
<html lang="en"     class="pb-page"  data-request-id="1d5a7481-e71b-4367-90b8-100e208da3f4"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.9b00797;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-5"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Opportunities and Challenges in Phenotypic Screening for Neurodegenerative Disease Research" /></meta><meta name="dc.Creator" content="Dean G.  Brown" /></meta><meta name="dc.Creator" content="Heike J.  Wobst" /></meta><meta name="dc.Description" content="Toxic misfolded proteins potentially underly many neurodegenerative diseases, but individual targets which regulate these proteins and their downstream detrimental effects are often unknown. Phenot..." /></meta><meta name="Description" content="Toxic misfolded proteins potentially underly many neurodegenerative diseases, but individual targets which regulate these proteins and their downstream detrimental effects are often unknown. Phenot..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 3, 2019" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00797" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00797" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00797" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00797" /></link>
        
    
    

<title>Opportunities and Challenges in Phenotypic Screening for Neurodegenerative Disease Research | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00797" /></meta><meta property="og:title" content="Opportunities and Challenges in Phenotypic Screening for Neurodegenerative Disease Research" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/large/jm-2019-00797d_0008.jpeg" /></meta><meta property="og:description" content="Toxic misfolded proteins potentially underly many neurodegenerative diseases, but individual targets which regulate these proteins and their downstream detrimental effects are often unknown. Phenotypic screening is an unbiased method to screen for novel targets and therapeutic molecules and span the range from primitive model organisms such as Sacchaomyces cerevisiae, which allow for high-throughput screening to patient-derived cell-lines that have a close connection to the disease biology but are limited in screening capacity. This perspective will review current phenotypic models, as well as the chemical screening strategies most often employed. Advances in in 3D cell cultures, high-content screens, robotic microscopy, CRISPR screening, and use of machine learning methods to process the enormous amount of data generated by these screens are certain to change the paradigm for phenotypic screening and will be discussed." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00797"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00797">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00797&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00797&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00797&amp;href=/doi/10.1021/acs.jmedchem.9b00797" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 1823-1840</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00940" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.9b01226" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Opportunities and Challenges in Phenotypic Screening for Neurodegenerative Disease Research</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Dean G. Brown</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dean G. Brown</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hit Discovery, Discovery Sciences, BioPharmaceuticals R&D, AstraZeneca, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</div></div><span class="conrtib-corresp">For D.G.B.: phone, +1-484-639-4153; E-mail, <a href="/cdn-cgi/l/email-protection#3d59585c53135f4f524a537d5c4e494f5c475853585e5c135e5250"><span class="__cf_email__" data-cfemail="2f4b4a4e41014d5d4058416f4e5c5b5d4e554a414a4c4e014c4042">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dean+G.++Brown">Dean G. Brown</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7130-3928" title="Orcid link">http://orcid.org/0000-0002-7130-3928</a></div></div></span></li><span class="comma-separator"> and </span><li><span><span class="hlFld-ContribAuthor">Heike J. Wobst</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Heike J. Wobst</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Neuroscience, BioPharmaceuticals R&D, AstraZeneca, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</div></div><span class="conrtib-corresp">For H.J.W.: phone, 1-617-892-1824; E-mail, <a href="/cdn-cgi/l/email-protection#226a474b49470c754d4051566243584c4757504d0c414d4f"><span class="__cf_email__" data-cfemail="3a725f53515f146d5558494e7a5b40545f4f485514595557">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Heike+J.++Wobst">Heike J. Wobst</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00797&amp;href=/doi/10.1021%2Facs.jmedchem.9b00797" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 1823–1840</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 3, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>15 May 2019</li><li><span class="item_label"><b>Published</b> online</span>3 July 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 March 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00797" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00797</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1823%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DDean%2BG.%2BBrown%252C%2BHeike%2BJ.%2BWobst%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D5%26contentID%3Dacs.jmedchem.9b00797%26title%3DOpportunities%2Band%2BChallenges%2Bin%2BPhenotypic%2BScreening%2Bfor%2BNeurodegenerative%2BDisease%2BResearch%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1840%26publicationDate%3DMarch%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00797"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4813</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">11</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00797" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Opportunities and Challenges in Phenotypic Screening for Neurodegenerative Disease Research&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Dean&quot;,&quot;last_name&quot;:&quot;G. Brown&quot;},{&quot;first_name&quot;:&quot;Heike&quot;,&quot;last_name&quot;:&quot;J. Wobst&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;03&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;1823-1840&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00797&quot;},&quot;abstract&quot;:&quot;Toxic misfolded proteins potentially underly many neurodegenerative diseases, but individual targets which regulate these proteins and their downstream detrimental effects are often unknown. Phenotypic screening is an unbiased method to screen for novel targets and therapeutic molecules and span the range from primitive model organisms such as Sacchaomyces cerevisiae, which allow for high-throughput screening to patient-derived cell-lines that have a close connection to the disease biology but are limited in screening capacity. This perspective will review current phenotypic models, as well as the chemical screening strategies most often employed. Advances in in 3D cell cultures, high-content screens, robotic microscopy, CRISPR screening, and use of machine learning methods to process the enormous amount of data generated by these screens are certain to change the paradigm for phenotypic screening and will be discussed.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00797&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00797" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00797&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00797" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00797&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00797" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00797&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00797&amp;href=/doi/10.1021/acs.jmedchem.9b00797" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00797" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00797" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00797%26sid%3Dliteratum%253Aachs%26pmid%3D31268707%26genre%3Darticle%26aulast%3DBrown%26date%3D2020%26atitle%3DOpportunities%2Band%2BChallenges%2Bin%2BPhenotypic%2BScreening%2Bfor%2BNeurodegenerative%2BDisease%2BResearch%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D5%26spage%3D1823%26epage%3D1840%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=290729" title="Nervous system diseases">Nervous system diseases</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/jmcmar.2020.63.issue-5/20200312/jmcmar.2020.63.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/medium/jm-2019-00797d_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/large/jm-2019-00797d_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00797&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Toxic misfolded proteins potentially underly many neurodegenerative diseases, but individual targets which regulate these proteins and their downstream detrimental effects are often unknown. Phenotypic screening is an unbiased method to screen for novel targets and therapeutic molecules and span the range from primitive model organisms such as <i>Sacchaomyces cerevisiae</i>, which allow for high-throughput screening to patient-derived cell-lines that have a close connection to the disease biology but are limited in screening capacity. This perspective will review current phenotypic models, as well as the chemical screening strategies most often employed. Advances in in 3D cell cultures, high-content screens, robotic microscopy, CRISPR screening, and use of machine learning methods to process the enormous amount of data generated by these screens are certain to change the paradigm for phenotypic screening and will be discussed.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/63/5"><issue-title>Women in Medicinal Chemistry</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35481" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35481" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Historically, phenotypic screening was the origin for many important classes of drugs.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Even though molecular target-based methods were widely adopted beginning in the late 1980s, phenotypic screening continues to provide productive routes for the discovery of new drugs. An analysis of first-in-class (FIC) drugs approved from 1998 to 2008 demonstrated that ∼62% (28 drugs) could be traced back to phenotypic screening approaches.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Of these 28 first-in-class drugs, seven were approved for infectious diseases. This trend has continued in the past decade, with several recent examples of new drugs discovered by phenotypic screening for bacterial and viral infections,<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> including the first novel drug for tuberculosis in more than 40 years.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> In addition, a significant number of drugs for central nervous system (CNS) indications have been discovered using phenotypic screening approaches. In the 1998–2008 analysis of FIC drugs, seven were approved for CNS indications. These include aripiprazole (schizophrenia), varenicline (smoking cessation), levetiracetam (antiepileptic), and refinamide (antiepileptic). A large number of CNS drugs (12) were discovered as part of an effort to identify novel antiepileptic drugs through the National Institutes of Health (NIH) drug-screening program Anticonvulsant Drug Development Program, by screening in rodent models of seizure. Examples from this effort include lamotrigine, gabapentin, and pregabalin.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">Phenotypic screens are attractive because, if properly designed, they may represent a more holistic model to the biology of interest as compared to a biochemical screen (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Furthermore, phenotypic screens are often the gateway to novel target-based drug discovery. Examples in this category include HDAC inhibitors such a vorinostat,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> mTOR inhibitors such as rapamycin,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> BET bromodomain inhibitors,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and inhibitors of the cholesterol transporter Niemann–Pick C1-Like 1 (NPC1L1).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> These targets were all discovered using phenotypic screens and subsequently molecular target-based approaches were enabled.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/medium/jm-2019-00797d_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/large/jm-2019-00797d_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic of phenotypic screening approaches. An appropriate cellular screening model is chosen according to the type of screening library and the phenotype of interest. Screening libraries can be divided into three classes: small molecule compound libraries, knockdown or deletion libraries, or overexpression libraries. Knockdown or deletion libraries contain either siRNA, shRNA, or gRNA for CRISPR screens. For CRISPR screens, the chosen cell type requires expression of Cas9 endonuclease. Cells can also express a disease gene that causes the phenotype of interest (e.g., overexpression of tau, α-synuclein, or Htt fragments) and/or a reporter gene (e.g., GFP-labeled LC3 to visualize autophagic flux, a minigene to assess splicing, a luciferase-labeled transgene to visualize up-or downregulation of protein levels).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/large/jm-2019-00797d_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00797&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Neurodegenerative diseases such as Alzheimer’s disease, Parkinson’s disease, Huntington’s disease, and frontotemporal dementia (FTD) are debilitating disorders, and there are currently no disease-modifying therapies available to patients. The World Health Organization has predicted that neurodegenerative diseases will become the second leading cause of death in 20 years’ time, after cardiovascular diseases.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Consequently, there is an urgent need to find novel drugs which can slow down or arrest the course of these disorders. Many of these diseases share common mechanisms and phenotypes (e.g., aberrant protein aggregation, impaired autophagic flux, excitotoxicity, mitochondrial dysfunction).<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(11−13)</a> Phenotypic screens are used to interrogate these and other disease mechanisms in a variety of model systems. Here we review current screening strategies in neurodegeneration, along with specific examples of compounds and novel targets discovered with these methods. We also discuss best practices for developing a robust screening cascade, along with methods for hit confirmation and target validation.</div><div id="sec1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Nonmammalian Model Organisms for Phenotypic Small Molecule Screening</h3><div class="NLM_p">Disease models have been generated using nonmammalian organisms such as <i>Saccharomyces cerevisiae</i> (yeast),<a onclick="showRef(event, 'ref14 ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref17 ref18">(14−18)</a><i>Caenorhabditis elegans</i> (nematode worm),<a onclick="showRef(event, 'ref19 ref20 ref21 ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21 ref22 ref23 ref24">(19−24)</a><i>Drosophila melanogaster</i> (fruit fly),<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> and <i>Danio rerio</i> (zebrafish)<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> for a number of neurodegenerative diseases including Alzheimer’s disease, Parkinson’s disease, Huntington’s disease, and amyotrophic lateral sclerosis (ALS) (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Some of these model organisms offer the advantage of high screening capacity and well-studied genomes. Yeast, for example, have a simple and compact genome of about 6K genes with few and often nonessential introns.<a onclick="showRef(event, 'ref14 ref29'); return false;" href="javascript:void(0);" class="ref ref14 ref29">(14,29)</a> However, only about a third of yeast genes have a human orthologue.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Furthermore, sequence identity between human-yeast orthologues varies greatly depending on the gene in question, ranging from 9–92% and averaging 32% across the genome.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Despite the limited overlap between human and yeast genome, important yeast genes can be connected to human disease, a fact that has been exploited in a number of small molecule screens. In one recent example, yeast were engineered to overexpress the Parkinson’s disease gene α-synuclein, which causes a toxicity phenotype, and a compound library was screened for suppressors of this toxic phenotype.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Using spontaneous drug-resistant mutants and genetics, one of the chemical series found in the yeast screen allowed the research team to identify the human orthologue target stearoyl-CoA desaturase. Potent compounds to the human target were synthesized (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, <b>1</b>, YTZ-465) and were shown to protect against α-synuclein toxicity in induced pluripotent stem cells (iPSCs). Another example from this screen provided an unknown mechanism which reverses endoplasmic reticulum (ER)-to-Golgi trafficking defects and mitochondrial dysfunction (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, <b>2</b>).<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Model Organisms Used in Phenotypic Screening</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col /></col><col /></col><col /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">species</th><th class="colsep0 rowsep0" align="center">estimated number of protein-coding genes</th><th class="colsep0 rowsep0" align="center">protein-coding genes with orthologues in <i>H. sapiens</i> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0"><i>S. cerevisiae</i> (yeast)</td><td class="colsep0 rowsep0">∼6200<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></td><td class="colsep0 rowsep0">∼37<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0"><i>C. elegans</i> (nematode)</td><td class="colsep0 rowsep0">∼20250<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></td><td class="colsep0 rowsep0">∼38<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0"><i>D. melanogaster</i> (fruit fly)</td><td class="colsep0 rowsep0">∼14000<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></td><td class="colsep0 rowsep0">∼46<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0"><i>D. rerio</i> (zebrafish)</td><td class="colsep0 rowsep0">∼26000<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></td><td class="colsep0 rowsep0">∼70<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0"><i>M. musculus</i> (mouse)</td><td class="colsep0 rowsep0">∼25000<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></td><td class="colsep0 rowsep0">∼99<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></td></tr></tbody></table></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Summary of Phenotypic Screens in Neurodegeneration<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/medium/jm-2019-00797d_0004.gif" alt="" id="gr2" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/medium/jm-2019-00797d_0005.gif" alt="" id="GRAPHIC-d7e412-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/medium/jm-2019-00797d_0006.gif" alt="" id="GRAPHIC-d7e413-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/medium/jm-2019-00797d_0007.gif" alt="" id="GRAPHIC-d7e414-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">(1) Background pharmacology of compound is the known pharmacology of the hit at the time of the screen. (2) MOA in phenotypic assay is the mechanism elucidated in the publication. (3) A set of 1K compounds was selected based on complexity and drug-like properties.</p></div></div><div></div></div><div class="NLM_p">The nematode <i>C. elegans</i> has been used to model several neurodegenerative diseases such as Alzheimer’s disease, FTD, ALS, and Parkinson’s disease.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Key advantages of <i>C. elegans</i> are its transparent body, simple neuronal circuit, well-understood genetics, and rapid reproductive cycle. Although widely used, very few examples are reported in the literature of chemical screens and hits and subsequent deconvolution of targets. One example from Mondal et al. reported on the screen of 1000 compounds from a library of Food and Drug Administration (FDA)-approved drugs in a <i>C. elegans</i> model of poly glutamine (polyQ) aggregation.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> PolyQ diseases such as Huntington’s disease and several forms of spinocerebellar ataxias are a class of neurodegenerative disorders characterized by expansion of a CAG trinucleotide repeat sequence encoding a polyQ repeat stretch. While in healthy individuals these CAG repeats are usually 40 or less (dependent on the gene and the individual), in the disease state the repeat length can reach >100. In the case of Huntingtin, the gene associated with Huntington’s disease, the expanded polyQ stretch makes the resulting protein aggregation-prone. In this screen dronedarone <b>3</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), an antiarrhythmic, was found to reduce polyQ aggregation by 46% at a 50 μM concentration. In another example, Yao et al. used a <i>C. elegans</i> model of Parkinson’s disease to demonstrate that the leucine-rich repeat kinase 2 (LRRK2) inhibitor LRRK2-IN1 <b>4</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) rescues R1441C- and G2019S-LRRK2-induced dopaminergic deficits and neurodegeneration.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> This example serves to illustrate that the <i>C. elegans</i> model can recapitulate known human target biology using a well-known Parkinson’s disease target.</div><div class="NLM_p">In another recent paper, Halliday et al. screened 1040 compounds from the NINDS (National Institute for Neurological Disorders and Stroke) library containing 75% FDA-approved drugs in a <i>C. elegans</i> model of tunicamycin-induced developmental delay.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Tunicamycin is a nucleoside antibiotic that activates the unfolded protein response (UPR). Two compounds were identified, trazadone and dibenzoylmethane, which inhibit UPR-induced eIF2α-P signaling, resulting in normalization of protein synthesis rates (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, <b>5</b> and <b>6</b>). Both compounds also demonstrated neuroprotection in prion-infected mice as well as a P301L-tau model of FTD, thus highlighting the translatability of certain pathways and genes in <i>C. elegans</i> into mammalian models of neurodegeneration.</div><div class="NLM_p">Zebrafish (<i>D. rerio</i>) is another attractive model organism to study neurodegenerative disorders. Zebrafish are easy to maintain and have a relatively short generation time but crucially are vertebrates. Furthermore, zebrafish larvae are transparent, allowing for live imaging using fluorescent labels.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> A recent review summarized more than 65 chemical screens in zebrafish, but none were aimed at identifying drugs for neurodegenerative diseases.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> However, zebrafish behavioral screens have been used to identify novel neuroactive drugs based on clustering of elicited responses. In two published papers, stereotypical quantitative behavioral features were extracted from two distinct locomotor paradigms and thousands of compounds were then tested for their effects on these behavioral features.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Compounds were clustered according to their “fingerprint” or “barcode”, the multidimensional impact of the compounds on these behavioral features. In these examples, fingerprints of behavioral activity can be ascribed to known drugs and then used to match compounds from a drug screen to infer the mechanism of action. While not widely used as a screening model for neurodegeneration, zebrafish have been used to validate compounds identified in cellular screens. For example, Corman et al. demonstrated that the BET bromodomain inhibitor PFI-1 (<b>7</b>), identified in a cellular toxicity screen, increases survival in developing zebrafish exposed to PR20 peptide, a frequently used stressor to elicit toxicity associated with C9ORF72-ALS.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></div><div class="NLM_p">The fruit fly <i>D. melanogaster</i> has become a useful translational model for neurological diseases and in particular for neurodegenerative diseases. The advantages of this model organism are its short life span, inexpensive maintenance, and ready access to transgenic and knockout lines for the use of genetic modifier screens.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> In one recent study, McGurk et al. performed a genetic modifier screen in flies expressing the ALS-associated human transactive response DNA binding protein 43 kDa (TDP-43), which results in an eye degeneration phenotype.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> They identified tankyrase (a member of the Poly-ADP ribose polymerization or “PARP” family), which attenuated TDP-43 associated toxicity in the TDP-43 fly model. The team further established that tankyrase-1/2 inhibitors such as XAV939 reduced the formation of arsenite-induced cytoplasmic TDP-43 foci in mammalian cells (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, <b>8</b>).</div><div class="NLM_p">While <i>Drosophila</i> is a widely used model organism for genetic screens in neurodegenerative disease paradigms (for an overview, see Lenz et al.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a>), compound screens have not been frequently reported. One example was published by Lawal et al., where a set of ∼1K known drugs were screened against a mutant vesicular monoamine transporter (VMAT) strain to identify potentiators of neurotransmitter release as potential therapeutic agents for the treatment of Parkinson’s disease and depression. The screen yielded several hits, including the chemotherapy agent dacarbazine <b>9</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> In another study, a Drosophila model of fragile X-associated tremor/ataxia syndrome (FXTAS) was used to test 2000 FDA-approved drugs and natural products (Spectrum Collection) to identify compounds that rescue the lethality phenotype induced by expression of CGG repeats, a mutation found in the FMR1 gene in affected patients.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Among the identified hits were xylazine <b>10</b> and fluocinolone acetonide <b>11</b>, a phospholipase A2 inhibitor (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_p last">Although rare, mouse screening models are also sometimes employed. One recent example was reported by Pieper et al., who identified small molecules capable of enhancing neuron formation in the hippocampus of adult mice in vivo. A set of 1K molecules were screened, and eight were found to enhance neuron formation in the subgranular zone of the dentate gyrus, with the most promising candidate P7C3 <b>12</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Furthermore, advances in rodent behavioral profiling platforms allow for test compounds to be compared to known drugs and tool compounds but has been mostly applied to identifying novel psychiatric drugs such as the recent SEP-363856 for schizophrenia.<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> These profiling models have also been proposed for neurodegenerative diseases such as Huntington’s disease, where behavioral phenotypes such as locomotor deficits, cognitive impairment, or circadian rhythm phenotypes can be monitored in high capacity.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></div></div><div id="sec1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Immortalized Cell Lines</h3><div class="NLM_p">Perhaps the historical mainstay of neuroscience-driven phenotypic screens is based on the use of immortalized cell lines. Immortalized cells are often considered a starting point for high-throughput phenotypic screens, as cell lines are widely commercially available, do not typically require special skills to culture and are inexpensive to maintain (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Furthermore, many cell lines are of human origin, thus ruling out any species effects that could impede translation of the results from the cellular screen.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/medium/jm-2019-00797d_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/large/jm-2019-00797d_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Overview of cell models used in phenotypic screening for neurodegeneration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/large/jm-2019-00797d_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00797&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Published screening approaches can be divided into several classes: (1) screens for molecules that rescue a toxicity phenotype, (2) screens for modulators of a specific pathway, and (3) screens for modulators of a specific target gene or protein. Screens for compounds which rescue a toxic phenotype are an attractive model in neurodegeneration research. In neurodegenerative diseases such as Parkinson’s disease, ALS, and FTD, disease-associated proteins misfold and aggregate, causing toxicity through either gain-of-function or loss-of-function mechanisms or a combination of both. In one recent paper, Corman et al. tested >4K compounds to identify suppressors of PR20 toxicity in U-2 OS cells.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> PR20 is a 20-mer repeat of proline and arginine amino acids and is frequently used to elicit toxicity in C9ORF72-ALS and -FTD models. Hexanucleotide repeat expansions in C9ORF72 are associated with ALS and FTD. These expansions lead to the non-ATG-dependent translation of five dipeptide repeats (GP, GA, PA, GR, and PR), of which the PR dipeptide repeat is toxic in vitro.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> The screen identified the histone deacetylase (HDAC) inhibitor sodium phenylbutyrate as well as the BET bromodomain inhibitors PFI-1 <b>7</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) and bromosporine.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Of these, both PFI-1 and sodium phenylbutyrate also improved survival in zebrafish treated with PR20 peptide. The authors were able to demonstrate that the BET inhibitors were responsible for rescue of nucleolar stress caused by PR20 as well as actinomycin D, suggesting a very general effect of these inhibitors. It should be noted that BET bromodomain inhibitors, and PFI-1 in particular, are active in a variety of phenotypic screens such as in assays measuring reactivation of HIV-1 latency,<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> promotion of HSV infection,<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> reduction of human collagen <i>Col2a1</i> gene expression,<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> and reduction of inflammation as measured by etoposide release of IL-6 and IL-8.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> This background information may help underscore that compounds which interfere with BET domains may cause gross general pharmacological effects that may confound the interpretation of a variety of phenotypic screens.</div><div class="NLM_p">To screen for compounds that rescue toxicity induced by viral overexpression of wild-type α-synuclein in a cellular model of Parkinson’s disease, Höllerhage et al. utilized Lund human mesencephalic (LUHMES) cells, which can be differentiated into dopamine-like cells.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> A set of 1600 FDA-approved drugs were screened at concentrations of 3 and 10 μM in this model, and cell death was quantified by high content imaging. Of the identified 53 hits, 22 were active at the lower concentration and nine at both concentrations. This disconnect between screening concentrations and activity can be due to the following factors: (1) natural variability in these types of assays, (2) confounding off-target activity at the higher concentrations such as cytotoxicity, or (3) compound-related issues such as poor solubility. Only dipyridamole <b>13</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), a nonspecific phosphodiesterase (PDE) inhibitor, was active at both concentrations after triplicate experiments.</div><div class="NLM_p">Pathway-specific phenotypic screens can likewise be done in immortalized cell lines, where one is able to understand and modulate a specific pathway. One of the more interesting tool molecules to originate from a pathway-specific phenotypic screen is ISRIB <b>14</b> (“integrated stress response inhibitor)”, which was discovered by screening ∼100K molecules in modified HEK293T cells treated with the ER stressor thapsigargin to trigger a luciferase end-point of PERK activation (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> ISRIB was identified as a potent inhibitor of the PERK activation pathway.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Deconvolution studies identified ISRIB as a stabilizer of activated eIF2b dimers with the potential to improve cognitive deficits as was demonstrated by improved memory consolidation in rodents.<a onclick="showRef(event, 'ref51 ref71'); return false;" href="javascript:void(0);" class="ref ref51 ref71">(51,71)</a> ISRIB has become one of the leading tool molecules in the study of UPR and stress response in neurological disease; however, very poor solubility may confound interpretation of results and limit its use as a tool compound.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> Another very notable example of a major advance in the field using immortalized cell lines was the discovery of small molecules which could correct mis-splicing in survival motor neuron 2 (SMN2) such as RG7800 <b>15</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) as a treatment for spinal muscular atrophy (SMA).<a onclick="showRef(event, 'ref52 ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref52 ref53 ref54">(52−54)</a> This was discovered in a high-throughput chemical screen in HEK293 cells expressing an SMN2 mini-gene. The mechanism of action was ultimately elucidated and determined to be stabilization of the SMN2 pre-mRNA interaction with U1 small nuclear ribonucleic protein, resulting in enhanced production of full length SMN.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a></div><div class="NLM_p last">Because misfolding and aggregation of disease-associated proteins is thought to be causative of, or significantly contribute to, neurodegenerative processes in diseases such as ALS, Parkinson’s disease, and FTD, correcting or preventing these misfolding events is a widely investigated therapeutic approach. In one such instance, Pickhardt et al. tested a chemogenomic library of 1649 molecules in N2a cells expressing an aggregation-prone version of the repeat domain of tau.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> Aggregated tau protein is the common hallmark of a number of neurodegenerative diseases termed tauopathies, including Alzheimer’s disease, FTD-tau, progressive supranuclear palsy, and corticobasal degeneration.<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> The screen identified 18 inhibitors of tau aggregation, including several HDAC inhibitors and p38 MAPK inhibitors such as LY2228820 <b>16</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a></div></div><div id="sec1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Primary Cell Assays</h3><div class="NLM_p">Primary neuronal or other CNS cell types can provide a more physiological setting for compound screens. For example, primary neurons do not divide, differentiate, and mature over time in culture and form synaptic connections. However, large-scale small molecule screens are hampered by several factors. Primary cultures require live animals as well as training to obtain the cells.  It is also more difficult to produce sufficient cell numbers as the cultures require differentiation over many days, and reagents are more expensive (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Furthermore, primary CNS cells are obtained from mice or rats, which can introduce species bias or result in species-specific effects. However, despite these limitations, a number of small-scale screens have yielded interesting chemical matter. In one such screen, a NINDS library of ∼1K known bioactive molecules and FDA-approved drugs was tested to identify inhibitors of Aβ-induced neurite degeneration in mouse cortical neurons.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Of the 36 identified hits, several belonged to the class of nonsteroidal anti-inflammatory drugs (NSAID) including metacetamol <b>17</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). In another instance, Schmidt et al. synthesized a small collection of quinoline-derived compounds and tested these for potential neuroprotective and neuritogenic effects.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> The compounds were designed based on lembehyne A, a compound known for its neurotrophic activity, and were screened in both spontaneous and -methyl-4-phenylpyridinium (MPP+)-induced Parkinson’s disease models of mesencephalic dopaminergic neuron degeneration. Neuroprotection was assessed by quantifying the number of tyrosine hydroxylase (TH)-positive dopaminergic neurons, while neuritogenesis was assessed by quantifying neurite length per dopaminergic neuron. An example hit from this screen <b>18</b> is exemplified in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>.</div><div class="NLM_p">Huang et al. sought to identify compounds for the possible treatment of Angelman syndrome, a neurodevelopmental disorder.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Angelman syndrome is caused by mutations or deletion of the maternal allele for ubiquitin protein ligase E3A (UBE3A). In neurons, the paternal allele is epigenetically silenced, while the maternal allele is expressed. A screen was conducted using 2306 compounds sourced from several libraries for hits that “unsilence” the expression of the paternal UBE3A allele. For this approach, cortical neurons were cultured from mice harboring a paternal UBE3A-YFP reporter and upregulation of this reporter was measured using high content imaging. The initial screen yielded only one hit, the FDA-approved topoisomerase type I inhibitor irinotecan <b>19</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Further structure–activity relationship (SAR) analysis and testing of chemically distinct topoisomerase type I and II inhibitors led to the identification of 11 additional type I and 4 type II inhibitors. The irinotecan analogue topotecan was also found to unsilence UBE3A in several brain regions of <i>Ube3a</i><sup><i>m</i>+/<i>pYFP</i></sup> mice.</div><div class="NLM_p last">Rubinstein et al. have reported on a screen of L-type calcium channel blockers in primary neurons expressing the autophagy marker construct mRFP-GFP-LC3.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> The screening hit felodipine <b>20</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) also demonstrated activity in a tau zebrafish model as well as in A53T α-synuclein-expressing iPSC-derived neurons and a transgenic mouse model with the same mutation.</div></div><div id="sec1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> iPSC Lines</h3><div class="NLM_p">Given the limitations of scalability with primary neuronal cells, most phenotypic screens to date are carried out in immortalized cell lines. Immortalized cells are a replenishable and scalable resource, generally easy to maintain, and widely available (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). However, neuronal cell lines such as SH-SY5Y cells, N2a cells, or NSC-34 cells do not fully recapitulate the features of neuronal cells, such as generation of spontaneous action potentials, formation of mature synaptic connections, or expression of relevant neurotransmitter receptors. Furthermore, immortalized cell models usually rely on expression of a transgene or addition of a stressor and are thus not suitable to mimic the pathobiology of sporadic disease. In this regard, induced pluripotent stem cells (iPSCs) have revolutionized the field. iPSCs, once generated by reprogramming somatic cells, can be indefinitely replenished and can be differentiated into many different cell types while retaining the genetic background of the donor or patient, thus making them a valuable resource for high content screening. However, like every other type of cell or model organism amenable to high throughput screening, iPSC technology comes with a number of caveats. The process of differentiation can be time-consuming, expensive, and produce variable results. Furthermore, in the case of neuronal differentiation, cells may resemble neurons by morphology but not have the electrophysiological or electrochemical characteristics of a mature neuron. Finally, the observed phenotype may arise as a result of the specific genetic background of the donor, thus making assay optimization and validation in multiple lines from different donors necessary if modeling sporadic disease or require the generation of isogenic control lines when modeling familial disease.</div><div class="NLM_p">To date, a number of small molecule screens have been performed in iPSCs to identify novel drugs for neurodegenerative diseases. In one such instance, Fujimori et al. performed a drug screen of >1200 FDA-approved compounds using <i>Fused In Sarcoma</i> (FUS)- and TDP-43-iPSC-derived motor neurons by assessing multiple phenotypes with high content imaging including neurite length, stress granule formation, FUS/TDP-43 aggregation, and cell death.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> The screen identified the D2 agonist ropinirole <b>21</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) as a top modulator of these phenotypes. The efficacy of ropinirole was further demonstrated in 24 sporadic ALS-derived iPSC lines. Ropinirole was shown to reduce apoptosis, reactive oxygen species, neuronal atrophy, and protein aggregation in several of the sporadic lines. The authors demonstrated that ropinirole may be working in these models through both a dopamine and a dopamine-independent pathway, as evidenced by lack of complete reversal of effects with dopamine antagonists.</div><div class="NLM_p last">In a screen to identify compounds that lower levels of tau, Wang et al. tested the “Library of Pharmacologically Active Compounds” (LOPAC), a commercial set of 1280 bioactive molecules, in iPSC-derived glutamatergic neurons.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> The adrenergic receptor agonists moxonidine <b>22</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) and metaproterenol <b>23</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) were shown to lower tau in a dose-dependent manner. In another paper, familial Alzheimer’s disease (FAD)-derived iPSCs with the PSEN1 G384A mutation were differentiated into cortical neurons and >1200 bioactive molecules were tested to identify Aβ-lowering compounds.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> Of the 27 hits, six leading compounds were identified and tested in all possible combinations. A cocktail of bromocriptine <b>24</b>, cromolyn <b>25</b>, and topiramate <b>26</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) was further validated in four additional FAD lines, four sporadic Alzheimer’s lines, and six control lines. The authors did not attempt to dissect and understand the basis of the synergistic effect. Although this result is quite interesting, it will be a challenge to carry this combination forward in more advanced models because the efficacy will greatly depend on the pharmacokinetics, exposure, and target engagement of each compound within every model.</div></div><div id="sec1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> 3D Cultures</h3><div class="NLM_p">Currently, most cell-based high throughput compound screens are carried out using a single cell type grown in monolayers. However, these nonphysiological conditions do not reproduce the highly complex architecture, cell-to-cell interactions and microenvironment cells share within the context of a tissue. Thus, complex 3D culture systems, organoids, organs-on-chips, and hydrogel scaffolds are being developed and refined to more accurately allow cells to “behave” as they would in vivo (for a recent review on 3D cultures in drug discovery, see Fang and Eglen<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a>). In the future, complex 3D culture systems and organoids may be more widely adopted in phenotypic screening. However, to the best of our knowledge, no low or high throughput small molecule screens to investigate neurodegenerative phenotypes using 3D culture systems have been performed, reflecting the novelty of the approach, the technical challenges, and the high costs associated with 3D cultures. However, several publications detail the effects of specifically selected compounds on neurodegeneration-associated phenotypes in 3D culture systems. For example, a recently developed “ALS-on-a-chip” platform was developed to test the effects of two drugs, the mammalian target of rapamycin (mTOR) inhibitor rapamycin <b>27</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) and the Src/c-Abl kinase inhibitor bosutinib <b>28</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The system consists of a three-dimensional iPSC-derived muscle fiber bundle as well as iPSC-derived motor neurons obtained from a sporadic ALS (sALS) patient in a microfluidic device.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> Neural activity was optogenetically induced to elicit controlled muscle fiber contractions. Cotreatment with rapamycin <b>27</b> and bosutinib <b>28</b> increased the force of muscle contractions and decreased levels of caspase 3/7.</div><div class="NLM_p">In a 3D triculture system of neurons, astrocytes, and microglia, Parker et al. modeled neurodegeneration and neuroinflammation associated with Alzheimer’s disease.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> Cells were derived from neural progenitor cells (NPCs) overexpressing amyloid precursor protein (APP) with the Swedish and London mutations. These neuron/astrocyte cocultures secreted more Aβ<sub>40</sub> and Aβ<sub>42</sub>, phospho-tau, and pro-inflammatory cytokines compared to control cultures. Microglia added to the microfluidic platform were recruited to form 3D cocultures and showed increased activation, leading to neuronal death.</div><div class="NLM_p">A number of publications have described protocols to generate brain and spinal organoids.<a onclick="showRef(event, 'ref64 ref77 ref78 ref79 ref80'); return false;" href="javascript:void(0);" class="ref ref64 ref77 ref78 ref79 ref80">(64,77−80)</a> Brain organoids are derived from pluripotent or induced pluripotent stem cells that self-assemble into 3D neuronal structures similar to certain brain regions. Organoids can be used to study a variety of diseases such as neurodevelopmental disorders, epilepsy, or neurodegenerative diseases. For example, a recent study showed that 110-day old brain organoids grown from iPSCs derived from a patient with familial Alzheimer’s disease recapitulated features of the disease such as increased Aβ<sub>42</sub>/Aβ<sub>40</sub> production, hyperphosphorylated tau, and apoptotic cell death.<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> In another report, Ng et al. described a spinal organoid model of SMA using differentiated patient derived pluripotent stem cells.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> These organoids demonstrated a reduction in motor neuron survival at day 35 compared to wild-type cells. Also, of note, the cells expressed high levels of cell cycle-dependent kinases (CDK) and cyclins, which led them to hypothesize that CDK inhibitors would be effective in rescue of the neurodegeneration. By screening a focused set of CDK inhibitors, they subsequently identified the CDK4/6 inhibitor PD 0332991 <b>29</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_p last">However, current limitations of organoids include the lack of vasculature, which is indispensable to overcome limits in the size of the assemblies as well as cell death due to lack of oxygen and nutrients, difficulties in establishing mature synaptic connections and a lack of diversity in the number of different cell types that can be incorporated into current models.<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a></div></div><div id="sec1_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> High-Content Imaging Methods</h3><div class="NLM_p last">Many phenotypic screens rely on high content imaging technologies. Fluorescent protein tags as well as fluorescent dyes and probes allow diverse cellular processes to be visualized, such as autophagy, mitochondrial function, neuronal activity, proteasomal degradation, or protein translation. Robotic microscopy platforms have further enabled the monitoring of phenotypes in a longitudinal manner. In this setup, 96- or 384-well plates are automatically moved from incubator to microscope and back, and images of the same cell can be taken over days without human intervention.<a onclick="showRef(event, 'ref83 ref84'); return false;" href="javascript:void(0);" class="ref ref83 ref84">(83,84)</a> Because of the volume and complexity of the data generated in these experiments, manual annotation is often impossible. As a result, machine-learning methods have been developed to track individual cells and identify meaningful patterns and cellular features. For example, Finkbeiner et al. have described a deep neural network model to process and interpret low-quality and out-of-focus images which historically would require human intervention, enabling full automation of image analysis.<a onclick="showRef(event, 'ref85 ref86'); return false;" href="javascript:void(0);" class="ref ref85 ref86">(85,86)</a> Data is then analyzed with the same statistical methods used in clinical trials, as each cell is viewed as an “individual”.<a onclick="showRef(event, 'ref84 ref87'); return false;" href="javascript:void(0);" class="ref ref84 ref87">(84,87)</a> In one example of such an approach to study small molecule pharmacological effects, Tsetkov et al. demonstrated that the Akt inhibitor 10-NCP <b>30</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) induces autophagy. Robotic microscopy and longitudinal tracking of individual neurons revealed that the compound improved survival in a neuronal model of Huntington’s disease and increased the fraction of neurons that clear inclusion bodies formed from the mutant Huntingtin exon 1 fragment.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> In another example, using a method termed “in silico labeling”, Christiansen et al. were able to predict fluorescent labels, including cell type and cell death, from transmitted light images of fixed as well as live cells.<a onclick="showRef(event, 'ref88'); return false;" href="javascript:void(0);" class="ref ref88">(88)</a> With such an approach, expensive staining or labeling techniques with limited multiplexing possibilities and the potential to introduce labeling artifacts may in the future be eschewed. Automated platforms of the kind described above, that can perform phenotypic screens, process the raw data and analyze it using deep neural network algorithms, are becoming ever more sophisticated, and will increase both the capacity of the screens as well as the quality and size of the data collected.</div></div><div id="sec1_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Genetic Screens</h3><div class="NLM_p">Unbiased phenotypic screens using small molecule libraries can be used to uncover new targets. This approach, however, can be fraught with difficulties. For example, chemical libraries may contain a large fraction of molecules without a known target annotation or with multiple annotated targets, thus making for complex deconvolution. They also do not cover the classically “undruggable” part of the genome. Off-target effects, rather than the effect on the annotated target, can be responsible for rescuing the observed phenotype, which has to be ascertained by genetic studies. Furthermore, small molecules might lack potency in the species of the cell line or organism chosen. These pitfalls can be avoided by utilizing genetic screens to identify suppressors or enhancers of the observed phenotype. In such screens, a fraction or the entirety of the annotated genome is either overexpressed or knocked down/out using approaches such as siRNA, yeast deletion strains, or genome-editing strategies.</div><div class="NLM_p">Using an siRNA-based assay to knock down the kinome of 719 kinases, Mason et al. identified 24 kinases that regulate levels of the growth factor Progranulin (PGRN).<a onclick="showRef(event, 'ref89'); return false;" href="javascript:void(0);" class="ref ref89">(89)</a> Mutations in the gene encoding progranulin, GRN, are a cause of FTD as a result of haploinsufficiency. Thus, upregulation of progranulin could be of potential therapeutic benefit. Among the top gene hits were the Parkinson’s disease-associated LRRK2 and the necroptosis regulator receptor-interacting serine/threonine-protein kinase 1 (RIPK1).</div><div class="NLM_p">Genetic modifiers of TDP-43 toxicity have been discovered in plasmid overexpression as well as deletion screens in yeast.<a onclick="showRef(event, 'ref90 ref91 ref92 ref93'); return false;" href="javascript:void(0);" class="ref ref90 ref91 ref92 ref93">(90−93)</a> In one study, 5500 yeast genes were individually transformed into a yeast strain that overexpresses TDP-43 under the control of a galactose-inducible promoter.<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> Of the 40 hit genes identified, 13 suppressed and 27 enhanced TDP-43 toxicity when overexpressed. One enhancer of toxicity was Pab1-binding protein 1 (PBP1), an orthologue of the human <i>Ataxin-2</i> (ATXN2) gene. A subsequently published paper showed that genetic or antisense oligonucleotide (ASO)-based suppression of ataxin-2 effectively improved motor function and extended lifespan in a TDP-43 mouse model.<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> Employing the opposite approach, Armarkola et al. mated a galactose-inducible TDP-43 strain to the yeast haploid deletion collection of nonessential genes to screen for genes that suppress or enhance toxicity of TDP-43 when deleted.<a onclick="showRef(event, 'ref90'); return false;" href="javascript:void(0);" class="ref ref90">(90)</a> This screen identified six enhancers and eight suppressors of toxicity. Among the suppressors were pbp1, confirming the results from Elden et al., as well as RNA lariat debranching enzyme 1 (dbr1), which also rescued TDP-43 toxicity in a primary neuronal assay assessed by robotic microscopy.</div><div class="NLM_p">Genome-wide loss of function screens utilize either RNA interference (RNAi) or genome editing strategies. While an invaluable technology to this date, RNAi-based approaches have a number of caveats: (1) genes are post-transcriptionally knocked down rather than knocked out, thus, there can still be residual gene expression, (2) the RNAi machinery works predominantly in the cytoplasm, making it difficult to target nuclear RNA species such as long noncoding RNA (lncRNA), and (3) RNA transcripts with limited complementarity can be recognized by the RNAi probe, causing off-target knockdown.<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a> Genome editing strategies can overcome some of these limitations, and the advent of clustered regularly interspaced palindromic repeat (CRISPR)/Cas9 technology served as a watershed moment for the field. In this genome editing method, the endonuclease Cas9 binds to a guide RNA (gRNA) encoding a sequence complementary to a genomic sequence of interest. Having been “guided” by the gRNA to the genomic target sequence, Cas9 induces the formation of a double-strand break, which is repaired by nonhomologous end joining, resulting in insertions or deletions which cause a loss of gene function.<a onclick="showRef(event, 'ref95 ref96'); return false;" href="javascript:void(0);" class="ref ref95 ref96">(95,96)</a> CRISPR knockout screens are now routinely used to identify modifiers of a given phenotype of interest genome-wide. In one recent example, Kraemer et al. performed a genome-wide knockout screen of ∼20500 genes with a lentiviral sgRNA library covering 10 sgRNAs per gene to identify modifiers of C9ORF72 dipeptide repeat (DPR) toxicity.<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a> In the screen, human K562 cells stably expressing Cas9 were infected with a pooled genome-wide lentiviral sgRNA library. After infection, the cell population was split and half the cells were either left untreated or treated with either toxic PR20 or GR20 at concentrations required to kill 50% of cells. Deep sequencing was used to identify sgRNAs that were enriched (i.e., the knockout was protective) or depleted (i.e., the knockout sensitized cells to the toxic effects of the DPRs) in the pooled cell population. The screens revealed 215 and 387 modifiers of PR20 and GR20 toxicity, respectively. The top ∼200 modifiers of PR20 toxicity in K562 cells were subsequently tested in primary mouse cortical neurons. The two strongest knockout suppressors of toxicity were Rab7 and Tmx2.<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a></div><div class="NLM_p last">Genetic screens for suppressors or enhancers of phenotypic screens in neurodegeneration hold promising potential to identify pathways and genes which would be difficult or impossible to identify using annotated chemical screening decks. The most ambitious goal of these screens is to identify new targets which can be modified with genetic or small molecule therapeutic approaches and slow down, halt, or reverse the disease. However, the output from these screens is simply a list of genes, most of which will not translate into a viable therapeutic target. For example, the observed modulatory effect may be specific to the cell type used in the screen or in the case of knockdown or knockout screens may result from off-target effects rather than from the intended target. Furthermore, a genetic knockdown or knockout of a classically druggable target may have detrimental effects that a small molecule inhibitor may not. It is, for example, conceivable that inhibiting a kinase chemically may have a protective effect in the chosen cell assay while knocking out the same kinase may have a detrimental effect due to possible additional scaffolding functions of the enzyme. Last, many of the identified genes may encode classically undruggable targets. New therapeutic modalities such as CRISPR, siRNAs, ASOs, or anticalins may overcome this particular barrier in theory and represent a rapidly growing area of drug discovery and development. For example, one ASO (inotersen<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a>) and one siRNA (patisiran<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a>) were both approved by the FDA in 2018, with many more in the pipeline.<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> Connecting key genes in pathways and understanding cellular and disease context are critical steps in the field, which will lead to insight into cellular responses to disease, important biomarkers, and hopefully novel therapeutic approaches.</div></div><div id="sec1_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Screening Collections</h3><div id="sec1_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> Drug and Drug-Like Libraries</h4><div class="NLM_p last">There are two key decisions that a project team will face when deciding to construct a phenotypic screen. One of those decisions is the selection of a primary screening model and subsequent orthogonal models to confirm the results. The second decision is selecting the chemical library to screen. Given that the primary screening model may dictate the size of the collection screened, these two decisions are often linked together. For example, primary cells and iPSC lines are not typically amenable to a large high-throughput screen, and typically sets on the order of 1000–5000 compounds or less are used. In these circumstances where a only small set of compounds can be screened, a popular approach is to screen drug or drug-like collections, of which many examples are cited in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. There are at least two arguments underlying the rationale for this approach. Opportunistically, if a hit is found, it could potentially move very quickly into the clinic, a topic referred to as “drug repurposing”. Drug repurposing is an attractive opportunity because it will save significant time and costs for regulatory approval. However, examples of drug repurposing are rare and are often either found as an observation in a clinical trial, or the new indication is an obvious connection to the original indication.<a onclick="showRef(event, 'ref101'); return false;" href="javascript:void(0);" class="ref ref101">(101)</a> Drug repurposing of a known marketed drug to a completely unexpected and unrelated disease indication is a low-chance endeavor. The second rationale for using these decks is one of finding unknown polypharmacology. These are drugs which are known to have in vivo efficacy against a human disease. If one is to search for unknown polypharmacological effects, a known drug seems a logical place to start because these compounds have some effect (e.g., as opposed to a random set of compounds which might not have any effect).</div></div><div id="sec1_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Annotated Decks</h4><div class="NLM_p">Annotated screening decks harness the power of legacy data which has been collected on an individual compound and can help identify the key target pharmacology. Drug and drug-like libraries are an example of a class of annotated decks. However, the coffers of many large pharmaceutical companies contain large historical data sets from HTS screens on millions of compounds. This data can be useful in several aspects. For example, a large number of hits annotated to the same target can support a hypothesis of the MOA, particularly if the hits include multiple and diverse scaffolds. The other utility of this data is that the level of prior activity can be understood and taken into context, for example, identification of pan-assay interference compounds (PAINS).<a onclick="showRef(event, 'ref102'); return false;" href="javascript:void(0);" class="ref ref102">(102)</a> This can be a major advantage as significant time and money can be saved if such compounds can be eliminated early in the cascade. One important aspect to consider when screening an annotated set is the potency of the primary pharmacology in comparison to the concentration used in the phenotypic assay. For example, a single-digit nanomolar kinase inhibitor which is active at 50 μM in a mammalian cell line may be cause for skepticism because most kinase inhibitors will likely have off-target kinase activity at such high concentrations. However, if the kinase inhibitor is active in a nonmammalian model with low homology to the human target (e.g., yeast), this could explain the weak activity. There is no database to help the medicinal chemist understand the loss or gain in potency when moving between different species, especially if these species are only distantly related. A chemist must rely on intuition and other contextual data to help make decisions on what series to advance. In these examples, it is best to treat the phenotypic data as qualitative and keep all the possible hits alive until they can be confirmed in an appropriate and definitive mammalian cell model.</div><div class="NLM_p last">Although the preliminary goal of these phenotypic screens is to identify a hit compound, a longer-term view of a chemistry strategy should take into consideration the CNS-like properties of hit compounds. If a hit chemical series has constraints that limit blood–brain barrier (BBB) penetration, it may be very difficult to validate the target in vivo and further to develop such a drug. In silico tools such as the Pfizer MPO model<a onclick="showRef(event, 'ref103'); return false;" href="javascript:void(0);" class="ref ref103">(103)</a> could help prioritize compounds for screening with better CNS-like properties. From our perspective, however, if the goal is to find a tool compound or a novel target, then the initial strategy should focus on well-annotated sets that cover as much diverse biology as possible. If the goal is to try and repurpose a known compound, then it is more appropriate to bias that set toward compounds which have, or are predicted to have, CNS penetration.</div></div><div id="sec1_8_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Random Screening Libraries</h4><div class="NLM_p last">In contrast to annotated sets, large and diverse random screening libraries often do not have associated background pharmacology. However, random screening libraries offer chemical diversity. If hits are found, these are more likely to represent novel chemical space in comparison to annotated decks of known compounds. If the random set also has many related analogues, one can get a sense of immediate SAR and begin to form hypotheses around what substituents are critical to activity. However, screening a large set of compounds produced primarily based on chemical diversity may result in the need for significant target deconvolution campaigns. There is an ongoing debate on whether target identification of a phenotypic hit is necessary for progression of a compound through development,<a onclick="showRef(event, 'ref104'); return false;" href="javascript:void(0);" class="ref ref104">(104)</a> as it is not required for FDA approval. However, identification of a target may allow for greater understanding of clinical outcomes (e.g., toxicity, target engagement or lack thereof to explain a clinical finding). An understanding of the target biology may also facilitate more focused SAR campaigns.</div></div></div><div id="sec1_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Selection of Appropriate Screening Library</h3><div class="NLM_p last">Selection of a screening library puts in motion the fate of finding quality hits and leads for the project. A “bigger is better” screening approach comes with the cost of having to invest significant resources in hit validation and deconvolution. If the model is not amenable to screening large numbers of compounds, then a project team must have a hypothesis on the types of molecules which could potentially work or must take a risk with a small random screen. Screening of FDA-approved drug libraries has proven to be a very popular approach and represents 7 of the 30 screens outlined in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Another strategy often employed is to screen “bioactive molecules”, (11/30 screens) which have significant overlap with FDA-approved drug libraries but also have pharmacologically active molecules which are not marketed drugs (e.g., NINDS set, LOPAC set). Another approach if the model is low-throughput is to focus the screen around a pathway or target (e.g., <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, entry 4 = PARP, entry 7 = LRRK2, entry 29 = CDK, and entry 30 = Akt). These focused approaches lend themselves to providing an immediate hypothesis regarding a potential mechanism of action.</div></div><div id="sec1_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Building a Screening Cascade</h3><div class="NLM_p">In addition to the selection of a compound library, the choice of screening models and downstream cascade to identify and validate hits is the cornerstone of a successful phenotypic screen. In our experience, a phenotypic screen in neurodegeneration has a cascade such as that outlined in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>. As discussed, a variety of primary screening models can be employed at the top of the cascade, each with their own intrinsic advantages and liabilities. Ideally, a disease model which is as close to the human disease state as possible is the first choice for a primary screen, bearing in mind that such models may be limited in scope by capacity and technical feasibility.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/medium/jm-2019-00797d_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/large/jm-2019-00797d_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Generic hit-validation cascade for phenotypic screening.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/large/jm-2019-00797d_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00797&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A key challenge in establishing a primary screening model is to validate the assay. Without a benchmark or internal standard this can prove difficult. Furthermore, even if an internal standard does exist, it is not necessarily obvious how much efficacy is required for a clinical candidate. For example, how much reduction of a toxic protein is needed for a drug to be effective? Is this accurately reflected in the cell model? Is it sufficient to reduce levels of a toxic protein but not have a profound effect on cell death? An important step is thus the identification of a tool compound that is validated throughout the screening cascade and that the researchers can use to address these questions.</div><div class="NLM_p">An important consideration for the primary screen is the screening concentration. Ideally one should screen at least at two concentrations in the initial primary screen (e.g., 1 and 10 μM). If a compound is toxic at the higher concentration, it could be missed and judged to be inactive if only that concentration was screened. Screening at two concentrations (or more) also creates greater confidence in hits if they are active at both concentrations. Furthermore, if the activity at the higher concentration is due to multiple target pharmacology, the lower concentrations may help to differentiate the involvement of various targets. The hits from this initial screen should then be run in full dose–response mode.</div><div class="NLM_p">From a chemistry perspective, the confidence around the hit will increase if close structural analogues of the hit also show activity. However, if the assay is particularly noisy it may not provide meaningful SAR, or worse, it may provide misleading information to the chemist. Multiple repeats of hits in the assay should be performed to provide a level of confidence to the medicinal chemist to inform analogue synthesis.</div><div class="NLM_p last">In addition, a counter-screen should be employed to mitigate false positives. For example, if the initial screen is designed to identify compounds which reduce the levels of a fluorescently labeled toxic protein, using a nontoxic protein with the same fluorescent label would be an appropriate counter-screen. It is important to conduct the counter-screen in a dose–response mode as single concentration screening is not sensitive enough to allow a chemist to definitively distinguish between a real hit and a false positive in the primary screen. Following the counter-screen, the remaining hits should be tested for cytotoxicity to rule out confounding effects. Confirmation of hits from phenotypic screens in an orthogonal assay is a critical next step to prioritize hits and identify true lead series. For example, if an immortalized cell line was used in the initial screen, then primary cells or an iPSC line would be appropriate for confirmation screens. Finally, confirmation of a lead molecule in an animal model of disease could be attempted if the lead compound has acceptable pharmacokinetic properties for in vivo testing. If this is not the case, a chemistry optimization campaign may be required. Furthermore, in order to be effective, the animal model must express the target, the target must have sufficient homology to the human target, and it must retain enough affinity for the compound to be efficacious. If the target is unknown, this cannot be ascertained and subsequently presents significant risk because neurodegenerative in vivo models are expensive and time-consuming.</div></div><div id="sec1_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Target Deconvolution</h3><div class="NLM_p">Once hits are confirmed in the cascade, target deconvolution efforts can be undertaken to understand a possible mechanism of action (MOA). If compounds have known annotation, there are several strategies that a medicinal chemist can employ to help bolster confidence in the target. For example, other compounds with the same background pharmacology could be tested, ideally from different chemical series. If multiple chemical series with the same pharmacology show activity, this will increase the confidence in the hit. Another strategy would be to test compounds which work on the same target or in the same pathway but have the opposite effect. For example, if the hit is an agonist at a certain receptor, one can test an antagonist to block the effect, if such a molecule exists. Genetic knockdown or overexpression experiments can also be used to validate the target. However, if the target is unknown, there are a variety of methods which could be deployed at this stage. While not required for regulatory approval, elucidation and understanding of the target will greatly simplify the development process. In silico and public databases are available to help the medicinal chemist to form hypotheses about potential MOAs. These include public databases, where one can search for similar compounds and their associated bioactivity, and ligand-based methods which attempt to predict target class. A comprehensive list of these tools has been reported by Koutsoukas et al. and is a valuable resource for these exercises.<a onclick="showRef(event, 'ref105'); return false;" href="javascript:void(0);" class="ref ref105">(105)</a> In vitro screens and panels can also be helpful to provide a hypothesis around the MOA. Large panels of known targets, including enzymes, G-protein-coupled receptors (GPCRs), ion channels, and kinases are commercially available.<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a> These can either be screened at single concentration or in dose–response, but the expenses of the exercise increase with increasing amount of data quality and target lists, and there is a significant chance that the target of the hit is not included in the panel. If the team has formed a specific hypothesis based on the scaffold (e.g., suspected kinase inhibitor), these panel screens are more informative because they can help to bolster support of the hypothesis. Gene expression profiling can also be used to elucidate new targets and mechanisms. In this approach, genes and gene clusters which are up- or downregulated in the presence of a compound are compared to expression profiles generated by compounds with known pharmacology.<a onclick="showRef(event, 'ref107 ref108'); return false;" href="javascript:void(0);" class="ref ref107 ref108">(107,108)</a> Advances in single cell transcriptional profiling are likely to make this a more impactful technology for neurodegenerative phenotypic screening and may allow for direct interrogation of individual cells which exhibit the phenotype.<a onclick="showRef(event, 'ref109 ref110'); return false;" href="javascript:void(0);" class="ref ref109 ref110">(109,110)</a> Cell-based phenotypic panels can also be used in a similar fashion to compare phenotypic responses of test compounds. The signatures of known compounds (i.e., the aggregated response in many cell lines and assay types) are used as a reference to identify potential pathways that the unknown hit could modulate.<a onclick="showRef(event, 'ref111'); return false;" href="javascript:void(0);" class="ref ref111">(111)</a> All of these methods require some prior knowledge of known targets, known pathways or known gene expression profiles and are thus referred to as biased approaches. There is also a risk that the results of these approaches may “cement” a false hypothesis that may be hard to break. For example, a positive hit against any particular target in a panel, even if the hit is very potent, does not necessarily mean that the compound is working through this mechanism in the phenotypic screen. On the basis of our experience, potent hits using these methods can be seductive but can also mislead the project team.</div><div class="NLM_p">In contrast to biased approaches, unbiased approaches such as chemoproteomic-based methods can provide direct detection of the target proteins in cells or cell lysates and are thus especially useful for identifying unknown targets.<a onclick="showRef(event, 'ref112 ref113'); return false;" href="javascript:void(0);" class="ref ref112 ref113">(112,113)</a> For these methods, the original hit compound needs to be derivatized with a label (e.g., photoaffinity), which can covalently tag the target proteins. This requires a chemistry campaign to explore the various vectors of the hit molecule and confirm that addition of a label does not result in a loss of the phenotypic activity. Unlabeled methods do exist, such as multiplexed quantitative mass-spectrometry-based cellular thermal shift assays (MS-CETSA), which measures thermal stability of the entire proteome in the presence and absence of compound.<a onclick="showRef(event, 'ref105 ref114 ref115'); return false;" href="javascript:void(0);" class="ref ref105 ref114 ref115">(105,114,115)</a> A recent example of this technology was the discovery of a novel target (purine nucleoside phosphorylase) in <i>Plasmodium falciparum</i> (the causative parasite of malaria).</div><div class="NLM_p">While a powerful technique for target deconvolution, chemoproteomics techniques require expensive investments in equipment, skilled MS expertise, and synthetic chemistry. Even then, the outcome is not guaranteed to produce a novel target, as a large number of hits may be pulled out, none of which may ultimately account for the phenotypic response. Follow-up genetic and pathway-probing experiments are necessary to confirm these hits.</div><div class="NLM_p last">Building a screening cascade, accessing screening libraries and target deconvolution exercise require significant resources (e.g., computational chemistry, computational biology, chemical biology), which may be especially hard for small biotech companies and academic laboratories, many of whom are developing these innovative screens. One solution to this dilemma is to work within a precompetitive model, where large pharmaceutical companies share their resources in collaboration with smaller institutions. Open innovation models such as these have been described previously<a onclick="showRef(event, 'ref116 ref117'); return false;" href="javascript:void(0);" class="ref ref116 ref117">(116,117)</a> and, in conjunction with funding agencies as a third partner, may benefit all parties involved by sharing resources and forming collaborative working relationships.</div></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Summary</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83830" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83830" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In conclusion, phenotypic screens in neurodegeneration are being facilitated by advances in screening models all the way from primitive organisms such as yeast to complex 3D cell models.</div><div class="NLM_p last">The outcome of a screen is critically dependent on the design of the assay, the nature of the screening collection, the concentrations of the test molecules and an appropriate triage effort to confirm the hits. Recent examples in ALS, Huntington’s disease, Parkinson’s disease, and Alzheimer’s disease illustrate that a diverse range of techniques can be used to identify hits and lead chemical series that mitigate neurodegeneration-associated phenotypes. In this review, we have discussed 30 recent screens, several of which have resulted in novel targets such as the discovery of stearoyl-CoA desaturase for Parkinson’s disease,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> eIF2b for cognitive defects,<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> and SMN2 splicing for SMA.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> In addition to small molecule screens, genetic screens have also yielded new targets, for example, Ataxin-2 for the treatment of ALS. These discoveries will hopefully advance the field toward new disease-modifying therapies. More novel targets are likely to be discovered with wider access to patient cell lines, automated screening methods, and innovative target deconvolution technologies.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b00797" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16387" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16387" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dean G. Brown</span> - <span class="hlFld-Affiliation affiliation">Hit
Discovery, Discovery Sciences, BioPharmaceuticals R&D, AstraZeneca, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7130-3928" title="Orcid link">http://orcid.org/0000-0002-7130-3928</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#7d19181c13531f0f120a133d1c0e090f1c071813181e1c531e1210"><span class="__cf_email__" data-cfemail="432726222d6d21312c342d03223037312239262d2620226d202c2e">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Heike J. Wobst</span> - <span class="hlFld-Affiliation affiliation">Neuroscience,
BioPharmaceuticals R&D, AstraZeneca, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#7c3419151719522b131e0f083c1d061219090e13521f1311"><span class="__cf_email__" data-cfemail="602805090b054e370f02131420011a0e0515120f4e030f0d">[email protected]</span></a></span></div></li></ul></li><li><ul></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80337" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80337" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Dean G. Brown</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00797&amp;id=BIO-d7e1067-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Dean Brown</b> is Director of Chemistry at AstraZeneca within the Hit Discovery Department, Discovery Science, R&D BioPharmaceuticals Unit. He obtained a B.Sc. in chemistry at Abilene Christian University (Abilene, TX) and a Ph.D at the University of Minnesota (Minneapolis) in organic chemistry. He has over 20 years of experience in industry with AstraZeneca. Dean has been responsible for building many new scientific programs in both neuroscience and infection, several which have resulted in successful transition to clinical trials. His scientific interests are in lead generation, library design, DNA-encoded library screening, synthetic chemistry, and applications of computational chemistry. Dean is listed as an author or coauthor on more than 50 publications and patent applications in medicinal chemistry and drug design, including granted patents on clinical candidates.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Heike J. Wobst</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00797&amp;id=BIO-d7e1072-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Heike Wobst</b> is a Principal Scientist at AstraZeneca within the Neuroscience Department, R&D BioPharmaceuticals Unit. She obtained a B.Sc. in Molecular Biotechnology from Heidelberg University, an M.Sc. in Molecular Medicine from Charité Universitätsmedizin Berlin, and a D.Phil. in Physiology, Anatomy, and Genetics from the University of Oxford. Her scientific interests are focused on molecular mechanisms of neurodegeneration and therapeutic intervention strategies targeting these mechanisms.</p></figure></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01114" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01114" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Nick Brandon and Damian Crowther for their helpful advice.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">ALS</td><td class="NLM_def"><p class="first last">amyotrophic lateral sclerosis</p></td></tr><tr><td class="NLM_term">APP</td><td class="NLM_def"><p class="first last">amyloid precursor protein</p></td></tr><tr><td class="NLM_term">ASO</td><td class="NLM_def"><p class="first last">antisense oligonucleotide</p></td></tr><tr><td class="NLM_term">ATXN2</td><td class="NLM_def"><p class="first last">ataxin-2</p></td></tr><tr><td class="NLM_term">BBB</td><td class="NLM_def"><p class="first last">blood–brain barrier</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra-terminal motif</p></td></tr><tr><td class="NLM_term">C9ORF72</td><td class="NLM_def"><p class="first last">chromosome 9 open reading frame 72</p></td></tr><tr><td class="NLM_term">CAG</td><td class="NLM_def"><p class="first last">cytosine–adenine–guanine trinucleotide repeat</p></td></tr><tr><td class="NLM_term">CDK</td><td class="NLM_def"><p class="first last">cyclin-dependent kinase</p></td></tr><tr><td class="NLM_term">CGG</td><td class="NLM_def"><p class="first last">cytosine–guanine–guanine trinucleotide repeat</p></td></tr><tr><td class="NLM_term">CNS</td><td class="NLM_def"><p class="first last">central nervous system</p></td></tr><tr><td class="NLM_term">CRISPR</td><td class="NLM_def"><p class="first last">clustered regularly interspaced short palindromic repeats</p></td></tr><tr><td class="NLM_term">DPR</td><td class="NLM_def"><p class="first last">dipeptide repeat</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">endoplasmic reticulum</p></td></tr><tr><td class="NLM_term">FAD</td><td class="NLM_def"><p class="first last">familial Alzheimer’s disease</p></td></tr><tr><td class="NLM_term">FDA</td><td class="NLM_def"><p class="first last">Food and Drug Administration</p></td></tr><tr><td class="NLM_term">FIC</td><td class="NLM_def"><p class="first last">first-in-class</p></td></tr><tr><td class="NLM_term">FTD</td><td class="NLM_def"><p class="first last">frontotemporal dementia</p></td></tr><tr><td class="NLM_term">FUS</td><td class="NLM_def"><p class="first last">fused in sarcoma</p></td></tr><tr><td class="NLM_term">FXTAS</td><td class="NLM_def"><p class="first last">fragile X-associated tremor/ataxia syndrome</p></td></tr><tr><td class="NLM_term">GFP</td><td class="NLM_def"><p class="first last">green fluorescent protein</p></td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G-protein-coupled receptor</p></td></tr><tr><td class="NLM_term">gRNA</td><td class="NLM_def"><p class="first last">guide RNA</p></td></tr><tr><td class="NLM_term">HDAC</td><td class="NLM_def"><p class="first last">histone deacetylase</p></td></tr><tr><td class="NLM_term">iPSC</td><td class="NLM_def"><p class="first last">induced pluripotent stem cells</p></td></tr><tr><td class="NLM_term">ISRIB</td><td class="NLM_def"><p class="first last">integrated stress response inhibitor</p></td></tr><tr><td class="NLM_term">lncRNA</td><td class="NLM_def"><p class="first last">long noncoding RNA</p></td></tr><tr><td class="NLM_term">LOPAC</td><td class="NLM_def"><p class="first last">Library of Pharmacologically Active Compounds</p></td></tr><tr><td class="NLM_term">LRRK2</td><td class="NLM_def"><p class="first last">leucine-rich repeat kinase 2</p></td></tr><tr><td class="NLM_term">LUHMES</td><td class="NLM_def"><p class="first last">Lund human mesencephalic</p></td></tr><tr><td class="NLM_term">MAPK</td><td class="NLM_def"><p class="first last">mitogen activated protein kinase</p></td></tr><tr><td class="NLM_term">MOA</td><td class="NLM_def"><p class="first last">mechanism of action</p></td></tr><tr><td class="NLM_term">MPP</td><td class="NLM_def"><p class="first last">methyl-4-phenylpyridinium</p></td></tr><tr><td class="NLM_term">MS</td><td class="NLM_def"><p class="first last">mass spectroscopy</p></td></tr><tr><td class="NLM_term">mTOR</td><td class="NLM_def"><p class="first last">mammalian target of rapamycin</p></td></tr><tr><td class="NLM_term">NIH</td><td class="NLM_def"><p class="first last">National Institutes of Health</p></td></tr><tr><td class="NLM_term">NINDS</td><td class="NLM_def"><p class="first last">National Institute for Neurological Disorders and Stroke</p></td></tr><tr><td class="NLM_term">NPC</td><td class="NLM_def"><p class="first last">neural progenitor cell</p></td></tr><tr><td class="NLM_term">NSAID</td><td class="NLM_def"><p class="first last">nonsteroidal anti-inflammatory drug</p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">pan-assay interference compounds</p></td></tr><tr><td class="NLM_term">PAR</td><td class="NLM_def"><p class="first last">poly(ADP ribose)</p></td></tr><tr><td class="NLM_term">PARP</td><td class="NLM_def"><p class="first last">poly(ADP ribose) polymerase</p></td></tr><tr><td class="NLM_term">PBP1</td><td class="NLM_def"><p class="first last">Pab1-binding protein</p></td></tr><tr><td class="NLM_term">PDE</td><td class="NLM_def"><p class="first last">phosphodiesterase</p></td></tr><tr><td class="NLM_term">PGRN</td><td class="NLM_def"><p class="first last">progranulin</p></td></tr><tr><td class="NLM_term">polyQ</td><td class="NLM_def"><p class="first last">polyglutamine repeat</p></td></tr><tr><td class="NLM_term">RIPK1</td><td class="NLM_def"><p class="first last">receptor-interacting serine/threonine-protein kinase 1</p></td></tr><tr><td class="NLM_term">RNAi</td><td class="NLM_def"><p class="first last">RNA interference</p></td></tr><tr><td class="NLM_term">sALS</td><td class="NLM_def"><p class="first last">sporadic amyotrophic lateral sclerosis</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationships</p></td></tr><tr><td class="NLM_term">SMA</td><td class="NLM_def"><p class="first last">spinal muscular atrophy</p></td></tr><tr><td class="NLM_term">snRNP</td><td class="NLM_def"><p class="first last">small nuclear ribonucleic protein</p></td></tr><tr><td class="NLM_term">TDP-43</td><td class="NLM_def"><p class="first last">transactive response DNA binding protein 43 kDa</p></td></tr><tr><td class="NLM_term">TH</td><td class="NLM_def"><p class="first last">tyrosine hydroxylase</p></td></tr><tr><td class="NLM_term">UBE3A</td><td class="NLM_def"><p class="first last">ubiquitin protein ligase E3A</p></td></tr><tr><td class="NLM_term">UPR</td><td class="NLM_def"><p class="first last">unfolded protein response</p></td></tr><tr><td class="NLM_term">VMAT</td><td class="NLM_def"><p class="first last">vesicular monoamine transporter</p></td></tr><tr><td class="NLM_term">WT</td><td class="NLM_def"><p class="first last">wild-type</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67488" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67488" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 117 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorne, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKew, J. C.</span></span> <span> </span><span class="NLM_article-title">Phenotypic Screens as a Renewed Approach for Drug Discovery</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2013.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.drudis.2013.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23850704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1emt7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=1067-1073&author=W.+Zhengauthor=N.+Thorneauthor=J.+C.+McKew&title=Phenotypic+Screens+as+a+Renewed+Approach+for+Drug+Discovery&doi=10.1016%2Fj.drudis.2013.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Phenotypic screens as a renewed approach for drug discovery</span></div><div class="casAuthors">Zheng, Wei; Thorne, Natasha; McKew, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">21-22</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The significant redn. in the no. of newly approved drugs in the past decade has been partially attributed to failures in discovery and validation of new targets.  Evaluation of recently approved new drugs has revealed that the no. of approved drugs discovered through phenotypic screens, an original drug screening paradigm, has exceeded those discovered through the mol. target-based approach.  Phenotypic screening is thus gaining new momentum in drug discovery with the hope that this approach may revitalize drug discovery and improve the success rate of drug approval through the discovery of viable lead compds. and identification of novel drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_cPfNTmnrfbVg90H21EOLACvtfcHk0lgiCyO0ueRSWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1emt7vO&md5=9d92e1f61043ac2dd11bfe5f32944cff</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DThorne%26aufirst%3DN.%26aulast%3DMcKew%26aufirst%3DJ.%2BC.%26atitle%3DPhenotypic%2520Screens%2520as%2520a%2520Renewed%2520Approach%2520for%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2013%26volume%3D18%26spage%3D1067%26epage%3D1073%26doi%3D10.1016%2Fj.drudis.2013.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Swinney, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anthony, J.</span></span> <span> </span><span class="NLM_article-title">How Were New Medicines Discovered?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">507</span>– <span class="NLM_lpage">519</span>, <span class="refDoi"> DOI: 10.1038/nrd3480</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnrd3480" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=21701501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlCmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=507-519&author=D.+C.+Swinneyauthor=J.+Anthony&title=How+Were+New+Medicines+Discovered%3F&doi=10.1038%2Fnrd3480"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">How were new medicines discovered?</span></div><div class="casAuthors">Swinney, David C.; Anthony, Jason</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">507-519</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Preclin. strategies that are used to identify potential drug candidates include target-based screening, phenotypic screening, modification of natural substances and biol.-based approaches.  To investigate whether some strategies have been more successful than others in the discovery of new drugs, we analyzed the discovery strategies and the mol. mechanism of action (MMOA) for new mol. entities and new biologics that were approved by the US Food and Drug Administration between 1999 and 2008.  Out of the 259 agents that were approved, 75 were first-in-class drugs with new MMOAs, and out of these, 50 (67%) were small mols. and 25 (33%) were biologics.  The results also show that the contribution of phenotypic screening to the discovery of first-in-class small-mol. drugs exceeded that of target-based approaches - with 28 and 17 of these drugs coming from the two approaches, resp. - in an era in which the major focus was on target-based approaches.  We postulate that a target-centric approach for first-in-class drugs, without consideration of an optimal MMOA, may contribute to the current high attrition rates and low productivity in pharmaceutical research and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE8Ah1iNz76bVg90H21EOLACvtfcHk0li8tAu0D1ti7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlCmsbs%253D&md5=7503620f8fbadb0a829f6ea2029936a2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd3480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3480%26sid%3Dliteratum%253Aachs%26aulast%3DSwinney%26aufirst%3DD.%2BC.%26aulast%3DAnthony%26aufirst%3DJ.%26atitle%3DHow%2520Were%2520New%2520Medicines%2520Discovered%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D507%26epage%3D519%26doi%3D10.1038%2Fnrd3480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aulner, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danckaert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ihm, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shum, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shorte, S. L.</span></span> <span> </span><span class="NLM_article-title">Next-Generation Phenotypic Screening in Early Drug Discovery for Infectious Diseases</span>. <i>Trends Parasitol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">559</span>– <span class="NLM_lpage">570</span>, <span class="refDoi"> DOI: 10.1016/j.pt.2019.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.pt.2019.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=31176583" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BB3M3ksFWmtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2019&pages=559-570&author=N.+Aulnerauthor=A.+Danckaertauthor=J.+Ihmauthor=D.+Shumauthor=S.+L.+Shorte&title=Next-Generation+Phenotypic+Screening+in+Early+Drug+Discovery+for+Infectious+Diseases&doi=10.1016%2Fj.pt.2019.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Next-Generation Phenotypic Screening in Early Drug Discovery for Infectious Diseases</span></div><div class="casAuthors">Aulner Nathalie; Danckaert Anne; Ihm JongEun; Shum David; Shorte Spencer L</div><div class="citationInfo"><span class="NLM_cas:title">Trends in parasitology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">559-570</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cell-based phenotypic screening has proven to be valuable, notably in recapitulating relevant biological conditions, for example, the host cell/pathogen niche.  However, the corresponding methodological complexity is not readily compatible with high-throughput pipelines, and fails to inform either molecular target or mechanism of action, which frustrates conventional drug-discovery roadmaps.  We review the state-of-the-art and emerging technologies that suggest new strategies for harnessing value from the complexity of phenotypic screening and augmenting powerful utility for translational drug discovery.  Advances in cellular, molecular, and bioinformatics technologies are converging at a cutting edge where the complexity of phenotypic screening may no longer be considered a hinderance but rather a catalyst to chemotherapeutic discovery for infectious diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSLgYTGAfXc8JPq5LRczm8ffW6udTcc2ebMhVA3VGC_Fbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3ksFWmtw%253D%253D&md5=a0ff8c0384a7171183c90ed6586c37b1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.pt.2019.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pt.2019.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DAulner%26aufirst%3DN.%26aulast%3DDanckaert%26aufirst%3DA.%26aulast%3DIhm%26aufirst%3DJ.%26aulast%3DShum%26aufirst%3DD.%26aulast%3DShorte%26aufirst%3DS.%2BL.%26atitle%3DNext-Generation%2520Phenotypic%2520Screening%2520in%2520Early%2520Drug%2520Discovery%2520for%2520Infectious%2520Diseases%26jtitle%3DTrends%2520Parasitol.%26date%3D2019%26volume%3D35%26spage%3D559%26epage%3D570%26doi%3D10.1016%2Fj.pt.2019.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mdluli, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaneko, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, A.</span></span> <span> </span><span class="NLM_article-title">The Tuberculosis Drug Discovery and Development Pipeline and Emerging Drug Targets</span>. <i>Cold Spring Harbor Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">a021154</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a021154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1101%2Fcshperspect.a021154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25635061" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=a021154&author=K.+Mdluliauthor=T.+Kanekoauthor=A.+Upton&title=The+Tuberculosis+Drug+Discovery+and+Development+Pipeline+and+Emerging+Drug+Targets&doi=10.1101%2Fcshperspect.a021154"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a021154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a021154%26sid%3Dliteratum%253Aachs%26aulast%3DMdluli%26aufirst%3DK.%26aulast%3DKaneko%26aufirst%3DT.%26aulast%3DUpton%26aufirst%3DA.%26atitle%3DThe%2520Tuberculosis%2520Drug%2520Discovery%2520and%2520Development%2520Pipeline%2520and%2520Emerging%2520Drug%2520Targets%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Med.%26date%3D2015%26volume%3D5%26spage%3Da021154%26doi%3D10.1101%2Fcshperspect.a021154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, R.</span></span> <span> </span><span class="NLM_article-title">First Novel Anti-Tuberculosis Drug in 40 Years</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">91</span>, <span class="refDoi"> DOI: 10.1038/nbt0213-89</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnbt0213-89" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23392485" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3sXit1Kisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=89-91&author=R.+Osborne&title=First+Novel+Anti-Tuberculosis+Drug+in+40+Years&doi=10.1038%2Fnbt0213-89"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">First novel anti-tuberculosis drug in 40 years</span></div><div class="casAuthors">Osborne, Randy</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">89-91</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9-XOgWa1Q27Vg90H21EOLACvtfcHk0li8tAu0D1ti7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXit1Kisrk%253D&md5=352170811a489381ea320397f4a6ed46</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnbt0213-89&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt0213-89%26sid%3Dliteratum%253Aachs%26aulast%3DOsborne%26aufirst%3DR.%26atitle%3DFirst%2520Novel%2520Anti-Tuberculosis%2520Drug%2520in%252040%2520Years%26jtitle%3DNat.%2520Biotechnol.%26date%3D2013%26volume%3D31%26spage%3D89%26epage%3D91%26doi%3D10.1038%2Fnbt0213-89" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brodie, M. J.</span></span> <span> </span><span class="NLM_article-title">Antiepileptic Drug Therapy the Story So Far</span>. <i>Seizure</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">650</span>– <span class="NLM_lpage">655</span>, <span class="refDoi"> DOI: 10.1016/j.seizure.2010.10.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.seizure.2010.10.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=21075011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC3M%252FhtlOrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2010&pages=650-655&author=M.+J.+Brodie&title=Antiepileptic+Drug+Therapy+the+Story+So+Far&doi=10.1016%2Fj.seizure.2010.10.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Antiepileptic drug therapy the story so far</span></div><div class="casAuthors">Brodie Martin J</div><div class="citationInfo"><span class="NLM_cas:title">Seizure</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">650-5</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The story began on 11th May 1857 when Charles Locock commented in the Lancet on his use of potassium bromide in 15 cases of "hysterical" epilepsy in young women.  The next development was the serendipitous discovery of the anticonvulsant properties of phenobarbital by Alfred Hauptmann in 1912.  This predated by more than 20 years the screening of potential therapeutic agents against "electrical seizures" in cats by Houston Merritt and Tracy Putnam.  The result was the launching of phenytoin in 1938.  Next came primidone, ethosuximide, carbamazepine and valproic acid, all of which can be regarded as first generation antiepileptic drugs (AEDs).  Shortly after their synthesis, the benzodiazepines were rapidly recognised as having anticonvulsant activity.  The modern era focused on the systematic screening of many thousands of compounds against rodent seizure models under the Anticonvulsant Drug Development Program in the US.  This resulted in the global licensing, in chronological order, of vigabatrin, zonisamide, oxcarbazepine, lamotrigine, felbamate, gabapentin, topiramate, tiagabine, levetiracetam, pregabalin and lacosamide.  Rufinamide is available in the US and Europe for Lennox-Gastaut syndrome and stiripentol has been made available for Dravet syndrome under the European orphan drug scheme.  Eslicarbazepine can be prescribed in Europe for partial seizures, but not in the US.  Has all this activity improved the lives of people with epilepsy? The short answer is-probably yes, but not by very much! This paper will conclude with a precis of the views of a selected group of paediatric and adult epileptologists on the advances in pharmacological management achieved over the last 20 years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQkQS9UEkVleJI8NvIb08rbfW6udTcc2eagiZNA8ppRn7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M%252FhtlOrtg%253D%253D&md5=6e97b89170a2b0b33f57bd397f016105</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.seizure.2010.10.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.seizure.2010.10.027%26sid%3Dliteratum%253Aachs%26aulast%3DBrodie%26aufirst%3DM.%2BJ.%26atitle%3DAntiepileptic%2520Drug%2520Therapy%2520the%2520Story%2520So%2520Far%26jtitle%3DSeizure%26date%3D2010%26volume%3D19%26spage%3D650%26epage%3D655%26doi%3D10.1016%2Fj.seizure.2010.10.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Z.</span></span> <span> </span><span class="NLM_article-title">Phenotypic Screens Targeting Neurodegenerative Diseases</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1177/1087057113499777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1177%2F1087057113499777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23958650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWgtbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=1-16&author=M.+Zhangauthor=G.+Luoauthor=Y.+Zhouauthor=S.+Wangauthor=Z.+Zhong&title=Phenotypic+Screens+Targeting+Neurodegenerative+Diseases&doi=10.1177%2F1087057113499777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Phenotypic screens targeting neurodegenerative diseases</span></div><div class="casAuthors">Zhang, Minhua; Luo, Guangrui; Zhou, Yanjiao; Wang, Shaohui; Zhong, Zhong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-16, 16 pp.</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Neurodegenerative diseases affect millions of people worldwide, and the incidences increase as the population ages.  Disease-modifying therapy that prevents or slows disease progression is still lacking, making neurodegenerative diseases an area of high unmet medical need.  Target-based drug discovery for disease-modifying agents has been ongoing for many years, without much success due to incomplete understanding of the mol. mechanisms underlying neurodegeneration.  Phenotypic screening, starting with a disease-relevant phenotype to screen for compds. that change the outcome of biol. pathways rather than activities at certain specific targets, offers an alternative approach to find small mols. or targets that modulate the key characteristics of neurodegeneration.  Phenotypic screens that focus on amelioration of disease-specific toxins, protection of neurons from degeneration, or promotion of neuroregeneration could be potential fertile grounds for discovering therapeutic agents for neurodegenerative diseases.  In this review, we will summarize the progress of compd. screening using these phenotypic-based strategies for this area, with a highlight on unique considerations for disease models, assays, and screening methodologies.  We will further provide our perspectives on how best to use phenotypic screening to develop drug leads for neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEhrbJWCDltbVg90H21EOLACvtfcHk0lj0UZwJhJ5zwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWgtbfN&md5=d462b3885bb85f3593316d0e71f4e672</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1177%2F1087057113499777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113499777%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DZ.%26atitle%3DPhenotypic%2520Screens%2520Targeting%2520Neurodegenerative%2520Diseases%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2014%26volume%3D19%26spage%3D1%26epage%3D16%26doi%3D10.1177%2F1087057113499777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, D.</span></span> <span> </span><span class="NLM_article-title">Phenotypic Screening in Cancer Drug Discovery - Past, Present and Future</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">588</span>– <span class="NLM_lpage">602</span>, <span class="refDoi"> DOI: 10.1038/nrd4366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnrd4366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25033736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=588-602&author=J.+G.+Moffatauthor=J.+Rudolphauthor=D.+Bailey&title=Phenotypic+Screening+in+Cancer+Drug+Discovery+-+Past%2C+Present+and+Future&doi=10.1038%2Fnrd4366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Phenotypic screening in cancer drug discovery - past, present and future</span></div><div class="casAuthors">Moffat, John G.; Rudolph, Joachim; Bailey, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">588-602</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There has been a resurgence of interest in the use of phenotypic screens in drug discovery as an alternative to target-focused approaches.  Given that oncol. is currently the most active therapeutic area, and also one in which target-focused approaches have been particularly prominent in the past two decades, we investigated the contribution of phenotypic assays to oncol. drug discovery by analyzing the origins of all new small-mol. cancer drugs approved by the US Food and Drug Administration (FDA) over the past 15 years and those currently in clin. development.  Although the majority of these drugs originated from target-based discovery, we identified a significant no. whose discovery depended on phenotypic screening approaches.  We postulate that the contribution of phenotypic screening to cancer drug discovery has been hampered by a reliance on 'classical' nonspecific drug effects such as cytotoxicity and mitotic arrest, exacerbated by a paucity of mechanistically defined cellular models for therapeutically translatable cancer phenotypes.  However, tech. and biol. advances that enable such mechanistically informed phenotypic models have the potential to empower phenotypic drug discovery in oncol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTWtoTIgWqvbVg90H21EOLACvtfcHk0lj0UZwJhJ5zwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7L&md5=0e8efed90e5bc4b47f959b83a6ab5b96</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd4366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4366%26sid%3Dliteratum%253Aachs%26aulast%3DMoffat%26aufirst%3DJ.%2BG.%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DBailey%26aufirst%3DD.%26atitle%3DPhenotypic%2520Screening%2520in%2520Cancer%2520Drug%2520Discovery%2520-%2520Past%252C%2520Present%2520and%2520Future%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D588%26epage%3D602%26doi%3D10.1038%2Fnrd4366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delves, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magny, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clement, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boullay, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandel, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of Small Molecule Inhibitors of the Bet Family Bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">3827</span>– <span class="NLM_lpage">3838</span>, <span class="refDoi"> DOI: 10.1021/jm200108t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200108t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3827-3838&author=C.+W.+Chungauthor=H.+Costeauthor=J.+H.+Whiteauthor=O.+Mirguetauthor=J.+Wildeauthor=R.+L.+Gosminiauthor=C.+Delvesauthor=S.+M.+Magnyauthor=R.+Woodwardauthor=S.+A.+Hughesauthor=E.+V.+Boursierauthor=H.+Flynnauthor=A.+M.+Bouillotauthor=P.+Bamboroughauthor=J.+M.+Brusqauthor=F.+J.+Gellibertauthor=E.+J.+Jonesauthor=A.+M.+Riouauthor=P.+Homesauthor=S.+L.+Martinauthor=I.+J.+Uingsauthor=J.+Toumauthor=C.+A.+Clementauthor=A.+B.+Boullayauthor=R.+L.+Grimleyauthor=F.+M.+Blandelauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=J.+Kirilovskyauthor=E.+Nicodeme&title=Discovery+and+Characterization+of+Small+Molecule+Inhibitors+of+the+Bet+Family+Bromodomains&doi=10.1021%2Fjm200108t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of small molecule inhibitors of the BET family bromodomains</span></div><div class="casAuthors">Chung, Chun-wa; Coste, Herve; White, Julia H.; Mirguet, Olivier; Wilde, Jonathan; Gosmini, Romain L.; Delves, Chris; Magny, Sylvie M.; Woodward, Robert; Hughes, Stephen A.; Boursier, Eric V.; Flynn, Helen; Bouillot, Anne M.; Bamborough, Paul; Brusq, Jean-Marie G.; Gellibert, Francoise J.; Jones, Emma J.; Riou, Alizon M.; Homes, Paul; Martin, Sandrine L.; Uings, Iain J.; Toum, Jerome; Clement, Catherine A.; Boullay, Anne-Benedicte; Grimley, Rachel L.; Blandel, Florence M.; Prinjha, Rab K.; Lee, Kevin; Kirilovsky, Jorge; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3827-3838</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic mechanisms of gene regulation have a profound role in normal development and disease processes.  An integral part of this mechanism occurs through lysine acetylation of histone tails which are recognized by bromodomains.  While the biol. and structural characterization of many bromodomain contg. proteins has advanced considerably, the therapeutic tractability of this protein family is only now becoming understood.  This paper describes the discovery and mol. characterization of potent (nM) small mol. inhibitors that disrupt the function of the BET family of bromodomains (Brd2, Brd3, and Brd4).  By using a combination of phenotypic screening, chemoproteomics, and biophys. studies, we have discovered that the protein-protein interactions between bromodomains and acetylated histones can be antagonized by selective small mols. that bind at the acetylated lysine recognition pocket.  X-ray crystal structures of compds. bound into bromodomains of Brd2 and Brd4 elucidate the mol. interactions of binding and explain the precisely defined stereochem. required for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VauuYkT5YLVg90H21EOLACvtfcHk0lj5Qsg1a3k0cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D&md5=ebc0f6062321dece877259d775f1c19b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm200108t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200108t%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%2BH.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DWilde%26aufirst%3DJ.%26aulast%3DGosmini%26aufirst%3DR.%2BL.%26aulast%3DDelves%26aufirst%3DC.%26aulast%3DMagny%26aufirst%3DS.%2BM.%26aulast%3DWoodward%26aufirst%3DR.%26aulast%3DHughes%26aufirst%3DS.%2BA.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DFlynn%26aufirst%3DH.%26aulast%3DBouillot%26aufirst%3DA.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBrusq%26aufirst%3DJ.%2BM.%26aulast%3DGellibert%26aufirst%3DF.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DRiou%26aufirst%3DA.%2BM.%26aulast%3DHomes%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%2BL.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DClement%26aufirst%3DC.%2BA.%26aulast%3DBoullay%26aufirst%3DA.%2BB.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBlandel%26aufirst%3DF.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520the%2520Bet%2520Family%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D3827%26epage%3D3838%26doi%3D10.1021%2Fjm200108t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gammon, K.</span></span> <span> </span><span class="NLM_article-title">Neurodegenerative Disease: Brain Windfall</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>515</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">300</span>, <span class="refDoi"> DOI: 10.1038/nj7526-299a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnj7526-299a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25396246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC2M3ot12kuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=515&publication_year=2014&pages=299-300&author=K.+Gammon&title=Neurodegenerative+Disease%3A+Brain+Windfall&doi=10.1038%2Fnj7526-299a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Neurodegenerative disease: brain windfall</span></div><div class="casAuthors">Gammon Katharine</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">515</span>
        (<span class="NLM_cas:issue">7526</span>),
    <span class="NLM_cas:pages">299-300</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQDfOo1BTaCdG52AMpI4GK2fW6udTcc2eYyN1xm3zEPWLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3ot12kuw%253D%253D&md5=1ee40ff6fc8fab21a70d1df7a0b12674</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnj7526-299a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnj7526-299a%26sid%3Dliteratum%253Aachs%26aulast%3DGammon%26aufirst%3DK.%26atitle%3DNeurodegenerative%2520Disease%253A%2520Brain%2520Windfall%26jtitle%3DNature%26date%3D2014%26volume%3D515%26spage%3D299%26epage%3D300%26doi%3D10.1038%2Fnj7526-299a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span> </span><span class="NLM_article-title">Focus on Neurodegenerative Disease</span>.  <i>Nat.
Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume">21</span>,  <span class="NLM_fpage">1293</span>.<span class="refDoi"> DOI: 10.1038/s41593-018-0250-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41593-018-0250-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30258240" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Focus+on+Neurodegenerative+Disease.+Nat.%0ANeurosci.+2018%2C+21%2C+1293.+10.1038%2Fs41593-018-0250-x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fs41593-018-0250-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41593-018-0250-x%26sid%3Dliteratum%253Aachs%26atitle%3DFocus%2520on%2520Neurodegenerative%2520Disease%26jtitle%3DNat.%250ANeurosci.%26date%3D2018%26volume%3D21%26spage%3D1293%26doi%3D10.1038%2Fs41593-018-0250-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soto, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritzkow, S.</span></span> <span> </span><span class="NLM_article-title">Protein Misfolding, Aggregation, and Conformational Strains in Neurodegenerative Diseases</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1332</span>– <span class="NLM_lpage">1340</span>, <span class="refDoi"> DOI: 10.1038/s41593-018-0235-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41593-018-0235-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30250260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslymtrfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2018&pages=1332-1340&author=C.+Sotoauthor=S.+Pritzkow&title=Protein+Misfolding%2C+Aggregation%2C+and+Conformational+Strains+in+Neurodegenerative+Diseases&doi=10.1038%2Fs41593-018-0235-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Protein misfolding, aggregation, and conformational strains in neurodegenerative diseases</span></div><div class="casAuthors">Soto, Claudio; Pritzkow, Sandra</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1332-1340</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A hallmark event in neurodegenerative diseases (NDs) is the misfolding, aggregation, and accumulation of proteins, leading to cellular dysfunction, loss of synaptic connections, and brain damage.  Despite the involvement of distinct proteins in different NDs, the process of protein misfolding and aggregation is remarkably similar.  A recent breakthrough in the field was the discovery that misfolded protein aggregates can self-propagate through seeding and spread the pathol. abnormalities between cells and tissues in a manner akin to the behavior of infectious prions in prion diseases.  This discovery has vast implications for understanding the mechanisms involved in the initiation and progression of NDs, as well as for the design of novel strategies for treatment and diagnosis.  In this Review, we provide a crit. discussion of the role of protein misfolding and aggregation in NDs.  Commonalities and differences between distinct protein aggregates will be highlighted, in addn. to evidence supporting the hypothesis that misfolded aggregates can be transmissible by the prion principle.  We will also describe the mol. basis and implications for prion-like conformational strains, cross-interaction between different misfolded proteins in the brain, and how these concepts can be applied to the development of novel strategies for therapy and diagnosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosF34qbEjX8LVg90H21EOLACvtfcHk0ljflND1Hi4yKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslymtrfN&md5=0d18590c1c796f289939f7125dc56f16</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fs41593-018-0235-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41593-018-0235-9%26sid%3Dliteratum%253Aachs%26aulast%3DSoto%26aufirst%3DC.%26aulast%3DPritzkow%26aufirst%3DS.%26atitle%3DProtein%2520Misfolding%252C%2520Aggregation%252C%2520and%2520Conformational%2520Strains%2520in%2520Neurodegenerative%2520Diseases%26jtitle%3DNat.%2520Neurosci.%26date%3D2018%26volume%3D21%26spage%3D1332%26epage%3D1340%26doi%3D10.1038%2Fs41593-018-0235-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, J.</span></span> <span> </span><span class="NLM_article-title">Notes on Continuing Education. The First Hour: Creating a Learning Evironment</span>. <i>J. Contin. Educ. Nurs.</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">46</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=258237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADyaL3c%252FgslWhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1979&pages=42-46&author=J.+Murphy&title=Notes+on+Continuing+Education.+The+First+Hour%3A+Creating+a+Learning+Evironment"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Notes on continuing education. The first hour: creating a learning evironment</span></div><div class="casAuthors">Murphy J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of continuing education in nursing</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">42-6</span>
        ISSN:<span class="NLM_cas:issn">0022-0124</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRrfSCfuWDe30NHe_gFldTbfW6udTcc2eZT1ykh-sVY3rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL3c%252FgslWhtw%253D%253D&md5=ccd924a323aa58778f665abe82520e39</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMurphy%26aufirst%3DJ.%26atitle%3DNotes%2520on%2520Continuing%2520Education.%2520The%2520First%2520Hour%253A%2520Creating%2520a%2520Learning%2520Evironment%26jtitle%3DJ.%2520Contin.%2520Educ.%2520Nurs.%26date%3D1979%26volume%3D10%26spage%3D42%26epage%3D46" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Botstein, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chervitz, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherry, J. M.</span></span> <span> </span><span class="NLM_article-title">Yeast as a Model Organism</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">1259</span>– <span class="NLM_lpage">1260</span>, <span class="refDoi"> DOI: 10.1126/science.277.5330.1259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fscience.277.5330.1259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=9297238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADyaK2sXlslaitr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=1997&pages=1259-1260&author=D.+Botsteinauthor=S.+A.+Chervitzauthor=J.+M.+Cherry&title=Yeast+as+a+Model+Organism&doi=10.1126%2Fscience.277.5330.1259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Yeast as a model organism</span></div><div class="casAuthors">Botstein, David; Chervitz, Steven A.; Cherry, J. Michael</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">5330</span>),
    <span class="NLM_cas:pages">1259-1260</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoilx40K3MqprVg90H21EOLACvtfcHk0ljflND1Hi4yKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlslaitr4%253D&md5=8df86f99c1f1b836f131cba2ab3f51f9</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1126%2Fscience.277.5330.1259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.277.5330.1259%26sid%3Dliteratum%253Aachs%26aulast%3DBotstein%26aufirst%3DD.%26aulast%3DChervitz%26aufirst%3DS.%2BA.%26aulast%3DCherry%26aufirst%3DJ.%2BM.%26atitle%3DYeast%2520as%2520a%2520Model%2520Organism%26jtitle%3DScience%26date%3D1997%26volume%3D277%26spage%3D1259%26epage%3D1260%26doi%3D10.1126%2Fscience.277.5330.1259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goffeau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrell, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussey, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dujon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galibert, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoheisel, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacq, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mewes, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murakami, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philippsen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tettelin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, S. G.</span></span> <span> </span><span class="NLM_article-title">Life with 6000 Genes</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>274</i></span>,  <span class="NLM_fpage">546</span>– <span class="NLM_lpage">567</span>, <span class="refDoi"> DOI: 10.1126/science.274.5287.546</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fscience.274.5287.546" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=8849441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADyaK28XmsVWlt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=546-567&author=A.+Goffeauauthor=B.+G.+Barrellauthor=H.+Busseyauthor=R.+W.+Davisauthor=B.+Dujonauthor=H.+Feldmannauthor=F.+Galibertauthor=J.+D.+Hoheiselauthor=C.+Jacqauthor=M.+Johnstonauthor=E.+J.+Louisauthor=H.+W.+Mewesauthor=Y.+Murakamiauthor=P.+Philippsenauthor=H.+Tettelinauthor=S.+G.+Oliver&title=Life+with+6000+Genes&doi=10.1126%2Fscience.274.5287.546"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Life with 6000 genes</span></div><div class="casAuthors">Goffeau, A.; Barrell, B. G.; Bussey, H.; Davis, R. W.; Dujon, B.; Feldmann, H.; Galibert, F.; Hoheisel, J. D.; Jacq, C.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">5287</span>),
    <span class="NLM_cas:pages">546, 563-567</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The genome of the yeast Saccharomyces cerevisiae has been completely sequenced through a worldwide collaboration.  The sequence of 12,068 kilobases defines 5885 potential protein-encoding genes, ∼140 genes specifying rRNA, 40 genes for small nuclear RNA mols., and 275 tRNA genes.  In addn., the complete sequence provides information about the higher order organization of yeast's 16 chromosomes and allows some insight into their evolutionary history.  The genome shows a considerable amt. of apparent genetic redundancy, and one of the major problems to be tackled during the next stage of the yeast genome project is to elucidate the biol. functions of all of these genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqAJbfgr4k5rVg90H21EOLACvtfcHk0ljflND1Hi4yKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmsVWlt7k%253D&md5=e7d4f76e51df1c61544dff929b6ce8e0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1126%2Fscience.274.5287.546&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.274.5287.546%26sid%3Dliteratum%253Aachs%26aulast%3DGoffeau%26aufirst%3DA.%26aulast%3DBarrell%26aufirst%3DB.%2BG.%26aulast%3DBussey%26aufirst%3DH.%26aulast%3DDavis%26aufirst%3DR.%2BW.%26aulast%3DDujon%26aufirst%3DB.%26aulast%3DFeldmann%26aufirst%3DH.%26aulast%3DGalibert%26aufirst%3DF.%26aulast%3DHoheisel%26aufirst%3DJ.%2BD.%26aulast%3DJacq%26aufirst%3DC.%26aulast%3DJohnston%26aufirst%3DM.%26aulast%3DLouis%26aufirst%3DE.%2BJ.%26aulast%3DMewes%26aufirst%3DH.%2BW.%26aulast%3DMurakami%26aufirst%3DY.%26aulast%3DPhilippsen%26aufirst%3DP.%26aulast%3DTettelin%26aufirst%3DH.%26aulast%3DOliver%26aufirst%3DS.%2BG.%26atitle%3DLife%2520with%25206000%2520Genes%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D546%26epage%3D567%26doi%3D10.1126%2Fscience.274.5287.546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Outeiro, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giorgini, F.</span></span> <span> </span><span class="NLM_article-title">Yeast as a Drug Discovery Platform in Huntington’s and Parkinson’s Diseases</span>. <i>Biotechnol. J.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">258</span>– <span class="NLM_lpage">269</span>, <span class="refDoi"> DOI: 10.1002/biot.200500043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1002%2Fbiot.200500043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=16897706" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtlKqtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=258-269&author=T.+F.+Outeiroauthor=F.+Giorgini&title=Yeast+as+a+Drug+Discovery+Platform+in+Huntington%E2%80%99s+and+Parkinson%E2%80%99s+Diseases&doi=10.1002%2Fbiot.200500043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Yeast as a drug discovery platform in Huntington's and Parkinson's diseases</span></div><div class="casAuthors">Outeiro, Tiago Fleming; Giorgini, Flaviano</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">258-269</span>CODEN:
                <span class="NLM_cas:coden">BJIOAM</span>;
        ISSN:<span class="NLM_cas:issn">1860-6768</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The high degree of conservation of cellular and mol. processes between the budding yeast Saccharomyces cerevisiae and higher eukaryotes have made it a valuable system for numerous studies of the basic mechanisms behind devastating illnesses such as cancer, infectious disease, and neurodegenerative disorders.  Several studies in yeast have already contributed to the basic understanding of cellular dysfunction in both Huntington's and Parkinson's disease.  Functional genomics approaches currently being undertaken in yeast may lead to novel insights into the genes and pathways that modulate neuronal cell dysfunction and death in these diseases.  In addn., the budding yeast constitutes a valuable system for identification of new drug targets, both via target-based and non-target-based drug screening.  Importantly, yeast can be used as a cellular platform to analyze the cellular effects of candidate compds., which is crit. for the development of effective therapeutics.  While the mol. mechanisms that underlie neurodegeneration will ultimately have to be tested in neuronal and animal models, there are several distinct advantages to using simple model organisms to elucidate fundamental aspects of protein aggregation, amyloid toxicity, and cellular dysfunction.  Here, we review recent studies that have shown that amyloid formation by disease-causing proteins and many of the resulting cellular deficits can be faithfully recapitulated in yeast.  In addn., we discuss new yeast-based techniques for screening candidate therapeutic compds. for Huntington's and Parkinson's diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfBHyXW8Cd8bVg90H21EOLACvtfcHk0lguRL5nmdAeSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmtlKqtb0%253D&md5=0a56d1b8f18dc7f01b58b6bd8ed24a13</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fbiot.200500043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbiot.200500043%26sid%3Dliteratum%253Aachs%26aulast%3DOuteiro%26aufirst%3DT.%2BF.%26aulast%3DGiorgini%26aufirst%3DF.%26atitle%3DYeast%2520as%2520a%2520Drug%2520Discovery%2520Platform%2520in%2520Huntington%25E2%2580%2599s%2520and%2520Parkinson%25E2%2580%2599s%2520Diseases%26jtitle%3DBiotechnol.%2520J.%26date%3D2006%26volume%3D1%26spage%3D258%26epage%3D269%26doi%3D10.1002%2Fbiot.200500043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Outeiro, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindquist, S.</span></span> <span> </span><span class="NLM_article-title">Yeast Cells Provide Insight into Alpha-Synuclein Biology and Pathobiology</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>302</i></span>,  <span class="NLM_fpage">1772</span>– <span class="NLM_lpage">1775</span>, <span class="refDoi"> DOI: 10.1126/science.1090439</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fscience.1090439" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=14657500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpsVWns74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=302&publication_year=2003&pages=1772-1775&author=T.+F.+Outeiroauthor=S.+Lindquist&title=Yeast+Cells+Provide+Insight+into+Alpha-Synuclein+Biology+and+Pathobiology&doi=10.1126%2Fscience.1090439"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Yeast Cells Provide Insight into Alpha-Synuclein Biology and Pathobiology</span></div><div class="casAuthors">Outeiro, Tiago Fleming; Lindquist, Susan</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">302</span>
        (<span class="NLM_cas:issue">5651</span>),
    <span class="NLM_cas:pages">1772-1775</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Alpha-synuclein is implicated in several neurodegenerative disorders, such as Parkinson's disease and multiple system atrophy, yet its functions remain obscure.  When expressed in yeast, alpha-synuclein assocd. with the plasma membrane in a highly selective manner, before forming cytoplasmic inclusions through a concn.-dependent, nucleated process.  Alpha-synuclein inhibited phospholipase D, induced lipid droplet accumulation, and affected vesicle trafficking.  This readily manipulable system provides an opportunity to dissect the mol. pathways underlying normal alpha-synuclein biol. and the pathogenic consequences of its misfolding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoO71orB7yEqrVg90H21EOLACvtfcHk0lguRL5nmdAeSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpsVWns74%253D&md5=3683c72f65e8639d5ad01569a724d9ea</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1126%2Fscience.1090439&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1090439%26sid%3Dliteratum%253Aachs%26aulast%3DOuteiro%26aufirst%3DT.%2BF.%26aulast%3DLindquist%26aufirst%3DS.%26atitle%3DYeast%2520Cells%2520Provide%2520Insight%2520into%2520Alpha-Synuclein%2520Biology%2520and%2520Pathobiology%26jtitle%3DScience%26date%3D2003%26volume%3D302%26spage%3D1772%26epage%3D1775%26doi%3D10.1126%2Fscience.1090439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tenreiro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munder, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alberti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Outeiro, T. F.</span></span> <span> </span><span class="NLM_article-title">Harnessing the Power of Yeast to Unravel the Molecular Basis of Neurodegeneration</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1111/jnc.12271</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1111%2Fjnc.12271" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23600759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWqs7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2013&pages=438-452&author=S.+Tenreiroauthor=M.+C.+Munderauthor=S.+Albertiauthor=T.+F.+Outeiro&title=Harnessing+the+Power+of+Yeast+to+Unravel+the+Molecular+Basis+of+Neurodegeneration&doi=10.1111%2Fjnc.12271"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing the power of yeast to unravel the molecular basis of neurodegeneration</span></div><div class="casAuthors">Tenreiro, Sandra; Munder, Matthias C.; Alberti, Simon; Outeiro, Tiago F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">3&4</span>),
    <span class="NLM_cas:pages">438-452</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Several neurodegenerative diseases, such as Parkinson's disease (PD), Alzheimer's disease (AD), Huntington's disease (HD), amyotrophic lateral sclerosis (ALS), or prion diseases, are known for their intimate assocn. with protein misfolding and aggregation.  These disorders are characterized by the loss of specific neuronal populations in the brain and are highly assocd. with aging, suggesting a decline in proteostasis capacity may contribute to pathogenesis.  Nevertheless, the precise mol. mechanisms that lead to the selective demise of neurons remain poorly understood.  As a consequence, appropriate therapeutic approaches and effective treatments are largely lacking.  The development of cellular and animal models that faithfully reproduce central aspects of neurodegeneration has been crucial for advancing our understanding of these diseases.  Approaches involving the sequential use of different model systems, starting with simpler cellular models and ending with validation in more complex animal models, resulted in the discovery of promising therapeutic targets and small mols. with therapeutic potential.  Within this framework, the simple and well-characterized eukaryote Saccharomyces cerevisiae, also known as budding yeast, is being increasingly used to study the mol. basis of several neurodegenerative disorders.  Yeast provides an unprecedented toolbox for the dissection of complex biol. processes and pathways.  Here, we summarize how yeast models are adding to our current understanding of several neurodegenerative disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvEkLUxN4SHrVg90H21EOLACvtfcHk0lguRL5nmdAeSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWqs7fP&md5=8eb7b8266d5a994c78ec2057ab238cd8</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Fjnc.12271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjnc.12271%26sid%3Dliteratum%253Aachs%26aulast%3DTenreiro%26aufirst%3DS.%26aulast%3DMunder%26aufirst%3DM.%2BC.%26aulast%3DAlberti%26aufirst%3DS.%26aulast%3DOuteiro%26aufirst%3DT.%2BF.%26atitle%3DHarnessing%2520the%2520Power%2520of%2520Yeast%2520to%2520Unravel%2520the%2520Molecular%2520Basis%2520of%2520Neurodegeneration%26jtitle%3DJ.%2520Neurochem.%26date%3D2013%26volume%3D127%26spage%3D438%26epage%3D452%26doi%3D10.1111%2Fjnc.12271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deak, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mieyal, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roder, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson-Delfosse, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. G.</span></span> <span> </span><span class="NLM_article-title">Kinase Inhibitors Arrest Neurodegeneration in Cell and C. Elegans Models of Lrrk2 Toxicity</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">328</span>– <span class="NLM_lpage">344</span>, <span class="refDoi"> DOI: 10.1093/hmg/dds431</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1093%2Fhmg%2Fdds431" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23065705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvV2gtr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2013&pages=328-344&author=C.+Yaoauthor=W.+M.+Johnsonauthor=Y.+Gaoauthor=W.+Wangauthor=J.+Zhangauthor=M.+Deakauthor=D.+R.+Alessiauthor=X.+Zhuauthor=J.+J.+Mieyalauthor=H.+Roderauthor=A.+L.+Wilson-Delfosseauthor=S.+G.+Chen&title=Kinase+Inhibitors+Arrest+Neurodegeneration+in+Cell+and+C.+Elegans+Models+of+Lrrk2+Toxicity&doi=10.1093%2Fhmg%2Fdds431"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitors arrest neurodegeneration in cell and C. elegans models of LRRK2 toxicity</span></div><div class="casAuthors">Yao, Chen; Johnson, William M.; Gao, Yue; Wang, Wen; Zhang, Jinwei; Deak, Maria; Alessi, Dario R.; Zhu, Xiongwei; Mieyal, John J.; Roder, Hanno; Wilson-Delfosse, Amy L.; Chen, Shu G.</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">328-344</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Mutations in leucine-rich repeat kinase 2 (LRRK2) are the most frequent known cause of late-onset Parkinson's disease (PD).  To explore the therapeutic potential of small mols. targeting the LRRK2 kinase domain, we characterized two LRRK2 kinase inhibitors, TTT-3002 and LRRK2-IN1, for their effects against LRRK2 activity in vitro and in Caenorhabditis elegans models of LRRK2-linked neurodegeneration.  TTT-3002 and LRRK2-IN1 potently inhibited in vitro kinase activity of LRRK2 wild-type and mutant proteins, attenuated phosphorylation of cellular LRRK2 and rescued neurotoxicity of mutant LRRK2 in transfected cells.  To establish whether LRRK2 kinase inhibitors can mitigate pathogenesis caused by different mutations including G2019S and R1441C located within and outside of the LRRK2 kinase domain, resp., we evaluated effects of TTT-3002 and LRRK2-IN1 against R1441C- and G2019S-induced neurodegeneration in C. elegans models.  TTT-3002 and LRRK2-IN1 rescued the behavioral deficit characteristic of dopaminergic impairment in transgenic C. elegans expressing human R1441C- and G2019S-LRRK2.  The inhibitors displayed nanomolar to low micromolar rescue potency when administered either pre-symptomatically or post-symptomatically, indicating both prevention and reversal of the dopaminergic deficit.  The same treatments also led to long-lasting prevention and rescue of neurodegeneration.  In contrast, TTT-3002 and LRRK2-IN1 were ineffective against the neurodegenerative phenotype in transgenic worms carrying the inhibitor-resistant A2016T mutation of LRRK2, suggesting that they elicit neuroprotective effects in vivo by targeting LRRK2 specifically.  Our findings indicate that the LRRK2 kinase activity is crit. for neurodegeneration caused by R1441C and G2019S mutations, suggesting that kinase inhibition of LRRK2 may represent a promising therapeutic strategy for PD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7q5ZpYDN5-LVg90H21EOLACvtfcHk0lguRL5nmdAeSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvV2gtr%252FE&md5=bbdd541fecf70129b8d42cfbd17eb5ad</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2Fdds431&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252Fdds431%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DC.%26aulast%3DJohnson%26aufirst%3DW.%2BM.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDeak%26aufirst%3DM.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DMieyal%26aufirst%3DJ.%2BJ.%26aulast%3DRoder%26aufirst%3DH.%26aulast%3DWilson-Delfosse%26aufirst%3DA.%2BL.%26aulast%3DChen%26aufirst%3DS.%2BG.%26atitle%3DKinase%2520Inhibitors%2520Arrest%2520Neurodegeneration%2520in%2520Cell%2520and%2520C.%2520Elegans%2520Models%2520of%2520Lrrk2%2520Toxicity%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2013%26volume%3D22%26spage%3D328%26epage%3D344%26doi%3D10.1093%2Fhmg%2Fdds431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barclay, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgoyne, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, A.</span></span> <span> </span><span class="NLM_article-title">Using C. Elegans to Discover Therapeutic Compounds for Ageing-Associated Neurodegenerative Diseases</span>. <i>Chem. Cent J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">65</span>, <span class="refDoi"> DOI: 10.1186/s13065-015-0143-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1186%2Fs13065-015-0143-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26617668" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC28Xpt1yht78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=65&author=X.+Chenauthor=J.+W.+Barclayauthor=R.+D.+Burgoyneauthor=A.+Morgan&title=Using+C.+Elegans+to+Discover+Therapeutic+Compounds+for+Ageing-Associated+Neurodegenerative+Diseases&doi=10.1186%2Fs13065-015-0143-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Using C. elegans to discover therapeutic compounds for ageing-associated neurodegenerative diseases</span></div><div class="casAuthors">Chen, Xi; Barclay, Jeff W.; Burgoyne, Robert D.; Morgan, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry Central Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">65/1-65/20</span>CODEN:
                <span class="NLM_cas:coden">CCJHA2</span>;
        ISSN:<span class="NLM_cas:issn">1752-153X</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Age-assocd. neurodegenerative disorders such as Alzheimer's disease are a major public health challenge, due to the demog. increase in the proportion of older individuals in society.  However, the relatively few currently approved drugs for these conditions provide only symptomatic relief.  A major goal of neurodegeneration research is therefore to identify potential new therapeutic compds. that can slow or even reverse disease progression, either by impacting directly on the neurodegenerative process or by activating endogenous physiol. neuroprotective mechanisms that decline with ageing.  This requires model systems that can recapitulate key features of human neurodegenerative diseases that are also amenable to compd. screening approaches.  Mammalian models are very powerful, but are prohibitively expensive for high-throughput drug screens.  Given the highly conserved neurol. pathways between mammals and invertebrates, Caenorhabditis elegans has emerged as a powerful tool for neuroprotective compd. screening.  Here we describe how C. elegans has been used to model various human ageing assocd. neurodegenerative diseases and provide an extensive list of compds. that have therapeutic activity in these worm models and so may have translational potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHxFDUBEtY-LVg90H21EOLACvtfcHk0lh68ax6QBDpfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xpt1yht78%253D&md5=09b5bd342e455e23e8bd26bb67cfa0f4</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1186%2Fs13065-015-0143-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13065-015-0143-y%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DBarclay%26aufirst%3DJ.%2BW.%26aulast%3DBurgoyne%26aufirst%3DR.%2BD.%26aulast%3DMorgan%26aufirst%3DA.%26atitle%3DUsing%2520C.%2520Elegans%2520to%2520Discover%2520Therapeutic%2520Compounds%2520for%2520Ageing-Associated%2520Neurodegenerative%2520Diseases%26jtitle%3DChem.%2520Cent%2520J.%26date%3D2015%26volume%3D9%26spage%3D65%26doi%3D10.1186%2Fs13065-015-0143-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegarty, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gokce, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghorashian, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Yakar, A.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Microfluidics Providing High-Resolution and High-Throughput Screening of Caenorhabditis Elegans Poly-Glutamine Aggregation Model</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13023</span>, <span class="refDoi"> DOI: 10.1038/ncomms13023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fncomms13023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=27725672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1GnurjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=13023&author=S.+Mondalauthor=E.+Hegartyauthor=C.+Martinauthor=S.+K.+Gokceauthor=N.+Ghorashianauthor=A.+Ben-Yakar&title=Large-Scale+Microfluidics+Providing+High-Resolution+and+High-Throughput+Screening+of+Caenorhabditis+Elegans+Poly-Glutamine+Aggregation+Model&doi=10.1038%2Fncomms13023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Large-scale microfluidics providing high-resolution and high-throughput screening of Caenorhabditis elegans poly-glutamine aggregation model</span></div><div class="casAuthors">Mondal, Sudip; Hegarty, Evan; Martin, Chris; Gokce, Sertan Kutal; Ghorashian, Navid; Ben-Yakar, Adela</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13023</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Next generation drug screening could benefit greatly from in vivo studies, using small animal models such as Caenorhabditis elegans for hit identification and lead optimization.  Current in vivo assays can operate either at low throughput with high resoln. or with low resoln. at high throughput.  To enable both high-throughput and high-resoln. imaging of C. elegans, we developed an automated microfluidic platform.  This platform can image 15 z-stacks of ∼4,000 C. elegans from 96 different populations using a large-scale chip with a micron resoln. in 16 min.  Using this platform, we screened ∼100,000 animals of the poly-glutamine aggregation model on 25 chips.  We tested the efficacy of ∼1,000 FDA-approved drugs in improving the aggregation phenotype of the model and identified four confirmed hits.  This robust platform now enables high-content screening of various C. elegans disease models at the speed and cost of in vitro cell-based assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaM5q1Y9A3WbVg90H21EOLACvtfcHk0lh68ax6QBDpfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1GnurjK&md5=2acb85ed4d0665977ac466ce94b7bb71</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fncomms13023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13023%26sid%3Dliteratum%253Aachs%26aulast%3DMondal%26aufirst%3DS.%26aulast%3DHegarty%26aufirst%3DE.%26aulast%3DMartin%26aufirst%3DC.%26aulast%3DGokce%26aufirst%3DS.%2BK.%26aulast%3DGhorashian%26aufirst%3DN.%26aulast%3DBen-Yakar%26aufirst%3DA.%26atitle%3DLarge-Scale%2520Microfluidics%2520Providing%2520High-Resolution%2520and%2520High-Throughput%2520Screening%2520of%2520Caenorhabditis%2520Elegans%2520Poly-Glutamine%2520Aggregation%2520Model%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D13023%26doi%3D10.1038%2Fncomms13023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Reilly, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luke, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perlmutter, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pak, S. C. C.</span></span> <span> </span><span class="NLM_article-title">Elegans in High-Throughput Drug Discovery</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>69–70</i></span>,  <span class="NLM_fpage">247</span>– <span class="NLM_lpage">253</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2013.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.addr.2013.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=24333896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOhsLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69%E2%80%9370&publication_year=2014&pages=247-253&author=L.+P.+O%E2%80%99Reillyauthor=C.+J.+Lukeauthor=D.+H.+Perlmutterauthor=G.+A.+Silvermanauthor=S.+C.+C.+Pak&title=Elegans+in+High-Throughput+Drug+Discovery&doi=10.1016%2Fj.addr.2013.12.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">C. elegans in high-throughput drug discovery</span></div><div class="casAuthors">O'Reilly, Linda P.; Luke, Cliff J.; Perlmutter, David H.; Silverman, Gary A.; Pak, Stephen C.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">69-70</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">247-253</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Caenorhabditis elegans has been proven to be a useful model organism for investigating mol. and cellular aspects of numerous human diseases.  More recently, investigators have explored the use of this organism as a tool for drug discovery.  Although earlier drug screens were labor-intensive and low in throughput, recent advances in high-throughput liq. workflows, imaging platforms and data anal. software have made C. elegans a viable option for automated high-throughput drug screens.  This review will outline the evolution of C. elegans-based drug screening, discuss the inherent challenges of using C. elegans, and highlight recent technol. advances that have paved the way for future drug screens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHxn3l10Eju7Vg90H21EOLACvtfcHk0lh68ax6QBDpfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOhsLbM&md5=112e983d48ccf79328f65e8af70f7c33</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2013.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2013.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Reilly%26aufirst%3DL.%2BP.%26aulast%3DLuke%26aufirst%3DC.%2BJ.%26aulast%3DPerlmutter%26aufirst%3DD.%2BH.%26aulast%3DSilverman%26aufirst%3DG.%2BA.%26aulast%3DPak%26aufirst%3DS.%2BC.%2BC.%26atitle%3DElegans%2520in%2520High-Throughput%2520Drug%2520Discovery%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2014%26volume%3D69%25E2%2580%259370%26spage%3D247%26epage%3D253%26doi%3D10.1016%2Fj.addr.2013.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shaye, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwald, I.</span></span> <span> </span><span class="NLM_article-title">Ortholist: A Compendium of C. Elegans Genes with Human Orthologs</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e20085</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0020085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1371%2Fjournal.pone.0020085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=21647448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFCku7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=D.+D.+Shayeauthor=I.+Greenwald&title=Ortholist%3A+A+Compendium+of+C.+Elegans+Genes+with+Human+Orthologs&doi=10.1371%2Fjournal.pone.0020085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">OrthoList: a compendium of C. elegans genes with human orthologs</span></div><div class="casAuthors">Shaye, Daniel D.; Greenwald, Iva</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e20085</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">C. elegans is an important model for genetic studies relevant to human biol. and disease.  We sought to assess the orthol. between C. elegans and human genes to understand better the relationship between their genomes and to generate a compelling list of candidates to streamline RNAi-based screens in this model.  We performed a meta-anal. of results from four orthol. prediction programs and generated a compendium, "OrthoList", contg. 7,663 C. elegans protein-coding genes.  Various assessments indicate that OrthoList has extensive coverage with low false-pos. and false-neg. rates.  Part of this evaluation examd. the conservation of components of the receptor tyrosine kinase, Notch, Wnt, TGF-β and insulin signaling pathways, and led us to update compendia of conserved C. elegans kinases, nuclear hormone receptors, F-box proteins, and transcription factors.  Comparison with two published genome-wide RNAi screens indicated that virtually all of the conserved hits would have been obtained had just the OrthoList set (∼38% of the genome) been targeted.  We compiled OrthoList by InterPro domains and Gene Ontol. annotation, making it easy to identify C. elegans orthologs of human disease genes for potential functional anal.  We anticipate that OrthoList will be of considerable utility to C. elegans researchers for streamlining RNAi screens, by focusing on genes with apparent human orthologs, thus reducing screening effort by ∼60%.  Moreover, we find that OrthoList provides a useful basis for annotating orthol. and reveals more C. elegans orthologs of human genes in various functional groups, such as transcription factors, than previously described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxUjExRD4rXrVg90H21EOLACvtfcHk0ljhduE-oWi7Cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvFCku7s%253D&md5=c73566c6c292b890536340ddd8c66580</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0020085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0020085%26sid%3Dliteratum%253Aachs%26aulast%3DShaye%26aufirst%3DD.%2BD.%26aulast%3DGreenwald%26aufirst%3DI.%26atitle%3DOrtholist%253A%2520A%2520Compendium%2520of%2520C.%2520Elegans%2520Genes%2520with%2520Human%2520Orthologs%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0020085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marfil, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span> <span> </span><span class="NLM_article-title">Use of Caenorhabditis Elegans as a Model to Study Alzheimer’s Disease and Other Neurodegenerative Diseases</span>. <i>Front. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">279</span>, <span class="refDoi"> DOI: 10.3389/fgene.2014.00279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.3389%2Ffgene.2014.00279" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25250042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC2M7kt1CktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=279&author=A.+G.+Alexanderauthor=V.+Marfilauthor=C.+Li&title=Use+of+Caenorhabditis+Elegans+as+a+Model+to+Study+Alzheimer%E2%80%99s+Disease+and+Other+Neurodegenerative+Diseases&doi=10.3389%2Ffgene.2014.00279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Use of Caenorhabditis elegans as a model to study Alzheimer's disease and other neurodegenerative diseases</span></div><div class="casAuthors">Alexander Adanna G; Li Chris; Marfil Vanessa</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in genetics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">279</span>
        ISSN:<span class="NLM_cas:issn">1664-8021</span>.
    </div><div class="casAbstract">Advances in research and technology has increased our quality of life, allowed us to combat diseases, and achieve increased longevity.  Unfortunately, increased longevity is accompanied by a rise in the incidences of age-related diseases such as Alzheimer's disease (AD).  AD is the sixth leading cause of death, and one of the leading causes of dementia amongst the aged population in the USA.  It is a progressive neurodegenerative disorder, characterized by the prevalence of extracellular Aβ plaques and intracellular neurofibrillary tangles, derived from the proteolysis of the amyloid precursor protein (APP) and the hyperphosphorylation of microtubule-associated protein tau, respectively.  Despite years of extensive research, the molecular mechanisms that underlie the pathology of AD remain unclear.  Model organisms, such as the nematode, Caenorhabditis elegans, present a complementary approach to addressing these questions.  C. elegans has many advantages as a model system to study AD and other neurodegenerative diseases.  Like their mammalian counterparts, they have complex biochemical pathways, most of which are conserved.  Genes in which mutations are correlated with AD have counterparts in C. elegans, including an APP-related gene, apl-1, a tau homolog, ptl-1, and presenilin homologs, such as sel-12 and hop-1.  Since the neuronal connectivity in C. elegans has already been established, C. elegans is also advantageous in modeling learning and memory impairments seen during AD.  This article addresses the insights C. elegans provide in studying AD and other neurodegenerative diseases.  Additionally, we explore the advantages and drawbacks associated with using this model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRa3vraTEWAPtFxYDaSSP8XfW6udTcc2eZMWMy707KL6rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M7kt1CktA%253D%253D&md5=7324b86ac9589e7e2aa8185a6903327f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.3389%2Ffgene.2014.00279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffgene.2014.00279%26sid%3Dliteratum%253Aachs%26aulast%3DAlexander%26aufirst%3DA.%2BG.%26aulast%3DMarfil%26aufirst%3DV.%26aulast%3DLi%26aufirst%3DC.%26atitle%3DUse%2520of%2520Caenorhabditis%2520Elegans%2520as%2520a%2520Model%2520to%2520Study%2520Alzheimer%25E2%2580%2599s%2520Disease%2520and%2520Other%2520Neurodegenerative%2520Diseases%26jtitle%3DFront.%2520Genet.%26date%3D2014%26volume%3D5%26spage%3D279%26doi%3D10.3389%2Ffgene.2014.00279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGurk, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonini, N. M.</span></span> <span> </span><span class="NLM_article-title">Drosophila as an in Vivo Model for Human Neurodegenerative Disease</span>. <i>Genetics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>201</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">402</span>, <span class="refDoi"> DOI: 10.1534/genetics.115.179457</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1534%2Fgenetics.115.179457" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26447127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC28XlslGqs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=201&publication_year=2015&pages=377-402&author=L.+McGurkauthor=A.+Bersonauthor=N.+M.+Bonini&title=Drosophila+as+an+in+Vivo+Model+for+Human+Neurodegenerative+Disease&doi=10.1534%2Fgenetics.115.179457"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Drosophila as an in vivo model for human neurodegenerative disease</span></div><div class="casAuthors">McGurk, Leeanne; Berson, Amit; Bonini, Nancy M.</div><div class="citationInfo"><span class="NLM_cas:title">Genetics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">201</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-402</span>CODEN:
                <span class="NLM_cas:coden">GENTAE</span>;
        ISSN:<span class="NLM_cas:issn">0016-6731</span>.
    
            (<span class="NLM_cas:orgname">Genetics Society of America</span>)
        </div><div class="casAbstract">With the increase in the ageing population, neurodegenerative disease is devastating to families and poses a huge burden on society.  The brain and spinal cord are extraordinarily complex: they consist of a highly organized network of neuronal and support cells that communicate in a highly specialized manner.  One approach to tackling problems of such complexity is to address the scientific questions in simpler, yet analogous, systems.  The fruit fly, Drosophila melanogaster, has been proven tremendously valuable as a model organism, enabling many major discoveries in neuroscientific disease research.  The plethora of genetic tools available in Drosophila allows for exquisite targeted manipulation of the genome.  Due to its relatively short lifespan, complex questions of brain function can be addressed more rapidly than in other model organisms, such as the mouse.  Here we discuss features of the fly as a model for human neurodegenerative disease.  There are many distinct fly models for a range of neurodegenerative diseases; we focus on select studies from models of polyglutamine disease and amyotrophic lateral sclerosis that illustrate the type and range of insights that can be gleaned.  In discussion of these models, we underscore strengths of the fly in providing understanding into mechanisms and pathways, as a foundation for translational and therapeutic research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhbyqPD12u_LVg90H21EOLACvtfcHk0ljhduE-oWi7Cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlslGqs7g%253D&md5=f30a80149c7f18e7a1247b326134d528</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1534%2Fgenetics.115.179457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1534%252Fgenetics.115.179457%26sid%3Dliteratum%253Aachs%26aulast%3DMcGurk%26aufirst%3DL.%26aulast%3DBerson%26aufirst%3DA.%26aulast%3DBonini%26aufirst%3DN.%2BM.%26atitle%3DDrosophila%2520as%2520an%2520in%2520Vivo%2520Model%2520for%2520Human%2520Neurodegenerative%2520Disease%26jtitle%3DGenetics%26date%3D2015%26volume%3D201%26spage%3D377%26epage%3D402%26doi%3D10.1534%2Fgenetics.115.179457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potocki, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chien, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gribskov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bier, E.</span></span> <span> </span><span class="NLM_article-title">A Systematic Analysis of Human Disease-Associated Gene Sequences in Drosophila Melanogaster</span>. <i>Genome Res.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1114</span>– <span class="NLM_lpage">1125</span>, <span class="refDoi"> DOI: 10.1101/gr.169101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1101%2Fgr.169101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=11381037" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BD3MXkt12ru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=1114-1125&author=L.+T.+Reiterauthor=L.+Potockiauthor=S.+Chienauthor=M.+Gribskovauthor=E.+Bier&title=A+Systematic+Analysis+of+Human+Disease-Associated+Gene+Sequences+in+Drosophila+Melanogaster&doi=10.1101%2Fgr.169101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A systematic analysis of human disease-associated gene sequences in Drosophila melanogaster</span></div><div class="casAuthors">Reiter, Lawrence T.; Potocki, Lorraine; Chien, Sam; Gribskov, Michael; Bier, Ethan</div><div class="citationInfo"><span class="NLM_cas:title">Genome Research</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1114-1125</span>CODEN:
                <span class="NLM_cas:coden">GEREFS</span>;
        ISSN:<span class="NLM_cas:issn">1088-9051</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">We performed a systematic BLAST anal. of 929 human disease gene entries assocd. with at least one mutant allele in the Online Mendelian Inheritance in Man (OMIM) database against the recently completed genome sequence of Drosophila melanogaster.  The results of this search have been formatted as an updateable and searchable online database called Homophila.  Our anal. identified 714 distinct human disease genes (77% of disease genes searched) matching 548 unique Drosophila sequences, which we have summarized by disease category.  This breakdown into disease classes creates a picture of disease genes that are amenable to study using Drosophila as the model organism.  Of the 548 Drosophila genes related to human disease genes, 153 are assocd. with known mutant alleles and 56 more are tagged by P-element insertions in or near the gene.  Examples of how to use the database to identify Drosophila genes related to human disease genes are presented.  We anticipate that cross-genomic anal. of human disease genes using the power of Drosophila second-site modifier screens will promote interaction between human and Drosophila research groups, accelerating the understanding of the pathogenesis of human genetic disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodyv8TO5yrqbVg90H21EOLACvtfcHk0liY-INgwR0Kjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXkt12ru7s%253D&md5=9eec727a7b935b3f5fe3783e93fe55d5</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1101%2Fgr.169101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgr.169101%26sid%3Dliteratum%253Aachs%26aulast%3DReiter%26aufirst%3DL.%2BT.%26aulast%3DPotocki%26aufirst%3DL.%26aulast%3DChien%26aufirst%3DS.%26aulast%3DGribskov%26aufirst%3DM.%26aulast%3DBier%26aufirst%3DE.%26atitle%3DA%2520Systematic%2520Analysis%2520of%2520Human%2520Disease-Associated%2520Gene%2520Sequences%2520in%2520Drosophila%2520Melanogaster%26jtitle%3DGenome%2520Res.%26date%3D2001%26volume%3D11%26spage%3D1114%26epage%3D1125%26doi%3D10.1101%2Fgr.169101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rihel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prober, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arvanites, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schier, A. F.</span></span> <span> </span><span class="NLM_article-title">Zebrafish Behavioral Profiling Links Drugs to Biological Targets and Rest/Wake Regulation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>327</i></span>,  <span class="NLM_fpage">348</span>– <span class="NLM_lpage">351</span>, <span class="refDoi"> DOI: 10.1126/science.1183090</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fscience.1183090" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=20075256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktlWksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=348-351&author=J.+Rihelauthor=D.+A.+Proberauthor=A.+Arvanitesauthor=K.+Lamauthor=S.+Zimmermanauthor=S.+Jangauthor=S.+J.+Haggartyauthor=D.+Kokelauthor=L.+L.+Rubinauthor=R.+T.+Petersonauthor=A.+F.+Schier&title=Zebrafish+Behavioral+Profiling+Links+Drugs+to+Biological+Targets+and+Rest%2FWake+Regulation&doi=10.1126%2Fscience.1183090"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Zebrafish Behavioral Profiling Links Drugs to Biological Targets and Rest/Wake Regulation</span></div><div class="casAuthors">Rihel, Jason; Prober, David A.; Arvanites, Anthony; Lam, Kelvin; Zimmerman, Steven; Jang, Sumin; Haggarty, Stephen J.; Kokel, David; Rubin, Lee L.; Peterson, Randall T.; Schier, Alexander F.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5963</span>),
    <span class="NLM_cas:pages">348-351</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A major obstacle for the discovery of psychoactive drugs is the inability to predict how small mols. will alter complex behaviors.  We report the development and application of a high-throughput, quant. screen for drugs that alter the behavior of larval zebrafish.  We found that the multidimensional nature of obsd. phenotypes enabled the hierarchical clustering of mols. according to shared behaviors.  Behavioral profiling revealed conserved functions of psychotropic mols. and predicted the mechanisms of action of poorly characterized compds.  In addn., behavioral profiling implicated new factors such as ether-a-go-go-related gene (ERG) potassium channels and immunomodulators in the control of rest and locomotor activity.  These results demonstrate the power of high-throughput behavioral profiling in zebrafish to discover and characterize psychotropic drugs and to dissect the pharmacol. of complex behaviors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbwHl9CPNU_7Vg90H21EOLACvtfcHk0liY-INgwR0Kjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktlWksw%253D%253D&md5=6a039c0c06cba3a364230aa24f84e13e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1126%2Fscience.1183090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1183090%26sid%3Dliteratum%253Aachs%26aulast%3DRihel%26aufirst%3DJ.%26aulast%3DProber%26aufirst%3DD.%2BA.%26aulast%3DArvanites%26aufirst%3DA.%26aulast%3DLam%26aufirst%3DK.%26aulast%3DZimmerman%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DS.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DKokel%26aufirst%3DD.%26aulast%3DRubin%26aufirst%3DL.%2BL.%26aulast%3DPeterson%26aufirst%3DR.%2BT.%26aulast%3DSchier%26aufirst%3DA.%2BF.%26atitle%3DZebrafish%2520Behavioral%2520Profiling%2520Links%2520Drugs%2520to%2520Biological%2520Targets%2520and%2520Rest%252FWake%2520Regulation%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D348%26epage%3D351%26doi%3D10.1126%2Fscience.1183090" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">MacRae, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, R. T.</span></span> <span> </span><span class="NLM_article-title">Zebrafish as Tools for Drug Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">721</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1038/nrd4627</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnrd4627" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26361349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyrtbzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=721-731&author=C.+A.+MacRaeauthor=R.+T.+Peterson&title=Zebrafish+as+Tools+for+Drug+Discovery&doi=10.1038%2Fnrd4627"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Zebrafish as tools for drug discovery</span></div><div class="casAuthors">MacRae, Calum A.; Peterson, Randall T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">721-731</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The zebrafish has become a prominent vertebrate model for disease and has already contributed to several examples of successful phenotype-based drug discovery.  For the zebrafish to become useful in drug development more broadly, key hurdles must be overcome, including a more comprehensive elucidation of the similarities and differences between human and zebrafish biol.  Recent studies have begun to establish the capabilities and limitations of zebrafish for disease modeling, drug screening, target identification, pharmacol., and toxicol.  As our understanding increases and as the technologies for manipulating zebrafish improve, it is hoped that the zebrafish will have a key role in accelerating the emergence of precision medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGcnVtHqzJXbVg90H21EOLACvtfcHk0liY-INgwR0Kjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyrtbzJ&md5=1f8733e7468a76738df9c130c493a3b8</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnrd4627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4627%26sid%3Dliteratum%253Aachs%26aulast%3DMacRae%26aufirst%3DC.%2BA.%26aulast%3DPeterson%26aufirst%3DR.%2BT.%26atitle%3DZebrafish%2520as%2520Tools%2520for%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D721%26epage%3D731%26doi%3D10.1038%2Fnrd4627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parenteau, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veronneau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacombe, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecez, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gervais-Bird, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koh, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunelle, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellinger, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chabot, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abou Elela, S.</span></span> <span> </span><span class="NLM_article-title">Deletion of Many Yeast Introns Reveals a Minority of Genes That Require Splicing for Function</span>. <i>Mol. Biol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1932</span>– <span class="NLM_lpage">1941</span>, <span class="refDoi"> DOI: 10.1091/mbc.e07-12-1254</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1091%2Fmbc.e07-12-1254" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=18287520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlslelsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2008&pages=1932-1941&author=J.+Parenteauauthor=M.+Durandauthor=S.+Veronneauauthor=A.+A.+Lacombeauthor=G.+Morinauthor=V.+Guerinauthor=B.+Cecezauthor=J.+Gervais-Birdauthor=C.+S.+Kohauthor=D.+Brunelleauthor=R.+J.+Wellingerauthor=B.+Chabotauthor=S.+Abou+Elela&title=Deletion+of+Many+Yeast+Introns+Reveals+a+Minority+of+Genes+That+Require+Splicing+for+Function&doi=10.1091%2Fmbc.e07-12-1254"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Deletion of many yeast introns reveals a minority of genes that require splicing for function</span></div><div class="casAuthors">Parenteau, Julie; Durand, Mathieu; Veronneau, Steeve; Lacombe, Andree-Anne; Morin, Genevieve; Guerin, Valerie; Cecez, Bojana; Gervais-Bird, Julien; Koh, Chu-Shin; Brunelle, David; Wellinger, Raymund J.; Chabot, Benoit; Elela, Sherif Abou</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology of the Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1932-1941</span>CODEN:
                <span class="NLM_cas:coden">MBCEEV</span>;
        ISSN:<span class="NLM_cas:issn">1939-4586</span>.
    
            (<span class="NLM_cas:orgname">American Society for Cell Biology</span>)
        </div><div class="casAbstract">Splicing regulates gene expression and contributes to proteomic diversity in higher eukaryotes.  However, in yeast only 283 of the 6000 genes contain introns and their impact on cell function is not clear.  To assess the contribution of introns to cell function, we initiated large-scale intron deletions in yeast with the ultimate goal of creating an intron-free model eukaryote.  We show that about one-third of yeast introns are not essential for growth.  Only three intron deletions caused severe growth defects, but normal growth was restored in all cases by expressing the intronless mRNA from a heterologous promoter.  Twenty percent of the intron deletions caused minor phenotypes under different growth conditions.  Strikingly, the combined deletion of all introns from the 15 cytoskeleton-related genes did not affect growth or strain fitness.  Together, our results show that although the presence of introns may optimize gene expression and provide benefit under stress, a majority of introns could be removed with minor consequences on growth under lab. conditions, supporting the view that many introns could be phased out of Saccharomyces cerevisiae without blocking cell growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvTOfcnp6-VbVg90H21EOLACvtfcHk0lj3vzgjN8HhWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlslelsrc%253D&md5=85dee50c2433d9800359059e7d97afae</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1091%2Fmbc.e07-12-1254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1091%252Fmbc.e07-12-1254%26sid%3Dliteratum%253Aachs%26aulast%3DParenteau%26aufirst%3DJ.%26aulast%3DDurand%26aufirst%3DM.%26aulast%3DVeronneau%26aufirst%3DS.%26aulast%3DLacombe%26aufirst%3DA.%2BA.%26aulast%3DMorin%26aufirst%3DG.%26aulast%3DGuerin%26aufirst%3DV.%26aulast%3DCecez%26aufirst%3DB.%26aulast%3DGervais-Bird%26aufirst%3DJ.%26aulast%3DKoh%26aufirst%3DC.%2BS.%26aulast%3DBrunelle%26aufirst%3DD.%26aulast%3DWellinger%26aufirst%3DR.%2BJ.%26aulast%3DChabot%26aufirst%3DB.%26aulast%3DAbou%2BElela%26aufirst%3DS.%26atitle%3DDeletion%2520of%2520Many%2520Yeast%2520Introns%2520Reveals%2520a%2520Minority%2520of%2520Genes%2520That%2520Require%2520Splicing%2520for%2520Function%26jtitle%3DMol.%2520Biol.%2520Cell%26date%3D2008%26volume%3D19%26spage%3D1932%26epage%3D1941%26doi%3D10.1091%2Fmbc.e07-12-1254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonnhammer, E. L.</span></span> <span> </span><span class="NLM_article-title">Inparanoid: A Comprehensive Database of Eukaryotic Orthologs</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">D476</span>– <span class="NLM_lpage">D480</span>, <span class="refDoi"> DOI: 10.1093/nar/gki107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1093%2Fnar%2Fgki107" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2004&pages=D476-D480&author=K.+P.+O%E2%80%99Brienauthor=M.+Remmauthor=E.+L.+Sonnhammer&title=Inparanoid%3A+A+Comprehensive+Database+of+Eukaryotic+Orthologs&doi=10.1093%2Fnar%2Fgki107"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgki107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgki107%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DK.%2BP.%26aulast%3DRemm%26aufirst%3DM.%26aulast%3DSonnhammer%26aufirst%3DE.%2BL.%26atitle%3DInparanoid%253A%2520A%2520Comprehensive%2520Database%2520of%2520Eukaryotic%2520Orthologs%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2004%26volume%3D33%26spage%3DD476%26epage%3DD480%26doi%3D10.1093%2Fnar%2Fgki107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kachroo, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurent, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yellman, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilke, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, E. M.</span></span> <span> </span><span class="NLM_article-title">Evolution. Systematic Humanization of Yeast Genes Reveals Conserved Functions and Genetic Modularity</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">921</span>– <span class="NLM_lpage">925</span>, <span class="refDoi"> DOI: 10.1126/science.aaa0769</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fscience.aaa0769" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25999509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosFaksbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=921-925&author=A.+H.+Kachrooauthor=J.+M.+Laurentauthor=C.+M.+Yellmanauthor=A.+G.+Meyerauthor=C.+O.+Wilkeauthor=E.+M.+Marcotte&title=Evolution.+Systematic+Humanization+of+Yeast+Genes+Reveals+Conserved+Functions+and+Genetic+Modularity&doi=10.1126%2Fscience.aaa0769"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic humanization of yeast genes reveals conserved functions and genetic modularity</span></div><div class="casAuthors">Kachroo, Aashiq H.; Laurent, Jon M.; Yellman, Christopher M.; Meyer, Austin G.; Wilke, Claus O.; Marcotte, Edward M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6237</span>),
    <span class="NLM_cas:pages">921-925</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">To det. whether genes retain ancestral functions over a billion years of evolution and to identify principles of deep evolutionary divergence, we replaced 414 essential yeast genes with their human orthologs, assaying for complementation of lethal growth defects upon loss of the yeast genes.  Nearly half (47%) of the yeast genes could be successfully humanized.  Sequence similarity and expression only partly predicted replaceability.  Instead, replaceability depended strongly on gene modules: Genes in the same process tended to be similarly replaceable (e.g., sterol biosynthesis) or not (e.g., DNA replication initiation).  Simulations confirmed that selection for specific function can maintain replaceability despite extensive sequence divergence.  Crit. ancestral functions of many essential genes are thus retained in a pathway-specific manner, resilient to drift in sequences, splicing, and protein interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqPJvZKreAPLVg90H21EOLACvtfcHk0lj3vzgjN8HhWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosFaksbg%253D&md5=422322e621ad3d1ad31e99195f9e3ead</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaa0769&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaa0769%26sid%3Dliteratum%253Aachs%26aulast%3DKachroo%26aufirst%3DA.%2BH.%26aulast%3DLaurent%26aufirst%3DJ.%2BM.%26aulast%3DYellman%26aufirst%3DC.%2BM.%26aulast%3DMeyer%26aufirst%3DA.%2BG.%26aulast%3DWilke%26aufirst%3DC.%2BO.%26aulast%3DMarcotte%26aufirst%3DE.%2BM.%26atitle%3DEvolution.%2520Systematic%2520Humanization%2520of%2520Yeast%2520Genes%2520Reveals%2520Conserved%2520Functions%2520and%2520Genetic%2520Modularity%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D921%26epage%3D925%26doi%3D10.1126%2Fscience.aaa0769" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tardiff, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piotrowski, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aron, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bacherman, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pires, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramaniam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doshi, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadlish, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raja, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solis, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khurana, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Bourdonnec, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scannevin, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhodes, K. J.</span></span> <span> </span><span class="NLM_article-title">Inhibiting Stearoyl-Coa Desaturase Ameliorates Alpha-Synuclein Cytotoxicity</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2742</span>– <span class="NLM_lpage">2754</span>, e2731 <span class="refDoi"> DOI: 10.1016/j.celrep.2018.11.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.celrep.2018.11.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30517862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVCit7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=2742-2754&author=B.+M.+Vincentauthor=D.+F.+Tardiffauthor=J.+S.+Piotrowskiauthor=R.+Aronauthor=M.+C.+Lucasauthor=C.+Y.+Chungauthor=H.+Bachermanauthor=Y.+Chenauthor=M.+Piresauthor=R.+Subramaniamauthor=D.+B.+Doshiauthor=H.+Sadlishauthor=W.+K.+Rajaauthor=E.+J.+Solisauthor=V.+Khuranaauthor=B.+Le+Bourdonnecauthor=R.+H.+Scannevinauthor=K.+J.+Rhodes&title=Inhibiting+Stearoyl-Coa+Desaturase+Ameliorates+Alpha-Synuclein+Cytotoxicity&doi=10.1016%2Fj.celrep.2018.11.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting Stearoyl-CoA Desaturase Ameliorates α-Synuclein Cytotoxicity</span></div><div class="casAuthors">Vincent, Benjamin M.; Tardiff, Daniel F.; Piotrowski, Jeff S.; Aron, Rebecca; Lucas, Matthew C.; Chung, Chee Yeun; Bacherman, Helene; Chen, YiQun; Pires, Michelle; Subramaniam, Radha; Doshi, Dimple B.; Sadlish, Heather; Raja, Waseem K.; Solis, Eric J.; Khurana, Vikram; Le Bourdonnec, Bertrand; Scannevin, Robert H.; Rhodes, Kenneth J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2742-2754.e31</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The lack of disease-modifying treatments for neurodegenerative disease stems in part from our rudimentary understanding of disease mechanisms and the paucity of targets for therapeutic intervention.  Here we used an integrated discovery paradigm to identify a new therapeutic target for diseases caused by α-synuclein (α-syn), a small lipid-binding protein that misfolds and aggregates in Parkinson's disease and other disorders.  Using unbiased phenotypic screening, we identified a series of compds. that were cytoprotective against α-syn-mediated toxicity by inhibiting the highly conserved enzyme stearoyl-CoA desaturase (SCD).  Critically, reducing the levels of unsatd. membrane lipids by inhibiting SCD reduced α-syn toxicity in human induced pluripotent stem cell (iPSC) neuronal models.  Taken together, these findings suggest that inhibition of fatty acid desatn. has potential as a therapeutic approach for the treatment of Parkinson's disease and other synucleinopathies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2xtNxbIcEk7Vg90H21EOLACvtfcHk0lgIXaCralzPeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVCit7fM&md5=1e1a2ffb251ff2a91f638b9a247cb2c0</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2018.11.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2018.11.028%26sid%3Dliteratum%253Aachs%26aulast%3DVincent%26aufirst%3DB.%2BM.%26aulast%3DTardiff%26aufirst%3DD.%2BF.%26aulast%3DPiotrowski%26aufirst%3DJ.%2BS.%26aulast%3DAron%26aufirst%3DR.%26aulast%3DLucas%26aufirst%3DM.%2BC.%26aulast%3DChung%26aufirst%3DC.%2BY.%26aulast%3DBacherman%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DPires%26aufirst%3DM.%26aulast%3DSubramaniam%26aufirst%3DR.%26aulast%3DDoshi%26aufirst%3DD.%2BB.%26aulast%3DSadlish%26aufirst%3DH.%26aulast%3DRaja%26aufirst%3DW.%2BK.%26aulast%3DSolis%26aufirst%3DE.%2BJ.%26aulast%3DKhurana%26aufirst%3DV.%26aulast%3DLe%2BBourdonnec%26aufirst%3DB.%26aulast%3DScannevin%26aufirst%3DR.%2BH.%26aulast%3DRhodes%26aufirst%3DK.%2BJ.%26atitle%3DInhibiting%2520Stearoyl-Coa%2520Desaturase%2520Ameliorates%2520Alpha-Synuclein%2520Cytotoxicity%26jtitle%3DCell%2520Rep.%26date%3D2018%26volume%3D25%26spage%3D2742%26epage%3D2754%26doi%3D10.1016%2Fj.celrep.2018.11.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auluck, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Outeiro, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeger-Lotem, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kritzer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tardiff, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strathearn, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamamichi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraenkel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCaffery, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rochet, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindquist, S.</span></span> <span> </span><span class="NLM_article-title">Compounds from an Unbiased Chemical Screen Reverse Both Er-to-Golgi Trafficking Defects and Mitochondrial Dysfunction in Parkinson’s Disease Models</span>. <i>Dis. Models &amp; Mech.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">194</span>– <span class="NLM_lpage">208</span>, <span class="refDoi"> DOI: 10.1242/dmm.004267</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1242%2Fdmm.004267" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=20038714" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2010&pages=194-208&author=L.+J.+Suauthor=P.+K.+Auluckauthor=T.+F.+Outeiroauthor=E.+Yeger-Lotemauthor=J.+A.+Kritzerauthor=D.+F.+Tardiffauthor=K.+E.+Strathearnauthor=F.+Liuauthor=S.+Caoauthor=S.+Hamamichiauthor=K.+J.+Hillauthor=K.+A.+Caldwellauthor=G.+W.+Bellauthor=E.+Fraenkelauthor=A.+A.+Cooperauthor=G.+A.+Caldwellauthor=J.+M.+McCafferyauthor=J.+C.+Rochetauthor=S.+Lindquist&title=Compounds+from+an+Unbiased+Chemical+Screen+Reverse+Both+Er-to-Golgi+Trafficking+Defects+and+Mitochondrial+Dysfunction+in+Parkinson%E2%80%99s+Disease+Models&doi=10.1242%2Fdmm.004267"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1242%2Fdmm.004267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fdmm.004267%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DL.%2BJ.%26aulast%3DAuluck%26aufirst%3DP.%2BK.%26aulast%3DOuteiro%26aufirst%3DT.%2BF.%26aulast%3DYeger-Lotem%26aufirst%3DE.%26aulast%3DKritzer%26aufirst%3DJ.%2BA.%26aulast%3DTardiff%26aufirst%3DD.%2BF.%26aulast%3DStrathearn%26aufirst%3DK.%2BE.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DCao%26aufirst%3DS.%26aulast%3DHamamichi%26aufirst%3DS.%26aulast%3DHill%26aufirst%3DK.%2BJ.%26aulast%3DCaldwell%26aufirst%3DK.%2BA.%26aulast%3DBell%26aufirst%3DG.%2BW.%26aulast%3DFraenkel%26aufirst%3DE.%26aulast%3DCooper%26aufirst%3DA.%2BA.%26aulast%3DCaldwell%26aufirst%3DG.%2BA.%26aulast%3DMcCaffery%26aufirst%3DJ.%2BM.%26aulast%3DRochet%26aufirst%3DJ.%2BC.%26aulast%3DLindquist%26aufirst%3DS.%26atitle%3DCompounds%2520from%2520an%2520Unbiased%2520Chemical%2520Screen%2520Reverse%2520Both%2520Er-to-Golgi%2520Trafficking%2520Defects%2520and%2520Mitochondrial%2520Dysfunction%2520in%2520Parkinson%25E2%2580%2599s%2520Disease%2520Models%26jtitle%3DDis.%2520Models%2520%2526amp%253B%2520Mech.%26date%3D2010%26volume%3D3%26spage%3D194%26epage%3D208%26doi%3D10.1242%2Fdmm.004267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halliday, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radford, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zents, K. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molloy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verity, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortori, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bushell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallucci, G. R.</span></span> <span> </span><span class="NLM_article-title">Repurposed Drugs Targeting Eif2&Alpha;-P-Mediated Translational Repression Prevent Neurodegeneration in Mice</span>. <i>Brain</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">1768</span>– <span class="NLM_lpage">1783</span>, <span class="refDoi"> DOI: 10.1093/brain/awx074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1093%2Fbrain%2Fawx074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28430857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC1cvpvFGnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2017&pages=1768-1783&author=M.+Hallidayauthor=H.+Radfordauthor=K.+A.+M.+Zentsauthor=C.+Molloyauthor=J.+A.+Morenoauthor=N.+C.+Verityauthor=E.+Smithauthor=C.+A.+Ortoriauthor=D.+A.+Barrettauthor=M.+Bushellauthor=G.+R.+Mallucci&title=Repurposed+Drugs+Targeting+Eif2%26Alpha%3B-P-Mediated+Translational+Repression+Prevent+Neurodegeneration+in+Mice&doi=10.1093%2Fbrain%2Fawx074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Repurposed drugs targeting eIF2&alpha;-P-mediated translational repression prevent neurodegeneration in mice</span></div><div class="casAuthors">Halliday Mark; Radford Helois; Molloy Collin; Moreno Julie A; Verity Nicholas C; Smith Ewan; Bushell Martin; Mallucci Giovanna R; Zents Karlijn A M; Mallucci Giovanna R; Ortori Catharine A; Barrett David A</div><div class="citationInfo"><span class="NLM_cas:title">Brain : a journal of neurology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1768-1783</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">See Mercado and Hetz (doi:10.1093/brain/awx107) for a scientific commentary on this article.Signalling through the PERK/eIF2&alpha;-P branch of the unfolded protein response plays a critical role in controlling protein synthesis rates in cells.  This pathway is overactivated in brains of patients with Alzheimer&rsquo;s disease and related disorders and has recently emerged as a promising therapeutic target for these currently untreatable conditions.  Thus, in mouse models of neurodegenerative disease, prolonged overactivation of PERK/eIF2&alpha;-P signalling causes sustained attenuation of protein synthesis, leading to memory impairment and neuronal loss.  Re-establishing translation rates by inhibition of eIF2&alpha;-P activity, genetically or pharmacologically, restores memory and prevents neurodegeneration and extends survival.  However, the experimental compounds used preclinically are unsuitable for use in humans, due to associated toxicity or poor pharmacokinetic properties.  To discover compounds that have anti-eIF2&alpha;-P activity suitable for clinical use, we performed phenotypic screens on a NINDS small molecule library of 1040 drugs.  We identified two compounds, trazodone hydrochloride and dibenzoylmethane, which reversed eIF2&alpha;-P-mediated translational attenuation in vitro and in vivo.  Both drugs were markedly neuroprotective in two mouse models of neurodegeneration, using clinically relevant doses over a prolonged period of time, without systemic toxicity.  Thus, in prion-diseased mice, both trazodone and dibenzoylmethane treatment restored memory deficits, abrogated development of neurological signs, prevented neurodegeneration and significantly prolonged survival.  In tauopathy-frontotemporal dementia mice, both drugs were neuroprotective, rescued memory deficits and reduced hippocampal atrophy.  Further, trazodone reduced p-tau burden.  These compounds therefore represent potential new disease-modifying treatments for dementia.  Trazodone in particular, a licensed drug, should now be tested in clinical trials in patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTdeaJRg2dW4UNHe_gFldTbfW6udTcc2ebSZB-bIvMNXrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cvpvFGnsw%253D%253D&md5=fb5be53ef4ec468839c4c85278c43f43</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1093%2Fbrain%2Fawx074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbrain%252Fawx074%26sid%3Dliteratum%253Aachs%26aulast%3DHalliday%26aufirst%3DM.%26aulast%3DRadford%26aufirst%3DH.%26aulast%3DZents%26aufirst%3DK.%2BA.%2BM.%26aulast%3DMolloy%26aufirst%3DC.%26aulast%3DMoreno%26aufirst%3DJ.%2BA.%26aulast%3DVerity%26aufirst%3DN.%2BC.%26aulast%3DSmith%26aufirst%3DE.%26aulast%3DOrtori%26aufirst%3DC.%2BA.%26aulast%3DBarrett%26aufirst%3DD.%2BA.%26aulast%3DBushell%26aufirst%3DM.%26aulast%3DMallucci%26aufirst%3DG.%2BR.%26atitle%3DRepurposed%2520Drugs%2520Targeting%2520Eif2%2526Alpha%253B-P-Mediated%2520Translational%2520Repression%2520Prevent%2520Neurodegeneration%2520in%2520Mice%26jtitle%3DBrain%26date%3D2017%26volume%3D140%26spage%3D1768%26epage%3D1783%26doi%3D10.1093%2Fbrain%2Fawx074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luesch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supek, F.</span></span> <span> </span><span class="NLM_article-title">A Genome-Wide Overexpression Screen in Yeast for Small-Molecule Target Identification</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2004.10.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.chembiol.2004.10.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=15664515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmtlKmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2005&pages=55-63&author=H.+Lueschauthor=T.+Y.+Wuauthor=P.+Renauthor=N.+S.+Grayauthor=P.+G.+Schultzauthor=F.+Supek&title=A+Genome-Wide+Overexpression+Screen+in+Yeast+for+Small-Molecule+Target+Identification&doi=10.1016%2Fj.chembiol.2004.10.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A Genome-Wide Overexpression Screen in Yeast for Small-Molecule Target Identification</span></div><div class="casAuthors">Luesch, Hendrik; Wu, Tom Y. H.; Ren, Pingda; Gray, Nathanael S.; Schultz, Peter G.; Supek, Frantisek</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-63</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We describe a multicopy gene suppression screen of drug sensitivity in Saccharomyces cerevisiae that facilitates the identification of cellular targets of small mols.  An array of yeast transformants harboring a multicopy yeast genomic library was screened for resistance to growth inhibitors.  Comparison of array growth patterns for several such inhibitors allowed the differentiation of general and mol.-specific genetic suppressors.  Specific resistance to phenylaminopyrimidine (1), an inhibitor identified from a kinase-directed library, was assocd. with the overexpression of Pkc1 and a subset of downstream kinases.  Components of two other pathways (pheromone response/filamentous growth and Pho85 kinase) that genetically interact with the PKC1 MAPK signaling cascade were also identified.  Consistent with the suppression screen, inhibitor 1 bound to Pkc1 in yeast cell lysate and inhibited its activity in vitro.  These results demonstrate the utility of this approach for the rapid deconvolution of small-mol. targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCcPHiH0TixrVg90H21EOLACvtfcHk0li5XVCrYCYcKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmtlKmsg%253D%253D&md5=a2cd6a2fc60ea8c785f1d375dd6d8178</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2004.10.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2004.10.015%26sid%3Dliteratum%253Aachs%26aulast%3DLuesch%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DT.%2BY.%26aulast%3DRen%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26aulast%3DSupek%26aufirst%3DF.%26atitle%3DA%2520Genome-Wide%2520Overexpression%2520Screen%2520in%2520Yeast%2520for%2520Small-Molecule%2520Target%2520Identification%26jtitle%3DChem.%2520Biol.%26date%3D2005%26volume%3D12%26spage%3D55%26epage%3D63%26doi%3D10.1016%2Fj.chembiol.2004.10.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Underwood, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwald, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaye, D. D.</span></span> <span> </span><span class="NLM_article-title">Ortholist 2: A New Comparative Genomic Analysis of Human and Caenorhabditis Elegans Genes</span>. <i>Genetics</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>210</i></span>,  <span class="NLM_fpage">445</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1534/genetics.118.301307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1534%2Fgenetics.118.301307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30120140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjsVyit7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=210&publication_year=2018&pages=445-461&author=W.+Kimauthor=R.+S.+Underwoodauthor=I.+Greenwaldauthor=D.+D.+Shaye&title=Ortholist+2%3A+A+New+Comparative+Genomic+Analysis+of+Human+and+Caenorhabditis+Elegans+Genes&doi=10.1534%2Fgenetics.118.301307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">OrthoList 2: a new comparative genomic analysis of human and Caenorhabditis elegans genes</span></div><div class="casAuthors">Kim, Woojin; Underwood, Ryan S.; Greenwald, Iva; Shaye, Daniel D.</div><div class="citationInfo"><span class="NLM_cas:title">Genetics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">210</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">445-461</span>CODEN:
                <span class="NLM_cas:coden">GENTAE</span>;
        ISSN:<span class="NLM_cas:issn">0016-6731</span>.
    
            (<span class="NLM_cas:orgname">Genetics Society of America</span>)
        </div><div class="casAbstract">OrthoList, a compendium of Caenorhabditis elegans genes with human orthologs compiled in 2011 by a meta-anal. of four orthol.-prediction methods, has been a popular tool for identifying conserved genes for research into biol. and disease mechanisms.  However, the efficacy of orthol. prediction depends on the accuracy of gene-model predictions, an ongoing process, and orthol.-prediction algorithms have also been updated over time.  Here we present OrthoList 2 (OL2), a new comparative genomic anal. between C. elegans and humans, and the first assessment of how changes over time affect the landscape of predicted orthologs between two species.  Although we find that updates to the orthol.-prediction methods significantly changed the landscape of C. elegans-human orthologs predicted by individual programs and-unexpectedly-reduced agreement among them, we also show that our meta-anal. approach "buffered" against changes in gene content.  We show that adding results from more programs did not lead to many addns. to the list and discuss reasons to avoid assigning "scores" based on support by individual orthol.-prediction programs; the treatment of "legacy" genes no longer predicted by these programs; and the practical difficulties of updating due to encountering deprecated, changed, or retired gene identifiers.  In addn., we consider what other criteria may support claims of orthol. and alternative approaches to find potential orthologs that elude identification by these programs.  Finally, we created a new web-based tool that allows for rapid searches of OL2 by gene identifiers, protein domains [InterPro and SMART (Simple Modular Architecture Research Tool], or human disease assocns. ([OMIM (Online Mendelian Inheritence in Man], and also includes available RNA-interference resources to facilitate potential translational cross-species studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt-i-dD0MQ0bVg90H21EOLACvtfcHk0li5XVCrYCYcKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjsVyit7c%253D&md5=17f047bb4d7d01c737c692041715d261</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1534%2Fgenetics.118.301307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1534%252Fgenetics.118.301307%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DW.%26aulast%3DUnderwood%26aufirst%3DR.%2BS.%26aulast%3DGreenwald%26aufirst%3DI.%26aulast%3DShaye%26aufirst%3DD.%2BD.%26atitle%3DOrtholist%25202%253A%2520A%2520New%2520Comparative%2520Genomic%2520Analysis%2520of%2520Human%2520and%2520Caenorhabditis%2520Elegans%2520Genes%26jtitle%3DGenetics%26date%3D2018%26volume%3D210%26spage%3D445%26epage%3D461%26doi%3D10.1534%2Fgenetics.118.301307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hales, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korey, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larracuente, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, D. M.</span></span> <span> </span><span class="NLM_article-title">Genetics on the Fly: A Primer on the Drosophila Model System</span>. <i>Genetics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>201</i></span>,  <span class="NLM_fpage">815</span>– <span class="NLM_lpage">842</span>, <span class="refDoi"> DOI: 10.1534/genetics.115.183392</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1534%2Fgenetics.115.183392" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26564900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvFGqt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=201&publication_year=2015&pages=815-842&author=K.+G.+Halesauthor=C.+A.+Koreyauthor=A.+M.+Larracuenteauthor=D.+M.+Roberts&title=Genetics+on+the+Fly%3A+A+Primer+on+the+Drosophila+Model+System&doi=10.1534%2Fgenetics.115.183392"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Genetics on the fly: a primer on the Drosophila model system</span></div><div class="casAuthors">Hales, Karen G.; Korey, Christopher A.; Larracuente, Amanda M.; Roberts, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Genetics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">201</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">815-842</span>CODEN:
                <span class="NLM_cas:coden">GENTAE</span>;
        ISSN:<span class="NLM_cas:issn">0016-6731</span>.
    
            (<span class="NLM_cas:orgname">Genetics Society of America</span>)
        </div><div class="casAbstract">Fruit flies of the genus Drosophila have been an attractive and effective genetic model organism since Thomas Hunt Morgan and colleagues made seminal discoveries with them a century ago.  Work with Drosophila has enabled dramatic advances in cell and developmental biol., neurobiol. and behavior, mol. biol., evolutionary and population genetics, and other fields.  With more tissue types and observable behaviors than in other short-generation model organisms, and with vast genome data available for many species within the genus, the fly's tractable complexity will continue to enable exciting opportunities to explore mechanisms of complex developmental programs, behaviors, and broader evolutionary questions.  This primer describes the organism's natural history, the features of sequenced genomes within the genus, the wide range of available genetic tools and online resources, the types of biol. questions Drosophila can help address, and historical milestones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmpAqURQTda7Vg90H21EOLACvtfcHk0li5XVCrYCYcKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvFGqt7s%253D&md5=30f5b0caf36e6649d7338882143777ab</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1534%2Fgenetics.115.183392&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1534%252Fgenetics.115.183392%26sid%3Dliteratum%253Aachs%26aulast%3DHales%26aufirst%3DK.%2BG.%26aulast%3DKorey%26aufirst%3DC.%2BA.%26aulast%3DLarracuente%26aufirst%3DA.%2BM.%26aulast%3DRoberts%26aufirst%3DD.%2BM.%26atitle%3DGenetics%2520on%2520the%2520Fly%253A%2520A%2520Primer%2520on%2520the%2520Drosophila%2520Model%2520System%26jtitle%3DGenetics%26date%3D2015%26volume%3D201%26spage%3D815%26epage%3D842%26doi%3D10.1534%2Fgenetics.115.183392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shih, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodge, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrade-Navarro, M. A.</span></span> <span> </span><span class="NLM_article-title">Comparison of Inter- and Intraspecies Variation in Humans and Fruit Flies</span>. <i>Genomics Data</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">54</span>, <span class="refDoi"> DOI: 10.1016/j.gdata.2014.11.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.gdata.2014.11.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26484147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC28zjslSqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2015&pages=49-54&author=J.+Shihauthor=R.+Hodgeauthor=M.+A.+Andrade-Navarro&title=Comparison+of+Inter-+and+Intraspecies+Variation+in+Humans+and+Fruit+Flies&doi=10.1016%2Fj.gdata.2014.11.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of inter- and intraspecies variation in humans and fruit flies</span></div><div class="casAuthors">Shih Juliann; Hodge Russ; Andrade-Navarro Miguel A</div><div class="citationInfo"><span class="NLM_cas:title">Genomics data</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">49-54</span>
        ISSN:<span class="NLM_cas:issn">2213-5960</span>.
    </div><div class="casAbstract">Variation is essential to species survival and adaptation during evolution.  This variation is conferred by the imperfection of biochemical processes, such as mutations and alterations in DNA sequences, and can also be seen within genomes through processes such as the generation of antibodies.  Recent sequencing projects have produced multiple versions of the genomes of humans and fruit flies (Drosophila melanogaster).  These give us a chance to study how individual gene sequences vary within and between species.  Here we arranged human and fly genes in orthologous pairs and compared such within-species variability with their degree of conservation between flies and humans.  We observed that a significant number of proteins associated with mRNA translation are highly conserved between species and yet are highly variable within each species.  The fact that we observe this in two species whose lineages separated more than 700 million years ago suggests that this is the result of a very ancient process.  We hypothesize that this effect might be attributed to a positive selection for variability of virus-interacting proteins that confers a general resistance to viral hijacking of the mRNA translation machinery within populations.  Our analysis points to this and to other processes resulting in positive selection for gene variation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQENEMGYK1lr7YtC3U5jeOifW6udTcc2eYhDm8fPrwclbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28zjslSqsg%253D%253D&md5=da95eeefb36ab976b285c1748967044d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.gdata.2014.11.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gdata.2014.11.010%26sid%3Dliteratum%253Aachs%26aulast%3DShih%26aufirst%3DJ.%26aulast%3DHodge%26aufirst%3DR.%26aulast%3DAndrade-Navarro%26aufirst%3DM.%2BA.%26atitle%3DComparison%2520of%2520Inter-%2520and%2520Intraspecies%2520Variation%2520in%2520Humans%2520and%2520Fruit%2520Flies%26jtitle%3DGenomics%2520Data%26date%3D2015%26volume%3D3%26spage%3D49%26epage%3D54%26doi%3D10.1016%2Fj.gdata.2014.11.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Howe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torroja, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrance, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berthelot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muffato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphray, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaren, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaren, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sealy, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caccamo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauch, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chow, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quintais, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerra-Assuncao, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eyre, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redmond, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laird, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenyon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donaldson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehra, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almeida-King, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loveland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trevanion, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quail, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLay, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plumb, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riddle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Threadgold, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harden, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ware, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Begum, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortimore, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerry, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heath, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillimore, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tracey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corby, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glithero, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howden, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panagiotidis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beasley, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auger, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raisen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambridge, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leongamornlert, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGuire, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilderthorp, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manthravadi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nichol, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitehead, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kay, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murnane, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphries, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sycamore, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunders, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babbage, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammond, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashreghi-Mohammadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barr, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wray, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellington, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellwood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodmansey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tromans, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grafham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skuce, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandian, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimberley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fosker, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehring, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dooley, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ersan-Urun, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eser, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geiger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geisler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karotki, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konantz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konantz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oberlander, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph-Geiger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teucke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raddatz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osoegawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rapp, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widaa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langford, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuster, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Searle, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enright, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geisler, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plasterk, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westerfield, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jong, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zon, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Postlethwait, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nusslein-Volhard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crollius, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stemple, D. L.</span></span> <span> </span><span class="NLM_article-title">The Zebrafish Reference Genome Sequence and Its Relationship to the Human Genome</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>496</i></span>,  <span class="NLM_fpage">498</span>– <span class="NLM_lpage">503</span>, <span class="refDoi"> DOI: 10.1038/nature12111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnature12111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23594743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvF2isb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=496&publication_year=2013&pages=498-503&author=K.+Howeauthor=M.+D.+Clarkauthor=C.+F.+Torrojaauthor=J.+Torranceauthor=C.+Berthelotauthor=M.+Muffatoauthor=J.+E.+Collinsauthor=S.+Humphrayauthor=K.+McLarenauthor=L.+Matthewsauthor=S.+McLarenauthor=I.+Sealyauthor=M.+Caccamoauthor=C.+Churcherauthor=C.+Scottauthor=J.+C.+Barrettauthor=R.+Kochauthor=G.+J.+Rauchauthor=S.+Whiteauthor=W.+Chowauthor=B.+Kilianauthor=L.+T.+Quintaisauthor=J.+A.+Guerra-Assuncaoauthor=Y.+Zhouauthor=Y.+Guauthor=J.+Yenauthor=J.+H.+Vogelauthor=T.+Eyreauthor=S.+Redmondauthor=R.+Banerjeeauthor=J.+Chiauthor=B.+Fuauthor=E.+Langleyauthor=S.+F.+Maguireauthor=G.+K.+Lairdauthor=D.+Lloydauthor=E.+Kenyonauthor=S.+Donaldsonauthor=H.+Sehraauthor=J.+Almeida-Kingauthor=J.+Lovelandauthor=S.+Trevanionauthor=M.+Jonesauthor=M.+Quailauthor=D.+Willeyauthor=A.+Huntauthor=J.+Burtonauthor=S.+Simsauthor=K.+McLayauthor=B.+Plumbauthor=J.+Davisauthor=C.+Cleeauthor=K.+Oliverauthor=R.+Clarkauthor=C.+Riddleauthor=D.+Elliottauthor=G.+Threadgoldauthor=G.+Hardenauthor=D.+Wareauthor=S.+Begumauthor=B.+Mortimoreauthor=G.+Kerryauthor=P.+Heathauthor=B.+Phillimoreauthor=A.+Traceyauthor=N.+Corbyauthor=M.+Dunnauthor=C.+Johnsonauthor=J.+Woodauthor=S.+Clarkauthor=S.+Pelanauthor=G.+Griffithsauthor=M.+Smithauthor=R.+Glitheroauthor=P.+Howdenauthor=N.+Barkerauthor=C.+Lloydauthor=C.+Stevensauthor=J.+Harleyauthor=K.+Holtauthor=G.+Panagiotidisauthor=J.+Lovellauthor=H.+Beasleyauthor=C.+Hendersonauthor=D.+Gordonauthor=K.+Augerauthor=D.+Wrightauthor=J.+Collinsauthor=C.+Raisenauthor=L.+Dyerauthor=K.+Leungauthor=L.+Robertsonauthor=K.+Ambridgeauthor=D.+Leongamornlertauthor=S.+McGuireauthor=R.+Gilderthorpauthor=C.+Griffithsauthor=D.+Manthravadiauthor=S.+Nicholauthor=G.+Barkerauthor=S.+Whiteheadauthor=M.+Kayauthor=J.+Brownauthor=C.+Murnaneauthor=E.+Grayauthor=M.+Humphriesauthor=N.+Sycamoreauthor=D.+Barkerauthor=D.+Saundersauthor=J.+Wallisauthor=A.+Babbageauthor=S.+Hammondauthor=M.+Mashreghi-Mohammadiauthor=L.+Barrauthor=S.+Martinauthor=P.+Wrayauthor=A.+Ellingtonauthor=N.+Matthewsauthor=M.+Ellwoodauthor=R.+Woodmanseyauthor=G.+Clarkauthor=J.+Cooperauthor=A.+Tromansauthor=D.+Grafhamauthor=C.+Skuceauthor=R.+Pandianauthor=R.+Andrewsauthor=E.+Harrisonauthor=A.+Kimberleyauthor=J.+Garnettauthor=N.+Foskerauthor=R.+Hallauthor=P.+Garnerauthor=D.+Kellyauthor=C.+Birdauthor=S.+Palmerauthor=I.+Gehringauthor=A.+Bergerauthor=C.+M.+Dooleyauthor=Z.+Ersan-Urunauthor=C.+Eserauthor=H.+Geigerauthor=M.+Geislerauthor=L.+Karotkiauthor=A.+Kirnauthor=J.+Konantzauthor=M.+Konantzauthor=M.+Oberlanderauthor=S.+Rudolph-Geigerauthor=M.+Teuckeauthor=C.+Lanzauthor=G.+Raddatzauthor=K.+Osoegawaauthor=B.+Zhuauthor=A.+Rappauthor=S.+Widaaauthor=C.+Langfordauthor=F.+Yangauthor=S.+C.+Schusterauthor=N.+P.+Carterauthor=J.+Harrowauthor=Z.+Ningauthor=J.+Herreroauthor=S.+M.+Searleauthor=A.+Enrightauthor=R.+Geislerauthor=R.+H.+Plasterkauthor=C.+Leeauthor=M.+Westerfieldauthor=P.+J.+de+Jongauthor=L.+I.+Zonauthor=J.+H.+Postlethwaitauthor=C.+Nusslein-Volhardauthor=T.+J.+Hubbardauthor=H.+R.+Crolliusauthor=J.+Rogersauthor=D.+L.+Stemple&title=The+Zebrafish+Reference+Genome+Sequence+and+Its+Relationship+to+the+Human+Genome&doi=10.1038%2Fnature12111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The zebrafish reference genome sequence and its relationship to the human genome</span></div><div class="casAuthors">Howe, Kerstin; Clark, Matthew D.; Torroja, Carlos F.; Torrance, James; Berthelot, Camille; Muffato, Matthieu; Collins, John E.; Humphray, Sean; McLaren, Karen; Matthews, Lucy; McLaren, Stuart; Sealy, Ian; Caccamo, Mario; Churcher, Carol; Scott, Carol; Barrett, Jeffrey C.; Koch, Romke; Rauch, Gerd-Joerg; White, Simon; Chow, William; Kilian, Britt; Quintais, Leonor T.; Guerra-Assuncao, Jose A.; Zhou, Yi; Gu, Yong; Yen, Jennifer; Vogel, Jan-Hinnerk; Eyre, Tina; Redmond, Seth; Banerjee, Ruby; Chi, Jianxiang; Fu, Beiyuan; Langley, Elizabeth; Maguire, Sean F.; Laird, Gavin K.; Lloyd, David; Kenyon, Emma; Donaldson, Sarah; Sehra, Harminder; Almeida-King, Jeff; Loveland, Jane; Trevanion, Stephen; Jones, Matt; Quail, Mike; Willey, Dave; Hunt, Adrienne; Burton, John; Sims, Sarah; McLay, Kirsten; Plumb, Bob; Davis, Joy; Clee, Chris; Oliver, Karen; Clark, Richard; Riddle, Clare; Eliott, David; Threadgold, Glen; Harden, Glenn; Ware, Darren; Begum, Sharmin; Mortimore, Beverley; Kerry, Giselle; Heath, Paul; Phillimore, Benjamin; Tracey, Alan; Corby, Nicole; Dunn, Matthew; Johnson, Christopher; Wood, Jonathan; Clark, Susan; Pelan, Sarah; Griffiths, Guy; Smith, Michelle; Glithero, Rebecca; Howden, Philip; Barker, Nicholas; Lloyd, Christine; Stevens, Christopher; Harley, Joanna; Holt, Karen; Panagiotidis, Georgios; Lovell, Jamieson; Beasley, Helen; Henderson, Carl; Gordon, Daria; Auger, Katherine; Wright, Deborah; Collins, Joanna; Raisen, Claire; Dyer, Lauren; Leung, Kenric; Robertson, Lauren; Ambridge, Kirsty; Leongamornlert, Daniel; McGuire, Sarah; Gilderthorp, Ruth; Griffiths, Coline; Manthravadi, Deepa; Nichol, Sarah; Barker, Gary; Whitehead, Siobhan; Kay, Michael; Brown, Jacqueline; Murnane, Clare; Gray, Emma; Humphries, Matthew; Sycamore, Neil; Barker, Darren; Saunders, David; Wallis, Justene; Babbage, Anne; Hammond, Sian; Mashreghi-Mohammadi, Maryam; Barr, Lucy; Martin, Sancha; Wray, Paul; Ellington, Andrew; Matthews, Nicholas; Ellwood, Matthew; Woodmansey, Rebecca; Clark, Graham; Cooper, James D.; Tromans, Anthony; Grafham, Darren; Skuce, Carl; Pandian, Richard; Andrews, Robert; Harrison, Elliot; Kimberley, Andrew; Garnett, Jane; Fosker, Nigel; Hall, Rebekah; Garner, Patrick; Kelly, Daniel; Bird, Christine; Palmer, Sophie; Gehring, Ines; Berger, Andrea; Dooley, Christopher M.; Ersan-Ueruen, Zuebeyde; Eser, Cigdem; Geiger, Horst; Geisler, Maria; Karotki, Lena; Kirn, Anette; Konantz, Judith; Konantz, Martina; Oberlaender, Martina; Rudolph-Geiger, Silke; Teucke, Mathias; Lanz, Christa; Raddatz, Guenter; Osoegawa, Kazutoyo; Zhu, Baoli; Rapp, Amanda; Widaa, Sara; Langford, Cordelia; Yang, Fengtang; Schuster, Stephan C.; Carter, Nigel P.; Harrow, Jennifer; Ning, Zemin; Herrero, Javier; Searle, Steve M. J.; Enright, Anton; Geisler, Robert; Plasterk, Ronald H. A.; Lee, Charles; Westerfield, Monte; de Jong, Pieter J.; Zon, Leonard I.; Postlethwait, John H.; Nuesslein-Volhard, Christiane; Hubbard, Tim J. P.; Crollius, Hugues Roest; Rogers, Jane; Stemple, Derek L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">496</span>
        (<span class="NLM_cas:issue">7446</span>),
    <span class="NLM_cas:pages">498-503</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Zebrafish have become a popular organism for the study of vertebrate gene function.  The virtually transparent embryos of this species, and the ability to accelerate genetic studies by gene knockdown or overexpression, have led to the widespread use of zebrafish in the detailed investigation of vertebrate gene function and increasingly, the study of human genetic disease.  However, for effective modeling of human genetic disease it is important to understand the extent to which zebrafish genes and gene structures are related to orthologous human genes.  To examine this, the authors generated a high-quality sequence assembly of the zebrafish genome, made up of an overlapping set of completely sequenced large-insert clones that were ordered and oriented using a high-resoln. high-d. meiotic map.  Detailed automatic and manual annotation provides evidence of >26,000 protein-coding genes, the largest gene set of any vertebrate so far sequenced.  Comparison to the human ref. genome shows that ∼70% of human genes have at least one obvious zebrafish ortholog.  In addn., the high quality of this genome assembly provides a clearer understanding of key genomic features such as a unique repeat content, a scarcity of pseudogenes, an enrichment of zebrafish-specific genes on chromosome 4 and chromosomal regions that influence sex detn.  The draft, annotated, genome sequence is deposited in GenBank/EMBL/DDBJ with WGS project accession nos. CABZ01000001-CABZ01119119, CU651604-CU651628, and FR648399-FR652957.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY5Bb_pwj9k7Vg90H21EOLACvtfcHk0ljR6cQHCbnTiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvF2isb0%253D&md5=9c7c77d0e97a41c5f64802b892b2701d</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnature12111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12111%26sid%3Dliteratum%253Aachs%26aulast%3DHowe%26aufirst%3DK.%26aulast%3DClark%26aufirst%3DM.%2BD.%26aulast%3DTorroja%26aufirst%3DC.%2BF.%26aulast%3DTorrance%26aufirst%3DJ.%26aulast%3DBerthelot%26aufirst%3DC.%26aulast%3DMuffato%26aufirst%3DM.%26aulast%3DCollins%26aufirst%3DJ.%2BE.%26aulast%3DHumphray%26aufirst%3DS.%26aulast%3DMcLaren%26aufirst%3DK.%26aulast%3DMatthews%26aufirst%3DL.%26aulast%3DMcLaren%26aufirst%3DS.%26aulast%3DSealy%26aufirst%3DI.%26aulast%3DCaccamo%26aufirst%3DM.%26aulast%3DChurcher%26aufirst%3DC.%26aulast%3DScott%26aufirst%3DC.%26aulast%3DBarrett%26aufirst%3DJ.%2BC.%26aulast%3DKoch%26aufirst%3DR.%26aulast%3DRauch%26aufirst%3DG.%2BJ.%26aulast%3DWhite%26aufirst%3DS.%26aulast%3DChow%26aufirst%3DW.%26aulast%3DKilian%26aufirst%3DB.%26aulast%3DQuintais%26aufirst%3DL.%2BT.%26aulast%3DGuerra-Assuncao%26aufirst%3DJ.%2BA.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DGu%26aufirst%3DY.%26aulast%3DYen%26aufirst%3DJ.%26aulast%3DVogel%26aufirst%3DJ.%2BH.%26aulast%3DEyre%26aufirst%3DT.%26aulast%3DRedmond%26aufirst%3DS.%26aulast%3DBanerjee%26aufirst%3DR.%26aulast%3DChi%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DB.%26aulast%3DLangley%26aufirst%3DE.%26aulast%3DMaguire%26aufirst%3DS.%2BF.%26aulast%3DLaird%26aufirst%3DG.%2BK.%26aulast%3DLloyd%26aufirst%3DD.%26aulast%3DKenyon%26aufirst%3DE.%26aulast%3DDonaldson%26aufirst%3DS.%26aulast%3DSehra%26aufirst%3DH.%26aulast%3DAlmeida-King%26aufirst%3DJ.%26aulast%3DLoveland%26aufirst%3DJ.%26aulast%3DTrevanion%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DM.%26aulast%3DQuail%26aufirst%3DM.%26aulast%3DWilley%26aufirst%3DD.%26aulast%3DHunt%26aufirst%3DA.%26aulast%3DBurton%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DS.%26aulast%3DMcLay%26aufirst%3DK.%26aulast%3DPlumb%26aufirst%3DB.%26aulast%3DDavis%26aufirst%3DJ.%26aulast%3DClee%26aufirst%3DC.%26aulast%3DOliver%26aufirst%3DK.%26aulast%3DClark%26aufirst%3DR.%26aulast%3DRiddle%26aufirst%3DC.%26aulast%3DElliott%26aufirst%3DD.%26aulast%3DThreadgold%26aufirst%3DG.%26aulast%3DHarden%26aufirst%3DG.%26aulast%3DWare%26aufirst%3DD.%26aulast%3DBegum%26aufirst%3DS.%26aulast%3DMortimore%26aufirst%3DB.%26aulast%3DKerry%26aufirst%3DG.%26aulast%3DHeath%26aufirst%3DP.%26aulast%3DPhillimore%26aufirst%3DB.%26aulast%3DTracey%26aufirst%3DA.%26aulast%3DCorby%26aufirst%3DN.%26aulast%3DDunn%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DC.%26aulast%3DWood%26aufirst%3DJ.%26aulast%3DClark%26aufirst%3DS.%26aulast%3DPelan%26aufirst%3DS.%26aulast%3DGriffiths%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DM.%26aulast%3DGlithero%26aufirst%3DR.%26aulast%3DHowden%26aufirst%3DP.%26aulast%3DBarker%26aufirst%3DN.%26aulast%3DLloyd%26aufirst%3DC.%26aulast%3DStevens%26aufirst%3DC.%26aulast%3DHarley%26aufirst%3DJ.%26aulast%3DHolt%26aufirst%3DK.%26aulast%3DPanagiotidis%26aufirst%3DG.%26aulast%3DLovell%26aufirst%3DJ.%26aulast%3DBeasley%26aufirst%3DH.%26aulast%3DHenderson%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DAuger%26aufirst%3DK.%26aulast%3DWright%26aufirst%3DD.%26aulast%3DCollins%26aufirst%3DJ.%26aulast%3DRaisen%26aufirst%3DC.%26aulast%3DDyer%26aufirst%3DL.%26aulast%3DLeung%26aufirst%3DK.%26aulast%3DRobertson%26aufirst%3DL.%26aulast%3DAmbridge%26aufirst%3DK.%26aulast%3DLeongamornlert%26aufirst%3DD.%26aulast%3DMcGuire%26aufirst%3DS.%26aulast%3DGilderthorp%26aufirst%3DR.%26aulast%3DGriffiths%26aufirst%3DC.%26aulast%3DManthravadi%26aufirst%3DD.%26aulast%3DNichol%26aufirst%3DS.%26aulast%3DBarker%26aufirst%3DG.%26aulast%3DWhitehead%26aufirst%3DS.%26aulast%3DKay%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DMurnane%26aufirst%3DC.%26aulast%3DGray%26aufirst%3DE.%26aulast%3DHumphries%26aufirst%3DM.%26aulast%3DSycamore%26aufirst%3DN.%26aulast%3DBarker%26aufirst%3DD.%26aulast%3DSaunders%26aufirst%3DD.%26aulast%3DWallis%26aufirst%3DJ.%26aulast%3DBabbage%26aufirst%3DA.%26aulast%3DHammond%26aufirst%3DS.%26aulast%3DMashreghi-Mohammadi%26aufirst%3DM.%26aulast%3DBarr%26aufirst%3DL.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DWray%26aufirst%3DP.%26aulast%3DEllington%26aufirst%3DA.%26aulast%3DMatthews%26aufirst%3DN.%26aulast%3DEllwood%26aufirst%3DM.%26aulast%3DWoodmansey%26aufirst%3DR.%26aulast%3DClark%26aufirst%3DG.%26aulast%3DCooper%26aufirst%3DJ.%26aulast%3DTromans%26aufirst%3DA.%26aulast%3DGrafham%26aufirst%3DD.%26aulast%3DSkuce%26aufirst%3DC.%26aulast%3DPandian%26aufirst%3DR.%26aulast%3DAndrews%26aufirst%3DR.%26aulast%3DHarrison%26aufirst%3DE.%26aulast%3DKimberley%26aufirst%3DA.%26aulast%3DGarnett%26aufirst%3DJ.%26aulast%3DFosker%26aufirst%3DN.%26aulast%3DHall%26aufirst%3DR.%26aulast%3DGarner%26aufirst%3DP.%26aulast%3DKelly%26aufirst%3DD.%26aulast%3DBird%26aufirst%3DC.%26aulast%3DPalmer%26aufirst%3DS.%26aulast%3DGehring%26aufirst%3DI.%26aulast%3DBerger%26aufirst%3DA.%26aulast%3DDooley%26aufirst%3DC.%2BM.%26aulast%3DErsan-Urun%26aufirst%3DZ.%26aulast%3DEser%26aufirst%3DC.%26aulast%3DGeiger%26aufirst%3DH.%26aulast%3DGeisler%26aufirst%3DM.%26aulast%3DKarotki%26aufirst%3DL.%26aulast%3DKirn%26aufirst%3DA.%26aulast%3DKonantz%26aufirst%3DJ.%26aulast%3DKonantz%26aufirst%3DM.%26aulast%3DOberlander%26aufirst%3DM.%26aulast%3DRudolph-Geiger%26aufirst%3DS.%26aulast%3DTeucke%26aufirst%3DM.%26aulast%3DLanz%26aufirst%3DC.%26aulast%3DRaddatz%26aufirst%3DG.%26aulast%3DOsoegawa%26aufirst%3DK.%26aulast%3DZhu%26aufirst%3DB.%26aulast%3DRapp%26aufirst%3DA.%26aulast%3DWidaa%26aufirst%3DS.%26aulast%3DLangford%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DF.%26aulast%3DSchuster%26aufirst%3DS.%2BC.%26aulast%3DCarter%26aufirst%3DN.%2BP.%26aulast%3DHarrow%26aufirst%3DJ.%26aulast%3DNing%26aufirst%3DZ.%26aulast%3DHerrero%26aufirst%3DJ.%26aulast%3DSearle%26aufirst%3DS.%2BM.%26aulast%3DEnright%26aufirst%3DA.%26aulast%3DGeisler%26aufirst%3DR.%26aulast%3DPlasterk%26aufirst%3DR.%2BH.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DWesterfield%26aufirst%3DM.%26aulast%3Dde%2BJong%26aufirst%3DP.%2BJ.%26aulast%3DZon%26aufirst%3DL.%2BI.%26aulast%3DPostlethwait%26aufirst%3DJ.%2BH.%26aulast%3DNusslein-Volhard%26aufirst%3DC.%26aulast%3DHubbard%26aufirst%3DT.%2BJ.%26aulast%3DCrollius%26aufirst%3DH.%2BR.%26aulast%3DRogers%26aufirst%3DJ.%26aulast%3DStemple%26aufirst%3DD.%2BL.%26atitle%3DThe%2520Zebrafish%2520Reference%2520Genome%2520Sequence%2520and%2520Its%2520Relationship%2520to%2520the%2520Human%2520Genome%26jtitle%3DNature%26date%3D2013%26volume%3D496%26spage%3D498%26epage%3D503%26doi%3D10.1038%2Fnature12111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guenet, J. L.</span></span> <span> </span><span class="NLM_article-title">The Mouse Genome</span>. <i>Genome Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1729</span>– <span class="NLM_lpage">1740</span>, <span class="refDoi"> DOI: 10.1101/gr.3728305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1101%2Fgr.3728305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=16339371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvFA%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=1729-1740&author=J.+L.+Guenet&title=The+Mouse+Genome&doi=10.1101%2Fgr.3728305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The mouse genome</span></div><div class="casAuthors">Guenet, Jean Louis</div><div class="citationInfo"><span class="NLM_cas:title">Genome Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1729-1740</span>CODEN:
                <span class="NLM_cas:coden">GEREFS</span>;
        ISSN:<span class="NLM_cas:issn">1088-9051</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  The house mouse has been used as a privileged model organism since the early days of genetics, and the numerous expts. made with this small mammal have regularly contributed to enrich our knowledge of mammalian biol. and pathol., ranging from embryonic development to metabolic disease, histocompatibility, immunol., behavior, and cancer.  Over the past two decades, a no. of large-scale integrated and concerted projects have been undertaken that will probably open a new era in the genetics of the species.  The sequencing of the genome, which will allow researchers to make comparisons with other mammals and identify regions conserved by evolution, is probably the most important project, but many other initiatives, such as the massive prodn. of point or chromosomal mutations assocd. with comprehensive and standardized phenotyping of the mutant phenotypes, will help annotation of the ∼25,000 genes packed in the mouse genome.  In the same way, and as another consequence of the sequencing, the discovery of many single nucleotide polymorphisms and the development of new tools and resources, like the Collaborative Cross, will contribute to the development of modern quant. genetics.  It is clear that mouse genetics has changed dramatically over the last 10-15 years and its future looks promising.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUyX4TuG7CNLVg90H21EOLACvtfcHk0ljR6cQHCbnTiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvFA%253D&md5=12eb5fb4a2caa43bb3e08b750e4bf1f9</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1101%2Fgr.3728305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgr.3728305%26sid%3Dliteratum%253Aachs%26aulast%3DGuenet%26aufirst%3DJ.%2BL.%26atitle%3DThe%2520Mouse%2520Genome%26jtitle%3DGenome%2520Res.%26date%3D2005%26volume%3D15%26spage%3D1729%26epage%3D1740%26doi%3D10.1101%2Fgr.3728305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saleem, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannan, R. R.</span></span> <span> </span><span class="NLM_article-title">Zebrafish: An Emerging Real-Time Model System to Study Alzheimer’s Disease and Neurospecific Drug Discovery</span>. <i>Cell Death Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">45</span>, <span class="refDoi"> DOI: 10.1038/s41420-018-0109-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41420-018-0109-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30302279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BB3cznsVGqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=45&author=S.+Saleemauthor=R.+R.+Kannan&title=Zebrafish%3A+An+Emerging+Real-Time+Model+System+to+Study+Alzheimer%E2%80%99s+Disease+and+Neurospecific+Drug+Discovery&doi=10.1038%2Fs41420-018-0109-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Zebrafish: an emerging real-time model system to study Alzheimer's disease and neurospecific drug discovery</span></div><div class="casAuthors">Saleem Suraiya; Kannan Rajaretinam Rajesh</div><div class="citationInfo"><span class="NLM_cas:title">Cell death discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45</span>
        ISSN:<span class="NLM_cas:issn">2058-7716</span>.
    </div><div class="casAbstract">Zebrafish (Danio rerio) is emerging as an increasingly successful model for translational research on human neurological disorders.  In this review, we appraise the high degree of neurological and behavioural resemblance of zebrafish with humans.  It is highly validated as a powerful vertebrate model for investigating human neurodegenerative diseases.  The neuroanatomic and neurochemical pathways of zebrafish brain exhibit a profound resemblance with the human brain.  Physiological, emotional and social behavioural pattern similarities between them have also been well established.  Interestingly, zebrafish models have been used successfully to simulate the pathology of Alzheimer's disease (AD) as well as Tauopathy.  Their relatively simple nervous system and the optical transparency of the embryos permit real-time neurological imaging.  Here, we further elaborate on the use of recent real-time imaging techniques to obtain vital insights into the neurodegeneration that occurs in AD.  Zebrafish is adeptly suitable for Ca(2+) imaging, which provides a better understanding of neuronal activity and axonal dystrophy in a non-invasive manner.  Three-dimensional imaging in zebrafish is a rapidly evolving technique, which allows the visualisation of the whole organism for an elaborate in vivo functional and neurophysiological analysis in disease condition.  Suitability to high-throughput screening and similarity with humans makes zebrafish an excellent model for screening neurospecific compounds.  Thus, the zebrafish model can be pivotal in bridging the gap from the bench to the bedside.  This fish is becoming an increasingly successful model to understand AD with further scope for investigation in neurodevelopment and neurodegeneration, which promises exciting research opportunities in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhy659z6uLnbmQt-ICR3qNfW6udTcc2eawhh7vGlBrtbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cznsVGqsQ%253D%253D&md5=a789b9a83b2db7e36fd21581b5f4de1c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fs41420-018-0109-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41420-018-0109-7%26sid%3Dliteratum%253Aachs%26aulast%3DSaleem%26aufirst%3DS.%26aulast%3DKannan%26aufirst%3DR.%2BR.%26atitle%3DZebrafish%253A%2520An%2520Emerging%2520Real-Time%2520Model%2520System%2520to%2520Study%2520Alzheimer%25E2%2580%2599s%2520Disease%2520and%2520Neurospecific%2520Drug%2520Discovery%26jtitle%3DCell%2520Death%2520Discovery%26date%3D2018%26volume%3D4%26spage%3D45%26doi%3D10.1038%2Fs41420-018-0109-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Corman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggblad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brautigam, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafarga, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lidemalm, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huhn, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carreras-Puigvert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Capetillo, O.</span></span> <span> </span><span class="NLM_article-title">A Chemical Screen Identifies Compounds Limiting the Toxicity of C9orf72 Dipeptide Repeats</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">243</span>, e235 <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.10.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.chembiol.2018.10.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30527999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVChs77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=235-243&author=A.+Cormanauthor=B.+Jungauthor=M.+Haggbladauthor=L.+Brautigamauthor=V.+Lafargaauthor=L.+Lidemalmauthor=D.+Huhnauthor=J.+Carreras-Puigvertauthor=O.+Fernandez-Capetillo&title=A+Chemical+Screen+Identifies+Compounds+Limiting+the+Toxicity+of+C9orf72+Dipeptide+Repeats&doi=10.1016%2Fj.chembiol.2018.10.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemical Screen Identifies Compounds Limiting the Toxicity of C9ORF72 Dipeptide Repeats</span></div><div class="casAuthors">Corman, Alba; Jung, Bomi; Haeggblad, Maria; Braeutigam, Lars; Lafarga, Vanesa; Lidemalm, Louise; Huhn, Daniela; Carreras-Puigvert, Jordi; Fernandez-Capetillo, Oscar</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">235-243.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The expansion of GGGGCC repeats within the first intron of C9ORF72 constitutes the most common cause of amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD).  Through repeat-assocd. non-ATG translation, these expansions are translated into dipeptide repeats (DPRs), some of which accumulate at nucleoli and lead to cell death.  We here performed a chem. screen to identify compds. reducing the toxicity of ALS-related poly(PR) peptides.  Our screening identified sodium phenylbutyrate, currently in clin. trials, and BET Bromodomain inhibitors as modifiers of poly(PR) toxicity in cell lines and developing zebrafish embryos.  Mechanistically, we show that BET Bromodomain inhibitors rescue the nucleolar stress induced by poly(PR) or actinomycin D, alleviating the effects of the DPR in nucleolus-related functions such as mRNA splicing or translation.  Our work suggests that BET Bromodomain inhibitors might have beneficial effects in diseases linked to nucleolar stress such as ALS/FTD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpymE6MfWVDlrVg90H21EOLACvtfcHk0lijjCuB6lpYyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVChs77K&md5=02b65f132ba1aee5ca18aa6140ec9989</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.10.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.10.020%26sid%3Dliteratum%253Aachs%26aulast%3DCorman%26aufirst%3DA.%26aulast%3DJung%26aufirst%3DB.%26aulast%3DHaggblad%26aufirst%3DM.%26aulast%3DBrautigam%26aufirst%3DL.%26aulast%3DLafarga%26aufirst%3DV.%26aulast%3DLidemalm%26aufirst%3DL.%26aulast%3DHuhn%26aufirst%3DD.%26aulast%3DCarreras-Puigvert%26aufirst%3DJ.%26aulast%3DFernandez-Capetillo%26aufirst%3DO.%26atitle%3DA%2520Chemical%2520Screen%2520Identifies%2520Compounds%2520Limiting%2520the%2520Toxicity%2520of%2520C9orf72%2520Dipeptide%2520Repeats%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D235%26epage%3D243%26doi%3D10.1016%2Fj.chembiol.2018.10.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGurk, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomes, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mojsilovic-Petrovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalb, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shorter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonini, N. M.</span></span> <span> </span><span class="NLM_article-title">Poly(Adp-Ribose) Prevents Pathological Phase Separation of Tdp-43 by Promoting Liquid Demixing and Stress Granule Localization</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">703</span>– <span class="NLM_lpage">717</span>, e709 <span class="refDoi"> DOI: 10.1016/j.molcel.2018.07.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.molcel.2018.07.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30100264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVOksr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2018&pages=703-717&author=L.+McGurkauthor=E.+Gomesauthor=L.+Guoauthor=J.+Mojsilovic-Petrovicauthor=V.+Tranauthor=R.+G.+Kalbauthor=J.+Shorterauthor=N.+M.+Bonini&title=Poly%28Adp-Ribose%29+Prevents+Pathological+Phase+Separation+of+Tdp-43+by+Promoting+Liquid+Demixing+and+Stress+Granule+Localization&doi=10.1016%2Fj.molcel.2018.07.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Poly(ADP-Ribose) Prevents Pathological Phase Separation of TDP-43 by Promoting Liquid Demixing and Stress Granule Localization</span></div><div class="casAuthors">McGurk, Leeanne; Gomes, Edward; Guo, Lin; Mojsilovic-Petrovic, Jelena; Tran, Van; Kalb, Robert G.; Shorter, James; Bonini, Nancy M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">703-717.e9</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In amyotrophic lateral sclerosis (ALS) and frontotemporal degeneration (FTD), cytoplasmic aggregates of hyperphosphorylated TDP-43 accumulate and colocalize with some stress granule components, but how pathol. TDP-43 aggregation is nucleated remains unknown.  In Drosophila, we establish that downregulation of tankyrase, a poly(ADP-ribose) (PAR) polymerase, reduces TDP-43 accumulation in the cytoplasm and potently mitigates neurodegeneration.  We establish that TDP-43 non-covalently binds to PAR via PAR-binding motifs embedded within its nuclear localization sequence.  PAR binding promotes liq.-liq. phase sepn. of TDP-43 in vitro and is required for TDP-43 accumulation in stress granules in mammalian cells and neurons.  Stress granule localization initially protects TDP-43 from disease-assocd. phosphorylation, but upon long-term stress, stress granules resolve, leaving behind aggregates of phosphorylated TDP-43.  Finally, small-mol. inhibition of Tankyrase-1/2 in mammalian cells inhibits formation of cytoplasmic TDP-43 foci without affecting stress granule assembly.  Thus, Tankyrase inhibition antagonizes TDP-43-assocd. pathol. and neurodegeneration and could have therapeutic utility for ALS and FTD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZPl6-6iH2o7Vg90H21EOLACvtfcHk0lijjCuB6lpYyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVOksr3O&md5=d3b22a855fac3079e9dcbe01e40546a7</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2018.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2018.07.002%26sid%3Dliteratum%253Aachs%26aulast%3DMcGurk%26aufirst%3DL.%26aulast%3DGomes%26aufirst%3DE.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DMojsilovic-Petrovic%26aufirst%3DJ.%26aulast%3DTran%26aufirst%3DV.%26aulast%3DKalb%26aufirst%3DR.%2BG.%26aulast%3DShorter%26aufirst%3DJ.%26aulast%3DBonini%26aufirst%3DN.%2BM.%26atitle%3DPoly%2528Adp-Ribose%2529%2520Prevents%2520Pathological%2520Phase%2520Separation%2520of%2520Tdp-43%2520by%2520Promoting%2520Liquid%2520Demixing%2520and%2520Stress%2520Granule%2520Localization%26jtitle%3DMol.%2520Cell%26date%3D2018%26volume%3D71%26spage%3D703%26epage%3D717%26doi%3D10.1016%2Fj.molcel.2018.07.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karsten, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voigt, A.</span></span> <span> </span><span class="NLM_article-title">Drosophila as a Screening Tool to Study Human Neurodegenerative Diseases</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">460</span>, <span class="refDoi"> DOI: 10.1111/jnc.12446</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1111%2Fjnc.12446" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=24028575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWqs7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2013&pages=453-460&author=S.+Lenzauthor=P.+Karstenauthor=J.+B.+Schulzauthor=A.+Voigt&title=Drosophila+as+a+Screening+Tool+to+Study+Human+Neurodegenerative+Diseases&doi=10.1111%2Fjnc.12446"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Drosophila as a screening tool to study human neurodegenerative diseases</span></div><div class="casAuthors">Lenz, Sarah; Karsten, Peter; Schulz, Joerg B.; Voigt, Aaron</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">3&4</span>),
    <span class="NLM_cas:pages">453-460</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  In an aging society, research involving neurodegenerative disorders is of paramount importance.  Over the past few years, research on Alzheimer's and Parkinson's diseases has made tremendous progress.  Exptl. studies, however, rely mostly on transgenic animal models, preferentially using mice.  Although expts. on mice have enormous advantages, they also have some inherent limitations, some of which can be overcome by the use of Drosophila melanogaster as an exptl. animal.  Among the major advantages of using the fly is its small genome, which can also be modified very easily.  The fact that its genome lends itself to diverse alterations (e. g. mutagenesis, transposons) has made the fly a useful organism to perform large-scale and genome-wide screening approaches.  This has opened up an entirely new field of exptl. research aiming to elucidate genetic interactions and screen for modifiers of disease processes in vivo.  Here, we provide a brief overview of how flies can be used to analyze mol. mechanisms underlying human neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmYVEkUe1F5rVg90H21EOLACvtfcHk0lgH__T6P4BSCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWqs7jM&md5=0f91df624a0557edce36710db73d1e52</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1111%2Fjnc.12446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjnc.12446%26sid%3Dliteratum%253Aachs%26aulast%3DLenz%26aufirst%3DS.%26aulast%3DKarsten%26aufirst%3DP.%26aulast%3DSchulz%26aufirst%3DJ.%2BB.%26aulast%3DVoigt%26aufirst%3DA.%26atitle%3DDrosophila%2520as%2520a%2520Screening%2520Tool%2520to%2520Study%2520Human%2520Neurodegenerative%2520Diseases%26jtitle%3DJ.%2520Neurochem.%26date%3D2013%26volume%3D127%26spage%3D453%26epage%3D460%26doi%3D10.1111%2Fjnc.12446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lawal, H. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Djapri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biedermann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawless, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maidment, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krantz, D. E.</span></span> <span> </span><span class="NLM_article-title">Drosophila Modifier Screens to Identify Novel Neuropsychiatric Drugs Including Aminergic Agents for the Possible Treatment of Parkinson’s Disease and Depression</span>. <i>Mol. Psychiatry</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1038/mp.2012.170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fmp.2012.170" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23229049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVClsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=235-242&author=H.+O.+Lawalauthor=A.+Terrellauthor=H.+A.+Lamauthor=C.+Djapriauthor=J.+Jangauthor=R.+Hadiauthor=L.+Robertsauthor=V.+Shahiauthor=M.+T.+Chouauthor=T.+Biedermannauthor=B.+Huangauthor=G.+M.+Lawlessauthor=N.+T.+Maidmentauthor=D.+E.+Krantz&title=Drosophila+Modifier+Screens+to+Identify+Novel+Neuropsychiatric+Drugs+Including+Aminergic+Agents+for+the+Possible+Treatment+of+Parkinson%E2%80%99s+Disease+and+Depression&doi=10.1038%2Fmp.2012.170"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Drosophila modifier screens to identify novel neuropsychiatric drugs including aminergic agents for the possible treatment of Parkinson's disease and depression</span></div><div class="casAuthors">Lawal, H. O.; Terrell, A.; Lam, H. A.; Djapri, C.; Jang, J.; Hadi, R.; Roberts, L.; Shahi, V.; Chou, M.-T.; Biedermann, T.; Huang, B.; Lawless, G. M.; Maidment, N. T.; Krantz, D. E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Psychiatry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">MOPSFQ</span>;
        ISSN:<span class="NLM_cas:issn">1359-4184</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small mols. that increase the presynaptic function of aminergic cells may provide neuroprotection in Parkinson's disease (PD) as well as treatments for attention deficit hyperactivity disorder (ADHD) and depression.  Model genetic organisms such as Drosophila melanogaster may enhance the detection of new drugs via modifier or 'enhancer/suppressor' screens, but this technique has not been applied to processes relevant to psychiatry.  To identify new aminergic drugs in vivo, we used a mutation in the Drosophila vesicular monoamine transporter (dVMAT) as a sensitized genetic background and performed a suppressor screen.  We fed dVMAT mutant larvae ∼1000 known drugs and quantitated rescue (suppression) of an amine-dependent locomotor deficit in the larva.  To det. which drugs might specifically potentiate neurotransmitter release, we performed an addnl. secondary screen for drugs that require presynaptic amine storage to rescue larval locomotion.  Using addnl. larval locomotion and adult fertility assays, we validated that at least one compd. previously used clin. as an antineoplastic agent potentiates the presynaptic function of aminergic circuits.  We suggest that structurally similar agents might be used to development treatments for PD, depression and ADHD, and that modifier screens in Drosophila provide a new strategy to screen for neuropsychiatric drugs.  More generally, our findings demonstrate the power of physiol. based screens for identifying bioactive agents for select neurotransmitter systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGop9GIOvrEeDLVg90H21EOLACvtfcHk0lgH__T6P4BSCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVClsL8%253D&md5=f70ee6ba04c7db3d98f99ac836d2cf5c</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fmp.2012.170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmp.2012.170%26sid%3Dliteratum%253Aachs%26aulast%3DLawal%26aufirst%3DH.%2BO.%26aulast%3DTerrell%26aufirst%3DA.%26aulast%3DLam%26aufirst%3DH.%2BA.%26aulast%3DDjapri%26aufirst%3DC.%26aulast%3DJang%26aufirst%3DJ.%26aulast%3DHadi%26aufirst%3DR.%26aulast%3DRoberts%26aufirst%3DL.%26aulast%3DShahi%26aufirst%3DV.%26aulast%3DChou%26aufirst%3DM.%2BT.%26aulast%3DBiedermann%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DLawless%26aufirst%3DG.%2BM.%26aulast%3DMaidment%26aufirst%3DN.%2BT.%26aulast%3DKrantz%26aufirst%3DD.%2BE.%26atitle%3DDrosophila%2520Modifier%2520Screens%2520to%2520Identify%2520Novel%2520Neuropsychiatric%2520Drugs%2520Including%2520Aminergic%2520Agents%2520for%2520the%2520Possible%2520Treatment%2520of%2520Parkinson%25E2%2580%2599s%2520Disease%2520and%2520Depression%26jtitle%3DMol.%2520Psychiatry%26date%3D2014%26volume%3D19%26spage%3D235%26epage%3D242%26doi%3D10.1038%2Fmp.2012.170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qurashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, P.</span></span> <span> </span><span class="NLM_article-title">Chemical Screen Reveals Small Molecules Suppressing Fragile X Premutation Rcgg Repeat-Mediated Neurodegeneration in Drosophila</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2068</span>– <span class="NLM_lpage">2075</span>, <span class="refDoi"> DOI: 10.1093/hmg/dds024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1093%2Fhmg%2Fdds024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=22298836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVeqtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=2068-2075&author=A.+Qurashiauthor=H.+Liuauthor=L.+Rayauthor=D.+L.+Nelsonauthor=R.+Duanauthor=P.+Jin&title=Chemical+Screen+Reveals+Small+Molecules+Suppressing+Fragile+X+Premutation+Rcgg+Repeat-Mediated+Neurodegeneration+in+Drosophila&doi=10.1093%2Fhmg%2Fdds024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical screen reveals small molecules suppressing fragile X premutation rCGG repeat-mediated neurodegeneration in Drosophila</span></div><div class="casAuthors">Qurashi, Abrar; Liu, Huijie; Ray, Laurie; Nelson, David L.; Duan, Ranhui; Jin, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2068-2075</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Fragile X-assocd. tremor/ataxia syndrome (FXTAS) is a progressive neurodegenerative disorder recognized in fragile X premutation carriers.  Using Drosophila, we previously identified elongated non-coding CGG repeats in FMR1 allele as the pathogenic cause of FXTAS.  Here, we use this same FXTAS Drosophila model to conduct a chem. screen that reveals small mols. that can ameliorate the toxic effects of fragile X premutation ribo-CGG (rCGG) repeats, among them several known phospholipase A2 (PLA2) inhibitors.  We show that specific inhibition of PLA2 activity could mitigate the neuronal deficits caused by fragile X premutation rCGG repeats, including lethality and locomotion deficits.  Furthermore, through a genetic screen, we identified a PLA2 Drosophila ortholog that specifically modulates rCGG repeat-mediated neuronal toxicity.  Our results demonstrate the utility of Drosophila models for unbiased small mol. screens and point to PLA2 as a possible therapeutic target to treat FXTAS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsCtze8pzp_bVg90H21EOLACvtfcHk0lgH__T6P4BSCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVeqtrs%253D&md5=ccd2cad64b01503be7870c6e33fd09ab</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2Fdds024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252Fdds024%26sid%3Dliteratum%253Aachs%26aulast%3DQurashi%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DRay%26aufirst%3DL.%26aulast%3DNelson%26aufirst%3DD.%2BL.%26aulast%3DDuan%26aufirst%3DR.%26aulast%3DJin%26aufirst%3DP.%26atitle%3DChemical%2520Screen%2520Reveals%2520Small%2520Molecules%2520Suppressing%2520Fragile%2520X%2520Premutation%2520Rcgg%2520Repeat-Mediated%2520Neurodegeneration%2520in%2520Drosophila%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2012%26volume%3D21%26spage%3D2068%26epage%3D2075%26doi%3D10.1093%2Fhmg%2Fdds024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pieper, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capota, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estill, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntington, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldman, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capota, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Britt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotti, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ure, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brat, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacMillan, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naidoo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melito, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Brabander, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ready, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKnight, S. L.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Proneurogenic, Neuroprotective Chemical</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">39</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2010.06.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.cell.2010.06.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=20603013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpt1Cjt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2010&pages=39-51&author=A.+A.+Pieperauthor=S.+Xieauthor=E.+Capotaauthor=S.+J.+Estillauthor=J.+Zhongauthor=J.+M.+Longauthor=G.+L.+Beckerauthor=P.+Huntingtonauthor=S.+E.+Goldmanauthor=C.+H.+Shenauthor=M.+Capotaauthor=J.+K.+Brittauthor=T.+Kottiauthor=K.+Ureauthor=D.+J.+Bratauthor=N.+S.+Williamsauthor=K.+S.+MacMillanauthor=J.+Naidooauthor=L.+Melitoauthor=J.+Hsiehauthor=J.+De+Brabanderauthor=J.+M.+Readyauthor=S.+L.+McKnight&title=Discovery+of+a+Proneurogenic%2C+Neuroprotective+Chemical&doi=10.1016%2Fj.cell.2010.06.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a proneurogenic, neuroprotective chemical</span></div><div class="casAuthors">Pieper, Andrew A.; Xie, Shanhai; Capota, Emanuela; Estill, Sandi Jo; Zhong, Jeannie; Long, Jeffrey M.; Becker, Ginger L.; Huntington, Paula; Goldman, Shauna E.; Shen, Ching-Han; Capota, Maria; Britt, Jeremiah K.; Kotti, Tiina; Ure, Kerstin; Brat, Daniel J.; Williams, Noelle S.; MacMillan, Karen S.; Naidoo, Jacinth; Melito, Lisa; Hsieh, Jenny; De Brabander, Jef; Ready, Joseph M.; McKnight, Steven L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">39-51</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">An in vivo screen was performed in search of chems. capable of enhancing neuron formation in the hippocampus of adult mice.  Eight of 1000 small mols. tested enhanced neuron formation in the subgranular zone of the dentate gyrus.  Among these was an aminopropyl carbazole, designated P7C3, endowed with favorable pharmacol. properties.  In vivo studies gave evidence that P7C3 exerts its proneurogenic activity by protecting newborn neurons from apoptosis.  Mice missing the gene encoding neuronal PAS domain protein 3 (NPAS3) are devoid of hippocampal neurogenesis and display malformation and electrophysiol. dysfunction of the dentate gyrus.  Prolonged administration of P7C3 to npas3-/- mice cor. these deficits by normalizing levels of apoptosis of newborn hippocampal neurons.  Prolonged administration of P7C3 to aged rats also enhanced neurogenesis in the dentate gyrus, impeded neuron death, and preserved cognitive capacity as a function of terminal aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDL4C8LqlFL7Vg90H21EOLACvtfcHk0lhAeZIE4GxwcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpt1Cjt7Y%253D&md5=7ed01874bd3e1b5cde587b2bc244741d</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2010.06.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2010.06.018%26sid%3Dliteratum%253Aachs%26aulast%3DPieper%26aufirst%3DA.%2BA.%26aulast%3DXie%26aufirst%3DS.%26aulast%3DCapota%26aufirst%3DE.%26aulast%3DEstill%26aufirst%3DS.%2BJ.%26aulast%3DZhong%26aufirst%3DJ.%26aulast%3DLong%26aufirst%3DJ.%2BM.%26aulast%3DBecker%26aufirst%3DG.%2BL.%26aulast%3DHuntington%26aufirst%3DP.%26aulast%3DGoldman%26aufirst%3DS.%2BE.%26aulast%3DShen%26aufirst%3DC.%2BH.%26aulast%3DCapota%26aufirst%3DM.%26aulast%3DBritt%26aufirst%3DJ.%2BK.%26aulast%3DKotti%26aufirst%3DT.%26aulast%3DUre%26aufirst%3DK.%26aulast%3DBrat%26aufirst%3DD.%2BJ.%26aulast%3DWilliams%26aufirst%3DN.%2BS.%26aulast%3DMacMillan%26aufirst%3DK.%2BS.%26aulast%3DNaidoo%26aufirst%3DJ.%26aulast%3DMelito%26aufirst%3DL.%26aulast%3DHsieh%26aufirst%3DJ.%26aulast%3DDe%2BBrabander%26aufirst%3DJ.%26aulast%3DReady%26aufirst%3DJ.%2BM.%26aulast%3DMcKnight%26aufirst%3DS.%2BL.%26atitle%3DDiscovery%2520of%2520a%2520Proneurogenic%252C%2520Neuroprotective%2520Chemical%26jtitle%3DCell%26date%3D2010%26volume%3D142%26spage%3D39%26epage%3D51%26doi%3D10.1016%2Fj.cell.2010.06.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alexandrov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanania, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leahy, E.</span></span> <span> </span><span class="NLM_article-title">Reprint Of: Highthroughtput Analysis of Behavior for Drug Discovery</span>. <i>Eur. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>753</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">134</span>, <span class="refDoi"> DOI: 10.1016/j.ejphar.2015.02.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.ejphar.2015.02.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25744878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktFKisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=753&publication_year=2015&pages=127-134&author=V.+Alexandrovauthor=D.+Brunnerauthor=T.+Hananiaauthor=E.+Leahy&title=Reprint+Of%3A+Highthroughtput+Analysis+of+Behavior+for+Drug+Discovery&doi=10.1016%2Fj.ejphar.2015.02.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Reprint of: Highthroughtput analysis of behavior for drug discovery</span></div><div class="casAuthors">Alexandrov, Vadim; Brunner, Dani; Hanania, Taleen; Leahy, Emer</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">753</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">127-134</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Drug testing with traditional behavioral assays constitutes a major bottleneck in the development of novel therapies.  PsychoGenics developed three comprehensive highthroughtput systems, SmartCube, NeuroCube and PhenoCube systems, to increase the efficiency of the drug screening and phenotyping in rodents.  These three systems capture different domains of behavior, namely, cognitive, motor, circadian, social, anxiety-like, gait and others, using custom-built computer vision software and machine learning algorithms for anal.  This review exemplifies the use of the three systems and explains how they can advance drug screening with their applications to phenotyping of disease models, drug screening, selection of lead candidates, behavior-driven lead optimization, and drug repurposing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpftTYagkadvbVg90H21EOLACvtfcHk0lhAeZIE4GxwcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktFKisrY%253D&md5=9c59a0180a0f0f3bb198ded8079ecd49</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.ejphar.2015.02.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejphar.2015.02.037%26sid%3Dliteratum%253Aachs%26aulast%3DAlexandrov%26aufirst%3DV.%26aulast%3DBrunner%26aufirst%3DD.%26aulast%3DHanania%26aufirst%3DT.%26aulast%3DLeahy%26aufirst%3DE.%26atitle%3DReprint%2520Of%253A%2520Highthroughtput%2520Analysis%2520of%2520Behavior%2520for%2520Drug%2520Discovery%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D2015%26volume%3D753%26spage%3D127%26epage%3D134%26doi%3D10.1016%2Fj.ejphar.2015.02.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span> <i>Sunovion and PsychoGenics Announce that SEP-363856 Has Received
FDA
Breakthrough Therapy Designation for the Treatment of People with
Schizophrenia</i>; <span class="NLM_publisher-name">Sunovion Pharmaceuticals,
Inc.</span>, May 10, <span class="NLM_year">2019</span>; <a href="https://news.sunovion.com/press-release/sunovion-and-psychogenics-announce-sep-363856-has-received-fda-breakthrough-therapy" class="extLink">https://news.sunovion.com/press-release/sunovion-and-psychogenics-announce-sep-363856-has-received-fda-breakthrough-therapy</a> (accessed July 2, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Sunovion+and+PsychoGenics+Announce+that+SEP-363856+Has+Received%0AFDA%0ABreakthrough+Therapy+Designation+for+the+Treatment+of+People+with%0ASchizophrenia%3B+Sunovion+Pharmaceuticals%2C%0AInc.%2C+May+10%2C+2019%3B+https%3A%2F%2Fnews.sunovion.com%2Fpress-release%2Fsunovion-and-psychogenics-announce-sep-363856-has-received-fda-breakthrough-therapy+%28accessed+July+2%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DSunovion%2520and%2520PsychoGenics%2520Announce%2520that%2520SEP-363856%2520Has%2520Received%250AFDA%250ABreakthrough%2520Therapy%2520Designation%2520for%2520the%2520Treatment%2520of%2520People%2520with%250ASchizophrenia%26pub%3DSunovion%2520Pharmaceuticals%252C%250AInc%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hollerhage, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moebius, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melms, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goebel, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakroun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koeglsperger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oertel, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bickle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoglinger, G. U.</span></span> <span> </span><span class="NLM_article-title">Protective Efficacy of Phosphodiesterase-1 Inhibition against Alpha-Synuclein Toxicity Revealed by Compound Screening in Luhmes Cells</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">11469</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-11664-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41598-017-11664-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28904388" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC1cbnvFGjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=11469&author=M.+Hollerhageauthor=C.+Moebiusauthor=J.+Melmsauthor=W.+H.+Chiuauthor=J.+N.+Goebelauthor=T.+Chakrounauthor=T.+Koeglspergerauthor=W.+H.+Oertelauthor=T.+W.+Roslerauthor=M.+Bickleauthor=G.+U.+Hoglinger&title=Protective+Efficacy+of+Phosphodiesterase-1+Inhibition+against+Alpha-Synuclein+Toxicity+Revealed+by+Compound+Screening+in+Luhmes+Cells&doi=10.1038%2Fs41598-017-11664-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Protective efficacy of phosphodiesterase-1 inhibition against alpha-synuclein toxicity revealed by compound screening in LUHMES cells</span></div><div class="casAuthors">Hollerhage Matthias; Melms Johannes; Chakroun Tasnim; Koeglsperger Thomas; Hoglinger Gunter U; Hollerhage Matthias; Melms Johannes; Rosler Thomas W; Hoglinger Gunter U; Moebius Claudia; Bickle Marc; Chiu Wei-Hua; Goebel Joachim N; Oertel Wolfgang H; Chakroun Tasnim; Rosler Thomas W; Hoglinger Gunter U; Koeglsperger Thomas; Oertel Wolfgang H</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11469</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">α-synuclein-induced neurotoxicity is a core pathogenic event in neurodegenerative synucleinopathies such as Parkinson's disease, dementia with Lewy bodies, or multiple system atrophy.  There is currently no disease-modifying therapy available for these diseases.  We screened 1,600 FDA-approved drugs for their efficacy to protect LUHMES cells from degeneration induced by wild-type α-synuclein and identified dipyridamole, a non-selective phosphodiesterase inhibitor, as top hit.  Systematic analysis of other phosphodiesterase inhibitors identified a specific phosphodiesterase 1 inhibitor as most potent to rescue from α-synuclein toxicity.  Protection was mediated by an increase of cGMP and associated with the reduction of a specific α-synuclein oligomeric species.  RNA interference experiments confirmed PDE1A and to a smaller extent PDE1C as molecular targets accounting for the protective efficacy.  PDE1 inhibition also rescued dopaminergic neurons from wild-type α-synuclein induced degeneration in the substantia nigra of mice.  In conclusion, this work identifies inhibition of PDE1A in particular as promising target for neuroprotective treatment of synucleinopathies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSVXi-Kda499iO2kE7krGOBfW6udTcc2eZbLwBeyEjcq7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbnvFGjsg%253D%253D&md5=5d8dcfa2a0e540ba23ac26c29f79f0c2</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-11664-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-11664-5%26sid%3Dliteratum%253Aachs%26aulast%3DHollerhage%26aufirst%3DM.%26aulast%3DMoebius%26aufirst%3DC.%26aulast%3DMelms%26aufirst%3DJ.%26aulast%3DChiu%26aufirst%3DW.%2BH.%26aulast%3DGoebel%26aufirst%3DJ.%2BN.%26aulast%3DChakroun%26aufirst%3DT.%26aulast%3DKoeglsperger%26aufirst%3DT.%26aulast%3DOertel%26aufirst%3DW.%2BH.%26aulast%3DRosler%26aufirst%3DT.%2BW.%26aulast%3DBickle%26aufirst%3DM.%26aulast%3DHoglinger%26aufirst%3DG.%2BU.%26atitle%3DProtective%2520Efficacy%2520of%2520Phosphodiesterase-1%2520Inhibition%2520against%2520Alpha-Synuclein%2520Toxicity%2520Revealed%2520by%2520Compound%2520Screening%2520in%2520Luhmes%2520Cells%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D11469%26doi%3D10.1038%2Fs41598-017-11664-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sidrauski, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acosta-Alvear, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khoutorsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedantham, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hearn, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamache, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ang, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okreglak, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashkenazi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nader, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renslo, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonenberg, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, P.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Brake-Release of Mrna Translation Enhances Cognitive Memory</span>. <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e00498</span> <span class="refDoi"> DOI: 10.7554/eLife.00498</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.7554%2FeLife.00498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23741617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFKht7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&author=C.+Sidrauskiauthor=D.+Acosta-Alvearauthor=A.+Khoutorskyauthor=P.+Vedanthamauthor=B.+R.+Hearnauthor=H.+Liauthor=K.+Gamacheauthor=C.+M.+Gallagherauthor=K.+K.+Angauthor=C.+Wilsonauthor=V.+Okreglakauthor=A.+Ashkenaziauthor=B.+Hannauthor=K.+Naderauthor=M.+R.+Arkinauthor=A.+R.+Rensloauthor=N.+Sonenbergauthor=P.+Walter&title=Pharmacological+Brake-Release+of+Mrna+Translation+Enhances+Cognitive+Memory&doi=10.7554%2FeLife.00498"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological brake-release of mRNA translation enhances cognitive memory</span></div><div class="casAuthors">Sidrauski, Carmela; Acosta-Alvear, Diego; Khoutorsky, Arkady; Vedantham, Punitha; Hearn, Brian R.; Li, Han; Gamache, Karine; Gallagher, Ciara M.; Ang, Kneey K-H.; Wilson, Chris; Okreglak, Voytek; Ashkenazi, Avi; Hann, Byron; Nader, Karim; Arkin, Michelle R.; Renslo, Adam R.; Sonenberg, Nahum; Walter, Peter</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e00498/1-e00498/22</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Phosphorylation of the a-subunit of initiation factor 2 (eIF2) controls protein synthesis by a conserved mechanism.  In metazoa, distinct stress conditions activate different eIF2α kinases (PERK, PKR, GCN2, and HRI) that converge on phosphorylating a unique serine in eIF2α.  This collection of signaling pathways is termed the 'integrated stress response' (ISR).  EIF2α phosphorylation diminishes protein synthesis, while allowing preferential translation of some mRNAs.  Starting with a cell-based screen for inhibitors of PERK signaling, we identified a small mol., named ISRIB, that potently (IC50 = 5 nM) reverses the effects of eIF2α phosphorylation.  ISRIB reduces the viability of cells subjected to PERK-activation by chronic endoplasmic reticulum stress.  EIF2α phosphorylation is implicated in memory consolidation.  Remarkably, ISRIB-treated mice display significant enhancement in spatial and fear-assocd. learning.  Thus, memory consolidation is inherently limited by the ISR, and ISRIB releases this brake.  As such, ISRIB promises to contribute to our understanding and treatment of cognitive disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaBg1X22tUYrVg90H21EOLACvtfcHk0li4bWcBCDnDuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntFKht7Y%253D&md5=1e74b0506a5ce689ed502ccc5f5776c7</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.7554%2FeLife.00498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.00498%26sid%3Dliteratum%253Aachs%26aulast%3DSidrauski%26aufirst%3DC.%26aulast%3DAcosta-Alvear%26aufirst%3DD.%26aulast%3DKhoutorsky%26aufirst%3DA.%26aulast%3DVedantham%26aufirst%3DP.%26aulast%3DHearn%26aufirst%3DB.%2BR.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DGamache%26aufirst%3DK.%26aulast%3DGallagher%26aufirst%3DC.%2BM.%26aulast%3DAng%26aufirst%3DK.%2BK.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DOkreglak%26aufirst%3DV.%26aulast%3DAshkenazi%26aufirst%3DA.%26aulast%3DHann%26aufirst%3DB.%26aulast%3DNader%26aufirst%3DK.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DRenslo%26aufirst%3DA.%2BR.%26aulast%3DSonenberg%26aufirst%3DN.%26aulast%3DWalter%26aufirst%3DP.%26atitle%3DPharmacological%2520Brake-Release%2520of%2520Mrna%2520Translation%2520Enhances%2520Cognitive%2520Memory%26jtitle%3DElife%26date%3D2013%26volume%3D2%26doi%3D10.7554%2FeLife.00498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Naryshkin, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weetall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karp, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woll, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabbeta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazirani, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharyya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Risher, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheedy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turpoff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trifillis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welch, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colacino, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babiak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almstead, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peltz, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eng, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mull, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynes, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontoura, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santarelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haehnke, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmucki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebeling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivaramakrishnan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paushkin, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, F.</span></span> <span> </span><span class="NLM_article-title">Motor Neuron Disease. Smn2 Splicing Modifiers Improve Motor Function and Longevity in Mice with Spinal Muscular Atrophy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>345</i></span>,  <span class="NLM_fpage">688</span>– <span class="NLM_lpage">693</span>, <span class="refDoi"> DOI: 10.1126/science.1250127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fscience.1250127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25104390" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12gtLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=345&publication_year=2014&pages=688-693&author=N.+A.+Naryshkinauthor=M.+Weetallauthor=A.+Dakkaauthor=J.+Narasimhanauthor=X.+Zhaoauthor=Z.+Fengauthor=K.+K.+Lingauthor=G.+M.+Karpauthor=H.+Qiauthor=M.+G.+Wollauthor=G.+Chenauthor=N.+Zhangauthor=V.+Gabbetaauthor=P.+Vaziraniauthor=A.+Bhattacharyyaauthor=B.+Furiaauthor=N.+Risherauthor=J.+Sheedyauthor=R.+Kongauthor=J.+Maauthor=A.+Turpoffauthor=C.+S.+Leeauthor=X.+Zhangauthor=Y.+C.+Moonauthor=P.+Trifillisauthor=E.+M.+Welchauthor=J.+M.+Colacinoauthor=J.+Babiakauthor=N.+G.+Almsteadauthor=S.+W.+Peltzauthor=L.+A.+Engauthor=K.+S.+Chenauthor=J.+L.+Mullauthor=M.+S.+Lynesauthor=L.+L.+Rubinauthor=P.+Fontouraauthor=L.+Santarelliauthor=D.+Haehnkeauthor=K.+D.+McCarthyauthor=R.+Schmuckiauthor=M.+Ebelingauthor=M.+Sivaramakrishnanauthor=C.+P.+Koauthor=S.+V.+Paushkinauthor=H.+Ratniauthor=I.+Gerlachauthor=A.+Ghoshauthor=F.+Metzger&title=Motor+Neuron+Disease.+Smn2+Splicing+Modifiers+Improve+Motor+Function+and+Longevity+in+Mice+with+Spinal+Muscular+Atrophy&doi=10.1126%2Fscience.1250127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">SMN2 splicing modifiers improve motor function and longevity in mice with spinal muscular atrophy</span></div><div class="casAuthors">Naryshkin, Nikolai A.; Weetall, Marla; Dakka, Amal; Narasimhan, Jana; Zhao, Xin; Feng, Zhihua; Ling, Karen K. Y.; Karp, Gary M.; Qi, Hongyan; Woll, Matthew G.; Chen, Guangming; Zhang, Nanjing; Gabbeta, Vijayalakshmi; Vazirani, Priya; Bhattacharyya, Anuradha; Furia, Bansri; Risher, Nicole; Sheedy, Josephine; Kong, Ronald; Ma, Jiyuan; Turpoff, Anthony; Lee, Chang-Sun; Zhang, Xiaoyan; Moon, Young-Choon; Trifillis, Panayiota; Welch, Ellen M.; Colacino, Joseph M.; Babiak, John; Almstead, Neil G.; Peltz, Stuart W.; Eng, Loren A.; Chen, Karen S.; Mull, Jesse L.; Lynes, Maureen S.; Rubin, Lee L.; Fontoura, Paulo; Santarelli, Luca; Haehnke, Daniel; McCarthy, Kathleen D.; Schmucki, Roland; Ebeling, Martin; Sivaramakrishnan, Manaswini; Ko, Chien-Ping; Paushkin, Sergey V.; Ratni, Hasane; Gerlach, Irene; Ghosh, Anirvan; Metzger, Friedrich</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">345</span>
        (<span class="NLM_cas:issue">6197</span>),
    <span class="NLM_cas:pages">688-693</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Spinal muscular atrophy (SMA) is a genetic disease caused by mutation or deletion of the survival of motor neuron 1 (SMN1) gene.  A paralogous gene in humans, SMN2, produces low, insufficient levels of functional SMN protein due to alternative splicing that truncates the transcript.  The decreased levels of SMN protein lead to progressive neuromuscular degeneration and high rates of mortality.  Through chem. screening and optimization, we identified orally available small mols. that shift the balance of SMN2 splicing toward the prodn. of full-length SMN2 mRNA with high selectivity.  Administration of these compds. to Δ7 mice, a model of severe SMA, led to an increase in SMN protein levels, improvement of motor function, and protection of the neuromuscular circuit.  These compds. also extended the life span of the mice.  Selective SMN2 splicing modifiers may have therapeutic potential for patients with SMA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxZ8QSg5KSzLVg90H21EOLACvtfcHk0li4bWcBCDnDuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12gtLfO&md5=f321c411bb384f95a9fe4d4debcbb195</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1126%2Fscience.1250127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1250127%26sid%3Dliteratum%253Aachs%26aulast%3DNaryshkin%26aufirst%3DN.%2BA.%26aulast%3DWeetall%26aufirst%3DM.%26aulast%3DDakka%26aufirst%3DA.%26aulast%3DNarasimhan%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DLing%26aufirst%3DK.%2BK.%26aulast%3DKarp%26aufirst%3DG.%2BM.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DWoll%26aufirst%3DM.%2BG.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DGabbeta%26aufirst%3DV.%26aulast%3DVazirani%26aufirst%3DP.%26aulast%3DBhattacharyya%26aufirst%3DA.%26aulast%3DFuria%26aufirst%3DB.%26aulast%3DRisher%26aufirst%3DN.%26aulast%3DSheedy%26aufirst%3DJ.%26aulast%3DKong%26aufirst%3DR.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DTurpoff%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DC.%2BS.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMoon%26aufirst%3DY.%2BC.%26aulast%3DTrifillis%26aufirst%3DP.%26aulast%3DWelch%26aufirst%3DE.%2BM.%26aulast%3DColacino%26aufirst%3DJ.%2BM.%26aulast%3DBabiak%26aufirst%3DJ.%26aulast%3DAlmstead%26aufirst%3DN.%2BG.%26aulast%3DPeltz%26aufirst%3DS.%2BW.%26aulast%3DEng%26aufirst%3DL.%2BA.%26aulast%3DChen%26aufirst%3DK.%2BS.%26aulast%3DMull%26aufirst%3DJ.%2BL.%26aulast%3DLynes%26aufirst%3DM.%2BS.%26aulast%3DRubin%26aufirst%3DL.%2BL.%26aulast%3DFontoura%26aufirst%3DP.%26aulast%3DSantarelli%26aufirst%3DL.%26aulast%3DHaehnke%26aufirst%3DD.%26aulast%3DMcCarthy%26aufirst%3DK.%2BD.%26aulast%3DSchmucki%26aufirst%3DR.%26aulast%3DEbeling%26aufirst%3DM.%26aulast%3DSivaramakrishnan%26aufirst%3DM.%26aulast%3DKo%26aufirst%3DC.%2BP.%26aulast%3DPaushkin%26aufirst%3DS.%2BV.%26aulast%3DRatni%26aufirst%3DH.%26aulast%3DGerlach%26aufirst%3DI.%26aulast%3DGhosh%26aufirst%3DA.%26aulast%3DMetzger%26aufirst%3DF.%26atitle%3DMotor%2520Neuron%2520Disease.%2520Smn2%2520Splicing%2520Modifiers%2520Improve%2520Motor%2520Function%2520and%2520Longevity%2520in%2520Mice%2520with%2520Spinal%2520Muscular%2520Atrophy%26jtitle%3DScience%26date%3D2014%26volume%3D345%26spage%3D688%26epage%3D693%26doi%3D10.1126%2Fscience.1250127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ratni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karp, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weetall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naryshkin, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paushkin, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turpoff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woll, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazirani, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinard, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Pierson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuerck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muster, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirchner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, F.</span></span> <span> </span><span class="NLM_article-title">Specific Correction of Alternative Survival Motor Neuron 2 Splicing by Small Molecules: Discovery of a Potential Novel Medicine to Treat Spinal Muscular Atrophy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">6086</span>– <span class="NLM_lpage">6100</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00459</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00459" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslWgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6086-6100&author=H.+Ratniauthor=G.+M.+Karpauthor=M.+Weetallauthor=N.+A.+Naryshkinauthor=S.+V.+Paushkinauthor=K.+S.+Chenauthor=K.+D.+McCarthyauthor=H.+Qiauthor=A.+Turpoffauthor=M.+G.+Wollauthor=X.+Zhangauthor=N.+Zhangauthor=T.+Yangauthor=A.+Dakkaauthor=P.+Vaziraniauthor=X.+Zhaoauthor=E.+Pinardauthor=L.+Greenauthor=P.+David-Piersonauthor=D.+Tuerckauthor=A.+Poirierauthor=W.+Musterauthor=S.+Kirchnerauthor=L.+Muellerauthor=I.+Gerlachauthor=F.+Metzger&title=Specific+Correction+of+Alternative+Survival+Motor+Neuron+2+Splicing+by+Small+Molecules%3A+Discovery+of+a+Potential+Novel+Medicine+to+Treat+Spinal+Muscular+Atrophy&doi=10.1021%2Facs.jmedchem.6b00459"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Specific Correction of Alternative Survival Motor Neuron 2 Splicing by Small Molecules: Discovery of a Potential Novel Medicine To Treat Spinal Muscular Atrophy</span></div><div class="casAuthors">Ratni, Hasane; Karp, Gary M.; Weetall, Marla; Naryshkin, Nikolai A.; Paushkin, Sergey V.; Chen, Karen S.; McCarthy, Kathleen D.; Qi, Hongyan; Turpoff, Anthony; Woll, Matthew G.; Zhang, Xiaoyan; Zhang, Nanjing; Yang, Tianle; Dakka, Amal; Vazirani, Priya; Zhao, Xin; Pinard, Emmanuel; Green, Luke; David-Pierson, Pascale; Tuerck, Dietrich; Poirier, Agnes; Muster, Wolfgang; Kirchner, Stephan; Mueller, Lutz; Gerlach, Irene; Metzger, Friedrich</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6086-6100</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Spinal muscular atrophy (SMA) is the leading genetic cause of infant and toddler mortality, and there is currently no approved therapy available.  SMA is caused by mutation or deletion of the survival motor neuron 1 (SMN1) gene.  These mutations or deletions result in low levels of functional SMN protein.  SMN2, a paralogous gene to SMN1, undergoes alternative splicing and exclusion of exon 7, producing an unstable, truncated SMNΔ7 protein.  Herein, we report the identification of a pyridopyrimidinone series of small mols. that modify the alternative splicing of SMN2, increasing the prodn. of full-length SMN2 mRNA.  Upon oral administration of our small mols., the levels of full-length SMN protein were restored in two mouse models of SMA.  In-depth lead optimization in the pyridopyrimidinone series culminated in the selection of compd. 3 (RG7800) (I), the first small mol. SMN2 splicing modifier to enter human clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDRpxc7_n787Vg90H21EOLACvtfcHk0lh73MDvzKD5_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslWgurw%253D&md5=14431b4127804cc623c2365298ad849b</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00459%26sid%3Dliteratum%253Aachs%26aulast%3DRatni%26aufirst%3DH.%26aulast%3DKarp%26aufirst%3DG.%2BM.%26aulast%3DWeetall%26aufirst%3DM.%26aulast%3DNaryshkin%26aufirst%3DN.%2BA.%26aulast%3DPaushkin%26aufirst%3DS.%2BV.%26aulast%3DChen%26aufirst%3DK.%2BS.%26aulast%3DMcCarthy%26aufirst%3DK.%2BD.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DTurpoff%26aufirst%3DA.%26aulast%3DWoll%26aufirst%3DM.%2BG.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DDakka%26aufirst%3DA.%26aulast%3DVazirani%26aufirst%3DP.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DPinard%26aufirst%3DE.%26aulast%3DGreen%26aufirst%3DL.%26aulast%3DDavid-Pierson%26aufirst%3DP.%26aulast%3DTuerck%26aufirst%3DD.%26aulast%3DPoirier%26aufirst%3DA.%26aulast%3DMuster%26aufirst%3DW.%26aulast%3DKirchner%26aufirst%3DS.%26aulast%3DMueller%26aufirst%3DL.%26aulast%3DGerlach%26aufirst%3DI.%26aulast%3DMetzger%26aufirst%3DF.%26atitle%3DSpecific%2520Correction%2520of%2520Alternative%2520Survival%2520Motor%2520Neuron%25202%2520Splicing%2520by%2520Small%2520Molecules%253A%2520Discovery%2520of%2520a%2520Potential%2520Novel%2520Medicine%2520to%2520Treat%2520Spinal%2520Muscular%2520Atrophy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6086%26epage%3D6100%26doi%3D10.1021%2Facs.jmedchem.6b00459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Woll, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turpoff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almstead, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naryshkin, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabbeta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welch, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Risher, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheedy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weetall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karp, G. M.</span></span> <span> </span><span class="NLM_article-title">Discovery and Optimization of Small Molecule Splicing Modifiers of Survival Motor Neuron 2 as a Treatment for Spinal Muscular Atrophy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">6070</span>– <span class="NLM_lpage">6085</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00460</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00460" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslWgur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6070-6085&author=M.+G.+Wollauthor=H.+Qiauthor=A.+Turpoffauthor=N.+Zhangauthor=X.+Zhangauthor=G.+Chenauthor=C.+Liauthor=S.+Huangauthor=T.+Yangauthor=Y.+C.+Moonauthor=C.+S.+Leeauthor=S.+Choiauthor=N.+G.+Almsteadauthor=N.+A.+Naryshkinauthor=A.+Dakkaauthor=J.+Narasimhanauthor=V.+Gabbetaauthor=E.+Welchauthor=X.+Zhaoauthor=N.+Risherauthor=J.+Sheedyauthor=M.+Weetallauthor=G.+M.+Karp&title=Discovery+and+Optimization+of+Small+Molecule+Splicing+Modifiers+of+Survival+Motor+Neuron+2+as+a+Treatment+for+Spinal+Muscular+Atrophy&doi=10.1021%2Facs.jmedchem.6b00460"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Small Molecule Splicing Modifiers of Survival Motor Neuron 2 as a Treatment for Spinal Muscular Atrophy</span></div><div class="casAuthors">Woll, Matthew G.; Qi, Hongyan; Turpoff, Anthony; Zhang, Nanjing; Zhang, Xiaoyan; Chen, Guangming; Li, Chunshi; Huang, Song; Yang, Tianle; Moon, Young-Choon; Lee, Chang-Sun; Choi, Soongyu; Almstead, Neil G.; Naryshkin, Nikolai A.; Dakka, Amal; Narasimhan, Jana; Gabbeta, Vijayalakshmi; Welch, Ellen; Zhao, Xin; Risher, Nicole; Sheedy, Josephine; Weetall, Marla; Karp, Gary M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6070-6085</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The underlying cause of spinal muscular atrophy (SMA) is a deficiency of the survival motor neuron (SMN) protein.  Starting from hits identified in a high-throughput screening campaign and through structure-activity relationship investigations, we have developed small mols. that potently shift the alternative splicing of the SMN2 exon 7, resulting in increased prodn. of the full-length SMN mRNA and protein.  Three novel chem. series, represented by compds. 9, 14, and 20, have been optimized to increase the level of SMN protein by >50% in SMA patient-derived fibroblasts at concns. of <160 nM.  Daily administration of these compds. to severe SMA Δ7 mice results in an increased prodn. of SMN protein in disease-relevant tissues and a significant increase in median survival time in a dose-dependent manner.  Our work supports the development of an orally administered small mol. for the treatment of patients with SMA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF6iU-0q9serVg90H21EOLACvtfcHk0lh73MDvzKD5_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslWgur0%253D&md5=9b32bd7380625b6ddd3e97479f98e79b</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00460&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00460%26sid%3Dliteratum%253Aachs%26aulast%3DWoll%26aufirst%3DM.%2BG.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DTurpoff%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DMoon%26aufirst%3DY.%2BC.%26aulast%3DLee%26aufirst%3DC.%2BS.%26aulast%3DChoi%26aufirst%3DS.%26aulast%3DAlmstead%26aufirst%3DN.%2BG.%26aulast%3DNaryshkin%26aufirst%3DN.%2BA.%26aulast%3DDakka%26aufirst%3DA.%26aulast%3DNarasimhan%26aufirst%3DJ.%26aulast%3DGabbeta%26aufirst%3DV.%26aulast%3DWelch%26aufirst%3DE.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DRisher%26aufirst%3DN.%26aulast%3DSheedy%26aufirst%3DJ.%26aulast%3DWeetall%26aufirst%3DM.%26aulast%3DKarp%26aufirst%3DG.%2BM.%26atitle%3DDiscovery%2520and%2520Optimization%2520of%2520Small%2520Molecule%2520Splicing%2520Modifiers%2520of%2520Survival%2520Motor%2520Neuron%25202%2520as%2520a%2520Treatment%2520for%2520Spinal%2520Muscular%2520Atrophy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6070%26epage%3D6085%26doi%3D10.1021%2Facs.jmedchem.6b00460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pickhardt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tassoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurkowsky, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitanovic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fava, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandelkow, E.</span></span> <span> </span><span class="NLM_article-title">Screening of a Neuronal Cell Model of Tau Pathology for Therapeutic Compounds</span>. <i>Neurobiol. Aging</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1016/j.neurobiolaging.2018.11.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.neurobiolaging.2018.11.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30640040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslelsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2019&pages=24-34&author=M.+Pickhardtauthor=M.+Tassoniauthor=P.+Dennerauthor=B.+Kurkowskyauthor=A.+Kitanovicauthor=C.+Mohlauthor=E.+Favaauthor=E.+Mandelkow&title=Screening+of+a+Neuronal+Cell+Model+of+Tau+Pathology+for+Therapeutic+Compounds&doi=10.1016%2Fj.neurobiolaging.2018.11.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Screening of a neuronal cell model of tau pathology for therapeutic compounds</span></div><div class="casAuthors">Pickhardt, Marcus; Tassoni, Michele; Denner, Philip; Kurkowsky, Birgit; Kitanovic, Ana; Moehl, Christoph; Fava, Eugenio; Mandelkow, Eckhard</div><div class="citationInfo"><span class="NLM_cas:title">Neurobiology of Aging</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">24-34</span>CODEN:
                <span class="NLM_cas:coden">NEAGDO</span>;
        ISSN:<span class="NLM_cas:issn">0197-4580</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">We have developed a cell-based phenotypic automated high-content screening approach for N2a cells expressing the pro-aggregant repeat domain of tau protein (tauRDΔK), which allows anal. of a chemogenomic library of 1649 compds. for their effect on the inhibition or stimulation of intracellular tau aggregation.  We identified several inhibitors and stimulators of aggregation and achieved a screening reproducibility >85% for all data.  We identified 18 potential inhibitors (= 1.1% of the library) and 10 stimulators (= 0.6% of the library) of tau aggregation in this cell model of tau pathol.  The results provide insights into the regulation of cellular tau aggregation and the pathways involved in this process (e.g., involving signaling via p38 mitogen-activated protein kinase, histone deacetylases, vascular endothelial growth factor, rho/ROCK).  For example, inhibitors of protein kinases (e.g., p38) can reduce tau aggregation, whereas inhibitors of deacetylases (histone deacetylases) can enhance aggregation.  These observations are compatible with reports that phosphorylated or acetylated tau promotes pathol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ6kH-hNDYULVg90H21EOLACvtfcHk0ljsoRFoCIe3Ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslelsw%253D%253D&md5=ef48fc4956a7c130bd6a4f50283d5aa5</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.neurobiolaging.2018.11.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neurobiolaging.2018.11.026%26sid%3Dliteratum%253Aachs%26aulast%3DPickhardt%26aufirst%3DM.%26aulast%3DTassoni%26aufirst%3DM.%26aulast%3DDenner%26aufirst%3DP.%26aulast%3DKurkowsky%26aufirst%3DB.%26aulast%3DKitanovic%26aufirst%3DA.%26aulast%3DMohl%26aufirst%3DC.%26aulast%3DFava%26aufirst%3DE.%26aulast%3DMandelkow%26aufirst%3DE.%26atitle%3DScreening%2520of%2520a%2520Neuronal%2520Cell%2520Model%2520of%2520Tau%2520Pathology%2520for%2520Therapeutic%2520Compounds%26jtitle%3DNeurobiol.%2520Aging%26date%3D2019%26volume%3D76%26spage%3D24%26epage%3D34%26doi%3D10.1016%2Fj.neurobiolaging.2018.11.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ofengeim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polydoro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degterev, A.</span></span> <span> </span><span class="NLM_article-title">Identification of Small Molecule Inhibitors of Neurite Loss Induced by Abeta Peptide Using High Content Screening</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">8714</span>– <span class="NLM_lpage">8723</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.290957</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1074%2Fjbc.M111.290957" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=22277654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC38XjvVeht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=8714-8723&author=D.+Ofengeimauthor=P.+Shiauthor=B.+Miaoauthor=J.+Fanauthor=X.+Xiaauthor=Y.+Fanauthor=M.+M.+Lipinskiauthor=T.+Hashimotoauthor=M.+Polydoroauthor=J.+Yuanauthor=S.+T.+Wongauthor=A.+Degterev&title=Identification+of+Small+Molecule+Inhibitors+of+Neurite+Loss+Induced+by+Abeta+Peptide+Using+High+Content+Screening&doi=10.1074%2Fjbc.M111.290957"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Small Molecule Inhibitors of Neurite Loss Induced by Aβ peptide using High Content Screening</span></div><div class="casAuthors">Ofengeim, Dimitry; Shi, Peng; Miao, Benchun; Fan, Jing; Xia, Xiaofeng; Fan, Yubo; Lipinski, Marta M.; Hashimoto, Tadafumi; Polydoro, Manuela; Yuan, Junying; Wong, Stephen T. C.; Degterev, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">8714-8723</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Multiple lines of evidence indicate a strong relationship between Aβ peptide-induced neurite degeneration and the progressive loss of cognitive functions in Alzheimer disease (AD) patients and in AD animal models.  This prompted us to develop a high content screening assay (HCS) and Neurite Image Quantitator (NeuriteIQ) software to quantify the loss of neuronal projections induced by Aβ peptide neurons and enable us to identify new classes of neurite-protective small mols., which may represent new leads for AD drug discovery.  We identified thirty-six inhibitors of Aβ-induced neurite loss in the 1,040-compd. National Institute of Neurol. Disorders and Stroke (NINDS) custom collection of known bioactives and FDA approved drugs.  Activity clustering showed that non-steroidal anti-inflammatory drugs (NSAIDs) were significantly enriched among the hits.  Notably, NSAIDs have previously attracted significant attention as potential drugs for AD; however their mechanism of action remains controversial.  Our data revealed that cyclooxygenase-2 (COX-2) expression was increased following Aβ treatment.  Furthermore, multiple distinct classes of COX inhibitors efficiently blocked neurite loss in primary neurons, suggesting that increased COX activity contributes to Aβ peptide-induced neurite loss.  Finally, we discovered that the detrimental effect of COX activity on neurite integrity may be mediated through the inhibition of peroxisome proliferator-activated receptor γ (PPARγ) activity.  Overall, our work establishes the feasibility of identifying small mol. inhibitors of Aβ-induced neurite loss using the NeuriteIQ pipeline and provides novel insights into the mechanisms of neuroprotection by NSAIDs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1btFfJUyhMbVg90H21EOLACvtfcHk0ljsoRFoCIe3Ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjvVeht7c%253D&md5=05a4cac624f04f8882491bada44355fa</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.290957&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.290957%26sid%3Dliteratum%253Aachs%26aulast%3DOfengeim%26aufirst%3DD.%26aulast%3DShi%26aufirst%3DP.%26aulast%3DMiao%26aufirst%3DB.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DXia%26aufirst%3DX.%26aulast%3DFan%26aufirst%3DY.%26aulast%3DLipinski%26aufirst%3DM.%2BM.%26aulast%3DHashimoto%26aufirst%3DT.%26aulast%3DPolydoro%26aufirst%3DM.%26aulast%3DYuan%26aufirst%3DJ.%26aulast%3DWong%26aufirst%3DS.%2BT.%26aulast%3DDegterev%26aufirst%3DA.%26atitle%3DIdentification%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520Neurite%2520Loss%2520Induced%2520by%2520Abeta%2520Peptide%2520Using%2520High%2520Content%2520Screening%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D8714%26epage%3D8723%26doi%3D10.1074%2Fjbc.M111.290957" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Champy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seon-Meniel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franck, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raisman-Vozari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figadere, B.</span></span> <span> </span><span class="NLM_article-title">Chemicals Possessing a Neurotrophin-Like Activity on Dopaminergic Neurons in Primary Culture</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">e6215</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0006215</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1371%2Fjournal.pone.0006215" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=19593444" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&author=F.+Schmidtauthor=P.+Champyauthor=B.+Seon-Menielauthor=X.+Franckauthor=R.+Raisman-Vozariauthor=B.+Figadere&title=Chemicals+Possessing+a+Neurotrophin-Like+Activity+on+Dopaminergic+Neurons+in+Primary+Culture&doi=10.1371%2Fjournal.pone.0006215"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0006215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0006215%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DF.%26aulast%3DChampy%26aufirst%3DP.%26aulast%3DSeon-Meniel%26aufirst%3DB.%26aulast%3DFranck%26aufirst%3DX.%26aulast%3DRaisman-Vozari%26aufirst%3DR.%26aulast%3DFigadere%26aufirst%3DB.%26atitle%3DChemicals%2520Possessing%2520a%2520Neurotrophin-Like%2520Activity%2520on%2520Dopaminergic%2520Neurons%2520in%2520Primary%2520Culture%26jtitle%3DPLoS%2520One%26date%3D2009%26volume%3D4%26doi%3D10.1371%2Fjournal.pone.0006215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mabb, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, I. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miriyala, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor-Blake, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sciaky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutton, J. W.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zylka, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpot, B. D.</span></span> <span> </span><span class="NLM_article-title">Topoisomerase Inhibitors Unsilence the Dormant Allele of Ube3a in Neurons</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>481</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">189</span>, <span class="refDoi"> DOI: 10.1038/nature10726</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnature10726" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1GjsL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=481&publication_year=2012&pages=185-189&author=H.+S.+Huangauthor=J.+A.+Allenauthor=A.+M.+Mabbauthor=I.+F.+Kingauthor=J.+Miriyalaauthor=B.+Taylor-Blakeauthor=N.+Sciakyauthor=J.+W.+Duttonauthor=H.+M.+Leeauthor=X.+Chenauthor=J.+Jinauthor=A.+S.+Bridgesauthor=M.+J.+Zylkaauthor=B.+L.+Rothauthor=B.+D.+Philpot&title=Topoisomerase+Inhibitors+Unsilence+the+Dormant+Allele+of+Ube3a+in+Neurons&doi=10.1038%2Fnature10726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Topoisomerase inhibitors unsilence the dormant allele of Ube3a in neurons</span></div><div class="casAuthors">Huang, Hsien-Sung; Allen, John A.; Mabb, Angela M.; King, Ian F.; Miriyala, Jayalakshmi; Taylor-Blake, Bonnie; Sciaky, Noah; Dutton, J. Walter; Lee, Hyeong-Min; Chen, Xin; Jin, Jian; Bridges, Arlene S.; Zylka, Mark J.; Roth, Bryan L.; Philpot, Benjamin D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">481</span>
        (<span class="NLM_cas:issue">7380</span>),
    <span class="NLM_cas:pages">185-189</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Angelman syndrome is a severe neurodevelopmental disorder caused by deletion or mutation of the maternal allele of the ubiquitin protein ligase E3A (UBE3A).  In neurons, the paternal allele of UBE3A is intact but epigenetically silenced, raising the possibility that Angelman syndrome could be treated by activating this silenced allele to restore functional UBE3A protein.  Using an unbiased, high-content screen in primary cortical neurons from mice, we identify twelve topoisomerase I inhibitors and four topoisomerase II inhibitors that unsilence the paternal Ube3a allele.  These drugs included topotecan, irinotecan, etoposide and dexrazoxane (ICRF-187).  At nanomolar concns., topotecan upregulated catalytically active UBE3A in neurons from maternal Ube3a-null mice.  Topotecan concomitantly downregulated expression of the Ube3a antisense transcript that overlaps the paternal copy of Ube3a antisense transcript that overlaps the paternal copy of Ube3a.  These results indicate that topotecan unsilences Ube3a in cis by reducing transcription of an imprinted antisense RNA.  When administered in vivo, topotecan unsilenced the paternal Ube3a allele in several regions of the nervous system, including neurons in the hippocampus, neocortex, striatum, cerebellum and spinal cord.  Paternal expression of Ube3a remained elevated in a subset of spinal cord neurons for at least 12 wk after cessation of topotecan treatment, indicating that transient topoisomerase inhibition can have enduring effects on gene expression.  Although potential off-target effects remain to be investigated, these findings suggest a therapeutic strategy for reactivating the functional but dormant allele of Ube3a in patients with Angelman syndrome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyX6PY0uT7lLVg90H21EOLACvtfcHk0lhPI0F-mrBlSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1GjsL%252FM&md5=659adcf73a6c9f6b432dd34431ae1308</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnature10726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10726%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%2BS.%26aulast%3DAllen%26aufirst%3DJ.%2BA.%26aulast%3DMabb%26aufirst%3DA.%2BM.%26aulast%3DKing%26aufirst%3DI.%2BF.%26aulast%3DMiriyala%26aufirst%3DJ.%26aulast%3DTaylor-Blake%26aufirst%3DB.%26aulast%3DSciaky%26aufirst%3DN.%26aulast%3DDutton%26aufirst%3DJ.%2BW.%26aulast%3DLee%26aufirst%3DH.%2BM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DBridges%26aufirst%3DA.%2BS.%26aulast%3DZylka%26aufirst%3DM.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DPhilpot%26aufirst%3DB.%2BD.%26atitle%3DTopoisomerase%2520Inhibitors%2520Unsilence%2520the%2520Dormant%2520Allele%2520of%2520Ube3a%2520in%2520Neurons%26jtitle%3DNature%26date%3D2012%26volume%3D481%26spage%3D185%26epage%3D189%26doi%3D10.1038%2Fnature10726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqi, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menzies, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamatakou, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karabiyik, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ureshino, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricketts, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimenez-Sanchez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteban, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tortorella, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzakopian, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandes, H. J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karran, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubinsztein, D. C.</span></span> <span> </span><span class="NLM_article-title">Felodipine Induces Autophagy in Mouse Brains with Pharmacokinetics Amenable to Repurposing</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1817</span>, <span class="refDoi"> DOI: 10.1038/s41467-019-09494-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41467-019-09494-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=31000720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BB3M%252FmvF2kuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1817&author=F.+H.+Siddiqiauthor=F.+M.+Menziesauthor=A.+Lopezauthor=E.+Stamatakouauthor=C.+Karabiyikauthor=R.+Ureshinoauthor=T.+Rickettsauthor=M.+Jimenez-Sanchezauthor=M.+A.+Estebanauthor=L.+Laiauthor=M.+D.+Tortorellaauthor=Z.+Luoauthor=H.+Liuauthor=E.+Metzakopianauthor=H.+J.+R.+Fernandesauthor=A.+Bassettauthor=E.+Karranauthor=B.+L.+Millerauthor=A.+Flemingauthor=D.+C.+Rubinsztein&title=Felodipine+Induces+Autophagy+in+Mouse+Brains+with+Pharmacokinetics+Amenable+to+Repurposing&doi=10.1038%2Fs41467-019-09494-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Felodipine induces autophagy in mouse brains with pharmacokinetics amenable to repurposing</span></div><div class="casAuthors">Siddiqi Farah H; Menzies Fiona M; Lopez Ana; Stamatakou Eleanna; Karabiyik Cansu; Ureshino Rodrigo; Ricketts Thomas; Jimenez-Sanchez Maria; Fleming Angeleen; Rubinsztein David C; Siddiqi Farah H; Stamatakou Eleanna; Rubinsztein David C; Lopez Ana; Ureshino Rodrigo; Fleming Angeleen; Jimenez-Sanchez Maria; Esteban Miguel Angel; Lai Liangxue; Tortorella Micky D; Luo Zhiwei; Liu Hao; Metzakopian Emmanouil; Fernandes Hugo J R; Bassett Andrew; Karran Eric; Miller Bruce L</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1817</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Neurodegenerative diseases like Alzheimer's disease, Parkinson's disease and Huntington's disease manifest with the neuronal accumulation of toxic proteins.  Since autophagy upregulation enhances the clearance of such proteins and ameliorates their toxicities in animal models, we and others have sought to re-position/re-profile existing compounds used in humans to identify those that may induce autophagy in the brain.  A key challenge with this approach is to assess if any hits identified can induce neuronal autophagy at concentrations that would be seen in humans taking the drug for its conventional indication.  Here we report that felodipine, an L-type calcium channel blocker and anti-hypertensive drug, induces autophagy and clears diverse aggregate-prone, neurodegenerative disease-associated proteins.  Felodipine can clear mutant α-synuclein in mouse brains at plasma concentrations similar to those that would be seen in humans taking the drug.  This is associated with neuroprotection in mice, suggesting the promise of this compound for use in neurodegeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSPsrrt_ir8Uv6AExegM5LLfW6udTcc2eZe-OkpmKhf4rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M%252FmvF2kuw%253D%253D&md5=09a695108aebc0b2d27275acf143063e</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fs41467-019-09494-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-019-09494-2%26sid%3Dliteratum%253Aachs%26aulast%3DSiddiqi%26aufirst%3DF.%2BH.%26aulast%3DMenzies%26aufirst%3DF.%2BM.%26aulast%3DLopez%26aufirst%3DA.%26aulast%3DStamatakou%26aufirst%3DE.%26aulast%3DKarabiyik%26aufirst%3DC.%26aulast%3DUreshino%26aufirst%3DR.%26aulast%3DRicketts%26aufirst%3DT.%26aulast%3DJimenez-Sanchez%26aufirst%3DM.%26aulast%3DEsteban%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DL.%26aulast%3DTortorella%26aufirst%3DM.%2BD.%26aulast%3DLuo%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DMetzakopian%26aufirst%3DE.%26aulast%3DFernandes%26aufirst%3DH.%2BJ.%2BR.%26aulast%3DBassett%26aufirst%3DA.%26aulast%3DKarran%26aufirst%3DE.%26aulast%3DMiller%26aufirst%3DB.%2BL.%26aulast%3DFleming%26aufirst%3DA.%26aulast%3DRubinsztein%26aufirst%3DD.%2BC.%26atitle%3DFelodipine%2520Induces%2520Autophagy%2520in%2520Mouse%2520Brains%2520with%2520Pharmacokinetics%2520Amenable%2520to%2520Repurposing%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D1817%26doi%3D10.1038%2Fs41467-019-09494-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujimori, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otomo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atsuta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akiyama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobue, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okano, H.</span></span> <span> </span><span class="NLM_article-title">Modeling Sporadic Als in Ipsc-Derived Motor Neurons Identifies a Potential Therapeutic Agent</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1579</span>– <span class="NLM_lpage">1589</span>, <span class="refDoi"> DOI: 10.1038/s41591-018-0140-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41591-018-0140-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30127392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFGjur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=1579-1589&author=K.+Fujimoriauthor=M.+Ishikawaauthor=A.+Otomoauthor=N.+Atsutaauthor=R.+Nakamuraauthor=T.+Akiyamaauthor=S.+Hadanoauthor=M.+Aokiauthor=H.+Sayaauthor=G.+Sobueauthor=H.+Okano&title=Modeling+Sporadic+Als+in+Ipsc-Derived+Motor+Neurons+Identifies+a+Potential+Therapeutic+Agent&doi=10.1038%2Fs41591-018-0140-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling sporadic ALS in iPSC-derived motor neurons identifies a potential therapeutic agent</span></div><div class="casAuthors">Fujimori, Koki; Ishikawa, Mitsuru; Otomo, Asako; Atsuta, Naoki; Nakamura, Ryoichi; Akiyama, Tetsuya; Hadano, Shinji; Aoki, Masashi; Saya, Hideyuki; Sobue, Gen; Okano, Hideyuki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1579-1589</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Amyotrophic lateral sclerosis (ALS) is a heterogeneous motor neuron disease for which no effective treatment is available, despite decades of research into SOD1-mutant familial ALS (FALS).  The majority of ALS patients have no familial history, making the modeling of sporadic ALS (SALS) essential to the development of ALS therapeutics.  However, as mutations underlying ALS pathogenesis have not yet been identified, it remains difficult to establish useful models of SALS.  Using induced pluripotent stem cell (iPSC) technol. to generate stem and differentiated cells retaining the patients' full genetic information, we have established a large no. of in vitro cellular models of SALS.  These models showed phenotypic differences in their pattern of neuronal degeneration, types of abnormal protein aggregates, cell death mechanisms, and onset and progression of these phenotypes in vitro among cases.  We therefore developed a system for case clustering capable of subdividing these heterogeneous SALS models by their in vitro characteristics.  We further evaluated multiple-phenotype rescue of these subclassified SALS models using agents selected from non-SOD1 FALS models, and identified ropinirole as a potential therapeutic candidate.  Integration of the datasets acquired in this study permitted the visualization of mol. pathologies shared across a wide range of SALS models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZe3w-U7d4qbVg90H21EOLACvtfcHk0lhPI0F-mrBlSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFGjur3M&md5=e858757be57e2ad32448221abc6b5eb1</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fs41591-018-0140-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-018-0140-5%26sid%3Dliteratum%253Aachs%26aulast%3DFujimori%26aufirst%3DK.%26aulast%3DIshikawa%26aufirst%3DM.%26aulast%3DOtomo%26aufirst%3DA.%26aulast%3DAtsuta%26aufirst%3DN.%26aulast%3DNakamura%26aufirst%3DR.%26aulast%3DAkiyama%26aufirst%3DT.%26aulast%3DHadano%26aufirst%3DS.%26aulast%3DAoki%26aufirst%3DM.%26aulast%3DSaya%26aufirst%3DH.%26aulast%3DSobue%26aufirst%3DG.%26aulast%3DOkano%26aufirst%3DH.%26atitle%3DModeling%2520Sporadic%2520Als%2520in%2520Ipsc-Derived%2520Motor%2520Neurons%2520Identifies%2520a%2520Potential%2520Therapeutic%2520Agent%26jtitle%3DNat.%2520Med.%26date%3D2018%26volume%3D24%26spage%3D1579%26epage%3D1589%26doi%3D10.1038%2Fs41591-018-0140-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tracy, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sohn, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludwig, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer-Franke, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karch, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, L.</span></span> <span> </span><span class="NLM_article-title">Scalable Production of Ipsc-Derived Human Neurons to Identify Tau-Lowering Compounds by High-Content Screening</span>. <i>Stem Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1221</span>– <span class="NLM_lpage">1233</span>, <span class="refDoi"> DOI: 10.1016/j.stemcr.2017.08.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.stemcr.2017.08.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28966121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOrtrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=1221-1233&author=C.+Wangauthor=M.+E.+Wardauthor=R.+Chenauthor=K.+Liuauthor=T.+E.+Tracyauthor=X.+Chenauthor=M.+Xieauthor=P.+D.+Sohnauthor=C.+Ludwigauthor=A.+Meyer-Frankeauthor=C.+M.+Karchauthor=S.+Dingauthor=L.+Gan&title=Scalable+Production+of+Ipsc-Derived+Human+Neurons+to+Identify+Tau-Lowering+Compounds+by+High-Content+Screening&doi=10.1016%2Fj.stemcr.2017.08.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Scalable Production of iPSC-Derived Human Neurons to Identify Tau-Lowering Compounds by High-Content Screening</span></div><div class="casAuthors">Wang, Chao; Ward, Michael E.; Chen, Robert; Liu, Kai; Tracy, Tara E.; Chen, Xu; Xie, Min; Sohn, Peter Dongmin; Ludwig, Connor; Meyer-Franke, Anke; Karch, Celeste M.; Ding, Sheng; Gan, Li</div><div class="citationInfo"><span class="NLM_cas:title">Stem Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1221-1233</span>CODEN:
                <span class="NLM_cas:coden">SCRTFL</span>;
        ISSN:<span class="NLM_cas:issn">2213-6711</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Lowering total tau levels is an attractive therapeutic strategy for Alzheimer's disease and other tauopathies.  High-throughput screening in neurons derived from human induced pluripotent stem cells (iPSCs) is a powerful tool to identify tau-targeted therapeutics.  However, such screens have been hampered by heterogeneous neuronal prodn., high cost and low yield, and multi-step differentiation procedures.  We engineered an isogenic iPSC line that harbors an inducible neurogenin 2 transgene, a transcription factor that rapidly converts iPSCs to neurons, integrated at the AAVS1 locus.  Using a simplified two-step protocol, we differentiated these iPSCs into cortical glutamatergic neurons with minimal well-to-well variability.  We developed a robust high-content screening assay to identify tau-lowering compds. in LOPAC and identified adrenergic receptors agonists as a class of compds. that reduce endogenous human tau.  These techniques enable the use of human neurons for high-throughput screening of drugs to treat neurodegenerative disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhY3PvM_2Qf7Vg90H21EOLACvtfcHk0lgsrLsvw_jrGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOrtrnJ&md5=194fb07160abc9b2d1f798328385839c</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.stemcr.2017.08.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.stemcr.2017.08.019%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWard%26aufirst%3DM.%2BE.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DK.%26aulast%3DTracy%26aufirst%3DT.%2BE.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DXie%26aufirst%3DM.%26aulast%3DSohn%26aufirst%3DP.%2BD.%26aulast%3DLudwig%26aufirst%3DC.%26aulast%3DMeyer-Franke%26aufirst%3DA.%26aulast%3DKarch%26aufirst%3DC.%2BM.%26aulast%3DDing%26aufirst%3DS.%26aulast%3DGan%26aufirst%3DL.%26atitle%3DScalable%2520Production%2520of%2520Ipsc-Derived%2520Human%2520Neurons%2520to%2520Identify%2520Tau-Lowering%2520Compounds%2520by%2520High-Content%2520Screening%26jtitle%3DStem%2520Cell%2520Rep.%26date%3D2017%26volume%3D9%26spage%3D1221%26epage%3D1233%26doi%3D10.1016%2Fj.stemcr.2017.08.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kondo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funayama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyake, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woltjen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawakatsu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izumi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaji, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, H.</span></span> <span> </span><span class="NLM_article-title">Ipsc-Based Compound Screening and in Vitro Trials Identify a Synergistic Anti-Amyloid Beta Combination for Alzheimer’s Disease</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2304</span>– <span class="NLM_lpage">2312</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.10.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.celrep.2017.10.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=29166618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOrs7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=2304-2312&author=T.+Kondoauthor=K.+Imamuraauthor=M.+Funayamaauthor=K.+Tsukitaauthor=M.+Miyakeauthor=A.+Ohtaauthor=K.+Woltjenauthor=M.+Nakagawaauthor=T.+Asadaauthor=T.+Araiauthor=S.+Kawakatsuauthor=Y.+Izumiauthor=R.+Kajiauthor=N.+Iwataauthor=H.+Inoue&title=Ipsc-Based+Compound+Screening+and+in+Vitro+Trials+Identify+a+Synergistic+Anti-Amyloid+Beta+Combination+for+Alzheimer%E2%80%99s+Disease&doi=10.1016%2Fj.celrep.2017.10.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">iPSC-Based Compound Screening and In Vitro Trials Identify a Synergistic Anti-amyloid β Combination for Alzheimer's Disease</span></div><div class="casAuthors">Kondo, Takayuki; Imamura, Keiko; Funayama, Misato; Tsukita, Kayoko; Miyake, Michiyo; Ohta, Akira; Woltjen, Knut; Nakagawa, Masato; Asada, Takashi; Arai, Tetsuaki; Kawakatsu, Shinobu; Izumi, Yuishin; Kaji, Ryuji; Iwata, Nobuhisa; Inoue, Haruhisa</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2304-2312</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">In the process of drug development, in vitro studies do not always adequately predict human-specific drug responsiveness in clin. trials.  Here, we applied the advantage of human iPSC-derived neurons, which offer human-specific drug responsiveness, to screen and evaluate therapeutic candidates for Alzheimer's disease (AD).  Using AD patient neurons with nearly 100% purity from iPSCs, we established a robust and reproducible assay for amyloid β peptide (Aβ), a pathogenic mol. in AD, and screened a pharmaceutical compd. library.  We acquired 27 Aβ-lowering screen hits, prioritized hits by chem. structure-based clustering, and selected 6 leading compds.  Next, to maximize the anti-Aβ effect, we selected a synergistic combination of bromocriptine, cromolyn, and topiramate as an anti-Aβ cocktail.  Finally, using neurons from familial and sporadic AD patients, we found that the cocktail showed a significant and potent anti-Aβ effect on patient cells.  This human iPSC-based platform promises to be useful for AD drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprnljAp1qpz7Vg90H21EOLACvtfcHk0lgsrLsvw_jrGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOrs7fN&md5=62a79bcd33bdcb3e529e7890fa04f18a</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.10.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.10.109%26sid%3Dliteratum%253Aachs%26aulast%3DKondo%26aufirst%3DT.%26aulast%3DImamura%26aufirst%3DK.%26aulast%3DFunayama%26aufirst%3DM.%26aulast%3DTsukita%26aufirst%3DK.%26aulast%3DMiyake%26aufirst%3DM.%26aulast%3DOhta%26aufirst%3DA.%26aulast%3DWoltjen%26aufirst%3DK.%26aulast%3DNakagawa%26aufirst%3DM.%26aulast%3DAsada%26aufirst%3DT.%26aulast%3DArai%26aufirst%3DT.%26aulast%3DKawakatsu%26aufirst%3DS.%26aulast%3DIzumi%26aufirst%3DY.%26aulast%3DKaji%26aufirst%3DR.%26aulast%3DIwata%26aufirst%3DN.%26aulast%3DInoue%26aufirst%3DH.%26atitle%3DIpsc-Based%2520Compound%2520Screening%2520and%2520in%2520Vitro%2520Trials%2520Identify%2520a%2520Synergistic%2520Anti-Amyloid%2520Beta%2520Combination%2520for%2520Alzheimer%25E2%2580%2599s%2520Disease%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D21%26spage%3D2304%26epage%3D2312%26doi%3D10.1016%2Fj.celrep.2017.10.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uzel, S. G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamm, R. D.</span></span> <span> </span><span class="NLM_article-title">Microphysiological 3d Model of Amyotrophic Lateral Sclerosis (Als) from Human Ips-Derived Muscle Cells and Optogenetic Motor Neurons</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">eaat5847</span> <span class="refDoi"> DOI: 10.1126/sciadv.aat5847</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fsciadv.aat5847" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30324134" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&author=T.+Osakiauthor=S.+G.+M.+Uzelauthor=R.+D.+Kamm&title=Microphysiological+3d+Model+of+Amyotrophic+Lateral+Sclerosis+%28Als%29+from+Human+Ips-Derived+Muscle+Cells+and+Optogenetic+Motor+Neurons&doi=10.1126%2Fsciadv.aat5847"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.aat5847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.aat5847%26sid%3Dliteratum%253Aachs%26aulast%3DOsaki%26aufirst%3DT.%26aulast%3DUzel%26aufirst%3DS.%2BG.%2BM.%26aulast%3DKamm%26aufirst%3DR.%2BD.%26atitle%3DMicrophysiological%25203d%2520Model%2520of%2520Amyotrophic%2520Lateral%2520Sclerosis%2520%2528Als%2529%2520from%2520Human%2520Ips-Derived%2520Muscle%2520Cells%2520and%2520Optogenetic%2520Motor%2520Neurons%26jtitle%3DSci.%2520Adv.%26date%3D2018%26volume%3D4%26doi%3D10.1126%2Fsciadv.aat5847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hor, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soh, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, V. J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santosa, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winanto</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, B. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soh, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, S. Y.</span></span> <span> </span><span class="NLM_article-title">Cell Cycle Inhibitors Protect Motor Neurons in an Organoid Model of Spinal Muscular Atrophy</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1100</span>, <span class="refDoi"> DOI: 10.1038/s41419-018-1081-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41419-018-1081-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30368521" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BB3cvktVGitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1100&author=J.+H.+Horauthor=E.+S.+Sohauthor=L.+Y.+Tanauthor=V.+J.+W.+Limauthor=M.+M.+Santosaauthor=+Winantoauthor=B.+X.+Hoauthor=Y.+Fanauthor=B.+S.+Sohauthor=S.+Y.+Ng&title=Cell+Cycle+Inhibitors+Protect+Motor+Neurons+in+an+Organoid+Model+of+Spinal+Muscular+Atrophy&doi=10.1038%2Fs41419-018-1081-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Cell cycle inhibitors protect motor neurons in an organoid model of Spinal Muscular Atrophy</span></div><div class="casAuthors">Hor Jin Hui; Soh Eunice Shi-Yi; Tan Li Yi; Lim Valerie Jing Wen; Santosa Munirah Mohamad; Winanto; Ho Beatrice Xuan; Soh Boon-Seng; Ng Shi-Yan; Hor Jin Hui; Ho Beatrice Xuan; Soh Boon-Seng; Soh Eunice Shi-Yi; Winanto; Fan Yong; Soh Boon-Seng; Ng Shi-Yan; Ng Shi-Yan; Ng Shi-Yan</div><div class="citationInfo"><span class="NLM_cas:title">Cell death & disease</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1100</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Spinal Muscular Atrophy (SMA) is caused by genetic mutations in the SMN1 gene, resulting in drastically reduced levels of Survival of Motor Neuron (SMN) protein.  Although SMN is ubiquitously expressed, spinal motor neurons are one of the most affected cell types.  Previous studies have identified pathways uniquely activated in SMA motor neurons, including a hyperactivated ER stress pathway, neuronal hyperexcitability, and defective spliceosomes.  To investigate why motor neurons are more affected than other neural types, we developed a spinal organoid model of SMA.  We demonstrate overt motor neuron degeneration in SMA spinal organoids, and this degeneration can be prevented using a small molecule inhibitor of CDK4/6, indicating that spinal organoids are an ideal platform for therapeutic discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT8cW2iZzjIi1w2ofyr07jXfW6udTcc2eZoSi8db07NH7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cvktVGitQ%253D%253D&md5=d36b00f8b6d2f678179f6d5a6c5e7c7a</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1038%2Fs41419-018-1081-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41419-018-1081-0%26sid%3Dliteratum%253Aachs%26aulast%3DHor%26aufirst%3DJ.%2BH.%26aulast%3DSoh%26aufirst%3DE.%2BS.%26aulast%3DTan%26aufirst%3DL.%2BY.%26aulast%3DLim%26aufirst%3DV.%2BJ.%2BW.%26aulast%3DSantosa%26aufirst%3DM.%2BM.%26aulast%3DWinanto%26aulast%3DHo%26aufirst%3DB.%2BX.%26aulast%3DFan%26aufirst%3DY.%26aulast%3DSoh%26aufirst%3DB.%2BS.%26aulast%3DNg%26aufirst%3DS.%2BY.%26atitle%3DCell%2520Cycle%2520Inhibitors%2520Protect%2520Motor%2520Neurons%2520in%2520an%2520Organoid%2520Model%2520of%2520Spinal%2520Muscular%2520Atrophy%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2018%26volume%3D9%26spage%3D1100%26doi%3D10.1038%2Fs41419-018-1081-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsvetkov, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrasate, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pleiss, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span> <span> </span><span class="NLM_article-title">A Small-Molecule Scaffold Induces Autophagy in Primary Neurons and Protects against Toxicity in a Huntington Disease Model</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">16982</span>– <span class="NLM_lpage">16987</span>, <span class="refDoi"> DOI: 10.1073/pnas.1004498107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1073%2Fpnas.1004498107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=20833817" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Oht7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=16982-16987&author=A.+S.+Tsvetkovauthor=J.+Millerauthor=M.+Arrasateauthor=J.+S.+Wongauthor=M.+A.+Pleissauthor=S.+Finkbeiner&title=A+Small-Molecule+Scaffold+Induces+Autophagy+in+Primary+Neurons+and+Protects+against+Toxicity+in+a+Huntington+Disease+Model&doi=10.1073%2Fpnas.1004498107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">A small-molecule scaffold induces autophagy in primary neurons and protects against toxicity in a Huntington disease model</span></div><div class="casAuthors">Tsvetkov, Andrey S.; Miller, Jason; Arrasate, Montserrat; Wong, Jinny S.; Pleiss, Michael A.; Finkbeiner, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">16982-16987, S16982/1-S16982/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Autophagy is an intracellular turnover pathway.  It has special relevance for neurodegenerative proteinopathies, such as Alzheimer disease, Parkinson disease, and Huntington disease (HD), which are characterized by the accumulation of misfolded proteins.  Although induction of autophagy enhances clearance of misfolded protein and has therefore been suggested as a therapy for proteinopathies, neurons appear to be less responsive to classic autophagy inducers than nonneuronal cells.  Searching for improved inducers of neuronal autophagy, we discovered an N10-substituted phenoxazine that, at proper doses, potently and safely up-regulated autophagy in neurons in an Akt- and mTOR-independent fashion.  In a neuron model of HD, this compd. was neuroprotective and decreased the accumulation of diffuse and aggregated misfolded protein.  A structure/activity anal. with structurally similar compds. approved by the US Food and Drug Administration revealed a defined pharmacophore for inducing neuronal autophagy.  This pharmacophore should prove useful in studying autophagy in neurons and in developing therapies for neurodegenerative proteinopathies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtnUfvW65_mrVg90H21EOLACvtfcHk0liPNzYv_ofbOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Oht7fP&md5=aaefe7b3fc1c0e12c53b67c660219767</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1004498107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1004498107%26sid%3Dliteratum%253Aachs%26aulast%3DTsvetkov%26aufirst%3DA.%2BS.%26aulast%3DMiller%26aufirst%3DJ.%26aulast%3DArrasate%26aufirst%3DM.%26aulast%3DWong%26aufirst%3DJ.%2BS.%26aulast%3DPleiss%26aufirst%3DM.%2BA.%26aulast%3DFinkbeiner%26aufirst%3DS.%26atitle%3DA%2520Small-Molecule%2520Scaffold%2520Induces%2520Autophagy%2520in%2520Primary%2520Neurons%2520and%2520Protects%2520against%2520Toxicity%2520in%2520a%2520Huntington%2520Disease%2520Model%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D16982%26epage%3D16987%26doi%3D10.1073%2Fpnas.1004498107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theodoropoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKnight, S. L.</span></span> <span> </span><span class="NLM_article-title">Poly-Dipeptides Encoded by the C9orf72 Repeats Bind Nucleoli, Impede Rna Biogenesis, and Kill Cells</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>345</i></span>,  <span class="NLM_fpage">1139</span>– <span class="NLM_lpage">1145</span>, <span class="refDoi"> DOI: 10.1126/science.1254917</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fscience.1254917" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25081482" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=345&publication_year=2014&pages=1139-1145&author=I.+Kwonauthor=S.+Xiangauthor=M.+Katoauthor=L.+Wuauthor=P.+Theodoropoulosauthor=T.+Wangauthor=J.+Kimauthor=J.+Yunauthor=Y.+Xieauthor=S.+L.+McKnight&title=Poly-Dipeptides+Encoded+by+the+C9orf72+Repeats+Bind+Nucleoli%2C+Impede+Rna+Biogenesis%2C+and+Kill+Cells&doi=10.1126%2Fscience.1254917"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Poly-dipeptides encoded by the C9orf72 repeats bind nucleoli, impede RNA biogenesis, and kill cells</span></div><div class="casAuthors">Kwon, Ilmin; Xiang, Siheng; Kato, Masato; Wu, Leeju; Theodoropoulos, Pano; Wang, Tao; Kim, Jiwoong; Yun, Jonghyun; Xie, Yang; McKnight, Steven L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">345</span>
        (<span class="NLM_cas:issue">6201</span>),
    <span class="NLM_cas:pages">1139-1145</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Many RNA regulatory proteins controlling pre-mRNA splicing contain serine:arginine (SR) repeats.  Here, we found that these SR domains bound hydrogel droplets composed of fibrous polymers of the low-complexity domain of heterogeneous ribonucleoprotein A2 (hnRNPA2).  Hydrogel binding was reversed upon phosphorylation of the SR domain by CDC2-like kinases 1 and 2 (CLK1/2).  Mutated variants of the SR domains changing serine to glycine (SR-to-GR variants) also bound to hnRNPA2 hydrogels but were not affected by CLK1/2.  When expressed in mammalian cells, these variants bound nucleoli.  The translation products of the sense and antisense transcripts of the expansion repeats assocd. with the C9orf72 gene altered in neurodegenerative disease encode GRn and PRn repeat polypeptides.  Both peptides bound to hnRNPA2 hydrogels independent of CLK1/2 activity.  When applied to cultured cells, both peptides entered cells, migrated to the nucleus, bound nucleoli, and poisoned RNA biogenesis, which caused cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUOC94n_nes7Vg90H21EOLACvtfcHk0liPNzYv_ofbOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtbzM&md5=f7bc9754f14face0bffa45434eeb5025</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1126%2Fscience.1254917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1254917%26sid%3Dliteratum%253Aachs%26aulast%3DKwon%26aufirst%3DI.%26aulast%3DXiang%26aufirst%3DS.%26aulast%3DKato%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DTheodoropoulos%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DYun%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DMcKnight%26aufirst%3DS.%2BL.%26atitle%3DPoly-Dipeptides%2520Encoded%2520by%2520the%2520C9orf72%2520Repeats%2520Bind%2520Nucleoli%252C%2520Impede%2520Rna%2520Biogenesis%252C%2520and%2520Kill%2520Cells%26jtitle%3DScience%26date%3D2014%26volume%3D345%26spage%3D1139%26epage%3D1145%26doi%3D10.1126%2Fscience.1254917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span> <span> </span><span class="NLM_article-title">Bet Inhibitors Rvx-208 and Pfi-1 Reactivate Hiv-1 from Latency</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">16646</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-16816-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41598-017-16816-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=29192216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC1M3ms1Gjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=16646&author=P.+Luauthor=Y.+Shenauthor=H.+Yangauthor=Y.+Wangauthor=Z.+Jiangauthor=X.+Yangauthor=Y.+Zhongauthor=H.+Panauthor=J.+Xuauthor=H.+Luauthor=H.+Zhu&title=Bet+Inhibitors+Rvx-208+and+Pfi-1+Reactivate+Hiv-1+from+Latency&doi=10.1038%2Fs41598-017-16816-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitors RVX-208 and PFI-1 reactivate HIV-1 from latency</span></div><div class="casAuthors">Lu Panpan; Yang He; Wang Yanan; Jiang Zhengtao; Yang Xinyi; Zhong Yangcheng; Pan Hanyu; Zhu Huanzhang; Shen Yinzhong; Xu Jianqing; Lu Hongzhou</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">16646</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Persistent latent reservoir in resting CD4+ T cells is a major obstacle in curing HIV-1 infection.  Effective strategies for eradication of the HIV-1 reservoir are urgently needed.  We report here for the first time that two BET inhibitors, RVX-208, which has entered phase II clinical trials for diverse cardiovascular disorders, and PFI-1, which has been widely studied in oncology, can reactivate HIV-1 from latency.  RVX-208 and PFI-1 treatment alone or in combination with other latency reversing agents efficiently reactivated HIV-1 transcription through an up-regulation of P-TEFb by increasing CDK9 Thr-186 phosphorylation in latently infected Jurkat T cells in vitro.  The two BET inhibitors also reactivated HIV-1 transcription in cART treated patient-derived resting CD4+ T cells ex vivo, without influence on global immune cell activation.  Our findings, in combination with previous reports, further confirm that BET inhibitors are a group of leading compounds for combating HIV-1 latency for viral eradication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTMmRurTV1xdu3MPKNEWB5RfW6udTcc2eaeK2DpGCPBsbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3ms1Gjtw%253D%253D&md5=51309d6a5fefc61001d79cbb93b727ee</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-16816-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-16816-1%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DP.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZhong%26aufirst%3DY.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DH.%26atitle%3DBet%2520Inhibitors%2520Rvx-208%2520and%2520Pfi-1%2520Reactivate%2520Hiv-1%2520from%2520Latency%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D16646%26doi%3D10.1038%2Fs41598-017-16816-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, R. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span> <span> </span><span class="NLM_article-title">An Epigenetic Compound Library Screen Identifies Bet Inhibitors That Promote Hsv-1 and −2 Replication by Bridging P-Tefb to Viral Gene Promoters through Brd4</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e1005950</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1005950</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1371%2Fjournal.ppat.1005950" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=27764245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFCntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&author=K.+Renauthor=W.+Zhangauthor=X.+Chenauthor=Y.+Maauthor=Y.+Daiauthor=Y.+Fanauthor=Y.+Houauthor=R.+X.+Tanauthor=E.+Li&title=An+Epigenetic+Compound+Library+Screen+Identifies+Bet+Inhibitors+That+Promote+Hsv-1+and+%E2%88%922+Replication+by+Bridging+P-Tefb+to+Viral+Gene+Promoters+through+Brd4&doi=10.1371%2Fjournal.ppat.1005950"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">An epigenetic compound library screen identifies BET inhibitors that promote HSV-1 and -2 replication by bridging P-TEFb to viral gene promoters through BRD4</span></div><div class="casAuthors">Ren, Ke; Zhang, Wei; Chen, Xiaoqing; Ma, Yingyu; Dai, Yue; Fan, Yimei; Hou, Yayi; Tan, Ren Xiang; Li, Erguang</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1005950/1-e1005950/23</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The human HSV-1 and -2 are common pathogens of human diseases.  Both host and viral factors are involved in HSV lytic infection, although detailed mechanisms remain elusive.  By screening a chem. library of epigenetic regulation, we identified bromodomain-contg. protein 4 (BRD4) as a crit. player in HSV infection.  We show that treatment with pan BD domain inhibitor enhanced both HSV infection.  Using JQ1 as a probe, we found that JQ1, a defined BD1 inhibitor, acts through BRD4 protein since knockdown of BRD4 expression ablated JQ1 effect on HSV infection.  BRD4 regulates HSV replication through complex formation involving CDK9 and RNAP II; whereas, JQ1 promotes HSV-1 infection by allocating the complex to HSV gene promoters.  Therefore, suppression of BRD4 expression or inhibition of CDK9 activity impeded HSV infection.  Our data support a model that JQ1 enhances HSV infection by switching BRD4 to transcription regulation of viral gene expression from chromatin targeting since transient expression of BRD4 BD1 or BD1/2 domain had similar effect to that by JQ1 treatment.  In addn. to the identification that BRD4 is a modulator for JQ1 action on HSV infection, this study demonstrates BRD4 has an essential role in HSV infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAdy32pDZ3HrVg90H21EOLACvtfcHk0ljf-YsQLp62Ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFCntbk%253D&md5=a2a78d63cb07da2680752067b4e8b5ae</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1005950&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1005950%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DDai%26aufirst%3DY.%26aulast%3DFan%26aufirst%3DY.%26aulast%3DHou%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DR.%2BX.%26aulast%3DLi%26aufirst%3DE.%26atitle%3DAn%2520Epigenetic%2520Compound%2520Library%2520Screen%2520Identifies%2520Bet%2520Inhibitors%2520That%2520Promote%2520Hsv-1%2520and%2520%25E2%2588%25922%2520Replication%2520by%2520Bridging%2520P-Tefb%2520to%2520Viral%2520Gene%2520Promoters%2520through%2520Brd4%26jtitle%3DPLoS%2520Pathog.%26date%3D2016%26volume%3D12%26doi%3D10.1371%2Fjournal.ppat.1005950" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, W.</span></span> <span> </span><span class="NLM_article-title">Bromodomain and Extra-Terminal (Bet) Protein Inhibitors Suppress Chondrocyte Differentiation and Restrain Bone Growth</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">26647</span>– <span class="NLM_lpage">26657</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.749697</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1074%2Fjbc.M116.749697" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=27821592" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFCmurnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=26647-26657&author=N.+Niuauthor=R.+Shaoauthor=G.+Yanauthor=W.+Zou&title=Bromodomain+and+Extra-Terminal+%28Bet%29+Protein+Inhibitors+Suppress+Chondrocyte+Differentiation+and+Restrain+Bone+Growth&doi=10.1074%2Fjbc.M116.749697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain and Extra-terminal (BET) Protein Inhibitors Suppress Chondrocyte Differentiation and Restrain Bone Growth</span></div><div class="casAuthors">Niu, Ningning; Shao, Rui; Yan, Guang; Zou, Weiguo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">26647-26657</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Small mol. inhibitors for bromodomain and extra-terminal (BET) proteins have recently emerged as potential therapeutic agents in clin. trials for various cancers.  However, to date, it is unknown whether these inhibitors have side effects on bone structures.  Here, we report that inhibition of BET bromodomain proteins may suppress chondrocyte differentiation and restrain bone growth.  We generated a luciferase reporter system using the chondrogenic cell line ATDC5 in which the luciferase gene was driven by the promoter of Col2a1, an elementary collagen of the chondrocyte.  The Col2a1-luciferase ATDC5 system was used for rapidly screening both activators and repressors of human collagen Col2a1 gene expression, and we found that BET bromodomain inhibitors reduce the Col2a1-luciferase.  Consistent with the luciferase assay, BET inhibitors decrease the expression of Col2a1.  Furthermore, we constructed a zebrafish line in which the enhanced green fluorescent protein (EGFP) expression was driven by col2a1 promoter.  The transgenic (col2a1-EGFP) zebrafish line demonstrated that BET inhibitors I-BET151 and (+)-JQ1 may affect EGFP expression in zebrafish.  Furthermore, we found that I-BET151 and (+)-JQ1 may affect chondrocyte differentiation in vitro and inhibit zebrafish growth in vivo.  Mechanistic anal. revealed that BET inhibitors influenced the depletion of RNA polymerase II from the Col2a1 promoter.  Collectively, these results suggest that BET bromodomain inhibition may have side effects on skeletal bone structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr97hnBq7K0OrVg90H21EOLACvtfcHk0ljf-YsQLp62Ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFCmurnF&md5=d0518ae2f1e7d71ca1e87148848f8707</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.749697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.749697%26sid%3Dliteratum%253Aachs%26aulast%3DNiu%26aufirst%3DN.%26aulast%3DShao%26aufirst%3DR.%26aulast%3DYan%26aufirst%3DG.%26aulast%3DZou%26aufirst%3DW.%26atitle%3DBromodomain%2520and%2520Extra-Terminal%2520%2528Bet%2529%2520Protein%2520Inhibitors%2520Suppress%2520Chondrocyte%2520Differentiation%2520and%2520Restrain%2520Bone%2520Growth%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D26647%26epage%3D26657%26doi%3D10.1074%2Fjbc.M116.749697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hytti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokarz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maatta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piippo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korhonen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suuronen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honkakoski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaarniranta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahtela-Kakkonen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauppinen, A.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Bet Bromodomains Alleviates Inflammation in Human Rpe Cells</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>110–111</i></span>,  <span class="NLM_fpage">71</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2016.04.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.bcp.2016.04.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=27106081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC28XmsVaqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110%E2%80%93111&publication_year=2016&pages=71-79&author=M.+Hyttiauthor=P.+Tokarzauthor=E.+Maattaauthor=N.+Piippoauthor=E.+Korhonenauthor=T.+Suuronenauthor=P.+Honkakoskiauthor=K.+Kaarnirantaauthor=M.+Lahtela-Kakkonenauthor=A.+Kauppinen&title=Inhibition+of+Bet+Bromodomains+Alleviates+Inflammation+in+Human+Rpe+Cells&doi=10.1016%2Fj.bcp.2016.04.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET bromodomains alleviates inflammation in human RPE cells</span></div><div class="casAuthors">Hytti, M.; Tokarz, P.; Maatta, E.; Piippo, N.; Korhonen, E.; Suuronen, T.; Honkakoski, P.; Kaarniranta, K.; Lahtela-Kakkonen, M.; Kauppinen, A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">110-111</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">71-79</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins are vital for controlling the expression of many pro-inflammatory genes.  Consequently, compds. capable of inhibiting specific bromodomain-facilitated protein-protein interactions would be predicted to alleviate inflammation, making them valuable agents in the treatment of diseases caused by dysregulated inflammation, such as age-related macular degeneration.  Here, the authors assessed the ability of known inhibitors JQ-1, PFI-1, and IBET-151 to protect from the inflammation and cell death caused by etoposide exposure in the human retinal pigment epithelial cell line, ARPE-19.  The potential anti-inflammatory effects of the bromodomain inhibitors were assessed by ELISA profiling.  The involvement of NF-κB and SIRT1 in inflammatory signaling was monitored by ELISA and western blotting.  Furthermore, SIRT1 was knocked down using a specific siRNA or inhibited by EX-527 to elucidate its role in the inflammatory reaction.  The bromodomain inhibitors effectively decreased etoposide-induced release of IL-6 and IL-8.  This anti-inflammatory effect was not related to SIRT1 activity, although all bromodomain inhibitors decreased the extent of acetylation of p53 at the SIRT1 deacetylation site.  Overall, since bromodomain inhibitors display anti-inflammatory properties in human retinal pigment epithelial cells, these compds. may represent a new way of alleviating the inflammation underlying the onset of age-related macular degeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocwtuJwbNv5rVg90H21EOLACvtfcHk0lj_sxsm80XUfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmsVaqsbk%253D&md5=689201c89b6400fe064d3e20ac719bee</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2016.04.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2016.04.009%26sid%3Dliteratum%253Aachs%26aulast%3DHytti%26aufirst%3DM.%26aulast%3DTokarz%26aufirst%3DP.%26aulast%3DMaatta%26aufirst%3DE.%26aulast%3DPiippo%26aufirst%3DN.%26aulast%3DKorhonen%26aufirst%3DE.%26aulast%3DSuuronen%26aufirst%3DT.%26aulast%3DHonkakoski%26aufirst%3DP.%26aulast%3DKaarniranta%26aufirst%3DK.%26aulast%3DLahtela-Kakkonen%26aufirst%3DM.%26aulast%3DKauppinen%26aufirst%3DA.%26atitle%3DInhibition%2520of%2520Bet%2520Bromodomains%2520Alleviates%2520Inflammation%2520in%2520Human%2520Rpe%2520Cells%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2016%26volume%3D110%25E2%2580%2593111%26spage%3D71%26epage%3D79%26doi%3D10.1016%2Fj.bcp.2016.04.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sidrauski, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kampmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hearn, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedantham, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaishankar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sokabe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendez, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newton, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verschueren, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krogan, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renslo, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, P.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Dimerization and Activation of the Exchange Factor Eif2b Antagonizes the Integrated Stress Response</span>. <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">e07314</span> <span class="refDoi"> DOI: 10.7554/eLife.07314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.7554%2FeLife.07314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25875391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC28XltVylsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2015&author=C.+Sidrauskiauthor=J.+C.+Tsaiauthor=M.+Kampmannauthor=B.+R.+Hearnauthor=P.+Vedanthamauthor=P.+Jaishankarauthor=M.+Sokabeauthor=A.+S.+Mendezauthor=B.+W.+Newtonauthor=E.+L.+Tangauthor=E.+Verschuerenauthor=J.+R.+Johnsonauthor=N.+J.+Kroganauthor=C.+S.+Fraserauthor=J.+S.+Weissmanauthor=A.+R.+Rensloauthor=P.+Walter&title=Pharmacological+Dimerization+and+Activation+of+the+Exchange+Factor+Eif2b+Antagonizes+the+Integrated+Stress+Response&doi=10.7554%2FeLife.07314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological dimerization and activation of the exchange factor eIF2B antagonizes the integrated stress response</span></div><div class="casAuthors">Sidrauski, Carmela; Tsai, Jordan C.; Kampmann, Martin; Hearn, Brian R.; Vedantham, Punitha; Jaishankar, Priyadarshini; Sokabe, Masaaki; Mendez, Aaron S.; Newton, Billy W.; Tang, Edward L.; Verschueren, Erik; Johnson, Jeffrey R.; Krogan, Nevan J.; Fraser, Christopher S.; Weissman, Jonathan S.; Renslo, Adam R.; Walter, Peter</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e07314/1-e07314/27</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">The general translation initiation factor eIF2 is a major translational control point.  Multiple signaling pathways in the integrated stress response phosphorylate eIF2 serine-51, inhibiting nucleotide exchange by eIF2B.  ISRIB, a potent drug-like small mol., renders cells insensitive to eIF2α phosphorylation and enhances cognitive function in rodents by blocking long-term depression.  ISRIB was identified in a phenotypic cell-based screen, and its mechanism of action remained unknown.  We now report that ISRIB is an activator of eIF2B.  Our reporter-based shRNA screen revealed an eIF2B requirement for ISRIB activity.  Our results define ISRIB as a sym. mol., show ISRIB-mediated stabilization of activated eIF2B dimers, and suggest that eIF2B4 (δ-subunit) contributes to the ISRIB binding site.  We also developed new ISRIB analogs, improving its EC50 to 600 pM in cell culture.  By modulating eIF2B function, ISRIB promises to be an invaluable tool in proof-of-principle studies aiming to ameliorate cognitive defects resulting from neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrw4eI3eO6AbVg90H21EOLACvtfcHk0lj_sxsm80XUfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltVylsLY%253D&md5=35563489035c0a80eb4e1e8046e7c28a</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.7554%2FeLife.07314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.07314%26sid%3Dliteratum%253Aachs%26aulast%3DSidrauski%26aufirst%3DC.%26aulast%3DTsai%26aufirst%3DJ.%2BC.%26aulast%3DKampmann%26aufirst%3DM.%26aulast%3DHearn%26aufirst%3DB.%2BR.%26aulast%3DVedantham%26aufirst%3DP.%26aulast%3DJaishankar%26aufirst%3DP.%26aulast%3DSokabe%26aufirst%3DM.%26aulast%3DMendez%26aufirst%3DA.%2BS.%26aulast%3DNewton%26aufirst%3DB.%2BW.%26aulast%3DTang%26aufirst%3DE.%2BL.%26aulast%3DVerschueren%26aufirst%3DE.%26aulast%3DJohnson%26aufirst%3DJ.%2BR.%26aulast%3DKrogan%26aufirst%3DN.%2BJ.%26aulast%3DFraser%26aufirst%3DC.%2BS.%26aulast%3DWeissman%26aufirst%3DJ.%2BS.%26aulast%3DRenslo%26aufirst%3DA.%2BR.%26aulast%3DWalter%26aufirst%3DP.%26atitle%3DPharmacological%2520Dimerization%2520and%2520Activation%2520of%2520the%2520Exchange%2520Factor%2520Eif2b%2520Antagonizes%2520the%2520Integrated%2520Stress%2520Response%26jtitle%3DElife%26date%3D2015%26volume%3D4%26doi%3D10.7554%2FeLife.07314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeBon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sidrauski, C.</span></span> <span> </span><span class="NLM_article-title">The Small Molecule Isrib Rescues the Stability and Activity of Vanishing White Matter Disease Eif2b Mutant Complexes</span>. <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e32733</span>, <span class="refDoi"> DOI: 10.7554/eLife.32733</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.7554%2FeLife.32733" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=29489452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVSjtLfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=e32733&author=Y.+L.+Wongauthor=L.+LeBonauthor=R.+Edaljiauthor=H.+B.+Limauthor=C.+Sunauthor=C.+Sidrauski&title=The+Small+Molecule+Isrib+Rescues+the+Stability+and+Activity+of+Vanishing+White+Matter+Disease+Eif2b+Mutant+Complexes&doi=10.7554%2FeLife.32733"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">The small molecule ISRIB rescues the stability and activity of vanishing white matter disease eIF2B mutant complexes</span></div><div class="casAuthors">Wong, Yao Liang; LeBon, Lauren; Edalji, Rohinton; Ben Lim, Hock; Sun, Chaohong; Sidrauski, Carmela</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e32733/1-e32733/23</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">EIF2B is a dedicated guanine nucleotide exchange factor for eIF2, the GTPase that is essential to initiate mRNA translation.  The integrated stress response (ISR) signaling pathway inhibits eIF2B activity, attenuates global protein synthesis and upregulates a set of stress-response proteins.  Partial loss-of-function mutations in eIF2B cause a neurodegenerative disorder called Vanishing White Matter Disease (VWMD).  Previously, we showed that the small mol. ISRIB is a specific activator of eIF2B.  Here, we report that various VWMD mutations destabilize the decameric eIF2B holoenzyme and impair its enzymic activity.  ISRIB stabilizes VWMD mutant eIF2B in the decameric form and restores the residual catalytic activity to wild-type levels.  Moreover, ISRIB blocks activation of the ISR in cells carrying these mutations.  As such, ISRIB promises to be an invaluable tool in proof-of-concept studies aiming to ameliorate defects resulting from inappropriate or pathol. activation of the ISR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkRGD2klbIorVg90H21EOLACvtfcHk0liN9UO_fgmKYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVSjtLfL&md5=7d897c1e9bd512843ee90aedb4babf89</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.7554%2FeLife.32733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.32733%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DY.%2BL.%26aulast%3DLeBon%26aufirst%3DL.%26aulast%3DEdalji%26aufirst%3DR.%26aulast%3DLim%26aufirst%3DH.%2BB.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DSidrauski%26aufirst%3DC.%26atitle%3DThe%2520Small%2520Molecule%2520Isrib%2520Rescues%2520the%2520Stability%2520and%2520Activity%2520of%2520Vanishing%2520White%2520Matter%2520Disease%2520Eif2b%2520Mutant%2520Complexes%26jtitle%3DElife%26date%3D2018%26volume%3D7%26spage%3De32733%26doi%3D10.7554%2FeLife.32733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Palacino, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swalley, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Hoosear, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beibel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renaud, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salcius, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servais, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLellan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuierer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blommers, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaustein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berenshteyn, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacoste, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roma, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaud, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishman, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dales, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivasankaran, R.</span></span> <span> </span><span class="NLM_article-title">Smn2 Splice Modulators Enhance U1-Pre-Mrna Association and Rescue Sma Mice</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">511</span>– <span class="NLM_lpage">517</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnchembio.1837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26030728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=511-517&author=J.+Palacinoauthor=S.+E.+Swalleyauthor=C.+Songauthor=A.+K.+Cheungauthor=L.+Shuauthor=X.+Zhangauthor=M.+Van+Hoosearauthor=Y.+Shinauthor=D.+N.+Chinauthor=C.+G.+Kellerauthor=M.+Beibelauthor=N.+A.+Renaudauthor=T.+M.+Smithauthor=M.+Salciusauthor=X.+Shiauthor=M.+Hildauthor=R.+Servaisauthor=M.+Jainauthor=L.+Dengauthor=C.+Bullockauthor=M.+McLellanauthor=S.+Schuiererauthor=L.+Murphyauthor=M.+J.+Blommersauthor=C.+Blausteinauthor=F.+Berenshteynauthor=A.+Lacosteauthor=J.+R.+Thomasauthor=G.+Romaauthor=G.+A.+Michaudauthor=B.+S.+Tsengauthor=J.+A.+Porterauthor=V.+E.+Myerauthor=J.+A.+Tallaricoauthor=L.+G.+Hamannauthor=D.+Curtisauthor=M.+C.+Fishmanauthor=W.+F.+Dietrichauthor=N.+A.+Dalesauthor=R.+Sivasankaran&title=Smn2+Splice+Modulators+Enhance+U1-Pre-Mrna+Association+and+Rescue+Sma+Mice&doi=10.1038%2Fnchembio.1837"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">SMN2 splice modulators enhance U1-pre-mRNA association and rescue SMA mice</span></div><div class="casAuthors">Palacino, James; Swalley, Susanne E.; Song, Cheng; Cheung, Atwood K.; Shu, Lei; Zhang, Xiaolu; Van Hoosear, Mailin; Shin, Youngah; Chin, Donovan N.; Keller, Caroline Gubser; Beibel, Martin; Renaud, Nicole A.; Smith, Thomas M.; Salcius, Michael; Shi, Xiaoying; Hild, Marc; Servais, Rebecca; Jain, Monish; Deng, Lin; Bullock, Caroline; McLellan, Michael; Schuierer, Sven; Murphy, Leo; Blommers, Marcel J. J.; Blaustein, Cecile; Berenshteyn, Frada; Lacoste, Arnaud; Thomas, Jason R.; Roma, Guglielmo; Michaud, Gregory A.; Tseng, Brian S.; Porter, Jeffery A.; Myer, Vic E.; Tallarico, John A.; Hamann, Lawrence G.; Curtis, Daniel; Fishman, Mark C.; Dietrich, William F.; Dales, Natalie A.; Sivasankaran, Rajeev</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">511-517</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Spinal muscular atrophy (SMA), which results from the loss of expression of the survival of motor neuron-1 (SMN1) gene, represents the most common genetic cause of pediatric mortality.  A duplicate copy (SMN2) is inefficiently spliced, producing a truncated and unstable protein.  We describe herein a potent, orally active, small-mol. enhancer of SMN2 splicing that elevates full-length SMN protein and extends survival in a severe SMA mouse model.  We demonstrate that the mol. mechanism of action is via stabilization of the transient double-strand RNA structure formed by the SMN2 pre-mRNA and U1 small nuclear ribonucleic protein (snRNP) complex.  The binding affinity of U1 snRNP to the 5' splice site is increased in a sequence-selective manner, discrete from constitutive recognition.  This new mechanism demonstrates the feasibility of small mol.-mediated, sequence-selective splice modulation and the potential for leveraging this strategy in other splicing diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdMqLIC0lw17Vg90H21EOLACvtfcHk0liN9UO_fgmKYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7zN&md5=15e56c8b3f48feffa150e975a81633a0</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1837%26sid%3Dliteratum%253Aachs%26aulast%3DPalacino%26aufirst%3DJ.%26aulast%3DSwalley%26aufirst%3DS.%2BE.%26aulast%3DSong%26aufirst%3DC.%26aulast%3DCheung%26aufirst%3DA.%2BK.%26aulast%3DShu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DVan%2BHoosear%26aufirst%3DM.%26aulast%3DShin%26aufirst%3DY.%26aulast%3DChin%26aufirst%3DD.%2BN.%26aulast%3DKeller%26aufirst%3DC.%2BG.%26aulast%3DBeibel%26aufirst%3DM.%26aulast%3DRenaud%26aufirst%3DN.%2BA.%26aulast%3DSmith%26aufirst%3DT.%2BM.%26aulast%3DSalcius%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DServais%26aufirst%3DR.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DBullock%26aufirst%3DC.%26aulast%3DMcLellan%26aufirst%3DM.%26aulast%3DSchuierer%26aufirst%3DS.%26aulast%3DMurphy%26aufirst%3DL.%26aulast%3DBlommers%26aufirst%3DM.%2BJ.%26aulast%3DBlaustein%26aufirst%3DC.%26aulast%3DBerenshteyn%26aufirst%3DF.%26aulast%3DLacoste%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DJ.%2BR.%26aulast%3DRoma%26aufirst%3DG.%26aulast%3DMichaud%26aufirst%3DG.%2BA.%26aulast%3DTseng%26aufirst%3DB.%2BS.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DTallarico%26aufirst%3DJ.%2BA.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DCurtis%26aufirst%3DD.%26aulast%3DFishman%26aufirst%3DM.%2BC.%26aulast%3DDietrich%26aufirst%3DW.%2BF.%26aulast%3DDales%26aufirst%3DN.%2BA.%26aulast%3DSivasankaran%26aufirst%3DR.%26atitle%3DSmn2%2520Splice%2520Modulators%2520Enhance%2520U1-Pre-Mrna%2520Association%2520and%2520Rescue%2520Sma%2520Mice%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D511%26epage%3D517%26doi%3D10.1038%2Fnchembio.1837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, C. X.</span></span> <span> </span><span class="NLM_article-title">Tau and Neurodegenerative Disease: The Story So Far</span>. <i>Nat. Rev. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1038/nrneurol.2015.225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnrneurol.2015.225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26635213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVSiurvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=15-27&author=K.+Iqbalauthor=F.+Liuauthor=C.+X.+Gong&title=Tau+and+Neurodegenerative+Disease%3A+The+Story+So+Far&doi=10.1038%2Fnrneurol.2015.225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Tau and neurodegenerative disease: the story so far</span></div><div class="casAuthors">Iqbal, Khalid; Liu, Fei; Gong, Cheng-Xin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Neurology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-27</span>CODEN:
                <span class="NLM_cas:coden">NRNACP</span>;
        ISSN:<span class="NLM_cas:issn">1759-4758</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In 1975, tau protein was isolated as a microtubule-assocd. factor from the porcine brain.  In the previous year, a paired helical filament (PHF) protein had been identified in neurofibrillary tangles in the brains of individuals with Alzheimer disease (AD), but it was not until 1986 that the PHF protein and tau were discovered to be one and the same.  In the AD brain, tau was found to be abnormally hyperphosphorylated, and it inhibited rather than promoted in vitro microtubule assembly.  Almost 80 disease-causing exonic missense and intronic silent mutations in the tau gene have been found in familial cases of frontotemporal dementia but, to date, no such mutation has been found in AD.  The first phase I clin. trial of an active tau immunization vaccine in patients with AD was recently completed.  Assays for tau levels in cerebrospinal fluid and plasma are now available, and tau radiotracers for PET are under development.  In this article, we provide an overview of the pivotal discoveries in the tau research field over the past 40 years.  We also review the current status of the field, including disease mechanisms and therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlP4cbMS6zdLVg90H21EOLACvtfcHk0lhWwn4UAA_ZEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVSiurvL&md5=94154d6c70f398a683821fe31ad3660e</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1038%2Fnrneurol.2015.225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrneurol.2015.225%26sid%3Dliteratum%253Aachs%26aulast%3DIqbal%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DGong%26aufirst%3DC.%2BX.%26atitle%3DTau%2520and%2520Neurodegenerative%2520Disease%253A%2520The%2520Story%2520So%2520Far%26jtitle%3DNat.%2520Rev.%2520Neurol.%26date%3D2016%26volume%3D12%26spage%3D15%26epage%3D27%26doi%3D10.1038%2Fnrneurol.2015.225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eglen, R. M.</span></span> <span> </span><span class="NLM_article-title">Three-Dimensional Cell Cultures in Drug Discovery and Development</span>. <i>SLAS Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">456</span>– <span class="NLM_lpage">472</span>, <span class="refDoi"> DOI: 10.1177/1087057117696795</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1177%2F1087057117696795" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28520521" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKktrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=456-472&author=Y.+Fangauthor=R.+M.+Eglen&title=Three-Dimensional+Cell+Cultures+in+Drug+Discovery+and+Development&doi=10.1177%2F1087057117696795"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Three-Dimensional Cell Cultures in Drug Discovery and Development</span></div><div class="casAuthors">Fang, Ye; Eglen, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">456-472</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  The past decades have witnessed significant efforts toward the development of three-dimensional (3D) cell cultures as systems that better mimic in vivo physiol.  Today, 3D cell cultures are emerging, not only as a new tool in early drug discovery but also as potential therapeutics to treat disease.  In this review, we assess leading 3D cell culture technologies and their impact on drug discovery, including spheroids, organoids, scaffolds, hydrogels, organs-on-chips, and 3D bioprinting.  We also discuss the implementation of these technologies in compd. identification, screening, and development, ranging from disease modeling to assessment of efficacy and safety profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8_ypNaFyNyLVg90H21EOLACvtfcHk0lhWwn4UAA_ZEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKktrg%253D&md5=1f8a292f0f288fca4fd5d4c2909e58fc</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1177%2F1087057117696795&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057117696795%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DY.%26aulast%3DEglen%26aufirst%3DR.%2BM.%26atitle%3DThree-Dimensional%2520Cell%2520Cultures%2520in%2520Drug%2520Discovery%2520and%2520Development%26jtitle%3DSLAS%2520Discovery%26date%3D2017%26volume%3D22%26spage%3D456%26epage%3D472%26doi%3D10.1177%2F1087057117696795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetzel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marriott, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dreau, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Avanzo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanzi, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H.</span></span> <span> </span><span class="NLM_article-title">A 3d Human Triculture System Modeling Neurodegeneration and Neuroinflammation in Alzheimer’s Disease</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">951</span>, <span class="refDoi"> DOI: 10.1038/s41593-018-0175-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41593-018-0175-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=29950669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1eisrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2018&pages=941-951&author=J.+Parkauthor=I.+Wetzelauthor=I.+Marriottauthor=D.+Dreauauthor=C.+D%E2%80%99Avanzoauthor=D.+Y.+Kimauthor=R.+E.+Tanziauthor=H.+Cho&title=A+3d+Human+Triculture+System+Modeling+Neurodegeneration+and+Neuroinflammation+in+Alzheimer%E2%80%99s+Disease&doi=10.1038%2Fs41593-018-0175-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">A 3D human triculture system modeling neurodegeneration and neuroinflammation in Alzheimer's disease</span></div><div class="casAuthors">Park, Joseph; Wetzel, Isaac; Marriott, Ian; Dreau, Didier; D'Avanzo, Carla; Kim, Doo Yeon; Tanzi, Rudolph E.; Cho, Hansang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-951</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Alzheimer's disease (AD) is characterized by beta-amyloid accumulation, phosphorylated tau formation, hyperactivation of glial cells, and neuronal loss.  The mechanisms of AD pathogenesis, however, remain poorly understood, partially due to the lack of relevant models that can comprehensively recapitulate multistage intercellular interactions in human AD brains.  Here we present a new three-dimensional (3D) human AD triculture model using neurons, astrocytes, and microglia in a 3D microfluidic platform.  Our model provided key representative AD features: beta-amyloid aggregation, phosphorylated tau accumulation, and neuroinflammatory activity.  In particular, the model mirrored microglial recruitment, neurotoxic activities such as axonal cleavage, and NO release damaging AD neurons and astrocytes.  Our model will serve to facilitate the development of more precise human brain models for basic mechanistic studies in neural-glial interactions and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW5hk7KxWJFLVg90H21EOLACvtfcHk0lhtkJKNr7zTZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1eisrrP&md5=681b07976a82487fb1f3bc8e7867417b</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1038%2Fs41593-018-0175-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41593-018-0175-4%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DJ.%26aulast%3DWetzel%26aufirst%3DI.%26aulast%3DMarriott%26aufirst%3DI.%26aulast%3DDreau%26aufirst%3DD.%26aulast%3DD%25E2%2580%2599Avanzo%26aufirst%3DC.%26aulast%3DKim%26aufirst%3DD.%2BY.%26aulast%3DTanzi%26aufirst%3DR.%2BE.%26aulast%3DCho%26aufirst%3DH.%26atitle%3DA%25203d%2520Human%2520Triculture%2520System%2520Modeling%2520Neurodegeneration%2520and%2520Neuroinflammation%2520in%2520Alzheimer%25E2%2580%2599s%2520Disease%26jtitle%3DNat.%2520Neurosci.%26date%3D2018%26volume%3D21%26spage%3D941%26epage%3D951%26doi%3D10.1038%2Fs41593-018-0175-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monzel, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smits, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemmer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hachi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Wuellen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarazo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruggemann, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boussaad, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, R. M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwamborn, J. C.</span></span> <span> </span><span class="NLM_article-title">Derivation of Human Midbrain-Specific Organoids from Neuroepithelial Stem Cells</span>. <i>Stem Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1144</span>– <span class="NLM_lpage">1154</span>, <span class="refDoi"> DOI: 10.1016/j.stemcr.2017.03.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.stemcr.2017.03.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28416282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmt12rurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1144-1154&author=A.+S.+Monzelauthor=L.+M.+Smitsauthor=K.+Hemmerauthor=S.+Hachiauthor=E.+L.+Morenoauthor=T.+van+Wuellenauthor=J.+Jarazoauthor=J.+Walterauthor=I.+Bruggemannauthor=I.+Boussaadauthor=E.+Bergerauthor=R.+M.+T.+Flemingauthor=S.+Bologninauthor=J.+C.+Schwamborn&title=Derivation+of+Human+Midbrain-Specific+Organoids+from+Neuroepithelial+Stem+Cells&doi=10.1016%2Fj.stemcr.2017.03.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Derivation of Human Midbrain-Specific Organoids from Neuroepithelial Stem Cells</span></div><div class="casAuthors">Monzel, Anna S.; Smits, Lisa M.; Hemmer, Kathrin; Hachi, Siham; Moreno, Edinson Lucumi; van Wuellen, Thea; Jarazo, Javier; Walter, Jonas; Bruggemann, Inga; Boussaad, Ibrahim; Berger, Emanuel; Fleming, Ronan M. T.; Bolognin, Silvia; Schwamborn, Jens C.</div><div class="citationInfo"><span class="NLM_cas:title">Stem Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1144-1154</span>CODEN:
                <span class="NLM_cas:coden">SCRTFL</span>;
        ISSN:<span class="NLM_cas:issn">2213-6711</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Research on human brain development and neurol. diseases is limited by the lack of advanced exptl. in vitro models that truly recapitulate the complexity of the human brain.  Here, we describe a robust human brain organoid system that is highly specific to the midbrain derived from regionally patterned neuroepithelial stem cells.  These human midbrain organoids contain spatially organized groups of dopaminergic neurons, which make them an attractive model for the study of Parkinson's disease.  Midbrain organoids are characterized in detail for neuronal, astroglial, and oligodendrocyte differentiation.  Furthermore, we show the presence of synaptic connections and electrophysiol. activity.  The complexity of this model is further highlighted by the myelination of neurites.  The present midbrain organoid system has the potential to be used for advanced in vitro disease modeling and therapy development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXJwLKJ8guk7Vg90H21EOLACvtfcHk0lhtkJKNr7zTZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmt12rurk%253D&md5=dc5e89c31a678846fa536b8bef5091b1</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1016%2Fj.stemcr.2017.03.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.stemcr.2017.03.010%26sid%3Dliteratum%253Aachs%26aulast%3DMonzel%26aufirst%3DA.%2BS.%26aulast%3DSmits%26aufirst%3DL.%2BM.%26aulast%3DHemmer%26aufirst%3DK.%26aulast%3DHachi%26aufirst%3DS.%26aulast%3DMoreno%26aufirst%3DE.%2BL.%26aulast%3Dvan%2BWuellen%26aufirst%3DT.%26aulast%3DJarazo%26aufirst%3DJ.%26aulast%3DWalter%26aufirst%3DJ.%26aulast%3DBruggemann%26aufirst%3DI.%26aulast%3DBoussaad%26aufirst%3DI.%26aulast%3DBerger%26aufirst%3DE.%26aulast%3DFleming%26aufirst%3DR.%2BM.%2BT.%26aulast%3DBolognin%26aufirst%3DS.%26aulast%3DSchwamborn%26aufirst%3DJ.%2BC.%26atitle%3DDerivation%2520of%2520Human%2520Midbrain-Specific%2520Organoids%2520from%2520Neuroepithelial%2520Stem%2520Cells%26jtitle%3DStem%2520Cell%2520Rep.%26date%3D2017%26volume%3D8%26spage%3D1144%26epage%3D1154%26doi%3D10.1016%2Fj.stemcr.2017.03.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Birey, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makinson, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Islam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzler, K. R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panagiotakos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thom, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Rourke, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinmetz, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallmayer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huguenard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasca, S. P.</span></span> <span> </span><span class="NLM_article-title">Assembly of Functionally Integrated Human Forebrain Spheroids</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>545</i></span>,  <span class="NLM_fpage">54</span>– <span class="NLM_lpage">59</span>, <span class="refDoi"> DOI: 10.1038/nature22330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnature22330" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28445465" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmslehs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=545&publication_year=2017&pages=54-59&author=F.+Bireyauthor=J.+Andersenauthor=C.+D.+Makinsonauthor=S.+Islamauthor=W.+Weiauthor=N.+Huberauthor=H.+C.+Fanauthor=K.+R.+C.+Metzlerauthor=G.+Panagiotakosauthor=N.+Thomauthor=N.+A.+O%E2%80%99Rourkeauthor=L.+M.+Steinmetzauthor=J.+A.+Bernsteinauthor=J.+Hallmayerauthor=J.+R.+Huguenardauthor=S.+P.+Pasca&title=Assembly+of+Functionally+Integrated+Human+Forebrain+Spheroids&doi=10.1038%2Fnature22330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Assembly of functionally integrated human forebrain spheroids</span></div><div class="casAuthors">Birey, Fikri; Andersen, Jimena; Makinson, Christopher D.; Islam, Saiful; Wei, Wu; Huber, Nina; Fan, H. Christina; Metzler, Kimberly R. Cordes; Panagiotakos, Georgia; Thom, Nicholas; O'Rourke, Nancy A.; Steinmetz, Lars M.; Bernstein, Jonathan A.; Hallmayer, Joachim; Huguenard, John R.; Pasca, Sergiu P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">545</span>
        (<span class="NLM_cas:issue">7652</span>),
    <span class="NLM_cas:pages">54-59</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The development of the nervous system involves a coordinated succession of events including the migration of GABAergic (γ-aminobutyric-acid-releasing) neurons from ventral to dorsal forebrain and their integration into cortical circuits.  However, these interregional interactions have not yet been modelled with human cells.  Here we generate three-dimensional spheroids from human pluripotent stem cells that resemble either the dorsal or ventral forebrain and contain cortical glutamatergic or GABAergic neurons.  These subdomain-specific forebrain spheroids can be assembled in vitro to recapitulate the saltatory migration of interneurons obsd. in the fetal forebrain.  Using this system, we find that in Timothy syndrome - a neurodevelopmental disorder that is caused by mutations in the CaV1.2 calcium channel - interneurons display abnormal migratory saltations.  We also show that after migration, interneurons functionally integrate with glutamatergic neurons to form a microphysiol. system.  We anticipate that this approach will be useful for studying neural development and disease, and for deriving spheroids that resemble other brain regions to assemble circuits in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-CnPnrt7jDrVg90H21EOLACvtfcHk0lhtkJKNr7zTZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmslehs70%253D&md5=60fb5a24730aef9ad1c3fd5e6a1f2336</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1038%2Fnature22330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22330%26sid%3Dliteratum%253Aachs%26aulast%3DBirey%26aufirst%3DF.%26aulast%3DAndersen%26aufirst%3DJ.%26aulast%3DMakinson%26aufirst%3DC.%2BD.%26aulast%3DIslam%26aufirst%3DS.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DHuber%26aufirst%3DN.%26aulast%3DFan%26aufirst%3DH.%2BC.%26aulast%3DMetzler%26aufirst%3DK.%2BR.%2BC.%26aulast%3DPanagiotakos%26aufirst%3DG.%26aulast%3DThom%26aufirst%3DN.%26aulast%3DO%25E2%2580%2599Rourke%26aufirst%3DN.%2BA.%26aulast%3DSteinmetz%26aufirst%3DL.%2BM.%26aulast%3DBernstein%26aufirst%3DJ.%2BA.%26aulast%3DHallmayer%26aufirst%3DJ.%26aulast%3DHuguenard%26aufirst%3DJ.%2BR.%26aulast%3DPasca%26aufirst%3DS.%2BP.%26atitle%3DAssembly%2520of%2520Functionally%2520Integrated%2520Human%2520Forebrain%2520Spheroids%26jtitle%3DNature%26date%3D2017%26volume%3D545%26spage%3D54%26epage%3D59%26doi%3D10.1038%2Fnature22330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadoshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakaguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ando, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eiraku, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasai, Y.</span></span> <span> </span><span class="NLM_article-title">Self-Organization of Axial Polarity, inside-out Layer Pattern, and Species-Specific Progenitor Dynamics in Human Es Cell-Derived Neocortex</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">20284</span>– <span class="NLM_lpage">20289</span>, <span class="refDoi"> DOI: 10.1073/pnas.1315710110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1073%2Fpnas.1315710110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=24277810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKmsLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=20284-20289&author=T.+Kadoshimaauthor=H.+Sakaguchiauthor=T.+Nakanoauthor=M.+Soenauthor=S.+Andoauthor=M.+Eirakuauthor=Y.+Sasai&title=Self-Organization+of+Axial+Polarity%2C+inside-out+Layer+Pattern%2C+and+Species-Specific+Progenitor+Dynamics+in+Human+Es+Cell-Derived+Neocortex&doi=10.1073%2Fpnas.1315710110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Self-organization of axial polarity, inside-out layer pattern, and species-specific progenitor dynamics in human ES cell-derived neocortex</span></div><div class="casAuthors">Kadoshima, Taisuke; Sakaguchi, Hideya; Nakano, Tokushige; Soen, Mika; Ando, Satoshi; Eiraku, Mototsugu; Sasai, Yoshiki</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">20284-20289,S20284/1-S20284/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Here, using further optimized 3D culture that allows highly selective induction and long-term growth of human ES cell (hESC)-derived cortical neuroepithelium, we demonstrate unique aspects of self-organization in human neocorticogenesis.  Self-organized cortical tissue spontaneously forms a polarity along the dorsocaudal-ventrorostral axis and undergoes region-specific rolling morphogenesis that generates a semispherical structure.  The neuroepithelium self-forms a multilayered structure including three neuronal zones (subplate, cortical plate, and Cajal-Retzius cell zones) and three progenitor zones (ventricular, subventricular, and intermediate zones) in the same apical-basal order as seen in the human fetal cortex in the early second trimester.  In the cortical plate, late-born neurons tend to localize more basally to early-born neurons, consistent with the inside-out pattern seen in vivo.  Furthermore, the outer subventricular zone contains basal progenitors that share characteristics with outer radial glia abundantly found in the human, but not mouse, fetal brain.  Thus, human neocorticogenesis involves intrinsic programs that enable the emergence of complex neocortical features.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIJ-H67pTIwLVg90H21EOLACvtfcHk0li0LVnspeCNUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKmsLzL&md5=c71a7e0b4a1d8fa4da52771ec449d1e6</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1315710110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1315710110%26sid%3Dliteratum%253Aachs%26aulast%3DKadoshima%26aufirst%3DT.%26aulast%3DSakaguchi%26aufirst%3DH.%26aulast%3DNakano%26aufirst%3DT.%26aulast%3DSoen%26aufirst%3DM.%26aulast%3DAndo%26aufirst%3DS.%26aulast%3DEiraku%26aufirst%3DM.%26aulast%3DSasai%26aufirst%3DY.%26atitle%3DSelf-Organization%2520of%2520Axial%2520Polarity%252C%2520inside-out%2520Layer%2520Pattern%252C%2520and%2520Species-Specific%2520Progenitor%2520Dynamics%2520in%2520Human%2520Es%2520Cell-Derived%2520Neocortex%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D20284%26epage%3D20289%26doi%3D10.1073%2Fpnas.1315710110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lancaster, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenzel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bicknell, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurles, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homfray, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penninger, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoblich, J. A.</span></span> <span> </span><span class="NLM_article-title">Cerebral Organoids Model Human Brain Development and Microcephaly</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>501</i></span>,  <span class="NLM_fpage">373</span>– <span class="NLM_lpage">379</span>, <span class="refDoi"> DOI: 10.1038/nature12517</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnature12517" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23995685" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlCntb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=501&publication_year=2013&pages=373-379&author=M.+A.+Lancasterauthor=M.+Rennerauthor=C.+A.+Martinauthor=D.+Wenzelauthor=L.+S.+Bicknellauthor=M.+E.+Hurlesauthor=T.+Homfrayauthor=J.+M.+Penningerauthor=A.+P.+Jacksonauthor=J.+A.+Knoblich&title=Cerebral+Organoids+Model+Human+Brain+Development+and+Microcephaly&doi=10.1038%2Fnature12517"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Cerebral organoids model human brain development and microcephaly</span></div><div class="casAuthors">Lancaster, Madeline A.; Renner, Magdalena; Martin, Carol-Anne; Wenzel, Daniel; Bicknell, Louise S.; Hurles, Matthew E.; Homfray, Tessa; Penninger, Josef M.; Jackson, Andrew P.; Knoblich, Juergen A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">501</span>
        (<span class="NLM_cas:issue">7467</span>),
    <span class="NLM_cas:pages">373-379</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The complexity of the human brain has made it difficult to study many brain disorders in model organisms, highlighting the need for an in vitro model of human brain development.  Here we have developed a human pluripotent stem cell-derived three-dimensional organoid culture system, termed cerebral organoids, that develop various discrete, although interdependent, brain regions.  These include a cerebral cortex contg. progenitor populations that organize and produce mature cortical neuron subtypes.  Furthermore, cerebral organoids are shown to recapitulate features of human cortical development, namely characteristic progenitor zone organization with abundant outer radial glial stem cells.  Finally, we use RNA interference and patient-specific induced pluripotent stem cells to model microcephaly, a disorder that has been difficult to recapitulate in mice.  We demonstrate premature neuronal differentiation in patient organoids, a defect that could help to explain the disease phenotype.  Together, these data show that three-dimensional organoids can recapitulate development and disease even in this most complex human tissue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1ooY8YMg5k7Vg90H21EOLACvtfcHk0li0LVnspeCNUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlCntb3F&md5=96195248e51aa62a5a5d761aa56814a1</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1038%2Fnature12517&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12517%26sid%3Dliteratum%253Aachs%26aulast%3DLancaster%26aufirst%3DM.%2BA.%26aulast%3DRenner%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DC.%2BA.%26aulast%3DWenzel%26aufirst%3DD.%26aulast%3DBicknell%26aufirst%3DL.%2BS.%26aulast%3DHurles%26aufirst%3DM.%2BE.%26aulast%3DHomfray%26aufirst%3DT.%26aulast%3DPenninger%26aufirst%3DJ.%2BM.%26aulast%3DJackson%26aufirst%3DA.%2BP.%26aulast%3DKnoblich%26aufirst%3DJ.%2BA.%26atitle%3DCerebral%2520Organoids%2520Model%2520Human%2520Brain%2520Development%2520and%2520Microcephaly%26jtitle%3DNature%26date%3D2013%26volume%3D501%26spage%3D373%26epage%3D379%26doi%3D10.1038%2Fnature12517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armijo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bravo-Alegria, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becerra-Calixto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mays, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soto, C.</span></span> <span> </span><span class="NLM_article-title">Modeling Amyloid Beta and Tau Pathology in Human Cerebral Organoids</span>. <i>Mol. Psychiatry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2363</span>– <span class="NLM_lpage">2374</span>, <span class="refDoi"> DOI: 10.1038/s41380-018-0229-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41380-018-0229-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30171212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1GltLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=2363-2374&author=C.+Gonzalezauthor=E.+Armijoauthor=J.+Bravo-Alegriaauthor=A.+Becerra-Calixtoauthor=C.+E.+Maysauthor=C.+Soto&title=Modeling+Amyloid+Beta+and+Tau+Pathology+in+Human+Cerebral+Organoids&doi=10.1038%2Fs41380-018-0229-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling amyloid beta and tau pathology in human cerebral organoids</span></div><div class="casAuthors">Gonzalez, Cesar; Armijo, Enrique; Bravo-Alegria, Javiera; Becerra-Calixto, Andrea; Mays, Charles E.; Soto, Claudio</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Psychiatry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2363-2374</span>CODEN:
                <span class="NLM_cas:coden">MOPSFQ</span>;
        ISSN:<span class="NLM_cas:issn">1359-4184</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The typical abnormalities obsd. in the brain of Alzheimer's disease (AD) patients include synaptic alterations, neuronal death, brain inflammation, and the accumulation of protein aggregates in the form of amyloid plaques and neurofibrillary tangles.  Despite the development of many animal and in vitro models for AD, there is a lack of an exptl. approach that fully recapitulates essential aspects of the disease in human cells.  Here, we report the generation of a new model to study AD, consisting of cerebral organoids (COs) produced from human-induced pluripotent stem cells (iPSCs).  Under our exptl. conditions, COs grow to form three-dimensional (3D) structures contg. neural areas with cortical-like organization.  Anal. of COs by histol. and biochem. methods revealed that organoids produced from iPSCs derived from patients affected by familial AD or Down syndrome (DS) spontaneously develop over time pathol. features of AD, including accumulation of structures highly reminiscent to amyloid plaques and neurofibrillary tangles.  These pathol. abnormalities were not obsd. in COs generated from various controls, including human iPSCs from healthy individuals, human iPSCs from patients affected by Creutzfeldt-Jakob disease, mouse embryonic stem cells (ESCs), or mouse iPSCs.  These findings enable modeling genetic AD in a human cellular context in a 3D cortical-like tissue developed in vitro from patient-specific stem cells.  This system provides a more relevant disease model compared to pre-existing methods and offers a new platform for discovery of novel targets and screening of drugs for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZw32-DOdVa7Vg90H21EOLACvtfcHk0li0LVnspeCNUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1GltLzE&md5=e2ec2923964014d8c9bb145e3d5eeae3</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1038%2Fs41380-018-0229-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41380-018-0229-8%26sid%3Dliteratum%253Aachs%26aulast%3DGonzalez%26aufirst%3DC.%26aulast%3DArmijo%26aufirst%3DE.%26aulast%3DBravo-Alegria%26aufirst%3DJ.%26aulast%3DBecerra-Calixto%26aufirst%3DA.%26aulast%3DMays%26aufirst%3DC.%2BE.%26aulast%3DSoto%26aufirst%3DC.%26atitle%3DModeling%2520Amyloid%2520Beta%2520and%2520Tau%2520Pathology%2520in%2520Human%2520Cerebral%2520Organoids%26jtitle%3DMol.%2520Psychiatry%26date%3D2018%26volume%3D23%26spage%3D2363%26epage%3D2374%26doi%3D10.1038%2Fs41380-018-0229-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trujillo, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muotri, A. R.</span></span> <span> </span><span class="NLM_article-title">Brain Organoids and the Study of Neurodevelopment</span>. <i>Trends Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">982</span>– <span class="NLM_lpage">990</span>, <span class="refDoi"> DOI: 10.1016/j.molmed.2018.09.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.molmed.2018.09.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30377071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFyntrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=982-990&author=C.+A.+Trujilloauthor=A.+R.+Muotri&title=Brain+Organoids+and+the+Study+of+Neurodevelopment&doi=10.1016%2Fj.molmed.2018.09.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Brain Organoids and the Study of Neurodevelopment</span></div><div class="casAuthors">Trujillo, Cleber A.; Muotri, Alysson R.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Molecular Medicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">982-990</span>CODEN:
                <span class="NLM_cas:coden">TMMRCY</span>;
        ISSN:<span class="NLM_cas:issn">1471-4914</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Brain organoids are 3D self-assembled structures composed of hundreds of thousands to millions of cells that resemble the cellular organization and transcriptional and epigenetic signature of a developing human brain.  Advancements using brain organoids have been made to elucidate the genetic basis of certain neurodevelopmental disorders, such as microcephaly and autism; and to investigate the impact of environmental factors to the brain, such as during Zika virus infection.  It remains to be explored how far brain organoids can functionally mature and process external information.  An improved brain organoid model might reproduce important aspects of the human brain in a more reproducible and high-throughput fashion.  This novel and complementary approach in the neuroscience toolbox opens perspectives to understand the fundamental features of the human neurodevelopment, with implications to personalize therapeutic opportunities for neurol. disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNGYKN-bnYlbVg90H21EOLACvtfcHk0likJrQWyGXUQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFyntrnF&md5=3a805e099e9e8c56b5f0239c2b158d4d</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2018.09.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2018.09.005%26sid%3Dliteratum%253Aachs%26aulast%3DTrujillo%26aufirst%3DC.%2BA.%26aulast%3DMuotri%26aufirst%3DA.%2BR.%26atitle%3DBrain%2520Organoids%2520and%2520the%2520Study%2520of%2520Neurodevelopment%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2018%26volume%3D24%26spage%3D982%26epage%3D990%26doi%3D10.1016%2Fj.molmed.2018.09.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arrasate, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span> <span> </span><span class="NLM_article-title">Automated Microscope System for Determining Factors That Predict Neuronal Fate</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">3840</span>– <span class="NLM_lpage">3845</span>, <span class="refDoi"> DOI: 10.1073/pnas.0409777102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1073%2Fpnas.0409777102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=15738408" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BD2MXisVOgt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=3840-3845&author=M.+Arrasateauthor=S.+Finkbeiner&title=Automated+Microscope+System+for+Determining+Factors+That+Predict+Neuronal+Fate&doi=10.1073%2Fpnas.0409777102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Automated microscope system for determining factors that predict neuronal fate</span></div><div class="casAuthors">Arrasate, Montserrat; Finkbeiner, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3840-3845</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Unraveling cause-and-effect relationships in the nervous system is challenging because some biol. processes begin stochastically, take a significant amt. of time to unfold, and affect small neuronal subpopulations that can be difficult to isolate and measure.  Single-cell approaches are slow, subject to user bias, and sometimes too laborious to achieve sample sizes large enough to detect important effects.  Here, we describe an automated imaging and anal. system that enables us to follow the fates of individual cells and intracellular proteins over time.  Observations can be quantified in a high-throughput manner with minimal user bias.  We have adapted survival anal. methods to det. whether and how factors measured during longitudinal anal. predict a particular biol. outcome.  The ability to monitor complex processes at single-cell resoln. quickly, quant., and over long intervals should have wide applications for biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUzK6HSFyV1rVg90H21EOLACvtfcHk0likJrQWyGXUQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXisVOgt7Y%253D&md5=8d2173976e3b9eb4b50b38cac219444c</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0409777102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0409777102%26sid%3Dliteratum%253Aachs%26aulast%3DArrasate%26aufirst%3DM.%26aulast%3DFinkbeiner%26aufirst%3DS.%26atitle%3DAutomated%2520Microscope%2520System%2520for%2520Determining%2520Factors%2520That%2520Predict%2520Neuronal%2520Fate%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D3840%26epage%3D3845%26doi%3D10.1073%2Fpnas.0409777102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haston, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span> <span> </span><span class="NLM_article-title">Clinical Trials in a Dish: The Potential of Pluripotent Stem Cells to Develop Therapies for Neurodegenerative Diseases</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">489</span>– <span class="NLM_lpage">510</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010715-103548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1146%2Fannurev-pharmtox-010715-103548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26514199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslGmsbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=489-510&author=K.+M.+Hastonauthor=S.+Finkbeiner&title=Clinical+Trials+in+a+Dish%3A+The+Potential+of+Pluripotent+Stem+Cells+to+Develop+Therapies+for+Neurodegenerative+Diseases&doi=10.1146%2Fannurev-pharmtox-010715-103548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical Trials in a Dish: The Potential of Pluripotent Stem Cells to Develop Therapies for Neurodegenerative Diseases</span></div><div class="casAuthors">Haston, Kelly M.; Finkbeiner, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">489-510</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Neurodegenerative diseases are a leading cause of death.  No disease-modifying therapies are available, and preclin. animal model data have routinely failed to translate into success for therapeutics.  Induced pluripotent stem cell (iPSC) biol. holds great promise for human in vitro disease modeling because these cells can give rise to any cell in the human brain and display phenotypes specific to neurodegenerative diseases previously identified in postmortem and clin. samples.  Here, we explore the potential and caveats of iPSC technol. as a platform for drug development and screening, and the future potential to use large cohorts of disease-bearing iPSCs to perform clin. trials in a dish.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIV9RCr3Mpi7Vg90H21EOLACvtfcHk0likJrQWyGXUQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslGmsbbO&md5=0d4d798b015770e8568ad1710f8ff3e6</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010715-103548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010715-103548%26sid%3Dliteratum%253Aachs%26aulast%3DHaston%26aufirst%3DK.%2BM.%26aulast%3DFinkbeiner%26aufirst%3DS.%26atitle%3DClinical%2520Trials%2520in%2520a%2520Dish%253A%2520The%2520Potential%2520of%2520Pluripotent%2520Stem%2520Cells%2520to%2520Develop%2520Therapies%2520for%2520Neurodegenerative%2520Diseases%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2016%26volume%3D56%26spage%3D489%26epage%3D510%26doi%3D10.1146%2Fannurev-pharmtox-010715-103548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michael
Ando, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanaswamy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christiansen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoyer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roat, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueden, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shankar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, P.</span></span> <span> </span><span class="NLM_article-title">Assessing Microscope Image Focus Quality with Deep Learning</span>. <i>BMC Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">77</span>, <span class="refDoi"> DOI: 10.1186/s12859-018-2087-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1186%2Fs12859-018-2087-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=29540156" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC1Mnht1aisg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=77&author=S.+J.+Yangauthor=M.+Berndlauthor=D.+Michael%0AAndoauthor=M.+Barchauthor=A.+Narayanaswamyauthor=E.+Christiansenauthor=S.+Hoyerauthor=C.+Roatauthor=J.+Hungauthor=C.+T.+Ruedenauthor=A.+Shankarauthor=S.+Finkbeinerauthor=P.+Nelson&title=Assessing+Microscope+Image+Focus+Quality+with+Deep+Learning&doi=10.1186%2Fs12859-018-2087-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing microscope image focus quality with deep learning</span></div><div class="casAuthors">Yang Samuel J; Berndl Marc; Michael Ando D; Narayanaswamy Arunachalam; Christiansen Eric; Hoyer Stephan; Roat Chris; Shankar Asim; Nelson Philip; Barch Mariya; Finkbeiner Steven; Hung Jane; Hung Jane; Rueden Curtis T; Finkbeiner Steven</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">77</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Large image datasets acquired on automated microscopes typically have some fraction of low quality, out-of-focus images, despite the use of hardware autofocus systems.  Identification of these images using automated image analysis with high accuracy is important for obtaining a clean, unbiased image dataset.  Complicating this task is the fact that image focus quality is only well-defined in foreground regions of images, and as a result, most previous approaches only enable a computation of the relative difference in quality between two or more images, rather than an absolute measure of quality.  RESULTS:  We present a deep neural network model capable of predicting an absolute measure of image focus on a single image in isolation, without any user-specified parameters.  The model operates at the image-patch level, and also outputs a measure of prediction certainty, enabling interpretable predictions.  The model was trained on only 384 in-focus Hoechst (nuclei) stain images of U2OS cells, which were synthetically defocused to one of 11 absolute defocus levels during training.  The trained model can generalize on previously unseen real Hoechst stain images, identifying the absolute image focus to within one defocus level (approximately 3 pixel blur diameter difference) with 95% accuracy.  On a simpler binary in/out-of-focus classification task, the trained model outperforms previous approaches on both Hoechst and Phalloidin (actin) stain images (F-scores of 0.89 and 0.86, respectively over 0.84 and 0.83), despite only having been presented Hoechst stain images during training.  Lastly, we observe qualitatively that the model generalizes to two additional stains, Hoechst and Tubulin, of an unseen cell type (Human MCF-7) acquired on a different instrument.  CONCLUSIONS:  Our deep neural network enables classification of out-of-focus microscope images with both higher accuracy and greater precision than previous approaches via interpretable patch-level focus and certainty predictions.  The use of synthetically defocused images precludes the need for a manually annotated training dataset.  The model also generalizes to different image and cell types.  The framework for model training and image prediction is available as a free software library and the pre-trained model is available for immediate use in Fiji (ImageJ) and CellProfiler.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS54d2r4ujj7a__Jf7-eMvJfW6udTcc2ebnPCSXytbcw7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mnht1aisg%253D%253D&md5=dc1ec2714da913b75c050af0449ed8b3</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1186%2Fs12859-018-2087-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12859-018-2087-4%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DS.%2BJ.%26aulast%3DBerndl%26aufirst%3DM.%26aulast%3DMichael%2BAndo%26aufirst%3DD.%26aulast%3DBarch%26aufirst%3DM.%26aulast%3DNarayanaswamy%26aufirst%3DA.%26aulast%3DChristiansen%26aufirst%3DE.%26aulast%3DHoyer%26aufirst%3DS.%26aulast%3DRoat%26aufirst%3DC.%26aulast%3DHung%26aufirst%3DJ.%26aulast%3DRueden%26aufirst%3DC.%2BT.%26aulast%3DShankar%26aufirst%3DA.%26aulast%3DFinkbeiner%26aufirst%3DS.%26aulast%3DNelson%26aufirst%3DP.%26atitle%3DAssessing%2520Microscope%2520Image%2520Focus%2520Quality%2520with%2520Deep%2520Learning%26jtitle%3DBMC%2520Bioinformatics%26date%3D2018%26volume%3D19%26spage%3D77%26doi%3D10.1186%2Fs12859-018-2087-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frumkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kassner, P. D.</span></span> <span> </span><span class="NLM_article-title">Cell-Based Screening: Extracting Meaning from Complex Data</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">160</span>– <span class="NLM_lpage">174</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2015.02.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.neuron.2015.02.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25856492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsVCisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2015&pages=160-174&author=S.+Finkbeinerauthor=M.+Frumkinauthor=P.+D.+Kassner&title=Cell-Based+Screening%3A+Extracting+Meaning+from+Complex+Data&doi=10.1016%2Fj.neuron.2015.02.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-Based Screening: Extracting Meaning from Complex Data</span></div><div class="casAuthors">Finkbeiner, Steven; Frumkin, Michael; Kassner, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">160-174</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Unbiased discovery approaches have the potential to uncover neurobiol. insights into CNS disease and lead to the development of therapies.  Here, we review lessons learned from imaging-based screening approaches and recent advances in these areas, including powerful new computational tools to synthesize complex data into more useful knowledge that can reliably guide future research and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobSk3HekAKbbVg90H21EOLACvtfcHk0lhZVTfJ-s6oZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsVCisLg%253D&md5=8aff9eec9955222985f8f10b1d4e22ab</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2015.02.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2015.02.023%26sid%3Dliteratum%253Aachs%26aulast%3DFinkbeiner%26aufirst%3DS.%26aulast%3DFrumkin%26aufirst%3DM.%26aulast%3DKassner%26aufirst%3DP.%2BD.%26atitle%3DCell-Based%2520Screening%253A%2520Extracting%2520Meaning%2520from%2520Complex%2520Data%26jtitle%3DNeuron%26date%3D2015%26volume%3D86%26spage%3D160%26epage%3D174%26doi%3D10.1016%2Fj.neuron.2015.02.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daub, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span> <span> </span><span class="NLM_article-title">High-Content Screening of Primary Neurons: Ready for Prime Time</span>. <i>Curr. Opin. Neurobiol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">537</span>– <span class="NLM_lpage">543</span>, <span class="refDoi"> DOI: 10.1016/j.conb.2009.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.conb.2009.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=19889533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsV2gs7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=537-543&author=A.+Daubauthor=P.+Sharmaauthor=S.+Finkbeiner&title=High-Content+Screening+of+Primary+Neurons%3A+Ready+for+Prime+Time&doi=10.1016%2Fj.conb.2009.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">High-content screening of primary neurons: Ready for prime time</span></div><div class="casAuthors">Daub, Aaron; Sharma, Punita; Finkbeiner, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Neurobiology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">537-543</span>CODEN:
                <span class="NLM_cas:coden">COPUEN</span>;
        ISSN:<span class="NLM_cas:issn">0959-4388</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">High-content screening (HCS), historically limited to drug-development companies, is now a powerful and affordable technol. for academic researchers.  Through automated routines, this technol. acquires large datasets of fluorescence images depicting the functional states of thousands to millions of cells.  Information on shapes, textures, intensities, and localizations is then used to create unique representations, or 'phenotypic signatures,' of each cell.  These signatures quantify physiol. or diseased states, for example, dendritic arborization, drug response, or cell coping strategies.  Live-cell imaging in HCS adds the ability to correlate cellular events at different points in time, thereby allowing sensitivities and observations not possible with fixed endpoint anal.  HCS with live-cell imaging therefore provides an unprecedented capability to detect spatiotemporal changes in cells and is particularly suited for time-dependent, stochastic processes such as neurodegenerative disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6q9y6kFFysLVg90H21EOLACvtfcHk0lhZVTfJ-s6oZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsV2gs7nP&md5=fa17456b6bafd063e024e34f657bf494</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1016%2Fj.conb.2009.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.conb.2009.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DDaub%26aufirst%3DA.%26aulast%3DSharma%26aufirst%3DP.%26aulast%3DFinkbeiner%26aufirst%3DS.%26atitle%3DHigh-Content%2520Screening%2520of%2520Primary%2520Neurons%253A%2520Ready%2520for%2520Prime%2520Time%26jtitle%3DCurr.%2520Opin.%2520Neurobiol.%26date%3D2009%26volume%3D19%26spage%3D537%26epage%3D543%26doi%3D10.1016%2Fj.conb.2009.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Christiansen, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ando, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javaherian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skibinski, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipnick, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mount, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neil, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goyal, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedus, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poplin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span> <span> </span><span class="NLM_article-title">In Silico Labeling: Predicting Fluorescent Labels in Unlabeled Images</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>173</i></span>,  <span class="NLM_fpage">792</span>– <span class="NLM_lpage">803</span>, e719 <span class="refDoi"> DOI: 10.1016/j.cell.2018.03.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.cell.2018.03.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=29656897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnsFClu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=792-803&author=E.+M.+Christiansenauthor=S.+J.+Yangauthor=D.+M.+Andoauthor=A.+Javaherianauthor=G.+Skibinskiauthor=S.+Lipnickauthor=E.+Mountauthor=A.+O%E2%80%99Neilauthor=K.+Shahauthor=A.+K.+Leeauthor=P.+Goyalauthor=W.+Fedusauthor=R.+Poplinauthor=A.+Estevaauthor=M.+Berndlauthor=L.+L.+Rubinauthor=P.+Nelsonauthor=S.+Finkbeiner&title=In+Silico+Labeling%3A+Predicting+Fluorescent+Labels+in+Unlabeled+Images&doi=10.1016%2Fj.cell.2018.03.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">In Silico Labeling: Predicting Fluorescent Labels in Unlabeled Images</span></div><div class="casAuthors">Christiansen, Eric M.; Yang, Samuel J.; Ando, D. Michael; Javaherian, Ashkan; Skibinski, Gaia; Lipnick, Scott; Mount, Elliot; O'Neil, Alison; Shah, Kevan; Lee, Alicia K.; Goyal, Piyush; Fedus, William; Poplin, Ryan; Esteva, Andre; Berndl, Marc; Rubin, Lee L.; Nelson, Philip; Finkbeiner, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">173</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">792-803.e19</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Microscopy is a central method in life sciences.  Many popular methods, such as antibody labeling, are used to add phys. fluorescent labels to specific cellular constituents.  However, these approaches have significant drawbacks, including inconsistency; limitations in the no. of simultaneous labels because of spectral overlap; and necessary perturbations of the expt., such as fixing the cells, to generate the measurement.  Here, we show that a computational machine-learning approach, which we call "in silico labeling" (ISL), reliably predicts some fluorescent labels from transmitted-light images of unlabeled fixed or live biol. samples.  ISL predicts a range of labels, such as those for nuclei, cell type (e.g., neural), and cell state (e.g., cell death).  Because prediction happens in silico, the method is consistent, is not limited by spectral overlap, and does not disturb the expt.  ISL generates biol. measurements that would otherwise be problematic or impossible to acquire.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprM6l-lkvHfLVg90H21EOLACvtfcHk0lhZVTfJ-s6oZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnsFClu7k%253D&md5=45cbd1f004ac1b093f37903c5bae4c83</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.03.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.03.040%26sid%3Dliteratum%253Aachs%26aulast%3DChristiansen%26aufirst%3DE.%2BM.%26aulast%3DYang%26aufirst%3DS.%2BJ.%26aulast%3DAndo%26aufirst%3DD.%2BM.%26aulast%3DJavaherian%26aufirst%3DA.%26aulast%3DSkibinski%26aufirst%3DG.%26aulast%3DLipnick%26aufirst%3DS.%26aulast%3DMount%26aufirst%3DE.%26aulast%3DO%25E2%2580%2599Neil%26aufirst%3DA.%26aulast%3DShah%26aufirst%3DK.%26aulast%3DLee%26aufirst%3DA.%2BK.%26aulast%3DGoyal%26aufirst%3DP.%26aulast%3DFedus%26aufirst%3DW.%26aulast%3DPoplin%26aufirst%3DR.%26aulast%3DEsteva%26aufirst%3DA.%26aulast%3DBerndl%26aufirst%3DM.%26aulast%3DRubin%26aufirst%3DL.%2BL.%26aulast%3DNelson%26aufirst%3DP.%26aulast%3DFinkbeiner%26aufirst%3DS.%26atitle%3DIn%2520Silico%2520Labeling%253A%2520Predicting%2520Fluorescent%2520Labels%2520in%2520Unlabeled%2520Images%26jtitle%3DCell%26date%3D2018%26volume%3D173%26spage%3D792%26epage%3D803%26doi%3D10.1016%2Fj.cell.2018.03.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mason, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elia, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span> <span> </span><span class="NLM_article-title">The Receptor-Interacting Serine/Threonine Protein Kinase 1 (Ripk1) Regulates Progranulin Levels</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">3262</span>– <span class="NLM_lpage">3272</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.752006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1074%2Fjbc.M116.752006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28069809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtlyqtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=3262-3272&author=A.+R.+Masonauthor=L.+P.+Eliaauthor=S.+Finkbeiner&title=The+Receptor-Interacting+Serine%2FThreonine+Protein+Kinase+1+%28Ripk1%29+Regulates+Progranulin+Levels&doi=10.1074%2Fjbc.M116.752006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">The Receptor-interacting Serine/Threonine Protein Kinase 1 (RIPK1) Regulates Progranulin Levels</span></div><div class="casAuthors">Mason, Amanda R.; Elia, Lisa P.; Finkbeiner, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3262-3272</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Progranulin (PGRN), a secreted growth factor, is a key regulator of inflammation and is genetically linked to two common and devastating neurodegenerative diseases.  Haploinsufficiency mutations in GRN, the gene encoding PGRN, cause frontotemporal dementia (FTD), and a GRN SNP confers significantly increased risk for Alzheimer's disease (AD).  Because cellular and animal data indicate that increasing PGRN can reverse phenotypes of both FTD and AD, modulating PGRN level has been proposed as a therapeutic strategy for both diseases.  However, little is known about the regulation of PGRN levels.  In this study, we performed an siRNA-based screen of the kinome to identify genetic regulators of PGRN levels in a rodent cell-based model system.  We found that knocking down receptor-interacting serine/threonine protein kinase 1 (Ripk1) increased both intracellular and extracellular PGRN protein levels by increasing the translation rate of PGRN without affecting mRNA levels.  We obsd. this effect in Neuro2a cells, wild-type primary mouse neurons, and Grn-haploinsufficient primary neurons from an FTD mouse model.  We found that the effect of RIPK1 on PGRN is independent of the kinase activity of RIPK1 and occurs through a novel signaling pathway.  These data suggest that targeting RIPK1 may be a therapeutic strategy in both AD and FTD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3vcUnBMUQS7Vg90H21EOLACvtfcHk0lgnmGO7jgKwig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtlyqtbs%253D&md5=be49c50b92b5866bb9bf3f288e230491</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.752006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.752006%26sid%3Dliteratum%253Aachs%26aulast%3DMason%26aufirst%3DA.%2BR.%26aulast%3DElia%26aufirst%3DL.%2BP.%26aulast%3DFinkbeiner%26aufirst%3DS.%26atitle%3DThe%2520Receptor-Interacting%2520Serine%252FThreonine%2520Protein%2520Kinase%25201%2520%2528Ripk1%2529%2520Regulates%2520Progranulin%2520Levels%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D3262%26epage%3D3272%26doi%3D10.1074%2Fjbc.M116.752006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Armakola, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higgins, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figley, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barmada, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scarborough, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shorter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krogan, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farese, R. V.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gitler, A. D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Rna Lariat Debranching Enzyme Suppresses Tdp-43 Toxicity in Als Disease Models</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1302</span>– <span class="NLM_lpage">1309</span>, <span class="refDoi"> DOI: 10.1038/ng.2434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fng.2434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23104007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFOmt7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=1302-1309&author=M.+Armakolaauthor=M.+J.+Higginsauthor=M.+D.+Figleyauthor=S.+J.+Barmadaauthor=E.+A.+Scarboroughauthor=Z.+Diazauthor=X.+Fangauthor=J.+Shorterauthor=N.+J.+Kroganauthor=S.+Finkbeinerauthor=R.+V.+Fareseauthor=A.+D.+Gitler&title=Inhibition+of+Rna+Lariat+Debranching+Enzyme+Suppresses+Tdp-43+Toxicity+in+Als+Disease+Models&doi=10.1038%2Fng.2434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of RNA lariat debranching enzyme suppresses TDP-43 toxicity in ALS disease models</span></div><div class="casAuthors">Armakola, Maria; Higgins, Matthew J.; Figley, Matthew D.; Barmada, Sami J.; Scarborough, Emily A.; Diaz, Zamia; Fang, Xiaodong; Shorter, James; Krogan, Nevan J.; Finkbeiner, Steven; Farese, Robert V.; Gitler, Aaron D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1302-1309</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Amyotrophic lateral sclerosis (ALS) is a devastating neurodegenerative disease primarily affecting motor neurons.  Mutations in the gene encoding TDP-43 cause some forms of the disease, and cytoplasmic TDP-43 aggregates accumulate in degenerating neurons of most individuals with ALS.  Thus, strategies aimed at targeting the toxicity of cytoplasmic TDP-43 aggregates may be effective.  Here, we report results from two genome-wide loss-of-function TDP-43 toxicity suppressor screens in yeast.  The strongest suppressor of TDP-43 toxicity was deletion of DBR1, which encodes an RNA lariat debranching enzyme.  We show that, in the absence of Dbr1 enzymic activity, intronic lariats accumulate in the cytoplasm and likely act as decoys to sequester TDP-43, preventing it from interfering with essential cellular RNAs and RNA-binding proteins.  Knockdown of Dbr1 in a human neuronal cell line or in primary rat neurons is also sufficient to rescue TDP-43 toxicity.  Our findings provide insight into TDP-43-mediated cytotoxicity and suggest that decreasing Dbr1 activity could be a potential therapeutic approach for ALS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHPDqiwEnEcbVg90H21EOLACvtfcHk0lgnmGO7jgKwig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFOmt7jI&md5=571f3fa8d80cd5b9a06fc6d9426e782e</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1038%2Fng.2434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2434%26sid%3Dliteratum%253Aachs%26aulast%3DArmakola%26aufirst%3DM.%26aulast%3DHiggins%26aufirst%3DM.%2BJ.%26aulast%3DFigley%26aufirst%3DM.%2BD.%26aulast%3DBarmada%26aufirst%3DS.%2BJ.%26aulast%3DScarborough%26aufirst%3DE.%2BA.%26aulast%3DDiaz%26aufirst%3DZ.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DShorter%26aufirst%3DJ.%26aulast%3DKrogan%26aufirst%3DN.%2BJ.%26aulast%3DFinkbeiner%26aufirst%3DS.%26aulast%3DFarese%26aufirst%3DR.%2BV.%26aulast%3DGitler%26aufirst%3DA.%2BD.%26atitle%3DInhibition%2520of%2520Rna%2520Lariat%2520Debranching%2520Enzyme%2520Suppresses%2520Tdp-43%2520Toxicity%2520in%2520Als%2520Disease%2520Models%26jtitle%3DNat.%2520Genet.%26date%3D2012%26volume%3D44%26spage%3D1302%26epage%3D1309%26doi%3D10.1038%2Fng.2434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giaever, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nislow, C.</span></span> <span> </span><span class="NLM_article-title">The Yeast Deletion Collection: A Decade of Functional Genomics</span>. <i>Genetics</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>197</i></span>,  <span class="NLM_fpage">451</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1534/genetics.114.161620</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1534%2Fgenetics.114.161620" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=24939991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=197&publication_year=2014&pages=451-465&author=G.+Giaeverauthor=C.+Nislow&title=The+Yeast+Deletion+Collection%3A+A+Decade+of+Functional+Genomics&doi=10.1534%2Fgenetics.114.161620"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">The yeast deletion collection: a decade of functional genomics</span></div><div class="casAuthors">Giaever, Guri; Nislow, Corey</div><div class="citationInfo"><span class="NLM_cas:title">Genetics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">197</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">451-465, 15</span>CODEN:
                <span class="NLM_cas:coden">GENTAE</span>;
        ISSN:<span class="NLM_cas:issn">0016-6731</span>.
    
            (<span class="NLM_cas:orgname">Genetics Society of America</span>)
        </div><div class="casAbstract">A review.  The yeast deletion collections comprise >21,000 mutant strains that carry precise start-to-stop deletions of ∼6000 open reading frames.  This collection includes heterozygous and homozygous diploids, and haploids of both MATa and MATα mating types.  The yeast deletion collection, or yeast knockout (YKO) set, represents the first and only complete, systematically constructed deletion collection available for any organism.  Conceived during the Saccharomyces cerevisiae sequencing project, work on the project began in 1998 and was completed in 2002.  The YKO strains have been used in numerous labs. in >1000 genome-wide screens.  This landmark genome project has inspired development of numerous genome-wide technologies in organisms from yeast to man.  Notable spinoff technologies include synthetic genetic array and HIPHOP chemogenomics.  In this retrospective, we briefly describe the yeast deletion project and some of its most noteworthy biol. contributions and the impact that these collections have had on the yeast research community and on genomics in general.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmljNCsYgzn7Vg90H21EOLACvtfcHk0lgnmGO7jgKwig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLrN&md5=f33c484976824362db6d4f5ca13ad38f</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1534%2Fgenetics.114.161620&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1534%252Fgenetics.114.161620%26sid%3Dliteratum%253Aachs%26aulast%3DGiaever%26aufirst%3DG.%26aulast%3DNislow%26aufirst%3DC.%26atitle%3DThe%2520Yeast%2520Deletion%2520Collection%253A%2520A%2520Decade%2520of%2520Functional%2520Genomics%26jtitle%3DGenetics%26date%3D2014%26volume%3D197%26spage%3D451%26epage%3D465%26doi%3D10.1534%2Fgenetics.114.161620" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elden, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen-Plotkin, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armakola, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geser, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clay-Falcone, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCluskey, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juhr, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruber, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rub, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auburger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojanowski, J. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Deerlin, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonini, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gitler, A. D.</span></span> <span> </span><span class="NLM_article-title">Ataxin-2 Intermediate-Length Polyglutamine Expansions Are Associated with Increased Risk for Als</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>466</i></span>,  <span class="NLM_fpage">1069</span>– <span class="NLM_lpage">1075</span>, <span class="refDoi"> DOI: 10.1038/nature09320</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnature09320" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=20740007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVOnurvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2010&pages=1069-1075&author=A.+C.+Eldenauthor=H.+J.+Kimauthor=M.+P.+Hartauthor=A.+S.+Chen-Plotkinauthor=B.+S.+Johnsonauthor=X.+Fangauthor=M.+Armakolaauthor=F.+Geserauthor=R.+Greeneauthor=M.+M.+Luauthor=A.+Padmanabhanauthor=D.+Clay-Falconeauthor=L.+McCluskeyauthor=L.+Elmanauthor=D.+Juhrauthor=P.+J.+Gruberauthor=U.+Rubauthor=G.+Auburgerauthor=J.+Q.+Trojanowskiauthor=V.+M.+Leeauthor=V.+M.+Van+Deerlinauthor=N.+M.+Boniniauthor=A.+D.+Gitler&title=Ataxin-2+Intermediate-Length+Polyglutamine+Expansions+Are+Associated+with+Increased+Risk+for+Als&doi=10.1038%2Fnature09320"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Ataxin-2 intermediate-length polyglutamine expansions are associated with increased risk for ALS</span></div><div class="casAuthors">Elden, Andrew C.; Kim, Hyung-Jun; Hart, Michael P.; Chen-Plotkin, Alice S.; Johnson, Brian S.; Fang, Xiaodong; Armakola, Maria; Geser, Felix; Greene, Robert; Lu, Min Min; Padmanabhan, Arun; Clay-Falcone, Dana; McCluskey, Leo; Elman, Lauren; Juhr, Denise; Gruber, Peter J.; Rb, Udo; Auburger, Georg; Trojanowski, John Q.; Lee, Virginia M.-Y.; Van Deerlin, Vivianna M.; Bonini, Nancy M.; Gitler, Aaron D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">466</span>
        (<span class="NLM_cas:issue">7310</span>),
    <span class="NLM_cas:pages">1069-1075</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The causes of amyotrophic lateral sclerosis (ALS), a devastating human neurodegenerative disease, are poorly understood, although the protein TDP-43 has been suggested to have a crit. role in disease pathogenesis.  Here we show that ataxin 2 (ATXN2), a polyglutamine (polyQ) protein mutated in spinocerebellar ataxia type 2, is a potent modifier of TDP-43 toxicity in animal and cellular models.  ATXN2 and TDP-43 assoc. in a complex that depends on RNA.  In spinal cord neurons of ALS patients, ATXN2 is abnormally localized; likewise, TDP-43 shows mislocalization in spinocerebellar ataxia type 2.  To assess the involvement of ATXN2 in ALS, we analyzed the length of the polyQ repeat in the ATXN2 gene in 915 ALS patients.  We found that intermediate-length polyQ expansions (2733 glutamines) in ATXN2 were significantly assocd. with ALS.  These data establish ATXN2 as a relatively common ALS susceptibility gene.  Furthermore, these findings indicate that the TDP-43ATXN2 interaction may be a promising target for therapeutic intervention in ALS and other TDP-43 proteinopathies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopssdThk2l_7Vg90H21EOLACvtfcHk0lizYohIzqcRfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVOnurvF&md5=2de17a182a5b150f81fbdece00813bcd</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1038%2Fnature09320&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09320%26sid%3Dliteratum%253Aachs%26aulast%3DElden%26aufirst%3DA.%2BC.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DHart%26aufirst%3DM.%2BP.%26aulast%3DChen-Plotkin%26aufirst%3DA.%2BS.%26aulast%3DJohnson%26aufirst%3DB.%2BS.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DArmakola%26aufirst%3DM.%26aulast%3DGeser%26aufirst%3DF.%26aulast%3DGreene%26aufirst%3DR.%26aulast%3DLu%26aufirst%3DM.%2BM.%26aulast%3DPadmanabhan%26aufirst%3DA.%26aulast%3DClay-Falcone%26aufirst%3DD.%26aulast%3DMcCluskey%26aufirst%3DL.%26aulast%3DElman%26aufirst%3DL.%26aulast%3DJuhr%26aufirst%3DD.%26aulast%3DGruber%26aufirst%3DP.%2BJ.%26aulast%3DRub%26aufirst%3DU.%26aulast%3DAuburger%26aufirst%3DG.%26aulast%3DTrojanowski%26aufirst%3DJ.%2BQ.%26aulast%3DLee%26aufirst%3DV.%2BM.%26aulast%3DVan%2BDeerlin%26aufirst%3DV.%2BM.%26aulast%3DBonini%26aufirst%3DN.%2BM.%26aulast%3DGitler%26aufirst%3DA.%2BD.%26atitle%3DAtaxin-2%2520Intermediate-Length%2520Polyglutamine%2520Expansions%2520Are%2520Associated%2520with%2520Increased%2520Risk%2520for%2520Als%26jtitle%3DNature%26date%3D2010%26volume%3D466%26spage%3D1069%26epage%3D1075%26doi%3D10.1038%2Fnature09320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Armakola, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gitler, A. D.</span></span> <span> </span><span class="NLM_article-title">Tdp-43 Toxicity in Yeast</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">238</span>– <span class="NLM_lpage">245</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2010.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.ymeth.2010.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=21115123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFGisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2011&pages=238-245&author=M.+Armakolaauthor=M.+P.+Hartauthor=A.+D.+Gitler&title=Tdp-43+Toxicity+in+Yeast&doi=10.1016%2Fj.ymeth.2010.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">TDP-43 toxicity in yeast</span></div><div class="casAuthors">Armakola, Maria; Hart, Michael P.; Gitler, Aaron D.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">238-245</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The budding yeast Saccharomyces cerevisiae is an emerging tool for investigating the mol. pathways that underpin several human neurodegenerative disorders assocd. with protein misfolding.  Amyotrophic lateral sclerosis (ALS) is a devastating adult onset neurodegenerative disease primarily affecting motor neurons.  The protein TDP-43 has recently been demonstrated to play an important role in the disease, however, the mechanisms by which TDP-43 contributes to pathogenesis are unclear.  To explore the mechanistic details that result in aberrant accumulation of TDP-43 and to discover potential strategies for therapeutic intervention, we employed a yeast TDP-43 proteinopathy model system.  These studies allowed us to det. the regions of TDP-43 required for aggregation and toxicity and to define the effects of ALS-linked mutant forms of TDP-43.  We have also been able to harness the power of yeast genetics to identify potent modifiers of TDP-43 toxicity using high-throughput yeast genetic screens.  Here, we describe the methods and approaches that we have used in order to gain insight into TDP-43 biol. and its role in disease.  These approaches are readily adaptable to other neurodegenerative disease proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpekRGMTYUokbVg90H21EOLACvtfcHk0lizYohIzqcRfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFGisLc%253D&md5=786a16a829e3648867537ea8ce10aed8</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2010.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2010.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DArmakola%26aufirst%3DM.%26aulast%3DHart%26aufirst%3DM.%2BP.%26aulast%3DGitler%26aufirst%3DA.%2BD.%26atitle%3DTdp-43%2520Toxicity%2520in%2520Yeast%26jtitle%3DMethods%26date%3D2011%26volume%3D53%26spage%3D238%26epage%3D245%26doi%3D10.1016%2Fj.ymeth.2010.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Becker, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bieri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knowles, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafar-Nejad, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soriano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auburger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulst, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rigo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gitler, A. D.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Reduction of Ataxin-2 Extends Lifespan and Reduces Pathology in Tdp-43 Mice</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>544</i></span>,  <span class="NLM_fpage">367</span>– <span class="NLM_lpage">371</span>, <span class="refDoi"> DOI: 10.1038/nature22038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnature22038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28405022" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvVOjsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=544&publication_year=2017&pages=367-371&author=L.+A.+Beckerauthor=B.+Huangauthor=G.+Bieriauthor=R.+Maauthor=D.+A.+Knowlesauthor=P.+Jafar-Nejadauthor=J.+Messingauthor=H.+J.+Kimauthor=A.+Sorianoauthor=G.+Auburgerauthor=S.+M.+Pulstauthor=J.+P.+Taylorauthor=F.+Rigoauthor=A.+D.+Gitler&title=Therapeutic+Reduction+of+Ataxin-2+Extends+Lifespan+and+Reduces+Pathology+in+Tdp-43+Mice&doi=10.1038%2Fnature22038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic reduction of ataxin-2 extends lifespan and reduces pathology in TDP-43 mice</span></div><div class="casAuthors">Becker, Lindsay A.; Huang, Brenda; Bieri, Gregor; Ma, Rosanna; Knowles, David A.; Jafar-Nejad, Paymaan; Messing, James; Kim, Hong Joo; Soriano, Armand; Auburger, Georg; Pulst, Stefan M.; Taylor, J. Paul; Rigo, Frank; Gitler, Aaron D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">544</span>
        (<span class="NLM_cas:issue">7650</span>),
    <span class="NLM_cas:pages">367-371</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Amyotrophic lateral sclerosis (ALS) is a rapidly progressing neurodegenerative disease that is characterized by motor neuron loss and that leads to paralysis and death 2-5 years after disease onset.  Nearly all patients with ALS have aggregates of the RNA-binding protein TDP-43 in their brains and spinal cords, and rare mutations in the gene encoding TDP-43 can cause ALS.  There are no effective TDP-43-directed therapies for ALS or related TDP-43 proteinopathies, such as frontotemporal dementia.  Antisense oligonucleotides (ASOs) and RNA-interference approaches are emerging as attractive therapeutic strategies in neurol. diseases.  Indeed, treatment of a rat model of inherited ALS (caused by a mutation in Sod1) with ASOs against Sod1 has been shown to substantially slow disease progression.  However, as SOD1 mutations account for only around 2-5% of ALS cases, addnl. therapeutic strategies are needed.  Silencing TDP-43 itself is probably not appropriate, given its crit. cellular functions.  Here we present a promising alternative therapeutic strategy for ALS that involves targeting ataxin-2.  A decrease in ataxin-2 suppresses TDP-43 toxicity in yeast and flies, and intermediate-length polyglutamine expansions in the ataxin-2 gene increase risk of ALS.  We used two independent approaches to test whether decreasing ataxin-2 levels could mitigate disease in a mouse model of TDP-43 proteinopathy.  First, we crossed ataxin-2 knockout mice with TDP-43 (also known as TARDBP) transgenic mice.  The decrease in ataxin-2 reduced aggregation of TDP-43, markedly increased survival and improved motor function.  Second, in a more therapeutically applicable approach, we administered ASOs targeting ataxin-2 to the central nervous system of TDP-43 transgenic mice.  This single treatment markedly extended survival.  Because TDP-43 aggregation is a component of nearly all cases of ALS, targeting ataxin-2 could represent a broadly effective therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf0ZIIBhNmhrVg90H21EOLACvtfcHk0lizYohIzqcRfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvVOjsbY%253D&md5=a14750d78aa2e30996bd163a7dec8f05</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1038%2Fnature22038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22038%26sid%3Dliteratum%253Aachs%26aulast%3DBecker%26aufirst%3DL.%2BA.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DBieri%26aufirst%3DG.%26aulast%3DMa%26aufirst%3DR.%26aulast%3DKnowles%26aufirst%3DD.%2BA.%26aulast%3DJafar-Nejad%26aufirst%3DP.%26aulast%3DMessing%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DSoriano%26aufirst%3DA.%26aulast%3DAuburger%26aufirst%3DG.%26aulast%3DPulst%26aufirst%3DS.%2BM.%26aulast%3DTaylor%26aufirst%3DJ.%2BP.%26aulast%3DRigo%26aufirst%3DF.%26aulast%3DGitler%26aufirst%3DA.%2BD.%26atitle%3DTherapeutic%2520Reduction%2520of%2520Ataxin-2%2520Extends%2520Lifespan%2520and%2520Reduces%2520Pathology%2520in%2520Tdp-43%2520Mice%26jtitle%3DNature%26date%3D2017%26volume%3D544%26spage%3D367%26epage%3D371%26doi%3D10.1038%2Fnature22038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boettcher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McManus, M. T.</span></span> <span> </span><span class="NLM_article-title">Choosing the Right Tool for the Job: Rnai, Talen, or Crispr</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">575</span>– <span class="NLM_lpage">585</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.04.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.molcel.2015.04.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26000843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpt1Sns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=575-585&author=M.+Boettcherauthor=M.+T.+McManus&title=Choosing+the+Right+Tool+for+the+Job%3A+Rnai%2C+Talen%2C+or+Crispr&doi=10.1016%2Fj.molcel.2015.04.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Choosing the Right Tool for the Job: RNAi, TALEN, or CRISPR</span></div><div class="casAuthors">Boettcher, Michael; McManus, Michael T.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">575-585</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The most widely used approach for defining gene function is to reduce or completely disrupt its normal expression.  For over a decade, RNAi has ruled the lab, offering a magic bullet to disrupt gene expression in many organisms.  However, new biotechnol. tools-specifically CRISPR-based technologies-have become available and are squeezing out RNAi dominance in mammalian cell studies.  These seemingly competing technologies leave research investigators with the question: "Which technol. should I use in my expt.".  This review offers a practical resource to compare and contrast these technologies, guiding the investigator when and where to use this fantastic array of powerful tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZmgnhFd9v1rVg90H21EOLACvtfcHk0li3L-lcxIS7DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpt1Sns70%253D&md5=c13cac6bc64dc636d817673787ad7ce7</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.04.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.04.028%26sid%3Dliteratum%253Aachs%26aulast%3DBoettcher%26aufirst%3DM.%26aulast%3DMcManus%26aufirst%3DM.%2BT.%26atitle%3DChoosing%2520the%2520Right%2520Tool%2520for%2520the%2520Job%253A%2520Rnai%252C%2520Talen%252C%2520or%2520Crispr%26jtitle%3DMol.%2520Cell%26date%3D2015%26volume%3D58%26spage%3D575%26epage%3D585%26doi%3D10.1016%2Fj.molcel.2015.04.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jinek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chylinski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fonfara, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doudna, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charpentier, E.</span></span> <span> </span><span class="NLM_article-title">A Programmable Dual-Rna-Guided DNA Endonuclease in Adaptive Bacterial Immunity</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>337</i></span>,  <span class="NLM_fpage">816</span>– <span class="NLM_lpage">821</span>, <span class="refDoi"> DOI: 10.1126/science.1225829</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fscience.1225829" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=22745249" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFOqsb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=337&publication_year=2012&pages=816-821&author=M.+Jinekauthor=K.+Chylinskiauthor=I.+Fonfaraauthor=M.+Hauerauthor=J.+A.+Doudnaauthor=E.+Charpentier&title=A+Programmable+Dual-Rna-Guided+DNA+Endonuclease+in+Adaptive+Bacterial+Immunity&doi=10.1126%2Fscience.1225829"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity</span></div><div class="casAuthors">Jinek, Martin; Chylinski, Krzysztof; Fonfara, Ines; Hauer, Michael; Doudna, Jennifer A.; Charpentier, Emmanuelle</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">337</span>
        (<span class="NLM_cas:issue">6096</span>),
    <span class="NLM_cas:pages">816-821</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-assocd. (Cas) systems provide bacteria and archaea with adaptive immunity against viruses and plasmids by using CRISPR RNAs (crRNAs) to guide the silencing of invading nucleic acids.  We show here that in a subset of these systems, the mature crRNA that is base-paired to trans-activating crRNA (tracrRNA) forms a two-RNA structure that directs the CRISPR-assocd. protein Cas9 to introduce double-stranded (ds) breaks in target DNA.  At sites complementary to the crRNA-guide sequence, the Cas9 HNH nuclease domain cleaves the complementary strand, whereas the Cas9 RuvC-like domain cleaves the noncomplementary strand.  The dual-tracrRNA:crRNA, when engineered as a single RNA chimera, also directs sequence-specific Cas9 dsDNA cleavage.  Our study reveals a family of endonucleases that use dual-RNAs for site-specific DNA cleavage and highlights the potential to exploit the system for RNA-programmable genome editing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMRnSchW9tkbVg90H21EOLACvtfcHk0li3L-lcxIS7DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFOqsb3L&md5=8ad6d5a9839f62f75a1f805a2c1c83f4</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1126%2Fscience.1225829&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1225829%26sid%3Dliteratum%253Aachs%26aulast%3DJinek%26aufirst%3DM.%26aulast%3DChylinski%26aufirst%3DK.%26aulast%3DFonfara%26aufirst%3DI.%26aulast%3DHauer%26aufirst%3DM.%26aulast%3DDoudna%26aufirst%3DJ.%2BA.%26aulast%3DCharpentier%26aufirst%3DE.%26atitle%3DA%2520Programmable%2520Dual-Rna-Guided%2520DNA%2520Endonuclease%2520in%2520Adaptive%2520Bacterial%2520Immunity%26jtitle%3DScience%26date%3D2012%26volume%3D337%26spage%3D816%26epage%3D821%26doi%3D10.1126%2Fscience.1225829" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haney, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgens, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jovicic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couthouis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ousey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bieri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hertz, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tessier-Lavigne, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichida, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassik, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gitler, A. D.</span></span> <span> </span><span class="NLM_article-title">Crispr-Cas9 Screens in Human Cells and Primary Neurons Identify Modifiers of C9orf72 Dipeptide-Repeat-Protein Toxicity</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">612</span>, <span class="refDoi"> DOI: 10.1038/s41588-018-0070-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41588-018-0070-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=29507424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltVyit7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2018&pages=603-612&author=N.+J.+Kramerauthor=M.+S.+Haneyauthor=D.+W.+Morgensauthor=A.+Jovicicauthor=J.+Couthouisauthor=A.+Liauthor=J.+Ouseyauthor=R.+Maauthor=G.+Bieriauthor=C.+K.+Tsuiauthor=Y.+Shiauthor=N.+T.+Hertzauthor=M.+Tessier-Lavigneauthor=J.+K.+Ichidaauthor=M.+C.+Bassikauthor=A.+D.+Gitler&title=Crispr-Cas9+Screens+in+Human+Cells+and+Primary+Neurons+Identify+Modifiers+of+C9orf72+Dipeptide-Repeat-Protein+Toxicity&doi=10.1038%2Fs41588-018-0070-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">CRISPR-Cas9 screens in human cells and primary neurons identify modifiers of C9ORF72 dipeptide-repeat-protein toxicity</span></div><div class="casAuthors">Kramer, Nicholas J.; Haney, Michael S.; Morgens, David W.; Jovicic, Ana; Couthouis, Julien; Li, Amy; Ousey, James; Ma, Rosanna; Bieri, Gregor; Tsui, C. Kimberly; Shi, Yingxiao; Hertz, Nicholas T.; Tessier-Lavigne, Marc; Ichida, Justin K.; Bassik, Michael C.; Gitler, Aaron D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">603-612</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Hexanucleotide-repeat expansions in the C9ORF72 gene are the most common cause of amyotrophic lateral sclerosis and frontotemporal dementia (c9ALS/FTD).  The nucleotide-repeat expansions are translated into dipeptide-repeat (DPR) proteins, which are aggregation prone and may contribute to neurodegeneration.  We used the CRISPR-Cas9 system to perform genome-wide gene-knockout screens for suppressors and enhancers of C9ORF72 DPR toxicity in human cells.  We validated hits by performing secondary CRISPR-Cas9 screens in primary mouse neurons.  We uncovered potent modifiers of DPR toxicity whose gene products function in nucleocytoplasmic transport, the endoplasmic reticulum (ER), proteasome, RNA-processing pathways, and chromatin modification.  One modifier, TMX2, modulated the ER-stress signature elicited by C9ORF72 DPRs in neurons and improved survival of human induced motor neurons from patients with C9ORF72 ALS.  Together, our results demonstrate the promise of CRISPR-Cas9 screens in defining mechanisms of neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpavbRLvjRmTrVg90H21EOLACvtfcHk0li3L-lcxIS7DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltVyit7c%253D&md5=e043408638fab13b2fb6939961ad00ed</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1038%2Fs41588-018-0070-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41588-018-0070-7%26sid%3Dliteratum%253Aachs%26aulast%3DKramer%26aufirst%3DN.%2BJ.%26aulast%3DHaney%26aufirst%3DM.%2BS.%26aulast%3DMorgens%26aufirst%3DD.%2BW.%26aulast%3DJovicic%26aufirst%3DA.%26aulast%3DCouthouis%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DA.%26aulast%3DOusey%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DR.%26aulast%3DBieri%26aufirst%3DG.%26aulast%3DTsui%26aufirst%3DC.%2BK.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DHertz%26aufirst%3DN.%2BT.%26aulast%3DTessier-Lavigne%26aufirst%3DM.%26aulast%3DIchida%26aufirst%3DJ.%2BK.%26aulast%3DBassik%26aufirst%3DM.%2BC.%26aulast%3DGitler%26aufirst%3DA.%2BD.%26atitle%3DCrispr-Cas9%2520Screens%2520in%2520Human%2520Cells%2520and%2520Primary%2520Neurons%2520Identify%2520Modifiers%2520of%2520C9orf72%2520Dipeptide-Repeat-Protein%2520Toxicity%26jtitle%3DNat.%2520Genet.%26date%3D2018%26volume%3D50%26spage%3D603%26epage%3D612%26doi%3D10.1038%2Fs41588-018-0070-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keam, S. J.</span></span> <span> </span><span class="NLM_article-title">Inotersen: First Global Approval</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">1371</span>– <span class="NLM_lpage">1376</span>, <span class="refDoi"> DOI: 10.1007/s40265-018-0968-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1007%2Fs40265-018-0968-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30120737" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFCku7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2018&pages=1371-1376&author=S.+J.+Keam&title=Inotersen%3A+First+Global+Approval&doi=10.1007%2Fs40265-018-0968-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Inotersen: First Global Approval</span></div><div class="casAuthors">Keam, Susan J.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1371-1376</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Ionis Pharmaceuticals and Akcea Therapeutics have developed inotersen (Tegsedi), an antisense oligonucleotide inhibitor of mutant and wild-type human transthyretin (TTR), for the treatment of hereditary transthyretin amyloidosis (hATTR).  Mutation of the TTR gene results in accumulation of TTR protein fragments as amyloid deposits throughout the organs in patients with hATTR, including the peripheral nervous system and the heart.  Treatment with inotersen, which selectively binds to TTR mRNA, prevents the synthesis of TTR protein in the liver, thus reducing further amyloid deposition throughout the body.  S.c. administration of inotersen significantly reduced neurol. progression and improved health-related quality of life in patients with hATTR and polyneuropathy in a phase III trial.  Based on these results, inotersen was recently approved in the EU for the treatment of stage 1 or 2 polyneuropathy in adult patients with hATTR and is under evaluation in the USA and Canada for a similar indication.  This article summarizes the milestones in the development of inotersen leading to this first approval.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxb4Bx5DMw4rVg90H21EOLACvtfcHk0lilSIDwVphU7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFCku7bE&md5=aecfa49c97a998da9692c3900bf14ca8</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1007%2Fs40265-018-0968-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-018-0968-5%26sid%3Dliteratum%253Aachs%26aulast%3DKeam%26aufirst%3DS.%2BJ.%26atitle%3DInotersen%253A%2520First%2520Global%2520Approval%26jtitle%3DDrugs%26date%3D2018%26volume%3D78%26spage%3D1371%26epage%3D1376%26doi%3D10.1007%2Fs40265-018-0968-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span> <span> </span><span class="NLM_article-title">Clinical Advances of Sirna Therapeutics</span>. <i>J. Gene Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>, <span class="NLM_elocation-id">e3097</span> <span class="refDoi"> DOI: 10.1002/jgm.3097</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1002%2Fjgm.3097" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=31069898" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=B.+Huauthor=Y.+Wengauthor=X.+H.+Xiaauthor=X.+J.+Liangauthor=Y.+Huang&title=Clinical+Advances+of+Sirna+Therapeutics&doi=10.1002%2Fjgm.3097"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1002%2Fjgm.3097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjgm.3097%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DB.%26aulast%3DWeng%26aufirst%3DY.%26aulast%3DXia%26aufirst%3DX.%2BH.%26aulast%3DLiang%26aufirst%3DX.%2BJ.%26aulast%3DHuang%26aufirst%3DY.%26atitle%3DClinical%2520Advances%2520of%2520Sirna%2520Therapeutics%26jtitle%3DJ.%2520Gene%2520Med.%26date%3D2019%26doi%3D10.1002%2Fjgm.3097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, C. F.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Antisense Oligonucleotides Are Coming of Age</span>. <i>Annu. Rev. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">321</span>, <span class="refDoi"> DOI: 10.1146/annurev-med-041217-010829</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1146%2Fannurev-med-041217-010829" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30691367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvF2lsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2019&pages=307-321&author=C.+F.+Bennett&title=Therapeutic+Antisense+Oligonucleotides+Are+Coming+of+Age&doi=10.1146%2Fannurev-med-041217-010829"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic Antisense Oligonucleotides Are Coming of Age</span></div><div class="casAuthors">Bennett, C. Frank</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Medicine</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">307-321</span>CODEN:
                <span class="NLM_cas:coden">ARMCAH</span>;
        ISSN:<span class="NLM_cas:issn">0066-4219</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The first published description of therapeutic applications of antisense oligonucleotide (ASO) technol. occurred in the late 1970s and was followed by the founding of com. companies focused on developing antisense therapeutics in the late 1980s.  Since the late 1980s, there has been steady progress in improving the technol. platform, taking advantage of advances in oligonucleotide chem. and formulations as well as increased understanding of the distribution and safety of ASOs.  There are several approved ASO drugs and a broad pipeline in development.  In addn., advances in understanding human disease, including the genetic basis for most monogenic diseases and the availability of the full human genome sequence, have created numerous therapeutic applications for the technol.  I summarize the state of the technol. and highlight how advances in the technol. position ASOs to be an important contributor to future medicines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQKzgrxHUp1LVg90H21EOLACvtfcHk0lilSIDwVphU7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvF2lsbs%253D&md5=6c7f4a07724b43b769c8014ca3108e6a</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1146%2Fannurev-med-041217-010829&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-med-041217-010829%26sid%3Dliteratum%253Aachs%26aulast%3DBennett%26aufirst%3DC.%2BF.%26atitle%3DTherapeutic%2520Antisense%2520Oligonucleotides%2520Are%2520Coming%2520of%2520Age%26jtitle%3DAnnu.%2520Rev.%2520Med.%26date%3D2019%26volume%3D70%26spage%3D307%26epage%3D321%26doi%3D10.1146%2Fannurev-med-041217-010829" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bostrom, J.</span></span> <span> </span><span class="NLM_article-title">Where Do Recent Small Molecule Clinical Development Candidates Come From?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9442</span>– <span class="NLM_lpage">9468</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00675</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00675" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFKhsb7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9442-9468&author=D.+G.+Brownauthor=J.+Bostrom&title=Where+Do+Recent+Small+Molecule+Clinical+Development+Candidates+Come+From%3F&doi=10.1021%2Facs.jmedchem.8b00675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Where Do Recent Small Molecule Clinical Development Candidates Come From?</span></div><div class="casAuthors">Brown, Dean G.; Bostrom, Jonas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9442-9468</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An anal. of 66 published clin. candidates from Journal of Medicinal Chem. has been conducted to shed light on which lead generation strategies are most frequently employed in identifying drug candidates.  The most frequent lead generation strategy (producing a drug candidate) was based on starting points derived from previously known compds. (43%) followed by random high throughput screening (29%).  The remainder of approaches included focused screening, structure-based drug design (SBDD), fragment-based lead generation (FBLG), and DNA-encoded library screening (DEL).  An anal. of physicochem. properties on the hit-to-clin. pairs shows an av. increase in mol. wt. (ΔMW = +85) but no change in lipophilicity (ΔclogP = -0.2), although exceptions are noted.  The majority (>50%) of clin. candidates were found to be structurally very different from their starting point and were more complex.  Finally, several reports of noncovalent scaffolds modified by a covalent warhead using SBDD approaches are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKVZOYYj87bLVg90H21EOLACvtfcHk0lilSIDwVphU7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFKhsb7F&md5=211341fa12156e9512650d3352893a5e</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00675%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DBostrom%26aufirst%3DJ.%26atitle%3DWhere%2520Do%2520Recent%2520Small%2520Molecule%2520Clinical%2520Development%2520Candidates%2520Come%2520From%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9442%26epage%3D9468%26doi%3D10.1021%2Facs.jmedchem.8b00675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (Pains) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28Pains%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lhidPk1fotQEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528Pains%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wager, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalobos, A.</span></span> <span> </span><span class="NLM_article-title">Moving Beyond Rules: The Development of a Central Nervous System Multiparameter Optimization (Cns Mpo) Approach to Enable Alignment of Druglike Properties</span>. <i>ACS Chem. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">449</span>, <span class="refDoi"> DOI: 10.1021/cn100008c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cn100008c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvVejtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=435-449&author=T.+T.+Wagerauthor=X.+Houauthor=P.+R.+Verhoestauthor=A.+Villalobos&title=Moving+Beyond+Rules%3A+The+Development+of+a+Central+Nervous+System+Multiparameter+Optimization+%28Cns+Mpo%29+Approach+to+Enable+Alignment+of+Druglike+Properties&doi=10.1021%2Fcn100008c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Moving beyond Rules: The Development of a Central Nervous System Multiparameter Optimization (CNS MPO) Approach To Enable Alignment of Druglike Properties</span></div><div class="casAuthors">Wager, Travis T.; Hou, Xinjun; Verhoest, Patrick R.; Villalobos, Anabella</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">435-449</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The interplay among commonly used physicochem. properties in drug design was examd. and utilized to create a prospective design tool focused on the alignment of key druglike attributes.  Using a set of six physicochem. parameters ((a) lipophilicity, calcd. partition coeff. (ClogP); (b) calcd. distribution coeff. at pH = 7.4 (ClogD); (c) mol. wt. (MW); (d) topol. polar surface area (TPSA); (e) no. of hydrogen bond donors (HBD); (f) most basic center (pKa)), a druglikeness central nervous system multiparameter optimization (CNS MPO) algorithm was built and applied to a set of marketed CNS drugs (N = 119) and Pfizer CNS candidates (N = 108), as well as to a large diversity set of Pfizer proprietary compds. (N = 11 303).  The novel CNS MPO algorithm showed that 74% of marketed CNS drugs displayed a high CNS MPO score (MPO desirability score ≥ 4, using a scale of 0-6), in comparison to 60% of the Pfizer CNS candidates.  This anal. suggests that this algorithm could potentially be used to identify compds. with a higher probability of successfully testing hypotheses in the clinic.  In addn., a relationship between an increasing CNS MPO score and alignment of key in vitro attributes of drug discovery (favorable permeability, P-glycoprotein (P-gp) efflux, metabolic stability, and safety) was seen in the marketed CNS drug set, the Pfizer candidate set, and the Pfizer proprietary diversity set.  The CNS MPO scoring function offers advantages over hard cutoffs or utilization of single parameters to optimize structure-activity relationships (SAR) by expanding medicinal chem. design space through a holistic assessment approach.  Based on six physicochem. properties commonly used by medicinal chemists, the CNS MPO function may be used prospectively at the design stage to accelerate the identification of compds. with increased probability of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB3Mvkgm_X5bVg90H21EOLACvtfcHk0lhidPk1fotQEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvVejtr8%253D&md5=e94998f65a420be6ee4dec4a987cb983</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1021%2Fcn100008c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcn100008c%26sid%3Dliteratum%253Aachs%26aulast%3DWager%26aufirst%3DT.%2BT.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DVillalobos%26aufirst%3DA.%26atitle%3DMoving%2520Beyond%2520Rules%253A%2520The%2520Development%2520of%2520a%2520Central%2520Nervous%2520System%2520Multiparameter%2520Optimization%2520%2528Cns%2520Mpo%2529%2520Approach%2520to%2520Enable%2520Alignment%2520of%2520Druglike%2520Properties%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2010%26volume%3D1%26spage%3D435%26epage%3D449%26doi%3D10.1021%2Fcn100008c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prunotto, M.</span></span> <span> </span><span class="NLM_article-title">Opportunities and Challenges in Phenotypic Drug Discovery: An Industry Perspective</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">543</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnrd.2017.111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28685762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFelsLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=531-543&author=J.+G.+Moffatauthor=F.+Vincentauthor=J.+A.+Leeauthor=J.+Ederauthor=M.+Prunotto&title=Opportunities+and+Challenges+in+Phenotypic+Drug+Discovery%3A+An+Industry+Perspective&doi=10.1038%2Fnrd.2017.111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Opportunities and challenges in phenotypic drug discovery: an industry perspective</span></div><div class="casAuthors">Moffat, John G.; Vincent, Fabien; Lee, Jonathan A.; Eder, Jorg; Prunotto, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">531-543</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Phenotypic drug discovery (PDD) approaches do not rely on knowledge of the identity of a specific drug target or a hypothesis about its role in disease, in contrast to the target-based strategies that have been widely used in the pharmaceutical industry in the past three decades.  However, in recent years, there has been a resurgence in interest in PDD approaches based on their potential to address the incompletely understood complexity of diseases and their promise of delivering first-in-class drugs, as well as major advances in the tools for cell-based phenotypic screening.  Nevertheless, PDD approaches also have considerable challenges, such as hit validation and target deconvolution.  This article focuses on the lessons learned by researchers engaged in PDD in the pharmaceutical industry and considers the impact of 'omics' knowledge in defining a cellular disease phenotype in the era of precision medicine, introducing the concept of a chain of translatability.  We particularly aim to identify features and areas in which PDD can best deliver value to drug discovery portfolios and can contribute to the identification and the development of novel medicines, and to illustrate the challenges and uncertainties that are assocd. with PDD in order to help set realistic expectations with regard to its benefits and costs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGrD5TsU8_kLVg90H21EOLACvtfcHk0lhidPk1fotQEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFelsLrJ&md5=d83196715c879a81d9eabcf544e303db</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.111%26sid%3Dliteratum%253Aachs%26aulast%3DMoffat%26aufirst%3DJ.%2BG.%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DLee%26aufirst%3DJ.%2BA.%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DPrunotto%26aufirst%3DM.%26atitle%3DOpportunities%2520and%2520Challenges%2520in%2520Phenotypic%2520Drug%2520Discovery%253A%2520An%2520Industry%2520Perspective%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D531%26epage%3D543%26doi%3D10.1038%2Fnrd.2017.111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koutsoukas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simms, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirchmair, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bond, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitmore, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, A.</span></span> <span> </span><span class="NLM_article-title">From in Silico Target Prediction to Multi-Target Drug Design: Current Databases, Methods and Applications</span>. <i>J. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">2554</span>– <span class="NLM_lpage">2574</span>, <span class="refDoi"> DOI: 10.1016/j.jprot.2011.05.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.jprot.2011.05.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=21621023" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOjtrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2011&pages=2554-2574&author=A.+Koutsoukasauthor=B.+Simmsauthor=J.+Kirchmairauthor=P.+J.+Bondauthor=A.+V.+Whitmoreauthor=S.+Zimmerauthor=M.+P.+Youngauthor=J.+L.+Jenkinsauthor=M.+Glickauthor=R.+C.+Glenauthor=A.+Bender&title=From+in+Silico+Target+Prediction+to+Multi-Target+Drug+Design%3A+Current+Databases%2C+Methods+and+Applications&doi=10.1016%2Fj.jprot.2011.05.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">From in silico target prediction to multi-target drug design: Current databases, methods and applications</span></div><div class="casAuthors">Koutsoukas, Alexios; Simms, Benjamin; Kirchmair, Johannes; Bond, Peter J.; Whitmore, Alan V.; Zimmer, Steven; Young, Malcolm P.; Jenkins, Jeremy L.; Glick, Meir; Glen, Robert C.; Bender, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteomics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2554-2574</span>CODEN:
                <span class="NLM_cas:coden">JPORFQ</span>;
        ISSN:<span class="NLM_cas:issn">1874-3919</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Given the tremendous growth of bioactivity databases, the use of computational tools to predict protein targets of small mols. has been gaining importance in recent years.  Applications span a wide range, from the designed polypharmacol.' of compds. to mode-of-action anal.  In this review, we firstly survey databases that can be used for ligand-based target prediction and which have grown tremendously in size in the past.  We furthermore outline methods for target prediction that exist, both based on the knowledge of bioactivities from the ligand side and methods that can be applied in situations when a protein structure is known.  Applications of successful in silico target identification attempts are discussed in detail, which were based partly or in whole on computational target predictions in the first instance.  This includes the authors' own experience using target prediction tools, in this case considering phenotypic antibacterial screens and the anal. of high-throughput screening data.  Finally, we will conclude with the prospective application of databases to not only predict, retrospectively, the protein targets of a small mol., but also how to design ligands with desired polypharmacol. in a prospective manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQAhthzL8wiLVg90H21EOLACvtfcHk0lgkk1gPl_ssfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOjtrfE&md5=3fd699639ef3faf347537ef400030dd9</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1016%2Fj.jprot.2011.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jprot.2011.05.011%26sid%3Dliteratum%253Aachs%26aulast%3DKoutsoukas%26aufirst%3DA.%26aulast%3DSimms%26aufirst%3DB.%26aulast%3DKirchmair%26aufirst%3DJ.%26aulast%3DBond%26aufirst%3DP.%2BJ.%26aulast%3DWhitmore%26aufirst%3DA.%2BV.%26aulast%3DZimmer%26aufirst%3DS.%26aulast%3DYoung%26aufirst%3DM.%2BP.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DBender%26aufirst%3DA.%26atitle%3DFrom%2520in%2520Silico%2520Target%2520Prediction%2520to%2520Multi-Target%2520Drug%2520Design%253A%2520Current%2520Databases%252C%2520Methods%2520and%2520Applications%26jtitle%3DJ.%2520Proteomics%26date%3D2011%26volume%3D74%26spage%3D2554%26epage%3D2574%26doi%3D10.1016%2Fj.jprot.2011.05.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dancik, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span> <span> </span><span class="NLM_article-title">Target Identification and Mechanism of Action in Chemical Biology and Drug Discovery</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">232</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnchembio.1199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23508189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFWmsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=232-240&author=M.+Schenoneauthor=V.+Dancikauthor=B.+K.+Wagnerauthor=P.+A.+Clemons&title=Target+Identification+and+Mechanism+of+Action+in+Chemical+Biology+and+Drug+Discovery&doi=10.1038%2Fnchembio.1199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Target identification and mechanism of action in chemical biology and drug discovery</span></div><div class="casAuthors">Schenone, Monica; Dancik, Vlado; Wagner, Bridget K.; Clemons, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">232-240</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Target-identification and mechanism-of-action studies have important roles in small-mol. probe and drug discovery.  Biol. and technol. advances have resulted in the increasing use of cell-based assays to discover new biol. active small mols.  Such studies allow small-mol. action to be tested in a more disease-relevant setting at the outset, but they require follow-up studies to det. the precise protein target or targets responsible for the obsd. phenotype.  Target identification can be approached by direct biochem. methods, genetic interactions or computational inference.  In many cases, however, combinations of approaches may be required to fully characterize on-target and off-target effects and to understand mechanisms of small-mol. action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMV_Mv_6-O-rVg90H21EOLACvtfcHk0lgkk1gPl_ssfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFWmsLs%253D&md5=7aa5556d8dfbf259ab7e60fda1d8fdc2</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1199%26sid%3Dliteratum%253Aachs%26aulast%3DSchenone%26aufirst%3DM.%26aulast%3DDancik%26aufirst%3DV.%26aulast%3DWagner%26aufirst%3DB.%2BK.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26atitle%3DTarget%2520Identification%2520and%2520Mechanism%2520of%2520Action%2520in%2520Chemical%2520Biology%2520and%2520Drug%2520Discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D232%26epage%3D240%26doi%3D10.1038%2Fnchembio.1199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corsello, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peck, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natoli, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gould, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tubelli, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asiedu, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahr, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirschman, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donahue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Julian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wadden, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liberzon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toder, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orzechowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enache, O. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccioni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vrcic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosains, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davison, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ardlie, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenside, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read-Button, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronco, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doench, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bittker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span> <span> </span><span class="NLM_article-title">A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>171</i></span>,  <span class="NLM_fpage">1437</span>– <span class="NLM_lpage">1452</span>, e1417 <span class="refDoi"> DOI: 10.1016/j.cell.2017.10.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.cell.2017.10.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=29195078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWmsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452&author=A.+Subramanianauthor=R.+Narayanauthor=S.+M.+Corselloauthor=D.+D.+Peckauthor=T.+E.+Natoliauthor=X.+Luauthor=J.+Gouldauthor=J.+F.+Davisauthor=A.+A.+Tubelliauthor=J.+K.+Asieduauthor=D.+L.+Lahrauthor=J.+E.+Hirschmanauthor=Z.+Liuauthor=M.+Donahueauthor=B.+Julianauthor=M.+Khanauthor=D.+Waddenauthor=I.+C.+Smithauthor=D.+Lamauthor=A.+Liberzonauthor=C.+Toderauthor=M.+Bagulauthor=M.+Orzechowskiauthor=O.+M.+Enacheauthor=F.+Piccioniauthor=S.+A.+Johnsonauthor=N.+J.+Lyonsauthor=A.+H.+Bergerauthor=A.+F.+Shamjiauthor=A.+N.+Brooksauthor=A.+Vrcicauthor=C.+Flynnauthor=J.+Rosainsauthor=D.+Y.+Takedaauthor=R.+Huauthor=D.+Davisonauthor=J.+Lambauthor=K.+Ardlieauthor=L.+Hogstromauthor=P.+Greensideauthor=N.+S.+Grayauthor=P.+A.+Clemonsauthor=S.+Silverauthor=X.+Wuauthor=W.+N.+Zhaoauthor=W.+Read-Buttonauthor=X.+Wuauthor=S.+J.+Haggartyauthor=L.+V.+Roncoauthor=J.+S.+Boehmauthor=S.+L.+Schreiberauthor=J.+G.+Doenchauthor=J.+A.+Bittkerauthor=D.+E.+Rootauthor=B.+Wongauthor=T.+R.+Golub&title=A+Next+Generation+Connectivity+Map%3A+L1000+Platform+and+the+First+1%2C000%2C000+Profiles&doi=10.1016%2Fj.cell.2017.10.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">A next generation connectivity map: L1000 platform and the first 1,000,000 profiles</span></div><div class="casAuthors">Subramanian, Aravind; Narayan, Rajiv; Corsello, Steven M.; Peck, David D.; Natoli, Ted E.; Lu, Xiaodong; Gould, Joshua; Davis, John F.; Tubelli, Andrew A.; Asiedu, Jacob K.; Lahr, David L.; Hirschman, Jodi E.; Liu, Zihan; Donahue, Melanie; Julian, Bina; Khan, Mariya; Wadden, David; Smith, Ian C.; Lam, Daniel; Liberzon, Arthur; Toder, Courtney; Bagul, Mukta; Orzechowski, Marek; Enache, Oana M.; Piccioni, Federica; Johnson, Sarah A.; Lyons, Nicholas J.; Berger, Alice H.; Shamji, Alykhan F.; Brooks, Angela N.; Vrcic, Anita; Flynn, Corey; Rosains, Jacqueline; Takeda, David Y.; Hu, Roger; Davison, Desiree; Lamb, Justin; Ardlie, Kristin; Hogstrom, Larson; Greenside, Peyton; Gray, Nathanael S.; Clemons, Paul A.; Silver, Serena; Wu, Xiaoyun; Zhao, Wen-Ning; Read-Button, Willis; Wu, Xiaohua; Haggarty, Stephen J.; Ronco, Lucienne V.; Boehm, Jesse S.; Schreiber, Stuart L.; Doench, John G.; Bittker, Joshua A.; Root, David E.; Wong, Bang; Golub, Todd R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1437-1452.e17</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We previously piloted the concept of a Connectivity Map (CMap), whereby genes, drugs, and disease states are connected by virtue of common gene-expression signatures.  Here, we report more than a 1000-fold scale-up of the CMap as part of the NIH LINCS Consortium, made possible by a new, low-cost, high-throughput reduced representation expression profiling method that we term L1000.  We show that L1000 is highly reproducible, comparable to RNA sequencing, and suitable for computational inference of the expression levels of 81% of non-measured transcripts.  We further show that the expanded CMap can be used to discover mechanism of action of small mols., functionally annotate genetic variants of disease genes, and inform clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIWrWy5IxY-rVg90H21EOLACvtfcHk0lgkk1gPl_ssfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWmsbrE&md5=d2b13f2b80fe94021ac03f6a8c859a4a</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.10.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.10.049%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DNarayan%26aufirst%3DR.%26aulast%3DCorsello%26aufirst%3DS.%2BM.%26aulast%3DPeck%26aufirst%3DD.%2BD.%26aulast%3DNatoli%26aufirst%3DT.%2BE.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DGould%26aufirst%3DJ.%26aulast%3DDavis%26aufirst%3DJ.%2BF.%26aulast%3DTubelli%26aufirst%3DA.%2BA.%26aulast%3DAsiedu%26aufirst%3DJ.%2BK.%26aulast%3DLahr%26aufirst%3DD.%2BL.%26aulast%3DHirschman%26aufirst%3DJ.%2BE.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DDonahue%26aufirst%3DM.%26aulast%3DJulian%26aufirst%3DB.%26aulast%3DKhan%26aufirst%3DM.%26aulast%3DWadden%26aufirst%3DD.%26aulast%3DSmith%26aufirst%3DI.%2BC.%26aulast%3DLam%26aufirst%3DD.%26aulast%3DLiberzon%26aufirst%3DA.%26aulast%3DToder%26aufirst%3DC.%26aulast%3DBagul%26aufirst%3DM.%26aulast%3DOrzechowski%26aufirst%3DM.%26aulast%3DEnache%26aufirst%3DO.%2BM.%26aulast%3DPiccioni%26aufirst%3DF.%26aulast%3DJohnson%26aufirst%3DS.%2BA.%26aulast%3DLyons%26aufirst%3DN.%2BJ.%26aulast%3DBerger%26aufirst%3DA.%2BH.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DBrooks%26aufirst%3DA.%2BN.%26aulast%3DVrcic%26aufirst%3DA.%26aulast%3DFlynn%26aufirst%3DC.%26aulast%3DRosains%26aufirst%3DJ.%26aulast%3DTakeda%26aufirst%3DD.%2BY.%26aulast%3DHu%26aufirst%3DR.%26aulast%3DDavison%26aufirst%3DD.%26aulast%3DLamb%26aufirst%3DJ.%26aulast%3DArdlie%26aufirst%3DK.%26aulast%3DHogstrom%26aufirst%3DL.%26aulast%3DGreenside%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DSilver%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DW.%2BN.%26aulast%3DRead-Button%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DRonco%26aufirst%3DL.%2BV.%26aulast%3DBoehm%26aufirst%3DJ.%2BS.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DDoench%26aufirst%3DJ.%2BG.%26aulast%3DBittker%26aufirst%3DJ.%2BA.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DWong%26aufirst%3DB.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26atitle%3DA%2520Next%2520Generation%2520Connectivity%2520Map%253A%2520L1000%2520Platform%2520and%2520the%2520First%25201%252C000%252C000%2520Profiles%26jtitle%3DCell%26date%3D2017%26volume%3D171%26spage%3D1437%26epage%3D1452%26doi%3D10.1016%2Fj.cell.2017.10.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cong, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. M.</span></span> <span> </span><span class="NLM_article-title">Chemical Genetics-Based Target Identification in Drug Discovery</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010611-134639</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1146%2Fannurev-pharmtox-010611-134639" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=21819237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC38XjsV2ntro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=57-78&author=F.+Congauthor=A.+K.+Cheungauthor=S.+M.+Huang&title=Chemical+Genetics-Based+Target+Identification+in+Drug+Discovery&doi=10.1146%2Fannurev-pharmtox-010611-134639"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical genetics-based target identification in drug discovery</span></div><div class="casAuthors">Cong, Feng; Cheung, Atwood K.; Huang, Shih-Min A.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">57-78</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Identification of novel, validated targets remains a top priority in modern drug discovery.  Chem. genetics represents a powerful approach to the discovery of new targets.  Unlike the traditional target-based screen that relies on a predefined, sometimes poorly validated target, a chem. genetics-based phenotypic screen probes the entire mol. signaling pathway in an efficient and unbiased manner for the most drug-sensitive node.  The most significant obstacle assocd. with this approach is identification of the efficacy targets of small-mol. probes.  The huge potential of chem. genetics cannot be realized without the establishment of reliable mechanisms for target identification.  In this article, we describe each essential element of the chem. genetics process, discuss common challenges that the field is facing, and critically review various biochem. and genetics approaches recently developed for target deconvolution.  We also attempt to summarize lessons that we have collectively learned and provide a practical perspective to facilitate the advancement of chem. genetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5yremnJyOOLVg90H21EOLACvtfcHk0lhs-Li2w73oTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjsV2ntro%253D&md5=97752e1589da40955b89293512f6303e</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010611-134639&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010611-134639%26sid%3Dliteratum%253Aachs%26aulast%3DCong%26aufirst%3DF.%26aulast%3DCheung%26aufirst%3DA.%2BK.%26aulast%3DHuang%26aufirst%3DS.%2BM.%26atitle%3DChemical%2520Genetics-Based%2520Target%2520Identification%2520in%2520Drug%2520Discovery%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2012%26volume%3D52%26spage%3D57%26epage%3D78%26doi%3D10.1146%2Fannurev-pharmtox-010611-134639" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tay, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagar</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dautzenberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinz, M.</span></span> <span> </span><span class="NLM_article-title">Unique Microglia Recovery Population Revealed by Single-Cell Rnaseq Following Neurodegeneration</span>. <i>Acta Neuropathol. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">87</span>, <span class="refDoi"> DOI: 10.1186/s40478-018-0584-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1186%2Fs40478-018-0584-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30185219" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjt1Gnu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=87&author=T.+L.+Tayauthor=+Sagarauthor=J.+Dautzenbergauthor=D.+Grunauthor=M.+Prinz&title=Unique+Microglia+Recovery+Population+Revealed+by+Single-Cell+Rnaseq+Following+Neurodegeneration&doi=10.1186%2Fs40478-018-0584-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Unique microglia recovery population revealed by single-cell RNAseq following neurodegeneration</span></div><div class="casAuthors">Tay, Tuan Leng; Sagar; Dautzenberg, Jana; Gruen, Dominic; Prinz, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Acta Neuropathologica Communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">87/1-87/11</span>CODEN:
                <span class="NLM_cas:coden">ANCCC9</span>;
        ISSN:<span class="NLM_cas:issn">2051-5960</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Microglia are brain immune cells that constantly survey their environment to maintain homeostasis.  Enhanced microglial reactivity and proliferation are typical hallmarks of neurodegenerative diseases.  Whether specific disease-linked microglial subsets exist during the entire course of neurodegeneration, including the recovery phase, is currently unclear.  Taking a single-cell RNA-sequencing approach in a susceptibility gene-free model of nerve injury, we identified a microglial subpopulation that upon acute neurodegeneration shares a conserved gene regulatory profile compared to previously reported chronic and destructive neurodegeneration transgenic mouse models.  Our data also revealed rapid shifts in gene regulation that defined microglial subsets at peak and resoln. of neurodegeneration.  Finally, our discovery of a unique transient microglial subpopulation at the onset of recovery may provide novel targets for modulating microglia mediated restoration of brain health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5TIFKyIY_6LVg90H21EOLACvtfcHk0lhs-Li2w73oTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjt1Gnu7c%253D&md5=aeb726a790fa829d3a2b0cbc4ea9f20b</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1186%2Fs40478-018-0584-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs40478-018-0584-3%26sid%3Dliteratum%253Aachs%26aulast%3DTay%26aufirst%3DT.%2BL.%26aulast%3DSagar%26aulast%3DDautzenberg%26aufirst%3DJ.%26aulast%3DGrun%26aufirst%3DD.%26aulast%3DPrinz%26aufirst%3DM.%26atitle%3DUnique%2520Microglia%2520Recovery%2520Population%2520Revealed%2520by%2520Single-Cell%2520Rnaseq%2520Following%2520Neurodegeneration%26jtitle%3DActa%2520Neuropathol.%2520Commun.%26date%3D2018%26volume%3D6%26spage%3D87%26doi%3D10.1186%2Fs40478-018-0584-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deczkowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keren-Shaul, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colonna, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amit, I.</span></span> <span> </span><span class="NLM_article-title">Disease-Associated Microglia: A Universal Immune Sensor of Neurodegeneration</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>173</i></span>,  <span class="NLM_fpage">1073</span>– <span class="NLM_lpage">1081</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.cell.2018.05.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=29775591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpvVemuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=1073-1081&author=A.+Deczkowskaauthor=H.+Keren-Shaulauthor=A.+Weinerauthor=M.+Colonnaauthor=M.+Schwartzauthor=I.+Amit&title=Disease-Associated+Microglia%3A+A+Universal+Immune+Sensor+of+Neurodegeneration&doi=10.1016%2Fj.cell.2018.05.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Disease-Associated Microglia: A Universal Immune Sensor of Neurodegeneration</span></div><div class="casAuthors">Deczkowska, Aleksandra; Keren-Shaul, Hadas; Weiner, Assaf; Colonna, Marco; Schwartz, Michal; Amit, Ido</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">173</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1073-1081</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  A major challenge in the field of neurodegenerative diseases and brain aging is to identify the body's intrinsic mechanism that could sense the central nervous system (CNS) damage early and protect the brain from neurodegeneration.  Accumulating evidence suggests that disease-assocd. microglia (DAM), a recently identified subset of CNS resident macrophages found at sites of neurodegeneration, might play such a protective role.  Here, we propose that microglia are endowed with a dedicated sensory mechanism, which includes the Trem2 signaling pathway, to detect damage within the CNS in the form of neurodegeneration-assocd. mol. patterns (NAMPs).  Combining data from transcriptional anal. of DAM at single-cell level and from human genome-wide assocn. studies (GWASs), we discuss potential function of different DAM pathways in the diseased brain and outline how manipulating DAM may create new therapeutic opportunities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqusr5xqDAy-7Vg90H21EOLACvtfcHk0lhs-Li2w73oTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpvVemuro%253D&md5=1f10ff8be131c68e10df99f49af41c3c</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DDeczkowska%26aufirst%3DA.%26aulast%3DKeren-Shaul%26aufirst%3DH.%26aulast%3DWeiner%26aufirst%3DA.%26aulast%3DColonna%26aufirst%3DM.%26aulast%3DSchwartz%26aufirst%3DM.%26aulast%3DAmit%26aufirst%3DI.%26atitle%3DDisease-Associated%2520Microglia%253A%2520A%2520Universal%2520Immune%2520Sensor%2520of%2520Neurodegeneration%26jtitle%3DCell%26date%3D2018%26volume%3D173%26spage%3D1073%26epage%3D1081%26doi%3D10.1016%2Fj.cell.2018.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hammitzsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, F. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Mossawi, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Wit, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vecellio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wordsworth, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowness, P.</span></span> <span> </span><span class="NLM_article-title">Cbp30, a Selective Cbp/P300 Bromodomain Inhibitor, Suppresses Human Th17 Responses</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">10768</span>– <span class="NLM_lpage">10773</span>, <span class="refDoi"> DOI: 10.1073/pnas.1501956112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1073%2Fpnas.1501956112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26261308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlSit7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=10768-10773&author=A.+Hammitzschauthor=C.+Tallantauthor=O.+Fedorovauthor=A.+O%E2%80%99Mahonyauthor=P.+E.+Brennanauthor=D.+A.+Hayauthor=F.+O.+Martinezauthor=M.+H.+Al-Mossawiauthor=J.+de+Witauthor=M.+Vecellioauthor=C.+Wellsauthor=P.+Wordsworthauthor=S.+Mullerauthor=S.+Knappauthor=P.+Bowness&title=Cbp30%2C+a+Selective+Cbp%2FP300+Bromodomain+Inhibitor%2C+Suppresses+Human+Th17+Responses&doi=10.1073%2Fpnas.1501956112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">CBP30, a selective CBP/p300 bromodomain inhibitor, suppresses human Th17 responses</span></div><div class="casAuthors">Hammitzsch, Ariane; Tallant, Cynthia; Fedorov, Oleg; O'Mahony, Alison; Brennan, Paul E.; Hay, Duncan A.; Martinez, Fernando O.; Al-Mossawi, M. Hussein; de Wit, Jelle; Vecellio, Matteo; Wells, Christopher; Wordsworth, Paul; Muller, Susanne; Knapp, Stefan; Bowness, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">10768-10773</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Th17 responses are crit. to a variety of human autoimmune diseases, and therapeutic targeting with monoclonal antibodies against IL-17 and IL-23 has shown considerable promise.  Here, the authors report data to support selective bromodomain blockade of the transcriptional coactivators CBP (CREB binding protein) and p300 as an alternative approach to inhibit human Th17 responses.  The authors show that CBP30 has marked mol. specificity for the bromodomains of CBP and p300, compared with 43 other bromodomains.  In unbiased cellular testing on a diverse panel of cultured primary human cells, CBP30 reduced immune cell prodn. of IL-17A and other proinflammatory cytokines.  CBP30 also inhibited IL-17A secretion by Th17 cells from healthy donors and patients with ankylosing spondylitis and psoriatic arthritis.  Transcriptional profiling of human T cells after CBP30 treatment showed a much more restricted effect on gene expression than that obsd. with the pan-BET (bromo and extraterminal domain protein family) bromodomain inhibitor JQ1.  This selective targeting of the CBP/p300 bromodomain by CBP30 will potentially lead to fewer side effects than with the broadly acting epigenetic inhibitors currently in clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4fYjOUV2xW7Vg90H21EOLACvtfcHk0lhe4_f01NvFpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlSit7jI&md5=fb9de88ecdd6441628ae1455ac11b65c</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1501956112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1501956112%26sid%3Dliteratum%253Aachs%26aulast%3DHammitzsch%26aufirst%3DA.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DMartinez%26aufirst%3DF.%2BO.%26aulast%3DAl-Mossawi%26aufirst%3DM.%2BH.%26aulast%3Dde%2BWit%26aufirst%3DJ.%26aulast%3DVecellio%26aufirst%3DM.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DWordsworth%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBowness%26aufirst%3DP.%26atitle%3DCbp30%252C%2520a%2520Selective%2520Cbp%252FP300%2520Bromodomain%2520Inhibitor%252C%2520Suppresses%2520Human%2520Th17%2520Responses%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D10768%26epage%3D10773%26doi%3D10.1073%2Fpnas.1501956112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span> <span> </span><span class="NLM_article-title">Identifying Compound Efficacy Targets in Phenotypic Drug Discovery</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">82</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2015.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.drudis.2015.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26272035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCisrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=82-89&author=M.+Schirleauthor=J.+L.+Jenkins&title=Identifying+Compound+Efficacy+Targets+in+Phenotypic+Drug+Discovery&doi=10.1016%2Fj.drudis.2015.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying compound efficacy targets in phenotypic drug discovery</span></div><div class="casAuthors">Schirle, Markus; Jenkins, Jeremy L.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">82-89</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The identification of the efficacy target(s) for hits from phenotypic compd. screens remains a key step to progress compds. into drug development.  In addn. to efficacy targets, the characterization of epistatic proteins influencing compd. activity often facilitates the elucidation of the underlying mechanism of action; and, further, early detn. of off-targets that cause potentially unwanted secondary phenotypes helps in assessing potential liabilities.  This short review discusses the most important technologies currently available for characterizing the direct and indirect target space of bioactive compds. following phenotypic screening.  We present a comprehensive strategy employing complementary approaches to balance individual technol. strengths and weaknesses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNnxrH8t-we7Vg90H21EOLACvtfcHk0lhe4_f01NvFpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCisrvF&md5=b373d239d72058a1b826ae1550bb4693</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26atitle%3DIdentifying%2520Compound%2520Efficacy%2520Targets%2520in%2520Phenotypic%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D82%26epage%3D89%26doi%3D10.1016%2Fj.drudis.2015.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laraia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">Natural Product Inspired Compound Collections: Evolutionary Principle, Chemical Synthesis, Phenotypic Screening, and Target Identification</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2017.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.ddtec.2017.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28647090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC1cjhsFaiug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=75-82&author=L.+Laraiaauthor=H.+Waldmann&title=Natural+Product+Inspired+Compound+Collections%3A+Evolutionary+Principle%2C+Chemical+Synthesis%2C+Phenotypic+Screening%2C+and+Target+Identification&doi=10.1016%2Fj.ddtec.2017.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Natural product inspired compound collections: evolutionary principle, chemical synthesis, phenotypic screening, and target identification</span></div><div class="casAuthors">Laraia Luca; Waldmann Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">75-82</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Natural products have been an excellent and abundant source of therapeutics for many decades.  To expand on their success, and explore areas of chemical space not covered by biosynthesis, the synthesis of natural product-inspired compound collections has emerged as a viable strategy.  Herein we describe the principles behind biology-oriented synthesis and related approaches, the requirements for development of novel chemistry and how phenotypic screens are a very fruitful way to explore the bioactivity of compounds made using these approaches.  Finally, we summarize state-of-the-art techniques to identify the biological targets of any hits identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRF0mGW8Hm3edXdcedVHT5tfW6udTcc2eZnWqKPQ9nmw7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjhsFaiug%253D%253D&md5=e544a538d8be64334ab0afcac72af5ca</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2017.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2017.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DLaraia%26aufirst%3DL.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DNatural%2520Product%2520Inspired%2520Compound%2520Collections%253A%2520Evolutionary%2520Principle%252C%2520Chemical%2520Synthesis%252C%2520Phenotypic%2520Screening%252C%2520and%2520Target%2520Identification%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2017%26volume%3D23%26spage%3D75%26epage%3D82%26doi%3D10.1016%2Fj.ddtec.2017.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olek, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colinge, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span> <span> </span><span class="NLM_article-title">Proteome-Wide Drug and Metabolite Interaction Mapping by Thermal-Stability Profiling</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1055</span>– <span class="NLM_lpage">1057</span>, <span class="refDoi"> DOI: 10.1038/nmeth.3590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnmeth.3590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26389571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFersL%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=1055-1057&author=K.+V.+Huberauthor=K.+M.+Olekauthor=A.+C.+Mullerauthor=C.+S.+Tanauthor=K.+L.+Bennettauthor=J.+Colingeauthor=G.+Superti-Furga&title=Proteome-Wide+Drug+and+Metabolite+Interaction+Mapping+by+Thermal-Stability+Profiling&doi=10.1038%2Fnmeth.3590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome-wide drug and metabolite interaction mapping by thermal-stability profiling</span></div><div class="casAuthors">Huber, Kilian V. M.; Olek, Karin M.; Mueller, Andre C.; Tan, Chris Soon Heng; Bennett, Keiryn L.; Colinge, Jacques; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1055-1057</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thermal stabilization of proteins after ligand binding provides an efficient means to assess the binding of small mols. to proteins.  We show here that in combination with quant. mass spectrometry, the approach allows for the systematic survey of protein engagement by cellular metabolites and drugs.  We profiled the targets of the drugs methotrexate and (S)-crizotinib and the metabolite 2'3'-cGAMP in intact cells and identified the 2'3'-cGAMP cognate transmembrane receptor STING, involved in immune signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrow66ugdg1oLVg90H21EOLACvtfcHk0ljd2qLUUIHKeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFersL%252FI&md5=619a2c08c4f151f4e3ab8bd8fc45c2b6</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.3590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.3590%26sid%3Dliteratum%253Aachs%26aulast%3DHuber%26aufirst%3DK.%2BV.%26aulast%3DOlek%26aufirst%3DK.%2BM.%26aulast%3DMuller%26aufirst%3DA.%2BC.%26aulast%3DTan%26aufirst%3DC.%2BS.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DColinge%26aufirst%3DJ.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DProteome-Wide%2520Drug%2520and%2520Metabolite%2520Interaction%2520Mapping%2520by%2520Thermal-Stability%2520Profiling%26jtitle%3DNat.%2520Methods%26date%3D2015%26volume%3D12%26spage%3D1055%26epage%3D1057%26doi%3D10.1038%2Fnmeth.3590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref115"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref115'); return false;" data-citation="" class="refNumLink">115</a></strong><div class="NLM_citation" id="cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martinez
Molina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1126/science.1233606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fscience.1233606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23828940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC3sjotlOqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=84-87&author=D.+Martinez%0AMolinaauthor=R.+Jafariauthor=M.+Ignatushchenkoauthor=T.+Sekiauthor=E.+A.+Larssonauthor=C.+Danauthor=L.+Sreekumarauthor=Y.+Caoauthor=P.+Nordlund&title=Monitoring+Drug+Target+Engagement+in+Cells+and+Tissues+Using+the+Cellular+Thermal+Shift+Assay&doi=10.1126%2Fscience.1233606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring drug target engagement in cells and tissues using the cellular thermal shift assay</span></div><div class="casAuthors">Martinez Molina Daniel; Jafari Rozbeh; Ignatushchenko Marina; Seki Takahiro; Larsson E Andreas; Dan Chen; Sreekumar Lekshmy; Cao Yihai; Nordlund Par</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6141</span>),
    <span class="NLM_cas:pages">84-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The efficacy of therapeutics is dependent on a drug binding to its cognate target.  Optimization of target engagement by drugs in cells is often challenging, because drug binding cannot be monitored inside cells.  We have developed a method for evaluating drug binding to target proteins in cells and tissue samples.  This cellular thermal shift assay (CETSA) is based on the biophysical principle of ligand-induced thermal stabilization of target proteins.  Using this assay, we validated drug binding for a set of important clinical targets and monitored processes of drug transport and activation, off-target effects and drug resistance in cancer cell lines, as well as drug distribution in tissues.  CETSA is likely to become a valuable tool for the validation and optimization of drug target engagement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRRn5K_mF1rcrHHuJhbyicZfW6udTcc2ebg4aH8qQ9mzrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sjotlOqtA%253D%253D&md5=a73bb49adb0d4f445223ea6f74fe3c10</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1126%2Fscience.1233606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1233606%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DSreekumar%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DMonitoring%2520Drug%2520Target%2520Engagement%2520in%2520Cells%2520and%2520Tissues%2520Using%2520the%2520Cellular%2520Thermal%2520Shift%2520Assay%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D84%26epage%3D87%26doi%3D10.1126%2Fscience.1233606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref116"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref116'); return false;" data-citation="" class="refNumLink">116</a></strong><div class="NLM_citation" id="cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rees, S.</span></span> <span> </span><span class="NLM_article-title">The Promise of Open Innovation in Drug Discovery: An Industry Perspective</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1835</span>– <span class="NLM_lpage">1838</span>, <span class="refDoi"> DOI: 10.4155/fmc.15.125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.4155%2Ffmc.15.125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26420551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1KrsL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=1835-1838&author=S.+Rees&title=The+Promise+of+Open+Innovation+in+Drug+Discovery%3A+An+Industry+Perspective&doi=10.4155%2Ffmc.15.125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">The promise of open innovation in drug discovery: an industry perspective</span></div><div class="casAuthors">Rees, Steve</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1835-1838</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyo6y8yseLhrVg90H21EOLACvtfcHk0ljd2qLUUIHKeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1KrsL3F&md5=a3f5209a6733e563df179e85ebe5842a</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.4155%2Ffmc.15.125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.15.125%26sid%3Dliteratum%253Aachs%26aulast%3DRees%26aufirst%3DS.%26atitle%3DThe%2520Promise%2520of%2520Open%2520Innovation%2520in%2520Drug%2520Discovery%253A%2520An%2520Industry%2520Perspective%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2015%26volume%3D7%26spage%3D1835%26epage%3D1838%26doi%3D10.4155%2Ffmc.15.125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref117"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref117'); return false;" data-citation="" class="refNumLink">117</a></strong><div class="NLM_citation" id="cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plump, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ringel, M.</span></span> <span> </span><span class="NLM_article-title">Racing to Define Pharmaceutical R&D External Innovation Models</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">370</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.10.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.drudis.2014.10.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25448753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC2MzisFKhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=361-370&author=L.+Wangauthor=A.+Plumpauthor=M.+Ringel&title=Racing+to+Define+Pharmaceutical+R%26D+External+Innovation+Models&doi=10.1016%2Fj.drudis.2014.10.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Racing to define pharmaceutical R&D external innovation models</span></div><div class="casAuthors">Wang Liangsu; Plump Andrew; Ringel Michael</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">361-70</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The pharmaceutical industry continues to face fundamental challenges because of issues with research and development (R&D) productivity and rising customer expectations.  To lower R&D costs, move beyond me-too therapies, and create more transformative portfolios, pharmaceutical companies are actively capitalizing on external innovation through precompetitive collaboration with academia, cultivation of biotech start-ups, and proactive licensing and acquisitions.  Here, we review the varying innovation strategies used by pharmaceutical companies, compare and contrast these models, and identify the trends in external innovation.  We also discuss factors that influence these external innovation models and propose a preliminary set of metrics that could be used as leading indicators of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSeCY3Qkc1ezIYPsCkJIw00fW6udTcc2ebg4aH8qQ9mzrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MzisFKhtA%253D%253D&md5=8c13d04ba3a38750269162f27c5af721</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.10.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.10.008%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DPlump%26aufirst%3DA.%26aulast%3DRingel%26aufirst%3DM.%26atitle%3DRacing%2520to%2520Define%2520Pharmaceutical%2520R%2526D%2520External%2520Innovation%2520Models%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D361%26epage%3D370%26doi%3D10.1016%2Fj.drudis.2014.10.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114'],'ref115':['cit115'],'ref116':['cit116'],'ref117':['cit117']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 11 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Wayne E. Childers, Khaled M. Elokely, <span class="NLM_string-name hlFld-ContribAuthor">Magid Abou-Gharbia</span>. </span><span class="cited-content_cbyCitation_article-title">The Resurrection of Phenotypic Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (10)
                                     , 1820-1828. <a href="https://doi.org/10.1021/acsmedchemlett.0c00006" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00006%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DThe%252BResurrection%252Bof%252BPhenotypic%252BDrug%252BDiscovery%26aulast%3DChilders%26aufirst%3DWayne%2BE.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D03012020%26date%3D06032020%26date%3D24032020%26date%3D06032020%26volume%3D11%26issue%3D10%26spage%3D1820%26epage%3D1828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Courtney  MacMullen</span>, <span class="hlFld-ContribAuthor ">Ronald L.  Davis</span>. </span><span class="cited-content_cbyCitation_article-title">High-Throughput Phenotypic Assay for Compounds That Influence Mitochondrial Health Using iPSC-Derived Human Neurons. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2021,</strong> <em>26 </em>
                                    (6)
                                     , 811-822. <a href="https://doi.org/10.1177/24725552211000671" title="DOI URL">https://doi.org/10.1177/24725552211000671</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/24725552211000671&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F24725552211000671%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DHigh-Throughput%252BPhenotypic%252BAssay%252Bfor%252BCompounds%252BThat%252BInfluence%252BMitochondrial%252BHealth%252BUsing%252BiPSC-Derived%252BHuman%252BNeurons%26aulast%3DMacMullen%26aufirst%3DCourtney%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D6%26spage%3D811%26epage%3D822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ganiyu  Oboh</span>, <span class="hlFld-ContribAuthor ">Folasade L.  Oladun</span>, <span class="hlFld-ContribAuthor ">Ayokunle O.  Ademosun</span>, <span class="hlFld-ContribAuthor ">Opeyemi B.  Ogunsuyi</span>. </span><span class="cited-content_cbyCitation_article-title">Anticholinesterase activity and antioxidant properties of Heinsia crinita and Pterocarpus soyauxii in Drosophila melanogaster model. </span><span class="cited-content_cbyCitation_journal-name">Journal of Ayurveda and Integrative Medicine</span><span> <strong>2021,</strong> <em>12 </em>
                                    (2)
                                     , 254-260. <a href="https://doi.org/10.1016/j.jaim.2020.10.004" title="DOI URL">https://doi.org/10.1016/j.jaim.2020.10.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jaim.2020.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jaim.2020.10.004%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Ayurveda%2520and%2520Integrative%2520Medicine%26atitle%3DAnticholinesterase%252Bactivity%252Band%252Bantioxidant%252Bproperties%252Bof%252BHeinsia%252Bcrinita%252Band%252BPterocarpus%252Bsoyauxii%252Bin%252BDrosophila%252Bmelanogaster%252Bmodel%26aulast%3DOboh%26aufirst%3DGaniyu%26date%3D2021%26volume%3D12%26issue%3D2%26spage%3D254%26epage%3D260" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natesh  Singh</span>, <span class="hlFld-ContribAuthor ">Ludovic  Chaput</span>, <span class="hlFld-ContribAuthor ">Bruno O  Villoutreix</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual screening web servers: designing chemical probes and drug candidates in the cyberspace. </span><span class="cited-content_cbyCitation_journal-name">Briefings in Bioinformatics</span><span> <strong>2021,</strong> <em>22 </em>
                                    (2)
                                     , 1790-1818. <a href="https://doi.org/10.1093/bib/bbaa034" title="DOI URL">https://doi.org/10.1093/bib/bbaa034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bib/bbaa034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbib%2Fbbaa034%26sid%3Dliteratum%253Aachs%26jtitle%3DBriefings%2520in%2520Bioinformatics%26atitle%3DVirtual%252Bscreening%252Bweb%252Bservers%25253A%252Bdesigning%252Bchemical%252Bprobes%252Band%252Bdrug%252Bcandidates%252Bin%252Bthe%252Bcyberspace%26aulast%3DSingh%26aufirst%3DNatesh%26date%3D2021%26date%3D2020%26volume%3D22%26issue%3D2%26spage%3D1790%26epage%3D1818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mario A.  Tan</span>, <span class="hlFld-ContribAuthor ">Elena  Zakharova</span>, <span class="hlFld-ContribAuthor ">Seong Soo A.  An</span>. </span><span class="cited-content_cbyCitation_article-title">Diaportheone A Analogues Instigate a Neuroprotective Effect by Protecting Neuroblastoma SH-SY5Y Cells from Oxidative Stress. </span><span class="cited-content_cbyCitation_journal-name">Biology</span><span> <strong>2021,</strong> <em>10 </em>
                                    (3)
                                     , 199. <a href="https://doi.org/10.3390/biology10030199" title="DOI URL">https://doi.org/10.3390/biology10030199</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biology10030199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiology10030199%26sid%3Dliteratum%253Aachs%26jtitle%3DBiology%26atitle%3DDiaportheone%252BA%252BAnalogues%252BInstigate%252Ba%252BNeuroprotective%252BEffect%252Bby%252BProtecting%252BNeuroblastoma%252BSH-SY5Y%252BCells%252Bfrom%252BOxidative%252BStress%26aulast%3DTan%26aufirst%3DMario%2BA.%26date%3D2021%26date%3D2021%26volume%3D10%26issue%3D3%26spage%3D199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Liqian  Gao</span>, <span class="hlFld-ContribAuthor ">Wei  Wang</span>, <span class="hlFld-ContribAuthor ">Xuan  Wang</span>, <span class="hlFld-ContribAuthor ">Fen  Yang</span>, <span class="hlFld-ContribAuthor ">Liuxing  Xie</span>, <span class="hlFld-ContribAuthor ">Jun  Shen</span>, <span class="hlFld-ContribAuthor ">Margaret A.  Brimble</span>, <span class="hlFld-ContribAuthor ">Qicai  Xiao</span>, <span class="hlFld-ContribAuthor ">Shao Q.  Yao</span>. </span><span class="cited-content_cbyCitation_article-title">Fluorescent probes for bioimaging of potential biomarkers in Parkinson's disease. </span><span class="cited-content_cbyCitation_journal-name">Chemical Society Reviews</span><span> <strong>2021,</strong> <em>50 </em>
                                    (2)
                                     , 1219-1250. <a href="https://doi.org/10.1039/D0CS00115E" title="DOI URL">https://doi.org/10.1039/D0CS00115E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CS00115E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CS00115E%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Society%2520Reviews%26atitle%3DFluorescent%252Bprobes%252Bfor%252Bbioimaging%252Bof%252Bpotential%252Bbiomarkers%252Bin%252BParkinson%252527s%252Bdisease%26aulast%3DGao%26aufirst%3DLiqian%26date%3D2021%26date%3D2021%26volume%3D50%26issue%3D2%26spage%3D1219%26epage%3D1250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jyoti  Ahlawat</span>, <span class="hlFld-ContribAuthor ">Mahesh  Narayan</span>. </span><span class="cited-content_cbyCitation_article-title">Organic Carbon Dots for Mitigating Neurodegenerative Diseases. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 273-287. <a href="https://doi.org/10.1007/978-3-030-61985-5_10" title="DOI URL">https://doi.org/10.1007/978-3-030-61985-5_10</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-61985-5_10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-61985-5_10%26sid%3Dliteratum%253Aachs%26atitle%3DOrganic%252BCarbon%252BDots%252Bfor%252BMitigating%252BNeurodegenerative%252BDiseases%26aulast%3DAhlawat%26aufirst%3DJyoti%26date%3D2021%26date%3D2021%26spage%3D273%26epage%3D287%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DBiobased%252BNanotechnology%252Bfor%252BGreen%252BApplications%26aulast%3DSarma%26aufirst%3DHemen%26date%3D2021%26volume%3D63" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A. Katherine  Hatstat</span>, <span class="hlFld-ContribAuthor ">Hannah D.  Ahrendt</span>, <span class="hlFld-ContribAuthor ">Matthew W.  Foster</span>, <span class="hlFld-ContribAuthor ">Leland  Mayne</span>, <span class="hlFld-ContribAuthor ">M. Arthur  Moseley</span>, <span class="hlFld-ContribAuthor ">S. Walter  Englander</span>, <span class="hlFld-ContribAuthor ">Dewey G.  McCafferty</span>. </span><span class="cited-content_cbyCitation_article-title">Characterization of Small-Molecule-Induced Changes in Parkinson's-Related Trafficking via the Nedd4 Ubiquitin Signaling Cascade. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (1)
                                     , 14-25.e9. <a href="https://doi.org/10.1016/j.chembiol.2020.10.008" title="DOI URL">https://doi.org/10.1016/j.chembiol.2020.10.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2020.10.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2020.10.008%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DCharacterization%252Bof%252BSmall-Molecule-Induced%252BChanges%252Bin%252BParkinson%252527s-Related%252BTrafficking%252Bvia%252Bthe%252BNedd4%252BUbiquitin%252BSignaling%252BCascade%26aulast%3DHatstat%26aufirst%3DA.%2BKatherine%26date%3D2021%26volume%3D28%26issue%3D1%26spage%3D14%26epage%3D25.e9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jin  Zhou</span>, <span class="hlFld-ContribAuthor ">Paramesh  Jangili</span>, <span class="hlFld-ContribAuthor ">Subin  Son</span>, <span class="hlFld-ContribAuthor ">Myung Sun  Ji</span>, <span class="hlFld-ContribAuthor ">Miae  Won</span>, <span class="hlFld-ContribAuthor ">Jong Seung  Kim</span>. </span><span class="cited-content_cbyCitation_article-title">Fluorescent Diagnostic Probes in Neurodegenerative Diseases. </span><span class="cited-content_cbyCitation_journal-name">Advanced Materials</span><span> <strong>2020,</strong> <em>32 </em>
                                    (51)
                                     , 2001945. <a href="https://doi.org/10.1002/adma.202001945" title="DOI URL">https://doi.org/10.1002/adma.202001945</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/adma.202001945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadma.202001945%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Materials%26atitle%3DFluorescent%252BDiagnostic%252BProbes%252Bin%252BNeurodegenerative%252BDiseases%26aulast%3DZhou%26aufirst%3DJin%26date%3D2020%26date%3D2020%26volume%3D32%26issue%3D51%26spage%3D2001945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ronald L.  Davis</span>. </span><span class="cited-content_cbyCitation_article-title">Mechanism of Action and Target Identification: A Matter of Timing in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">iScience</span><span> <strong>2020,</strong> <em>23 </em>
                                    (9)
                                     , 101487. <a href="https://doi.org/10.1016/j.isci.2020.101487" title="DOI URL">https://doi.org/10.1016/j.isci.2020.101487</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.isci.2020.101487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.isci.2020.101487%26sid%3Dliteratum%253Aachs%26jtitle%3DiScience%26atitle%3DMechanism%252Bof%252BAction%252Band%252BTarget%252BIdentification%25253A%252BA%252BMatter%252Bof%252BTiming%252Bin%252BDrug%252BDiscovery%26aulast%3DDavis%26aufirst%3DRonald%2BL.%26date%3D2020%26volume%3D23%26issue%3D9%26spage%3D101487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Walter H.  Moos</span>, <span class="hlFld-ContribAuthor ">Douglas V.  Faller</span>, <span class="hlFld-ContribAuthor ">Ioannis P.  Glavas</span>, <span class="hlFld-ContribAuthor ">David N.  Harpp</span>, <span class="hlFld-ContribAuthor ">Iphigenia  Kanara</span>, <span class="hlFld-ContribAuthor ">Anastasios N.  Mavrakis</span>, <span class="hlFld-ContribAuthor ">Julie  Pernokas</span>, <span class="hlFld-ContribAuthor ">Mark  Pernokas</span>, <span class="hlFld-ContribAuthor ">Carl A.  Pinkert</span>, <span class="hlFld-ContribAuthor ">Whitney R.  Powers</span>, <span class="hlFld-ContribAuthor ">Konstantina  Sampani</span>, <span class="hlFld-ContribAuthor ">Kosta  Steliou</span>, <span class="hlFld-ContribAuthor ">Demetrios G.  Vavvas</span>, <span class="hlFld-ContribAuthor ">Robert J.  Zamboni</span>, <span class="hlFld-ContribAuthor ">Krishna  Kodukula</span>, <span class="hlFld-ContribAuthor ">Xiaohong  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Klotho Pathways, Myelination Disorders, Neurodegenerative Diseases, and Epigenetic Drugs. </span><span class="cited-content_cbyCitation_journal-name">BioResearch Open Access</span><span> <strong>2020,</strong> <em>9 </em>
                                    (1)
                                     , 94-105. <a href="https://doi.org/10.1089/biores.2020.0004" title="DOI URL">https://doi.org/10.1089/biores.2020.0004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1089/biores.2020.0004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1089%2Fbiores.2020.0004%26sid%3Dliteratum%253Aachs%26jtitle%3DBioResearch%2520Open%2520Access%26atitle%3DKlotho%252BPathways%25252C%252BMyelination%252BDisorders%25252C%252BNeurodegenerative%252BDiseases%25252C%252Band%252BEpigenetic%252BDrugs%26aulast%3DMoos%26aufirst%3DWalter%2BH.%26date%3D2020%26volume%3D9%26issue%3D1%26spage%3D94%26epage%3D105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/medium/jm-2019-00797d_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/large/jm-2019-00797d_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00797&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/medium/jm-2019-00797d_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/large/jm-2019-00797d_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic of phenotypic screening approaches. An appropriate cellular screening model is chosen according to the type of screening library and the phenotype of interest. Screening libraries can be divided into three classes: small molecule compound libraries, knockdown or deletion libraries, or overexpression libraries. Knockdown or deletion libraries contain either siRNA, shRNA, or gRNA for CRISPR screens. For CRISPR screens, the chosen cell type requires expression of Cas9 endonuclease. Cells can also express a disease gene that causes the phenotype of interest (e.g., overexpression of tau, α-synuclein, or Htt fragments) and/or a reporter gene (e.g., GFP-labeled LC3 to visualize autophagic flux, a minigene to assess splicing, a luciferase-labeled transgene to visualize up-or downregulation of protein levels).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/large/jm-2019-00797d_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00797&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/medium/jm-2019-00797d_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/large/jm-2019-00797d_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Overview of cell models used in phenotypic screening for neurodegeneration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/large/jm-2019-00797d_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00797&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/medium/jm-2019-00797d_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/large/jm-2019-00797d_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Generic hit-validation cascade for phenotypic screening.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b00797/20200306/images/large/jm-2019-00797d_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00797&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i24">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72034" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72034" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 117 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorne, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKew, J. C.</span></span> <span> </span><span class="NLM_article-title">Phenotypic Screens as a Renewed Approach for Drug Discovery</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2013.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.drudis.2013.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23850704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1emt7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=1067-1073&author=W.+Zhengauthor=N.+Thorneauthor=J.+C.+McKew&title=Phenotypic+Screens+as+a+Renewed+Approach+for+Drug+Discovery&doi=10.1016%2Fj.drudis.2013.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Phenotypic screens as a renewed approach for drug discovery</span></div><div class="casAuthors">Zheng, Wei; Thorne, Natasha; McKew, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">21-22</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The significant redn. in the no. of newly approved drugs in the past decade has been partially attributed to failures in discovery and validation of new targets.  Evaluation of recently approved new drugs has revealed that the no. of approved drugs discovered through phenotypic screens, an original drug screening paradigm, has exceeded those discovered through the mol. target-based approach.  Phenotypic screening is thus gaining new momentum in drug discovery with the hope that this approach may revitalize drug discovery and improve the success rate of drug approval through the discovery of viable lead compds. and identification of novel drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_cPfNTmnrfbVg90H21EOLACvtfcHk0liWg5-wwLl4Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1emt7vO&md5=9d92e1f61043ac2dd11bfe5f32944cff</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DThorne%26aufirst%3DN.%26aulast%3DMcKew%26aufirst%3DJ.%2BC.%26atitle%3DPhenotypic%2520Screens%2520as%2520a%2520Renewed%2520Approach%2520for%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2013%26volume%3D18%26spage%3D1067%26epage%3D1073%26doi%3D10.1016%2Fj.drudis.2013.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Swinney, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anthony, J.</span></span> <span> </span><span class="NLM_article-title">How Were New Medicines Discovered?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">507</span>– <span class="NLM_lpage">519</span>, <span class="refDoi"> DOI: 10.1038/nrd3480</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnrd3480" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=21701501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlCmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=507-519&author=D.+C.+Swinneyauthor=J.+Anthony&title=How+Were+New+Medicines+Discovered%3F&doi=10.1038%2Fnrd3480"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">How were new medicines discovered?</span></div><div class="casAuthors">Swinney, David C.; Anthony, Jason</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">507-519</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Preclin. strategies that are used to identify potential drug candidates include target-based screening, phenotypic screening, modification of natural substances and biol.-based approaches.  To investigate whether some strategies have been more successful than others in the discovery of new drugs, we analyzed the discovery strategies and the mol. mechanism of action (MMOA) for new mol. entities and new biologics that were approved by the US Food and Drug Administration between 1999 and 2008.  Out of the 259 agents that were approved, 75 were first-in-class drugs with new MMOAs, and out of these, 50 (67%) were small mols. and 25 (33%) were biologics.  The results also show that the contribution of phenotypic screening to the discovery of first-in-class small-mol. drugs exceeded that of target-based approaches - with 28 and 17 of these drugs coming from the two approaches, resp. - in an era in which the major focus was on target-based approaches.  We postulate that a target-centric approach for first-in-class drugs, without consideration of an optimal MMOA, may contribute to the current high attrition rates and low productivity in pharmaceutical research and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE8Ah1iNz76bVg90H21EOLACvtfcHk0liWg5-wwLl4Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlCmsbs%253D&md5=7503620f8fbadb0a829f6ea2029936a2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd3480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3480%26sid%3Dliteratum%253Aachs%26aulast%3DSwinney%26aufirst%3DD.%2BC.%26aulast%3DAnthony%26aufirst%3DJ.%26atitle%3DHow%2520Were%2520New%2520Medicines%2520Discovered%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D507%26epage%3D519%26doi%3D10.1038%2Fnrd3480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aulner, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danckaert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ihm, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shum, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shorte, S. L.</span></span> <span> </span><span class="NLM_article-title">Next-Generation Phenotypic Screening in Early Drug Discovery for Infectious Diseases</span>. <i>Trends Parasitol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">559</span>– <span class="NLM_lpage">570</span>, <span class="refDoi"> DOI: 10.1016/j.pt.2019.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.pt.2019.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=31176583" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BB3M3ksFWmtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2019&pages=559-570&author=N.+Aulnerauthor=A.+Danckaertauthor=J.+Ihmauthor=D.+Shumauthor=S.+L.+Shorte&title=Next-Generation+Phenotypic+Screening+in+Early+Drug+Discovery+for+Infectious+Diseases&doi=10.1016%2Fj.pt.2019.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Next-Generation Phenotypic Screening in Early Drug Discovery for Infectious Diseases</span></div><div class="casAuthors">Aulner Nathalie; Danckaert Anne; Ihm JongEun; Shum David; Shorte Spencer L</div><div class="citationInfo"><span class="NLM_cas:title">Trends in parasitology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">559-570</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cell-based phenotypic screening has proven to be valuable, notably in recapitulating relevant biological conditions, for example, the host cell/pathogen niche.  However, the corresponding methodological complexity is not readily compatible with high-throughput pipelines, and fails to inform either molecular target or mechanism of action, which frustrates conventional drug-discovery roadmaps.  We review the state-of-the-art and emerging technologies that suggest new strategies for harnessing value from the complexity of phenotypic screening and augmenting powerful utility for translational drug discovery.  Advances in cellular, molecular, and bioinformatics technologies are converging at a cutting edge where the complexity of phenotypic screening may no longer be considered a hinderance but rather a catalyst to chemotherapeutic discovery for infectious diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSLgYTGAfXc8JPq5LRczm8ffW6udTcc2ea1ldOYPMO67Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3ksFWmtw%253D%253D&md5=a0ff8c0384a7171183c90ed6586c37b1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.pt.2019.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pt.2019.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DAulner%26aufirst%3DN.%26aulast%3DDanckaert%26aufirst%3DA.%26aulast%3DIhm%26aufirst%3DJ.%26aulast%3DShum%26aufirst%3DD.%26aulast%3DShorte%26aufirst%3DS.%2BL.%26atitle%3DNext-Generation%2520Phenotypic%2520Screening%2520in%2520Early%2520Drug%2520Discovery%2520for%2520Infectious%2520Diseases%26jtitle%3DTrends%2520Parasitol.%26date%3D2019%26volume%3D35%26spage%3D559%26epage%3D570%26doi%3D10.1016%2Fj.pt.2019.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mdluli, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaneko, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, A.</span></span> <span> </span><span class="NLM_article-title">The Tuberculosis Drug Discovery and Development Pipeline and Emerging Drug Targets</span>. <i>Cold Spring Harbor Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">a021154</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a021154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1101%2Fcshperspect.a021154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25635061" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=a021154&author=K.+Mdluliauthor=T.+Kanekoauthor=A.+Upton&title=The+Tuberculosis+Drug+Discovery+and+Development+Pipeline+and+Emerging+Drug+Targets&doi=10.1101%2Fcshperspect.a021154"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a021154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a021154%26sid%3Dliteratum%253Aachs%26aulast%3DMdluli%26aufirst%3DK.%26aulast%3DKaneko%26aufirst%3DT.%26aulast%3DUpton%26aufirst%3DA.%26atitle%3DThe%2520Tuberculosis%2520Drug%2520Discovery%2520and%2520Development%2520Pipeline%2520and%2520Emerging%2520Drug%2520Targets%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Med.%26date%3D2015%26volume%3D5%26spage%3Da021154%26doi%3D10.1101%2Fcshperspect.a021154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, R.</span></span> <span> </span><span class="NLM_article-title">First Novel Anti-Tuberculosis Drug in 40 Years</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">91</span>, <span class="refDoi"> DOI: 10.1038/nbt0213-89</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnbt0213-89" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23392485" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3sXit1Kisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=89-91&author=R.+Osborne&title=First+Novel+Anti-Tuberculosis+Drug+in+40+Years&doi=10.1038%2Fnbt0213-89"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">First novel anti-tuberculosis drug in 40 years</span></div><div class="casAuthors">Osborne, Randy</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">89-91</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9-XOgWa1Q27Vg90H21EOLACvtfcHk0lj8UBlPDxGNZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXit1Kisrk%253D&md5=352170811a489381ea320397f4a6ed46</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnbt0213-89&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt0213-89%26sid%3Dliteratum%253Aachs%26aulast%3DOsborne%26aufirst%3DR.%26atitle%3DFirst%2520Novel%2520Anti-Tuberculosis%2520Drug%2520in%252040%2520Years%26jtitle%3DNat.%2520Biotechnol.%26date%3D2013%26volume%3D31%26spage%3D89%26epage%3D91%26doi%3D10.1038%2Fnbt0213-89" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brodie, M. J.</span></span> <span> </span><span class="NLM_article-title">Antiepileptic Drug Therapy the Story So Far</span>. <i>Seizure</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">650</span>– <span class="NLM_lpage">655</span>, <span class="refDoi"> DOI: 10.1016/j.seizure.2010.10.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.seizure.2010.10.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=21075011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC3M%252FhtlOrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2010&pages=650-655&author=M.+J.+Brodie&title=Antiepileptic+Drug+Therapy+the+Story+So+Far&doi=10.1016%2Fj.seizure.2010.10.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Antiepileptic drug therapy the story so far</span></div><div class="casAuthors">Brodie Martin J</div><div class="citationInfo"><span class="NLM_cas:title">Seizure</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">650-5</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The story began on 11th May 1857 when Charles Locock commented in the Lancet on his use of potassium bromide in 15 cases of "hysterical" epilepsy in young women.  The next development was the serendipitous discovery of the anticonvulsant properties of phenobarbital by Alfred Hauptmann in 1912.  This predated by more than 20 years the screening of potential therapeutic agents against "electrical seizures" in cats by Houston Merritt and Tracy Putnam.  The result was the launching of phenytoin in 1938.  Next came primidone, ethosuximide, carbamazepine and valproic acid, all of which can be regarded as first generation antiepileptic drugs (AEDs).  Shortly after their synthesis, the benzodiazepines were rapidly recognised as having anticonvulsant activity.  The modern era focused on the systematic screening of many thousands of compounds against rodent seizure models under the Anticonvulsant Drug Development Program in the US.  This resulted in the global licensing, in chronological order, of vigabatrin, zonisamide, oxcarbazepine, lamotrigine, felbamate, gabapentin, topiramate, tiagabine, levetiracetam, pregabalin and lacosamide.  Rufinamide is available in the US and Europe for Lennox-Gastaut syndrome and stiripentol has been made available for Dravet syndrome under the European orphan drug scheme.  Eslicarbazepine can be prescribed in Europe for partial seizures, but not in the US.  Has all this activity improved the lives of people with epilepsy? The short answer is-probably yes, but not by very much! This paper will conclude with a precis of the views of a selected group of paediatric and adult epileptologists on the advances in pharmacological management achieved over the last 20 years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQkQS9UEkVleJI8NvIb08rbfW6udTcc2ea1ldOYPMO67Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M%252FhtlOrtg%253D%253D&md5=6e97b89170a2b0b33f57bd397f016105</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.seizure.2010.10.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.seizure.2010.10.027%26sid%3Dliteratum%253Aachs%26aulast%3DBrodie%26aufirst%3DM.%2BJ.%26atitle%3DAntiepileptic%2520Drug%2520Therapy%2520the%2520Story%2520So%2520Far%26jtitle%3DSeizure%26date%3D2010%26volume%3D19%26spage%3D650%26epage%3D655%26doi%3D10.1016%2Fj.seizure.2010.10.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Z.</span></span> <span> </span><span class="NLM_article-title">Phenotypic Screens Targeting Neurodegenerative Diseases</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1177/1087057113499777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1177%2F1087057113499777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23958650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWgtbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=1-16&author=M.+Zhangauthor=G.+Luoauthor=Y.+Zhouauthor=S.+Wangauthor=Z.+Zhong&title=Phenotypic+Screens+Targeting+Neurodegenerative+Diseases&doi=10.1177%2F1087057113499777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Phenotypic screens targeting neurodegenerative diseases</span></div><div class="casAuthors">Zhang, Minhua; Luo, Guangrui; Zhou, Yanjiao; Wang, Shaohui; Zhong, Zhong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-16, 16 pp.</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Neurodegenerative diseases affect millions of people worldwide, and the incidences increase as the population ages.  Disease-modifying therapy that prevents or slows disease progression is still lacking, making neurodegenerative diseases an area of high unmet medical need.  Target-based drug discovery for disease-modifying agents has been ongoing for many years, without much success due to incomplete understanding of the mol. mechanisms underlying neurodegeneration.  Phenotypic screening, starting with a disease-relevant phenotype to screen for compds. that change the outcome of biol. pathways rather than activities at certain specific targets, offers an alternative approach to find small mols. or targets that modulate the key characteristics of neurodegeneration.  Phenotypic screens that focus on amelioration of disease-specific toxins, protection of neurons from degeneration, or promotion of neuroregeneration could be potential fertile grounds for discovering therapeutic agents for neurodegenerative diseases.  In this review, we will summarize the progress of compd. screening using these phenotypic-based strategies for this area, with a highlight on unique considerations for disease models, assays, and screening methodologies.  We will further provide our perspectives on how best to use phenotypic screening to develop drug leads for neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEhrbJWCDltbVg90H21EOLACvtfcHk0lj8UBlPDxGNZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWgtbfN&md5=d462b3885bb85f3593316d0e71f4e672</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1177%2F1087057113499777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113499777%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DZ.%26atitle%3DPhenotypic%2520Screens%2520Targeting%2520Neurodegenerative%2520Diseases%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2014%26volume%3D19%26spage%3D1%26epage%3D16%26doi%3D10.1177%2F1087057113499777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, D.</span></span> <span> </span><span class="NLM_article-title">Phenotypic Screening in Cancer Drug Discovery - Past, Present and Future</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">588</span>– <span class="NLM_lpage">602</span>, <span class="refDoi"> DOI: 10.1038/nrd4366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnrd4366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25033736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=588-602&author=J.+G.+Moffatauthor=J.+Rudolphauthor=D.+Bailey&title=Phenotypic+Screening+in+Cancer+Drug+Discovery+-+Past%2C+Present+and+Future&doi=10.1038%2Fnrd4366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Phenotypic screening in cancer drug discovery - past, present and future</span></div><div class="casAuthors">Moffat, John G.; Rudolph, Joachim; Bailey, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">588-602</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There has been a resurgence of interest in the use of phenotypic screens in drug discovery as an alternative to target-focused approaches.  Given that oncol. is currently the most active therapeutic area, and also one in which target-focused approaches have been particularly prominent in the past two decades, we investigated the contribution of phenotypic assays to oncol. drug discovery by analyzing the origins of all new small-mol. cancer drugs approved by the US Food and Drug Administration (FDA) over the past 15 years and those currently in clin. development.  Although the majority of these drugs originated from target-based discovery, we identified a significant no. whose discovery depended on phenotypic screening approaches.  We postulate that the contribution of phenotypic screening to cancer drug discovery has been hampered by a reliance on 'classical' nonspecific drug effects such as cytotoxicity and mitotic arrest, exacerbated by a paucity of mechanistically defined cellular models for therapeutically translatable cancer phenotypes.  However, tech. and biol. advances that enable such mechanistically informed phenotypic models have the potential to empower phenotypic drug discovery in oncol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTWtoTIgWqvbVg90H21EOLACvtfcHk0ljIBvIPG6Xreg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7L&md5=0e8efed90e5bc4b47f959b83a6ab5b96</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd4366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4366%26sid%3Dliteratum%253Aachs%26aulast%3DMoffat%26aufirst%3DJ.%2BG.%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DBailey%26aufirst%3DD.%26atitle%3DPhenotypic%2520Screening%2520in%2520Cancer%2520Drug%2520Discovery%2520-%2520Past%252C%2520Present%2520and%2520Future%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D588%26epage%3D602%26doi%3D10.1038%2Fnrd4366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delves, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magny, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clement, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boullay, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandel, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of Small Molecule Inhibitors of the Bet Family Bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">3827</span>– <span class="NLM_lpage">3838</span>, <span class="refDoi"> DOI: 10.1021/jm200108t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200108t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3827-3838&author=C.+W.+Chungauthor=H.+Costeauthor=J.+H.+Whiteauthor=O.+Mirguetauthor=J.+Wildeauthor=R.+L.+Gosminiauthor=C.+Delvesauthor=S.+M.+Magnyauthor=R.+Woodwardauthor=S.+A.+Hughesauthor=E.+V.+Boursierauthor=H.+Flynnauthor=A.+M.+Bouillotauthor=P.+Bamboroughauthor=J.+M.+Brusqauthor=F.+J.+Gellibertauthor=E.+J.+Jonesauthor=A.+M.+Riouauthor=P.+Homesauthor=S.+L.+Martinauthor=I.+J.+Uingsauthor=J.+Toumauthor=C.+A.+Clementauthor=A.+B.+Boullayauthor=R.+L.+Grimleyauthor=F.+M.+Blandelauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=J.+Kirilovskyauthor=E.+Nicodeme&title=Discovery+and+Characterization+of+Small+Molecule+Inhibitors+of+the+Bet+Family+Bromodomains&doi=10.1021%2Fjm200108t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of small molecule inhibitors of the BET family bromodomains</span></div><div class="casAuthors">Chung, Chun-wa; Coste, Herve; White, Julia H.; Mirguet, Olivier; Wilde, Jonathan; Gosmini, Romain L.; Delves, Chris; Magny, Sylvie M.; Woodward, Robert; Hughes, Stephen A.; Boursier, Eric V.; Flynn, Helen; Bouillot, Anne M.; Bamborough, Paul; Brusq, Jean-Marie G.; Gellibert, Francoise J.; Jones, Emma J.; Riou, Alizon M.; Homes, Paul; Martin, Sandrine L.; Uings, Iain J.; Toum, Jerome; Clement, Catherine A.; Boullay, Anne-Benedicte; Grimley, Rachel L.; Blandel, Florence M.; Prinjha, Rab K.; Lee, Kevin; Kirilovsky, Jorge; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3827-3838</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic mechanisms of gene regulation have a profound role in normal development and disease processes.  An integral part of this mechanism occurs through lysine acetylation of histone tails which are recognized by bromodomains.  While the biol. and structural characterization of many bromodomain contg. proteins has advanced considerably, the therapeutic tractability of this protein family is only now becoming understood.  This paper describes the discovery and mol. characterization of potent (nM) small mol. inhibitors that disrupt the function of the BET family of bromodomains (Brd2, Brd3, and Brd4).  By using a combination of phenotypic screening, chemoproteomics, and biophys. studies, we have discovered that the protein-protein interactions between bromodomains and acetylated histones can be antagonized by selective small mols. that bind at the acetylated lysine recognition pocket.  X-ray crystal structures of compds. bound into bromodomains of Brd2 and Brd4 elucidate the mol. interactions of binding and explain the precisely defined stereochem. required for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VauuYkT5YLVg90H21EOLACvtfcHk0ljIBvIPG6Xreg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D&md5=ebc0f6062321dece877259d775f1c19b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm200108t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200108t%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%2BH.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DWilde%26aufirst%3DJ.%26aulast%3DGosmini%26aufirst%3DR.%2BL.%26aulast%3DDelves%26aufirst%3DC.%26aulast%3DMagny%26aufirst%3DS.%2BM.%26aulast%3DWoodward%26aufirst%3DR.%26aulast%3DHughes%26aufirst%3DS.%2BA.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DFlynn%26aufirst%3DH.%26aulast%3DBouillot%26aufirst%3DA.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBrusq%26aufirst%3DJ.%2BM.%26aulast%3DGellibert%26aufirst%3DF.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DRiou%26aufirst%3DA.%2BM.%26aulast%3DHomes%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%2BL.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DClement%26aufirst%3DC.%2BA.%26aulast%3DBoullay%26aufirst%3DA.%2BB.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBlandel%26aufirst%3DF.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520the%2520Bet%2520Family%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D3827%26epage%3D3838%26doi%3D10.1021%2Fjm200108t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gammon, K.</span></span> <span> </span><span class="NLM_article-title">Neurodegenerative Disease: Brain Windfall</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>515</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">300</span>, <span class="refDoi"> DOI: 10.1038/nj7526-299a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnj7526-299a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25396246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC2M3ot12kuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=515&publication_year=2014&pages=299-300&author=K.+Gammon&title=Neurodegenerative+Disease%3A+Brain+Windfall&doi=10.1038%2Fnj7526-299a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Neurodegenerative disease: brain windfall</span></div><div class="casAuthors">Gammon Katharine</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">515</span>
        (<span class="NLM_cas:issue">7526</span>),
    <span class="NLM_cas:pages">299-300</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQDfOo1BTaCdG52AMpI4GK2fW6udTcc2ebzZejR58cX6bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3ot12kuw%253D%253D&md5=1ee40ff6fc8fab21a70d1df7a0b12674</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnj7526-299a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnj7526-299a%26sid%3Dliteratum%253Aachs%26aulast%3DGammon%26aufirst%3DK.%26atitle%3DNeurodegenerative%2520Disease%253A%2520Brain%2520Windfall%26jtitle%3DNature%26date%3D2014%26volume%3D515%26spage%3D299%26epage%3D300%26doi%3D10.1038%2Fnj7526-299a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span> </span><span class="NLM_article-title">Focus on Neurodegenerative Disease</span>.  <i>Nat.
Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume">21</span>,  <span class="NLM_fpage">1293</span>.<span class="refDoi"> DOI: 10.1038/s41593-018-0250-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41593-018-0250-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30258240" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Focus+on+Neurodegenerative+Disease.+Nat.%0ANeurosci.+2018%2C+21%2C+1293.+10.1038%2Fs41593-018-0250-x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fs41593-018-0250-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41593-018-0250-x%26sid%3Dliteratum%253Aachs%26atitle%3DFocus%2520on%2520Neurodegenerative%2520Disease%26jtitle%3DNat.%250ANeurosci.%26date%3D2018%26volume%3D21%26spage%3D1293%26doi%3D10.1038%2Fs41593-018-0250-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soto, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritzkow, S.</span></span> <span> </span><span class="NLM_article-title">Protein Misfolding, Aggregation, and Conformational Strains in Neurodegenerative Diseases</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1332</span>– <span class="NLM_lpage">1340</span>, <span class="refDoi"> DOI: 10.1038/s41593-018-0235-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41593-018-0235-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30250260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslymtrfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2018&pages=1332-1340&author=C.+Sotoauthor=S.+Pritzkow&title=Protein+Misfolding%2C+Aggregation%2C+and+Conformational+Strains+in+Neurodegenerative+Diseases&doi=10.1038%2Fs41593-018-0235-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Protein misfolding, aggregation, and conformational strains in neurodegenerative diseases</span></div><div class="casAuthors">Soto, Claudio; Pritzkow, Sandra</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1332-1340</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A hallmark event in neurodegenerative diseases (NDs) is the misfolding, aggregation, and accumulation of proteins, leading to cellular dysfunction, loss of synaptic connections, and brain damage.  Despite the involvement of distinct proteins in different NDs, the process of protein misfolding and aggregation is remarkably similar.  A recent breakthrough in the field was the discovery that misfolded protein aggregates can self-propagate through seeding and spread the pathol. abnormalities between cells and tissues in a manner akin to the behavior of infectious prions in prion diseases.  This discovery has vast implications for understanding the mechanisms involved in the initiation and progression of NDs, as well as for the design of novel strategies for treatment and diagnosis.  In this Review, we provide a crit. discussion of the role of protein misfolding and aggregation in NDs.  Commonalities and differences between distinct protein aggregates will be highlighted, in addn. to evidence supporting the hypothesis that misfolded aggregates can be transmissible by the prion principle.  We will also describe the mol. basis and implications for prion-like conformational strains, cross-interaction between different misfolded proteins in the brain, and how these concepts can be applied to the development of novel strategies for therapy and diagnosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosF34qbEjX8LVg90H21EOLACvtfcHk0lggHiBkxwWY6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslymtrfN&md5=0d18590c1c796f289939f7125dc56f16</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fs41593-018-0235-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41593-018-0235-9%26sid%3Dliteratum%253Aachs%26aulast%3DSoto%26aufirst%3DC.%26aulast%3DPritzkow%26aufirst%3DS.%26atitle%3DProtein%2520Misfolding%252C%2520Aggregation%252C%2520and%2520Conformational%2520Strains%2520in%2520Neurodegenerative%2520Diseases%26jtitle%3DNat.%2520Neurosci.%26date%3D2018%26volume%3D21%26spage%3D1332%26epage%3D1340%26doi%3D10.1038%2Fs41593-018-0235-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, J.</span></span> <span> </span><span class="NLM_article-title">Notes on Continuing Education. The First Hour: Creating a Learning Evironment</span>. <i>J. Contin. Educ. Nurs.</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">46</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=258237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADyaL3c%252FgslWhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1979&pages=42-46&author=J.+Murphy&title=Notes+on+Continuing+Education.+The+First+Hour%3A+Creating+a+Learning+Evironment"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Notes on continuing education. The first hour: creating a learning evironment</span></div><div class="casAuthors">Murphy J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of continuing education in nursing</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">42-6</span>
        ISSN:<span class="NLM_cas:issn">0022-0124</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRrfSCfuWDe30NHe_gFldTbfW6udTcc2ebzZejR58cX6bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL3c%252FgslWhtw%253D%253D&md5=ccd924a323aa58778f665abe82520e39</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMurphy%26aufirst%3DJ.%26atitle%3DNotes%2520on%2520Continuing%2520Education.%2520The%2520First%2520Hour%253A%2520Creating%2520a%2520Learning%2520Evironment%26jtitle%3DJ.%2520Contin.%2520Educ.%2520Nurs.%26date%3D1979%26volume%3D10%26spage%3D42%26epage%3D46" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Botstein, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chervitz, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherry, J. M.</span></span> <span> </span><span class="NLM_article-title">Yeast as a Model Organism</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">1259</span>– <span class="NLM_lpage">1260</span>, <span class="refDoi"> DOI: 10.1126/science.277.5330.1259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fscience.277.5330.1259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=9297238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADyaK2sXlslaitr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=1997&pages=1259-1260&author=D.+Botsteinauthor=S.+A.+Chervitzauthor=J.+M.+Cherry&title=Yeast+as+a+Model+Organism&doi=10.1126%2Fscience.277.5330.1259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Yeast as a model organism</span></div><div class="casAuthors">Botstein, David; Chervitz, Steven A.; Cherry, J. Michael</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">5330</span>),
    <span class="NLM_cas:pages">1259-1260</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoilx40K3MqprVg90H21EOLACvtfcHk0lggHiBkxwWY6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlslaitr4%253D&md5=8df86f99c1f1b836f131cba2ab3f51f9</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1126%2Fscience.277.5330.1259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.277.5330.1259%26sid%3Dliteratum%253Aachs%26aulast%3DBotstein%26aufirst%3DD.%26aulast%3DChervitz%26aufirst%3DS.%2BA.%26aulast%3DCherry%26aufirst%3DJ.%2BM.%26atitle%3DYeast%2520as%2520a%2520Model%2520Organism%26jtitle%3DScience%26date%3D1997%26volume%3D277%26spage%3D1259%26epage%3D1260%26doi%3D10.1126%2Fscience.277.5330.1259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goffeau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrell, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussey, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dujon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galibert, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoheisel, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacq, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mewes, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murakami, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philippsen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tettelin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, S. G.</span></span> <span> </span><span class="NLM_article-title">Life with 6000 Genes</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>274</i></span>,  <span class="NLM_fpage">546</span>– <span class="NLM_lpage">567</span>, <span class="refDoi"> DOI: 10.1126/science.274.5287.546</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fscience.274.5287.546" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=8849441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADyaK28XmsVWlt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=546-567&author=A.+Goffeauauthor=B.+G.+Barrellauthor=H.+Busseyauthor=R.+W.+Davisauthor=B.+Dujonauthor=H.+Feldmannauthor=F.+Galibertauthor=J.+D.+Hoheiselauthor=C.+Jacqauthor=M.+Johnstonauthor=E.+J.+Louisauthor=H.+W.+Mewesauthor=Y.+Murakamiauthor=P.+Philippsenauthor=H.+Tettelinauthor=S.+G.+Oliver&title=Life+with+6000+Genes&doi=10.1126%2Fscience.274.5287.546"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Life with 6000 genes</span></div><div class="casAuthors">Goffeau, A.; Barrell, B. G.; Bussey, H.; Davis, R. W.; Dujon, B.; Feldmann, H.; Galibert, F.; Hoheisel, J. D.; Jacq, C.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">5287</span>),
    <span class="NLM_cas:pages">546, 563-567</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The genome of the yeast Saccharomyces cerevisiae has been completely sequenced through a worldwide collaboration.  The sequence of 12,068 kilobases defines 5885 potential protein-encoding genes, ∼140 genes specifying rRNA, 40 genes for small nuclear RNA mols., and 275 tRNA genes.  In addn., the complete sequence provides information about the higher order organization of yeast's 16 chromosomes and allows some insight into their evolutionary history.  The genome shows a considerable amt. of apparent genetic redundancy, and one of the major problems to be tackled during the next stage of the yeast genome project is to elucidate the biol. functions of all of these genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqAJbfgr4k5rVg90H21EOLACvtfcHk0lgJsuyYgmgZJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmsVWlt7k%253D&md5=e7d4f76e51df1c61544dff929b6ce8e0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1126%2Fscience.274.5287.546&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.274.5287.546%26sid%3Dliteratum%253Aachs%26aulast%3DGoffeau%26aufirst%3DA.%26aulast%3DBarrell%26aufirst%3DB.%2BG.%26aulast%3DBussey%26aufirst%3DH.%26aulast%3DDavis%26aufirst%3DR.%2BW.%26aulast%3DDujon%26aufirst%3DB.%26aulast%3DFeldmann%26aufirst%3DH.%26aulast%3DGalibert%26aufirst%3DF.%26aulast%3DHoheisel%26aufirst%3DJ.%2BD.%26aulast%3DJacq%26aufirst%3DC.%26aulast%3DJohnston%26aufirst%3DM.%26aulast%3DLouis%26aufirst%3DE.%2BJ.%26aulast%3DMewes%26aufirst%3DH.%2BW.%26aulast%3DMurakami%26aufirst%3DY.%26aulast%3DPhilippsen%26aufirst%3DP.%26aulast%3DTettelin%26aufirst%3DH.%26aulast%3DOliver%26aufirst%3DS.%2BG.%26atitle%3DLife%2520with%25206000%2520Genes%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D546%26epage%3D567%26doi%3D10.1126%2Fscience.274.5287.546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Outeiro, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giorgini, F.</span></span> <span> </span><span class="NLM_article-title">Yeast as a Drug Discovery Platform in Huntington’s and Parkinson’s Diseases</span>. <i>Biotechnol. J.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">258</span>– <span class="NLM_lpage">269</span>, <span class="refDoi"> DOI: 10.1002/biot.200500043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1002%2Fbiot.200500043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=16897706" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtlKqtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=258-269&author=T.+F.+Outeiroauthor=F.+Giorgini&title=Yeast+as+a+Drug+Discovery+Platform+in+Huntington%E2%80%99s+and+Parkinson%E2%80%99s+Diseases&doi=10.1002%2Fbiot.200500043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Yeast as a drug discovery platform in Huntington's and Parkinson's diseases</span></div><div class="casAuthors">Outeiro, Tiago Fleming; Giorgini, Flaviano</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">258-269</span>CODEN:
                <span class="NLM_cas:coden">BJIOAM</span>;
        ISSN:<span class="NLM_cas:issn">1860-6768</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The high degree of conservation of cellular and mol. processes between the budding yeast Saccharomyces cerevisiae and higher eukaryotes have made it a valuable system for numerous studies of the basic mechanisms behind devastating illnesses such as cancer, infectious disease, and neurodegenerative disorders.  Several studies in yeast have already contributed to the basic understanding of cellular dysfunction in both Huntington's and Parkinson's disease.  Functional genomics approaches currently being undertaken in yeast may lead to novel insights into the genes and pathways that modulate neuronal cell dysfunction and death in these diseases.  In addn., the budding yeast constitutes a valuable system for identification of new drug targets, both via target-based and non-target-based drug screening.  Importantly, yeast can be used as a cellular platform to analyze the cellular effects of candidate compds., which is crit. for the development of effective therapeutics.  While the mol. mechanisms that underlie neurodegeneration will ultimately have to be tested in neuronal and animal models, there are several distinct advantages to using simple model organisms to elucidate fundamental aspects of protein aggregation, amyloid toxicity, and cellular dysfunction.  Here, we review recent studies that have shown that amyloid formation by disease-causing proteins and many of the resulting cellular deficits can be faithfully recapitulated in yeast.  In addn., we discuss new yeast-based techniques for screening candidate therapeutic compds. for Huntington's and Parkinson's diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfBHyXW8Cd8bVg90H21EOLACvtfcHk0lgJsuyYgmgZJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmtlKqtb0%253D&md5=0a56d1b8f18dc7f01b58b6bd8ed24a13</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fbiot.200500043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbiot.200500043%26sid%3Dliteratum%253Aachs%26aulast%3DOuteiro%26aufirst%3DT.%2BF.%26aulast%3DGiorgini%26aufirst%3DF.%26atitle%3DYeast%2520as%2520a%2520Drug%2520Discovery%2520Platform%2520in%2520Huntington%25E2%2580%2599s%2520and%2520Parkinson%25E2%2580%2599s%2520Diseases%26jtitle%3DBiotechnol.%2520J.%26date%3D2006%26volume%3D1%26spage%3D258%26epage%3D269%26doi%3D10.1002%2Fbiot.200500043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Outeiro, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindquist, S.</span></span> <span> </span><span class="NLM_article-title">Yeast Cells Provide Insight into Alpha-Synuclein Biology and Pathobiology</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>302</i></span>,  <span class="NLM_fpage">1772</span>– <span class="NLM_lpage">1775</span>, <span class="refDoi"> DOI: 10.1126/science.1090439</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fscience.1090439" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=14657500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpsVWns74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=302&publication_year=2003&pages=1772-1775&author=T.+F.+Outeiroauthor=S.+Lindquist&title=Yeast+Cells+Provide+Insight+into+Alpha-Synuclein+Biology+and+Pathobiology&doi=10.1126%2Fscience.1090439"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Yeast Cells Provide Insight into Alpha-Synuclein Biology and Pathobiology</span></div><div class="casAuthors">Outeiro, Tiago Fleming; Lindquist, Susan</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">302</span>
        (<span class="NLM_cas:issue">5651</span>),
    <span class="NLM_cas:pages">1772-1775</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Alpha-synuclein is implicated in several neurodegenerative disorders, such as Parkinson's disease and multiple system atrophy, yet its functions remain obscure.  When expressed in yeast, alpha-synuclein assocd. with the plasma membrane in a highly selective manner, before forming cytoplasmic inclusions through a concn.-dependent, nucleated process.  Alpha-synuclein inhibited phospholipase D, induced lipid droplet accumulation, and affected vesicle trafficking.  This readily manipulable system provides an opportunity to dissect the mol. pathways underlying normal alpha-synuclein biol. and the pathogenic consequences of its misfolding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoO71orB7yEqrVg90H21EOLACvtfcHk0lgJsuyYgmgZJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpsVWns74%253D&md5=3683c72f65e8639d5ad01569a724d9ea</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1126%2Fscience.1090439&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1090439%26sid%3Dliteratum%253Aachs%26aulast%3DOuteiro%26aufirst%3DT.%2BF.%26aulast%3DLindquist%26aufirst%3DS.%26atitle%3DYeast%2520Cells%2520Provide%2520Insight%2520into%2520Alpha-Synuclein%2520Biology%2520and%2520Pathobiology%26jtitle%3DScience%26date%3D2003%26volume%3D302%26spage%3D1772%26epage%3D1775%26doi%3D10.1126%2Fscience.1090439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tenreiro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munder, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alberti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Outeiro, T. F.</span></span> <span> </span><span class="NLM_article-title">Harnessing the Power of Yeast to Unravel the Molecular Basis of Neurodegeneration</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1111/jnc.12271</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1111%2Fjnc.12271" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23600759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWqs7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2013&pages=438-452&author=S.+Tenreiroauthor=M.+C.+Munderauthor=S.+Albertiauthor=T.+F.+Outeiro&title=Harnessing+the+Power+of+Yeast+to+Unravel+the+Molecular+Basis+of+Neurodegeneration&doi=10.1111%2Fjnc.12271"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing the power of yeast to unravel the molecular basis of neurodegeneration</span></div><div class="casAuthors">Tenreiro, Sandra; Munder, Matthias C.; Alberti, Simon; Outeiro, Tiago F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">3&4</span>),
    <span class="NLM_cas:pages">438-452</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Several neurodegenerative diseases, such as Parkinson's disease (PD), Alzheimer's disease (AD), Huntington's disease (HD), amyotrophic lateral sclerosis (ALS), or prion diseases, are known for their intimate assocn. with protein misfolding and aggregation.  These disorders are characterized by the loss of specific neuronal populations in the brain and are highly assocd. with aging, suggesting a decline in proteostasis capacity may contribute to pathogenesis.  Nevertheless, the precise mol. mechanisms that lead to the selective demise of neurons remain poorly understood.  As a consequence, appropriate therapeutic approaches and effective treatments are largely lacking.  The development of cellular and animal models that faithfully reproduce central aspects of neurodegeneration has been crucial for advancing our understanding of these diseases.  Approaches involving the sequential use of different model systems, starting with simpler cellular models and ending with validation in more complex animal models, resulted in the discovery of promising therapeutic targets and small mols. with therapeutic potential.  Within this framework, the simple and well-characterized eukaryote Saccharomyces cerevisiae, also known as budding yeast, is being increasingly used to study the mol. basis of several neurodegenerative disorders.  Yeast provides an unprecedented toolbox for the dissection of complex biol. processes and pathways.  Here, we summarize how yeast models are adding to our current understanding of several neurodegenerative disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvEkLUxN4SHrVg90H21EOLACvtfcHk0lgJsuyYgmgZJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWqs7fP&md5=8eb7b8266d5a994c78ec2057ab238cd8</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Fjnc.12271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjnc.12271%26sid%3Dliteratum%253Aachs%26aulast%3DTenreiro%26aufirst%3DS.%26aulast%3DMunder%26aufirst%3DM.%2BC.%26aulast%3DAlberti%26aufirst%3DS.%26aulast%3DOuteiro%26aufirst%3DT.%2BF.%26atitle%3DHarnessing%2520the%2520Power%2520of%2520Yeast%2520to%2520Unravel%2520the%2520Molecular%2520Basis%2520of%2520Neurodegeneration%26jtitle%3DJ.%2520Neurochem.%26date%3D2013%26volume%3D127%26spage%3D438%26epage%3D452%26doi%3D10.1111%2Fjnc.12271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deak, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mieyal, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roder, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson-Delfosse, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. G.</span></span> <span> </span><span class="NLM_article-title">Kinase Inhibitors Arrest Neurodegeneration in Cell and C. Elegans Models of Lrrk2 Toxicity</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">328</span>– <span class="NLM_lpage">344</span>, <span class="refDoi"> DOI: 10.1093/hmg/dds431</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1093%2Fhmg%2Fdds431" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23065705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvV2gtr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2013&pages=328-344&author=C.+Yaoauthor=W.+M.+Johnsonauthor=Y.+Gaoauthor=W.+Wangauthor=J.+Zhangauthor=M.+Deakauthor=D.+R.+Alessiauthor=X.+Zhuauthor=J.+J.+Mieyalauthor=H.+Roderauthor=A.+L.+Wilson-Delfosseauthor=S.+G.+Chen&title=Kinase+Inhibitors+Arrest+Neurodegeneration+in+Cell+and+C.+Elegans+Models+of+Lrrk2+Toxicity&doi=10.1093%2Fhmg%2Fdds431"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitors arrest neurodegeneration in cell and C. elegans models of LRRK2 toxicity</span></div><div class="casAuthors">Yao, Chen; Johnson, William M.; Gao, Yue; Wang, Wen; Zhang, Jinwei; Deak, Maria; Alessi, Dario R.; Zhu, Xiongwei; Mieyal, John J.; Roder, Hanno; Wilson-Delfosse, Amy L.; Chen, Shu G.</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">328-344</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Mutations in leucine-rich repeat kinase 2 (LRRK2) are the most frequent known cause of late-onset Parkinson's disease (PD).  To explore the therapeutic potential of small mols. targeting the LRRK2 kinase domain, we characterized two LRRK2 kinase inhibitors, TTT-3002 and LRRK2-IN1, for their effects against LRRK2 activity in vitro and in Caenorhabditis elegans models of LRRK2-linked neurodegeneration.  TTT-3002 and LRRK2-IN1 potently inhibited in vitro kinase activity of LRRK2 wild-type and mutant proteins, attenuated phosphorylation of cellular LRRK2 and rescued neurotoxicity of mutant LRRK2 in transfected cells.  To establish whether LRRK2 kinase inhibitors can mitigate pathogenesis caused by different mutations including G2019S and R1441C located within and outside of the LRRK2 kinase domain, resp., we evaluated effects of TTT-3002 and LRRK2-IN1 against R1441C- and G2019S-induced neurodegeneration in C. elegans models.  TTT-3002 and LRRK2-IN1 rescued the behavioral deficit characteristic of dopaminergic impairment in transgenic C. elegans expressing human R1441C- and G2019S-LRRK2.  The inhibitors displayed nanomolar to low micromolar rescue potency when administered either pre-symptomatically or post-symptomatically, indicating both prevention and reversal of the dopaminergic deficit.  The same treatments also led to long-lasting prevention and rescue of neurodegeneration.  In contrast, TTT-3002 and LRRK2-IN1 were ineffective against the neurodegenerative phenotype in transgenic worms carrying the inhibitor-resistant A2016T mutation of LRRK2, suggesting that they elicit neuroprotective effects in vivo by targeting LRRK2 specifically.  Our findings indicate that the LRRK2 kinase activity is crit. for neurodegeneration caused by R1441C and G2019S mutations, suggesting that kinase inhibition of LRRK2 may represent a promising therapeutic strategy for PD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7q5ZpYDN5-LVg90H21EOLACvtfcHk0ljq7s0f4aIqCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvV2gtr%252FE&md5=bbdd541fecf70129b8d42cfbd17eb5ad</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2Fdds431&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252Fdds431%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DC.%26aulast%3DJohnson%26aufirst%3DW.%2BM.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDeak%26aufirst%3DM.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DMieyal%26aufirst%3DJ.%2BJ.%26aulast%3DRoder%26aufirst%3DH.%26aulast%3DWilson-Delfosse%26aufirst%3DA.%2BL.%26aulast%3DChen%26aufirst%3DS.%2BG.%26atitle%3DKinase%2520Inhibitors%2520Arrest%2520Neurodegeneration%2520in%2520Cell%2520and%2520C.%2520Elegans%2520Models%2520of%2520Lrrk2%2520Toxicity%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2013%26volume%3D22%26spage%3D328%26epage%3D344%26doi%3D10.1093%2Fhmg%2Fdds431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barclay, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgoyne, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, A.</span></span> <span> </span><span class="NLM_article-title">Using C. Elegans to Discover Therapeutic Compounds for Ageing-Associated Neurodegenerative Diseases</span>. <i>Chem. Cent J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">65</span>, <span class="refDoi"> DOI: 10.1186/s13065-015-0143-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1186%2Fs13065-015-0143-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26617668" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC28Xpt1yht78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=65&author=X.+Chenauthor=J.+W.+Barclayauthor=R.+D.+Burgoyneauthor=A.+Morgan&title=Using+C.+Elegans+to+Discover+Therapeutic+Compounds+for+Ageing-Associated+Neurodegenerative+Diseases&doi=10.1186%2Fs13065-015-0143-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Using C. elegans to discover therapeutic compounds for ageing-associated neurodegenerative diseases</span></div><div class="casAuthors">Chen, Xi; Barclay, Jeff W.; Burgoyne, Robert D.; Morgan, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry Central Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">65/1-65/20</span>CODEN:
                <span class="NLM_cas:coden">CCJHA2</span>;
        ISSN:<span class="NLM_cas:issn">1752-153X</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Age-assocd. neurodegenerative disorders such as Alzheimer's disease are a major public health challenge, due to the demog. increase in the proportion of older individuals in society.  However, the relatively few currently approved drugs for these conditions provide only symptomatic relief.  A major goal of neurodegeneration research is therefore to identify potential new therapeutic compds. that can slow or even reverse disease progression, either by impacting directly on the neurodegenerative process or by activating endogenous physiol. neuroprotective mechanisms that decline with ageing.  This requires model systems that can recapitulate key features of human neurodegenerative diseases that are also amenable to compd. screening approaches.  Mammalian models are very powerful, but are prohibitively expensive for high-throughput drug screens.  Given the highly conserved neurol. pathways between mammals and invertebrates, Caenorhabditis elegans has emerged as a powerful tool for neuroprotective compd. screening.  Here we describe how C. elegans has been used to model various human ageing assocd. neurodegenerative diseases and provide an extensive list of compds. that have therapeutic activity in these worm models and so may have translational potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHxFDUBEtY-LVg90H21EOLACvtfcHk0ljq7s0f4aIqCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xpt1yht78%253D&md5=09b5bd342e455e23e8bd26bb67cfa0f4</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1186%2Fs13065-015-0143-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13065-015-0143-y%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DBarclay%26aufirst%3DJ.%2BW.%26aulast%3DBurgoyne%26aufirst%3DR.%2BD.%26aulast%3DMorgan%26aufirst%3DA.%26atitle%3DUsing%2520C.%2520Elegans%2520to%2520Discover%2520Therapeutic%2520Compounds%2520for%2520Ageing-Associated%2520Neurodegenerative%2520Diseases%26jtitle%3DChem.%2520Cent%2520J.%26date%3D2015%26volume%3D9%26spage%3D65%26doi%3D10.1186%2Fs13065-015-0143-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegarty, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gokce, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghorashian, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Yakar, A.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Microfluidics Providing High-Resolution and High-Throughput Screening of Caenorhabditis Elegans Poly-Glutamine Aggregation Model</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">13023</span>, <span class="refDoi"> DOI: 10.1038/ncomms13023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fncomms13023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=27725672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1GnurjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=13023&author=S.+Mondalauthor=E.+Hegartyauthor=C.+Martinauthor=S.+K.+Gokceauthor=N.+Ghorashianauthor=A.+Ben-Yakar&title=Large-Scale+Microfluidics+Providing+High-Resolution+and+High-Throughput+Screening+of+Caenorhabditis+Elegans+Poly-Glutamine+Aggregation+Model&doi=10.1038%2Fncomms13023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Large-scale microfluidics providing high-resolution and high-throughput screening of Caenorhabditis elegans poly-glutamine aggregation model</span></div><div class="casAuthors">Mondal, Sudip; Hegarty, Evan; Martin, Chris; Gokce, Sertan Kutal; Ghorashian, Navid; Ben-Yakar, Adela</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13023</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Next generation drug screening could benefit greatly from in vivo studies, using small animal models such as Caenorhabditis elegans for hit identification and lead optimization.  Current in vivo assays can operate either at low throughput with high resoln. or with low resoln. at high throughput.  To enable both high-throughput and high-resoln. imaging of C. elegans, we developed an automated microfluidic platform.  This platform can image 15 z-stacks of ∼4,000 C. elegans from 96 different populations using a large-scale chip with a micron resoln. in 16 min.  Using this platform, we screened ∼100,000 animals of the poly-glutamine aggregation model on 25 chips.  We tested the efficacy of ∼1,000 FDA-approved drugs in improving the aggregation phenotype of the model and identified four confirmed hits.  This robust platform now enables high-content screening of various C. elegans disease models at the speed and cost of in vitro cell-based assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaM5q1Y9A3WbVg90H21EOLACvtfcHk0ljq7s0f4aIqCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1GnurjK&md5=2acb85ed4d0665977ac466ce94b7bb71</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fncomms13023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13023%26sid%3Dliteratum%253Aachs%26aulast%3DMondal%26aufirst%3DS.%26aulast%3DHegarty%26aufirst%3DE.%26aulast%3DMartin%26aufirst%3DC.%26aulast%3DGokce%26aufirst%3DS.%2BK.%26aulast%3DGhorashian%26aufirst%3DN.%26aulast%3DBen-Yakar%26aufirst%3DA.%26atitle%3DLarge-Scale%2520Microfluidics%2520Providing%2520High-Resolution%2520and%2520High-Throughput%2520Screening%2520of%2520Caenorhabditis%2520Elegans%2520Poly-Glutamine%2520Aggregation%2520Model%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D13023%26doi%3D10.1038%2Fncomms13023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Reilly, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luke, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perlmutter, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pak, S. C. C.</span></span> <span> </span><span class="NLM_article-title">Elegans in High-Throughput Drug Discovery</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>69–70</i></span>,  <span class="NLM_fpage">247</span>– <span class="NLM_lpage">253</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2013.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.addr.2013.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=24333896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOhsLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69%E2%80%9370&publication_year=2014&pages=247-253&author=L.+P.+O%E2%80%99Reillyauthor=C.+J.+Lukeauthor=D.+H.+Perlmutterauthor=G.+A.+Silvermanauthor=S.+C.+C.+Pak&title=Elegans+in+High-Throughput+Drug+Discovery&doi=10.1016%2Fj.addr.2013.12.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">C. elegans in high-throughput drug discovery</span></div><div class="casAuthors">O'Reilly, Linda P.; Luke, Cliff J.; Perlmutter, David H.; Silverman, Gary A.; Pak, Stephen C.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">69-70</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">247-253</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Caenorhabditis elegans has been proven to be a useful model organism for investigating mol. and cellular aspects of numerous human diseases.  More recently, investigators have explored the use of this organism as a tool for drug discovery.  Although earlier drug screens were labor-intensive and low in throughput, recent advances in high-throughput liq. workflows, imaging platforms and data anal. software have made C. elegans a viable option for automated high-throughput drug screens.  This review will outline the evolution of C. elegans-based drug screening, discuss the inherent challenges of using C. elegans, and highlight recent technol. advances that have paved the way for future drug screens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHxn3l10Eju7Vg90H21EOLACvtfcHk0lis8CAu0Epteg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOhsLbM&md5=112e983d48ccf79328f65e8af70f7c33</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2013.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2013.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Reilly%26aufirst%3DL.%2BP.%26aulast%3DLuke%26aufirst%3DC.%2BJ.%26aulast%3DPerlmutter%26aufirst%3DD.%2BH.%26aulast%3DSilverman%26aufirst%3DG.%2BA.%26aulast%3DPak%26aufirst%3DS.%2BC.%2BC.%26atitle%3DElegans%2520in%2520High-Throughput%2520Drug%2520Discovery%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2014%26volume%3D69%25E2%2580%259370%26spage%3D247%26epage%3D253%26doi%3D10.1016%2Fj.addr.2013.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shaye, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwald, I.</span></span> <span> </span><span class="NLM_article-title">Ortholist: A Compendium of C. Elegans Genes with Human Orthologs</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e20085</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0020085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1371%2Fjournal.pone.0020085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=21647448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFCku7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=D.+D.+Shayeauthor=I.+Greenwald&title=Ortholist%3A+A+Compendium+of+C.+Elegans+Genes+with+Human+Orthologs&doi=10.1371%2Fjournal.pone.0020085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">OrthoList: a compendium of C. elegans genes with human orthologs</span></div><div class="casAuthors">Shaye, Daniel D.; Greenwald, Iva</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e20085</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">C. elegans is an important model for genetic studies relevant to human biol. and disease.  We sought to assess the orthol. between C. elegans and human genes to understand better the relationship between their genomes and to generate a compelling list of candidates to streamline RNAi-based screens in this model.  We performed a meta-anal. of results from four orthol. prediction programs and generated a compendium, "OrthoList", contg. 7,663 C. elegans protein-coding genes.  Various assessments indicate that OrthoList has extensive coverage with low false-pos. and false-neg. rates.  Part of this evaluation examd. the conservation of components of the receptor tyrosine kinase, Notch, Wnt, TGF-β and insulin signaling pathways, and led us to update compendia of conserved C. elegans kinases, nuclear hormone receptors, F-box proteins, and transcription factors.  Comparison with two published genome-wide RNAi screens indicated that virtually all of the conserved hits would have been obtained had just the OrthoList set (∼38% of the genome) been targeted.  We compiled OrthoList by InterPro domains and Gene Ontol. annotation, making it easy to identify C. elegans orthologs of human disease genes for potential functional anal.  We anticipate that OrthoList will be of considerable utility to C. elegans researchers for streamlining RNAi screens, by focusing on genes with apparent human orthologs, thus reducing screening effort by ∼60%.  Moreover, we find that OrthoList provides a useful basis for annotating orthol. and reveals more C. elegans orthologs of human genes in various functional groups, such as transcription factors, than previously described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxUjExRD4rXrVg90H21EOLACvtfcHk0lis8CAu0Epteg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvFCku7s%253D&md5=c73566c6c292b890536340ddd8c66580</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0020085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0020085%26sid%3Dliteratum%253Aachs%26aulast%3DShaye%26aufirst%3DD.%2BD.%26aulast%3DGreenwald%26aufirst%3DI.%26atitle%3DOrtholist%253A%2520A%2520Compendium%2520of%2520C.%2520Elegans%2520Genes%2520with%2520Human%2520Orthologs%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0020085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marfil, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span> <span> </span><span class="NLM_article-title">Use of Caenorhabditis Elegans as a Model to Study Alzheimer’s Disease and Other Neurodegenerative Diseases</span>. <i>Front. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">279</span>, <span class="refDoi"> DOI: 10.3389/fgene.2014.00279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.3389%2Ffgene.2014.00279" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25250042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC2M7kt1CktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=279&author=A.+G.+Alexanderauthor=V.+Marfilauthor=C.+Li&title=Use+of+Caenorhabditis+Elegans+as+a+Model+to+Study+Alzheimer%E2%80%99s+Disease+and+Other+Neurodegenerative+Diseases&doi=10.3389%2Ffgene.2014.00279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Use of Caenorhabditis elegans as a model to study Alzheimer's disease and other neurodegenerative diseases</span></div><div class="casAuthors">Alexander Adanna G; Li Chris; Marfil Vanessa</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in genetics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">279</span>
        ISSN:<span class="NLM_cas:issn">1664-8021</span>.
    </div><div class="casAbstract">Advances in research and technology has increased our quality of life, allowed us to combat diseases, and achieve increased longevity.  Unfortunately, increased longevity is accompanied by a rise in the incidences of age-related diseases such as Alzheimer's disease (AD).  AD is the sixth leading cause of death, and one of the leading causes of dementia amongst the aged population in the USA.  It is a progressive neurodegenerative disorder, characterized by the prevalence of extracellular Aβ plaques and intracellular neurofibrillary tangles, derived from the proteolysis of the amyloid precursor protein (APP) and the hyperphosphorylation of microtubule-associated protein tau, respectively.  Despite years of extensive research, the molecular mechanisms that underlie the pathology of AD remain unclear.  Model organisms, such as the nematode, Caenorhabditis elegans, present a complementary approach to addressing these questions.  C. elegans has many advantages as a model system to study AD and other neurodegenerative diseases.  Like their mammalian counterparts, they have complex biochemical pathways, most of which are conserved.  Genes in which mutations are correlated with AD have counterparts in C. elegans, including an APP-related gene, apl-1, a tau homolog, ptl-1, and presenilin homologs, such as sel-12 and hop-1.  Since the neuronal connectivity in C. elegans has already been established, C. elegans is also advantageous in modeling learning and memory impairments seen during AD.  This article addresses the insights C. elegans provide in studying AD and other neurodegenerative diseases.  Additionally, we explore the advantages and drawbacks associated with using this model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRa3vraTEWAPtFxYDaSSP8XfW6udTcc2ebSTHwFOBnPdrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M7kt1CktA%253D%253D&md5=7324b86ac9589e7e2aa8185a6903327f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.3389%2Ffgene.2014.00279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffgene.2014.00279%26sid%3Dliteratum%253Aachs%26aulast%3DAlexander%26aufirst%3DA.%2BG.%26aulast%3DMarfil%26aufirst%3DV.%26aulast%3DLi%26aufirst%3DC.%26atitle%3DUse%2520of%2520Caenorhabditis%2520Elegans%2520as%2520a%2520Model%2520to%2520Study%2520Alzheimer%25E2%2580%2599s%2520Disease%2520and%2520Other%2520Neurodegenerative%2520Diseases%26jtitle%3DFront.%2520Genet.%26date%3D2014%26volume%3D5%26spage%3D279%26doi%3D10.3389%2Ffgene.2014.00279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGurk, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonini, N. M.</span></span> <span> </span><span class="NLM_article-title">Drosophila as an in Vivo Model for Human Neurodegenerative Disease</span>. <i>Genetics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>201</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">402</span>, <span class="refDoi"> DOI: 10.1534/genetics.115.179457</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1534%2Fgenetics.115.179457" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26447127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC28XlslGqs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=201&publication_year=2015&pages=377-402&author=L.+McGurkauthor=A.+Bersonauthor=N.+M.+Bonini&title=Drosophila+as+an+in+Vivo+Model+for+Human+Neurodegenerative+Disease&doi=10.1534%2Fgenetics.115.179457"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Drosophila as an in vivo model for human neurodegenerative disease</span></div><div class="casAuthors">McGurk, Leeanne; Berson, Amit; Bonini, Nancy M.</div><div class="citationInfo"><span class="NLM_cas:title">Genetics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">201</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-402</span>CODEN:
                <span class="NLM_cas:coden">GENTAE</span>;
        ISSN:<span class="NLM_cas:issn">0016-6731</span>.
    
            (<span class="NLM_cas:orgname">Genetics Society of America</span>)
        </div><div class="casAbstract">With the increase in the ageing population, neurodegenerative disease is devastating to families and poses a huge burden on society.  The brain and spinal cord are extraordinarily complex: they consist of a highly organized network of neuronal and support cells that communicate in a highly specialized manner.  One approach to tackling problems of such complexity is to address the scientific questions in simpler, yet analogous, systems.  The fruit fly, Drosophila melanogaster, has been proven tremendously valuable as a model organism, enabling many major discoveries in neuroscientific disease research.  The plethora of genetic tools available in Drosophila allows for exquisite targeted manipulation of the genome.  Due to its relatively short lifespan, complex questions of brain function can be addressed more rapidly than in other model organisms, such as the mouse.  Here we discuss features of the fly as a model for human neurodegenerative disease.  There are many distinct fly models for a range of neurodegenerative diseases; we focus on select studies from models of polyglutamine disease and amyotrophic lateral sclerosis that illustrate the type and range of insights that can be gleaned.  In discussion of these models, we underscore strengths of the fly in providing understanding into mechanisms and pathways, as a foundation for translational and therapeutic research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhbyqPD12u_LVg90H21EOLACvtfcHk0lgkDWCmGBxycg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlslGqs7g%253D&md5=f30a80149c7f18e7a1247b326134d528</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1534%2Fgenetics.115.179457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1534%252Fgenetics.115.179457%26sid%3Dliteratum%253Aachs%26aulast%3DMcGurk%26aufirst%3DL.%26aulast%3DBerson%26aufirst%3DA.%26aulast%3DBonini%26aufirst%3DN.%2BM.%26atitle%3DDrosophila%2520as%2520an%2520in%2520Vivo%2520Model%2520for%2520Human%2520Neurodegenerative%2520Disease%26jtitle%3DGenetics%26date%3D2015%26volume%3D201%26spage%3D377%26epage%3D402%26doi%3D10.1534%2Fgenetics.115.179457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potocki, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chien, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gribskov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bier, E.</span></span> <span> </span><span class="NLM_article-title">A Systematic Analysis of Human Disease-Associated Gene Sequences in Drosophila Melanogaster</span>. <i>Genome Res.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1114</span>– <span class="NLM_lpage">1125</span>, <span class="refDoi"> DOI: 10.1101/gr.169101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1101%2Fgr.169101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=11381037" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BD3MXkt12ru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=1114-1125&author=L.+T.+Reiterauthor=L.+Potockiauthor=S.+Chienauthor=M.+Gribskovauthor=E.+Bier&title=A+Systematic+Analysis+of+Human+Disease-Associated+Gene+Sequences+in+Drosophila+Melanogaster&doi=10.1101%2Fgr.169101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A systematic analysis of human disease-associated gene sequences in Drosophila melanogaster</span></div><div class="casAuthors">Reiter, Lawrence T.; Potocki, Lorraine; Chien, Sam; Gribskov, Michael; Bier, Ethan</div><div class="citationInfo"><span class="NLM_cas:title">Genome Research</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1114-1125</span>CODEN:
                <span class="NLM_cas:coden">GEREFS</span>;
        ISSN:<span class="NLM_cas:issn">1088-9051</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">We performed a systematic BLAST anal. of 929 human disease gene entries assocd. with at least one mutant allele in the Online Mendelian Inheritance in Man (OMIM) database against the recently completed genome sequence of Drosophila melanogaster.  The results of this search have been formatted as an updateable and searchable online database called Homophila.  Our anal. identified 714 distinct human disease genes (77% of disease genes searched) matching 548 unique Drosophila sequences, which we have summarized by disease category.  This breakdown into disease classes creates a picture of disease genes that are amenable to study using Drosophila as the model organism.  Of the 548 Drosophila genes related to human disease genes, 153 are assocd. with known mutant alleles and 56 more are tagged by P-element insertions in or near the gene.  Examples of how to use the database to identify Drosophila genes related to human disease genes are presented.  We anticipate that cross-genomic anal. of human disease genes using the power of Drosophila second-site modifier screens will promote interaction between human and Drosophila research groups, accelerating the understanding of the pathogenesis of human genetic disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodyv8TO5yrqbVg90H21EOLACvtfcHk0lh6TNhfVqCY9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXkt12ru7s%253D&md5=9eec727a7b935b3f5fe3783e93fe55d5</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1101%2Fgr.169101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgr.169101%26sid%3Dliteratum%253Aachs%26aulast%3DReiter%26aufirst%3DL.%2BT.%26aulast%3DPotocki%26aufirst%3DL.%26aulast%3DChien%26aufirst%3DS.%26aulast%3DGribskov%26aufirst%3DM.%26aulast%3DBier%26aufirst%3DE.%26atitle%3DA%2520Systematic%2520Analysis%2520of%2520Human%2520Disease-Associated%2520Gene%2520Sequences%2520in%2520Drosophila%2520Melanogaster%26jtitle%3DGenome%2520Res.%26date%3D2001%26volume%3D11%26spage%3D1114%26epage%3D1125%26doi%3D10.1101%2Fgr.169101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rihel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prober, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arvanites, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schier, A. F.</span></span> <span> </span><span class="NLM_article-title">Zebrafish Behavioral Profiling Links Drugs to Biological Targets and Rest/Wake Regulation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>327</i></span>,  <span class="NLM_fpage">348</span>– <span class="NLM_lpage">351</span>, <span class="refDoi"> DOI: 10.1126/science.1183090</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fscience.1183090" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=20075256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktlWksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=348-351&author=J.+Rihelauthor=D.+A.+Proberauthor=A.+Arvanitesauthor=K.+Lamauthor=S.+Zimmermanauthor=S.+Jangauthor=S.+J.+Haggartyauthor=D.+Kokelauthor=L.+L.+Rubinauthor=R.+T.+Petersonauthor=A.+F.+Schier&title=Zebrafish+Behavioral+Profiling+Links+Drugs+to+Biological+Targets+and+Rest%2FWake+Regulation&doi=10.1126%2Fscience.1183090"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Zebrafish Behavioral Profiling Links Drugs to Biological Targets and Rest/Wake Regulation</span></div><div class="casAuthors">Rihel, Jason; Prober, David A.; Arvanites, Anthony; Lam, Kelvin; Zimmerman, Steven; Jang, Sumin; Haggarty, Stephen J.; Kokel, David; Rubin, Lee L.; Peterson, Randall T.; Schier, Alexander F.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5963</span>),
    <span class="NLM_cas:pages">348-351</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A major obstacle for the discovery of psychoactive drugs is the inability to predict how small mols. will alter complex behaviors.  We report the development and application of a high-throughput, quant. screen for drugs that alter the behavior of larval zebrafish.  We found that the multidimensional nature of obsd. phenotypes enabled the hierarchical clustering of mols. according to shared behaviors.  Behavioral profiling revealed conserved functions of psychotropic mols. and predicted the mechanisms of action of poorly characterized compds.  In addn., behavioral profiling implicated new factors such as ether-a-go-go-related gene (ERG) potassium channels and immunomodulators in the control of rest and locomotor activity.  These results demonstrate the power of high-throughput behavioral profiling in zebrafish to discover and characterize psychotropic drugs and to dissect the pharmacol. of complex behaviors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbwHl9CPNU_7Vg90H21EOLACvtfcHk0lh6TNhfVqCY9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktlWksw%253D%253D&md5=6a039c0c06cba3a364230aa24f84e13e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1126%2Fscience.1183090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1183090%26sid%3Dliteratum%253Aachs%26aulast%3DRihel%26aufirst%3DJ.%26aulast%3DProber%26aufirst%3DD.%2BA.%26aulast%3DArvanites%26aufirst%3DA.%26aulast%3DLam%26aufirst%3DK.%26aulast%3DZimmerman%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DS.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DKokel%26aufirst%3DD.%26aulast%3DRubin%26aufirst%3DL.%2BL.%26aulast%3DPeterson%26aufirst%3DR.%2BT.%26aulast%3DSchier%26aufirst%3DA.%2BF.%26atitle%3DZebrafish%2520Behavioral%2520Profiling%2520Links%2520Drugs%2520to%2520Biological%2520Targets%2520and%2520Rest%252FWake%2520Regulation%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D348%26epage%3D351%26doi%3D10.1126%2Fscience.1183090" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">MacRae, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, R. T.</span></span> <span> </span><span class="NLM_article-title">Zebrafish as Tools for Drug Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">721</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1038/nrd4627</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnrd4627" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26361349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyrtbzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=721-731&author=C.+A.+MacRaeauthor=R.+T.+Peterson&title=Zebrafish+as+Tools+for+Drug+Discovery&doi=10.1038%2Fnrd4627"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Zebrafish as tools for drug discovery</span></div><div class="casAuthors">MacRae, Calum A.; Peterson, Randall T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">721-731</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The zebrafish has become a prominent vertebrate model for disease and has already contributed to several examples of successful phenotype-based drug discovery.  For the zebrafish to become useful in drug development more broadly, key hurdles must be overcome, including a more comprehensive elucidation of the similarities and differences between human and zebrafish biol.  Recent studies have begun to establish the capabilities and limitations of zebrafish for disease modeling, drug screening, target identification, pharmacol., and toxicol.  As our understanding increases and as the technologies for manipulating zebrafish improve, it is hoped that the zebrafish will have a key role in accelerating the emergence of precision medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGcnVtHqzJXbVg90H21EOLACvtfcHk0lhbgSik4eJSdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyrtbzJ&md5=1f8733e7468a76738df9c130c493a3b8</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnrd4627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4627%26sid%3Dliteratum%253Aachs%26aulast%3DMacRae%26aufirst%3DC.%2BA.%26aulast%3DPeterson%26aufirst%3DR.%2BT.%26atitle%3DZebrafish%2520as%2520Tools%2520for%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D721%26epage%3D731%26doi%3D10.1038%2Fnrd4627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parenteau, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veronneau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacombe, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecez, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gervais-Bird, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koh, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunelle, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellinger, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chabot, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abou Elela, S.</span></span> <span> </span><span class="NLM_article-title">Deletion of Many Yeast Introns Reveals a Minority of Genes That Require Splicing for Function</span>. <i>Mol. Biol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1932</span>– <span class="NLM_lpage">1941</span>, <span class="refDoi"> DOI: 10.1091/mbc.e07-12-1254</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1091%2Fmbc.e07-12-1254" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=18287520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlslelsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2008&pages=1932-1941&author=J.+Parenteauauthor=M.+Durandauthor=S.+Veronneauauthor=A.+A.+Lacombeauthor=G.+Morinauthor=V.+Guerinauthor=B.+Cecezauthor=J.+Gervais-Birdauthor=C.+S.+Kohauthor=D.+Brunelleauthor=R.+J.+Wellingerauthor=B.+Chabotauthor=S.+Abou+Elela&title=Deletion+of+Many+Yeast+Introns+Reveals+a+Minority+of+Genes+That+Require+Splicing+for+Function&doi=10.1091%2Fmbc.e07-12-1254"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Deletion of many yeast introns reveals a minority of genes that require splicing for function</span></div><div class="casAuthors">Parenteau, Julie; Durand, Mathieu; Veronneau, Steeve; Lacombe, Andree-Anne; Morin, Genevieve; Guerin, Valerie; Cecez, Bojana; Gervais-Bird, Julien; Koh, Chu-Shin; Brunelle, David; Wellinger, Raymund J.; Chabot, Benoit; Elela, Sherif Abou</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Biology of the Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1932-1941</span>CODEN:
                <span class="NLM_cas:coden">MBCEEV</span>;
        ISSN:<span class="NLM_cas:issn">1939-4586</span>.
    
            (<span class="NLM_cas:orgname">American Society for Cell Biology</span>)
        </div><div class="casAbstract">Splicing regulates gene expression and contributes to proteomic diversity in higher eukaryotes.  However, in yeast only 283 of the 6000 genes contain introns and their impact on cell function is not clear.  To assess the contribution of introns to cell function, we initiated large-scale intron deletions in yeast with the ultimate goal of creating an intron-free model eukaryote.  We show that about one-third of yeast introns are not essential for growth.  Only three intron deletions caused severe growth defects, but normal growth was restored in all cases by expressing the intronless mRNA from a heterologous promoter.  Twenty percent of the intron deletions caused minor phenotypes under different growth conditions.  Strikingly, the combined deletion of all introns from the 15 cytoskeleton-related genes did not affect growth or strain fitness.  Together, our results show that although the presence of introns may optimize gene expression and provide benefit under stress, a majority of introns could be removed with minor consequences on growth under lab. conditions, supporting the view that many introns could be phased out of Saccharomyces cerevisiae without blocking cell growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvTOfcnp6-VbVg90H21EOLACvtfcHk0lhbgSik4eJSdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlslelsrc%253D&md5=85dee50c2433d9800359059e7d97afae</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1091%2Fmbc.e07-12-1254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1091%252Fmbc.e07-12-1254%26sid%3Dliteratum%253Aachs%26aulast%3DParenteau%26aufirst%3DJ.%26aulast%3DDurand%26aufirst%3DM.%26aulast%3DVeronneau%26aufirst%3DS.%26aulast%3DLacombe%26aufirst%3DA.%2BA.%26aulast%3DMorin%26aufirst%3DG.%26aulast%3DGuerin%26aufirst%3DV.%26aulast%3DCecez%26aufirst%3DB.%26aulast%3DGervais-Bird%26aufirst%3DJ.%26aulast%3DKoh%26aufirst%3DC.%2BS.%26aulast%3DBrunelle%26aufirst%3DD.%26aulast%3DWellinger%26aufirst%3DR.%2BJ.%26aulast%3DChabot%26aufirst%3DB.%26aulast%3DAbou%2BElela%26aufirst%3DS.%26atitle%3DDeletion%2520of%2520Many%2520Yeast%2520Introns%2520Reveals%2520a%2520Minority%2520of%2520Genes%2520That%2520Require%2520Splicing%2520for%2520Function%26jtitle%3DMol.%2520Biol.%2520Cell%26date%3D2008%26volume%3D19%26spage%3D1932%26epage%3D1941%26doi%3D10.1091%2Fmbc.e07-12-1254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonnhammer, E. L.</span></span> <span> </span><span class="NLM_article-title">Inparanoid: A Comprehensive Database of Eukaryotic Orthologs</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">D476</span>– <span class="NLM_lpage">D480</span>, <span class="refDoi"> DOI: 10.1093/nar/gki107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1093%2Fnar%2Fgki107" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2004&pages=D476-D480&author=K.+P.+O%E2%80%99Brienauthor=M.+Remmauthor=E.+L.+Sonnhammer&title=Inparanoid%3A+A+Comprehensive+Database+of+Eukaryotic+Orthologs&doi=10.1093%2Fnar%2Fgki107"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgki107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgki107%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DK.%2BP.%26aulast%3DRemm%26aufirst%3DM.%26aulast%3DSonnhammer%26aufirst%3DE.%2BL.%26atitle%3DInparanoid%253A%2520A%2520Comprehensive%2520Database%2520of%2520Eukaryotic%2520Orthologs%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2004%26volume%3D33%26spage%3DD476%26epage%3DD480%26doi%3D10.1093%2Fnar%2Fgki107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kachroo, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurent, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yellman, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilke, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, E. M.</span></span> <span> </span><span class="NLM_article-title">Evolution. Systematic Humanization of Yeast Genes Reveals Conserved Functions and Genetic Modularity</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">921</span>– <span class="NLM_lpage">925</span>, <span class="refDoi"> DOI: 10.1126/science.aaa0769</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fscience.aaa0769" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25999509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosFaksbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=921-925&author=A.+H.+Kachrooauthor=J.+M.+Laurentauthor=C.+M.+Yellmanauthor=A.+G.+Meyerauthor=C.+O.+Wilkeauthor=E.+M.+Marcotte&title=Evolution.+Systematic+Humanization+of+Yeast+Genes+Reveals+Conserved+Functions+and+Genetic+Modularity&doi=10.1126%2Fscience.aaa0769"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic humanization of yeast genes reveals conserved functions and genetic modularity</span></div><div class="casAuthors">Kachroo, Aashiq H.; Laurent, Jon M.; Yellman, Christopher M.; Meyer, Austin G.; Wilke, Claus O.; Marcotte, Edward M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6237</span>),
    <span class="NLM_cas:pages">921-925</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">To det. whether genes retain ancestral functions over a billion years of evolution and to identify principles of deep evolutionary divergence, we replaced 414 essential yeast genes with their human orthologs, assaying for complementation of lethal growth defects upon loss of the yeast genes.  Nearly half (47%) of the yeast genes could be successfully humanized.  Sequence similarity and expression only partly predicted replaceability.  Instead, replaceability depended strongly on gene modules: Genes in the same process tended to be similarly replaceable (e.g., sterol biosynthesis) or not (e.g., DNA replication initiation).  Simulations confirmed that selection for specific function can maintain replaceability despite extensive sequence divergence.  Crit. ancestral functions of many essential genes are thus retained in a pathway-specific manner, resilient to drift in sequences, splicing, and protein interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqPJvZKreAPLVg90H21EOLACvtfcHk0lhbgSik4eJSdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosFaksbg%253D&md5=422322e621ad3d1ad31e99195f9e3ead</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaa0769&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaa0769%26sid%3Dliteratum%253Aachs%26aulast%3DKachroo%26aufirst%3DA.%2BH.%26aulast%3DLaurent%26aufirst%3DJ.%2BM.%26aulast%3DYellman%26aufirst%3DC.%2BM.%26aulast%3DMeyer%26aufirst%3DA.%2BG.%26aulast%3DWilke%26aufirst%3DC.%2BO.%26aulast%3DMarcotte%26aufirst%3DE.%2BM.%26atitle%3DEvolution.%2520Systematic%2520Humanization%2520of%2520Yeast%2520Genes%2520Reveals%2520Conserved%2520Functions%2520and%2520Genetic%2520Modularity%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D921%26epage%3D925%26doi%3D10.1126%2Fscience.aaa0769" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tardiff, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piotrowski, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aron, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bacherman, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pires, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramaniam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doshi, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadlish, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raja, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solis, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khurana, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Bourdonnec, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scannevin, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhodes, K. J.</span></span> <span> </span><span class="NLM_article-title">Inhibiting Stearoyl-Coa Desaturase Ameliorates Alpha-Synuclein Cytotoxicity</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2742</span>– <span class="NLM_lpage">2754</span>, e2731 <span class="refDoi"> DOI: 10.1016/j.celrep.2018.11.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.celrep.2018.11.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30517862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVCit7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=2742-2754&author=B.+M.+Vincentauthor=D.+F.+Tardiffauthor=J.+S.+Piotrowskiauthor=R.+Aronauthor=M.+C.+Lucasauthor=C.+Y.+Chungauthor=H.+Bachermanauthor=Y.+Chenauthor=M.+Piresauthor=R.+Subramaniamauthor=D.+B.+Doshiauthor=H.+Sadlishauthor=W.+K.+Rajaauthor=E.+J.+Solisauthor=V.+Khuranaauthor=B.+Le+Bourdonnecauthor=R.+H.+Scannevinauthor=K.+J.+Rhodes&title=Inhibiting+Stearoyl-Coa+Desaturase+Ameliorates+Alpha-Synuclein+Cytotoxicity&doi=10.1016%2Fj.celrep.2018.11.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting Stearoyl-CoA Desaturase Ameliorates α-Synuclein Cytotoxicity</span></div><div class="casAuthors">Vincent, Benjamin M.; Tardiff, Daniel F.; Piotrowski, Jeff S.; Aron, Rebecca; Lucas, Matthew C.; Chung, Chee Yeun; Bacherman, Helene; Chen, YiQun; Pires, Michelle; Subramaniam, Radha; Doshi, Dimple B.; Sadlish, Heather; Raja, Waseem K.; Solis, Eric J.; Khurana, Vikram; Le Bourdonnec, Bertrand; Scannevin, Robert H.; Rhodes, Kenneth J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2742-2754.e31</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The lack of disease-modifying treatments for neurodegenerative disease stems in part from our rudimentary understanding of disease mechanisms and the paucity of targets for therapeutic intervention.  Here we used an integrated discovery paradigm to identify a new therapeutic target for diseases caused by α-synuclein (α-syn), a small lipid-binding protein that misfolds and aggregates in Parkinson's disease and other disorders.  Using unbiased phenotypic screening, we identified a series of compds. that were cytoprotective against α-syn-mediated toxicity by inhibiting the highly conserved enzyme stearoyl-CoA desaturase (SCD).  Critically, reducing the levels of unsatd. membrane lipids by inhibiting SCD reduced α-syn toxicity in human induced pluripotent stem cell (iPSC) neuronal models.  Taken together, these findings suggest that inhibition of fatty acid desatn. has potential as a therapeutic approach for the treatment of Parkinson's disease and other synucleinopathies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2xtNxbIcEk7Vg90H21EOLACvtfcHk0ljWru72-S5NGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVCit7fM&md5=1e1a2ffb251ff2a91f638b9a247cb2c0</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2018.11.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2018.11.028%26sid%3Dliteratum%253Aachs%26aulast%3DVincent%26aufirst%3DB.%2BM.%26aulast%3DTardiff%26aufirst%3DD.%2BF.%26aulast%3DPiotrowski%26aufirst%3DJ.%2BS.%26aulast%3DAron%26aufirst%3DR.%26aulast%3DLucas%26aufirst%3DM.%2BC.%26aulast%3DChung%26aufirst%3DC.%2BY.%26aulast%3DBacherman%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DPires%26aufirst%3DM.%26aulast%3DSubramaniam%26aufirst%3DR.%26aulast%3DDoshi%26aufirst%3DD.%2BB.%26aulast%3DSadlish%26aufirst%3DH.%26aulast%3DRaja%26aufirst%3DW.%2BK.%26aulast%3DSolis%26aufirst%3DE.%2BJ.%26aulast%3DKhurana%26aufirst%3DV.%26aulast%3DLe%2BBourdonnec%26aufirst%3DB.%26aulast%3DScannevin%26aufirst%3DR.%2BH.%26aulast%3DRhodes%26aufirst%3DK.%2BJ.%26atitle%3DInhibiting%2520Stearoyl-Coa%2520Desaturase%2520Ameliorates%2520Alpha-Synuclein%2520Cytotoxicity%26jtitle%3DCell%2520Rep.%26date%3D2018%26volume%3D25%26spage%3D2742%26epage%3D2754%26doi%3D10.1016%2Fj.celrep.2018.11.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auluck, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Outeiro, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeger-Lotem, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kritzer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tardiff, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strathearn, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamamichi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraenkel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCaffery, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rochet, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindquist, S.</span></span> <span> </span><span class="NLM_article-title">Compounds from an Unbiased Chemical Screen Reverse Both Er-to-Golgi Trafficking Defects and Mitochondrial Dysfunction in Parkinson’s Disease Models</span>. <i>Dis. Models &amp; Mech.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">194</span>– <span class="NLM_lpage">208</span>, <span class="refDoi"> DOI: 10.1242/dmm.004267</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1242%2Fdmm.004267" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=20038714" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2010&pages=194-208&author=L.+J.+Suauthor=P.+K.+Auluckauthor=T.+F.+Outeiroauthor=E.+Yeger-Lotemauthor=J.+A.+Kritzerauthor=D.+F.+Tardiffauthor=K.+E.+Strathearnauthor=F.+Liuauthor=S.+Caoauthor=S.+Hamamichiauthor=K.+J.+Hillauthor=K.+A.+Caldwellauthor=G.+W.+Bellauthor=E.+Fraenkelauthor=A.+A.+Cooperauthor=G.+A.+Caldwellauthor=J.+M.+McCafferyauthor=J.+C.+Rochetauthor=S.+Lindquist&title=Compounds+from+an+Unbiased+Chemical+Screen+Reverse+Both+Er-to-Golgi+Trafficking+Defects+and+Mitochondrial+Dysfunction+in+Parkinson%E2%80%99s+Disease+Models&doi=10.1242%2Fdmm.004267"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1242%2Fdmm.004267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fdmm.004267%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DL.%2BJ.%26aulast%3DAuluck%26aufirst%3DP.%2BK.%26aulast%3DOuteiro%26aufirst%3DT.%2BF.%26aulast%3DYeger-Lotem%26aufirst%3DE.%26aulast%3DKritzer%26aufirst%3DJ.%2BA.%26aulast%3DTardiff%26aufirst%3DD.%2BF.%26aulast%3DStrathearn%26aufirst%3DK.%2BE.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DCao%26aufirst%3DS.%26aulast%3DHamamichi%26aufirst%3DS.%26aulast%3DHill%26aufirst%3DK.%2BJ.%26aulast%3DCaldwell%26aufirst%3DK.%2BA.%26aulast%3DBell%26aufirst%3DG.%2BW.%26aulast%3DFraenkel%26aufirst%3DE.%26aulast%3DCooper%26aufirst%3DA.%2BA.%26aulast%3DCaldwell%26aufirst%3DG.%2BA.%26aulast%3DMcCaffery%26aufirst%3DJ.%2BM.%26aulast%3DRochet%26aufirst%3DJ.%2BC.%26aulast%3DLindquist%26aufirst%3DS.%26atitle%3DCompounds%2520from%2520an%2520Unbiased%2520Chemical%2520Screen%2520Reverse%2520Both%2520Er-to-Golgi%2520Trafficking%2520Defects%2520and%2520Mitochondrial%2520Dysfunction%2520in%2520Parkinson%25E2%2580%2599s%2520Disease%2520Models%26jtitle%3DDis.%2520Models%2520%2526amp%253B%2520Mech.%26date%3D2010%26volume%3D3%26spage%3D194%26epage%3D208%26doi%3D10.1242%2Fdmm.004267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halliday, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radford, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zents, K. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molloy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verity, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortori, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bushell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallucci, G. R.</span></span> <span> </span><span class="NLM_article-title">Repurposed Drugs Targeting Eif2&Alpha;-P-Mediated Translational Repression Prevent Neurodegeneration in Mice</span>. <i>Brain</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">1768</span>– <span class="NLM_lpage">1783</span>, <span class="refDoi"> DOI: 10.1093/brain/awx074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1093%2Fbrain%2Fawx074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28430857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC1cvpvFGnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2017&pages=1768-1783&author=M.+Hallidayauthor=H.+Radfordauthor=K.+A.+M.+Zentsauthor=C.+Molloyauthor=J.+A.+Morenoauthor=N.+C.+Verityauthor=E.+Smithauthor=C.+A.+Ortoriauthor=D.+A.+Barrettauthor=M.+Bushellauthor=G.+R.+Mallucci&title=Repurposed+Drugs+Targeting+Eif2%26Alpha%3B-P-Mediated+Translational+Repression+Prevent+Neurodegeneration+in+Mice&doi=10.1093%2Fbrain%2Fawx074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Repurposed drugs targeting eIF2&alpha;-P-mediated translational repression prevent neurodegeneration in mice</span></div><div class="casAuthors">Halliday Mark; Radford Helois; Molloy Collin; Moreno Julie A; Verity Nicholas C; Smith Ewan; Bushell Martin; Mallucci Giovanna R; Zents Karlijn A M; Mallucci Giovanna R; Ortori Catharine A; Barrett David A</div><div class="citationInfo"><span class="NLM_cas:title">Brain : a journal of neurology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1768-1783</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">See Mercado and Hetz (doi:10.1093/brain/awx107) for a scientific commentary on this article.Signalling through the PERK/eIF2&alpha;-P branch of the unfolded protein response plays a critical role in controlling protein synthesis rates in cells.  This pathway is overactivated in brains of patients with Alzheimer&rsquo;s disease and related disorders and has recently emerged as a promising therapeutic target for these currently untreatable conditions.  Thus, in mouse models of neurodegenerative disease, prolonged overactivation of PERK/eIF2&alpha;-P signalling causes sustained attenuation of protein synthesis, leading to memory impairment and neuronal loss.  Re-establishing translation rates by inhibition of eIF2&alpha;-P activity, genetically or pharmacologically, restores memory and prevents neurodegeneration and extends survival.  However, the experimental compounds used preclinically are unsuitable for use in humans, due to associated toxicity or poor pharmacokinetic properties.  To discover compounds that have anti-eIF2&alpha;-P activity suitable for clinical use, we performed phenotypic screens on a NINDS small molecule library of 1040 drugs.  We identified two compounds, trazodone hydrochloride and dibenzoylmethane, which reversed eIF2&alpha;-P-mediated translational attenuation in vitro and in vivo.  Both drugs were markedly neuroprotective in two mouse models of neurodegeneration, using clinically relevant doses over a prolonged period of time, without systemic toxicity.  Thus, in prion-diseased mice, both trazodone and dibenzoylmethane treatment restored memory deficits, abrogated development of neurological signs, prevented neurodegeneration and significantly prolonged survival.  In tauopathy-frontotemporal dementia mice, both drugs were neuroprotective, rescued memory deficits and reduced hippocampal atrophy.  Further, trazodone reduced p-tau burden.  These compounds therefore represent potential new disease-modifying treatments for dementia.  Trazodone in particular, a licensed drug, should now be tested in clinical trials in patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTdeaJRg2dW4UNHe_gFldTbfW6udTcc2ea4-lAVpv0_Arntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cvpvFGnsw%253D%253D&md5=fb5be53ef4ec468839c4c85278c43f43</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1093%2Fbrain%2Fawx074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbrain%252Fawx074%26sid%3Dliteratum%253Aachs%26aulast%3DHalliday%26aufirst%3DM.%26aulast%3DRadford%26aufirst%3DH.%26aulast%3DZents%26aufirst%3DK.%2BA.%2BM.%26aulast%3DMolloy%26aufirst%3DC.%26aulast%3DMoreno%26aufirst%3DJ.%2BA.%26aulast%3DVerity%26aufirst%3DN.%2BC.%26aulast%3DSmith%26aufirst%3DE.%26aulast%3DOrtori%26aufirst%3DC.%2BA.%26aulast%3DBarrett%26aufirst%3DD.%2BA.%26aulast%3DBushell%26aufirst%3DM.%26aulast%3DMallucci%26aufirst%3DG.%2BR.%26atitle%3DRepurposed%2520Drugs%2520Targeting%2520Eif2%2526Alpha%253B-P-Mediated%2520Translational%2520Repression%2520Prevent%2520Neurodegeneration%2520in%2520Mice%26jtitle%3DBrain%26date%3D2017%26volume%3D140%26spage%3D1768%26epage%3D1783%26doi%3D10.1093%2Fbrain%2Fawx074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luesch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supek, F.</span></span> <span> </span><span class="NLM_article-title">A Genome-Wide Overexpression Screen in Yeast for Small-Molecule Target Identification</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2004.10.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.chembiol.2004.10.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=15664515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmtlKmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2005&pages=55-63&author=H.+Lueschauthor=T.+Y.+Wuauthor=P.+Renauthor=N.+S.+Grayauthor=P.+G.+Schultzauthor=F.+Supek&title=A+Genome-Wide+Overexpression+Screen+in+Yeast+for+Small-Molecule+Target+Identification&doi=10.1016%2Fj.chembiol.2004.10.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A Genome-Wide Overexpression Screen in Yeast for Small-Molecule Target Identification</span></div><div class="casAuthors">Luesch, Hendrik; Wu, Tom Y. H.; Ren, Pingda; Gray, Nathanael S.; Schultz, Peter G.; Supek, Frantisek</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-63</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We describe a multicopy gene suppression screen of drug sensitivity in Saccharomyces cerevisiae that facilitates the identification of cellular targets of small mols.  An array of yeast transformants harboring a multicopy yeast genomic library was screened for resistance to growth inhibitors.  Comparison of array growth patterns for several such inhibitors allowed the differentiation of general and mol.-specific genetic suppressors.  Specific resistance to phenylaminopyrimidine (1), an inhibitor identified from a kinase-directed library, was assocd. with the overexpression of Pkc1 and a subset of downstream kinases.  Components of two other pathways (pheromone response/filamentous growth and Pho85 kinase) that genetically interact with the PKC1 MAPK signaling cascade were also identified.  Consistent with the suppression screen, inhibitor 1 bound to Pkc1 in yeast cell lysate and inhibited its activity in vitro.  These results demonstrate the utility of this approach for the rapid deconvolution of small-mol. targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCcPHiH0TixrVg90H21EOLACvtfcHk0ljJOkUiP2UYlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmtlKmsg%253D%253D&md5=a2cd6a2fc60ea8c785f1d375dd6d8178</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2004.10.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2004.10.015%26sid%3Dliteratum%253Aachs%26aulast%3DLuesch%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DT.%2BY.%26aulast%3DRen%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26aulast%3DSupek%26aufirst%3DF.%26atitle%3DA%2520Genome-Wide%2520Overexpression%2520Screen%2520in%2520Yeast%2520for%2520Small-Molecule%2520Target%2520Identification%26jtitle%3DChem.%2520Biol.%26date%3D2005%26volume%3D12%26spage%3D55%26epage%3D63%26doi%3D10.1016%2Fj.chembiol.2004.10.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Underwood, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwald, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaye, D. D.</span></span> <span> </span><span class="NLM_article-title">Ortholist 2: A New Comparative Genomic Analysis of Human and Caenorhabditis Elegans Genes</span>. <i>Genetics</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>210</i></span>,  <span class="NLM_fpage">445</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1534/genetics.118.301307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1534%2Fgenetics.118.301307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30120140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjsVyit7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=210&publication_year=2018&pages=445-461&author=W.+Kimauthor=R.+S.+Underwoodauthor=I.+Greenwaldauthor=D.+D.+Shaye&title=Ortholist+2%3A+A+New+Comparative+Genomic+Analysis+of+Human+and+Caenorhabditis+Elegans+Genes&doi=10.1534%2Fgenetics.118.301307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">OrthoList 2: a new comparative genomic analysis of human and Caenorhabditis elegans genes</span></div><div class="casAuthors">Kim, Woojin; Underwood, Ryan S.; Greenwald, Iva; Shaye, Daniel D.</div><div class="citationInfo"><span class="NLM_cas:title">Genetics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">210</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">445-461</span>CODEN:
                <span class="NLM_cas:coden">GENTAE</span>;
        ISSN:<span class="NLM_cas:issn">0016-6731</span>.
    
            (<span class="NLM_cas:orgname">Genetics Society of America</span>)
        </div><div class="casAbstract">OrthoList, a compendium of Caenorhabditis elegans genes with human orthologs compiled in 2011 by a meta-anal. of four orthol.-prediction methods, has been a popular tool for identifying conserved genes for research into biol. and disease mechanisms.  However, the efficacy of orthol. prediction depends on the accuracy of gene-model predictions, an ongoing process, and orthol.-prediction algorithms have also been updated over time.  Here we present OrthoList 2 (OL2), a new comparative genomic anal. between C. elegans and humans, and the first assessment of how changes over time affect the landscape of predicted orthologs between two species.  Although we find that updates to the orthol.-prediction methods significantly changed the landscape of C. elegans-human orthologs predicted by individual programs and-unexpectedly-reduced agreement among them, we also show that our meta-anal. approach "buffered" against changes in gene content.  We show that adding results from more programs did not lead to many addns. to the list and discuss reasons to avoid assigning "scores" based on support by individual orthol.-prediction programs; the treatment of "legacy" genes no longer predicted by these programs; and the practical difficulties of updating due to encountering deprecated, changed, or retired gene identifiers.  In addn., we consider what other criteria may support claims of orthol. and alternative approaches to find potential orthologs that elude identification by these programs.  Finally, we created a new web-based tool that allows for rapid searches of OL2 by gene identifiers, protein domains [InterPro and SMART (Simple Modular Architecture Research Tool], or human disease assocns. ([OMIM (Online Mendelian Inheritence in Man], and also includes available RNA-interference resources to facilitate potential translational cross-species studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt-i-dD0MQ0bVg90H21EOLACvtfcHk0ljJOkUiP2UYlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjsVyit7c%253D&md5=17f047bb4d7d01c737c692041715d261</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1534%2Fgenetics.118.301307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1534%252Fgenetics.118.301307%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DW.%26aulast%3DUnderwood%26aufirst%3DR.%2BS.%26aulast%3DGreenwald%26aufirst%3DI.%26aulast%3DShaye%26aufirst%3DD.%2BD.%26atitle%3DOrtholist%25202%253A%2520A%2520New%2520Comparative%2520Genomic%2520Analysis%2520of%2520Human%2520and%2520Caenorhabditis%2520Elegans%2520Genes%26jtitle%3DGenetics%26date%3D2018%26volume%3D210%26spage%3D445%26epage%3D461%26doi%3D10.1534%2Fgenetics.118.301307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hales, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korey, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larracuente, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, D. M.</span></span> <span> </span><span class="NLM_article-title">Genetics on the Fly: A Primer on the Drosophila Model System</span>. <i>Genetics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>201</i></span>,  <span class="NLM_fpage">815</span>– <span class="NLM_lpage">842</span>, <span class="refDoi"> DOI: 10.1534/genetics.115.183392</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1534%2Fgenetics.115.183392" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26564900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvFGqt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=201&publication_year=2015&pages=815-842&author=K.+G.+Halesauthor=C.+A.+Koreyauthor=A.+M.+Larracuenteauthor=D.+M.+Roberts&title=Genetics+on+the+Fly%3A+A+Primer+on+the+Drosophila+Model+System&doi=10.1534%2Fgenetics.115.183392"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Genetics on the fly: a primer on the Drosophila model system</span></div><div class="casAuthors">Hales, Karen G.; Korey, Christopher A.; Larracuente, Amanda M.; Roberts, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Genetics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">201</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">815-842</span>CODEN:
                <span class="NLM_cas:coden">GENTAE</span>;
        ISSN:<span class="NLM_cas:issn">0016-6731</span>.
    
            (<span class="NLM_cas:orgname">Genetics Society of America</span>)
        </div><div class="casAbstract">Fruit flies of the genus Drosophila have been an attractive and effective genetic model organism since Thomas Hunt Morgan and colleagues made seminal discoveries with them a century ago.  Work with Drosophila has enabled dramatic advances in cell and developmental biol., neurobiol. and behavior, mol. biol., evolutionary and population genetics, and other fields.  With more tissue types and observable behaviors than in other short-generation model organisms, and with vast genome data available for many species within the genus, the fly's tractable complexity will continue to enable exciting opportunities to explore mechanisms of complex developmental programs, behaviors, and broader evolutionary questions.  This primer describes the organism's natural history, the features of sequenced genomes within the genus, the wide range of available genetic tools and online resources, the types of biol. questions Drosophila can help address, and historical milestones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmpAqURQTda7Vg90H21EOLACvtfcHk0ljJOkUiP2UYlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvFGqt7s%253D&md5=30f5b0caf36e6649d7338882143777ab</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1534%2Fgenetics.115.183392&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1534%252Fgenetics.115.183392%26sid%3Dliteratum%253Aachs%26aulast%3DHales%26aufirst%3DK.%2BG.%26aulast%3DKorey%26aufirst%3DC.%2BA.%26aulast%3DLarracuente%26aufirst%3DA.%2BM.%26aulast%3DRoberts%26aufirst%3DD.%2BM.%26atitle%3DGenetics%2520on%2520the%2520Fly%253A%2520A%2520Primer%2520on%2520the%2520Drosophila%2520Model%2520System%26jtitle%3DGenetics%26date%3D2015%26volume%3D201%26spage%3D815%26epage%3D842%26doi%3D10.1534%2Fgenetics.115.183392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shih, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodge, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrade-Navarro, M. A.</span></span> <span> </span><span class="NLM_article-title">Comparison of Inter- and Intraspecies Variation in Humans and Fruit Flies</span>. <i>Genomics Data</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">54</span>, <span class="refDoi"> DOI: 10.1016/j.gdata.2014.11.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.gdata.2014.11.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26484147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC28zjslSqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2015&pages=49-54&author=J.+Shihauthor=R.+Hodgeauthor=M.+A.+Andrade-Navarro&title=Comparison+of+Inter-+and+Intraspecies+Variation+in+Humans+and+Fruit+Flies&doi=10.1016%2Fj.gdata.2014.11.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of inter- and intraspecies variation in humans and fruit flies</span></div><div class="casAuthors">Shih Juliann; Hodge Russ; Andrade-Navarro Miguel A</div><div class="citationInfo"><span class="NLM_cas:title">Genomics data</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">49-54</span>
        ISSN:<span class="NLM_cas:issn">2213-5960</span>.
    </div><div class="casAbstract">Variation is essential to species survival and adaptation during evolution.  This variation is conferred by the imperfection of biochemical processes, such as mutations and alterations in DNA sequences, and can also be seen within genomes through processes such as the generation of antibodies.  Recent sequencing projects have produced multiple versions of the genomes of humans and fruit flies (Drosophila melanogaster).  These give us a chance to study how individual gene sequences vary within and between species.  Here we arranged human and fly genes in orthologous pairs and compared such within-species variability with their degree of conservation between flies and humans.  We observed that a significant number of proteins associated with mRNA translation are highly conserved between species and yet are highly variable within each species.  The fact that we observe this in two species whose lineages separated more than 700 million years ago suggests that this is the result of a very ancient process.  We hypothesize that this effect might be attributed to a positive selection for variability of virus-interacting proteins that confers a general resistance to viral hijacking of the mRNA translation machinery within populations.  Our analysis points to this and to other processes resulting in positive selection for gene variation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQENEMGYK1lr7YtC3U5jeOifW6udTcc2eY9mrMpR5Ce-7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28zjslSqsg%253D%253D&md5=da95eeefb36ab976b285c1748967044d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.gdata.2014.11.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gdata.2014.11.010%26sid%3Dliteratum%253Aachs%26aulast%3DShih%26aufirst%3DJ.%26aulast%3DHodge%26aufirst%3DR.%26aulast%3DAndrade-Navarro%26aufirst%3DM.%2BA.%26atitle%3DComparison%2520of%2520Inter-%2520and%2520Intraspecies%2520Variation%2520in%2520Humans%2520and%2520Fruit%2520Flies%26jtitle%3DGenomics%2520Data%26date%3D2015%26volume%3D3%26spage%3D49%26epage%3D54%26doi%3D10.1016%2Fj.gdata.2014.11.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Howe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torroja, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrance, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berthelot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muffato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphray, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaren, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaren, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sealy, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caccamo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauch, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chow, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quintais, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerra-Assuncao, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eyre, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redmond, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laird, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenyon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donaldson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehra, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almeida-King, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loveland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trevanion, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quail, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLay, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plumb, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riddle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Threadgold, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harden, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ware, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Begum, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortimore, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerry, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heath, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillimore, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tracey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corby, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glithero, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howden, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panagiotidis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beasley, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auger, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raisen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambridge, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leongamornlert, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGuire, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilderthorp, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manthravadi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nichol, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitehead, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kay, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murnane, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphries, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sycamore, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunders, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babbage, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammond, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashreghi-Mohammadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barr, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wray, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellington, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellwood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodmansey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tromans, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grafham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skuce, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandian, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimberley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fosker, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehring, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dooley, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ersan-Urun, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eser, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geiger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geisler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karotki, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konantz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konantz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oberlander, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph-Geiger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teucke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raddatz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osoegawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rapp, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widaa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langford, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuster, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Searle, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enright, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geisler, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plasterk, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westerfield, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jong, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zon, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Postlethwait, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nusslein-Volhard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crollius, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stemple, D. L.</span></span> <span> </span><span class="NLM_article-title">The Zebrafish Reference Genome Sequence and Its Relationship to the Human Genome</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>496</i></span>,  <span class="NLM_fpage">498</span>– <span class="NLM_lpage">503</span>, <span class="refDoi"> DOI: 10.1038/nature12111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnature12111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23594743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvF2isb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=496&publication_year=2013&pages=498-503&author=K.+Howeauthor=M.+D.+Clarkauthor=C.+F.+Torrojaauthor=J.+Torranceauthor=C.+Berthelotauthor=M.+Muffatoauthor=J.+E.+Collinsauthor=S.+Humphrayauthor=K.+McLarenauthor=L.+Matthewsauthor=S.+McLarenauthor=I.+Sealyauthor=M.+Caccamoauthor=C.+Churcherauthor=C.+Scottauthor=J.+C.+Barrettauthor=R.+Kochauthor=G.+J.+Rauchauthor=S.+Whiteauthor=W.+Chowauthor=B.+Kilianauthor=L.+T.+Quintaisauthor=J.+A.+Guerra-Assuncaoauthor=Y.+Zhouauthor=Y.+Guauthor=J.+Yenauthor=J.+H.+Vogelauthor=T.+Eyreauthor=S.+Redmondauthor=R.+Banerjeeauthor=J.+Chiauthor=B.+Fuauthor=E.+Langleyauthor=S.+F.+Maguireauthor=G.+K.+Lairdauthor=D.+Lloydauthor=E.+Kenyonauthor=S.+Donaldsonauthor=H.+Sehraauthor=J.+Almeida-Kingauthor=J.+Lovelandauthor=S.+Trevanionauthor=M.+Jonesauthor=M.+Quailauthor=D.+Willeyauthor=A.+Huntauthor=J.+Burtonauthor=S.+Simsauthor=K.+McLayauthor=B.+Plumbauthor=J.+Davisauthor=C.+Cleeauthor=K.+Oliverauthor=R.+Clarkauthor=C.+Riddleauthor=D.+Elliottauthor=G.+Threadgoldauthor=G.+Hardenauthor=D.+Wareauthor=S.+Begumauthor=B.+Mortimoreauthor=G.+Kerryauthor=P.+Heathauthor=B.+Phillimoreauthor=A.+Traceyauthor=N.+Corbyauthor=M.+Dunnauthor=C.+Johnsonauthor=J.+Woodauthor=S.+Clarkauthor=S.+Pelanauthor=G.+Griffithsauthor=M.+Smithauthor=R.+Glitheroauthor=P.+Howdenauthor=N.+Barkerauthor=C.+Lloydauthor=C.+Stevensauthor=J.+Harleyauthor=K.+Holtauthor=G.+Panagiotidisauthor=J.+Lovellauthor=H.+Beasleyauthor=C.+Hendersonauthor=D.+Gordonauthor=K.+Augerauthor=D.+Wrightauthor=J.+Collinsauthor=C.+Raisenauthor=L.+Dyerauthor=K.+Leungauthor=L.+Robertsonauthor=K.+Ambridgeauthor=D.+Leongamornlertauthor=S.+McGuireauthor=R.+Gilderthorpauthor=C.+Griffithsauthor=D.+Manthravadiauthor=S.+Nicholauthor=G.+Barkerauthor=S.+Whiteheadauthor=M.+Kayauthor=J.+Brownauthor=C.+Murnaneauthor=E.+Grayauthor=M.+Humphriesauthor=N.+Sycamoreauthor=D.+Barkerauthor=D.+Saundersauthor=J.+Wallisauthor=A.+Babbageauthor=S.+Hammondauthor=M.+Mashreghi-Mohammadiauthor=L.+Barrauthor=S.+Martinauthor=P.+Wrayauthor=A.+Ellingtonauthor=N.+Matthewsauthor=M.+Ellwoodauthor=R.+Woodmanseyauthor=G.+Clarkauthor=J.+Cooperauthor=A.+Tromansauthor=D.+Grafhamauthor=C.+Skuceauthor=R.+Pandianauthor=R.+Andrewsauthor=E.+Harrisonauthor=A.+Kimberleyauthor=J.+Garnettauthor=N.+Foskerauthor=R.+Hallauthor=P.+Garnerauthor=D.+Kellyauthor=C.+Birdauthor=S.+Palmerauthor=I.+Gehringauthor=A.+Bergerauthor=C.+M.+Dooleyauthor=Z.+Ersan-Urunauthor=C.+Eserauthor=H.+Geigerauthor=M.+Geislerauthor=L.+Karotkiauthor=A.+Kirnauthor=J.+Konantzauthor=M.+Konantzauthor=M.+Oberlanderauthor=S.+Rudolph-Geigerauthor=M.+Teuckeauthor=C.+Lanzauthor=G.+Raddatzauthor=K.+Osoegawaauthor=B.+Zhuauthor=A.+Rappauthor=S.+Widaaauthor=C.+Langfordauthor=F.+Yangauthor=S.+C.+Schusterauthor=N.+P.+Carterauthor=J.+Harrowauthor=Z.+Ningauthor=J.+Herreroauthor=S.+M.+Searleauthor=A.+Enrightauthor=R.+Geislerauthor=R.+H.+Plasterkauthor=C.+Leeauthor=M.+Westerfieldauthor=P.+J.+de+Jongauthor=L.+I.+Zonauthor=J.+H.+Postlethwaitauthor=C.+Nusslein-Volhardauthor=T.+J.+Hubbardauthor=H.+R.+Crolliusauthor=J.+Rogersauthor=D.+L.+Stemple&title=The+Zebrafish+Reference+Genome+Sequence+and+Its+Relationship+to+the+Human+Genome&doi=10.1038%2Fnature12111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The zebrafish reference genome sequence and its relationship to the human genome</span></div><div class="casAuthors">Howe, Kerstin; Clark, Matthew D.; Torroja, Carlos F.; Torrance, James; Berthelot, Camille; Muffato, Matthieu; Collins, John E.; Humphray, Sean; McLaren, Karen; Matthews, Lucy; McLaren, Stuart; Sealy, Ian; Caccamo, Mario; Churcher, Carol; Scott, Carol; Barrett, Jeffrey C.; Koch, Romke; Rauch, Gerd-Joerg; White, Simon; Chow, William; Kilian, Britt; Quintais, Leonor T.; Guerra-Assuncao, Jose A.; Zhou, Yi; Gu, Yong; Yen, Jennifer; Vogel, Jan-Hinnerk; Eyre, Tina; Redmond, Seth; Banerjee, Ruby; Chi, Jianxiang; Fu, Beiyuan; Langley, Elizabeth; Maguire, Sean F.; Laird, Gavin K.; Lloyd, David; Kenyon, Emma; Donaldson, Sarah; Sehra, Harminder; Almeida-King, Jeff; Loveland, Jane; Trevanion, Stephen; Jones, Matt; Quail, Mike; Willey, Dave; Hunt, Adrienne; Burton, John; Sims, Sarah; McLay, Kirsten; Plumb, Bob; Davis, Joy; Clee, Chris; Oliver, Karen; Clark, Richard; Riddle, Clare; Eliott, David; Threadgold, Glen; Harden, Glenn; Ware, Darren; Begum, Sharmin; Mortimore, Beverley; Kerry, Giselle; Heath, Paul; Phillimore, Benjamin; Tracey, Alan; Corby, Nicole; Dunn, Matthew; Johnson, Christopher; Wood, Jonathan; Clark, Susan; Pelan, Sarah; Griffiths, Guy; Smith, Michelle; Glithero, Rebecca; Howden, Philip; Barker, Nicholas; Lloyd, Christine; Stevens, Christopher; Harley, Joanna; Holt, Karen; Panagiotidis, Georgios; Lovell, Jamieson; Beasley, Helen; Henderson, Carl; Gordon, Daria; Auger, Katherine; Wright, Deborah; Collins, Joanna; Raisen, Claire; Dyer, Lauren; Leung, Kenric; Robertson, Lauren; Ambridge, Kirsty; Leongamornlert, Daniel; McGuire, Sarah; Gilderthorp, Ruth; Griffiths, Coline; Manthravadi, Deepa; Nichol, Sarah; Barker, Gary; Whitehead, Siobhan; Kay, Michael; Brown, Jacqueline; Murnane, Clare; Gray, Emma; Humphries, Matthew; Sycamore, Neil; Barker, Darren; Saunders, David; Wallis, Justene; Babbage, Anne; Hammond, Sian; Mashreghi-Mohammadi, Maryam; Barr, Lucy; Martin, Sancha; Wray, Paul; Ellington, Andrew; Matthews, Nicholas; Ellwood, Matthew; Woodmansey, Rebecca; Clark, Graham; Cooper, James D.; Tromans, Anthony; Grafham, Darren; Skuce, Carl; Pandian, Richard; Andrews, Robert; Harrison, Elliot; Kimberley, Andrew; Garnett, Jane; Fosker, Nigel; Hall, Rebekah; Garner, Patrick; Kelly, Daniel; Bird, Christine; Palmer, Sophie; Gehring, Ines; Berger, Andrea; Dooley, Christopher M.; Ersan-Ueruen, Zuebeyde; Eser, Cigdem; Geiger, Horst; Geisler, Maria; Karotki, Lena; Kirn, Anette; Konantz, Judith; Konantz, Martina; Oberlaender, Martina; Rudolph-Geiger, Silke; Teucke, Mathias; Lanz, Christa; Raddatz, Guenter; Osoegawa, Kazutoyo; Zhu, Baoli; Rapp, Amanda; Widaa, Sara; Langford, Cordelia; Yang, Fengtang; Schuster, Stephan C.; Carter, Nigel P.; Harrow, Jennifer; Ning, Zemin; Herrero, Javier; Searle, Steve M. J.; Enright, Anton; Geisler, Robert; Plasterk, Ronald H. A.; Lee, Charles; Westerfield, Monte; de Jong, Pieter J.; Zon, Leonard I.; Postlethwait, John H.; Nuesslein-Volhard, Christiane; Hubbard, Tim J. P.; Crollius, Hugues Roest; Rogers, Jane; Stemple, Derek L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">496</span>
        (<span class="NLM_cas:issue">7446</span>),
    <span class="NLM_cas:pages">498-503</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Zebrafish have become a popular organism for the study of vertebrate gene function.  The virtually transparent embryos of this species, and the ability to accelerate genetic studies by gene knockdown or overexpression, have led to the widespread use of zebrafish in the detailed investigation of vertebrate gene function and increasingly, the study of human genetic disease.  However, for effective modeling of human genetic disease it is important to understand the extent to which zebrafish genes and gene structures are related to orthologous human genes.  To examine this, the authors generated a high-quality sequence assembly of the zebrafish genome, made up of an overlapping set of completely sequenced large-insert clones that were ordered and oriented using a high-resoln. high-d. meiotic map.  Detailed automatic and manual annotation provides evidence of >26,000 protein-coding genes, the largest gene set of any vertebrate so far sequenced.  Comparison to the human ref. genome shows that ∼70% of human genes have at least one obvious zebrafish ortholog.  In addn., the high quality of this genome assembly provides a clearer understanding of key genomic features such as a unique repeat content, a scarcity of pseudogenes, an enrichment of zebrafish-specific genes on chromosome 4 and chromosomal regions that influence sex detn.  The draft, annotated, genome sequence is deposited in GenBank/EMBL/DDBJ with WGS project accession nos. CABZ01000001-CABZ01119119, CU651604-CU651628, and FR648399-FR652957.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY5Bb_pwj9k7Vg90H21EOLACvtfcHk0lgulf6kaeup-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvF2isb0%253D&md5=9c7c77d0e97a41c5f64802b892b2701d</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnature12111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12111%26sid%3Dliteratum%253Aachs%26aulast%3DHowe%26aufirst%3DK.%26aulast%3DClark%26aufirst%3DM.%2BD.%26aulast%3DTorroja%26aufirst%3DC.%2BF.%26aulast%3DTorrance%26aufirst%3DJ.%26aulast%3DBerthelot%26aufirst%3DC.%26aulast%3DMuffato%26aufirst%3DM.%26aulast%3DCollins%26aufirst%3DJ.%2BE.%26aulast%3DHumphray%26aufirst%3DS.%26aulast%3DMcLaren%26aufirst%3DK.%26aulast%3DMatthews%26aufirst%3DL.%26aulast%3DMcLaren%26aufirst%3DS.%26aulast%3DSealy%26aufirst%3DI.%26aulast%3DCaccamo%26aufirst%3DM.%26aulast%3DChurcher%26aufirst%3DC.%26aulast%3DScott%26aufirst%3DC.%26aulast%3DBarrett%26aufirst%3DJ.%2BC.%26aulast%3DKoch%26aufirst%3DR.%26aulast%3DRauch%26aufirst%3DG.%2BJ.%26aulast%3DWhite%26aufirst%3DS.%26aulast%3DChow%26aufirst%3DW.%26aulast%3DKilian%26aufirst%3DB.%26aulast%3DQuintais%26aufirst%3DL.%2BT.%26aulast%3DGuerra-Assuncao%26aufirst%3DJ.%2BA.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DGu%26aufirst%3DY.%26aulast%3DYen%26aufirst%3DJ.%26aulast%3DVogel%26aufirst%3DJ.%2BH.%26aulast%3DEyre%26aufirst%3DT.%26aulast%3DRedmond%26aufirst%3DS.%26aulast%3DBanerjee%26aufirst%3DR.%26aulast%3DChi%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DB.%26aulast%3DLangley%26aufirst%3DE.%26aulast%3DMaguire%26aufirst%3DS.%2BF.%26aulast%3DLaird%26aufirst%3DG.%2BK.%26aulast%3DLloyd%26aufirst%3DD.%26aulast%3DKenyon%26aufirst%3DE.%26aulast%3DDonaldson%26aufirst%3DS.%26aulast%3DSehra%26aufirst%3DH.%26aulast%3DAlmeida-King%26aufirst%3DJ.%26aulast%3DLoveland%26aufirst%3DJ.%26aulast%3DTrevanion%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DM.%26aulast%3DQuail%26aufirst%3DM.%26aulast%3DWilley%26aufirst%3DD.%26aulast%3DHunt%26aufirst%3DA.%26aulast%3DBurton%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DS.%26aulast%3DMcLay%26aufirst%3DK.%26aulast%3DPlumb%26aufirst%3DB.%26aulast%3DDavis%26aufirst%3DJ.%26aulast%3DClee%26aufirst%3DC.%26aulast%3DOliver%26aufirst%3DK.%26aulast%3DClark%26aufirst%3DR.%26aulast%3DRiddle%26aufirst%3DC.%26aulast%3DElliott%26aufirst%3DD.%26aulast%3DThreadgold%26aufirst%3DG.%26aulast%3DHarden%26aufirst%3DG.%26aulast%3DWare%26aufirst%3DD.%26aulast%3DBegum%26aufirst%3DS.%26aulast%3DMortimore%26aufirst%3DB.%26aulast%3DKerry%26aufirst%3DG.%26aulast%3DHeath%26aufirst%3DP.%26aulast%3DPhillimore%26aufirst%3DB.%26aulast%3DTracey%26aufirst%3DA.%26aulast%3DCorby%26aufirst%3DN.%26aulast%3DDunn%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DC.%26aulast%3DWood%26aufirst%3DJ.%26aulast%3DClark%26aufirst%3DS.%26aulast%3DPelan%26aufirst%3DS.%26aulast%3DGriffiths%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DM.%26aulast%3DGlithero%26aufirst%3DR.%26aulast%3DHowden%26aufirst%3DP.%26aulast%3DBarker%26aufirst%3DN.%26aulast%3DLloyd%26aufirst%3DC.%26aulast%3DStevens%26aufirst%3DC.%26aulast%3DHarley%26aufirst%3DJ.%26aulast%3DHolt%26aufirst%3DK.%26aulast%3DPanagiotidis%26aufirst%3DG.%26aulast%3DLovell%26aufirst%3DJ.%26aulast%3DBeasley%26aufirst%3DH.%26aulast%3DHenderson%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DAuger%26aufirst%3DK.%26aulast%3DWright%26aufirst%3DD.%26aulast%3DCollins%26aufirst%3DJ.%26aulast%3DRaisen%26aufirst%3DC.%26aulast%3DDyer%26aufirst%3DL.%26aulast%3DLeung%26aufirst%3DK.%26aulast%3DRobertson%26aufirst%3DL.%26aulast%3DAmbridge%26aufirst%3DK.%26aulast%3DLeongamornlert%26aufirst%3DD.%26aulast%3DMcGuire%26aufirst%3DS.%26aulast%3DGilderthorp%26aufirst%3DR.%26aulast%3DGriffiths%26aufirst%3DC.%26aulast%3DManthravadi%26aufirst%3DD.%26aulast%3DNichol%26aufirst%3DS.%26aulast%3DBarker%26aufirst%3DG.%26aulast%3DWhitehead%26aufirst%3DS.%26aulast%3DKay%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DMurnane%26aufirst%3DC.%26aulast%3DGray%26aufirst%3DE.%26aulast%3DHumphries%26aufirst%3DM.%26aulast%3DSycamore%26aufirst%3DN.%26aulast%3DBarker%26aufirst%3DD.%26aulast%3DSaunders%26aufirst%3DD.%26aulast%3DWallis%26aufirst%3DJ.%26aulast%3DBabbage%26aufirst%3DA.%26aulast%3DHammond%26aufirst%3DS.%26aulast%3DMashreghi-Mohammadi%26aufirst%3DM.%26aulast%3DBarr%26aufirst%3DL.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DWray%26aufirst%3DP.%26aulast%3DEllington%26aufirst%3DA.%26aulast%3DMatthews%26aufirst%3DN.%26aulast%3DEllwood%26aufirst%3DM.%26aulast%3DWoodmansey%26aufirst%3DR.%26aulast%3DClark%26aufirst%3DG.%26aulast%3DCooper%26aufirst%3DJ.%26aulast%3DTromans%26aufirst%3DA.%26aulast%3DGrafham%26aufirst%3DD.%26aulast%3DSkuce%26aufirst%3DC.%26aulast%3DPandian%26aufirst%3DR.%26aulast%3DAndrews%26aufirst%3DR.%26aulast%3DHarrison%26aufirst%3DE.%26aulast%3DKimberley%26aufirst%3DA.%26aulast%3DGarnett%26aufirst%3DJ.%26aulast%3DFosker%26aufirst%3DN.%26aulast%3DHall%26aufirst%3DR.%26aulast%3DGarner%26aufirst%3DP.%26aulast%3DKelly%26aufirst%3DD.%26aulast%3DBird%26aufirst%3DC.%26aulast%3DPalmer%26aufirst%3DS.%26aulast%3DGehring%26aufirst%3DI.%26aulast%3DBerger%26aufirst%3DA.%26aulast%3DDooley%26aufirst%3DC.%2BM.%26aulast%3DErsan-Urun%26aufirst%3DZ.%26aulast%3DEser%26aufirst%3DC.%26aulast%3DGeiger%26aufirst%3DH.%26aulast%3DGeisler%26aufirst%3DM.%26aulast%3DKarotki%26aufirst%3DL.%26aulast%3DKirn%26aufirst%3DA.%26aulast%3DKonantz%26aufirst%3DJ.%26aulast%3DKonantz%26aufirst%3DM.%26aulast%3DOberlander%26aufirst%3DM.%26aulast%3DRudolph-Geiger%26aufirst%3DS.%26aulast%3DTeucke%26aufirst%3DM.%26aulast%3DLanz%26aufirst%3DC.%26aulast%3DRaddatz%26aufirst%3DG.%26aulast%3DOsoegawa%26aufirst%3DK.%26aulast%3DZhu%26aufirst%3DB.%26aulast%3DRapp%26aufirst%3DA.%26aulast%3DWidaa%26aufirst%3DS.%26aulast%3DLangford%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DF.%26aulast%3DSchuster%26aufirst%3DS.%2BC.%26aulast%3DCarter%26aufirst%3DN.%2BP.%26aulast%3DHarrow%26aufirst%3DJ.%26aulast%3DNing%26aufirst%3DZ.%26aulast%3DHerrero%26aufirst%3DJ.%26aulast%3DSearle%26aufirst%3DS.%2BM.%26aulast%3DEnright%26aufirst%3DA.%26aulast%3DGeisler%26aufirst%3DR.%26aulast%3DPlasterk%26aufirst%3DR.%2BH.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DWesterfield%26aufirst%3DM.%26aulast%3Dde%2BJong%26aufirst%3DP.%2BJ.%26aulast%3DZon%26aufirst%3DL.%2BI.%26aulast%3DPostlethwait%26aufirst%3DJ.%2BH.%26aulast%3DNusslein-Volhard%26aufirst%3DC.%26aulast%3DHubbard%26aufirst%3DT.%2BJ.%26aulast%3DCrollius%26aufirst%3DH.%2BR.%26aulast%3DRogers%26aufirst%3DJ.%26aulast%3DStemple%26aufirst%3DD.%2BL.%26atitle%3DThe%2520Zebrafish%2520Reference%2520Genome%2520Sequence%2520and%2520Its%2520Relationship%2520to%2520the%2520Human%2520Genome%26jtitle%3DNature%26date%3D2013%26volume%3D496%26spage%3D498%26epage%3D503%26doi%3D10.1038%2Fnature12111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guenet, J. L.</span></span> <span> </span><span class="NLM_article-title">The Mouse Genome</span>. <i>Genome Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1729</span>– <span class="NLM_lpage">1740</span>, <span class="refDoi"> DOI: 10.1101/gr.3728305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1101%2Fgr.3728305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=16339371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvFA%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=1729-1740&author=J.+L.+Guenet&title=The+Mouse+Genome&doi=10.1101%2Fgr.3728305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The mouse genome</span></div><div class="casAuthors">Guenet, Jean Louis</div><div class="citationInfo"><span class="NLM_cas:title">Genome Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1729-1740</span>CODEN:
                <span class="NLM_cas:coden">GEREFS</span>;
        ISSN:<span class="NLM_cas:issn">1088-9051</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  The house mouse has been used as a privileged model organism since the early days of genetics, and the numerous expts. made with this small mammal have regularly contributed to enrich our knowledge of mammalian biol. and pathol., ranging from embryonic development to metabolic disease, histocompatibility, immunol., behavior, and cancer.  Over the past two decades, a no. of large-scale integrated and concerted projects have been undertaken that will probably open a new era in the genetics of the species.  The sequencing of the genome, which will allow researchers to make comparisons with other mammals and identify regions conserved by evolution, is probably the most important project, but many other initiatives, such as the massive prodn. of point or chromosomal mutations assocd. with comprehensive and standardized phenotyping of the mutant phenotypes, will help annotation of the ∼25,000 genes packed in the mouse genome.  In the same way, and as another consequence of the sequencing, the discovery of many single nucleotide polymorphisms and the development of new tools and resources, like the Collaborative Cross, will contribute to the development of modern quant. genetics.  It is clear that mouse genetics has changed dramatically over the last 10-15 years and its future looks promising.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUyX4TuG7CNLVg90H21EOLACvtfcHk0lhcDjIgmv5C6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvFA%253D&md5=12eb5fb4a2caa43bb3e08b750e4bf1f9</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1101%2Fgr.3728305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgr.3728305%26sid%3Dliteratum%253Aachs%26aulast%3DGuenet%26aufirst%3DJ.%2BL.%26atitle%3DThe%2520Mouse%2520Genome%26jtitle%3DGenome%2520Res.%26date%3D2005%26volume%3D15%26spage%3D1729%26epage%3D1740%26doi%3D10.1101%2Fgr.3728305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saleem, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannan, R. R.</span></span> <span> </span><span class="NLM_article-title">Zebrafish: An Emerging Real-Time Model System to Study Alzheimer’s Disease and Neurospecific Drug Discovery</span>. <i>Cell Death Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">45</span>, <span class="refDoi"> DOI: 10.1038/s41420-018-0109-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41420-018-0109-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30302279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BB3cznsVGqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=45&author=S.+Saleemauthor=R.+R.+Kannan&title=Zebrafish%3A+An+Emerging+Real-Time+Model+System+to+Study+Alzheimer%E2%80%99s+Disease+and+Neurospecific+Drug+Discovery&doi=10.1038%2Fs41420-018-0109-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Zebrafish: an emerging real-time model system to study Alzheimer's disease and neurospecific drug discovery</span></div><div class="casAuthors">Saleem Suraiya; Kannan Rajaretinam Rajesh</div><div class="citationInfo"><span class="NLM_cas:title">Cell death discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45</span>
        ISSN:<span class="NLM_cas:issn">2058-7716</span>.
    </div><div class="casAbstract">Zebrafish (Danio rerio) is emerging as an increasingly successful model for translational research on human neurological disorders.  In this review, we appraise the high degree of neurological and behavioural resemblance of zebrafish with humans.  It is highly validated as a powerful vertebrate model for investigating human neurodegenerative diseases.  The neuroanatomic and neurochemical pathways of zebrafish brain exhibit a profound resemblance with the human brain.  Physiological, emotional and social behavioural pattern similarities between them have also been well established.  Interestingly, zebrafish models have been used successfully to simulate the pathology of Alzheimer's disease (AD) as well as Tauopathy.  Their relatively simple nervous system and the optical transparency of the embryos permit real-time neurological imaging.  Here, we further elaborate on the use of recent real-time imaging techniques to obtain vital insights into the neurodegeneration that occurs in AD.  Zebrafish is adeptly suitable for Ca(2+) imaging, which provides a better understanding of neuronal activity and axonal dystrophy in a non-invasive manner.  Three-dimensional imaging in zebrafish is a rapidly evolving technique, which allows the visualisation of the whole organism for an elaborate in vivo functional and neurophysiological analysis in disease condition.  Suitability to high-throughput screening and similarity with humans makes zebrafish an excellent model for screening neurospecific compounds.  Thus, the zebrafish model can be pivotal in bridging the gap from the bench to the bedside.  This fish is becoming an increasingly successful model to understand AD with further scope for investigation in neurodevelopment and neurodegeneration, which promises exciting research opportunities in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhy659z6uLnbmQt-ICR3qNfW6udTcc2eZF5p5k9ugI_bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cznsVGqsQ%253D%253D&md5=a789b9a83b2db7e36fd21581b5f4de1c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fs41420-018-0109-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41420-018-0109-7%26sid%3Dliteratum%253Aachs%26aulast%3DSaleem%26aufirst%3DS.%26aulast%3DKannan%26aufirst%3DR.%2BR.%26atitle%3DZebrafish%253A%2520An%2520Emerging%2520Real-Time%2520Model%2520System%2520to%2520Study%2520Alzheimer%25E2%2580%2599s%2520Disease%2520and%2520Neurospecific%2520Drug%2520Discovery%26jtitle%3DCell%2520Death%2520Discovery%26date%3D2018%26volume%3D4%26spage%3D45%26doi%3D10.1038%2Fs41420-018-0109-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Corman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggblad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brautigam, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafarga, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lidemalm, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huhn, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carreras-Puigvert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Capetillo, O.</span></span> <span> </span><span class="NLM_article-title">A Chemical Screen Identifies Compounds Limiting the Toxicity of C9orf72 Dipeptide Repeats</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">243</span>, e235 <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.10.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.chembiol.2018.10.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30527999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVChs77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=235-243&author=A.+Cormanauthor=B.+Jungauthor=M.+Haggbladauthor=L.+Brautigamauthor=V.+Lafargaauthor=L.+Lidemalmauthor=D.+Huhnauthor=J.+Carreras-Puigvertauthor=O.+Fernandez-Capetillo&title=A+Chemical+Screen+Identifies+Compounds+Limiting+the+Toxicity+of+C9orf72+Dipeptide+Repeats&doi=10.1016%2Fj.chembiol.2018.10.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemical Screen Identifies Compounds Limiting the Toxicity of C9ORF72 Dipeptide Repeats</span></div><div class="casAuthors">Corman, Alba; Jung, Bomi; Haeggblad, Maria; Braeutigam, Lars; Lafarga, Vanesa; Lidemalm, Louise; Huhn, Daniela; Carreras-Puigvert, Jordi; Fernandez-Capetillo, Oscar</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">235-243.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The expansion of GGGGCC repeats within the first intron of C9ORF72 constitutes the most common cause of amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD).  Through repeat-assocd. non-ATG translation, these expansions are translated into dipeptide repeats (DPRs), some of which accumulate at nucleoli and lead to cell death.  We here performed a chem. screen to identify compds. reducing the toxicity of ALS-related poly(PR) peptides.  Our screening identified sodium phenylbutyrate, currently in clin. trials, and BET Bromodomain inhibitors as modifiers of poly(PR) toxicity in cell lines and developing zebrafish embryos.  Mechanistically, we show that BET Bromodomain inhibitors rescue the nucleolar stress induced by poly(PR) or actinomycin D, alleviating the effects of the DPR in nucleolus-related functions such as mRNA splicing or translation.  Our work suggests that BET Bromodomain inhibitors might have beneficial effects in diseases linked to nucleolar stress such as ALS/FTD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpymE6MfWVDlrVg90H21EOLACvtfcHk0lhcDjIgmv5C6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVChs77K&md5=02b65f132ba1aee5ca18aa6140ec9989</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.10.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.10.020%26sid%3Dliteratum%253Aachs%26aulast%3DCorman%26aufirst%3DA.%26aulast%3DJung%26aufirst%3DB.%26aulast%3DHaggblad%26aufirst%3DM.%26aulast%3DBrautigam%26aufirst%3DL.%26aulast%3DLafarga%26aufirst%3DV.%26aulast%3DLidemalm%26aufirst%3DL.%26aulast%3DHuhn%26aufirst%3DD.%26aulast%3DCarreras-Puigvert%26aufirst%3DJ.%26aulast%3DFernandez-Capetillo%26aufirst%3DO.%26atitle%3DA%2520Chemical%2520Screen%2520Identifies%2520Compounds%2520Limiting%2520the%2520Toxicity%2520of%2520C9orf72%2520Dipeptide%2520Repeats%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D235%26epage%3D243%26doi%3D10.1016%2Fj.chembiol.2018.10.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGurk, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomes, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mojsilovic-Petrovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalb, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shorter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonini, N. M.</span></span> <span> </span><span class="NLM_article-title">Poly(Adp-Ribose) Prevents Pathological Phase Separation of Tdp-43 by Promoting Liquid Demixing and Stress Granule Localization</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">703</span>– <span class="NLM_lpage">717</span>, e709 <span class="refDoi"> DOI: 10.1016/j.molcel.2018.07.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.molcel.2018.07.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30100264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVOksr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2018&pages=703-717&author=L.+McGurkauthor=E.+Gomesauthor=L.+Guoauthor=J.+Mojsilovic-Petrovicauthor=V.+Tranauthor=R.+G.+Kalbauthor=J.+Shorterauthor=N.+M.+Bonini&title=Poly%28Adp-Ribose%29+Prevents+Pathological+Phase+Separation+of+Tdp-43+by+Promoting+Liquid+Demixing+and+Stress+Granule+Localization&doi=10.1016%2Fj.molcel.2018.07.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Poly(ADP-Ribose) Prevents Pathological Phase Separation of TDP-43 by Promoting Liquid Demixing and Stress Granule Localization</span></div><div class="casAuthors">McGurk, Leeanne; Gomes, Edward; Guo, Lin; Mojsilovic-Petrovic, Jelena; Tran, Van; Kalb, Robert G.; Shorter, James; Bonini, Nancy M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">703-717.e9</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In amyotrophic lateral sclerosis (ALS) and frontotemporal degeneration (FTD), cytoplasmic aggregates of hyperphosphorylated TDP-43 accumulate and colocalize with some stress granule components, but how pathol. TDP-43 aggregation is nucleated remains unknown.  In Drosophila, we establish that downregulation of tankyrase, a poly(ADP-ribose) (PAR) polymerase, reduces TDP-43 accumulation in the cytoplasm and potently mitigates neurodegeneration.  We establish that TDP-43 non-covalently binds to PAR via PAR-binding motifs embedded within its nuclear localization sequence.  PAR binding promotes liq.-liq. phase sepn. of TDP-43 in vitro and is required for TDP-43 accumulation in stress granules in mammalian cells and neurons.  Stress granule localization initially protects TDP-43 from disease-assocd. phosphorylation, but upon long-term stress, stress granules resolve, leaving behind aggregates of phosphorylated TDP-43.  Finally, small-mol. inhibition of Tankyrase-1/2 in mammalian cells inhibits formation of cytoplasmic TDP-43 foci without affecting stress granule assembly.  Thus, Tankyrase inhibition antagonizes TDP-43-assocd. pathol. and neurodegeneration and could have therapeutic utility for ALS and FTD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZPl6-6iH2o7Vg90H21EOLACvtfcHk0lhOA1uc9JTaXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVOksr3O&md5=d3b22a855fac3079e9dcbe01e40546a7</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2018.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2018.07.002%26sid%3Dliteratum%253Aachs%26aulast%3DMcGurk%26aufirst%3DL.%26aulast%3DGomes%26aufirst%3DE.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DMojsilovic-Petrovic%26aufirst%3DJ.%26aulast%3DTran%26aufirst%3DV.%26aulast%3DKalb%26aufirst%3DR.%2BG.%26aulast%3DShorter%26aufirst%3DJ.%26aulast%3DBonini%26aufirst%3DN.%2BM.%26atitle%3DPoly%2528Adp-Ribose%2529%2520Prevents%2520Pathological%2520Phase%2520Separation%2520of%2520Tdp-43%2520by%2520Promoting%2520Liquid%2520Demixing%2520and%2520Stress%2520Granule%2520Localization%26jtitle%3DMol.%2520Cell%26date%3D2018%26volume%3D71%26spage%3D703%26epage%3D717%26doi%3D10.1016%2Fj.molcel.2018.07.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karsten, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voigt, A.</span></span> <span> </span><span class="NLM_article-title">Drosophila as a Screening Tool to Study Human Neurodegenerative Diseases</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">460</span>, <span class="refDoi"> DOI: 10.1111/jnc.12446</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1111%2Fjnc.12446" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=24028575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWqs7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2013&pages=453-460&author=S.+Lenzauthor=P.+Karstenauthor=J.+B.+Schulzauthor=A.+Voigt&title=Drosophila+as+a+Screening+Tool+to+Study+Human+Neurodegenerative+Diseases&doi=10.1111%2Fjnc.12446"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Drosophila as a screening tool to study human neurodegenerative diseases</span></div><div class="casAuthors">Lenz, Sarah; Karsten, Peter; Schulz, Joerg B.; Voigt, Aaron</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">3&4</span>),
    <span class="NLM_cas:pages">453-460</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  In an aging society, research involving neurodegenerative disorders is of paramount importance.  Over the past few years, research on Alzheimer's and Parkinson's diseases has made tremendous progress.  Exptl. studies, however, rely mostly on transgenic animal models, preferentially using mice.  Although expts. on mice have enormous advantages, they also have some inherent limitations, some of which can be overcome by the use of Drosophila melanogaster as an exptl. animal.  Among the major advantages of using the fly is its small genome, which can also be modified very easily.  The fact that its genome lends itself to diverse alterations (e. g. mutagenesis, transposons) has made the fly a useful organism to perform large-scale and genome-wide screening approaches.  This has opened up an entirely new field of exptl. research aiming to elucidate genetic interactions and screen for modifiers of disease processes in vivo.  Here, we provide a brief overview of how flies can be used to analyze mol. mechanisms underlying human neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmYVEkUe1F5rVg90H21EOLACvtfcHk0lhOA1uc9JTaXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWqs7jM&md5=0f91df624a0557edce36710db73d1e52</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1111%2Fjnc.12446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjnc.12446%26sid%3Dliteratum%253Aachs%26aulast%3DLenz%26aufirst%3DS.%26aulast%3DKarsten%26aufirst%3DP.%26aulast%3DSchulz%26aufirst%3DJ.%2BB.%26aulast%3DVoigt%26aufirst%3DA.%26atitle%3DDrosophila%2520as%2520a%2520Screening%2520Tool%2520to%2520Study%2520Human%2520Neurodegenerative%2520Diseases%26jtitle%3DJ.%2520Neurochem.%26date%3D2013%26volume%3D127%26spage%3D453%26epage%3D460%26doi%3D10.1111%2Fjnc.12446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lawal, H. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Djapri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biedermann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawless, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maidment, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krantz, D. E.</span></span> <span> </span><span class="NLM_article-title">Drosophila Modifier Screens to Identify Novel Neuropsychiatric Drugs Including Aminergic Agents for the Possible Treatment of Parkinson’s Disease and Depression</span>. <i>Mol. Psychiatry</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1038/mp.2012.170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fmp.2012.170" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23229049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVClsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=235-242&author=H.+O.+Lawalauthor=A.+Terrellauthor=H.+A.+Lamauthor=C.+Djapriauthor=J.+Jangauthor=R.+Hadiauthor=L.+Robertsauthor=V.+Shahiauthor=M.+T.+Chouauthor=T.+Biedermannauthor=B.+Huangauthor=G.+M.+Lawlessauthor=N.+T.+Maidmentauthor=D.+E.+Krantz&title=Drosophila+Modifier+Screens+to+Identify+Novel+Neuropsychiatric+Drugs+Including+Aminergic+Agents+for+the+Possible+Treatment+of+Parkinson%E2%80%99s+Disease+and+Depression&doi=10.1038%2Fmp.2012.170"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Drosophila modifier screens to identify novel neuropsychiatric drugs including aminergic agents for the possible treatment of Parkinson's disease and depression</span></div><div class="casAuthors">Lawal, H. O.; Terrell, A.; Lam, H. A.; Djapri, C.; Jang, J.; Hadi, R.; Roberts, L.; Shahi, V.; Chou, M.-T.; Biedermann, T.; Huang, B.; Lawless, G. M.; Maidment, N. T.; Krantz, D. E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Psychiatry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">MOPSFQ</span>;
        ISSN:<span class="NLM_cas:issn">1359-4184</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small mols. that increase the presynaptic function of aminergic cells may provide neuroprotection in Parkinson's disease (PD) as well as treatments for attention deficit hyperactivity disorder (ADHD) and depression.  Model genetic organisms such as Drosophila melanogaster may enhance the detection of new drugs via modifier or 'enhancer/suppressor' screens, but this technique has not been applied to processes relevant to psychiatry.  To identify new aminergic drugs in vivo, we used a mutation in the Drosophila vesicular monoamine transporter (dVMAT) as a sensitized genetic background and performed a suppressor screen.  We fed dVMAT mutant larvae ∼1000 known drugs and quantitated rescue (suppression) of an amine-dependent locomotor deficit in the larva.  To det. which drugs might specifically potentiate neurotransmitter release, we performed an addnl. secondary screen for drugs that require presynaptic amine storage to rescue larval locomotion.  Using addnl. larval locomotion and adult fertility assays, we validated that at least one compd. previously used clin. as an antineoplastic agent potentiates the presynaptic function of aminergic circuits.  We suggest that structurally similar agents might be used to development treatments for PD, depression and ADHD, and that modifier screens in Drosophila provide a new strategy to screen for neuropsychiatric drugs.  More generally, our findings demonstrate the power of physiol. based screens for identifying bioactive agents for select neurotransmitter systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGop9GIOvrEeDLVg90H21EOLACvtfcHk0lhOA1uc9JTaXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVClsL8%253D&md5=f70ee6ba04c7db3d98f99ac836d2cf5c</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fmp.2012.170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmp.2012.170%26sid%3Dliteratum%253Aachs%26aulast%3DLawal%26aufirst%3DH.%2BO.%26aulast%3DTerrell%26aufirst%3DA.%26aulast%3DLam%26aufirst%3DH.%2BA.%26aulast%3DDjapri%26aufirst%3DC.%26aulast%3DJang%26aufirst%3DJ.%26aulast%3DHadi%26aufirst%3DR.%26aulast%3DRoberts%26aufirst%3DL.%26aulast%3DShahi%26aufirst%3DV.%26aulast%3DChou%26aufirst%3DM.%2BT.%26aulast%3DBiedermann%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DLawless%26aufirst%3DG.%2BM.%26aulast%3DMaidment%26aufirst%3DN.%2BT.%26aulast%3DKrantz%26aufirst%3DD.%2BE.%26atitle%3DDrosophila%2520Modifier%2520Screens%2520to%2520Identify%2520Novel%2520Neuropsychiatric%2520Drugs%2520Including%2520Aminergic%2520Agents%2520for%2520the%2520Possible%2520Treatment%2520of%2520Parkinson%25E2%2580%2599s%2520Disease%2520and%2520Depression%26jtitle%3DMol.%2520Psychiatry%26date%3D2014%26volume%3D19%26spage%3D235%26epage%3D242%26doi%3D10.1038%2Fmp.2012.170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qurashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, P.</span></span> <span> </span><span class="NLM_article-title">Chemical Screen Reveals Small Molecules Suppressing Fragile X Premutation Rcgg Repeat-Mediated Neurodegeneration in Drosophila</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2068</span>– <span class="NLM_lpage">2075</span>, <span class="refDoi"> DOI: 10.1093/hmg/dds024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1093%2Fhmg%2Fdds024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=22298836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVeqtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=2068-2075&author=A.+Qurashiauthor=H.+Liuauthor=L.+Rayauthor=D.+L.+Nelsonauthor=R.+Duanauthor=P.+Jin&title=Chemical+Screen+Reveals+Small+Molecules+Suppressing+Fragile+X+Premutation+Rcgg+Repeat-Mediated+Neurodegeneration+in+Drosophila&doi=10.1093%2Fhmg%2Fdds024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical screen reveals small molecules suppressing fragile X premutation rCGG repeat-mediated neurodegeneration in Drosophila</span></div><div class="casAuthors">Qurashi, Abrar; Liu, Huijie; Ray, Laurie; Nelson, David L.; Duan, Ranhui; Jin, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2068-2075</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Fragile X-assocd. tremor/ataxia syndrome (FXTAS) is a progressive neurodegenerative disorder recognized in fragile X premutation carriers.  Using Drosophila, we previously identified elongated non-coding CGG repeats in FMR1 allele as the pathogenic cause of FXTAS.  Here, we use this same FXTAS Drosophila model to conduct a chem. screen that reveals small mols. that can ameliorate the toxic effects of fragile X premutation ribo-CGG (rCGG) repeats, among them several known phospholipase A2 (PLA2) inhibitors.  We show that specific inhibition of PLA2 activity could mitigate the neuronal deficits caused by fragile X premutation rCGG repeats, including lethality and locomotion deficits.  Furthermore, through a genetic screen, we identified a PLA2 Drosophila ortholog that specifically modulates rCGG repeat-mediated neuronal toxicity.  Our results demonstrate the utility of Drosophila models for unbiased small mol. screens and point to PLA2 as a possible therapeutic target to treat FXTAS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsCtze8pzp_bVg90H21EOLACvtfcHk0lhC2VVoqF2vDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVeqtrs%253D&md5=ccd2cad64b01503be7870c6e33fd09ab</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2Fdds024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252Fdds024%26sid%3Dliteratum%253Aachs%26aulast%3DQurashi%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DRay%26aufirst%3DL.%26aulast%3DNelson%26aufirst%3DD.%2BL.%26aulast%3DDuan%26aufirst%3DR.%26aulast%3DJin%26aufirst%3DP.%26atitle%3DChemical%2520Screen%2520Reveals%2520Small%2520Molecules%2520Suppressing%2520Fragile%2520X%2520Premutation%2520Rcgg%2520Repeat-Mediated%2520Neurodegeneration%2520in%2520Drosophila%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2012%26volume%3D21%26spage%3D2068%26epage%3D2075%26doi%3D10.1093%2Fhmg%2Fdds024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pieper, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capota, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estill, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntington, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldman, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capota, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Britt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotti, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ure, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brat, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacMillan, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naidoo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melito, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Brabander, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ready, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKnight, S. L.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Proneurogenic, Neuroprotective Chemical</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">39</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2010.06.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.cell.2010.06.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=20603013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpt1Cjt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2010&pages=39-51&author=A.+A.+Pieperauthor=S.+Xieauthor=E.+Capotaauthor=S.+J.+Estillauthor=J.+Zhongauthor=J.+M.+Longauthor=G.+L.+Beckerauthor=P.+Huntingtonauthor=S.+E.+Goldmanauthor=C.+H.+Shenauthor=M.+Capotaauthor=J.+K.+Brittauthor=T.+Kottiauthor=K.+Ureauthor=D.+J.+Bratauthor=N.+S.+Williamsauthor=K.+S.+MacMillanauthor=J.+Naidooauthor=L.+Melitoauthor=J.+Hsiehauthor=J.+De+Brabanderauthor=J.+M.+Readyauthor=S.+L.+McKnight&title=Discovery+of+a+Proneurogenic%2C+Neuroprotective+Chemical&doi=10.1016%2Fj.cell.2010.06.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a proneurogenic, neuroprotective chemical</span></div><div class="casAuthors">Pieper, Andrew A.; Xie, Shanhai; Capota, Emanuela; Estill, Sandi Jo; Zhong, Jeannie; Long, Jeffrey M.; Becker, Ginger L.; Huntington, Paula; Goldman, Shauna E.; Shen, Ching-Han; Capota, Maria; Britt, Jeremiah K.; Kotti, Tiina; Ure, Kerstin; Brat, Daniel J.; Williams, Noelle S.; MacMillan, Karen S.; Naidoo, Jacinth; Melito, Lisa; Hsieh, Jenny; De Brabander, Jef; Ready, Joseph M.; McKnight, Steven L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">39-51</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">An in vivo screen was performed in search of chems. capable of enhancing neuron formation in the hippocampus of adult mice.  Eight of 1000 small mols. tested enhanced neuron formation in the subgranular zone of the dentate gyrus.  Among these was an aminopropyl carbazole, designated P7C3, endowed with favorable pharmacol. properties.  In vivo studies gave evidence that P7C3 exerts its proneurogenic activity by protecting newborn neurons from apoptosis.  Mice missing the gene encoding neuronal PAS domain protein 3 (NPAS3) are devoid of hippocampal neurogenesis and display malformation and electrophysiol. dysfunction of the dentate gyrus.  Prolonged administration of P7C3 to npas3-/- mice cor. these deficits by normalizing levels of apoptosis of newborn hippocampal neurons.  Prolonged administration of P7C3 to aged rats also enhanced neurogenesis in the dentate gyrus, impeded neuron death, and preserved cognitive capacity as a function of terminal aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDL4C8LqlFL7Vg90H21EOLACvtfcHk0lhC2VVoqF2vDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpt1Cjt7Y%253D&md5=7ed01874bd3e1b5cde587b2bc244741d</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2010.06.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2010.06.018%26sid%3Dliteratum%253Aachs%26aulast%3DPieper%26aufirst%3DA.%2BA.%26aulast%3DXie%26aufirst%3DS.%26aulast%3DCapota%26aufirst%3DE.%26aulast%3DEstill%26aufirst%3DS.%2BJ.%26aulast%3DZhong%26aufirst%3DJ.%26aulast%3DLong%26aufirst%3DJ.%2BM.%26aulast%3DBecker%26aufirst%3DG.%2BL.%26aulast%3DHuntington%26aufirst%3DP.%26aulast%3DGoldman%26aufirst%3DS.%2BE.%26aulast%3DShen%26aufirst%3DC.%2BH.%26aulast%3DCapota%26aufirst%3DM.%26aulast%3DBritt%26aufirst%3DJ.%2BK.%26aulast%3DKotti%26aufirst%3DT.%26aulast%3DUre%26aufirst%3DK.%26aulast%3DBrat%26aufirst%3DD.%2BJ.%26aulast%3DWilliams%26aufirst%3DN.%2BS.%26aulast%3DMacMillan%26aufirst%3DK.%2BS.%26aulast%3DNaidoo%26aufirst%3DJ.%26aulast%3DMelito%26aufirst%3DL.%26aulast%3DHsieh%26aufirst%3DJ.%26aulast%3DDe%2BBrabander%26aufirst%3DJ.%26aulast%3DReady%26aufirst%3DJ.%2BM.%26aulast%3DMcKnight%26aufirst%3DS.%2BL.%26atitle%3DDiscovery%2520of%2520a%2520Proneurogenic%252C%2520Neuroprotective%2520Chemical%26jtitle%3DCell%26date%3D2010%26volume%3D142%26spage%3D39%26epage%3D51%26doi%3D10.1016%2Fj.cell.2010.06.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alexandrov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanania, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leahy, E.</span></span> <span> </span><span class="NLM_article-title">Reprint Of: Highthroughtput Analysis of Behavior for Drug Discovery</span>. <i>Eur. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>753</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">134</span>, <span class="refDoi"> DOI: 10.1016/j.ejphar.2015.02.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.ejphar.2015.02.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25744878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktFKisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=753&publication_year=2015&pages=127-134&author=V.+Alexandrovauthor=D.+Brunnerauthor=T.+Hananiaauthor=E.+Leahy&title=Reprint+Of%3A+Highthroughtput+Analysis+of+Behavior+for+Drug+Discovery&doi=10.1016%2Fj.ejphar.2015.02.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Reprint of: Highthroughtput analysis of behavior for drug discovery</span></div><div class="casAuthors">Alexandrov, Vadim; Brunner, Dani; Hanania, Taleen; Leahy, Emer</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">753</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">127-134</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Drug testing with traditional behavioral assays constitutes a major bottleneck in the development of novel therapies.  PsychoGenics developed three comprehensive highthroughtput systems, SmartCube, NeuroCube and PhenoCube systems, to increase the efficiency of the drug screening and phenotyping in rodents.  These three systems capture different domains of behavior, namely, cognitive, motor, circadian, social, anxiety-like, gait and others, using custom-built computer vision software and machine learning algorithms for anal.  This review exemplifies the use of the three systems and explains how they can advance drug screening with their applications to phenotyping of disease models, drug screening, selection of lead candidates, behavior-driven lead optimization, and drug repurposing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpftTYagkadvbVg90H21EOLACvtfcHk0lg8UKwQrP_VBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktFKisrY%253D&md5=9c59a0180a0f0f3bb198ded8079ecd49</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.ejphar.2015.02.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejphar.2015.02.037%26sid%3Dliteratum%253Aachs%26aulast%3DAlexandrov%26aufirst%3DV.%26aulast%3DBrunner%26aufirst%3DD.%26aulast%3DHanania%26aufirst%3DT.%26aulast%3DLeahy%26aufirst%3DE.%26atitle%3DReprint%2520Of%253A%2520Highthroughtput%2520Analysis%2520of%2520Behavior%2520for%2520Drug%2520Discovery%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D2015%26volume%3D753%26spage%3D127%26epage%3D134%26doi%3D10.1016%2Fj.ejphar.2015.02.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span> <i>Sunovion and PsychoGenics Announce that SEP-363856 Has Received
FDA
Breakthrough Therapy Designation for the Treatment of People with
Schizophrenia</i>; <span class="NLM_publisher-name">Sunovion Pharmaceuticals,
Inc.</span>, May 10, <span class="NLM_year">2019</span>; <a href="https://news.sunovion.com/press-release/sunovion-and-psychogenics-announce-sep-363856-has-received-fda-breakthrough-therapy" class="extLink">https://news.sunovion.com/press-release/sunovion-and-psychogenics-announce-sep-363856-has-received-fda-breakthrough-therapy</a> (accessed July 2, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Sunovion+and+PsychoGenics+Announce+that+SEP-363856+Has+Received%0AFDA%0ABreakthrough+Therapy+Designation+for+the+Treatment+of+People+with%0ASchizophrenia%3B+Sunovion+Pharmaceuticals%2C%0AInc.%2C+May+10%2C+2019%3B+https%3A%2F%2Fnews.sunovion.com%2Fpress-release%2Fsunovion-and-psychogenics-announce-sep-363856-has-received-fda-breakthrough-therapy+%28accessed+July+2%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DSunovion%2520and%2520PsychoGenics%2520Announce%2520that%2520SEP-363856%2520Has%2520Received%250AFDA%250ABreakthrough%2520Therapy%2520Designation%2520for%2520the%2520Treatment%2520of%2520People%2520with%250ASchizophrenia%26pub%3DSunovion%2520Pharmaceuticals%252C%250AInc%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hollerhage, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moebius, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melms, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goebel, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakroun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koeglsperger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oertel, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosler, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bickle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoglinger, G. U.</span></span> <span> </span><span class="NLM_article-title">Protective Efficacy of Phosphodiesterase-1 Inhibition against Alpha-Synuclein Toxicity Revealed by Compound Screening in Luhmes Cells</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">11469</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-11664-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41598-017-11664-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28904388" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC1cbnvFGjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=11469&author=M.+Hollerhageauthor=C.+Moebiusauthor=J.+Melmsauthor=W.+H.+Chiuauthor=J.+N.+Goebelauthor=T.+Chakrounauthor=T.+Koeglspergerauthor=W.+H.+Oertelauthor=T.+W.+Roslerauthor=M.+Bickleauthor=G.+U.+Hoglinger&title=Protective+Efficacy+of+Phosphodiesterase-1+Inhibition+against+Alpha-Synuclein+Toxicity+Revealed+by+Compound+Screening+in+Luhmes+Cells&doi=10.1038%2Fs41598-017-11664-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Protective efficacy of phosphodiesterase-1 inhibition against alpha-synuclein toxicity revealed by compound screening in LUHMES cells</span></div><div class="casAuthors">Hollerhage Matthias; Melms Johannes; Chakroun Tasnim; Koeglsperger Thomas; Hoglinger Gunter U; Hollerhage Matthias; Melms Johannes; Rosler Thomas W; Hoglinger Gunter U; Moebius Claudia; Bickle Marc; Chiu Wei-Hua; Goebel Joachim N; Oertel Wolfgang H; Chakroun Tasnim; Rosler Thomas W; Hoglinger Gunter U; Koeglsperger Thomas; Oertel Wolfgang H</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11469</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">α-synuclein-induced neurotoxicity is a core pathogenic event in neurodegenerative synucleinopathies such as Parkinson's disease, dementia with Lewy bodies, or multiple system atrophy.  There is currently no disease-modifying therapy available for these diseases.  We screened 1,600 FDA-approved drugs for their efficacy to protect LUHMES cells from degeneration induced by wild-type α-synuclein and identified dipyridamole, a non-selective phosphodiesterase inhibitor, as top hit.  Systematic analysis of other phosphodiesterase inhibitors identified a specific phosphodiesterase 1 inhibitor as most potent to rescue from α-synuclein toxicity.  Protection was mediated by an increase of cGMP and associated with the reduction of a specific α-synuclein oligomeric species.  RNA interference experiments confirmed PDE1A and to a smaller extent PDE1C as molecular targets accounting for the protective efficacy.  PDE1 inhibition also rescued dopaminergic neurons from wild-type α-synuclein induced degeneration in the substantia nigra of mice.  In conclusion, this work identifies inhibition of PDE1A in particular as promising target for neuroprotective treatment of synucleinopathies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSVXi-Kda499iO2kE7krGOBfW6udTcc2eb5jzrE4RyNALntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbnvFGjsg%253D%253D&md5=5d8dcfa2a0e540ba23ac26c29f79f0c2</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-11664-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-11664-5%26sid%3Dliteratum%253Aachs%26aulast%3DHollerhage%26aufirst%3DM.%26aulast%3DMoebius%26aufirst%3DC.%26aulast%3DMelms%26aufirst%3DJ.%26aulast%3DChiu%26aufirst%3DW.%2BH.%26aulast%3DGoebel%26aufirst%3DJ.%2BN.%26aulast%3DChakroun%26aufirst%3DT.%26aulast%3DKoeglsperger%26aufirst%3DT.%26aulast%3DOertel%26aufirst%3DW.%2BH.%26aulast%3DRosler%26aufirst%3DT.%2BW.%26aulast%3DBickle%26aufirst%3DM.%26aulast%3DHoglinger%26aufirst%3DG.%2BU.%26atitle%3DProtective%2520Efficacy%2520of%2520Phosphodiesterase-1%2520Inhibition%2520against%2520Alpha-Synuclein%2520Toxicity%2520Revealed%2520by%2520Compound%2520Screening%2520in%2520Luhmes%2520Cells%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D11469%26doi%3D10.1038%2Fs41598-017-11664-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sidrauski, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acosta-Alvear, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khoutorsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedantham, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hearn, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamache, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ang, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okreglak, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashkenazi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nader, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renslo, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonenberg, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, P.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Brake-Release of Mrna Translation Enhances Cognitive Memory</span>. <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e00498</span> <span class="refDoi"> DOI: 10.7554/eLife.00498</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.7554%2FeLife.00498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23741617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFKht7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&author=C.+Sidrauskiauthor=D.+Acosta-Alvearauthor=A.+Khoutorskyauthor=P.+Vedanthamauthor=B.+R.+Hearnauthor=H.+Liauthor=K.+Gamacheauthor=C.+M.+Gallagherauthor=K.+K.+Angauthor=C.+Wilsonauthor=V.+Okreglakauthor=A.+Ashkenaziauthor=B.+Hannauthor=K.+Naderauthor=M.+R.+Arkinauthor=A.+R.+Rensloauthor=N.+Sonenbergauthor=P.+Walter&title=Pharmacological+Brake-Release+of+Mrna+Translation+Enhances+Cognitive+Memory&doi=10.7554%2FeLife.00498"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological brake-release of mRNA translation enhances cognitive memory</span></div><div class="casAuthors">Sidrauski, Carmela; Acosta-Alvear, Diego; Khoutorsky, Arkady; Vedantham, Punitha; Hearn, Brian R.; Li, Han; Gamache, Karine; Gallagher, Ciara M.; Ang, Kneey K-H.; Wilson, Chris; Okreglak, Voytek; Ashkenazi, Avi; Hann, Byron; Nader, Karim; Arkin, Michelle R.; Renslo, Adam R.; Sonenberg, Nahum; Walter, Peter</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e00498/1-e00498/22</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Phosphorylation of the a-subunit of initiation factor 2 (eIF2) controls protein synthesis by a conserved mechanism.  In metazoa, distinct stress conditions activate different eIF2α kinases (PERK, PKR, GCN2, and HRI) that converge on phosphorylating a unique serine in eIF2α.  This collection of signaling pathways is termed the 'integrated stress response' (ISR).  EIF2α phosphorylation diminishes protein synthesis, while allowing preferential translation of some mRNAs.  Starting with a cell-based screen for inhibitors of PERK signaling, we identified a small mol., named ISRIB, that potently (IC50 = 5 nM) reverses the effects of eIF2α phosphorylation.  ISRIB reduces the viability of cells subjected to PERK-activation by chronic endoplasmic reticulum stress.  EIF2α phosphorylation is implicated in memory consolidation.  Remarkably, ISRIB-treated mice display significant enhancement in spatial and fear-assocd. learning.  Thus, memory consolidation is inherently limited by the ISR, and ISRIB releases this brake.  As such, ISRIB promises to contribute to our understanding and treatment of cognitive disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaBg1X22tUYrVg90H21EOLACvtfcHk0ljjVQUHWIogRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntFKht7Y%253D&md5=1e74b0506a5ce689ed502ccc5f5776c7</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.7554%2FeLife.00498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.00498%26sid%3Dliteratum%253Aachs%26aulast%3DSidrauski%26aufirst%3DC.%26aulast%3DAcosta-Alvear%26aufirst%3DD.%26aulast%3DKhoutorsky%26aufirst%3DA.%26aulast%3DVedantham%26aufirst%3DP.%26aulast%3DHearn%26aufirst%3DB.%2BR.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DGamache%26aufirst%3DK.%26aulast%3DGallagher%26aufirst%3DC.%2BM.%26aulast%3DAng%26aufirst%3DK.%2BK.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DOkreglak%26aufirst%3DV.%26aulast%3DAshkenazi%26aufirst%3DA.%26aulast%3DHann%26aufirst%3DB.%26aulast%3DNader%26aufirst%3DK.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DRenslo%26aufirst%3DA.%2BR.%26aulast%3DSonenberg%26aufirst%3DN.%26aulast%3DWalter%26aufirst%3DP.%26atitle%3DPharmacological%2520Brake-Release%2520of%2520Mrna%2520Translation%2520Enhances%2520Cognitive%2520Memory%26jtitle%3DElife%26date%3D2013%26volume%3D2%26doi%3D10.7554%2FeLife.00498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Naryshkin, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weetall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karp, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woll, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabbeta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazirani, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharyya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Risher, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheedy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turpoff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trifillis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welch, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colacino, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babiak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almstead, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peltz, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eng, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mull, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynes, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontoura, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santarelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haehnke, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmucki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebeling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivaramakrishnan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paushkin, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, F.</span></span> <span> </span><span class="NLM_article-title">Motor Neuron Disease. Smn2 Splicing Modifiers Improve Motor Function and Longevity in Mice with Spinal Muscular Atrophy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>345</i></span>,  <span class="NLM_fpage">688</span>– <span class="NLM_lpage">693</span>, <span class="refDoi"> DOI: 10.1126/science.1250127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fscience.1250127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25104390" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12gtLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=345&publication_year=2014&pages=688-693&author=N.+A.+Naryshkinauthor=M.+Weetallauthor=A.+Dakkaauthor=J.+Narasimhanauthor=X.+Zhaoauthor=Z.+Fengauthor=K.+K.+Lingauthor=G.+M.+Karpauthor=H.+Qiauthor=M.+G.+Wollauthor=G.+Chenauthor=N.+Zhangauthor=V.+Gabbetaauthor=P.+Vaziraniauthor=A.+Bhattacharyyaauthor=B.+Furiaauthor=N.+Risherauthor=J.+Sheedyauthor=R.+Kongauthor=J.+Maauthor=A.+Turpoffauthor=C.+S.+Leeauthor=X.+Zhangauthor=Y.+C.+Moonauthor=P.+Trifillisauthor=E.+M.+Welchauthor=J.+M.+Colacinoauthor=J.+Babiakauthor=N.+G.+Almsteadauthor=S.+W.+Peltzauthor=L.+A.+Engauthor=K.+S.+Chenauthor=J.+L.+Mullauthor=M.+S.+Lynesauthor=L.+L.+Rubinauthor=P.+Fontouraauthor=L.+Santarelliauthor=D.+Haehnkeauthor=K.+D.+McCarthyauthor=R.+Schmuckiauthor=M.+Ebelingauthor=M.+Sivaramakrishnanauthor=C.+P.+Koauthor=S.+V.+Paushkinauthor=H.+Ratniauthor=I.+Gerlachauthor=A.+Ghoshauthor=F.+Metzger&title=Motor+Neuron+Disease.+Smn2+Splicing+Modifiers+Improve+Motor+Function+and+Longevity+in+Mice+with+Spinal+Muscular+Atrophy&doi=10.1126%2Fscience.1250127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">SMN2 splicing modifiers improve motor function and longevity in mice with spinal muscular atrophy</span></div><div class="casAuthors">Naryshkin, Nikolai A.; Weetall, Marla; Dakka, Amal; Narasimhan, Jana; Zhao, Xin; Feng, Zhihua; Ling, Karen K. Y.; Karp, Gary M.; Qi, Hongyan; Woll, Matthew G.; Chen, Guangming; Zhang, Nanjing; Gabbeta, Vijayalakshmi; Vazirani, Priya; Bhattacharyya, Anuradha; Furia, Bansri; Risher, Nicole; Sheedy, Josephine; Kong, Ronald; Ma, Jiyuan; Turpoff, Anthony; Lee, Chang-Sun; Zhang, Xiaoyan; Moon, Young-Choon; Trifillis, Panayiota; Welch, Ellen M.; Colacino, Joseph M.; Babiak, John; Almstead, Neil G.; Peltz, Stuart W.; Eng, Loren A.; Chen, Karen S.; Mull, Jesse L.; Lynes, Maureen S.; Rubin, Lee L.; Fontoura, Paulo; Santarelli, Luca; Haehnke, Daniel; McCarthy, Kathleen D.; Schmucki, Roland; Ebeling, Martin; Sivaramakrishnan, Manaswini; Ko, Chien-Ping; Paushkin, Sergey V.; Ratni, Hasane; Gerlach, Irene; Ghosh, Anirvan; Metzger, Friedrich</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">345</span>
        (<span class="NLM_cas:issue">6197</span>),
    <span class="NLM_cas:pages">688-693</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Spinal muscular atrophy (SMA) is a genetic disease caused by mutation or deletion of the survival of motor neuron 1 (SMN1) gene.  A paralogous gene in humans, SMN2, produces low, insufficient levels of functional SMN protein due to alternative splicing that truncates the transcript.  The decreased levels of SMN protein lead to progressive neuromuscular degeneration and high rates of mortality.  Through chem. screening and optimization, we identified orally available small mols. that shift the balance of SMN2 splicing toward the prodn. of full-length SMN2 mRNA with high selectivity.  Administration of these compds. to Δ7 mice, a model of severe SMA, led to an increase in SMN protein levels, improvement of motor function, and protection of the neuromuscular circuit.  These compds. also extended the life span of the mice.  Selective SMN2 splicing modifiers may have therapeutic potential for patients with SMA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxZ8QSg5KSzLVg90H21EOLACvtfcHk0ljjVQUHWIogRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12gtLfO&md5=f321c411bb384f95a9fe4d4debcbb195</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1126%2Fscience.1250127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1250127%26sid%3Dliteratum%253Aachs%26aulast%3DNaryshkin%26aufirst%3DN.%2BA.%26aulast%3DWeetall%26aufirst%3DM.%26aulast%3DDakka%26aufirst%3DA.%26aulast%3DNarasimhan%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DLing%26aufirst%3DK.%2BK.%26aulast%3DKarp%26aufirst%3DG.%2BM.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DWoll%26aufirst%3DM.%2BG.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DGabbeta%26aufirst%3DV.%26aulast%3DVazirani%26aufirst%3DP.%26aulast%3DBhattacharyya%26aufirst%3DA.%26aulast%3DFuria%26aufirst%3DB.%26aulast%3DRisher%26aufirst%3DN.%26aulast%3DSheedy%26aufirst%3DJ.%26aulast%3DKong%26aufirst%3DR.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DTurpoff%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DC.%2BS.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMoon%26aufirst%3DY.%2BC.%26aulast%3DTrifillis%26aufirst%3DP.%26aulast%3DWelch%26aufirst%3DE.%2BM.%26aulast%3DColacino%26aufirst%3DJ.%2BM.%26aulast%3DBabiak%26aufirst%3DJ.%26aulast%3DAlmstead%26aufirst%3DN.%2BG.%26aulast%3DPeltz%26aufirst%3DS.%2BW.%26aulast%3DEng%26aufirst%3DL.%2BA.%26aulast%3DChen%26aufirst%3DK.%2BS.%26aulast%3DMull%26aufirst%3DJ.%2BL.%26aulast%3DLynes%26aufirst%3DM.%2BS.%26aulast%3DRubin%26aufirst%3DL.%2BL.%26aulast%3DFontoura%26aufirst%3DP.%26aulast%3DSantarelli%26aufirst%3DL.%26aulast%3DHaehnke%26aufirst%3DD.%26aulast%3DMcCarthy%26aufirst%3DK.%2BD.%26aulast%3DSchmucki%26aufirst%3DR.%26aulast%3DEbeling%26aufirst%3DM.%26aulast%3DSivaramakrishnan%26aufirst%3DM.%26aulast%3DKo%26aufirst%3DC.%2BP.%26aulast%3DPaushkin%26aufirst%3DS.%2BV.%26aulast%3DRatni%26aufirst%3DH.%26aulast%3DGerlach%26aufirst%3DI.%26aulast%3DGhosh%26aufirst%3DA.%26aulast%3DMetzger%26aufirst%3DF.%26atitle%3DMotor%2520Neuron%2520Disease.%2520Smn2%2520Splicing%2520Modifiers%2520Improve%2520Motor%2520Function%2520and%2520Longevity%2520in%2520Mice%2520with%2520Spinal%2520Muscular%2520Atrophy%26jtitle%3DScience%26date%3D2014%26volume%3D345%26spage%3D688%26epage%3D693%26doi%3D10.1126%2Fscience.1250127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ratni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karp, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weetall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naryshkin, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paushkin, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turpoff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woll, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazirani, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinard, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Pierson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuerck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muster, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirchner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, F.</span></span> <span> </span><span class="NLM_article-title">Specific Correction of Alternative Survival Motor Neuron 2 Splicing by Small Molecules: Discovery of a Potential Novel Medicine to Treat Spinal Muscular Atrophy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">6086</span>– <span class="NLM_lpage">6100</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00459</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00459" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslWgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6086-6100&author=H.+Ratniauthor=G.+M.+Karpauthor=M.+Weetallauthor=N.+A.+Naryshkinauthor=S.+V.+Paushkinauthor=K.+S.+Chenauthor=K.+D.+McCarthyauthor=H.+Qiauthor=A.+Turpoffauthor=M.+G.+Wollauthor=X.+Zhangauthor=N.+Zhangauthor=T.+Yangauthor=A.+Dakkaauthor=P.+Vaziraniauthor=X.+Zhaoauthor=E.+Pinardauthor=L.+Greenauthor=P.+David-Piersonauthor=D.+Tuerckauthor=A.+Poirierauthor=W.+Musterauthor=S.+Kirchnerauthor=L.+Muellerauthor=I.+Gerlachauthor=F.+Metzger&title=Specific+Correction+of+Alternative+Survival+Motor+Neuron+2+Splicing+by+Small+Molecules%3A+Discovery+of+a+Potential+Novel+Medicine+to+Treat+Spinal+Muscular+Atrophy&doi=10.1021%2Facs.jmedchem.6b00459"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Specific Correction of Alternative Survival Motor Neuron 2 Splicing by Small Molecules: Discovery of a Potential Novel Medicine To Treat Spinal Muscular Atrophy</span></div><div class="casAuthors">Ratni, Hasane; Karp, Gary M.; Weetall, Marla; Naryshkin, Nikolai A.; Paushkin, Sergey V.; Chen, Karen S.; McCarthy, Kathleen D.; Qi, Hongyan; Turpoff, Anthony; Woll, Matthew G.; Zhang, Xiaoyan; Zhang, Nanjing; Yang, Tianle; Dakka, Amal; Vazirani, Priya; Zhao, Xin; Pinard, Emmanuel; Green, Luke; David-Pierson, Pascale; Tuerck, Dietrich; Poirier, Agnes; Muster, Wolfgang; Kirchner, Stephan; Mueller, Lutz; Gerlach, Irene; Metzger, Friedrich</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6086-6100</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Spinal muscular atrophy (SMA) is the leading genetic cause of infant and toddler mortality, and there is currently no approved therapy available.  SMA is caused by mutation or deletion of the survival motor neuron 1 (SMN1) gene.  These mutations or deletions result in low levels of functional SMN protein.  SMN2, a paralogous gene to SMN1, undergoes alternative splicing and exclusion of exon 7, producing an unstable, truncated SMNΔ7 protein.  Herein, we report the identification of a pyridopyrimidinone series of small mols. that modify the alternative splicing of SMN2, increasing the prodn. of full-length SMN2 mRNA.  Upon oral administration of our small mols., the levels of full-length SMN protein were restored in two mouse models of SMA.  In-depth lead optimization in the pyridopyrimidinone series culminated in the selection of compd. 3 (RG7800) (I), the first small mol. SMN2 splicing modifier to enter human clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDRpxc7_n787Vg90H21EOLACvtfcHk0ljnZZjUDnFC1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslWgurw%253D&md5=14431b4127804cc623c2365298ad849b</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00459%26sid%3Dliteratum%253Aachs%26aulast%3DRatni%26aufirst%3DH.%26aulast%3DKarp%26aufirst%3DG.%2BM.%26aulast%3DWeetall%26aufirst%3DM.%26aulast%3DNaryshkin%26aufirst%3DN.%2BA.%26aulast%3DPaushkin%26aufirst%3DS.%2BV.%26aulast%3DChen%26aufirst%3DK.%2BS.%26aulast%3DMcCarthy%26aufirst%3DK.%2BD.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DTurpoff%26aufirst%3DA.%26aulast%3DWoll%26aufirst%3DM.%2BG.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DDakka%26aufirst%3DA.%26aulast%3DVazirani%26aufirst%3DP.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DPinard%26aufirst%3DE.%26aulast%3DGreen%26aufirst%3DL.%26aulast%3DDavid-Pierson%26aufirst%3DP.%26aulast%3DTuerck%26aufirst%3DD.%26aulast%3DPoirier%26aufirst%3DA.%26aulast%3DMuster%26aufirst%3DW.%26aulast%3DKirchner%26aufirst%3DS.%26aulast%3DMueller%26aufirst%3DL.%26aulast%3DGerlach%26aufirst%3DI.%26aulast%3DMetzger%26aufirst%3DF.%26atitle%3DSpecific%2520Correction%2520of%2520Alternative%2520Survival%2520Motor%2520Neuron%25202%2520Splicing%2520by%2520Small%2520Molecules%253A%2520Discovery%2520of%2520a%2520Potential%2520Novel%2520Medicine%2520to%2520Treat%2520Spinal%2520Muscular%2520Atrophy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6086%26epage%3D6100%26doi%3D10.1021%2Facs.jmedchem.6b00459" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Woll, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turpoff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almstead, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naryshkin, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabbeta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welch, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Risher, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheedy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weetall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karp, G. M.</span></span> <span> </span><span class="NLM_article-title">Discovery and Optimization of Small Molecule Splicing Modifiers of Survival Motor Neuron 2 as a Treatment for Spinal Muscular Atrophy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">6070</span>– <span class="NLM_lpage">6085</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00460</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00460" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslWgur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6070-6085&author=M.+G.+Wollauthor=H.+Qiauthor=A.+Turpoffauthor=N.+Zhangauthor=X.+Zhangauthor=G.+Chenauthor=C.+Liauthor=S.+Huangauthor=T.+Yangauthor=Y.+C.+Moonauthor=C.+S.+Leeauthor=S.+Choiauthor=N.+G.+Almsteadauthor=N.+A.+Naryshkinauthor=A.+Dakkaauthor=J.+Narasimhanauthor=V.+Gabbetaauthor=E.+Welchauthor=X.+Zhaoauthor=N.+Risherauthor=J.+Sheedyauthor=M.+Weetallauthor=G.+M.+Karp&title=Discovery+and+Optimization+of+Small+Molecule+Splicing+Modifiers+of+Survival+Motor+Neuron+2+as+a+Treatment+for+Spinal+Muscular+Atrophy&doi=10.1021%2Facs.jmedchem.6b00460"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Small Molecule Splicing Modifiers of Survival Motor Neuron 2 as a Treatment for Spinal Muscular Atrophy</span></div><div class="casAuthors">Woll, Matthew G.; Qi, Hongyan; Turpoff, Anthony; Zhang, Nanjing; Zhang, Xiaoyan; Chen, Guangming; Li, Chunshi; Huang, Song; Yang, Tianle; Moon, Young-Choon; Lee, Chang-Sun; Choi, Soongyu; Almstead, Neil G.; Naryshkin, Nikolai A.; Dakka, Amal; Narasimhan, Jana; Gabbeta, Vijayalakshmi; Welch, Ellen; Zhao, Xin; Risher, Nicole; Sheedy, Josephine; Weetall, Marla; Karp, Gary M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6070-6085</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The underlying cause of spinal muscular atrophy (SMA) is a deficiency of the survival motor neuron (SMN) protein.  Starting from hits identified in a high-throughput screening campaign and through structure-activity relationship investigations, we have developed small mols. that potently shift the alternative splicing of the SMN2 exon 7, resulting in increased prodn. of the full-length SMN mRNA and protein.  Three novel chem. series, represented by compds. 9, 14, and 20, have been optimized to increase the level of SMN protein by >50% in SMA patient-derived fibroblasts at concns. of <160 nM.  Daily administration of these compds. to severe SMA Δ7 mice results in an increased prodn. of SMN protein in disease-relevant tissues and a significant increase in median survival time in a dose-dependent manner.  Our work supports the development of an orally administered small mol. for the treatment of patients with SMA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF6iU-0q9serVg90H21EOLACvtfcHk0ljnZZjUDnFC1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslWgur0%253D&md5=9b32bd7380625b6ddd3e97479f98e79b</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00460&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00460%26sid%3Dliteratum%253Aachs%26aulast%3DWoll%26aufirst%3DM.%2BG.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DTurpoff%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DMoon%26aufirst%3DY.%2BC.%26aulast%3DLee%26aufirst%3DC.%2BS.%26aulast%3DChoi%26aufirst%3DS.%26aulast%3DAlmstead%26aufirst%3DN.%2BG.%26aulast%3DNaryshkin%26aufirst%3DN.%2BA.%26aulast%3DDakka%26aufirst%3DA.%26aulast%3DNarasimhan%26aufirst%3DJ.%26aulast%3DGabbeta%26aufirst%3DV.%26aulast%3DWelch%26aufirst%3DE.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DRisher%26aufirst%3DN.%26aulast%3DSheedy%26aufirst%3DJ.%26aulast%3DWeetall%26aufirst%3DM.%26aulast%3DKarp%26aufirst%3DG.%2BM.%26atitle%3DDiscovery%2520and%2520Optimization%2520of%2520Small%2520Molecule%2520Splicing%2520Modifiers%2520of%2520Survival%2520Motor%2520Neuron%25202%2520as%2520a%2520Treatment%2520for%2520Spinal%2520Muscular%2520Atrophy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6070%26epage%3D6085%26doi%3D10.1021%2Facs.jmedchem.6b00460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pickhardt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tassoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurkowsky, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitanovic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fava, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandelkow, E.</span></span> <span> </span><span class="NLM_article-title">Screening of a Neuronal Cell Model of Tau Pathology for Therapeutic Compounds</span>. <i>Neurobiol. Aging</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1016/j.neurobiolaging.2018.11.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.neurobiolaging.2018.11.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30640040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslelsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2019&pages=24-34&author=M.+Pickhardtauthor=M.+Tassoniauthor=P.+Dennerauthor=B.+Kurkowskyauthor=A.+Kitanovicauthor=C.+Mohlauthor=E.+Favaauthor=E.+Mandelkow&title=Screening+of+a+Neuronal+Cell+Model+of+Tau+Pathology+for+Therapeutic+Compounds&doi=10.1016%2Fj.neurobiolaging.2018.11.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Screening of a neuronal cell model of tau pathology for therapeutic compounds</span></div><div class="casAuthors">Pickhardt, Marcus; Tassoni, Michele; Denner, Philip; Kurkowsky, Birgit; Kitanovic, Ana; Moehl, Christoph; Fava, Eugenio; Mandelkow, Eckhard</div><div class="citationInfo"><span class="NLM_cas:title">Neurobiology of Aging</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">24-34</span>CODEN:
                <span class="NLM_cas:coden">NEAGDO</span>;
        ISSN:<span class="NLM_cas:issn">0197-4580</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">We have developed a cell-based phenotypic automated high-content screening approach for N2a cells expressing the pro-aggregant repeat domain of tau protein (tauRDΔK), which allows anal. of a chemogenomic library of 1649 compds. for their effect on the inhibition or stimulation of intracellular tau aggregation.  We identified several inhibitors and stimulators of aggregation and achieved a screening reproducibility >85% for all data.  We identified 18 potential inhibitors (= 1.1% of the library) and 10 stimulators (= 0.6% of the library) of tau aggregation in this cell model of tau pathol.  The results provide insights into the regulation of cellular tau aggregation and the pathways involved in this process (e.g., involving signaling via p38 mitogen-activated protein kinase, histone deacetylases, vascular endothelial growth factor, rho/ROCK).  For example, inhibitors of protein kinases (e.g., p38) can reduce tau aggregation, whereas inhibitors of deacetylases (histone deacetylases) can enhance aggregation.  These observations are compatible with reports that phosphorylated or acetylated tau promotes pathol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ6kH-hNDYULVg90H21EOLACvtfcHk0li5tcEHIzPlEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslelsw%253D%253D&md5=ef48fc4956a7c130bd6a4f50283d5aa5</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.neurobiolaging.2018.11.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neurobiolaging.2018.11.026%26sid%3Dliteratum%253Aachs%26aulast%3DPickhardt%26aufirst%3DM.%26aulast%3DTassoni%26aufirst%3DM.%26aulast%3DDenner%26aufirst%3DP.%26aulast%3DKurkowsky%26aufirst%3DB.%26aulast%3DKitanovic%26aufirst%3DA.%26aulast%3DMohl%26aufirst%3DC.%26aulast%3DFava%26aufirst%3DE.%26aulast%3DMandelkow%26aufirst%3DE.%26atitle%3DScreening%2520of%2520a%2520Neuronal%2520Cell%2520Model%2520of%2520Tau%2520Pathology%2520for%2520Therapeutic%2520Compounds%26jtitle%3DNeurobiol.%2520Aging%26date%3D2019%26volume%3D76%26spage%3D24%26epage%3D34%26doi%3D10.1016%2Fj.neurobiolaging.2018.11.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ofengeim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polydoro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degterev, A.</span></span> <span> </span><span class="NLM_article-title">Identification of Small Molecule Inhibitors of Neurite Loss Induced by Abeta Peptide Using High Content Screening</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">8714</span>– <span class="NLM_lpage">8723</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.290957</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1074%2Fjbc.M111.290957" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=22277654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC38XjvVeht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=8714-8723&author=D.+Ofengeimauthor=P.+Shiauthor=B.+Miaoauthor=J.+Fanauthor=X.+Xiaauthor=Y.+Fanauthor=M.+M.+Lipinskiauthor=T.+Hashimotoauthor=M.+Polydoroauthor=J.+Yuanauthor=S.+T.+Wongauthor=A.+Degterev&title=Identification+of+Small+Molecule+Inhibitors+of+Neurite+Loss+Induced+by+Abeta+Peptide+Using+High+Content+Screening&doi=10.1074%2Fjbc.M111.290957"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Small Molecule Inhibitors of Neurite Loss Induced by Aβ peptide using High Content Screening</span></div><div class="casAuthors">Ofengeim, Dimitry; Shi, Peng; Miao, Benchun; Fan, Jing; Xia, Xiaofeng; Fan, Yubo; Lipinski, Marta M.; Hashimoto, Tadafumi; Polydoro, Manuela; Yuan, Junying; Wong, Stephen T. C.; Degterev, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">8714-8723</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Multiple lines of evidence indicate a strong relationship between Aβ peptide-induced neurite degeneration and the progressive loss of cognitive functions in Alzheimer disease (AD) patients and in AD animal models.  This prompted us to develop a high content screening assay (HCS) and Neurite Image Quantitator (NeuriteIQ) software to quantify the loss of neuronal projections induced by Aβ peptide neurons and enable us to identify new classes of neurite-protective small mols., which may represent new leads for AD drug discovery.  We identified thirty-six inhibitors of Aβ-induced neurite loss in the 1,040-compd. National Institute of Neurol. Disorders and Stroke (NINDS) custom collection of known bioactives and FDA approved drugs.  Activity clustering showed that non-steroidal anti-inflammatory drugs (NSAIDs) were significantly enriched among the hits.  Notably, NSAIDs have previously attracted significant attention as potential drugs for AD; however their mechanism of action remains controversial.  Our data revealed that cyclooxygenase-2 (COX-2) expression was increased following Aβ treatment.  Furthermore, multiple distinct classes of COX inhibitors efficiently blocked neurite loss in primary neurons, suggesting that increased COX activity contributes to Aβ peptide-induced neurite loss.  Finally, we discovered that the detrimental effect of COX activity on neurite integrity may be mediated through the inhibition of peroxisome proliferator-activated receptor γ (PPARγ) activity.  Overall, our work establishes the feasibility of identifying small mol. inhibitors of Aβ-induced neurite loss using the NeuriteIQ pipeline and provides novel insights into the mechanisms of neuroprotection by NSAIDs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1btFfJUyhMbVg90H21EOLACvtfcHk0li5tcEHIzPlEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjvVeht7c%253D&md5=05a4cac624f04f8882491bada44355fa</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.290957&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.290957%26sid%3Dliteratum%253Aachs%26aulast%3DOfengeim%26aufirst%3DD.%26aulast%3DShi%26aufirst%3DP.%26aulast%3DMiao%26aufirst%3DB.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DXia%26aufirst%3DX.%26aulast%3DFan%26aufirst%3DY.%26aulast%3DLipinski%26aufirst%3DM.%2BM.%26aulast%3DHashimoto%26aufirst%3DT.%26aulast%3DPolydoro%26aufirst%3DM.%26aulast%3DYuan%26aufirst%3DJ.%26aulast%3DWong%26aufirst%3DS.%2BT.%26aulast%3DDegterev%26aufirst%3DA.%26atitle%3DIdentification%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520Neurite%2520Loss%2520Induced%2520by%2520Abeta%2520Peptide%2520Using%2520High%2520Content%2520Screening%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D8714%26epage%3D8723%26doi%3D10.1074%2Fjbc.M111.290957" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Champy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seon-Meniel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franck, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raisman-Vozari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figadere, B.</span></span> <span> </span><span class="NLM_article-title">Chemicals Possessing a Neurotrophin-Like Activity on Dopaminergic Neurons in Primary Culture</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">e6215</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0006215</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1371%2Fjournal.pone.0006215" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=19593444" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&author=F.+Schmidtauthor=P.+Champyauthor=B.+Seon-Menielauthor=X.+Franckauthor=R.+Raisman-Vozariauthor=B.+Figadere&title=Chemicals+Possessing+a+Neurotrophin-Like+Activity+on+Dopaminergic+Neurons+in+Primary+Culture&doi=10.1371%2Fjournal.pone.0006215"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0006215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0006215%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DF.%26aulast%3DChampy%26aufirst%3DP.%26aulast%3DSeon-Meniel%26aufirst%3DB.%26aulast%3DFranck%26aufirst%3DX.%26aulast%3DRaisman-Vozari%26aufirst%3DR.%26aulast%3DFigadere%26aufirst%3DB.%26atitle%3DChemicals%2520Possessing%2520a%2520Neurotrophin-Like%2520Activity%2520on%2520Dopaminergic%2520Neurons%2520in%2520Primary%2520Culture%26jtitle%3DPLoS%2520One%26date%3D2009%26volume%3D4%26doi%3D10.1371%2Fjournal.pone.0006215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mabb, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, I. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miriyala, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor-Blake, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sciaky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutton, J. W.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zylka, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpot, B. D.</span></span> <span> </span><span class="NLM_article-title">Topoisomerase Inhibitors Unsilence the Dormant Allele of Ube3a in Neurons</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>481</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">189</span>, <span class="refDoi"> DOI: 10.1038/nature10726</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnature10726" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1GjsL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=481&publication_year=2012&pages=185-189&author=H.+S.+Huangauthor=J.+A.+Allenauthor=A.+M.+Mabbauthor=I.+F.+Kingauthor=J.+Miriyalaauthor=B.+Taylor-Blakeauthor=N.+Sciakyauthor=J.+W.+Duttonauthor=H.+M.+Leeauthor=X.+Chenauthor=J.+Jinauthor=A.+S.+Bridgesauthor=M.+J.+Zylkaauthor=B.+L.+Rothauthor=B.+D.+Philpot&title=Topoisomerase+Inhibitors+Unsilence+the+Dormant+Allele+of+Ube3a+in+Neurons&doi=10.1038%2Fnature10726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Topoisomerase inhibitors unsilence the dormant allele of Ube3a in neurons</span></div><div class="casAuthors">Huang, Hsien-Sung; Allen, John A.; Mabb, Angela M.; King, Ian F.; Miriyala, Jayalakshmi; Taylor-Blake, Bonnie; Sciaky, Noah; Dutton, J. Walter; Lee, Hyeong-Min; Chen, Xin; Jin, Jian; Bridges, Arlene S.; Zylka, Mark J.; Roth, Bryan L.; Philpot, Benjamin D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">481</span>
        (<span class="NLM_cas:issue">7380</span>),
    <span class="NLM_cas:pages">185-189</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Angelman syndrome is a severe neurodevelopmental disorder caused by deletion or mutation of the maternal allele of the ubiquitin protein ligase E3A (UBE3A).  In neurons, the paternal allele of UBE3A is intact but epigenetically silenced, raising the possibility that Angelman syndrome could be treated by activating this silenced allele to restore functional UBE3A protein.  Using an unbiased, high-content screen in primary cortical neurons from mice, we identify twelve topoisomerase I inhibitors and four topoisomerase II inhibitors that unsilence the paternal Ube3a allele.  These drugs included topotecan, irinotecan, etoposide and dexrazoxane (ICRF-187).  At nanomolar concns., topotecan upregulated catalytically active UBE3A in neurons from maternal Ube3a-null mice.  Topotecan concomitantly downregulated expression of the Ube3a antisense transcript that overlaps the paternal copy of Ube3a antisense transcript that overlaps the paternal copy of Ube3a.  These results indicate that topotecan unsilences Ube3a in cis by reducing transcription of an imprinted antisense RNA.  When administered in vivo, topotecan unsilenced the paternal Ube3a allele in several regions of the nervous system, including neurons in the hippocampus, neocortex, striatum, cerebellum and spinal cord.  Paternal expression of Ube3a remained elevated in a subset of spinal cord neurons for at least 12 wk after cessation of topotecan treatment, indicating that transient topoisomerase inhibition can have enduring effects on gene expression.  Although potential off-target effects remain to be investigated, these findings suggest a therapeutic strategy for reactivating the functional but dormant allele of Ube3a in patients with Angelman syndrome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyX6PY0uT7lLVg90H21EOLACvtfcHk0lgO2I-yRxQtuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1GjsL%252FM&md5=659adcf73a6c9f6b432dd34431ae1308</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnature10726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10726%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%2BS.%26aulast%3DAllen%26aufirst%3DJ.%2BA.%26aulast%3DMabb%26aufirst%3DA.%2BM.%26aulast%3DKing%26aufirst%3DI.%2BF.%26aulast%3DMiriyala%26aufirst%3DJ.%26aulast%3DTaylor-Blake%26aufirst%3DB.%26aulast%3DSciaky%26aufirst%3DN.%26aulast%3DDutton%26aufirst%3DJ.%2BW.%26aulast%3DLee%26aufirst%3DH.%2BM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DBridges%26aufirst%3DA.%2BS.%26aulast%3DZylka%26aufirst%3DM.%2BJ.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DPhilpot%26aufirst%3DB.%2BD.%26atitle%3DTopoisomerase%2520Inhibitors%2520Unsilence%2520the%2520Dormant%2520Allele%2520of%2520Ube3a%2520in%2520Neurons%26jtitle%3DNature%26date%3D2012%26volume%3D481%26spage%3D185%26epage%3D189%26doi%3D10.1038%2Fnature10726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqi, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menzies, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamatakou, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karabiyik, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ureshino, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricketts, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimenez-Sanchez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteban, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tortorella, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzakopian, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandes, H. J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karran, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubinsztein, D. C.</span></span> <span> </span><span class="NLM_article-title">Felodipine Induces Autophagy in Mouse Brains with Pharmacokinetics Amenable to Repurposing</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1817</span>, <span class="refDoi"> DOI: 10.1038/s41467-019-09494-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41467-019-09494-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=31000720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BB3M%252FmvF2kuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1817&author=F.+H.+Siddiqiauthor=F.+M.+Menziesauthor=A.+Lopezauthor=E.+Stamatakouauthor=C.+Karabiyikauthor=R.+Ureshinoauthor=T.+Rickettsauthor=M.+Jimenez-Sanchezauthor=M.+A.+Estebanauthor=L.+Laiauthor=M.+D.+Tortorellaauthor=Z.+Luoauthor=H.+Liuauthor=E.+Metzakopianauthor=H.+J.+R.+Fernandesauthor=A.+Bassettauthor=E.+Karranauthor=B.+L.+Millerauthor=A.+Flemingauthor=D.+C.+Rubinsztein&title=Felodipine+Induces+Autophagy+in+Mouse+Brains+with+Pharmacokinetics+Amenable+to+Repurposing&doi=10.1038%2Fs41467-019-09494-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Felodipine induces autophagy in mouse brains with pharmacokinetics amenable to repurposing</span></div><div class="casAuthors">Siddiqi Farah H; Menzies Fiona M; Lopez Ana; Stamatakou Eleanna; Karabiyik Cansu; Ureshino Rodrigo; Ricketts Thomas; Jimenez-Sanchez Maria; Fleming Angeleen; Rubinsztein David C; Siddiqi Farah H; Stamatakou Eleanna; Rubinsztein David C; Lopez Ana; Ureshino Rodrigo; Fleming Angeleen; Jimenez-Sanchez Maria; Esteban Miguel Angel; Lai Liangxue; Tortorella Micky D; Luo Zhiwei; Liu Hao; Metzakopian Emmanouil; Fernandes Hugo J R; Bassett Andrew; Karran Eric; Miller Bruce L</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1817</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Neurodegenerative diseases like Alzheimer's disease, Parkinson's disease and Huntington's disease manifest with the neuronal accumulation of toxic proteins.  Since autophagy upregulation enhances the clearance of such proteins and ameliorates their toxicities in animal models, we and others have sought to re-position/re-profile existing compounds used in humans to identify those that may induce autophagy in the brain.  A key challenge with this approach is to assess if any hits identified can induce neuronal autophagy at concentrations that would be seen in humans taking the drug for its conventional indication.  Here we report that felodipine, an L-type calcium channel blocker and anti-hypertensive drug, induces autophagy and clears diverse aggregate-prone, neurodegenerative disease-associated proteins.  Felodipine can clear mutant α-synuclein in mouse brains at plasma concentrations similar to those that would be seen in humans taking the drug.  This is associated with neuroprotection in mice, suggesting the promise of this compound for use in neurodegeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSPsrrt_ir8Uv6AExegM5LLfW6udTcc2ea7O1IUF3c6CLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M%252FmvF2kuw%253D%253D&md5=09a695108aebc0b2d27275acf143063e</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fs41467-019-09494-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-019-09494-2%26sid%3Dliteratum%253Aachs%26aulast%3DSiddiqi%26aufirst%3DF.%2BH.%26aulast%3DMenzies%26aufirst%3DF.%2BM.%26aulast%3DLopez%26aufirst%3DA.%26aulast%3DStamatakou%26aufirst%3DE.%26aulast%3DKarabiyik%26aufirst%3DC.%26aulast%3DUreshino%26aufirst%3DR.%26aulast%3DRicketts%26aufirst%3DT.%26aulast%3DJimenez-Sanchez%26aufirst%3DM.%26aulast%3DEsteban%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DL.%26aulast%3DTortorella%26aufirst%3DM.%2BD.%26aulast%3DLuo%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DMetzakopian%26aufirst%3DE.%26aulast%3DFernandes%26aufirst%3DH.%2BJ.%2BR.%26aulast%3DBassett%26aufirst%3DA.%26aulast%3DKarran%26aufirst%3DE.%26aulast%3DMiller%26aufirst%3DB.%2BL.%26aulast%3DFleming%26aufirst%3DA.%26aulast%3DRubinsztein%26aufirst%3DD.%2BC.%26atitle%3DFelodipine%2520Induces%2520Autophagy%2520in%2520Mouse%2520Brains%2520with%2520Pharmacokinetics%2520Amenable%2520to%2520Repurposing%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D1817%26doi%3D10.1038%2Fs41467-019-09494-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujimori, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otomo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atsuta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akiyama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobue, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okano, H.</span></span> <span> </span><span class="NLM_article-title">Modeling Sporadic Als in Ipsc-Derived Motor Neurons Identifies a Potential Therapeutic Agent</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1579</span>– <span class="NLM_lpage">1589</span>, <span class="refDoi"> DOI: 10.1038/s41591-018-0140-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41591-018-0140-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30127392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFGjur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=1579-1589&author=K.+Fujimoriauthor=M.+Ishikawaauthor=A.+Otomoauthor=N.+Atsutaauthor=R.+Nakamuraauthor=T.+Akiyamaauthor=S.+Hadanoauthor=M.+Aokiauthor=H.+Sayaauthor=G.+Sobueauthor=H.+Okano&title=Modeling+Sporadic+Als+in+Ipsc-Derived+Motor+Neurons+Identifies+a+Potential+Therapeutic+Agent&doi=10.1038%2Fs41591-018-0140-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling sporadic ALS in iPSC-derived motor neurons identifies a potential therapeutic agent</span></div><div class="casAuthors">Fujimori, Koki; Ishikawa, Mitsuru; Otomo, Asako; Atsuta, Naoki; Nakamura, Ryoichi; Akiyama, Tetsuya; Hadano, Shinji; Aoki, Masashi; Saya, Hideyuki; Sobue, Gen; Okano, Hideyuki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1579-1589</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Amyotrophic lateral sclerosis (ALS) is a heterogeneous motor neuron disease for which no effective treatment is available, despite decades of research into SOD1-mutant familial ALS (FALS).  The majority of ALS patients have no familial history, making the modeling of sporadic ALS (SALS) essential to the development of ALS therapeutics.  However, as mutations underlying ALS pathogenesis have not yet been identified, it remains difficult to establish useful models of SALS.  Using induced pluripotent stem cell (iPSC) technol. to generate stem and differentiated cells retaining the patients' full genetic information, we have established a large no. of in vitro cellular models of SALS.  These models showed phenotypic differences in their pattern of neuronal degeneration, types of abnormal protein aggregates, cell death mechanisms, and onset and progression of these phenotypes in vitro among cases.  We therefore developed a system for case clustering capable of subdividing these heterogeneous SALS models by their in vitro characteristics.  We further evaluated multiple-phenotype rescue of these subclassified SALS models using agents selected from non-SOD1 FALS models, and identified ropinirole as a potential therapeutic candidate.  Integration of the datasets acquired in this study permitted the visualization of mol. pathologies shared across a wide range of SALS models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZe3w-U7d4qbVg90H21EOLACvtfcHk0lgxnymwNZevPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFGjur3M&md5=e858757be57e2ad32448221abc6b5eb1</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fs41591-018-0140-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-018-0140-5%26sid%3Dliteratum%253Aachs%26aulast%3DFujimori%26aufirst%3DK.%26aulast%3DIshikawa%26aufirst%3DM.%26aulast%3DOtomo%26aufirst%3DA.%26aulast%3DAtsuta%26aufirst%3DN.%26aulast%3DNakamura%26aufirst%3DR.%26aulast%3DAkiyama%26aufirst%3DT.%26aulast%3DHadano%26aufirst%3DS.%26aulast%3DAoki%26aufirst%3DM.%26aulast%3DSaya%26aufirst%3DH.%26aulast%3DSobue%26aufirst%3DG.%26aulast%3DOkano%26aufirst%3DH.%26atitle%3DModeling%2520Sporadic%2520Als%2520in%2520Ipsc-Derived%2520Motor%2520Neurons%2520Identifies%2520a%2520Potential%2520Therapeutic%2520Agent%26jtitle%3DNat.%2520Med.%26date%3D2018%26volume%3D24%26spage%3D1579%26epage%3D1589%26doi%3D10.1038%2Fs41591-018-0140-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tracy, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sohn, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludwig, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer-Franke, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karch, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, L.</span></span> <span> </span><span class="NLM_article-title">Scalable Production of Ipsc-Derived Human Neurons to Identify Tau-Lowering Compounds by High-Content Screening</span>. <i>Stem Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1221</span>– <span class="NLM_lpage">1233</span>, <span class="refDoi"> DOI: 10.1016/j.stemcr.2017.08.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.stemcr.2017.08.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28966121" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOrtrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=1221-1233&author=C.+Wangauthor=M.+E.+Wardauthor=R.+Chenauthor=K.+Liuauthor=T.+E.+Tracyauthor=X.+Chenauthor=M.+Xieauthor=P.+D.+Sohnauthor=C.+Ludwigauthor=A.+Meyer-Frankeauthor=C.+M.+Karchauthor=S.+Dingauthor=L.+Gan&title=Scalable+Production+of+Ipsc-Derived+Human+Neurons+to+Identify+Tau-Lowering+Compounds+by+High-Content+Screening&doi=10.1016%2Fj.stemcr.2017.08.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Scalable Production of iPSC-Derived Human Neurons to Identify Tau-Lowering Compounds by High-Content Screening</span></div><div class="casAuthors">Wang, Chao; Ward, Michael E.; Chen, Robert; Liu, Kai; Tracy, Tara E.; Chen, Xu; Xie, Min; Sohn, Peter Dongmin; Ludwig, Connor; Meyer-Franke, Anke; Karch, Celeste M.; Ding, Sheng; Gan, Li</div><div class="citationInfo"><span class="NLM_cas:title">Stem Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1221-1233</span>CODEN:
                <span class="NLM_cas:coden">SCRTFL</span>;
        ISSN:<span class="NLM_cas:issn">2213-6711</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Lowering total tau levels is an attractive therapeutic strategy for Alzheimer's disease and other tauopathies.  High-throughput screening in neurons derived from human induced pluripotent stem cells (iPSCs) is a powerful tool to identify tau-targeted therapeutics.  However, such screens have been hampered by heterogeneous neuronal prodn., high cost and low yield, and multi-step differentiation procedures.  We engineered an isogenic iPSC line that harbors an inducible neurogenin 2 transgene, a transcription factor that rapidly converts iPSCs to neurons, integrated at the AAVS1 locus.  Using a simplified two-step protocol, we differentiated these iPSCs into cortical glutamatergic neurons with minimal well-to-well variability.  We developed a robust high-content screening assay to identify tau-lowering compds. in LOPAC and identified adrenergic receptors agonists as a class of compds. that reduce endogenous human tau.  These techniques enable the use of human neurons for high-throughput screening of drugs to treat neurodegenerative disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhY3PvM_2Qf7Vg90H21EOLACvtfcHk0lgxnymwNZevPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOrtrnJ&md5=194fb07160abc9b2d1f798328385839c</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.stemcr.2017.08.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.stemcr.2017.08.019%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWard%26aufirst%3DM.%2BE.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DK.%26aulast%3DTracy%26aufirst%3DT.%2BE.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DXie%26aufirst%3DM.%26aulast%3DSohn%26aufirst%3DP.%2BD.%26aulast%3DLudwig%26aufirst%3DC.%26aulast%3DMeyer-Franke%26aufirst%3DA.%26aulast%3DKarch%26aufirst%3DC.%2BM.%26aulast%3DDing%26aufirst%3DS.%26aulast%3DGan%26aufirst%3DL.%26atitle%3DScalable%2520Production%2520of%2520Ipsc-Derived%2520Human%2520Neurons%2520to%2520Identify%2520Tau-Lowering%2520Compounds%2520by%2520High-Content%2520Screening%26jtitle%3DStem%2520Cell%2520Rep.%26date%3D2017%26volume%3D9%26spage%3D1221%26epage%3D1233%26doi%3D10.1016%2Fj.stemcr.2017.08.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kondo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funayama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyake, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woltjen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawakatsu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izumi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaji, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, H.</span></span> <span> </span><span class="NLM_article-title">Ipsc-Based Compound Screening and in Vitro Trials Identify a Synergistic Anti-Amyloid Beta Combination for Alzheimer’s Disease</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2304</span>– <span class="NLM_lpage">2312</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.10.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.celrep.2017.10.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=29166618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOrs7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=2304-2312&author=T.+Kondoauthor=K.+Imamuraauthor=M.+Funayamaauthor=K.+Tsukitaauthor=M.+Miyakeauthor=A.+Ohtaauthor=K.+Woltjenauthor=M.+Nakagawaauthor=T.+Asadaauthor=T.+Araiauthor=S.+Kawakatsuauthor=Y.+Izumiauthor=R.+Kajiauthor=N.+Iwataauthor=H.+Inoue&title=Ipsc-Based+Compound+Screening+and+in+Vitro+Trials+Identify+a+Synergistic+Anti-Amyloid+Beta+Combination+for+Alzheimer%E2%80%99s+Disease&doi=10.1016%2Fj.celrep.2017.10.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">iPSC-Based Compound Screening and In Vitro Trials Identify a Synergistic Anti-amyloid β Combination for Alzheimer's Disease</span></div><div class="casAuthors">Kondo, Takayuki; Imamura, Keiko; Funayama, Misato; Tsukita, Kayoko; Miyake, Michiyo; Ohta, Akira; Woltjen, Knut; Nakagawa, Masato; Asada, Takashi; Arai, Tetsuaki; Kawakatsu, Shinobu; Izumi, Yuishin; Kaji, Ryuji; Iwata, Nobuhisa; Inoue, Haruhisa</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2304-2312</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">In the process of drug development, in vitro studies do not always adequately predict human-specific drug responsiveness in clin. trials.  Here, we applied the advantage of human iPSC-derived neurons, which offer human-specific drug responsiveness, to screen and evaluate therapeutic candidates for Alzheimer's disease (AD).  Using AD patient neurons with nearly 100% purity from iPSCs, we established a robust and reproducible assay for amyloid β peptide (Aβ), a pathogenic mol. in AD, and screened a pharmaceutical compd. library.  We acquired 27 Aβ-lowering screen hits, prioritized hits by chem. structure-based clustering, and selected 6 leading compds.  Next, to maximize the anti-Aβ effect, we selected a synergistic combination of bromocriptine, cromolyn, and topiramate as an anti-Aβ cocktail.  Finally, using neurons from familial and sporadic AD patients, we found that the cocktail showed a significant and potent anti-Aβ effect on patient cells.  This human iPSC-based platform promises to be useful for AD drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprnljAp1qpz7Vg90H21EOLACvtfcHk0li7-ssGsTGS8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOrs7fN&md5=62a79bcd33bdcb3e529e7890fa04f18a</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.10.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.10.109%26sid%3Dliteratum%253Aachs%26aulast%3DKondo%26aufirst%3DT.%26aulast%3DImamura%26aufirst%3DK.%26aulast%3DFunayama%26aufirst%3DM.%26aulast%3DTsukita%26aufirst%3DK.%26aulast%3DMiyake%26aufirst%3DM.%26aulast%3DOhta%26aufirst%3DA.%26aulast%3DWoltjen%26aufirst%3DK.%26aulast%3DNakagawa%26aufirst%3DM.%26aulast%3DAsada%26aufirst%3DT.%26aulast%3DArai%26aufirst%3DT.%26aulast%3DKawakatsu%26aufirst%3DS.%26aulast%3DIzumi%26aufirst%3DY.%26aulast%3DKaji%26aufirst%3DR.%26aulast%3DIwata%26aufirst%3DN.%26aulast%3DInoue%26aufirst%3DH.%26atitle%3DIpsc-Based%2520Compound%2520Screening%2520and%2520in%2520Vitro%2520Trials%2520Identify%2520a%2520Synergistic%2520Anti-Amyloid%2520Beta%2520Combination%2520for%2520Alzheimer%25E2%2580%2599s%2520Disease%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D21%26spage%3D2304%26epage%3D2312%26doi%3D10.1016%2Fj.celrep.2017.10.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uzel, S. G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamm, R. D.</span></span> <span> </span><span class="NLM_article-title">Microphysiological 3d Model of Amyotrophic Lateral Sclerosis (Als) from Human Ips-Derived Muscle Cells and Optogenetic Motor Neurons</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">eaat5847</span> <span class="refDoi"> DOI: 10.1126/sciadv.aat5847</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fsciadv.aat5847" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30324134" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&author=T.+Osakiauthor=S.+G.+M.+Uzelauthor=R.+D.+Kamm&title=Microphysiological+3d+Model+of+Amyotrophic+Lateral+Sclerosis+%28Als%29+from+Human+Ips-Derived+Muscle+Cells+and+Optogenetic+Motor+Neurons&doi=10.1126%2Fsciadv.aat5847"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.aat5847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.aat5847%26sid%3Dliteratum%253Aachs%26aulast%3DOsaki%26aufirst%3DT.%26aulast%3DUzel%26aufirst%3DS.%2BG.%2BM.%26aulast%3DKamm%26aufirst%3DR.%2BD.%26atitle%3DMicrophysiological%25203d%2520Model%2520of%2520Amyotrophic%2520Lateral%2520Sclerosis%2520%2528Als%2529%2520from%2520Human%2520Ips-Derived%2520Muscle%2520Cells%2520and%2520Optogenetic%2520Motor%2520Neurons%26jtitle%3DSci.%2520Adv.%26date%3D2018%26volume%3D4%26doi%3D10.1126%2Fsciadv.aat5847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hor, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soh, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, V. J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santosa, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winanto</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, B. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soh, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, S. Y.</span></span> <span> </span><span class="NLM_article-title">Cell Cycle Inhibitors Protect Motor Neurons in an Organoid Model of Spinal Muscular Atrophy</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1100</span>, <span class="refDoi"> DOI: 10.1038/s41419-018-1081-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41419-018-1081-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30368521" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BB3cvktVGitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1100&author=J.+H.+Horauthor=E.+S.+Sohauthor=L.+Y.+Tanauthor=V.+J.+W.+Limauthor=M.+M.+Santosaauthor=+Winantoauthor=B.+X.+Hoauthor=Y.+Fanauthor=B.+S.+Sohauthor=S.+Y.+Ng&title=Cell+Cycle+Inhibitors+Protect+Motor+Neurons+in+an+Organoid+Model+of+Spinal+Muscular+Atrophy&doi=10.1038%2Fs41419-018-1081-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Cell cycle inhibitors protect motor neurons in an organoid model of Spinal Muscular Atrophy</span></div><div class="casAuthors">Hor Jin Hui; Soh Eunice Shi-Yi; Tan Li Yi; Lim Valerie Jing Wen; Santosa Munirah Mohamad; Winanto; Ho Beatrice Xuan; Soh Boon-Seng; Ng Shi-Yan; Hor Jin Hui; Ho Beatrice Xuan; Soh Boon-Seng; Soh Eunice Shi-Yi; Winanto; Fan Yong; Soh Boon-Seng; Ng Shi-Yan; Ng Shi-Yan; Ng Shi-Yan</div><div class="citationInfo"><span class="NLM_cas:title">Cell death & disease</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1100</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Spinal Muscular Atrophy (SMA) is caused by genetic mutations in the SMN1 gene, resulting in drastically reduced levels of Survival of Motor Neuron (SMN) protein.  Although SMN is ubiquitously expressed, spinal motor neurons are one of the most affected cell types.  Previous studies have identified pathways uniquely activated in SMA motor neurons, including a hyperactivated ER stress pathway, neuronal hyperexcitability, and defective spliceosomes.  To investigate why motor neurons are more affected than other neural types, we developed a spinal organoid model of SMA.  We demonstrate overt motor neuron degeneration in SMA spinal organoids, and this degeneration can be prevented using a small molecule inhibitor of CDK4/6, indicating that spinal organoids are an ideal platform for therapeutic discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT8cW2iZzjIi1w2ofyr07jXfW6udTcc2eYM72TNEkAOErntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cvktVGitQ%253D%253D&md5=d36b00f8b6d2f678179f6d5a6c5e7c7a</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1038%2Fs41419-018-1081-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41419-018-1081-0%26sid%3Dliteratum%253Aachs%26aulast%3DHor%26aufirst%3DJ.%2BH.%26aulast%3DSoh%26aufirst%3DE.%2BS.%26aulast%3DTan%26aufirst%3DL.%2BY.%26aulast%3DLim%26aufirst%3DV.%2BJ.%2BW.%26aulast%3DSantosa%26aufirst%3DM.%2BM.%26aulast%3DWinanto%26aulast%3DHo%26aufirst%3DB.%2BX.%26aulast%3DFan%26aufirst%3DY.%26aulast%3DSoh%26aufirst%3DB.%2BS.%26aulast%3DNg%26aufirst%3DS.%2BY.%26atitle%3DCell%2520Cycle%2520Inhibitors%2520Protect%2520Motor%2520Neurons%2520in%2520an%2520Organoid%2520Model%2520of%2520Spinal%2520Muscular%2520Atrophy%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2018%26volume%3D9%26spage%3D1100%26doi%3D10.1038%2Fs41419-018-1081-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsvetkov, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrasate, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pleiss, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span> <span> </span><span class="NLM_article-title">A Small-Molecule Scaffold Induces Autophagy in Primary Neurons and Protects against Toxicity in a Huntington Disease Model</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">16982</span>– <span class="NLM_lpage">16987</span>, <span class="refDoi"> DOI: 10.1073/pnas.1004498107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1073%2Fpnas.1004498107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=20833817" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Oht7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=16982-16987&author=A.+S.+Tsvetkovauthor=J.+Millerauthor=M.+Arrasateauthor=J.+S.+Wongauthor=M.+A.+Pleissauthor=S.+Finkbeiner&title=A+Small-Molecule+Scaffold+Induces+Autophagy+in+Primary+Neurons+and+Protects+against+Toxicity+in+a+Huntington+Disease+Model&doi=10.1073%2Fpnas.1004498107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">A small-molecule scaffold induces autophagy in primary neurons and protects against toxicity in a Huntington disease model</span></div><div class="casAuthors">Tsvetkov, Andrey S.; Miller, Jason; Arrasate, Montserrat; Wong, Jinny S.; Pleiss, Michael A.; Finkbeiner, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">16982-16987, S16982/1-S16982/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Autophagy is an intracellular turnover pathway.  It has special relevance for neurodegenerative proteinopathies, such as Alzheimer disease, Parkinson disease, and Huntington disease (HD), which are characterized by the accumulation of misfolded proteins.  Although induction of autophagy enhances clearance of misfolded protein and has therefore been suggested as a therapy for proteinopathies, neurons appear to be less responsive to classic autophagy inducers than nonneuronal cells.  Searching for improved inducers of neuronal autophagy, we discovered an N10-substituted phenoxazine that, at proper doses, potently and safely up-regulated autophagy in neurons in an Akt- and mTOR-independent fashion.  In a neuron model of HD, this compd. was neuroprotective and decreased the accumulation of diffuse and aggregated misfolded protein.  A structure/activity anal. with structurally similar compds. approved by the US Food and Drug Administration revealed a defined pharmacophore for inducing neuronal autophagy.  This pharmacophore should prove useful in studying autophagy in neurons and in developing therapies for neurodegenerative proteinopathies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtnUfvW65_mrVg90H21EOLACvtfcHk0li7-ssGsTGS8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Oht7fP&md5=aaefe7b3fc1c0e12c53b67c660219767</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1004498107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1004498107%26sid%3Dliteratum%253Aachs%26aulast%3DTsvetkov%26aufirst%3DA.%2BS.%26aulast%3DMiller%26aufirst%3DJ.%26aulast%3DArrasate%26aufirst%3DM.%26aulast%3DWong%26aufirst%3DJ.%2BS.%26aulast%3DPleiss%26aufirst%3DM.%2BA.%26aulast%3DFinkbeiner%26aufirst%3DS.%26atitle%3DA%2520Small-Molecule%2520Scaffold%2520Induces%2520Autophagy%2520in%2520Primary%2520Neurons%2520and%2520Protects%2520against%2520Toxicity%2520in%2520a%2520Huntington%2520Disease%2520Model%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D16982%26epage%3D16987%26doi%3D10.1073%2Fpnas.1004498107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theodoropoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKnight, S. L.</span></span> <span> </span><span class="NLM_article-title">Poly-Dipeptides Encoded by the C9orf72 Repeats Bind Nucleoli, Impede Rna Biogenesis, and Kill Cells</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>345</i></span>,  <span class="NLM_fpage">1139</span>– <span class="NLM_lpage">1145</span>, <span class="refDoi"> DOI: 10.1126/science.1254917</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fscience.1254917" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25081482" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=345&publication_year=2014&pages=1139-1145&author=I.+Kwonauthor=S.+Xiangauthor=M.+Katoauthor=L.+Wuauthor=P.+Theodoropoulosauthor=T.+Wangauthor=J.+Kimauthor=J.+Yunauthor=Y.+Xieauthor=S.+L.+McKnight&title=Poly-Dipeptides+Encoded+by+the+C9orf72+Repeats+Bind+Nucleoli%2C+Impede+Rna+Biogenesis%2C+and+Kill+Cells&doi=10.1126%2Fscience.1254917"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Poly-dipeptides encoded by the C9orf72 repeats bind nucleoli, impede RNA biogenesis, and kill cells</span></div><div class="casAuthors">Kwon, Ilmin; Xiang, Siheng; Kato, Masato; Wu, Leeju; Theodoropoulos, Pano; Wang, Tao; Kim, Jiwoong; Yun, Jonghyun; Xie, Yang; McKnight, Steven L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">345</span>
        (<span class="NLM_cas:issue">6201</span>),
    <span class="NLM_cas:pages">1139-1145</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Many RNA regulatory proteins controlling pre-mRNA splicing contain serine:arginine (SR) repeats.  Here, we found that these SR domains bound hydrogel droplets composed of fibrous polymers of the low-complexity domain of heterogeneous ribonucleoprotein A2 (hnRNPA2).  Hydrogel binding was reversed upon phosphorylation of the SR domain by CDC2-like kinases 1 and 2 (CLK1/2).  Mutated variants of the SR domains changing serine to glycine (SR-to-GR variants) also bound to hnRNPA2 hydrogels but were not affected by CLK1/2.  When expressed in mammalian cells, these variants bound nucleoli.  The translation products of the sense and antisense transcripts of the expansion repeats assocd. with the C9orf72 gene altered in neurodegenerative disease encode GRn and PRn repeat polypeptides.  Both peptides bound to hnRNPA2 hydrogels independent of CLK1/2 activity.  When applied to cultured cells, both peptides entered cells, migrated to the nucleus, bound nucleoli, and poisoned RNA biogenesis, which caused cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUOC94n_nes7Vg90H21EOLACvtfcHk0ljFdCmf4eo2oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtbzM&md5=f7bc9754f14face0bffa45434eeb5025</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1126%2Fscience.1254917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1254917%26sid%3Dliteratum%253Aachs%26aulast%3DKwon%26aufirst%3DI.%26aulast%3DXiang%26aufirst%3DS.%26aulast%3DKato%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DTheodoropoulos%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DYun%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DMcKnight%26aufirst%3DS.%2BL.%26atitle%3DPoly-Dipeptides%2520Encoded%2520by%2520the%2520C9orf72%2520Repeats%2520Bind%2520Nucleoli%252C%2520Impede%2520Rna%2520Biogenesis%252C%2520and%2520Kill%2520Cells%26jtitle%3DScience%26date%3D2014%26volume%3D345%26spage%3D1139%26epage%3D1145%26doi%3D10.1126%2Fscience.1254917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span> <span> </span><span class="NLM_article-title">Bet Inhibitors Rvx-208 and Pfi-1 Reactivate Hiv-1 from Latency</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">16646</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-16816-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41598-017-16816-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=29192216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC1M3ms1Gjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=16646&author=P.+Luauthor=Y.+Shenauthor=H.+Yangauthor=Y.+Wangauthor=Z.+Jiangauthor=X.+Yangauthor=Y.+Zhongauthor=H.+Panauthor=J.+Xuauthor=H.+Luauthor=H.+Zhu&title=Bet+Inhibitors+Rvx-208+and+Pfi-1+Reactivate+Hiv-1+from+Latency&doi=10.1038%2Fs41598-017-16816-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitors RVX-208 and PFI-1 reactivate HIV-1 from latency</span></div><div class="casAuthors">Lu Panpan; Yang He; Wang Yanan; Jiang Zhengtao; Yang Xinyi; Zhong Yangcheng; Pan Hanyu; Zhu Huanzhang; Shen Yinzhong; Xu Jianqing; Lu Hongzhou</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">16646</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Persistent latent reservoir in resting CD4+ T cells is a major obstacle in curing HIV-1 infection.  Effective strategies for eradication of the HIV-1 reservoir are urgently needed.  We report here for the first time that two BET inhibitors, RVX-208, which has entered phase II clinical trials for diverse cardiovascular disorders, and PFI-1, which has been widely studied in oncology, can reactivate HIV-1 from latency.  RVX-208 and PFI-1 treatment alone or in combination with other latency reversing agents efficiently reactivated HIV-1 transcription through an up-regulation of P-TEFb by increasing CDK9 Thr-186 phosphorylation in latently infected Jurkat T cells in vitro.  The two BET inhibitors also reactivated HIV-1 transcription in cART treated patient-derived resting CD4+ T cells ex vivo, without influence on global immune cell activation.  Our findings, in combination with previous reports, further confirm that BET inhibitors are a group of leading compounds for combating HIV-1 latency for viral eradication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTMmRurTV1xdu3MPKNEWB5RfW6udTcc2eaTflUelREa7rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3ms1Gjtw%253D%253D&md5=51309d6a5fefc61001d79cbb93b727ee</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-16816-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-16816-1%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DP.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZhong%26aufirst%3DY.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DH.%26atitle%3DBet%2520Inhibitors%2520Rvx-208%2520and%2520Pfi-1%2520Reactivate%2520Hiv-1%2520from%2520Latency%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D16646%26doi%3D10.1038%2Fs41598-017-16816-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, R. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span> <span> </span><span class="NLM_article-title">An Epigenetic Compound Library Screen Identifies Bet Inhibitors That Promote Hsv-1 and −2 Replication by Bridging P-Tefb to Viral Gene Promoters through Brd4</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e1005950</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1005950</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1371%2Fjournal.ppat.1005950" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=27764245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFCntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&author=K.+Renauthor=W.+Zhangauthor=X.+Chenauthor=Y.+Maauthor=Y.+Daiauthor=Y.+Fanauthor=Y.+Houauthor=R.+X.+Tanauthor=E.+Li&title=An+Epigenetic+Compound+Library+Screen+Identifies+Bet+Inhibitors+That+Promote+Hsv-1+and+%E2%88%922+Replication+by+Bridging+P-Tefb+to+Viral+Gene+Promoters+through+Brd4&doi=10.1371%2Fjournal.ppat.1005950"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">An epigenetic compound library screen identifies BET inhibitors that promote HSV-1 and -2 replication by bridging P-TEFb to viral gene promoters through BRD4</span></div><div class="casAuthors">Ren, Ke; Zhang, Wei; Chen, Xiaoqing; Ma, Yingyu; Dai, Yue; Fan, Yimei; Hou, Yayi; Tan, Ren Xiang; Li, Erguang</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1005950/1-e1005950/23</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The human HSV-1 and -2 are common pathogens of human diseases.  Both host and viral factors are involved in HSV lytic infection, although detailed mechanisms remain elusive.  By screening a chem. library of epigenetic regulation, we identified bromodomain-contg. protein 4 (BRD4) as a crit. player in HSV infection.  We show that treatment with pan BD domain inhibitor enhanced both HSV infection.  Using JQ1 as a probe, we found that JQ1, a defined BD1 inhibitor, acts through BRD4 protein since knockdown of BRD4 expression ablated JQ1 effect on HSV infection.  BRD4 regulates HSV replication through complex formation involving CDK9 and RNAP II; whereas, JQ1 promotes HSV-1 infection by allocating the complex to HSV gene promoters.  Therefore, suppression of BRD4 expression or inhibition of CDK9 activity impeded HSV infection.  Our data support a model that JQ1 enhances HSV infection by switching BRD4 to transcription regulation of viral gene expression from chromatin targeting since transient expression of BRD4 BD1 or BD1/2 domain had similar effect to that by JQ1 treatment.  In addn. to the identification that BRD4 is a modulator for JQ1 action on HSV infection, this study demonstrates BRD4 has an essential role in HSV infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAdy32pDZ3HrVg90H21EOLACvtfcHk0ljMqOVQAUSPLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFCntbk%253D&md5=a2a78d63cb07da2680752067b4e8b5ae</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1005950&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1005950%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DDai%26aufirst%3DY.%26aulast%3DFan%26aufirst%3DY.%26aulast%3DHou%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DR.%2BX.%26aulast%3DLi%26aufirst%3DE.%26atitle%3DAn%2520Epigenetic%2520Compound%2520Library%2520Screen%2520Identifies%2520Bet%2520Inhibitors%2520That%2520Promote%2520Hsv-1%2520and%2520%25E2%2588%25922%2520Replication%2520by%2520Bridging%2520P-Tefb%2520to%2520Viral%2520Gene%2520Promoters%2520through%2520Brd4%26jtitle%3DPLoS%2520Pathog.%26date%3D2016%26volume%3D12%26doi%3D10.1371%2Fjournal.ppat.1005950" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, W.</span></span> <span> </span><span class="NLM_article-title">Bromodomain and Extra-Terminal (Bet) Protein Inhibitors Suppress Chondrocyte Differentiation and Restrain Bone Growth</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">26647</span>– <span class="NLM_lpage">26657</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.749697</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1074%2Fjbc.M116.749697" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=27821592" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFCmurnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=26647-26657&author=N.+Niuauthor=R.+Shaoauthor=G.+Yanauthor=W.+Zou&title=Bromodomain+and+Extra-Terminal+%28Bet%29+Protein+Inhibitors+Suppress+Chondrocyte+Differentiation+and+Restrain+Bone+Growth&doi=10.1074%2Fjbc.M116.749697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain and Extra-terminal (BET) Protein Inhibitors Suppress Chondrocyte Differentiation and Restrain Bone Growth</span></div><div class="casAuthors">Niu, Ningning; Shao, Rui; Yan, Guang; Zou, Weiguo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">26647-26657</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Small mol. inhibitors for bromodomain and extra-terminal (BET) proteins have recently emerged as potential therapeutic agents in clin. trials for various cancers.  However, to date, it is unknown whether these inhibitors have side effects on bone structures.  Here, we report that inhibition of BET bromodomain proteins may suppress chondrocyte differentiation and restrain bone growth.  We generated a luciferase reporter system using the chondrogenic cell line ATDC5 in which the luciferase gene was driven by the promoter of Col2a1, an elementary collagen of the chondrocyte.  The Col2a1-luciferase ATDC5 system was used for rapidly screening both activators and repressors of human collagen Col2a1 gene expression, and we found that BET bromodomain inhibitors reduce the Col2a1-luciferase.  Consistent with the luciferase assay, BET inhibitors decrease the expression of Col2a1.  Furthermore, we constructed a zebrafish line in which the enhanced green fluorescent protein (EGFP) expression was driven by col2a1 promoter.  The transgenic (col2a1-EGFP) zebrafish line demonstrated that BET inhibitors I-BET151 and (+)-JQ1 may affect EGFP expression in zebrafish.  Furthermore, we found that I-BET151 and (+)-JQ1 may affect chondrocyte differentiation in vitro and inhibit zebrafish growth in vivo.  Mechanistic anal. revealed that BET inhibitors influenced the depletion of RNA polymerase II from the Col2a1 promoter.  Collectively, these results suggest that BET bromodomain inhibition may have side effects on skeletal bone structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr97hnBq7K0OrVg90H21EOLACvtfcHk0ljMqOVQAUSPLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFCmurnF&md5=d0518ae2f1e7d71ca1e87148848f8707</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.749697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.749697%26sid%3Dliteratum%253Aachs%26aulast%3DNiu%26aufirst%3DN.%26aulast%3DShao%26aufirst%3DR.%26aulast%3DYan%26aufirst%3DG.%26aulast%3DZou%26aufirst%3DW.%26atitle%3DBromodomain%2520and%2520Extra-Terminal%2520%2528Bet%2529%2520Protein%2520Inhibitors%2520Suppress%2520Chondrocyte%2520Differentiation%2520and%2520Restrain%2520Bone%2520Growth%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D26647%26epage%3D26657%26doi%3D10.1074%2Fjbc.M116.749697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hytti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokarz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maatta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piippo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korhonen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suuronen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honkakoski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaarniranta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahtela-Kakkonen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauppinen, A.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Bet Bromodomains Alleviates Inflammation in Human Rpe Cells</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>110–111</i></span>,  <span class="NLM_fpage">71</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2016.04.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.bcp.2016.04.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=27106081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC28XmsVaqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110%E2%80%93111&publication_year=2016&pages=71-79&author=M.+Hyttiauthor=P.+Tokarzauthor=E.+Maattaauthor=N.+Piippoauthor=E.+Korhonenauthor=T.+Suuronenauthor=P.+Honkakoskiauthor=K.+Kaarnirantaauthor=M.+Lahtela-Kakkonenauthor=A.+Kauppinen&title=Inhibition+of+Bet+Bromodomains+Alleviates+Inflammation+in+Human+Rpe+Cells&doi=10.1016%2Fj.bcp.2016.04.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET bromodomains alleviates inflammation in human RPE cells</span></div><div class="casAuthors">Hytti, M.; Tokarz, P.; Maatta, E.; Piippo, N.; Korhonen, E.; Suuronen, T.; Honkakoski, P.; Kaarniranta, K.; Lahtela-Kakkonen, M.; Kauppinen, A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">110-111</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">71-79</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins are vital for controlling the expression of many pro-inflammatory genes.  Consequently, compds. capable of inhibiting specific bromodomain-facilitated protein-protein interactions would be predicted to alleviate inflammation, making them valuable agents in the treatment of diseases caused by dysregulated inflammation, such as age-related macular degeneration.  Here, the authors assessed the ability of known inhibitors JQ-1, PFI-1, and IBET-151 to protect from the inflammation and cell death caused by etoposide exposure in the human retinal pigment epithelial cell line, ARPE-19.  The potential anti-inflammatory effects of the bromodomain inhibitors were assessed by ELISA profiling.  The involvement of NF-κB and SIRT1 in inflammatory signaling was monitored by ELISA and western blotting.  Furthermore, SIRT1 was knocked down using a specific siRNA or inhibited by EX-527 to elucidate its role in the inflammatory reaction.  The bromodomain inhibitors effectively decreased etoposide-induced release of IL-6 and IL-8.  This anti-inflammatory effect was not related to SIRT1 activity, although all bromodomain inhibitors decreased the extent of acetylation of p53 at the SIRT1 deacetylation site.  Overall, since bromodomain inhibitors display anti-inflammatory properties in human retinal pigment epithelial cells, these compds. may represent a new way of alleviating the inflammation underlying the onset of age-related macular degeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocwtuJwbNv5rVg90H21EOLACvtfcHk0ljMqOVQAUSPLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmsVaqsbk%253D&md5=689201c89b6400fe064d3e20ac719bee</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2016.04.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2016.04.009%26sid%3Dliteratum%253Aachs%26aulast%3DHytti%26aufirst%3DM.%26aulast%3DTokarz%26aufirst%3DP.%26aulast%3DMaatta%26aufirst%3DE.%26aulast%3DPiippo%26aufirst%3DN.%26aulast%3DKorhonen%26aufirst%3DE.%26aulast%3DSuuronen%26aufirst%3DT.%26aulast%3DHonkakoski%26aufirst%3DP.%26aulast%3DKaarniranta%26aufirst%3DK.%26aulast%3DLahtela-Kakkonen%26aufirst%3DM.%26aulast%3DKauppinen%26aufirst%3DA.%26atitle%3DInhibition%2520of%2520Bet%2520Bromodomains%2520Alleviates%2520Inflammation%2520in%2520Human%2520Rpe%2520Cells%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2016%26volume%3D110%25E2%2580%2593111%26spage%3D71%26epage%3D79%26doi%3D10.1016%2Fj.bcp.2016.04.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sidrauski, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kampmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hearn, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedantham, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaishankar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sokabe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendez, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newton, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verschueren, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krogan, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renslo, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, P.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Dimerization and Activation of the Exchange Factor Eif2b Antagonizes the Integrated Stress Response</span>. <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">e07314</span> <span class="refDoi"> DOI: 10.7554/eLife.07314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.7554%2FeLife.07314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25875391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC28XltVylsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2015&author=C.+Sidrauskiauthor=J.+C.+Tsaiauthor=M.+Kampmannauthor=B.+R.+Hearnauthor=P.+Vedanthamauthor=P.+Jaishankarauthor=M.+Sokabeauthor=A.+S.+Mendezauthor=B.+W.+Newtonauthor=E.+L.+Tangauthor=E.+Verschuerenauthor=J.+R.+Johnsonauthor=N.+J.+Kroganauthor=C.+S.+Fraserauthor=J.+S.+Weissmanauthor=A.+R.+Rensloauthor=P.+Walter&title=Pharmacological+Dimerization+and+Activation+of+the+Exchange+Factor+Eif2b+Antagonizes+the+Integrated+Stress+Response&doi=10.7554%2FeLife.07314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological dimerization and activation of the exchange factor eIF2B antagonizes the integrated stress response</span></div><div class="casAuthors">Sidrauski, Carmela; Tsai, Jordan C.; Kampmann, Martin; Hearn, Brian R.; Vedantham, Punitha; Jaishankar, Priyadarshini; Sokabe, Masaaki; Mendez, Aaron S.; Newton, Billy W.; Tang, Edward L.; Verschueren, Erik; Johnson, Jeffrey R.; Krogan, Nevan J.; Fraser, Christopher S.; Weissman, Jonathan S.; Renslo, Adam R.; Walter, Peter</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e07314/1-e07314/27</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">The general translation initiation factor eIF2 is a major translational control point.  Multiple signaling pathways in the integrated stress response phosphorylate eIF2 serine-51, inhibiting nucleotide exchange by eIF2B.  ISRIB, a potent drug-like small mol., renders cells insensitive to eIF2α phosphorylation and enhances cognitive function in rodents by blocking long-term depression.  ISRIB was identified in a phenotypic cell-based screen, and its mechanism of action remained unknown.  We now report that ISRIB is an activator of eIF2B.  Our reporter-based shRNA screen revealed an eIF2B requirement for ISRIB activity.  Our results define ISRIB as a sym. mol., show ISRIB-mediated stabilization of activated eIF2B dimers, and suggest that eIF2B4 (δ-subunit) contributes to the ISRIB binding site.  We also developed new ISRIB analogs, improving its EC50 to 600 pM in cell culture.  By modulating eIF2B function, ISRIB promises to be an invaluable tool in proof-of-principle studies aiming to ameliorate cognitive defects resulting from neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrw4eI3eO6AbVg90H21EOLACvtfcHk0lheIgnzBpGPxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltVylsLY%253D&md5=35563489035c0a80eb4e1e8046e7c28a</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.7554%2FeLife.07314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.07314%26sid%3Dliteratum%253Aachs%26aulast%3DSidrauski%26aufirst%3DC.%26aulast%3DTsai%26aufirst%3DJ.%2BC.%26aulast%3DKampmann%26aufirst%3DM.%26aulast%3DHearn%26aufirst%3DB.%2BR.%26aulast%3DVedantham%26aufirst%3DP.%26aulast%3DJaishankar%26aufirst%3DP.%26aulast%3DSokabe%26aufirst%3DM.%26aulast%3DMendez%26aufirst%3DA.%2BS.%26aulast%3DNewton%26aufirst%3DB.%2BW.%26aulast%3DTang%26aufirst%3DE.%2BL.%26aulast%3DVerschueren%26aufirst%3DE.%26aulast%3DJohnson%26aufirst%3DJ.%2BR.%26aulast%3DKrogan%26aufirst%3DN.%2BJ.%26aulast%3DFraser%26aufirst%3DC.%2BS.%26aulast%3DWeissman%26aufirst%3DJ.%2BS.%26aulast%3DRenslo%26aufirst%3DA.%2BR.%26aulast%3DWalter%26aufirst%3DP.%26atitle%3DPharmacological%2520Dimerization%2520and%2520Activation%2520of%2520the%2520Exchange%2520Factor%2520Eif2b%2520Antagonizes%2520the%2520Integrated%2520Stress%2520Response%26jtitle%3DElife%26date%3D2015%26volume%3D4%26doi%3D10.7554%2FeLife.07314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeBon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sidrauski, C.</span></span> <span> </span><span class="NLM_article-title">The Small Molecule Isrib Rescues the Stability and Activity of Vanishing White Matter Disease Eif2b Mutant Complexes</span>. <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e32733</span>, <span class="refDoi"> DOI: 10.7554/eLife.32733</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.7554%2FeLife.32733" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=29489452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVSjtLfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=e32733&author=Y.+L.+Wongauthor=L.+LeBonauthor=R.+Edaljiauthor=H.+B.+Limauthor=C.+Sunauthor=C.+Sidrauski&title=The+Small+Molecule+Isrib+Rescues+the+Stability+and+Activity+of+Vanishing+White+Matter+Disease+Eif2b+Mutant+Complexes&doi=10.7554%2FeLife.32733"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">The small molecule ISRIB rescues the stability and activity of vanishing white matter disease eIF2B mutant complexes</span></div><div class="casAuthors">Wong, Yao Liang; LeBon, Lauren; Edalji, Rohinton; Ben Lim, Hock; Sun, Chaohong; Sidrauski, Carmela</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e32733/1-e32733/23</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">EIF2B is a dedicated guanine nucleotide exchange factor for eIF2, the GTPase that is essential to initiate mRNA translation.  The integrated stress response (ISR) signaling pathway inhibits eIF2B activity, attenuates global protein synthesis and upregulates a set of stress-response proteins.  Partial loss-of-function mutations in eIF2B cause a neurodegenerative disorder called Vanishing White Matter Disease (VWMD).  Previously, we showed that the small mol. ISRIB is a specific activator of eIF2B.  Here, we report that various VWMD mutations destabilize the decameric eIF2B holoenzyme and impair its enzymic activity.  ISRIB stabilizes VWMD mutant eIF2B in the decameric form and restores the residual catalytic activity to wild-type levels.  Moreover, ISRIB blocks activation of the ISR in cells carrying these mutations.  As such, ISRIB promises to be an invaluable tool in proof-of-concept studies aiming to ameliorate defects resulting from inappropriate or pathol. activation of the ISR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkRGD2klbIorVg90H21EOLACvtfcHk0lheIgnzBpGPxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVSjtLfL&md5=7d897c1e9bd512843ee90aedb4babf89</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.7554%2FeLife.32733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.32733%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DY.%2BL.%26aulast%3DLeBon%26aufirst%3DL.%26aulast%3DEdalji%26aufirst%3DR.%26aulast%3DLim%26aufirst%3DH.%2BB.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DSidrauski%26aufirst%3DC.%26atitle%3DThe%2520Small%2520Molecule%2520Isrib%2520Rescues%2520the%2520Stability%2520and%2520Activity%2520of%2520Vanishing%2520White%2520Matter%2520Disease%2520Eif2b%2520Mutant%2520Complexes%26jtitle%3DElife%26date%3D2018%26volume%3D7%26spage%3De32733%26doi%3D10.7554%2FeLife.32733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Palacino, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swalley, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Hoosear, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beibel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renaud, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salcius, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servais, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLellan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuierer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blommers, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaustein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berenshteyn, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacoste, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roma, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaud, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishman, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dales, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivasankaran, R.</span></span> <span> </span><span class="NLM_article-title">Smn2 Splice Modulators Enhance U1-Pre-Mrna Association and Rescue Sma Mice</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">511</span>– <span class="NLM_lpage">517</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnchembio.1837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26030728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=511-517&author=J.+Palacinoauthor=S.+E.+Swalleyauthor=C.+Songauthor=A.+K.+Cheungauthor=L.+Shuauthor=X.+Zhangauthor=M.+Van+Hoosearauthor=Y.+Shinauthor=D.+N.+Chinauthor=C.+G.+Kellerauthor=M.+Beibelauthor=N.+A.+Renaudauthor=T.+M.+Smithauthor=M.+Salciusauthor=X.+Shiauthor=M.+Hildauthor=R.+Servaisauthor=M.+Jainauthor=L.+Dengauthor=C.+Bullockauthor=M.+McLellanauthor=S.+Schuiererauthor=L.+Murphyauthor=M.+J.+Blommersauthor=C.+Blausteinauthor=F.+Berenshteynauthor=A.+Lacosteauthor=J.+R.+Thomasauthor=G.+Romaauthor=G.+A.+Michaudauthor=B.+S.+Tsengauthor=J.+A.+Porterauthor=V.+E.+Myerauthor=J.+A.+Tallaricoauthor=L.+G.+Hamannauthor=D.+Curtisauthor=M.+C.+Fishmanauthor=W.+F.+Dietrichauthor=N.+A.+Dalesauthor=R.+Sivasankaran&title=Smn2+Splice+Modulators+Enhance+U1-Pre-Mrna+Association+and+Rescue+Sma+Mice&doi=10.1038%2Fnchembio.1837"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">SMN2 splice modulators enhance U1-pre-mRNA association and rescue SMA mice</span></div><div class="casAuthors">Palacino, James; Swalley, Susanne E.; Song, Cheng; Cheung, Atwood K.; Shu, Lei; Zhang, Xiaolu; Van Hoosear, Mailin; Shin, Youngah; Chin, Donovan N.; Keller, Caroline Gubser; Beibel, Martin; Renaud, Nicole A.; Smith, Thomas M.; Salcius, Michael; Shi, Xiaoying; Hild, Marc; Servais, Rebecca; Jain, Monish; Deng, Lin; Bullock, Caroline; McLellan, Michael; Schuierer, Sven; Murphy, Leo; Blommers, Marcel J. J.; Blaustein, Cecile; Berenshteyn, Frada; Lacoste, Arnaud; Thomas, Jason R.; Roma, Guglielmo; Michaud, Gregory A.; Tseng, Brian S.; Porter, Jeffery A.; Myer, Vic E.; Tallarico, John A.; Hamann, Lawrence G.; Curtis, Daniel; Fishman, Mark C.; Dietrich, William F.; Dales, Natalie A.; Sivasankaran, Rajeev</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">511-517</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Spinal muscular atrophy (SMA), which results from the loss of expression of the survival of motor neuron-1 (SMN1) gene, represents the most common genetic cause of pediatric mortality.  A duplicate copy (SMN2) is inefficiently spliced, producing a truncated and unstable protein.  We describe herein a potent, orally active, small-mol. enhancer of SMN2 splicing that elevates full-length SMN protein and extends survival in a severe SMA mouse model.  We demonstrate that the mol. mechanism of action is via stabilization of the transient double-strand RNA structure formed by the SMN2 pre-mRNA and U1 small nuclear ribonucleic protein (snRNP) complex.  The binding affinity of U1 snRNP to the 5' splice site is increased in a sequence-selective manner, discrete from constitutive recognition.  This new mechanism demonstrates the feasibility of small mol.-mediated, sequence-selective splice modulation and the potential for leveraging this strategy in other splicing diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdMqLIC0lw17Vg90H21EOLACvtfcHk0lheIgnzBpGPxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7zN&md5=15e56c8b3f48feffa150e975a81633a0</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1837%26sid%3Dliteratum%253Aachs%26aulast%3DPalacino%26aufirst%3DJ.%26aulast%3DSwalley%26aufirst%3DS.%2BE.%26aulast%3DSong%26aufirst%3DC.%26aulast%3DCheung%26aufirst%3DA.%2BK.%26aulast%3DShu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DVan%2BHoosear%26aufirst%3DM.%26aulast%3DShin%26aufirst%3DY.%26aulast%3DChin%26aufirst%3DD.%2BN.%26aulast%3DKeller%26aufirst%3DC.%2BG.%26aulast%3DBeibel%26aufirst%3DM.%26aulast%3DRenaud%26aufirst%3DN.%2BA.%26aulast%3DSmith%26aufirst%3DT.%2BM.%26aulast%3DSalcius%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DServais%26aufirst%3DR.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DBullock%26aufirst%3DC.%26aulast%3DMcLellan%26aufirst%3DM.%26aulast%3DSchuierer%26aufirst%3DS.%26aulast%3DMurphy%26aufirst%3DL.%26aulast%3DBlommers%26aufirst%3DM.%2BJ.%26aulast%3DBlaustein%26aufirst%3DC.%26aulast%3DBerenshteyn%26aufirst%3DF.%26aulast%3DLacoste%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DJ.%2BR.%26aulast%3DRoma%26aufirst%3DG.%26aulast%3DMichaud%26aufirst%3DG.%2BA.%26aulast%3DTseng%26aufirst%3DB.%2BS.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DTallarico%26aufirst%3DJ.%2BA.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DCurtis%26aufirst%3DD.%26aulast%3DFishman%26aufirst%3DM.%2BC.%26aulast%3DDietrich%26aufirst%3DW.%2BF.%26aulast%3DDales%26aufirst%3DN.%2BA.%26aulast%3DSivasankaran%26aufirst%3DR.%26atitle%3DSmn2%2520Splice%2520Modulators%2520Enhance%2520U1-Pre-Mrna%2520Association%2520and%2520Rescue%2520Sma%2520Mice%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D511%26epage%3D517%26doi%3D10.1038%2Fnchembio.1837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, C. X.</span></span> <span> </span><span class="NLM_article-title">Tau and Neurodegenerative Disease: The Story So Far</span>. <i>Nat. Rev. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1038/nrneurol.2015.225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnrneurol.2015.225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26635213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVSiurvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=15-27&author=K.+Iqbalauthor=F.+Liuauthor=C.+X.+Gong&title=Tau+and+Neurodegenerative+Disease%3A+The+Story+So+Far&doi=10.1038%2Fnrneurol.2015.225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Tau and neurodegenerative disease: the story so far</span></div><div class="casAuthors">Iqbal, Khalid; Liu, Fei; Gong, Cheng-Xin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Neurology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-27</span>CODEN:
                <span class="NLM_cas:coden">NRNACP</span>;
        ISSN:<span class="NLM_cas:issn">1759-4758</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In 1975, tau protein was isolated as a microtubule-assocd. factor from the porcine brain.  In the previous year, a paired helical filament (PHF) protein had been identified in neurofibrillary tangles in the brains of individuals with Alzheimer disease (AD), but it was not until 1986 that the PHF protein and tau were discovered to be one and the same.  In the AD brain, tau was found to be abnormally hyperphosphorylated, and it inhibited rather than promoted in vitro microtubule assembly.  Almost 80 disease-causing exonic missense and intronic silent mutations in the tau gene have been found in familial cases of frontotemporal dementia but, to date, no such mutation has been found in AD.  The first phase I clin. trial of an active tau immunization vaccine in patients with AD was recently completed.  Assays for tau levels in cerebrospinal fluid and plasma are now available, and tau radiotracers for PET are under development.  In this article, we provide an overview of the pivotal discoveries in the tau research field over the past 40 years.  We also review the current status of the field, including disease mechanisms and therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlP4cbMS6zdLVg90H21EOLACvtfcHk0lgiZaVbbFQCCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVSiurvL&md5=94154d6c70f398a683821fe31ad3660e</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1038%2Fnrneurol.2015.225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrneurol.2015.225%26sid%3Dliteratum%253Aachs%26aulast%3DIqbal%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DGong%26aufirst%3DC.%2BX.%26atitle%3DTau%2520and%2520Neurodegenerative%2520Disease%253A%2520The%2520Story%2520So%2520Far%26jtitle%3DNat.%2520Rev.%2520Neurol.%26date%3D2016%26volume%3D12%26spage%3D15%26epage%3D27%26doi%3D10.1038%2Fnrneurol.2015.225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eglen, R. M.</span></span> <span> </span><span class="NLM_article-title">Three-Dimensional Cell Cultures in Drug Discovery and Development</span>. <i>SLAS Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">456</span>– <span class="NLM_lpage">472</span>, <span class="refDoi"> DOI: 10.1177/1087057117696795</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1177%2F1087057117696795" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28520521" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKktrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=456-472&author=Y.+Fangauthor=R.+M.+Eglen&title=Three-Dimensional+Cell+Cultures+in+Drug+Discovery+and+Development&doi=10.1177%2F1087057117696795"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Three-Dimensional Cell Cultures in Drug Discovery and Development</span></div><div class="casAuthors">Fang, Ye; Eglen, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">456-472</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  The past decades have witnessed significant efforts toward the development of three-dimensional (3D) cell cultures as systems that better mimic in vivo physiol.  Today, 3D cell cultures are emerging, not only as a new tool in early drug discovery but also as potential therapeutics to treat disease.  In this review, we assess leading 3D cell culture technologies and their impact on drug discovery, including spheroids, organoids, scaffolds, hydrogels, organs-on-chips, and 3D bioprinting.  We also discuss the implementation of these technologies in compd. identification, screening, and development, ranging from disease modeling to assessment of efficacy and safety profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8_ypNaFyNyLVg90H21EOLACvtfcHk0lgiZaVbbFQCCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKktrg%253D&md5=1f8a292f0f288fca4fd5d4c2909e58fc</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1177%2F1087057117696795&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057117696795%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DY.%26aulast%3DEglen%26aufirst%3DR.%2BM.%26atitle%3DThree-Dimensional%2520Cell%2520Cultures%2520in%2520Drug%2520Discovery%2520and%2520Development%26jtitle%3DSLAS%2520Discovery%26date%3D2017%26volume%3D22%26spage%3D456%26epage%3D472%26doi%3D10.1177%2F1087057117696795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetzel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marriott, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dreau, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Avanzo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanzi, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H.</span></span> <span> </span><span class="NLM_article-title">A 3d Human Triculture System Modeling Neurodegeneration and Neuroinflammation in Alzheimer’s Disease</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">951</span>, <span class="refDoi"> DOI: 10.1038/s41593-018-0175-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41593-018-0175-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=29950669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1eisrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2018&pages=941-951&author=J.+Parkauthor=I.+Wetzelauthor=I.+Marriottauthor=D.+Dreauauthor=C.+D%E2%80%99Avanzoauthor=D.+Y.+Kimauthor=R.+E.+Tanziauthor=H.+Cho&title=A+3d+Human+Triculture+System+Modeling+Neurodegeneration+and+Neuroinflammation+in+Alzheimer%E2%80%99s+Disease&doi=10.1038%2Fs41593-018-0175-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">A 3D human triculture system modeling neurodegeneration and neuroinflammation in Alzheimer's disease</span></div><div class="casAuthors">Park, Joseph; Wetzel, Isaac; Marriott, Ian; Dreau, Didier; D'Avanzo, Carla; Kim, Doo Yeon; Tanzi, Rudolph E.; Cho, Hansang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-951</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Alzheimer's disease (AD) is characterized by beta-amyloid accumulation, phosphorylated tau formation, hyperactivation of glial cells, and neuronal loss.  The mechanisms of AD pathogenesis, however, remain poorly understood, partially due to the lack of relevant models that can comprehensively recapitulate multistage intercellular interactions in human AD brains.  Here we present a new three-dimensional (3D) human AD triculture model using neurons, astrocytes, and microglia in a 3D microfluidic platform.  Our model provided key representative AD features: beta-amyloid aggregation, phosphorylated tau accumulation, and neuroinflammatory activity.  In particular, the model mirrored microglial recruitment, neurotoxic activities such as axonal cleavage, and NO release damaging AD neurons and astrocytes.  Our model will serve to facilitate the development of more precise human brain models for basic mechanistic studies in neural-glial interactions and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW5hk7KxWJFLVg90H21EOLACvtfcHk0lgCE1lnNAPMoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1eisrrP&md5=681b07976a82487fb1f3bc8e7867417b</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1038%2Fs41593-018-0175-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41593-018-0175-4%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DJ.%26aulast%3DWetzel%26aufirst%3DI.%26aulast%3DMarriott%26aufirst%3DI.%26aulast%3DDreau%26aufirst%3DD.%26aulast%3DD%25E2%2580%2599Avanzo%26aufirst%3DC.%26aulast%3DKim%26aufirst%3DD.%2BY.%26aulast%3DTanzi%26aufirst%3DR.%2BE.%26aulast%3DCho%26aufirst%3DH.%26atitle%3DA%25203d%2520Human%2520Triculture%2520System%2520Modeling%2520Neurodegeneration%2520and%2520Neuroinflammation%2520in%2520Alzheimer%25E2%2580%2599s%2520Disease%26jtitle%3DNat.%2520Neurosci.%26date%3D2018%26volume%3D21%26spage%3D941%26epage%3D951%26doi%3D10.1038%2Fs41593-018-0175-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monzel, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smits, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemmer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hachi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Wuellen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarazo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruggemann, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boussaad, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, R. M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwamborn, J. C.</span></span> <span> </span><span class="NLM_article-title">Derivation of Human Midbrain-Specific Organoids from Neuroepithelial Stem Cells</span>. <i>Stem Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1144</span>– <span class="NLM_lpage">1154</span>, <span class="refDoi"> DOI: 10.1016/j.stemcr.2017.03.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.stemcr.2017.03.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28416282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmt12rurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1144-1154&author=A.+S.+Monzelauthor=L.+M.+Smitsauthor=K.+Hemmerauthor=S.+Hachiauthor=E.+L.+Morenoauthor=T.+van+Wuellenauthor=J.+Jarazoauthor=J.+Walterauthor=I.+Bruggemannauthor=I.+Boussaadauthor=E.+Bergerauthor=R.+M.+T.+Flemingauthor=S.+Bologninauthor=J.+C.+Schwamborn&title=Derivation+of+Human+Midbrain-Specific+Organoids+from+Neuroepithelial+Stem+Cells&doi=10.1016%2Fj.stemcr.2017.03.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Derivation of Human Midbrain-Specific Organoids from Neuroepithelial Stem Cells</span></div><div class="casAuthors">Monzel, Anna S.; Smits, Lisa M.; Hemmer, Kathrin; Hachi, Siham; Moreno, Edinson Lucumi; van Wuellen, Thea; Jarazo, Javier; Walter, Jonas; Bruggemann, Inga; Boussaad, Ibrahim; Berger, Emanuel; Fleming, Ronan M. T.; Bolognin, Silvia; Schwamborn, Jens C.</div><div class="citationInfo"><span class="NLM_cas:title">Stem Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1144-1154</span>CODEN:
                <span class="NLM_cas:coden">SCRTFL</span>;
        ISSN:<span class="NLM_cas:issn">2213-6711</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Research on human brain development and neurol. diseases is limited by the lack of advanced exptl. in vitro models that truly recapitulate the complexity of the human brain.  Here, we describe a robust human brain organoid system that is highly specific to the midbrain derived from regionally patterned neuroepithelial stem cells.  These human midbrain organoids contain spatially organized groups of dopaminergic neurons, which make them an attractive model for the study of Parkinson's disease.  Midbrain organoids are characterized in detail for neuronal, astroglial, and oligodendrocyte differentiation.  Furthermore, we show the presence of synaptic connections and electrophysiol. activity.  The complexity of this model is further highlighted by the myelination of neurites.  The present midbrain organoid system has the potential to be used for advanced in vitro disease modeling and therapy development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXJwLKJ8guk7Vg90H21EOLACvtfcHk0lgCE1lnNAPMoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmt12rurk%253D&md5=dc5e89c31a678846fa536b8bef5091b1</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1016%2Fj.stemcr.2017.03.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.stemcr.2017.03.010%26sid%3Dliteratum%253Aachs%26aulast%3DMonzel%26aufirst%3DA.%2BS.%26aulast%3DSmits%26aufirst%3DL.%2BM.%26aulast%3DHemmer%26aufirst%3DK.%26aulast%3DHachi%26aufirst%3DS.%26aulast%3DMoreno%26aufirst%3DE.%2BL.%26aulast%3Dvan%2BWuellen%26aufirst%3DT.%26aulast%3DJarazo%26aufirst%3DJ.%26aulast%3DWalter%26aufirst%3DJ.%26aulast%3DBruggemann%26aufirst%3DI.%26aulast%3DBoussaad%26aufirst%3DI.%26aulast%3DBerger%26aufirst%3DE.%26aulast%3DFleming%26aufirst%3DR.%2BM.%2BT.%26aulast%3DBolognin%26aufirst%3DS.%26aulast%3DSchwamborn%26aufirst%3DJ.%2BC.%26atitle%3DDerivation%2520of%2520Human%2520Midbrain-Specific%2520Organoids%2520from%2520Neuroepithelial%2520Stem%2520Cells%26jtitle%3DStem%2520Cell%2520Rep.%26date%3D2017%26volume%3D8%26spage%3D1144%26epage%3D1154%26doi%3D10.1016%2Fj.stemcr.2017.03.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Birey, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makinson, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Islam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzler, K. R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panagiotakos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thom, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Rourke, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinmetz, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallmayer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huguenard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasca, S. P.</span></span> <span> </span><span class="NLM_article-title">Assembly of Functionally Integrated Human Forebrain Spheroids</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>545</i></span>,  <span class="NLM_fpage">54</span>– <span class="NLM_lpage">59</span>, <span class="refDoi"> DOI: 10.1038/nature22330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnature22330" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28445465" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmslehs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=545&publication_year=2017&pages=54-59&author=F.+Bireyauthor=J.+Andersenauthor=C.+D.+Makinsonauthor=S.+Islamauthor=W.+Weiauthor=N.+Huberauthor=H.+C.+Fanauthor=K.+R.+C.+Metzlerauthor=G.+Panagiotakosauthor=N.+Thomauthor=N.+A.+O%E2%80%99Rourkeauthor=L.+M.+Steinmetzauthor=J.+A.+Bernsteinauthor=J.+Hallmayerauthor=J.+R.+Huguenardauthor=S.+P.+Pasca&title=Assembly+of+Functionally+Integrated+Human+Forebrain+Spheroids&doi=10.1038%2Fnature22330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Assembly of functionally integrated human forebrain spheroids</span></div><div class="casAuthors">Birey, Fikri; Andersen, Jimena; Makinson, Christopher D.; Islam, Saiful; Wei, Wu; Huber, Nina; Fan, H. Christina; Metzler, Kimberly R. Cordes; Panagiotakos, Georgia; Thom, Nicholas; O'Rourke, Nancy A.; Steinmetz, Lars M.; Bernstein, Jonathan A.; Hallmayer, Joachim; Huguenard, John R.; Pasca, Sergiu P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">545</span>
        (<span class="NLM_cas:issue">7652</span>),
    <span class="NLM_cas:pages">54-59</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The development of the nervous system involves a coordinated succession of events including the migration of GABAergic (γ-aminobutyric-acid-releasing) neurons from ventral to dorsal forebrain and their integration into cortical circuits.  However, these interregional interactions have not yet been modelled with human cells.  Here we generate three-dimensional spheroids from human pluripotent stem cells that resemble either the dorsal or ventral forebrain and contain cortical glutamatergic or GABAergic neurons.  These subdomain-specific forebrain spheroids can be assembled in vitro to recapitulate the saltatory migration of interneurons obsd. in the fetal forebrain.  Using this system, we find that in Timothy syndrome - a neurodevelopmental disorder that is caused by mutations in the CaV1.2 calcium channel - interneurons display abnormal migratory saltations.  We also show that after migration, interneurons functionally integrate with glutamatergic neurons to form a microphysiol. system.  We anticipate that this approach will be useful for studying neural development and disease, and for deriving spheroids that resemble other brain regions to assemble circuits in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-CnPnrt7jDrVg90H21EOLACvtfcHk0lgCE1lnNAPMoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmslehs70%253D&md5=60fb5a24730aef9ad1c3fd5e6a1f2336</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1038%2Fnature22330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22330%26sid%3Dliteratum%253Aachs%26aulast%3DBirey%26aufirst%3DF.%26aulast%3DAndersen%26aufirst%3DJ.%26aulast%3DMakinson%26aufirst%3DC.%2BD.%26aulast%3DIslam%26aufirst%3DS.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DHuber%26aufirst%3DN.%26aulast%3DFan%26aufirst%3DH.%2BC.%26aulast%3DMetzler%26aufirst%3DK.%2BR.%2BC.%26aulast%3DPanagiotakos%26aufirst%3DG.%26aulast%3DThom%26aufirst%3DN.%26aulast%3DO%25E2%2580%2599Rourke%26aufirst%3DN.%2BA.%26aulast%3DSteinmetz%26aufirst%3DL.%2BM.%26aulast%3DBernstein%26aufirst%3DJ.%2BA.%26aulast%3DHallmayer%26aufirst%3DJ.%26aulast%3DHuguenard%26aufirst%3DJ.%2BR.%26aulast%3DPasca%26aufirst%3DS.%2BP.%26atitle%3DAssembly%2520of%2520Functionally%2520Integrated%2520Human%2520Forebrain%2520Spheroids%26jtitle%3DNature%26date%3D2017%26volume%3D545%26spage%3D54%26epage%3D59%26doi%3D10.1038%2Fnature22330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadoshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakaguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ando, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eiraku, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasai, Y.</span></span> <span> </span><span class="NLM_article-title">Self-Organization of Axial Polarity, inside-out Layer Pattern, and Species-Specific Progenitor Dynamics in Human Es Cell-Derived Neocortex</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">20284</span>– <span class="NLM_lpage">20289</span>, <span class="refDoi"> DOI: 10.1073/pnas.1315710110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1073%2Fpnas.1315710110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=24277810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKmsLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=20284-20289&author=T.+Kadoshimaauthor=H.+Sakaguchiauthor=T.+Nakanoauthor=M.+Soenauthor=S.+Andoauthor=M.+Eirakuauthor=Y.+Sasai&title=Self-Organization+of+Axial+Polarity%2C+inside-out+Layer+Pattern%2C+and+Species-Specific+Progenitor+Dynamics+in+Human+Es+Cell-Derived+Neocortex&doi=10.1073%2Fpnas.1315710110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Self-organization of axial polarity, inside-out layer pattern, and species-specific progenitor dynamics in human ES cell-derived neocortex</span></div><div class="casAuthors">Kadoshima, Taisuke; Sakaguchi, Hideya; Nakano, Tokushige; Soen, Mika; Ando, Satoshi; Eiraku, Mototsugu; Sasai, Yoshiki</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">20284-20289,S20284/1-S20284/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Here, using further optimized 3D culture that allows highly selective induction and long-term growth of human ES cell (hESC)-derived cortical neuroepithelium, we demonstrate unique aspects of self-organization in human neocorticogenesis.  Self-organized cortical tissue spontaneously forms a polarity along the dorsocaudal-ventrorostral axis and undergoes region-specific rolling morphogenesis that generates a semispherical structure.  The neuroepithelium self-forms a multilayered structure including three neuronal zones (subplate, cortical plate, and Cajal-Retzius cell zones) and three progenitor zones (ventricular, subventricular, and intermediate zones) in the same apical-basal order as seen in the human fetal cortex in the early second trimester.  In the cortical plate, late-born neurons tend to localize more basally to early-born neurons, consistent with the inside-out pattern seen in vivo.  Furthermore, the outer subventricular zone contains basal progenitors that share characteristics with outer radial glia abundantly found in the human, but not mouse, fetal brain.  Thus, human neocorticogenesis involves intrinsic programs that enable the emergence of complex neocortical features.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIJ-H67pTIwLVg90H21EOLACvtfcHk0lioP7kVfclLqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKmsLzL&md5=c71a7e0b4a1d8fa4da52771ec449d1e6</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1315710110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1315710110%26sid%3Dliteratum%253Aachs%26aulast%3DKadoshima%26aufirst%3DT.%26aulast%3DSakaguchi%26aufirst%3DH.%26aulast%3DNakano%26aufirst%3DT.%26aulast%3DSoen%26aufirst%3DM.%26aulast%3DAndo%26aufirst%3DS.%26aulast%3DEiraku%26aufirst%3DM.%26aulast%3DSasai%26aufirst%3DY.%26atitle%3DSelf-Organization%2520of%2520Axial%2520Polarity%252C%2520inside-out%2520Layer%2520Pattern%252C%2520and%2520Species-Specific%2520Progenitor%2520Dynamics%2520in%2520Human%2520Es%2520Cell-Derived%2520Neocortex%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D20284%26epage%3D20289%26doi%3D10.1073%2Fpnas.1315710110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lancaster, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenzel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bicknell, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurles, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homfray, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penninger, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoblich, J. A.</span></span> <span> </span><span class="NLM_article-title">Cerebral Organoids Model Human Brain Development and Microcephaly</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>501</i></span>,  <span class="NLM_fpage">373</span>– <span class="NLM_lpage">379</span>, <span class="refDoi"> DOI: 10.1038/nature12517</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnature12517" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23995685" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlCntb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=501&publication_year=2013&pages=373-379&author=M.+A.+Lancasterauthor=M.+Rennerauthor=C.+A.+Martinauthor=D.+Wenzelauthor=L.+S.+Bicknellauthor=M.+E.+Hurlesauthor=T.+Homfrayauthor=J.+M.+Penningerauthor=A.+P.+Jacksonauthor=J.+A.+Knoblich&title=Cerebral+Organoids+Model+Human+Brain+Development+and+Microcephaly&doi=10.1038%2Fnature12517"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Cerebral organoids model human brain development and microcephaly</span></div><div class="casAuthors">Lancaster, Madeline A.; Renner, Magdalena; Martin, Carol-Anne; Wenzel, Daniel; Bicknell, Louise S.; Hurles, Matthew E.; Homfray, Tessa; Penninger, Josef M.; Jackson, Andrew P.; Knoblich, Juergen A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">501</span>
        (<span class="NLM_cas:issue">7467</span>),
    <span class="NLM_cas:pages">373-379</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The complexity of the human brain has made it difficult to study many brain disorders in model organisms, highlighting the need for an in vitro model of human brain development.  Here we have developed a human pluripotent stem cell-derived three-dimensional organoid culture system, termed cerebral organoids, that develop various discrete, although interdependent, brain regions.  These include a cerebral cortex contg. progenitor populations that organize and produce mature cortical neuron subtypes.  Furthermore, cerebral organoids are shown to recapitulate features of human cortical development, namely characteristic progenitor zone organization with abundant outer radial glial stem cells.  Finally, we use RNA interference and patient-specific induced pluripotent stem cells to model microcephaly, a disorder that has been difficult to recapitulate in mice.  We demonstrate premature neuronal differentiation in patient organoids, a defect that could help to explain the disease phenotype.  Together, these data show that three-dimensional organoids can recapitulate development and disease even in this most complex human tissue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1ooY8YMg5k7Vg90H21EOLACvtfcHk0lioP7kVfclLqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlCntb3F&md5=96195248e51aa62a5a5d761aa56814a1</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1038%2Fnature12517&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12517%26sid%3Dliteratum%253Aachs%26aulast%3DLancaster%26aufirst%3DM.%2BA.%26aulast%3DRenner%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DC.%2BA.%26aulast%3DWenzel%26aufirst%3DD.%26aulast%3DBicknell%26aufirst%3DL.%2BS.%26aulast%3DHurles%26aufirst%3DM.%2BE.%26aulast%3DHomfray%26aufirst%3DT.%26aulast%3DPenninger%26aufirst%3DJ.%2BM.%26aulast%3DJackson%26aufirst%3DA.%2BP.%26aulast%3DKnoblich%26aufirst%3DJ.%2BA.%26atitle%3DCerebral%2520Organoids%2520Model%2520Human%2520Brain%2520Development%2520and%2520Microcephaly%26jtitle%3DNature%26date%3D2013%26volume%3D501%26spage%3D373%26epage%3D379%26doi%3D10.1038%2Fnature12517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armijo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bravo-Alegria, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becerra-Calixto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mays, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soto, C.</span></span> <span> </span><span class="NLM_article-title">Modeling Amyloid Beta and Tau Pathology in Human Cerebral Organoids</span>. <i>Mol. Psychiatry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2363</span>– <span class="NLM_lpage">2374</span>, <span class="refDoi"> DOI: 10.1038/s41380-018-0229-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41380-018-0229-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30171212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1GltLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=2363-2374&author=C.+Gonzalezauthor=E.+Armijoauthor=J.+Bravo-Alegriaauthor=A.+Becerra-Calixtoauthor=C.+E.+Maysauthor=C.+Soto&title=Modeling+Amyloid+Beta+and+Tau+Pathology+in+Human+Cerebral+Organoids&doi=10.1038%2Fs41380-018-0229-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling amyloid beta and tau pathology in human cerebral organoids</span></div><div class="casAuthors">Gonzalez, Cesar; Armijo, Enrique; Bravo-Alegria, Javiera; Becerra-Calixto, Andrea; Mays, Charles E.; Soto, Claudio</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Psychiatry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2363-2374</span>CODEN:
                <span class="NLM_cas:coden">MOPSFQ</span>;
        ISSN:<span class="NLM_cas:issn">1359-4184</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The typical abnormalities obsd. in the brain of Alzheimer's disease (AD) patients include synaptic alterations, neuronal death, brain inflammation, and the accumulation of protein aggregates in the form of amyloid plaques and neurofibrillary tangles.  Despite the development of many animal and in vitro models for AD, there is a lack of an exptl. approach that fully recapitulates essential aspects of the disease in human cells.  Here, we report the generation of a new model to study AD, consisting of cerebral organoids (COs) produced from human-induced pluripotent stem cells (iPSCs).  Under our exptl. conditions, COs grow to form three-dimensional (3D) structures contg. neural areas with cortical-like organization.  Anal. of COs by histol. and biochem. methods revealed that organoids produced from iPSCs derived from patients affected by familial AD or Down syndrome (DS) spontaneously develop over time pathol. features of AD, including accumulation of structures highly reminiscent to amyloid plaques and neurofibrillary tangles.  These pathol. abnormalities were not obsd. in COs generated from various controls, including human iPSCs from healthy individuals, human iPSCs from patients affected by Creutzfeldt-Jakob disease, mouse embryonic stem cells (ESCs), or mouse iPSCs.  These findings enable modeling genetic AD in a human cellular context in a 3D cortical-like tissue developed in vitro from patient-specific stem cells.  This system provides a more relevant disease model compared to pre-existing methods and offers a new platform for discovery of novel targets and screening of drugs for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZw32-DOdVa7Vg90H21EOLACvtfcHk0ljguelgsLaC0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1GltLzE&md5=e2ec2923964014d8c9bb145e3d5eeae3</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1038%2Fs41380-018-0229-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41380-018-0229-8%26sid%3Dliteratum%253Aachs%26aulast%3DGonzalez%26aufirst%3DC.%26aulast%3DArmijo%26aufirst%3DE.%26aulast%3DBravo-Alegria%26aufirst%3DJ.%26aulast%3DBecerra-Calixto%26aufirst%3DA.%26aulast%3DMays%26aufirst%3DC.%2BE.%26aulast%3DSoto%26aufirst%3DC.%26atitle%3DModeling%2520Amyloid%2520Beta%2520and%2520Tau%2520Pathology%2520in%2520Human%2520Cerebral%2520Organoids%26jtitle%3DMol.%2520Psychiatry%26date%3D2018%26volume%3D23%26spage%3D2363%26epage%3D2374%26doi%3D10.1038%2Fs41380-018-0229-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trujillo, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muotri, A. R.</span></span> <span> </span><span class="NLM_article-title">Brain Organoids and the Study of Neurodevelopment</span>. <i>Trends Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">982</span>– <span class="NLM_lpage">990</span>, <span class="refDoi"> DOI: 10.1016/j.molmed.2018.09.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.molmed.2018.09.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30377071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFyntrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=982-990&author=C.+A.+Trujilloauthor=A.+R.+Muotri&title=Brain+Organoids+and+the+Study+of+Neurodevelopment&doi=10.1016%2Fj.molmed.2018.09.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Brain Organoids and the Study of Neurodevelopment</span></div><div class="casAuthors">Trujillo, Cleber A.; Muotri, Alysson R.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Molecular Medicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">982-990</span>CODEN:
                <span class="NLM_cas:coden">TMMRCY</span>;
        ISSN:<span class="NLM_cas:issn">1471-4914</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Brain organoids are 3D self-assembled structures composed of hundreds of thousands to millions of cells that resemble the cellular organization and transcriptional and epigenetic signature of a developing human brain.  Advancements using brain organoids have been made to elucidate the genetic basis of certain neurodevelopmental disorders, such as microcephaly and autism; and to investigate the impact of environmental factors to the brain, such as during Zika virus infection.  It remains to be explored how far brain organoids can functionally mature and process external information.  An improved brain organoid model might reproduce important aspects of the human brain in a more reproducible and high-throughput fashion.  This novel and complementary approach in the neuroscience toolbox opens perspectives to understand the fundamental features of the human neurodevelopment, with implications to personalize therapeutic opportunities for neurol. disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNGYKN-bnYlbVg90H21EOLACvtfcHk0ljguelgsLaC0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFyntrnF&md5=3a805e099e9e8c56b5f0239c2b158d4d</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2018.09.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2018.09.005%26sid%3Dliteratum%253Aachs%26aulast%3DTrujillo%26aufirst%3DC.%2BA.%26aulast%3DMuotri%26aufirst%3DA.%2BR.%26atitle%3DBrain%2520Organoids%2520and%2520the%2520Study%2520of%2520Neurodevelopment%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2018%26volume%3D24%26spage%3D982%26epage%3D990%26doi%3D10.1016%2Fj.molmed.2018.09.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arrasate, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span> <span> </span><span class="NLM_article-title">Automated Microscope System for Determining Factors That Predict Neuronal Fate</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">3840</span>– <span class="NLM_lpage">3845</span>, <span class="refDoi"> DOI: 10.1073/pnas.0409777102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1073%2Fpnas.0409777102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=15738408" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BD2MXisVOgt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=3840-3845&author=M.+Arrasateauthor=S.+Finkbeiner&title=Automated+Microscope+System+for+Determining+Factors+That+Predict+Neuronal+Fate&doi=10.1073%2Fpnas.0409777102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Automated microscope system for determining factors that predict neuronal fate</span></div><div class="casAuthors">Arrasate, Montserrat; Finkbeiner, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3840-3845</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Unraveling cause-and-effect relationships in the nervous system is challenging because some biol. processes begin stochastically, take a significant amt. of time to unfold, and affect small neuronal subpopulations that can be difficult to isolate and measure.  Single-cell approaches are slow, subject to user bias, and sometimes too laborious to achieve sample sizes large enough to detect important effects.  Here, we describe an automated imaging and anal. system that enables us to follow the fates of individual cells and intracellular proteins over time.  Observations can be quantified in a high-throughput manner with minimal user bias.  We have adapted survival anal. methods to det. whether and how factors measured during longitudinal anal. predict a particular biol. outcome.  The ability to monitor complex processes at single-cell resoln. quickly, quant., and over long intervals should have wide applications for biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUzK6HSFyV1rVg90H21EOLACvtfcHk0ljguelgsLaC0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXisVOgt7Y%253D&md5=8d2173976e3b9eb4b50b38cac219444c</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0409777102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0409777102%26sid%3Dliteratum%253Aachs%26aulast%3DArrasate%26aufirst%3DM.%26aulast%3DFinkbeiner%26aufirst%3DS.%26atitle%3DAutomated%2520Microscope%2520System%2520for%2520Determining%2520Factors%2520That%2520Predict%2520Neuronal%2520Fate%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D3840%26epage%3D3845%26doi%3D10.1073%2Fpnas.0409777102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haston, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span> <span> </span><span class="NLM_article-title">Clinical Trials in a Dish: The Potential of Pluripotent Stem Cells to Develop Therapies for Neurodegenerative Diseases</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">489</span>– <span class="NLM_lpage">510</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010715-103548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1146%2Fannurev-pharmtox-010715-103548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26514199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslGmsbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=489-510&author=K.+M.+Hastonauthor=S.+Finkbeiner&title=Clinical+Trials+in+a+Dish%3A+The+Potential+of+Pluripotent+Stem+Cells+to+Develop+Therapies+for+Neurodegenerative+Diseases&doi=10.1146%2Fannurev-pharmtox-010715-103548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical Trials in a Dish: The Potential of Pluripotent Stem Cells to Develop Therapies for Neurodegenerative Diseases</span></div><div class="casAuthors">Haston, Kelly M.; Finkbeiner, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">489-510</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Neurodegenerative diseases are a leading cause of death.  No disease-modifying therapies are available, and preclin. animal model data have routinely failed to translate into success for therapeutics.  Induced pluripotent stem cell (iPSC) biol. holds great promise for human in vitro disease modeling because these cells can give rise to any cell in the human brain and display phenotypes specific to neurodegenerative diseases previously identified in postmortem and clin. samples.  Here, we explore the potential and caveats of iPSC technol. as a platform for drug development and screening, and the future potential to use large cohorts of disease-bearing iPSCs to perform clin. trials in a dish.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIV9RCr3Mpi7Vg90H21EOLACvtfcHk0liPPjh0kpq4xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslGmsbbO&md5=0d4d798b015770e8568ad1710f8ff3e6</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010715-103548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010715-103548%26sid%3Dliteratum%253Aachs%26aulast%3DHaston%26aufirst%3DK.%2BM.%26aulast%3DFinkbeiner%26aufirst%3DS.%26atitle%3DClinical%2520Trials%2520in%2520a%2520Dish%253A%2520The%2520Potential%2520of%2520Pluripotent%2520Stem%2520Cells%2520to%2520Develop%2520Therapies%2520for%2520Neurodegenerative%2520Diseases%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2016%26volume%3D56%26spage%3D489%26epage%3D510%26doi%3D10.1146%2Fannurev-pharmtox-010715-103548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michael
Ando, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanaswamy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christiansen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoyer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roat, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueden, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shankar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, P.</span></span> <span> </span><span class="NLM_article-title">Assessing Microscope Image Focus Quality with Deep Learning</span>. <i>BMC Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">77</span>, <span class="refDoi"> DOI: 10.1186/s12859-018-2087-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1186%2Fs12859-018-2087-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=29540156" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC1Mnht1aisg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=77&author=S.+J.+Yangauthor=M.+Berndlauthor=D.+Michael%0AAndoauthor=M.+Barchauthor=A.+Narayanaswamyauthor=E.+Christiansenauthor=S.+Hoyerauthor=C.+Roatauthor=J.+Hungauthor=C.+T.+Ruedenauthor=A.+Shankarauthor=S.+Finkbeinerauthor=P.+Nelson&title=Assessing+Microscope+Image+Focus+Quality+with+Deep+Learning&doi=10.1186%2Fs12859-018-2087-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing microscope image focus quality with deep learning</span></div><div class="casAuthors">Yang Samuel J; Berndl Marc; Michael Ando D; Narayanaswamy Arunachalam; Christiansen Eric; Hoyer Stephan; Roat Chris; Shankar Asim; Nelson Philip; Barch Mariya; Finkbeiner Steven; Hung Jane; Hung Jane; Rueden Curtis T; Finkbeiner Steven</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">77</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Large image datasets acquired on automated microscopes typically have some fraction of low quality, out-of-focus images, despite the use of hardware autofocus systems.  Identification of these images using automated image analysis with high accuracy is important for obtaining a clean, unbiased image dataset.  Complicating this task is the fact that image focus quality is only well-defined in foreground regions of images, and as a result, most previous approaches only enable a computation of the relative difference in quality between two or more images, rather than an absolute measure of quality.  RESULTS:  We present a deep neural network model capable of predicting an absolute measure of image focus on a single image in isolation, without any user-specified parameters.  The model operates at the image-patch level, and also outputs a measure of prediction certainty, enabling interpretable predictions.  The model was trained on only 384 in-focus Hoechst (nuclei) stain images of U2OS cells, which were synthetically defocused to one of 11 absolute defocus levels during training.  The trained model can generalize on previously unseen real Hoechst stain images, identifying the absolute image focus to within one defocus level (approximately 3 pixel blur diameter difference) with 95% accuracy.  On a simpler binary in/out-of-focus classification task, the trained model outperforms previous approaches on both Hoechst and Phalloidin (actin) stain images (F-scores of 0.89 and 0.86, respectively over 0.84 and 0.83), despite only having been presented Hoechst stain images during training.  Lastly, we observe qualitatively that the model generalizes to two additional stains, Hoechst and Tubulin, of an unseen cell type (Human MCF-7) acquired on a different instrument.  CONCLUSIONS:  Our deep neural network enables classification of out-of-focus microscope images with both higher accuracy and greater precision than previous approaches via interpretable patch-level focus and certainty predictions.  The use of synthetically defocused images precludes the need for a manually annotated training dataset.  The model also generalizes to different image and cell types.  The framework for model training and image prediction is available as a free software library and the pre-trained model is available for immediate use in Fiji (ImageJ) and CellProfiler.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS54d2r4ujj7a__Jf7-eMvJfW6udTcc2eZZQPm5uVm717ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mnht1aisg%253D%253D&md5=dc1ec2714da913b75c050af0449ed8b3</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1186%2Fs12859-018-2087-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12859-018-2087-4%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DS.%2BJ.%26aulast%3DBerndl%26aufirst%3DM.%26aulast%3DMichael%2BAndo%26aufirst%3DD.%26aulast%3DBarch%26aufirst%3DM.%26aulast%3DNarayanaswamy%26aufirst%3DA.%26aulast%3DChristiansen%26aufirst%3DE.%26aulast%3DHoyer%26aufirst%3DS.%26aulast%3DRoat%26aufirst%3DC.%26aulast%3DHung%26aufirst%3DJ.%26aulast%3DRueden%26aufirst%3DC.%2BT.%26aulast%3DShankar%26aufirst%3DA.%26aulast%3DFinkbeiner%26aufirst%3DS.%26aulast%3DNelson%26aufirst%3DP.%26atitle%3DAssessing%2520Microscope%2520Image%2520Focus%2520Quality%2520with%2520Deep%2520Learning%26jtitle%3DBMC%2520Bioinformatics%26date%3D2018%26volume%3D19%26spage%3D77%26doi%3D10.1186%2Fs12859-018-2087-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frumkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kassner, P. D.</span></span> <span> </span><span class="NLM_article-title">Cell-Based Screening: Extracting Meaning from Complex Data</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">160</span>– <span class="NLM_lpage">174</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2015.02.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.neuron.2015.02.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25856492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsVCisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2015&pages=160-174&author=S.+Finkbeinerauthor=M.+Frumkinauthor=P.+D.+Kassner&title=Cell-Based+Screening%3A+Extracting+Meaning+from+Complex+Data&doi=10.1016%2Fj.neuron.2015.02.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-Based Screening: Extracting Meaning from Complex Data</span></div><div class="casAuthors">Finkbeiner, Steven; Frumkin, Michael; Kassner, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">160-174</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Unbiased discovery approaches have the potential to uncover neurobiol. insights into CNS disease and lead to the development of therapies.  Here, we review lessons learned from imaging-based screening approaches and recent advances in these areas, including powerful new computational tools to synthesize complex data into more useful knowledge that can reliably guide future research and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobSk3HekAKbbVg90H21EOLACvtfcHk0liPPjh0kpq4xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsVCisLg%253D&md5=8aff9eec9955222985f8f10b1d4e22ab</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2015.02.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2015.02.023%26sid%3Dliteratum%253Aachs%26aulast%3DFinkbeiner%26aufirst%3DS.%26aulast%3DFrumkin%26aufirst%3DM.%26aulast%3DKassner%26aufirst%3DP.%2BD.%26atitle%3DCell-Based%2520Screening%253A%2520Extracting%2520Meaning%2520from%2520Complex%2520Data%26jtitle%3DNeuron%26date%3D2015%26volume%3D86%26spage%3D160%26epage%3D174%26doi%3D10.1016%2Fj.neuron.2015.02.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daub, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span> <span> </span><span class="NLM_article-title">High-Content Screening of Primary Neurons: Ready for Prime Time</span>. <i>Curr. Opin. Neurobiol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">537</span>– <span class="NLM_lpage">543</span>, <span class="refDoi"> DOI: 10.1016/j.conb.2009.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.conb.2009.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=19889533" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsV2gs7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=537-543&author=A.+Daubauthor=P.+Sharmaauthor=S.+Finkbeiner&title=High-Content+Screening+of+Primary+Neurons%3A+Ready+for+Prime+Time&doi=10.1016%2Fj.conb.2009.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">High-content screening of primary neurons: Ready for prime time</span></div><div class="casAuthors">Daub, Aaron; Sharma, Punita; Finkbeiner, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Neurobiology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">537-543</span>CODEN:
                <span class="NLM_cas:coden">COPUEN</span>;
        ISSN:<span class="NLM_cas:issn">0959-4388</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">High-content screening (HCS), historically limited to drug-development companies, is now a powerful and affordable technol. for academic researchers.  Through automated routines, this technol. acquires large datasets of fluorescence images depicting the functional states of thousands to millions of cells.  Information on shapes, textures, intensities, and localizations is then used to create unique representations, or 'phenotypic signatures,' of each cell.  These signatures quantify physiol. or diseased states, for example, dendritic arborization, drug response, or cell coping strategies.  Live-cell imaging in HCS adds the ability to correlate cellular events at different points in time, thereby allowing sensitivities and observations not possible with fixed endpoint anal.  HCS with live-cell imaging therefore provides an unprecedented capability to detect spatiotemporal changes in cells and is particularly suited for time-dependent, stochastic processes such as neurodegenerative disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6q9y6kFFysLVg90H21EOLACvtfcHk0ljz2gztH5dU5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsV2gs7nP&md5=fa17456b6bafd063e024e34f657bf494</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1016%2Fj.conb.2009.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.conb.2009.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DDaub%26aufirst%3DA.%26aulast%3DSharma%26aufirst%3DP.%26aulast%3DFinkbeiner%26aufirst%3DS.%26atitle%3DHigh-Content%2520Screening%2520of%2520Primary%2520Neurons%253A%2520Ready%2520for%2520Prime%2520Time%26jtitle%3DCurr.%2520Opin.%2520Neurobiol.%26date%3D2009%26volume%3D19%26spage%3D537%26epage%3D543%26doi%3D10.1016%2Fj.conb.2009.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Christiansen, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ando, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javaherian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skibinski, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipnick, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mount, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neil, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goyal, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedus, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poplin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span> <span> </span><span class="NLM_article-title">In Silico Labeling: Predicting Fluorescent Labels in Unlabeled Images</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>173</i></span>,  <span class="NLM_fpage">792</span>– <span class="NLM_lpage">803</span>, e719 <span class="refDoi"> DOI: 10.1016/j.cell.2018.03.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.cell.2018.03.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=29656897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnsFClu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=792-803&author=E.+M.+Christiansenauthor=S.+J.+Yangauthor=D.+M.+Andoauthor=A.+Javaherianauthor=G.+Skibinskiauthor=S.+Lipnickauthor=E.+Mountauthor=A.+O%E2%80%99Neilauthor=K.+Shahauthor=A.+K.+Leeauthor=P.+Goyalauthor=W.+Fedusauthor=R.+Poplinauthor=A.+Estevaauthor=M.+Berndlauthor=L.+L.+Rubinauthor=P.+Nelsonauthor=S.+Finkbeiner&title=In+Silico+Labeling%3A+Predicting+Fluorescent+Labels+in+Unlabeled+Images&doi=10.1016%2Fj.cell.2018.03.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">In Silico Labeling: Predicting Fluorescent Labels in Unlabeled Images</span></div><div class="casAuthors">Christiansen, Eric M.; Yang, Samuel J.; Ando, D. Michael; Javaherian, Ashkan; Skibinski, Gaia; Lipnick, Scott; Mount, Elliot; O'Neil, Alison; Shah, Kevan; Lee, Alicia K.; Goyal, Piyush; Fedus, William; Poplin, Ryan; Esteva, Andre; Berndl, Marc; Rubin, Lee L.; Nelson, Philip; Finkbeiner, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">173</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">792-803.e19</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Microscopy is a central method in life sciences.  Many popular methods, such as antibody labeling, are used to add phys. fluorescent labels to specific cellular constituents.  However, these approaches have significant drawbacks, including inconsistency; limitations in the no. of simultaneous labels because of spectral overlap; and necessary perturbations of the expt., such as fixing the cells, to generate the measurement.  Here, we show that a computational machine-learning approach, which we call "in silico labeling" (ISL), reliably predicts some fluorescent labels from transmitted-light images of unlabeled fixed or live biol. samples.  ISL predicts a range of labels, such as those for nuclei, cell type (e.g., neural), and cell state (e.g., cell death).  Because prediction happens in silico, the method is consistent, is not limited by spectral overlap, and does not disturb the expt.  ISL generates biol. measurements that would otherwise be problematic or impossible to acquire.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprM6l-lkvHfLVg90H21EOLACvtfcHk0ljz2gztH5dU5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnsFClu7k%253D&md5=45cbd1f004ac1b093f37903c5bae4c83</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.03.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.03.040%26sid%3Dliteratum%253Aachs%26aulast%3DChristiansen%26aufirst%3DE.%2BM.%26aulast%3DYang%26aufirst%3DS.%2BJ.%26aulast%3DAndo%26aufirst%3DD.%2BM.%26aulast%3DJavaherian%26aufirst%3DA.%26aulast%3DSkibinski%26aufirst%3DG.%26aulast%3DLipnick%26aufirst%3DS.%26aulast%3DMount%26aufirst%3DE.%26aulast%3DO%25E2%2580%2599Neil%26aufirst%3DA.%26aulast%3DShah%26aufirst%3DK.%26aulast%3DLee%26aufirst%3DA.%2BK.%26aulast%3DGoyal%26aufirst%3DP.%26aulast%3DFedus%26aufirst%3DW.%26aulast%3DPoplin%26aufirst%3DR.%26aulast%3DEsteva%26aufirst%3DA.%26aulast%3DBerndl%26aufirst%3DM.%26aulast%3DRubin%26aufirst%3DL.%2BL.%26aulast%3DNelson%26aufirst%3DP.%26aulast%3DFinkbeiner%26aufirst%3DS.%26atitle%3DIn%2520Silico%2520Labeling%253A%2520Predicting%2520Fluorescent%2520Labels%2520in%2520Unlabeled%2520Images%26jtitle%3DCell%26date%3D2018%26volume%3D173%26spage%3D792%26epage%3D803%26doi%3D10.1016%2Fj.cell.2018.03.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mason, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elia, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span> <span> </span><span class="NLM_article-title">The Receptor-Interacting Serine/Threonine Protein Kinase 1 (Ripk1) Regulates Progranulin Levels</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">3262</span>– <span class="NLM_lpage">3272</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.752006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1074%2Fjbc.M116.752006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28069809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtlyqtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=3262-3272&author=A.+R.+Masonauthor=L.+P.+Eliaauthor=S.+Finkbeiner&title=The+Receptor-Interacting+Serine%2FThreonine+Protein+Kinase+1+%28Ripk1%29+Regulates+Progranulin+Levels&doi=10.1074%2Fjbc.M116.752006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">The Receptor-interacting Serine/Threonine Protein Kinase 1 (RIPK1) Regulates Progranulin Levels</span></div><div class="casAuthors">Mason, Amanda R.; Elia, Lisa P.; Finkbeiner, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3262-3272</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Progranulin (PGRN), a secreted growth factor, is a key regulator of inflammation and is genetically linked to two common and devastating neurodegenerative diseases.  Haploinsufficiency mutations in GRN, the gene encoding PGRN, cause frontotemporal dementia (FTD), and a GRN SNP confers significantly increased risk for Alzheimer's disease (AD).  Because cellular and animal data indicate that increasing PGRN can reverse phenotypes of both FTD and AD, modulating PGRN level has been proposed as a therapeutic strategy for both diseases.  However, little is known about the regulation of PGRN levels.  In this study, we performed an siRNA-based screen of the kinome to identify genetic regulators of PGRN levels in a rodent cell-based model system.  We found that knocking down receptor-interacting serine/threonine protein kinase 1 (Ripk1) increased both intracellular and extracellular PGRN protein levels by increasing the translation rate of PGRN without affecting mRNA levels.  We obsd. this effect in Neuro2a cells, wild-type primary mouse neurons, and Grn-haploinsufficient primary neurons from an FTD mouse model.  We found that the effect of RIPK1 on PGRN is independent of the kinase activity of RIPK1 and occurs through a novel signaling pathway.  These data suggest that targeting RIPK1 may be a therapeutic strategy in both AD and FTD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3vcUnBMUQS7Vg90H21EOLACvtfcHk0ljz2gztH5dU5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtlyqtbs%253D&md5=be49c50b92b5866bb9bf3f288e230491</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.752006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.752006%26sid%3Dliteratum%253Aachs%26aulast%3DMason%26aufirst%3DA.%2BR.%26aulast%3DElia%26aufirst%3DL.%2BP.%26aulast%3DFinkbeiner%26aufirst%3DS.%26atitle%3DThe%2520Receptor-Interacting%2520Serine%252FThreonine%2520Protein%2520Kinase%25201%2520%2528Ripk1%2529%2520Regulates%2520Progranulin%2520Levels%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D3262%26epage%3D3272%26doi%3D10.1074%2Fjbc.M116.752006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Armakola, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higgins, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figley, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barmada, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scarborough, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shorter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krogan, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farese, R. V.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gitler, A. D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Rna Lariat Debranching Enzyme Suppresses Tdp-43 Toxicity in Als Disease Models</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1302</span>– <span class="NLM_lpage">1309</span>, <span class="refDoi"> DOI: 10.1038/ng.2434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fng.2434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23104007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFOmt7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=1302-1309&author=M.+Armakolaauthor=M.+J.+Higginsauthor=M.+D.+Figleyauthor=S.+J.+Barmadaauthor=E.+A.+Scarboroughauthor=Z.+Diazauthor=X.+Fangauthor=J.+Shorterauthor=N.+J.+Kroganauthor=S.+Finkbeinerauthor=R.+V.+Fareseauthor=A.+D.+Gitler&title=Inhibition+of+Rna+Lariat+Debranching+Enzyme+Suppresses+Tdp-43+Toxicity+in+Als+Disease+Models&doi=10.1038%2Fng.2434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of RNA lariat debranching enzyme suppresses TDP-43 toxicity in ALS disease models</span></div><div class="casAuthors">Armakola, Maria; Higgins, Matthew J.; Figley, Matthew D.; Barmada, Sami J.; Scarborough, Emily A.; Diaz, Zamia; Fang, Xiaodong; Shorter, James; Krogan, Nevan J.; Finkbeiner, Steven; Farese, Robert V.; Gitler, Aaron D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1302-1309</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Amyotrophic lateral sclerosis (ALS) is a devastating neurodegenerative disease primarily affecting motor neurons.  Mutations in the gene encoding TDP-43 cause some forms of the disease, and cytoplasmic TDP-43 aggregates accumulate in degenerating neurons of most individuals with ALS.  Thus, strategies aimed at targeting the toxicity of cytoplasmic TDP-43 aggregates may be effective.  Here, we report results from two genome-wide loss-of-function TDP-43 toxicity suppressor screens in yeast.  The strongest suppressor of TDP-43 toxicity was deletion of DBR1, which encodes an RNA lariat debranching enzyme.  We show that, in the absence of Dbr1 enzymic activity, intronic lariats accumulate in the cytoplasm and likely act as decoys to sequester TDP-43, preventing it from interfering with essential cellular RNAs and RNA-binding proteins.  Knockdown of Dbr1 in a human neuronal cell line or in primary rat neurons is also sufficient to rescue TDP-43 toxicity.  Our findings provide insight into TDP-43-mediated cytotoxicity and suggest that decreasing Dbr1 activity could be a potential therapeutic approach for ALS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHPDqiwEnEcbVg90H21EOLACvtfcHk0ljKY58tqURJew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFOmt7jI&md5=571f3fa8d80cd5b9a06fc6d9426e782e</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1038%2Fng.2434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2434%26sid%3Dliteratum%253Aachs%26aulast%3DArmakola%26aufirst%3DM.%26aulast%3DHiggins%26aufirst%3DM.%2BJ.%26aulast%3DFigley%26aufirst%3DM.%2BD.%26aulast%3DBarmada%26aufirst%3DS.%2BJ.%26aulast%3DScarborough%26aufirst%3DE.%2BA.%26aulast%3DDiaz%26aufirst%3DZ.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DShorter%26aufirst%3DJ.%26aulast%3DKrogan%26aufirst%3DN.%2BJ.%26aulast%3DFinkbeiner%26aufirst%3DS.%26aulast%3DFarese%26aufirst%3DR.%2BV.%26aulast%3DGitler%26aufirst%3DA.%2BD.%26atitle%3DInhibition%2520of%2520Rna%2520Lariat%2520Debranching%2520Enzyme%2520Suppresses%2520Tdp-43%2520Toxicity%2520in%2520Als%2520Disease%2520Models%26jtitle%3DNat.%2520Genet.%26date%3D2012%26volume%3D44%26spage%3D1302%26epage%3D1309%26doi%3D10.1038%2Fng.2434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giaever, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nislow, C.</span></span> <span> </span><span class="NLM_article-title">The Yeast Deletion Collection: A Decade of Functional Genomics</span>. <i>Genetics</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>197</i></span>,  <span class="NLM_fpage">451</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1534/genetics.114.161620</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1534%2Fgenetics.114.161620" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=24939991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=197&publication_year=2014&pages=451-465&author=G.+Giaeverauthor=C.+Nislow&title=The+Yeast+Deletion+Collection%3A+A+Decade+of+Functional+Genomics&doi=10.1534%2Fgenetics.114.161620"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">The yeast deletion collection: a decade of functional genomics</span></div><div class="casAuthors">Giaever, Guri; Nislow, Corey</div><div class="citationInfo"><span class="NLM_cas:title">Genetics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">197</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">451-465, 15</span>CODEN:
                <span class="NLM_cas:coden">GENTAE</span>;
        ISSN:<span class="NLM_cas:issn">0016-6731</span>.
    
            (<span class="NLM_cas:orgname">Genetics Society of America</span>)
        </div><div class="casAbstract">A review.  The yeast deletion collections comprise >21,000 mutant strains that carry precise start-to-stop deletions of ∼6000 open reading frames.  This collection includes heterozygous and homozygous diploids, and haploids of both MATa and MATα mating types.  The yeast deletion collection, or yeast knockout (YKO) set, represents the first and only complete, systematically constructed deletion collection available for any organism.  Conceived during the Saccharomyces cerevisiae sequencing project, work on the project began in 1998 and was completed in 2002.  The YKO strains have been used in numerous labs. in >1000 genome-wide screens.  This landmark genome project has inspired development of numerous genome-wide technologies in organisms from yeast to man.  Notable spinoff technologies include synthetic genetic array and HIPHOP chemogenomics.  In this retrospective, we briefly describe the yeast deletion project and some of its most noteworthy biol. contributions and the impact that these collections have had on the yeast research community and on genomics in general.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmljNCsYgzn7Vg90H21EOLACvtfcHk0ljKY58tqURJew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLrN&md5=f33c484976824362db6d4f5ca13ad38f</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1534%2Fgenetics.114.161620&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1534%252Fgenetics.114.161620%26sid%3Dliteratum%253Aachs%26aulast%3DGiaever%26aufirst%3DG.%26aulast%3DNislow%26aufirst%3DC.%26atitle%3DThe%2520Yeast%2520Deletion%2520Collection%253A%2520A%2520Decade%2520of%2520Functional%2520Genomics%26jtitle%3DGenetics%26date%3D2014%26volume%3D197%26spage%3D451%26epage%3D465%26doi%3D10.1534%2Fgenetics.114.161620" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elden, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen-Plotkin, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armakola, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geser, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clay-Falcone, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCluskey, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juhr, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruber, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rub, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auburger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojanowski, J. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Deerlin, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonini, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gitler, A. D.</span></span> <span> </span><span class="NLM_article-title">Ataxin-2 Intermediate-Length Polyglutamine Expansions Are Associated with Increased Risk for Als</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>466</i></span>,  <span class="NLM_fpage">1069</span>– <span class="NLM_lpage">1075</span>, <span class="refDoi"> DOI: 10.1038/nature09320</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnature09320" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=20740007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVOnurvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2010&pages=1069-1075&author=A.+C.+Eldenauthor=H.+J.+Kimauthor=M.+P.+Hartauthor=A.+S.+Chen-Plotkinauthor=B.+S.+Johnsonauthor=X.+Fangauthor=M.+Armakolaauthor=F.+Geserauthor=R.+Greeneauthor=M.+M.+Luauthor=A.+Padmanabhanauthor=D.+Clay-Falconeauthor=L.+McCluskeyauthor=L.+Elmanauthor=D.+Juhrauthor=P.+J.+Gruberauthor=U.+Rubauthor=G.+Auburgerauthor=J.+Q.+Trojanowskiauthor=V.+M.+Leeauthor=V.+M.+Van+Deerlinauthor=N.+M.+Boniniauthor=A.+D.+Gitler&title=Ataxin-2+Intermediate-Length+Polyglutamine+Expansions+Are+Associated+with+Increased+Risk+for+Als&doi=10.1038%2Fnature09320"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Ataxin-2 intermediate-length polyglutamine expansions are associated with increased risk for ALS</span></div><div class="casAuthors">Elden, Andrew C.; Kim, Hyung-Jun; Hart, Michael P.; Chen-Plotkin, Alice S.; Johnson, Brian S.; Fang, Xiaodong; Armakola, Maria; Geser, Felix; Greene, Robert; Lu, Min Min; Padmanabhan, Arun; Clay-Falcone, Dana; McCluskey, Leo; Elman, Lauren; Juhr, Denise; Gruber, Peter J.; Rb, Udo; Auburger, Georg; Trojanowski, John Q.; Lee, Virginia M.-Y.; Van Deerlin, Vivianna M.; Bonini, Nancy M.; Gitler, Aaron D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">466</span>
        (<span class="NLM_cas:issue">7310</span>),
    <span class="NLM_cas:pages">1069-1075</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The causes of amyotrophic lateral sclerosis (ALS), a devastating human neurodegenerative disease, are poorly understood, although the protein TDP-43 has been suggested to have a crit. role in disease pathogenesis.  Here we show that ataxin 2 (ATXN2), a polyglutamine (polyQ) protein mutated in spinocerebellar ataxia type 2, is a potent modifier of TDP-43 toxicity in animal and cellular models.  ATXN2 and TDP-43 assoc. in a complex that depends on RNA.  In spinal cord neurons of ALS patients, ATXN2 is abnormally localized; likewise, TDP-43 shows mislocalization in spinocerebellar ataxia type 2.  To assess the involvement of ATXN2 in ALS, we analyzed the length of the polyQ repeat in the ATXN2 gene in 915 ALS patients.  We found that intermediate-length polyQ expansions (2733 glutamines) in ATXN2 were significantly assocd. with ALS.  These data establish ATXN2 as a relatively common ALS susceptibility gene.  Furthermore, these findings indicate that the TDP-43ATXN2 interaction may be a promising target for therapeutic intervention in ALS and other TDP-43 proteinopathies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopssdThk2l_7Vg90H21EOLACvtfcHk0ljKY58tqURJew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVOnurvF&md5=2de17a182a5b150f81fbdece00813bcd</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1038%2Fnature09320&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09320%26sid%3Dliteratum%253Aachs%26aulast%3DElden%26aufirst%3DA.%2BC.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DHart%26aufirst%3DM.%2BP.%26aulast%3DChen-Plotkin%26aufirst%3DA.%2BS.%26aulast%3DJohnson%26aufirst%3DB.%2BS.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DArmakola%26aufirst%3DM.%26aulast%3DGeser%26aufirst%3DF.%26aulast%3DGreene%26aufirst%3DR.%26aulast%3DLu%26aufirst%3DM.%2BM.%26aulast%3DPadmanabhan%26aufirst%3DA.%26aulast%3DClay-Falcone%26aufirst%3DD.%26aulast%3DMcCluskey%26aufirst%3DL.%26aulast%3DElman%26aufirst%3DL.%26aulast%3DJuhr%26aufirst%3DD.%26aulast%3DGruber%26aufirst%3DP.%2BJ.%26aulast%3DRub%26aufirst%3DU.%26aulast%3DAuburger%26aufirst%3DG.%26aulast%3DTrojanowski%26aufirst%3DJ.%2BQ.%26aulast%3DLee%26aufirst%3DV.%2BM.%26aulast%3DVan%2BDeerlin%26aufirst%3DV.%2BM.%26aulast%3DBonini%26aufirst%3DN.%2BM.%26aulast%3DGitler%26aufirst%3DA.%2BD.%26atitle%3DAtaxin-2%2520Intermediate-Length%2520Polyglutamine%2520Expansions%2520Are%2520Associated%2520with%2520Increased%2520Risk%2520for%2520Als%26jtitle%3DNature%26date%3D2010%26volume%3D466%26spage%3D1069%26epage%3D1075%26doi%3D10.1038%2Fnature09320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Armakola, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gitler, A. D.</span></span> <span> </span><span class="NLM_article-title">Tdp-43 Toxicity in Yeast</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">238</span>– <span class="NLM_lpage">245</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2010.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.ymeth.2010.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=21115123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFGisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2011&pages=238-245&author=M.+Armakolaauthor=M.+P.+Hartauthor=A.+D.+Gitler&title=Tdp-43+Toxicity+in+Yeast&doi=10.1016%2Fj.ymeth.2010.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">TDP-43 toxicity in yeast</span></div><div class="casAuthors">Armakola, Maria; Hart, Michael P.; Gitler, Aaron D.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">238-245</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The budding yeast Saccharomyces cerevisiae is an emerging tool for investigating the mol. pathways that underpin several human neurodegenerative disorders assocd. with protein misfolding.  Amyotrophic lateral sclerosis (ALS) is a devastating adult onset neurodegenerative disease primarily affecting motor neurons.  The protein TDP-43 has recently been demonstrated to play an important role in the disease, however, the mechanisms by which TDP-43 contributes to pathogenesis are unclear.  To explore the mechanistic details that result in aberrant accumulation of TDP-43 and to discover potential strategies for therapeutic intervention, we employed a yeast TDP-43 proteinopathy model system.  These studies allowed us to det. the regions of TDP-43 required for aggregation and toxicity and to define the effects of ALS-linked mutant forms of TDP-43.  We have also been able to harness the power of yeast genetics to identify potent modifiers of TDP-43 toxicity using high-throughput yeast genetic screens.  Here, we describe the methods and approaches that we have used in order to gain insight into TDP-43 biol. and its role in disease.  These approaches are readily adaptable to other neurodegenerative disease proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpekRGMTYUokbVg90H21EOLACvtfcHk0lhTEtWGPH2Huw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFGisLc%253D&md5=786a16a829e3648867537ea8ce10aed8</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2010.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2010.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DArmakola%26aufirst%3DM.%26aulast%3DHart%26aufirst%3DM.%2BP.%26aulast%3DGitler%26aufirst%3DA.%2BD.%26atitle%3DTdp-43%2520Toxicity%2520in%2520Yeast%26jtitle%3DMethods%26date%3D2011%26volume%3D53%26spage%3D238%26epage%3D245%26doi%3D10.1016%2Fj.ymeth.2010.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Becker, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bieri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knowles, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafar-Nejad, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soriano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auburger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulst, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rigo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gitler, A. D.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Reduction of Ataxin-2 Extends Lifespan and Reduces Pathology in Tdp-43 Mice</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>544</i></span>,  <span class="NLM_fpage">367</span>– <span class="NLM_lpage">371</span>, <span class="refDoi"> DOI: 10.1038/nature22038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnature22038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28405022" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvVOjsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=544&publication_year=2017&pages=367-371&author=L.+A.+Beckerauthor=B.+Huangauthor=G.+Bieriauthor=R.+Maauthor=D.+A.+Knowlesauthor=P.+Jafar-Nejadauthor=J.+Messingauthor=H.+J.+Kimauthor=A.+Sorianoauthor=G.+Auburgerauthor=S.+M.+Pulstauthor=J.+P.+Taylorauthor=F.+Rigoauthor=A.+D.+Gitler&title=Therapeutic+Reduction+of+Ataxin-2+Extends+Lifespan+and+Reduces+Pathology+in+Tdp-43+Mice&doi=10.1038%2Fnature22038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic reduction of ataxin-2 extends lifespan and reduces pathology in TDP-43 mice</span></div><div class="casAuthors">Becker, Lindsay A.; Huang, Brenda; Bieri, Gregor; Ma, Rosanna; Knowles, David A.; Jafar-Nejad, Paymaan; Messing, James; Kim, Hong Joo; Soriano, Armand; Auburger, Georg; Pulst, Stefan M.; Taylor, J. Paul; Rigo, Frank; Gitler, Aaron D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">544</span>
        (<span class="NLM_cas:issue">7650</span>),
    <span class="NLM_cas:pages">367-371</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Amyotrophic lateral sclerosis (ALS) is a rapidly progressing neurodegenerative disease that is characterized by motor neuron loss and that leads to paralysis and death 2-5 years after disease onset.  Nearly all patients with ALS have aggregates of the RNA-binding protein TDP-43 in their brains and spinal cords, and rare mutations in the gene encoding TDP-43 can cause ALS.  There are no effective TDP-43-directed therapies for ALS or related TDP-43 proteinopathies, such as frontotemporal dementia.  Antisense oligonucleotides (ASOs) and RNA-interference approaches are emerging as attractive therapeutic strategies in neurol. diseases.  Indeed, treatment of a rat model of inherited ALS (caused by a mutation in Sod1) with ASOs against Sod1 has been shown to substantially slow disease progression.  However, as SOD1 mutations account for only around 2-5% of ALS cases, addnl. therapeutic strategies are needed.  Silencing TDP-43 itself is probably not appropriate, given its crit. cellular functions.  Here we present a promising alternative therapeutic strategy for ALS that involves targeting ataxin-2.  A decrease in ataxin-2 suppresses TDP-43 toxicity in yeast and flies, and intermediate-length polyglutamine expansions in the ataxin-2 gene increase risk of ALS.  We used two independent approaches to test whether decreasing ataxin-2 levels could mitigate disease in a mouse model of TDP-43 proteinopathy.  First, we crossed ataxin-2 knockout mice with TDP-43 (also known as TARDBP) transgenic mice.  The decrease in ataxin-2 reduced aggregation of TDP-43, markedly increased survival and improved motor function.  Second, in a more therapeutically applicable approach, we administered ASOs targeting ataxin-2 to the central nervous system of TDP-43 transgenic mice.  This single treatment markedly extended survival.  Because TDP-43 aggregation is a component of nearly all cases of ALS, targeting ataxin-2 could represent a broadly effective therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf0ZIIBhNmhrVg90H21EOLACvtfcHk0lhTEtWGPH2Huw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvVOjsbY%253D&md5=a14750d78aa2e30996bd163a7dec8f05</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1038%2Fnature22038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22038%26sid%3Dliteratum%253Aachs%26aulast%3DBecker%26aufirst%3DL.%2BA.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DBieri%26aufirst%3DG.%26aulast%3DMa%26aufirst%3DR.%26aulast%3DKnowles%26aufirst%3DD.%2BA.%26aulast%3DJafar-Nejad%26aufirst%3DP.%26aulast%3DMessing%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DSoriano%26aufirst%3DA.%26aulast%3DAuburger%26aufirst%3DG.%26aulast%3DPulst%26aufirst%3DS.%2BM.%26aulast%3DTaylor%26aufirst%3DJ.%2BP.%26aulast%3DRigo%26aufirst%3DF.%26aulast%3DGitler%26aufirst%3DA.%2BD.%26atitle%3DTherapeutic%2520Reduction%2520of%2520Ataxin-2%2520Extends%2520Lifespan%2520and%2520Reduces%2520Pathology%2520in%2520Tdp-43%2520Mice%26jtitle%3DNature%26date%3D2017%26volume%3D544%26spage%3D367%26epage%3D371%26doi%3D10.1038%2Fnature22038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boettcher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McManus, M. T.</span></span> <span> </span><span class="NLM_article-title">Choosing the Right Tool for the Job: Rnai, Talen, or Crispr</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">575</span>– <span class="NLM_lpage">585</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.04.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.molcel.2015.04.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26000843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpt1Sns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=575-585&author=M.+Boettcherauthor=M.+T.+McManus&title=Choosing+the+Right+Tool+for+the+Job%3A+Rnai%2C+Talen%2C+or+Crispr&doi=10.1016%2Fj.molcel.2015.04.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Choosing the Right Tool for the Job: RNAi, TALEN, or CRISPR</span></div><div class="casAuthors">Boettcher, Michael; McManus, Michael T.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">575-585</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The most widely used approach for defining gene function is to reduce or completely disrupt its normal expression.  For over a decade, RNAi has ruled the lab, offering a magic bullet to disrupt gene expression in many organisms.  However, new biotechnol. tools-specifically CRISPR-based technologies-have become available and are squeezing out RNAi dominance in mammalian cell studies.  These seemingly competing technologies leave research investigators with the question: "Which technol. should I use in my expt.".  This review offers a practical resource to compare and contrast these technologies, guiding the investigator when and where to use this fantastic array of powerful tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZmgnhFd9v1rVg90H21EOLACvtfcHk0lixtJclArc5gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpt1Sns70%253D&md5=c13cac6bc64dc636d817673787ad7ce7</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.04.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.04.028%26sid%3Dliteratum%253Aachs%26aulast%3DBoettcher%26aufirst%3DM.%26aulast%3DMcManus%26aufirst%3DM.%2BT.%26atitle%3DChoosing%2520the%2520Right%2520Tool%2520for%2520the%2520Job%253A%2520Rnai%252C%2520Talen%252C%2520or%2520Crispr%26jtitle%3DMol.%2520Cell%26date%3D2015%26volume%3D58%26spage%3D575%26epage%3D585%26doi%3D10.1016%2Fj.molcel.2015.04.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jinek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chylinski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fonfara, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doudna, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charpentier, E.</span></span> <span> </span><span class="NLM_article-title">A Programmable Dual-Rna-Guided DNA Endonuclease in Adaptive Bacterial Immunity</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>337</i></span>,  <span class="NLM_fpage">816</span>– <span class="NLM_lpage">821</span>, <span class="refDoi"> DOI: 10.1126/science.1225829</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fscience.1225829" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=22745249" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFOqsb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=337&publication_year=2012&pages=816-821&author=M.+Jinekauthor=K.+Chylinskiauthor=I.+Fonfaraauthor=M.+Hauerauthor=J.+A.+Doudnaauthor=E.+Charpentier&title=A+Programmable+Dual-Rna-Guided+DNA+Endonuclease+in+Adaptive+Bacterial+Immunity&doi=10.1126%2Fscience.1225829"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity</span></div><div class="casAuthors">Jinek, Martin; Chylinski, Krzysztof; Fonfara, Ines; Hauer, Michael; Doudna, Jennifer A.; Charpentier, Emmanuelle</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">337</span>
        (<span class="NLM_cas:issue">6096</span>),
    <span class="NLM_cas:pages">816-821</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-assocd. (Cas) systems provide bacteria and archaea with adaptive immunity against viruses and plasmids by using CRISPR RNAs (crRNAs) to guide the silencing of invading nucleic acids.  We show here that in a subset of these systems, the mature crRNA that is base-paired to trans-activating crRNA (tracrRNA) forms a two-RNA structure that directs the CRISPR-assocd. protein Cas9 to introduce double-stranded (ds) breaks in target DNA.  At sites complementary to the crRNA-guide sequence, the Cas9 HNH nuclease domain cleaves the complementary strand, whereas the Cas9 RuvC-like domain cleaves the noncomplementary strand.  The dual-tracrRNA:crRNA, when engineered as a single RNA chimera, also directs sequence-specific Cas9 dsDNA cleavage.  Our study reveals a family of endonucleases that use dual-RNAs for site-specific DNA cleavage and highlights the potential to exploit the system for RNA-programmable genome editing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMRnSchW9tkbVg90H21EOLACvtfcHk0lixtJclArc5gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFOqsb3L&md5=8ad6d5a9839f62f75a1f805a2c1c83f4</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1126%2Fscience.1225829&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1225829%26sid%3Dliteratum%253Aachs%26aulast%3DJinek%26aufirst%3DM.%26aulast%3DChylinski%26aufirst%3DK.%26aulast%3DFonfara%26aufirst%3DI.%26aulast%3DHauer%26aufirst%3DM.%26aulast%3DDoudna%26aufirst%3DJ.%2BA.%26aulast%3DCharpentier%26aufirst%3DE.%26atitle%3DA%2520Programmable%2520Dual-Rna-Guided%2520DNA%2520Endonuclease%2520in%2520Adaptive%2520Bacterial%2520Immunity%26jtitle%3DScience%26date%3D2012%26volume%3D337%26spage%3D816%26epage%3D821%26doi%3D10.1126%2Fscience.1225829" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haney, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgens, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jovicic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couthouis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ousey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bieri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hertz, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tessier-Lavigne, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichida, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassik, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gitler, A. D.</span></span> <span> </span><span class="NLM_article-title">Crispr-Cas9 Screens in Human Cells and Primary Neurons Identify Modifiers of C9orf72 Dipeptide-Repeat-Protein Toxicity</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">612</span>, <span class="refDoi"> DOI: 10.1038/s41588-018-0070-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fs41588-018-0070-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=29507424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltVyit7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2018&pages=603-612&author=N.+J.+Kramerauthor=M.+S.+Haneyauthor=D.+W.+Morgensauthor=A.+Jovicicauthor=J.+Couthouisauthor=A.+Liauthor=J.+Ouseyauthor=R.+Maauthor=G.+Bieriauthor=C.+K.+Tsuiauthor=Y.+Shiauthor=N.+T.+Hertzauthor=M.+Tessier-Lavigneauthor=J.+K.+Ichidaauthor=M.+C.+Bassikauthor=A.+D.+Gitler&title=Crispr-Cas9+Screens+in+Human+Cells+and+Primary+Neurons+Identify+Modifiers+of+C9orf72+Dipeptide-Repeat-Protein+Toxicity&doi=10.1038%2Fs41588-018-0070-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">CRISPR-Cas9 screens in human cells and primary neurons identify modifiers of C9ORF72 dipeptide-repeat-protein toxicity</span></div><div class="casAuthors">Kramer, Nicholas J.; Haney, Michael S.; Morgens, David W.; Jovicic, Ana; Couthouis, Julien; Li, Amy; Ousey, James; Ma, Rosanna; Bieri, Gregor; Tsui, C. Kimberly; Shi, Yingxiao; Hertz, Nicholas T.; Tessier-Lavigne, Marc; Ichida, Justin K.; Bassik, Michael C.; Gitler, Aaron D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">603-612</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Hexanucleotide-repeat expansions in the C9ORF72 gene are the most common cause of amyotrophic lateral sclerosis and frontotemporal dementia (c9ALS/FTD).  The nucleotide-repeat expansions are translated into dipeptide-repeat (DPR) proteins, which are aggregation prone and may contribute to neurodegeneration.  We used the CRISPR-Cas9 system to perform genome-wide gene-knockout screens for suppressors and enhancers of C9ORF72 DPR toxicity in human cells.  We validated hits by performing secondary CRISPR-Cas9 screens in primary mouse neurons.  We uncovered potent modifiers of DPR toxicity whose gene products function in nucleocytoplasmic transport, the endoplasmic reticulum (ER), proteasome, RNA-processing pathways, and chromatin modification.  One modifier, TMX2, modulated the ER-stress signature elicited by C9ORF72 DPRs in neurons and improved survival of human induced motor neurons from patients with C9ORF72 ALS.  Together, our results demonstrate the promise of CRISPR-Cas9 screens in defining mechanisms of neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpavbRLvjRmTrVg90H21EOLACvtfcHk0lixtJclArc5gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltVyit7c%253D&md5=e043408638fab13b2fb6939961ad00ed</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1038%2Fs41588-018-0070-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41588-018-0070-7%26sid%3Dliteratum%253Aachs%26aulast%3DKramer%26aufirst%3DN.%2BJ.%26aulast%3DHaney%26aufirst%3DM.%2BS.%26aulast%3DMorgens%26aufirst%3DD.%2BW.%26aulast%3DJovicic%26aufirst%3DA.%26aulast%3DCouthouis%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DA.%26aulast%3DOusey%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DR.%26aulast%3DBieri%26aufirst%3DG.%26aulast%3DTsui%26aufirst%3DC.%2BK.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DHertz%26aufirst%3DN.%2BT.%26aulast%3DTessier-Lavigne%26aufirst%3DM.%26aulast%3DIchida%26aufirst%3DJ.%2BK.%26aulast%3DBassik%26aufirst%3DM.%2BC.%26aulast%3DGitler%26aufirst%3DA.%2BD.%26atitle%3DCrispr-Cas9%2520Screens%2520in%2520Human%2520Cells%2520and%2520Primary%2520Neurons%2520Identify%2520Modifiers%2520of%2520C9orf72%2520Dipeptide-Repeat-Protein%2520Toxicity%26jtitle%3DNat.%2520Genet.%26date%3D2018%26volume%3D50%26spage%3D603%26epage%3D612%26doi%3D10.1038%2Fs41588-018-0070-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keam, S. J.</span></span> <span> </span><span class="NLM_article-title">Inotersen: First Global Approval</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">1371</span>– <span class="NLM_lpage">1376</span>, <span class="refDoi"> DOI: 10.1007/s40265-018-0968-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1007%2Fs40265-018-0968-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30120737" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFCku7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2018&pages=1371-1376&author=S.+J.+Keam&title=Inotersen%3A+First+Global+Approval&doi=10.1007%2Fs40265-018-0968-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Inotersen: First Global Approval</span></div><div class="casAuthors">Keam, Susan J.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1371-1376</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Ionis Pharmaceuticals and Akcea Therapeutics have developed inotersen (Tegsedi), an antisense oligonucleotide inhibitor of mutant and wild-type human transthyretin (TTR), for the treatment of hereditary transthyretin amyloidosis (hATTR).  Mutation of the TTR gene results in accumulation of TTR protein fragments as amyloid deposits throughout the organs in patients with hATTR, including the peripheral nervous system and the heart.  Treatment with inotersen, which selectively binds to TTR mRNA, prevents the synthesis of TTR protein in the liver, thus reducing further amyloid deposition throughout the body.  S.c. administration of inotersen significantly reduced neurol. progression and improved health-related quality of life in patients with hATTR and polyneuropathy in a phase III trial.  Based on these results, inotersen was recently approved in the EU for the treatment of stage 1 or 2 polyneuropathy in adult patients with hATTR and is under evaluation in the USA and Canada for a similar indication.  This article summarizes the milestones in the development of inotersen leading to this first approval.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxb4Bx5DMw4rVg90H21EOLACvtfcHk0lixtJclArc5gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFCku7bE&md5=aecfa49c97a998da9692c3900bf14ca8</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1007%2Fs40265-018-0968-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-018-0968-5%26sid%3Dliteratum%253Aachs%26aulast%3DKeam%26aufirst%3DS.%2BJ.%26atitle%3DInotersen%253A%2520First%2520Global%2520Approval%26jtitle%3DDrugs%26date%3D2018%26volume%3D78%26spage%3D1371%26epage%3D1376%26doi%3D10.1007%2Fs40265-018-0968-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span> <span> </span><span class="NLM_article-title">Clinical Advances of Sirna Therapeutics</span>. <i>J. Gene Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>, <span class="NLM_elocation-id">e3097</span> <span class="refDoi"> DOI: 10.1002/jgm.3097</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1002%2Fjgm.3097" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=31069898" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=B.+Huauthor=Y.+Wengauthor=X.+H.+Xiaauthor=X.+J.+Liangauthor=Y.+Huang&title=Clinical+Advances+of+Sirna+Therapeutics&doi=10.1002%2Fjgm.3097"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1002%2Fjgm.3097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjgm.3097%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DB.%26aulast%3DWeng%26aufirst%3DY.%26aulast%3DXia%26aufirst%3DX.%2BH.%26aulast%3DLiang%26aufirst%3DX.%2BJ.%26aulast%3DHuang%26aufirst%3DY.%26atitle%3DClinical%2520Advances%2520of%2520Sirna%2520Therapeutics%26jtitle%3DJ.%2520Gene%2520Med.%26date%3D2019%26doi%3D10.1002%2Fjgm.3097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, C. F.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Antisense Oligonucleotides Are Coming of Age</span>. <i>Annu. Rev. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">321</span>, <span class="refDoi"> DOI: 10.1146/annurev-med-041217-010829</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1146%2Fannurev-med-041217-010829" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30691367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvF2lsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2019&pages=307-321&author=C.+F.+Bennett&title=Therapeutic+Antisense+Oligonucleotides+Are+Coming+of+Age&doi=10.1146%2Fannurev-med-041217-010829"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic Antisense Oligonucleotides Are Coming of Age</span></div><div class="casAuthors">Bennett, C. Frank</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Medicine</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">307-321</span>CODEN:
                <span class="NLM_cas:coden">ARMCAH</span>;
        ISSN:<span class="NLM_cas:issn">0066-4219</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The first published description of therapeutic applications of antisense oligonucleotide (ASO) technol. occurred in the late 1970s and was followed by the founding of com. companies focused on developing antisense therapeutics in the late 1980s.  Since the late 1980s, there has been steady progress in improving the technol. platform, taking advantage of advances in oligonucleotide chem. and formulations as well as increased understanding of the distribution and safety of ASOs.  There are several approved ASO drugs and a broad pipeline in development.  In addn., advances in understanding human disease, including the genetic basis for most monogenic diseases and the availability of the full human genome sequence, have created numerous therapeutic applications for the technol.  I summarize the state of the technol. and highlight how advances in the technol. position ASOs to be an important contributor to future medicines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQKzgrxHUp1LVg90H21EOLACvtfcHk0lis42Ly6ynHBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvF2lsbs%253D&md5=6c7f4a07724b43b769c8014ca3108e6a</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1146%2Fannurev-med-041217-010829&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-med-041217-010829%26sid%3Dliteratum%253Aachs%26aulast%3DBennett%26aufirst%3DC.%2BF.%26atitle%3DTherapeutic%2520Antisense%2520Oligonucleotides%2520Are%2520Coming%2520of%2520Age%26jtitle%3DAnnu.%2520Rev.%2520Med.%26date%3D2019%26volume%3D70%26spage%3D307%26epage%3D321%26doi%3D10.1146%2Fannurev-med-041217-010829" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bostrom, J.</span></span> <span> </span><span class="NLM_article-title">Where Do Recent Small Molecule Clinical Development Candidates Come From?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9442</span>– <span class="NLM_lpage">9468</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00675</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00675" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFKhsb7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9442-9468&author=D.+G.+Brownauthor=J.+Bostrom&title=Where+Do+Recent+Small+Molecule+Clinical+Development+Candidates+Come+From%3F&doi=10.1021%2Facs.jmedchem.8b00675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Where Do Recent Small Molecule Clinical Development Candidates Come From?</span></div><div class="casAuthors">Brown, Dean G.; Bostrom, Jonas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9442-9468</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An anal. of 66 published clin. candidates from Journal of Medicinal Chem. has been conducted to shed light on which lead generation strategies are most frequently employed in identifying drug candidates.  The most frequent lead generation strategy (producing a drug candidate) was based on starting points derived from previously known compds. (43%) followed by random high throughput screening (29%).  The remainder of approaches included focused screening, structure-based drug design (SBDD), fragment-based lead generation (FBLG), and DNA-encoded library screening (DEL).  An anal. of physicochem. properties on the hit-to-clin. pairs shows an av. increase in mol. wt. (ΔMW = +85) but no change in lipophilicity (ΔclogP = -0.2), although exceptions are noted.  The majority (>50%) of clin. candidates were found to be structurally very different from their starting point and were more complex.  Finally, several reports of noncovalent scaffolds modified by a covalent warhead using SBDD approaches are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKVZOYYj87bLVg90H21EOLACvtfcHk0lis42Ly6ynHBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFKhsb7F&md5=211341fa12156e9512650d3352893a5e</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00675%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DBostrom%26aufirst%3DJ.%26atitle%3DWhere%2520Do%2520Recent%2520Small%2520Molecule%2520Clinical%2520Development%2520Candidates%2520Come%2520From%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9442%26epage%3D9468%26doi%3D10.1021%2Facs.jmedchem.8b00675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (Pains) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28Pains%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lievGiG07ho4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528Pains%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wager, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalobos, A.</span></span> <span> </span><span class="NLM_article-title">Moving Beyond Rules: The Development of a Central Nervous System Multiparameter Optimization (Cns Mpo) Approach to Enable Alignment of Druglike Properties</span>. <i>ACS Chem. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">449</span>, <span class="refDoi"> DOI: 10.1021/cn100008c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cn100008c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvVejtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=435-449&author=T.+T.+Wagerauthor=X.+Houauthor=P.+R.+Verhoestauthor=A.+Villalobos&title=Moving+Beyond+Rules%3A+The+Development+of+a+Central+Nervous+System+Multiparameter+Optimization+%28Cns+Mpo%29+Approach+to+Enable+Alignment+of+Druglike+Properties&doi=10.1021%2Fcn100008c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Moving beyond Rules: The Development of a Central Nervous System Multiparameter Optimization (CNS MPO) Approach To Enable Alignment of Druglike Properties</span></div><div class="casAuthors">Wager, Travis T.; Hou, Xinjun; Verhoest, Patrick R.; Villalobos, Anabella</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">435-449</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The interplay among commonly used physicochem. properties in drug design was examd. and utilized to create a prospective design tool focused on the alignment of key druglike attributes.  Using a set of six physicochem. parameters ((a) lipophilicity, calcd. partition coeff. (ClogP); (b) calcd. distribution coeff. at pH = 7.4 (ClogD); (c) mol. wt. (MW); (d) topol. polar surface area (TPSA); (e) no. of hydrogen bond donors (HBD); (f) most basic center (pKa)), a druglikeness central nervous system multiparameter optimization (CNS MPO) algorithm was built and applied to a set of marketed CNS drugs (N = 119) and Pfizer CNS candidates (N = 108), as well as to a large diversity set of Pfizer proprietary compds. (N = 11 303).  The novel CNS MPO algorithm showed that 74% of marketed CNS drugs displayed a high CNS MPO score (MPO desirability score ≥ 4, using a scale of 0-6), in comparison to 60% of the Pfizer CNS candidates.  This anal. suggests that this algorithm could potentially be used to identify compds. with a higher probability of successfully testing hypotheses in the clinic.  In addn., a relationship between an increasing CNS MPO score and alignment of key in vitro attributes of drug discovery (favorable permeability, P-glycoprotein (P-gp) efflux, metabolic stability, and safety) was seen in the marketed CNS drug set, the Pfizer candidate set, and the Pfizer proprietary diversity set.  The CNS MPO scoring function offers advantages over hard cutoffs or utilization of single parameters to optimize structure-activity relationships (SAR) by expanding medicinal chem. design space through a holistic assessment approach.  Based on six physicochem. properties commonly used by medicinal chemists, the CNS MPO function may be used prospectively at the design stage to accelerate the identification of compds. with increased probability of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB3Mvkgm_X5bVg90H21EOLACvtfcHk0lievGiG07ho4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvVejtr8%253D&md5=e94998f65a420be6ee4dec4a987cb983</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1021%2Fcn100008c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcn100008c%26sid%3Dliteratum%253Aachs%26aulast%3DWager%26aufirst%3DT.%2BT.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DVillalobos%26aufirst%3DA.%26atitle%3DMoving%2520Beyond%2520Rules%253A%2520The%2520Development%2520of%2520a%2520Central%2520Nervous%2520System%2520Multiparameter%2520Optimization%2520%2528Cns%2520Mpo%2529%2520Approach%2520to%2520Enable%2520Alignment%2520of%2520Druglike%2520Properties%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2010%26volume%3D1%26spage%3D435%26epage%3D449%26doi%3D10.1021%2Fcn100008c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prunotto, M.</span></span> <span> </span><span class="NLM_article-title">Opportunities and Challenges in Phenotypic Drug Discovery: An Industry Perspective</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">543</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnrd.2017.111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28685762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFelsLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=531-543&author=J.+G.+Moffatauthor=F.+Vincentauthor=J.+A.+Leeauthor=J.+Ederauthor=M.+Prunotto&title=Opportunities+and+Challenges+in+Phenotypic+Drug+Discovery%3A+An+Industry+Perspective&doi=10.1038%2Fnrd.2017.111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Opportunities and challenges in phenotypic drug discovery: an industry perspective</span></div><div class="casAuthors">Moffat, John G.; Vincent, Fabien; Lee, Jonathan A.; Eder, Jorg; Prunotto, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">531-543</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Phenotypic drug discovery (PDD) approaches do not rely on knowledge of the identity of a specific drug target or a hypothesis about its role in disease, in contrast to the target-based strategies that have been widely used in the pharmaceutical industry in the past three decades.  However, in recent years, there has been a resurgence in interest in PDD approaches based on their potential to address the incompletely understood complexity of diseases and their promise of delivering first-in-class drugs, as well as major advances in the tools for cell-based phenotypic screening.  Nevertheless, PDD approaches also have considerable challenges, such as hit validation and target deconvolution.  This article focuses on the lessons learned by researchers engaged in PDD in the pharmaceutical industry and considers the impact of 'omics' knowledge in defining a cellular disease phenotype in the era of precision medicine, introducing the concept of a chain of translatability.  We particularly aim to identify features and areas in which PDD can best deliver value to drug discovery portfolios and can contribute to the identification and the development of novel medicines, and to illustrate the challenges and uncertainties that are assocd. with PDD in order to help set realistic expectations with regard to its benefits and costs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGrD5TsU8_kLVg90H21EOLACvtfcHk0liEBENBifzEMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFelsLrJ&md5=d83196715c879a81d9eabcf544e303db</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.111%26sid%3Dliteratum%253Aachs%26aulast%3DMoffat%26aufirst%3DJ.%2BG.%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DLee%26aufirst%3DJ.%2BA.%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DPrunotto%26aufirst%3DM.%26atitle%3DOpportunities%2520and%2520Challenges%2520in%2520Phenotypic%2520Drug%2520Discovery%253A%2520An%2520Industry%2520Perspective%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D531%26epage%3D543%26doi%3D10.1038%2Fnrd.2017.111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koutsoukas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simms, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirchmair, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bond, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitmore, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, A.</span></span> <span> </span><span class="NLM_article-title">From in Silico Target Prediction to Multi-Target Drug Design: Current Databases, Methods and Applications</span>. <i>J. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">2554</span>– <span class="NLM_lpage">2574</span>, <span class="refDoi"> DOI: 10.1016/j.jprot.2011.05.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.jprot.2011.05.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=21621023" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOjtrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2011&pages=2554-2574&author=A.+Koutsoukasauthor=B.+Simmsauthor=J.+Kirchmairauthor=P.+J.+Bondauthor=A.+V.+Whitmoreauthor=S.+Zimmerauthor=M.+P.+Youngauthor=J.+L.+Jenkinsauthor=M.+Glickauthor=R.+C.+Glenauthor=A.+Bender&title=From+in+Silico+Target+Prediction+to+Multi-Target+Drug+Design%3A+Current+Databases%2C+Methods+and+Applications&doi=10.1016%2Fj.jprot.2011.05.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">From in silico target prediction to multi-target drug design: Current databases, methods and applications</span></div><div class="casAuthors">Koutsoukas, Alexios; Simms, Benjamin; Kirchmair, Johannes; Bond, Peter J.; Whitmore, Alan V.; Zimmer, Steven; Young, Malcolm P.; Jenkins, Jeremy L.; Glick, Meir; Glen, Robert C.; Bender, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteomics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2554-2574</span>CODEN:
                <span class="NLM_cas:coden">JPORFQ</span>;
        ISSN:<span class="NLM_cas:issn">1874-3919</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Given the tremendous growth of bioactivity databases, the use of computational tools to predict protein targets of small mols. has been gaining importance in recent years.  Applications span a wide range, from the designed polypharmacol.' of compds. to mode-of-action anal.  In this review, we firstly survey databases that can be used for ligand-based target prediction and which have grown tremendously in size in the past.  We furthermore outline methods for target prediction that exist, both based on the knowledge of bioactivities from the ligand side and methods that can be applied in situations when a protein structure is known.  Applications of successful in silico target identification attempts are discussed in detail, which were based partly or in whole on computational target predictions in the first instance.  This includes the authors' own experience using target prediction tools, in this case considering phenotypic antibacterial screens and the anal. of high-throughput screening data.  Finally, we will conclude with the prospective application of databases to not only predict, retrospectively, the protein targets of a small mol., but also how to design ligands with desired polypharmacol. in a prospective manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQAhthzL8wiLVg90H21EOLACvtfcHk0liEBENBifzEMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOjtrfE&md5=3fd699639ef3faf347537ef400030dd9</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1016%2Fj.jprot.2011.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jprot.2011.05.011%26sid%3Dliteratum%253Aachs%26aulast%3DKoutsoukas%26aufirst%3DA.%26aulast%3DSimms%26aufirst%3DB.%26aulast%3DKirchmair%26aufirst%3DJ.%26aulast%3DBond%26aufirst%3DP.%2BJ.%26aulast%3DWhitmore%26aufirst%3DA.%2BV.%26aulast%3DZimmer%26aufirst%3DS.%26aulast%3DYoung%26aufirst%3DM.%2BP.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DBender%26aufirst%3DA.%26atitle%3DFrom%2520in%2520Silico%2520Target%2520Prediction%2520to%2520Multi-Target%2520Drug%2520Design%253A%2520Current%2520Databases%252C%2520Methods%2520and%2520Applications%26jtitle%3DJ.%2520Proteomics%26date%3D2011%26volume%3D74%26spage%3D2554%26epage%3D2574%26doi%3D10.1016%2Fj.jprot.2011.05.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dancik, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span> <span> </span><span class="NLM_article-title">Target Identification and Mechanism of Action in Chemical Biology and Drug Discovery</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">232</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnchembio.1199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23508189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFWmsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=232-240&author=M.+Schenoneauthor=V.+Dancikauthor=B.+K.+Wagnerauthor=P.+A.+Clemons&title=Target+Identification+and+Mechanism+of+Action+in+Chemical+Biology+and+Drug+Discovery&doi=10.1038%2Fnchembio.1199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Target identification and mechanism of action in chemical biology and drug discovery</span></div><div class="casAuthors">Schenone, Monica; Dancik, Vlado; Wagner, Bridget K.; Clemons, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">232-240</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Target-identification and mechanism-of-action studies have important roles in small-mol. probe and drug discovery.  Biol. and technol. advances have resulted in the increasing use of cell-based assays to discover new biol. active small mols.  Such studies allow small-mol. action to be tested in a more disease-relevant setting at the outset, but they require follow-up studies to det. the precise protein target or targets responsible for the obsd. phenotype.  Target identification can be approached by direct biochem. methods, genetic interactions or computational inference.  In many cases, however, combinations of approaches may be required to fully characterize on-target and off-target effects and to understand mechanisms of small-mol. action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMV_Mv_6-O-rVg90H21EOLACvtfcHk0liEBENBifzEMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFWmsLs%253D&md5=7aa5556d8dfbf259ab7e60fda1d8fdc2</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1199%26sid%3Dliteratum%253Aachs%26aulast%3DSchenone%26aufirst%3DM.%26aulast%3DDancik%26aufirst%3DV.%26aulast%3DWagner%26aufirst%3DB.%2BK.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26atitle%3DTarget%2520Identification%2520and%2520Mechanism%2520of%2520Action%2520in%2520Chemical%2520Biology%2520and%2520Drug%2520Discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D232%26epage%3D240%26doi%3D10.1038%2Fnchembio.1199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corsello, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peck, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natoli, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gould, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tubelli, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asiedu, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahr, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirschman, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donahue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Julian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wadden, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liberzon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toder, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orzechowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enache, O. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccioni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vrcic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosains, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davison, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ardlie, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenside, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read-Button, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronco, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doench, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bittker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span> <span> </span><span class="NLM_article-title">A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>171</i></span>,  <span class="NLM_fpage">1437</span>– <span class="NLM_lpage">1452</span>, e1417 <span class="refDoi"> DOI: 10.1016/j.cell.2017.10.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.cell.2017.10.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=29195078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWmsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452&author=A.+Subramanianauthor=R.+Narayanauthor=S.+M.+Corselloauthor=D.+D.+Peckauthor=T.+E.+Natoliauthor=X.+Luauthor=J.+Gouldauthor=J.+F.+Davisauthor=A.+A.+Tubelliauthor=J.+K.+Asieduauthor=D.+L.+Lahrauthor=J.+E.+Hirschmanauthor=Z.+Liuauthor=M.+Donahueauthor=B.+Julianauthor=M.+Khanauthor=D.+Waddenauthor=I.+C.+Smithauthor=D.+Lamauthor=A.+Liberzonauthor=C.+Toderauthor=M.+Bagulauthor=M.+Orzechowskiauthor=O.+M.+Enacheauthor=F.+Piccioniauthor=S.+A.+Johnsonauthor=N.+J.+Lyonsauthor=A.+H.+Bergerauthor=A.+F.+Shamjiauthor=A.+N.+Brooksauthor=A.+Vrcicauthor=C.+Flynnauthor=J.+Rosainsauthor=D.+Y.+Takedaauthor=R.+Huauthor=D.+Davisonauthor=J.+Lambauthor=K.+Ardlieauthor=L.+Hogstromauthor=P.+Greensideauthor=N.+S.+Grayauthor=P.+A.+Clemonsauthor=S.+Silverauthor=X.+Wuauthor=W.+N.+Zhaoauthor=W.+Read-Buttonauthor=X.+Wuauthor=S.+J.+Haggartyauthor=L.+V.+Roncoauthor=J.+S.+Boehmauthor=S.+L.+Schreiberauthor=J.+G.+Doenchauthor=J.+A.+Bittkerauthor=D.+E.+Rootauthor=B.+Wongauthor=T.+R.+Golub&title=A+Next+Generation+Connectivity+Map%3A+L1000+Platform+and+the+First+1%2C000%2C000+Profiles&doi=10.1016%2Fj.cell.2017.10.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">A next generation connectivity map: L1000 platform and the first 1,000,000 profiles</span></div><div class="casAuthors">Subramanian, Aravind; Narayan, Rajiv; Corsello, Steven M.; Peck, David D.; Natoli, Ted E.; Lu, Xiaodong; Gould, Joshua; Davis, John F.; Tubelli, Andrew A.; Asiedu, Jacob K.; Lahr, David L.; Hirschman, Jodi E.; Liu, Zihan; Donahue, Melanie; Julian, Bina; Khan, Mariya; Wadden, David; Smith, Ian C.; Lam, Daniel; Liberzon, Arthur; Toder, Courtney; Bagul, Mukta; Orzechowski, Marek; Enache, Oana M.; Piccioni, Federica; Johnson, Sarah A.; Lyons, Nicholas J.; Berger, Alice H.; Shamji, Alykhan F.; Brooks, Angela N.; Vrcic, Anita; Flynn, Corey; Rosains, Jacqueline; Takeda, David Y.; Hu, Roger; Davison, Desiree; Lamb, Justin; Ardlie, Kristin; Hogstrom, Larson; Greenside, Peyton; Gray, Nathanael S.; Clemons, Paul A.; Silver, Serena; Wu, Xiaoyun; Zhao, Wen-Ning; Read-Button, Willis; Wu, Xiaohua; Haggarty, Stephen J.; Ronco, Lucienne V.; Boehm, Jesse S.; Schreiber, Stuart L.; Doench, John G.; Bittker, Joshua A.; Root, David E.; Wong, Bang; Golub, Todd R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1437-1452.e17</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We previously piloted the concept of a Connectivity Map (CMap), whereby genes, drugs, and disease states are connected by virtue of common gene-expression signatures.  Here, we report more than a 1000-fold scale-up of the CMap as part of the NIH LINCS Consortium, made possible by a new, low-cost, high-throughput reduced representation expression profiling method that we term L1000.  We show that L1000 is highly reproducible, comparable to RNA sequencing, and suitable for computational inference of the expression levels of 81% of non-measured transcripts.  We further show that the expanded CMap can be used to discover mechanism of action of small mols., functionally annotate genetic variants of disease genes, and inform clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIWrWy5IxY-rVg90H21EOLACvtfcHk0lhdOr4zUImYvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWmsbrE&md5=d2b13f2b80fe94021ac03f6a8c859a4a</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.10.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.10.049%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DNarayan%26aufirst%3DR.%26aulast%3DCorsello%26aufirst%3DS.%2BM.%26aulast%3DPeck%26aufirst%3DD.%2BD.%26aulast%3DNatoli%26aufirst%3DT.%2BE.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DGould%26aufirst%3DJ.%26aulast%3DDavis%26aufirst%3DJ.%2BF.%26aulast%3DTubelli%26aufirst%3DA.%2BA.%26aulast%3DAsiedu%26aufirst%3DJ.%2BK.%26aulast%3DLahr%26aufirst%3DD.%2BL.%26aulast%3DHirschman%26aufirst%3DJ.%2BE.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DDonahue%26aufirst%3DM.%26aulast%3DJulian%26aufirst%3DB.%26aulast%3DKhan%26aufirst%3DM.%26aulast%3DWadden%26aufirst%3DD.%26aulast%3DSmith%26aufirst%3DI.%2BC.%26aulast%3DLam%26aufirst%3DD.%26aulast%3DLiberzon%26aufirst%3DA.%26aulast%3DToder%26aufirst%3DC.%26aulast%3DBagul%26aufirst%3DM.%26aulast%3DOrzechowski%26aufirst%3DM.%26aulast%3DEnache%26aufirst%3DO.%2BM.%26aulast%3DPiccioni%26aufirst%3DF.%26aulast%3DJohnson%26aufirst%3DS.%2BA.%26aulast%3DLyons%26aufirst%3DN.%2BJ.%26aulast%3DBerger%26aufirst%3DA.%2BH.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DBrooks%26aufirst%3DA.%2BN.%26aulast%3DVrcic%26aufirst%3DA.%26aulast%3DFlynn%26aufirst%3DC.%26aulast%3DRosains%26aufirst%3DJ.%26aulast%3DTakeda%26aufirst%3DD.%2BY.%26aulast%3DHu%26aufirst%3DR.%26aulast%3DDavison%26aufirst%3DD.%26aulast%3DLamb%26aufirst%3DJ.%26aulast%3DArdlie%26aufirst%3DK.%26aulast%3DHogstrom%26aufirst%3DL.%26aulast%3DGreenside%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DSilver%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DW.%2BN.%26aulast%3DRead-Button%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DRonco%26aufirst%3DL.%2BV.%26aulast%3DBoehm%26aufirst%3DJ.%2BS.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DDoench%26aufirst%3DJ.%2BG.%26aulast%3DBittker%26aufirst%3DJ.%2BA.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DWong%26aufirst%3DB.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26atitle%3DA%2520Next%2520Generation%2520Connectivity%2520Map%253A%2520L1000%2520Platform%2520and%2520the%2520First%25201%252C000%252C000%2520Profiles%26jtitle%3DCell%26date%3D2017%26volume%3D171%26spage%3D1437%26epage%3D1452%26doi%3D10.1016%2Fj.cell.2017.10.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cong, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. M.</span></span> <span> </span><span class="NLM_article-title">Chemical Genetics-Based Target Identification in Drug Discovery</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010611-134639</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1146%2Fannurev-pharmtox-010611-134639" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=21819237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC38XjsV2ntro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=57-78&author=F.+Congauthor=A.+K.+Cheungauthor=S.+M.+Huang&title=Chemical+Genetics-Based+Target+Identification+in+Drug+Discovery&doi=10.1146%2Fannurev-pharmtox-010611-134639"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical genetics-based target identification in drug discovery</span></div><div class="casAuthors">Cong, Feng; Cheung, Atwood K.; Huang, Shih-Min A.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">57-78</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Identification of novel, validated targets remains a top priority in modern drug discovery.  Chem. genetics represents a powerful approach to the discovery of new targets.  Unlike the traditional target-based screen that relies on a predefined, sometimes poorly validated target, a chem. genetics-based phenotypic screen probes the entire mol. signaling pathway in an efficient and unbiased manner for the most drug-sensitive node.  The most significant obstacle assocd. with this approach is identification of the efficacy targets of small-mol. probes.  The huge potential of chem. genetics cannot be realized without the establishment of reliable mechanisms for target identification.  In this article, we describe each essential element of the chem. genetics process, discuss common challenges that the field is facing, and critically review various biochem. and genetics approaches recently developed for target deconvolution.  We also attempt to summarize lessons that we have collectively learned and provide a practical perspective to facilitate the advancement of chem. genetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5yremnJyOOLVg90H21EOLACvtfcHk0lhdOr4zUImYvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjsV2ntro%253D&md5=97752e1589da40955b89293512f6303e</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010611-134639&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010611-134639%26sid%3Dliteratum%253Aachs%26aulast%3DCong%26aufirst%3DF.%26aulast%3DCheung%26aufirst%3DA.%2BK.%26aulast%3DHuang%26aufirst%3DS.%2BM.%26atitle%3DChemical%2520Genetics-Based%2520Target%2520Identification%2520in%2520Drug%2520Discovery%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2012%26volume%3D52%26spage%3D57%26epage%3D78%26doi%3D10.1146%2Fannurev-pharmtox-010611-134639" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tay, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagar</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dautzenberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinz, M.</span></span> <span> </span><span class="NLM_article-title">Unique Microglia Recovery Population Revealed by Single-Cell Rnaseq Following Neurodegeneration</span>. <i>Acta Neuropathol. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">87</span>, <span class="refDoi"> DOI: 10.1186/s40478-018-0584-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1186%2Fs40478-018-0584-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=30185219" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjt1Gnu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=87&author=T.+L.+Tayauthor=+Sagarauthor=J.+Dautzenbergauthor=D.+Grunauthor=M.+Prinz&title=Unique+Microglia+Recovery+Population+Revealed+by+Single-Cell+Rnaseq+Following+Neurodegeneration&doi=10.1186%2Fs40478-018-0584-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Unique microglia recovery population revealed by single-cell RNAseq following neurodegeneration</span></div><div class="casAuthors">Tay, Tuan Leng; Sagar; Dautzenberg, Jana; Gruen, Dominic; Prinz, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Acta Neuropathologica Communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">87/1-87/11</span>CODEN:
                <span class="NLM_cas:coden">ANCCC9</span>;
        ISSN:<span class="NLM_cas:issn">2051-5960</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Microglia are brain immune cells that constantly survey their environment to maintain homeostasis.  Enhanced microglial reactivity and proliferation are typical hallmarks of neurodegenerative diseases.  Whether specific disease-linked microglial subsets exist during the entire course of neurodegeneration, including the recovery phase, is currently unclear.  Taking a single-cell RNA-sequencing approach in a susceptibility gene-free model of nerve injury, we identified a microglial subpopulation that upon acute neurodegeneration shares a conserved gene regulatory profile compared to previously reported chronic and destructive neurodegeneration transgenic mouse models.  Our data also revealed rapid shifts in gene regulation that defined microglial subsets at peak and resoln. of neurodegeneration.  Finally, our discovery of a unique transient microglial subpopulation at the onset of recovery may provide novel targets for modulating microglia mediated restoration of brain health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5TIFKyIY_6LVg90H21EOLACvtfcHk0ljBEaCgcidasg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjt1Gnu7c%253D&md5=aeb726a790fa829d3a2b0cbc4ea9f20b</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1186%2Fs40478-018-0584-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs40478-018-0584-3%26sid%3Dliteratum%253Aachs%26aulast%3DTay%26aufirst%3DT.%2BL.%26aulast%3DSagar%26aulast%3DDautzenberg%26aufirst%3DJ.%26aulast%3DGrun%26aufirst%3DD.%26aulast%3DPrinz%26aufirst%3DM.%26atitle%3DUnique%2520Microglia%2520Recovery%2520Population%2520Revealed%2520by%2520Single-Cell%2520Rnaseq%2520Following%2520Neurodegeneration%26jtitle%3DActa%2520Neuropathol.%2520Commun.%26date%3D2018%26volume%3D6%26spage%3D87%26doi%3D10.1186%2Fs40478-018-0584-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deczkowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keren-Shaul, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colonna, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amit, I.</span></span> <span> </span><span class="NLM_article-title">Disease-Associated Microglia: A Universal Immune Sensor of Neurodegeneration</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>173</i></span>,  <span class="NLM_fpage">1073</span>– <span class="NLM_lpage">1081</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.cell.2018.05.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=29775591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpvVemuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=1073-1081&author=A.+Deczkowskaauthor=H.+Keren-Shaulauthor=A.+Weinerauthor=M.+Colonnaauthor=M.+Schwartzauthor=I.+Amit&title=Disease-Associated+Microglia%3A+A+Universal+Immune+Sensor+of+Neurodegeneration&doi=10.1016%2Fj.cell.2018.05.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Disease-Associated Microglia: A Universal Immune Sensor of Neurodegeneration</span></div><div class="casAuthors">Deczkowska, Aleksandra; Keren-Shaul, Hadas; Weiner, Assaf; Colonna, Marco; Schwartz, Michal; Amit, Ido</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">173</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1073-1081</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  A major challenge in the field of neurodegenerative diseases and brain aging is to identify the body's intrinsic mechanism that could sense the central nervous system (CNS) damage early and protect the brain from neurodegeneration.  Accumulating evidence suggests that disease-assocd. microglia (DAM), a recently identified subset of CNS resident macrophages found at sites of neurodegeneration, might play such a protective role.  Here, we propose that microglia are endowed with a dedicated sensory mechanism, which includes the Trem2 signaling pathway, to detect damage within the CNS in the form of neurodegeneration-assocd. mol. patterns (NAMPs).  Combining data from transcriptional anal. of DAM at single-cell level and from human genome-wide assocn. studies (GWASs), we discuss potential function of different DAM pathways in the diseased brain and outline how manipulating DAM may create new therapeutic opportunities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqusr5xqDAy-7Vg90H21EOLACvtfcHk0ljBEaCgcidasg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpvVemuro%253D&md5=1f10ff8be131c68e10df99f49af41c3c</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DDeczkowska%26aufirst%3DA.%26aulast%3DKeren-Shaul%26aufirst%3DH.%26aulast%3DWeiner%26aufirst%3DA.%26aulast%3DColonna%26aufirst%3DM.%26aulast%3DSchwartz%26aufirst%3DM.%26aulast%3DAmit%26aufirst%3DI.%26atitle%3DDisease-Associated%2520Microglia%253A%2520A%2520Universal%2520Immune%2520Sensor%2520of%2520Neurodegeneration%26jtitle%3DCell%26date%3D2018%26volume%3D173%26spage%3D1073%26epage%3D1081%26doi%3D10.1016%2Fj.cell.2018.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hammitzsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, F. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Mossawi, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Wit, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vecellio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wordsworth, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowness, P.</span></span> <span> </span><span class="NLM_article-title">Cbp30, a Selective Cbp/P300 Bromodomain Inhibitor, Suppresses Human Th17 Responses</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">10768</span>– <span class="NLM_lpage">10773</span>, <span class="refDoi"> DOI: 10.1073/pnas.1501956112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1073%2Fpnas.1501956112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26261308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlSit7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=10768-10773&author=A.+Hammitzschauthor=C.+Tallantauthor=O.+Fedorovauthor=A.+O%E2%80%99Mahonyauthor=P.+E.+Brennanauthor=D.+A.+Hayauthor=F.+O.+Martinezauthor=M.+H.+Al-Mossawiauthor=J.+de+Witauthor=M.+Vecellioauthor=C.+Wellsauthor=P.+Wordsworthauthor=S.+Mullerauthor=S.+Knappauthor=P.+Bowness&title=Cbp30%2C+a+Selective+Cbp%2FP300+Bromodomain+Inhibitor%2C+Suppresses+Human+Th17+Responses&doi=10.1073%2Fpnas.1501956112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">CBP30, a selective CBP/p300 bromodomain inhibitor, suppresses human Th17 responses</span></div><div class="casAuthors">Hammitzsch, Ariane; Tallant, Cynthia; Fedorov, Oleg; O'Mahony, Alison; Brennan, Paul E.; Hay, Duncan A.; Martinez, Fernando O.; Al-Mossawi, M. Hussein; de Wit, Jelle; Vecellio, Matteo; Wells, Christopher; Wordsworth, Paul; Muller, Susanne; Knapp, Stefan; Bowness, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">10768-10773</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Th17 responses are crit. to a variety of human autoimmune diseases, and therapeutic targeting with monoclonal antibodies against IL-17 and IL-23 has shown considerable promise.  Here, the authors report data to support selective bromodomain blockade of the transcriptional coactivators CBP (CREB binding protein) and p300 as an alternative approach to inhibit human Th17 responses.  The authors show that CBP30 has marked mol. specificity for the bromodomains of CBP and p300, compared with 43 other bromodomains.  In unbiased cellular testing on a diverse panel of cultured primary human cells, CBP30 reduced immune cell prodn. of IL-17A and other proinflammatory cytokines.  CBP30 also inhibited IL-17A secretion by Th17 cells from healthy donors and patients with ankylosing spondylitis and psoriatic arthritis.  Transcriptional profiling of human T cells after CBP30 treatment showed a much more restricted effect on gene expression than that obsd. with the pan-BET (bromo and extraterminal domain protein family) bromodomain inhibitor JQ1.  This selective targeting of the CBP/p300 bromodomain by CBP30 will potentially lead to fewer side effects than with the broadly acting epigenetic inhibitors currently in clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4fYjOUV2xW7Vg90H21EOLACvtfcHk0ljBEaCgcidasg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlSit7jI&md5=fb9de88ecdd6441628ae1455ac11b65c</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1501956112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1501956112%26sid%3Dliteratum%253Aachs%26aulast%3DHammitzsch%26aufirst%3DA.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DMartinez%26aufirst%3DF.%2BO.%26aulast%3DAl-Mossawi%26aufirst%3DM.%2BH.%26aulast%3Dde%2BWit%26aufirst%3DJ.%26aulast%3DVecellio%26aufirst%3DM.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DWordsworth%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBowness%26aufirst%3DP.%26atitle%3DCbp30%252C%2520a%2520Selective%2520Cbp%252FP300%2520Bromodomain%2520Inhibitor%252C%2520Suppresses%2520Human%2520Th17%2520Responses%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D10768%26epage%3D10773%26doi%3D10.1073%2Fpnas.1501956112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span> <span> </span><span class="NLM_article-title">Identifying Compound Efficacy Targets in Phenotypic Drug Discovery</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">82</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2015.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.drudis.2015.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26272035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCisrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=82-89&author=M.+Schirleauthor=J.+L.+Jenkins&title=Identifying+Compound+Efficacy+Targets+in+Phenotypic+Drug+Discovery&doi=10.1016%2Fj.drudis.2015.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying compound efficacy targets in phenotypic drug discovery</span></div><div class="casAuthors">Schirle, Markus; Jenkins, Jeremy L.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">82-89</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The identification of the efficacy target(s) for hits from phenotypic compd. screens remains a key step to progress compds. into drug development.  In addn. to efficacy targets, the characterization of epistatic proteins influencing compd. activity often facilitates the elucidation of the underlying mechanism of action; and, further, early detn. of off-targets that cause potentially unwanted secondary phenotypes helps in assessing potential liabilities.  This short review discusses the most important technologies currently available for characterizing the direct and indirect target space of bioactive compds. following phenotypic screening.  We present a comprehensive strategy employing complementary approaches to balance individual technol. strengths and weaknesses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNnxrH8t-we7Vg90H21EOLACvtfcHk0lghYEzBKOQpww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCisrvF&md5=b373d239d72058a1b826ae1550bb4693</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26atitle%3DIdentifying%2520Compound%2520Efficacy%2520Targets%2520in%2520Phenotypic%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D82%26epage%3D89%26doi%3D10.1016%2Fj.drudis.2015.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laraia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">Natural Product Inspired Compound Collections: Evolutionary Principle, Chemical Synthesis, Phenotypic Screening, and Target Identification</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2017.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.ddtec.2017.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=28647090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC1cjhsFaiug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=75-82&author=L.+Laraiaauthor=H.+Waldmann&title=Natural+Product+Inspired+Compound+Collections%3A+Evolutionary+Principle%2C+Chemical+Synthesis%2C+Phenotypic+Screening%2C+and+Target+Identification&doi=10.1016%2Fj.ddtec.2017.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Natural product inspired compound collections: evolutionary principle, chemical synthesis, phenotypic screening, and target identification</span></div><div class="casAuthors">Laraia Luca; Waldmann Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">75-82</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Natural products have been an excellent and abundant source of therapeutics for many decades.  To expand on their success, and explore areas of chemical space not covered by biosynthesis, the synthesis of natural product-inspired compound collections has emerged as a viable strategy.  Herein we describe the principles behind biology-oriented synthesis and related approaches, the requirements for development of novel chemistry and how phenotypic screens are a very fruitful way to explore the bioactivity of compounds made using these approaches.  Finally, we summarize state-of-the-art techniques to identify the biological targets of any hits identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRF0mGW8Hm3edXdcedVHT5tfW6udTcc2eZe62-Y1vWom7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjhsFaiug%253D%253D&md5=e544a538d8be64334ab0afcac72af5ca</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2017.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2017.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DLaraia%26aufirst%3DL.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DNatural%2520Product%2520Inspired%2520Compound%2520Collections%253A%2520Evolutionary%2520Principle%252C%2520Chemical%2520Synthesis%252C%2520Phenotypic%2520Screening%252C%2520and%2520Target%2520Identification%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2017%26volume%3D23%26spage%3D75%26epage%3D82%26doi%3D10.1016%2Fj.ddtec.2017.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olek, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colinge, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span> <span> </span><span class="NLM_article-title">Proteome-Wide Drug and Metabolite Interaction Mapping by Thermal-Stability Profiling</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1055</span>– <span class="NLM_lpage">1057</span>, <span class="refDoi"> DOI: 10.1038/nmeth.3590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1038%2Fnmeth.3590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26389571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFersL%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=1055-1057&author=K.+V.+Huberauthor=K.+M.+Olekauthor=A.+C.+Mullerauthor=C.+S.+Tanauthor=K.+L.+Bennettauthor=J.+Colingeauthor=G.+Superti-Furga&title=Proteome-Wide+Drug+and+Metabolite+Interaction+Mapping+by+Thermal-Stability+Profiling&doi=10.1038%2Fnmeth.3590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome-wide drug and metabolite interaction mapping by thermal-stability profiling</span></div><div class="casAuthors">Huber, Kilian V. M.; Olek, Karin M.; Mueller, Andre C.; Tan, Chris Soon Heng; Bennett, Keiryn L.; Colinge, Jacques; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1055-1057</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thermal stabilization of proteins after ligand binding provides an efficient means to assess the binding of small mols. to proteins.  We show here that in combination with quant. mass spectrometry, the approach allows for the systematic survey of protein engagement by cellular metabolites and drugs.  We profiled the targets of the drugs methotrexate and (S)-crizotinib and the metabolite 2'3'-cGAMP in intact cells and identified the 2'3'-cGAMP cognate transmembrane receptor STING, involved in immune signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrow66ugdg1oLVg90H21EOLACvtfcHk0lghYEzBKOQpww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFersL%252FI&md5=619a2c08c4f151f4e3ab8bd8fc45c2b6</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.3590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.3590%26sid%3Dliteratum%253Aachs%26aulast%3DHuber%26aufirst%3DK.%2BV.%26aulast%3DOlek%26aufirst%3DK.%2BM.%26aulast%3DMuller%26aufirst%3DA.%2BC.%26aulast%3DTan%26aufirst%3DC.%2BS.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DColinge%26aufirst%3DJ.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DProteome-Wide%2520Drug%2520and%2520Metabolite%2520Interaction%2520Mapping%2520by%2520Thermal-Stability%2520Profiling%26jtitle%3DNat.%2520Methods%26date%3D2015%26volume%3D12%26spage%3D1055%26epage%3D1057%26doi%3D10.1038%2Fnmeth.3590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref115"><div class="reference"><strong class="refLabel"><a href="#ref115" class="rightTabRefNumLink">115</a></strong><div class="NLM_citation" id="rightTab-cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martinez
Molina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1126/science.1233606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1126%2Fscience.1233606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=23828940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC3sjotlOqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=84-87&author=D.+Martinez%0AMolinaauthor=R.+Jafariauthor=M.+Ignatushchenkoauthor=T.+Sekiauthor=E.+A.+Larssonauthor=C.+Danauthor=L.+Sreekumarauthor=Y.+Caoauthor=P.+Nordlund&title=Monitoring+Drug+Target+Engagement+in+Cells+and+Tissues+Using+the+Cellular+Thermal+Shift+Assay&doi=10.1126%2Fscience.1233606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring drug target engagement in cells and tissues using the cellular thermal shift assay</span></div><div class="casAuthors">Martinez Molina Daniel; Jafari Rozbeh; Ignatushchenko Marina; Seki Takahiro; Larsson E Andreas; Dan Chen; Sreekumar Lekshmy; Cao Yihai; Nordlund Par</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6141</span>),
    <span class="NLM_cas:pages">84-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The efficacy of therapeutics is dependent on a drug binding to its cognate target.  Optimization of target engagement by drugs in cells is often challenging, because drug binding cannot be monitored inside cells.  We have developed a method for evaluating drug binding to target proteins in cells and tissue samples.  This cellular thermal shift assay (CETSA) is based on the biophysical principle of ligand-induced thermal stabilization of target proteins.  Using this assay, we validated drug binding for a set of important clinical targets and monitored processes of drug transport and activation, off-target effects and drug resistance in cancer cell lines, as well as drug distribution in tissues.  CETSA is likely to become a valuable tool for the validation and optimization of drug target engagement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRRn5K_mF1rcrHHuJhbyicZfW6udTcc2eYsWLMODIcqT7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sjotlOqtA%253D%253D&md5=a73bb49adb0d4f445223ea6f74fe3c10</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1126%2Fscience.1233606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1233606%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DSreekumar%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DMonitoring%2520Drug%2520Target%2520Engagement%2520in%2520Cells%2520and%2520Tissues%2520Using%2520the%2520Cellular%2520Thermal%2520Shift%2520Assay%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D84%26epage%3D87%26doi%3D10.1126%2Fscience.1233606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref116"><div class="reference"><strong class="refLabel"><a href="#ref116" class="rightTabRefNumLink">116</a></strong><div class="NLM_citation" id="rightTab-cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rees, S.</span></span> <span> </span><span class="NLM_article-title">The Promise of Open Innovation in Drug Discovery: An Industry Perspective</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1835</span>– <span class="NLM_lpage">1838</span>, <span class="refDoi"> DOI: 10.4155/fmc.15.125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.4155%2Ffmc.15.125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=26420551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1KrsL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=1835-1838&author=S.+Rees&title=The+Promise+of+Open+Innovation+in+Drug+Discovery%3A+An+Industry+Perspective&doi=10.4155%2Ffmc.15.125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">The promise of open innovation in drug discovery: an industry perspective</span></div><div class="casAuthors">Rees, Steve</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1835-1838</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyo6y8yseLhrVg90H21EOLACvtfcHk0ljFsgcHDyuYOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1KrsL3F&md5=a3f5209a6733e563df179e85ebe5842a</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.4155%2Ffmc.15.125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.15.125%26sid%3Dliteratum%253Aachs%26aulast%3DRees%26aufirst%3DS.%26atitle%3DThe%2520Promise%2520of%2520Open%2520Innovation%2520in%2520Drug%2520Discovery%253A%2520An%2520Industry%2520Perspective%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2015%26volume%3D7%26spage%3D1835%26epage%3D1838%26doi%3D10.4155%2Ffmc.15.125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref117"><div class="reference"><strong class="refLabel"><a href="#ref117" class="rightTabRefNumLink">117</a></strong><div class="NLM_citation" id="rightTab-cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plump, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ringel, M.</span></span> <span> </span><span class="NLM_article-title">Racing to Define Pharmaceutical R&D External Innovation Models</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">370</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.10.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=10.1016%2Fj.drudis.2014.10.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=25448753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;key=1%3ACAS%3A280%3ADC%252BC2MzisFKhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=361-370&author=L.+Wangauthor=A.+Plumpauthor=M.+Ringel&title=Racing+to+Define+Pharmaceutical+R%26D+External+Innovation+Models&doi=10.1016%2Fj.drudis.2014.10.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Racing to define pharmaceutical R&D external innovation models</span></div><div class="casAuthors">Wang Liangsu; Plump Andrew; Ringel Michael</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">361-70</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The pharmaceutical industry continues to face fundamental challenges because of issues with research and development (R&D) productivity and rising customer expectations.  To lower R&D costs, move beyond me-too therapies, and create more transformative portfolios, pharmaceutical companies are actively capitalizing on external innovation through precompetitive collaboration with academia, cultivation of biotech start-ups, and proactive licensing and acquisitions.  Here, we review the varying innovation strategies used by pharmaceutical companies, compare and contrast these models, and identify the trends in external innovation.  We also discuss factors that influence these external innovation models and propose a preliminary set of metrics that could be used as leading indicators of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSeCY3Qkc1ezIYPsCkJIw00fW6udTcc2eYsWLMODIcqT7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MzisFKhtA%253D%253D&md5=8c13d04ba3a38750269162f27c5af721</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.10.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.10.008%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DPlump%26aufirst%3DA.%26aulast%3DRingel%26aufirst%3DM.%26atitle%3DRacing%2520to%2520Define%2520Pharmaceutical%2520R%2526D%2520External%2520Innovation%2520Models%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D361%26epage%3D370%26doi%3D10.1016%2Fj.drudis.2014.10.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00797&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00797%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-5" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00797" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799a81a6aa23cee","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
