
<html lang="en"     class="pb-page"  data-request-id="83a42760-af40-466f-89fe-134bdba2c6db"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;issue:issue:10.1021/jmcmar.2017.60.issue-16;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.6b01577;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Informing the Selection of Screening Hit Series with in Silico Absorption, Distribution, Metabolism, Excretion, and Toxicity Profiles" /></meta><meta name="dc.Creator" content="John M.  Sanders" /></meta><meta name="dc.Creator" content="Douglas C.  Beshore" /></meta><meta name="dc.Creator" content="J. Christopher  Culberson" /></meta><meta name="dc.Creator" content="James I.  Fells" /></meta><meta name="dc.Creator" content="Jason E.  Imbriglio" /></meta><meta name="dc.Creator" content="Hakan  Gunaydin" /></meta><meta name="dc.Creator" content="Andrew M.  Haidle" /></meta><meta name="dc.Creator" content="Marc  Labroli" /></meta><meta name="dc.Creator" content="Brian E.  Mattioni" /></meta><meta name="dc.Creator" content="Nunzio  Sciammetta" /></meta><meta name="dc.Creator" content="William D.  Shipe" /></meta><meta name="dc.Creator" content="Robert P.  Sheridan" /></meta><meta name="dc.Creator" content="Linda M.  Suen" /></meta><meta name="dc.Creator" content="Andreas  Verras" /></meta><meta name="dc.Creator" content="Abbas  Walji" /></meta><meta name="dc.Creator" content="Elizabeth M.  Joshi" /></meta><meta name="dc.Creator" content="Tjerk  Bueters" /></meta><meta name="dc.Description" content="High-throughput screening (HTS) has enabled millions of compounds to be assessed for biological activity, but challenges remain in the prioritization of hit series. While biological, absorption, di..." /></meta><meta name="Description" content="High-throughput screening (HTS) has enabled millions of compounds to be assessed for biological activity, but challenges remain in the prioritization of hit series. While biological, absorption, di..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 5, 2017" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b01577" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01577" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b01577" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01577" /></link>
        
    
    

<title>Informing the Selection of Screening Hit Series with in Silico Absorption, Distribution, Metabolism, Excretion, and Toxicity Profiles | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b01577" /></meta><meta property="og:title" content="Informing the Selection of Screening Hit Series with in Silico Absorption, Distribution, Metabolism, Excretion, and Toxicity Profiles" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0009.jpeg" /></meta><meta property="og:description" content="High-throughput screening (HTS) has enabled millions of compounds to be assessed for biological activity, but challenges remain in the prioritization of hit series. While biological, absorption, distribution, metabolism, excretion, and toxicity (ADMET), purity, and structural data are routinely used to select chemical matter for further follow-up, the scarcity of historical ADMET data for screening hits limits our understanding of early hit compounds. Herein, we describe a process that utilizes a battery of in-house quantitative structure–activity relationship (QSAR) models to generate in silico ADMET profiles for hit series to enable more complete characterizations of HTS chemical matter. These profiles allow teams to quickly assess hit series for desirable ADMET properties or suspected liabilities that may require significant optimization. Accordingly, these in silico data can direct ADMET experimentation and profoundly impact the progression of hit series. Several prospective examples are presented to substantiate the value of this approach." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b01577"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01577">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b01577&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b01577&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b01577&amp;href=/doi/10.1021/acs.jmedchem.6b01577" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 16</span><span class="cit-fg-pageRange">, 6771-6780</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/16" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.7b00057" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Informing the Selection of Screening Hit Series with in Silico Absorption, Distribution, Metabolism, Excretion, and Toxicity Profiles</span></h1><h2 class="article_header-subtitle"><span class="hlFld-SubTitle">Miniperspective</span></h2><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John+M.++Sanders">John M. Sanders</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>⊥</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-3788-4220" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Douglas+C.++Beshore">Douglas C. Beshore</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=J.+Christopher++Culberson">J. Christopher Culberson</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=James+I.++Fells">James I. Fells</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jason+E.++Imbriglio">Jason E. Imbriglio</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hakan++Gunaydin">Hakan Gunaydin</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+M.++Haidle">Andrew M. Haidle</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Marc++Labroli">Marc Labroli</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Brian+E.++Mattioni">Brian E. Mattioni</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Nunzio++Sciammetta">Nunzio Sciammetta</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=William+D.++Shipe">William D. Shipe</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Robert+P.++Sheridan">Robert P. Sheridan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-6549-1635" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Linda+M.++Suen">Linda M. Suen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andreas++Verras">Andreas Verras</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Abbas++Walji">Abbas Walji</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Elizabeth+M.++Joshi">Elizabeth M. Joshi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-3267-0634" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tjerk++Bueters">Tjerk Bueters</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text"><sup>†</sup>Modeling & Informatics, <sup>‡</sup>Discovery Chemistry, and <sup>§</sup>Pharmacokinetics, Pharmacodynamics, and Drug Metabolism, Merck & Co., Inc., Kenilworth, New Jersey 07065, United States</span></div><div class="corresp-info"><strong>*</strong>J.M.S.: e-mail, <a href="/cdn-cgi/l/email-protection#402a2f282e1f33212e24253233002d2532232b6e232f2d"><span class="__cf_email__" data-cfemail="305a5f585e6f43515e54554243705d5542535b1e535f5d">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>D.C.B.: e-mail, <a href="/cdn-cgi/l/email-protection#12767d67757e73614d7077617a7d6077527f776071793c717d7f"><span class="__cf_email__" data-cfemail="60040f15070c01133f020513080f1205200d0512030b4e030f0d">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01577&amp;href=/doi/10.1021%2Facs.jmedchem.6b01577" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 16</span><span class="cit-pageRange">, 6771–6780</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 18, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>26 October 2016</li><li><span class="item_label"><b>Published</b> online</span>5 May 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 August 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b01577" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01577</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6771%26pageCount%3D10%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJohn%2BM.%2BSanders%252C%2BDouglas%2BC.%2BBeshore%252C%2BJ.%2BChristopher%2BCulberson%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D16%26contentID%3Dacs.jmedchem.6b01577%26title%3DInforming%2Bthe%2BSelection%2Bof%2BScreening%2BHit%2BSeries%2Bwith%2Bin%2BSilico%2BAbsorption%252C%2BDistribution%252C%2BMetabolism%252C%2BExcretion%252C%2Band%2BToxicity%2BProfiles%26numPages%3D10%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6780%26publicationDate%3DAugust%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b01577"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2654</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">12</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b01577" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Informing the Selection of Screening Hit Series with in Silico Absorption, Distribution, Metabolism, Excretion, and Toxicity Profiles&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;M. Sanders&quot;},{&quot;first_name&quot;:&quot;Douglas&quot;,&quot;last_name&quot;:&quot;C. Beshore&quot;},{&quot;first_name&quot;:&quot;J.&quot;,&quot;last_name&quot;:&quot;Christopher Culberson&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;I. Fells&quot;},{&quot;first_name&quot;:&quot;Jason&quot;,&quot;last_name&quot;:&quot;E. Imbriglio&quot;},{&quot;first_name&quot;:&quot;Hakan&quot;,&quot;last_name&quot;:&quot;Gunaydin&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;M. Haidle&quot;},{&quot;first_name&quot;:&quot;Marc&quot;,&quot;last_name&quot;:&quot;Labroli&quot;},{&quot;first_name&quot;:&quot;Brian&quot;,&quot;last_name&quot;:&quot;E. Mattioni&quot;},{&quot;first_name&quot;:&quot;Nunzio&quot;,&quot;last_name&quot;:&quot;Sciammetta&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;D. Shipe&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;P. Sheridan&quot;},{&quot;first_name&quot;:&quot;Linda&quot;,&quot;last_name&quot;:&quot;M. Suen&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;Verras&quot;},{&quot;first_name&quot;:&quot;Abbas&quot;,&quot;last_name&quot;:&quot;Walji&quot;},{&quot;first_name&quot;:&quot;Elizabeth&quot;,&quot;last_name&quot;:&quot;M. Joshi&quot;},{&quot;first_name&quot;:&quot;Tjerk&quot;,&quot;last_name&quot;:&quot;Bueters&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;16&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;6771-6780&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b01577&quot;},&quot;abstract&quot;:&quot;High-throughput screening (HTS) has enabled millions of compounds to be assessed for biological activity, but challenges remain in the prioritization of hit series. While biological, absorption, distribution, metabolism, excretion, and toxicity (ADMET), purity, and structural data are routinely used to select chemical matter for further follow-up, the scarcity of historical ADMET data for screening hits limits our understanding of early hit compounds. Herein, we describe a process that utilizes a battery of in-house quantitative structure–activity relationship (QSAR) models to generate in silico ADMET profiles for hit series to enable more complete characterizations of HTS chemical matter. These profiles allow teams to quickly assess hit series for desirable ADMET properties or suspected liabilities that may require significant optimization. Accordingly, these in silico data can direct ADMET experimentation and profoundly impact the progression of hit series. Several prospective examples are presented to subs&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01577&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01577" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01577&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01577" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01577&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01577" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01577&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01577&amp;href=/doi/10.1021/acs.jmedchem.6b01577" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b01577" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b01577" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (1 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b01577%26sid%3Dliteratum%253Aachs%26pmid%3D28418656%26genre%3Darticle%26aulast%3DSanders%26date%3D2017%26atitle%3DInforming%2Bthe%2BSelection%2Bof%2BScreening%2BHit%2BSeries%2Bwith%2Bin%2BSilico%2BAbsorption%252C%2BDistribution%252C%2BMetabolism%252C%2BExcretion%252C%2Band%2BToxicity%2BProfiles%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D16%26spage%3D6771%26epage%3D6780%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=291465" title="Structure activity relationship">Structure activity relationship</a>,</li><li><a href="/action/doSearch?ConceptID=292202" title="Drug discovery">Drug discovery</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/16" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/jmcmar.2017.60.issue-16/20170824/jmcmar.2017.60.issue-16.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/medium/jm-2016-015772_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">High-throughput screening (HTS) has enabled millions of compounds to be assessed for biological activity, but challenges remain in the prioritization of hit series. While biological, absorption, distribution, metabolism, excretion, and toxicity (ADMET), purity, and structural data are routinely used to select chemical matter for further follow-up, the scarcity of historical ADMET data for screening hits limits our understanding of early hit compounds. Herein, we describe a process that utilizes a battery of in-house quantitative structure–activity relationship (QSAR) models to generate in silico ADMET profiles for hit series to enable more complete characterizations of HTS chemical matter. These profiles allow teams to quickly assess hit series for desirable ADMET properties or suspected liabilities that may require significant optimization. Accordingly, these in silico data can direct ADMET experimentation and profoundly impact the progression of hit series. Several prospective examples are presented to substantiate the value of this approach.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51581" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51581" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">For decades, corporate and academic drug discovery groups have used robotic high throughput screening (HTS) techniques to enable the exploration of large compound collections for specific and desired biochemical and cell-based activities.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> Primary screens of millions of compounds are typically conducted using one concentration and a single replicate (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). Activity in this primary screen is used to advance a significantly smaller subset of the initial library (e.g., typically ∼1%) to a confirmation screen, often carried out at the same concentration as the primary screen but with additional replicates. Beyond the confirmation screen, several replicates of 8- or 10-point concentration response curves are usually generated for an even smaller subset of the collection and these experiments are often performed using additional and orthogonal proteins and formats in order to better understand selectivity, pharmacology, and mechanisms of action.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> This frequently marks the end of the HTS campaign itself and is accompanied by the transfer of a substantial dossier to the discovery program team, which must then decide how to allocate biological and chemical resources in order to identify and prioritize chemical starting points for a drug discovery program.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/medium/jm-2016-015772_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic depicting the work flow and decision points during and after a typical HTS campaign. Orange boxes indicate assays conducted in a high-throughput format; blue boxes indicate assays conducted by the program team; the open arrow indicates the transition from the HTS screening group to the local project team.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The first task the team must complete is biological validation of the hits identified during the screening campaign (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). At our company, this process submits samples of hit molecules that are available within our sample collection to biochemical, cell-based, and biophysical assays of on- and off-target systems and ensures that the activity observed in the HTS campaign arises from bona fide ligand–target interactions. Following biological validation, samples must retain the desired biological activity following repurification or resynthesis in order to be considered chemically validated (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). This represents the first step of post-HTS wet chemistry support and often requires a significant time and resource investment, necessitating a careful selection process before individual compounds receive this level of attention.</div><div class="NLM_p">In order to inform decision-making in the processes described above, HTS screening data are presented to drug discovery teams together with information on compound purity, physical properties, selectivity through prior HTS and non-HTS testing histories, ADMET properties, mechanism of action, and chemical structure-related data. These data sets, while extremely powerful and informative for driving decisions during the biological and chemical validation steps, have not eliminated the need for significant chemistry investments to optimize compounds during lead optimization. In fact, it is nearly always the case that compounds identified through HTS require several years of chemical modification and biological experimentation before a suitable drug candidate is identified.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">One of the challenges of lead optimization is the discovery of a compound that possesses the required combination of potency, selectivity, ADMET, safety, formulation, and manufacturing properties.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Of these criteria, a substantial portion of the optimization effort is often devoted to solving ADMET issues. While understanding the ADMET properties of hit series is widely recognized as being key to a successful optimization campaign,<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6-8)</a> it is often challenging to compare hit series to one another at the earliest stages of lead identification because only a small fraction of the compound collection has any historical measured ADMET data.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> As a result of this, teams use calculated parameters<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> such as molecular weight, log <i>P</i>, polar surface area (PSA), Lipinski’s Ro5,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and their intuition about chemical structure and viability as surrogates for experimental ADMET data during hit prioritization.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> The generation of experimental ADMET data is often delayed until hit series have been expanded and chemically evolved into more “mature” series at which time they frequently reveal the need for extensive ADMET optimization in order to return a pipeline candidate with an acceptable dose and dosing frequency.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><div class="NLM_p">Because resources are finite and chemical space is relatively unlimited, it is imperative that discovery program teams selectively invest available resources in the chemical matter having the highest likelihood of delivering an efficacious and safe drug candidate in a timely fashion.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13, 14)</a> Working from the hypothesis that the good ADMET properties of a promising hit series would be retained during lead optimization, as drug candidates often strongly resemble screening hits, we reasoned that the identification of compound classes possessing favorable overall property profiles during the initial assessment of HTS hits would facilitate and accelerate drug discovery efforts.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></div><div class="NLM_p">Rather than rely on simple physiochemical parameters (i.e., log <i>P</i>), we have taken the approach of using QSAR models for ADMET end points to generate in silico ADMET (isADMET) profiles for hit series in order to inform teams and enable more thorough characterizations of their HTS hits. These profiles are useful for the identification of series with promising ADMET properties, the identification and assessment of tractability around ADMET liabilities, and the compilation of a portfolio of hits with different strengths and weaknesses to hedge against unforeseen complications at later discovery stages. When these profiles are leveraged early in the lead identification process, prior to a discovery team committing significant resources and time to hit series (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>), isADMET profiles can meaningfully alter the perceived tractability of screening hits and, we assert, alter the course of a drug discovery program effort.</div><div class="NLM_p last">To date, there are a limited number of reports describing the in silico characterization of HTS hits using QSAR models built for specific ADMET end points in addition to (or in place of) routinely calculated physical properties,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> but we are unaware of descriptions of more complete in silico profiling. Described herein is a strategy that we have developed to broadly characterize chemical starting points for drug discovery programs by integrating detailed isADMET properties with decision-enabling visualizations to inform discovery teams on series selection and potential solutions to ADMET issues. These visualizations allow for the rapid digestion of large amounts of data and facilitate the identification of promising chemical series and/or liabilities that these series may possess. Several prospective examples are presented to substantiate the value of the approach.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56454" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56454" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Over the years, our company has internally developed a number of QSAR models based on ADMET data obtained for project compounds in our corporate collection. Models are available for human and rat intrinsic clearance (CL<sub>int</sub>), human P-glycoprotein (P-gp)-mediated efflux, and apparent permeability (<i>P</i><sub>app</sub>),<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> in addition to others.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> These models are typically built on highly diverse data sets comprising tens of thousands of unique chemical structures, and we have demonstrated that these models offer good performance for the prospective evaluation of chemical matter (data not shown). While these models are not necessarily quantitatively accurate, they are in all cases able to meaningfully enrich for categorical compound properties. For instance, when predicting the human P-gp efflux ratio for a given compound, a prediction ratio greater than 4 has been shown, through subsequent measurement, to significantly enrich for experimentally determined P-gp efflux substrates (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A). An accurate numerical prediction of the efflux ratio is not necessary for the purposes described herein, and this model offers significant value to teams in the categorical classification of a molecule as a P-gp efflux substrate or nonsubstrate. Although this example is particularly relevant for programs directed at CNS targets, we generally find that the categorical classification afforded by our QSAR models adds significant value during the early phases of a program when measured data are often scarce, prohibitively expensive to generate, or not able to be generated due to a compound being unavailable.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/medium/jm-2016-015772_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Categorization accuracy of human P-gp efflux ratio by the QSAR model for human P-gp efflux ratio (A) and by PSA (B, C). QSAR data shown are prospective prediction values made for 2403 compounds measured between January 2015 and March 2016 for compounds from more than 50 therapeutic programs. Bar charts are colored by experimental P-gp efflux ratio in a MDR1-overexpressing LLC-PK1 cell line for compounds demonstrating good <i>P</i><sub>app</sub> and a lack of endogenous transport in a control LLC-PK1 cell line.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Although physical properties have long been used as surrogates for ADMET parameters, it has been our experience, and that of others,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> that QSAR models can outperform simple physical property-based predictions of a specific end point while providing key structural insights. For example, we have compared categorical predictions made by our human P-gp QSAR model with predictions obtained using PSA calculations (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>), which are routinely applied for this purpose.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> While PSA is useful for enriching for P-gp nonsubstrates and substrates in low- and high-PSA ranges, respectively (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B), the classification accuracy does not rise to the level of performance observed when using our internal P-gp QSAR model unless a wide PSA middle bin of 80–120 Å<sup>2</sup> is adopted (now capturing 1214 compounds, <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>C, versus the 379 compounds contained by the middle bin of predicted P-gp efflux ratios in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A). This enrichment comes at the expense of the numbers of true negative and true positive P-gp substrates falling into low- and high-PSA ranges (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A and <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>C) and limits the utility of using PSA as a single-parameter decision-making tool for this end point. Findings of this nature motivated us to utilize explicit QSAR predictions for ADMET properties instead of exclusively making use of physical property-based approaches.</div><div class="NLM_p">To account for metabolism, we have utilized intrinsic clearance (CL<sub>int</sub>) rather than apparent clearance because correcting for nonspecific binding to microsomal proteins (f<sub>u,mic</sub>) has been shown to improve in vitro–in vivo correlations.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> In evaluating in silico and in vitro CL<sub>int</sub>, the free fraction of compound is incorporated to give an unbound CL<sub>int</sub> using the following relationship:<span class="NLM_disp-formula" id="ueq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/jm-2016-015772_m001.gif" alt="" /></img></span>where CL<sub>int,app</sub> represents the apparent loss of parent from a metabolic stability assessment in microsomes of the appropriate species and <i>f</i><sub>u,mic</sub> is either the measured microsomal unbound fraction or a predicted value. In this work, measured CL<sub>int</sub> values are calculated from measured CL<sub>int,app</sub> and <i>f</i><sub>u,mic</sub> values, whereas predicted CL<sub>int</sub> values are calculated from QSAR predictions of CL<sub>int,app</sub> and QSAR or physical-properties-based predictions of <i>f</i><sub>u,mic</sub>.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> In addition to using QSAR models for microsomal CL<sub>int</sub>, we also make use of predicted rat in vivo intrinsic hepatic clearance, CL<sub>int</sub>, in order to account for compounds whose primary clearance pathway may not be driven by cytochrome P450-mediated oxidative metabolism. In this case, in vivo CL<sub>int</sub> is back-calculated from the well-stirred model of hepatic clearance using QSAR predictions for in vivo plasma clearance (CL<sub>p</sub>) and unbound fraction in plasma (<i>f</i><sub>u,p</sub>) and an assumed blood-to-plasma ratio of unity.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div><div class="NLM_p">We have defined property guidelines in our isADMET battery that we believe are appropriate to inform teams on HTS chemical matter (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Predicted values for compounds are displayed in a heat map, where color gradients are defined by desired and undesired ranges (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) for a given end point (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). We favor visualization of these data via heat maps because they facilitate the interpretation of large data sets and are the standard practice of genome-wide association studies (GWAS) and other large data analysis platforms.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Desirable predictions are rendered in green, moderate or indeterminate predictions are rendered in yellow, and undesirable predictions are rendered in red, though we emphasize categorical enrichment over absolute numerical accuracy. The value of this approach is that it permits dozens of series to be rapidly assessed and qualitatively compared to one another. The gradient values shown in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> were largely based on the experience of the authors and institutional knowledge regarding what property space is generally considered to be desirable or undesirable for each end point. For example, molecules with human in vitro CL<sub>int</sub> values greater than 250 mL·min<sup>–1</sup>·kg<sup>–1</sup> may require significant effort to identify a clinical candidate with adequate bioavailability and half-life to support a once-daily, low dose oral therapy. Similarly, compounds with <i>P</i><sub>app</sub> values exceeding 20 × 10<sup>–6</sup> cm·s<sup>−1</sup> will frequently have unrestricted distribution and complete absorption, whereas values below 10 × 10<sup>–6</sup> cm·s<sup>−1</sup> may be associated with disposition complicated by drug transporters and/or incomplete and variable absorption. Coloring can also reflect experimental variability of the assays. For example, we believe that a predicted P-gp efflux ratio of >5 is more indicative of a bona fide P-gp substrate than is a prediction of ∼3. Accordingly, we use yellow in the color gradient to indicate a middle ground prediction space where there is less certainty and some caution is warranted (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A). In practice, the coloring of these properties requires adjustment by program teams to address program-specific needs. For instance, a CNS program would render small predicted ratios for P-gp efflux as green (desirable) whereas a program seeking minimally CNS-penetrant matter would render these values as red (undesirable). Failure to understand and modify these gradients on a program-specific basis can result in the identification of chemical matter with an undesirable or intractable initial property profile.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/medium/jm-2016-015772_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Representative isADMET heat maps for single hit series displaying potentially desirable (A) and undesirable (B) predicted property profiles. Rows represent individual compounds. In the legend, the red coloring used for in vivo CL<sub>int</sub> predictions of 0 accounts for compounds with predicted CL<sub>p</sub> values that exceed hepatic blood flow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. isADMET Heat Map Color Gradient Set Points Used for Lead Identification Projects</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">in silico model</th><th class="colsep0 rowsep0" align="center">desired value (green)</th><th class="colsep0 rowsep0" align="center">moderate value (yellow)</th><th class="colsep0 rowsep0" align="center">undesired value (red)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>P</i><sub>app</sub> (10<sup>–6</sup> cm·s<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P-gp efflux ratio</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human microsomal CL<sub>int</sub> (mL·min<sup>–1</sup>·kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">125</td><td class="colsep0 rowsep0" align="left">250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat microsomal CL<sub>int</sub> (mL·min<sup>–1</sup>·kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">250</td><td class="colsep0 rowsep0" align="left">500</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat in vivo CL<sub>int</sub> (mL·min<sup>–1</sup>·kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">2500</td><td class="colsep0 rowsep0" align="left">5000</td></tr></tbody></table></div></div><div class="NLM_p">We have also chosen to treat HTS hits as chemical series by grouping similar compounds together and displaying the isADMET profile of a given series as a single heat map. Series can be defined in myriad ways, but prior to biological validation we assemble series using hierarchical clustering of titrated compounds and prior to chemical validation we use topological similarity searching in our corporate collection to expand each HTS hit into a comprehensive series. The motivation for using different methods at different stages is related to convenience; at early stages, the inclusion of isADMET profiles for hits and their neighbors can be burdensome due to a lack of hit attrition (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). At later stages, for example prior to chemical validation, leveraging the full sample repository to build comprehensive heat maps is preferable to restricting heat maps to active molecules uncovered during the screen. In many cases, the molecules that are titrated during an HTS campaign are a subset of active molecules identified in the campaign. In order to match the capacity of subsequent screening steps, representatives of active series are chosen for assay as the goal of the campaign is to identify chemical matter rather than to fully characterize its SAR. Further, our screening collection is a subset of the complete sample repository and a large number of molecules would not contribute to heat maps constructed from screened compounds. By using topological similarity searching to define the chemical space surrounding HTS hits, we capture many of the compounds that would be considered by a chemistry team to be relevant to a decision-making process about a series. This is beneficial even if neighbor molecules are inactive or have unknown activity at the biological target of interest since analogs can inform on the sensitivity of isADMET properties to perturbations of chemical structure and thus the perceived tractability of ADMET properties in each series.</div><div class="NLM_p last">It is our assumption that series that are structurally diverse and well-behaved (i.e., they exhibit desirable predicted properties independent of chemical permutation) will retain desirable ADMET properties during an optimization campaign. Put differently, these series should not require extensive ADMET optimization relative to a series displaying extensive ADMET liabilities at the time of its identification. These concepts are foundational to fragment-based drug discovery, which emphasizes ligand efficiency, and are equally applicable to HTS starting points if attention is paid to ADMET properties.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32501" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32501" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">At the outset of our investigation, we first attempted to address whether or not the use of isADMET profiles would result in the identification or prioritization of different chemical matter than other methods of hit selection. To address this, we retrospectively analyzed the output of several HTS campaigns and compared the decisions that were made by the project teams that prosecuted these campaigns to decisions that would be made on the basis of standard metrics such as the combination of ligand binding efficiency (LBE) and lipophilic ligand efficiency (LLE)<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> or the attractiveness of isADMET profiles.</div><div class="NLM_p">By way of example, project 1 was a drug discovery program that pursued a small molecule therapeutic for the treatment of thrombosis. As the project 1 team analyzed their HTS hits, they used LBE and LLE in their decision-making, and as a result the compounds that were biologically validated all have LBE ≥ 0.26. For the retrospective analysis, we made the following categorical classifications based on LBE and LLE: biologically validated hits with LLE ≥ 3.5 and LBE ≥ 0.26 or LBE ≥ 0.36 and LLE ≥ 2 were classified as “good”; hits with 0.26 ≤ LBE ≤ 0.36 and 2 ≤ LLE < 3 or LBE ≥ 0.36 and LLE < 2 were classified as “moderate”; hits with LBE < 0.36 and LLE < 2 were classified as “poor.” We compared the biologically validated hits, now classified in terms of their LBE and LLE values, with selections made based on the appearance of isADMET profiles. Interpretation of isADMET heat maps is subjective, but our observations are that teams can readily visualize and identify series as being desirable (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A) and undesirable (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B) once they have generally familiarized themselves with the chemical matter; the moderate series generate more discussion in terms of their overall classification. <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a> shows how the hit series from project 1 scored, with the magenta data points representing series that received synthetic effort to enumerate SAR and perform other characterizations. From this analysis, we concluded that the chemical matter prioritized by isADMET profiles partially overlaps the matter prioritized by the team as well as by LBE and LLE.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/medium/jm-2016-015772_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Retrospective comparison of different hit selection methods for project 1. Data points shown are representative molecules from distinct chemical series that were biologically validated. Representative molecules were chosen to indicate the highest level of activity reached by a given series during project 1. Data points are jittered for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">An interesting finding of this exercise is that one of the series chemically pursued by the team in order to build additional SAR and further characterize the series’ properties generated a poor isADMET profile in our retrospective study (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>, top left). At the time this series was pursued, the team had demonstrated in a binding assay that the mechanism of action for the series was differentiated from other hits found in the HTS campaign. As a result of this pharmacology difference, the team was compelled to advance this series in order to preserve diversity in mechanism of action. This example illustrates that there are many criteria beyond physical properties, potency, and ADMET concerns that must be considered when resourcing decisions are made on drug discovery programs. Factors such as pharmacological mechanism of action, intellectual freedom to operate, and other considerations must be taken into account in order to advance an appropriately balanced chemical portfolio.</div><div class="NLM_p">We next describe the incorporation of isADMET profiles into the real-time prosecution of three HTS campaigns from our neuroscience portfolio. Each of these projects sought the discovery of a small molecule therapeutic targeting a protein located in the CNS. In order to address one of the principal challenges of CNS programs, we initially chose to evaluate hit series based upon their propensity for central penetration coupled with attractive ADMET properties, leveraging QSAR models for human P-gp efflux ratio, <i>P</i><sub>app</sub>, human microsomal CL<sub>int</sub>, rat microsomal CL<sub>int</sub>, and rat in vivo CL<sub>int</sub> together with the coloring criteria outlined in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>.</div><div class="NLM_p">Project 2 provided an opportunity to directly and prospectively compare decision-making with and without isADMET profiles. Following biological validation, the program team began selecting compounds for chemical validation using physical properties (e.g., MW, ClogP, PSA, etc.) and on-target potency in addition to subjective characteristics such as chemical attractiveness and synthetic tractability. In addition to the compounds initially chosen for chemical validation, a supplemental group of chemical series was identified based solely on a combination of potency and isADMET profiles generated for the hit molecules and related chemical matter following the method described above. Prior to chemical validation, the project 2 team submitted representative compounds from each of the series chosen by both methods to a panel of in vitro ADMET-related assays (P-gp efflux, <i>P</i><sub>app</sub>, and microsomal CL<sub>int</sub>) to confirm their isADMET profiles. As shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>, most of the series identified by both approaches exhibited in vitro ADMET properties that were encouraging for hit series (low CL<sub>int</sub> and <i>P</i><sub>app</sub> and non-efflux substrates of P-gp). This finding validates the prospective application of in silico models for the identification of compounds with good overall in vitro ADMET-related properties. That is, our QSAR models prospectively identified compounds with desirable properties, reinforcing the validation work shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>. More importantly, compounds identified for their performance in isADMET models may have little overlap with compounds prioritized by other considerations and provide additional chemical starting points, as found with project 1.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/medium/jm-2016-015772_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Plot of human microsomal CL<sub>int</sub> vs human P-gp efflux ratio for project 2 hits. Each data point reflects a representative molecule chosen to characterize a unique chemical series. Blue data points represent compounds chosen by the chemistry team for chemical validation; squares represent compounds chosen for their isADMET properties. Note that there are several overlapping compounds identified as blue squares.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In the case of project 2, we were able to generate enthusiasm for series that were not initially chosen by the team by advancing compounds to our in vitro ADMET assays prior to chemical validation. Consideration of isADMET data at this point can alter the perceived risks or attributes of a chemical series and alter decision-making such that series are progressed or halted based on their isADMET profiles. Because in vitro ADMET assays require less material than does the chemical validation process, we were able to leverage isADMET profiles to collect additional data for series that were not initially chosen for chemical validation. Compounds demonstrating measured properties consistent with good ADMET starting points were then viewed with a heightened interest that warranted resynthesis or, at a minimum, the submission of related molecules to the primary assay in order to increase our understanding of the series. In the case of project 2, perceptions of chemical matter fitness were altered by this approach and chemistry was initiated for several series in order to perform chemical validation and expand their SAR following in vitro confirmation of favorable isADMET properties.</div><div class="NLM_p">An interesting case that emerged from project 2 was the identification of a “series” containing two compounds with an attractive, albeit limited, isADMET profile due to the absence of a significant body of related chemical matter in our sample collection (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>A). Following in vitro confirmation of the isADMET properties for one of the two members of series 1, the team virtually enumerated a small library of prophetic compounds to more thoroughly characterize the series from an isADMET perspective. One of the interesting results from this exercise was the observation that some substituents, such as an unsubstituted phenyl group, consistently enriched for high CL<sub>int</sub> predictions as indicated by the bands of orange and red in the human and rat microsomal CL<sub>int</sub> columns as well as the rat in vivo CL<sub>int</sub> column (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>B). Derivatives containing an oxazole in place of a phenyl group at one of the positions in the core resulted in systematic lowering of CL<sub>int</sub> predictions (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>B), consistent with changes in lipophilicity. From this, we were able to demonstrate that the chemical space surrounding the series was generally consistent with favorable isADMET properties (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>B). We note that in the absence of these isADMET profiles, this series would have been viewed as being “high risk” due to its limited size and likely would have been deprioritized accordingly. As such, we expect virtual libraries like that described for project 2 will become more frequently leveraged to help teams anticipate how the ADMET properties of a series may evolve. Additionally, it is foreseeable that some of the chemical functionalities that frequently appear in SAR exploration, and which consistently result in undesirable ADMET properties (e.g., higher CL<sub>int</sub> predictions in the case of phenyl), will be replaced by functionally similar but more ADMET-sensitive groups (e.g., oxazole in place of phenyl).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/medium/jm-2016-015772_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) isADMET heat map for series 1 from project 2, which only contained two compounds from our sample collection. (B) isADMET heat map for a virtual library enumerated around the core of series 1 from project 2. Rows represent individual compounds. The boxed regions indicate where one R-group was held constant across library members (phenyl and oxazole, as indicated). In the legend, the red coloring used for in vivo CL<sub>int</sub> predictions of 0 accounts for compounds with predicted CL<sub>p</sub> values that exceed hepatic blood flow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Projects 3 and 4 were CNS targets from our neuroscience portfolio. In both cases, following HTS there were tens of biologically validated hits from different chemically distinct series. The projects 3 and 4 teams used isADMET profiles to characterize their chemical matter and to select representative molecules from promising series for in vitro ADMET confirmation. It is worth noting that the compounds submitted for in vitro ADMET characterization were chosen to sample the chemical diversity available within the hit series families while still maintaining good isADMET properties. Additionally, the demonstration of on-target activity was deliberately ignored for a compound to obtain in vitro ADMET characterization. The goal of these studies was to probe how widely the structures of a given series could be modified while still retaining desired ADMET properties, not to screen molecules in order to justify additional resource-intensive studies. Preservation of good in vitro ADMET properties within a chemical series was interpreted as a series having intrinsically good ADMET properties.</div><div class="NLM_p">As described above, the projects 3 and 4 teams only advanced series with encouraging isADMET profiles to in vitro ADMET characterization. At the time, these teams took the strategic position that series which were less promising from an isADMET perspective would be deprioritized and only pursued in the event that the series initially deemed to be promising were shown not to be suitable for lead optimization efforts. Fortunately, both teams were pleased to confirm that a number of series identified through utilization of isADMET profiling displayed good overall in vitro ADMET profiles and revisiting series predicted to be unfavorable was not required.</div><div class="NLM_p">Following experimental characterization of several examples of each chemically validated series, we found that series predicted to be attractive from an isADMET perspective were confirmed to display attractive lead-like properties (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>). Compounds that were prospectively identified by QSAR as having low human microsomal CL<sub>int</sub> values in general exhibited good enrichment for low to moderate CL<sub>int</sub> (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>A and <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>B). Generally speaking, data for compounds fall within 3-fold of their expected values and are categorically consistent with the predictions.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/medium/jm-2016-015772_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Predicted vs measured scatterplots for human microsomal CL<sub>int</sub> (A, project 3; B, project 4), <i>P</i><sub>app</sub> (C, project 3; D, project 4), and human P-gp efflux ratio (E, project 3; F, project 4). Compounds are shaped and colored by series and the colors and shapes differ between projects 3 and 4. CL<sub>int</sub> values are measured in mL·min<sup>–1</sup>·kg<sup>–1</sup>; <i>P</i><sub>app</sub> values are measured in 10<sup>–6</sup> cm·s<sup>–1</sup>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p"><a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>C–F depicts the relationship between experimental and predicted <i>P</i><sub>app</sub> and human P-gp efflux ratios, in which compounds are shaped and colored by series with the line as unity. Measured <i>P</i><sub>app</sub> values were, with few exceptions, categorically consistent with predictions (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>C and <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>D). Similarly, the predictions for P-gp efflux generally led to the identification of non-P-gp substrates or series with tractable P-gp efflux (orange and green-brown triangles, <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>E, and green squares, <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>F) with the systematic exception of one series (blue pentagons, <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>E). Employing this strategy, both teams were able to leverage isADMET predictions to prioritize efforts toward identifying chemical matter that possessed favorable measured ADMET properties.</div><div class="NLM_p">We have also incorporated isADMET predictions into multiparameter optimization (MPO) scoring functions. Teams are frequently confronted with many series possessing similar isADMET profiles, and their differentiation, classification, and assignment to different workflows can become complicated. Converting isADMET profiles to numerical scores facilitates consistency in decision-making while also enabling the sorting of hit series by their isADMET profile scores. For instance, a program team seeking molecules that are peripherally restricted could devise a scoring function that rewards series displaying high predicted P-gp efflux ratios and low predicted rat in vivo CL<sub>int</sub>. Scores averaged over each series would then allow teams to sort their hits to quickly identify hit series that consistently display the desired property profile. This may be particularly beneficial when there are large numbers of compounds that require prioritization, such as prior to biological validation or earlier steps in the screening process (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). However, we caution that overweighting isADMET profiles prior to acquiring broader biological data sets can result in a loss of pharmacological diversity. Biological validation often includes detailed studies important for identifying and maintaining a healthy portfolio of chemical matter (for example, by demonstrating binding to a specific site on a protein in order to affect a particular pharmacodynamic response). In principle, diversity is never irrevocably lost because teams can always return to earlier steps in the screening process to “retriage” with different criteria; in practice, this can be difficult to accomplish for reasons technical and nontechnical in nature.</div><div class="NLM_p">Project 5, an infectious disease program, serves as an example where an MPO scoring function based on isADMET properties resulted in the identification of a series that addressed a key liability. A number of series identified by project 5 displayed a risk associated with the inhibition of hERG<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> based on their isADMET profiles. For one of these series, series 3, nine molecules had historical data from our hERG binding assay and five of these displayed <i>K</i><sub>i</sub> values ranging from 100 nM to 10 μM. Although predictions for this series were not prospectively validated, our QSAR model flagged the series as having a propensity for hERG binding as indicated by the large proportion of molecules predicted to display ∼1 μM affinity in our binding assay (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>A). In order to help the team focus on series predicted to be less prone to hERG binding, an MPO scoring function based on predicted hERG binding affinity and on-target potency and selectivity was employed. Through this approach, the team immediately identified a member from series 4 that was anticipated to display a substantial advantage over other series with respect to hERG binding. Although this series displays undesirable predicted CL<sub>int</sub> properties (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>B), it still attracted attention since hERG binding and inhibition represented a key liability for the team. The potential for improved hERG binding by series 4 is clearly demonstrated in the expanded series predictions, shown in <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>B, by the lower fraction of compounds predicted to exhibit hERG binding. Following experimental characterization in our hERG binding assay, a set of seven compounds from series 4 were shown not to display hERG channel binding (<i>K</i><sub>i</sub> > 60 μM, data not shown), in agreement with our QSAR model predictions.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/medium/jm-2016-015772_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. isADMET heat maps from project 5: (A) series 3, which was consistently predicted to bind to hERG; (B) series 4, which displayed less propensity for predicted hERG binding. Rows represent individual compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In addition to the case examples described above, it is worth noting that other predicted end points can be added to the profiles without significantly complicating their analysis. Leveraging heat maps assists in the digestion of large data sets (in terms of number of rows and columns), and consequently we now include other end points related to safety concerns and drug–drug interaction potential (reversible cytochrome P450 inhibition, CYP3A induction, and hERG, Ca<sub>v</sub>1.2, and Na<sub>v</sub>1.5 ion channel inhibition). We believe that any statistically robust QSAR model which adds data that are relevant to decision-making processes can and should be utilized within this approach.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76617" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76617" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Described herein is an approach to inform drug discovery programs teams of the in silico properties of the molecules identified through screening and incorporate these properties into isADMET profiles. The intent of this review is to raise awareness of and advocate for the utilization of QSAR models in the characterization, prioritization, and execution on chemical matter as early as possible in the discovery process. By incorporating these data in readily digestible heat map visualizations, teams are enabled to rapidly analyze the anticipated ADMET strengths and liabilities of their hit series. The retrospective and prospective application of our approach has shown that leveraging isADMET profiles results in the identification and selection of previously unidentified chemical matter. We continue to develop and refine this approach as part of our hit identification practices and encourage the medicinal chemistry community to consider similar approaches in their drug discovery efforts.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.6b01577" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07420" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07420" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John M. Sanders</span> - <span class="hlFld-Affiliation affiliation">†Modeling & Informatics, ‡Discovery Chemistry, and §Pharmacokinetics,
Pharmacodynamics,
and Drug Metabolism, Merck & Co., Inc., Kenilworth, New Jersey 07065, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3788-4220" title="Orcid link">http://orcid.org/0000-0002-3788-4220</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#650f0a0d0b3a16040b0100171625080017060e4b060a08"><span class="__cf_email__" data-cfemail="650f0a0d0b3a16040b0100171625080017060e4b060a08">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Douglas C. Beshore</span> - <span class="hlFld-Affiliation affiliation">†Modeling & Informatics, ‡Discovery Chemistry, and §Pharmacokinetics,
Pharmacodynamics,
and Drug Metabolism, Merck & Co., Inc., Kenilworth, New Jersey 07065, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#45212a30222924361a2720362d2a372005282037262e6b262a28"><span class="__cf_email__" data-cfemail="dbbfb4aebcb7baa884b9bea8b3b4a9be9bb6bea9b8b0f5b8b4b6">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">J. Christopher Culberson</span> - <span class="hlFld-Affiliation affiliation">†Modeling & Informatics, ‡Discovery Chemistry, and §Pharmacokinetics,
Pharmacodynamics,
and Drug Metabolism, Merck & Co., Inc., Kenilworth, New Jersey 07065, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James I. Fells</span> - <span class="hlFld-Affiliation affiliation">†Modeling & Informatics, ‡Discovery Chemistry, and §Pharmacokinetics,
Pharmacodynamics,
and Drug Metabolism, Merck & Co., Inc., Kenilworth, New Jersey 07065, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jason E. Imbriglio</span> - <span class="hlFld-Affiliation affiliation">†Modeling & Informatics, ‡Discovery Chemistry, and §Pharmacokinetics,
Pharmacodynamics,
and Drug Metabolism, Merck & Co., Inc., Kenilworth, New Jersey 07065, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hakan Gunaydin</span> - <span class="hlFld-Affiliation affiliation">†Modeling & Informatics, ‡Discovery Chemistry, and §Pharmacokinetics,
Pharmacodynamics,
and Drug Metabolism, Merck & Co., Inc., Kenilworth, New Jersey 07065, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew M. Haidle</span> - <span class="hlFld-Affiliation affiliation">†Modeling & Informatics, ‡Discovery Chemistry, and §Pharmacokinetics,
Pharmacodynamics,
and Drug Metabolism, Merck & Co., Inc., Kenilworth, New Jersey 07065, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marc Labroli</span> - <span class="hlFld-Affiliation affiliation">†Modeling & Informatics, ‡Discovery Chemistry, and §Pharmacokinetics,
Pharmacodynamics,
and Drug Metabolism, Merck & Co., Inc., Kenilworth, New Jersey 07065, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brian E. Mattioni</span> - <span class="hlFld-Affiliation affiliation">†Modeling & Informatics, ‡Discovery Chemistry, and §Pharmacokinetics,
Pharmacodynamics,
and Drug Metabolism, Merck & Co., Inc., Kenilworth, New Jersey 07065, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nunzio Sciammetta</span> - <span class="hlFld-Affiliation affiliation">†Modeling & Informatics, ‡Discovery Chemistry, and §Pharmacokinetics,
Pharmacodynamics,
and Drug Metabolism, Merck & Co., Inc., Kenilworth, New Jersey 07065, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William D. Shipe</span> - <span class="hlFld-Affiliation affiliation">†Modeling & Informatics, ‡Discovery Chemistry, and §Pharmacokinetics,
Pharmacodynamics,
and Drug Metabolism, Merck & Co., Inc., Kenilworth, New Jersey 07065, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert P. Sheridan</span> - <span class="hlFld-Affiliation affiliation">†Modeling & Informatics, ‡Discovery Chemistry, and §Pharmacokinetics,
Pharmacodynamics,
and Drug Metabolism, Merck & Co., Inc., Kenilworth, New Jersey 07065, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6549-1635" title="Orcid link">http://orcid.org/0000-0002-6549-1635</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Linda M. Suen</span> - <span class="hlFld-Affiliation affiliation">†Modeling & Informatics, ‡Discovery Chemistry, and §Pharmacokinetics,
Pharmacodynamics,
and Drug Metabolism, Merck & Co., Inc., Kenilworth, New Jersey 07065, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas Verras</span> - <span class="hlFld-Affiliation affiliation">†Modeling & Informatics, ‡Discovery Chemistry, and §Pharmacokinetics,
Pharmacodynamics,
and Drug Metabolism, Merck & Co., Inc., Kenilworth, New Jersey 07065, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Abbas Walji</span> - <span class="hlFld-Affiliation affiliation">†Modeling & Informatics, ‡Discovery Chemistry, and §Pharmacokinetics,
Pharmacodynamics,
and Drug Metabolism, Merck & Co., Inc., Kenilworth, New Jersey 07065, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elizabeth M. Joshi</span> - <span class="hlFld-Affiliation affiliation">†Modeling & Informatics, ‡Discovery Chemistry, and §Pharmacokinetics,
Pharmacodynamics,
and Drug Metabolism, Merck & Co., Inc., Kenilworth, New Jersey 07065, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3267-0634" title="Orcid link">http://orcid.org/0000-0003-3267-0634</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tjerk Bueters</span> - <span class="hlFld-Affiliation affiliation">†Modeling & Informatics, ‡Discovery Chemistry, and §Pharmacokinetics,
Pharmacodynamics,
and Drug Metabolism, Merck & Co., Inc., Kenilworth, New Jersey 07065, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>J.M.S. and D.C.B. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): Authors are current or former employees of Merck & Co., Inc., Whitehouse Station, NJ, U.S., and potentially own stock and/or hold stock options in the Company.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23131" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23131" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="9" class="article__inlineFigure"><h2 class="fig-label">John M. Sanders</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=BIO-d130e1065-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>John M. Sanders</b> is a computational chemist at Merck & Co., Inc. in West Point, PA, U.S. He obtained a B.A. in Chemistry from Colgate University (2000) and a Ph.D. in Chemistry from the University of Illinois under the guidance of Eric Oldfield (2005).</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Douglas C. Beshore</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=BIO-d130e1070-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Douglas C. Beshore</b> is a medicinal chemist at Merck & Co., Inc. in West Point, PA, U.S. He obtained his B.S. in Biochemistry from Albright College (1997) and his Ph.D. from the University of Pennsylvania (2007) under the guidance of Prof. Amos B Smith, III.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="11" class="article__inlineFigure"><h2 class="fig-label">J. Christopher Culberson</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=BIO-d130e1075-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>J. Christopher Culberson</b> joined Merck & Co., Inc. in West Point, PA, U.S., in 1991 after spending 4 years at Nutrasweet designing new sweeteners. As a member of the molecular modeling group, he was directly involved in oxytocin antagonists, HIV-RT inhibitors, and farnesyl transferase inhibitor projects, as well as having an interest in the hERG ion channel. He actively participated in the design and analysis of our screening collection and screening results in order to build a chemical collection that supports drug discovery efforts. He received his Ph.D. degree from University of Florida in Physical Chemistry under the direction of M. C. Zerner and W. L. Luken.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="12" class="article__inlineFigure"><h2 class="fig-label">James I. Fells</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=BIO-d130e1080-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>James I. Fells</b> is a computational chemist in the Modeling & Informatics group at Merck & Co., Inc. in Rahway, NJ, U.S. Prior to joining this company in 2014, he obtained his M.S. and Ph.D. in Organic Chemistry at The University of Memphis, followed by a postdoctoral fellowship in Pharmacology at The University of Tennessee Health Science Center.</p></figure></div><div class="bio" rid="ath5"><figure id="ath5" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Jason E. Imbriglio</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=BIO-d130e1085-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Jason E. Imbriglio</b> received his Ph.D. from the University of Arizona and was an NIH Postdoctoral Fellow at Boston College. Jason joined Merck & Co., Inc. in Rahway, NJ, U.S., in the discovery chemistry group in 2004, where he currently serves as a Director. Jason’s work at this company has included all phases of discovery chemistry, including chemical biology, target validation, lead identification, and lead optimization, across a number of different therapeutic areas. Jason is an alumnus of the University of Massachusetts, Amherst.</p></figure></div><div class="bio" rid="ath6"><figure id="ath6" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Hakan Gunaydin</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=BIO-d130e1090-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Hakan Gunaydin</b> is a computational chemist at Merck & Co., Inc. in Boston, MA, U.S. He earned B.S. and M.S. degrees in Chemistry from Bogazici University in Turkey in 2001 and 2003, respectively. He earned his Ph.D. in Biochemistry and Molecular Biology from UCLA in 2008 working in the Houk laboratory on the modeling of reaction mechanisms and de novo enzyme design. He worked at Amgen, Inc. for 5 years before joining his current company in 2014.</p></figure></div><div class="bio" rid="ath7"><figure id="ath7" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Andrew M. Haidle</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=BIO-d130e1095-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Andrew M. Haidle</b> is a Principal Scientist in Medicinal Chemistry at Merck & Co., Inc. in Boston, MA, U.S. He earned his B. S. in Chemistry and Cell/Molecular Biology in 1998 from the University of Michigan, where he worked on bioorganic chemistry projects involving oligosaccharyltransferase in the laboratory of Prof. James Coward. He then pursued the total synthesis of members of the cytochalasin natural product family under the direction of Prof. Andrew G. Myers at Harvard University, obtaining his Ph.D. from the Department of Chemistry and Chemical Biology in 2004. He has worked on both early and late stage discovery projects in multiple therapeutic areas at Pfizer Inc. and his current company.</p></figure></div><div class="bio" rid="ath8"><figure id="ath8" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Marc Labroli</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=BIO-d130e1100-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Marc Labroli</b> is an Associate Principal Scientist in Medicinal Chemistry at Merck & Co., Inc. in West Point, PA, U.S. He earned a B.S. in Chemistry from Villanova University in 1992, working in the laboratory of Dr. Walter Zajac. He then earned a Ph.D. in Chemistry from The University of Virginia in 1997 under the supervision of Professor Timothy Macdonald, working on topoisomerase inhibitors. Following postdoctoral research studies at the Scripps Research Institute in La Jolla, CA, under the supervision of Professor Dale Boger he joined Schering-Plough in 2000, which merged with his current company in 2009.</p></figure></div><div class="bio" rid="ath9"><figure id="ath9" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Brian E. Mattioni</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=BIO-d130e1105-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Brian E. Mattioni</b> is an ADME scientist at Eli Lilly in Indianapolis, Indiana. He earned his B.S. in Chemistry from Wingate University and Ph.D. in Computational Chemistry from the Pennsylvania State University under Professor Peter Jurs.</p></figure></div><div class="bio" rid="ath10"><figure id="ath10" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Nunzio Sciammetta</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=BIO-d130e1110-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Nunzio Sciammetta</b> is a medicinal chemist at Merck & Co., Inc. in Boston, MA, U.S. He leads a medicinal chemistry team with a focus on enabling synthesis and applying computational design and new chemical technologies to medicinal chemistry projects. His current research interests include parallel medicinal chemistry, property based target design, hit-to-lead discovery, and bRo5 macrocycles in drug discovery. He joined his current company in 2012 and prior to this was a medicinal chemist at the Pfizer Sandwich Laboratories, U.K., and Leo Pharma, Denmark. Nunzio obtained a Ph.D. in Organic Synthesis from the University of Manchester, U.K. (1994), under the supervision of Dr. Andrew C. Regan and undertook postdoctoral studies at the University of Milan, Italy (Prof. C. Scolastico) and Leeds University, U.K. (Prof. R. Grigg).</p></figure></div><div class="bio" rid="ath11"><figure id="ath11" data-index="19" class="article__inlineFigure"><h2 class="fig-label">William D. Shipe</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=BIO-d130e1115-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>William D. Shipe</b> is an Associate Principal Scientist in medicinal chemistry at Merck & Co., Inc. in West Point, PA, U.S. He earned a B.S. in Chemistry with a minor in Mathematics from The Pennsylvania State University in 1999, working in the laboratory of Prof. Raymond L. Funk on the development of new methods for asymmetric induction via chiral auxiliaries and catalysts. He then earned a Ph.D. in Chemistry from The Scripps Research Institute in 2004 under the supervision of Prof. Erik J. Sorensen, completing enantioselective total syntheses of the diterpene guanacastepenes A and E. After leaving Scripps in 2003, he spent 1 year as a visiting graduate student at Princeton University before joining his current company in 2004.</p></figure></div><div class="bio" rid="ath12"><figure id="ath12" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Robert P. Sheridan</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=BIO-d130e1121-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Robert P. Sheridan</b> is a computational chemist and cheminformatics expert at Merck & Co., Inc. in Rahway, NJ, U.S. He earned his Ph.D. in Biochemistry from Princeton University and did his postdoctoral work at Fox Chase Cancer Center and Rutgers University before joining Lederle Laboratories in 1983. He joined his current company in 1991 and concentrated on developing new methods in molecular modeling, first in the field of virtual screening and currently in QSAR.</p></figure></div><div class="bio" rid="ath13"><figure id="ath13" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Linda M. Suen</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=BIO-d130e1126-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Linda M. Suen</b> is a Senior Scientist in medicinal chemistry at Merck & Co., Inc. in West Point, PA, U.S. She earned a B.A. in Biochemistry from Barnard College and Ph.D. in Organic Chemistry from Columbia University under the supervision of Prof. James Leighton.</p></figure></div><div class="bio" rid="ath14"><figure id="ath14" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Andreas Verras</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=BIO-d130e1131-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Andreas Verras</b> is a computational chemist at Merck & Co., Inc. in Kenilworth, NJ, U.S. He earned a B.S. in Chemistry and a B.A. in Creative Writing from Emory University. His Ph.D. was conducted at UCSF in the laboratories of Paul Ortiz de Montellano and Tack Kuntz. Previous to joining his current company he worked at Syngenta Crop Protection in Basel, Switzerland.</p></figure></div><div class="bio" rid="ath15"><figure id="ath15" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Abbas Walji</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=BIO-d130e1136-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Abbas Walji</b> received his Ph.D. from University at Buffalo, The State University of New York and was a postdoctoral scholar at the California Institute of Technology and Princeton University. Abbas joined Merck & Co., Inc. in West Point, PA, U.S., in 2007, where he currently serves as a medicinal chemist in the Discovery Chemistry Modalities group.</p></figure></div><div class="bio" rid="ath16"><figure id="ath16" data-index="24" class="article__inlineFigure"><h2 class="fig-label">Elizabeth M. Joshi</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=BIO-d130e1141-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Elizabeth M. Joshi</b> obtained a B.S. at Mary Washington College, followed by her Ph.D. in Chemistry under the supervision of Professor Timothy L. Macdonald at the University of Virginia. Following a number of years working as an ADME research scientist at Eli Lilly and Company, she joined Merck & Co., Inc. in Kenilworth, NJ, U.S. in 2013. Her work has focused on model development in the area of drug induced liver injury and drug metabolism, as well as the early application of in silico ADME tools to inform decision making in discovery.</p></figure></div><div class="bio" rid="ath17"><figure id="ath17" data-index="25" class="article__inlineFigure"><h2 class="fig-label">Tjerk Bueters</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=BIO-d130e1146-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Tjerk Bueters</b> works at Merck & Co., Inc. in West Point, PA, U.S., as a lead for translational PKPD offering strategic, educational, and hands-on support to develop and execute on translational plans in the neuroscience and cardiovascular disease areas. In addition, he is serving on several multidisciplinary drug discovery and early development teams, in which he is responsible for the overall pharmacokinetic, pharmacodynamics, and drug metabolism contributions. Before joining his current company, he has worked 8 years at the CNS & Pain unit of AstraZeneca in similar roles. Dr. Bueters has a Ph.D. in Quantitative Pharmacology obtained at Leiden University, The Netherlands, and did his postdoctoral research within neuroscience at the Karolinska Institute in Sweden.</p></figure></div><div class="ack" id="ACK-d130e1061-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66637" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66637" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank M. Kate Holloway for careful reading of the manuscript and constructive comments and David M. Tellers for discussion and support.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i27" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i27"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i28" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i28"> Abbreviations</h2><tr><td class="NLM_term">ADMET</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, excretion, and toxicity</p></td></tr><tr><td class="NLM_term">CL<sub>int</sub></td><td class="NLM_def"><p class="first last">intrinsic clearance</p></td></tr><tr><td class="NLM_term">CL<sub>p</sub></td><td class="NLM_def"><p class="first last">plasma clearance</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high throughput screening</p></td></tr><tr><td class="NLM_term">f<sub>u,mic</sub></td><td class="NLM_def"><p class="first last">unbound fraction in microsomes</p></td></tr><tr><td class="NLM_term">f<sub>u,p</sub></td><td class="NLM_def"><p class="first last">unbound fraction in plasma</p></td></tr><tr><td class="NLM_term">isADMET</td><td class="NLM_def"><p class="first last">in silico absorption, distribution, metabolism, excretion, and toxicity</p></td></tr><tr><td class="NLM_term"><i>P</i><sub>app</sub></td><td class="NLM_def"><p class="first last">apparent permeability</p></td></tr><tr><td class="NLM_term">P-gp</td><td class="NLM_def"><p class="first last">P-glycoprotein</p></td></tr><tr><td class="NLM_term">QSAR</td><td class="NLM_def"><p class="first last">quantitative structure–activity relationship</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04435" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04435" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 26 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Mayr, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuerst, P.</span><span> </span><span class="NLM_article-title">The future of high-throughput screening</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">443</span><span class="NLM_x">–</span> <span class="NLM_lpage">448</span><span class="refDoi"> DOI: 10.1177/1087057108319644</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1177%2F1087057108319644" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=18660458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpt1eitLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=443-448&author=L.+M.+Mayrauthor=P.+Fuerst&title=The+future+of+high-throughput+screening&doi=10.1177%2F1087057108319644"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The future of high-throughput screening</span></div><div class="casAuthors">Mayr, Lorenz M.; Fuerst, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">443-448</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) is a well-established process in lead discovery for pharma and biotech companies and is now also being set up for basic and applied research in academia and some research hospitals.  Since its first advent in the early to mid-1990s, the field of HTS has seen not only a continuous change in technol. and processes but also an adaptation to various needs in lead discovery.  HTS has now evolved into a quite mature discipline of modern drug discovery.  Whereas in previous years, much emphasis has been put toward a steady increase in capacity ("quant. increase") via various strategies in the fields of automation and miniaturization, the past years have seen a steady shift toward higher content and quality ("quality increase") for these biol. test systems.  Today, many experts in the field see HTS at the crossroads with the need to decide either toward further increase in throughput or more focus toward relevance of biol. data.  In this article, the authors describe the development of HTS over the past decade and point out their own ideas for future directions of HTS in biomedical research.  They predict that the trend toward further miniaturization will slow down with the implementation of 384-well, 1536-well, and 384 low-vol.-well plates.  The authors predict that, ultimately, each hit-finding strategy will be much more project related, tailor-made, and better integrated into the broader drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOoQxntLKDvbVg90H21EOLACvtfcHk0lg0ypxk5fz1Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpt1eitLk%253D&md5=7ed3a27bdc7526c8aadcdca01ed3640b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1177%2F1087057108319644&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057108319644%26sid%3Dliteratum%253Aachs%26aulast%3DMayr%26aufirst%3DL.%2BM.%26aulast%3DFuerst%26aufirst%3DP.%26atitle%3DThe%2520future%2520of%2520high-throughput%2520screening%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2008%26volume%3D13%26spage%3D443%26epage%3D448%26doi%3D10.1177%2F1087057108319644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Macarron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bojanic, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burns, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cirovic, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garyantes, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, D. V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hertzberg, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paslay, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schopfer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sittampalam, G. S.</span><span> </span><span class="NLM_article-title">Impact of high-throughput screening in biomedical research</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">188</span><span class="NLM_x">–</span> <span class="NLM_lpage">195</span><span class="refDoi"> DOI: 10.1038/nrd3368</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1038%2Fnrd3368" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=21358738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFensbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=188-195&author=R.+Macarronauthor=M.+N.+Banksauthor=D.+Bojanicauthor=D.+J.+Burnsauthor=D.+A.+Cirovicauthor=T.+Garyantesauthor=D.+V.+S.+Greenauthor=R.+P.+Hertzbergauthor=W.+P.+Janzenauthor=J.+W.+Paslayauthor=U.+Schopferauthor=G.+S.+Sittampalam&title=Impact+of+high-throughput+screening+in+biomedical+research&doi=10.1038%2Fnrd3368"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of high-throughput screening in biomedical research</span></div><div class="casAuthors">Macarron, Ricardo; Banks, Martyn N.; Bojanic, Dejan; Burns, David J.; Cirovic, Dragan A.; Garyantes, Tina; Green, Darren V. S.; Hertzberg, Robert P.; Janzen, William P.; Paslay, Jeff W.; Schopfer, Ulrich; Sittampalam, G. Sitta</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">188-195</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) has been postulated in several quarters to be a contributory factor to the decline in productivity in the pharmaceutical industry.  Moreover, it has been blamed for stifling the creativity that drug discovery demands.  In this article, we aim to dispel these myths and present the case for the use of HTS as part of a proven scientific tool kit, the wider use of which is essential for the discovery of new chemotypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLnrSVehADbLVg90H21EOLACvtfcHk0lirY6_6HOPIsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFensbY%253D&md5=cb9853227506f3a3d774791cff9fd79d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd3368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3368%26sid%3Dliteratum%253Aachs%26aulast%3DMacarron%26aufirst%3DR.%26aulast%3DBanks%26aufirst%3DM.%2BN.%26aulast%3DBojanic%26aufirst%3DD.%26aulast%3DBurns%26aufirst%3DD.%2BJ.%26aulast%3DCirovic%26aufirst%3DD.%2BA.%26aulast%3DGaryantes%26aufirst%3DT.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DHertzberg%26aufirst%3DR.%2BP.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DPaslay%26aufirst%3DJ.%2BW.%26aulast%3DSchopfer%26aufirst%3DU.%26aulast%3DSittampalam%26aufirst%3DG.%2BS.%26atitle%3DImpact%2520of%2520high-throughput%2520screening%2520in%2520biomedical%2520research%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D188%26epage%3D195%26doi%3D10.1038%2Fnrd3368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Thorne, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auld, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span> </span><span class="NLM_article-title">Apparent activity in high-throughput screening: origins of compound-dependent assay interference</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">315</span><span class="NLM_x">–</span> <span class="NLM_lpage">324</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2010.03.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1016%2Fj.cbpa.2010.03.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=20417149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=315-324&author=N.+Thorneauthor=D.+S.+Auldauthor=J.+Inglese&title=Apparent+activity+in+high-throughput+screening%3A+origins+of+compound-dependent+assay+interference&doi=10.1016%2Fj.cbpa.2010.03.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Apparent activity in high-throughput screening: origins of compound-dependent assay interference</span></div><div class="casAuthors">Thorne, Natasha; Auld, Douglas S.; Inglese, James</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">315-324</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Expansive compd. collections made up of structurally heterogeneous chems., the activities of which are largely undefined, present challenging problems for high-throughput screening (HTS).  Foremost is differentiating whether the activity for a given compd. in an assay is directed against the targeted biol., or is the result of surreptitious compd. activity involving the assay detection system.  Such compd. interference can be esp. difficult to identify if it is reproducible and concn.-dependent - characteristics generally attributed to compds. with genuine activity.  While reactive chem. groups on compds. were once thought to be the primary source of compd. interference in assays used in HTS, recent work suggests that other factors, such as compd. aggregation, may play a more significant role in many assay formats.  Considerable progress has been made to profile representative compd. libraries in an effort to identify chem. classes susceptible to producing compd. interference, such as compds. commonly found to inhibit the reporter enzyme firefly luciferase.  Such work has also led to the development of practices that have the potential to significantly reduce compd. interference, for example, through the addn. of non-ionic detergent to assay buffer to reduce aggregation-based inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBawORe7gR97Vg90H21EOLACvtfcHk0lirY6_6HOPIsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsL8%253D&md5=4d3dd71f66ce937002874106edaff12e</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.03.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.03.020%26sid%3Dliteratum%253Aachs%26aulast%3DThorne%26aufirst%3DN.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DInglese%26aufirst%3DJ.%26atitle%3DApparent%2520activity%2520in%2520high-throughput%2520screening%253A%2520origins%2520of%2520compound-dependent%2520assay%2520interference%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D315%26epage%3D324%26doi%3D10.1016%2Fj.cbpa.2010.03.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Hughes, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalindjian, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, K. L.</span><span> </span><span class="NLM_article-title">Principles of early drug discovery</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">162</span><span class="NLM_x">, </span> <span class="NLM_fpage">1239</span><span class="NLM_x">–</span> <span class="NLM_lpage">1249</span><span class="refDoi"> DOI: 10.1111/j.1476-5381.2010.01127.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1111%2Fj.1476-5381.2010.01127.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=21091654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivF2lt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=2011&pages=1239-1249&author=J.+P.+Hughesauthor=S.+Reesauthor=S.+B.+Kalindjianauthor=K.+L.+Philpott&title=Principles+of+early+drug+discovery&doi=10.1111%2Fj.1476-5381.2010.01127.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Principles of early drug discovery</span></div><div class="casAuthors">Hughes, J. P.; Rees, S.; Kalindjian, S. B.; Philpott, K. L.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1239-1249</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Developing a new drug from original idea to the launch of a finished product is a complex process which can take 12-15 years and cost in excess of $1 billion.  The idea for a target can come from a variety of sources including academic and clin. research and from the com. sector.  It may take many years to build up a body of supporting evidence before selecting a target for a costly drug discovery program.  Once a target was chosen, the pharmaceutical industry and more recently some academic centers have streamlined a no. of early processes to identify mols. which possess suitable characteristics to make acceptable drugs.  This review will look at key preclin. stages of the drug discovery process, from initial target identification and validation, through assay development, high throughput screening, hit identification, lead optimization and finally the selection of a candidate mol. for clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooCc74_F_90rVg90H21EOLACvtfcHk0lirY6_6HOPIsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivF2lt7o%253D&md5=05a0016f45b155f9fb5822f5c7bcc35d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2010.01127.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2010.01127.x%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DJ.%2BP.%26aulast%3DRees%26aufirst%3DS.%26aulast%3DKalindjian%26aufirst%3DS.%2BB.%26aulast%3DPhilpott%26aufirst%3DK.%2BL.%26atitle%3DPrinciples%2520of%2520early%2520drug%2520discovery%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2011%26volume%3D162%26spage%3D1239%26epage%3D1249%26doi%3D10.1111%2Fj.1476-5381.2010.01127.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Kerns, E. H.; Di, L.</span> <span class="citation_source-book">Drug-like Properties: Concepts, Structure Design and Methods. From ADME to Toxicity Optimization</span>; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">San Diego, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=E.+H.+Kerns&author=L.+Di&title=Drug-like+Properties%3A+Concepts%2C+Structure+Design+and+Methods.+From+ADME+to+Toxicity+Optimization"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKerns%26aufirst%3DE.%2BH.%26btitle%3DDrug-like%2520Properties%253A%2520Concepts%252C%2520Structure%2520Design%2520and%2520Methods.%2520From%2520ADME%2520to%2520Toxicity%2520Optimization%26pub%3DAcademic%2520Press%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Bleicher, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bohm, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alanine, A. I.</span><span> </span><span class="NLM_article-title">Hit and lead generation: beyond high-throughput screening</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">369</span><span class="NLM_x">–</span> <span class="NLM_lpage">378</span><span class="refDoi"> DOI: 10.1038/nrd1086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1038%2Fnrd1086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=12750740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjslamtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=369-378&author=K.+H.+Bleicherauthor=H.+J.+Bohmauthor=K.+Mullerauthor=A.+I.+Alanine&title=Hit+and+lead+generation%3A+beyond+high-throughput+screening&doi=10.1038%2Fnrd1086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A guide to drug discovery: Hit and lead generation: beyond high-throughput screening</span></div><div class="casAuthors">Bleicher, Konrad H.; Boehm, Hans-Joachim; Mueller, Klaus; Alanine, Alexander I.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">369-378</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The identification of small-mol. modulators of protein function, and the process of transforming these into high-content lead series, are key activities in modern drug discovery.  The decisions taken during this process have far-reaching consequences for success later in lead optimization and even more crucially in clin. development.  Recently, there has been an increased focus on these activities due to escalating downstream costs resulting from high clin. failure rates.  In addn., the vast emerging opportunities from efforts in functional genomics and proteomics demands a departure from the linear process of identification, evaluation and refinement activities towards a more integrated parallel process.  This calls for flexible, fast and cost-effective strategies to meet the demands of producing high-content lead series with improved prospects for clin. success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9IQCsTR8tobVg90H21EOLACvtfcHk0ljDcBl3ZTMkPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjslamtb8%253D&md5=b54e55c1f0d34fbd172c8aa9b996591d</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd1086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1086%26sid%3Dliteratum%253Aachs%26aulast%3DBleicher%26aufirst%3DK.%2BH.%26aulast%3DBohm%26aufirst%3DH.%2BJ.%26aulast%3DMuller%26aufirst%3DK.%26aulast%3DAlanine%26aufirst%3DA.%2BI.%26atitle%3DHit%2520and%2520lead%2520generation%253A%2520beyond%2520high-throughput%2520screening%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D369%26epage%3D378%26doi%3D10.1038%2Fnrd1086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Davis, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keeling, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steele, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomkinson, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tinker, A. C.</span><span> </span><span class="NLM_article-title">Components of successful lead generation</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">421</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.2174/1568026053828411</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.2174%2F1568026053828411" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=15892684" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkvFGhtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2005&pages=421-439&author=A.+M.+Davisauthor=D.+J.+Keelingauthor=J.+Steeleauthor=N.+P.+Tomkinsonauthor=A.+C.+Tinker&title=Components+of+successful+lead+generation&doi=10.2174%2F1568026053828411"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Components of successful lead generation</span></div><div class="casAuthors">Davis, Andrew M.; Keeling, David J.; Steele, John; Tomkinson, Nicholas P.; Tinker, Alan C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">421-439</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  This review aims to give an overview of current good practice in the prosecution of Lead Generation.  It will assess experiences across the field as judged from the contents of the limited no. of peer-review disclosures to date.  It will also rely heavily on the experiences of the authors from many campaigns within this organization.  Its focus will be on the assembly of an appropriate compd. collection for application in High Throughput Screening (HTS), the prosecution of HTS, the profiling of HTS output and, lastly the Hit-to-Lead optimization of selected HTS output.  Excluded from the scope are detailed aspects of library design [1], parallel synthesis [2], virtual library applications [3], virtual screening [4] and fragment screening [5] approaches, all of which have been the subject of recent reviews.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox5UYz5NF4m7Vg90H21EOLACvtfcHk0ljDcBl3ZTMkPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkvFGhtL4%253D&md5=ef29c85464f4ecbc63787210fda1b778</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2174%2F1568026053828411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026053828411%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DA.%2BM.%26aulast%3DKeeling%26aufirst%3DD.%2BJ.%26aulast%3DSteele%26aufirst%3DJ.%26aulast%3DTomkinson%26aufirst%3DN.%2BP.%26aulast%3DTinker%26aufirst%3DA.%2BC.%26atitle%3DComponents%2520of%2520successful%2520lead%2520generation%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2005%26volume%3D5%26spage%3D421%26epage%3D439%26doi%3D10.2174%2F1568026053828411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Ballard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brassil, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bui, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dolgos, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petersson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tunek, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webborn, P. J.</span><span> </span><span class="NLM_article-title">The right compound in the right assay at the right time: an integrated discovery DMPK strategy</span> <span class="citation_source-journal">Drug Metab. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">252</span><span class="refDoi"> DOI: 10.3109/03602532.2012.691099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.3109%2F03602532.2012.691099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=22697420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVelsLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=224-252&author=P.+Ballardauthor=P.+Brassilauthor=K.+H.+Buiauthor=H.+Dolgosauthor=C.+Peterssonauthor=A.+Tunekauthor=P.+J.+Webborn&title=The+right+compound+in+the+right+assay+at+the+right+time%3A+an+integrated+discovery+DMPK+strategy&doi=10.3109%2F03602532.2012.691099"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The right compound in the right assay at the right time: an integrated discovery DMPK strategy</span></div><div class="casAuthors">Ballard, Peter; Brassil, Patrick; Bui, Khanh H.; Dolgos, Hugues; Petersson, Carl; Tunek, Anders; Webborn, Peter J. H.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">224-252</span>CODEN:
                <span class="NLM_cas:coden">DMTRAR</span>;
        ISSN:<span class="NLM_cas:issn">0360-2532</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  The high rate of attrition during drug development and its assocd. high research and development (R&D) cost have put pressure on pharmaceutical companies to ensure that candidate drugs going to clin. testing have the appropriate quality such that the biol. hypothesis could be evaluated.  To help achieve this ambition, drug metab. and pharmacokinetic (DMPK) science and increasing investment have been deployed earlier in the R&D process.  To gain max. return on investment, it is essential that DMPK concepts are both appropriately integrated into the compd. design process and that compd. selection is focused on accurate prediction of likely outcomes in patients.  This article describes key principles that underpin the contribution of DMPK science for small-mol. research based on 15 years of discovery support in a major pharmaceutical company.  It does not aim to describe the breadth and depth of DMPK science, but more the practical application for decision making in real-world situations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolEqf17mivQbVg90H21EOLACvtfcHk0ljDcBl3ZTMkPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVelsLzF&md5=105fbbcfba6213caaadf41c8b10cf408</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.3109%2F03602532.2012.691099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F03602532.2012.691099%26sid%3Dliteratum%253Aachs%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DBrassil%26aufirst%3DP.%26aulast%3DBui%26aufirst%3DK.%2BH.%26aulast%3DDolgos%26aufirst%3DH.%26aulast%3DPetersson%26aufirst%3DC.%26aulast%3DTunek%26aufirst%3DA.%26aulast%3DWebborn%26aufirst%3DP.%2BJ.%26atitle%3DThe%2520right%2520compound%2520in%2520the%2520right%2520assay%2520at%2520the%2520right%2520time%253A%2520an%2520integrated%2520discovery%2520DMPK%2520strategy%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D2012%26volume%3D44%26spage%3D224%26epage%3D252%26doi%3D10.3109%2F03602532.2012.691099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">van de Waterbeemd, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gifford, E.</span><span> </span><span class="NLM_article-title">ADMET in silico modelling: towards prediction paradise?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">192</span><span class="NLM_x">–</span> <span class="NLM_lpage">204</span><span class="refDoi"> DOI: 10.1038/nrd1032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1038%2Fnrd1032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=12612645" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhsFKrtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=192-204&author=H.+van+de+Waterbeemdauthor=E.+Gifford&title=ADMET+in+silico+modelling%3A+towards+prediction+paradise%3F&doi=10.1038%2Fnrd1032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">ADMET in silico modelling: Towards prediction paradise?</span></div><div class="casAuthors">van de Waterbeemd, Han; Gifford, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">192-204</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with refs.  Following studies in the late 1990s that indicated that poor pharmacokinetics and toxicity were important causes of costly late-stage failures in drug development, it has become widely appreciated that these areas should be considered as early as possible in the drug discovery process.  However, in recent years, combinatorial chem. and high-throughput screening have significantly increased the no. of compds. for which early data on absorption, distribution, metab., excretion (ADME) and toxicity (T) are needed, which has in turn driven the development of a variety of medium and high-throughput in vitro ADMET screens.  Here, we describe how in silico approaches will further increase our ability to predict and model the most relevant pharmacokinetic, metabolic and toxicity endpoints, thereby accelerating the drug discovery process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr8bI29kgKZLVg90H21EOLACvtfcHk0ljnYjvyKW2K9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhsFKrtLo%253D&md5=898a1edb3b33a862185c889930314439</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrd1032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1032%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bde%2BWaterbeemd%26aufirst%3DH.%26aulast%3DGifford%26aufirst%3DE.%26atitle%3DADMET%2520in%2520silico%2520modelling%253A%2520towards%2520prediction%2520paradise%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D192%26epage%3D204%26doi%3D10.1038%2Fnrd1032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominy, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feeney, P. J.</span><span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span> <span class="citation_source-journal">Adv. Drug Delivery Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span><span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0ljnYjvyKW2K9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Leeson, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Springthorpe, B.</span><span> </span><span class="NLM_article-title">The influence of drug-like concepts on decision-making in medicinal chemistry</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">881</span><span class="NLM_x">–</span> <span class="NLM_lpage">890</span><span class="refDoi"> DOI: 10.1038/nrd2445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1038%2Fnrd2445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=17971784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=881-890&author=P.+D.+Leesonauthor=B.+Springthorpe&title=The+influence+of+drug-like+concepts+on+decision-making+in+medicinal+chemistry&doi=10.1038%2Fnrd2445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The influence of drug-like concepts on decision-making in medicinal chemistry</span></div><div class="casAuthors">Leeson, Paul D.; Springthorpe, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">881-890</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite the wide acceptance of drug-like principles such as the 'rule of five', this anal. of mols. currently being synthesized in leading pharmaceutical companies reveals that their phys. properties differ significantly from those of recently discovered oral drugs.  The marked increase in lipophilicity in particular could increase the likelihood of attrition in drug development.  The application of guidelines linked to the concept of drug-likeness, such as the 'rule of five', has gained wide acceptance as an approach to reduce attrition in drug discovery and development.  However, despite this acceptance, anal. of recent trends reveals that the phys. properties of mols. that are currently being synthesized in leading drug discovery companies differ significantly from those of recently discovered oral drugs and compds. in clin. development.  The consequences of the marked increase in lipophilicity - the most important drug-like phys. property - include a greater likelihood of lack of selectivity and attrition in drug development.  Tackling the threat of compd.-related toxicol. attrition needs to move to the mainstream of medicinal chem. decision-making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTUeonLD6iebVg90H21EOLACvtfcHk0ljnYjvyKW2K9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL&md5=5665a9e2dc266a1ee096e20625757ef7</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrd2445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2445%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DSpringthorpe%26aufirst%3DB.%26atitle%3DThe%2520influence%2520of%2520drug-like%2520concepts%2520on%2520decision-making%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D881%26epage%3D890%26doi%3D10.1038%2Fnrd2445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Bueters, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Visser, S. A.</span><span> </span><span class="NLM_article-title">Optimization of human dose prediction by using quantitative and translational pharmacology in drug discovery</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">2351</span><span class="NLM_x">–</span> <span class="NLM_lpage">2369</span><span class="refDoi"> DOI: 10.4155/fmc.15.143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.4155%2Ffmc.15.143" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=26599348" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFKlu7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=2351-2369&author=T.+Buetersauthor=C.+Gibsonauthor=S.+A.+Visser&title=Optimization+of+human+dose+prediction+by+using+quantitative+and+translational+pharmacology+in+drug+discovery&doi=10.4155%2Ffmc.15.143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of human dose prediction by using quantitative and translational pharmacology in drug discovery</span></div><div class="casAuthors">Bueters, Tjerk; Gibson, Christopher; G Visser, Sandra A.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2351-2369</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">In this perspective article, we explain how quant. and translational pharmacol., when well-implemented, is believed to lead to improved clin. candidates and drug targets that are differentiated from current treatment options.  Quant. and translational pharmacol. aims to build and continuously improve the quant. relationship between drug exposure, target engagement, efficacy, safety and its interspecies relationship at every phase of drug discovery.  Drug hunters should consider and apply these concepts to develop compds. with a higher probability of interrogating the clin. biol. hypothesis.  We offer different approaches to set an initial effective concn. or pharmacokinetic-pharmacodynamic target in man and to predict human pharmacokinetics that det. together the predicted human dose and dose schedule.  All concepts are illustrated with ample literature examples.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoy3M0R3KWa1LVg90H21EOLACvtfcHk0ljnYjvyKW2K9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFKlu7%252FO&md5=c30f4756c965a5dbca10d7b5c57b59c9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.4155%2Ffmc.15.143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.15.143%26sid%3Dliteratum%253Aachs%26aulast%3DBueters%26aufirst%3DT.%26aulast%3DGibson%26aufirst%3DC.%26aulast%3DVisser%26aufirst%3DS.%2BA.%26atitle%3DOptimization%2520of%2520human%2520dose%2520prediction%2520by%2520using%2520quantitative%2520and%2520translational%2520pharmacology%2520in%2520drug%2520discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2015%26volume%3D7%26spage%3D2351%26epage%3D2369%26doi%3D10.4155%2Ffmc.15.143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Gleeson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montanari, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J.</span><span> </span><span class="NLM_article-title">Probing the links between in vitro potency, ADMET and physicochemical parameters</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">197</span><span class="NLM_x">–</span> <span class="NLM_lpage">208</span><span class="refDoi"> DOI: 10.1038/nrd3367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1038%2Fnrd3367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=21358739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFemsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=197-208&author=M.+P.+Gleesonauthor=A.+Herseyauthor=D.+Montanariauthor=J.+Overington&title=Probing+the+links+between+in+vitro+potency%2C+ADMET+and+physicochemical+parameters&doi=10.1038%2Fnrd3367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the links between in vitro potency, ADMET and physicochemical parameters</span></div><div class="casAuthors">Gleeson, M. Paul; Hersey, Anne; Montanari, Dino; Overington, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">197-208</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A common underlying assumption in current drug discovery strategies is that compds. with higher in vitro potency at their target(s) have greater potential to translate into successful, low-dose therapeutics.  This has led to the development of screening cascades with in vitro potency embedded as an early filter.  However, this approach is beginning to be questioned, given the bias in physicochem. properties that it can introduce early in lead generation and optimization, which is due to the often diametrically opposed relationship between physicochem. parameters assocd. with high in vitro potency and those assocd. with desirable absorption, distribution, metab., excretion and toxicity (ADMET) characteristics.  Here, we describe analyses that probe these issues further using the ChEMBL database, which includes more than 500,000 drug discovery and marketed oral drug compds.  Key findings include: first, that oral drugs seldom possess nanomolar potency (50 nM on av.); second, that many oral drugs have considerable off-target activity; and third, that in vitro potency does not correlate strongly with the therapeutic dose.  These findings suggest that the perceived benefit of high in vitro potency may be negated by poorer ADMET properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovOslFSgIkUbVg90H21EOLACvtfcHk0lgiOjDp2ftbgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFemsr8%253D&md5=e275044516e9b0245b71c9603477ef74</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnrd3367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3367%26sid%3Dliteratum%253Aachs%26aulast%3DGleeson%26aufirst%3DM.%2BP.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DMontanari%26aufirst%3DD.%26aulast%3DOverington%26aufirst%3DJ.%26atitle%3DProbing%2520the%2520links%2520between%2520in%2520vitro%2520potency%252C%2520ADMET%2520and%2520physicochemical%2520parameters%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D197%26epage%3D208%26doi%3D10.1038%2Fnrd3367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Hop, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davidson, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duignan, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federico, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janiszewski, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krueger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lebowitz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liston, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steyn, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soglia, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troutman, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umland, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whalen, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zelesky, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, S. X.</span><span> </span><span class="NLM_article-title">High throughput ADME screening: practical considerations, impact on the portfolio and enabler of in silico ADME models</span> <span class="citation_source-journal">Curr. Drug Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">847</span><span class="NLM_x">–</span> <span class="NLM_lpage">853</span><span class="refDoi"> DOI: 10.2174/138920008786485092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.2174%2F138920008786485092" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=18991580" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivF2nsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=847-853&author=C.+E.+Hopauthor=M.+J.+Coleauthor=R.+E.+Davidsonauthor=D.+B.+Duignanauthor=J.+Federicoauthor=J.+S.+Janiszewskiauthor=K.+Jenkinsauthor=S.+Kruegerauthor=R.+Lebowitzauthor=T.+E.+Listonauthor=W.+Mitchellauthor=M.+Snyderauthor=S.+J.+Steynauthor=J.+R.+Sogliaauthor=C.+Taylorauthor=M.+D.+Troutmanauthor=J.+Umlandauthor=M.+Westauthor=K.+M.+Whalenauthor=V.+Zeleskyauthor=S.+X.+Zhao&title=High+throughput+ADME+screening%3A+practical+considerations%2C+impact+on+the+portfolio+and+enabler+of+in+silico+ADME+models&doi=10.2174%2F138920008786485092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">High throughput ADME screening: practical considerations, impact on the portfolio and enabler of in silico ADME models</span></div><div class="casAuthors">Hop, Cornelis E. C. A.; Cole, Mark J.; Davidson, Ralph E.; Duignan, David B.; Federico, James; Janiszewski, John S.; Jenkins, Kelly; Krueger, Suzanne; Lebowitz, Rebecca; Liston, Theodore E.; Mitchell, Walter; Snyder, Mark; Steyn, Stefan J.; Soglia, John R.; Taylor, Christine; Troutman, Matt D.; Umland, John; West, Michael; Whalen, Kevin M.; Zelesky, Veronica; Zhao, Sabrina X.</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Metabolism</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">847-853</span>CODEN:
                <span class="NLM_cas:coden">CDMUBU</span>;
        ISSN:<span class="NLM_cas:issn">1389-2002</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Evaluation and optimization of drug metab. and pharmacokinetic data plays an important role in drug discovery and development and several reliable in vitro ADME models are available.  Recently higher throughput in vitro ADME screening facilities have been established in order to be able to evaluate an appreciable fraction of synthesized compds.  The ADME screening process can be dissected in five distinct steps: (1) plate management of compds. in need of in vitro ADME data, (2) optimization of the MS/MS method for the compds., (3) in vitro ADME expts. and sample clean up, (4) collection and redn. of the raw LC-MS/MS data and (5) archival of the processed ADME data.  All steps will be described in detail and the value of the data on drug discovery projects will be discussed as well.  Finally, in vitro ADME screening can generate large quantities of data obtained under identical conditions to allow building of reliable in silico models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHIB4wuoFXErVg90H21EOLACvtfcHk0lgiOjDp2ftbgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivF2nsLk%253D&md5=69acd6725561ddb30157d22dab69e329</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.2174%2F138920008786485092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138920008786485092%26sid%3Dliteratum%253Aachs%26aulast%3DHop%26aufirst%3DC.%2BE.%26aulast%3DCole%26aufirst%3DM.%2BJ.%26aulast%3DDavidson%26aufirst%3DR.%2BE.%26aulast%3DDuignan%26aufirst%3DD.%2BB.%26aulast%3DFederico%26aufirst%3DJ.%26aulast%3DJaniszewski%26aufirst%3DJ.%2BS.%26aulast%3DJenkins%26aufirst%3DK.%26aulast%3DKrueger%26aufirst%3DS.%26aulast%3DLebowitz%26aufirst%3DR.%26aulast%3DListon%26aufirst%3DT.%2BE.%26aulast%3DMitchell%26aufirst%3DW.%26aulast%3DSnyder%26aufirst%3DM.%26aulast%3DSteyn%26aufirst%3DS.%2BJ.%26aulast%3DSoglia%26aufirst%3DJ.%2BR.%26aulast%3DTaylor%26aufirst%3DC.%26aulast%3DTroutman%26aufirst%3DM.%2BD.%26aulast%3DUmland%26aufirst%3DJ.%26aulast%3DWest%26aufirst%3DM.%26aulast%3DWhalen%26aufirst%3DK.%2BM.%26aulast%3DZelesky%26aufirst%3DV.%26aulast%3DZhao%26aufirst%3DS.%2BX.%26atitle%3DHigh%2520throughput%2520ADME%2520screening%253A%2520practical%2520considerations%252C%2520impact%2520on%2520the%2520portfolio%2520and%2520enabler%2520of%2520in%2520silico%2520ADME%2520models%26jtitle%3DCurr.%2520Drug%2520Metab.%26date%3D2008%26volume%3D9%26spage%3D847%26epage%3D853%26doi%3D10.2174%2F138920008786485092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Leeson, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. J.</span><span> </span><span class="NLM_article-title">Molecular property design: does everyone get it?</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">722</span><span class="NLM_x">–</span> <span class="NLM_lpage">725</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00157</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00157" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptVGksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=722-725&author=P.+D.+Leesonauthor=R.+J.+Young&title=Molecular+property+design%3A+does+everyone+get+it%3F&doi=10.1021%2Facsmedchemlett.5b00157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Property Design: Does Everyone Get It?</span></div><div class="casAuthors">Leeson, Paul D.; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">722-725</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The principles of mol. property optimization in drug design have been understood for decades, yet much drug discovery activity today is conducted at the periphery of historical druglike property space.  Lead optimization trajectories aimed at reducing physicochem. risk, assisted by ligand efficiency metrics, could help to reduce clin. attrition rates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_t8gIn79arVg90H21EOLACvtfcHk0lgiOjDp2ftbgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptVGksr4%253D&md5=5a807a8d049ce8cf4adc186891a9234d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00157%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DMolecular%2520property%2520design%253A%2520does%2520everyone%2520get%2520it%253F%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D722%26epage%3D725%26doi%3D10.1021%2Facsmedchemlett.5b00157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Desai, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawada, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raub, T. J.</span><span> </span><span class="NLM_article-title">Integration of in silico and in vitro tools for scaffold optimization during drug discovery: predicting P-glycoprotein efflux</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1249</span><span class="NLM_x">–</span> <span class="NLM_lpage">1261</span><span class="refDoi"> DOI: 10.1021/mp300555n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/mp300555n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1aju7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=1249-1261&author=P.+V.+Desaiauthor=G.+A.+Sawadaauthor=I.+A.+Watsonauthor=T.+J.+Raub&title=Integration+of+in+silico+and+in+vitro+tools+for+scaffold+optimization+during+drug+discovery%3A+predicting+P-glycoprotein+efflux&doi=10.1021%2Fmp300555n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Integration of in Silico and in Vitro Tools for Scaffold Optimization during Drug Discovery: Predicting P-Glycoprotein Efflux</span></div><div class="casAuthors">Desai, Prashant V.; Sawada, Geri A.; Watson, Ian A.; Raub, Thomas J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1249-1261</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In silico tools are regularly utilized for designing and prioritizing compds. to address challenges related to drug metab. and pharmacokinetics (DMPK) during the process of drug discovery.  P-Glycoprotein (P-gp) is a member of the ATP-binding cassette (ABC) transporters with broad substrate specificity that plays a significant role in absorption and distribution of drugs that are P-gp substrates.  As a result, screening for P-gp transport has now become routine in the drug discovery process.  Typically, bidirectional permeability assays are employed to assess in vitro P-gp efflux.  In this article, we use P-gp as an example to illustrate a well-validated methodol. to effectively integrate in silico and in vitro tools to identify and resolve key barriers during the early stages of drug discovery.  A detailed account of development and application of in silico tools such as simple guidelines based on physicochem. properties and more complex quant. structure-activity relationship (QSAR) models is provided.  The tools were developed based on structurally diverse data for more than 2000 compds. generated using a robust P-gp substrate assay over the past several years.  Anal. of physicochem. properties revealed a significantly lower proportion (<10%) of P-gp substrates among the compds. with topol. polar surface area (TPSA) <60 Å2 and the most basic cpKa <8.  In contrast, this proportion of substrates was greater than 75% for compds. with TPSA >60 Å2 and the most basic cpKa >8.  Among the various QSAR models evaluated to predict P-gp efflux, the Bagging model provided optimum prediction performance for prospective validation based on chronol. test sets.  Four sequential versions of the model were built with increasing nos. of compds. to train the models as new data became available.  Except for the first version with the smallest training set, the QSAR models exhibited robust prediction profiles with pos. prediction values (PPV) and neg. prediction values (NPV) exceeding 80%.  The QSAR model demonstrated better concordance with the manual P-gp substrate assay than an automated P-gp substrate screen.  The in silico and the in vitro tools have been effectively integrated during early stages of drug discovery to resolve P-gp-related challenges exemplified by several case studies.  Key learning based on our experience with P-gp can be widely applicable across other DMPK-related challenges.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Q5Y8HssxH7Vg90H21EOLACvtfcHk0lgHvicURBIRqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1aju7o%253D&md5=3bd5239b80270c0f4ed85dffb86704ed</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fmp300555n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp300555n%26sid%3Dliteratum%253Aachs%26aulast%3DDesai%26aufirst%3DP.%2BV.%26aulast%3DSawada%26aufirst%3DG.%2BA.%26aulast%3DWatson%26aufirst%3DI.%2BA.%26aulast%3DRaub%26aufirst%3DT.%2BJ.%26atitle%3DIntegration%2520of%2520in%2520silico%2520and%2520in%2520vitro%2520tools%2520for%2520scaffold%2520optimization%2520during%2520drug%2520discovery%253A%2520predicting%2520P-glycoprotein%2520efflux%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2013%26volume%3D10%26spage%3D1249%26epage%3D1261%26doi%3D10.1021%2Fmp300555n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Sherer, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verras, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madeira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagmann, W. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bleasby, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cornell, W. D.</span><span> </span><span class="NLM_article-title">QSAR prediction of passive permeability in the LLC-PK1 cell line: trends in molecular properties and cross-prediction of Caco-2 permeabilities</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">231</span><span class="NLM_x">–</span> <span class="NLM_lpage">245</span><span class="refDoi"> DOI: 10.1002/minf.201100157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1002%2Fminf.201100157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=27477094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC38XjsFOnt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=231-245&author=E.+C.+Shererauthor=A.+Verrasauthor=M.+Madeiraauthor=W.+K.+Hagmannauthor=R.+P.+Sheridanauthor=D.+Robertsauthor=K.+Bleasbyauthor=W.+D.+Cornell&title=QSAR+prediction+of+passive+permeability+in+the+LLC-PK1+cell+line%3A+trends+in+molecular+properties+and+cross-prediction+of+Caco-2+permeabilities&doi=10.1002%2Fminf.201100157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">QSAR Prediction of Passive Permeability in the LLC-PK1 Cell Line: Trends in Molecular Properties and Cross-Prediction of Caco-2 Permeabilities</span></div><div class="casAuthors">Sherer, Edward C.; Verras, Andreas; Madeira, Maria; Hagmann, William K.; Sheridan, Robert P.; Roberts, Drew; Bleasby, Kelly; Cornell, Wendy D.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3-4</span>),
    <span class="NLM_cas:pages">231-245</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A QSAR model for predicting passive permeability (Papp) was derived from Papp values measured in the LLC-PK1 cell line.  The QSAR method and descriptor set that performed best in terms of cross-validation was random forest with a combination of AP, DP, and MOE2D descriptors.  The QSAR model was used to predict the Caco-2 cell permeability for 313 compds. described in the literature with good success.  We find that passive permeability for different cell lines can be predicted with similar mol. properties and descriptors.  It is shown that the variation in exptl. measurements of Papp is smaller than the error in QSAR predictions indicating that predictions are not quant. perfect, although qual. useful.  We get better predictions if the training set is large and diverse, rather than smaller and more internally consistent.  This is because prediction accuracy falls off quickly with decreasing similarity to the training set and it is therefore better to have as large a training set as possible.  While single phys. parameters are not as good as a full QSAR model in predicting Papp, logD seems the most important parameter.  Intermediate values of logD are assocd. with higher Papp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-PMQffMXQQrVg90H21EOLACvtfcHk0lgHvicURBIRqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjsFOnt7s%253D&md5=e21d5d9a43bae6f9ec7a9c5dadf1bfa1</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fminf.201100157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201100157%26sid%3Dliteratum%253Aachs%26aulast%3DSherer%26aufirst%3DE.%2BC.%26aulast%3DVerras%26aufirst%3DA.%26aulast%3DMadeira%26aufirst%3DM.%26aulast%3DHagmann%26aufirst%3DW.%2BK.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DRoberts%26aufirst%3DD.%26aulast%3DBleasby%26aufirst%3DK.%26aulast%3DCornell%26aufirst%3DW.%2BD.%26atitle%3DQSAR%2520prediction%2520of%2520passive%2520permeability%2520in%2520the%2520LLC-PK1%2520cell%2520line%253A%2520trends%2520in%2520molecular%2520properties%2520and%2520cross-prediction%2520of%2520Caco-2%2520permeabilities%26jtitle%3DMol.%2520Inf.%26date%3D2012%26volume%3D31%26spage%3D231%26epage%3D245%26doi%3D10.1002%2Fminf.201100157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMasters, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voigt, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wildey, M. J.</span><span> </span><span class="NLM_article-title">eCounterscreening: using QSAR predictions to prioritize testing for off-target activities and setting the balance between benefit and risk</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">231</span><span class="NLM_x">–</span> <span class="NLM_lpage">238</span><span class="refDoi"> DOI: 10.1021/ci500666m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500666m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFKgsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=231-238&author=R.+P.+Sheridanauthor=D.+R.+McMastersauthor=J.+H.+Voigtauthor=M.+J.+Wildey&title=eCounterscreening%3A+using+QSAR+predictions+to+prioritize+testing+for+off-target+activities+and+setting+the+balance+between+benefit+and+risk&doi=10.1021%2Fci500666m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">eCounterscreening: Using QSAR Predictions to Prioritize Testing for Off-Target Activities and Setting the Balance between Benefit and Risk</span></div><div class="casAuthors">Sheridan, Robert P.; McMasters, Daniel R.; Voigt, Johannes H.; Wildey, Mary Jo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">231-238</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">During drug development, compds. are tested against counterscreens, a panel of off-target activities that would be undesirable for a drug to have.  Testing every compd. against every counterscreen is generally too costly in terms of time and money, and we need to find a rational way of prioritizing counterscreen testing.  Here we present the eCounterscreening paradigm, wherein predictions from QSAR models for counterscreen activity are used to generate a recommendation as to whether a specific compd. in a specific project should be tested against a specific counterscreen.  The rules behind the recommendations, which can be summarized in a risk-benefit plot specific for a counterscreen/project combination, are based on a previously assembled database of prospective QSAR predictions.  The recommendations require two user-defined cutoffs: the level of activity in a specific counterscreen that is considered undesirable and the level of risk the chemist is willing to accept that an undesired counterscreen activity will go undetected.  We demonstrate in a simulated prospective expt. that eCounterscreening can be used to postpone a large fraction of counterscreen testing and still have an acceptably low risk of undetected counterscreen activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjw-Fazb67UbVg90H21EOLACvtfcHk0lgHvicURBIRqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFKgsw%253D%253D&md5=7e433fcc51077f0cce1fe04dc65f47cf</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fci500666m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500666m%26sid%3Dliteratum%253Aachs%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DMcMasters%26aufirst%3DD.%2BR.%26aulast%3DVoigt%26aufirst%3DJ.%2BH.%26aulast%3DWildey%26aufirst%3DM.%2BJ.%26atitle%3DeCounterscreening%253A%2520using%2520QSAR%2520predictions%2520to%2520prioritize%2520testing%2520for%2520off-target%2520activities%2520and%2520setting%2520the%2520balance%2520between%2520benefit%2520and%2520risk%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D231%26epage%3D238%26doi%3D10.1021%2Fci500666m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Mahar Doan, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphreys, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wring, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shampine, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serabjit-Singh, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adkison, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polli, J. W.</span><span> </span><span class="NLM_article-title">Passive permeability and P-glycoprotein-mediated efflux differentiate central nervous system (CNS) and non-CNS marketed drugs</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">1029</span><span class="NLM_x">–</span> <span class="NLM_lpage">1037</span><span class="refDoi"> DOI: 10.1124/jpet.102.039255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1124%2Fjpet.102.039255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=12438524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A280%3ADC%252BD38nptVKmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2002&pages=1029-1037&author=K.+M.+Mahar+Doanauthor=J.+E.+Humphreysauthor=L.+O.+Websterauthor=S.+A.+Wringauthor=L.+J.+Shampineauthor=C.+J.+Serabjit-Singhauthor=K.+K.+Adkisonauthor=J.+W.+Polli&title=Passive+permeability+and+P-glycoprotein-mediated+efflux+differentiate+central+nervous+system+%28CNS%29+and+non-CNS+marketed+drugs&doi=10.1124%2Fjpet.102.039255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Passive permeability and P-glycoprotein-mediated efflux differentiate central nervous system (CNS) and non-CNS marketed drugs</span></div><div class="casAuthors">Mahar Doan Kelly M; Humphreys Joan E; Webster Lindsey O; Wring Stephen A; Shampine Larry J; Serabjit-Singh Cosette J; Adkison Kimberly K; Polli Joseph W</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of pharmacology and experimental therapeutics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1029-37</span>
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    </div><div class="casAbstract">Membrane permeability and P-glycoprotein (Pgp) can be limiting factors for blood-brain barrier penetration.  The objectives of this study were to determine whether there are differences in the in vitro permeability, Pgp substrate profiles, and physicochemical properties of drugs for central nervous system (CNS) and non-CNS indications, and whether these differences are useful criteria in selecting compounds for drug development.  Apparent permeability (P(app)) and Pgp substrate profiles for 93 CNS (n = 48) and non-CNS (n = 45) drugs were determined by monolayer efflux.  Calcein-AM inhibition assays were used to supplement the efflux results.  The CNS set (2 of 48, 4.2%) had a 7-fold lower incidence of passive permeability values <150 nm/s compared with the non-CNS set (13 of 45, 28.9%).  The majority of drugs (72.0%, 67 of 93) were not Pgp substrates; however, 49.5% (46 of 93) were positive in the calcein-AM assay when tested at 100 microM.  The CNS drug set (n = 7 of 48, 14.6%) had a 3-fold lower incidence of Pgp-mediated efflux than the non-CNS drug set (n = 19 of 45, 42.2%).  Analysis of 18 physicochemical properties revealed that the CNS drug set had fewer hydrogen bond donors, fewer positive charges, greater lipophilicity, lower polar surface area, and reduced flexibility compared with the non-CNS group (p < 0.05), properties that enhance membrane permeability.  This study on a large, diverse set of marketed compounds clearly demonstrates that permeability, Pgp-mediated efflux, and certain physicochemical properties are factors that differentiate CNS and non-CNS drugs.  For CNS delivery, a drug should ideally have an in vitro passive permeability >150 nm/s and not be a good (B --> A/A --> B ratio <2.5) Pgp substrate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT_HiV1Xqt18Vf1XRtfYM0dfW6udTcc2eb5DdICiyuDtbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD38nptVKmsw%253D%253D&md5=b841902aa8502de47c1e1603c80e09fb</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1124%2Fjpet.102.039255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.102.039255%26sid%3Dliteratum%253Aachs%26aulast%3DMahar%2BDoan%26aufirst%3DK.%2BM.%26aulast%3DHumphreys%26aufirst%3DJ.%2BE.%26aulast%3DWebster%26aufirst%3DL.%2BO.%26aulast%3DWring%26aufirst%3DS.%2BA.%26aulast%3DShampine%26aufirst%3DL.%2BJ.%26aulast%3DSerabjit-Singh%26aufirst%3DC.%2BJ.%26aulast%3DAdkison%26aufirst%3DK.%2BK.%26aulast%3DPolli%26aufirst%3DJ.%2BW.%26atitle%3DPassive%2520permeability%2520and%2520P-glycoprotein-mediated%2520efflux%2520differentiate%2520central%2520nervous%2520system%2520%2528CNS%2529%2520and%2520non-CNS%2520marketed%2520drugs%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2002%26volume%3D303%26spage%3D1029%26epage%3D1037%26doi%3D10.1124%2Fjpet.102.039255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Giuliano, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jairaj, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zafiu, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laufer, R.</span><span> </span><span class="NLM_article-title">Direct determination of unbound intrinsic drug clearance in the microsomal stability assay</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">1319</span><span class="NLM_x">–</span> <span class="NLM_lpage">1324</span><span class="refDoi"> DOI: 10.1124/dmd.105.005033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1124%2Fdmd.105.005033" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=1319-1324&author=C.+Giulianoauthor=M.+Jairajauthor=C.+M.+Zafiuauthor=R.+Laufer&title=Direct+determination+of+unbound+intrinsic+drug+clearance+in+the+microsomal+stability+assay&doi=10.1124%2Fdmd.105.005033"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1124%2Fdmd.105.005033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.105.005033%26sid%3Dliteratum%253Aachs%26aulast%3DGiuliano%26aufirst%3DC.%26aulast%3DJairaj%26aufirst%3DM.%26aulast%3DZafiu%26aufirst%3DC.%2BM.%26aulast%3DLaufer%26aufirst%3DR.%26atitle%3DDirect%2520determination%2520of%2520unbound%2520intrinsic%2520drug%2520clearance%2520in%2520the%2520microsomal%2520stability%2520assay%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2005%26volume%3D33%26spage%3D1319%26epage%3D1324%26doi%3D10.1124%2Fdmd.105.005033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Turner, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeo, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tucker, G. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rostami-Hodjegan, A.</span><span> </span><span class="NLM_article-title">Prediction of nonspecific hepatic microsomal binding from readily available physicochemical properties</span> <span class="citation_source-journal">Drug Metab. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">162</span><span class="NLM_x">–</span> <span class="NLM_lpage">162</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2006&pages=162-162&author=D.+B.+Turnerauthor=K.+R.+Yeoauthor=G.+T.+Tuckerauthor=A.+Rostami-Hodjegan&title=Prediction+of+nonspecific+hepatic+microsomal+binding+from+readily+available+physicochemical+properties"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTurner%26aufirst%3DD.%2BB.%26aulast%3DYeo%26aufirst%3DK.%2BR.%26aulast%3DTucker%26aufirst%3DG.%2BT.%26aulast%3DRostami-Hodjegan%26aufirst%3DA.%26atitle%3DPrediction%2520of%2520nonspecific%2520hepatic%2520microsomal%2520binding%2520from%2520readily%2520available%2520physicochemical%2520properties%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D2006%26volume%3D38%26spage%3D162%26epage%3D162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Yang, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jamei, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeo, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rostami-Hodjegan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tucker, G. T.</span><span> </span><span class="NLM_article-title">Misuse of the well-stirred model of hepatic drug clearance</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">501</span><span class="NLM_x">–</span> <span class="NLM_lpage">502</span><span class="refDoi"> DOI: 10.1124/dmd.106.013359</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1124%2Fdmd.106.013359" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=501-502&author=J.+S.+Yangauthor=M.+Jameiauthor=K.+R.+Yeoauthor=A.+Rostami-Hodjeganauthor=G.+T.+Tucker&title=Misuse+of+the+well-stirred+model+of+hepatic+drug+clearance&doi=10.1124%2Fdmd.106.013359"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1124%2Fdmd.106.013359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.106.013359%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%2BS.%26aulast%3DJamei%26aufirst%3DM.%26aulast%3DYeo%26aufirst%3DK.%2BR.%26aulast%3DRostami-Hodjegan%26aufirst%3DA.%26aulast%3DTucker%26aufirst%3DG.%2BT.%26atitle%3DMisuse%2520of%2520the%2520well-stirred%2520model%2520of%2520hepatic%2520drug%2520clearance%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2007%26volume%3D35%26spage%3D501%26epage%3D502%26doi%3D10.1124%2Fdmd.106.013359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Eisen, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spellman, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botstein, D.</span><span> </span><span class="NLM_article-title">Cluster analysis and display of genome-wide expression patterns</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">14863</span><span class="NLM_x">–</span> <span class="NLM_lpage">14868</span><span class="refDoi"> DOI: 10.1073/pnas.95.25.14863</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1073%2Fpnas.95.25.14863" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=9843981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADyaK1cXotVGmurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=14863-14868&author=M.+B.+Eisenauthor=P.+T.+Spellmanauthor=P.+O.+Brownauthor=D.+Botstein&title=Cluster+analysis+and+display+of+genome-wide+expression+patterns&doi=10.1073%2Fpnas.95.25.14863"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Cluster analysis and display of genome-wide expression patterns</span></div><div class="casAuthors">Eisen, Michael B.; Spellman, Paul T.; Brown, Patrick O.; Botstein, David</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">14863-14868</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A system of cluster anal. for genome-wide expression data from DNA microarray hybridization is described that uses std. statistical algorithms to arrange genes according to similarity in pattern of gene expression.  The output is displayed graphically, conveying the clustering and the underlying expression data simultaneously in a form intuitive for biologists.  We have found in the budding yeast Saccharomyces cerevisiae that clustering gene expression data groups together efficiently genes of known similar function, and we find a similar tendency in human data.  Thus patterns seen in genome-wide expression expts. can be interpreted as indications of the status of cellular processes.  Also, coexpression of genes of known function with poorly characterized or novel genes may provide a simple means of gaining leads to the functions of many genes for which information is not available currently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMnoYB4klPc7Vg90H21EOLACvtfcHk0lh90cEvt4MCkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXotVGmurk%253D&md5=6ef0b72943b7e2e928966e665c3a9628</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1073%2Fpnas.95.25.14863&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.95.25.14863%26sid%3Dliteratum%253Aachs%26aulast%3DEisen%26aufirst%3DM.%2BB.%26aulast%3DSpellman%26aufirst%3DP.%2BT.%26aulast%3DBrown%26aufirst%3DP.%2BO.%26aulast%3DBotstein%26aufirst%3DD.%26atitle%3DCluster%2520analysis%2520and%2520display%2520of%2520genome-wide%2520expression%2520patterns%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1998%26volume%3D95%26spage%3D14863%26epage%3D14868%26doi%3D10.1073%2Fpnas.95.25.14863" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Erlanson, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDowell, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, T.</span><span> </span><span class="NLM_article-title">Fragment-based drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">3463</span><span class="NLM_x">–</span> <span class="NLM_lpage">3482</span><span class="refDoi"> DOI: 10.1021/jm040031v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm040031v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFeltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=3463-3482&author=D.+A.+Erlansonauthor=R.+S.+McDowellauthor=T.+O%E2%80%99Brien&title=Fragment-based+drug+discovery&doi=10.1021%2Fjm040031v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; McDowell, Robert S.; O'Brien, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3463-3482</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The authors discuss steps in fragment-based drug design such as the development of the library of fragments, fragment screening methods, in situ fragment assembly.  The authors demonstrated that the fragment-based methods of drug discovery are widespread in industry and academia and that this methods have been successful in generation new drug leads with high potency and improved pharmacokinetic properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6YMqrpLJZY7Vg90H21EOLACvtfcHk0lhz2XaH9a5wXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFeltLg%253D&md5=ad9c404ed14d03d889ceae44b4ef991a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm040031v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040031v%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DMcDowell%26aufirst%3DR.%2BS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26atitle%3DFragment-based%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D3463%26epage%3D3482%26doi%3D10.1021%2Fjm040031v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keseru, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leeson, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynolds, C. H.</span><span> </span><span class="NLM_article-title">The role of ligand efficiency metrics in drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">121</span><span class="refDoi"> DOI: 10.1038/nrd4163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1038%2Fnrd4163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=24481311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=105-121&author=A.+L.+Hopkinsauthor=G.+M.+Keseruauthor=P.+D.+Leesonauthor=D.+C.+Reesauthor=C.+H.+Reynolds&title=The+role+of+ligand+efficiency+metrics+in+drug+discovery&doi=10.1038%2Fnrd4163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ligand efficiency metrics in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.; Keserue, Gyoergy M.; Leeson, Paul D.; Rees, David C.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-121</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The judicious application of ligand or binding efficiency metrics, which quantify the mol. properties required to obtain binding affinity for a drug target, is gaining traction in the selection and optimization of fragments, hits and leads.  Retrospective anal. of recently marketed oral drugs shows that they frequently have highly optimized ligand efficiency values for their targets.  Optimizing ligand efficiency metrics based on both mol. mass and lipophilicity, when set in the context of the specific target, has the potential to ameliorate the inflation of these properties that has been obsd. in current medicinal chem. practice, and to increase the quality of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq96uhvE1OZ5LVg90H21EOLACvtfcHk0lhz2XaH9a5wXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D&md5=344030e59f499180f79d311302a27532</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrd4163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4163%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DThe%2520role%2520of%2520ligand%2520efficiency%2520metrics%2520in%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D105%26epage%3D121%26doi%3D10.1038%2Fnrd4163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Sanguinetti, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tristani-Firouzi, M.</span><span> </span><span class="NLM_article-title">hERG potassium channels and cardiac arrhythmia</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">440</span><span class="NLM_x">, </span> <span class="NLM_fpage">463</span><span class="NLM_x">–</span> <span class="NLM_lpage">469</span><span class="refDoi"> DOI: 10.1038/nature04710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1038%2Fnature04710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=16554806" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BD28Xis1OlsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=440&publication_year=2006&pages=463-469&author=M.+C.+Sanguinettiauthor=M.+Tristani-Firouzi&title=hERG+potassium+channels+and+cardiac+arrhythmia&doi=10.1038%2Fnature04710"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">hERG potassium channels and cardiac arrhythmia</span></div><div class="casAuthors">Sanguinetti, Michael C.; Tristani-Firouzi, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">440</span>
        (<span class="NLM_cas:issue">7083</span>),
    <span class="NLM_cas:pages">463-469</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  HERG potassium channels are essential for normal elec. activity in the heart.  Inherited mutations in the HERG gene cause long QT syndrome, a disorder that predisposes individuals to life-threatening arrhythmias.  Arrhythmia can also be induced by a blockage of hERG channels by a surprisingly diverse group of drugs.  This side effect is a common reason for drug failure in preclin. safety trials.  Insights gained from the crystal structures of other potassium channels have helped our understanding of the block of hERG channels and the mechanisms of gating.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopS3WBEpuAUrVg90H21EOLACvtfcHk0lhz2XaH9a5wXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xis1OlsLs%253D&md5=3f355dcd51da3488d73c7c91cbe8d3b8</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnature04710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04710%26sid%3Dliteratum%253Aachs%26aulast%3DSanguinetti%26aufirst%3DM.%2BC.%26aulast%3DTristani-Firouzi%26aufirst%3DM.%26atitle%3DhERG%2520potassium%2520channels%2520and%2520cardiac%2520arrhythmia%26jtitle%3DNature%26date%3D2006%26volume%3D440%26spage%3D463%26epage%3D469%26doi%3D10.1038%2Fnature04710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 12 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Elena L. Cáceres, Nicholas C. Mew, <span class="NLM_string-name hlFld-ContribAuthor">Michael J. Keiser</span>. </span><span class="cited-content_cbyCitation_article-title">Adding Stochastic Negative Examples into Machine Learning Improves Molecular Bioactivity Prediction. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 5957-5970. <a href="https://doi.org/10.1021/acs.jcim.0c00565" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00565</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00565&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00565%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAdding%252BStochastic%252BNegative%252BExamples%252Binto%252BMachine%252BLearning%252BImproves%252BMolecular%252BBioactivity%252BPrediction%26aulast%3DC%25C3%25A1ceres%26aufirst%3DElena%2BL.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D21052020%26date%3D27112020%26volume%3D60%26issue%3D12%26spage%3D5957%26epage%3D5970" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Evan N. Feinberg, Elizabeth Joshi, Vijay S. Pande, <span class="NLM_string-name hlFld-ContribAuthor">Alan C. Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Improvement in ADMET Prediction with Multitask Deep Featurization. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (16)
                                     , 8835-8848. <a href="https://doi.org/10.1021/acs.jmedchem.9b02187" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b02187</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b02187&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b02187%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DImprovement%252Bin%252BADMET%252BPrediction%252Bwith%252BMultitask%252BDeep%252BFeaturization%26aulast%3DFeinberg%26aufirst%3DEvan%2BN.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D31122019%26date%3D12052020%26date%3D14042020%26volume%3D63%26issue%3D16%26spage%3D8835%26epage%3D8848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xiaoshen Ma, David L. Sloman, Yongxin Han, <span class="NLM_string-name hlFld-ContribAuthor">David J. Bennett</span>. </span><span class="cited-content_cbyCitation_article-title">A Selective Synthesis of 2,2-Difluorobicyclo[1.1.1]pentane Analogues: “BCP-F2”. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2019,</strong> <em>21 </em>
                                    (18)
                                     , 7199-7203. <a href="https://doi.org/10.1021/acs.orglett.9b02026" title="DOI URL">https://doi.org/10.1021/acs.orglett.9b02026</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.9b02026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.9b02026%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DA%252BSelective%252BSynthesis%252Bof%252B2%25252C2-Difluorobicyclo%25255B1.1.1%25255Dpentane%252BAnalogues%25253A%252B%2525E2%252580%25259CBCP-F2%2525E2%252580%25259D%26aulast%3DMa%26aufirst%3DXiaoshen%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D12062019%26date%3D11072019%26volume%3D21%26issue%3D18%26spage%3D7199%26epage%3D7203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Nayra
R. Perestelo, Gabriel G. Llanos, Carolina P. Reyes, Angel Amesty, Kartheek Sooda, Saeed Afshinjavid, Ignacio A. Jiménez, Farideh Javid, <span class="NLM_string-name hlFld-ContribAuthor">Isabel L. Bazzocchi</span>. </span><span class="cited-content_cbyCitation_article-title">Expanding the Chemical Space of Withaferin A by Incorporating Silicon To Improve Its Clinical Potential on Human Ovarian Carcinoma Cells. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (9)
                                     , 4571-4585. <a href="https://doi.org/10.1021/acs.jmedchem.9b00146" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00146</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00146%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DExpanding%252Bthe%252BChemical%252BSpace%252Bof%252BWithaferin%252BA%252Bby%252BIncorporating%252BSilicon%252BTo%252BImprove%252BIts%252BClinical%252BPotential%252Bon%252BHuman%252BOvarian%252BCarcinoma%252BCells%26aulast%3DPerestelo%26aufirst%3DNayra%2BR.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D24012019%26date%3D01052019%26date%3D22042019%26volume%3D62%26issue%3D9%26spage%3D4571%26epage%3D4585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sushovan  Jena</span>, <span class="hlFld-ContribAuthor ">Punnagai  Munusami</span>, <span class="hlFld-ContribAuthor ">Balamurali  MM</span>, <span class="hlFld-ContribAuthor ">Kaushik  Chanda</span>. </span><span class="cited-content_cbyCitation_article-title">Computationally approached inhibition potential of Tinospora cordifolia towards COVID-19 targets. </span><span class="cited-content_cbyCitation_journal-name">VirusDisease</span><span> <strong>2021,</strong> <em>32 </em>
                                    (1)
                                     , 65-77. <a href="https://doi.org/10.1007/s13337-021-00666-7" title="DOI URL">https://doi.org/10.1007/s13337-021-00666-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s13337-021-00666-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs13337-021-00666-7%26sid%3Dliteratum%253Aachs%26jtitle%3DVirusDisease%26atitle%3DComputationally%252Bapproached%252Binhibition%252Bpotential%252Bof%252BTinospora%252Bcordifolia%252Btowards%252BCOVID-19%252Btargets%26aulast%3DJena%26aufirst%3DSushovan%26date%3D2021%26date%3D2021%26volume%3D32%26issue%3D1%26spage%3D65%26epage%3D77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elena L  Cáceres</span>, <span class="hlFld-ContribAuthor ">Matthew  Tudor</span>, <span class="hlFld-ContribAuthor ">Alan C  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Deep learning approaches in predicting ADMET properties. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2020,</strong> <em>12 </em>
                                    (22)
                                     , 1995-1999. <a href="https://doi.org/10.4155/fmc-2020-0259" title="DOI URL">https://doi.org/10.4155/fmc-2020-0259</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0259%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DDeep%252Blearning%252Bapproaches%252Bin%252Bpredicting%252BADMET%252Bproperties%26aulast%3DC%25C3%25A1ceres%26aufirst%3DElena%2BL%26date%3D2020%26volume%3D12%26issue%3D22%26spage%3D1995%26epage%3D1999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrew E  Brereton</span>, <span class="hlFld-ContribAuthor ">Stephen  MacKinnon</span>, <span class="hlFld-ContribAuthor ">Zhaleh  Safikhani</span>, <span class="hlFld-ContribAuthor ">Shawn  Reeves</span>, <span class="hlFld-ContribAuthor ">Sana  Alwash</span>, <span class="hlFld-ContribAuthor ">Vijay  Shahani</span>, <span class="hlFld-ContribAuthor ">Andreas  Windemuth</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting drug properties with parameter-free machine learning: pareto-optimal embedded modeling (POEM). </span><span class="cited-content_cbyCitation_journal-name">Machine Learning: Science and Technology</span><span> <strong>2020,</strong> <em>1 </em>
                                    (2)
                                     , 025008. <a href="https://doi.org/10.1088/2632-2153/ab891b" title="DOI URL">https://doi.org/10.1088/2632-2153/ab891b</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1088/2632-2153/ab891b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1088%2F2632-2153%2Fab891b%26sid%3Dliteratum%253Aachs%26jtitle%3DMachine%2520Learning%253A%2520Science%2520and%2520Technology%26atitle%3DPredicting%252Bdrug%252Bproperties%252Bwith%252Bparameter-free%252Bmachine%252Blearning%25253A%252Bpareto-optimal%252Bembedded%252Bmodeling%252B%252528POEM%252529%26aulast%3DBrereton%26aufirst%3DAndrew%2BE%26date%3D2020%26date%3D2020%26volume%3D1%26issue%3D2%26spage%3D025008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alla P.  Toropova</span>, <span class="hlFld-ContribAuthor ">Andrey A.  Toropov</span>. </span><span class="cited-content_cbyCitation_article-title">Whether the Validation of the Predictive Potential of Toxicity Models is a Solved Task?. </span><span class="cited-content_cbyCitation_journal-name">Current Topics in Medicinal Chemistry</span><span> <strong>2019,</strong> <em>19 </em>
                                    (29)
                                     , 2643-2657. <a href="https://doi.org/10.2174/1568026619666191105111817" title="DOI URL">https://doi.org/10.2174/1568026619666191105111817</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1568026619666191105111817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1568026619666191105111817%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Topics%2520in%2520Medicinal%2520Chemistry%26atitle%3DWhether%252Bthe%252BValidation%252Bof%252Bthe%252BPredictive%252BPotential%252Bof%252BToxicity%252BModels%252Bis%252Ba%252BSolved%252BTask%25253F%26aulast%3DToropova%26aufirst%3DAlla%2BP.%26date%3D2019%26volume%3D19%26issue%3D29%26spage%3D2643%26epage%3D2657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jennifer L  Wilson</span>. </span><span class="cited-content_cbyCitation_article-title">A scientist engineer’s contribution to therapeutic discovery and development. </span><span class="cited-content_cbyCitation_journal-name">Experimental Biology and Medicine</span><span> <strong>2018,</strong> <em>243 </em>
                                    (14)
                                     , 1125-1132. <a href="https://doi.org/10.1177/1535370218813974" title="DOI URL">https://doi.org/10.1177/1535370218813974</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/1535370218813974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F1535370218813974%26sid%3Dliteratum%253Aachs%26jtitle%3DExperimental%2520Biology%2520and%2520Medicine%26atitle%3DA%252Bscientist%252Bengineer%2525E2%252580%252599s%252Bcontribution%252Bto%252Btherapeutic%252Bdiscovery%252Band%252Bdevelopment%26aulast%3DWilson%26aufirst%3DJennifer%2BL%26date%3D2018%26date%3D2018%26volume%3D243%26issue%3D14%26spage%3D1125%26epage%3D1132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulio  Ragusa</span>, <span class="hlFld-ContribAuthor ">Serena  Bencivenni</span>, <span class="hlFld-ContribAuthor ">Paula  Morales</span>, <span class="hlFld-ContribAuthor ">Tyra  Callaway</span>, <span class="hlFld-ContribAuthor ">Dow P.  Hurst</span>, <span class="hlFld-ContribAuthor ">Battistina  Asproni</span>, <span class="hlFld-ContribAuthor ">Stefania  Merighi</span>, <span class="hlFld-ContribAuthor ">Giovanni  Loriga</span>, <span class="hlFld-ContribAuthor ">Gerard A.  Pinna</span>, <span class="hlFld-ContribAuthor ">Patricia H.  Reggio</span>, <span class="hlFld-ContribAuthor ">Stefania  Gessi</span>, <span class="hlFld-ContribAuthor ">Gabriele  Murineddu</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, Pharmacological Evaluation, and Docking Studies of Novel Pyridazinone-Based Cannabinoid Receptor Type 2 Ligands. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2018,</strong> <em>13 </em>
                                    (11)
                                     , 1102-1114. <a href="https://doi.org/10.1002/cmdc.201800152" title="DOI URL">https://doi.org/10.1002/cmdc.201800152</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201800152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201800152%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DSynthesis%25252C%252BPharmacological%252BEvaluation%25252C%252Band%252BDocking%252BStudies%252Bof%252BNovel%252BPyridazinone-Based%252BCannabinoid%252BReceptor%252BType%2525E2%252580%2525852%252BLigands%26aulast%3DRagusa%26aufirst%3DGiulio%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D11%26spage%3D1102%26epage%3D1114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mingyue  Zheng</span>, <span class="hlFld-ContribAuthor ">Jihui  Zhao</span>, <span class="hlFld-ContribAuthor ">Chen  Cui</span>, <span class="hlFld-ContribAuthor ">Zunyun  Fu</span>, <span class="hlFld-ContribAuthor ">Xutong  Li</span>, <span class="hlFld-ContribAuthor ">Xiaohong  Liu</span>, <span class="hlFld-ContribAuthor ">Xiaoyu  Ding</span>, <span class="hlFld-ContribAuthor ">Xiaoqin  Tan</span>, <span class="hlFld-ContribAuthor ">Fei  Li</span>, <span class="hlFld-ContribAuthor ">Xiaomin  Luo</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Computational chemical biology and drug design: Facilitating protein structure, function, and modulation studies. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2018,</strong> <em>38 </em>
                                    (3)
                                     , 914-950. <a href="https://doi.org/10.1002/med.21483" title="DOI URL">https://doi.org/10.1002/med.21483</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21483%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DComputational%252Bchemical%252Bbiology%252Band%252Bdrug%252Bdesign%25253A%252BFacilitating%252Bprotein%252Bstructure%25252C%252Bfunction%25252C%252Band%252Bmodulation%252Bstudies%26aulast%3DZheng%26aufirst%3DMingyue%26date%3D2018%26date%3D2018%26volume%3D38%26issue%3D3%26spage%3D914%26epage%3D950" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marilia N. N.  Lima</span>, <span class="hlFld-ContribAuthor ">Cleber C.  Melo-Filho</span>, <span class="hlFld-ContribAuthor ">Gustavo C.  Cassiano</span>, <span class="hlFld-ContribAuthor ">Bruno J.  Neves</span>, <span class="hlFld-ContribAuthor ">Vinicius M.  Alves</span>, <span class="hlFld-ContribAuthor ">Rodolpho C.  Braga</span>, <span class="hlFld-ContribAuthor ">Pedro V. L.  Cravo</span>, <span class="hlFld-ContribAuthor ">Eugene N.  Muratov</span>, <span class="hlFld-ContribAuthor ">Juliana  Calit</span>, <span class="hlFld-ContribAuthor ">Daniel Y.  Bargieri</span>, <span class="hlFld-ContribAuthor ">Fabio T. M.  Costa</span>, <span class="hlFld-ContribAuthor ">Carolina H.  Andrade</span>. </span><span class="cited-content_cbyCitation_article-title">QSAR-Driven Design and Discovery of Novel Compounds With Antiplasmodial and Transmission Blocking Activities. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2018,</strong> <em>9 </em><a href="https://doi.org/10.3389/fphar.2018.00146" title="DOI URL">https://doi.org/10.3389/fphar.2018.00146</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2018.00146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2018.00146%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DQSAR-Driven%252BDesign%252Band%252BDiscovery%252Bof%252BNovel%252BCompounds%252BWith%252BAntiplasmodial%252Band%252BTransmission%252BBlocking%252BActivities%26aulast%3DLima%26aufirst%3DMarilia%2BN.%2BN.%26date%3D2018%26date%3D2018%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/medium/jm-2016-015772_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/medium/jm-2016-015772_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic depicting the work flow and decision points during and after a typical HTS campaign. Orange boxes indicate assays conducted in a high-throughput format; blue boxes indicate assays conducted by the program team; the open arrow indicates the transition from the HTS screening group to the local project team.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/medium/jm-2016-015772_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Categorization accuracy of human P-gp efflux ratio by the QSAR model for human P-gp efflux ratio (A) and by PSA (B, C). QSAR data shown are prospective prediction values made for 2403 compounds measured between January 2015 and March 2016 for compounds from more than 50 therapeutic programs. Bar charts are colored by experimental P-gp efflux ratio in a MDR1-overexpressing LLC-PK1 cell line for compounds demonstrating good <i>P</i><sub>app</sub> and a lack of endogenous transport in a control LLC-PK1 cell line.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/medium/jm-2016-015772_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Representative isADMET heat maps for single hit series displaying potentially desirable (A) and undesirable (B) predicted property profiles. Rows represent individual compounds. In the legend, the red coloring used for in vivo CL<sub>int</sub> predictions of 0 accounts for compounds with predicted CL<sub>p</sub> values that exceed hepatic blood flow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/medium/jm-2016-015772_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Retrospective comparison of different hit selection methods for project 1. Data points shown are representative molecules from distinct chemical series that were biologically validated. Representative molecules were chosen to indicate the highest level of activity reached by a given series during project 1. Data points are jittered for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/medium/jm-2016-015772_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Plot of human microsomal CL<sub>int</sub> vs human P-gp efflux ratio for project 2 hits. Each data point reflects a representative molecule chosen to characterize a unique chemical series. Blue data points represent compounds chosen by the chemistry team for chemical validation; squares represent compounds chosen for their isADMET properties. Note that there are several overlapping compounds identified as blue squares.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/medium/jm-2016-015772_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) isADMET heat map for series 1 from project 2, which only contained two compounds from our sample collection. (B) isADMET heat map for a virtual library enumerated around the core of series 1 from project 2. Rows represent individual compounds. The boxed regions indicate where one R-group was held constant across library members (phenyl and oxazole, as indicated). In the legend, the red coloring used for in vivo CL<sub>int</sub> predictions of 0 accounts for compounds with predicted CL<sub>p</sub> values that exceed hepatic blood flow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/medium/jm-2016-015772_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Predicted vs measured scatterplots for human microsomal CL<sub>int</sub> (A, project 3; B, project 4), <i>P</i><sub>app</sub> (C, project 3; D, project 4), and human P-gp efflux ratio (E, project 3; F, project 4). Compounds are shaped and colored by series and the colors and shapes differ between projects 3 and 4. CL<sub>int</sub> values are measured in mL·min<sup>–1</sup>·kg<sup>–1</sup>; <i>P</i><sub>app</sub> values are measured in 10<sup>–6</sup> cm·s<sup>–1</sup>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/medium/jm-2016-015772_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. isADMET heat maps from project 5: (A) series 3, which was consistently predicted to bind to hERG; (B) series 4, which displayed less propensity for predicted hERG binding. Rows represent individual compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-16/acs.jmedchem.6b01577/20170818/images/large/jm-2016-015772_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01577&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i29">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26325" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26325" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 26 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Mayr, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuerst, P.</span><span> </span><span class="NLM_article-title">The future of high-throughput screening</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">443</span><span class="NLM_x">–</span> <span class="NLM_lpage">448</span><span class="refDoi"> DOI: 10.1177/1087057108319644</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1177%2F1087057108319644" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=18660458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpt1eitLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=443-448&author=L.+M.+Mayrauthor=P.+Fuerst&title=The+future+of+high-throughput+screening&doi=10.1177%2F1087057108319644"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The future of high-throughput screening</span></div><div class="casAuthors">Mayr, Lorenz M.; Fuerst, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">443-448</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) is a well-established process in lead discovery for pharma and biotech companies and is now also being set up for basic and applied research in academia and some research hospitals.  Since its first advent in the early to mid-1990s, the field of HTS has seen not only a continuous change in technol. and processes but also an adaptation to various needs in lead discovery.  HTS has now evolved into a quite mature discipline of modern drug discovery.  Whereas in previous years, much emphasis has been put toward a steady increase in capacity ("quant. increase") via various strategies in the fields of automation and miniaturization, the past years have seen a steady shift toward higher content and quality ("quality increase") for these biol. test systems.  Today, many experts in the field see HTS at the crossroads with the need to decide either toward further increase in throughput or more focus toward relevance of biol. data.  In this article, the authors describe the development of HTS over the past decade and point out their own ideas for future directions of HTS in biomedical research.  They predict that the trend toward further miniaturization will slow down with the implementation of 384-well, 1536-well, and 384 low-vol.-well plates.  The authors predict that, ultimately, each hit-finding strategy will be much more project related, tailor-made, and better integrated into the broader drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOoQxntLKDvbVg90H21EOLACvtfcHk0lgRIibszzKRuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpt1eitLk%253D&md5=7ed3a27bdc7526c8aadcdca01ed3640b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1177%2F1087057108319644&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057108319644%26sid%3Dliteratum%253Aachs%26aulast%3DMayr%26aufirst%3DL.%2BM.%26aulast%3DFuerst%26aufirst%3DP.%26atitle%3DThe%2520future%2520of%2520high-throughput%2520screening%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2008%26volume%3D13%26spage%3D443%26epage%3D448%26doi%3D10.1177%2F1087057108319644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Macarron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bojanic, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burns, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cirovic, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garyantes, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, D. V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hertzberg, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paslay, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schopfer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sittampalam, G. S.</span><span> </span><span class="NLM_article-title">Impact of high-throughput screening in biomedical research</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">188</span><span class="NLM_x">–</span> <span class="NLM_lpage">195</span><span class="refDoi"> DOI: 10.1038/nrd3368</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1038%2Fnrd3368" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=21358738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFensbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=188-195&author=R.+Macarronauthor=M.+N.+Banksauthor=D.+Bojanicauthor=D.+J.+Burnsauthor=D.+A.+Cirovicauthor=T.+Garyantesauthor=D.+V.+S.+Greenauthor=R.+P.+Hertzbergauthor=W.+P.+Janzenauthor=J.+W.+Paslayauthor=U.+Schopferauthor=G.+S.+Sittampalam&title=Impact+of+high-throughput+screening+in+biomedical+research&doi=10.1038%2Fnrd3368"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of high-throughput screening in biomedical research</span></div><div class="casAuthors">Macarron, Ricardo; Banks, Martyn N.; Bojanic, Dejan; Burns, David J.; Cirovic, Dragan A.; Garyantes, Tina; Green, Darren V. S.; Hertzberg, Robert P.; Janzen, William P.; Paslay, Jeff W.; Schopfer, Ulrich; Sittampalam, G. Sitta</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">188-195</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) has been postulated in several quarters to be a contributory factor to the decline in productivity in the pharmaceutical industry.  Moreover, it has been blamed for stifling the creativity that drug discovery demands.  In this article, we aim to dispel these myths and present the case for the use of HTS as part of a proven scientific tool kit, the wider use of which is essential for the discovery of new chemotypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLnrSVehADbLVg90H21EOLACvtfcHk0lgRIibszzKRuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFensbY%253D&md5=cb9853227506f3a3d774791cff9fd79d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd3368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3368%26sid%3Dliteratum%253Aachs%26aulast%3DMacarron%26aufirst%3DR.%26aulast%3DBanks%26aufirst%3DM.%2BN.%26aulast%3DBojanic%26aufirst%3DD.%26aulast%3DBurns%26aufirst%3DD.%2BJ.%26aulast%3DCirovic%26aufirst%3DD.%2BA.%26aulast%3DGaryantes%26aufirst%3DT.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DHertzberg%26aufirst%3DR.%2BP.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DPaslay%26aufirst%3DJ.%2BW.%26aulast%3DSchopfer%26aufirst%3DU.%26aulast%3DSittampalam%26aufirst%3DG.%2BS.%26atitle%3DImpact%2520of%2520high-throughput%2520screening%2520in%2520biomedical%2520research%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D188%26epage%3D195%26doi%3D10.1038%2Fnrd3368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Thorne, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auld, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span> </span><span class="NLM_article-title">Apparent activity in high-throughput screening: origins of compound-dependent assay interference</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">315</span><span class="NLM_x">–</span> <span class="NLM_lpage">324</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2010.03.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1016%2Fj.cbpa.2010.03.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=20417149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=315-324&author=N.+Thorneauthor=D.+S.+Auldauthor=J.+Inglese&title=Apparent+activity+in+high-throughput+screening%3A+origins+of+compound-dependent+assay+interference&doi=10.1016%2Fj.cbpa.2010.03.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Apparent activity in high-throughput screening: origins of compound-dependent assay interference</span></div><div class="casAuthors">Thorne, Natasha; Auld, Douglas S.; Inglese, James</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">315-324</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Expansive compd. collections made up of structurally heterogeneous chems., the activities of which are largely undefined, present challenging problems for high-throughput screening (HTS).  Foremost is differentiating whether the activity for a given compd. in an assay is directed against the targeted biol., or is the result of surreptitious compd. activity involving the assay detection system.  Such compd. interference can be esp. difficult to identify if it is reproducible and concn.-dependent - characteristics generally attributed to compds. with genuine activity.  While reactive chem. groups on compds. were once thought to be the primary source of compd. interference in assays used in HTS, recent work suggests that other factors, such as compd. aggregation, may play a more significant role in many assay formats.  Considerable progress has been made to profile representative compd. libraries in an effort to identify chem. classes susceptible to producing compd. interference, such as compds. commonly found to inhibit the reporter enzyme firefly luciferase.  Such work has also led to the development of practices that have the potential to significantly reduce compd. interference, for example, through the addn. of non-ionic detergent to assay buffer to reduce aggregation-based inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBawORe7gR97Vg90H21EOLACvtfcHk0ljHeHtfQbTtmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsL8%253D&md5=4d3dd71f66ce937002874106edaff12e</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.03.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.03.020%26sid%3Dliteratum%253Aachs%26aulast%3DThorne%26aufirst%3DN.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DInglese%26aufirst%3DJ.%26atitle%3DApparent%2520activity%2520in%2520high-throughput%2520screening%253A%2520origins%2520of%2520compound-dependent%2520assay%2520interference%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D315%26epage%3D324%26doi%3D10.1016%2Fj.cbpa.2010.03.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Hughes, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalindjian, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, K. L.</span><span> </span><span class="NLM_article-title">Principles of early drug discovery</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">162</span><span class="NLM_x">, </span> <span class="NLM_fpage">1239</span><span class="NLM_x">–</span> <span class="NLM_lpage">1249</span><span class="refDoi"> DOI: 10.1111/j.1476-5381.2010.01127.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1111%2Fj.1476-5381.2010.01127.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=21091654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivF2lt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=2011&pages=1239-1249&author=J.+P.+Hughesauthor=S.+Reesauthor=S.+B.+Kalindjianauthor=K.+L.+Philpott&title=Principles+of+early+drug+discovery&doi=10.1111%2Fj.1476-5381.2010.01127.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Principles of early drug discovery</span></div><div class="casAuthors">Hughes, J. P.; Rees, S.; Kalindjian, S. B.; Philpott, K. L.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1239-1249</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Developing a new drug from original idea to the launch of a finished product is a complex process which can take 12-15 years and cost in excess of $1 billion.  The idea for a target can come from a variety of sources including academic and clin. research and from the com. sector.  It may take many years to build up a body of supporting evidence before selecting a target for a costly drug discovery program.  Once a target was chosen, the pharmaceutical industry and more recently some academic centers have streamlined a no. of early processes to identify mols. which possess suitable characteristics to make acceptable drugs.  This review will look at key preclin. stages of the drug discovery process, from initial target identification and validation, through assay development, high throughput screening, hit identification, lead optimization and finally the selection of a candidate mol. for clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooCc74_F_90rVg90H21EOLACvtfcHk0ljHeHtfQbTtmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivF2lt7o%253D&md5=05a0016f45b155f9fb5822f5c7bcc35d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2010.01127.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2010.01127.x%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DJ.%2BP.%26aulast%3DRees%26aufirst%3DS.%26aulast%3DKalindjian%26aufirst%3DS.%2BB.%26aulast%3DPhilpott%26aufirst%3DK.%2BL.%26atitle%3DPrinciples%2520of%2520early%2520drug%2520discovery%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2011%26volume%3D162%26spage%3D1239%26epage%3D1249%26doi%3D10.1111%2Fj.1476-5381.2010.01127.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Kerns, E. H.; Di, L.</span> <span class="citation_source-book">Drug-like Properties: Concepts, Structure Design and Methods. From ADME to Toxicity Optimization</span>; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">San Diego, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=E.+H.+Kerns&author=L.+Di&title=Drug-like+Properties%3A+Concepts%2C+Structure+Design+and+Methods.+From+ADME+to+Toxicity+Optimization"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKerns%26aufirst%3DE.%2BH.%26btitle%3DDrug-like%2520Properties%253A%2520Concepts%252C%2520Structure%2520Design%2520and%2520Methods.%2520From%2520ADME%2520to%2520Toxicity%2520Optimization%26pub%3DAcademic%2520Press%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Bleicher, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bohm, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alanine, A. I.</span><span> </span><span class="NLM_article-title">Hit and lead generation: beyond high-throughput screening</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">369</span><span class="NLM_x">–</span> <span class="NLM_lpage">378</span><span class="refDoi"> DOI: 10.1038/nrd1086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1038%2Fnrd1086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=12750740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjslamtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=369-378&author=K.+H.+Bleicherauthor=H.+J.+Bohmauthor=K.+Mullerauthor=A.+I.+Alanine&title=Hit+and+lead+generation%3A+beyond+high-throughput+screening&doi=10.1038%2Fnrd1086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A guide to drug discovery: Hit and lead generation: beyond high-throughput screening</span></div><div class="casAuthors">Bleicher, Konrad H.; Boehm, Hans-Joachim; Mueller, Klaus; Alanine, Alexander I.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">369-378</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The identification of small-mol. modulators of protein function, and the process of transforming these into high-content lead series, are key activities in modern drug discovery.  The decisions taken during this process have far-reaching consequences for success later in lead optimization and even more crucially in clin. development.  Recently, there has been an increased focus on these activities due to escalating downstream costs resulting from high clin. failure rates.  In addn., the vast emerging opportunities from efforts in functional genomics and proteomics demands a departure from the linear process of identification, evaluation and refinement activities towards a more integrated parallel process.  This calls for flexible, fast and cost-effective strategies to meet the demands of producing high-content lead series with improved prospects for clin. success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9IQCsTR8tobVg90H21EOLACvtfcHk0ljHeHtfQbTtmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjslamtb8%253D&md5=b54e55c1f0d34fbd172c8aa9b996591d</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd1086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1086%26sid%3Dliteratum%253Aachs%26aulast%3DBleicher%26aufirst%3DK.%2BH.%26aulast%3DBohm%26aufirst%3DH.%2BJ.%26aulast%3DMuller%26aufirst%3DK.%26aulast%3DAlanine%26aufirst%3DA.%2BI.%26atitle%3DHit%2520and%2520lead%2520generation%253A%2520beyond%2520high-throughput%2520screening%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D369%26epage%3D378%26doi%3D10.1038%2Fnrd1086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Davis, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keeling, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steele, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomkinson, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tinker, A. C.</span><span> </span><span class="NLM_article-title">Components of successful lead generation</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">421</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.2174/1568026053828411</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.2174%2F1568026053828411" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=15892684" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkvFGhtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2005&pages=421-439&author=A.+M.+Davisauthor=D.+J.+Keelingauthor=J.+Steeleauthor=N.+P.+Tomkinsonauthor=A.+C.+Tinker&title=Components+of+successful+lead+generation&doi=10.2174%2F1568026053828411"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Components of successful lead generation</span></div><div class="casAuthors">Davis, Andrew M.; Keeling, David J.; Steele, John; Tomkinson, Nicholas P.; Tinker, Alan C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">421-439</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  This review aims to give an overview of current good practice in the prosecution of Lead Generation.  It will assess experiences across the field as judged from the contents of the limited no. of peer-review disclosures to date.  It will also rely heavily on the experiences of the authors from many campaigns within this organization.  Its focus will be on the assembly of an appropriate compd. collection for application in High Throughput Screening (HTS), the prosecution of HTS, the profiling of HTS output and, lastly the Hit-to-Lead optimization of selected HTS output.  Excluded from the scope are detailed aspects of library design [1], parallel synthesis [2], virtual library applications [3], virtual screening [4] and fragment screening [5] approaches, all of which have been the subject of recent reviews.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox5UYz5NF4m7Vg90H21EOLACvtfcHk0ljHeHtfQbTtmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkvFGhtL4%253D&md5=ef29c85464f4ecbc63787210fda1b778</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2174%2F1568026053828411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026053828411%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DA.%2BM.%26aulast%3DKeeling%26aufirst%3DD.%2BJ.%26aulast%3DSteele%26aufirst%3DJ.%26aulast%3DTomkinson%26aufirst%3DN.%2BP.%26aulast%3DTinker%26aufirst%3DA.%2BC.%26atitle%3DComponents%2520of%2520successful%2520lead%2520generation%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2005%26volume%3D5%26spage%3D421%26epage%3D439%26doi%3D10.2174%2F1568026053828411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Ballard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brassil, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bui, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dolgos, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petersson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tunek, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webborn, P. J.</span><span> </span><span class="NLM_article-title">The right compound in the right assay at the right time: an integrated discovery DMPK strategy</span> <span class="citation_source-journal">Drug Metab. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">252</span><span class="refDoi"> DOI: 10.3109/03602532.2012.691099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.3109%2F03602532.2012.691099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=22697420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVelsLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=224-252&author=P.+Ballardauthor=P.+Brassilauthor=K.+H.+Buiauthor=H.+Dolgosauthor=C.+Peterssonauthor=A.+Tunekauthor=P.+J.+Webborn&title=The+right+compound+in+the+right+assay+at+the+right+time%3A+an+integrated+discovery+DMPK+strategy&doi=10.3109%2F03602532.2012.691099"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The right compound in the right assay at the right time: an integrated discovery DMPK strategy</span></div><div class="casAuthors">Ballard, Peter; Brassil, Patrick; Bui, Khanh H.; Dolgos, Hugues; Petersson, Carl; Tunek, Anders; Webborn, Peter J. H.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">224-252</span>CODEN:
                <span class="NLM_cas:coden">DMTRAR</span>;
        ISSN:<span class="NLM_cas:issn">0360-2532</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  The high rate of attrition during drug development and its assocd. high research and development (R&D) cost have put pressure on pharmaceutical companies to ensure that candidate drugs going to clin. testing have the appropriate quality such that the biol. hypothesis could be evaluated.  To help achieve this ambition, drug metab. and pharmacokinetic (DMPK) science and increasing investment have been deployed earlier in the R&D process.  To gain max. return on investment, it is essential that DMPK concepts are both appropriately integrated into the compd. design process and that compd. selection is focused on accurate prediction of likely outcomes in patients.  This article describes key principles that underpin the contribution of DMPK science for small-mol. research based on 15 years of discovery support in a major pharmaceutical company.  It does not aim to describe the breadth and depth of DMPK science, but more the practical application for decision making in real-world situations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolEqf17mivQbVg90H21EOLACvtfcHk0lhgUwb8QRPTZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVelsLzF&md5=105fbbcfba6213caaadf41c8b10cf408</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.3109%2F03602532.2012.691099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F03602532.2012.691099%26sid%3Dliteratum%253Aachs%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DBrassil%26aufirst%3DP.%26aulast%3DBui%26aufirst%3DK.%2BH.%26aulast%3DDolgos%26aufirst%3DH.%26aulast%3DPetersson%26aufirst%3DC.%26aulast%3DTunek%26aufirst%3DA.%26aulast%3DWebborn%26aufirst%3DP.%2BJ.%26atitle%3DThe%2520right%2520compound%2520in%2520the%2520right%2520assay%2520at%2520the%2520right%2520time%253A%2520an%2520integrated%2520discovery%2520DMPK%2520strategy%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D2012%26volume%3D44%26spage%3D224%26epage%3D252%26doi%3D10.3109%2F03602532.2012.691099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">van de Waterbeemd, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gifford, E.</span><span> </span><span class="NLM_article-title">ADMET in silico modelling: towards prediction paradise?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">192</span><span class="NLM_x">–</span> <span class="NLM_lpage">204</span><span class="refDoi"> DOI: 10.1038/nrd1032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1038%2Fnrd1032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=12612645" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhsFKrtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=192-204&author=H.+van+de+Waterbeemdauthor=E.+Gifford&title=ADMET+in+silico+modelling%3A+towards+prediction+paradise%3F&doi=10.1038%2Fnrd1032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">ADMET in silico modelling: Towards prediction paradise?</span></div><div class="casAuthors">van de Waterbeemd, Han; Gifford, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">192-204</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with refs.  Following studies in the late 1990s that indicated that poor pharmacokinetics and toxicity were important causes of costly late-stage failures in drug development, it has become widely appreciated that these areas should be considered as early as possible in the drug discovery process.  However, in recent years, combinatorial chem. and high-throughput screening have significantly increased the no. of compds. for which early data on absorption, distribution, metab., excretion (ADME) and toxicity (T) are needed, which has in turn driven the development of a variety of medium and high-throughput in vitro ADMET screens.  Here, we describe how in silico approaches will further increase our ability to predict and model the most relevant pharmacokinetic, metabolic and toxicity endpoints, thereby accelerating the drug discovery process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr8bI29kgKZLVg90H21EOLACvtfcHk0lhgUwb8QRPTZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhsFKrtLo%253D&md5=898a1edb3b33a862185c889930314439</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrd1032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1032%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bde%2BWaterbeemd%26aufirst%3DH.%26aulast%3DGifford%26aufirst%3DE.%26atitle%3DADMET%2520in%2520silico%2520modelling%253A%2520towards%2520prediction%2520paradise%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D192%26epage%3D204%26doi%3D10.1038%2Fnrd1032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominy, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feeney, P. J.</span><span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span> <span class="citation_source-journal">Adv. Drug Delivery Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span><span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0lhgUwb8QRPTZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Leeson, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Springthorpe, B.</span><span> </span><span class="NLM_article-title">The influence of drug-like concepts on decision-making in medicinal chemistry</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">881</span><span class="NLM_x">–</span> <span class="NLM_lpage">890</span><span class="refDoi"> DOI: 10.1038/nrd2445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1038%2Fnrd2445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=17971784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=881-890&author=P.+D.+Leesonauthor=B.+Springthorpe&title=The+influence+of+drug-like+concepts+on+decision-making+in+medicinal+chemistry&doi=10.1038%2Fnrd2445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The influence of drug-like concepts on decision-making in medicinal chemistry</span></div><div class="casAuthors">Leeson, Paul D.; Springthorpe, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">881-890</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite the wide acceptance of drug-like principles such as the 'rule of five', this anal. of mols. currently being synthesized in leading pharmaceutical companies reveals that their phys. properties differ significantly from those of recently discovered oral drugs.  The marked increase in lipophilicity in particular could increase the likelihood of attrition in drug development.  The application of guidelines linked to the concept of drug-likeness, such as the 'rule of five', has gained wide acceptance as an approach to reduce attrition in drug discovery and development.  However, despite this acceptance, anal. of recent trends reveals that the phys. properties of mols. that are currently being synthesized in leading drug discovery companies differ significantly from those of recently discovered oral drugs and compds. in clin. development.  The consequences of the marked increase in lipophilicity - the most important drug-like phys. property - include a greater likelihood of lack of selectivity and attrition in drug development.  Tackling the threat of compd.-related toxicol. attrition needs to move to the mainstream of medicinal chem. decision-making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTUeonLD6iebVg90H21EOLACvtfcHk0ljj0f9ItErfcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL&md5=5665a9e2dc266a1ee096e20625757ef7</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrd2445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2445%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DSpringthorpe%26aufirst%3DB.%26atitle%3DThe%2520influence%2520of%2520drug-like%2520concepts%2520on%2520decision-making%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D881%26epage%3D890%26doi%3D10.1038%2Fnrd2445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Bueters, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Visser, S. A.</span><span> </span><span class="NLM_article-title">Optimization of human dose prediction by using quantitative and translational pharmacology in drug discovery</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">2351</span><span class="NLM_x">–</span> <span class="NLM_lpage">2369</span><span class="refDoi"> DOI: 10.4155/fmc.15.143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.4155%2Ffmc.15.143" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=26599348" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFKlu7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=2351-2369&author=T.+Buetersauthor=C.+Gibsonauthor=S.+A.+Visser&title=Optimization+of+human+dose+prediction+by+using+quantitative+and+translational+pharmacology+in+drug+discovery&doi=10.4155%2Ffmc.15.143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of human dose prediction by using quantitative and translational pharmacology in drug discovery</span></div><div class="casAuthors">Bueters, Tjerk; Gibson, Christopher; G Visser, Sandra A.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2351-2369</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">In this perspective article, we explain how quant. and translational pharmacol., when well-implemented, is believed to lead to improved clin. candidates and drug targets that are differentiated from current treatment options.  Quant. and translational pharmacol. aims to build and continuously improve the quant. relationship between drug exposure, target engagement, efficacy, safety and its interspecies relationship at every phase of drug discovery.  Drug hunters should consider and apply these concepts to develop compds. with a higher probability of interrogating the clin. biol. hypothesis.  We offer different approaches to set an initial effective concn. or pharmacokinetic-pharmacodynamic target in man and to predict human pharmacokinetics that det. together the predicted human dose and dose schedule.  All concepts are illustrated with ample literature examples.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoy3M0R3KWa1LVg90H21EOLACvtfcHk0ljj0f9ItErfcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFKlu7%252FO&md5=c30f4756c965a5dbca10d7b5c57b59c9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.4155%2Ffmc.15.143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.15.143%26sid%3Dliteratum%253Aachs%26aulast%3DBueters%26aufirst%3DT.%26aulast%3DGibson%26aufirst%3DC.%26aulast%3DVisser%26aufirst%3DS.%2BA.%26atitle%3DOptimization%2520of%2520human%2520dose%2520prediction%2520by%2520using%2520quantitative%2520and%2520translational%2520pharmacology%2520in%2520drug%2520discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2015%26volume%3D7%26spage%3D2351%26epage%3D2369%26doi%3D10.4155%2Ffmc.15.143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Gleeson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montanari, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J.</span><span> </span><span class="NLM_article-title">Probing the links between in vitro potency, ADMET and physicochemical parameters</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">197</span><span class="NLM_x">–</span> <span class="NLM_lpage">208</span><span class="refDoi"> DOI: 10.1038/nrd3367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1038%2Fnrd3367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=21358739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFemsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=197-208&author=M.+P.+Gleesonauthor=A.+Herseyauthor=D.+Montanariauthor=J.+Overington&title=Probing+the+links+between+in+vitro+potency%2C+ADMET+and+physicochemical+parameters&doi=10.1038%2Fnrd3367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the links between in vitro potency, ADMET and physicochemical parameters</span></div><div class="casAuthors">Gleeson, M. Paul; Hersey, Anne; Montanari, Dino; Overington, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">197-208</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A common underlying assumption in current drug discovery strategies is that compds. with higher in vitro potency at their target(s) have greater potential to translate into successful, low-dose therapeutics.  This has led to the development of screening cascades with in vitro potency embedded as an early filter.  However, this approach is beginning to be questioned, given the bias in physicochem. properties that it can introduce early in lead generation and optimization, which is due to the often diametrically opposed relationship between physicochem. parameters assocd. with high in vitro potency and those assocd. with desirable absorption, distribution, metab., excretion and toxicity (ADMET) characteristics.  Here, we describe analyses that probe these issues further using the ChEMBL database, which includes more than 500,000 drug discovery and marketed oral drug compds.  Key findings include: first, that oral drugs seldom possess nanomolar potency (50 nM on av.); second, that many oral drugs have considerable off-target activity; and third, that in vitro potency does not correlate strongly with the therapeutic dose.  These findings suggest that the perceived benefit of high in vitro potency may be negated by poorer ADMET properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovOslFSgIkUbVg90H21EOLACvtfcHk0ljj0f9ItErfcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFemsr8%253D&md5=e275044516e9b0245b71c9603477ef74</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnrd3367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3367%26sid%3Dliteratum%253Aachs%26aulast%3DGleeson%26aufirst%3DM.%2BP.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DMontanari%26aufirst%3DD.%26aulast%3DOverington%26aufirst%3DJ.%26atitle%3DProbing%2520the%2520links%2520between%2520in%2520vitro%2520potency%252C%2520ADMET%2520and%2520physicochemical%2520parameters%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D197%26epage%3D208%26doi%3D10.1038%2Fnrd3367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Hop, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davidson, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duignan, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federico, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janiszewski, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krueger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lebowitz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liston, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchell, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snyder, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steyn, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soglia, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troutman, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umland, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whalen, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zelesky, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, S. X.</span><span> </span><span class="NLM_article-title">High throughput ADME screening: practical considerations, impact on the portfolio and enabler of in silico ADME models</span> <span class="citation_source-journal">Curr. Drug Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">847</span><span class="NLM_x">–</span> <span class="NLM_lpage">853</span><span class="refDoi"> DOI: 10.2174/138920008786485092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.2174%2F138920008786485092" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=18991580" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivF2nsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=847-853&author=C.+E.+Hopauthor=M.+J.+Coleauthor=R.+E.+Davidsonauthor=D.+B.+Duignanauthor=J.+Federicoauthor=J.+S.+Janiszewskiauthor=K.+Jenkinsauthor=S.+Kruegerauthor=R.+Lebowitzauthor=T.+E.+Listonauthor=W.+Mitchellauthor=M.+Snyderauthor=S.+J.+Steynauthor=J.+R.+Sogliaauthor=C.+Taylorauthor=M.+D.+Troutmanauthor=J.+Umlandauthor=M.+Westauthor=K.+M.+Whalenauthor=V.+Zeleskyauthor=S.+X.+Zhao&title=High+throughput+ADME+screening%3A+practical+considerations%2C+impact+on+the+portfolio+and+enabler+of+in+silico+ADME+models&doi=10.2174%2F138920008786485092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">High throughput ADME screening: practical considerations, impact on the portfolio and enabler of in silico ADME models</span></div><div class="casAuthors">Hop, Cornelis E. C. A.; Cole, Mark J.; Davidson, Ralph E.; Duignan, David B.; Federico, James; Janiszewski, John S.; Jenkins, Kelly; Krueger, Suzanne; Lebowitz, Rebecca; Liston, Theodore E.; Mitchell, Walter; Snyder, Mark; Steyn, Stefan J.; Soglia, John R.; Taylor, Christine; Troutman, Matt D.; Umland, John; West, Michael; Whalen, Kevin M.; Zelesky, Veronica; Zhao, Sabrina X.</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Metabolism</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">847-853</span>CODEN:
                <span class="NLM_cas:coden">CDMUBU</span>;
        ISSN:<span class="NLM_cas:issn">1389-2002</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Evaluation and optimization of drug metab. and pharmacokinetic data plays an important role in drug discovery and development and several reliable in vitro ADME models are available.  Recently higher throughput in vitro ADME screening facilities have been established in order to be able to evaluate an appreciable fraction of synthesized compds.  The ADME screening process can be dissected in five distinct steps: (1) plate management of compds. in need of in vitro ADME data, (2) optimization of the MS/MS method for the compds., (3) in vitro ADME expts. and sample clean up, (4) collection and redn. of the raw LC-MS/MS data and (5) archival of the processed ADME data.  All steps will be described in detail and the value of the data on drug discovery projects will be discussed as well.  Finally, in vitro ADME screening can generate large quantities of data obtained under identical conditions to allow building of reliable in silico models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHIB4wuoFXErVg90H21EOLACvtfcHk0lg2xZvNJOgajA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivF2nsLk%253D&md5=69acd6725561ddb30157d22dab69e329</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.2174%2F138920008786485092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138920008786485092%26sid%3Dliteratum%253Aachs%26aulast%3DHop%26aufirst%3DC.%2BE.%26aulast%3DCole%26aufirst%3DM.%2BJ.%26aulast%3DDavidson%26aufirst%3DR.%2BE.%26aulast%3DDuignan%26aufirst%3DD.%2BB.%26aulast%3DFederico%26aufirst%3DJ.%26aulast%3DJaniszewski%26aufirst%3DJ.%2BS.%26aulast%3DJenkins%26aufirst%3DK.%26aulast%3DKrueger%26aufirst%3DS.%26aulast%3DLebowitz%26aufirst%3DR.%26aulast%3DListon%26aufirst%3DT.%2BE.%26aulast%3DMitchell%26aufirst%3DW.%26aulast%3DSnyder%26aufirst%3DM.%26aulast%3DSteyn%26aufirst%3DS.%2BJ.%26aulast%3DSoglia%26aufirst%3DJ.%2BR.%26aulast%3DTaylor%26aufirst%3DC.%26aulast%3DTroutman%26aufirst%3DM.%2BD.%26aulast%3DUmland%26aufirst%3DJ.%26aulast%3DWest%26aufirst%3DM.%26aulast%3DWhalen%26aufirst%3DK.%2BM.%26aulast%3DZelesky%26aufirst%3DV.%26aulast%3DZhao%26aufirst%3DS.%2BX.%26atitle%3DHigh%2520throughput%2520ADME%2520screening%253A%2520practical%2520considerations%252C%2520impact%2520on%2520the%2520portfolio%2520and%2520enabler%2520of%2520in%2520silico%2520ADME%2520models%26jtitle%3DCurr.%2520Drug%2520Metab.%26date%3D2008%26volume%3D9%26spage%3D847%26epage%3D853%26doi%3D10.2174%2F138920008786485092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Leeson, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. J.</span><span> </span><span class="NLM_article-title">Molecular property design: does everyone get it?</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">722</span><span class="NLM_x">–</span> <span class="NLM_lpage">725</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00157</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00157" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptVGksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=722-725&author=P.+D.+Leesonauthor=R.+J.+Young&title=Molecular+property+design%3A+does+everyone+get+it%3F&doi=10.1021%2Facsmedchemlett.5b00157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Property Design: Does Everyone Get It?</span></div><div class="casAuthors">Leeson, Paul D.; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">722-725</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The principles of mol. property optimization in drug design have been understood for decades, yet much drug discovery activity today is conducted at the periphery of historical druglike property space.  Lead optimization trajectories aimed at reducing physicochem. risk, assisted by ligand efficiency metrics, could help to reduce clin. attrition rates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_t8gIn79arVg90H21EOLACvtfcHk0lg2xZvNJOgajA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptVGksr4%253D&md5=5a807a8d049ce8cf4adc186891a9234d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00157%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DMolecular%2520property%2520design%253A%2520does%2520everyone%2520get%2520it%253F%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D722%26epage%3D725%26doi%3D10.1021%2Facsmedchemlett.5b00157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Desai, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawada, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raub, T. J.</span><span> </span><span class="NLM_article-title">Integration of in silico and in vitro tools for scaffold optimization during drug discovery: predicting P-glycoprotein efflux</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1249</span><span class="NLM_x">–</span> <span class="NLM_lpage">1261</span><span class="refDoi"> DOI: 10.1021/mp300555n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/mp300555n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1aju7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=1249-1261&author=P.+V.+Desaiauthor=G.+A.+Sawadaauthor=I.+A.+Watsonauthor=T.+J.+Raub&title=Integration+of+in+silico+and+in+vitro+tools+for+scaffold+optimization+during+drug+discovery%3A+predicting+P-glycoprotein+efflux&doi=10.1021%2Fmp300555n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Integration of in Silico and in Vitro Tools for Scaffold Optimization during Drug Discovery: Predicting P-Glycoprotein Efflux</span></div><div class="casAuthors">Desai, Prashant V.; Sawada, Geri A.; Watson, Ian A.; Raub, Thomas J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1249-1261</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In silico tools are regularly utilized for designing and prioritizing compds. to address challenges related to drug metab. and pharmacokinetics (DMPK) during the process of drug discovery.  P-Glycoprotein (P-gp) is a member of the ATP-binding cassette (ABC) transporters with broad substrate specificity that plays a significant role in absorption and distribution of drugs that are P-gp substrates.  As a result, screening for P-gp transport has now become routine in the drug discovery process.  Typically, bidirectional permeability assays are employed to assess in vitro P-gp efflux.  In this article, we use P-gp as an example to illustrate a well-validated methodol. to effectively integrate in silico and in vitro tools to identify and resolve key barriers during the early stages of drug discovery.  A detailed account of development and application of in silico tools such as simple guidelines based on physicochem. properties and more complex quant. structure-activity relationship (QSAR) models is provided.  The tools were developed based on structurally diverse data for more than 2000 compds. generated using a robust P-gp substrate assay over the past several years.  Anal. of physicochem. properties revealed a significantly lower proportion (<10%) of P-gp substrates among the compds. with topol. polar surface area (TPSA) <60 Å2 and the most basic cpKa <8.  In contrast, this proportion of substrates was greater than 75% for compds. with TPSA >60 Å2 and the most basic cpKa >8.  Among the various QSAR models evaluated to predict P-gp efflux, the Bagging model provided optimum prediction performance for prospective validation based on chronol. test sets.  Four sequential versions of the model were built with increasing nos. of compds. to train the models as new data became available.  Except for the first version with the smallest training set, the QSAR models exhibited robust prediction profiles with pos. prediction values (PPV) and neg. prediction values (NPV) exceeding 80%.  The QSAR model demonstrated better concordance with the manual P-gp substrate assay than an automated P-gp substrate screen.  The in silico and the in vitro tools have been effectively integrated during early stages of drug discovery to resolve P-gp-related challenges exemplified by several case studies.  Key learning based on our experience with P-gp can be widely applicable across other DMPK-related challenges.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Q5Y8HssxH7Vg90H21EOLACvtfcHk0lg2xZvNJOgajA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1aju7o%253D&md5=3bd5239b80270c0f4ed85dffb86704ed</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fmp300555n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp300555n%26sid%3Dliteratum%253Aachs%26aulast%3DDesai%26aufirst%3DP.%2BV.%26aulast%3DSawada%26aufirst%3DG.%2BA.%26aulast%3DWatson%26aufirst%3DI.%2BA.%26aulast%3DRaub%26aufirst%3DT.%2BJ.%26atitle%3DIntegration%2520of%2520in%2520silico%2520and%2520in%2520vitro%2520tools%2520for%2520scaffold%2520optimization%2520during%2520drug%2520discovery%253A%2520predicting%2520P-glycoprotein%2520efflux%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2013%26volume%3D10%26spage%3D1249%26epage%3D1261%26doi%3D10.1021%2Fmp300555n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Sherer, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verras, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madeira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagmann, W. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bleasby, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cornell, W. D.</span><span> </span><span class="NLM_article-title">QSAR prediction of passive permeability in the LLC-PK1 cell line: trends in molecular properties and cross-prediction of Caco-2 permeabilities</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">231</span><span class="NLM_x">–</span> <span class="NLM_lpage">245</span><span class="refDoi"> DOI: 10.1002/minf.201100157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1002%2Fminf.201100157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=27477094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC38XjsFOnt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=231-245&author=E.+C.+Shererauthor=A.+Verrasauthor=M.+Madeiraauthor=W.+K.+Hagmannauthor=R.+P.+Sheridanauthor=D.+Robertsauthor=K.+Bleasbyauthor=W.+D.+Cornell&title=QSAR+prediction+of+passive+permeability+in+the+LLC-PK1+cell+line%3A+trends+in+molecular+properties+and+cross-prediction+of+Caco-2+permeabilities&doi=10.1002%2Fminf.201100157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">QSAR Prediction of Passive Permeability in the LLC-PK1 Cell Line: Trends in Molecular Properties and Cross-Prediction of Caco-2 Permeabilities</span></div><div class="casAuthors">Sherer, Edward C.; Verras, Andreas; Madeira, Maria; Hagmann, William K.; Sheridan, Robert P.; Roberts, Drew; Bleasby, Kelly; Cornell, Wendy D.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3-4</span>),
    <span class="NLM_cas:pages">231-245</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A QSAR model for predicting passive permeability (Papp) was derived from Papp values measured in the LLC-PK1 cell line.  The QSAR method and descriptor set that performed best in terms of cross-validation was random forest with a combination of AP, DP, and MOE2D descriptors.  The QSAR model was used to predict the Caco-2 cell permeability for 313 compds. described in the literature with good success.  We find that passive permeability for different cell lines can be predicted with similar mol. properties and descriptors.  It is shown that the variation in exptl. measurements of Papp is smaller than the error in QSAR predictions indicating that predictions are not quant. perfect, although qual. useful.  We get better predictions if the training set is large and diverse, rather than smaller and more internally consistent.  This is because prediction accuracy falls off quickly with decreasing similarity to the training set and it is therefore better to have as large a training set as possible.  While single phys. parameters are not as good as a full QSAR model in predicting Papp, logD seems the most important parameter.  Intermediate values of logD are assocd. with higher Papp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-PMQffMXQQrVg90H21EOLACvtfcHk0lhJoaMkergqFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjsFOnt7s%253D&md5=e21d5d9a43bae6f9ec7a9c5dadf1bfa1</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fminf.201100157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201100157%26sid%3Dliteratum%253Aachs%26aulast%3DSherer%26aufirst%3DE.%2BC.%26aulast%3DVerras%26aufirst%3DA.%26aulast%3DMadeira%26aufirst%3DM.%26aulast%3DHagmann%26aufirst%3DW.%2BK.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DRoberts%26aufirst%3DD.%26aulast%3DBleasby%26aufirst%3DK.%26aulast%3DCornell%26aufirst%3DW.%2BD.%26atitle%3DQSAR%2520prediction%2520of%2520passive%2520permeability%2520in%2520the%2520LLC-PK1%2520cell%2520line%253A%2520trends%2520in%2520molecular%2520properties%2520and%2520cross-prediction%2520of%2520Caco-2%2520permeabilities%26jtitle%3DMol.%2520Inf.%26date%3D2012%26volume%3D31%26spage%3D231%26epage%3D245%26doi%3D10.1002%2Fminf.201100157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMasters, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voigt, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wildey, M. J.</span><span> </span><span class="NLM_article-title">eCounterscreening: using QSAR predictions to prioritize testing for off-target activities and setting the balance between benefit and risk</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">231</span><span class="NLM_x">–</span> <span class="NLM_lpage">238</span><span class="refDoi"> DOI: 10.1021/ci500666m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500666m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFKgsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=231-238&author=R.+P.+Sheridanauthor=D.+R.+McMastersauthor=J.+H.+Voigtauthor=M.+J.+Wildey&title=eCounterscreening%3A+using+QSAR+predictions+to+prioritize+testing+for+off-target+activities+and+setting+the+balance+between+benefit+and+risk&doi=10.1021%2Fci500666m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">eCounterscreening: Using QSAR Predictions to Prioritize Testing for Off-Target Activities and Setting the Balance between Benefit and Risk</span></div><div class="casAuthors">Sheridan, Robert P.; McMasters, Daniel R.; Voigt, Johannes H.; Wildey, Mary Jo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">231-238</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">During drug development, compds. are tested against counterscreens, a panel of off-target activities that would be undesirable for a drug to have.  Testing every compd. against every counterscreen is generally too costly in terms of time and money, and we need to find a rational way of prioritizing counterscreen testing.  Here we present the eCounterscreening paradigm, wherein predictions from QSAR models for counterscreen activity are used to generate a recommendation as to whether a specific compd. in a specific project should be tested against a specific counterscreen.  The rules behind the recommendations, which can be summarized in a risk-benefit plot specific for a counterscreen/project combination, are based on a previously assembled database of prospective QSAR predictions.  The recommendations require two user-defined cutoffs: the level of activity in a specific counterscreen that is considered undesirable and the level of risk the chemist is willing to accept that an undesired counterscreen activity will go undetected.  We demonstrate in a simulated prospective expt. that eCounterscreening can be used to postpone a large fraction of counterscreen testing and still have an acceptably low risk of undetected counterscreen activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjw-Fazb67UbVg90H21EOLACvtfcHk0lhJoaMkergqFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFKgsw%253D%253D&md5=7e433fcc51077f0cce1fe04dc65f47cf</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fci500666m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500666m%26sid%3Dliteratum%253Aachs%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DMcMasters%26aufirst%3DD.%2BR.%26aulast%3DVoigt%26aufirst%3DJ.%2BH.%26aulast%3DWildey%26aufirst%3DM.%2BJ.%26atitle%3DeCounterscreening%253A%2520using%2520QSAR%2520predictions%2520to%2520prioritize%2520testing%2520for%2520off-target%2520activities%2520and%2520setting%2520the%2520balance%2520between%2520benefit%2520and%2520risk%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D231%26epage%3D238%26doi%3D10.1021%2Fci500666m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Mahar Doan, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphreys, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wring, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shampine, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serabjit-Singh, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adkison, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polli, J. W.</span><span> </span><span class="NLM_article-title">Passive permeability and P-glycoprotein-mediated efflux differentiate central nervous system (CNS) and non-CNS marketed drugs</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">1029</span><span class="NLM_x">–</span> <span class="NLM_lpage">1037</span><span class="refDoi"> DOI: 10.1124/jpet.102.039255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1124%2Fjpet.102.039255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=12438524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A280%3ADC%252BD38nptVKmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2002&pages=1029-1037&author=K.+M.+Mahar+Doanauthor=J.+E.+Humphreysauthor=L.+O.+Websterauthor=S.+A.+Wringauthor=L.+J.+Shampineauthor=C.+J.+Serabjit-Singhauthor=K.+K.+Adkisonauthor=J.+W.+Polli&title=Passive+permeability+and+P-glycoprotein-mediated+efflux+differentiate+central+nervous+system+%28CNS%29+and+non-CNS+marketed+drugs&doi=10.1124%2Fjpet.102.039255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Passive permeability and P-glycoprotein-mediated efflux differentiate central nervous system (CNS) and non-CNS marketed drugs</span></div><div class="casAuthors">Mahar Doan Kelly M; Humphreys Joan E; Webster Lindsey O; Wring Stephen A; Shampine Larry J; Serabjit-Singh Cosette J; Adkison Kimberly K; Polli Joseph W</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of pharmacology and experimental therapeutics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1029-37</span>
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    </div><div class="casAbstract">Membrane permeability and P-glycoprotein (Pgp) can be limiting factors for blood-brain barrier penetration.  The objectives of this study were to determine whether there are differences in the in vitro permeability, Pgp substrate profiles, and physicochemical properties of drugs for central nervous system (CNS) and non-CNS indications, and whether these differences are useful criteria in selecting compounds for drug development.  Apparent permeability (P(app)) and Pgp substrate profiles for 93 CNS (n = 48) and non-CNS (n = 45) drugs were determined by monolayer efflux.  Calcein-AM inhibition assays were used to supplement the efflux results.  The CNS set (2 of 48, 4.2%) had a 7-fold lower incidence of passive permeability values <150 nm/s compared with the non-CNS set (13 of 45, 28.9%).  The majority of drugs (72.0%, 67 of 93) were not Pgp substrates; however, 49.5% (46 of 93) were positive in the calcein-AM assay when tested at 100 microM.  The CNS drug set (n = 7 of 48, 14.6%) had a 3-fold lower incidence of Pgp-mediated efflux than the non-CNS drug set (n = 19 of 45, 42.2%).  Analysis of 18 physicochemical properties revealed that the CNS drug set had fewer hydrogen bond donors, fewer positive charges, greater lipophilicity, lower polar surface area, and reduced flexibility compared with the non-CNS group (p < 0.05), properties that enhance membrane permeability.  This study on a large, diverse set of marketed compounds clearly demonstrates that permeability, Pgp-mediated efflux, and certain physicochemical properties are factors that differentiate CNS and non-CNS drugs.  For CNS delivery, a drug should ideally have an in vitro passive permeability >150 nm/s and not be a good (B --> A/A --> B ratio <2.5) Pgp substrate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT_HiV1Xqt18Vf1XRtfYM0dfW6udTcc2eb6siARjqDinbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD38nptVKmsw%253D%253D&md5=b841902aa8502de47c1e1603c80e09fb</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1124%2Fjpet.102.039255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.102.039255%26sid%3Dliteratum%253Aachs%26aulast%3DMahar%2BDoan%26aufirst%3DK.%2BM.%26aulast%3DHumphreys%26aufirst%3DJ.%2BE.%26aulast%3DWebster%26aufirst%3DL.%2BO.%26aulast%3DWring%26aufirst%3DS.%2BA.%26aulast%3DShampine%26aufirst%3DL.%2BJ.%26aulast%3DSerabjit-Singh%26aufirst%3DC.%2BJ.%26aulast%3DAdkison%26aufirst%3DK.%2BK.%26aulast%3DPolli%26aufirst%3DJ.%2BW.%26atitle%3DPassive%2520permeability%2520and%2520P-glycoprotein-mediated%2520efflux%2520differentiate%2520central%2520nervous%2520system%2520%2528CNS%2529%2520and%2520non-CNS%2520marketed%2520drugs%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2002%26volume%3D303%26spage%3D1029%26epage%3D1037%26doi%3D10.1124%2Fjpet.102.039255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Giuliano, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jairaj, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zafiu, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laufer, R.</span><span> </span><span class="NLM_article-title">Direct determination of unbound intrinsic drug clearance in the microsomal stability assay</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">1319</span><span class="NLM_x">–</span> <span class="NLM_lpage">1324</span><span class="refDoi"> DOI: 10.1124/dmd.105.005033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1124%2Fdmd.105.005033" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=1319-1324&author=C.+Giulianoauthor=M.+Jairajauthor=C.+M.+Zafiuauthor=R.+Laufer&title=Direct+determination+of+unbound+intrinsic+drug+clearance+in+the+microsomal+stability+assay&doi=10.1124%2Fdmd.105.005033"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1124%2Fdmd.105.005033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.105.005033%26sid%3Dliteratum%253Aachs%26aulast%3DGiuliano%26aufirst%3DC.%26aulast%3DJairaj%26aufirst%3DM.%26aulast%3DZafiu%26aufirst%3DC.%2BM.%26aulast%3DLaufer%26aufirst%3DR.%26atitle%3DDirect%2520determination%2520of%2520unbound%2520intrinsic%2520drug%2520clearance%2520in%2520the%2520microsomal%2520stability%2520assay%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2005%26volume%3D33%26spage%3D1319%26epage%3D1324%26doi%3D10.1124%2Fdmd.105.005033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Turner, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeo, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tucker, G. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rostami-Hodjegan, A.</span><span> </span><span class="NLM_article-title">Prediction of nonspecific hepatic microsomal binding from readily available physicochemical properties</span> <span class="citation_source-journal">Drug Metab. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">162</span><span class="NLM_x">–</span> <span class="NLM_lpage">162</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2006&pages=162-162&author=D.+B.+Turnerauthor=K.+R.+Yeoauthor=G.+T.+Tuckerauthor=A.+Rostami-Hodjegan&title=Prediction+of+nonspecific+hepatic+microsomal+binding+from+readily+available+physicochemical+properties"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTurner%26aufirst%3DD.%2BB.%26aulast%3DYeo%26aufirst%3DK.%2BR.%26aulast%3DTucker%26aufirst%3DG.%2BT.%26aulast%3DRostami-Hodjegan%26aufirst%3DA.%26atitle%3DPrediction%2520of%2520nonspecific%2520hepatic%2520microsomal%2520binding%2520from%2520readily%2520available%2520physicochemical%2520properties%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D2006%26volume%3D38%26spage%3D162%26epage%3D162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Yang, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jamei, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeo, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rostami-Hodjegan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tucker, G. T.</span><span> </span><span class="NLM_article-title">Misuse of the well-stirred model of hepatic drug clearance</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">501</span><span class="NLM_x">–</span> <span class="NLM_lpage">502</span><span class="refDoi"> DOI: 10.1124/dmd.106.013359</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1124%2Fdmd.106.013359" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=501-502&author=J.+S.+Yangauthor=M.+Jameiauthor=K.+R.+Yeoauthor=A.+Rostami-Hodjeganauthor=G.+T.+Tucker&title=Misuse+of+the+well-stirred+model+of+hepatic+drug+clearance&doi=10.1124%2Fdmd.106.013359"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1124%2Fdmd.106.013359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.106.013359%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%2BS.%26aulast%3DJamei%26aufirst%3DM.%26aulast%3DYeo%26aufirst%3DK.%2BR.%26aulast%3DRostami-Hodjegan%26aufirst%3DA.%26aulast%3DTucker%26aufirst%3DG.%2BT.%26atitle%3DMisuse%2520of%2520the%2520well-stirred%2520model%2520of%2520hepatic%2520drug%2520clearance%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2007%26volume%3D35%26spage%3D501%26epage%3D502%26doi%3D10.1124%2Fdmd.106.013359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Eisen, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spellman, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botstein, D.</span><span> </span><span class="NLM_article-title">Cluster analysis and display of genome-wide expression patterns</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">14863</span><span class="NLM_x">–</span> <span class="NLM_lpage">14868</span><span class="refDoi"> DOI: 10.1073/pnas.95.25.14863</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1073%2Fpnas.95.25.14863" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=9843981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADyaK1cXotVGmurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=14863-14868&author=M.+B.+Eisenauthor=P.+T.+Spellmanauthor=P.+O.+Brownauthor=D.+Botstein&title=Cluster+analysis+and+display+of+genome-wide+expression+patterns&doi=10.1073%2Fpnas.95.25.14863"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Cluster analysis and display of genome-wide expression patterns</span></div><div class="casAuthors">Eisen, Michael B.; Spellman, Paul T.; Brown, Patrick O.; Botstein, David</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">14863-14868</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A system of cluster anal. for genome-wide expression data from DNA microarray hybridization is described that uses std. statistical algorithms to arrange genes according to similarity in pattern of gene expression.  The output is displayed graphically, conveying the clustering and the underlying expression data simultaneously in a form intuitive for biologists.  We have found in the budding yeast Saccharomyces cerevisiae that clustering gene expression data groups together efficiently genes of known similar function, and we find a similar tendency in human data.  Thus patterns seen in genome-wide expression expts. can be interpreted as indications of the status of cellular processes.  Also, coexpression of genes of known function with poorly characterized or novel genes may provide a simple means of gaining leads to the functions of many genes for which information is not available currently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMnoYB4klPc7Vg90H21EOLACvtfcHk0ljhOq4En8HjDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXotVGmurk%253D&md5=6ef0b72943b7e2e928966e665c3a9628</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1073%2Fpnas.95.25.14863&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.95.25.14863%26sid%3Dliteratum%253Aachs%26aulast%3DEisen%26aufirst%3DM.%2BB.%26aulast%3DSpellman%26aufirst%3DP.%2BT.%26aulast%3DBrown%26aufirst%3DP.%2BO.%26aulast%3DBotstein%26aufirst%3DD.%26atitle%3DCluster%2520analysis%2520and%2520display%2520of%2520genome-wide%2520expression%2520patterns%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1998%26volume%3D95%26spage%3D14863%26epage%3D14868%26doi%3D10.1073%2Fpnas.95.25.14863" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Erlanson, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDowell, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, T.</span><span> </span><span class="NLM_article-title">Fragment-based drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">3463</span><span class="NLM_x">–</span> <span class="NLM_lpage">3482</span><span class="refDoi"> DOI: 10.1021/jm040031v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm040031v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFeltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=3463-3482&author=D.+A.+Erlansonauthor=R.+S.+McDowellauthor=T.+O%E2%80%99Brien&title=Fragment-based+drug+discovery&doi=10.1021%2Fjm040031v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; McDowell, Robert S.; O'Brien, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3463-3482</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The authors discuss steps in fragment-based drug design such as the development of the library of fragments, fragment screening methods, in situ fragment assembly.  The authors demonstrated that the fragment-based methods of drug discovery are widespread in industry and academia and that this methods have been successful in generation new drug leads with high potency and improved pharmacokinetic properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6YMqrpLJZY7Vg90H21EOLACvtfcHk0ljhOq4En8HjDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFeltLg%253D&md5=ad9c404ed14d03d889ceae44b4ef991a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm040031v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040031v%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DMcDowell%26aufirst%3DR.%2BS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26atitle%3DFragment-based%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D3463%26epage%3D3482%26doi%3D10.1021%2Fjm040031v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keseru, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leeson, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynolds, C. H.</span><span> </span><span class="NLM_article-title">The role of ligand efficiency metrics in drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">121</span><span class="refDoi"> DOI: 10.1038/nrd4163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1038%2Fnrd4163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=24481311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=105-121&author=A.+L.+Hopkinsauthor=G.+M.+Keseruauthor=P.+D.+Leesonauthor=D.+C.+Reesauthor=C.+H.+Reynolds&title=The+role+of+ligand+efficiency+metrics+in+drug+discovery&doi=10.1038%2Fnrd4163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ligand efficiency metrics in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.; Keserue, Gyoergy M.; Leeson, Paul D.; Rees, David C.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-121</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The judicious application of ligand or binding efficiency metrics, which quantify the mol. properties required to obtain binding affinity for a drug target, is gaining traction in the selection and optimization of fragments, hits and leads.  Retrospective anal. of recently marketed oral drugs shows that they frequently have highly optimized ligand efficiency values for their targets.  Optimizing ligand efficiency metrics based on both mol. mass and lipophilicity, when set in the context of the specific target, has the potential to ameliorate the inflation of these properties that has been obsd. in current medicinal chem. practice, and to increase the quality of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq96uhvE1OZ5LVg90H21EOLACvtfcHk0ljhOq4En8HjDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D&md5=344030e59f499180f79d311302a27532</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrd4163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4163%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DThe%2520role%2520of%2520ligand%2520efficiency%2520metrics%2520in%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D105%26epage%3D121%26doi%3D10.1038%2Fnrd4163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Sanguinetti, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tristani-Firouzi, M.</span><span> </span><span class="NLM_article-title">hERG potassium channels and cardiac arrhythmia</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">440</span><span class="NLM_x">, </span> <span class="NLM_fpage">463</span><span class="NLM_x">–</span> <span class="NLM_lpage">469</span><span class="refDoi"> DOI: 10.1038/nature04710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=10.1038%2Fnature04710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=16554806" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;key=1%3ACAS%3A528%3ADC%252BD28Xis1OlsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=440&publication_year=2006&pages=463-469&author=M.+C.+Sanguinettiauthor=M.+Tristani-Firouzi&title=hERG+potassium+channels+and+cardiac+arrhythmia&doi=10.1038%2Fnature04710"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">hERG potassium channels and cardiac arrhythmia</span></div><div class="casAuthors">Sanguinetti, Michael C.; Tristani-Firouzi, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">440</span>
        (<span class="NLM_cas:issue">7083</span>),
    <span class="NLM_cas:pages">463-469</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  HERG potassium channels are essential for normal elec. activity in the heart.  Inherited mutations in the HERG gene cause long QT syndrome, a disorder that predisposes individuals to life-threatening arrhythmias.  Arrhythmia can also be induced by a blockage of hERG channels by a surprisingly diverse group of drugs.  This side effect is a common reason for drug failure in preclin. safety trials.  Insights gained from the crystal structures of other potassium channels have helped our understanding of the block of hERG channels and the mechanisms of gating.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopS3WBEpuAUrVg90H21EOLACvtfcHk0linOnq9c9xDtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xis1OlsLs%253D&md5=3f355dcd51da3488d73c7c91cbe8d3b8</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnature04710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04710%26sid%3Dliteratum%253Aachs%26aulast%3DSanguinetti%26aufirst%3DM.%2BC.%26aulast%3DTristani-Firouzi%26aufirst%3DM.%26atitle%3DhERG%2520potassium%2520channels%2520and%2520cardiac%2520arrhythmia%26jtitle%3DNature%26date%3D2006%26volume%3D440%26spage%3D463%26epage%3D469%26doi%3D10.1038%2Fnature04710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b01577&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-16%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b01577%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b01577" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                1MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679952aecc7a3cb2","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
