
<html lang="en"     class="pb-page"  data-request-id="a19148e8-1fdd-4a56-aa7f-309e84349f8a"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2018.61.issue-23;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.8b01407;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Cell-Active Small Molecule Inhibitors of the DNA-Damage Repair Enzyme Poly(ADP-ribose) Glycohydrolase (PARG): Discovery and Optimization of Orally Bioavailable Quinazolinedione Sulfonamides" /></meta><meta name="dc.Creator" content="Bohdan  Waszkowycz" /></meta><meta name="dc.Creator" content="Kate M.  Smith" /></meta><meta name="dc.Creator" content="Alison E.  McGonagle" /></meta><meta name="dc.Creator" content="Allan M.  Jordan" /></meta><meta name="dc.Creator" content="Ben  Acton" /></meta><meta name="dc.Creator" content="Emma E.  Fairweather" /></meta><meta name="dc.Creator" content="Louise A.  Griffiths" /></meta><meta name="dc.Creator" content="Niall M.  Hamilton" /></meta><meta name="dc.Creator" content="Nicola S.  Hamilton" /></meta><meta name="dc.Creator" content="James R.  Hitchin" /></meta><meta name="dc.Creator" content="Colin P.  Hutton" /></meta><meta name="dc.Creator" content="Dominic I.  James" /></meta><meta name="dc.Creator" content="Clifford D.  Jones" /></meta><meta name="dc.Creator" content="Stuart  Jones" /></meta><meta name="dc.Creator" content="Daniel P.  Mould" /></meta><meta name="dc.Creator" content="Helen F.  Small" /></meta><meta name="dc.Creator" content="Alexandra I. J.  Stowell" /></meta><meta name="dc.Creator" content="Julie A.  Tucker" /></meta><meta name="dc.Creator" content="Ian D.  Waddell" /></meta><meta name="dc.Creator" content="Donald J.  Ogilvie" /></meta><meta name="dc.Description" content="DNA damage repair enzymes are promising targets in the development of new therapeutic agents for a wide range of cancers and potentially other diseases. The enzyme poly(ADP-ribose) glycohydrolase (..." /></meta><meta name="Description" content="DNA damage repair enzymes are promising targets in the development of new therapeutic agents for a wide range of cancers and potentially other diseases. The enzyme poly(ADP-ribose) glycohydrolase (..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 7, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01407" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01407" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01407" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01407" /></link>
        
    
    

<title>Cell-Active Small Molecule Inhibitors of the DNA-Damage Repair Enzyme Poly(ADP-ribose) Glycohydrolase (PARG): Discovery and Optimization of Orally Bioavailable Quinazolinedione Sulfonamides | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01407" /></meta><meta property="og:title" content="Cell-Active Small Molecule Inhibitors of the DNA-Damage Repair Enzyme Poly(ADP-ribose) Glycohydrolase (PARG): Discovery and Optimization of Orally Bioavailable Quinazolinedione Sulfonamides" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0030.jpeg" /></meta><meta property="og:description" content="DNA damage repair enzymes are promising targets in the development of new therapeutic agents for a wide range of cancers and potentially other diseases. The enzyme poly(ADP-ribose) glycohydrolase (PARG) plays a pivotal role in the regulation of DNA repair mechanisms; however, the lack of potent drug-like inhibitors for use in cellular and in vivo models has limited the investigation of its potential as a novel therapeutic target. Using the crystal structure of human PARG in complex with the weakly active and cytotoxic anthraquinone 8a, novel quinazolinedione sulfonamides PARG inhibitors have been identified by means of structure-based virtual screening and library design. 1-Oxetan-3-ylmethyl derivatives 33d and 35d were selected for preliminary investigations in vivo. X-ray crystal structures help rationalize the observed structure–activity relationships of these novel inhibitors." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01407"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01407">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01407&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01407&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01407&amp;href=/doi/10.1021/acs.jmedchem.8b01407" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 23</span><span class="cit-fg-pageRange">, 10767-10792</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/23" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01398" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b01428" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Cell-Active Small Molecule Inhibitors of the DNA-Damage Repair Enzyme Poly(ADP-ribose) Glycohydrolase (PARG): Discovery and Optimization of Orally Bioavailable Quinazolinedione Sulfonamides</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Bohdan Waszkowycz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bohdan Waszkowycz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Manchester Institute, The University of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bohdan++Waszkowycz">Bohdan Waszkowycz</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8572-2018" title="Orcid link">http://orcid.org/0000-0001-8572-2018</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kate M. Smith</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kate M. Smith</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Manchester Institute, The University of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kate+M.++Smith">Kate M. Smith</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8940-2804" title="Orcid link">http://orcid.org/0000-0001-8940-2804</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alison E. McGonagle</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alison E. McGonagle</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Manchester Institute, The University of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alison+E.++McGonagle">Alison E. McGonagle</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Allan M. Jordan</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Allan M. Jordan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Manchester Institute, The University of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +44 (0)161-306-6280. E-mail: <a href="/cdn-cgi/l/email-protection#1978757578773773766b7d7877597a6b6c72377478777a717c6a6d7c6b37787a376c72"><span class="__cf_email__" data-cfemail="67060b0b0609490d0815030609270415120c490a0609040f021413021549060449120c">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Allan+M.++Jordan">Allan M. Jordan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3449-3993" title="Orcid link">http://orcid.org/0000-0003-3449-3993</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ben Acton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ben Acton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Manchester Institute, The University of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ben++Acton">Ben Acton</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Emma E. Fairweather</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Emma E. Fairweather</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Manchester Institute, The University of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Emma+E.++Fairweather">Emma E. Fairweather</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Louise A. Griffiths</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Louise A. Griffiths</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Manchester Institute, The University of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Louise+A.++Griffiths">Louise A. Griffiths</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Niall M. Hamilton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Niall M. Hamilton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Manchester Institute, The University of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Niall+M.++Hamilton">Niall M. Hamilton</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicola S. Hamilton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicola S. Hamilton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Manchester Institute, The University of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicola+S.++Hamilton">Nicola S. Hamilton</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James R. Hitchin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James R. Hitchin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Manchester Institute, The University of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+R.++Hitchin">James R. Hitchin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Colin P. Hutton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Colin P. Hutton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Manchester Institute, The University of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Colin+P.++Hutton">Colin P. Hutton</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dominic I. James</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dominic I. James</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Manchester Institute, The University of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dominic+I.++James">Dominic I. James</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Clifford D. Jones</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Clifford D. Jones</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology Innovative Medicines Unit, AstraZeneca, Alderley Park, Macclesfield Cheshire SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Clifford+D.++Jones">Clifford D. Jones</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stuart Jones</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stuart Jones</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Manchester Institute, The University of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stuart++Jones">Stuart Jones</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Daniel P. Mould</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniel P. Mould</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Manchester Institute, The University of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniel+P.++Mould">Daniel P. Mould</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6501-4938" title="Orcid link">http://orcid.org/0000-0001-6501-4938</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Helen F. Small</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Helen F. Small</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Manchester Institute, The University of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Helen+F.++Small">Helen F. Small</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alexandra I. J. Stowell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alexandra I. J. Stowell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Manchester Institute, The University of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alexandra+I.+J.++Stowell">Alexandra I. J. Stowell</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Julie A. Tucker</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Julie A. Tucker</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structure and Biophysics, Discovery Sciences, AstraZeneca, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Julie+A.++Tucker">Julie A. Tucker</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ian D. Waddell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ian D. Waddell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Manchester Institute, The University of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ian+D.++Waddell">Ian D. Waddell</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Donald J. Ogilvie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Donald J. Ogilvie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Manchester Institute, The University of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Donald+J.++Ogilvie">Donald J. Ogilvie</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01407&amp;href=/doi/10.1021%2Facs.jmedchem.8b01407" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 23</span><span class="cit-pageRange">, 10767–10792</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 7, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>11 September 2018</li><li><span class="item_label"><b>Published</b> online</span>7 November 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 December 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01407" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01407</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10767%26pageCount%3D26%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DBohdan%2BWaszkowycz%252C%2BKate%2BM.%2BSmith%252C%2BAlison%2BE.%2BMcGonagle%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D23%26contentID%3Dacs.jmedchem.8b01407%26title%3DCell-Active%2BSmall%2BMolecule%2BInhibitors%2Bof%2Bthe%2BDNA-Damage%2BRepair%2BEnzyme%2BPoly%2528ADP-ribose%2529%2BGlycohydrolase%2B%2528PARG%2529%253A%2BDiscovery%2Band%2BOptimization%2Bof%2BOrally%2BBioavailable%2BQuinazolinedione%2BSulfonamides%26numPages%3D26%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10792%26publicationDate%3DDecember%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01407"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2888</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">13</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01407" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Cell-Active Small Molecule Inhibitors of the DNA-Damage Repair Enzyme Poly(ADP-ribose) Glycohydrolase (PARG): Discovery and Optimization of Orally Bioavailable Quinazolinedione Sulfonamides&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Bohdan&quot;,&quot;last_name&quot;:&quot;Waszkowycz&quot;},{&quot;first_name&quot;:&quot;Kate&quot;,&quot;last_name&quot;:&quot;M. Smith&quot;},{&quot;first_name&quot;:&quot;Alison&quot;,&quot;last_name&quot;:&quot;E. McGonagle&quot;},{&quot;first_name&quot;:&quot;Allan&quot;,&quot;last_name&quot;:&quot;M. Jordan&quot;},{&quot;first_name&quot;:&quot;Ben&quot;,&quot;last_name&quot;:&quot;Acton&quot;},{&quot;first_name&quot;:&quot;Emma&quot;,&quot;last_name&quot;:&quot;E. Fairweather&quot;},{&quot;first_name&quot;:&quot;Louise&quot;,&quot;last_name&quot;:&quot;A. Griffiths&quot;},{&quot;first_name&quot;:&quot;Niall&quot;,&quot;last_name&quot;:&quot;M. Hamilton&quot;},{&quot;first_name&quot;:&quot;Nicola&quot;,&quot;last_name&quot;:&quot;S. Hamilton&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;R. Hitchin&quot;},{&quot;first_name&quot;:&quot;Colin&quot;,&quot;last_name&quot;:&quot;P. Hutton&quot;},{&quot;first_name&quot;:&quot;Dominic&quot;,&quot;last_name&quot;:&quot;I. James&quot;},{&quot;first_name&quot;:&quot;Clifford&quot;,&quot;last_name&quot;:&quot;D. Jones&quot;},{&quot;first_name&quot;:&quot;Stuart&quot;,&quot;last_name&quot;:&quot;Jones&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;P. Mould&quot;},{&quot;first_name&quot;:&quot;Helen&quot;,&quot;last_name&quot;:&quot;F. Small&quot;},{&quot;first_name&quot;:&quot;Alexandra&quot;,&quot;last_name&quot;:&quot;I. J. Stowell&quot;},{&quot;first_name&quot;:&quot;Julie&quot;,&quot;last_name&quot;:&quot;A. Tucker&quot;},{&quot;first_name&quot;:&quot;Ian&quot;,&quot;last_name&quot;:&quot;D. Waddell&quot;},{&quot;first_name&quot;:&quot;Donald&quot;,&quot;last_name&quot;:&quot;J. Ogilvie&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;07&quot;,&quot;issue&quot;:&quot;23&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;10767-10792&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01407&quot;},&quot;abstract&quot;:&quot;DNA damage repair enzymes are promising targets in the development of new therapeutic agents for a wide range of cancers and potentially other diseases. The enzyme poly(ADP-ribose) glycohydrolase (PARG) plays a pivotal role in the regulation of DNA repair mechanisms; however, the lack of potent drug-like inhibitors for use in cellular and in vivo models has limited the investigation of its potential as a novel therapeutic target. Using the crystal structure of human PARG in complex with the weakly active and cytotoxic anthraquinone 8a, novel quinazolinedione sulfonamides PARG inhibitors have been identified by means of structure-based virtual screening and library design. 1-Oxetan-3-ylmethyl derivatives 33d and 35d were selected for preliminary investigations in vivo. X-ray crystal structures help rationalize the observed structure–activity relationships of these novel inhibitors.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01407&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01407" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01407&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01407" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01407&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01407" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01407&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01407&amp;href=/doi/10.1021/acs.jmedchem.8b01407" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01407" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01407" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (9 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01407%26sid%3Dliteratum%253Aachs%26pmid%3D30403352%26genre%3Darticle%26aulast%3DWaszkowycz%26date%3D2018%26atitle%3DCell-Active%2BSmall%2BMolecule%2BInhibitors%2Bof%2Bthe%2BDNA-Damage%2BRepair%2BEnzyme%2BPoly%2528ADP-ribose%2529%2BGlycohydrolase%2B%2528PARG%2529%253A%2BDiscovery%2Band%2BOptimization%2Bof%2BOrally%2BBioavailable%2BQuinazolinedione%2BSulfonamides%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D23%26spage%3D10767%26epage%3D10792%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=290676" title="Substituents">Substituents</a>,</li><li><a href="/action/doSearch?ConceptID=291763" title="Noncovalent interactions">Noncovalent interactions</a>,</li><li><a href="/action/doSearch?ConceptID=290716" title="Sulfones">Sulfones</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/23" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/jmcmar.2018.61.issue-23/20181213/jmcmar.2018.61.issue-23.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0030.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0030.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">DNA damage repair enzymes are promising targets in the development of new therapeutic agents for a wide range of cancers and potentially other diseases. The enzyme poly(ADP-ribose) glycohydrolase (PARG) plays a pivotal role in the regulation of DNA repair mechanisms; however, the lack of potent drug-like inhibitors for use in cellular and in vivo models has limited the investigation of its potential as a novel therapeutic target. Using the crystal structure of human PARG in complex with the weakly active and cytotoxic anthraquinone <b>8a</b>, novel quinazolinedione sulfonamides PARG inhibitors have been identified by means of structure-based virtual screening and library design. 1-Oxetan-3-ylmethyl derivatives <b>33d</b> and <b>35d</b> were selected for preliminary investigations in vivo. X-ray crystal structures help rationalize the observed structure–activity relationships of these novel inhibitors.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20068" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20068" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Exploiting the DNA damage response for cancer therapy relies on the fact that rapidly proliferating cancer cells have higher incidences of DNA damage, defective DNA repair pathways, genomic instability, or a combination of these facets.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> Poly(ADP-ribosylation) is a post-translational modification that plays an important role in the repair of damaged sections of DNA.<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a> Poly(ADP-ribose) polymerases (PARPs), in particular PARP1, signal the presence of DNA damage and facilitate DNA repair (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Upon DNA damage, PARP1 binds to single-strand break (SSB) sites and autoribosylates using NAD+ as a substrate to form poly(ADP-ribose) (PAR) chains. These PAR chains serve to recruit DNA repair proteins, such as XRCC1, to the site of DNA damage. Poly(ADP-ribose) glycohydrolase (PARG) systematically degrades the ADP-ribose polymers on the PARP enzyme, which is essential for DNA repair to occur; however, the precise order of events in this catalytic cycle is still unknown. <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> shows how inhibiting PARG leads to the persistence of PAR chains, with predicted consequences on NAD recycling, PARP recycling, and SSB repair. Although PARP inhibitors have received much attention as cancer therapies,<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> with olaparib approved by the FDA,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> the dearth of selective, cell-permeable small molecule PARG inhibitors has hampered assessment of the therapeutic potential of targeting PARG in human cancer.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> Additionally, while there are 17 known PARP family members, of which only a subset are inhibited by the PARP inhibitors, no close homologues of PARG exist.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> This provides an attractive target for drug discovery, given the cell’s apparent reliance on this single enzyme in nuclear DNA damage repair.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Consequences of PARG inhibition in DNA SSB repair.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Crystal structures of the bacterial, protozoal, and mammalian PARG catalytic domains have served to elucidate the structural basis of PAR substrate recognition and catalytic mechanism.<a onclick="showRef(event, 'ref13 ref14 ref15 ref16 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15 ref16 ref17 ref18 ref19">(13−19)</a> The C-terminal catalytic domain of PARG comprises a macrodomain, an evolutionarily conserved subunit that binds ADP-ribose and is involved in the regulation of PAR turnover and signaling,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> with a PARG-specific catalytic loop. PARG demonstrates a preference for hydrolyzing the long, unbranched chains of PAR, which interact with an extended substrate binding site that is mostly solvent-exposed and polar in character.</div><div class="NLM_p">To date, identification of potent and selective PARG inhibitors has been hampered by the lack of a suitable HTS-compatible assay system, prompting our recent development of a robust homogeneous time-resolved fluorescence (HTRF) based assay format.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Inhibitors reported in the literature include: ADP-ribose mimetics (e.g., ADP-HPD, <b>1</b>),<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> tannins (e.g., monogalloyl glucose, <b>2</b>),<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> salicylanilides (<b>3</b>),<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> rhodanine-based compounds (<b>4</b>),<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and phenolic hydrazide hydrazones (<b>5</b>)<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). These compounds are typically weakly potent, nonspecific inhibitors with poor cell permeability, only demonstrating activity in cell lysates.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> For example, compounds <b>3</b> and <b>4</b> did not demonstrate significant PARG inhibitory activity in our biochemical assay (activities >70 μM in our hands). Although compound <b>4</b> contains the known pan-assay interference (PAINs)<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> motif rhodanine, it has been successfully crystallized in complex with protozoal PARG, demonstrating binding to the ADP-ribose site.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Reported PARG inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Given the lack of drug-like, cell permeable inhibitors reported to date, there remains a need for selective tool compounds with improved physicochemical properties to validate the therapeutic potential of PARG inhibition.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> To address this need, we recently reported the first cell-active chemical probes against this enzyme.<a onclick="showRef(event, 'ref12 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref12 ref28 ref29">(12,28,29)</a> Herein, we describe the early discovery of several novel lead-like chemotypes arising from a program of virtual screening and structure-guided library design, inspired by the crystal structure of a weakly active HTS hit bound to human PARG. We further report our efforts to optimize one of these chemotypes in order to afford more potent in vitro tool compounds and derivatives with physicochemical properties suitable for preliminary in vivo pharmacokinetic (PK) assessment.</div></div><div id="sec1_1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15873" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15873" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">For preliminary hit expansion, compounds were generally prepared using traditional chlorosulfonation techniques to prepare the sulfonyl chloride, which was then used for sulfonamide formation. Anthraquinones <b>8a</b>–<b>o</b> were prepared in this manner as detailed in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, as was compounds <b>16</b> described in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, with purification by preparative HPLC leading to variable recoveries of material. Symmetrically substituted quinazolinediones <b>12a</b>–<b>c</b>, <b>13a</b>,<b>b</b> were prepared by alkylation of the quinazolinedione, followed by chlorosulfonation and sulfonamide formation (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a><named-content content-type="anchor" rid="sch3" type="simple"></named-content>). For the unsymmetrically substituted quinazolinediones <b>22a</b>–<b>h</b>, a route was developed which installed the acetyl protected alkyl sulfonamide and allowed diversification in the final step, as detailed in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Analogues <b>8a</b>–<b>o</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents: (a) SOCl<sub>2</sub>, DMF, reflux, 53%; (b) RNH<sub>2</sub>, Et<sub>3</sub>N, DCM, 15%–quant.</p></p></figure><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>12a</b>–<b>c</b>, <b>13a</b>,<b>b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents: (a) RI, K<sub>2</sub>CO<sub>3</sub>, DMF, 30–41%; (b) ClSO<sub>3</sub>H, 60 °C, 64–67%; (c) R′NH<sub>2</sub>, Et<sub>3</sub>N, DCM, 18–76%.</p></p></figure><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>16</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents: (a) ClSO<sub>3</sub>H, 60 °C, 70%; (b) RNH<sub>2</sub>, <i>i</i>-Pr<sub>2</sub>NEt, THF, DCM, 43%.</p></p></figure><figure id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Analogues <b>22a</b>–<b>t</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents: (a) CDI, DMF, 140 °C, 90%; (b) ClSO<sub>3</sub>H, 60 °C, 99%; (c) 1-methylcyclopropanamine hydrochloride, Et<sub>3</sub>N, DCM, 89%; (d) Ac<sub>2</sub>O, pyridine, DMAP, 83%; (e) BrCH<sub>2</sub>CN, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 75%; (f) R-X (where X = I or Br), K<sub>2</sub>CO<sub>3</sub>, DMF, then NH<sub>3</sub>, THF, 28–88%; (g) R-X (where X = Cl, Br or I), K<sub>2</sub>CO<sub>3</sub>, DMF, then K<sub>2</sub>CO<sub>3</sub>, MeOH 20–32%; (h) R-OH, DIAD, PS-PPh<sub>3</sub>, DMF, then K<sub>2</sub>CO<sub>3</sub>, MeOH, 93%.</p></p></figure><div class="NLM_p"><a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a> describes the synthesis of <b>27a</b>–<b>au</b>, with elaboration at the N3 vector. The quinazolinedione core was formed from <i>N</i>-methylanthranilic acid and sodium cyanate. Subsequent chlorosulfonylation and sulfonamide formation afforded key intermediate <b>26</b>, which was alkylated to give 1-methyl-3-substituted quinazolinediones <b>27a</b>–<b>au</b>. <a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a> shows the synthesis of quinazolinediones with elaboration at both N1 and N3 vectors. Chlorosulfonylation of isatoic anhydride afforded <b>30</b>. Careful temperature control in the subsequent sulfonamide formation was vital to minimize anhydride ring-opening with amine <b>28</b>. Once the sulfonamide formation was complete at a low temperature, the appropriate primary amine was added to the reaction mixture and warmed to room temperature (RT) to induce anhydride ring-opening and decarboxylation to give amide intermediates <b>31a</b>–<b>d</b>. Ring closure to afford quinazolinediones <b>32a</b>–<b>d</b> was carried out using triphosgene, and subsequent alkylation at N1 afforded compounds <b>33</b>–<b>36</b>. In some cases, Mitsunobu reaction using diisopropylazodicarboxylate (DIAD) resulted in an inseparable mixture of product and reduced DIAD byproduct. Fortunately, this problem was surmounted by using di(4-chlorobenzyl)azodicarboxylate (DCAD),<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> as the hydrazine byproduct in this case is easily removed by trituration. The approach to 3-(5-methyl-1,3,4-thiadiazol-2-yl)methyl quinazolinediones <b>41a</b> and <b>41e</b> was slightly different (<a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>). The 3-substituent bearing the thiadiazole was introduced prior to the chlorosulfonylation reaction as there are no free sites on the five-membered heteroaryl ring that can undergo chlorosulfonylation in this case.</div><figure id="sch5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0015.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of <b>27a</b>–<b>at</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaOCN, NaOH, AcOH, H<sub>2</sub>O, 50 °C, 1 h, 79%; (b) ClSO<sub>3</sub>H, 50 °C, 16 h, quant; (c) amine <b>28</b>, Et<sub>3</sub>N, DCM, RT, 3 h, 93%; (d) (i) NaH, DMF, RT, 5 min, (ii) R-X (X = Br, Cl, I, OTs), RT, 6–59%; (e) R-OH, DIAD (or DCAD), PS-PPh<sub>3</sub>, DMF, RT, 10–21%; (f) R-B(OH)<sub>2</sub>, Cu(OAc)<sub>2</sub>, Et<sub>3</sub>N, 4 Å molecular sieves, DCM, RT, 18%.</p></p></figure><figure id="sch6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0016.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of <b>33</b>–<b>36</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) ClSO<sub>3</sub>H, 50 °C, 4 h, 79%; (b) (i) amine <b>28</b>, Et<sub>3</sub>N, DCM, −10 °C, 30 min, (ii) R<sub>2</sub>-NH<sub>2</sub>, Et<sub>3</sub>N, −10 °C–RT, 18 h, 20–67%; (c) triphosgene, THF, 0 °C–RT, 18 h 78–97%; (d) (i) NaH, DMF, RT, 5 min, (ii) R<sub>1</sub>-X (X = Br, I, OMs, OTs), RT; (e) R<sub>1</sub>-OH, DIAD (or DCAD), PS-PPh<sub>3</sub>, DMF, RT, 16–75%.</p></p></figure><figure id="sch7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0017.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of <b>41a</b> and <b>41e</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) amine <b>42</b>, Et<sub>3</sub>N, DMF, 60 °C for 5 h, then RT for 16 h, 65%; (b) triphosgene, THF, 0 °C–RT, 18 h, 58%; (c) ClSO<sub>3</sub>H, 50 °C, 2 h, 69%; (d) amine <b>28</b>, Et<sub>3</sub>N, DCM, 0 °C–RT, 18 h, 32%; (e) EtI, K<sub>2</sub>CO<sub>3</sub>, DMF, μW, 80 °C, 20 min, 20% for <b>41a</b>, and 5-(chloromethyl)-1,3-dimethyl-1H-pyrazole, K<sub>2</sub>CO<sub>3</sub>, DMF, μW, 80 °C, 20 min, 44% for <b>41e</b>.</p></p></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14100" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14100" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Crystallographic Binding Mode and Initial Optimization of Anthraquinone <b>8a</b></h3><div class="NLM_p">The low druggability of PARG was highlighted by a high-throughput screen at AstraZeneca, comprising 1.4 million compounds, which resulted in only three confirmed hits of related chemical structure, exemplified by the anthraquinone derivative <b>8a</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). Despite this compound’s modest inhibitory potency (EC<sub>50</sub> = 81 μM) in an HTRF biochemical assay, we were encouraged by its approximately equipotent activity in an acute (1 h) cell-based proof-of-mechanism PAR chain persistence assay,<a onclick="showRef(event, 'ref21 ref31'); return false;" href="javascript:void(0);" class="ref ref21 ref31">(21,31)</a> where the compound was shown to block the PARG-mediated degradation of PAR chains in a dose-dependent manner. However, the limited solubility of the compound precluded an accurate estimation of EC<sub>50</sub>, and further progression of this anthraquinone derivative was additionally hampered by the observation of nonspecific 72 h cytotoxicity at cell-active doses.</div><figure id="fig3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Anthraquinone HTS hit <b>8a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Determination of the cocrystal structure of <b>8a</b> with the catalytic domain of human PARG at 1.9 Å resolution confirmed the direct interaction of the inhibitor with the PAR binding site and revealed some unexpected binding contacts that prompted our efforts to pursue structure-guided optimization of this series. As was previously reported, the substrate binding site in human PARG comprises an extended, relatively solvent-exposed channel.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> The anthraquinone <b>8a</b> is observed to bind in the region occupied by the adenine moiety of the truncated PAR analogue, ADP-ribose (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). However, the plane of the anthraquinone ring system is oriented approximately orthogonal to that of the adenine moiety of ADP-ribose, with the result that <b>8a</b> is highly solvent-exposed compared to the relatively well-buried adenine. The significant difference in the shape of the binding site between the two crystal structures can be attributed to Phe902 adopting a different rotamer when bound to <b>8a</b> compared with ADP-ribose (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). As a consequence, the anthraquinone core of <b>8a</b> is able to form extensive face-to-face aromatic stacking interactions with both Phe902 and Tyr795. Although this rotamer of Phe902 is incompatible with substrate binding, in the published <i>apo</i> structure of human PARG (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4A0D">4A0D</a>) both rotamers of Phe902 are observed, highlighting the flexibility of this side chain.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div><figure id="fig4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Comparison of the X-ray crystal structures of the catalytic domain of (A) human PARG bound to <b>8a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMM">6HMM</a>) and (B) ADP-ribose (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4B1H">4B1H</a>), shown in the same coordinate frame. Protein solvent-accessible surfaces are colored by electrostatic potential. (C) Detail of relative binding orientations of <b>8a</b> (cyan carbon atoms) and ADP-ribose (green) highlights movement of Phe902. (D) <b>8a</b> and ADP ribose bound to entire PARG domain, colored by atom type.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The binding mode of <b>8a</b> is anchored by a network of three hydrogen bonds from the sulfonamide moiety to the side chain of Glu727 and the backbones of Ile726 and Gln754, with the <i>tert</i>-butyl group deeply buried in a mainly hydrophobic cavity (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). We note that in the crystal structure of PARG in complex with ADP-ribose, the adenine moiety forms hydrogen bonds to the Glu727 side chain and Ile726 backbone <i>NH</i> (as well as, several water-mediated contacts); hence, this is a rare example of a sulfonamide moiety mimicking the hydrogen-bond network of adenine.</div><figure id="fig5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Detail of the hydrogen-bonding environment around <b>8a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMM">6HMM</a>). (B) Protein solvent-accessible surface highlights the exposure to solvent of the anthraquinone core and the buried pocket around the <i>tert</i>-butyl moiety. Hydrogen atoms modeled using Maestro (Schrödinger LLC). (C) Protein–ligand interaction diagram with hydrogen bonds indicated by pink arrows and aromatic interactions by green arrows.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Optimization of <b>8a</b> focused on a series of hydrophobic and polar replacements of the <i>tert</i>-butyl moiety to explore the limits of the constrained binding pocket observed around the <i>N</i>-alkyl sulfonamide headgroup (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). A preference was observed for compact branched or cyclic alkyl groups, notably for the simple cyclopropyl <b>8f</b>, and particularly the methylcyclopropyl and cyanocyclopropyl derivatives (<b>8h</b> and <b>8i</b>), which yielded single-digit micromolar biochemical activities. A series of unbranched polar-substituted groups (e.g., <b>8j</b>–<b>n</b>) were designed to probe a narrow solvent-filled channel extending from the <i>tert</i>-butyl pocket, but these all proved ineffective. <i>N</i>-Methylation of the sulfonamide, or replacement with alternative linkers such as amides and the reversed sulfonamide abolished activity (data not shown), emphasizing a strict requirement for the specific network of hydrogen-bonding contacts observed in the crystal structure. In summary, the steep SAR highlighted the limited opportunities for optimization of the <i>N</i>-alkyl sulfonamide moiety. Although an encouraging improvement in activity had been achieved for <b>8i</b> and <b>8h</b> compared with <b>8a</b>, cell toxicity remained an issue with this series and was presumed to be related to the anthraquinone ring system, a motif reminiscent of known DNA intercalators.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Initial Optimization of the Sulfonamide <i>N</i>-Alkyl Moiety of <b>8a</b><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0018.gif" alt="" id="GRAPHIC-d7e1113-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">PARG biochemical EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">PARG Cell EC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8a</b></td><td class="colsep0 rowsep0" align="left"><i>tert</i>-butyl</td><td class="colsep0 rowsep0" align="center">81 (21)</td><td class="colsep0 rowsep0" align="center">∼30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8b</b></td><td class="colsep0 rowsep0" align="left">methyl</td><td class="colsep0 rowsep0" align="center">>30<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8c</b></td><td class="colsep0 rowsep0" align="left">ethyl</td><td class="colsep0 rowsep0" align="center">110 (7.1)</td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8d</b></td><td class="colsep0 rowsep0" align="left">1,1-dimethylpropyl</td><td class="colsep0 rowsep0" align="center">49 (27)</td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8e</b></td><td class="colsep0 rowsep0" align="left">cyclopropylmethyl</td><td class="colsep0 rowsep0" align="center">24 (5.2)</td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8f</b></td><td class="colsep0 rowsep0" align="left">cyclopropyl</td><td class="colsep0 rowsep0" align="center">13 (5.3)</td><td class="colsep0 rowsep0" align="center">22 (6.8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8g</b></td><td class="colsep0 rowsep0" align="left">cyclobutyl</td><td class="colsep0 rowsep0" align="center">>50<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8h</b></td><td class="colsep0 rowsep0" align="left">1-methylcyclopropyl</td><td class="colsep0 rowsep0" align="center">6.0 (2.3)</td><td class="colsep0 rowsep0" align="center">6.6 (3.5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8i</b></td><td class="colsep0 rowsep0" align="left">1-cyanocyclopropyl</td><td class="colsep0 rowsep0" align="center">3.3 (2.3)</td><td class="colsep0 rowsep0" align="center">19 (5.2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8j</b></td><td class="colsep0 rowsep0" align="left">carbamoylmethyl</td><td class="colsep0 rowsep0" align="center">>150</td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8k</b></td><td class="colsep0 rowsep0" align="left">2-carbamoylethyl</td><td class="colsep0 rowsep0" align="center">>150</td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8l</b></td><td class="colsep0 rowsep0" align="left">2-hydroxyethyl</td><td class="colsep0 rowsep0" align="center">>150</td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8m</b></td><td class="colsep0 rowsep0" align="left">2-methoxyethyl</td><td class="colsep0 rowsep0" align="center">>150</td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8n</b></td><td class="colsep0 rowsep0" align="left">2-methoxy-1,1-dimethyl-ethyl</td><td class="colsep0 rowsep0" align="center">>150</td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8o</b></td><td class="colsep0 rowsep0" align="left">2-cyanoethyl</td><td class="colsep0 rowsep0" align="center">88 (18)</td><td class="colsep0 rowsep0" align="center">nd</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Biological data are stated as the geometric mean of at least three independent determinations unless otherwise stated, with standard deviations quoted in parentheses. nd = not determined.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Maximum concentration limited by interference and/or solubility issues.</p></div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Scaffold-Hopping by Virtual Screening and Library Design</h3><div class="NLM_p">In parallel with elaboration of the anthraquinone sulfonamide moiety, a structure-based virtual screening campaign was initiated to identify alternatives to the anthraquinone ring system that might obviate the observed cell toxicity and prove more amenable to optimization. As the hydrogen-bonding network observed around the sulfonamide appeared to be a key pharmacophore, we decided to retain this moiety while exploring alternative bi- and tricyclic ring systems that would maintain the observed stacking interactions with Tyr795 and Phe902, and introduce more drug-like physical properties.</div><div class="NLM_p">To this end, commercially available screening libraries were initially triaged to identify and select all aryl sulphonamides bearing a variety of aliphatic <i>N</i>-substituents. These selections were then filtered for suitable drug-like properties and any PAINS or undesirable structural motifs removed. These approaches yielded around 31000 putative hits which were then docked into the induced cavity observed in the X-ray structure of <b>8a</b> bound to human PARG (Glide SP docking, Schrodinger, LLC, New York). Manual inspection of the top five docking poses per ligand ensured maintenance of the key sulphonamide binding motif and credible binding modes and from this manual analysis, a set of 68 compounds was selected for purchase and assessment, based on an assessment of binding mode, site complementarity and docking score.</div><div class="NLM_p">The biochemical affinity of these initial compounds proved to be very weak, with the notable exception of the quinazolinedione <b>12e</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Replacing the sulfonamide <i>N</i>-cyclopropyl substituent of <b>12e</b> with <i>N</i>-ethyl (<b>12d</b>, <b>13c</b>) resulted in a large drop in activity, consistent with the SAR observed for the anthraquinone scaffold (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The only other <i>N</i>-cyclopropyl example in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, the tricyclic scaffold <b>43</b>, displayed modest activity but with significantly reduced ligand efficiency compared with <b>12e</b>. While some examples with the <i>N</i>-<i>tert</i>-butyl moiety, such as <b>44</b> and <b>45</b>, were of similar activity to the anthraquinone <b>8a</b>, other bicyclic systems proved to be inactive (<b>16</b>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Example Hits from the Initial Virtual Screen of Commercial Compounds<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0019.gif" alt="" id="gr18" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Biological data are stated as the geometric mean of at least three independent determinations, with standard deviations quoted in parentheses.</p></div></div><div></div></div><div class="NLM_p">To determine if the weak level of activity for these compounds was a consequence of suboptimal <i>N</i>-alkyl substitution on the sulfonamide, analogues of selected cores were synthesized with the preferred <i>N</i>-alkyl substituents identified from the optimization of <b>8a</b>, e.g., methylcyclopropyl and cyanocyclopropyl (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> and <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). This modification resulted in a significant gain in biochemical activity for the quinazolinediones <b>12b</b>, <b>12c</b>, <b>13a</b>, and <b>13b</b>, which demonstrated a greater than 20-fold improvement compared with the <i>tert</i>-butyl analogue <b>12a</b> and were accompanied by an increase in biochemical lipophilic ligand efficiency (LLE)<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> from a value of 2.1 for <b>12a</b> to 3.9 and 4.7 for <b>12b</b> and <b>12c</b>, respectively. Notably, the quinazolinediones also demonstrated a similar level of potency in the cell assay, with the best examples achieving low micromolar potency. However, <i>N</i>-methylcyclopropyl derivatives of other ring systems such as the naphthalene <b>16</b> did not achieve levels of activity to justify further consideration. From docking studies, the quinazolinedione scaffold hop was predicted to achieve very similar binding modes to the anthraquinone <b>8a</b>, while offering different vectors for enhancing binding affinity through additional protein interactions (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Optimization of Sulfonamide <i>N</i>-Alkyl Groups for Hits from the First Round of Virtual Screening<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0020.gif" alt="" id="gr19" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Biological data are stated as the geometric mean of at least three independent determinations, with standard deviations quoted in parentheses. nd = not determined.</p></div></div><div></div></div><figure id="fig6" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Predicted binding modes of quinazolinedione <b>12b</b> (docked into PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMM">6HMM</a>). The N1 position presents a vector toward solvent and the shelf adjacent to Phe902, while the N3 position presents a vector toward the pocket occupied by the ribose moiety of ADP.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The requirement for specific <i>N</i>-alkyl sulfonamide substituents, such as <i>tert</i>-butyl and cyclopropyl, in order to achieve significant levels of potency suggested that potentially useful scaffold hops may have been missed in the original virtual screening campaign because of the limited commercial availability of appropriate sulfonamides. Therefore, a second round of virtual screening was initiated based on docking virtual libraries generated from the in silico enumeration of a diverse set of commercially available aryl sulfonyl chlorides with several of our preferred alkyl amines. A further set of scaffold hops was designed on the basis of 2D similarity to the active cores identified previously. Synthesis of selected examples from these approaches resulted in a chemically diverse set of additional scaffolds, several of which displayed promising levels of activity, although none proved to be superior in potency or lipophilic ligand efficiency to the quinazolinediones (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01407/suppl_file/jm8b01407_si_001.pdf" class="ext-link">Table S2</a>). Note that for synthetic expediency, no attempt was made to optimize fully the substituents around each distinct ring system; rather, the aim of the exercise was to perform a rapid exploration of synthetically tractable SAR to understand whether any particular ring systems were significantly favored or disfavored. In general, we observed that fully aromatic/delocalized bicyclic cores were preferred over partially saturated ring systems, consistent with the modeled binding modes in which the aromatic core is tightly stacked between Tyr795 and Phe902. Among aromatic systems, there appears to be a preference for one or more hydrogen bond acceptor groups such as a carbonyl oxygen (as in the quinazolinedione <b>12b</b>) or a nitrogen lone pair (as in the quinoline <b>S12</b>, Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01407/suppl_file/jm8b01407_si_001.pdf" class="ext-link">Table S2</a>). In contrast, incorporation of a nitrogen atom in the benzo ring bearing the sulfonamide resulted in between 3- and 5-fold loss of activity (e.g., compound <b>S1</b>, Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01407/suppl_file/jm8b01407_si_001.pdf" class="ext-link">Table S2</a>). We note that, in the X-ray crystal structure of PARG in complex with <b>8a</b>, a number of polar protein atoms are located in close proximity to the anthraquinone ring system, including the potential hydrogen bond donor Tyr795 <i>OH</i>, and it may be that a combination of hydrogen bonding (perhaps through water) and enhanced π-stacking interactions account for the preferences seen for certain ring systems over others.</div><div class="NLM_p">Overall, by utilizing this approach, we succeeded in moving from an unattractive HTS hit to several lead-like scaffolds displaying improved ligand efficiency and physicochemical properties. Comparison of the initial hit compound <b>8a</b> with <b>12b</b> reveals a reduction in molecular weight and log <i>P</i>, with associated increases in LE and LLE (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). In addition, the scaffold hop is devoid of the acute nonspecific cytotoxicity of the parent anthraquinone. On the basis of its promising potency, synthetic tractability, and favorable physicochemical profile, the quinazolinedione series was prioritized as an attractive starting-point for further optimization.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Comparison of Measured and Predicted Physicochemical Properties of Representative Compounds</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">MW</th><th class="colsep0 rowsep0" align="center">measured log <i>D</i><sub>7.4</sub></th><th class="colsep0 rowsep0" align="center" char=".">calculated log <i>P</i><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">turbidimetric solubility (μM)</th><th class="colsep0 rowsep0" align="center" char=".">LE<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">LLE<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">cytotoxicity (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8a</b></td><td class="colsep0 rowsep0" align="char" char=".">343.4</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="char" char=".">3.18</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">0.23</td><td class="colsep0 rowsep0" align="char" char=".">0.91</td><td class="colsep0 rowsep0" align="center">16 (2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12b</b></td><td class="colsep0 rowsep0" align="char" char=".">323.4</td><td class="colsep0 rowsep0" align="center">1.73</td><td class="colsep0 rowsep0" align="char" char=".">1.50</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">0.34</td><td class="colsep0 rowsep0" align="char" char=".">3.9</td><td class="colsep0 rowsep0" align="center">>30</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">XlogP calculated using Dotmatics Elemental package (Dotmatics, Bishops Stortford, UK).</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Ligand efficiency calculated from biochemical EC<sub>50</sub> (i.e., LE = 1.4(pEC<sub>50</sub>)/HAC, where HAC is the number of heavy atoms).</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">Reference <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a>. nd = not determined.</p></div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Optimization of Quinazolinedione N1 Position</h3><div class="NLM_p">Initial exploration of the SAR around the quinazolinedione ring system focused on substitution at the N1 position (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>), as this vector presented an opportunity for interaction with a solvent-exposed shelf adjacent to Phe902 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). Methylene-linked saturated and aromatic ring systems were of interest as a strategy to improve binding affinity, as modeling suggested these would readily access the Phe902 shelf, while more conformationally flexible groups were deemed likely to extend into solvent and thus would be more useful to modulate physicochemical properties.</div><div class="NLM_p">In general, substitution at N1 yielded relatively modest gains in biochemical and cellular EC<sub>50</sub> (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>), with the ethyl analogue <b>22a</b> similar in potency to the parent methyl <b>12b</b>. Incorporation of more polar groups, such as in <b>22b</b> and <b>22c</b>, resulted in low micromolar compounds with improved LLE. We were pleased to observe that simple lipophilic groups such as the prop-2-ynyl <b>22d</b> and cyclopropylmethyl <b>22f</b> yielded our first submicromolar compounds in both enzyme and cell assays. This level of cellular potency represents a significant advance in the identification of cell-active PARG tool compounds when compared with our original HTS hit and also other inhibitors described in the literature.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Evaluation of Quinazolinedione N1 Substituents<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0021.gif" alt="" id="GRAPHIC-d7e1711-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">biochemical EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">cell EC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">4.0 (2.0)</td><td class="colsep0 rowsep0" align="center">8.1 (9.2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12b</b></td><td class="colsep0 rowsep0" align="left">methyl</td><td class="colsep0 rowsep0" align="center">3.9 (1.7)</td><td class="colsep0 rowsep0" align="center">1.8 (1.1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22a</b></td><td class="colsep0 rowsep0" align="left">ethyl</td><td class="colsep0 rowsep0" align="center">1.3 (1.9)</td><td class="colsep0 rowsep0" align="center">3.6 (1.3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22b</b></td><td class="colsep0 rowsep0" align="left">2-methoxyethyl</td><td class="colsep0 rowsep0" align="center">1.1 (0.13)</td><td class="colsep0 rowsep0" align="center">6.1 (3.1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22c</b></td><td class="colsep0 rowsep0" align="left">2-(dimethylamino)ethyl</td><td class="colsep0 rowsep0" align="center">2.0 (0.77)</td><td class="colsep0 rowsep0" align="center">6.4 (5.3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22d</b></td><td class="colsep0 rowsep0" align="left">prop-2-ynyl</td><td class="colsep0 rowsep0" align="center">0.71 (0.17)</td><td class="colsep0 rowsep0" align="center">0.25 (0.088)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22e</b></td><td class="colsep0 rowsep0" align="left">cyanomethyl</td><td class="colsep0 rowsep0" align="center">2.8 (1.6)</td><td class="colsep0 rowsep0" align="center">8.9 (4.5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22f</b></td><td class="colsep0 rowsep0" align="left">cyclopropylmethyl</td><td class="colsep0 rowsep0" align="center">0.44 (0.18)</td><td class="colsep0 rowsep0" align="center">0.62 (0.62)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22g</b></td><td class="colsep0 rowsep0" align="left">cyclohexylmethyl</td><td class="colsep0 rowsep0" align="center">0.56 (0.28)</td><td class="colsep0 rowsep0" align="center">2.7 (1.6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22h</b></td><td class="colsep0 rowsep0" align="left">benzyl</td><td class="colsep0 rowsep0" align="center">1.3 (0.62)</td><td class="colsep0 rowsep0" align="center">2.3 (0.97)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Biological data are stated as the geometric mean of at least three independent determinations, with standard deviations quoted in parentheses.</p></div></div></div><div class="NLM_p">In comparison with <b>22f</b>, larger carbocyclic substituents (e.g., <b>22g</b> and <b>22h</b>) did not result in any further gain in potency. The predicted binding mode of the benzylic derivative <b>22h</b> was confirmed by soaking the compound into preformed crystals of human PARG, resulting in a 2.9 Å resolution X-ray crystal structure (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). The compound was observed to bind similarly to <b>8a</b>, with the quinazolinedione ring system well aligned with the anthraquinone of <b>8a</b> and forming aromatic stacking interactions with Tyr795 and Phe902. The sulfonamide moiety forms the same network of three hydrogen bonds as observed for <b>8a</b>, with the <i>N</i>-methylcyclopropyl group occupying a similar location to the <i>tert</i>-butyl of <b>8a</b>. As these groups are very similar in size, the significant improvement in activity observed for the methylcyclopropyl may be due to the greater δ character of the cyclopropyl ring;<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> we note close nonbonded contacts between the cyclopropyl ring and the side chains of Gln754 and Glu727, with a potential CH···O hydrogen bond to the backbone carbonyl of Gln754. The N1 benzyl group partly occupies the Phe902 shelf, stacking against the Phe902–Gly903 peptide bond. The interaction between the benzyl group and the Phe902 shelf appears suboptimal in terms of shape complementarity, which likely accounts for the modest potency of <b>22h</b> and suggested opportunities for further optimization. Pleasingly, the determined crystallographic structure of this derivative closely matched the binding mode predicted by modeling, facilitating the prioritization of latter derivatives (such as the improved analogue <b>22f</b>) for synthesis and precluding the requirement for extensive iterative crystallographic determination.</div><figure id="fig7" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. X-ray crystal structure of <b>22h</b> bound to the catalytic domain of human PARG at 2.9 Å resolution (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMN">6HMN</a>). (A) Interaction of the <i>N</i>1-benzyl moiety with the Phe902 shelf. (B) Detail of the hydrogen-bonding environment around the sulfonamide, with a putative CH···O interaction highlighted by the cyan dashed line. Hydrogen atoms modeled using Maestro (Schrödinger LLC). (C) Ligand interaction diagram colored as in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We determined to further investigate structure–activity relationships (SAR) at the N1 position of the quinazolinedione, as this vector presented a good opportunity for interaction with a solvent-exposed shelf adjacent to Phe902. It can be seen from the crystal structure of <b>22h</b> in complex with PARG that the N1 vector projects away from the PARG active site toward solvent (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). A library of more elaborate <i>N</i>1-quinazolinediones (with N3 fixed as methyl) was therefore synthesized to explore whether the interactions with this shelf could be optimized in order to boost affinity. Further N1 variations are also shown in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Evaluation of Quinazolinedione N1 Substituents<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0022.gif" alt="" id="GRAPHIC-d7e1951-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0023.gif" alt="" id="gr20" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">Biological data are stated as the geometric mean of at least three independent determinations, unless otherwise stated, with standard deviations quoted in parentheses.</p></div></div><div></div></div><div class="NLM_p last">The results in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a> show that substitution at the 4-position of the phenyl ring of the benzyl derivatives was usually favored over the 2- or 3-substituted analogues. Although the SAR was generally quite flat, only modest improvements in potency were observed compared to the parent <i>N</i>1-benzyl analogue <b>22h</b>. No improvement was observed with pyridylmethyl derivatives <b>22o</b>–<b>q</b>. However, significant improvements were obtained with methylene-linked 5-membered heteroaryl substituents. Pyrazole derivative <b>22r</b> showed low micromolar activity, and this was enhanced further by methylation at the 3-positon (<b>22s</b>). Bis-methylated thiazole derivative <b>22t</b> gave the most potent quinazolinedione in the series. Modeling suggested that these substituted 5-membered heterocycles achieved improved lipophilic contacts with the Phe902 shelf compared with benzyl derivative <b>22h</b>. Although some polar protein sites are accessible in this region, it was not apparent from the SAR that any productive hydrogen-bonding interactions were being achieved.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Exploring the N3 Vector</h3><div class="NLM_p">We then turned our attention to exploring the SAR at the N3 vector of the quinazolinedione. Although this vector also points primarily toward solvent, its proximity to the ribose binding pocket suggested that additional productive binding interactions may be achievable. A library with a range of substituents at N3 (with N1 fixed as methyl) was modeled into the crystal structure of PARG (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMN">6HMN</a>), and a selection of analogues was prepared in order to investigate what is tolerated at this position. A representative set of N3 variations is shown in <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Elaboration at the N3 Vector<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0024.gif" alt="" id="GRAPHIC-d7e2009-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0025.gif" alt="" id="gr21" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">All EC<sub>50</sub> data are means of at least <i>n</i> = 4 independent measurements, with standard deviations quoted in parentheses.</p></div></div><div></div></div><div class="NLM_p">We initially investigated lipophilic alkyl and aryl groups and found that these gave either no improvement over the 3-methyl analogue <b>12b</b> or were detrimental to activity, consistent with the generally polar nature of the protein surface in this region. However, introducing polarity to the substituent at the N3-position yielded promising compounds with activity in the low micromolar range (e.g., <b>27g</b>, <b>27i</b>, <b>27j</b>, and <b>27l</b>). Modeling studies suggested that the hydrogen bond acceptor atom common to these groups may form a hydrogen bond with Asn869. The difference in activity observed across the pyridyl isomers suggested that the position of this hydrogen bond acceptor was important, and this was further demonstrated by using a series of methylene-linked heterocyclic substituents at N3. Initial exploration with this motif resulted in moderate potencies. However, moving from thiazol-2-ylmethyl derivative <b>27r</b> (13 μM) to thiazol-5-yl isomer <b>27s</b> resulted in a 10-fold improvement in potency to 1.2 μM. Methylation of the thiazole (<b>27t</b>) gave a further boost to potency to yield a submicromolar inhibitor.</div><div class="NLM_p">Determination of the X-ray crystal structure of <b>27t</b> bound to the human PARG catalytic domain revealed that the quinazolinedione sulfonamide exhibited a similar binding mode to that observed for the <i>N</i>1-benzyl derivative <b>22h</b> (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). The N3 substituent of <b>27t</b> accesses a narrow solvent-exposed ledge above Phe902, potentially making a favorable edge-to-face stacking interaction with the Phe902 side chain. The crystal structure confirmed the hypothesis that the N atom in the thiazole ring can make a hydrogen bond interaction with the Asn869 side chain, as had been suggested by modeling. This explains the noticeable gain in potency for this specific thiazole isomer compared to other heterocycles lacking an appropriately positioned hydrogen bond acceptor (e.g., <b>27q</b>). The reduced affinity noted for furan <b>27n</b> and oxazole <b>27q</b> is in line with the weaker hydrogen bonding strength expected from an aromatic oxygen atom compared with a nitrogen lone pair. The 2-methyl group on the thiazole ring of <b>27t</b> makes an additional hydrophobic contact with the β-carbon of the Asp904 side chain. Overall, this data suggests that the complementarity in shape and properties of the 2-methylthiazole substituent enables productive binding to an apparently unpromising, narrow shelf on the protein surface. The relatively limited conformational flexibility around the N3 methylene, due to the steric restrictions imposed by the adjacent carbonyl oxygens of the quinazolinedione core, may also help to stabilize the observed binding mode.</div><figure id="fig8" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) Crystal structure of <b>27t</b> bound to human PARG at 2.06 Å resolution (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMK">6HMK</a>). Hydrogen atoms modeled using Maestro (Schrödinger LLC). (B) Protein–ligand interaction diagram with hydrogen bonds indicated by pink arrows and aromatic interactions by green arrows.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The data in <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a> indicated that the SAR at the N3 vector of the quinazolinedione was steeper than at the N1 vector. Changing the position of the nitrogen atom in the heteroaryl ring evidently had a significant effect on the PARG activity, and we were keen to explore the interaction of the nitrogen with Asn869 further. A focused set of quinazolinediones substituted at the N3 position with methylene-linked 5-membered N-containing heterocycles was therefore synthesized (<a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>).</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Exploring the Interaction with Asn869: Focused N3 Library<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0026.gif" alt="" id="GRAPHIC-d7e2111-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0027.gif" alt="" id="gr22" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup><sup>a</sup></sup><p class="last">All EC<sub>50</sub> data are means of at least <i>n</i> = 4 independent measurements, with standard deviations quoted in parentheses.</p></div></div><div></div></div><div class="NLM_p">The X-ray crystal structure of <b>27t</b> bound to PARG (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>) showed that the methyl group on the thiazole ring was in close contact with the β-carbon of the side chain of Asp904. The protein surface in this region is relatively polar due to the proximity of the carboxylate moiety of Asp904, as well as the backbone carbonyls of Thr901 and Phe902; hence, we hoped that the introduction of a more polar substituent in place of the methyl group would allow formation of an additional hydrogen bond. Disappointingly, however, aminothiazole <b>27u</b> showed no improvement in potency compared to <b>27t</b>. Methylation at the 4-position of the thiazole (<b>27v</b>) was tolerated with only a modest loss in PARG inhibitory activity, while the 4-phenyl analogue <b>27w</b> resulted in a significant reduction in activity, likely due to steric clashes with both the protein and the quinazolinedione core. Having prepared a broad range of methylene-linked heteroaryl substituents, we observed that the best examples (with activities <0.5 μM) contained a nitrogen atom at the position essential for maintenance of the key hydrogen bond interaction with Asn869. The importance of having the correct orientation of the nitrogen lone pair is demonstrated by the striking differences in PARG activity observed between regioisomer pairs <b>27ah</b> and <b>27ai</b> and <b>27an</b> and <b>27ao</b>. Although <i>des</i>-methyl analogue <b>27aj</b> showed a similar biochemical activity to <b>27ai</b>, it displayed a significant drop-off in cell activity (data not shown), probably due to low permeability. Overall, 3-pyrazolylmethyl and 3-isoxazolylmethyl substituents emerged as interesting, less lipophilic alternatives to the 3-thiazolylmethyl substituent of <b>27t</b>. Interestingly, introduction of a nitrogen lone pair <i>ortho</i> to the methylene linker is generally detrimental to PARG activity (e.g., <b>27y</b>, <b>27ad</b>, <b>27ae</b>, <b>27ah</b>, <b>27ao</b>, and <b>27ar</b>), likely as a result of conformational strain due to the unfavorable electrostatic interaction of the lone pair with the carbonyl oxygen atoms of the quinazolinedione core in the bound conformation.</div><div class="NLM_p"><a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a> summarizes the key SAR at the N3 vector of the quinazolinedione. A methylene-linked 5-membered heterocycle has been used as an example, however, the model is equally applicable to 3-substituents containing pyridyl (<b>27g</b>), ketone (<b>27i</b> and <b>27j</b>), and nitrile (<b>27l</b>) functional groups. As noted earlier, overall, the SAR is much steeper at the quinazolinedione N3 vector compared to the N1 vector, with specific pendant functional groups enabling a significant boost in PARG inhibition.</div><figure id="fig9" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Key SAR at the 3-position of the quinazolinedione.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Simultaneous Elaboration at Both the N1 and N3 Vectors</h3><div class="NLM_p">With a better understanding of the SAR at the N1 and N3 vectors in isolation, we next investigated the effect of combining the optimized N1 and N3 substituents in the same compound in a bid to further increase potency. A representative set of N1,N3-substituted quinazolinediones is shown in <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">Table <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">9</a></a>.</div><div class="NLM_table-wrap" id="tbl9"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 9. Simultaneous Elaboration at Both N1 and N3 Vectors<a class="ref internalNav" href="#t9fn1" aria-label="a">a</a>,<a class="ref internalNav" href="#t9fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0028.gif" alt="" id="GRAPHIC-d7e2236-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0029.gif" alt="" id="gr23" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t9fn1"><div class="footnote" id="t9fn1"><sup><sup>a</sup></sup><p class="last">All biochemical and cellular EC<sub>50</sub> data are means of at least <i>n</i> = 4 independent measurements, with standard deviations quoted in parentheses.</p></div><div class="footnote" id="t9fn2"><sup><sup>b</sup></sup><p class="last">Human microsomal clearance (mouse microsomal clearance is shown in brackets for selected compounds).</p></div><div class="footnote" id="t9fn3"><sup><sup>c</sup></sup><p class="last">In vitro tool compound <b>34e</b> (PDD00017273)<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> is available to purchase.</p></div><div class="footnote" id="t9fn4"><sup><sup>d</sup></sup><p class="last"><b>34f</b> has previously been reported as PDDX-04/PDD00017272.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></p></div></div><div></div></div><div class="NLM_p">Combining substitution at both N1 and N3 vectors led to a significant improvement in potency, and biochemical EC<sub>50</sub>s as low as 4.8 nM were achieved (compound <b>34f</b>). For 3-pyrazolylmethyl quinazolinediones (<b>33a</b>–<b>f</b>), up to a 100-fold increase in potency was achieved over the 1-methyl analogue (<b>27ai</b>). Pleasingly, in our cellular proof-of-mechanism PAR chain persistence assay,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> the inhibition of PARG in cells was in line with the biochemical EC<sub>50</sub> values.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></div><div class="NLM_p">The X-ray crystal structure of <b>33f</b> was solved in complex with the catalytic domain of human PARG (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>) and revealed the same binding configuration of the quinazolinedione core and sulfonamide as observed for <b>27t</b>. For <b>33f</b>, the N1 and N3 substituents adopt a “pincer” configuration around the Phe902 side chain. The pyrazole nitrogen makes the key hydrogen bonding interaction with Asn869, and the dimethylthiazole moiety at N1 forms extensive hydrophobic contacts with the Phe902 shelf, consistent with earlier modeling studies.</div><figure id="fig10" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (A) Crystal structure of <b>33f</b> bound to human PARG at 2.25 Å resolution (PBD <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HML">6HML</a>). Hydrogen atoms modeled using Maestro (Schrödinger LLC). (B) Protein–ligand interaction diagram with key hydrogen bond interactions highlighted (pink arrows). (C) Overlay of crystal structures of <b>33f</b> (green carbons and ribbons) with <b>8a</b> (blue carbons and ribbons), showing maintenance of initial binding mode.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Encouraged by these good levels of potency in the biochemical and cellular assays, we obtained in vitro human microsomal clearance data to assess which derivatives might be candidates for further studies in vivo. Quinazolinediones <b>34b</b>, <b>34c</b>, <b>34e</b>, and <b>34f</b>, bearing a 3-(2-methylthiazol-5-yl)methyl group, showed excellent potency in our biochemical and cellular assays, however, their in vitro microsomal stabilities were poor, which we attributed to their high lipophilicity (e.g., <b>34f</b> calculated log <i>P</i><a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> = 4.7). Despite high intrinsic clearance, <b>34e</b> has been extensively characterized as a potent and selective chemical probe and used as an in vitro tool compound to study the effects of small molecule inhibition of PARG<a onclick="showRef(event, 'ref12 ref38'); return false;" href="javascript:void(0);" class="ref ref12 ref38">(12,38)</a> to evaluate the chemotherapeutic potential of PARG inhibitors. 3-(1-Methylpyrazol-4-yl)methyl derivatives <b>33a</b>–<b>f</b> maintained a similar level of potency to the 3-(2-methylthiazol-5-yl)methyl compounds, but were slightly less lipophilic (e.g., <b>33f</b> calculated log <i>P</i><a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> = 3.6) and consequently showed significantly better stability in human liver microsomes, although clearance remained moderate–high. In contrast to the majority of the 3-(1-methylpyrazol-4-yl)methyl quinazolinediones, 1-oxetan-3-ylmethyl derivative <b>33d</b> had low clearance in vitro and was a useful candidate for further investigation. 3-(3-Methylisoxazol-5-yl)methyl quinazolinediones <b>35a</b>–<b>e</b> exhibited a slight reduction in potency in our biochemical and cellular assays compared to 3-(1-methylpyrazol-4-yl)methyl derivatives <b>33</b> while maintaining a similar level of in vitro microsomal stability. A selection of 3-(5-methyl-1,3,4-thiadiazol-2-yl) and 3-cyanomethyl quinazolinediones <b>36b</b>–<b>e</b>, <b>41a</b>, and <b>41e</b> also showed good potency and human microsomal stability; unfortunately, however, their high in vitro mouse clearance precluded them from further studies.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Cytotoxicity and Selectivity</h3><div class="NLM_p">Pleasingly, all compounds which showed a cell activity of <0.3 μM in this series (exemplified by <b>33d</b>, <b>33e</b>, <b>34e</b>, and <b>35d</b>) showed more than a 100-fold window to cytotoxicity in HeLa cells after 72 h of exposure at PARG cell-active doses.</div><div class="NLM_p last">The selectivity of a set of quinazolinedione derivatives (<b>22f</b>, <b>33e</b>, <b>34c</b>, <b>33e</b>, and <b>35f</b>) was investigated by measuring activity against PARP1 and ARH3, a functionally related glycohydrolase enzyme. Encouragingly, none of the compounds tested showed any activity in either the PARP1 and ARH3 assays, giving EC<sub>50</sub> values of >150 and >100 μM, respectively.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Pharmacokinetic Investigations</h3><div class="NLM_p">1-Oxetan-3-ylmethyl derivatives <b>33d</b> and <b>35d</b> showed no cytotoxicity and a good balance of cell potency and human and mouse microsomal stability and were therefore selected for further profiling. For the preliminary pharmacokinetic (PK) investigations, mice were dosed intravenously or orally, and tail vein blood samples were analyzed at regular time-points (<a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">Table <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">10</a></a>).</div><div class="NLM_table-wrap" id="tbl10"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 10. In Vitro and in Vivo Parameters for <b>33d</b> and <b>35d</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>33d</b></th><th class="colsep0 rowsep0" align="center"><b>35d</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">solubility (μM)</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Caco-2 A–B mean <i>P</i><sub>app</sub> (10<sup>–6</sup> cm/s); (efflux ratio)</td><td class="colsep0 rowsep0" align="left">1.0 (34)</td><td class="colsep0 rowsep0" align="left">1.9 (21)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Mo Hep CL<sub>int</sub> (μL/min/10<sup>6</sup> cells)</td><td class="colsep0 rowsep0" align="left">24</td><td class="colsep0 rowsep0" align="left">25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CYP inhibition (IC<sub>50</sub>, μM, 5 isoforms)<a class="ref internalNav" href="#t10fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IV AUC (h·ng/mL)<a class="ref internalNav" href="#t10fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">250</td><td class="colsep0 rowsep0" align="left">90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IV <i>T</i><sub>1/2</sub> (min)</td><td class="colsep0 rowsep0" align="left">22</td><td class="colsep0 rowsep0" align="left">12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">IV <i>V</i><sub>D</sub> (L/kg)</td><td class="colsep0 rowsep0" align="left">1.4</td><td class="colsep0 rowsep0" align="left">2.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CL<sub>iv</sub> (L/h/kg)</td><td class="colsep0 rowsep0" align="left">4.0</td><td class="colsep0 rowsep0" align="left">10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>F</i> (%)</td><td class="colsep0 rowsep0" align="left">41</td><td class="colsep0 rowsep0" align="left">25</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t10fn1"><div class="footnote" id="t10fn1"><sup><sup>a</sup></sup><p class="last">CYP1A, CYP2C19, CYP2C9, CYP2D6, and CYP3A4 isoforms tested.</p></div><div class="footnote" id="t10fn2"><sup><sup>b</sup></sup><p class="last">Male CD-1 mice were dosed intravenously (<i>n</i> = 2) at a nominal dose level of 1 mg/kg, or orally (<i>n</i> = 2) at a nominal dose level of 5 mg/kg.</p></div></div></div><div class="NLM_p last">The in vivo PK data showed that both compounds were cleared rapidly, with <b>35d</b> having an IV clearance greater than liver blood flow, indicating that extrahepatic clearance mechanisms might be involved. Despite this, we were pleased to see moderate levels of bioavailability (<i>F</i>), especially as the in vitro data had shown high hepatocyte clearance, low permeability and high efflux.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64656" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64656" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">To date, the elucidation of the precise role of PARG in DNA damage repair and its potential as a novel therapeutic target has been severely limited by the lack of drug-like tool compounds suitable for use in cellular and in vivo models. Starting from a weakly active HTS hit <b>8a</b>, we have identified several lead-like scaffold hops that show a significant improvement in affinity in biochemical and cellular assays, without the nonspecific cellular toxicity of the parent anthraquinone. With the ligand binding site highly solvent-exposed, the tight binding interactions around the <i>N</i>-alkyl sulfonamide moiety of <b>8a</b> proved to be an essential pharmacophore that was exploited successfully by pursuing a structure-based design approach. An initial virtual screen identified a number of scaffold hops of interest; however, the steep SAR around the sulfonamide necessitated the synthesis of a specific set of <i>N</i>-alkyl groups in order to achieve a satisfactory level of potency. By pursuing a focused library design strategy, a number of structurally diverse bicyclic ring systems were found to be tolerated in the aromatic cleft between Tyr795 and Phe902, with the quinazolinedione scaffold deemed of particular interest based on level of activity, drug-like profile, and potential for further substitution to target additional features on the protein surface. High resolution crystal structures of some key compounds bound to human PARG helped to explain the observed SAR and to guide further design. Excellent levels of potency have been achieved, and in vitro tool compounds such as <b>34e</b>, with low nanomolar cellular activity, have been provided to elucidate the biological effects of PARG inhibition. These novel PARG inhibitors show no nonmechanistic toxicity, are selective over PARP1 and a functionally related glycohydrolase (ARH3), and represent promising lead compounds for the discovery of a novel class of chemotherapeutics. Preliminary investigations of 1-oxetan-3-ylmethyl derivatives <b>33d</b> and <b>35d</b> in vivo also demonstrated encouraging levels of oral bioavailability. Further progress on these efforts will be reported in due course.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55027" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55027" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> General Chemistry</h3><div class="NLM_p">Flash chromatography was performed using prepacked silica gel cartridges (KP-Sil SNAP, Biotage, Hengoed UK or RediSep Rf, Isco). Thin layer chromatography was conducted on 5 cm × 10 cm plates coated with Merck type 60 F<sub>254</sub> silica gel to a thickness of 0.25 mm. All reagents obtained from commercial sources were used without further purification. Anhydrous solvents were obtained from the Sigma-Aldrich Chemical Co. Ltd. or Fisher Chemicals Ltd. and used without further drying. HPLC grade solvents were obtained from Fisher Chemicals Ltd. All compounds were >95% purity as determined by examination of the LC–MS chromatograms at two independent pHs and the <sup>1</sup>H NMR spectra, unless otherwise specifically indicated. Where Cl or Br were present, expected isotopic distribution patterns were observed. Proton (<sup>1</sup>H) and carbon (<sup>13</sup>C) NMR spectra were recorded on a 300 MHz Bruker spectrometer. Solutions were typically prepared in either deuterochloroform (CDCl<sub>3</sub>) or deuterated dimethyl sulfoxide (DMSO-<i>d</i><sub>6</sub>) with chemical shifts referenced to tetramethylsilane (TMS) or deuterated solvent as an internal standard. <sup>1</sup>H NMR data are reported indicating the chemical shift (δ), the integration (e.g., 1H), the multiplicity (s, singlet; d, doublet; t, triplet; q, quartet; m, multiplet; br, broad; dd, doublet of doublets; etc.), and the coupling constant (<i>J</i>) in Hz (app implies apparent coupling on broadened signals). Deuterated solvents were obtained from Goss. LC–MS analyses were performed on a Waters Acquity UPLC system fitted with BEH C18 1.7 μM columns (2.1 mm × 50 mm) and with UV diode array detection (210–400 nm). Positive and negative mass ion detection was performed using a Waters SQD detector. Analyses were performed with either buffered acidic or basic solvents, with gradients as detailed in the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01407/suppl_file/jm8b01407_si_001.pdf" class="ext-link">Supporting Information</a>. Some compounds were purified by preparative HPLC on a Waters FractionLynx MS autopurification system, with a Waters XBridge 5 μm C18, 100 mm × 19 mm i.d. column, running at a flow rate of 20 mL/min with UV diode array detection (210–400 nm) and mass-directed collection using both positive and negative mass ion detection. Purifications were performed using buffered acidic or basic solvent systems as appropriate. Compound retention times on the system were routinely assessed using a 30–50 μL test injection and a standard gradient, then purified using an appropriately chosen focused gradient as detailed in the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01407/suppl_file/jm8b01407_si_001.pdf" class="ext-link">Supporting Information</a>, based upon observed retention time.</div><div class="NLM_p">Compounds <b>12d</b>,<b>e</b>, <b>13c</b>, <b>16b</b>, and <b>43</b>–<b>45</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) were purchased from either ChemBridge, ChemDiv, or Princeton BioMolecular Research.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 9,10-Dioxoanthracene-2-sulfonyl Chloride (<b>7</b>)</h4><div class="NLM_p last">Thionyl chloride (20 mL, 274 mmol) and DMF (1 mL) were sequentially added to 9,10-anthraquinone-2-sulfonic acid, sodium salt, monohydrate (5 g, 16 mmol) in a flask under nitrogen. The reaction mixture was heated at reflux for 2 h. The reaction mixture was left to cool to ambient temperature overnight and a precipitate formed. The reaction mixture was poured onto stirring ice and EtOAc (50 mL) was added to rinse the flask into the ice solution. The precipitate was filtered and dried in the vacuum oven to yield a pale-yellow solid, <b>7</b> (2.6 g, 53%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 9.00 (d, <i>J</i> = 2.0 Hz, 1H), 8.60 (d, <i>J</i> = 8.3 Hz, 1H), 8.35–8.47 (m, 3H), 7.89–7.96 (m, 2H). LC–MS <i>m</i>/<i>z</i> 305.5 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <i>N</i>-<i>tert</i>-Butyl-9,10-dioxo-anthracene-2-sulfonamide (<b>8a</b>)</h4><div class="NLM_p last">Triethylamine (91 μL, 0.65 mmol) and <i>tert</i>-butylamine (68.5 μL, 0.65 mmol) were added sequentially to a solution of <b>7</b> (100 mg, 0.33 mmol) in DCM (5 mL) at ambient temperature. The reaction mixture was stirred at ambient temperature overnight. The reaction mixture was diluted with DCM (5 mL) and washed with water (10 mL) and saturated aq sodium bicarbonate solution (10 mL). The organic phase was passed through a hydrophobic frit and evaporated to dryness. The crude product was purified by preparative HPLC (high pH) and lyophilized to dryness to yield <b>8a</b> as a white solid (48 mg, 43%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.61 (d, <i>J</i> = 1.8 Hz, 1H), 8.39 (d, <i>J</i> = 8.2 Hz, 1H), 8.31 (dd, <i>J</i> = 1.9, 8.1 Hz, 1H), 8.22–8.28 (m, 2H), 7.94–8.02 (m, 3H), 1.14 (s, 9H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 181.8, 181.7, 149.3, 135.0, 134.82, 134.77, 133.8, 133.1, 131.3, 128.1, 126.9, 124.4, 53.8, 29.8. HRMS (ESI) <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>18</sub>H<sub>17</sub>NO<sub>4</sub>NaS 366.0770, found 366.0765.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> <i>N</i>-Methyl-9,10-dioxo-anthracene-2-sulfonamide (<b>8b</b>)</h4><div class="NLM_p last">Methylamine solution (2 M in THF) (0.82 mL, 1.6 mmol) was added dropwise to a solution of <b>7</b> (100 mg, 0.33 mmol) in DCM (5 mL) cooled to 0 °C. The reaction mixture was allowed to warm to ambient temperature and stirred for 2 h. The reaction mixture was diluted with DCM (5 mL) and washed with water (10 mL) before passing through a hydrophobic frit and evaporating to dryness to yield <b>8b</b> as a beige solid (76 mg, 77%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) ä 8.52 (d, <i>J</i> = 1.8 Hz, 1H), 8.42 (d, <i>J</i> = 8.1 Hz, 1H), 8.21–8.29 (m, 3H), 7.90–8.03 (m, 3H), 2.48 (d, <i>J</i> = 3.7 Hz, 3H). LC–MS <i>m</i>/<i>z</i> 302.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <i>N</i>-Ethyl-9,10-dioxo-anthracene-2-sulfonamide (<b>8c</b>)</h4><div class="NLM_p last">Compound <b>8c</b> was prepared in a manner analogous to that for <b>8b</b>, purified by preparative HPLC (high pH) (white solid, 26 mg, 26%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.54 (d, <i>J</i> = 1.9 Hz, 1H), 8.41 (d, <i>J</i> = 8.2 Hz, 1H), 8.22–8.29 (m, 3H), 7.88–8.02 (m, 3H), 2.86 (q, <i>J</i> = 7.2 Hz, 2H), 1.00 (t, <i>J</i> = 7.2 Hz, 3H). LC–MS <i>m</i>/<i>z</i> 316.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> <i>N</i>-(1,1-Dimethylpropyl)-9,10-dioxo-anthracene-2-sulfonamide (<b>8d</b>)</h4><div class="NLM_p last">Compound <b>8d</b> was prepared in a manner analogous to that for <b>8b</b> (beige solid, 88 mg, quant.). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.83 (d, <i>J</i> = 1.9 Hz, 1H), 8.46 (d, <i>J</i> = 8.2 Hz, 1H), 8.34–8.43 (m, 2H), 8.31 (dd, <i>J</i> = 1.9, 8.2 Hz, 1H), 7.85–7.91 (m, 2H), 4.62 (s, 1H), 1.61 (q, <i>J</i> = 7.5 Hz, 2H), 1.23 (s, 6H), 0.88 (t, <i>J</i> = 7.4 Hz, 3H). LC–MS <i>m</i>/<i>z</i> 356.9 [M – H]<sup>−</sup>. LC–MS purity, 90–95%.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> <i>N</i>-(Cyclopropylmethyl)-9,10-dioxo-anthracene-2-sulfonamide (<b>8e</b>)</h4><div class="NLM_p last">Triethylamine (40 μL, 0.29 mmol) and cyclopropanemethylamine (23 μL, 0.27 mmol) were added sequentially to a solution of <b>7</b> (75 mg, 0.24 mmol) in DCM (5 mL) cooled to 0 °C. After addition of the reagents, the cooling was removed and the reaction mixture was allowed to warm to ambient temperature and stirred for 3 h. The reaction mixture was diluted with DCM (5 mL) and washed with water (10 mL) and saturated aq sodium bicarbonate solution (10 mL) before passing through a hydrophobic frit and evaporating to dryness. The crude product was purified by automated column chromatography, eluent 0–5% MeOH in DCM, to yield <b>8e</b> as a pale-yellow solid (54 mg, 64%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.56 (d, <i>J</i> = 1.6 Hz, 1H), 8.39 (d, <i>J</i> = 8.0 Hz, 1H), 8.17–8.32 (m, 4H), 7.94–8.01 (m, 2H), 2.74 (t, <i>J</i> = 6.5 Hz, 2H), 0.72–0.86 (m, 1H), 0.29–0.37 (m, 2H), 0.05–0.12 (m, 2H). LC–MS <i>m</i>/<i>z</i> 340.5 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> <i>N</i>-Cyclopropyl-9,10-dioxo-anthracene-2-sulfonamide (<b>8f</b>)</h4><div class="NLM_p last">Compound <b>8f</b> was prepared in a manner analogous to that for <b>8e</b>, stirring at ambient temperature for 1.5 h (pale-yellow solid, 39 mg, 49%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.56 (d, <i>J</i> = 1.8 Hz, 1H), 8.43 (d, <i>J</i> = 8.1 Hz, 1H), 8.34 (s, 1H), 8.23–8.31 (m, 3H), 7.95–8.01 (m, 2H), 2.16–2.26 (m, 1H), 0.37–0.55 (m, 4H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 181.8, 181.7, 145.2, 135.5, 133.8, 133.11, 133.08, 131.9, 128.2, 127.0, 124.9, 117.7, 38.7, 38.5, 24.1, 5.2. HRMS (ESI) <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>17</sub>H<sub>13</sub>NO<sub>4</sub>Na 350.0457, found 350.0449.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> <i>N</i>-Cyclobutyl-9,10-dioxo-anthracene-2-sulfonamide (<b>8g</b>)</h4><div class="NLM_p last">Compound <b>8g</b> was prepared in a manner analogous to that for <b>8a</b> (beige solid, 60 mg, 77%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.54 (d, <i>J</i> = 1.9 Hz, 1H), 8.39 (d, <i>J</i> = 8.1 Hz, 1H), 8.20–8.30 (m, 4H), 7.93–8.03 (m, 2H), 3.72 (quin, <i>J</i> = 8.2 Hz, 1H), 1.86–2.00 (m, 2H), 1.65–1.85 (m, 2H), 1.40–1.55 (m, 2H). LC–MS <i>m</i>/<i>z</i> 340.5 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> <i>N</i>-(1-Methylcyclopropyl)-9,10-dioxo-anthracene-2-sulfonamide (<b>8h</b>)</h4><div class="NLM_p last">Triethylamine (181 μL, 1.3 mmol) and 1-methylcyclopropanamine hydrochloride (70 mg, 0.65 mmol) were added sequentially to a solution of <b>7</b> (100 mg, 0.33 mmol) in DCM (5 mL) at 0 °C. The reaction mixture was allowed to warm to ambient temperature and stirred for 1.5 h. The reaction mixture was diluted with DCM (10 mL), 2 M HCl (10 mL) was added, and the reaction mixture was stirred for 10 min. The organic phase was separated, the aqueous phase was further extracted with DCM (10 mL), and the combined organic phase was passed through a hydrophobic frit and evaporated to dryness. The resulting solid was triturated with diethyl ether, filtered, and dried to yield <b>8h</b> as a pale-yellow solid (100 mg, 89%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.55 (d, <i>J</i> = 1.9 Hz, 1H), 8.52 (s, 1H), 8.40 (dd, <i>J</i> = 0.4, 8.1 Hz, 1H), 8.21–8.29 (m, 3H), 7.93–8.01 (m, 2H), 1.11 (s, 3H), 0.54–0.70 (m, 2H), 0.35–0.51 (m, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 181.74, 181.68, 147.8, 135.3, 134.83, 134.78, 133.8, 133.10, 133.06, 131.4, 128.2, 126.9, 124.5, 124.1, 117.7, 31.1, 24.0, 13.1. HRMS (ESI) <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>18</sub>H<sub>15</sub>NO<sub>4</sub>NaS 364.0614, found 364.0607. LC–MS purity, 90–95%.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>N</i>-(1-Cyanocyclopropyl)-9,10-dioxo-anthracene-2-sulfonamide (<b>8i</b>)</h4><div class="NLM_p last">Triethylamine (75 μL, 0.54 mmol) and 1-amino-1-cyclopropanecarbonitrile hydrochloride (32 mg, 0.27 mmol) were added sequentially to a solution of <b>7</b> (75 mg, 0.24 mmol) in DCM (5 mL) cooled to 0 °C. After addition of the reagents, the cooling was removed and the reaction mixture was allowed to warm to ambient temperature and stirring continued. After 5 h, further 1-amino-1-cyclopropanecarbonitrile hydrochloride (32 mg, 0.27 mmol) and triethylamine (75 μL, 0.54 mmol) were added to the reaction mixture, which was left to stir at ambient temperature for 48 h. LC–MS indicated ∼50% conversion to the desired product. The reaction mixture was heated at reflux for 5 h, and no further conversion to product observed by LC–MS. The reaction mixture was diluted with DCM (10 mL) and water (10 mL), separated, and the organic phase washed with saturated aq sodium bicarbonate (10 mL), then passed through a hydrophobic frit and evaporated to dryness. The crude product was purified by preparative HPLC (high pH) to yield <b>8i</b> as a cream solid (31 mg, 35%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.58 (d, <i>J</i> = 1.6 Hz, 1H), 8.43 (d, <i>J</i> = 8.2 Hz, 1H), 8.31 (dd, <i>J</i> = 1.9, 8.2 Hz, 1H), 8.23–8.29 (m, 2H), 7.96–8.02 (m, 2H), 1.39–1.45 (m, 2H), 1.20–1.27 (m, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 181.7, 181.5, 144.8, 135.8, 135.0, 134.9, 133.8, 133.04, 132.96, 132.0, 128.4, 127.03, 126.99, 125.1, 119.4, 117.7, 21.9, 15.8. HRMS (ESI) <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>18</sub>H<sub>12</sub>N<sub>2</sub>O<sub>4</sub>S 375.0410, found 375.0403.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 2-[(9,10-Dioxo-2-anthryl)sulfonylamino]acetamide (<b>8j</b>)</h4><div class="NLM_p last">Compound <b>8j</b> was prepared in a manner analogous to that for <b>8h</b>, stirring at ambient temperature for 4 h, and the crude product was purified by preparative HPLC (high pH) (cream solid, 12 mg, 15%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.55 (d, <i>J</i> = 1.7 Hz, 1H), 8.38 (d, <i>J</i> = 8.1 Hz, 1H), 8.20–8.30 (m, 4H), 7.93–8.03 (m, 2H), 7.34 (br s, 1H), 7.05 (br s, 1H), 3.32 (s, 2H). LC–MS <i>m</i>/<i>z</i> 343.5 [M – H]<sup>−</sup>. LC–MS purity, 90–95%.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 3-[(9,10-Dioxo-2-anthryl)sulfonylamino]propanamide (<b>8k</b>)</h4><div class="NLM_p last">Compound <b>8k</b> was prepared in a manner analogous to that for <b>8h</b>, stirring at ambient temperature for 4 h (pale-yellow solid, 27 mg, 32%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.54 (d, <i>J</i> = 1.8 Hz, 1H), 8.41 (d, <i>J</i> = 8.1 Hz, 1H), 8.23–8.30 (m, 3H), 8.10 (t, <i>J</i> = 5.9 Hz, 1H), 7.94–8.01 (m, 2H), 7.31 (br s, 1H), 6.83 (br s, 1H), 2.95–3.05 (m, 2H), 2.23 (t, <i>J</i> = 7.3 Hz, 2H). LC–MS <i>m</i>/<i>z</i> 357.5 [M – H]<sup>−</sup>. LC–MS purity, 90–95%.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> <i>N</i>-(2-Hydroxyethyl)-9,10-dioxo-anthracene-2-sulfonamide (<b>8l</b>)</h4><div class="NLM_p last">Compound <b>8l</b> was prepared in a manner analogous to that for <b>8a</b>, the crude product was triturated with DCM/diethyl ether, and the filtrate was isolated and dried in at 40 °C in a vacuum oven (cream solid, 59.5 mg, 79%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.55 (d, <i>J</i> = 1.9 Hz, 1H), 8.40 (d, <i>J</i> = 8.1 Hz, 1H), 8.19–8.33 (m, 3H), 8.11 (t, <i>J</i> = 5.8 Hz, 1H), 7.93–8.03 (m, 2H), 4.72 (t, <i>J</i> = 5.5 Hz, 1H), 3.38 (q, <i>J</i> = 6.0 Hz, 2H), 2.88 (q, <i>J</i> = 5.9 Hz, 2H). LC–MS <i>m</i>/<i>z</i> 330.2 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> <i>N</i>-(2-Methoxyethyl)-9,10-dioxo-anthracene-2-sulfonamide (<b>8m</b>)</h4><div class="NLM_p last">Compound <b>8m</b> was prepared in a manner analogous to that for <b>8a</b>, stirring at ambient temperature for 2 h (cream solid, 42 mg, 53%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.55 (d, <i>J</i> = 1.9 Hz, 1H), 8.39 (d, <i>J</i> = 8.1 Hz, 1H), 8.19–8.31 (m, 4H), 7.94–8.01 (m, 2H), 3.30 (t, <i>J</i> = 5.6 Hz, 2H), 3.11 (s, 3H), 3.01 (t, <i>J</i> = 5.5 Hz, 2H). LC–MS <i>m</i>/<i>z</i> 344.4 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> <i>N</i>-(2-Methoxy-1,1-dimethyl–ethyl)-9,10-dioxo-anthracene-2-sulfonamide (<b>8n</b>)</h4><div class="NLM_p last">Compound <b>8n</b> was prepared in a manner analogous to that for <b>8h</b>, stirring at ambient temperature overnight, and the crude product was purified by preparative HPLC (high pH) (beige solid, 26 mg, 31%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.61 (d, <i>J</i> = 1.6 Hz, 1H), 8.38 (d, <i>J</i> = 8.1 Hz, 1H), 8.21–8.32 (m, 3H), 7.92–8.05 (m, 3H), 3.14 (s, 2H), 2.99 (s, 3H), 1.11 (s, 6H). LC–MS <i>m</i>/<i>z</i> 372.5 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> <i>N</i>-(2-Cyanoethyl)-9,10-dioxo-anthracene-2-sulfonamide (<b>8o</b>)</h4><div class="NLM_p last">Compound <b>8o</b> was prepared in a manner analogous to that for <b>8a</b>, stirring at ambient temperature for 2 h (cream solid, 40 mg, 52%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.49–8.59 (m, 2H), 8.41 (d, <i>J</i> = 8.1 Hz, 1H), 8.21–8.34 (m, 3H), 7.93–8.03 (m, 2H), 3.08 (app q, <i>J</i> = 6.2 Hz, 2H), 2.65 (t, <i>J</i> = 6.4 Hz, 2H). LC–MS <i>m</i>/<i>z</i> 339.6 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 1,3-Dimethylquinazoline-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>10a</b>)</h4><div class="NLM_p last">A suspension of benzoylene urea (2.0 g, 12 mmol) in DMF (24 mL) was treated with potassium carbonate (8.5 g, 62 mmol) and was left to stir, under nitrogen at ambient temperature, for 30 min. The solution was then treated with iodomethane (2.3 mL, 37 mmol) and stirred at ambient temperature for 2 h. The suspension was filtered, and water (40 mL) and EtOAc (40 mL) were added to the filtrate. The organic layer was separated, washed with brine (2 × 20 mL), dried over MgSO<sub>4</sub>, and concentrated to give 1-<b>10a</b> as a white solid (0.96 g, 41%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.06 (dd, <i>J</i> = 1.7, 7.8 Hz, 1H), 7.81–7.75 (m, 1H), 7.47 (d, <i>J</i> = 8.5 Hz, 1H), 7.31 (app t, <i>J</i> = 7.5 Hz, 1H), 3.53 (s, 3H), 3.32 (s, 3H).</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 1,3-Diethylquinazoline-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>10b</b>)</h4><div class="NLM_p last">Compound <b>10b</b> was prepared in a manner analogous to that for <b>10a</b> (white solid, 807 mg, 30%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.07 (dd, <i>J</i> = 1.6, 7.9 Hz, 1H), 7.78 (ddd, <i>J</i> = 1.8, 7.3, 8.5 Hz, 1H), 7.51 (d, <i>J</i> = 8.6 Hz, 1H), 7.30 (app t, <i>J</i> = 7.5 Hz, 1H), 4.15 (q, <i>J</i> = 7.2 Hz, 2H), 3.99 (q, <i>J</i> = 7.2 Hz, 2H), 1.22 (t, <i>J</i> = 7.1 Hz, 3H), 1.16 (t, <i>J</i> = 7.1 Hz, 3H).</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 1,3-Dimethyl-2,4-dioxo-1,2,3,4-tetrahydroquinazoline-6-sulfonyl Chloride (<b>11a</b>)</h4><div class="NLM_p last">Chlorosulfonic acid (1.7 mL, 25 mmol) was added portionwise to stirring <b>10a</b> (960 mg, 5.1 mmol) cooled to 5–10 °C. The reaction mixture was heated at 60 °C for 4 h. The reaction mixture was cooled and poured into crushed ice, causing the formation of an off-white precipitate. The precipitate was filtered, washed with water, and dried in the vacuum oven to give <b>11a</b> as a white solid (0.98 g, 67%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.25 (d, <i>J</i> = 2.1 Hz, 1H), 7.93 (dd, <i>J</i> = 2.2, 8.7 Hz, 1H), 7.42 (d, <i>J</i> = 8.8 Hz, 1H), 3.52 (s, 3H), 3.31 (s, 3H).</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 1,3-Diethyl-2,4-dioxo-1,2,3,4-tetrahydroquinazoline-6-sulfonyl Chloride (<b>11b</b>)</h4><div class="NLM_p last">Compound <b>11b</b> was prepared in a manner analogous to that for <b>11a</b> (white solid, 657 mg, 64%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.26 (d, <i>J</i> = 2.5 Hz, 1H), 7.92 (dd, <i>J</i> = 2.1, 8.7 Hz, 1H), 7.46 (d, <i>J</i> = 8.9 Hz, 1H), 4.14 (q, <i>J</i> = 6.8 Hz, 2H), 3.99 (q, <i>J</i> = 7.0 Hz, 2H), 1.25–1.13 (m, 6H).</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> <i>N</i>-<i>tert</i>-Butyl-1,3-dimethyl-2,4-dioxo-quinazoline-6-sulfonamide (<b>12a</b>)</h4><div class="NLM_p last"><b>11a</b> (100 mg, 0.35 mmol), <i>tert</i>-butylamine (50 μL, 0.52 mmol), and triethylamine (0.10 mL, 0.69 mmol) in DCM (5 mL) was stirred at ambient temperature for 1 h. The reaction mixture was diluted with 2 M HCl (5 mL) and DCM (5 mL) and stirred vigorously for 10 min, then filtered through a hydrophobic frit. The organic layer was concentrated to dryness and the crude product purified by preparative HPLC (high pH) to yield <b>12a</b> as a white solid (83 mg, 73%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.44 (d, <i>J</i> = 2.3 Hz, 1H), 8.12 (dd, <i>J</i> = 2.4, 8.9 Hz, 1H), 7.70 (s, 1H), 7.63 (d, <i>J</i> = 8.9 Hz, 1H), 3.55 (s, 3H), 3.32 (s, 3H), 1.10 (s, 9H). LC–MS <i>m</i>/<i>z</i> 325.4 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 1,3-Dimethyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>12b</b>)</h4><div class="NLM_p last">Compound <b>12b</b> was prepared in a manner analogous to that for <b>12a</b> (white solid, 91 mg, 76%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.41 (d, <i>J</i> = 2.3 Hz, 1H), 8.19 (s, 1H), 8.07 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.65 (d, <i>J</i> = 8.9 Hz, 1H), 3.56 (s, 3H), 3.33 (s, 3H), 1.07 (s, 3H), 0.49–0.69 (m, 2H), 0.32–0.47 (m, 2H). LC–MS <i>m</i>/<i>z</i> 324.5 [M + H]<sup>+</sup>. <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.7, 150.5, 142.8, 136.7, 132.5, 126.4, 117.7, 115.8, 114.8, 31.0, 30.9, 28.3, 23.9, 12.9. HRMS (ESI) <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>14</sub>H<sub>17</sub>N<sub>3</sub>O<sub>4</sub>NaS 346.0832, found 346.0825. LC–MS purity, 90–95%.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> <i>N</i>-(1-Cyanocyclopropyl)-1,3-dimethyl-2,4-dioxo-quinazoline-6-sulfonamide (<b>12c</b>)</h4><div class="NLM_p last">Compound <b>12c</b> was prepared in a manner analogous to that for <b>12a</b>, the crude product was purified by automated column chromatography, eluent 0–100% EtOAc in <i>iso</i>-hexane (white solid, 21 mg, 18%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.27 (br s, 1H), 8.46 (s, 1H), 8.13 (d, <i>J</i> = 8.9 Hz, 1H), 7.71 (d, <i>J</i> = 8.9 Hz, 1H), 3.57 (s, 3H), 1.38–1.54 (m, 2H), 1.15–1.32 (m, 2H). 3H N-CH<sub>3</sub> not observed (under water peak). LC–MS <i>m</i>/<i>z</i> 335.5 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 1,3-Diethyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>13a</b>)</h4><div class="NLM_p last">Compound <b>13a</b> was prepared in a manner analogous to that for <b>12a</b> using <b>11b</b>, the reaction mixture was stirred at ambient temperature overnight, and the crude product was purified by automated column chromatography, eluent 0–100% EtOAc in <i>iso</i>-hexane (white solid, 52 mg, 47%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.42 (d, <i>J</i> = 2.3 Hz, 1H), 8.20 (s, 1H), 8.06 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.70 (d, <i>J</i> = 8.9 Hz, 1H), 4.17 (q, <i>J</i> = 6.9 Hz, 2H), 3.99 (q, <i>J</i> = 6.9 Hz, 2H), 1.23 (t, <i>J</i> = 7.1 Hz, 3H), 1.18 (t, <i>J</i> = 7.1 Hz, 3H), 1.08 (s, 3H), 0.52–0.64 (m, 2H), 0.36–0.44 (m, 2H). LC–MS <i>m</i>/<i>z</i> 351.6 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> <i>N</i>-(1-Cyanocyclopropyl)-1,3-diethyl-2,4-dioxo-quinazoline-6-sulfonamide (<b>13b</b>)</h4><div class="NLM_p last">Compound <b>13b</b> was prepared in a manner analogous to that for <b>12a</b> using <b>11b</b>, the reaction mixture was heated at 50 °C overnight and the crude product purified by automated column chromatography, eluent 0–100% EtOAc in <i>iso</i>-hexane (white solid, 36 mg, 44%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.26 (s, 1H), 8.47 (d, <i>J</i> = 2.3 Hz, 1H), 8.12 (dd, <i>J</i> = 2.3, 9.0 Hz, 1H), 7.76 (d, <i>J</i> = 9.0 Hz, 1H), 4.19 (q, <i>J</i> = 7.1 Hz, 2H), 4.00 (q, <i>J</i> = 7.0 Hz, 2H), 1.40–1.49 (m, 2H), 1.13–1.31 (m, 8H). LC–MS <i>m</i>/<i>z</i> 363.6 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> <i>N</i>-(1-Methylcyclopropyl)naphthalene-2-sulfonamide (<b>16</b>)</h4><div class="NLM_p last">To a solution of 2-naphthylsulfonyl chloride (100 mg, 0.44 mmol) in THF (5 mL) cooled in an ice bath was added a mixture of <i>N</i>,<i>N</i>-diisopropylethylamine (0.17 mL, 0.95 mmol) and 1-methylcyclopropanamine hydrochloride (0.050 mL, 0.46 mmol) in DCM (3 mL), and the mixture was stirred at ambient temperature overnight. The mixture was diluted with water (10 mL) and extracted with DCM (2 × 10 mL) dried over sodium sulfate and evaporated to dryness. The crude product mixture was purified by preparative HPLC (high pH) to yield <b>16</b> as a white solid (50 mg, 43%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.45 (d, <i>J</i> = 1.5 Hz, 1H), 8.09–8.20 (m, 3H), 8.02–8.07 (m, 1H), 7.80 (dd, <i>J</i> = 1.9, 8.7 Hz, 1H), 7.64–7.74 (m, 2H), 1.04 (s, 3H), 0.52–0.69 (m, 2H), 0.29–0.43 (m, 2H). LC–MS <i>m</i>/<i>z</i> 262.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 3-Methyl-1<i>H</i>-quinazoline-2,4-dione (<b>18</b>)</h4><div class="NLM_p last">A solution of 2-amino-<i>N</i>-methylbenzamide (15 g, 0.10 mol) and 1,1-carbonyldiimidazole (21 g, 0.13 mol) in DMF (150 mL) was heated at 135–145 °C overnight. The reaction showed 30% starting material by LC–MS, so more 1,1-carbonyldiimidazole (14 g, 0.086 mol) was added and the reaction mixture was heated at 140 °C overnight. The reaction showed complete conversion and so was cooled and poured into ice/water (300 mL) and stirred for 10 min. The suspension was filtered and the product was washed with water. The solid was dried in the vacuum oven at 50 °C overnight to give <b>18</b> as a white solid (16 g, 90%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.44 (br s, 1H), 7.93 (dd, <i>J</i> = 1.5, 7.8 Hz, 1H), 7.65 (app t, <i>J</i> = 7.7 Hz, 1H), 7.23–7.15 (m, 2H), 3.33 (s, 3H).</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 3-Methyl-2,4-dioxo-1,2,3,4-tetrahydroquinazoline-6-sulfonyl Chloride (<b>19</b>)</h4><div class="NLM_p last"><b>18</b> (15 g, 0.085 mol) was added portionwise with cooling to chlorosulfonic acid (60 mL, 0.9 mol) maintaining the temperature <20 °C. The reaction mixture was heated to 60 °C and stirred for 2 h then allowed to cool. The reaction mixture was added carefully to ice (500 mL) over 30 min. The resultant precipitate was stirred for 30 min, collected by filtration, and washed with water. The solid was dried in the vacuum oven to give <b>19</b> as a white solid (23 g, 99%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.54 (s, 1H), 8.15 (d, <i>J</i> = 2.1 Hz, 1H), 7.83 (dd, <i>J</i> = 2.0, 8.4 Hz, 1H), 7.13 (d, <i>J</i> = 8.5 Hz, 1H), 3.25 (s, 3H).</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 3-Methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-1,2,3,4-tetrahydroquinazoline-6-sulfonamide (<b>20</b>)</h4><div class="NLM_p last"><b>19</b> (12 g, 44 mmol) and 1-methylcyclopropanamine hydrochloride (5.2 g, 48 mmol) were slurried in DCM (120 mL) at 20 °C. Triethylamine (13 mL, 96 mmol) was added over 15 min; this was exothermic, and the temperature was allowed to rise to 34 °C. As the reaction cooled, a precipitate formed. The mixture was stirred for 2 h at ambient temperature, then 1 M HCl (100 mL) was added and the mixture stirred for 25 min. The product was filtered from the biphasic mixture and washed with water (100 mL). The solid was dried in the vacuum oven to give <b>20</b> as a white solid (12 g, 89%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.83 (br s, 1H), 8.32 (d, <i>J</i> = 2.2 Hz, 1H), 8.13 (s, 1H), 7.99 (dd, <i>J</i> = 2.2, 8.6 Hz, 1H), 7.32 (d, <i>J</i> = 8.6 Hz, 1H), 1.06 (s, 3H), 0.62–0.55 (m, 2H), 0.42–0.35 (m, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.5, 150.2, 141.9, 136.5, 132.5, 126.4, 116.1, 113.7, 30.9, 27.2, 23.9, 13.0. HRMS (ESI) <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>13</sub>H<sub>15</sub>N<sub>3</sub>O<sub>4</sub>NaS 332.0675, found 332.0667.</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> <i>N</i>-((3-Methyl-2,4-dioxo-1,2,3,4-tetrahydroquinazolin-6-yl)sulfonyl)-<i>N</i>-(1-methylcyclopropyl)acetamide (<b>21</b>)</h4><div class="NLM_p last"><b>20</b> (11 g, 36 mmol) was slurried in pyridine (30 mL), and 4-dimethylaminopyridine (430 mg, 3.5 mmol) was added. Acetic anhydride (34 mL, 356 mmol) was added over 10 min, and the resultant thick slurry was stirred overnight at ambient temperature. The reaction mixture was diluted with EtOAc (25 mL) and filtered. The solid was washed with EtOAc (25 mL). The solid was transferred to a conical flask and slurried in water (40 mL) at 60 °C for 2 h. The slurry was cooled to 35 °C and filtered and washed with water (20 mL). The solid was dried in the vacuum oven to give <b>21</b> (10 g, 83%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.91 (br s, 1H), 8.39 (d, <i>J</i> = 2.3 Hz, 1H), 8.09 (dd, <i>J</i> = 2.3, 8.7 Hz, 1H), 7.33 (d, <i>J</i> = 8.8 Hz, 1H), 3.26 (s, 3H), 2.23 (s, 3H), 1.54 (s, 3H), 1.25–1.16 (m, 2H), 1.10–0.97 (m, 2H).</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 1-Ethyl-3-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>22a</b>)</h4><div class="NLM_p last">Potassium carbonate (47 mg, 0.34 mmol) was added to a solution of <b>21</b> (100 mg, 0.28 mmol) and iodoethane (34 μL, 0.43 mmol) in DMF (3 mL), and the reaction mixture was stirred at ambient temperature overnight, followed by heating at 50 °C for 3.5 h. EtOAc (25 mL) and water (25 mL) were added to the reaction mixture, and the organic phase was separated, passed through a hydrophobic frit, and evaporated to dryness. The crude intermediate was solubilized in THF (5 mL) and conc aq NH<sub>3</sub> (2 mL) was added, and the reaction mixture was heated at 50 °C for 29 h. The reaction mixture was evaporated to dryness, and the crude product was purified by automated column chromatography, eluent 0–50% EtOAc in <i>iso</i>-hexane, to yield <b>22a</b> as a yellow solid (27 mg, 28%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.74 (d, <i>J</i> = 2.3 Hz, 1H), 8.15 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.32 (d, <i>J</i> = 8.9 Hz, 1H), 4.95 (s, 1H), 4.25 (q, <i>J</i> = 7.1 Hz, 2H), 3.51 (s, 3H), 1.39 (t, <i>J</i> = 7.2 Hz, 3H), 1.26 (s, 3H), 0.74–0.83 (m, 2H), 0.47–0.55 (m, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.6, 150.0, 141.8, 136.6, 132.6, 126.7, 115.5, 115.0, 30.9, 28.2, 23.9, 13.0, 12.3. HRMS (ESI) <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>15</sub>H<sub>19</sub>N<sub>3</sub>O<sub>4</sub>NaS 360.0988, found 360.0979.</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 1-(2-Methoxyethyl)-3-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>22b</b>)</h4><div class="NLM_p last">A solution of <b>21</b> (75 mg, 0.19 mmol), potassium carbonate (32 mg, 0.23 mmol), and 2-bromoethyl methyl ether (27 μL, 0.29 mmol) in DMF (3 mL) was stirred at ambient temperature overnight. Methanol (3 mL) and potassium carbonate (13 mg, 0.10 mmol) were added to the reaction mixture, which was stirred at ambient temperature for 2 h. Water (10 mL) and brine (10 mL) were added to the reaction mixture, which was then extracted with EtOAc (50 mL), and the organic phase was separated, passed through a hydrophobic frit, and evaporated to dryness. The crude product was purified by preparative HPLC (high pH) to yield <b>22b</b> as a white solid (14 mg, 20%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.69 (d, <i>J</i> = 2.3 Hz, 1H), 8.09 (dd, <i>J</i> = 2.3, 9.0 Hz, 1H), 7.53 (d, <i>J</i> = 9.0 Hz, 1H), 4.95 (s, 1H), 4.35 (t, <i>J</i> = 5.3 Hz, 2H), 3.73 (t, <i>J</i> = 5.3 Hz, 2H), 3.49 (s, 3H), 3.32 (s, 3H), 1.24 (s, 3H), 0.73–0.80 (m, 2H), 0.46–0.54 (m, 2H). HRMS (ESI) <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>16</sub>H<sub>21</sub>N<sub>3</sub>O<sub>5</sub>NaS 390.1094, found 390.1084.</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 1-[2-(Dimethylamino)ethyl]-3-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>22c</b>)</h4><div class="NLM_p last">Compound <b>22c</b> was prepared in a manner analogous to that for <b>22b</b> (white solid, 18 mg, 25%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.73 (d, <i>J</i> = 2.3 Hz, 1H), 8.14 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.38 (d, <i>J</i> = 8.8 Hz, 1H), 4.96 (s, 1H), 4.30 (t, <i>J</i> = 7.3 Hz, 2H), 3.51 (s, 3H), 2.63 (t, <i>J</i> = 7.4 Hz, 2H), 2.37 (s, 6H), 1.26 (s, 3H), 0.75–0.81 (m, 2H), 0.47–0.55 (m, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.6, 150.3, 142.1, 136.6, 132.5, 115.7, 114.9, 55.7, 45.5, 41.7, 30.9, 28.3, 23.9, 13.0. HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>25</sub>N<sub>4</sub>O<sub>4</sub>S 381.1591, found 381.1578.</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 3-Methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-1-prop-2-ynyl-quinazoline-6-sulfonamide (<b>22d</b>)</h4><div class="NLM_p last"><b>21</b> (75 mg, 0.19 mmol), propargyl alcohol (22 μL, 0.38 mmol), and triphenylphosphine, polymer supported (240 mg, 0.38 mmol), were dissolved in chloroform (3 mL) under nitrogen at ambient temperature. After 2 min, diisopropyl azodicarboxylate (76 μL, 0.38 mmol) was added dropwise and the reaction was allowed to stir overnight at ambient temperature. Brine (5 mL) was added to the reaction mixture, which was then passed through a hydrophobic frit, the resulting aqueous phase was washed with DCM (10 mL), and the organic phase was evaporated to dryness. The resulting oil was solubilized in methanol (3 mL), potassium carbonate (27 mg, 0.19 mmol) was added, and the reaction mixture stirred at ambient temperature for 4.5 h. Brine (10 mL) and DCM (20 mL) were added to the reaction mixture, and the organic phase was separated with a hydrophobic frit and evaporated to dryness. The crude product was purified by automated column chromatography, eluent 0–80% EtOAc in <i>iso</i>-hexane, to yield <b>22d</b> as a white solid, (62 mg, 93%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.74 (d, <i>J</i> = 2.3 Hz, 1H), 8.19 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.49 (d, <i>J</i> = 8.8 Hz, 1H), 4.99 (d, <i>J</i> = 2.5 Hz, 2H), 4.98 (br s, 1H), 3.52 (s, 3H), 2.36 (t, <i>J</i> = 2.5 Hz, 1H), 1.26 (s, 3H), 0.75–0.83 (m, 2H), 0.49–0.56 (m, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.4, 149.9, 141.2, 137.3, 132.5, 126.6, 115.9, 115.2, 78.0, 75.7, 33.5, 30.9, 28.4, 23.9, 13.0. HRMS (ESI) <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>16</sub>H<sub>17</sub>N<sub>3</sub>O<sub>4</sub>NaS 370.0832, found 370.0823.</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 1-(Cyanomethyl)-3-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>22e</b>)</h4><div class="NLM_p last"><b>20</b> (50 mg, 0.16 mmol), bromoacetonitrile (12 μL, 0.18 mmol), and cesium carbonate (58 mg, 0.18 mmol) in DMF (2 mL) was stirred at ambient temperature for 5 h. Water (10 mL) was added and the reaction mixture extracted with EtOAc (2 × 10 mL), and the combined organic phase was passed through a hydrophobic frit and evaporated to dryness. The crude product was purified by preparative HPLC (high pH) to yield <b>22e</b> as a pale-yellow solid (43 mg, 75%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.44 (d, <i>J</i> = 2.3 Hz, 1H), 8.37 (br s, 1H), 8.15 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.79 (d, <i>J</i> = 8.8 Hz, 1H), 5.34 (s, 2H), 3.32 (s, 3H), 1.08 (s, 3H), 0.56–0.62 (m, 2H), 0.36–0.43 (m, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.4, 150.0, 140.9, 137.8, 132.7, 126.7, 115.7, 115.6, 115.3, 32.3, 31.0, 28.5, 23.9, 13.0. HRMS (ESI) <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>15</sub>H<sub>16</sub>N<sub>4</sub>O<sub>4</sub>NaS 371.0784, found 371.0775.</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 1-(Cyclopropylmethyl)-3-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>22f</b>)</h4><div class="NLM_p last">Compound <b>22f</b> was prepared in a manner analogous to that for <b>22b</b>, purified by automated column chromatography, eluent 0–30% EtOAc in DCM (white solid, 15 mg, 21%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.75 (d, <i>J</i> = 2.3 Hz, 1H), 8.15 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.44 (d, <i>J</i> = 8.9 Hz, 1H), 5.02 (s, 1H), 4.12 (d, <i>J</i> = 6.9 Hz, 2H), 3.51 (s, 3H), 1.26 (s, 3H), 0.83–0.75 (m, 2H), 0.64–0.49 (m, 6H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 192.6, 160.6, 150.6, 142.2, 136.7, 132.5, 126.7, 117.7, 116.0, 114.9, 47.4, 30.9, 28.4, 23.9, 13.0, 9.4, 3.7. HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>22</sub>N3O<sub>4</sub>S 364.1326, found 364.1318.</div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 1-(Cyclohexylmethyl)-3-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>22g</b>)</h4><div class="NLM_p last">Compound <b>22g</b> was prepared in a manner analogous to that for <b>22b</b>, and the reaction mixture was stirred at ambient temperature for 7 days and purified by automated column chromatography, eluent 0–50% EtOAc in <i>iso</i>-hexane (white solid, 25 mg, 32%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.73 (d, <i>J</i> = 2.3 Hz, 1H), 8.12 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.29 (d, <i>J</i> = 5.0 Hz, 1H), 4.94 (s, 1H), 4.05 (d, <i>J</i> = 7.1 Hz, 2H), 3.51 (s, 3H), 1.65–1.85 (m, 6H), 1.27 (s, 3H), 1.15–1.26 (m, 5H), 0.76–0.82 (m, 2H), 0.47–0.56 (m, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.6, 150.6, 142.4, 136.5, 132.3, 126.6, 124.5, 117.7, 116.2, 114.9, 35.8, 30.9, 29.9, 28.4, 25.8, 25.3, 23.9, 13.0. HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>28</sub>N3O<sub>4</sub>S 406.1795, found 406.1787.</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 1-Benzyl-3-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>22h</b>)</h4><div class="NLM_p last">Compound <b>22h</b> was prepared in a manner analogous to that for <b>22a</b> (white solid, 101 mg, 88%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.73 (d, <i>J</i> = 2.3 Hz, 1H), 8.01 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.28–7.40 (m, 4H), 7.21–7.25 (m, 2H), 5.43 (s, 2H), 4.90 (s, 1H), 3.58 (s, 3H), 1.24 (s, 3H), 0.68–0.78 (m, 2H), 0.46–0.53 (m, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.7, 150.9, 142.1, 136.9, 135.8, 132.4, 128.7, 127.3, 126.6, 126.5, 116.0, 115.3, 46.7, 30.9, 28.5, 23.9, 12.9. HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>22</sub>N<sub>3</sub>O<sub>4</sub>S 400.1326, found 400.1316.</div></div><div id="sec4_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 1-[(2-Fluorophenyl)methyl]-3-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>22i</b>)</h4><div class="NLM_p last">Compound <b>22i</b> was prepared in a manner analogous to that for <b>22a</b> (white powder, 40%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ = 8.74 (d, <i>J</i> = 2.3 Hz, 1H), 8.04 (dd, <i>J</i> = 2.3, 9.2 Hz, 1H), 7.33–7.21 (m, 2H), 7.17–7.09 (m, 3H), 5.48 (s, 2H), 4.88 (s, 1H), 3.58 (s, 3H), 1.24 (s, 3H), 0.78–0.74 (m, 2H), 0.53–0.48 (m, 2H). LC–MS <i>m</i>/<i>z</i> 418.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 1-[(3-Fluorophenyl)methyl]-3-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>22j</b>)</h4><div class="NLM_p last">Compound <b>22j</b> was prepared in a manner analogous to that for <b>22a</b> (white powder, 55%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ = 8.72 (d, <i>J</i> = 2.3 Hz, 1H), 8.01 (dd, <i>J</i> = 2.3, 9.2 Hz, 1H), 7.36–7.29 (m, 1H), 7.16 (d, <i>J</i> = 8.7 Hz, 1H), 7.05–6.92 (m, 3H), 5.39 (s, 2H), 4.95 (s, 1H), 3.56 (s, 3H), 1.22 (s, 3H), 0.77–0.72 (m, 2H), 0.52–0.45 (m, 2H). LC–MS <i>m</i>/<i>z</i> 416.0 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 1-[(4-Fluorophenyl)methyl]-3-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>22k</b>)</h4><div class="NLM_p last">Compound <b>22k</b> was prepared in a manner analogous to that for <b>22a</b> (white powder, 28%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ = 8.72 (d, <i>J</i> = 2.3 Hz, 1H), 8.01 (dd, <i>J</i> = 2.3, 9.2 Hz, 1H), 7.28–7.18 (m, 3H), 7.08–7.00 (m, 2H), 5.37 (s, 2H), 4.93 (s, 1H), 3.55 (s, 3H), 1.22 (s, 3H), 0.77–0.71 (m, 2H), 0.52–0.45 (m, 2H). LC–MS <i>m</i>/<i>z</i> 416.1 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 1-[(2-Methoxyphenyl)methyl]-3-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>22l</b>)</h4><div class="NLM_p last">Compound <b>22l</b> was prepared in a manner analogous to that for <b>22a</b> (white powder, 17%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ = 8.70 (d, <i>J</i> = 2.3 Hz, 1H), 7.98 (dd, <i>J</i> = 2.3, 9.2 Hz, 1H), 7.31–7.19 (m, 2H), 6.97–6.91 (m, 2H), 6.88–6.82 (m, 1H), 5.40 (s, 2H), 4.90 (s, 1H), 3.94 (s, 3H), 3.56 (s, 3H), 1.22 (s, 3H), 0.78–0.71 (m, 2H), 0.51–0.42 (m, 2H). LC–MS <i>m</i>/<i>z</i> 428.0 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 1-[(3-Methoxyphenyl)methyl]-3-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>22m</b>)</h4><div class="NLM_p last">Compound <b>22m</b> was prepared in a manner analogous to that for <b>22a</b> (white powder, 45%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ = 8.71 (d, <i>J</i> = 1.8 Hz, 1H), 7.99 (dd, <i>J</i> = 2.3, 8.7 Hz, 1H), 7.29–7.23 (m, 1H), 7.21 (d, <i>J</i> = 8.7 Hz, 1H), 6.85–6.75 (m, 3H), 5.37 (s, 2H), 5.06 (s, 1H), 3.79–3.74 (m, 3H), 3.55 (s, 3H), 1.21 (s, 3H), 0.78–0.70 (m, 2H), 0.50–0.38 (m, 2H). LC–MS <i>m</i>/<i>z</i> 428.0 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 1-[(4-Methoxyphenyl)methyl]-3-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>22n</b>)</h4><div class="NLM_p last">Compound <b>22n</b> was prepared in a manner analogous to that for <b>22a (</b>white powder, 20%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ = 8.71 (d, <i>J</i> = 2.3 Hz, 1H), 8.00 (dd, <i>J</i> = 2.3, 8.7 Hz, 1H), 7.29–7.24 (m, 1H), 7.19 (d, <i>J</i> = 8.7 Hz, 2H), 6.87 (d, <i>J</i> = 8.7 Hz, 2H), 5.34 (s, 2H), 4.90 (s, 1H), 3.78 (s, 3H), 3.55 (s, 3H), 1.22 (s, 3H), 0.77–0.69 (m, 2H), 0.50–0.44 (m, 2H). LC–MS <i>m</i>/<i>z</i> 876.1 [2 M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_1_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 3-Methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-1-(2-pyridylmethyl)quinazoline-6-sulfonamide (<b>22o</b>)</h4><div class="NLM_p last">Compound <b>22o</b> was prepared in a manner analogous to that for <b>22a</b> (white powder, 25%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ = 8.69 (d, <i>J</i> = 2.3 Hz, 1H), 8.56–8.52 (m, 1H), 8.01 (dd, <i>J</i> = 2.3, 8.7 Hz, 1H), 7.66 (dt, <i>J</i> = 1.8, 7.8 Hz, 1H), 7.47 (d, <i>J</i> = 8.7 Hz, 1H), 7.31 (d, <i>J</i> = 8.2 Hz, 1H), 7.22 (t, <i>J</i> = 6.1 Hz, 1H), 5.49 (s, 2H), 5.28 (s, 1H), 3.54 (s, 3H), 1.20 (s, 3H), 0.76–0.70 (m, 2H), 0.50–0.42 (m, 2H). LC–MS <i>m</i>/<i>z</i> 401.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> 3-Methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-1-(3-pyridylmethyl)quinazoline-6-sulfonamide (<b>22p</b>)</h4><div class="NLM_p last">Compound <b>22p</b> was prepared in a manner analogous to that for <b>22a</b> (yellow powder, 16%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ = 8.73 (d, <i>J</i> = 2.3 Hz, 1H), 8.65 (d, <i>J</i> = 2.3 Hz, 1H), 8.58 (dd, <i>J</i> = 1.4, 5.0 Hz, 1H), 8.04 (dd, <i>J</i> = 2.5, 8.9 Hz, 1H), 7.63 (br d, <i>J</i> = 8.2 Hz, 1H), 7.34–7.26 (m, 1H), 7.21 (d, <i>J</i> = 8.7 Hz, 1H), 5.43 (s, 2H), 4.99 (s, 1H), 3.55 (s, 3H), 1.23 (s, 3H), 0.79–0.69 (m, 2H), 0.54–0.46 (m, 2H). LC–MS <i>m</i>/<i>z</i> 401.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> 3-Methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-1-(4-pyridylmethyl)quinazoline-6-sulfonamide (<b>22q</b>)</h4><div class="NLM_p last">Compound <b>22q</b> was prepared in a manner analogous to that for <b>22a</b> (yellow powder, 27%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ = 8.74 (d, <i>J</i> = 2.3 Hz, 1H), 8.63–8.57 (m, 2H), 8.02 (dd, <i>J</i> = 2.3, 8.7 Hz, 1H), 7.16 (d, <i>J</i> = 6.0 Hz, 2H), 7.05 (d, <i>J</i> = 8.7 Hz, 1H), 5.41 (s, 2H), 4.96 (s, 1H), 3.56 (s, 3H), 1.23 (s, 3H), 0.77–0.71 (m, 2H), 0.53–0.47 (m, 2H). LC–MS <i>m</i>/<i>z</i> 401.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> 3-Methyl-<i>N</i>-(1-methylcyclopropyl)-1-[(2-methylpyrazol-3-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>22r</b>)</h4><div class="NLM_p last">Compound <b>22r</b> was prepared in a manner analogous to that for <b>22d</b> (white powder, 22%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ = 8.74 (d, <i>J</i> = 2.3 Hz, 1H), 8.10 (dd, <i>J</i> = 2.3, 9.2 Hz, 1H), 7.38 (d, <i>J</i> = 1.8 Hz, 1H), 7.30 (d, <i>J</i> = 8.7 Hz, 1H), 6.00 (d, <i>J</i> = 2.3 Hz, 1H), 5.39 (s, 2H), 4.94 (s, 1H), 4.01 (s, 3H), 3.52 (s, 3H), 1.24 (s, 3H), 0.79–0.73 (m, 2H), 0.54–0.47 (m, 2H). LC–MS <i>m</i>/<i>z</i> 404.0 [M + H]<sup>+</sup>. NMR purity, 85–90%.</div></div><div id="sec4_1_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> 1-[(2,5-Dimethylpyrazol-3-yl)methyl]-3-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>22s</b>)</h4><div class="NLM_p last">Compound <b>22s</b> was prepared in a manner analogous to that for <b>22d</b> (white powder, 13%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ = 8.74 (d, <i>J</i> = 2.3 Hz, 1H), 8.11 (br dd, <i>J</i> = 2.3, 9.2 Hz, 1H), 7.29 (d, <i>J</i> = 8.7 Hz, 1H), 5.76 (s, 1H), 5.33 (s, 2H), 4.92 (s, 1H), 3.92 (s, 3H), 3.51 (s, 3H), 2.15 (s, 3H), 1.25 (s, 3H), 0.79–0.74 (m, 2H), 0.53–0.49 (m, 2H). LC–MS <i>m</i>/<i>z</i> 418.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> 1-[(2,4-Dimethylthiazol-5-yl)methyl]-3-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>22t</b>)</h4><div class="NLM_p last">Compound <b>22t</b> was prepared in a manner analogous to that for <b>22d</b> (off-white powder, 5%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.44 (d, <i>J</i> = 2.3 Hz, 1H), 8.22 (s, 1H), 8.11 (dd, <i>J</i> = 2.4, 8.9 Hz, 1H), 7.67 (d, <i>J</i> = 8.9 Hz, 1H), 5.45 (s, 2H), 3.34 (s, 3H), 2.49 (s, 3H), 2.47 (s, 3H), 1.08 (s, 3H), 0.66–0.56 (m, 2H), 0.43–0.36 (m, 2H). LC–MS <i>m</i>/<i>z</i> 435.5 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> 1-Methylquinazoline-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>24</b>)</h4><div class="NLM_p last">Sodium cyanate (30 g, 0.46 mol) was added to a slurry of <i>N</i>-methylanthranilic acid (50 g, 0.33 mol) in water (1.75 L) and acetic acid (3.3 mL). The reaction mixture was heated to 50 °C for 1 h. The solution was slowly basified by the addition of sodium hydroxide (exothermic). The resulting solution was heated to 80 °C and stirred overnight. The reaction mixture was cooled to 0 °C and the resultant precipitate collected by filtration. The solid was dissolved in boiling water (200 mL) and acidified with conc sulfuric acid to pH 2. The slurry was cooled to RT and filtered. The solid was dried in a vacuum oven to give <b>24</b> as a white powder (53 g, 77%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 11.54 (br s, 1H), 8.00 (d, <i>J</i> = 7.7 Hz, 1H), 7.77 (t, <i>J</i> = 7.9 Hz, 1H), 7.43 (d, <i>J</i> = 8.6 Hz, 1H), 7.28 (t, <i>J</i> = 7.5 Hz, 1H), 3.33 (s, 3H).</div></div><div id="sec4_1_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> 1-Methyl-2,4-dioxo-1,2,3,4-tetrahydroquinazoline-6-sulfonyl Chloride (<b>25</b>)</h4><div class="NLM_p last">Compound <b>24</b> (25 g, 0.14 mol) was added portionwise to chlorosulfonic acid (125 mL, 1.9 mol) at 50 °C. The reaction mixture was heated to 50 °C and stirred overnight then allowed to cool. The reaction mixture was added carefully to ice/water (1.5 L), maintaining the temperature <20 °C. The resultant precipitate was collected by filtration and dried in an open atmosphere to give <b>19</b> as a white powder (44 g, 113%, contains water). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 11.58 (s, 1H), 8.19 (d, <i>J</i> = 2.1 Hz, 1H), 7.91 (dd, <i>J</i> = 2.1, 8.6 Hz, 1H), 7.38 (d, <i>J</i> = 8.7 Hz, 1H), 3.44 (s, 3H).</div></div><div id="sec4_1_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-1,2,3,4-tetrahydroquinazoline-6-sulfonamide (<b>26</b>)</h4><div class="NLM_p last">Triethylamine (8.1 g, 80 mmol) was added to a stirred solution of <b>25</b> (11.3 g, 36 mmol) in DCM (250 mL). 1-Methylcyclopropanamine hydrochloride (4.3 g, 40 mmol) was added portionwise over 10 min, and the reaction mixture was stirred at RT for 3 h. The reaction mixture was poured onto water (250 mL) and filtered to afford <b>26</b> as a white powder (10.5 g, 93%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 11.83 (br s, 1H), 8.35 (d, <i>J</i> = 2.4 Hz, 1H), 8.17 (s, 1H), 8.06 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.61 (d, <i>J</i> = 8.9 Hz, 1H), 3.48 (s, 3H), 1.07 (s, 3H), 0.64–0.55 (m, 2H), 0.44–0.35 (m, 2H).</div></div></div><div id="sec4_1_54" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> Preparation of Compounds <b>27a</b>–<b>au</b></h3><div id="sec4_1_54_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> General Method A</h4><div class="NLM_p last">A solution of <b>26</b> (50 mg, 0.16 mmol) in DMF (1 mL) was treated with NaH (60% w/w in mineral oil) (6.6 mg, 0.17 mmol), and the resulting mixture was stirred at RT for 5 min. Alkyl halide, mesylate or tosylate (0.17 mmol), was added (if the alkyl chloride, mesylate or tosylate, was used, NaI (5 mg, 0.030 mmol) was also added), and the mixture was stirred at RT until analysis revealed complete reaction. Water (2 mL) and ethyl acetate (3 mL) were added, and the layers were separated. The organic layer was washed with water (2 × 2 mL) and concentrated under reduced pressure. Purification by mass directed prep HPLC gave the alkylated products.</div></div><div id="sec4_1_54_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> General Method B</h4><div class="NLM_p last">A mixture of <b>26</b> (50 mg, 0.161 mmol), polymer supported triphenylphosphine (202 mg, 0.32 mmol), and alcohol (0.32 mmol) in DMF (1.5 mL) was treated with diisopropyl azodicarboxylate (64 μL, 0.32 mmol) and stirred at RT until analysis revealed complete reaction. The reaction mixture was quenched with water (0.5 mL), filtered, and washed with EtOAc. The filtrate was concentrated and purified by prep HPLC to afford the alkylated products.</div></div><div id="sec4_1_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> 3-Ethyl-1-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>27a</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl iodide) to afford a white powder, 59%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 8.73 (d, <i>J</i> = 2.3 Hz, 1H), 8.17 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.32 (d, <i>J</i> = 8.9 Hz, 1H), 4.97 (s, 1H), 4.18 (q, <i>J</i> = 7.0 Hz, 2H), 3.67 (s, 3H), 1.32 (t, <i>J</i> = 7.1 Hz, 3H), 1.26 (s, 3H), 0.84–0.71 (m, 2H), 0.56–0.49 (m, 2H). LC–MS <i>m</i>/<i>z</i> 336.6 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> 3-(Cyclopropylmethyl)-1-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>27b</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl bromide) to afford a white powder, 18%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 8.72 (d, <i>J</i> = 2.3 Hz, 1H), 8.16 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.32 (d, <i>J</i> = 8.9 Hz, 1H), 5.10 (s, 1H), 4.00 (d, <i>J</i> = 7.2 Hz, 2H), 3.67 (s, 3H), 1.28–1.22 (m, 4H), 0.81–0.71 (m, 2H), 0.55–0.43 (m, 6H). LC–MS <i>m</i>/<i>z</i> 364.5 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> 3-(Cyclohexylmethyl)-1-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>27c</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl bromide) to afford a beige powder, 30%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 8.63–8.59 (m, 1H), 8.48 (br s, 1H), 8.19–8.12 (m, 1H), 7.32 (d, <i>J</i> = 8.9 Hz, 1H), 3.63 (s, 3H), 3.13 (d, <i>J</i> = 7.1 Hz, 2H), 1.85–1.65 (m, 7H), 1.28 (s, 3H), 1.24–1.17 (m, 2H), 0.97–0.80 (m, 4H), 0.63 (s, 2H). LC–MS <i>m</i>/<i>z</i> 406.6 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-3-phenyl-quinazoline-6-sulfonamide (<b>27d</b>)</h4><div class="NLM_p last">Following the method of Chan et al.,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> a mixture of <b>26</b> (100 mg, 0.32 mmol), phenylboronic acid (79 mg, 0.65 mmol), copper acetate (59 mg, 0.32 mmol), triethylamine (0.09 mL, 0.65 mmol), and 4 Å molecular sieves was slurried in DCM (2 mL) and stirred at RT for 3 d. The reaction mixture was filtered through Celite, and the filtrate was concentrated to dryness and partitioned between water (10 mL) and DCM (10 mL). The organic layer was passed through a hydrophobic frit, concentrated to dryness, and purified by prep HPLC to afford <b>27d</b> (23 mg, 18%) as a white powder. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 8.75 (d, <i>J</i> = 2.3 Hz, 1H), 8.23 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.60–7.48 (m, 3H), 7.40 (d, <i>J</i> = 8.9 Hz, 1H), 7.32–7.29 (m, 2H), 4.98 (s, 1H), 3.71 (s, 3H), 1.29 (s, 3H), 0.83–0.77 (m, 2H), 0.57–0.52 (m, 2H). LC–MS <i>m</i>/<i>z</i> 386.6 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> 3-Benzyl-1-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>27e</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl bromide) to afford a white powder, 39%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 8.51 (s, 1H), 7.93 (br d, <i>J</i> = 8.8 Hz, 1H), 7.37–7.24 (m, 2H), 7.17–7.06 (m, 4H), 5.05 (s, 2H), 4.87 (s, 1H), 3.41 (s, 3H), 1.01 (s, 3H), 0.62–0.47 (m, 2H), 0.35–0.22 (m, 2H). LC–MS <i>m</i>/<i>z</i> 400.5 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-3-(2-pyridylmethyl)quinazoline-6-sulfonamide (<b>27f</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl bromide HBr salt plus an extra equivalent of base) to afford a white powder, 15%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ = 8.72 (d, <i>J</i> = 2.3 Hz, 1H), 8.50 (br d, <i>J</i> = 4.1 Hz, 1H), 8.16 (dd, <i>J</i> = 2.3, 8.7 Hz, 1H), 7.66 (dt, <i>J</i> = 1.6, 7.7 Hz, 1H), 7.32 (d, <i>J</i> = 8.7 Hz, 2H), 7.19–7.13 (m, 1H), 5.42 (s, 2H), 4.92 (s, 1H), 3.66 (s, 3H), 1.25 (s, 3H), 0.79–0.72 (m, 2H), 0.55–0.47 (m, 2H). LC–MS <i>m</i>/<i>z</i> 401.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_61" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-3-(3-pyridylmethyl)quinazoline-6-sulfonamide (<b>27g</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl bromide HBr salt plus an extra equivalent of base) to afford a white powder, 15%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ = 8.81 (br s, 1H), 8.67 (d, <i>J</i> = 2.3 Hz, 1H), 8.53 (br d, <i>J</i> = 2.3 Hz, 1H), 8.13 (dd, <i>J</i> = 2.3, 8.7 Hz, 1H), 7.92 (br d, <i>J</i> = 7.8 Hz, 1H), 7.33–7.27 (m, 2H), 5.28 (s, 2H), 5.20 (br s, 1H), 3.64 (s, 3H), 1.21 (s, 3H), 0.80–0.68 (m, 2H), 0.51–0.41 (m, 2H). LC–MS <i>m</i>/<i>z</i> 401.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_62" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-3-(4-pyridylmethyl)quinazoline-6-sulfonamide (<b>27h</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl bromide HBr salt plus an extra equivalent of base) to afford a white powder, 19%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ = 8.72 (d, <i>J</i> = 2.3 Hz, 1H), 8.61–8.52 (m, 2H), 8.22–8.11 (m, 1H), 7.43 (br d, <i>J</i> = 5.0 Hz, 2H), 7.32 (d, <i>J</i> = 9.2 Hz, 1H), 5.27 (s, 2H), 5.01 (s, 1H), 3.65 (s, 3H), 1.24 (s, 3H), 0.81–0.72 (m, 2H), 0.56–0.45 (m, 2H). LC–MS <i>m</i>/<i>z</i> 401.0 [M + H]<sup>+</sup>. LC–MS purity, 90–95%.</div></div><div id="sec4_1_63" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-3-phenacyl-quinazoline-6-sulfonamide (<b>27i</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl bromide) to afford a white powder, 37%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.43 (d, <i>J</i> = 2.3 Hz, 1H), 8.24 (s, 1H), 8.18–8.09 (m, 3H), 7.78–7.72 (m, 2H), 7.65–7.58 (m, 2H), 5.53 (s, 2H), 3.60 (s, 3H), 1.09 (s, 3H), 0.65–0.59 (m, 2H), 0.44–0.38 (m, 2H). LC–MS <i>m</i>/<i>z</i> 428.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_64" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-3-[2-oxo-2-(4-pyridyl)ethyl]quinazoline-6-sulfonamide (<b>27j</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl bromide HBr salt plus an extra equivalent of base) to afford a white powder, 6%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.90 (dd, <i>J</i> = 1.6, 4.7 Hz, 2H), 8.42 (d, <i>J</i> = 2.3 Hz, 1H), 8.24 (s, 1H), 8.16 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.98 (dd, <i>J</i> = 1.8, 4.7 Hz, 2H), 7.75 (d, <i>J</i> = 8.9 Hz, 1H), 5.56 (s, 2H), 3.60 (s, 3H), 1.09 (s, 3H), 0.66–0.56 (m, 2H), 0.44–0.39 (m, 2H). LC–MS <i>m</i>/<i>z</i> 429.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_65" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-3-prop-2-ynyl-quinazoline-6-sulfonamide (<b>27k</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 13%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.42 (d, <i>J</i> = 2.3 Hz, 1H), 8.23 (br s, 1H), 8.10 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.68 (d, <i>J</i> = 8.9 Hz, 1H), 4.69 (d, <i>J</i> = 2.4 Hz, 2H), 3.57 (s, 3H), 3.22–3.17 (m, 1H), 1.07 (s, 3H), 0.64–0.56 (m, 2H), 0.42–0.37 (m, 2H). LC–MS <i>m</i>/<i>z</i> 348.5 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_66" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99"> 3-(Cyanomethyl)-1-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>27l</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl bromide) to afford a white powder, 21%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.42 (d, <i>J</i> = 2.3 Hz, 1H), 8.21 (br s, 1H), 8.12 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.70 (d, <i>J</i> = 8.9 Hz, 1H), 4.97 (s, 2H), 3.58 (s, 3H), 1.08 (s, 3H), 0.67–0.52 (m, 2H), 0.47–0.34 (m, 2H). LC–MS <i>m</i>/<i>z</i> 349.5 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_67" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-(oxetan-3-ylmethyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>27m</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl tosylate) to afford a white powder, 42%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 8.72 (d, <i>J</i> = 2.3 Hz, 1H), 8.18 (d, <i>J</i> = 8.9 Hz, 1H), 7.36–7.30 (m, 1H), 4.96 (s, 1H), 4.82–4.74 (m, 2H), 4.71–4.64 (m, 2H), 4.44 (d, <i>J</i> = 6.7 Hz, 2H), 3.66 (s, 3H), 3.55–3.38 (m, 1H), 1.27 (s, 3H), 0.81–0.75 (m, 2H), 0.56–0.49 (m, 2H). LC–MS <i>m</i>/<i>z</i> 380.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_68" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> 3-(3-Furylmethyl)-1-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>27n</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 10%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 8.72 (d, <i>J</i> = 2.3 Hz, 1H), 8.15 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.60 (d, <i>J</i> = 0.7 Hz, 1H), 7.34 (t, <i>J</i> = 1.6 Hz, 1H), 7.30 (d, <i>J</i> = 8.9 Hz, 1H), 6.55 (s, 1H), 5.12 (s, 1H), 5.11 (s, 2H), 3.65 (s, 3H), 1.24 (s, 3H), 0.81–0.73 (m, 2H), 0.54–0.46 (m, 2H). LC–MS <i>m</i>/<i>z</i> 390.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_69" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-3-(3-thienylmethyl)quinazoline-6-sulfonamide (<b>27o</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl bromide) to afford a white powder, 25%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.30 (d, <i>J</i> = 2.3 Hz, 1H), 8.06 (br s, 1H), 7.96 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.54 (d, <i>J</i> = 8.9 Hz, 1H), 7.36–7.32 (m, 1H), 7.31–7.28 (m, 1H), 6.99 (dd, <i>J</i> = 1.3, 4.9 Hz, 1H), 4.99 (s, 2H), 3.44 (s, 3H), 0.95 (s, 3H), 0.51–0.44 (m, 2H), 0.31–0.23 (m, 2H). LC–MS <i>m</i>/<i>z</i> 406.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_70" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-[(2-methylthiazol-4-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>27p</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl chloride) to afford a white powder, 6%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.42 (d, <i>J</i> = 2.3 Hz, 1H), 8.22 (s, 1H), 8.11 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.68 (d, <i>J</i> = 8.9 Hz, 1H), 7.30 (s, 1H), 5.18 (s, 2H), 3.57 (s, 3H), 2.60 (s, 3H), 1.09 (s, 3H), 0.64–0.56 (m, 2H), 0.44–0.37 (m, 2H). LC–MS <i>m</i>/<i>z</i> 421.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_71" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-(oxazol-4-ylmethyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>27q</b>)</h4><div class="NLM_p last">Following the method of Lipshutz et al.,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> a mixture of 1,3-oxazol-4-ylmethanol (38 mg, 0.39 mmol), <b>26</b> (60 mg, 0.19 mmol), and polymer supported triphenylphosphine (242 mg, 0.39 mmol) in DMF (1 mL) was treated with a solution of bis(4-chlorobenzyl)azodicarboxylate (0.19 mL, 0.39 mmol) in DMF (1 mL) and stirred at RT for 2 h. The reaction mixture was filtered, and the filter cake washed with EtOAc (2 × 5 mL). The filtrate was concentrated, and then DCM (3 mL) was added. The flask was cooled in an ice bath, and the resulting hydrazine byproduct was collected by filtration and washed with DCM (2 × 5 mL). The DCM filtrate was concentrated and purified by prep HPLC to afford <b>12r</b> (10 mg, 13%) as a white powder. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.42 (d, <i>J</i> = 2.3 Hz, 1H), 8.28 (d, <i>J</i> = 0.9 Hz, 1H), 8.21 (s, 1H), 8.10 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 8.04 (d, <i>J</i> = 1.0 Hz, 1H), 7.68 (d, <i>J</i> = 8.9 Hz, 1H), 5.06 (s, 2H), 3.56 (s, 3H), 1.08 (s, 3H), 0.64–0.56 (m, 2H), 0.44–0.37 (m, 2H). LC–MS <i>m</i>/<i>z</i> 391.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_72" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-3-(thiazol-2-ylmethyl)quinazoline-6-sulfonamide (<b>27r</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 12%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.43 (d, <i>J</i> = 2.2 Hz, 1H), 8.34 (s, 1H), 8.12 (dd, <i>J</i> = 2.2, 8.9 Hz, 1H), 7.73–7.66 (m, 3H), 5.46 (s, 2H), 3.59 (s, 3H), 1.08 (s, 3H), 0.65–0.57 (m, 2H), 0.44–0.36 (m, 2H). LC–MS <i>m</i>/<i>z</i> 407.5 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_73" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-3-(thiazol-5-ylmethyl)quinazoline-6-sulfonamide (<b>27s</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 21%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 9.00 (s, 1H), 8.43 (d, <i>J</i> = 2.3 Hz, 1H), 8.20 (s, 1H), 8.09 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.93 (s, 1H), 7.66 (d, <i>J</i> = 8.9 Hz, 1H), 5.34 (s, 2H), 3.56 (s, 3H), 1.07 (s, 3H), 0.62–0.56 (m, 2H), 0.42–0.36 (m, 2H). LC–MS <i>m</i>/<i>z</i> 407.5 [M + H]<sup>+</sup>. LC–MS purity, 85–90%.</div></div><div id="sec4_1_74" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-[(2-methylthiazol-5-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>27t</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 16%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 8.73 (d, <i>J</i> = 2.3 Hz, 1H), 8.16 (dd, <i>J</i> = 2.4, 8.9 Hz, 1H), 7.75 (s, 1H), 7.30 (d, <i>J</i> = 8.9 Hz, 1H), 5.38 (s, 2H), 4.99 (s, 1H), 3.66 (s, 3H), 2.65 (s, 3H), 1.25 (s, 3H), 0.80–0.74 (m, 2H), 0.55–0.49 (m, 2H). LC–MS <i>m</i>/<i>z</i> 421.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_75" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> 3-[(2-Aminothiazol-5-yl)methyl]-1-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>27u</b>)</h4><div id="sec4_1_75_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i109" class="anchor-spacer"></div><h5 class="article-section__title" id="_i109">1.  Preparation of Alcohol with Protected Amino Group</h5><div class="NLM_p">(a) Following the method of Leblanc et al.,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> a mixture of methyl 2-aminothiazole-5-carboxylate (359 mg, 2.3 mmol), hexane-2,5-dione (0.29 mL, 2.5 mmol), and <i>p</i>-toluenesulfonic acid monohydrate (20 mg, 0.11 mmol) in toluene (15 mL) was heated to 80 °C for 3 h and then stirred at reflux for a further 17 h. The reaction mixture was then cooled to RT and concentrated. The crude material was columned, eluting with 0–50% EtOAc in hexane, then flushed with EtOAc to give methyl 2-(2,5-dimethylpyrrol-1-yl)thiazole-5-carboxylate as a pale-yellow oil (155 mg, 29%) (50% starting material was also recovered). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.43 (s, 1H), 5.94 (s, 2H), 3.88 (s, 3H), 2.25 (s, 6H).</div><div class="NLM_p last">(b) A solution of methyl 2-(2,5-dimethylpyrrol-1-yl)thiazole-5-carboxylate (1.2 g, 5.0 mmol) in THF (40 mL) was cooled to 0 °C and treated slowly with a solution of lithium aluminum hydride (1.0 M in THF, 5.5 mL, 5.5 mmol), and the mixture was stirred at RT for 20 min. The reaction mixture was cooled to 0 °C and carefully quenched with satd aq Rochelle’s salt (40 mL) and stirred vigorously at RT for 1 h. The mixture was allowed to settle, then the clear solution was decanted away from the sticky residue. The sticky residue was washed with EtOAc and decanted (×3) and the layers separated. The aqueous layer was re-extracted with EtOAc, and organic extracts washed with brine, passed through a hydrophobic frit, and concentrated to give an amber oil (989 mg, 96%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 7.65 (t, <i>J</i> = 0.9 Hz, 1H), 5.86 (s, 2H), 5.71 (t, <i>J</i> = 5.7 Hz, 1H), 4.71 (dd, <i>J</i> = 1.0, 5.7 Hz, 2H), 2.19–2.05 (m, 6H). LC–MS purity, 90–95%.</div></div><div id="sec4_1_75_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i110" class="anchor-spacer"></div><h5 class="article-section__title" id="_i110">2.  Quinazolinedione Alkylation</h5><div class="NLM_p last">General method B was followed to afford crude material which was progressed to the deprotection reaction without purification.</div></div><div id="sec4_1_75_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i111" class="anchor-spacer"></div><h5 class="article-section__title" id="_i111">3.  Deprotection</h5><div class="NLM_p last">A mixture of the intermediate from the previous step (100 mg, 0.16 mmol), hydroxylamine hydrochloride (171 mg, 2.5 mmol), and 2 M NaOH (0.82 mL, 1.6 mmol) in EtOH (2 mL), water (2 mL), and DMF (2 mL) was heated to reflux for 3 d. The reaction mixture was concentrated and the residue taken up in DCM, then saturated aq NaHCO<sub>3</sub> and water were added, stirred for 5 min, and then passed through a hydrophobic frit and washed with DCM. The combined organics were concentrated and the residue purified by prep HPLC to give <b>12v</b> (8 mg, 12%) as a white powder. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.42 (d, <i>J</i> = 2.4 Hz, 1H), 8.19 (br s, 1H), 8.08 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.65 (d, <i>J</i> = 8.9 Hz, 1H), 6.96–6.82 (m, 3H), 5.06 (s, 2H), 3.56 (s, 3H), 1.07 (s, 3H), 0.64–0.53 (m, 2H), 0.45–0.32 (m, 2H). LC–MS <i>m</i>/<i>z</i> 422.2 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_1_76" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112"> 3-[(2,4-Dimethylthiazol-5-yl)methyl]-1-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>27v</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 4%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.43 (d, <i>J</i> = 2.2 Hz, 1H), 8.20 (s, 1H), 8.09 (dd, <i>J</i> = 2.2, 8.9 Hz, 1H), 7.65 (d, <i>J</i> = 8.9 Hz, 1H), 5.19 (s, 2H), 3.56 (s, 3H), 2.51 (3H, s), 2.44 (s, 3H), 1.07 (s, 3H), 0.67–0.55 (m, 2H), 0.43–0.34 (m, 2H). LC–MS <i>m</i>/<i>z</i> 435.5 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_77" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-[(2-methyl-4-phenyl-thiazol-5-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>27w</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 6%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.40 (d, <i>J</i> = 2.3 Hz, 1H), 8.35 (s, 1H), 8.10 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.77–7.69 (m, 2H), 7.66 (d, <i>J</i> = 8.9 Hz, 1H), 7.54–7.37 (m, 3H), 5.41 (s, 2H), 3.54 (s, 3H), 2.59 (s, 3H), 1.08 (s, 3H), 0.64–0.55 (m, 2H), 0.43–0.36 (m, 2H). LC–MS <i>m</i>/<i>z</i> 497.6 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_78" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114"> 3-(Isothiazol-5-ylmethyl)-1-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>27x</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 4%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.46–8.42 (m, 2H), 8.21 (s, 1H), 8.13–8.07 (m, 1H), 7.68 (d, <i>J</i> = 8.9 Hz, 1H), 7.43 (d, <i>J</i> = 1.7 Hz, 1H), 5.41 (s, 2H), 3.57 (s, 3H), 1.08 (s, 3H), 0.63–0.57 (m, 2H), 0.42–0.37 (m, 2H). LC–MS <i>m</i>/<i>z</i> 407.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_79" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-3-(thiadiazol-4-ylmethyl)quinazoline-6-sulfonamide (<b>27y</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 24%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 8.75 (d, <i>J</i> = 2.2 Hz, 1H), 8.65 (s, 1H), 8.19 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.40–7.32 (m, 1H), 5.81 (s, 2H), 4.92 (s, 1H), 3.68 (s, 3H), 1.27 (s, 3H), 0.82–0.74 (m, 2H), 0.57–0.49 (m, 2H). LC–MS <i>m</i>/<i>z</i> 406.5 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_80" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-[(4-methylthiadiazol-5-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>27z</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 50%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 8.75 (d, <i>J</i> = 2.2 Hz, 1H), 8.20 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.34 (d, <i>J</i> = 8.9 Hz, 1H), 5.51 (s, 2H), 4.97 (s, 1H), 3.69 (s, 3H), 2.93 (s, 3H), 1.27 (s, 3H), 0.80–0.74 (m, 2H), 0.56–0.50 (m, 2H). LC–MS <i>m</i>/<i>z</i> 422.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_81" class="NLM_sec NLM_sec_level_3"><div id="ac_i117" class="anchor-spacer"></div><h4 class="article-section__title" id="_i117"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-[(5-methyl-1,3,4-thiadiazol-2-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>27aa</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl chloride) to afford a white powder, 34%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.43 (d, <i>J</i> = 2.3 Hz, 1H), 8.23 (s, 1H), 8.12 (dd, <i>J</i> = 2.2, 8.9 Hz, 1H), 7.70 (d, <i>J</i> = 8.9 Hz, 1H), 5.50 (s, 2H), 3.58 (s, 3H), 2.68 (s, 3H), 1.08 (s, 3H), 0.64–0.56 (m, 2H), 0.44–0.37 (m, 2H). LC–MS <i>m</i>/<i>z</i> 422.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_82" class="NLM_sec NLM_sec_level_3"><div id="ac_i118" class="anchor-spacer"></div><h4 class="article-section__title" id="_i118"> 3-(1<i>H</i>-Imidazol-4-ylmethyl)-1-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>27ab</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 5%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.42 (d, <i>J</i> = 2.3 Hz, 1H), 8.26 (s, 1H), 8.09 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.66 (d, <i>J</i> = 8.9 Hz, 1H), 7.48 (d, <i>J</i> = 0.9 Hz, 1H), 6.95 (s, 1H), 5.07 (s, 2H), 3.56 (s, 3H), 1.07 (s, 3H), 0.62–0.55 (m, 2H), 0.42–0.36 (m, 2H). LC–MS <i>m</i>/<i>z</i> 390.1 [M + H]<sup>+</sup>. LC–MS purity, 85–90%.</div></div><div id="sec4_1_83" class="NLM_sec NLM_sec_level_3"><div id="ac_i119" class="anchor-spacer"></div><h4 class="article-section__title" id="_i119"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-[(3-methylimidazol-4-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>27ac</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 23%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 8.73 (d, <i>J</i> = 2.3 Hz, 1H), 8.21–8.15 (m, 1H), 7.43 (s, 1H), 7.34–7.30 (m, 2H), 5.27 (s, 2H), 5.00 (s, 1H), 3.87 (s, 3H), 3.66 (s, 3H), 1.26 (s, 3H), 0.82–0.71 (m, 2H), 0.55–0.48 (m, 2H). LC–MS <i>m</i>/<i>z</i> 404.6 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_84" class="NLM_sec NLM_sec_level_3"><div id="ac_i120" class="anchor-spacer"></div><h4 class="article-section__title" id="_i120"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-[(4-methyl-1,2,4-triazol-3-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>27ad</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl chloride) to afford a white powder, 6%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.43 (d, <i>J</i> = 2.3 Hz, 1H), 8.40 (s, 1H), 8.23 (s, 1H), 8.13 (dd, <i>J</i> = 2.4, 8.9 Hz, 1H), 7.72 (d, <i>J</i> = 8.9 Hz, 1H), 5.27 (s, 2H), 3.75 (s, 3H), 3.58 (s, 3H), 1.08 (s, 3H), 0.65–0.57 (m, 2H), 0.44–0.37 (m, 2H). LC–MS <i>m</i>/<i>z</i> 405.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_85" class="NLM_sec NLM_sec_level_3"><div id="ac_i121" class="anchor-spacer"></div><h4 class="article-section__title" id="_i121"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-[(1-methyltetrazol-5-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>27ae</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl chloride) to afford a white powder, 30%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.43 (d, <i>J</i> = 2.2 Hz, 1H), 8.23 (s, 1H), 8.16–8.11 (m, 1H), 7.72 (d, <i>J</i> = 8.9 Hz, 1H), 5.46 (s, 2H), 4.17 (s, 3H), 3.58 (s, 3H), 1.08 (s, 3H), 0.64–0.58 (m, 2H), 0.43–0.38 (m, 2H). LC–MS <i>m</i>/<i>z</i> 406.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_86" class="NLM_sec NLM_sec_level_3"><div id="ac_i122" class="anchor-spacer"></div><h4 class="article-section__title" id="_i122"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-[(3-methyl-1<i>H</i>-pyrazol-5-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>27af</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 11%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 8.70 (d, <i>J</i> = 2.3 Hz, 1H), 8.14 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.30 (d, <i>J</i> = 9.1 Hz, 1H), 6.15 (s, 1H), 5.84 (s, 1H), 5.26 (s, 2H), 4.68 (br s, 1H), 3.65 (s, 3H), 2.25 (s, 3H), 1.24 (s, 3H), 0.80–0.72 (m, 2H), 0.51–0.44 (m, 2H). LC–MS <i>m</i>/<i>z</i> 402.2 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_87" class="NLM_sec NLM_sec_level_3"><div id="ac_i123" class="anchor-spacer"></div><h4 class="article-section__title" id="_i123"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-[(2-methylpyrazol-3-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>27ag</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 32%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ = 8.71 (d, <i>J</i> = 2.3 Hz, 1H), 8.15 (dd, <i>J</i> = 2.3, 8.7 Hz, 1H), 7.38 (d, <i>J</i> = 1.8 Hz, 1H), 7.30 (d, <i>J</i> = 8.7 Hz, 1H), 6.38 (d, <i>J</i> = 1.8 Hz, 1H), 5.29 (s, 2H), 4.93 (s, 1H), 4.08 (s, 3H), 3.64 (s, 3H), 1.23 (s, 3H), 0.79–0.73 (m, 2H), 0.55–0.48 (m, 2H). LC–MS <i>m</i>/<i>z</i> 404.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_88" class="NLM_sec NLM_sec_level_3"><div id="ac_i124" class="anchor-spacer"></div><h4 class="article-section__title" id="_i124"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-[(1-methylpyrazol-3-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>27ah</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 11%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 8.75 (d, <i>J</i> = 2.3 Hz, 1H), 8.16 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.31 (d, <i>J</i> = 8.9 Hz, 1H), 7.28–7.26 (m, 1H), 6.29 (d, <i>J</i> = 2.2 Hz, 1H), 5.32 (s, 2H), 4.95 (s, 1H), 3.86 (s, 3H), 3.67 (s, 3H), 1.26 (s, 3H), 0.81–0.75 (m, 2H), 0.55–0.49 (m, 2H). LC–MS <i>m</i>/<i>z</i> 404.6 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_89" class="NLM_sec NLM_sec_level_3"><div id="ac_i125" class="anchor-spacer"></div><h4 class="article-section__title" id="_i125"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-[(1-methylpyrazol-4-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>27ai</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 9%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.42 (d, <i>J</i> = 2.3 Hz, 1H), 8.19 (br s, 1H), 8.07 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.67 (d, <i>J</i> = 5.4 Hz, 1H), 7.64 (d, <i>J</i> = 8.9 Hz, 1H), 7.39 (s, 1H), 4.95 (s, 2H), 3.76 (s, 3H), 3.55 (s, 3H), 1.07 (s, 3H), 0.65–0.52 (m, 2H), 0.45–0.33 (m, 2H). LC–MS <i>m</i>/<i>z</i> 404.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_90" class="NLM_sec NLM_sec_level_3"><div id="ac_i126" class="anchor-spacer"></div><h4 class="article-section__title" id="_i126"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-3-(1<i>H</i>-pyrazol-4-ylmethyl)quinazoline-6-sulfonamide (<b>27aj</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 29%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 8.72 (d, <i>J</i> = 2.4 Hz, 1H), 8.15 (dd, <i>J</i> = 2.4, 8.9 Hz, 1H), 7.78 (s, 2H), 7.29 (d, <i>J</i> = 8.9 Hz, 1H), 5.18 (s, 2H), 4.97 (s, 1H), 3.65 (s, 3H), 2.18 (s, 1H), 1.24 (s, 3H), 0.80–0.74 (m, 2H), 0.54–0.48 (m, 2H). LC–MS <i>m</i>/<i>z</i> 390.4 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_91" class="NLM_sec NLM_sec_level_3"><div id="ac_i127" class="anchor-spacer"></div><h4 class="article-section__title" id="_i127"> 3-[(1-Ethylpyrazol-4-yl)methyl]-1-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>27ak</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 42%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 8.73 (d, <i>J</i> = 2.3 Hz, 1H), 8.15 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.65 (s, 1H), 7.60 (s, 1H), 7.35–7.30 (m, 1H), 5.13 (s, 2H), 5.06 (br s, 1H), 4.13 (q, <i>J</i> = 7.3 Hz, 2H), 3.65 (s, 3H), 1.47 (t, <i>J</i> = 7.3 Hz, 3H), 1.25 (s, 3H), 0.82–0.71 (m, 2H), 0.55–0.49 (m, 2H). LC–MS <i>m</i>/<i>z</i> 418.4 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_92" class="NLM_sec NLM_sec_level_3"><div id="ac_i128" class="anchor-spacer"></div><h4 class="article-section__title" id="_i128"> 3-[[1-(Cyanomethyl)pyrazol-4-yl]methyl]-1-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>27al</b>)</h4><div class="NLM_p last">A solution of <b>27ak</b> (45 mg, 0.12 mmol) and sodium hydride (60% w/w in mineral oil, 10 mg, 0.25 mmol) in DMF was stirred at ambient temperature for 30 min. Bromoacetonitrile (8.9 μL, 0.13 mmol) was then added to the reaction mixture, and the reaction mixture stirred at ambient temperature for 1 h. Water (1 mL) was carefully added to the reaction mixture, followed by 2 M HCl (1 mL). DCM (10 mL) was added, and the mixture was stirred vigorously for 10 min and then passed through a hydrophobic frit. The aqueous layer was washed with DCM (5 mL), and the combined organic phase concentrated to dryness in vacuo. The crude product was purified by prep HPLC to yield the desired product (6 mg, 12%) as a white powder. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.42 (d, <i>J</i> = 2.3 Hz, 1H), 8.08 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.87 (s, 1H), 7.65 (d, <i>J</i> = 8.9 Hz, 1H), 7.57 (s, 1H), 5.42 (s, 2H), 4.98 (s, 2H), 3.55 (s, 3H), 1.07 (s, 3H), 0.63–0.54 (m, 2H), 0.43–0.33 (m, 2H). LC–MS <i>m</i>/<i>z</i> 429.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_93" class="NLM_sec NLM_sec_level_3"><div id="ac_i129" class="anchor-spacer"></div><h4 class="article-section__title" id="_i129"> 3-(Isoxazol-5-ylmethyl)-1-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>27am</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl bromide) to afford a white powder, 29%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.50 (d, <i>J</i> = 1.8 Hz, 1H), 8.42 (d, <i>J</i> = 2.3 Hz, 1H), 8.23 (s, 1H), 8.12 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.70 (d, <i>J</i> = 8.9 Hz, 1H), 6.49 (d, <i>J</i> = 1.8 Hz, 1H), 5.29 (s, 2H), 3.57 (s, 3H), 1.08 (s, 3H), 0.64–0.56 (m, 2H), 0.43–0.37 (m, 2H). LC–MS <i>m</i>/<i>z</i> 391.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_94" class="NLM_sec NLM_sec_level_3"><div id="ac_i130" class="anchor-spacer"></div><h4 class="article-section__title" id="_i130"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-[(3-methylisoxazol-5-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>27an</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 38%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 8.73 (d, <i>J</i> = 2.3 Hz, 1H), 8.19 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.33 (d, <i>J</i> = 8.9 Hz, 1H), 6.16 (s, 1H), 5.39 (s, 2H), 4.95 (s, 1H), 3.67 (s, 3H), 2.27 (s, 3H), 1.26 (s, 3H), 0.80–0.75 (m, 2H), 0.55–0.50 (m, 2H). LC–MS <i>m</i>/<i>z</i> 405.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_95" class="NLM_sec NLM_sec_level_3"><div id="ac_i131" class="anchor-spacer"></div><h4 class="article-section__title" id="_i131"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-[(5-methylisoxazol-3-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>27ao</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl bromide) to afford a white powder, 18%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ = 8.71 (d, <i>J</i> = 2.3 Hz, 1H), 8.16 (dd, <i>J</i> = 2.3, 8.7 Hz, 1H), 7.34–7.27 (m, 1H), 6.02 (s, 1H), 5.31 (s, 2H), 4.94 (s, 1H), 3.65 (s, 3H), 2.37 (s, 3H), 1.24 (s, 3H), 0.79–0.72 (m, 2H), 0.53–0.47 (m, 2H). LC–MS <i>m</i>/<i>z</i> 446.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_96" class="NLM_sec NLM_sec_level_3"><div id="ac_i132" class="anchor-spacer"></div><h4 class="article-section__title" id="_i132"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-[(5-methylisoxazol-4-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>27ap</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 23%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.43 (d, <i>J</i> = 2.3 Hz, 1H), 8.38 (s, 1H), 8.08 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.65 (d, J = 8.9 Hz, 1H), 4.93 (s, 2H), 3.55 (s, 3H), 2.51 (s, 3H), 1.07 (s, 3H), 0.64–0.55 (m, 2H), 0.43–0.35 (m, 2H). LC–MS <i>m</i>/<i>z</i> 405.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_97" class="NLM_sec NLM_sec_level_3"><div id="ac_i133" class="anchor-spacer"></div><h4 class="article-section__title" id="_i133"> 3-[(3,5-Dimethylisoxazol-4-yl)methyl]-1-methyl-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>27aq</b>)</h4><div class="NLM_p last">Prepared by general method A (using the alkyl bromide) to afford a white powder, 30%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.43 (d, <i>J</i> = 2.3 Hz, 1H), 8.07 (dd, <i>J</i> = 2.4, 8.9 Hz, 1H), 7.64 (d, <i>J</i> = 8.9 Hz, 1H), 4.91 (s, 2H), 3.54 (s, 3H), 2.42 (s, 3H), 2.21 (s, 3H), 1.07 (s, 3H), 0.62–0.56 (m, 2H), 0.41–0.35 (m, 2H). LC–MS <i>m</i>/<i>z</i> 417.0 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_98" class="NLM_sec NLM_sec_level_3"><div id="ac_i134" class="anchor-spacer"></div><h4 class="article-section__title" id="_i134"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-[(3-methyl-1,2,4-oxadiazol-5-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>27ar</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 29%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.43 (d, <i>J</i> = 2.3 Hz, 1H), 8.25 (s, 1H), 8.15 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.73 (d, <i>J</i> = 8.9 Hz, 1H), 5.41 (s, 2H), 3.66–3.52 (m, 3H), 2.30 (s, 3H), 1.09 (s, 3H), 0.65–0.56 (m, 2H), 0.45–0.37 (m, 2H). LC–MS <i>m</i>/<i>z</i> 406.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_99" class="NLM_sec NLM_sec_level_3"><div id="ac_i135" class="anchor-spacer"></div><h4 class="article-section__title" id="_i135"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-[(5-methyl-1,3,4-oxadiazol-2-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>27as</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 10%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.33 (d, <i>J</i> = 2.3 Hz, 1H), 8.14 (br s, 1H), 8.04 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.62 (d, <i>J</i> = 8.9 Hz, 1H), 5.24 (s, 2H), 3.48 (s, 3H), 2.37 (s, 3H), 0.98 (s, 3H), 0.54–0.47 (m, 2H), 0.34–0.27 (m, 2H). LC–MS <i>m</i>/<i>z</i> 406.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_100" class="NLM_sec NLM_sec_level_3"><div id="ac_i136" class="anchor-spacer"></div><h4 class="article-section__title" id="_i136"> 1-Methyl-<i>N</i>-(1-methylcyclopropyl)-3-[(4-methyl-1,2,5-oxadiazol-3-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>27at</b>)</h4><div class="NLM_p last">Prepared by general method B to afford a white powder, 12%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.43 (d, J = 2.2 Hz, 1H), 8.22 (s, 1H), 8.15–8.09 (m, 1H), 7.70 (d, J = 8.9 Hz, 1H), 5.31 (s, 2H), 3.57 (s, 3H), 2.46 (s, 3H), 1.08 (s, 3H), 0.64–0.57 (m, 2H), 0.43–0.38 (m, 2H). LC–MS <i>m</i>/<i>z</i> 406.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_101" class="NLM_sec NLM_sec_level_3"><div id="ac_i137" class="anchor-spacer"></div><h4 class="article-section__title" id="_i137"> 2,4-Dioxo-1<i>H</i>-3,1-benzoxazine-6-sulfonyl Chloride (<b>30</b>)</h4><div class="NLM_p last">Isatoic acid anhydride <b>29</b> (5.0 g, 28 mmol) was added portionwise to stirring chlorosulfonic acid (9 mL, 140 mmol) at RT. A slight exotherm was observed during the addition, and the mixture turned very dark brown immediately. The mixture was heated at 50 °C with a base scrubber for 4 h. The reaction mixture was allowed to cool to RT and then added dropwise to stirring ice/water. The precipitated solid was filtered and oven-dried to give <b>30</b> (5.8 g, 79%) as a brown solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 11.82 (s, 1H), 8.06 (d, <i>J</i> = 2.0 Hz, 1H), 7.91 (dd, <i>J</i> = 2.0, 8.5 Hz, 1H), 7.11 (d, <i>J</i> = 8.5 Hz, 1H).</div></div><div id="sec4_1_102" class="NLM_sec NLM_sec_level_3"><div id="ac_i138" class="anchor-spacer"></div><h4 class="article-section__title" id="_i138"> 2-Amino-5-[(1-methylcyclopropyl)sulfamoyl]-<i>N</i>-[(2-methylthiazol-5-yl)methyl]benzamide (<b>31a</b>)</h4><div class="NLM_p last">A suspension of <b>30</b> (1.0 g, 3.8 mmol) in DMF (12 mL) was treated with 1-methylcyclopropanamine hydrochloride <b>28</b> (493 mg, 4.6 mmol) and cooled to −10 °C in an ice/MeOH bath. The suspension was treated dropwise with triethylamine (1.2 mL, 8.4 mmol), and the resulting solution was stirred at −10 °C for 30 min. A solution of (2-methyl-1,3-thiazol-5-yl)methanamine (400 mg, 3.1 mmol) in DMF (5 mL) was added dropwise to the reaction mixture, followed by triethylamine (1.1 mL, 7.6 mmol), and the reaction mixture was stirred at RT for 18 h. Water (25 mL) was added and the mixture was extracted with EtOAc (2 × 25 mL), the organic extracts were washed with brine (25 mL), dried (hydrophobic frit), and concentrated. The crude product was purified by automated column chromatography, eluting with 0–5% MeOH in DCM to afford <b>31a</b> (286 mg, 20%) as an off-white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 7.90 (d, <i>J</i> = 2.2 Hz, 1H), 7.62 (dd, <i>J</i> = 2.2, 8.8 Hz, 1H), 7.53 (br s, 1H), 7.12 (t, <i>J</i> = 6.0 Hz, 1H), 6.70 (d, <i>J</i> = 8.8 Hz, 1H), 6.24 (br s, 2H), 5.11 (s, 1H), 4.71 (d, <i>J</i> = 5.7 Hz, 2H), 3.08 (s, 1H), 2.68 (s, 3H), 1.18 (s, 3H), 0.80–0.72 (m, 2H), 0.47–0.42 (m, 2H).</div></div><div id="sec4_1_103" class="NLM_sec NLM_sec_level_3"><div id="ac_i139" class="anchor-spacer"></div><h4 class="article-section__title" id="_i139"> 2-Amino-5-[(1-methylcyclopropyl)sulfamoyl]-<i>N</i>-[(1-methylpyrazol-4-yl)methyl]benzamide (<b>31b</b>)</h4><div class="NLM_p last">Prepared in the same manner as <b>31a</b> using (1-methyl-1<i>H</i>-pyrazol-4-yl)methylamine, to afford the desired product as a white powder, 67%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.83 (t, <i>J</i> = 5.7 Hz, 1H), 7.87 (d, <i>J</i> = 2.3 Hz, 1H), 7.58 (s, 1H), 7.55 (s, 1H), 7.46 (dd, <i>J</i> = 2.2, 8.7 Hz, 1H), 7.35 (s, 1H), 7.09 (br s, 2H), 6.79 (d, <i>J</i> = 8.8 Hz, 1H), 4.22 (d, <i>J</i> = 5.7 Hz, 2H), 3.78 (s, 3H), 1.04 (s, 3H), 0.65–0.51 (m, 2H), 0.36–0.29 (m, 2H).</div></div><div id="sec4_1_104" class="NLM_sec NLM_sec_level_3"><div id="ac_i140" class="anchor-spacer"></div><h4 class="article-section__title" id="_i140"> 2-Amino-5-[(1-methylcyclopropyl)sulfamoyl]-<i>N</i>-[(3-methylisoxazol-5-yl)methyl]benzamide (<b>31c</b>)</h4><div class="NLM_p last">Prepared in the same manner as <b>32a</b> using (3-methylisoxazol-5-ylmethyl)amine, to afford the desired product as an off-white powder, 42%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 7.95 (d, <i>J</i> = 2.1 Hz, 1H), 7.65 (dd, <i>J</i> = 2.2, 9.1 Hz, 1H), 6.99 (br s, 1H), 6.72 (d, <i>J</i> = 8.6 Hz, 1H), 6.10 (s, 1H), 4.68 (d, <i>J</i> = 5.7 Hz, 2H), 2.29 (s, 3H), 1.22 (s, 3H), 0.79–0.75 (m, 2H), 0.49–0.44 (m, 2H).</div></div><div id="sec4_1_105" class="NLM_sec NLM_sec_level_3"><div id="ac_i141" class="anchor-spacer"></div><h4 class="article-section__title" id="_i141"> 2-Amino-<i>N</i>-(cyanomethyl)-5-[(1-methylcyclopropyl)sulfamoyl]benzamide (<b>31d</b>)</h4><div class="NLM_p last">Prepared in the same manner as <b>32a</b>, using aminoacetonitrile bisulfate plus an extra equivalent of trimethylamine, to afford the desired product as a white powder, 35%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 9.18 (t, <i>J</i> = 5.4 Hz, 1H), 7.94 (d, <i>J</i> = 2.1 Hz, 1H), 7.64 (s, 1H), 7.51 (dd, <i>J</i> = 2.1, 8.8 Hz, 1H), 7.24 (s, 2H), 6.85 (d, <i>J</i> = 8.9 Hz, 1H), 4.25 (d, <i>J</i> = 5.5 Hz, 2H), 1.06 (s, 3H), 0.62–0.56 (m, 2H), 0.38–0.30 (m, 2H).</div></div><div id="sec4_1_106" class="NLM_sec NLM_sec_level_3"><div id="ac_i142" class="anchor-spacer"></div><h4 class="article-section__title" id="_i142"> <i>N</i>-(1-Methylcyclopropyl)-3-[(2-methylthiazol-5-yl)methyl]-2,4-dioxo-1<i>H</i>-quinazoline-6-sulfonamide (<b>32a</b>)</h4><div class="NLM_p last">A solution of <b>31a</b> (283 mg, 0.74 mmol) in THF (8 mL) was cooled in an ice bath (reaction mixture fitted with NaOH scrubber) and treated with triphosgene (110 mg, 0.37 mmol), which caused precipitation of a white solid. The reaction mixture was stirred at RT for 18 h, then quenched with satd aq K<sub>2</sub>CO<sub>3</sub> (10 mL) and stirred for 3 h. The mixture was extracted with EtOAc (3 × 20 mL), and the organic extracts washed with brine, dried (hydrophobic frit), and concentrated to give <b>17a</b> (275 mg, 91%) as an off-white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 11.98 (s, 1H), 8.32 (d, <i>J</i> = 2.2 Hz, 1H), 8.14 (s, 1H), 8.01 (dd, <i>J</i> = 2.2, 8.6 Hz, 1H), 7.61 (s, 1H), 7.33 (d, <i>J</i> = 8.6 Hz, 1H), 5.19 (s, 2H), 1.06 (s, 3H), 0.65–0.51 (m, 2H), 0.45–0.31 (m, 2H).</div></div><div id="sec4_1_107" class="NLM_sec NLM_sec_level_3"><div id="ac_i143" class="anchor-spacer"></div><h4 class="article-section__title" id="_i143"> <i>N</i>-(1-Methylcyclopropyl)-3-[(1-methylpyrazol-4-yl)methyl]-2,4-dioxo-1<i>H</i>-quinazoline-6-sulfonamide (<b>32b</b>)</h4><div class="NLM_p last">Prepared from <b>31b</b> in the same manner as <b>32a</b>, with an ether trituration, to afford the desired product as a pale-yellow solid, 78%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ = 8.51 (d, <i>J</i> = 2.0 Hz, 1H), 8.05–8.03 (m, 1H), 7.68 (s, 1H), 7.54 (s, 1H), 7.29 (d, <i>J</i> = 8.8 Hz, 1H), 5.05 (s, 2H), 3.84 (s, 3H), 1.16 (s, 3H), 0.71–0.67 (m, 2H), 0.43–0.46 (m, 2H).</div></div><div id="sec4_1_108" class="NLM_sec NLM_sec_level_3"><div id="ac_i144" class="anchor-spacer"></div><h4 class="article-section__title" id="_i144"> <i>N</i>-(1-Methylcyclopropyl)-3-[(3-methylisoxazol-5-yl)methyl]-2,4-dioxo-1<i>H</i>-quinazoline-6-sulfonamide (<b>32c</b>)</h4><div class="NLM_p last">Prepared from <b>31c</b> in the same manner as <b>32a</b> to afford the desired product as a white solid, 97%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.03 (s, 1H), 8.31 (d, <i>J</i> = 8.3 Hz, 1H), 8.17 (s, 1H), 8.04–8.02 (m, 1H), 7.36 (d, <i>J</i> = 8.8 Hz, 1H), 6.33 (s, 1H), 5.16 (s, 2H), 2.17 (s, 3H), 1.07 (s, 3H), 0.58–0.56 (m, 2H), 0.40–0.37 (m, 2H).</div></div><div id="sec4_1_109" class="NLM_sec NLM_sec_level_3"><div id="ac_i145" class="anchor-spacer"></div><h4 class="article-section__title" id="_i145"> 3-(Cyanomethyl)-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-1H-quinazoline-6-sulfonamide (<b>32d</b>)</h4><div class="NLM_p last">Prepared from <b>31d</b> in the same manner as <b>32a</b>, to afford the desired product as an off-white solid, 84%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 12.13 (s, 1H), 8.33 (d, <i>J</i> = 2.2 Hz, 1H), 8.17 (s, 1H), 8.04 (dd, <i>J</i> = 2.2, 8.6 Hz, 1H), 7.36 (d, <i>J</i> = 8.7 Hz, 1H), 4.91 (s, 2H), 1.07 (s, 3H), 0.68–0.51 (m, 2H), 0.42–0.35 (m, 2H).</div></div></div><div id="sec4_1_110" class="NLM_sec NLM_sec_level_2"><div id="ac_i146" class="anchor-spacer"></div><h3 class="article-section__title" id="_i146"> Preparation of Compounds <b>33</b>–<b>36</b></h3><div id="sec4_1_110_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i147" class="anchor-spacer"></div><h4 class="article-section__title" id="_i147"> General Method C</h4><div class="NLM_p last">A mixture of 3-substituted quinazolinedione (<b>32a</b>, <b>32b</b>, <b>32c</b>, or <b>32d</b>; 0.21 mmol), K<sub>2</sub>CO<sub>3</sub> (34 mg, 0.25 mmol). and alkyl halide, mesylate or tosylate (if the alkyl chloride, mesylate or tosylate, was used, NaI (5 mg, 0.030 mmol) was also added) in DMF (2 mL) was heated to 80 °C under microwave irradiation for 20 min. The resulting mixture was diluted with DCM (5 mL) and water (2 mL), stirred vigorously for 10 min, then passed through a hydrophobic frit. The aqueous layer was re-extracted with DCM in a similar manner, and the combined organic extracts were concentrated and purified by prep HPLC to afford the <i>N</i>1-alkylated product.</div></div><div id="sec4_1_111" class="NLM_sec NLM_sec_level_3"><div id="ac_i148" class="anchor-spacer"></div><h4 class="article-section__title" id="_i148"> 1-Ethyl-<i>N</i>-(1-methylcyclopropyl)-3-[(1-methylpyrazol-4-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>33a</b>)</h4><div class="NLM_p last">Prepared by general method C (using the alkyl iodide) using <b>32b</b> to afford a white powder, 38%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.43 (d, <i>J</i> = 2.3 Hz, 1H), 8.20 (s, 1H), 8.06 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.70 (d, <i>J</i> = 9.0 Hz, 1H), 7.67 (s, 1H), 7.39 (s, 1H), 4.95 (s, 2H), 4.17 (q, <i>J</i> = 6.9 Hz, 2H), 3.76 (s, 3H), 1.23 (t, <i>J</i> = 7.0 Hz, 3H), 1.07 (s, 3H), 0.62–0.55 (m, 2H), 0.44–0.35 (m, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ = 160.3, 149.9, 142.3, 139.8, 136.7, 133.3, 128.9, 124.6, 116.5, 115.9, 114.1, 39.4, 38.9, 35.5, 31.8, 24.6, 14.1, 12.6. HRMS (ESI) <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>19</sub>H<sub>23</sub>N<sub>5</sub>O<sub>4</sub>SNa 440.1363, found 440.1350.</div></div><div id="sec4_1_112" class="NLM_sec NLM_sec_level_3"><div id="ac_i149" class="anchor-spacer"></div><h4 class="article-section__title" id="_i149"> <i>N</i>-(1-Methylcyclopropyl)-3-[(1-methylpyrazol-4-yl)methyl]-2,4-dioxo-1-prop-2-ynyl-quinazoline-6-sulfonamide (<b>33b</b>)</h4><div class="NLM_p last">Prepared by general method B, using <b>32b</b> as a starting material, to afford a white powder, 16%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.44 (d, <i>J</i> = 2.3 Hz, 1H), 8.24 (br s, 1H), 8.14 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.71 (s, 1H), 7.69 (s, <i>J</i> = 7.9 Hz, 1H), 7.40 (s, 1H), 4.99 (d, <i>J</i> = 2.2 Hz, 2H), 4.95 (s, 2H), 3.76 (s, 3H), 3.40 (s, 1H), 1.08 (s, 3H), 0.65–0.55 (m, 2H), 0.45–0.35 (m, 2H). HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>22</sub>N<sub>5</sub>O<sub>4</sub>S 428.1387, found 428.1378.</div></div><div id="sec4_1_113" class="NLM_sec NLM_sec_level_3"><div id="ac_i150" class="anchor-spacer"></div><h4 class="article-section__title" id="_i150"> 1-(Cyclopropylmethyl)-<i>N</i>-(1-methylcyclopropyl)-3-[(1-methylpyrazol-4-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>33c</b>)</h4><div class="NLM_p last">Prepared by general method B, using <b>32b</b> as a starting material to afford a white powder, 17%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.44 (d, <i>J</i> = 2.4 Hz, 1H), 8.38 (s, 1H), 8.07 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.80 (d, <i>J</i> = 8.9 Hz, 1H), 7.68 (s, 1H), 7.39 (s, 1H), 4.96 (s, 2H), 4.08 (d, <i>J</i> = 6.9 Hz, 2H), 3.76 (s, 3H), 1.24–1.19 (m, 1H), 1.08 (s, 3H), 0.63–0.56 (m, 2H), 0.52–0.44 (m, 4H), 0.42–0.37 (m, 2H). HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>26</sub>N<sub>5</sub>O<sub>4</sub>S 444.1700, found 444.1687.</div></div><div id="sec4_1_114" class="NLM_sec NLM_sec_level_3"><div id="ac_i151" class="anchor-spacer"></div><h4 class="article-section__title" id="_i151"> <i>N</i>-(1-Methylcyclopropyl)-3-[(1-methylpyrazol-4-yl)methyl]-1-(oxetan-3-ylmethyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>33d</b>)</h4><div class="NLM_p last">Prepared by general method C (using the alkyl tosylate), using <b>32b</b> as a starting material, to afford a white powder, 43%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.41 (d, <i>J</i> = 2.3 Hz, 1H), 8.37 (s, 1H), 8.02 (d, <i>J</i> = 8.9 Hz, 1H), 7.73–7.66 (m, 2H), 7.39 (s, 1H), 4.95 (s, 2H), 4.61 (dd, <i>J</i> = 6.2, 7.8 Hz, 2H), 4.52–4.44 (m, 4H), 3.76 (s, 3H), 3.44–3.39 (m, 1H), 1.07 (s, 3H), 0.59 (s, 2H), 0.42–0.37 (m, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ = 160.0, 150.7, 142.4, 140.2, 137.2, 133.3, 131.2, 129.1, 116.2, 116.0, 114.0, 74.7, 46.1, 38.9, 35.8, 33.9, 31.8, 24.6, 14.1. HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>26</sub>N<sub>5</sub>O<sub>5</sub>S 444.1649, found 444.1637.</div></div><div id="sec4_1_115" class="NLM_sec NLM_sec_level_3"><div id="ac_i152" class="anchor-spacer"></div><h4 class="article-section__title" id="_i152"> 1-[(2,5-Dimethylpyrazol-3-yl)methyl]-<i>N</i>-(1-methylcyclopropyl)-3-[(1-methylpyrazol-4-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>33e</b>)</h4><div class="NLM_p last">Prepared by general method B, using <b>32b</b> as a starting material to afford a white powder, 19%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.44 (d, <i>J</i> = 2.3 Hz, 1H), 8.21 (s, 1H), 8.02 (dd, <i>J</i> = 2.4, 8.9 Hz, 1H), 7.68 (s, 1H), 7.56 (d, <i>J</i> = 8.9 Hz, 1H), 7.39 (s, 1H), 5.73 (s, 1H), 5.37 (s, 2H), 4.97 (s, 2H), 3.82 (s, 3H), 3.76 (s, 3H), 1.99 (s, 3H), 1.07 (s, 3H), 0.62–0.55 (m, 2H), 0.42–0.35 (m, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ = 159.9, 150.2, 147.7, 142.0, 140.1, 137.6, 137.0, 133.3, 131.2, 128.8, 116.1, 116.0, 114.8, 105.4, 39.5, 38.9, 36.6, 36.0, 31.8, 24.6, 14.1, 13.2. HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>28</sub>N<sub>7</sub>O<sub>4</sub>S 498.1918, found 498.1905.</div></div><div id="sec4_1_116" class="NLM_sec NLM_sec_level_3"><div id="ac_i153" class="anchor-spacer"></div><h4 class="article-section__title" id="_i153"> 1-[(2,4-Dimethylthiazol-5-yl)methyl]-<i>N</i>-(1-methylcyclopropyl)-3-[(1-methylpyrazol-4-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>33f</b>)</h4><div class="NLM_p last">Prepared by general method B, using <b>32b</b> as a starting material, to afford a white powder, 21%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 8.72 (d, <i>J</i> = 2.3 Hz, 1H), 8.12 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.64 (s, 1H), 7.57 (s, 1H), 7.27–7.25 (m, 1H), 5.40 (s, 2H), 5.14 (s, 2H), 5.08 (s, 1H), 3.87 (s, 3H), 2.59 (s, 3H), 2.57 (s, 3H), 1.26 (s, 3H), 0.80–0.72 (m, 2H), 0.55–0.48 (m, 2H). HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>27</sub>N<sub>6</sub>O<sub>4</sub>S<sub>2</sub> 515.1530, found 515.1519. LC–MS purity, 90–95%.</div></div><div id="sec4_1_117" class="NLM_sec NLM_sec_level_3"><div id="ac_i154" class="anchor-spacer"></div><h4 class="article-section__title" id="_i154"> <i>N</i>-(1-Methylcyclopropyl)-3-[(2-methylthiazol-5-yl)methyl]-2,4-dioxo-1-prop-2-ynyl-quinazoline-6-sulfonamide (<b>34b</b>)</h4><div class="NLM_p last">Prepared by general method B, using <b>32a</b> as a starting material, to afford a white powder, 75%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.44 (d, <i>J</i> = 2.2 Hz, 1H), 8.15 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.71 (d, <i>J</i> = 8.9 Hz, 1H), 7.64 (s, 1H), 5.25 (s, 2H), 5.00 (d, <i>J</i> = 2.4 Hz, 2H), 3.44–3.39 (m, 1H), 2.57 (s, 3H), 1.08 (s, 3H), 0.65–0.54 (m, 2H), 0.48–0.32 (m, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 166.2, 159.8, 149.27, 142.5, 141.2, 137.6, 132.9, 131.8, 126.6, 116.2, 115.0, 77.8, 75.9, 37.2, 33.6, 30.9, 23.9, 18.7, 13.0. HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>21</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub> 445.0999, found 445.0987.</div></div><div id="sec4_1_118" class="NLM_sec NLM_sec_level_3"><div id="ac_i155" class="anchor-spacer"></div><h4 class="article-section__title" id="_i155"> 1-(Cyclopropylmethyl)-<i>N</i>-(1-methylcyclopropyl)-3-[(2-methylthiazol-5-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>34c</b>)</h4><div class="NLM_p last">Prepared by general method B, using <b>32a</b> as a starting material, to afford a white powder, 34%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.43 (d, <i>J</i> = 2.3 Hz, 1H), 8.08 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.81 (d, <i>J</i> = 8.9 Hz, 1H), 7.62 (s, 1H), 5.25 (s, 2H), 4.09 (d, <i>J</i> = 6.9 Hz, 2H), 2.57 (s, 3H), 1.22 (br s, 1H), 1.07 (s, 3H), 0.62–0.56 (m, 2H), 0.53–0.43 (m, 4H), 0.43–0.37 (m, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ = 160.1, 150.4, 142.7, 137.0, 133.5, 131.8, 128.9, 128.5, 115.6, 114.8, 108.9, 48.4, 37.2, 31.8, 24.7, 18.9, 14.1, 9.4, 4.2. HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>25</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub> 461.1312, found 461.1299.</div></div><div id="sec4_1_119" class="NLM_sec NLM_sec_level_3"><div id="ac_i156" class="anchor-spacer"></div><h4 class="article-section__title" id="_i156"> 1-[(2,5-Dimethylpyrazol-3-yl)methyl]-<i>N</i>-(1-methylcyclopropyl)-3-[(2-methylthiazol-5-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>34e</b>)</h4><div class="NLM_p last">Prepared by general method B, using <b>32a</b> as a starting material, to afford a white powder, 43%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.44 (d, <i>J</i> = 2.3 Hz, 1H), 8.21 (br s, 1H), 8.03 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.64 (s, 1H), 7.57 (d, <i>J</i> = 8.9 Hz, 1H), 5.75 (s, 1H), 5.38 (s, 2H), 5.27 (s, 2H), 3.82 (s, 3H), 2.58 (s, 3H), 1.99 (s, 3H), 1.07 (s, 3H), 0.58 (s, 2H), 0.39 (d, <i>J</i> = 2.0 Hz, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ = 159.8, 150.1, 147.8, 143.1, 142.0, 137.8, 136.4, 133.5, 131.2, 128.8, 123.9, 115.8, 115.0, 105.4, 39.6, 37.4, 36.7, 31.8, 24.6, 19.1, 14.1, 13.3. HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>27</sub>N<sub>6</sub>O<sub>4</sub>S<sub>2</sub> 515.1530, found 515.1518.</div></div><div id="sec4_1_120" class="NLM_sec NLM_sec_level_3"><div id="ac_i157" class="anchor-spacer"></div><h4 class="article-section__title" id="_i157"> 1-[(2,4-Dimethylthiazol-5-yl)methyl]-<i>N</i>-(1-methylcyclopropyl)-3-[(2-methylthiazol-5-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>34f</b>)</h4><div class="NLM_p last">Prepared by general method B, using <b>32a</b> as a starting material, to afford an off-white powder, 27%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.44 (d, <i>J</i> = 1.9 Hz, 1H), 8.23 (s, 1H), 8.12 (dd, <i>J</i> = 1.8, 8.9 Hz, 1H), 7.67 (s, 1H), 7.64 (s, 1H), 5.46 (s, 2H), 5.26 (s, 2H), 2.58 (s, 3H), 2.49 (s, 3H), 2.47 (s, 3H), 1.07 (s, 3H), 0.58 (s, 2H), 0.44–0.36 (m, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ = 167.6, 165.3, 159.8, 150.0, 143.0, 141.7, 137.8, 133.6, 131.2, 129.1, 124.7, 124.4, 115.8, 114.4, 40.8, 37.3, 31.9, 24.6, 23.4, 19.1, 15.4, 14.1. HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>26</sub>N<sub>5</sub>O<sub>4</sub>S<sub>3</sub> 532.1141, found 532.1129.</div></div><div id="sec4_1_121" class="NLM_sec NLM_sec_level_3"><div id="ac_i158" class="anchor-spacer"></div><h4 class="article-section__title" id="_i158"> 1-Ethyl-<i>N</i>-(1-methylcyclopropyl)-3-[(3-methylisoxazol-5-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>35a</b>)</h4><div class="NLM_p last">Prepared by general method C (using the alkyl iodide), using <b>32c</b> as a starting material, to afford a white powder, 26%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.43 (d, <i>J</i> = 2.3 Hz, 1H), 8.23 (br s, 1H), 8.10 (dd, <i>J</i> = 2.2, 8.9 Hz, 1H), 7.75 (d, <i>J</i> = 8.9 Hz, 1H), 6.33 (s, 1H), 5.22 (s, 2H), 4.18 (s, 2H), 2.18 (s, 3H), 1.24 (t, <i>J</i> = 7.0 Hz, 3H), 1.09 (s, 3H), 0.60 (s, 2H), 0.41 (d, <i>J</i> = 2.0 Hz, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ = 166.3, 160.1, 160.0, 149.70, 142.3, 137.1, 133.7, 129.1, 115.6, 114.4, 104.3, 83.6, 39.7, 36.8, 31.8, 24.6, 14.1, 12.5, 11.4. HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>23</sub>N<sub>4</sub>O<sub>5</sub>S 419.1384, found 419.1373.</div></div><div id="sec4_1_122" class="NLM_sec NLM_sec_level_3"><div id="ac_i159" class="anchor-spacer"></div><h4 class="article-section__title" id="_i159"> <i>N</i>-(1-Methylcyclopropyl)-3-[(3-methylisoxazol-5-yl)methyl]-2,4-dioxo-1-prop-2-ynyl-quinazoline-6-sulfonamide (<b>35b</b>)</h4><div class="NLM_p last">Prepared by general method C (using the alkyl bromide), using <b>32c</b> as a starting material, to afford a white powder, 64%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.44 (d, <i>J</i> = 2.2 Hz, 1H), 8.17 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.74 (d, <i>J</i> = 8.9 Hz, 1H), 6.35 (s, 1H), 5.22 (s, 2H), 5.01 (d, <i>J</i> = 2.3 Hz, 2H), 3.46–3.40 (m, 1H), 2.18 (s, 3H), 1.09 (s, 3H), 0.64–0.55 (m, 2H), 0.46–0.36 (m, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 166.9, 159.9, 159.7, 149.3, 141.3, 137.6, 132.9, 126.7, 116.2, 115.1, 103.5, 77.8, 75.9, 37.2, 33.7, 30.9, 23.9, 13.0, 10.9. HRMS (ESI) <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>20</sub>H<sub>20</sub>N<sub>4</sub>O<sub>5</sub>SNa 451.1047, found 451.1033.</div></div><div id="sec4_1_123" class="NLM_sec NLM_sec_level_3"><div id="ac_i160" class="anchor-spacer"></div><h4 class="article-section__title" id="_i160"> 1-(Cyclopropylmethyl)-<i>N</i>-(1-methylcyclopropyl)-3-[(3-methylisoxazol-5-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>35c</b>)</h4><div class="NLM_p last">Prepared by general method C (using the alkyl bromide), using <b>32c</b> as a starting material, to afford a white powder, 60%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.44 (d, <i>J</i> = 2.3 Hz, 1H), 8.24 (s, 1H), 8.11 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.84 (d, <i>J</i> = 8.9 Hz, 1H), 6.32 (s, 1H), 5.23 (s, 2H), 4.09 (d, <i>J</i> = 6.9 Hz, 2H), 2.18 (s, 3H), 1.30–1.18 (m, 1H), 1.09 (s, 3H), 0.65–0.56 (m, 2H), 0.54–0.35 (m, 6H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 167.0, 160.1, 160.0, 150.0, 142.4, 137.0, 132.9, 126.7, 116.3, 114.7, 103.4, 47.7, 37.1, 30.9, 23.9, 13.0, 10.9, 9.3, 3.7. HRMS (ESI) <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>21</sub>H<sub>24</sub>N<sub>4</sub>O<sub>5</sub>SNa 467.1360, found 467.1349.</div></div><div id="sec4_1_124" class="NLM_sec NLM_sec_level_3"><div id="ac_i161" class="anchor-spacer"></div><h4 class="article-section__title" id="_i161"> <i>N</i>-(1-Methylcyclopropyl)-3-[(3-methylisoxazol-5-yl)methyl]-1-(oxetan-3-ylmethyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>35d</b>)</h4><div class="NLM_p last">Prepared by general method C (using the alkyl tosylate), using <b>32c</b> as a starting material, to afford a white powder, 42%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.42 (d, <i>J</i> = 2.3 Hz, 1H), 8.36 (s, 1H), 8.06 (dd, <i>J</i> = 2.4, 8.9 Hz, 1H), 7.76 (d, <i>J</i> = 9.0 Hz, 1H), 6.33 (s, 1H), 5.21 (s, 2H), 4.61 (dd, <i>J</i> = 6.1, 7.8 Hz, 2H), 4.53–4.42 (m, 4H), 3.45–3.39 (m, 1H), 2.18 (s, 3H), 1.09 (s, 3H), 0.60 (s, 2H), 0.43–0.38 (m, 2H). <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 167.0, 160.1, 159.7, 150.4, 142.3, 137.1, 132.7, 126.6, 116.0, 115.0, 103.4, 73.8, 45.7, 37.1, 33.3, 30.9, 24.0, 13.0, 10.9. HRMS (ESI) <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>21</sub>H<sub>24</sub>N<sub>4</sub>O<sub>6</sub>SNa: 483.1309, found 483.1296.</div></div><div id="sec4_1_125" class="NLM_sec NLM_sec_level_3"><div id="ac_i162" class="anchor-spacer"></div><h4 class="article-section__title" id="_i162"> 1-[(2,5-Dimethylpyrazol-3-yl)methyl]-<i>N</i>-(1-methylcyclopropyl)-3-[(3-methylisoxazol-5-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>35e</b>)</h4><div class="NLM_p last">Prepared by general method C (using the alkyl mesylate), using <b>32c</b> as a starting material, to afford a white powder, 39%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.44 (d, <i>J</i> = 2.3 Hz, 1H), 8.23 (s, 1H), 8.06 (dd, <i>J</i> = 2.3, 8.8 Hz, 1H), 7.60 (d, <i>J</i> = 8.9 Hz, 1H), 6.36 (s, 1H), 5.80 (s, 1H), 5.38 (s, 2H), 5.23 (s, 2H), 3.81 (s, 3H), 2.18 (s, 3H), 2.00 (s, 3H), 1.08 (s, 3H), 0.64–0.55 (m, 2H), 0.44–0.36 (m, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ = 166.0, 160.1, 159.8, 150.1, 147.8, 142.0, 137.9, 136.6, 133.7, 128.8, 115.6, 115.2, 105.4, 104.4, 39.7, 37.0, 36.7, 31.8, 24.6, 14.1, 13.3, 11.4. HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>27</sub>N<sub>6</sub>O<sub>5</sub>S 499.1758, found 499.1747.</div></div><div id="sec4_1_126" class="NLM_sec NLM_sec_level_3"><div id="ac_i163" class="anchor-spacer"></div><h4 class="article-section__title" id="_i163"> 3-(Cyanomethyl)-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-1-prop-2-ynyl-quinazoline-6-sulfonamide <b>36b</b></h4><div class="NLM_p last">Prepared by general method C (using the alkyl bromide), using <b>32d</b> as a starting material, to afford a white powder, 37%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ = 8.75 (d, <i>J</i> = 2.3 Hz, 1H), 8.25 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.55 (d, <i>J</i> = 8.9 Hz, 1H), 5.03–4.95 (m, 5H), 2.41 (t, <i>J</i> = 2.4 Hz, 1H), 1.28 (s, 3H), 0.83–0.72 (m, 2H), 0.59–0.50 (m, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ = 159.2, 148.9, 141.4, 138.3, 134.2, 128.9, 115.5, 115.3, 114.0, 75.7, 74.8, 34.2, 31.9, 29.3, 24.6, 14.1. HRMS (ESI) <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>17</sub>H<sub>16</sub>N<sub>4</sub>O<sub>4</sub>SNa 395.0784, found 395.0776.</div></div><div id="sec4_1_127" class="NLM_sec NLM_sec_level_3"><div id="ac_i164" class="anchor-spacer"></div><h4 class="article-section__title" id="_i164"> 3-(Cyanomethyl)-1-(cyclopropylmethyl)-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>36c</b>)</h4><div class="NLM_p last">Prepared by general method C (using the alkyl bromide), using <b>32d</b> as a starting material, to afford a white powder, 60%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.44 (d, <i>J</i> = 2.3 Hz, 1H), 8.25 (s, 1H), 8.11 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.85 (d, <i>J</i> = 9.0 Hz, 1H), 4.97 (s, 2H), 4.10 (d, <i>J</i> = 6.9 Hz, 2H), 2.09 (s, 1H), 1.09 (s, 3H), 0.92–0.78 (m, 1H), 0.65–0.56 (m, 2H), 0.56–0.44 (m, 4H), 0.44–0.36 (m, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ = 159.7, 149.7, 142.6, 137.6, 134.1, 129.0, 115.2, 115.1, 114.3, 48.9, 31.8, 29.2, 24.6, 14.1, 9.3, 4.2. HRMS (ESI) <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>18</sub>H<sub>20</sub>N<sub>4</sub>O<sub>4</sub>SNa 411.1097, found 411.1088.</div></div><div id="sec4_1_128" class="NLM_sec NLM_sec_level_3"><div id="ac_i165" class="anchor-spacer"></div><h4 class="article-section__title" id="_i165"> 3-(Cyanomethyl)-1-[(2,5-dimethylpyrazol-3-yl)methyl]-<i>N</i>-(1-methylcyclopropyl)-2,4-dioxo-quinazoline-6-sulfonamide (<b>36e</b>)</h4><div class="NLM_p last">Prepared by general method B, using <b>32d</b> as a starting material, to afford a white powder, 17%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.44 (d, <i>J</i> = 2.2 Hz, 1H), 8.25 (br s, 1H), 8.06 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.58 (d, <i>J</i> = 9.0 Hz, 1H), 5.84 (s, 1H), 5.39 (s, 2H), 4.98 (s, 2H), 3.83 (s, 3H), 2.00 (s, 3H), 1.08 (s, 3H), 0.63–0.55 (m, 2H), 0.43–0.37 (m, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ = 159.2, 149.4, 148.0, 141.9, 138.3, 136.0, 134.2, 129.0, 115.4, 115.2, 114.0, 105.4, 39.9, 36.7, 31.9, 29.3, 24.6, 14.1, 13.3. HRMS (ESI) <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>18</sub>H<sub>20</sub>N<sub>4</sub>O<sub>4</sub>SNa 465.1315, found 465.1305.</div></div><div id="sec4_1_129" class="NLM_sec NLM_sec_level_3"><div id="ac_i166" class="anchor-spacer"></div><h4 class="article-section__title" id="_i166"> 2-Amino-<i>N</i>-[(5-methyl-1,3,4-thiadiazol-2-yl)methyl]benzamide (<b>37</b>)</h4><div class="NLM_p last">(5-Methyl-1,3,4-thiadiazol-2-yl)methanamine <b>42</b> (2.4 g, 17 mmol) was added to a magnetically stirred solution of isatoic acid anhydride <b>29</b> (2.7 g, 17 mmol) and triethylamine (2.5 mL, 18 mmol) in DMF (20 mL) at 0 °C. After the addition, the resulting mixture was heated to 60 °C for 5 h and then stirred at RT for 16 h. The solvent was removed in vacuo to give a light-brown solid, which was preabsorbed onto silica and columned, eluting with 0–5% MeOH/DCM to afford <b>37</b> (3.0 g, 65%) as a light-brown solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 9.12 (t, <i>J</i> = 5.8 Hz, 1H), 7.51 (dd, <i>J</i> = 1.4, 8.0 Hz, 1H), 7.16 (ddd, <i>J</i> = 1.5, 7.0, 8.3 Hz, 1H), 6.71 (dd, <i>J</i> = 1.1, 8.2 Hz, 1H), 6.59–6.43 (m, 3H), 4.72 (d, <i>J</i> = 5.9 Hz, 2H), 2.67 (s, 3H).</div></div><div id="sec4_1_130" class="NLM_sec NLM_sec_level_3"><div id="ac_i167" class="anchor-spacer"></div><h4 class="article-section__title" id="_i167"> 3-[(5-Methyl-1,3,4-thiadiazol-2-yl)methyl]-1<i>H</i>-quinazoline-2,4-dione (<b>38</b>)</h4><div class="NLM_p last">A solution of <b>37</b> (3.0 g, 11 mmol) in THF (100 mL) was cooled to 0 °C and treated with triphosgene (1.6 g, 5.4 mmol), causing precipitate formation. The mixture was stirred in the cool bath for 15 min and then stirred at RT for 18 h. The mixture was quenched with satd aq K<sub>2</sub>CO<sub>3</sub> (50 mL) and stirred at RT for 24 h. The resulting precipitate was collected by filtration, washed with water, and oven-dried to afford <b>24</b> (1.7 g, 58%) as an off-white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 11.80 (br s, 1H), 7.96 (d, <i>J</i> = 7.8 Hz, 1H), 7.69 (dt, <i>J</i> = 1.5, 7.7 Hz, 1H), 7.28–7.16 (m, 2H), 5.44 (s, 2H), 2.67 (s, 3H).</div></div><div id="sec4_1_131" class="NLM_sec NLM_sec_level_3"><div id="ac_i168" class="anchor-spacer"></div><h4 class="article-section__title" id="_i168"> 3-[(5-Methyl-1,3,4-thiadiazol-2-yl)methyl]-2,4-dioxo-1<i>H</i>-quinazoline-6-sulfonyl chloride (<b>39</b>)</h4><div class="NLM_p last">A solution of <b>38</b> (1.7 g, 6.2 mmol) in chlorosulfonic acid (5 mL, 6.2 mmol) was heated to 50 °C for 2 h and then allowed to cool to RT. The reaction mixture was then added dropwise to crushed ice (∼20 mL) at −10 °C and stirred for 20 min. The precipitated solid was collected by filtration, washed with water, and oven-dried to give a sticky solid. This was azeotroped with toluene (×3) to give the desired product (1.6 g, 69%) as a pale-yellow solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 11.76 (s, 1H), 8.16 (d, <i>J</i> = 2.0 Hz, 1H), 7.88 (dd, <i>J</i> = 2.0, 8.4 Hz, 1H), 7.17 (d, <i>J</i> = 8.5 Hz, 1H), 5.44 (s, 2H), 2.67 (s, 3H).</div></div><div id="sec4_1_132" class="NLM_sec NLM_sec_level_3"><div id="ac_i169" class="anchor-spacer"></div><h4 class="article-section__title" id="_i169"> <i>N</i>-(1-Methylcyclopropyl)-3-[(5-methyl-1,3,4-thiadiazol-2-yl)methyl]-2,4-dioxo-1<i>H</i>-quinazoline-6-sulfonamide (<b>40</b>)</h4><div class="NLM_p last">A mixture of <b>39</b> (411 mg, 1.1 mmol) and 1-methylcyclopropanamine hydrochloride (130 mg, 1.2 mmol) in DCM (20 mL) was cooled in an ice bath and treated with triethylamine (0.34 mL, 2.4 mmol). The reaction mixture was allowed to warm to RT and was stirred for 18 h. The reaction mixture was then concentrated, preabsorbed onto silica, and columned, eluting with 0–5% MeOH in DCM afford <b>40</b> (142 mg, 32%) as a white powder. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 12.07 (s, 1H), 8.33 (s, 1H), 8.16 (s, 1H), 8.04 (br d, <i>J</i> = 8.7 Hz, 1H), 7.37 (d, <i>J</i> = 8.6 Hz, 1H), 5.44 (s, 2H), 2.68 (s, 3H), 1.07 (s, 3H), 0.63–0.55 (m, 2H), 0.43–0.34 (m, 2H).</div></div><div id="sec4_1_133" class="NLM_sec NLM_sec_level_3"><div id="ac_i170" class="anchor-spacer"></div><h4 class="article-section__title" id="_i170"> 1-Ethyl-<i>N</i>-(1-methylcyclopropyl)-3-[(5-methyl-1,3,4-thiadiazol-2-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>41a</b>)</h4><div class="NLM_p last">Prepared by general method C (using the alkyl iodide), using <b>40</b> as a starting material, to afford a white powder, 20%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.44 (d, <i>J</i> = 2.3 Hz, 1H), 8.19 (br s, 1H), 8.11 (dd, <i>J</i> = 2.3, 8.9 Hz, 1H), 7.76 (d, <i>J</i> = 8.9 Hz, 1H), 5.50 (s, 2H), 4.20 (q, <i>J</i> = 7.1 Hz, 2H), 2.68 (s, 3H), 1.24 (t, <i>J</i> = 7.0 Hz, 3H), 1.08 (s, 3H), 0.63–0.56 (m, 2H), 0.43–0.37 (m, 2H). HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>22</sub>N<sub>5</sub>O<sub>4</sub>S<sub>2</sub> 436.1108, found 436.1096.</div></div><div id="sec4_1_134" class="NLM_sec NLM_sec_level_3"><div id="ac_i171" class="anchor-spacer"></div><h4 class="article-section__title" id="_i171"> 1-[(2,5-Dimethylpyrazol-3-yl)methyl]-<i>N</i>-(1-methylcyclopropyl)-3-[(5-methyl-1,3,4-thiadiazol-2-yl)methyl]-2,4-dioxo-quinazoline-6-sulfonamide (<b>41e</b>)</h4><div class="NLM_p last">Prepared by general method C (using the alkyl chloride), using <b>40</b> as a starting material, to afford a white powder, 44%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.45 (d, <i>J</i> = 2.3 Hz, 1H), 8.25 (s, 1H), 8.07 (dd, <i>J</i> = 2.4, 8.9 Hz, 1H), 7.61 (d, <i>J</i> = 8.9 Hz, 1H), 5.79 (s, 1H), 5.52 (s, 2H), 5.40 (s, 2H), 3.82 (s, 3H), 2.68 (s, 3H), 2.00 (s, 3H), 1.08 (s, 3H), 0.63–0.56 (m, 2H), 0.42–0.37 (m, 2H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>) δ = 166.6, 163.7, 159.8, 150.1, 147.5, 142.0, 138.1, 133.8, 129.0, 124.6, 115.7, 115.1, 105.6, 40.2, 39.7, 36.6, 31.9, 24.7, 15.6, 14.1, 13.0. HRMS (ESI) <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>26</sub>N<sub>7</sub>O<sub>4</sub>S<sub>2</sub> 516.1482, found 516.1470.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i172" class="anchor-spacer"></div><h3 class="article-section__title" id="_i172"> PARG Biochemical Assay</h3><div class="NLM_p last">This assay was conducted according to the published protocol.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Briefly, PARG in vitro assays were conducted in a total volume of 15 μL in a standard 384-well format. Then 5 μL of human full-length PARG (AstraZeneca),<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> used at a final reaction concentration of 65 pM, was added to 5 μL of Bt-NAD ribosylated PARP1 substrate (AstraZeneca) at a final reaction concentration of 4.8 nM in assay buffer (50 mM Tris pH 7.4, 0.1 mg/mL BSA, 3 mM EDTA, 0.4 mM EGTA, 1 mM DTT, 0.01% Tween 20, 50 mM KCl). The reaction was incubated at room temperature for 10 min, and then 5 μL of detection reagent was added. Detection reagent consists of 42 nM mAb anti-6HIS XL665 (CisBio: 61HISXLB) and 2.25 nM streptavidin europium cryptate (CisBio: 610SAKLB), both at 3× working stock concentrations (final concentrations of 14 nM and 0.75 nM respectively), in detection buffer (50 mM Tris pH 7.4, 0.1 mg/mL BSA, and 100 mM KF). Following incubation at room temperature for 60 min in the dark, TR-FRET signal was measured at λEx 340 nm and λEm 665 nm and λEm 620 nm using a PHERAstar FS plate reader (BMG Labtech). The ratio was calculated as [Em665/EM620] × 10<sup>4</sup> for each well and used to calculate percent inhibition for test compounds.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i173" class="anchor-spacer"></div><h3 class="article-section__title" id="_i173"> Cellular PAR Chain Assay</h3><div class="NLM_p">This assay was carried out according to the published protocol<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Briefly, HeLa cells were seeded into 384-well plates and, 16–24 h later, were treated with inhibitors (8 pt dose response, 0.01–30 μM) or vehicle (DMSO) control. After 1 h, cells were codosed with MMS at 50 μg/mL (final) and incubated for 1 h at 37 °C. Cells were fixed with ice-cold 95% methanol/phosphate buffered saline (PBS) for 15 min at −20 °C and then washed once with PBS at room temperature. Cells were then permeabilized using PBS/Triton 0.1% for 20 min and washed once in PBS before incubating with anti-PAR antibody (10H, Merck-Millipore) overnight at 4 °C. Cells were then incubated with antimouse Alexofluor 488-conjugated secondary antibody (A11029, ThermoFisher) at 1:1000 and Hoechst 33342 (1:5000), and images were captured on a CellInsight (ThermoFisher Scientific, Northumberland, UK) and analyzed using proprietary software. The mean of the intensity of nuclear spots at 488 nM was reported with 25 fields captured from duplicate wells and approximately 1100 cells analyzed per well.</div><div class="NLM_p last">For details of the enzyme assay–cell assay correlation, see Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01407/suppl_file/jm8b01407_si_001.pdf" class="ext-link">Figure S1</a>.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i174" class="anchor-spacer"></div><h3 class="article-section__title" id="_i174"> Cytotoxicity Assay</h3><div class="NLM_p last">HeLa cells were seeded in 30 μL media at 1 × 10<sup>4</sup> cells/mL in Greiner (no. 781091) 384-well plates. Then 16–24 h later, cells were treated with inhibitors (8 point dose response, 0.01–30 μM, triplicates) or vehicle (DMSO) control. The outer wells were left undosed to account for edge effects. After 72 h, 50 μL of 3.7% formaldehyde/PBS was added to each well and cells were fixed for 20 min. Cells were then rinsed twice with PBS and stained for 1 h with Hoechst 33342/PBS (1:2000) in the dark. After two further rinses with PBS, images were captured and nuclei counted on a CellInsight (ThermoFisher). The maximum number of fields (25) were captured from each triplicate well which approximated to at least 1000 nuclei in vehicle-dosed wells.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i175" class="anchor-spacer"></div><h3 class="article-section__title" id="_i175"> Cytotoxicity and Selectivity</h3><div class="NLM_p last">See ref <a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> for methods to measure PARP1 selectivity and ARH3 selectivity.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i176" class="anchor-spacer"></div><h3 class="article-section__title" id="_i176"> Pharmacokinetics</h3><div class="NLM_p">All studies were conducted after review by the Animal Welfare and Ethical Review Body at CRUK-MI and in accordance with the University of Manchester Policy on the use of animals in research. All work was carried out in compliance with the Animals (Scientific Procedures) Act 1986 and EU Directive 2010/63/EU.</div><div class="NLM_p last">Pharmacokinetics were studied in male CD-1 mice following single intravenous or oral administration. Compounds were formulated as solutions comprising up to 10% DMSO with up to 10% Tween80 in saline at a concentration of approximately 1 mM. Doses were administered at 1 mg/kg (IV) or 5 mg/kg (oral). Blood samples were collected as dried blood spots and assayed following solvent extraction through a phospholipid removal plate followed by LC–MS/MS analysis. The time-points for blood collection were as follows: oral (0.5, 1, 3, 5, and 24 h), IV (0.1, 0.25, 0.75, 2, 5, and 24 h). The resulting concentration–time data were analyzed by noncompartmental methods (PK Solver, Excel Add-In).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i177" class="anchor-spacer"></div><h3 class="article-section__title" id="_i177"> Structure-Based Virtual screening and Library Design</h3><div class="NLM_p">The X-ray crystal structure of the anthraquinone <b>8a</b> in complex with the human PARG catalytic domain (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMM">6HMM</a>) was the basis for virtual screening and protein–ligand docking studies using the Schrödinger modeling suite (Schrödinger, LLC, New York, NY, 2012–2015 releases). The Maestro Protein Preparation Wizard was used for initial processing of the X-ray crystal structure, applying default settings for hydrogen addition, optimization of hydrogen bonding networks and constrained geometry refinement, followed by visual assessment of the final protein model. Crystallographic solvent molecules were removed prior to generation of a receptor grid file for use in Glide docking. A grid box was defined based on the crystallographic pose of <b>8a</b>, extending the inner grid box to 14 Å in the direction of the PARG ribose-binding site so as to encompass the bound pose of ADP-ribose and hence to facilitate docking of ligands larger than the relatively compact anthraquinone <b>8a</b>. Hydrogen bonding constraint sites were defined as the protein atoms interacting with the sulfonamide moiety of <b>8a</b> (Glu727 carboxylate, Gln754 side chain NH<sub>2</sub>, Ile726 backbone <i>NH</i>).</div><div class="NLM_p">For the initial virtual screen, a set of commercially available aryl sulfonamides was generated by searching an in-house compilation of individual supplier catalogues, filtered for general drug-like property profiles.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Prompted by the steep SAR, we had observed around the sulfonamide moiety of <b>8a</b>, the substructure search was limited to aryl sulfonamides bearing a small <i>N</i>-alkyl substituent, as represented by the SMILES string cS(=O)(=O)[NH]CC. After removal of duplicates and checking of physicochemical property profiles using Canvas, a set of approximately 30000 compounds was prepared for docking using LigPrep and subsequently docked to the PARG protein grid using Glide SP, using default docking parameters other than inclusion of at least one of the H-bonding constraints around the sulfonamide moiety. From the docked poses, a selection was made on the basis of docking score, visually plausible binding interactions, lead-like property profile, and chemical diversity.</div><div class="NLM_p last">To extend the diversity of aryl sulfonamides for virtual screening, several rounds of virtual library building and docking were pursued based on lists of commercially available aryl sulfonyl chlorides, initially obtained by searching our internal vendor catalogue and subsequently the eMolecules Building Blocks collection (<a href="https://www.emolecules.com" class="extLink">https://www.emolecules.com</a>, accessed Oct 24, 2018). These lists were the basis for virtual library enumeration with favored <i>N</i>-alkyl substituents (typically methylcyclopropyl) using PipelinePilot (BIOVIA, San Diego, CA), followed by processing in LigPrep and docking in Glide as before. Ligand interaction diagrams in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>, <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>, and <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a> were generated within Maestro and images of protein–ligand complexes within the PyMOL Molecular Graphics System (Schrödinger, LLC, New York, NY).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i178"><a href="/doi/suppl/10.1021/acs.jmedchem.8b01407" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03081" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03081" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b01407" class="ext-link">10.1021/acs.jmedchem.8b01407</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">LC–MS methods and solvent gradients; preparative HPLC instrument and solvent gradients; summary of purity data; additional core-hops identified from virtual screening; preparative methods and spectroscopic data for key intermediates and compounds <b>S1</b>–<b>S17</b>; exemplar NMR and LC data; PARG cocrystal structure determination and refinement details for <b>8a</b>, <b>22h</b>, <b>27t</b>, and <b>33f</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01407/suppl_file/jm8b01407_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01407/suppl_file/jm8b01407_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01407/suppl_file/jm8b01407_si_001.pdf">jm8b01407_si_001.pdf (907.04 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01407/suppl_file/jm8b01407_si_002.csv">jm8b01407_si_002.csv (7.95 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and structure factors for X-ray crystal structures of <b>8a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMM">6HMM</a>), <b>22h</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMN">6HMN</a>), <b>27t</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMK">6HMK</a>), and <b>33f</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HML">6HML</a>) in complex with the human PARG catalytic domain have been deposited in the Protein Data Bank.</p><div class="testing" data-doi="10.1021/acs.jmedchem.8b01407" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67301" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67301" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Allan M. Jordan</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Manchester Institute, The University
of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3449-3993" title="Orcid link">http://orcid.org/0000-0003-3449-3993</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#aacbc6c6cbc484c0c5d8cecbc4eac9d8dfc184c7cbc4c9c2cfd9decfd884cbc984dfc1"><span class="__cf_email__" data-cfemail="06676a6a6768286c6974626768466574736d286b6768656e637572637428676528736d">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bohdan Waszkowycz</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Manchester Institute, The University
of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8572-2018" title="Orcid link">http://orcid.org/0000-0001-8572-2018</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kate M. Smith</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Manchester Institute, The University
of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8940-2804" title="Orcid link">http://orcid.org/0000-0001-8940-2804</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alison E. McGonagle</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Manchester Institute, The University
of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ben Acton</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Manchester Institute, The University
of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emma E. Fairweather</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Manchester Institute, The University
of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Louise A. Griffiths</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Manchester Institute, The University
of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Niall M. Hamilton</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Manchester Institute, The University
of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicola S. Hamilton</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Manchester Institute, The University
of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James R. Hitchin</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Manchester Institute, The University
of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Colin P. Hutton</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Manchester Institute, The University
of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dominic I. James</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Manchester Institute, The University
of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Clifford D. Jones</span> - <span class="hlFld-Affiliation affiliation">Oncology
Innovative Medicines Unit, AstraZeneca, Alderley Park, Macclesfield Cheshire SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stuart Jones</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Manchester Institute, The University
of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel P. Mould</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Manchester Institute, The University
of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6501-4938" title="Orcid link">http://orcid.org/0000-0001-6501-4938</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Helen F. Small</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Manchester Institute, The University
of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alexandra I. J. Stowell</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Manchester Institute, The University
of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Julie A. Tucker</span> - <span class="hlFld-Affiliation affiliation">Structure
and Biophysics, Discovery Sciences, AstraZeneca, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span>; 
    <span>Present Address:
                        For J.A.T.: Centre for Immunology and Infection, Department of Biology and HYMS, University of York, Heslington, York, YO10 5DD, UK</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ian D. Waddell</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Manchester Institute, The University
of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Donald J. Ogilvie</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Manchester Institute, The University
of Manchester, Alderley Park, Maccelsfield SK10 4TG, U.K.</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>B.W., K.M.S., A.McG. and A.M.J. contributed equally to the content and preparation of this manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i182">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71458" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71458" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by Cancer Research UK (grant C480/A11411 and C5759/A17098). We thank AstraZeneca for triaging HTS output and Mentxu Aiertza and Samantha Hitchin for assistance with compound logistics. We thank Ivan Ahel for helpful discussions regarding PARG biology. HRMS spectra were generated by Gareth Smith, School of Chemistry, University of Manchester, M13 9PL Manchester, UK. log <i>D</i> and turbidimetric solubility were provided by Cyprotex Discovery, Macclesfield, UK. We thank Giles Hassall (AstraZeneca) for expression and purification of the human PARG catalytic domain, Claire Brassington (AstraZeneca) for assistance with crystallization, Diamond Light Source for beamtime, and the AstraZeneca Alderley Park Protein Structure team and the staff of beamline I04-1 for assistance with crystal testing and X-ray diffraction data collection. JChem for Excel was used for structure property prediction and calculation and general data handling (JChem for Excel, version 6.1.5.781, 2008–2013, ChemAxon (<a href="http://www.chemaxon.com" class="extLink">http://www.chemaxon.com</a>). We thank Rae Lawrence for help in reformatting the graphics in this manuscript.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screen</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i184">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72875" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72875" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 41 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">The DNA Damage Response and Cancer Therapy</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>481</i></span>,  <span class="NLM_fpage">287</span>– <span class="NLM_lpage">294</span>, <span class="refDoi"> DOI: 10.1038/nature10760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1038%2Fnature10760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=22258607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC38Xps1GmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=481&publication_year=2012&pages=287-294&author=C.+J.+Lordauthor=A.+Ashworth&title=The+DNA+Damage+Response+and+Cancer+Therapy&doi=10.1038%2Fnature10760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The DNA damage response and cancer therapy</span></div><div class="casAuthors">Lord, Christopher J.; Ashworth, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">481</span>
        (<span class="NLM_cas:issue">7381</span>),
    <span class="NLM_cas:pages">287-294</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Genomic instability is one of the most pervasive characteristics of tumor cells and is probably the combined effect of DNA damage, tumor-specific DNA repair defects, and a failure to stop or stall the cell cycle before the damaged DNA is passed on to daughter cells.  Although these processes drive genomic instability and ultimately the disease process, they also provide therapeutic opportunities.  A better understanding of the cellular response to DNA damage will not only inform our knowledge of cancer development but also help to refine the classification as well as the treatment of the disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUS1sAbqFhmrVg90H21EOLACvtfcHk0liBe3ieZiG56Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xps1GmsQ%253D%253D&md5=75d12cdb52e8c9a39bced1611bf90298</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature10760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10760%26sid%3Dliteratum%253Aachs%26aulast%3DLord%26aufirst%3DC.%2BJ.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DThe%2520DNA%2520Damage%2520Response%2520and%2520Cancer%2520Therapy%26jtitle%3DNature%26date%3D2012%26volume%3D481%26spage%3D287%26epage%3D294%26doi%3D10.1038%2Fnature10760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hosoya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyagawa, K.</span></span> <span> </span><span class="NLM_article-title">Targeting DNA Damage Response in Cancer Therapy</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">370</span>– <span class="NLM_lpage">388</span>, <span class="refDoi"> DOI: 10.1111/cas.12366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1111%2Fcas.12366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=24484288" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlvFWitb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2014&pages=370-388&author=N.+Hosoyaauthor=K.+Miyagawa&title=Targeting+DNA+Damage+Response+in+Cancer+Therapy&doi=10.1111%2Fcas.12366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting DNA damage response in cancer therapy</span></div><div class="casAuthors">Hosoya, Noriko; Miyagawa, Kiyoshi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">370-388</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Cancer chemotherapy and radiotherapy are designed to kill cancer cells mostly by inducing DNA damage.  DNA damage is normally recognized and repaired by the intrinsic DNA damage response machinery.  If the damaged lesions are successfully repaired, the cells will survive.  In order to specifically and effectively kill cancer cells by therapies that induce DNA damage, it is important to take advantage of specific abnormalities in the DNA damage response machinery that are present in cancer cells but not in normal cells.  Such properties of cancer cells can provide biomarkers or targets for sensitization.  For example, defects or upregulation of the specific pathways that recognize or repair specific types of DNA damage can serve as biomarkers of favorable or poor response to therapies that induce such types of DNA damage.  Inhibition of a DNA damage response pathway may enhance the therapeutic effects in combination with the DNA-damaging agents.  Moreover, it may also be useful as a monotherapy when it achieves synthetic lethality, in which inhibition of a complementary DNA damage response pathway selectively kills cancer cells that have a defect in a particular DNA repair pathway.  The most striking application of this strategy is the treatment of cancers deficient in homologous recombination by poly(ADP-ribose) polymerase inhibitors.  In this review, we describe the impact of targeting the cancer-specific aberrations in the DNA damage response by explaining how these treatment strategies are currently being evaluated in preclin. or clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNrms7_IuKabVg90H21EOLACvtfcHk0ljuArWHQz48NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlvFWitb0%253D&md5=e5484fa4a619052a621c999349ec9b83</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1111%2Fcas.12366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcas.12366%26sid%3Dliteratum%253Aachs%26aulast%3DHosoya%26aufirst%3DN.%26aulast%3DMiyagawa%26aufirst%3DK.%26atitle%3DTargeting%2520DNA%2520Damage%2520Response%2520in%2520Cancer%2520Therapy%26jtitle%3DCancer%2520Sci.%26date%3D2014%26volume%3D105%26spage%3D370%26epage%3D388%26doi%3D10.1111%2Fcas.12366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, M. J.</span></span> <span> </span><span class="NLM_article-title">Targeting the DNA Damage Response in Cancer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.10.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1016%2Fj.molcel.2015.10.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=26590714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyqtbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2015&pages=547-560&author=M.+J.+O%E2%80%99Connor&title=Targeting+the+DNA+Damage+Response+in+Cancer&doi=10.1016%2Fj.molcel.2015.10.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the DNA Damage Response in Cancer</span></div><div class="casAuthors">O'Connor, Mark J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">547-560</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">An underlying hallmark of cancers is their genomic instability, which is assocd. with a greater propensity to accumulate DNA damage.  Historical treatment of cancer by radiotherapy and DNA-damaging chemotherapy is based on this principle, yet it is accompanied by significant collateral damage to normal tissue and unwanted side effects.  Targeted therapy based on inhibiting the DNA damage response (DDR) in cancers offers the potential for a greater therapeutic window by tailoring treatment to patients with tumors lacking specific DDR functions.  The recent approval of olaparib (Lynparza), the poly (ADP-ribose) polymerase (PARP) inhibitor for treating tumors harboring BRCA1 or BRCA2 mutations, represents the first medicine based on this principle, exploiting an underlying cause of tumor formation that also represents an Achilles' heel.  This review highlights the different concepts behind targeting DDR in cancer and how this can provide significant opportunities for DDR-based therapies in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWZ64MrshabrVg90H21EOLACvtfcHk0ljuArWHQz48NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyqtbvI&md5=b3f0d737d3e7dbf1f404b3f7658ab271</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.10.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.10.040%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DM.%2BJ.%26atitle%3DTargeting%2520the%2520DNA%2520Damage%2520Response%2520in%2520Cancer%26jtitle%3DMol.%2520Cell%26date%3D2015%26volume%3D60%26spage%3D547%26epage%3D560%26doi%3D10.1016%2Fj.molcel.2015.10.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mortusewicz, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fouquerel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amé, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonhardt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, V.</span></span> <span> </span><span class="NLM_article-title">PARG is Recruited to DNA Damage Sites Through Poly(ADP-ribose)- and PCNA-dependent Mechanisms</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">5045</span>– <span class="NLM_lpage">5056</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1093%2Fnar%2Fgkr099" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=5045-5056&author=O.+Mortusewiczauthor=E.+Fouquerelauthor=J.-C.+Am%C3%A9author=H.+Leonhardtauthor=V.+Schreiber&title=PARG+is+Recruited+to+DNA+Damage+Sites+Through+Poly%28ADP-ribose%29-+and+PCNA-dependent+Mechanisms&doi=10.1093%2Fnar%2Fgkr099"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr099%26sid%3Dliteratum%253Aachs%26aulast%3DMortusewicz%26aufirst%3DO.%26aulast%3DFouquerel%26aufirst%3DE.%26aulast%3DAm%25C3%25A9%26aufirst%3DJ.-C.%26aulast%3DLeonhardt%26aufirst%3DH.%26aulast%3DSchreiber%26aufirst%3DV.%26atitle%3DPARG%2520is%2520Recruited%2520to%2520DNA%2520Damage%2520Sites%2520Through%2520Poly%2528ADP-ribose%2529-%2520and%2520PCNA-dependent%2520Mechanisms%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3D5045%26epage%3D5056%26doi%3D10.1093%2Fnar%2Fgkr099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barkauskaite, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jankevicius, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladurner, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timinszky, G.</span></span> <span> </span><span class="NLM_article-title">The Recognition and Removal of Cellular Poly(ADP-ribose) Signals</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">3491</span>– <span class="NLM_lpage">3507</span>, <span class="refDoi"> DOI: 10.1111/febs.12358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1111%2Ffebs.12358" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2013&pages=3491-3507&author=E.+Barkauskaiteauthor=G.+Jankeviciusauthor=A.+G.+Ladurnerauthor=I.+Ahelauthor=G.+Timinszky&title=The+Recognition+and+Removal+of+Cellular+Poly%28ADP-ribose%29+Signals&doi=10.1111%2Ffebs.12358"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1111%2Ffebs.12358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.12358%26sid%3Dliteratum%253Aachs%26aulast%3DBarkauskaite%26aufirst%3DE.%26aulast%3DJankevicius%26aufirst%3DG.%26aulast%3DLadurner%26aufirst%3DA.%2BG.%26aulast%3DAhel%26aufirst%3DI.%26aulast%3DTiminszky%26aufirst%3DG.%26atitle%3DThe%2520Recognition%2520and%2520Removal%2520of%2520Cellular%2520Poly%2528ADP-ribose%2529%2520Signals%26jtitle%3DFEBS%2520J.%26date%3D2013%26volume%3D280%26spage%3D3491%26epage%3D3507%26doi%3D10.1111%2Ffebs.12358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span>; <span class="NLM_string-name">Koh, D.
W.</span></span> <span> </span><span class="NLM_article-title">Roles of Poly(ADP-Ribose) Glycohydrolase in DNA Damage and Apoptosis</span>.  <i>International Review of Cell and Molecular Biology</i>, <span class="NLM_contrib-group"><span class="NLM_string-name">Kwang, W. J.</span></span>, Ed. <span class="NLM_publisher-name">Academic Press</span>, <span class="NLM_year">2013</span>; Vol.  <span class="NLM_volume">304</span>, Chapter 5, pp  <span class="NLM_fpage">227</span>– <span class="NLM_lpage">281</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=227-281&author=X.+Feng&author=D.%0AW.+Kohauthor=W.+J.+Kwang&title=International+Review+of+Cell+and+Molecular+Biology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DX.%26atitle%3DRoles%2520of%2520Poly%2528ADP-Ribose%2529%2520Glycohydrolase%2520in%2520DNA%2520Damage%2520and%2520Apoptosis%26btitle%3DInternational%2520Review%2520of%2520Cell%2520and%2520Molecular%2520Biology%26aulast%3DKwang%26aufirst%3DW.%2BJ.%26pub%3DAcademic%2520Press%26date%3D2013%26volume%3D304%26spage%3D227%26epage%3D281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beumer, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamieh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tawbi, H.</span></span> <span> </span><span class="NLM_article-title">Role of PARP Inhibitors in Cancer Biology and Therapy</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">3907</span>– <span class="NLM_lpage">3921</span>, <span class="refDoi"> DOI: 10.2174/092986712802002464</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.2174%2F092986712802002464" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=22788767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht12nu73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=3907-3921&author=D.+Davarauthor=J.+H.+Beumerauthor=L.+Hamiehauthor=H.+Tawbi&title=Role+of+PARP+Inhibitors+in+Cancer+Biology+and+Therapy&doi=10.2174%2F092986712802002464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Role of PARP inhibitors in cancer biology and therapy</span></div><div class="casAuthors">Davar, D.; Beumer, J. H.; Hamieh, L.; Tawbi, H.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3907-3921</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Deeper understanding of DNA repair mechanisms and their potential value as therapeutic targets in oncol. heralded the clin. development of poly(ADP-ribose) polymerase (PARP) inhibitors.  Although initially developed to exploit synthetic lethality in models of cancer assocd. with defective DNA repair, our burgeoning knowledge of PARP biol. has resulted in these agents being exploited both in cancer with select chemotherapeutic agents and in non-malignant diseases.  In this review article, we briefly review the mechanisms of DNA repair and pre-clin. development of PARP inhibitors before discussing the clin. development of the various PARP inhibitors in depth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRQaaaS9jHi7Vg90H21EOLACvtfcHk0lh8HURjdhWOyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht12nu73I&md5=70350a0d3159a740ca4ae3dfe6b5db25</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2174%2F092986712802002464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986712802002464%26sid%3Dliteratum%253Aachs%26aulast%3DDavar%26aufirst%3DD.%26aulast%3DBeumer%26aufirst%3DJ.%2BH.%26aulast%3DHamieh%26aufirst%3DL.%26aulast%3DTawbi%26aufirst%3DH.%26atitle%3DRole%2520of%2520PARP%2520Inhibitors%2520in%2520Cancer%2520Biology%2520and%2520Therapy%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2012%26volume%3D19%26spage%3D3907%26epage%3D3921%26doi%3D10.2174%2F092986712802002464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ricks, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ison, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKee, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kluetz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazdur, R.</span></span> <span> </span><span class="NLM_article-title">Successes and Challenges of PARP Inhibitors in Cancer Therapy</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">222</span>, <span class="refDoi"> DOI: 10.3389/fonc.2015.00222</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.3389%2Ffonc.2015.00222" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=26528434" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A280%3ADC%252BC28zot1ykug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=222&author=T.+K.+Ricksauthor=H.-J.+Chiuauthor=G.+Isonauthor=G.+Kimauthor=A.+E.+McKeeauthor=P.+Kluetzauthor=R.+Pazdur&title=Successes+and+Challenges+of+PARP+Inhibitors+in+Cancer+Therapy&doi=10.3389%2Ffonc.2015.00222"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Successes and Challenges of PARP Inhibitors in Cancer Therapy</span></div><div class="casAuthors">Ricks Tiffany K; Chiu Haw-Jyh; Ison Gwynn; Kim Geoffrey; McKee Amy E; Kluetz Paul; Pazdur Richard</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">222</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQxHehOgh33Mntlzo9xastyfW6udTcc2ea88V18uRBx0Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28zot1ykug%253D%253D&md5=66046870d9d787d55c8806292c24764a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2015.00222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2015.00222%26sid%3Dliteratum%253Aachs%26aulast%3DRicks%26aufirst%3DT.%2BK.%26aulast%3DChiu%26aufirst%3DH.-J.%26aulast%3DIson%26aufirst%3DG.%26aulast%3DKim%26aufirst%3DG.%26aulast%3DMcKee%26aufirst%3DA.%2BE.%26aulast%3DKluetz%26aufirst%3DP.%26aulast%3DPazdur%26aufirst%3DR.%26atitle%3DSuccesses%2520and%2520Challenges%2520of%2520PARP%2520Inhibitors%2520in%2520Cancer%2520Therapy%26jtitle%3DFront.%2520Oncol.%26date%3D2015%26volume%3D5%26spage%3D222%26doi%3D10.3389%2Ffonc.2015.00222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frampton, J. E.</span></span> <span> </span><span class="NLM_article-title">Olaparib: A Review of Its Use as Maintenance Therapy in Patients with Ovarian Cancer</span>. <i>BioDrugs</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">150</span>, <span class="refDoi"> DOI: 10.1007/s40259-015-0125-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1007%2Fs40259-015-0125-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=143-150&author=J.+E.+Frampton&title=Olaparib%3A+A+Review+of+Its+Use+as+Maintenance+Therapy+in+Patients+with+Ovarian+Cancer&doi=10.1007%2Fs40259-015-0125-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2Fs40259-015-0125-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40259-015-0125-6%26sid%3Dliteratum%253Aachs%26aulast%3DFrampton%26aufirst%3DJ.%2BE.%26atitle%3DOlaparib%253A%2520A%2520Review%2520of%2520Its%2520Use%2520as%2520Maintenance%2520Therapy%2520in%2520Patients%2520with%2520Ovarian%2520Cancer%26jtitle%3DBioDrugs%26date%3D2015%26volume%3D29%26spage%3D143%26epage%3D150%26doi%3D10.1007%2Fs40259-015-0125-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fauzee, N. J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-l.</span></span> <span> </span><span class="NLM_article-title">PARP and PARG Inhibitors-New Therapeutic Targets in Cancer Treatment</span>. <i>Pathol. Oncol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">469</span>– <span class="NLM_lpage">478</span>, <span class="refDoi"> DOI: 10.1007/s12253-010-9266-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1007%2Fs12253-010-9266-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=20383759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlSjt7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=469-478&author=N.+J.+S.+Fauzeeauthor=J.+Panauthor=Y.-l.+Wang&title=PARP+and+PARG+Inhibitors-New+Therapeutic+Targets+in+Cancer+Treatment&doi=10.1007%2Fs12253-010-9266-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">PARP and PARG Inhibitors-New Therapeutic Targets in Cancer Treatment</span></div><div class="casAuthors">Fauzee, Nilufer Jasmine Selimah; Pan, Juan; Wang, Ya-lan</div><div class="citationInfo"><span class="NLM_cas:title">Pathology & Oncology Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">469-478</span>CODEN:
                <span class="NLM_cas:coden">POREFR</span>;
        ISSN:<span class="NLM_cas:issn">1219-4956</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Today, the no. of cancer patients throughout the world is increasing alarmingly and as per the World Health Organization (WHO) data and statistics the prediction for the year 2020 will be 15 million new cases as compared to only 10 million cases in year 2000 leaving us dumbfounded.  A lot of effort has been put in by researchers and scientists over decades to find drugs helpful in the treatment of cancers for the benefit of patients-The latest being the Poly ADP-ribose polymerase (PARP) and the Poly ADP-ribose glycohydrolase (PARG) inhibitors.  This review highlights their mechanism of action under the rationale of their use and current development in the field of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohbT47IdZPjbVg90H21EOLACvtfcHk0lgTJF1_GmmpGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlSjt7%252FF&md5=17a2c5402b4cb761abe03665edb4a41c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs12253-010-9266-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12253-010-9266-6%26sid%3Dliteratum%253Aachs%26aulast%3DFauzee%26aufirst%3DN.%2BJ.%2BS.%26aulast%3DPan%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.-l.%26atitle%3DPARP%2520and%2520PARG%2520Inhibitors-New%2520Therapeutic%2520Targets%2520in%2520Cancer%2520Treatment%26jtitle%3DPathol.%2520Oncol.%2520Res.%26date%3D2010%26volume%3D16%26spage%3D469%26epage%3D478%26doi%3D10.1007%2Fs12253-010-9266-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makhija, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandak, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaikwad, A. B</span></span> <span> </span><span class="NLM_article-title">PARG Inhibitors’ Success: A Long Way to Go!</span>. <i>Curr. Enzyme Inhib.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">93</span>, <span class="refDoi"> DOI: 10.2174/1573408010666141126220225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.2174%2F1573408010666141126220225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVGitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=81-93&author=A.+Pandeyauthor=P.+Makhijaauthor=P.+G.+Chandakauthor=A.+B+Gaikwad&title=PARG+Inhibitors%E2%80%99+Success%3A+A+Long+Way+to+Go%21&doi=10.2174%2F1573408010666141126220225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">"PARG Inhibitorsμ Success: A Long Way to Go!"</span></div><div class="casAuthors">Pandey, Anuradha; Makhija, Pooja; Chandak, Prakash G.; Gaikwad, Anil Bhanudas</div><div class="citationInfo"><span class="NLM_cas:title">Current Enzyme Inhibition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">81-93</span>CODEN:
                <span class="NLM_cas:coden">CEIUAG</span>;
        ISSN:<span class="NLM_cas:issn">1573-4080</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribosyl)ation is one of the most pertinent post translational modifications involved in regulation of chromatin structure, cell cycle progression, tissue development and differentiation and other vital biol. phenomena.  The enzymes that catalyze the synthesis and degrdn. of poly(ADP) ribose polymers are PARP and PARG, resp.  The role of PARP has been implicated in development of various diseases since a long time and hence it has evolved as an important pharmacol. target but a plethora of drawbacks assocd. with PARP inhibitors compelled the shift of focus towards PARG.  Recently PARG has evolved as an alternative target to overcome the hurdles being faced in the treatment of various conditions like multiple organ failure, ischemic organ damage, diabetic nephropathy, neurodegenerative diseases and cancer.  The review provides a compendium on PARG, its mode of action, inhibitors, and its therapeutic applications and also discusses the reasons due to which PARG inhibitors have not been able to reach the clin. trials.  PARG inhibitors, though far from success, definitely appear as alluring topics for further research as PARG emerges out as an eminent pharmacol. target in making which can shape the future of medicines to provide better therapy with reduced side effects and more efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreF7WC4XqPYbVg90H21EOLACvtfcHk0lgTJF1_GmmpGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVGitrw%253D&md5=c257034e60bfb73a0bcf395a03ba5fdd</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.2174%2F1573408010666141126220225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1573408010666141126220225%26sid%3Dliteratum%253Aachs%26aulast%3DPandey%26aufirst%3DA.%26aulast%3DMakhija%26aufirst%3DP.%26aulast%3DChandak%26aufirst%3DP.%2BG.%26aulast%3DGaikwad%26aufirst%3DA.%2BB%26atitle%3DPARG%2520Inhibitors%25E2%2580%2599%2520Success%253A%2520A%2520Long%2520Way%2520to%2520Go%2521%26jtitle%3DCurr.%2520Enzyme%2520Inhib.%26date%3D2014%26volume%3D10%26spage%3D81%26epage%3D93%26doi%3D10.2174%2F1573408010666141126220225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">James, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairweather, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hitchin, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutton, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGonagle, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stowell, A. I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddell, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waszkowycz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, D. J.</span></span> <span> </span><span class="NLM_article-title">First-in-Class Chemical Probes against Poly(ADP-ribose) Glycohydrolase (PARG) Inhibit DNA Repair with Differential Pharmacology to Olaparib</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3179</span>– <span class="NLM_lpage">3190</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00609</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00609" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=3179-3190&author=D.+I.+Jamesauthor=K.+M.+Smithauthor=A.+M.+Jordanauthor=E.+E.+Fairweatherauthor=L.+A.+Griffithsauthor=N.+S.+Hamiltonauthor=J.+R.+Hitchinauthor=C.+P.+Huttonauthor=S.+Jonesauthor=P.+Kellyauthor=A.+E.+McGonagleauthor=H.+Smallauthor=A.+I.+J.+Stowellauthor=J.+Tuckerauthor=I.+D.+Waddellauthor=B.+Waszkowyczauthor=D.+J.+Ogilvie&title=First-in-Class+Chemical+Probes+against+Poly%28ADP-ribose%29+Glycohydrolase+%28PARG%29+Inhibit+DNA+Repair+with+Differential+Pharmacology+to+Olaparib&doi=10.1021%2Facschembio.6b00609"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00609%26sid%3Dliteratum%253Aachs%26aulast%3DJames%26aufirst%3DD.%2BI.%26aulast%3DSmith%26aufirst%3DK.%2BM.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DFairweather%26aufirst%3DE.%2BE.%26aulast%3DGriffiths%26aufirst%3DL.%2BA.%26aulast%3DHamilton%26aufirst%3DN.%2BS.%26aulast%3DHitchin%26aufirst%3DJ.%2BR.%26aulast%3DHutton%26aufirst%3DC.%2BP.%26aulast%3DJones%26aufirst%3DS.%26aulast%3DKelly%26aufirst%3DP.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DSmall%26aufirst%3DH.%26aulast%3DStowell%26aufirst%3DA.%2BI.%2BJ.%26aulast%3DTucker%26aufirst%3DJ.%26aulast%3DWaddell%26aufirst%3DI.%2BD.%26aulast%3DWaszkowycz%26aufirst%3DB.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26atitle%3DFirst-in-Class%2520Chemical%2520Probes%2520against%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%2520%2528PARG%2529%2520Inhibit%2520DNA%2520Repair%2520with%2520Differential%2520Pharmacology%2520to%2520Olaparib%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D3179%26epage%3D3190%26doi%3D10.1021%2Facschembio.6b00609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slade, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunstan, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barkauskaite, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weston, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafite, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leys, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, I.</span></span> <span> </span><span class="NLM_article-title">The Structure and Catalytic Mechanism of a Poly(ADP-ribose) Glycohydrolase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>477</i></span>,  <span class="NLM_fpage">616</span>– <span class="NLM_lpage">620</span>, <span class="refDoi"> DOI: 10.1038/nature10404</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1038%2Fnature10404" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=21892188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFers7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=477&publication_year=2011&pages=616-620&author=D.+Sladeauthor=M.+S.+Dunstanauthor=E.+Barkauskaiteauthor=R.+Westonauthor=P.+Lafiteauthor=N.+Dixonauthor=M.+Ahelauthor=D.+Leysauthor=I.+Ahel&title=The+Structure+and+Catalytic+Mechanism+of+a+Poly%28ADP-ribose%29+Glycohydrolase&doi=10.1038%2Fnature10404"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The structure and catalytic mechanism of a poly(ADP-ribose) glycohydrolase</span></div><div class="casAuthors">Slade, Dea; Dunstan, Mark S.; Barkauskaite, Eva; Weston, Ria; Lafite, Pierre; Dixon, Neil; Ahel, Marijan; Leys, David; Ahel, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">477</span>
        (<span class="NLM_cas:issue">7366</span>),
    <span class="NLM_cas:pages">616-620</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Post-translational modification of proteins by poly(ADP-ribosyl)ation regulates many cellular pathways that are crit. for genome stability, including DNA repair, chromatin structure, mitosis, and apoptosis.  Poly(ADP-ribose) (PAR) is composed of repeating ADP-ribose units linked via a unique glycosidic ribose-ribose bond, and is synthesized from NAD by PAR polymerases.  PAR glycohydrolase (PARG) is the only protein capable of specific hydrolysis of the ribose-ribose bonds present in PAR chains; its deficiency leads to cell death.  Here, the authors show that filamentous fungi and a no. of bacteria possess a divergent form of PARG that has all the main characteristics of the human PARG enzyme.  The authors present the 1st crystal structure of PARG of Thermomonospora curvata, which reveals that the PARG catalytic domain is a distant member of the ubiquitous ADP-ribose-binding macrodomain family.  High-resoln. structures of T. curvata PARG in complexes with ADP-ribose and the PARG inhibitor, ADP (hydroxymethyl)pyrrolidinediol (ADP-HPD), complemented by biochem. studies, allowed the authors to propose a model for PAR binding and catalysis by PARG.  The insights into the PARG structure and catalytic mechanism should greatly improve the understanding of how PARG activity controls reversible protein poly(ADP-ribosyl)ation and potentially of how the defects in this regulation are linked to human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjFsMUTjjlOrVg90H21EOLACvtfcHk0lgFISrbIlmQwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFers7rE&md5=7e539f310e582800407bddd95efca01f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature10404&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10404%26sid%3Dliteratum%253Aachs%26aulast%3DSlade%26aufirst%3DD.%26aulast%3DDunstan%26aufirst%3DM.%2BS.%26aulast%3DBarkauskaite%26aufirst%3DE.%26aulast%3DWeston%26aufirst%3DR.%26aulast%3DLafite%26aufirst%3DP.%26aulast%3DDixon%26aufirst%3DN.%26aulast%3DAhel%26aufirst%3DM.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DAhel%26aufirst%3DI.%26atitle%3DThe%2520Structure%2520and%2520Catalytic%2520Mechanism%2520of%2520a%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%26jtitle%3DNature%26date%3D2011%26volume%3D477%26spage%3D616%26epage%3D620%26doi%3D10.1038%2Fnature10404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dunstan, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barkauskaite, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafite, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knezevic, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brassington, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leys, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, I.</span></span> <span> </span><span class="NLM_article-title">Structure and Mechanism of a Canonical Poly(ADP-ribose) Glycohydrolase</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">878</span>, <span class="refDoi"> DOI: 10.1038/ncomms1889</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1038%2Fncomms1889" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=22673905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A280%3ADC%252BC38npsFOrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=878&author=M.+S.+Dunstanauthor=E.+Barkauskaiteauthor=P.+Lafiteauthor=C.+E.+Knezevicauthor=A.+Brassingtonauthor=M.+Ahelauthor=P.+J.+Hergenrotherauthor=D.+Leysauthor=I.+Ahel&title=Structure+and+Mechanism+of+a+Canonical+Poly%28ADP-ribose%29+Glycohydrolase&doi=10.1038%2Fncomms1889"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and mechanism of a canonical poly(ADP-ribose) glycohydrolase</span></div><div class="casAuthors">Dunstan Mark S; Barkauskaite Eva; Lafite Pierre; Knezevic Claire E; Brassington Amy; Ahel Marijan; Hergenrother Paul J; Leys David; Ahel Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">878</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Poly(ADP-ribosyl)ation is a reversible post-translational protein modification involved in the regulation of a number of cellular processes including DNA repair, chromatin structure, mitosis, transcription, checkpoint activation, apoptosis and asexual development.  The reversion of poly(ADP-ribosyl)ation is catalysed by poly(ADP-ribose) (PAR) glycohydrolase (PARG), which specifically targets the unique PAR (1''-2') ribose-ribose bonds.  Here we report the structure and mechanism of the first canonical PARG from the protozoan Tetrahymena thermophila.  In addition, we reveal the structure of T. thermophila PARG in a complex with a novel rhodanine-containing mammalian PARG inhibitor RBPI-3.  Our data demonstrate that the protozoan PARG represents a good model for human PARG and is therefore likely to prove useful in guiding structure-based discovery of new classes of PARG inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTWJIiwTxgYVv6BottYdIAWfW6udTcc2eZiq-hGHhRFYrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38npsFOrtQ%253D%253D&md5=42bea8394a83a8c32f6fb422e5862cbc</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fncomms1889&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms1889%26sid%3Dliteratum%253Aachs%26aulast%3DDunstan%26aufirst%3DM.%2BS.%26aulast%3DBarkauskaite%26aufirst%3DE.%26aulast%3DLafite%26aufirst%3DP.%26aulast%3DKnezevic%26aufirst%3DC.%2BE.%26aulast%3DBrassington%26aufirst%3DA.%26aulast%3DAhel%26aufirst%3DM.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DAhel%26aufirst%3DI.%26atitle%3DStructure%2520and%2520Mechanism%2520of%2520a%2520Canonical%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%26jtitle%3DNat.%2520Commun.%26date%3D2012%26volume%3D3%26spage%3D878%26doi%3D10.1038%2Fncomms1889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, I.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, C. M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegeman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Classen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tainer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellenberger, T.</span></span> <span> </span><span class="NLM_article-title">Structure of Mammalian Poly(ADP-ribose) Glycohydrolase Reveals a Flexible Tyrosine Clasp as a Substrate-binding Element</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">653</span>– <span class="NLM_lpage">656</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1038%2Fnsmb.2305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=22609859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC38XntF2js7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=653-656&author=I.-K.+Kimauthor=J.+R.+Kieferauthor=C.+M.+W.+Hoauthor=R.+A.+Stegemanauthor=S.+Classenauthor=J.+A.+Tainerauthor=T.+Ellenberger&title=Structure+of+Mammalian+Poly%28ADP-ribose%29+Glycohydrolase+Reveals+a+Flexible+Tyrosine+Clasp+as+a+Substrate-binding+Element&doi=10.1038%2Fnsmb.2305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of mammalian poly(ADP-ribose) glycohydrolase reveals a flexible tyrosine clasp as a substrate-binding element</span></div><div class="casAuthors">Kim, In-Kwon; Kiefer, James R.; Ho, Chris M. W.; Stegeman, Roderick A.; Classen, Scott; Tainer, John A.; Ellenberger, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">653-656</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Reversible post-translational modification by poly(ADP-ribose) (PAR) regulates chromatin structure, DNA repair and cell fate in response to genotoxic stress.  PAR glycohydrolase (PARG) removes PAR chains from poly ADP-ribosylated proteins to restore protein function and release oligo(ADP-ribose) chains to signal damage.  Here we report crystal structures of mammalian PARG and its complex with a substrate mimic that reveal an open substrate-binding site and a unique 'tyrosine clasp' enabling endoglycosidic cleavage of branched PAR chains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLggg2gkhJj7Vg90H21EOLACvtfcHk0lgFISrbIlmQwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntF2js7c%253D&md5=fda9258dddd2b7dbd9609291d757f7e5</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2305%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DI.-K.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DHo%26aufirst%3DC.%2BM.%2BW.%26aulast%3DStegeman%26aufirst%3DR.%2BA.%26aulast%3DClassen%26aufirst%3DS.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26aulast%3DEllenberger%26aufirst%3DT.%26atitle%3DStructure%2520of%2520Mammalian%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%2520Reveals%2520a%2520Flexible%2520Tyrosine%2520Clasp%2520as%2520a%2520Substrate-binding%2520Element%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D653%26epage%3D656%26doi%3D10.1038%2Fnsmb.2305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brassington, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durant, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassall, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holdgate, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAlister, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, M.</span></span> <span> </span><span class="NLM_article-title">Structures of the Human Poly(ADP-Ribose) Glycohydrolase Catalytic Domain Confirm Catalytic Mechanism and Explain Inhibition by ADP-HPD Derivatives</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e50889</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0050889</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1371%2Fjournal.pone.0050889" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&author=J.+A.+Tuckerauthor=N.+Bennettauthor=C.+Brassingtonauthor=S.+T.+Durantauthor=G.+Hassallauthor=G.+Holdgateauthor=M.+McAlisterauthor=J.+W.+M.+Nissinkauthor=C.+Trumanauthor=M.+Watson&title=Structures+of+the+Human+Poly%28ADP-Ribose%29+Glycohydrolase+Catalytic+Domain+Confirm+Catalytic+Mechanism+and+Explain+Inhibition+by+ADP-HPD+Derivatives&doi=10.1371%2Fjournal.pone.0050889"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0050889&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0050889%26sid%3Dliteratum%253Aachs%26aulast%3DTucker%26aufirst%3DJ.%2BA.%26aulast%3DBennett%26aufirst%3DN.%26aulast%3DBrassington%26aufirst%3DC.%26aulast%3DDurant%26aufirst%3DS.%2BT.%26aulast%3DHassall%26aufirst%3DG.%26aulast%3DHoldgate%26aufirst%3DG.%26aulast%3DMcAlister%26aufirst%3DM.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DTruman%26aufirst%3DC.%26aulast%3DWatson%26aufirst%3DM.%26atitle%3DStructures%2520of%2520the%2520Human%2520Poly%2528ADP-Ribose%2529%2520Glycohydrolase%2520Catalytic%2520Domain%2520Confirm%2520Catalytic%2520Mechanism%2520and%2520Explain%2520Inhibition%2520by%2520ADP-HPD%2520Derivatives%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26doi%3D10.1371%2Fjournal.pone.0050889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barkauskaite, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brassington, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warwicker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunstan, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafite, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchison, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leys, D.</span></span> <span> </span><span class="NLM_article-title">Visualization of Poly(ADP-ribose) Bound to PARG Reveals Inherent Balance Between <i>Exo</i>- and <i>Endo</i>-Glycohydrolase Activities</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">2164</span>, <span class="refDoi"> DOI: 10.1038/ncomms3164</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1038%2Fncomms3164" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=23917065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A280%3ADC%252BC3sfntFSquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2164&author=E.+Barkauskaiteauthor=A.+Brassingtonauthor=E.+S.+Tanauthor=J.+Warwickerauthor=M.+S.+Dunstanauthor=B.+Banosauthor=P.+Lafiteauthor=M.+Ahelauthor=T.+J.+Mitchisonauthor=I.+Ahelauthor=D.+Leys&title=Visualization+of+Poly%28ADP-ribose%29+Bound+to+PARG+Reveals+Inherent+Balance+Between+Exo-+and+Endo-Glycohydrolase+Activities&doi=10.1038%2Fncomms3164"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Visualization of poly(ADP-ribose) bound to PARG reveals inherent balance between exo- and endo-glycohydrolase activities</span></div><div class="casAuthors">Barkauskaite Eva; Brassington Amy; Tan Edwin S; Warwicker Jim; Dunstan Mark S; Banos Benito; Lafite Pierre; Ahel Marijan; Mitchison Timothy J; Ahel Ivan; Leys David</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2164</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Poly-ADP-ribosylation is a post-translational modification that regulates processes involved in genome stability.  Breakdown of the poly(ADP-ribose) (PAR) polymer is catalysed by poly(ADP-ribose) glycohydrolase (PARG), whose endo-glycohydrolase activity generates PAR fragments.  Here we present the crystal structure of PARG incorporating the PAR substrate.  The two terminal ADP-ribose units of the polymeric substrate are bound in exo-mode.  Biochemical and modelling studies reveal that PARG acts predominantly as an exo-glycohydrolase.  This preference is linked to Phe902 (human numbering), which is responsible for low-affinity binding of the substrate in endo-mode.  Our data reveal the mechanism of poly-ADP-ribosylation reversal, with ADP-ribose as the dominant product, and suggest that the release of apoptotic PAR fragments occurs at unusual PAR/PARG ratios.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSRNfG0TEYbZfSAs_PE1QvifW6udTcc2eZbuqDyoW9gd7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sfntFSquw%253D%253D&md5=53b696cff64b0dcb983cf0c35a839a46</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fncomms3164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms3164%26sid%3Dliteratum%253Aachs%26aulast%3DBarkauskaite%26aufirst%3DE.%26aulast%3DBrassington%26aufirst%3DA.%26aulast%3DTan%26aufirst%3DE.%2BS.%26aulast%3DWarwicker%26aufirst%3DJ.%26aulast%3DDunstan%26aufirst%3DM.%2BS.%26aulast%3DBanos%26aufirst%3DB.%26aulast%3DLafite%26aufirst%3DP.%26aulast%3DAhel%26aufirst%3DM.%26aulast%3DMitchison%26aufirst%3DT.%2BJ.%26aulast%3DAhel%26aufirst%3DI.%26aulast%3DLeys%26aufirst%3DD.%26atitle%3DVisualization%2520of%2520Poly%2528ADP-ribose%2529%2520Bound%2520to%2520PARG%2520Reveals%2520Inherent%2520Balance%2520Between%2520Exo-%2520and%2520Endo-Glycohydrolase%2520Activities%26jtitle%3DNat.%2520Commun.%26date%3D2013%26volume%3D4%26spage%3D2164%26doi%3D10.1038%2Fncomms3164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Žaja, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikoč, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barkauskaite, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, I.</span></span> <span> </span><span class="NLM_article-title">Molecular Insights into Poly(ADP-ribose) Recognition and Processing</span>. <i>Biomolecules</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.3390/biom3010001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.3390%2Fbiom3010001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=1-17&author=R.+%C5%BDajaauthor=A.+Miko%C4%8Dauthor=E.+Barkauskaiteauthor=I.+Ahel&title=Molecular+Insights+into+Poly%28ADP-ribose%29+Recognition+and+Processing&doi=10.3390%2Fbiom3010001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.3390%2Fbiom3010001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiom3010001%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%25BDaja%26aufirst%3DR.%26aulast%3DMiko%25C4%258D%26aufirst%3DA.%26aulast%3DBarkauskaite%26aufirst%3DE.%26aulast%3DAhel%26aufirst%3DI.%26atitle%3DMolecular%2520Insights%2520into%2520Poly%2528ADP-ribose%2529%2520Recognition%2520and%2520Processing%26jtitle%3DBiomolecules%26date%3D2013%26volume%3D3%26spage%3D1%26epage%3D17%26doi%3D10.3390%2Fbiom3010001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagné, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span> <span> </span><span class="NLM_article-title">Crystallographic and Biochemical Analysis of the Mouse Poly(ADP-ribose) Glycohydrolase</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e86010</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0086010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1371%2Fjournal.pone.0086010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&author=Z.+Wangauthor=J.-P.+Gagn%C3%A9author=G.+G.+Poirierauthor=W.+Xu&title=Crystallographic+and+Biochemical+Analysis+of+the+Mouse+Poly%28ADP-ribose%29+Glycohydrolase&doi=10.1371%2Fjournal.pone.0086010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0086010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0086010%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DGagn%25C3%25A9%26aufirst%3DJ.-P.%26aulast%3DPoirier%26aufirst%3DG.%2BG.%26aulast%3DXu%26aufirst%3DW.%26atitle%3DCrystallographic%2520and%2520Biochemical%2520Analysis%2520of%2520the%2520Mouse%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26doi%3D10.1371%2Fjournal.pone.0086010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rack, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, I.</span></span> <span> </span><span class="NLM_article-title">Macrodomains: Structure, Function, Evolution, and Catalytic Activities</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">431</span>– <span class="NLM_lpage">454</span>, <span class="refDoi"> DOI: 10.1146/annurev-biochem-060815-014935</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1146%2Fannurev-biochem-060815-014935" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=26844395" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1WrsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2016&pages=431-454&author=J.+G.+Rackauthor=D.+Perinaauthor=I.+Ahel&title=Macrodomains%3A+Structure%2C+Function%2C+Evolution%2C+and+Catalytic+Activities&doi=10.1146%2Fannurev-biochem-060815-014935"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Macrodomains: Structure, Function, Evolution, and Catalytic Activities</span></div><div class="casAuthors">Rack, Johannes Gregor Matthias; Perina, Dragutin; Ahel, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">431-454</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  Recent developments indicate that macrodomains, an ancient and diverse protein domain family, are key players in the recognition, interpretation, and turnover of ADP-ribose (ADPr) signaling.  Crucial to this is the ability of macrodomains to recognize ADPr either directly, in the form of a metabolic deriv., or as a modification covalently bound to proteins.  Thus, macrodomains regulate a wide variety of cellular and organismal processes, including DNA damage repair, signal transduction, and immune response.  Their importance is further indicated by the fact that dysregulation or mutation of a macrodomain is assocd. with several diseases, including cancer, developmental defects, and neurodegeneration.  In this review, we summarize the current insights into macrodomain evolution and how this evolution influenced their structural and functional diversification.  We highlight some aspects of macrodomain roles in pathobiol. as well as their emerging potential as therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozF6aFyzAOS7Vg90H21EOLACvtfcHk0ljz2zFr7mvH1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1WrsLY%253D&md5=cdd198a0cdbdb8d41adf66109794cb5a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-060815-014935&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-060815-014935%26sid%3Dliteratum%253Aachs%26aulast%3DRack%26aufirst%3DJ.%2BG.%26aulast%3DPerina%26aufirst%3DD.%26aulast%3DAhel%26aufirst%3DI.%26atitle%3DMacrodomains%253A%2520Structure%252C%2520Function%252C%2520Evolution%252C%2520and%2520Catalytic%2520Activities%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2016%26volume%3D85%26spage%3D431%26epage%3D454%26doi%3D10.1146%2Fannurev-biochem-060815-014935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stowell, A. I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddell, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardern, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, D. J.</span></span> <span> </span><span class="NLM_article-title">A High-throughput Screening-compatible Homogeneous Time-resolved Fluorescence Assay Measuring the Glycohydrolase Activity of Human Poly(ADP-ribose) Glycohydrolase</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>503</i></span>,  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2016.03.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1016%2Fj.ab.2016.03.016" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2016&pages=58-64&author=A.+I.+J.+Stowellauthor=D.+I.+Jamesauthor=I.+D.+Waddellauthor=N.+Bennettauthor=C.+Trumanauthor=I.+M.+Hardernauthor=D.+J.+Ogilvie&title=A+High-throughput+Screening-compatible+Homogeneous+Time-resolved+Fluorescence+Assay+Measuring+the+Glycohydrolase+Activity+of+Human+Poly%28ADP-ribose%29+Glycohydrolase&doi=10.1016%2Fj.ab.2016.03.016"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2016.03.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2016.03.016%26sid%3Dliteratum%253Aachs%26aulast%3DStowell%26aufirst%3DA.%2BI.%2BJ.%26aulast%3DJames%26aufirst%3DD.%2BI.%26aulast%3DWaddell%26aufirst%3DI.%2BD.%26aulast%3DBennett%26aufirst%3DN.%26aulast%3DTruman%26aufirst%3DC.%26aulast%3DHardern%26aufirst%3DI.%2BM.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26atitle%3DA%2520High-throughput%2520Screening-compatible%2520Homogeneous%2520Time-resolved%2520Fluorescence%2520Assay%2520Measuring%2520the%2520Glycohydrolase%2520Activity%2520of%2520Human%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%26jtitle%3DAnal.%2520Biochem.%26date%3D2016%26volume%3D503%26spage%3D58%26epage%3D64%26doi%3D10.1016%2Fj.ab.2016.03.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koh, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsinghani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slama, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. K.</span></span> <span> </span><span class="NLM_article-title">SAR Analysis of Adenosine Diphosphate (Hydroxymethyl)pyrrolidinediol Inhibition of Poly(ADP-ribose) Glycohydrolase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">4322</span>– <span class="NLM_lpage">4332</span>, <span class="refDoi"> DOI: 10.1021/jm020541u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020541u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=4322-4332&author=D.+W.+Kohauthor=D.+L.+Coyleauthor=N.+Mehtaauthor=S.+Ramsinghaniauthor=H.+Kimauthor=J.+T.+Slamaauthor=M.+K.+Jacobson&title=SAR+Analysis+of+Adenosine+Diphosphate+%28Hydroxymethyl%29pyrrolidinediol+Inhibition+of+Poly%28ADP-ribose%29+Glycohydrolase&doi=10.1021%2Fjm020541u"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm020541u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020541u%26sid%3Dliteratum%253Aachs%26aulast%3DKoh%26aufirst%3DD.%2BW.%26aulast%3DCoyle%26aufirst%3DD.%2BL.%26aulast%3DMehta%26aufirst%3DN.%26aulast%3DRamsinghani%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DSlama%26aufirst%3DJ.%2BT.%26aulast%3DJacobson%26aufirst%3DM.%2BK.%26atitle%3DSAR%2520Analysis%2520of%2520Adenosine%2520Diphosphate%2520%2528Hydroxymethyl%2529pyrrolidinediol%2520Inhibition%2520of%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D4322%26epage%3D4332%26doi%3D10.1021%2Fjm020541u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Formentini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arapistas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacomelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitozzi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menichetti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giovannelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarugi, A.</span></span> <span> </span><span class="NLM_article-title">Mono-galloyl Glucose Derivatives are Potent Poly(ADP-ribose) Glycohydrolase (PARG) Inhibitors and Partially Reduce PARP-1-dependent Cell Death</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">1235</span>– <span class="NLM_lpage">1249</span>, <span class="refDoi"> DOI: 10.1038/bjp.2008.370</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1038%2Fbjp.2008.370" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=18806807" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVKlt7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2008&pages=1235-1249&author=L.+Formentiniauthor=P.+Arapistasauthor=M.+Pittelliauthor=M.+Jacomelliauthor=V.+Pitozziauthor=S.+Menichettiauthor=A.+Romaniauthor=L.+Giovannelliauthor=F.+Moroniauthor=A.+Chiarugi&title=Mono-galloyl+Glucose+Derivatives+are+Potent+Poly%28ADP-ribose%29+Glycohydrolase+%28PARG%29+Inhibitors+and+Partially+Reduce+PARP-1-dependent+Cell+Death&doi=10.1038%2Fbjp.2008.370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Mono-galloyl glucose derivatives are potent poly(ADP-ribose) glycohydrolase (PARG) inhibitors and partially reduce PARP-1-dependent cell death</span></div><div class="casAuthors">Formentini, L.; Arapistas, P.; Pittelli, M.; Jacomelli, M.; Pitozzi, V.; Menichetti, S.; Romani, A.; Giovannelli, L.; Moroni, F.; Chiarugi, A.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1235-1249</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Maintenance of poly(ADP-ribose) (PAR) polymers at homoeostatic levels by PAR glycohydrolase (PARG) is central in cell functioning and survival.  Yet the pharmacol. relevance of PARG inhibitors is still debated.  Gallotannin, a complex mixt. of hydrolyzable tannins from oak gall, inhibits PARG but which of its constituents is responsible for the inhibition and whether the pharmacodynamic properties are due to its antioxidant properties, has not yet been established.  A structure-activity relationship study was conducted on different natural and synthetic tannins/galloyl derivs. as potential PARG inhibitors, using a novel in vitro enzymic assay.  Cytotoxicity was assayed in cultured HeLa cells.  Mono-galloyl glucose compds. were potent inhibitors of PARG, with activities similar to that of ADP-(hydroxymethyl) pyrrolidinediol, the most potent PARG inhibitor yet identified.  When tested on HeLa cells exposed to the PAR polymerase (PARP)-1-activating compd. 1-methyl-3-nitro-1-nitrosoguanidine (MNNG), 3-galloyl glucose weakly inhibited PAR degrdn.  Conversely, the more lipophilic, 3-galloyl-1,2-O-isopropylidene glucose, despite being inactive on the pure enzyme, efficiently prolonged the half-life of the polymers in intact HeLa cells.  Also, PARG inhibitors, but not radical scavengers, reduced, in part, cell death caused by MNNG.  Taken together, our findings identify mono-galloyl glucose derivs. as potent PARG inhibitors, and emphasize the active function of this enzyme in cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdKsn9DOPnL7Vg90H21EOLACvtfcHk0ljBFcMYLZJpbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVKlt7rE&md5=f28e30c70b4e12b491b7b9e354f03311</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fbjp.2008.370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjp.2008.370%26sid%3Dliteratum%253Aachs%26aulast%3DFormentini%26aufirst%3DL.%26aulast%3DArapistas%26aufirst%3DP.%26aulast%3DPittelli%26aufirst%3DM.%26aulast%3DJacomelli%26aufirst%3DM.%26aulast%3DPitozzi%26aufirst%3DV.%26aulast%3DMenichetti%26aufirst%3DS.%26aulast%3DRomani%26aufirst%3DA.%26aulast%3DGiovannelli%26aufirst%3DL.%26aulast%3DMoroni%26aufirst%3DF.%26aulast%3DChiarugi%26aufirst%3DA.%26atitle%3DMono-galloyl%2520Glucose%2520Derivatives%2520are%2520Potent%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%2520%2528PARG%2529%2520Inhibitors%2520and%2520Partially%2520Reduce%2520PARP-1-dependent%2520Cell%2520Death%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2008%26volume%3D155%26spage%3D1235%26epage%3D1249%26doi%3D10.1038%2Fbjp.2008.370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steffen, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damodaran, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beroza, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. K.</span></span> <span> </span><span class="NLM_article-title">Discovery and Structure-Activity Relationships of Modified Salicylanilides as Cell Permeable Inhibitors of Poly(ADP-ribose) Glycohydrolase (PARG)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">5403</span>– <span class="NLM_lpage">5413</span>, <span class="refDoi"> DOI: 10.1021/jm200325s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200325s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFyksLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=5403-5413&author=J.+D.+Steffenauthor=D.+L.+Coyleauthor=K.+Damodaranauthor=P.+Berozaauthor=M.+K.+Jacobson&title=Discovery+and+Structure-Activity+Relationships+of+Modified+Salicylanilides+as+Cell+Permeable+Inhibitors+of+Poly%28ADP-ribose%29+Glycohydrolase+%28PARG%29&doi=10.1021%2Fjm200325s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Structure-Activity Relationships of Modified Salicylanilides as Cell Permeable Inhibitors of Poly(ADP-ribose) Glycohydrolase (PARG)</span></div><div class="casAuthors">Steffen, Jamin D.; Coyle, Donna L.; Damodaran, Komath; Beroza, Paul; Jacobson, Myron K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5403-5413</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The metab. of poly(ADP-ribose) (PAR) in response to DNA strand breaks, which involves the concerted activities of poly(ADP-ribose) polymerases (PARPs) and poly(ADP-ribose) glycohydrolase (PARG), modulates cell recovery or cell death depending upon the level of DNA damage.  While PARP inhibitors show high promise in clin. trials because of their low toxicity and selectivity for BRCA related cancers, evaluation of the therapeutic potential of PARG is limited by the lack of well-validated cell permeable inhibitors.  In this study, target-related affinity profiling (TRAP), an alternative to high-throughput screening, was used to identify a no. of druglike compds. from several chem. classes that demonstrated PARG inhibition in the low-micromolar range.  A no. of analogs of one of the most active chemotypes were synthesized to explore the structure-activity relationship (SAR) for that series.  This led to the discovery of a putative pharmacophore for PARG inhibition that contains a modified salicylanilide structure.  Interestingly, these compds. also inhibit PARP-1, indicating strong homol. in the active sites of PARG and PARP-1 and raising a new challenge for development of PARG specific inhibitors.  The cellular activity of a lead inhibitor was demonstrated by the inhibition of both PARP and PARG activity in squamous cell carcinoma cells, although preferential inhibition of PARG relative to PARP was obsd.  The ability of inhibitors to modulate PAR metab. via simultaneous effects on PARPs and PARG may represent a new approach for therapeutic development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1rgu2zajJJ7Vg90H21EOLACvtfcHk0ljBFcMYLZJpbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFyksLc%253D&md5=340fde4d20e1a35348d94f7e75ff3641</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm200325s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200325s%26sid%3Dliteratum%253Aachs%26aulast%3DSteffen%26aufirst%3DJ.%2BD.%26aulast%3DCoyle%26aufirst%3DD.%2BL.%26aulast%3DDamodaran%26aufirst%3DK.%26aulast%3DBeroza%26aufirst%3DP.%26aulast%3DJacobson%26aufirst%3DM.%2BK.%26atitle%3DDiscovery%2520and%2520Structure-Activity%2520Relationships%2520of%2520Modified%2520Salicylanilides%2520as%2520Cell%2520Permeable%2520Inhibitors%2520of%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%2520%2528PARG%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D5403%26epage%3D5413%26doi%3D10.1021%2Fjm200325s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finch, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knezevic, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nottbohm, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Partlow, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Selective Small Molecule Inhibition of Poly(ADP-Ribose) Glycohydrolase (PARG)</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">563</span>– <span class="NLM_lpage">570</span>, <span class="refDoi"> DOI: 10.1021/cb200506t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb200506t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=563-570&author=K.+E.+Finchauthor=C.+E.+Knezevicauthor=A.+C.+Nottbohmauthor=K.+C.+Partlowauthor=P.+J.+Hergenrother&title=Selective+Small+Molecule+Inhibition+of+Poly%28ADP-Ribose%29+Glycohydrolase+%28PARG%29&doi=10.1021%2Fcb200506t"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fcb200506t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb200506t%26sid%3Dliteratum%253Aachs%26aulast%3DFinch%26aufirst%3DK.%2BE.%26aulast%3DKnezevic%26aufirst%3DC.%2BE.%26aulast%3DNottbohm%26aufirst%3DA.%2BC.%26aulast%3DPartlow%26aufirst%3DK.%2BC.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DSelective%2520Small%2520Molecule%2520Inhibition%2520of%2520Poly%2528ADP-Ribose%2529%2520Glycohydrolase%2520%2528PARG%2529%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D563%26epage%3D570%26doi%3D10.1021%2Fcb200506t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Islam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koizumi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodera, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okawara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masutani, M.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of Phenolic Hydrazide Hydrazones as Potent Poly(ADP-ribose) Glycohydrolase (PARG) Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3802</span>– <span class="NLM_lpage">3806</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2014.06.065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1016%2Fj.bmcl.2014.06.065" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=3802-3806&author=R.+Islamauthor=F.+Koizumiauthor=Y.+Koderaauthor=K.+Inoueauthor=T.+Okawaraauthor=M.+Masutani&title=Design+and+Synthesis+of+Phenolic+Hydrazide+Hydrazones+as+Potent+Poly%28ADP-ribose%29+Glycohydrolase+%28PARG%29+Inhibitors&doi=10.1016%2Fj.bmcl.2014.06.065"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.06.065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.06.065%26sid%3Dliteratum%253Aachs%26aulast%3DIslam%26aufirst%3DR.%26aulast%3DKoizumi%26aufirst%3DF.%26aulast%3DKodera%26aufirst%3DY.%26aulast%3DInoue%26aufirst%3DK.%26aulast%3DOkawara%26aufirst%3DT.%26aulast%3DMasutani%26aufirst%3DM.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520Phenolic%2520Hydrazide%2520Hydrazones%2520as%2520Potent%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%2520%2528PARG%2529%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D3802%26epage%3D3806%26doi%3D10.1016%2Fj.bmcl.2014.06.065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Biosassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Biosassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Biosassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGonagle, A. E.</span>; <span class="NLM_string-name">Jordan, A. M.</span>; <span class="NLM_string-name">Waszkowycz, B.</span>; <span class="NLM_string-name">Hutton, C. P.</span>; <span class="NLM_string-name">Waddell, I. D.</span>; <span class="NLM_string-name">Hitchin, J. R.</span>; <span class="NLM_string-name">Smith, K. M.</span>; <span class="NLM_string-name">Hamilton, N. M.</span></span> <span> </span><span class="NLM_article-title">2,4-Dioxo-quinazoline-6-sulphonamide Derivatives as Inhibitors of PARG</span>.  <i>PCT Int. Appl.</i> <span class="NLM_patent">WO2015079251</span> June 4, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=A.+E.+McGonagle&author=A.+M.+Jordan&author=B.+Waszkowycz&author=C.+P.+Hutton&author=I.+D.+Waddell&author=J.+R.+Hitchin&author=K.+M.+Smith&author=N.+M.+Hamilton&title=2%2C4-Dioxo-quinazoline-6-sulphonamide+Derivatives+as+Inhibitors+of+PARG"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26atitle%3D2%252C4-Dioxo-quinazoline-6-sulphonamide%2520Derivatives%2520as%2520Inhibitors%2520of%2520PARG%26jtitle%3DPCT%2520Int.%2520Appl.%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGonagle, A. E.</span>; <span class="NLM_string-name">Jordan, A. M.</span>; <span class="NLM_string-name">Waszkowycz, B.</span>; <span class="NLM_string-name">Hutton, C. P.</span>; <span class="NLM_string-name">Waddell, I. D.</span>; <span class="NLM_string-name">Hitchin, J. R.</span>; <span class="NLM_string-name">Smith, K. M.</span>; <span class="NLM_string-name">Hamilton, N. M.</span></span> <span> </span><span class="NLM_article-title">PARG Inhibitory Compounds</span>.  <i>PCT Int. Appl.</i> <span class="NLM_patent">WO2016097749</span> June 23, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=A.+E.+McGonagle&author=A.+M.+Jordan&author=B.+Waszkowycz&author=C.+P.+Hutton&author=I.+D.+Waddell&author=J.+R.+Hitchin&author=K.+M.+Smith&author=N.+M.+Hamilton&title=PARG+Inhibitory+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26atitle%3DPARG%2520Inhibitory%2520Compounds%26jtitle%3DPCT%2520Int.%2520Appl.%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipshutz, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corral, R.</span></span> <span> </span><span class="NLM_article-title">Simplification of the Mitsunobu Reaction. Di-p-chlorobenzyl Azodicarboxylate:  A New Azodicarboxylate</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">5069</span>– <span class="NLM_lpage">5072</span>, <span class="refDoi"> DOI: 10.1021/ol0618757</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol0618757" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2006&pages=5069-5072&author=B.+H.+Lipshutzauthor=D.+W.+Chungauthor=B.+Richauthor=R.+Corral&title=Simplification+of+the+Mitsunobu+Reaction.+Di-p-chlorobenzyl+Azodicarboxylate%3A%E2%80%89+A+New+Azodicarboxylate&doi=10.1021%2Fol0618757"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fol0618757&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol0618757%26sid%3Dliteratum%253Aachs%26aulast%3DLipshutz%26aufirst%3DB.%2BH.%26aulast%3DChung%26aufirst%3DD.%2BW.%26aulast%3DRich%26aufirst%3DB.%26aulast%3DCorral%26aufirst%3DR.%26atitle%3DSimplification%2520of%2520the%2520Mitsunobu%2520Reaction.%2520Di-p-chlorobenzyl%2520Azodicarboxylate%253A%25E2%2580%2589%2520A%2520New%2520Azodicarboxylate%26jtitle%3DOrg.%2520Lett.%26date%3D2006%26volume%3D8%26spage%3D5069%26epage%3D5072%26doi%3D10.1021%2Fol0618757" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">James, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durant, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckersley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairweather, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shea, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddell, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, D. J.</span></span> <span> </span><span class="NLM_article-title">An Assay to Measure Poly(ADP ribose) Glycohydrolase (PARG) Activity in Cells</span>. <i>F1000Research</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">736</span>– <span class="NLM_lpage">746</span>, <span class="refDoi"> DOI: 10.12688/f1000research.8463.2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.12688%2Ff1000research.8463.2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=27610220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A280%3ADC%252BC2szptVyqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=736-746&author=D.+I.+Jamesauthor=S.+Durantauthor=K.+Eckersleyauthor=E.+Fairweatherauthor=L.+A.+Griffithsauthor=N.+Hamiltonauthor=P.+Kellyauthor=M.+O%E2%80%99Connorauthor=K.+Sheaauthor=I.+D.+Waddellauthor=D.+J.+Ogilvie&title=An+Assay+to+Measure+Poly%28ADP+ribose%29+Glycohydrolase+%28PARG%29+Activity+in+Cells&doi=10.12688%2Ff1000research.8463.2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">An assay to measure poly(ADP ribose) glycohydrolase (PARG) activity in cells</span></div><div class="casAuthors">James Dominic I; Fairweather Emma; Griffiths Louise A; Hamilton Nicola; Kelly Paul; Waddell Ian D; Ogilvie Donald J; Durant Stephen; Eckersley Kay; O'Connor Mark; Shea Kerry</div><div class="citationInfo"><span class="NLM_cas:title">F1000Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">736</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">After a DNA damage signal multiple polymers of ADP ribose attached to poly(ADP) ribose (PAR) polymerases (PARPs) are broken down by the enzyme poly(ADP) ribose glycohydrolase (PARG).  Inhibition of PARG leads to a failure of DNA repair and small molecule inhibition of PARG has been a goal for many years.  To determine whether biochemical inhibitors of PARG are active in cells we have designed an immunofluorescence assay to detect nuclear PAR after DNA damage.  This 384-well assay is suitable for medium throughput high-content screening and can detect cell-permeable inhibitors of PARG from nM to μM potency.  In addition, the assay has been shown to work in murine cells and in a variety of human cancer cells.  Furthermore, the assay is suitable for detecting the DNA damage response induced by treatment with temozolomide and methylmethane sulfonate (MMS).  Lastly, the assay has been shown to be robust over a period of several years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSzT50tY2lEhWBAiYWy6n3SfW6udTcc2ebudzle-KwYSLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szptVyqsA%253D%253D&md5=e51d923a92a7a1fd09c578c2ee9cf917</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.12688%2Ff1000research.8463.2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.12688%252Ff1000research.8463.2%26sid%3Dliteratum%253Aachs%26aulast%3DJames%26aufirst%3DD.%2BI.%26aulast%3DDurant%26aufirst%3DS.%26aulast%3DEckersley%26aufirst%3DK.%26aulast%3DFairweather%26aufirst%3DE.%26aulast%3DGriffiths%26aufirst%3DL.%2BA.%26aulast%3DHamilton%26aufirst%3DN.%26aulast%3DKelly%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DM.%26aulast%3DShea%26aufirst%3DK.%26aulast%3DWaddell%26aufirst%3DI.%2BD.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26atitle%3DAn%2520Assay%2520to%2520Measure%2520Poly%2528ADP%2520ribose%2529%2520Glycohydrolase%2520%2528PARG%2529%2520Activity%2520in%2520Cells%26jtitle%3DF1000Research%26date%3D2016%26volume%3D5%26spage%3D736%26epage%3D746%26doi%3D10.12688%2Ff1000research.8463.2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gean, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katzhendle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Shoshan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ringel, I.</span></span> <span> </span><span class="NLM_article-title">Preparation, Characterization, and Anticancer Activity of a Series of <i>cis</i>-PtCl<sub>2</sub> Complexes Linked to Anthraquinone Intercalators</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">414</span>– <span class="NLM_lpage">420</span>, <span class="refDoi"> DOI: 10.1021/jm00105a063</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00105a063" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADyaK3MXlsVOgtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1991&pages=414-420&author=D.+Gibsonauthor=K.+F.+Geanauthor=J.+Katzhendleauthor=R.+Ben-Shoshanauthor=A.+Ramuauthor=I.+Ringel&title=Preparation%2C+Characterization%2C+and+Anticancer+Activity+of+a+Series+of+cis-PtCl2+Complexes+Linked+to+Anthraquinone+Intercalators&doi=10.1021%2Fjm00105a063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Preparation, characterization, and anticancer activity of a series of cis-PtCl2 complexes linked to anthraquinone intercalators</span></div><div class="casAuthors">Gibson, Dan; Gean, Keria Fiorella; Katzhendle, Jehoshua; Ben-Shoshan, Raphael; Ramu, Avner; Ringel, Israel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">414-20</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">cis-PtL2X2 [L = a monodentate AQZ(CH2)nNH2 and L2 = a bidentate AQ-Z(CH2)nNH(CH2)2NH2; AQ = anthraquinonyl, X = Cl, I, Z = NH, O] in which anthraquinone intercalators are tethered to the cis-PtCl2 unit via an (aminoalkyl)amino, (oxyalkyl)amino, or polyethylene glycol (aminoethyl)amino linker chains was prepd. and screened in vitro against P388 leukemia.  In vivo toxicity studies were carried out on selected complexes.  All complexes were characterized by means of elemental anal., and 195Pt NMR and FTIR spectroscopies.  The 1:1 Pt-intercalcalator complexes displayed much higher in vitro cytotoxic activities than the 1:2 Pt-intercalator complexes.  The X = Cl complexes were consistently more active than their X = I counterparts.  Among the 1:1 Pt-intercalator complexes those with the shorter linker chains (n = 2, 3) exhibited the highest cytotoxic activities.  cis-PtL2Cl2 (L = 2-[[2-(anthraquinon-1-ylamino)ethyl]amino]ethyl]amine, C2-[[3-(anthraquinon-1-ylamino)propyl]amino]ethyl]amine, and [2-[[3-(anthraquinon-1-yloxy)propyl]amino]ethyl]amine-N,N']) were as active in vitro as cisplatin (ED50 = 2-4 × 10-7 M) while on a molar basis their acute in vivo toxicity was lower than that of cisplatin.  In vivo screening against P388 leukemia indicated that these complexes have activity comparable to cisplatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtfcLsaH7LE7Vg90H21EOLACvtfcHk0lhdzP7ar97Z-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXlsVOgtw%253D%253D&md5=170132b6f7c62b91af0abd77ae6715e9</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm00105a063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00105a063%26sid%3Dliteratum%253Aachs%26aulast%3DGibson%26aufirst%3DD.%26aulast%3DGean%26aufirst%3DK.%2BF.%26aulast%3DKatzhendle%26aufirst%3DJ.%26aulast%3DBen-Shoshan%26aufirst%3DR.%26aulast%3DRamu%26aufirst%3DA.%26aulast%3DRingel%26aufirst%3DI.%26atitle%3DPreparation%252C%2520Characterization%252C%2520and%2520Anticancer%2520Activity%2520of%2520a%2520Series%2520of%2520cis-PtCl2%2520Complexes%2520Linked%2520to%2520Anthraquinone%2520Intercalators%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1991%26volume%3D34%26spage%3D414%26epage%3D420%26doi%3D10.1021%2Fjm00105a063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="note"><p class="first last">Lipophilic ligand efficiency calculated as biochemical pEC<sub>50</sub> – XlogP.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Talele, T. T.</span></span> <span> </span><span class="NLM_article-title">The “Cyclopropyl Fragment” is a Versatile Player that Frequently Appears in Preclinical/Clinical Drug Molecules</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8712</span>– <span class="NLM_lpage">8756</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00472</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00472" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslWguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8712-8756&author=T.+T.+Talele&title=The+%E2%80%9CCyclopropyl+Fragment%E2%80%9D+is+a+Versatile+Player+that+Frequently+Appears+in+Preclinical%2FClinical+Drug+Molecules&doi=10.1021%2Facs.jmedchem.6b00472"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">The "Cyclopropyl Fragment" is a Versatile Player that Frequently Appears in Preclinical/Clinical Drug Molecules</span></div><div class="casAuthors">Talele, Tanaji T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8712-8756</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, there has been an increasing use of the cyclopropyl ring in drug development to transition drug candidates from the preclin. to clin. stage.  Important features of the cyclopropane ring are, the (1) coplanarity of the three carbon atoms, (2) relatively shorter (1.51 Å) C-C bonds, (3) enhanced π-character of C-C bonds, and (4) C-H bonds are shorter and stronger than those in alkanes.  The present review will focus on the contributions that a cyclopropyl ring makes to the properties of drugs contg. it.  Consequently, the cyclopropyl ring addresses multiple roadblocks that can occur during drug discovery such as (a) enhancing potency, (b) reducing off-target effects, (c) increasing metabolic stability, (d) increasing brain permeability, (e) decreasing plasma clearance, (f) contributing to an entropically more favorable binding to the receptor, (g) conformational restriction of peptides/peptidomimetics to prevent proteolytic hydrolysis, and (h) altering drug pKa to reduce its P-glycoprotein efflux ratio.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVIJAG1XVRG7Vg90H21EOLACvtfcHk0lhdzP7ar97Z-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslWguro%253D&md5=b5586458ec4ab5e2d7586f5739663d1f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00472%26sid%3Dliteratum%253Aachs%26aulast%3DTalele%26aufirst%3DT.%2BT.%26atitle%3DThe%2520%25E2%2580%259CCyclopropyl%2520Fragment%25E2%2580%259D%2520is%2520a%2520Versatile%2520Player%2520that%2520Frequently%2520Appears%2520in%2520Preclinical%252FClinical%2520Drug%2520Molecules%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8712%26epage%3D8756%26doi%3D10.1021%2Facs.jmedchem.6b00472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span> <i>PARG inhibitor
PDD00017273 Small Molecule (Tool Compound)</i>; <span class="NLM_publisher-name">Ximbio</span>, <span class="NLM_year">2018</span>; <a href="https://ximbio.com/reagent/152787/parg-inhibitor-pdd00017273-small-molecule-tool-compound" class="extLink">https://ximbio.com/reagent/152787/parg-inhibitor-pdd00017273-small-molecule-tool-compound</a> (accessed Oct 24, 2018.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+PARG+inhibitor%0APDD00017273+Small+Molecule+%28Tool+Compound%29%3B+Ximbio%2C+2018%3B+https%3A%2F%2Fximbio.com%2Freagent%2F152787%2Fparg-inhibitor-pdd00017273-small-molecule-tool-compound+%28accessed+Oct+24%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DPARG%2520inhibitor%250APDD00017273%2520Small%2520Molecule%2520%2528Tool%2520Compound%2529%26pub%3DXimbio%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span> See, for example<span class="NLM_contrib-group"><span class="NLM_string-name">Gogola, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duarte, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Ruiter, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiegant, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bruijn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerrero Llobet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vis, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annunziato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den Broek, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barazas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kersbergen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van de Ven, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarsounas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vugt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wessels, L. F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartkova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gromova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andújar-Sánchez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Attikum, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonkers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rottenberg, S.</span></span> <span> </span><span class="NLM_article-title">Selective Loss of PARG Restores PARylation and Counteracts PARP Inhibitor-Mediated Synthetic Lethality</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1078</span>– <span class="NLM_lpage">1093</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2018.05.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1016%2Fj.ccell.2018.05.008" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2018&pages=1078-1093&author=E.+Gogolaauthor=A.+A.+Duarteauthor=J.+R.+de+Ruiterauthor=W.+W.+Wiegantauthor=J.+A.+Schmidauthor=R.+de+Bruijnauthor=D.+I.+Jamesauthor=S.+Guerrero+Llobetauthor=D.+J.+Visauthor=S.+Annunziatoauthor=B.+van+den+Broekauthor=M.+Barazasauthor=A.+Kersbergenauthor=M.+van+de+Venauthor=M.+Tarsounasauthor=D.+J.+Ogilvieauthor=M.+van+Vugtauthor=L.+F.+A.+Wesselsauthor=J.+Bartkovaauthor=I.+Gromovaauthor=M.+And%C3%BAjar-S%C3%A1nchezauthor=J.+Bartekauthor=M.+Lopesauthor=H.+van+Attikumauthor=P.+Borstauthor=J.+Jonkersauthor=S.+Rottenberg&title=Selective+Loss+of+PARG+Restores+PARylation+and+Counteracts+PARP+Inhibitor-Mediated+Synthetic+Lethality&doi=10.1016%2Fj.ccell.2018.05.008"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2018.05.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2018.05.008%26sid%3Dliteratum%253Aachs%26aulast%3DGogola%26aufirst%3DE.%26aulast%3DDuarte%26aufirst%3DA.%2BA.%26aulast%3Dde%2BRuiter%26aufirst%3DJ.%2BR.%26aulast%3DWiegant%26aufirst%3DW.%2BW.%26aulast%3DSchmid%26aufirst%3DJ.%2BA.%26aulast%3Dde%2BBruijn%26aufirst%3DR.%26aulast%3DJames%26aufirst%3DD.%2BI.%26aulast%3DGuerrero%2BLlobet%26aufirst%3DS.%26aulast%3DVis%26aufirst%3DD.%2BJ.%26aulast%3DAnnunziato%26aufirst%3DS.%26aulast%3Dvan%2Bden%2BBroek%26aufirst%3DB.%26aulast%3DBarazas%26aufirst%3DM.%26aulast%3DKersbergen%26aufirst%3DA.%26aulast%3Dvan%2Bde%2BVen%26aufirst%3DM.%26aulast%3DTarsounas%26aufirst%3DM.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26aulast%3Dvan%2BVugt%26aufirst%3DM.%26aulast%3DWessels%26aufirst%3DL.%2BF.%2BA.%26aulast%3DBartkova%26aufirst%3DJ.%26aulast%3DGromova%26aufirst%3DI.%26aulast%3DAnd%25C3%25BAjar-S%25C3%25A1nchez%26aufirst%3DM.%26aulast%3DBartek%26aufirst%3DJ.%26aulast%3DLopes%26aufirst%3DM.%26aulast%3Dvan%2BAttikum%26aufirst%3DH.%26aulast%3DBorst%26aufirst%3DP.%26aulast%3DJonkers%26aufirst%3DJ.%26aulast%3DRottenberg%26aufirst%3DS.%26atitle%3DSelective%2520Loss%2520of%2520PARG%2520Restores%2520PARylation%2520and%2520Counteracts%2520PARP%2520Inhibitor-Mediated%2520Synthetic%2520Lethality%26jtitle%3DCancer%2520Cell%26date%3D2018%26volume%3D33%26spage%3D1078%26epage%3D1093%26doi%3D10.1016%2Fj.ccell.2018.05.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="note"><p class="first last">See <a href="/doi/suppl/10.1021/acs.jmedchem.8b01407/suppl_file/jm8b01407_si_001.pdf" class="ext-link">Supporting Information</a> for correlation between biochemical and cellular pEC<sub>50</sub>s for the whole quinazolinedione series.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span> <i>PDD00017273</i>; <span class="NLM_publisher-name">Chemical
Probes Portal</span>, <span class="NLM_year">2018</span>; <a href="http://www.chemicalprobes.org/pdd00017273" class="extLink">http://www.chemicalprobes.org/pdd00017273</a> (accessed Oct 24, 2018).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+PDD00017273%3B+Chemical%0AProbes+Portal%2C+2018%3B+http%3A%2F%2Fwww.chemicalprobes.org%2Fpdd00017273+%28accessed+Oct+24%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DPDD00017273%26pub%3DChemical%250AProbes%2520Portal%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span>XLogP calculated
using <i>Dotmatics Elemental
Suite</i>; <span class="NLM_publisher-name">Dotmatics</span>: <span class="NLM_publisher-name">Bishops Stortford, UK</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=XLogP+calculated%0Ausing+Dotmatics+Elemental%0ASuite%3B+Dotmatics%3A+Bishops+Stortford%2C+UK."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DDotmatics%2520Elemental%250ASuite%26pub%3DDotmatics%26pub%3DBishops%2520Stortford%252C%2520UK" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, D. M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monaco, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winters, M. P.</span></span> <span> </span><span class="NLM_article-title">New N- and O-arylations with Phenylboronic Acids and Cupric Acetate</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">2933</span>– <span class="NLM_lpage">2936</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(98)00503-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1016%2FS0040-4039%2898%2900503-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADyaK1cXjtVOlu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1998&pages=2933-2936&author=D.+M.+T.+Chanauthor=K.+L.+Monacoauthor=R.-P.+Wangauthor=M.+P.+Winters&title=New+N-+and+O-arylations+with+Phenylboronic+Acids+and+Cupric+Acetate&doi=10.1016%2FS0040-4039%2898%2900503-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">New N- and O-arylation with phenylboronic acids and cupric acetate</span></div><div class="casAuthors">Chan, Dominic M. T.; Monaco, Kevin L.; Wang, Ru-Ping; Winters, Michael P.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2933-2936</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A new method of arylating N-H and O-H contg. compds. at room temp. with phenylboronic acids and cupric acetate in the presence of a tertiary amine promoter is described.  Substrates include phenols, amines, anilines, amides, imides, ureas, carbamates, and sulfonamides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQZxWi8FJntLVg90H21EOLACvtfcHk0lhdqw1ypegZgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjtVOlu7w%253D&md5=56efb58adc883aba3ca4ab908f6864ec</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2898%2900503-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252898%252900503-6%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DD.%2BM.%2BT.%26aulast%3DMonaco%26aufirst%3DK.%2BL.%26aulast%3DWang%26aufirst%3DR.-P.%26aulast%3DWinters%26aufirst%3DM.%2BP.%26atitle%3DNew%2520N-%2520and%2520O-arylations%2520with%2520Phenylboronic%2520Acids%2520and%2520Cupric%2520Acetate%26jtitle%3DTetrahedron%2520Lett.%26date%3D1998%26volume%3D39%26spage%3D2933%26epage%3D2936%26doi%3D10.1016%2FS0040-4039%2898%2900503-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McLeod, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boudreault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, Y.</span></span> <span> </span><span class="NLM_article-title">Synthetic Application of Monoprotected Hydrazines toward the Synthesis of 1-Aminopyrroles</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1180</span>– <span class="NLM_lpage">1183</span>, <span class="refDoi"> DOI: 10.1021/jo9518260</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo9518260" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=1996&pages=1180-1183&author=M.+McLeodauthor=N.+Boudreaultauthor=Y.+Leblanc&title=Synthetic+Application+of+Monoprotected+Hydrazines+toward+the+Synthesis+of+1-Aminopyrroles&doi=10.1021%2Fjo9518260"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjo9518260&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo9518260%26sid%3Dliteratum%253Aachs%26aulast%3DMcLeod%26aufirst%3DM.%26aulast%3DBoudreault%26aufirst%3DN.%26aulast%3DLeblanc%26aufirst%3DY.%26atitle%3DSynthetic%2520Application%2520of%2520Monoprotected%2520Hydrazines%2520toward%2520the%2520Synthesis%2520of%25201-Aminopyrroles%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1996%26volume%3D61%26spage%3D1180%26epage%3D1183%26doi%3D10.1021%2Fjo9518260" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':[],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':[],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 13 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Lloyd</span>. </span><span class="cited-content_cbyCitation_article-title">High-Throughput Screening for the Discovery of Enzyme Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (19)
                                     , 10742-10772. <a href="https://doi.org/10.1021/acs.jmedchem.0c00523" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00523</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00523%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHigh-Throughput%252BScreening%252Bfor%252Bthe%252BDiscovery%252Bof%252BEnzyme%252BInhibitors%26aulast%3DLloyd%26aufirst%3DMatthew%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30032020%26date%3D10062020%26date%3D20052020%26volume%3D63%26issue%3D19%26spage%3D10742%26epage%3D10772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Irene  Paterniti</span>, <span class="hlFld-ContribAuthor ">Sarah Adriana  Scuderi</span>, <span class="hlFld-ContribAuthor ">Giovanna  Casili</span>, <span class="hlFld-ContribAuthor ">Marika  Lanza</span>, <span class="hlFld-ContribAuthor ">Marzia  Mare</span>, <span class="hlFld-ContribAuthor ">Raffella  Giuffrida</span>, <span class="hlFld-ContribAuthor ">Cristina  Colarossi</span>, <span class="hlFld-ContribAuthor ">Marco  Portelli</span>, <span class="hlFld-ContribAuthor ">Salvatore  Cuzzocrea</span>, <span class="hlFld-ContribAuthor ">Emanuela  Esposito</span>. </span><span class="cited-content_cbyCitation_article-title">Poly (ADP-Ribose) Polymerase Inhibitor, ABT888, Improved Cisplatin Effect in Human Oral Cell Carcinoma. </span><span class="cited-content_cbyCitation_journal-name">Biomedicines</span><span> <strong>2021,</strong> <em>9 </em>
                                    (7)
                                     , 771. <a href="https://doi.org/10.3390/biomedicines9070771" title="DOI URL">https://doi.org/10.3390/biomedicines9070771</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biomedicines9070771&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiomedicines9070771%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomedicines%26atitle%3DPoly%252B%252528ADP-Ribose%252529%252BPolymerase%252BInhibitor%25252C%252BABT888%25252C%252BImproved%252BCisplatin%252BEffect%252Bin%252BHuman%252BOral%252BCell%252BCarcinoma%26aulast%3DPaterniti%26aufirst%3DIrene%26date%3D2021%26date%3D2021%26volume%3D9%26issue%3D7%26spage%3D771" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wei  Fu</span>, <span class="hlFld-ContribAuthor ">Huiqiao  Yao</span>, <span class="hlFld-ContribAuthor ">Mareike  Bütepage</span>, <span class="hlFld-ContribAuthor ">Qianqian  Zhao</span>, <span class="hlFld-ContribAuthor ">Bernhard  Lüscher</span>, <span class="hlFld-ContribAuthor ">Jinyu  Li</span>. </span><span class="cited-content_cbyCitation_article-title">The search for inhibitors of macrodomains for targeting the readers and erasers of mono-ADP-ribosylation. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2021,</strong> <em>15 </em><a href="https://doi.org/10.1016/j.drudis.2021.05.007" title="DOI URL">https://doi.org/10.1016/j.drudis.2021.05.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2021.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2021.05.007%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DThe%252Bsearch%252Bfor%252Binhibitors%252Bof%252Bmacrodomains%252Bfor%252Btargeting%252Bthe%252Breaders%252Band%252Berasers%252Bof%252Bmono-ADP-ribosylation%26aulast%3DFu%26aufirst%3DWei%26date%3D2021%26volume%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Manqi  Zhang</span>, <span class="hlFld-ContribAuthor ">Yanhao  Lai</span>, <span class="hlFld-ContribAuthor ">Judy L.  Vasquez</span>, <span class="hlFld-ContribAuthor ">Dominic I.  James</span>, <span class="hlFld-ContribAuthor ">Kate M.  Smith</span>, <span class="hlFld-ContribAuthor ">Ian D.  Waddell</span>, <span class="hlFld-ContribAuthor ">Donald J.  Ogilvie</span>, <span class="hlFld-ContribAuthor ">Yuan  Liu</span>, <span class="hlFld-ContribAuthor ">Irina U.  Agoulnik</span>. </span><span class="cited-content_cbyCitation_article-title">Androgen Receptor and Poly(ADP-ribose) Glycohydrolase Inhibition Increases Efficiency of Androgen Ablation in Prostate Cancer Cells. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2020,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-020-60849-y" title="DOI URL">https://doi.org/10.1038/s41598-020-60849-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-020-60849-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-020-60849-y%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DAndrogen%252BReceptor%252Band%252BPoly%252528ADP-ribose%252529%252BGlycohydrolase%252BInhibition%252BIncreases%252BEfficiency%252Bof%252BAndrogen%252BAblation%252Bin%252BProstate%252BCancer%252BCells%26aulast%3DZhang%26aufirst%3DManqi%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Johannes Gregor Matthias  Rack</span>, <span class="hlFld-ContribAuthor ">Luca  Palazzo</span>, <span class="hlFld-ContribAuthor ">Ivan  Ahel</span>. </span><span class="cited-content_cbyCitation_article-title">(ADP-ribosyl)hydrolases: structure, function, and biology. </span><span class="cited-content_cbyCitation_journal-name">Genes & Development</span><span> <strong>2020,</strong> <em>34 </em>
                                    (5-6)
                                     , 263-284. <a href="https://doi.org/10.1101/gad.334631.119" title="DOI URL">https://doi.org/10.1101/gad.334631.119</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1101/gad.334631.119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1101%2Fgad.334631.119%26sid%3Dliteratum%253Aachs%26jtitle%3DGenes%2520%2526%2520Development%26atitle%3D%252528ADP-ribosyl%252529hydrolases%25253A%252Bstructure%25252C%252Bfunction%25252C%252Band%252Bbiology%26aulast%3DRack%26aufirst%3DJohannes%2BGregor%2BMatthias%26date%3D2020%26date%3D2020%26volume%3D34%26issue%3D5-6%26spage%3D263%26epage%3D284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuto  Shibui</span>, <span class="hlFld-ContribAuthor ">Takahiro  Oyama</span>, <span class="hlFld-ContribAuthor ">Miwa  Okazawa</span>, <span class="hlFld-ContribAuthor ">Atsushi  Yoshimori</span>, <span class="hlFld-ContribAuthor ">Hideaki  Abe</span>, <span class="hlFld-ContribAuthor ">Fumiaki  Uchiumi</span>, <span class="hlFld-ContribAuthor ">Sei-ichi  Tanuma</span>. </span><span class="cited-content_cbyCitation_article-title">Structural insights into the active site of poly(ADP-ribose) glycohydrolase using docking modes of 6-hydroxy-3H-xanthen-3-one derivative inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2020,</strong> <em>28 </em>
                                    (3)
                                     , 115249. <a href="https://doi.org/10.1016/j.bmc.2019.115249" title="DOI URL">https://doi.org/10.1016/j.bmc.2019.115249</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2019.115249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2019.115249%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DStructural%252Binsights%252Binto%252Bthe%252Bactive%252Bsite%252Bof%252Bpoly%252528ADP-ribose%252529%252Bglycohydrolase%252Busing%252Bdocking%252Bmodes%252Bof%252B6-hydroxy-3H-xanthen-3-one%252Bderivative%252Binhibitors%26aulast%3DShibui%26aufirst%3DYuto%26date%3D2020%26volume%3D28%26issue%3D3%26spage%3D115249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Arindam  Chatterjee</span>. </span><span class="cited-content_cbyCitation_article-title">Computational methods and tools for sustainable and green approaches in drug discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 965-988. <a href="https://doi.org/10.1016/B978-0-12-817592-7.00027-7" title="DOI URL">https://doi.org/10.1016/B978-0-12-817592-7.00027-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-817592-7.00027-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-817592-7.00027-7%26sid%3Dliteratum%253Aachs%26atitle%3DComputational%252Bmethods%252Band%252Btools%252Bfor%252Bsustainable%252Band%252Bgreen%252Bapproaches%252Bin%252Bdrug%252Bdiscovery%26aulast%3DChatterjee%26aufirst%3DArindam%26date%3D2020%26spage%3D965%26epage%3D988%26pub%3DElsevier%26atitle%3DGreen%252BApproaches%252Bin%252BMedicinal%252BChemistry%252Bfor%252BSustainable%252BDrug%252BDesign%26date%3D2020%26volume%3D47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">K. Alison  Rinderspacher</span>. </span><span class="cited-content_cbyCitation_article-title">Six-Membered Ring Systems: Diazines and Benzo Derivatives. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 465-504. <a href="https://doi.org/10.1016/B978-0-12-819962-6.00013-0" title="DOI URL">https://doi.org/10.1016/B978-0-12-819962-6.00013-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-819962-6.00013-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-819962-6.00013-0%26sid%3Dliteratum%253Aachs%26atitle%3DSix-Membered%252BRing%252BSystems%25253A%252BDiazines%252Band%252BBenzo%252BDerivatives%26aulast%3DRinderspacher%26aufirst%3DK.%2BAlison%26date%3D2020%26spage%3D465%26epage%3D504%26pub%3DElsevier%26date%3D2020%26volume%3D31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bijendra  Khadka</span>, <span class="hlFld-ContribAuthor ">Tonuka  Chatterjee</span>, <span class="hlFld-ContribAuthor ">Bhagwati P.  Gupta</span>, <span class="hlFld-ContribAuthor ">Radhey S.  Gupta</span>. </span><span class="cited-content_cbyCitation_article-title">Genomic Analyses Identify Novel Molecular Signatures Specific for the Caenorhabditis and other Nematode Taxa Providing Novel Means for Genetic and Biochemical Studies. </span><span class="cited-content_cbyCitation_journal-name">Genes</span><span> <strong>2019,</strong> <em>10 </em>
                                    (10)
                                     , 739. <a href="https://doi.org/10.3390/genes10100739" title="DOI URL">https://doi.org/10.3390/genes10100739</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/genes10100739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fgenes10100739%26sid%3Dliteratum%253Aachs%26jtitle%3DGenes%26atitle%3DGenomic%252BAnalyses%252BIdentify%252BNovel%252BMolecular%252BSignatures%252BSpecific%252Bfor%252Bthe%252BCaenorhabditis%252Band%252Bother%252BNematode%252BTaxa%252BProviding%252BNovel%252BMeans%252Bfor%252BGenetic%252Band%252BBiochemical%252BStudies%26aulast%3DKhadka%26aufirst%3DBijendra%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D10%26spage%3D739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aditi  Jain</span>, <span class="hlFld-ContribAuthor ">Lebaron C.  Agostini</span>, <span class="hlFld-ContribAuthor ">Grace A.  McCarthy</span>, <span class="hlFld-ContribAuthor ">Saswati N.  Chand</span>, <span class="hlFld-ContribAuthor ">AnnJosette  Ramirez</span>, <span class="hlFld-ContribAuthor ">Avinoam  Nevler</span>, <span class="hlFld-ContribAuthor ">Joseph  Cozzitorto</span>, <span class="hlFld-ContribAuthor ">Christopher W.  Schultz</span>, <span class="hlFld-ContribAuthor ">Cinthya Yabar  Lowder</span>, <span class="hlFld-ContribAuthor ">Kate M.  Smith</span>, <span class="hlFld-ContribAuthor ">Ian D.  Waddell</span>, <span class="hlFld-ContribAuthor ">Maria  Raitses-Gurevich</span>, <span class="hlFld-ContribAuthor ">Chani  Stossel</span>, <span class="hlFld-ContribAuthor ">Yulia Glick  Gorman</span>, <span class="hlFld-ContribAuthor ">Dikla  Atias</span>, <span class="hlFld-ContribAuthor ">Charles J.  Yeo</span>, <span class="hlFld-ContribAuthor ">Jordan M.  Winter</span>, <span class="hlFld-ContribAuthor ">Kenneth P.  Olive</span>, <span class="hlFld-ContribAuthor ">Talia  Golan</span>, <span class="hlFld-ContribAuthor ">Michael J.  Pishvaian</span>, <span class="hlFld-ContribAuthor ">Donald  Ogilvie</span>, <span class="hlFld-ContribAuthor ">Dominic I.  James</span>, <span class="hlFld-ContribAuthor ">Allan M.  Jordan</span>, <span class="hlFld-ContribAuthor ">Jonathan R.  Brody</span>. </span><span class="cited-content_cbyCitation_article-title">Poly (ADP) Ribose Glycohydrolase Can Be Effectively Targeted in Pancreatic Cancer. </span><span class="cited-content_cbyCitation_journal-name">Cancer Research</span><span> <strong>2019,</strong> <em>79 </em>
                                    (17)
                                     , 4491-4502. <a href="https://doi.org/10.1158/0008-5472.CAN-18-3645" title="DOI URL">https://doi.org/10.1158/0008-5472.CAN-18-3645</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/0008-5472.CAN-18-3645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F0008-5472.CAN-18-3645%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Research%26atitle%3DPoly%252B%252528ADP%252529%252BRibose%252BGlycohydrolase%252BCan%252BBe%252BEffectively%252BTargeted%252Bin%252BPancreatic%252BCancer%26aulast%3DJain%26aufirst%3DAditi%26date%3D2019%26date%3D2019%26volume%3D79%26issue%3D17%26spage%3D4491%26epage%3D4502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sei-ichi  Tanuma</span>, <span class="hlFld-ContribAuthor ">Yuto  Shibui</span>, <span class="hlFld-ContribAuthor ">Takahiro  Oyama</span>, <span class="hlFld-ContribAuthor ">Fumiaki  Uchiumi</span>, <span class="hlFld-ContribAuthor ">Hideaki  Abe</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting poly(ADP-ribose) glycohydrolase to draw apoptosis codes in cancer. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2019,</strong> <em>167 </em>, 163-172. <a href="https://doi.org/10.1016/j.bcp.2019.06.004" title="DOI URL">https://doi.org/10.1016/j.bcp.2019.06.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2019.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2019.06.004%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DTargeting%252Bpoly%252528ADP-ribose%252529%252Bglycohydrolase%252Bto%252Bdraw%252Bapoptosis%252Bcodes%252Bin%252Bcancer%26aulast%3DTanuma%26aufirst%3DSei-ichi%26date%3D2019%26volume%3D167%26spage%3D163%26epage%3D172" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olivia  Slater</span>, <span class="hlFld-ContribAuthor ">Maria  Kontoyianni</span>. </span><span class="cited-content_cbyCitation_article-title">The compromise of virtual screening and its impact on drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2019,</strong> <em>14 </em>
                                    (7)
                                     , 619-637. <a href="https://doi.org/10.1080/17460441.2019.1604677" title="DOI URL">https://doi.org/10.1080/17460441.2019.1604677</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2019.1604677&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2019.1604677%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DThe%252Bcompromise%252Bof%252Bvirtual%252Bscreening%252Band%252Bits%252Bimpact%252Bon%252Bdrug%252Bdiscovery%26aulast%3DSlater%26aufirst%3DOlivia%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D7%26spage%3D619%26epage%3D637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ben  Acton</span>, <span class="hlFld-ContribAuthor ">Helen F.  Small</span>, <span class="hlFld-ContribAuthor ">Kate M.  Smith</span>, <span class="hlFld-ContribAuthor ">Alison  McGonagle</span>, <span class="hlFld-ContribAuthor ">Alexandra I.J.  Stowell</span>, <span class="hlFld-ContribAuthor ">Dominic I.  James</span>, <span class="hlFld-ContribAuthor ">Niall M.  Hamilton</span>, <span class="hlFld-ContribAuthor ">Nicola  Hamilton</span>, <span class="hlFld-ContribAuthor ">James R.  Hitchin</span>, <span class="hlFld-ContribAuthor ">Colin P.  Hutton</span>, <span class="hlFld-ContribAuthor ">Ian D.  Waddell</span>, <span class="hlFld-ContribAuthor ">Donald J.  Ogilvie</span>, <span class="hlFld-ContribAuthor ">Allan M.  Jordan</span>. </span><span class="cited-content_cbyCitation_article-title">Fluoromethylcyclopropylamine derivatives as potential in vivo toxicophores – A cautionary disclosure. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (4)
                                     , 560-562. <a href="https://doi.org/10.1016/j.bmcl.2018.12.066" title="DOI URL">https://doi.org/10.1016/j.bmcl.2018.12.066</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2018.12.066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2018.12.066%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DFluoromethylcyclopropylamine%252Bderivatives%252Bas%252Bpotential%252Bin%252Bvivo%252Btoxicophores%252B%2525E2%252580%252593%252BA%252Bcautionary%252Bdisclosure%26aulast%3DActon%26aufirst%3DBen%26date%3D2019%26volume%3D29%26issue%3D4%26spage%3D560%26epage%3D562" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0030.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0030.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Consequences of PARG inhibition in DNA SSB repair.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Reported PARG inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Analogues <b>8a</b>–<b>o</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents: (a) SOCl<sub>2</sub>, DMF, reflux, 53%; (b) RNH<sub>2</sub>, Et<sub>3</sub>N, DCM, 15%–quant.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>12a</b>–<b>c</b>, <b>13a</b>,<b>b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents: (a) RI, K<sub>2</sub>CO<sub>3</sub>, DMF, 30–41%; (b) ClSO<sub>3</sub>H, 60 °C, 64–67%; (c) R′NH<sub>2</sub>, Et<sub>3</sub>N, DCM, 18–76%.</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>16</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents: (a) ClSO<sub>3</sub>H, 60 °C, 70%; (b) RNH<sub>2</sub>, <i>i</i>-Pr<sub>2</sub>NEt, THF, DCM, 43%.</p></p></figure><figure data-id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Analogues <b>22a</b>–<b>t</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents: (a) CDI, DMF, 140 °C, 90%; (b) ClSO<sub>3</sub>H, 60 °C, 99%; (c) 1-methylcyclopropanamine hydrochloride, Et<sub>3</sub>N, DCM, 89%; (d) Ac<sub>2</sub>O, pyridine, DMAP, 83%; (e) BrCH<sub>2</sub>CN, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 75%; (f) R-X (where X = I or Br), K<sub>2</sub>CO<sub>3</sub>, DMF, then NH<sub>3</sub>, THF, 28–88%; (g) R-X (where X = Cl, Br or I), K<sub>2</sub>CO<sub>3</sub>, DMF, then K<sub>2</sub>CO<sub>3</sub>, MeOH 20–32%; (h) R-OH, DIAD, PS-PPh<sub>3</sub>, DMF, then K<sub>2</sub>CO<sub>3</sub>, MeOH, 93%.</p></p></figure><figure data-id="sch5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0015.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of <b>27a</b>–<b>at</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaOCN, NaOH, AcOH, H<sub>2</sub>O, 50 °C, 1 h, 79%; (b) ClSO<sub>3</sub>H, 50 °C, 16 h, quant; (c) amine <b>28</b>, Et<sub>3</sub>N, DCM, RT, 3 h, 93%; (d) (i) NaH, DMF, RT, 5 min, (ii) R-X (X = Br, Cl, I, OTs), RT, 6–59%; (e) R-OH, DIAD (or DCAD), PS-PPh<sub>3</sub>, DMF, RT, 10–21%; (f) R-B(OH)<sub>2</sub>, Cu(OAc)<sub>2</sub>, Et<sub>3</sub>N, 4 Å molecular sieves, DCM, RT, 18%.</p></p></figure><figure data-id="sch6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0016.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of <b>33</b>–<b>36</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) ClSO<sub>3</sub>H, 50 °C, 4 h, 79%; (b) (i) amine <b>28</b>, Et<sub>3</sub>N, DCM, −10 °C, 30 min, (ii) R<sub>2</sub>-NH<sub>2</sub>, Et<sub>3</sub>N, −10 °C–RT, 18 h, 20–67%; (c) triphosgene, THF, 0 °C–RT, 18 h 78–97%; (d) (i) NaH, DMF, RT, 5 min, (ii) R<sub>1</sub>-X (X = Br, I, OMs, OTs), RT; (e) R<sub>1</sub>-OH, DIAD (or DCAD), PS-PPh<sub>3</sub>, DMF, RT, 16–75%.</p></p></figure><figure data-id="sch7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0017.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of <b>41a</b> and <b>41e</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) amine <b>42</b>, Et<sub>3</sub>N, DMF, 60 °C for 5 h, then RT for 16 h, 65%; (b) triphosgene, THF, 0 °C–RT, 18 h, 58%; (c) ClSO<sub>3</sub>H, 50 °C, 2 h, 69%; (d) amine <b>28</b>, Et<sub>3</sub>N, DCM, 0 °C–RT, 18 h, 32%; (e) EtI, K<sub>2</sub>CO<sub>3</sub>, DMF, μW, 80 °C, 20 min, 20% for <b>41a</b>, and 5-(chloromethyl)-1,3-dimethyl-1H-pyrazole, K<sub>2</sub>CO<sub>3</sub>, DMF, μW, 80 °C, 20 min, 44% for <b>41e</b>.</p></p></figure><figure data-id="fig3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Anthraquinone HTS hit <b>8a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Comparison of the X-ray crystal structures of the catalytic domain of (A) human PARG bound to <b>8a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMM">6HMM</a>) and (B) ADP-ribose (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4B1H">4B1H</a>), shown in the same coordinate frame. Protein solvent-accessible surfaces are colored by electrostatic potential. (C) Detail of relative binding orientations of <b>8a</b> (cyan carbon atoms) and ADP-ribose (green) highlights movement of Phe902. (D) <b>8a</b> and ADP ribose bound to entire PARG domain, colored by atom type.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Detail of the hydrogen-bonding environment around <b>8a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMM">6HMM</a>). (B) Protein solvent-accessible surface highlights the exposure to solvent of the anthraquinone core and the buried pocket around the <i>tert</i>-butyl moiety. Hydrogen atoms modeled using Maestro (Schrödinger LLC). (C) Protein–ligand interaction diagram with hydrogen bonds indicated by pink arrows and aromatic interactions by green arrows.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Predicted binding modes of quinazolinedione <b>12b</b> (docked into PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMM">6HMM</a>). The N1 position presents a vector toward solvent and the shelf adjacent to Phe902, while the N3 position presents a vector toward the pocket occupied by the ribose moiety of ADP.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. X-ray crystal structure of <b>22h</b> bound to the catalytic domain of human PARG at 2.9 Å resolution (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMN">6HMN</a>). (A) Interaction of the <i>N</i>1-benzyl moiety with the Phe902 shelf. (B) Detail of the hydrogen-bonding environment around the sulfonamide, with a putative CH···O interaction highlighted by the cyan dashed line. Hydrogen atoms modeled using Maestro (Schrödinger LLC). (C) Ligand interaction diagram colored as in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) Crystal structure of <b>27t</b> bound to human PARG at 2.06 Å resolution (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMK">6HMK</a>). Hydrogen atoms modeled using Maestro (Schrödinger LLC). (B) Protein–ligand interaction diagram with hydrogen bonds indicated by pink arrows and aromatic interactions by green arrows.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Key SAR at the 3-position of the quinazolinedione.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/medium/jm-2018-01407e_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (A) Crystal structure of <b>33f</b> bound to human PARG at 2.25 Å resolution (PBD <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HML">6HML</a>). Hydrogen atoms modeled using Maestro (Schrödinger LLC). (B) Protein–ligand interaction diagram with key hydrogen bond interactions highlighted (pink arrows). (C) Overlay of crystal structures of <b>33f</b> (green carbons and ribbons) with <b>8a</b> (blue carbons and ribbons), showing maintenance of initial binding mode.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01407/20181207/images/large/jm-2018-01407e_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01407&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i184">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45278" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45278" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 41 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">The DNA Damage Response and Cancer Therapy</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>481</i></span>,  <span class="NLM_fpage">287</span>– <span class="NLM_lpage">294</span>, <span class="refDoi"> DOI: 10.1038/nature10760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1038%2Fnature10760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=22258607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC38Xps1GmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=481&publication_year=2012&pages=287-294&author=C.+J.+Lordauthor=A.+Ashworth&title=The+DNA+Damage+Response+and+Cancer+Therapy&doi=10.1038%2Fnature10760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The DNA damage response and cancer therapy</span></div><div class="casAuthors">Lord, Christopher J.; Ashworth, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">481</span>
        (<span class="NLM_cas:issue">7381</span>),
    <span class="NLM_cas:pages">287-294</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Genomic instability is one of the most pervasive characteristics of tumor cells and is probably the combined effect of DNA damage, tumor-specific DNA repair defects, and a failure to stop or stall the cell cycle before the damaged DNA is passed on to daughter cells.  Although these processes drive genomic instability and ultimately the disease process, they also provide therapeutic opportunities.  A better understanding of the cellular response to DNA damage will not only inform our knowledge of cancer development but also help to refine the classification as well as the treatment of the disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUS1sAbqFhmrVg90H21EOLACvtfcHk0lhclybNBB4vhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xps1GmsQ%253D%253D&md5=75d12cdb52e8c9a39bced1611bf90298</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature10760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10760%26sid%3Dliteratum%253Aachs%26aulast%3DLord%26aufirst%3DC.%2BJ.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DThe%2520DNA%2520Damage%2520Response%2520and%2520Cancer%2520Therapy%26jtitle%3DNature%26date%3D2012%26volume%3D481%26spage%3D287%26epage%3D294%26doi%3D10.1038%2Fnature10760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hosoya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyagawa, K.</span></span> <span> </span><span class="NLM_article-title">Targeting DNA Damage Response in Cancer Therapy</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">370</span>– <span class="NLM_lpage">388</span>, <span class="refDoi"> DOI: 10.1111/cas.12366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1111%2Fcas.12366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=24484288" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlvFWitb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2014&pages=370-388&author=N.+Hosoyaauthor=K.+Miyagawa&title=Targeting+DNA+Damage+Response+in+Cancer+Therapy&doi=10.1111%2Fcas.12366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting DNA damage response in cancer therapy</span></div><div class="casAuthors">Hosoya, Noriko; Miyagawa, Kiyoshi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">370-388</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Cancer chemotherapy and radiotherapy are designed to kill cancer cells mostly by inducing DNA damage.  DNA damage is normally recognized and repaired by the intrinsic DNA damage response machinery.  If the damaged lesions are successfully repaired, the cells will survive.  In order to specifically and effectively kill cancer cells by therapies that induce DNA damage, it is important to take advantage of specific abnormalities in the DNA damage response machinery that are present in cancer cells but not in normal cells.  Such properties of cancer cells can provide biomarkers or targets for sensitization.  For example, defects or upregulation of the specific pathways that recognize or repair specific types of DNA damage can serve as biomarkers of favorable or poor response to therapies that induce such types of DNA damage.  Inhibition of a DNA damage response pathway may enhance the therapeutic effects in combination with the DNA-damaging agents.  Moreover, it may also be useful as a monotherapy when it achieves synthetic lethality, in which inhibition of a complementary DNA damage response pathway selectively kills cancer cells that have a defect in a particular DNA repair pathway.  The most striking application of this strategy is the treatment of cancers deficient in homologous recombination by poly(ADP-ribose) polymerase inhibitors.  In this review, we describe the impact of targeting the cancer-specific aberrations in the DNA damage response by explaining how these treatment strategies are currently being evaluated in preclin. or clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNrms7_IuKabVg90H21EOLACvtfcHk0ljGzkwwNaw7vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlvFWitb0%253D&md5=e5484fa4a619052a621c999349ec9b83</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1111%2Fcas.12366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcas.12366%26sid%3Dliteratum%253Aachs%26aulast%3DHosoya%26aufirst%3DN.%26aulast%3DMiyagawa%26aufirst%3DK.%26atitle%3DTargeting%2520DNA%2520Damage%2520Response%2520in%2520Cancer%2520Therapy%26jtitle%3DCancer%2520Sci.%26date%3D2014%26volume%3D105%26spage%3D370%26epage%3D388%26doi%3D10.1111%2Fcas.12366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, M. J.</span></span> <span> </span><span class="NLM_article-title">Targeting the DNA Damage Response in Cancer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.10.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1016%2Fj.molcel.2015.10.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=26590714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyqtbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2015&pages=547-560&author=M.+J.+O%E2%80%99Connor&title=Targeting+the+DNA+Damage+Response+in+Cancer&doi=10.1016%2Fj.molcel.2015.10.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the DNA Damage Response in Cancer</span></div><div class="casAuthors">O'Connor, Mark J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">547-560</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">An underlying hallmark of cancers is their genomic instability, which is assocd. with a greater propensity to accumulate DNA damage.  Historical treatment of cancer by radiotherapy and DNA-damaging chemotherapy is based on this principle, yet it is accompanied by significant collateral damage to normal tissue and unwanted side effects.  Targeted therapy based on inhibiting the DNA damage response (DDR) in cancers offers the potential for a greater therapeutic window by tailoring treatment to patients with tumors lacking specific DDR functions.  The recent approval of olaparib (Lynparza), the poly (ADP-ribose) polymerase (PARP) inhibitor for treating tumors harboring BRCA1 or BRCA2 mutations, represents the first medicine based on this principle, exploiting an underlying cause of tumor formation that also represents an Achilles' heel.  This review highlights the different concepts behind targeting DDR in cancer and how this can provide significant opportunities for DDR-based therapies in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWZ64MrshabrVg90H21EOLACvtfcHk0ljGzkwwNaw7vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyqtbvI&md5=b3f0d737d3e7dbf1f404b3f7658ab271</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.10.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.10.040%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DM.%2BJ.%26atitle%3DTargeting%2520the%2520DNA%2520Damage%2520Response%2520in%2520Cancer%26jtitle%3DMol.%2520Cell%26date%3D2015%26volume%3D60%26spage%3D547%26epage%3D560%26doi%3D10.1016%2Fj.molcel.2015.10.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mortusewicz, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fouquerel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amé, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonhardt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, V.</span></span> <span> </span><span class="NLM_article-title">PARG is Recruited to DNA Damage Sites Through Poly(ADP-ribose)- and PCNA-dependent Mechanisms</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">5045</span>– <span class="NLM_lpage">5056</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1093%2Fnar%2Fgkr099" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=5045-5056&author=O.+Mortusewiczauthor=E.+Fouquerelauthor=J.-C.+Am%C3%A9author=H.+Leonhardtauthor=V.+Schreiber&title=PARG+is+Recruited+to+DNA+Damage+Sites+Through+Poly%28ADP-ribose%29-+and+PCNA-dependent+Mechanisms&doi=10.1093%2Fnar%2Fgkr099"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr099%26sid%3Dliteratum%253Aachs%26aulast%3DMortusewicz%26aufirst%3DO.%26aulast%3DFouquerel%26aufirst%3DE.%26aulast%3DAm%25C3%25A9%26aufirst%3DJ.-C.%26aulast%3DLeonhardt%26aufirst%3DH.%26aulast%3DSchreiber%26aufirst%3DV.%26atitle%3DPARG%2520is%2520Recruited%2520to%2520DNA%2520Damage%2520Sites%2520Through%2520Poly%2528ADP-ribose%2529-%2520and%2520PCNA-dependent%2520Mechanisms%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3D5045%26epage%3D5056%26doi%3D10.1093%2Fnar%2Fgkr099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barkauskaite, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jankevicius, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladurner, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timinszky, G.</span></span> <span> </span><span class="NLM_article-title">The Recognition and Removal of Cellular Poly(ADP-ribose) Signals</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">3491</span>– <span class="NLM_lpage">3507</span>, <span class="refDoi"> DOI: 10.1111/febs.12358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1111%2Ffebs.12358" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2013&pages=3491-3507&author=E.+Barkauskaiteauthor=G.+Jankeviciusauthor=A.+G.+Ladurnerauthor=I.+Ahelauthor=G.+Timinszky&title=The+Recognition+and+Removal+of+Cellular+Poly%28ADP-ribose%29+Signals&doi=10.1111%2Ffebs.12358"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1111%2Ffebs.12358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.12358%26sid%3Dliteratum%253Aachs%26aulast%3DBarkauskaite%26aufirst%3DE.%26aulast%3DJankevicius%26aufirst%3DG.%26aulast%3DLadurner%26aufirst%3DA.%2BG.%26aulast%3DAhel%26aufirst%3DI.%26aulast%3DTiminszky%26aufirst%3DG.%26atitle%3DThe%2520Recognition%2520and%2520Removal%2520of%2520Cellular%2520Poly%2528ADP-ribose%2529%2520Signals%26jtitle%3DFEBS%2520J.%26date%3D2013%26volume%3D280%26spage%3D3491%26epage%3D3507%26doi%3D10.1111%2Ffebs.12358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span>; <span class="NLM_string-name">Koh, D.
W.</span></span> <span> </span><span class="NLM_article-title">Roles of Poly(ADP-Ribose) Glycohydrolase in DNA Damage and Apoptosis</span>.  <i>International Review of Cell and Molecular Biology</i>, <span class="NLM_contrib-group"><span class="NLM_string-name">Kwang, W. J.</span></span>, Ed. <span class="NLM_publisher-name">Academic Press</span>, <span class="NLM_year">2013</span>; Vol.  <span class="NLM_volume">304</span>, Chapter 5, pp  <span class="NLM_fpage">227</span>– <span class="NLM_lpage">281</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=227-281&author=X.+Feng&author=D.%0AW.+Kohauthor=W.+J.+Kwang&title=International+Review+of+Cell+and+Molecular+Biology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DX.%26atitle%3DRoles%2520of%2520Poly%2528ADP-Ribose%2529%2520Glycohydrolase%2520in%2520DNA%2520Damage%2520and%2520Apoptosis%26btitle%3DInternational%2520Review%2520of%2520Cell%2520and%2520Molecular%2520Biology%26aulast%3DKwang%26aufirst%3DW.%2BJ.%26pub%3DAcademic%2520Press%26date%3D2013%26volume%3D304%26spage%3D227%26epage%3D281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beumer, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamieh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tawbi, H.</span></span> <span> </span><span class="NLM_article-title">Role of PARP Inhibitors in Cancer Biology and Therapy</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">3907</span>– <span class="NLM_lpage">3921</span>, <span class="refDoi"> DOI: 10.2174/092986712802002464</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.2174%2F092986712802002464" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=22788767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht12nu73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=3907-3921&author=D.+Davarauthor=J.+H.+Beumerauthor=L.+Hamiehauthor=H.+Tawbi&title=Role+of+PARP+Inhibitors+in+Cancer+Biology+and+Therapy&doi=10.2174%2F092986712802002464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Role of PARP inhibitors in cancer biology and therapy</span></div><div class="casAuthors">Davar, D.; Beumer, J. H.; Hamieh, L.; Tawbi, H.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3907-3921</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Deeper understanding of DNA repair mechanisms and their potential value as therapeutic targets in oncol. heralded the clin. development of poly(ADP-ribose) polymerase (PARP) inhibitors.  Although initially developed to exploit synthetic lethality in models of cancer assocd. with defective DNA repair, our burgeoning knowledge of PARP biol. has resulted in these agents being exploited both in cancer with select chemotherapeutic agents and in non-malignant diseases.  In this review article, we briefly review the mechanisms of DNA repair and pre-clin. development of PARP inhibitors before discussing the clin. development of the various PARP inhibitors in depth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRQaaaS9jHi7Vg90H21EOLACvtfcHk0ljPyxcv_LnxjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht12nu73I&md5=70350a0d3159a740ca4ae3dfe6b5db25</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2174%2F092986712802002464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986712802002464%26sid%3Dliteratum%253Aachs%26aulast%3DDavar%26aufirst%3DD.%26aulast%3DBeumer%26aufirst%3DJ.%2BH.%26aulast%3DHamieh%26aufirst%3DL.%26aulast%3DTawbi%26aufirst%3DH.%26atitle%3DRole%2520of%2520PARP%2520Inhibitors%2520in%2520Cancer%2520Biology%2520and%2520Therapy%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2012%26volume%3D19%26spage%3D3907%26epage%3D3921%26doi%3D10.2174%2F092986712802002464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ricks, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ison, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKee, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kluetz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazdur, R.</span></span> <span> </span><span class="NLM_article-title">Successes and Challenges of PARP Inhibitors in Cancer Therapy</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">222</span>, <span class="refDoi"> DOI: 10.3389/fonc.2015.00222</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.3389%2Ffonc.2015.00222" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=26528434" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A280%3ADC%252BC28zot1ykug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=222&author=T.+K.+Ricksauthor=H.-J.+Chiuauthor=G.+Isonauthor=G.+Kimauthor=A.+E.+McKeeauthor=P.+Kluetzauthor=R.+Pazdur&title=Successes+and+Challenges+of+PARP+Inhibitors+in+Cancer+Therapy&doi=10.3389%2Ffonc.2015.00222"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Successes and Challenges of PARP Inhibitors in Cancer Therapy</span></div><div class="casAuthors">Ricks Tiffany K; Chiu Haw-Jyh; Ison Gwynn; Kim Geoffrey; McKee Amy E; Kluetz Paul; Pazdur Richard</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">222</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQxHehOgh33Mntlzo9xastyfW6udTcc2eZ3riKCyxrzp7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28zot1ykug%253D%253D&md5=66046870d9d787d55c8806292c24764a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2015.00222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2015.00222%26sid%3Dliteratum%253Aachs%26aulast%3DRicks%26aufirst%3DT.%2BK.%26aulast%3DChiu%26aufirst%3DH.-J.%26aulast%3DIson%26aufirst%3DG.%26aulast%3DKim%26aufirst%3DG.%26aulast%3DMcKee%26aufirst%3DA.%2BE.%26aulast%3DKluetz%26aufirst%3DP.%26aulast%3DPazdur%26aufirst%3DR.%26atitle%3DSuccesses%2520and%2520Challenges%2520of%2520PARP%2520Inhibitors%2520in%2520Cancer%2520Therapy%26jtitle%3DFront.%2520Oncol.%26date%3D2015%26volume%3D5%26spage%3D222%26doi%3D10.3389%2Ffonc.2015.00222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frampton, J. E.</span></span> <span> </span><span class="NLM_article-title">Olaparib: A Review of Its Use as Maintenance Therapy in Patients with Ovarian Cancer</span>. <i>BioDrugs</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">150</span>, <span class="refDoi"> DOI: 10.1007/s40259-015-0125-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1007%2Fs40259-015-0125-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=143-150&author=J.+E.+Frampton&title=Olaparib%3A+A+Review+of+Its+Use+as+Maintenance+Therapy+in+Patients+with+Ovarian+Cancer&doi=10.1007%2Fs40259-015-0125-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2Fs40259-015-0125-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40259-015-0125-6%26sid%3Dliteratum%253Aachs%26aulast%3DFrampton%26aufirst%3DJ.%2BE.%26atitle%3DOlaparib%253A%2520A%2520Review%2520of%2520Its%2520Use%2520as%2520Maintenance%2520Therapy%2520in%2520Patients%2520with%2520Ovarian%2520Cancer%26jtitle%3DBioDrugs%26date%3D2015%26volume%3D29%26spage%3D143%26epage%3D150%26doi%3D10.1007%2Fs40259-015-0125-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fauzee, N. J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-l.</span></span> <span> </span><span class="NLM_article-title">PARP and PARG Inhibitors-New Therapeutic Targets in Cancer Treatment</span>. <i>Pathol. Oncol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">469</span>– <span class="NLM_lpage">478</span>, <span class="refDoi"> DOI: 10.1007/s12253-010-9266-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1007%2Fs12253-010-9266-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=20383759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlSjt7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=469-478&author=N.+J.+S.+Fauzeeauthor=J.+Panauthor=Y.-l.+Wang&title=PARP+and+PARG+Inhibitors-New+Therapeutic+Targets+in+Cancer+Treatment&doi=10.1007%2Fs12253-010-9266-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">PARP and PARG Inhibitors-New Therapeutic Targets in Cancer Treatment</span></div><div class="casAuthors">Fauzee, Nilufer Jasmine Selimah; Pan, Juan; Wang, Ya-lan</div><div class="citationInfo"><span class="NLM_cas:title">Pathology & Oncology Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">469-478</span>CODEN:
                <span class="NLM_cas:coden">POREFR</span>;
        ISSN:<span class="NLM_cas:issn">1219-4956</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Today, the no. of cancer patients throughout the world is increasing alarmingly and as per the World Health Organization (WHO) data and statistics the prediction for the year 2020 will be 15 million new cases as compared to only 10 million cases in year 2000 leaving us dumbfounded.  A lot of effort has been put in by researchers and scientists over decades to find drugs helpful in the treatment of cancers for the benefit of patients-The latest being the Poly ADP-ribose polymerase (PARP) and the Poly ADP-ribose glycohydrolase (PARG) inhibitors.  This review highlights their mechanism of action under the rationale of their use and current development in the field of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohbT47IdZPjbVg90H21EOLACvtfcHk0ljLOZmdLT3bRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlSjt7%252FF&md5=17a2c5402b4cb761abe03665edb4a41c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs12253-010-9266-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12253-010-9266-6%26sid%3Dliteratum%253Aachs%26aulast%3DFauzee%26aufirst%3DN.%2BJ.%2BS.%26aulast%3DPan%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.-l.%26atitle%3DPARP%2520and%2520PARG%2520Inhibitors-New%2520Therapeutic%2520Targets%2520in%2520Cancer%2520Treatment%26jtitle%3DPathol.%2520Oncol.%2520Res.%26date%3D2010%26volume%3D16%26spage%3D469%26epage%3D478%26doi%3D10.1007%2Fs12253-010-9266-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makhija, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandak, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaikwad, A. B</span></span> <span> </span><span class="NLM_article-title">PARG Inhibitors’ Success: A Long Way to Go!</span>. <i>Curr. Enzyme Inhib.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">93</span>, <span class="refDoi"> DOI: 10.2174/1573408010666141126220225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.2174%2F1573408010666141126220225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVGitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=81-93&author=A.+Pandeyauthor=P.+Makhijaauthor=P.+G.+Chandakauthor=A.+B+Gaikwad&title=PARG+Inhibitors%E2%80%99+Success%3A+A+Long+Way+to+Go%21&doi=10.2174%2F1573408010666141126220225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">"PARG Inhibitorsμ Success: A Long Way to Go!"</span></div><div class="casAuthors">Pandey, Anuradha; Makhija, Pooja; Chandak, Prakash G.; Gaikwad, Anil Bhanudas</div><div class="citationInfo"><span class="NLM_cas:title">Current Enzyme Inhibition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">81-93</span>CODEN:
                <span class="NLM_cas:coden">CEIUAG</span>;
        ISSN:<span class="NLM_cas:issn">1573-4080</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribosyl)ation is one of the most pertinent post translational modifications involved in regulation of chromatin structure, cell cycle progression, tissue development and differentiation and other vital biol. phenomena.  The enzymes that catalyze the synthesis and degrdn. of poly(ADP) ribose polymers are PARP and PARG, resp.  The role of PARP has been implicated in development of various diseases since a long time and hence it has evolved as an important pharmacol. target but a plethora of drawbacks assocd. with PARP inhibitors compelled the shift of focus towards PARG.  Recently PARG has evolved as an alternative target to overcome the hurdles being faced in the treatment of various conditions like multiple organ failure, ischemic organ damage, diabetic nephropathy, neurodegenerative diseases and cancer.  The review provides a compendium on PARG, its mode of action, inhibitors, and its therapeutic applications and also discusses the reasons due to which PARG inhibitors have not been able to reach the clin. trials.  PARG inhibitors, though far from success, definitely appear as alluring topics for further research as PARG emerges out as an eminent pharmacol. target in making which can shape the future of medicines to provide better therapy with reduced side effects and more efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreF7WC4XqPYbVg90H21EOLACvtfcHk0lj7KtwFvwEcJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVGitrw%253D&md5=c257034e60bfb73a0bcf395a03ba5fdd</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.2174%2F1573408010666141126220225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1573408010666141126220225%26sid%3Dliteratum%253Aachs%26aulast%3DPandey%26aufirst%3DA.%26aulast%3DMakhija%26aufirst%3DP.%26aulast%3DChandak%26aufirst%3DP.%2BG.%26aulast%3DGaikwad%26aufirst%3DA.%2BB%26atitle%3DPARG%2520Inhibitors%25E2%2580%2599%2520Success%253A%2520A%2520Long%2520Way%2520to%2520Go%2521%26jtitle%3DCurr.%2520Enzyme%2520Inhib.%26date%3D2014%26volume%3D10%26spage%3D81%26epage%3D93%26doi%3D10.2174%2F1573408010666141126220225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">James, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairweather, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hitchin, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutton, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGonagle, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stowell, A. I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddell, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waszkowycz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, D. J.</span></span> <span> </span><span class="NLM_article-title">First-in-Class Chemical Probes against Poly(ADP-ribose) Glycohydrolase (PARG) Inhibit DNA Repair with Differential Pharmacology to Olaparib</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3179</span>– <span class="NLM_lpage">3190</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00609</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00609" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=3179-3190&author=D.+I.+Jamesauthor=K.+M.+Smithauthor=A.+M.+Jordanauthor=E.+E.+Fairweatherauthor=L.+A.+Griffithsauthor=N.+S.+Hamiltonauthor=J.+R.+Hitchinauthor=C.+P.+Huttonauthor=S.+Jonesauthor=P.+Kellyauthor=A.+E.+McGonagleauthor=H.+Smallauthor=A.+I.+J.+Stowellauthor=J.+Tuckerauthor=I.+D.+Waddellauthor=B.+Waszkowyczauthor=D.+J.+Ogilvie&title=First-in-Class+Chemical+Probes+against+Poly%28ADP-ribose%29+Glycohydrolase+%28PARG%29+Inhibit+DNA+Repair+with+Differential+Pharmacology+to+Olaparib&doi=10.1021%2Facschembio.6b00609"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00609%26sid%3Dliteratum%253Aachs%26aulast%3DJames%26aufirst%3DD.%2BI.%26aulast%3DSmith%26aufirst%3DK.%2BM.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DFairweather%26aufirst%3DE.%2BE.%26aulast%3DGriffiths%26aufirst%3DL.%2BA.%26aulast%3DHamilton%26aufirst%3DN.%2BS.%26aulast%3DHitchin%26aufirst%3DJ.%2BR.%26aulast%3DHutton%26aufirst%3DC.%2BP.%26aulast%3DJones%26aufirst%3DS.%26aulast%3DKelly%26aufirst%3DP.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DSmall%26aufirst%3DH.%26aulast%3DStowell%26aufirst%3DA.%2BI.%2BJ.%26aulast%3DTucker%26aufirst%3DJ.%26aulast%3DWaddell%26aufirst%3DI.%2BD.%26aulast%3DWaszkowycz%26aufirst%3DB.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26atitle%3DFirst-in-Class%2520Chemical%2520Probes%2520against%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%2520%2528PARG%2529%2520Inhibit%2520DNA%2520Repair%2520with%2520Differential%2520Pharmacology%2520to%2520Olaparib%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D3179%26epage%3D3190%26doi%3D10.1021%2Facschembio.6b00609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slade, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunstan, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barkauskaite, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weston, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafite, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leys, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, I.</span></span> <span> </span><span class="NLM_article-title">The Structure and Catalytic Mechanism of a Poly(ADP-ribose) Glycohydrolase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>477</i></span>,  <span class="NLM_fpage">616</span>– <span class="NLM_lpage">620</span>, <span class="refDoi"> DOI: 10.1038/nature10404</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1038%2Fnature10404" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=21892188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFers7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=477&publication_year=2011&pages=616-620&author=D.+Sladeauthor=M.+S.+Dunstanauthor=E.+Barkauskaiteauthor=R.+Westonauthor=P.+Lafiteauthor=N.+Dixonauthor=M.+Ahelauthor=D.+Leysauthor=I.+Ahel&title=The+Structure+and+Catalytic+Mechanism+of+a+Poly%28ADP-ribose%29+Glycohydrolase&doi=10.1038%2Fnature10404"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The structure and catalytic mechanism of a poly(ADP-ribose) glycohydrolase</span></div><div class="casAuthors">Slade, Dea; Dunstan, Mark S.; Barkauskaite, Eva; Weston, Ria; Lafite, Pierre; Dixon, Neil; Ahel, Marijan; Leys, David; Ahel, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">477</span>
        (<span class="NLM_cas:issue">7366</span>),
    <span class="NLM_cas:pages">616-620</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Post-translational modification of proteins by poly(ADP-ribosyl)ation regulates many cellular pathways that are crit. for genome stability, including DNA repair, chromatin structure, mitosis, and apoptosis.  Poly(ADP-ribose) (PAR) is composed of repeating ADP-ribose units linked via a unique glycosidic ribose-ribose bond, and is synthesized from NAD by PAR polymerases.  PAR glycohydrolase (PARG) is the only protein capable of specific hydrolysis of the ribose-ribose bonds present in PAR chains; its deficiency leads to cell death.  Here, the authors show that filamentous fungi and a no. of bacteria possess a divergent form of PARG that has all the main characteristics of the human PARG enzyme.  The authors present the 1st crystal structure of PARG of Thermomonospora curvata, which reveals that the PARG catalytic domain is a distant member of the ubiquitous ADP-ribose-binding macrodomain family.  High-resoln. structures of T. curvata PARG in complexes with ADP-ribose and the PARG inhibitor, ADP (hydroxymethyl)pyrrolidinediol (ADP-HPD), complemented by biochem. studies, allowed the authors to propose a model for PAR binding and catalysis by PARG.  The insights into the PARG structure and catalytic mechanism should greatly improve the understanding of how PARG activity controls reversible protein poly(ADP-ribosyl)ation and potentially of how the defects in this regulation are linked to human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjFsMUTjjlOrVg90H21EOLACvtfcHk0lj7KtwFvwEcJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFers7rE&md5=7e539f310e582800407bddd95efca01f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature10404&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10404%26sid%3Dliteratum%253Aachs%26aulast%3DSlade%26aufirst%3DD.%26aulast%3DDunstan%26aufirst%3DM.%2BS.%26aulast%3DBarkauskaite%26aufirst%3DE.%26aulast%3DWeston%26aufirst%3DR.%26aulast%3DLafite%26aufirst%3DP.%26aulast%3DDixon%26aufirst%3DN.%26aulast%3DAhel%26aufirst%3DM.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DAhel%26aufirst%3DI.%26atitle%3DThe%2520Structure%2520and%2520Catalytic%2520Mechanism%2520of%2520a%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%26jtitle%3DNature%26date%3D2011%26volume%3D477%26spage%3D616%26epage%3D620%26doi%3D10.1038%2Fnature10404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dunstan, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barkauskaite, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafite, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knezevic, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brassington, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leys, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, I.</span></span> <span> </span><span class="NLM_article-title">Structure and Mechanism of a Canonical Poly(ADP-ribose) Glycohydrolase</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">878</span>, <span class="refDoi"> DOI: 10.1038/ncomms1889</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1038%2Fncomms1889" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=22673905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A280%3ADC%252BC38npsFOrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=878&author=M.+S.+Dunstanauthor=E.+Barkauskaiteauthor=P.+Lafiteauthor=C.+E.+Knezevicauthor=A.+Brassingtonauthor=M.+Ahelauthor=P.+J.+Hergenrotherauthor=D.+Leysauthor=I.+Ahel&title=Structure+and+Mechanism+of+a+Canonical+Poly%28ADP-ribose%29+Glycohydrolase&doi=10.1038%2Fncomms1889"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and mechanism of a canonical poly(ADP-ribose) glycohydrolase</span></div><div class="casAuthors">Dunstan Mark S; Barkauskaite Eva; Lafite Pierre; Knezevic Claire E; Brassington Amy; Ahel Marijan; Hergenrother Paul J; Leys David; Ahel Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">878</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Poly(ADP-ribosyl)ation is a reversible post-translational protein modification involved in the regulation of a number of cellular processes including DNA repair, chromatin structure, mitosis, transcription, checkpoint activation, apoptosis and asexual development.  The reversion of poly(ADP-ribosyl)ation is catalysed by poly(ADP-ribose) (PAR) glycohydrolase (PARG), which specifically targets the unique PAR (1''-2') ribose-ribose bonds.  Here we report the structure and mechanism of the first canonical PARG from the protozoan Tetrahymena thermophila.  In addition, we reveal the structure of T. thermophila PARG in a complex with a novel rhodanine-containing mammalian PARG inhibitor RBPI-3.  Our data demonstrate that the protozoan PARG represents a good model for human PARG and is therefore likely to prove useful in guiding structure-based discovery of new classes of PARG inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTWJIiwTxgYVv6BottYdIAWfW6udTcc2eZ3Zk3MEmy_Zbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38npsFOrtQ%253D%253D&md5=42bea8394a83a8c32f6fb422e5862cbc</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fncomms1889&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms1889%26sid%3Dliteratum%253Aachs%26aulast%3DDunstan%26aufirst%3DM.%2BS.%26aulast%3DBarkauskaite%26aufirst%3DE.%26aulast%3DLafite%26aufirst%3DP.%26aulast%3DKnezevic%26aufirst%3DC.%2BE.%26aulast%3DBrassington%26aufirst%3DA.%26aulast%3DAhel%26aufirst%3DM.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26aulast%3DLeys%26aufirst%3DD.%26aulast%3DAhel%26aufirst%3DI.%26atitle%3DStructure%2520and%2520Mechanism%2520of%2520a%2520Canonical%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%26jtitle%3DNat.%2520Commun.%26date%3D2012%26volume%3D3%26spage%3D878%26doi%3D10.1038%2Fncomms1889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, I.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, C. M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegeman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Classen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tainer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellenberger, T.</span></span> <span> </span><span class="NLM_article-title">Structure of Mammalian Poly(ADP-ribose) Glycohydrolase Reveals a Flexible Tyrosine Clasp as a Substrate-binding Element</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">653</span>– <span class="NLM_lpage">656</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1038%2Fnsmb.2305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=22609859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC38XntF2js7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=653-656&author=I.-K.+Kimauthor=J.+R.+Kieferauthor=C.+M.+W.+Hoauthor=R.+A.+Stegemanauthor=S.+Classenauthor=J.+A.+Tainerauthor=T.+Ellenberger&title=Structure+of+Mammalian+Poly%28ADP-ribose%29+Glycohydrolase+Reveals+a+Flexible+Tyrosine+Clasp+as+a+Substrate-binding+Element&doi=10.1038%2Fnsmb.2305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of mammalian poly(ADP-ribose) glycohydrolase reveals a flexible tyrosine clasp as a substrate-binding element</span></div><div class="casAuthors">Kim, In-Kwon; Kiefer, James R.; Ho, Chris M. W.; Stegeman, Roderick A.; Classen, Scott; Tainer, John A.; Ellenberger, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">653-656</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Reversible post-translational modification by poly(ADP-ribose) (PAR) regulates chromatin structure, DNA repair and cell fate in response to genotoxic stress.  PAR glycohydrolase (PARG) removes PAR chains from poly ADP-ribosylated proteins to restore protein function and release oligo(ADP-ribose) chains to signal damage.  Here we report crystal structures of mammalian PARG and its complex with a substrate mimic that reveal an open substrate-binding site and a unique 'tyrosine clasp' enabling endoglycosidic cleavage of branched PAR chains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLggg2gkhJj7Vg90H21EOLACvtfcHk0lhUffrHhnJ4Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntF2js7c%253D&md5=fda9258dddd2b7dbd9609291d757f7e5</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2305%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DI.-K.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DHo%26aufirst%3DC.%2BM.%2BW.%26aulast%3DStegeman%26aufirst%3DR.%2BA.%26aulast%3DClassen%26aufirst%3DS.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26aulast%3DEllenberger%26aufirst%3DT.%26atitle%3DStructure%2520of%2520Mammalian%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%2520Reveals%2520a%2520Flexible%2520Tyrosine%2520Clasp%2520as%2520a%2520Substrate-binding%2520Element%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D653%26epage%3D656%26doi%3D10.1038%2Fnsmb.2305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brassington, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durant, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassall, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holdgate, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAlister, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, M.</span></span> <span> </span><span class="NLM_article-title">Structures of the Human Poly(ADP-Ribose) Glycohydrolase Catalytic Domain Confirm Catalytic Mechanism and Explain Inhibition by ADP-HPD Derivatives</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e50889</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0050889</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1371%2Fjournal.pone.0050889" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&author=J.+A.+Tuckerauthor=N.+Bennettauthor=C.+Brassingtonauthor=S.+T.+Durantauthor=G.+Hassallauthor=G.+Holdgateauthor=M.+McAlisterauthor=J.+W.+M.+Nissinkauthor=C.+Trumanauthor=M.+Watson&title=Structures+of+the+Human+Poly%28ADP-Ribose%29+Glycohydrolase+Catalytic+Domain+Confirm+Catalytic+Mechanism+and+Explain+Inhibition+by+ADP-HPD+Derivatives&doi=10.1371%2Fjournal.pone.0050889"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0050889&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0050889%26sid%3Dliteratum%253Aachs%26aulast%3DTucker%26aufirst%3DJ.%2BA.%26aulast%3DBennett%26aufirst%3DN.%26aulast%3DBrassington%26aufirst%3DC.%26aulast%3DDurant%26aufirst%3DS.%2BT.%26aulast%3DHassall%26aufirst%3DG.%26aulast%3DHoldgate%26aufirst%3DG.%26aulast%3DMcAlister%26aufirst%3DM.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DTruman%26aufirst%3DC.%26aulast%3DWatson%26aufirst%3DM.%26atitle%3DStructures%2520of%2520the%2520Human%2520Poly%2528ADP-Ribose%2529%2520Glycohydrolase%2520Catalytic%2520Domain%2520Confirm%2520Catalytic%2520Mechanism%2520and%2520Explain%2520Inhibition%2520by%2520ADP-HPD%2520Derivatives%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26doi%3D10.1371%2Fjournal.pone.0050889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barkauskaite, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brassington, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warwicker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunstan, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafite, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchison, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leys, D.</span></span> <span> </span><span class="NLM_article-title">Visualization of Poly(ADP-ribose) Bound to PARG Reveals Inherent Balance Between <i>Exo</i>- and <i>Endo</i>-Glycohydrolase Activities</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">2164</span>, <span class="refDoi"> DOI: 10.1038/ncomms3164</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1038%2Fncomms3164" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=23917065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A280%3ADC%252BC3sfntFSquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2164&author=E.+Barkauskaiteauthor=A.+Brassingtonauthor=E.+S.+Tanauthor=J.+Warwickerauthor=M.+S.+Dunstanauthor=B.+Banosauthor=P.+Lafiteauthor=M.+Ahelauthor=T.+J.+Mitchisonauthor=I.+Ahelauthor=D.+Leys&title=Visualization+of+Poly%28ADP-ribose%29+Bound+to+PARG+Reveals+Inherent+Balance+Between+Exo-+and+Endo-Glycohydrolase+Activities&doi=10.1038%2Fncomms3164"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Visualization of poly(ADP-ribose) bound to PARG reveals inherent balance between exo- and endo-glycohydrolase activities</span></div><div class="casAuthors">Barkauskaite Eva; Brassington Amy; Tan Edwin S; Warwicker Jim; Dunstan Mark S; Banos Benito; Lafite Pierre; Ahel Marijan; Mitchison Timothy J; Ahel Ivan; Leys David</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2164</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Poly-ADP-ribosylation is a post-translational modification that regulates processes involved in genome stability.  Breakdown of the poly(ADP-ribose) (PAR) polymer is catalysed by poly(ADP-ribose) glycohydrolase (PARG), whose endo-glycohydrolase activity generates PAR fragments.  Here we present the crystal structure of PARG incorporating the PAR substrate.  The two terminal ADP-ribose units of the polymeric substrate are bound in exo-mode.  Biochemical and modelling studies reveal that PARG acts predominantly as an exo-glycohydrolase.  This preference is linked to Phe902 (human numbering), which is responsible for low-affinity binding of the substrate in endo-mode.  Our data reveal the mechanism of poly-ADP-ribosylation reversal, with ADP-ribose as the dominant product, and suggest that the release of apoptotic PAR fragments occurs at unusual PAR/PARG ratios.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSRNfG0TEYbZfSAs_PE1QvifW6udTcc2ebrWQKSm3s_47ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sfntFSquw%253D%253D&md5=53b696cff64b0dcb983cf0c35a839a46</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fncomms3164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms3164%26sid%3Dliteratum%253Aachs%26aulast%3DBarkauskaite%26aufirst%3DE.%26aulast%3DBrassington%26aufirst%3DA.%26aulast%3DTan%26aufirst%3DE.%2BS.%26aulast%3DWarwicker%26aufirst%3DJ.%26aulast%3DDunstan%26aufirst%3DM.%2BS.%26aulast%3DBanos%26aufirst%3DB.%26aulast%3DLafite%26aufirst%3DP.%26aulast%3DAhel%26aufirst%3DM.%26aulast%3DMitchison%26aufirst%3DT.%2BJ.%26aulast%3DAhel%26aufirst%3DI.%26aulast%3DLeys%26aufirst%3DD.%26atitle%3DVisualization%2520of%2520Poly%2528ADP-ribose%2529%2520Bound%2520to%2520PARG%2520Reveals%2520Inherent%2520Balance%2520Between%2520Exo-%2520and%2520Endo-Glycohydrolase%2520Activities%26jtitle%3DNat.%2520Commun.%26date%3D2013%26volume%3D4%26spage%3D2164%26doi%3D10.1038%2Fncomms3164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Žaja, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikoč, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barkauskaite, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, I.</span></span> <span> </span><span class="NLM_article-title">Molecular Insights into Poly(ADP-ribose) Recognition and Processing</span>. <i>Biomolecules</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.3390/biom3010001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.3390%2Fbiom3010001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=1-17&author=R.+%C5%BDajaauthor=A.+Miko%C4%8Dauthor=E.+Barkauskaiteauthor=I.+Ahel&title=Molecular+Insights+into+Poly%28ADP-ribose%29+Recognition+and+Processing&doi=10.3390%2Fbiom3010001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.3390%2Fbiom3010001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiom3010001%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%25BDaja%26aufirst%3DR.%26aulast%3DMiko%25C4%258D%26aufirst%3DA.%26aulast%3DBarkauskaite%26aufirst%3DE.%26aulast%3DAhel%26aufirst%3DI.%26atitle%3DMolecular%2520Insights%2520into%2520Poly%2528ADP-ribose%2529%2520Recognition%2520and%2520Processing%26jtitle%3DBiomolecules%26date%3D2013%26volume%3D3%26spage%3D1%26epage%3D17%26doi%3D10.3390%2Fbiom3010001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagné, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poirier, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span> <span> </span><span class="NLM_article-title">Crystallographic and Biochemical Analysis of the Mouse Poly(ADP-ribose) Glycohydrolase</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e86010</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0086010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1371%2Fjournal.pone.0086010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&author=Z.+Wangauthor=J.-P.+Gagn%C3%A9author=G.+G.+Poirierauthor=W.+Xu&title=Crystallographic+and+Biochemical+Analysis+of+the+Mouse+Poly%28ADP-ribose%29+Glycohydrolase&doi=10.1371%2Fjournal.pone.0086010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0086010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0086010%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DGagn%25C3%25A9%26aufirst%3DJ.-P.%26aulast%3DPoirier%26aufirst%3DG.%2BG.%26aulast%3DXu%26aufirst%3DW.%26atitle%3DCrystallographic%2520and%2520Biochemical%2520Analysis%2520of%2520the%2520Mouse%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26doi%3D10.1371%2Fjournal.pone.0086010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rack, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, I.</span></span> <span> </span><span class="NLM_article-title">Macrodomains: Structure, Function, Evolution, and Catalytic Activities</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">431</span>– <span class="NLM_lpage">454</span>, <span class="refDoi"> DOI: 10.1146/annurev-biochem-060815-014935</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1146%2Fannurev-biochem-060815-014935" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=26844395" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1WrsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2016&pages=431-454&author=J.+G.+Rackauthor=D.+Perinaauthor=I.+Ahel&title=Macrodomains%3A+Structure%2C+Function%2C+Evolution%2C+and+Catalytic+Activities&doi=10.1146%2Fannurev-biochem-060815-014935"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Macrodomains: Structure, Function, Evolution, and Catalytic Activities</span></div><div class="casAuthors">Rack, Johannes Gregor Matthias; Perina, Dragutin; Ahel, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">431-454</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  Recent developments indicate that macrodomains, an ancient and diverse protein domain family, are key players in the recognition, interpretation, and turnover of ADP-ribose (ADPr) signaling.  Crucial to this is the ability of macrodomains to recognize ADPr either directly, in the form of a metabolic deriv., or as a modification covalently bound to proteins.  Thus, macrodomains regulate a wide variety of cellular and organismal processes, including DNA damage repair, signal transduction, and immune response.  Their importance is further indicated by the fact that dysregulation or mutation of a macrodomain is assocd. with several diseases, including cancer, developmental defects, and neurodegeneration.  In this review, we summarize the current insights into macrodomain evolution and how this evolution influenced their structural and functional diversification.  We highlight some aspects of macrodomain roles in pathobiol. as well as their emerging potential as therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozF6aFyzAOS7Vg90H21EOLACvtfcHk0lgidcUo_ythsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1WrsLY%253D&md5=cdd198a0cdbdb8d41adf66109794cb5a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-060815-014935&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-060815-014935%26sid%3Dliteratum%253Aachs%26aulast%3DRack%26aufirst%3DJ.%2BG.%26aulast%3DPerina%26aufirst%3DD.%26aulast%3DAhel%26aufirst%3DI.%26atitle%3DMacrodomains%253A%2520Structure%252C%2520Function%252C%2520Evolution%252C%2520and%2520Catalytic%2520Activities%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2016%26volume%3D85%26spage%3D431%26epage%3D454%26doi%3D10.1146%2Fannurev-biochem-060815-014935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stowell, A. I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddell, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardern, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, D. J.</span></span> <span> </span><span class="NLM_article-title">A High-throughput Screening-compatible Homogeneous Time-resolved Fluorescence Assay Measuring the Glycohydrolase Activity of Human Poly(ADP-ribose) Glycohydrolase</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>503</i></span>,  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2016.03.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1016%2Fj.ab.2016.03.016" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2016&pages=58-64&author=A.+I.+J.+Stowellauthor=D.+I.+Jamesauthor=I.+D.+Waddellauthor=N.+Bennettauthor=C.+Trumanauthor=I.+M.+Hardernauthor=D.+J.+Ogilvie&title=A+High-throughput+Screening-compatible+Homogeneous+Time-resolved+Fluorescence+Assay+Measuring+the+Glycohydrolase+Activity+of+Human+Poly%28ADP-ribose%29+Glycohydrolase&doi=10.1016%2Fj.ab.2016.03.016"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2016.03.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2016.03.016%26sid%3Dliteratum%253Aachs%26aulast%3DStowell%26aufirst%3DA.%2BI.%2BJ.%26aulast%3DJames%26aufirst%3DD.%2BI.%26aulast%3DWaddell%26aufirst%3DI.%2BD.%26aulast%3DBennett%26aufirst%3DN.%26aulast%3DTruman%26aufirst%3DC.%26aulast%3DHardern%26aufirst%3DI.%2BM.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26atitle%3DA%2520High-throughput%2520Screening-compatible%2520Homogeneous%2520Time-resolved%2520Fluorescence%2520Assay%2520Measuring%2520the%2520Glycohydrolase%2520Activity%2520of%2520Human%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%26jtitle%3DAnal.%2520Biochem.%26date%3D2016%26volume%3D503%26spage%3D58%26epage%3D64%26doi%3D10.1016%2Fj.ab.2016.03.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koh, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsinghani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slama, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. K.</span></span> <span> </span><span class="NLM_article-title">SAR Analysis of Adenosine Diphosphate (Hydroxymethyl)pyrrolidinediol Inhibition of Poly(ADP-ribose) Glycohydrolase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">4322</span>– <span class="NLM_lpage">4332</span>, <span class="refDoi"> DOI: 10.1021/jm020541u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020541u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=4322-4332&author=D.+W.+Kohauthor=D.+L.+Coyleauthor=N.+Mehtaauthor=S.+Ramsinghaniauthor=H.+Kimauthor=J.+T.+Slamaauthor=M.+K.+Jacobson&title=SAR+Analysis+of+Adenosine+Diphosphate+%28Hydroxymethyl%29pyrrolidinediol+Inhibition+of+Poly%28ADP-ribose%29+Glycohydrolase&doi=10.1021%2Fjm020541u"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm020541u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020541u%26sid%3Dliteratum%253Aachs%26aulast%3DKoh%26aufirst%3DD.%2BW.%26aulast%3DCoyle%26aufirst%3DD.%2BL.%26aulast%3DMehta%26aufirst%3DN.%26aulast%3DRamsinghani%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DSlama%26aufirst%3DJ.%2BT.%26aulast%3DJacobson%26aufirst%3DM.%2BK.%26atitle%3DSAR%2520Analysis%2520of%2520Adenosine%2520Diphosphate%2520%2528Hydroxymethyl%2529pyrrolidinediol%2520Inhibition%2520of%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D4322%26epage%3D4332%26doi%3D10.1021%2Fjm020541u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Formentini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arapistas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacomelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitozzi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menichetti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giovannelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarugi, A.</span></span> <span> </span><span class="NLM_article-title">Mono-galloyl Glucose Derivatives are Potent Poly(ADP-ribose) Glycohydrolase (PARG) Inhibitors and Partially Reduce PARP-1-dependent Cell Death</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">1235</span>– <span class="NLM_lpage">1249</span>, <span class="refDoi"> DOI: 10.1038/bjp.2008.370</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1038%2Fbjp.2008.370" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=18806807" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVKlt7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2008&pages=1235-1249&author=L.+Formentiniauthor=P.+Arapistasauthor=M.+Pittelliauthor=M.+Jacomelliauthor=V.+Pitozziauthor=S.+Menichettiauthor=A.+Romaniauthor=L.+Giovannelliauthor=F.+Moroniauthor=A.+Chiarugi&title=Mono-galloyl+Glucose+Derivatives+are+Potent+Poly%28ADP-ribose%29+Glycohydrolase+%28PARG%29+Inhibitors+and+Partially+Reduce+PARP-1-dependent+Cell+Death&doi=10.1038%2Fbjp.2008.370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Mono-galloyl glucose derivatives are potent poly(ADP-ribose) glycohydrolase (PARG) inhibitors and partially reduce PARP-1-dependent cell death</span></div><div class="casAuthors">Formentini, L.; Arapistas, P.; Pittelli, M.; Jacomelli, M.; Pitozzi, V.; Menichetti, S.; Romani, A.; Giovannelli, L.; Moroni, F.; Chiarugi, A.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1235-1249</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Maintenance of poly(ADP-ribose) (PAR) polymers at homoeostatic levels by PAR glycohydrolase (PARG) is central in cell functioning and survival.  Yet the pharmacol. relevance of PARG inhibitors is still debated.  Gallotannin, a complex mixt. of hydrolyzable tannins from oak gall, inhibits PARG but which of its constituents is responsible for the inhibition and whether the pharmacodynamic properties are due to its antioxidant properties, has not yet been established.  A structure-activity relationship study was conducted on different natural and synthetic tannins/galloyl derivs. as potential PARG inhibitors, using a novel in vitro enzymic assay.  Cytotoxicity was assayed in cultured HeLa cells.  Mono-galloyl glucose compds. were potent inhibitors of PARG, with activities similar to that of ADP-(hydroxymethyl) pyrrolidinediol, the most potent PARG inhibitor yet identified.  When tested on HeLa cells exposed to the PAR polymerase (PARP)-1-activating compd. 1-methyl-3-nitro-1-nitrosoguanidine (MNNG), 3-galloyl glucose weakly inhibited PAR degrdn.  Conversely, the more lipophilic, 3-galloyl-1,2-O-isopropylidene glucose, despite being inactive on the pure enzyme, efficiently prolonged the half-life of the polymers in intact HeLa cells.  Also, PARG inhibitors, but not radical scavengers, reduced, in part, cell death caused by MNNG.  Taken together, our findings identify mono-galloyl glucose derivs. as potent PARG inhibitors, and emphasize the active function of this enzyme in cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdKsn9DOPnL7Vg90H21EOLACvtfcHk0lgidcUo_ythsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVKlt7rE&md5=f28e30c70b4e12b491b7b9e354f03311</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fbjp.2008.370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjp.2008.370%26sid%3Dliteratum%253Aachs%26aulast%3DFormentini%26aufirst%3DL.%26aulast%3DArapistas%26aufirst%3DP.%26aulast%3DPittelli%26aufirst%3DM.%26aulast%3DJacomelli%26aufirst%3DM.%26aulast%3DPitozzi%26aufirst%3DV.%26aulast%3DMenichetti%26aufirst%3DS.%26aulast%3DRomani%26aufirst%3DA.%26aulast%3DGiovannelli%26aufirst%3DL.%26aulast%3DMoroni%26aufirst%3DF.%26aulast%3DChiarugi%26aufirst%3DA.%26atitle%3DMono-galloyl%2520Glucose%2520Derivatives%2520are%2520Potent%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%2520%2528PARG%2529%2520Inhibitors%2520and%2520Partially%2520Reduce%2520PARP-1-dependent%2520Cell%2520Death%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2008%26volume%3D155%26spage%3D1235%26epage%3D1249%26doi%3D10.1038%2Fbjp.2008.370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steffen, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damodaran, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beroza, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. K.</span></span> <span> </span><span class="NLM_article-title">Discovery and Structure-Activity Relationships of Modified Salicylanilides as Cell Permeable Inhibitors of Poly(ADP-ribose) Glycohydrolase (PARG)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">5403</span>– <span class="NLM_lpage">5413</span>, <span class="refDoi"> DOI: 10.1021/jm200325s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200325s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFyksLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=5403-5413&author=J.+D.+Steffenauthor=D.+L.+Coyleauthor=K.+Damodaranauthor=P.+Berozaauthor=M.+K.+Jacobson&title=Discovery+and+Structure-Activity+Relationships+of+Modified+Salicylanilides+as+Cell+Permeable+Inhibitors+of+Poly%28ADP-ribose%29+Glycohydrolase+%28PARG%29&doi=10.1021%2Fjm200325s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Structure-Activity Relationships of Modified Salicylanilides as Cell Permeable Inhibitors of Poly(ADP-ribose) Glycohydrolase (PARG)</span></div><div class="casAuthors">Steffen, Jamin D.; Coyle, Donna L.; Damodaran, Komath; Beroza, Paul; Jacobson, Myron K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5403-5413</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The metab. of poly(ADP-ribose) (PAR) in response to DNA strand breaks, which involves the concerted activities of poly(ADP-ribose) polymerases (PARPs) and poly(ADP-ribose) glycohydrolase (PARG), modulates cell recovery or cell death depending upon the level of DNA damage.  While PARP inhibitors show high promise in clin. trials because of their low toxicity and selectivity for BRCA related cancers, evaluation of the therapeutic potential of PARG is limited by the lack of well-validated cell permeable inhibitors.  In this study, target-related affinity profiling (TRAP), an alternative to high-throughput screening, was used to identify a no. of druglike compds. from several chem. classes that demonstrated PARG inhibition in the low-micromolar range.  A no. of analogs of one of the most active chemotypes were synthesized to explore the structure-activity relationship (SAR) for that series.  This led to the discovery of a putative pharmacophore for PARG inhibition that contains a modified salicylanilide structure.  Interestingly, these compds. also inhibit PARP-1, indicating strong homol. in the active sites of PARG and PARP-1 and raising a new challenge for development of PARG specific inhibitors.  The cellular activity of a lead inhibitor was demonstrated by the inhibition of both PARP and PARG activity in squamous cell carcinoma cells, although preferential inhibition of PARG relative to PARP was obsd.  The ability of inhibitors to modulate PAR metab. via simultaneous effects on PARPs and PARG may represent a new approach for therapeutic development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1rgu2zajJJ7Vg90H21EOLACvtfcHk0lg1rjemJDU_2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFyksLc%253D&md5=340fde4d20e1a35348d94f7e75ff3641</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm200325s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200325s%26sid%3Dliteratum%253Aachs%26aulast%3DSteffen%26aufirst%3DJ.%2BD.%26aulast%3DCoyle%26aufirst%3DD.%2BL.%26aulast%3DDamodaran%26aufirst%3DK.%26aulast%3DBeroza%26aufirst%3DP.%26aulast%3DJacobson%26aufirst%3DM.%2BK.%26atitle%3DDiscovery%2520and%2520Structure-Activity%2520Relationships%2520of%2520Modified%2520Salicylanilides%2520as%2520Cell%2520Permeable%2520Inhibitors%2520of%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%2520%2528PARG%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D5403%26epage%3D5413%26doi%3D10.1021%2Fjm200325s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finch, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knezevic, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nottbohm, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Partlow, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Selective Small Molecule Inhibition of Poly(ADP-Ribose) Glycohydrolase (PARG)</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">563</span>– <span class="NLM_lpage">570</span>, <span class="refDoi"> DOI: 10.1021/cb200506t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb200506t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=563-570&author=K.+E.+Finchauthor=C.+E.+Knezevicauthor=A.+C.+Nottbohmauthor=K.+C.+Partlowauthor=P.+J.+Hergenrother&title=Selective+Small+Molecule+Inhibition+of+Poly%28ADP-Ribose%29+Glycohydrolase+%28PARG%29&doi=10.1021%2Fcb200506t"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fcb200506t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb200506t%26sid%3Dliteratum%253Aachs%26aulast%3DFinch%26aufirst%3DK.%2BE.%26aulast%3DKnezevic%26aufirst%3DC.%2BE.%26aulast%3DNottbohm%26aufirst%3DA.%2BC.%26aulast%3DPartlow%26aufirst%3DK.%2BC.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DSelective%2520Small%2520Molecule%2520Inhibition%2520of%2520Poly%2528ADP-Ribose%2529%2520Glycohydrolase%2520%2528PARG%2529%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D563%26epage%3D570%26doi%3D10.1021%2Fcb200506t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Islam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koizumi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodera, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okawara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masutani, M.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of Phenolic Hydrazide Hydrazones as Potent Poly(ADP-ribose) Glycohydrolase (PARG) Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3802</span>– <span class="NLM_lpage">3806</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2014.06.065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1016%2Fj.bmcl.2014.06.065" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=3802-3806&author=R.+Islamauthor=F.+Koizumiauthor=Y.+Koderaauthor=K.+Inoueauthor=T.+Okawaraauthor=M.+Masutani&title=Design+and+Synthesis+of+Phenolic+Hydrazide+Hydrazones+as+Potent+Poly%28ADP-ribose%29+Glycohydrolase+%28PARG%29+Inhibitors&doi=10.1016%2Fj.bmcl.2014.06.065"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.06.065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.06.065%26sid%3Dliteratum%253Aachs%26aulast%3DIslam%26aufirst%3DR.%26aulast%3DKoizumi%26aufirst%3DF.%26aulast%3DKodera%26aufirst%3DY.%26aulast%3DInoue%26aufirst%3DK.%26aulast%3DOkawara%26aufirst%3DT.%26aulast%3DMasutani%26aufirst%3DM.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520Phenolic%2520Hydrazide%2520Hydrazones%2520as%2520Potent%2520Poly%2528ADP-ribose%2529%2520Glycohydrolase%2520%2528PARG%2529%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D3802%26epage%3D3806%26doi%3D10.1016%2Fj.bmcl.2014.06.065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Biosassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Biosassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Biosassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGonagle, A. E.</span>; <span class="NLM_string-name">Jordan, A. M.</span>; <span class="NLM_string-name">Waszkowycz, B.</span>; <span class="NLM_string-name">Hutton, C. P.</span>; <span class="NLM_string-name">Waddell, I. D.</span>; <span class="NLM_string-name">Hitchin, J. R.</span>; <span class="NLM_string-name">Smith, K. M.</span>; <span class="NLM_string-name">Hamilton, N. M.</span></span> <span> </span><span class="NLM_article-title">2,4-Dioxo-quinazoline-6-sulphonamide Derivatives as Inhibitors of PARG</span>.  <i>PCT Int. Appl.</i> <span class="NLM_patent">WO2015079251</span> June 4, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=A.+E.+McGonagle&author=A.+M.+Jordan&author=B.+Waszkowycz&author=C.+P.+Hutton&author=I.+D.+Waddell&author=J.+R.+Hitchin&author=K.+M.+Smith&author=N.+M.+Hamilton&title=2%2C4-Dioxo-quinazoline-6-sulphonamide+Derivatives+as+Inhibitors+of+PARG"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26atitle%3D2%252C4-Dioxo-quinazoline-6-sulphonamide%2520Derivatives%2520as%2520Inhibitors%2520of%2520PARG%26jtitle%3DPCT%2520Int.%2520Appl.%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGonagle, A. E.</span>; <span class="NLM_string-name">Jordan, A. M.</span>; <span class="NLM_string-name">Waszkowycz, B.</span>; <span class="NLM_string-name">Hutton, C. P.</span>; <span class="NLM_string-name">Waddell, I. D.</span>; <span class="NLM_string-name">Hitchin, J. R.</span>; <span class="NLM_string-name">Smith, K. M.</span>; <span class="NLM_string-name">Hamilton, N. M.</span></span> <span> </span><span class="NLM_article-title">PARG Inhibitory Compounds</span>.  <i>PCT Int. Appl.</i> <span class="NLM_patent">WO2016097749</span> June 23, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=A.+E.+McGonagle&author=A.+M.+Jordan&author=B.+Waszkowycz&author=C.+P.+Hutton&author=I.+D.+Waddell&author=J.+R.+Hitchin&author=K.+M.+Smith&author=N.+M.+Hamilton&title=PARG+Inhibitory+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26atitle%3DPARG%2520Inhibitory%2520Compounds%26jtitle%3DPCT%2520Int.%2520Appl.%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipshutz, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corral, R.</span></span> <span> </span><span class="NLM_article-title">Simplification of the Mitsunobu Reaction. Di-p-chlorobenzyl Azodicarboxylate:  A New Azodicarboxylate</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">5069</span>– <span class="NLM_lpage">5072</span>, <span class="refDoi"> DOI: 10.1021/ol0618757</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol0618757" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2006&pages=5069-5072&author=B.+H.+Lipshutzauthor=D.+W.+Chungauthor=B.+Richauthor=R.+Corral&title=Simplification+of+the+Mitsunobu+Reaction.+Di-p-chlorobenzyl+Azodicarboxylate%3A%E2%80%89+A+New+Azodicarboxylate&doi=10.1021%2Fol0618757"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fol0618757&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol0618757%26sid%3Dliteratum%253Aachs%26aulast%3DLipshutz%26aufirst%3DB.%2BH.%26aulast%3DChung%26aufirst%3DD.%2BW.%26aulast%3DRich%26aufirst%3DB.%26aulast%3DCorral%26aufirst%3DR.%26atitle%3DSimplification%2520of%2520the%2520Mitsunobu%2520Reaction.%2520Di-p-chlorobenzyl%2520Azodicarboxylate%253A%25E2%2580%2589%2520A%2520New%2520Azodicarboxylate%26jtitle%3DOrg.%2520Lett.%26date%3D2006%26volume%3D8%26spage%3D5069%26epage%3D5072%26doi%3D10.1021%2Fol0618757" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">James, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durant, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckersley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairweather, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffiths, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shea, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddell, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, D. J.</span></span> <span> </span><span class="NLM_article-title">An Assay to Measure Poly(ADP ribose) Glycohydrolase (PARG) Activity in Cells</span>. <i>F1000Research</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">736</span>– <span class="NLM_lpage">746</span>, <span class="refDoi"> DOI: 10.12688/f1000research.8463.2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.12688%2Ff1000research.8463.2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=27610220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A280%3ADC%252BC2szptVyqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=736-746&author=D.+I.+Jamesauthor=S.+Durantauthor=K.+Eckersleyauthor=E.+Fairweatherauthor=L.+A.+Griffithsauthor=N.+Hamiltonauthor=P.+Kellyauthor=M.+O%E2%80%99Connorauthor=K.+Sheaauthor=I.+D.+Waddellauthor=D.+J.+Ogilvie&title=An+Assay+to+Measure+Poly%28ADP+ribose%29+Glycohydrolase+%28PARG%29+Activity+in+Cells&doi=10.12688%2Ff1000research.8463.2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">An assay to measure poly(ADP ribose) glycohydrolase (PARG) activity in cells</span></div><div class="casAuthors">James Dominic I; Fairweather Emma; Griffiths Louise A; Hamilton Nicola; Kelly Paul; Waddell Ian D; Ogilvie Donald J; Durant Stephen; Eckersley Kay; O'Connor Mark; Shea Kerry</div><div class="citationInfo"><span class="NLM_cas:title">F1000Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">736</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">After a DNA damage signal multiple polymers of ADP ribose attached to poly(ADP) ribose (PAR) polymerases (PARPs) are broken down by the enzyme poly(ADP) ribose glycohydrolase (PARG).  Inhibition of PARG leads to a failure of DNA repair and small molecule inhibition of PARG has been a goal for many years.  To determine whether biochemical inhibitors of PARG are active in cells we have designed an immunofluorescence assay to detect nuclear PAR after DNA damage.  This 384-well assay is suitable for medium throughput high-content screening and can detect cell-permeable inhibitors of PARG from nM to μM potency.  In addition, the assay has been shown to work in murine cells and in a variety of human cancer cells.  Furthermore, the assay is suitable for detecting the DNA damage response induced by treatment with temozolomide and methylmethane sulfonate (MMS).  Lastly, the assay has been shown to be robust over a period of several years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSzT50tY2lEhWBAiYWy6n3SfW6udTcc2eZtHz3jiTR42Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szptVyqsA%253D%253D&md5=e51d923a92a7a1fd09c578c2ee9cf917</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.12688%2Ff1000research.8463.2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.12688%252Ff1000research.8463.2%26sid%3Dliteratum%253Aachs%26aulast%3DJames%26aufirst%3DD.%2BI.%26aulast%3DDurant%26aufirst%3DS.%26aulast%3DEckersley%26aufirst%3DK.%26aulast%3DFairweather%26aufirst%3DE.%26aulast%3DGriffiths%26aufirst%3DL.%2BA.%26aulast%3DHamilton%26aufirst%3DN.%26aulast%3DKelly%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DM.%26aulast%3DShea%26aufirst%3DK.%26aulast%3DWaddell%26aufirst%3DI.%2BD.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26atitle%3DAn%2520Assay%2520to%2520Measure%2520Poly%2528ADP%2520ribose%2529%2520Glycohydrolase%2520%2528PARG%2529%2520Activity%2520in%2520Cells%26jtitle%3DF1000Research%26date%3D2016%26volume%3D5%26spage%3D736%26epage%3D746%26doi%3D10.12688%2Ff1000research.8463.2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gean, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katzhendle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Shoshan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ringel, I.</span></span> <span> </span><span class="NLM_article-title">Preparation, Characterization, and Anticancer Activity of a Series of <i>cis</i>-PtCl<sub>2</sub> Complexes Linked to Anthraquinone Intercalators</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">414</span>– <span class="NLM_lpage">420</span>, <span class="refDoi"> DOI: 10.1021/jm00105a063</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00105a063" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADyaK3MXlsVOgtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1991&pages=414-420&author=D.+Gibsonauthor=K.+F.+Geanauthor=J.+Katzhendleauthor=R.+Ben-Shoshanauthor=A.+Ramuauthor=I.+Ringel&title=Preparation%2C+Characterization%2C+and+Anticancer+Activity+of+a+Series+of+cis-PtCl2+Complexes+Linked+to+Anthraquinone+Intercalators&doi=10.1021%2Fjm00105a063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Preparation, characterization, and anticancer activity of a series of cis-PtCl2 complexes linked to anthraquinone intercalators</span></div><div class="casAuthors">Gibson, Dan; Gean, Keria Fiorella; Katzhendle, Jehoshua; Ben-Shoshan, Raphael; Ramu, Avner; Ringel, Israel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">414-20</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">cis-PtL2X2 [L = a monodentate AQZ(CH2)nNH2 and L2 = a bidentate AQ-Z(CH2)nNH(CH2)2NH2; AQ = anthraquinonyl, X = Cl, I, Z = NH, O] in which anthraquinone intercalators are tethered to the cis-PtCl2 unit via an (aminoalkyl)amino, (oxyalkyl)amino, or polyethylene glycol (aminoethyl)amino linker chains was prepd. and screened in vitro against P388 leukemia.  In vivo toxicity studies were carried out on selected complexes.  All complexes were characterized by means of elemental anal., and 195Pt NMR and FTIR spectroscopies.  The 1:1 Pt-intercalcalator complexes displayed much higher in vitro cytotoxic activities than the 1:2 Pt-intercalator complexes.  The X = Cl complexes were consistently more active than their X = I counterparts.  Among the 1:1 Pt-intercalator complexes those with the shorter linker chains (n = 2, 3) exhibited the highest cytotoxic activities.  cis-PtL2Cl2 (L = 2-[[2-(anthraquinon-1-ylamino)ethyl]amino]ethyl]amine, C2-[[3-(anthraquinon-1-ylamino)propyl]amino]ethyl]amine, and [2-[[3-(anthraquinon-1-yloxy)propyl]amino]ethyl]amine-N,N']) were as active in vitro as cisplatin (ED50 = 2-4 × 10-7 M) while on a molar basis their acute in vivo toxicity was lower than that of cisplatin.  In vivo screening against P388 leukemia indicated that these complexes have activity comparable to cisplatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtfcLsaH7LE7Vg90H21EOLACvtfcHk0lhQEM97IMlVvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXlsVOgtw%253D%253D&md5=170132b6f7c62b91af0abd77ae6715e9</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm00105a063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00105a063%26sid%3Dliteratum%253Aachs%26aulast%3DGibson%26aufirst%3DD.%26aulast%3DGean%26aufirst%3DK.%2BF.%26aulast%3DKatzhendle%26aufirst%3DJ.%26aulast%3DBen-Shoshan%26aufirst%3DR.%26aulast%3DRamu%26aufirst%3DA.%26aulast%3DRingel%26aufirst%3DI.%26atitle%3DPreparation%252C%2520Characterization%252C%2520and%2520Anticancer%2520Activity%2520of%2520a%2520Series%2520of%2520cis-PtCl2%2520Complexes%2520Linked%2520to%2520Anthraquinone%2520Intercalators%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1991%26volume%3D34%26spage%3D414%26epage%3D420%26doi%3D10.1021%2Fjm00105a063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="note"><p class="first last">Lipophilic ligand efficiency calculated as biochemical pEC<sub>50</sub> – XlogP.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Talele, T. T.</span></span> <span> </span><span class="NLM_article-title">The “Cyclopropyl Fragment” is a Versatile Player that Frequently Appears in Preclinical/Clinical Drug Molecules</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8712</span>– <span class="NLM_lpage">8756</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00472</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00472" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslWguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8712-8756&author=T.+T.+Talele&title=The+%E2%80%9CCyclopropyl+Fragment%E2%80%9D+is+a+Versatile+Player+that+Frequently+Appears+in+Preclinical%2FClinical+Drug+Molecules&doi=10.1021%2Facs.jmedchem.6b00472"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">The "Cyclopropyl Fragment" is a Versatile Player that Frequently Appears in Preclinical/Clinical Drug Molecules</span></div><div class="casAuthors">Talele, Tanaji T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8712-8756</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, there has been an increasing use of the cyclopropyl ring in drug development to transition drug candidates from the preclin. to clin. stage.  Important features of the cyclopropane ring are, the (1) coplanarity of the three carbon atoms, (2) relatively shorter (1.51 Å) C-C bonds, (3) enhanced π-character of C-C bonds, and (4) C-H bonds are shorter and stronger than those in alkanes.  The present review will focus on the contributions that a cyclopropyl ring makes to the properties of drugs contg. it.  Consequently, the cyclopropyl ring addresses multiple roadblocks that can occur during drug discovery such as (a) enhancing potency, (b) reducing off-target effects, (c) increasing metabolic stability, (d) increasing brain permeability, (e) decreasing plasma clearance, (f) contributing to an entropically more favorable binding to the receptor, (g) conformational restriction of peptides/peptidomimetics to prevent proteolytic hydrolysis, and (h) altering drug pKa to reduce its P-glycoprotein efflux ratio.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVIJAG1XVRG7Vg90H21EOLACvtfcHk0lhQEM97IMlVvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslWguro%253D&md5=b5586458ec4ab5e2d7586f5739663d1f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00472%26sid%3Dliteratum%253Aachs%26aulast%3DTalele%26aufirst%3DT.%2BT.%26atitle%3DThe%2520%25E2%2580%259CCyclopropyl%2520Fragment%25E2%2580%259D%2520is%2520a%2520Versatile%2520Player%2520that%2520Frequently%2520Appears%2520in%2520Preclinical%252FClinical%2520Drug%2520Molecules%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8712%26epage%3D8756%26doi%3D10.1021%2Facs.jmedchem.6b00472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span> <i>PARG inhibitor
PDD00017273 Small Molecule (Tool Compound)</i>; <span class="NLM_publisher-name">Ximbio</span>, <span class="NLM_year">2018</span>; <a href="https://ximbio.com/reagent/152787/parg-inhibitor-pdd00017273-small-molecule-tool-compound" class="extLink">https://ximbio.com/reagent/152787/parg-inhibitor-pdd00017273-small-molecule-tool-compound</a> (accessed Oct 24, 2018.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+PARG+inhibitor%0APDD00017273+Small+Molecule+%28Tool+Compound%29%3B+Ximbio%2C+2018%3B+https%3A%2F%2Fximbio.com%2Freagent%2F152787%2Fparg-inhibitor-pdd00017273-small-molecule-tool-compound+%28accessed+Oct+24%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DPARG%2520inhibitor%250APDD00017273%2520Small%2520Molecule%2520%2528Tool%2520Compound%2529%26pub%3DXimbio%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span> See, for example<span class="NLM_contrib-group"><span class="NLM_string-name">Gogola, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duarte, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Ruiter, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiegant, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bruijn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerrero Llobet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vis, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annunziato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den Broek, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barazas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kersbergen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van de Ven, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarsounas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vugt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wessels, L. F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartkova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gromova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andújar-Sánchez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Attikum, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonkers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rottenberg, S.</span></span> <span> </span><span class="NLM_article-title">Selective Loss of PARG Restores PARylation and Counteracts PARP Inhibitor-Mediated Synthetic Lethality</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1078</span>– <span class="NLM_lpage">1093</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2018.05.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1016%2Fj.ccell.2018.05.008" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2018&pages=1078-1093&author=E.+Gogolaauthor=A.+A.+Duarteauthor=J.+R.+de+Ruiterauthor=W.+W.+Wiegantauthor=J.+A.+Schmidauthor=R.+de+Bruijnauthor=D.+I.+Jamesauthor=S.+Guerrero+Llobetauthor=D.+J.+Visauthor=S.+Annunziatoauthor=B.+van+den+Broekauthor=M.+Barazasauthor=A.+Kersbergenauthor=M.+van+de+Venauthor=M.+Tarsounasauthor=D.+J.+Ogilvieauthor=M.+van+Vugtauthor=L.+F.+A.+Wesselsauthor=J.+Bartkovaauthor=I.+Gromovaauthor=M.+And%C3%BAjar-S%C3%A1nchezauthor=J.+Bartekauthor=M.+Lopesauthor=H.+van+Attikumauthor=P.+Borstauthor=J.+Jonkersauthor=S.+Rottenberg&title=Selective+Loss+of+PARG+Restores+PARylation+and+Counteracts+PARP+Inhibitor-Mediated+Synthetic+Lethality&doi=10.1016%2Fj.ccell.2018.05.008"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2018.05.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2018.05.008%26sid%3Dliteratum%253Aachs%26aulast%3DGogola%26aufirst%3DE.%26aulast%3DDuarte%26aufirst%3DA.%2BA.%26aulast%3Dde%2BRuiter%26aufirst%3DJ.%2BR.%26aulast%3DWiegant%26aufirst%3DW.%2BW.%26aulast%3DSchmid%26aufirst%3DJ.%2BA.%26aulast%3Dde%2BBruijn%26aufirst%3DR.%26aulast%3DJames%26aufirst%3DD.%2BI.%26aulast%3DGuerrero%2BLlobet%26aufirst%3DS.%26aulast%3DVis%26aufirst%3DD.%2BJ.%26aulast%3DAnnunziato%26aufirst%3DS.%26aulast%3Dvan%2Bden%2BBroek%26aufirst%3DB.%26aulast%3DBarazas%26aufirst%3DM.%26aulast%3DKersbergen%26aufirst%3DA.%26aulast%3Dvan%2Bde%2BVen%26aufirst%3DM.%26aulast%3DTarsounas%26aufirst%3DM.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26aulast%3Dvan%2BVugt%26aufirst%3DM.%26aulast%3DWessels%26aufirst%3DL.%2BF.%2BA.%26aulast%3DBartkova%26aufirst%3DJ.%26aulast%3DGromova%26aufirst%3DI.%26aulast%3DAnd%25C3%25BAjar-S%25C3%25A1nchez%26aufirst%3DM.%26aulast%3DBartek%26aufirst%3DJ.%26aulast%3DLopes%26aufirst%3DM.%26aulast%3Dvan%2BAttikum%26aufirst%3DH.%26aulast%3DBorst%26aufirst%3DP.%26aulast%3DJonkers%26aufirst%3DJ.%26aulast%3DRottenberg%26aufirst%3DS.%26atitle%3DSelective%2520Loss%2520of%2520PARG%2520Restores%2520PARylation%2520and%2520Counteracts%2520PARP%2520Inhibitor-Mediated%2520Synthetic%2520Lethality%26jtitle%3DCancer%2520Cell%26date%3D2018%26volume%3D33%26spage%3D1078%26epage%3D1093%26doi%3D10.1016%2Fj.ccell.2018.05.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="note"><p class="first last">See <a href="/doi/suppl/10.1021/acs.jmedchem.8b01407/suppl_file/jm8b01407_si_001.pdf" class="ext-link">Supporting Information</a> for correlation between biochemical and cellular pEC<sub>50</sub>s for the whole quinazolinedione series.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span> <i>PDD00017273</i>; <span class="NLM_publisher-name">Chemical
Probes Portal</span>, <span class="NLM_year">2018</span>; <a href="http://www.chemicalprobes.org/pdd00017273" class="extLink">http://www.chemicalprobes.org/pdd00017273</a> (accessed Oct 24, 2018).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+PDD00017273%3B+Chemical%0AProbes+Portal%2C+2018%3B+http%3A%2F%2Fwww.chemicalprobes.org%2Fpdd00017273+%28accessed+Oct+24%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DPDD00017273%26pub%3DChemical%250AProbes%2520Portal%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span>XLogP calculated
using <i>Dotmatics Elemental
Suite</i>; <span class="NLM_publisher-name">Dotmatics</span>: <span class="NLM_publisher-name">Bishops Stortford, UK</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=XLogP+calculated%0Ausing+Dotmatics+Elemental%0ASuite%3B+Dotmatics%3A+Bishops+Stortford%2C+UK."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DDotmatics%2520Elemental%250ASuite%26pub%3DDotmatics%26pub%3DBishops%2520Stortford%252C%2520UK" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, D. M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monaco, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winters, M. P.</span></span> <span> </span><span class="NLM_article-title">New N- and O-arylations with Phenylboronic Acids and Cupric Acetate</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">2933</span>– <span class="NLM_lpage">2936</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(98)00503-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=10.1016%2FS0040-4039%2898%2900503-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;key=1%3ACAS%3A528%3ADyaK1cXjtVOlu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1998&pages=2933-2936&author=D.+M.+T.+Chanauthor=K.+L.+Monacoauthor=R.-P.+Wangauthor=M.+P.+Winters&title=New+N-+and+O-arylations+with+Phenylboronic+Acids+and+Cupric+Acetate&doi=10.1016%2FS0040-4039%2898%2900503-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">New N- and O-arylation with phenylboronic acids and cupric acetate</span></div><div class="casAuthors">Chan, Dominic M. T.; Monaco, Kevin L.; Wang, Ru-Ping; Winters, Michael P.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2933-2936</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A new method of arylating N-H and O-H contg. compds. at room temp. with phenylboronic acids and cupric acetate in the presence of a tertiary amine promoter is described.  Substrates include phenols, amines, anilines, amides, imides, ureas, carbamates, and sulfonamides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQZxWi8FJntLVg90H21EOLACvtfcHk0ljED65Z4BkiMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjtVOlu7w%253D&md5=56efb58adc883aba3ca4ab908f6864ec</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2898%2900503-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252898%252900503-6%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DD.%2BM.%2BT.%26aulast%3DMonaco%26aufirst%3DK.%2BL.%26aulast%3DWang%26aufirst%3DR.-P.%26aulast%3DWinters%26aufirst%3DM.%2BP.%26atitle%3DNew%2520N-%2520and%2520O-arylations%2520with%2520Phenylboronic%2520Acids%2520and%2520Cupric%2520Acetate%26jtitle%3DTetrahedron%2520Lett.%26date%3D1998%26volume%3D39%26spage%3D2933%26epage%3D2936%26doi%3D10.1016%2FS0040-4039%2898%2900503-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McLeod, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boudreault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, Y.</span></span> <span> </span><span class="NLM_article-title">Synthetic Application of Monoprotected Hydrazines toward the Synthesis of 1-Aminopyrroles</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1180</span>– <span class="NLM_lpage">1183</span>, <span class="refDoi"> DOI: 10.1021/jo9518260</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo9518260" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=1996&pages=1180-1183&author=M.+McLeodauthor=N.+Boudreaultauthor=Y.+Leblanc&title=Synthetic+Application+of+Monoprotected+Hydrazines+toward+the+Synthesis+of+1-Aminopyrroles&doi=10.1021%2Fjo9518260"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjo9518260&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo9518260%26sid%3Dliteratum%253Aachs%26aulast%3DMcLeod%26aufirst%3DM.%26aulast%3DBoudreault%26aufirst%3DN.%26aulast%3DLeblanc%26aufirst%3DY.%26atitle%3DSynthetic%2520Application%2520of%2520Monoprotected%2520Hydrazines%2520toward%2520the%2520Synthesis%2520of%25201-Aminopyrroles%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1996%26volume%3D61%26spage%3D1180%26epage%3D1183%26doi%3D10.1021%2Fjo9518260" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4A0D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4A0D','PDB','4A0D'); return false;">PDB: 4A0D</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMM','PDB','6HMM'); return false;">PDB: 6HMM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4B1H" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4B1H','PDB','4B1H'); return false;">PDB: 4B1H</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMN','PDB','6HMN'); return false;">PDB: 6HMN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMK','PDB','6HMK'); return false;">PDB: 6HMK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HML" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HML','PDB','6HML'); return false;">PDB: 6HML</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i178"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01407">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_26833"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b01407">10.1021/acs.jmedchem.8b01407</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">LC–MS methods and solvent gradients; preparative HPLC instrument and solvent gradients; summary of purity data; additional core-hops identified from virtual screening; preparative methods and spectroscopic data for key intermediates and compounds <b>S1</b>–<b>S17</b>; exemplar NMR and LC data; PARG cocrystal structure determination and refinement details for <b>8a</b>, <b>22h</b>, <b>27t</b>, and <b>33f</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01407/suppl_file/jm8b01407_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01407/suppl_file/jm8b01407_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01407/suppl_file/jm8b01407_si_001.pdf">jm8b01407_si_001.pdf (907.04 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01407/suppl_file/jm8b01407_si_002.csv">jm8b01407_si_002.csv (7.95 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and structure factors for X-ray crystal structures of <b>8a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMM">6HMM</a>), <b>22h</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMN">6HMN</a>), <b>27t</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HMK">6HMK</a>), and <b>33f</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HML">6HML</a>) in complex with the human PARG catalytic domain have been deposited in the Protein Data Bank.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01407&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-23%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01407%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01407" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                9MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679976bae9c7d1c7","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
