
<html lang="en"     class="pb-page"  data-request-id="04935402-c16d-4f34-8493-e66ae23002df"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;article:article:10.1021/acs.jmedchem.1c00420;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2021.64.issue-12;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Development of HDAC Inhibitors Exhibiting Therapeutic Potential in T-Cell Prolymphocytic Leukemia" /></meta><meta name="dc.Creator" content="Krimo  Toutah" /></meta><meta name="dc.Creator" content="Nabanita  Nawar" /></meta><meta name="dc.Creator" content="Sanna  Timonen" /></meta><meta name="dc.Creator" content="Helena  Sorger" /></meta><meta name="dc.Creator" content="Yasir S.  Raouf" /></meta><meta name="dc.Creator" content="Shazreh  Bukhari" /></meta><meta name="dc.Creator" content="Jana  von Jan" /></meta><meta name="dc.Creator" content="Aleksandr  Ianevski" /></meta><meta name="dc.Creator" content="Justyna M.  Gawel" /></meta><meta name="dc.Creator" content="Olasunkanmi O.  Olaoye" /></meta><meta name="dc.Creator" content="Mulu  Geletu" /></meta><meta name="dc.Creator" content="Ayah  Abdeldayem" /></meta><meta name="dc.Creator" content="Johan  Israelian" /></meta><meta name="dc.Creator" content="Tudor B.  Radu" /></meta><meta name="dc.Creator" content="Abootaleb  Sedighi" /></meta><meta name="dc.Creator" content="Muzaffar N.  Bhatti" /></meta><meta name="dc.Creator" content="Muhammad Murtaza  Hassan" /></meta><meta name="dc.Creator" content="Pimyupa  Manaswiyoungkul" /></meta><meta name="dc.Creator" content="Andrew E.  Shouksmith" /></meta><meta name="dc.Creator" content="Heidi A.  Neubauer" /></meta><meta name="dc.Creator" content="Elvin D.  de Araujo" /></meta><meta name="dc.Creator" content="Tero  Aittokallio" /></meta><meta name="dc.Creator" content="Oliver H.  Krämer" /></meta><meta name="dc.Creator" content="Richard  Moriggl" /></meta><meta name="dc.Creator" content="Satu  Mustjoki" /></meta><meta name="dc.Creator" content="Marco  Herling" /></meta><meta name="dc.Creator" content="Patrick T.  Gunning" /></meta><meta name="dc.Description" content="Epigenetic targeting has emerged as an efficacious therapy for hematological cancers. The rare and incurable T-cell prolymphocytic leukemia (T-PLL) is known for its aggressive clinical course. Curr..." /></meta><meta name="Description" content="Epigenetic targeting has emerged as an efficacious therapy for hematological cancers. The rare and incurable T-cell prolymphocytic leukemia (T-PLL) is known for its aggressive clinical course. Curr..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 8, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00420" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 The Authors. Published by American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00420" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00420" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00420" /></link>
        
    
    

<title>Development of HDAC Inhibitors Exhibiting Therapeutic Potential in T-Cell Prolymphocytic Leukemia | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00420" /></meta><meta property="og:title" content="Development of HDAC Inhibitors Exhibiting Therapeutic Potential in T-Cell Prolymphocytic Leukemia" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0010.jpeg" /></meta><meta property="og:description" content="Epigenetic targeting has emerged as an efficacious therapy for hematological cancers. The rare and incurable T-cell prolymphocytic leukemia (T-PLL) is known for its aggressive clinical course. Current epigenetic agents such as histone deacetylase (HDAC) inhibitors are increasingly used for targeted therapy. Through a structure–activity relationship (SAR) study, we developed an HDAC6 inhibitor KT-531, which exhibited higher potency in T-PLL compared to other hematological cancers. KT-531 displayed strong HDAC6 inhibitory potency and selectivity, on-target biological activity, and a safe therapeutic window in nontransformed cell lines. In primary T-PLL patient cells, where HDAC6 was found to be overexpressed, KT-531 exhibited strong biological responses, and safety in healthy donor samples. Notably, combination studies in T-PLL patient samples demonstrated KT-531 synergizes with approved cancer drugs, bendamustine, idasanutlin, and venetoclax. Our work suggests HDAC inhibition in T-PLL could afford sufficient therapeutic windows to achieve durable remission either as stand-alone or in combination with targeted drugs." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00420"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00420">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00420&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00420&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00420&amp;href=/doi/10.1021/acs.jmedchem.1c00420" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 12</span><span class="cit-fg-pageRange">, 8486-8509</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/12" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00414" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00430" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Development of HDAC Inhibitors Exhibiting Therapeutic Potential in T-Cell Prolymphocytic Leukemia</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Krimo Toutah</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Krimo Toutah</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical and Physical Sciences, University of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Krimo++Toutah">Krimo Toutah</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nabanita Nawar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nabanita Nawar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical and Physical Sciences, University of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</div><div class="loa-info-affiliations-info">Department of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nabanita++Nawar">Nabanita Nawar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sanna Timonen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sanna Timonen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hematology Research Unit Helsinki, Helsinki University Hospital Comprehensive Cancer Center, Helsinki, 00029 HUS, Finland</div><div class="loa-info-affiliations-info">Translational Immunology Research Program and Department of Clinical Chemistry and Hematology, University of Helsinki, Helsinki, 00014 Helsinki, Finland</div><div class="loa-info-affiliations-info">Institute for Molecular Medicine Finland (FIMM), HiLIFE, University of Helsinki, Helsinki, 00014 Helsinki, Finland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sanna++Timonen">Sanna Timonen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Helena Sorger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Helena Sorger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Animal Breeding and Genetics, University of Veterinary Medicine Vienna, A-1210 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Helena++Sorger">Helena Sorger</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yasir S. Raouf</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yasir S. Raouf</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical and Physical Sciences, University of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</div><div class="loa-info-affiliations-info">Department of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yasir+S.++Raouf">Yasir S. Raouf</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shazreh Bukhari</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shazreh Bukhari</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical and Physical Sciences, University of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</div><div class="loa-info-affiliations-info">Department of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shazreh++Bukhari">Shazreh Bukhari</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jana von Jan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jana von Jan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Internal Medicine, Center for Integrated Oncology Aachen-Bonn-Cologne-Duesseldorf (CIO ABCD), University of Cologne (UoC), 50923 Cologne, Germany</div><div class="loa-info-affiliations-info">Excellence Cluster for Cellular Stress Response and Aging-Associated Diseases (CECAD), UoC, 50923 Cologne, Germany</div><div class="loa-info-affiliations-info">Center for Molecular Medicine Cologne (CMMC), UoC, 50923 Cologne, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jana++von+Jan">Jana von Jan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Aleksandr Ianevski</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Aleksandr Ianevski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute for Molecular Medicine Finland (FIMM), HiLIFE, University of Helsinki, Helsinki, 00014 Helsinki, Finland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Aleksandr++Ianevski">Aleksandr Ianevski</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Justyna M. Gawel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Justyna M. Gawel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical and Physical Sciences, University of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Justyna+M.++Gawel">Justyna M. Gawel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Olasunkanmi O. Olaoye</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Olasunkanmi O. Olaoye</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical and Physical Sciences, University of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</div><div class="loa-info-affiliations-info">Department of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Olasunkanmi+O.++Olaoye">Olasunkanmi O. Olaoye</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mulu Geletu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mulu Geletu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical and Physical Sciences, University of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mulu++Geletu">Mulu Geletu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ayah Abdeldayem</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ayah Abdeldayem</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical and Physical Sciences, University of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</div><div class="loa-info-affiliations-info">Department of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ayah++Abdeldayem">Ayah Abdeldayem</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Johan Israelian</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Johan Israelian</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical and Physical Sciences, University of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</div><div class="loa-info-affiliations-info">Department of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Johan++Israelian">Johan Israelian</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tudor B. Radu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tudor B. Radu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical and Physical Sciences, University of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</div><div class="loa-info-affiliations-info">Department of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tudor+B.++Radu">Tudor B. Radu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Abootaleb Sedighi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Abootaleb Sedighi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical and Physical Sciences, University of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Abootaleb++Sedighi">Abootaleb Sedighi</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5974-1792" title="Orcid link">https://orcid.org/0000-0001-5974-1792</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Muzaffar N. Bhatti</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Muzaffar N. Bhatti</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical and Physical Sciences, University of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Muzaffar+N.++Bhatti">Muzaffar N. Bhatti</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Muhammad Murtaza Hassan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Muhammad Murtaza Hassan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical and Physical Sciences, University of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</div><div class="loa-info-affiliations-info">Department of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Muhammad+Murtaza++Hassan">Muhammad Murtaza Hassan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pimyupa Manaswiyoungkul</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pimyupa Manaswiyoungkul</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical and Physical Sciences, University of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</div><div class="loa-info-affiliations-info">Department of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pimyupa++Manaswiyoungkul">Pimyupa Manaswiyoungkul</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrew E. Shouksmith</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrew E. Shouksmith</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical and Physical Sciences, University of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+E.++Shouksmith">Andrew E. Shouksmith</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Heidi A. Neubauer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Heidi A. Neubauer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Animal Breeding and Genetics, University of Veterinary Medicine Vienna, A-1210 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Heidi+A.++Neubauer">Heidi A. Neubauer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elvin D. de Araujo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elvin D. de Araujo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centre for Medicinal Chemistry, University of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elvin+D.++de+Araujo">Elvin D. de Araujo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tero Aittokallio</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tero Aittokallio</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute for Molecular Medicine Finland (FIMM), HiLIFE, University of Helsinki, Helsinki, 00014 Helsinki, Finland</div><div class="loa-info-affiliations-info">Department of Cancer Genetics, Institute for Cancer Research, Oslo University Hospital, 0424 Oslo, Norway</div><div class="loa-info-affiliations-info">Oslo Centre for Biostatistics and Epidemiology, University of Oslo, 0316 Oslo, Norway</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tero++Aittokallio">Tero Aittokallio</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-0886-9769" title="Orcid link">https://orcid.org/0000-0002-0886-9769</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Oliver H. Krämer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Oliver H. Krämer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Toxicology, University Medical Center, 55131 Mainz, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Oliver+H.++Kr%C3%A4mer">Oliver H. Krämer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Richard Moriggl</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Richard Moriggl</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Animal Breeding and Genetics, University of Veterinary Medicine Vienna, A-1210 Vienna, Austria</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#5b293238333a293f75363429323c3c371b2d3e2f363e3f2e3532753a38753a2f"><span class="__cf_email__" data-cfemail="493b202a21283b2d6724263b202e2e25093f2c3d242c2d3c272067282a67283d">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Richard++Moriggl">Richard Moriggl</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Satu Mustjoki</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Satu Mustjoki</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hematology Research Unit Helsinki, Helsinki University Hospital Comprehensive Cancer Center, Helsinki, 00029 HUS, Finland</div><div class="loa-info-affiliations-info">Translational Immunology Research Program and Department of Clinical Chemistry and Hematology, University of Helsinki, Helsinki, 00014 Helsinki, Finland</div><div class="loa-info-affiliations-info">iCAN Digital Precision Cancer Medicine Flagship, 00014 Helsinki, Finland</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#83f0e2f7f6adeef6f0f7e9ece8eac3ebe6eff0eaede8eaade5ea"><span class="__cf_email__" data-cfemail="8af9ebfeffa4e7fff9fee0e5e1e3cae2efe6f9e3e4e1e3a4ece3">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Satu++Mustjoki">Satu Mustjoki</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marco Herling</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marco Herling</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Internal Medicine, Center for Integrated Oncology Aachen-Bonn-Cologne-Duesseldorf (CIO ABCD), University of Cologne (UoC), 50923 Cologne, Germany</div><div class="loa-info-affiliations-info">Excellence Cluster for Cellular Stress Response and Aging-Associated Diseases (CECAD), UoC, 50923 Cologne, Germany</div><div class="loa-info-affiliations-info">Center for Molecular Medicine Cologne (CMMC), UoC, 50923 Cologne, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#2f424e5d4c4001474a5d434641486f5a440244404a4341014b4a"><span class="__cf_email__" data-cfemail="472a26352428692f22352b2e292007322c6a2c28222b29692322">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marco++Herling">Marco Herling</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Patrick T. Gunning</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Patrick T. Gunning</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical and Physical Sciences, University of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</div><div class="loa-info-affiliations-info">Department of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</div><div class="loa-info-affiliations-info">Centre for Medicinal Chemistry, University of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#acdccdd8dec5cfc782cbd9c2c2c5c2cbecd9d8c3dec3c2d8c382cfcd"><span class="__cf_email__" data-cfemail="5828392c2a313b33763f2d363631363f182d2c372a37362c37763b39">[email protected]</span></a>. Tel: 905-828-5354.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Patrick+T.++Gunning">Patrick T. Gunning</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-0654-735X" title="Orcid link">https://orcid.org/0000-0003-0654-735X</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00420&amp;href=/doi/10.1021%2Facs.jmedchem.1c00420" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 12</span><span class="cit-pageRange">, 8486–8509</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 8, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 March 2021</li><li><span class="item_label"><b>Published</b> online</span>8 June 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 June 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00420" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00420</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 The Authors. Published by American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="https://acsopenscience.org/open-access/licensing-options/"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by"><a href="https://creativecommons.org/licenses/by/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_cc-1612971093887.svg" alt="CC: Creative Commons" /></img><span><strong>CC: </strong>Creative Commons</span></a><a href="https://creativecommons.org/licenses/by/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_by-1612971093257.svg" alt="BY: Credit must be given to the creator" /></img><span><strong>BY: </strong>Credit must be given to the creator</span></a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00420"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1354</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00420" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Development of HDAC Inhibitors Exhibiting Therapeutic Potential in T-Cell Prolymphocytic Leukemia&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Krimo&quot;,&quot;last_name&quot;:&quot;Toutah&quot;},{&quot;first_name&quot;:&quot;Nabanita&quot;,&quot;last_name&quot;:&quot;Nawar&quot;},{&quot;first_name&quot;:&quot;Sanna&quot;,&quot;last_name&quot;:&quot;Timonen&quot;},{&quot;first_name&quot;:&quot;Helena&quot;,&quot;last_name&quot;:&quot;Sorger&quot;},{&quot;first_name&quot;:&quot;Yasir&quot;,&quot;last_name&quot;:&quot;S. Raouf&quot;},{&quot;first_name&quot;:&quot;Shazreh&quot;,&quot;last_name&quot;:&quot;Bukhari&quot;},{&quot;first_name&quot;:&quot;Jana&quot;,&quot;last_name&quot;:&quot;von Jan&quot;},{&quot;first_name&quot;:&quot;Aleksandr&quot;,&quot;last_name&quot;:&quot;Ianevski&quot;},{&quot;first_name&quot;:&quot;Justyna&quot;,&quot;last_name&quot;:&quot;M. Gawel&quot;},{&quot;first_name&quot;:&quot;Olasunkanmi&quot;,&quot;last_name&quot;:&quot;O. Olaoye&quot;},{&quot;first_name&quot;:&quot;Mulu&quot;,&quot;last_name&quot;:&quot;Geletu&quot;},{&quot;first_name&quot;:&quot;Ayah&quot;,&quot;last_name&quot;:&quot;Abdeldayem&quot;},{&quot;first_name&quot;:&quot;Johan&quot;,&quot;last_name&quot;:&quot;Israelian&quot;},{&quot;first_name&quot;:&quot;Tudor&quot;,&quot;last_name&quot;:&quot;B. Radu&quot;},{&quot;first_name&quot;:&quot;Abootaleb&quot;,&quot;last_name&quot;:&quot;Sedighi&quot;},{&quot;first_name&quot;:&quot;Muzaffar&quot;,&quot;last_name&quot;:&quot;N. Bhatti&quot;},{&quot;first_name&quot;:&quot;Muhammad&quot;,&quot;last_name&quot;:&quot;Murtaza Hassan&quot;},{&quot;first_name&quot;:&quot;Pimyupa&quot;,&quot;last_name&quot;:&quot;Manaswiyoungkul&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;E. Shouksmith&quot;},{&quot;first_name&quot;:&quot;Heidi&quot;,&quot;last_name&quot;:&quot;A. Neubauer&quot;},{&quot;first_name&quot;:&quot;Elvin&quot;,&quot;last_name&quot;:&quot;D. de Araujo&quot;},{&quot;first_name&quot;:&quot;Tero&quot;,&quot;last_name&quot;:&quot;Aittokallio&quot;},{&quot;first_name&quot;:&quot;Oliver&quot;,&quot;last_name&quot;:&quot;H. Krämer&quot;},{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;Moriggl&quot;},{&quot;first_name&quot;:&quot;Satu&quot;,&quot;last_name&quot;:&quot;Mustjoki&quot;},{&quot;first_name&quot;:&quot;Marco&quot;,&quot;last_name&quot;:&quot;Herling&quot;},{&quot;first_name&quot;:&quot;Patrick&quot;,&quot;last_name&quot;:&quot;T. Gunning&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;08&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;8486-8509&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00420&quot;},&quot;abstract&quot;:&quot;Epigenetic targeting has emerged as an efficacious therapy for hematological cancers. The rare and incurable T-cell prolymphocytic leukemia (T-PLL) is known for its aggressive clinical course. Current epigenetic agents such as histone deacetylase (HDAC) inhibitors are increasingly used for targeted therapy. Through a structure–activity relationship (SAR) study, we developed an HDAC6 inhibitor KT-531, which exhibited higher potency in T-PLL compared to other hematological cancers. KT-531 displayed strong HDAC6 inhibitory potency and selectivity, on-target biological activity, and a safe therapeutic window in nontransformed cell lines. In primary T-PLL patient cells, where HDAC6 was found to be overexpressed, KT-531 exhibited strong biological responses, and safety in healthy donor samples. Notably, combination studies in T-PLL patient samples demonstrated KT-531 synergizes with approved cancer drugs, bendamustine, idasanutlin, and venetoclax. Our work suggests HDAC inhibition in T-PLL could afford sufficient t&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00420&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00420" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00420&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00420" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00420&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00420" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00420&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00420&amp;href=/doi/10.1021/acs.jmedchem.1c00420" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00420" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00420" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00420%26sid%3Dliteratum%253Aachs%26pmid%3D34101461%26genre%3Darticle%26aulast%3DToutah%26date%3D2021%26atitle%3DDevelopment%2Bof%2BHDAC%2BInhibitors%2BExhibiting%2BTherapeutic%2BPotential%2Bin%2BT-Cell%2BProlymphocytic%2BLeukemia%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D12%26spage%3D8486%26epage%3D8509%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (10)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291070" title="Pharmaceuticals">Pharmaceuticals</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/12" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Epigenetic targeting has emerged as an efficacious therapy for hematological cancers. The rare and incurable T-cell prolymphocytic leukemia (T-PLL) is known for its aggressive clinical course. Current epigenetic agents such as histone deacetylase (HDAC) inhibitors are increasingly used for targeted therapy. Through a structure–activity relationship (SAR) study, we developed an HDAC6 inhibitor KT-531, which exhibited higher potency in T-PLL compared to other hematological cancers. KT-531 displayed strong HDAC6 inhibitory potency and selectivity, on-target biological activity, and a safe therapeutic window in nontransformed cell lines. In primary T-PLL patient cells, where <i>HDAC6</i> was found to be overexpressed, KT-531 exhibited strong biological responses, and safety in healthy donor samples. Notably, combination studies in T-PLL patient samples demonstrated KT-531 synergizes with approved cancer drugs, bendamustine, idasanutlin, and venetoclax. Our work suggests HDAC inhibition in T-PLL could afford sufficient therapeutic windows to achieve durable remission either as stand-alone or in combination with targeted drugs.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span></span> Special Issue</h4><p class="last">Published as part of the <i>Journal of Medicinal Chemistry</i> special issue “<a href="/page/vi/epigenetics" class="ext-link">Epigenetics 2022</a>”.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65373" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65373" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Epigenetic regulation of gene expression in the onset and progression of cancer has fueled therapeutic strategies against a number of molecular targets. Histone deacetylases (HDACs) have been identified as targets for reversing cancer-associated epigenetic states.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1−4)</a> HDACs remove acetyl groups from the amino-terminal lysine residues of histones and nonhistone proteins. This can lead to the formation of a condensed chromatin structure that consequently limits the binding of transcription factors to promoter sequences. Conversely, histone acetyltransferases (HATs) reverse this activity by catalyzing the transfer of an acetyl group from acetyl-CoA to lysine residues.<a onclick="showRef(event, 'ref2 ref5'); return false;" href="javascript:void(0);" class="ref ref2 ref5">(2,5)</a> These post-translational modifications have been identified as key regulatory strategies for a multitude of processes such as signaling networks, gene expression, transcription, cell cycle, and metabolism pathways. The activity of key regulatory proteins such as the central tumor suppressor p53, STATs, or HIF family members as well as key epigenetic regulatory proteins such as p300/CBP HATs themselves is also modulated by acetylation reactions. Hence, they also represent important targets of HDAC inhibitors (HDACi).<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6−8)</a></div><div class="NLM_p">The human HDAC superfamily consists of 18 nuclear and cytoplasmic proteins, distributed into 4 distinct classes according to their sequence similarity; Class I (1, 2, 3, 8), Class IIa (4, 5, 7, 9), Class IIb (6, 10), Class III (NAD<sup>+</sup>-dependent Sir2), and Class IV (11). Defined nonhistone substrates have been identified for certain HDAC isoforms.<a onclick="showRef(event, 'ref3 ref4 ref7 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref7 ref9 ref10">(3,4,7,9,10)</a> In particular, HDAC6 is responsible for the deacetylation of α-tubulin, HSP90, Cortactin, and Peroxiredoxin. HDAC6 is also recognized as a KDAC (lysine deacetylase). As such, HDAC6 plays a key role in microtubule dynamics and chaperone activities, DNA damage repair pathways, and oncogenic stress responses through the regulation of cell migration, immune-cell synapse formation, protein trafficking, and degradation.<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(11−13)</a> The upregulation of HDAC6 activity has been directly correlated with cellular proliferation, metastasis, and mitosis, highlighting the impact of this epigenetic regulator in cancer pathogenesis.<a onclick="showRef(event, 'ref11 ref12 ref14'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref14">(11,12,14)</a></div><div class="NLM_p">In the past two decades, successful efforts have led to four HDACi receiving U.S. Food and Drug Administration (FDA) approval for hematological cancers: vorinostat (SAHA, <b>a</b>), romidepsin (FK-228, <b>b</b>), belinostat (PXD-101, <b>c</b>), and panobinostat (LBH-589, <b>d</b>), with several in clinical trials (quisinostat, <b>e</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Figure S1a</a>). In general, these HDACi exhibit broad spectrum HDAC activity, which limits their therapeutic tolerability owing to side effects such as fatigue, diarrhea, vomiting, anorexia, asthenia, weight loss, and thrombocytopenia.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> Selective HDACi are hypothesized to minimize side effects associated with pan-inhibitors.<a onclick="showRef(event, 'ref2 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref2 ref15 ref16">(2,15,16)</a> Since the identification of the first selective HDAC6 inhibitor tubacin (<b>f</b>) in 2003, a number of HDAC6-selective inhibitors have been developed, notably tubastatin A (<b>g</b>), ricolinostat (ACY-1215, <b>h</b>), citarinostat (ACY-241, <b>i</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Figure S1a</a>), KA2507, and marbostat-100. Ricolinostat, citarinostat, and KA2507 are currently under evaluation in advanced clinical trials for hematological and solid tumors.<a onclick="showRef(event, 'ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19">(17−19)</a> Although modest selectivity toward HDAC6 (5–6-fold) has been achieved over the nearest HDAC family member <i>in vitro</i>, the selectivity profile of these clinical candidates <i>in vivo</i> remains largely indiscriminatory toward other HDACs.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Newer inhibitors such as marbostat-100 have attained higher selectivity than current clinical candidates and are well-tolerated <i>in vitro</i> and <i>in vivo</i>.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> The benign phenotype of HDAC6 knockout mice suggests that HDAC6 inhibition is a safe therapeutic strategy.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Overexpression of HDAC6 and confirmed dependency profiles in multiple hematological malignancies have accelerated preclinical and clinical studies of HDAC6-selective inhibitors as single agents, and in combination with lenalidomide, pomalidomide, paclitaxel, bortezomib, or dexamethasone.<a onclick="showRef(event, 'ref18 ref20 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref18 ref20 ref24 ref25 ref26">(18,20,24−26)</a></div><div class="NLM_p">T-cell prolymphocytic leukemia (T-PLL) is one of the most aggressive forms of hematological neoplasms.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Systematic analysis, largely by sequencing efforts of T-PLL, has revealed insights into the molecular landscape of this condition, notably through recognition of recurrent <i>T-cell leukemia 1A</i> (<i>TCL1A</i>) oncogene rearrangements, damaging lesions of the <i>ataxia telangiectasia mutated</i> (<i>ATM</i>) tumor suppressor gene, and gain-of-function mutations of <i>Janus-activated kinase–signal transducer and activator of transcription factor</i> (JAK-STAT) molecules.<a onclick="showRef(event, 'ref28 ref29 ref30 ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30 ref31 ref32 ref33">(28−33)</a> These functional cancer genomic insights in T-PLL were also validated biochemically, suggesting that aberrant DNA damage and hyper-cytokine and -growth factor responses trigger neoplastic T-cell outgrowth. These responses are associated with fulminant migratory properties of T-PLL cells, and this migratory T-cell phenotype could be under HDAC6 control.<a onclick="showRef(event, 'ref27 ref32 ref34'); return false;" href="javascript:void(0);" class="ref ref27 ref32 ref34">(27,32,34)</a> Improved biological understanding has yet to translate into a therapeutic application, as patient outcomes remain poor (median overall survival <20 months), owing to the aggressive tumor growth and insufficient responses to conventional chemotherapy. The current first-line treatment for T-PLL, the monoclonal anti-CD52 antibody alemtuzumab, induces high response rates, although eventual relapses are dominant.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Pan-HDACi have received FDA approval for the treatment of other mature T-cell neoplasms such as cutaneous T-cell lymphomas (CTCL) and peripheral T-cell lymphomas (PTCL) but not for T-PLL.<a onclick="showRef(event, 'ref2 ref36 ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref2 ref36 ref37 ref38 ref39">(2,36−39)</a></div><div class="NLM_p">In the absence of drugs that confer sustained tumor control in T-PLL, we focused on studying novel HDACi scaffolds for potential therapeutic use in these patients. Recent <i>in vitro</i> studies in T-PLL have also shown impressive efficacy of pan-HDACi and marked synergisms with the MDM2 inhibitor idasanutlin.<a onclick="showRef(event, 'ref29 ref40'); return false;" href="javascript:void(0);" class="ref ref29 ref40">(29,40)</a> The intertwined mechanisms of HDAC inhibition with the activity of tumor suppressor protein p53, whose repression is a hallmark in T-PLL, may be contributing to the therapeutic benefit of epigenetic modulation in this aggressive diseased state.<a onclick="showRef(event, 'ref40 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref40 ref41 ref42">(40−42)</a> Moreover, the BH3-mimetic venetoclax (ABT-199) has emerged as a breakthrough in the treatment of hematologic neoplasms. Venetoclax can induce apoptosis by targeting the B-cell lymphoma 2 (BCL-2) family of proteins. <i>Ex vivo</i> and human studies of T-PLL patients have highlighted an enhanced sensitivity to venetoclax.<a onclick="showRef(event, 'ref41 ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref41 ref43 ref44 ref45">(41,43−45)</a></div><div class="NLM_p last">Here, we report the identification of a novel perfluorinated benzenesulfonamide HDAC inhibitor, KT-531 (<b>14</b>), which showed high <i>in vitro</i> HDAC6 selectivity (39-fold HDAC6 selectivity; compared to ∼6-fold selectivity of clinical candidates citarinostat and ricolinostat) with low nanomolar potency (IC<sub>50</sub> = 8.5 nM) against HDAC6 in a functional <i>in vitro</i> activity assay. Furthermore, KT-531 demonstrated biological potency in multiple hematological cancer cell models (acute myeloid leukemia (AML), PTCL, and T-cell acute lymphoblastic leukemia (T-ALL)) and limited cytotoxicity in nonmalignant cell types as well as no observable toxicity <i>in vivo</i> (CD-1 mice). Notably, KT-531 exhibited strong potency (IC<sub>50</sub> = 0.42 μM) in the T-ALL/T-PLL-like cell line SUP-T11. Data mining using Oncomine revealed <i>HDAC6</i> was the only HDAC member to be selectively overexpressed in T-PLL patient samples but not in other mature or immature T-cell malignancies, supporting the rationale for HDAC6-targeting in treatment of T-PLL.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> KT-531 was subsequently tested in ten primary T-PLL patient samples, where it exhibited promising drug sensitivity scores (DSS), with a therapeutic window for T-PLL over healthy donor-derived peripheral blood mononuclear cells (PBMCs). Finally, KT-531 showed high synergy with chemotherapeutic agents idasanutlin, bendamustine, and venetoclax in T-PLL patient samples, representing a novel, efficacious, and potentially safer combination for T-PLL treatment. To the best of our knowledge, KT-531 is the first HDAC6 inhibitor to show efficacy in T-PLL patient samples.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00405" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00405" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Chemistry</h3><div class="NLM_p">Herein, we describe a new class of perfluorinated benzenesulfonamide HDAC6-selective inhibitors, which originated from our recently reported HDAC6-selective inhibitor, JG-265 (<b>1</b>).<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> A comprehensive structure–activity relationship (SAR) study of <b>1</b> was designed to identify inhibitors that retained/amplified HDAC6 potency and selectivity, while improving the limited pharmacokinetic (PK) profile which precluded its advancement to preclinical studies (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a,b).<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. HDACi <b>1</b>−<b>14</b> and their <i>in vitro</i> activity and selectivity profiles. (a) Chemical structures of HDACi <b>1</b>–<b>14</b>. (b) Biochemical HDAC inhibition against recombinant enzymes (Nanosyn) and cellular cytotoxicity results in MV4-11 (AML) and MRC-9 (healthy fibroblasts). Note: **not HDAC6 selective, *selectivity ≥ indicated value (beyond limit of experiment). A color gradient is applied for visualization of differential HDAC6 activity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Synthetic pathways used to prepare the final compounds (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a) are outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. In order to synthesize a subset of the desired inhibitor library using <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>a (upper branch), 4-formyl benzoic acid was protected using benzyl bromide to form benzyl 4-formylbenzoate (<b>S1</b>) (70%), which was reductively aminated with the appropriate amines to form the desired secondary amine (<b>S2</b>, <b>S2c</b>) (83%). The corresponding amine was coupled to polyhalogenated benzenesulfonyl chloride (<b>S3</b>) via an amine sulfonylation reaction to yield the sulfonamide precursors (<b>S4a</b>, <b>S4c–S4g</b>, <b>S4i</b>) (61–91%). If the precursor polyhalogenated benzenesulfonyl chloride (<b>S3</b>) was commercially unavailable, it was synthesized via the sulfonylation of the corresponding polyhalogenated benzene with chlorosulfonic acid at 150 °C for 2 h. After removal of the benzylic ester protecting group (<b>S5a</b>, <b>S5c</b>–<b>S5g</b>, <b>S 5f’</b>, <b>S5i</b>, <b>S5l</b>) (71–98%) from the sulfonamide precursors, it was coupled to tetrahydropyranyl (THP) or benzyl-protected hydroxylamines (<b>S6a</b>, <b>S6c</b>–<b>S6g</b>, <b>S6f’</b>, <b>S6i</b>, <b>S6l</b>) (61–76%). A final deprotection using <i>H</i><sub>2</sub>(<i>g</i>), Pd/C, or 4 M HCl (35–81%) yielded the final molecules (<b>2</b>, <b>3</b>, <b>5</b>, <b>7</b>, <b>9</b>, <b>11</b>–<b>13</b>).</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Conditions for Preparation of <b>2</b>–<b>14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) Reagents and conditions to synthesize <b>2</b>–<b>9</b> and <b>11</b>–<b>14</b>. (i) BnBr, Cs<sub>2</sub>CO<sub>3</sub>, DMF, RT, 24 h; (a) (i) R–NH<sub>2</sub>, AcOH, DCE, RT, 2 h; (ii) NaBH(OAc)<sub>3</sub>, RT, 16 h; (b) R′SO<sub>2</sub>Cl, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 3–16 h, RT; (c) H<sub>2</sub>, 10% Pd/C, THF/MeOH (2:1), RT, 16 h; (g) HSO<sub>3</sub>Cl, 3 h, 150°C; (b) RNH<sub>2</sub>, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, RT, 3–16 h <b>d)</b> C<sub>12</sub>H<sub>15</sub>BrO<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, DMF, RT, 24 h; <b>e)</b> 4M HCl/dioxane, 0°C-RT, 3 h; (f) (i) (COCl)<sub>2</sub>, THF, DMF, 0°C, 1 h; (ii) H<sub>2</sub>N-OTHP, <sup>i</sup>Pr<sub>2</sub>NEt, THF, RT, 16 h; (f′) H<sub>2</sub>N-OBn, EDCI, HOBt, Et<sub>3</sub>N, DMF, RT, 16–24 h; (c) H<sub>2</sub>, 10% Pd/C, THF/MeOH (2:1), RT, 16–24 h; (e) 4M HCl/dioxane, 0°C-RT, 3 h. <b>b</b> Reagents and conditions to synthesize <b>10. (i)</b> BnBr, Cs<sub>2</sub>CO<sub>3</sub>, DMF, RT, 24 h; <b>a)</b> (i) <sup>c</sup>Pr-NH<sub>2</sub>, AcOH, DCE, RT, 2 h; (ii) NaBH(OAc)<sub>3</sub>, RT, 16 h; (b) PFBSCl, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 3–16 h, RT; (e) H<sub>2</sub>, 10% Pd/C, THF/MeOH (2:1), RT, 18 h; (f) (i) (COCl)<sub>2</sub>, THF, DMF, 0°C, 1 h; (ii) H<sub>2</sub>N-OTHP, <sup>i</sup>Pr<sub>2</sub>NEt, THF, RT, 16 h; (e) H<sub>2</sub>, 10% Pd/C, THF/MeOH (2:1), RT, 6 h.</p></p></figure><div class="NLM_p">The alternate strategy (lower branch of <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>a) was used to design compounds <b>4</b>, <b>6</b>, and <b>14</b>. Polyfluorobenzenesulfonyl chloride (prepared as previously described if not commercially available) was coupled to the respective amines to generate the desired sulfonamides (<b>S9b</b>–<b>d</b>) (68–83%), which were subsequently coupled to 4-bromomethylbenzoic <i>tert</i>-butyl ester to form the corresponding <i>tert</i>-butyl ester-protected benzoic acids (<b>S4b</b>, <b>S11a</b>,<b>b</b>) (27–82%). Following acid-mediated deprotection (<b>S5b</b>,<b>j</b>,<b>k</b>) (99%), coupling to O-(Tetrahydro-2H-pyran-2-yl)hydroxylamine yielded the protected hydroxamic acid (<b>S6b</b>,<b>j</b>,<b>k</b>) (88–98%) which was readily deprotected to reveal the final compounds <b>4</b> (50%), <b>6</b> (49%), and <b>14</b> (60%). Compound <b>1</b> was generated as previously reported.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></div><div class="NLM_p last">A similar synthetic route (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>b) was utilized to furnish 3-(((<i>N</i>-cyclopropyl-2,3,4,5,6-pentafluorophenyl)sulfonamido)methyl)-<i>N</i>-hydroxybenzamide, by beginning with the benzyl ester protection of 3-carboxybenzaldehyde (<b>S1b</b>) (98%) and reductively aminating with cyclopropylamine to yield benzyl 3-((cyclopropylamino)methyl)benzoate (<b>S2b</b>) (93%). Coupling to pentafluorobenzenesulfonyl chloride (<b>S4h</b>) (78%) and following the aforementioned deprotection yielded 3-(((<i>N</i>-cyclopropyl-2,3,4,5,6-pentafluorophenyl)sulfonamido)methyl)benzoic acid (<b>S5h</b>) (98%). The synthesis was completed through conversion to the hydroxamate ester (<b>S6h</b>) (70%) and deprotecting with acid to furnish compound <b>10</b> (66.7%).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Structure–Activity Relationships, <i>In Vitro</i> HDAC Inhibition, and <i>In Cellulo</i> Cytotoxicity</h3><div class="NLM_p">The prepared library (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a) was screened against HDAC3, 6, 8, 11 (representative of groups I, II, and IV) to determine <i>in vitro</i> activity inhibition profiles (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b).<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> Concurrently, cellular activity was analyzed in a model cancer cell line (MV4-11) and healthy fibroblasts (MRC-9) to correlate biochemical inhibition with cellular potency and therapeutic window, and other HDACi (SAHA/vorinostat, ricolinostat, and citarinostat), as well as parent compound <b>1</b>, were included for parallel comparison.</div><div class="NLM_p">First, the role of the pentafluorobenzamide moiety was investigated by substituting it with a pentafluorobenzenesulfonamide (PFBS), <b>2</b>. This resulted in a substantial loss of activity against HDAC6 (IC<sub>50</sub>[HDAC6] = 389 nM) vs <b>1</b> (IC<sub>50</sub>[HDAC6] = 5.9 nM) and a subsequent loss in cellular potency (IC<sub>50</sub>[MV4-11] = 11.07 μM) vs <b>1</b> (IC<sub>50</sub>[MV4-11] = 1.85 μM). The introduction of an acidic sulfonamide group likely incurs unfavorable electrostatic interactions with tunnel residues. This hypothesis is supported by the <i>N</i>-methylated analogue, <b>3</b>, being ∼200 fold more potent. Moreover, substitutions on the <i>N</i>-sulfonamide with alkyl groups of different sizes (−CH<sub>3</sub>, <b>3</b>; −<sup>i</sup>Pr, <b>4</b>; −<sup>c</sup>Pr, <b>5</b>) showed improved HDAC6 potency (∼200-fold for <b>3</b>, ∼100-fold for <b>4</b>, and ∼430-fold for <b>5</b>) and cellular IC<sub>50</sub>’s (0.31–1.04 μM). One of the most notable improvements included the selectivity jump from 36-fold for the parent compound <b>1</b> to 163-fold for <b>4</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b). Furthermore, <b>5</b>, with an <i>N</i>-<sup>c</sup>Pr substituent, exhibited picomolar inhibitory activity (IC<sub>50</sub> = 0.9 nM) toward HDAC6.</div><div class="NLM_p">The <i>N</i>-<sup>c</sup>Pr group of <b>5</b> was retained, and the importance of the halogenation and electrophilicity was explored next. Diminishing cellular cytotoxicity in MV4-11 cells was observed by successively removing of fluorine atoms from <b>5</b>. Although mono-, tri-, and tetrafluorobenzenesulfonamides (<b>6</b>–<b>9</b>) retained biochemical HDAC6 inhibition (IC<sub>50</sub> = 8–14 nM), cellular efficacy was reduced by ∼5–10-fold (IC<sub>50</sub> = 5–9 μM), suggesting that substitution of the phenyl sulfonamide ring is important for biological activity. HDAC selectivity profiles ranged from 20-fold for <b>8</b> to 163-fold for <b>4</b>. Selectivity toward HDAC isoforms was modulated by varying the position of the hydroxamic acid, specifically, <i>para-</i> to <i>meta-</i>, which switched selectivity toward HDAC8 (4-fold) (<b>10</b>).</div><div class="NLM_p">Although covalent modification of HDAC6 was not observed (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Figure S2</a>), the PFBS group is susceptible to nucleophilic aromatic substitution by biological nucleophiles. A previously described high-throughput <i>in vitro</i> glutathione-reactivity assay was conducted, where compounds were incubated in the presence of glutathione (GSH) and the formation of a GSH-compound adduct was monitored over time via LC/MS/MS.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Fluorinated analogues <b>3</b> and <b>4</b> were found to react readily with glutathione, which precluded further advancement (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Table S1a</a>). The exceptional picomolar HDAC6 potency of <b>5</b> and moderate GSH stability (half-life of 20 min) warranted further ADME analysis. In hepatocytes, <b>5</b> was cleared rapidly, with a half-life of 3 min (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Table S1b</a>). In contrast, <b>6</b>, lacking the <i>para</i> F, was unreactive toward GSH (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Table S1a</a>). However, in mouse plasma, <b>6</b> had a low half-life of 25 min (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Table S2a</a>).</div><div class="NLM_p">We next investigated strategies to improve stability by reducing the electrophilicity of the perfluorinated cap group. The phenyl group was substituted with a variety of halogen combinations (<b>11</b>–<b>14</b>). A homologue of <b>5</b>, missing the 4-fluoro substituent, <b>13</b>, showed significantly reduced potency toward HDAC6 (∼700-fold). The loss of HDAC6 inhibition was concomitant with reduced biological activity in the cytotoxicity experiments. Analogues where fluorine substituents were replaced with less electronegative chlorine atoms exhibited comparable HDAC6 inhibition and cellular activity, but reduced selectivity. Inhibitor <b>14</b> (KT-531) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a), with a tetrafluorobenzene substituent led to a substantial increase in stability. While no discernible reactivity with GSH was observed after 24 h (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Table S1a</a>), KT-531 showed cellular activity (IC<sub>50</sub> [MV4-11] = 0.42 μM), single digit nanomolar potency for HDAC6 (IC<sub>50</sub> = 8.5 nM), and a promising selectivity profile (39-fold over the next closest target, HDAC8) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b). KT-531 exhibited a superior <i>in vitro</i> profile when compared to the current literature HDAC6 inhibitor, Nexturastat, which had comparable HDAC6 potency (IC<sub>50</sub> = 12.4 nM) but a lower selectivity margin (19-fold; next nearest target HDAC3 with IC<sub>50</sub> = 238 nM) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Table S2b</a>).<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51,52)</a> KT-531 also demonstrated higher cytotoxicity than Nexturastat in MV4-11 cancer cells (IC<sub>50-Nexturastat</sub> = 1.68 μM, IC<sub>50-KT-531</sub> = 0.42 μM) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Table S2b</a>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b).</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. <i>In vitro</i> and <i>in silico</i> confirmation of superior HDAC6-selectivity and potency of compound <b>14</b> (KT-531). (a) X-ray crystal structure of <b>14</b>. (b) Dose–response curve representing the percent inhibition and HDAC6 selectivity of citarinostat against HDAC2–4,6,8,11 via EMSA activity assay (Nanosyn). (c) Dose–response curve representing the percent inhibition and HDAC6-selectivity of <b>14</b> against HDAC1–11. (d) <i>In vitro</i> HDAC inhibition IC<sub>50</sub> values of citarinostat (i) and <b>14</b> against HDAC1–11. A color gradient is applied for visualization of differential activity. (e) <i>In silico</i> docking binding pose of <b>5</b> in human HDAC6 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU">5EDU</a>), with Zn<sup>2+</sup> (yellow sphere), hydrogen bonds (yellow dashed line), π–π stacking (green dashed line), and salt bridges (purple dashed line) (N, blue; O, red; C, green; F, blue-green). Catalytic triad residues and portions of the protein have been omitted for clarity. (f) <i>In silico</i> docking binding pose of <b>14</b> in human HDAC6 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU">5EDU</a>), with Zn<sup>2+</sup> (yellow sphere), hydrogen bonds (yellow dashed line), π–π stacking (green dashed line), and salt bridges (purple dashed line) (N, blue; O, red; C, yellow; F, green). Catalytic triad residues and portions of the protein have been omitted for clarity. (g) <i>In silico</i> docking binding pose of <b>14</b> in human HDAC8 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1T64">1T64</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The binding kinetics profile of KT-531 was also investigated by determining the association (<i>k</i><sub>on</sub>)/dissociation (<i>k</i><sub>off</sub>) rates as well as residence time parameters. The <i>in vitro</i> residence time (1/<i>k</i><sub>obs</sub>) for the KT-531–HDAC6 complex was determined to be 23.18 min, as compared to 10.72 min for <b>1</b>. This is a significant improvement over <b>5</b>, which exhibited a residence time of 1.09 min (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Table S3</a>). Reversible ligand binding was confirmed for <b>5</b> and <b>14</b> with unchanged IC<sub>50</sub> values in the 6 h time-dependent assay (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Figure S2</a>). The <i>K</i><sub>i</sub> of KT-531 for HDAC6 was determined to be 17.3 nM in a time-dependent inhibition <i>K</i><sub>on</sub> experiment (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Figure S3</a>).</div><div class="NLM_p last">The selectivity profile of KT-531 for all HDAC families (Class I, II, and IV) was determined using an activity-based EMSA assay (Nanosyn, USA). KT-531 showed a clean selectivity profile, with an IC<sub>50</sub> of >1000 nM for HDAC1–5, 7, and 9–11 and modest potency against HDAC8 (IC<sub>50</sub> = 334 nM). This ∼40-fold selectivity was higher than that of HDAC6-selective clinical candidates, ricolinostat and citarinostat (both 5–6-fold selective, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>b–d). Unlike KT-531, both clinical candidates inhibit Class I HDACs, specifically HDAC3 (IC<sub>50</sub> of citarinostat = 12.5 nM).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> <i>In Silico</i> Modeling and Docking Studies</h3><div class="NLM_p">Docking simulations between <i>hs</i>HDAC6 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU">5EDU</a>, catalytic domain 2) and <b>5</b> using Schrödinger’s Maestro v11.9 (Schrödinger, LLC, New York, 2019) were performed to understand the role of the PFBS ring in deriving potency toward HDAC6 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>e and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">S4</a>).<a onclick="showRef(event, 'ref53 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref53 ref54 ref55">(53−55)</a> The core interactions in the catalytic tunnel included sandwich-type π–π interactions of <b>5</b> with Phe620 and Phe680, coupled with an additional T-shaped π–π stacking of the PFBS ring with Phe679 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>e). Hydrogen bonding interactions between the fluorine substituents and the nearby backbone of Phe679 likely contributes to the increase in overall affinity. The flexible sulfonyl center and the benzylic −CH<sub>2</sub>– kink atom serve to accommodate these interactions (Δ<i>G</i><sub>B</sub> = −8.84 kcal mol<sup>–1</sup>). The fluorine atoms were deemed crucial for efficacy and HDAC6 pocket-occupancy, as replacement of the <i>para</i> fluorine to a chlorine atom led to a significant decline in potency (∼77-fold loss in potency from 0.9 to 70 nM). Through docking simulations of <b>13</b>, loss of activity was predicted to be due to the bulkier chlorine substituent not being well accommodated in the HDAC6 catalytic tunnel outlet (Δ<i>G</i><sub>B</sub> = −6.67 kcal mol<sup>–1</sup>) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Figure S5</a>).</div><div class="NLM_p last">Similar analyses of <b>14</b> (KT-531), showed the same key interactions with Phe620 and Phe680 in the HDAC6 pocket (Δ<i>G</i><sub>B</sub> = −8.30 kcal mol<sup>–1</sup>), and hydrogen bonding with the nearby Phe679 amide, despite the loss of F (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>f). When docked in the HDAC8 catalytic pocket, KT-531 lost the tunnel π–π stacking interactions, and only retained a T-shaped stacking interaction with the proximal Tyr100 (Δ<i>G</i><sub>B</sub> = −5.70 kcal mol<sup>–1</sup>) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>g). The fluorinated ring was unable to effectively bind to the opening of a catalytic tunnel characterized by an intractable, sterically hindered topology. Although π–F interactions are also likely, Maestro does not have the functionality to distinguish such interactions. Overall, these findings provide a structural rationale for the high potency of KT-531 toward HDAC6 (IC<sub>50</sub> = 8.5 nM), and the corresponding lower affinity toward HDAC8 (IC<sub>50</sub> = 334 nM).</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> <i>In Cellulo</i> Pharmacology</h3><div class="NLM_p">KT-531 was examined for its ability to induce acetylation of α-tubulin, a prominent substrate of HDAC6, and histone H3 (HDAC Class I substrate).<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Initially, model cancer cells (HeLa) were subjected to 6 h incubation with increasing doses of KT-531 or citarinostat and target engagement was analyzed via immunofluorescence assays. In KT-531-treated cells, significant acetylation of α-tubulin was observed, with no observable change in acetylation of histone H3 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a,b). Citarinostat induced weaker acetylation as compared to KT-531 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c,d).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. <i>In cellulo</i> pharmacology of compound <b>14</b> (KT-531) in comparison to citarinostat. (a) Immunofluorescence analysis of α-tubulin acetylation and histone H3 acetylation following 6 h treatment of KT-531 in cells. (acetylated α-tubulin in red, acetylated histone H3 in green, and nuclear stain DAPI in blue). This data is representative of three independent experiments. (b) Quantification of fluorescent signals that correspond to levels of acetylated α-tubulin and acetylated histone H3 in cells dosed with KT-531 at indicated concentrations. (c) Immunofluorescence analysis of α-tubulin acetylation and histone H3 acetylation following 6 h treatment of citarinostat in cells (acetylated α-tubulin in red, acetylated histone H3 in green, and nuclear stain DAPI in blue). This data is representative of three independent experiments. (d) Quantification of fluorescent signals that correspond to levels of acetylated α-tubulin and acetylated histone H3 in cells dosed with citarinostat at indicated concentrations. *<i>p</i> ≤ 0.05; ****<i>p</i> ≤ 0.001; ns, nonsignificant; two-way ANOVA with Tukey’s multiple comparisons test. (e) Western blot illustrating α-tubulin acetylation and histone H3 acetylation levels in MV4-11 AML cells following 6 h treatment with varying concentrations of KT-531 and clinical candidate citarinostat (i). Protein extracts were prepared, resolved by SDS-PAGE and immunoblotted with acetylated α-tubulin, acetylated histone H3, and HSC70 antibodies. A representative Western blot of three independent experiments is shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next, KT-531-treatment of a hematopoietic cancer cell line, MV4-11, revealed that, at higher inhibitor concentrations, acetylated α-tubulin levels were comparable between citarinostat and KT-531 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>d and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>e). Off-target acetylated histone H3 levels were higher with citarinostat in comparison to KT-531, aligning with <i>in vitro</i> findings (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>d and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>e).</div><div class="NLM_p">To understand the origin of the cytotoxic activity, fluorescence-activated cell sorting (FACS) was employed to evaluate KT-531 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a,b). Annexin V/PI staining revealed 15% of cells underwent early apoptosis (Annexin V+/PI) upon 18 h treatment with 0.5 μM KT-531, in contrast to 9% for citarinostat at the same dose (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">S6</a>). At 2 μM KT-531, only 8% of cells remained healthy and nonapoptotic, whereas 47% of cells remained as nonapoptotic following 2 μM treatment with citarinostat. Similar trends were observed for late apoptotic populations (Annexin V+/PI+) (22% for KT-531 versus 10% for citarinostat at a 1 μM dose). Cleavage of PARP-1 and Caspase-3 at 0.5 μM treatment of KT-531 confirmed the apoptotic response signature (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>d). The absence of cleaved Caspase-3 and PARP-1 at higher concentrations of citarinostat (5 μM) recapitulated FACS findings that KT-531 (<b>14</b>) induces apoptosis more strongly than the clinical molecule (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Figure S7</a>).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>14</b> (KT-531) induces apoptosis in a dose-dependent manner. (a) Flow cytometric data of MV4-11 cells treated with increasing concentrations of KT-531 (0–4 μM) for 18 h and analyzed for apoptosis via Annexin V/PI staining. Representative dot blots from three independent experiments are presented. (b) Percentage of cell populations in healthy, early apoptosis, and late apoptosis phases following treatment with KT-531 for 18 h at indicated concentrations. (c) Percentage of cell populations in healthy, early apoptosis, and late apoptosis phases following treatment with citarinostat for 18 h at indicated concentrations. Flow cytometric data for citarinostat is available in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Figure S6</a>. (d) Cleavage of caspase-3 and PARP-1 upon dose-escalation of KT-531 for 6 h in MV4-11 cells. Protein extracts were prepared and subjected to SDS-PAGE and immunoblotting with procaspase 3, cleaved caspase-3, PARP-1, cleaved PARP-1, and HSC70 antibodies. A representative Western blot of three independent experiments is shown. Data for citarinostat is available in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Figure S7</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Therapeutic Potential of KT-531 in T-Cell Neoplasms</h3><div class="NLM_p">Similarities in the mutational landscape of myeloproliferative and lymphoid neoplasms suggest that similar treatments might be effective.<a onclick="showRef(event, 'ref57 ref58'); return false;" href="javascript:void(0);" class="ref ref57 ref58">(57,58)</a> Since biological activity was demonstrated in AML, KT-531 was further assessed in a collection of PTCL (<i>n</i> = 8) and T-ALL (<i>n</i> = 3) model cell lines covering various subtypes to understand if efficacy was transferable (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. KT-531 shows selective cellular cytotoxicity in PTCL malignancies. Heat map of IC<sub>50</sub> values calculated from drug response analysis of KT-531 (<b>14</b>) and citarinostat from one representative out of three independent experiments. Cell lines are classified according to the respective disease subtype. Abbreviations: AML, acute myeloid leukemia; ANKL, aggressive NK-leukemia; LGL, large granular lymphocyte leukemia; T-NHL, T-cell non-Hodgkin’s lymphoma; ALK<sup>+</sup> ALCL, anaplastic large cell lymphoma (anaplastic lymphoma kinase positive); CTCL, cutaneous T-cell lymphoma; T-ALL, T-cell acute lymphoblastic leukemia.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">KT-531 displayed selective cytotoxicity in different PTCL cell lines. A striking observation of this screen in PTCL and T-ALL models was the indiscriminate behavior of citarinostat toward all branches of these malignancies, in contrast to the more selective cellular behavior of KT-531 in certain subtypes. Unlike citarinostat and FDA-approved pan-HDACi romidepsin and vorinostat, KT-531 was not very active in CTCL models, such as Myla and Hut78 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).<a onclick="showRef(event, 'ref40 ref43 ref59'); return false;" href="javascript:void(0);" class="ref ref40 ref43 ref59">(40,43,59)</a> Another distinction of KT-531 from citarinostat was its strong activity in the T-ALL/T-PLL-like cell line SUP-T11 (IC<sub>50</sub> of 0.42 μM; 9-fold more potent than citarinostat). SUP-T11 (derived from a patient with mature T-ALL) harbors a TRA/TRD-TCL1A translocation and consequent aberrant expression of the TCL1A protein, which is considered the genetic hallmark of T-PLL. SUP-T11 is, hence, the closest T-PLL-model derived from a patient so far.<a onclick="showRef(event, 'ref57 ref59 ref60'); return false;" href="javascript:void(0);" class="ref ref57 ref59 ref60">(57,59,60)</a> These findings also correlate with the HDAC6 mRNA expression levels available in Broad Institute Cancer Cell Line Encyclopedia (CCLE; <a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Figure S8</a>). The poor activity of KT-531 in Hut78 cells is consistent with the relatively low HDAC6 expression. The highest sensitivity to KT-531 was observed for T-ALL cell models SUP-T11, DND-41 and MOLT4, which had much higher HDAC6 expression levels. Indeed, the sensitivity of HDAC6 inhibitor KT-531 exhibits significant correlation with the available HDAC6 mRNA expression levels in T-cell lines (<i>R</i><sup>2</sup> = 0.9119, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Figure S8b</a>).</div><div class="NLM_p">The inhibitors were also assessed in nontransformed cellular systems to determine their therapeutic window. KT-531 displayed an IC<sub>50</sub> of 22 μM in MRC-9 (lung) cells, revealing a clear therapeutic window. This supports the improved safety profile of HDAC6-targeting inhibitors compared to pan-HDACi. NHF (normal human fibroblasts), HUVEC (primary human umbilical vein endothelial cells), and pPF (primary pooled fibroblasts from 250 donors) were also assessed against citarinostat and KT-531, and while still possessing a therapeutic window both molecules were significantly more potent in these normal cell types (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a). The cellular safety margins for KT-531 are comparable to those of citarinostat and the FDA-approved HDACi SAHA (vorinostat) and belinostat (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Table S4</a>). In addition, the safety of KT-531 was tested <i>in vivo</i> with a 5 day treatment of CD-1 mice (20 mg/kg via P.O. (oral administration)), which revealed no visible signs of toxicity or weight loss (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>b).</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Preclinical findings show promising therapeutic window and pharmacokinetic properties of <b>14</b> (KT-531). (a) Cytotoxicity of KT-531 (<b>14</b>) and citarinostat in healthy noncancerous cell lines (MRC-9, normal human fibroblasts (NHF), human umbilical vein endothelial cells (HUVEC), primary pooled fibroblasts from 250 donors (pPF)). (b) Average mouse weight of CD-1 mice (20 mg/kg, P.O.) dosed with KT-531 (<b>14</b>) and vehicle daily for 5 days. (c) Pharmacokinetic assessment of KT-531 (<b>14</b>) in mouse hepatocytes, PAMPA assay, mouse plasma and <i>in vivo</i> in CD-1 mice (IP, 20 mg/kg).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> HDAC Expression in T-PLL Tumor Cells</h3><div class="NLM_p">Oncomine database mining of published T-PLL patient gene expression data in relation to healthy T-cells revealed specific overexpression of <i>HDAC6</i> (1.7-fold) compared to other HDAC family members in T-PLL (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>a).<a onclick="showRef(event, 'ref46 ref59'); return false;" href="javascript:void(0);" class="ref ref46 ref59">(46,59)</a> Interestingly, no increase in <i>HDAC6</i> expression levels was observed in available data sets of patients with other mature or immature T-cell malignancies, including anaplastic large cell lymphoma (ALCL), angioimmunoblastic T-cell lymphoma (AITL), PTCL not otherwise specified (PTCL-NOS), and T-ALL (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b), suggesting that <i>HDAC6</i> overexpression may be specific to T-PLL cancer. These data support the hypothesis that HDAC6 could be a beneficial target in T-PLL, and adverse effects of pan-HDAC inhibition could be minimized through the specific abrogation of HDAC6.</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. HDAC6 is significantly overexpressed in T-PLL patient samples, but not in other T-cell cancers. (a) mRNA expression levels of various <i>HDAC</i> genes (1–6, 9, and 11) in T-cell prolymphocytic leukemia (T-PLL) patient samples, compared to healthy T-cell (CD3<sup>+</sup>) control samples. Data were extracted from the Oncomine Platform, from the Durig Leukemia data set. <i>HDAC7</i>, <i>8</i>, and <i>10</i> expression levels were not reported in this data set. ns; not significant (<i>p</i> > 0.05). (b) mRNA expression levels of <i>HDAC6</i> in various T-cell neoplasia patient samples, compared to healthy control cells (from left to right, Eckerle Lymphoma: NK cell, T-cell, NK/T-cell; Piccaluga Lymphoma: CD4<sup>+</sup> T-cell, CD8<sup>+</sup> T-cell, T-cell; Zhang Leukemia: hematogone, bone marrow; Haferlach Leukemia: peripheral blood mononuclear cells). Data were extracted from the Oncomine Platform, from the Eckerle Lymphoma, Piccaluga Lymphoma, Zhang Leukemia, and Haferlach Leukemia data sets. All comparisons have a <i>p</i>-value > 0.05 and/or a fold-change of ≤1.2 and are not considered significant. Representation: boxes as interquartile range, horizontal line as the mean, and whiskers as lower and upper limits. Abbreviations: ALCL, anaplastic large cell lymphoma; ALK, anaplastic lymphoma kinase; PTCL-NOS, peripheral T-cell lymphoma, not otherwise specified; AITL, angioimmunoblastic T-cell lymphoma; NK, natural killer.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> KT-531 Shows Anticancer Activity and Therapeutic Margin in T-PLL Patient Samples</h3><div class="NLM_p">A high-throughput <i>ex vivo</i> drug sensitivity and resistance testing (DSRT) platform covering 306 approved and investigational oncology drugs has identified a potential therapeutic role for HDACi in T-PLL.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Drug sensitivity scores (DSS) that capture and integrate multiparametric dose–response relationships into a single metric have previously identified T-PLL selective drug responses of FDA-approved HDACi panobinostat and belinostat.<a onclick="showRef(event, 'ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref61 ref62">(61,62)</a> Therefore, we performed similar studies with KT-531 on primary T-PLL patient samples.</div><div class="NLM_p">In comparison to the pan-HDACi, vorinostat (SAHA), KT-531 resulted in a more potent drug response (<i>p</i> = 0.0007; Wilcoxon test), which was also comparable to other pan-HDACi, belinostat and tinostamustine (EDOS101) (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>a). Anticancer effects of HDACi have been synergistically potentiated by other chemotherapeutic agents in hematopoietic cancers.<a onclick="showRef(event, 'ref63 ref64 ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref63 ref64 ref65 ref66">(63−66)</a> Combinations of KT-531 with promising investigational drugs for T-PLL, notably idasanutlin, bendamustine, and venetoclax, were performed to identify potential synergisms. All combinations revealed synergistic relationships in patient samples, highlighting that HDAC inhibition may offer potential advantages in T-PLL (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>b).<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> Comparison of the potency in T-PLL patient samples to potency in PBMCs from healthy donors revealed a therapeutic margin for this novel combination of HDACi and venetoclax (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>c). Collectively, the data supports the hypothesis that HDAC6 inhibition, alone or in drug combination, could be a promising therapy for T-PLL.</div><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. KT-531 (<b>14</b>) shows promising efficacy and safety in T-PLL primary patient samples. (a) Drug sensitivity scores (DSS)<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> of selected HDACi in T-PLL primary patient samples. (b) Box plot of T-PLL synergy scores of KT-531 (<b>14</b>) with bendamustine, idasanutlin, and venetoclax in T-PLL patient samples. (c) Box plot of combination potency cropped to TC50 (toxic concentration in 50% of PBMC population) of KT-531 (<b>14</b>) and venetoclax in T-PLL patient samples and healthy PBMC controls.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Pharmacokinetic Studies</h3><div class="NLM_p">A preliminary <i>in vitro</i> ADME assessment of selected inhibitors was conducted. The pentafluorobenzene-sulfonamide analogue (<b>5</b>) exhibited a poor stability profile (<i>t</i><sub>1/2</sub> = ∼20 min in incubation with GSH, <i>t</i><sub>1/2</sub> < 2.26 min in mouse hepatocytes, <i>t</i><sub>1/2</sub> = 3.84 min in human hepatocytes, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Table S1b</a>). However, the removal of more than one fluorine from <b>5</b> was shown to be detrimental for antileukemic activity (<b>6</b>–<b>9</b>), leaving KT-531 as the most prominent lead for initial PK evaluation. Notably, no discernible S<sub>N</sub>Ar reaction with GSH was observed with KT-531 within 24 h of incubation (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Table S1a</a>).</div><div class="NLM_p last">The parallel artificial membrane permeability assay (PAMPA) and human plasma were used to evaluate the permeability and stability of compound KT-531, respectively. A −log <i>P</i><sub>e</sub> value of 5.44 was reported for KT-531 in the PAMPA assay, and <i>t</i><sub>1/2</sub> of 37 min in mouse hepatocytes and 41 min in human plasma showed promise for <i>in vivo</i> bioavailability (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>c). <i>In vivo</i> pharmacokinetic studies on KT-531 were performed via intraperitoneal (IP) administration of compound in male CD-1 mice, at a dose of 20 mg/kg. Compound KT-531 reached a maximum concentration (<i>C</i><sub>max</sub>) of 493 ng/mL, with <i>t</i><sub>1/2</sub> of 63 min and an overall exposure of 1576 h·ng/mL (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>c). In comparison, and under the same conditions, citarinostat was reported to have a shorter <i>t</i><sub>1/2</sub> of 15 min but higher <i>C</i><sub>max</sub> and overall exposure (<i>C</i><sub>max</sub> = 5640 ng/mL, AUC = 4163 h·ng/mL; <a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Figure S9</a>), supporting the hypothesis that citarinostat is a more readily absorbed compound.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Discussion and Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48772" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48772" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">T-PLL is a very aggressive T-cell malignancy that does not benefit from “classical treatments” and urgently demands rationally designed targeted therapies. Available treatment options do not prevent inevitable patient relapses, and a novel treatment approach specifically tailored to treat T-PLL may resolve the hurdles of short survival (<20 months) and absence of curative therapy. The promising efficacy but limited tolerability of pan-HDAC inhibitors in this group of patients highlights the need to selectively and safely target the T-PLL-relevant HDAC isoforms. Critically, only the <i>HDAC6</i> isozyme of the HDAC family of enzymes was found to be overexpressed in primary T-PLL patient samples.</div><div class="NLM_p">A comprehensive SAR study led to the identification of a preclinical candidate, KT-531 which demonstrated exceptional anticancer effects in T-PLL. This lead candidate exhibited (i) nanomolar inhibition of HDAC6 in a functional EMSA assay (IC<sub>50</sub> = 8.5 nM), (ii) 39-fold <i>in vitro</i> selectivity for HDAC6 over other HDAC isozymes, and (iii) inhibition of cellular proliferation in a variety of hematologic model cell lines (AML, PTCL, T-ALL) while possessing a therapeutic window in heathy cells (MRC-9, NHF, pPF, HUVEC) and safety in 5 day toxicity trials in CD-1 mice. Since HDAC8 was the next nearest target of KT-531, HDAC8 selective inhibitors published in the literature were tested and used as control compounds to ensure that biological activity is not reliant on HDAC8 inhibition. Both PCI-34051 (120-fold selective for HDAC8) and MMH-410 (15-fold selective for HDAC8) exhibited nanomolar HDAC8 potency but no observable activity in MV4-11 and MRC9 cells (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Table S5</a>).<a onclick="showRef(event, 'ref68 ref69'); return false;" href="javascript:void(0);" class="ref ref68 ref69">(68,69)</a> Given the established pathogenetic role of HDAC modulation in AML and PTCL, cell lines representing these disease types were used to profile the library and investigate the cellular pharmacology of KT-531.</div><div class="NLM_p">The promising cellular potency of KT-531 in the T-PLL quasi-model SUP-T11 prompted further investigation in this novel indication. The absence of activity in some branches of hematological neoplasms, such as CTCL, indicated that the higher selectivity of KT-531 in contrast to FDA-approved pan-HDAC drugs may be driving its unique and selective cytotoxicity profile. KT-531 exhibited comparable potency to clinically used HDACi, belinostat and vorinostat, in T-PLL patient cells, with a significant therapeutic window when compared to cytotoxicity in treated PBMCs from healthy donors. Given the well-validated synergism displayed by epigenetic agents with other compounds, KT-531 was tested in combination with promising investigational drugs for T-PLL, namely, idasanutlin, bendamustine, and venetoclax. The marked synergy of all pairings supports the clinical combination of more selective HDAC6i with these chemotherapeutic agents.</div><div class="NLM_p last">When coupled with the reported increased frequency of upregulated <i>HDAC6</i>, as compared to other HDAC isoforms in primary T-PLL patient cells, HDAC6i is a potential therapeutic avenue in this incurable condition and warrants more in-depth exploration. Further studies are necessary to explain the superior effects of KT-531 over citarinostat. Although the <i>in cellulo</i> and <i>in vitro</i> studies support KT-531 as a more HDAC6-selective inhibitor than citarinostat, the ∼39-fold HDAC6 enzymatic selectivity of KT-531 is not fully translated with same selectivity margins in cells. This is evident from the Western blots, where acetylation levels of histone H3 were concomitant with acetylation levels of α-tubulin, suggesting that the mechanism of action may not be solely dependent on HDAC6 inhibition. Such weak cellular selectivity profiles are common, as observed in multiple clinical trials of the modestly HDAC6-selective drug candidate ricolinostat where acetylation of histones was similar to acetylation of α-tubulin.<a onclick="showRef(event, 'ref20 ref26'); return false;" href="javascript:void(0);" class="ref ref20 ref26">(20,26)</a> Based on the data generated, an HDACi combination approach will be investigated in preclinical models of T-PLL, specifically through development of HDAC6 inhibitors with greater cellular selectivity and enhanced pharmacokinetic properties. Subsequent studies will be focused on deciphering the functional significance and mechanism of action of selective HDAC6 inhibition in T-PLL.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42744" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42744" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> General Procedures</h3><div class="NLM_p">All solvents and chemicals were used as purchased without further purification. Reactions were carried out in oven-dried glassware and monitored by thin-layer chromatography (TLC) using Merck silica gel 60 F<sub>254</sub> on aluminum sheets (visualized by 254/365 nm UV light and/or staining with KMnO<sub>4</sub>). Column chromatography was carried out using Biotage Isolera One and Isolera Prime purification systems, with industry-standard SNAP 10, 25, 50, and 100 g cartridges loaded with 40–60 μm silica gel from VWR International (average pore size 60 Å), eluting at 12–40 mL/min and detecting compounds by UV measuring at 254 and 298 nm. Semipreparative HPLC was conducted using a Waters 2487 Dual λ Absorbance Detector, equipped with a Symmetry C18 4.6 mm × 150 mm cartridge. Compounds were detected by UV at 254 nm with sensitivity set to between 0.0001 and 4.0000 AUFS (default settings), eluting at 20 mL/min and using gradient mixtures of (A) Milli-Q water with 0.1% (v/v) TFA and (B) HPLC-grade acetonitrile.</div><div class="NLM_p">Inhibitor purity was evaluated by using a Hewlett-Packard Series 1100 analytical HPLC system fitted with a Phenomenex Luna 5.0 μm C18 4.6 mm × 150 mm cartridge, with eluent flow set at 1.200 mL/min and using gradient mixtures of (A) Milli-Q water with 0.1% (v/v) TFA and (B) HPLC-grade acetonitrile. Retention times for the target compound are given, followed by purities in their respective order. Biologically evaluated compounds are ≥95% chemically pure.</div><div class="NLM_p last">A 400 MHz Bruker NMR instrument was utilized to obtain <sup>1</sup>H, <sup>13</sup>C, and <sup>19</sup>F NMR spectra in CDCl<sub>3</sub> (99.8 atom % D), CD<sub>3</sub>CN (99.8 atom % D), or MeOH-<i>d</i><sub>4</sub> (99.8 atom % D), as indicated (<sup>1</sup>H at 400 MHz, <sup>13</sup>C at 100 MHz, and <sup>19</sup>F at 54 MHz, unless otherwise stated). Samples were analyzed with 16 scans for <sup>1</sup>H, 256 scans for <sup>13</sup>C, and 8 scans for <sup>19</sup>F, in 8” 3 mm A-grade glass tubes. Chemical shifts (δ) are reported in parts per million (ppm), after calibration to residual isotopic solvent, and coupling constants (<i>J</i>) are reported in hertz (Hz). Multiplicities are described as singlet (s), doublet (d), triplet (t), quartet (q), pentet (p), sextet (sex), septet (sep), multiplet (m), broad (br), or a combination of these. Low-resolution mass spectrometry (LRMS) was carried out using a Waters LC-MS system in ESI mode, fitted with a Micromass ZQ MS and an Alliance 2690 LC. Analysis was performed on a Waters Xterra C18 3.0 × 150 mm column containing 3.5 μm beads by direct injection into the mass spectrometer. High-resolution mass spectrometry (HRMS) was carried out using an Agilent 6538 UHD Q-TOF MS system in ESI mode with a mass accuracy ±1 mDa.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Synthetic Procedures</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Reductive Amination (a)</h4><div class="NLM_p last">The appropriate alkyl amine (1.1 equiv) was added in one go to a solution of the appropriate aldehyde (1.0 equiv) in 1,2-DCE (0.1 M) and stirred at RT for 10 min before addition of glacial acetic acid (2.0 equiv) and further stirring for 30 min. Sodium triacetoxyborohydride (2.0 equiv) was added in one go, and the reaction was stirred for 24 h before pouring over saturated aqueous sodium bicarbonate. The layers were separated, and the aqueous layer was extracted with CH<sub>2</sub>Cl<sub>2</sub>. The combined organic layer was washed with saturated aqueous sodium bicarbonate, followed by brine, dried (MgSO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. Column chromatography isolated the target compound.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Amine Sulfonylation (b)</h4><div class="NLM_p last">The sulfonyl chloride (1.0–1.1 equiv) was added in one go to an ice-cooled solution of the appropriate amine (1.1 equiv) and triethylamine or DIPEA (2.0 equiv) in THF, CH<sub>2</sub>Cl<sub>2</sub>, or CH<sub>3</sub>Cl (0.1–0.2 M) and stirred at room temperature, and the disappearance of the limiting agent was monitored by TLC (hexanes/EtOAc 8:2). After 3–16 h, the reaction was quenched with 1 M HCl. The layers were separated, and the aqueous layer was extracted with CH<sub>2</sub>Cl<sub>2</sub>. The organic layer was dried over MgSO<sub>4</sub>, filtered, concentrated <i>in vacuo</i>, and purified by column chromatography to isolate the target compound.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Benzyl Deprotection by Hydrogenation (c)</h4><div class="NLM_p last">To a nitrogen-purged solution of the benzyl or hydroxamate ester (1.0 equiv) in 2:1 mixture of THF and methanol or 100% methanol (0.05–0.1 M) was charged 10% Pd/C (0.1 equiv). The flask was purged with hydrogen, and then the reaction was stirred for 2 h under hydrogen atmosphere (balloon) before filtration through Celite, washed with THF, and concentrated <i>in vacuo</i>. Carboxylic acids were isolated without further purification. Preparative HPLC was used to isolate hydroxamic acids.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> SN<sub>2</sub> Substitution with Benzyl Bromides/Iodides (d)</h4><div class="NLM_p last">Benzyl bromide/iodide (1.0 equiv) was added to a solution of the amine (1 equiv) and Cs<sub>2</sub>CO<sub>3</sub> (1.5 equiv) in DMF (0.1 M). After 3–16 h (reaction monitored by TLC, hexanes/EtOAc 6:4). The reaction mixture was then diluted in EtOAc and saturated aqueous sodium bicarbonate. The organic layer was washed with saturated aqueous sodium bicarbonate (1×), water (3×), and brine (1×), and the aqueous layer was extracted once with EtOAc. The combined organic layer was dried over MgSO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to isolate the target compound without further purification.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Acid-Mediated Hydrolysis of Carboxylate or Hydroxamate Esters (e)</h4><div class="NLM_p last">The carboxylate or hydroxamate ester was charged in a round-bottom flask with 4 M HCl in dioxane (0.3 M final concentration) at RT in air. After 3–16 h, the solvent was removed <i>in vacuo</i>. Carboxylic acid intermediates were used in the next step without further purification, whereas final hydroxamic acids were purified using preparative HPLC.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Formation of Hydroxamate Esters (f)</h4><div class="NLM_p last">Oxalyl chloride (4 equiv) was added dropwise to a solution of the appropriate carboxylic acid (1.0 equiv) in THF (0.05–0.2 M) and DMF (1 to 2 drops) at 0 °C and stirred for 50 min −3 h. The reaction was concentrated to dryness <i>in vacuo</i> before redissolving in dry THF (0.2 M) and mixing with diisopropylethylamine or triethylamine (2.0 equiv) followed by the <i>O</i>-protected hydroxylamine or its hydrochloride salt (1.0–2.0 equiv). After 16 h, the reaction was quenched with 1 M HCl and the layers were separated. The organic layer was washed with 1 M HCl and the combined aqueous layer was extracted with EtOAc or CH<sub>2</sub>Cl<sub>2</sub>. The organic layer was dried (MgSO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>, and purified by column chromatography to isolate the target compound.</div></div><div id="sec400" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Alternate Procedure for Formation of Hydroxamate Esters (f′)</h4><div class="NLM_p last">Triethylamine (3.0 equiv) was charged in one go to a solution of the appropriate carboxylic acid (1.0 equiv), 1-(3-(dimethylamino)propyl)-3-ethylcarbodiimide hydrochloride (EDC.HCl, 1.5 equiv), and 1-hydroxybenzotriazole monohydrate (HOBt, 1.1 equiv) in DMF (0.1–0.2 M) at RT in air. After 10 min, the O-protected hydroxylamine (2.0 equiv) was added in one go, and the reaction was stirred for 16–24 h before quenching with 0.1 M HCl. The layers were separated, the organic layer was washed with 0.1 M HCl, and the aqueous layer was extracted with EtOAc or CH<sub>2</sub>Cl<sub>2</sub>. The organic layer was dried (MgSO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>, and purified by column chromatography to isolate the target compound.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Preparation of Sulfonyl Chlorides S3 from Polyhalogenated Benzene Rings (g)</h4><div class="NLM_p last">The polyhalogenated benzene ring was added to neat chlorosulfonic acid (5 equiv) in a round-bottom flask under argon atmosphere, and the resulting solution stirred at RT for 5 min. The reaction was then heated to reflux (150 °C) for 2 h. The reaction turned dark over this period of time. The reaction was cooled to RT, and the crude material was added slowly to a mixture of ice and water. The aqueous phase was extracted three times with EtOAc, washed with brine, and concentrated to afford a dark brown oil that was flushed through a bed of silica and used without purification in subsequent step.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>tert-</i>Butyloxycarbonyl (Boc) Protection (h)</h4><div class="NLM_p last">Di-<i>tert</i>-butyl dicarbonate (2.0 equiv) in THF (0.63 M) was added in one go to a solution of the appropriate amine hydrochloride (1.0 equiv) in THF and distilled water (1:1). Sodium bicarbonate (3.0 equiv) in distilled water (0.95 M) was added to the reaction mixture in one go and stirred at RT in air. After 19 h, the reaction was quenched by addition of 1 M HCl until pH 3 and the layers were separated. The aqueous layer was extracted with EtOAc and the combined organic layer was dried (MgSO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i> to isolate the target compound without further purification.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Carboxyl Benzylation (i)</h4><div class="NLM_p last">The appropriate benzoic acid (1.0 equiv) and cesium carbonate (1.1–1.2 equiv) were suspended in DMF (0.33–0.5 M) and stirred at RT for 20 min in air, before addition of benzyl bromide (1.0 equiv) in one go. After 24 h, the reaction was concentrated <i>in vacuo</i> and partitioned between EtOAc and distilled water. The layers were separated, and the aqueous layer was extracted with EtOAc. The combined organic layer was dried (MgSO<sub>4</sub>), filtered, and concentrated <i>in vacuo</i>. Column chromatography isolated the target compound.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Boc Deprotection (j)</h4><div class="NLM_p last">The appropriate carbamate (1.0 equiv) was charged with ice-cooled 4 M HCl in dioxane (0.1 M) and stirred at RT in air. After 3 h, the solvent was removed <i>in vacuo</i>, azeotroped with CH<sub>2</sub>Cl<sub>2</sub>, to isolate the target compound without further purification.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 2,3,4,5,6-Pentafluoro-<i>N</i>-(4-(hydroxycarbamoyl)benzyl)benzamide (<b>1</b>)</h3><div class="NLM_p last">The product was obtained using synthetic procedure (c) as a white-solid (39%). <sup>1</sup>H NMR δ/ppm (400 MHz, DMSO-<i>d</i><sub><i>6</i></sub>) 4.54 (d, <i>J</i> = 5.9 Hz, 2H), 7.38 (d, <i>J</i> = 8.3 Hz, 2H), 7.74 (d, <i>J</i> = 8.3 Hz, 2H), 9.02 (s, 1H), 9.51 (t, <i>J</i> = 5.9 Hz, 1H), 11.19 (s, 1H). <sup>13</sup>C NMR δ/ppm (100 MHz, DMSO-<i>d</i><sub><i>6</i></sub>) 42.5, 127.1, 127.1, 131.7, 135.8, 138.1, 141.4, 141.9, 144.2, 156.8, 164.0. <sup>19</sup>F NMR δ/ppm (54 MHz, DMSO-<i>d</i><sub>6</sub>) −161.1 to −161.3 (m, 2F), −152.9 (t, <i>J</i> = 22.1 Hz, 1F), −142.0 to −142.2 (m, 2F). LRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> calcd for [C<sub>15</sub>H<sub>9</sub>F<sub>5</sub>N<sub>2</sub>O<sub>3</sub>Na]<sup>+</sup>: 383.04, found: 383.16. HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> calcd for [C<sub>15</sub>H<sub>10</sub>F<sub>5</sub>N<sub>2</sub>O<sub>3</sub>]<sup>+</sup>: 361.0606, found: 361.0607. HPLC (I) <i>t</i><sub>R</sub> = 10.48 min (98.7%).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> <i>N</i>-Hydroxy-4-(((perfluorophenyl)sulfonamido)methyl)benzamide (<b>2</b>)</h3><div class="NLM_p last">The product was obtained using synthetic procedure (e) after purification using preparative HPLC and lyophilization as a white powder (66.7%). <sup>1</sup>H NMR (500 MHz, acetonitrile-<i>d</i><sub>3</sub>) δ 10.73 (s, 1H), 7.78 (d, <i>J</i> = 8.4 Hz, 2H), 7.44 (d, <i>J</i> = 8.3 Hz, 2H), 4.47 (s, 2H), <i>hydroxamic acid OH, NH, and sulfonamide NH protons were not observed</i>. <sup>13</sup>C NMR (126 MHz, cd<sub>3</sub>cn) δ 162.6, 140.0, 131.7, 130.1, 129.0, 126.5, 126.3, 125.0, 46.2. <sup>19</sup>F NMR (376 MHz, acetone-<i>d</i><sub>6</sub>) δ −136.68 to −139.04 (m, 2F), −151.43 to −153.09 (m, 1F), −161.00 to −162.87 (m, 2F). HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> calcd for [C<sub>14</sub>H<sub>10</sub>F<sub>5</sub>N<sub>2</sub>O<sub>4</sub>S]<sup>+</sup>: 397.03, found: 397.0269. HPLC (I) <i>t</i><sub>R</sub> = 16.99 min (97%).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> <i>N</i>-Hydroxy-4-(((2,3,4,5,6-pentafluoro-<i>N</i>-methylphenyl)sulfonamido)methyl)benzamide (<b>3</b>)</h3><div class="NLM_p last">The product was obtained using synthetic procedure (c) after purification using preparative HPLC and lyophilization as a white powder (35.2%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN) δ 7.83 (d, <i>J</i> = 8.4 Hz, 2H rotamer #1), 7.78 (d, <i>J</i> = 8.3 Hz, 2H rotamer#2), 7.47 (d, <i>J</i> = 8.3 Hz, 2H rotamer#2), 7.37 (d, <i>J</i> = 8.2 Hz, 2H rotamer#1), 4.48 (s, 2H, rotamer#2), 4.44 (s, 2H rotamer#1), 2.85 (s, 3H), <i>hydroxamic acid NH and OH protons were not observed</i>. <sup>13</sup>C NMR (126 MHz, acetonitrile-<i>d</i><sub>3</sub>) δ 160.6, 139.5, 136.6, 133.6, 131.5, 129.0, 128.2, 53.2, 34.1, 23.9. <sup>19</sup>F NMR (376 MHz, chloroform-<i>d</i>) δ −134.19 to −136.14 (m, 2F), −146.11 (tt, <i>J</i> = 21.1, 6.7 Hz, 1F), −157.50 to −159.50 (m, 2F). HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> calcd for [C<sub>15</sub>H<sub>12</sub>F<sub>5</sub>N<sub>2</sub>O<sub>4</sub>S]<sup>+</sup>: 411.04; found 411.0425. HPLC (I) <i>t</i><sub>R</sub> = 14.54 min (97%).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> <i>N</i>-Hydroxy-4-(((2,3,4,5,6-pentafluoro-<i>N</i>-isopropylphenyl)sulfonamido)methyl)benzamide (<b>4</b>)</h3><div class="NLM_p last">The product was obtained using synthetic procedure (e) after purification using preparative HPLC and lyophilization as a white powder (50%). <sup>1</sup>H NMR (400 MHz, acetonitrile-<i>d</i><sub>3</sub>) δ 9.76 (s, 1H), 7.72 (d, 2H), 7.51 (d, 2H), 4.60 (s, 2H), 4.31 (p, <i>J</i> = 6.8 Hz, 1H), 1.11 (d, <i>J</i> = 6.8 Hz, 6H), <i>hydroxamic acid OH proton was not observed</i>. <sup>13</sup>C NMR (126 MHz, acetonitrile-<i>d</i><sub>3</sub>) δ 165.1, 145.7, 143.7, 142.9, 130.9, 127.9, 127.0, 125.6, 117.4, 51.4, 46.3, 20.4. <sup>19</sup>F NMR (376 MHz, acetonitrile-<i>d</i><sub>3</sub>) δ −137.37 to −137.57 (m, 2F), −149.44 (tt, <i>J</i> = 20.2, 6.6 Hz, 1F), −161.26 to −161.46 (m, 2F). HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> calcd for [C<sub>17</sub>H<sub>16</sub>F<sub>5</sub>N<sub>2</sub>O<sub>4</sub>S]<sup>+</sup>: 439.07; found 439.0740. HPLC (I) <i>t</i><sub>R</sub> = 14.24 min (100%).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 4-(((<i>N</i>-Cyclopropyl-2,3,4,5,6-pentafluorophenyl)sulfonamido)methyl)-<i>N</i>-hydroxybenzamide (<b>5</b>)</h3><div class="NLM_p last">The product was obtained using synthetic procedure (c) after purification using preparative HPLC and lyophilization as a white powder (53%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ: 10.83 (s, 1H), 7.87 (d, <i>J</i> = 8.3 Hz, 2H), [7.52 (d, <i>J</i> = 8.3 Hz, 2H rotamer #1), 7.46 (d, <i>J</i> = 8.0 Hz, 2H rotamer #2)], [4.68 (s, 2H rotamer #1), 4.65 (s, 2H rotamer #2)], 2.59–2.49 (m, 1H), 0.86–0.51 (m, 4H), <i>hydroxamic acid OH proton was not observed</i>. <sup>13</sup>C NMR (101 MHz, acetone) δ 160.1, 144.4, 141.5, 133.9, 129.8, 128.0, 124.6, 54.2, 31.1, 13.1, 8.2. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ: −134.84 to −135.26 (m, 2F), −145.36 (t, <i>J</i> = 20.5 Hz, 1F), −157.84 to −158.96 (m, 2F). HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> calcd for [C<sub>17</sub>H<sub>14</sub>F<sub>5</sub>N<sub>2</sub>O<sub>4</sub>S]<sup>+</sup>: 437.0589, found: 437.0588. HPLC (I) <i>t</i><sub>R</sub> = 20.23 min (97.5%).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 4-(((<i>N</i>-Cyclopropyl-2,3,5,6-tetrafluorophenyl)sulfonamido)methyl)-<i>N</i>-hydroxybenzamide (<b>6</b>)</h3><div class="NLM_p last">The product was obtained using synthetic procedure (e) after purification using preparative HPLC and lyophilization as a white powder (49%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.81 (s, 1H), 8.02–7.79 (m, 3H), 7.53 (d, <i>J</i> = 7.9 Hz, 2H rotamer#1), 7.46 (d, <i>J</i> = 7.7 Hz, 2H rotamer#2), 4.70 (s, 1H) (rotamer #1), 4.67 (s, 1H) (rotamer #2), 2.56 (p, <i>J</i> = 6.4, 3.5, 2.8 Hz, 1H), 0.80–0.61 (m, 4H), <i>hydroxamic acid OH proton was not observed</i>. <sup>13</sup>C NMR (126 MHz, acetonitrile-<i>d</i><sub>3</sub>) δ 161.8, 147.6, 140.9, 138.4, 131.2, 128.3, 127.2, 125.7, 120.5, 111.1, 53.7, 30.4, 6.4. <sup>19</sup>F NMR (376 MHz, acetone-<i>d</i><sub>6</sub>) δ −137.15 to −137.73 (m, 2F), −137.79 to −138.20 (m, 2F). LRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for [C<sub>17</sub>H<sub>13</sub>F<sub>4</sub>N<sub>2</sub>O<sub>4</sub>S]<sup>−</sup>: 417.05, found: 417.48. HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> calcd for [C<sub>17</sub>H<sub>14</sub>F<sub>4</sub>N<sub>2</sub>O<sub>4</sub>S]<sup>+</sup>: 419.0683, found: 419.0677. HPLC (I) <i>t</i><sub>R</sub> = 20.29 min (100%).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 4-(((<i>N</i>-Cyclopropyl-4-fluorophenyl)sulfonamido)methyl)-<i>N</i>-hydroxybenzamide (<b>7</b>)</h3><div class="NLM_p last">The product was obtained using synthetic procedure (c) after purification using preparative HPLC and lyophilization as a white powder (59%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ: 10.28 (s, 1H), 8.03 (d, <i>J</i> = 8.9, 5.2 Hz, 2H), 7.85 (d, <i>J</i> = 8.3 Hz, 2H), 7.55 (d, <i>J</i> = 8.2 Hz, 2H), 7.45 (d, <i>J</i> = 8.8 Hz, 2H), 4.52 (s, 2H), 2.26–2.19 (m, 1H), 0.79–0.68 (m, 4H), <i>hydroxamic acid OH proton was not observed</i>. <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ: 130.7, 130.6, 128.5, 127.0, 116.4, 116.2, 54.1, 31.1, 6.8. <sup>19</sup>F NMR (376 MHz, acetone-<i>d</i><sub>6</sub>) δ: −107.29 to −107.46 (m, 1F). HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> calcd for [C<sub>17</sub>H<sub>18</sub>FN<sub>2</sub>O<sub>4</sub>S]<sup>+</sup>: 365.0966, found: 365.0969. HPLC (I) <i>t</i><sub>R</sub> = 23.04 min (99%).</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 4-(((<i>N</i>-Cyclopropyl-3,4,5-trifluorophenyl)sulfonamido)methyl)-<i>N</i>-hydroxybenzamide (<b>8</b>)</h3><div class="NLM_p last">The product was obtained using synthetic procedure (c) after purification using preparative HPLC and lyophilization as a white powder (60%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ: 10.83 (s, 1H), 7.84 (d, <i>J</i> = 8.3 Hz, 2H), 7.79–7.69 (m, 2H), [7.48 (d, <i>J</i> = 8.3 Hz, 2H rotamer #1), 7.42 (d, <i>J</i> = 8.0 Hz, 2H rotamer #2)], [4.53 (s, 2H rotamer #1), 4.50 (s, 2H rotamer #2)], 2.35–2.25 (m, 1H), 0.77–0.64 (m, 4H), <i>hydroxamic acid OH proton was not observed</i>. <sup>13</sup>C NMR (101 MHz, acetone) δ 202.5, 170.7, 152.6, 142.0, 136.9, 129.3, 127.9, 114.0, 113.7, 55.0, 31.9, 7.6. <sup>19</sup>F NMR (376 MHz, acetone-<i>d</i><sub>6</sub>) δ: −132.40 to −132.79 (m, 1F), −155.19 to −155.55 (m, 2F). HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> calcd for [C<sub>17</sub>H<sub>16</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub>S]<sup>+</sup>: 401.0777, found: 401.0772. HPLC (I) <i>t</i><sub>R</sub> = 26.05 min (98.8%).</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 4-(((<i>N</i>-Cyclopropyl-2,4,6-trifluorophenyl)sulfonamido)methyl)-<i>N</i>-hydroxybenzamide (<b>9</b>)</h3><div class="NLM_p last">The product was obtained using synthetic procedure (c) after purification using preparative HPLC and lyophilization as a white powder (63%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ: 10.86 (s, 1H), 7.89–7.82 (m, 2H), 7.52 (d, <i>J</i> = 8.0 Hz, 1H), 7.46 (d, <i>J</i> = 7.9 Hz, 1H), 7.28–7.18 (m, 2H), [4.66 (s, 2H rotamer #1), 4.63 (s, 2H rotamer #2)], 2.62–2.37 (m, 1H), 0.70–0.62 (m, 4H), <i>hydroxamic acid OH proton was not observed</i>. <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ: 160.5, 141.2, 138.8, 131.6, 128.3, 128.2, 127.1, 125.7, 102.8, 53.6, 30.1, 6.2. <sup>19</sup>F NMR (376 MHz, acetone-<i>d</i><sub>6</sub>) δ: −100.83 to −101.14 (m, 1F), −103.06 (dt, <i>J</i> = 39.8, 10.5 Hz, 2F). HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> calcd for [C<sub>17</sub>H<sub>16</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub>S]<sup>+</sup>: 401.0777, found: 401.0772. HPLC (I) <i>t</i><sub>R</sub> = 15.16 min (99%).</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 3-(((<i>N</i>-Cyclopropyl-2,3,4,5,6-pentafluorophenyl)sulfonamido)methyl)-<i>N</i>-hydroxybenzamide (<b>10</b>)</h3><div class="NLM_p last">The product was obtained using synthetic procedure (e) after purification using preparative HPLC and lyophilization as a white powder (66.7%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.78 (s, 1H), 7.70 (d, <i>J</i> = 7.7, 1.5 Hz, 1H), 7.61 (d, <i>J</i> = 7.7, 1.6 Hz, 1H), 7.48 (dd, <i>J</i> = 7.7 Hz, 1H), 4.64 (s, 2H), 2.48 (p, <i>J</i> = 6.4, 3.2 Hz, 1H), 0.80–0.61 (m, 4H). <i>hydroxamic acid OH and NH protons were not observed</i>. <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>) δ 139.4, 135.6, 130.8, 87.8, 31.6, 30.9, 30.4, 22.7, 14.1, 7.0. <sup>19</sup>F NMR (376 MHz, chloroform-<i>d</i>) δ −134.85 (2F), −145.19 (1F), −158.25 (d, <i>J</i> = 18.1 Hz, 2F). HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> calcd for [C<sub>17</sub>H<sub>13</sub>F<sub>5</sub>N<sub>2</sub>O<sub>4</sub>S]<sup>+</sup>: 436.05, found: 437.05, HPLC (I) <i>t</i><sub>R</sub> = 19.59 min (98.1%).</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 4-(((3,5-Dichloro-<i>N</i>-cyclopropyl-2,4,6-trifluorophenyl)sulfonamido)methyl)-<i>N</i>-hydroxybenzamide (<b>11</b>)</h3><div class="NLM_p last">The product was obtained using synthetic procedure (e) after purification using preparative HPLC and lyophilization as a white powder (81%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN) δ 9.91 (s, 1H), 7.75 (d, <i>J</i> = 7.8 Hz, 2H), 7.48 (d, <i>J</i> = 7.7 Hz, 2H), 4.63 (s, 2H), 2.49 (p, <i>J</i> = 6.6, 3.5 Hz, 1H), 0.74–0.65 (m, 4H), <i>hydroxamic acid OH proton was not observed</i>. <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>CN) δ 177.2, 157.5, 157.1, 153.2, 140.3, 129.2, 128.1, 54.7, 31.4, 7.4. <sup>19</sup>F NMR (376 MHz, CD<sub>3</sub>CN) δ −105.08 (t, <i>J</i> = 6.9 Hz, 1F), −107.87 (d, <i>J</i> = 6.9 Hz, 2F). HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> calcd for [C<sub>17</sub>H<sub>14</sub>F<sub>3</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>4</sub>S]<sup>+</sup>: 468.9998, found: 468.9997. HPLC (I) <i>t</i><sub>R</sub> = 20.78 min (100%).</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 4-(((3-Chloro-<i>N</i>-cyclopropyl-2,4,6-trifluorophenyl)sulfonamido)methyl)-<i>N</i>-hydroxybenzamide (<b>12</b>)</h3><div class="NLM_p last">The product was obtained using synthetic procedure (e) after purification using preparative HPLC and lyophilization as a white powder (77%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN) δ: 7.75 (d, <i>J</i> = 7.8 Hz, 2H), 7.48 (d, <i>J</i> = 7.6 Hz, 2H), 7.29–7.16 (m, 1H), 4.61 (s, 2H), 2.59–2.40 (m, 1H), 0.77–0.56 (m, 4H), <i>hydroxamic acid OH proton was not observed</i>. <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>CN) δ 148.0, 145.6, 144.6, 141.5, 132.1, 129.8, 128.1, 104.5, 104.0, 103.6, 54.6, 30.8, 6.7. <sup>19</sup>F NMR (376 MHz, CD<sub>3</sub>CN) δ: −104.04 (q, <i>J</i> = 7.5 Hz, 1F), −105.46 (t, <i>J</i> = 10.4 Hz, 1F), −105.71 (d, <i>J</i> = 7.7 Hz, 1F). HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> calcd for [C<sub>17</sub>H<sub>15</sub>F<sub>3</sub>ClN<sub>2</sub>O<sub>4</sub>S]<sup>+</sup>: 435.0388, found: 435.0392. HPLC (I) <i>t</i><sub>R</sub> = 19.19 min (99.2%).</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 4-(((4-Chloro-<i>N</i>-cyclopropyl-2,3,5,6-tetrafluorophenyl)sulfonamido)methyl)-<i>N</i>-hydroxybenzamide (<b>13</b>)</h3><div class="NLM_p last">The product was obtained using synthetic procedure (e) after purification using preparative HPLC and lyophilization as a white powder (71%). <sup>1</sup>H NMR (400 MHz, acetone) δ 7.87 (d, <i>J</i> = 7.8 Hz, 2H), 7.52 (d, <i>J</i> = 7.6 Hz, 2H), 4.69 (s, 2H), 2.89–2.45 (m, 1H), 1.05–0.52 (m, 4H). <sup>19</sup>F NMR (376 MHz, acetone) δ −136.45 to −136.81 (m, 2F), −140.16 to −140.46 (m, 2F). <sup>13</sup>C NMR (101 MHz, acetone) δ 146.0, 145.8, 143.0, 140.7, 131.7, 128.3, 127.2, 118.6, 53.7, 30.3, 6.4. HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> calcd for [C<sub>17</sub>H<sub>14</sub>F<sub>4</sub>ClN<sub>2</sub>O<sub>4</sub>S]<sup>+</sup>: 453.0293, found: 453.0294. HPLC (I) <i>t</i><sub>R</sub> = 20.45 min (100%).</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 4-(((<i>N</i>-Cyclopropyl-2,3,4,5-tetrafluorophenyl)sulfonamido)methyl)-<i>N</i>-hydroxybenzamide (<b>14</b>)</h3><div class="NLM_p last">The product was obtained using synthetic procedure (e) after purification using preparative HPLC and lyophilization as a white powder (60%). <sup>1</sup>H NMR (400 MHz, DMSO) δ 11.28 (s, 1H), 8.04–7.84 (m, 1H), 7.81–7.69 (m, 2H), 7.58–7.20 (m, 2H), 4.59 (s, 2H), 2.48 (p, <i>J</i> = 7.1, 3.9 Hz, 1H), 0.76–0.61 (m, 4H), <i>hydroxamic acid OH proton was not observed</i>. <sup>13</sup>C NMR (126 MHz, acetone) δ 141.0, 132.5, 130.8, 129.2, 129.1, 128.0, 126.2, 113.7, 113.5, 53.5, 31.2, 31.0, 28.1, 7.4. <sup>19</sup>F NMR (376 MHz, acetone-<i>d</i><sub>6</sub>) δ: −134.84 (ddt, <i>J</i> = 21.3, 14.0, 7.4 Hz, 1F), −138.26 to −138.43 (m, 1F), −149.58 (tt, <i>J</i> = 20.0, 8.2 Hz, 1F), −153.95 (ddt, <i>J</i> = 21.9, 19.2, 3.4 Hz, 1F). HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> calcd for [C<sub>17</sub>H<sub>15</sub>F<sub>4</sub>N<sub>2</sub>O<sub>4</sub>S]<sup>+</sup>: 419.0681, found: 419.0683. HPLC (I) <i>t</i><sub>R</sub> = 20.43 min (100%).</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Intermediate Compound Characterization for All New Molecules</h3><div id="sec4_17_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> Benzyl 4-Formylbenzoate (<b>S1</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (i) as a light yellow oil (70%). <sup>1</sup>H (400 MHz, CDCl<sub>3</sub>) δ: 10.10 (s, 1H); 8.23 (d, <i>J</i> = 8.3 Hz, 2H), 7.95 (d, <i>J</i> = 8.5 Hz, 2H), 7.35–7.47 (m, 5H), 5.40 (s, 2H). <sup>13</sup>C (101 MHz, CDCl<sub>3</sub>) δ: 191.6, 165.4, 139.2, 135.6, 135.1, 130.3, 128.7, 128.5, 128.4, 67.3. LRMS (ESI) <i>mass not observed</i>.</div></div><div id="sec4_17_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> Benzyl 4-((Cyclopropylamino)methyl)benzoate (<b>S2</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (a) as a pale yellow oil (83%). <sup>1</sup>H (400 MHz, CDCl<sub>3</sub>) δ: 8.07 (d, <i>J</i> = 8.2 Hz, 2H), 7.48 (d, <i>J</i> = 7.3 Hz, 2H), 7.45–7.36 (m, 5H), 5.39 (s, 2H), 3.92 (s, 2H), 2.16 (dd, <i>J</i> = 6.6, 3.7 Hz, 1H), 0.46 (s, 2H), 0.41 (s, 2H). <sup>13</sup>C (101 MHz, CDCl<sub>3</sub>) δ: 166.4, 146.2, 136.2, 129.8, 128.7, 128.6, 128.2, 128.1, 66.6, 53.4, 53.1, 51.6. LRMS (ESI+) <i>m</i>/<i>z</i> calcd for [C<sub>18</sub>H<sub>20</sub>NO<sub>2</sub>]<sup>+</sup>: 282.14, found: 282.30.</div></div><div id="sec4_17_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> <i>N</i>-Cyclopropyl-2,3,4,5,6-pentafluorobenzenesulfonamide (<b>S9a</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (b) as a clear oil without further purification (80%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ: 5.47 (s, 1H), 2.47–2.39 (m, 1H), 0.82–0.70 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ: 156.8, 145.5, 144.8, 116.5, 39.9, 27.8. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ: −137.91 to −138.14 (m, 2F), −148.02 (s, 1F), −160.05 to −161.11 (m, 2F). LRMS (ESI+) <i>m</i>/<i>z</i> calcd for [C<sub>9</sub>H<sub>6</sub>F<sub>5</sub>NO<sub>2</sub>SNa]<sup>+</sup>: 309.99, found: 310.24.</div></div><div id="sec4_17_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> <i>N</i>-Cyclopropyl-2,3,4,5-tetrafluorobenzenesulfonamide (<b>S9b</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (b) as a yellow oil without further purification (83%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN) δ: 7.92–7.30 (m, 1H), 6.40 (s, 1H), 2.34 (p, <i>J</i> = 6.9, 3.6 Hz, 1H), 0.65–0.53 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ: 156.8, 145.5, 144.8, 140.4, 117.3, 23.9, 4.9. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ: −135.25 (s, 1F), −135.39 to −135.78 (m, 1F), −146.02 to −147.09 (m, 1F), −151.30 (s, 1F). LRMS (ESI-) <i>m</i>/<i>z</i> calcd for [C<sub>9</sub>H<sub>6</sub>F<sub>4</sub>NO<sub>2</sub>S]<sup>−</sup>: 268.01, found: 268.02.</div></div><div id="sec4_17_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 2,3,4,5-Tetrafluorobenzenesulfonyl Chloride Used for <b>S9b</b></h4><div class="NLM_p last">The product was obtained using synthetic procedure (g) as a clear, yellow oil without further purification (80%). <sup>1</sup>H NMR (400 MHz, acetonitrile-<i>d</i><sub>3</sub>) δ 7.92–7.66 (m, 1H). <sup>13</sup>C NMR (101 MHz, acetonitrile-<i>d</i><sub>3</sub>) δ 147.8, 147.3, 146.7, 145.3, 144.6, 140.5, 116.6. <sup>19</sup>F NMR (376 MHz, acetonitrile-<i>d</i><sub>3</sub>) δ −133.82 to −134.01 (m, 1F), −135.65 to −136.10 (m, 1F), −142.87 (s, 1F), −151.06 to −151.57 (m, 1F). LRMS (ESI+) found 229.04 (corresponding to M-1 for the acid showing rapid compound hydrolysis under MS conditions).</div></div><div id="sec4_17_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> Benzyl 4-(((<i>N</i>-Cyclopropyl-2,3,4,5,6-pentafluorophenyl)sulfonamido)methyl)benzoate (<b>S4a</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (b) as a viscous pale yellow oil (75%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ: 8.09 (d, <i>J</i> = 8.3 Hz, 2H), 7.53–7.46 (m, 4H), 7.45–7.35 (m, 3H), 5.40 (s, 2H), 4.62 (s, 2H), 2.39 (p, <i>J</i> = 5.3 Hz, 1H), 0.72 (s, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ: 166.0, 141.4, 135.9, 130.1, 129.9, 128.6, 128.3, 128.3, 128.2, 66.8, 54.2, 30.4, 6.9. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ: −134.77 (s, 2F), −145.18 to −145.41 (m, 1F), −158.10 to −158.36 (m, 2F). LRMS (ESI+) <i>m</i>/<i>z</i> calcd for [C<sub>24</sub>H<sub>18</sub>F<sub>5</sub>NO<sub>4</sub>SNa]<sup>+</sup>: 534.08, found: 534.32.</div></div><div id="sec4_17_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> <i>tert</i>-Butyl 4-(((<i>N</i>-Cyclopropyl-2,3,4,5-tetrafluorophenyl)sulfonamido)methyl)benzoate (<b>S4b</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (d) as a viscous pale yellow viscous oil (82%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ: 8.09 (d, <i>J</i> = 8.3 Hz, 2H), 7.68–7.55 (m, 1H), 7.51–7.45 (m, 4H), 7.45–7.33 (m, 3H), 5.40 (s, 2H), 4.61 (s, 2H), 2.32 (p, <i>J</i> = 6.8, 5.6, 4.5, 1.0 Hz, 1H), 0.65 (d, <i>J</i> = 4.4 Hz, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ: 166.0, 141.7, 136.0, 130.3, 130.0, 129.8, 128.6, 128.3, 128.2, 128.1, 112.7, 66.7, 54.1, 30.3, 7.1. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ: −133.29 (s, 1F), −134.73 to −136.53 (m, 1F), −145.42 to −146.92 (m, 1F), −150.93 to −151.01 (m, 1F). LRMS (ESI+) <i>m</i>/<i>z</i> calcd for [C<sub>24</sub>H<sub>19</sub>F<sub>4</sub>NO<sub>4</sub>SNa]<sup>+</sup>: 516.09, found: 516.36.</div></div><div id="sec4_17_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> Benzyl 4-(((<i>N</i>-Cyclopropyl-3,4,5-trifluorophenyl)sulfonamido)methyl)benzoate (<b>S4c</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (b) as a yellow oil (91%). <sup>1</sup>H NMR (400 MHz, THF-<i>d</i><sub>8</sub>) δ: 8.07 (d, <i>J</i> = 8.3 Hz, 2H), 7.76 (t, <i>J</i> = 6.6 Hz, 2H), 7.55–7.49 (m, 4H), 7.45–7.09 (m, 3H), 5.40 (s, 2H), 4.51 (s, 2H), 2.20 (tt, <i>J</i> = 7.0, 3.4 Hz, 1H), 0.85–0.48 (m, 2H). <sup>13</sup>C NMR (101 MHz, THF-<i>d</i><sub>8</sub>) δ: 165.2, 150.1, 143.7, 142.6, 141.2, 136.5, 135.1, 129.7, 129.5, 128.4, 128.3, 127.9, 112.9, 54.2, 27.7, 6.7. <sup>19</sup>F NMR (376 MHz, THF-<i>d</i><sub>8</sub>) δ: −132.18 (s, 1F), −154.98 (s, 2F). LRMS (ESI+) <i>m</i>/<i>z</i> calcd for [C<sub>24</sub>H<sub>20</sub>F<sub>3</sub>NO<sub>4</sub>SNa]<sup>+</sup>: 498.10, found: 497.60.</div></div><div id="sec4_17_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Benzyl 4-(((<i>N</i>-Cyclopropyl-2,4,6-trifluorophenyl)sulfonamido)methyl)benzoate (<b>S4d</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (b) as a yellow oil (81%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ: 8.08 (d, <i>J</i> = 8.3 Hz, 2H), 7.51–7.45 (m, 2H), 7.44–7.33 (m, 2H), 6.81 (t, <i>J</i> = 8.6 Hz, 1H), 5.39 (s, 2H), 4.61 (s, 2H), 2.36 (p, <i>J</i> = 7.0, 3.9 Hz, 1H), 0.97–0.56 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ: 166.0, 142.3, 136.0, 129.9, 129.6, 128.6, 128.2, 128.1, 126.5, 120.8, 102.1, 66.7, 54.0, 30.3, 6.8. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ: −98.44 (s, 1F), −101.59 (s, 2F). LRMS (ESI+) <i>m</i>/<i>z</i> calcd for [C<sub>24</sub>H<sub>20</sub>F<sub>3</sub>NO<sub>4</sub>SNa]<sup>+</sup>: 498.10, found: 498.30.</div></div><div id="sec4_17_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> Benzyl 4-(((<i>N</i>-Cyclopropyl-4-fluorophenyl)sulfonamido)methyl)benzoate (<b>S4e</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (b) as a yellow oil (92%). <sup>1</sup>H NMR (400 MHz, THF) δ 8.07 (d, <i>J</i> = 8.3 Hz, 2H), 8.03–7.97 (m, 2H), 7.57–7.48 (m, 4H), 7.46–7.32 (m, 5H), 5.40 (s, 2H), 4.46 (s, 2H), 2.15–2.04 (m, 1H), 0.71–0.53 (m, 4H). <sup>13</sup>C NMR (101 MHz, THF-<i>d</i><sub>8</sub>) δ: 166.4, 165.3, 163.9, 136.6, 135.0, 130.5, 129.5, 129.5, 128.4, 128.0, 127.9, 116.1, 115.8, 54.1, 31.0, 6.6. <sup>19</sup>F NMR (376 MHz, THF-<i>d</i><sub>8</sub>) δ: −107.05 (s, 1F). LRMS (ESI+) <i>m</i>/<i>z</i> calcd for [C<sub>24</sub>H<sub>22</sub>FNO<sub>4</sub>SNa]<sup>+</sup>: 462.11, found: 462.46.</div></div><div id="sec4_17_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> Benzyl 4-(((3,5-Dichloro-<i>N</i>-cyclopropyl-2,4,6-trifluorophenyl)sulfonamido)methyl)benzoate (<b>S4f</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (b) as a yellow oil (61%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ: 8.09 (s, 2H), 7.57–7.44 (m, 4H), 7.45–7.34 (m, 3H), 5.40 (s, 2H), 4.63 (s, 2H), 2.41 (p, <i>J</i> = 5.8, 5.3 Hz, 1H), 0.99–0.20 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ: 130.0, 128.6, 128.2, 128.2, 126.7, 125.6, 123.4, 107.8, 67.9, 67.7, 29.1, 23.9, 7.0. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ: −101.81 (s, 1F), −105.89 (s, 2F). LRMS (ESI) <i>mass not observed</i>.</div></div><div id="sec4_17_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> Benzyl 4-(((4-Chloro-<i>N</i>-cyclopropyl-2,3,5,6-tetrafluorophenyl)sulfonamido)methyl)benzoate (<b>S4g</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (b) as a yellow oil (61%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ: 8.09 (d, <i>J</i> = 8.3 Hz, 2H), 7.53–7.46 (m, 4H), 7.46–7.34 (m, 3H), 5.40 (s, 2H), 4.63 (s, 2H), 2.46–2.37 (m, 1H), 0.77–0.67 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ: 165.9, 141.4, 135.9, 134.5, 130.0, 129.9, 129.5, 128.6, 128.3, 128.2, 66.8, 54.2, 30.4, 7.0. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ: −134.91 to −135.16 (m, 2F), −137.36 to −137.61 (m, 2F). LRMS (ESI+) <i>m</i>/<i>z</i> calcd for [C<sub>24</sub>H<sub>18</sub>ClF<sub>4</sub>NO<sub>4</sub>SNa]<sup>+</sup>: 550.05, found: 550.30.</div></div><div id="sec4_17_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 4-(((<i>N</i>-Cyclopropyl-2,3,4,5,6-pentafluorophenyl)sulfonamido)methyl)benzoic Acid (<b>S5a</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (c) as a white solid (98%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ: 8.12 (d, <i>J</i> = 8.3 Hz, 2H), 7.52 (d, <i>J</i> = 8.1 Hz, 2H), 4.65 (s, 2H), 2.42 (p, <i>J</i> = 5.2 Hz, 1H), 0.83–0.63 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 167.5, 165.6, 165.4, 150.9, 142.1, 130.7, 130.0, 128.4, 53.6, 30.7, 6.8. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ: −134.65 to −134.83 (m, 2F), −145.18 (s, 1F), −158.16 to −159.22 (m, 2F). LRMS (ESI-) <i>m</i>/<i>z</i> calcd for [C<sub>17</sub>H<sub>11</sub>F<sub>5</sub>NO<sub>4</sub>S]<sup>−</sup>: 420.03, found: 420.38.</div></div><div id="sec4_17_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 4-(((<i>N</i>-Cyclopropyl-3,4,5-trifluorophenyl)sulfonamido)methyl)benzoic Acid (<b>S5c</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (c) as an amorphous solid in a quantitative yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 7.92 (d, <i>J</i> = 8.0 Hz, 2H), 7.84 (m, 2H), 7.40 (d, <i>J</i> = 8.0 Hz, 2H), 4.46 (s, 2H), 2.25 (p, <i>J</i> = 8.8, 6.8, 3.7 Hz, 1H), 0.74–0.53 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 129.7, 128.3, 113.5, 97.6, 66.1, 54.1, 33.6, 31.4, 23.7, 7.2. <sup>19</sup>F NMR (376 MHz, DMSO-<i>d</i><sub>6</sub>) δ: −130.95 to −131.32 (m, 2F), −153.44 (s, 1F). LRMS (ESI-) <i>m</i>/<i>z</i> calcd for [C<sub>17</sub>H<sub>13</sub>F<sub>3</sub>NO<sub>4</sub>S]<sup>−</sup>: 384.05, found: 383.98.</div></div><div id="sec4_17_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 4-(((<i>N</i>-Cyclopropyl-2,4,6-trifluorophenyl)sulfonamido)methyl)benzoic Acid (<b>S5d</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (c) as an amorphous solid in a quantitative yield. <sup>1</sup>H NMR (400 MHz, THF-<i>d</i><sub>8</sub>) δ: 8.03 (d, <i>J</i> = 7.9 Hz, 2H), 7.52 (d, <i>J</i> = 7.9 Hz, 2H), 7.17 (t, <i>J</i> = 9.4 Hz, 2H), 4.63 (s, 2H), 2.20 (p, <i>J</i> = 7.6 Hz, 1H), 0.94–0.47 (m, 2H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>CN) δ: 169.2, 142.4, 140.7, 140.1, 139.8, 130.1, 110.0, 60.6, 30.5, 6.2. <sup>19</sup>F NMR (376 MHz, THF-<i>d</i><sub>8</sub>) δ: −100.67 (s, 1F), −102.55 (s, 2F). LRMS (ESI-) <i>m</i>/<i>z</i> calcd for [C<sub>17</sub>H<sub>13</sub>F<sub>3</sub>NO<sub>4</sub>S]<sup>−</sup>: 384.05, found: 384.21.</div></div><div id="sec4_17_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 4-(((3,5-Dichloro-<i>N</i>-cyclopropyl-2,4,6-trifluorophenyl)sulfonamido)methyl)benzoic Acid (<b>S5f</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (c) as an amorphous solid. <b>S5f</b> and byproduct <b>S5f’</b> were isolated by reverse phase chromatography water/ACN (100:0 → 0:100) as amorphous solids in a quantitative yield, in a 75:25 proportion, respectively. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ: 8.05 (d, <i>J</i> = 8.3 Hz, 2H), 7.54 (d, <i>J</i> = 8.4 Hz, 2H), 4.69 (s, 2H), 2.55 (p, <i>J</i> = 6.7, 4.9, 3.6 Hz, 1H), 0.80–0.66 (m, 4H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>CN) δ: 170.0, 141.1, 140.5, 140.1, 133.2, 132.1, 130.0, 130.1, 111.5, 70.9, 31.3, 7.0. <sup>19</sup>F NMR (376 MHz, methanol-<i>d</i><sub>4</sub>) δ: −105.06 (s, 1F), −107.80 (s, 2F). LRMS (ESI+) <i>m</i>/<i>z</i> calcd for [C<sub>16</sub>H<sub>11</sub>BrF<sub>5</sub>NO<sub>2</sub>SNa]<sup>+</sup>: 454.96, found: 454.31.</div></div><div id="sec4_17_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 4-(((4-Chloro-<i>N</i>-cyclopropyl-2,3,5,6-tetrafluorophenyl)sulfonamido)methyl)benzoic Acid (<b>S5g</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (c) as an amorphous solid in a quantitative yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ: 8.10 (d, <i>J</i> = 7.9 Hz, 2H), 7.50 (d, <i>J</i> = 7.9 Hz, 2H), 4.64 (s, 2H), 2.48–2.32 (m, 1H) 0.79–0.63 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ: 161.2, 150.9, 150.8, 145.6, 145.2, 107.7, 107.4, 106.1, 67.5, 29.0, 23.7. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ: −135.11 to −135.34 (m, 2F), −137.65 to −137.84 (m, 2F). LRMS (ESI-) <i>m</i>/<i>z</i> calcd for [C<sub>17</sub>H<sub>11</sub>ClF<sub>4</sub>NO<sub>4</sub>S]<sup>−</sup>: 436.00, found: 436.31.</div></div><div id="sec4_17_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> <i>N</i>-(Benzyloxy)-4-(((<i>N</i>-cyclopropyl-2,3,4,5,6-pentafluorophenyl)sulfonamido)methyl)benzamide (<b>S6a</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (f) as an amorphous solid (62%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ: 9.11 (s, 1H), 7.68 (d, <i>J</i> = 8.2 Hz, 2H), 7.52–7.31 (m, 7H), 5.03 (s, 2H), 4.57 (s, 2H), 2.37 (p, <i>J</i> = 5.5, 4.8 Hz, 1H), 0.73–0.64 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ: 163.4, 155.9, 154.3, 145.6, 140.4, 135.2, 131.6, 129.3, 128.8, 128.6, 128.4, 127.5, 78.4, 54.1, 30.4, 6.9. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ: −134.90 (s, 2F), −145.01 to −145.29 (m, 1F), −158.00 to −158.30 (m, 2F). LRMS (ESI+) <i>m</i>/<i>z</i> calcd for [C<sub>24</sub>H<sub>19</sub>F<sub>5</sub>N<sub>2</sub>O<sub>4</sub>SNa]<sup>+</sup>: 549.09, found: 549.43.</div></div><div id="sec4_17_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 4-(((3,5-Dichloro-<i>N</i>-cyclopropyl-2,4,6-trifluorophenyl)sulfonamido)methyl)-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-2-yl)oxy)benzamide (<b>S6b</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (f) as an amorphous solid (88%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ: 9.24 (s, 1H), 7.76 (d, <i>J</i> = 8.1 Hz, 2H), 7.61 (m, 1H), 7.43 (d, <i>J</i> = 8.0 Hz, 2H), 5.08 (s, 1H), 4.56 (s, 2H), 4.03 (dd, <i>J</i> = 10.4, 9.1, 2.9 Hz, 1H), 3.70–3.61 (m, 1H), 1.99–1.75 (m, <i>J</i> = 11.0, 9.8, 5.3 Hz, 3H), 1.75–1.47 (m, <i>J</i> = 14.5, 10.6, 5.1 Hz, 4H), 1.33–1.17 (m, 1H), 0.69–0.49 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ: 140.7, 131.6, 128.5, 127.6, 112.7, 102.7, 68.5, 62.6, 54.0, 30.3, 28.0, 27.7, 25.0, 22.1, 18.6, 7.0. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ: −133.36 to −133.98 (m, 1F), −135.31 (s, 1F), −145.79 to −146.76 (m, 1F), −150.49 to −151.20 (m, 1F). LRMS (ESI) <i>mass not observed</i>.</div></div><div id="sec4_17_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> <i>N</i>-(Benzyloxy)-4-(((<i>N</i>-cyclopropyl-3,4,5-trifluorophenyl)sulfonamido)methyl)benzamide (<b>S6c</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (f) as an amorphous solid (71%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN) δ: 9.99 (s, 1H), 7.76–7.67 (m, 2H), 7.67–7.57 (m, 2H), 7.53–7.46 (m, 2H), 7.47–7.36 (m, 5H), 4.99 (s, 2H), 4.43 (s, 2H), 2.26–2.18 (m, 1H), 0.72–0.64 (m, 4H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>CN) δ: 164.9, 152.2, 149.7, 144.0, 141.1, 136.0, 134.4, 131.7, 129.2, 128.5, 128.4, 127.2, 114.6, 77.7, 54.1, 31.1, 6.8. <sup>19</sup>F NMR (376 MHz, CD<sub>3</sub>CN) δ: −132.13 to −132.99 (m, 2F), −154.83 (s, 1F). LRMS (ESI-) <i>m</i>/<i>z</i> calcd for [C<sub>24</sub>H<sub>21</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub>SCl]<sup>−</sup>: 525.09, found: 525.19.</div></div><div id="sec4_17_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> <i>N</i>-(Benzyloxy)-4-(((<i>N</i>-cyclopropyl-2,4,6-trifluorophenyl)sulfonamido)methyl)benzamide (<b>S6d</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (f) as an off-white solid (61%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN) δ: 10.01 (s, 1H), 7.80–7.65 (m, 2H), 7.57–7.34 (m,7H), 7.12–6.97 (m, 2H), 4.99 (s, 2H), 4.59 (s, 2H), 2.27–2.16 (m, 1H), 0.72–0.58 (m, 4H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>CN) δ: 178.0, 154.3, 151.0, 150.4, 141.4, 135.9, 131.7, 129.2, 128.5, 128.4, 127.3, 117.3, 77.8, 53.5, 30.2, 6.3. <sup>19</sup>F NMR (376 MHz, CD<sub>3</sub>CN) δ: −100.83 to −101.19 (m, 1F), −103.33 to −103.70 (m, 2F). LRMS (ESI-) <i>m</i>/<i>z</i> calcd for [C<sub>24</sub>H<sub>21</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub>SCl]<sup>−</sup>: 525.09, found: 525.33.</div></div><div id="sec4_17_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> <i>N</i>-(Benzyloxy)-4-(((<i>N</i>-cyclopropyl-4-fluorophenyl)sulfonamido)methyl)benzamide (<b>S6e</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (f) as a yellow solid (67%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN) δ: 7.96–7.83 (m, 2H), 7.71–7.63 (m, 2H), 7.52–7.46 (m, 2H), 7.45–7.29 (m, 7H), 4.40 (s, 2H), 2.17–2.05 (m, 1H), 0.67–0.57 (m, 4H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>CN) δ: 169.9, 152.3, 151.9, 151.8, 151.5, 150.9, 142.2, 136.4, 131.0, 129.0, 128.3, 112.5, 112.3, 77.8, 53.0, 23.3, 21.5, 13.9, 6.0. <sup>19</sup>F NMR (376 MHz, acetone-<i>d</i><sub>6</sub>) δ: −107.10 to −107.39 (m, 1F). LRMS (ESI-) <i>m</i>/<i>z</i> calcd for [C<sub>24</sub>H<sub>23</sub>FN<sub>2</sub>O<sub>4</sub>SCl]<sup>−</sup>: 489.11, found: 489.35.</div></div><div id="sec4_17_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 4-(((3,5-Dichloro-<i>N</i>-cyclopropyl-2,4,6-trifluorophenyl)sulfonamido)methyl)-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-2-yl)oxy)benzamide (<b>S6f</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (f) as an amorphous solid (82%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ: 9.14 (s, 1H), 7.76 (d, <i>J</i> = 8.1 Hz, 2H), 7.45 (d, <i>J</i> = 8.0 Hz, 2H), 5.10 (s, 1H), 4.60 (s, 2H), 4.12–3.91 (m, 1H), 3.71–3.62 (m, 1H), 1.97–1.82 (m, 3H), 1.74–1.56 (m, 3H), 1.33–1.23 (m, 1H), 0.69 (d, <i>J</i> = 5.3 Hz, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ: 156.4, 145.4, 143.2, 140.5, 131.7, 128.5, 127.6, 102.7, 62.6, 54.2, 30.4, 28.0, 25.0, 18.6, 7.0. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ: −101.73 (s, 1F), −105.97 to −106.12 (m, 2F). LRMS (ESI) <i>mass not observed</i>.</div></div><div id="sec4_17_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 4-(((3-Chloro-<i>N</i>-cyclopropyl-2,4,6-trifluorophenyl)sulfonamido)methyl)-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-2-yl)oxy)benzamide (<b>S6f’</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (f) as an amorphous solid (73%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ: 9.12 (s, 1H), 7.76 (d, <i>J</i> = 8.2 Hz, 2H), 7.45 (d, <i>J</i> = 8.0 Hz, 2H), 6.96 (m, 1H), 5.09 (s, 1H), 4.59 (s, 2H), 4.12–3.95 (m, 1H), 3.73–3.57 (m, 1H), 2.36 (p, <i>J</i> = 5.3 Hz, 1H), 1.99–1.74 (m, 4H), 1.74–1.47 (m, 4H), 0.75–0.58 (m, <i>J</i> = 5.3 Hz, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ: 165.7, 145.1, 144.2, 144.0, 140.8, 131.6, 127.6, 102.7, 62.6, 54.1, 30.3, 28.0, 25.0, 18.6, 6.9. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>) δ: −101.01 (s, 1F), −103.18 to −104.54 (m, 1F), −104.00 (t, <i>J</i> = 9.9 Hz, 1F). LRMS (ESI) <i>mass not observed</i>.</div></div><div id="sec4_17_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 4-(((Perfluorophenyl)sulfonamido)methyl)-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-2-yl)oxy)benzamide (<b>S6l</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (f) as an off-white solid (70%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 7.77 (d, <i>J</i> = 8.3 Hz, 2H), 7.44 (d, <i>J</i> = 8.2 Hz, 2H), 5.08 (s, 1H), 4.46 (s, 2H), 2.13–1.28 (m, 8H). <sup>13</sup>C NMR (101 MHz, acetone) δ 205.5, 154.2, 145.7, 140.3, 139.1, 132.0, 127.9, 102.0, 62.3, 47.9, 28.0, 25.4, 25.3, 25.0, 19.9, 18.4. <sup>19</sup>F NMR (376 MHz, acetone-<i>d</i><sub>6</sub>) δ −138.08 to −139.04 (m, 2F), −150.03 (s, 1F), −161.43 to −162.65 (m, 2F). LRMS (ESI-) <i>m</i>/<i>z</i> calcd for [C<sub>19</sub>H<sub>16</sub>F<sub>5</sub>N<sub>2</sub>O5<sub>S</sub>]<sup>−</sup>: 479.07, found: 479.38.</div></div><div id="sec4_17_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> Benzyl 4-(Aminomethyl)benzoate Hydrochloride (<b>S2c</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (i) as a white solid (83%). <sup>1</sup>H δ/ppm (400 MHz, DMSO-d6) 4.10 (s, 2H), 5.36 (s, 2H), 7.34–7.48 (m, 5H), 7.66 (d, <i>J</i> = 8.2 Hz, 2H), 8.02 (d, <i>J</i> = 8.2 Hz, 2H); <sup>13</sup>C δ/ppm (100 MHz, DMSO-<i>d6</i>) 41.7, 66.3, 127.9, 128.1, 128.5, 129.2, 129.4, 129.5, 136.1, 139.6, 165.2; LRMS (ESI+) <i>m</i>/<i>z</i> calcd for [C<sub>15</sub>H<sub>15</sub>NO<sub>2</sub>Na]<sup>+</sup>: 264.10, found: 264.08.</div></div><div id="sec4_17_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> Benzyl 4-(((Perfluorophenyl)sulfonamido)methyl)benzoate (<b>S4i</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (b) as a light yellow solid (69%). <sup>1</sup>H δ/ppm (400 MHz, CDCl<sub>3</sub>) 4.67 (d, <i>J</i> = 5.9 Hz, 2H), 5.34 (s, 2H), 6.51 (s, 1H), 7.33–7.45 (m, 7H), 8.03 (d, <i>J</i> = 8.3 Hz, 2H); <sup>13</sup>C δ/ppm (100 MHz, CDCl<sub>3</sub>) 44.0, 67.0, 117.4, 127.6, 128.3, 128.5, 128.8, 129.9, 130.4, 136.0, 138.9, 142.2, 143.2, 145.6, 157.6, 166.2; <sup>19</sup>F δ/ppm (376 MHz, CDCl<sub>3</sub>) −159.6 to −160.0 (m, 2F), - 150.1 (s, 1F), −140.2 to −140.4 (m, 2F); LRMS (ESI+) <i>m</i>/<i>z</i> calcd for [C<sub>22</sub>H<sub>14</sub>F<sub>5</sub>NO<sub>3</sub>Na]<sup>+</sup>: 458.08, found: 458.41.</div></div><div id="sec4_17_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> <i>tert</i>-Butyl 4-(((2,3,4,5,6-pentafluoro-<i>N</i>-isopropylphenyl)sulfonamido)methyl)benzoate (<b>S11a</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (d) as a viscous pale yellow oil (27%). <sup>1</sup>H NMR (400 MHz, acetonitrile-<i>d</i><sub>3</sub>) δ 7.92 (d, <i>J</i> = 8.4 Hz, 2H), 7.48 (d, <i>J</i> = 8.4 Hz, 2H), 4.60 (s, 2H), 4.33 (p, <i>J</i> = 6.8 Hz, 1H), 1.60 (s, 9H), 1.11 (d, <i>J</i> = 6.8 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ: 165.50, 141.43, 135.97, 130.10, 129.93, 128.64, 128.31, 128.29, 127.60, 65.01, 54.25, 32.24, 7.07. <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.5, 147.4, 147.3, 145.8, 145.0, 138.8, 133.6, 133.5, 112.0, 112., 39.71, 39.6, 23.6, 17.6, 12.3. <sup>19</sup>F NMR (376 MHz, acetonitrile-<i>d</i><sub>3</sub>) δ −137.37 to −137.57 (m, 2F), −149.44 (s, 1F), −161.26 to −161.46 (m, 2F). LRMS (ESI-) <i>m</i>/<i>z</i> calcd for [C<sub>21</sub>H<sub>22</sub>ClF<sub>5</sub>NO<sub>4</sub>S]<sup>+</sup>: 514.09, found: 514.40.</div></div><div id="sec4_17_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 4-(((2,3,4,5,6-Pentafluoro-<i>N</i>-isopropylphenyl)sulfonamido)methyl)benzoic Acid (<b>S5j</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (e) as an off-white solid in a quantitative yield. <sup>1</sup>H NMR (400 MHz, acetonitrile-<i>d</i><sub>3</sub>) δ 7.98 (d, <i>J</i> = 8.3 Hz, 2H), 7.52 (d, <i>J</i> = 8.3 Hz, 2H), 4.62 (s, 2H), 4.32 (p, <i>J</i> = 6.8 Hz, 1H), 1.10 (d, <i>J</i> = 6.8 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ: 165.5, 141.4, 135.9, 135.7, 129.0, 128.9, 128.3, 127.9, 124.1, 72.0, 50.5, 20.3. <sup>19</sup>F NMR (376 MHz, acetonitrile-<i>d</i><sub>3</sub>) δ −137.37 to −137.57 (m, 2F), −149.44 (s, 1F), −161.26 to −161.46 (m, 2F). LRMS (ESI) <i>mass not observed</i>.</div></div><div id="sec4_17_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> <i>N</i>-Cyclopropyl-2,3,5,6-tetrafluorobenzenesulfonamide (<b>S9d</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (b) as a yellow oil without further purification (68%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) 7.39–7.28 (m, 1H), 2.47–2.35 (m, 1H), 0.74–0.63 (m, 4H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>CN) δ: 140.3, 133.9, 120.3, 15.7, 4.9. <sup>19</sup>F NMR (376 MHz, acetonitrile-<i>d</i><sub>3</sub>) δ −137.67 to −138.01 (m, 2F), −137.79 to −138.07 (m, 2F). LRMS (ESI) <i>mass not observed</i>.</div></div><div id="sec4_17_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> <i>tert</i>-Butyl 4-(((<i>N</i>-Cyclopropyl-2,3,5,6-tetrafluorophenyl)sulfonamido)methyl)benzoate (<b>S11b</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (d) as a pale yellow viscous oil (77.4%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.97 (d, <i>J</i> = 8.2 Hz, 2H), 7.44 (d, <i>J</i> = 8.2 Hz, 2H), 7.39–7.28 (m, 1H), 4.61 (s, 2H), 2.44–2.33 (m, 1H), 1.60 (s, 9H), 0.74–0.63 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ: 166.0, 141.7, 136.0, 130.3, 130.0, 129.8, 128.6, 128.3, 128.2, 128.1, 67.7, 54.1, 30.2, 7.0. <sup>19</sup>F NMR (376 MHz, chloroform-<i>d</i>) δ −135.41 (s, 2F), −135.86 to −136.09 (m, 2F). LRMS (ESI) <i>mass not observed</i>.</div></div><div id="sec4_17_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> Benzyl 3-Formylbenzoate (<b>S1b</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (i) as a light yellow oil (98%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 10.07 (s, 1H), 8.56 (m, 1H), 8.34 (dd, <i>J</i> = 7.7, 1.5 Hz, 1H), 8.09 (dd, <i>J</i> = 7.7, 1.5 Hz, 1H), 7.63 (m, 1H), 7.51–7.46 (m, 2H), 7.45–7.36 (m, 3H), 5.42 (s, 2H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>) δ 191.3, 165.3, 136.6, 135.6, 135.2, 133.1, 131.3, 131.2, 129.3, 128.7, 128.5, 128.4, 67.2. LRMS (ESI+) <i>m</i>/<i>z</i> calcd for [C<sub>15</sub>H<sub>12</sub>O<sub>3</sub>]<sup>+</sup>: 240.08, found: 241.11.</div></div><div id="sec4_17_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> Benzyl 3-((Cyclopropylamino)methyl)benzoate (<b>S2b</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (a) as a pale yellow oil (93%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.06 (s, 1H), 8.00 (dt, <i>J</i> = 7.8, 1.5 Hz, 1H), 7.55 (dt, <i>J</i> = 7.7, 1.5 Hz, 1H), 7.49 (d, 2H), 7.42 (dt, <i>J</i> = 7.7, 1.9 Hz, 3H), 7.38 (d, 1H), 5.40 (s, 2H), 3.90 (s, 2H), 2.16 (p, 1H), 0.48–0.45 (m, 2H), 0.45–0.40 (m, 2H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>) δ 166.5, 141.0, 136.1, 133.0, 130.2, 129.4, 128.6, 128.4, 128.3, 128.2, 128.2, 66.7, 53.3, 43.5, 30.1, 6.5. LRMS (ESI+) <i>m</i>/<i>z</i> calcd for [C<sub>18</sub>H<sub>19</sub>NO<sub>2</sub>]<sup>+</sup>: 281.36, found: 282.11.</div></div><div id="sec4_17_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> Benzyl 3-(((<i>N</i>-Cyclopropyl-2,3,4,5,6-pentafluorophenyl)sulfonamido)methyl)benzoate (<b>S4h</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (b) 78%. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.07–8.02 (m, 2H), 7.67 (m, 1H), 7.50 (s, 1H), 7.48 (s, 2H), 7.45–7.40 (m, 3H), 5.40 (s, 2H), 4.62 (s, 2H), 2.40 (p, 1H), 0.74 (s, 2H), 0.73 (s, 2H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>) δ 166.0, 136.4, 135.9, 133.0, 130.5, 129.5, 129.3, 128.9, 128.6, 128.3, 128.2, 66.8, 54.1, 30.3, 7.1. <sup>19</sup>F NMR (376 MHz, chloroform-<i>d</i>) δ −134.65 (dq, <i>J</i> = 21.0, 6.9, 6.0 Hz, 2F), −143.09 to −149.37 (m, 1F), −156.12 to −161.92 (m, 2F). LRMS (ESI+) <i>m</i>/<i>z</i> calcd for [C<sub>24</sub>H<sub>18</sub>F<sub>5</sub>NO<sub>4</sub>S]<sup>+</sup>: 511.09, found: 512.11.</div></div><div id="sec4_17_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> 3-(((<i>N</i>-Cyclopropyl-2,3,4,5,6-pentafluorophenyl)sulfonamido)methyl)benzoic Acid (<b>S5h</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (c) as an amorphous white solid in a quantitative yield (98%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.06 (s, 1H), 8.01 (d, <i>J</i> = 7.8 Hz, 1H), 7.66 (d, <i>J</i> = 7.6 Hz, 1H), 7.52 (t, <i>J</i> = 7.7 Hz, 1H), 4.70 (s, 2H), 2.54 (p, <i>J</i> = 6.9, 3.8 Hz, 1H), 0.78–0.61 (m, 4H). <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 167.4, 137.2, 132.4, 132.0, 129.4, 128.9, 128.6, 53.7, 30.1, 6.4. <sup>19</sup>F NMR (376 MHz, acetone-<i>d</i><sub>6</sub>) δ −136.80 (s, 2F), −145.37 to −152.60 (m, 1F), −161.51 to −162.30 (m, 2F). LRMS (ESI+) <i>m</i>/<i>z</i> calcd for [C<sub>17</sub>H<sub>12</sub>F<sub>5</sub>NO<sub>4</sub>S]<sup>+</sup>: 421.04, found: 422.21.</div></div><div id="sec4_17_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> 3-(((<i>N</i>-Cyclopropyl-2,3,4,5,6-pentafluorophenyl)sulfonamido)methyl)-<i>N</i>-((tetrahydro-2<i>H</i>-pyran-2-yl)oxy)benzamide (<b>S6h</b>)</h4><div class="NLM_p last">The product was obtained using synthetic procedure (f) as a clear oil (84.5%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.32 (s, 1H), 7.73 (s, 1H), 7.68 (d, <i>J</i> = 7.7 Hz, 1H), 7.56 (d, <i>J</i> = 7.8 Hz, 1H), 7.41 (t, <i>J</i> = 7.7 Hz, 1H), 5.08 (s, 1H), 4.56 (s, 2H), 1.93–1.75 (m, 4H), 1.73–1.53 (m, 4H), 0.68 (d, <i>J</i> = 5.7 Hz, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 146.0, 143.4, 139.5, 138.1, 133.1, 131.6, 129.0, 127.4, 126.8, 115.1, 101.5, 61.9, 53.7, 30.6, 28.4, 25.2, 18., 14.25, 6.9. <sup>19</sup>F NMR (376 MHz, chloroform-<i>d</i>) δ −134.85 to −135.69 (m, 2F), −145.52 to −146.03 (m, 1F), −154.98 to −162.40 (m, 2F). LRMS (ESI+) <i>m</i>/<i>z</i> calcd for [C<sub>22</sub>H<sub>21</sub>F<sub>5</sub>N<sub>2</sub>O<sub>5</sub>S]<sup>+</sup>: 520.11, found: 520.54.</div></div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> Western Blotting</h3><div class="NLM_p last">Briefly, MV4-11 cells were incubated with inhibitors prior to cell lysis with radioimmunoprecipitation assay (RIPA) buffer (20 mM Tris pH 7.4, 150 mM NaCl, 0.5% deoxycholate, 1% Triton X-100, and 0.1% sodium dodecyl sulfate (SDS)). Total protein content was determined through a BCA assay (ThermoFisher), resolved via a 4–20% polyacrylamide SDS gel, and transferred to a nitrocellulose membrane (Bio-Rad). The membranes were blocked with a 5% solution (skimmed milk powder in PBS-T). This was followed by incubation at 4 °C (overnight) with the following antibodies: acetylated α-tubulin mouse monoclonal (MABT868, EMD Millipore), acetylated histone H3 (Ac-Lys18, 07–354, Sigma), PARP-1 (ab227244, Abcam), apoptosis Western blot cocktail (136812, Abcam), cleaved PARP-1 (ab32561, Abcam), and HSC70 (sc-7298, Santa Cruz). Following overnight incubation, horseradish peroxidase (HRP)-conjugated goat anti-mouse IgG secondary antibody (7076, Cell Signaling) or HRP-linked anti-rabbit IgG secondary antibody (7074, Cell Signaling) was applied to the membrane in a 1:5000 dilution. The bands were visualized using clarity Western ECL substrate luminal/enhancer solution and peroxide solution. Western blotting analysis was carried out using Image lab software (Bio-Rad).<a onclick="showRef(event, 'ref47 ref50 ref70'); return false;" href="javascript:void(0);" class="ref ref47 ref50 ref70">(47,50,70)</a></div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> Cytotoxicity Assays</h3><div class="NLM_p">HeLa cells were grown in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 10% fetal bovine serum (FBS) (Sigma-Aldrich). MV4-11 were grown in Iscove’s modified Dulbecco’s medium (IMDM) supplemented with 10% FBS. MOLM-13 and MRC-9 cells were maintained in RPMI-1640 supplemented with 10% FBS. HeLa, MOLM-13, MRC-9, and MV4-11 were obtained from the American Type Culture Collection (ATCC, USA).</div><div class="NLM_p">The PTCL cell lines, KHYG-1, NK-92, SNK6, DERL-2, Mac1, 1305 Myla, Hut78, SU-DHL-1 or T-ALL cell lines SUP-T11, DND-41, and MOLT4 were maintained in RPMI-1640 supplemented with 10% FBS, 0.06 g/L penicillin/0.1 g/L streptomycin (Pen/Strep, Gibco), and 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine (Gibco). Culture media of KHYG-1, NK-92, SNK6, and DERL-2 cells was additionally supplemented with 2.5 ng/mL recombinant human IL-2 (ImmunoTools GmbH, Germany). The authenticity of PTCL cell lines was confirmed by analysis of highly polymorphic short tandem repeat loci (STR) using the PowerPlex 16 HS System (Promega; performed by Microsynth AG, Switzerland).</div><div class="NLM_p">Hut78 cells were obtained from CLS Cell Lines Service GmbH, Germany. Mac1, SU-DHL-1, HH, DERL-2, KHYG-1, NK-92, SUP-T11, and YT cell lines were obtained from the Deutsche Sammlung von Mikroorganismen and Zellkulturen GmbH (DSMZ, Germany). SNK6 and NK-YS were a generous gift from Dr. Wing C. Chan (City of Hope Medical Center, Duarte, CA, USA), and the DND-41 cell line was a generous gift from A. Thomas Look (Dana-Farber Cancer Institute, Boston, MA, USA). Myla cells were a generous gift from Dr. K. Kaltoft, Institute of Human Genetics, University of Aarhus (Aarhus, Denmark) via European Collection of Authenticated Cell Cultures (ECACC). MOLT4 cells were obtained from ATCC, USA.</div><div class="NLM_p">NHF and pPF were grown with Cell System growth medium, supplemented with Culture Boost, whereas the U87-MG cells were grown in DMEM (and 10% FBS). HUVECs were grown in Vascular Cell Basal medium supplemented with Endothelial Cell Grow Kit-VEGF. NHF and pPF were obtained from Cell Systems, USA. U87-MG and HUVEC cell lines were obtained from ATCC, USA.</div><div class="NLM_p">Cell lines were regularly tested for mycoplasma using the MycoAlert mycoplasma detection kit (Lonza Group AG, Switzerland). All cell lines were cultured at 37 °C in a humidified atmosphere containing 5% CO<sub>2</sub>. Experiments were performed within 20 passages after cell resuscitation. None of the above-mentioned cell lines are listed in the register of cell lines that are known to be misidentified through cross-contamination.</div><div class="NLM_p last">Cells were plated in 96-well flat-bottom sterile culture plates with low-evaporation lids (Costar #3997). The inhibitors and a vehicle control (0.5% DMSO) were added to the cells following 24 h. After 72 h, Cell Titer-Blue (Promega #G808A) was added to each well (20 μL), and the fluorescence was measured at 560/590 nm using a Cytation S63 spectrophotometer (BioTek) or a GloMax Discover Microplate Reader (Promega, Madison, WI, USA). GraphPad Prism 6.0 (GraphPad Software Inc.) was used to determine the IC<sub>50</sub> values.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> Immunofluorescence Assay</h3><div class="NLM_p last">HeLa cells were plated to subconfluency on a 96-well plate (clear bottom black from Fisher Scientific). As above, inhibitors were introduced following 24 h. The cells were washed with 1× PBS, fixed with 4% formaldehyde (Millipore Sigma), permeabilized with 1% Triton X-100 (Millipore Sigma), and blocked in 5% bovine serum albumin (BSA) (BioShop) for 1 h at room temperature. Following this treatment, the cells were incubated in a cocktail composed of acetylated α-tubulin mouse monoclonal (1:100 dilution, MD Millipore) and acetylated histone H3 (1:50 dilution, Ac-Lys18, Sigma) antibodies. Secondary antibodies anti-mouse Alexa Fluor 647 and anti-rabbit Alexa Fluor 488 were used for immunofluorescent detection. Cells were counterstained for nucleic acids using 4′,6-diamidino-2-phenylindole (DAPI) (ThermoFisher Scientific). The cells were imaged with a Cytation S63 spectrophotometer and the fluorescence intensity was determined using ImageJ (<i>n</i> = 3). Two-way ANOVA with Tukey’s multiple comparisons test were performed using GraphPad Prism 6.0 (GraphPad Software Inc.). A <i>p</i>-value ≤ 0.05 was considered significant.</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> FACS Apoptosis Detection Assay</h3><div class="NLM_p last">MV4-11 cells were prepared, dosed, and washed with cold 1× PBS. The resulting cell pellets were resuspended in 1× Binding Buffer (1 × 10<sup>6</sup> cell/mL) from the FITC Annexin V Apoptosis Detection Kit I (BD Pharmingen). Subsequently, the dyes annexin V (5 μL) and propidium iodide (PI, 5 μL) were added to 2.5 × 10<sup>5</sup> cells (250 μL). The suspension was thoroughly mixed and incubated in the dark for 15 min. Following the addition of 250 μL of 1× binding buffer, the cells were analyzed by flow cytometry within 1 h using the Cytoflex S system (Beckman Coulter).</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> Maximum Tolerated Dose Studies</h3><div class="NLM_p last">Acute toxicity of the lead inhibitors was studied using 4–12 week old male CD-1 mice (Charles River). A concentration of 20 mg/kg of inhibitor via oral gavage was administered to mice (<i>n</i> = 5) over 5 days. The vehicle consisted of 10% DMSO, 5% Cremophor EL, and 85% saline. Mice were monitored daily for toxicity effects including excessive changes in weight and hair as well as changes to posture and inactivity. All mouse studies were approved by the Local Animal Care Committee at the University of Toronto and Office of Laboratory Animal Welfare (OLAW). The mice were housed in Tecniplast Blueline ventilated cages with environmental enrichment and fed a Teklad 2019 diet supplemented with municipal water.</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> Computational Modeling and Docking</h3><div class="NLM_p last">Computational modeling and docking was performed with Schrödinger Maestro 11.9.011 software using Glide, Epik, Optimized Potentials for Liquid Simulations 3e (OPLS3e) force-field, Glide, LigPrep and the Protein Preparation Wizard. All ligand poses as well as protein structure images were generated with Maestro version11.9.011. Relevant compounds were prepared via a ligand preparation workflow (Epik to generate possible protonation states at target pH 7.0 ± 2.0) and generating tautomers. Protein structures were retrieved from the protein data bank (<a href="https://www.rcsb.org" class="extLink">https://www.rcsb.org</a>): HDAC6, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU">5EDU</a>; HDAC8, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1T64">1T64</a>. For protein model preparation, appropriate bond orders were assigned as well as including hydrogen atoms and creating zero-order bonds to any metal centers. Any of the loops and side chains that were not present in the protein structures were added using Prime. Furthermore, any water molecules that were located >5.0 Å from heteroatoms were removed. All residues with steric clashes were individually minimized (OPLS3e). Ramachandran plot analysis was used to examine the structure output. Receptor grids (10 × 10 × 10 Å<sup>3</sup>) were defined surrounding the cocrystallized ligands (Trichostatin A, TSA) within the HDAC active site during docking. The relevant HDACi were screened against HDAC6 and HDAC8 using standard precision (SP) docking, and the top 25 binding poses were generated. In all simulations, the ligands were flexible and the protein remained static (excluding rotatable groups). After postdocking minimizations, the top binding pose for each ligand was produced and analyzed for protein–ligand interactions using Maestro 11.9.011 and Ligand Interaction Diagrams.</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> Permeability Determination by Lipid-PAMPA</h3><div class="NLM_p last">A 1.8% solution (w/v) of lecithin in dodecane was added to each acceptor plate well (top), followed by application of the artificial membrane and addition of 300 μL of PBS (pH 7.4) solution to each well of the acceptor plate. Compounds were added to the donor plate and incubated at 25 °C, 60 rpm for 16 h. After incubation, aliquots of 50 μL from each acceptor well and donor plate were transferred into a 96-well plate, vortexed at 750 rpm for 100 s and centrifuged at 3220g for 20 min. Compound concentrations and effective permeability (<i>P</i><sub>e</sub>) were determined by LC/MS/MS.</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> Stability in Mouse Plasma</h3><div class="NLM_p">The plasma stability of the compounds was evaluated in mouse plasma provided from Sigma-Aldrich, Oakville, Canada. The main stock solutions of the compounds were prepared at 10 mM in DMSO, which was diluted to 100 μM in 50% acetonitrile (ACN)/50% water. Aliquots of 356.5 μL of plasma proteins were placed in a heating block and allowed to equilibrate at 37 °C for 5 min. Then, 3.5 μL of 100 μM compound solutions were added to each vial (final concentration of 1 μM). Three quality control (QC) samples at 100, 500, and 1000 nM were prepared in 0.5% ACN. The vial contents were transferred in 50 μL aliquots at time points of 0.5, 5, 10, 20, 30, and 60 mi to a 96-well autosampler plate containing 150 μL of protein precipitation solution. Ice-cold ACN containing internals standards (100 nM glyburide) were used as the protein precipitation solution. After centrifugation at 4 °C, 5500 rpm for 15 min, the supernatant was diluted in MQ water, and injected into the LC/MS/MS for quantitative analysis.</div><div class="NLM_p last">The mobile phase consisted of (A) 0.1% (v/v) formic acid in Milli-Q water; (B) 0.1% (v/v) formic acid in acetonitrile. Gradients were run over 15 min and proceeded as follows: A:B, 85:15, 0.0–1 min, 85:15 → 10:90, 1–7 min, 10:90, 7–9 min, 10:90 → 85:15, 9–9.5 min, 85:15, 9.5–15 min. The analytical column was a Waters T3 HSS iKey 1.7 μm (50 × 0.15 mm) column. The MS data was collected via multiple reaction monitoring in positive ion mode. All calculations were carried out using Microsoft Excel. Peak areas were determined from extracted ion chromatograms. The <i>in vitro</i> half-life (<i>t</i><sub>1/2</sub>) of parent compounds were determined by regression analysis of the percent parent disappearance vs time curve.</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> Glutathione Stability Assay</h3><div class="NLM_p last">Compound was added to PBS and 5 mM GSH to reach a final concentration of 5 μM. Samples were incubated at 25 °C at 60 rpm, and aliquots were taken at 0, 30, 60, and 120 min and quenched with internal standard, IS (100 nM alprazolam, 200 nM caffeine, 200 nM Labetalol, and 100 nM tolbutamide). Samples were vortexed and centrifuged for 45 min at 3220<i>g</i>. Aliquots were diluted by ultrapure water, and used for LC/MS/MS analysis. Peak area ratios were determined and percent compounds remaining was calculated by the following equation:<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_m001.gif" alt="" id="_i89" /></img></span>where peak area ratio (<i>t</i> min) is the peak area ratio of control and test compounds at <i>t</i> min; peak area ratio<sub>0min</sub> is the peak area ratio of control and test compounds at the zero time point. <i>In vitro</i> half-life (<i>t</i><sub>1/2</sub>) = −(0.693/<i>k</i>).</div></div><div id="sec4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> Reversibility of Inhibition (<i>K</i><sub>off</sub> Kinetics)</h3><div class="NLM_p last">The enzyme was preincubated with compound or DMSO, followed by dilution of the [compound/enzyme] complex into buffer with a substrate peptide to bring the compound concentration ∼10× below its respective IC<sub>50</sub> value. Progress curves were obtained after dilution, and enzyme recovery was fitted with the following equation: % conv = <i>V</i><sub>s</sub> × <i>t</i> + (<i>V</i><sub>i</sub> – <i>V</i><sub>s</sub>)/<i>K</i><sub>obs</sub> app × (1 – exp(−<i>K</i><sub>obs</sub> app × t), in which <i>V</i><sub>i</sub> is the initial velocity, <i>V</i><sub>s</sub> is steady state velocity, and <i>K</i><sub>obs</sub> is the observed rate constant. The obtained <i>K</i><sub>obs</sub> rate constant was used to determine the reversibility of inhibition and calculate residence time as 1/2<i>T</i><sub>res</sub> = ln2 × 1/<i>K</i><sub>obs</sub>.</div></div><div id="sec4_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> Determination of <i>K</i><sub>i</sub> (<i>K</i><sub>on</sub> Kinetics)</h3><div class="NLM_p last">Serial dilutions of compound were mixed with 3 nM HDAC6 enzyme and substrate peptide in assay buffer. Progress curves were acquired without preincubation for ∼5 h on a Labchip3000 instrument. Progress curves in the presence of compound were fit with the following equation: ((<i>A</i> + (<i>V</i><sub>s</sub> × <i>x</i>)) + (((<i>V</i><sub>o</sub> – <i>V</i><sub>s</sub>) × (1 – exp((−1 × <i>K</i><sub>obs</sub>) × t)))/<i>K</i><sub>obs</sub>)). To determine observed rate of inhibition, V<sub>o</sub> was locked to that in control (V<sub>i</sub>, DMSO only). Fractional steady state velocity (<i>V</i><sub>s</sub>/<i>V</i><sub>i</sub>) and <i>K</i><sub>obs</sub> values were plotted against [compound] to determine <i>K</i><sub>i</sub> and residence time.</div></div><div id="sec4_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> <i>In Vitro</i> Stability in Mouse Hepatocytes</h3><div class="NLM_p last">The inhibitors (and controls) were formulated to concentrations of 100 μM in 50% acetonitrile/50% water. The medium (William’s E Medium supplemented with GlutaMAX) and hepatocyte thawing medium were allowed to reach room temperature and the hepatocytes and thawing medium and centrifuged at 100 g for 10 min. The supernatant was removed, and hepatocytes resuspended in the incubation medium (∼1.5 × 10<sup>6</sup> cells/mL). The cells were diluted to 0.5 × 10<sup>6</sup> viable cells/mL. A volume of 198 μL of hepatocytes was transferred to a 96-well noncoated plate and incubated at 37 °C for 10 min. Following this, 2 μL of the 100 μM test compounds or positive control was transferred into respective wells to start the reaction. At different time points (0, 15, 30, 60, 90, and 120 min), 25 μL aliquots were mixed with 6 volumes (150 μL) of acetonitrile containing internal standard, IS (100 nM alprazolam, 200 nM labetalol, 200 nM caffeine, and 200 nM diclofenac) to terminate the reaction. Samples were vortexed for 5 min and centrifuged for 45 min at 3220<i>g</i>, diluted by ultrapure water, followed by LC/MS/MS analysis. The <i>in vitro</i> half-life (<i>T</i><sub>1/2</sub>) was determined by regression analysis of the percent parent disappearance vs time curve (<i>T</i><sub>1/2</sub> = 0.693/k) using Microsoft Excel. Conversion of the <i>in vitro</i><i>T</i><sub>1/2</sub> into the <i>in vitro</i> intrinsic clearance (<i>in vitro</i> CL<sub>int.</sub> in μL/min/10<sup>6</sup> cells) was performed using the following equation (mean of duplicate determinations): CL<sub>int</sub> = <i>kV</i>/<i>N</i>, where <i>V</i> = incubation volume (0.2 mL) and <i>N</i> = number of hepatocytes per well (0.1 × 10<sup>6</sup> cells)</div></div><div id="sec4_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> <i>In Vivo</i> PK Study in CD-1 Male Mice (Pharmaron, MA, USA)</h3><div class="NLM_p last">The test compounds were formulated within a 4 mg/mL solution (10% DMA, 65% PEG400, 25% saline). CD-1 mice were administered the test compound (20 mg/kg, I.P) once, and blood samples were obtained from each mouse at 0.25, 0.5, 1, 2, 4, 8, and 24 h postdose. The working solutions of 5 μL at different concentrations (2, 4, 10, 20, 100, 200, 1000, 2000 ng/mL) were added to CD-1 mouse plasma (10 μL) to generate calibration standards of 1, 2, 5, 10, 50, 100, 500, and 1000 ng/mL. Four quality control (QC) samples at 2, 5, 50, and 800 ng/mL for plasma were prepared independently of calibration curves. Standards, QC samples, and unknown samples (total volume 15 μL) were added to acetonitrile (200 μL) containing IS (2 ng/mL verapamil, and 50 ng/mL dexamethasone) for precipitation of protein. Samples were vortexed and centrifuged (4 °C, 3900 rpm, 15 min), and the supernatant was diluted 3-fold with ultrapure water. Diluted supernatant was injected into the LC/MS/MS system for quantitative analysis.</div></div><div id="sec4_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> HDAC Target Engagement (Nanosyn, CA, USA)</h3><div class="NLM_p last">The full-length human HDACs were recombinantly expressed and purified from SF9 insect cells. All compounds were serially prediluted in DMSO from a top concentration of 10 or 1 μM and introduced into the 384-well plates through an acoustic dispenser (Labcyte550) into the reaction buffer (100 mM HEPES, pH 7.5, 25 mM KCl, 0.1% BSA, 0.01% Triton X-100, and enzyme). The total concentration of DMSO was 1% in all wells. The deacetylation reaction was initiated by introduction of a FAM-labeled acetylated peptide substrate. The reaction was monitored through the change in the relative fluorescence intensity of the substrate and product peaks. The activity was determined as the product sum ratio (PSR): <i>P</i>/(<i>S</i> + <i>P</i>), where <i>P</i> = product peak height and <i>S</i> = substrate peak height. The negative control (0% inhibition in absence of inhibitor, DMSO only) and positive control (100% inhibition in absence of enzyme) were determined from <i>n</i> = 4 samples. Percent inhibition (<i>P</i><sub>inh</sub>) was determined through the equation: <i>P</i><sub>inh</sub> = (PSR<sub>0%</sub> – PSR<sub>inh</sub>)/(PSR<sub>0%</sub> – PSR<sub>100%</sub>) × 100, where PSR<sub>inh</sub> is the product sum ratio in the presence of inhibitor, PSR<sub>0%</sub> is the product sum ratio in absence of inhibitor, and PSR<sub>100%</sub> is the product sum ratio in 100% inhibition samples. Inhibition curves (<i>P</i><sub>inh</sub> vs inhibitor concentration) were fitted by a four-parameter sigmoid dose–response model using XLfit software (IDBS) to determine the IC<sub>50</sub> values.</div></div><div id="sec4_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i95" class="anchor-spacer"></div><h3 class="article-section__title" id="_i95"> Primary T-PLL Patient Sample Studies</h3><div class="NLM_p">Primary T-PLL cells were isolated from peripheral blood (PB) of 10 T-PLL patients. All patients were diagnosed according to WHO criteria based on clinical features, immunophenotyping (flow cytometry and histochemisty; including TCL1 expression), and FISH/karyotyping.<a onclick="showRef(event, 'ref71 ref72'); return false;" href="javascript:void(0);" class="ref ref71 ref72">(71,72)</a> Samples were obtained from patients under IRB-approved protocols following written informed consent according to the Declaration of Helsinki. Collection and use of patient material have been approved for research purpose by the ethics committee of the University Hospital of Cologne (EudraCT-Nr.: #2008-001421-34 and AZ11-319) and Helsinki (303/13/03/01/2011).</div><div class="NLM_p last">All patients except for one (patient 9: alemtuzumab and bendamustine) were untreated before the time of sample collection, and all the patients, except for two (Patient 8 (50%) and Patient 10 (25%)) had a > 80% T-cell fraction of their PB leukocytes. PB mononuclear cells (PBMCs) of T-PLL samples were obtained by density gradient centrifugation (Histopaque; Sigma-Aldrich). All drug screening in primary T-PLL patient samples were conducted as previously described.<a onclick="showRef(event, 'ref34 ref73'); return false;" href="javascript:void(0);" class="ref ref34 ref73">(34,73)</a></div></div><div id="sec4_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> Oncomine Gene Expression Analysis</h3><div class="NLM_p last"><i>HDAC</i> gene expression data and associated statistical analyses were extracted from the Oncomine Research Premium Edition database (Thermo Fisher, Ann Arbor, MI),<a onclick="showRef(event, 'ref46 ref59'); return false;" href="javascript:void(0);" class="ref ref46 ref59">(46,59)</a> from the data sets described in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a> using the following reporters: <i>HDAC1</i> (201209_at), <i>HDAC2</i> (201833_at), <i>HDAC3</i> (216326_s_at), <i>HDAC4</i> (204225_at), <i>HDAC5</i> (202455_at), <i>HDAC6</i> (206846_s_at), <i>HDAC9</i> (205659_at), and <i>HDAC11</i> (219847_at).</div></div><div id="sec4_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i97" class="anchor-spacer"></div><h3 class="article-section__title" id="_i97"> Statistical Testing</h3><div class="NLM_p last">Statistical analysis was performed using two-way ANOVA with Tukey’s multiple comparisons test in GraphPad Prism 6.0 (GraphPad Software Inc.). To determine differences between drug treatments in T-PLL primary samples, Wilcoxon test was performed in R with the function wilcox.test. A <i>p</i>-value of <0.05 was considered significant. <i>HDAC</i> gene expression data and associated statistical analyses were extracted from the Oncomine Research Premium Edition database (Thermo Fisher, Ann Arbor, MI) from the Durig Leukemia data set.<a onclick="showRef(event, 'ref46 ref59'); return false;" href="javascript:void(0);" class="ref ref46 ref59">(46,59)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i98"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00420" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07816" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07816" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00420?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00420</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Raw data, structures of inhibitors, additional biochemical assay results, docking results, flow cytometry, pharmacokinetic evaluations, cytotoxicity assays, and compound characterizations (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline"><i>In silico</i> docking binding pose of <b>5</b> in human HDAC6 (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><i>In silico</i> docking binding pose of <b>14</b> in human HDAC6 (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><i>In silico</i> docking binding pose of <b>14</b> in human HDAC8 (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_005.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><i>In vitro</i> IC<sub>50</sub> values for <b>1</b>–<b>14</b>, SAHA, ricolinostat, and citarinostat (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_006.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline"><i>In vitro</i> IC<sub>50</sub> values for <b>14</b> and citarinostat (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_007.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Pharmacology of <b>14</b> and citarinostat (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_008.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Cellular IC<sub>50</sub> values for <b>14</b> and citarinostat (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_009.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Pharmacokinetic parameters of <b>14</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_010.xlsx" class="ext-link">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf">jm1c00420_si_001.pdf (9.31 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_002.csv">jm1c00420_si_002.csv (1.37 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_003.pdb">jm1c00420_si_003.pdb (4.32 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_004.pdb">jm1c00420_si_004.pdb (4.32 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_005.pdb">jm1c00420_si_005.pdb (4.32 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_006.xlsx">jm1c00420_si_006.xlsx (10.35 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_007.xlsx">jm1c00420_si_007.xlsx (9.4 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_008.xlsx">jm1c00420_si_008.xlsx (10.24 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_009.xlsx">jm1c00420_si_009.xlsx (9.15 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_010.xlsx">jm1c00420_si_010.xlsx (9.59 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00420" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41524" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41524" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard Moriggl</span> - <span class="hlFld-Affiliation affiliation">Institute
of Animal Breeding and Genetics, University
of Veterinary Medicine Vienna, A-1210 Vienna, Austria</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#6b190208030a190f45060419020c0c072b1d0e1f060e0f1e0502450a08450a1f"><span class="__cf_email__" data-cfemail="60120903080112044e0d0f120907070c201605140d0504150e094e01034e0114">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Satu Mustjoki</span> - <span class="hlFld-Affiliation affiliation">Hematology
Research Unit Helsinki, Helsinki University
Hospital Comprehensive Cancer Center, Helsinki, 00029 HUS, Finland</span>; 
    <span class="hlFld-Affiliation affiliation">Translational
Immunology Research Program and Department of Clinical Chemistry and
Hematology, University of Helsinki, Helsinki, 00014 Helsinki, Finland</span>; 
    <span class="hlFld-Affiliation affiliation">iCAN Digital Precision Cancer Medicine
Flagship, 00014 Helsinki, Finland</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#91e2f0e5e4bffce4e2e5fbfefaf8d1f9f4fde2f8fffaf8bff7f8"><span class="__cf_email__" data-cfemail="63100217164d0e161017090c080a230b060f100a0d080a4d050a">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marco Herling</span> - <span class="hlFld-Affiliation affiliation">Department
of Internal Medicine, Center for Integrated Oncology Aachen-Bonn-Cologne-Duesseldorf
(CIO ABCD), University of Cologne (UoC), 50923 Cologne, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Excellence
Cluster for Cellular Stress Response and Aging-Associated Diseases
(CECAD), UoC, 50923 Cologne, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Center
for Molecular Medicine Cologne (CMMC), UoC, 50923 Cologne, Germany</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#610c0013020e4f0904130d080f0621140a4c0a0e040d0f4f0504"><span class="__cf_email__" data-cfemail="b1dcd0c3d2de9fd9d4c3ddd8dfd6f1c4da9cdaded4dddf9fd5d4">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Patrick T. Gunning</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical and Physical Sciences, University
of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Centre
for Medicinal Chemistry, University of Toronto
Mississauga, 3359 Mississauga
Road, Mississauga, Ontario L5L 1C6, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-0654-735X" title="Orcid link">https://orcid.org/0000-0003-0654-735X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#90e0f1e4e2f9f3fbbef7e5fefef9fef7d0e5e4ffe2fffee4ffbef3f1"><span class="__cf_email__" data-cfemail="403021343229232b6e27352e2e292e270035342f322f2e342f6e2321">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Krimo Toutah</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical and Physical Sciences, University
of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nabanita Nawar</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical and Physical Sciences, University
of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sanna Timonen</span> - <span class="hlFld-Affiliation affiliation">Hematology
Research Unit Helsinki, Helsinki University
Hospital Comprehensive Cancer Center, Helsinki, 00029 HUS, Finland</span>; 
    <span class="hlFld-Affiliation affiliation">Translational
Immunology Research Program and Department of Clinical Chemistry and
Hematology, University of Helsinki, Helsinki, 00014 Helsinki, Finland</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
for Molecular Medicine Finland (FIMM), HiLIFE, University of Helsinki, Helsinki, 00014 Helsinki, Finland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Helena Sorger</span> - <span class="hlFld-Affiliation affiliation">Institute
of Animal Breeding and Genetics, University
of Veterinary Medicine Vienna, A-1210 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yasir S. Raouf</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical and Physical Sciences, University
of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shazreh Bukhari</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical and Physical Sciences, University
of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jana von Jan</span> - <span class="hlFld-Affiliation affiliation">Department
of Internal Medicine, Center for Integrated Oncology Aachen-Bonn-Cologne-Duesseldorf
(CIO ABCD), University of Cologne (UoC), 50923 Cologne, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Excellence
Cluster for Cellular Stress Response and Aging-Associated Diseases
(CECAD), UoC, 50923 Cologne, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Center
for Molecular Medicine Cologne (CMMC), UoC, 50923 Cologne, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aleksandr Ianevski</span> - <span class="hlFld-Affiliation affiliation">Institute
for Molecular Medicine Finland (FIMM), HiLIFE, University of Helsinki, Helsinki, 00014 Helsinki, Finland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Justyna M. Gawel</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical and Physical Sciences, University
of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Olasunkanmi O. Olaoye</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical and Physical Sciences, University
of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mulu Geletu</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical and Physical Sciences, University
of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ayah Abdeldayem</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical and Physical Sciences, University
of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Johan Israelian</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical and Physical Sciences, University
of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tudor B. Radu</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical and Physical Sciences, University
of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Abootaleb Sedighi</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical and Physical Sciences, University
of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5974-1792" title="Orcid link">https://orcid.org/0000-0001-5974-1792</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Muzaffar N. Bhatti</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical and Physical Sciences, University
of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Muhammad Murtaza Hassan</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical and Physical Sciences, University
of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pimyupa Manaswiyoungkul</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical and Physical Sciences, University
of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Toronto, 80 St. George Street, Toronto, Ontario M5S 3H6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew E. Shouksmith</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical and Physical Sciences, University
of Toronto Mississauga, 3359 Mississauga Road, Mississauga, Ontario L5L 1C6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Heidi A. Neubauer</span> - <span class="hlFld-Affiliation affiliation">Institute
of Animal Breeding and Genetics, University
of Veterinary Medicine Vienna, A-1210 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elvin D. de Araujo</span> - <span class="hlFld-Affiliation affiliation">Centre
for Medicinal Chemistry, University of Toronto
Mississauga, 3359 Mississauga
Road, Mississauga, Ontario L5L 1C6, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tero Aittokallio</span> - <span class="hlFld-Affiliation affiliation">Institute
for Molecular Medicine Finland (FIMM), HiLIFE, University of Helsinki, Helsinki, 00014 Helsinki, Finland</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Genetics, Institute for Cancer Research, Oslo University Hospital, 0424 Oslo, Norway</span>; 
    <span class="hlFld-Affiliation affiliation">Oslo Centre
for Biostatistics and Epidemiology, University
of Oslo, 0316 Oslo, Norway</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-0886-9769" title="Orcid link">https://orcid.org/0000-0002-0886-9769</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oliver H. Krämer</span> - <span class="hlFld-Affiliation affiliation">Department
of Toxicology, University Medical Center, 55131 Mainz, Germany</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>K.T. and N.N. contributed equally. K.T., N.N., R.M., M.H., and P.T.G. conceptualized the study and designed experiments. K.T., N.N., Y.S.R., M.N.B., A.A., O.O.O., and A.E.S. synthesized/characterized the compounds. N.N., H.S., S.B., J.M.G., M.G., H.A.N., and R.M. carried out in cellulo biological evaluation and analysis. J.I., P.M., O.O.O., T.B.R., A.S., M.M.H., and E.D.A. performed biophysical evaluation and analysis. Y.S.R. performed in silico studies and analysis. H.A.N. performed Oncomine analysis. J.V.J., A.I., S.T., T.A., S.M., and M.H. performed synergy studies and analysis. N.N., E.D.A., M.H., O.H.K., H.A.N., R.M., and P.T.G. wrote the manuscript with contributions from all authors.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i100">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33115" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33115" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors would like to thank Ruth Villalonga for her contributions with cytotoxicity assays using NHF, pPF and HUVEC. They would also like to thank Andrew Sedmihradsky and family, who have raised funds, awareness for DMD and generated support for Max’s Big Fellowship. Funding: This work was supported by the EU consortia ERAPerMed “JAKSTAT-TARGET” (S.M., H.A.N., M.H., P.T.G.) and the “ERANETPLL” (M.H., R.M.). S.M. was funded by European Research Council (M-IMM project), Academy of Finland, Sigrid Juselius Foundation and the Cancer Foundation Finland. T.A. was funded by the Academy of Finland (grants 310507, 313267, 326238), Cancer Society of Finland, Sigrid Jusélius Foundation and Helse Sør-Øst. R.M., H.A.N., and H.S. were supported by the Austrian Science Fund (FWF) (SFB-F04707, SFB-F06105). R.M. and H.A.N. were also generously supported by a private cancer metabolism grant donation from Liechtenstein. P.T.G is supported by research grants from NSERC (RGPIN-2014-05767), CIHR (MOP-130424, MOP-137036), Canada Research Chair (950-232042), Canadian Cancer Society (703963), Canadian Breast Cancer Foundation (705456), Leukemia and Lymphoma Society of Canada and infrastructure grants from CFI (33536) and the Ontario Research Fund (34876). O.O.O. is supported by an OGS Fellowship. Y.S.R and N.N are supported by grants from Jesse’s Journey.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">ABT-199</td><td class="NLM_def"><p class="first last">venetoclax</p></td></tr><tr><td class="NLM_term">AcOH</td><td class="NLM_def"><p class="first last">acetic acid</p></td></tr><tr><td class="NLM_term">MeCN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">ADME</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, and excretion</p></td></tr><tr><td class="NLM_term">AITL</td><td class="NLM_def"><p class="first last">angioimmunoblastic T-cell lymphoma</p></td></tr><tr><td class="NLM_term">ALCL</td><td class="NLM_def"><p class="first last">Anaplastic large cell lymphoma</p></td></tr><tr><td class="NLM_term">ALK</td><td class="NLM_def"><p class="first last">anaplastic lymphoma kinase</p></td></tr><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myeloid leukemia</p></td></tr><tr><td class="NLM_term">ATM</td><td class="NLM_def"><p class="first last">ataxia telangiectasia mutated</p></td></tr><tr><td class="NLM_term">AUC</td><td class="NLM_def"><p class="first last">area under the curve</p></td></tr><tr><td class="NLM_term">BCL</td><td class="NLM_def"><p class="first last">B-cell lymphoma</p></td></tr><tr><td class="NLM_term">Boc</td><td class="NLM_def"><p class="first last"><i>tert</i>-butoxycarbonyl</p></td></tr><tr><td class="NLM_term">FAM</td><td class="NLM_def"><p class="first last">carboxyfluorescein</p></td></tr><tr><td class="NLM_term">CCLE</td><td class="NLM_def"><p class="first last">Cancer Cell Line Encyclopedia</p></td></tr><tr><td class="NLM_term">CoA</td><td class="NLM_def"><p class="first last">coenzyme A</p></td></tr><tr><td class="NLM_term"><i>C</i><sub>max</sub></td><td class="NLM_def"><p class="first last">maximum concentration the drug achieves</p></td></tr><tr><td class="NLM_term">Cs<sub>2</sub>CO<sub>3</sub></td><td class="NLM_def"><p class="first last">cesium carbonate</p></td></tr><tr><td class="NLM_term">CTCL</td><td class="NLM_def"><p class="first last">cutaneous T-cell lymphomas</p></td></tr><tr><td class="NLM_term">DCE</td><td class="NLM_def"><p class="first last">dichloroethane</p></td></tr><tr><td class="NLM_term">DCM and CH<sub>2</sub>Cl<sub>2</sub></td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">DNA</td><td class="NLM_def"><p class="first last">deoxyribonucleic acid</p></td></tr><tr><td class="NLM_term"><i>i</i>Pr<sub>2</sub>NEt</td><td class="NLM_def"><p class="first last">diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DSRT</td><td class="NLM_def"><p class="first last">drug sensitivity and resistance testing</p></td></tr><tr><td class="NLM_term">DSS</td><td class="NLM_def"><p class="first last">drug sensitivity score, EMSA, electrophoretic mobility shift assay</p></td></tr><tr><td class="NLM_term">FACS</td><td class="NLM_def"><p class="first last">fluorescence-activated cell sorting</p></td></tr><tr><td class="NLM_term">FITC</td><td class="NLM_def"><p class="first last">fluorescein isothiocyanate</p></td></tr><tr><td class="NLM_term">FDA</td><td class="NLM_def"><p class="first last">U.S. Food and Drug administration</p></td></tr><tr><td class="NLM_term">GSH</td><td class="NLM_def"><p class="first last">reduced glutathione</p></td></tr><tr><td class="NLM_term"><i>t</i><sub>1/2</sub></td><td class="NLM_def"><p class="first last">half-life</p></td></tr><tr><td class="NLM_term">h</td><td class="NLM_def"><p class="first last">hour</p></td></tr><tr><td class="NLM_term">HCl</td><td class="NLM_def"><p class="first last">hydrochloric acid</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">HRMS</td><td class="NLM_def"><p class="first last">high-resolution mass spectrometry</p></td></tr><tr><td class="NLM_term">HPOB</td><td class="NLM_def"><p class="first last"><i>N</i>-hydroxy-4-(2-[(2-hydroxyethyl)(phenyl)amino]-2-oxoethyl)benzamide</p></td></tr><tr><td class="NLM_term">HSP</td><td class="NLM_def"><p class="first last">heat shock protein</p></td></tr><tr><td class="NLM_term">HUVEC</td><td class="NLM_def"><p class="first last">normal human umbilical vein endothelial cells</p></td></tr><tr><td class="NLM_term">HIF</td><td class="NLM_def"><p class="first last">hypoxia-inducible factors</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">half maximal inhibitory concentration</p></td></tr><tr><td class="NLM_term">IS</td><td class="NLM_def"><p class="first last">internal standard</p></td></tr><tr><td class="NLM_term">IP</td><td class="NLM_def"><p class="first last">intraperitoneal</p></td></tr><tr><td class="NLM_term">JAK</td><td class="NLM_def"><p class="first last">Janus-activated kinase</p></td></tr><tr><td class="NLM_term">LC/MS</td><td class="NLM_def"><p class="first last">liquid chromatography/mass spectrometry</p></td></tr><tr><td class="NLM_term">LRMS</td><td class="NLM_def"><p class="first last">low-resolution mass spectrometry</p></td></tr><tr><td class="NLM_term">MaTCL</td><td class="NLM_def"><p class="first last">mature T-cell leukemias and lymphomas</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">min</td><td class="NLM_def"><p class="first last">minutes</p></td></tr><tr><td class="NLM_term">NAD<sup>+</sup></td><td class="NLM_def"><p class="first last">nicotinamide adenine dinucleotide</p></td></tr><tr><td class="NLM_term">NHF</td><td class="NLM_def"><p class="first last">normal human fibroblasts</p></td></tr><tr><td class="NLM_term">NK</td><td class="NLM_def"><p class="first last">natural killer</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">(COCl)<sub>2</sub></td><td class="NLM_def"><p class="first last">oxalyl chloride</p></td></tr><tr><td class="NLM_term">p300/CBP</td><td class="NLM_def"><p class="first last">EA1 binding protein 300/CREB-binding protein</p></td></tr><tr><td class="NLM_term">Pd(OAc)<sub>2</sub></td><td class="NLM_def"><p class="first last">palladium acetate</p></td></tr><tr><td class="NLM_term">Pd/C</td><td class="NLM_def"><p class="first last">palladium on carbon</p></td></tr><tr><td class="NLM_term">PAMPA</td><td class="NLM_def"><p class="first last">parallel artificial membrane permeability assay</p></td></tr><tr><td class="NLM_term">PBMCs</td><td class="NLM_def"><p class="first last">peripheral blood mononuclear cells</p></td></tr><tr><td class="NLM_term"><i>P</i><sub>e</sub></td><td class="NLM_def"><p class="first last">effective permeability</p></td></tr><tr><td class="NLM_term">PFBSCl</td><td class="NLM_def"><p class="first last">pentafluorobenzyl sulfonyl chloride</p></td></tr><tr><td class="NLM_term">PTCL</td><td class="NLM_def"><p class="first last">peripheral T-cell lymphomas</p></td></tr><tr><td class="NLM_term">PTCL-NOS</td><td class="NLM_def"><p class="first last">peripheral T-cell lymphoma, not otherwise specified</p></td></tr><tr><td class="NLM_term">PG</td><td class="NLM_def"><p class="first last">protecting group</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetic</p></td></tr><tr><td class="NLM_term">P.O.</td><td class="NLM_def"><p class="first last">oral administration</p></td></tr><tr><td class="NLM_term">pPF</td><td class="NLM_def"><p class="first last">primary pooled fibroblasts</p></td></tr><tr><td class="NLM_term">rpm</td><td class="NLM_def"><p class="first last">revolutions per minute</p></td></tr><tr><td class="NLM_term">RT</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term">Na(AcO)<sub>3</sub>BH</td><td class="NLM_def"><p class="first last">SCT, stem cell transplantation</p></td></tr><tr><td class="NLM_term">sDSS</td><td class="NLM_def"><p class="first last">selective drug sensitivity score, sodium triacetoxyborohydride</p></td></tr><tr><td class="NLM_term">SI</td><td class="NLM_def"><p class="first last">Supporting Information</p></td></tr><tr><td class="NLM_term">T-ALL</td><td class="NLM_def"><p class="first last">T-cell acute lymphocytic leukemia</p></td></tr><tr><td class="NLM_term">TC50</td><td class="NLM_def"><p class="first last">toxic concentration in 50% of PBMC population</p></td></tr><tr><td class="NLM_term">TCL1A</td><td class="NLM_def"><p class="first last">T-cell leukemia 1A</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">Et<sub>3</sub>N</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SAHA</td><td class="NLM_def"><p class="first last">suberanilohydroxamic acid</p></td></tr><tr><td class="NLM_term">Sir2</td><td class="NLM_def"><p class="first last">sirtuin</p></td></tr><tr><td class="NLM_term">STAT</td><td class="NLM_def"><p class="first last">signal transducer and activator of transcription factor</p></td></tr><tr><td class="NLM_term">T-PLL</td><td class="NLM_def"><p class="first last">T-cell prolymphocytic leukemia</p></td></tr><tr><td class="NLM_term">ZBG</td><td class="NLM_def"><p class="first last">zinc-binding group</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i102">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67414" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67414" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 73 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lund, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Lohuizen, M.</span></span> <span> </span><span class="NLM_article-title">Epigenetics and Cancer</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2315</span>– <span class="NLM_lpage">2335</span>, <span class="refDoi"> DOI: 10.1101/gad.1232504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1101%2Fgad.1232504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=15466484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2cXot1yrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2004&pages=2315-2335&author=A.+H.+Lundauthor=M.+Van+Lohuizen&title=Epigenetics+and+Cancer&doi=10.1101%2Fgad.1232504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetics and cancer</span></div><div class="casAuthors">Lund, Anders H.; van Lohuizen, Maarten</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2315-2335</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  Epigenetic mechanisms act to change the accessibility of chromatin to transcriptional regulation locally and globally via modifications of the DNA and by modification or rearrangement of nucleosomes.  Epigenetic gene regulation collaborates with genetic alterations in cancer development.  This is evident from every aspect of tumor biol. including cell growth and differentiation, cell cycle control, DNA repair, angiogenesis, migration, and evasion of host immunosurveillance.  In contrast to genetic cancer causes, the possibility of reversing epigenetic codes may provide new targets for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLkbbp11uuWrVg90H21EOLACvtfcHk0liSSCqq7EcxUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXot1yrtL8%253D&md5=7e9499e9bc7e5638f2d8e4ff80528343</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1101%2Fgad.1232504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1232504%26sid%3Dliteratum%253Aachs%26aulast%3DLund%26aufirst%3DA.%2BH.%26aulast%3DVan%2BLohuizen%26aufirst%3DM.%26atitle%3DEpigenetics%2520and%2520Cancer%26jtitle%3DGenes%2520Dev.%26date%3D2004%26volume%3D18%26spage%3D2315%26epage%3D2335%26doi%3D10.1101%2Fgad.1232504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eckschlager, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stiborova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hrabeta, J.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylase Inhibitors as Anticancer Drugs</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1414</span>, <span class="refDoi"> DOI: 10.3390/ijms18071414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.3390%2Fijms18071414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=28671573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVGgt73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=1414&author=T.+Eckschlagerauthor=J.+Plchauthor=M.+Stiborovaauthor=J.+Hrabeta&title=Histone+Deacetylase+Inhibitors+as+Anticancer+Drugs&doi=10.3390%2Fijms18071414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase inhibitors as anticancer drugs</span></div><div class="casAuthors">Eckschlager, Tomas; Plch, Johana; Stiborova, Marie; Hrabeta, Jan</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1414/1-1414/25</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Carcinogenesis cannot be explained only by genetic alterations, but also involves epigenetic processes.  Modification of histones by acetylation plays a key role in epigenetic regulation of gene expression and is controlled by the balance between histone deacetylases (HDAC) and histone acetyltransferases (HAT).  HDAC inhibitors induce cancer cell cycle arrest, differentiation and cell death, reduce angiogenesis and modulate immune response.  Mechanisms of anticancer effects of HDAC inhibitors are not uniform; they may be different and depend on the cancer type, HDAC inhibitors, doses, etc.  HDAC inhibitors seem to be promising anti-cancer drugs particularly in the combination with other anti-cancer drugs and/or radiotherapy.  HDAC inhibitors vorinostat, romidepsin and belinostat have been approved for some T-cell lymphoma and panobinostat for multiple myeloma.  Other HDAC inhibitors are in clin. trials for the treatment of hematol. and solid malignancies.  The results of such studies are promising but further larger studies are needed.  Because of the reversibility of epigenetic changes during cancer development, the potency of epigenetic therapies seems to be of great importance.  Here, we summarize the data on different classes of HDAC inhibitors, mechanisms of their actions and discuss novel results of preclin. and clin. studies, including the combination with other therapeutic modalities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPNVhoLBAHnrVg90H21EOLACvtfcHk0liSSCqq7EcxUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVGgt73P&md5=6018c7709a09548214bfe34bd91107a9</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3390%2Fijms18071414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms18071414%26sid%3Dliteratum%253Aachs%26aulast%3DEckschlager%26aufirst%3DT.%26aulast%3DPlch%26aufirst%3DJ.%26aulast%3DStiborova%26aufirst%3DM.%26aulast%3DHrabeta%26aufirst%3DJ.%26atitle%3DHistone%2520Deacetylase%2520Inhibitors%2520as%2520Anticancer%2520Drugs%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2017%26volume%3D18%26spage%3D1414%26doi%3D10.3390%2Fijms18071414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, S.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylases as Targets for Treatment of Multiple Diseases</span>. <i>Clin. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">651</span>– <span class="NLM_lpage">662</span>, <span class="refDoi"> DOI: 10.1042/CS20120504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1042%2FCS20120504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=23414309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC3sXislOmt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2013&pages=651-662&author=J.+Tangauthor=H.+Yanauthor=S.+Zhuang&title=Histone+Deacetylases+as+Targets+for+Treatment+of+Multiple+Diseases&doi=10.1042%2FCS20120504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylases as targets for treatment of multiple diseases</span></div><div class="casAuthors">Tang, Jinhua; Yan, Haidong; Zhuang, Shougang</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Science</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">651-662</span>CODEN:
                <span class="NLM_cas:coden">CSCIAE</span>;
        ISSN:<span class="NLM_cas:issn">1470-8736</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  HDACs (histone deacetylases) are a group of enzymes that deacetylate histones as well as non-histone proteins.  They are known as modulators of gene transcription and are assocd. with proliferation and differentiation of a variety of cell types and the pathogenesis of some diseases.  Recently, HDACs have come to be considered crucial targets in various diseases, including cancer, interstitial fibrosis, autoimmune and inflammatory diseases, and metabolic disorders.  Pharmacol. inhibitors of HDACs have been used or tested to treat those diseases.  In the present review, we will examine the application of HDAC inhibitors in a variety of diseases with the focus on their effects of anti-cancer, fibrosis, anti-inflammatory, immunomodulatory activity and regulating metabolic disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP-M9OvYul7bVg90H21EOLACvtfcHk0lgEzE7ANBggXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXislOmt7c%253D&md5=cc0d32170884436c10d5010f7342c6bd</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1042%2FCS20120504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FCS20120504%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DZhuang%26aufirst%3DS.%26atitle%3DHistone%2520Deacetylases%2520as%2520Targets%2520for%2520Treatment%2520of%2520Multiple%2520Diseases%26jtitle%3DClin.%2520Sci.%26date%3D2013%26volume%3D124%26spage%3D651%26epage%3D662%26doi%3D10.1042%2FCS20120504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pandolfi, P. P.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylases and Transcriptional Therapy with Their Inhibitors</span>. <i>Cancer Chemother. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">S17</span>, <span class="refDoi"> DOI: 10.1007/s002800100322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1007%2Fs002800100322" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=11587360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD3MXot1GisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2001&pages=S17&author=P.+P.+Pandolfi&title=Histone+Deacetylases+and+Transcriptional+Therapy+with+Their+Inhibitors&doi=10.1007%2Fs002800100322"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylases and transcriptional therapy with their inhibitors</span></div><div class="casAuthors">Pandolfi, Pier Paolo</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Chemotherapy and Pharmacology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">S17-S19</span>CODEN:
                <span class="NLM_cas:coden">CCPHDZ</span>;
        ISSN:<span class="NLM_cas:issn">0344-5704</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag</span>)
        </div><div class="casAbstract">A review.  Acute promyelocytic leukemia (APL) is characterized by the expansion of malignant myeloid cells blocked at the promyelocytic stage of hemopoietic development and is invariably assocd. with reciprocal chromosomal translocations involving the retinoic acid receptor α (RARα) gene on chromosome 17.  RARα variably fuses to PML, PLZF, NPM, NuMA, and Stat5B genes (X genes/proteins).  These translocations are balanced and reciprocal, thus leading to the generation of X-RARα and RARα-X fusion genes of which the products coexist in the APL blast.  The invariable involvement in these translocations of RARα, a prototypical transcription factor, makes APL a compelling example of aberrant transcriptional mechanisms in the etiopathogenesis of cancer.  This paper focuses on the recent progress in defining the mol. mechanisms underlying APL pathogenesis and addresses how this new understanding has allowed the proposal and development of novel therapeutic strategies with compds. such as histone deacetylase inhibitors and inorg. arsenicals such as As2O3 which are currently being tested in murine leukemia models as well as in human APL patients.  In particular, the crucial role played by the aberrant transcriptional activities of X-RARα and RARα-X fusion proteins in APL pathogenesis is discussed by reviewing the relevant therapeutic implications resulting from this anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQNb7pqi3oCrVg90H21EOLACvtfcHk0lgEzE7ANBggXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXot1GisLc%253D&md5=56665a664a42902528ff33cbbbe5b570</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2Fs002800100322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs002800100322%26sid%3Dliteratum%253Aachs%26aulast%3DPandolfi%26aufirst%3DP.%2BP.%26atitle%3DHistone%2520Deacetylases%2520and%2520Transcriptional%2520Therapy%2520with%2520Their%2520Inhibitors%26jtitle%3DCancer%2520Chemother.%2520Pharmacol.%26date%3D2001%26volume%3D48%26spage%3DS17%26doi%3D10.1007%2Fs002800100322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thiagalingam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mineva, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiagalingam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponte, J. F.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylases: Unique Players in Shaping the Epigenetic Histone Code</span>. <i>Ann. N. Y. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>983</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1111/j.1749-6632.2003.tb05964.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1111%2Fj.1749-6632.2003.tb05964.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=12724214" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjt1yrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=983&publication_year=2003&pages=84-100&author=S.+Thiagalingamauthor=K.+H.+Chengauthor=H.+J.+Leeauthor=N.+Minevaauthor=A.+Thiagalingamauthor=J.+F.+Ponte&title=Histone+Deacetylases%3A+Unique+Players+in+Shaping+the+Epigenetic+Histone+Code&doi=10.1111%2Fj.1749-6632.2003.tb05964.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylases: unique players in shaping the epigenetic histone code</span></div><div class="casAuthors">Thiagalingam, Sam; Cheng, Kuang-Hung; Lee, Hyunjoo J.; Mineva, Nora; Thiagalingam, Arunthathi; Ponte, Jose F.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the New York Academy of Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">983</span>
        (<span class="NLM_cas:issue">Epigenetics in Cancer Prevention</span>),
    <span class="NLM_cas:pages">84-100</span>CODEN:
                <span class="NLM_cas:coden">ANYAA9</span>;
        ISSN:<span class="NLM_cas:issn">0077-8923</span>.
    
            (<span class="NLM_cas:orgname">New York Academy of Sciences</span>)
        </div><div class="casAbstract">A review.  The epigenome is defined by DNA methylation patterns and the assocd. posttranslational modifications of histones.  This histone code dets. the expression status of individual genes dependent upon their localization on the chromatin.  The silencing of gene expression is assocd. with deacetylated histones, which are often found to be assocd. with regions of DNA methylation as well as methylation at the lysine 4 residue of histone 3.  In contrast, the activation of gene expression is assocd. with acetylated histones and methylation at the lysine 9 residue of histone 3.  The histone deacetylases play a major role in keeping the balance between the acetylated and deacetylated states of chromatin.  Histone deacetylases (HDACs) are divided into three classes: class I HDACs (HDACs 1, 2, 3, and 8) are similar to the yeast RPD3 protein and localize to the nucleus; class II HDACs (HDACs 4, 5, 6, 7, 9, and 10) are homologous to the yeast HDA1 protein and are found in both the nucleus and cytoplasm; and class III HDACs form a structurally distinct class of NAD-dependent enzymes that are similar to the yeast SIR2 proteins.  Since inappropriate silencing of crit. genes can result in one or both hits of tumor suppressor gene (TSG) inactivation in cancer, theor. the reactivation of affected TSGs could have an enormous therapeutic value in preventing and treating cancer.  Indeed, several HDAC inhibitors are currently being developed and tested for their potency in cancer chemotherapy.  Importantly, these agents are also potentially applicable to chemoprevention if their toxicity can be minimized.  Despite the toxic side effects and lack of specificity of some of the inhibitors, progress is being made.  With the elucidation of the structures, functions and modes of action of HDACs, finding agents that may be targeted to specific HDACs and potentially reactivate expression of only a defined set of affected genes in cancer will be more attainable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJPAGtCnCINbVg90H21EOLACvtfcHk0lio7YZ5el92nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjt1yrs7g%253D&md5=e288e2f402f1cab9f1d51ddffc0ac87c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1111%2Fj.1749-6632.2003.tb05964.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1749-6632.2003.tb05964.x%26sid%3Dliteratum%253Aachs%26aulast%3DThiagalingam%26aufirst%3DS.%26aulast%3DCheng%26aufirst%3DK.%2BH.%26aulast%3DLee%26aufirst%3DH.%2BJ.%26aulast%3DMineva%26aufirst%3DN.%26aulast%3DThiagalingam%26aufirst%3DA.%26aulast%3DPonte%26aufirst%3DJ.%2BF.%26atitle%3DHistone%2520Deacetylases%253A%2520Unique%2520Players%2520in%2520Shaping%2520the%2520Epigenetic%2520Histone%2520Code%26jtitle%3DAnn.%2520N.%2520Y.%2520Acad.%2520Sci.%26date%3D2003%26volume%3D983%26spage%3D84%26epage%3D100%26doi%3D10.1111%2Fj.1749-6632.2003.tb05964.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haberland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, E. N.</span></span> <span> </span><span class="NLM_article-title">The Many Roles of Histone Deacetylases in Development and Physiology: Implications for Disease and Therapy</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1038/nrg2485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fnrg2485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=19065135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFWis7bI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=32-42&author=M.+Haberlandauthor=R.+L.+Montgomeryauthor=E.+N.+Olson&title=The+Many+Roles+of+Histone+Deacetylases+in+Development+and+Physiology%3A+Implications+for+Disease+and+Therapy&doi=10.1038%2Fnrg2485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The many roles of histone deacetylases in development and physiology: implications for disease and therapy</span></div><div class="casAuthors">Haberland, Michael; Montgomery, Rusty L.; Olson, Eric N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-42</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The expression of many histone deacetylase (HDAC) isoforms in eukaryotic cells raises questions regarding their specificity and the programs of gene expression that they control.  HDAC knockout mice are a powerful tool for addressing these questions and have revealed that individual HDACs have specific functions in development and disease.  Histone deacetylases (HDACs) are part of a vast family of enzymes that have crucial roles in numerous biol. processes, largely through their repressive influence on transcription.  The expression of many HDAC isoforms in eukaryotic cells raises questions about their possible specificity or redundancy, and whether they control global or specific programs of gene expression.  Recent analyses of HDAC knockout mice have revealed highly specific functions of individual HDACs in development and disease.  Mutant mice lacking individual HDACs are a powerful tool for defining the functions of HDACs in vivo and the mol. targets of HDAC inhibitors in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMWcIo3mrsM7Vg90H21EOLACvtfcHk0lio7YZ5el92nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFWis7bI&md5=0678d8655e86994cd4f0174c1e9d0eb1</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrg2485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg2485%26sid%3Dliteratum%253Aachs%26aulast%3DHaberland%26aufirst%3DM.%26aulast%3DMontgomery%26aufirst%3DR.%2BL.%26aulast%3DOlson%26aufirst%3DE.%2BN.%26atitle%3DThe%2520Many%2520Roles%2520of%2520Histone%2520Deacetylases%2520in%2520Development%2520and%2520Physiology%253A%2520Implications%2520for%2520Disease%2520and%2520Therapy%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2009%26volume%3D10%26spage%3D32%26epage%3D42%26doi%3D10.1038%2Fnrg2485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allfrey, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faulkner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirsky, A. E.</span></span> <span> </span><span class="NLM_article-title">Acetylation and Methylation of Histones and Their Possible Role in the Regulation of RNA</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1964</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">786</span>– <span class="NLM_lpage">794</span>, <span class="refDoi"> DOI: 10.1073/pnas.51.5.786</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1073%2Fpnas.51.5.786" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=14172992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADyaF2cXktlekurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1964&pages=786-794&author=V.+G.+Allfreyauthor=R.+Faulknerauthor=A.+E.+Mirsky&title=Acetylation+and+Methylation+of+Histones+and+Their+Possible+Role+in+the+Regulation+of+RNA&doi=10.1073%2Fpnas.51.5.786"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Acetylation and methylation of histones and their possible role in the regulation of ribonucleic acid (RNA) synthesis</span></div><div class="casAuthors">Allfrey, V. G.; Faulkner, R.; Mirsky, A. E.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1964</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">786-94</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Isolated calf thymus nuclei was incubated with 14C-labeled Na acetate.  Evidence that acetate was incorporated as acetyl groups attached to histones included: (1) following chromatography the label was eluted together with basic proteins; (2) acetyl-labeled histones were sepd. by electrophoresis; (3) acetate-14C was not removed from histones by org. solvents; (4) over 75% of the label remained after histones were treated with trichloroacetic acid.  Most extensive acetate uptake occurred with arginine-rich histones.  Puromycin did not inhibit the acetylation, indicating histones were modified by acetylation after completion of the polypeptide chain.  Acetylation lowered the histone effectiveness as inhibitor of the RNA polymerase reaction, and a dynamic and reversible mechanism was suggested for activation as well as repression of RNA synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpL5QIm_JdO1bVg90H21EOLACvtfcHk0lio7YZ5el92nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF2cXktlekurg%253D&md5=7a80f7a8a12636d4bc47930ee5eff217</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.51.5.786&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.51.5.786%26sid%3Dliteratum%253Aachs%26aulast%3DAllfrey%26aufirst%3DV.%2BG.%26aulast%3DFaulkner%26aufirst%3DR.%26aulast%3DMirsky%26aufirst%3DA.%2BE.%26atitle%3DAcetylation%2520and%2520Methylation%2520of%2520Histones%2520and%2520Their%2520Possible%2520Role%2520in%2520the%2520Regulation%2520of%2520RNA%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1964%26volume%3D51%26spage%3D786%26epage%3D794%26doi%3D10.1073%2Fpnas.51.5.786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, M.</span></span> <span> </span><span class="NLM_article-title">Epigenome-Based Personalized Medicine in Human Cancer</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">119</span>– <span class="NLM_lpage">133</span>, <span class="refDoi"> DOI: 10.2217/epi.15.84</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.2217%2Fepi.15.84" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=26344672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC28Xos1aqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=119-133&author=W.+Yanauthor=J.+G.+Hermanauthor=M.+Guo&title=Epigenome-Based+Personalized+Medicine+in+Human+Cancer&doi=10.2217%2Fepi.15.84"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenome-based personalized medicine in human cancer</span></div><div class="casAuthors">Yan, Wenji; Herman, James G.; Guo, Mingzhou</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">119-133</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">Cancer genome sequencing has created an opportunity for precision medicine.  Thus far, genetic alterations can only be used to guide treatment for small subsets of certain cancer types with these key alterations.  Similar to mutations, epigenetic events are equally suitable for personalized medicine.  DNA methylation alterations have been used to identify tumor-specific drug responsive markers.  Methylation of MGMT sensitizes gliomas to alkylating agents is an example of epigenetic personalized medicine.  Recent studies have revealed that 5-azacytidine and decitabine show activity in myelodysplasia, lung and other cancers.  There are currently at least 20 kinds of histone deacetylase inhibitors in clin. testing.  Inhibitors targeting other epigenetic regulators are being clin. tested, such as EZH2 inhibitor EPZ-6438.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_za-KvY9spLVg90H21EOLACvtfcHk0lh83GggDL2Q3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xos1aqtA%253D%253D&md5=f872718075e1a9b7f7788965fe143c53</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.2217%2Fepi.15.84&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.15.84%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DW.%26aulast%3DHerman%26aufirst%3DJ.%2BG.%26aulast%3DGuo%26aufirst%3DM.%26atitle%3DEpigenome-Based%2520Personalized%2520Medicine%2520in%2520Human%2520Cancer%26jtitle%3DEpigenomics%26date%3D2016%26volume%3D8%26spage%3D119%26epage%3D133%26doi%3D10.2217%2Fepi.15.84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lombardi, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowling, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Structure, Mechanism, and Inhibition of Histone Deacetylases and Related Metalloenzymes</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">735</span>– <span class="NLM_lpage">743</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2011.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1016%2Fj.sbi.2011.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=21872466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFOhurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=735-743&author=P.+M.+Lombardiauthor=K.+E.+Coleauthor=D.+P.+Dowlingauthor=D.+W.+Christianson&title=Structure%2C+Mechanism%2C+and+Inhibition+of+Histone+Deacetylases+and+Related+Metalloenzymes&doi=10.1016%2Fj.sbi.2011.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structure, mechanism, and inhibition of histone deacetylases and related metalloenzymes</span></div><div class="casAuthors">Lombardi, Patrick M.; Cole, Kathryn E.; Dowling, Daniel P.; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">735-743</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Metal-dependent histone deacetylases (HDACs) catalyze the hydrolysis of acetyl-L-lysine side-chains in histone and nonhistone proteins to yield L-lysine and acetate.  This chem. plays a crit. role in the regulation of numerous biol. processes.  Aberrant HDAC activity is implicated in various diseases, and HDACs are validated targets for drug design.  Two HDAC inhibitors are currently approved for cancer chemotherapy, and other inhibitors are in clin. trials.  To date, x-ray crystal structures are available for 4 human HDACs (2, 4, 7, and 8) and 3 HDAC-related deacetylases from bacteria [histone deacetylase-like protein (HDLP), histone deacetylase-like amidohydrolase (HDAH), and acetylpolyamine amidohydrolase (APAH)].  Structural comparisons among these enzymes reveal a conserved constellation of active site residues, suggesting a common mechanism for the metal-dependent hydrolysis of acetylated substrates.  Structural analyses of HDACs and HDAC-related deacetylases guide the design of tight-binding inhibitors, and future prospects for developing isoenzyme-specific inhibitors are quite promising.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2SnZUQ2FtTrVg90H21EOLACvtfcHk0lh83GggDL2Q3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFOhurrJ&md5=edab41900a9f233d7ff30c507cfcce66</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2011.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2011.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DLombardi%26aufirst%3DP.%2BM.%26aulast%3DCole%26aufirst%3DK.%2BE.%26aulast%3DDowling%26aufirst%3DD.%2BP.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DStructure%252C%2520Mechanism%252C%2520and%2520Inhibition%2520of%2520Histone%2520Deacetylases%2520and%2520Related%2520Metalloenzymes%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2011%26volume%3D21%26spage%3D735%26epage%3D743%26doi%3D10.1016%2Fj.sbi.2011.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feinberg, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tycko, B.</span></span> <span> </span><span class="NLM_article-title">The History of Cancer Epigenetics</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">153</span>, <span class="refDoi"> DOI: 10.1038/nrc1279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fnrc1279" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=14732866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2cXks1CktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=143-153&author=A.+P.+Feinbergauthor=B.+Tycko&title=The+History+of+Cancer+Epigenetics&doi=10.1038%2Fnrc1279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Timeline: The history of cancer epigenetics</span></div><div class="casAuthors">Feinberg, Andrew P.; Tycko, Benjamin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">143-153</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Since its discovery in 1983, the epigenetics of human cancer has been in the shadows of human cancer genetics.  But this area has become increasingly visible with a growing understanding of specific epigenetic mechanisms and their role in cancer, including hypomethylation, hypermethylation, loss of imprinting and chromatin modification.  This timeline traces the field from its conception to the present day.  It also addresses the genetic basis of epigenetic changes - an emerging area that promises to unite cancer genetics and epigenetics, and might serve as a model for understanding the epigenetic basis of human disease more generally.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIbizd6X65lrVg90H21EOLACvtfcHk0lioEbmapZqlZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXks1CktA%253D%253D&md5=1e1c0ccf32e48422fd1842593b5062be</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrc1279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1279%26sid%3Dliteratum%253Aachs%26aulast%3DFeinberg%26aufirst%3DA.%2BP.%26aulast%3DTycko%26aufirst%3DB.%26atitle%3DThe%2520History%2520of%2520Cancer%2520Epigenetics%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2004%26volume%3D4%26spage%3D143%26epage%3D153%26doi%3D10.1038%2Fnrc1279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tran, A. D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmo, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salam, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkelstein, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabarriti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xenias, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazitschek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawaguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheetz, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulinski, J. C.</span></span> <span> </span><span class="NLM_article-title">HDAC6 Deacetylation of Tubulin Modulates Dynamics of Cellular Adhesions</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">1469</span>– <span class="NLM_lpage">1479</span>, <span class="refDoi"> DOI: 10.1242/jcs.03431</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1242%2Fjcs.03431" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=17389687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlsFSjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2007&pages=1469-1479&author=A.+D.+A.+Tranauthor=T.+P.+Marmoauthor=A.+A.+Salamauthor=S.+Cheauthor=E.+Finkelsteinauthor=R.+Kabarritiauthor=H.+S.+Xeniasauthor=R.+Mazitschekauthor=C.+Hubbertauthor=Y.+Kawaguchiauthor=M.+P.+Sheetzauthor=T.+Yaoauthor=J.+C.+Bulinski&title=HDAC6+Deacetylation+of+Tubulin+Modulates+Dynamics+of+Cellular+Adhesions&doi=10.1242%2Fjcs.03431"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">HDAC6 deacetylation of tubulin modulates dynamics of cellular adhesions</span></div><div class="casAuthors">Tran, Andy Dong-Anh; Marmo, Timothy P.; Salam, Ambar A.; Che, Sally; Finkelstein, Erik; Kabarriti, Rafi; Xenias, Harry S.; Mazitschek, Ralph; Hubbert, Charlotte; Kawaguchi, Yoshiharu; Sheetz, Michael P.; Yao, Tso-Pang; Bulinski, J. Chloe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1469-1479</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Genetic or pharmacol. alteration of the activity of histone deacetylase 6 (HDAC6) induces a parallel alteration in cell migration.  Using tubacin to block deacetylation of α-tubulin, and not other HDAC6 substrates, yielded a motility redn. equiv. to agents that block all NAD-independent HDACs.  Accordingly, the authors investigated how the failure to deacetylate tubulin contributes to decreased motility in HDAC6-inhibited cells.  Testing the hypothesis that motility is reduced because cellular adhesion is altered, it was found that inhibiting HDAC6 activity toward tubulin rapidly increased the total adhesion area.  Next, the authors investigated the mechanism of the adhesion area increase.  Formation of adhesions proceeded normally and cell spreading was more rapid in the absence of active HDAC6; however, photobleaching assays and adhesion breakdown showed that adhesion turnover was slower.  To test the role of hyperacetylated tubulin in altering adhesion turnover, the authors measured microtubule dynamics in HDAC6-inhibited cells because dynamic microtubules are required to target adhesions for turnover.  HDAC6 inhibition yielded a decrease in microtubule dynamics that was sufficient to decrease focal adhesion turnover.  Thus, the results suggest a scenario in which the decreased dynamics of hyperacetylated microtubules in HDAC6-inhibited cells compromises their capacity to mediate the focal adhesion dynamics required for rapid cell migration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTqrWHdqcFXrVg90H21EOLACvtfcHk0lioEbmapZqlZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlsFSjtbs%253D&md5=6e25ead47afba657cf58bf4287df25fb</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1242%2Fjcs.03431&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.03431%26sid%3Dliteratum%253Aachs%26aulast%3DTran%26aufirst%3DA.%2BD.%2BA.%26aulast%3DMarmo%26aufirst%3DT.%2BP.%26aulast%3DSalam%26aufirst%3DA.%2BA.%26aulast%3DChe%26aufirst%3DS.%26aulast%3DFinkelstein%26aufirst%3DE.%26aulast%3DKabarriti%26aufirst%3DR.%26aulast%3DXenias%26aufirst%3DH.%2BS.%26aulast%3DMazitschek%26aufirst%3DR.%26aulast%3DHubbert%26aufirst%3DC.%26aulast%3DKawaguchi%26aufirst%3DY.%26aulast%3DSheetz%26aufirst%3DM.%2BP.%26aulast%3DYao%26aufirst%3DT.%26aulast%3DBulinski%26aufirst%3DJ.%2BC.%26atitle%3DHDAC6%2520Deacetylation%2520of%2520Tubulin%2520Modulates%2520Dynamics%2520of%2520Cellular%2520Adhesions%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2007%26volume%3D120%26spage%3D1469%26epage%3D1479%26doi%3D10.1242%2Fjcs.03431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubbert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guardiola, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawaguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nixon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, T. P.</span></span> <span> </span><span class="NLM_article-title">HDAC6 Is a Microtubule-Associated Deacetylase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>417</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">458</span>, <span class="refDoi"> DOI: 10.1038/417455a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2F417455a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=12024216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD38XjvVersr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=417&publication_year=2002&pages=455-458&author=C.+Hubbertauthor=A.+Guardiolaauthor=R.+Shaoauthor=Y.+Kawaguchiauthor=A.+Itoauthor=A.+Nixonauthor=M.+Yoshidaauthor=X.+F.+Wangauthor=T.+P.+Yao&title=HDAC6+Is+a+Microtubule-Associated+Deacetylase&doi=10.1038%2F417455a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">HDAC6 is a microtubule-associated deacetylase</span></div><div class="casAuthors">Hubbert, Charlotte; Guardiola, Amaris; Shao, Rong; Kawaguchi, Yoshiharu; Ito, Akihiro; Nixon, Andrew; Yoshida, Minoru; Wang, Xiao-Fan; Yao, Tso-Pang</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">417</span>
        (<span class="NLM_cas:issue">6887</span>),
    <span class="NLM_cas:pages">455-458</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Reversible acetylation of α-tubulin has been implicated in regulating microtubule stability and function.  The distribution of acetylated α-tubulin is tightly controlled and stereotypic.  Acetylated α-tubulin is most abundant in stable microtubules but is absent from dynamic cellular structures such as neuronal growth cones and the leading edges of fibroblasts.  However, the enzymes responsible for regulating tubulin acetylation and deacetylation are not known.  Here we report that a member of the histone deacetylase family, HDAC6, functions as a tubulin deacetylase.  HDAC6 is localized exclusively in the cytoplasm, where it assocs. with microtubules and localizes with the microtubule motor complex contg. p150glued.  In vivo, the overexpression of HDAC6 leads to a global deacetylation of α-tubulin, whereas a decrease in HDAC6 increases α-tubulin acetylation.  In vitro, purified HDAC6 potently deacetylates α-tubulin in assembled microtubules.  Furthermore, overexpression of HDAC6 promotes chemotactic cell movement, supporting the idea that HDAC6-mediated deacetylation regulates microtubule-dependent cell motility.  Our results show that HDAC6 is the tubulin deacetylase, and provide evidence that reversible acetylation regulates important biol. processes beyond histone metab. and gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp76Er3MrWtO7Vg90H21EOLACvtfcHk0lgwjAjrPln1OA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjvVersr4%253D&md5=f18cbb0af388925c606e822358ba3da6</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2F417455a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F417455a%26sid%3Dliteratum%253Aachs%26aulast%3DHubbert%26aufirst%3DC.%26aulast%3DGuardiola%26aufirst%3DA.%26aulast%3DShao%26aufirst%3DR.%26aulast%3DKawaguchi%26aufirst%3DY.%26aulast%3DIto%26aufirst%3DA.%26aulast%3DNixon%26aufirst%3DA.%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DX.%2BF.%26aulast%3DYao%26aufirst%3DT.%2BP.%26atitle%3DHDAC6%2520Is%2520a%2520Microtubule-Associated%2520Deacetylase%26jtitle%3DNature%26date%3D2002%26volume%3D417%26spage%3D455%26epage%3D458%26doi%3D10.1038%2F417455a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krämer, O. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahboobi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellmer, A.</span></span> <span> </span><span class="NLM_article-title">Drugging the HDAC6-HSP90 Interplay in Malignant Cells</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">501</span>– <span class="NLM_lpage">509</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2014.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1016%2Fj.tips.2014.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=25234862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A280%3ADC%252BC2M7ivFKgtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=501-509&author=O.+H.+Kr%C3%A4merauthor=S.+Mahboobiauthor=A.+Sellmer&title=Drugging+the+HDAC6-HSP90+Interplay+in+Malignant+Cells&doi=10.1016%2Fj.tips.2014.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging the HDAC6-HSP90 interplay in malignant cells</span></div><div class="casAuthors">Kramer Oliver H; Mahboobi Siavosh; Sellmer Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Trends in pharmacological sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">501-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Acetylation and deacetylation cycles regulate crucial biological processes.  The enzymes deacetylating lysine residues are termed histone deacetylases (HDACs).  Eighteen deacetylases have been isolated from mammalian cells.  There is an intense search underway for individual functions of these enzymes and for selective histone deacetylase inhibitors (HDACi).  HDAC6 in particular has unique cytoprotective functions that rely on its ability to ensure protein homeostasis and to prevent protein aggregation.  The chaperone heat shock protein 90 (HSP90) also safeguards proteins and is deacetylated by HDAC6.  Current data illustrate the complexity and importance of the HDAC6-HSP90 interplay.  In this review, we discuss how recently identified HSP90-dependent regulators of posttranslational modifications (PTMs) of HDAC6 dictate its functions, and how HDACi-induced acetylation of HSP90 might control oncologically relevant proteins, especially in leukemic cells.  Additionally, we discuss small molecules blocking HDAC6 and how such agents could become therapeutically relevant.  We summarize structure-function relationships that determine the specificity of drugs against HDAC6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS6kyzjnmxFum52AMpI4GK2fW6udTcc2eZj8Yt21UTVurntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M7ivFKgtw%253D%253D&md5=4aa468225dd58345f164ca1a10e324a2</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2014.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2014.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DKr%25C3%25A4mer%26aufirst%3DO.%2BH.%26aulast%3DMahboobi%26aufirst%3DS.%26aulast%3DSellmer%26aufirst%3DA.%26atitle%3DDrugging%2520the%2520HDAC6-HSP90%2520Interplay%2520in%2520Malignant%2520Cells%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2014%26volume%3D35%26spage%3D501%26epage%3D509%26doi%3D10.1016%2Fj.tips.2014.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthias, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthias, P.</span></span> <span> </span><span class="NLM_article-title">HDAC-6 Interacts with and Deacetylates Tubulin and Microtubules in Vivo</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1168</span>– <span class="NLM_lpage">1179</span>, <span class="refDoi"> DOI: 10.1093/emboj/cdg115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1093%2Femboj%2Fcdg115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=12606581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD3sXit1yls7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=1168-1179&author=Y.+Zhangauthor=N.+Liauthor=C.+Caronauthor=G.+Matthiasauthor=D.+Hessauthor=S.+Khochbinauthor=P.+Matthias&title=HDAC-6+Interacts+with+and+Deacetylates+Tubulin+and+Microtubules+in+Vivo&doi=10.1093%2Femboj%2Fcdg115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">HDAC-6 interacts with and deacetylates tubulin and microtubules in vivo</span></div><div class="casAuthors">Zhang, Yu; Li, Na; Caron, Cecile; Matthias, Gabriele; Hess, Daniel; Khochbin, Saadi; Matthias, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1168-1179</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Microtubules are cylindrical cytoskeletal structures found in almost all eukaryotic cell types which are involved in a great variety of cellular processes.  Reversible acetylation on the ε-amino group of α-tubulin Lys40 marks stabilized microtubule structures and may contribute to regulating microtubule dynamics.  Yet, the enzymes catalyzing this acetylation/deacetylation have remained unidentified until recently.  Here we report that β-tubulin interacts with histone deacetylase-6 (HDAC-6) in a yeast two-hybrid assay and in vitro.  We find that HDAC-6 is a micro tubule-assocd. protein capable of deacetylating α-tubulin in vivo and in vitro.  HDAC-6's microtubule binding and deacetylation functions both depend on the hdac domains.  Overexpression of HDAC-6 in mammalian cells leads to tubulin hypoacetylation.  In contrast, inhibition of HDAC-6 function by two independent mechanisms-pharmacol. (HDAC inhibitors) or genetic (targeted inactivation of HDAC-6 in embryonic stem cells)-leads to hyperacetylation of tubulin and microtubules.  Taken together, our data provide evidence that HDAC-6 might act as a dual deacetylase for tubulin and histones, and suggest the possibility that acetylated non-histone proteins might represent novel targets for pharmacol. therapy by HDAC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBqjHB5aVYPrVg90H21EOLACvtfcHk0lhKnBYWA2Lzxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXit1yls7c%253D&md5=0d2b6dd552163866b85f14471ac600f7</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1093%2Femboj%2Fcdg115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252Fcdg115%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DN.%26aulast%3DCaron%26aufirst%3DC.%26aulast%3DMatthias%26aufirst%3DG.%26aulast%3DHess%26aufirst%3DD.%26aulast%3DKhochbin%26aufirst%3DS.%26aulast%3DMatthias%26aufirst%3DP.%26atitle%3DHDAC-6%2520Interacts%2520with%2520and%2520Deacetylates%2520Tubulin%2520and%2520Microtubules%2520in%2520Vivo%26jtitle%3DEMBO%2520J.%26date%3D2003%26volume%3D22%26spage%3D1168%26epage%3D1179%26doi%3D10.1093%2Femboj%2Fcdg115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cosenza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, S.</span></span> <span> </span><span class="NLM_article-title">The Therapeutic Strategy of HDAC6 Inhibitors in Lymphoproliferative Disease</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2337</span>, <span class="refDoi"> DOI: 10.3390/ijms19082337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.3390%2Fijms19082337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsVGhtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=2337&author=M.+Cosenzaauthor=S.+Pozzi&title=The+Therapeutic+Strategy+of+HDAC6+Inhibitors+in+Lymphoproliferative+Disease&doi=10.3390%2Fijms19082337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The therapeutic strategy of HDAC6 inhibitors in lymphoproliferative disease</span></div><div class="casAuthors">Cosenza, Maria; Pozzi, Samantha</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2337/1-2337/26</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Histone deacetylases (HDACs) are master regulators of chromatin remodeling, acting as epigenetic regulators of gene expression.  In the last decade, inhibition of HDACs has become a target for specific epigenetic modifications related to cancer development.  Overexpression of HDAC has been obsd. in several hematol. malignancies.  Therefore, the observation that HDACs might play a role in various hematol. malignancies has brought to the development of HDAC inhibitors as potential antitumor agents.  Recently, the class IIb, HDAC6, has emerged as one potential selective HDACi.  This isoenzyme represents an important pharmacol. target for selective inhibition.  Its selectivity may reduce the toxicity related to the off-target effects of pan-HDAC inhibitors.  HDAC6 has also been studied in cancer esp. for its ability to coordinate a variety of cellular processes that are important for cancer pathogenesis.  HDAC6 has been reported to be overexpressed in lymphoid cells and its inhibition has demonstrated activity in preclin. and clin. study of lymphoproliferative disease.  Various studies of HDAC6 inhibitors alone and in combination with other agents provide strong scientific rationale for the evaluation of these new agents in the clin. setting of hematol. malignancies.  In this review, we describe the HDACs, their inhibitors, and the recent advances of HDAC6 inhibitors, their mechanisms of action and role in lymphoproliferative disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr40AmGWIhm0rVg90H21EOLACvtfcHk0lhKnBYWA2Lzxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsVGhtrY%253D&md5=e2f63affb8a194d516a2465e772f46c1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.3390%2Fijms19082337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms19082337%26sid%3Dliteratum%253Aachs%26aulast%3DCosenza%26aufirst%3DM.%26aulast%3DPozzi%26aufirst%3DS.%26atitle%3DThe%2520Therapeutic%2520Strategy%2520of%2520HDAC6%2520Inhibitors%2520in%2520Lymphoproliferative%2520Disease%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2018%26volume%3D19%26spage%3D2337%26doi%3D10.3390%2Fijms19082337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormack, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams-Truax, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mita, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burris, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butterfoss, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dugan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culver, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prince, H. M.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylase Inhibitor Panobinostat Induces Clinical Responses with Associated Alterations in Gene Expression Profiles in Cutaneous T-Cell Lymphoma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">4500</span>– <span class="NLM_lpage">4510</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-07-4262</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1158%2F1078-0432.CCR-07-4262" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=18628465" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD1cXosFGitL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=4500-4510&author=L.+Ellisauthor=Y.+Panauthor=G.+K.+Smythauthor=D.+J.+Georgeauthor=C.+McCormackauthor=R.+Williams-Truaxauthor=M.+Mitaauthor=J.+Beckauthor=H.+Burrisauthor=G.+Ryanauthor=P.+Atadjaauthor=D.+Butterfossauthor=M.+Duganauthor=K.+Culverauthor=R.+W.+Johnstoneauthor=H.+M.+Prince&title=Histone+Deacetylase+Inhibitor+Panobinostat+Induces+Clinical+Responses+with+Associated+Alterations+in+Gene+Expression+Profiles+in+Cutaneous+T-Cell+Lymphoma&doi=10.1158%2F1078-0432.CCR-07-4262"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Deacetylase Inhibitor Panobinostat Induces Clinical Responses with Associated Alterations in Gene Expression Profiles in Cutaneous T-Cell Lymphoma</span></div><div class="casAuthors">Ellis, Leigh; Pan, Yan; Smyth, Gordon K.; George, Daniel J.; McCormack, Chris; Williams-Truax, Roxanne; Mita, Monica; Beck, Joachim; Burris, Howard; Ryan, Gail; Atadja, Peter; Butterfoss, Dale; Dugan, Margaret; Culver, Kenneth; Johnstone, Ricky W.; Prince, H. Miles</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4500-4510</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Histone deacetylase inhibitors can alter gene expression and mediate diverse antitumor activities.  Herein, we report the safety and activity of the histone deacetylase inhibitor panobinostat (LBH589) in cutaneous T-cell lymphoma (CTCL) and identify genes commonly regulated by panobinostat.  Panobinostat was administered orally to patients with CTCL on Monday, Wednesday, and Friday of each week on a 28-day cycle.  A dose of 30 mg was considered excessively toxic, and subsequent patients were treated at the expanded max. tolerated dose of 20 mg.  Biopsies from six patients taken 0, 4, 8, and 24 h after administration were subjected to microarray gene expression profiling and real-time quant. PCR of selected genes.  Patients attained a complete response (n = 2), attained a partial response (n = 4), achieved stable disease with ongoing improvement (n = 1), and progressed on treatment (n = 2).  Microarray data showed distinct gene expression response profiles over time following panobinostat treatment, with the majority of genes being repressed.  Twenty-three genes were commonly regulated by panobinostat in all patients tested.  Panobinostat is well tolerated and induces clin. responses in CTCL patients.  Microarray analyses of tumor samples indicate that panobinostat induces rapid changes in gene expression, and surprisingly more genes are repressed than are activated.  A unique set of genes that can mediate biol. responses such as apoptosis, immune regulation, and angiogenesis were commonly regulated in response to panobinostat.  These genes are potential mol. biomarkers for panobinostat activity and are strong candidates for the future assessment of their functional role(s) in mediating the antitumor responses of panobinostat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKdLh4qldQ37Vg90H21EOLACvtfcHk0liAUPP8OI5dQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXosFGitL0%253D&md5=0b53d3b837c0f89a700287855ffcc961</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-07-4262&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-07-4262%26sid%3Dliteratum%253Aachs%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DPan%26aufirst%3DY.%26aulast%3DSmyth%26aufirst%3DG.%2BK.%26aulast%3DGeorge%26aufirst%3DD.%2BJ.%26aulast%3DMcCormack%26aufirst%3DC.%26aulast%3DWilliams-Truax%26aufirst%3DR.%26aulast%3DMita%26aufirst%3DM.%26aulast%3DBeck%26aufirst%3DJ.%26aulast%3DBurris%26aufirst%3DH.%26aulast%3DRyan%26aufirst%3DG.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DButterfoss%26aufirst%3DD.%26aulast%3DDugan%26aufirst%3DM.%26aulast%3DCulver%26aufirst%3DK.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26aulast%3DPrince%26aufirst%3DH.%2BM.%26atitle%3DHistone%2520Deacetylase%2520Inhibitor%2520Panobinostat%2520Induces%2520Clinical%2520Responses%2520with%2520Associated%2520Alterations%2520in%2520Gene%2520Expression%2520Profiles%2520in%2520Cutaneous%2520T-Cell%2520Lymphoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2008%26volume%3D14%26spage%3D4500%26epage%3D4510%26doi%3D10.1158%2F1078-0432.CCR-07-4262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koeller, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grozinger, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">Domain-Selective Small-Molecule Inhibitor of Histone Deacetylase 6 (HDAC6)-Mediated Tubulin Deacetylation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">4389</span>– <span class="NLM_lpage">4394</span>, <span class="refDoi"> DOI: 10.1073/pnas.0430973100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1073%2Fpnas.0430973100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=12677000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjt12gtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2003&pages=4389-4394&author=S.+J.+Haggartyauthor=K.+M.+Koellerauthor=J.+C.+Wongauthor=C.+M.+Grozingerauthor=S.+L.+Schreiber&title=Domain-Selective+Small-Molecule+Inhibitor+of+Histone+Deacetylase+6+%28HDAC6%29-Mediated+Tubulin+Deacetylation&doi=10.1073%2Fpnas.0430973100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Domain-selective small-molecule inhibitor of histone deacetylase 6 (HDAC6)-mediated tubulin deacetylation</span></div><div class="casAuthors">Haggarty, Stephen J.; Koeller, Kathryn M.; Wong, Jason C.; Grozinger, Christina M.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4389-4394</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein acetylation, esp. histone acetylation, is the subject of both research and clin. investigation.  At least four small-mol. histone deacetylase inhibitors are currently in clin. trials for the treatment of cancer.  These and other inhibitors also affect microtubule acetylation.  A multidimensional, chem. genetic screen of 7392 small mols. was used to discover "tubacin," which inhibits α-tubulin deacetylation in mammalian cells.  Tubacin does not affect the level of histone acetylation, gene-expression patterns, or cell-cycle progression.  We provide evidence that class II histone deacetylase 6 (HDAC6) is the intracellular target of tubacin.  Only one of the two catalytic domains of HDAC6 possesses tubulin deacetylase activity, and only this domain is bound by tubacin.  Tubacin treatment did not affect the stability of microtubules but did decrease cell motility.  HDAC6 overexpression disrupted the localization of p58, a protein that mediates binding of Golgi elements to microtubules.  Our results highlight the role of α-tubulin acetylation in mediating the localization of microtubule-assocd. proteins.  They also suggest that small mols. that selectively inhibit HDAC6-mediated α-tubulin deacetylation, a first example of which is tubacin, might have therapeutic applications as antimetastatic and antiangiogenic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbQBWjEZr47rVg90H21EOLACvtfcHk0liAUPP8OI5dQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjt12gtL0%253D&md5=260da1ede64782ddbddda31b740b9782</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0430973100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0430973100%26sid%3Dliteratum%253Aachs%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DKoeller%26aufirst%3DK.%2BM.%26aulast%3DWong%26aufirst%3DJ.%2BC.%26aulast%3DGrozinger%26aufirst%3DC.%2BM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DDomain-Selective%2520Small-Molecule%2520Inhibitor%2520of%2520Histone%2520Deacetylase%25206%2520%2528HDAC6%2529-Mediated%2520Tubulin%2520Deacetylation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2003%26volume%3D100%26spage%3D4389%26epage%3D4394%26doi%3D10.1073%2Fpnas.0430973100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almeciga-Pinto, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarpe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Duzer, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazitschek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quayle, S. N.</span></span> <span> </span><span class="NLM_article-title">Selective HDAC Inhibition by ACY-241 Enhances the Activity of Paclitaxel in Solid Tumor Models</span>. <i>Oncotarget.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2694</span>– <span class="NLM_lpage">2707</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13738</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.18632%2Foncotarget.13738" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=27926524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A280%3ADC%252BC2sjksVSmuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=2694-2707&author=P.+Huangauthor=I.+Almeciga-Pintoauthor=M.+Jarpeauthor=J.+H.+van+Duzerauthor=R.+Mazitschekauthor=M.+Yangauthor=S.+S.+Jonesauthor=S.+N.+Quayle&title=Selective+HDAC+Inhibition+by+ACY-241+Enhances+the+Activity+of+Paclitaxel+in+Solid+Tumor+Models&doi=10.18632%2Foncotarget.13738"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Selective HDAC inhibition by ACY-241 enhances the activity of paclitaxel in solid tumor models</span></div><div class="casAuthors">Huang Pengyu; Almeciga-Pinto Ingrid; Jarpe Matthew; van Duzer John H; Yang Min; Jones Simon S; Quayle Steven N; Mazitschek Ralph</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">2694-2707</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">ACY-241 is a novel, orally available and selective histone deacetylase (HDAC) 6 inhibitor in Phase 1b clinical development in multiple myeloma (NCT 02400242).  Like the structurally related drug ACY-1215 (ricolinostat), ACY-241 has the potential for a substantially reduced side effect profile versus current nonselective HDAC inhibitor drug candidates due to reduced potency against Class I HDACs while retaining the potential for anticancer effectiveness.  We now show that combination treatment of xenograft models with paclitaxel and either ricolinostat or ACY-241 significantly suppresses solid tumor growth.  In cell lines from multiple solid tumor lineages, combination treatment with ACY-241 and paclitaxel enhanced inhibition of proliferation and increased cell death relative to either single agent alone.  Combination treatment with ACY-241 and paclitaxel also resulted in more frequent occurrence of mitotic cells with abnormal multipolar spindles and aberrant mitoses, consistent with the observed increase of aneuploid cells.  At the molecular level, multipolar mitotic spindle formation was observed to be NuMA-dependent and γ-tubulin independent, suggesting that treatment-induced multipolar spindle formation does not depend on centrosomal amplification.  The significantly enhanced efficacy of ACY-241 plus paclitaxel observed here, in addition to the anticipated superior safety profile of a selective HDAC6 inhibitor versus pan-HDAC inhibitors, provides a strong rationale for clinical development of this combination in patients with advanced solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQFLMaYGUFFIBfKb6ELK8t0fW6udTcc2ebspStEUns9oLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjksVSmuw%253D%253D&md5=b8a516375b03fc65536598ab9c0e4a21</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13738%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DP.%26aulast%3DAlmeciga-Pinto%26aufirst%3DI.%26aulast%3DJarpe%26aufirst%3DM.%26aulast%3Dvan%2BDuzer%26aufirst%3DJ.%2BH.%26aulast%3DMazitschek%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DS.%2BS.%26aulast%3DQuayle%26aufirst%3DS.%2BN.%26atitle%3DSelective%2520HDAC%2520Inhibition%2520by%2520ACY-241%2520Enhances%2520the%2520Activity%2520of%2520Paclitaxel%2520in%2520Solid%2520Tumor%2520Models%26jtitle%3DOncotarget.%26date%3D2017%26volume%3D8%26spage%3D2694%26epage%3D2707%26doi%3D10.18632%2Foncotarget.13738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hideshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarpe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Duzer, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazitschek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogier, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirstea, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodig, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scullen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canavese, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, N.</span></span> <span> </span><span class="NLM_article-title">Preclinical Activity, Pharmacodynamic, and Pharmacokinetic Properties of a Selective HDAC6 Inhibitor, ACY-1215, in Combination with Bortezomib in Multiple Myeloma</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">2579</span>– <span class="NLM_lpage">2589</span>, <span class="refDoi"> DOI: 10.1182/blood-2011-10-387365</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1182%2Fblood-2011-10-387365" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=22262760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVOntb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2012&pages=2579-2589&author=L.+Santoauthor=T.+Hideshimaauthor=A.+L.+Kungauthor=J.+C.+Tsengauthor=D.+Tamangauthor=M.+Yangauthor=M.+Jarpeauthor=J.+H.+Van+Duzerauthor=R.+Mazitschekauthor=W.+C.+Ogierauthor=D.+Cirsteaauthor=S.+Rodigauthor=H.+Edaauthor=T.+Scullenauthor=M.+Canaveseauthor=J.+Bradnerauthor=K.+C.+Andersonauthor=S.+S.+Jonesauthor=N.+Raje&title=Preclinical+Activity%2C+Pharmacodynamic%2C+and+Pharmacokinetic+Properties+of+a+Selective+HDAC6+Inhibitor%2C+ACY-1215%2C+in+Combination+with+Bortezomib+in+Multiple+Myeloma&doi=10.1182%2Fblood-2011-10-387365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical activity, pharmacodynamic, and pharmacokinetic properties of a selective HDAC6 inhibitor, ACY-1215, in combination with bortezomib in multiple myeloma</span></div><div class="casAuthors">Santo, Loredana; Hideshima, Teru; Kung, Andrew L.; Tseng, Jen-Chieh; Tamang, David; Yang, Min; Jarpe, Matthew; van Duzer, John H.; Mazitschek, Ralph; Ogier, Walter C.; Cirstea, Diana; Rodig, Scott; Eda, Homare; Scullen, Tyler; Canavese, Miriam; Bradner, James; Anderson, Kenneth C.; Jones, Simon S.; Raje, Noopur</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2579-2589</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Histone deacetylase (HDAC) enzymic activity has been linked to the transcription of DNA in cancers including multiple myeloma (MM).  Therefore, HDAC inhibitors used alone and in combination are being actively studied as novel therapies in MM.  In the present study, we investigated the preclin. activity of ACY-1215, an HDAC6-selective inhibitor, alone and in combination with bortezomib in MM.  Low doses of ACY-1215 combined with bortezomib triggered synergistic anti-MM activity, resulting in protracted endoplasmic reticulum stress and apoptosis via activation of caspase-3, caspase-8, and caspase-9 and poly (ADP) ribosome polymerase.  In vivo, the anti-MM activity of ACY-1215 in combination with bortezomib was confirmed using 2 different xenograft SCID mouse models: human MM injected s.c. (the plasmacytoma model) and luciferase-expressing human MM injected i.v. (the disseminated MM model).  Tumor growth was significantly delayed and overall survival was significantly prolonged in animals treated with the combination therapy.  Pharmacokinetic data showed peak plasma levels of ACY-1215 at 4 h after treatment coincident with an increase in acetylated α-tubulin, a marker of HDAC6 inhibition, by immunohistochem. and Western blot anal.  These studies provide preclin. rationale for acetylated α-tubulin use as a pharmacodynamic biomarker in future clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoelmsk1wo_LVg90H21EOLACvtfcHk0lhqPau6yaNozQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVOntb4%253D&md5=094298bdd3d5b91c7060d8b294857846</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1182%2Fblood-2011-10-387365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2011-10-387365%26sid%3Dliteratum%253Aachs%26aulast%3DSanto%26aufirst%3DL.%26aulast%3DHideshima%26aufirst%3DT.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DTseng%26aufirst%3DJ.%2BC.%26aulast%3DTamang%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DJarpe%26aufirst%3DM.%26aulast%3DVan%2BDuzer%26aufirst%3DJ.%2BH.%26aulast%3DMazitschek%26aufirst%3DR.%26aulast%3DOgier%26aufirst%3DW.%2BC.%26aulast%3DCirstea%26aufirst%3DD.%26aulast%3DRodig%26aufirst%3DS.%26aulast%3DEda%26aufirst%3DH.%26aulast%3DScullen%26aufirst%3DT.%26aulast%3DCanavese%26aufirst%3DM.%26aulast%3DBradner%26aufirst%3DJ.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DJones%26aufirst%3DS.%2BS.%26aulast%3DRaje%26aufirst%3DN.%26atitle%3DPreclinical%2520Activity%252C%2520Pharmacodynamic%252C%2520and%2520Pharmacokinetic%2520Properties%2520of%2520a%2520Selective%2520HDAC6%2520Inhibitor%252C%2520ACY-1215%252C%2520in%2520Combination%2520with%2520Bortezomib%2520in%2520Multiple%2520Myeloma%26jtitle%3DBlood%26date%3D2012%26volume%3D119%26spage%3D2579%26epage%3D2589%26doi%3D10.1182%2Fblood-2011-10-387365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogl, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagannath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hari, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlowski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supko, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wheeler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markelewicz, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lonial, S.</span></span> <span> </span><span class="NLM_article-title">Ricolinostat, the First Selective Histone Deacetylase 6 Inhibitor, in Combination with Bortezomib and Dexamethasone for Relapsed or Refractory Multiple Myeloma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3307</span>– <span class="NLM_lpage">3315</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-16-2526</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1158%2F1078-0432.CCR-16-2526" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=28053023" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFSnurbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=3307-3315&author=D.+T.+Voglauthor=N.+Rajeauthor=S.+Jagannathauthor=P.+Richardsonauthor=P.+Hariauthor=R.+Orlowskiauthor=J.+G.+Supkoauthor=D.+Tamangauthor=M.+Yangauthor=S.+S.+Jonesauthor=C.+Wheelerauthor=R.+J.+Markelewiczauthor=S.+Lonial&title=Ricolinostat%2C+the+First+Selective+Histone+Deacetylase+6+Inhibitor%2C+in+Combination+with+Bortezomib+and+Dexamethasone+for+Relapsed+or+Refractory+Multiple+Myeloma&doi=10.1158%2F1078-0432.CCR-16-2526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Ricolinostat, the First Selective Histone Deacetylase 6 Inhibitor, in Combination with Bortezomib and Dexamethasone for Relapsed or Refractory Multiple Myeloma</span></div><div class="casAuthors">Vogl, Dan T.; Raje, Noopur; Jagannath, Sundar; Richardson, Paul; Hari, Parameswaran; Orlowski, Robert; Supko, Jeffrey G.; Tamang, David; Yang, Min; Jones, Simon S.; Wheeler, Catherine; Markelewicz, Robert J.; Lonial, Sagar</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3307-3315</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Histone deacetylase (HDAC) inhibition improves the efficacy of proteasome inhibition for multiple myeloma but adds substantial toxicity.  Preclin. models suggest that the obsd. synergy is due to the role of HDAC6 in mediating resistance to proteasome inhibition via the aggresome/autophagy pathway of protein degrdn.  Exptl. Design: We conducted a phase I/II trial of the HDAC6-selective inhibitor ricolinostat to define the safety, preliminary efficacy, and recommended phase II dose in combination with std. proteasome inhibitor therapy.  Patients with relapsed or refractory multiple myeloma received oral ricolinostat on days 1-5 and 8-12 of each 21-day cycle.  Results: Single-agent ricolinostat therapy resulted in neither significant toxicity nor clin. responses.  Combination therapy with bortezomib and dexamethasone was well-tolerated during dose escalation but led to dose-limiting diarrhea in an expansion cohort at a ricolinostat dose of 160 mg twice daily.  Combination therapy at a ricolinostat dose of 160 mg daily in a second expansion cohort was well tolerated, with less severe hematol., gastrointestinal, and constitutional toxicities compared with published data on nonselective HDAC inhibitors.  The overall response rate in combination with daily ricolinostat at ≥160 mg was 37%.  The response rate to combination therapy among bortezomib-refractory patients was 14%.  Samples taken during therapy showed dose-dependent increases of acetylated tubulin in peripheral blood lymphocytes.  Conclusions: At the recommended phase II dose of ricolinostat of 160 mg daily, the combination with bortezomib and dexamethasone is safe, well-tolerated, and active, suggesting that selective inhibition of HDAC6 is a promising approach to multiple myeloma therapy.  Clin Cancer Res; 23(13); 3307-15. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFQogphgLWbLVg90H21EOLACvtfcHk0ljK7DAwL6RW5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFSnurbL&md5=2dd174501d2f331aeb0eff42072001d1</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-16-2526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-16-2526%26sid%3Dliteratum%253Aachs%26aulast%3DVogl%26aufirst%3DD.%2BT.%26aulast%3DRaje%26aufirst%3DN.%26aulast%3DJagannath%26aufirst%3DS.%26aulast%3DRichardson%26aufirst%3DP.%26aulast%3DHari%26aufirst%3DP.%26aulast%3DOrlowski%26aufirst%3DR.%26aulast%3DSupko%26aufirst%3DJ.%2BG.%26aulast%3DTamang%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DS.%2BS.%26aulast%3DWheeler%26aufirst%3DC.%26aulast%3DMarkelewicz%26aufirst%3DR.%2BJ.%26aulast%3DLonial%26aufirst%3DS.%26atitle%3DRicolinostat%252C%2520the%2520First%2520Selective%2520Histone%2520Deacetylase%25206%2520Inhibitor%252C%2520in%2520Combination%2520with%2520Bortezomib%2520and%2520Dexamethasone%2520for%2520Relapsed%2520or%2520Refractory%2520Multiple%2520Myeloma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D3307%26epage%3D3315%26doi%3D10.1158%2F1078-0432.CCR-16-2526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leonhardt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellmer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krämer, O. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dove, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahboobi, S.</span></span> <span> </span><span class="NLM_article-title">Design and Biological Evaluation of Tetrahydro-β-Carboline Derivatives as Highly Potent Histone Deacetylase 6 (HDAC6) Inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">357</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.04.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1016%2Fj.ejmech.2018.04.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=29738953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpt1agtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2018&pages=329-357&author=M.+Leonhardtauthor=A.+Sellmerauthor=O.+H.+Kr%C3%A4merauthor=S.+Doveauthor=S.+Elzauthor=B.+Krausauthor=M.+Beyerauthor=S.+Mahboobi&title=Design+and+Biological+Evaluation+of+Tetrahydro-%CE%B2-Carboline+Derivatives+as+Highly+Potent+Histone+Deacetylase+6+%28HDAC6%29+Inhibitors&doi=10.1016%2Fj.ejmech.2018.04.046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Design and biological evaluation of tetrahydro-β-carboline derivatives as highly potent histone deacetylase 6 (HDAC6) inhibitors</span></div><div class="casAuthors">Leonhardt, Michel; Sellmer, Andreas; Kraemer, Oliver H.; Dove, Stefan; Elz, Sigurd; Kraus, Birgit; Beyer, Mandy; Mahboobi, Siavosh</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">329-357</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Various diseases are related to epigenetic modifications.  Histone deacetylases (HDACs) and histone acetyltransferases (HATs) det. the pattern of histone acetylation, and thus are involved in the regulation of gene expression.  First generation histone deacetylase inhibitors (HDACi) are unselective, hinder all different kinds of zinc dependent HDACs and addnl. cause several side effects.  Subsequently, selective HDACi are gaining more and more interest.  Esp., selective histone deacetylase 6 inhibitors (HDAC6i) are supposed to be less toxic.  Here the authors present a successful optimization study of tubastatin A, the synthesis and biol. evaluation of new inhibitors based on hydroxamic acids linked to various tetrahydro-β-carboline derivs.  The potency of the authors' selective HDAC6 inhibitors, exhibiting IC50 values in a range of 1-10 nM towards HDAC6, was evaluated with the help of a recombinant human HDAC6 enzyme assay.  Selectivity was proofed in cellular assays by the hyperacetylation of surrogate parameter α-tubulin in the absence of acetylated histone H3 analyzed by Western Blot.  The authors show that all synthesized compds., with varies modifications of the rigid cap group, were selective and potent HDAC6 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO3jAwqO8N0LVg90H21EOLACvtfcHk0ljK7DAwL6RW5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpt1agtbY%253D&md5=668b7ae179e680a0ae7116589e315cb5</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.04.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.04.046%26sid%3Dliteratum%253Aachs%26aulast%3DLeonhardt%26aufirst%3DM.%26aulast%3DSellmer%26aufirst%3DA.%26aulast%3DKr%25C3%25A4mer%26aufirst%3DO.%2BH.%26aulast%3DDove%26aufirst%3DS.%26aulast%3DElz%26aufirst%3DS.%26aulast%3DKraus%26aufirst%3DB.%26aulast%3DBeyer%26aufirst%3DM.%26aulast%3DMahboobi%26aufirst%3DS.%26atitle%3DDesign%2520and%2520Biological%2520Evaluation%2520of%2520Tetrahydro-%25CE%25B2-Carboline%2520Derivatives%2520as%2520Highly%2520Potent%2520Histone%2520Deacetylase%25206%2520%2528HDAC6%2529%2520Inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D152%26spage%3D329%26epage%3D357%26doi%3D10.1016%2Fj.ejmech.2018.04.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sellmer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stangl, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grünstein, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonhardt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pongratz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichhorn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Striegl, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenei-Lanzl, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dove, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Straub, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krämer, O. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahboobi, S.</span></span> <span> </span><span class="NLM_article-title">Marbostat-100 Defines a New Class of Potent and Selective Antiinflammatory and Antirheumatic Histone Deacetylase 6 Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">3454</span>– <span class="NLM_lpage">3477</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01593</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01593" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmtFCnsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=3454-3477&author=A.+Sellmerauthor=H.+Stanglauthor=M.+Beyerauthor=E.+Gr%C3%BCnsteinauthor=M.+Leonhardtauthor=H.+Pongratzauthor=E.+Eichhornauthor=S.+Elzauthor=B.+Strieglauthor=Z.+Jenei-Lanzlauthor=S.+Doveauthor=R.+H.+Straubauthor=O.+H.+Kr%C3%A4merauthor=S.+Mahboobi&title=Marbostat-100+Defines+a+New+Class+of+Potent+and+Selective+Antiinflammatory+and+Antirheumatic+Histone+Deacetylase+6+Inhibitors&doi=10.1021%2Facs.jmedchem.7b01593"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Marbostat-100 Defines a New Class of Potent and Selective Antiinflammatory and Antirheumatic Histone Deacetylase 6 Inhibitors</span></div><div class="casAuthors">Sellmer, Andreas; Stangl, Hubert; Beyer, Mandy; Gruenstein, Elisabeth; Leonhardt, Michel; Pongratz, Herwig; Eichhorn, Emerich; Elz, Sigurd; Striegl, Birgit; Jenei-Lanzl, Zsuzsa; Dove, Stefan; Straub, Rainer H.; Kraemer, Oliver H.; Mahboobi, Siavosh</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3454-3477</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic modifiers of the histone deacetylase (HDAC) family contribute to autoimmunity, cancer, HIV infection, inflammation, and neurodegeneration.  Hence, histone deacetylase inhibitors (HDACi), which alter protein acetylation, gene expression patterns, and cell fate decisions, represent promising new drugs for the therapy of these diseases.  Whereas pan-HDACi inhibit all 11 Zn2+-dependent histone deacetylases (HDACs) and cause a broad spectrum of side effects, specific inhibitors of histone deacetylase 6 (HDAC6i) are supposed to have less side effects.  The authors present the synthesis and biol. evaluation of Marbostats, novel HDAC6i that contain the hydroxamic acid moiety linked to tetrahydro-β-carboline derivs.  The lead compd. Marbostat-100 is a more potent and more selective HDAC6i than previously established well-characterized compds. in vitro as well as in cells.  Moreover, Marbostat-100 is well tolerated by mice and effective against collagen type II induced arthritis.  Thus, Marbostat-100 represents a most selective known HDAC6i and the possibility for clin. evaluation of a HDAC isoform-specific drug.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqixUQJJP-7bVg90H21EOLACvtfcHk0lgR2sLpeWtJJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmtFCnsbg%253D&md5=0f6c62f03f50501728d43fef66d0d64b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01593&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01593%26sid%3Dliteratum%253Aachs%26aulast%3DSellmer%26aufirst%3DA.%26aulast%3DStangl%26aufirst%3DH.%26aulast%3DBeyer%26aufirst%3DM.%26aulast%3DGr%25C3%25BCnstein%26aufirst%3DE.%26aulast%3DLeonhardt%26aufirst%3DM.%26aulast%3DPongratz%26aufirst%3DH.%26aulast%3DEichhorn%26aufirst%3DE.%26aulast%3DElz%26aufirst%3DS.%26aulast%3DStriegl%26aufirst%3DB.%26aulast%3DJenei-Lanzl%26aufirst%3DZ.%26aulast%3DDove%26aufirst%3DS.%26aulast%3DStraub%26aufirst%3DR.%2BH.%26aulast%3DKr%25C3%25A4mer%26aufirst%3DO.%2BH.%26aulast%3DMahboobi%26aufirst%3DS.%26atitle%3DMarbostat-100%2520Defines%2520a%2520New%2520Class%2520of%2520Potent%2520and%2520Selective%2520Antiinflammatory%2520and%2520Antirheumatic%2520Histone%2520Deacetylase%25206%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D3454%26epage%3D3477%26doi%3D10.1021%2Facs.jmedchem.7b01593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cubizolles, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseaux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kneissel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chua, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizeracki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthias, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alt, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthias, P.</span></span> <span> </span><span class="NLM_article-title">Mice Lacking Histone Deacetylase 6 Have Hyperacetylated Tubulin but Are Viable and Develop Normally</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1688</span>– <span class="NLM_lpage">1701</span>, <span class="refDoi"> DOI: 10.1128/MCB.01154-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1128%2FMCB.01154-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=18180281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD1cXivVajtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=1688-1701&author=Y.+Zhangauthor=S.+Kwonauthor=T.+Yamaguchiauthor=F.+Cubizollesauthor=S.+Rousseauxauthor=M.+Kneisselauthor=C.+Caoauthor=N.+Liauthor=H.-L.+Chengauthor=K.+Chuaauthor=D.+Lombardauthor=A.+Mizerackiauthor=G.+Matthiasauthor=F.+W.+Altauthor=S.+Khochbinauthor=P.+Matthias&title=Mice+Lacking+Histone+Deacetylase+6+Have+Hyperacetylated+Tubulin+but+Are+Viable+and+Develop+Normally&doi=10.1128%2FMCB.01154-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Mice lacking histone deacetylase 6 have hyperacetylated tubulin but are viable and develop normally</span></div><div class="casAuthors">Zhang, Yu; Kwon, SoHee; Yamaguchi, Teppei; Cubizolles, Fabien; Rousseaux, Sophie; Kneissel, Michaela; Cao, Chun; Li, Na; Cheng, Hwei-Ling; Chua, Katrin; Lombard, David; Mizeracki, Adam; Matthias, Gabriele; Alt, Frederick W.; Khochbin, Saadi; Matthias, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1688-1701</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Posttranslational modifications play important roles in regulating protein structure and function.  Histone deacetylase 6 (HDAC6) is a mostly cytoplasmic class II HDAC, which has a unique structure with two catalytic domains and a domain binding ubiquitin with high affinity.  This enzyme was recently identified as a multisubstrate protein deacetylase that can act on acetylated histone tails, α-tubulin and Hsp90.  To investigate the in vivo functions of HDAC6 and the relevance of tubulin acetylation/deacetylation, we targeted the HDAC6 gene by homologous recombination in embryonic stem cells and generated knockout mice.  HDAC6-deficient mice are viable and fertile and show hyperacetylated tubulin in most tissues.  The highest level of expression of HDAC6 is seen in the testis, yet development and function of this organ are normal in the absence of HDAC6.  Likewise, lymphoid development is normal, but the immune response is moderately affected.  Furthermore, the lack of HDAC6 results in a small increase in cancellous bone mineral d., indicating that this deacetylase plays a minor role in bone biol.  HDAC6-deficient mouse embryonic fibroblasts show apparently normal microtubule organization and stability and also show increased Hsp90 acetylation correlating with impaired Hsp90 function.  Collectively, these data demonstrate that mice survive well without HDAC6 and that tubulin hyperacetylation is not detrimental to normal mammalian development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1on7_hqKefrVg90H21EOLACvtfcHk0lgR2sLpeWtJJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXivVajtLo%253D&md5=9c83b56ee31ea67a04357b31467ed655</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1128%2FMCB.01154-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01154-06%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DKwon%26aufirst%3DS.%26aulast%3DYamaguchi%26aufirst%3DT.%26aulast%3DCubizolles%26aufirst%3DF.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DKneissel%26aufirst%3DM.%26aulast%3DCao%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DN.%26aulast%3DCheng%26aufirst%3DH.-L.%26aulast%3DChua%26aufirst%3DK.%26aulast%3DLombard%26aufirst%3DD.%26aulast%3DMizeracki%26aufirst%3DA.%26aulast%3DMatthias%26aufirst%3DG.%26aulast%3DAlt%26aufirst%3DF.%2BW.%26aulast%3DKhochbin%26aufirst%3DS.%26aulast%3DMatthias%26aufirst%3DP.%26atitle%3DMice%2520Lacking%2520Histone%2520Deacetylase%25206%2520Have%2520Hyperacetylated%2520Tubulin%2520but%2520Are%2520Viable%2520and%2520Develop%2520Normally%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2008%26volume%3D28%26spage%3D1688%26epage%3D1701%26doi%3D10.1128%2FMCB.01154-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hideshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cottini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakubikova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorgun, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mimura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tai, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span> <span> </span><span class="NLM_article-title">Rational Combination Treatment with Histone Deacetylase Inhibitors and Immunomodulatory Drugs in Multiple Myeloma</span>. <i>Blood Cancer J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">e312</span>, <span class="refDoi"> DOI: 10.1038/bcj.2015.38</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fbcj.2015.38" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=25978432" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=e312&author=T.+Hideshimaauthor=F.+Cottiniauthor=H.+Ohguchiauthor=J.+Jakubikovaauthor=G.+Gorgunauthor=N.+Mimuraauthor=Y.+T.+Taiauthor=N.+C.+Munshiauthor=P.+G.+Richardsonauthor=K.+C.+Anderson&title=Rational+Combination+Treatment+with+Histone+Deacetylase+Inhibitors+and+Immunomodulatory+Drugs+in+Multiple+Myeloma&doi=10.1038%2Fbcj.2015.38"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fbcj.2015.38&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbcj.2015.38%26sid%3Dliteratum%253Aachs%26aulast%3DHideshima%26aufirst%3DT.%26aulast%3DCottini%26aufirst%3DF.%26aulast%3DOhguchi%26aufirst%3DH.%26aulast%3DJakubikova%26aufirst%3DJ.%26aulast%3DGorgun%26aufirst%3DG.%26aulast%3DMimura%26aufirst%3DN.%26aulast%3DTai%26aufirst%3DY.%2BT.%26aulast%3DMunshi%26aufirst%3DN.%2BC.%26aulast%3DRichardson%26aufirst%3DP.%2BG.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26atitle%3DRational%2520Combination%2520Treatment%2520with%2520Histone%2520Deacetylase%2520Inhibitors%2520and%2520Immunomodulatory%2520Drugs%2520in%2520Multiple%2520Myeloma%26jtitle%3DBlood%2520Cancer%2520J.%26date%3D2015%26volume%3D5%26spage%3De312%26doi%3D10.1038%2Fbcj.2015.38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heider, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzler, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Von Kaiser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosche, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rötzer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rademacher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakob, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleissner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuckelkorn, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kloetzel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sezer, O.</span></span> <span> </span><span class="NLM_article-title">Synergistic Interaction of the Histone Deacetylase Inhibitor SAHA with the Proteasome Inhibitor Bortezomib in Mantle Cell Lymphoma</span>. <i>Eur. J. Haematol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">142</span>, <span class="refDoi"> DOI: 10.1111/j.1600-0609.2007.00995.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1111%2Fj.1600-0609.2007.00995.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=18005386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD1cXivFahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2008&pages=133-142&author=U.+Heiderauthor=I.+Metzlerauthor=M.+Von+Kaiserauthor=M.+Roscheauthor=J.+Sterzauthor=S.+R%C3%B6tzerauthor=J.+Rademacherauthor=C.+Jakobauthor=C.+Fleissnerauthor=U.+Kuckelkornauthor=P.+Kloetzelauthor=O.+Sezer&title=Synergistic+Interaction+of+the+Histone+Deacetylase+Inhibitor+SAHA+with+the+Proteasome+Inhibitor+Bortezomib+in+Mantle+Cell+Lymphoma&doi=10.1111%2Fj.1600-0609.2007.00995.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Synergistic interaction of the histone deacetylase inhibitor SAHA with the proteasome inhibitor bortezomib in mantle cell lymphoma</span></div><div class="casAuthors">Heider, Ulrike; von Metzler, Ivana; Kaiser, Martin; Rosche, Marleen; Sterz, Jan; Roetzer, Susanne; Rademacher, Jessica; Jakob, Christian; Fleissner, Claudia; Kuckelkorn, Ulrike; Kloetzel, Peter-Michael; Sezer, Orhan</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Haematology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-142</span>CODEN:
                <span class="NLM_cas:coden">EJHAEC</span>;
        ISSN:<span class="NLM_cas:issn">0902-4441</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Mantle cell lymphoma (MCL) is an incurable B cell lymphoma, and novel treatment strategies are urgently needed.  We evaluated the effects of combined treatment with the proteasome inhibitor bortezomib and the histone deacetylase inhibitor (HDACi) suberoylanilide hydroxamic acid (SAHA) on MCL.  Bortezomib acts by targeting the proteasome, and - among other mechanisms - results in a reduced nuclear factor-kappa B (NF-κB) activity.  HDACi promote histone acetylation, and also interfere with NF-κB signaling.  Human MCL cell lines (JeKo-1, Granta-519 and Hbl-2) were exposed to bortezomib and/or SAHA.  Cell viability and apoptosis were quantified by the MTT and annexin-V assay, resp.  Reactive oxygen species (ROS) were analyzed using the fluorophore H2DCFDA.  In addn., activated caspases, proteasome- and NF-κB activity were quantified.  Combined incubation with bortezomib and SAHA resulted in synergistic cytotoxic effects, as indicated by combination index values <1 using the median effect method of Chou and Talalay.  The combination of both inhibitors led to a strong increase in apoptosis as compared to single agents and was accompanied by enhanced ROS generation, while each agent alone only modestly induced ROS.  The free radical scavenger N-acetyl-l-cysteine blocked the ROS generation and reduced the apoptosis significantly.  In addn., coexposure of bortezomib and SAHA led to increased caspase-3, -8, and -9 activity, marked redn. of proteasome activity and decrease of NF-κB activity.  This is the first report giving evidence that SAHA and bortezomib synergistically induce apoptosis in MCL cells.  These data build the framework for clin. trials using combined proteasome and histone deacetylase inhibition in the treatment of MCL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopNeVtuIe4nrVg90H21EOLACvtfcHk0lj4DWyMCQxogA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXivFahsbg%253D&md5=2e251f33843ecad588e8aa8ca20f4984</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1111%2Fj.1600-0609.2007.00995.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1600-0609.2007.00995.x%26sid%3Dliteratum%253Aachs%26aulast%3DHeider%26aufirst%3DU.%26aulast%3DMetzler%26aufirst%3DI.%26aulast%3DVon%2BKaiser%26aufirst%3DM.%26aulast%3DRosche%26aufirst%3DM.%26aulast%3DSterz%26aufirst%3DJ.%26aulast%3DR%25C3%25B6tzer%26aufirst%3DS.%26aulast%3DRademacher%26aufirst%3DJ.%26aulast%3DJakob%26aufirst%3DC.%26aulast%3DFleissner%26aufirst%3DC.%26aulast%3DKuckelkorn%26aufirst%3DU.%26aulast%3DKloetzel%26aufirst%3DP.%26aulast%3DSezer%26aufirst%3DO.%26atitle%3DSynergistic%2520Interaction%2520of%2520the%2520Histone%2520Deacetylase%2520Inhibitor%2520SAHA%2520with%2520the%2520Proteasome%2520Inhibitor%2520Bortezomib%2520in%2520Mantle%2520Cell%2520Lymphoma%26jtitle%3DEur.%2520J.%2520Haematol.%26date%3D2008%26volume%3D80%26spage%3D133%26epage%3D142%26doi%3D10.1111%2Fj.1600-0609.2007.00995.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yee, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bensinger, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supko, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voorhees, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berdeja, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Libby, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birrer, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamang, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wheeler, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markelewicz, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, N. S.</span></span> <span> </span><span class="NLM_article-title">Ricolinostat plus Lenalidomide, and Dexamethasone in Relapsed or Refractory Multiple Myeloma: A Multicentre Phase 1b Trial</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1569</span>– <span class="NLM_lpage">1578</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(16)30375-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1016%2FS1470-2045%2816%2930375-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=27646843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhsl2isb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=1569-1578&author=A.+J.+Yeeauthor=W.+I.+Bensingerauthor=J.+G.+Supkoauthor=P.+M.+Voorheesauthor=J.+G.+Berdejaauthor=P.+G.+Richardsonauthor=E.+N.+Libbyauthor=E.+E.+Wallaceauthor=N.+E.+Birrerauthor=J.+N.+Burkeauthor=D.+L.+Tamangauthor=M.+Yangauthor=S.+S.+Jonesauthor=C.+A.+Wheelerauthor=R.+J.+Markelewiczauthor=N.+S.+Raje&title=Ricolinostat+plus+Lenalidomide%2C+and+Dexamethasone+in+Relapsed+or+Refractory+Multiple+Myeloma%3A+A+Multicentre+Phase+1b+Trial&doi=10.1016%2FS1470-2045%2816%2930375-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Ricolinostat plus lenalidomide, and dexamethasone in relapsed or refractory multiple myeloma: a multicentre phase 1b trial</span></div><div class="casAuthors">Yee, Andrew J.; Bensinger, William I.; Supko, Jeffrey G.; Voorhees, Peter M.; Berdeja, Jesus G.; Richardson, Paul G.; Libby, Edward N.; Wallace, Ellen E.; Birrer, Nicole E.; Burke, Jill N.; Tamang, David L.; Yang, Min; Jones, Simon S.; Wheeler, Catherine A.; Markelewicz, Robert J.; Raje, Noopur S.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1569-1578</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Histone deacetylase (HDAC) inhibitors are an important new class of therapeutics for treating multiple myeloma.  Ricolinostat (ACY-1215) is the first oral selective HDAC6 inhibitor with reduced class I HDAC activity to be studied clin.  Motivated by findings from preclin. studies showing potent synergistic activity with ricolinostat and lenalidomide, our goal was to assess the safety and preliminary activity of the combination of ricolinostat with lenalidomide and dexamethasone in relapsed or refractory multiple myeloma.  In this multicenter phase 1b trial, we recruited patients aged 18 years or older with previously treated relapsed or refractory multiple myeloma from five cancer centers in the USA.  Inclusion criteria included a Karnofsky Performance Status score of at least 70, measureable disease, adequate bone marrow reserve, adequate hepatic function, and a creatinine clearance of at least 50 mL per min.  Exclusion criteria included previous exposure to HDAC inhibitors; previous allogeneic stem-cell transplantation; previous autologous stem-cell transplantation within 12 wk of baseline; active systemic infection; malignancy within the last 5 years; known or suspected HIV, hepatitis B, or hepatitis C infection; a QTc Fridericia of more than 480 ms; and substantial cardiovascular, gastrointestinal, psychiatric, or other medical disorders.  We gave escalating doses (from 40-240 mg once daily to 160 mg twice daily) of oral ricolinostat according to a std. 3 + 3 design according to three different regimens on days 1-21 with a conventional 28 day schedule of oral lenalidomide (from 15 mg [in one cohort] to 25 mg [in all other cohorts] once daily) and oral dexamethasone (40 mg weekly).  Primary outcomes were dose-limiting toxicities, the max. tolerated dose of ricolinostat in this combination, and the dose and schedule of ricolinostat recommended for further phase 2 investigation.  Secondary outcomes were the pharmacokinetics and pharmacodynamics of ricolinostat in this combination and the preliminary anti-tumor activity of this treatment.  The trial is closed to accrual and is registered at ClinicalTrials.gov, no. NCT01583283.  Between July 12, 2012, and Aug 20, 2015, we enrolled 38 patients.  We obsd. two dose-limiting toxicities with ricolinostat 160 mg twice daily: one (2%) grade 3 syncope and one (2%) grade 3 myalgia event in different cohorts.  A max. tolerated dose was not reached.  We chose ricolinostat 160 mg once daily on days 1-21 of a 28 day cycle as the recommended dose for future phase 2 studies in combination with lenalidomide 25 mg and dexamethasone 40 mg.  The most common adverse events were fatigue (grade 1-2 in 14 [37%] patients; grade 3 in seven [18%]) and diarrhea (grade 1-2 in 15 [39%] patients; grade 3 in two [5%]).  Our pharmacodynamic studies showed that at clin. relevant doses, ricolinostat selectively inhibits HDAC6 while retaining a low and tolerable level of class I HDAC inhibition.  The pharmacokinetics of ricolinostat and lenalidomide were not affected by co-administration.  In a preliminary assessment of antitumor activity, 21 (55% [95% CI 38-71]) of 38 patients had an overall response.  The findings from this study provide preliminary evidence that ricolinostat is a safe and well tolerated selective HDAC6 inhibitor, which might partner well with lenalidomide and dexamethasone to enhance their efficacy in relapsed or refractory multiple myeloma.Acetylon Pharmaceuticals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWXWsO-vNKY7Vg90H21EOLACvtfcHk0lj4DWyMCQxogA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhsl2isb7J&md5=59fea63c11b7bce7be6bd74f195af04b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2816%2930375-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252816%252930375-8%26sid%3Dliteratum%253Aachs%26aulast%3DYee%26aufirst%3DA.%2BJ.%26aulast%3DBensinger%26aufirst%3DW.%2BI.%26aulast%3DSupko%26aufirst%3DJ.%2BG.%26aulast%3DVoorhees%26aufirst%3DP.%2BM.%26aulast%3DBerdeja%26aufirst%3DJ.%2BG.%26aulast%3DRichardson%26aufirst%3DP.%2BG.%26aulast%3DLibby%26aufirst%3DE.%2BN.%26aulast%3DWallace%26aufirst%3DE.%2BE.%26aulast%3DBirrer%26aufirst%3DN.%2BE.%26aulast%3DBurke%26aufirst%3DJ.%2BN.%26aulast%3DTamang%26aufirst%3DD.%2BL.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DS.%2BS.%26aulast%3DWheeler%26aufirst%3DC.%2BA.%26aulast%3DMarkelewicz%26aufirst%3DR.%2BJ.%26aulast%3DRaje%26aufirst%3DN.%2BS.%26atitle%3DRicolinostat%2520plus%2520Lenalidomide%252C%2520and%2520Dexamethasone%2520in%2520Relapsed%2520or%2520Refractory%2520Multiple%2520Myeloma%253A%2520A%2520Multicentre%2520Phase%25201b%2520Trial%26jtitle%3DLancet%2520Oncol.%26date%3D2016%26volume%3D17%26spage%3D1569%26epage%3D1578%26doi%3D10.1016%2FS1470-2045%2816%2930375-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khoury, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Washington, L. B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keating, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D.</span></span> <span> </span><span class="NLM_article-title">A Systematic Approach to Diagnosis of Mature T-Cell Leukemias Reveals Heterogeneity among WHO Categories</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">328</span>– <span class="NLM_lpage">335</span>, <span class="refDoi"> DOI: 10.1182/blood-2004-01-0002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1182%2Fblood-2004-01-0002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=15044256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvFSnsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2004&pages=328-335&author=M.+Herlingauthor=J.+D.+Khouryauthor=L.+B.+T.+Washingtonauthor=M.+Duvicauthor=M.+J.+Keatingauthor=D.+Jones&title=A+Systematic+Approach+to+Diagnosis+of+Mature+T-Cell+Leukemias+Reveals+Heterogeneity+among+WHO+Categories&doi=10.1182%2Fblood-2004-01-0002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A systematic approach to diagnosis of mature T-cell leukemias reveals heterogeneity among WHO categories</span></div><div class="casAuthors">Herling, Marco; Khoury, Joseph D.; Washington, LaBaron T.; Duvic, Madeleine; Keating, Michael J.; Jones, Dan</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">328-335</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The current World Health Organization (WHO) classification of hematopoietic malignancies defines several types of mature T-cell leukemia including T-cell prolymphocytic leukemia (T-PLL), Sezary syndrome (SS), and T-cell large granular lymphocytic (T-LGL) leukemia.  These neoplasms can show overlapping features with each other and with T-cell lymphomas involving peripheral blood (PB).  We analyzed the spectrum of clinicopathol. features in 102 mature T-cell leukemias and compared them to 10 hepatosplenic T-cell lymphomas that involved PB.  T-PLL, defined as a T-cell leukemia showing rapidly rising PB lymphocyte counts, was the only tumor type expressing the oncoprotein TCL1 (71% of cases) and could present with relatively low lymphocyte levels or small tumor cell morphol.  SS, defined by accompanying erythrodermic skin disease, was frequently assocd. with peripheral eosinophilia but could also develop high nos. of prolymphocytes, esp. late in the disease course.  T-LGL leukemia, defined by accompanying cytopenias or autoimmune phenomena (or both), had the best clin. outcome and generally showed the lowest circulating lymphocyte levels with only a few cases developing marked lymphocytosis.  Using the dominant clin. or phenotypic feature, we describe here the degree of overlap among currently recognized WHO categories and identify areas where further clarification is needed.  Our results indicate that incorporation of addnl. criteria, such as TCL1 expression status and hematol. parameters, can assist in a more accurate classification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS0GwbkDkfirVg90H21EOLACvtfcHk0lgZbeukQDcKMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvFSnsbc%253D&md5=7fa603b6f7fb2cf31c14a978107ab173</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1182%2Fblood-2004-01-0002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2004-01-0002%26sid%3Dliteratum%253Aachs%26aulast%3DHerling%26aufirst%3DM.%26aulast%3DKhoury%26aufirst%3DJ.%2BD.%26aulast%3DWashington%26aufirst%3DL.%2BB.%2BT.%26aulast%3DDuvic%26aufirst%3DM.%26aulast%3DKeating%26aufirst%3DM.%2BJ.%26aulast%3DJones%26aufirst%3DD.%26atitle%3DA%2520Systematic%2520Approach%2520to%2520Diagnosis%2520of%2520Mature%2520T-Cell%2520Leukemias%2520Reveals%2520Heterogeneity%2520among%2520WHO%2520Categories%26jtitle%3DBlood%26date%3D2004%26volume%3D104%26spage%3D328%26epage%3D335%26doi%3D10.1182%2Fblood-2004-01-0002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stengel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zenger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perglerová, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnittger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haferlach, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haferlach, C.</span></span> <span> </span><span class="NLM_article-title">Genetic Characterization of T-PLL Reveals Two Major Biologic Subgroups and <i>JAK3</i> Mutations as Prognostic Marker</span>. <i>Genes, Chromosomes Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">82</span>– <span class="NLM_lpage">94</span>, <span class="refDoi"> DOI: 10.1002/gcc.22313</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1002%2Fgcc.22313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=26493028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs12qtL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=82-94&author=A.+Stengelauthor=W.+Kernauthor=M.+Zengerauthor=K.+Perglerov%C3%A1author=S.+Schnittgerauthor=T.+Haferlachauthor=C.+Haferlach&title=Genetic+Characterization+of+T-PLL+Reveals+Two+Major+Biologic+Subgroups+and+JAK3+Mutations+as+Prognostic+Marker&doi=10.1002%2Fgcc.22313"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic characterization of T-PLL reveals two major biologic subgroups and JAK3 mutations as prognostic marker</span></div><div class="casAuthors">Stengel, Anna; Kern, Wolfgang; Zenger, Melanie; Perglerova, Karolina; Schnittger, Susanne; Haferlach, Torsten; Haferlach, Claudia</div><div class="citationInfo"><span class="NLM_cas:title">Genes, Chromosomes & Cancer</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">82-94</span>CODEN:
                <span class="NLM_cas:coden">GCCAES</span>;
        ISSN:<span class="NLM_cas:issn">1045-2257</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">T-cell prolymphocytic leukemia (T-PLL) is a rare post-thymic T-cell neoplasm with aggressive clin. course and short overall survival.  So far, due to the rareness of this disease, genetic data are available only from individual cases or small cohorts.  In our study, we aimed at performing a comprehensive cytogenetic and mol. genetic characterization of T-PLL comprising the largest cohort of patients with T-PLL analyzed so far, including correlations between the resp. markers and their impact on prognosis.  Genetic abnormalities were found in all 51 cases with T-PLL, most frequently involving the TCRA/D locus (86%).  Deletions were detected for ATM (69%) and TP53 (31%), whereas i(8)(q10) was obsd. in 61% of cases.  Mutations in ATM, TP53, JAK1, and JAK3 were detected in 73, 14, 6, and 21% of patients, resp.  Addnl., BCOR mutations were obsd. for the first time in a lymphoid malignancy (8%).  Two distinct genetic subgroups of T-PLL were identified: A large subset (86% of patients) showed abnormalities involving the TCRA/D locus activating the proto-oncogenes TCL1 or MTCP1, while the second group was characterized by a high frequency of TP53 mutations (4/7 cases).  Further, analyses of overall survival identified JAK3 mutations as important prognostic marker, showing a significant neg. impact. © 2015 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ohuCRXjR9LVg90H21EOLACvtfcHk0lgZbeukQDcKMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs12qtL7F&md5=082cc0079a3832dc732c4350c0fc4b33</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fgcc.22313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fgcc.22313%26sid%3Dliteratum%253Aachs%26aulast%3DStengel%26aufirst%3DA.%26aulast%3DKern%26aufirst%3DW.%26aulast%3DZenger%26aufirst%3DM.%26aulast%3DPerglerov%25C3%25A1%26aufirst%3DK.%26aulast%3DSchnittger%26aufirst%3DS.%26aulast%3DHaferlach%26aufirst%3DT.%26aulast%3DHaferlach%26aufirst%3DC.%26atitle%3DGenetic%2520Characterization%2520of%2520T-PLL%2520Reveals%2520Two%2520Major%2520Biologic%2520Subgroups%2520and%2520JAK3%2520Mutations%2520as%2520Prognostic%2520Marker%26jtitle%3DGenes%252C%2520Chromosomes%2520Cancer%26date%3D2016%26volume%3D55%26spage%3D82%26epage%3D94%26doi%3D10.1002%2Fgcc.22313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crispatzu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oberbeck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pützer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Von Jan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasyutina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weit, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pflug, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riabinska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, M. S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altmuller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanasa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haferlach, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stilgenbauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopfinger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peifer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brummendorf, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurnberg, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elenitoba-Johnson, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriggl, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinhardt, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, m. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustjoki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newrzela, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frommolt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span> <span> </span><span class="NLM_article-title">Actionable Perturbations of Damage Responses by TCL1/ATM and Epigenetic Lesions Form the Basis of T-PLL</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">22</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-02688-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fs41467-017-02688-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=29317637" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVehtLnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1-22&author=A.+Schraderauthor=G.+Crispatzuauthor=S.+Oberbeckauthor=P.+Mayerauthor=S.+P%C3%BCtzerauthor=J.+Von+Janauthor=E.+Vasyutinaauthor=K.+Warnerauthor=N.+Weitauthor=N.+Pflugauthor=T.+Braunauthor=E.+I.+Anderssonauthor=B.+Yadavauthor=A.+Riabinskaauthor=M.+S.+V.+Ferreiraauthor=F.+Beierauthor=J.+Altmullerauthor=M.+Lanasaauthor=C.+D.+Herlingauthor=T.+Haferlachauthor=S.+Stilgenbauerauthor=G.+Hopfingerauthor=M.+Peiferauthor=T.+H.+Brummendorfauthor=P.+Nurnbergauthor=K.+S.+Elenitoba-Johnsonauthor=S.+Zhaauthor=M.+Hallekauthor=R.+Morigglauthor=H.+C.+Reinhardtauthor=m.+H.+Sternauthor=S.+Mustjokiauthor=S.+Newrzelaauthor=P.+Frommoltauthor=M.+Herling&title=Actionable+Perturbations+of+Damage+Responses+by+TCL1%2FATM+and+Epigenetic+Lesions+Form+the+Basis+of+T-PLL&doi=10.1038%2Fs41467-017-02688-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A sustained change in the supply of parental care causes adaptive evolution of offspring morphology</span></div><div class="casAuthors">Jarrett, Benjamin J. M.; Evans, Emma; Haynes, Hannah B.; Leaf, Miranda R.; Rebar, Darren; Duarte, Ana; Schrader, Matthew; Kilner, Rebecca M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-8</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Although cooperative social interactions within species are considered an important driver of evolutionary change, few studies have exptl. demonstrated that they cause adaptive evolution.  Here we address this problem by studying the burying beetle Nicrophorus vespilloides.  In this species, parents and larvae work together to obtain nourishment for larvae from the carrion breeding resource: parents feed larvae and larvae also self-feed.  We established exptl. evolving populations in which we varied the assistance that parents provided for their offspring and investigated how offspring evolved in response.  We show that in populations where parents predictably supplied more care, larval mandibles evolved to be smaller in relation to larval mass, and larvae were correspondingly less self-sufficient.  Previous work has shown that antagonistic social interactions can generate escalating evolutionary arms races.  Our study shows that cooperative interactions can yield the opposite evolutionary outcome: when one party invests more, the other evolves to invest less.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbNTa4adWXq7Vg90H21EOLACvtfcHk0liU6NrP59fmpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVehtLnI&md5=83d8d47cb89b14cb513c9d0f8a1039bb</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-02688-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-02688-6%26sid%3Dliteratum%253Aachs%26aulast%3DSchrader%26aufirst%3DA.%26aulast%3DCrispatzu%26aufirst%3DG.%26aulast%3DOberbeck%26aufirst%3DS.%26aulast%3DMayer%26aufirst%3DP.%26aulast%3DP%25C3%25BCtzer%26aufirst%3DS.%26aulast%3DVon%2BJan%26aufirst%3DJ.%26aulast%3DVasyutina%26aufirst%3DE.%26aulast%3DWarner%26aufirst%3DK.%26aulast%3DWeit%26aufirst%3DN.%26aulast%3DPflug%26aufirst%3DN.%26aulast%3DBraun%26aufirst%3DT.%26aulast%3DAndersson%26aufirst%3DE.%2BI.%26aulast%3DYadav%26aufirst%3DB.%26aulast%3DRiabinska%26aufirst%3DA.%26aulast%3DFerreira%26aufirst%3DM.%2BS.%2BV.%26aulast%3DBeier%26aufirst%3DF.%26aulast%3DAltmuller%26aufirst%3DJ.%26aulast%3DLanasa%26aufirst%3DM.%26aulast%3DHerling%26aufirst%3DC.%2BD.%26aulast%3DHaferlach%26aufirst%3DT.%26aulast%3DStilgenbauer%26aufirst%3DS.%26aulast%3DHopfinger%26aufirst%3DG.%26aulast%3DPeifer%26aufirst%3DM.%26aulast%3DBrummendorf%26aufirst%3DT.%2BH.%26aulast%3DNurnberg%26aufirst%3DP.%26aulast%3DElenitoba-Johnson%26aufirst%3DK.%2BS.%26aulast%3DZha%26aufirst%3DS.%26aulast%3DHallek%26aufirst%3DM.%26aulast%3DMoriggl%26aufirst%3DR.%26aulast%3DReinhardt%26aufirst%3DH.%2BC.%26aulast%3DStern%26aufirst%3Dm.%2BH.%26aulast%3DMustjoki%26aufirst%3DS.%26aulast%3DNewrzela%26aufirst%3DS.%26aulast%3DFrommolt%26aufirst%3DP.%26aulast%3DHerling%26aufirst%3DM.%26atitle%3DActionable%2520Perturbations%2520of%2520Damage%2520Responses%2520by%2520TCL1%252FATM%2520and%2520Epigenetic%2520Lesions%2520Form%2520the%2520Basis%2520of%2520T-PLL%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26spage%3D1%26epage%3D22%26doi%3D10.1038%2Fs41467-017-02688-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teitell, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konopleva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravandi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D.</span></span> <span> </span><span class="NLM_article-title">High TCL1 Expression and Intact T-Cell Receptor Signaling Define a Hyperproliferative Subset of T-Cell Prolymphocytic Leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">328</span>– <span class="NLM_lpage">337</span>, <span class="refDoi"> DOI: 10.1182/blood-2007-07-101519</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1182%2Fblood-2007-07-101519" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=17890451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtVKnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2008&pages=328-337&author=M.+Herlingauthor=K.+A.+Patelauthor=M.+A.+Teitellauthor=M.+Konoplevaauthor=F.+Ravandiauthor=R.+Kobayashiauthor=D.+Jones&title=High+TCL1+Expression+and+Intact+T-Cell+Receptor+Signaling+Define+a+Hyperproliferative+Subset+of+T-Cell+Prolymphocytic+Leukemia&doi=10.1182%2Fblood-2007-07-101519"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">High TCL1 expression and intact T-cell receptor signaling define a hyperproliferative subset of T-cell prolymphocytic leukemia</span></div><div class="casAuthors">Herling, Marco; Patel, Kaushali A.; Teitell, Michael A.; Konopleva, Marina; Ravandi, Farhad; Kobayashi, Ryuji; Jones, Dan</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">328-337</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The T-cell leukemia 1 (TCL1) oncoprotein is overexpressed by chromosomal rearrangement in the majority of cases of T-cell prolymphocytic leukemia (T-PLL).  In vitro, TCL1 can modulate the activity of the serine-threonine kinase AKT, a down-stream effector of T-cell receptor (TCR) signaling.  In a series of 86 T-PLL tumors, the authors show that expression of TCR, and levels of TCL1 and activated AKT are adverse prognostic markers.  High-level TCL1 in TCR-expressing T-PLL is assocd. with higher presenting white blood cell counts, faster tumor cell doubling, and enhanced in vitro growth response to TCR engagement.  In primary tumors and TCL1-transfected T-cell lines, TCR engagement leads to rapid recruitment of TCL1 and AKT to transient membrane activation complexes that include TCR-assocd. tyrosine kinases, including LCK.  Pharmacol. inhibition of AKT activation alters the localization, stability, and levels of these transient TCL1-AKT complexes and reduces tumor cell growth.  Exptl. introduction and knockdown of TCL1 influence the kinetics and strength of TCR-mediated AKT activation.  The authors propose that in T-PLL, TCL1 represents a highly regulated, targetable modulator of TCR-mediated AKT growth signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtog4z542fFbVg90H21EOLACvtfcHk0liU6NrP59fmpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtVKnsw%253D%253D&md5=d7a199cc58464e4e4ecba59239c457e2</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1182%2Fblood-2007-07-101519&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2007-07-101519%26sid%3Dliteratum%253Aachs%26aulast%3DHerling%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DK.%2BA.%26aulast%3DTeitell%26aufirst%3DM.%2BA.%26aulast%3DKonopleva%26aufirst%3DM.%26aulast%3DRavandi%26aufirst%3DF.%26aulast%3DKobayashi%26aufirst%3DR.%26aulast%3DJones%26aufirst%3DD.%26atitle%3DHigh%2520TCL1%2520Expression%2520and%2520Intact%2520T-Cell%2520Receptor%2520Signaling%2520Define%2520a%2520Hyperproliferative%2520Subset%2520of%2520T-Cell%2520Prolymphocytic%2520Leukemia%26jtitle%3DBlood%26date%3D2008%26volume%3D111%26spage%3D328%26epage%3D337%26doi%3D10.1182%2Fblood-2007-07-101519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kiel, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velusamy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolland, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahasrabuddhe, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozel, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betz, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medeiros, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elenitoba-Johnson, K. S. J.</span></span> <span> </span><span class="NLM_article-title">Integrated Genomic Sequencing Reveals Mutational Landscape of T-Cell Prolymphocytic Leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">1460</span>– <span class="NLM_lpage">1472</span>, <span class="refDoi"> DOI: 10.1182/blood-2014-03-559542</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1182%2Fblood-2014-03-559542" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=24825865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFOksbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2014&pages=1460-1472&author=M.+J.+Kielauthor=T.+Velusamyauthor=D.+Rollandauthor=A.+A.+Sahasrabuddheauthor=F.+Chungauthor=N.+G.+Baileyauthor=A.+Schraderauthor=B.+Liauthor=J.+Z.+Liauthor=A.+B.+Ozelauthor=B.+L.+Betzauthor=R.+N.+Mirandaauthor=L.+J.+Medeirosauthor=L.+Zhaoauthor=M.+Herlingauthor=M.+S.+Limauthor=K.+S.+J.+Elenitoba-Johnson&title=Integrated+Genomic+Sequencing+Reveals+Mutational+Landscape+of+T-Cell+Prolymphocytic+Leukemia&doi=10.1182%2Fblood-2014-03-559542"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated genomic sequencing reveals mutational landscape of T-cell prolymphocytic leukemia</span></div><div class="casAuthors">Kiel, Mark J.; Velusamy, Thirunavukkarasu; Rolland, Delphine; Sahasrabuddhe, Anagh A.; Chung, Fuzon; Bailey, Nathanael G.; Schrader, Alexandra; Li, Bo; Li, Jun Z.; Ozel, Ayse B.; Betz, Bryan L.; Miranda, Roberto N.; Medeiros, L. Jeffrey; Zhao, Lili; Herling, Marco; Lim, Megan S.; Elenitoba-Johnson, Kojo S. J.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1460-1472</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The comprehensive genetic alterations underlying the pathogenesis of T-cell prolymphocytic leukemia (T-PLL) are unknown.  To address this, we performed whole-genome sequencing (WGS), whole-exome sequencing (WES), high-resoln. copy-no. anal., and Sanger resequencing of a large cohort of T-PLL.  WGS and WES identified novel mutations in recurrently altered genes not previously implicated in T-PLL including EZH2, FBXW10, and CHEK2.  Strikingly, WGS and/or WES showed largely mutually exclusive mutations affecting IL2RG, JAK1, JAK3, or STAT5B in 38 of 50 T-PLL genomes (76.0%).  Notably, gain-of-function IL2RG mutations are novel and have not been reported in any form of cancer.  Further, high-frequency mutations in STAT5B have not been previously reported in T-PLL.  Functionally, IL2RG-JAK1-JAK3-STAT5B mutations led to signal transducer and activator of transcription 5 (STAT5) hyperactivation, transformed Ba/F3 cells resulting in cytokine-independent growth, and/or enhanced colony formation in Jurkat T cells.  Importantly, primary T-PLL cells exhibited constitutive activation of STAT5, and targeted pharmacol. inhibition of STAT5 with pimozide induced apoptosis in primary T-PLL cells.  These results for the first time provide a portrait of the mutational landscape of T-PLL and implicate deregulation of DNA repair and epigenetic modulators as well as high-frequency mutational activation of the IL2RG-JAK1-JAK3-STAT5B axis in the pathogenesis of T-PLL.  These findings offer opportunities for novel targeted therapies in this aggressive leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7rpUv0RHH5bVg90H21EOLACvtfcHk0lgl0vEQFYK7nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFOksbjN&md5=f199c9a7ef7b596f38de29347fab47db</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1182%2Fblood-2014-03-559542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2014-03-559542%26sid%3Dliteratum%253Aachs%26aulast%3DKiel%26aufirst%3DM.%2BJ.%26aulast%3DVelusamy%26aufirst%3DT.%26aulast%3DRolland%26aufirst%3DD.%26aulast%3DSahasrabuddhe%26aufirst%3DA.%2BA.%26aulast%3DChung%26aufirst%3DF.%26aulast%3DBailey%26aufirst%3DN.%2BG.%26aulast%3DSchrader%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DJ.%2BZ.%26aulast%3DOzel%26aufirst%3DA.%2BB.%26aulast%3DBetz%26aufirst%3DB.%2BL.%26aulast%3DMiranda%26aufirst%3DR.%2BN.%26aulast%3DMedeiros%26aufirst%3DL.%2BJ.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DHerling%26aufirst%3DM.%26aulast%3DLim%26aufirst%3DM.%2BS.%26aulast%3DElenitoba-Johnson%26aufirst%3DK.%2BS.%2BJ.%26atitle%3DIntegrated%2520Genomic%2520Sequencing%2520Reveals%2520Mutational%2520Landscape%2520of%2520T-Cell%2520Prolymphocytic%2520Leukemia%26jtitle%3DBlood%26date%3D2014%26volume%3D124%26spage%3D1460%26epage%3D1472%26doi%3D10.1182%2Fblood-2014-03-559542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wahnschaffe, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timonen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giri, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagle, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almusa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellanger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haferlach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durig, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siebrt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustjoki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aittokallio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span> <span> </span><span class="NLM_article-title">Jak/Stat-Activating Genomic Alterations Are a Hallmark of T-PLL</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1833</span>, <span class="refDoi"> DOI: 10.3390/cancers11121833</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.3390%2Fcancers11121833" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptVyiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=1833&author=L.+Wahnschaffeauthor=T.+Braunauthor=S.+Timonenauthor=A.+K.+Giriauthor=A.+Schraderauthor=P.+Wagleauthor=H.+Almusaauthor=P.+Johanssonauthor=D.+Bellangerauthor=C.+L%C3%B3pezauthor=C.+Haferlachauthor=M.+Sternauthor=J.+Durigauthor=R.+Siebrtauthor=S.+Mustjokiauthor=T.+Aittokallioauthor=M.+Herling&title=Jak%2FStat-Activating+Genomic+Alterations+Are+a+Hallmark+of+T-PLL&doi=10.3390%2Fcancers11121833"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">JAK/STAT-activating genomic alterations are a hallmark of T-PLL</span></div><div class="casAuthors">Wahnschaffe, Linus; Braun, Till; Timonen, Sanna; Giri, Anil K.; Schrader, Alexandra; Wagle, Prerana; Almusa, Henrikki; Johansson, Patricia; Bellanger, Dorine; Lopez, Cristina; Haferlach, Claudia; Stern, Marc-Henri; Duerig, Jan; Siebert, Reiner; Mustjoki, Satu; Aittokallio, Tero; Herling, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1833</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">T-cell prolymphocytic leukemia (T-PLL) is a rare and poor-prognostic mature T-cell leukemia.  Recent studies detected genomic aberrations affecting JAK and STAT genes in T-PLL.  Due to the limited no. of primary patient samples available, genomic analyses of the JAK/STAT pathway have been performed in rather small cohorts.  Therefore, we conducted-via a primary-data based pipeline-a meta-anal. that re-evaluated the genomic landscape of T-PLL.  It included all available data sets with sequence information on JAK or STAT gene loci in 275 T-PLL.  We eliminated overlapping cases and detd. a cumulative rate of 62.1% of cases with mutated JAK or STAT genes.  Most frequently, JAK1 (6.3%), JAK3 (36.4%), and STAT5B (18.8%) carried somatic single-nucleotide variants (SNVs), with missense mutations in the SH2 or pseudokinase domains as most prevalent.  Importantly, these lesions were predominantly subclonal.  We did not detect any strong assocn. between mutations of a JAK or STAT gene with clin. characteristics.  Irresp. of the presence of gain-of-function (GOF) SNVs, basal phosphorylation of STAT5B was elevated in all analyzed T-PLL.  Fittingly, a significant proportion of genes encoding for potential neg. regulators of STAT5B showed genomic losses (in 71.4% of T-PLL in total, in 68.4% of T-PLL without any JAK or STAT mutations).  They included DUSP4, CD45, TCPTP, SHP1, SOCS1, SOCS3, and HDAC9.  Overall, considering such losses of neg. regulators and the GOF mutations in JAK and STAT genes, a total of 89.8% of T-PLL revealed a genomic aberration potentially explaining enhanced STAT5B activity.  In essence, we present a comprehensive meta-anal. on the highly prevalent genomic lesions that affect genes encoding JAK/STAT signaling components.  This provides an overview of possible modes of activation of this pathway in a large cohort of T-PLL.  In light of new advances in JAK/STAT inhibitor development, we also outline translational contexts for harnessing active JAK/STAT signaling, which has emerged as a 'secondary' hallmark of T-PLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGria6G9Upve5rVg90H21EOLACvtfcHk0lgl0vEQFYK7nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptVyiurc%253D&md5=7d5ff1c2af7f605cd645c4f4e4d38cda</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.3390%2Fcancers11121833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers11121833%26sid%3Dliteratum%253Aachs%26aulast%3DWahnschaffe%26aufirst%3DL.%26aulast%3DBraun%26aufirst%3DT.%26aulast%3DTimonen%26aufirst%3DS.%26aulast%3DGiri%26aufirst%3DA.%2BK.%26aulast%3DSchrader%26aufirst%3DA.%26aulast%3DWagle%26aufirst%3DP.%26aulast%3DAlmusa%26aufirst%3DH.%26aulast%3DJohansson%26aufirst%3DP.%26aulast%3DBellanger%26aufirst%3DD.%26aulast%3DL%25C3%25B3pez%26aufirst%3DC.%26aulast%3DHaferlach%26aufirst%3DC.%26aulast%3DStern%26aufirst%3DM.%26aulast%3DDurig%26aufirst%3DJ.%26aulast%3DSiebrt%26aufirst%3DR.%26aulast%3DMustjoki%26aufirst%3DS.%26aulast%3DAittokallio%26aufirst%3DT.%26aulast%3DHerling%26aufirst%3DM.%26atitle%3DJak%252FStat-Activating%2520Genomic%2520Alterations%2520Are%2520a%2520Hallmark%2520of%2520T-PLL%26jtitle%3DCancers%26date%3D2019%26volume%3D11%26spage%3D1833%26doi%3D10.3390%2Fcancers11121833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Araujo, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdogan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubauer, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meneksedag-Erol, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manaswiyoungkul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qadree, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israelian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suske, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, H. T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boersma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tangermann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rulicke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravichandran, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audette, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauscher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriggl, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span> <span> </span><span class="NLM_article-title">Structural and Functional Consequences of the STAT5BN642H Driver Mutation</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2517</span>, <span class="refDoi"> DOI: 10.1038/s41467-019-10422-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fs41467-019-10422-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=31175292" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A280%3ADC%252BB3M3ktlyltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=2517&author=E.+D.+de%0AAraujoauthor=F.+Erdoganauthor=H.+A.+Neubauerauthor=D.+Meneksedag-Erolauthor=P.+Manaswiyoungkulauthor=M.+S.+Eramauthor=H.-S.+Seoauthor=A.+K.+Qadreeauthor=J.+Israelianauthor=A.+Orlovaauthor=T.+Suskeauthor=H.+T.+T.+Phamauthor=A.+Boersmaauthor=S.+Tangermannauthor=L.+Kennerauthor=T.+Rulickeauthor=A.+Dongauthor=M.+Ravichandranauthor=P.+J.+Brownauthor=G.+F.+Audetteauthor=S.+Rauscherauthor=S.+Dhe-Paganonauthor=R.+Morigglauthor=P.+T.+Gunning&title=Structural+and+Functional+Consequences+of+the+STAT5BN642H+Driver+Mutation&doi=10.1038%2Fs41467-019-10422-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and functional consequences of the STAT5B(N642H) driver mutation</span></div><div class="casAuthors">de Araujo Elvin D; Erdogan Fettah; Meneksedag-Erol Deniz; Manaswiyoungkul Pimyupa; Qadree Abdul K; Israelian Johan; Rauscher Sarah; Gunning Patrick T; de Araujo Elvin D; Erdogan Fettah; Manaswiyoungkul Pimyupa; Qadree Abdul K; Israelian Johan; Rauscher Sarah; Gunning Patrick T; Neubauer Heidi A; Orlova Anna; Suske Tobias; Pham Ha T T; Moriggl Richard; Neubauer Heidi A; Orlova Anna; Pham Ha T T; Kenner Lukas; Moriggl Richard; Meneksedag-Erol Deniz; Rauscher Sarah; Eram Mohammad S; Seo Hyuk-Soo; Dhe-Paganon Sirano; Seo Hyuk-Soo; Dhe-Paganon Sirano; Boersma Auke; Rulicke Thomas; Tangermann Simone; Kenner Lukas; Kenner Lukas; Dong Aiping; Ravichandran Manimekalai; Brown Peter J; Audette Gerald F; Moriggl Richard</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2517</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Hyper-activated STAT5B variants are high value oncology targets for pharmacologic intervention.  STAT5B(N642H), a frequently-occurring oncogenic driver mutation, promotes aggressive T-cell leukemia/lymphoma in patient carriers, although the molecular origins remain unclear.  Herein, we emphasize the aggressive nature of STAT5B(N642H) in driving T-cell neoplasia upon hematopoietic expression in transgenic mice, revealing evidence of multiple T-cell subset organ infiltration.  Notably, we demonstrate STAT5B(N642H)-driven transformation of γδ T-cells in in vivo syngeneic transplant models, comparable to STAT5B(N642H) patient γδ T-cell entities.  Importantly, we present human STAT5B and STAT5B(N642H) crystal structures, which propose alternative mutation-mediated SH2 domain conformations.  Our biophysical data suggests STAT5B(N642H) can adopt a hyper-activated and hyper-inactivated state with resistance to dephosphorylation.  MD simulations support sustained interchain cross-domain interactions in STAT5B(N642H), conferring kinetic stability to the mutant anti-parallel dimer.  This study provides a molecular explanation for the STAT5B(N642H) activating potential, and insights into pre-clinical models for targeted intervention of hyper-activated STAT5B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSf39tGczghJI3yKPtYViAGfW6udTcc2ebWdRJl0LERtbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3ktlyltw%253D%253D&md5=393a1fb7ed0316b22b04c8fde6e3ad54</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41467-019-10422-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-019-10422-7%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BAraujo%26aufirst%3DE.%2BD.%26aulast%3DErdogan%26aufirst%3DF.%26aulast%3DNeubauer%26aufirst%3DH.%2BA.%26aulast%3DMeneksedag-Erol%26aufirst%3DD.%26aulast%3DManaswiyoungkul%26aufirst%3DP.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DQadree%26aufirst%3DA.%2BK.%26aulast%3DIsraelian%26aufirst%3DJ.%26aulast%3DOrlova%26aufirst%3DA.%26aulast%3DSuske%26aufirst%3DT.%26aulast%3DPham%26aufirst%3DH.%2BT.%2BT.%26aulast%3DBoersma%26aufirst%3DA.%26aulast%3DTangermann%26aufirst%3DS.%26aulast%3DKenner%26aufirst%3DL.%26aulast%3DRulicke%26aufirst%3DT.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DRavichandran%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DAudette%26aufirst%3DG.%2BF.%26aulast%3DRauscher%26aufirst%3DS.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DMoriggl%26aufirst%3DR.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26atitle%3DStructural%2520and%2520Functional%2520Consequences%2520of%2520the%2520STAT5BN642H%2520Driver%2520Mutation%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D2517%26doi%3D10.1038%2Fs41467-019-10422-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timonen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koschmieder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wennerberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustjoki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aittokallio, T.</span></span> <span> </span><span class="NLM_article-title">Patient-Customized Drug Combination Prediction and Testing for t-Cell Prolymphocytic Leukemia Patients</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">2407</span>– <span class="NLM_lpage">2418</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-17-3644</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1158%2F0008-5472.CAN-17-3644" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=29483097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosF2qtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2018&pages=2407-2418&author=L.+Heauthor=J.+Tangauthor=E.+I.+Anderssonauthor=S.+Timonenauthor=S.+Koschmiederauthor=K.+Wennerbergauthor=S.+Mustjokiauthor=T.+Aittokallio&title=Patient-Customized+Drug+Combination+Prediction+and+Testing+for+t-Cell+Prolymphocytic+Leukemia+Patients&doi=10.1158%2F0008-5472.CAN-17-3644"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Patient-customized drug combination prediction and testing for t-cell prolymphocytic leukemia patients</span></div><div class="casAuthors">He, Liye; Tang, Jing; Andersson, Emma I.; Timonen, Sanna; Koschmieder, Steffen; Wennerberg, Krister; Mustjoki, Satu; Aittokallio, Tero</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2407-2418</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The mol. pathways that drive cancer progression and treatment resistance are highly redundant and variable between individual patients with the same cancer type.  To tackle this complex rewiring of pathway cross-talk, personalized combination treatments targeting multiple cancer growth and survival pathways are required.  Here we implemented a computational-exptl. drug combination prediction and testing (DCPT) platform for efficient in silico prioritization and ex vivo testing in patient-derived samples to identify customized synergistic combinations for individual cancer patients.  DCPT used drug-target interaction networks to traverse the massive combinatorial search spaces among 218 compds. (a total of 23,653 pairwise combinations) and identified cancer-selective synergies by using differential single-compd. sensitivity profiles between patient cells and healthy controls, hence reducing the likelihood of toxic combination effects.  A polypharmacol.-based machine learning modeling and network visualization made use of baseline genomic and mol. profiles to guide patient-specific combination testing and clin. translation phases.  Using T-cell prolymphocytic leukemia (T-PLL) as a first case study, we show how the DCPT platform successfully predicted distinct synergistic combinations for each of the three T-PLL patients, each presenting with different resistance patterns and synergy mechanisms.  In total, 10 of 24 (42%) of selective combination predictions were exptl. confirmed to show synergy in patient-derived samples ex vivo.  The identified selective synergies among approved drugs, including tacrolimus and temsirolimus combined with BCL-2 inhibitor venetoclax, may offer novel drug repurposing opportunities for treating T-PLL.  Significance: An integrated use of functional drug screening combined with genomic and mol. profiling enables patient-customized prediction and testing of drug combination synergies for T-PLL patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokJsA7GQacZbVg90H21EOLACvtfcHk0ljuWAJBcze8jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosF2qtbo%253D&md5=a6a97c5c2e41d663aebcc028f1331e59</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-17-3644&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-17-3644%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DL.%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DAndersson%26aufirst%3DE.%2BI.%26aulast%3DTimonen%26aufirst%3DS.%26aulast%3DKoschmieder%26aufirst%3DS.%26aulast%3DWennerberg%26aufirst%3DK.%26aulast%3DMustjoki%26aufirst%3DS.%26aulast%3DAittokallio%26aufirst%3DT.%26atitle%3DPatient-Customized%2520Drug%2520Combination%2520Prediction%2520and%2520Testing%2520for%2520t-Cell%2520Prolymphocytic%2520Leukemia%2520Patients%26jtitle%3DCancer%2520Res.%26date%3D2018%26volume%3D78%26spage%3D2407%26epage%3D2418%26doi%3D10.1158%2F0008-5472.CAN-17-3644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dearden, C.</span></span> <span> </span><span class="NLM_article-title">How I Treat Prolymphocytic Leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">538</span>– <span class="NLM_lpage">551</span>, <span class="refDoi"> DOI: 10.1182/blood-2012-01-380139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1182%2Fblood-2012-01-380139" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=22649104" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFOntrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2012&pages=538-551&author=C.+Dearden&title=How+I+Treat+Prolymphocytic+Leukemia&doi=10.1182%2Fblood-2012-01-380139"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">How I treat prolymphocytic leukemia</span></div><div class="casAuthors">Dearden, Claire</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">538-551</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">A review.  T- and B-cell subtypes of prolymphocytic leukemia (PLL) are rare, aggressive lymphoid malignancies with characteristic morphol., immunophenotypic, cytogenetic, and mol. features.  Recent studies have highlighted the role of specific oncogenes, such as TCL-1, MTCP-1, and ATM in the case of T-cell and TP53 mutations in the case of B-cell prolymphocytic leukemia.  Despite the advances in the understanding of the biol. of these conditions, the prognosis for these patients remains poor with short survival and no curative therapy.  The advent of monoclonal antibodies has improved treatment options.  Currently, the best treatment for T-PLL is i.v. alemtuzumab, which has resulted in very high response rates of more than 90% when given as first-line treatment and a significant improvement in survival.  Consolidation of remissions with autologous or allogeneic stem cell transplantation further prolongs survival, and the latter may offer potential cure.  In B-PLL, rituximab-based combination chemo-immunotherapy is effective in fitter patients.  TP53 abnormalities are common and, as for chronic lymphocytic leukemia, these patients should be managed using an alemtuzumab-based therapy.  The role of allogeneic transplant with nonmyeloablative conditioning needs to be explored further in both T- and B-cell PLL to broaden the patient eligibility for what may be a curative treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlLkh6TX_EP7Vg90H21EOLACvtfcHk0ljuWAJBcze8jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFOntrjF&md5=7b3dbf767a348de953e7f91b151e34f1</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1182%2Fblood-2012-01-380139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2012-01-380139%26sid%3Dliteratum%253Aachs%26aulast%3DDearden%26aufirst%3DC.%26atitle%3DHow%2520I%2520Treat%2520Prolymphocytic%2520Leukemia%26jtitle%3DBlood%26date%3D2012%26volume%3D120%26spage%3D538%26epage%3D551%26doi%3D10.1182%2Fblood-2012-01-380139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hockly, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richon, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sathasivam, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghazi-Noori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowden, P. A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffan, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, G. P.</span></span> <span> </span><span class="NLM_article-title">Suberoylanilide Hydroxamic Acid, a Histone Deacetylase Inhibitor, Ameliorates Motor Deficits in a Mouse Model of Huntington’s Disease</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">2041</span>– <span class="NLM_lpage">2046</span>, <span class="refDoi"> DOI: 10.1073/pnas.0437870100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1073%2Fpnas.0437870100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=12576549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhsFGrsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2003&pages=2041-2046&author=E.+Hocklyauthor=V.+M.+Richonauthor=B.+Woodmanauthor=D.+L.+Smithauthor=X.+Zhouauthor=E.+Rosaauthor=K.+Sathasivamauthor=S.+Ghazi-Nooriauthor=A.+Mahalauthor=P.+A.+S.+Lowdenauthor=J.+S.+Steffanauthor=J.+L.+Marshauthor=L.+M.+Thompsonauthor=C.+M.+Lewisauthor=P.+A.+Marksauthor=G.+P.+Bates&title=Suberoylanilide+Hydroxamic+Acid%2C+a+Histone+Deacetylase+Inhibitor%2C+Ameliorates+Motor+Deficits+in+a+Mouse+Model+of+Huntington%E2%80%99s+Disease&doi=10.1073%2Fpnas.0437870100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Suberoylanilide hydroxamic acid, a histone deacetylase inhibitor, ameliorates motor deficits in a mouse model of Huntington's disease</span></div><div class="casAuthors">Hockly, Emma; Richon, Victoria M.; Woodman, Benjamin; Smith, Donna L.; Zhou, Xianbo; Rosa, Eddie; Sathasivam, Kirupa; Ghazi-Noori, Shabnam; Mahal, Amarbirpal; Lowden, Philip A. S.; Steffan, Joan S.; Marsh, J. Lawrence; Thompson, Leslie M.; Lewis, Cathryn M.; Marks, Paul A.; Bates, Gillian P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2041-2046</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is an inherited, progressive neurol. disorder that is caused by a CAG/polyglutamine repeat expansion and for which there is no effective therapy.  Recent evidence indicates that transcriptional dysregulation may contribute to the mol. pathogenesis of this disease.  Supporting this view, administration of histone deacetylase (HDAC) inhibitors has been shown to rescue lethality and photoreceptor neurodegeneration in a Drosophila model of polyglutamine disease.  To further explore the therapeutic potential of HDAC inhibitors, we have conducted preclin. trials with suberoylanilide hydroxamic acid (SAHA), a potent HDAC inhibitor, in the R6/2 HD mouse model.  We show that SAHA crosses the blood-brain barrier and increases histone acetylation in the brain.  We found that SAHA could be administered orally in drinking water when complexed with cyclodextrins.  SAHA dramatically improved the motor impairment in R6/2 mice, clearly validating the pursuit of this class of compds. as HD therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpy-3Y7VV7UbVg90H21EOLACvtfcHk0lhH35vkmTj69w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhsFGrsbs%253D&md5=4096dcbfeeeafd00c51ccb01e4ee29a5</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0437870100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0437870100%26sid%3Dliteratum%253Aachs%26aulast%3DHockly%26aufirst%3DE.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DWoodman%26aufirst%3DB.%26aulast%3DSmith%26aufirst%3DD.%2BL.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DRosa%26aufirst%3DE.%26aulast%3DSathasivam%26aufirst%3DK.%26aulast%3DGhazi-Noori%26aufirst%3DS.%26aulast%3DMahal%26aufirst%3DA.%26aulast%3DLowden%26aufirst%3DP.%2BA.%2BS.%26aulast%3DSteffan%26aufirst%3DJ.%2BS.%26aulast%3DMarsh%26aufirst%3DJ.%2BL.%26aulast%3DThompson%26aufirst%3DL.%2BM.%26aulast%3DLewis%26aufirst%3DC.%2BM.%26aulast%3DMarks%26aufirst%3DP.%2BA.%26aulast%3DBates%26aufirst%3DG.%2BP.%26atitle%3DSuberoylanilide%2520Hydroxamic%2520Acid%252C%2520a%2520Histone%2520Deacetylase%2520Inhibitor%252C%2520Ameliorates%2520Motor%2520Deficits%2520in%2520a%2520Mouse%2520Model%2520of%2520Huntington%25E2%2580%2599s%2520Disease%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2003%26volume%3D100%26spage%3D2041%26epage%3D2046%26doi%3D10.1073%2Fpnas.0437870100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakimoto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chumakov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koeffler, H. P.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylase Inhibitor, Suberoylanilide Hydroxamic Acid (Vorinostat, SAHA) Profoundly Inhibits the Growth of Human Pancreatic Cancer Cells</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>121</i></span>,  <span class="NLM_fpage">656</span>– <span class="NLM_lpage">665</span>, <span class="refDoi"> DOI: 10.1002/ijc.22558</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1002%2Fijc.22558" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=17417771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnt1Crsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2007&pages=656-665&author=T.+Kumagaiauthor=N.+Wakimotoauthor=D.+Yinauthor=S.+Geryauthor=N.+Kawamataauthor=N.+Takaiauthor=N.+Komatsuauthor=A.+Chumakovauthor=Y.+Imaiauthor=H.+P.+Koeffler&title=Histone+Deacetylase+Inhibitor%2C+Suberoylanilide+Hydroxamic+Acid+%28Vorinostat%2C+SAHA%29+Profoundly+Inhibits+the+Growth+of+Human+Pancreatic+Cancer+Cells&doi=10.1002%2Fijc.22558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase inhibitor, suberoylanilide hydroxamic acid (Vorinostat, SAHA) profoundly inhibits the growth of human pancreatic cancer cells</span></div><div class="casAuthors">Kumagai, Takashi; Wakimoto, Naoki; Yin, Dong; Gery, Sigal; Kawamata, Norihiko; Takai, Noriyuki; Komatsu, Naoki; Chumakov, Alexy; Imai, Yasufumi; Koeffler, H. Phillip</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">656-665</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Tumor suppressor genes are often silenced in human cancer; this can occur by transcriptional repression by deacetylation in the promoter regions, mediated by histone deacetylase (HDAC).  HDAC inhibitors can block cancer cell growth by restoring expression of tumor suppressor genes.  In this study, we investigated the effects of a HDAC inhibitor, suberoylanilide hydroxamic acid (SAHA) on pancreatic cancer cells.  SAHA inhibited the growth of 6 pancreatic cancer cell lines in a dose-dependent manner as measured by MTT and clonogenic assays (ED50 ≈10-6 M) assocd. with induction of apoptosis, G2 cell cycle arrest and also induced differentiation as indicated by morphol. and increased expression of cytokeratin 7.  It increased expression of p21WAF1 (independent of the mutational status of p53), C/EBPα, RARα and E-cadherin; these genes have been assocd. with decreased proliferation in other cancers.  SAHA decreased cyclin B1 expression; this cyclin normally promotes progression through G2 of the cell cycle.  SAHA mediated acetylation of histone H3 globally, as well as, assocd. with the p21WAF1 promoter, as measured by chromatin immunopptn.  SAHA also decreased levels of c-myc and cyclin D1, independent of an active β-catenin pathway.  In further studies, the combination of SAHA and an inhibitor of DNA methylation, 5-Aza-2'-deoxycytidine, had an enhanced antiproliferative effect on pancreatic cancer cells.  In summary, SAHA inhibited the growth of human pancreatic cancer cells by inducing apoptosis, differentiation and cell cycle arrest, as well as increase in the expression of several tumor suppressor genes.  SAHA is a novel, promising therapeutic agent for human pancreatic cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVgeJjEzQr77Vg90H21EOLACvtfcHk0lhH35vkmTj69w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnt1Crsrs%253D&md5=4c7224c82f75d160b934fb0560f73eb5</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fijc.22558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.22558%26sid%3Dliteratum%253Aachs%26aulast%3DKumagai%26aufirst%3DT.%26aulast%3DWakimoto%26aufirst%3DN.%26aulast%3DYin%26aufirst%3DD.%26aulast%3DGery%26aufirst%3DS.%26aulast%3DKawamata%26aufirst%3DN.%26aulast%3DTakai%26aufirst%3DN.%26aulast%3DKomatsu%26aufirst%3DN.%26aulast%3DChumakov%26aufirst%3DA.%26aulast%3DImai%26aufirst%3DY.%26aulast%3DKoeffler%26aufirst%3DH.%2BP.%26atitle%3DHistone%2520Deacetylase%2520Inhibitor%252C%2520Suberoylanilide%2520Hydroxamic%2520Acid%2520%2528Vorinostat%252C%2520SAHA%2529%2520Profoundly%2520Inhibits%2520the%2520Growth%2520of%2520Human%2520Pancreatic%2520Cancer%2520Cells%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2007%26volume%3D121%26spage%3D656%26epage%3D665%26doi%3D10.1002%2Fijc.22558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruefli, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ausserlechner, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tainton, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kofler, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span> <span> </span><span class="NLM_article-title">The Histone Deacetylase Inhibitor and Chemotherapeutic Agent Suberoylanilide Hydroxamic Acid (SAHA) Induces a Cell-Death Pathway Characterized by Cleavage of Bid and Production of Reactive Oxygen Species</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">10833</span>– <span class="NLM_lpage">10838</span>, <span class="refDoi"> DOI: 10.1073/pnas.191208598</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1073%2Fpnas.191208598" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=11535817" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntVGmsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=10833-10838&author=A.+A.+Ruefliauthor=M.+J.+Ausserlechnerauthor=D.+Bernhardauthor=V.+R.+Suttonauthor=K.+M.+Taintonauthor=R.+Koflerauthor=M.+J.+Smythauthor=R.+W.+Johnstone&title=The+Histone+Deacetylase+Inhibitor+and+Chemotherapeutic+Agent+Suberoylanilide+Hydroxamic+Acid+%28SAHA%29+Induces+a+Cell-Death+Pathway+Characterized+by+Cleavage+of+Bid+and+Production+of+Reactive+Oxygen+Species&doi=10.1073%2Fpnas.191208598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The histone deacetylase inhibitor and chemotherapeutic agent suberoylanilide hydroxamic acid (SAHA) induces a cell-death pathway characterized by cleavage of Bid and production of reactive oxygen species</span></div><div class="casAuthors">Ruefli, Astrid A.; Ausserlechner, Michael J.; Bernhard, David; Sutton, Vivien R.; Tainton, Kellie M.; Kofler, Reinhard; Smyth, Mark J.; Johnstone, Ricky W.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">10833-10838</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Many chemotherapeutic agents induce mitochondrial-membrane disruption to initiate apoptosis.  However, the upstream events leading to drug-induced mitochondrial perturbation have remained poorly defined.  We have used a variety of physiol. and pharmacol. inhibitors of distinct apoptotic pathways to analyze the manner by which suberoylanilide hydroxamic acid (SAHA), a chemotherapeutic agent and histone deacetylase inhibitor, induces cell death.  We demonstrate that SAHA initiates cell death by inducing mitochondria-mediated death pathways characterized by cytochrome c release and the prodn. of reactive oxygen species, and does not require the activation of key caspases such as caspase-8 or -3.  We provide evidence that mitochondrial disruption is achieved by means of the cleavage of the BH3-only proapoptotic Bcl-2 family member Bid.  SAHA-induced Bid cleavage was not blocked by caspase inhibitors or the overexpression of Bcl-2 but did require the transcriptional regulatory activity of SAHA.  These data provide evidence of a mechanism of cell death mediated by transcriptional events that result in the cleavage of Bid, disruption of the mitochondrial membrane, and prodn. of reactive oxygen species to induce cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKvHbIDNsJq7Vg90H21EOLACvtfcHk0lgtFp1gDOrr5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntVGmsLY%253D&md5=0b35cff7a6f2af9bb236e6c7ae942db9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1073%2Fpnas.191208598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.191208598%26sid%3Dliteratum%253Aachs%26aulast%3DRuefli%26aufirst%3DA.%2BA.%26aulast%3DAusserlechner%26aufirst%3DM.%2BJ.%26aulast%3DBernhard%26aufirst%3DD.%26aulast%3DSutton%26aufirst%3DV.%2BR.%26aulast%3DTainton%26aufirst%3DK.%2BM.%26aulast%3DKofler%26aufirst%3DR.%26aulast%3DSmyth%26aufirst%3DM.%2BJ.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26atitle%3DThe%2520Histone%2520Deacetylase%2520Inhibitor%2520and%2520Chemotherapeutic%2520Agent%2520Suberoylanilide%2520Hydroxamic%2520Acid%2520%2528SAHA%2529%2520Induces%2520a%2520Cell-Death%2520Pathway%2520Characterized%2520by%2520Cleavage%2520of%2520Bid%2520and%2520Production%2520of%2520Reactive%2520Oxygen%2520Species%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D10833%26epage%3D10838%26doi%3D10.1073%2Fpnas.191208598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwitkowski, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Valle, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricci, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saber, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habtemariam, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloomquist, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehrotra, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorff, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charlab, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kane, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaminskas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Justice, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrell, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazdur, R.</span></span> <span> </span><span class="NLM_article-title">FDA Approval: Belinostat for the Treatment of Patients with Relapsed or Refractory Peripheral T-Cell Lymphoma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2666</span>– <span class="NLM_lpage">2670</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-14-3119</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1158%2F1078-0432.CCR-14-3119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=25802282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVajurfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=2666-2670&author=H.+Z.+Leeauthor=V.+E.+Kwitkowskiauthor=P.+L.+Del+Valleauthor=M.+S.+Ricciauthor=H.+Saberauthor=B.+A.+Habtemariamauthor=J.+Bullockauthor=E.+Bloomquistauthor=Y.+L.+Shenauthor=X.+H.+Chenauthor=J.+Brownauthor=N.+Mehrotraauthor=S.+Dorffauthor=R.+Charlabauthor=R.+C.+Kaneauthor=E.+Kaminskasauthor=R.+Justiceauthor=A.+T.+Farrellauthor=R.+Pazdur&title=FDA+Approval%3A+Belinostat+for+the+Treatment+of+Patients+with+Relapsed+or+Refractory+Peripheral+T-Cell+Lymphoma&doi=10.1158%2F1078-0432.CCR-14-3119"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">FDA Approval: Belinostat for the Treatment of Patients with Relapsed or Refractory Peripheral T-cell Lymphoma</span></div><div class="casAuthors">Lee, Hyon-Zu; Kwitkowski, Virginia E.; Del Valle, Pedro L.; Ricci, M. Stacey; Saber, Haleh; Habtemariam, Bahru A.; Bullock, Julie; Bloomquist, Erik; Shen, Yuan Li; Chen, Xiao-Hong; Brown, Janice; Mehrotra, Nitin; Dorff, Sarah; Charlab, Rosane; Kane, Robert C.; Kaminskas, Edvardas; Justice, Robert; Farrell, Ann T.; Pazdur, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2666-2670</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">On July 3, 2014, the FDA granted accelerated approval for belinostat (Beleodaq; Spectrum Pharmaceuticals, Inc.), a histone deacetylase inhibitor, for the treatment of patients with relapsed or refractory peripheral T-cell lymphoma (PTCL).  A single-arm, open-label, multicenter, international trial in the indicated patient population was submitted in support of the application.  Belinostat was administered i.v. at a dose of 1000 mg/m2 over 30 min once daily on days 1 to 5 of a 21-day cycle.  The primary efficacy endpoint was overall response rate (ORR) based on central radiol. readings by an independent review committee.  The ORR was 25.8% [95% confidence interval (CI), 18.3-34.6] in 120 patients that had confirmed diagnoses of PTCL by the Central Pathol. Review Group.  The complete and partial response rates were 10.8% (95% CI, 5.9-17.8) and 15.0% (95% CI, 9.1-22.7), resp.  The median duration of response, the key secondary efficacy endpoint, was 8.4 mo (95% CI, 4.5-29.4).  The most common adverse reactions (>25%) were nausea, fatigue, pyrexia, anemia, and vomiting.  Grade 3/4 toxicities (≥5.0%) included anemia, thrombocytopenia, dyspnea, neutropenia, fatigue, and pneumonia.  Belinostat is the third drug to receive accelerated approval for the treatment of relapsed or refractory PTCL.  Clin Cancer Res; 21(12); 2666-70. ©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1RbkvuHSk6rVg90H21EOLACvtfcHk0lgtFp1gDOrr5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVajurfM&md5=c1e64c9d01d18272d25ad03001e7041a</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-3119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-3119%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%2BZ.%26aulast%3DKwitkowski%26aufirst%3DV.%2BE.%26aulast%3DDel%2BValle%26aufirst%3DP.%2BL.%26aulast%3DRicci%26aufirst%3DM.%2BS.%26aulast%3DSaber%26aufirst%3DH.%26aulast%3DHabtemariam%26aufirst%3DB.%2BA.%26aulast%3DBullock%26aufirst%3DJ.%26aulast%3DBloomquist%26aufirst%3DE.%26aulast%3DShen%26aufirst%3DY.%2BL.%26aulast%3DChen%26aufirst%3DX.%2BH.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DMehrotra%26aufirst%3DN.%26aulast%3DDorff%26aufirst%3DS.%26aulast%3DCharlab%26aufirst%3DR.%26aulast%3DKane%26aufirst%3DR.%2BC.%26aulast%3DKaminskas%26aufirst%3DE.%26aulast%3DJustice%26aufirst%3DR.%26aulast%3DFarrell%26aufirst%3DA.%2BT.%26aulast%3DPazdur%26aufirst%3DR.%26atitle%3DFDA%2520Approval%253A%2520Belinostat%2520for%2520the%2520Treatment%2520of%2520Patients%2520with%2520Relapsed%2520or%2520Refractory%2520Peripheral%2520T-Cell%2520Lymphoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26spage%3D2666%26epage%3D2670%26doi%3D10.1158%2F1078-0432.CCR-14-3119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pützer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varghese, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Jan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giri, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riet, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timonen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oberbeck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuusanmäki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustjoki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aittokallio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newrzela, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span> <span> </span><span class="NLM_article-title">Reinstated P53 Response and High Anti-T-Cell Leukemia Activity by the Novel Alkylating Deacetylase Inhibitor Tinostamustine</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">2513</span>– <span class="NLM_lpage">2518</span>, <span class="refDoi"> DOI: 10.1038/s41375-020-0772-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fs41375-020-0772-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=32099034" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2020&pages=2513-2518&author=S.+P%C3%BCtzerauthor=L.+Vargheseauthor=J.+von+Janauthor=T.+Braunauthor=A.+K.+Giriauthor=P.+Mayerauthor=N.+Rietauthor=S.+Timonenauthor=S.+Oberbeckauthor=H.+Kuusanm%C3%A4kiauthor=S.+Mustjokiauthor=M.+Sternauthor=T.+Aittokallioauthor=S.+Newrzelaauthor=A.+Schraderauthor=M.+Herling&title=Reinstated+P53+Response+and+High+Anti-T-Cell+Leukemia+Activity+by+the+Novel+Alkylating+Deacetylase+Inhibitor+Tinostamustine&doi=10.1038%2Fs41375-020-0772-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fs41375-020-0772-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41375-020-0772-6%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25BCtzer%26aufirst%3DS.%26aulast%3DVarghese%26aufirst%3DL.%26aulast%3Dvon%2BJan%26aufirst%3DJ.%26aulast%3DBraun%26aufirst%3DT.%26aulast%3DGiri%26aufirst%3DA.%2BK.%26aulast%3DMayer%26aufirst%3DP.%26aulast%3DRiet%26aufirst%3DN.%26aulast%3DTimonen%26aufirst%3DS.%26aulast%3DOberbeck%26aufirst%3DS.%26aulast%3DKuusanm%25C3%25A4ki%26aufirst%3DH.%26aulast%3DMustjoki%26aufirst%3DS.%26aulast%3DStern%26aufirst%3DM.%26aulast%3DAittokallio%26aufirst%3DT.%26aulast%3DNewrzela%26aufirst%3DS.%26aulast%3DSchrader%26aufirst%3DA.%26aulast%3DHerling%26aufirst%3DM.%26atitle%3DReinstated%2520P53%2520Response%2520and%2520High%2520Anti-T-Cell%2520Leukemia%2520Activity%2520by%2520the%2520Novel%2520Alkylating%2520Deacetylase%2520Inhibitor%2520Tinostamustine%26jtitle%3DLeukemia%26date%3D2020%26volume%3D34%26spage%3D2513%26epage%3D2518%26doi%3D10.1038%2Fs41375-020-0772-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pützer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufva, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crispatzu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oleś, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adnan-Awad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagstrom, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellanger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mpindi, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldfors, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pemovska, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietarinen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lauhio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuesta-Mateos, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koschmieder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brummendorf, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kytola, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savolainen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siitonen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellonen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallioneimi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wennerberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, M_H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aittokallio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zenz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustjoki, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel Drug Sensitivities in T-PLL by High-Throughput Ex Vivo Drug Testing and Mutation Profiling</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">774</span>– <span class="NLM_lpage">787</span>, <span class="refDoi"> DOI: 10.1038/leu.2017.252</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fleu.2017.252" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=28804127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKmu7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=774-787&author=E.+I.+Anderssonauthor=S.+P%C3%BCtzerauthor=B.+Yadavauthor=O.+Dufvaauthor=S.+Khanauthor=L.+Heauthor=L.+Sellnerauthor=A.+Schraderauthor=G.+Crispatzuauthor=M.+Ole%C5%9Bauthor=H.+Zhangauthor=S.+Adnan-Awadauthor=S.+Lagstromauthor=D.+Bellangerauthor=J.+P.+Mpindiauthor=S.+Eldforsauthor=T.+Pemovskaauthor=P.+Pietarinenauthor=A.+Lauhioauthor=K.+Tomskaauthor=C.+Cuesta-Mateosauthor=E.+Faberauthor=S.+Koschmiederauthor=T.+H.+Brummendorfauthor=S.+Kytolaauthor=E.+Savolainenauthor=T.+Siitonenauthor=P.+Ellonenauthor=O.+Kallioneimiauthor=K.+Wennerbergauthor=W.+Dingauthor=M_H.+Sternauthor=W.+Huberauthor=S.+Andersauthor=J.+Tangauthor=T.+Aittokallioauthor=T.+Zenzauthor=M.+Herlingauthor=S.+Mustjoki&title=Discovery+of+Novel+Drug+Sensitivities+in+T-PLL+by+High-Throughput+Ex+Vivo+Drug+Testing+and+Mutation+Profiling&doi=10.1038%2Fleu.2017.252"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel drug sensitivities in T-PLL by high-throughput ex vivo drug testing and mutation profiling</span></div><div class="casAuthors">Andersson, E. I.; Putzer, S.; Yadav, B.; Dufva, O.; Khan, S.; He, L.; Sellner, L.; Schrader, A.; Crispatzu, G.; Oles, M.; Zhang, H.; Adnan-Awad, S.; Lagstrom, S.; Bellanger, D.; Mpindi, J. P.; Eldfors, S.; Pemovska, T.; Pietarinen, P.; Lauhio, A.; Tomska, K.; Cuesta-Mateos, C.; Faber, E.; Koschmieder, S.; Brummendorf, T. H.; Kytola, S.; Savolainen, E.-R.; Siitonen, T.; Ellonen, P.; Kallioniemi, O.; Wennerberg, K.; Ding, W.; Stern, M.-H.; Huber, W.; Anders, S.; Tang, J.; Aittokallio, T.; Zenz, T.; Herling, M.; Mustjoki, S.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">774-787</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">T-cell prolymphocytic leukemia (T-PLL) is a rare and aggressive neoplasm of mature T-cells with an urgent need for rationally designed therapies to address its notoriously chemo-refractory behavior.  The median survival of T-PLL patients is <2 years and clin. trials are difficult to execute.  Here we systematically explored the diversity of drug responses in T-PLL patient samples using an ex vivo drug sensitivity and resistance testing platform and correlated the findings with somatic mutations and gene expression profiles.  Intriguingly, all T-PLL samples were sensitive to the cyclin-dependent kinase inhibitor SNS-032, which overcame stromal-cell-mediated protection and elicited robust p53-activation and apoptosis.  Across all patients, the most effective classes of compds. were histone deacetylase, phosphoinositide-3 kinase/AKT/mammalian target of rapamycin, heat-shock protein 90 and BH3-family protein inhibitors as well as p53 activators, indicating previously unexplored, novel targeted approaches for treating T-PLL.  Although Janus-activated kinase-signal transducer and activator of transcription factor (JAK-STAT) pathway mutations were common in T-PLL (71% of patients), JAK-STAT inhibitor responses were not directly linked to those or other T-PLL-specific lesions.  Overall, we found that genetic markers do not readily translate into novel effective therapeutic vulnerabilities.  In conclusion, novel classes of compds. with high efficacy in T-PLL were discovered with the comprehensive ex vivo drug screening platform warranting further studies of synergisms and clin. testing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNlAKw7-znF7Vg90H21EOLACvtfcHk0lihupb0caom7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKmu7%252FI&md5=70fedb2dce547d45c70300ec7759ee07</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fleu.2017.252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2017.252%26sid%3Dliteratum%253Aachs%26aulast%3DAndersson%26aufirst%3DE.%2BI.%26aulast%3DP%25C3%25BCtzer%26aufirst%3DS.%26aulast%3DYadav%26aufirst%3DB.%26aulast%3DDufva%26aufirst%3DO.%26aulast%3DKhan%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DL.%26aulast%3DSellner%26aufirst%3DL.%26aulast%3DSchrader%26aufirst%3DA.%26aulast%3DCrispatzu%26aufirst%3DG.%26aulast%3DOle%25C5%259B%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DAdnan-Awad%26aufirst%3DS.%26aulast%3DLagstrom%26aufirst%3DS.%26aulast%3DBellanger%26aufirst%3DD.%26aulast%3DMpindi%26aufirst%3DJ.%2BP.%26aulast%3DEldfors%26aufirst%3DS.%26aulast%3DPemovska%26aufirst%3DT.%26aulast%3DPietarinen%26aufirst%3DP.%26aulast%3DLauhio%26aufirst%3DA.%26aulast%3DTomska%26aufirst%3DK.%26aulast%3DCuesta-Mateos%26aufirst%3DC.%26aulast%3DFaber%26aufirst%3DE.%26aulast%3DKoschmieder%26aufirst%3DS.%26aulast%3DBrummendorf%26aufirst%3DT.%2BH.%26aulast%3DKytola%26aufirst%3DS.%26aulast%3DSavolainen%26aufirst%3DE.%26aulast%3DSiitonen%26aufirst%3DT.%26aulast%3DEllonen%26aufirst%3DP.%26aulast%3DKallioneimi%26aufirst%3DO.%26aulast%3DWennerberg%26aufirst%3DK.%26aulast%3DDing%26aufirst%3DW.%26aulast%3DStern%26aufirst%3DM_H.%26aulast%3DHuber%26aufirst%3DW.%26aulast%3DAnders%26aufirst%3DS.%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DAittokallio%26aufirst%3DT.%26aulast%3DZenz%26aufirst%3DT.%26aulast%3DHerling%26aufirst%3DM.%26aulast%3DMustjoki%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520Novel%2520Drug%2520Sensitivities%2520in%2520T-PLL%2520by%2520High-Throughput%2520Ex%2520Vivo%2520Drug%2520Testing%2520and%2520Mutation%2520Profiling%26jtitle%3DLeukemia%26date%3D2018%26volume%3D32%26spage%3D774%26epage%3D787%26doi%3D10.1038%2Fleu.2017.252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mrakovcic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleinheinz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fröhlich, L. F.</span></span> <span> </span><span class="NLM_article-title">P53 at the Crossroads between Different Types of Hdac Inhibitor-Mediated Cancer Cell Death</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">2415</span>, <span class="refDoi"> DOI: 10.3390/ijms20102415</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.3390%2Fijms20102415" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosVGmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=2415&author=M.+Mrakovcicauthor=J.+Kleinheinzauthor=L.+F.+Fr%C3%B6hlich&title=P53+at+the+Crossroads+between+Different+Types+of+Hdac+Inhibitor-Mediated+Cancer+Cell+Death&doi=10.3390%2Fijms20102415"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">p53 at the crossroads between different types of HDAC inhibitor-mediated cancer cell death</span></div><div class="casAuthors">Mrakovcic, Maria; Kleinheinz, Johannes; Froehlich, Leopold F.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2415/1-2415/32</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Cancer is a complex genetic and epigenetic-based disease that has developed an armada of mechanisms to escape cell death.  The deregulation of apoptosis and autophagy, which are basic processes essential for normal cellular activity, are commonly encountered during the development of human tumors.  In order to assist the cancer cell in defeating the imbalance between cell growth and cell death, histone deacetylase inhibitors (HDACi) have been employed to reverse epigenetically deregulated gene expression caused by aberrant post-translational protein modifications.  These interfere with histone acetyltransferase- and deacetylase-mediated acetylation of both histone and non-histone proteins, and thereby exert a wide array of HDACi-stimulated cytotoxic effects.  Key determinants of HDACi lethality that interfere with cellular growth in a multitude of tumor cells are apoptosis and autophagy, which are either mutually exclusive or activated in combination.  Here, we compile known mol. signals and pathways involved in the HDACi-triggered induction of apoptosis and autophagy.  Currently, the factors that det. the mode of HDACi-elicited cell death are mostly unclear.  Correspondingly, we also summarized as yet established intertwined mechanisms, in particular with respect to the oncogenic tumor suppressor protein p53, that drive the interplay between apoptosis and autophagy in response to HDACi.  In this context, we also note the significance to det. the presence of functional p53 protein levels in the cancer cell.  The confirmation of the context-dependent function of autophagy will pave the way to improve the benefit from HDACi-mediated cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUKduySxvx97Vg90H21EOLACvtfcHk0lihupb0caom7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosVGmsg%253D%253D&md5=d74b939314d54d581024c3ef97664733</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.3390%2Fijms20102415&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms20102415%26sid%3Dliteratum%253Aachs%26aulast%3DMrakovcic%26aufirst%3DM.%26aulast%3DKleinheinz%26aufirst%3DJ.%26aulast%3DFr%25C3%25B6hlich%26aufirst%3DL.%2BF.%26atitle%3DP53%2520at%2520the%2520Crossroads%2520between%2520Different%2520Types%2520of%2520Hdac%2520Inhibitor-Mediated%2520Cancer%2520Cell%2520Death%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D20%26spage%3D2415%26doi%3D10.3390%2Fijms20102415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boidol, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornauth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Der Kouwe, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prutsch, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazianka, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gültekin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoermann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayerhoefer, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopfinger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauswirth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panny, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aretin, M.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgarth, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sperr, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonitsch-Klupp, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriggl, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jager, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubicek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staber, P. B.</span></span> <span> </span><span class="NLM_article-title">First-in-Human Response of BCL-2 Inhibitor Venetoclax in T-Cell Prolymphocytic Leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">2499</span>– <span class="NLM_lpage">2503</span>, <span class="refDoi"> DOI: 10.1182/blood-2017-05-785683</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1182%2Fblood-2017-05-785683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=28972014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1erurfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2017&pages=2499-2503&author=B.+Boidolauthor=C.+Kornauthauthor=E.+Van+Der+Kouweauthor=N.+Prutschauthor=L.+Kaziankaauthor=S.+G%C3%BCltekinauthor=G.+Hoermannauthor=M.+E.+Mayerhoeferauthor=G.+Hopfingerauthor=A.+Hauswirthauthor=M.+Pannyauthor=M.-B.+Aretinauthor=B.+Hilgarthauthor=W.+R.+Sperrauthor=P.+Valentauthor=I.+Simonitsch-Kluppauthor=R.+Morigglauthor=P.+Merkelauthor=L.+Kennerauthor=U.+Jagerauthor=S.+Kubicekauthor=P.+B.+Staber&title=First-in-Human+Response+of+BCL-2+Inhibitor+Venetoclax+in+T-Cell+Prolymphocytic+Leukemia&doi=10.1182%2Fblood-2017-05-785683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">First-in-human response of BCL-2 inhibitor venetoclax in T-cell prolymphocytic leukemia</span></div><div class="casAuthors">Boidol, Bernd; Kornauth, Christoph; van der Kouwe, Emiel; Prutsch, Nicole; Kazianka, Lukas; Gultekin, Sinan; Hoermann, Gregor; Mayerhoefer, Marius E.; Hopfinger, Georg; Hauswirth, Alexander; Panny, Michael; Aretin, Marie-Bernadette; Hilgarth, Bernadette; Sperr, Wolfgang R.; Valent, Peter; Simonitsch-Klupp, Ingrid; Moriggl, Richard; Merkel, Olaf; Kenner, Lukas; Jager, Ulrich; Kubicek, Stefan; Staber, Philipp B.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2499-2503</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">1528-0020</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">T-cell prolymphocytic leukemia (T-PLL) is a rare and aggressive T-lymphoid malignancy usually refractory to current treatment strategies and assocd. with short overall survival.  By applying next-generation functional testing of primary patient-derived lymphoma cells using a library of 106 US Food and Drug Administration (FDA)-approved anticancer drugs or compds. currently in clin. development, we set out to identify novel effective treatments for T-PLL patients.  We found that the B-cell lymphoma 2 (BCL-2) inhibitor venetoclax (ABT-199) demonstrated the strongest T-PLL-specific response when comparing individual ex vivo drug response in 86 patients with refractory hematol. malignancies.  Mechanistically, responses to venetoclax correlated with protein expression of BCL-2 but not with expression of the BCL-2 family members myeloid cell leukemia 1 (MCL-1) and BCL-XL in lymphoma cells.  BCL-2 expression was inversely correlated with the expression of MCL-1.  Based on the ex vivo responses, venetoclax treatment was commenced in 2 late-stage refractory T-PLL patients resulting in clin. responses.  Our findings demonstrate first evidence of single-agent activity of venetoclax both ex vivo and in humans, offering a novel agent in T-PLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGCxVs1rSztrVg90H21EOLACvtfcHk0lihupb0caom7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1erurfI&md5=d24bb98d7284eb6484c9a11134b49837</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1182%2Fblood-2017-05-785683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2017-05-785683%26sid%3Dliteratum%253Aachs%26aulast%3DBoidol%26aufirst%3DB.%26aulast%3DKornauth%26aufirst%3DC.%26aulast%3DVan%2BDer%2BKouwe%26aufirst%3DE.%26aulast%3DPrutsch%26aufirst%3DN.%26aulast%3DKazianka%26aufirst%3DL.%26aulast%3DG%25C3%25BCltekin%26aufirst%3DS.%26aulast%3DHoermann%26aufirst%3DG.%26aulast%3DMayerhoefer%26aufirst%3DM.%2BE.%26aulast%3DHopfinger%26aufirst%3DG.%26aulast%3DHauswirth%26aufirst%3DA.%26aulast%3DPanny%26aufirst%3DM.%26aulast%3DAretin%26aufirst%3DM.-B.%26aulast%3DHilgarth%26aufirst%3DB.%26aulast%3DSperr%26aufirst%3DW.%2BR.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DSimonitsch-Klupp%26aufirst%3DI.%26aulast%3DMoriggl%26aufirst%3DR.%26aulast%3DMerkel%26aufirst%3DP.%26aulast%3DKenner%26aufirst%3DL.%26aulast%3DJager%26aufirst%3DU.%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DStaber%26aufirst%3DP.%2BB.%26atitle%3DFirst-in-Human%2520Response%2520of%2520BCL-2%2520Inhibitor%2520Venetoclax%2520in%2520T-Cell%2520Prolymphocytic%2520Leukemia%26jtitle%3DBlood%26date%3D2017%26volume%3D130%26spage%3D2499%26epage%3D2503%26doi%3D10.1182%2Fblood-2017-05-785683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Souers, A. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leverson, J. D</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boghaert, E. R</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackler, S. L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catron, N. D</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dayton, B. D</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enschede, S. H</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairbrother, W. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D. C S</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hymowitz, S. G</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khaw, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maecker, H. L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, K. C</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, K. D</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitten, M. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nimmer, P. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oleksijew, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, D. C</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, A. W</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seymour, J. F</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, M. L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, G. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tahir, S. K</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendt, M. D</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, J. C</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humerickhouse, R. A</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W</span></span> <span> </span><span class="NLM_article-title">ABT-199, a Potent and Selective BCL-2 Inhibitor, Achieves Antitumor Activity While Sparing Platelets</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">208</span>, <span class="refDoi"> DOI: 10.1038/nm.3048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fnm.3048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=23291630" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjslagtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=202-208&author=A.+J+Souersauthor=J.+D+Leversonauthor=E.+R+Boghaertauthor=S.+L+Acklerauthor=N.+D+Catronauthor=J.+Chenauthor=B.+D+Daytonauthor=H.+Dingauthor=S.+H+Enschedeauthor=W.+J+Fairbrotherauthor=D.+C+S+Huangauthor=S.+G+Hymowitzauthor=S.+Jinauthor=S.+L.+Khawauthor=P.+J+Kovarauthor=L.+T+Lamauthor=J.+Leeauthor=H.+L+Maeckerauthor=K.+C+Marshauthor=K.+D+Masonauthor=M.+J+Mittenauthor=P.+M+Nimmerauthor=A.+Oleksijewauthor=C.+H+Parkauthor=C.-M.+Parkauthor=D.+C+Phillipsauthor=A.+W+Robertsauthor=D.+Sampathauthor=J.+F+Seymourauthor=M.+L+Smithauthor=G.+M+Sullivanauthor=S.+K+Tahirauthor=C.+Tseauthor=M.+D+Wendtauthor=Y.+Xiaoauthor=J.+C+Xueauthor=H.+Zhangauthor=R.+A+Humerickhouseauthor=S.+H+Rosenbergauthor=S.+W+Elmore&title=ABT-199%2C+a+Potent+and+Selective+BCL-2+Inhibitor%2C+Achieves+Antitumor+Activity+While+Sparing+Platelets&doi=10.1038%2Fnm.3048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">ABT-199, a potent and selective BCL-2 inhibitor, achieves antitumor activity while sparing platelets</span></div><div class="casAuthors">Souers, Andrew J.; Leverson, Joel D.; Boghaert, Erwin R.; Ackler, Scott L.; Catron, Nathaniel D.; Chen, Jun; Dayton, Brian D.; Ding, Hong; Enschede, Sari H.; Fairbrother, Wayne J.; Huang, David C. S.; Hymowitz, Sarah G.; Jin, Sha; Khaw, Seong Lin; Kovar, Peter J.; Lam, Lloyd T.; Lee, Jackie; Maecker, Heather L.; Marsh, Kennan C.; Mason, Kylie D.; Mitten, Michael J.; Nimmer, Paul M.; Oleksijew, Anatol; Park, Chang H.; Park, Cheol-Min; Phillips, Darren C.; Roberts, Andrew W.; Sampath, Deepak; Seymour, John F.; Smith, Morey L.; Sullivan, Gerard M.; Tahir, Stephen K.; Tse, Chris; Wendt, Michael D.; Xiao, Yu; Xue, John C.; Zhang, Haichao; Humerickhouse, Rod A.; Rosenberg, Saul H.; Elmore, Steven W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">202-208</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins in the B cell CLL/lymphoma 2 (BCL-2) family are key regulators of the apoptotic process.  This family comprises proapoptotic and prosurvival proteins, and shifting the balance toward the latter is an established mechanism whereby cancer cells evade apoptosis.  The therapeutic potential of directly inhibiting prosurvival proteins was unveiled with the development of navitoclax, a selective inhibitor of both BCL-2 and BCL-2-like 1 (BCL-XL), which has shown clin. efficacy in some BCL-2-dependent hematol. cancers.  However, concomitant on-target thrombocytopenia caused by BCL-XL inhibition limits the efficacy achievable with this agent.  Here we report the re-engineering of navitoclax to create a highly potent, orally bioavailable and BCL-2-selective inhibitor, ABT-199.  This compd. inhibits the growth of BCL-2-dependent tumors in vivo and spares human platelets.  A single dose of ABT-199 in three patients with refractory chronic lymphocytic leukemia resulted in tumor lysis within 24 h.  These data indicate that selective pharmacol. inhibition of BCL-2 shows promise for the treatment of BCL-2-dependent hematol. cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqi7P7koWwJHLVg90H21EOLACvtfcHk0ljzAYGII547gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjslagtQ%253D%253D&md5=6a24b72303eb2062da653f1f61ac74fb</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnm.3048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3048%26sid%3Dliteratum%253Aachs%26aulast%3DSouers%26aufirst%3DA.%2BJ%26aulast%3DLeverson%26aufirst%3DJ.%2BD%26aulast%3DBoghaert%26aufirst%3DE.%2BR%26aulast%3DAckler%26aufirst%3DS.%2BL%26aulast%3DCatron%26aufirst%3DN.%2BD%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DDayton%26aufirst%3DB.%2BD%26aulast%3DDing%26aufirst%3DH.%26aulast%3DEnschede%26aufirst%3DS.%2BH%26aulast%3DFairbrother%26aufirst%3DW.%2BJ%26aulast%3DHuang%26aufirst%3DD.%2BC%2BS%26aulast%3DHymowitz%26aufirst%3DS.%2BG%26aulast%3DJin%26aufirst%3DS.%26aulast%3DKhaw%26aufirst%3DS.%2BL.%26aulast%3DKovar%26aufirst%3DP.%2BJ%26aulast%3DLam%26aufirst%3DL.%2BT%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DMaecker%26aufirst%3DH.%2BL%26aulast%3DMarsh%26aufirst%3DK.%2BC%26aulast%3DMason%26aufirst%3DK.%2BD%26aulast%3DMitten%26aufirst%3DM.%2BJ%26aulast%3DNimmer%26aufirst%3DP.%2BM%26aulast%3DOleksijew%26aufirst%3DA.%26aulast%3DPark%26aufirst%3DC.%2BH%26aulast%3DPark%26aufirst%3DC.-M.%26aulast%3DPhillips%26aufirst%3DD.%2BC%26aulast%3DRoberts%26aufirst%3DA.%2BW%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DSeymour%26aufirst%3DJ.%2BF%26aulast%3DSmith%26aufirst%3DM.%2BL%26aulast%3DSullivan%26aufirst%3DG.%2BM%26aulast%3DTahir%26aufirst%3DS.%2BK%26aulast%3DTse%26aufirst%3DC.%26aulast%3DWendt%26aufirst%3DM.%2BD%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DXue%26aufirst%3DJ.%2BC%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DHumerickhouse%26aufirst%3DR.%2BA%26aulast%3DRosenberg%26aufirst%3DS.%2BH%26aulast%3DElmore%26aufirst%3DS.%2BW%26atitle%3DABT-199%252C%2520a%2520Potent%2520and%2520Selective%2520BCL-2%2520Inhibitor%252C%2520Achieves%2520Antitumor%2520Activity%2520While%2520Sparing%2520Platelets%26jtitle%3DNat.%2520Med.%26date%3D2013%26volume%3D19%26spage%3D202%26epage%3D208%26doi%3D10.1038%2Fnm.3048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davids, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagel, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahl, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puvvada, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerecitano, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kipps, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gressick, L.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Targeting BCL2 with Venetoclax in Relapsed Chronic Lymphocytic Leukemia</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>374</i></span>,  <span class="NLM_fpage">311</span>– <span class="NLM_lpage">322</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa1513257</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1056%2FNEJMoa1513257" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=26639348" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtV2murrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=374&publication_year=2016&pages=311-322&author=A.+W.+Robertsauthor=M.+S.+Davidsauthor=J.+M.+Pagelauthor=B.+S.+Kahlauthor=S.+D.+Puvvadaauthor=J.+F.+Gerecitanoauthor=T.+J.+Kippsauthor=M.+A.+Andersonauthor=J.+R.+Brownauthor=L.+Gressick&title=Targeting+BCL2+with+Venetoclax+in+Relapsed+Chronic+Lymphocytic+Leukemia&doi=10.1056%2FNEJMoa1513257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting BCL2 with venetoclax in relapsed chronic lymphocytic leukemia</span></div><div class="casAuthors">Roberts, Andrew W.; Davids, Matthew S.; Pagel, John M.; Kahl, Brad S.; Puvvada, Soham D.; Gerecitano, John F.; Kipps, Thomas J.; Anderson, Mary Ann; Brown, Jennifer R.; Gressick, Lori; Wong, Shekman; Dunbar, Martin; Zhu, Ming; Desai, Monali B.; Cerri, Elisa; Enschede, Sari Heitner; Humerickhouse, Rod A.; Wierda, William G.; Seymour, John F.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">374</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">311-322</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">BACKGROUND New treatments have improved outcomes for patients with relapsed chronic lymphocytic leukemia (CLL), but complete remissions remain uncommon.  Venetoclax has a distinct mechanism of action; it targets BCL2, a protein central to the survival of CLL cells.  METHODS We conducted a phase 1 dose-escalation study of daily oral venetoclax in patients with relapsed or refractory CLL or small lymphocytic lymphoma (SLL) to assess safety, pharmacokinetic profile, and efficacy.  In the dose-escalation phase, 56 patients received active treatment in one of eight dose groups that ranged from 150 to 1200 mg per day.  In an expansion cohort, 60 addnl. patients were treated with a weekly stepwise ramp-up in doses as high as 400 mg per day.  RESULTS The majority of the study patients had received multiple previous treatments, and 89% had poor prognostic clin. or genetic features.  Venetoclax was active at all dose levels.  Clin. tumor lysis syndrome occurred in 3 of 56 patients in the dose-escalation cohort, with one death.  After adjustments to the dose-escalation schedule, clin. tumor lysis syndrome did not occur in any of the 60 patients in the expansion cohort.  Other toxic effects included mild diarrhea (in 52% of the patients), upper respiratory tract infection (in 48%), nausea (in 47%), and grade 3 or 4 neutropenia (in 41%).  A max. tolerated dose was not identified.  Among the 116 patients who received venetoclax, 92 (79%) had a response.  Response rates ranged from 71 to 79% among patients in subgroups with an adverse prognosis, including those with resistance to fludarabine, those with chromosome 17p deletions (deletion 17p CLL), and those with unmutated IGHV.  Complete remissions occurred in 20% of the patients, including 5% who had no minimal residual disease on flow cytometry.  The 15-mo progression-free survival est. for the 400-mg dose groups was 69%.  CONCLUSIONS Selective targeting of BCL2 with venetoclax had a manageable safety profile and induced substantial responses in patients with relapsed CLL or SLL, including those with poor prognostic features.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHQgYgnKl88rVg90H21EOLACvtfcHk0ljzAYGII547gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtV2murrP&md5=b371a37b76ee88cfc6ac47ea0eeea0e6</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1513257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1513257%26sid%3Dliteratum%253Aachs%26aulast%3DRoberts%26aufirst%3DA.%2BW.%26aulast%3DDavids%26aufirst%3DM.%2BS.%26aulast%3DPagel%26aufirst%3DJ.%2BM.%26aulast%3DKahl%26aufirst%3DB.%2BS.%26aulast%3DPuvvada%26aufirst%3DS.%2BD.%26aulast%3DGerecitano%26aufirst%3DJ.%2BF.%26aulast%3DKipps%26aufirst%3DT.%2BJ.%26aulast%3DAnderson%26aufirst%3DM.%2BA.%26aulast%3DBrown%26aufirst%3DJ.%2BR.%26aulast%3DGressick%26aufirst%3DL.%26atitle%3DTargeting%2520BCL2%2520with%2520Venetoclax%2520in%2520Relapsed%2520Chronic%2520Lymphocytic%2520Leukemia%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2016%26volume%3D374%26spage%3D311%26epage%3D322%26doi%3D10.1056%2FNEJMoa1513257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rhodes, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshpande, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varambally, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrette, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pander, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaiyan, A. M.</span></span> <span> </span><span class="NLM_article-title">ONCOMINE: A Cancer Microarray Database and Integrated Data-Mining Platform</span>. <i>Neoplasia.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">6</span>, <span class="refDoi"> DOI: 10.1016/S1476-5586(04)80047-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1016%2FS1476-5586%2804%2980047-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=15068665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjslSgtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2004&pages=1-6&author=D.+R.+Rhodesauthor=J.+Yuauthor=K.+Shankerauthor=N.+Deshpandeauthor=R.+Varamballyauthor=D.+Ghoshauthor=T.+Barretteauthor=A.+Panderauthor=A.+M.+Chinnaiyan&title=ONCOMINE%3A+A+Cancer+Microarray+Database+and+Integrated+Data-Mining+Platform&doi=10.1016%2FS1476-5586%2804%2980047-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">ONCOMINE: a cancer microarray database and integrated data-mining platform</span></div><div class="casAuthors">Rhodes, Daniel R.; Yu, Jianjun; Shanker, K.; Deshpande, Nandan; Varambally, Radhika; Ghosh, Debashis; Barrette, Terrence; Pandey, Akhilesh; Chinnaiyan, Arul M.</div><div class="citationInfo"><span class="NLM_cas:title">Neoplasia (Ann Arbor, MI, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-6</span>CODEN:
                <span class="NLM_cas:coden">NEOPFL</span>;
        ISSN:<span class="NLM_cas:issn">1522-8002</span>.
    
            (<span class="NLM_cas:orgname">Neoplasia Press Inc.</span>)
        </div><div class="casAbstract">DNA microarray technol. has led to an explosion of oncogenomic analyses, generating a wealth of data and uncovering the complex gene expression patterns of cancer.  Unfortunately, due to the lack of a unifying bioinformatic resource, the majority of these data sit stagnant and disjointed following publication, massively underutilized by the cancer research community.  Here, we present ONCOMINE, a cancer microarray database and web-based data-mining platform aimed at facilitating discovery from genome-wide expression analyses.  To date, ONCOMINE contains 65 gene expression datasets comprising nearly 48 million gene expression measurements form over 4700 microarray expts.  Differential expression analyses comparing most major types of cancer with resp. normal tissues as well as a variety of cancer subtypes and clin.-based and pathol.-based analyses are available for exploration.  Data can be queried and visualized for a selected gene across all analyses or for multiple genes in a selected anal.  Furthermore, gene sets can be limited to clin. important annotations including secreted, kinase, membrane, and known gene-drug target pairs to facilitate the discovery of novel biomarkers and therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUXX6_gcnyLrVg90H21EOLACvtfcHk0ljjo_v7pj-C6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjslSgtrY%253D&md5=84d851435b55438b4a6ecd11ab2cd5b3</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2FS1476-5586%2804%2980047-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1476-5586%252804%252980047-2%26sid%3Dliteratum%253Aachs%26aulast%3DRhodes%26aufirst%3DD.%2BR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DShanker%26aufirst%3DK.%26aulast%3DDeshpande%26aufirst%3DN.%26aulast%3DVarambally%26aufirst%3DR.%26aulast%3DGhosh%26aufirst%3DD.%26aulast%3DBarrette%26aufirst%3DT.%26aulast%3DPander%26aufirst%3DA.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DONCOMINE%253A%2520A%2520Cancer%2520Microarray%2520Database%2520and%2520Integrated%2520Data-Mining%2520Platform%26jtitle%3DNeoplasia.%26date%3D2004%26volume%3D6%26spage%3D1%26epage%3D6%26doi%3D10.1016%2FS1476-5586%2804%2980047-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gawel, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shouksmith, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raouf, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nawar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toutah, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukhari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manaswiyoungkul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olaoye, O. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israelian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radu, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabral, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedighi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Araujo, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span> <span> </span><span class="NLM_article-title">PTG-0861: A Novel HDAC6-Selective Inhibitor as a Therapeutic Strategy in Acute Myeloid Leukaemia</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>201</i></span>,  <span class="NLM_fpage">112411</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112411</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1016%2Fj.ejmech.2020.112411" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=32615502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlSht73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=201&publication_year=2020&pages=112411&author=J.+M.+Gawelauthor=A.+E.+Shouksmithauthor=Y.+S.+Raoufauthor=N.+Nawarauthor=K.+Toutahauthor=S.+Bukhariauthor=P.+Manaswiyoungkulauthor=O.+O.+Olaoyeauthor=J.+Israelianauthor=T.+B.+Raduauthor=A.+D.+Cabralauthor=D.+Sinaauthor=A.+Sedighiauthor=E.+D.+de+Araujoauthor=P.+T.+Gunning&title=PTG-0861%3A+A+Novel+HDAC6-Selective+Inhibitor+as+a+Therapeutic+Strategy+in+Acute+Myeloid+Leukaemia&doi=10.1016%2Fj.ejmech.2020.112411"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">PTG-0861: A novel HDAC6-selective inhibitor as a therapeutic strategy in acute myeloid leukaemia</span></div><div class="casAuthors">Gawel, Justyna M.; Shouksmith, Andrew E.; Raouf, Yasir S.; Nawar, Nabanita; Toutah, Krimo; Bukhari, Shazreh; Manaswiyoungkul, Pimyupa; Olaoye, Olasunkanmi O.; Israelian, Johan; Radu, Tudor B.; Cabral, Aaron D.; Sina, Diana; Sedighi, Abootaleb; de Araujo, Elvin D.; Gunning, Patrick T.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">201</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112411</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Dysregulated Histone Deacetylase (HDAC) activity across multiple human pathologies have highlighted this family of epigenetic enzymes as crit. druggable targets, amenable to small mol. intervention.  While efficacious, current approaches using non-selective HDAC inhibitors (HDACi) have been shown to cause a range of undesirable clin. toxicities.  To circumvent this, recent efforts have focused on the design of highly selective HDACi as a novel therapeutic strategy.  Beyond roles in regulating transcription, the unique HDAC6 (with two catalytic domains) regulates the deacetylation of α-tubulin; promoting growth factor-controlled cell motility, cell division, and metastatic hallmarks.  Recent studies have linked aberrant HDAC6 function in various hematol. cancers including acute myeloid leukemia and multiple myeloma.  Herein, we report the discovery, in vitro characterization, and biol. evaluation of PTG-0861 (JG-265), a novel HDAC6-selective inhibitor with strong isoenzyme-selectivity (∼36x ) and low nanomolar potency (IC50 = 5.92 nM) against HDAC6.  This selectivity profile was rationalized via in silico docking studies and also obsd. in cellulo through cellular target engagement.  Moreover, PTG-0861 achieved relevant potency against several blood cancer cell lines (e.g. MV4-11, MM1S), while showing limited cytotoxicity against non-malignant cells (e.g. NHF, HUVEC) and CD-1 mice.  In examg. compd. stability and cellular permeability, PTG-0861 revealed a promising in vitro pharmacokinetic (PK) profile.  Altogether, in this study we identified a novel and potent HDAC6-selective inhibitor (∼4x more selective than current clin. stds. - citarinostat, ricolinostat), which achieves cellular target engagement, efficacy in hematol. cancer cells with a promising safety profile and in vitro PK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGUpACNMyWt7Vg90H21EOLACvtfcHk0ljjo_v7pj-C6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlSht73E&md5=c5b2e399be96a785b52019f04e1c08b8</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112411%26sid%3Dliteratum%253Aachs%26aulast%3DGawel%26aufirst%3DJ.%2BM.%26aulast%3DShouksmith%26aufirst%3DA.%2BE.%26aulast%3DRaouf%26aufirst%3DY.%2BS.%26aulast%3DNawar%26aufirst%3DN.%26aulast%3DToutah%26aufirst%3DK.%26aulast%3DBukhari%26aufirst%3DS.%26aulast%3DManaswiyoungkul%26aufirst%3DP.%26aulast%3DOlaoye%26aufirst%3DO.%2BO.%26aulast%3DIsraelian%26aufirst%3DJ.%26aulast%3DRadu%26aufirst%3DT.%2BB.%26aulast%3DCabral%26aufirst%3DA.%2BD.%26aulast%3DSina%26aufirst%3DD.%26aulast%3DSedighi%26aufirst%3DA.%26aulast%3Dde%2BAraujo%26aufirst%3DE.%2BD.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26atitle%3DPTG-0861%253A%2520A%2520Novel%2520HDAC6-Selective%2520Inhibitor%2520as%2520a%2520Therapeutic%2520Strategy%2520in%2520Acute%2520Myeloid%2520Leukaemia%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D201%26spage%3D112411%26doi%3D10.1016%2Fj.ejmech.2020.112411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shouksmith, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gawel, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nawar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raouf, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukhari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manaswiyoungkul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olaoye, O. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabral, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedighi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Araujo, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span> <span> </span><span class="NLM_article-title">Class I/IIb-Selective HDAC Inhibitor Exhibits Oral Bioavailability and Therapeutic Efficacy in Acute Myeloid Leukemia</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00471</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00471" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlygtLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=56-64&author=A.+E.+Shouksmithauthor=J.+M.+Gawelauthor=N.+Nawarauthor=D.+Sinaauthor=Y.+S.+Raoufauthor=S.+Bukhariauthor=L.+Heauthor=A.+E.+Johnsauthor=P.+Manaswiyoungkulauthor=O.+O.+Olaoyeauthor=A.+D.+Cabralauthor=A.+Sedighiauthor=E.+D.+de+Araujoauthor=P.+T.+Gunning&title=Class+I%2FIIb-Selective+HDAC+Inhibitor+Exhibits+Oral+Bioavailability+and+Therapeutic+Efficacy+in+Acute+Myeloid+Leukemia&doi=10.1021%2Facsmedchemlett.9b00471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Class I/IIb-Selective HDAC Inhibitor Exhibits Oral Bioavailability and Therapeutic Efficacy in Acute Myeloid Leukemia</span></div><div class="casAuthors">Shouksmith, Andrew E.; Gawel, Justyna M.; Nawar, Nabanita; Sina, Diana; Raouf, Yasir S.; Bukhari, Shazreh; He, Liying; Johns, Alexandra E.; Manaswiyoungkul, Pimyupa; Olaoye, Olasunkanmi O.; Cabral, Aaron D.; Sedighi, Abootaleb; de Araujo, Elvin D.; Gunning, Patrick T.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-64</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The HDAC inhibitor 4-tert-butyl-N-(4-(hydroxycarbamoyl)phenyl)benzamide (AES-350, I) was identified as a promising preclin. candidate for the treatment of acute myeloid leukemia (AML), an aggressive malignancy with a meagre 24% 5-yr survival rate.  Through screening of low-mol.-wt. analogs derived from the previously discovered novel HDAC inhibitor, AES-135 (II), compd. I demonstrated greater HDAC isoform selectivity, higher cytotoxicity in MV4-11 cells, an improved therapeutic window, and more efficient absorption through cellular and lipid membranes.  Compd. I also demonstrated improved oral bioavailability compared to SAHA in mouse models.  A broad spectrum of expts., including FACS, ELISA, and Western blotting, were performed to support our hypothesis that I dose-dependently triggers apoptosis in AML cells through HDAC inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8ikqWn7KZc7Vg90H21EOLACvtfcHk0lidMc3Cnb5jtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlygtLrK&md5=3c21f11cda2a27e00f30a7cd3c700785</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00471%26sid%3Dliteratum%253Aachs%26aulast%3DShouksmith%26aufirst%3DA.%2BE.%26aulast%3DGawel%26aufirst%3DJ.%2BM.%26aulast%3DNawar%26aufirst%3DN.%26aulast%3DSina%26aufirst%3DD.%26aulast%3DRaouf%26aufirst%3DY.%2BS.%26aulast%3DBukhari%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DL.%26aulast%3DJohns%26aufirst%3DA.%2BE.%26aulast%3DManaswiyoungkul%26aufirst%3DP.%26aulast%3DOlaoye%26aufirst%3DO.%2BO.%26aulast%3DCabral%26aufirst%3DA.%2BD.%26aulast%3DSedighi%26aufirst%3DA.%26aulast%3Dde%2BAraujo%26aufirst%3DE.%2BD.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26atitle%3DClass%2520I%252FIIb-Selective%2520HDAC%2520Inhibitor%2520Exhibits%2520Oral%2520Bioavailability%2520and%2520Therapeutic%2520Efficacy%2520in%2520Acute%2520Myeloid%2520Leukemia%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D56%26epage%3D64%26doi%3D10.1021%2Facsmedchemlett.9b00471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olaoye, O. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nawar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geletu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedighi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukhari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raouf, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manaswiyoungkul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdogan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdeldayem, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabral, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toutah, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shouksmith, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gawel, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israelian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radu, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kachhiyapatel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Araujo, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span> <span> </span><span class="NLM_article-title">Unique Molecular Interaction with the Histone Deacetylase 6 Catalytic Tunnel: Crystallographic and Biological Characterization of a Model Chemotype</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">2691</span>– <span class="NLM_lpage">2704</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01922</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01922" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BB3MXjs12hu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2021&pages=2691-2704&author=O.+O.+Olaoyeauthor=P.+R.+Watsonauthor=N.+Nawarauthor=M.+Geletuauthor=A.+Sedighiauthor=S.+Bukhariauthor=Y.+S.+Raoufauthor=P.+Manaswiyoungkulauthor=F.+Erdoganauthor=A.+Abdeldayemauthor=A.+D.+Cabralauthor=M.+M.+Hassanauthor=K.+Toutahauthor=A.+E.+Shouksmithauthor=J.+M.+Gawelauthor=J.+Israelianauthor=T.+B.+Raduauthor=N.+Kachhiyapatelauthor=E.+D.+de+Araujoauthor=D.+W.+Christiansonauthor=P.+T.+Gunning&title=Unique+Molecular+Interaction+with+the+Histone+Deacetylase+6+Catalytic+Tunnel%3A+Crystallographic+and+Biological+Characterization+of+a+Model+Chemotype&doi=10.1021%2Facs.jmedchem.0c01922"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Unique Molecular Interaction with the Histone Deacetylase 6 Catalytic Tunnel: Crystallographic and Biological Characterization of a Model Chemotype</span></div><div class="casAuthors">Olaoye, Olasunkanmi O.; Watson, Paris R.; Nawar, Nabanita; Geletu, Mulu; Sedighi, Abootaleb; Bukhari, Shazreh; Raouf, Yasir S.; Manaswiyoungkul, Pimyupa; Erdogan, Fettah; Abdeldayem, Ayah; Cabral, Aaron D.; Hassan, Muhammad Murtaza; Toutah, Krimo; Shouksmith, Andrew E.; Gawel, Justyna M.; Israelian, Johan; Radu, Tudor B.; Kachhiyapatel, Niyati; de Araujo, Elvin D.; Christianson, David W.; Gunning, Patrick T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2691-2704</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone deacetylase 6 (HDAC6) is involved in multiple regulatory processes, ranging from cellular stress to intracellular transport.  Inhibition of aberrant HDAC6 activity in several cancers and neurol. diseases has been shown to be efficacious in both preclin. and clin. studies.  While selective HDAC6 targeting has been pursued as an alternative to pan-HDAC drugs, identifying truly selective mol. templates has not been trivial.  Herein, we report a structure-activity relationship study yielding TO-317, which potently binds HDAC6 catalytic domain 2 (Ki = 0.7 nM) and inhibits the enzyme function (IC50 = 2 nM).  TO-317 exhibits 158-fold selectivity for HDAC6 over other HDAC isoenzymes by binding the catalytic Zn2+ and, uniquely, making a never seen before direct hydrogen bond with the Zn2+ coordinating residue, His614.  This novel structural motif targeting the second-sphere His614 interaction, obsd. in a 1.84 Å resoln. crystal structure with drHDAC6 from zebrafish, can provide new pharmacophores for identifying enthalpically driven, high-affinity, HDAC6-selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZoHL-PIpylbVg90H21EOLACvtfcHk0lidMc3Cnb5jtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXjs12hu74%253D&md5=1fc291767413b2b65282c6a7009cb17e</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01922%26sid%3Dliteratum%253Aachs%26aulast%3DOlaoye%26aufirst%3DO.%2BO.%26aulast%3DWatson%26aufirst%3DP.%2BR.%26aulast%3DNawar%26aufirst%3DN.%26aulast%3DGeletu%26aufirst%3DM.%26aulast%3DSedighi%26aufirst%3DA.%26aulast%3DBukhari%26aufirst%3DS.%26aulast%3DRaouf%26aufirst%3DY.%2BS.%26aulast%3DManaswiyoungkul%26aufirst%3DP.%26aulast%3DErdogan%26aufirst%3DF.%26aulast%3DAbdeldayem%26aufirst%3DA.%26aulast%3DCabral%26aufirst%3DA.%2BD.%26aulast%3DHassan%26aufirst%3DM.%2BM.%26aulast%3DToutah%26aufirst%3DK.%26aulast%3DShouksmith%26aufirst%3DA.%2BE.%26aulast%3DGawel%26aufirst%3DJ.%2BM.%26aulast%3DIsraelian%26aufirst%3DJ.%26aulast%3DRadu%26aufirst%3DT.%2BB.%26aulast%3DKachhiyapatel%26aufirst%3DN.%26aulast%3Dde%2BAraujo%26aufirst%3DE.%2BD.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26atitle%3DUnique%2520Molecular%2520Interaction%2520with%2520the%2520Histone%2520Deacetylase%25206%2520Catalytic%2520Tunnel%253A%2520Crystallographic%2520and%2520Biological%2520Characterization%2520of%2520a%2520Model%2520Chemotype%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2021%26volume%3D64%26spage%3D2691%26epage%3D2704%26doi%3D10.1021%2Facs.jmedchem.0c01922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shouksmith, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimard, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gawel, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raouf, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geletu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger-Becvar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Araujo, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luchman, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heaton, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakhshinyan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adile, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venugopal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Hare, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deininger, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konieczny, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishel, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterization of AES-135, a Hydroxamic Acid-Based HDAC Inhibitor That Prolongs Survival in an Orthotopic Mouse Model of Pancreatic Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2651</span>– <span class="NLM_lpage">2665</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01957</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01957" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjt1Slu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2651-2665&author=A.+E.+Shouksmithauthor=F.+Shahauthor=M.+L.+Grimardauthor=J.+M.+Gawelauthor=Y.+S.+Raoufauthor=M.+Geletuauthor=A.+Berger-Becvarauthor=E.+D.+De+Araujoauthor=H.+A.+Luchmanauthor=W.+L.+Heatonauthor=D.+Bakhshinyanauthor=A.+A.+Adileauthor=C.+Venugopalauthor=T.+O%E2%80%99Hareauthor=M.+W.+Deiningerauthor=S.+K.+Singhauthor=S.+F.+Koniecznyauthor=S.+Weissauthor=M.+L.+Fishelauthor=P.+T.+Gunning&title=Identification+and+Characterization+of+AES-135%2C+a+Hydroxamic+Acid-Based+HDAC+Inhibitor+That+Prolongs+Survival+in+an+Orthotopic+Mouse+Model+of+Pancreatic+Cancer&doi=10.1021%2Facs.jmedchem.8b01957"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of AES-135, a hydroxamic acid-based HDAC inhibitor that prolongs survival in an orthotopic mouse model of pancreatic cancer</span></div><div class="casAuthors">Shouksmith, Andrew E.; Shah, Fenil; Grimard, Michelle L.; Gawel, Justyna M.; Raouf, Yasir S.; Geletu, Mulu; Berger-Becvar, Angelika; de Araujo, Elvin D.; Luchman, H. Artee; Heaton, William L.; Bakhshinyan, David; Adile, Ashley A.; Venugopal, Chitra; O'Hare, Thomas; Deininger, Michael W.; Singh, Sheila K.; Konieczny, Stephen F.; Weiss, Samuel; Fishel, Melissa L.; Gunning, Patrick T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2651-2665</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pancreatic ductal adenocarcinoma (PDAC) is an aggressive, incurable cancer with a 20% 1 yr survival rate.  While std.-of-care therapy can prolong life in a small fraction of cases, PDAC is inherently resistant to current treatments, and novel therapies are urgently required.  Histone deacetylase (HDAC) inhibitors are effective in killing pancreatic cancer cells in in vitro PDAC studies, and although there are a few clin. studies investigating combination therapy including HDAC inhibitors, no HDAC drug or combination therapy with an HDAC drug has been approved for the treatment of PDAC.  We developed an inhibitor of HDACs, AES-135, that exhibits nanomolar inhibitory activity against HDAC3, HDAC6, and HDAC11 in biochem. assays.  In a three-dimensional coculture model, AES-135 kills low-passage patient-derived tumor spheroids selectively over surrounding cancer-assocd. fibroblasts and has excellent pharmacokinetic properties in vivo.  In an orthotopic murine model of pancreatic cancer, AES-135 prolongs survival significantly, therefore representing a candidate for further preclin. testing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsx6K4JeAb-bVg90H21EOLACvtfcHk0lidMc3Cnb5jtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjt1Slu78%253D&md5=2b90becae883cc5552a701a614d8c452</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01957&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01957%26sid%3Dliteratum%253Aachs%26aulast%3DShouksmith%26aufirst%3DA.%2BE.%26aulast%3DShah%26aufirst%3DF.%26aulast%3DGrimard%26aufirst%3DM.%2BL.%26aulast%3DGawel%26aufirst%3DJ.%2BM.%26aulast%3DRaouf%26aufirst%3DY.%2BS.%26aulast%3DGeletu%26aufirst%3DM.%26aulast%3DBerger-Becvar%26aufirst%3DA.%26aulast%3DDe%2BAraujo%26aufirst%3DE.%2BD.%26aulast%3DLuchman%26aufirst%3DH.%2BA.%26aulast%3DHeaton%26aufirst%3DW.%2BL.%26aulast%3DBakhshinyan%26aufirst%3DD.%26aulast%3DAdile%26aufirst%3DA.%2BA.%26aulast%3DVenugopal%26aufirst%3DC.%26aulast%3DO%25E2%2580%2599Hare%26aufirst%3DT.%26aulast%3DDeininger%26aufirst%3DM.%2BW.%26aulast%3DSingh%26aufirst%3DS.%2BK.%26aulast%3DKonieczny%26aufirst%3DS.%2BF.%26aulast%3DWeiss%26aufirst%3DS.%26aulast%3DFishel%26aufirst%3DM.%2BL.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520AES-135%252C%2520a%2520Hydroxamic%2520Acid-Based%2520HDAC%2520Inhibitor%2520That%2520Prolongs%2520Survival%2520in%2520an%2520Orthotopic%2520Mouse%2520Model%2520of%2520Pancreatic%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2651%26epage%3D2665%26doi%3D10.1021%2Facs.jmedchem.8b01957" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bergman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Villarroel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villagra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sotomayor, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">Selective Histone Deacetylase 6 Inhibitors Bearing Substituted Urea Linkers Inhibit Melanoma Cell Growth</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9891</span>– <span class="NLM_lpage">9899</span>, <span class="refDoi"> DOI: 10.1021/jm301098e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301098e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9891-9899&author=J.+A.+Bergmanauthor=K.+Woanauthor=P.+Perez-Villarroelauthor=A.+Villagraauthor=E.+M.+Sotomayorauthor=A.+P.+Kozikowski&title=Selective+Histone+Deacetylase+6+Inhibitors+Bearing+Substituted+Urea+Linkers+Inhibit+Melanoma+Cell+Growth&doi=10.1021%2Fjm301098e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Histone Deacetylase 6 Inhibitors Bearing Substituted Urea Linkers Inhibit Melanoma Cell Growth</span></div><div class="casAuthors">Bergman, Joel A.; Woan, Karrune; Perez-Villarroel, Patricio; Villagra, Alejandro; Sotomayor, Eduardo M.; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9891-9899</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The incidence of malignant melanoma has dramatically increased in recent years thus requiring the need for improved therapeutic strategies.  In our efforts to design selective histone deactylase inhibitors (HDACI), we discovered that the aryl urea 1 is a modestly potent yet nonselective inhibitor.  Structure-activity relationship studies revealed that adding substituents to the nitrogen atom of the urea so as to generate compds. bearing a branched linker group results in increased potency and selectivity for HDAC6.  Compd. 5g shows low nanomolar inhibitory potency against HDAC6 and a selectivity of ∼600-fold relative to the inhibition of HDAC1.  These HDACIs were evaluated for their ability to inhibit the growth of B16 melanoma cells with the most potent and selective HDAC6I being found to decrease tumor cell growth.  To the best of our knowledge, this work constitutes the first report of HDAC6-selective inhibitors that possess antiproliferative effects against melanoma cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTkTtPLqvwsrVg90H21EOLACvtfcHk0liJ032v_q9KIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsrbJ&md5=d6be4c0efb3922fc35d814ba00ae9ed1</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm301098e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301098e%26sid%3Dliteratum%253Aachs%26aulast%3DBergman%26aufirst%3DJ.%2BA.%26aulast%3DWoan%26aufirst%3DK.%26aulast%3DPerez-Villarroel%26aufirst%3DP.%26aulast%3DVillagra%26aufirst%3DA.%26aulast%3DSotomayor%26aufirst%3DE.%2BM.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DSelective%2520Histone%2520Deacetylase%25206%2520Inhibitors%2520Bearing%2520Substituted%2520Urea%2520Linkers%2520Inhibit%2520Melanoma%2520Cell%2520Growth%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9891%26epage%3D9899%26doi%3D10.1021%2Fjm301098e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Woan, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lienlaf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Villaroel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrios, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahakian, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canales, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marante, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malley, K. S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seto, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinilla-Ibarz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarnaik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Celis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sotomayor, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villagra, A.</span></span> <span> </span><span class="NLM_article-title">Targeting Histone Deacetylase 6 Mediates a Dual Anti-Melanoma Effect: Enhanced Antitumor Immunity and Impaired Cell Proliferation</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1447</span>– <span class="NLM_lpage">1457</span>, <span class="refDoi"> DOI: 10.1016/j.molonc.2015.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1016%2Fj.molonc.2015.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=25957812" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsF2gtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=1447-1457&author=K.+V.+Woanauthor=M.+Lienlafauthor=P.+Perez-Villaroelauthor=C.+Leeauthor=F.+Chengauthor=T.+Knoxauthor=D.+M.+Woodsauthor=K.+Barriosauthor=J.+Powersauthor=E.+Sahakianauthor=H.+W.+Wangauthor=J.+Canalesauthor=D.+Maranteauthor=K.+S.+M.+Malleyauthor=J.+Bergmanauthor=E.+Setoauthor=A.+Kozikowskiauthor=J.+Pinilla-Ibarzauthor=A.+Sarnaikauthor=E.+Celisauthor=J.+Weberauthor=E.+M.+Sotomayorauthor=A.+Villagra&title=Targeting+Histone+Deacetylase+6+Mediates+a+Dual+Anti-Melanoma+Effect%3A+Enhanced+Antitumor+Immunity+and+Impaired+Cell+Proliferation&doi=10.1016%2Fj.molonc.2015.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting histone deacetylase 6 mediates a dual anti-melanoma effect: Enhanced antitumor immunity and impaired cell proliferation</span></div><div class="casAuthors">Woan, K. V.; Lienlaf, M.; Perez-Villaroel, P.; Lee, C.; Cheng, F.; Knox, T.; Woods, D. M.; Barrios, K.; Powers, J.; Sahakian, E.; Wang, H. W.; Canales, J.; Marante, D.; Smalley, K. S. M.; Bergman, J.; Seto, E.; Kozikowski, A.; Pinilla-Ibarz, J.; Sarnaik, A.; Celis, E.; Weber, J.; Sotomayor, E. M.; Villagra, A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1447-1457</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-7891</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The median survival for metastatic melanoma is in the realm of 8-16 mo and there are few therapies that offer significant improvement in overall survival.  One of the recent advances in cancer treatment focuses on epigenetic modifiers to alter the survivability and immunogenicity of cancer cells.  Our group and others have previously demonstrated that pan-HDAC inhibitors induce apoptosis, cell cycle arrest and changes in the immunogenicity of melanoma cells.  Here we interrogated specific HDACs which may be responsible for this effect.  We found that both genetic abrogation and pharmacol. inhibition of HDAC6 decreases in vitro proliferation and induces G1 arrest of melanoma cell lines without inducing apoptosis.  Moreover, targeting this mol. led to an important upregulation in the expression of tumor assocd. antigens and MHC class I, suggesting a potential improvement in the immunogenicity of these cells.  Of note, this anti-melanoma activity was operative regardless of mutational status of the cells.  These effects translated into a pronounced delay of in vivo melanoma tumor growth which was, at least in part, dependent on intact immunity as evidenced by the restoration of tumor growth after CD4+ and CD8+ depletion.  Given our findings, we provide the initial rationale for the further development of selective HDAC6 inhibitors as potential therapeutic anti-melanoma agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDIQ5LKbZF8bVg90H21EOLACvtfcHk0lg2mCD5lDQAYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsF2gtLw%253D&md5=7534df9db01f52a71f97f0e112db37cc</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.molonc.2015.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molonc.2015.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DWoan%26aufirst%3DK.%2BV.%26aulast%3DLienlaf%26aufirst%3DM.%26aulast%3DPerez-Villaroel%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DCheng%26aufirst%3DF.%26aulast%3DKnox%26aufirst%3DT.%26aulast%3DWoods%26aufirst%3DD.%2BM.%26aulast%3DBarrios%26aufirst%3DK.%26aulast%3DPowers%26aufirst%3DJ.%26aulast%3DSahakian%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DH.%2BW.%26aulast%3DCanales%26aufirst%3DJ.%26aulast%3DMarante%26aufirst%3DD.%26aulast%3DMalley%26aufirst%3DK.%2BS.%2BM.%26aulast%3DBergman%26aufirst%3DJ.%26aulast%3DSeto%26aufirst%3DE.%26aulast%3DKozikowski%26aufirst%3DA.%26aulast%3DPinilla-Ibarz%26aufirst%3DJ.%26aulast%3DSarnaik%26aufirst%3DA.%26aulast%3DCelis%26aufirst%3DE.%26aulast%3DWeber%26aufirst%3DJ.%26aulast%3DSotomayor%26aufirst%3DE.%2BM.%26aulast%3DVillagra%26aufirst%3DA.%26atitle%3DTargeting%2520Histone%2520Deacetylase%25206%2520Mediates%2520a%2520Dual%2520Anti-Melanoma%2520Effect%253A%2520Enhanced%2520Antitumor%2520Immunity%2520and%2520Impaired%2520Cell%2520Proliferation%26jtitle%3DMol.%2520Oncol.%26date%3D2015%26volume%3D9%26spage%3D1447%26epage%3D1457%26doi%3D10.1016%2Fj.molonc.2015.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanschagrin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span> <span> </span><span class="NLM_article-title">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6177</span>– <span class="NLM_lpage">6196</span>, <span class="refDoi"> DOI: 10.1021/jm051256o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+Precision+Glide%3A+Docking+and+Scoring+Incorporating+a+Model+of+Hydrophobic+Enclosure+for+Protein-Ligand+Complexes&doi=10.1021%2Fjm051256o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span></div><div class="casAuthors">Friesner, Richard A.; Murphy, Robert B.; Repasky, Matthew P.; Frye, Leah L.; Greenwood, Jeremy R.; Halgren, Thomas A.; Sanschagrin, Paul C.; Mainz, Daniel T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6177-6196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel scoring function to est. protein-ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol.  In addn. to unique water desolvation energy terms, protein-ligand structural motifs leading to enhanced binding affinity are included:(1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral-neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged-charged hydrogen bonds.  The XP scoring function and docking protocol have been developed to reproduce exptl. binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, resp.) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance.  Enrichment results demonstrate the importance of the novel XP mol. recognition and water scoring in sepg. active and inactive ligands and avoiding false positives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZiwN12vaY7Vg90H21EOLACvtfcHk0lg2mCD5lDQAYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D&md5=ea428c82ead0d8c27f8c1a7b694a1edf</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520Precision%2520Glide%253A%2520Docking%2520and%2520Scoring%2520Incorporating%2520a%2520Model%2520of%2520Hydrophobic%2520Enclosure%2520for%2520Protein-Ligand%2520Complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196%26doi%3D10.1021%2Fjm051256o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 2. Enrichment Factors in Database Screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+2.+Enrichment+Factors+in+Database+Screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lhiYfKattNWGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25202.%2520Enrichment%2520Factors%2520in%2520Database%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+1.+Method+and+Assessment+of+Docking+Accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lhiYfKattNWGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25201.%2520Method%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Padige, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negmeldin, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pflum, M. K. H.</span></span> <span> </span><span class="NLM_article-title">Development of an ELISA-Based HDAC Activity Assay for Characterization of Isoform-Selective Inhibitors</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1277</span>– <span class="NLM_lpage">1285</span>, <span class="refDoi"> DOI: 10.1177/1087057115598118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1177%2F1087057115598118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=26232305" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC28XlvVCktLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=1277-1285&author=G.+Padigeauthor=A.+T.+Negmeldinauthor=M.+K.+H.+Pflum&title=Development+of+an+ELISA-Based+HDAC+Activity+Assay+for+Characterization+of+Isoform-Selective+Inhibitors&doi=10.1177%2F1087057115598118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Development of an ELISA-based HDAC activity assay for characterization of isoform-selective inhibitors</span></div><div class="casAuthors">Padige, Geetha; Negmeldin, Ahmed T.; Pflum, Mary Kay H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1277-1285</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Histone deacetylase (HDAC) proteins are promising targets for cancer treatment, with several HDAC inhibitors used clin. as anticancer drugs.  Most HDAC inhibitors nonspecifically interact with all or many of the 11 HDAC isoforms.  Isoform-selective HDAC inhibitors would be useful tools to dissect the individual functions of HDAC proteins in cancer formation, in addn. to potentially displaying effective anticancer properties.  We report here a robust HDAC activity assay for screening selective HDAC inhibitors, which is inspired by the traditional ELISA (ELISA).  The key feature of the ELISA-based HDAC activity assay is use of mammalian cell-derived HDAC isoforms instead of recombinant proteins.  Importantly, the assay was validated with several known HDAC inhibitors.  The ELISA based HDAC activity assay will facilitate the characterization of isoform-selective HDAC inhibitors against mammalian cell-derived HDAC proteins, which will enhance HDAC-centered cancer research and provide a foundation for anticancer drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCPMJ2yMSbrLVg90H21EOLACvtfcHk0lhiYfKattNWGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlvVCktLs%253D&md5=f7cc8b4f3d289c8f616c6ccef6c9459e</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1177%2F1087057115598118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057115598118%26sid%3Dliteratum%253Aachs%26aulast%3DPadige%26aufirst%3DG.%26aulast%3DNegmeldin%26aufirst%3DA.%2BT.%26aulast%3DPflum%26aufirst%3DM.%2BK.%2BH.%26atitle%3DDevelopment%2520of%2520an%2520ELISA-Based%2520HDAC%2520Activity%2520Assay%2520for%2520Characterization%2520of%2520Isoform-Selective%2520Inhibitors%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2015%26volume%3D20%26spage%3D1277%26epage%3D1285%26doi%3D10.1177%2F1087057115598118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crispatzu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Ghaili, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weit, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florou, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newrzela, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span> <span> </span><span class="NLM_article-title">Models for Mature T-Cell Lymphomas-A Critical Appraisal of Experimental Systems and Their Contribution to Current T-Cell Tumorigenic Concepts</span>. <i>Critical Reviews in Oncology/Hematology.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">680</span>– <span class="NLM_lpage">695</span>, <span class="refDoi"> DOI: 10.1016/j.critrevonc.2013.07.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1016%2Fj.critrevonc.2013.07.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=23972664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A280%3ADC%252BC3sbjvVegsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2013&pages=680-695&author=K.+Warnerauthor=G.+Crispatzuauthor=N.+Al-Ghailiauthor=N.+Weitauthor=V.+Florouauthor=M.+J.+Youauthor=S.+Newrzelaauthor=M.+Herling&title=Models+for+Mature+T-Cell+Lymphomas-A+Critical+Appraisal+of+Experimental+Systems+and+Their+Contribution+to+Current+T-Cell+Tumorigenic+Concepts&doi=10.1016%2Fj.critrevonc.2013.07.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Models for mature T-cell lymphomas--a critical appraisal of experimental systems and their contribution to current T-cell tumorigenic concepts</span></div><div class="casAuthors">Warner Kathrin; Crispatzu Giuliano; Al-Ghaili Nabil; Weit Nicole; Florou Vaia; You M James; Newrzela Sebastian; Herling Marco</div><div class="citationInfo"><span class="NLM_cas:title">Critical reviews in oncology/hematology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">680-95</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Mature T-cell lymphomas/leukemias (MTCL) have been understudied lymphoid neoplasms that currently receive growing attention.  Our historically rudimentary molecular understanding and dissatisfactory interventional success in this complex and for the most part poor-prognostic group of tumors is only slightly improving.  A major limiting aspect in further progress in these rare neoplasms is the lack of suitable model systems that would substantially facilitate pathogenic studies and pre-clinical drug evaluations.  Such representations of MTCL have thus far not been systematically appraised.  We therefore provide an overview on existing models and point out their particular advantages and limitations in the context of the specific scientific questions.  After addressing issues of species-specific differences and classifications, we summarize data on MTCL cell lines of human as well as murine origin, on murine strain predispositions to MTCL, on available models of genetically engineered mice, and on transplant systems.  From an in-silico meta-analysis of available primary data of gene expression profiles on human MTCL we cross-reference genes reported to transform T-cells in mice and reflect on their general vs entity-restricted relevance and on target-promoter influences.  Overall, we identify the urgent need for new models of higher fidelity to human MTCL with respect to their increasingly recognized diversity and to predictions of drug response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRG8YhZZYBEXs25P7XSLhESfW6udTcc2ebUrrvrPY18y7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sbjvVegsQ%253D%253D&md5=5f857d45628624e7f732ff2ddf6bb4fc</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.critrevonc.2013.07.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.critrevonc.2013.07.014%26sid%3Dliteratum%253Aachs%26aulast%3DWarner%26aufirst%3DK.%26aulast%3DCrispatzu%26aufirst%3DG.%26aulast%3DAl-Ghaili%26aufirst%3DN.%26aulast%3DWeit%26aufirst%3DN.%26aulast%3DFlorou%26aufirst%3DV.%26aulast%3DYou%26aufirst%3DM.%2BJ.%26aulast%3DNewrzela%26aufirst%3DS.%26aulast%3DHerling%26aufirst%3DM.%26atitle%3DModels%2520for%2520Mature%2520T-Cell%2520Lymphomas-A%2520Critical%2520Appraisal%2520of%2520Experimental%2520Systems%2520and%2520Their%2520Contribution%2520to%2520Current%2520T-Cell%2520Tumorigenic%2520Concepts%26jtitle%3DCritical%2520Reviews%2520in%2520Oncology%252FHematology.%26date%3D2013%26volume%3D88%26spage%3D680%26epage%3D695%26doi%3D10.1016%2Fj.critrevonc.2013.07.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Orlova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wingelhofer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubauer, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger-Becvar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miklós Keserű, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriggl, R.</span></span> <span> </span><span class="NLM_article-title">Emerging Therapeutic Targets in Myeloproliferative Neoplasms and Peripheral T-Cell Leukemia and Lymphomas</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">57</span>, <span class="refDoi"> DOI: 10.1080/14728222.2018.1406924</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1080%2F14728222.2018.1406924" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=29148847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFaht7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=45-57&author=A.+Orlovaauthor=B.+Wingelhoferauthor=H.+A.+Neubauerauthor=B.+Maurerauthor=A.+Berger-Becvarauthor=G.+Mikl%C3%B3s+Keser%C5%B1author=P.+T.+Gunningauthor=P.+Valentauthor=R.+Moriggl&title=Emerging+Therapeutic+Targets+in+Myeloproliferative+Neoplasms+and+Peripheral+T-Cell+Leukemia+and+Lymphomas&doi=10.1080%2F14728222.2018.1406924"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging therapeutic targets in myeloproliferative neoplasms and peripheral T-cell leukemia and lymphomas</span></div><div class="casAuthors">Orlova, Anna; Wingelhofer, Bettina; Neubauer, Heidi A.; Maurer, Barbara; Berger-Becvar, Angelika; Keseru, Gyoergy Miklos; Gunning, Patrick T.; Valent, Peter; Moriggl, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-57</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  : Hematopoietic neoplasms are often driven by gain-of-function mutations of the JAK-STAT pathway together with mutations in chromatin remodeling and DNA damage control pathways.  The interconnection between the JAK-STAT pathway, epigenetic regulation or DNA damage control is still poorly understood in cancer cell biol.  Here, we focus on a broader description of mutational insights into myeloproliferative neoplasms and peripheral T-cell leukemia and lymphomas, since sequencing efforts have identified similar combinations of driver mutations in these diseases covering different lineages.  We summarize how these pathways might be interconnected in normal or cancer cells, which have lost differentiation capacity and drive oncogene transcription.  Due to similarities in driver mutations including epigenetic enzymes, JAK-STAT pathway activation and mutated checkpoint control through TP53, we hypothesize that similar therapeutic approaches could be of benefit in these diseases.  We give an overview of how driver mutations in these malignancies contribute to hematopoietic cancer initiation or progression, and how these pathways can be targeted with currently available tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIQGAsv8OacLVg90H21EOLACvtfcHk0lgkY2GzoloWzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFaht7%252FM&md5=989c76e5954a7e46ccf27fa8ebea5530</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1080%2F14728222.2018.1406924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14728222.2018.1406924%26sid%3Dliteratum%253Aachs%26aulast%3DOrlova%26aufirst%3DA.%26aulast%3DWingelhofer%26aufirst%3DB.%26aulast%3DNeubauer%26aufirst%3DH.%2BA.%26aulast%3DMaurer%26aufirst%3DB.%26aulast%3DBerger-Becvar%26aufirst%3DA.%26aulast%3DMikl%25C3%25B3s%2BKeser%25C5%25B1%26aufirst%3DG.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DMoriggl%26aufirst%3DR.%26atitle%3DEmerging%2520Therapeutic%2520Targets%2520in%2520Myeloproliferative%2520Neoplasms%2520and%2520Peripheral%2520T-Cell%2520Leukemia%2520and%2520Lymphomas%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2018%26volume%3D22%26spage%3D45%26epage%3D57%26doi%3D10.1080%2F14728222.2018.1406924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dürig, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bug, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein-Hitpass, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jöns, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Subero, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baudis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dührsen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siebert, R.</span></span> <span> </span><span class="NLM_article-title">Combined Single Nucleotide Polymorphism-Based Genomic Mapping and Global Gene Expression Profiling Identifies Novel Chromosomal Imbalances, Mechanisms and Candidate Genes Important in the Pathogenesis of T-Cell Prolymphocytic Leukemia with Inv(14)(Q11q32)</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2153</span>– <span class="NLM_lpage">2163</span>, <span class="refDoi"> DOI: 10.1038/sj.leu.2404877</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fsj.leu.2404877" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=17713554" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVGmsb7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=2153-2163&author=J.+D%C3%BCrigauthor=S.+Bugauthor=L.+Klein-Hitpassauthor=T.+Boesauthor=T.+J%C3%B6nsauthor=J.+I.+Martin-Suberoauthor=L.+Harderauthor=M.+Baudisauthor=U.+D%C3%BChrsenauthor=R.+Siebert&title=Combined+Single+Nucleotide+Polymorphism-Based+Genomic+Mapping+and+Global+Gene+Expression+Profiling+Identifies+Novel+Chromosomal+Imbalances%2C+Mechanisms+and+Candidate+Genes+Important+in+the+Pathogenesis+of+T-Cell+Prolymphocytic+Leukemia+with+Inv%2814%29%28Q11q32%29&doi=10.1038%2Fsj.leu.2404877"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Combined single nucleotide polymorphism-based genomic mapping and global gene expression profiling identifies novel chromosomal imbalances, mechanisms and candidate genes important in the pathogenesis of T-cell prolymphocytic leukemia with inv(14)(q11q32)</span></div><div class="casAuthors">Duerig, J.; Bug, S.; Klein-Hitpass, L.; Boes, T.; Joens, T.; Martin-Subero, J. I.; Harder, L.; Baudis, M.; Duehrsen, U.; Siebert, R.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2153-2163</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">T-cell prolymphocytic leukemia (T-PLL) is a rare aggressive lymphoma derived from mature T cells, which is, in most cases, characterized by the presence of an inv(14)(q11q32)/t(14;14)(q11;q32) and a characteristic pattern of secondary chromosomal aberrations.  DNA microarray technol. was employed to compare the transcriptomes of eight immunomagnetically purified CD3+ normal donor-derived peripheral blood cell samples, with five highly purified inv(14)/t(14;14)-pos. T-PLL blood samples.  Between the two exptl. groups, 734 genes were identified as differentially expressed, including functionally important genes involved in lymphomagenesis, cell cycle regulation, apoptosis and DNA repair.  Notably, the differentially expressed genes were found to be significantly enriched in genomic regions affected by recurrent chromosomal imbalances.  Upregulated genes clustered on chromosome arms 6p and 8q, and downregulated genes on 6q, 8p, 10p, 11q and 18p.  High-resoln. copy-no. detn. using single nucleotide polymorphism chip technol. in 12 inv(14)/t(14;14)-pos. T-PLL including those analyzed for gene expression, refined chromosomal breakpoints as well as regions of imbalances.  In conclusion, combined transcriptional and mol. cytogenetic profiling identified novel specific chromosomal loci and genes that are likely to be involved in disease progression and suggests a gene dosage effect as a pathogenic mechanism in T-PLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9hzDAmZA-srVg90H21EOLACvtfcHk0lgkY2GzoloWzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVGmsb7I&md5=611eb203d8c8daa965c5d5e29b2a96cf</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fsj.leu.2404877&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.leu.2404877%26sid%3Dliteratum%253Aachs%26aulast%3DD%25C3%25BCrig%26aufirst%3DJ.%26aulast%3DBug%26aufirst%3DS.%26aulast%3DKlein-Hitpass%26aufirst%3DL.%26aulast%3DBoes%26aufirst%3DT.%26aulast%3DJ%25C3%25B6ns%26aufirst%3DT.%26aulast%3DMartin-Subero%26aufirst%3DJ.%2BI.%26aulast%3DHarder%26aufirst%3DL.%26aulast%3DBaudis%26aufirst%3DM.%26aulast%3DD%25C3%25BChrsen%26aufirst%3DU.%26aulast%3DSiebert%26aufirst%3DR.%26atitle%3DCombined%2520Single%2520Nucleotide%2520Polymorphism-Based%2520Genomic%2520Mapping%2520and%2520Global%2520Gene%2520Expression%2520Profiling%2520Identifies%2520Novel%2520Chromosomal%2520Imbalances%252C%2520Mechanisms%2520and%2520Candidate%2520Genes%2520Important%2520in%2520the%2520Pathogenesis%2520of%2520T-Cell%2520Prolymphocytic%2520Leukemia%2520with%2520Inv%252814%2529%2528Q11q32%2529%26jtitle%3DLeukemia%26date%3D2007%26volume%3D21%26spage%3D2153%26epage%3D2163%26doi%3D10.1038%2Fsj.leu.2404877" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patil, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieslak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhart, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toprak, U. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagener, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiehle, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altmüller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franitza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scholz, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayne, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahearne, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheffold, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jebaraj, B. M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyer, M. J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurnberg, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durig, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottcher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schesner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stilgenbauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macintyre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siebert, R.</span></span> <span> </span><span class="NLM_article-title">Reconstruction of Rearranged T-Cell Receptor Loci by Whole Genome and Transcriptome Sequencing Gives Insights into the Initial Steps of T-Cell Prolymphocytic Leukemia</span>. <i>Genes, Chromosomes Cancer</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">267</span>, <span class="refDoi"> DOI: 10.1002/gcc.22821</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1002%2Fgcc.22821" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=31677197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlt1WgtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=261-267&author=P.+Patilauthor=A.+Cieslakauthor=S.+H.+Bernhartauthor=U.+H.+Toprakauthor=R.+Wagenerauthor=C.+L%C3%B3pezauthor=L.+Wiehleauthor=S.+Bensauthor=J.+Altm%C3%BCllerauthor=M.+Franitzaauthor=I.+Scholzauthor=S.+Jayneauthor=M.+J.+Ahearneauthor=A.+Scheffoldauthor=B.+M.+C.+Jebarajauthor=C.+Schneiderauthor=D.+Costaauthor=T.+Braunauthor=A.+Schraderauthor=E.+Campoauthor=M.+J.+S.+Dyerauthor=P.+Nurnbergauthor=J.+Durigauthor=S.+Bottcherauthor=M.+Schesnerauthor=M.+Herlingauthor=S.+Stilgenbauerauthor=E.+Macintyreauthor=R.+Siebert&title=Reconstruction+of+Rearranged+T-Cell+Receptor+Loci+by+Whole+Genome+and+Transcriptome+Sequencing+Gives+Insights+into+the+Initial+Steps+of+T-Cell+Prolymphocytic+Leukemia&doi=10.1002%2Fgcc.22821"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Reconstruction of rearranged T-cell receptor loci by whole genome and transcriptome sequencing gives insights into the initial steps of T-cell prolymphocytic leukemia</span></div><div class="casAuthors">Patil, Paurnima; Cieslak, Agata; Bernhart, Stephan H.; Toprak, Umut H.; Wagener, Rabea; Lopez, Cristina; Wiehle, Laura; Bens, Susanne; Altmueller, Janine; Franitza, Marek; Scholz, Ingrid; Jayne, Sandrine; Ahearne, Matthew J.; Scheffold, Annika; Jebaraj, Billy M. C.; Schneider, Christof; Costa, Dolors; Braun, Till; Schrader, Alexandra; Campo, Elias; Dyer, Martin J. S.; Nuernberg, Peter; Duerig, Jan; Johansson, Patricia; Boettcher, Sebastian; Schlesner, Matthias; Herling, Marco; Stilgenbauer, Stephan; Macintyre, Elizabeth; Siebert, Reiner</div><div class="citationInfo"><span class="NLM_cas:title">Genes, Chromosomes & Cancer</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">261-267</span>CODEN:
                <span class="NLM_cas:coden">GCCAES</span>;
        ISSN:<span class="NLM_cas:issn">1045-2257</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">T-cell prolymphocytic leukemia (T-PLL) is an aggressive tumor with leukemic presentation of mature T-lymphocytes.  Here, we aimed at characterizing the initial events in the mol. pathogenesis of T-PLL and particularly, at detg. the point in T-cell differentiation when the hallmark oncogenic events, i.e., inv(14)(q11q32)/t(14;14)(q11;q32) and t(X;14)(q28;q11) occur.  To this end, we mined whole genome and transcriptome sequencing data of 17 and 11 T-PLL cases, resp.  Mapping of the 14q32.1 locus breakpoints identified only TCL1A, which was moreover significantly overexpressed in T-PLL as compared to benign CD4+ and CD8+ T-cells, as the only common oncogenic target of aberrations.  In cases with t(14;14), the breakpoints mapped telomeric and in cases with inv(14) centromeric or in the 3'-untranslated region of TCL1A.  Regarding the T-cell receptor alpha (TRA) locus-TCL1A breakpoint junctions, all 17 breakpoints involved recombination signal sequences and 15 junctions contained nontemplated (N-) nucleotides.  All T-PLL cases studied carried in-frame TRA rearrangements on the intact allele, which skewed significantly toward usage of distal/central TRAV/TRAJ gene segments as compared to the illegitimate TRA rearrangements.  Our findings suggest that the oncogenic TRA-TCL1A/MTCP1 rearrangements in T-PLL occur during opening of the TRA locus, i.e., during the progression from CD4+ immature single pos. to early double pos. thymocyte stage, just before physiol. TCL1A expression is silenced.  The cell carrying such an oncogenic event continues maturation and rearranges the second TRA allele to achieve a functional T-cell receptor.  Thereafter, it switches off RAG and DNTT expression in line with the mature T-cell phenotype at presentation of T-PLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnLsOdYbvKB7Vg90H21EOLACvtfcHk0lj1Y0pXm9F3kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlt1WgtLw%253D&md5=4e0f8b13b816524524d1de9bf03d5e93</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fgcc.22821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fgcc.22821%26sid%3Dliteratum%253Aachs%26aulast%3DPatil%26aufirst%3DP.%26aulast%3DCieslak%26aufirst%3DA.%26aulast%3DBernhart%26aufirst%3DS.%2BH.%26aulast%3DToprak%26aufirst%3DU.%2BH.%26aulast%3DWagener%26aufirst%3DR.%26aulast%3DL%25C3%25B3pez%26aufirst%3DC.%26aulast%3DWiehle%26aufirst%3DL.%26aulast%3DBens%26aufirst%3DS.%26aulast%3DAltm%25C3%25BCller%26aufirst%3DJ.%26aulast%3DFranitza%26aufirst%3DM.%26aulast%3DScholz%26aufirst%3DI.%26aulast%3DJayne%26aufirst%3DS.%26aulast%3DAhearne%26aufirst%3DM.%2BJ.%26aulast%3DScheffold%26aufirst%3DA.%26aulast%3DJebaraj%26aufirst%3DB.%2BM.%2BC.%26aulast%3DSchneider%26aufirst%3DC.%26aulast%3DCosta%26aufirst%3DD.%26aulast%3DBraun%26aufirst%3DT.%26aulast%3DSchrader%26aufirst%3DA.%26aulast%3DCampo%26aufirst%3DE.%26aulast%3DDyer%26aufirst%3DM.%2BJ.%2BS.%26aulast%3DNurnberg%26aufirst%3DP.%26aulast%3DDurig%26aufirst%3DJ.%26aulast%3DBottcher%26aufirst%3DS.%26aulast%3DSchesner%26aufirst%3DM.%26aulast%3DHerling%26aufirst%3DM.%26aulast%3DStilgenbauer%26aufirst%3DS.%26aulast%3DMacintyre%26aufirst%3DE.%26aulast%3DSiebert%26aufirst%3DR.%26atitle%3DReconstruction%2520of%2520Rearranged%2520T-Cell%2520Receptor%2520Loci%2520by%2520Whole%2520Genome%2520and%2520Transcriptome%2520Sequencing%2520Gives%2520Insights%2520into%2520the%2520Initial%2520Steps%2520of%2520T-Cell%2520Prolymphocytic%2520Leukemia%26jtitle%3DGenes%252C%2520Chromosomes%2520Cancer%26date%3D2020%26volume%3D59%26spage%3D261%26epage%3D267%26doi%3D10.1002%2Fgcc.22821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pützer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufva, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crispatzu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oles, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adnan-Awad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagstrom, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellanger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mpindi, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldfors, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pemovska, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietarinen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lauhio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuesta-Matoes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koschmieder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brummendorf, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kytola, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savolainen, E.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siitonen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellonen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallioniemi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wennerberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aittokallio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zenz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustjoki, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel Drug Sensitivities in T-PLL by High-Throughput Ex Vivo Drug Testing and Mutation Profiling</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">774</span>– <span class="NLM_lpage">787</span>, <span class="refDoi"> DOI: 10.1038/leu.2017.252</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fleu.2017.252" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=28804127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKmu7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=774-787&author=E.+I.+Anderssonauthor=S.+P%C3%BCtzerauthor=B.+Yadavauthor=O.+Dufvaauthor=S.+Khanauthor=L.+Heauthor=L.+Sellnerauthor=A.+Schraderauthor=G.+Crispatzuauthor=M.+Olesauthor=H.+Zhangauthor=S.+Adnan-Awadauthor=S.+Lagstromauthor=D.+Bellangerauthor=J.+P.+Mpindiauthor=S.+Eldforsauthor=T.+Pemovskaauthor=S.+Pietarinenauthor=A.+Lauhioauthor=K.+Tomskaauthor=C.+Cuesta-Matoesauthor=E.+Faberauthor=S.+Koschmiederauthor=T.+H.+Brummendorfauthor=S.+Kytolaauthor=E.-R.+Savolainenauthor=T.+Siitonenauthor=P.+Ellonenauthor=O.+Kallioniemiauthor=K.+Wennerbergauthor=W.+Dingauthor=M.+Sternauthor=W.+Huberauthor=S.+Andersauthor=J.+Tangauthor=T.+Aittokallioauthor=T.+Zenzauthor=M.+Herlingauthor=S.+Mustjoki&title=Discovery+of+Novel+Drug+Sensitivities+in+T-PLL+by+High-Throughput+Ex+Vivo+Drug+Testing+and+Mutation+Profiling&doi=10.1038%2Fleu.2017.252"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel drug sensitivities in T-PLL by high-throughput ex vivo drug testing and mutation profiling</span></div><div class="casAuthors">Andersson, E. I.; Putzer, S.; Yadav, B.; Dufva, O.; Khan, S.; He, L.; Sellner, L.; Schrader, A.; Crispatzu, G.; Oles, M.; Zhang, H.; Adnan-Awad, S.; Lagstrom, S.; Bellanger, D.; Mpindi, J. P.; Eldfors, S.; Pemovska, T.; Pietarinen, P.; Lauhio, A.; Tomska, K.; Cuesta-Mateos, C.; Faber, E.; Koschmieder, S.; Brummendorf, T. H.; Kytola, S.; Savolainen, E.-R.; Siitonen, T.; Ellonen, P.; Kallioniemi, O.; Wennerberg, K.; Ding, W.; Stern, M.-H.; Huber, W.; Anders, S.; Tang, J.; Aittokallio, T.; Zenz, T.; Herling, M.; Mustjoki, S.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">774-787</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">T-cell prolymphocytic leukemia (T-PLL) is a rare and aggressive neoplasm of mature T-cells with an urgent need for rationally designed therapies to address its notoriously chemo-refractory behavior.  The median survival of T-PLL patients is <2 years and clin. trials are difficult to execute.  Here we systematically explored the diversity of drug responses in T-PLL patient samples using an ex vivo drug sensitivity and resistance testing platform and correlated the findings with somatic mutations and gene expression profiles.  Intriguingly, all T-PLL samples were sensitive to the cyclin-dependent kinase inhibitor SNS-032, which overcame stromal-cell-mediated protection and elicited robust p53-activation and apoptosis.  Across all patients, the most effective classes of compds. were histone deacetylase, phosphoinositide-3 kinase/AKT/mammalian target of rapamycin, heat-shock protein 90 and BH3-family protein inhibitors as well as p53 activators, indicating previously unexplored, novel targeted approaches for treating T-PLL.  Although Janus-activated kinase-signal transducer and activator of transcription factor (JAK-STAT) pathway mutations were common in T-PLL (71% of patients), JAK-STAT inhibitor responses were not directly linked to those or other T-PLL-specific lesions.  Overall, we found that genetic markers do not readily translate into novel effective therapeutic vulnerabilities.  In conclusion, novel classes of compds. with high efficacy in T-PLL were discovered with the comprehensive ex vivo drug screening platform warranting further studies of synergisms and clin. testing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNlAKw7-znF7Vg90H21EOLACvtfcHk0lj1Y0pXm9F3kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKmu7%252FI&md5=70fedb2dce547d45c70300ec7759ee07</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fleu.2017.252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2017.252%26sid%3Dliteratum%253Aachs%26aulast%3DAndersson%26aufirst%3DE.%2BI.%26aulast%3DP%25C3%25BCtzer%26aufirst%3DS.%26aulast%3DYadav%26aufirst%3DB.%26aulast%3DDufva%26aufirst%3DO.%26aulast%3DKhan%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DL.%26aulast%3DSellner%26aufirst%3DL.%26aulast%3DSchrader%26aufirst%3DA.%26aulast%3DCrispatzu%26aufirst%3DG.%26aulast%3DOles%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DAdnan-Awad%26aufirst%3DS.%26aulast%3DLagstrom%26aufirst%3DS.%26aulast%3DBellanger%26aufirst%3DD.%26aulast%3DMpindi%26aufirst%3DJ.%2BP.%26aulast%3DEldfors%26aufirst%3DS.%26aulast%3DPemovska%26aufirst%3DT.%26aulast%3DPietarinen%26aufirst%3DS.%26aulast%3DLauhio%26aufirst%3DA.%26aulast%3DTomska%26aufirst%3DK.%26aulast%3DCuesta-Matoes%26aufirst%3DC.%26aulast%3DFaber%26aufirst%3DE.%26aulast%3DKoschmieder%26aufirst%3DS.%26aulast%3DBrummendorf%26aufirst%3DT.%2BH.%26aulast%3DKytola%26aufirst%3DS.%26aulast%3DSavolainen%26aufirst%3DE.-R.%26aulast%3DSiitonen%26aufirst%3DT.%26aulast%3DEllonen%26aufirst%3DP.%26aulast%3DKallioniemi%26aufirst%3DO.%26aulast%3DWennerberg%26aufirst%3DK.%26aulast%3DDing%26aufirst%3DW.%26aulast%3DStern%26aufirst%3DM.%26aulast%3DHuber%26aufirst%3DW.%26aulast%3DAnders%26aufirst%3DS.%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DAittokallio%26aufirst%3DT.%26aulast%3DZenz%26aufirst%3DT.%26aulast%3DHerling%26aufirst%3DM.%26aulast%3DMustjoki%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520Novel%2520Drug%2520Sensitivities%2520in%2520T-PLL%2520by%2520High-Throughput%2520Ex%2520Vivo%2520Drug%2520Testing%2520and%2520Mutation%2520Profiling%26jtitle%3DLeukemia%26date%3D2018%26volume%3D32%26spage%3D774%26epage%3D787%26doi%3D10.1038%2Fleu.2017.252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pemovska, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szwajda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulesskiy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kontro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karjalainen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majumder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murumägi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knowles, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porkka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heckman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallioniemi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wennerberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aittokallio, T.</span></span> <span> </span><span class="NLM_article-title">Quantitative Scoring of Differential Drug Sensitivity for Individually Optimized Anticancer Therapies</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">5193</span>, <span class="refDoi"> DOI: 10.1038/srep05193</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fsrep05193" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2015&pages=5193&author=B.+Yadavauthor=T.+Pemovskaauthor=A.+Szwajdaauthor=E.+Kulesskiyauthor=M.+Kontroauthor=R.+Karjalainenauthor=M.+Majumderauthor=D.+Malaniauthor=A.+Murum%C3%A4giauthor=J.+Knowlesauthor=K.+Porkkaauthor=C.+Heckmanauthor=O.+Kallioniemiauthor=K.+Wennerbergauthor=T.+Aittokallio&title=Quantitative+Scoring+of+Differential+Drug+Sensitivity+for+Individually+Optimized+Anticancer+Therapies&doi=10.1038%2Fsrep05193"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1038%2Fsrep05193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep05193%26sid%3Dliteratum%253Aachs%26aulast%3DYadav%26aufirst%3DB.%26aulast%3DPemovska%26aufirst%3DT.%26aulast%3DSzwajda%26aufirst%3DA.%26aulast%3DKulesskiy%26aufirst%3DE.%26aulast%3DKontro%26aufirst%3DM.%26aulast%3DKarjalainen%26aufirst%3DR.%26aulast%3DMajumder%26aufirst%3DM.%26aulast%3DMalani%26aufirst%3DD.%26aulast%3DMurum%25C3%25A4gi%26aufirst%3DA.%26aulast%3DKnowles%26aufirst%3DJ.%26aulast%3DPorkka%26aufirst%3DK.%26aulast%3DHeckman%26aufirst%3DC.%26aulast%3DKallioniemi%26aufirst%3DO.%26aulast%3DWennerberg%26aufirst%3DK.%26aulast%3DAittokallio%26aufirst%3DT.%26atitle%3DQuantitative%2520Scoring%2520of%2520Differential%2520Drug%2520Sensitivity%2520for%2520Individually%2520Optimized%2520Anticancer%2520Therapies%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D4%26spage%3D5193%26doi%3D10.1038%2Fsrep05193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cosenza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Civallero, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcheselli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, S.</span></span> <span> </span><span class="NLM_article-title">Ricolinostat, a Selective HDAC6 Inhibitor, Shows Anti-Lymphoma Cell Activity Alone and in Combination with Bendamustine</span>. <i>Apoptosis</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">827</span>– <span class="NLM_lpage">840</span>, <span class="refDoi"> DOI: 10.1007/s10495-017-1364-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1007%2Fs10495-017-1364-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=28315173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslKktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=827-840&author=M.+Cosenzaauthor=M.+Civalleroauthor=L.+Marcheselliauthor=S.+Sacchiauthor=S.+Pozzi&title=Ricolinostat%2C+a+Selective+HDAC6+Inhibitor%2C+Shows+Anti-Lymphoma+Cell+Activity+Alone+and+in+Combination+with+Bendamustine&doi=10.1007%2Fs10495-017-1364-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Ricolinostat, a selective HDAC6 inhibitor, shows anti-lymphoma cell activity alone and in combination with bendamustine</span></div><div class="casAuthors">Cosenza, Maria; Civallero, Monica; Marcheselli, Luigi; Sacchi, Stefano; Pozzi, Samantha</div><div class="citationInfo"><span class="NLM_cas:title">Apoptosis</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">827-840</span>CODEN:
                <span class="NLM_cas:coden">APOPFN</span>;
        ISSN:<span class="NLM_cas:issn">1360-8185</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Histone deacetylase inhibitors (HDACis) have emerged as a new class of anticancer agents, targeting the biol. process including cell cycle and apoptosis.  We investigated and explained the anticancer effects of an HDAC6 inhibitor, ricolinostat alone and in combination with bendamustine in lymphoma cell lines.  Cell viability was measured by MTT assay.  Apoptosis, reactive oxygen species (ROS) generation, Bcl-2 protein expression, cell cycle progression and tubuline expression were detd. by flow cytometry.  The effects of ricolinostat alone and in combination on the caspases, PI3K/Akt, Bcl-2 pathways, ER stress and UPR were assessed by immunoblotting.  Ricolinostat shows anti lymphoma activity when used as single agent and its capability to induce apoptosis is synergistically potentiated by the bendamustine in lymphoma cell lines.  Drug combination reduced the proportion of cells in the G0/G1 and S phases and caused an increase of "sub-G0/G1" peak.  The synergistic effect accompanied with the increased ROS, activation of caspase-8, -9, and -3, the cleavage of PARP and modulated by Bcl-2 proteins family.  In addn., the exposure of ricolinostat induced the acetylation level of α-tubulin, the extend of which was not further modified by bendamustine.  Finally, the apoptosis effect of ricolinostat/bendamustine may be mediated by a corresponding effect on microtubule stabilization.  Our data suggest that ricolinostat in combination with bendamustine may be a novel combination with potential for use as an antitumor agent in lymphoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhV5zrMw5A6rVg90H21EOLACvtfcHk0liRhoqMkKp7XQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslKktr8%253D&md5=6ac58def2f10c37fe434c78dd074dcfd</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1007%2Fs10495-017-1364-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10495-017-1364-4%26sid%3Dliteratum%253Aachs%26aulast%3DCosenza%26aufirst%3DM.%26aulast%3DCivallero%26aufirst%3DM.%26aulast%3DMarcheselli%26aufirst%3DL.%26aulast%3DSacchi%26aufirst%3DS.%26aulast%3DPozzi%26aufirst%3DS.%26atitle%3DRicolinostat%252C%2520a%2520Selective%2520HDAC6%2520Inhibitor%252C%2520Shows%2520Anti-Lymphoma%2520Cell%2520Activity%2520Alone%2520and%2520in%2520Combination%2520with%2520Bendamustine%26jtitle%3DApoptosis%26date%3D2017%26volume%3D22%26spage%3D827%26epage%3D840%26doi%3D10.1007%2Fs10495-017-1364-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">North, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almeciga-Pinto, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quayle, S. N.</span></span> <span> </span><span class="NLM_article-title">Enhancement of Pomalidomide Anti-Tumor Response with ACY-241, a Selective HDAC6 Inhibitor</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0173507</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0173507</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1371%2Fjournal.pone.0173507" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=28264055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVGjtrfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=B.+J.+Northauthor=I.+Almeciga-Pintoauthor=D.+Tamangauthor=M.+Yangauthor=S.+S.+Jonesauthor=S.+N.+Quayle&title=Enhancement+of+Pomalidomide+Anti-Tumor+Response+with+ACY-241%2C+a+Selective+HDAC6+Inhibitor&doi=10.1371%2Fjournal.pone.0173507"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Enhancement of pomalidomide anti-tumor response with ACY-241, a selective HDAC6 inhibitor</span></div><div class="casAuthors">North, Brian J.; Almeciga-Pinto, Ingrid; Tamang, David; Yang, Min; Jones, Simon S.; Quayle, Steven N.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e0173507/1-e0173507/16</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Thalidomide-based Immunomodulatory Drugs (IMiDs), including lenalidomide and pomalidomide, are effective therapeutics for multiple myeloma.  These agents have been approved with, or are under clin. development with, other targeted therapies including proteasome inhibitors, αCD38 monoclonal antibodies, as well as histone deacetylase (HDAC) inhibitors for combination therapy.  HDAC inhibitors broadly targeting Class I and IIb HDACs have shown potent preclin. efficacy but have frequently demonstrated an undesirable safety profile in combination therapy approaches in clin. studies.  Therefore, development of more selective HDAC inhibitors could provide enhanced efficacy with reduced side effects in combination with IMiDs for the treatment of B-cell malignancies, including multiple myeloma.  Here, the second generation selective HDAC6 inhibitor citarinostat (ACY-241), with a more favorable safety profile than non-selective pan-HDAC inhibitors, is shown to synergize with pomalidomide in in vitro assays through promoting greater apoptosis and cell cycle arrest.  Furthermore, utilizing a multiple myeloma in vivo murine xenograft model, combination treatment with pomalidomide and ACY-241 leads to increased tumor growth inhibition.  At the mol. level, combination treatment with ACY-241 and pomalidomide leads to greater suppression of the pro-survival factors survivin, Myc, and IRF4.  The results presented here demonstrate synergy between pomalidomide and ACY-241 in both in vitro and in vivo preclin. models, providing further impetus for clin. development of ACY-241 for use in combination with IMiDs for patients with multiple myeloma and potentially other B-cell malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaSAg6exJCNbVg90H21EOLACvtfcHk0liASspMc1h2jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVGjtrfL&md5=175ef137cd119027f0462d07784375b5</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0173507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0173507%26sid%3Dliteratum%253Aachs%26aulast%3DNorth%26aufirst%3DB.%2BJ.%26aulast%3DAlmeciga-Pinto%26aufirst%3DI.%26aulast%3DTamang%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DS.%2BS.%26aulast%3DQuayle%26aufirst%3DS.%2BN.%26atitle%3DEnhancement%2520of%2520Pomalidomide%2520Anti-Tumor%2520Response%2520with%2520ACY-241%252C%2520a%2520Selective%2520HDAC6%2520Inhibitor%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0173507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, Y. H.</span></span> <span> </span><span class="NLM_article-title">Synergistic Antitumor Effects of the Combined Treatment with an HDAC6 Inhibitor and a COX-2 Inhibitor through Activation of PTEN</span>. <i>Oncol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">2657</span>– <span class="NLM_lpage">2666</span>, <span class="refDoi"> DOI: 10.3892/or.2017.5981</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.3892%2For.2017.5981" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=29048666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVKjsrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=2657-2666&author=G.+Zhangauthor=Y.+H.+Gan&title=Synergistic+Antitumor+Effects+of+the+Combined+Treatment+with+an+HDAC6+Inhibitor+and+a+COX-2+Inhibitor+through+Activation+of+PTEN&doi=10.3892%2For.2017.5981"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Synergistic antitumor effects of the combined treatment with an HDAC6 inhibitor and a COX-2 inhibitor through activation of PTEN</span></div><div class="casAuthors">Zhang, Guanhua; Gan, Ye-Hua</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2657-2666</span>CODEN:
                <span class="NLM_cas:coden">OCRPEW</span>;
        ISSN:<span class="NLM_cas:issn">1791-2431</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Chemotherapy is one of the most effective nonsurgical treatments for various types of tumor.  Identifying different combinations of antitumor agents that can produce synergistic antitumor effects remains an important clin. strategy.  In the present study, we showed that the combination of histone deacetylase 6 (HDAC6) inhibitor tubastatin A together with cyclooxygenase-2 (COX-2) inhibitor celecoxib resulted in synergistic antitumor effects in CAL 27 and SACC-83 cells.  Treatment with celecoxib alone promoted the membrane translocation of phosphatase and tensin homolog (PTEN), indicating PTEN activation, and consequently led to protein kinase B (AKT) dephosphorylation (inactivation).  Similarly, treatment with an HDAC6 inhibitor alone promoted PTEN membrane translocation and correspondingly dephosphorylated AKT.  The combination of celecoxib and an HDAC6 inhibitor synergistically increased PTEN membrane translocation and inactivated AKT.  Moreover, celecoxib enhanced the HDAC6 inhibitor-induced antitumor effects in PTEN-deficient U-87 MG cells that had been stably transfected with wild-type PTEN, but not in the same cell line stably transfected with mutant PTEN-K163R, which cannot be activated by HDAC6 inhibitors.  In summary, the results indicated that the COX-2 inhibitor celecoxib enhanced the HDAC6 inhibitor-induced antitumor effects by activating the PTEN/AKT signaling pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPVy27HVeeLLVg90H21EOLACvtfcHk0liASspMc1h2jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVKjsrrE&md5=0ee7ef927d63553a2ebcbbdd06fbd661</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.3892%2For.2017.5981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252For.2017.5981%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DGan%26aufirst%3DY.%2BH.%26atitle%3DSynergistic%2520Antitumor%2520Effects%2520of%2520the%2520Combined%2520Treatment%2520with%2520an%2520HDAC6%2520Inhibitor%2520and%2520a%2520COX-2%2520Inhibitor%2520through%2520Activation%2520of%2520PTEN%26jtitle%3DOncol.%2520Rep.%26date%3D2017%26volume%3D38%26spage%3D2657%26epage%3D2666%26doi%3D10.3892%2For.2017.5981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hideshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paranal, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colling, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estiu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazitschek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cottini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mimura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorgun, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tai, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrasco, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Discovery of Selective Small-Molecule HDAC6 Inhibitor for Overcoming Proteasome Inhibitor Resistance in Multiple Myeloma</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">13162</span>– <span class="NLM_lpage">13167</span>, <span class="refDoi"> DOI: 10.1073/pnas.1608067113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1073%2Fpnas.1608067113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=27799547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslKnu73M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=13162-13167&author=T.+Hideshimaauthor=J.+Qiauthor=R.+M.+Paranalauthor=W.+Tangauthor=E.+Greenbergauthor=N.+Westauthor=M.+E.+Collingauthor=G.+Estiuauthor=R.+Mazitschekauthor=J.+A.+Perryauthor=H.+Ohguchiauthor=F.+Cottiniauthor=N.+Mimuraauthor=G.+Gorgunauthor=Y.-T.+Taiauthor=P.+G.+Richardsonauthor=R.+D.+Carrascoauthor=O.+Wiestauthor=S.+L.+Schreiberauthor=K.+C.+Andersonauthor=J.+E.+Bradner&title=Discovery+of+Selective+Small-Molecule+HDAC6+Inhibitor+for+Overcoming+Proteasome+Inhibitor+Resistance+in+Multiple+Myeloma&doi=10.1073%2Fpnas.1608067113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective small-molecule HDAC6 inhibitor for overcoming proteasome inhibitor resistance in multiple myeloma</span></div><div class="casAuthors">Hideshima, Teru; Qi, Jun; Paranal, Ronald M.; Tang, Weiping; Greenberg, Edward; West, Nathan; Colling, Meaghan E.; Estiu, Guillermina; Mazitschek, Ralph; Perry, Jennifer A.; Ohguchi, Hiroto; Cottini, Francesca; Mimura, Naoya; Gorgun, Gullu; Tai, Yu-Tzu; Richardson, Paul G.; Carrasco, Ruben D.; Wiest, Olaf; Schreiber, Stuart L.; Anderson, Kenneth C.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">13162-13167</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Multiple myeloma (MM) has proven clin. susceptible to modulation of pathways of protein homeostasis.  Blockade of proteasomal degrdn. of polyubiquitinated misfolded proteins by the proteasome inhibitor bortezomib (BTZ) achieves responses and prolongs survival in MM, but long-term treatment with BTZ leads to drug-resistant relapse in most patients.  In a proof-of-concept study, the authors previously demonstrated that blocking aggresomal breakdown of polyubiquitinated misfolded proteins with the histone deacetylase 6 (HDAC6) inhibitor tubacin enhances BTZ-induced cytotoxicity in MM cells in vitro.  However, these foundational studies were limited by the pharmacol. liabilities of tubacin as a chem. probe with only in vitro utility.  Emerging from a focused library synthesis, a potent, selective, and bioavailable HDAC6 inhibitor, WT161, was created to study the mechanism of action of HDAC6 inhibition in MM alone and in combination with BTZ.  WT161 in combination with BTZ triggers significant accumulation of polyubiquitinated proteins and cell stress, followed by caspase activation and apoptosis.  More importantly, this combination treatment was effective in BTZ-resistant cells and in the presence of bone marrow stromal cells, which have been shown to mediate MM cell drug resistance.  The activity of WT161 was confirmed in the human MM cell xenograft mouse model and established the framework for clin. trials of the combination treatment to improve patient outcomes in MM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxE_VYaPFUSbVg90H21EOLACvtfcHk0liASspMc1h2jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslKnu73M&md5=2515fcba03631917d0ca7656d1f62a7e</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1608067113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1608067113%26sid%3Dliteratum%253Aachs%26aulast%3DHideshima%26aufirst%3DT.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DGreenberg%26aufirst%3DE.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DColling%26aufirst%3DM.%2BE.%26aulast%3DEstiu%26aufirst%3DG.%26aulast%3DMazitschek%26aufirst%3DR.%26aulast%3DPerry%26aufirst%3DJ.%2BA.%26aulast%3DOhguchi%26aufirst%3DH.%26aulast%3DCottini%26aufirst%3DF.%26aulast%3DMimura%26aufirst%3DN.%26aulast%3DGorgun%26aufirst%3DG.%26aulast%3DTai%26aufirst%3DY.-T.%26aulast%3DRichardson%26aufirst%3DP.%2BG.%26aulast%3DCarrasco%26aufirst%3DR.%2BD.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDiscovery%2520of%2520Selective%2520Small-Molecule%2520HDAC6%2520Inhibitor%2520for%2520Overcoming%2520Proteasome%2520Inhibitor%2520Resistance%2520in%2520Multiple%2520Myeloma%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D13162%26epage%3D13167%26doi%3D10.1073%2Fpnas.1608067113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahman, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piekarz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robey, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figg, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, S. E.</span></span> <span> </span><span class="NLM_article-title">Romidepsin: A New Therapy for Cutaneous T-Cell Lymphoma and a Potential Therapy for Solid Tumors</span>. <i>Expert Rev. Anticancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">997</span>– <span class="NLM_lpage">1008</span>, <span class="refDoi"> DOI: 10.1586/era.10.88</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1586%2Fera.10.88" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=20645688" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptFWqs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=997-1008&author=C.+Grantauthor=F.+Rahmanauthor=R.+Piekarzauthor=C.+Peerauthor=R.+Fryeauthor=R.+W.+Robeyauthor=E.+R.+Gardnerauthor=W.+D.+Figgauthor=S.+E.+Bates&title=Romidepsin%3A+A+New+Therapy+for+Cutaneous+T-Cell+Lymphoma+and+a+Potential+Therapy+for+Solid+Tumors&doi=10.1586%2Fera.10.88"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Romidepsin: a new therapy for cutaneous T-cell lymphoma and a potential therapy for solid tumors</span></div><div class="casAuthors">Grant, Cliona; Rahman, Fahd; Piekarz, Richard; Peer, Cody; Frye, Robin; Robey, Robert W.; Gardner, Erin R.; Figg, William D.; Bates, Susan E.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Review of Anticancer Therapy</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">997-1008</span>CODEN:
                <span class="NLM_cas:coden">ERATBJ</span>;
        ISSN:<span class="NLM_cas:issn">1473-7140</span>.
    
            (<span class="NLM_cas:orgname">Expert Reviews Ltd.</span>)
        </div><div class="casAbstract">A review.  Romidepsin is a histone deacetylase inhibitor (HDI), approved by the US FDA for the treatment of cutaneous T-cell lymphoma (CTCL).  Although various mechanisms have been proposed for the activity of HDIs, including induction of genes controlling cell cycle, acetylation of cytoplasmic proteins and direct induction of apoptosis, the mechanism underlying activity of romidepsin and other HDIs in CTCL is not known.  Romidepsin induces long-lasting responses.  The side-effect profile is similar to that of other HDIs, causing fatigue, nausea and thrombocytopenia.  Management of the CTCL population requires vigilence to prevent infection with skin contaminants, and monitoring of potassium and magnesium, electrolytes found to be low in a large proportion of patients.  Electrocardiog. (ECG) changes are common but are not assocd. with myocardial damage.  When mol. end points were evaluated in 61 patients enrolled on a Phase II trial with romidepsin, response was assocd. with persistence of acetylated histone H3, suggesting that drug exposure is important in effective therapy with romidepsin.  Future studies will endeavor to identify combination strategies to increase the efficacy both in resistant CTCL and in solid tumors and to identify biomarkers of response that will allow selection of patients most likely to benefit from the therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhGXcIbdqpk7Vg90H21EOLACvtfcHk0liSJjLqs_v4FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptFWqs74%253D&md5=b28fd17c77991e4662f7c0e08d6d7297</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1586%2Fera.10.88&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1586%252Fera.10.88%26sid%3Dliteratum%253Aachs%26aulast%3DGrant%26aufirst%3DC.%26aulast%3DRahman%26aufirst%3DF.%26aulast%3DPiekarz%26aufirst%3DR.%26aulast%3DPeer%26aufirst%3DC.%26aulast%3DFrye%26aufirst%3DR.%26aulast%3DRobey%26aufirst%3DR.%2BW.%26aulast%3DGardner%26aufirst%3DE.%2BR.%26aulast%3DFigg%26aufirst%3DW.%2BD.%26aulast%3DBates%26aufirst%3DS.%2BE.%26atitle%3DRomidepsin%253A%2520A%2520New%2520Therapy%2520for%2520Cutaneous%2520T-Cell%2520Lymphoma%2520and%2520a%2520Potential%2520Therapy%2520for%2520Solid%2520Tumors%26jtitle%3DExpert%2520Rev.%2520Anticancer%2520Ther.%26date%3D2010%26volume%3D10%26spage%3D997%26epage%3D1008%26doi%3D10.1586%2Fera.10.88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balasubramanian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirisawad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buggy, J. J.</span></span> <span> </span><span class="NLM_article-title">A Novel Histone Deacetylase 8 (HDAC8)-Specific Inhibitor PCI-34051 Induces Apoptosis in T-Cell Lymphomas</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1026</span>– <span class="NLM_lpage">1034</span>, <span class="refDoi"> DOI: 10.1038/leu.2008.9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fleu.2008.9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=18256683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvVSrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=1026-1034&author=S.+Balasubramanianauthor=J.+Ramosauthor=W.+Luoauthor=M.+Sirisawadauthor=E.+Vernerauthor=J.+J.+Buggy&title=A+Novel+Histone+Deacetylase+8+%28HDAC8%29-Specific+Inhibitor+PCI-34051+Induces+Apoptosis+in+T-Cell+Lymphomas&doi=10.1038%2Fleu.2008.9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">A novel histone deacetylase 8 (HDAC8)-specific inhibitor PCI-34051 induces apoptosis in T-cell lymphomas</span></div><div class="casAuthors">Balasubramanian, S.; Ramos, J.; Luo, W.; Sirisawad, M.; Verner, E.; Buggy, J. J.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1026-1034</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We have developed a potent, histone deacetylase 8 (HDAC8)-specific inhibitor PCI-34051 with >200-fold selectivity over the other HDAC isoforms.  PCI-34051 induces caspase-dependent apoptosis in cell lines derived from T-cell lymphomas or leukemias, but not in other hematopoietic or solid tumor lines.  Unlike broad-spectrum HDAC inhibitors, PCI-34051 does not cause detectable histone or tubulin acetylation.  Cells defective in T-cell receptor signaling were still sensitive to PCI-34051-induced apoptosis, whereas a phospholipase C-γ1 (PLCγ1)-defective line was resistant.  Jurkat cells showed a dose-dependent decrease in PCI-34051-induced apoptosis upon treatment with a PLC inhibitor U73122, but not with an inactive analog.  We found that rapid intracellular calcium mobilization from endoplasmic reticulum (ER) and later cytochrome c release from mitochondria are essential for the apoptotic mechanism.  The rapid Ca2+ flux was dependent on PCI-34051 concn., and was blocked by the PLC inhibitor U73122.  Further, apoptosis was blocked by Ca2+ chelators (BAPTA) and enhanced by Ca2+ effectors (thapsigargin), supporting this model.  These studies show that HDAC8-selective inhibitors have a unique mechanism of action involving PLCγ1 activation and calcium-induced apoptosis, and could offer benefits including a greater therapeutic index for treating T-cell malignancies.  Leukemia (2008) 22, 1026-1034; doi:10.1038/leu.2008.9; published online 7 Feb. 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpW-VB4Wb3h1bVg90H21EOLACvtfcHk0liSJjLqs_v4FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvVSrtr0%253D&md5=03446a05b396ea9497e4f6f9254cae03</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1038%2Fleu.2008.9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2008.9%26sid%3Dliteratum%253Aachs%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DRamos%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DW.%26aulast%3DSirisawad%26aufirst%3DM.%26aulast%3DVerner%26aufirst%3DE.%26aulast%3DBuggy%26aufirst%3DJ.%2BJ.%26atitle%3DA%2520Novel%2520Histone%2520Deacetylase%25208%2520%2528HDAC8%2529-Specific%2520Inhibitor%2520PCI-34051%2520Induces%2520Apoptosis%2520in%2520T-Cell%2520Lymphomas%26jtitle%3DLeukemia%26date%3D2008%26volume%3D22%26spage%3D1026%26epage%3D1034%26doi%3D10.1038%2Fleu.2008.9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israelian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nawar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manaswiyoungkul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raouf, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedighi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olaoye, O. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdogan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabral, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angeles, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altintas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Araujo, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span> <span> </span><span class="NLM_article-title">Characterization of Conformationally Constrained Benzanilide Scaffolds for Potent and Selective HDAC8 Targeting</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">8634</span>– <span class="NLM_lpage">8648</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01025</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01025" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtl2rs7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=8634-8648&author=M.+M.+Hassanauthor=J.+Israelianauthor=N.+Nawarauthor=G.+Gandaauthor=P.+Manaswiyoungkulauthor=Y.+S.+Raoufauthor=D.+Armstrongauthor=A.+Sedighiauthor=O.+O.+Olaoyeauthor=F.+Erdoganauthor=A.+D.+Cabralauthor=F.+Angelesauthor=R.+Altintasauthor=E.+D.+de+Araujoauthor=P.+T.+Gunning&title=Characterization+of+Conformationally+Constrained+Benzanilide+Scaffolds+for+Potent+and+Selective+HDAC8+Targeting&doi=10.1021%2Facs.jmedchem.0c01025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of Conformationally Constrained Benzanilide Scaffolds for Potent and Selective HDAC8 Targeting</span></div><div class="casAuthors">Hassan, Muhammad Murtaza; Israelian, Johan; Nawar, Nabanita; Ganda, Giovanni; Manaswiyoungkul, Pimyupa; Raouf, Yasir S.; Armstrong, David; Sedighi, Abootaleb; Olaoye, Olasunkanmi O.; Erdogan, Fettah; Cabral, Aaron D.; Angeles, Fabrizio; Altintas, Rabia; de Araujo, Elvin D.; Gunning, Patrick T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8634-8648</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone deacetylases (HDACs) are an attractive therapeutic target for a variety of human diseases.  Currently, all four FDA-approved HDAC-targeting drugs are nonselective, pan-HDAC inhibitors, exhibiting adverse side effects at therapeutic doses.  Although selective HDAC inhibition has been proposed to mitigate toxicity, the targeted catalytic domains are highly conserved.  Herein, we describe a series of rationally designed, conformationally constrained, benzanilide foldamers which selectively bind the catalytic tunnel of HDAC8.  The series includes benzanilides, MMH371, MMH409, and MMH410, which exhibit potent in vitro HDAC8 activity (IC50 = 66, 23, and 66 nM, resp.) and up to 410-fold selectivity for HDAC8 over the next targeted HDAC.  Exptl. and computational analyses of the benzanilide structure docked with human HDAC8 enzyme showed the adoption of a low-energy L-shaped conformer that favors HDAC8 selectivity.  The conformationally constrained HDAC8 inhibitors present an alternative biol. probe for further detg. the clin. utility and safety of pharmacol. knockdown of HDAC8 in diseased cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRExtnmcfaMrVg90H21EOLACvtfcHk0ljsjqX26y4raA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtl2rs7%252FN&md5=d7a4513990c97e0802308c444663c02e</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01025%26sid%3Dliteratum%253Aachs%26aulast%3DHassan%26aufirst%3DM.%2BM.%26aulast%3DIsraelian%26aufirst%3DJ.%26aulast%3DNawar%26aufirst%3DN.%26aulast%3DGanda%26aufirst%3DG.%26aulast%3DManaswiyoungkul%26aufirst%3DP.%26aulast%3DRaouf%26aufirst%3DY.%2BS.%26aulast%3DArmstrong%26aufirst%3DD.%26aulast%3DSedighi%26aufirst%3DA.%26aulast%3DOlaoye%26aufirst%3DO.%2BO.%26aulast%3DErdogan%26aufirst%3DF.%26aulast%3DCabral%26aufirst%3DA.%2BD.%26aulast%3DAngeles%26aufirst%3DF.%26aulast%3DAltintas%26aufirst%3DR.%26aulast%3Dde%2BAraujo%26aufirst%3DE.%2BD.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26atitle%3DCharacterization%2520of%2520Conformationally%2520Constrained%2520Benzanilide%2520Scaffolds%2520for%2520Potent%2520and%2520Selective%2520HDAC8%2520Targeting%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D8634%26epage%3D8648%26doi%3D10.1021%2Facs.jmedchem.0c01025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shouksmith, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gawel, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nawar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raouf, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukhari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manaswiyoungkul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olaoye, O. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabral, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedighi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Araujo, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span> <span> </span><span class="NLM_article-title">Class I/IIb-Selective HDAC Inhibitor Exhibits Oral Bioavailability and Therapeutic Efficacy in Acute Myeloid Leukemia</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00471</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00471" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlygtLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=56-64&author=A.+E.+Shouksmithauthor=J.+M.+Gawelauthor=N.+Nawarauthor=D.+Sinaauthor=Y.+S.+Raoufauthor=S.+Bukhariauthor=L.+Heauthor=A.+E.+Johnsauthor=P.+Manaswiyoungkulauthor=O.+O.+Olaoyeauthor=A.+D.+Cabralauthor=A.+Sedighiauthor=E.+D.+de+Araujoauthor=P.+T.+Gunning&title=Class+I%2FIIb-Selective+HDAC+Inhibitor+Exhibits+Oral+Bioavailability+and+Therapeutic+Efficacy+in+Acute+Myeloid+Leukemia&doi=10.1021%2Facsmedchemlett.9b00471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Class I/IIb-Selective HDAC Inhibitor Exhibits Oral Bioavailability and Therapeutic Efficacy in Acute Myeloid Leukemia</span></div><div class="casAuthors">Shouksmith, Andrew E.; Gawel, Justyna M.; Nawar, Nabanita; Sina, Diana; Raouf, Yasir S.; Bukhari, Shazreh; He, Liying; Johns, Alexandra E.; Manaswiyoungkul, Pimyupa; Olaoye, Olasunkanmi O.; Cabral, Aaron D.; Sedighi, Abootaleb; de Araujo, Elvin D.; Gunning, Patrick T.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-64</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The HDAC inhibitor 4-tert-butyl-N-(4-(hydroxycarbamoyl)phenyl)benzamide (AES-350, I) was identified as a promising preclin. candidate for the treatment of acute myeloid leukemia (AML), an aggressive malignancy with a meagre 24% 5-yr survival rate.  Through screening of low-mol.-wt. analogs derived from the previously discovered novel HDAC inhibitor, AES-135 (II), compd. I demonstrated greater HDAC isoform selectivity, higher cytotoxicity in MV4-11 cells, an improved therapeutic window, and more efficient absorption through cellular and lipid membranes.  Compd. I also demonstrated improved oral bioavailability compared to SAHA in mouse models.  A broad spectrum of expts., including FACS, ELISA, and Western blotting, were performed to support our hypothesis that I dose-dependently triggers apoptosis in AML cells through HDAC inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8ikqWn7KZc7Vg90H21EOLACvtfcHk0ljsjqX26y4raA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlygtLrK&md5=3c21f11cda2a27e00f30a7cd3c700785</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00471%26sid%3Dliteratum%253Aachs%26aulast%3DShouksmith%26aufirst%3DA.%2BE.%26aulast%3DGawel%26aufirst%3DJ.%2BM.%26aulast%3DNawar%26aufirst%3DN.%26aulast%3DSina%26aufirst%3DD.%26aulast%3DRaouf%26aufirst%3DY.%2BS.%26aulast%3DBukhari%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DL.%26aulast%3DJohns%26aufirst%3DA.%2BE.%26aulast%3DManaswiyoungkul%26aufirst%3DP.%26aulast%3DOlaoye%26aufirst%3DO.%2BO.%26aulast%3DCabral%26aufirst%3DA.%2BD.%26aulast%3DSedighi%26aufirst%3DA.%26aulast%3Dde%2BAraujo%26aufirst%3DE.%2BD.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26atitle%3DClass%2520I%252FIIb-Selective%2520HDAC%2520Inhibitor%2520Exhibits%2520Oral%2520Bioavailability%2520and%2520Therapeutic%2520Efficacy%2520in%2520Acute%2520Myeloid%2520Leukemia%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D56%26epage%3D64%26doi%3D10.1021%2Facsmedchemlett.9b00471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Staber, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellido, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davids, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadia, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shustov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tournilhac, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaja, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porkka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoermann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonitsch-Klupp, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haferlach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubicek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayerhoefer, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopfinger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dearden, C.</span></span> <span> </span><span class="NLM_article-title">Consensus Criteria for Diagnosis, Staging, and Treatment Response Assessment of T-Cell Prolymphocytic Leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">1132</span>– <span class="NLM_lpage">1143</span>, <span class="refDoi"> DOI: 10.1182/blood.2019000402</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1182%2Fblood.2019000402" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=31292114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1GjsLbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2019&pages=1132-1143&author=P.+B.+Staberauthor=M.+Herlingauthor=M.+Bellidoauthor=E.+D.+Jacobsenauthor=M.+S.+Davidsauthor=T.+M.+Kadiaauthor=A.+Shustovauthor=O.+Tournilhacauthor=E.+Bachyauthor=F.+Zajaauthor=K.+Porkkaauthor=G.+Hoermannauthor=I.+Simonitsch-Kluppauthor=C.+Haferlachauthor=S.+Kubicekauthor=M.+E.+Mayerhoeferauthor=G.+Hopfingerauthor=U.+Jaegerauthor=C.+Dearden&title=Consensus+Criteria+for+Diagnosis%2C+Staging%2C+and+Treatment+Response+Assessment+of+T-Cell+Prolymphocytic+Leukemia&doi=10.1182%2Fblood.2019000402"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Consensus criteria for diagnosis, staging, and treatment response assessment of T-cell prolymphocytic leukemia</span></div><div class="casAuthors">Staber, Philipp B.; Herling, Marco; Bellido, Mar; Jacobsen, Eric D.; Davids, Matthew S.; Kadia, Tapan Mahendra; Shustov, Andrei; Tournilhac, Olivier; Bachy, Emmanuel; Zaja, Francesco; Porkka, Kimmo; Hoermann, Gregor; Simonitsch-Klupp, Ingrid; Haferlach, Claudia; Kubicek, Stefan; Mayerhoefer, Marius E.; Hopfinger, Georg; Jaeger, Ulrich; Dearden, Claire</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1132-1143</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">1528-0020</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">T-cell prolymphocytic leukemia (T-PLL) is a rare, mature T-cell neoplasm with a heterogeneous clin. course.  With the advent of novel treatment options that will potentially change the management of patients with T-PLL, it has become necessary to produce consensus guidelines for the design and conduct of clin. trials.  The T-PLL International Study group (TPLL-ISG) set out to define standardized criteria for diagnosis, treatment indication, and evaluation of response.  These criteria will facilitate comparison of results from clin. trials in T-PLL, and will thus support clin. decision making, as well as the approval of new therapeutics by healthcare authorities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_-kVKgvO1zrVg90H21EOLACvtfcHk0ljsjqX26y4raA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1GjsLbJ&md5=77e8f358b7e24f0ad1ff286d6e2d8e72</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1182%2Fblood.2019000402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood.2019000402%26sid%3Dliteratum%253Aachs%26aulast%3DStaber%26aufirst%3DP.%2BB.%26aulast%3DHerling%26aufirst%3DM.%26aulast%3DBellido%26aufirst%3DM.%26aulast%3DJacobsen%26aufirst%3DE.%2BD.%26aulast%3DDavids%26aufirst%3DM.%2BS.%26aulast%3DKadia%26aufirst%3DT.%2BM.%26aulast%3DShustov%26aufirst%3DA.%26aulast%3DTournilhac%26aufirst%3DO.%26aulast%3DBachy%26aufirst%3DE.%26aulast%3DZaja%26aufirst%3DF.%26aulast%3DPorkka%26aufirst%3DK.%26aulast%3DHoermann%26aufirst%3DG.%26aulast%3DSimonitsch-Klupp%26aufirst%3DI.%26aulast%3DHaferlach%26aufirst%3DC.%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DMayerhoefer%26aufirst%3DM.%2BE.%26aulast%3DHopfinger%26aufirst%3DG.%26aulast%3DJaeger%26aufirst%3DU.%26aulast%3DDearden%26aufirst%3DC.%26atitle%3DConsensus%2520Criteria%2520for%2520Diagnosis%252C%2520Staging%252C%2520and%2520Treatment%2520Response%2520Assessment%2520of%2520T-Cell%2520Prolymphocytic%2520Leukemia%26jtitle%3DBlood%26date%3D2019%26volume%3D134%26spage%3D1132%26epage%3D1143%26doi%3D10.1182%2Fblood.2019000402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khoury, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Washington, L. B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keating, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D.</span></span> <span> </span><span class="NLM_article-title">A Systematic Approach to Diagnosis of Mature T-Cell Leukemias Reveals Heterogeneity among WHO Categories</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">328</span>– <span class="NLM_lpage">335</span>, <span class="refDoi"> DOI: 10.1182/blood-2004-01-0002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1182%2Fblood-2004-01-0002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=15044256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvFSnsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2004&pages=328-335&author=M.+Herlingauthor=J.+D.+Khouryauthor=L.+B.+T.+Washingtonauthor=M.+Duvicauthor=M.+J.+Keatingauthor=D.+Jones&title=A+Systematic+Approach+to+Diagnosis+of+Mature+T-Cell+Leukemias+Reveals+Heterogeneity+among+WHO+Categories&doi=10.1182%2Fblood-2004-01-0002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">A systematic approach to diagnosis of mature T-cell leukemias reveals heterogeneity among WHO categories</span></div><div class="casAuthors">Herling, Marco; Khoury, Joseph D.; Washington, LaBaron T.; Duvic, Madeleine; Keating, Michael J.; Jones, Dan</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">328-335</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The current World Health Organization (WHO) classification of hematopoietic malignancies defines several types of mature T-cell leukemia including T-cell prolymphocytic leukemia (T-PLL), Sezary syndrome (SS), and T-cell large granular lymphocytic (T-LGL) leukemia.  These neoplasms can show overlapping features with each other and with T-cell lymphomas involving peripheral blood (PB).  We analyzed the spectrum of clinicopathol. features in 102 mature T-cell leukemias and compared them to 10 hepatosplenic T-cell lymphomas that involved PB.  T-PLL, defined as a T-cell leukemia showing rapidly rising PB lymphocyte counts, was the only tumor type expressing the oncoprotein TCL1 (71% of cases) and could present with relatively low lymphocyte levels or small tumor cell morphol.  SS, defined by accompanying erythrodermic skin disease, was frequently assocd. with peripheral eosinophilia but could also develop high nos. of prolymphocytes, esp. late in the disease course.  T-LGL leukemia, defined by accompanying cytopenias or autoimmune phenomena (or both), had the best clin. outcome and generally showed the lowest circulating lymphocyte levels with only a few cases developing marked lymphocytosis.  Using the dominant clin. or phenotypic feature, we describe here the degree of overlap among currently recognized WHO categories and identify areas where further clarification is needed.  Our results indicate that incorporation of addnl. criteria, such as TCL1 expression status and hematol. parameters, can assist in a more accurate classification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS0GwbkDkfirVg90H21EOLACvtfcHk0lhl23KL881U7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvFSnsbc%253D&md5=7fa603b6f7fb2cf31c14a978107ab173</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1182%2Fblood-2004-01-0002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2004-01-0002%26sid%3Dliteratum%253Aachs%26aulast%3DHerling%26aufirst%3DM.%26aulast%3DKhoury%26aufirst%3DJ.%2BD.%26aulast%3DWashington%26aufirst%3DL.%2BB.%2BT.%26aulast%3DDuvic%26aufirst%3DM.%26aulast%3DKeating%26aufirst%3DM.%2BJ.%26aulast%3DJones%26aufirst%3DD.%26atitle%3DA%2520Systematic%2520Approach%2520to%2520Diagnosis%2520of%2520Mature%2520T-Cell%2520Leukemias%2520Reveals%2520Heterogeneity%2520among%2520WHO%2520Categories%26jtitle%3DBlood%26date%3D2004%26volume%3D104%26spage%3D328%26epage%3D335%26doi%3D10.1182%2Fblood-2004-01-0002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ianevski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timonen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kononov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aittokallio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giri, A. K.</span></span> <span> </span><span class="NLM_article-title">SynToxProfiler: An Interactive Analysis of Drug Combination Synergy, Toxicity and Efficacy</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>, <span class="NLM_elocation-id">e1007604</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1007604</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1371%2Fjournal.pcbi.1007604" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=32012154" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BB3cXovFWjsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&author=A.+Ianevskiauthor=S.+Timonenauthor=A.+Kononovauthor=T.+Aittokallioauthor=A.+K.+Giri&title=SynToxProfiler%3A+An+Interactive+Analysis+of+Drug+Combination+Synergy%2C+Toxicity+and+Efficacy&doi=10.1371%2Fjournal.pcbi.1007604"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">SynToxProfiler: an interactive analysis of drug combination synergy, toxicity and efficacy</span></div><div class="casAuthors">Ianevski, Aleksandr; Timonen, Sanna; Kononov, Alexander; Aittokallio, Tero; Giri, Anil K.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e1007604</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Drug combinations are becoming a std. treatment of many complex diseases due to their capability to overcome resistance to monotherapy.  In the current preclin. drug combination screening, the top combinations for further study are often selected based on synergy alone, without considering the combination efficacy and toxicity effects, even though these are crit. determinants for the clin. success of a therapy.  To promote the prioritization of drug combinations based on integrated anal. of synergy, efficacy and toxicity profiles, we implemented a web-based open-source tool, SynToxProfiler (Synergy-Toxicity-Profiler).  When applied to 20 anti-cancer drug combinations tested both in healthy control and T-cell prolymphocytic leukemia (T-PLL) patient cells, as well as to 77 anti-viral drug pairs tested in Huh7 liver cell line with and without Ebola virus infection, SynToxProfiler prioritized as top hits those synergistic drug pairs that showed higher selective efficacy (difference between efficacy and toxicity), which offers an improved likelihood for clin. success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4EyvmeDt93LVg90H21EOLACvtfcHk0lhl23KL881U7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXovFWjsL4%253D&md5=c8b64ed8bc0258e5b4c674ae3195bda8</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1007604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1007604%26sid%3Dliteratum%253Aachs%26aulast%3DIanevski%26aufirst%3DA.%26aulast%3DTimonen%26aufirst%3DS.%26aulast%3DKononov%26aufirst%3DA.%26aulast%3DAittokallio%26aufirst%3DT.%26aulast%3DGiri%26aufirst%3DA.%2BK.%26atitle%3DSynToxProfiler%253A%2520An%2520Interactive%2520Analysis%2520of%2520Drug%2520Combination%2520Synergy%252C%2520Toxicity%2520and%2520Efficacy%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2020%26volume%3D16%26doi%3D10.1371%2Fjournal.pcbi.1007604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. HDACi <b>1</b>−<b>14</b> and their <i>in vitro</i> activity and selectivity profiles. (a) Chemical structures of HDACi <b>1</b>–<b>14</b>. (b) Biochemical HDAC inhibition against recombinant enzymes (Nanosyn) and cellular cytotoxicity results in MV4-11 (AML) and MRC-9 (healthy fibroblasts). Note: **not HDAC6 selective, *selectivity ≥ indicated value (beyond limit of experiment). A color gradient is applied for visualization of differential HDAC6 activity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Conditions for Preparation of <b>2</b>–<b>14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) Reagents and conditions to synthesize <b>2</b>–<b>9</b> and <b>11</b>–<b>14</b>. (i) BnBr, Cs<sub>2</sub>CO<sub>3</sub>, DMF, RT, 24 h; (a) (i) R–NH<sub>2</sub>, AcOH, DCE, RT, 2 h; (ii) NaBH(OAc)<sub>3</sub>, RT, 16 h; (b) R′SO<sub>2</sub>Cl, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 3–16 h, RT; (c) H<sub>2</sub>, 10% Pd/C, THF/MeOH (2:1), RT, 16 h; (g) HSO<sub>3</sub>Cl, 3 h, 150°C; (b) RNH<sub>2</sub>, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, RT, 3–16 h <b>d)</b> C<sub>12</sub>H<sub>15</sub>BrO<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, DMF, RT, 24 h; <b>e)</b> 4M HCl/dioxane, 0°C-RT, 3 h; (f) (i) (COCl)<sub>2</sub>, THF, DMF, 0°C, 1 h; (ii) H<sub>2</sub>N-OTHP, <sup>i</sup>Pr<sub>2</sub>NEt, THF, RT, 16 h; (f′) H<sub>2</sub>N-OBn, EDCI, HOBt, Et<sub>3</sub>N, DMF, RT, 16–24 h; (c) H<sub>2</sub>, 10% Pd/C, THF/MeOH (2:1), RT, 16–24 h; (e) 4M HCl/dioxane, 0°C-RT, 3 h. <b>b</b> Reagents and conditions to synthesize <b>10. (i)</b> BnBr, Cs<sub>2</sub>CO<sub>3</sub>, DMF, RT, 24 h; <b>a)</b> (i) <sup>c</sup>Pr-NH<sub>2</sub>, AcOH, DCE, RT, 2 h; (ii) NaBH(OAc)<sub>3</sub>, RT, 16 h; (b) PFBSCl, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 3–16 h, RT; (e) H<sub>2</sub>, 10% Pd/C, THF/MeOH (2:1), RT, 18 h; (f) (i) (COCl)<sub>2</sub>, THF, DMF, 0°C, 1 h; (ii) H<sub>2</sub>N-OTHP, <sup>i</sup>Pr<sub>2</sub>NEt, THF, RT, 16 h; (e) H<sub>2</sub>, 10% Pd/C, THF/MeOH (2:1), RT, 6 h.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. <i>In vitro</i> and <i>in silico</i> confirmation of superior HDAC6-selectivity and potency of compound <b>14</b> (KT-531). (a) X-ray crystal structure of <b>14</b>. (b) Dose–response curve representing the percent inhibition and HDAC6 selectivity of citarinostat against HDAC2–4,6,8,11 via EMSA activity assay (Nanosyn). (c) Dose–response curve representing the percent inhibition and HDAC6-selectivity of <b>14</b> against HDAC1–11. (d) <i>In vitro</i> HDAC inhibition IC<sub>50</sub> values of citarinostat (i) and <b>14</b> against HDAC1–11. A color gradient is applied for visualization of differential activity. (e) <i>In silico</i> docking binding pose of <b>5</b> in human HDAC6 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU">5EDU</a>), with Zn<sup>2+</sup> (yellow sphere), hydrogen bonds (yellow dashed line), π–π stacking (green dashed line), and salt bridges (purple dashed line) (N, blue; O, red; C, green; F, blue-green). Catalytic triad residues and portions of the protein have been omitted for clarity. (f) <i>In silico</i> docking binding pose of <b>14</b> in human HDAC6 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU">5EDU</a>), with Zn<sup>2+</sup> (yellow sphere), hydrogen bonds (yellow dashed line), π–π stacking (green dashed line), and salt bridges (purple dashed line) (N, blue; O, red; C, yellow; F, green). Catalytic triad residues and portions of the protein have been omitted for clarity. (g) <i>In silico</i> docking binding pose of <b>14</b> in human HDAC8 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1T64">1T64</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. <i>In cellulo</i> pharmacology of compound <b>14</b> (KT-531) in comparison to citarinostat. (a) Immunofluorescence analysis of α-tubulin acetylation and histone H3 acetylation following 6 h treatment of KT-531 in cells. (acetylated α-tubulin in red, acetylated histone H3 in green, and nuclear stain DAPI in blue). This data is representative of three independent experiments. (b) Quantification of fluorescent signals that correspond to levels of acetylated α-tubulin and acetylated histone H3 in cells dosed with KT-531 at indicated concentrations. (c) Immunofluorescence analysis of α-tubulin acetylation and histone H3 acetylation following 6 h treatment of citarinostat in cells (acetylated α-tubulin in red, acetylated histone H3 in green, and nuclear stain DAPI in blue). This data is representative of three independent experiments. (d) Quantification of fluorescent signals that correspond to levels of acetylated α-tubulin and acetylated histone H3 in cells dosed with citarinostat at indicated concentrations. *<i>p</i> ≤ 0.05; ****<i>p</i> ≤ 0.001; ns, nonsignificant; two-way ANOVA with Tukey’s multiple comparisons test. (e) Western blot illustrating α-tubulin acetylation and histone H3 acetylation levels in MV4-11 AML cells following 6 h treatment with varying concentrations of KT-531 and clinical candidate citarinostat (i). Protein extracts were prepared, resolved by SDS-PAGE and immunoblotted with acetylated α-tubulin, acetylated histone H3, and HSC70 antibodies. A representative Western blot of three independent experiments is shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>14</b> (KT-531) induces apoptosis in a dose-dependent manner. (a) Flow cytometric data of MV4-11 cells treated with increasing concentrations of KT-531 (0–4 μM) for 18 h and analyzed for apoptosis via Annexin V/PI staining. Representative dot blots from three independent experiments are presented. (b) Percentage of cell populations in healthy, early apoptosis, and late apoptosis phases following treatment with KT-531 for 18 h at indicated concentrations. (c) Percentage of cell populations in healthy, early apoptosis, and late apoptosis phases following treatment with citarinostat for 18 h at indicated concentrations. Flow cytometric data for citarinostat is available in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Figure S6</a>. (d) Cleavage of caspase-3 and PARP-1 upon dose-escalation of KT-531 for 6 h in MV4-11 cells. Protein extracts were prepared and subjected to SDS-PAGE and immunoblotting with procaspase 3, cleaved caspase-3, PARP-1, cleaved PARP-1, and HSC70 antibodies. A representative Western blot of three independent experiments is shown. Data for citarinostat is available in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf" class="ext-link">Figure S7</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. KT-531 shows selective cellular cytotoxicity in PTCL malignancies. Heat map of IC<sub>50</sub> values calculated from drug response analysis of KT-531 (<b>14</b>) and citarinostat from one representative out of three independent experiments. Cell lines are classified according to the respective disease subtype. Abbreviations: AML, acute myeloid leukemia; ANKL, aggressive NK-leukemia; LGL, large granular lymphocyte leukemia; T-NHL, T-cell non-Hodgkin’s lymphoma; ALK<sup>+</sup> ALCL, anaplastic large cell lymphoma (anaplastic lymphoma kinase positive); CTCL, cutaneous T-cell lymphoma; T-ALL, T-cell acute lymphoblastic leukemia.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Preclinical findings show promising therapeutic window and pharmacokinetic properties of <b>14</b> (KT-531). (a) Cytotoxicity of KT-531 (<b>14</b>) and citarinostat in healthy noncancerous cell lines (MRC-9, normal human fibroblasts (NHF), human umbilical vein endothelial cells (HUVEC), primary pooled fibroblasts from 250 donors (pPF)). (b) Average mouse weight of CD-1 mice (20 mg/kg, P.O.) dosed with KT-531 (<b>14</b>) and vehicle daily for 5 days. (c) Pharmacokinetic assessment of KT-531 (<b>14</b>) in mouse hepatocytes, PAMPA assay, mouse plasma and <i>in vivo</i> in CD-1 mice (IP, 20 mg/kg).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. HDAC6 is significantly overexpressed in T-PLL patient samples, but not in other T-cell cancers. (a) mRNA expression levels of various <i>HDAC</i> genes (1–6, 9, and 11) in T-cell prolymphocytic leukemia (T-PLL) patient samples, compared to healthy T-cell (CD3<sup>+</sup>) control samples. Data were extracted from the Oncomine Platform, from the Durig Leukemia data set. <i>HDAC7</i>, <i>8</i>, and <i>10</i> expression levels were not reported in this data set. ns; not significant (<i>p</i> > 0.05). (b) mRNA expression levels of <i>HDAC6</i> in various T-cell neoplasia patient samples, compared to healthy control cells (from left to right, Eckerle Lymphoma: NK cell, T-cell, NK/T-cell; Piccaluga Lymphoma: CD4<sup>+</sup> T-cell, CD8<sup>+</sup> T-cell, T-cell; Zhang Leukemia: hematogone, bone marrow; Haferlach Leukemia: peripheral blood mononuclear cells). Data were extracted from the Oncomine Platform, from the Eckerle Lymphoma, Piccaluga Lymphoma, Zhang Leukemia, and Haferlach Leukemia data sets. All comparisons have a <i>p</i>-value > 0.05 and/or a fold-change of ≤1.2 and are not considered significant. Representation: boxes as interquartile range, horizontal line as the mean, and whiskers as lower and upper limits. Abbreviations: ALCL, anaplastic large cell lymphoma; ALK, anaplastic lymphoma kinase; PTCL-NOS, peripheral T-cell lymphoma, not otherwise specified; AITL, angioimmunoblastic T-cell lymphoma; NK, natural killer.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/medium/jm1c00420_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. KT-531 (<b>14</b>) shows promising efficacy and safety in T-PLL primary patient samples. (a) Drug sensitivity scores (DSS)<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> of selected HDACi in T-PLL primary patient samples. (b) Box plot of T-PLL synergy scores of KT-531 (<b>14</b>) with bendamustine, idasanutlin, and venetoclax in T-PLL patient samples. (c) Box plot of combination potency cropped to TC50 (toxic concentration in 50% of PBMC population) of KT-531 (<b>14</b>) and venetoclax in T-PLL patient samples and healthy PBMC controls.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00420/20210616/images/large/jm1c00420_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00420&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i102">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42356" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42356" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 73 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lund, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Lohuizen, M.</span></span> <span> </span><span class="NLM_article-title">Epigenetics and Cancer</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2315</span>– <span class="NLM_lpage">2335</span>, <span class="refDoi"> DOI: 10.1101/gad.1232504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1101%2Fgad.1232504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=15466484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2cXot1yrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2004&pages=2315-2335&author=A.+H.+Lundauthor=M.+Van+Lohuizen&title=Epigenetics+and+Cancer&doi=10.1101%2Fgad.1232504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetics and cancer</span></div><div class="casAuthors">Lund, Anders H.; van Lohuizen, Maarten</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2315-2335</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  Epigenetic mechanisms act to change the accessibility of chromatin to transcriptional regulation locally and globally via modifications of the DNA and by modification or rearrangement of nucleosomes.  Epigenetic gene regulation collaborates with genetic alterations in cancer development.  This is evident from every aspect of tumor biol. including cell growth and differentiation, cell cycle control, DNA repair, angiogenesis, migration, and evasion of host immunosurveillance.  In contrast to genetic cancer causes, the possibility of reversing epigenetic codes may provide new targets for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLkbbp11uuWrVg90H21EOLACvtfcHk0liB-XHYq3-EcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXot1yrtL8%253D&md5=7e9499e9bc7e5638f2d8e4ff80528343</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1101%2Fgad.1232504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1232504%26sid%3Dliteratum%253Aachs%26aulast%3DLund%26aufirst%3DA.%2BH.%26aulast%3DVan%2BLohuizen%26aufirst%3DM.%26atitle%3DEpigenetics%2520and%2520Cancer%26jtitle%3DGenes%2520Dev.%26date%3D2004%26volume%3D18%26spage%3D2315%26epage%3D2335%26doi%3D10.1101%2Fgad.1232504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eckschlager, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stiborova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hrabeta, J.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylase Inhibitors as Anticancer Drugs</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1414</span>, <span class="refDoi"> DOI: 10.3390/ijms18071414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.3390%2Fijms18071414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=28671573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVGgt73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=1414&author=T.+Eckschlagerauthor=J.+Plchauthor=M.+Stiborovaauthor=J.+Hrabeta&title=Histone+Deacetylase+Inhibitors+as+Anticancer+Drugs&doi=10.3390%2Fijms18071414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase inhibitors as anticancer drugs</span></div><div class="casAuthors">Eckschlager, Tomas; Plch, Johana; Stiborova, Marie; Hrabeta, Jan</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1414/1-1414/25</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Carcinogenesis cannot be explained only by genetic alterations, but also involves epigenetic processes.  Modification of histones by acetylation plays a key role in epigenetic regulation of gene expression and is controlled by the balance between histone deacetylases (HDAC) and histone acetyltransferases (HAT).  HDAC inhibitors induce cancer cell cycle arrest, differentiation and cell death, reduce angiogenesis and modulate immune response.  Mechanisms of anticancer effects of HDAC inhibitors are not uniform; they may be different and depend on the cancer type, HDAC inhibitors, doses, etc.  HDAC inhibitors seem to be promising anti-cancer drugs particularly in the combination with other anti-cancer drugs and/or radiotherapy.  HDAC inhibitors vorinostat, romidepsin and belinostat have been approved for some T-cell lymphoma and panobinostat for multiple myeloma.  Other HDAC inhibitors are in clin. trials for the treatment of hematol. and solid malignancies.  The results of such studies are promising but further larger studies are needed.  Because of the reversibility of epigenetic changes during cancer development, the potency of epigenetic therapies seems to be of great importance.  Here, we summarize the data on different classes of HDAC inhibitors, mechanisms of their actions and discuss novel results of preclin. and clin. studies, including the combination with other therapeutic modalities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPNVhoLBAHnrVg90H21EOLACvtfcHk0liB-XHYq3-EcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVGgt73P&md5=6018c7709a09548214bfe34bd91107a9</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3390%2Fijms18071414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms18071414%26sid%3Dliteratum%253Aachs%26aulast%3DEckschlager%26aufirst%3DT.%26aulast%3DPlch%26aufirst%3DJ.%26aulast%3DStiborova%26aufirst%3DM.%26aulast%3DHrabeta%26aufirst%3DJ.%26atitle%3DHistone%2520Deacetylase%2520Inhibitors%2520as%2520Anticancer%2520Drugs%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2017%26volume%3D18%26spage%3D1414%26doi%3D10.3390%2Fijms18071414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, S.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylases as Targets for Treatment of Multiple Diseases</span>. <i>Clin. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">651</span>– <span class="NLM_lpage">662</span>, <span class="refDoi"> DOI: 10.1042/CS20120504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1042%2FCS20120504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=23414309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC3sXislOmt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2013&pages=651-662&author=J.+Tangauthor=H.+Yanauthor=S.+Zhuang&title=Histone+Deacetylases+as+Targets+for+Treatment+of+Multiple+Diseases&doi=10.1042%2FCS20120504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylases as targets for treatment of multiple diseases</span></div><div class="casAuthors">Tang, Jinhua; Yan, Haidong; Zhuang, Shougang</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Science</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">651-662</span>CODEN:
                <span class="NLM_cas:coden">CSCIAE</span>;
        ISSN:<span class="NLM_cas:issn">1470-8736</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  HDACs (histone deacetylases) are a group of enzymes that deacetylate histones as well as non-histone proteins.  They are known as modulators of gene transcription and are assocd. with proliferation and differentiation of a variety of cell types and the pathogenesis of some diseases.  Recently, HDACs have come to be considered crucial targets in various diseases, including cancer, interstitial fibrosis, autoimmune and inflammatory diseases, and metabolic disorders.  Pharmacol. inhibitors of HDACs have been used or tested to treat those diseases.  In the present review, we will examine the application of HDAC inhibitors in a variety of diseases with the focus on their effects of anti-cancer, fibrosis, anti-inflammatory, immunomodulatory activity and regulating metabolic disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpP-M9OvYul7bVg90H21EOLACvtfcHk0liB-XHYq3-EcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXislOmt7c%253D&md5=cc0d32170884436c10d5010f7342c6bd</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1042%2FCS20120504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FCS20120504%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DZhuang%26aufirst%3DS.%26atitle%3DHistone%2520Deacetylases%2520as%2520Targets%2520for%2520Treatment%2520of%2520Multiple%2520Diseases%26jtitle%3DClin.%2520Sci.%26date%3D2013%26volume%3D124%26spage%3D651%26epage%3D662%26doi%3D10.1042%2FCS20120504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pandolfi, P. P.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylases and Transcriptional Therapy with Their Inhibitors</span>. <i>Cancer Chemother. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">S17</span>, <span class="refDoi"> DOI: 10.1007/s002800100322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1007%2Fs002800100322" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=11587360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD3MXot1GisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2001&pages=S17&author=P.+P.+Pandolfi&title=Histone+Deacetylases+and+Transcriptional+Therapy+with+Their+Inhibitors&doi=10.1007%2Fs002800100322"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylases and transcriptional therapy with their inhibitors</span></div><div class="casAuthors">Pandolfi, Pier Paolo</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Chemotherapy and Pharmacology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">S17-S19</span>CODEN:
                <span class="NLM_cas:coden">CCPHDZ</span>;
        ISSN:<span class="NLM_cas:issn">0344-5704</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag</span>)
        </div><div class="casAbstract">A review.  Acute promyelocytic leukemia (APL) is characterized by the expansion of malignant myeloid cells blocked at the promyelocytic stage of hemopoietic development and is invariably assocd. with reciprocal chromosomal translocations involving the retinoic acid receptor α (RARα) gene on chromosome 17.  RARα variably fuses to PML, PLZF, NPM, NuMA, and Stat5B genes (X genes/proteins).  These translocations are balanced and reciprocal, thus leading to the generation of X-RARα and RARα-X fusion genes of which the products coexist in the APL blast.  The invariable involvement in these translocations of RARα, a prototypical transcription factor, makes APL a compelling example of aberrant transcriptional mechanisms in the etiopathogenesis of cancer.  This paper focuses on the recent progress in defining the mol. mechanisms underlying APL pathogenesis and addresses how this new understanding has allowed the proposal and development of novel therapeutic strategies with compds. such as histone deacetylase inhibitors and inorg. arsenicals such as As2O3 which are currently being tested in murine leukemia models as well as in human APL patients.  In particular, the crucial role played by the aberrant transcriptional activities of X-RARα and RARα-X fusion proteins in APL pathogenesis is discussed by reviewing the relevant therapeutic implications resulting from this anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQNb7pqi3oCrVg90H21EOLACvtfcHk0liB-XHYq3-EcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXot1GisLc%253D&md5=56665a664a42902528ff33cbbbe5b570</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2Fs002800100322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs002800100322%26sid%3Dliteratum%253Aachs%26aulast%3DPandolfi%26aufirst%3DP.%2BP.%26atitle%3DHistone%2520Deacetylases%2520and%2520Transcriptional%2520Therapy%2520with%2520Their%2520Inhibitors%26jtitle%3DCancer%2520Chemother.%2520Pharmacol.%26date%3D2001%26volume%3D48%26spage%3DS17%26doi%3D10.1007%2Fs002800100322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thiagalingam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mineva, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiagalingam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponte, J. F.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylases: Unique Players in Shaping the Epigenetic Histone Code</span>. <i>Ann. N. Y. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>983</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1111/j.1749-6632.2003.tb05964.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1111%2Fj.1749-6632.2003.tb05964.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=12724214" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjt1yrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=983&publication_year=2003&pages=84-100&author=S.+Thiagalingamauthor=K.+H.+Chengauthor=H.+J.+Leeauthor=N.+Minevaauthor=A.+Thiagalingamauthor=J.+F.+Ponte&title=Histone+Deacetylases%3A+Unique+Players+in+Shaping+the+Epigenetic+Histone+Code&doi=10.1111%2Fj.1749-6632.2003.tb05964.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylases: unique players in shaping the epigenetic histone code</span></div><div class="casAuthors">Thiagalingam, Sam; Cheng, Kuang-Hung; Lee, Hyunjoo J.; Mineva, Nora; Thiagalingam, Arunthathi; Ponte, Jose F.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the New York Academy of Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">983</span>
        (<span class="NLM_cas:issue">Epigenetics in Cancer Prevention</span>),
    <span class="NLM_cas:pages">84-100</span>CODEN:
                <span class="NLM_cas:coden">ANYAA9</span>;
        ISSN:<span class="NLM_cas:issn">0077-8923</span>.
    
            (<span class="NLM_cas:orgname">New York Academy of Sciences</span>)
        </div><div class="casAbstract">A review.  The epigenome is defined by DNA methylation patterns and the assocd. posttranslational modifications of histones.  This histone code dets. the expression status of individual genes dependent upon their localization on the chromatin.  The silencing of gene expression is assocd. with deacetylated histones, which are often found to be assocd. with regions of DNA methylation as well as methylation at the lysine 4 residue of histone 3.  In contrast, the activation of gene expression is assocd. with acetylated histones and methylation at the lysine 9 residue of histone 3.  The histone deacetylases play a major role in keeping the balance between the acetylated and deacetylated states of chromatin.  Histone deacetylases (HDACs) are divided into three classes: class I HDACs (HDACs 1, 2, 3, and 8) are similar to the yeast RPD3 protein and localize to the nucleus; class II HDACs (HDACs 4, 5, 6, 7, 9, and 10) are homologous to the yeast HDA1 protein and are found in both the nucleus and cytoplasm; and class III HDACs form a structurally distinct class of NAD-dependent enzymes that are similar to the yeast SIR2 proteins.  Since inappropriate silencing of crit. genes can result in one or both hits of tumor suppressor gene (TSG) inactivation in cancer, theor. the reactivation of affected TSGs could have an enormous therapeutic value in preventing and treating cancer.  Indeed, several HDAC inhibitors are currently being developed and tested for their potency in cancer chemotherapy.  Importantly, these agents are also potentially applicable to chemoprevention if their toxicity can be minimized.  Despite the toxic side effects and lack of specificity of some of the inhibitors, progress is being made.  With the elucidation of the structures, functions and modes of action of HDACs, finding agents that may be targeted to specific HDACs and potentially reactivate expression of only a defined set of affected genes in cancer will be more attainable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJPAGtCnCINbVg90H21EOLACvtfcHk0liAydslRi9HCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjt1yrs7g%253D&md5=e288e2f402f1cab9f1d51ddffc0ac87c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1111%2Fj.1749-6632.2003.tb05964.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1749-6632.2003.tb05964.x%26sid%3Dliteratum%253Aachs%26aulast%3DThiagalingam%26aufirst%3DS.%26aulast%3DCheng%26aufirst%3DK.%2BH.%26aulast%3DLee%26aufirst%3DH.%2BJ.%26aulast%3DMineva%26aufirst%3DN.%26aulast%3DThiagalingam%26aufirst%3DA.%26aulast%3DPonte%26aufirst%3DJ.%2BF.%26atitle%3DHistone%2520Deacetylases%253A%2520Unique%2520Players%2520in%2520Shaping%2520the%2520Epigenetic%2520Histone%2520Code%26jtitle%3DAnn.%2520N.%2520Y.%2520Acad.%2520Sci.%26date%3D2003%26volume%3D983%26spage%3D84%26epage%3D100%26doi%3D10.1111%2Fj.1749-6632.2003.tb05964.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haberland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, E. N.</span></span> <span> </span><span class="NLM_article-title">The Many Roles of Histone Deacetylases in Development and Physiology: Implications for Disease and Therapy</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1038/nrg2485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fnrg2485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=19065135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFWis7bI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=32-42&author=M.+Haberlandauthor=R.+L.+Montgomeryauthor=E.+N.+Olson&title=The+Many+Roles+of+Histone+Deacetylases+in+Development+and+Physiology%3A+Implications+for+Disease+and+Therapy&doi=10.1038%2Fnrg2485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The many roles of histone deacetylases in development and physiology: implications for disease and therapy</span></div><div class="casAuthors">Haberland, Michael; Montgomery, Rusty L.; Olson, Eric N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-42</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The expression of many histone deacetylase (HDAC) isoforms in eukaryotic cells raises questions regarding their specificity and the programs of gene expression that they control.  HDAC knockout mice are a powerful tool for addressing these questions and have revealed that individual HDACs have specific functions in development and disease.  Histone deacetylases (HDACs) are part of a vast family of enzymes that have crucial roles in numerous biol. processes, largely through their repressive influence on transcription.  The expression of many HDAC isoforms in eukaryotic cells raises questions about their possible specificity or redundancy, and whether they control global or specific programs of gene expression.  Recent analyses of HDAC knockout mice have revealed highly specific functions of individual HDACs in development and disease.  Mutant mice lacking individual HDACs are a powerful tool for defining the functions of HDACs in vivo and the mol. targets of HDAC inhibitors in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMWcIo3mrsM7Vg90H21EOLACvtfcHk0liAydslRi9HCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFWis7bI&md5=0678d8655e86994cd4f0174c1e9d0eb1</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrg2485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg2485%26sid%3Dliteratum%253Aachs%26aulast%3DHaberland%26aufirst%3DM.%26aulast%3DMontgomery%26aufirst%3DR.%2BL.%26aulast%3DOlson%26aufirst%3DE.%2BN.%26atitle%3DThe%2520Many%2520Roles%2520of%2520Histone%2520Deacetylases%2520in%2520Development%2520and%2520Physiology%253A%2520Implications%2520for%2520Disease%2520and%2520Therapy%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2009%26volume%3D10%26spage%3D32%26epage%3D42%26doi%3D10.1038%2Fnrg2485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allfrey, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faulkner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirsky, A. E.</span></span> <span> </span><span class="NLM_article-title">Acetylation and Methylation of Histones and Their Possible Role in the Regulation of RNA</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1964</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">786</span>– <span class="NLM_lpage">794</span>, <span class="refDoi"> DOI: 10.1073/pnas.51.5.786</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1073%2Fpnas.51.5.786" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=14172992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADyaF2cXktlekurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1964&pages=786-794&author=V.+G.+Allfreyauthor=R.+Faulknerauthor=A.+E.+Mirsky&title=Acetylation+and+Methylation+of+Histones+and+Their+Possible+Role+in+the+Regulation+of+RNA&doi=10.1073%2Fpnas.51.5.786"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Acetylation and methylation of histones and their possible role in the regulation of ribonucleic acid (RNA) synthesis</span></div><div class="casAuthors">Allfrey, V. G.; Faulkner, R.; Mirsky, A. E.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1964</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">786-94</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Isolated calf thymus nuclei was incubated with 14C-labeled Na acetate.  Evidence that acetate was incorporated as acetyl groups attached to histones included: (1) following chromatography the label was eluted together with basic proteins; (2) acetyl-labeled histones were sepd. by electrophoresis; (3) acetate-14C was not removed from histones by org. solvents; (4) over 75% of the label remained after histones were treated with trichloroacetic acid.  Most extensive acetate uptake occurred with arginine-rich histones.  Puromycin did not inhibit the acetylation, indicating histones were modified by acetylation after completion of the polypeptide chain.  Acetylation lowered the histone effectiveness as inhibitor of the RNA polymerase reaction, and a dynamic and reversible mechanism was suggested for activation as well as repression of RNA synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpL5QIm_JdO1bVg90H21EOLACvtfcHk0liAydslRi9HCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF2cXktlekurg%253D&md5=7a80f7a8a12636d4bc47930ee5eff217</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.51.5.786&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.51.5.786%26sid%3Dliteratum%253Aachs%26aulast%3DAllfrey%26aufirst%3DV.%2BG.%26aulast%3DFaulkner%26aufirst%3DR.%26aulast%3DMirsky%26aufirst%3DA.%2BE.%26atitle%3DAcetylation%2520and%2520Methylation%2520of%2520Histones%2520and%2520Their%2520Possible%2520Role%2520in%2520the%2520Regulation%2520of%2520RNA%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1964%26volume%3D51%26spage%3D786%26epage%3D794%26doi%3D10.1073%2Fpnas.51.5.786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, M.</span></span> <span> </span><span class="NLM_article-title">Epigenome-Based Personalized Medicine in Human Cancer</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">119</span>– <span class="NLM_lpage">133</span>, <span class="refDoi"> DOI: 10.2217/epi.15.84</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.2217%2Fepi.15.84" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=26344672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC28Xos1aqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=119-133&author=W.+Yanauthor=J.+G.+Hermanauthor=M.+Guo&title=Epigenome-Based+Personalized+Medicine+in+Human+Cancer&doi=10.2217%2Fepi.15.84"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenome-based personalized medicine in human cancer</span></div><div class="casAuthors">Yan, Wenji; Herman, James G.; Guo, Mingzhou</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">119-133</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">Cancer genome sequencing has created an opportunity for precision medicine.  Thus far, genetic alterations can only be used to guide treatment for small subsets of certain cancer types with these key alterations.  Similar to mutations, epigenetic events are equally suitable for personalized medicine.  DNA methylation alterations have been used to identify tumor-specific drug responsive markers.  Methylation of MGMT sensitizes gliomas to alkylating agents is an example of epigenetic personalized medicine.  Recent studies have revealed that 5-azacytidine and decitabine show activity in myelodysplasia, lung and other cancers.  There are currently at least 20 kinds of histone deacetylase inhibitors in clin. testing.  Inhibitors targeting other epigenetic regulators are being clin. tested, such as EZH2 inhibitor EPZ-6438.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_za-KvY9spLVg90H21EOLACvtfcHk0ljx7ftbtiBc-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xos1aqtA%253D%253D&md5=f872718075e1a9b7f7788965fe143c53</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.2217%2Fepi.15.84&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.15.84%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DW.%26aulast%3DHerman%26aufirst%3DJ.%2BG.%26aulast%3DGuo%26aufirst%3DM.%26atitle%3DEpigenome-Based%2520Personalized%2520Medicine%2520in%2520Human%2520Cancer%26jtitle%3DEpigenomics%26date%3D2016%26volume%3D8%26spage%3D119%26epage%3D133%26doi%3D10.2217%2Fepi.15.84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lombardi, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowling, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Structure, Mechanism, and Inhibition of Histone Deacetylases and Related Metalloenzymes</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">735</span>– <span class="NLM_lpage">743</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2011.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1016%2Fj.sbi.2011.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=21872466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFOhurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=735-743&author=P.+M.+Lombardiauthor=K.+E.+Coleauthor=D.+P.+Dowlingauthor=D.+W.+Christianson&title=Structure%2C+Mechanism%2C+and+Inhibition+of+Histone+Deacetylases+and+Related+Metalloenzymes&doi=10.1016%2Fj.sbi.2011.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structure, mechanism, and inhibition of histone deacetylases and related metalloenzymes</span></div><div class="casAuthors">Lombardi, Patrick M.; Cole, Kathryn E.; Dowling, Daniel P.; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">735-743</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Metal-dependent histone deacetylases (HDACs) catalyze the hydrolysis of acetyl-L-lysine side-chains in histone and nonhistone proteins to yield L-lysine and acetate.  This chem. plays a crit. role in the regulation of numerous biol. processes.  Aberrant HDAC activity is implicated in various diseases, and HDACs are validated targets for drug design.  Two HDAC inhibitors are currently approved for cancer chemotherapy, and other inhibitors are in clin. trials.  To date, x-ray crystal structures are available for 4 human HDACs (2, 4, 7, and 8) and 3 HDAC-related deacetylases from bacteria [histone deacetylase-like protein (HDLP), histone deacetylase-like amidohydrolase (HDAH), and acetylpolyamine amidohydrolase (APAH)].  Structural comparisons among these enzymes reveal a conserved constellation of active site residues, suggesting a common mechanism for the metal-dependent hydrolysis of acetylated substrates.  Structural analyses of HDACs and HDAC-related deacetylases guide the design of tight-binding inhibitors, and future prospects for developing isoenzyme-specific inhibitors are quite promising.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2SnZUQ2FtTrVg90H21EOLACvtfcHk0ljx7ftbtiBc-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFOhurrJ&md5=edab41900a9f233d7ff30c507cfcce66</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2011.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2011.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DLombardi%26aufirst%3DP.%2BM.%26aulast%3DCole%26aufirst%3DK.%2BE.%26aulast%3DDowling%26aufirst%3DD.%2BP.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DStructure%252C%2520Mechanism%252C%2520and%2520Inhibition%2520of%2520Histone%2520Deacetylases%2520and%2520Related%2520Metalloenzymes%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2011%26volume%3D21%26spage%3D735%26epage%3D743%26doi%3D10.1016%2Fj.sbi.2011.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feinberg, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tycko, B.</span></span> <span> </span><span class="NLM_article-title">The History of Cancer Epigenetics</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">153</span>, <span class="refDoi"> DOI: 10.1038/nrc1279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fnrc1279" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=14732866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2cXks1CktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=143-153&author=A.+P.+Feinbergauthor=B.+Tycko&title=The+History+of+Cancer+Epigenetics&doi=10.1038%2Fnrc1279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Timeline: The history of cancer epigenetics</span></div><div class="casAuthors">Feinberg, Andrew P.; Tycko, Benjamin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">143-153</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Since its discovery in 1983, the epigenetics of human cancer has been in the shadows of human cancer genetics.  But this area has become increasingly visible with a growing understanding of specific epigenetic mechanisms and their role in cancer, including hypomethylation, hypermethylation, loss of imprinting and chromatin modification.  This timeline traces the field from its conception to the present day.  It also addresses the genetic basis of epigenetic changes - an emerging area that promises to unite cancer genetics and epigenetics, and might serve as a model for understanding the epigenetic basis of human disease more generally.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIbizd6X65lrVg90H21EOLACvtfcHk0ljx7ftbtiBc-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXks1CktA%253D%253D&md5=1e1c0ccf32e48422fd1842593b5062be</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrc1279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1279%26sid%3Dliteratum%253Aachs%26aulast%3DFeinberg%26aufirst%3DA.%2BP.%26aulast%3DTycko%26aufirst%3DB.%26atitle%3DThe%2520History%2520of%2520Cancer%2520Epigenetics%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2004%26volume%3D4%26spage%3D143%26epage%3D153%26doi%3D10.1038%2Fnrc1279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tran, A. D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmo, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salam, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkelstein, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabarriti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xenias, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazitschek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawaguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheetz, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulinski, J. C.</span></span> <span> </span><span class="NLM_article-title">HDAC6 Deacetylation of Tubulin Modulates Dynamics of Cellular Adhesions</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">1469</span>– <span class="NLM_lpage">1479</span>, <span class="refDoi"> DOI: 10.1242/jcs.03431</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1242%2Fjcs.03431" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=17389687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlsFSjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2007&pages=1469-1479&author=A.+D.+A.+Tranauthor=T.+P.+Marmoauthor=A.+A.+Salamauthor=S.+Cheauthor=E.+Finkelsteinauthor=R.+Kabarritiauthor=H.+S.+Xeniasauthor=R.+Mazitschekauthor=C.+Hubbertauthor=Y.+Kawaguchiauthor=M.+P.+Sheetzauthor=T.+Yaoauthor=J.+C.+Bulinski&title=HDAC6+Deacetylation+of+Tubulin+Modulates+Dynamics+of+Cellular+Adhesions&doi=10.1242%2Fjcs.03431"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">HDAC6 deacetylation of tubulin modulates dynamics of cellular adhesions</span></div><div class="casAuthors">Tran, Andy Dong-Anh; Marmo, Timothy P.; Salam, Ambar A.; Che, Sally; Finkelstein, Erik; Kabarriti, Rafi; Xenias, Harry S.; Mazitschek, Ralph; Hubbert, Charlotte; Kawaguchi, Yoshiharu; Sheetz, Michael P.; Yao, Tso-Pang; Bulinski, J. Chloe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1469-1479</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Genetic or pharmacol. alteration of the activity of histone deacetylase 6 (HDAC6) induces a parallel alteration in cell migration.  Using tubacin to block deacetylation of α-tubulin, and not other HDAC6 substrates, yielded a motility redn. equiv. to agents that block all NAD-independent HDACs.  Accordingly, the authors investigated how the failure to deacetylate tubulin contributes to decreased motility in HDAC6-inhibited cells.  Testing the hypothesis that motility is reduced because cellular adhesion is altered, it was found that inhibiting HDAC6 activity toward tubulin rapidly increased the total adhesion area.  Next, the authors investigated the mechanism of the adhesion area increase.  Formation of adhesions proceeded normally and cell spreading was more rapid in the absence of active HDAC6; however, photobleaching assays and adhesion breakdown showed that adhesion turnover was slower.  To test the role of hyperacetylated tubulin in altering adhesion turnover, the authors measured microtubule dynamics in HDAC6-inhibited cells because dynamic microtubules are required to target adhesions for turnover.  HDAC6 inhibition yielded a decrease in microtubule dynamics that was sufficient to decrease focal adhesion turnover.  Thus, the results suggest a scenario in which the decreased dynamics of hyperacetylated microtubules in HDAC6-inhibited cells compromises their capacity to mediate the focal adhesion dynamics required for rapid cell migration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTqrWHdqcFXrVg90H21EOLACvtfcHk0ljx7ftbtiBc-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlsFSjtbs%253D&md5=6e25ead47afba657cf58bf4287df25fb</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1242%2Fjcs.03431&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.03431%26sid%3Dliteratum%253Aachs%26aulast%3DTran%26aufirst%3DA.%2BD.%2BA.%26aulast%3DMarmo%26aufirst%3DT.%2BP.%26aulast%3DSalam%26aufirst%3DA.%2BA.%26aulast%3DChe%26aufirst%3DS.%26aulast%3DFinkelstein%26aufirst%3DE.%26aulast%3DKabarriti%26aufirst%3DR.%26aulast%3DXenias%26aufirst%3DH.%2BS.%26aulast%3DMazitschek%26aufirst%3DR.%26aulast%3DHubbert%26aufirst%3DC.%26aulast%3DKawaguchi%26aufirst%3DY.%26aulast%3DSheetz%26aufirst%3DM.%2BP.%26aulast%3DYao%26aufirst%3DT.%26aulast%3DBulinski%26aufirst%3DJ.%2BC.%26atitle%3DHDAC6%2520Deacetylation%2520of%2520Tubulin%2520Modulates%2520Dynamics%2520of%2520Cellular%2520Adhesions%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2007%26volume%3D120%26spage%3D1469%26epage%3D1479%26doi%3D10.1242%2Fjcs.03431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubbert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guardiola, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawaguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nixon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, T. P.</span></span> <span> </span><span class="NLM_article-title">HDAC6 Is a Microtubule-Associated Deacetylase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>417</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">458</span>, <span class="refDoi"> DOI: 10.1038/417455a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2F417455a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=12024216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD38XjvVersr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=417&publication_year=2002&pages=455-458&author=C.+Hubbertauthor=A.+Guardiolaauthor=R.+Shaoauthor=Y.+Kawaguchiauthor=A.+Itoauthor=A.+Nixonauthor=M.+Yoshidaauthor=X.+F.+Wangauthor=T.+P.+Yao&title=HDAC6+Is+a+Microtubule-Associated+Deacetylase&doi=10.1038%2F417455a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">HDAC6 is a microtubule-associated deacetylase</span></div><div class="casAuthors">Hubbert, Charlotte; Guardiola, Amaris; Shao, Rong; Kawaguchi, Yoshiharu; Ito, Akihiro; Nixon, Andrew; Yoshida, Minoru; Wang, Xiao-Fan; Yao, Tso-Pang</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">417</span>
        (<span class="NLM_cas:issue">6887</span>),
    <span class="NLM_cas:pages">455-458</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Reversible acetylation of α-tubulin has been implicated in regulating microtubule stability and function.  The distribution of acetylated α-tubulin is tightly controlled and stereotypic.  Acetylated α-tubulin is most abundant in stable microtubules but is absent from dynamic cellular structures such as neuronal growth cones and the leading edges of fibroblasts.  However, the enzymes responsible for regulating tubulin acetylation and deacetylation are not known.  Here we report that a member of the histone deacetylase family, HDAC6, functions as a tubulin deacetylase.  HDAC6 is localized exclusively in the cytoplasm, where it assocs. with microtubules and localizes with the microtubule motor complex contg. p150glued.  In vivo, the overexpression of HDAC6 leads to a global deacetylation of α-tubulin, whereas a decrease in HDAC6 increases α-tubulin acetylation.  In vitro, purified HDAC6 potently deacetylates α-tubulin in assembled microtubules.  Furthermore, overexpression of HDAC6 promotes chemotactic cell movement, supporting the idea that HDAC6-mediated deacetylation regulates microtubule-dependent cell motility.  Our results show that HDAC6 is the tubulin deacetylase, and provide evidence that reversible acetylation regulates important biol. processes beyond histone metab. and gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp76Er3MrWtO7Vg90H21EOLACvtfcHk0lhAt5PFbYmNiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjvVersr4%253D&md5=f18cbb0af388925c606e822358ba3da6</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2F417455a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F417455a%26sid%3Dliteratum%253Aachs%26aulast%3DHubbert%26aufirst%3DC.%26aulast%3DGuardiola%26aufirst%3DA.%26aulast%3DShao%26aufirst%3DR.%26aulast%3DKawaguchi%26aufirst%3DY.%26aulast%3DIto%26aufirst%3DA.%26aulast%3DNixon%26aufirst%3DA.%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DX.%2BF.%26aulast%3DYao%26aufirst%3DT.%2BP.%26atitle%3DHDAC6%2520Is%2520a%2520Microtubule-Associated%2520Deacetylase%26jtitle%3DNature%26date%3D2002%26volume%3D417%26spage%3D455%26epage%3D458%26doi%3D10.1038%2F417455a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krämer, O. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahboobi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellmer, A.</span></span> <span> </span><span class="NLM_article-title">Drugging the HDAC6-HSP90 Interplay in Malignant Cells</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">501</span>– <span class="NLM_lpage">509</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2014.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1016%2Fj.tips.2014.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=25234862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A280%3ADC%252BC2M7ivFKgtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=501-509&author=O.+H.+Kr%C3%A4merauthor=S.+Mahboobiauthor=A.+Sellmer&title=Drugging+the+HDAC6-HSP90+Interplay+in+Malignant+Cells&doi=10.1016%2Fj.tips.2014.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging the HDAC6-HSP90 interplay in malignant cells</span></div><div class="casAuthors">Kramer Oliver H; Mahboobi Siavosh; Sellmer Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Trends in pharmacological sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">501-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Acetylation and deacetylation cycles regulate crucial biological processes.  The enzymes deacetylating lysine residues are termed histone deacetylases (HDACs).  Eighteen deacetylases have been isolated from mammalian cells.  There is an intense search underway for individual functions of these enzymes and for selective histone deacetylase inhibitors (HDACi).  HDAC6 in particular has unique cytoprotective functions that rely on its ability to ensure protein homeostasis and to prevent protein aggregation.  The chaperone heat shock protein 90 (HSP90) also safeguards proteins and is deacetylated by HDAC6.  Current data illustrate the complexity and importance of the HDAC6-HSP90 interplay.  In this review, we discuss how recently identified HSP90-dependent regulators of posttranslational modifications (PTMs) of HDAC6 dictate its functions, and how HDACi-induced acetylation of HSP90 might control oncologically relevant proteins, especially in leukemic cells.  Additionally, we discuss small molecules blocking HDAC6 and how such agents could become therapeutically relevant.  We summarize structure-function relationships that determine the specificity of drugs against HDAC6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS6kyzjnmxFum52AMpI4GK2fW6udTcc2eZ4-cEtux14Pbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M7ivFKgtw%253D%253D&md5=4aa468225dd58345f164ca1a10e324a2</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2014.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2014.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DKr%25C3%25A4mer%26aufirst%3DO.%2BH.%26aulast%3DMahboobi%26aufirst%3DS.%26aulast%3DSellmer%26aufirst%3DA.%26atitle%3DDrugging%2520the%2520HDAC6-HSP90%2520Interplay%2520in%2520Malignant%2520Cells%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2014%26volume%3D35%26spage%3D501%26epage%3D509%26doi%3D10.1016%2Fj.tips.2014.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthias, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthias, P.</span></span> <span> </span><span class="NLM_article-title">HDAC-6 Interacts with and Deacetylates Tubulin and Microtubules in Vivo</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1168</span>– <span class="NLM_lpage">1179</span>, <span class="refDoi"> DOI: 10.1093/emboj/cdg115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1093%2Femboj%2Fcdg115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=12606581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD3sXit1yls7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=1168-1179&author=Y.+Zhangauthor=N.+Liauthor=C.+Caronauthor=G.+Matthiasauthor=D.+Hessauthor=S.+Khochbinauthor=P.+Matthias&title=HDAC-6+Interacts+with+and+Deacetylates+Tubulin+and+Microtubules+in+Vivo&doi=10.1093%2Femboj%2Fcdg115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">HDAC-6 interacts with and deacetylates tubulin and microtubules in vivo</span></div><div class="casAuthors">Zhang, Yu; Li, Na; Caron, Cecile; Matthias, Gabriele; Hess, Daniel; Khochbin, Saadi; Matthias, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1168-1179</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Microtubules are cylindrical cytoskeletal structures found in almost all eukaryotic cell types which are involved in a great variety of cellular processes.  Reversible acetylation on the ε-amino group of α-tubulin Lys40 marks stabilized microtubule structures and may contribute to regulating microtubule dynamics.  Yet, the enzymes catalyzing this acetylation/deacetylation have remained unidentified until recently.  Here we report that β-tubulin interacts with histone deacetylase-6 (HDAC-6) in a yeast two-hybrid assay and in vitro.  We find that HDAC-6 is a micro tubule-assocd. protein capable of deacetylating α-tubulin in vivo and in vitro.  HDAC-6's microtubule binding and deacetylation functions both depend on the hdac domains.  Overexpression of HDAC-6 in mammalian cells leads to tubulin hypoacetylation.  In contrast, inhibition of HDAC-6 function by two independent mechanisms-pharmacol. (HDAC inhibitors) or genetic (targeted inactivation of HDAC-6 in embryonic stem cells)-leads to hyperacetylation of tubulin and microtubules.  Taken together, our data provide evidence that HDAC-6 might act as a dual deacetylase for tubulin and histones, and suggest the possibility that acetylated non-histone proteins might represent novel targets for pharmacol. therapy by HDAC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBqjHB5aVYPrVg90H21EOLACvtfcHk0lh_OwWGwC2qWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXit1yls7c%253D&md5=0d2b6dd552163866b85f14471ac600f7</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1093%2Femboj%2Fcdg115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252Fcdg115%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DN.%26aulast%3DCaron%26aufirst%3DC.%26aulast%3DMatthias%26aufirst%3DG.%26aulast%3DHess%26aufirst%3DD.%26aulast%3DKhochbin%26aufirst%3DS.%26aulast%3DMatthias%26aufirst%3DP.%26atitle%3DHDAC-6%2520Interacts%2520with%2520and%2520Deacetylates%2520Tubulin%2520and%2520Microtubules%2520in%2520Vivo%26jtitle%3DEMBO%2520J.%26date%3D2003%26volume%3D22%26spage%3D1168%26epage%3D1179%26doi%3D10.1093%2Femboj%2Fcdg115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cosenza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, S.</span></span> <span> </span><span class="NLM_article-title">The Therapeutic Strategy of HDAC6 Inhibitors in Lymphoproliferative Disease</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2337</span>, <span class="refDoi"> DOI: 10.3390/ijms19082337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.3390%2Fijms19082337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsVGhtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=2337&author=M.+Cosenzaauthor=S.+Pozzi&title=The+Therapeutic+Strategy+of+HDAC6+Inhibitors+in+Lymphoproliferative+Disease&doi=10.3390%2Fijms19082337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The therapeutic strategy of HDAC6 inhibitors in lymphoproliferative disease</span></div><div class="casAuthors">Cosenza, Maria; Pozzi, Samantha</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2337/1-2337/26</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Histone deacetylases (HDACs) are master regulators of chromatin remodeling, acting as epigenetic regulators of gene expression.  In the last decade, inhibition of HDACs has become a target for specific epigenetic modifications related to cancer development.  Overexpression of HDAC has been obsd. in several hematol. malignancies.  Therefore, the observation that HDACs might play a role in various hematol. malignancies has brought to the development of HDAC inhibitors as potential antitumor agents.  Recently, the class IIb, HDAC6, has emerged as one potential selective HDACi.  This isoenzyme represents an important pharmacol. target for selective inhibition.  Its selectivity may reduce the toxicity related to the off-target effects of pan-HDAC inhibitors.  HDAC6 has also been studied in cancer esp. for its ability to coordinate a variety of cellular processes that are important for cancer pathogenesis.  HDAC6 has been reported to be overexpressed in lymphoid cells and its inhibition has demonstrated activity in preclin. and clin. study of lymphoproliferative disease.  Various studies of HDAC6 inhibitors alone and in combination with other agents provide strong scientific rationale for the evaluation of these new agents in the clin. setting of hematol. malignancies.  In this review, we describe the HDACs, their inhibitors, and the recent advances of HDAC6 inhibitors, their mechanisms of action and role in lymphoproliferative disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr40AmGWIhm0rVg90H21EOLACvtfcHk0lh_OwWGwC2qWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsVGhtrY%253D&md5=e2f63affb8a194d516a2465e772f46c1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.3390%2Fijms19082337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms19082337%26sid%3Dliteratum%253Aachs%26aulast%3DCosenza%26aufirst%3DM.%26aulast%3DPozzi%26aufirst%3DS.%26atitle%3DThe%2520Therapeutic%2520Strategy%2520of%2520HDAC6%2520Inhibitors%2520in%2520Lymphoproliferative%2520Disease%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2018%26volume%3D19%26spage%3D2337%26doi%3D10.3390%2Fijms19082337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormack, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams-Truax, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mita, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burris, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butterfoss, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dugan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culver, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prince, H. M.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylase Inhibitor Panobinostat Induces Clinical Responses with Associated Alterations in Gene Expression Profiles in Cutaneous T-Cell Lymphoma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">4500</span>– <span class="NLM_lpage">4510</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-07-4262</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1158%2F1078-0432.CCR-07-4262" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=18628465" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD1cXosFGitL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=4500-4510&author=L.+Ellisauthor=Y.+Panauthor=G.+K.+Smythauthor=D.+J.+Georgeauthor=C.+McCormackauthor=R.+Williams-Truaxauthor=M.+Mitaauthor=J.+Beckauthor=H.+Burrisauthor=G.+Ryanauthor=P.+Atadjaauthor=D.+Butterfossauthor=M.+Duganauthor=K.+Culverauthor=R.+W.+Johnstoneauthor=H.+M.+Prince&title=Histone+Deacetylase+Inhibitor+Panobinostat+Induces+Clinical+Responses+with+Associated+Alterations+in+Gene+Expression+Profiles+in+Cutaneous+T-Cell+Lymphoma&doi=10.1158%2F1078-0432.CCR-07-4262"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Deacetylase Inhibitor Panobinostat Induces Clinical Responses with Associated Alterations in Gene Expression Profiles in Cutaneous T-Cell Lymphoma</span></div><div class="casAuthors">Ellis, Leigh; Pan, Yan; Smyth, Gordon K.; George, Daniel J.; McCormack, Chris; Williams-Truax, Roxanne; Mita, Monica; Beck, Joachim; Burris, Howard; Ryan, Gail; Atadja, Peter; Butterfoss, Dale; Dugan, Margaret; Culver, Kenneth; Johnstone, Ricky W.; Prince, H. Miles</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4500-4510</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Histone deacetylase inhibitors can alter gene expression and mediate diverse antitumor activities.  Herein, we report the safety and activity of the histone deacetylase inhibitor panobinostat (LBH589) in cutaneous T-cell lymphoma (CTCL) and identify genes commonly regulated by panobinostat.  Panobinostat was administered orally to patients with CTCL on Monday, Wednesday, and Friday of each week on a 28-day cycle.  A dose of 30 mg was considered excessively toxic, and subsequent patients were treated at the expanded max. tolerated dose of 20 mg.  Biopsies from six patients taken 0, 4, 8, and 24 h after administration were subjected to microarray gene expression profiling and real-time quant. PCR of selected genes.  Patients attained a complete response (n = 2), attained a partial response (n = 4), achieved stable disease with ongoing improvement (n = 1), and progressed on treatment (n = 2).  Microarray data showed distinct gene expression response profiles over time following panobinostat treatment, with the majority of genes being repressed.  Twenty-three genes were commonly regulated by panobinostat in all patients tested.  Panobinostat is well tolerated and induces clin. responses in CTCL patients.  Microarray analyses of tumor samples indicate that panobinostat induces rapid changes in gene expression, and surprisingly more genes are repressed than are activated.  A unique set of genes that can mediate biol. responses such as apoptosis, immune regulation, and angiogenesis were commonly regulated in response to panobinostat.  These genes are potential mol. biomarkers for panobinostat activity and are strong candidates for the future assessment of their functional role(s) in mediating the antitumor responses of panobinostat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKdLh4qldQ37Vg90H21EOLACvtfcHk0lh_OwWGwC2qWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXosFGitL0%253D&md5=0b53d3b837c0f89a700287855ffcc961</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-07-4262&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-07-4262%26sid%3Dliteratum%253Aachs%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DPan%26aufirst%3DY.%26aulast%3DSmyth%26aufirst%3DG.%2BK.%26aulast%3DGeorge%26aufirst%3DD.%2BJ.%26aulast%3DMcCormack%26aufirst%3DC.%26aulast%3DWilliams-Truax%26aufirst%3DR.%26aulast%3DMita%26aufirst%3DM.%26aulast%3DBeck%26aufirst%3DJ.%26aulast%3DBurris%26aufirst%3DH.%26aulast%3DRyan%26aufirst%3DG.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DButterfoss%26aufirst%3DD.%26aulast%3DDugan%26aufirst%3DM.%26aulast%3DCulver%26aufirst%3DK.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26aulast%3DPrince%26aufirst%3DH.%2BM.%26atitle%3DHistone%2520Deacetylase%2520Inhibitor%2520Panobinostat%2520Induces%2520Clinical%2520Responses%2520with%2520Associated%2520Alterations%2520in%2520Gene%2520Expression%2520Profiles%2520in%2520Cutaneous%2520T-Cell%2520Lymphoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2008%26volume%3D14%26spage%3D4500%26epage%3D4510%26doi%3D10.1158%2F1078-0432.CCR-07-4262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koeller, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grozinger, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span> <span> </span><span class="NLM_article-title">Domain-Selective Small-Molecule Inhibitor of Histone Deacetylase 6 (HDAC6)-Mediated Tubulin Deacetylation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">4389</span>– <span class="NLM_lpage">4394</span>, <span class="refDoi"> DOI: 10.1073/pnas.0430973100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1073%2Fpnas.0430973100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=12677000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjt12gtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2003&pages=4389-4394&author=S.+J.+Haggartyauthor=K.+M.+Koellerauthor=J.+C.+Wongauthor=C.+M.+Grozingerauthor=S.+L.+Schreiber&title=Domain-Selective+Small-Molecule+Inhibitor+of+Histone+Deacetylase+6+%28HDAC6%29-Mediated+Tubulin+Deacetylation&doi=10.1073%2Fpnas.0430973100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Domain-selective small-molecule inhibitor of histone deacetylase 6 (HDAC6)-mediated tubulin deacetylation</span></div><div class="casAuthors">Haggarty, Stephen J.; Koeller, Kathryn M.; Wong, Jason C.; Grozinger, Christina M.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4389-4394</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein acetylation, esp. histone acetylation, is the subject of both research and clin. investigation.  At least four small-mol. histone deacetylase inhibitors are currently in clin. trials for the treatment of cancer.  These and other inhibitors also affect microtubule acetylation.  A multidimensional, chem. genetic screen of 7392 small mols. was used to discover "tubacin," which inhibits α-tubulin deacetylation in mammalian cells.  Tubacin does not affect the level of histone acetylation, gene-expression patterns, or cell-cycle progression.  We provide evidence that class II histone deacetylase 6 (HDAC6) is the intracellular target of tubacin.  Only one of the two catalytic domains of HDAC6 possesses tubulin deacetylase activity, and only this domain is bound by tubacin.  Tubacin treatment did not affect the stability of microtubules but did decrease cell motility.  HDAC6 overexpression disrupted the localization of p58, a protein that mediates binding of Golgi elements to microtubules.  Our results highlight the role of α-tubulin acetylation in mediating the localization of microtubule-assocd. proteins.  They also suggest that small mols. that selectively inhibit HDAC6-mediated α-tubulin deacetylation, a first example of which is tubacin, might have therapeutic applications as antimetastatic and antiangiogenic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbQBWjEZr47rVg90H21EOLACvtfcHk0lh_OwWGwC2qWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjt12gtL0%253D&md5=260da1ede64782ddbddda31b740b9782</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0430973100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0430973100%26sid%3Dliteratum%253Aachs%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DKoeller%26aufirst%3DK.%2BM.%26aulast%3DWong%26aufirst%3DJ.%2BC.%26aulast%3DGrozinger%26aufirst%3DC.%2BM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DDomain-Selective%2520Small-Molecule%2520Inhibitor%2520of%2520Histone%2520Deacetylase%25206%2520%2528HDAC6%2529-Mediated%2520Tubulin%2520Deacetylation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2003%26volume%3D100%26spage%3D4389%26epage%3D4394%26doi%3D10.1073%2Fpnas.0430973100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almeciga-Pinto, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarpe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Duzer, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazitschek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quayle, S. N.</span></span> <span> </span><span class="NLM_article-title">Selective HDAC Inhibition by ACY-241 Enhances the Activity of Paclitaxel in Solid Tumor Models</span>. <i>Oncotarget.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2694</span>– <span class="NLM_lpage">2707</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13738</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.18632%2Foncotarget.13738" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=27926524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A280%3ADC%252BC2sjksVSmuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=2694-2707&author=P.+Huangauthor=I.+Almeciga-Pintoauthor=M.+Jarpeauthor=J.+H.+van+Duzerauthor=R.+Mazitschekauthor=M.+Yangauthor=S.+S.+Jonesauthor=S.+N.+Quayle&title=Selective+HDAC+Inhibition+by+ACY-241+Enhances+the+Activity+of+Paclitaxel+in+Solid+Tumor+Models&doi=10.18632%2Foncotarget.13738"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Selective HDAC inhibition by ACY-241 enhances the activity of paclitaxel in solid tumor models</span></div><div class="casAuthors">Huang Pengyu; Almeciga-Pinto Ingrid; Jarpe Matthew; van Duzer John H; Yang Min; Jones Simon S; Quayle Steven N; Mazitschek Ralph</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">2694-2707</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">ACY-241 is a novel, orally available and selective histone deacetylase (HDAC) 6 inhibitor in Phase 1b clinical development in multiple myeloma (NCT 02400242).  Like the structurally related drug ACY-1215 (ricolinostat), ACY-241 has the potential for a substantially reduced side effect profile versus current nonselective HDAC inhibitor drug candidates due to reduced potency against Class I HDACs while retaining the potential for anticancer effectiveness.  We now show that combination treatment of xenograft models with paclitaxel and either ricolinostat or ACY-241 significantly suppresses solid tumor growth.  In cell lines from multiple solid tumor lineages, combination treatment with ACY-241 and paclitaxel enhanced inhibition of proliferation and increased cell death relative to either single agent alone.  Combination treatment with ACY-241 and paclitaxel also resulted in more frequent occurrence of mitotic cells with abnormal multipolar spindles and aberrant mitoses, consistent with the observed increase of aneuploid cells.  At the molecular level, multipolar mitotic spindle formation was observed to be NuMA-dependent and γ-tubulin independent, suggesting that treatment-induced multipolar spindle formation does not depend on centrosomal amplification.  The significantly enhanced efficacy of ACY-241 plus paclitaxel observed here, in addition to the anticipated superior safety profile of a selective HDAC6 inhibitor versus pan-HDAC inhibitors, provides a strong rationale for clinical development of this combination in patients with advanced solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQFLMaYGUFFIBfKb6ELK8t0fW6udTcc2eaR-y6dyufD17ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjksVSmuw%253D%253D&md5=b8a516375b03fc65536598ab9c0e4a21</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13738%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DP.%26aulast%3DAlmeciga-Pinto%26aufirst%3DI.%26aulast%3DJarpe%26aufirst%3DM.%26aulast%3Dvan%2BDuzer%26aufirst%3DJ.%2BH.%26aulast%3DMazitschek%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DS.%2BS.%26aulast%3DQuayle%26aufirst%3DS.%2BN.%26atitle%3DSelective%2520HDAC%2520Inhibition%2520by%2520ACY-241%2520Enhances%2520the%2520Activity%2520of%2520Paclitaxel%2520in%2520Solid%2520Tumor%2520Models%26jtitle%3DOncotarget.%26date%3D2017%26volume%3D8%26spage%3D2694%26epage%3D2707%26doi%3D10.18632%2Foncotarget.13738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hideshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarpe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Duzer, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazitschek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogier, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirstea, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodig, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scullen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canavese, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, N.</span></span> <span> </span><span class="NLM_article-title">Preclinical Activity, Pharmacodynamic, and Pharmacokinetic Properties of a Selective HDAC6 Inhibitor, ACY-1215, in Combination with Bortezomib in Multiple Myeloma</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">2579</span>– <span class="NLM_lpage">2589</span>, <span class="refDoi"> DOI: 10.1182/blood-2011-10-387365</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1182%2Fblood-2011-10-387365" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=22262760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVOntb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2012&pages=2579-2589&author=L.+Santoauthor=T.+Hideshimaauthor=A.+L.+Kungauthor=J.+C.+Tsengauthor=D.+Tamangauthor=M.+Yangauthor=M.+Jarpeauthor=J.+H.+Van+Duzerauthor=R.+Mazitschekauthor=W.+C.+Ogierauthor=D.+Cirsteaauthor=S.+Rodigauthor=H.+Edaauthor=T.+Scullenauthor=M.+Canaveseauthor=J.+Bradnerauthor=K.+C.+Andersonauthor=S.+S.+Jonesauthor=N.+Raje&title=Preclinical+Activity%2C+Pharmacodynamic%2C+and+Pharmacokinetic+Properties+of+a+Selective+HDAC6+Inhibitor%2C+ACY-1215%2C+in+Combination+with+Bortezomib+in+Multiple+Myeloma&doi=10.1182%2Fblood-2011-10-387365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical activity, pharmacodynamic, and pharmacokinetic properties of a selective HDAC6 inhibitor, ACY-1215, in combination with bortezomib in multiple myeloma</span></div><div class="casAuthors">Santo, Loredana; Hideshima, Teru; Kung, Andrew L.; Tseng, Jen-Chieh; Tamang, David; Yang, Min; Jarpe, Matthew; van Duzer, John H.; Mazitschek, Ralph; Ogier, Walter C.; Cirstea, Diana; Rodig, Scott; Eda, Homare; Scullen, Tyler; Canavese, Miriam; Bradner, James; Anderson, Kenneth C.; Jones, Simon S.; Raje, Noopur</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2579-2589</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Histone deacetylase (HDAC) enzymic activity has been linked to the transcription of DNA in cancers including multiple myeloma (MM).  Therefore, HDAC inhibitors used alone and in combination are being actively studied as novel therapies in MM.  In the present study, we investigated the preclin. activity of ACY-1215, an HDAC6-selective inhibitor, alone and in combination with bortezomib in MM.  Low doses of ACY-1215 combined with bortezomib triggered synergistic anti-MM activity, resulting in protracted endoplasmic reticulum stress and apoptosis via activation of caspase-3, caspase-8, and caspase-9 and poly (ADP) ribosome polymerase.  In vivo, the anti-MM activity of ACY-1215 in combination with bortezomib was confirmed using 2 different xenograft SCID mouse models: human MM injected s.c. (the plasmacytoma model) and luciferase-expressing human MM injected i.v. (the disseminated MM model).  Tumor growth was significantly delayed and overall survival was significantly prolonged in animals treated with the combination therapy.  Pharmacokinetic data showed peak plasma levels of ACY-1215 at 4 h after treatment coincident with an increase in acetylated α-tubulin, a marker of HDAC6 inhibition, by immunohistochem. and Western blot anal.  These studies provide preclin. rationale for acetylated α-tubulin use as a pharmacodynamic biomarker in future clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoelmsk1wo_LVg90H21EOLACvtfcHk0lhwVhrURXOftA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVOntb4%253D&md5=094298bdd3d5b91c7060d8b294857846</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1182%2Fblood-2011-10-387365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2011-10-387365%26sid%3Dliteratum%253Aachs%26aulast%3DSanto%26aufirst%3DL.%26aulast%3DHideshima%26aufirst%3DT.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DTseng%26aufirst%3DJ.%2BC.%26aulast%3DTamang%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DJarpe%26aufirst%3DM.%26aulast%3DVan%2BDuzer%26aufirst%3DJ.%2BH.%26aulast%3DMazitschek%26aufirst%3DR.%26aulast%3DOgier%26aufirst%3DW.%2BC.%26aulast%3DCirstea%26aufirst%3DD.%26aulast%3DRodig%26aufirst%3DS.%26aulast%3DEda%26aufirst%3DH.%26aulast%3DScullen%26aufirst%3DT.%26aulast%3DCanavese%26aufirst%3DM.%26aulast%3DBradner%26aufirst%3DJ.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DJones%26aufirst%3DS.%2BS.%26aulast%3DRaje%26aufirst%3DN.%26atitle%3DPreclinical%2520Activity%252C%2520Pharmacodynamic%252C%2520and%2520Pharmacokinetic%2520Properties%2520of%2520a%2520Selective%2520HDAC6%2520Inhibitor%252C%2520ACY-1215%252C%2520in%2520Combination%2520with%2520Bortezomib%2520in%2520Multiple%2520Myeloma%26jtitle%3DBlood%26date%3D2012%26volume%3D119%26spage%3D2579%26epage%3D2589%26doi%3D10.1182%2Fblood-2011-10-387365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogl, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagannath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hari, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlowski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supko, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wheeler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markelewicz, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lonial, S.</span></span> <span> </span><span class="NLM_article-title">Ricolinostat, the First Selective Histone Deacetylase 6 Inhibitor, in Combination with Bortezomib and Dexamethasone for Relapsed or Refractory Multiple Myeloma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3307</span>– <span class="NLM_lpage">3315</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-16-2526</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1158%2F1078-0432.CCR-16-2526" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=28053023" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFSnurbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=3307-3315&author=D.+T.+Voglauthor=N.+Rajeauthor=S.+Jagannathauthor=P.+Richardsonauthor=P.+Hariauthor=R.+Orlowskiauthor=J.+G.+Supkoauthor=D.+Tamangauthor=M.+Yangauthor=S.+S.+Jonesauthor=C.+Wheelerauthor=R.+J.+Markelewiczauthor=S.+Lonial&title=Ricolinostat%2C+the+First+Selective+Histone+Deacetylase+6+Inhibitor%2C+in+Combination+with+Bortezomib+and+Dexamethasone+for+Relapsed+or+Refractory+Multiple+Myeloma&doi=10.1158%2F1078-0432.CCR-16-2526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Ricolinostat, the First Selective Histone Deacetylase 6 Inhibitor, in Combination with Bortezomib and Dexamethasone for Relapsed or Refractory Multiple Myeloma</span></div><div class="casAuthors">Vogl, Dan T.; Raje, Noopur; Jagannath, Sundar; Richardson, Paul; Hari, Parameswaran; Orlowski, Robert; Supko, Jeffrey G.; Tamang, David; Yang, Min; Jones, Simon S.; Wheeler, Catherine; Markelewicz, Robert J.; Lonial, Sagar</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3307-3315</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Histone deacetylase (HDAC) inhibition improves the efficacy of proteasome inhibition for multiple myeloma but adds substantial toxicity.  Preclin. models suggest that the obsd. synergy is due to the role of HDAC6 in mediating resistance to proteasome inhibition via the aggresome/autophagy pathway of protein degrdn.  Exptl. Design: We conducted a phase I/II trial of the HDAC6-selective inhibitor ricolinostat to define the safety, preliminary efficacy, and recommended phase II dose in combination with std. proteasome inhibitor therapy.  Patients with relapsed or refractory multiple myeloma received oral ricolinostat on days 1-5 and 8-12 of each 21-day cycle.  Results: Single-agent ricolinostat therapy resulted in neither significant toxicity nor clin. responses.  Combination therapy with bortezomib and dexamethasone was well-tolerated during dose escalation but led to dose-limiting diarrhea in an expansion cohort at a ricolinostat dose of 160 mg twice daily.  Combination therapy at a ricolinostat dose of 160 mg daily in a second expansion cohort was well tolerated, with less severe hematol., gastrointestinal, and constitutional toxicities compared with published data on nonselective HDAC inhibitors.  The overall response rate in combination with daily ricolinostat at ≥160 mg was 37%.  The response rate to combination therapy among bortezomib-refractory patients was 14%.  Samples taken during therapy showed dose-dependent increases of acetylated tubulin in peripheral blood lymphocytes.  Conclusions: At the recommended phase II dose of ricolinostat of 160 mg daily, the combination with bortezomib and dexamethasone is safe, well-tolerated, and active, suggesting that selective inhibition of HDAC6 is a promising approach to multiple myeloma therapy.  Clin Cancer Res; 23(13); 3307-15. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFQogphgLWbLVg90H21EOLACvtfcHk0liZu_GXkG1jwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFSnurbL&md5=2dd174501d2f331aeb0eff42072001d1</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-16-2526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-16-2526%26sid%3Dliteratum%253Aachs%26aulast%3DVogl%26aufirst%3DD.%2BT.%26aulast%3DRaje%26aufirst%3DN.%26aulast%3DJagannath%26aufirst%3DS.%26aulast%3DRichardson%26aufirst%3DP.%26aulast%3DHari%26aufirst%3DP.%26aulast%3DOrlowski%26aufirst%3DR.%26aulast%3DSupko%26aufirst%3DJ.%2BG.%26aulast%3DTamang%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DS.%2BS.%26aulast%3DWheeler%26aufirst%3DC.%26aulast%3DMarkelewicz%26aufirst%3DR.%2BJ.%26aulast%3DLonial%26aufirst%3DS.%26atitle%3DRicolinostat%252C%2520the%2520First%2520Selective%2520Histone%2520Deacetylase%25206%2520Inhibitor%252C%2520in%2520Combination%2520with%2520Bortezomib%2520and%2520Dexamethasone%2520for%2520Relapsed%2520or%2520Refractory%2520Multiple%2520Myeloma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D3307%26epage%3D3315%26doi%3D10.1158%2F1078-0432.CCR-16-2526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leonhardt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellmer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krämer, O. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dove, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahboobi, S.</span></span> <span> </span><span class="NLM_article-title">Design and Biological Evaluation of Tetrahydro-β-Carboline Derivatives as Highly Potent Histone Deacetylase 6 (HDAC6) Inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">357</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.04.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1016%2Fj.ejmech.2018.04.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=29738953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpt1agtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2018&pages=329-357&author=M.+Leonhardtauthor=A.+Sellmerauthor=O.+H.+Kr%C3%A4merauthor=S.+Doveauthor=S.+Elzauthor=B.+Krausauthor=M.+Beyerauthor=S.+Mahboobi&title=Design+and+Biological+Evaluation+of+Tetrahydro-%CE%B2-Carboline+Derivatives+as+Highly+Potent+Histone+Deacetylase+6+%28HDAC6%29+Inhibitors&doi=10.1016%2Fj.ejmech.2018.04.046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Design and biological evaluation of tetrahydro-β-carboline derivatives as highly potent histone deacetylase 6 (HDAC6) inhibitors</span></div><div class="casAuthors">Leonhardt, Michel; Sellmer, Andreas; Kraemer, Oliver H.; Dove, Stefan; Elz, Sigurd; Kraus, Birgit; Beyer, Mandy; Mahboobi, Siavosh</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">329-357</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Various diseases are related to epigenetic modifications.  Histone deacetylases (HDACs) and histone acetyltransferases (HATs) det. the pattern of histone acetylation, and thus are involved in the regulation of gene expression.  First generation histone deacetylase inhibitors (HDACi) are unselective, hinder all different kinds of zinc dependent HDACs and addnl. cause several side effects.  Subsequently, selective HDACi are gaining more and more interest.  Esp., selective histone deacetylase 6 inhibitors (HDAC6i) are supposed to be less toxic.  Here the authors present a successful optimization study of tubastatin A, the synthesis and biol. evaluation of new inhibitors based on hydroxamic acids linked to various tetrahydro-β-carboline derivs.  The potency of the authors' selective HDAC6 inhibitors, exhibiting IC50 values in a range of 1-10 nM towards HDAC6, was evaluated with the help of a recombinant human HDAC6 enzyme assay.  Selectivity was proofed in cellular assays by the hyperacetylation of surrogate parameter α-tubulin in the absence of acetylated histone H3 analyzed by Western Blot.  The authors show that all synthesized compds., with varies modifications of the rigid cap group, were selective and potent HDAC6 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO3jAwqO8N0LVg90H21EOLACvtfcHk0liZu_GXkG1jwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpt1agtbY%253D&md5=668b7ae179e680a0ae7116589e315cb5</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.04.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.04.046%26sid%3Dliteratum%253Aachs%26aulast%3DLeonhardt%26aufirst%3DM.%26aulast%3DSellmer%26aufirst%3DA.%26aulast%3DKr%25C3%25A4mer%26aufirst%3DO.%2BH.%26aulast%3DDove%26aufirst%3DS.%26aulast%3DElz%26aufirst%3DS.%26aulast%3DKraus%26aufirst%3DB.%26aulast%3DBeyer%26aufirst%3DM.%26aulast%3DMahboobi%26aufirst%3DS.%26atitle%3DDesign%2520and%2520Biological%2520Evaluation%2520of%2520Tetrahydro-%25CE%25B2-Carboline%2520Derivatives%2520as%2520Highly%2520Potent%2520Histone%2520Deacetylase%25206%2520%2528HDAC6%2529%2520Inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D152%26spage%3D329%26epage%3D357%26doi%3D10.1016%2Fj.ejmech.2018.04.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sellmer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stangl, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grünstein, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonhardt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pongratz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichhorn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Striegl, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenei-Lanzl, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dove, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Straub, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krämer, O. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahboobi, S.</span></span> <span> </span><span class="NLM_article-title">Marbostat-100 Defines a New Class of Potent and Selective Antiinflammatory and Antirheumatic Histone Deacetylase 6 Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">3454</span>– <span class="NLM_lpage">3477</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01593</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01593" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmtFCnsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=3454-3477&author=A.+Sellmerauthor=H.+Stanglauthor=M.+Beyerauthor=E.+Gr%C3%BCnsteinauthor=M.+Leonhardtauthor=H.+Pongratzauthor=E.+Eichhornauthor=S.+Elzauthor=B.+Strieglauthor=Z.+Jenei-Lanzlauthor=S.+Doveauthor=R.+H.+Straubauthor=O.+H.+Kr%C3%A4merauthor=S.+Mahboobi&title=Marbostat-100+Defines+a+New+Class+of+Potent+and+Selective+Antiinflammatory+and+Antirheumatic+Histone+Deacetylase+6+Inhibitors&doi=10.1021%2Facs.jmedchem.7b01593"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Marbostat-100 Defines a New Class of Potent and Selective Antiinflammatory and Antirheumatic Histone Deacetylase 6 Inhibitors</span></div><div class="casAuthors">Sellmer, Andreas; Stangl, Hubert; Beyer, Mandy; Gruenstein, Elisabeth; Leonhardt, Michel; Pongratz, Herwig; Eichhorn, Emerich; Elz, Sigurd; Striegl, Birgit; Jenei-Lanzl, Zsuzsa; Dove, Stefan; Straub, Rainer H.; Kraemer, Oliver H.; Mahboobi, Siavosh</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3454-3477</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic modifiers of the histone deacetylase (HDAC) family contribute to autoimmunity, cancer, HIV infection, inflammation, and neurodegeneration.  Hence, histone deacetylase inhibitors (HDACi), which alter protein acetylation, gene expression patterns, and cell fate decisions, represent promising new drugs for the therapy of these diseases.  Whereas pan-HDACi inhibit all 11 Zn2+-dependent histone deacetylases (HDACs) and cause a broad spectrum of side effects, specific inhibitors of histone deacetylase 6 (HDAC6i) are supposed to have less side effects.  The authors present the synthesis and biol. evaluation of Marbostats, novel HDAC6i that contain the hydroxamic acid moiety linked to tetrahydro-β-carboline derivs.  The lead compd. Marbostat-100 is a more potent and more selective HDAC6i than previously established well-characterized compds. in vitro as well as in cells.  Moreover, Marbostat-100 is well tolerated by mice and effective against collagen type II induced arthritis.  Thus, Marbostat-100 represents a most selective known HDAC6i and the possibility for clin. evaluation of a HDAC isoform-specific drug.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqixUQJJP-7bVg90H21EOLACvtfcHk0liZu_GXkG1jwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmtFCnsbg%253D&md5=0f6c62f03f50501728d43fef66d0d64b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01593&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01593%26sid%3Dliteratum%253Aachs%26aulast%3DSellmer%26aufirst%3DA.%26aulast%3DStangl%26aufirst%3DH.%26aulast%3DBeyer%26aufirst%3DM.%26aulast%3DGr%25C3%25BCnstein%26aufirst%3DE.%26aulast%3DLeonhardt%26aufirst%3DM.%26aulast%3DPongratz%26aufirst%3DH.%26aulast%3DEichhorn%26aufirst%3DE.%26aulast%3DElz%26aufirst%3DS.%26aulast%3DStriegl%26aufirst%3DB.%26aulast%3DJenei-Lanzl%26aufirst%3DZ.%26aulast%3DDove%26aufirst%3DS.%26aulast%3DStraub%26aufirst%3DR.%2BH.%26aulast%3DKr%25C3%25A4mer%26aufirst%3DO.%2BH.%26aulast%3DMahboobi%26aufirst%3DS.%26atitle%3DMarbostat-100%2520Defines%2520a%2520New%2520Class%2520of%2520Potent%2520and%2520Selective%2520Antiinflammatory%2520and%2520Antirheumatic%2520Histone%2520Deacetylase%25206%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D3454%26epage%3D3477%26doi%3D10.1021%2Facs.jmedchem.7b01593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cubizolles, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousseaux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kneissel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chua, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizeracki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthias, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alt, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khochbin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthias, P.</span></span> <span> </span><span class="NLM_article-title">Mice Lacking Histone Deacetylase 6 Have Hyperacetylated Tubulin but Are Viable and Develop Normally</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1688</span>– <span class="NLM_lpage">1701</span>, <span class="refDoi"> DOI: 10.1128/MCB.01154-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1128%2FMCB.01154-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=18180281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD1cXivVajtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=1688-1701&author=Y.+Zhangauthor=S.+Kwonauthor=T.+Yamaguchiauthor=F.+Cubizollesauthor=S.+Rousseauxauthor=M.+Kneisselauthor=C.+Caoauthor=N.+Liauthor=H.-L.+Chengauthor=K.+Chuaauthor=D.+Lombardauthor=A.+Mizerackiauthor=G.+Matthiasauthor=F.+W.+Altauthor=S.+Khochbinauthor=P.+Matthias&title=Mice+Lacking+Histone+Deacetylase+6+Have+Hyperacetylated+Tubulin+but+Are+Viable+and+Develop+Normally&doi=10.1128%2FMCB.01154-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Mice lacking histone deacetylase 6 have hyperacetylated tubulin but are viable and develop normally</span></div><div class="casAuthors">Zhang, Yu; Kwon, SoHee; Yamaguchi, Teppei; Cubizolles, Fabien; Rousseaux, Sophie; Kneissel, Michaela; Cao, Chun; Li, Na; Cheng, Hwei-Ling; Chua, Katrin; Lombard, David; Mizeracki, Adam; Matthias, Gabriele; Alt, Frederick W.; Khochbin, Saadi; Matthias, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1688-1701</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Posttranslational modifications play important roles in regulating protein structure and function.  Histone deacetylase 6 (HDAC6) is a mostly cytoplasmic class II HDAC, which has a unique structure with two catalytic domains and a domain binding ubiquitin with high affinity.  This enzyme was recently identified as a multisubstrate protein deacetylase that can act on acetylated histone tails, α-tubulin and Hsp90.  To investigate the in vivo functions of HDAC6 and the relevance of tubulin acetylation/deacetylation, we targeted the HDAC6 gene by homologous recombination in embryonic stem cells and generated knockout mice.  HDAC6-deficient mice are viable and fertile and show hyperacetylated tubulin in most tissues.  The highest level of expression of HDAC6 is seen in the testis, yet development and function of this organ are normal in the absence of HDAC6.  Likewise, lymphoid development is normal, but the immune response is moderately affected.  Furthermore, the lack of HDAC6 results in a small increase in cancellous bone mineral d., indicating that this deacetylase plays a minor role in bone biol.  HDAC6-deficient mouse embryonic fibroblasts show apparently normal microtubule organization and stability and also show increased Hsp90 acetylation correlating with impaired Hsp90 function.  Collectively, these data demonstrate that mice survive well without HDAC6 and that tubulin hyperacetylation is not detrimental to normal mammalian development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1on7_hqKefrVg90H21EOLACvtfcHk0lic52ybL3jImw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXivVajtLo%253D&md5=9c83b56ee31ea67a04357b31467ed655</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1128%2FMCB.01154-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01154-06%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DKwon%26aufirst%3DS.%26aulast%3DYamaguchi%26aufirst%3DT.%26aulast%3DCubizolles%26aufirst%3DF.%26aulast%3DRousseaux%26aufirst%3DS.%26aulast%3DKneissel%26aufirst%3DM.%26aulast%3DCao%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DN.%26aulast%3DCheng%26aufirst%3DH.-L.%26aulast%3DChua%26aufirst%3DK.%26aulast%3DLombard%26aufirst%3DD.%26aulast%3DMizeracki%26aufirst%3DA.%26aulast%3DMatthias%26aufirst%3DG.%26aulast%3DAlt%26aufirst%3DF.%2BW.%26aulast%3DKhochbin%26aufirst%3DS.%26aulast%3DMatthias%26aufirst%3DP.%26atitle%3DMice%2520Lacking%2520Histone%2520Deacetylase%25206%2520Have%2520Hyperacetylated%2520Tubulin%2520but%2520Are%2520Viable%2520and%2520Develop%2520Normally%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2008%26volume%3D28%26spage%3D1688%26epage%3D1701%26doi%3D10.1128%2FMCB.01154-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hideshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cottini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakubikova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorgun, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mimura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tai, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span> <span> </span><span class="NLM_article-title">Rational Combination Treatment with Histone Deacetylase Inhibitors and Immunomodulatory Drugs in Multiple Myeloma</span>. <i>Blood Cancer J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">e312</span>, <span class="refDoi"> DOI: 10.1038/bcj.2015.38</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fbcj.2015.38" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=25978432" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=e312&author=T.+Hideshimaauthor=F.+Cottiniauthor=H.+Ohguchiauthor=J.+Jakubikovaauthor=G.+Gorgunauthor=N.+Mimuraauthor=Y.+T.+Taiauthor=N.+C.+Munshiauthor=P.+G.+Richardsonauthor=K.+C.+Anderson&title=Rational+Combination+Treatment+with+Histone+Deacetylase+Inhibitors+and+Immunomodulatory+Drugs+in+Multiple+Myeloma&doi=10.1038%2Fbcj.2015.38"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fbcj.2015.38&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbcj.2015.38%26sid%3Dliteratum%253Aachs%26aulast%3DHideshima%26aufirst%3DT.%26aulast%3DCottini%26aufirst%3DF.%26aulast%3DOhguchi%26aufirst%3DH.%26aulast%3DJakubikova%26aufirst%3DJ.%26aulast%3DGorgun%26aufirst%3DG.%26aulast%3DMimura%26aufirst%3DN.%26aulast%3DTai%26aufirst%3DY.%2BT.%26aulast%3DMunshi%26aufirst%3DN.%2BC.%26aulast%3DRichardson%26aufirst%3DP.%2BG.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26atitle%3DRational%2520Combination%2520Treatment%2520with%2520Histone%2520Deacetylase%2520Inhibitors%2520and%2520Immunomodulatory%2520Drugs%2520in%2520Multiple%2520Myeloma%26jtitle%3DBlood%2520Cancer%2520J.%26date%3D2015%26volume%3D5%26spage%3De312%26doi%3D10.1038%2Fbcj.2015.38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heider, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzler, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Von Kaiser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosche, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rötzer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rademacher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakob, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleissner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuckelkorn, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kloetzel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sezer, O.</span></span> <span> </span><span class="NLM_article-title">Synergistic Interaction of the Histone Deacetylase Inhibitor SAHA with the Proteasome Inhibitor Bortezomib in Mantle Cell Lymphoma</span>. <i>Eur. J. Haematol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">142</span>, <span class="refDoi"> DOI: 10.1111/j.1600-0609.2007.00995.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1111%2Fj.1600-0609.2007.00995.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=18005386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD1cXivFahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2008&pages=133-142&author=U.+Heiderauthor=I.+Metzlerauthor=M.+Von+Kaiserauthor=M.+Roscheauthor=J.+Sterzauthor=S.+R%C3%B6tzerauthor=J.+Rademacherauthor=C.+Jakobauthor=C.+Fleissnerauthor=U.+Kuckelkornauthor=P.+Kloetzelauthor=O.+Sezer&title=Synergistic+Interaction+of+the+Histone+Deacetylase+Inhibitor+SAHA+with+the+Proteasome+Inhibitor+Bortezomib+in+Mantle+Cell+Lymphoma&doi=10.1111%2Fj.1600-0609.2007.00995.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Synergistic interaction of the histone deacetylase inhibitor SAHA with the proteasome inhibitor bortezomib in mantle cell lymphoma</span></div><div class="casAuthors">Heider, Ulrike; von Metzler, Ivana; Kaiser, Martin; Rosche, Marleen; Sterz, Jan; Roetzer, Susanne; Rademacher, Jessica; Jakob, Christian; Fleissner, Claudia; Kuckelkorn, Ulrike; Kloetzel, Peter-Michael; Sezer, Orhan</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Haematology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-142</span>CODEN:
                <span class="NLM_cas:coden">EJHAEC</span>;
        ISSN:<span class="NLM_cas:issn">0902-4441</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Mantle cell lymphoma (MCL) is an incurable B cell lymphoma, and novel treatment strategies are urgently needed.  We evaluated the effects of combined treatment with the proteasome inhibitor bortezomib and the histone deacetylase inhibitor (HDACi) suberoylanilide hydroxamic acid (SAHA) on MCL.  Bortezomib acts by targeting the proteasome, and - among other mechanisms - results in a reduced nuclear factor-kappa B (NF-κB) activity.  HDACi promote histone acetylation, and also interfere with NF-κB signaling.  Human MCL cell lines (JeKo-1, Granta-519 and Hbl-2) were exposed to bortezomib and/or SAHA.  Cell viability and apoptosis were quantified by the MTT and annexin-V assay, resp.  Reactive oxygen species (ROS) were analyzed using the fluorophore H2DCFDA.  In addn., activated caspases, proteasome- and NF-κB activity were quantified.  Combined incubation with bortezomib and SAHA resulted in synergistic cytotoxic effects, as indicated by combination index values <1 using the median effect method of Chou and Talalay.  The combination of both inhibitors led to a strong increase in apoptosis as compared to single agents and was accompanied by enhanced ROS generation, while each agent alone only modestly induced ROS.  The free radical scavenger N-acetyl-l-cysteine blocked the ROS generation and reduced the apoptosis significantly.  In addn., coexposure of bortezomib and SAHA led to increased caspase-3, -8, and -9 activity, marked redn. of proteasome activity and decrease of NF-κB activity.  This is the first report giving evidence that SAHA and bortezomib synergistically induce apoptosis in MCL cells.  These data build the framework for clin. trials using combined proteasome and histone deacetylase inhibition in the treatment of MCL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopNeVtuIe4nrVg90H21EOLACvtfcHk0lic52ybL3jImw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXivFahsbg%253D&md5=2e251f33843ecad588e8aa8ca20f4984</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1111%2Fj.1600-0609.2007.00995.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1600-0609.2007.00995.x%26sid%3Dliteratum%253Aachs%26aulast%3DHeider%26aufirst%3DU.%26aulast%3DMetzler%26aufirst%3DI.%26aulast%3DVon%2BKaiser%26aufirst%3DM.%26aulast%3DRosche%26aufirst%3DM.%26aulast%3DSterz%26aufirst%3DJ.%26aulast%3DR%25C3%25B6tzer%26aufirst%3DS.%26aulast%3DRademacher%26aufirst%3DJ.%26aulast%3DJakob%26aufirst%3DC.%26aulast%3DFleissner%26aufirst%3DC.%26aulast%3DKuckelkorn%26aufirst%3DU.%26aulast%3DKloetzel%26aufirst%3DP.%26aulast%3DSezer%26aufirst%3DO.%26atitle%3DSynergistic%2520Interaction%2520of%2520the%2520Histone%2520Deacetylase%2520Inhibitor%2520SAHA%2520with%2520the%2520Proteasome%2520Inhibitor%2520Bortezomib%2520in%2520Mantle%2520Cell%2520Lymphoma%26jtitle%3DEur.%2520J.%2520Haematol.%26date%3D2008%26volume%3D80%26spage%3D133%26epage%3D142%26doi%3D10.1111%2Fj.1600-0609.2007.00995.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yee, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bensinger, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supko, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voorhees, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berdeja, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Libby, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birrer, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamang, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wheeler, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markelewicz, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, N. S.</span></span> <span> </span><span class="NLM_article-title">Ricolinostat plus Lenalidomide, and Dexamethasone in Relapsed or Refractory Multiple Myeloma: A Multicentre Phase 1b Trial</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1569</span>– <span class="NLM_lpage">1578</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(16)30375-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1016%2FS1470-2045%2816%2930375-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=27646843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhsl2isb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=1569-1578&author=A.+J.+Yeeauthor=W.+I.+Bensingerauthor=J.+G.+Supkoauthor=P.+M.+Voorheesauthor=J.+G.+Berdejaauthor=P.+G.+Richardsonauthor=E.+N.+Libbyauthor=E.+E.+Wallaceauthor=N.+E.+Birrerauthor=J.+N.+Burkeauthor=D.+L.+Tamangauthor=M.+Yangauthor=S.+S.+Jonesauthor=C.+A.+Wheelerauthor=R.+J.+Markelewiczauthor=N.+S.+Raje&title=Ricolinostat+plus+Lenalidomide%2C+and+Dexamethasone+in+Relapsed+or+Refractory+Multiple+Myeloma%3A+A+Multicentre+Phase+1b+Trial&doi=10.1016%2FS1470-2045%2816%2930375-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Ricolinostat plus lenalidomide, and dexamethasone in relapsed or refractory multiple myeloma: a multicentre phase 1b trial</span></div><div class="casAuthors">Yee, Andrew J.; Bensinger, William I.; Supko, Jeffrey G.; Voorhees, Peter M.; Berdeja, Jesus G.; Richardson, Paul G.; Libby, Edward N.; Wallace, Ellen E.; Birrer, Nicole E.; Burke, Jill N.; Tamang, David L.; Yang, Min; Jones, Simon S.; Wheeler, Catherine A.; Markelewicz, Robert J.; Raje, Noopur S.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1569-1578</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Histone deacetylase (HDAC) inhibitors are an important new class of therapeutics for treating multiple myeloma.  Ricolinostat (ACY-1215) is the first oral selective HDAC6 inhibitor with reduced class I HDAC activity to be studied clin.  Motivated by findings from preclin. studies showing potent synergistic activity with ricolinostat and lenalidomide, our goal was to assess the safety and preliminary activity of the combination of ricolinostat with lenalidomide and dexamethasone in relapsed or refractory multiple myeloma.  In this multicenter phase 1b trial, we recruited patients aged 18 years or older with previously treated relapsed or refractory multiple myeloma from five cancer centers in the USA.  Inclusion criteria included a Karnofsky Performance Status score of at least 70, measureable disease, adequate bone marrow reserve, adequate hepatic function, and a creatinine clearance of at least 50 mL per min.  Exclusion criteria included previous exposure to HDAC inhibitors; previous allogeneic stem-cell transplantation; previous autologous stem-cell transplantation within 12 wk of baseline; active systemic infection; malignancy within the last 5 years; known or suspected HIV, hepatitis B, or hepatitis C infection; a QTc Fridericia of more than 480 ms; and substantial cardiovascular, gastrointestinal, psychiatric, or other medical disorders.  We gave escalating doses (from 40-240 mg once daily to 160 mg twice daily) of oral ricolinostat according to a std. 3 + 3 design according to three different regimens on days 1-21 with a conventional 28 day schedule of oral lenalidomide (from 15 mg [in one cohort] to 25 mg [in all other cohorts] once daily) and oral dexamethasone (40 mg weekly).  Primary outcomes were dose-limiting toxicities, the max. tolerated dose of ricolinostat in this combination, and the dose and schedule of ricolinostat recommended for further phase 2 investigation.  Secondary outcomes were the pharmacokinetics and pharmacodynamics of ricolinostat in this combination and the preliminary anti-tumor activity of this treatment.  The trial is closed to accrual and is registered at ClinicalTrials.gov, no. NCT01583283.  Between July 12, 2012, and Aug 20, 2015, we enrolled 38 patients.  We obsd. two dose-limiting toxicities with ricolinostat 160 mg twice daily: one (2%) grade 3 syncope and one (2%) grade 3 myalgia event in different cohorts.  A max. tolerated dose was not reached.  We chose ricolinostat 160 mg once daily on days 1-21 of a 28 day cycle as the recommended dose for future phase 2 studies in combination with lenalidomide 25 mg and dexamethasone 40 mg.  The most common adverse events were fatigue (grade 1-2 in 14 [37%] patients; grade 3 in seven [18%]) and diarrhea (grade 1-2 in 15 [39%] patients; grade 3 in two [5%]).  Our pharmacodynamic studies showed that at clin. relevant doses, ricolinostat selectively inhibits HDAC6 while retaining a low and tolerable level of class I HDAC inhibition.  The pharmacokinetics of ricolinostat and lenalidomide were not affected by co-administration.  In a preliminary assessment of antitumor activity, 21 (55% [95% CI 38-71]) of 38 patients had an overall response.  The findings from this study provide preliminary evidence that ricolinostat is a safe and well tolerated selective HDAC6 inhibitor, which might partner well with lenalidomide and dexamethasone to enhance their efficacy in relapsed or refractory multiple myeloma.Acetylon Pharmaceuticals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWXWsO-vNKY7Vg90H21EOLACvtfcHk0lg87bhOW4ewRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhsl2isb7J&md5=59fea63c11b7bce7be6bd74f195af04b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2816%2930375-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252816%252930375-8%26sid%3Dliteratum%253Aachs%26aulast%3DYee%26aufirst%3DA.%2BJ.%26aulast%3DBensinger%26aufirst%3DW.%2BI.%26aulast%3DSupko%26aufirst%3DJ.%2BG.%26aulast%3DVoorhees%26aufirst%3DP.%2BM.%26aulast%3DBerdeja%26aufirst%3DJ.%2BG.%26aulast%3DRichardson%26aufirst%3DP.%2BG.%26aulast%3DLibby%26aufirst%3DE.%2BN.%26aulast%3DWallace%26aufirst%3DE.%2BE.%26aulast%3DBirrer%26aufirst%3DN.%2BE.%26aulast%3DBurke%26aufirst%3DJ.%2BN.%26aulast%3DTamang%26aufirst%3DD.%2BL.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DS.%2BS.%26aulast%3DWheeler%26aufirst%3DC.%2BA.%26aulast%3DMarkelewicz%26aufirst%3DR.%2BJ.%26aulast%3DRaje%26aufirst%3DN.%2BS.%26atitle%3DRicolinostat%2520plus%2520Lenalidomide%252C%2520and%2520Dexamethasone%2520in%2520Relapsed%2520or%2520Refractory%2520Multiple%2520Myeloma%253A%2520A%2520Multicentre%2520Phase%25201b%2520Trial%26jtitle%3DLancet%2520Oncol.%26date%3D2016%26volume%3D17%26spage%3D1569%26epage%3D1578%26doi%3D10.1016%2FS1470-2045%2816%2930375-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khoury, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Washington, L. B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keating, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D.</span></span> <span> </span><span class="NLM_article-title">A Systematic Approach to Diagnosis of Mature T-Cell Leukemias Reveals Heterogeneity among WHO Categories</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">328</span>– <span class="NLM_lpage">335</span>, <span class="refDoi"> DOI: 10.1182/blood-2004-01-0002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1182%2Fblood-2004-01-0002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=15044256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvFSnsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2004&pages=328-335&author=M.+Herlingauthor=J.+D.+Khouryauthor=L.+B.+T.+Washingtonauthor=M.+Duvicauthor=M.+J.+Keatingauthor=D.+Jones&title=A+Systematic+Approach+to+Diagnosis+of+Mature+T-Cell+Leukemias+Reveals+Heterogeneity+among+WHO+Categories&doi=10.1182%2Fblood-2004-01-0002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A systematic approach to diagnosis of mature T-cell leukemias reveals heterogeneity among WHO categories</span></div><div class="casAuthors">Herling, Marco; Khoury, Joseph D.; Washington, LaBaron T.; Duvic, Madeleine; Keating, Michael J.; Jones, Dan</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">328-335</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The current World Health Organization (WHO) classification of hematopoietic malignancies defines several types of mature T-cell leukemia including T-cell prolymphocytic leukemia (T-PLL), Sezary syndrome (SS), and T-cell large granular lymphocytic (T-LGL) leukemia.  These neoplasms can show overlapping features with each other and with T-cell lymphomas involving peripheral blood (PB).  We analyzed the spectrum of clinicopathol. features in 102 mature T-cell leukemias and compared them to 10 hepatosplenic T-cell lymphomas that involved PB.  T-PLL, defined as a T-cell leukemia showing rapidly rising PB lymphocyte counts, was the only tumor type expressing the oncoprotein TCL1 (71% of cases) and could present with relatively low lymphocyte levels or small tumor cell morphol.  SS, defined by accompanying erythrodermic skin disease, was frequently assocd. with peripheral eosinophilia but could also develop high nos. of prolymphocytes, esp. late in the disease course.  T-LGL leukemia, defined by accompanying cytopenias or autoimmune phenomena (or both), had the best clin. outcome and generally showed the lowest circulating lymphocyte levels with only a few cases developing marked lymphocytosis.  Using the dominant clin. or phenotypic feature, we describe here the degree of overlap among currently recognized WHO categories and identify areas where further clarification is needed.  Our results indicate that incorporation of addnl. criteria, such as TCL1 expression status and hematol. parameters, can assist in a more accurate classification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS0GwbkDkfirVg90H21EOLACvtfcHk0lg87bhOW4ewRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvFSnsbc%253D&md5=7fa603b6f7fb2cf31c14a978107ab173</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1182%2Fblood-2004-01-0002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2004-01-0002%26sid%3Dliteratum%253Aachs%26aulast%3DHerling%26aufirst%3DM.%26aulast%3DKhoury%26aufirst%3DJ.%2BD.%26aulast%3DWashington%26aufirst%3DL.%2BB.%2BT.%26aulast%3DDuvic%26aufirst%3DM.%26aulast%3DKeating%26aufirst%3DM.%2BJ.%26aulast%3DJones%26aufirst%3DD.%26atitle%3DA%2520Systematic%2520Approach%2520to%2520Diagnosis%2520of%2520Mature%2520T-Cell%2520Leukemias%2520Reveals%2520Heterogeneity%2520among%2520WHO%2520Categories%26jtitle%3DBlood%26date%3D2004%26volume%3D104%26spage%3D328%26epage%3D335%26doi%3D10.1182%2Fblood-2004-01-0002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stengel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zenger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perglerová, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnittger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haferlach, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haferlach, C.</span></span> <span> </span><span class="NLM_article-title">Genetic Characterization of T-PLL Reveals Two Major Biologic Subgroups and <i>JAK3</i> Mutations as Prognostic Marker</span>. <i>Genes, Chromosomes Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">82</span>– <span class="NLM_lpage">94</span>, <span class="refDoi"> DOI: 10.1002/gcc.22313</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1002%2Fgcc.22313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=26493028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs12qtL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=82-94&author=A.+Stengelauthor=W.+Kernauthor=M.+Zengerauthor=K.+Perglerov%C3%A1author=S.+Schnittgerauthor=T.+Haferlachauthor=C.+Haferlach&title=Genetic+Characterization+of+T-PLL+Reveals+Two+Major+Biologic+Subgroups+and+JAK3+Mutations+as+Prognostic+Marker&doi=10.1002%2Fgcc.22313"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic characterization of T-PLL reveals two major biologic subgroups and JAK3 mutations as prognostic marker</span></div><div class="casAuthors">Stengel, Anna; Kern, Wolfgang; Zenger, Melanie; Perglerova, Karolina; Schnittger, Susanne; Haferlach, Torsten; Haferlach, Claudia</div><div class="citationInfo"><span class="NLM_cas:title">Genes, Chromosomes & Cancer</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">82-94</span>CODEN:
                <span class="NLM_cas:coden">GCCAES</span>;
        ISSN:<span class="NLM_cas:issn">1045-2257</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">T-cell prolymphocytic leukemia (T-PLL) is a rare post-thymic T-cell neoplasm with aggressive clin. course and short overall survival.  So far, due to the rareness of this disease, genetic data are available only from individual cases or small cohorts.  In our study, we aimed at performing a comprehensive cytogenetic and mol. genetic characterization of T-PLL comprising the largest cohort of patients with T-PLL analyzed so far, including correlations between the resp. markers and their impact on prognosis.  Genetic abnormalities were found in all 51 cases with T-PLL, most frequently involving the TCRA/D locus (86%).  Deletions were detected for ATM (69%) and TP53 (31%), whereas i(8)(q10) was obsd. in 61% of cases.  Mutations in ATM, TP53, JAK1, and JAK3 were detected in 73, 14, 6, and 21% of patients, resp.  Addnl., BCOR mutations were obsd. for the first time in a lymphoid malignancy (8%).  Two distinct genetic subgroups of T-PLL were identified: A large subset (86% of patients) showed abnormalities involving the TCRA/D locus activating the proto-oncogenes TCL1 or MTCP1, while the second group was characterized by a high frequency of TP53 mutations (4/7 cases).  Further, analyses of overall survival identified JAK3 mutations as important prognostic marker, showing a significant neg. impact. © 2015 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ohuCRXjR9LVg90H21EOLACvtfcHk0lg87bhOW4ewRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs12qtL7F&md5=082cc0079a3832dc732c4350c0fc4b33</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fgcc.22313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fgcc.22313%26sid%3Dliteratum%253Aachs%26aulast%3DStengel%26aufirst%3DA.%26aulast%3DKern%26aufirst%3DW.%26aulast%3DZenger%26aufirst%3DM.%26aulast%3DPerglerov%25C3%25A1%26aufirst%3DK.%26aulast%3DSchnittger%26aufirst%3DS.%26aulast%3DHaferlach%26aufirst%3DT.%26aulast%3DHaferlach%26aufirst%3DC.%26atitle%3DGenetic%2520Characterization%2520of%2520T-PLL%2520Reveals%2520Two%2520Major%2520Biologic%2520Subgroups%2520and%2520JAK3%2520Mutations%2520as%2520Prognostic%2520Marker%26jtitle%3DGenes%252C%2520Chromosomes%2520Cancer%26date%3D2016%26volume%3D55%26spage%3D82%26epage%3D94%26doi%3D10.1002%2Fgcc.22313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crispatzu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oberbeck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pützer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Von Jan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasyutina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weit, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pflug, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riabinska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, M. S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altmuller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanasa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haferlach, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stilgenbauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopfinger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peifer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brummendorf, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurnberg, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elenitoba-Johnson, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriggl, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinhardt, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, m. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustjoki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newrzela, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frommolt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span> <span> </span><span class="NLM_article-title">Actionable Perturbations of Damage Responses by TCL1/ATM and Epigenetic Lesions Form the Basis of T-PLL</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">22</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-02688-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fs41467-017-02688-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=29317637" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVehtLnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1-22&author=A.+Schraderauthor=G.+Crispatzuauthor=S.+Oberbeckauthor=P.+Mayerauthor=S.+P%C3%BCtzerauthor=J.+Von+Janauthor=E.+Vasyutinaauthor=K.+Warnerauthor=N.+Weitauthor=N.+Pflugauthor=T.+Braunauthor=E.+I.+Anderssonauthor=B.+Yadavauthor=A.+Riabinskaauthor=M.+S.+V.+Ferreiraauthor=F.+Beierauthor=J.+Altmullerauthor=M.+Lanasaauthor=C.+D.+Herlingauthor=T.+Haferlachauthor=S.+Stilgenbauerauthor=G.+Hopfingerauthor=M.+Peiferauthor=T.+H.+Brummendorfauthor=P.+Nurnbergauthor=K.+S.+Elenitoba-Johnsonauthor=S.+Zhaauthor=M.+Hallekauthor=R.+Morigglauthor=H.+C.+Reinhardtauthor=m.+H.+Sternauthor=S.+Mustjokiauthor=S.+Newrzelaauthor=P.+Frommoltauthor=M.+Herling&title=Actionable+Perturbations+of+Damage+Responses+by+TCL1%2FATM+and+Epigenetic+Lesions+Form+the+Basis+of+T-PLL&doi=10.1038%2Fs41467-017-02688-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A sustained change in the supply of parental care causes adaptive evolution of offspring morphology</span></div><div class="casAuthors">Jarrett, Benjamin J. M.; Evans, Emma; Haynes, Hannah B.; Leaf, Miranda R.; Rebar, Darren; Duarte, Ana; Schrader, Matthew; Kilner, Rebecca M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-8</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Although cooperative social interactions within species are considered an important driver of evolutionary change, few studies have exptl. demonstrated that they cause adaptive evolution.  Here we address this problem by studying the burying beetle Nicrophorus vespilloides.  In this species, parents and larvae work together to obtain nourishment for larvae from the carrion breeding resource: parents feed larvae and larvae also self-feed.  We established exptl. evolving populations in which we varied the assistance that parents provided for their offspring and investigated how offspring evolved in response.  We show that in populations where parents predictably supplied more care, larval mandibles evolved to be smaller in relation to larval mass, and larvae were correspondingly less self-sufficient.  Previous work has shown that antagonistic social interactions can generate escalating evolutionary arms races.  Our study shows that cooperative interactions can yield the opposite evolutionary outcome: when one party invests more, the other evolves to invest less.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbNTa4adWXq7Vg90H21EOLACvtfcHk0lhHZO4dG_quvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVehtLnI&md5=83d8d47cb89b14cb513c9d0f8a1039bb</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-02688-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-02688-6%26sid%3Dliteratum%253Aachs%26aulast%3DSchrader%26aufirst%3DA.%26aulast%3DCrispatzu%26aufirst%3DG.%26aulast%3DOberbeck%26aufirst%3DS.%26aulast%3DMayer%26aufirst%3DP.%26aulast%3DP%25C3%25BCtzer%26aufirst%3DS.%26aulast%3DVon%2BJan%26aufirst%3DJ.%26aulast%3DVasyutina%26aufirst%3DE.%26aulast%3DWarner%26aufirst%3DK.%26aulast%3DWeit%26aufirst%3DN.%26aulast%3DPflug%26aufirst%3DN.%26aulast%3DBraun%26aufirst%3DT.%26aulast%3DAndersson%26aufirst%3DE.%2BI.%26aulast%3DYadav%26aufirst%3DB.%26aulast%3DRiabinska%26aufirst%3DA.%26aulast%3DFerreira%26aufirst%3DM.%2BS.%2BV.%26aulast%3DBeier%26aufirst%3DF.%26aulast%3DAltmuller%26aufirst%3DJ.%26aulast%3DLanasa%26aufirst%3DM.%26aulast%3DHerling%26aufirst%3DC.%2BD.%26aulast%3DHaferlach%26aufirst%3DT.%26aulast%3DStilgenbauer%26aufirst%3DS.%26aulast%3DHopfinger%26aufirst%3DG.%26aulast%3DPeifer%26aufirst%3DM.%26aulast%3DBrummendorf%26aufirst%3DT.%2BH.%26aulast%3DNurnberg%26aufirst%3DP.%26aulast%3DElenitoba-Johnson%26aufirst%3DK.%2BS.%26aulast%3DZha%26aufirst%3DS.%26aulast%3DHallek%26aufirst%3DM.%26aulast%3DMoriggl%26aufirst%3DR.%26aulast%3DReinhardt%26aufirst%3DH.%2BC.%26aulast%3DStern%26aufirst%3Dm.%2BH.%26aulast%3DMustjoki%26aufirst%3DS.%26aulast%3DNewrzela%26aufirst%3DS.%26aulast%3DFrommolt%26aufirst%3DP.%26aulast%3DHerling%26aufirst%3DM.%26atitle%3DActionable%2520Perturbations%2520of%2520Damage%2520Responses%2520by%2520TCL1%252FATM%2520and%2520Epigenetic%2520Lesions%2520Form%2520the%2520Basis%2520of%2520T-PLL%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26spage%3D1%26epage%3D22%26doi%3D10.1038%2Fs41467-017-02688-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teitell, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konopleva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravandi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D.</span></span> <span> </span><span class="NLM_article-title">High TCL1 Expression and Intact T-Cell Receptor Signaling Define a Hyperproliferative Subset of T-Cell Prolymphocytic Leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">328</span>– <span class="NLM_lpage">337</span>, <span class="refDoi"> DOI: 10.1182/blood-2007-07-101519</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1182%2Fblood-2007-07-101519" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=17890451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtVKnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2008&pages=328-337&author=M.+Herlingauthor=K.+A.+Patelauthor=M.+A.+Teitellauthor=M.+Konoplevaauthor=F.+Ravandiauthor=R.+Kobayashiauthor=D.+Jones&title=High+TCL1+Expression+and+Intact+T-Cell+Receptor+Signaling+Define+a+Hyperproliferative+Subset+of+T-Cell+Prolymphocytic+Leukemia&doi=10.1182%2Fblood-2007-07-101519"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">High TCL1 expression and intact T-cell receptor signaling define a hyperproliferative subset of T-cell prolymphocytic leukemia</span></div><div class="casAuthors">Herling, Marco; Patel, Kaushali A.; Teitell, Michael A.; Konopleva, Marina; Ravandi, Farhad; Kobayashi, Ryuji; Jones, Dan</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">328-337</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The T-cell leukemia 1 (TCL1) oncoprotein is overexpressed by chromosomal rearrangement in the majority of cases of T-cell prolymphocytic leukemia (T-PLL).  In vitro, TCL1 can modulate the activity of the serine-threonine kinase AKT, a down-stream effector of T-cell receptor (TCR) signaling.  In a series of 86 T-PLL tumors, the authors show that expression of TCR, and levels of TCL1 and activated AKT are adverse prognostic markers.  High-level TCL1 in TCR-expressing T-PLL is assocd. with higher presenting white blood cell counts, faster tumor cell doubling, and enhanced in vitro growth response to TCR engagement.  In primary tumors and TCL1-transfected T-cell lines, TCR engagement leads to rapid recruitment of TCL1 and AKT to transient membrane activation complexes that include TCR-assocd. tyrosine kinases, including LCK.  Pharmacol. inhibition of AKT activation alters the localization, stability, and levels of these transient TCL1-AKT complexes and reduces tumor cell growth.  Exptl. introduction and knockdown of TCL1 influence the kinetics and strength of TCR-mediated AKT activation.  The authors propose that in T-PLL, TCL1 represents a highly regulated, targetable modulator of TCR-mediated AKT growth signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtog4z542fFbVg90H21EOLACvtfcHk0lhHZO4dG_quvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtVKnsw%253D%253D&md5=d7a199cc58464e4e4ecba59239c457e2</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1182%2Fblood-2007-07-101519&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2007-07-101519%26sid%3Dliteratum%253Aachs%26aulast%3DHerling%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DK.%2BA.%26aulast%3DTeitell%26aufirst%3DM.%2BA.%26aulast%3DKonopleva%26aufirst%3DM.%26aulast%3DRavandi%26aufirst%3DF.%26aulast%3DKobayashi%26aufirst%3DR.%26aulast%3DJones%26aufirst%3DD.%26atitle%3DHigh%2520TCL1%2520Expression%2520and%2520Intact%2520T-Cell%2520Receptor%2520Signaling%2520Define%2520a%2520Hyperproliferative%2520Subset%2520of%2520T-Cell%2520Prolymphocytic%2520Leukemia%26jtitle%3DBlood%26date%3D2008%26volume%3D111%26spage%3D328%26epage%3D337%26doi%3D10.1182%2Fblood-2007-07-101519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kiel, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velusamy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolland, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahasrabuddhe, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozel, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betz, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medeiros, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elenitoba-Johnson, K. S. J.</span></span> <span> </span><span class="NLM_article-title">Integrated Genomic Sequencing Reveals Mutational Landscape of T-Cell Prolymphocytic Leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">1460</span>– <span class="NLM_lpage">1472</span>, <span class="refDoi"> DOI: 10.1182/blood-2014-03-559542</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1182%2Fblood-2014-03-559542" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=24825865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFOksbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2014&pages=1460-1472&author=M.+J.+Kielauthor=T.+Velusamyauthor=D.+Rollandauthor=A.+A.+Sahasrabuddheauthor=F.+Chungauthor=N.+G.+Baileyauthor=A.+Schraderauthor=B.+Liauthor=J.+Z.+Liauthor=A.+B.+Ozelauthor=B.+L.+Betzauthor=R.+N.+Mirandaauthor=L.+J.+Medeirosauthor=L.+Zhaoauthor=M.+Herlingauthor=M.+S.+Limauthor=K.+S.+J.+Elenitoba-Johnson&title=Integrated+Genomic+Sequencing+Reveals+Mutational+Landscape+of+T-Cell+Prolymphocytic+Leukemia&doi=10.1182%2Fblood-2014-03-559542"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated genomic sequencing reveals mutational landscape of T-cell prolymphocytic leukemia</span></div><div class="casAuthors">Kiel, Mark J.; Velusamy, Thirunavukkarasu; Rolland, Delphine; Sahasrabuddhe, Anagh A.; Chung, Fuzon; Bailey, Nathanael G.; Schrader, Alexandra; Li, Bo; Li, Jun Z.; Ozel, Ayse B.; Betz, Bryan L.; Miranda, Roberto N.; Medeiros, L. Jeffrey; Zhao, Lili; Herling, Marco; Lim, Megan S.; Elenitoba-Johnson, Kojo S. J.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1460-1472</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The comprehensive genetic alterations underlying the pathogenesis of T-cell prolymphocytic leukemia (T-PLL) are unknown.  To address this, we performed whole-genome sequencing (WGS), whole-exome sequencing (WES), high-resoln. copy-no. anal., and Sanger resequencing of a large cohort of T-PLL.  WGS and WES identified novel mutations in recurrently altered genes not previously implicated in T-PLL including EZH2, FBXW10, and CHEK2.  Strikingly, WGS and/or WES showed largely mutually exclusive mutations affecting IL2RG, JAK1, JAK3, or STAT5B in 38 of 50 T-PLL genomes (76.0%).  Notably, gain-of-function IL2RG mutations are novel and have not been reported in any form of cancer.  Further, high-frequency mutations in STAT5B have not been previously reported in T-PLL.  Functionally, IL2RG-JAK1-JAK3-STAT5B mutations led to signal transducer and activator of transcription 5 (STAT5) hyperactivation, transformed Ba/F3 cells resulting in cytokine-independent growth, and/or enhanced colony formation in Jurkat T cells.  Importantly, primary T-PLL cells exhibited constitutive activation of STAT5, and targeted pharmacol. inhibition of STAT5 with pimozide induced apoptosis in primary T-PLL cells.  These results for the first time provide a portrait of the mutational landscape of T-PLL and implicate deregulation of DNA repair and epigenetic modulators as well as high-frequency mutational activation of the IL2RG-JAK1-JAK3-STAT5B axis in the pathogenesis of T-PLL.  These findings offer opportunities for novel targeted therapies in this aggressive leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7rpUv0RHH5bVg90H21EOLACvtfcHk0liXOE1lKSTi-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFOksbjN&md5=f199c9a7ef7b596f38de29347fab47db</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1182%2Fblood-2014-03-559542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2014-03-559542%26sid%3Dliteratum%253Aachs%26aulast%3DKiel%26aufirst%3DM.%2BJ.%26aulast%3DVelusamy%26aufirst%3DT.%26aulast%3DRolland%26aufirst%3DD.%26aulast%3DSahasrabuddhe%26aufirst%3DA.%2BA.%26aulast%3DChung%26aufirst%3DF.%26aulast%3DBailey%26aufirst%3DN.%2BG.%26aulast%3DSchrader%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DJ.%2BZ.%26aulast%3DOzel%26aufirst%3DA.%2BB.%26aulast%3DBetz%26aufirst%3DB.%2BL.%26aulast%3DMiranda%26aufirst%3DR.%2BN.%26aulast%3DMedeiros%26aufirst%3DL.%2BJ.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DHerling%26aufirst%3DM.%26aulast%3DLim%26aufirst%3DM.%2BS.%26aulast%3DElenitoba-Johnson%26aufirst%3DK.%2BS.%2BJ.%26atitle%3DIntegrated%2520Genomic%2520Sequencing%2520Reveals%2520Mutational%2520Landscape%2520of%2520T-Cell%2520Prolymphocytic%2520Leukemia%26jtitle%3DBlood%26date%3D2014%26volume%3D124%26spage%3D1460%26epage%3D1472%26doi%3D10.1182%2Fblood-2014-03-559542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wahnschaffe, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timonen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giri, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagle, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almusa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellanger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haferlach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durig, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siebrt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustjoki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aittokallio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span> <span> </span><span class="NLM_article-title">Jak/Stat-Activating Genomic Alterations Are a Hallmark of T-PLL</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1833</span>, <span class="refDoi"> DOI: 10.3390/cancers11121833</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.3390%2Fcancers11121833" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptVyiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=1833&author=L.+Wahnschaffeauthor=T.+Braunauthor=S.+Timonenauthor=A.+K.+Giriauthor=A.+Schraderauthor=P.+Wagleauthor=H.+Almusaauthor=P.+Johanssonauthor=D.+Bellangerauthor=C.+L%C3%B3pezauthor=C.+Haferlachauthor=M.+Sternauthor=J.+Durigauthor=R.+Siebrtauthor=S.+Mustjokiauthor=T.+Aittokallioauthor=M.+Herling&title=Jak%2FStat-Activating+Genomic+Alterations+Are+a+Hallmark+of+T-PLL&doi=10.3390%2Fcancers11121833"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">JAK/STAT-activating genomic alterations are a hallmark of T-PLL</span></div><div class="casAuthors">Wahnschaffe, Linus; Braun, Till; Timonen, Sanna; Giri, Anil K.; Schrader, Alexandra; Wagle, Prerana; Almusa, Henrikki; Johansson, Patricia; Bellanger, Dorine; Lopez, Cristina; Haferlach, Claudia; Stern, Marc-Henri; Duerig, Jan; Siebert, Reiner; Mustjoki, Satu; Aittokallio, Tero; Herling, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1833</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">T-cell prolymphocytic leukemia (T-PLL) is a rare and poor-prognostic mature T-cell leukemia.  Recent studies detected genomic aberrations affecting JAK and STAT genes in T-PLL.  Due to the limited no. of primary patient samples available, genomic analyses of the JAK/STAT pathway have been performed in rather small cohorts.  Therefore, we conducted-via a primary-data based pipeline-a meta-anal. that re-evaluated the genomic landscape of T-PLL.  It included all available data sets with sequence information on JAK or STAT gene loci in 275 T-PLL.  We eliminated overlapping cases and detd. a cumulative rate of 62.1% of cases with mutated JAK or STAT genes.  Most frequently, JAK1 (6.3%), JAK3 (36.4%), and STAT5B (18.8%) carried somatic single-nucleotide variants (SNVs), with missense mutations in the SH2 or pseudokinase domains as most prevalent.  Importantly, these lesions were predominantly subclonal.  We did not detect any strong assocn. between mutations of a JAK or STAT gene with clin. characteristics.  Irresp. of the presence of gain-of-function (GOF) SNVs, basal phosphorylation of STAT5B was elevated in all analyzed T-PLL.  Fittingly, a significant proportion of genes encoding for potential neg. regulators of STAT5B showed genomic losses (in 71.4% of T-PLL in total, in 68.4% of T-PLL without any JAK or STAT mutations).  They included DUSP4, CD45, TCPTP, SHP1, SOCS1, SOCS3, and HDAC9.  Overall, considering such losses of neg. regulators and the GOF mutations in JAK and STAT genes, a total of 89.8% of T-PLL revealed a genomic aberration potentially explaining enhanced STAT5B activity.  In essence, we present a comprehensive meta-anal. on the highly prevalent genomic lesions that affect genes encoding JAK/STAT signaling components.  This provides an overview of possible modes of activation of this pathway in a large cohort of T-PLL.  In light of new advances in JAK/STAT inhibitor development, we also outline translational contexts for harnessing active JAK/STAT signaling, which has emerged as a 'secondary' hallmark of T-PLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGria6G9Upve5rVg90H21EOLACvtfcHk0liXOE1lKSTi-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptVyiurc%253D&md5=7d5ff1c2af7f605cd645c4f4e4d38cda</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.3390%2Fcancers11121833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers11121833%26sid%3Dliteratum%253Aachs%26aulast%3DWahnschaffe%26aufirst%3DL.%26aulast%3DBraun%26aufirst%3DT.%26aulast%3DTimonen%26aufirst%3DS.%26aulast%3DGiri%26aufirst%3DA.%2BK.%26aulast%3DSchrader%26aufirst%3DA.%26aulast%3DWagle%26aufirst%3DP.%26aulast%3DAlmusa%26aufirst%3DH.%26aulast%3DJohansson%26aufirst%3DP.%26aulast%3DBellanger%26aufirst%3DD.%26aulast%3DL%25C3%25B3pez%26aufirst%3DC.%26aulast%3DHaferlach%26aufirst%3DC.%26aulast%3DStern%26aufirst%3DM.%26aulast%3DDurig%26aufirst%3DJ.%26aulast%3DSiebrt%26aufirst%3DR.%26aulast%3DMustjoki%26aufirst%3DS.%26aulast%3DAittokallio%26aufirst%3DT.%26aulast%3DHerling%26aufirst%3DM.%26atitle%3DJak%252FStat-Activating%2520Genomic%2520Alterations%2520Are%2520a%2520Hallmark%2520of%2520T-PLL%26jtitle%3DCancers%26date%3D2019%26volume%3D11%26spage%3D1833%26doi%3D10.3390%2Fcancers11121833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Araujo, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdogan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubauer, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meneksedag-Erol, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manaswiyoungkul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qadree, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israelian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suske, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, H. T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boersma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tangermann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rulicke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravichandran, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audette, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauscher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriggl, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span> <span> </span><span class="NLM_article-title">Structural and Functional Consequences of the STAT5BN642H Driver Mutation</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2517</span>, <span class="refDoi"> DOI: 10.1038/s41467-019-10422-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fs41467-019-10422-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=31175292" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A280%3ADC%252BB3M3ktlyltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=2517&author=E.+D.+de%0AAraujoauthor=F.+Erdoganauthor=H.+A.+Neubauerauthor=D.+Meneksedag-Erolauthor=P.+Manaswiyoungkulauthor=M.+S.+Eramauthor=H.-S.+Seoauthor=A.+K.+Qadreeauthor=J.+Israelianauthor=A.+Orlovaauthor=T.+Suskeauthor=H.+T.+T.+Phamauthor=A.+Boersmaauthor=S.+Tangermannauthor=L.+Kennerauthor=T.+Rulickeauthor=A.+Dongauthor=M.+Ravichandranauthor=P.+J.+Brownauthor=G.+F.+Audetteauthor=S.+Rauscherauthor=S.+Dhe-Paganonauthor=R.+Morigglauthor=P.+T.+Gunning&title=Structural+and+Functional+Consequences+of+the+STAT5BN642H+Driver+Mutation&doi=10.1038%2Fs41467-019-10422-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and functional consequences of the STAT5B(N642H) driver mutation</span></div><div class="casAuthors">de Araujo Elvin D; Erdogan Fettah; Meneksedag-Erol Deniz; Manaswiyoungkul Pimyupa; Qadree Abdul K; Israelian Johan; Rauscher Sarah; Gunning Patrick T; de Araujo Elvin D; Erdogan Fettah; Manaswiyoungkul Pimyupa; Qadree Abdul K; Israelian Johan; Rauscher Sarah; Gunning Patrick T; Neubauer Heidi A; Orlova Anna; Suske Tobias; Pham Ha T T; Moriggl Richard; Neubauer Heidi A; Orlova Anna; Pham Ha T T; Kenner Lukas; Moriggl Richard; Meneksedag-Erol Deniz; Rauscher Sarah; Eram Mohammad S; Seo Hyuk-Soo; Dhe-Paganon Sirano; Seo Hyuk-Soo; Dhe-Paganon Sirano; Boersma Auke; Rulicke Thomas; Tangermann Simone; Kenner Lukas; Kenner Lukas; Dong Aiping; Ravichandran Manimekalai; Brown Peter J; Audette Gerald F; Moriggl Richard</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2517</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Hyper-activated STAT5B variants are high value oncology targets for pharmacologic intervention.  STAT5B(N642H), a frequently-occurring oncogenic driver mutation, promotes aggressive T-cell leukemia/lymphoma in patient carriers, although the molecular origins remain unclear.  Herein, we emphasize the aggressive nature of STAT5B(N642H) in driving T-cell neoplasia upon hematopoietic expression in transgenic mice, revealing evidence of multiple T-cell subset organ infiltration.  Notably, we demonstrate STAT5B(N642H)-driven transformation of γδ T-cells in in vivo syngeneic transplant models, comparable to STAT5B(N642H) patient γδ T-cell entities.  Importantly, we present human STAT5B and STAT5B(N642H) crystal structures, which propose alternative mutation-mediated SH2 domain conformations.  Our biophysical data suggests STAT5B(N642H) can adopt a hyper-activated and hyper-inactivated state with resistance to dephosphorylation.  MD simulations support sustained interchain cross-domain interactions in STAT5B(N642H), conferring kinetic stability to the mutant anti-parallel dimer.  This study provides a molecular explanation for the STAT5B(N642H) activating potential, and insights into pre-clinical models for targeted intervention of hyper-activated STAT5B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSf39tGczghJI3yKPtYViAGfW6udTcc2eaoezhxYZAuZ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3ktlyltw%253D%253D&md5=393a1fb7ed0316b22b04c8fde6e3ad54</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41467-019-10422-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-019-10422-7%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BAraujo%26aufirst%3DE.%2BD.%26aulast%3DErdogan%26aufirst%3DF.%26aulast%3DNeubauer%26aufirst%3DH.%2BA.%26aulast%3DMeneksedag-Erol%26aufirst%3DD.%26aulast%3DManaswiyoungkul%26aufirst%3DP.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DQadree%26aufirst%3DA.%2BK.%26aulast%3DIsraelian%26aufirst%3DJ.%26aulast%3DOrlova%26aufirst%3DA.%26aulast%3DSuske%26aufirst%3DT.%26aulast%3DPham%26aufirst%3DH.%2BT.%2BT.%26aulast%3DBoersma%26aufirst%3DA.%26aulast%3DTangermann%26aufirst%3DS.%26aulast%3DKenner%26aufirst%3DL.%26aulast%3DRulicke%26aufirst%3DT.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DRavichandran%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DAudette%26aufirst%3DG.%2BF.%26aulast%3DRauscher%26aufirst%3DS.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DMoriggl%26aufirst%3DR.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26atitle%3DStructural%2520and%2520Functional%2520Consequences%2520of%2520the%2520STAT5BN642H%2520Driver%2520Mutation%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D2517%26doi%3D10.1038%2Fs41467-019-10422-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timonen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koschmieder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wennerberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustjoki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aittokallio, T.</span></span> <span> </span><span class="NLM_article-title">Patient-Customized Drug Combination Prediction and Testing for t-Cell Prolymphocytic Leukemia Patients</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">2407</span>– <span class="NLM_lpage">2418</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-17-3644</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1158%2F0008-5472.CAN-17-3644" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=29483097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosF2qtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2018&pages=2407-2418&author=L.+Heauthor=J.+Tangauthor=E.+I.+Anderssonauthor=S.+Timonenauthor=S.+Koschmiederauthor=K.+Wennerbergauthor=S.+Mustjokiauthor=T.+Aittokallio&title=Patient-Customized+Drug+Combination+Prediction+and+Testing+for+t-Cell+Prolymphocytic+Leukemia+Patients&doi=10.1158%2F0008-5472.CAN-17-3644"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Patient-customized drug combination prediction and testing for t-cell prolymphocytic leukemia patients</span></div><div class="casAuthors">He, Liye; Tang, Jing; Andersson, Emma I.; Timonen, Sanna; Koschmieder, Steffen; Wennerberg, Krister; Mustjoki, Satu; Aittokallio, Tero</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2407-2418</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The mol. pathways that drive cancer progression and treatment resistance are highly redundant and variable between individual patients with the same cancer type.  To tackle this complex rewiring of pathway cross-talk, personalized combination treatments targeting multiple cancer growth and survival pathways are required.  Here we implemented a computational-exptl. drug combination prediction and testing (DCPT) platform for efficient in silico prioritization and ex vivo testing in patient-derived samples to identify customized synergistic combinations for individual cancer patients.  DCPT used drug-target interaction networks to traverse the massive combinatorial search spaces among 218 compds. (a total of 23,653 pairwise combinations) and identified cancer-selective synergies by using differential single-compd. sensitivity profiles between patient cells and healthy controls, hence reducing the likelihood of toxic combination effects.  A polypharmacol.-based machine learning modeling and network visualization made use of baseline genomic and mol. profiles to guide patient-specific combination testing and clin. translation phases.  Using T-cell prolymphocytic leukemia (T-PLL) as a first case study, we show how the DCPT platform successfully predicted distinct synergistic combinations for each of the three T-PLL patients, each presenting with different resistance patterns and synergy mechanisms.  In total, 10 of 24 (42%) of selective combination predictions were exptl. confirmed to show synergy in patient-derived samples ex vivo.  The identified selective synergies among approved drugs, including tacrolimus and temsirolimus combined with BCL-2 inhibitor venetoclax, may offer novel drug repurposing opportunities for treating T-PLL.  Significance: An integrated use of functional drug screening combined with genomic and mol. profiling enables patient-customized prediction and testing of drug combination synergies for T-PLL patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokJsA7GQacZbVg90H21EOLACvtfcHk0lie7GqK0TAJUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosF2qtbo%253D&md5=a6a97c5c2e41d663aebcc028f1331e59</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-17-3644&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-17-3644%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DL.%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DAndersson%26aufirst%3DE.%2BI.%26aulast%3DTimonen%26aufirst%3DS.%26aulast%3DKoschmieder%26aufirst%3DS.%26aulast%3DWennerberg%26aufirst%3DK.%26aulast%3DMustjoki%26aufirst%3DS.%26aulast%3DAittokallio%26aufirst%3DT.%26atitle%3DPatient-Customized%2520Drug%2520Combination%2520Prediction%2520and%2520Testing%2520for%2520t-Cell%2520Prolymphocytic%2520Leukemia%2520Patients%26jtitle%3DCancer%2520Res.%26date%3D2018%26volume%3D78%26spage%3D2407%26epage%3D2418%26doi%3D10.1158%2F0008-5472.CAN-17-3644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dearden, C.</span></span> <span> </span><span class="NLM_article-title">How I Treat Prolymphocytic Leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">538</span>– <span class="NLM_lpage">551</span>, <span class="refDoi"> DOI: 10.1182/blood-2012-01-380139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1182%2Fblood-2012-01-380139" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=22649104" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFOntrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2012&pages=538-551&author=C.+Dearden&title=How+I+Treat+Prolymphocytic+Leukemia&doi=10.1182%2Fblood-2012-01-380139"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">How I treat prolymphocytic leukemia</span></div><div class="casAuthors">Dearden, Claire</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">538-551</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">A review.  T- and B-cell subtypes of prolymphocytic leukemia (PLL) are rare, aggressive lymphoid malignancies with characteristic morphol., immunophenotypic, cytogenetic, and mol. features.  Recent studies have highlighted the role of specific oncogenes, such as TCL-1, MTCP-1, and ATM in the case of T-cell and TP53 mutations in the case of B-cell prolymphocytic leukemia.  Despite the advances in the understanding of the biol. of these conditions, the prognosis for these patients remains poor with short survival and no curative therapy.  The advent of monoclonal antibodies has improved treatment options.  Currently, the best treatment for T-PLL is i.v. alemtuzumab, which has resulted in very high response rates of more than 90% when given as first-line treatment and a significant improvement in survival.  Consolidation of remissions with autologous or allogeneic stem cell transplantation further prolongs survival, and the latter may offer potential cure.  In B-PLL, rituximab-based combination chemo-immunotherapy is effective in fitter patients.  TP53 abnormalities are common and, as for chronic lymphocytic leukemia, these patients should be managed using an alemtuzumab-based therapy.  The role of allogeneic transplant with nonmyeloablative conditioning needs to be explored further in both T- and B-cell PLL to broaden the patient eligibility for what may be a curative treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlLkh6TX_EP7Vg90H21EOLACvtfcHk0lie7GqK0TAJUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFOntrjF&md5=7b3dbf767a348de953e7f91b151e34f1</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1182%2Fblood-2012-01-380139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2012-01-380139%26sid%3Dliteratum%253Aachs%26aulast%3DDearden%26aufirst%3DC.%26atitle%3DHow%2520I%2520Treat%2520Prolymphocytic%2520Leukemia%26jtitle%3DBlood%26date%3D2012%26volume%3D120%26spage%3D538%26epage%3D551%26doi%3D10.1182%2Fblood-2012-01-380139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hockly, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richon, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sathasivam, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghazi-Noori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowden, P. A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffan, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, G. P.</span></span> <span> </span><span class="NLM_article-title">Suberoylanilide Hydroxamic Acid, a Histone Deacetylase Inhibitor, Ameliorates Motor Deficits in a Mouse Model of Huntington’s Disease</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">2041</span>– <span class="NLM_lpage">2046</span>, <span class="refDoi"> DOI: 10.1073/pnas.0437870100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1073%2Fpnas.0437870100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=12576549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhsFGrsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2003&pages=2041-2046&author=E.+Hocklyauthor=V.+M.+Richonauthor=B.+Woodmanauthor=D.+L.+Smithauthor=X.+Zhouauthor=E.+Rosaauthor=K.+Sathasivamauthor=S.+Ghazi-Nooriauthor=A.+Mahalauthor=P.+A.+S.+Lowdenauthor=J.+S.+Steffanauthor=J.+L.+Marshauthor=L.+M.+Thompsonauthor=C.+M.+Lewisauthor=P.+A.+Marksauthor=G.+P.+Bates&title=Suberoylanilide+Hydroxamic+Acid%2C+a+Histone+Deacetylase+Inhibitor%2C+Ameliorates+Motor+Deficits+in+a+Mouse+Model+of+Huntington%E2%80%99s+Disease&doi=10.1073%2Fpnas.0437870100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Suberoylanilide hydroxamic acid, a histone deacetylase inhibitor, ameliorates motor deficits in a mouse model of Huntington's disease</span></div><div class="casAuthors">Hockly, Emma; Richon, Victoria M.; Woodman, Benjamin; Smith, Donna L.; Zhou, Xianbo; Rosa, Eddie; Sathasivam, Kirupa; Ghazi-Noori, Shabnam; Mahal, Amarbirpal; Lowden, Philip A. S.; Steffan, Joan S.; Marsh, J. Lawrence; Thompson, Leslie M.; Lewis, Cathryn M.; Marks, Paul A.; Bates, Gillian P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2041-2046</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is an inherited, progressive neurol. disorder that is caused by a CAG/polyglutamine repeat expansion and for which there is no effective therapy.  Recent evidence indicates that transcriptional dysregulation may contribute to the mol. pathogenesis of this disease.  Supporting this view, administration of histone deacetylase (HDAC) inhibitors has been shown to rescue lethality and photoreceptor neurodegeneration in a Drosophila model of polyglutamine disease.  To further explore the therapeutic potential of HDAC inhibitors, we have conducted preclin. trials with suberoylanilide hydroxamic acid (SAHA), a potent HDAC inhibitor, in the R6/2 HD mouse model.  We show that SAHA crosses the blood-brain barrier and increases histone acetylation in the brain.  We found that SAHA could be administered orally in drinking water when complexed with cyclodextrins.  SAHA dramatically improved the motor impairment in R6/2 mice, clearly validating the pursuit of this class of compds. as HD therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpy-3Y7VV7UbVg90H21EOLACvtfcHk0lie7GqK0TAJUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhsFGrsbs%253D&md5=4096dcbfeeeafd00c51ccb01e4ee29a5</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0437870100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0437870100%26sid%3Dliteratum%253Aachs%26aulast%3DHockly%26aufirst%3DE.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DWoodman%26aufirst%3DB.%26aulast%3DSmith%26aufirst%3DD.%2BL.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DRosa%26aufirst%3DE.%26aulast%3DSathasivam%26aufirst%3DK.%26aulast%3DGhazi-Noori%26aufirst%3DS.%26aulast%3DMahal%26aufirst%3DA.%26aulast%3DLowden%26aufirst%3DP.%2BA.%2BS.%26aulast%3DSteffan%26aufirst%3DJ.%2BS.%26aulast%3DMarsh%26aufirst%3DJ.%2BL.%26aulast%3DThompson%26aufirst%3DL.%2BM.%26aulast%3DLewis%26aufirst%3DC.%2BM.%26aulast%3DMarks%26aufirst%3DP.%2BA.%26aulast%3DBates%26aufirst%3DG.%2BP.%26atitle%3DSuberoylanilide%2520Hydroxamic%2520Acid%252C%2520a%2520Histone%2520Deacetylase%2520Inhibitor%252C%2520Ameliorates%2520Motor%2520Deficits%2520in%2520a%2520Mouse%2520Model%2520of%2520Huntington%25E2%2580%2599s%2520Disease%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2003%26volume%3D100%26spage%3D2041%26epage%3D2046%26doi%3D10.1073%2Fpnas.0437870100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakimoto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chumakov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koeffler, H. P.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylase Inhibitor, Suberoylanilide Hydroxamic Acid (Vorinostat, SAHA) Profoundly Inhibits the Growth of Human Pancreatic Cancer Cells</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>121</i></span>,  <span class="NLM_fpage">656</span>– <span class="NLM_lpage">665</span>, <span class="refDoi"> DOI: 10.1002/ijc.22558</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1002%2Fijc.22558" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=17417771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnt1Crsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2007&pages=656-665&author=T.+Kumagaiauthor=N.+Wakimotoauthor=D.+Yinauthor=S.+Geryauthor=N.+Kawamataauthor=N.+Takaiauthor=N.+Komatsuauthor=A.+Chumakovauthor=Y.+Imaiauthor=H.+P.+Koeffler&title=Histone+Deacetylase+Inhibitor%2C+Suberoylanilide+Hydroxamic+Acid+%28Vorinostat%2C+SAHA%29+Profoundly+Inhibits+the+Growth+of+Human+Pancreatic+Cancer+Cells&doi=10.1002%2Fijc.22558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase inhibitor, suberoylanilide hydroxamic acid (Vorinostat, SAHA) profoundly inhibits the growth of human pancreatic cancer cells</span></div><div class="casAuthors">Kumagai, Takashi; Wakimoto, Naoki; Yin, Dong; Gery, Sigal; Kawamata, Norihiko; Takai, Noriyuki; Komatsu, Naoki; Chumakov, Alexy; Imai, Yasufumi; Koeffler, H. Phillip</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">656-665</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Tumor suppressor genes are often silenced in human cancer; this can occur by transcriptional repression by deacetylation in the promoter regions, mediated by histone deacetylase (HDAC).  HDAC inhibitors can block cancer cell growth by restoring expression of tumor suppressor genes.  In this study, we investigated the effects of a HDAC inhibitor, suberoylanilide hydroxamic acid (SAHA) on pancreatic cancer cells.  SAHA inhibited the growth of 6 pancreatic cancer cell lines in a dose-dependent manner as measured by MTT and clonogenic assays (ED50 ≈10-6 M) assocd. with induction of apoptosis, G2 cell cycle arrest and also induced differentiation as indicated by morphol. and increased expression of cytokeratin 7.  It increased expression of p21WAF1 (independent of the mutational status of p53), C/EBPα, RARα and E-cadherin; these genes have been assocd. with decreased proliferation in other cancers.  SAHA decreased cyclin B1 expression; this cyclin normally promotes progression through G2 of the cell cycle.  SAHA mediated acetylation of histone H3 globally, as well as, assocd. with the p21WAF1 promoter, as measured by chromatin immunopptn.  SAHA also decreased levels of c-myc and cyclin D1, independent of an active β-catenin pathway.  In further studies, the combination of SAHA and an inhibitor of DNA methylation, 5-Aza-2'-deoxycytidine, had an enhanced antiproliferative effect on pancreatic cancer cells.  In summary, SAHA inhibited the growth of human pancreatic cancer cells by inducing apoptosis, differentiation and cell cycle arrest, as well as increase in the expression of several tumor suppressor genes.  SAHA is a novel, promising therapeutic agent for human pancreatic cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVgeJjEzQr77Vg90H21EOLACvtfcHk0lhe2YWv_jAfWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnt1Crsrs%253D&md5=4c7224c82f75d160b934fb0560f73eb5</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fijc.22558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.22558%26sid%3Dliteratum%253Aachs%26aulast%3DKumagai%26aufirst%3DT.%26aulast%3DWakimoto%26aufirst%3DN.%26aulast%3DYin%26aufirst%3DD.%26aulast%3DGery%26aufirst%3DS.%26aulast%3DKawamata%26aufirst%3DN.%26aulast%3DTakai%26aufirst%3DN.%26aulast%3DKomatsu%26aufirst%3DN.%26aulast%3DChumakov%26aufirst%3DA.%26aulast%3DImai%26aufirst%3DY.%26aulast%3DKoeffler%26aufirst%3DH.%2BP.%26atitle%3DHistone%2520Deacetylase%2520Inhibitor%252C%2520Suberoylanilide%2520Hydroxamic%2520Acid%2520%2528Vorinostat%252C%2520SAHA%2529%2520Profoundly%2520Inhibits%2520the%2520Growth%2520of%2520Human%2520Pancreatic%2520Cancer%2520Cells%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2007%26volume%3D121%26spage%3D656%26epage%3D665%26doi%3D10.1002%2Fijc.22558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruefli, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ausserlechner, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tainton, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kofler, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span> <span> </span><span class="NLM_article-title">The Histone Deacetylase Inhibitor and Chemotherapeutic Agent Suberoylanilide Hydroxamic Acid (SAHA) Induces a Cell-Death Pathway Characterized by Cleavage of Bid and Production of Reactive Oxygen Species</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">10833</span>– <span class="NLM_lpage">10838</span>, <span class="refDoi"> DOI: 10.1073/pnas.191208598</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1073%2Fpnas.191208598" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=11535817" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntVGmsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=10833-10838&author=A.+A.+Ruefliauthor=M.+J.+Ausserlechnerauthor=D.+Bernhardauthor=V.+R.+Suttonauthor=K.+M.+Taintonauthor=R.+Koflerauthor=M.+J.+Smythauthor=R.+W.+Johnstone&title=The+Histone+Deacetylase+Inhibitor+and+Chemotherapeutic+Agent+Suberoylanilide+Hydroxamic+Acid+%28SAHA%29+Induces+a+Cell-Death+Pathway+Characterized+by+Cleavage+of+Bid+and+Production+of+Reactive+Oxygen+Species&doi=10.1073%2Fpnas.191208598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The histone deacetylase inhibitor and chemotherapeutic agent suberoylanilide hydroxamic acid (SAHA) induces a cell-death pathway characterized by cleavage of Bid and production of reactive oxygen species</span></div><div class="casAuthors">Ruefli, Astrid A.; Ausserlechner, Michael J.; Bernhard, David; Sutton, Vivien R.; Tainton, Kellie M.; Kofler, Reinhard; Smyth, Mark J.; Johnstone, Ricky W.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">10833-10838</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Many chemotherapeutic agents induce mitochondrial-membrane disruption to initiate apoptosis.  However, the upstream events leading to drug-induced mitochondrial perturbation have remained poorly defined.  We have used a variety of physiol. and pharmacol. inhibitors of distinct apoptotic pathways to analyze the manner by which suberoylanilide hydroxamic acid (SAHA), a chemotherapeutic agent and histone deacetylase inhibitor, induces cell death.  We demonstrate that SAHA initiates cell death by inducing mitochondria-mediated death pathways characterized by cytochrome c release and the prodn. of reactive oxygen species, and does not require the activation of key caspases such as caspase-8 or -3.  We provide evidence that mitochondrial disruption is achieved by means of the cleavage of the BH3-only proapoptotic Bcl-2 family member Bid.  SAHA-induced Bid cleavage was not blocked by caspase inhibitors or the overexpression of Bcl-2 but did require the transcriptional regulatory activity of SAHA.  These data provide evidence of a mechanism of cell death mediated by transcriptional events that result in the cleavage of Bid, disruption of the mitochondrial membrane, and prodn. of reactive oxygen species to induce cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKvHbIDNsJq7Vg90H21EOLACvtfcHk0lhe2YWv_jAfWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntVGmsLY%253D&md5=0b35cff7a6f2af9bb236e6c7ae942db9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1073%2Fpnas.191208598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.191208598%26sid%3Dliteratum%253Aachs%26aulast%3DRuefli%26aufirst%3DA.%2BA.%26aulast%3DAusserlechner%26aufirst%3DM.%2BJ.%26aulast%3DBernhard%26aufirst%3DD.%26aulast%3DSutton%26aufirst%3DV.%2BR.%26aulast%3DTainton%26aufirst%3DK.%2BM.%26aulast%3DKofler%26aufirst%3DR.%26aulast%3DSmyth%26aufirst%3DM.%2BJ.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26atitle%3DThe%2520Histone%2520Deacetylase%2520Inhibitor%2520and%2520Chemotherapeutic%2520Agent%2520Suberoylanilide%2520Hydroxamic%2520Acid%2520%2528SAHA%2529%2520Induces%2520a%2520Cell-Death%2520Pathway%2520Characterized%2520by%2520Cleavage%2520of%2520Bid%2520and%2520Production%2520of%2520Reactive%2520Oxygen%2520Species%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D10833%26epage%3D10838%26doi%3D10.1073%2Fpnas.191208598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwitkowski, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Valle, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricci, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saber, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habtemariam, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloomquist, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehrotra, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorff, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charlab, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kane, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaminskas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Justice, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrell, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazdur, R.</span></span> <span> </span><span class="NLM_article-title">FDA Approval: Belinostat for the Treatment of Patients with Relapsed or Refractory Peripheral T-Cell Lymphoma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2666</span>– <span class="NLM_lpage">2670</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-14-3119</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1158%2F1078-0432.CCR-14-3119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=25802282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVajurfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=2666-2670&author=H.+Z.+Leeauthor=V.+E.+Kwitkowskiauthor=P.+L.+Del+Valleauthor=M.+S.+Ricciauthor=H.+Saberauthor=B.+A.+Habtemariamauthor=J.+Bullockauthor=E.+Bloomquistauthor=Y.+L.+Shenauthor=X.+H.+Chenauthor=J.+Brownauthor=N.+Mehrotraauthor=S.+Dorffauthor=R.+Charlabauthor=R.+C.+Kaneauthor=E.+Kaminskasauthor=R.+Justiceauthor=A.+T.+Farrellauthor=R.+Pazdur&title=FDA+Approval%3A+Belinostat+for+the+Treatment+of+Patients+with+Relapsed+or+Refractory+Peripheral+T-Cell+Lymphoma&doi=10.1158%2F1078-0432.CCR-14-3119"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">FDA Approval: Belinostat for the Treatment of Patients with Relapsed or Refractory Peripheral T-cell Lymphoma</span></div><div class="casAuthors">Lee, Hyon-Zu; Kwitkowski, Virginia E.; Del Valle, Pedro L.; Ricci, M. Stacey; Saber, Haleh; Habtemariam, Bahru A.; Bullock, Julie; Bloomquist, Erik; Shen, Yuan Li; Chen, Xiao-Hong; Brown, Janice; Mehrotra, Nitin; Dorff, Sarah; Charlab, Rosane; Kane, Robert C.; Kaminskas, Edvardas; Justice, Robert; Farrell, Ann T.; Pazdur, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2666-2670</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">On July 3, 2014, the FDA granted accelerated approval for belinostat (Beleodaq; Spectrum Pharmaceuticals, Inc.), a histone deacetylase inhibitor, for the treatment of patients with relapsed or refractory peripheral T-cell lymphoma (PTCL).  A single-arm, open-label, multicenter, international trial in the indicated patient population was submitted in support of the application.  Belinostat was administered i.v. at a dose of 1000 mg/m2 over 30 min once daily on days 1 to 5 of a 21-day cycle.  The primary efficacy endpoint was overall response rate (ORR) based on central radiol. readings by an independent review committee.  The ORR was 25.8% [95% confidence interval (CI), 18.3-34.6] in 120 patients that had confirmed diagnoses of PTCL by the Central Pathol. Review Group.  The complete and partial response rates were 10.8% (95% CI, 5.9-17.8) and 15.0% (95% CI, 9.1-22.7), resp.  The median duration of response, the key secondary efficacy endpoint, was 8.4 mo (95% CI, 4.5-29.4).  The most common adverse reactions (>25%) were nausea, fatigue, pyrexia, anemia, and vomiting.  Grade 3/4 toxicities (≥5.0%) included anemia, thrombocytopenia, dyspnea, neutropenia, fatigue, and pneumonia.  Belinostat is the third drug to receive accelerated approval for the treatment of relapsed or refractory PTCL.  Clin Cancer Res; 21(12); 2666-70. ©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1RbkvuHSk6rVg90H21EOLACvtfcHk0lhe2YWv_jAfWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVajurfM&md5=c1e64c9d01d18272d25ad03001e7041a</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-3119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-3119%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%2BZ.%26aulast%3DKwitkowski%26aufirst%3DV.%2BE.%26aulast%3DDel%2BValle%26aufirst%3DP.%2BL.%26aulast%3DRicci%26aufirst%3DM.%2BS.%26aulast%3DSaber%26aufirst%3DH.%26aulast%3DHabtemariam%26aufirst%3DB.%2BA.%26aulast%3DBullock%26aufirst%3DJ.%26aulast%3DBloomquist%26aufirst%3DE.%26aulast%3DShen%26aufirst%3DY.%2BL.%26aulast%3DChen%26aufirst%3DX.%2BH.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DMehrotra%26aufirst%3DN.%26aulast%3DDorff%26aufirst%3DS.%26aulast%3DCharlab%26aufirst%3DR.%26aulast%3DKane%26aufirst%3DR.%2BC.%26aulast%3DKaminskas%26aufirst%3DE.%26aulast%3DJustice%26aufirst%3DR.%26aulast%3DFarrell%26aufirst%3DA.%2BT.%26aulast%3DPazdur%26aufirst%3DR.%26atitle%3DFDA%2520Approval%253A%2520Belinostat%2520for%2520the%2520Treatment%2520of%2520Patients%2520with%2520Relapsed%2520or%2520Refractory%2520Peripheral%2520T-Cell%2520Lymphoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26spage%3D2666%26epage%3D2670%26doi%3D10.1158%2F1078-0432.CCR-14-3119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pützer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varghese, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Jan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giri, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riet, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timonen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oberbeck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuusanmäki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustjoki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aittokallio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newrzela, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span> <span> </span><span class="NLM_article-title">Reinstated P53 Response and High Anti-T-Cell Leukemia Activity by the Novel Alkylating Deacetylase Inhibitor Tinostamustine</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">2513</span>– <span class="NLM_lpage">2518</span>, <span class="refDoi"> DOI: 10.1038/s41375-020-0772-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fs41375-020-0772-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=32099034" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2020&pages=2513-2518&author=S.+P%C3%BCtzerauthor=L.+Vargheseauthor=J.+von+Janauthor=T.+Braunauthor=A.+K.+Giriauthor=P.+Mayerauthor=N.+Rietauthor=S.+Timonenauthor=S.+Oberbeckauthor=H.+Kuusanm%C3%A4kiauthor=S.+Mustjokiauthor=M.+Sternauthor=T.+Aittokallioauthor=S.+Newrzelaauthor=A.+Schraderauthor=M.+Herling&title=Reinstated+P53+Response+and+High+Anti-T-Cell+Leukemia+Activity+by+the+Novel+Alkylating+Deacetylase+Inhibitor+Tinostamustine&doi=10.1038%2Fs41375-020-0772-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fs41375-020-0772-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41375-020-0772-6%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25BCtzer%26aufirst%3DS.%26aulast%3DVarghese%26aufirst%3DL.%26aulast%3Dvon%2BJan%26aufirst%3DJ.%26aulast%3DBraun%26aufirst%3DT.%26aulast%3DGiri%26aufirst%3DA.%2BK.%26aulast%3DMayer%26aufirst%3DP.%26aulast%3DRiet%26aufirst%3DN.%26aulast%3DTimonen%26aufirst%3DS.%26aulast%3DOberbeck%26aufirst%3DS.%26aulast%3DKuusanm%25C3%25A4ki%26aufirst%3DH.%26aulast%3DMustjoki%26aufirst%3DS.%26aulast%3DStern%26aufirst%3DM.%26aulast%3DAittokallio%26aufirst%3DT.%26aulast%3DNewrzela%26aufirst%3DS.%26aulast%3DSchrader%26aufirst%3DA.%26aulast%3DHerling%26aufirst%3DM.%26atitle%3DReinstated%2520P53%2520Response%2520and%2520High%2520Anti-T-Cell%2520Leukemia%2520Activity%2520by%2520the%2520Novel%2520Alkylating%2520Deacetylase%2520Inhibitor%2520Tinostamustine%26jtitle%3DLeukemia%26date%3D2020%26volume%3D34%26spage%3D2513%26epage%3D2518%26doi%3D10.1038%2Fs41375-020-0772-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pützer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufva, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crispatzu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oleś, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adnan-Awad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagstrom, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellanger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mpindi, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldfors, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pemovska, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietarinen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lauhio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuesta-Mateos, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koschmieder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brummendorf, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kytola, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savolainen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siitonen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellonen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallioneimi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wennerberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, M_H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aittokallio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zenz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustjoki, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel Drug Sensitivities in T-PLL by High-Throughput Ex Vivo Drug Testing and Mutation Profiling</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">774</span>– <span class="NLM_lpage">787</span>, <span class="refDoi"> DOI: 10.1038/leu.2017.252</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fleu.2017.252" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=28804127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKmu7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=774-787&author=E.+I.+Anderssonauthor=S.+P%C3%BCtzerauthor=B.+Yadavauthor=O.+Dufvaauthor=S.+Khanauthor=L.+Heauthor=L.+Sellnerauthor=A.+Schraderauthor=G.+Crispatzuauthor=M.+Ole%C5%9Bauthor=H.+Zhangauthor=S.+Adnan-Awadauthor=S.+Lagstromauthor=D.+Bellangerauthor=J.+P.+Mpindiauthor=S.+Eldforsauthor=T.+Pemovskaauthor=P.+Pietarinenauthor=A.+Lauhioauthor=K.+Tomskaauthor=C.+Cuesta-Mateosauthor=E.+Faberauthor=S.+Koschmiederauthor=T.+H.+Brummendorfauthor=S.+Kytolaauthor=E.+Savolainenauthor=T.+Siitonenauthor=P.+Ellonenauthor=O.+Kallioneimiauthor=K.+Wennerbergauthor=W.+Dingauthor=M_H.+Sternauthor=W.+Huberauthor=S.+Andersauthor=J.+Tangauthor=T.+Aittokallioauthor=T.+Zenzauthor=M.+Herlingauthor=S.+Mustjoki&title=Discovery+of+Novel+Drug+Sensitivities+in+T-PLL+by+High-Throughput+Ex+Vivo+Drug+Testing+and+Mutation+Profiling&doi=10.1038%2Fleu.2017.252"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel drug sensitivities in T-PLL by high-throughput ex vivo drug testing and mutation profiling</span></div><div class="casAuthors">Andersson, E. I.; Putzer, S.; Yadav, B.; Dufva, O.; Khan, S.; He, L.; Sellner, L.; Schrader, A.; Crispatzu, G.; Oles, M.; Zhang, H.; Adnan-Awad, S.; Lagstrom, S.; Bellanger, D.; Mpindi, J. P.; Eldfors, S.; Pemovska, T.; Pietarinen, P.; Lauhio, A.; Tomska, K.; Cuesta-Mateos, C.; Faber, E.; Koschmieder, S.; Brummendorf, T. H.; Kytola, S.; Savolainen, E.-R.; Siitonen, T.; Ellonen, P.; Kallioniemi, O.; Wennerberg, K.; Ding, W.; Stern, M.-H.; Huber, W.; Anders, S.; Tang, J.; Aittokallio, T.; Zenz, T.; Herling, M.; Mustjoki, S.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">774-787</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">T-cell prolymphocytic leukemia (T-PLL) is a rare and aggressive neoplasm of mature T-cells with an urgent need for rationally designed therapies to address its notoriously chemo-refractory behavior.  The median survival of T-PLL patients is <2 years and clin. trials are difficult to execute.  Here we systematically explored the diversity of drug responses in T-PLL patient samples using an ex vivo drug sensitivity and resistance testing platform and correlated the findings with somatic mutations and gene expression profiles.  Intriguingly, all T-PLL samples were sensitive to the cyclin-dependent kinase inhibitor SNS-032, which overcame stromal-cell-mediated protection and elicited robust p53-activation and apoptosis.  Across all patients, the most effective classes of compds. were histone deacetylase, phosphoinositide-3 kinase/AKT/mammalian target of rapamycin, heat-shock protein 90 and BH3-family protein inhibitors as well as p53 activators, indicating previously unexplored, novel targeted approaches for treating T-PLL.  Although Janus-activated kinase-signal transducer and activator of transcription factor (JAK-STAT) pathway mutations were common in T-PLL (71% of patients), JAK-STAT inhibitor responses were not directly linked to those or other T-PLL-specific lesions.  Overall, we found that genetic markers do not readily translate into novel effective therapeutic vulnerabilities.  In conclusion, novel classes of compds. with high efficacy in T-PLL were discovered with the comprehensive ex vivo drug screening platform warranting further studies of synergisms and clin. testing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNlAKw7-znF7Vg90H21EOLACvtfcHk0lg0EC3gQrSwVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKmu7%252FI&md5=70fedb2dce547d45c70300ec7759ee07</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fleu.2017.252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2017.252%26sid%3Dliteratum%253Aachs%26aulast%3DAndersson%26aufirst%3DE.%2BI.%26aulast%3DP%25C3%25BCtzer%26aufirst%3DS.%26aulast%3DYadav%26aufirst%3DB.%26aulast%3DDufva%26aufirst%3DO.%26aulast%3DKhan%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DL.%26aulast%3DSellner%26aufirst%3DL.%26aulast%3DSchrader%26aufirst%3DA.%26aulast%3DCrispatzu%26aufirst%3DG.%26aulast%3DOle%25C5%259B%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DAdnan-Awad%26aufirst%3DS.%26aulast%3DLagstrom%26aufirst%3DS.%26aulast%3DBellanger%26aufirst%3DD.%26aulast%3DMpindi%26aufirst%3DJ.%2BP.%26aulast%3DEldfors%26aufirst%3DS.%26aulast%3DPemovska%26aufirst%3DT.%26aulast%3DPietarinen%26aufirst%3DP.%26aulast%3DLauhio%26aufirst%3DA.%26aulast%3DTomska%26aufirst%3DK.%26aulast%3DCuesta-Mateos%26aufirst%3DC.%26aulast%3DFaber%26aufirst%3DE.%26aulast%3DKoschmieder%26aufirst%3DS.%26aulast%3DBrummendorf%26aufirst%3DT.%2BH.%26aulast%3DKytola%26aufirst%3DS.%26aulast%3DSavolainen%26aufirst%3DE.%26aulast%3DSiitonen%26aufirst%3DT.%26aulast%3DEllonen%26aufirst%3DP.%26aulast%3DKallioneimi%26aufirst%3DO.%26aulast%3DWennerberg%26aufirst%3DK.%26aulast%3DDing%26aufirst%3DW.%26aulast%3DStern%26aufirst%3DM_H.%26aulast%3DHuber%26aufirst%3DW.%26aulast%3DAnders%26aufirst%3DS.%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DAittokallio%26aufirst%3DT.%26aulast%3DZenz%26aufirst%3DT.%26aulast%3DHerling%26aufirst%3DM.%26aulast%3DMustjoki%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520Novel%2520Drug%2520Sensitivities%2520in%2520T-PLL%2520by%2520High-Throughput%2520Ex%2520Vivo%2520Drug%2520Testing%2520and%2520Mutation%2520Profiling%26jtitle%3DLeukemia%26date%3D2018%26volume%3D32%26spage%3D774%26epage%3D787%26doi%3D10.1038%2Fleu.2017.252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mrakovcic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleinheinz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fröhlich, L. F.</span></span> <span> </span><span class="NLM_article-title">P53 at the Crossroads between Different Types of Hdac Inhibitor-Mediated Cancer Cell Death</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">2415</span>, <span class="refDoi"> DOI: 10.3390/ijms20102415</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.3390%2Fijms20102415" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosVGmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=2415&author=M.+Mrakovcicauthor=J.+Kleinheinzauthor=L.+F.+Fr%C3%B6hlich&title=P53+at+the+Crossroads+between+Different+Types+of+Hdac+Inhibitor-Mediated+Cancer+Cell+Death&doi=10.3390%2Fijms20102415"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">p53 at the crossroads between different types of HDAC inhibitor-mediated cancer cell death</span></div><div class="casAuthors">Mrakovcic, Maria; Kleinheinz, Johannes; Froehlich, Leopold F.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2415/1-2415/32</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Cancer is a complex genetic and epigenetic-based disease that has developed an armada of mechanisms to escape cell death.  The deregulation of apoptosis and autophagy, which are basic processes essential for normal cellular activity, are commonly encountered during the development of human tumors.  In order to assist the cancer cell in defeating the imbalance between cell growth and cell death, histone deacetylase inhibitors (HDACi) have been employed to reverse epigenetically deregulated gene expression caused by aberrant post-translational protein modifications.  These interfere with histone acetyltransferase- and deacetylase-mediated acetylation of both histone and non-histone proteins, and thereby exert a wide array of HDACi-stimulated cytotoxic effects.  Key determinants of HDACi lethality that interfere with cellular growth in a multitude of tumor cells are apoptosis and autophagy, which are either mutually exclusive or activated in combination.  Here, we compile known mol. signals and pathways involved in the HDACi-triggered induction of apoptosis and autophagy.  Currently, the factors that det. the mode of HDACi-elicited cell death are mostly unclear.  Correspondingly, we also summarized as yet established intertwined mechanisms, in particular with respect to the oncogenic tumor suppressor protein p53, that drive the interplay between apoptosis and autophagy in response to HDACi.  In this context, we also note the significance to det. the presence of functional p53 protein levels in the cancer cell.  The confirmation of the context-dependent function of autophagy will pave the way to improve the benefit from HDACi-mediated cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUKduySxvx97Vg90H21EOLACvtfcHk0ljvDjmmWe3vCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosVGmsg%253D%253D&md5=d74b939314d54d581024c3ef97664733</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.3390%2Fijms20102415&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms20102415%26sid%3Dliteratum%253Aachs%26aulast%3DMrakovcic%26aufirst%3DM.%26aulast%3DKleinheinz%26aufirst%3DJ.%26aulast%3DFr%25C3%25B6hlich%26aufirst%3DL.%2BF.%26atitle%3DP53%2520at%2520the%2520Crossroads%2520between%2520Different%2520Types%2520of%2520Hdac%2520Inhibitor-Mediated%2520Cancer%2520Cell%2520Death%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D20%26spage%3D2415%26doi%3D10.3390%2Fijms20102415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boidol, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornauth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Der Kouwe, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prutsch, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazianka, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gültekin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoermann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayerhoefer, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopfinger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauswirth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panny, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aretin, M.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgarth, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sperr, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonitsch-Klupp, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriggl, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jager, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubicek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staber, P. B.</span></span> <span> </span><span class="NLM_article-title">First-in-Human Response of BCL-2 Inhibitor Venetoclax in T-Cell Prolymphocytic Leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">2499</span>– <span class="NLM_lpage">2503</span>, <span class="refDoi"> DOI: 10.1182/blood-2017-05-785683</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1182%2Fblood-2017-05-785683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=28972014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1erurfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2017&pages=2499-2503&author=B.+Boidolauthor=C.+Kornauthauthor=E.+Van+Der+Kouweauthor=N.+Prutschauthor=L.+Kaziankaauthor=S.+G%C3%BCltekinauthor=G.+Hoermannauthor=M.+E.+Mayerhoeferauthor=G.+Hopfingerauthor=A.+Hauswirthauthor=M.+Pannyauthor=M.-B.+Aretinauthor=B.+Hilgarthauthor=W.+R.+Sperrauthor=P.+Valentauthor=I.+Simonitsch-Kluppauthor=R.+Morigglauthor=P.+Merkelauthor=L.+Kennerauthor=U.+Jagerauthor=S.+Kubicekauthor=P.+B.+Staber&title=First-in-Human+Response+of+BCL-2+Inhibitor+Venetoclax+in+T-Cell+Prolymphocytic+Leukemia&doi=10.1182%2Fblood-2017-05-785683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">First-in-human response of BCL-2 inhibitor venetoclax in T-cell prolymphocytic leukemia</span></div><div class="casAuthors">Boidol, Bernd; Kornauth, Christoph; van der Kouwe, Emiel; Prutsch, Nicole; Kazianka, Lukas; Gultekin, Sinan; Hoermann, Gregor; Mayerhoefer, Marius E.; Hopfinger, Georg; Hauswirth, Alexander; Panny, Michael; Aretin, Marie-Bernadette; Hilgarth, Bernadette; Sperr, Wolfgang R.; Valent, Peter; Simonitsch-Klupp, Ingrid; Moriggl, Richard; Merkel, Olaf; Kenner, Lukas; Jager, Ulrich; Kubicek, Stefan; Staber, Philipp B.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2499-2503</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">1528-0020</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">T-cell prolymphocytic leukemia (T-PLL) is a rare and aggressive T-lymphoid malignancy usually refractory to current treatment strategies and assocd. with short overall survival.  By applying next-generation functional testing of primary patient-derived lymphoma cells using a library of 106 US Food and Drug Administration (FDA)-approved anticancer drugs or compds. currently in clin. development, we set out to identify novel effective treatments for T-PLL patients.  We found that the B-cell lymphoma 2 (BCL-2) inhibitor venetoclax (ABT-199) demonstrated the strongest T-PLL-specific response when comparing individual ex vivo drug response in 86 patients with refractory hematol. malignancies.  Mechanistically, responses to venetoclax correlated with protein expression of BCL-2 but not with expression of the BCL-2 family members myeloid cell leukemia 1 (MCL-1) and BCL-XL in lymphoma cells.  BCL-2 expression was inversely correlated with the expression of MCL-1.  Based on the ex vivo responses, venetoclax treatment was commenced in 2 late-stage refractory T-PLL patients resulting in clin. responses.  Our findings demonstrate first evidence of single-agent activity of venetoclax both ex vivo and in humans, offering a novel agent in T-PLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGCxVs1rSztrVg90H21EOLACvtfcHk0ljvDjmmWe3vCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1erurfI&md5=d24bb98d7284eb6484c9a11134b49837</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1182%2Fblood-2017-05-785683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2017-05-785683%26sid%3Dliteratum%253Aachs%26aulast%3DBoidol%26aufirst%3DB.%26aulast%3DKornauth%26aufirst%3DC.%26aulast%3DVan%2BDer%2BKouwe%26aufirst%3DE.%26aulast%3DPrutsch%26aufirst%3DN.%26aulast%3DKazianka%26aufirst%3DL.%26aulast%3DG%25C3%25BCltekin%26aufirst%3DS.%26aulast%3DHoermann%26aufirst%3DG.%26aulast%3DMayerhoefer%26aufirst%3DM.%2BE.%26aulast%3DHopfinger%26aufirst%3DG.%26aulast%3DHauswirth%26aufirst%3DA.%26aulast%3DPanny%26aufirst%3DM.%26aulast%3DAretin%26aufirst%3DM.-B.%26aulast%3DHilgarth%26aufirst%3DB.%26aulast%3DSperr%26aufirst%3DW.%2BR.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DSimonitsch-Klupp%26aufirst%3DI.%26aulast%3DMoriggl%26aufirst%3DR.%26aulast%3DMerkel%26aufirst%3DP.%26aulast%3DKenner%26aufirst%3DL.%26aulast%3DJager%26aufirst%3DU.%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DStaber%26aufirst%3DP.%2BB.%26atitle%3DFirst-in-Human%2520Response%2520of%2520BCL-2%2520Inhibitor%2520Venetoclax%2520in%2520T-Cell%2520Prolymphocytic%2520Leukemia%26jtitle%3DBlood%26date%3D2017%26volume%3D130%26spage%3D2499%26epage%3D2503%26doi%3D10.1182%2Fblood-2017-05-785683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Souers, A. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leverson, J. D</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boghaert, E. R</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackler, S. L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catron, N. D</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dayton, B. D</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enschede, S. H</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairbrother, W. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D. C S</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hymowitz, S. G</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khaw, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maecker, H. L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, K. C</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, K. D</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitten, M. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nimmer, P. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oleksijew, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, D. C</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, A. W</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seymour, J. F</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, M. L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, G. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tahir, S. K</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendt, M. D</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, J. C</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humerickhouse, R. A</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W</span></span> <span> </span><span class="NLM_article-title">ABT-199, a Potent and Selective BCL-2 Inhibitor, Achieves Antitumor Activity While Sparing Platelets</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">208</span>, <span class="refDoi"> DOI: 10.1038/nm.3048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fnm.3048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=23291630" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjslagtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=202-208&author=A.+J+Souersauthor=J.+D+Leversonauthor=E.+R+Boghaertauthor=S.+L+Acklerauthor=N.+D+Catronauthor=J.+Chenauthor=B.+D+Daytonauthor=H.+Dingauthor=S.+H+Enschedeauthor=W.+J+Fairbrotherauthor=D.+C+S+Huangauthor=S.+G+Hymowitzauthor=S.+Jinauthor=S.+L.+Khawauthor=P.+J+Kovarauthor=L.+T+Lamauthor=J.+Leeauthor=H.+L+Maeckerauthor=K.+C+Marshauthor=K.+D+Masonauthor=M.+J+Mittenauthor=P.+M+Nimmerauthor=A.+Oleksijewauthor=C.+H+Parkauthor=C.-M.+Parkauthor=D.+C+Phillipsauthor=A.+W+Robertsauthor=D.+Sampathauthor=J.+F+Seymourauthor=M.+L+Smithauthor=G.+M+Sullivanauthor=S.+K+Tahirauthor=C.+Tseauthor=M.+D+Wendtauthor=Y.+Xiaoauthor=J.+C+Xueauthor=H.+Zhangauthor=R.+A+Humerickhouseauthor=S.+H+Rosenbergauthor=S.+W+Elmore&title=ABT-199%2C+a+Potent+and+Selective+BCL-2+Inhibitor%2C+Achieves+Antitumor+Activity+While+Sparing+Platelets&doi=10.1038%2Fnm.3048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">ABT-199, a potent and selective BCL-2 inhibitor, achieves antitumor activity while sparing platelets</span></div><div class="casAuthors">Souers, Andrew J.; Leverson, Joel D.; Boghaert, Erwin R.; Ackler, Scott L.; Catron, Nathaniel D.; Chen, Jun; Dayton, Brian D.; Ding, Hong; Enschede, Sari H.; Fairbrother, Wayne J.; Huang, David C. S.; Hymowitz, Sarah G.; Jin, Sha; Khaw, Seong Lin; Kovar, Peter J.; Lam, Lloyd T.; Lee, Jackie; Maecker, Heather L.; Marsh, Kennan C.; Mason, Kylie D.; Mitten, Michael J.; Nimmer, Paul M.; Oleksijew, Anatol; Park, Chang H.; Park, Cheol-Min; Phillips, Darren C.; Roberts, Andrew W.; Sampath, Deepak; Seymour, John F.; Smith, Morey L.; Sullivan, Gerard M.; Tahir, Stephen K.; Tse, Chris; Wendt, Michael D.; Xiao, Yu; Xue, John C.; Zhang, Haichao; Humerickhouse, Rod A.; Rosenberg, Saul H.; Elmore, Steven W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">202-208</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins in the B cell CLL/lymphoma 2 (BCL-2) family are key regulators of the apoptotic process.  This family comprises proapoptotic and prosurvival proteins, and shifting the balance toward the latter is an established mechanism whereby cancer cells evade apoptosis.  The therapeutic potential of directly inhibiting prosurvival proteins was unveiled with the development of navitoclax, a selective inhibitor of both BCL-2 and BCL-2-like 1 (BCL-XL), which has shown clin. efficacy in some BCL-2-dependent hematol. cancers.  However, concomitant on-target thrombocytopenia caused by BCL-XL inhibition limits the efficacy achievable with this agent.  Here we report the re-engineering of navitoclax to create a highly potent, orally bioavailable and BCL-2-selective inhibitor, ABT-199.  This compd. inhibits the growth of BCL-2-dependent tumors in vivo and spares human platelets.  A single dose of ABT-199 in three patients with refractory chronic lymphocytic leukemia resulted in tumor lysis within 24 h.  These data indicate that selective pharmacol. inhibition of BCL-2 shows promise for the treatment of BCL-2-dependent hematol. cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqi7P7koWwJHLVg90H21EOLACvtfcHk0ljvDjmmWe3vCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjslagtQ%253D%253D&md5=6a24b72303eb2062da653f1f61ac74fb</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnm.3048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3048%26sid%3Dliteratum%253Aachs%26aulast%3DSouers%26aufirst%3DA.%2BJ%26aulast%3DLeverson%26aufirst%3DJ.%2BD%26aulast%3DBoghaert%26aufirst%3DE.%2BR%26aulast%3DAckler%26aufirst%3DS.%2BL%26aulast%3DCatron%26aufirst%3DN.%2BD%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DDayton%26aufirst%3DB.%2BD%26aulast%3DDing%26aufirst%3DH.%26aulast%3DEnschede%26aufirst%3DS.%2BH%26aulast%3DFairbrother%26aufirst%3DW.%2BJ%26aulast%3DHuang%26aufirst%3DD.%2BC%2BS%26aulast%3DHymowitz%26aufirst%3DS.%2BG%26aulast%3DJin%26aufirst%3DS.%26aulast%3DKhaw%26aufirst%3DS.%2BL.%26aulast%3DKovar%26aufirst%3DP.%2BJ%26aulast%3DLam%26aufirst%3DL.%2BT%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DMaecker%26aufirst%3DH.%2BL%26aulast%3DMarsh%26aufirst%3DK.%2BC%26aulast%3DMason%26aufirst%3DK.%2BD%26aulast%3DMitten%26aufirst%3DM.%2BJ%26aulast%3DNimmer%26aufirst%3DP.%2BM%26aulast%3DOleksijew%26aufirst%3DA.%26aulast%3DPark%26aufirst%3DC.%2BH%26aulast%3DPark%26aufirst%3DC.-M.%26aulast%3DPhillips%26aufirst%3DD.%2BC%26aulast%3DRoberts%26aufirst%3DA.%2BW%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DSeymour%26aufirst%3DJ.%2BF%26aulast%3DSmith%26aufirst%3DM.%2BL%26aulast%3DSullivan%26aufirst%3DG.%2BM%26aulast%3DTahir%26aufirst%3DS.%2BK%26aulast%3DTse%26aufirst%3DC.%26aulast%3DWendt%26aufirst%3DM.%2BD%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DXue%26aufirst%3DJ.%2BC%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DHumerickhouse%26aufirst%3DR.%2BA%26aulast%3DRosenberg%26aufirst%3DS.%2BH%26aulast%3DElmore%26aufirst%3DS.%2BW%26atitle%3DABT-199%252C%2520a%2520Potent%2520and%2520Selective%2520BCL-2%2520Inhibitor%252C%2520Achieves%2520Antitumor%2520Activity%2520While%2520Sparing%2520Platelets%26jtitle%3DNat.%2520Med.%26date%3D2013%26volume%3D19%26spage%3D202%26epage%3D208%26doi%3D10.1038%2Fnm.3048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davids, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagel, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahl, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puvvada, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerecitano, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kipps, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gressick, L.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Targeting BCL2 with Venetoclax in Relapsed Chronic Lymphocytic Leukemia</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>374</i></span>,  <span class="NLM_fpage">311</span>– <span class="NLM_lpage">322</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa1513257</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1056%2FNEJMoa1513257" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=26639348" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtV2murrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=374&publication_year=2016&pages=311-322&author=A.+W.+Robertsauthor=M.+S.+Davidsauthor=J.+M.+Pagelauthor=B.+S.+Kahlauthor=S.+D.+Puvvadaauthor=J.+F.+Gerecitanoauthor=T.+J.+Kippsauthor=M.+A.+Andersonauthor=J.+R.+Brownauthor=L.+Gressick&title=Targeting+BCL2+with+Venetoclax+in+Relapsed+Chronic+Lymphocytic+Leukemia&doi=10.1056%2FNEJMoa1513257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting BCL2 with venetoclax in relapsed chronic lymphocytic leukemia</span></div><div class="casAuthors">Roberts, Andrew W.; Davids, Matthew S.; Pagel, John M.; Kahl, Brad S.; Puvvada, Soham D.; Gerecitano, John F.; Kipps, Thomas J.; Anderson, Mary Ann; Brown, Jennifer R.; Gressick, Lori; Wong, Shekman; Dunbar, Martin; Zhu, Ming; Desai, Monali B.; Cerri, Elisa; Enschede, Sari Heitner; Humerickhouse, Rod A.; Wierda, William G.; Seymour, John F.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">374</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">311-322</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">BACKGROUND New treatments have improved outcomes for patients with relapsed chronic lymphocytic leukemia (CLL), but complete remissions remain uncommon.  Venetoclax has a distinct mechanism of action; it targets BCL2, a protein central to the survival of CLL cells.  METHODS We conducted a phase 1 dose-escalation study of daily oral venetoclax in patients with relapsed or refractory CLL or small lymphocytic lymphoma (SLL) to assess safety, pharmacokinetic profile, and efficacy.  In the dose-escalation phase, 56 patients received active treatment in one of eight dose groups that ranged from 150 to 1200 mg per day.  In an expansion cohort, 60 addnl. patients were treated with a weekly stepwise ramp-up in doses as high as 400 mg per day.  RESULTS The majority of the study patients had received multiple previous treatments, and 89% had poor prognostic clin. or genetic features.  Venetoclax was active at all dose levels.  Clin. tumor lysis syndrome occurred in 3 of 56 patients in the dose-escalation cohort, with one death.  After adjustments to the dose-escalation schedule, clin. tumor lysis syndrome did not occur in any of the 60 patients in the expansion cohort.  Other toxic effects included mild diarrhea (in 52% of the patients), upper respiratory tract infection (in 48%), nausea (in 47%), and grade 3 or 4 neutropenia (in 41%).  A max. tolerated dose was not identified.  Among the 116 patients who received venetoclax, 92 (79%) had a response.  Response rates ranged from 71 to 79% among patients in subgroups with an adverse prognosis, including those with resistance to fludarabine, those with chromosome 17p deletions (deletion 17p CLL), and those with unmutated IGHV.  Complete remissions occurred in 20% of the patients, including 5% who had no minimal residual disease on flow cytometry.  The 15-mo progression-free survival est. for the 400-mg dose groups was 69%.  CONCLUSIONS Selective targeting of BCL2 with venetoclax had a manageable safety profile and induced substantial responses in patients with relapsed CLL or SLL, including those with poor prognostic features.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHQgYgnKl88rVg90H21EOLACvtfcHk0lg76EIUB4vIVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtV2murrP&md5=b371a37b76ee88cfc6ac47ea0eeea0e6</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1513257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1513257%26sid%3Dliteratum%253Aachs%26aulast%3DRoberts%26aufirst%3DA.%2BW.%26aulast%3DDavids%26aufirst%3DM.%2BS.%26aulast%3DPagel%26aufirst%3DJ.%2BM.%26aulast%3DKahl%26aufirst%3DB.%2BS.%26aulast%3DPuvvada%26aufirst%3DS.%2BD.%26aulast%3DGerecitano%26aufirst%3DJ.%2BF.%26aulast%3DKipps%26aufirst%3DT.%2BJ.%26aulast%3DAnderson%26aufirst%3DM.%2BA.%26aulast%3DBrown%26aufirst%3DJ.%2BR.%26aulast%3DGressick%26aufirst%3DL.%26atitle%3DTargeting%2520BCL2%2520with%2520Venetoclax%2520in%2520Relapsed%2520Chronic%2520Lymphocytic%2520Leukemia%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2016%26volume%3D374%26spage%3D311%26epage%3D322%26doi%3D10.1056%2FNEJMoa1513257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rhodes, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshpande, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varambally, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrette, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pander, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaiyan, A. M.</span></span> <span> </span><span class="NLM_article-title">ONCOMINE: A Cancer Microarray Database and Integrated Data-Mining Platform</span>. <i>Neoplasia.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">6</span>, <span class="refDoi"> DOI: 10.1016/S1476-5586(04)80047-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1016%2FS1476-5586%2804%2980047-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=15068665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjslSgtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2004&pages=1-6&author=D.+R.+Rhodesauthor=J.+Yuauthor=K.+Shankerauthor=N.+Deshpandeauthor=R.+Varamballyauthor=D.+Ghoshauthor=T.+Barretteauthor=A.+Panderauthor=A.+M.+Chinnaiyan&title=ONCOMINE%3A+A+Cancer+Microarray+Database+and+Integrated+Data-Mining+Platform&doi=10.1016%2FS1476-5586%2804%2980047-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">ONCOMINE: a cancer microarray database and integrated data-mining platform</span></div><div class="casAuthors">Rhodes, Daniel R.; Yu, Jianjun; Shanker, K.; Deshpande, Nandan; Varambally, Radhika; Ghosh, Debashis; Barrette, Terrence; Pandey, Akhilesh; Chinnaiyan, Arul M.</div><div class="citationInfo"><span class="NLM_cas:title">Neoplasia (Ann Arbor, MI, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-6</span>CODEN:
                <span class="NLM_cas:coden">NEOPFL</span>;
        ISSN:<span class="NLM_cas:issn">1522-8002</span>.
    
            (<span class="NLM_cas:orgname">Neoplasia Press Inc.</span>)
        </div><div class="casAbstract">DNA microarray technol. has led to an explosion of oncogenomic analyses, generating a wealth of data and uncovering the complex gene expression patterns of cancer.  Unfortunately, due to the lack of a unifying bioinformatic resource, the majority of these data sit stagnant and disjointed following publication, massively underutilized by the cancer research community.  Here, we present ONCOMINE, a cancer microarray database and web-based data-mining platform aimed at facilitating discovery from genome-wide expression analyses.  To date, ONCOMINE contains 65 gene expression datasets comprising nearly 48 million gene expression measurements form over 4700 microarray expts.  Differential expression analyses comparing most major types of cancer with resp. normal tissues as well as a variety of cancer subtypes and clin.-based and pathol.-based analyses are available for exploration.  Data can be queried and visualized for a selected gene across all analyses or for multiple genes in a selected anal.  Furthermore, gene sets can be limited to clin. important annotations including secreted, kinase, membrane, and known gene-drug target pairs to facilitate the discovery of novel biomarkers and therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUXX6_gcnyLrVg90H21EOLACvtfcHk0lg76EIUB4vIVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjslSgtrY%253D&md5=84d851435b55438b4a6ecd11ab2cd5b3</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2FS1476-5586%2804%2980047-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1476-5586%252804%252980047-2%26sid%3Dliteratum%253Aachs%26aulast%3DRhodes%26aufirst%3DD.%2BR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DShanker%26aufirst%3DK.%26aulast%3DDeshpande%26aufirst%3DN.%26aulast%3DVarambally%26aufirst%3DR.%26aulast%3DGhosh%26aufirst%3DD.%26aulast%3DBarrette%26aufirst%3DT.%26aulast%3DPander%26aufirst%3DA.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DONCOMINE%253A%2520A%2520Cancer%2520Microarray%2520Database%2520and%2520Integrated%2520Data-Mining%2520Platform%26jtitle%3DNeoplasia.%26date%3D2004%26volume%3D6%26spage%3D1%26epage%3D6%26doi%3D10.1016%2FS1476-5586%2804%2980047-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gawel, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shouksmith, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raouf, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nawar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toutah, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukhari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manaswiyoungkul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olaoye, O. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israelian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radu, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabral, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedighi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Araujo, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span> <span> </span><span class="NLM_article-title">PTG-0861: A Novel HDAC6-Selective Inhibitor as a Therapeutic Strategy in Acute Myeloid Leukaemia</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>201</i></span>,  <span class="NLM_fpage">112411</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112411</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1016%2Fj.ejmech.2020.112411" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=32615502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlSht73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=201&publication_year=2020&pages=112411&author=J.+M.+Gawelauthor=A.+E.+Shouksmithauthor=Y.+S.+Raoufauthor=N.+Nawarauthor=K.+Toutahauthor=S.+Bukhariauthor=P.+Manaswiyoungkulauthor=O.+O.+Olaoyeauthor=J.+Israelianauthor=T.+B.+Raduauthor=A.+D.+Cabralauthor=D.+Sinaauthor=A.+Sedighiauthor=E.+D.+de+Araujoauthor=P.+T.+Gunning&title=PTG-0861%3A+A+Novel+HDAC6-Selective+Inhibitor+as+a+Therapeutic+Strategy+in+Acute+Myeloid+Leukaemia&doi=10.1016%2Fj.ejmech.2020.112411"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">PTG-0861: A novel HDAC6-selective inhibitor as a therapeutic strategy in acute myeloid leukaemia</span></div><div class="casAuthors">Gawel, Justyna M.; Shouksmith, Andrew E.; Raouf, Yasir S.; Nawar, Nabanita; Toutah, Krimo; Bukhari, Shazreh; Manaswiyoungkul, Pimyupa; Olaoye, Olasunkanmi O.; Israelian, Johan; Radu, Tudor B.; Cabral, Aaron D.; Sina, Diana; Sedighi, Abootaleb; de Araujo, Elvin D.; Gunning, Patrick T.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">201</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112411</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Dysregulated Histone Deacetylase (HDAC) activity across multiple human pathologies have highlighted this family of epigenetic enzymes as crit. druggable targets, amenable to small mol. intervention.  While efficacious, current approaches using non-selective HDAC inhibitors (HDACi) have been shown to cause a range of undesirable clin. toxicities.  To circumvent this, recent efforts have focused on the design of highly selective HDACi as a novel therapeutic strategy.  Beyond roles in regulating transcription, the unique HDAC6 (with two catalytic domains) regulates the deacetylation of α-tubulin; promoting growth factor-controlled cell motility, cell division, and metastatic hallmarks.  Recent studies have linked aberrant HDAC6 function in various hematol. cancers including acute myeloid leukemia and multiple myeloma.  Herein, we report the discovery, in vitro characterization, and biol. evaluation of PTG-0861 (JG-265), a novel HDAC6-selective inhibitor with strong isoenzyme-selectivity (∼36x ) and low nanomolar potency (IC50 = 5.92 nM) against HDAC6.  This selectivity profile was rationalized via in silico docking studies and also obsd. in cellulo through cellular target engagement.  Moreover, PTG-0861 achieved relevant potency against several blood cancer cell lines (e.g. MV4-11, MM1S), while showing limited cytotoxicity against non-malignant cells (e.g. NHF, HUVEC) and CD-1 mice.  In examg. compd. stability and cellular permeability, PTG-0861 revealed a promising in vitro pharmacokinetic (PK) profile.  Altogether, in this study we identified a novel and potent HDAC6-selective inhibitor (∼4x more selective than current clin. stds. - citarinostat, ricolinostat), which achieves cellular target engagement, efficacy in hematol. cancer cells with a promising safety profile and in vitro PK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGUpACNMyWt7Vg90H21EOLACvtfcHk0liu6OV2111kwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlSht73E&md5=c5b2e399be96a785b52019f04e1c08b8</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112411%26sid%3Dliteratum%253Aachs%26aulast%3DGawel%26aufirst%3DJ.%2BM.%26aulast%3DShouksmith%26aufirst%3DA.%2BE.%26aulast%3DRaouf%26aufirst%3DY.%2BS.%26aulast%3DNawar%26aufirst%3DN.%26aulast%3DToutah%26aufirst%3DK.%26aulast%3DBukhari%26aufirst%3DS.%26aulast%3DManaswiyoungkul%26aufirst%3DP.%26aulast%3DOlaoye%26aufirst%3DO.%2BO.%26aulast%3DIsraelian%26aufirst%3DJ.%26aulast%3DRadu%26aufirst%3DT.%2BB.%26aulast%3DCabral%26aufirst%3DA.%2BD.%26aulast%3DSina%26aufirst%3DD.%26aulast%3DSedighi%26aufirst%3DA.%26aulast%3Dde%2BAraujo%26aufirst%3DE.%2BD.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26atitle%3DPTG-0861%253A%2520A%2520Novel%2520HDAC6-Selective%2520Inhibitor%2520as%2520a%2520Therapeutic%2520Strategy%2520in%2520Acute%2520Myeloid%2520Leukaemia%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D201%26spage%3D112411%26doi%3D10.1016%2Fj.ejmech.2020.112411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shouksmith, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gawel, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nawar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raouf, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukhari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manaswiyoungkul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olaoye, O. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabral, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedighi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Araujo, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span> <span> </span><span class="NLM_article-title">Class I/IIb-Selective HDAC Inhibitor Exhibits Oral Bioavailability and Therapeutic Efficacy in Acute Myeloid Leukemia</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00471</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00471" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlygtLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=56-64&author=A.+E.+Shouksmithauthor=J.+M.+Gawelauthor=N.+Nawarauthor=D.+Sinaauthor=Y.+S.+Raoufauthor=S.+Bukhariauthor=L.+Heauthor=A.+E.+Johnsauthor=P.+Manaswiyoungkulauthor=O.+O.+Olaoyeauthor=A.+D.+Cabralauthor=A.+Sedighiauthor=E.+D.+de+Araujoauthor=P.+T.+Gunning&title=Class+I%2FIIb-Selective+HDAC+Inhibitor+Exhibits+Oral+Bioavailability+and+Therapeutic+Efficacy+in+Acute+Myeloid+Leukemia&doi=10.1021%2Facsmedchemlett.9b00471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Class I/IIb-Selective HDAC Inhibitor Exhibits Oral Bioavailability and Therapeutic Efficacy in Acute Myeloid Leukemia</span></div><div class="casAuthors">Shouksmith, Andrew E.; Gawel, Justyna M.; Nawar, Nabanita; Sina, Diana; Raouf, Yasir S.; Bukhari, Shazreh; He, Liying; Johns, Alexandra E.; Manaswiyoungkul, Pimyupa; Olaoye, Olasunkanmi O.; Cabral, Aaron D.; Sedighi, Abootaleb; de Araujo, Elvin D.; Gunning, Patrick T.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-64</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The HDAC inhibitor 4-tert-butyl-N-(4-(hydroxycarbamoyl)phenyl)benzamide (AES-350, I) was identified as a promising preclin. candidate for the treatment of acute myeloid leukemia (AML), an aggressive malignancy with a meagre 24% 5-yr survival rate.  Through screening of low-mol.-wt. analogs derived from the previously discovered novel HDAC inhibitor, AES-135 (II), compd. I demonstrated greater HDAC isoform selectivity, higher cytotoxicity in MV4-11 cells, an improved therapeutic window, and more efficient absorption through cellular and lipid membranes.  Compd. I also demonstrated improved oral bioavailability compared to SAHA in mouse models.  A broad spectrum of expts., including FACS, ELISA, and Western blotting, were performed to support our hypothesis that I dose-dependently triggers apoptosis in AML cells through HDAC inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8ikqWn7KZc7Vg90H21EOLACvtfcHk0liu6OV2111kwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlygtLrK&md5=3c21f11cda2a27e00f30a7cd3c700785</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00471%26sid%3Dliteratum%253Aachs%26aulast%3DShouksmith%26aufirst%3DA.%2BE.%26aulast%3DGawel%26aufirst%3DJ.%2BM.%26aulast%3DNawar%26aufirst%3DN.%26aulast%3DSina%26aufirst%3DD.%26aulast%3DRaouf%26aufirst%3DY.%2BS.%26aulast%3DBukhari%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DL.%26aulast%3DJohns%26aufirst%3DA.%2BE.%26aulast%3DManaswiyoungkul%26aufirst%3DP.%26aulast%3DOlaoye%26aufirst%3DO.%2BO.%26aulast%3DCabral%26aufirst%3DA.%2BD.%26aulast%3DSedighi%26aufirst%3DA.%26aulast%3Dde%2BAraujo%26aufirst%3DE.%2BD.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26atitle%3DClass%2520I%252FIIb-Selective%2520HDAC%2520Inhibitor%2520Exhibits%2520Oral%2520Bioavailability%2520and%2520Therapeutic%2520Efficacy%2520in%2520Acute%2520Myeloid%2520Leukemia%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D56%26epage%3D64%26doi%3D10.1021%2Facsmedchemlett.9b00471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olaoye, O. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nawar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geletu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedighi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukhari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raouf, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manaswiyoungkul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdogan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdeldayem, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabral, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toutah, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shouksmith, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gawel, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israelian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radu, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kachhiyapatel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Araujo, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span> <span> </span><span class="NLM_article-title">Unique Molecular Interaction with the Histone Deacetylase 6 Catalytic Tunnel: Crystallographic and Biological Characterization of a Model Chemotype</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">2691</span>– <span class="NLM_lpage">2704</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01922</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01922" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BB3MXjs12hu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2021&pages=2691-2704&author=O.+O.+Olaoyeauthor=P.+R.+Watsonauthor=N.+Nawarauthor=M.+Geletuauthor=A.+Sedighiauthor=S.+Bukhariauthor=Y.+S.+Raoufauthor=P.+Manaswiyoungkulauthor=F.+Erdoganauthor=A.+Abdeldayemauthor=A.+D.+Cabralauthor=M.+M.+Hassanauthor=K.+Toutahauthor=A.+E.+Shouksmithauthor=J.+M.+Gawelauthor=J.+Israelianauthor=T.+B.+Raduauthor=N.+Kachhiyapatelauthor=E.+D.+de+Araujoauthor=D.+W.+Christiansonauthor=P.+T.+Gunning&title=Unique+Molecular+Interaction+with+the+Histone+Deacetylase+6+Catalytic+Tunnel%3A+Crystallographic+and+Biological+Characterization+of+a+Model+Chemotype&doi=10.1021%2Facs.jmedchem.0c01922"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Unique Molecular Interaction with the Histone Deacetylase 6 Catalytic Tunnel: Crystallographic and Biological Characterization of a Model Chemotype</span></div><div class="casAuthors">Olaoye, Olasunkanmi O.; Watson, Paris R.; Nawar, Nabanita; Geletu, Mulu; Sedighi, Abootaleb; Bukhari, Shazreh; Raouf, Yasir S.; Manaswiyoungkul, Pimyupa; Erdogan, Fettah; Abdeldayem, Ayah; Cabral, Aaron D.; Hassan, Muhammad Murtaza; Toutah, Krimo; Shouksmith, Andrew E.; Gawel, Justyna M.; Israelian, Johan; Radu, Tudor B.; Kachhiyapatel, Niyati; de Araujo, Elvin D.; Christianson, David W.; Gunning, Patrick T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2691-2704</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone deacetylase 6 (HDAC6) is involved in multiple regulatory processes, ranging from cellular stress to intracellular transport.  Inhibition of aberrant HDAC6 activity in several cancers and neurol. diseases has been shown to be efficacious in both preclin. and clin. studies.  While selective HDAC6 targeting has been pursued as an alternative to pan-HDAC drugs, identifying truly selective mol. templates has not been trivial.  Herein, we report a structure-activity relationship study yielding TO-317, which potently binds HDAC6 catalytic domain 2 (Ki = 0.7 nM) and inhibits the enzyme function (IC50 = 2 nM).  TO-317 exhibits 158-fold selectivity for HDAC6 over other HDAC isoenzymes by binding the catalytic Zn2+ and, uniquely, making a never seen before direct hydrogen bond with the Zn2+ coordinating residue, His614.  This novel structural motif targeting the second-sphere His614 interaction, obsd. in a 1.84 Å resoln. crystal structure with drHDAC6 from zebrafish, can provide new pharmacophores for identifying enthalpically driven, high-affinity, HDAC6-selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZoHL-PIpylbVg90H21EOLACvtfcHk0liu6OV2111kwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXjs12hu74%253D&md5=1fc291767413b2b65282c6a7009cb17e</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01922%26sid%3Dliteratum%253Aachs%26aulast%3DOlaoye%26aufirst%3DO.%2BO.%26aulast%3DWatson%26aufirst%3DP.%2BR.%26aulast%3DNawar%26aufirst%3DN.%26aulast%3DGeletu%26aufirst%3DM.%26aulast%3DSedighi%26aufirst%3DA.%26aulast%3DBukhari%26aufirst%3DS.%26aulast%3DRaouf%26aufirst%3DY.%2BS.%26aulast%3DManaswiyoungkul%26aufirst%3DP.%26aulast%3DErdogan%26aufirst%3DF.%26aulast%3DAbdeldayem%26aufirst%3DA.%26aulast%3DCabral%26aufirst%3DA.%2BD.%26aulast%3DHassan%26aufirst%3DM.%2BM.%26aulast%3DToutah%26aufirst%3DK.%26aulast%3DShouksmith%26aufirst%3DA.%2BE.%26aulast%3DGawel%26aufirst%3DJ.%2BM.%26aulast%3DIsraelian%26aufirst%3DJ.%26aulast%3DRadu%26aufirst%3DT.%2BB.%26aulast%3DKachhiyapatel%26aufirst%3DN.%26aulast%3Dde%2BAraujo%26aufirst%3DE.%2BD.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26atitle%3DUnique%2520Molecular%2520Interaction%2520with%2520the%2520Histone%2520Deacetylase%25206%2520Catalytic%2520Tunnel%253A%2520Crystallographic%2520and%2520Biological%2520Characterization%2520of%2520a%2520Model%2520Chemotype%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2021%26volume%3D64%26spage%3D2691%26epage%3D2704%26doi%3D10.1021%2Facs.jmedchem.0c01922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shouksmith, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimard, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gawel, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raouf, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geletu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger-Becvar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Araujo, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luchman, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heaton, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakhshinyan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adile, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venugopal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Hare, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deininger, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konieczny, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishel, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterization of AES-135, a Hydroxamic Acid-Based HDAC Inhibitor That Prolongs Survival in an Orthotopic Mouse Model of Pancreatic Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2651</span>– <span class="NLM_lpage">2665</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01957</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01957" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjt1Slu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2651-2665&author=A.+E.+Shouksmithauthor=F.+Shahauthor=M.+L.+Grimardauthor=J.+M.+Gawelauthor=Y.+S.+Raoufauthor=M.+Geletuauthor=A.+Berger-Becvarauthor=E.+D.+De+Araujoauthor=H.+A.+Luchmanauthor=W.+L.+Heatonauthor=D.+Bakhshinyanauthor=A.+A.+Adileauthor=C.+Venugopalauthor=T.+O%E2%80%99Hareauthor=M.+W.+Deiningerauthor=S.+K.+Singhauthor=S.+F.+Koniecznyauthor=S.+Weissauthor=M.+L.+Fishelauthor=P.+T.+Gunning&title=Identification+and+Characterization+of+AES-135%2C+a+Hydroxamic+Acid-Based+HDAC+Inhibitor+That+Prolongs+Survival+in+an+Orthotopic+Mouse+Model+of+Pancreatic+Cancer&doi=10.1021%2Facs.jmedchem.8b01957"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of AES-135, a hydroxamic acid-based HDAC inhibitor that prolongs survival in an orthotopic mouse model of pancreatic cancer</span></div><div class="casAuthors">Shouksmith, Andrew E.; Shah, Fenil; Grimard, Michelle L.; Gawel, Justyna M.; Raouf, Yasir S.; Geletu, Mulu; Berger-Becvar, Angelika; de Araujo, Elvin D.; Luchman, H. Artee; Heaton, William L.; Bakhshinyan, David; Adile, Ashley A.; Venugopal, Chitra; O'Hare, Thomas; Deininger, Michael W.; Singh, Sheila K.; Konieczny, Stephen F.; Weiss, Samuel; Fishel, Melissa L.; Gunning, Patrick T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2651-2665</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pancreatic ductal adenocarcinoma (PDAC) is an aggressive, incurable cancer with a 20% 1 yr survival rate.  While std.-of-care therapy can prolong life in a small fraction of cases, PDAC is inherently resistant to current treatments, and novel therapies are urgently required.  Histone deacetylase (HDAC) inhibitors are effective in killing pancreatic cancer cells in in vitro PDAC studies, and although there are a few clin. studies investigating combination therapy including HDAC inhibitors, no HDAC drug or combination therapy with an HDAC drug has been approved for the treatment of PDAC.  We developed an inhibitor of HDACs, AES-135, that exhibits nanomolar inhibitory activity against HDAC3, HDAC6, and HDAC11 in biochem. assays.  In a three-dimensional coculture model, AES-135 kills low-passage patient-derived tumor spheroids selectively over surrounding cancer-assocd. fibroblasts and has excellent pharmacokinetic properties in vivo.  In an orthotopic murine model of pancreatic cancer, AES-135 prolongs survival significantly, therefore representing a candidate for further preclin. testing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsx6K4JeAb-bVg90H21EOLACvtfcHk0ljoF9e2KnTA-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjt1Slu78%253D&md5=2b90becae883cc5552a701a614d8c452</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01957&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01957%26sid%3Dliteratum%253Aachs%26aulast%3DShouksmith%26aufirst%3DA.%2BE.%26aulast%3DShah%26aufirst%3DF.%26aulast%3DGrimard%26aufirst%3DM.%2BL.%26aulast%3DGawel%26aufirst%3DJ.%2BM.%26aulast%3DRaouf%26aufirst%3DY.%2BS.%26aulast%3DGeletu%26aufirst%3DM.%26aulast%3DBerger-Becvar%26aufirst%3DA.%26aulast%3DDe%2BAraujo%26aufirst%3DE.%2BD.%26aulast%3DLuchman%26aufirst%3DH.%2BA.%26aulast%3DHeaton%26aufirst%3DW.%2BL.%26aulast%3DBakhshinyan%26aufirst%3DD.%26aulast%3DAdile%26aufirst%3DA.%2BA.%26aulast%3DVenugopal%26aufirst%3DC.%26aulast%3DO%25E2%2580%2599Hare%26aufirst%3DT.%26aulast%3DDeininger%26aufirst%3DM.%2BW.%26aulast%3DSingh%26aufirst%3DS.%2BK.%26aulast%3DKonieczny%26aufirst%3DS.%2BF.%26aulast%3DWeiss%26aufirst%3DS.%26aulast%3DFishel%26aufirst%3DM.%2BL.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520AES-135%252C%2520a%2520Hydroxamic%2520Acid-Based%2520HDAC%2520Inhibitor%2520That%2520Prolongs%2520Survival%2520in%2520an%2520Orthotopic%2520Mouse%2520Model%2520of%2520Pancreatic%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2651%26epage%3D2665%26doi%3D10.1021%2Facs.jmedchem.8b01957" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bergman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Villarroel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villagra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sotomayor, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">Selective Histone Deacetylase 6 Inhibitors Bearing Substituted Urea Linkers Inhibit Melanoma Cell Growth</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9891</span>– <span class="NLM_lpage">9899</span>, <span class="refDoi"> DOI: 10.1021/jm301098e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301098e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9891-9899&author=J.+A.+Bergmanauthor=K.+Woanauthor=P.+Perez-Villarroelauthor=A.+Villagraauthor=E.+M.+Sotomayorauthor=A.+P.+Kozikowski&title=Selective+Histone+Deacetylase+6+Inhibitors+Bearing+Substituted+Urea+Linkers+Inhibit+Melanoma+Cell+Growth&doi=10.1021%2Fjm301098e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Histone Deacetylase 6 Inhibitors Bearing Substituted Urea Linkers Inhibit Melanoma Cell Growth</span></div><div class="casAuthors">Bergman, Joel A.; Woan, Karrune; Perez-Villarroel, Patricio; Villagra, Alejandro; Sotomayor, Eduardo M.; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9891-9899</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The incidence of malignant melanoma has dramatically increased in recent years thus requiring the need for improved therapeutic strategies.  In our efforts to design selective histone deactylase inhibitors (HDACI), we discovered that the aryl urea 1 is a modestly potent yet nonselective inhibitor.  Structure-activity relationship studies revealed that adding substituents to the nitrogen atom of the urea so as to generate compds. bearing a branched linker group results in increased potency and selectivity for HDAC6.  Compd. 5g shows low nanomolar inhibitory potency against HDAC6 and a selectivity of ∼600-fold relative to the inhibition of HDAC1.  These HDACIs were evaluated for their ability to inhibit the growth of B16 melanoma cells with the most potent and selective HDAC6I being found to decrease tumor cell growth.  To the best of our knowledge, this work constitutes the first report of HDAC6-selective inhibitors that possess antiproliferative effects against melanoma cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTkTtPLqvwsrVg90H21EOLACvtfcHk0ljoF9e2KnTA-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsrbJ&md5=d6be4c0efb3922fc35d814ba00ae9ed1</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm301098e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301098e%26sid%3Dliteratum%253Aachs%26aulast%3DBergman%26aufirst%3DJ.%2BA.%26aulast%3DWoan%26aufirst%3DK.%26aulast%3DPerez-Villarroel%26aufirst%3DP.%26aulast%3DVillagra%26aufirst%3DA.%26aulast%3DSotomayor%26aufirst%3DE.%2BM.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DSelective%2520Histone%2520Deacetylase%25206%2520Inhibitors%2520Bearing%2520Substituted%2520Urea%2520Linkers%2520Inhibit%2520Melanoma%2520Cell%2520Growth%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9891%26epage%3D9899%26doi%3D10.1021%2Fjm301098e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Woan, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lienlaf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Villaroel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrios, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahakian, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canales, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marante, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malley, K. S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seto, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinilla-Ibarz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarnaik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Celis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sotomayor, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villagra, A.</span></span> <span> </span><span class="NLM_article-title">Targeting Histone Deacetylase 6 Mediates a Dual Anti-Melanoma Effect: Enhanced Antitumor Immunity and Impaired Cell Proliferation</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1447</span>– <span class="NLM_lpage">1457</span>, <span class="refDoi"> DOI: 10.1016/j.molonc.2015.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1016%2Fj.molonc.2015.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=25957812" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsF2gtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=1447-1457&author=K.+V.+Woanauthor=M.+Lienlafauthor=P.+Perez-Villaroelauthor=C.+Leeauthor=F.+Chengauthor=T.+Knoxauthor=D.+M.+Woodsauthor=K.+Barriosauthor=J.+Powersauthor=E.+Sahakianauthor=H.+W.+Wangauthor=J.+Canalesauthor=D.+Maranteauthor=K.+S.+M.+Malleyauthor=J.+Bergmanauthor=E.+Setoauthor=A.+Kozikowskiauthor=J.+Pinilla-Ibarzauthor=A.+Sarnaikauthor=E.+Celisauthor=J.+Weberauthor=E.+M.+Sotomayorauthor=A.+Villagra&title=Targeting+Histone+Deacetylase+6+Mediates+a+Dual+Anti-Melanoma+Effect%3A+Enhanced+Antitumor+Immunity+and+Impaired+Cell+Proliferation&doi=10.1016%2Fj.molonc.2015.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting histone deacetylase 6 mediates a dual anti-melanoma effect: Enhanced antitumor immunity and impaired cell proliferation</span></div><div class="casAuthors">Woan, K. V.; Lienlaf, M.; Perez-Villaroel, P.; Lee, C.; Cheng, F.; Knox, T.; Woods, D. M.; Barrios, K.; Powers, J.; Sahakian, E.; Wang, H. W.; Canales, J.; Marante, D.; Smalley, K. S. M.; Bergman, J.; Seto, E.; Kozikowski, A.; Pinilla-Ibarz, J.; Sarnaik, A.; Celis, E.; Weber, J.; Sotomayor, E. M.; Villagra, A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1447-1457</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-7891</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The median survival for metastatic melanoma is in the realm of 8-16 mo and there are few therapies that offer significant improvement in overall survival.  One of the recent advances in cancer treatment focuses on epigenetic modifiers to alter the survivability and immunogenicity of cancer cells.  Our group and others have previously demonstrated that pan-HDAC inhibitors induce apoptosis, cell cycle arrest and changes in the immunogenicity of melanoma cells.  Here we interrogated specific HDACs which may be responsible for this effect.  We found that both genetic abrogation and pharmacol. inhibition of HDAC6 decreases in vitro proliferation and induces G1 arrest of melanoma cell lines without inducing apoptosis.  Moreover, targeting this mol. led to an important upregulation in the expression of tumor assocd. antigens and MHC class I, suggesting a potential improvement in the immunogenicity of these cells.  Of note, this anti-melanoma activity was operative regardless of mutational status of the cells.  These effects translated into a pronounced delay of in vivo melanoma tumor growth which was, at least in part, dependent on intact immunity as evidenced by the restoration of tumor growth after CD4+ and CD8+ depletion.  Given our findings, we provide the initial rationale for the further development of selective HDAC6 inhibitors as potential therapeutic anti-melanoma agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDIQ5LKbZF8bVg90H21EOLACvtfcHk0ljoF9e2KnTA-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsF2gtLw%253D&md5=7534df9db01f52a71f97f0e112db37cc</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.molonc.2015.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molonc.2015.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DWoan%26aufirst%3DK.%2BV.%26aulast%3DLienlaf%26aufirst%3DM.%26aulast%3DPerez-Villaroel%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DCheng%26aufirst%3DF.%26aulast%3DKnox%26aufirst%3DT.%26aulast%3DWoods%26aufirst%3DD.%2BM.%26aulast%3DBarrios%26aufirst%3DK.%26aulast%3DPowers%26aufirst%3DJ.%26aulast%3DSahakian%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DH.%2BW.%26aulast%3DCanales%26aufirst%3DJ.%26aulast%3DMarante%26aufirst%3DD.%26aulast%3DMalley%26aufirst%3DK.%2BS.%2BM.%26aulast%3DBergman%26aufirst%3DJ.%26aulast%3DSeto%26aufirst%3DE.%26aulast%3DKozikowski%26aufirst%3DA.%26aulast%3DPinilla-Ibarz%26aufirst%3DJ.%26aulast%3DSarnaik%26aufirst%3DA.%26aulast%3DCelis%26aufirst%3DE.%26aulast%3DWeber%26aufirst%3DJ.%26aulast%3DSotomayor%26aufirst%3DE.%2BM.%26aulast%3DVillagra%26aufirst%3DA.%26atitle%3DTargeting%2520Histone%2520Deacetylase%25206%2520Mediates%2520a%2520Dual%2520Anti-Melanoma%2520Effect%253A%2520Enhanced%2520Antitumor%2520Immunity%2520and%2520Impaired%2520Cell%2520Proliferation%26jtitle%3DMol.%2520Oncol.%26date%3D2015%26volume%3D9%26spage%3D1447%26epage%3D1457%26doi%3D10.1016%2Fj.molonc.2015.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanschagrin, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span> <span> </span><span class="NLM_article-title">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6177</span>– <span class="NLM_lpage">6196</span>, <span class="refDoi"> DOI: 10.1021/jm051256o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+Precision+Glide%3A+Docking+and+Scoring+Incorporating+a+Model+of+Hydrophobic+Enclosure+for+Protein-Ligand+Complexes&doi=10.1021%2Fjm051256o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span></div><div class="casAuthors">Friesner, Richard A.; Murphy, Robert B.; Repasky, Matthew P.; Frye, Leah L.; Greenwood, Jeremy R.; Halgren, Thomas A.; Sanschagrin, Paul C.; Mainz, Daniel T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6177-6196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel scoring function to est. protein-ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol.  In addn. to unique water desolvation energy terms, protein-ligand structural motifs leading to enhanced binding affinity are included:(1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral-neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged-charged hydrogen bonds.  The XP scoring function and docking protocol have been developed to reproduce exptl. binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, resp.) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance.  Enrichment results demonstrate the importance of the novel XP mol. recognition and water scoring in sepg. active and inactive ligands and avoiding false positives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZiwN12vaY7Vg90H21EOLACvtfcHk0lhaHcoUy4CPYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D&md5=ea428c82ead0d8c27f8c1a7b694a1edf</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520Precision%2520Glide%253A%2520Docking%2520and%2520Scoring%2520Incorporating%2520a%2520Model%2520of%2520Hydrophobic%2520Enclosure%2520for%2520Protein-Ligand%2520Complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196%26doi%3D10.1021%2Fjm051256o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 2. Enrichment Factors in Database Screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+2.+Enrichment+Factors+in+Database+Screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lhaHcoUy4CPYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25202.%2520Enrichment%2520Factors%2520in%2520Database%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+1.+Method+and+Assessment+of+Docking+Accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lhGq0TyISBHhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25201.%2520Method%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Padige, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negmeldin, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pflum, M. K. H.</span></span> <span> </span><span class="NLM_article-title">Development of an ELISA-Based HDAC Activity Assay for Characterization of Isoform-Selective Inhibitors</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1277</span>– <span class="NLM_lpage">1285</span>, <span class="refDoi"> DOI: 10.1177/1087057115598118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1177%2F1087057115598118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=26232305" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC28XlvVCktLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=1277-1285&author=G.+Padigeauthor=A.+T.+Negmeldinauthor=M.+K.+H.+Pflum&title=Development+of+an+ELISA-Based+HDAC+Activity+Assay+for+Characterization+of+Isoform-Selective+Inhibitors&doi=10.1177%2F1087057115598118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Development of an ELISA-based HDAC activity assay for characterization of isoform-selective inhibitors</span></div><div class="casAuthors">Padige, Geetha; Negmeldin, Ahmed T.; Pflum, Mary Kay H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1277-1285</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Histone deacetylase (HDAC) proteins are promising targets for cancer treatment, with several HDAC inhibitors used clin. as anticancer drugs.  Most HDAC inhibitors nonspecifically interact with all or many of the 11 HDAC isoforms.  Isoform-selective HDAC inhibitors would be useful tools to dissect the individual functions of HDAC proteins in cancer formation, in addn. to potentially displaying effective anticancer properties.  We report here a robust HDAC activity assay for screening selective HDAC inhibitors, which is inspired by the traditional ELISA (ELISA).  The key feature of the ELISA-based HDAC activity assay is use of mammalian cell-derived HDAC isoforms instead of recombinant proteins.  Importantly, the assay was validated with several known HDAC inhibitors.  The ELISA based HDAC activity assay will facilitate the characterization of isoform-selective HDAC inhibitors against mammalian cell-derived HDAC proteins, which will enhance HDAC-centered cancer research and provide a foundation for anticancer drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCPMJ2yMSbrLVg90H21EOLACvtfcHk0lhGq0TyISBHhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlvVCktLs%253D&md5=f7cc8b4f3d289c8f616c6ccef6c9459e</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1177%2F1087057115598118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057115598118%26sid%3Dliteratum%253Aachs%26aulast%3DPadige%26aufirst%3DG.%26aulast%3DNegmeldin%26aufirst%3DA.%2BT.%26aulast%3DPflum%26aufirst%3DM.%2BK.%2BH.%26atitle%3DDevelopment%2520of%2520an%2520ELISA-Based%2520HDAC%2520Activity%2520Assay%2520for%2520Characterization%2520of%2520Isoform-Selective%2520Inhibitors%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2015%26volume%3D20%26spage%3D1277%26epage%3D1285%26doi%3D10.1177%2F1087057115598118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crispatzu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Ghaili, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weit, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florou, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newrzela, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span> <span> </span><span class="NLM_article-title">Models for Mature T-Cell Lymphomas-A Critical Appraisal of Experimental Systems and Their Contribution to Current T-Cell Tumorigenic Concepts</span>. <i>Critical Reviews in Oncology/Hematology.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">680</span>– <span class="NLM_lpage">695</span>, <span class="refDoi"> DOI: 10.1016/j.critrevonc.2013.07.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1016%2Fj.critrevonc.2013.07.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=23972664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A280%3ADC%252BC3sbjvVegsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2013&pages=680-695&author=K.+Warnerauthor=G.+Crispatzuauthor=N.+Al-Ghailiauthor=N.+Weitauthor=V.+Florouauthor=M.+J.+Youauthor=S.+Newrzelaauthor=M.+Herling&title=Models+for+Mature+T-Cell+Lymphomas-A+Critical+Appraisal+of+Experimental+Systems+and+Their+Contribution+to+Current+T-Cell+Tumorigenic+Concepts&doi=10.1016%2Fj.critrevonc.2013.07.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Models for mature T-cell lymphomas--a critical appraisal of experimental systems and their contribution to current T-cell tumorigenic concepts</span></div><div class="casAuthors">Warner Kathrin; Crispatzu Giuliano; Al-Ghaili Nabil; Weit Nicole; Florou Vaia; You M James; Newrzela Sebastian; Herling Marco</div><div class="citationInfo"><span class="NLM_cas:title">Critical reviews in oncology/hematology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">680-95</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Mature T-cell lymphomas/leukemias (MTCL) have been understudied lymphoid neoplasms that currently receive growing attention.  Our historically rudimentary molecular understanding and dissatisfactory interventional success in this complex and for the most part poor-prognostic group of tumors is only slightly improving.  A major limiting aspect in further progress in these rare neoplasms is the lack of suitable model systems that would substantially facilitate pathogenic studies and pre-clinical drug evaluations.  Such representations of MTCL have thus far not been systematically appraised.  We therefore provide an overview on existing models and point out their particular advantages and limitations in the context of the specific scientific questions.  After addressing issues of species-specific differences and classifications, we summarize data on MTCL cell lines of human as well as murine origin, on murine strain predispositions to MTCL, on available models of genetically engineered mice, and on transplant systems.  From an in-silico meta-analysis of available primary data of gene expression profiles on human MTCL we cross-reference genes reported to transform T-cells in mice and reflect on their general vs entity-restricted relevance and on target-promoter influences.  Overall, we identify the urgent need for new models of higher fidelity to human MTCL with respect to their increasingly recognized diversity and to predictions of drug response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRG8YhZZYBEXs25P7XSLhESfW6udTcc2eaJooFyqdyQzbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sbjvVegsQ%253D%253D&md5=5f857d45628624e7f732ff2ddf6bb4fc</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.critrevonc.2013.07.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.critrevonc.2013.07.014%26sid%3Dliteratum%253Aachs%26aulast%3DWarner%26aufirst%3DK.%26aulast%3DCrispatzu%26aufirst%3DG.%26aulast%3DAl-Ghaili%26aufirst%3DN.%26aulast%3DWeit%26aufirst%3DN.%26aulast%3DFlorou%26aufirst%3DV.%26aulast%3DYou%26aufirst%3DM.%2BJ.%26aulast%3DNewrzela%26aufirst%3DS.%26aulast%3DHerling%26aufirst%3DM.%26atitle%3DModels%2520for%2520Mature%2520T-Cell%2520Lymphomas-A%2520Critical%2520Appraisal%2520of%2520Experimental%2520Systems%2520and%2520Their%2520Contribution%2520to%2520Current%2520T-Cell%2520Tumorigenic%2520Concepts%26jtitle%3DCritical%2520Reviews%2520in%2520Oncology%252FHematology.%26date%3D2013%26volume%3D88%26spage%3D680%26epage%3D695%26doi%3D10.1016%2Fj.critrevonc.2013.07.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Orlova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wingelhofer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubauer, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger-Becvar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miklós Keserű, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriggl, R.</span></span> <span> </span><span class="NLM_article-title">Emerging Therapeutic Targets in Myeloproliferative Neoplasms and Peripheral T-Cell Leukemia and Lymphomas</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">57</span>, <span class="refDoi"> DOI: 10.1080/14728222.2018.1406924</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1080%2F14728222.2018.1406924" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=29148847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFaht7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=45-57&author=A.+Orlovaauthor=B.+Wingelhoferauthor=H.+A.+Neubauerauthor=B.+Maurerauthor=A.+Berger-Becvarauthor=G.+Mikl%C3%B3s+Keser%C5%B1author=P.+T.+Gunningauthor=P.+Valentauthor=R.+Moriggl&title=Emerging+Therapeutic+Targets+in+Myeloproliferative+Neoplasms+and+Peripheral+T-Cell+Leukemia+and+Lymphomas&doi=10.1080%2F14728222.2018.1406924"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging therapeutic targets in myeloproliferative neoplasms and peripheral T-cell leukemia and lymphomas</span></div><div class="casAuthors">Orlova, Anna; Wingelhofer, Bettina; Neubauer, Heidi A.; Maurer, Barbara; Berger-Becvar, Angelika; Keseru, Gyoergy Miklos; Gunning, Patrick T.; Valent, Peter; Moriggl, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-57</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  : Hematopoietic neoplasms are often driven by gain-of-function mutations of the JAK-STAT pathway together with mutations in chromatin remodeling and DNA damage control pathways.  The interconnection between the JAK-STAT pathway, epigenetic regulation or DNA damage control is still poorly understood in cancer cell biol.  Here, we focus on a broader description of mutational insights into myeloproliferative neoplasms and peripheral T-cell leukemia and lymphomas, since sequencing efforts have identified similar combinations of driver mutations in these diseases covering different lineages.  We summarize how these pathways might be interconnected in normal or cancer cells, which have lost differentiation capacity and drive oncogene transcription.  Due to similarities in driver mutations including epigenetic enzymes, JAK-STAT pathway activation and mutated checkpoint control through TP53, we hypothesize that similar therapeutic approaches could be of benefit in these diseases.  We give an overview of how driver mutations in these malignancies contribute to hematopoietic cancer initiation or progression, and how these pathways can be targeted with currently available tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIQGAsv8OacLVg90H21EOLACvtfcHk0lgiXKP98dn0EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFaht7%252FM&md5=989c76e5954a7e46ccf27fa8ebea5530</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1080%2F14728222.2018.1406924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14728222.2018.1406924%26sid%3Dliteratum%253Aachs%26aulast%3DOrlova%26aufirst%3DA.%26aulast%3DWingelhofer%26aufirst%3DB.%26aulast%3DNeubauer%26aufirst%3DH.%2BA.%26aulast%3DMaurer%26aufirst%3DB.%26aulast%3DBerger-Becvar%26aufirst%3DA.%26aulast%3DMikl%25C3%25B3s%2BKeser%25C5%25B1%26aufirst%3DG.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DMoriggl%26aufirst%3DR.%26atitle%3DEmerging%2520Therapeutic%2520Targets%2520in%2520Myeloproliferative%2520Neoplasms%2520and%2520Peripheral%2520T-Cell%2520Leukemia%2520and%2520Lymphomas%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2018%26volume%3D22%26spage%3D45%26epage%3D57%26doi%3D10.1080%2F14728222.2018.1406924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dürig, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bug, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein-Hitpass, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jöns, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Subero, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baudis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dührsen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siebert, R.</span></span> <span> </span><span class="NLM_article-title">Combined Single Nucleotide Polymorphism-Based Genomic Mapping and Global Gene Expression Profiling Identifies Novel Chromosomal Imbalances, Mechanisms and Candidate Genes Important in the Pathogenesis of T-Cell Prolymphocytic Leukemia with Inv(14)(Q11q32)</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2153</span>– <span class="NLM_lpage">2163</span>, <span class="refDoi"> DOI: 10.1038/sj.leu.2404877</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fsj.leu.2404877" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=17713554" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVGmsb7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=2153-2163&author=J.+D%C3%BCrigauthor=S.+Bugauthor=L.+Klein-Hitpassauthor=T.+Boesauthor=T.+J%C3%B6nsauthor=J.+I.+Martin-Suberoauthor=L.+Harderauthor=M.+Baudisauthor=U.+D%C3%BChrsenauthor=R.+Siebert&title=Combined+Single+Nucleotide+Polymorphism-Based+Genomic+Mapping+and+Global+Gene+Expression+Profiling+Identifies+Novel+Chromosomal+Imbalances%2C+Mechanisms+and+Candidate+Genes+Important+in+the+Pathogenesis+of+T-Cell+Prolymphocytic+Leukemia+with+Inv%2814%29%28Q11q32%29&doi=10.1038%2Fsj.leu.2404877"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Combined single nucleotide polymorphism-based genomic mapping and global gene expression profiling identifies novel chromosomal imbalances, mechanisms and candidate genes important in the pathogenesis of T-cell prolymphocytic leukemia with inv(14)(q11q32)</span></div><div class="casAuthors">Duerig, J.; Bug, S.; Klein-Hitpass, L.; Boes, T.; Joens, T.; Martin-Subero, J. I.; Harder, L.; Baudis, M.; Duehrsen, U.; Siebert, R.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2153-2163</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">T-cell prolymphocytic leukemia (T-PLL) is a rare aggressive lymphoma derived from mature T cells, which is, in most cases, characterized by the presence of an inv(14)(q11q32)/t(14;14)(q11;q32) and a characteristic pattern of secondary chromosomal aberrations.  DNA microarray technol. was employed to compare the transcriptomes of eight immunomagnetically purified CD3+ normal donor-derived peripheral blood cell samples, with five highly purified inv(14)/t(14;14)-pos. T-PLL blood samples.  Between the two exptl. groups, 734 genes were identified as differentially expressed, including functionally important genes involved in lymphomagenesis, cell cycle regulation, apoptosis and DNA repair.  Notably, the differentially expressed genes were found to be significantly enriched in genomic regions affected by recurrent chromosomal imbalances.  Upregulated genes clustered on chromosome arms 6p and 8q, and downregulated genes on 6q, 8p, 10p, 11q and 18p.  High-resoln. copy-no. detn. using single nucleotide polymorphism chip technol. in 12 inv(14)/t(14;14)-pos. T-PLL including those analyzed for gene expression, refined chromosomal breakpoints as well as regions of imbalances.  In conclusion, combined transcriptional and mol. cytogenetic profiling identified novel specific chromosomal loci and genes that are likely to be involved in disease progression and suggests a gene dosage effect as a pathogenic mechanism in T-PLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9hzDAmZA-srVg90H21EOLACvtfcHk0lgiXKP98dn0EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVGmsb7I&md5=611eb203d8c8daa965c5d5e29b2a96cf</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fsj.leu.2404877&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.leu.2404877%26sid%3Dliteratum%253Aachs%26aulast%3DD%25C3%25BCrig%26aufirst%3DJ.%26aulast%3DBug%26aufirst%3DS.%26aulast%3DKlein-Hitpass%26aufirst%3DL.%26aulast%3DBoes%26aufirst%3DT.%26aulast%3DJ%25C3%25B6ns%26aufirst%3DT.%26aulast%3DMartin-Subero%26aufirst%3DJ.%2BI.%26aulast%3DHarder%26aufirst%3DL.%26aulast%3DBaudis%26aufirst%3DM.%26aulast%3DD%25C3%25BChrsen%26aufirst%3DU.%26aulast%3DSiebert%26aufirst%3DR.%26atitle%3DCombined%2520Single%2520Nucleotide%2520Polymorphism-Based%2520Genomic%2520Mapping%2520and%2520Global%2520Gene%2520Expression%2520Profiling%2520Identifies%2520Novel%2520Chromosomal%2520Imbalances%252C%2520Mechanisms%2520and%2520Candidate%2520Genes%2520Important%2520in%2520the%2520Pathogenesis%2520of%2520T-Cell%2520Prolymphocytic%2520Leukemia%2520with%2520Inv%252814%2529%2528Q11q32%2529%26jtitle%3DLeukemia%26date%3D2007%26volume%3D21%26spage%3D2153%26epage%3D2163%26doi%3D10.1038%2Fsj.leu.2404877" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patil, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieslak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhart, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toprak, U. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagener, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiehle, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altmüller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franitza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scholz, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayne, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahearne, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheffold, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jebaraj, B. M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyer, M. J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurnberg, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durig, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottcher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schesner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stilgenbauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macintyre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siebert, R.</span></span> <span> </span><span class="NLM_article-title">Reconstruction of Rearranged T-Cell Receptor Loci by Whole Genome and Transcriptome Sequencing Gives Insights into the Initial Steps of T-Cell Prolymphocytic Leukemia</span>. <i>Genes, Chromosomes Cancer</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">267</span>, <span class="refDoi"> DOI: 10.1002/gcc.22821</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1002%2Fgcc.22821" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=31677197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlt1WgtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=261-267&author=P.+Patilauthor=A.+Cieslakauthor=S.+H.+Bernhartauthor=U.+H.+Toprakauthor=R.+Wagenerauthor=C.+L%C3%B3pezauthor=L.+Wiehleauthor=S.+Bensauthor=J.+Altm%C3%BCllerauthor=M.+Franitzaauthor=I.+Scholzauthor=S.+Jayneauthor=M.+J.+Ahearneauthor=A.+Scheffoldauthor=B.+M.+C.+Jebarajauthor=C.+Schneiderauthor=D.+Costaauthor=T.+Braunauthor=A.+Schraderauthor=E.+Campoauthor=M.+J.+S.+Dyerauthor=P.+Nurnbergauthor=J.+Durigauthor=S.+Bottcherauthor=M.+Schesnerauthor=M.+Herlingauthor=S.+Stilgenbauerauthor=E.+Macintyreauthor=R.+Siebert&title=Reconstruction+of+Rearranged+T-Cell+Receptor+Loci+by+Whole+Genome+and+Transcriptome+Sequencing+Gives+Insights+into+the+Initial+Steps+of+T-Cell+Prolymphocytic+Leukemia&doi=10.1002%2Fgcc.22821"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Reconstruction of rearranged T-cell receptor loci by whole genome and transcriptome sequencing gives insights into the initial steps of T-cell prolymphocytic leukemia</span></div><div class="casAuthors">Patil, Paurnima; Cieslak, Agata; Bernhart, Stephan H.; Toprak, Umut H.; Wagener, Rabea; Lopez, Cristina; Wiehle, Laura; Bens, Susanne; Altmueller, Janine; Franitza, Marek; Scholz, Ingrid; Jayne, Sandrine; Ahearne, Matthew J.; Scheffold, Annika; Jebaraj, Billy M. C.; Schneider, Christof; Costa, Dolors; Braun, Till; Schrader, Alexandra; Campo, Elias; Dyer, Martin J. S.; Nuernberg, Peter; Duerig, Jan; Johansson, Patricia; Boettcher, Sebastian; Schlesner, Matthias; Herling, Marco; Stilgenbauer, Stephan; Macintyre, Elizabeth; Siebert, Reiner</div><div class="citationInfo"><span class="NLM_cas:title">Genes, Chromosomes & Cancer</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">261-267</span>CODEN:
                <span class="NLM_cas:coden">GCCAES</span>;
        ISSN:<span class="NLM_cas:issn">1045-2257</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">T-cell prolymphocytic leukemia (T-PLL) is an aggressive tumor with leukemic presentation of mature T-lymphocytes.  Here, we aimed at characterizing the initial events in the mol. pathogenesis of T-PLL and particularly, at detg. the point in T-cell differentiation when the hallmark oncogenic events, i.e., inv(14)(q11q32)/t(14;14)(q11;q32) and t(X;14)(q28;q11) occur.  To this end, we mined whole genome and transcriptome sequencing data of 17 and 11 T-PLL cases, resp.  Mapping of the 14q32.1 locus breakpoints identified only TCL1A, which was moreover significantly overexpressed in T-PLL as compared to benign CD4+ and CD8+ T-cells, as the only common oncogenic target of aberrations.  In cases with t(14;14), the breakpoints mapped telomeric and in cases with inv(14) centromeric or in the 3'-untranslated region of TCL1A.  Regarding the T-cell receptor alpha (TRA) locus-TCL1A breakpoint junctions, all 17 breakpoints involved recombination signal sequences and 15 junctions contained nontemplated (N-) nucleotides.  All T-PLL cases studied carried in-frame TRA rearrangements on the intact allele, which skewed significantly toward usage of distal/central TRAV/TRAJ gene segments as compared to the illegitimate TRA rearrangements.  Our findings suggest that the oncogenic TRA-TCL1A/MTCP1 rearrangements in T-PLL occur during opening of the TRA locus, i.e., during the progression from CD4+ immature single pos. to early double pos. thymocyte stage, just before physiol. TCL1A expression is silenced.  The cell carrying such an oncogenic event continues maturation and rearranges the second TRA allele to achieve a functional T-cell receptor.  Thereafter, it switches off RAG and DNTT expression in line with the mature T-cell phenotype at presentation of T-PLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnLsOdYbvKB7Vg90H21EOLACvtfcHk0ljU-W58SpDDAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlt1WgtLw%253D&md5=4e0f8b13b816524524d1de9bf03d5e93</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fgcc.22821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fgcc.22821%26sid%3Dliteratum%253Aachs%26aulast%3DPatil%26aufirst%3DP.%26aulast%3DCieslak%26aufirst%3DA.%26aulast%3DBernhart%26aufirst%3DS.%2BH.%26aulast%3DToprak%26aufirst%3DU.%2BH.%26aulast%3DWagener%26aufirst%3DR.%26aulast%3DL%25C3%25B3pez%26aufirst%3DC.%26aulast%3DWiehle%26aufirst%3DL.%26aulast%3DBens%26aufirst%3DS.%26aulast%3DAltm%25C3%25BCller%26aufirst%3DJ.%26aulast%3DFranitza%26aufirst%3DM.%26aulast%3DScholz%26aufirst%3DI.%26aulast%3DJayne%26aufirst%3DS.%26aulast%3DAhearne%26aufirst%3DM.%2BJ.%26aulast%3DScheffold%26aufirst%3DA.%26aulast%3DJebaraj%26aufirst%3DB.%2BM.%2BC.%26aulast%3DSchneider%26aufirst%3DC.%26aulast%3DCosta%26aufirst%3DD.%26aulast%3DBraun%26aufirst%3DT.%26aulast%3DSchrader%26aufirst%3DA.%26aulast%3DCampo%26aufirst%3DE.%26aulast%3DDyer%26aufirst%3DM.%2BJ.%2BS.%26aulast%3DNurnberg%26aufirst%3DP.%26aulast%3DDurig%26aufirst%3DJ.%26aulast%3DBottcher%26aufirst%3DS.%26aulast%3DSchesner%26aufirst%3DM.%26aulast%3DHerling%26aufirst%3DM.%26aulast%3DStilgenbauer%26aufirst%3DS.%26aulast%3DMacintyre%26aufirst%3DE.%26aulast%3DSiebert%26aufirst%3DR.%26atitle%3DReconstruction%2520of%2520Rearranged%2520T-Cell%2520Receptor%2520Loci%2520by%2520Whole%2520Genome%2520and%2520Transcriptome%2520Sequencing%2520Gives%2520Insights%2520into%2520the%2520Initial%2520Steps%2520of%2520T-Cell%2520Prolymphocytic%2520Leukemia%26jtitle%3DGenes%252C%2520Chromosomes%2520Cancer%26date%3D2020%26volume%3D59%26spage%3D261%26epage%3D267%26doi%3D10.1002%2Fgcc.22821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pützer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufva, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crispatzu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oles, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adnan-Awad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagstrom, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellanger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mpindi, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldfors, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pemovska, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietarinen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lauhio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuesta-Matoes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koschmieder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brummendorf, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kytola, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savolainen, E.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siitonen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellonen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallioniemi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wennerberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aittokallio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zenz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustjoki, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel Drug Sensitivities in T-PLL by High-Throughput Ex Vivo Drug Testing and Mutation Profiling</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">774</span>– <span class="NLM_lpage">787</span>, <span class="refDoi"> DOI: 10.1038/leu.2017.252</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fleu.2017.252" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=28804127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKmu7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=774-787&author=E.+I.+Anderssonauthor=S.+P%C3%BCtzerauthor=B.+Yadavauthor=O.+Dufvaauthor=S.+Khanauthor=L.+Heauthor=L.+Sellnerauthor=A.+Schraderauthor=G.+Crispatzuauthor=M.+Olesauthor=H.+Zhangauthor=S.+Adnan-Awadauthor=S.+Lagstromauthor=D.+Bellangerauthor=J.+P.+Mpindiauthor=S.+Eldforsauthor=T.+Pemovskaauthor=S.+Pietarinenauthor=A.+Lauhioauthor=K.+Tomskaauthor=C.+Cuesta-Matoesauthor=E.+Faberauthor=S.+Koschmiederauthor=T.+H.+Brummendorfauthor=S.+Kytolaauthor=E.-R.+Savolainenauthor=T.+Siitonenauthor=P.+Ellonenauthor=O.+Kallioniemiauthor=K.+Wennerbergauthor=W.+Dingauthor=M.+Sternauthor=W.+Huberauthor=S.+Andersauthor=J.+Tangauthor=T.+Aittokallioauthor=T.+Zenzauthor=M.+Herlingauthor=S.+Mustjoki&title=Discovery+of+Novel+Drug+Sensitivities+in+T-PLL+by+High-Throughput+Ex+Vivo+Drug+Testing+and+Mutation+Profiling&doi=10.1038%2Fleu.2017.252"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel drug sensitivities in T-PLL by high-throughput ex vivo drug testing and mutation profiling</span></div><div class="casAuthors">Andersson, E. I.; Putzer, S.; Yadav, B.; Dufva, O.; Khan, S.; He, L.; Sellner, L.; Schrader, A.; Crispatzu, G.; Oles, M.; Zhang, H.; Adnan-Awad, S.; Lagstrom, S.; Bellanger, D.; Mpindi, J. P.; Eldfors, S.; Pemovska, T.; Pietarinen, P.; Lauhio, A.; Tomska, K.; Cuesta-Mateos, C.; Faber, E.; Koschmieder, S.; Brummendorf, T. H.; Kytola, S.; Savolainen, E.-R.; Siitonen, T.; Ellonen, P.; Kallioniemi, O.; Wennerberg, K.; Ding, W.; Stern, M.-H.; Huber, W.; Anders, S.; Tang, J.; Aittokallio, T.; Zenz, T.; Herling, M.; Mustjoki, S.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">774-787</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">T-cell prolymphocytic leukemia (T-PLL) is a rare and aggressive neoplasm of mature T-cells with an urgent need for rationally designed therapies to address its notoriously chemo-refractory behavior.  The median survival of T-PLL patients is <2 years and clin. trials are difficult to execute.  Here we systematically explored the diversity of drug responses in T-PLL patient samples using an ex vivo drug sensitivity and resistance testing platform and correlated the findings with somatic mutations and gene expression profiles.  Intriguingly, all T-PLL samples were sensitive to the cyclin-dependent kinase inhibitor SNS-032, which overcame stromal-cell-mediated protection and elicited robust p53-activation and apoptosis.  Across all patients, the most effective classes of compds. were histone deacetylase, phosphoinositide-3 kinase/AKT/mammalian target of rapamycin, heat-shock protein 90 and BH3-family protein inhibitors as well as p53 activators, indicating previously unexplored, novel targeted approaches for treating T-PLL.  Although Janus-activated kinase-signal transducer and activator of transcription factor (JAK-STAT) pathway mutations were common in T-PLL (71% of patients), JAK-STAT inhibitor responses were not directly linked to those or other T-PLL-specific lesions.  Overall, we found that genetic markers do not readily translate into novel effective therapeutic vulnerabilities.  In conclusion, novel classes of compds. with high efficacy in T-PLL were discovered with the comprehensive ex vivo drug screening platform warranting further studies of synergisms and clin. testing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNlAKw7-znF7Vg90H21EOLACvtfcHk0lgWDWKjljUYdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKmu7%252FI&md5=70fedb2dce547d45c70300ec7759ee07</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fleu.2017.252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2017.252%26sid%3Dliteratum%253Aachs%26aulast%3DAndersson%26aufirst%3DE.%2BI.%26aulast%3DP%25C3%25BCtzer%26aufirst%3DS.%26aulast%3DYadav%26aufirst%3DB.%26aulast%3DDufva%26aufirst%3DO.%26aulast%3DKhan%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DL.%26aulast%3DSellner%26aufirst%3DL.%26aulast%3DSchrader%26aufirst%3DA.%26aulast%3DCrispatzu%26aufirst%3DG.%26aulast%3DOles%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DAdnan-Awad%26aufirst%3DS.%26aulast%3DLagstrom%26aufirst%3DS.%26aulast%3DBellanger%26aufirst%3DD.%26aulast%3DMpindi%26aufirst%3DJ.%2BP.%26aulast%3DEldfors%26aufirst%3DS.%26aulast%3DPemovska%26aufirst%3DT.%26aulast%3DPietarinen%26aufirst%3DS.%26aulast%3DLauhio%26aufirst%3DA.%26aulast%3DTomska%26aufirst%3DK.%26aulast%3DCuesta-Matoes%26aufirst%3DC.%26aulast%3DFaber%26aufirst%3DE.%26aulast%3DKoschmieder%26aufirst%3DS.%26aulast%3DBrummendorf%26aufirst%3DT.%2BH.%26aulast%3DKytola%26aufirst%3DS.%26aulast%3DSavolainen%26aufirst%3DE.-R.%26aulast%3DSiitonen%26aufirst%3DT.%26aulast%3DEllonen%26aufirst%3DP.%26aulast%3DKallioniemi%26aufirst%3DO.%26aulast%3DWennerberg%26aufirst%3DK.%26aulast%3DDing%26aufirst%3DW.%26aulast%3DStern%26aufirst%3DM.%26aulast%3DHuber%26aufirst%3DW.%26aulast%3DAnders%26aufirst%3DS.%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DAittokallio%26aufirst%3DT.%26aulast%3DZenz%26aufirst%3DT.%26aulast%3DHerling%26aufirst%3DM.%26aulast%3DMustjoki%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520Novel%2520Drug%2520Sensitivities%2520in%2520T-PLL%2520by%2520High-Throughput%2520Ex%2520Vivo%2520Drug%2520Testing%2520and%2520Mutation%2520Profiling%26jtitle%3DLeukemia%26date%3D2018%26volume%3D32%26spage%3D774%26epage%3D787%26doi%3D10.1038%2Fleu.2017.252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pemovska, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szwajda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulesskiy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kontro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karjalainen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majumder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murumägi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knowles, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porkka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heckman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallioniemi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wennerberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aittokallio, T.</span></span> <span> </span><span class="NLM_article-title">Quantitative Scoring of Differential Drug Sensitivity for Individually Optimized Anticancer Therapies</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">5193</span>, <span class="refDoi"> DOI: 10.1038/srep05193</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fsrep05193" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2015&pages=5193&author=B.+Yadavauthor=T.+Pemovskaauthor=A.+Szwajdaauthor=E.+Kulesskiyauthor=M.+Kontroauthor=R.+Karjalainenauthor=M.+Majumderauthor=D.+Malaniauthor=A.+Murum%C3%A4giauthor=J.+Knowlesauthor=K.+Porkkaauthor=C.+Heckmanauthor=O.+Kallioniemiauthor=K.+Wennerbergauthor=T.+Aittokallio&title=Quantitative+Scoring+of+Differential+Drug+Sensitivity+for+Individually+Optimized+Anticancer+Therapies&doi=10.1038%2Fsrep05193"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1038%2Fsrep05193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep05193%26sid%3Dliteratum%253Aachs%26aulast%3DYadav%26aufirst%3DB.%26aulast%3DPemovska%26aufirst%3DT.%26aulast%3DSzwajda%26aufirst%3DA.%26aulast%3DKulesskiy%26aufirst%3DE.%26aulast%3DKontro%26aufirst%3DM.%26aulast%3DKarjalainen%26aufirst%3DR.%26aulast%3DMajumder%26aufirst%3DM.%26aulast%3DMalani%26aufirst%3DD.%26aulast%3DMurum%25C3%25A4gi%26aufirst%3DA.%26aulast%3DKnowles%26aufirst%3DJ.%26aulast%3DPorkka%26aufirst%3DK.%26aulast%3DHeckman%26aufirst%3DC.%26aulast%3DKallioniemi%26aufirst%3DO.%26aulast%3DWennerberg%26aufirst%3DK.%26aulast%3DAittokallio%26aufirst%3DT.%26atitle%3DQuantitative%2520Scoring%2520of%2520Differential%2520Drug%2520Sensitivity%2520for%2520Individually%2520Optimized%2520Anticancer%2520Therapies%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D4%26spage%3D5193%26doi%3D10.1038%2Fsrep05193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cosenza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Civallero, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcheselli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, S.</span></span> <span> </span><span class="NLM_article-title">Ricolinostat, a Selective HDAC6 Inhibitor, Shows Anti-Lymphoma Cell Activity Alone and in Combination with Bendamustine</span>. <i>Apoptosis</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">827</span>– <span class="NLM_lpage">840</span>, <span class="refDoi"> DOI: 10.1007/s10495-017-1364-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1007%2Fs10495-017-1364-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=28315173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslKktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=827-840&author=M.+Cosenzaauthor=M.+Civalleroauthor=L.+Marcheselliauthor=S.+Sacchiauthor=S.+Pozzi&title=Ricolinostat%2C+a+Selective+HDAC6+Inhibitor%2C+Shows+Anti-Lymphoma+Cell+Activity+Alone+and+in+Combination+with+Bendamustine&doi=10.1007%2Fs10495-017-1364-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Ricolinostat, a selective HDAC6 inhibitor, shows anti-lymphoma cell activity alone and in combination with bendamustine</span></div><div class="casAuthors">Cosenza, Maria; Civallero, Monica; Marcheselli, Luigi; Sacchi, Stefano; Pozzi, Samantha</div><div class="citationInfo"><span class="NLM_cas:title">Apoptosis</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">827-840</span>CODEN:
                <span class="NLM_cas:coden">APOPFN</span>;
        ISSN:<span class="NLM_cas:issn">1360-8185</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Histone deacetylase inhibitors (HDACis) have emerged as a new class of anticancer agents, targeting the biol. process including cell cycle and apoptosis.  We investigated and explained the anticancer effects of an HDAC6 inhibitor, ricolinostat alone and in combination with bendamustine in lymphoma cell lines.  Cell viability was measured by MTT assay.  Apoptosis, reactive oxygen species (ROS) generation, Bcl-2 protein expression, cell cycle progression and tubuline expression were detd. by flow cytometry.  The effects of ricolinostat alone and in combination on the caspases, PI3K/Akt, Bcl-2 pathways, ER stress and UPR were assessed by immunoblotting.  Ricolinostat shows anti lymphoma activity when used as single agent and its capability to induce apoptosis is synergistically potentiated by the bendamustine in lymphoma cell lines.  Drug combination reduced the proportion of cells in the G0/G1 and S phases and caused an increase of "sub-G0/G1" peak.  The synergistic effect accompanied with the increased ROS, activation of caspase-8, -9, and -3, the cleavage of PARP and modulated by Bcl-2 proteins family.  In addn., the exposure of ricolinostat induced the acetylation level of α-tubulin, the extend of which was not further modified by bendamustine.  Finally, the apoptosis effect of ricolinostat/bendamustine may be mediated by a corresponding effect on microtubule stabilization.  Our data suggest that ricolinostat in combination with bendamustine may be a novel combination with potential for use as an antitumor agent in lymphoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhV5zrMw5A6rVg90H21EOLACvtfcHk0lhgpIQL9wJ5oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslKktr8%253D&md5=6ac58def2f10c37fe434c78dd074dcfd</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1007%2Fs10495-017-1364-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10495-017-1364-4%26sid%3Dliteratum%253Aachs%26aulast%3DCosenza%26aufirst%3DM.%26aulast%3DCivallero%26aufirst%3DM.%26aulast%3DMarcheselli%26aufirst%3DL.%26aulast%3DSacchi%26aufirst%3DS.%26aulast%3DPozzi%26aufirst%3DS.%26atitle%3DRicolinostat%252C%2520a%2520Selective%2520HDAC6%2520Inhibitor%252C%2520Shows%2520Anti-Lymphoma%2520Cell%2520Activity%2520Alone%2520and%2520in%2520Combination%2520with%2520Bendamustine%26jtitle%3DApoptosis%26date%3D2017%26volume%3D22%26spage%3D827%26epage%3D840%26doi%3D10.1007%2Fs10495-017-1364-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">North, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almeciga-Pinto, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quayle, S. N.</span></span> <span> </span><span class="NLM_article-title">Enhancement of Pomalidomide Anti-Tumor Response with ACY-241, a Selective HDAC6 Inhibitor</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0173507</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0173507</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1371%2Fjournal.pone.0173507" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=28264055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVGjtrfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=B.+J.+Northauthor=I.+Almeciga-Pintoauthor=D.+Tamangauthor=M.+Yangauthor=S.+S.+Jonesauthor=S.+N.+Quayle&title=Enhancement+of+Pomalidomide+Anti-Tumor+Response+with+ACY-241%2C+a+Selective+HDAC6+Inhibitor&doi=10.1371%2Fjournal.pone.0173507"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Enhancement of pomalidomide anti-tumor response with ACY-241, a selective HDAC6 inhibitor</span></div><div class="casAuthors">North, Brian J.; Almeciga-Pinto, Ingrid; Tamang, David; Yang, Min; Jones, Simon S.; Quayle, Steven N.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e0173507/1-e0173507/16</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Thalidomide-based Immunomodulatory Drugs (IMiDs), including lenalidomide and pomalidomide, are effective therapeutics for multiple myeloma.  These agents have been approved with, or are under clin. development with, other targeted therapies including proteasome inhibitors, αCD38 monoclonal antibodies, as well as histone deacetylase (HDAC) inhibitors for combination therapy.  HDAC inhibitors broadly targeting Class I and IIb HDACs have shown potent preclin. efficacy but have frequently demonstrated an undesirable safety profile in combination therapy approaches in clin. studies.  Therefore, development of more selective HDAC inhibitors could provide enhanced efficacy with reduced side effects in combination with IMiDs for the treatment of B-cell malignancies, including multiple myeloma.  Here, the second generation selective HDAC6 inhibitor citarinostat (ACY-241), with a more favorable safety profile than non-selective pan-HDAC inhibitors, is shown to synergize with pomalidomide in in vitro assays through promoting greater apoptosis and cell cycle arrest.  Furthermore, utilizing a multiple myeloma in vivo murine xenograft model, combination treatment with pomalidomide and ACY-241 leads to increased tumor growth inhibition.  At the mol. level, combination treatment with ACY-241 and pomalidomide leads to greater suppression of the pro-survival factors survivin, Myc, and IRF4.  The results presented here demonstrate synergy between pomalidomide and ACY-241 in both in vitro and in vivo preclin. models, providing further impetus for clin. development of ACY-241 for use in combination with IMiDs for patients with multiple myeloma and potentially other B-cell malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaSAg6exJCNbVg90H21EOLACvtfcHk0lhgpIQL9wJ5oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVGjtrfL&md5=175ef137cd119027f0462d07784375b5</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0173507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0173507%26sid%3Dliteratum%253Aachs%26aulast%3DNorth%26aufirst%3DB.%2BJ.%26aulast%3DAlmeciga-Pinto%26aufirst%3DI.%26aulast%3DTamang%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DS.%2BS.%26aulast%3DQuayle%26aufirst%3DS.%2BN.%26atitle%3DEnhancement%2520of%2520Pomalidomide%2520Anti-Tumor%2520Response%2520with%2520ACY-241%252C%2520a%2520Selective%2520HDAC6%2520Inhibitor%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0173507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, Y. H.</span></span> <span> </span><span class="NLM_article-title">Synergistic Antitumor Effects of the Combined Treatment with an HDAC6 Inhibitor and a COX-2 Inhibitor through Activation of PTEN</span>. <i>Oncol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">2657</span>– <span class="NLM_lpage">2666</span>, <span class="refDoi"> DOI: 10.3892/or.2017.5981</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.3892%2For.2017.5981" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=29048666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVKjsrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=2657-2666&author=G.+Zhangauthor=Y.+H.+Gan&title=Synergistic+Antitumor+Effects+of+the+Combined+Treatment+with+an+HDAC6+Inhibitor+and+a+COX-2+Inhibitor+through+Activation+of+PTEN&doi=10.3892%2For.2017.5981"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Synergistic antitumor effects of the combined treatment with an HDAC6 inhibitor and a COX-2 inhibitor through activation of PTEN</span></div><div class="casAuthors">Zhang, Guanhua; Gan, Ye-Hua</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2657-2666</span>CODEN:
                <span class="NLM_cas:coden">OCRPEW</span>;
        ISSN:<span class="NLM_cas:issn">1791-2431</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Chemotherapy is one of the most effective nonsurgical treatments for various types of tumor.  Identifying different combinations of antitumor agents that can produce synergistic antitumor effects remains an important clin. strategy.  In the present study, we showed that the combination of histone deacetylase 6 (HDAC6) inhibitor tubastatin A together with cyclooxygenase-2 (COX-2) inhibitor celecoxib resulted in synergistic antitumor effects in CAL 27 and SACC-83 cells.  Treatment with celecoxib alone promoted the membrane translocation of phosphatase and tensin homolog (PTEN), indicating PTEN activation, and consequently led to protein kinase B (AKT) dephosphorylation (inactivation).  Similarly, treatment with an HDAC6 inhibitor alone promoted PTEN membrane translocation and correspondingly dephosphorylated AKT.  The combination of celecoxib and an HDAC6 inhibitor synergistically increased PTEN membrane translocation and inactivated AKT.  Moreover, celecoxib enhanced the HDAC6 inhibitor-induced antitumor effects in PTEN-deficient U-87 MG cells that had been stably transfected with wild-type PTEN, but not in the same cell line stably transfected with mutant PTEN-K163R, which cannot be activated by HDAC6 inhibitors.  In summary, the results indicated that the COX-2 inhibitor celecoxib enhanced the HDAC6 inhibitor-induced antitumor effects by activating the PTEN/AKT signaling pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPVy27HVeeLLVg90H21EOLACvtfcHk0lhgpIQL9wJ5oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVKjsrrE&md5=0ee7ef927d63553a2ebcbbdd06fbd661</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.3892%2For.2017.5981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252For.2017.5981%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DGan%26aufirst%3DY.%2BH.%26atitle%3DSynergistic%2520Antitumor%2520Effects%2520of%2520the%2520Combined%2520Treatment%2520with%2520an%2520HDAC6%2520Inhibitor%2520and%2520a%2520COX-2%2520Inhibitor%2520through%2520Activation%2520of%2520PTEN%26jtitle%3DOncol.%2520Rep.%26date%3D2017%26volume%3D38%26spage%3D2657%26epage%3D2666%26doi%3D10.3892%2For.2017.5981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hideshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paranal, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colling, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estiu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazitschek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cottini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mimura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorgun, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tai, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrasco, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Discovery of Selective Small-Molecule HDAC6 Inhibitor for Overcoming Proteasome Inhibitor Resistance in Multiple Myeloma</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">13162</span>– <span class="NLM_lpage">13167</span>, <span class="refDoi"> DOI: 10.1073/pnas.1608067113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1073%2Fpnas.1608067113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=27799547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslKnu73M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=13162-13167&author=T.+Hideshimaauthor=J.+Qiauthor=R.+M.+Paranalauthor=W.+Tangauthor=E.+Greenbergauthor=N.+Westauthor=M.+E.+Collingauthor=G.+Estiuauthor=R.+Mazitschekauthor=J.+A.+Perryauthor=H.+Ohguchiauthor=F.+Cottiniauthor=N.+Mimuraauthor=G.+Gorgunauthor=Y.-T.+Taiauthor=P.+G.+Richardsonauthor=R.+D.+Carrascoauthor=O.+Wiestauthor=S.+L.+Schreiberauthor=K.+C.+Andersonauthor=J.+E.+Bradner&title=Discovery+of+Selective+Small-Molecule+HDAC6+Inhibitor+for+Overcoming+Proteasome+Inhibitor+Resistance+in+Multiple+Myeloma&doi=10.1073%2Fpnas.1608067113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective small-molecule HDAC6 inhibitor for overcoming proteasome inhibitor resistance in multiple myeloma</span></div><div class="casAuthors">Hideshima, Teru; Qi, Jun; Paranal, Ronald M.; Tang, Weiping; Greenberg, Edward; West, Nathan; Colling, Meaghan E.; Estiu, Guillermina; Mazitschek, Ralph; Perry, Jennifer A.; Ohguchi, Hiroto; Cottini, Francesca; Mimura, Naoya; Gorgun, Gullu; Tai, Yu-Tzu; Richardson, Paul G.; Carrasco, Ruben D.; Wiest, Olaf; Schreiber, Stuart L.; Anderson, Kenneth C.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">13162-13167</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Multiple myeloma (MM) has proven clin. susceptible to modulation of pathways of protein homeostasis.  Blockade of proteasomal degrdn. of polyubiquitinated misfolded proteins by the proteasome inhibitor bortezomib (BTZ) achieves responses and prolongs survival in MM, but long-term treatment with BTZ leads to drug-resistant relapse in most patients.  In a proof-of-concept study, the authors previously demonstrated that blocking aggresomal breakdown of polyubiquitinated misfolded proteins with the histone deacetylase 6 (HDAC6) inhibitor tubacin enhances BTZ-induced cytotoxicity in MM cells in vitro.  However, these foundational studies were limited by the pharmacol. liabilities of tubacin as a chem. probe with only in vitro utility.  Emerging from a focused library synthesis, a potent, selective, and bioavailable HDAC6 inhibitor, WT161, was created to study the mechanism of action of HDAC6 inhibition in MM alone and in combination with BTZ.  WT161 in combination with BTZ triggers significant accumulation of polyubiquitinated proteins and cell stress, followed by caspase activation and apoptosis.  More importantly, this combination treatment was effective in BTZ-resistant cells and in the presence of bone marrow stromal cells, which have been shown to mediate MM cell drug resistance.  The activity of WT161 was confirmed in the human MM cell xenograft mouse model and established the framework for clin. trials of the combination treatment to improve patient outcomes in MM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxE_VYaPFUSbVg90H21EOLACvtfcHk0liUFZnFAu1d1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslKnu73M&md5=2515fcba03631917d0ca7656d1f62a7e</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1608067113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1608067113%26sid%3Dliteratum%253Aachs%26aulast%3DHideshima%26aufirst%3DT.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DGreenberg%26aufirst%3DE.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DColling%26aufirst%3DM.%2BE.%26aulast%3DEstiu%26aufirst%3DG.%26aulast%3DMazitschek%26aufirst%3DR.%26aulast%3DPerry%26aufirst%3DJ.%2BA.%26aulast%3DOhguchi%26aufirst%3DH.%26aulast%3DCottini%26aufirst%3DF.%26aulast%3DMimura%26aufirst%3DN.%26aulast%3DGorgun%26aufirst%3DG.%26aulast%3DTai%26aufirst%3DY.-T.%26aulast%3DRichardson%26aufirst%3DP.%2BG.%26aulast%3DCarrasco%26aufirst%3DR.%2BD.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDiscovery%2520of%2520Selective%2520Small-Molecule%2520HDAC6%2520Inhibitor%2520for%2520Overcoming%2520Proteasome%2520Inhibitor%2520Resistance%2520in%2520Multiple%2520Myeloma%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D13162%26epage%3D13167%26doi%3D10.1073%2Fpnas.1608067113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahman, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piekarz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robey, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figg, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, S. E.</span></span> <span> </span><span class="NLM_article-title">Romidepsin: A New Therapy for Cutaneous T-Cell Lymphoma and a Potential Therapy for Solid Tumors</span>. <i>Expert Rev. Anticancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">997</span>– <span class="NLM_lpage">1008</span>, <span class="refDoi"> DOI: 10.1586/era.10.88</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1586%2Fera.10.88" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=20645688" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptFWqs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=997-1008&author=C.+Grantauthor=F.+Rahmanauthor=R.+Piekarzauthor=C.+Peerauthor=R.+Fryeauthor=R.+W.+Robeyauthor=E.+R.+Gardnerauthor=W.+D.+Figgauthor=S.+E.+Bates&title=Romidepsin%3A+A+New+Therapy+for+Cutaneous+T-Cell+Lymphoma+and+a+Potential+Therapy+for+Solid+Tumors&doi=10.1586%2Fera.10.88"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Romidepsin: a new therapy for cutaneous T-cell lymphoma and a potential therapy for solid tumors</span></div><div class="casAuthors">Grant, Cliona; Rahman, Fahd; Piekarz, Richard; Peer, Cody; Frye, Robin; Robey, Robert W.; Gardner, Erin R.; Figg, William D.; Bates, Susan E.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Review of Anticancer Therapy</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">997-1008</span>CODEN:
                <span class="NLM_cas:coden">ERATBJ</span>;
        ISSN:<span class="NLM_cas:issn">1473-7140</span>.
    
            (<span class="NLM_cas:orgname">Expert Reviews Ltd.</span>)
        </div><div class="casAbstract">A review.  Romidepsin is a histone deacetylase inhibitor (HDI), approved by the US FDA for the treatment of cutaneous T-cell lymphoma (CTCL).  Although various mechanisms have been proposed for the activity of HDIs, including induction of genes controlling cell cycle, acetylation of cytoplasmic proteins and direct induction of apoptosis, the mechanism underlying activity of romidepsin and other HDIs in CTCL is not known.  Romidepsin induces long-lasting responses.  The side-effect profile is similar to that of other HDIs, causing fatigue, nausea and thrombocytopenia.  Management of the CTCL population requires vigilence to prevent infection with skin contaminants, and monitoring of potassium and magnesium, electrolytes found to be low in a large proportion of patients.  Electrocardiog. (ECG) changes are common but are not assocd. with myocardial damage.  When mol. end points were evaluated in 61 patients enrolled on a Phase II trial with romidepsin, response was assocd. with persistence of acetylated histone H3, suggesting that drug exposure is important in effective therapy with romidepsin.  Future studies will endeavor to identify combination strategies to increase the efficacy both in resistant CTCL and in solid tumors and to identify biomarkers of response that will allow selection of patients most likely to benefit from the therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhGXcIbdqpk7Vg90H21EOLACvtfcHk0lhmXyDMVcEMxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptFWqs74%253D&md5=b28fd17c77991e4662f7c0e08d6d7297</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1586%2Fera.10.88&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1586%252Fera.10.88%26sid%3Dliteratum%253Aachs%26aulast%3DGrant%26aufirst%3DC.%26aulast%3DRahman%26aufirst%3DF.%26aulast%3DPiekarz%26aufirst%3DR.%26aulast%3DPeer%26aufirst%3DC.%26aulast%3DFrye%26aufirst%3DR.%26aulast%3DRobey%26aufirst%3DR.%2BW.%26aulast%3DGardner%26aufirst%3DE.%2BR.%26aulast%3DFigg%26aufirst%3DW.%2BD.%26aulast%3DBates%26aufirst%3DS.%2BE.%26atitle%3DRomidepsin%253A%2520A%2520New%2520Therapy%2520for%2520Cutaneous%2520T-Cell%2520Lymphoma%2520and%2520a%2520Potential%2520Therapy%2520for%2520Solid%2520Tumors%26jtitle%3DExpert%2520Rev.%2520Anticancer%2520Ther.%26date%3D2010%26volume%3D10%26spage%3D997%26epage%3D1008%26doi%3D10.1586%2Fera.10.88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balasubramanian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirisawad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buggy, J. J.</span></span> <span> </span><span class="NLM_article-title">A Novel Histone Deacetylase 8 (HDAC8)-Specific Inhibitor PCI-34051 Induces Apoptosis in T-Cell Lymphomas</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1026</span>– <span class="NLM_lpage">1034</span>, <span class="refDoi"> DOI: 10.1038/leu.2008.9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1038%2Fleu.2008.9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=18256683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvVSrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=1026-1034&author=S.+Balasubramanianauthor=J.+Ramosauthor=W.+Luoauthor=M.+Sirisawadauthor=E.+Vernerauthor=J.+J.+Buggy&title=A+Novel+Histone+Deacetylase+8+%28HDAC8%29-Specific+Inhibitor+PCI-34051+Induces+Apoptosis+in+T-Cell+Lymphomas&doi=10.1038%2Fleu.2008.9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">A novel histone deacetylase 8 (HDAC8)-specific inhibitor PCI-34051 induces apoptosis in T-cell lymphomas</span></div><div class="casAuthors">Balasubramanian, S.; Ramos, J.; Luo, W.; Sirisawad, M.; Verner, E.; Buggy, J. J.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1026-1034</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We have developed a potent, histone deacetylase 8 (HDAC8)-specific inhibitor PCI-34051 with >200-fold selectivity over the other HDAC isoforms.  PCI-34051 induces caspase-dependent apoptosis in cell lines derived from T-cell lymphomas or leukemias, but not in other hematopoietic or solid tumor lines.  Unlike broad-spectrum HDAC inhibitors, PCI-34051 does not cause detectable histone or tubulin acetylation.  Cells defective in T-cell receptor signaling were still sensitive to PCI-34051-induced apoptosis, whereas a phospholipase C-γ1 (PLCγ1)-defective line was resistant.  Jurkat cells showed a dose-dependent decrease in PCI-34051-induced apoptosis upon treatment with a PLC inhibitor U73122, but not with an inactive analog.  We found that rapid intracellular calcium mobilization from endoplasmic reticulum (ER) and later cytochrome c release from mitochondria are essential for the apoptotic mechanism.  The rapid Ca2+ flux was dependent on PCI-34051 concn., and was blocked by the PLC inhibitor U73122.  Further, apoptosis was blocked by Ca2+ chelators (BAPTA) and enhanced by Ca2+ effectors (thapsigargin), supporting this model.  These studies show that HDAC8-selective inhibitors have a unique mechanism of action involving PLCγ1 activation and calcium-induced apoptosis, and could offer benefits including a greater therapeutic index for treating T-cell malignancies.  Leukemia (2008) 22, 1026-1034; doi:10.1038/leu.2008.9; published online 7 Feb. 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpW-VB4Wb3h1bVg90H21EOLACvtfcHk0lhmXyDMVcEMxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvVSrtr0%253D&md5=03446a05b396ea9497e4f6f9254cae03</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1038%2Fleu.2008.9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2008.9%26sid%3Dliteratum%253Aachs%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DRamos%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DW.%26aulast%3DSirisawad%26aufirst%3DM.%26aulast%3DVerner%26aufirst%3DE.%26aulast%3DBuggy%26aufirst%3DJ.%2BJ.%26atitle%3DA%2520Novel%2520Histone%2520Deacetylase%25208%2520%2528HDAC8%2529-Specific%2520Inhibitor%2520PCI-34051%2520Induces%2520Apoptosis%2520in%2520T-Cell%2520Lymphomas%26jtitle%3DLeukemia%26date%3D2008%26volume%3D22%26spage%3D1026%26epage%3D1034%26doi%3D10.1038%2Fleu.2008.9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israelian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nawar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manaswiyoungkul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raouf, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedighi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olaoye, O. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdogan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabral, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angeles, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altintas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Araujo, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span> <span> </span><span class="NLM_article-title">Characterization of Conformationally Constrained Benzanilide Scaffolds for Potent and Selective HDAC8 Targeting</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">8634</span>– <span class="NLM_lpage">8648</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01025</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01025" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtl2rs7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=8634-8648&author=M.+M.+Hassanauthor=J.+Israelianauthor=N.+Nawarauthor=G.+Gandaauthor=P.+Manaswiyoungkulauthor=Y.+S.+Raoufauthor=D.+Armstrongauthor=A.+Sedighiauthor=O.+O.+Olaoyeauthor=F.+Erdoganauthor=A.+D.+Cabralauthor=F.+Angelesauthor=R.+Altintasauthor=E.+D.+de+Araujoauthor=P.+T.+Gunning&title=Characterization+of+Conformationally+Constrained+Benzanilide+Scaffolds+for+Potent+and+Selective+HDAC8+Targeting&doi=10.1021%2Facs.jmedchem.0c01025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of Conformationally Constrained Benzanilide Scaffolds for Potent and Selective HDAC8 Targeting</span></div><div class="casAuthors">Hassan, Muhammad Murtaza; Israelian, Johan; Nawar, Nabanita; Ganda, Giovanni; Manaswiyoungkul, Pimyupa; Raouf, Yasir S.; Armstrong, David; Sedighi, Abootaleb; Olaoye, Olasunkanmi O.; Erdogan, Fettah; Cabral, Aaron D.; Angeles, Fabrizio; Altintas, Rabia; de Araujo, Elvin D.; Gunning, Patrick T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8634-8648</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone deacetylases (HDACs) are an attractive therapeutic target for a variety of human diseases.  Currently, all four FDA-approved HDAC-targeting drugs are nonselective, pan-HDAC inhibitors, exhibiting adverse side effects at therapeutic doses.  Although selective HDAC inhibition has been proposed to mitigate toxicity, the targeted catalytic domains are highly conserved.  Herein, we describe a series of rationally designed, conformationally constrained, benzanilide foldamers which selectively bind the catalytic tunnel of HDAC8.  The series includes benzanilides, MMH371, MMH409, and MMH410, which exhibit potent in vitro HDAC8 activity (IC50 = 66, 23, and 66 nM, resp.) and up to 410-fold selectivity for HDAC8 over the next targeted HDAC.  Exptl. and computational analyses of the benzanilide structure docked with human HDAC8 enzyme showed the adoption of a low-energy L-shaped conformer that favors HDAC8 selectivity.  The conformationally constrained HDAC8 inhibitors present an alternative biol. probe for further detg. the clin. utility and safety of pharmacol. knockdown of HDAC8 in diseased cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRExtnmcfaMrVg90H21EOLACvtfcHk0ljW2GPzac0koA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtl2rs7%252FN&md5=d7a4513990c97e0802308c444663c02e</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01025%26sid%3Dliteratum%253Aachs%26aulast%3DHassan%26aufirst%3DM.%2BM.%26aulast%3DIsraelian%26aufirst%3DJ.%26aulast%3DNawar%26aufirst%3DN.%26aulast%3DGanda%26aufirst%3DG.%26aulast%3DManaswiyoungkul%26aufirst%3DP.%26aulast%3DRaouf%26aufirst%3DY.%2BS.%26aulast%3DArmstrong%26aufirst%3DD.%26aulast%3DSedighi%26aufirst%3DA.%26aulast%3DOlaoye%26aufirst%3DO.%2BO.%26aulast%3DErdogan%26aufirst%3DF.%26aulast%3DCabral%26aufirst%3DA.%2BD.%26aulast%3DAngeles%26aufirst%3DF.%26aulast%3DAltintas%26aufirst%3DR.%26aulast%3Dde%2BAraujo%26aufirst%3DE.%2BD.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26atitle%3DCharacterization%2520of%2520Conformationally%2520Constrained%2520Benzanilide%2520Scaffolds%2520for%2520Potent%2520and%2520Selective%2520HDAC8%2520Targeting%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D8634%26epage%3D8648%26doi%3D10.1021%2Facs.jmedchem.0c01025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shouksmith, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gawel, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nawar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raouf, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukhari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manaswiyoungkul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olaoye, O. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabral, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedighi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Araujo, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span> <span> </span><span class="NLM_article-title">Class I/IIb-Selective HDAC Inhibitor Exhibits Oral Bioavailability and Therapeutic Efficacy in Acute Myeloid Leukemia</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00471</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00471" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlygtLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=56-64&author=A.+E.+Shouksmithauthor=J.+M.+Gawelauthor=N.+Nawarauthor=D.+Sinaauthor=Y.+S.+Raoufauthor=S.+Bukhariauthor=L.+Heauthor=A.+E.+Johnsauthor=P.+Manaswiyoungkulauthor=O.+O.+Olaoyeauthor=A.+D.+Cabralauthor=A.+Sedighiauthor=E.+D.+de+Araujoauthor=P.+T.+Gunning&title=Class+I%2FIIb-Selective+HDAC+Inhibitor+Exhibits+Oral+Bioavailability+and+Therapeutic+Efficacy+in+Acute+Myeloid+Leukemia&doi=10.1021%2Facsmedchemlett.9b00471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Class I/IIb-Selective HDAC Inhibitor Exhibits Oral Bioavailability and Therapeutic Efficacy in Acute Myeloid Leukemia</span></div><div class="casAuthors">Shouksmith, Andrew E.; Gawel, Justyna M.; Nawar, Nabanita; Sina, Diana; Raouf, Yasir S.; Bukhari, Shazreh; He, Liying; Johns, Alexandra E.; Manaswiyoungkul, Pimyupa; Olaoye, Olasunkanmi O.; Cabral, Aaron D.; Sedighi, Abootaleb; de Araujo, Elvin D.; Gunning, Patrick T.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-64</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The HDAC inhibitor 4-tert-butyl-N-(4-(hydroxycarbamoyl)phenyl)benzamide (AES-350, I) was identified as a promising preclin. candidate for the treatment of acute myeloid leukemia (AML), an aggressive malignancy with a meagre 24% 5-yr survival rate.  Through screening of low-mol.-wt. analogs derived from the previously discovered novel HDAC inhibitor, AES-135 (II), compd. I demonstrated greater HDAC isoform selectivity, higher cytotoxicity in MV4-11 cells, an improved therapeutic window, and more efficient absorption through cellular and lipid membranes.  Compd. I also demonstrated improved oral bioavailability compared to SAHA in mouse models.  A broad spectrum of expts., including FACS, ELISA, and Western blotting, were performed to support our hypothesis that I dose-dependently triggers apoptosis in AML cells through HDAC inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8ikqWn7KZc7Vg90H21EOLACvtfcHk0lhanfeZjO8RPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlygtLrK&md5=3c21f11cda2a27e00f30a7cd3c700785</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00471%26sid%3Dliteratum%253Aachs%26aulast%3DShouksmith%26aufirst%3DA.%2BE.%26aulast%3DGawel%26aufirst%3DJ.%2BM.%26aulast%3DNawar%26aufirst%3DN.%26aulast%3DSina%26aufirst%3DD.%26aulast%3DRaouf%26aufirst%3DY.%2BS.%26aulast%3DBukhari%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DL.%26aulast%3DJohns%26aufirst%3DA.%2BE.%26aulast%3DManaswiyoungkul%26aufirst%3DP.%26aulast%3DOlaoye%26aufirst%3DO.%2BO.%26aulast%3DCabral%26aufirst%3DA.%2BD.%26aulast%3DSedighi%26aufirst%3DA.%26aulast%3Dde%2BAraujo%26aufirst%3DE.%2BD.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26atitle%3DClass%2520I%252FIIb-Selective%2520HDAC%2520Inhibitor%2520Exhibits%2520Oral%2520Bioavailability%2520and%2520Therapeutic%2520Efficacy%2520in%2520Acute%2520Myeloid%2520Leukemia%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D56%26epage%3D64%26doi%3D10.1021%2Facsmedchemlett.9b00471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Staber, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellido, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davids, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadia, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shustov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tournilhac, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaja, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porkka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoermann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonitsch-Klupp, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haferlach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubicek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayerhoefer, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopfinger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dearden, C.</span></span> <span> </span><span class="NLM_article-title">Consensus Criteria for Diagnosis, Staging, and Treatment Response Assessment of T-Cell Prolymphocytic Leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">1132</span>– <span class="NLM_lpage">1143</span>, <span class="refDoi"> DOI: 10.1182/blood.2019000402</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1182%2Fblood.2019000402" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=31292114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1GjsLbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2019&pages=1132-1143&author=P.+B.+Staberauthor=M.+Herlingauthor=M.+Bellidoauthor=E.+D.+Jacobsenauthor=M.+S.+Davidsauthor=T.+M.+Kadiaauthor=A.+Shustovauthor=O.+Tournilhacauthor=E.+Bachyauthor=F.+Zajaauthor=K.+Porkkaauthor=G.+Hoermannauthor=I.+Simonitsch-Kluppauthor=C.+Haferlachauthor=S.+Kubicekauthor=M.+E.+Mayerhoeferauthor=G.+Hopfingerauthor=U.+Jaegerauthor=C.+Dearden&title=Consensus+Criteria+for+Diagnosis%2C+Staging%2C+and+Treatment+Response+Assessment+of+T-Cell+Prolymphocytic+Leukemia&doi=10.1182%2Fblood.2019000402"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Consensus criteria for diagnosis, staging, and treatment response assessment of T-cell prolymphocytic leukemia</span></div><div class="casAuthors">Staber, Philipp B.; Herling, Marco; Bellido, Mar; Jacobsen, Eric D.; Davids, Matthew S.; Kadia, Tapan Mahendra; Shustov, Andrei; Tournilhac, Olivier; Bachy, Emmanuel; Zaja, Francesco; Porkka, Kimmo; Hoermann, Gregor; Simonitsch-Klupp, Ingrid; Haferlach, Claudia; Kubicek, Stefan; Mayerhoefer, Marius E.; Hopfinger, Georg; Jaeger, Ulrich; Dearden, Claire</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1132-1143</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">1528-0020</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">T-cell prolymphocytic leukemia (T-PLL) is a rare, mature T-cell neoplasm with a heterogeneous clin. course.  With the advent of novel treatment options that will potentially change the management of patients with T-PLL, it has become necessary to produce consensus guidelines for the design and conduct of clin. trials.  The T-PLL International Study group (TPLL-ISG) set out to define standardized criteria for diagnosis, treatment indication, and evaluation of response.  These criteria will facilitate comparison of results from clin. trials in T-PLL, and will thus support clin. decision making, as well as the approval of new therapeutics by healthcare authorities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_-kVKgvO1zrVg90H21EOLACvtfcHk0lhanfeZjO8RPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1GjsLbJ&md5=77e8f358b7e24f0ad1ff286d6e2d8e72</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1182%2Fblood.2019000402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood.2019000402%26sid%3Dliteratum%253Aachs%26aulast%3DStaber%26aufirst%3DP.%2BB.%26aulast%3DHerling%26aufirst%3DM.%26aulast%3DBellido%26aufirst%3DM.%26aulast%3DJacobsen%26aufirst%3DE.%2BD.%26aulast%3DDavids%26aufirst%3DM.%2BS.%26aulast%3DKadia%26aufirst%3DT.%2BM.%26aulast%3DShustov%26aufirst%3DA.%26aulast%3DTournilhac%26aufirst%3DO.%26aulast%3DBachy%26aufirst%3DE.%26aulast%3DZaja%26aufirst%3DF.%26aulast%3DPorkka%26aufirst%3DK.%26aulast%3DHoermann%26aufirst%3DG.%26aulast%3DSimonitsch-Klupp%26aufirst%3DI.%26aulast%3DHaferlach%26aufirst%3DC.%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DMayerhoefer%26aufirst%3DM.%2BE.%26aulast%3DHopfinger%26aufirst%3DG.%26aulast%3DJaeger%26aufirst%3DU.%26aulast%3DDearden%26aufirst%3DC.%26atitle%3DConsensus%2520Criteria%2520for%2520Diagnosis%252C%2520Staging%252C%2520and%2520Treatment%2520Response%2520Assessment%2520of%2520T-Cell%2520Prolymphocytic%2520Leukemia%26jtitle%3DBlood%26date%3D2019%26volume%3D134%26spage%3D1132%26epage%3D1143%26doi%3D10.1182%2Fblood.2019000402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khoury, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Washington, L. B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keating, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D.</span></span> <span> </span><span class="NLM_article-title">A Systematic Approach to Diagnosis of Mature T-Cell Leukemias Reveals Heterogeneity among WHO Categories</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">328</span>– <span class="NLM_lpage">335</span>, <span class="refDoi"> DOI: 10.1182/blood-2004-01-0002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1182%2Fblood-2004-01-0002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=15044256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvFSnsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2004&pages=328-335&author=M.+Herlingauthor=J.+D.+Khouryauthor=L.+B.+T.+Washingtonauthor=M.+Duvicauthor=M.+J.+Keatingauthor=D.+Jones&title=A+Systematic+Approach+to+Diagnosis+of+Mature+T-Cell+Leukemias+Reveals+Heterogeneity+among+WHO+Categories&doi=10.1182%2Fblood-2004-01-0002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">A systematic approach to diagnosis of mature T-cell leukemias reveals heterogeneity among WHO categories</span></div><div class="casAuthors">Herling, Marco; Khoury, Joseph D.; Washington, LaBaron T.; Duvic, Madeleine; Keating, Michael J.; Jones, Dan</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">328-335</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The current World Health Organization (WHO) classification of hematopoietic malignancies defines several types of mature T-cell leukemia including T-cell prolymphocytic leukemia (T-PLL), Sezary syndrome (SS), and T-cell large granular lymphocytic (T-LGL) leukemia.  These neoplasms can show overlapping features with each other and with T-cell lymphomas involving peripheral blood (PB).  We analyzed the spectrum of clinicopathol. features in 102 mature T-cell leukemias and compared them to 10 hepatosplenic T-cell lymphomas that involved PB.  T-PLL, defined as a T-cell leukemia showing rapidly rising PB lymphocyte counts, was the only tumor type expressing the oncoprotein TCL1 (71% of cases) and could present with relatively low lymphocyte levels or small tumor cell morphol.  SS, defined by accompanying erythrodermic skin disease, was frequently assocd. with peripheral eosinophilia but could also develop high nos. of prolymphocytes, esp. late in the disease course.  T-LGL leukemia, defined by accompanying cytopenias or autoimmune phenomena (or both), had the best clin. outcome and generally showed the lowest circulating lymphocyte levels with only a few cases developing marked lymphocytosis.  Using the dominant clin. or phenotypic feature, we describe here the degree of overlap among currently recognized WHO categories and identify areas where further clarification is needed.  Our results indicate that incorporation of addnl. criteria, such as TCL1 expression status and hematol. parameters, can assist in a more accurate classification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS0GwbkDkfirVg90H21EOLACvtfcHk0ljVxBn4lzyqiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvFSnsbc%253D&md5=7fa603b6f7fb2cf31c14a978107ab173</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1182%2Fblood-2004-01-0002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2004-01-0002%26sid%3Dliteratum%253Aachs%26aulast%3DHerling%26aufirst%3DM.%26aulast%3DKhoury%26aufirst%3DJ.%2BD.%26aulast%3DWashington%26aufirst%3DL.%2BB.%2BT.%26aulast%3DDuvic%26aufirst%3DM.%26aulast%3DKeating%26aufirst%3DM.%2BJ.%26aulast%3DJones%26aufirst%3DD.%26atitle%3DA%2520Systematic%2520Approach%2520to%2520Diagnosis%2520of%2520Mature%2520T-Cell%2520Leukemias%2520Reveals%2520Heterogeneity%2520among%2520WHO%2520Categories%26jtitle%3DBlood%26date%3D2004%26volume%3D104%26spage%3D328%26epage%3D335%26doi%3D10.1182%2Fblood-2004-01-0002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ianevski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timonen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kononov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aittokallio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giri, A. K.</span></span> <span> </span><span class="NLM_article-title">SynToxProfiler: An Interactive Analysis of Drug Combination Synergy, Toxicity and Efficacy</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>, <span class="NLM_elocation-id">e1007604</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1007604</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=10.1371%2Fjournal.pcbi.1007604" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=32012154" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;key=1%3ACAS%3A528%3ADC%252BB3cXovFWjsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&author=A.+Ianevskiauthor=S.+Timonenauthor=A.+Kononovauthor=T.+Aittokallioauthor=A.+K.+Giri&title=SynToxProfiler%3A+An+Interactive+Analysis+of+Drug+Combination+Synergy%2C+Toxicity+and+Efficacy&doi=10.1371%2Fjournal.pcbi.1007604"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">SynToxProfiler: an interactive analysis of drug combination synergy, toxicity and efficacy</span></div><div class="casAuthors">Ianevski, Aleksandr; Timonen, Sanna; Kononov, Alexander; Aittokallio, Tero; Giri, Anil K.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e1007604</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Drug combinations are becoming a std. treatment of many complex diseases due to their capability to overcome resistance to monotherapy.  In the current preclin. drug combination screening, the top combinations for further study are often selected based on synergy alone, without considering the combination efficacy and toxicity effects, even though these are crit. determinants for the clin. success of a therapy.  To promote the prioritization of drug combinations based on integrated anal. of synergy, efficacy and toxicity profiles, we implemented a web-based open-source tool, SynToxProfiler (Synergy-Toxicity-Profiler).  When applied to 20 anti-cancer drug combinations tested both in healthy control and T-cell prolymphocytic leukemia (T-PLL) patient cells, as well as to 77 anti-viral drug pairs tested in Huh7 liver cell line with and without Ebola virus infection, SynToxProfiler prioritized as top hits those synergistic drug pairs that showed higher selective efficacy (difference between efficacy and toxicity), which offers an improved likelihood for clin. success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4EyvmeDt93LVg90H21EOLACvtfcHk0ljVxBn4lzyqiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXovFWjsL4%253D&md5=c8b64ed8bc0258e5b4c674ae3195bda8</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1007604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1007604%26sid%3Dliteratum%253Aachs%26aulast%3DIanevski%26aufirst%3DA.%26aulast%3DTimonen%26aufirst%3DS.%26aulast%3DKononov%26aufirst%3DA.%26aulast%3DAittokallio%26aufirst%3DT.%26aulast%3DGiri%26aufirst%3DA.%2BK.%26atitle%3DSynToxProfiler%253A%2520An%2520Interactive%2520Analysis%2520of%2520Drug%2520Combination%2520Synergy%252C%2520Toxicity%2520and%2520Efficacy%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2020%26volume%3D16%26doi%3D10.1371%2Fjournal.pcbi.1007604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU','PDB','5EDU'); return false;">PDB: 5EDU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1T64" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1T64','PDB','1T64'); return false;">PDB: 1T64</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i98"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00420">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_72103"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00420?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00420</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Raw data, structures of inhibitors, additional biochemical assay results, docking results, flow cytometry, pharmacokinetic evaluations, cytotoxicity assays, and compound characterizations (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_002.csv">CSV</a>)</p></li><li><p class="inline"><i>In silico</i> docking binding pose of <b>5</b> in human HDAC6 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_003.pdb">PDB</a>)</p></li><li><p class="inline"><i>In silico</i> docking binding pose of <b>14</b> in human HDAC6 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_004.pdb">PDB</a>)</p></li><li><p class="inline"><i>In silico</i> docking binding pose of <b>14</b> in human HDAC8 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_005.pdb">PDB</a>)</p></li><li><p class="inline"><i>In vitro</i> IC<sub>50</sub> values for <b>1</b>–<b>14</b>, SAHA, ricolinostat, and citarinostat (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_006.xlsx">XLSX</a>)</p></li><li><p class="inline"><i>In vitro</i> IC<sub>50</sub> values for <b>14</b> and citarinostat (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_007.xlsx">XLSX</a>)</p></li><li><p class="inline">Pharmacology of <b>14</b> and citarinostat (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_008.xlsx">XLSX</a>)</p></li><li><p class="inline">Cellular IC<sub>50</sub> values for <b>14</b> and citarinostat (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_009.xlsx">XLSX</a>)</p></li><li><p class="inline">Pharmacokinetic parameters of <b>14</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_010.xlsx">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_001.pdf">jm1c00420_si_001.pdf (9.31 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_002.csv">jm1c00420_si_002.csv (1.37 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_003.pdb">jm1c00420_si_003.pdb (4.32 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_004.pdb">jm1c00420_si_004.pdb (4.32 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_005.pdb">jm1c00420_si_005.pdb (4.32 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_006.xlsx">jm1c00420_si_006.xlsx (10.35 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_007.xlsx">jm1c00420_si_007.xlsx (9.4 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_008.xlsx">jm1c00420_si_008.xlsx (10.24 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_009.xlsx">jm1c00420_si_009.xlsx (9.15 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00420/suppl_file/jm1c00420_si_010.xlsx">jm1c00420_si_010.xlsx (9.59 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00420&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00420%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-12%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00420" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679995837edc3d4e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
