
<html lang="en"     class="pb-page"  data-request-id="445a03e2-0a50-4d87-a61b-f73b71d380d4"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2020.63.issue-15;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c00529;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structural Analysis of Potent Hybrid HIV-1 Protease Inhibitors Containing Bis-tetrahydrofuran in a Pseudosymmetric Dipeptide Isostere" /></meta><meta name="dc.Creator" content="Linah N.  Rusere" /></meta><meta name="dc.Creator" content="Gordon J.  Lockbaum" /></meta><meta name="dc.Creator" content="Mina  Henes" /></meta><meta name="dc.Creator" content="Sook-Kyung  Lee" /></meta><meta name="dc.Creator" content="Ean  Spielvogel" /></meta><meta name="dc.Creator" content="Desaboini Nageswara  Rao" /></meta><meta name="dc.Creator" content="Klajdi  Kosovrasti" /></meta><meta name="dc.Creator" content="Ellen A.  Nalivaika" /></meta><meta name="dc.Creator" content="Ronald  Swanstrom" /></meta><meta name="dc.Creator" content="Nese  Kurt Yilmaz" /></meta><meta name="dc.Creator" content="Celia A.  Schiffer" /></meta><meta name="dc.Creator" content="Akbar  Ali" /></meta><meta name="dc.Description" content="The design, synthesis, and X-ray structural analysis of hybrid HIV-1 protease inhibitors (PIs) containing bis-tetrahydrofuran (bis-THF) in a pseudo-C2-symmetric dipeptide isostere are described. A ..." /></meta><meta name="Description" content="The design, synthesis, and X-ray structural analysis of hybrid HIV-1 protease inhibitors (PIs) containing bis-tetrahydrofuran (bis-THF) in a pseudo-C2-symmetric dipeptide isostere are described. A ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 16, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00529" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00529" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00529" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00529" /></link>
        
    
    

<title>Structural Analysis of Potent Hybrid HIV-1 Protease Inhibitors Containing Bis-tetrahydrofuran in a Pseudosymmetric Dipeptide Isostere | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00529" /></meta><meta property="og:title" content="Structural Analysis of Potent Hybrid HIV-1 Protease Inhibitors Containing Bis-tetrahydrofuran in a Pseudosymmetric Dipeptide Isostere" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0013.jpeg" /></meta><meta property="og:description" content="The design, synthesis, and X-ray structural analysis of hybrid HIV-1 protease inhibitors (PIs) containing bis-tetrahydrofuran (bis-THF) in a pseudo-C2-symmetric dipeptide isostere are described. A series of PIs were synthesized by incorporating bis-THF of darunavir on either side of the Phe-Phe isostere of lopinavir in combination with hydrophobic amino acids on the opposite P2/P2′ position. Structure–activity relationship studies indicated that the bis-THF moiety can be attached at either the P2 or P2′ position without significantly affecting potency. However, the group on the opposite P2/P2′ position had a dramatic effect on potency depending on the size and shape of the side chain. Cocrystal structures of inhibitors with wild-type HIV-1 protease revealed that the bis-THF moiety retained similar interactions as observed in the darunavir–protease complex regardless of the position on the Phe-Phe isostere. Analyses of cocrystal structures and molecular dynamics simulations provide insights into optimizing HIV-1 PIs containing bis-THF in non-sulfonamide dipeptide isosteres." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00529"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00529">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00529&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00529&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00529&amp;href=/doi/10.1021/acs.jmedchem.0c00529" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 15</span><span class="cit-fg-pageRange">, 8296-8313</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/15" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00130" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00546" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structural Analysis of Potent Hybrid HIV-1 Protease Inhibitors Containing Bis-tetrahydrofuran in a Pseudosymmetric Dipeptide Isostere</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Linah N. Rusere</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Linah N. Rusere</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, Worcester, Massachusetts 01605, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Linah+N.++Rusere">Linah N. Rusere</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gordon J. Lockbaum</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gordon J. Lockbaum</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, Worcester, Massachusetts 01605, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gordon+J.++Lockbaum">Gordon J. Lockbaum</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2720-6984" title="Orcid link">http://orcid.org/0000-0003-2720-6984</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mina Henes</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mina Henes</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, Worcester, Massachusetts 01605, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mina++Henes">Mina Henes</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sook-Kyung Lee</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sook-Kyung Lee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, and the UNC Center for AIDS Research, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sook-Kyung++Lee">Sook-Kyung Lee</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ean Spielvogel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ean Spielvogel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, and the UNC Center for AIDS Research, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ean++Spielvogel">Ean Spielvogel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Desaboini Nageswara Rao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Desaboini Nageswara Rao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, Worcester, Massachusetts 01605, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Desaboini+Nageswara++Rao">Desaboini Nageswara Rao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Klajdi Kosovrasti</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Klajdi Kosovrasti</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, Worcester, Massachusetts 01605, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Klajdi++Kosovrasti">Klajdi Kosovrasti</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ellen A. Nalivaika</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ellen A. Nalivaika</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, Worcester, Massachusetts 01605, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ellen+A.++Nalivaika">Ellen A. Nalivaika</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ronald Swanstrom</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ronald Swanstrom</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, and the UNC Center for AIDS Research, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ronald++Swanstrom">Ronald Swanstrom</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nese Kurt Yilmaz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nese Kurt Yilmaz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, Worcester, Massachusetts 01605, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nese++Kurt+Yilmaz">Nese Kurt Yilmaz</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5036-676X" title="Orcid link">http://orcid.org/0000-0002-5036-676X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Celia A. Schiffer</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Celia A. Schiffer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, Worcester, Massachusetts 01605, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#357650595c541b66565d5c535350477540585446465850511b505140"><span class="__cf_email__" data-cfemail="9ad9fff6f3fbb4c9f9f2f3fcfcffe8daeff7fbe9e9f7fffeb4fffeef">[email protected]</span></a>. Phone: +1 508 856 8008. Fax: +1 508 856 6464.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Celia+A.++Schiffer">Celia A. Schiffer</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2270-6613" title="Orcid link">http://orcid.org/0000-0003-2270-6613</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Akbar Ali</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Akbar Ali</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, Worcester, Massachusetts 01605, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#8dcce6efecffa3cce1e4cdf8e0ecfefee0e8e9a3e8e9f8"><span class="__cf_email__" data-cfemail="7938121b180b57381510390c14180a0a141c1d571c1d0c">[email protected]</span></a>. Phone: +1 508 856 8873. Fax: +1 508 856 6464.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Akbar++Ali">Akbar Ali</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3491-791X" title="Orcid link">http://orcid.org/0000-0003-3491-791X</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00529&amp;href=/doi/10.1021%2Facs.jmedchem.0c00529" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 15</span><span class="cit-pageRange">, 8296–8313</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 16, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>31 March 2020</li><li><span class="item_label"><b>Published</b> online</span>16 July 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 August 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00529" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00529</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8296%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DLinah%2BN.%2BRusere%252C%2BGordon%2BJ.%2BLockbaum%252C%2BMina%2BHenes%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D15%26contentID%3Dacs.jmedchem.0c00529%26title%3DStructural%2BAnalysis%2Bof%2BPotent%2BHybrid%2BHIV-1%2BProtease%2BInhibitors%2BContaining%2BBis-tetrahydrofuran%2Bin%2Ba%2BPseudosymmetric%2BDipeptide%2BIsostere%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8313%26publicationDate%3DAugust%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00529"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">660</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">1</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00529" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structural Analysis of Potent Hybrid HIV-1 Protease Inhibitors Containing Bis-tetrahydrofuran in a Pseudosymmetric Dipeptide Isostere&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Linah&quot;,&quot;last_name&quot;:&quot;N. Rusere&quot;},{&quot;first_name&quot;:&quot;Gordon&quot;,&quot;last_name&quot;:&quot;J. Lockbaum&quot;},{&quot;first_name&quot;:&quot;Mina&quot;,&quot;last_name&quot;:&quot;Henes&quot;},{&quot;first_name&quot;:&quot;Sook-Kyung&quot;,&quot;last_name&quot;:&quot;Lee&quot;},{&quot;first_name&quot;:&quot;Ean&quot;,&quot;last_name&quot;:&quot;Spielvogel&quot;},{&quot;first_name&quot;:&quot;Desaboini&quot;,&quot;last_name&quot;:&quot;Nageswara Rao&quot;},{&quot;first_name&quot;:&quot;Klajdi&quot;,&quot;last_name&quot;:&quot;Kosovrasti&quot;},{&quot;first_name&quot;:&quot;Ellen&quot;,&quot;last_name&quot;:&quot;A. Nalivaika&quot;},{&quot;first_name&quot;:&quot;Ronald&quot;,&quot;last_name&quot;:&quot;Swanstrom&quot;},{&quot;first_name&quot;:&quot;Nese&quot;,&quot;last_name&quot;:&quot;Kurt Yilmaz&quot;},{&quot;first_name&quot;:&quot;Celia&quot;,&quot;last_name&quot;:&quot;A. Schiffer&quot;},{&quot;first_name&quot;:&quot;Akbar&quot;,&quot;last_name&quot;:&quot;Ali&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;16&quot;,&quot;issue&quot;:&quot;15&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;8296-8313&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00529&quot;},&quot;abstract&quot;:&quot;The design, synthesis, and X-ray structural analysis of hybrid HIV-1 protease inhibitors (PIs) containing bis-tetrahydrofuran (bis-THF) in a pseudo-C2-symmetric dipeptide isostere are described. A series of PIs were synthesized by incorporating bis-THF of darunavir on either side of the Phe-Phe isostere of lopinavir in combination with hydrophobic amino acids on the opposite P2/P2′ position. Structure–activity relationship studies indicated that the bis-THF moiety can be attached at either the P2 or P2′ position without significantly affecting potency. However, the group on the opposite P2/P2′ position had a dramatic effect on potency depending on the size and shape of the side chain. Cocrystal structures of inhibitors with wild-type HIV-1 protease revealed that the bis-THF moiety retained similar interactions as observed in the darunavir–protease complex regardless of the position on the Phe-Phe isostere. Analyses of cocrystal structures and molecular dynamics simulations provide insights into optimizing HIV&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00529&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00529" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00529&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00529" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00529&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00529" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00529&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00529&amp;href=/doi/10.1021/acs.jmedchem.0c00529" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00529" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00529" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (10 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00529%26sid%3Dliteratum%253Aachs%26pmid%3D32672965%26genre%3Darticle%26aulast%3DRusere%26date%3D2020%26atitle%3DStructural%2BAnalysis%2Bof%2BPotent%2BHybrid%2BHIV-1%2BProtease%2BInhibitors%2BContaining%2BBis-tetrahydrofuran%2Bin%2Ba%2BPseudosymmetric%2BDipeptide%2BIsostere%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D15%26spage%3D8296%26epage%3D8313%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291120" title="Hydrophobicity">Hydrophobicity</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291763" title="Noncovalent interactions">Noncovalent interactions</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/15" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/jmcmar.2020.63.issue-15/20200813/jmcmar.2020.63.issue-15.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The design, synthesis, and X-ray structural analysis of hybrid HIV-1 protease inhibitors (PIs) containing bis-tetrahydrofuran (bis-THF) in a pseudo-C<sub>2</sub>-symmetric dipeptide isostere are described. A series of PIs were synthesized by incorporating bis-THF of darunavir on either side of the Phe-Phe isostere of lopinavir in combination with hydrophobic amino acids on the opposite P2/P2′ position. Structure–activity relationship studies indicated that the bis-THF moiety can be attached at either the P2 or P2′ position without significantly affecting potency. However, the group on the opposite P2/P2′ position had a dramatic effect on potency depending on the size and shape of the side chain. Cocrystal structures of inhibitors with wild-type HIV-1 protease revealed that the bis-THF moiety retained similar interactions as observed in the darunavir–protease complex regardless of the position on the Phe-Phe isostere. Analyses of cocrystal structures and molecular dynamics simulations provide insights into optimizing HIV-1 PIs containing bis-THF in non-sulfonamide dipeptide isosteres.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16163" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16163" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Combinations of drugs targeting HIV-1 enzymes essential for viral replication have been highly effective in reducing viral load in infected individuals and significantly increasing their life expectancy.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> HIV-1 protease inhibitors (PIs) have played an important role as a highly potent component of combination antiretroviral therapy (cART) for people living with HIV-1. There are nine HIV-1 PIs approved by the US FDA; however, clinical use of most of these has diminished over the years due to side effects, unfavorable pharmacokinetics, and drug resistance. Currently, only three PIs, darunavir (DRV), lopinavir (LPV), and atazanavir (ATV) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) – boosted with small doses of ritonavir (RTV) – are being used in cART as they have comparatively higher potency and better resistance profiles.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of HIV-1 protease inhibitors darunavir (DRV) (<b>1</b>), lopinavir (LPV) (<b>2</b>), atazanavir (ATV) (<b>3</b>), previously reported hybrid compounds <b>4</b> and <b>5</b>, and new hybrid compounds designed by incorporating the bis-THF moiety of DRV into the pseudosymmetric Phe-Phe dipeptide isostere of LPV at the P2 (<b>12a</b>–<b>h</b>) or P2′ position (<b>15a</b>–<b>c</b>), in combination with an ATV-like hydrophobic amino acid moiety at the P2′ or P2 position, respectively. The canonical nomenclature is used to indicate the position of each inhibitor moiety.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Most HIV-1 PIs are substrate-based peptidomimetics containing a hydroxyl group as transition-state mimetic and diverse moieties that target the S1/S1′ and S2/S2′ subsites of the enzyme.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> Among the PIs in clinical use, DRV is the most potent with the highest barrier to resistance and efficacy against multidrug-resistant HIV-1 strains.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> The excellent potency and resistance profiles of DRV are attributed to the bis-tetrahydrofuran (bis-THF) moiety, which forms strong hydrogen bonding interactions with the main-chain NH of Asp29 and Asp30 in the S2 subsite of the enzyme.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Moreover, the extensive van der Waals (vdW) interactions of the bis-THF moiety, along with the sulfonamide-based Phe-Leu dipeptide isostere and the 4-aminobenzene group, also significantly contribute to the potency of DRV.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Due to these unique binding characteristics, the P2 bis-THF moiety of DRV has emerged as a privileged ligand for targeting the S2 subsite of HIV-1 protease especially when incorporated into the (<i>R</i>)-(hydroxyethylamino)sulfonamide isostere.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></div><div class="NLM_p">Efforts to improve potency, pharmacokinetics, and resistance profiles of HIV-1 PIs have led to the discovery of exceptionally potent compounds exhibiting a wide range of properties.<a onclick="showRef(event, 'ref5 ref10 ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref5 ref10 ref11 ref12 ref13 ref14">(5,10−14)</a> In recent years, these efforts have focused mainly on exploring DRV analogues with similar sulfonamide-based dipeptide isosteres and P2/P2′ moieties. Many analogues of DRV with modifications at the P1/P1′ and P2/P2′ positions have been explored extensively,<a onclick="showRef(event, 'ref5 ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref5 ref15 ref16 ref17 ref18">(5,15−18)</a> including novel bis-THF analogues showing improved hydrogen bonding and vdW interactions with the protease.<a onclick="showRef(event, 'ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21 ref22">(19−22)</a> Moreover, detailed structural analyses informing structure-based design of inhibitors with the DRV scaffold have been reported numerous times including recent structures determined by neutron crystallography.<a onclick="showRef(event, 'ref5 ref7 ref8 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref5 ref7 ref8 ref23 ref24 ref25">(5,7,8,23−25)</a></div><div class="NLM_p">However, very few inhibitors containing the bis-THF moiety in non-sulfonamide dipeptide isosteres have been explored.<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26−28)</a> Previously, Chen and colleagues at Abbott Laboratories explored PIs with bis-THF on either side of the pseudo-C<sub>2</sub>-symmetric Phe-Phe dipeptide isostere of LPV,<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> while Cannizzaro and co-workers at Gilead Sciences designed similar analogues using the aza-dipeptide isostere of ATV.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Although some of the Phe-Phe isostere-based compounds, such as <b>4</b> and <b>5</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), exhibited significant antiviral activity against wild-type HIV-1, in the absence of direct comparison, it is unclear if their potency is similar to that of DRV.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Also, due to limited structure–activity relationship studies, the S2/S2′ subsite preference of the bis-THF moiety in the context of the Phe-Phe isostere remains unclear,<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> as is the identity of an optimal moiety for targeting the opposite S2/S2′ subsite.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Moreover, no cocrystal structures of such hybrid PIs have been reported, limiting opportunities for structure-guided design and optimization.</div><div class="NLM_p">In this study, we investigate HIV-1 PIs containing the bis-THF moiety of DRV in the Phe-Phe dipeptide isostere of LPV with the aim to identify compounds with potency similar to DRV but with an alternate non-sulfonamide scaffold. Protease–inhibitor molecular interactions are characterized by X-ray crystallography and molecular dynamics (MD) simulations to aid structure-guided optimization of these hybrid PIs. In addition, the interdependence of HIV-1 protease subsites is explored to provide insight into subsite preferences and influence of different inhibitor moieties on binding and potency.</div><div class="NLM_p last">The active site of HIV-1 protease exhibits subsite interdependence in substrate recognition and inhibitor binding,<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29,30)</a> but this feature of the protease has rarely been considered in inhibitor design. While identifying optimal moieties for targeting each of the subsites in the protease active site is important, understanding of the subsite interdependence and selection of the best combination of moieties are also crucial to achieving higher potency especially against drug-resistant variants.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The bis-THF moiety potentially can be incorporated in diverse dipeptide isosteres in combination with suitable ligands targeting the other subsites. Thus, a series of hybrid HIV-1 PIs were synthesized by incorporating the bis-THF moiety of DRV on either side of the pseudo-C<sub>2</sub>-symmetric Phe-Phe dipeptide isostere of LPV in combination with ATV-like hydrophobic P2/P2′ ligands of varying sizes and shapes (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). The hybrid PIs were tested for inhibitory activity against wild-type protease and two drug-resistant variants and for antiviral activity in cellular assays. We also determined high-resolution X-ray crystal structures of all designed hybrid compounds and previously reported PI <b>4</b> bound to wild-type HIV-1 protease, and characterized the binding interactions of bis-THF in the context of the Phe-Phe isostere. In addition, molecular dynamics simulations were utilized to assess the stability of protease–inhibitor interactions in solution. The comprehensive structural and dynamic analyses of hybrid PIs have provided insights into inhibitor binding and interdependence of protease subsites.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34786" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34786" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The synthesis of hybrid PIs incorporating the bis-THF moiety at either the P2 or P2′ position is outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The required Phe-Phe dipeptide isostere core intermediate <b>7</b> was synthesized from phenylalanine following previously reported methods with minor modifications (see the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Supporting Information</a> for details).<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Reaction of the dipeptide isostere <b>7</b> with bis-THF activated carbonate <b>8</b> in the presence of DIEA provided intermediate <b>9</b>. Debenzylation of <b>9</b> using ammonium formate and 10% palladium on activated carbon in methanol gave the corresponding deprotected amine <b>10</b>, which was subsequently coupled with the amino acid derivatives <b>11a</b>–<b>h</b> using HATU and DIEA to provide the target compounds <b>12a</b>–<b>h</b> with bis-THF at the P2 position. While <i>N</i>-(methoxycarbonyl)-protected amino acid derivatives <b>11a</b> and <b>11e</b> were commercially available, intermediates <b>11b</b>–<b>d</b> and <b>11f</b>–<b>h</b> were prepared from the corresponding amino acids by reaction with methyl chloroformate under basic conditions following a reported procedure.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Protease Inhibitors Incorporating Bis-tetrahydrofuran in a Pseudosymmetric Phe-Phe Dipeptide Isostere<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) K<sub>2</sub>CO<sub>3</sub>, KOH, BnCl, H<sub>2</sub>O, 100 °C, 5 h; (b) NaNH<sub>2</sub>, CH<sub>3</sub>CN, MTBE, 0 °C, 2 h, then BnMgCl (2 M in THF), RT, 14 h; (c) NaBH<sub>4</sub>, MsOH, <i>i</i>-PrOH, EGDME, 0 °C, 12 h; (d) NaBH<sub>4</sub>, TEA, DMA, 0–15 °C, 2 h; (e) DIEA, CH<sub>3</sub>CN, RT, 24 h; (f) Pd/C, HCO<sub>2</sub>NH<sub>4</sub>, MeOH, 50 °C, 12 h; (g) <i>N</i>-(methoxycarbonyl)-capped amino acid, HATU, DIEA, DMF, RT, 4 h.</p></p></figure><div class="NLM_p last">The PIs <b>15a</b>–<b>c</b> with the bis-THF at the P2′ position were synthesized using a similar reaction sequence. Coupling of the dipeptide isostere <b>7</b> with acids <b>11a</b>–<b>c</b> under HATU peptide coupling conditions gave the corresponding intermediates <b>13a</b>–<b>c</b>. Catalytic debenzylation of compounds <b>13a</b>–<b>c</b> followed by coupling with bis-THF activated carbonate <b>8</b> in the presence of DIEA provided the target compounds <b>15a</b>–<b>c</b>. A pair of previously reported PIs <b>4</b> and <b>5</b> containing the dimethylphenoxy acetate and bis-THF moieties as the P2 or P2′ ligands in the Phe-Phe isostere were also synthesized using the same reaction sequences.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21632" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21632" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We explored hybrid HIV-1 PIs containing the bis-THF moiety of DRV in the Phe-Phe isostere of LPV with the goal of characterizing the binding interactions of this P2 ligand in the context of a pseudo-C<sub>2</sub>-symmetric dipeptide isostere. The bis-THF moiety was used in combination with diverse <i>N</i>-(methoxycarbonyl)-capped amino acid derivatives similar to the P2/P2′ moiety of ATV to optimize hydrogen bonding and vdW interactions in both the S2 and S2′ subsites of HIV-1 protease. To explore the subsite preference for the bis-THF moiety, pairs of compounds were synthesized with the bis-THF moiety attached at either the P2 (<b>12a</b>–<b>c</b>) or P2′ (<b>15a</b>−<b>c</b>) position. Considering potential subsite interdependence, incorporating a relatively small and conformationally flexible moiety at the other P2/P2′ position was expected to allow the bis-THF moiety to optimally interact with the backbone atoms in the S2/S2′ subsite, likely improving overall inhibitor binding with the protease. Structure–activity relationships were explored by incorporating hydrophobic amino acids of varying sizes and shapes to identify an optimal ligand for targeting the other S2/S2′ subsite.</div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Enzyme Inhibition and Antiviral Assays</h3><div class="NLM_p">The potency of PIs was assessed using biochemical and antiviral assays. The enzyme inhibition constants (<i>K</i>i) were determined against wild-type protease and two drug-resistant variants (I84V and I50V/A71V) using a highly sensitive fluorogenic assay (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Both I50V and I84V are major protease inhibitor resistance mutations that reduce susceptibility to LPV and DRV. For a subset of compounds, antiviral potencies (EC<sub>50</sub>) were determined against wild-type HIV-1 (NL4–3 strain) using a cell-based antiviral assay. LPV and DRV were used as controls in all assays.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Enzyme Inhibitory Activity and Antiviral Potency Values of Hybrid HIV-1 Protease Inhibitors</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0012.gif" alt="" id="gr12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Enzyme inhibition constants against wild-type HIV-1 protease and drug-resistant variants.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Antiviral potency against wild-type HIV-1; NT = not tested.</p></div></div><div></div></div><div class="NLM_p">Compounds <b>12a</b> and <b>15a</b>, with the bis-THF moiety at the P2 and P2′ position, respectively, and the <i>N</i>-(methoxycarbonyl)-<i>tert</i>-leucine at the opposite P2/P2′ position, showed excellent inhibitory potencies against wild-type protease and the drug-resistant variants. Compared to DRV and LPV, both compounds were less active, particularly against the I84V variant. Interestingly, compounds <b>12a</b> and <b>15a</b> exhibited similar <i>K</i>i values, though the latter was 4-fold more active against the I50V/A71V variant, indicating no clear S2/S2′ subsite preference for the bis-THF moiety. However, in antiviral assays, compound <b>12a</b> was 3-fold more potent than <b>15a</b> against wild-type HIV-1. Compound <b>12a</b>, with the bis-THF moiety at the P2 position, exhibited excellent antiviral potency (EC<sub>50</sub> = 11 nM) comparable to LPV (EC<sub>50</sub> = 12 nM) and DRV (EC<sub>50</sub> = 5 nM). A similar potency profile was observed for previously reported compounds <b>4</b> and <b>5</b> containing the bis-THF and the dimethylphenoxy acetate moieties as the P2 and P2′ ligands. Compound <b>5</b> exhibited 2-fold better antiviral potency than <b>4</b>, indicating a reverse subsite preference for the bis-THF moiety relative to the <b>12a</b>/<b>15a</b> pair. The small variation in the antiviral potencies of compounds <b>12a</b>/<b>15a</b> and <b>4</b>/<b>5</b> indicates that the position of the bis-THF moiety with respect to the central hydroxyl group of the Phe-Phe isostere, though not critical for binding, may affect potency in cellular assays.</div><div class="NLM_p">Although the potency of the hybrid compounds was not particularly sensitive to the position of the bis-THF moiety, compounds with different hydrophobic amino acid derivatives at the opposite P2/P2′ position showed markedly different inhibitory potencies. The compounds with the cycloleucine moiety, <b>12b</b> and <b>15b</b>, were significantly less active (>2500-fold and 560-fold, respectively) than the corresponding <i>tert</i>-leucine analogues <b>12a</b> and <b>15a</b>. The dramatic loss of potency is likely due to the altered shape and flexibility of the hydrophobic amino acid moiety. This was further supported by the fact that the corresponding cyclopentylglycine analogues <b>12c</b> and <b>15c</b>, though also an order of magnitude less active than <b>12a</b> and <b>15a</b>, still exhibited sub-nanomolar to low nM potency against wild-type protease and drug-resistant variants. Together, these data confirm that the hydrophobic amino acid moiety at the opposite P2/P2′ position significantly influences potency in both biochemical and antiviral assays.</div><div class="NLM_p last">Since potency of the compounds with bis-THF moiety attached to the Phe-Phe isostere largely depended on the hydrophobic amino acid moiety at the opposite side of the core scaffold, we investigated the size and shape requirements for optimally targeting the S2′ subsite. Analogues of compound <b>12a</b> were synthesized with diverse hydrophobic amino acid derivatives as P2′ ligands while keeping the bis-THF moiety at the P2 position. The alanine analogue <b>12d</b> exhibited excellent inhibitory potency with <i>K</i>i values against resistant variants comparable to LPV and DRV. The corresponding valine analogue <b>12e</b> was 4-fold less active against wild-type protease but maintained similar inhibitory potency to <b>12a</b> against the resistant variants. However, compared to <b>12a</b>, compounds <b>12d</b> and <b>12e</b> were 6- and 5-fold less active in antiviral assays, respectively, indicating that factors other than binding affinity are responsible for lower antiviral potency. The cyclopropyl analogue <b>12f</b> was considerably less active than <b>12a</b>, with <i>K</i>i values similar to the cyclopentyl analogue <b>12c</b>, strongly suggesting that cyclic moieties at the P2/P2′ position are unfavorable. Replacement of <i>tert</i>-leucine with an isoleucine (compound <b>12g</b>) or alloisoleucine (compound <b>12h</b>) moiety at the P2′ position also resulted in slightly lower activity against wild-type protease but similar or better potency against resistant variants. This further confirmed earlier findings that the P2/P2′ group opposite to the bis-THF moiety on the Phe-Phe core has a significant impact on potency. These results are in contrast to the sulfonamide-based dipeptide isostere of DRV, where diverse P2′ modifications result in relatively minor changes in potency.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> The identity of the P2/P2′ moiety opposite to the bis-THF on the Phe-Phe core considerably impacts potency in both biochemical and antiviral assays.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Analysis of Protease–Inhibitor Complexes</h3><div class="NLM_p">To characterize protease–inhibitor molecular interactions, 12 high-resolution (1.8–2.0 Å) crystal structures of hybrid PIs and compound <b>4</b> bound to wild-type HIV-1 protease of the NL4–3 strain were determined. A cocrystal structure of LPV bound to the same protease enzyme was also determined for direct comparison. The crystallographic data collection and refinement statistics are summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Table S1</a>. Following convention, protease chains were assigned chain A (nonprime) or chain B (prime) depending on the interactions between the central hydroxyl group of the Phe-Phe isostere and the catalytic aspartates Asp25/Asp25′.</div><div class="NLM_p">LPV and DRV achieve low picomolar affinity to wild-type HIV-1 protease by making a number of hydrogen bonds and vdW interactions with the active site residues (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Figure S1</a>). The interactions of the transition-state mimetic secondary hydroxyl group with the catalytic aspartates, together with the vdW interactions of the adjacent P1/P1′ moieties, are critical for overall inhibitor binding. Together, these interactions determine the positions of the P2/P2′ moieties in the active site, which in turn determine the patterns of hydrogen bonding and vdW interactions of these moieties with the protease. DRV contains a sulfonamide-based Phe-Leu dipeptide isostere core with the (<i>R</i>)-configuration of the secondary hydroxyl group, whereas LPV has a pseudo-C<sub>2</sub>-symmetric Phe-Phe dipeptide isostere with the (<i>S</i>)-configuration of the hydroxyl group. Though chemically quite distinct, the core scaffolds of LPV and DRV make largely similar hydrogen bonding and vdW interactions with the protease. These inhibitors, however, differ in their interactions with the protease in the S2 and S2′ subsites.</div><div class="NLM_p">The bis-THF moiety of DRV forms direct hydrogen bonds with the main-chain NH of Asp29 and Asp30 in the S2 subsite, while the 4-aminobenzene makes a direct hydrogen bond with the main-chain carbonyl of Asp30′ as well as a water-mediated interaction with the side-chain carboxylate of the same residue in the S2′ subsite (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Figure S1</a>). The cyclic urea moiety of LPV makes direct hydrogen bonds with the main-chain NH and side-chain carboxylate of Asp29. The same moiety also makes water-mediated interactions with the main-chain carbonyl of Gly27 and the side-chain carboxylate of Asp29 in the S2 subsite. However, in contrast to the P2′ moiety of DRV, the dimethylphenoxy acetate moiety of LPV only makes hydrophobic interactions in the S2′ subsite (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Figure S1</a>). The hybrid compounds combine the pseudo-C<sub>2</sub>-symmetric Phe-Phe isostere of LPV with the bis-THF moiety of DRV and an ATV-like amino acid moiety to enhance direct hydrogen bonding and vdW interactions in the S2 and S2′ subsites.</div><div class="NLM_p">The cocrystal structures of hybrid compounds revealed similar binding conformations of the Phe-Phe isostere and the bis-THF moiety as observed in the protease complexes with LPV and DRV, respectively (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Importantly, the bis-THF moiety is positioned to interact with the main-chain NH of residues Asp29 and Asp30, mimicking the binding interactions of DRV (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). This binding conformation is maintained in all hybrid compounds including <b>4</b> regardless of the position of the bis-THF moiety. As a result, within each compound series (with the bis-THF moiety at either the P2 or P2′ position), the structures aligned well with variations occurring only at the hydrophobic amino acid moiety (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Overlays of protease cocrystal structures bound to LPV (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJB">6PJB</a>) (gray, top) and DRV (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DGX">6DGX</a>) (brown, bottom) with compounds <b>12a</b> (bis-THF at P2; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJD">6PJD</a>) (green), <b>15a</b> (bis-THF at P2′; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJM">6PJM</a>) (orange), and <b>4</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJC">6PJC</a>) (salmon) revealed similar binding conformations of the Phe-Phe isostere and the bis-THF moiety as observed in the protease complexes with LPV and DRV, respectively. Note that DRV is flipped 180° to compare with bis-THF of <b>15a</b> at the P2′ position.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structures of wild-type HIV-1 protease in complex with LPV (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJB">6PJB</a>) (A) and hybrid compounds <b>4</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJC">6PJC</a>) (B), <b>12a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJD">6PJD</a>) (C), and <b>15a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJM">6PJM</a>) (D). The bis-THF moiety maintains the hydrogen bonding interactions with the main-chain NH of Asp29/Asp29′ and Asp30/Asp30′ in the S2/S2′ subsites regardless of its position on the Phe-Phe core. Hydrogen bonds and water-mediated interactions as determined by PyMol are shown as black dashed lines with waters shown as red spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Superposition of HIV-1 protease complexes with compounds <b>12a</b>–<b>h</b> (A) and compounds <b>15a</b>–<b>c</b> (B), showing similar binding conformations of the bis-THF moiety regardless of the position and minor differences at the Phe-Phe isostere but major differences at the P2/P2′ amino acid moiety. The protease active site is shown in a ribbon representation, with bound inhibitors depicted as sticks.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In all complexes, the central hydroxyl group of the Phe-Phe core maintains the key hydrogen bonds with the side-chain carboxylates of the catalytic residues Asp25/Asp25′. Nevertheless, subtle differences in hydrogen bonding and vdW interactions were observed between the Phe-Phe cores in LPV and the hybrid compounds. Notably, one of the four water-mediated interactions with the flap residue Ile50 is consistently longer (2.9–3.4 Å) than what is observed in the LPV structure (2.7 Å) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Table S2</a>). The two amide NH groups on each side of the Phe-Phe core could potentially interact with the backbone carbonyl of Gly27 and Gly27′, but in the LPV structure, only the Gly27 is close enough to form a hydrogen bond (2.9 versus 3.9 Å) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). This phenomenon is also observed in the protease complex with compound <b>4</b> (2.9 versus 3.7 Å) indicating that the large hydrophobic P2′ moiety prevents the additional hydrogen bond from occurring with Gly27′. In contrast, in all hybrid compound structures, the core NH groups are within 3.5 Å from the backbone carbonyls of Gly27 and Gly27′ (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Table S2</a>), although simultaneous interactions with both Gly27/Gly27′ may not be stable. Nevertheless, the combination of bis-THF and a flexible hydrophobic amino acid moiety allows both core NH groups to interact with the main-chain carbonyls of residues Gly27/Gly27′.</div><div class="NLM_p">The Phe-Phe isostere in hybrid compounds, in addition to maintaining key hydrogen bonding interactions with the protease, makes a number of favorable vdW interactions with the hydrophobic residues that make up the S1/S1′ subsites. Despite the same phenylalanine side chains at the P1 and P1′ positions, there are a few notable differences in vdW contacts with the protease (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Figure S2</a>). Compared to LPV, the P1/P1′ phenyl group adjacent to the amino acid moiety is shifted toward Val82 resulting in improved vdW contacts with that residue, particularly for compounds with larger P2/P2′ amino acid moieties. These localized gains in vdW contacts are slightly lower for compounds <b>15a</b>–<b>c</b> with bis-THF at the P2′ position than their corresponding analogues <b>12a</b>–<b>c</b>. The shift of the P1/P1′ phenyl group toward Val82, creating more vdW contacts, resulted in a nearly equal and opposite loss of contacts with Ile50 in the same subsite. The packing of the phenyl group adjacent to the bis-THF moiety resulted in a small increase in vdW contacts with the 80s loop (residues 81, 82, and 84) but a decrease in contacts with flap residue Gly49. For compounds <b>15a</b>–<b>c</b>, gains in vdW contacts are smaller and the losses are greater. Overall, despite localized losses and gains in vdW interactions for both series of compounds, the packing of the P1 and P1′ groups remains generally similar to that of LPV.</div><div class="NLM_p">The cocrystal structures of hybrid compounds (and <b>4</b>) also revealed similar binding interactions of the bis-THF moiety as in the DRV–protease complex regardless of the position, with the two ring oxygen atoms positioned to form hydrogen bonds with the main-chain NH of Asp29 and Asp30. However, there are subtle differences in the position and puckering of the bis-THF rings, causing the carbamate-linked THF ring oxygen atom to slightly shift away from the backbone NH of Asp30 compared to DRV (3.1–3.4 versus 2.9 Å) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>), whereas the oxygen atom of the other THF ring maintains a similar distance to the backbone NH of Asp29 (2.7–2.9 versus 2.8 Å) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Table S2</a>). For compounds <b>12a</b>–<b>c</b>, these subtle changes in the bis-THF moiety arise from the differences in the binding of core isosteres, where the P1 benzyl group and the carbamate are shifted toward the B-chain compared to their position in DRV. This is likely due to the additional carbon in the core or to accommodate the larger P1′ group. In compounds <b>15a</b>–<b>c</b>, the core scaffold from the central hydroxyl to the bis-THF moiety is identical. However, the opposite stereochemistry of the central hydroxyl causes a subtle shift preventing the bis-THF moiety from optimally interacting with Asp30′ (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Despite these minor differences in the position of the bis-THF moiety, the hybrid compounds make similar hydrogen bonding interactions with the protease in the S2/S2′ subsite compared to DRV.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comparison of the binding conformations of the bis-THF moiety and protease–inhibitor intramolecular distances with the main-chain NH of Asp29 and Asp30 in the S2/S2′ subsite of HIV-1 protease. As observed in the DRV–protease complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DGX">6DGX</a>), the bis-THF moiety retained similar interactions with the protease in complexes with compound <b>4</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJC">6PJC</a>) (top), the most potent hybrid compounds <b>12a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJD">6PJD</a>) and <b>15a</b> (PBD <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJM">6PJM</a>) (middle), and the least potent hybrid compounds <b>12b</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJE">6PJE</a>) and <b>15b</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJN">6PJN</a>) (bottom), irrespective of the position on the Phe-Phe core.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The bis-THF moiety of DRV is critical to its potency, not only for the hydrogen bonding interactions with Asp29 and Asp30 but also vdW packing with residues in the S2 subsite, particularly Val32 and Ile47. In hybrid compounds, the minor change in the position and puckering of the bis-THF moiety causes reduced vdW interactions with Ala28 and Asp30 but increased interactions with Ile47 and Gly48 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Figure S3</a>). The main-chain carbonyl of Gly48 is oriented toward the hydrophobic region of bis-THF, so enhanced contacts with this residue may be unfavorable. In contrast, compounds <b>4</b>, <b>15a</b>, and <b>15b</b> do not make these unfavorable contacts with Gly48. With the exception of enhanced interactions with Ile47, the bis-THF moiety in most hybrid compounds makes less optimal vdW interactions with the protease than those of DRV.</div><div class="NLM_p">The major difference between the hybrid compounds and the previously reported Phe-Phe core-based bis-THF containing PIs <b>4</b> and <b>5</b> is the introduction of an ATV-like amino acid moiety to target the opposite S2/S2′ subsite. The amino acid side chain interacts with hydrophobic protease residues, while the <i>N</i>-(methoxycarbonyl) capping group allows for additional hydrogen bonding interactions with the protease. As already noted, LPV’s P2′ moiety does not form any hydrogen bonds, and DRV’s P2′ moiety makes one direct and one water-mediated interaction in the S2′ subsite (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Figure S1</a>). In most hybrid compounds, the carbonyl oxygen of the carbamate makes one direct hydrogen bond with the backbone NH of Asp29′ (Asp29), and the carbamate NH makes another direct hydrogen bond with the backbone carbonyl of Gly48′ (Gly48) in the flaps (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). There are also a number of coordinated water molecules that allow for additional water-mediated interactions. The carbonyl oxygen of the carbamate coordinates the same water-mediated interactions as the cyclic urea moiety of LPV, with the backbone carbonyl of Gly27′ (Gly27) and the side-chain carboxylate of Asp29′ (Asp29) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). This network of polar interactions is maintained by all amino acid moieties in both the S2′ and S2 subsites except the cycloleucine in compound <b>12b</b>.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Comparison of protease–inhibitor direct hydrogen bonds and water-mediated interactions in the S2/S2′ subsites for the cyclic urea moiety of LPV (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJB">6PJB</a>) (A), <i>N</i>-(methoxycarbonyl)-<i>tert</i>-leucine of ATV (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EKY">3EKY</a>) (B), and the <i>N</i>-(methoxycarbonyl)-capped amino acid moieties of hybrid compound <b>15a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJM">6PJM</a>) (C), <b>12a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJD">6PJD</a>) (D), <b>15b</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJN">6PJN</a>) (E), and <b>12b</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJE">6PJE</a>) (F). Polar interactions are shown as black dashed lines (as determined by PyMol) with waters shown as red spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The carbamate of compound <b>12b</b> is flipped relative to all other inhibitors losing the direct hydrogen bond to flap residue Gly48′ (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>F). This flip positions the carbonyl oxygen of the carbamate close to the carbonyl oxygen of Gly48′ (3.3 Å), creating a very unfavorable interaction and preventing compound <b>12b</b> from utilizing the same water-mediated interactions as LPV and most hybrid compounds. Instead, <b>12b</b> forms a hydrogen bond with a “backside” water in the S2′ subsite, creating a water-mediated interaction with the main-chain carbonyl and NH of Asp30′ (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>F). The marked changes in the hydrogen bonding interactions in the S2/S2′ subsite likely underlie the greatly reduced potency of <b>12b</b> compared to all other hybrid compounds. Unlike <b>12b</b>, the P2 carbamate group in compound <b>15b</b> maintained a similar conformation as in the <i>tert</i>-leucine analogue <b>15a</b> but with a slight change in the position of the carbonyl oxygen away from Asp29. This shift resulted in increased hydrogen bond distance between the P2 carbamate carbonyl and the main-chain NH of Asp29 compared to <b>15a</b> (3.3 versus 2.9 Å). However, the reduced potency of <b>15b</b> could not result from minor differences in hydrogen bonding interactions alone, highlighting the significance of nonpolar interactions for potency.</div><div class="NLM_p">The protease–inhibitor vdW interactions in the S2/S2′ subsite varied significantly depending on the size and shape of the hydrophobic amino acid moiety (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Figure S4</a>). Compounds <b>12a</b> and <b>15a</b> with the <i>tert</i>-leucine make highly distributed vdW contacts with a number of residues. In contrast, compared to <i>tert</i>-leucine, the cycloleucine side chain in <b>12b</b> predominantly interacts with Ile50 and I84V while losing significant contacts with residues 28–30 (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). The cycloleucine side chain of <b>15b</b> packs in the same hydrophobic area but makes less vdW contacts than <b>12b</b>. Compound <b>12c</b> with cyclopentyl glycine also has enhanced vdW contacts with Ile50 but without experiencing losses at other residues as observed for <b>12b</b>. In general, the cyclic hydrophobic amino acid derivatives make more localized vdW interactions in the S2/S2′ subsite while the acyclic moieties make more distributed contacts. Thus, in addition to altered polar interactions, the differences in vdW contacts due to the size and shape of the hydrophobic amino acid moiety are likely responsible for the varied inhibitor potencies.</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Comparison of binding conformation (A) and van der Waals packing with residues in the S2′ subsite for the most and least potent inhibitors <b>12a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJD">6PJD</a>) (B, green sticks) and <b>12b</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJE">6PJE</a>) (C, blue sticks). Compound <b>12a</b> makes more extensive contacts with the protease as estimated by relative vdW contact energy than <b>12b</b> (−91 versus −86 kcal/mol), with a major difference occurring at the S2′ subsite (−27 versus −23 kcal/mol). Protease residues are colored red to blue for highest to lowest vdW interactions with the inhibitor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Molecular Dynamics (MD) Simulations</h3><div class="NLM_p">Starting from the cocrystal structures, molecular dynamics (MD) simulations were utilized to interrogate the stability of protease–inhibitor interactions in the dynamic ensemble of the complexes. Three replicates of fully hydrated 100 ns MD simulations were performed on each protease–inhibitor complex. All simulations reached convergence (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Figure S5</a>).</div><div class="NLM_p">The protease active site and flap dynamics were analyzed by measuring C<sub>α</sub> distances between specific residues. The analysis focused on the most and least potent pairs of compounds containing bis-THF at either the P2 or P2′ position (<b>12a</b>/<b>15a</b> and <b>12b</b>/<b>15b</b>) compared to LPV. The cocrystal structures suggested that the flaps and the 80s loops surrounding the inhibitor may be perturbed when bound to different compounds. To monitor dynamics of the flaps, distance distributions between C<sub>α</sub> atoms of Ile50-Ile84′, Ile50′-Ile84, and Ile50-Ile50′ were calculated during the MD trajectories. Distances for Ile84-Ile84′ and Pro81-Pro81′ were used to probe the expansion or narrowing of the “lower” and “upper” side wall of the active site, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Figure S6</a>). In protease complexes with hybrid compounds, the active site remained largely unchanged in the lower part of the active site as indicated by a single, narrow distribution of Ile84-Ile84′ distance centered around 15.2 Å. However, compared to LPV, the shift in the Pro81-Pro81′ distance distribution with compounds <b>12a</b>, <b>15a</b>, <b>12b</b>, and <b>15b</b> indicated that the active site was narrower in the upper portion close to the flaps. In addition, the active site was asymmetrically longer (Ile50′-Ile84 or Ile50-Ile84′), sampling bimodal distance distributions, suggesting the sampling of a semiopen conformation of the flaps. This was supported by the Ile50-Ile50′ distance distribution, with one major peak around 5.3 Å (closed flaps) and another ∼10 Å corresponding to semiopen flaps with compounds <b>15a</b> and <b>15b</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Figure S6</a>). An increased distance between residues Ile50 and Ile50′ in a relatively short time (nanoseconds) during MD simulations has been previously suggested to indicate flap opening.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> Thus, perturbed flap dynamics and narrowing of the active site may indicate unfavorable binding of these compounds compared to LPV, which is supported by the <i>K</i>i data.</div><div class="NLM_p">Next, the stability of inhibitor binding was analyzed by calculating root-mean-square fluctuations (RMSF) of each atom (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). In all complexes, the bis-THF moiety often displayed the lowest RMSF among all moieties regardless of its position on the Phe-Phe core. Notably, even in the least active compounds with nanomolar potency (<b>12b</b> and <b>15b</b>), the bis-THF moiety showed fluctuations similar to those observed in <b>12a</b> and <b>15a</b>. Moreover, the phenylalanine side chain adjacent to the bis-THF moiety displayed lower RMSF than the distal phenylalanine side chain despite the pseudosymmetric nature of the Phe-Phe core. The P2/P2′ amino acid moieties showed varying RMSF profiles, independent of the size or shape of the hydrophobic side chain. The phenylalanine adjacent to the amino acid moiety consistently displayed greater RMSF, with the exception of <b>12e</b>. The varying fluctuations of the P1/P1′ moiety depending on the size and shape of the P2/P2′ moiety indicate interdependence between the S1/S1′ and S2/S2′ subsites. The P2/P2′ amino acid moiety also affected the fluctuations of the bis-THF moiety but to a much lesser extent. This result suggests that, in hybrid compounds, the bis-THF moiety and the adjacent P1/P1′ moiety remain relatively stable in the protease active site regardless of the position on the Phe-Phe core. In contrast, the fluctuations of the amino acid and the adjacent P1/P1′ moieties appear to destabilize overall inhibitor binding. Thus, due to interdependence between the S1/S1′ and S2/S2′ subsites, the optimization of the bis-THF containing hybrid PIs would require modification of the amino acid moiety together with the adjacent P1/P1′ group.</div><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Inhibitor root-mean-square fluctuation (RMSF) of each non-hydrogen atom of DRV, LPV, and hybrid compounds mapped onto their crystal structures. Warmer colors indicate larger fluctuations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The relationship between inhibitor and protease conformational dynamics was examined via simultaneous monitoring of inhibitor bond rotations and protease residue distances. The dihedral angles of all inhibitor rotatable bonds were calculated throughout the MD simulations. Given the bimodal sampling of the flap distances (Ile50-Ile50′, Ile50-Ile84′, and Ile50′-Ile84), we examined whether the semiopen conformation of flaps was associated with certain inhibitor bond rotations. In protease complexes with compounds <b>12a</b> and <b>15a</b>, the two most potent inhibitors, the separation of the flap tips (Ile50-Ile50′) was associated with the conformational sampling of the dihedral controlling the amino acid moiety (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). In this semiopen flap conformation, the Ile50-Ile50′ residues at the top of the flaps also lost intraprotease vdW contacts with residues 32, 47–49, and 84 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Figure S7</a>). Loss of intraprotease vdW interactions could result in a less stable protease–inhibitor complex, leading to an overall decrease in inhibitor binding affinity.</div><figure id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. The dihedral angle (ϕ<sub>5</sub>) responsible for conformational sampling of the <i>tert</i>-leucine moiety on <b>12a</b> (A) and <b>15a</b> (B) is associated with increased Ile50-Ile50′ distance sampling.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In addition, protein dynamics were compared by calculating the RMSF of C<sub>α</sub> atoms. Consistent with our observations from the distance distributions, there were significant increases in fluctuations at the flaps and 80s loop (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Figure S8</a>). In complex with LPV, the flaps and active site show little fluctuations comparable to those with DRV. However, compounds containing bis-THF at the P2′ position showed increased fluctuations at both flaps, with fluctuations asymmetrically greater at the B chain flap, contacting the amino acid moiety. The increased protein fluctuations could impact the stability of protease–inhibitor hydrogen bonding interactions.</div><div class="NLM_p">In the cocrystal structures, the hybrid compounds make a number of direct and water-mediated interactions with the protease active site, including the central hydroxyl with catalytic residues, coordinated water with Ile50/50′, the Phe-Phe core nitrogen atoms with Gly27, bis-THF with Asp29 and Asp30, carbamate with Asp29 and Gly48, and the coordinated S3′ water with Gly27′ and Asp29′. To assess if the increased protein and ligand fluctuations impact hydrogen bonding stability, the patterns and frequencies of these hydrogen bonds were monitored throughout the MD simulations (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Figure S9</a>). In MD simulations, LPV and the hybrid compounds did not maintain all hydrogen bonds observed in the cocrystal structures. LPV and all compounds with bis-THF at the P2 position (<b>4</b> and <b>12a</b>–<b>h</b>) maintained a hydrogen bond with Asp25′ (66–99% frequencies) but did not form a hydrogen bond with Asp25, whereas compounds <b>15a</b> and <b>15b</b> containing bis-THF at the P2 position formed hydrogen bonds with Asp25 and Asp25′ at roughly 100 and 50% frequencies, respectively. In contrast, the hydrogen bond between compound <b>15c</b> and Asp25 was unstable, while the hydrogen bond with Asp25′ was highly stable (92% frequency). Hybrid compounds maintained the network of water-mediated interactions with the main-chain NH of Ile50 and Ile50′ at varying stability. The interactions with Ile50 were observed to be less stable than the ones with Ile50′ (21–65 versus 49–78%). For most of the hybrid compounds, the hydrogen bonds between the Phe-Phe core NH groups and the main-chain carbonyls of Gly27/Gly27′ were either not observed or were very unstable.</div><figure id="fig10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0002.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Protease–inhibitor hydrogen bond frequencies from MD simulations mapped onto the cocrystal structures of compounds <b>12a</b> (A), <b>12b</b> (B), <b>15a</b> (C), and <b>15b</b> (D). The protease side chains and inhibitors are shown as sticks; chains A and B are colored cyan and magenta, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The bis-THF moiety in all hybrid compounds maintained two hydrogen bonds with the main-chain NH of Asp29/Asp29′ and Asp30/Asp30′ at high frequency regardless of the position on the Phe-Phe isostere (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Figure S9</a>). In contrast, the P2/P2′ amino acid moiety had varying effects on hydrogen bonding interactions in the S2/S2′ subsite. The carbamate nitrogen of hybrid compounds formed a hydrogen bond with the backbone carbonyl of Gly48′ at moderate to high frequency, but this interaction was not observed for the least potent compound <b>12b</b> and was less stable for <b>15b</b>. The carbonyl oxygen of the P2′ amino acid moiety in compounds <b>12a</b> and <b>12c</b> formed relatively stable water-mediated interactions with the side chain of Asp30′. In contrast, the carbonyl oxygen of the P2 moiety in compounds <b>15a</b> and <b>15c</b> formed three low-frequency (14–39%) water-mediated interactions, bridging the main-chain carbonyl of Gly27 and the side-chain carboxylate of Asp29. These data suggest that, for most hybrid compounds, the key hydrogen bonding interactions of the bis-THF moiety and the Phe-Phe isostere with protease remain relatively stable. In addition, the size and shape of the P2/P2′ amino acid moiety not only influence the interactions in the S2/S2′ subsites but can also propagate changes in the water-mediated interactions between the inhibitor and Ile50-Ile50′.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50354" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50354" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have explored hybrid HIV-1 PIs containing the bis-THF moiety of DRV on either side of the Phe-Phe dipeptide isostere of LPV to ascertain whether this moiety could maintain key interactions with the protease and improve potency against wild-type and primary drug-resistant variants of the enzyme. Most of the hybrid compounds retained picomolar biochemical potency irrespective of the position of bis-THF relative to the central hydroxyl group, likely due to the pseudo-C<sub>2</sub>-symmetric nature of the Phe-Phe isostere. However, in both compound series, the identity of the other P2/P2′ moiety significantly affected potency, favoring relatively flexible, hydrophobic moieties. The cocrystal structures of hybrid compounds revealed that the bis-THF moiety maintains the key hydrogen bonding interactions with the protease in the S2/S2′ subsite as observed in the DRV–protease complex regardless of its position on the Phe-Phe isostere. In contrast, the binding of the hydrophobic amino acid moiety in the other S2/S2′ subsite was greatly influenced by the size, shape, and flexibility of the hydrophobic group, which affected hydrogen bonding and vdW interactions with the protease. Moreover, this moiety appears to influence binding of the adjacent P1/P1′ group as well as the water-mediated interactions between the inhibitor and flap residues, indicating dynamic interdependence between the protease subsites. This is further supported by the relatively stable binding interactions of the bis-THF moiety and the adjacent P1/P1′ group. The dynamic interdependence between subsites in the protease active site can be exploited to optimize inhibitor potency against drug-resistant protease variants. The detailed structural characterization of hybrid HIV-1 PIs containing bis-THF in non-sulfonamide dipeptide isosteres offers opportunities for structure-guided optimization of these promising inhibitors.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16834" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16834" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> General</h3><div class="NLM_p last">All reactions were performed in oven-dried round-bottom flasks fitted with rubber septa under an argon atmosphere unless otherwise noted. All reagents and solvents, including anhydrous solvents, were purchased from commercial sources and used as received. Flash column chromatography was performed using an automated Teledyne ISCO CombiFlash Rf+ system equipped with a UV–vis detector using disposable Redisep Gold high-performance silica gel columns or was performed manually using silica gel (230–400 mesh, EMD Millipore). Thin-layer chromatography (TLC) was performed using silica gel (60 F<sub>254</sub>)-coated aluminum plates (EMD Millipore), and spots were visualized by exposure to ultraviolet light (UV), exposure to iodine adsorbed on silica gel, and/or staining with alcohol solutions of phosphomolybdic acid (PMA) and ninhydrin followed by brief heating. <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were acquired using Varian Mercury 400 MHz and Bruker Avance III HD 500 MHz NMR instruments. Chemical shifts are reported in ppm (δ scale) with the residual solvent signal used as a reference, and coupling constant (<i>J</i>) values are reported in hertz (Hz). Data are presented as follows: chemical shift, multiplicity (s = singlet, d = doublet, dd = doublet of doublet, dd = doublet of triplet, t = triplet, m = multiplet, br s = broad singlet), coupling constant in Hz, and integration. High-resolution mass spectra (HRMS) were recorded using a Thermo Scientific Orbitrap Velos Pro mass spectrometer coupled with a Thermo Scientific Accela 1250 UPLC and an autosampler using electrospray ionization (ESI) in the positive mode. The purity of final compounds was determined by analytical HPLC and was found to be ≥95% pure. HPLC was performed using an Agilent 1200 system equipped with a multiple-wavelength detector and a manual injector under the following conditions: column, Phenomenex Hypersil-BDS-5u-C18 (5 μm, 4.6 mm × 250 mm, 130 Å); solvent A, H<sub>2</sub>O containing 0.1% trifluoroacetic acid (TFA); solvent B, CH<sub>3</sub>CN containing 0.1% TFA; gradient, 20% B to 100% B over 15 min followed by 100% B over 5 min; injection volume, 20 μL; flow rate, 1 mL/min. The wavelengths of detection were 254 and 280 nm. Retention times and purity data for each target compound are provided in the <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> (3<i>R</i>,3a<i>S</i>,6a<i>R</i>)-Hexahydrofuro[2,3-<i>b</i>]furan-3-yl ((2<i>S</i>,4<i>S</i>,5<i>S</i>)-5-(Dibenzylamino)-4-hydroxy-1,6-diphenylhexan-2-yl)carbamate (<b>9</b>)</h3><div class="NLM_p last">A solution of the Phe-Phe core intermediate <b>7</b> (3.00 g, 6.46 mmol) in anhydrous CH<sub>3</sub>CN (50 mL) was cooled to 0 °C and treated with diisopropylethylamine (DIEA) (3.37 mL, 19.35 mmol) followed by bis-THF activated carbonate <b>8</b> (1.93 g, 7.10 mmol). The reaction mixture was warmed to room temperature and stirred for 24 h, concentrated under reduced pressure, and dried under high vacuum. The residue was purified by flash column chromatography (RediSep Gold, 80 g, gradient elution with 0–100% ethyl acetate/hexanes) to provide the intermediate <b>9</b> (4.01 g, 100%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.34–7.17 (m, 16 H), 7.10 (d, <i>J</i> = 7.5 Hz, 2 H), 7.01 (d, <i>J</i> = 7.0 Hz, 2 H), 5.69 (d, <i>J</i> = 5.0 Hz, 1 H), 5.29 (d, <i>J</i> = 6.0 Hz, 1 H), 5.12 (q, <i>J</i> = 6.5 Hz, 1 H), 4.52 (br s, 1 H), 4.02 (dd, <i>J</i> = 9.0, 7.0 Hz, 1 H), 3.95–3.73 (m, 6 H), 3.65–3.58 (m, 1 H), 3.38 (d, <i>J</i> = 13.0 Hz, 2 H), 3.10 (dd, <i>J</i> = 14.0, 5.5 Hz, 1 H), 3.04–2.97 (m, 1 H), 2.85–2.76 (m, 2 H), 2.66 (dd, <i>J</i> = 13.0, 6.5 Hz, 1 H), 2.54 (dd, <i>J</i> = 14.5, 7.0 Hz, 1 H), 1.87–1.75 (m, 2 H), 1.52 (dd, <i>J</i> = 14.5, 3.5 Hz, 1 H), 1.11 (dt, <i>J</i> = 14.5, 8.5 Hz, 1 H) ppm; <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 155.41, 140.15, 138.74, 138.18, 129.61, 129.15, 128.86, 128.69, 128.40, 127.53, 126.47, 126.40, 109.48, 73.00, 71.03, 69.77, 69.74, 64.31, 53.95, 52.28, 45.49, 41.35, 38.15, 32.15, 26.09 ppm; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>39</sub>H<sub>45</sub>N<sub>2</sub>O<sub>5</sub>, 621.3323; found 621.3306.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> (3<i>R</i>,3a<i>S</i>,6a<i>R</i>)-Hexahydrofuro[2,3-<i>b</i>]furan-3-yl ((2<i>S</i>,4<i>S</i>,5<i>S</i>)-4-Hydroxy-5-((<i>S</i>)-2-((methoxycarbonyl)amino)-3,3-dimethylbutanamido)-1,6-diphenylhexan-2-yl)carbamate (<b>12a</b>)</h3><div class="NLM_p last">A solution of compound <b>9</b> (0.50 g, 0.81 mmol) in anhydrous MeOH (10 mL) was treated with ammonium formate (0.30 g, 4.83 mmol) and 10% Pd/C (0.1 g), and the reaction mixture was stirred at 50 °C for 14 h. The reaction mixture was filtered through a pad of Celite, and the filtrate was concentrated under reduced pressure. The resulting deprotected amine <b>10</b> was dissolved in anhydrous CH<sub>2</sub>Cl<sub>2</sub> (10 mL), and the solution was cooled to 0 °C. <i>N</i>-(methoxycarbonyl)-<span class="smallcaps smallerCapital">l</span>-<i>tert</i>-leucine <b>11a</b> (0.18 g, 0.97 mmol), DIEA (0.280 mL, 1.61 mmol), and HATU (0.48 g, 1.26 mmol) were added. The reaction mixture was stirred at room temperature for 16 h, concentrated under reduced pressure, and dried under high vacuum. The residue was purified by flash column chromatography (RediSep Gold, 24 g, gradient elution with 0–15% methanol/dichloromethane) to provide the target compound <b>12a</b> (0.27 g, 55%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.25–7.13 (m, 8 H), 7.07 (d, <i>J</i> = 7.0 Hz, 2 H), 6.57 (d, <i>J</i> = 9.0 Hz, 1 H), 5.65 (d, <i>J</i> = 5.0 Hz, 1 H), 5.46 (d, <i>J</i> = 9.0 Hz, 1 H), 5.12 (d, <i>J</i> = 8.0 Hz, 1 H), 5.04 (q, <i>J</i> = 6.5 Hz, 1 H), 4.15 (q, <i>J</i> = 7.0 Hz, 1 H), 4.06–3.96 (m, 1 H), 3.93 (dd, <i>J</i> = 9.5, 6.5 Hz, 1 H), 3.87–3.63 (m, 4 H), 3.66 (s, 3 H, overlapping), 2.96–2.81 (m, 3 H), 2.80 (s, 2 H), 2.66 (dd, <i>J</i> = 14.0, 7.5 Hz, 1 H), 1.74–1.55 (m, 4 H), 0.93 (s, 9 H) ppm; <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 170.93, 157.18, 155.58, 137.95, 137.39, 129.39, 128.67, 128.60, 126.82, 126.60, 109.44, 73.47, 70.97, 69.74, 69.67, 63.44, 54.83, 52.59, 50.95, 45.51, 41.72, 40.19, 38.35, 34.21, 26.71, 26.02 ppm; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>46</sub>N<sub>3</sub>O<sub>8</sub>, 612.3280; found 612.3276; Anal. HPLC: <i>t</i><sub>R</sub> 10.86 min, purity 99%.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> (3<i>R</i>,3a<i>S</i>,6a<i>R</i>)-Hexahydrofuro[2,3-<i>b</i>]furan-3-yl ((2<i>S</i>,4<i>S</i>,5<i>S</i>)-4-Hydroxy-5-(1-((methoxycarbonyl)amino)cyclopentane-1-carboxamido)-1,6-diphenylhexan-2-yl)carbamate (<b>12b</b>)</h3><div class="NLM_p last">The same procedure was used as described above for compound <b>12a</b>. Compound <b>9</b> (1.00 g, 1.61 mmol) was treated with ammonium formate (0.61 g, 9.66 mmol) and 10% Pd/C (0.25 g) to give the corresponding deprotected amine <b>10</b>, which was coupled with <i>N</i>-(methoxycarbonyl)-cycloleucine <b>11b</b> (0.36 g, 1.93 mmol) using DIEA (1.12 mL, 6.44 mmol) and HATU (0.96 g, 2.51 mmol) to provide the target compound <b>12b</b> (0.66 g, 67%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.28–7.22 (m, 4 H), 7.21–7.16 (m, 4 H), 7.14 (d, <i>J</i> = 7.0 Hz, 2 H), 6.61 (d, <i>J</i> = 8.5 Hz, 1 H), 5.66 (d, <i>J</i> = 5.0 Hz, 1 H), 5.07 (app q, <i>J</i> = 6.5 Hz, 2 H), 4.96 (br s, 1 H), 4.07 (q, <i>J</i> = 7.5 Hz, 1 H), 4.02–3.93 (m, 2 H), 3.90–3.84 (m, 1 H), 3.76–3.65 (m, 3 H), 3.63 (s, 3 H), 3.35 (br s, 1 H), 2.99–2.93 (m, 1 H), 2.90–2.84 (m, 2 H), 2.83 (dd, <i>J</i> = 14.5, 7.0 Hz, 1 H, overlapping), 2.75 (dd, <i>J</i> = 12.5, 7.0 Hz, 1 H), 2.27–2.18 (m, 1 H), 2.00–1.91 (m, 1 H), 1.86–1.59 (m, 10 H) ppm; <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 174.05, 156.34, 155.58, 138.29, 137.85, 129.53, 129.34, 128.57, 126.64, 126.55, 109.45, 73.31, 70.91, 69.74, 67.49, 55.07, 52.53, 51.16, 45.50, 41.84, 39.19, 37.80, 36.60, 26.00, 24.11 ppm; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>44</sub>N<sub>3</sub>O<sub>8</sub>, 610.3123; found 610.3117; Anal. HPLC: <i>t</i><sub>R</sub> 9.74 min, purity 96%.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> (3<i>R</i>,3a<i>S</i>,6a<i>R</i>)-Hexahydrofuro[2,3-<i>b</i>]furan-3-yl ((2<i>S</i>,4<i>S</i>,5<i>S</i>)-5-((<i>S</i>)-2-Cyclopentyl-2-((methoxycarbonyl)amino)acetamido)-4-hydroxy-1,6-diphenylhexan-2-yl)carbamate (<b>12c</b>)</h3><div class="NLM_p last">The same procedure was used as described above for compound <b>12a</b>. Compound <b>9</b> (0.50 g, 0.81 mmol) was treated with ammonium formate (0.30 g, 4.83 mmol) and 10% Pd/C (0.13 g) to give the corresponding deprotected amine <b>10</b>, which was coupled with <i>N</i>-(methoxycarbonyl)-<span class="smallcaps smallerCapital">l</span>-cyclopentylglycine <b>11c</b> (0.19 g, 0.97 mmol) using DIEA (0.566 mL, 3.22 mmol) and HATU (0.48 g, 1.26 mmol) to provide the target compound <b>12c</b> (0.32 g, 64%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.29–7.22 (m, 4 H), 7.21–7.16 (m, 4 H), 7.11 (d, <i>J</i> = 7.5 Hz, 2 H), 6.35 (d, <i>J</i> = 6.5 Hz, 1 H), 5.67 (d, <i>J</i> = 5.0 Hz, 1 H), 5.12 (d, <i>J</i> = 7.5 Hz, 1 H), 5.07 (q, <i>J</i> = 6.5 Hz, 1 H), 4.94 (d, <i>J</i> = 8.5 Hz, 1 H), 4.11 (app q, <i>J</i> = 8.5 Hz, 1 H), 4.03–3.93 (m, 2 H), 3.92–3.83 (m, 2 H), 3.77–3.66 (m, 3 H), 3.67 (s, 3 H, overlapping), 3.30 (br s, 1 H), 2.99–2.93 (m, 1 H), 2.91–2.85 (m, 2 H), 2.82 (dd, <i>J</i> = 14.5, 6.5 Hz, 1 H), 2.70 (dd, <i>J</i> = 13.5, 7.5 Hz, 1 H), 2.22–2.11 (m, 1 H), 1.76–1.46 (m, 10 H), 1.27–1.14 (m, 2 H) ppm; <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 172.09, 157.16, 155.62, 138.05, 137.55, 129.43, 129.36, 128.66, 128.62, 126.80, 126.62, 109.46, 73.47, 70.93, 70.15, 69.73, 59.47, 54.91, 52.63, 51.02, 45.53, 41.94, 41.81, 39.98, 38.17, 29.59, 28.73, 26.00, 25.51, 25.28 ppm; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>34</sub>H<sub>46</sub>N<sub>3</sub>O<sub>8</sub>, 624.3280; found 624.3273; Anal. HPLC: <i>t</i><sub>R</sub> 10.98 min, purity 96%.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> (3<i>R</i>,3a<i>S</i>,6a<i>R</i>)-Hexahydrofuro[2,3-<i>b</i>]furan-3-yl ((2<i>S</i>,4<i>S</i>,5<i>S</i>)-4-Hydroxy-5-((<i>S</i>)-2-((methoxycarbonyl)amino)propanamido)-1,6-diphenylhexan-2-yl)carbamate (<b>12d</b>)</h3><div class="NLM_p last">The same procedure was used as described above for compound <b>12a</b>. Compound <b>9</b> (0.45 g, 0.73 mmol) was treated with ammonium formate (0.28 g, 4.36 mmol) and 10% Pd/C (0.11 g) to give the corresponding deprotected amine <b>10</b>, which was coupled with <i>N</i>-(methoxycarbonyl)-<span class="smallcaps smallerCapital">l</span>-alanine <b>11d</b> (0.14 g, 0.95 mmol) using DIEA (0.377 mL, 2.18 mmol) and HATU (0.47 g, 1.23 mmol) to provide the target compound <b>12d</b> (0.20 g, 47%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.28–7.22 (m, 4 H), 7.21–7.16 (m, 4 H), 7.11 (d, <i>J</i> = 7.5 Hz, 2 H), 6.39 (d, <i>J</i> = 9.0 Hz, 1 H), 5.66 (d, <i>J</i> = 5.0 Hz, 1 H), 5.17–5.09 (m, 1 H), 5.06 (q, <i>J</i> = 6.0 Hz, 1 H), 4.99 (d, <i>J</i> = 8.0 Hz, 1 H), 4.16–4.03 (m, 2 H), 3.99–3.91 (m, 2 H), 3.85 (td, <i>J</i> = 8.0, 2.0 Hz, 1 H), 3.77–3.65 (m, 3 H), 3.67 (s, 3 H, overlapping), 2.97–2.91 (m, 1 H), 2.88 (d, <i>J</i> = 7.5 Hz, 2 H), 2.81 (dd, <i>J</i> = 13.5, 5.5 Hz, 1 H), 2.71 (dd, <i>J</i> = 14.0, 8.0 Hz, 1 H), 1.75–1.58 (m, 4 H), 1.25 (d, <i>J</i> = 7.0 Hz, 3 H) ppm; <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 172.73, 156.74, 155.66, 138.05, 137.61, 129.44, 129.36, 128.65, 126.79, 126.66, 109.45, 73.48, 70.96, 70.03, 69.73, 54.92, 52.61, 51.01, 45.54, 41.82, 39.87, 37.99, 26.00, 18.76 ppm; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>40</sub>N<sub>3</sub>O<sub>8</sub>, 570.2810; found 570.2811; Anal. HPLC: <i>t</i><sub>R</sub> 9.14 min, purity 99%.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> (3<i>R</i>,3a<i>S</i>,6a<i>R</i>)-Hexahydrofuro[2,3-<i>b</i>]furan-3-yl ((2<i>S</i>,4<i>S</i>,5<i>S</i>)-4-Hydroxy-5-((<i>S</i>)-2-((methoxycarbonyl)amino)-3-methylbutanamido)-1,6-diphenylhexan-2-yl)carbamate (<b>12e</b>)</h3><div class="NLM_p last">The same procedure was used as described above for compound <b>12a</b>. Compound <b>9</b> (0.44 g, 0.70 mmol) was treated with ammonium formate (0.27 g, 4.22 mmol) and 10% Pd/C (0.11 g) to give the corresponding deprotected amine <b>10</b>, which was coupled with <i>N</i>-(methoxycarbonyl)-<span class="smallcaps smallerCapital">l</span>-valine <b>11e</b> (0.15 g, 0.84 mmol) using DIEA (0.364 mL, 2.11 mmol) and HATU (0.42 g, 1.10 mmol) to provide the target compound <b>12e</b> (0.29 g, 70%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.29–7.23 (m, 4 H), 7.22–7.17 (m, 4 H), 7.10 (d, <i>J</i> = 7.5 Hz, 2 H), 6.30 (d, <i>J</i> = 9.0 Hz, 1 H), 5.67 (d, <i>J</i> = 5.0 Hz, 1 H), 5.12–5.04 (m, 2 H), 4.93 (d, <i>J</i> = 8.5 Hz, 1 H), 4.12 (q, <i>J</i> = 7.5 Hz, 1 H), 4.01–3.93 (m, 2 H), 3.92–3.83 (m, 2 H), 3.75–3.67 (m, 3 H), 3.68 (s, 3 H, overlapping), 3.26 (br s, 1 H), 2.99–2.93 (m, 1 H), 2.91–2.84 (m, 2 H), 2.81 (dd, <i>J</i> = 14.5, 6.5 Hz, 1 H), 2.70 (dd, <i>J</i> = 13.5, 7.5 Hz, 1 H), 2.15–2.05 (m, 1 H), 1.77–1.57 (m, 4 H), 0.90 (d, <i>J</i> = 7.0 Hz, 3 H), 0.78 (d, <i>J</i> = 6.5 Hz, 3 H) ppm; <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 171.58, 157.28, 155.67, 138.00, 137.49, 129.42, 129.35, 128.67, 126.82, 126.66, 109.45, 73.50, 70.93, 70.12, 69.72, 61.01, 54.88, 52.67, 51.09, 45.52, 41.86, 40.04, 38.26, 30.50, 26.01, 19.54, 17.54 ppm; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>32</sub>H<sub>44</sub>N<sub>3</sub>O<sub>8</sub>, 598.3123; found 598.3122; Anal. HPLC: <i>t</i><sub>R</sub> 10.23 min, purity 99%.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> (3<i>R</i>,3a<i>S</i>,6a<i>R</i>)-Hexahydrofuro[2,3-<i>b</i>]furan-3-yl ((2<i>S</i>,4<i>S</i>,5<i>S</i>)-5-((<i>S</i>)-2-Cyclopropyl-2-((methoxycarbonyl)amino)acetamido)-4-hydroxy-1,6-diphenylhexan-2-yl)carbamate (<b>12f</b>)</h3><div class="NLM_p last">The same procedure was used as described above for compound <b>12a</b>. Compound <b>9</b> (0.46 g, 0.75 mmol) was treated with ammonium formate (0.28 g, 4.49 mmol) and 10% Pd/C (0.12 g) to give the corresponding deprotected amine <b>10</b>, which was coupled with <i>N</i>-(methoxycarbonyl)-<span class="smallcaps smallerCapital">l</span>-cyclopropylglycine <b>11f</b> (0.18 g, 1.05 mmol) using DIEA (0.390 mL, 2.25 mmol) and HATU (0.42 g, 1.05 mmol) to provide the target compound <b>12f</b> (0.27 g, 61%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.29–7.22 (m, 4 H), 7.21–7.15 (m, 4 H, overlapping), 7.10 (d, <i>J</i> = 7.5 Hz, 2 H), 6.46 (d, <i>J</i> = 7.5 Hz, 1 H), 5.66 (d, <i>J</i> = 5.0 Hz, 1 H), 5.49 (d, <i>J</i> = 6.5 Hz, 1 H), 5.10–5.02 (m, 2 H), 4.10 (app q, <i>J</i> = 8.0 Hz, 1 H), 4.02–3.91 (m, 2 H), 3.89–3.82 (m, 1 H), 3.78–3.64 (m, 3 H), 3.65 (s, 3 H, overlapping), 3.54–3.47 (br s, 1 H), 3.46–3.39 (m, 1 H), 2.98–2.91 (m, 1 H), 2.89 (d, <i>J</i> = 7.5 Hz, 2 H), 2.81 (dd, <i>J</i> = 14.0, 6.0 Hz, 1 H), 2.70 (dd, <i>J</i> = 13.5, 7.5 Hz, 1 H), 1.75–1.57 (m, 4 H), 0.96 (br s, 1 H), 0.55–0.44 (m, 2 H), 0.43–0.33 (m, 2 H) ppm; <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 171.65, 156.96, 155.58, 138.08, 137.65, 129.44, 129.36, 128.62, 128.59, 126.72, 126.62, 109.43, 73.41, 70.94, 69.70, 59.32, 54.96, 52.56, 50.82, 45.54, 41.63, 39.78, 37.95, 25.98, 14.59, 3.68, 3.40 ppm; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>32</sub>H<sub>42</sub>N<sub>3</sub>O<sub>8</sub>, 596.2967; found 596.2957; Anal. HPLC: <i>t</i><sub>R</sub> 10.54 min, purity 99%.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> (3<i>R</i>,3a<i>S</i>,6a<i>R</i>)-Hexahydrofuro[2,3-<i>b</i>]furan-3-yl ((2<i>S</i>,4<i>S</i>,5<i>S</i>)-4-Hydroxy-5-((2<i>S</i>,3<i>S</i>)-2-((methoxycarbonyl)amino)-3-methylpentanamido)-1,6-diphenylhexan-2-yl)carbamate (<b>12g</b>)</h3><div class="NLM_p last">The same procedure was used as described above for compound <b>12a</b>. Compound <b>9</b> (0.44 g, 0.70 mmol) was treated with ammonium formate (0.27 g, 4.22 mmol) and 10% Pd/C (0.11 g) to give the corresponding deprotected amine <b>10</b>, which was coupled with <i>N</i>-(methoxycarbonyl)-<span class="smallcaps smallerCapital">l</span>-isoleucine <b>11g</b> (0.15 g, 0.84 mmol) using DIEA (0.364 mL, 2.11 mmol) and HATU (0.42 g, 1.10 mmol) to provide the target compound <b>12g</b> (0.29 g, 40%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.29–7.23 (m, 4 H), 7.22–7.17 (m, 4 H), 7.11 (d, <i>J</i> = 7.5 Hz, 2 H), 6.26 (d, <i>J</i> = 9.0 Hz, 1 H), 5.67 (d, <i>J</i> = 5.5 Hz, 1 H), 5.07 (q, <i>J</i> = 6.5 Hz, 1 H), 5.02 (d, <i>J</i> = 8.0 Hz, 1 H), 4.89 (d, <i>J</i> = 8.0 Hz, 1 H), 4.13 (q, <i>J</i> = 7.0 Hz, 1 H), 4.00–3.90 (m, 3 H), 3.89–3.84 (m, 1 H), 3.75–3.67 (m, 3 H), 3.68 (s, 3 H, overlapping), 3.16 (br s, 1 H), 2.99–2.93 (m, 1 H), 2.91–2.84 (m, 2 H), 2.81 (dd, <i>J</i> = 15.5, 7.5 Hz, 1 H), 2.71 (dd, <i>J</i> = 14.0, 8.0 Hz, 1 H), 1.90–1.80 (m, 1 H), 1.77–1.56 (m, 4 H), 1.33–1.22 (m, 1 H), 1.03–0.93 (m, 1 H), 0.87 (d, <i>J</i> = 7.0 Hz, 3 H), 0.84 (t, <i>J</i> = 7.5 Hz, 3 H) ppm; <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 171.58, 157.21, 155.66, 137.99, 137.51, 129.42, 129.36, 128.68, 128.64, 126.82, 126.64, 109.45, 73.49, 70.91, 70.24, 69.73, 60.40, 56.02, 54.80, 52.67, 51.10, 45.52, 43.96, 41.90, 40.06, 38.25, 36.82, 26.00, 24.48, 18.78, 17.40, 15.85, 12.65, 11.57 ppm; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>46</sub>N<sub>3</sub>O<sub>8</sub>, 612.3280; found 612.3275; Anal. HPLC: <i>t</i><sub>R</sub> 10.82 min, purity 99%.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> (3<i>R</i>,3a<i>S</i>,6a<i>R</i>)-Hexahydrofuro[2,3-<i>b</i>]furan-3-yl ((2<i>S</i>,4<i>S</i>,5<i>S</i>)-4-Hydroxy-5-((2<i>S</i>,3<i>R</i>)-2-((methoxycarbonyl)amino)-3-methylpentanamido)-1,6-diphenylhexan-2-yl)carbamate (<b>12h</b>)</h3><div class="NLM_p last">The same procedure was used as described above for compound <b>12a</b>. Compound <b>9</b> (0.53 g, 0.86 mmol) was treated with ammonium formate (0.33 g, 5.17 mmol) and 10% Pd/C (0.13 g) to give the corresponding deprotected amine <b>10</b>, which was coupled with <i>N</i>-(methoxycarbonyl)-<span class="smallcaps smallerCapital">l</span>-alloisoleucine <b>11h</b> (0.15 g, 0.84 mmol) using DIEA (0.450 mL, 2.59 mmol) and HATU (0.59 g, 1.55 mmol) to provide the target compound <b>12h</b> (0.30 g, 58%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.29–7.23 (m, 4 H), 7.22–7.17 (m, 4 H), 7.11 (d, <i>J</i> = 7.0 Hz, 2 H), 6.28 (d, <i>J</i> = 9.0 Hz, 1 H), 5.67 (d, <i>J</i> = 5.5 Hz, 1 H), 5.07 (q, <i>J</i> = 6.5 Hz, 1 H), 4.96 (d, <i>J</i> = 7.5 Hz, 1 H), 4.93 (d, <i>J</i> = 8.0 Hz, 1 H), 4.16–4.09 (m, 1 H), 4.07 (dd, <i>J</i> = 7.0, 4.5 Hz, 1 H), 3.97 (dd, <i>J</i> = 9.5, 6.5 Hz, 2 H), 3.90–3.84 (m, 1 H), 3.75–3.67 (m, 3 H), 3.69 (s, 3 H, overlapping), 2.99–2.93 (m, 1 H), 2.90–2.84 (m, 2 H), 2.81 (dd, <i>J</i> = 15.5, 7.5 Hz, 1 H), 2.72 (dd, <i>J</i> = 14.0, 8.0 Hz, 1 H), 1.96–1.88 (m, 1 H), 1.76–1.57 (m, 4 H), 1.40–1.30 (m, 1 H), 1.21–1.11 (m, 1 H), 0.90 (t, <i>J</i> = 7.0 Hz, 3 H), 0.67 (d, <i>J</i> = 6.5 Hz, 3 H) ppm; <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 171.84, 157.29, 155.66, 138.02, 137.54, 129.42, 129.34, 128.67, 126.81, 126.67, 109.45, 73.48, 70.89, 70.36, 69.73, 59.12, 54.88, 52.71, 51.16, 45.52, 41.96, 39.95, 38.22, 36.82, 26.52, 26.00, 14.17, 11.80 ppm; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>46</sub>N<sub>3</sub>O<sub>8</sub>, 612.3280; found 612.3277; Anal. HPLC: <i>t</i><sub>R</sub> 10.87 min, purity 95%.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Methyl ((<i>S</i>)-1-(((2<i>S</i>,4<i>S</i>,5<i>S</i>)-5-(Dibenzylamino)-4-hydroxy-1,6-diphenylhexan-2-yl)amino)-3,3-dimethyl-1-oxobutan-2-yl)carbamate (<b>13a</b>)</h3><div class="NLM_p last">A solution of the Phe-Phe core intermediate <b>7</b> (1.00 g, 2.16 mmol) in DMF (15 mL) was cooled to 0 °C and treated with <i>N</i>-(methoxycarbonyl)-<span class="smallcaps smallerCapital">l</span>-<i>tert</i>-leucine <b>11a</b> (0.48 g, 2.58 mmol), DIEA (1.1 mL, 6.30 mmol), and HATU (1.28 g, 3.36 mmol). The resulting reaction mixture was stirred at room temperature for 4 h, concentrated under reduced pressure, and dried under high vacuum. The residue was purified by flash column chromatography (RediSep Gold, 24 g, gradient elution with 0–10% methanol/dichloromethane) to give the intermediate <b>13a</b> (1.20 g, 87%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.34–7.29 (m, 4 H), 7.28–7.23 (m, 4 H), 7.22–7.14 (m, 8 H), 7.06 (d, <i>J</i> = 7.5 Hz, 2 H), 7.00 (d, <i>J</i> = 6.5 Hz, 2 H), 6.55 (d, <i>J</i> = 6.0 Hz, 1 H), 5.42 (d, <i>J</i> = 9.0 Hz, 1 H), 4.65 (br s, 1 H), 4.06–3.97 (m, 1 H), 3.91–3.79 (m, 3 H), 3.66 (s, 3 H), 3.53 (t, <i>J</i> = 8.5 Hz, 1 H), 3.36 (d, <i>J</i> = 13.5 Hz, 2 H), 3.06 (dd, <i>J</i> = 14.5, 5.5 Hz, 1 H), 2.92 (dd, <i>J</i> = 13.5, 5.0 Hz, 1 H), 2.76 (q, <i>J</i> = 7.0 Hz, 1 H), 2.59–2.46 (m, 2 H), 1.50 (dd, <i>J</i> = 14.5, 5.0 Hz, 1 H), 1.17–1.08 (m, 1 H), 0.99 (s, 9 H) ppm; <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 170.57, 157.02, 140.03, 138.77, 138.10, 129.54, 129.15, 129.06, 128.80, 128.64, 128.36, 127.49, 126.40, 126.29, 69.83, 64.22, 63.31, 53.90, 52.34, 51.54, 41.12, 37.42, 34.78, 32.01, 26.60 ppm; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>40</sub>H<sub>50</sub>N<sub>3</sub>O<sub>4</sub>, 636.3796; found 636.3793.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Methyl (1-(((2<i>S</i>,4<i>S</i>,5<i>S</i>)-5-(Dibenzylamino)-4-hydroxy-1,6-diphenylhexan-2-yl)carbamoyl)cyclopentyl)carbamate (<b>13b</b>)</h3><div class="NLM_p last">The same procedure was used as described above for <b>13a</b>. The Phe-Phe core intermediate <b>7</b> (1.00 g, 2.16 mmol) was coupled with <i>N</i>-(methoxycarbonyl)-cycloleucine <b>11b</b> (0.47 g, 2.50 mmol) using DIEA (1.1 mL, 6.30 mmol) and HATU (1.21 g, 3.18 mmol) to provide <b>13b</b> (1.30 g, 95%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.33–7.27 (m, 4 H), 7.26–7.15 (m, 12 H), 7.11 (d, <i>J</i> = 7.5 Hz, 2 H), 7.04 (d, <i>J</i> = 7.0 Hz, 2 H), 6.80 (br s, 1 H), 5.00 (br s, 1 H), 4.32 (br s, 1 H), 4.04 (m, 1 H), 3.93 (d, <i>J</i> = 13.0 Hz, 2 H), 3.55 (s, 4 H), 3.39 (d, <i>J</i> = 13.0 Hz, 2 H), 3.05 (dd, <i>J</i> = 13.5, 5.0 Hz, 1 H), 2.81 (dd, <i>J</i> = 13.5, 5.0 Hz 1 H), 2.74 (app q, <i>J</i> = 7.5 Hz, 1 H), 2.70–2.59 (m, 2 H), 2.28–2.17 (m, 1 H), 2.12–2.04 (m, 1 H), 1.93–1.78 (m, 1 H), 1.77–1.65 (m, 4 H), 1.51 (dd, <i>J</i> = 14.5, 3.5 Hz, 1 H), 1.35–1.24 (m, 1 H) ppm; <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 173.64, 156.09, 140.40, 139.32, 138.38, 129.69, 129.27, 129.10, 128.74, 128.57, 128.30, 127.33, 126.28, 70.03, 67.31, 64.31, 54.27, 52.20, 50.93, 41.33, 38.17, 37.23, 36.65, 31.65, 24.30, 24.26 ppm; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>40</sub>H<sub>48</sub>N<sub>3</sub>O<sub>4</sub>, 634.3640; found 634.3615.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Methyl ((<i>S</i>)-1-Cyclopentyl-2-(((2<i>S</i>,4<i>S</i>,5<i>S</i>)-5-(dibenzylamino)-4-hydroxy-1,6-diphenylhexan-2-yl)amino)-2-oxoethyl)carbamate (<b>13c</b>)</h3><div class="NLM_p last">The same procedure was used as described above for <b>13a</b>. The Phe-Phe core intermediate <b>7</b> (1.04 g, 2.23 mmol) was coupled with <i>N</i>-(methoxycarbonyl)-cyclopentylglycine <b>11c</b> (0.45 g, 2.23 mmol) using DIEA (1.17 mL, 6.69 mmol) and HATU (1.02 g, 2.68 mmol) to give <b>13c</b> (1.30 g, 90%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.34–7.28 (m, 4 H), 7.27–7.22 (m, 4 H), 7.21–7.14 (m, 8 H), 7.06 (d, <i>J</i> = 7.0 Hz, 2 H), 7.00 (d, <i>J</i> = 6.5 Hz, 2 H), 6.57 (d, <i>J</i> = 5.5 Hz, 1 H), 5.22 (d, <i>J</i> = 8.0 Hz, 1 H), 4.66 (br s, 1 H), 4.06–3.98 (m, 1 H), 3.97–3.90 (m, 1 H), 3.86 (d, <i>J</i> = 13.0 Hz, 2 H), 3.64 (s, 3 H), 3.53 (t, <i>J</i> = 8.5 Hz, 1 H), 3.36 (d, <i>J</i> = 13.0 Hz, 2 H), 3.05 (dd, <i>J</i> = 14.5, 5.5 Hz, 1 H), 2.88 (dd, <i>J</i> = 13.5, 5.0 Hz, 1 H), 2.76 (q, <i>J</i> = 6.5 Hz, 1 H), 2.59–2.47 (m, 2 H), 2.23 (app q, <i>J</i> = 8.0 Hz, 1 H), 1.74–1.54 (m, 4 H), 1.53–1.43 (m, 3 H), 1.40–1.22 (m, 2 H), 1.15–1.06 (m, 1 H) ppm; <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 171.64, 157.03, 140.10, 138.79, 138.14, 129.56, 129.16, 129.11, 128.83, 128.66, 128.38, 127.51, 126.42, 126.33, 70.00, 64.20, 58.96, 53.88, 52.38, 51.53, 42.75, 41.40, 37.59, 32.05, 29.29, 28.55, 25.46, 25.20 ppm; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>41</sub>H<sub>50</sub>N<sub>3</sub>O<sub>4</sub>, 648.3796; found 648.3795.</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> (3<i>R</i>,3a<i>S</i>,6a<i>R</i>)-Hexahydrofuro[2,3-<i>b</i>]furan-3-yl ((2<i>S</i>,3<i>S</i>,5<i>S</i>)-3-Hydroxy-5-((<i>S</i>)-2-((methoxycarbonyl)amino)-3,3-dimethylbutanamido)-1,6-diphenylhexan-2-yl)carbamate (<b>15a</b>)</h3><div class="NLM_p last">A solution of intermediate <b>13a</b> (1.20 g, 1.89 mmol) in anhydrous MeOH (15 mL) was treated with ammonium formate (0.71 g, 11.32 mmol) and 10% Pd/C (0.3 g). The resulting reaction mixture was stirred at 50 °C for 15 h, cooled to room temperature, and filtered through a pad of Celite, and the filtrate was concentrated under reduced pressure. The resulting deprotected amine was dissolved in acetonitrile (10 mL), and the solution was cooled to 0 °C. DIEA (0.98 mL, 5.66 mmol) was added followed by bis-THF activated carbonate <b>8</b> (0.56 g, 2.08 mmol). The reaction mixture was warmed up to room temperature and stirred for 24 h. The solvents were removed under reduced pressure, and the residue was purified by automated flash chromatography using a silica gel column (RediSep Gold, 40 g, gradient elution with 0–15% methanol/dichloromethane) to provide the target compound <b>15a</b> (0.36 g, 31%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.29–7.22 (m, 4 H), 7.21–7.16 (m, 4 H), 7.10 (d, <i>J</i> = 7.0 Hz, 2 H), 6.07 (d, <i>J</i> = 6.0 Hz, 1 H), 5.67 (d, <i>J</i> = 5.0 Hz, 1 H), 5.36–5.22 (m, 1 H), 5.10 (d, <i>J</i> = 9.5 Hz, 1 H), 5.05 (q, <i>J</i> = 6.5 Hz, 1 H), 4.22–4.13 (m, 1 H), 3.98 (dd, <i>J</i> = 9.5, 6.5 Hz, 1 H), 3.87 (td, <i>J</i> = 8.5, 3.0 Hz, 1 H), 3.82–3.58 (m, 6 H), 3.67 (s, 3 H, overlapping), 2.98–2.90 (m, 1 H), 2.88–2.73 (m, 4 H), 1.73–1.57 (m, 4 H), 0.91 (s, 9 H) ppm; <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 171.09, 157.19, 155.83, 138.13, 137.12, 129.41, 129.32, 128.73, 128.64, 126.91, 126.61, 109.47, 73.39, 70.98, 70.35, 69.78, 63.35, 56.60, 52.65, 49.38, 45.49, 41.43, 40.37, 38.87, 34.35, 26.65, 25.98 ppm; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>46</sub>N<sub>3</sub>O<sub>8</sub>, 612.3280; found 612.3278; Anal. HPLC: <i>t</i><sub>R</sub> 10.70 min, purity 97%.</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> (3<i>R</i>,3a<i>S</i>,6a<i>R</i>)-Hexahydrofuro[2,3-<i>b</i>]furan-3-yl ((2<i>S</i>,3<i>S</i>,5<i>S</i>)-3-Hydroxy-5-(1-((methoxycarbonyl)amino)cyclopentane-1-carboxamido)-1,6-diphenylhexan-2-yl)carbamate (<b>15b</b>)</h3><div class="NLM_p last">The same procedure was used as described above for <b>15a</b>. Compound <b>13b</b> (0.70 g, 1.10 mmol) was treated with ammonium formate (0.42 g, 6.63 mmol) and 10% Pd/C (0.17 g) to give the corresponding deprotected amine, which was subsequently treated with DIEA (0.380 mL, 2.20 mmol) and bis-THF activated carbonate <b>8</b> (0.33 g, 1.21 mmol) to provide the target compound <b>15b</b> (0.23 g, 35%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.29–7.22 (m, 4 H), 7.21–7.16 (m, 4 H), 7.11 (d, <i>J</i> = 7.0 Hz, 2 H), 6.49 (d, <i>J</i> = 7.0 Hz, 1 H), 5.67 (d, <i>J</i> = 5.5 Hz, 1 H), 5.14 (d, <i>J</i> = 9.5 Hz, 1 H), 5.07 (q, <i>J</i> = 6.5 Hz, 1 H), 4.97 (s, 1 H), 4.23 (br s, 1 H), 3.99 (dd, <i>J</i> = 9.5, 6.5 Hz, 1 H), 3.94 (br s, 1 H), 3.88 (td, <i>J</i> = 8.0, 2.0 Hz, 1 H), 3.85–3.69 (m, 4 H), 3.65 (s, 3 H), 2.98–2.91 (m, 1 H), 2.89 (dd, <i>J</i> = 14.0, 7.5 Hz, 1 H, overlapping), 2.83 (dd, <i>J</i> = 15.5, 8.0 Hz, 2 H), 2.71 (dd, <i>J</i> = 13.5, 7.5 Hz, 1 H), 2.19–2.11 (m, 1 H), 1.94–1.59 (m, 11 H) ppm; <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 173.71, 156.67, 155.77, 138.34, 137.62, 129.52, 129.40, 128.56, 126.73, 126.49, 109.51, 73.29, 71.11, 70.40, 69.81, 67.39, 57.24, 52.62, 49.96, 45.50, 41.98, 40.64, 39.01, 37.14, 36.65, 26.03, 24.01, 23.81 ppm; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>44</sub>N<sub>3</sub>O<sub>8</sub>, 610.3123; found 610.3118; Anal. HPLC: <i>t</i><sub>R</sub> 7.73 min, purity 95%.</div></div><div id="sec5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> (3<i>R</i>,3a<i>S</i>,6a<i>R</i>)-Hexahydrofuro[2,3-<i>b</i>]furan-3-yl ((2<i>S</i>,3<i>S</i>,5<i>S</i>)-5-((<i>S</i>)-2-Cyclopentyl-2-((methoxycarbonyl)amino)acetamido)-3-hydroxy-1,6-diphenylhexan-2-yl)carbamate (<b>15c</b>)</h3><div class="NLM_p last">The same procedure was used as described above for <b>15a</b>. Compound <b>13c</b> (0.70 g, 1.08 mmol) was treated with ammonium formate (0.41 g, 6.48 mmol) and 10% Pd/C (0.17 g) to give the corresponding deprotected amine, which was subsequently treated with DIEA (0.570 mL, 3.24 mmol) and bis-THF activated carbonate <b>8</b> (0.32 g, 1.19 mmol) to provide the target compound <b>15c</b> (0.26 g, 39%) as a white solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.29–7.22 (m, 4 H), 7.21–7.16 (m, 4 H), 7.11 (d, <i>J</i> = 7.5 Hz, 2 H), 6.29 (br s, 1 H), 5.67 (d, <i>J</i> = 5.0 Hz, 1 H), 5.19–5.07 (m, 2 H), 5.05 (q, <i>J</i> = 6.0 Hz, 1 H, overlapping), 4.24–4.15 (m, 1 H), 3.98 (dd, <i>J</i> = 9.5, 6.0 Hz, 1 H), 3.87 (td, <i>J</i> = 8.0, 2.0 Hz, 1 H), 3.85–3.78 (m, 2 H, overlapping), 3.77–3.63 (m, 3 H), 3.67 (s, 3 H, overlapping), 2.99–2.91 (m, 1 H), 2.90–2.71 (m, 4 H), 2.22–2.09 (m, 1 H), 1.83–1.45 (m, 10 H), 1.22–1.07 (m, 2 H) ppm; <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 172.11, 157.29, 155.83, 138.24, 137.34, 129.45, 129.34, 128.63, 128.60, 126.80, 126.56, 109.48, 73.36, 71.00, 70.33, 69.79, 59.46, 56.68, 52.71, 49.33, 45.54, 41.70, 41.46, 40.51, 38.92, 29.48, 28.71, 25.97, 25.47, 25.22 ppm; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>34</sub>H<sub>46</sub>N<sub>3</sub>O<sub>8</sub>, 624.3279; found 624.3282; Anal. HPLC: <i>t</i><sub>R</sub> 10.83 min, purity 99%.</div></div><div id="sec5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Protease Gene Construction</h3><div class="NLM_p last">Protease gene construction was carried out as previously described.<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36,37)</a> The NL4–3 strain has four naturally occurring polymorphisms in the protease relative to the SF2 strain. In short, the protease variant genes (I84V, I50V/A71V) were constructed using QuikChange site-directed mutagenesis (Genewiz) onto NL4–3 wild-type protease on a pET11a plasmid with codon optimization for protein expression in <i>Escherichia coli</i>. A Q7K mutation was included to prevent autoproteolysis.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div></div><div id="sec5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Protein Expression and Purification</h3><div class="NLM_p last">The expression, isolation, and purification of WT and mutant HIV-1 proteases used for the kinetic assays and crystallization were carried out as previously described.<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36,37)</a> Briefly, the gene encoding the HIV protease was subcloned into the heat-inducible pXC35 expression vector (ATCC) and transformed into <i>E. coli</i> TAP-106 cells. Cells grown in 6 L of Terrific Broth were lysed with a cell disruptor, and the protein was purified from inclusion bodies.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> The inclusion body centrifugation pellet was dissolved in 50% acetic acid followed by another round of centrifugation to remove impurities. Size-exclusion chromatography was used to separate high-molecular-weight proteins from the desired protease. This was carried out using a 2.1 L Sephadex G-75 superfine column (Millipore Sigma) equilibrated with 50% acetic acid. The cleanest fractions of HIV protease were refolded into a 10-fold dilution of 0.05 M sodium acetate at pH 5.5, 5% ethylene glycol, 10% glycerol, and 5 mM DTT. The folded protein was concentrated down to 1–2 mg/mL and stored. This stored protease was used in <i>K</i>i assays. For crystallography, a final purification was performed with a Pharmacia Superdex 75 FPLC column equilibrated with 0.05 M sodium acetate at pH 5.5, 5% ethylene glycol, 10% glycerol, and 5 mM DTT. Protease fractions purified from the size-exclusion column were concentrated to 1–2 mg/mL using an Amicon Ultra-15 10-kDa device (Millipore) for crystallization.</div></div><div id="sec5_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Enzyme Inhibition Assays</h3><div class="NLM_p last">The enzyme inhibition assays were carried out as previously described.<a onclick="showRef(event, 'ref33 ref40'); return false;" href="javascript:void(0);" class="ref ref33 ref40">(33,40)</a> To determine the enzyme inhibition constant (<i>K</i>i), in a 96-well plate, each inhibitor was serially diluted, including a no drug control, and incubated with 0.35 nM protein for 1 h. A 10-amino acid substrate containing an optimized protease cleavage site with an EDANS/DABCYL FRET pair was dissolved in 4% DMSO at 120 μM. Using an Envision plate reader, 5 μL of the 120 μM substrate was added to the 96-well plate to a final concentration of 10 μM. Fluorescence was observed with an excitation at 340 nm and emission at 492 nm and monitored for 200 counts for approximately 60 min. Data was analyzed with Prism7. DRV was used as a control in all assays.</div></div><div id="sec5_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Antiviral Assays</h3><div class="NLM_p last">293T and TZM-BL<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> cells (NIH AIDS Research and Reference Reagent Program) were maintained in Dulbecco’s modified Eagle’s medium supplemented with 10% fetal calf serum in the presence of penicillin and streptomycin at 37 °C with 5% CO<sub>2</sub>. To determine the concentration of drugs achieving 50% inhibition of infection compared with the drug-free control, 4.5 × 10<sup>6</sup> 293T cells were seeded onto a 10 cm plate 24 h before transfection. Cells were transfected with 8 μg of either the wild-type plasmid or infectious molecular clone pNL-CH derived from the pNL4–3 clone of HIV-1 using a FuGENE 6 transfection reagent (Roche). The culture supernatant of 293T cells transfected with the wild-type HIV-1 was removed 18 h after transfection, and the cells were washed with 1× PBS. The 293T cells were collected and transferred to the wells of a 24-well plate. Briefly, each drug was serially diluted in the culture medium, and the dilutions were added to the wells of a 24-well plate. The 293T cells (0.5 × 10<sup>6</sup> per well) collected from the transfection were added to the wells containing various concentrations of drug. The culture supernatant containing virus particles was harvested 18 h after the 293T cells were reseeded in the presence of drug. This supernatant was filtered through a 0.45 μm pore size membrane (Millipore) to remove cell debris and then used to infect 2 × 10<sup>4</sup> TZM-BL cells in a 96-well plate following a procedure previously described.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> The culture supernatant was removed from each well 48 h of post-infection, and the cells were washed with 1× PBS. For the luciferase assay, infected TZM-BL cells were lysed in 1× reporter lysis buffer (Promega), and the cells were kept at −80 °C. After one freeze–thaw cycle, the cell lysates were transferred into a 96-well assay plate (Costar), and luciferase activity was measured using a luminometer (Promega). The culture supernatant harvested from 293T cells reseeded in the absence of drugs was used as a drug-free control. EC<sub>50</sub> was determined based on a dose–response curve generated using GraphPad Prism (version 7.0).</div></div><div id="sec5_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Protein Crystallization</h3><div class="NLM_p last">The condition reliably producing cocrystals of NL4–3 WT protease bound to PIs was discovered and optimized as previously described.<a onclick="showRef(event, 'ref40 ref43'); return false;" href="javascript:void(0);" class="ref ref40 ref43">(40,43)</a> Briefly, all cocrystals were grown at room temperature by the hanging drop vapor diffusion method in a 24-well VDX hanging-drop tray (Hampton Research) with a protease concentration of 1.4–1.7 mg/mL with 3-fold molar excess of inhibitors and mixed with the precipitant solution at a 1:2 ratio. The reservoir solution was 23–27% (w/v) ammonium sulfate with 0.1 M bis-Tris-methane buffer at pH 5.5, and the crystallization drops were set with 2 μL of well solution and 1 μL of protein–inhibitor solution and microseeded with a cat whisker. Diffraction-quality crystals were obtained within 1 week. As data were collected at 100 K, cryogenic conditions contained the precipitant solution supplemented with 25% glycerol.</div></div><div id="sec5_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> X-ray Data Collection and Structure Solution</h3><div class="NLM_p">X-ray diffraction data were collected and solved as previously described.<a onclick="showRef(event, 'ref36 ref40 ref43'); return false;" href="javascript:void(0);" class="ref ref36 ref40 ref43">(36,40,43)</a> Diffraction-quality crystals were flash-frozen under a cryostream when mounting the crystals either at our in-house Rigaku_Saturn944 X-ray system or the Chicago APS Synchrotron Beamline 23-1D-D. The cocrystal diffraction intensities from the Rigaku system were indexed, integrated, and scaled using HKL3000.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Structures were solved using molecular replacement with PHASER.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Model building and refinement were performed using Coot<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> and Phenix.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Ligands were designed in Maestro, and the output sdf files were used in the Phenix program eLBOW<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> to generate cif files containing atomic positions and constraints necessary for ligand refinement. Iterative rounds of crystallographic refinement were carried out until convergence was achieved. To limit bias throughout the refinement process, 5% of the data were reserved for the free <i>R</i>-value calculation.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> MolProbity<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> was applied to evaluate the final structures before deposition in the PDB. Structure analysis, superposition, and figure generation were done using PyMOL.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> X-ray data collection and crystallographic refinement statistics are presented in the Supporting Information (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">Table S1</a>).</div><div class="NLM_p last">The cocrystal structures of all new hybrid compounds were solved in the <i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub> space group with one protease homodimer in the asymmetric unit and only one orientation of the bound inhibitor in the active site, which was crucial for direct comparison of inhibitor structures. The cocrystal structure of compound <b>4</b> was solved in the <i>P</i>2<sub>1</sub> space group with two protease homodimers in the asymmetric unit and one inhibitor bound to each dimer in one orientation, allowing one <i>P</i>2<sub>1</sub> dimer to be directly compared to the crystal structures solved in the <i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub> space group. Three structures (<b>10a</b>, <b>10b</b>, <b>13c</b>) had significant electron density at the flap tips (residues 50–51) indicating that the backbone atoms interacted in two conformations as both hydrogen bond donors and acceptors. Figure generation and structural analysis calculations (distance differences, vdW, hydrogen bonds) were performed with the flap tips modeled in one conformation while the other conformation was excluded. Compound <b>5</b> formed cocrystals but were inadequate for X-ray data collection.</div></div><div id="sec5_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Intermolecular vdW Contact Analysis of Crystal Structures</h3><div class="NLM_p last">To calculate the intermolecular vdW interaction energies the crystal structures were prepared using the Schrödinger Protein Preparation Wizard.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Hydrogen atoms were added, protonation states were determined, and the structures were minimized. The protease active site was monoprotonated at Asp25. Subsequently, force field parameters were assigned using the OPLS3 force field.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Interaction energies between the inhibitor and protease were estimated using a simplified Lennard-Jones potential, as previously described in detail.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Briefly, the vdW energy was calculated for pairwise interactions depending on the types of atoms interacting and the distance between them. For each protease residue, the change in vdW interactions relative to a reference complex in the same space group was also calculated for each variant structure.</div></div><div id="sec5_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Molecular Dynamics Simulations</h3><div id="sec5_24_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> System Preparation</h4><div class="NLM_p last">High-resolution crystal structures of LPV and the designed inhibitors bound to WT protease were prepared using the Protein Preparation Wizard using Maestro within the Schrödinger Suite<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> as previously described.<a onclick="showRef(event, 'ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref55 ref56">(55,56)</a> Briefly, missing atoms were added using Prime,<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> and PROPKA<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> was used to determine the protonation state of the side chains at pH 7.0. The catalytic aspartic acid with a p<i>K</i>a higher than 7.0 was protonated, whereas the one with a p<i>K</i>a less than 7.0 was unprotonated. Cocrystallized fragments such as phosphate ions were removed. Lastly, the structure was minimized to a convergence criterion of 0.3 Å using Impref.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a></div></div><div id="sec5_24_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> Molecular Dynamics Simulations</h4><div class="NLM_p last">The prepared systems were placed in a cubic TIP3P implicit water box measuring 12 Å on each side. Molecular dynamics simulations were carried out as previously described<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> using Desmond within the Schrödinger Suite.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Briefly, chloride ions were used to neutralize the system, and 0.15 M salt was added using sodium and chloride ions. The OPLS3 force field was used to parametrize the ligand and protein. Prior to starting the 100 ns MD simulations, the solvated system was minimized using the stepwise procedure described previously.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> Triplicates of 100 ns simulations for LPV and the designed inhibitors in complex with WT protease each with a randomized velocity were started using a protocol previously developed.<a onclick="showRef(event, 'ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref55 ref56">(55,56)</a> The root-mean-square deviation (RMSD) and root-mean-square fluctuation (RMSF) were calculated using tools within the Schrödinger Suite.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00529" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86368" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86368" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00529?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00529</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Synthesis details and analytical data of Phe-Phe isostere <b>7</b>, amino acid derivatives <b>11a</b>–<b>h</b>, and inhibitors <b>4</b>–<b>5</b>; binding modes of parent compounds, comparison of per residue vdW contacts, RMSD analysis, distance distribution, intraprotease vdW interactions, protein RMSF analysis, and protease–inhibitor hydrogen bond frequencies from MD simulations; X-ray data collection and refinement statistics; and intermolecular hydrogen bonds between the protease and inhibitors (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf">jm0c00529_si_001.pdf (28.12 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_002.csv">jm0c00529_si_002.csv (2.46 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The PDB accession codes for X-ray cocrystal structures of wild-type HIV-1 protease with LPV, <b>4</b>, <b>12a</b>, <b>12b</b>, <b>12c</b>, <b>12d</b>, <b>12e</b>, <b>12f</b>, <b>12g</b>, <b>12h</b>, <b>15a</b>, <b>15b</b>, and <b>15c</b> are 6DGX, 6PJB, 6PJC, 6PJD, 6PJE, 6PJF, 6PJG, 6PJH, 6PJI, 6PJK, 6PJL, 6PJM, 6PJN, and 6PJO, respectively. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00529" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64833" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64833" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Celia A. Schiffer</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, Worcester, Massachusetts 01605, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2270-6613" title="Orcid link">http://orcid.org/0000-0003-2270-6613</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c586a0a9aca4eb96a6adaca3a3a0b785b0a8a4b6b6a8a0a1eba0a1b0"><span class="__cf_email__" data-cfemail="4e0d2b22272f601d2d262728282b3c0e3b232f3d3d232b2a602b2a3b">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Akbar Ali</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, Worcester, Massachusetts 01605, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3491-791X" title="Orcid link">http://orcid.org/0000-0003-3491-791X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#97d6fcf5f6e5b9d6fbfed7e2faf6e4e4faf2f3b9f2f3e2"><span class="__cf_email__" data-cfemail="cc8da7aeadbee28da0a58cb9a1adbfbfa1a9a8e2a9a8b9">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Linah N. Rusere</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, Worcester, Massachusetts 01605, United States</span>; 
    <span>Present Address:
                        Raybow Pharmaceutical USA, 158 McLean Road, Brevard, North Carolina 28712, United States (L.N.R.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gordon J. Lockbaum</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, Worcester, Massachusetts 01605, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2720-6984" title="Orcid link">http://orcid.org/0000-0003-2720-6984</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mina Henes</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, Worcester, Massachusetts 01605, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sook-Kyung Lee</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, and the UNC Center for AIDS Research, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ean Spielvogel</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, and the UNC Center for AIDS Research, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Desaboini Nageswara Rao</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, Worcester, Massachusetts 01605, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Klajdi Kosovrasti</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, Worcester, Massachusetts 01605, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ellen A. Nalivaika</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, Worcester, Massachusetts 01605, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ronald Swanstrom</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, and the UNC Center for AIDS Research, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nese Kurt Yilmaz</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Pharmacology, University of Massachusetts Medical School, Worcester, Massachusetts 01605, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5036-676X" title="Orcid link">http://orcid.org/0000-0002-5036-676X</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>L.N.R. and G.J.L. contributed equally to this work.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This work was supported by a grant from the National Institute of General Medical Sciences of the NIH (P01 GM109767).</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11750" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11750" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This research used resources of the Advanced Photon Source, a U.S. Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Argonne National Laboratory under Contract No. DE-AC02-06CH11357. GM/<a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="c2818382839291">[email protected]</a> has been funded in whole or in part with Federal funds from the National Cancer Institute (ACB-12002) and the National Institute of General Medical Sciences (AGM-12006). The Eiger 16M detector was funded by an NIH–Office of Research Infrastructure Programs, High-End Instrumentation Grant (1S10OD012289-01A1). We thank the beamline specialists at 23-ID-D for their help in data collection. We thank Ellen A. Nalivaika for providing the HIV-1 protease variants, the AIDS Research and Reference Reagent Program (NIAD, NIH) for reference protease inhibitors, and Drs. William Royer, Brian Kelch, and members of the Schiffer and Miller laboratories for helpful discussions. This work was supported by a grant from the National Institute of General Medical Sciences of the NIH (P01 GM109767).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ATV</td><td class="NLM_def"><p class="first last">atazanavir</p></td></tr><tr><td class="NLM_term">HIV-1</td><td class="NLM_def"><p class="first last">human immunodeficiency virus type 1</p></td></tr><tr><td class="NLM_term">bis-THF</td><td class="NLM_def"><p class="first last">bis-tetrahydrofuran</p></td></tr><tr><td class="NLM_term">Boc</td><td class="NLM_def"><p class="first last"><i>tert</i>-butoxycarbonyl</p></td></tr><tr><td class="NLM_term">DIEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DRV</td><td class="NLM_def"><p class="first last">darunavir</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate</p></td></tr><tr><td class="NLM_term">FRET</td><td class="NLM_def"><p class="first last">fluorescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">LPV</td><td class="NLM_def"><p class="first last">lopinavir</p></td></tr><tr><td class="NLM_term">MDR</td><td class="NLM_def"><p class="first last">multidrug-resistant</p></td></tr><tr><td class="NLM_term">PIs</td><td class="NLM_def"><p class="first last">protease inhibitors</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationships</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78018" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78018" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 59 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cihlar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fordyce, M.</span></span> <span> </span><span class="NLM_article-title">Current status and prospects of HIV treatment</span>. <i>Curr. Opin. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">50</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1016/j.coviro.2016.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fj.coviro.2016.03.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=27023283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC28Xksl2nsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2016&pages=50-56&author=T.+Cihlarauthor=M.+Fordyce&title=Current+status+and+prospects+of+HIV+treatment&doi=10.1016%2Fj.coviro.2016.03.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Current status and prospects of HIV treatment</span></div><div class="casAuthors">Cihlar, Tomas; Fordyce, Marshall</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Virology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">50-56</span>CODEN:
                <span class="NLM_cas:coden">COVUAF</span>;
        ISSN:<span class="NLM_cas:issn">1879-6257</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">Current antiviral treatments can reduce HIV-assocd. morbidity, prolong survival, and prevent HIV transmission.  Combination antiretroviral therapy (cART) contg. preferably three active drugs from two or more classes is required for durable virol. suppression.  Regimen selection is based on virol. efficacy, potential for adverse effects, pill burden and dosing frequency, drug-drug interaction potential, resistance test results, comorbid conditions, social status, and cost.  With prolonged virol. suppression, improved clin. outcomes, and longer survival, patients will be exposed to antiretroviral agents for decades.  Therefore, maximizing the safety and tolerability of cART is a high priority.  Emergence of resistance and/or lack of tolerability in individual patients require availability of a range of treatment options.  Development of new drugs is focused on improving safety (e.g. tenofovir alafenamide) and/or resistance profile (e.g. doravirine) within the existing drug classes, combination therapies with improved adherence (e.g. single-tablet regimens), novel mechanisms of action (e.g. attachment inhibitors, maturation inhibitors, broadly neutralizing antibodies), and treatment simplification with infrequent dosing (e.g. long-acting injectables).  In parallel with cART innovations, research and development efforts focused on agents that target persistent HIV reservoirs may lead to prolonged drug-free remission and HIV cure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpHDvTzsN1MbVg90H21EOLACvtfcHk0li1UbtpMczjiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xksl2nsrw%253D&md5=8f134da24f9491b3b2ac68bf7086de0f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.coviro.2016.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coviro.2016.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DCihlar%26aufirst%3DT.%26aulast%3DFordyce%26aufirst%3DM.%26atitle%3DCurrent%2520status%2520and%2520prospects%2520of%2520HIV%2520treatment%26jtitle%3DCurr.%2520Opin.%2520Virol.%26date%3D2016%26volume%3D18%26spage%3D50%26epage%3D56%26doi%3D10.1016%2Fj.coviro.2016.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saag, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benson, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandhi, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoy, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landovitz, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mugavero, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sax, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchbinder, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Rio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eron, J. J.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fatkenheuer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunthard, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volberding, P. A.</span></span> <span> </span><span class="NLM_article-title">Antiretroviral drugs for treatment and prevention of HIV infection in adults: 2018 recommendations of the international antiviral society-USA panel</span>. <i>JAMA</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>320</i></span>,  <span class="NLM_fpage">379</span>– <span class="NLM_lpage">396</span>, <span class="refDoi"> DOI: 10.1001/jama.2018.8431</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1001%2Fjama.2018.8431" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=30043070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVehtrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2018&pages=379-396&author=M.+S.+Saagauthor=C.+A.+Bensonauthor=R.+T.+Gandhiauthor=J.+F.+Hoyauthor=R.+J.+Landovitzauthor=M.+J.+Mugaveroauthor=P.+E.+Saxauthor=D.+M.+Smithauthor=M.+A.+Thompsonauthor=S.+P.+Buchbinderauthor=C.+Del+Rioauthor=J.+J.+Eronauthor=G.+Fatkenheuerauthor=H.+F.+Gunthardauthor=J.-M.+Molinaauthor=D.+M.+Jacobsenauthor=P.+A.+Volberding&title=Antiretroviral+drugs+for+treatment+and+prevention+of+HIV+infection+in+adults%3A+2018+recommendations+of+the+international+antiviral+society-USA+panel&doi=10.1001%2Fjama.2018.8431"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Antiretroviral drugs for treatment and prevention of HIV infection in adults: 2018 recommendations of the International Antiviral Society-USA Panel</span></div><div class="casAuthors">Saag, Michael S.; Benson, Constance A.; Gandhi, Rajesh T.; Hoy, Jennifer F.; Landovitz, Raphael J.; Mugavero, Michael J.; Sax, Paul E.; Smith, Davey M.; Thompson, Melanie A.; Buchbinder, Susan P.; del Rio, Carlos; Eron, Joseph J., Jr.; Fatkenheuer, Gerd; Gunthard, Huldrych F.; Molina, Jean-Michel; Jacobsen, Donna M.; Volberding, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">JAMA, the Journal of the American Medical Association</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">320</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">379-396</span>CODEN:
                <span class="NLM_cas:coden">JAMAAP</span>;
        ISSN:<span class="NLM_cas:issn">1538-3598</span>.
    
            (<span class="NLM_cas:orgname">American Medical Association</span>)
        </div><div class="casAbstract">Importance Antiretroviral therapy (ART) is the cornerstone of prevention and management of HIV infection. objective To evaluate new data and treatments and incorporate this information into updated recommendations for initiating therapy, monitoring individuals starting therapy, changing regimens, and preventing HIV infection for individuals at risk. evidence review New evidence collected since the International Antiviral Society-USA 2016 recommendations via monthly PubMed and EMBASE literature searches up to Apr. 2018; data presented at peer-reviewed scientific conferences.  A volunteer panel of experts in HIV research and patient care considered these data and updated previous recommendations. findings ART is recommended for virtually all HIV-infected individuals, as soon as possible after HIV diagnosis.  Immediate initiation (eg, rapid start), if clin. appropriate, requires adequate staffing, specialized services, and careful selection of medical therapy.  An integrase strand transfer inhibitor (InSTI) plus 2 nucleoside reverse transcriptase inhibitors (NRTIs) is generally recommended for initial therapy, with unique patient circumstances (eg, concomitant diseases and conditions, potential for pregnancy, cost) guiding the treatment choice.  CD4 cell count, HIV RNA level, genotype, and other lab. tests for general health and co-infections are recommended at specified points before and during ART.  If a regimen switch is indicated, treatment history, tolerability, adherence, and drug resistance history should first be assessed; 2 or 3 active drugs are recommended for a new regimen.  HIV testing is recommended at least once for anyone who has ever been sexually active and more often for individuals at ongoing risk for infection.  Preexposure prophylaxis with tenofovir disoproxil fumarate/emtricitabine and appropriate monitoring is recommended for individuals at risk for HIV. conclusions and relevance Advances in HIV prevention and treatment with antiretroviral drugs continue to improve clin. management and outcomes for individuals at risk for and living with HIV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZMBk4meAznbVg90H21EOLACvtfcHk0li1UbtpMczjiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVehtrrJ&md5=d2b4b70b43f71b3900fcd80abb54755d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1001%2Fjama.2018.8431&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Fjama.2018.8431%26sid%3Dliteratum%253Aachs%26aulast%3DSaag%26aufirst%3DM.%2BS.%26aulast%3DBenson%26aufirst%3DC.%2BA.%26aulast%3DGandhi%26aufirst%3DR.%2BT.%26aulast%3DHoy%26aufirst%3DJ.%2BF.%26aulast%3DLandovitz%26aufirst%3DR.%2BJ.%26aulast%3DMugavero%26aufirst%3DM.%2BJ.%26aulast%3DSax%26aufirst%3DP.%2BE.%26aulast%3DSmith%26aufirst%3DD.%2BM.%26aulast%3DThompson%26aufirst%3DM.%2BA.%26aulast%3DBuchbinder%26aufirst%3DS.%2BP.%26aulast%3DDel%2BRio%26aufirst%3DC.%26aulast%3DEron%26aufirst%3DJ.%2BJ.%26aulast%3DFatkenheuer%26aufirst%3DG.%26aulast%3DGunthard%26aufirst%3DH.%2BF.%26aulast%3DMolina%26aufirst%3DJ.-M.%26aulast%3DJacobsen%26aufirst%3DD.%2BM.%26aulast%3DVolberding%26aufirst%3DP.%2BA.%26atitle%3DAntiretroviral%2520drugs%2520for%2520treatment%2520and%2520prevention%2520of%2520HIV%2520infection%2520in%2520adults%253A%25202018%2520recommendations%2520of%2520the%2520international%2520antiviral%2520society-USA%2520panel%26jtitle%3DJAMA%26date%3D2018%26volume%3D320%26spage%3D379%26epage%3D396%26doi%3D10.1001%2Fjama.2018.8431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Panel on Antiretroviral Guidelines
for Adults and Adolescents</span>.  <i>Guidelines
for the Use of Antiretroviral Agents in Adults and Adolescents with
HIV</i>; Department of Health and Human Services. Available at <a href="https://files.aidsinfo.nih.gov/contentfiles/lvguidelines/AdultandAdolescentGL.pdf" class="extLink">https://files.aidsinfo.nih.gov/contentfiles/lvguidelines/AdultandAdolescentGL.pdf</a>. (Accessed February 15, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Panel+on+Antiretroviral+Guidelines%0Afor+Adults+and+Adolescents.+Guidelines%0Afor+the+Use+of+Antiretroviral+Agents+in+Adults+and+Adolescents+with%0AHIV%3B+Department+of+Health+and+Human+Services.+Available+at+https%3A%2F%2Ffiles.aidsinfo.nih.gov%2Fcontentfiles%2Flvguidelines%2FAdultandAdolescentGL.pdf.+%28Accessed+February+15%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DGuidelines%250Afor%2520the%2520Use%2520of%2520Antiretroviral%2520Agents%2520in%2520Adults%2520and%2520Adolescents%2520with%250AHIV" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandaranayake, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murzycki, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nalam, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nalivaika, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Özen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prabu-Jeyabalan, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thayer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Molecular basis for drug resistance in HIV-1 protease</span>. <i>Viruses</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2509</span>– <span class="NLM_lpage">2535</span>, <span class="refDoi"> DOI: 10.3390/v2112509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.3390%2Fv2112509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=21994628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFKjsrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=2509-2535&author=A.+Aliauthor=R.+M.+Bandaranayakeauthor=Y.+Caiauthor=N.+M.+Kingauthor=M.+Kolliauthor=S.+Mittalauthor=J.+F.+Murzyckiauthor=M.+N.+Nalamauthor=E.+A.+Nalivaikaauthor=A.+%C3%96zenauthor=M.+M.+Prabu-Jeyabalanauthor=K.+Thayerauthor=C.+A.+Schiffer&title=Molecular+basis+for+drug+resistance+in+HIV-1+protease&doi=10.3390%2Fv2112509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for drug resistance in HIV-1 protease</span></div><div class="casAuthors">Ali, Akbar; Bandaranayake, Rajintha M.; Cai, Yufeng; King, Nancy M.; Kolli, Madhavi; Mittal, Seema; Murzycki, Jennifer F.; Nalam, Madhavi N. L.; Nalivaika, Ellen A.; Ozen, Aysegul; Prabu-Jeyabalan, Moses M.; Thayer, Kelly; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Viruses</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2509-2535</span>CODEN:
                <span class="NLM_cas:coden">VIRUBR</span>;
        ISSN:<span class="NLM_cas:issn">1999-4915</span>.
    
            (<span class="NLM_cas:orgname">Molecular Diversity Preservation International</span>)
        </div><div class="casAbstract">A review.  HIV-1 protease is one of the major antiviral targets in the treatment of patients infected with HIV-1.  The nine FDA approved HIV-1 protease inhibitors were developed with extensive use of structure-based drug design, thus the at. details of how the inhibitors bind are well characterized.  From this structural understanding the mol. basis for drug resistance in HIV-1 protease can be elucidated.  Selected mutations in response to therapy and diversity between clades in HIV-1 protease have altered the shape of the active site, potentially altered the dynamics and even altered the sequence of the cleavage sites in the Gag polyprotein.  All of these interdependent changes act in synergy to confer drug resistance while simultaneously maintaining the fitness of the virus.  New strategies, such as incorporation of the substrate envelope constraint to design robust inhibitors that incorporate details of HIV-1 protease's function and decrease the probability of drug resistance, are necessary to continue to effectively target this key protein in HIV-1 life cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBf4i5iKmj8rVg90H21EOLACvtfcHk0lg_iBxFuZuHag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFKjsrzL&md5=3c5a791d1277a18b800c5849403333b5</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3390%2Fv2112509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fv2112509%26sid%3Dliteratum%253Aachs%26aulast%3DAli%26aufirst%3DA.%26aulast%3DBandaranayake%26aufirst%3DR.%2BM.%26aulast%3DCai%26aufirst%3DY.%26aulast%3DKing%26aufirst%3DN.%2BM.%26aulast%3DKolli%26aufirst%3DM.%26aulast%3DMittal%26aufirst%3DS.%26aulast%3DMurzycki%26aufirst%3DJ.%2BF.%26aulast%3DNalam%26aufirst%3DM.%2BN.%26aulast%3DNalivaika%26aufirst%3DE.%2BA.%26aulast%3D%25C3%2596zen%26aufirst%3DA.%26aulast%3DPrabu-Jeyabalan%26aufirst%3DM.%2BM.%26aulast%3DThayer%26aufirst%3DK.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DMolecular%2520basis%2520for%2520drug%2520resistance%2520in%2520HIV-1%2520protease%26jtitle%3DViruses%26date%3D2010%26volume%3D2%26spage%3D2509%26epage%3D2535%26doi%3D10.3390%2Fv2112509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osswald, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prato, G.</span></span> <span> </span><span class="NLM_article-title">Recent progress in the development of HIV-1 protease inhibitors for the treatment of HIV/AIDS</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5172</span>– <span class="NLM_lpage">5208</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01697</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01697" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5172-5208&author=A.+K.+Ghoshauthor=H.+L.+Osswaldauthor=G.+Prato&title=Recent+progress+in+the+development+of+HIV-1+protease+inhibitors+for+the+treatment+of+HIV%2FAIDS&doi=10.1021%2Facs.jmedchem.5b01697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Progress in the Development of HIV-1 Protease Inhibitors for the Treatment of HIV/AIDS</span></div><div class="casAuthors">Ghosh, Arun K.; Osswald, Heather L.; Prato, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5172-5208</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HIV-1 protease inhibitors continue to play an important role in the treatment of HIV/AIDS, transforming this deadly ailment into a more manageable chronic infection.  Over the years, intensive research has led to a variety of approved protease inhibitors for the treatment of HIV/AIDS.  In this review, we outline current drug design and medicinal chem. efforts toward the development of next-generation protease inhibitors beyond the currently approved drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4DisuSRbQXbVg90H21EOLACvtfcHk0lg_iBxFuZuHag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhsL0%253D&md5=e0994f3d84ab627d3ac3f757e8d11725</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01697%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DOsswald%26aufirst%3DH.%2BL.%26aulast%3DPrato%26aufirst%3DG.%26atitle%3DRecent%2520progress%2520in%2520the%2520development%2520of%2520HIV-1%2520protease%2520inhibitors%2520for%2520the%2520treatment%2520of%2520HIV%252FAIDS%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5172%26epage%3D5208%26doi%3D10.1021%2Facs.jmedchem.5b01697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Surleraux, D. L. N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tahri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verschueren, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pille, G. M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de
Kock, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonckers, T. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peeters, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Meyer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azijn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauwels, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bethune, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prabu-Jeyabalan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigerinck, P. B. T. P.</span></span> <span> </span><span class="NLM_article-title">Discovery and selection of TMC114, a next generation HIV-1 protease inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1813</span>– <span class="NLM_lpage">1822</span>, <span class="refDoi"> DOI: 10.1021/jm049560p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049560p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotVOlu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=1813-1822&author=D.+L.+N.+G.+Surlerauxauthor=A.+Tahriauthor=W.+G.+Verschuerenauthor=G.+M.+E.+Pilleauthor=H.+A.+de%0AKockauthor=T.+H.+M.+Jonckersauthor=A.+Peetersauthor=S.+De+Meyerauthor=H.+Azijnauthor=R.+Pauwelsauthor=M.+P.+de+Bethuneauthor=N.+M.+Kingauthor=M.+Prabu-Jeyabalanauthor=C.+A.+Schifferauthor=P.+B.+T.+P.+Wigerinck&title=Discovery+and+selection+of+TMC114%2C+a+next+generation+HIV-1+protease+inhibitor&doi=10.1021%2Fjm049560p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Selection of TMC114, a Next Generation HIV-1 Protease Inhibitor</span></div><div class="casAuthors">Surleraux, Dominique L. N. G.; Tahri, Abdellah; Verschueren, Wim G.; Pille, Geert M. E.; de Kock, Herman A.; Jonckers, Tim H. M.; Peeters, Anik; De Meyer, Sandra; Azijn, Hilde; Pauwels, Rudi; de Bethune, Marie-Pierre; King, Nancy M.; Prabu-Jeyabalan, Moses; Schiffer, Celia A.; Wigerinck, Piet B. T. P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1813-1822</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The screening of known HIV-1 protease inhibitors against a panel of multidrug-resistant viruses revealed the potent activity of TMC126 on drug-resistant mutants.  In comparison to amprenavir, the improved affinity of TMC126 is largely the result of one extra hydrogen bond to the backbone of the protein in the P2 pocket.  Modification of the substitution pattern on the phenylsulfonamide P2' substituent of TMC126 created an interesting SAR, with the close analog TMC114 being found to have a similar antiviral activity against the mutant and the wild-type viruses.  X-ray and thermodn. studies on both wild-type and mutant enzymes showed an extremely high enthalpy driven affinity of TMC114 for HIV-1 protease.  In vitro selection of mutants resistant to TMC114 starting from wild-type virus proved to be extremely difficult; this was not the case for other close analogs.  Therefore, the extra H-bond to the backbone in the P2 pocket cannot be the only explanation for the interesting antiviral profile of TMC114.  Absorption studies in animals indicated that TMC114 has pharmacokinetic properties comparable to currently approved HIV-1 protease inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdYfZu3Lc6mbVg90H21EOLACvtfcHk0lh-3D7Of6OF2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotVOlu7Y%253D&md5=35809ccdec57ebb6afa7af1b80362f74</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm049560p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049560p%26sid%3Dliteratum%253Aachs%26aulast%3DSurleraux%26aufirst%3DD.%2BL.%2BN.%2BG.%26aulast%3DTahri%26aufirst%3DA.%26aulast%3DVerschueren%26aufirst%3DW.%2BG.%26aulast%3DPille%26aufirst%3DG.%2BM.%2BE.%26aulast%3Dde%2BKock%26aufirst%3DH.%2BA.%26aulast%3DJonckers%26aufirst%3DT.%2BH.%2BM.%26aulast%3DPeeters%26aufirst%3DA.%26aulast%3DDe%2BMeyer%26aufirst%3DS.%26aulast%3DAzijn%26aufirst%3DH.%26aulast%3DPauwels%26aufirst%3DR.%26aulast%3Dde%2BBethune%26aufirst%3DM.%2BP.%26aulast%3DKing%26aufirst%3DN.%2BM.%26aulast%3DPrabu-Jeyabalan%26aufirst%3DM.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26aulast%3DWigerinck%26aufirst%3DP.%2BB.%2BT.%2BP.%26atitle%3DDiscovery%2520and%2520selection%2520of%2520TMC114%252C%2520a%2520next%2520generation%2520HIV-1%2520protease%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D1813%26epage%3D1822%26doi%3D10.1021%2Fjm049560p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapsal, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuya, H.</span></span> <span> </span><span class="NLM_article-title">Design of HIV protease inhibitors targeting protein backbone: an effective strategy for combating drug resistance</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1021/ar7001232</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar7001232" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFyhsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2008&pages=78-86&author=A.+K.+Ghoshauthor=B.+D.+Chapsalauthor=I.+T.+Weberauthor=H.+Mitsuya&title=Design+of+HIV+protease+inhibitors+targeting+protein+backbone%3A+an+effective+strategy+for+combating+drug+resistance&doi=10.1021%2Far7001232"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Design of HIV Protease Inhibitors Targeting Protein Backbone: An Effective Strategy for Combating Drug Resistance</span></div><div class="casAuthors">Ghosh, Arun K.; Chapsal, Bruno D.; Weber, Irene T.; Mitsuya, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-86</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The discovery of HIV protease inhibitors (PIs) followed by their incorporation into highly active antiretroviral therapy (HAART) was a major turning point for the management of HIV/AIDS.  However, the rapid emergence of drug resistance has become a major concern.  Our structure-based design of HIV-1 PIs that can specifically interact with the enzyme backbone atoms has led to the development of compds. with unprecedented potency and drug-resistance profiles.  Particularly notable is the development of a stereochem. defined bis(tetrahydrofuranyl)urethane as the non-peptidic P2 ligand.  Incorporation of this ligand has provided a variety of exceedingly potent HIV-1 PIs with superior activity against multi-PI-resistant variants relative to other FDA-approved PIs.  Darunavir, a PI recently approved for the treatment of drug-resistant HIV, directly resulted from this strategy.  Our detailed protein-darunavir X-ray crystal structures have clearly demonstrated extensive interactions with the protein backbone in the enzyme active site.  This present design concept targeting the protein backbone may serve as a powerful strategy to combat drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGZCuOKomAEbVg90H21EOLACvtfcHk0lh-3D7Of6OF2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFyhsbo%253D&md5=a8ee6f368273a5fece6ebbaf9bafc750</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Far7001232&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far7001232%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DChapsal%26aufirst%3DB.%2BD.%26aulast%3DWeber%26aufirst%3DI.%2BT.%26aulast%3DMitsuya%26aufirst%3DH.%26atitle%3DDesign%2520of%2520HIV%2520protease%2520inhibitors%2520targeting%2520protein%2520backbone%253A%2520an%2520effective%2520strategy%2520for%2520combating%2520drug%2520resistance%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2008%26volume%3D41%26spage%3D78%26epage%3D86%26doi%3D10.1021%2Far7001232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boross, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaddis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leshchenko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, I. T.</span></span> <span> </span><span class="NLM_article-title">High resolution crystal structures of HIV-1 protease with a potent non-peptide inhibitor (UIC-94017) active against multi-drug-resistant clinical strains</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>338</i></span>,  <span class="NLM_fpage">341</span>– <span class="NLM_lpage">352</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2004.02.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fj.jmb.2004.02.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=15066436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2cXivVeltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=338&publication_year=2004&pages=341-352&author=Y.+Tieauthor=P.+I.+Borossauthor=Y.+F.+Wangauthor=L.+Gaddisauthor=A.+K.+Hussainauthor=S.+Leshchenkoauthor=A.+K.+Ghoshauthor=J.+M.+Louisauthor=R.+W.+Harrisonauthor=I.+T.+Weber&title=High+resolution+crystal+structures+of+HIV-1+protease+with+a+potent+non-peptide+inhibitor+%28UIC-94017%29+active+against+multi-drug-resistant+clinical+strains&doi=10.1016%2Fj.jmb.2004.02.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">High Resolution Crystal Structures of HIV-1 Protease with a Potent Non-peptide Inhibitor (UIC-94017) Active Against Multi-drug-resistant Clinical Strains</span></div><div class="casAuthors">Tie, Yunfeng; Boross, Peter I.; Wang, Yuan-Fang; Gaddis, Laquasha; Hussain, Azhar K.; Leshchenko, Sofiya; Ghosh, Arun K.; Louis, John M.; Harrison, Robert W.; Weber, Irene T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">338</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">341-352</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The compd. UIC-94017 (TMC-114) is a second-generation HIV protease inhibitor with improved pharmacokinetics that is chem. related to the clin. inhibitor amprenavir.  UIC-94017 is a broad-spectrum potent inhibitor active against HIV-1 clin. isolates with minimal cytotoxicity.  We have detd. the high-resoln. crystal structures of UIC-94017 in complexes with wild-type HIV-1 protease (PR) and mutant proteases PRV82A and PRI84V that are common in drug-resistant HIV.  The structures were refined at resolns. of 1.10 - 1.53 Å.  The crystal structures of PR and PRI84V with UIC-94017 ternary complexes show that the inhibitor binds to the protease in two overlapping positions, while the PRV82A complex had one ordered inhibitor.  In all three structures, UIC-94017 forms hydrogen bonds with the conserved main-chain atoms of Asp29 and Asp30 of the protease.  These interactions are proposed to be crit. for the potency of this compd. against HIV isolates that are resistant to multiple protease inhibitors.  Other small differences were obsd. in the interactions of the mutants with UIC-94017 as compared to PR.  PRV82A showed differences in the position of the main-chain atoms of residue 82 compared to PR structure that better accommodated the inhibitor.  Finally, the 1.10 Å resoln. structure of PRV82A with UIC-94017 showed an unusual distribution of electron d. for the catalytic aspartate residues, which is discussed in relation to the reaction mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHW51Not1WA7Vg90H21EOLACvtfcHk0lh-3D7Of6OF2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXivVeltL0%253D&md5=82ecc0bb118a886b254ac579ffe9c5fc</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2004.02.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2004.02.052%26sid%3Dliteratum%253Aachs%26aulast%3DTie%26aufirst%3DY.%26aulast%3DBoross%26aufirst%3DP.%2BI.%26aulast%3DWang%26aufirst%3DY.%2BF.%26aulast%3DGaddis%26aufirst%3DL.%26aulast%3DHussain%26aufirst%3DA.%2BK.%26aulast%3DLeshchenko%26aufirst%3DS.%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DLouis%26aufirst%3DJ.%2BM.%26aulast%3DHarrison%26aufirst%3DR.%2BW.%26aulast%3DWeber%26aufirst%3DI.%2BT.%26atitle%3DHigh%2520resolution%2520crystal%2520structures%2520of%2520HIV-1%2520protease%2520with%2520a%2520potent%2520non-peptide%2520inhibitor%2520%2528UIC-94017%2529%2520active%2520against%2520multi-drug-resistant%2520clinical%2520strains%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D338%26spage%3D341%26epage%3D352%26doi%3D10.1016%2Fj.jmb.2004.02.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramu Sridhar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumaragurubaran, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuya, H.</span></span> <span> </span><span class="NLM_article-title">Bis-tetrahydrofuran: a privileged ligand for darunavir and a new generation of hiv protease inhibitors that combat drug resistance</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">939</span>– <span class="NLM_lpage">950</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200600103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1002%2Fcmdc.200600103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=16927344" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsVCmu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=939-950&author=A.+K.+Ghoshauthor=P.+Ramu+Sridharauthor=N.+Kumaragurubaranauthor=Y.+Kohauthor=I.+T.+Weberauthor=H.+Mitsuya&title=Bis-tetrahydrofuran%3A+a+privileged+ligand+for+darunavir+and+a+new+generation+of+hiv+protease+inhibitors+that+combat+drug+resistance&doi=10.1002%2Fcmdc.200600103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Bis-tetrahydrofuran: a privileged ligand for darunavir and a new generation of HIV protease inhibitors that combat drug resistance</span></div><div class="casAuthors">Ghosh, Arun K.; Sridhar, Perali Ramu; Kumaragurubaran, Nagaswamy; Koh, Yasuhiro; Weber, Irene T.; Mitsuya, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">939-950</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The structure-based design of bis-tetrahydrofuranyl urethane has emerged as a privileged nonpeptide P2 ligand for a variety of highly potent HIV-1 protease inhibitors.  Incorporation of this ligand provided HIV protease inhibitors with exceedingly potent antiviral activity and superior activity against multi-PI-resistant variants relative to other FDA-approved PIs.  Recently, TMC-114 (darunavir) was approved by the FDA for treatment of drug-resistant HIV.  GW640385 (brecanavir), which incorporates bis-THF as the P2 ligand, is currently in Phase-III clin. development.  The bis-THF ligand was specifically designed to fill in the hydrophobic S2 pocket effectively and to promote extensive hydrogen bonding with the protein backbone in the enzyme S2 site.  The protein-ligand x-ray crystal structures with TMC-114 and other inhibitors with the bis-THF ligand revealed extensive interactions with the backbone of residues Asp29 and Asp30 at the S2 site.  The current design concept targeting the protein backbone may serve as an important guide to combat drug resistance.  Further design and synthesis of conceptually novel inhibitors are in progress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMpT49sY24kLVg90H21EOLACvtfcHk0lhYVyiwVu_-uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsVCmu7w%253D&md5=e9826330e807476b916d69e86ff55a40</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200600103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200600103%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DRamu%2BSridhar%26aufirst%3DP.%26aulast%3DKumaragurubaran%26aufirst%3DN.%26aulast%3DKoh%26aufirst%3DY.%26aulast%3DWeber%26aufirst%3DI.%2BT.%26aulast%3DMitsuya%26aufirst%3DH.%26atitle%3DBis-tetrahydrofuran%253A%2520a%2520privileged%2520ligand%2520for%2520darunavir%2520and%2520a%2520new%2520generation%2520of%2520hiv%2520protease%2520inhibitors%2520that%2520combat%2520drug%2520resistance%26jtitle%3DChemMedChem%26date%3D2006%26volume%3D1%26spage%3D939%26epage%3D950%26doi%3D10.1002%2Fcmdc.200600103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hazen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferris, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craig, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaldor, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samano, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furfine, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spaltenstein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tung, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St Clair, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanlon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boone, L.</span></span> <span> </span><span class="NLM_article-title">In vitro antiviral activity of the novel, tyrosyl-based human immunodeficiency virus (HIV) type 1 protease inhibitor brecanavir (GW640385) in combination with other antiretrovirals and against a panel of protease inhibitor-resistant HIV</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">3147</span>– <span class="NLM_lpage">3154</span>, <span class="refDoi"> DOI: 10.1128/AAC.00401-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1128%2FAAC.00401-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=17620375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVais73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2007&pages=3147-3154&author=R.+Hazenauthor=R.+Harveyauthor=R.+Ferrisauthor=C.+Craigauthor=P.+Yatesauthor=P.+Griffinauthor=J.+Millerauthor=I.+Kaldorauthor=J.+Rayauthor=V.+Samanoauthor=E.+Furfineauthor=A.+Spaltensteinauthor=M.+Haleauthor=R.+Tungauthor=M.+St+Clairauthor=M.+Hanlonauthor=L.+Boone&title=In+vitro+antiviral+activity+of+the+novel%2C+tyrosyl-based+human+immunodeficiency+virus+%28HIV%29+type+1+protease+inhibitor+brecanavir+%28GW640385%29+in+combination+with+other+antiretrovirals+and+against+a+panel+of+protease+inhibitor-resistant+HIV&doi=10.1128%2FAAC.00401-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro antiviral activity of the novel, tyrosyl-based human immunodeficiency virus (HIV) type 1 protease inhibitor brecanavir (GW640385) in combination with other antiretrovirals and against a panel of protease inhibitor-resistant HIV</span></div><div class="casAuthors">Hazen, Richard; Harvey, Robert; Ferris, Robert; Craig, Charles; Yates, Phillip; Griffin, Philip; Miller, John; Kaldor, Istvan; Ray, John; Samano, Vincente; Furfine, Eric; Spaltenstein, Andrew; Hale, Michael; Tung, Roger; St. Clair, Marty; Hanlon, Mary; Boone, Lawrence</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3147-3154</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Brecanavir, a novel tyrosyl-based arylsulfonamide, high-affinity, human immunodeficiency virus type 1 (HIV-1) protease inhibitor (PI), has been evaluated for anti-HIV activity in several in vitro assays.  Preclin. assessment of brecanavir indicated that this compd. potently inhibited HIV-1 in cell culture assays with 50% effective concns. (EC50s) of 0.2 to 0.53 nM and was equally active against HIV strains utilizing either the CXCR4 or CCR5 coreceptor, as was found with other PIs.  The presence of up to 40% human serum decreased the anti-HIV-1 activity of brecanavir by 5.2-fold, but under these conditions the compd. retained single-digit nanomolar EC50s.  When brecanavir was tested in combination with nucleoside reverse transcriptase inhibitors, the antiviral activity of brecanavir was synergistic with the effects of stavudine and additive to the effects of zidovudine, tenofovir, dideoxycytidine, didanosine, adefovir, abacavir, lamivudine, and emtricitabine.  Brecanavir was synergistic with the nonnucleoside reverse transcriptase inhibitor nevirapine or delavirdine and was additive to the effects of efavirenz.  In combination with other PIs, brecanavir was additive to the activities of indinavir, lopinavir, nelfinavir, ritonavir, amprenavir, saquinavir, and atazanavir.  Clin. HIV isolates from PI-experienced patients were evaluated for sensitivity to brecanavir and other PIs in a recombinant virus assay.  Brecanavir had a <5-fold increase in EC50s against 80% of patient isolates tested and had a greater mean in vitro potency than amprenavir, indinavir, lopinavir, atazanavir, tipranavir, and darunavir.  Brecanavir is by a substantial margin the most potent and broadly active antiviral agent among the PIs tested in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqplFzzXvQJ9rVg90H21EOLACvtfcHk0lhYVyiwVu_-uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVais73F&md5=a84fb4b36143fb6c9e92db4895ad1d9d</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1128%2FAAC.00401-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00401-07%26sid%3Dliteratum%253Aachs%26aulast%3DHazen%26aufirst%3DR.%26aulast%3DHarvey%26aufirst%3DR.%26aulast%3DFerris%26aufirst%3DR.%26aulast%3DCraig%26aufirst%3DC.%26aulast%3DYates%26aufirst%3DP.%26aulast%3DGriffin%26aufirst%3DP.%26aulast%3DMiller%26aufirst%3DJ.%26aulast%3DKaldor%26aufirst%3DI.%26aulast%3DRay%26aufirst%3DJ.%26aulast%3DSamano%26aufirst%3DV.%26aulast%3DFurfine%26aufirst%3DE.%26aulast%3DSpaltenstein%26aufirst%3DA.%26aulast%3DHale%26aufirst%3DM.%26aulast%3DTung%26aufirst%3DR.%26aulast%3DSt%2BClair%26aufirst%3DM.%26aulast%3DHanlon%26aufirst%3DM.%26aulast%3DBoone%26aufirst%3DL.%26atitle%3DIn%2520vitro%2520antiviral%2520activity%2520of%2520the%2520novel%252C%2520tyrosyl-based%2520human%2520immunodeficiency%2520virus%2520%2528HIV%2529%2520type%25201%2520protease%2520inhibitor%2520brecanavir%2520%2528GW640385%2529%2520in%2520combination%2520with%2520other%2520antiretrovirals%2520and%2520against%2520a%2520panel%2520of%2520protease%2520inhibitor-resistant%2520HIV%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2007%26volume%3D51%26spage%3D3147%26epage%3D3154%26doi%3D10.1128%2FAAC.00401-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cihlar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulato, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leavitt, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stray, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W. A.</span></span> <span> </span><span class="NLM_article-title">Suppression of HIV-1 protease inhibitor resistance by phosphonate-mediated solvent anchoring</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>363</i></span>,  <span class="NLM_fpage">635</span>– <span class="NLM_lpage">647</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2006.07.073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fj.jmb.2006.07.073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=16979654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVCrurbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=363&publication_year=2006&pages=635-647&author=T.+Cihlarauthor=G.+X.+Heauthor=X.+Liuauthor=J.+M.+Chenauthor=M.+Hatadaauthor=S.+Swaminathanauthor=M.+J.+McDermottauthor=Z.+Y.+Yangauthor=A.+S.+Mulatoauthor=X.+Chenauthor=S.+A.+Leavittauthor=K.+M.+Strayauthor=W.+A.+Lee&title=Suppression+of+HIV-1+protease+inhibitor+resistance+by+phosphonate-mediated+solvent+anchoring&doi=10.1016%2Fj.jmb.2006.07.073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of HIV-1 Protease Inhibitor Resistance by Phosphonate-mediated Solvent Anchoring</span></div><div class="casAuthors">Cihlar, Tomas; He, Gong-Xin; Liu, Xiaohong; Chen, James M.; Hatada, Marcos; Swaminathan, Swami; McDermott, Martin J.; Yang, Zheng-Yu; Mulato, Andrew S.; Chen, Xiaowu; Leavitt, Stephanie A.; Stray, Kirsten M.; Lee, William A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">363</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">635-647</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The introduction of human immunodeficiency virus type 1 (HIV-1) protease inhibitors (PIs) markedly improved the clin. outcome and control of HIV-1 infection.  However, cross-resistance among PIs due to a wide spectrum of mutations in viral protease is a major factor limiting their broader clin. use.  Here we report on the suppression of PI resistance using a covalent attachment of a phosphonic acid motif to a peptidomimetic inhibitor scaffold.  The resulting phosphonate analogs maintain high binding affinity to HIV-1 protease, potent antiretroviral activity, and unlike the parent mols., display no loss of potency against a panel of clin. important PI-resistant HIV-1 strains.  As shown by crystallog. anal., the phosphonate moiety is highly exposed to solvent with no discernable interactions with any of the enzyme active site or surface residues.  We term this effect "solvent anchoring" and demonstrate that it is driven by a favorable change in the inhibitor binding entropy upon the interaction with mutant enzymes.  This type of thermodn. behavior, which was not found with the parent scaffold fully buried in the enzyme active site, is a result of the increased degeneracy of inhibitor binding states, allowing effective mol. adaptation to the expanded cavity vol. of mutant proteases.  This strategy, which is applicable to various PI scaffolds, should facilitate the design of novel PIs and potentially other antiviral therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgTKggUJt9RLVg90H21EOLACvtfcHk0lguZ17g0cfdOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVCrurbE&md5=46b1cae8f52f1acb3246c84ae6e8708e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2006.07.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2006.07.073%26sid%3Dliteratum%253Aachs%26aulast%3DCihlar%26aufirst%3DT.%26aulast%3DHe%26aufirst%3DG.%2BX.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%2BM.%26aulast%3DHatada%26aufirst%3DM.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DMcDermott%26aufirst%3DM.%2BJ.%26aulast%3DYang%26aufirst%3DZ.%2BY.%26aulast%3DMulato%26aufirst%3DA.%2BS.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLeavitt%26aufirst%3DS.%2BA.%26aulast%3DStray%26aufirst%3DK.%2BM.%26aulast%3DLee%26aufirst%3DW.%2BA.%26atitle%3DSuppression%2520of%2520HIV-1%2520protease%2520inhibitor%2520resistance%2520by%2520phosphonate-mediated%2520solvent%2520anchoring%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D363%26spage%3D635%26epage%3D647%26doi%3D10.1016%2Fj.jmb.2006.07.073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dandache, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sévigny, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yelle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stranix, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapiro, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wainberg, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. J.</span></span> <span> </span><span class="NLM_article-title">In vitro antiviral activity and cross-resistance profile of PL-100, a novel protease inhibitor of human immunodeficiency virus type 1</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">4036</span>– <span class="NLM_lpage">4043</span>, <span class="refDoi"> DOI: 10.1128/AAC.00149-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1128%2FAAC.00149-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=17638694" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1ygtL3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2007&pages=4036-4043&author=S.+Dandacheauthor=G.+S%C3%A9vignyauthor=J.+Yelleauthor=B.+R.+Stranixauthor=N.+Parkinauthor=J.+M.+Schapiroauthor=M.+A.+Wainbergauthor=J.+J.+Wu&title=In+vitro+antiviral+activity+and+cross-resistance+profile+of+PL-100%2C+a+novel+protease+inhibitor+of+human+immunodeficiency+virus+type+1&doi=10.1128%2FAAC.00149-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro antiviral activity and cross-resistance profile of PL-100, a novel protease inhibitor of human immunodeficiency virus type 1</span></div><div class="casAuthors">Dandache, Serge; Sevigny, Guy; Yelle, Jocelyn; Stranix, Brent R.; Parkin, Neil; Schapiro, Jonathan M.; Wainberg, Mark A.; Wu, Jinzi J.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4036-4043</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Despite the success of highly active antiretroviral therapy, the current emergence and spread of drug-resistant variants of human immunodeficiency virus (HIV) stress the need for new inhibitors with distinct properties.  We designed, produced, and screened a library of compds. based on an original L-lysine scaffold for their potentials as HIV type 1 (HIV-1) protease inhibitors (PI).  One candidate compd., PL-100, emerged as a specific and noncytotoxic PI that exhibited potent inhibition of HIV-1 protease and viral replication in vitro (Ki, ∼36 pM, and 50% effective concn. [EC50], ∼16 nM, resp.).  To confirm that PL-100 possessed a favorable resistance profile, we performed a cross-resistance study using a panel of 63 viral strains from PI-experienced patients selected for the presence of primary PI mutations known to confer resistance to multiple PIs now in clin. use.  The results showed that PL-100 retained excellent antiviral activity against almost all of these PI-resistant viruses and that its performance in this regard was superior to those of atazanavir, amprenavir, indinavir, lopinavir, nelfinavir, and saquinavir.  In almost every case, the increase in the EC50 for PL-100 obsd. with viruses contg. multiple mutations in protease was far less than that obtained with the other drugs tested.  These data underscore the potential for PL-100 to be used in the treatment of drug-resistant HIV disease and argue for its further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3g2z6DZZfnbVg90H21EOLACvtfcHk0lguZ17g0cfdOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1ygtL3M&md5=165d8269fa4fcc5f6592a58250570fe9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1128%2FAAC.00149-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00149-07%26sid%3Dliteratum%253Aachs%26aulast%3DDandache%26aufirst%3DS.%26aulast%3DS%25C3%25A9vigny%26aufirst%3DG.%26aulast%3DYelle%26aufirst%3DJ.%26aulast%3DStranix%26aufirst%3DB.%2BR.%26aulast%3DParkin%26aufirst%3DN.%26aulast%3DSchapiro%26aufirst%3DJ.%2BM.%26aulast%3DWainberg%26aufirst%3DM.%2BA.%26aulast%3DWu%26aufirst%3DJ.%2BJ.%26atitle%3DIn%2520vitro%2520antiviral%2520activity%2520and%2520cross-resistance%2520profile%2520of%2520PL-100%252C%2520a%2520novel%2520protease%2520inhibitor%2520of%2520human%2520immunodeficiency%2520virus%2520type%25201%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2007%26volume%3D51%26spage%3D4036%26epage%3D4043%26doi%3D10.1128%2FAAC.00149-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kesteleyn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amssoms, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepens, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hache, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verschueren, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van De Vreken, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rombauts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meurs, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterkens, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoops, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baert, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masungi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dierynck, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundgren, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jönsson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkes, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalayanov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenquist, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samuelsson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Emelen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thuring, J. W.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of HIV-1 protease inhibitors for a long-acting injectable drug application</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">310</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.10.095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fj.bmcl.2012.10.095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=23177258" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsleitrnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=310-317&author=B.+Kesteleynauthor=K.+Amssomsauthor=W.+Schepensauthor=G.+Hacheauthor=W.+Verschuerenauthor=W.+Van+De+Vrekenauthor=K.+Rombautsauthor=G.+Meursauthor=P.+Sterkensauthor=B.+Stoopsauthor=L.+Baertauthor=N.+Austinauthor=J.+Wegnerauthor=C.+Masungiauthor=I.+Dierynckauthor=S.+Lundgrenauthor=D.+J%C3%B6nssonauthor=K.+Parkesauthor=G.+Kalayanovauthor=H.+Wallbergauthor=A.+Rosenquistauthor=B.+Samuelssonauthor=K.+Van+Emelenauthor=J.+W.+Thuring&title=Design+and+synthesis+of+HIV-1+protease+inhibitors+for+a+long-acting+injectable+drug+application&doi=10.1016%2Fj.bmcl.2012.10.095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of HIV-1 protease inhibitors for a long-acting injectable drug application</span></div><div class="casAuthors">Kesteleyn, Bart; Amssoms, Katie; Schepens, Wim; Hache, Geerwin; Verschueren, Wim; Van De Vreken, Wim; Rombauts, Klara; Meurs, Greet; Sterkens, Patrick; Stoops, Bart; Baert, Lieven; Austin, Nigel; Wegner, Jorg; Masungi, Chantal; Dierynck, Inge; Lundgren, Stina; Jonsson, Daniel; Parkes, Kevin; Kalayanov, Genadiy; Wallberg, Hans; Rosenquist, Asa; Samuelsson, Bertil; Van Emelen, Kristof; Thuring, Jan Willem</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">310-317</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The design and synthesis of novel HIV-1 protease inhibitors (PIs) (1-22), which display high potency against HIV-1 wild-type and multi-PI-resistant HIV-mutant clin. isolates, are described.  Lead optimization was initiated from compd. 1, a Phe-Phe hydroxyethylene peptidomimetic PI, and was directed towards the discovery of new PIs suitable for a long-acting (LA) injectable drug application.  Introducing a heterocyclic 6-methoxy-3-pyridinyl or a 6-(dimethylamino)-3-pyridinyl moiety (R3) at the para-position of the P1' benzyl fragment generated compds. with antiviral potency in the low single digit nanomolar range.  Halogenation or alkylation of the metabolic hot spots on the various arom. rings resulted in PIs with high stability against degrdn. in human liver microsomes and low plasma clearance in rats.  Replacing the chromanolamine moiety (R1) in the P2 protease binding site by a cyclopentanolamine or a cyclohexanolamine deriv. provided a series of high clearance PIs (16-22) with EC50s on wild-type HIV-1 in the range of 0.8-1.8 nM.  PIs 18 and 22, formulated as nanosuspensions, showed gradual but sustained and complete release from the injection site over two months in rats, and were therefore identified as interesting candidates for a LA injectable drug application for treating HIV/AIDS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojsvYYFkCohrVg90H21EOLACvtfcHk0lguZ17g0cfdOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsleitrnM&md5=a90562b30218a52c55496037245c5e06</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.10.095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.10.095%26sid%3Dliteratum%253Aachs%26aulast%3DKesteleyn%26aufirst%3DB.%26aulast%3DAmssoms%26aufirst%3DK.%26aulast%3DSchepens%26aufirst%3DW.%26aulast%3DHache%26aufirst%3DG.%26aulast%3DVerschueren%26aufirst%3DW.%26aulast%3DVan%2BDe%2BVreken%26aufirst%3DW.%26aulast%3DRombauts%26aufirst%3DK.%26aulast%3DMeurs%26aufirst%3DG.%26aulast%3DSterkens%26aufirst%3DP.%26aulast%3DStoops%26aufirst%3DB.%26aulast%3DBaert%26aufirst%3DL.%26aulast%3DAustin%26aufirst%3DN.%26aulast%3DWegner%26aufirst%3DJ.%26aulast%3DMasungi%26aufirst%3DC.%26aulast%3DDierynck%26aufirst%3DI.%26aulast%3DLundgren%26aufirst%3DS.%26aulast%3DJ%25C3%25B6nsson%26aufirst%3DD.%26aulast%3DParkes%26aufirst%3DK.%26aulast%3DKalayanov%26aufirst%3DG.%26aulast%3DWallberg%26aufirst%3DH.%26aulast%3DRosenquist%26aufirst%3DA.%26aulast%3DSamuelsson%26aufirst%3DB.%26aulast%3DVan%2BEmelen%26aufirst%3DK.%26aulast%3DThuring%26aufirst%3DJ.%2BW.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520HIV-1%2520protease%2520inhibitors%2520for%2520a%2520long-acting%2520injectable%2520drug%2520application%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D310%26epage%3D317%26doi%3D10.1016%2Fj.bmcl.2012.10.095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bungard, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiscount, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loughran, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manikowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crespo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keertikar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morriello, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamford, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddell, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diamond, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahnck-Teets, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fay, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCauley, J. A.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of piperazine sulfonamide cores leading to highly potent HIV-1 protease inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1292</span>– <span class="NLM_lpage">1297</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00386</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00386" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVSnurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1292-1297&author=C.+J.+Bungardauthor=P.+D.+Williamsauthor=J.+Schulzauthor=C.+M.+Wiscountauthor=M.+K.+Hollowayauthor=H.+M.+Loughranauthor=J.+J.+Manikowskiauthor=H.+P.+Suauthor=D.+J.+Bennettauthor=L.+Changauthor=X.+J.+Chuauthor=A.+Crespoauthor=M.+P.+Dwyerauthor=K.+Keertikarauthor=G.+J.+Morrielloauthor=A.+W.+Stamfordauthor=S.+T.+Waddellauthor=B.+Zhongauthor=B.+Huauthor=T.+Jiauthor=T.+L.+Diamondauthor=C.+Bahnck-Teetsauthor=S.+S.+Carrollauthor=J.+F.+Fayauthor=X.+Minauthor=W.+Morrisauthor=J.+E.+Ballardauthor=M.+D.+Millerauthor=J.+A.+McCauley&title=Design+and+synthesis+of+piperazine+sulfonamide+cores+leading+to+highly+potent+HIV-1+protease+inhibitors&doi=10.1021%2Facsmedchemlett.7b00386"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Piperazine Sulfonamide Cores Leading to Highly Potent HIV-1 Protease Inhibitors</span></div><div class="casAuthors">Bungard, Christopher J.; Williams, Peter D.; Schulz, Jurgen; Wiscount, Catherine M.; Holloway, M. Katharine; Loughran, H. Marie; Manikowski, Jesse J.; Su, Hua-Poo; Bennett, David J.; Chang, Lehua; Chu, Xin-Jie; Crespo, Alejandro; Dwyer, Michael P.; Keertikar, Kartik; Morriello, Gregori J.; Stamford, Andrew W.; Waddell, Sherman T.; Zhong, Bin; Hu, Bin; Ji, Tao; Diamond, Tracy L.; Bahnck-Teets, Carolyn; Carroll, Steven S.; Fay, John F.; Min, Xu; Morris, William; Ballard, Jeanine E.; Miller, Michael D.; McCauley, John A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1292-1297</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using the HIV-1 protease binding mode of MK-8718 and PL-100 as inspiration, a novel aspartate binding bicyclic piperazine sulfonamide core was designed and synthesized.  The resulting HIV-1 protease inhibitor contg. this core showed an 60-fold increase in enzyme binding affinity and a 10-fold increase in antiviral activity relative to MK-8718.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrku3wJBBdY07Vg90H21EOLACvtfcHk0lioS-tCAVdXiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVSnurvK&md5=c063d6663bd7ebede67974942cf1ea78</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00386&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00386%26sid%3Dliteratum%253Aachs%26aulast%3DBungard%26aufirst%3DC.%2BJ.%26aulast%3DWilliams%26aufirst%3DP.%2BD.%26aulast%3DSchulz%26aufirst%3DJ.%26aulast%3DWiscount%26aufirst%3DC.%2BM.%26aulast%3DHolloway%26aufirst%3DM.%2BK.%26aulast%3DLoughran%26aufirst%3DH.%2BM.%26aulast%3DManikowski%26aufirst%3DJ.%2BJ.%26aulast%3DSu%26aufirst%3DH.%2BP.%26aulast%3DBennett%26aufirst%3DD.%2BJ.%26aulast%3DChang%26aufirst%3DL.%26aulast%3DChu%26aufirst%3DX.%2BJ.%26aulast%3DCrespo%26aufirst%3DA.%26aulast%3DDwyer%26aufirst%3DM.%2BP.%26aulast%3DKeertikar%26aufirst%3DK.%26aulast%3DMorriello%26aufirst%3DG.%2BJ.%26aulast%3DStamford%26aufirst%3DA.%2BW.%26aulast%3DWaddell%26aufirst%3DS.%2BT.%26aulast%3DZhong%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DJi%26aufirst%3DT.%26aulast%3DDiamond%26aufirst%3DT.%2BL.%26aulast%3DBahnck-Teets%26aufirst%3DC.%26aulast%3DCarroll%26aufirst%3DS.%2BS.%26aulast%3DFay%26aufirst%3DJ.%2BF.%26aulast%3DMin%26aufirst%3DX.%26aulast%3DMorris%26aufirst%3DW.%26aulast%3DBallard%26aufirst%3DJ.%2BE.%26aulast%3DMiller%26aufirst%3DM.%2BD.%26aulast%3DMcCauley%26aufirst%3DJ.%2BA.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520piperazine%2520sulfonamide%2520cores%2520leading%2520to%2520highly%2520potent%2520HIV-1%2520protease%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D1292%26epage%3D1297%26doi%3D10.1021%2Facsmedchemlett.7b00386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brieger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furfine, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanlon, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazen, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaldor, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLean, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sammond, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spaltenstein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tung, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherrill, R. G.</span></span> <span> </span><span class="NLM_article-title">Ultra-potent P1 modified arylsulfonamide HIV protease inhibitors: the discovery of GW0385</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1788</span>– <span class="NLM_lpage">1794</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2006.01.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fj.bmcl.2006.01.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=16458505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvVyrtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=1788-1794&author=J.+F.+Millerauthor=C.+W.+Andrewsauthor=M.+Briegerauthor=E.+S.+Furfineauthor=M.+R.+Haleauthor=M.+H.+Hanlonauthor=R.+J.+Hazenauthor=I.+Kaldorauthor=E.+W.+McLeanauthor=D.+Reynoldsauthor=D.+M.+Sammondauthor=A.+Spaltensteinauthor=R.+Tungauthor=E.+M.+Turnerauthor=R.+X.+Xuauthor=R.+G.+Sherrill&title=Ultra-potent+P1+modified+arylsulfonamide+HIV+protease+inhibitors%3A+the+discovery+of+GW0385&doi=10.1016%2Fj.bmcl.2006.01.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Ultra-potent P1 modified arylsulfonamide HIV protease inhibitors: The discovery of GW0385</span></div><div class="casAuthors">Miller, John F.; Andrews, C. Webster; Brieger, Michael; Furfine, Eric S.; Hale, Michael R.; Hanlon, Mary H.; Hazen, Richard J.; Kaldor, Istvan; McLean, Ed W.; Reynolds, David; Sammond, Douglas M.; Spaltenstein, Andrew; Tung, Roger; Turner, Elizabeth M.; Xu, Robert X.; Sherrill, Ronald G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1788-1794</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A novel series of P1 modified HIV protease inhibitors was synthesized and evaluated for in vitro antiviral activity against wild-type virus and protease inhibitor-resistant viruses.  Optimization of the P1 moiety resulted in compds. with femtomolar enzyme activities and cellular antiviral activities in the low nanomolar range culminating in the identification of clin. candidate GW0385.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojt0KYD8JW_rVg90H21EOLACvtfcHk0lioS-tCAVdXiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhvVyrtLs%253D&md5=50f237c39e7eb19475315ab50240719e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2006.01.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2006.01.035%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DJ.%2BF.%26aulast%3DAndrews%26aufirst%3DC.%2BW.%26aulast%3DBrieger%26aufirst%3DM.%26aulast%3DFurfine%26aufirst%3DE.%2BS.%26aulast%3DHale%26aufirst%3DM.%2BR.%26aulast%3DHanlon%26aufirst%3DM.%2BH.%26aulast%3DHazen%26aufirst%3DR.%2BJ.%26aulast%3DKaldor%26aufirst%3DI.%26aulast%3DMcLean%26aufirst%3DE.%2BW.%26aulast%3DReynolds%26aufirst%3DD.%26aulast%3DSammond%26aufirst%3DD.%2BM.%26aulast%3DSpaltenstein%26aufirst%3DA.%26aulast%3DTung%26aufirst%3DR.%26aulast%3DTurner%26aufirst%3DE.%2BM.%26aulast%3DXu%26aufirst%3DR.%2BX.%26aulast%3DSherrill%26aufirst%3DR.%2BG.%26atitle%3DUltra-potent%2520P1%2520modified%2520arylsulfonamide%2520HIV%2520protease%2520inhibitors%253A%2520the%2520discovery%2520of%2520GW0385%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D1788%26epage%3D1794%26doi%3D10.1016%2Fj.bmcl.2006.01.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, G.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callebaut, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cihlar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimilli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenberg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stray, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strickley, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, C. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span> <span> </span><span class="NLM_article-title">Discovery of GS-8374, a potent human immunodeficiency virus type 1 protease inhibitor with a superior resistance profile</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">1093</span>– <span class="NLM_lpage">1098</span>, <span class="refDoi"> DOI: 10.1039/c1md00147g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1039%2Fc1md00147g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVSktr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=1093-1098&author=G.-X.+Heauthor=Z.-Y.+Yangauthor=M.+Williamsauthor=C.+Callebautauthor=T.+Cihlarauthor=B.+P.+Murrayauthor=C.+Yangauthor=M.+L.+Mitchellauthor=H.+Liuauthor=J.+Wangauthor=M.+Arimilliauthor=E.+Eisenbergauthor=K.+M.+Strayauthor=L.+K.+Tsaiauthor=M.+Hatadaauthor=X.+Chenauthor=J.+M.+Chenauthor=Y.+Wangauthor=M.+S.+Leeauthor=R.+G.+Strickleyauthor=Q.+Iwataauthor=X.+Zhengauthor=C.+U.+Kimauthor=S.+Swaminathanauthor=M.+C.+Desaiauthor=W.+A.+Leeauthor=L.+Xu&title=Discovery+of+GS-8374%2C+a+potent+human+immunodeficiency+virus+type+1+protease+inhibitor+with+a+superior+resistance+profile&doi=10.1039%2Fc1md00147g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of GS-8374, a potent human immunodeficiency virus type 1 protease inhibitor with a superior resistance profile</span></div><div class="casAuthors">He, Gong-Xing; Yang, Zheng-Yu; Williams, Matthew; Callebaut, Christian; Cihlar, Tomas; Murray, Bernard P.; Yang, Chris; Mitchell, Michael L.; Liu, Hongtao; Wang, Jianying; Arimilli, Murty; Eisenberg, Eugene; Stray, Kirsten M.; Tsai, Luong K.; Hatada, Marcos; Chen, Xiaowu; Chen, James M.; Wang, Yujin; Lee, Melody S.; Strickley, Robert G.; Iwata, Quynh; Zheng, Xubin; Kim, Choung U.; Swaminathan, Swami; Desai, Manoj C.; Lee, William A.; Xu, Lianhong</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1093-1098</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Introduction of a unique phosphonate moiety at the P1 position of the TMC-126 (3) scaffold provided a series of novel HIV-1 protease inhibitors (PIs) with an improved resistance profile against highly resistant variants.  Optimization of the linker and phosphonate moieties lead to the identification of GS-8374 (1).  Compd. 1 is a potent and orally bioavailable HIV-1 PI with a superior resistance profile.  Synthesis and characterization of 1 are reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdQ4VQyn9Rb7Vg90H21EOLACvtfcHk0ljzXNk5pTRACA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVSktr3F&md5=ea0c8e0cd95eea21571c7c1148ef9af5</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1039%2Fc1md00147g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1md00147g%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DG.-X.%26aulast%3DYang%26aufirst%3DZ.-Y.%26aulast%3DWilliams%26aufirst%3DM.%26aulast%3DCallebaut%26aufirst%3DC.%26aulast%3DCihlar%26aufirst%3DT.%26aulast%3DMurray%26aufirst%3DB.%2BP.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DMitchell%26aufirst%3DM.%2BL.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DArimilli%26aufirst%3DM.%26aulast%3DEisenberg%26aufirst%3DE.%26aulast%3DStray%26aufirst%3DK.%2BM.%26aulast%3DTsai%26aufirst%3DL.%2BK.%26aulast%3DHatada%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%2BM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DM.%2BS.%26aulast%3DStrickley%26aufirst%3DR.%2BG.%26aulast%3DIwata%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DKim%26aufirst%3DC.%2BU.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DDesai%26aufirst%3DM.%2BC.%26aulast%3DLee%26aufirst%3DW.%2BA.%26aulast%3DXu%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520GS-8374%252C%2520a%2520potent%2520human%2520immunodeficiency%2520virus%2520type%25201%2520protease%2520inhibitor%2520with%2520a%2520superior%2520resistance%2520profile%26jtitle%3DMedChemComm%26date%3D2011%26volume%3D2%26spage%3D1093%26epage%3D1098%26doi%3D10.1039%2Fc1md00147g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Windsor, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palte, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukesh, J. C.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gold, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forest, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raines, R. T.</span></span> <span> </span><span class="NLM_article-title">Sub-picomolar inhibition of HIV-1 protease with a boronic acid</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">14015</span>– <span class="NLM_lpage">14018</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b07366</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b07366" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFaltbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=14015-14018&author=I.+W.+Windsorauthor=M.+J.+Palteauthor=J.+C.+Lukeshauthor=B.+Goldauthor=K.+T.+Forestauthor=R.+T.+Raines&title=Sub-picomolar+inhibition+of+HIV-1+protease+with+a+boronic+acid&doi=10.1021%2Fjacs.8b07366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Sub-picomolar Inhibition of HIV-1 Protease with a Boronic Acid</span></div><div class="casAuthors">Windsor, Ian W.; Palte, Michael J.; Lukesh, John C.; Gold, Brian; Forest, Katrina T.; Raines, Ronald T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">14015-14018</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Boronic acids have been typecast as moieties for covalent complexation and are employed only rarely as agents for non-covalent recognition.  By exploiting the profuse ability of a boronic acid group to form hydrogen bonds, we have developed an inhibitor of HIV-1 protease with extraordinary affinity.  Specifically, we find that replacing an aniline moiety in darunavir with a phenylboronic acid leads to 20-fold greater affinity for the protease.  X-ray crystallog. demonstrates that the boronic acid group participates in three hydrogen bonds, more than the amino group of darunavir or any other analog.  Importantly, the boronic acid maintains its hydrogen bonds and its affinity for the drug-resistant D30N variant of HIV-1 protease.  The BOH···OC hydrogen bonds between the boronic acid hydroxy group and Asp30 (or Asn30) of the protease are short (rO···O = 2.2 Å), and d. functional theory anal. reveals a high degree of covalency.  These data highlight the utility of boronic acids as versatile functional groups in the design of small-mol. ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8UYVpx53VSrVg90H21EOLACvtfcHk0ljzXNk5pTRACA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFaltbnI&md5=ad7d7a58f8e3cad8c6fea7cb2823741c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b07366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b07366%26sid%3Dliteratum%253Aachs%26aulast%3DWindsor%26aufirst%3DI.%2BW.%26aulast%3DPalte%26aufirst%3DM.%2BJ.%26aulast%3DLukesh%26aufirst%3DJ.%2BC.%26aulast%3DGold%26aufirst%3DB.%26aulast%3DForest%26aufirst%3DK.%2BT.%26aulast%3DRaines%26aufirst%3DR.%2BT.%26atitle%3DSub-picomolar%2520inhibition%2520of%2520HIV-1%2520protease%2520with%2520a%2520boronic%2520acid%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D14015%26epage%3D14018%26doi%3D10.1021%2Fjacs.8b07366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nalam, M. N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, G. S. K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anjum, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yilmaz, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tidor, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rana, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Substrate envelope-designed potent HIV-1 protease inhibitors to avoid drug resistance</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1116</span>– <span class="NLM_lpage">1124</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2013.07.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fj.chembiol.2013.07.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=24012370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVWis7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=1116-1124&author=M.+N.+L.+Nalamauthor=A.+Aliauthor=G.+S.+K.+K.+Reddyauthor=H.+Caoauthor=S.+G.+Anjumauthor=M.+D.+Altmanauthor=N.+K.+Yilmazauthor=B.+Tidorauthor=T.+M.+Ranaauthor=C.+A.+Schiffer&title=Substrate+envelope-designed+potent+HIV-1+protease+inhibitors+to+avoid+drug+resistance&doi=10.1016%2Fj.chembiol.2013.07.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate Envelope-Designed Potent HIV-1 Protease Inhibitors to Avoid Drug Resistance</span></div><div class="casAuthors">Nalam, Madhavi N. L.; Ali, Akbar; Reddy, G. S. Kiran Kumar; Cao, Hong; Anjum, Saima G.; Altman, Michael D.; Yilmaz, Nese Kurt; Tidor, Bruce; Rana, Tariq M.; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1116-1124</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The rapid evolution of HIV under selective drug pressure has led to multidrug resistant (MDR) strains that evade std. therapies.  We designed highly potent HIV-1 protease inhibitors (PIs) using the substrate envelope model, which confines inhibitors within the consensus vol. of natural substrates, providing inhibitors less susceptible to resistance because a mutation affecting such inhibitors will simultaneously affect viral substrate processing.  The designed PIs share a common chem. scaffold but utilize various moieties that optimally fill the substrate envelope, as confirmed by crystal structures.  The designed PIs retain robust binding to MDR protease variants and display exceptional antiviral potencies against different clades of HIV as well as a panel of 12 drug-resistant viral strains.  The substrate envelope model proves to be a powerful strategy to develop potent and robust inhibitors that avoid drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm5ioan7rs5LVg90H21EOLACvtfcHk0lidRWXkq7jfZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVWis7zI&md5=2dc40eee276c4627593ee5857e2a7670</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2013.07.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2013.07.014%26sid%3Dliteratum%253Aachs%26aulast%3DNalam%26aufirst%3DM.%2BN.%2BL.%26aulast%3DAli%26aufirst%3DA.%26aulast%3DReddy%26aufirst%3DG.%2BS.%2BK.%2BK.%26aulast%3DCao%26aufirst%3DH.%26aulast%3DAnjum%26aufirst%3DS.%2BG.%26aulast%3DAltman%26aufirst%3DM.%2BD.%26aulast%3DYilmaz%26aufirst%3DN.%2BK.%26aulast%3DTidor%26aufirst%3DB.%26aulast%3DRana%26aufirst%3DT.%2BM.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DSubstrate%2520envelope-designed%2520potent%2520HIV-1%2520protease%2520inhibitors%2520to%2520avoid%2520drug%2520resistance%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D1116%26epage%3D1124%26doi%3D10.1016%2Fj.chembiol.2013.07.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higashi-Kuwata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulut, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoki-Ogata, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takamatsu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yedidi, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishida, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takamune, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyalapatla, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osswald, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jono, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarchoan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misumi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuya, H.</span></span> <span> </span><span class="NLM_article-title">A novel central nervous system-penetrating protease inhibitor overcomes human immunodeficiency virus 1 resistance with unprecedented aM to pM potency</span>. <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e28020</span> <span class="refDoi"> DOI: 10.7554/eLife.28020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.7554%2FeLife.28020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=29039736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlKkt7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&author=M.+Aokiauthor=H.+Hayashiauthor=K.+V.+Raoauthor=D.+Dasauthor=N.+Higashi-Kuwataauthor=H.+Bulutauthor=H.+Aoki-Ogataauthor=Y.+Takamatsuauthor=R.+S.+Yedidiauthor=D.+A.+Davisauthor=S.+I.+Hattoriauthor=N.+Nishidaauthor=K.+Hasegawaauthor=N.+Takamuneauthor=P.+R.+Nyalapatlaauthor=H.+L.+Osswaldauthor=H.+Jonoauthor=H.+Saitoauthor=R.+Yarchoanauthor=S.+Misumiauthor=A.+K.+Ghoshauthor=H.+Mitsuya&title=A+novel+central+nervous+system-penetrating+protease+inhibitor+overcomes+human+immunodeficiency+virus+1+resistance+with+unprecedented+aM+to+pM+potency&doi=10.7554%2FeLife.28020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A novel central nervous system-penetrating protease inhibitor overcomes human immunodeficiency virus 1 resistance with unprecedented aM to pM potency</span></div><div class="casAuthors">Aoki, Manabu; Hayashi, Hironori; Rao, Kalapala Venkateswara; Das, Debananda; Higashi-Kuwata, Nobuyo; Bulut, Haydar; Aoki-Ogata, Hiromi; Takamatsu, Yuki; Yedidi, Ravikiran S.; Davis, David A.; Hattori, Shin-ichiro; Nishida, Noriko; Hasegawa, Kazuya; Takamune, Nobutoki; Nyalapatla, Prasanth R.; Osswald, Heather L.; Jono, Hirofumi; Saito, Hideyuki; Yarchoan, Robert; Misumi, Shogo; Ghosh, Arun K.; Mitsuya, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e28020/1-e28020/25</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Antiretroviral therapy for HIV-1 infection/AIDS has significantly extended the life expectancy of HIV-1-infected individuals and reduced HIV-1 transmission at very high rates.  However, certain individuals who initially achieve viral suppression to undetectable levels may eventually suffer treatment failure mainly due to adverse effects and the emergence of drug-resistant HIV-1 variants.  Here, we report GRL-142, a novel HIV-1 protease inhibitor contg. an unprecedented 6-5-5-ring-fused crown-like tetrahydropyranofuran, which has extremely potent activity against all HIV-1 strains examd. with IC50 values of attomolar-to-picomolar concns., virtually no effects on cellular growth, extremely high genetic barrier against the emergence of drug-resistant variants, and favorable intracellular and central nervous system penetration.  GRL-142 forms optimum polar, van der Waals, and halogen bond interactions with HIV-1 protease and strongly blocks protease dimerization, demonstrating that combined multiple optimizing elements significantly enhance mol. and at. interactions with a target protein and generate unprecedentedly potent and practically favorable agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfcZXKE5Ic5rVg90H21EOLACvtfcHk0lidRWXkq7jfZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlKkt7vE&md5=2c90963399dbc034de9e800d17a12ee1</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.7554%2FeLife.28020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.28020%26sid%3Dliteratum%253Aachs%26aulast%3DAoki%26aufirst%3DM.%26aulast%3DHayashi%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DK.%2BV.%26aulast%3DDas%26aufirst%3DD.%26aulast%3DHigashi-Kuwata%26aufirst%3DN.%26aulast%3DBulut%26aufirst%3DH.%26aulast%3DAoki-Ogata%26aufirst%3DH.%26aulast%3DTakamatsu%26aufirst%3DY.%26aulast%3DYedidi%26aufirst%3DR.%2BS.%26aulast%3DDavis%26aufirst%3DD.%2BA.%26aulast%3DHattori%26aufirst%3DS.%2BI.%26aulast%3DNishida%26aufirst%3DN.%26aulast%3DHasegawa%26aufirst%3DK.%26aulast%3DTakamune%26aufirst%3DN.%26aulast%3DNyalapatla%26aufirst%3DP.%2BR.%26aulast%3DOsswald%26aufirst%3DH.%2BL.%26aulast%3DJono%26aufirst%3DH.%26aulast%3DSaito%26aufirst%3DH.%26aulast%3DYarchoan%26aufirst%3DR.%26aulast%3DMisumi%26aufirst%3DS.%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DMitsuya%26aufirst%3DH.%26atitle%3DA%2520novel%2520central%2520nervous%2520system-penetrating%2520protease%2520inhibitor%2520overcomes%2520human%2520immunodeficiency%2520virus%25201%2520resistance%2520with%2520unprecedented%2520aM%2520to%2520pM%2520potency%26jtitle%3DElife%26date%3D2017%26volume%3D6%26doi%3D10.7554%2FeLife.28020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyalapatla, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovela, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brindisi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osswald, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agniswamy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuya, H.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of highly potent HIV-1 protease inhibitors containing tricyclic fused ring systems as novel P2 ligands: structure-activity studies, biological and X-ray structural analysis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4561</span>– <span class="NLM_lpage">4577</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00298</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00298" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpsVahtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4561-4577&author=A.+K.+Ghoshauthor=P.+R.+Nyalapatlaauthor=S.+Kovelaauthor=K.+V.+Raoauthor=M.+Brindisiauthor=H.+L.+Osswaldauthor=M.+Amanoauthor=M.+Aokiauthor=J.+Agniswamyauthor=Y.-F.+Wangauthor=I.+T.+Weberauthor=H.+Mitsuya&title=Design+and+synthesis+of+highly+potent+HIV-1+protease+inhibitors+containing+tricyclic+fused+ring+systems+as+novel+P2+ligands%3A+structure-activity+studies%2C+biological+and+X-ray+structural+analysis&doi=10.1021%2Facs.jmedchem.8b00298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Highly Potent HIV-1 Protease Inhibitors Containing Tricyclic Fused Ring Systems as Novel P2 Ligands: Structure-Activity Studies, Biological and X-ray Structural Analysis</span></div><div class="casAuthors">Ghosh, Arun K.; R. Nyalapatla, Prasanth; Kovela, Satish; Rao, Kalapala Venkateswara; Brindisi, Margherita; Osswald, Heather L.; Amano, Masayuki; Aoki, Manabu; Agniswamy, Johnson; Wang, Yuan-Fang; Weber, Irene T.; Mitsuya, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4561-4577</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design, synthesis, and biol. evaluation of a new class of HIV-1 protease inhibitors contg. stereochem. defined fused tricyclic polyethers as the P2 ligands and a variety of sulfonamide derivs. as the P2' ligands are described.  A no. of ring sizes and various substituent effects were investigated to enhance the ligand-backbone interactions in the protease active site.  Inhibitors I and II contg. this unprecedented fused 6-5-5 ring system as the P2 ligand, an aminobenzothiazole as the P2' ligand, and a difluorophenylmethyl as the P1 ligand exhibited exceptional enzyme inhibitory potency and maintained excellent antiviral activity against a panel of highly multidrug-resistant HIV-1 variants.  The umbrella-like P2 ligand for these inhibitors has been synthesized efficiently in an optically active form using a Pauson-Khand cyclization reaction as the key step.  The racemic alcs. were resolved efficiently using a lipase catalyzed enzymic resoln.  Two high resoln. X-ray structures of inhibitor-bound HIV-1 protease revealed extensive interactions with the backbone atoms of HIV-1 protease and provided mol. insight into the binding properties of these new inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobuOIwTJfrX7Vg90H21EOLACvtfcHk0liTC0Rv6I1Hyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpsVahtb4%253D&md5=55ddb084c94a598748cb922796cffa22</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00298%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DNyalapatla%26aufirst%3DP.%2BR.%26aulast%3DKovela%26aufirst%3DS.%26aulast%3DRao%26aufirst%3DK.%2BV.%26aulast%3DBrindisi%26aufirst%3DM.%26aulast%3DOsswald%26aufirst%3DH.%2BL.%26aulast%3DAmano%26aufirst%3DM.%26aulast%3DAoki%26aufirst%3DM.%26aulast%3DAgniswamy%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.-F.%26aulast%3DWeber%26aufirst%3DI.%2BT.%26aulast%3DMitsuya%26aufirst%3DH.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520highly%2520potent%2520HIV-1%2520protease%2520inhibitors%2520containing%2520tricyclic%2520fused%2520ring%2520systems%2520as%2520novel%2520P2%2520ligands%253A%2520structure-activity%2520studies%252C%2520biological%2520and%2520X-ray%2520structural%2520analysis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4561%26epage%3D4577%26doi%3D10.1021%2Facs.jmedchem.8b00298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyalapatla, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osswald, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martyr, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agniswamy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulut, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuya, H.</span></span> <span> </span><span class="NLM_article-title">Design and development of highly potent HIV-1 protease inhibitors with a crown-like oxotricyclic core as the P2-ligand to combat multidrug-resistant HIV variants</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4267</span>– <span class="NLM_lpage">4278</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00172</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00172" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFKlsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4267-4278&author=A.+K.+Ghoshauthor=K.+V.+Raoauthor=P.+R.+Nyalapatlaauthor=H.+L.+Osswaldauthor=C.+D.+Martyrauthor=M.+Aokiauthor=H.+Hayashiauthor=J.+Agniswamyauthor=Y.+F.+Wangauthor=H.+Bulutauthor=D.+Dasauthor=I.+T.+Weberauthor=H.+Mitsuya&title=Design+and+development+of+highly+potent+HIV-1+protease+inhibitors+with+a+crown-like+oxotricyclic+core+as+the+P2-ligand+to+combat+multidrug-resistant+HIV+variants&doi=10.1021%2Facs.jmedchem.7b00172"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Development of Highly Potent HIV-1 Protease Inhibitors with a Crown-Like Oxotricyclic Core as the P2-Ligand To Combat Multidrug-Resistant HIV Variants</span></div><div class="casAuthors">Ghosh, Arun K.; Rao, Kalapala Venkateswara; Nyalapatla, Prasanth R.; Osswald, Heather L.; Martyr, Cuthbert D.; Aoki, Manabu; Hayashi, Hironori; Agniswamy, Johnson; Wang, Yuan-Fang; Bulut, Haydar; Das, Debananda; Weber, Irene T.; Mitsuya, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4267-4278</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Design, synthesis, and evaluation of a new class of exceptionally potent HIV-1 protease inhibitors is reported.  Inhibitor I displayed superior antiviral activity and drug-resistance profiles.  In fact, this inhibitor showed several orders of magnitude improved antiviral activity over the FDA approved drug, darunavir.  This inhibitor incorporates an unprecedented 6-5-5 ring-fused crown-like tetrahydropyranofuran as the P2 ligand and an aminobenzothiazole as the P2' ligand with the (R)-hydroxyethylsulfonamide isostere.  The crown-like P2 ligand for this inhibitor has been synthesized efficiently in an optically active form using a chiral Diels-Alder catalyst providing a key intermediate in high enantiomeric purity.  Two high resoln. X-ray structures of inhibitor-bound HIV-1 protease revealed extensive interactions with the backbone atoms of HIV-1 protease and provided mol. insight into the binding properties of these new inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpA8W8TXM4SXbVg90H21EOLACvtfcHk0liTC0Rv6I1Hyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFKlsLw%253D&md5=03b84db839ddd5d19195e5dd8bf43642</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00172&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00172%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DRao%26aufirst%3DK.%2BV.%26aulast%3DNyalapatla%26aufirst%3DP.%2BR.%26aulast%3DOsswald%26aufirst%3DH.%2BL.%26aulast%3DMartyr%26aufirst%3DC.%2BD.%26aulast%3DAoki%26aufirst%3DM.%26aulast%3DHayashi%26aufirst%3DH.%26aulast%3DAgniswamy%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%2BF.%26aulast%3DBulut%26aufirst%3DH.%26aulast%3DDas%26aufirst%3DD.%26aulast%3DWeber%26aufirst%3DI.%2BT.%26aulast%3DMitsuya%26aufirst%3DH.%26atitle%3DDesign%2520and%2520development%2520of%2520highly%2520potent%2520HIV-1%2520protease%2520inhibitors%2520with%2520a%2520crown-like%2520oxotricyclic%2520core%2520as%2520the%2520P2-ligand%2520to%2520combat%2520multidrug-resistant%2520HIV%2520variants%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4267%26epage%3D4278%26doi%3D10.1021%2Facs.jmedchem.7b00172" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapsal, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldridge, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffey, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuya, H.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of potent HIV-1 protease inhibitors incorporating hexahydrofuropyranol-derived high affinity P<sub>2</sub> ligands: structure-activity studies and biological evaluation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">622</span>– <span class="NLM_lpage">634</span>, <span class="refDoi"> DOI: 10.1021/jm1012787</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012787" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslSq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=622-634&author=A.+K.+Ghoshauthor=B.+D.+Chapsalauthor=A.+Baldridgeauthor=M.+P.+Steffeyauthor=D.+E.+Waltersauthor=Y.+Kohauthor=M.+Amanoauthor=H.+Mitsuya&title=Design+and+synthesis+of+potent+HIV-1+protease+inhibitors+incorporating+hexahydrofuropyranol-derived+high+affinity+P2+ligands%3A+structure-activity+studies+and+biological+evaluation&doi=10.1021%2Fjm1012787"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Potent HIV-1 Protease Inhibitors Incorporating Hexahydrofuropyranol-Derived High Affinity P2 Ligands: Structure-Activity Studies and Biological Evaluation</span></div><div class="casAuthors">Ghosh, Arun K.; Chapsal, Bruno D.; Baldridge, Abigail; Steffey, Melinda P.; Walters, D. Eric; Koh, Yasuhiro; Amano, Masayuki; Mitsuya, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">622-634</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Substituted furopyranol, benzofuranol, cyclopentapyranol, and indanol arylsulfonyl phenylpropylcarbamate HIV-1 protease inhibitors such as I are prepd. as HIV-1 protease inhibitors.  Hexahydrofuranopyranol II is prepd. in seven steps from nonracemic cyclopentenofuranone III; conversion to a 4-nitrophenyl carbamate and coupling with an arylsulfonylphenylpropylamine yields I.  I inhibits multidrug resistant HIV-1 strains with comparable effectiveness to the HIV-1 inhibitor darunavir.  A model of I bound to the active site of HIV-1 protease is calcd. based on the crystal structure of a related protease inhibitor bound to HIV-1 protease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrY4F-41rD7jbVg90H21EOLACvtfcHk0liTC0Rv6I1Hyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslSq&md5=511b20b9bb4c1d31aafc8e969eea9328</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm1012787&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012787%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DChapsal%26aufirst%3DB.%2BD.%26aulast%3DBaldridge%26aufirst%3DA.%26aulast%3DSteffey%26aufirst%3DM.%2BP.%26aulast%3DWalters%26aufirst%3DD.%2BE.%26aulast%3DKoh%26aufirst%3DY.%26aulast%3DAmano%26aufirst%3DM.%26aulast%3DMitsuya%26aufirst%3DH.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520potent%2520HIV-1%2520protease%2520inhibitors%2520incorporating%2520hexahydrofuropyranol-derived%2520high%2520affinity%2520P2%2520ligands%253A%2520structure-activity%2520studies%2520and%2520biological%2520evaluation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D622%26epage%3D634%26doi%3D10.1021%2Fjm1012787" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">King, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prabu-Jeyabalan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nalivaika, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Combating susceptibility to drug resistance: lessons from HIV-1 protease</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1333</span>– <span class="NLM_lpage">1338</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2004.08.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fj.chembiol.2004.08.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=15489160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2cXos1Ciu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=1333-1338&author=N.+M.+Kingauthor=M.+Prabu-Jeyabalanauthor=E.+A.+Nalivaikaauthor=C.+A.+Schiffer&title=Combating+susceptibility+to+drug+resistance%3A+lessons+from+HIV-1+protease&doi=10.1016%2Fj.chembiol.2004.08.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Combating Susceptibility to Drug Resistance Lessons from HIV-1 Protease</span></div><div class="casAuthors">King, Nancy M.; Prabu-Jeyabalan, Moses; Nalivaika, Ellen A.; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1333-1338</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Drug resistance is a major obstacle in modern medicine.  However, resistance is rarely considered in drug development and may inadvertently be facilitated, as many designed inhibitors contact residues that can mutate to confer resistance, without significantly impairing function.  Contemporary drug design often ignores the detailed at. basis for function and primarily focuses on disrupting the target's activity, which is necessary but not sufficient for developing a robust drug.  In this study, we examine the impact of drug-resistant mutations in HIV-1 protease on substrate recognition and demonstrate that most primary active site mutations do not extensively contact substrates, but are crit. to inhibitor binding.  We propose a general, structure-based strategy to reduce the probability of drug resistance by designing inhibitors that interact only with those residues that are essential for function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbsk2AyTcCmbVg90H21EOLACvtfcHk0li8RTW-DK9iZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXos1Ciu78%253D&md5=701599b15cf9ecc2c026ad4eac650a90</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2004.08.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2004.08.010%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DN.%2BM.%26aulast%3DPrabu-Jeyabalan%26aufirst%3DM.%26aulast%3DNalivaika%26aufirst%3DE.%2BA.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DCombating%2520susceptibility%2520to%2520drug%2520resistance%253A%2520lessons%2520from%2520HIV-1%2520protease%26jtitle%3DChem.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D1333%26epage%3D1338%26doi%3D10.1016%2Fj.chembiol.2004.08.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerlits, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wymore, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parks, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blakeley, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovalevsky, A.</span></span> <span> </span><span class="NLM_article-title">Long-range electrostatics-induced two-proton transfer captured by neutron crystallography in an enzyme catalytic site</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">4924</span>– <span class="NLM_lpage">4927</span>, <span class="refDoi"> DOI: 10.1002/anie.201509989</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1002%2Fanie.201509989" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC28XktVaju70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=4924-4927&author=O.+Gerlitsauthor=T.+Wymoreauthor=A.+Dasauthor=C.+H.+Shenauthor=J.+M.+Parksauthor=J.+C.+Smithauthor=K.+L.+Weissauthor=D.+A.+Keenauthor=M.+P.+Blakeleyauthor=J.+M.+Louisauthor=P.+Langanauthor=I.+T.+Weberauthor=A.+Kovalevsky&title=Long-range+electrostatics-induced+two-proton+transfer+captured+by+neutron+crystallography+in+an+enzyme+catalytic+site&doi=10.1002%2Fanie.201509989"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Long-range electrostatics-induced two-proton transfer captured by neutron crystallography in an enzyme catalytic site</span></div><div class="casAuthors">Gerlits, Oksana; Wymore, Troy; Das, Amit; Shen, Chen-Hsiang; Parks, Jerry M.; Smith, Jeremy C.; Weiss, Kevin L.; Keen, David A.; Blakeley, Matthew P.; Louis, John M.; Langan, Paul; Weber, Irene T.; Kovalevsky, Andrey</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4924-4927</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Neutron crystallog. was used to directly locate 2 protons before and after a pH-induced 2-proton transfer between catalytic Asp residues and the OH group of the bound clin. drug, darunavir, located in the catalytic site of enzyme HIV-1 protease.  The 2-proton transfer was triggered by electrostatic effects arising from protonation state changes of surface residues far from the active site.  The mechanism and pH effect were supported by quantum mechanics/mol. mechanics (QM/MM) calcns.  The low-pH proton configuration in the catalytic site was deemed crit. for the catalytic action of this enzyme and may apply more generally to other aspartic proteases.  Neutrons therefore represent a superb probe to obtain structural details for proton transfer reactions in biol. systems at a truly at. level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzmyxWbnMIhLVg90H21EOLACvtfcHk0li8RTW-DK9iZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktVaju70%253D&md5=198ac991fe2274b9f140e13b7e7aae35</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fanie.201509989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201509989%26sid%3Dliteratum%253Aachs%26aulast%3DGerlits%26aufirst%3DO.%26aulast%3DWymore%26aufirst%3DT.%26aulast%3DDas%26aufirst%3DA.%26aulast%3DShen%26aufirst%3DC.%2BH.%26aulast%3DParks%26aufirst%3DJ.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BC.%26aulast%3DWeiss%26aufirst%3DK.%2BL.%26aulast%3DKeen%26aufirst%3DD.%2BA.%26aulast%3DBlakeley%26aufirst%3DM.%2BP.%26aulast%3DLouis%26aufirst%3DJ.%2BM.%26aulast%3DLangan%26aufirst%3DP.%26aulast%3DWeber%26aufirst%3DI.%2BT.%26aulast%3DKovalevsky%26aufirst%3DA.%26atitle%3DLong-range%2520electrostatics-induced%2520two-proton%2520transfer%2520captured%2520by%2520neutron%2520crystallography%2520in%2520an%2520enzyme%2520catalytic%2520site%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D4924%26epage%3D4927%26doi%3D10.1002%2Fanie.201509989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kovalevsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlits, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beltran, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blakeley, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, I. T.</span></span> <span> </span><span class="NLM_article-title">Proton transfer and drug binding details revealed in neutron diffraction studies of wild-type and drug resistant HIV-1 protease</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>634</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">279</span>, <span class="refDoi"> DOI: 10.1016/bs.mie.2019.12.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fbs.mie.2019.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=32093836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A280%3ADC%252BB387ls1KitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=634&publication_year=2020&pages=257-279&author=A.+Kovalevskyauthor=O.+Gerlitsauthor=K.+Beltranauthor=K.+L.+Weissauthor=D.+A.+Keenauthor=M.+P.+Blakeleyauthor=J.+M.+Louisauthor=I.+T.+Weber&title=Proton+transfer+and+drug+binding+details+revealed+in+neutron+diffraction+studies+of+wild-type+and+drug+resistant+HIV-1+protease&doi=10.1016%2Fbs.mie.2019.12.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Proton transfer and drug binding details revealed in neutron diffraction studies of wild-type and drug resistant HIV-1 protease</span></div><div class="casAuthors">Kovalevsky Andrey; Gerlits Oksana; Beltran Kaira; Weiss Kevin L; Keen David A; Blakeley Matthew P; Louis John M; Weber Irene T</div><div class="citationInfo"><span class="NLM_cas:title">Methods in enzymology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">634</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">257-279</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">HIV-1 protease is an essential therapeutic target for the design and development of antiviral inhibitors to treat AIDS.  We used room temperature neutron crystallography to accurately determine hydrogen atom positions in several protease complexes with clinical drugs, amprenavir and darunavir.  Hydrogen bonding interactions were carefully mapped to provide an unprecedented picture of drug binding to the protease target.  We demonstrate that hydrogen atom positions within the enzyme catalytic site can be altered by introducing drug resistant mutations and by protonating surface residues that trigger proton transfer reactions between the catalytic Asp residues and the hydroxyl group of darunavir.  When protein perdeuteration is not feasible, we validate the use of initial H/D exchange with unfolded protein and partial deuteration in pure D2O with hydrogenous glycerol to maximize deuterium incorporation into the protein, with no detrimental effects on the growth of quality crystals suitable for neutron diffraction experiments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQM8q3kfnWTrftos8R3JUhXfW6udTcc2eYyQyGKCovjHLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB387ls1KitQ%253D%253D&md5=859de4c89f45cbaa73b1120155c96241</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fbs.mie.2019.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.mie.2019.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DKovalevsky%26aufirst%3DA.%26aulast%3DGerlits%26aufirst%3DO.%26aulast%3DBeltran%26aufirst%3DK.%26aulast%3DWeiss%26aufirst%3DK.%2BL.%26aulast%3DKeen%26aufirst%3DD.%2BA.%26aulast%3DBlakeley%26aufirst%3DM.%2BP.%26aulast%3DLouis%26aufirst%3DJ.%2BM.%26aulast%3DWeber%26aufirst%3DI.%2BT.%26atitle%3DProton%2520transfer%2520and%2520drug%2520binding%2520details%2520revealed%2520in%2520neutron%2520diffraction%2520studies%2520of%2520wild-type%2520and%2520drug%2520resistant%2520HIV-1%2520protease%26jtitle%3DMethods%2520Enzymol.%26date%3D2020%26volume%3D634%26spage%3D257%26epage%3D279%26doi%3D10.1016%2Fbs.mie.2019.12.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempf, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sham, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasavanonda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wideburg, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saldivar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norbeck, D. W.</span></span> <span> </span><span class="NLM_article-title">Synthesis and SAR studies of potent HIV protease inhibitors containing novel dimethylphenoxyl acetates as P2 ligands</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">3657</span>– <span class="NLM_lpage">3660</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2003.08.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fj.bmcl.2003.08.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=14552751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnvFGjtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=3657-3660&author=X.+Chenauthor=D.+J.+Kempfauthor=L.+Liauthor=H.+L.+Shamauthor=S.+Vasavanondaauthor=N.+E.+Wideburgauthor=A.+Saldivarauthor=K.+C.+Marshauthor=E.+McDonaldauthor=D.+W.+Norbeck&title=Synthesis+and+SAR+studies+of+potent+HIV+protease+inhibitors+containing+novel+dimethylphenoxyl+acetates+as+P2+ligands&doi=10.1016%2Fj.bmcl.2003.08.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and SAR studies of potent HIV protease inhibitors containing novel dimethylphenoxyl acetates as P2 ligands</span></div><div class="casAuthors">Chen, Xiaoqi; Kempf, Dale J.; Li, Lin; Sham, Hing L.; Vasavanonda, Sudthida; Wideburg, Norman E.; Saldivar, Ayda; Marsh, Kennan C.; McDonald, Edith; Norbeck, Daniel W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3657-3660</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">Iso-Pr substituted 4-thioazolyl valine side chains are highly optimized P2-P3 ligands for C2 symmetry-based HIV protease inhibitors, as exemplified by the drug ritonavir.  Replacement of the side chain with the conformationally constrained hexahydrofurofuranyloxy P2 ligand in combination with a dimethylphenoxyacetate on the other end of the ritonavir core diamine yielded highly potent HIV protease inhibitors.  The in vitro antiviral activity in MT4 cells increased by 10- and 20-fold, resp., in the absence and presence of 50% human serum compared to ritonavir.  The structure-activity relationships of inhibitor series with this combination of ligands were investigated.  Preliminary pharmacokinetic studies in rats indicated rapid elimination of the inhibitors from the blood, and the plasma levels were not significantly enhanced by coadministration with ritonavir.  However, the novel structural features and the high intrinsic antiviral potency of this series provides potential for the future exploration of prodrug strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2XII92gr9NbVg90H21EOLACvtfcHk0lha2qFfDSCpWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnvFGjtb0%253D&md5=f72b5c606df040470e2796bb850edf4d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2003.08.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2003.08.043%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DKempf%26aufirst%3DD.%2BJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DSham%26aufirst%3DH.%2BL.%26aulast%3DVasavanonda%26aufirst%3DS.%26aulast%3DWideburg%26aufirst%3DN.%2BE.%26aulast%3DSaldivar%26aufirst%3DA.%26aulast%3DMarsh%26aufirst%3DK.%2BC.%26aulast%3DMcDonald%26aufirst%3DE.%26aulast%3DNorbeck%26aufirst%3DD.%2BW.%26atitle%3DSynthesis%2520and%2520SAR%2520studies%2520of%2520potent%2520HIV%2520protease%2520inhibitors%2520containing%2520novel%2520dimethylphenoxyl%2520acetates%2520as%2520P2%2520ligands%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2003%26volume%3D13%26spage%3D3657%26epage%3D3660%26doi%3D10.1016%2Fj.bmcl.2003.08.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempf, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sham, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wideburg, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saldivar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasavanonda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norbeck, D. W.</span></span> <span> </span><span class="NLM_article-title">Evaluation of furofuran as a P2 ligand for symmetry-based HIV protease inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">2847</span>– <span class="NLM_lpage">2852</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(96)00528-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2FS0960-894X%2896%2900528-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADyaK2sXhtlagtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1996&pages=2847-2852&author=X.+Chenauthor=L.+Liauthor=D.+J.+Kempfauthor=H.+Shamauthor=N.+E.+Wideburgauthor=A.+Saldivarauthor=S.+Vasavanondaauthor=K.+C.+Marshauthor=E.+McDonaldauthor=D.+W.+Norbeck&title=Evaluation+of+furofuran+as+a+P2+ligand+for+symmetry-based+HIV+protease+inhibitors&doi=10.1016%2FS0960-894X%2896%2900528-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of furofuran as a P2 ligand for symmetry-based HIV protease inhibitors</span></div><div class="casAuthors">Chen, Xiaoqi; Li, Lin; Kempf, Dale J.; Sham, Hing; Wideburg, Norman E.; Saldivar, Ayda; Vasavanonda, Sudthida; Marsh, Kennan C.; McDonald, Edith; Norbeck, Daniel W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2847-2852</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The hexahydrofurofuranyloxy group was evaluated as a conformationally constrained P2 ligand for symmetry-based HIV protease inhibitors.  A no. of compds. showed nM level activity against HIV in MT4 cells and lower protein binding than the licensed protease inhibitor ritonavir.  However, replacement of 5-thiazole of ritonavir with a furofuran caused a redn. of the bioavailability in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo80kdyUn-rDLVg90H21EOLACvtfcHk0lha2qFfDSCpWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhtlagtA%253D%253D&md5=a577546f996c402549d6537d3e4db3a1</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2896%2900528-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252896%252900528-8%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DKempf%26aufirst%3DD.%2BJ.%26aulast%3DSham%26aufirst%3DH.%26aulast%3DWideburg%26aufirst%3DN.%2BE.%26aulast%3DSaldivar%26aufirst%3DA.%26aulast%3DVasavanonda%26aufirst%3DS.%26aulast%3DMarsh%26aufirst%3DK.%2BC.%26aulast%3DMcDonald%26aufirst%3DE.%26aulast%3DNorbeck%26aufirst%3DD.%2BW.%26atitle%3DEvaluation%2520of%2520furofuran%2520as%2520a%2520P2%2520ligand%2520for%2520symmetry-based%2520HIV%2520protease%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1996%26volume%3D6%26spage%3D2847%26epage%3D2852%26doi%3D10.1016%2FS0960-894X%2896%2900528-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cannizzaro, C. E.</span>; <span class="NLM_string-name">Yang, Z.-Y.</span>; <span class="NLM_string-name">Desai, M. C.</span>; <span class="NLM_string-name">Mitchell, M. L.</span>; <span class="NLM_string-name">Xu, L.</span>; <span class="NLM_string-name">Swaminathan, S.</span>; <span class="NLM_string-name">Chen, J. M.</span></span> <span> </span><span class="NLM_article-title">Antiviral protease inhibitors</span>. PCT Int. App. <span class="NLM_patent">WO2008011117</span>, <span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=C.+E.+Cannizzaro&author=Z.-Y.+Yang&author=M.+C.+Desai&author=M.+L.+Mitchell&author=L.+Xu&author=S.+Swaminathan&author=J.+M.+Chen&title=Antiviral+protease+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCannizzaro%26aufirst%3DC.%2BE.%26atitle%3DAntiviral%2520protease%2520inhibitors%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Özen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haliloǧlu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Dynamics of preferential substrate recognition in HIV-1 protease: redefining the substrate envelope</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>410</i></span>,  <span class="NLM_fpage">726</span>– <span class="NLM_lpage">744</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2011.03.053</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fj.jmb.2011.03.053" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=21762811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovF2mtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=410&publication_year=2011&pages=726-744&author=A.+%C3%96zenauthor=T.+Halilo%C7%A7luauthor=C.+A.+Schiffer&title=Dynamics+of+preferential+substrate+recognition+in+HIV-1+protease%3A+redefining+the+substrate+envelope&doi=10.1016%2Fj.jmb.2011.03.053"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamics of Preferential Substrate Recognition in HIV-1 Protease: Redefining the Substrate Envelope</span></div><div class="casAuthors">Ozen, Aysegul; Haliloglu, Turkan; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">410</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">726-744</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Human immunodeficiency virus type 1 (HIV-1) protease (PR) permits viral maturation by processing the gag and gag-pro-pol polyproteins.  HIV-1 PR inhibitors (PIs) are used in combination antiviral therapy but the emergence of drug resistance has limited their efficacy.  The rapid evolution of HIV-1 necessitates consideration of drug resistance in novel drug design.  Drug-resistant HIV-1 PR variants no longer inhibited efficiently, continue to hydrolyze the natural viral substrates.  Though highly diverse in sequence, the HIV-1 PR substrates bind in a conserved three-dimensional shape we termed the substrate envelope.  Earlier, we showed that resistance mutations arise where PIs protrude beyond the substrate envelope, because these regions are crucial for drug binding but not for substrate recognition.  We extend this model by considering the role of protein dynamics in the interaction of HIV-1 PR with its substrates.  We simulated the mol. dynamics of seven PR-substrate complexes to est. the conformational flexibility of the bound substrates.  Interdependence of substrate-protease interactions might compensate for variations in cleavage-site sequences and explain how a diverse set of sequences are recognized as substrates by the same enzyme.  This diversity might be essential for regulating sequential processing of substrates.  We define a dynamic substrate envelope as a more accurate representation of PR-substrate interactions.  This dynamic substrate envelope, described by a probability distribution function, is a powerful tool for drug design efforts targeting ensembles of resistant HIV-1 PR variants with the aim of developing drugs that are less susceptible to resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2avfbPli-5LVg90H21EOLACvtfcHk0lha2qFfDSCpWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovF2mtr0%253D&md5=726b923be7b92c8cbc241a8947c427ff</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2011.03.053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2011.03.053%26sid%3Dliteratum%253Aachs%26aulast%3D%25C3%2596zen%26aufirst%3DA.%26aulast%3DHalilo%25C7%25A7lu%26aufirst%3DT.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DDynamics%2520of%2520preferential%2520substrate%2520recognition%2520in%2520HIV-1%2520protease%253A%2520redefining%2520the%2520substrate%2520envelope%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D410%26spage%3D726%26epage%3D744%26doi%3D10.1016%2Fj.jmb.2011.03.053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paulsen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leidner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ragland, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurt Yilmaz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Interdependence of inhibitor recognition in HIV-1 protease</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2300</span>– <span class="NLM_lpage">2309</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.6b01262</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.6b01262" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltlWgs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=2300-2309&author=J.+L.+Paulsenauthor=F.+Leidnerauthor=D.+A.+Raglandauthor=N.+Kurt+Yilmazauthor=C.+A.+Schiffer&title=Interdependence+of+inhibitor+recognition+in+HIV-1+protease&doi=10.1021%2Facs.jctc.6b01262"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Interdependence of Inhibitor Recognition in HIV-1 Protease</span></div><div class="casAuthors">Paulsen, Janet L.; Leidner, Florian; Ragland, Debra A.; Kurt Yilmaz, Nese; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2300-2309</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. recognition is a highly interdependent process.  Subsite couplings within the active site of proteases are most often revealed through conditional amino acid preferences in substrate recognition.  However, the potential effect of these couplings on inhibition and thus inhibitor design are largely unexplored.  The present study examines the interdependency of subsites in HIV-1 protease using a focused library of protease inhibitors, to aid in future inhibitor design.  Previously a series of darunavir (DRV) analogs was designed to systematically probe the S1' and S2' subsites.  Co-crystal structures of these analogs with HIV-1 protease provide the ideal opportunity to probe subsite interdependency.  All-atom mol. dynamics simulations starting from these structures were performed, and systematically analyzed in terms of at. fluctuations, intermol. interactions and water structure.  These analyses reveal that the S1' subsite highly influences other subsites: the extension of the hydrophobic P1' moiety results in (1) reduced van der Waals contacts in the P2' subsite, (2) more variability in the hydrogen bond frequencies with catalytic residues and the flap water, and (3) changes in the occupancy of conserved water sites both proximal and distal to the active site.  In addn., one of the monomers in this homodimeric enzyme has at. fluctuations more highly correlated with DRV than the other monomer.  These relationships intricately link the HIV-1 protease subsites, and are crit. to understanding of mol. recognition and inhibitor binding.  More broadly, the interdependency of subsite recognition within an active site requires consideration in the selection of chem. moieties in drug design; this strategy is in contrast to what is traditionally done with independent optimization of chem. moieties of an inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtIHU1NPM7ObVg90H21EOLACvtfcHk0lhuTzKtJD5FzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltlWgs7g%253D&md5=0710306cc5e01e808fca75475c4ca293</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.6b01262&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.6b01262%26sid%3Dliteratum%253Aachs%26aulast%3DPaulsen%26aufirst%3DJ.%2BL.%26aulast%3DLeidner%26aufirst%3DF.%26aulast%3DRagland%26aufirst%3DD.%2BA.%26aulast%3DKurt%2BYilmaz%26aufirst%3DN.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DInterdependence%2520of%2520inhibitor%2520recognition%2520in%2520HIV-1%2520protease%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2017%26volume%3D13%26spage%3D2300%26epage%3D2309%26doi%3D10.1021%2Facs.jctc.6b01262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haight, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuk, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhagavatula, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannick, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerdesky, F. A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menzia, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parekh, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robbins, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scarpetti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tien, J.-H. J.</span></span> <span> </span><span class="NLM_article-title">Reduction of an enaminone: synthesis of the diamino alcohol core of ritonavir</span>. <i>Org. Process Res. Dev.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">94</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1021/op9802071</span> </span><div class="citationLinks">[<a href="/doi/10.1021/op9802071" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADyaK1MXhslOntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1999&pages=94-100&author=A.+R.+Haightauthor=T.+L.+Stukauthor=M.+S.+Allenauthor=L.+Bhagavatulaauthor=M.+Fitzgeraldauthor=S.+M.+Hannickauthor=F.+A.+J.+Kerdeskyauthor=J.+A.+Menziaauthor=S.+I.+Parekhauthor=T.+A.+Robbinsauthor=D.+Scarpettiauthor=J.-H.+J.+Tien&title=Reduction+of+an+enaminone%3A+synthesis+of+the+diamino+alcohol+core+of+ritonavir&doi=10.1021%2Fop9802071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Reduction of an Enaminone: Synthesis of the Diamino Alcohol Core of Ritonavir</span></div><div class="casAuthors">Haight, Anthony R.; Stuk, Timothy L.; Allen, Michael S.; Bhagavatula, Lakshmi; Fitzgerald, Michael; Hannick, Steven M.; Kerdesky, Francis A. J.; Menzia, Jerome A.; Parekh, Shyamal I.; Robbins, Timothy A.; Scarpetti, David; Tien, Jien-Heh J.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Process Research & Development</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">94-100</span>CODEN:
                <span class="NLM_cas:coden">OPRDFK</span>;
        ISSN:<span class="NLM_cas:issn">1083-6160</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The redn. of (5S)-2-amino-5-dibenzylamino-4-oxo-1,6-diphenylhex-2-ene (I) was optimized for diastereoselectivity and overall conversion to (2S,3S,5S)-5-amino-2-dibenzylamino-3-hydroxy-1,6-diphenylhexane (II).  A two-step redn. sequence is described wherein the enamine is reduced with a borane-sulfonate deriv. followed by redn. of the resulting ketone with sodium borohydride.  The desired II was obtained with 84% diastereoselectivity and an acyclic 1,4 stereoinduction ratio of 14:1.  This methodol. has been used to produce multikilogram quantities of the diamino alc. core of Ritonavir (III) and should be general to the synthesis of related diamino hydroxyethylene isosteres.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVtDKnLMDXYLVg90H21EOLACvtfcHk0lhuTzKtJD5FzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhslOntg%253D%253D&md5=a3961137cde012f2c6cc62eeabb3727c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fop9802071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fop9802071%26sid%3Dliteratum%253Aachs%26aulast%3DHaight%26aufirst%3DA.%2BR.%26aulast%3DStuk%26aufirst%3DT.%2BL.%26aulast%3DAllen%26aufirst%3DM.%2BS.%26aulast%3DBhagavatula%26aufirst%3DL.%26aulast%3DFitzgerald%26aufirst%3DM.%26aulast%3DHannick%26aufirst%3DS.%2BM.%26aulast%3DKerdesky%26aufirst%3DF.%2BA.%2BJ.%26aulast%3DMenzia%26aufirst%3DJ.%2BA.%26aulast%3DParekh%26aufirst%3DS.%2BI.%26aulast%3DRobbins%26aufirst%3DT.%2BA.%26aulast%3DScarpetti%26aufirst%3DD.%26aulast%3DTien%26aufirst%3DJ.-H.%2BJ.%26atitle%3DReduction%2520of%2520an%2520enaminone%253A%2520synthesis%2520of%2520the%2520diamino%2520alcohol%2520core%2520of%2520ritonavir%26jtitle%3DOrg.%2520Process%2520Res.%2520Dev.%26date%3D1999%26volume%3D3%26spage%3D94%26epage%3D100%26doi%3D10.1021%2Fop9802071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engle, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J. Q.</span></span> <span> </span><span class="NLM_article-title">Ligand-accelerated C-H activation reactions: evidence for a switch of mechanism</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">14137</span>– <span class="NLM_lpage">14151</span>, <span class="refDoi"> DOI: 10.1021/ja105044s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja105044s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFygsbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=14137-14151&author=K.+M.+Engleauthor=D.+H.+Wangauthor=J.+Q.+Yu&title=Ligand-accelerated+C-H+activation+reactions%3A+evidence+for+a+switch+of+mechanism&doi=10.1021%2Fja105044s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-Accelerated C-H Activation Reactions: Evidence for a Switch of Mechanism</span></div><div class="casAuthors">Engle, Keary M.; Wang, Dong-Hui; Yu, Jin-Quan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">14137-14151</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Initial rate studies have revealed dramatic acceleration in aerobic Pd(II)-catalyzed C-H olefination reactions of phenylacetic acids when mono-N-protected amino acids are used as ligands.  In light of these findings, systematic ligand tuning was undertaken, which has resulted in drastic improvements in substrate scope, reaction rate, and catalyst turnover.  We present evidence from intermol. competition studies and kinetic isotope effect expts. that implies that the obsd. rate increases are a result of acceleration in the C-H cleavage step.  Furthermore, these studies suggest that the origin of this phenomenon is a change in the mechanism of C-H cleavage from electrophilic palladation to proton abstraction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_PdoqifF8irVg90H21EOLACvtfcHk0lhuTzKtJD5FzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFygsbzM&md5=b4011c8813e2fad95a9a7cb423d2623b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fja105044s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja105044s%26sid%3Dliteratum%253Aachs%26aulast%3DEngle%26aufirst%3DK.%2BM.%26aulast%3DWang%26aufirst%3DD.%2BH.%26aulast%3DYu%26aufirst%3DJ.%2BQ.%26atitle%3DLigand-accelerated%2520C-H%2520activation%2520reactions%253A%2520evidence%2520for%2520a%2520switch%2520of%2520mechanism%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D14137%26epage%3D14151%26doi%3D10.1021%2Fja105044s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Windsor, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raines, R. T.</span></span> <span> </span><span class="NLM_article-title">Fluorogenic Assay for Inhibitors of HIV-1 Protease with Sub-picomolar Affinity</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">11286</span>, <span class="refDoi"> DOI: 10.1038/srep11286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1038%2Fsrep11286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=26261098" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVKhs77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=11286&author=I.+W.+Windsorauthor=R.+T.+Raines&title=Fluorogenic+Assay+for+Inhibitors+of+HIV-1+Protease+with+Sub-picomolar+Affinity&doi=10.1038%2Fsrep11286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorogenic Assay for Inhibitors of HIV-1 Protease with Sub-picomolar Affinity</span></div><div class="casAuthors">Windsor, Ian W.; Raines, Ronald T.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11286</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A fluorogenic substrate for HIV-1 protease was designed and used as the basis for a hypersensitive assay.  The substrate exhibits a kcat of 7.4 s-1, KM of 15 μM, and an increase in fluorescence intensity of 104-fold upon cleavage, thus providing sensitivity that is unmatched in a continuous assay of HIV-1 protease.  These properties enabled the enzyme concn. in an activity assay to be reduced to 25 pM, which is close to the Kd value of the protease dimer.  By fitting inhibition data to Morrison's equation, Ki values of amprenavir, darunavir, and tipranavir were detd. to be 135, 10, and 82 pM, resp.  This assay, which is capable of measuring Ki values as low as 0.25 pM, is well-suited for characterizing the next generation of HIV-1 protease inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjTs91FgAshbVg90H21EOLACvtfcHk0lh-p0Gi0FAC1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVKhs77F&md5=0371262982d14e659af3d60f2691a3c2</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fsrep11286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep11286%26sid%3Dliteratum%253Aachs%26aulast%3DWindsor%26aufirst%3DI.%2BW.%26aulast%3DRaines%26aufirst%3DR.%2BT.%26atitle%3DFluorogenic%2520Assay%2520for%2520Inhibitors%2520of%2520HIV-1%2520Protease%2520with%2520Sub-picomolar%2520Affinity%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D11286%26doi%3D10.1038%2Fsrep11286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Curling of flap tips in HIV-1 protease as a mechanism for substrate entry and tolerance of drug resistance</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1259</span>– <span class="NLM_lpage">1265</span>, <span class="refDoi"> DOI: 10.1016/S0969-2126(00)00537-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2FS0969-2126%2800%2900537-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=11188690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhtFentrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2000&pages=1259-1265&author=W.+R.+P.+Scottauthor=C.+A.+Schiffer&title=Curling+of+flap+tips+in+HIV-1+protease+as+a+mechanism+for+substrate+entry+and+tolerance+of+drug+resistance&doi=10.1016%2FS0969-2126%2800%2900537-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Curling of Flap Tips in HIV-1 Protease as a Mechanism for Substrate Entry and Tolerance of Drug Resistance</span></div><div class="casAuthors">Scott, W. R. P.; Schiffer, C. A.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1259-1265</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Background: The human immunodeficiency virus type 1 (HIV-1) protease is an essential viral protein that is a major drug target in the fight against Acquired Immune Deficiency Syndrome (AIDS).  Access to the active site of this homodimeric enzyme is gained when two large flaps, one from each monomer, open.  The flap movements are therefore central to the function of the enzyme, yet detg. how these flaps move at an at. level has not been exptl. possible.  In the present study, the authors observe the flaps of HIV-1 protease completely opening during a 10 ns solvated mol. dynamics simulation starting from the unliganded crystal structure.  This movement is on the time scale obsd. by NMR relaxation data.  The highly flexible tips of the flaps, with the sequence Gly-Gly-Ile-Gly-Gly, are seen curling back into the protein and thereby burying many hydrophobic residues.  This curled-in conformational change has never been previously described.  Previous models of this movement, with the flaps as rigid levers, are not consistent with the exptl. data.  The residues that participate in this hydrophobic cluster as a result of the conformational change are highly sensitive to mutation and often contribute to drug resistance when they do change.  However, several of these residues are not part of the active site cavity, and their essential role in causing drug resistance could possibly be rationalized if this conformational change actually occurs.  Trapping HIV-1 protease in this inactive conformation would provide a unique opportunity for future drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp79jBAv68z07Vg90H21EOLACvtfcHk0lh-p0Gi0FAC1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhtFentrY%253D&md5=84338adcf206539ad8a71ce378eca4c0</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2800%2900537-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252800%252900537-2%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DW.%2BR.%2BP.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DCurling%2520of%2520flap%2520tips%2520in%2520HIV-1%2520protease%2520as%2520a%2520mechanism%2520for%2520substrate%2520entry%2520and%2520tolerance%2520of%2520drug%2520resistance%26jtitle%3DStructure%26date%3D2000%26volume%3D8%26spage%3D1259%26epage%3D1265%26doi%3D10.1016%2FS0969-2126%2800%2900537-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span> <span> </span><span class="NLM_article-title">Structural insights into HIV-1 protease flap opening processes and key intermediates</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">45121</span>– <span class="NLM_lpage">45128</span>, <span class="refDoi"> DOI: 10.1039/C7RA09691G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1039%2FC7RA09691G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeqt7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=45121-45128&author=Y.+Yuauthor=J.+Wangauthor=Z.+Chenauthor=G.+Wangauthor=Q.+Shaoauthor=J.+Shiauthor=W.+Zhu&title=Structural+insights+into+HIV-1+protease+flap+opening+processes+and+key+intermediates&doi=10.1039%2FC7RA09691G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into HIV-1 protease flap opening processes and key intermediates</span></div><div class="casAuthors">Yu, Yuqi; Wang, Jinan; Chen, Zhaoqiang; Wang, Guimin; Shao, Qiang; Shi, Jiye; Zhu, Weiliang</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">71</span>),
    <span class="NLM_cas:pages">45121-45128</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">HIV-1 protease (PR) is an effective drug target for antiviral inhibitors.  The conformational dynamics in the flaps of HIV-1 PR plays a crucial role in the mechanism of substrate binding.  Here, the structural properties of the functionally important intermediate states of the flap opening transition of HIV-1 PR have been characterized by enhanced sampling mol. dynamics simulation as well as long-time conventional non-enhanced simulations at at. level.  Not only crystallog. measured "closed" and "semi-open" structures but also a novel "curled" structure of HIV-1 PR is captured by both kinds of simulations qual. and quant.  The observation of the "curled" intermediate state helps to connect all other functionally important states to provide an integrated view of the transition pathway of the flap opening of HIV-1 PR (closed → curled → semi-open → fully open).  The key residue-residue interactions which are broken or formed in the transition are analyzed to reveal the inherent driving force for the protein conformational transition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5kSh0BjTWNLVg90H21EOLACvtfcHk0lh-p0Gi0FAC1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeqt7nN&md5=afbad6dff4ee880b3906bf08cdf2a7fd</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1039%2FC7RA09691G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7RA09691G%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DShao%26aufirst%3DQ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DW.%26atitle%3DStructural%2520insights%2520into%2520HIV-1%2520protease%2520flap%2520opening%2520processes%2520and%2520key%2520intermediates%26jtitle%3DRSC%2520Adv.%26date%3D2017%26volume%3D7%26spage%3D45121%26epage%3D45128%26doi%3D10.1039%2FC7RA09691G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Özen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurt Yilmaz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Structural basis and distal effects of Gag substrate coevolution in drug resistance to HIV-1 protease</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">15993</span>– <span class="NLM_lpage">15998</span>, <span class="refDoi"> DOI: 10.1073/pnas.1414063111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1073%2Fpnas.1414063111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=25355911" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVemsrnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=15993-15998&author=A.+%C3%96zenauthor=K.-H.+Linauthor=N.+Kurt+Yilmazauthor=C.+A.+Schiffer&title=Structural+basis+and+distal+effects+of+Gag+substrate+coevolution+in+drug+resistance+to+HIV-1+protease&doi=10.1073%2Fpnas.1414063111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis and distal effects of Gag substrate coevolution in drug resistance to HIV-1 protease</span></div><div class="casAuthors">Ozen, Aysegul; Lin, Kuan-Hung; Yilmaz, Nese Kurt; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">15993-15998</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Drug resistance mutations in response to HIV-1 protease inhibitors are selected not only in the drug target but elsewhere in the viral genome, esp. at the protease cleavage sites in the precursor protein Gag.  To understand the mol. basis of this protease-substrate coevolution, we solved the crystal structures of drug resistant I50V/A71V HIV-1 protease with p1-p6 substrates bearing coevolved mutations.  Analyses of the protease-substrate interactions reveal that compensatory coevolved mutations in the substrate do not restore interactions lost due to protease mutations, but instead establish other interactions that are not restricted to the site of mutation.  Mutation of a substrate residue has distal effects on other residues' interactions as well, including through the induction of a conformational change in the protease.  Addnl., mol. dynamics simulations suggest that restoration of active site dynamics is an addnl. constraint in the selection of coevolved mutations.  Hence, protease-substrate coevolution permits mutational, structural, and dynamic changes via mol. mechanisms that involve distal effects contributing to drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-njLLQk8frLVg90H21EOLACvtfcHk0lii8yg7kO-cLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVemsrnE&md5=b7ebcc341026ba01cd0453883dcca480</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1414063111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1414063111%26sid%3Dliteratum%253Aachs%26aulast%3D%25C3%2596zen%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DK.-H.%26aulast%3DKurt%2BYilmaz%26aufirst%3DN.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DStructural%2520basis%2520and%2520distal%2520effects%2520of%2520Gag%2520substrate%2520coevolution%2520in%2520drug%2520resistance%2520to%2520HIV-1%2520protease%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D15993%26epage%3D15998%26doi%3D10.1073%2Fpnas.1414063111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">King, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prabu-Jeyabalan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nalivaika, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zepp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heefner, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Lack of synergy for inhibitors targeting a multi-drug-resistant HIV-1 protease</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">418</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1110/ps.25502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1110%2Fps.25502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=11790852" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD38XosFyrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2002&pages=418-429&author=N.+M.+Kingauthor=L.+Melnickauthor=M.+Prabu-Jeyabalanauthor=E.+A.+Nalivaikaauthor=S.+S.+Yangauthor=Y.+Gaoauthor=X.+Nieauthor=C.+Zeppauthor=D.+L.+Heefnerauthor=C.+A.+Schiffer&title=Lack+of+synergy+for+inhibitors+targeting+a+multi-drug-resistant+HIV-1+protease&doi=10.1110%2Fps.25502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Lack of synergy for inhibitors targeting a multi-drug-resistant HIV-1 protease</span></div><div class="casAuthors">King, Nancy M.; Melnick, Laurence; Prabu-Jeyabalan, Moses; Nalivaika, Ellen A.; Yang, Shiow-Shong; Gao, Yun; Nie, Xiaoyi; Zepp, Charles; Heefner, Donald L.; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">418-429</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The three-dimensional structures of indinavir and three newly synthesized indinavir analogs in complex with a multi-drug-resistant variant (L63P, V82T, I84V) of HIV-1 protease were detd. to ∼2.2 Å resoln.  Two of the three analogs have only a single modification of indinavir, and their binding affinities to the variant HIV-1 protease are enhanced over that of indinavir.  However, when both modifications were combined into a single compd., the binding affinity to the protease variant was reduced.  On close examn., the structural rearrangements in the protease that occur in the tightest binding inhibitor complex are mutually exclusive with the structural rearrangements seen in the second tightest inhibitor complex.  This occurs as adaptations in the S1 pocket of one monomer propagate through the dimer and affect the conformation of the S1 loop near P81 of the other monomer.  Therefore, structural rearrangements that occur within the protease when it binds to an inhibitor with a single modification must be accounted for in the design of inhibitors with multiple modifications.  This consideration is necessary to develop inhibitors that bind sufficiently tightly to drug-resistant variants of HIV-1 protease to potentially become the next generation of therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRhi_yayOc3bVg90H21EOLACvtfcHk0lii8yg7kO-cLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosFyrsg%253D%253D&md5=9855843d5da7ecf81a51aa0734175cc0</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1110%2Fps.25502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.25502%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DN.%2BM.%26aulast%3DMelnick%26aufirst%3DL.%26aulast%3DPrabu-Jeyabalan%26aufirst%3DM.%26aulast%3DNalivaika%26aufirst%3DE.%2BA.%26aulast%3DYang%26aufirst%3DS.%2BS.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DNie%26aufirst%3DX.%26aulast%3DZepp%26aufirst%3DC.%26aulast%3DHeefner%26aufirst%3DD.%2BL.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DLack%2520of%2520synergy%2520for%2520inhibitors%2520targeting%2520a%2520multi-drug-resistant%2520HIV-1%2520protease%26jtitle%3DProtein%2520Sci.%26date%3D2002%26volume%3D11%26spage%3D418%26epage%3D429%26doi%3D10.1110%2Fps.25502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rose, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, C. S.</span></span> <span> </span><span class="NLM_article-title">Regulation of autoproteolysis of the HIV-1 and HIV-2 proteases with engineered amino acid substitutions</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>268</i></span>,  <span class="NLM_fpage">11939</span>– <span class="NLM_lpage">11945</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=8505318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADyaK3sXltFWqtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=268&publication_year=1993&pages=11939-11945&author=J.+R.+Roseauthor=R.+Saltoauthor=C.+S.+Craik&title=Regulation+of+autoproteolysis+of+the+HIV-1+and+HIV-2+proteases+with+engineered+amino+acid+substitutions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of autoproteolysis of the HIV-1 and HIV-2 proteases with engineered amino acid substitutions</span></div><div class="casAuthors">Rose, Jason R.; Salto, Rafael; Craik, Charles S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">268</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">11939-45</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Autoproteolysis of the retroviral aspartyl proteases is a major obstacle to purifn. and anal. of these enzymes.  A mutagenic approach to rendering autolytic cleavage sites less labile was applied to the primary cleavage site between Leu5 and Trp6 in human immunodeficiency virus-1 (HIV-1) protease.  From predictions based on known substrates it was concluded that amino acids Lys or Ser in place of Gln at position 7 would prevent cleavage at the Leu5-Trp6 peptide bond, therefore stabilizing the protein.  Autoproteolytic stability was enhanced at least 100-fold by these mutations.  At longer time points the protease was degraded at secondary sites which contained adequate substrate sequences but were conformationally restricted.  Conversely, a mutation in HIV-2 protease which changed Lys7 to Gln rendered the protein 3-fold less stable and shifted the position of the initial autoproteolytic cleavage from Phe3-Ser4 to Leu5-Trp6.  The effects of these mutations demonstrate that small changes in protein sequence can have a major impact on their autoproteolytic stability.  The work described here suggests a general method for stabilizing proteases and perhaps other recombinantly produced proteins to autolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM8gNMxq7onbVg90H21EOLACvtfcHk0lii8yg7kO-cLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXltFWqtbc%253D&md5=2b87aa648119fbad8d132accb82c1c7d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DJ.%2BR.%26aulast%3DSalto%26aufirst%3DR.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26atitle%3DRegulation%2520of%2520autoproteolysis%2520of%2520the%2520HIV-1%2520and%2520HIV-2%2520proteases%2520with%2520engineered%2520amino%2520acid%2520substitutions%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1993%26volume%3D268%26spage%3D11939%26epage%3D11945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hui, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomasselli, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reardon, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lull, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunner, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomich, C.-S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinrikson, R. L.</span></span> <span> </span><span class="NLM_article-title">Large scale purification and refolding of HIV-1 protease from Escherichia coli inclusion bodies</span>. <i>J. Protein Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">323</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.1007/BF01028194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1007%2FBF01028194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=8397790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADyaK3sXlvFyhtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1993&pages=323-327&author=J.+O.+Huiauthor=A.+G.+Tomasselliauthor=I.+M.+Reardonauthor=J.+M.+Lullauthor=D.+P.+Brunnerauthor=C.-S.+C.+Tomichauthor=R.+L.+Heinrikson&title=Large+scale+purification+and+refolding+of+HIV-1+protease+from+Escherichia+coli+inclusion+bodies&doi=10.1007%2FBF01028194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Large scale purification and refolding of HIV-1 protease from Escherichia coli inclusion bodies</span></div><div class="casAuthors">Hui, John O.; Tomasselli, Alfredo G.; Reardon, Ilene M.; Lull, June M.; Brunner, David P.; Tomich, Che Shen C.; Heinrikson, Robert L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Protein Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">323-7</span>CODEN:
                <span class="NLM_cas:coden">JPCHD2</span>;
        ISSN:<span class="NLM_cas:issn">0277-8033</span>.
    </div><div class="casAbstract">Retropepsin (I), the aspartic protease encoded by human immunodeficiency virus type 1 (HIV-1), was engineered in E. coli as a construct in which the natural 99-residue polypeptide was preceded by an N-terminal methionine initiator.  Inclusion bodies harboring recombinant HIV-1 I were dissolved in 50% HOAc and the soln. was subjected to gel filtration on a column of Sephadex G-75.  I, eluted in the 2nd of 2 peaks, migrated in SDS-PAGE as a single sharp band of mol. wt. ∼10,000.  Purified HIV-1 I was refolded into an active enzyme by dilg. a soln. of the protein in 50% HOAc with 25 vols. of buffer at pH 5.5.  This method of purifn., which was also applied to the purifn. of HIV-2 I, provides a single-step procedure to produce 100-mg quantities of fully active enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXYIBwJp92ELVg90H21EOLACvtfcHk0lgq0I0EUm2Dsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXlvFyhtrk%253D&md5=61cd315aa7b58c9bf84bb1f6c614be8a</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1007%2FBF01028194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01028194%26sid%3Dliteratum%253Aachs%26aulast%3DHui%26aufirst%3DJ.%2BO.%26aulast%3DTomasselli%26aufirst%3DA.%2BG.%26aulast%3DReardon%26aufirst%3DI.%2BM.%26aulast%3DLull%26aufirst%3DJ.%2BM.%26aulast%3DBrunner%26aufirst%3DD.%2BP.%26aulast%3DTomich%26aufirst%3DC.-S.%2BC.%26aulast%3DHeinrikson%26aufirst%3DR.%2BL.%26atitle%3DLarge%2520scale%2520purification%2520and%2520refolding%2520of%2520HIV-1%2520protease%2520from%2520Escherichia%2520coli%2520inclusion%2520bodies%26jtitle%3DJ.%2520Protein%2520Chem.%26date%3D1993%26volume%3D12%26spage%3D323%26epage%3D327%26doi%3D10.1007%2FBF01028194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rusere, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockbaum, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosovrasti, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spielvogel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nalivaika, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanstrom, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yilmaz, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, A.</span></span> <span> </span><span class="NLM_article-title">HIV-1 protease inhibitors incorporating stereochemically defined P2’ ligands to optimize hydrogen bonding in the substrate envelope</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">8062</span>– <span class="NLM_lpage">8079</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00838</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00838" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFamsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=8062-8079&author=L.+N.+Rusereauthor=G.+J.+Lockbaumauthor=S.+K.+Leeauthor=M.+Henesauthor=K.+Kosovrastiauthor=E.+Spielvogelauthor=E.+A.+Nalivaikaauthor=R.+Swanstromauthor=N.+K.+Yilmazauthor=C.+A.+Schifferauthor=A.+Ali&title=HIV-1+protease+inhibitors+incorporating+stereochemically+defined+P2%E2%80%99+ligands+to+optimize+hydrogen+bonding+in+the+substrate+envelope&doi=10.1021%2Facs.jmedchem.9b00838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">HIV-1 Protease Inhibitors Incorporating Stereochemically Defined P2' Ligands To Optimize Hydrogen Bonding in the Substrate Envelope</span></div><div class="casAuthors">Rusere, Linah N.; Lockbaum, Gordon J.; Lee, Sook-Kyung; Henes, Mina; Kosovrasti, Klajdi; Spielvogel, Ean; Nalivaika, Ellen A.; Swanstrom, Ronald; Yilmaz, Nese Kurt; Schiffer, Celia A.; Ali, Akbar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8062-8079</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A structure-guided design strategy was used to improve the resistance profile of HIV-1 protease inhibitors by optimizing hydrogen bonding and van der Waals interactions with the protease while staying within the substrate envelope.  Stereoisomers of 4-(1-hydroxyethyl)benzene and 4-(1,2-dihydroxyethyl)benzene moieties were explored as P2' ligands providing pairs of diastereoisomers epimeric at P2', which exhibited distinct potency profiles depending on the configuration of the hydroxyl group and size of the P1' group.  While compds. with the 4-(1-hydroxyethyl)benzene P2' moiety maintained excellent antiviral potency against a panel of multidrug-resistant HIV-1 strains, analogs with the polar 4-(1,2-dihydroxyethyl)benzene moiety were less potent, and only the (R)-epimer incorporating a larger 2-ethylbutyl P1' group showed improved potency.  Crystal structures of protease-inhibitor complexes revealed strong hydrogen bonding interactions of both (R)- and (S)-stereoisomers of the hydroxyethyl group with Asp30'.  Notably, the (R)-dihydroxyethyl group was involved in a unique pattern of direct hydrogen bonding interactions with the backbone amides of Asp29' and Asp30'.  The SAR data and anal. of crystal structures provide insights for optimizing these promising HIV-1 protease inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYYCrKYi2QMbVg90H21EOLACvtfcHk0lgq0I0EUm2Dsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFamsrrJ&md5=d4a719a7aaf985b4ef27c7613f68a37d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00838%26sid%3Dliteratum%253Aachs%26aulast%3DRusere%26aufirst%3DL.%2BN.%26aulast%3DLockbaum%26aufirst%3DG.%2BJ.%26aulast%3DLee%26aufirst%3DS.%2BK.%26aulast%3DHenes%26aufirst%3DM.%26aulast%3DKosovrasti%26aufirst%3DK.%26aulast%3DSpielvogel%26aufirst%3DE.%26aulast%3DNalivaika%26aufirst%3DE.%2BA.%26aulast%3DSwanstrom%26aufirst%3DR.%26aulast%3DYilmaz%26aufirst%3DN.%2BK.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26aulast%3DAli%26aufirst%3DA.%26atitle%3DHIV-1%2520protease%2520inhibitors%2520incorporating%2520stereochemically%2520defined%2520P2%25E2%2580%2599%2520ligands%2520to%2520optimize%2520hydrogen%2520bonding%2520in%2520the%2520substrate%2520envelope%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D8062%26epage%3D8079%26doi%3D10.1021%2Facs.jmedchem.9b00838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decker, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arani, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilby, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saag, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kappes, J. C.</span></span> <span> </span><span class="NLM_article-title">Emergence of resistant human immunodeficiency virus type 1 in patients receiving fusion inhibitor (T-20) monotherapy</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1896</span>– <span class="NLM_lpage">1905</span>, <span class="refDoi"> DOI: 10.1128/AAC.46.6.1896-1905.2002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1128%2FAAC.46.6.1896-1905.2002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=12019106" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD38XktFCmsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2002&pages=1896-1905&author=X.+Weiauthor=J.+M.+Deckerauthor=H.+Liuauthor=Z.+Zhangauthor=R.+B.+Araniauthor=J.+M.+Kilbyauthor=M.+S.+Saagauthor=X.+Wuauthor=G.+M.+Shawauthor=J.+C.+Kappes&title=Emergence+of+resistant+human+immunodeficiency+virus+type+1+in+patients+receiving+fusion+inhibitor+%28T-20%29+monotherapy&doi=10.1128%2FAAC.46.6.1896-1905.2002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Emergence of resistant human immunodeficiency virus type 1 in patients receiving fusion inhibitor (T-20) monotherapy</span></div><div class="casAuthors">Wei, Xiping; Decker, Julie M.; Liu, Hongmei; Zhang, Zee; Arani, Ramin B.; Kilby, J. Michael; Saag, Michael S.; Wu, Xiaoyun; Shaw, George M.; Kappes, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1896-1905</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The synthetic peptide T-20 (enfuvirtide) represents the first of a new class of antiretroviral compds. to demonstrate in vivo potency by targeting a step in viral entry.  T-20 inhibits a conformational change in the human immunodeficiency virus type 1 (HIV-1) transmembrane glycoprotein (gp41) that is required for fusion between HIV-1 and target cell membranes.  The initial phase I clin. trial of T-20 treatment for HIV-infected patients thus provided a unique opportunity to evaluate the emergence of resistant virus in vivo to this novel class of antiretroviral agents.  All four patients who received an intermediate dose of T-20 (30 mg twice daily) had an initial decline in plasma viral load over the first 10 days but a rising trend by day 14, suggestive of selection for resistant virus.  Plasma virus derived from patients enrolled in all dosage groups of the phase I T-20 trial was analyzed by population sequencing before and after treatment.  While no mutations were found within a highly conserved 3-amino-acid sequence (GIV) known to be crit. for fusion at baseline, after 14 days of therapy, virus from one patient in the 30-mg dose group (30-1) developed a mutation in this motif, specifically an aspartic acid (D) substitution for glycine (G) at position 36.  Multiple env clones were derived from the plasma virus of all four patients in the 30-mg dosage group.  Sequence anal. of 49 clones derived from the plasma of patient 30-1 on day 14 revealed that 25 clones contained the G36D mutation, while 8 contained the V38A mutation.  Dual mutations involving G36D and other residues within the HR1 domain were also identified.  In 5 of the 49 env clones, other mutations involving residues 32 (Q32R or Q32H) and 39 (Q39R) were found in combination with G36D.  Cloned env sequences derived from the plasma virus of subject 30-3 also had single mutations in the GIV sequence (V38M and I37V) detectable following therapy with T-20.  The plasma virus from subjects 30-2 and 30-4 did not contain changes within the GIV sequence.  To analyze the biol. resistance properties of these mutations, we developed a novel single-cycle HIV-1 entry assay using JC53BL cells which express β-galactosidase and luciferase under control of the HIV-1 long terminal repeat.  Full-length env clones were derived from the plasma virus of patients 30-1 and 30-3 and used to generate pseudotyped virus stocks.  The mean 50% inhibition concns. (IC50s) for mutants G36D and V38A (patient 30-1) were 2.3 μg/mL and 11.2 μg/mL, resp., statistically significant increases of 9.1- and 45-fold, resp., compared with those of wild-type Env.  The IC50 for the V38 M mutation (patient 30-3) was 7.6 μg/mL, an 8-fold increase compared with that of the wild type.  The I37V mutation resulted in an IC50 3.2-fold greater than that of the wild type.  Envs with double mutations (Q32R plus G36D and Q32H plus G36D) exhibited a level of resistance similar to that of G36D alone.  These findings provide the first evidence for the rapid emergence of clin. resistance to a novel class of HIV-1 entry inhibitors and may be relevant to future treatment strategies involving these agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTmBSTKEK3v7Vg90H21EOLACvtfcHk0lgq0I0EUm2Dsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktFCmsb8%253D&md5=65977bf843ca8ec9411a88b50ef71206</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1128%2FAAC.46.6.1896-1905.2002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.46.6.1896-1905.2002%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DX.%26aulast%3DDecker%26aufirst%3DJ.%2BM.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DArani%26aufirst%3DR.%2BB.%26aulast%3DKilby%26aufirst%3DJ.%2BM.%26aulast%3DSaag%26aufirst%3DM.%2BS.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DShaw%26aufirst%3DG.%2BM.%26aulast%3DKappes%26aufirst%3DJ.%2BC.%26atitle%3DEmergence%2520of%2520resistant%2520human%2520immunodeficiency%2520virus%2520type%25201%2520in%2520patients%2520receiving%2520fusion%2520inhibitor%2520%2528T-20%2529%2520monotherapy%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2002%26volume%3D46%26spage%3D1896%26epage%3D1905%26doi%3D10.1128%2FAAC.46.6.1896-1905.2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanstrom, R.</span></span> <span> </span><span class="NLM_article-title">A strongly transdominant mutation in the human immunodeficiency virus type 1 gag gene defines an Achilles heel in the virus life cycle</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">8536</span>– <span class="NLM_lpage">8543</span>, <span class="refDoi"> DOI: 10.1128/JVI.00317-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1128%2FJVI.00317-09" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=19515760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFahur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2009&pages=8536-8543&author=S.+K.+Leeauthor=J.+Harrisauthor=R.+Swanstrom&title=A+strongly+transdominant+mutation+in+the+human+immunodeficiency+virus+type+1+gag+gene+defines+an+Achilles+heel+in+the+virus+life+cycle&doi=10.1128%2FJVI.00317-09"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A strongly transdominant mutation in the human immunodeficiency virus type 1 gag gene defines an Achilles heel in the virus life cycle</span></div><div class="casAuthors">Lee, Sook-Kyung; Harris, Janera; Swanstrom, Ronald</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8536-8543</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The human immunodeficiency virus type 1 (HIV-1) protease (PR) makes five obligatory cleavages in the viral Gag polyprotein precursor.  The cleavage events release the virion structural proteins from the precursor and allow the virion to undergo maturation to become infectious.  The protease cleavage between the matrix protein (MA) domain and the adjacent capsid protein (CA) domain releases CA from the membrane-anchored MA and allows the N terminus of CA to refold into a structure that facilitates the formation of hexamer arrays that represent the structural unit of the capsid shell.  In this study, we analyzed the extent to which each of the HIV-1 Gag processing sites must be cleaved by substituting the P1-position amino acid at each processing site with Ile.  A mutation that blocks cleavage at the MA/CA processing site (Y132I) displayed a strong transdominant effect when tested in a phenotypic mixing strategy, inhibiting virion infectivity with a 50% inhibitory concn. of only 4% of the mutant relative to the wild type.  This mutation is 10- to 20-fold more potent in phenotypic mixing than an inactivating mutation in the viral protease, the target of many successful inhibitors, and more potent than an inactivating mutation at any of the other Gag cleavage sites.  The transdominant effect is manifested as the assembly of an aberrant virion core.  Virus contg. 20% of the Y132I mutant and 80% of the wild type (to assess the transdominant effect on infectivity) was blocked either before reverse transcription (RT) or at an early RT step.  The ability of a small amt. of the MA/CA fusion protein to poison the oligomeric assembly of infectious virus identifies an essential step in the complex process of virion formation and maturation.  The effect of a small-mol. inhibitor that is able to block MA/CA cleavage even partially would be amplified by this transdominant neg. effect on the highly orchestrated process of virion assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLsJRGkg0icbVg90H21EOLACvtfcHk0lhKbsrjD5vsyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFahur3K&md5=34901ec3b0bac668f50a3d36bc0b341b</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1128%2FJVI.00317-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00317-09%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DS.%2BK.%26aulast%3DHarris%26aufirst%3DJ.%26aulast%3DSwanstrom%26aufirst%3DR.%26atitle%3DA%2520strongly%2520transdominant%2520mutation%2520in%2520the%2520human%2520immunodeficiency%2520virus%2520type%25201%2520gag%2520gene%2520defines%2520an%2520Achilles%2520heel%2520in%2520the%2520virus%2520life%2520cycle%26jtitle%3DJ.%2520Virol.%26date%3D2009%26volume%3D83%26spage%3D8536%26epage%3D8543%26doi%3D10.1128%2FJVI.00317-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lockbaum, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leidner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusere, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosovrasti, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nachum, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nalivaika, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yilmaz, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Structural Adaptation of Darunavir Analogues against Primary Mutations in HIV-1 Protease</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">316</span>– <span class="NLM_lpage">325</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.8b00336</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.8b00336" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFWrsLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&pages=316-325&author=G.+J.+Lockbaumauthor=F.+Leidnerauthor=L.+N.+Rusereauthor=M.+Henesauthor=K.+Kosovrastiauthor=G.+S.+Nachumauthor=E.+A.+Nalivaikaauthor=A.+Aliauthor=N.+K.+Yilmazauthor=C.+A.+Schiffer&title=Structural+Adaptation+of+Darunavir+Analogues+against+Primary+Mutations+in+HIV-1+Protease&doi=10.1021%2Facsinfecdis.8b00336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Adaptation of Darunavir Analogues against Primary Mutations in HIV-1 Protease</span></div><div class="casAuthors">Lockbaum, Gordon J.; Leidner, Florian; Rusere, Linah N.; Henes, Mina; Kosovrasti, Klajdi; Nachum, Gily S.; Nalivaika, Ellen A.; Ali, Akbar; Yilmaz, Nese Kurt; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">316-325</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HIV-1 protease is one of the prime targets of agents used in antiretroviral therapy against HIV.  However, under selective pressure of protease inhibitors, primary mutations at the active site weaken inhibitor binding to confer resistance.  Darunavir (DRV) is the most potent HIV-1 protease inhibitor in clinic; resistance is limited, as DRV fits well within the substrate envelope.  Nevertheless, resistance is obsd. due to hydrophobic changes at residues including I50, V82, and I84 that line the S1/S1' pocket within the active site.  Through enzyme inhibition assays and a series of 12 crystal structures, the authors interrogated susceptibility of DRV and two potent analogs to primary S1' mutations.  The analogs had modifications at the hydrophobic P1' moiety compared to DRV to better occupy the unexploited space in the S1' pocket where the primary mutations were located.  Considerable losses of potency were obsd. against protease variants with I84V and I50V mutations for all three inhibitors.  The crystal structures revealed an unexpected conformational change in the flap region of I50V protease bound to the analog with the largest P1' moiety, indicating interdependency between the S1' subsite and the flap region.  Collective anal. of protease-inhibitor interactions in the crystal structures using principle component anal. was able to distinguish inhibitor identity and relative potency solely based on van der Waals contacts.  The authors' results reveal the complexity of the interplay between inhibitor P1' moiety and S1' mutations and validate principle component analyses as a useful tool for distinguishing resistance and inhibitor potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpimZoVRCvWL7Vg90H21EOLACvtfcHk0lhKbsrjD5vsyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFWrsLfJ&md5=498898235e84267a1e528d02ff3b345a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.8b00336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.8b00336%26sid%3Dliteratum%253Aachs%26aulast%3DLockbaum%26aufirst%3DG.%2BJ.%26aulast%3DLeidner%26aufirst%3DF.%26aulast%3DRusere%26aufirst%3DL.%2BN.%26aulast%3DHenes%26aufirst%3DM.%26aulast%3DKosovrasti%26aufirst%3DK.%26aulast%3DNachum%26aufirst%3DG.%2BS.%26aulast%3DNalivaika%26aufirst%3DE.%2BA.%26aulast%3DAli%26aufirst%3DA.%26aulast%3DYilmaz%26aufirst%3DN.%2BK.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DStructural%2520Adaptation%2520of%2520Darunavir%2520Analogues%2520against%2520Primary%2520Mutations%2520in%2520HIV-1%2520Protease%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2019%26volume%3D5%26spage%3D316%26epage%3D325%26doi%3D10.1021%2Facsinfecdis.8b00336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">[20] Processing of X-ray diffraction data collected in oscillation mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=%5B20%5D+Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lhKbsrjD5vsyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3D%255B20%255D%2520Processing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lh98X2RiD7jcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lh98X2RiD7jcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lh98X2RiD7jcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1074</span>– <span class="NLM_lpage">1080</span>, <span class="refDoi"> DOI: 10.1107/S0907444909029436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1107%2FS0907444909029436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=19770504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=1074-1080&author=N.+W.+Moriartyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adams&title=electronic+Ligand+Builder+and+Optimization+Workbench+%28eLBOW%29%3A+a+tool+for+ligand+coordinate+and+restraint+generation&doi=10.1107%2FS0907444909029436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span></div><div class="casAuthors">Moriarty, Nigel W.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The electronic Ligand Builder and Optimization Workbench (eLBOW) is a program module of the PHENIX suite of computational crystallog. software.  It is designed to be a flexible procedure that uses simple and fast quantum-chem. techniques to provide chem. accurate information for novel and known ligands alike.  A variety of input formats and options allow the attainment of a no. of diverse goals including geometry optimization and generation of restraints.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojnN4AVY_DwrVg90H21EOLACvtfcHk0lh98X2RiD7jcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN&md5=3f9a9c6c31c9a4b014a9f151fb6aa8d9</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444909029436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909029436%26sid%3Dliteratum%253Aachs%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3Delectronic%2520Ligand%2520Builder%2520and%2520Optimization%2520Workbench%2520%2528eLBOW%2529%253A%2520a%2520tool%2520for%2520ligand%2520coordinate%2520and%2520restraint%2520generation%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D1074%26epage%3D1080%26doi%3D10.1107%2FS0907444909029436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brünger, A. T.</span></span> <span> </span><span class="NLM_article-title">Free R value: a novel statistical quantity for assessing the accuracy of crystal structures</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">472</span>– <span class="NLM_lpage">475</span>, <span class="refDoi"> DOI: 10.1038/355472a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1038%2F355472a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=18481394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADyaK38XktFOrsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=1992&pages=472-475&author=A.+T.+Br%C3%BCnger&title=Free+R+value%3A+a+novel+statistical+quantity+for+assessing+the+accuracy+of+crystal+structures&doi=10.1038%2F355472a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Free R value:  a novel statistical quantity for assessing the accuracy of crystal structures</span></div><div class="casAuthors">Bruenger, Axel T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6359</span>),
    <span class="NLM_cas:pages">472-5</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">A reliable and unbiased indicator of the accuracy is proposed for fitting at. models for macromol. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmCvKGOC-AKrVg90H21EOLACvtfcHk0lgRBz0lt7rSdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XktFOrsrc%253D&md5=a382d593937b2394d9354ccba49cdbc3</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2F355472a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F355472a0%26sid%3Dliteratum%253Aachs%26aulast%3DBr%25C3%25BCnger%26aufirst%3DA.%2BT.%26atitle%3DFree%2520R%2520value%253A%2520a%2520novel%2520statistical%2520quantity%2520for%2520assessing%2520the%2520accuracy%2520of%2520crystal%2520structures%26jtitle%3DNature%26date%3D1992%26volume%3D355%26spage%3D472%26epage%3D475%26doi%3D10.1038%2F355472a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leaver-Fay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Block, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snoeyink, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: all-atom contacts and structure validation for proteins and nucleic acids</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">W375</span>– <span class="NLM_lpage">W383</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1093%2Fnar%2Fgkm216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=17452350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A280%3ADC%252BD2svhsVGhtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W375-W383&author=I.+W.+Davisauthor=A.+Leaver-Fayauthor=V.+B.+Chenauthor=J.+N.+Blockauthor=G.+J.+Kapralauthor=X.+Wangauthor=L.+W.+Murrayauthor=W.+B.+Arendallauthor=J.+Snoeyinkauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+contacts+and+structure+validation+for+proteins+and+nucleic+acids&doi=10.1093%2Fnar%2Fgkm216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom contacts and structure validation for proteins and nucleic acids</span></div><div class="casAuthors">Davis Ian W; Leaver-Fay Andrew; Chen Vincent B; Block Jeremy N; Kapral Gary J; Wang Xueyi; Murray Laura W; Arendall W Bryan 3rd; Snoeyink Jack; Richardson Jane S; Richardson David C</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic acids research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Web Server issue</span>),
    <span class="NLM_cas:pages">W375-83</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">MolProbity is a general-purpose web server offering quality validation for 3D structures of proteins, nucleic acids and complexes.  It provides detailed all-atom contact analysis of any steric problems within the molecules as well as updated dihedral-angle diagnostics, and it can calculate and display the H-bond and van der Waals contacts in the interfaces between components.  An integral step in the process is the addition and full optimization of all hydrogen atoms, both polar and nonpolar.  New analysis functions have been added for RNA, for interfaces, and for NMR ensembles.  Additionally, both the web site and major component programs have been rewritten to improve speed, convenience, clarity and integration with other resources.  MolProbity results are reported in multiple forms: as overall numeric scores, as lists or charts of local problems, as downloadable PDB and graphics files, and most notably as informative, manipulable 3D kinemage graphics shown online in the KiNG viewer.  This service is available free to all users at http://molprobity.biochem.duke.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRuyhuetoNS59bHMBt0mMIXfW6udTcc2ebYBX44edUctbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svhsVGhtQ%253D%253D&md5=726ca1e7e162ab2a4a05048f493731fb</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm216%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DLeaver-Fay%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DBlock%26aufirst%3DJ.%2BN.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DSnoeyink%26aufirst%3DJ.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520contacts%2520and%2520structure%2520validation%2520for%2520proteins%2520and%2520nucleic%2520acids%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW375%26epage%3DW383%26doi%3D10.1093%2Fnar%2Fgkm216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">PyMOL</span> <i>The PyMOL Molecular Graphics System</i>, Version 2.3; <span class="NLM_publisher-name">Schrödinger, LLC</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=PyMOL+The+PyMOL+Molecular+Graphics+System%2C+Version+2.3%3B+Schr%C3%B6dinger%2C+LLC."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Schrödinger</span> <i>Schrödinger Release 2019–2</i>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York, NY, United States</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=Schr%C3%B6dinger&title=Schr%C3%B6dinger+Release+2019%E2%80%932"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26btitle%3DSchr%25C3%25B6dinger%2520Release%25202019%25E2%2580%25932%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maple, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reboul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lupyan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlgren, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaus, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerutti, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krilov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">OPLS3: a force field providing broad coverage of drug-like small molecules and proteins</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">281</span>– <span class="NLM_lpage">296</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.5b00864</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00864" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVCjtbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=281-296&author=E.+Harderauthor=W.+Dammauthor=J.+Mapleauthor=C.+Wuauthor=M.+Reboulauthor=J.+Y.+Xiangauthor=L.+Wangauthor=D.+Lupyanauthor=M.+K.+Dahlgrenauthor=J.+L.+Knightauthor=J.+W.+Kausauthor=D.+S.+Ceruttiauthor=G.+Krilovauthor=W.+L.+Jorgensenauthor=R.+Abelauthor=R.+A.+Friesner&title=OPLS3%3A+a+force+field+providing+broad+coverage+of+drug-like+small+molecules+and+proteins&doi=10.1021%2Facs.jctc.5b00864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">OPLS3: A Force Field Providing Broad Coverage of Drug-like Small Molecules and Proteins</span></div><div class="casAuthors">Harder, Edward; Damm, Wolfgang; Maple, Jon; Wu, Chuanjie; Reboul, Mark; Xiang, Jin Yu; Wang, Lingle; Lupyan, Dmitry; Dahlgren, Markus K.; Knight, Jennifer L.; Kaus, Joseph W.; Cerutti, David S.; Krilov, Goran; Jorgensen, William L.; Abel, Robert; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">281-296</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The parametrization and validation of the OPLS3 force field for small mols. and proteins are reported.  Enhancements with respect to the previous version (OPLS2.1) include the addn. of off-atom charge sites to represent halogen bonding and aryl nitrogen lone pairs as well as a complete refit of peptide dihedral parameters to better model the native structure of proteins.  To adequately cover medicinal chem. space, OPLS3 employs over an order of magnitude more ref. data and assocd. parameter types relative to other commonly used small mol. force fields (e.g., MMFF and OPLS_2005).  As a consequence, OPLS3 achieves a high level of accuracy across performance benchmarks that assess small mol. conformational propensities and solvation.  The newly fitted peptide dihedrals lead to significant improvements in the representation of secondary structure elements in simulated peptides and native structure stability over a no. of proteins.  Together, the improvements made to both the small mol. and protein force field lead to a high level of accuracy in predicting protein-ligand binding measured over a wide range of targets and ligands (less than 1 kcal/mol RMS error) representing a 30% improvement over earlier variants of the OPLS force field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY_jrOhcBRgrVg90H21EOLACvtfcHk0li6yIvaKLy8WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVCjtbfE&md5=42663f8cfa84b80a67132bbb13b9b7ce</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00864%26sid%3Dliteratum%253Aachs%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DMaple%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DReboul%26aufirst%3DM.%26aulast%3DXiang%26aufirst%3DJ.%2BY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLupyan%26aufirst%3DD.%26aulast%3DDahlgren%26aufirst%3DM.%2BK.%26aulast%3DKnight%26aufirst%3DJ.%2BL.%26aulast%3DKaus%26aufirst%3DJ.%2BW.%26aulast%3DCerutti%26aufirst%3DD.%2BS.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DOPLS3%253A%2520a%2520force%2520field%2520providing%2520broad%2520coverage%2520of%2520drug-like%2520small%2520molecules%2520and%2520proteins%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D281%26epage%3D296%26doi%3D10.1021%2Facs.jctc.5b00864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nalam, M. N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, G. S. K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chellappan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kairys, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Özen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tidor, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rana, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Evaluating the substrate-envelope hypothesis: structural analysis of novel HIV-1 protease inhibitors designed to be robust against drug resistance</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">5368</span>– <span class="NLM_lpage">5378</span>, <span class="refDoi"> DOI: 10.1128/JVI.02531-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1128%2FJVI.02531-09" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=20237088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmtVWgtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2010&pages=5368-5378&author=M.+N.+L.+Nalamauthor=A.+Aliauthor=M.+D.+Altmanauthor=G.+S.+K.+K.+Reddyauthor=S.+Chellappanauthor=V.+Kairysauthor=A.+%C3%96zenauthor=H.+Caoauthor=M.+K.+Gilsonauthor=B.+Tidorauthor=T.+M.+Ranaauthor=C.+A.+Schiffer&title=Evaluating+the+substrate-envelope+hypothesis%3A+structural+analysis+of+novel+HIV-1+protease+inhibitors+designed+to+be+robust+against+drug+resistance&doi=10.1128%2FJVI.02531-09"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluating the substrate-envelope hypothesis: structural analysis of novel HIV-1 protease inhibitors designed to be robust against drug resistance</span></div><div class="casAuthors">Nalam, Madhavi N. L.; Ali, Akbar; Altman, Michael D.; Reddy, G. S. Kiran Kumar; Chellappan, Sripriya; Kairys, Visvaldas; Ozen, Aysegul; Cao, Hong; Gilson, Michael K.; Tidor, Bruce; Rana, Tariq M.; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5368-5378</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Drug resistance mutations in HIV-1 protease selectively alter inhibitor binding without significantly affecting substrate recognition and cleavage.  This alteration in mol. recognition led us to develop the substrate-envelope hypothesis which predicts that HIV-1 protease inhibitors that fit within the overlapping consensus vol. of the substrates are less likely to be susceptible to drug-resistant mutations, as a mutation impacting such inhibitors would simultaneously impact the processing of substrates.  To evaluate this hypothesis, over 130 HIV-1 protease inhibitors were designed and synthesized using three different approaches with and without substrate-envelope constraints.  A subset of 16 representative inhibitors with binding affinities to wild-type protease ranging from 58 nM to 0.8 pM was chosen for crystallog. anal.  The inhibitor-protease complexes revealed that tightly binding inhibitors (at the picomolar level of affinity) appear to "lock" into the protease active site by forming hydrogen bonds to particular active-site residues.  Both this hydrogen bonding pattern and subtle variations in protein-ligand van der Waals interactions distinguish nanomolar from picomolar inhibitors.  In general, inhibitors that fit within the substrate envelope, regardless of whether they are picomolar or nanomolar, have flatter profiles with respect to drug-resistant protease variants than inhibitors that protrude beyond the substrate envelope; this provides a strong rationale for incorporating substrate-envelope constraints into structure-based design strategies to develop new HIV-1 protease inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1FdWkZ1EQDLVg90H21EOLACvtfcHk0li6yIvaKLy8WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmtVWgtrc%253D&md5=09e55d24a9ff7382cefd2fccc5206ada</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1128%2FJVI.02531-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02531-09%26sid%3Dliteratum%253Aachs%26aulast%3DNalam%26aufirst%3DM.%2BN.%2BL.%26aulast%3DAli%26aufirst%3DA.%26aulast%3DAltman%26aufirst%3DM.%2BD.%26aulast%3DReddy%26aufirst%3DG.%2BS.%2BK.%2BK.%26aulast%3DChellappan%26aufirst%3DS.%26aulast%3DKairys%26aufirst%3DV.%26aulast%3D%25C3%2596zen%26aufirst%3DA.%26aulast%3DCao%26aufirst%3DH.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26aulast%3DTidor%26aufirst%3DB.%26aulast%3DRana%26aufirst%3DT.%2BM.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DEvaluating%2520the%2520substrate-envelope%2520hypothesis%253A%2520structural%2520analysis%2520of%2520novel%2520HIV-1%2520protease%2520inhibitors%2520designed%2520to%2520be%2520robust%2520against%2520drug%2520resistance%26jtitle%3DJ.%2520Virol.%26date%3D2010%26volume%3D84%26spage%3D5368%26epage%3D5378%26doi%3D10.1128%2FJVI.02531-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leidner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurt Yilmaz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Hydration structure and dynamics of inhibitor-bound HIV-1 protease</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2784</span>– <span class="NLM_lpage">2796</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.8b00097</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.8b00097" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlvVKmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=2784-2796&author=F.+Leidnerauthor=N.+Kurt+Yilmazauthor=J.+Paulsenauthor=Y.+A.+Mullerauthor=C.+A.+Schiffer&title=Hydration+structure+and+dynamics+of+inhibitor-bound+HIV-1+protease&doi=10.1021%2Facs.jctc.8b00097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Hydration Structure and Dynamics of Inhibitor-Bound HIV-1 Protease</span></div><div class="casAuthors">Leidner, Florian; Kurt Yilmaz, Nese; Paulsen, Janet; Muller, Yves A.; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2784-2796</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Water is an essential in many biol. processes, and the hydration structure plays a crit. role in facilitating protein folding, dynamics and ligand binding.  A variety of biophys. spectroscopic techniques have been used to probe the water solvating proteins, often complemented with mol. dynamics (MD) simulations to resolve the spatial and dynamic features of the hydration shell, but comparing relative water structure is challenging.  In this study 1 μs MD simulations were performed to identify and characterize hydration sites around HIV-1 protease bound to an inhibitor, darunavir (DRV).  The water d., hydration site occupancy, extent and anisotropy of fluctuations, coordinated water mols., and hydrogen bonds were characterized and compared to the properties of bulk water.  The water d. of the principal hydration shell was found to be higher than bulk, dependent on the topol. and physio-chem. identity of the biomol. surface.  The dynamics of water mols. occupying principal hydration sites was highly dependent on the no. of water-water interactions, and inversely correlated with hydrogen bonds to the protein-inhibitor complex.  While many waters were conserved following the symmetry of homodimeric HIV protease, the asymmetry induced by DRV resulted in asym. lower-occupancy hydration sites at the concave surface of the active site.  Key interactions between water mols. and the protease, that stabilize the protein in the inhibited form, were altered in a drug resistant variant of the protease indicating that modulation of solvent-solute interactions might play a key role in conveying drug resistance.  Our anal. provides insights into the interplay between an enzyme inhibitor complex and the hydration shell and has implications in elucidating water structure in a variety of biol. processes and applications including ligand binding, inhibitor design and resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-XstjSseAurVg90H21EOLACvtfcHk0li6yIvaKLy8WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlvVKmsbs%253D&md5=f8801ad50103c24efc2bec9dc442f9a1</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.8b00097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.8b00097%26sid%3Dliteratum%253Aachs%26aulast%3DLeidner%26aufirst%3DF.%26aulast%3DKurt%2BYilmaz%26aufirst%3DN.%26aulast%3DPaulsen%26aufirst%3DJ.%26aulast%3DMuller%26aufirst%3DY.%2BA.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DHydration%2520structure%2520and%2520dynamics%2520of%2520inhibitor-bound%2520HIV-1%2520protease%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2018%26volume%3D14%26spage%3D2784%26epage%3D2796%26doi%3D10.1021%2Facs.jctc.8b00097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockbaum, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosovrasti, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leidner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nachum, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nalivaika, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spielvogel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanstrom, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolon, D. N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurt
Yilmaz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Picomolar to micromolar: elucidating the role of distal mutations in HIV-1 protease in conferring drug resistance</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2441</span>– <span class="NLM_lpage">2452</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00370</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00370" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVOhtr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=2441-2452&author=M.+Henesauthor=G.+J.+Lockbaumauthor=K.+Kosovrastiauthor=F.+Leidnerauthor=G.+S.+Nachumauthor=E.+A.+Nalivaikaauthor=S.+K.+Leeauthor=E.+Spielvogelauthor=S.+Zhouauthor=R.+Swanstromauthor=D.+N.+A.+Bolonauthor=N.+Kurt%0AYilmazauthor=C.+A.+Schiffer&title=Picomolar+to+micromolar%3A+elucidating+the+role+of+distal+mutations+in+HIV-1+protease+in+conferring+drug+resistance&doi=10.1021%2Facschembio.9b00370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Picomolar to Micromolar: Elucidating the Role of Distal Mutations in HIV-1 Protease in Conferring Drug Resistance</span></div><div class="casAuthors">Henes, Mina; Lockbaum, Gordon J.; Kosovrasti, Klajdi; Leidner, Florian; Nachum, Gily S.; Nalivaika, Ellen A.; Lee, Sook-Kyung; Spielvogel, Ean; Zhou, Shuntai; Swanstrom, Ronald; Bolon, Daniel N. A.; Kurt Yilmaz, Nese; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2441-2452</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug resistance continues to be a growing global problem.  The efficacy of small mol. inhibitors is threatened by pools of genetic diversity in all systems, including antibacterials, antifungals, cancer therapeutics, and antivirals.  Resistant variants often include combinations of active site mutations and distal "secondary" mutations, which are thought to compensate for losses in enzymic activity.  HIV-1 protease is the ideal model system to investigate these combinations and underlying mol. mechanisms of resistance.  Darunavir (DRV) binds wild-type (WT) HIV-1 protease with a potency of <5 pM, but we have identified a protease variant that loses potency to DRV 150 000-fold, with 11 mutations in and outside the active site.  To elucidate the roles of these mutations in DRV resistance, we used a multidisciplinary approach, combining enzymic assays, crystallog., and mol. dynamics simulations.  Anal. of protease variants with 1, 2, 4, 8, 9, 10, and 11 mutations showed that the primary active site mutations caused ∼50-fold loss in potency (2 mutations), while distal mutations outside the active site further decreased DRV potency from 13 nM (8 mutations) to 0.76 μM (11 mutations).  Crystal structures and simulations revealed that distal mutations induce subtle changes that are dynamically propagated through the protease.  Our results reveal that changes remote from the active site directly and dramatically impact the potency of the inhibitor.  Moreover, we find interdependent effects of mutations in conferring high levels of resistance.  These mechanisms of resistance are likely applicable to many other quickly evolving drug targets, and the insights may have implications for the design of more robust inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2wI5Ah0DMVbVg90H21EOLACvtfcHk0li4mQXFvrz6Lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVOhtr7F&md5=cc0a728eca73920e31dfbe0098c67424</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00370%26sid%3Dliteratum%253Aachs%26aulast%3DHenes%26aufirst%3DM.%26aulast%3DLockbaum%26aufirst%3DG.%2BJ.%26aulast%3DKosovrasti%26aufirst%3DK.%26aulast%3DLeidner%26aufirst%3DF.%26aulast%3DNachum%26aufirst%3DG.%2BS.%26aulast%3DNalivaika%26aufirst%3DE.%2BA.%26aulast%3DLee%26aufirst%3DS.%2BK.%26aulast%3DSpielvogel%26aufirst%3DE.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DSwanstrom%26aufirst%3DR.%26aulast%3DBolon%26aufirst%3DD.%2BN.%2BA.%26aulast%3DKurt%2BYilmaz%26aufirst%3DN.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DPicomolar%2520to%2520micromolar%253A%2520elucidating%2520the%2520role%2520of%2520distal%2520mutations%2520in%2520HIV-1%2520protease%2520in%2520conferring%2520drug%2520resistance%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D2441%26epage%3D2452%26doi%3D10.1021%2Facschembio.9b00370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honig, B.</span></span> <span> </span><span class="NLM_article-title">On the role of the crystal environment in determining protein side-chain conformations</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>320</i></span>,  <span class="NLM_fpage">597</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1016/S0022-2836(02)00470-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2FS0022-2836%2802%2900470-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=12096912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD38XltVKmu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2002&pages=597-608&author=M.+P.+Jacobsonauthor=R.+A.+Friesnerauthor=Z.+Xiangauthor=B.+Honig&title=On+the+role+of+the+crystal+environment+in+determining+protein+side-chain+conformations&doi=10.1016%2FS0022-2836%2802%2900470-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">On the Role of the Crystal Environment in Determining Protein Side-chain Conformations</span></div><div class="casAuthors">Jacobson, Matthew P.; Friesner, Richard A.; Xiang, Zhexin; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">320</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">597-608</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">The role of crystal packing in detg. the obsd. conformations of amino acid side-chains in protein crystals is investigated by (1) anal. of a database of proteins that have been crystd. in different unit cells (space group or unit cell dimensions) and (2) theor. predictions of side-chain conformations with the crystal environment explicitly represented.  Both of these approaches indicate that the crystal environment plays an important role in detg. the conformations of polar side-chains on the surfaces of proteins.  Inclusion of the crystal environment permits a more sensitive measurement of the achievable accuracy of side-chain prediction programs, when validating against structures obtained by x-ray crystallog.  Our side-chain prediction program uses an all-atom force field and a Generalized Born model of solvation and is thus capable of modeling simple packing effects (i.e. van der Waals interactions), electrostatic effects, and desolvation, which are all important mechanisms by which the crystal environment impacts obsd. side-chain conformations.  Our results are also relevant to the understanding of changes in side-chain conformation that may result from ligand docking and protein-protein assocn., insofar as the results reveal how side-chain conformations change in response to their local environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgHEOVpIEiErVg90H21EOLACvtfcHk0li4mQXFvrz6Lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltVKmu70%253D&md5=006de6bd2d0f233ab32d6798dc1a3fbc</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900470-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900470-9%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DXiang%26aufirst%3DZ.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DOn%2520the%2520role%2520of%2520the%2520crystal%2520environment%2520in%2520determining%2520protein%2520side-chain%2520conformations%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D320%26spage%3D597%26epage%3D608%26doi%3D10.1016%2FS0022-2836%2802%2900470-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, M. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Søndergaard, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rostkowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. H.</span></span> <span> </span><span class="NLM_article-title">PROPKA3: consistent treatment of internal and surface residues in empirical pK<sub>a</sub> predictions</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">525</span>– <span class="NLM_lpage">537</span>, <span class="refDoi"> DOI: 10.1021/ct100578z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct100578z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1aqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=525-537&author=M.+H.+M.+Olssonauthor=C.+R.+S%C3%B8ndergaardauthor=M.+Rostkowskiauthor=J.+H.+Jensen&title=PROPKA3%3A+consistent+treatment+of+internal+and+surface+residues+in+empirical+pKa+predictions&doi=10.1021%2Fct100578z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">PROPKA3: Consistent Treatment of Internal and Surface Residues in Empirical pKa Predictions</span></div><div class="casAuthors">Olsson, Mats H. M.; Sondergaard, Chresten R.; Rostkowski, Michal; Jensen, Jan H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">525-537</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors have revised the rules and parameters for one of the most commonly used empirical pKa predictors, PROPKA, based on better phys. description of the desolvation and dielec. response for the protein.  The authors have introduced a new and consistent approach to interpolate the description between the previously distinct classifications into internal and surface residues, which otherwise is found to give rise to an erratic and discontinuous behavior.  Since the goal of this study is to lay out the framework and validate the concept, it focuses on Asp and Glu residues where the protein pKa values and structures are assumed to be more reliable.  The new and improved implementation is evaluated and discussed; it is found to agree better with expt. than the previous implementation (in parentheses): rmsd = 0.79 (0.91) for Asp and Glu, 0.75 (0.97) for Tyr, 0.65 (0.72) for Lys, and 1.00 (1.37) for His residues.  The most significant advance, however, is in reducing the no. of outliers and removing unreasonable sensitivity to small structural changes that arise from classifying residues as either internal or surface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm6urTZNdjY7Vg90H21EOLACvtfcHk0li4mQXFvrz6Lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1aqsA%253D%253D&md5=9b1666b1c56e1129789e62948eb4d001</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fct100578z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct100578z%26sid%3Dliteratum%253Aachs%26aulast%3DOlsson%26aufirst%3DM.%2BH.%2BM.%26aulast%3DS%25C3%25B8ndergaard%26aufirst%3DC.%2BR.%26aulast%3DRostkowski%26aufirst%3DM.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26atitle%3DPROPKA3%253A%2520consistent%2520treatment%2520of%2520internal%2520and%2520surface%2520residues%2520in%2520empirical%2520pKa%2520predictions%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D525%26epage%3D537%26doi%3D10.1021%2Fct100578z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felts, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maple, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philipp, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallicchio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, R. M.</span></span> <span> </span><span class="NLM_article-title">Integrated Modeling Program, Applied Chemical Theory (IMPACT)</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1752</span>– <span class="NLM_lpage">1780</span>, <span class="refDoi"> DOI: 10.1002/jcc.20292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1002%2Fjcc.20292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=16211539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1752-1780&author=J.+L.+Banksauthor=H.+S.+Beardauthor=Y.+Caoauthor=A.+E.+Choauthor=W.+Dammauthor=R.+Faridauthor=A.+K.+Feltsauthor=T.+A.+Halgrenauthor=D.+T.+Mainzauthor=J.+R.+Mapleauthor=R.+Murphyauthor=D.+M.+Philippauthor=M.+P.+Repaskyauthor=L.+Y.+Zhangauthor=B.+J.+Berneauthor=R.+A.+Friesnerauthor=E.+Gallicchioauthor=R.+M.+Levy&title=Integrated+Modeling+Program%2C+Applied+Chemical+Theory+%28IMPACT%29&doi=10.1002%2Fjcc.20292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated modeling program, applied chemical theory (IMPACT)</span></div><div class="casAuthors">Banks, Jay L.; Beard, Hege S.; Cao, Yixiang; Cho, Art E.; Damm, Wolfgang; Farid, Ramy; Felts, Anthony K.; Halgren, Thomas A.; Mainz, Daniel T.; Maple, Jon R.; Murphy, Robert; Philipp, Dean M.; Repasky, Matthew P.; Zhang, Linda Y.; Berne, Bruce J.; Friesner, Richard A.; Gallicchio, Emilio; Levy, Ronald M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1752-1780</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  We provide an overview of the IMPACT mol. mechanics program with an emphasis on recent developments and a description of its current functionality.  With respect to core mol. mechanics technologies we include a status report for the fixed charge and polarizable force fields that can be used with the program and illustrate how the force fields, when used together with new atom typing and parameter assignment modules, have greatly expanded the coverage of org. compds. and medicinally relevant ligands.  As we discuss in this review, explicit solvent simulations have been used to guide our design of implicit solvent models based on the generalized Born framework and a novel nonpolar estimator that have recently been incorporated into the program.  With IMPACT it is possible to use several different advanced conformational sampling algorithms based on combining features of mol. dynamics and Monte Carlo simulations.  The program includes two specialized mol. mechanics modules: Glide, a high-throughput docking program, and QSite, a mixed quantum mechanics/mol. mechanics module.  These modules employ the IMPACT infrastructure as a starting point for the construction of the protein model and assignment of mol. mechanics parameters, but have then been developed to meet specialized objectives with respect to sampling and the energy function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZX2FzcGWGM7Vg90H21EOLACvtfcHk0lj4f9DErvEN1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbI&md5=0094f15727c7fb77a0fae16ccf8292f0</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20292%26sid%3Dliteratum%253Aachs%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DCho%26aufirst%3DA.%2BE.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DFelts%26aufirst%3DA.%2BK.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DMaple%26aufirst%3DJ.%2BR.%26aulast%3DMurphy%26aufirst%3DR.%26aulast%3DPhilipp%26aufirst%3DD.%2BM.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DZhang%26aufirst%3DL.%2BY.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DGallicchio%26aufirst%3DE.%26aulast%3DLevy%26aufirst%3DR.%2BM.%26atitle%3DIntegrated%2520Modeling%2520Program%252C%2520Applied%2520Chemical%2520Theory%2520%2528IMPACT%2529%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1752%26epage%3D1780%26doi%3D10.1002%2Fjcc.20292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 1 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Mark Aldren M. Feliciano, <span class="NLM_string-name hlFld-ContribAuthor">Brian Gold</span>. </span><span class="cited-content_cbyCitation_article-title">Unique, yet Typical Oxyanion Holes in Aspartic Proteases. </span><span class="cited-content_cbyCitation_journal-name">ACS Catalysis</span><span> <strong>2020,</strong> <em>10 </em>
                                    (23)
                                     , 14201-14209. <a href="https://doi.org/10.1021/acscatal.0c03624" title="DOI URL">https://doi.org/10.1021/acscatal.0c03624</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscatal.0c03624&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscatal.0c03624%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Catalysis%26atitle%3DUnique%25252C%252Byet%252BTypical%252BOxyanion%252BHoles%252Bin%252BAspartic%252BProteases%26aulast%3DFeliciano%26aufirst%3DMark%2BAldren%2BM.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D18082020%26date%3D19102020%26date%3D20112020%26volume%3D10%26issue%3D23%26spage%3D14201%26epage%3D14209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of HIV-1 protease inhibitors darunavir (DRV) (<b>1</b>), lopinavir (LPV) (<b>2</b>), atazanavir (ATV) (<b>3</b>), previously reported hybrid compounds <b>4</b> and <b>5</b>, and new hybrid compounds designed by incorporating the bis-THF moiety of DRV into the pseudosymmetric Phe-Phe dipeptide isostere of LPV at the P2 (<b>12a</b>–<b>h</b>) or P2′ position (<b>15a</b>–<b>c</b>), in combination with an ATV-like hydrophobic amino acid moiety at the P2′ or P2 position, respectively. The canonical nomenclature is used to indicate the position of each inhibitor moiety.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Protease Inhibitors Incorporating Bis-tetrahydrofuran in a Pseudosymmetric Phe-Phe Dipeptide Isostere<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) K<sub>2</sub>CO<sub>3</sub>, KOH, BnCl, H<sub>2</sub>O, 100 °C, 5 h; (b) NaNH<sub>2</sub>, CH<sub>3</sub>CN, MTBE, 0 °C, 2 h, then BnMgCl (2 M in THF), RT, 14 h; (c) NaBH<sub>4</sub>, MsOH, <i>i</i>-PrOH, EGDME, 0 °C, 12 h; (d) NaBH<sub>4</sub>, TEA, DMA, 0–15 °C, 2 h; (e) DIEA, CH<sub>3</sub>CN, RT, 24 h; (f) Pd/C, HCO<sub>2</sub>NH<sub>4</sub>, MeOH, 50 °C, 12 h; (g) <i>N</i>-(methoxycarbonyl)-capped amino acid, HATU, DIEA, DMF, RT, 4 h.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Overlays of protease cocrystal structures bound to LPV (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJB">6PJB</a>) (gray, top) and DRV (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DGX">6DGX</a>) (brown, bottom) with compounds <b>12a</b> (bis-THF at P2; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJD">6PJD</a>) (green), <b>15a</b> (bis-THF at P2′; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJM">6PJM</a>) (orange), and <b>4</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJC">6PJC</a>) (salmon) revealed similar binding conformations of the Phe-Phe isostere and the bis-THF moiety as observed in the protease complexes with LPV and DRV, respectively. Note that DRV is flipped 180° to compare with bis-THF of <b>15a</b> at the P2′ position.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structures of wild-type HIV-1 protease in complex with LPV (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJB">6PJB</a>) (A) and hybrid compounds <b>4</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJC">6PJC</a>) (B), <b>12a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJD">6PJD</a>) (C), and <b>15a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJM">6PJM</a>) (D). The bis-THF moiety maintains the hydrogen bonding interactions with the main-chain NH of Asp29/Asp29′ and Asp30/Asp30′ in the S2/S2′ subsites regardless of its position on the Phe-Phe core. Hydrogen bonds and water-mediated interactions as determined by PyMol are shown as black dashed lines with waters shown as red spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Superposition of HIV-1 protease complexes with compounds <b>12a</b>–<b>h</b> (A) and compounds <b>15a</b>–<b>c</b> (B), showing similar binding conformations of the bis-THF moiety regardless of the position and minor differences at the Phe-Phe isostere but major differences at the P2/P2′ amino acid moiety. The protease active site is shown in a ribbon representation, with bound inhibitors depicted as sticks.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comparison of the binding conformations of the bis-THF moiety and protease–inhibitor intramolecular distances with the main-chain NH of Asp29 and Asp30 in the S2/S2′ subsite of HIV-1 protease. As observed in the DRV–protease complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DGX">6DGX</a>), the bis-THF moiety retained similar interactions with the protease in complexes with compound <b>4</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJC">6PJC</a>) (top), the most potent hybrid compounds <b>12a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJD">6PJD</a>) and <b>15a</b> (PBD <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJM">6PJM</a>) (middle), and the least potent hybrid compounds <b>12b</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJE">6PJE</a>) and <b>15b</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJN">6PJN</a>) (bottom), irrespective of the position on the Phe-Phe core.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Comparison of protease–inhibitor direct hydrogen bonds and water-mediated interactions in the S2/S2′ subsites for the cyclic urea moiety of LPV (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJB">6PJB</a>) (A), <i>N</i>-(methoxycarbonyl)-<i>tert</i>-leucine of ATV (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EKY">3EKY</a>) (B), and the <i>N</i>-(methoxycarbonyl)-capped amino acid moieties of hybrid compound <b>15a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJM">6PJM</a>) (C), <b>12a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJD">6PJD</a>) (D), <b>15b</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJN">6PJN</a>) (E), and <b>12b</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJE">6PJE</a>) (F). Polar interactions are shown as black dashed lines (as determined by PyMol) with waters shown as red spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Comparison of binding conformation (A) and van der Waals packing with residues in the S2′ subsite for the most and least potent inhibitors <b>12a</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJD">6PJD</a>) (B, green sticks) and <b>12b</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJE">6PJE</a>) (C, blue sticks). Compound <b>12a</b> makes more extensive contacts with the protease as estimated by relative vdW contact energy than <b>12b</b> (−91 versus −86 kcal/mol), with a major difference occurring at the S2′ subsite (−27 versus −23 kcal/mol). Protease residues are colored red to blue for highest to lowest vdW interactions with the inhibitor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Inhibitor root-mean-square fluctuation (RMSF) of each non-hydrogen atom of DRV, LPV, and hybrid compounds mapped onto their crystal structures. Warmer colors indicate larger fluctuations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. The dihedral angle (ϕ<sub>5</sub>) responsible for conformational sampling of the <i>tert</i>-leucine moiety on <b>12a</b> (A) and <b>15a</b> (B) is associated with increased Ile50-Ile50′ distance sampling.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/medium/jm0c00529_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0002.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Protease–inhibitor hydrogen bond frequencies from MD simulations mapped onto the cocrystal structures of compounds <b>12a</b> (A), <b>12b</b> (B), <b>15a</b> (C), and <b>15b</b> (D). The protease side chains and inhibitors are shown as sticks; chains A and B are colored cyan and magenta, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00529/20200806/images/large/jm0c00529_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00529&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88772" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88772" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 59 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cihlar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fordyce, M.</span></span> <span> </span><span class="NLM_article-title">Current status and prospects of HIV treatment</span>. <i>Curr. Opin. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">50</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1016/j.coviro.2016.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fj.coviro.2016.03.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=27023283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC28Xksl2nsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2016&pages=50-56&author=T.+Cihlarauthor=M.+Fordyce&title=Current+status+and+prospects+of+HIV+treatment&doi=10.1016%2Fj.coviro.2016.03.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Current status and prospects of HIV treatment</span></div><div class="casAuthors">Cihlar, Tomas; Fordyce, Marshall</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Virology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">50-56</span>CODEN:
                <span class="NLM_cas:coden">COVUAF</span>;
        ISSN:<span class="NLM_cas:issn">1879-6257</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">Current antiviral treatments can reduce HIV-assocd. morbidity, prolong survival, and prevent HIV transmission.  Combination antiretroviral therapy (cART) contg. preferably three active drugs from two or more classes is required for durable virol. suppression.  Regimen selection is based on virol. efficacy, potential for adverse effects, pill burden and dosing frequency, drug-drug interaction potential, resistance test results, comorbid conditions, social status, and cost.  With prolonged virol. suppression, improved clin. outcomes, and longer survival, patients will be exposed to antiretroviral agents for decades.  Therefore, maximizing the safety and tolerability of cART is a high priority.  Emergence of resistance and/or lack of tolerability in individual patients require availability of a range of treatment options.  Development of new drugs is focused on improving safety (e.g. tenofovir alafenamide) and/or resistance profile (e.g. doravirine) within the existing drug classes, combination therapies with improved adherence (e.g. single-tablet regimens), novel mechanisms of action (e.g. attachment inhibitors, maturation inhibitors, broadly neutralizing antibodies), and treatment simplification with infrequent dosing (e.g. long-acting injectables).  In parallel with cART innovations, research and development efforts focused on agents that target persistent HIV reservoirs may lead to prolonged drug-free remission and HIV cure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpHDvTzsN1MbVg90H21EOLACvtfcHk0lgNCRLW-_Xkhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xksl2nsrw%253D&md5=8f134da24f9491b3b2ac68bf7086de0f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.coviro.2016.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coviro.2016.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DCihlar%26aufirst%3DT.%26aulast%3DFordyce%26aufirst%3DM.%26atitle%3DCurrent%2520status%2520and%2520prospects%2520of%2520HIV%2520treatment%26jtitle%3DCurr.%2520Opin.%2520Virol.%26date%3D2016%26volume%3D18%26spage%3D50%26epage%3D56%26doi%3D10.1016%2Fj.coviro.2016.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saag, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benson, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandhi, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoy, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landovitz, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mugavero, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sax, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchbinder, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Rio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eron, J. J.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fatkenheuer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunthard, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volberding, P. A.</span></span> <span> </span><span class="NLM_article-title">Antiretroviral drugs for treatment and prevention of HIV infection in adults: 2018 recommendations of the international antiviral society-USA panel</span>. <i>JAMA</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>320</i></span>,  <span class="NLM_fpage">379</span>– <span class="NLM_lpage">396</span>, <span class="refDoi"> DOI: 10.1001/jama.2018.8431</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1001%2Fjama.2018.8431" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=30043070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVehtrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2018&pages=379-396&author=M.+S.+Saagauthor=C.+A.+Bensonauthor=R.+T.+Gandhiauthor=J.+F.+Hoyauthor=R.+J.+Landovitzauthor=M.+J.+Mugaveroauthor=P.+E.+Saxauthor=D.+M.+Smithauthor=M.+A.+Thompsonauthor=S.+P.+Buchbinderauthor=C.+Del+Rioauthor=J.+J.+Eronauthor=G.+Fatkenheuerauthor=H.+F.+Gunthardauthor=J.-M.+Molinaauthor=D.+M.+Jacobsenauthor=P.+A.+Volberding&title=Antiretroviral+drugs+for+treatment+and+prevention+of+HIV+infection+in+adults%3A+2018+recommendations+of+the+international+antiviral+society-USA+panel&doi=10.1001%2Fjama.2018.8431"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Antiretroviral drugs for treatment and prevention of HIV infection in adults: 2018 recommendations of the International Antiviral Society-USA Panel</span></div><div class="casAuthors">Saag, Michael S.; Benson, Constance A.; Gandhi, Rajesh T.; Hoy, Jennifer F.; Landovitz, Raphael J.; Mugavero, Michael J.; Sax, Paul E.; Smith, Davey M.; Thompson, Melanie A.; Buchbinder, Susan P.; del Rio, Carlos; Eron, Joseph J., Jr.; Fatkenheuer, Gerd; Gunthard, Huldrych F.; Molina, Jean-Michel; Jacobsen, Donna M.; Volberding, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">JAMA, the Journal of the American Medical Association</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">320</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">379-396</span>CODEN:
                <span class="NLM_cas:coden">JAMAAP</span>;
        ISSN:<span class="NLM_cas:issn">1538-3598</span>.
    
            (<span class="NLM_cas:orgname">American Medical Association</span>)
        </div><div class="casAbstract">Importance Antiretroviral therapy (ART) is the cornerstone of prevention and management of HIV infection. objective To evaluate new data and treatments and incorporate this information into updated recommendations for initiating therapy, monitoring individuals starting therapy, changing regimens, and preventing HIV infection for individuals at risk. evidence review New evidence collected since the International Antiviral Society-USA 2016 recommendations via monthly PubMed and EMBASE literature searches up to Apr. 2018; data presented at peer-reviewed scientific conferences.  A volunteer panel of experts in HIV research and patient care considered these data and updated previous recommendations. findings ART is recommended for virtually all HIV-infected individuals, as soon as possible after HIV diagnosis.  Immediate initiation (eg, rapid start), if clin. appropriate, requires adequate staffing, specialized services, and careful selection of medical therapy.  An integrase strand transfer inhibitor (InSTI) plus 2 nucleoside reverse transcriptase inhibitors (NRTIs) is generally recommended for initial therapy, with unique patient circumstances (eg, concomitant diseases and conditions, potential for pregnancy, cost) guiding the treatment choice.  CD4 cell count, HIV RNA level, genotype, and other lab. tests for general health and co-infections are recommended at specified points before and during ART.  If a regimen switch is indicated, treatment history, tolerability, adherence, and drug resistance history should first be assessed; 2 or 3 active drugs are recommended for a new regimen.  HIV testing is recommended at least once for anyone who has ever been sexually active and more often for individuals at ongoing risk for infection.  Preexposure prophylaxis with tenofovir disoproxil fumarate/emtricitabine and appropriate monitoring is recommended for individuals at risk for HIV. conclusions and relevance Advances in HIV prevention and treatment with antiretroviral drugs continue to improve clin. management and outcomes for individuals at risk for and living with HIV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZMBk4meAznbVg90H21EOLACvtfcHk0ljJ8z6o6RBdtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVehtrrJ&md5=d2b4b70b43f71b3900fcd80abb54755d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1001%2Fjama.2018.8431&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Fjama.2018.8431%26sid%3Dliteratum%253Aachs%26aulast%3DSaag%26aufirst%3DM.%2BS.%26aulast%3DBenson%26aufirst%3DC.%2BA.%26aulast%3DGandhi%26aufirst%3DR.%2BT.%26aulast%3DHoy%26aufirst%3DJ.%2BF.%26aulast%3DLandovitz%26aufirst%3DR.%2BJ.%26aulast%3DMugavero%26aufirst%3DM.%2BJ.%26aulast%3DSax%26aufirst%3DP.%2BE.%26aulast%3DSmith%26aufirst%3DD.%2BM.%26aulast%3DThompson%26aufirst%3DM.%2BA.%26aulast%3DBuchbinder%26aufirst%3DS.%2BP.%26aulast%3DDel%2BRio%26aufirst%3DC.%26aulast%3DEron%26aufirst%3DJ.%2BJ.%26aulast%3DFatkenheuer%26aufirst%3DG.%26aulast%3DGunthard%26aufirst%3DH.%2BF.%26aulast%3DMolina%26aufirst%3DJ.-M.%26aulast%3DJacobsen%26aufirst%3DD.%2BM.%26aulast%3DVolberding%26aufirst%3DP.%2BA.%26atitle%3DAntiretroviral%2520drugs%2520for%2520treatment%2520and%2520prevention%2520of%2520HIV%2520infection%2520in%2520adults%253A%25202018%2520recommendations%2520of%2520the%2520international%2520antiviral%2520society-USA%2520panel%26jtitle%3DJAMA%26date%3D2018%26volume%3D320%26spage%3D379%26epage%3D396%26doi%3D10.1001%2Fjama.2018.8431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Panel on Antiretroviral Guidelines
for Adults and Adolescents</span>.  <i>Guidelines
for the Use of Antiretroviral Agents in Adults and Adolescents with
HIV</i>; Department of Health and Human Services. Available at <a href="https://files.aidsinfo.nih.gov/contentfiles/lvguidelines/AdultandAdolescentGL.pdf" class="extLink">https://files.aidsinfo.nih.gov/contentfiles/lvguidelines/AdultandAdolescentGL.pdf</a>. (Accessed February 15, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Panel+on+Antiretroviral+Guidelines%0Afor+Adults+and+Adolescents.+Guidelines%0Afor+the+Use+of+Antiretroviral+Agents+in+Adults+and+Adolescents+with%0AHIV%3B+Department+of+Health+and+Human+Services.+Available+at+https%3A%2F%2Ffiles.aidsinfo.nih.gov%2Fcontentfiles%2Flvguidelines%2FAdultandAdolescentGL.pdf.+%28Accessed+February+15%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DGuidelines%250Afor%2520the%2520Use%2520of%2520Antiretroviral%2520Agents%2520in%2520Adults%2520and%2520Adolescents%2520with%250AHIV" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandaranayake, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murzycki, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nalam, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nalivaika, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Özen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prabu-Jeyabalan, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thayer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Molecular basis for drug resistance in HIV-1 protease</span>. <i>Viruses</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2509</span>– <span class="NLM_lpage">2535</span>, <span class="refDoi"> DOI: 10.3390/v2112509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.3390%2Fv2112509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=21994628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFKjsrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=2509-2535&author=A.+Aliauthor=R.+M.+Bandaranayakeauthor=Y.+Caiauthor=N.+M.+Kingauthor=M.+Kolliauthor=S.+Mittalauthor=J.+F.+Murzyckiauthor=M.+N.+Nalamauthor=E.+A.+Nalivaikaauthor=A.+%C3%96zenauthor=M.+M.+Prabu-Jeyabalanauthor=K.+Thayerauthor=C.+A.+Schiffer&title=Molecular+basis+for+drug+resistance+in+HIV-1+protease&doi=10.3390%2Fv2112509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for drug resistance in HIV-1 protease</span></div><div class="casAuthors">Ali, Akbar; Bandaranayake, Rajintha M.; Cai, Yufeng; King, Nancy M.; Kolli, Madhavi; Mittal, Seema; Murzycki, Jennifer F.; Nalam, Madhavi N. L.; Nalivaika, Ellen A.; Ozen, Aysegul; Prabu-Jeyabalan, Moses M.; Thayer, Kelly; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Viruses</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2509-2535</span>CODEN:
                <span class="NLM_cas:coden">VIRUBR</span>;
        ISSN:<span class="NLM_cas:issn">1999-4915</span>.
    
            (<span class="NLM_cas:orgname">Molecular Diversity Preservation International</span>)
        </div><div class="casAbstract">A review.  HIV-1 protease is one of the major antiviral targets in the treatment of patients infected with HIV-1.  The nine FDA approved HIV-1 protease inhibitors were developed with extensive use of structure-based drug design, thus the at. details of how the inhibitors bind are well characterized.  From this structural understanding the mol. basis for drug resistance in HIV-1 protease can be elucidated.  Selected mutations in response to therapy and diversity between clades in HIV-1 protease have altered the shape of the active site, potentially altered the dynamics and even altered the sequence of the cleavage sites in the Gag polyprotein.  All of these interdependent changes act in synergy to confer drug resistance while simultaneously maintaining the fitness of the virus.  New strategies, such as incorporation of the substrate envelope constraint to design robust inhibitors that incorporate details of HIV-1 protease's function and decrease the probability of drug resistance, are necessary to continue to effectively target this key protein in HIV-1 life cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBf4i5iKmj8rVg90H21EOLACvtfcHk0liJzpSk3WiA-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFKjsrzL&md5=3c5a791d1277a18b800c5849403333b5</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3390%2Fv2112509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fv2112509%26sid%3Dliteratum%253Aachs%26aulast%3DAli%26aufirst%3DA.%26aulast%3DBandaranayake%26aufirst%3DR.%2BM.%26aulast%3DCai%26aufirst%3DY.%26aulast%3DKing%26aufirst%3DN.%2BM.%26aulast%3DKolli%26aufirst%3DM.%26aulast%3DMittal%26aufirst%3DS.%26aulast%3DMurzycki%26aufirst%3DJ.%2BF.%26aulast%3DNalam%26aufirst%3DM.%2BN.%26aulast%3DNalivaika%26aufirst%3DE.%2BA.%26aulast%3D%25C3%2596zen%26aufirst%3DA.%26aulast%3DPrabu-Jeyabalan%26aufirst%3DM.%2BM.%26aulast%3DThayer%26aufirst%3DK.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DMolecular%2520basis%2520for%2520drug%2520resistance%2520in%2520HIV-1%2520protease%26jtitle%3DViruses%26date%3D2010%26volume%3D2%26spage%3D2509%26epage%3D2535%26doi%3D10.3390%2Fv2112509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osswald, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prato, G.</span></span> <span> </span><span class="NLM_article-title">Recent progress in the development of HIV-1 protease inhibitors for the treatment of HIV/AIDS</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5172</span>– <span class="NLM_lpage">5208</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01697</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01697" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5172-5208&author=A.+K.+Ghoshauthor=H.+L.+Osswaldauthor=G.+Prato&title=Recent+progress+in+the+development+of+HIV-1+protease+inhibitors+for+the+treatment+of+HIV%2FAIDS&doi=10.1021%2Facs.jmedchem.5b01697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Progress in the Development of HIV-1 Protease Inhibitors for the Treatment of HIV/AIDS</span></div><div class="casAuthors">Ghosh, Arun K.; Osswald, Heather L.; Prato, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5172-5208</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HIV-1 protease inhibitors continue to play an important role in the treatment of HIV/AIDS, transforming this deadly ailment into a more manageable chronic infection.  Over the years, intensive research has led to a variety of approved protease inhibitors for the treatment of HIV/AIDS.  In this review, we outline current drug design and medicinal chem. efforts toward the development of next-generation protease inhibitors beyond the currently approved drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4DisuSRbQXbVg90H21EOLACvtfcHk0lgt5HFzYBt8NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhsL0%253D&md5=e0994f3d84ab627d3ac3f757e8d11725</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01697%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DOsswald%26aufirst%3DH.%2BL.%26aulast%3DPrato%26aufirst%3DG.%26atitle%3DRecent%2520progress%2520in%2520the%2520development%2520of%2520HIV-1%2520protease%2520inhibitors%2520for%2520the%2520treatment%2520of%2520HIV%252FAIDS%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5172%26epage%3D5208%26doi%3D10.1021%2Facs.jmedchem.5b01697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Surleraux, D. L. N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tahri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verschueren, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pille, G. M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de
Kock, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonckers, T. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peeters, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Meyer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azijn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauwels, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bethune, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prabu-Jeyabalan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigerinck, P. B. T. P.</span></span> <span> </span><span class="NLM_article-title">Discovery and selection of TMC114, a next generation HIV-1 protease inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1813</span>– <span class="NLM_lpage">1822</span>, <span class="refDoi"> DOI: 10.1021/jm049560p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049560p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotVOlu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=1813-1822&author=D.+L.+N.+G.+Surlerauxauthor=A.+Tahriauthor=W.+G.+Verschuerenauthor=G.+M.+E.+Pilleauthor=H.+A.+de%0AKockauthor=T.+H.+M.+Jonckersauthor=A.+Peetersauthor=S.+De+Meyerauthor=H.+Azijnauthor=R.+Pauwelsauthor=M.+P.+de+Bethuneauthor=N.+M.+Kingauthor=M.+Prabu-Jeyabalanauthor=C.+A.+Schifferauthor=P.+B.+T.+P.+Wigerinck&title=Discovery+and+selection+of+TMC114%2C+a+next+generation+HIV-1+protease+inhibitor&doi=10.1021%2Fjm049560p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Selection of TMC114, a Next Generation HIV-1 Protease Inhibitor</span></div><div class="casAuthors">Surleraux, Dominique L. N. G.; Tahri, Abdellah; Verschueren, Wim G.; Pille, Geert M. E.; de Kock, Herman A.; Jonckers, Tim H. M.; Peeters, Anik; De Meyer, Sandra; Azijn, Hilde; Pauwels, Rudi; de Bethune, Marie-Pierre; King, Nancy M.; Prabu-Jeyabalan, Moses; Schiffer, Celia A.; Wigerinck, Piet B. T. P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1813-1822</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The screening of known HIV-1 protease inhibitors against a panel of multidrug-resistant viruses revealed the potent activity of TMC126 on drug-resistant mutants.  In comparison to amprenavir, the improved affinity of TMC126 is largely the result of one extra hydrogen bond to the backbone of the protein in the P2 pocket.  Modification of the substitution pattern on the phenylsulfonamide P2' substituent of TMC126 created an interesting SAR, with the close analog TMC114 being found to have a similar antiviral activity against the mutant and the wild-type viruses.  X-ray and thermodn. studies on both wild-type and mutant enzymes showed an extremely high enthalpy driven affinity of TMC114 for HIV-1 protease.  In vitro selection of mutants resistant to TMC114 starting from wild-type virus proved to be extremely difficult; this was not the case for other close analogs.  Therefore, the extra H-bond to the backbone in the P2 pocket cannot be the only explanation for the interesting antiviral profile of TMC114.  Absorption studies in animals indicated that TMC114 has pharmacokinetic properties comparable to currently approved HIV-1 protease inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdYfZu3Lc6mbVg90H21EOLACvtfcHk0lgt5HFzYBt8NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotVOlu7Y%253D&md5=35809ccdec57ebb6afa7af1b80362f74</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm049560p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049560p%26sid%3Dliteratum%253Aachs%26aulast%3DSurleraux%26aufirst%3DD.%2BL.%2BN.%2BG.%26aulast%3DTahri%26aufirst%3DA.%26aulast%3DVerschueren%26aufirst%3DW.%2BG.%26aulast%3DPille%26aufirst%3DG.%2BM.%2BE.%26aulast%3Dde%2BKock%26aufirst%3DH.%2BA.%26aulast%3DJonckers%26aufirst%3DT.%2BH.%2BM.%26aulast%3DPeeters%26aufirst%3DA.%26aulast%3DDe%2BMeyer%26aufirst%3DS.%26aulast%3DAzijn%26aufirst%3DH.%26aulast%3DPauwels%26aufirst%3DR.%26aulast%3Dde%2BBethune%26aufirst%3DM.%2BP.%26aulast%3DKing%26aufirst%3DN.%2BM.%26aulast%3DPrabu-Jeyabalan%26aufirst%3DM.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26aulast%3DWigerinck%26aufirst%3DP.%2BB.%2BT.%2BP.%26atitle%3DDiscovery%2520and%2520selection%2520of%2520TMC114%252C%2520a%2520next%2520generation%2520HIV-1%2520protease%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D1813%26epage%3D1822%26doi%3D10.1021%2Fjm049560p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapsal, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuya, H.</span></span> <span> </span><span class="NLM_article-title">Design of HIV protease inhibitors targeting protein backbone: an effective strategy for combating drug resistance</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1021/ar7001232</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar7001232" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFyhsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2008&pages=78-86&author=A.+K.+Ghoshauthor=B.+D.+Chapsalauthor=I.+T.+Weberauthor=H.+Mitsuya&title=Design+of+HIV+protease+inhibitors+targeting+protein+backbone%3A+an+effective+strategy+for+combating+drug+resistance&doi=10.1021%2Far7001232"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Design of HIV Protease Inhibitors Targeting Protein Backbone: An Effective Strategy for Combating Drug Resistance</span></div><div class="casAuthors">Ghosh, Arun K.; Chapsal, Bruno D.; Weber, Irene T.; Mitsuya, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-86</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The discovery of HIV protease inhibitors (PIs) followed by their incorporation into highly active antiretroviral therapy (HAART) was a major turning point for the management of HIV/AIDS.  However, the rapid emergence of drug resistance has become a major concern.  Our structure-based design of HIV-1 PIs that can specifically interact with the enzyme backbone atoms has led to the development of compds. with unprecedented potency and drug-resistance profiles.  Particularly notable is the development of a stereochem. defined bis(tetrahydrofuranyl)urethane as the non-peptidic P2 ligand.  Incorporation of this ligand has provided a variety of exceedingly potent HIV-1 PIs with superior activity against multi-PI-resistant variants relative to other FDA-approved PIs.  Darunavir, a PI recently approved for the treatment of drug-resistant HIV, directly resulted from this strategy.  Our detailed protein-darunavir X-ray crystal structures have clearly demonstrated extensive interactions with the protein backbone in the enzyme active site.  This present design concept targeting the protein backbone may serve as a powerful strategy to combat drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGZCuOKomAEbVg90H21EOLACvtfcHk0lhzbpI4eFBsVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFyhsbo%253D&md5=a8ee6f368273a5fece6ebbaf9bafc750</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Far7001232&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far7001232%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DChapsal%26aufirst%3DB.%2BD.%26aulast%3DWeber%26aufirst%3DI.%2BT.%26aulast%3DMitsuya%26aufirst%3DH.%26atitle%3DDesign%2520of%2520HIV%2520protease%2520inhibitors%2520targeting%2520protein%2520backbone%253A%2520an%2520effective%2520strategy%2520for%2520combating%2520drug%2520resistance%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2008%26volume%3D41%26spage%3D78%26epage%3D86%26doi%3D10.1021%2Far7001232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boross, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaddis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leshchenko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, I. T.</span></span> <span> </span><span class="NLM_article-title">High resolution crystal structures of HIV-1 protease with a potent non-peptide inhibitor (UIC-94017) active against multi-drug-resistant clinical strains</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>338</i></span>,  <span class="NLM_fpage">341</span>– <span class="NLM_lpage">352</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2004.02.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fj.jmb.2004.02.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=15066436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2cXivVeltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=338&publication_year=2004&pages=341-352&author=Y.+Tieauthor=P.+I.+Borossauthor=Y.+F.+Wangauthor=L.+Gaddisauthor=A.+K.+Hussainauthor=S.+Leshchenkoauthor=A.+K.+Ghoshauthor=J.+M.+Louisauthor=R.+W.+Harrisonauthor=I.+T.+Weber&title=High+resolution+crystal+structures+of+HIV-1+protease+with+a+potent+non-peptide+inhibitor+%28UIC-94017%29+active+against+multi-drug-resistant+clinical+strains&doi=10.1016%2Fj.jmb.2004.02.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">High Resolution Crystal Structures of HIV-1 Protease with a Potent Non-peptide Inhibitor (UIC-94017) Active Against Multi-drug-resistant Clinical Strains</span></div><div class="casAuthors">Tie, Yunfeng; Boross, Peter I.; Wang, Yuan-Fang; Gaddis, Laquasha; Hussain, Azhar K.; Leshchenko, Sofiya; Ghosh, Arun K.; Louis, John M.; Harrison, Robert W.; Weber, Irene T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">338</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">341-352</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The compd. UIC-94017 (TMC-114) is a second-generation HIV protease inhibitor with improved pharmacokinetics that is chem. related to the clin. inhibitor amprenavir.  UIC-94017 is a broad-spectrum potent inhibitor active against HIV-1 clin. isolates with minimal cytotoxicity.  We have detd. the high-resoln. crystal structures of UIC-94017 in complexes with wild-type HIV-1 protease (PR) and mutant proteases PRV82A and PRI84V that are common in drug-resistant HIV.  The structures were refined at resolns. of 1.10 - 1.53 Å.  The crystal structures of PR and PRI84V with UIC-94017 ternary complexes show that the inhibitor binds to the protease in two overlapping positions, while the PRV82A complex had one ordered inhibitor.  In all three structures, UIC-94017 forms hydrogen bonds with the conserved main-chain atoms of Asp29 and Asp30 of the protease.  These interactions are proposed to be crit. for the potency of this compd. against HIV isolates that are resistant to multiple protease inhibitors.  Other small differences were obsd. in the interactions of the mutants with UIC-94017 as compared to PR.  PRV82A showed differences in the position of the main-chain atoms of residue 82 compared to PR structure that better accommodated the inhibitor.  Finally, the 1.10 Å resoln. structure of PRV82A with UIC-94017 showed an unusual distribution of electron d. for the catalytic aspartate residues, which is discussed in relation to the reaction mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHW51Not1WA7Vg90H21EOLACvtfcHk0lhzbpI4eFBsVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXivVeltL0%253D&md5=82ecc0bb118a886b254ac579ffe9c5fc</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2004.02.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2004.02.052%26sid%3Dliteratum%253Aachs%26aulast%3DTie%26aufirst%3DY.%26aulast%3DBoross%26aufirst%3DP.%2BI.%26aulast%3DWang%26aufirst%3DY.%2BF.%26aulast%3DGaddis%26aufirst%3DL.%26aulast%3DHussain%26aufirst%3DA.%2BK.%26aulast%3DLeshchenko%26aufirst%3DS.%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DLouis%26aufirst%3DJ.%2BM.%26aulast%3DHarrison%26aufirst%3DR.%2BW.%26aulast%3DWeber%26aufirst%3DI.%2BT.%26atitle%3DHigh%2520resolution%2520crystal%2520structures%2520of%2520HIV-1%2520protease%2520with%2520a%2520potent%2520non-peptide%2520inhibitor%2520%2528UIC-94017%2529%2520active%2520against%2520multi-drug-resistant%2520clinical%2520strains%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D338%26spage%3D341%26epage%3D352%26doi%3D10.1016%2Fj.jmb.2004.02.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramu Sridhar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumaragurubaran, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuya, H.</span></span> <span> </span><span class="NLM_article-title">Bis-tetrahydrofuran: a privileged ligand for darunavir and a new generation of hiv protease inhibitors that combat drug resistance</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">939</span>– <span class="NLM_lpage">950</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200600103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1002%2Fcmdc.200600103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=16927344" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsVCmu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=939-950&author=A.+K.+Ghoshauthor=P.+Ramu+Sridharauthor=N.+Kumaragurubaranauthor=Y.+Kohauthor=I.+T.+Weberauthor=H.+Mitsuya&title=Bis-tetrahydrofuran%3A+a+privileged+ligand+for+darunavir+and+a+new+generation+of+hiv+protease+inhibitors+that+combat+drug+resistance&doi=10.1002%2Fcmdc.200600103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Bis-tetrahydrofuran: a privileged ligand for darunavir and a new generation of HIV protease inhibitors that combat drug resistance</span></div><div class="casAuthors">Ghosh, Arun K.; Sridhar, Perali Ramu; Kumaragurubaran, Nagaswamy; Koh, Yasuhiro; Weber, Irene T.; Mitsuya, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">939-950</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The structure-based design of bis-tetrahydrofuranyl urethane has emerged as a privileged nonpeptide P2 ligand for a variety of highly potent HIV-1 protease inhibitors.  Incorporation of this ligand provided HIV protease inhibitors with exceedingly potent antiviral activity and superior activity against multi-PI-resistant variants relative to other FDA-approved PIs.  Recently, TMC-114 (darunavir) was approved by the FDA for treatment of drug-resistant HIV.  GW640385 (brecanavir), which incorporates bis-THF as the P2 ligand, is currently in Phase-III clin. development.  The bis-THF ligand was specifically designed to fill in the hydrophobic S2 pocket effectively and to promote extensive hydrogen bonding with the protein backbone in the enzyme S2 site.  The protein-ligand x-ray crystal structures with TMC-114 and other inhibitors with the bis-THF ligand revealed extensive interactions with the backbone of residues Asp29 and Asp30 at the S2 site.  The current design concept targeting the protein backbone may serve as an important guide to combat drug resistance.  Further design and synthesis of conceptually novel inhibitors are in progress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMpT49sY24kLVg90H21EOLACvtfcHk0lgX24gtfMDTPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsVCmu7w%253D&md5=e9826330e807476b916d69e86ff55a40</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200600103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200600103%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DRamu%2BSridhar%26aufirst%3DP.%26aulast%3DKumaragurubaran%26aufirst%3DN.%26aulast%3DKoh%26aufirst%3DY.%26aulast%3DWeber%26aufirst%3DI.%2BT.%26aulast%3DMitsuya%26aufirst%3DH.%26atitle%3DBis-tetrahydrofuran%253A%2520a%2520privileged%2520ligand%2520for%2520darunavir%2520and%2520a%2520new%2520generation%2520of%2520hiv%2520protease%2520inhibitors%2520that%2520combat%2520drug%2520resistance%26jtitle%3DChemMedChem%26date%3D2006%26volume%3D1%26spage%3D939%26epage%3D950%26doi%3D10.1002%2Fcmdc.200600103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hazen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferris, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craig, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaldor, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samano, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furfine, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spaltenstein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tung, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St Clair, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanlon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boone, L.</span></span> <span> </span><span class="NLM_article-title">In vitro antiviral activity of the novel, tyrosyl-based human immunodeficiency virus (HIV) type 1 protease inhibitor brecanavir (GW640385) in combination with other antiretrovirals and against a panel of protease inhibitor-resistant HIV</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">3147</span>– <span class="NLM_lpage">3154</span>, <span class="refDoi"> DOI: 10.1128/AAC.00401-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1128%2FAAC.00401-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=17620375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVais73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2007&pages=3147-3154&author=R.+Hazenauthor=R.+Harveyauthor=R.+Ferrisauthor=C.+Craigauthor=P.+Yatesauthor=P.+Griffinauthor=J.+Millerauthor=I.+Kaldorauthor=J.+Rayauthor=V.+Samanoauthor=E.+Furfineauthor=A.+Spaltensteinauthor=M.+Haleauthor=R.+Tungauthor=M.+St+Clairauthor=M.+Hanlonauthor=L.+Boone&title=In+vitro+antiviral+activity+of+the+novel%2C+tyrosyl-based+human+immunodeficiency+virus+%28HIV%29+type+1+protease+inhibitor+brecanavir+%28GW640385%29+in+combination+with+other+antiretrovirals+and+against+a+panel+of+protease+inhibitor-resistant+HIV&doi=10.1128%2FAAC.00401-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro antiviral activity of the novel, tyrosyl-based human immunodeficiency virus (HIV) type 1 protease inhibitor brecanavir (GW640385) in combination with other antiretrovirals and against a panel of protease inhibitor-resistant HIV</span></div><div class="casAuthors">Hazen, Richard; Harvey, Robert; Ferris, Robert; Craig, Charles; Yates, Phillip; Griffin, Philip; Miller, John; Kaldor, Istvan; Ray, John; Samano, Vincente; Furfine, Eric; Spaltenstein, Andrew; Hale, Michael; Tung, Roger; St. Clair, Marty; Hanlon, Mary; Boone, Lawrence</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3147-3154</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Brecanavir, a novel tyrosyl-based arylsulfonamide, high-affinity, human immunodeficiency virus type 1 (HIV-1) protease inhibitor (PI), has been evaluated for anti-HIV activity in several in vitro assays.  Preclin. assessment of brecanavir indicated that this compd. potently inhibited HIV-1 in cell culture assays with 50% effective concns. (EC50s) of 0.2 to 0.53 nM and was equally active against HIV strains utilizing either the CXCR4 or CCR5 coreceptor, as was found with other PIs.  The presence of up to 40% human serum decreased the anti-HIV-1 activity of brecanavir by 5.2-fold, but under these conditions the compd. retained single-digit nanomolar EC50s.  When brecanavir was tested in combination with nucleoside reverse transcriptase inhibitors, the antiviral activity of brecanavir was synergistic with the effects of stavudine and additive to the effects of zidovudine, tenofovir, dideoxycytidine, didanosine, adefovir, abacavir, lamivudine, and emtricitabine.  Brecanavir was synergistic with the nonnucleoside reverse transcriptase inhibitor nevirapine or delavirdine and was additive to the effects of efavirenz.  In combination with other PIs, brecanavir was additive to the activities of indinavir, lopinavir, nelfinavir, ritonavir, amprenavir, saquinavir, and atazanavir.  Clin. HIV isolates from PI-experienced patients were evaluated for sensitivity to brecanavir and other PIs in a recombinant virus assay.  Brecanavir had a <5-fold increase in EC50s against 80% of patient isolates tested and had a greater mean in vitro potency than amprenavir, indinavir, lopinavir, atazanavir, tipranavir, and darunavir.  Brecanavir is by a substantial margin the most potent and broadly active antiviral agent among the PIs tested in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqplFzzXvQJ9rVg90H21EOLACvtfcHk0lgX24gtfMDTPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVais73F&md5=a84fb4b36143fb6c9e92db4895ad1d9d</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1128%2FAAC.00401-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00401-07%26sid%3Dliteratum%253Aachs%26aulast%3DHazen%26aufirst%3DR.%26aulast%3DHarvey%26aufirst%3DR.%26aulast%3DFerris%26aufirst%3DR.%26aulast%3DCraig%26aufirst%3DC.%26aulast%3DYates%26aufirst%3DP.%26aulast%3DGriffin%26aufirst%3DP.%26aulast%3DMiller%26aufirst%3DJ.%26aulast%3DKaldor%26aufirst%3DI.%26aulast%3DRay%26aufirst%3DJ.%26aulast%3DSamano%26aufirst%3DV.%26aulast%3DFurfine%26aufirst%3DE.%26aulast%3DSpaltenstein%26aufirst%3DA.%26aulast%3DHale%26aufirst%3DM.%26aulast%3DTung%26aufirst%3DR.%26aulast%3DSt%2BClair%26aufirst%3DM.%26aulast%3DHanlon%26aufirst%3DM.%26aulast%3DBoone%26aufirst%3DL.%26atitle%3DIn%2520vitro%2520antiviral%2520activity%2520of%2520the%2520novel%252C%2520tyrosyl-based%2520human%2520immunodeficiency%2520virus%2520%2528HIV%2529%2520type%25201%2520protease%2520inhibitor%2520brecanavir%2520%2528GW640385%2529%2520in%2520combination%2520with%2520other%2520antiretrovirals%2520and%2520against%2520a%2520panel%2520of%2520protease%2520inhibitor-resistant%2520HIV%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2007%26volume%3D51%26spage%3D3147%26epage%3D3154%26doi%3D10.1128%2FAAC.00401-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cihlar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulato, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leavitt, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stray, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W. A.</span></span> <span> </span><span class="NLM_article-title">Suppression of HIV-1 protease inhibitor resistance by phosphonate-mediated solvent anchoring</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>363</i></span>,  <span class="NLM_fpage">635</span>– <span class="NLM_lpage">647</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2006.07.073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fj.jmb.2006.07.073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=16979654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVCrurbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=363&publication_year=2006&pages=635-647&author=T.+Cihlarauthor=G.+X.+Heauthor=X.+Liuauthor=J.+M.+Chenauthor=M.+Hatadaauthor=S.+Swaminathanauthor=M.+J.+McDermottauthor=Z.+Y.+Yangauthor=A.+S.+Mulatoauthor=X.+Chenauthor=S.+A.+Leavittauthor=K.+M.+Strayauthor=W.+A.+Lee&title=Suppression+of+HIV-1+protease+inhibitor+resistance+by+phosphonate-mediated+solvent+anchoring&doi=10.1016%2Fj.jmb.2006.07.073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of HIV-1 Protease Inhibitor Resistance by Phosphonate-mediated Solvent Anchoring</span></div><div class="casAuthors">Cihlar, Tomas; He, Gong-Xin; Liu, Xiaohong; Chen, James M.; Hatada, Marcos; Swaminathan, Swami; McDermott, Martin J.; Yang, Zheng-Yu; Mulato, Andrew S.; Chen, Xiaowu; Leavitt, Stephanie A.; Stray, Kirsten M.; Lee, William A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">363</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">635-647</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The introduction of human immunodeficiency virus type 1 (HIV-1) protease inhibitors (PIs) markedly improved the clin. outcome and control of HIV-1 infection.  However, cross-resistance among PIs due to a wide spectrum of mutations in viral protease is a major factor limiting their broader clin. use.  Here we report on the suppression of PI resistance using a covalent attachment of a phosphonic acid motif to a peptidomimetic inhibitor scaffold.  The resulting phosphonate analogs maintain high binding affinity to HIV-1 protease, potent antiretroviral activity, and unlike the parent mols., display no loss of potency against a panel of clin. important PI-resistant HIV-1 strains.  As shown by crystallog. anal., the phosphonate moiety is highly exposed to solvent with no discernable interactions with any of the enzyme active site or surface residues.  We term this effect "solvent anchoring" and demonstrate that it is driven by a favorable change in the inhibitor binding entropy upon the interaction with mutant enzymes.  This type of thermodn. behavior, which was not found with the parent scaffold fully buried in the enzyme active site, is a result of the increased degeneracy of inhibitor binding states, allowing effective mol. adaptation to the expanded cavity vol. of mutant proteases.  This strategy, which is applicable to various PI scaffolds, should facilitate the design of novel PIs and potentially other antiviral therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgTKggUJt9RLVg90H21EOLACvtfcHk0liiER7kbQQ2Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVCrurbE&md5=46b1cae8f52f1acb3246c84ae6e8708e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2006.07.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2006.07.073%26sid%3Dliteratum%253Aachs%26aulast%3DCihlar%26aufirst%3DT.%26aulast%3DHe%26aufirst%3DG.%2BX.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%2BM.%26aulast%3DHatada%26aufirst%3DM.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DMcDermott%26aufirst%3DM.%2BJ.%26aulast%3DYang%26aufirst%3DZ.%2BY.%26aulast%3DMulato%26aufirst%3DA.%2BS.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLeavitt%26aufirst%3DS.%2BA.%26aulast%3DStray%26aufirst%3DK.%2BM.%26aulast%3DLee%26aufirst%3DW.%2BA.%26atitle%3DSuppression%2520of%2520HIV-1%2520protease%2520inhibitor%2520resistance%2520by%2520phosphonate-mediated%2520solvent%2520anchoring%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D363%26spage%3D635%26epage%3D647%26doi%3D10.1016%2Fj.jmb.2006.07.073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dandache, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sévigny, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yelle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stranix, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapiro, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wainberg, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. J.</span></span> <span> </span><span class="NLM_article-title">In vitro antiviral activity and cross-resistance profile of PL-100, a novel protease inhibitor of human immunodeficiency virus type 1</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">4036</span>– <span class="NLM_lpage">4043</span>, <span class="refDoi"> DOI: 10.1128/AAC.00149-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1128%2FAAC.00149-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=17638694" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1ygtL3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2007&pages=4036-4043&author=S.+Dandacheauthor=G.+S%C3%A9vignyauthor=J.+Yelleauthor=B.+R.+Stranixauthor=N.+Parkinauthor=J.+M.+Schapiroauthor=M.+A.+Wainbergauthor=J.+J.+Wu&title=In+vitro+antiviral+activity+and+cross-resistance+profile+of+PL-100%2C+a+novel+protease+inhibitor+of+human+immunodeficiency+virus+type+1&doi=10.1128%2FAAC.00149-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro antiviral activity and cross-resistance profile of PL-100, a novel protease inhibitor of human immunodeficiency virus type 1</span></div><div class="casAuthors">Dandache, Serge; Sevigny, Guy; Yelle, Jocelyn; Stranix, Brent R.; Parkin, Neil; Schapiro, Jonathan M.; Wainberg, Mark A.; Wu, Jinzi J.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4036-4043</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Despite the success of highly active antiretroviral therapy, the current emergence and spread of drug-resistant variants of human immunodeficiency virus (HIV) stress the need for new inhibitors with distinct properties.  We designed, produced, and screened a library of compds. based on an original L-lysine scaffold for their potentials as HIV type 1 (HIV-1) protease inhibitors (PI).  One candidate compd., PL-100, emerged as a specific and noncytotoxic PI that exhibited potent inhibition of HIV-1 protease and viral replication in vitro (Ki, ∼36 pM, and 50% effective concn. [EC50], ∼16 nM, resp.).  To confirm that PL-100 possessed a favorable resistance profile, we performed a cross-resistance study using a panel of 63 viral strains from PI-experienced patients selected for the presence of primary PI mutations known to confer resistance to multiple PIs now in clin. use.  The results showed that PL-100 retained excellent antiviral activity against almost all of these PI-resistant viruses and that its performance in this regard was superior to those of atazanavir, amprenavir, indinavir, lopinavir, nelfinavir, and saquinavir.  In almost every case, the increase in the EC50 for PL-100 obsd. with viruses contg. multiple mutations in protease was far less than that obtained with the other drugs tested.  These data underscore the potential for PL-100 to be used in the treatment of drug-resistant HIV disease and argue for its further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3g2z6DZZfnbVg90H21EOLACvtfcHk0liiER7kbQQ2Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1ygtL3M&md5=165d8269fa4fcc5f6592a58250570fe9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1128%2FAAC.00149-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00149-07%26sid%3Dliteratum%253Aachs%26aulast%3DDandache%26aufirst%3DS.%26aulast%3DS%25C3%25A9vigny%26aufirst%3DG.%26aulast%3DYelle%26aufirst%3DJ.%26aulast%3DStranix%26aufirst%3DB.%2BR.%26aulast%3DParkin%26aufirst%3DN.%26aulast%3DSchapiro%26aufirst%3DJ.%2BM.%26aulast%3DWainberg%26aufirst%3DM.%2BA.%26aulast%3DWu%26aufirst%3DJ.%2BJ.%26atitle%3DIn%2520vitro%2520antiviral%2520activity%2520and%2520cross-resistance%2520profile%2520of%2520PL-100%252C%2520a%2520novel%2520protease%2520inhibitor%2520of%2520human%2520immunodeficiency%2520virus%2520type%25201%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2007%26volume%3D51%26spage%3D4036%26epage%3D4043%26doi%3D10.1128%2FAAC.00149-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kesteleyn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amssoms, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepens, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hache, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verschueren, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van De Vreken, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rombauts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meurs, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterkens, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoops, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baert, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masungi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dierynck, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundgren, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jönsson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkes, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalayanov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenquist, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samuelsson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Emelen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thuring, J. W.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of HIV-1 protease inhibitors for a long-acting injectable drug application</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">310</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.10.095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fj.bmcl.2012.10.095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=23177258" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsleitrnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=310-317&author=B.+Kesteleynauthor=K.+Amssomsauthor=W.+Schepensauthor=G.+Hacheauthor=W.+Verschuerenauthor=W.+Van+De+Vrekenauthor=K.+Rombautsauthor=G.+Meursauthor=P.+Sterkensauthor=B.+Stoopsauthor=L.+Baertauthor=N.+Austinauthor=J.+Wegnerauthor=C.+Masungiauthor=I.+Dierynckauthor=S.+Lundgrenauthor=D.+J%C3%B6nssonauthor=K.+Parkesauthor=G.+Kalayanovauthor=H.+Wallbergauthor=A.+Rosenquistauthor=B.+Samuelssonauthor=K.+Van+Emelenauthor=J.+W.+Thuring&title=Design+and+synthesis+of+HIV-1+protease+inhibitors+for+a+long-acting+injectable+drug+application&doi=10.1016%2Fj.bmcl.2012.10.095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of HIV-1 protease inhibitors for a long-acting injectable drug application</span></div><div class="casAuthors">Kesteleyn, Bart; Amssoms, Katie; Schepens, Wim; Hache, Geerwin; Verschueren, Wim; Van De Vreken, Wim; Rombauts, Klara; Meurs, Greet; Sterkens, Patrick; Stoops, Bart; Baert, Lieven; Austin, Nigel; Wegner, Jorg; Masungi, Chantal; Dierynck, Inge; Lundgren, Stina; Jonsson, Daniel; Parkes, Kevin; Kalayanov, Genadiy; Wallberg, Hans; Rosenquist, Asa; Samuelsson, Bertil; Van Emelen, Kristof; Thuring, Jan Willem</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">310-317</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The design and synthesis of novel HIV-1 protease inhibitors (PIs) (1-22), which display high potency against HIV-1 wild-type and multi-PI-resistant HIV-mutant clin. isolates, are described.  Lead optimization was initiated from compd. 1, a Phe-Phe hydroxyethylene peptidomimetic PI, and was directed towards the discovery of new PIs suitable for a long-acting (LA) injectable drug application.  Introducing a heterocyclic 6-methoxy-3-pyridinyl or a 6-(dimethylamino)-3-pyridinyl moiety (R3) at the para-position of the P1' benzyl fragment generated compds. with antiviral potency in the low single digit nanomolar range.  Halogenation or alkylation of the metabolic hot spots on the various arom. rings resulted in PIs with high stability against degrdn. in human liver microsomes and low plasma clearance in rats.  Replacing the chromanolamine moiety (R1) in the P2 protease binding site by a cyclopentanolamine or a cyclohexanolamine deriv. provided a series of high clearance PIs (16-22) with EC50s on wild-type HIV-1 in the range of 0.8-1.8 nM.  PIs 18 and 22, formulated as nanosuspensions, showed gradual but sustained and complete release from the injection site over two months in rats, and were therefore identified as interesting candidates for a LA injectable drug application for treating HIV/AIDS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojsvYYFkCohrVg90H21EOLACvtfcHk0liiER7kbQQ2Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsleitrnM&md5=a90562b30218a52c55496037245c5e06</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.10.095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.10.095%26sid%3Dliteratum%253Aachs%26aulast%3DKesteleyn%26aufirst%3DB.%26aulast%3DAmssoms%26aufirst%3DK.%26aulast%3DSchepens%26aufirst%3DW.%26aulast%3DHache%26aufirst%3DG.%26aulast%3DVerschueren%26aufirst%3DW.%26aulast%3DVan%2BDe%2BVreken%26aufirst%3DW.%26aulast%3DRombauts%26aufirst%3DK.%26aulast%3DMeurs%26aufirst%3DG.%26aulast%3DSterkens%26aufirst%3DP.%26aulast%3DStoops%26aufirst%3DB.%26aulast%3DBaert%26aufirst%3DL.%26aulast%3DAustin%26aufirst%3DN.%26aulast%3DWegner%26aufirst%3DJ.%26aulast%3DMasungi%26aufirst%3DC.%26aulast%3DDierynck%26aufirst%3DI.%26aulast%3DLundgren%26aufirst%3DS.%26aulast%3DJ%25C3%25B6nsson%26aufirst%3DD.%26aulast%3DParkes%26aufirst%3DK.%26aulast%3DKalayanov%26aufirst%3DG.%26aulast%3DWallberg%26aufirst%3DH.%26aulast%3DRosenquist%26aufirst%3DA.%26aulast%3DSamuelsson%26aufirst%3DB.%26aulast%3DVan%2BEmelen%26aufirst%3DK.%26aulast%3DThuring%26aufirst%3DJ.%2BW.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520HIV-1%2520protease%2520inhibitors%2520for%2520a%2520long-acting%2520injectable%2520drug%2520application%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D310%26epage%3D317%26doi%3D10.1016%2Fj.bmcl.2012.10.095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bungard, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiscount, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loughran, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manikowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crespo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keertikar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morriello, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamford, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddell, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diamond, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahnck-Teets, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fay, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCauley, J. A.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of piperazine sulfonamide cores leading to highly potent HIV-1 protease inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1292</span>– <span class="NLM_lpage">1297</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00386</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00386" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVSnurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1292-1297&author=C.+J.+Bungardauthor=P.+D.+Williamsauthor=J.+Schulzauthor=C.+M.+Wiscountauthor=M.+K.+Hollowayauthor=H.+M.+Loughranauthor=J.+J.+Manikowskiauthor=H.+P.+Suauthor=D.+J.+Bennettauthor=L.+Changauthor=X.+J.+Chuauthor=A.+Crespoauthor=M.+P.+Dwyerauthor=K.+Keertikarauthor=G.+J.+Morrielloauthor=A.+W.+Stamfordauthor=S.+T.+Waddellauthor=B.+Zhongauthor=B.+Huauthor=T.+Jiauthor=T.+L.+Diamondauthor=C.+Bahnck-Teetsauthor=S.+S.+Carrollauthor=J.+F.+Fayauthor=X.+Minauthor=W.+Morrisauthor=J.+E.+Ballardauthor=M.+D.+Millerauthor=J.+A.+McCauley&title=Design+and+synthesis+of+piperazine+sulfonamide+cores+leading+to+highly+potent+HIV-1+protease+inhibitors&doi=10.1021%2Facsmedchemlett.7b00386"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Piperazine Sulfonamide Cores Leading to Highly Potent HIV-1 Protease Inhibitors</span></div><div class="casAuthors">Bungard, Christopher J.; Williams, Peter D.; Schulz, Jurgen; Wiscount, Catherine M.; Holloway, M. Katharine; Loughran, H. Marie; Manikowski, Jesse J.; Su, Hua-Poo; Bennett, David J.; Chang, Lehua; Chu, Xin-Jie; Crespo, Alejandro; Dwyer, Michael P.; Keertikar, Kartik; Morriello, Gregori J.; Stamford, Andrew W.; Waddell, Sherman T.; Zhong, Bin; Hu, Bin; Ji, Tao; Diamond, Tracy L.; Bahnck-Teets, Carolyn; Carroll, Steven S.; Fay, John F.; Min, Xu; Morris, William; Ballard, Jeanine E.; Miller, Michael D.; McCauley, John A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1292-1297</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using the HIV-1 protease binding mode of MK-8718 and PL-100 as inspiration, a novel aspartate binding bicyclic piperazine sulfonamide core was designed and synthesized.  The resulting HIV-1 protease inhibitor contg. this core showed an 60-fold increase in enzyme binding affinity and a 10-fold increase in antiviral activity relative to MK-8718.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrku3wJBBdY07Vg90H21EOLACvtfcHk0lhxc_KnYFhpig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVSnurvK&md5=c063d6663bd7ebede67974942cf1ea78</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00386&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00386%26sid%3Dliteratum%253Aachs%26aulast%3DBungard%26aufirst%3DC.%2BJ.%26aulast%3DWilliams%26aufirst%3DP.%2BD.%26aulast%3DSchulz%26aufirst%3DJ.%26aulast%3DWiscount%26aufirst%3DC.%2BM.%26aulast%3DHolloway%26aufirst%3DM.%2BK.%26aulast%3DLoughran%26aufirst%3DH.%2BM.%26aulast%3DManikowski%26aufirst%3DJ.%2BJ.%26aulast%3DSu%26aufirst%3DH.%2BP.%26aulast%3DBennett%26aufirst%3DD.%2BJ.%26aulast%3DChang%26aufirst%3DL.%26aulast%3DChu%26aufirst%3DX.%2BJ.%26aulast%3DCrespo%26aufirst%3DA.%26aulast%3DDwyer%26aufirst%3DM.%2BP.%26aulast%3DKeertikar%26aufirst%3DK.%26aulast%3DMorriello%26aufirst%3DG.%2BJ.%26aulast%3DStamford%26aufirst%3DA.%2BW.%26aulast%3DWaddell%26aufirst%3DS.%2BT.%26aulast%3DZhong%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DJi%26aufirst%3DT.%26aulast%3DDiamond%26aufirst%3DT.%2BL.%26aulast%3DBahnck-Teets%26aufirst%3DC.%26aulast%3DCarroll%26aufirst%3DS.%2BS.%26aulast%3DFay%26aufirst%3DJ.%2BF.%26aulast%3DMin%26aufirst%3DX.%26aulast%3DMorris%26aufirst%3DW.%26aulast%3DBallard%26aufirst%3DJ.%2BE.%26aulast%3DMiller%26aufirst%3DM.%2BD.%26aulast%3DMcCauley%26aufirst%3DJ.%2BA.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520piperazine%2520sulfonamide%2520cores%2520leading%2520to%2520highly%2520potent%2520HIV-1%2520protease%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D1292%26epage%3D1297%26doi%3D10.1021%2Facsmedchemlett.7b00386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brieger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furfine, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanlon, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazen, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaldor, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLean, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sammond, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spaltenstein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tung, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherrill, R. G.</span></span> <span> </span><span class="NLM_article-title">Ultra-potent P1 modified arylsulfonamide HIV protease inhibitors: the discovery of GW0385</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1788</span>– <span class="NLM_lpage">1794</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2006.01.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fj.bmcl.2006.01.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=16458505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvVyrtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=1788-1794&author=J.+F.+Millerauthor=C.+W.+Andrewsauthor=M.+Briegerauthor=E.+S.+Furfineauthor=M.+R.+Haleauthor=M.+H.+Hanlonauthor=R.+J.+Hazenauthor=I.+Kaldorauthor=E.+W.+McLeanauthor=D.+Reynoldsauthor=D.+M.+Sammondauthor=A.+Spaltensteinauthor=R.+Tungauthor=E.+M.+Turnerauthor=R.+X.+Xuauthor=R.+G.+Sherrill&title=Ultra-potent+P1+modified+arylsulfonamide+HIV+protease+inhibitors%3A+the+discovery+of+GW0385&doi=10.1016%2Fj.bmcl.2006.01.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Ultra-potent P1 modified arylsulfonamide HIV protease inhibitors: The discovery of GW0385</span></div><div class="casAuthors">Miller, John F.; Andrews, C. Webster; Brieger, Michael; Furfine, Eric S.; Hale, Michael R.; Hanlon, Mary H.; Hazen, Richard J.; Kaldor, Istvan; McLean, Ed W.; Reynolds, David; Sammond, Douglas M.; Spaltenstein, Andrew; Tung, Roger; Turner, Elizabeth M.; Xu, Robert X.; Sherrill, Ronald G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1788-1794</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A novel series of P1 modified HIV protease inhibitors was synthesized and evaluated for in vitro antiviral activity against wild-type virus and protease inhibitor-resistant viruses.  Optimization of the P1 moiety resulted in compds. with femtomolar enzyme activities and cellular antiviral activities in the low nanomolar range culminating in the identification of clin. candidate GW0385.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojt0KYD8JW_rVg90H21EOLACvtfcHk0lhxc_KnYFhpig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhvVyrtLs%253D&md5=50f237c39e7eb19475315ab50240719e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2006.01.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2006.01.035%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DJ.%2BF.%26aulast%3DAndrews%26aufirst%3DC.%2BW.%26aulast%3DBrieger%26aufirst%3DM.%26aulast%3DFurfine%26aufirst%3DE.%2BS.%26aulast%3DHale%26aufirst%3DM.%2BR.%26aulast%3DHanlon%26aufirst%3DM.%2BH.%26aulast%3DHazen%26aufirst%3DR.%2BJ.%26aulast%3DKaldor%26aufirst%3DI.%26aulast%3DMcLean%26aufirst%3DE.%2BW.%26aulast%3DReynolds%26aufirst%3DD.%26aulast%3DSammond%26aufirst%3DD.%2BM.%26aulast%3DSpaltenstein%26aufirst%3DA.%26aulast%3DTung%26aufirst%3DR.%26aulast%3DTurner%26aufirst%3DE.%2BM.%26aulast%3DXu%26aufirst%3DR.%2BX.%26aulast%3DSherrill%26aufirst%3DR.%2BG.%26atitle%3DUltra-potent%2520P1%2520modified%2520arylsulfonamide%2520HIV%2520protease%2520inhibitors%253A%2520the%2520discovery%2520of%2520GW0385%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D1788%26epage%3D1794%26doi%3D10.1016%2Fj.bmcl.2006.01.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, G.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callebaut, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cihlar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimilli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenberg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stray, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strickley, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, C. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span> <span> </span><span class="NLM_article-title">Discovery of GS-8374, a potent human immunodeficiency virus type 1 protease inhibitor with a superior resistance profile</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">1093</span>– <span class="NLM_lpage">1098</span>, <span class="refDoi"> DOI: 10.1039/c1md00147g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1039%2Fc1md00147g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVSktr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=1093-1098&author=G.-X.+Heauthor=Z.-Y.+Yangauthor=M.+Williamsauthor=C.+Callebautauthor=T.+Cihlarauthor=B.+P.+Murrayauthor=C.+Yangauthor=M.+L.+Mitchellauthor=H.+Liuauthor=J.+Wangauthor=M.+Arimilliauthor=E.+Eisenbergauthor=K.+M.+Strayauthor=L.+K.+Tsaiauthor=M.+Hatadaauthor=X.+Chenauthor=J.+M.+Chenauthor=Y.+Wangauthor=M.+S.+Leeauthor=R.+G.+Strickleyauthor=Q.+Iwataauthor=X.+Zhengauthor=C.+U.+Kimauthor=S.+Swaminathanauthor=M.+C.+Desaiauthor=W.+A.+Leeauthor=L.+Xu&title=Discovery+of+GS-8374%2C+a+potent+human+immunodeficiency+virus+type+1+protease+inhibitor+with+a+superior+resistance+profile&doi=10.1039%2Fc1md00147g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of GS-8374, a potent human immunodeficiency virus type 1 protease inhibitor with a superior resistance profile</span></div><div class="casAuthors">He, Gong-Xing; Yang, Zheng-Yu; Williams, Matthew; Callebaut, Christian; Cihlar, Tomas; Murray, Bernard P.; Yang, Chris; Mitchell, Michael L.; Liu, Hongtao; Wang, Jianying; Arimilli, Murty; Eisenberg, Eugene; Stray, Kirsten M.; Tsai, Luong K.; Hatada, Marcos; Chen, Xiaowu; Chen, James M.; Wang, Yujin; Lee, Melody S.; Strickley, Robert G.; Iwata, Quynh; Zheng, Xubin; Kim, Choung U.; Swaminathan, Swami; Desai, Manoj C.; Lee, William A.; Xu, Lianhong</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1093-1098</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Introduction of a unique phosphonate moiety at the P1 position of the TMC-126 (3) scaffold provided a series of novel HIV-1 protease inhibitors (PIs) with an improved resistance profile against highly resistant variants.  Optimization of the linker and phosphonate moieties lead to the identification of GS-8374 (1).  Compd. 1 is a potent and orally bioavailable HIV-1 PI with a superior resistance profile.  Synthesis and characterization of 1 are reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdQ4VQyn9Rb7Vg90H21EOLACvtfcHk0ljqm7JYuaZz6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVSktr3F&md5=ea0c8e0cd95eea21571c7c1148ef9af5</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1039%2Fc1md00147g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1md00147g%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DG.-X.%26aulast%3DYang%26aufirst%3DZ.-Y.%26aulast%3DWilliams%26aufirst%3DM.%26aulast%3DCallebaut%26aufirst%3DC.%26aulast%3DCihlar%26aufirst%3DT.%26aulast%3DMurray%26aufirst%3DB.%2BP.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DMitchell%26aufirst%3DM.%2BL.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DArimilli%26aufirst%3DM.%26aulast%3DEisenberg%26aufirst%3DE.%26aulast%3DStray%26aufirst%3DK.%2BM.%26aulast%3DTsai%26aufirst%3DL.%2BK.%26aulast%3DHatada%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%2BM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DM.%2BS.%26aulast%3DStrickley%26aufirst%3DR.%2BG.%26aulast%3DIwata%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DKim%26aufirst%3DC.%2BU.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DDesai%26aufirst%3DM.%2BC.%26aulast%3DLee%26aufirst%3DW.%2BA.%26aulast%3DXu%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520GS-8374%252C%2520a%2520potent%2520human%2520immunodeficiency%2520virus%2520type%25201%2520protease%2520inhibitor%2520with%2520a%2520superior%2520resistance%2520profile%26jtitle%3DMedChemComm%26date%3D2011%26volume%3D2%26spage%3D1093%26epage%3D1098%26doi%3D10.1039%2Fc1md00147g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Windsor, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palte, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukesh, J. C.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gold, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forest, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raines, R. T.</span></span> <span> </span><span class="NLM_article-title">Sub-picomolar inhibition of HIV-1 protease with a boronic acid</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">14015</span>– <span class="NLM_lpage">14018</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b07366</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b07366" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFaltbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=14015-14018&author=I.+W.+Windsorauthor=M.+J.+Palteauthor=J.+C.+Lukeshauthor=B.+Goldauthor=K.+T.+Forestauthor=R.+T.+Raines&title=Sub-picomolar+inhibition+of+HIV-1+protease+with+a+boronic+acid&doi=10.1021%2Fjacs.8b07366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Sub-picomolar Inhibition of HIV-1 Protease with a Boronic Acid</span></div><div class="casAuthors">Windsor, Ian W.; Palte, Michael J.; Lukesh, John C.; Gold, Brian; Forest, Katrina T.; Raines, Ronald T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">14015-14018</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Boronic acids have been typecast as moieties for covalent complexation and are employed only rarely as agents for non-covalent recognition.  By exploiting the profuse ability of a boronic acid group to form hydrogen bonds, we have developed an inhibitor of HIV-1 protease with extraordinary affinity.  Specifically, we find that replacing an aniline moiety in darunavir with a phenylboronic acid leads to 20-fold greater affinity for the protease.  X-ray crystallog. demonstrates that the boronic acid group participates in three hydrogen bonds, more than the amino group of darunavir or any other analog.  Importantly, the boronic acid maintains its hydrogen bonds and its affinity for the drug-resistant D30N variant of HIV-1 protease.  The BOH···OC hydrogen bonds between the boronic acid hydroxy group and Asp30 (or Asn30) of the protease are short (rO···O = 2.2 Å), and d. functional theory anal. reveals a high degree of covalency.  These data highlight the utility of boronic acids as versatile functional groups in the design of small-mol. ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8UYVpx53VSrVg90H21EOLACvtfcHk0ljqm7JYuaZz6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFaltbnI&md5=ad7d7a58f8e3cad8c6fea7cb2823741c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b07366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b07366%26sid%3Dliteratum%253Aachs%26aulast%3DWindsor%26aufirst%3DI.%2BW.%26aulast%3DPalte%26aufirst%3DM.%2BJ.%26aulast%3DLukesh%26aufirst%3DJ.%2BC.%26aulast%3DGold%26aufirst%3DB.%26aulast%3DForest%26aufirst%3DK.%2BT.%26aulast%3DRaines%26aufirst%3DR.%2BT.%26atitle%3DSub-picomolar%2520inhibition%2520of%2520HIV-1%2520protease%2520with%2520a%2520boronic%2520acid%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D14015%26epage%3D14018%26doi%3D10.1021%2Fjacs.8b07366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nalam, M. N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, G. S. K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anjum, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yilmaz, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tidor, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rana, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Substrate envelope-designed potent HIV-1 protease inhibitors to avoid drug resistance</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1116</span>– <span class="NLM_lpage">1124</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2013.07.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fj.chembiol.2013.07.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=24012370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVWis7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=1116-1124&author=M.+N.+L.+Nalamauthor=A.+Aliauthor=G.+S.+K.+K.+Reddyauthor=H.+Caoauthor=S.+G.+Anjumauthor=M.+D.+Altmanauthor=N.+K.+Yilmazauthor=B.+Tidorauthor=T.+M.+Ranaauthor=C.+A.+Schiffer&title=Substrate+envelope-designed+potent+HIV-1+protease+inhibitors+to+avoid+drug+resistance&doi=10.1016%2Fj.chembiol.2013.07.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate Envelope-Designed Potent HIV-1 Protease Inhibitors to Avoid Drug Resistance</span></div><div class="casAuthors">Nalam, Madhavi N. L.; Ali, Akbar; Reddy, G. S. Kiran Kumar; Cao, Hong; Anjum, Saima G.; Altman, Michael D.; Yilmaz, Nese Kurt; Tidor, Bruce; Rana, Tariq M.; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1116-1124</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The rapid evolution of HIV under selective drug pressure has led to multidrug resistant (MDR) strains that evade std. therapies.  We designed highly potent HIV-1 protease inhibitors (PIs) using the substrate envelope model, which confines inhibitors within the consensus vol. of natural substrates, providing inhibitors less susceptible to resistance because a mutation affecting such inhibitors will simultaneously affect viral substrate processing.  The designed PIs share a common chem. scaffold but utilize various moieties that optimally fill the substrate envelope, as confirmed by crystal structures.  The designed PIs retain robust binding to MDR protease variants and display exceptional antiviral potencies against different clades of HIV as well as a panel of 12 drug-resistant viral strains.  The substrate envelope model proves to be a powerful strategy to develop potent and robust inhibitors that avoid drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm5ioan7rs5LVg90H21EOLACvtfcHk0ljqm7JYuaZz6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVWis7zI&md5=2dc40eee276c4627593ee5857e2a7670</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2013.07.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2013.07.014%26sid%3Dliteratum%253Aachs%26aulast%3DNalam%26aufirst%3DM.%2BN.%2BL.%26aulast%3DAli%26aufirst%3DA.%26aulast%3DReddy%26aufirst%3DG.%2BS.%2BK.%2BK.%26aulast%3DCao%26aufirst%3DH.%26aulast%3DAnjum%26aufirst%3DS.%2BG.%26aulast%3DAltman%26aufirst%3DM.%2BD.%26aulast%3DYilmaz%26aufirst%3DN.%2BK.%26aulast%3DTidor%26aufirst%3DB.%26aulast%3DRana%26aufirst%3DT.%2BM.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DSubstrate%2520envelope-designed%2520potent%2520HIV-1%2520protease%2520inhibitors%2520to%2520avoid%2520drug%2520resistance%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D1116%26epage%3D1124%26doi%3D10.1016%2Fj.chembiol.2013.07.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higashi-Kuwata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulut, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoki-Ogata, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takamatsu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yedidi, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishida, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takamune, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyalapatla, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osswald, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jono, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarchoan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misumi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuya, H.</span></span> <span> </span><span class="NLM_article-title">A novel central nervous system-penetrating protease inhibitor overcomes human immunodeficiency virus 1 resistance with unprecedented aM to pM potency</span>. <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e28020</span> <span class="refDoi"> DOI: 10.7554/eLife.28020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.7554%2FeLife.28020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=29039736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlKkt7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&author=M.+Aokiauthor=H.+Hayashiauthor=K.+V.+Raoauthor=D.+Dasauthor=N.+Higashi-Kuwataauthor=H.+Bulutauthor=H.+Aoki-Ogataauthor=Y.+Takamatsuauthor=R.+S.+Yedidiauthor=D.+A.+Davisauthor=S.+I.+Hattoriauthor=N.+Nishidaauthor=K.+Hasegawaauthor=N.+Takamuneauthor=P.+R.+Nyalapatlaauthor=H.+L.+Osswaldauthor=H.+Jonoauthor=H.+Saitoauthor=R.+Yarchoanauthor=S.+Misumiauthor=A.+K.+Ghoshauthor=H.+Mitsuya&title=A+novel+central+nervous+system-penetrating+protease+inhibitor+overcomes+human+immunodeficiency+virus+1+resistance+with+unprecedented+aM+to+pM+potency&doi=10.7554%2FeLife.28020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A novel central nervous system-penetrating protease inhibitor overcomes human immunodeficiency virus 1 resistance with unprecedented aM to pM potency</span></div><div class="casAuthors">Aoki, Manabu; Hayashi, Hironori; Rao, Kalapala Venkateswara; Das, Debananda; Higashi-Kuwata, Nobuyo; Bulut, Haydar; Aoki-Ogata, Hiromi; Takamatsu, Yuki; Yedidi, Ravikiran S.; Davis, David A.; Hattori, Shin-ichiro; Nishida, Noriko; Hasegawa, Kazuya; Takamune, Nobutoki; Nyalapatla, Prasanth R.; Osswald, Heather L.; Jono, Hirofumi; Saito, Hideyuki; Yarchoan, Robert; Misumi, Shogo; Ghosh, Arun K.; Mitsuya, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e28020/1-e28020/25</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Antiretroviral therapy for HIV-1 infection/AIDS has significantly extended the life expectancy of HIV-1-infected individuals and reduced HIV-1 transmission at very high rates.  However, certain individuals who initially achieve viral suppression to undetectable levels may eventually suffer treatment failure mainly due to adverse effects and the emergence of drug-resistant HIV-1 variants.  Here, we report GRL-142, a novel HIV-1 protease inhibitor contg. an unprecedented 6-5-5-ring-fused crown-like tetrahydropyranofuran, which has extremely potent activity against all HIV-1 strains examd. with IC50 values of attomolar-to-picomolar concns., virtually no effects on cellular growth, extremely high genetic barrier against the emergence of drug-resistant variants, and favorable intracellular and central nervous system penetration.  GRL-142 forms optimum polar, van der Waals, and halogen bond interactions with HIV-1 protease and strongly blocks protease dimerization, demonstrating that combined multiple optimizing elements significantly enhance mol. and at. interactions with a target protein and generate unprecedentedly potent and practically favorable agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfcZXKE5Ic5rVg90H21EOLACvtfcHk0lhaw3sNYiwc6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlKkt7vE&md5=2c90963399dbc034de9e800d17a12ee1</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.7554%2FeLife.28020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.28020%26sid%3Dliteratum%253Aachs%26aulast%3DAoki%26aufirst%3DM.%26aulast%3DHayashi%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DK.%2BV.%26aulast%3DDas%26aufirst%3DD.%26aulast%3DHigashi-Kuwata%26aufirst%3DN.%26aulast%3DBulut%26aufirst%3DH.%26aulast%3DAoki-Ogata%26aufirst%3DH.%26aulast%3DTakamatsu%26aufirst%3DY.%26aulast%3DYedidi%26aufirst%3DR.%2BS.%26aulast%3DDavis%26aufirst%3DD.%2BA.%26aulast%3DHattori%26aufirst%3DS.%2BI.%26aulast%3DNishida%26aufirst%3DN.%26aulast%3DHasegawa%26aufirst%3DK.%26aulast%3DTakamune%26aufirst%3DN.%26aulast%3DNyalapatla%26aufirst%3DP.%2BR.%26aulast%3DOsswald%26aufirst%3DH.%2BL.%26aulast%3DJono%26aufirst%3DH.%26aulast%3DSaito%26aufirst%3DH.%26aulast%3DYarchoan%26aufirst%3DR.%26aulast%3DMisumi%26aufirst%3DS.%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DMitsuya%26aufirst%3DH.%26atitle%3DA%2520novel%2520central%2520nervous%2520system-penetrating%2520protease%2520inhibitor%2520overcomes%2520human%2520immunodeficiency%2520virus%25201%2520resistance%2520with%2520unprecedented%2520aM%2520to%2520pM%2520potency%26jtitle%3DElife%26date%3D2017%26volume%3D6%26doi%3D10.7554%2FeLife.28020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyalapatla, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovela, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brindisi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osswald, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agniswamy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuya, H.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of highly potent HIV-1 protease inhibitors containing tricyclic fused ring systems as novel P2 ligands: structure-activity studies, biological and X-ray structural analysis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4561</span>– <span class="NLM_lpage">4577</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00298</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00298" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpsVahtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4561-4577&author=A.+K.+Ghoshauthor=P.+R.+Nyalapatlaauthor=S.+Kovelaauthor=K.+V.+Raoauthor=M.+Brindisiauthor=H.+L.+Osswaldauthor=M.+Amanoauthor=M.+Aokiauthor=J.+Agniswamyauthor=Y.-F.+Wangauthor=I.+T.+Weberauthor=H.+Mitsuya&title=Design+and+synthesis+of+highly+potent+HIV-1+protease+inhibitors+containing+tricyclic+fused+ring+systems+as+novel+P2+ligands%3A+structure-activity+studies%2C+biological+and+X-ray+structural+analysis&doi=10.1021%2Facs.jmedchem.8b00298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Highly Potent HIV-1 Protease Inhibitors Containing Tricyclic Fused Ring Systems as Novel P2 Ligands: Structure-Activity Studies, Biological and X-ray Structural Analysis</span></div><div class="casAuthors">Ghosh, Arun K.; R. Nyalapatla, Prasanth; Kovela, Satish; Rao, Kalapala Venkateswara; Brindisi, Margherita; Osswald, Heather L.; Amano, Masayuki; Aoki, Manabu; Agniswamy, Johnson; Wang, Yuan-Fang; Weber, Irene T.; Mitsuya, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4561-4577</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design, synthesis, and biol. evaluation of a new class of HIV-1 protease inhibitors contg. stereochem. defined fused tricyclic polyethers as the P2 ligands and a variety of sulfonamide derivs. as the P2' ligands are described.  A no. of ring sizes and various substituent effects were investigated to enhance the ligand-backbone interactions in the protease active site.  Inhibitors I and II contg. this unprecedented fused 6-5-5 ring system as the P2 ligand, an aminobenzothiazole as the P2' ligand, and a difluorophenylmethyl as the P1 ligand exhibited exceptional enzyme inhibitory potency and maintained excellent antiviral activity against a panel of highly multidrug-resistant HIV-1 variants.  The umbrella-like P2 ligand for these inhibitors has been synthesized efficiently in an optically active form using a Pauson-Khand cyclization reaction as the key step.  The racemic alcs. were resolved efficiently using a lipase catalyzed enzymic resoln.  Two high resoln. X-ray structures of inhibitor-bound HIV-1 protease revealed extensive interactions with the backbone atoms of HIV-1 protease and provided mol. insight into the binding properties of these new inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobuOIwTJfrX7Vg90H21EOLACvtfcHk0lhaw3sNYiwc6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpsVahtb4%253D&md5=55ddb084c94a598748cb922796cffa22</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00298%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DNyalapatla%26aufirst%3DP.%2BR.%26aulast%3DKovela%26aufirst%3DS.%26aulast%3DRao%26aufirst%3DK.%2BV.%26aulast%3DBrindisi%26aufirst%3DM.%26aulast%3DOsswald%26aufirst%3DH.%2BL.%26aulast%3DAmano%26aufirst%3DM.%26aulast%3DAoki%26aufirst%3DM.%26aulast%3DAgniswamy%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.-F.%26aulast%3DWeber%26aufirst%3DI.%2BT.%26aulast%3DMitsuya%26aufirst%3DH.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520highly%2520potent%2520HIV-1%2520protease%2520inhibitors%2520containing%2520tricyclic%2520fused%2520ring%2520systems%2520as%2520novel%2520P2%2520ligands%253A%2520structure-activity%2520studies%252C%2520biological%2520and%2520X-ray%2520structural%2520analysis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4561%26epage%3D4577%26doi%3D10.1021%2Facs.jmedchem.8b00298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyalapatla, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osswald, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martyr, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agniswamy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulut, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuya, H.</span></span> <span> </span><span class="NLM_article-title">Design and development of highly potent HIV-1 protease inhibitors with a crown-like oxotricyclic core as the P2-ligand to combat multidrug-resistant HIV variants</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4267</span>– <span class="NLM_lpage">4278</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00172</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00172" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFKlsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4267-4278&author=A.+K.+Ghoshauthor=K.+V.+Raoauthor=P.+R.+Nyalapatlaauthor=H.+L.+Osswaldauthor=C.+D.+Martyrauthor=M.+Aokiauthor=H.+Hayashiauthor=J.+Agniswamyauthor=Y.+F.+Wangauthor=H.+Bulutauthor=D.+Dasauthor=I.+T.+Weberauthor=H.+Mitsuya&title=Design+and+development+of+highly+potent+HIV-1+protease+inhibitors+with+a+crown-like+oxotricyclic+core+as+the+P2-ligand+to+combat+multidrug-resistant+HIV+variants&doi=10.1021%2Facs.jmedchem.7b00172"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Development of Highly Potent HIV-1 Protease Inhibitors with a Crown-Like Oxotricyclic Core as the P2-Ligand To Combat Multidrug-Resistant HIV Variants</span></div><div class="casAuthors">Ghosh, Arun K.; Rao, Kalapala Venkateswara; Nyalapatla, Prasanth R.; Osswald, Heather L.; Martyr, Cuthbert D.; Aoki, Manabu; Hayashi, Hironori; Agniswamy, Johnson; Wang, Yuan-Fang; Bulut, Haydar; Das, Debananda; Weber, Irene T.; Mitsuya, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4267-4278</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Design, synthesis, and evaluation of a new class of exceptionally potent HIV-1 protease inhibitors is reported.  Inhibitor I displayed superior antiviral activity and drug-resistance profiles.  In fact, this inhibitor showed several orders of magnitude improved antiviral activity over the FDA approved drug, darunavir.  This inhibitor incorporates an unprecedented 6-5-5 ring-fused crown-like tetrahydropyranofuran as the P2 ligand and an aminobenzothiazole as the P2' ligand with the (R)-hydroxyethylsulfonamide isostere.  The crown-like P2 ligand for this inhibitor has been synthesized efficiently in an optically active form using a chiral Diels-Alder catalyst providing a key intermediate in high enantiomeric purity.  Two high resoln. X-ray structures of inhibitor-bound HIV-1 protease revealed extensive interactions with the backbone atoms of HIV-1 protease and provided mol. insight into the binding properties of these new inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpA8W8TXM4SXbVg90H21EOLACvtfcHk0liIFI0m7FFcWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFKlsLw%253D&md5=03b84db839ddd5d19195e5dd8bf43642</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00172&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00172%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DRao%26aufirst%3DK.%2BV.%26aulast%3DNyalapatla%26aufirst%3DP.%2BR.%26aulast%3DOsswald%26aufirst%3DH.%2BL.%26aulast%3DMartyr%26aufirst%3DC.%2BD.%26aulast%3DAoki%26aufirst%3DM.%26aulast%3DHayashi%26aufirst%3DH.%26aulast%3DAgniswamy%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%2BF.%26aulast%3DBulut%26aufirst%3DH.%26aulast%3DDas%26aufirst%3DD.%26aulast%3DWeber%26aufirst%3DI.%2BT.%26aulast%3DMitsuya%26aufirst%3DH.%26atitle%3DDesign%2520and%2520development%2520of%2520highly%2520potent%2520HIV-1%2520protease%2520inhibitors%2520with%2520a%2520crown-like%2520oxotricyclic%2520core%2520as%2520the%2520P2-ligand%2520to%2520combat%2520multidrug-resistant%2520HIV%2520variants%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4267%26epage%3D4278%26doi%3D10.1021%2Facs.jmedchem.7b00172" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapsal, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldridge, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffey, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuya, H.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of potent HIV-1 protease inhibitors incorporating hexahydrofuropyranol-derived high affinity P<sub>2</sub> ligands: structure-activity studies and biological evaluation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">622</span>– <span class="NLM_lpage">634</span>, <span class="refDoi"> DOI: 10.1021/jm1012787</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012787" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslSq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=622-634&author=A.+K.+Ghoshauthor=B.+D.+Chapsalauthor=A.+Baldridgeauthor=M.+P.+Steffeyauthor=D.+E.+Waltersauthor=Y.+Kohauthor=M.+Amanoauthor=H.+Mitsuya&title=Design+and+synthesis+of+potent+HIV-1+protease+inhibitors+incorporating+hexahydrofuropyranol-derived+high+affinity+P2+ligands%3A+structure-activity+studies+and+biological+evaluation&doi=10.1021%2Fjm1012787"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Potent HIV-1 Protease Inhibitors Incorporating Hexahydrofuropyranol-Derived High Affinity P2 Ligands: Structure-Activity Studies and Biological Evaluation</span></div><div class="casAuthors">Ghosh, Arun K.; Chapsal, Bruno D.; Baldridge, Abigail; Steffey, Melinda P.; Walters, D. Eric; Koh, Yasuhiro; Amano, Masayuki; Mitsuya, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">622-634</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Substituted furopyranol, benzofuranol, cyclopentapyranol, and indanol arylsulfonyl phenylpropylcarbamate HIV-1 protease inhibitors such as I are prepd. as HIV-1 protease inhibitors.  Hexahydrofuranopyranol II is prepd. in seven steps from nonracemic cyclopentenofuranone III; conversion to a 4-nitrophenyl carbamate and coupling with an arylsulfonylphenylpropylamine yields I.  I inhibits multidrug resistant HIV-1 strains with comparable effectiveness to the HIV-1 inhibitor darunavir.  A model of I bound to the active site of HIV-1 protease is calcd. based on the crystal structure of a related protease inhibitor bound to HIV-1 protease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrY4F-41rD7jbVg90H21EOLACvtfcHk0liIFI0m7FFcWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslSq&md5=511b20b9bb4c1d31aafc8e969eea9328</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm1012787&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012787%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DChapsal%26aufirst%3DB.%2BD.%26aulast%3DBaldridge%26aufirst%3DA.%26aulast%3DSteffey%26aufirst%3DM.%2BP.%26aulast%3DWalters%26aufirst%3DD.%2BE.%26aulast%3DKoh%26aufirst%3DY.%26aulast%3DAmano%26aufirst%3DM.%26aulast%3DMitsuya%26aufirst%3DH.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520potent%2520HIV-1%2520protease%2520inhibitors%2520incorporating%2520hexahydrofuropyranol-derived%2520high%2520affinity%2520P2%2520ligands%253A%2520structure-activity%2520studies%2520and%2520biological%2520evaluation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D622%26epage%3D634%26doi%3D10.1021%2Fjm1012787" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">King, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prabu-Jeyabalan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nalivaika, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Combating susceptibility to drug resistance: lessons from HIV-1 protease</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1333</span>– <span class="NLM_lpage">1338</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2004.08.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fj.chembiol.2004.08.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=15489160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2cXos1Ciu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=1333-1338&author=N.+M.+Kingauthor=M.+Prabu-Jeyabalanauthor=E.+A.+Nalivaikaauthor=C.+A.+Schiffer&title=Combating+susceptibility+to+drug+resistance%3A+lessons+from+HIV-1+protease&doi=10.1016%2Fj.chembiol.2004.08.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Combating Susceptibility to Drug Resistance Lessons from HIV-1 Protease</span></div><div class="casAuthors">King, Nancy M.; Prabu-Jeyabalan, Moses; Nalivaika, Ellen A.; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1333-1338</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Drug resistance is a major obstacle in modern medicine.  However, resistance is rarely considered in drug development and may inadvertently be facilitated, as many designed inhibitors contact residues that can mutate to confer resistance, without significantly impairing function.  Contemporary drug design often ignores the detailed at. basis for function and primarily focuses on disrupting the target's activity, which is necessary but not sufficient for developing a robust drug.  In this study, we examine the impact of drug-resistant mutations in HIV-1 protease on substrate recognition and demonstrate that most primary active site mutations do not extensively contact substrates, but are crit. to inhibitor binding.  We propose a general, structure-based strategy to reduce the probability of drug resistance by designing inhibitors that interact only with those residues that are essential for function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbsk2AyTcCmbVg90H21EOLACvtfcHk0liIFI0m7FFcWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXos1Ciu78%253D&md5=701599b15cf9ecc2c026ad4eac650a90</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2004.08.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2004.08.010%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DN.%2BM.%26aulast%3DPrabu-Jeyabalan%26aufirst%3DM.%26aulast%3DNalivaika%26aufirst%3DE.%2BA.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DCombating%2520susceptibility%2520to%2520drug%2520resistance%253A%2520lessons%2520from%2520HIV-1%2520protease%26jtitle%3DChem.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D1333%26epage%3D1338%26doi%3D10.1016%2Fj.chembiol.2004.08.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerlits, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wymore, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parks, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blakeley, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovalevsky, A.</span></span> <span> </span><span class="NLM_article-title">Long-range electrostatics-induced two-proton transfer captured by neutron crystallography in an enzyme catalytic site</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">4924</span>– <span class="NLM_lpage">4927</span>, <span class="refDoi"> DOI: 10.1002/anie.201509989</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1002%2Fanie.201509989" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC28XktVaju70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=4924-4927&author=O.+Gerlitsauthor=T.+Wymoreauthor=A.+Dasauthor=C.+H.+Shenauthor=J.+M.+Parksauthor=J.+C.+Smithauthor=K.+L.+Weissauthor=D.+A.+Keenauthor=M.+P.+Blakeleyauthor=J.+M.+Louisauthor=P.+Langanauthor=I.+T.+Weberauthor=A.+Kovalevsky&title=Long-range+electrostatics-induced+two-proton+transfer+captured+by+neutron+crystallography+in+an+enzyme+catalytic+site&doi=10.1002%2Fanie.201509989"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Long-range electrostatics-induced two-proton transfer captured by neutron crystallography in an enzyme catalytic site</span></div><div class="casAuthors">Gerlits, Oksana; Wymore, Troy; Das, Amit; Shen, Chen-Hsiang; Parks, Jerry M.; Smith, Jeremy C.; Weiss, Kevin L.; Keen, David A.; Blakeley, Matthew P.; Louis, John M.; Langan, Paul; Weber, Irene T.; Kovalevsky, Andrey</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4924-4927</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Neutron crystallog. was used to directly locate 2 protons before and after a pH-induced 2-proton transfer between catalytic Asp residues and the OH group of the bound clin. drug, darunavir, located in the catalytic site of enzyme HIV-1 protease.  The 2-proton transfer was triggered by electrostatic effects arising from protonation state changes of surface residues far from the active site.  The mechanism and pH effect were supported by quantum mechanics/mol. mechanics (QM/MM) calcns.  The low-pH proton configuration in the catalytic site was deemed crit. for the catalytic action of this enzyme and may apply more generally to other aspartic proteases.  Neutrons therefore represent a superb probe to obtain structural details for proton transfer reactions in biol. systems at a truly at. level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzmyxWbnMIhLVg90H21EOLACvtfcHk0lhOYug7YHLPpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktVaju70%253D&md5=198ac991fe2274b9f140e13b7e7aae35</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fanie.201509989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201509989%26sid%3Dliteratum%253Aachs%26aulast%3DGerlits%26aufirst%3DO.%26aulast%3DWymore%26aufirst%3DT.%26aulast%3DDas%26aufirst%3DA.%26aulast%3DShen%26aufirst%3DC.%2BH.%26aulast%3DParks%26aufirst%3DJ.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BC.%26aulast%3DWeiss%26aufirst%3DK.%2BL.%26aulast%3DKeen%26aufirst%3DD.%2BA.%26aulast%3DBlakeley%26aufirst%3DM.%2BP.%26aulast%3DLouis%26aufirst%3DJ.%2BM.%26aulast%3DLangan%26aufirst%3DP.%26aulast%3DWeber%26aufirst%3DI.%2BT.%26aulast%3DKovalevsky%26aufirst%3DA.%26atitle%3DLong-range%2520electrostatics-induced%2520two-proton%2520transfer%2520captured%2520by%2520neutron%2520crystallography%2520in%2520an%2520enzyme%2520catalytic%2520site%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D4924%26epage%3D4927%26doi%3D10.1002%2Fanie.201509989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kovalevsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlits, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beltran, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blakeley, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, I. T.</span></span> <span> </span><span class="NLM_article-title">Proton transfer and drug binding details revealed in neutron diffraction studies of wild-type and drug resistant HIV-1 protease</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>634</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">279</span>, <span class="refDoi"> DOI: 10.1016/bs.mie.2019.12.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fbs.mie.2019.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=32093836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A280%3ADC%252BB387ls1KitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=634&publication_year=2020&pages=257-279&author=A.+Kovalevskyauthor=O.+Gerlitsauthor=K.+Beltranauthor=K.+L.+Weissauthor=D.+A.+Keenauthor=M.+P.+Blakeleyauthor=J.+M.+Louisauthor=I.+T.+Weber&title=Proton+transfer+and+drug+binding+details+revealed+in+neutron+diffraction+studies+of+wild-type+and+drug+resistant+HIV-1+protease&doi=10.1016%2Fbs.mie.2019.12.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Proton transfer and drug binding details revealed in neutron diffraction studies of wild-type and drug resistant HIV-1 protease</span></div><div class="casAuthors">Kovalevsky Andrey; Gerlits Oksana; Beltran Kaira; Weiss Kevin L; Keen David A; Blakeley Matthew P; Louis John M; Weber Irene T</div><div class="citationInfo"><span class="NLM_cas:title">Methods in enzymology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">634</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">257-279</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">HIV-1 protease is an essential therapeutic target for the design and development of antiviral inhibitors to treat AIDS.  We used room temperature neutron crystallography to accurately determine hydrogen atom positions in several protease complexes with clinical drugs, amprenavir and darunavir.  Hydrogen bonding interactions were carefully mapped to provide an unprecedented picture of drug binding to the protease target.  We demonstrate that hydrogen atom positions within the enzyme catalytic site can be altered by introducing drug resistant mutations and by protonating surface residues that trigger proton transfer reactions between the catalytic Asp residues and the hydroxyl group of darunavir.  When protein perdeuteration is not feasible, we validate the use of initial H/D exchange with unfolded protein and partial deuteration in pure D2O with hydrogenous glycerol to maximize deuterium incorporation into the protein, with no detrimental effects on the growth of quality crystals suitable for neutron diffraction experiments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQM8q3kfnWTrftos8R3JUhXfW6udTcc2ebW_5P1H7OIx7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB387ls1KitQ%253D%253D&md5=859de4c89f45cbaa73b1120155c96241</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fbs.mie.2019.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.mie.2019.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DKovalevsky%26aufirst%3DA.%26aulast%3DGerlits%26aufirst%3DO.%26aulast%3DBeltran%26aufirst%3DK.%26aulast%3DWeiss%26aufirst%3DK.%2BL.%26aulast%3DKeen%26aufirst%3DD.%2BA.%26aulast%3DBlakeley%26aufirst%3DM.%2BP.%26aulast%3DLouis%26aufirst%3DJ.%2BM.%26aulast%3DWeber%26aufirst%3DI.%2BT.%26atitle%3DProton%2520transfer%2520and%2520drug%2520binding%2520details%2520revealed%2520in%2520neutron%2520diffraction%2520studies%2520of%2520wild-type%2520and%2520drug%2520resistant%2520HIV-1%2520protease%26jtitle%3DMethods%2520Enzymol.%26date%3D2020%26volume%3D634%26spage%3D257%26epage%3D279%26doi%3D10.1016%2Fbs.mie.2019.12.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempf, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sham, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasavanonda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wideburg, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saldivar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norbeck, D. W.</span></span> <span> </span><span class="NLM_article-title">Synthesis and SAR studies of potent HIV protease inhibitors containing novel dimethylphenoxyl acetates as P2 ligands</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">3657</span>– <span class="NLM_lpage">3660</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2003.08.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fj.bmcl.2003.08.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=14552751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnvFGjtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=3657-3660&author=X.+Chenauthor=D.+J.+Kempfauthor=L.+Liauthor=H.+L.+Shamauthor=S.+Vasavanondaauthor=N.+E.+Wideburgauthor=A.+Saldivarauthor=K.+C.+Marshauthor=E.+McDonaldauthor=D.+W.+Norbeck&title=Synthesis+and+SAR+studies+of+potent+HIV+protease+inhibitors+containing+novel+dimethylphenoxyl+acetates+as+P2+ligands&doi=10.1016%2Fj.bmcl.2003.08.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and SAR studies of potent HIV protease inhibitors containing novel dimethylphenoxyl acetates as P2 ligands</span></div><div class="casAuthors">Chen, Xiaoqi; Kempf, Dale J.; Li, Lin; Sham, Hing L.; Vasavanonda, Sudthida; Wideburg, Norman E.; Saldivar, Ayda; Marsh, Kennan C.; McDonald, Edith; Norbeck, Daniel W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3657-3660</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">Iso-Pr substituted 4-thioazolyl valine side chains are highly optimized P2-P3 ligands for C2 symmetry-based HIV protease inhibitors, as exemplified by the drug ritonavir.  Replacement of the side chain with the conformationally constrained hexahydrofurofuranyloxy P2 ligand in combination with a dimethylphenoxyacetate on the other end of the ritonavir core diamine yielded highly potent HIV protease inhibitors.  The in vitro antiviral activity in MT4 cells increased by 10- and 20-fold, resp., in the absence and presence of 50% human serum compared to ritonavir.  The structure-activity relationships of inhibitor series with this combination of ligands were investigated.  Preliminary pharmacokinetic studies in rats indicated rapid elimination of the inhibitors from the blood, and the plasma levels were not significantly enhanced by coadministration with ritonavir.  However, the novel structural features and the high intrinsic antiviral potency of this series provides potential for the future exploration of prodrug strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2XII92gr9NbVg90H21EOLACvtfcHk0lhOYug7YHLPpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnvFGjtb0%253D&md5=f72b5c606df040470e2796bb850edf4d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2003.08.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2003.08.043%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DKempf%26aufirst%3DD.%2BJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DSham%26aufirst%3DH.%2BL.%26aulast%3DVasavanonda%26aufirst%3DS.%26aulast%3DWideburg%26aufirst%3DN.%2BE.%26aulast%3DSaldivar%26aufirst%3DA.%26aulast%3DMarsh%26aufirst%3DK.%2BC.%26aulast%3DMcDonald%26aufirst%3DE.%26aulast%3DNorbeck%26aufirst%3DD.%2BW.%26atitle%3DSynthesis%2520and%2520SAR%2520studies%2520of%2520potent%2520HIV%2520protease%2520inhibitors%2520containing%2520novel%2520dimethylphenoxyl%2520acetates%2520as%2520P2%2520ligands%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2003%26volume%3D13%26spage%3D3657%26epage%3D3660%26doi%3D10.1016%2Fj.bmcl.2003.08.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kempf, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sham, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wideburg, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saldivar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasavanonda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norbeck, D. W.</span></span> <span> </span><span class="NLM_article-title">Evaluation of furofuran as a P2 ligand for symmetry-based HIV protease inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">2847</span>– <span class="NLM_lpage">2852</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(96)00528-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2FS0960-894X%2896%2900528-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADyaK2sXhtlagtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1996&pages=2847-2852&author=X.+Chenauthor=L.+Liauthor=D.+J.+Kempfauthor=H.+Shamauthor=N.+E.+Wideburgauthor=A.+Saldivarauthor=S.+Vasavanondaauthor=K.+C.+Marshauthor=E.+McDonaldauthor=D.+W.+Norbeck&title=Evaluation+of+furofuran+as+a+P2+ligand+for+symmetry-based+HIV+protease+inhibitors&doi=10.1016%2FS0960-894X%2896%2900528-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of furofuran as a P2 ligand for symmetry-based HIV protease inhibitors</span></div><div class="casAuthors">Chen, Xiaoqi; Li, Lin; Kempf, Dale J.; Sham, Hing; Wideburg, Norman E.; Saldivar, Ayda; Vasavanonda, Sudthida; Marsh, Kennan C.; McDonald, Edith; Norbeck, Daniel W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2847-2852</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The hexahydrofurofuranyloxy group was evaluated as a conformationally constrained P2 ligand for symmetry-based HIV protease inhibitors.  A no. of compds. showed nM level activity against HIV in MT4 cells and lower protein binding than the licensed protease inhibitor ritonavir.  However, replacement of 5-thiazole of ritonavir with a furofuran caused a redn. of the bioavailability in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo80kdyUn-rDLVg90H21EOLACvtfcHk0lhgOv_bArIeOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhtlagtA%253D%253D&md5=a577546f996c402549d6537d3e4db3a1</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2896%2900528-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252896%252900528-8%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DKempf%26aufirst%3DD.%2BJ.%26aulast%3DSham%26aufirst%3DH.%26aulast%3DWideburg%26aufirst%3DN.%2BE.%26aulast%3DSaldivar%26aufirst%3DA.%26aulast%3DVasavanonda%26aufirst%3DS.%26aulast%3DMarsh%26aufirst%3DK.%2BC.%26aulast%3DMcDonald%26aufirst%3DE.%26aulast%3DNorbeck%26aufirst%3DD.%2BW.%26atitle%3DEvaluation%2520of%2520furofuran%2520as%2520a%2520P2%2520ligand%2520for%2520symmetry-based%2520HIV%2520protease%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1996%26volume%3D6%26spage%3D2847%26epage%3D2852%26doi%3D10.1016%2FS0960-894X%2896%2900528-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cannizzaro, C. E.</span>; <span class="NLM_string-name">Yang, Z.-Y.</span>; <span class="NLM_string-name">Desai, M. C.</span>; <span class="NLM_string-name">Mitchell, M. L.</span>; <span class="NLM_string-name">Xu, L.</span>; <span class="NLM_string-name">Swaminathan, S.</span>; <span class="NLM_string-name">Chen, J. M.</span></span> <span> </span><span class="NLM_article-title">Antiviral protease inhibitors</span>. PCT Int. App. <span class="NLM_patent">WO2008011117</span>, <span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=C.+E.+Cannizzaro&author=Z.-Y.+Yang&author=M.+C.+Desai&author=M.+L.+Mitchell&author=L.+Xu&author=S.+Swaminathan&author=J.+M.+Chen&title=Antiviral+protease+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCannizzaro%26aufirst%3DC.%2BE.%26atitle%3DAntiviral%2520protease%2520inhibitors%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Özen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haliloǧlu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Dynamics of preferential substrate recognition in HIV-1 protease: redefining the substrate envelope</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>410</i></span>,  <span class="NLM_fpage">726</span>– <span class="NLM_lpage">744</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2011.03.053</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2Fj.jmb.2011.03.053" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=21762811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovF2mtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=410&publication_year=2011&pages=726-744&author=A.+%C3%96zenauthor=T.+Halilo%C7%A7luauthor=C.+A.+Schiffer&title=Dynamics+of+preferential+substrate+recognition+in+HIV-1+protease%3A+redefining+the+substrate+envelope&doi=10.1016%2Fj.jmb.2011.03.053"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamics of Preferential Substrate Recognition in HIV-1 Protease: Redefining the Substrate Envelope</span></div><div class="casAuthors">Ozen, Aysegul; Haliloglu, Turkan; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">410</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">726-744</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Human immunodeficiency virus type 1 (HIV-1) protease (PR) permits viral maturation by processing the gag and gag-pro-pol polyproteins.  HIV-1 PR inhibitors (PIs) are used in combination antiviral therapy but the emergence of drug resistance has limited their efficacy.  The rapid evolution of HIV-1 necessitates consideration of drug resistance in novel drug design.  Drug-resistant HIV-1 PR variants no longer inhibited efficiently, continue to hydrolyze the natural viral substrates.  Though highly diverse in sequence, the HIV-1 PR substrates bind in a conserved three-dimensional shape we termed the substrate envelope.  Earlier, we showed that resistance mutations arise where PIs protrude beyond the substrate envelope, because these regions are crucial for drug binding but not for substrate recognition.  We extend this model by considering the role of protein dynamics in the interaction of HIV-1 PR with its substrates.  We simulated the mol. dynamics of seven PR-substrate complexes to est. the conformational flexibility of the bound substrates.  Interdependence of substrate-protease interactions might compensate for variations in cleavage-site sequences and explain how a diverse set of sequences are recognized as substrates by the same enzyme.  This diversity might be essential for regulating sequential processing of substrates.  We define a dynamic substrate envelope as a more accurate representation of PR-substrate interactions.  This dynamic substrate envelope, described by a probability distribution function, is a powerful tool for drug design efforts targeting ensembles of resistant HIV-1 PR variants with the aim of developing drugs that are less susceptible to resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2avfbPli-5LVg90H21EOLACvtfcHk0lhgOv_bArIeOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovF2mtr0%253D&md5=726b923be7b92c8cbc241a8947c427ff</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2011.03.053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2011.03.053%26sid%3Dliteratum%253Aachs%26aulast%3D%25C3%2596zen%26aufirst%3DA.%26aulast%3DHalilo%25C7%25A7lu%26aufirst%3DT.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DDynamics%2520of%2520preferential%2520substrate%2520recognition%2520in%2520HIV-1%2520protease%253A%2520redefining%2520the%2520substrate%2520envelope%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D410%26spage%3D726%26epage%3D744%26doi%3D10.1016%2Fj.jmb.2011.03.053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paulsen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leidner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ragland, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurt Yilmaz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Interdependence of inhibitor recognition in HIV-1 protease</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2300</span>– <span class="NLM_lpage">2309</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.6b01262</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.6b01262" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltlWgs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=2300-2309&author=J.+L.+Paulsenauthor=F.+Leidnerauthor=D.+A.+Raglandauthor=N.+Kurt+Yilmazauthor=C.+A.+Schiffer&title=Interdependence+of+inhibitor+recognition+in+HIV-1+protease&doi=10.1021%2Facs.jctc.6b01262"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Interdependence of Inhibitor Recognition in HIV-1 Protease</span></div><div class="casAuthors">Paulsen, Janet L.; Leidner, Florian; Ragland, Debra A.; Kurt Yilmaz, Nese; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2300-2309</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. recognition is a highly interdependent process.  Subsite couplings within the active site of proteases are most often revealed through conditional amino acid preferences in substrate recognition.  However, the potential effect of these couplings on inhibition and thus inhibitor design are largely unexplored.  The present study examines the interdependency of subsites in HIV-1 protease using a focused library of protease inhibitors, to aid in future inhibitor design.  Previously a series of darunavir (DRV) analogs was designed to systematically probe the S1' and S2' subsites.  Co-crystal structures of these analogs with HIV-1 protease provide the ideal opportunity to probe subsite interdependency.  All-atom mol. dynamics simulations starting from these structures were performed, and systematically analyzed in terms of at. fluctuations, intermol. interactions and water structure.  These analyses reveal that the S1' subsite highly influences other subsites: the extension of the hydrophobic P1' moiety results in (1) reduced van der Waals contacts in the P2' subsite, (2) more variability in the hydrogen bond frequencies with catalytic residues and the flap water, and (3) changes in the occupancy of conserved water sites both proximal and distal to the active site.  In addn., one of the monomers in this homodimeric enzyme has at. fluctuations more highly correlated with DRV than the other monomer.  These relationships intricately link the HIV-1 protease subsites, and are crit. to understanding of mol. recognition and inhibitor binding.  More broadly, the interdependency of subsite recognition within an active site requires consideration in the selection of chem. moieties in drug design; this strategy is in contrast to what is traditionally done with independent optimization of chem. moieties of an inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtIHU1NPM7ObVg90H21EOLACvtfcHk0lhgOv_bArIeOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltlWgs7g%253D&md5=0710306cc5e01e808fca75475c4ca293</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.6b01262&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.6b01262%26sid%3Dliteratum%253Aachs%26aulast%3DPaulsen%26aufirst%3DJ.%2BL.%26aulast%3DLeidner%26aufirst%3DF.%26aulast%3DRagland%26aufirst%3DD.%2BA.%26aulast%3DKurt%2BYilmaz%26aufirst%3DN.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DInterdependence%2520of%2520inhibitor%2520recognition%2520in%2520HIV-1%2520protease%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2017%26volume%3D13%26spage%3D2300%26epage%3D2309%26doi%3D10.1021%2Facs.jctc.6b01262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haight, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuk, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhagavatula, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannick, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerdesky, F. A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menzia, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parekh, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robbins, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scarpetti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tien, J.-H. J.</span></span> <span> </span><span class="NLM_article-title">Reduction of an enaminone: synthesis of the diamino alcohol core of ritonavir</span>. <i>Org. Process Res. Dev.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">94</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1021/op9802071</span> </span><div class="citationLinks">[<a href="/doi/10.1021/op9802071" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADyaK1MXhslOntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1999&pages=94-100&author=A.+R.+Haightauthor=T.+L.+Stukauthor=M.+S.+Allenauthor=L.+Bhagavatulaauthor=M.+Fitzgeraldauthor=S.+M.+Hannickauthor=F.+A.+J.+Kerdeskyauthor=J.+A.+Menziaauthor=S.+I.+Parekhauthor=T.+A.+Robbinsauthor=D.+Scarpettiauthor=J.-H.+J.+Tien&title=Reduction+of+an+enaminone%3A+synthesis+of+the+diamino+alcohol+core+of+ritonavir&doi=10.1021%2Fop9802071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Reduction of an Enaminone: Synthesis of the Diamino Alcohol Core of Ritonavir</span></div><div class="casAuthors">Haight, Anthony R.; Stuk, Timothy L.; Allen, Michael S.; Bhagavatula, Lakshmi; Fitzgerald, Michael; Hannick, Steven M.; Kerdesky, Francis A. J.; Menzia, Jerome A.; Parekh, Shyamal I.; Robbins, Timothy A.; Scarpetti, David; Tien, Jien-Heh J.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Process Research & Development</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">94-100</span>CODEN:
                <span class="NLM_cas:coden">OPRDFK</span>;
        ISSN:<span class="NLM_cas:issn">1083-6160</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The redn. of (5S)-2-amino-5-dibenzylamino-4-oxo-1,6-diphenylhex-2-ene (I) was optimized for diastereoselectivity and overall conversion to (2S,3S,5S)-5-amino-2-dibenzylamino-3-hydroxy-1,6-diphenylhexane (II).  A two-step redn. sequence is described wherein the enamine is reduced with a borane-sulfonate deriv. followed by redn. of the resulting ketone with sodium borohydride.  The desired II was obtained with 84% diastereoselectivity and an acyclic 1,4 stereoinduction ratio of 14:1.  This methodol. has been used to produce multikilogram quantities of the diamino alc. core of Ritonavir (III) and should be general to the synthesis of related diamino hydroxyethylene isosteres.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVtDKnLMDXYLVg90H21EOLACvtfcHk0lifu6_PDmmGuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhslOntg%253D%253D&md5=a3961137cde012f2c6cc62eeabb3727c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fop9802071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fop9802071%26sid%3Dliteratum%253Aachs%26aulast%3DHaight%26aufirst%3DA.%2BR.%26aulast%3DStuk%26aufirst%3DT.%2BL.%26aulast%3DAllen%26aufirst%3DM.%2BS.%26aulast%3DBhagavatula%26aufirst%3DL.%26aulast%3DFitzgerald%26aufirst%3DM.%26aulast%3DHannick%26aufirst%3DS.%2BM.%26aulast%3DKerdesky%26aufirst%3DF.%2BA.%2BJ.%26aulast%3DMenzia%26aufirst%3DJ.%2BA.%26aulast%3DParekh%26aufirst%3DS.%2BI.%26aulast%3DRobbins%26aufirst%3DT.%2BA.%26aulast%3DScarpetti%26aufirst%3DD.%26aulast%3DTien%26aufirst%3DJ.-H.%2BJ.%26atitle%3DReduction%2520of%2520an%2520enaminone%253A%2520synthesis%2520of%2520the%2520diamino%2520alcohol%2520core%2520of%2520ritonavir%26jtitle%3DOrg.%2520Process%2520Res.%2520Dev.%26date%3D1999%26volume%3D3%26spage%3D94%26epage%3D100%26doi%3D10.1021%2Fop9802071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engle, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J. Q.</span></span> <span> </span><span class="NLM_article-title">Ligand-accelerated C-H activation reactions: evidence for a switch of mechanism</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">14137</span>– <span class="NLM_lpage">14151</span>, <span class="refDoi"> DOI: 10.1021/ja105044s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja105044s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFygsbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=14137-14151&author=K.+M.+Engleauthor=D.+H.+Wangauthor=J.+Q.+Yu&title=Ligand-accelerated+C-H+activation+reactions%3A+evidence+for+a+switch+of+mechanism&doi=10.1021%2Fja105044s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-Accelerated C-H Activation Reactions: Evidence for a Switch of Mechanism</span></div><div class="casAuthors">Engle, Keary M.; Wang, Dong-Hui; Yu, Jin-Quan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">14137-14151</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Initial rate studies have revealed dramatic acceleration in aerobic Pd(II)-catalyzed C-H olefination reactions of phenylacetic acids when mono-N-protected amino acids are used as ligands.  In light of these findings, systematic ligand tuning was undertaken, which has resulted in drastic improvements in substrate scope, reaction rate, and catalyst turnover.  We present evidence from intermol. competition studies and kinetic isotope effect expts. that implies that the obsd. rate increases are a result of acceleration in the C-H cleavage step.  Furthermore, these studies suggest that the origin of this phenomenon is a change in the mechanism of C-H cleavage from electrophilic palladation to proton abstraction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_PdoqifF8irVg90H21EOLACvtfcHk0lifu6_PDmmGuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFygsbzM&md5=b4011c8813e2fad95a9a7cb423d2623b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fja105044s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja105044s%26sid%3Dliteratum%253Aachs%26aulast%3DEngle%26aufirst%3DK.%2BM.%26aulast%3DWang%26aufirst%3DD.%2BH.%26aulast%3DYu%26aufirst%3DJ.%2BQ.%26atitle%3DLigand-accelerated%2520C-H%2520activation%2520reactions%253A%2520evidence%2520for%2520a%2520switch%2520of%2520mechanism%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D14137%26epage%3D14151%26doi%3D10.1021%2Fja105044s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Windsor, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raines, R. T.</span></span> <span> </span><span class="NLM_article-title">Fluorogenic Assay for Inhibitors of HIV-1 Protease with Sub-picomolar Affinity</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">11286</span>, <span class="refDoi"> DOI: 10.1038/srep11286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1038%2Fsrep11286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=26261098" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVKhs77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=11286&author=I.+W.+Windsorauthor=R.+T.+Raines&title=Fluorogenic+Assay+for+Inhibitors+of+HIV-1+Protease+with+Sub-picomolar+Affinity&doi=10.1038%2Fsrep11286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorogenic Assay for Inhibitors of HIV-1 Protease with Sub-picomolar Affinity</span></div><div class="casAuthors">Windsor, Ian W.; Raines, Ronald T.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11286</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A fluorogenic substrate for HIV-1 protease was designed and used as the basis for a hypersensitive assay.  The substrate exhibits a kcat of 7.4 s-1, KM of 15 μM, and an increase in fluorescence intensity of 104-fold upon cleavage, thus providing sensitivity that is unmatched in a continuous assay of HIV-1 protease.  These properties enabled the enzyme concn. in an activity assay to be reduced to 25 pM, which is close to the Kd value of the protease dimer.  By fitting inhibition data to Morrison's equation, Ki values of amprenavir, darunavir, and tipranavir were detd. to be 135, 10, and 82 pM, resp.  This assay, which is capable of measuring Ki values as low as 0.25 pM, is well-suited for characterizing the next generation of HIV-1 protease inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjTs91FgAshbVg90H21EOLACvtfcHk0lifu6_PDmmGuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVKhs77F&md5=0371262982d14e659af3d60f2691a3c2</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fsrep11286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep11286%26sid%3Dliteratum%253Aachs%26aulast%3DWindsor%26aufirst%3DI.%2BW.%26aulast%3DRaines%26aufirst%3DR.%2BT.%26atitle%3DFluorogenic%2520Assay%2520for%2520Inhibitors%2520of%2520HIV-1%2520Protease%2520with%2520Sub-picomolar%2520Affinity%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D11286%26doi%3D10.1038%2Fsrep11286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Curling of flap tips in HIV-1 protease as a mechanism for substrate entry and tolerance of drug resistance</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1259</span>– <span class="NLM_lpage">1265</span>, <span class="refDoi"> DOI: 10.1016/S0969-2126(00)00537-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2FS0969-2126%2800%2900537-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=11188690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhtFentrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2000&pages=1259-1265&author=W.+R.+P.+Scottauthor=C.+A.+Schiffer&title=Curling+of+flap+tips+in+HIV-1+protease+as+a+mechanism+for+substrate+entry+and+tolerance+of+drug+resistance&doi=10.1016%2FS0969-2126%2800%2900537-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Curling of Flap Tips in HIV-1 Protease as a Mechanism for Substrate Entry and Tolerance of Drug Resistance</span></div><div class="casAuthors">Scott, W. R. P.; Schiffer, C. A.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1259-1265</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Background: The human immunodeficiency virus type 1 (HIV-1) protease is an essential viral protein that is a major drug target in the fight against Acquired Immune Deficiency Syndrome (AIDS).  Access to the active site of this homodimeric enzyme is gained when two large flaps, one from each monomer, open.  The flap movements are therefore central to the function of the enzyme, yet detg. how these flaps move at an at. level has not been exptl. possible.  In the present study, the authors observe the flaps of HIV-1 protease completely opening during a 10 ns solvated mol. dynamics simulation starting from the unliganded crystal structure.  This movement is on the time scale obsd. by NMR relaxation data.  The highly flexible tips of the flaps, with the sequence Gly-Gly-Ile-Gly-Gly, are seen curling back into the protein and thereby burying many hydrophobic residues.  This curled-in conformational change has never been previously described.  Previous models of this movement, with the flaps as rigid levers, are not consistent with the exptl. data.  The residues that participate in this hydrophobic cluster as a result of the conformational change are highly sensitive to mutation and often contribute to drug resistance when they do change.  However, several of these residues are not part of the active site cavity, and their essential role in causing drug resistance could possibly be rationalized if this conformational change actually occurs.  Trapping HIV-1 protease in this inactive conformation would provide a unique opportunity for future drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp79jBAv68z07Vg90H21EOLACvtfcHk0lh60wg0tU1hag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhtFentrY%253D&md5=84338adcf206539ad8a71ce378eca4c0</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2800%2900537-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252800%252900537-2%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DW.%2BR.%2BP.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DCurling%2520of%2520flap%2520tips%2520in%2520HIV-1%2520protease%2520as%2520a%2520mechanism%2520for%2520substrate%2520entry%2520and%2520tolerance%2520of%2520drug%2520resistance%26jtitle%3DStructure%26date%3D2000%26volume%3D8%26spage%3D1259%26epage%3D1265%26doi%3D10.1016%2FS0969-2126%2800%2900537-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span> <span> </span><span class="NLM_article-title">Structural insights into HIV-1 protease flap opening processes and key intermediates</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">45121</span>– <span class="NLM_lpage">45128</span>, <span class="refDoi"> DOI: 10.1039/C7RA09691G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1039%2FC7RA09691G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeqt7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=45121-45128&author=Y.+Yuauthor=J.+Wangauthor=Z.+Chenauthor=G.+Wangauthor=Q.+Shaoauthor=J.+Shiauthor=W.+Zhu&title=Structural+insights+into+HIV-1+protease+flap+opening+processes+and+key+intermediates&doi=10.1039%2FC7RA09691G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into HIV-1 protease flap opening processes and key intermediates</span></div><div class="casAuthors">Yu, Yuqi; Wang, Jinan; Chen, Zhaoqiang; Wang, Guimin; Shao, Qiang; Shi, Jiye; Zhu, Weiliang</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">71</span>),
    <span class="NLM_cas:pages">45121-45128</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">HIV-1 protease (PR) is an effective drug target for antiviral inhibitors.  The conformational dynamics in the flaps of HIV-1 PR plays a crucial role in the mechanism of substrate binding.  Here, the structural properties of the functionally important intermediate states of the flap opening transition of HIV-1 PR have been characterized by enhanced sampling mol. dynamics simulation as well as long-time conventional non-enhanced simulations at at. level.  Not only crystallog. measured "closed" and "semi-open" structures but also a novel "curled" structure of HIV-1 PR is captured by both kinds of simulations qual. and quant.  The observation of the "curled" intermediate state helps to connect all other functionally important states to provide an integrated view of the transition pathway of the flap opening of HIV-1 PR (closed → curled → semi-open → fully open).  The key residue-residue interactions which are broken or formed in the transition are analyzed to reveal the inherent driving force for the protein conformational transition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5kSh0BjTWNLVg90H21EOLACvtfcHk0lh60wg0tU1hag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeqt7nN&md5=afbad6dff4ee880b3906bf08cdf2a7fd</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1039%2FC7RA09691G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7RA09691G%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DShao%26aufirst%3DQ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DW.%26atitle%3DStructural%2520insights%2520into%2520HIV-1%2520protease%2520flap%2520opening%2520processes%2520and%2520key%2520intermediates%26jtitle%3DRSC%2520Adv.%26date%3D2017%26volume%3D7%26spage%3D45121%26epage%3D45128%26doi%3D10.1039%2FC7RA09691G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Özen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurt Yilmaz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Structural basis and distal effects of Gag substrate coevolution in drug resistance to HIV-1 protease</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">15993</span>– <span class="NLM_lpage">15998</span>, <span class="refDoi"> DOI: 10.1073/pnas.1414063111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1073%2Fpnas.1414063111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=25355911" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVemsrnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=15993-15998&author=A.+%C3%96zenauthor=K.-H.+Linauthor=N.+Kurt+Yilmazauthor=C.+A.+Schiffer&title=Structural+basis+and+distal+effects+of+Gag+substrate+coevolution+in+drug+resistance+to+HIV-1+protease&doi=10.1073%2Fpnas.1414063111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis and distal effects of Gag substrate coevolution in drug resistance to HIV-1 protease</span></div><div class="casAuthors">Ozen, Aysegul; Lin, Kuan-Hung; Yilmaz, Nese Kurt; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">15993-15998</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Drug resistance mutations in response to HIV-1 protease inhibitors are selected not only in the drug target but elsewhere in the viral genome, esp. at the protease cleavage sites in the precursor protein Gag.  To understand the mol. basis of this protease-substrate coevolution, we solved the crystal structures of drug resistant I50V/A71V HIV-1 protease with p1-p6 substrates bearing coevolved mutations.  Analyses of the protease-substrate interactions reveal that compensatory coevolved mutations in the substrate do not restore interactions lost due to protease mutations, but instead establish other interactions that are not restricted to the site of mutation.  Mutation of a substrate residue has distal effects on other residues' interactions as well, including through the induction of a conformational change in the protease.  Addnl., mol. dynamics simulations suggest that restoration of active site dynamics is an addnl. constraint in the selection of coevolved mutations.  Hence, protease-substrate coevolution permits mutational, structural, and dynamic changes via mol. mechanisms that involve distal effects contributing to drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-njLLQk8frLVg90H21EOLACvtfcHk0lh60wg0tU1hag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVemsrnE&md5=b7ebcc341026ba01cd0453883dcca480</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1414063111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1414063111%26sid%3Dliteratum%253Aachs%26aulast%3D%25C3%2596zen%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DK.-H.%26aulast%3DKurt%2BYilmaz%26aufirst%3DN.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DStructural%2520basis%2520and%2520distal%2520effects%2520of%2520Gag%2520substrate%2520coevolution%2520in%2520drug%2520resistance%2520to%2520HIV-1%2520protease%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D15993%26epage%3D15998%26doi%3D10.1073%2Fpnas.1414063111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">King, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prabu-Jeyabalan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nalivaika, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zepp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heefner, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Lack of synergy for inhibitors targeting a multi-drug-resistant HIV-1 protease</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">418</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1110/ps.25502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1110%2Fps.25502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=11790852" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD38XosFyrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2002&pages=418-429&author=N.+M.+Kingauthor=L.+Melnickauthor=M.+Prabu-Jeyabalanauthor=E.+A.+Nalivaikaauthor=S.+S.+Yangauthor=Y.+Gaoauthor=X.+Nieauthor=C.+Zeppauthor=D.+L.+Heefnerauthor=C.+A.+Schiffer&title=Lack+of+synergy+for+inhibitors+targeting+a+multi-drug-resistant+HIV-1+protease&doi=10.1110%2Fps.25502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Lack of synergy for inhibitors targeting a multi-drug-resistant HIV-1 protease</span></div><div class="casAuthors">King, Nancy M.; Melnick, Laurence; Prabu-Jeyabalan, Moses; Nalivaika, Ellen A.; Yang, Shiow-Shong; Gao, Yun; Nie, Xiaoyi; Zepp, Charles; Heefner, Donald L.; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">418-429</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The three-dimensional structures of indinavir and three newly synthesized indinavir analogs in complex with a multi-drug-resistant variant (L63P, V82T, I84V) of HIV-1 protease were detd. to ∼2.2 Å resoln.  Two of the three analogs have only a single modification of indinavir, and their binding affinities to the variant HIV-1 protease are enhanced over that of indinavir.  However, when both modifications were combined into a single compd., the binding affinity to the protease variant was reduced.  On close examn., the structural rearrangements in the protease that occur in the tightest binding inhibitor complex are mutually exclusive with the structural rearrangements seen in the second tightest inhibitor complex.  This occurs as adaptations in the S1 pocket of one monomer propagate through the dimer and affect the conformation of the S1 loop near P81 of the other monomer.  Therefore, structural rearrangements that occur within the protease when it binds to an inhibitor with a single modification must be accounted for in the design of inhibitors with multiple modifications.  This consideration is necessary to develop inhibitors that bind sufficiently tightly to drug-resistant variants of HIV-1 protease to potentially become the next generation of therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRhi_yayOc3bVg90H21EOLACvtfcHk0ljav_ixP3RFBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosFyrsg%253D%253D&md5=9855843d5da7ecf81a51aa0734175cc0</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1110%2Fps.25502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.25502%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DN.%2BM.%26aulast%3DMelnick%26aufirst%3DL.%26aulast%3DPrabu-Jeyabalan%26aufirst%3DM.%26aulast%3DNalivaika%26aufirst%3DE.%2BA.%26aulast%3DYang%26aufirst%3DS.%2BS.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DNie%26aufirst%3DX.%26aulast%3DZepp%26aufirst%3DC.%26aulast%3DHeefner%26aufirst%3DD.%2BL.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DLack%2520of%2520synergy%2520for%2520inhibitors%2520targeting%2520a%2520multi-drug-resistant%2520HIV-1%2520protease%26jtitle%3DProtein%2520Sci.%26date%3D2002%26volume%3D11%26spage%3D418%26epage%3D429%26doi%3D10.1110%2Fps.25502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rose, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, C. S.</span></span> <span> </span><span class="NLM_article-title">Regulation of autoproteolysis of the HIV-1 and HIV-2 proteases with engineered amino acid substitutions</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>268</i></span>,  <span class="NLM_fpage">11939</span>– <span class="NLM_lpage">11945</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=8505318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADyaK3sXltFWqtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=268&publication_year=1993&pages=11939-11945&author=J.+R.+Roseauthor=R.+Saltoauthor=C.+S.+Craik&title=Regulation+of+autoproteolysis+of+the+HIV-1+and+HIV-2+proteases+with+engineered+amino+acid+substitutions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of autoproteolysis of the HIV-1 and HIV-2 proteases with engineered amino acid substitutions</span></div><div class="casAuthors">Rose, Jason R.; Salto, Rafael; Craik, Charles S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">268</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">11939-45</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Autoproteolysis of the retroviral aspartyl proteases is a major obstacle to purifn. and anal. of these enzymes.  A mutagenic approach to rendering autolytic cleavage sites less labile was applied to the primary cleavage site between Leu5 and Trp6 in human immunodeficiency virus-1 (HIV-1) protease.  From predictions based on known substrates it was concluded that amino acids Lys or Ser in place of Gln at position 7 would prevent cleavage at the Leu5-Trp6 peptide bond, therefore stabilizing the protein.  Autoproteolytic stability was enhanced at least 100-fold by these mutations.  At longer time points the protease was degraded at secondary sites which contained adequate substrate sequences but were conformationally restricted.  Conversely, a mutation in HIV-2 protease which changed Lys7 to Gln rendered the protein 3-fold less stable and shifted the position of the initial autoproteolytic cleavage from Phe3-Ser4 to Leu5-Trp6.  The effects of these mutations demonstrate that small changes in protein sequence can have a major impact on their autoproteolytic stability.  The work described here suggests a general method for stabilizing proteases and perhaps other recombinantly produced proteins to autolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM8gNMxq7onbVg90H21EOLACvtfcHk0ljav_ixP3RFBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXltFWqtbc%253D&md5=2b87aa648119fbad8d132accb82c1c7d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DJ.%2BR.%26aulast%3DSalto%26aufirst%3DR.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26atitle%3DRegulation%2520of%2520autoproteolysis%2520of%2520the%2520HIV-1%2520and%2520HIV-2%2520proteases%2520with%2520engineered%2520amino%2520acid%2520substitutions%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1993%26volume%3D268%26spage%3D11939%26epage%3D11945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hui, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomasselli, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reardon, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lull, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunner, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomich, C.-S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinrikson, R. L.</span></span> <span> </span><span class="NLM_article-title">Large scale purification and refolding of HIV-1 protease from Escherichia coli inclusion bodies</span>. <i>J. Protein Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">323</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.1007/BF01028194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1007%2FBF01028194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=8397790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADyaK3sXlvFyhtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1993&pages=323-327&author=J.+O.+Huiauthor=A.+G.+Tomasselliauthor=I.+M.+Reardonauthor=J.+M.+Lullauthor=D.+P.+Brunnerauthor=C.-S.+C.+Tomichauthor=R.+L.+Heinrikson&title=Large+scale+purification+and+refolding+of+HIV-1+protease+from+Escherichia+coli+inclusion+bodies&doi=10.1007%2FBF01028194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Large scale purification and refolding of HIV-1 protease from Escherichia coli inclusion bodies</span></div><div class="casAuthors">Hui, John O.; Tomasselli, Alfredo G.; Reardon, Ilene M.; Lull, June M.; Brunner, David P.; Tomich, Che Shen C.; Heinrikson, Robert L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Protein Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">323-7</span>CODEN:
                <span class="NLM_cas:coden">JPCHD2</span>;
        ISSN:<span class="NLM_cas:issn">0277-8033</span>.
    </div><div class="casAbstract">Retropepsin (I), the aspartic protease encoded by human immunodeficiency virus type 1 (HIV-1), was engineered in E. coli as a construct in which the natural 99-residue polypeptide was preceded by an N-terminal methionine initiator.  Inclusion bodies harboring recombinant HIV-1 I were dissolved in 50% HOAc and the soln. was subjected to gel filtration on a column of Sephadex G-75.  I, eluted in the 2nd of 2 peaks, migrated in SDS-PAGE as a single sharp band of mol. wt. ∼10,000.  Purified HIV-1 I was refolded into an active enzyme by dilg. a soln. of the protein in 50% HOAc with 25 vols. of buffer at pH 5.5.  This method of purifn., which was also applied to the purifn. of HIV-2 I, provides a single-step procedure to produce 100-mg quantities of fully active enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXYIBwJp92ELVg90H21EOLACvtfcHk0ljav_ixP3RFBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXlvFyhtrk%253D&md5=61cd315aa7b58c9bf84bb1f6c614be8a</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1007%2FBF01028194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01028194%26sid%3Dliteratum%253Aachs%26aulast%3DHui%26aufirst%3DJ.%2BO.%26aulast%3DTomasselli%26aufirst%3DA.%2BG.%26aulast%3DReardon%26aufirst%3DI.%2BM.%26aulast%3DLull%26aufirst%3DJ.%2BM.%26aulast%3DBrunner%26aufirst%3DD.%2BP.%26aulast%3DTomich%26aufirst%3DC.-S.%2BC.%26aulast%3DHeinrikson%26aufirst%3DR.%2BL.%26atitle%3DLarge%2520scale%2520purification%2520and%2520refolding%2520of%2520HIV-1%2520protease%2520from%2520Escherichia%2520coli%2520inclusion%2520bodies%26jtitle%3DJ.%2520Protein%2520Chem.%26date%3D1993%26volume%3D12%26spage%3D323%26epage%3D327%26doi%3D10.1007%2FBF01028194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rusere, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockbaum, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosovrasti, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spielvogel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nalivaika, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanstrom, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yilmaz, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, A.</span></span> <span> </span><span class="NLM_article-title">HIV-1 protease inhibitors incorporating stereochemically defined P2’ ligands to optimize hydrogen bonding in the substrate envelope</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">8062</span>– <span class="NLM_lpage">8079</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00838</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00838" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFamsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=8062-8079&author=L.+N.+Rusereauthor=G.+J.+Lockbaumauthor=S.+K.+Leeauthor=M.+Henesauthor=K.+Kosovrastiauthor=E.+Spielvogelauthor=E.+A.+Nalivaikaauthor=R.+Swanstromauthor=N.+K.+Yilmazauthor=C.+A.+Schifferauthor=A.+Ali&title=HIV-1+protease+inhibitors+incorporating+stereochemically+defined+P2%E2%80%99+ligands+to+optimize+hydrogen+bonding+in+the+substrate+envelope&doi=10.1021%2Facs.jmedchem.9b00838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">HIV-1 Protease Inhibitors Incorporating Stereochemically Defined P2' Ligands To Optimize Hydrogen Bonding in the Substrate Envelope</span></div><div class="casAuthors">Rusere, Linah N.; Lockbaum, Gordon J.; Lee, Sook-Kyung; Henes, Mina; Kosovrasti, Klajdi; Spielvogel, Ean; Nalivaika, Ellen A.; Swanstrom, Ronald; Yilmaz, Nese Kurt; Schiffer, Celia A.; Ali, Akbar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8062-8079</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A structure-guided design strategy was used to improve the resistance profile of HIV-1 protease inhibitors by optimizing hydrogen bonding and van der Waals interactions with the protease while staying within the substrate envelope.  Stereoisomers of 4-(1-hydroxyethyl)benzene and 4-(1,2-dihydroxyethyl)benzene moieties were explored as P2' ligands providing pairs of diastereoisomers epimeric at P2', which exhibited distinct potency profiles depending on the configuration of the hydroxyl group and size of the P1' group.  While compds. with the 4-(1-hydroxyethyl)benzene P2' moiety maintained excellent antiviral potency against a panel of multidrug-resistant HIV-1 strains, analogs with the polar 4-(1,2-dihydroxyethyl)benzene moiety were less potent, and only the (R)-epimer incorporating a larger 2-ethylbutyl P1' group showed improved potency.  Crystal structures of protease-inhibitor complexes revealed strong hydrogen bonding interactions of both (R)- and (S)-stereoisomers of the hydroxyethyl group with Asp30'.  Notably, the (R)-dihydroxyethyl group was involved in a unique pattern of direct hydrogen bonding interactions with the backbone amides of Asp29' and Asp30'.  The SAR data and anal. of crystal structures provide insights for optimizing these promising HIV-1 protease inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYYCrKYi2QMbVg90H21EOLACvtfcHk0ljav_ixP3RFBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFamsrrJ&md5=d4a719a7aaf985b4ef27c7613f68a37d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00838%26sid%3Dliteratum%253Aachs%26aulast%3DRusere%26aufirst%3DL.%2BN.%26aulast%3DLockbaum%26aufirst%3DG.%2BJ.%26aulast%3DLee%26aufirst%3DS.%2BK.%26aulast%3DHenes%26aufirst%3DM.%26aulast%3DKosovrasti%26aufirst%3DK.%26aulast%3DSpielvogel%26aufirst%3DE.%26aulast%3DNalivaika%26aufirst%3DE.%2BA.%26aulast%3DSwanstrom%26aufirst%3DR.%26aulast%3DYilmaz%26aufirst%3DN.%2BK.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26aulast%3DAli%26aufirst%3DA.%26atitle%3DHIV-1%2520protease%2520inhibitors%2520incorporating%2520stereochemically%2520defined%2520P2%25E2%2580%2599%2520ligands%2520to%2520optimize%2520hydrogen%2520bonding%2520in%2520the%2520substrate%2520envelope%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D8062%26epage%3D8079%26doi%3D10.1021%2Facs.jmedchem.9b00838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decker, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arani, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilby, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saag, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kappes, J. C.</span></span> <span> </span><span class="NLM_article-title">Emergence of resistant human immunodeficiency virus type 1 in patients receiving fusion inhibitor (T-20) monotherapy</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1896</span>– <span class="NLM_lpage">1905</span>, <span class="refDoi"> DOI: 10.1128/AAC.46.6.1896-1905.2002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1128%2FAAC.46.6.1896-1905.2002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=12019106" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD38XktFCmsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2002&pages=1896-1905&author=X.+Weiauthor=J.+M.+Deckerauthor=H.+Liuauthor=Z.+Zhangauthor=R.+B.+Araniauthor=J.+M.+Kilbyauthor=M.+S.+Saagauthor=X.+Wuauthor=G.+M.+Shawauthor=J.+C.+Kappes&title=Emergence+of+resistant+human+immunodeficiency+virus+type+1+in+patients+receiving+fusion+inhibitor+%28T-20%29+monotherapy&doi=10.1128%2FAAC.46.6.1896-1905.2002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Emergence of resistant human immunodeficiency virus type 1 in patients receiving fusion inhibitor (T-20) monotherapy</span></div><div class="casAuthors">Wei, Xiping; Decker, Julie M.; Liu, Hongmei; Zhang, Zee; Arani, Ramin B.; Kilby, J. Michael; Saag, Michael S.; Wu, Xiaoyun; Shaw, George M.; Kappes, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1896-1905</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The synthetic peptide T-20 (enfuvirtide) represents the first of a new class of antiretroviral compds. to demonstrate in vivo potency by targeting a step in viral entry.  T-20 inhibits a conformational change in the human immunodeficiency virus type 1 (HIV-1) transmembrane glycoprotein (gp41) that is required for fusion between HIV-1 and target cell membranes.  The initial phase I clin. trial of T-20 treatment for HIV-infected patients thus provided a unique opportunity to evaluate the emergence of resistant virus in vivo to this novel class of antiretroviral agents.  All four patients who received an intermediate dose of T-20 (30 mg twice daily) had an initial decline in plasma viral load over the first 10 days but a rising trend by day 14, suggestive of selection for resistant virus.  Plasma virus derived from patients enrolled in all dosage groups of the phase I T-20 trial was analyzed by population sequencing before and after treatment.  While no mutations were found within a highly conserved 3-amino-acid sequence (GIV) known to be crit. for fusion at baseline, after 14 days of therapy, virus from one patient in the 30-mg dose group (30-1) developed a mutation in this motif, specifically an aspartic acid (D) substitution for glycine (G) at position 36.  Multiple env clones were derived from the plasma virus of all four patients in the 30-mg dosage group.  Sequence anal. of 49 clones derived from the plasma of patient 30-1 on day 14 revealed that 25 clones contained the G36D mutation, while 8 contained the V38A mutation.  Dual mutations involving G36D and other residues within the HR1 domain were also identified.  In 5 of the 49 env clones, other mutations involving residues 32 (Q32R or Q32H) and 39 (Q39R) were found in combination with G36D.  Cloned env sequences derived from the plasma virus of subject 30-3 also had single mutations in the GIV sequence (V38M and I37V) detectable following therapy with T-20.  The plasma virus from subjects 30-2 and 30-4 did not contain changes within the GIV sequence.  To analyze the biol. resistance properties of these mutations, we developed a novel single-cycle HIV-1 entry assay using JC53BL cells which express β-galactosidase and luciferase under control of the HIV-1 long terminal repeat.  Full-length env clones were derived from the plasma virus of patients 30-1 and 30-3 and used to generate pseudotyped virus stocks.  The mean 50% inhibition concns. (IC50s) for mutants G36D and V38A (patient 30-1) were 2.3 μg/mL and 11.2 μg/mL, resp., statistically significant increases of 9.1- and 45-fold, resp., compared with those of wild-type Env.  The IC50 for the V38 M mutation (patient 30-3) was 7.6 μg/mL, an 8-fold increase compared with that of the wild type.  The I37V mutation resulted in an IC50 3.2-fold greater than that of the wild type.  Envs with double mutations (Q32R plus G36D and Q32H plus G36D) exhibited a level of resistance similar to that of G36D alone.  These findings provide the first evidence for the rapid emergence of clin. resistance to a novel class of HIV-1 entry inhibitors and may be relevant to future treatment strategies involving these agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTmBSTKEK3v7Vg90H21EOLACvtfcHk0lgks6-4ICQ--Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XktFCmsb8%253D&md5=65977bf843ca8ec9411a88b50ef71206</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1128%2FAAC.46.6.1896-1905.2002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.46.6.1896-1905.2002%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DX.%26aulast%3DDecker%26aufirst%3DJ.%2BM.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DArani%26aufirst%3DR.%2BB.%26aulast%3DKilby%26aufirst%3DJ.%2BM.%26aulast%3DSaag%26aufirst%3DM.%2BS.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DShaw%26aufirst%3DG.%2BM.%26aulast%3DKappes%26aufirst%3DJ.%2BC.%26atitle%3DEmergence%2520of%2520resistant%2520human%2520immunodeficiency%2520virus%2520type%25201%2520in%2520patients%2520receiving%2520fusion%2520inhibitor%2520%2528T-20%2529%2520monotherapy%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2002%26volume%3D46%26spage%3D1896%26epage%3D1905%26doi%3D10.1128%2FAAC.46.6.1896-1905.2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanstrom, R.</span></span> <span> </span><span class="NLM_article-title">A strongly transdominant mutation in the human immunodeficiency virus type 1 gag gene defines an Achilles heel in the virus life cycle</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">8536</span>– <span class="NLM_lpage">8543</span>, <span class="refDoi"> DOI: 10.1128/JVI.00317-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1128%2FJVI.00317-09" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=19515760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFahur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2009&pages=8536-8543&author=S.+K.+Leeauthor=J.+Harrisauthor=R.+Swanstrom&title=A+strongly+transdominant+mutation+in+the+human+immunodeficiency+virus+type+1+gag+gene+defines+an+Achilles+heel+in+the+virus+life+cycle&doi=10.1128%2FJVI.00317-09"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A strongly transdominant mutation in the human immunodeficiency virus type 1 gag gene defines an Achilles heel in the virus life cycle</span></div><div class="casAuthors">Lee, Sook-Kyung; Harris, Janera; Swanstrom, Ronald</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8536-8543</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The human immunodeficiency virus type 1 (HIV-1) protease (PR) makes five obligatory cleavages in the viral Gag polyprotein precursor.  The cleavage events release the virion structural proteins from the precursor and allow the virion to undergo maturation to become infectious.  The protease cleavage between the matrix protein (MA) domain and the adjacent capsid protein (CA) domain releases CA from the membrane-anchored MA and allows the N terminus of CA to refold into a structure that facilitates the formation of hexamer arrays that represent the structural unit of the capsid shell.  In this study, we analyzed the extent to which each of the HIV-1 Gag processing sites must be cleaved by substituting the P1-position amino acid at each processing site with Ile.  A mutation that blocks cleavage at the MA/CA processing site (Y132I) displayed a strong transdominant effect when tested in a phenotypic mixing strategy, inhibiting virion infectivity with a 50% inhibitory concn. of only 4% of the mutant relative to the wild type.  This mutation is 10- to 20-fold more potent in phenotypic mixing than an inactivating mutation in the viral protease, the target of many successful inhibitors, and more potent than an inactivating mutation at any of the other Gag cleavage sites.  The transdominant effect is manifested as the assembly of an aberrant virion core.  Virus contg. 20% of the Y132I mutant and 80% of the wild type (to assess the transdominant effect on infectivity) was blocked either before reverse transcription (RT) or at an early RT step.  The ability of a small amt. of the MA/CA fusion protein to poison the oligomeric assembly of infectious virus identifies an essential step in the complex process of virion formation and maturation.  The effect of a small-mol. inhibitor that is able to block MA/CA cleavage even partially would be amplified by this transdominant neg. effect on the highly orchestrated process of virion assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLsJRGkg0icbVg90H21EOLACvtfcHk0lgks6-4ICQ--Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFahur3K&md5=34901ec3b0bac668f50a3d36bc0b341b</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1128%2FJVI.00317-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00317-09%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DS.%2BK.%26aulast%3DHarris%26aufirst%3DJ.%26aulast%3DSwanstrom%26aufirst%3DR.%26atitle%3DA%2520strongly%2520transdominant%2520mutation%2520in%2520the%2520human%2520immunodeficiency%2520virus%2520type%25201%2520gag%2520gene%2520defines%2520an%2520Achilles%2520heel%2520in%2520the%2520virus%2520life%2520cycle%26jtitle%3DJ.%2520Virol.%26date%3D2009%26volume%3D83%26spage%3D8536%26epage%3D8543%26doi%3D10.1128%2FJVI.00317-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lockbaum, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leidner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusere, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosovrasti, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nachum, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nalivaika, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yilmaz, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Structural Adaptation of Darunavir Analogues against Primary Mutations in HIV-1 Protease</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">316</span>– <span class="NLM_lpage">325</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.8b00336</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.8b00336" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFWrsLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&pages=316-325&author=G.+J.+Lockbaumauthor=F.+Leidnerauthor=L.+N.+Rusereauthor=M.+Henesauthor=K.+Kosovrastiauthor=G.+S.+Nachumauthor=E.+A.+Nalivaikaauthor=A.+Aliauthor=N.+K.+Yilmazauthor=C.+A.+Schiffer&title=Structural+Adaptation+of+Darunavir+Analogues+against+Primary+Mutations+in+HIV-1+Protease&doi=10.1021%2Facsinfecdis.8b00336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Adaptation of Darunavir Analogues against Primary Mutations in HIV-1 Protease</span></div><div class="casAuthors">Lockbaum, Gordon J.; Leidner, Florian; Rusere, Linah N.; Henes, Mina; Kosovrasti, Klajdi; Nachum, Gily S.; Nalivaika, Ellen A.; Ali, Akbar; Yilmaz, Nese Kurt; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">316-325</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HIV-1 protease is one of the prime targets of agents used in antiretroviral therapy against HIV.  However, under selective pressure of protease inhibitors, primary mutations at the active site weaken inhibitor binding to confer resistance.  Darunavir (DRV) is the most potent HIV-1 protease inhibitor in clinic; resistance is limited, as DRV fits well within the substrate envelope.  Nevertheless, resistance is obsd. due to hydrophobic changes at residues including I50, V82, and I84 that line the S1/S1' pocket within the active site.  Through enzyme inhibition assays and a series of 12 crystal structures, the authors interrogated susceptibility of DRV and two potent analogs to primary S1' mutations.  The analogs had modifications at the hydrophobic P1' moiety compared to DRV to better occupy the unexploited space in the S1' pocket where the primary mutations were located.  Considerable losses of potency were obsd. against protease variants with I84V and I50V mutations for all three inhibitors.  The crystal structures revealed an unexpected conformational change in the flap region of I50V protease bound to the analog with the largest P1' moiety, indicating interdependency between the S1' subsite and the flap region.  Collective anal. of protease-inhibitor interactions in the crystal structures using principle component anal. was able to distinguish inhibitor identity and relative potency solely based on van der Waals contacts.  The authors' results reveal the complexity of the interplay between inhibitor P1' moiety and S1' mutations and validate principle component analyses as a useful tool for distinguishing resistance and inhibitor potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpimZoVRCvWL7Vg90H21EOLACvtfcHk0lh1HbZuwE46KQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFWrsLfJ&md5=498898235e84267a1e528d02ff3b345a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.8b00336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.8b00336%26sid%3Dliteratum%253Aachs%26aulast%3DLockbaum%26aufirst%3DG.%2BJ.%26aulast%3DLeidner%26aufirst%3DF.%26aulast%3DRusere%26aufirst%3DL.%2BN.%26aulast%3DHenes%26aufirst%3DM.%26aulast%3DKosovrasti%26aufirst%3DK.%26aulast%3DNachum%26aufirst%3DG.%2BS.%26aulast%3DNalivaika%26aufirst%3DE.%2BA.%26aulast%3DAli%26aufirst%3DA.%26aulast%3DYilmaz%26aufirst%3DN.%2BK.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DStructural%2520Adaptation%2520of%2520Darunavir%2520Analogues%2520against%2520Primary%2520Mutations%2520in%2520HIV-1%2520Protease%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2019%26volume%3D5%26spage%3D316%26epage%3D325%26doi%3D10.1021%2Facsinfecdis.8b00336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">[20] Processing of X-ray diffraction data collected in oscillation mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=%5B20%5D+Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lh1HbZuwE46KQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3D%255B20%255D%2520Processing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lh1HbZuwE46KQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lh1HbZuwE46KQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lg03Dj9neewaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1074</span>– <span class="NLM_lpage">1080</span>, <span class="refDoi"> DOI: 10.1107/S0907444909029436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1107%2FS0907444909029436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=19770504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=1074-1080&author=N.+W.+Moriartyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adams&title=electronic+Ligand+Builder+and+Optimization+Workbench+%28eLBOW%29%3A+a+tool+for+ligand+coordinate+and+restraint+generation&doi=10.1107%2FS0907444909029436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span></div><div class="casAuthors">Moriarty, Nigel W.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The electronic Ligand Builder and Optimization Workbench (eLBOW) is a program module of the PHENIX suite of computational crystallog. software.  It is designed to be a flexible procedure that uses simple and fast quantum-chem. techniques to provide chem. accurate information for novel and known ligands alike.  A variety of input formats and options allow the attainment of a no. of diverse goals including geometry optimization and generation of restraints.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojnN4AVY_DwrVg90H21EOLACvtfcHk0lg03Dj9neewaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN&md5=3f9a9c6c31c9a4b014a9f151fb6aa8d9</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444909029436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909029436%26sid%3Dliteratum%253Aachs%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3Delectronic%2520Ligand%2520Builder%2520and%2520Optimization%2520Workbench%2520%2528eLBOW%2529%253A%2520a%2520tool%2520for%2520ligand%2520coordinate%2520and%2520restraint%2520generation%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D1074%26epage%3D1080%26doi%3D10.1107%2FS0907444909029436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brünger, A. T.</span></span> <span> </span><span class="NLM_article-title">Free R value: a novel statistical quantity for assessing the accuracy of crystal structures</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">472</span>– <span class="NLM_lpage">475</span>, <span class="refDoi"> DOI: 10.1038/355472a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1038%2F355472a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=18481394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADyaK38XktFOrsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=1992&pages=472-475&author=A.+T.+Br%C3%BCnger&title=Free+R+value%3A+a+novel+statistical+quantity+for+assessing+the+accuracy+of+crystal+structures&doi=10.1038%2F355472a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Free R value:  a novel statistical quantity for assessing the accuracy of crystal structures</span></div><div class="casAuthors">Bruenger, Axel T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6359</span>),
    <span class="NLM_cas:pages">472-5</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">A reliable and unbiased indicator of the accuracy is proposed for fitting at. models for macromol. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmCvKGOC-AKrVg90H21EOLACvtfcHk0lg03Dj9neewaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XktFOrsrc%253D&md5=a382d593937b2394d9354ccba49cdbc3</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2F355472a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F355472a0%26sid%3Dliteratum%253Aachs%26aulast%3DBr%25C3%25BCnger%26aufirst%3DA.%2BT.%26atitle%3DFree%2520R%2520value%253A%2520a%2520novel%2520statistical%2520quantity%2520for%2520assessing%2520the%2520accuracy%2520of%2520crystal%2520structures%26jtitle%3DNature%26date%3D1992%26volume%3D355%26spage%3D472%26epage%3D475%26doi%3D10.1038%2F355472a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leaver-Fay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Block, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snoeyink, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: all-atom contacts and structure validation for proteins and nucleic acids</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">W375</span>– <span class="NLM_lpage">W383</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1093%2Fnar%2Fgkm216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=17452350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A280%3ADC%252BD2svhsVGhtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W375-W383&author=I.+W.+Davisauthor=A.+Leaver-Fayauthor=V.+B.+Chenauthor=J.+N.+Blockauthor=G.+J.+Kapralauthor=X.+Wangauthor=L.+W.+Murrayauthor=W.+B.+Arendallauthor=J.+Snoeyinkauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+contacts+and+structure+validation+for+proteins+and+nucleic+acids&doi=10.1093%2Fnar%2Fgkm216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom contacts and structure validation for proteins and nucleic acids</span></div><div class="casAuthors">Davis Ian W; Leaver-Fay Andrew; Chen Vincent B; Block Jeremy N; Kapral Gary J; Wang Xueyi; Murray Laura W; Arendall W Bryan 3rd; Snoeyink Jack; Richardson Jane S; Richardson David C</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic acids research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Web Server issue</span>),
    <span class="NLM_cas:pages">W375-83</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">MolProbity is a general-purpose web server offering quality validation for 3D structures of proteins, nucleic acids and complexes.  It provides detailed all-atom contact analysis of any steric problems within the molecules as well as updated dihedral-angle diagnostics, and it can calculate and display the H-bond and van der Waals contacts in the interfaces between components.  An integral step in the process is the addition and full optimization of all hydrogen atoms, both polar and nonpolar.  New analysis functions have been added for RNA, for interfaces, and for NMR ensembles.  Additionally, both the web site and major component programs have been rewritten to improve speed, convenience, clarity and integration with other resources.  MolProbity results are reported in multiple forms: as overall numeric scores, as lists or charts of local problems, as downloadable PDB and graphics files, and most notably as informative, manipulable 3D kinemage graphics shown online in the KiNG viewer.  This service is available free to all users at http://molprobity.biochem.duke.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRuyhuetoNS59bHMBt0mMIXfW6udTcc2eac2GAVixjJ6rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svhsVGhtQ%253D%253D&md5=726ca1e7e162ab2a4a05048f493731fb</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm216%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DLeaver-Fay%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DBlock%26aufirst%3DJ.%2BN.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DSnoeyink%26aufirst%3DJ.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520contacts%2520and%2520structure%2520validation%2520for%2520proteins%2520and%2520nucleic%2520acids%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW375%26epage%3DW383%26doi%3D10.1093%2Fnar%2Fgkm216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">PyMOL</span> <i>The PyMOL Molecular Graphics System</i>, Version 2.3; <span class="NLM_publisher-name">Schrödinger, LLC</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=PyMOL+The+PyMOL+Molecular+Graphics+System%2C+Version+2.3%3B+Schr%C3%B6dinger%2C+LLC."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Schrödinger</span> <i>Schrödinger Release 2019–2</i>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York, NY, United States</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=Schr%C3%B6dinger&title=Schr%C3%B6dinger+Release+2019%E2%80%932"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26btitle%3DSchr%25C3%25B6dinger%2520Release%25202019%25E2%2580%25932%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maple, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reboul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lupyan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlgren, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaus, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerutti, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krilov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">OPLS3: a force field providing broad coverage of drug-like small molecules and proteins</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">281</span>– <span class="NLM_lpage">296</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.5b00864</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00864" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVCjtbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=281-296&author=E.+Harderauthor=W.+Dammauthor=J.+Mapleauthor=C.+Wuauthor=M.+Reboulauthor=J.+Y.+Xiangauthor=L.+Wangauthor=D.+Lupyanauthor=M.+K.+Dahlgrenauthor=J.+L.+Knightauthor=J.+W.+Kausauthor=D.+S.+Ceruttiauthor=G.+Krilovauthor=W.+L.+Jorgensenauthor=R.+Abelauthor=R.+A.+Friesner&title=OPLS3%3A+a+force+field+providing+broad+coverage+of+drug-like+small+molecules+and+proteins&doi=10.1021%2Facs.jctc.5b00864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">OPLS3: A Force Field Providing Broad Coverage of Drug-like Small Molecules and Proteins</span></div><div class="casAuthors">Harder, Edward; Damm, Wolfgang; Maple, Jon; Wu, Chuanjie; Reboul, Mark; Xiang, Jin Yu; Wang, Lingle; Lupyan, Dmitry; Dahlgren, Markus K.; Knight, Jennifer L.; Kaus, Joseph W.; Cerutti, David S.; Krilov, Goran; Jorgensen, William L.; Abel, Robert; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">281-296</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The parametrization and validation of the OPLS3 force field for small mols. and proteins are reported.  Enhancements with respect to the previous version (OPLS2.1) include the addn. of off-atom charge sites to represent halogen bonding and aryl nitrogen lone pairs as well as a complete refit of peptide dihedral parameters to better model the native structure of proteins.  To adequately cover medicinal chem. space, OPLS3 employs over an order of magnitude more ref. data and assocd. parameter types relative to other commonly used small mol. force fields (e.g., MMFF and OPLS_2005).  As a consequence, OPLS3 achieves a high level of accuracy across performance benchmarks that assess small mol. conformational propensities and solvation.  The newly fitted peptide dihedrals lead to significant improvements in the representation of secondary structure elements in simulated peptides and native structure stability over a no. of proteins.  Together, the improvements made to both the small mol. and protein force field lead to a high level of accuracy in predicting protein-ligand binding measured over a wide range of targets and ligands (less than 1 kcal/mol RMS error) representing a 30% improvement over earlier variants of the OPLS force field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY_jrOhcBRgrVg90H21EOLACvtfcHk0lgUu66JR-9gZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVCjtbfE&md5=42663f8cfa84b80a67132bbb13b9b7ce</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00864%26sid%3Dliteratum%253Aachs%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DMaple%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DReboul%26aufirst%3DM.%26aulast%3DXiang%26aufirst%3DJ.%2BY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLupyan%26aufirst%3DD.%26aulast%3DDahlgren%26aufirst%3DM.%2BK.%26aulast%3DKnight%26aufirst%3DJ.%2BL.%26aulast%3DKaus%26aufirst%3DJ.%2BW.%26aulast%3DCerutti%26aufirst%3DD.%2BS.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DOPLS3%253A%2520a%2520force%2520field%2520providing%2520broad%2520coverage%2520of%2520drug-like%2520small%2520molecules%2520and%2520proteins%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D281%26epage%3D296%26doi%3D10.1021%2Facs.jctc.5b00864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nalam, M. N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, G. S. K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chellappan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kairys, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Özen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tidor, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rana, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Evaluating the substrate-envelope hypothesis: structural analysis of novel HIV-1 protease inhibitors designed to be robust against drug resistance</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">5368</span>– <span class="NLM_lpage">5378</span>, <span class="refDoi"> DOI: 10.1128/JVI.02531-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1128%2FJVI.02531-09" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=20237088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmtVWgtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2010&pages=5368-5378&author=M.+N.+L.+Nalamauthor=A.+Aliauthor=M.+D.+Altmanauthor=G.+S.+K.+K.+Reddyauthor=S.+Chellappanauthor=V.+Kairysauthor=A.+%C3%96zenauthor=H.+Caoauthor=M.+K.+Gilsonauthor=B.+Tidorauthor=T.+M.+Ranaauthor=C.+A.+Schiffer&title=Evaluating+the+substrate-envelope+hypothesis%3A+structural+analysis+of+novel+HIV-1+protease+inhibitors+designed+to+be+robust+against+drug+resistance&doi=10.1128%2FJVI.02531-09"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluating the substrate-envelope hypothesis: structural analysis of novel HIV-1 protease inhibitors designed to be robust against drug resistance</span></div><div class="casAuthors">Nalam, Madhavi N. L.; Ali, Akbar; Altman, Michael D.; Reddy, G. S. Kiran Kumar; Chellappan, Sripriya; Kairys, Visvaldas; Ozen, Aysegul; Cao, Hong; Gilson, Michael K.; Tidor, Bruce; Rana, Tariq M.; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5368-5378</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Drug resistance mutations in HIV-1 protease selectively alter inhibitor binding without significantly affecting substrate recognition and cleavage.  This alteration in mol. recognition led us to develop the substrate-envelope hypothesis which predicts that HIV-1 protease inhibitors that fit within the overlapping consensus vol. of the substrates are less likely to be susceptible to drug-resistant mutations, as a mutation impacting such inhibitors would simultaneously impact the processing of substrates.  To evaluate this hypothesis, over 130 HIV-1 protease inhibitors were designed and synthesized using three different approaches with and without substrate-envelope constraints.  A subset of 16 representative inhibitors with binding affinities to wild-type protease ranging from 58 nM to 0.8 pM was chosen for crystallog. anal.  The inhibitor-protease complexes revealed that tightly binding inhibitors (at the picomolar level of affinity) appear to "lock" into the protease active site by forming hydrogen bonds to particular active-site residues.  Both this hydrogen bonding pattern and subtle variations in protein-ligand van der Waals interactions distinguish nanomolar from picomolar inhibitors.  In general, inhibitors that fit within the substrate envelope, regardless of whether they are picomolar or nanomolar, have flatter profiles with respect to drug-resistant protease variants than inhibitors that protrude beyond the substrate envelope; this provides a strong rationale for incorporating substrate-envelope constraints into structure-based design strategies to develop new HIV-1 protease inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1FdWkZ1EQDLVg90H21EOLACvtfcHk0lgysgIpYL5boQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmtVWgtrc%253D&md5=09e55d24a9ff7382cefd2fccc5206ada</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1128%2FJVI.02531-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02531-09%26sid%3Dliteratum%253Aachs%26aulast%3DNalam%26aufirst%3DM.%2BN.%2BL.%26aulast%3DAli%26aufirst%3DA.%26aulast%3DAltman%26aufirst%3DM.%2BD.%26aulast%3DReddy%26aufirst%3DG.%2BS.%2BK.%2BK.%26aulast%3DChellappan%26aufirst%3DS.%26aulast%3DKairys%26aufirst%3DV.%26aulast%3D%25C3%2596zen%26aufirst%3DA.%26aulast%3DCao%26aufirst%3DH.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26aulast%3DTidor%26aufirst%3DB.%26aulast%3DRana%26aufirst%3DT.%2BM.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DEvaluating%2520the%2520substrate-envelope%2520hypothesis%253A%2520structural%2520analysis%2520of%2520novel%2520HIV-1%2520protease%2520inhibitors%2520designed%2520to%2520be%2520robust%2520against%2520drug%2520resistance%26jtitle%3DJ.%2520Virol.%26date%3D2010%26volume%3D84%26spage%3D5368%26epage%3D5378%26doi%3D10.1128%2FJVI.02531-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leidner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurt Yilmaz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Hydration structure and dynamics of inhibitor-bound HIV-1 protease</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2784</span>– <span class="NLM_lpage">2796</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.8b00097</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.8b00097" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlvVKmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=2784-2796&author=F.+Leidnerauthor=N.+Kurt+Yilmazauthor=J.+Paulsenauthor=Y.+A.+Mullerauthor=C.+A.+Schiffer&title=Hydration+structure+and+dynamics+of+inhibitor-bound+HIV-1+protease&doi=10.1021%2Facs.jctc.8b00097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Hydration Structure and Dynamics of Inhibitor-Bound HIV-1 Protease</span></div><div class="casAuthors">Leidner, Florian; Kurt Yilmaz, Nese; Paulsen, Janet; Muller, Yves A.; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2784-2796</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Water is an essential in many biol. processes, and the hydration structure plays a crit. role in facilitating protein folding, dynamics and ligand binding.  A variety of biophys. spectroscopic techniques have been used to probe the water solvating proteins, often complemented with mol. dynamics (MD) simulations to resolve the spatial and dynamic features of the hydration shell, but comparing relative water structure is challenging.  In this study 1 μs MD simulations were performed to identify and characterize hydration sites around HIV-1 protease bound to an inhibitor, darunavir (DRV).  The water d., hydration site occupancy, extent and anisotropy of fluctuations, coordinated water mols., and hydrogen bonds were characterized and compared to the properties of bulk water.  The water d. of the principal hydration shell was found to be higher than bulk, dependent on the topol. and physio-chem. identity of the biomol. surface.  The dynamics of water mols. occupying principal hydration sites was highly dependent on the no. of water-water interactions, and inversely correlated with hydrogen bonds to the protein-inhibitor complex.  While many waters were conserved following the symmetry of homodimeric HIV protease, the asymmetry induced by DRV resulted in asym. lower-occupancy hydration sites at the concave surface of the active site.  Key interactions between water mols. and the protease, that stabilize the protein in the inhibited form, were altered in a drug resistant variant of the protease indicating that modulation of solvent-solute interactions might play a key role in conveying drug resistance.  Our anal. provides insights into the interplay between an enzyme inhibitor complex and the hydration shell and has implications in elucidating water structure in a variety of biol. processes and applications including ligand binding, inhibitor design and resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-XstjSseAurVg90H21EOLACvtfcHk0lgysgIpYL5boQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlvVKmsbs%253D&md5=f8801ad50103c24efc2bec9dc442f9a1</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.8b00097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.8b00097%26sid%3Dliteratum%253Aachs%26aulast%3DLeidner%26aufirst%3DF.%26aulast%3DKurt%2BYilmaz%26aufirst%3DN.%26aulast%3DPaulsen%26aufirst%3DJ.%26aulast%3DMuller%26aufirst%3DY.%2BA.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DHydration%2520structure%2520and%2520dynamics%2520of%2520inhibitor-bound%2520HIV-1%2520protease%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2018%26volume%3D14%26spage%3D2784%26epage%3D2796%26doi%3D10.1021%2Facs.jctc.8b00097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockbaum, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosovrasti, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leidner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nachum, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nalivaika, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spielvogel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanstrom, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolon, D. N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurt
Yilmaz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Picomolar to micromolar: elucidating the role of distal mutations in HIV-1 protease in conferring drug resistance</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2441</span>– <span class="NLM_lpage">2452</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00370</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00370" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVOhtr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=2441-2452&author=M.+Henesauthor=G.+J.+Lockbaumauthor=K.+Kosovrastiauthor=F.+Leidnerauthor=G.+S.+Nachumauthor=E.+A.+Nalivaikaauthor=S.+K.+Leeauthor=E.+Spielvogelauthor=S.+Zhouauthor=R.+Swanstromauthor=D.+N.+A.+Bolonauthor=N.+Kurt%0AYilmazauthor=C.+A.+Schiffer&title=Picomolar+to+micromolar%3A+elucidating+the+role+of+distal+mutations+in+HIV-1+protease+in+conferring+drug+resistance&doi=10.1021%2Facschembio.9b00370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Picomolar to Micromolar: Elucidating the Role of Distal Mutations in HIV-1 Protease in Conferring Drug Resistance</span></div><div class="casAuthors">Henes, Mina; Lockbaum, Gordon J.; Kosovrasti, Klajdi; Leidner, Florian; Nachum, Gily S.; Nalivaika, Ellen A.; Lee, Sook-Kyung; Spielvogel, Ean; Zhou, Shuntai; Swanstrom, Ronald; Bolon, Daniel N. A.; Kurt Yilmaz, Nese; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2441-2452</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug resistance continues to be a growing global problem.  The efficacy of small mol. inhibitors is threatened by pools of genetic diversity in all systems, including antibacterials, antifungals, cancer therapeutics, and antivirals.  Resistant variants often include combinations of active site mutations and distal "secondary" mutations, which are thought to compensate for losses in enzymic activity.  HIV-1 protease is the ideal model system to investigate these combinations and underlying mol. mechanisms of resistance.  Darunavir (DRV) binds wild-type (WT) HIV-1 protease with a potency of <5 pM, but we have identified a protease variant that loses potency to DRV 150 000-fold, with 11 mutations in and outside the active site.  To elucidate the roles of these mutations in DRV resistance, we used a multidisciplinary approach, combining enzymic assays, crystallog., and mol. dynamics simulations.  Anal. of protease variants with 1, 2, 4, 8, 9, 10, and 11 mutations showed that the primary active site mutations caused ∼50-fold loss in potency (2 mutations), while distal mutations outside the active site further decreased DRV potency from 13 nM (8 mutations) to 0.76 μM (11 mutations).  Crystal structures and simulations revealed that distal mutations induce subtle changes that are dynamically propagated through the protease.  Our results reveal that changes remote from the active site directly and dramatically impact the potency of the inhibitor.  Moreover, we find interdependent effects of mutations in conferring high levels of resistance.  These mechanisms of resistance are likely applicable to many other quickly evolving drug targets, and the insights may have implications for the design of more robust inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2wI5Ah0DMVbVg90H21EOLACvtfcHk0lgysgIpYL5boQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVOhtr7F&md5=cc0a728eca73920e31dfbe0098c67424</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00370%26sid%3Dliteratum%253Aachs%26aulast%3DHenes%26aufirst%3DM.%26aulast%3DLockbaum%26aufirst%3DG.%2BJ.%26aulast%3DKosovrasti%26aufirst%3DK.%26aulast%3DLeidner%26aufirst%3DF.%26aulast%3DNachum%26aufirst%3DG.%2BS.%26aulast%3DNalivaika%26aufirst%3DE.%2BA.%26aulast%3DLee%26aufirst%3DS.%2BK.%26aulast%3DSpielvogel%26aufirst%3DE.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DSwanstrom%26aufirst%3DR.%26aulast%3DBolon%26aufirst%3DD.%2BN.%2BA.%26aulast%3DKurt%2BYilmaz%26aufirst%3DN.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DPicomolar%2520to%2520micromolar%253A%2520elucidating%2520the%2520role%2520of%2520distal%2520mutations%2520in%2520HIV-1%2520protease%2520in%2520conferring%2520drug%2520resistance%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D2441%26epage%3D2452%26doi%3D10.1021%2Facschembio.9b00370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honig, B.</span></span> <span> </span><span class="NLM_article-title">On the role of the crystal environment in determining protein side-chain conformations</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>320</i></span>,  <span class="NLM_fpage">597</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1016/S0022-2836(02)00470-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1016%2FS0022-2836%2802%2900470-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=12096912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD38XltVKmu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2002&pages=597-608&author=M.+P.+Jacobsonauthor=R.+A.+Friesnerauthor=Z.+Xiangauthor=B.+Honig&title=On+the+role+of+the+crystal+environment+in+determining+protein+side-chain+conformations&doi=10.1016%2FS0022-2836%2802%2900470-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">On the Role of the Crystal Environment in Determining Protein Side-chain Conformations</span></div><div class="casAuthors">Jacobson, Matthew P.; Friesner, Richard A.; Xiang, Zhexin; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">320</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">597-608</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">The role of crystal packing in detg. the obsd. conformations of amino acid side-chains in protein crystals is investigated by (1) anal. of a database of proteins that have been crystd. in different unit cells (space group or unit cell dimensions) and (2) theor. predictions of side-chain conformations with the crystal environment explicitly represented.  Both of these approaches indicate that the crystal environment plays an important role in detg. the conformations of polar side-chains on the surfaces of proteins.  Inclusion of the crystal environment permits a more sensitive measurement of the achievable accuracy of side-chain prediction programs, when validating against structures obtained by x-ray crystallog.  Our side-chain prediction program uses an all-atom force field and a Generalized Born model of solvation and is thus capable of modeling simple packing effects (i.e. van der Waals interactions), electrostatic effects, and desolvation, which are all important mechanisms by which the crystal environment impacts obsd. side-chain conformations.  Our results are also relevant to the understanding of changes in side-chain conformation that may result from ligand docking and protein-protein assocn., insofar as the results reveal how side-chain conformations change in response to their local environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgHEOVpIEiErVg90H21EOLACvtfcHk0lhhuTB0XOZ6og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltVKmu70%253D&md5=006de6bd2d0f233ab32d6798dc1a3fbc</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900470-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900470-9%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DXiang%26aufirst%3DZ.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DOn%2520the%2520role%2520of%2520the%2520crystal%2520environment%2520in%2520determining%2520protein%2520side-chain%2520conformations%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D320%26spage%3D597%26epage%3D608%26doi%3D10.1016%2FS0022-2836%2802%2900470-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, M. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Søndergaard, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rostkowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. H.</span></span> <span> </span><span class="NLM_article-title">PROPKA3: consistent treatment of internal and surface residues in empirical pK<sub>a</sub> predictions</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">525</span>– <span class="NLM_lpage">537</span>, <span class="refDoi"> DOI: 10.1021/ct100578z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct100578z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1aqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=525-537&author=M.+H.+M.+Olssonauthor=C.+R.+S%C3%B8ndergaardauthor=M.+Rostkowskiauthor=J.+H.+Jensen&title=PROPKA3%3A+consistent+treatment+of+internal+and+surface+residues+in+empirical+pKa+predictions&doi=10.1021%2Fct100578z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">PROPKA3: Consistent Treatment of Internal and Surface Residues in Empirical pKa Predictions</span></div><div class="casAuthors">Olsson, Mats H. M.; Sondergaard, Chresten R.; Rostkowski, Michal; Jensen, Jan H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">525-537</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors have revised the rules and parameters for one of the most commonly used empirical pKa predictors, PROPKA, based on better phys. description of the desolvation and dielec. response for the protein.  The authors have introduced a new and consistent approach to interpolate the description between the previously distinct classifications into internal and surface residues, which otherwise is found to give rise to an erratic and discontinuous behavior.  Since the goal of this study is to lay out the framework and validate the concept, it focuses on Asp and Glu residues where the protein pKa values and structures are assumed to be more reliable.  The new and improved implementation is evaluated and discussed; it is found to agree better with expt. than the previous implementation (in parentheses): rmsd = 0.79 (0.91) for Asp and Glu, 0.75 (0.97) for Tyr, 0.65 (0.72) for Lys, and 1.00 (1.37) for His residues.  The most significant advance, however, is in reducing the no. of outliers and removing unreasonable sensitivity to small structural changes that arise from classifying residues as either internal or surface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm6urTZNdjY7Vg90H21EOLACvtfcHk0lhhuTB0XOZ6og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1aqsA%253D%253D&md5=9b1666b1c56e1129789e62948eb4d001</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fct100578z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct100578z%26sid%3Dliteratum%253Aachs%26aulast%3DOlsson%26aufirst%3DM.%2BH.%2BM.%26aulast%3DS%25C3%25B8ndergaard%26aufirst%3DC.%2BR.%26aulast%3DRostkowski%26aufirst%3DM.%26aulast%3DJensen%26aufirst%3DJ.%2BH.%26atitle%3DPROPKA3%253A%2520consistent%2520treatment%2520of%2520internal%2520and%2520surface%2520residues%2520in%2520empirical%2520pKa%2520predictions%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D525%26epage%3D537%26doi%3D10.1021%2Fct100578z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felts, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maple, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philipp, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallicchio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, R. M.</span></span> <span> </span><span class="NLM_article-title">Integrated Modeling Program, Applied Chemical Theory (IMPACT)</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1752</span>– <span class="NLM_lpage">1780</span>, <span class="refDoi"> DOI: 10.1002/jcc.20292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=10.1002%2Fjcc.20292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=16211539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1752-1780&author=J.+L.+Banksauthor=H.+S.+Beardauthor=Y.+Caoauthor=A.+E.+Choauthor=W.+Dammauthor=R.+Faridauthor=A.+K.+Feltsauthor=T.+A.+Halgrenauthor=D.+T.+Mainzauthor=J.+R.+Mapleauthor=R.+Murphyauthor=D.+M.+Philippauthor=M.+P.+Repaskyauthor=L.+Y.+Zhangauthor=B.+J.+Berneauthor=R.+A.+Friesnerauthor=E.+Gallicchioauthor=R.+M.+Levy&title=Integrated+Modeling+Program%2C+Applied+Chemical+Theory+%28IMPACT%29&doi=10.1002%2Fjcc.20292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated modeling program, applied chemical theory (IMPACT)</span></div><div class="casAuthors">Banks, Jay L.; Beard, Hege S.; Cao, Yixiang; Cho, Art E.; Damm, Wolfgang; Farid, Ramy; Felts, Anthony K.; Halgren, Thomas A.; Mainz, Daniel T.; Maple, Jon R.; Murphy, Robert; Philipp, Dean M.; Repasky, Matthew P.; Zhang, Linda Y.; Berne, Bruce J.; Friesner, Richard A.; Gallicchio, Emilio; Levy, Ronald M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1752-1780</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  We provide an overview of the IMPACT mol. mechanics program with an emphasis on recent developments and a description of its current functionality.  With respect to core mol. mechanics technologies we include a status report for the fixed charge and polarizable force fields that can be used with the program and illustrate how the force fields, when used together with new atom typing and parameter assignment modules, have greatly expanded the coverage of org. compds. and medicinally relevant ligands.  As we discuss in this review, explicit solvent simulations have been used to guide our design of implicit solvent models based on the generalized Born framework and a novel nonpolar estimator that have recently been incorporated into the program.  With IMPACT it is possible to use several different advanced conformational sampling algorithms based on combining features of mol. dynamics and Monte Carlo simulations.  The program includes two specialized mol. mechanics modules: Glide, a high-throughput docking program, and QSite, a mixed quantum mechanics/mol. mechanics module.  These modules employ the IMPACT infrastructure as a starting point for the construction of the protein model and assignment of mol. mechanics parameters, but have then been developed to meet specialized objectives with respect to sampling and the energy function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZX2FzcGWGM7Vg90H21EOLACvtfcHk0lhhuTB0XOZ6og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbI&md5=0094f15727c7fb77a0fae16ccf8292f0</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20292%26sid%3Dliteratum%253Aachs%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DCho%26aufirst%3DA.%2BE.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DFelts%26aufirst%3DA.%2BK.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DMaple%26aufirst%3DJ.%2BR.%26aulast%3DMurphy%26aufirst%3DR.%26aulast%3DPhilipp%26aufirst%3DD.%2BM.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DZhang%26aufirst%3DL.%2BY.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DGallicchio%26aufirst%3DE.%26aulast%3DLevy%26aufirst%3DR.%2BM.%26atitle%3DIntegrated%2520Modeling%2520Program%252C%2520Applied%2520Chemical%2520Theory%2520%2528IMPACT%2529%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1752%26epage%3D1780%26doi%3D10.1002%2Fjcc.20292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJB','PDB','6PJB'); return false;">PDB: 6PJB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DGX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DGX','PDB','6DGX'); return false;">PDB: 6DGX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJD','PDB','6PJD'); return false;">PDB: 6PJD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJM','PDB','6PJM'); return false;">PDB: 6PJM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJC','PDB','6PJC'); return false;">PDB: 6PJC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJE','PDB','6PJE'); return false;">PDB: 6PJE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PJN','PDB','6PJN'); return false;">PDB: 6PJN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EKY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EKY','PDB','3EKY'); return false;">PDB: 3EKY</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i38"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00529">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_28874"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00529?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00529</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Synthesis details and analytical data of Phe-Phe isostere <b>7</b>, amino acid derivatives <b>11a</b>–<b>h</b>, and inhibitors <b>4</b>–<b>5</b>; binding modes of parent compounds, comparison of per residue vdW contacts, RMSD analysis, distance distribution, intraprotease vdW interactions, protein RMSF analysis, and protease–inhibitor hydrogen bond frequencies from MD simulations; X-ray data collection and refinement statistics; and intermolecular hydrogen bonds between the protease and inhibitors (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_001.pdf">jm0c00529_si_001.pdf (28.12 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00529/suppl_file/jm0c00529_si_002.csv">jm0c00529_si_002.csv (2.46 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The PDB accession codes for X-ray cocrystal structures of wild-type HIV-1 protease with LPV, <b>4</b>, <b>12a</b>, <b>12b</b>, <b>12c</b>, <b>12d</b>, <b>12e</b>, <b>12f</b>, <b>12g</b>, <b>12h</b>, <b>15a</b>, <b>15b</b>, and <b>15c</b> are 6DGX, 6PJB, 6PJC, 6PJD, 6PJE, 6PJF, 6PJG, 6PJH, 6PJI, 6PJK, 6PJL, 6PJM, 6PJN, and 6PJO, respectively. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00529&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-15%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00529%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00529" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                10MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679961d4582924b2","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
