
<html lang="en"     class="pb-page"  data-request-id="87f4259a-2160-4448-a793-acdd99a526f8"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.5b00929;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2015.58.issue-24;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Identification of Non-nucleoside Human Ribonucleotide Reductase Modulators" /></meta><meta name="dc.Creator" content="Md. Faiz  Ahmad" /></meta><meta name="dc.Creator" content="Sarah E.  Huff" /></meta><meta name="dc.Creator" content="John  Pink" /></meta><meta name="dc.Creator" content="Intekhab  Alam" /></meta><meta name="dc.Creator" content="Andrew  Zhang" /></meta><meta name="dc.Creator" content="Kay  Perry" /></meta><meta name="dc.Creator" content="Michael E.  Harris" /></meta><meta name="dc.Creator" content="Tessianna  Misko" /></meta><meta name="dc.Creator" content="Suheel K.  Porwal" /></meta><meta name="dc.Creator" content="Nancy L.  Oleinick" /></meta><meta name="dc.Creator" content="Masaru  Miyagi" /></meta><meta name="dc.Creator" content="Rajesh  Viswanathan" /></meta><meta name="dc.Creator" content="Chris Godfrey  Dealwis" /></meta><meta name="dc.Description" content="Ribonucleotide reductase (RR) catalyzes the rate-limiting step of dNTP synthesis and is an established cancer target. Drugs targeting RR are mainly nucleoside in nature. In this study, we sought to..." /></meta><meta name="Description" content="Ribonucleotide reductase (RR) catalyzes the rate-limiting step of dNTP synthesis and is an established cancer target. Drugs targeting RR are mainly nucleoside in nature. In this study, we sought to..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 9, 2015" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b00929" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2015 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b00929" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b00929" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b00929" /></link>
        
    
    

<title>Identification of Non-nucleoside Human Ribonucleotide Reductase Modulators | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b00929" /></meta><meta property="og:title" content="Identification of Non-nucleoside Human Ribonucleotide Reductase Modulators" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0007.jpeg" /></meta><meta property="og:description" content="Ribonucleotide reductase (RR) catalyzes the rate-limiting step of dNTP synthesis and is an established cancer target. Drugs targeting RR are mainly nucleoside in nature. In this study, we sought to identify non-nucleoside small-molecule inhibitors of RR. Using virtual screening, binding affinity, inhibition, and cell toxicity, we have discovered a class of small molecules that alter the equilibrium of inactive hexamers of RR, leading to its inhibition. Several unique chemical categories, including a phthalimide derivative, show micromolar IC50s and KDs while demonstrating cytotoxicity. A crystal structure of an active phthalimide binding at the targeted interface supports the noncompetitive mode of inhibition determined by kinetic studies. Furthermore, the phthalimide shifts the equilibrium from dimer to hexamer. Together, these data identify several novel non-nucleoside inhibitors of human RR which act by stabilizing the inactive form of the enzyme." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b00929"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b00929">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b00929&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b00929&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b00929&amp;href=/doi/10.1021/acs.jmedchem.5b00929" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2015</span><span class="cit-fg-volume">, 58</span><span class="cit-fg-issue">, 24</span><span class="cit-fg-pageRange">, 9498-9509</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/58/24" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01346" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b00792" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Identification of Non-nucleoside Human Ribonucleotide Reductase Modulators</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Md.+Faiz++Ahmad">Md. Faiz Ahmad</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sarah+E.++Huff">Sarah E. Huff</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John++Pink">John Pink</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Intekhab++Alam">Intekhab Alam</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrew++Zhang">Andrew Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kay++Perry">Kay Perry</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael+E.++Harris">Michael E. Harris</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tessianna++Misko">Tessianna Misko</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Suheel+K.++Porwal">Suheel K. Porwal</a></span><span class="author-xref-symbol author-aff-symbol"><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Nancy+L.++Oleinick">Nancy L. Oleinick</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Masaru++Miyagi">Masaru Miyagi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>¶</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rajesh++Viswanathan">Rajesh Viswanathan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chris+Godfrey++Dealwis">Chris Godfrey Dealwis</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>▽</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Pharmacology, School of Medicine, Case Western Reserve University, Cleveland, Ohio 44106, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Chemistry, Case Western Reserve University, Cleveland, Ohio 44106, United States</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Case Comprehensive Cancer Center, Case Western Reserve University, Cleveland, Ohio 44106, United States</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Northeastern-CAT at the Advanced Photon Source, Argonne National Laboratory, Argonne, Illinois 60439, United States</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Department of Biochemistry, School of Medicine, Case Western Reserve University, Cleveland, Ohio 44106, United States</span></div><div class="aff-info" id="aff6"><span class="aff-symbol">#</span> <span class="aff-text">Department of Radiation Oncology, School of Medicine, Case Western Reserve University, Cleveland, Ohio 44106, United States</span></div><div class="aff-info" id="aff7"><span class="aff-symbol">▽</span> <span class="aff-text">Center for Proteomics and the Department of Chemistry, Case Western Reserve University, Cleveland, Ohio 44106, United States</span></div><div class="aff-info" id="aff8"><span class="aff-symbol">○</span> <span class="aff-text">Department of Chemistry, Dehradun Institute of Technology, University of Deharadun, Dehradun 248197, India</span></div><div class="aff-info" id="aff9"><span class="aff-symbol">¶</span> <span class="aff-text">Center for Proteomics and Bioinformatics, Case Western Reserve University, Cleveland, Ohio 44106, United States</span></div><div class="corresp-info"><strong>*</strong>Phone: 1-216-368-1652. Fax: 1-216-368-1300. E-mail: <a href="/cdn-cgi/l/email-protection#9bf8f3e9f2e8b5fffefaf7ecf2e8dbf8fae8feb5feffee"><span class="__cf_email__" data-cfemail="e4878c968d97ca80818588938d97a487859781ca818091">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b00929&amp;href=/doi/10.1021%2Facs.jmedchem.5b00929" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2015</span></span><span class="cit-volume">, 58</span><span class="cit-issue">, 24</span><span class="cit-pageRange">, 9498–9509</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 21, 2015</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>15 April 2015</li><li><span class="item_label"><b>Published</b> online</span>9 December 2015</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 December 2015</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b00929" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b00929</a></div><div class="article_header-article-copyright"><strong>Copyright © 2015 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9498%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMd.%2BFaiz%2BAhmad%252C%2BSarah%2BE.%2BHuff%252C%2BJohn%2BPink%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D58%26issueNum%3D24%26contentID%3Dacs.jmedchem.5b00929%26title%3DIdentification%2Bof%2BNon-nucleoside%2BHuman%2BRibonucleotide%2BReductase%2BModulators%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9509%26publicationDate%3DDecember%2B2015">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b00929"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1368</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">8</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b00929" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Identification of Non-nucleoside Human Ribonucleotide Reductase Modulators&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Md.&quot;,&quot;last_name&quot;:&quot;Faiz Ahmad&quot;},{&quot;first_name&quot;:&quot;Sarah&quot;,&quot;last_name&quot;:&quot;E. Huff&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;Pink&quot;},{&quot;first_name&quot;:&quot;Intekhab&quot;,&quot;last_name&quot;:&quot;Alam&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Kay&quot;,&quot;last_name&quot;:&quot;Perry&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;E. Harris&quot;},{&quot;first_name&quot;:&quot;Tessianna&quot;,&quot;last_name&quot;:&quot;Misko&quot;},{&quot;first_name&quot;:&quot;Suheel&quot;,&quot;last_name&quot;:&quot;K. Porwal&quot;},{&quot;first_name&quot;:&quot;Nancy&quot;,&quot;last_name&quot;:&quot;L. Oleinick&quot;},{&quot;first_name&quot;:&quot;Masaru&quot;,&quot;last_name&quot;:&quot;Miyagi&quot;},{&quot;first_name&quot;:&quot;Rajesh&quot;,&quot;last_name&quot;:&quot;Viswanathan&quot;},{&quot;first_name&quot;:&quot;Chris&quot;,&quot;last_name&quot;:&quot;Godfrey Dealwis&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2015&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;09&quot;,&quot;issue&quot;:&quot;24&quot;,&quot;volume&quot;:&quot;58&quot;,&quot;pages&quot;:&quot;9498-9509&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b00929&quot;},&quot;abstract&quot;:&quot;Ribonucleotide reductase (RR) catalyzes the rate-limiting step of dNTP synthesis and is an established cancer target. Drugs targeting RR are mainly nucleoside in nature. In this study, we sought to identify non-nucleoside small-molecule inhibitors of RR. Using virtual screening, binding affinity, inhibition, and cell toxicity, we have discovered a class of small molecules that alter the equilibrium of inactive hexamers of RR, leading to its inhibition. Several unique chemical categories, including a phthalimide derivative, show micromolar IC50s and KDs while demonstrating cytotoxicity. A crystal structure of an active phthalimide binding at the targeted interface supports the noncompetitive mode of inhibition determined by kinetic studies. Furthermore, the phthalimide shifts the equilibrium from dimer to hexamer. Together, these data identify several novel non-nucleoside inhibitors of human RR which act by stabilizing the inactive form of the enzyme.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b00929&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b00929" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b00929&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b00929" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b00929&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b00929" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b00929&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b00929&amp;href=/doi/10.1021/acs.jmedchem.5b00929" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b00929" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b00929" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b00929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b00929%26sid%3Dliteratum%253Aachs%26pmid%3D26488902%26genre%3Darticle%26aulast%3DAhmad%26date%3D2015%26atitle%3DIdentification%2Bof%2BNon-nucleoside%2BHuman%2BRibonucleotide%2BReductase%2BModulators%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D58%26issue%3D24%26spage%3D9498%26epage%3D9509%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290817" title="Interfaces">Interfaces</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291912" title="Fluorescence">Fluorescence</a>,</li><li><a href="/action/doSearch?ConceptID=292425" title="Oligomers">Oligomers</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/58/24" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/jmcmar.2015.58.issue-24/20151224/jmcmar.2015.58.issue-24.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/medium/jm-2015-00929x_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00929&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Ribonucleotide reductase (RR) catalyzes the rate-limiting step of dNTP synthesis and is an established cancer target. Drugs targeting RR are mainly nucleoside in nature. In this study, we sought to identify non-nucleoside small-molecule inhibitors of RR. Using virtual screening, binding affinity, inhibition, and cell toxicity, we have discovered a class of small molecules that alter the equilibrium of inactive hexamers of RR, leading to its inhibition. Several unique chemical categories, including a phthalimide derivative, show micromolar IC<sub>50</sub>s and <i>K</i><sub>D</sub>s while demonstrating cytotoxicity. A crystal structure of an active phthalimide binding at the targeted interface supports the noncompetitive mode of inhibition determined by kinetic studies. Furthermore, the phthalimide shifts the equilibrium from dimer to hexamer. Together, these data identify several novel non-nucleoside inhibitors of human RR which act by stabilizing the inactive form of the enzyme.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04233" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04233" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Ribonucleotide reductase (RR) is crucial for rapidly proliferating cells, and inhibition of this enzyme has proven to be an effective strategy for anticancer therapy.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> Competitive nucleoside analogue inhibitors such as gemcitabine are some of the few drugs used to treat devastating cancers such as pancreatic cancer.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Other FDA approved drugs including gemcitabine, fludarabine, clofarabine, and cladarabine are nucleoside analogues that target allosteric sites of RR and irreversibly inhibit DNA replication by incorporation and chain termination.<a onclick="showRef(event, 'ref5 ref6 ref7 ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8 ref9 ref10">(5-10)</a> So far, all of the clinically used drugs that target the large subunit (hRRM1) of hRR are nucleoside analogues.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Nucleoside analogues lack on-target specificity for hRR.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> For example, gemcitabine is also known to cross-react with numerous other enzymes in addition to hRR including DNA polymerase,<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12, 13)</a> deoxycytidine deaminase (dCMP deaminase), thymidylate synthase, CTP-synthase,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and topoisomerase 1.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> We wish to discover a novel class of non-nucleoside inhibitors of hRR.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div><div class="NLM_p">The development of RR inhibitors has necessarily advanced along with our understanding of its structure and its enzymology, in particular its allosteric regulation. Ribonucleotide reductase is a multiprotein enzyme consisting of a large subunit called hRRM1 (α) containing the catalytic site and allosteric sites and a small subunit called hRRM2 (β) that houses the free radical required for initiating radical-based chemistry.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> The hRRM1 subunit catalyzes the conversion of four ribonucleoside diphosphates (UDP, CDP, GDP, and ADP) to their respective deoxy forms. During the S-phase of the cell cycle, these reduction reactions are allosterically controlled by binding of nucleotide triphosphates to two different sites on RR.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> The S-site is located at the dimer interface of hRRM1 and is involved in allosterically regulating substrate binding specificity (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A).<a onclick="showRef(event, 'ref18 ref19 ref20 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20 ref21 ref22 ref23">(18-23)</a> ATP is an allosteric activator, while dATP is an allosteric inhibitor, where both bind to the A-site (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A).<a onclick="showRef(event, 'ref18 ref24'); return false;" href="javascript:void(0);" class="ref ref18 ref24">(18, 24)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/medium/jm-2015-00929x_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. hRRM1 drug binding sites and fluorescence quenching identifies RR inhibitors. (A) Structure of the hRRM1 dimer with drug-target sites mapped. The M-site is the binding site for the new class of modulators that is the subject of this study. The A-site controls activity. The S-site controls specificity. The C-site is the catalytic site. The P-site binds the smaller R2 subunit derived peptide. (B and C) Tryptophan fluorescence quenching of hRRM1 in the presence of compounds <b>4</b> and <b>10</b>, respectively. (D and E) No tryptophan fluorescence quenching of hRRM1 by compounds <b>100207</b> and <b>184612</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00929&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Recent studies with RR have revealed the importance of oligomerization and its regulation as well as its inhibition.<a onclick="showRef(event, 'ref25 ref26 ref27 ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref25 ref26 ref27 ref28 ref29 ref30 ref31">(25-31)</a> Although the multimerization of RR is still a subject of investigation, the prevailing model is that RR minimally functions as an α<sub>2</sub>β<sub>2</sub> complex. At physiological concentrations of ATP (3 mM), hRRM1 exists predominantly as an active hexamer with a small population of dimer present.<a onclick="showRef(event, 'ref25 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref25 ref27 ref28 ref29">(25, 27-29)</a> When dATP is bound, the large subunit has also been shown to exist as a dimer and hexamer,<a onclick="showRef(event, 'ref25 ref27 ref29'); return false;" href="javascript:void(0);" class="ref ref25 ref27 ref29">(25, 27, 29)</a> although these forms of the enzyme are inactive, while baculovirus expressed mouse RR1 was observed to exist as a tetramer.<a onclick="showRef(event, 'ref25 ref32'); return false;" href="javascript:void(0);" class="ref ref25 ref32">(25, 32)</a></div><div class="NLM_p">Stabilization of the inactive dATP bound form might be an effective strategy for RR inhibition. In a recent report, the dATP hexamer was proposed to be stabilized by the protein IRBIT in cells.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Recently, hexamer formation has been shown to be important for inhibitors such as gemcitabine and clofarabine binding to RR.<a onclick="showRef(event, 'ref28 ref30'); return false;" href="javascript:void(0);" class="ref ref28 ref30">(28, 30)</a> For example, gemcitabine was shown to inactivate hRRM1 by inducing α<sub>6</sub>β<sub>6</sub> oligomers, while clofarabine was shown to bind hRRM1 hexamers with nanomolar affinity.<a onclick="showRef(event, 'ref28 ref30'); return false;" href="javascript:void(0);" class="ref ref28 ref30">(28, 30)</a> Indeed, the importance of hexamer formation was highlighted in a recent paper characterizing the non-nucleoside drug 5-NITP which is a moderate RR inhibitor.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> While this drug was shown to induce hRRM1 dimers, its low inhibitory potency is proposed to be due to its inability to form inactive hexamers.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div><div class="NLM_p">Because of the importance of hexamerization in drug-mediated inactivation, targeting the hexamer interface to develop specific small molecules that bind preferentially to the dATP-induced hexamer can potentially shift the equilibrium toward the inactive conformation. Similar strategies have been used against porphobilinogen synthase (PBGS), phenylalanine hydroxylase, and HIV integrase to discover small molecules that bind at the oligomeric interfaces.<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35, 36)</a> These proteins conform to the morpheein model of allostery,<a onclick="showRef(event, 'ref35 ref37'); return false;" href="javascript:void(0);" class="ref ref35 ref37">(35, 37)</a> where the change of oligomeric state is a prerequisite for allosteric regulation.</div><div class="NLM_p">Here, we describe a method that combines virtual screening with hit validation by biophysical methods, RR activity assays, and growth inhibition using cell culture. In this method, the hexamer interface was chosen as the docking site for virtual screening (M-site on <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A and <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A), and the top 51 of the top 76 hits were subjected to growth inhibition assays to assess cellular uptake and anticancer properties. The top 76 hits obtained from the virtual screen were also subjected to fluorescence quenching assays to verify binding to hRRM1. Collectively, these techniques yielded a broad group of compounds possessing redundant functional group architecture. From this subset, we identified 10 structurally unique chemical scaffolds and subjected them to <i>in vitro</i> enzymatic inhibition assays. We report compounds that exhibit micromolar affinity against RR where PB piperazine (compound <b>1</b>) demonstrated cytotoxicity against HCT-116 cell line with an IC<sub>50</sub> of 2.0 μM. Moreover, we were able to derive the crystal structure for OxoIsoIndoLys (compound <b>4</b>) which binds at the proposed hexamer interface at the N-terminus of hRRM1, suggesting that we have discovered a new RR modulator that binds at a previously unidentified site (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A). As these hits are non-nucleoside in nature and are unique chemical entities, they enable us to use a chemical biology platform guided by structure to develop new highly potent anticancer agents.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/medium/jm-2015-00929x_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compound <b>4</b> interactions with hRRM1. (A) Model of the hRRM1 hexamer based on the <i>S. cerevisiae</i> RR1 hexamer structure. Ribbon diagram of the hRRM1 hexamer packing arrangement. hRRM1 monomers are green and magenta. All of the four helix ATP-binding cones are red. The 16 N-terminal residues at the hexamer interface are in cyan. Effectors (TTP) bound at the S-site are drawn in brick red spheres, and compound <b>4</b> at the hexamer interface is drawn in blue spheres. (B) The |<i>F</i><sub>0</sub>| – |<i>F</i><sub>c</sub>| electron density for the phthalimide compound (blue) in complex with hRRM1 orthorhombic crystals. Density contoured at 3σ defines the phthalimide binding to hRRM1. (C) 2|<i>F</i><sub>0</sub>| – |<i>F</i><sub>c</sub>| electron density (blue) of the phthalimide compound contoured at 1σ after refinement in the phthalimide-hRRM1 orthorhombic complex. (D) Illustration of the A-site and M-site binding by dATP and compound <b>4</b>, respectively, Compound <b>4</b> is shown in magenta, and dATP is in yellow. (E) Lig plot analysis of compound <b>4</b> interactions with hRRM1. The phthalimide compound is shown in purple, carbon atoms in phthalimide are shown in black, oxygen atoms are in red, and nitrogen atoms are in blue. An amino acid residue from hRRM1 interacting with the phthalimide is shown in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00929&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44466" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44466" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> <i>In Silico</i> Screening Targeting the RR Hexamer Interface</h3><div class="NLM_p">The Cincinnati library consisting of 350,000 compounds was screened <i>in silico</i> using the Schrödinger software suite. The homologous model of the hRRM1 hexamer was constructed using the <i>S. cerevisiae</i> dATP-induced hexamer structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PAW">3PAW</a>). The docking site was defined as the inactive dATP hexamer interface that consists of the N-terminal 16 residues from adjacent dimers (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A). When determining hits, we carefully examined the docking poses (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Figure S4</a>) where common interactions were a good indication of a consensus binding site. For example, residues Ile 44, Gln 45, Met 1, His 2, Val 51, and Val 43 interact with all 10 compounds in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, which is a good indication that they are binding at the same site. The top 90 hits were subjected to a PAINS filter (<a href="http://cbligand.org/PAINS/" class="extLink">http://cbligand.org/PAINS/</a>), which identified 14 hits as violators that were then removed leaving 76 hits.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> A summary of the results from Schrödinger are provided in <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Table S1</a>. All top ranking hits are referred to by their corresponding Cincinnati library GRI numbers as described in <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Supporting Information</a>. Compounds discussed in the main text are identified in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Identification of 10 Novel hRRM1 Inhibitors Using <i>in Silico</i> Docking, Fluorescence Quenching, RR Inhibition Assays, and Growth Inhibition</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/medium/jm-2015-00929x_0006.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">Drug effect values are averaged from MDA-MB-231 and HCT116 cell lines.</p></div><div class="footnote" id="t1fn2"><p class="first last"><i>ND</i>: <i>K</i><sub>D</sub>’s not determined by fluorescence quenching due to the lack of sufficient quantities of the compound</p></div></div><div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Analysis of compound binding using intrinsic protein fluorescence</h3><div class="NLM_p last">The top 76 hits from the <i>in silico</i> screen were subjected to fluorescence quenching assays for binding to hRRM1 (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Table S1</a>). Ligands that exhibited 25% or more quenching were considered to have sufficient affinity for hRRM1. On the basis of this criterion, 51% of the ligands tested were considered as binding to hRRM1. As shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B–C and <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Table S1</a>, compound <b>4</b> shows 35% quenching, and compound <b>10</b> shows 40% quenching. The compounds that did not show any quenching (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>D and E) were not selected for further screening or studies. To test whether the observed quenching of tryptophan fluorescence of hRRM1 was due to binding controls, the ability of the compounds to quench the fluorescence of the free tryptophan analogue NATA was measured, and the binding of selected compounds was confirmed by thermal denaturation of hRRM1 in the presence of compounds (data not shown).<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Furthermore, nonspecific and artificial inhibition was eliminated by testing two unrelated compounds to the 76 hits using fluorescence quenching (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Figure S1</a>) which demonstrated no binding. Four of the 10 compounds reported in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> were subject to <i>K</i><sub>D</sub> determination using fluorescence quenching. The <i>K</i><sub>D</sub>’s ranged from 10–55 μM (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Figure S5</a>).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Chemical Classification and Inhibitory Potency of Non-nucleoside Ligands Inhibiting RR Reveals Broader Pharmacophore Diversity</h3><div class="NLM_p last">On the basis of their structure, the collection of hits can be broadly classified into 10 groups (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Table S2</a>) with distinct chemical scaffolds (see <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Table S7 and Figure S2</a> for HRMS and <sup>1</sup>H NMR data). A group of fluorenyl piperazines represented by PB-piperazine (compound <b>1</b>), which consists of a C<sub>2</sub>-symmetric <i>p</i>-cresol core tethered to two glycerol units that are derivatized as their corresponding fluorenylpiperazine units on either ends. The next class represented by TetraHThioDIM (compound <b>2</b>) featured varying substituent groups on a tetrahydrobenzothiophene heterocycle. The next class contained a diamino butanamide moiety connected to aliphatic chains of varying lengths. A representative member of this class is S-DiTDB (compound <b>3</b>), generally referred to as the DiaminoTDBamide class of inhibitors. The class represented by compound <b>4</b> (OxoIsoIndoLys) is uniquely defined through the presence of the phthalimide ring system at one end of the molecule and a hexafluoromonoketide group at the other terminus, tethered with an <span class="smallcaps smallerCapital">l</span>-lysine-α,α-dimethylglycine dipeptide scaffold. ButHyNitNap (compound <b>5</b>), generally referred to as the Hydroxy Naphthamide class, is representative of a class consisting of an <i>o</i>-naphthamide functionality featuring a polar <i>p</i>-nitrophenol substituent and tethered to a hydrophobic <i>tert</i>-pentyl phenoxy group through an amide linkage. An additional class of molecules is styrenyl sulfonamides, represented by DPSP (compound <b>6</b>, with a general name, DPS-benzoate) possessing a trans-configured sulfonamide moiety. Nmet GAVTVH (compound <b>7</b>) is peptidyl-like containing an <i>N</i>-methyl-acetylated guanidine-containing arginine at one of its termini and a series of <span class="smallcaps smallerCapital">l</span>-amino acids terminating with <span class="smallcaps smallerCapital">l</span>-histidine at its carboxy terminus. A class is defined by a (3,5-bis(benzyloxy)-phenyl-ethyl) amino pentanol group, represented by BoPEAP (compound <b>8</b>). A linear 3-hydroxy-keto amide represented the next class we identified, which also featured a peptidyl core and an acetoxy-ethoxy-oxopentamide scaffold, represented by AEOHydBen (compound <b>9</b>) (generally referred to as the AcetoxyHydBenzoate class). MePAMLL (compound <b>10</b>) was defined by <span class="smallcaps smallerCapital">l</span>-lysinyl <span class="smallcaps smallerCapital">l</span>-lysinate groups containing an aliphatic chain of varying length (C<sub>15</sub>H<sub>31</sub> for instance). In addition to the chemical classification names, the pharmacologic properties (including AlogP, polar surface area, and the corresponding binding efficiency) for each representative member are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Table S2</a>.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> RR Inhibition</h3><div class="NLM_p last">A representative of each class was tested for enzyme inhibition and revealed approximate IC<sub>50</sub> values in the micromolar range (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Table S3</a>). As a positive control, the well-known RR drug hydroxyurea’s IC<sub>50</sub> was determined using our two-point IC<sub>50</sub> derivation method, which was in extremely good agreement with published results<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Surprisingly, among the top four of the most potent inhibitors, compounds <b>5</b> and <b>8</b> and <b>1</b> and <b>9</b> (21.8, 23.6, 23.9, and 27.2 μM RR IC<sub>50</sub>, respectively), there is little structural similarity. They all possess multiple flat aromatic rings with <b>5</b> and <b>8</b> having distinct structural features that impart high hydrophobicity at one terminus and a polar functionality at the other. Compounds <b>1</b> and <b>9</b> like-wise possess distinct structural features yet have similar potency against the catalytic RR. The lack of apparent correlations between structural parameters such as polar surface area, solubility property (AlogP), and RR inhibition suggests that identifying pharmacophore <i>a priori</i> would be difficult.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Growth Inhibition of Established Cancer Cell Lines Including MDA-MB-231, HCT116, A549, and Panc1</h3><div class="NLM_p">Approximately 51 compounds were screened for their ability to inhibit growth and/or induce cell death of well-characterized cell lines representing common cancer types that are generally difficult to treat (triple negative breast cancer and colon cancer). Pancreatic cancer was also considered informative because gemcitabine is a core component of the current standard of care chemotherapy for this disease. From these screens, compounds were identified that showed significant (>50%) growth inhibitory activity against both MDA-MB-231 (triple negative breast cancer) and HCT-116 (DNA mismatch repair deficient colon cancer) cell lines at 1 μM, 10 μM, or 50 μM after a three day continuous exposure (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Table S4</a>, approximately 64% of the compounds did not show any significant growth inhibitory activity at 50 μM, and an additional 36% did not show significant activity at 10 μM. Although none of the compounds showed significant activity at 1 μM, 8.5% of the compounds did show significant activity at 10 μM. <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a> shows a more detailed growth inhibition study in the initially screened cell lines (MDA-MB-231 and HCT116) as well as two additional cell lines (A549 and Panc1) for compound <b>1</b>. The median effect doses (Dm) calculated using Calcusyn 2.0 are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Table S5</a>.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/medium/jm-2015-00929x_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Growth inhibition of MDA-MB-231 and HCT-116 cancer cells in a moderate throughput screen of candidate RR inhibitors. Cells were treated with 3 doses of candidate drugs (1 μM, 10 μM, and 50 μM) for 3 days in a standard growth inhibition assay in 96 well plates. Growth inhibition in duplicate wells of each drug/dose/cell line was assessed by measuring relative DNA content per well compared with that of untreated cells. Drug effect (1-relative growth) is plotted for the 10 μM and 1 μM dose groups (50 μM groups were not included for clarity).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00929&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/medium/jm-2015-00929x_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Growth inhibition of cancer cell lines by compound <b>1</b>. Cells were seeded into 96 well plates (2000 cells/well) and the following day incubated with the indicated concentrations of gemcitabine alone or in combination with compound <b>1</b> for 24 h. At this time, drugs were removed and replaced with control media in the gemcitabine alone groups, or compound <b>1</b> at the indicated concentrations in the compound <b>1</b> alone or combination groups for an additional 48 h. Relative growth was assessed by measuring DNA content in each well. Each drug concentration was assayed utilizing five replicates for each cell line. Results are representative of at least 2 experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00929&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Compound <b>1</b> demonstrated the greatest efficacy in this set of cell lines. Interestingly, compound <b>1</b> showed a very dramatic dose–response in three of the cell lines, with little growth inhibitory activity below 5 μM and nearly 100% growth inhibition at the next highest dose (10 μM). This is in contrast to the more gradual growth inhibitory activity of the other compounds tested in these experiments.</div><div class="NLM_p last">Combination studies were undertaken to determine if sublethal amounts of compound <b>1</b> could enhance the cytotoxicity of other agents including gemcitabine. MDA-MB-231 and HCT-116 cells were treated with sublethal doses of compound <b>1</b> (2.5 μM for MDA-MB-231 and 1.0 μM for HCT-116) in addition to a standard range of gemcitabine doses. Gemcitabine containing media were removed after 24 h, and media containing compound <b>1</b> was replaced and remained on the cells for the duration of the experiment (72 h). As shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Table S5</a>, the addition of sublethal amounts of compound <b>1</b> enhanced the cytotoxicity of gemcitabine, decreasing the relative Dms by 90% (MDA-MB-231, gemcitabine alone 0.92 μM; to 0.1 μM for gemcitabine plus compound <b>1</b>).</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> X-ray Structure Determination of the Phthalimide-Based Compound <b>4</b> in Complex with hRRM1</h3><div class="NLM_p">Of our 10 potential hits (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>), compound <b>4</b> (a phthalimide) was then subjected to X-ray crystallography studies in an effort to obtain a crystal structure of hRRM1 in complex with these M-site modulators.</div><div class="NLM_p">The X-ray structure of compound <b>4</b> was determined to 3.7 Å resolution in complex with hRRM1 (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). The binding of the compound was modeled to a homologous model derived from the hexameric <i>S. cerevisiae</i> structure that we had determined by X-ray crystallography, reported in 2011. Since <i>S. cerevisiae</i> and human enzymes share a sequence identity of 68% and structural homology greater than 80%, where the RMSD between the Cα atoms is less than 2 Å, it is easy to superimpose each RR1 dimer from one species to the other; hence, the same orientation matrix relating the dimeric structures from each species can be used to transpose the inhibitor to its correct location in the model. As the hRRM1 crystal form belongs to the orthorhombic class of crystals, the structure was easily transformed into the hexameric form of the homologous model using superposition (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A). This can be easily done by superimposing the orthorhombic dimer onto each of the three hexameric dimers in the coordinates PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PAW">3PAW</a>. Usually, when protein molecules are superimposed, the bound ligands are automatically placed in the correct orientation, providing a model complex structure. This transformation is required as the direct solution of the complex in the hexameric crystal form would lack the resolution (approximately 6 Å) to provide useful molecular details. Upon several rounds of refinement, the 2<i>F</i><sub>O</sub> – <i>F</i><sub>C</sub> and <i>F</i><sub>O</sub> – <i>F</i><sub>C</sub> Fourier omit maps revealed that there was ligand density at the proposed N-terminal hexamer interface (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B and C). Almost the entire compound <b>4</b> was visible in the 2<i>F</i><sub>O</sub> – <i>F</i><sub>C</sub> Fourier difference electron density map with the exception of the two terminal fluorinated methyl groups. Compound <b>4</b> makes mostly nonpolar interactions with the N-terminus of one dimer at the hexamer interface (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>D,E). <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>E depicts a ligand plot of the protein–ligand interactions. A comprehensive list of interactions is given in <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Table S6</a>, while the crucial interactions are summarized below. The oxygen atom which is bound to the C12 atom of the phthalimide ring forms a polar contact with the main chain nitrogen atom of Ala 49. Additionally, hydrophobic interactions are observed among C9, C10, C11, and C12 of compound <b>4</b> and Ala 48, Ala 49, and Ala 53. The C<sub>β</sub> atom of Ala 48 forms hydrophobic interactions with C10 of compound <b>4</b> at a distance of 3.5 Å. The nitrogen atom of Ala 49 forms hydrophobic interactions with C9, C10, C11, and C12 of the phthalimide ring at a distance of 3.5, 3.6, 3.7, and 3.6 Å, respectively (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>D,E). Additionally, C<sub>β</sub> of Ala 49 interacts with C11 and C12 atoms of the phthalimide ring at 3.6 and 3.7 Å, respectively. The C<sub>β</sub> atom of Ala 53 forms a van der Waals interaction with O7 of compound <b>4</b>. Also, residues that span the loop region formed by residues 45–52 are believed to be an important part of the dimer–dimer interface in the hRRM1 hexamer.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The surface accessible area of the phthalimide ligand when bound to the protein was 83 Å<sup>2</sup> derived with the AREAIMOL program using the solvent/probe radius to 1.4 Å.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Data Collection and Refinement Statistics for X-ray Crystal Structures of hRRM1 Bound with Compound <b>4</b></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Cell Dimensions</th><th class="colsep0 rowsep0" align="center"> </th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left"><i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>a</i>, <i>b</i>, <i>c</i>, (Å)</td><td class="colsep0 rowsep0" align="left">72.21, 112.11, 218.27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">wavelength (Å)</td><td class="colsep0 rowsep0" align="left">0.98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution (Å)</td><td class="colsep0 rowsep0" align="left">200.00–3.70</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">monomer per asymmetric unit</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unique reflections</td><td class="colsep0 rowsep0" align="left">19896</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>sym</sub><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">21.8 (89.7)<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>I</i>/σ(<i>I</i>)</td><td class="colsep0 rowsep0" align="left">28.8 (4.4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">(%) completeness</td><td class="colsep0 rowsep0" align="left">95.5 (95.7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">redundancy</td><td class="colsep0 rowsep0" align="left">3.2 (3.0)</td></tr></tbody></table><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">refinement</th><th class="colsep0 rowsep0" align="center"> </th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">number of reflections</td><td class="colsep0 rowsep0" align="left">18327</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>work</sub>/<i>R</i><sub>free</sub><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">23.2/28.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">number of atoms</td><td class="colsep0 rowsep0" align="left">11384</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein</td><td class="colsep0 rowsep0" align="left">11296</td></tr></tbody></table><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">RMS deviation from ideal</th><th class="colsep0 rowsep0" align="center"> </th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond length (Å)</td><td class="colsep0 rowsep0" align="left">0.009</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond angle (deg)</td><td class="colsep0 rowsep0" align="left">1.78</td></tr></tbody></table><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Ramachandran plot, residues in (%)</th><th class="colsep0 rowsep0" align="center"> </th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">core regions</td><td class="colsep0 rowsep0" align="left">80.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">allowed regions</td><td class="colsep0 rowsep0" align="left">17.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">generously allowed regions</td><td class="colsep0 rowsep0" align="left">1.5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last"><i>R</i><sub>sym</sub> = Σ<sub><i>hkl</i></sub>Σ<sub><i>i</i></sub>/<i>I</i><sub><i>i</i></sub>(<i>hkl</i>) – ⟨<i>I</i>(<i>hkl</i>)⟩/Σ<sub><i>hkl</i></sub>Σ<sub><i>i</i></sub><i>I</i><sub><i>i</i></sub>(<i>hkl</i>), where <i>I</i><sub><i>i</i></sub>(<i>hkl</i>) is the <i>i</i>th observation of reflection <i>hkl</i>, and ⟨<i>I</i>(<i>hkl</i>)⟩<i>i</i> is the weighted average intensity for all observations <i>i</i> of reflection <i>hkl</i>.</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last"><i>R</i><sub>work</sub> and <i>R</i><sub>free</sub> = Σ∥<i>F</i><sub>o</sub>| – |<i>F</i><sub>c</sub>∥/Σ|<i>F</i><sub>o</sub>|, where <i>F</i><sub>o</sub> and <i>F</i><sub>c</sub> are the observed and calculated structure factor amplitudes, respectively. For the calculation of <i>R</i><sub>free</sub>, 5% of the reflection data were selected and omitted from refinement.</p></div><div class="footnote" id="t2fn3"><sup>c</sup><p class="last">Values in parentheses are used for the highest-resolution shell.</p></div></div></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Mechanism of hRRM1 Inhibition by Compound <b>4</b></h3><div class="NLM_p">The mechanism of inhibition of hRRM1 by compound <b>4</b> was analyzed using steady-state kinetics (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Figure S3</a>). A plot of reaction velocity versus substrate concentration shows that the <i>v</i><sub>max</sub> is reached at NDP concentrations greater than 1 mM. Likewise, the plots for assays in the presence of compound <b>4</b> also showed substrate saturation at NDP concentrations greater than 1 mM; however, the <i>k</i><sub>cat</sub> in the presence of compound <b>4</b> decreases in response to increasing inhibitor concentration (0.67:1 at 32 μM; 0.30:1 at 64 μM) consistent with noncompetitive inhibition. A double reciprocal plot further demonstrates that compound <b>4</b> results in a decrease in <i>k</i><sub>cat</sub> while <i>K</i><sub>m</sub> is minimally affected. To quantitatively assess the mode of inhibition, the alpha value for the data set was calculated. A value of α = 1 denotes noncompetitive inhibition, α ≫ 1 denotes competitive inhibition, and α ≪ 1 denotes noncompetitive inhibition. An alpha value of 1.047 was obtained for compound <b>4</b> further supporting the interpretation of a noncompetitive mode of inhibition.</div><div class="NLM_p">Next, gel filtration experiments were conducted to study the impact of compound <b>4</b> on the oligomeric state of hRRM1 (see <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). hRRM1 mainly exists as a monomer (greater than 90%) with a small fraction of dimer (less than 10%) (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A). The addition of 1 mM phthalimide results in an increased population of the dimer compared to hRRM1 alone (approximately 40%) (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C and D). As previously observed,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> the addition of 50 μM dATP results in hexamer formation (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C). Upon addition of 1 mM phthalimide, the dimer peak diminishes significantly, while the hexamer appears to be enhanced considerably (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>D). This is observed when the area under the dATP hexamer peak in the presence of phthalimide was integrated and compared to the area under the native dATP hexamer peak (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C and D). In the presence of dATP, the area under the hexamer peak is three times that of the dimer peak. Upon addition of phthalimide, the area under the hexamer peak increases to seven times that of the dimer (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C and D) indicating that the presence of compound <b>4</b> can affect hRRM1 multimerization equilibrium.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/medium/jm-2015-00929x_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Effect of phthalimide binding on oligomerization in hRRM1 using gel filtration chromatography. (A) Chromatogram of hRRM1 with 1 mM concentration of compound <b>4</b> is shown in red, where the native hRRM1 in the absence of compound <b>4</b> is shown in black. (B) Standard curve for the determination of molecular masses (<i>Mr</i>) of RR. <i>K</i><sub>av</sub> = (<i>V</i><sub>e</sub> – <i>V</i><sub>0</sub>)/(<i>V</i><sub>t</sub> – <i>V</i><sub>0</sub>), where <i>V</i><sub>e</sub> = elution volume, <i>V</i><sub>0</sub> = void volume, and <i>V</i><sub>t</sub> = total volume. (C) hRRM1 hexamerization in the presence of dATP; at 50 μM dATP, the hexamers species are predominant with a small amount of dimers. The hexamer to dimer ratio based on integration of the peaks is approximately 3-fold. (D) The chromatogram of hRRM1 in the presence of 1 mM compound <b>4</b> and 50 μM dATP. The hexamer to dimer ratio based on integration of the peaks is approximately 7-fold.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00929&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48707" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48707" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Ribonucleotide reductase is a major cancer target. Drugs such as hydroxyurea, fludarabine, clofarabine, gemcitabine, and cladarabine are FDA-approved drugs for the treatment of cancer that target RR.<a onclick="showRef(event, 'ref5 ref7 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref5 ref7 ref41 ref42">(5, 7, 41, 42)</a> Hydroxyurea targets the small subunit of RR, while the others are nucleoside analogues that target the large subunit (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A). Our aim was to discover a class of small-molecule inhibitors that targeted hRRM1.</div><div class="NLM_p">In this study, we screened for small molecules that target RR using virtual screening where the hits were confirmed by fluorescence quenching and RR inhibition assays. We targeted the hexamer interface of RR in our <i>in silico</i> screen (<a class="ref internalNav" href="#fig1" aria-label="Figure1">Figure1</a>A and <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A). Recent reports have revealed the advantage of targeting protein–protein interfaces in drug discovery.<a onclick="showRef(event, 'ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref43 ref44 ref45">(43-45)</a> Out of the 76 compounds that were screened for binding to hRRM1 using fluorescence quenching, 51% of them produced at least 25% quenching indicating that they were binders (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Table S1</a>). We were able to obtain sufficient amounts for compounds <b>1</b>, <b>4</b>, <b>6</b>, and <b>8</b> to determine the dissociation constant using fluorescence quenching. These compounds’ <i>K</i><sub>D</sub>’s range from 10–55 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Figure S5</a>).</div><div class="NLM_p">As most of the results were considered chemically redundant, the compounds were condensed into groups of unique chemical classes (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). From a medicinal chemistry perspective, a few of the scaffolds represented in this group of 10 inhibitors have prior non-RR medicinal activities. The ethyl ester version of compound <b>1</b> (with PubChem ID: NSC-282192 and CAS 7325-88-4) was identified through NCI’s anticancer screen against mec2-1, rad50, and rad14 assays. Related compounds have also been reported as vectors for cell transformation and as antimicrobial agents.<a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46, 47)</a> Compound <b>3</b> has a C<sub>14</sub> alkyl chain; however, its C<sub>12</sub> analogue has been reported in the literature as antibacterial. The peptidyl scaffold presented in compound <b>7</b> is a commonly observed substructure in multiple angiotensin-related peptides. Compound <b>6</b>, the styrenyl sulfonate DPS Benzoate is an analogue of proparacaine, that has activity in ion channels and aryl hydrocarbon receptor and sigma-1-receptor modulation. No definitive medicinal activity reports are available, in the context of inhibition against RR, for compounds <b>4</b> (OxoIsoIndoLys), <b>5</b> (BHNaphthamide, CAS: 18643-47-5), <b>8</b> (BoPEAP), and <b>9</b> (AEOHydBen), and this study therefore is the first to report them.</div><div class="NLM_p">Of these, compound <b>5</b> (ButHyNitNap) was the most potent inhibitor of RR with an IC<sub>50</sub> of 21 ± 1.1 μM. In general, the IC<sub>50</sub>’s of these compounds ranged from 21–60 μM, closely shadowing their <i>K</i><sub>D</sub>’s. This suggests that the potency could be enhanced using a medicinal chemistry and/or structure-based drug design approach.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Although cocrystallization attempts with the phthalimide (compound <b>4</b>) derivative were conducted, they did not produce cocrystals possibly due to the high percentage of DMSO, which was essential for solubilizing the ligand.</div><div class="NLM_p">Soaking experiments with the phthalimide derivative successfully yielded its X-ray structure defining its binding site on the hRRM1 hexamer interface (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A and <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A–C). The structure reveals that the phthalimide binds at a surface pocket interacting with the β-cap at the ATP-binding cone. The surface accessibility of the free ligand is 606 Å<sup>2</sup> where upon binding the protein, the surface accessibility is reduced to 461 Å<sup>2</sup> indicating that 24% of the ligand is buried upon binding hRRM1, suggesting that the ligand binds in a surface pocket (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A). The loop including residues 47–49 in the β cap has been identified as an important region crucial for dATP and ATP binding, required for inactivation and activation of the enzyme, respectively.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> The pocket defined by phthalimide binding has not been previously observed as a ligand binding site for hRRM1. However, this binding pocket lies within approximately 8 Å of the activity site. <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>D shows the proximity and relationship between the M-site and A-site. Portions of the N-terminal β-cap (1–14, 48–51) and helix H3 (residues 53–70) are shared between the two sites (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>D). Interestingly, a surface pocket has been identified in a similar study for nuclear receptors where modulators bind.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div><div class="NLM_p">Our structural findings were further confirmed by analyzing the mechanism of inhibition of hRRM1 by compound <b>4</b> using steady-state kinetics (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Figure S3</a>). The velocity over substrate plots at multiple inhibition concentrations and the Lineweaver–Burk plots have profiles reminiscent of noncompetitive inhibitors where <i>K</i><sub>m</sub> remains the same, while <i>k</i><sub>cat</sub> changes. On the basis of this mode of inhibition, compound <b>4</b> will bind both the free enzyme and the enzyme–substrate complex. This model is consistent with our X-ray crystal structure where compound <b>4</b> binds at a site independent of the catalytic site.</div><div class="NLM_p">Targeting protein–protein interfaces such as the hexamer interface is gaining momentum in the postgenome era due to the abundance of such targets in the human genome.<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43, 44)</a> Although in the past protein–protein interface drug targets have been considered to be extremely challenging, new breakthroughs in drug discovery have produced several promising examples that are progressing through the late stages of clinical trials. The flat shapeless surfaces of the protein–protein interface were considered to be a major hurdle for drug design.<a onclick="showRef(event, 'ref43 ref45'); return false;" href="javascript:void(0);" class="ref ref43 ref45">(43, 45)</a> However, there are hotspots within the protein–protein interface of a few residues that can be targeted by small-molecule fragments to obtain high affinity inhibitors.<a onclick="showRef(event, 'ref44 ref51'); return false;" href="javascript:void(0);" class="ref ref44 ref51">(44, 51)</a> The structure of compound <b>4</b> suggests that the phthalimide fragment binds in such a hotspot at the hexamer interface, allowing us to design highly potent RR modulators (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B–E). Fragment-based drug design is considered a prudent strategy for targeting protein–protein interfaces.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> The phthalimide structure provides a good starting point for fragment-based drug design. Although the resolution of our structure is limited to 3.7 Å, similar resolution structures in combination with isothermal titration calorimetry (ITC) have been successfully used with the acetylcholine binding protein in fragment-based drug design.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></div><div class="NLM_p">The impact of the phthalimide compound on the oligomeric states of hRRM1 was investigated using gel filtration chromatography (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). Our studies show that the addition of the phthalimide into nucleoside-free hRRM1 shifts the equilibrium from monomer to dimer (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a> A). This is an interesting finding, as our structure (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>) reveals that the phthalimide binds at the hexamer interface, which is far from the dimerization domain near loop 1 (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A). These results indicate that phthalimide binding at the hexamer interface possibly stabilizes and promotes enhancement of the dimer population. Our studies on how the phthalimide affects the dATP hexamer is shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C and D. We observed that the dimer to hexamer equilibrium shifts in favor of the hexamer. These results support the notion that the phthalimide fragment binding at the hexamer interface strengthens the inactive hexamer formation. Proteins such as PBGS, phenylalanine hydroxylase, HIV-integrase, and pyruvate kinase, which follow the morpheein model of allostery, also are regulated by oligomerization, and small-molecule modulators have been shown to shift the equilibrium from lower order to higher order oligomers.<a onclick="showRef(event, 'ref35 ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref37">(35-37)</a></div><div class="NLM_p">Independently, 51 of the 76 hits from the <i>in silico</i> screen were tested for growth inhibition and cytotoxicity using cultured cancer cell lines (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). Compound <b>1</b>, a fluoronyl piperazine, was observed to enhance the cytotoxicity of gemcitabine when administered together (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). This observation suggests that RR modulators may be able to enhance the toxicity of existing drugs, when administered in combination. By varying the structural scaffold of these compounds systematically, more optimal structures may be found. Such studies are currently being pursued for two of most promising hits identified from this study.</div><div class="NLM_p last">In summary, we have conducted an <i>in silico</i> screen of potential inhibitors of hRRM1. Our studies have identified 10 potential hits that target hRRM1 with low to medium micromolar IC<sub>50</sub>’s. Remarkably, no 2 of these 10 compounds resemble each other closely. The structure of the phthalimide compound bound to hRRM1 confirms that it binds at the hexamer interface. In general, most of the compounds tested inhibited RR with IC<sub>50</sub> values in the micromolar range. This bears implications for exploration of these chemical spaces as novel non-nucleoside inhibitors of RR. Future work will involve the study of the structure–activity relationship (SAR) and the mechanism of inhibition of the remaining nine potential hits (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) using enzyme kinetics and X-ray crystallography. These studies will definitively identify their respective binding sites. The strategy used in this study can be adapted to obtain novel lead compounds directed against hRRM1 using additional chemical libraries.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57316" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57316" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Virtual Screening of the Cincinnati Library against hRRM1</h3><div class="NLM_p">In order to conduct virtual screening against hRRM1, we used a homologous model of the dATP-induced hexamer that was based on the <i>S. cerevisiae</i> structure (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a>). <i>S. cerevisiae</i> shares a 68% sequence identity and a greater than 80% structural homology with the human enzyme. The model was made by substituting the hRRM1 sequence onto the <i>S. cerevisiae</i> structure followed by energy minimization.</div><div class="NLM_p last"><i>In silico</i> docking of the Cincinnati chemical library (formerly the Proctor & Gamble chemical library) was performed independently using the Glide docking module of the Schrödinger 9.3 modeling software suite.<a onclick="showRef(event, 'ref53 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref53 ref54 ref55">(53-55)</a> The hits were scored using a docking function and a glide scoring function (glide score). The docking process is described in further detail in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Supporting Information</a>. When considering the best hits, more weight was given to careful examination of the docking poses; especially, consensus interactions with the same residues defining the docking site were high in our rankings. The top 90 hits were subjected to the PAINS filter using the Web site <a href="http://cbligand.org/PAINS/" class="extLink">http://cbligand.org/PAINS/</a>.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> This filter removed 14 hits.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Protein Expression and Purification</h3><div class="NLM_p last">The hRRM1 protein was expressed in <i>E. coli</i> BL21-codon plus (DE3)-RIL cells and purified using peptide affinity chromatography to homogeneity as previously described by Fairman et al.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The hRRM2 protein was also expressed in <i>E. coli</i> BL21-codon plus (DE3) cells and purified to homogeneity using Ni-NTA affinity chromatography, and the protein concentrations were measured as described.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Fluorescence Quenching</h3><div class="NLM_p last">Tryptophan fluorescence spectra of hRRM1 at 0.1 mg/mL in 50 mM Tris, pH 8.0, 5% glycerol, 5 mM MgCl<sub>2</sub>, and 10 mM DTT (Assay buffer) were recorded using a Jobin Yvon-Spex fluorescence spectrophotometer by exciting the sample at 295 nm. The protein samples were treated with ligand at a fixed concentration of 50 μM with the 74 compounds obtained from the virtual screen. The spectra were corrected for the inherent fluorescence contributions made by the ligand. Compounds exhibiting quenching greater than 25% were kept for further evaluation. We have used 0.1 mg/mL bovine serum albumin (BSA) and 1 μM <i>N</i>-acetyltryptophanamide (NATA) as controls. Controls for nonspecific or artificial inhibition were tested by subjecting two independent phthalimide derivatives derived from an unrelated chemical library to the Cincinnati library using the fluorescence quenching assay shown in <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Figure S1</a>.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> <i>K</i><sub>D</sub> Determination Using Fluorescence Quenching</h3><div class="NLM_p last">We attempted to measure the dissociation constant (<i>K</i><sub>D</sub>) for four compounds in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> (compounds <b>1</b>, <b>4</b>, <b>6</b>, and <b>8</b>) using the following procedure. Unfortunately, the six remaining compounds were not available in sufficient quantities to conduct the same experiments. Tryptophan fluorescence spectra of hRRM1 at 0.3 mg/mL in assay buffer were recorded using a Jobin Yvon-Spex fluorescence spectrophotometer by exciting the sample with 295 nm light. The hRRM1 sample was titrated with increasing concentrations (1.25 μM–400 μM) of the individual compounds at room temperature. The data were fitted by nonlinear regression using the one-site binding (hyperbola) equation <i>Y</i> = <i>B</i><sub>max</sub>·<i>X</i>/(<i>K</i><sub>d(app)</sub> + <i>X</i>), where <i>B</i><sub>max</sub> is the maximum extent of quenching, and <i>K</i><sub>d(app)</sub> is the apparent dissociation constant, using GraphPad Prism 6.0 software (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Figure S5</a>). Measurements were recorded in duplicate in order to estimate error.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Ribonucleotide Reductase Inhibition Assays and Mechanism</h3><div class="NLM_p last">The specific activity of hRR was determined <i>in vitro</i> using <sup>14</sup>C-ADP reduction assays as previously described.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28, 29)</a> The full assay protocol, including purification of hRRM2, is described in detail in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Supporting Information</a>. Briefly, we adopted a two-point method for IC<sub>50</sub> determination using the procedure described by Krippendorff et al.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> as only limited amounts of the hits were available. On the basis of this method, we used 5 and 100 μM concentrations of the ligand for measuring the IC<sub>50</sub>. As a control we used hydroxyurea, a common RR drug used against cancer, to validate our two-point method for deriving IC<sub>50</sub>. For instance, the two-point method gave an IC<sub>50</sub> of 0.997 μM, while a traditional IC<sub>50</sub> measurement using multiple points gave us an IC<sub>50</sub> of 1.07 μM, confirming that both methods give almost identical numbers. The product <sup>14</sup>C-dADP that formed during the reaction was separated from substrate <sup>14</sup>C-ADP using boronate affinity chromatography.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a><sup>14</sup>C-dADP was quantified by liquid scintillation counting using a Beckman LS6500 liquid scintillation counter. The IC<sub>50</sub> was defined as the concentration of any compound that reduced the specific activity of hRRM1 to 50% of the control activity. The inhibition mechanism was analyzed by generating double-reciprocal plots and fitting with the mixed-model equations as described in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Growth Inhibition Screening Assays for Determining Cellular Toxicity</h3><div class="NLM_p last">To assess cellular toxicity, growth inhibition assays were conducted using the standard MTT assay with the cancer cell lines MDA-MB-231, HCT-116, A549, and Panc1. Median effect doses (Dm) were calculated using Calcusyn, version 2.0. A detailed description of the procedures is described in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Crystallographic Studies of Compound <b>4</b> Bound to hRRM1</h3><div class="NLM_p last">A full description of the crystallization and structure solution are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Supporting Information</a>. Briefly, hRRM1 was crystallized as described.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> As compound <b>4</b> could not be cocrystallized with hRRM1, the TTP-bound orthorhombic crystals were soaked with 100–500 μM of compound <b>4</b> for 2 h, and the crystals were cryogenized and data collected at the NE-CAT beamline at APS. A full description of the refinement and model building is provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Gel Filtration Chromatography</h3><div class="NLM_p last">The effect of compound <b>4</b> on the oligomeric state of hRRM1 was assessed using gel filtration chromatography using a Superdex 200 (GE Healthcare) column as described.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> A full description of the procedure is found in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22"><a href="/doi/suppl/10.1021/acs.jmedchem.5b00929">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10065" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10065" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b00929" class="ext-link">10.1021/acs.jmedchem.5b00929</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Experimental methods for virtual screening, RR inhibition assays, inhibition mechanism, growth inhibition screening assays for determining cellular toxicity, crystallization of hRRM1, data collection, and structure determination; docking scores for Schrödinger; fluorescence quenching data; medicinal chemistry data; growth inhibition data for all compounds tested in MDA-MB-231 breast cancer and HCT-116 colon cancer cell lines; median effect doses (Dm) for compound <b>1</b>; interactions between compound <b>4</b> and hRRM1 (within 5 Å); enzyme kinetic mechanism of compound <b>4</b> inhibition of hRRM1; docking poses for the 10 compounds in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf">jm5b00929_si_001.pdf (10.04 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">4X3V.</p><div class="testing" data-doi="10.1021/acs.jmedchem.5b00929" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35574" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35574" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chris Godfrey Dealwis</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmacology, School of Medicine, Case Western Reserve University, Cleveland, Ohio 44106, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Center for Proteomics and the Department of Chemistry, Case Western Reserve University, Cleveland, Ohio 44106, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#74171c061d075a10111518031d0734171507115a111001"><span class="__cf_email__" data-cfemail="5e3d362c372d703a3b3f3229372d1e3d3f2d3b703b3a2b">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Md. Faiz Ahmad</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmacology, School of Medicine, Case Western Reserve University, Cleveland, Ohio 44106, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sarah E. Huff</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Case Western Reserve University, Cleveland, Ohio 44106, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John Pink</span> - <span class="hlFld-Affiliation affiliation">Case Comprehensive Cancer Center, Case
Western Reserve University, Cleveland, Ohio 44106, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Intekhab Alam</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmacology, School of Medicine, Case Western Reserve University, Cleveland, Ohio 44106, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew Zhang</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmacology, School of Medicine, Case Western Reserve University, Cleveland, Ohio 44106, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kay Perry</span> - <span class="hlFld-Affiliation affiliation">Northeastern-CAT at the Advanced Photon
Source, Argonne National Laboratory, Argonne, Illinois 60439, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael E. Harris</span> - <span class="hlFld-Affiliation affiliation">Department of Biochemistry, School of Medicine, Case Western Reserve University, Cleveland, Ohio 44106, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tessianna Misko</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmacology, School of Medicine, Case Western Reserve University, Cleveland, Ohio 44106, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Suheel K. Porwal</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Dehradun Institute of Technology, University of Deharadun, Dehradun 248197, India</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nancy L. Oleinick</span> - <span class="hlFld-Affiliation affiliation">Case Comprehensive Cancer Center, Case
Western Reserve University, Cleveland, Ohio 44106, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Radiation Oncology, School
of Medicine, Case Western Reserve University, Cleveland, Ohio 44106, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masaru Miyagi</span> - <span class="hlFld-Affiliation affiliation">Center for Proteomics
and Bioinformatics, Case Western Reserve
University, Cleveland, Ohio 44106, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rajesh Viswanathan</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Case Western Reserve University, Cleveland, Ohio 44106, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.F.A. and S.E.H. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d9797e2133-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17754" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17754" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was funded by the National Institutes of Health, grants R01GM100887 and R01CA100827. T.M. was supported by training grant 5R25CA148052-05. This research was also supported by the Translational Research & Pharmacology Core Facility of the Case Comprehensive Cancer Center (P30 CA43703) and the Early Clinical Trials of Anti-Cancer Agents with Phase I Emphasis UO1 grant (U01 CA062502). We thank the Case Western Reserve University School of Medicine and the University of Cincinnati for making the chemical library available to us for screening. We thank the members of NE-CAT at the APS and X29 at NSLS for assistance with data collection. The NE-CAT beamlines are supported by a grant from the National Institute of General Medical Sciences (P41 GM103403) from the National Institutes of Health. This research used resources of the Advanced Photon Source, a U.S. Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Argonne National Laboratory under Contract No. DE-AC02-06CH11357. We acknowledge Mr. Andrew Burr for assisting us with ligand interaction analysis. We appreciate William Seibel for sharing his expert knowledge on the Cincinnati library. We would like to acknowledge Banumathi Sankaran for screening crystals at the ALS beamline. We thank Kathleen Lundberg for her assistance in the mass spectrometry analysis.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i26" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i26"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i27" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i27"> Abbreviations Used</h2><tr><td class="NLM_term">NATA</td><td class="NLM_def"><p class="first last"><i>N-</i>acetyl-<i>L</i>-tryptophanamide</p></td></tr><tr><td class="NLM_term">hRR</td><td class="NLM_def"><p class="first last">human ribonucleotide reductase</p></td></tr><tr><td class="NLM_term">hRRM1</td><td class="NLM_def"><p class="first last">large subunit of human ribonucleotide reductase</p></td></tr><tr><td class="NLM_term"><i>Kav</i></td><td class="NLM_def"><p class="first last">partition coefficient</p></td></tr><tr><td class="NLM_term">hRRM2</td><td class="NLM_def"><p class="first last">small subunit of human ribonucleotide reductase</p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">pan assay interference compounds</p></td></tr><tr><td class="NLM_term">dATP</td><td class="NLM_def"><p class="first last">deoxyadenosine 5′-triphosphate</p></td></tr><tr><td class="NLM_term">Dm</td><td class="NLM_def"><p class="first last">median effect dose</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12201" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12201" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 57 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Jordheim, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seve, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tredan, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dumontet, C.</span><span> </span><span class="NLM_article-title">The ribonucleotide reductase large subunit (RRM1) as a predictive factor in patients with cancer</span> <span class="citation_source-journal">Lancet Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">702</span><span class="refDoi"> DOI: 10.1016/S1470-2045(10)70244-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1016%2FS1470-2045%2810%2970244-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=21163702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3MXot1ejur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=693-702&author=L.+P.+Jordheimauthor=P.+Seveauthor=O.+Tredanauthor=C.+Dumontet&title=The+ribonucleotide+reductase+large+subunit+%28RRM1%29+as+a+predictive+factor+in+patients+with+cancer&doi=10.1016%2FS1470-2045%2810%2970244-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The ribonucleotide reductase large subunit (RRM1) as a predictive factor in patients with cancer</span></div><div class="casAuthors">Jordheim, Lars Petter; Seve, Pascal; Tredan, Olivier; Dumontet, Charles</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">693-702</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Summary: The large subunit of human ribonucleotide reductase, RRM1, is involved in the regulation of cell proliferation, cell migration, tumor and metastasis development, and the synthesis of deoxyribonucleotides for DNA synthesis.  It is also a cellular target for the chemotherapeutic agent, gemcitabine.  RRM1 has been studied in a large no. of patients with different types of cancer, such as non-small-cell lung cancer, pancreatic cancer, breast cancer, and biliary tract cancer, to establish its prognostic or predictive value when patients were treated with gemcitabine, and mRNA expression and genetic variants as detd. by genotyping have in some cases been assocd. with clin. outcome of patients with cancer.  Here, we review preclin. and clin. studies of RRM1 assessment and discuss the further steps in the development of this clin. pertinent biomarker.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo06llwPkTEqbVg90H21EOLACvtfcHk0lga76QJ8lRZQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXot1ejur0%253D&md5=e64d3933721a6bcf856ae763ff8ec47a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2810%2970244-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252810%252970244-8%26sid%3Dliteratum%253Aachs%26aulast%3DJordheim%26aufirst%3DL.%2BP.%26aulast%3DSeve%26aufirst%3DP.%26aulast%3DTredan%26aufirst%3DO.%26aulast%3DDumontet%26aufirst%3DC.%26atitle%3DThe%2520ribonucleotide%2520reductase%2520large%2520subunit%2520%2528RRM1%2529%2520as%2520a%2520predictive%2520factor%2520in%2520patients%2520with%2520cancer%26jtitle%3DLancet%2520Oncol.%26date%3D2011%26volume%3D12%26spage%3D693%26epage%3D702%26doi%3D10.1016%2FS1470-2045%2810%2970244-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Jordheim, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guittet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lepoivre, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galmarini, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dumontet, C.</span><span> </span><span class="NLM_article-title">Increased expression of the large subunit of ribonucleotide reductase is involved in resistance to gemcitabine in human mammary adenocarcinoma cells</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1268</span><span class="NLM_x">–</span> <span class="NLM_lpage">1276</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-05-0121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1158%2F1535-7163.MCT-05-0121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=16093443" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnt1Shtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=1268-1276&author=L.+P.+Jordheimauthor=O.+Guittetauthor=M.+Lepoivreauthor=C.+M.+Galmariniauthor=C.+Dumontet&title=Increased+expression+of+the+large+subunit+of+ribonucleotide+reductase+is+involved+in+resistance+to+gemcitabine+in+human+mammary+adenocarcinoma+cells&doi=10.1158%2F1535-7163.MCT-05-0121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Increased expression of the large subunit of ribonucleotide reductase is involved in resistance to gemcitabine in human mammary adenocarcinoma cells</span></div><div class="casAuthors">Jordheim, Lars Petter; Guittet, Olivier; Lepoivre, Michel; Galmarini, Carlos M.; Dumontet, Charles</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1268-1276</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Resistance to cytotoxic nucleoside analogs is a major problem in cancer treatment.  The cellular mechanisms involved in this phenomenon have been studied for several years, and some factors have been identified.  However, this resistance seems to be multifactorial and more studies are needed to gain better insight into this domain.  For this purpose, we developed a gemcitabine-resistant cell line (MCF7 1K) from the human mammary adenocarcinoma MCF7 strain by prolonged exposure to gemcitabine in vitro.  MCF7 1K cells are highly resistant to gemcitabine (533-fold) and cross-resistance is obsd. with araC (47-fold), triapine (14-fold), and hydroxyurea (6.7-fold).  Quant. real-time reverse transcription-PCR and Western blot anal. showed an increase in the gene and protein expression of the large subunit of ribonucleotide reductase, R1.  Ribonucleotide reductase activity was also significantly increased in the gemcitabine-resistant cells.  Study of genomic DNA showed 12-fold increase in R1 gene dosage in MCF7 1K cells.  In contrast, the gene and protein expression of the small subunit of ribonucleotide reductase, R2, were not modified in this cell line.  These results show that gemcitabine resistance can be assocd. with genetic modifications of target genes in malignant cells, and suggest that the large subunit of human ribonucleotide reductase is involved in the cellular response to gemcitabine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9Uk0sNRqiYrVg90H21EOLACvtfcHk0liMjPQV2aWF4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnt1Shtbw%253D&md5=4b78ae54a2d3ba8c1195ad2f1c34edd9</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-05-0121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-05-0121%26sid%3Dliteratum%253Aachs%26aulast%3DJordheim%26aufirst%3DL.%2BP.%26aulast%3DGuittet%26aufirst%3DO.%26aulast%3DLepoivre%26aufirst%3DM.%26aulast%3DGalmarini%26aufirst%3DC.%2BM.%26aulast%3DDumontet%26aufirst%3DC.%26atitle%3DIncreased%2520expression%2520of%2520the%2520large%2520subunit%2520of%2520ribonucleotide%2520reductase%2520is%2520involved%2520in%2520resistance%2520to%2520gemcitabine%2520in%2520human%2520mammary%2520adenocarcinoma%2520cells%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2005%26volume%3D4%26spage%3D1268%26epage%3D1276%26doi%3D10.1158%2F1535-7163.MCT-05-0121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Shao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, Y.</span><span> </span><span class="NLM_article-title">Ribonucleotide reductase inhibitors and future drug design</span> <span class="citation_source-journal">Curr. Cancer Drug Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">409</span><span class="NLM_x">–</span> <span class="NLM_lpage">431</span><span class="refDoi"> DOI: 10.2174/156800906777723949</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.2174%2F156800906777723949" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=16918309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD28XnslCmsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2006&pages=409-431&author=J.+Shaoauthor=B.+Zhouauthor=B.+Chuauthor=Y.+Yen&title=Ribonucleotide+reductase+inhibitors+and+future+drug+design&doi=10.2174%2F156800906777723949"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Ribonucleotide reductase inhibitors and future drug design</span></div><div class="casAuthors">Shao, J.; Zhou, B.; Chu, Bernard; Yen, Y.</div><div class="citationInfo"><span class="NLM_cas:title">Current Cancer Drug Targets</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">409-431</span>CODEN:
                <span class="NLM_cas:coden">CCDTB9</span>;
        ISSN:<span class="NLM_cas:issn">1568-0096</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Ribonucleotide reductase (RR) is a multisubunit enzyme responsible for the redn. of ribonucleotides to their corresponding deoxyribonucleotides, which are building blocks for DNA replication and repair.  The key role of RR in DNA synthesis and cell growth control has made it an important target for anticancer therapy.  Increased RR activity has been assocd. with malignant transformation and tumor cell growth.  Efforts for new RR inhibitors have been made in basic and translational research.  In recent years, several RR inhibitors, including Triapine, Gemcitabine, and GTI-2040, have entered clin. trial or application.  Furthermore, the discovery of p53R2, a p53-inducible form of the small subunit of RR, raises the interest to develop subunit-specific RR inhibitors for cancer treatment.  This review compiles recent studies on (1) the structure, function, and regulation of two forms of RR; (2) the role in tumorigenesis of RR and the effect of RR inhibition in cancer treatment; (3) the classification, mechanisms of action, antitumor activity, and clin. trial and application of new RR inhibitors that have been used in clin. cancer chemotherapy or are being evaluated in clin. trials; (4) novel approaches for future RR inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaMYCdtQEZLVg90H21EOLACvtfcHk0liMjPQV2aWF4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnslCmsL8%253D&md5=cafea6b6ddf5aff0a4d75c09b120c2b9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2174%2F156800906777723949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156800906777723949%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DChu%26aufirst%3DB.%26aulast%3DYen%26aufirst%3DY.%26atitle%3DRibonucleotide%2520reductase%2520inhibitors%2520and%2520future%2520drug%2520design%26jtitle%3DCurr.%2520Cancer%2520Drug%2520Targets%26date%3D2006%26volume%3D6%26spage%3D409%26epage%3D431%26doi%3D10.2174%2F156800906777723949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Tang, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. C.</span><span> </span><span class="NLM_article-title">Novel therapeutic targets for pancreatic cancer</span> <span class="citation_source-journal">World J. Gastroenterol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">10825</span><span class="NLM_x">–</span> <span class="NLM_lpage">10844</span><span class="refDoi"> DOI: 10.3748/wjg.v20.i31.10825</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.3748%2Fwjg.v20.i31.10825" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=25152585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFajurbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=10825-10844&author=S.+C.+Tangauthor=Y.+C.+Chen&title=Novel+therapeutic+targets+for+pancreatic+cancer&doi=10.3748%2Fwjg.v20.i31.10825"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Novel therapeutic targets for pancreatic cancer</span></div><div class="casAuthors">Tang, Shing-Chun; Chen, Yang-Chao</div><div class="citationInfo"><span class="NLM_cas:title">World Journal of Gastroenterology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">10825-10844, 21 pp.</span>CODEN:
                <span class="NLM_cas:coden">WJGAF2</span>;
        ISSN:<span class="NLM_cas:issn">1007-9327</span>.
    
            (<span class="NLM_cas:orgname">Baishideng Publishing Group Inc.</span>)
        </div><div class="casAbstract">A review.  Pancreatic cancer has become the fourth leading cause of cancer death in the last two decades.  Only 3%-15% of patients diagnosed with pancreatic cancer had 5 yr survival rate.  Drug resistance, high metastasis, poor prognosis and tumor relapse contributed to the malignancies and difficulties in treating pancreatic cancer.  The current std. chemotherapy for pancreatic cancer is gemcitabine, however its efficacy is far from satisfactory, one of the reasons is due to the complex tumor microenvironment which decreases effective drug delivery to target cancer cell.  Studies of the mol. pathol. of pancreatic cancer have revealed that activation of KRAS, overexpression of cyclooxygenase-2, inactivation of p16INK4A and loss of p53 activities occurred in pancreatic cancer.  Co-administration of gemcitabine and targeting the mol. pathol. events happened in pancreatic cancer has brought an enhanced therapeutic effectiveness of gemcitabine.  Therefore, studies looking for novel targets in hindering pancreatic tumor growth are emerging rapidly.  In order to give a better understanding of the current findings and to seek the direction in future pancreatic cancer research; in this review we will focus on targets suppressing tumor metastasis and progression, KRAS activated downstream effectors, the relationship of Notch signaling and Nodal/Activin signaling with pancreatic cancer cells, the current findings of non-coding RNAs in inhibiting pancreatic cancer cell proliferation, brief discussion in transcription remodeling by epigenetic modifiers (e.g., HDAC, BMI1, EZH2) and the plausible therapeutic applications of cancer stem cell and hyaluronan in tumor environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4pSDAY_HLObVg90H21EOLACvtfcHk0liMjPQV2aWF4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFajurbK&md5=50dfb899e09c4f2dfd7708ddbe828972</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3748%2Fwjg.v20.i31.10825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3748%252Fwjg.v20.i31.10825%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DS.%2BC.%26aulast%3DChen%26aufirst%3DY.%2BC.%26atitle%3DNovel%2520therapeutic%2520targets%2520for%2520pancreatic%2520cancer%26jtitle%3DWorld%2520J.%2520Gastroenterol.%26date%3D2014%26volume%3D20%26spage%3D10825%26epage%3D10844%26doi%3D10.3748%2Fwjg.v20.i31.10825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Huang, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chubb, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plunkett, W.</span><span> </span><span class="NLM_article-title">Termination of DNA synthesis by 9-beta-D-arabinofuranosyl-2-fluoroadenine. A mechanism for cytotoxicity</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">265</span><span class="NLM_x">, </span> <span class="NLM_fpage">16617</span><span class="NLM_x">–</span> <span class="NLM_lpage">16625</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1697861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADyaK3cXlvVCgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=265&publication_year=1990&pages=16617-16625&author=P.+Huangauthor=S.+Chubbauthor=W.+Plunkett&title=Termination+of+DNA+synthesis+by+9-beta-D-arabinofuranosyl-2-fluoroadenine.+A+mechanism+for+cytotoxicity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Termination of DNA synthesis by 9-β-D-arabinofuranosyl-2-fluoroadenine.  A mechanism for cytotoxicity</span></div><div class="casAuthors">Huang, Peng; Chubb, Sherri; Plunkett, William</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">265</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">16617-25</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">The action of 9-β-D-arabinofuranosyl-2-fluoroadenine (F-ara-A) on DNA synthesis was evaluated in whole cells and in vitro.  F-ara-A was converted to its 5'-triphosphate (F-ara-ATP) in cells and then incorporated into DNA in a self-limiting manner.  More than 94% of the analog ws incorporated into DNA at the 3'-termini, indicating a chain termination action.  In vitro DNA primer extension expts. further revealed that F-ara-ATP competed with dATP for incorporation into the A site of the extending DNA strand.  The incorporation of F-ara-AMP into DNA resulted in termination of DNA strand elongation.  Human DNA polymerase α incorporated more F-ara-AMP into DNA than polymerase ε (proliferating cell nuclear antigen-independent DNA polymerase δ) and was more sensitive to inhibition by F-ara-ATP.  DNA polymerase ε was able to excise the incorporated F-ara-AMP from DNA in vitro.  The incorporation of F-ara-AMP into DNA was linearly correlated both with inhibition of DNA synthesis and with loss of clonogenicity; thus it may be the mechanism of its cytotoxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplGaiLMmReHLVg90H21EOLACvtfcHk0ljuRnDE--Wj2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXlvVCgurw%253D&md5=1ce760d8ae93741a29a2a690527a9449</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DP.%26aulast%3DChubb%26aufirst%3DS.%26aulast%3DPlunkett%26aufirst%3DW.%26atitle%3DTermination%2520of%2520DNA%2520synthesis%2520by%25209-beta-D-arabinofuranosyl-2-fluoroadenine.%2520A%2520mechanism%2520for%2520cytotoxicity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1990%26volume%3D265%26spage%3D16617%26epage%3D16625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Avery, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rehg, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lumm, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harwood, F. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santana, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blakley, R. L.</span><span> </span><span class="NLM_article-title">Biochemical pharmacology of 2-chlorodeoxyadenosine in malignant human hematopoietic cell lines and therapeutic effects of 2-bromodeoxyadenosine in drug combinations in mice</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">4972</span><span class="NLM_x">–</span> <span class="NLM_lpage">4978</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1989&pages=4972-4978&author=T.+L.+Averyauthor=J.+E.+Rehgauthor=W.+C.+Lummauthor=F.+C.+Harwoodauthor=V.+M.+Santanaauthor=R.+L.+Blakley&title=Biochemical+pharmacology+of+2-chlorodeoxyadenosine+in+malignant+human+hematopoietic+cell+lines+and+therapeutic+effects+of+2-bromodeoxyadenosine+in+drug+combinations+in+mice"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAvery%26aufirst%3DT.%2BL.%26aulast%3DRehg%26aufirst%3DJ.%2BE.%26aulast%3DLumm%26aufirst%3DW.%2BC.%26aulast%3DHarwood%26aufirst%3DF.%2BC.%26aulast%3DSantana%26aufirst%3DV.%2BM.%26aulast%3DBlakley%26aufirst%3DR.%2BL.%26atitle%3DBiochemical%2520pharmacology%2520of%25202-chlorodeoxyadenosine%2520in%2520malignant%2520human%2520hematopoietic%2520cell%2520lines%2520and%2520therapeutic%2520effects%2520of%25202-bromodeoxyadenosine%2520in%2520drug%2520combinations%2520in%2520mice%26jtitle%3DCancer%2520Res.%26date%3D1989%26volume%3D49%26spage%3D4972%26epage%3D4978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Griffig, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koob, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blakley, R. L.</span><span> </span><span class="NLM_article-title">Mechanisms of inhibition of DNA synthesis by 2-chlorodeoxyadenosine in human lymphoblastic cells</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6923</span><span class="NLM_x">–</span> <span class="NLM_lpage">6928</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1989&pages=6923-6928&author=J.+Griffigauthor=R.+Koobauthor=R.+L.+Blakley&title=Mechanisms+of+inhibition+of+DNA+synthesis+by+2-chlorodeoxyadenosine+in+human+lymphoblastic+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGriffig%26aufirst%3DJ.%26aulast%3DKoob%26aufirst%3DR.%26aulast%3DBlakley%26aufirst%3DR.%2BL.%26atitle%3DMechanisms%2520of%2520inhibition%2520of%2520DNA%2520synthesis%2520by%25202-chlorodeoxyadenosine%2520in%2520human%2520lymphoblastic%2520cells%26jtitle%3DCancer%2520Res.%26date%3D1989%26volume%3D49%26spage%3D6923%26epage%3D6928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Parker, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaddix, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brockman, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shortnacy, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montgomery, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Secrist, J. A.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, L. L.,  Jr.</span><span> </span><span class="NLM_article-title">Effects of 2-chloro-9-(2-deoxy-2-fluoro-beta-D-arabinofuranosyl)adenine on K562 cellular metabolism and the inhibition of human ribonucleotide reductase and DNA polymerases by its 5′-triphosphate</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2386</span><span class="NLM_x">–</span> <span class="NLM_lpage">2394</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1991&pages=2386-2394&author=W.+B.+Parkerauthor=S.+C.+Shaddixauthor=C.+H.+Changauthor=E.+L.+Whiteauthor=L.+M.+Roseauthor=R.+W.+Brockmanauthor=A.+T.+Shortnacyauthor=J.+A.+Montgomeryauthor=J.+A.+Secristauthor=L.+L.+Bennett&title=Effects+of+2-chloro-9-%282-deoxy-2-fluoro-beta-D-arabinofuranosyl%29adenine+on+K562+cellular+metabolism+and+the+inhibition+of+human+ribonucleotide+reductase+and+DNA+polymerases+by+its+5%E2%80%B2-triphosphate"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DParker%26aufirst%3DW.%2BB.%26aulast%3DShaddix%26aufirst%3DS.%2BC.%26aulast%3DChang%26aufirst%3DC.%2BH.%26aulast%3DWhite%26aufirst%3DE.%2BL.%26aulast%3DRose%26aufirst%3DL.%2BM.%26aulast%3DBrockman%26aufirst%3DR.%2BW.%26aulast%3DShortnacy%26aufirst%3DA.%2BT.%26aulast%3DMontgomery%26aufirst%3DJ.%2BA.%26aulast%3DSecrist%26aufirst%3DJ.%2BA.%26aulast%3DBennett%26aufirst%3DL.%2BL.%26atitle%3DEffects%2520of%25202-chloro-9-%25282-deoxy-2-fluoro-beta-D-arabinofuranosyl%2529adenine%2520on%2520K562%2520cellular%2520metabolism%2520and%2520the%2520inhibition%2520of%2520human%2520ribonucleotide%2520reductase%2520and%2520DNA%2520polymerases%2520by%2520its%25205%25E2%2580%25B2-triphosphate%26jtitle%3DCancer%2520Res.%26date%3D1991%26volume%3D51%26spage%3D2386%26epage%3D2394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Heinemann, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Y. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chubb, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hertel, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grindey, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plunkett, W.</span><span> </span><span class="NLM_article-title">Inhibition of ribonucleotide reduction in CCRF-CEM cells by 2′,2′-difluorodeoxycytidine</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">567</span><span class="NLM_x">–</span> <span class="NLM_lpage">572</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1990&pages=567-572&author=V.+Heinemannauthor=Y.+Z.+Xuauthor=S.+Chubbauthor=A.+Senauthor=L.+W.+Hertelauthor=G.+B.+Grindeyauthor=W.+Plunkett&title=Inhibition+of+ribonucleotide+reduction+in+CCRF-CEM+cells+by+2%E2%80%B2%2C2%E2%80%B2-difluorodeoxycytidine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHeinemann%26aufirst%3DV.%26aulast%3DXu%26aufirst%3DY.%2BZ.%26aulast%3DChubb%26aufirst%3DS.%26aulast%3DSen%26aufirst%3DA.%26aulast%3DHertel%26aufirst%3DL.%2BW.%26aulast%3DGrindey%26aufirst%3DG.%2BB.%26aulast%3DPlunkett%26aufirst%3DW.%26atitle%3DInhibition%2520of%2520ribonucleotide%2520reduction%2520in%2520CCRF-CEM%2520cells%2520by%25202%25E2%2580%25B2%252C2%25E2%2580%25B2-difluorodeoxycytidine%26jtitle%3DMol.%2520Pharmacol.%26date%3D1990%26volume%3D38%26spage%3D567%26epage%3D572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Parker, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaddix, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shewach, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hertel, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Secrist, J. A.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montgomery, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, L. L.,  Jr.</span><span> </span><span class="NLM_article-title">Comparison of the mechanism of cytotoxicity of 2-chloro-9-(2-deoxy-2- fluoro-beta-D-arabinofuranosyl)adenine, 2-chloro-9-(2-deoxy-2-fluoro- beta-D-ribofuranosyl)adenine, and 2-chloro-9-(2-deoxy-2,2-difluoro- beta-D-ribofuranosyl)adenine in CEM cells</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">515</span><span class="NLM_x">–</span> <span class="NLM_lpage">520</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=1999&pages=515-520&author=W.+B.+Parkerauthor=S.+C.+Shaddixauthor=L.+M.+Roseauthor=D.+S.+Shewachauthor=L.+W.+Hertelauthor=J.+A.+Secristauthor=J.+A.+Montgomeryauthor=L.+L.+Bennett&title=Comparison+of+the+mechanism+of+cytotoxicity+of+2-chloro-9-%282-deoxy-2-+fluoro-beta-D-arabinofuranosyl%29adenine%2C+2-chloro-9-%282-deoxy-2-fluoro-+beta-D-ribofuranosyl%29adenine%2C+and+2-chloro-9-%282-deoxy-2%2C2-difluoro-+beta-D-ribofuranosyl%29adenine+in+CEM+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DParker%26aufirst%3DW.%2BB.%26aulast%3DShaddix%26aufirst%3DS.%2BC.%26aulast%3DRose%26aufirst%3DL.%2BM.%26aulast%3DShewach%26aufirst%3DD.%2BS.%26aulast%3DHertel%26aufirst%3DL.%2BW.%26aulast%3DSecrist%26aufirst%3DJ.%2BA.%26aulast%3DMontgomery%26aufirst%3DJ.%2BA.%26aulast%3DBennett%26aufirst%3DL.%2BL.%26atitle%3DComparison%2520of%2520the%2520mechanism%2520of%2520cytotoxicity%2520of%25202-chloro-9-%25282-deoxy-2-%2520fluoro-beta-D-arabinofuranosyl%2529adenine%252C%25202-chloro-9-%25282-deoxy-2-fluoro-%2520beta-D-ribofuranosyl%2529adenine%252C%2520and%25202-chloro-9-%25282-deoxy-2%252C2-difluoro-%2520beta-D-ribofuranosyl%2529adenine%2520in%2520CEM%2520cells%26jtitle%3DMol.%2520Pharmacol.%26date%3D1999%26volume%3D55%26spage%3D515%26epage%3D520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Cerqueira, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandes, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramos, M. J.</span><span> </span><span class="NLM_article-title">Ribonucleotide reductase: a critical enzyme for cancer chemotherapy and antiviral agents</span> <span class="citation_source-journal">Recent Pat. Anti-Cancer Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">11</span><span class="NLM_x">–</span> <span class="NLM_lpage">29</span><span class="refDoi"> DOI: 10.2174/157489207779561408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.2174%2F157489207779561408" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=11-29&author=N.+M.+Cerqueiraauthor=P.+A.+Fernandesauthor=M.+J.+Ramos&title=Ribonucleotide+reductase%3A+a+critical+enzyme+for+cancer+chemotherapy+and+antiviral+agents&doi=10.2174%2F157489207779561408"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.2174%2F157489207779561408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157489207779561408%26sid%3Dliteratum%253Aachs%26aulast%3DCerqueira%26aufirst%3DN.%2BM.%26aulast%3DFernandes%26aufirst%3DP.%2BA.%26aulast%3DRamos%26aufirst%3DM.%2BJ.%26atitle%3DRibonucleotide%2520reductase%253A%2520a%2520critical%2520enzyme%2520for%2520cancer%2520chemotherapy%2520and%2520antiviral%2520agents%26jtitle%3DRecent%2520Pat.%2520Anti-Cancer%2520Drug%2520Discovery%26date%3D2007%26volume%3D2%26spage%3D11%26epage%3D29%26doi%3D10.2174%2F157489207779561408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohman, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stubbe, J.</span><span> </span><span class="NLM_article-title">Mechanism of inactivation of human ribonucleotide reductase with p53R2 by gemcitabine 5′-diphosphate</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">11612</span><span class="NLM_x">–</span> <span class="NLM_lpage">11621</span><span class="refDoi"> DOI: 10.1021/bi901588z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi901588z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVens7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=11612-11621&author=J.+Wangauthor=G.+J.+Lohmanauthor=J.+Stubbe&title=Mechanism+of+inactivation+of+human+ribonucleotide+reductase+with+p53R2+by+gemcitabine+5%E2%80%B2-diphosphate&doi=10.1021%2Fbi901588z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of Inactivation of Human Ribonucleotide Reductase with p53R2 by Gemcitabine 5'-Diphosphate</span></div><div class="casAuthors">Wang, Jun; Lohman, Gregory J. S.; Stubbe, JoAnne</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">11612-11621</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ribonucleotide reductases (RNRs) catalyze the conversion of nucleoside 5'-diphosphates to the corresponding deoxynucleotides, supplying the dNTPs required for DNA replication and DNA repair.  Class I RNRs require two subunits, α and β, for activity.  Humans possess two β subunits: one involved in S phase DNA replication (β), and a second one involved in mitochondrial DNA replication (β' or p53R2) and potentially DNA repair.  Gemcitabine (F2C) is used clin. as an anticancer agent, and its phosphorylated metabolites target many enzymes involved in nucleotide metab., including RNR.  The present investigation with α (specific activity of 400 nmol min-1 mg-1) and β' (0.6 Y·/β'2 and a specific activity of 420 nmol min-1 mg-1) establishes that F2CDP is a substoichiometric inactivator of RNR.  Incubation of this α/β' with [1'-3H]-F2CDP or [5-3H]-F2CDP and reisolation of the protein by Sephadex G-50 chromatog. resulted in recovery 0.5 equiv of covalently bound sugar and 0.03 equiv of tightly assocd. cytosine to α2.  SDS-PAGE anal. (loaded without boiling) of the inactivated RNR showed that 60% of α migrates as a 90 kDa protein and 40% as a 120 kDa protein.  Incubation of [1'-3H]-F2CDP with active site mutants C444S/A, C218S/A, and E431Q/D-α and the C-terminal tail C787S/A and C790S/A mutants reveals that no sugar label is bound to the active site mutants of α and that in the case of C218S-α, α migrates as a 90 kDa protein.  Anal. of the inactivated wt-α/β' RNR by size exclusion chromatog. indicates a quaternary structure of α6β'6.  A mechanism of inactivation common with hα/β is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9SOObMT8YvLVg90H21EOLACvtfcHk0lhFIgzE13F_lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVens7zJ&md5=8b1f3f09511b91e2bf328e4f0df18d1a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fbi901588z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi901588z%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLohman%26aufirst%3DG.%2BJ.%26aulast%3DStubbe%26aufirst%3DJ.%26atitle%3DMechanism%2520of%2520inactivation%2520of%2520human%2520ribonucleotide%2520reductase%2520with%2520p53R2%2520by%2520gemcitabine%25205%25E2%2580%25B2-diphosphate%26jtitle%3DBiochemistry%26date%3D2009%26volume%3D48%26spage%3D11612%26epage%3D11621%26doi%3D10.1021%2Fbi901588z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Huang, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chubb, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hertel, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grindey, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plunkett, W.</span><span> </span><span class="NLM_article-title">Action of 2′,2′-difluorodeoxycytidine on DNA synthesis</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">6110</span><span class="NLM_x">–</span> <span class="NLM_lpage">6117</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1718594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADyaK38XnsFOqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1991&pages=6110-6117&author=P.+Huangauthor=S.+Chubbauthor=L.+W.+Hertelauthor=G.+B.+Grindeyauthor=W.+Plunkett&title=Action+of+2%E2%80%B2%2C2%E2%80%B2-difluorodeoxycytidine+on+DNA+synthesis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Action of 2',2'-difluorodeoxycytidine on DNA synthesis</span></div><div class="casAuthors">Huang, Peng; Chubb, Sherri; Hertel, Larry W.; Grindey, Gerald B.; Plunkett, William</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6110-17</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    </div><div class="casAbstract">The action of the new deoxycytidine analog 2',2'-difluorodeoxycytidine (dFdC) on DNA synthesis was investigated in whole cells and in vitro assay systems with purified DNA polymerases.  DNA synthesis in human lymphoblastoid CEM cells was inhibited by dFdC in a concn.-dependent manner that could not be reversed by exogenous deoxynucleosides.  The analog was incorporated into cellular DNA; most of the incorporated dFdC 5'-monophosphate (dFdCMP) residues were in internucleotide linkage.  In vitro DNA primer extension assays demonstrated that dFdC 5'-triphosphate (dFdCTP) competed with deoxycytidine triphosphate for incorporation into the C sites of the growing DNA strand.  The ratios of the apparent Km values for the incorporation of dFdCTP and dCTP into a C site of M13mp19 DNA were 21.8 and 22.9 for DNA polymerases α and ε, resp.  The apparent Ki values of dFdCTP were 11.2 μM for DNA polymerase α and 14.4 μM for polymerase ε.  After dFdCMP incorporation, the primer was extended by one deoxynucleotide before a major pause in the polymn. process was obsd.  This was in contrast to the action of arabinosylcytosine 5'-triphosphate, which caused both DNA polymerases α and ε to pause at the site of incorporation.  The 3'→5' exonuclease activity of DNA polymerase ε was essentially unable to excise nucleotides from DNA contg. dFdCMP at either the 3'-end or at an internal position, whereas arabinosylcytosine monophosphate was removed from the 3'-terminus at 37% the rate for deoxynucleotides.  The cytotoxic activity of dFdC was strongly correlated with the amt. of dFdCMP incorporated into cellular DNA.  These results demonstrate qual. and quant. differences in the mol. actions of dFdC and arabinosylcytosine on DNA metab., but are consistent with an important role for such incorporation in the toxicity of dFdC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofN5itudznZLVg90H21EOLACvtfcHk0lhFIgzE13F_lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XnsFOqsA%253D%253D&md5=2e7f4ee419ecbee58845b1a983750858</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DP.%26aulast%3DChubb%26aufirst%3DS.%26aulast%3DHertel%26aufirst%3DL.%2BW.%26aulast%3DGrindey%26aufirst%3DG.%2BB.%26aulast%3DPlunkett%26aufirst%3DW.%26atitle%3DAction%2520of%25202%25E2%2580%25B2%252C2%25E2%2580%25B2-difluorodeoxycytidine%2520on%2520DNA%2520synthesis%26jtitle%3DCancer%2520Res.%26date%3D1991%26volume%3D51%26spage%3D6110%26epage%3D6117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Heinemann, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulz, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Issels, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plunkett, W.</span><span> </span><span class="NLM_article-title">Gemcitabine: a modulator of intracellular nucleotide and deoxynucleotide metabolism</span> <span class="citation_source-journal">Semin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">11</span><span class="NLM_x">–</span> <span class="NLM_lpage">18</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=7481839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADyaK2MXpsFGntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1995&pages=11-18&author=V.+Heinemannauthor=L.+Schulzauthor=R.+D.+Isselsauthor=W.+Plunkett&title=Gemcitabine%3A+a+modulator+of+intracellular+nucleotide+and+deoxynucleotide+metabolism"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Gemcitabine: a modulator of intracellular nucleotide and deoxynucleotide metabolism</span></div><div class="casAuthors">Heinemann, Volker; Schulz, Lothar; Issels, Rolf D.; Plunkett, William</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Oncology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">4, Suppl. 11</span>),
    <span class="NLM_cas:pages">11-18</span>CODEN:
                <span class="NLM_cas:coden">SOLGAV</span>;
        ISSN:<span class="NLM_cas:issn">0093-7754</span>.
    
            (<span class="NLM_cas:orgname">Saunders</span>)
        </div><div class="casAbstract">Gemcitabine (2',2'-difluorodeoxycytidine, dFdC) is a deoxycytidine (dCyd) analog that extensively modulates intracellular CTP and dCTP metab.  In Chinese hamster ovary (CHO) cells, a 4-h exposure to gemcitabine (100 μM) reduced cellular CTP and dCTP concns. to 5.9% and 50%, resp., of control values.  Intracellular UTP concns. increased, indicating a metabolic block at CTP synthetase.  Pool sizes of ATP and GTP remained unaffected.  In contrast, a CHO mutant deficient in deoxycytidine kinase, and thus unable to accumulate dFdCTP, maintained its CTP pools under identical conditions, suggesting that the CTP pool depletion was dependent on gemcitabine phosphorylation.  Neither 100 μM arabinosylcytosine nor 5 mM hydroxyurea affected CTP levels, indicating that inhibition of DNA synthesis by analog incorporation or by depletion of deoxynucleotide triphosphate pools was not the cause of the CTP pool perturbation.  Metabolic studies demonstrated that incorporation of [3H]uridine into the UTP pool was not impaired by gemcitabine, whereas the specific activity of the CTP pools decreased as a function of increasing gemcitabine concn. and time of exposure.  Comparable results were obtained by using 3-deazauridine, a known inhibitor of CTP synthetase.  It is concluded that high cellular concns. of dFdCTP deplete cellular CTP concns. by inhibition of the dCTP pool and also may be a limiting factor for RNA synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8-eqnW776frVg90H21EOLACvtfcHk0lh8Yq0CwrTrCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXpsFGntbc%253D&md5=39daaaf3c2730983939356963cce40ed</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHeinemann%26aufirst%3DV.%26aulast%3DSchulz%26aufirst%3DL.%26aulast%3DIssels%26aufirst%3DR.%2BD.%26aulast%3DPlunkett%26aufirst%3DW.%26atitle%3DGemcitabine%253A%2520a%2520modulator%2520of%2520intracellular%2520nucleotide%2520and%2520deoxynucleotide%2520metabolism%26jtitle%3DSemin.%2520Oncol.%26date%3D1995%26volume%3D22%26spage%3D11%26epage%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Gmeiner, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pon, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pourquier, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span> </span><span class="NLM_article-title">Structural basis for topoisomerase I inhibition by nucleoside analogs</span> <span class="citation_source-journal">Nucleosides, Nucleotides Nucleic Acids</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">653</span><span class="NLM_x">–</span> <span class="NLM_lpage">658</span><span class="refDoi"> DOI: 10.1081/NCN-120022604</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1081%2FNCN-120022604" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=14565246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntFChtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=653-658&author=W.+H.+Gmeinerauthor=S.+Yuauthor=R.+T.+Ponauthor=P.+Pourquierauthor=Y.+Pommier&title=Structural+basis+for+topoisomerase+I+inhibition+by+nucleoside+analogs&doi=10.1081%2FNCN-120022604"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Topoisomerase I Inhibition by Nucleoside Analogs</span></div><div class="casAuthors">Gmeiner, William H.; Yu, Shuyuan; Pon, Richard T.; Pourquier, Philippe; Pommier, Yves</div><div class="citationInfo"><span class="NLM_cas:title">Nucleosides, Nucleotides & Nucleic Acids</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5-8</span>),
    <span class="NLM_cas:pages">653-658</span>CODEN:
                <span class="NLM_cas:coden">NNNAFY</span>;
        ISSN:<span class="NLM_cas:issn">1525-7770</span>.
    
            (<span class="NLM_cas:orgname">Marcel Dekker, Inc.</span>)
        </div><div class="casAbstract">A review.  Nucleoside analogs such as 1-β-D-arabinofuranosyl cytidine (AraC) and 2', 2'-difluoro deoxycytidine (dFdC) are important components of the anticancer chemotherapeutic arsenal and are among the most effective anticancer drugs currently available.  Although both AraCTP and dFdCTP impede DNA replication through pausing of DNA polymerases, both nucleoside analogs are ultimately incorporated into replicated DNA and interfere in DNA-mediated processes.  Our labs. are investigating the structural basis for the poisoning of topoisomerase I (top1) due to antipyrimidine incorporation into duplex DNA.  We recently reported that both AraC and dFdC induce formation of top1 cleavage complexes, and poisoning of top1 contributes to the anticancer activities of both these drugs.  Recent NMR and thermodn. studies from our labs. provide insight into the mechanism by which AraC and dFdC poison top 1.  NMR studies from our labs. have revealed that the arabinosyl sugar of AraC adopted a C2'-endo conformation.  Although this is a B-type sugar pucker characteristic of duplex DNA, the conformation is rigid, and this lack of flexibility probably contributes to inhibition of the religation step of the top 1 reaction.  In contrast to AraC, NMR studies revealed dFdC adopted a C3' endo sugar pucker characteristic of RNA, rather than DNA duplexes.  DFdC substitution enhanced formation of top1 cleavage complexes, but did not inhibit religation.  The enhancement of top1 cleavage complexes most likely results from a combination of conformational and electrostatic effects.  The structural effects of dFdC and AraC are being further investigated in duplex DNA with well-defined top1 cleavage sites to analyze more specifically how these structural perturbations lead to enzyme poisoning.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqU2NZ1ttFrlrVg90H21EOLACvtfcHk0lh8Yq0CwrTrCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntFChtrs%253D&md5=c950a0a08d9777b215b38cfc639bc4db</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1081%2FNCN-120022604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1081%252FNCN-120022604%26sid%3Dliteratum%253Aachs%26aulast%3DGmeiner%26aufirst%3DW.%2BH.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DPon%26aufirst%3DR.%2BT.%26aulast%3DPourquier%26aufirst%3DP.%26aulast%3DPommier%26aufirst%3DY.%26atitle%3DStructural%2520basis%2520for%2520topoisomerase%2520I%2520inhibition%2520by%2520nucleoside%2520analogs%26jtitle%3DNucleosides%252C%2520Nucleotides%2520Nucleic%2520Acids%26date%3D2003%26volume%3D22%26spage%3D653%26epage%3D658%26doi%3D10.1081%2FNCN-120022604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Zhou, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yip, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaur, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Synold, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horne, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, Y.</span><span> </span><span class="NLM_article-title">A small-molecule blocking ribonucleotide reductase holoenzyme formation inhibits cancer cell growth and overcomes drug resistance</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">6484</span><span class="NLM_x">–</span> <span class="NLM_lpage">6493</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-1094</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1158%2F0008-5472.CAN-13-1094" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=6484-6493&author=B.+Zhouauthor=L.+Suauthor=S.+Huauthor=W.+Huauthor=M.+L.+Yipauthor=J.+Wuauthor=S.+Gaurauthor=D.+L.+Smithauthor=Y.+C.+Yuanauthor=T.+W.+Synoldauthor=D.+Horneauthor=Y.+Yen&title=A+small-molecule+blocking+ribonucleotide+reductase+holoenzyme+formation+inhibits+cancer+cell+growth+and+overcomes+drug+resistance&doi=10.1158%2F0008-5472.CAN-13-1094"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-1094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-1094%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DSu%26aufirst%3DL.%26aulast%3DHu%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DW.%26aulast%3DYip%26aufirst%3DM.%2BL.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DGaur%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DD.%2BL.%26aulast%3DYuan%26aufirst%3DY.%2BC.%26aulast%3DSynold%26aufirst%3DT.%2BW.%26aulast%3DHorne%26aufirst%3DD.%26aulast%3DYen%26aufirst%3DY.%26atitle%3DA%2520small-molecule%2520blocking%2520ribonucleotide%2520reductase%2520holoenzyme%2520formation%2520inhibits%2520cancer%2520cell%2520growth%2520and%2520overcomes%2520drug%2520resistance%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D6484%26epage%3D6493%26doi%3D10.1158%2F0008-5472.CAN-13-1094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Brown, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canellakis, Z. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reichard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thelander, L.</span><span> </span><span class="NLM_article-title">Ribonucleoside diphosphate reductase. Purification of the two subunits, proteins B1 and B2</span> <span class="citation_source-journal">Eur. J. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1969</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">561</span><span class="NLM_x">–</span> <span class="NLM_lpage">573</span><span class="refDoi"> DOI: 10.1111/j.1432-1033.1969.tb00646.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1111%2Fj.1432-1033.1969.tb00646.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=4896737" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADyaF1MXks1Ols7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1969&pages=561-573&author=N.+C.+Brownauthor=Z.+N.+Canellakisauthor=B.+Lundinauthor=P.+Reichardauthor=L.+Thelander&title=Ribonucleoside+diphosphate+reductase.+Purification+of+the+two+subunits%2C+proteins+B1+and+B2&doi=10.1111%2Fj.1432-1033.1969.tb00646.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Ribonucleoside diphosphate reductase.  Purification of the two subunits, proteins B1 and B2</span></div><div class="casAuthors">Brown, Neal Curtis; Canellakis, Z. N.; Lundin, B.; Reichard, Peter; Thelander, Lars</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Biochemistry</span>
        (<span class="NLM_cas:date">1969</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">561-73</span>CODEN:
                <span class="NLM_cas:coden">EJBCAI</span>;
        ISSN:<span class="NLM_cas:issn">0014-2956</span>.
    </div><div class="casAbstract">Ribonucleoside diphosphate reductase from Escherichia coli B consists of 2 nonidentical subunits, called proteins B1 and B2.  After purification from exts. of a partially derepressed thymine- requiring mutant, both subunits were obtained in essentially pure form.  Criteria of purity included the demonstration of const. specific enzyme activity in peaks from final chromatographic or electrophoretic steps and the behavior of the proteins during ultracentrifugation and anal. gel electrophoresis.  Detns. of mol. wts. by sedimentation equil. centrifugation gave values between 160,000 and 200,000 for protein B1 and 78,000 for protein B2.  Evidence was obtained for some dissocn. of protein B1 during centrifugation.  Sep., each subunit was completely inactive; together, under the proper conditions, they catalyzed the redn. of the 5'-diphosphates of cytidine, uridine, adenosine, and guanosine to the corresponding deoxyribonucleotides.  With the exception of deoxyadenosine 5'-triphosphate, the regulatory effects of nucleoside triphosphates were identical to those observed earlier with less pure prepns. of enzyme.  The general inhibitory effect of deoxyadenosine 5'-triphosphate at concns. higher than 1μM was obtained as before.  In addn., lower concns. of this nucleotide had a novel stimulatory effect on the redn. of CDP.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5SRt2D08lprVg90H21EOLACvtfcHk0lj56ENGAOP0sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF1MXks1Ols7s%253D&md5=e41cb10135d1abc3918c947683dd2e06</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1111%2Fj.1432-1033.1969.tb00646.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1432-1033.1969.tb00646.x%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DN.%2BC.%26aulast%3DCanellakis%26aufirst%3DZ.%2BN.%26aulast%3DLundin%26aufirst%3DB.%26aulast%3DReichard%26aufirst%3DP.%26aulast%3DThelander%26aufirst%3DL.%26atitle%3DRibonucleoside%2520diphosphate%2520reductase.%2520Purification%2520of%2520the%2520two%2520subunits%252C%2520proteins%2520B1%2520and%2520B2%26jtitle%3DEur.%2520J.%2520Biochem.%26date%3D1969%26volume%3D9%26spage%3D561%26epage%3D573%26doi%3D10.1111%2Fj.1432-1033.1969.tb00646.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Brown, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reichard, P.</span><span> </span><span class="NLM_article-title">Role of effector binding in allosteric control of ribonucleoside diphosphate reductase</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1969</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">39</span><span class="NLM_x">–</span> <span class="NLM_lpage">55</span><span class="refDoi"> DOI: 10.1016/0022-2836(69)90056-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1016%2F0022-2836%2869%2990056-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=1969&pages=39-55&author=N.+C.+Brownauthor=P.+Reichard&title=Role+of+effector+binding+in+allosteric+control+of+ribonucleoside+diphosphate+reductase&doi=10.1016%2F0022-2836%2869%2990056-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2869%2990056-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252869%252990056-4%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DN.%2BC.%26aulast%3DReichard%26aufirst%3DP.%26atitle%3DRole%2520of%2520effector%2520binding%2520in%2520allosteric%2520control%2520of%2520ribonucleoside%2520diphosphate%2520reductase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1969%26volume%3D46%26spage%3D39%26epage%3D55%26doi%3D10.1016%2F0022-2836%2869%2990056-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Eriksson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uhlin, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramaswamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ekberg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Regnstrom, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sjoberg, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eklund, H.</span><span> </span><span class="NLM_article-title">Binding of allosteric effectors to ribonucleotide reductase protein R1: reduction of active-site cysteines promotes substrate binding</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1077</span><span class="NLM_x">–</span> <span class="NLM_lpage">1092</span><span class="refDoi"> DOI: 10.1016/S0969-2126(97)00259-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1016%2FS0969-2126%2897%2900259-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=9309223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADyaK2sXmt1Wgsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1997&pages=1077-1092&author=M.+Erikssonauthor=U.+Uhlinauthor=S.+Ramaswamyauthor=M.+Ekbergauthor=K.+Regnstromauthor=B.+M.+Sjobergauthor=H.+Eklund&title=Binding+of+allosteric+effectors+to+ribonucleotide+reductase+protein+R1%3A+reduction+of+active-site+cysteines+promotes+substrate+binding&doi=10.1016%2FS0969-2126%2897%2900259-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Binding of allosteric effectors to ribonucleotide reductase protein R1: reduction of active-site cysteines promotes substrate binding</span></div><div class="casAuthors">Eriksson, Mathias; Uhlin, Ulla; Ramaswamy, S.; Ekberg, Monica; Regnstrom, Karin; Sjoberg, Britt-Marie; Eklund, Hans</div><div class="citationInfo"><span class="NLM_cas:title">Structure (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1077-1092</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Current Biology</span>)
        </div><div class="casAbstract">Ribonucleotide reductase (RNR) is an essential enzyme in DNA synthesis, catalyzing all de novo synthesis of deoxyribonucleotides.  The enzyme comprises two dimers, termed R1 and R2, and contains the redox active cysteine residues, Cys462 and Cys225.  The redn. of ribonucleotides to deoxyribonucleotides involves the transfer of free radicals.  The pathway for the radical has previously been suggested from crystallog. results, and is supported by site-directed mutagenesis studies.  Most RNRs are allosterically regulated through two different nucleotide-binding sites: one site controls general activity and the other controls substrate specificity.  Our aim has been to crystallog. demonstrate substrate binding and to locate the two effector-binding sites.  We report here the first crystal structure of RNR R1 in a reduced form.  The structure shows that upon redn. of the redox active cysteines, the sulfur atom of Cys462 becomes deeply buried.  The more accessible Cys225 moves to the former position of Cys462 making room for the substrate.  In addn., the structures of R1 in complexes with effector, effector analog and effector plus substrate provide information about these binding sites.  The substrate GDP binds in a cleft between two domains with its β-phosphate bound to the N termini of two helixes; the ribose forms hydrogen bonds to conserved residues.  Binding of dTTP at the allosteric substrate specificity site stabilizes three loops close to the dimer interface and the active site, whereas the general allosteric binding site is positioned far from the active site.  Binding of substrate at the active site of the enzyme is structurally regulated in two ways: binding of the correct substrate is regulated by the binding of allosteric effectors, and binding of the actual substrate occurs primarily when the active-site cysteines are reduced.  One of the loops stabilized upon binding of dTTP participates in the formation of the substrate-binding site through direct interaction with the nucleotide base.  The general allosteric effector site, located far from the active site, appears to regulate subunit interactions within the holoenzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZ9X-VPWPN7Vg90H21EOLACvtfcHk0lj56ENGAOP0sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmt1Wgsrc%253D&md5=3b1cb456c53f6d9ccd4f303e0f2c330d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2897%2900259-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252897%252900259-1%26sid%3Dliteratum%253Aachs%26aulast%3DEriksson%26aufirst%3DM.%26aulast%3DUhlin%26aufirst%3DU.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DEkberg%26aufirst%3DM.%26aulast%3DRegnstrom%26aufirst%3DK.%26aulast%3DSjoberg%26aufirst%3DB.%2BM.%26aulast%3DEklund%26aufirst%3DH.%26atitle%3DBinding%2520of%2520allosteric%2520effectors%2520to%2520ribonucleotide%2520reductase%2520protein%2520R1%253A%2520reduction%2520of%2520active-site%2520cysteines%2520promotes%2520substrate%2520binding%26jtitle%3DStructure%26date%3D1997%26volume%3D5%26spage%3D1077%26epage%3D1092%26doi%3D10.1016%2FS0969-2126%2897%2900259-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Larsson, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eliasson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reichard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Logan, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordlund, P.</span><span> </span><span class="NLM_article-title">Structural mechanism of allosteric substrate specificity regulation in a ribonucleotide reductase</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1142</span><span class="NLM_x">–</span> <span class="NLM_lpage">1149</span><span class="refDoi"> DOI: 10.1038/nsmb838</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1038%2Fnsmb838" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=15475969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpt1ejsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=1142-1149&author=K.+M.+Larssonauthor=A.+Jordanauthor=R.+Eliassonauthor=P.+Reichardauthor=D.+T.+Loganauthor=P.+Nordlund&title=Structural+mechanism+of+allosteric+substrate+specificity+regulation+in+a+ribonucleotide+reductase&doi=10.1038%2Fnsmb838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structural mechanism of allosteric substrate specificity regulation in a ribonucleotide reductase</span></div><div class="casAuthors">Larsson, Karl-Magnus; Jordan, Albert; Eliasson, Rolf; Reichard, Peter; Logan, Derek T.; Nordlund, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1142-1149</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Ribonucleotide reductases (RNRs) catalyze the redn. of ribonucleotides into deoxyribonucleotides, which constitute the precursor pools used for DNA synthesis and repair.  Imbalances in these pools increase mutational rates and are detrimental to the cell.  Balanced precursor pools are maintained primarily through the regulation of the RNR substrate specificity.  Here, the mol. mechanism of the allosteric substrate specificity regulation is revealed through the structures of a dimeric coenzyme B12-dependent RNR from Thermotoga maritima, both in complexes with four effector-substrate nucleotide pairs and in three complexes with only effector.  The mechanism is based on the flexibility of loop 2, a key structural element, which forms a bridge between the specificity effector and substrate nucleotides.  Substrate specificity is achieved as different effectors and their cognate substrates stabilize specific discrete loop 2 conformations.  The mechanism of substrate specificity regulation is probably general for most class I and class II RNRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkLGAmhsPRF7Vg90H21EOLACvtfcHk0lj56ENGAOP0sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpt1ejsL0%253D&md5=07cdbaa804737ec8af326306b1e3b2b8</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnsmb838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb838%26sid%3Dliteratum%253Aachs%26aulast%3DLarsson%26aufirst%3DK.%2BM.%26aulast%3DJordan%26aufirst%3DA.%26aulast%3DEliasson%26aufirst%3DR.%26aulast%3DReichard%26aufirst%3DP.%26aulast%3DLogan%26aufirst%3DD.%2BT.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DStructural%2520mechanism%2520of%2520allosteric%2520substrate%2520specificity%2520regulation%2520in%2520a%2520ribonucleotide%2520reductase%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D1142%26epage%3D1149%26doi%3D10.1038%2Fnsmb838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Xu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faber, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uchiki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairman, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Racca, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dealwis, C.</span><span> </span><span class="NLM_article-title">Structures of eukaryotic ribonucleotide reductase I provide insights into dNTP regulation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">4022</span><span class="NLM_x">–</span> <span class="NLM_lpage">4027</span><span class="refDoi"> DOI: 10.1073/pnas.0600443103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1073%2Fpnas.0600443103" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=4022-4027&author=H.+Xuauthor=C.+Faberauthor=T.+Uchikiauthor=J.+W.+Fairmanauthor=J.+Raccaauthor=C.+Dealwis&title=Structures+of+eukaryotic+ribonucleotide+reductase+I+provide+insights+into+dNTP+regulation&doi=10.1073%2Fpnas.0600443103"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0600443103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0600443103%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DH.%26aulast%3DFaber%26aufirst%3DC.%26aulast%3DUchiki%26aufirst%3DT.%26aulast%3DFairman%26aufirst%3DJ.%2BW.%26aulast%3DRacca%26aufirst%3DJ.%26aulast%3DDealwis%26aufirst%3DC.%26atitle%3DStructures%2520of%2520eukaryotic%2520ribonucleotide%2520reductase%2520I%2520provide%2520insights%2520into%2520dNTP%2520regulation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D4022%26epage%3D4027%26doi%3D10.1073%2Fpnas.0600443103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Kumar, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdulovic, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viberg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilsson, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunkel, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chabes, A.</span><span> </span><span class="NLM_article-title">Mechanisms of mutagenesis in vivo due to imbalanced dNTP pools</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">1360</span><span class="NLM_x">–</span> <span class="NLM_lpage">1371</span><span class="refDoi"> DOI: 10.1093/nar/gkq829</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1093%2Fnar%2Fgkq829" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=20961955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3MXis1Chs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=1360-1371&author=D.+Kumarauthor=A.+L.+Abdulovicauthor=J.+Vibergauthor=A.+K.+Nilssonauthor=T.+A.+Kunkelauthor=A.+Chabes&title=Mechanisms+of+mutagenesis+in+vivo+due+to+imbalanced+dNTP+pools&doi=10.1093%2Fnar%2Fgkq829"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of mutagenesis in vivo due to imbalanced dNTP pools</span></div><div class="casAuthors">Kumar, Dinesh; Abdulovic, Amy L.; Viberg, Joergen; Nilsson, Anna Karin; Kunkel, Thomas A.; Chabes, Andrei</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1360-1371</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The mechanisms by which imbalanced dNTPs induce mutations have been well characterized within a test tube, but not in vivo.  We have examd. mechanisms by which dNTP imbalances induce genome instability in strains of Saccharomyces cerevisiae with different amino acid substitutions in Rnr1, the large subunit of ribonucleotide reductase.  These strains have different dNTP imbalances that correlate with elevated CAN1 mutation rates, with both substitution and insertion-deletion rates increasing by 10- to 300-fold.  The locations of the mutations in a strain with elevated dTTP and dCTP are completely different from those in a strain with elevated dATP and dGTP.  Thus, imbalanced dNTPs reduce genome stability in a manner that is highly dependent on the nature and degree of the imbalance.  Mutagenesis is enhanced despite the availability of proofreading and mismatch repair.  The mutations can be explained by imbalanced dNTP-induced increases in misinsertion, strand misalignment and mismatch extension at the expense of proofreading.  This implies that the relative dNTP concns. measured in exts. are truly available to a replication fork in vivo.  An interesting mutational strand bias is obsd. in one rnr1 strain, suggesting that the S-phase checkpoint selectively prevents replication errors during leading strand replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEZ-Es5BD9hLVg90H21EOLACvtfcHk0ljby-7AUDxkdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXis1Chs74%253D&md5=1d61a978fd83aae91563ab8f3994afd2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq829&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq829%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DD.%26aulast%3DAbdulovic%26aufirst%3DA.%2BL.%26aulast%3DViberg%26aufirst%3DJ.%26aulast%3DNilsson%26aufirst%3DA.%2BK.%26aulast%3DKunkel%26aufirst%3DT.%2BA.%26aulast%3DChabes%26aufirst%3DA.%26atitle%3DMechanisms%2520of%2520mutagenesis%2520in%2520vivo%2520due%2520to%2520imbalanced%2520dNTP%2520pools%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3D1360%26epage%3D1371%26doi%3D10.1093%2Fnar%2Fgkq829" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Ahmad, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaushal, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wan, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wijerathna, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">An, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dealwis, C. G.</span><span> </span><span class="NLM_article-title">Role of arginine 293 and glutamine 288 in communication between catalytic and allosteric sites in yeast ribonucleotide reductase</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">419</span><span class="NLM_x">, </span> <span class="NLM_fpage">315</span><span class="NLM_x">–</span> <span class="NLM_lpage">329</span><span class="refDoi"> DOI: 10.1016/j.jmb.2012.03.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1016%2Fj.jmb.2012.03.014" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=419&publication_year=2012&pages=315-329&author=M.+F.+Ahmadauthor=P.+S.+Kaushalauthor=Q.+Wanauthor=S.+R.+Wijerathnaauthor=X.+Anauthor=M.+Huangauthor=C.+G.+Dealwis&title=Role+of+arginine+293+and+glutamine+288+in+communication+between+catalytic+and+allosteric+sites+in+yeast+ribonucleotide+reductase&doi=10.1016%2Fj.jmb.2012.03.014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2012.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2012.03.014%26sid%3Dliteratum%253Aachs%26aulast%3DAhmad%26aufirst%3DM.%2BF.%26aulast%3DKaushal%26aufirst%3DP.%2BS.%26aulast%3DWan%26aufirst%3DQ.%26aulast%3DWijerathna%26aufirst%3DS.%2BR.%26aulast%3DAn%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DDealwis%26aufirst%3DC.%2BG.%26atitle%3DRole%2520of%2520arginine%2520293%2520and%2520glutamine%2520288%2520in%2520communication%2520between%2520catalytic%2520and%2520allosteric%2520sites%2520in%2520yeast%2520ribonucleotide%2520reductase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D419%26spage%3D315%26epage%3D329%26doi%3D10.1016%2Fj.jmb.2012.03.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Uhlin, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eklund, H.</span><span> </span><span class="NLM_article-title">Structure of ribonucleotide reductase protein R1</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">370</span><span class="NLM_x">, </span> <span class="NLM_fpage">533</span><span class="NLM_x">–</span> <span class="NLM_lpage">539</span><span class="refDoi"> DOI: 10.1038/370533a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1038%2F370533a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=8052308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADyaK2cXmt1CqsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=370&publication_year=1994&pages=533-539&author=U.+Uhlinauthor=H.+Eklund&title=Structure+of+ribonucleotide+reductase+protein+R1&doi=10.1038%2F370533a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of ribonucleotide reductase protein R1</span></div><div class="casAuthors">Uhlin, Ulla; Eklund, Hans</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">370</span>
        (<span class="NLM_cas:issue">6490</span>),
    <span class="NLM_cas:pages">533-9</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The crystal structure ribonucleotide reductase (I) subunit R1 (in complex with subunit R2) at 2.5 Å is reported.  The 3-dimensional structure of the R2 subunit was previously reported and refined at 2.2 Å.  The R2 tyrosyl radical-based I reaction involves 5 cysteines.  Two redox-active R1 cysteines (Cys-225 and Cys-462) are located at adjacent antiparallel strands in a new type of 10-stranded α/β-barrel, and 2 others (Cys-754 and Cys-759) at the C-terminal end in a flexible arm.  The 5th cysteine (Cys-439), in a loop in the center of the barrel, is positioned to initiate the radical reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW-kijq3QSZbVg90H21EOLACvtfcHk0ljby-7AUDxkdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmt1CqsLc%253D&md5=89f9cdf8318bf918d61ef63012f24073</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2F370533a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F370533a0%26sid%3Dliteratum%253Aachs%26aulast%3DUhlin%26aufirst%3DU.%26aulast%3DEklund%26aufirst%3DH.%26atitle%3DStructure%2520of%2520ribonucleotide%2520reductase%2520protein%2520R1%26jtitle%3DNature%26date%3D1994%26volume%3D370%26spage%3D533%26epage%3D539%26doi%3D10.1038%2F370533a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Kashlan, O. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lear, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooperman, B. S.</span><span> </span><span class="NLM_article-title">A comprehensive model for the allosteric regulation of mammalian ribonucleotide reductase. Functional consequences of ATP- and dATP-induced oligomerization of the large subunit</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">462</span><span class="NLM_x">–</span> <span class="NLM_lpage">474</span><span class="refDoi"> DOI: 10.1021/bi011653a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi011653a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD3MXovF2rtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=462-474&author=O.+B.+Kashlanauthor=C.+P.+Scottauthor=J.+D.+Learauthor=B.+S.+Cooperman&title=A+comprehensive+model+for+the+allosteric+regulation+of+mammalian+ribonucleotide+reductase.+Functional+consequences+of+ATP-+and+dATP-induced+oligomerization+of+the+large+subunit&doi=10.1021%2Fbi011653a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A Comprehensive Model for the Allosteric Regulation of Mammalian Ribonucleotide Reductase. Functional Consequences of ATP- and dATP-Induced Oligomerization of the Large Subunit</span></div><div class="casAuthors">Kashlan, Ossama B.; Scott, Charles P.; Lear, James D.; Cooperman, Barry S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-474</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Redn. of NDPs by murine ribonucleotide reductase (mRR) requires catalytic (mR1) and free radical-contg. (mR2) subunits and is regulated by nucleoside triphosphate allosteric effectors.  Here we present a new, comprehensive, and quant. model for allosteric control of mRR enzymic activity based on mol. mass, ligand binding, and enzyme activity studies.  In this model, nucleotide binding to the specificity site (s-site) drives formation of an active R12R22 dimer, ATP or dATP binding to the adenine-specific site (a-site) results in formation of an inactive tetramer, and ATP binding to the newly described hexamerization site (h-site) drives formation of active R16R26 hexamer.  In contrast, an earlier phenomenol. model [Thelander, L., and Reichard, P. (1979) Annu. Rev. Biochem. 67, 71-98] (the "RT" model) ignores aggregation state changes and mistakenly rationalizes ATP activation vs. dATP inhibition as reflecting different functional consequences of ATP vs. dATP binding to the a-site.  Our results suggest that the R16R26 heterohexamer is the major active form of the enzyme in mammalian cells, and that the ATP concn. is the primary modulator of enzyme activity, coupling the rate of DNA biosynthesis with the energetic state of the cell.  Using the crystal structure of the Escherichia coli R1 hexamer as a model for the mR1 hexamer, a scheme is presented that rationalizes the slow isomerization of the tetramer form and suggests an explanation for the low enzymic activity of tetramers complexed with R2.  The similar specific activities of R12R22 and R16R26 are inconsistent with a proposed model for R22 docking with R12 [Uhlin, U., and Eklund, H. (1994) Nature 370, 533-539], and an alternative is suggested.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT8Nd4PPq087Vg90H21EOLACvtfcHk0ljby-7AUDxkdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXovF2rtLg%253D&md5=f1cd48444fc897f38017b562d49fc1e5</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fbi011653a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi011653a%26sid%3Dliteratum%253Aachs%26aulast%3DKashlan%26aufirst%3DO.%2BB.%26aulast%3DScott%26aufirst%3DC.%2BP.%26aulast%3DLear%26aufirst%3DJ.%2BD.%26aulast%3DCooperman%26aufirst%3DB.%2BS.%26atitle%3DA%2520comprehensive%2520model%2520for%2520the%2520allosteric%2520regulation%2520of%2520mammalian%2520ribonucleotide%2520reductase.%2520Functional%2520consequences%2520of%2520ATP-%2520and%2520dATP-induced%2520oligomerization%2520of%2520the%2520large%2520subunit%26jtitle%3DBiochemistry%26date%3D2002%26volume%3D41%26spage%3D462%26epage%3D474%26doi%3D10.1021%2Fbi011653a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Cooperman, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kashlan, O. B.</span><span> </span><span class="NLM_article-title">A comprehensive model for the allosteric regulation of Class Ia ribonucleotide reductases</span> <span class="citation_source-journal">Adv. Enzyme Regul.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span><span class="refDoi"> DOI: 10.1016/S0065-2571(02)00035-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1016%2FS0065-2571%2802%2900035-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2003&pages=167-82&author=B.+S.+Coopermanauthor=O.+B.+Kashlan&title=A+comprehensive+model+for+the+allosteric+regulation+of+Class+Ia+ribonucleotide+reductases&doi=10.1016%2FS0065-2571%2802%2900035-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS0065-2571%2802%2900035-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0065-2571%252802%252900035-3%26sid%3Dliteratum%253Aachs%26aulast%3DCooperman%26aufirst%3DB.%2BS.%26aulast%3DKashlan%26aufirst%3DO.%2BB.%26atitle%3DA%2520comprehensive%2520model%2520for%2520the%2520allosteric%2520regulation%2520of%2520Class%2520Ia%2520ribonucleotide%2520reductases%26jtitle%3DAdv.%2520Enzyme%2520Regul.%26date%3D2003%26volume%3D43%26spage%3D167%26epage%3D82%26doi%3D10.1016%2FS0065-2571%2802%2900035-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Rofougaran, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vodnala, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hofer, A.</span><span> </span><span class="NLM_article-title">Enzymatically active mammalian ribonucleotide reductase exists primarily as an alpha6beta2 octamer</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">27705</span><span class="NLM_x">–</span> <span class="NLM_lpage">27711</span><span class="refDoi"> DOI: 10.1074/jbc.M605573200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1074%2Fjbc.M605573200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=27705-27711&author=R.+Rofougaranauthor=M.+Vodnalaauthor=A.+Hofer&title=Enzymatically+active+mammalian+ribonucleotide+reductase+exists+primarily+as+an+alpha6beta2+octamer&doi=10.1074%2Fjbc.M605573200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M605573200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M605573200%26sid%3Dliteratum%253Aachs%26aulast%3DRofougaran%26aufirst%3DR.%26aulast%3DVodnala%26aufirst%3DM.%26aulast%3DHofer%26aufirst%3DA.%26atitle%3DEnzymatically%2520active%2520mammalian%2520ribonucleotide%2520reductase%2520exists%2520primarily%2520as%2520an%2520alpha6beta2%2520octamer%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D27705%26epage%3D27711%26doi%3D10.1074%2Fjbc.M605573200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohman, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stubbe, J.</span><span> </span><span class="NLM_article-title">Enhanced subunit interactions with gemcitabine-5′-diphosphate inhibit ribonucleotide reductases</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">14324</span><span class="NLM_x">–</span> <span class="NLM_lpage">14329</span><span class="refDoi"> DOI: 10.1073/pnas.0706803104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1073%2Fpnas.0706803104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=17726094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVGhtbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=14324-14329&author=J.+Wangauthor=G.+J.+Lohmanauthor=J.+Stubbe&title=Enhanced+subunit+interactions+with+gemcitabine-5%E2%80%B2-diphosphate+inhibit+ribonucleotide+reductases&doi=10.1073%2Fpnas.0706803104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced subunit interactions with gemcitabine-5'-diphosphate inhibit ribonucleotide reductases</span></div><div class="casAuthors">Wang, Jun; Lohman, Gregory J. S.; Stubbe, JoAnne</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">14324-14329</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Ribonucleotide reductases (RNRs) catalyze the conversion of nucleotides to deoxynucleotides in all organisms.  The class I RNRs are composed of two subunits, α and β, with proposed quaternary structures of α2β2, α6β2, or α6β6, depending on the organism.  The t subunits bind the nucleoside diphosphate substrates and the dNTP/ATP allosteric effectors that govern specificity and turnover.  The β2 subunit houses the diferric Y. (1 radical per β2) cofactor that is required to initiate nucleotide redn. 2',2'-Difluoro-2'-deoxycytidine (F2C) is presently used clin. in a variety of cancer treatments and the 5'-diphosphorylated F2C (F2CDP) is a potent inhibitor of RNRs.  The studies with [1'-3H]-F2CDP and [5-3H]-F2CDP have established that F2CDP is a substoichiometric mechanism based inhibitor (0.5 equiv F2CDP/α) of both the Escherichia coli and the human RNRs in the presence of reductant.  Inactivation is caused by covalent labeling of RNR by the sugar of F2CDP (0.5 equiv/α) and is accompanied by release of 0.5 equiv cytosine/α.  Inactivation also results in loss of 40% of β2 activity.  Studies using size exclusion chromatog. reveal that in the E. coli RNR, an α2β2 tight complex is generated subsequent to enzyme inactivation by F2CDP, whereas in the human RNR, an α6β6 tight complex is generated.  Isolation of these complexes establishes that the weak interactions of the subunits in the absence of nucleotides are substantially increased in the presence of F2CDP and ATP.  This information and the proposed asymmetry between the interactions of αnβn provide an explanation for complete inactivation of RNR with substoichiometric amts. of F2CDP.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoABu-J8ZIg4bVg90H21EOLACvtfcHk0lg3XctvokCbcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVGhtbbP&md5=28970ac58ef9ada8c69ff4f88bb7704c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0706803104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0706803104%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLohman%26aufirst%3DG.%2BJ.%26aulast%3DStubbe%26aufirst%3DJ.%26atitle%3DEnhanced%2520subunit%2520interactions%2520with%2520gemcitabine-5%25E2%2580%25B2-diphosphate%2520inhibit%2520ribonucleotide%2520reductases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D14324%26epage%3D14329%26doi%3D10.1073%2Fpnas.0706803104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Fairman, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wijerathna, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmad, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakano, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jha, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prendergast, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Welin, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flodin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordlund, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walz, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dealwis, C. G.</span><span> </span><span class="NLM_article-title">Structural basis for allosteric regulation of human ribonucleotide reductase by nucleotide-induced oligomerization</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">316</span><span class="NLM_x">–</span> <span class="NLM_lpage">322</span><span class="refDoi"> DOI: 10.1038/nsmb.2007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1038%2Fnsmb.2007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=21336276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1aisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=316-322&author=J.+W.+Fairmanauthor=S.+R.+Wijerathnaauthor=M.+F.+Ahmadauthor=H.+Xuauthor=R.+Nakanoauthor=S.+Jhaauthor=J.+Prendergastauthor=R.+M.+Welinauthor=S.+Flodinauthor=A.+Roosauthor=P.+Nordlundauthor=Z.+Liauthor=T.+Walzauthor=C.+G.+Dealwis&title=Structural+basis+for+allosteric+regulation+of+human+ribonucleotide+reductase+by+nucleotide-induced+oligomerization&doi=10.1038%2Fnsmb.2007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for allosteric regulation of human ribonucleotide reductase by nucleotide-induced oligomerization</span></div><div class="casAuthors">Fairman, James Wesley; Wijerathna, Sanath Ranjan; Ahmad, Md Faiz; Xu, Hai; Nakano, Ryo; Jha, Shalini; Prendergast, Jay; Welin, R. Martin; Flodin, Susanne; Roos, Annette; Nordlund, Par; Li, Zongli; Walz, Thomas; Dealwis, Chris Godfrey</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">316-322</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Ribonucleotide reductase (RR) is an αnβn (RR1-RR2) complex that maintains balanced dNTP pools by reducing NDPs to dNDPs.  RR1 is the catalytic subunit, and RR2 houses the free radical required for catalysis.  RR is allosterically regulated by its activator ATP and its inhibitor dATP, which regulate RR activity by inducing oligomerization of RR1.  Here, we report the first X-ray structures of human RR1 bound to TTP alone, dATP alone, TTP-GDP, TTP-ATP, and TTP-dATP.  These structures provide insights into regulation of RR by ATP or dATP.  At physiol. dATP concns., RR1 forms inactive hexamers.  We detd. the first X-ray structure of the RR1-dATP hexamer and used single-particle electron microscopy to visualize the α6-ββ'-dATP holocomplex.  Site-directed mutagenesis and functional assays confirm that hexamerization is a prerequisite for inhibition by dATP.  Our data indicate a mechanism for regulating RR activity by dATP-induced oligomerization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeexjNT7fmrLVg90H21EOLACvtfcHk0lg3XctvokCbcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1aisbk%253D&md5=e7e97e869a51f1724f5ccc2d4eaea116</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2007%26sid%3Dliteratum%253Aachs%26aulast%3DFairman%26aufirst%3DJ.%2BW.%26aulast%3DWijerathna%26aufirst%3DS.%2BR.%26aulast%3DAhmad%26aufirst%3DM.%2BF.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DNakano%26aufirst%3DR.%26aulast%3DJha%26aufirst%3DS.%26aulast%3DPrendergast%26aufirst%3DJ.%26aulast%3DWelin%26aufirst%3DR.%2BM.%26aulast%3DFlodin%26aufirst%3DS.%26aulast%3DRoos%26aufirst%3DA.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DWalz%26aufirst%3DT.%26aulast%3DDealwis%26aufirst%3DC.%2BG.%26atitle%3DStructural%2520basis%2520for%2520allosteric%2520regulation%2520of%2520human%2520ribonucleotide%2520reductase%2520by%2520nucleotide-induced%2520oligomerization%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D316%26epage%3D322%26doi%3D10.1038%2Fnsmb.2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Aye, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stubbe, J.</span><span> </span><span class="NLM_article-title">Clofarabine 5′-di and -triphosphates inhibit human ribonucleotide reductase by altering the quaternary structure of its large subunit</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">9815</span><span class="NLM_x">–</span> <span class="NLM_lpage">9820</span><span class="refDoi"> DOI: 10.1073/pnas.1013274108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1073%2Fpnas.1013274108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=21628579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvVyjt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=9815-9820&author=Y.+Ayeauthor=J.+Stubbe&title=Clofarabine+5%E2%80%B2-di+and+-triphosphates+inhibit+human+ribonucleotide+reductase+by+altering+the+quaternary+structure+of+its+large+subunit&doi=10.1073%2Fpnas.1013274108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Clofarabine 5'-di and -triphosphates inhibit human ribonucleotide reductase by altering the quaternary structure of its large subunit</span></div><div class="casAuthors">Aye, Yimon; Stubbe, Joanne</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9815-9820, S9815/1-S9815/11</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Human ribonucleotide reductases (hRNRs) catalyze the conversion of nucleotides to deoxynucleotides and are composed of α and β-subunits that form active αnβm (n, m = 2 or 6) complexes.  α Binds NDP substrates (CDP, UDP, ADP, and GDP, C site) as well as ATP and dNTPs (dATP, dGTP, TTP) allosteric effectors that control enzyme activity (A site) and substrate specificity (S site).  Clofarabine (ClF), an adenosine analog, is used in the treatment of refractory leukemias.  Its mode of cytotoxicity is thought to be assocd. in part with the triphosphate functioning as an allosteric inhibitor of hRNR.  Studies on the mechanism of inhibition of hRNR by ClF di- and triphosphates (ClFDP and ClFTP) are presented.  ClFTP is a reversible inhibitor (Ki = 40 nM) that rapidly inactivates hRNR.  However, with time, 50% of the activity is recovered.  D57N-α, a mutant with an altered A site, prevents inhibition by ClFTP, suggesting its A site binding.  ClFDP is a slow-binding, reversible inhibitor (Ki = 17 nM; t1/2 = 23 min).  CDP protects α from its inhibition.  The altered off-rate of CIFDP from E·ClFDP* by ClFTP (A site) or dGTP (S site) and its inhibition of D57N-α together implicate its C site binding.  Size exclusion chromatog. of hRNR or α alone with ClFDP or ClFTP, ±ATP or dGTP, reveals in each case that α forms a kinetically stable hexameric state.  This is the first example of hexamerization of a induced by an NDP analog that reversibly binds at the active site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpI_2N_uNXV87Vg90H21EOLACvtfcHk0ljzgCRNuyeOyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvVyjt7o%253D&md5=1843df4bccc24864da4eb14c91bc84ff</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1013274108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1013274108%26sid%3Dliteratum%253Aachs%26aulast%3DAye%26aufirst%3DY.%26aulast%3DStubbe%26aufirst%3DJ.%26atitle%3DClofarabine%25205%25E2%2580%25B2-di%2520and%2520-triphosphates%2520inhibit%2520human%2520ribonucleotide%2520reductase%2520by%2520altering%2520the%2520quaternary%2520structure%2520of%2520its%2520large%2520subunit%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D9815%26epage%3D9820%26doi%3D10.1073%2Fpnas.1013274108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Aye, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brignole, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chittuluru, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drennan, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asturias, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stubbe, J.</span><span> </span><span class="NLM_article-title">Clofarabine targets the large subunit (alpha) of human ribonucleotide reductase in live cells by assembly into persistent hexamers</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">799</span><span class="NLM_x">–</span> <span class="NLM_lpage">805</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2012.05.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1016%2Fj.chembiol.2012.05.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=22840768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFahs7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=799-805&author=Y.+Ayeauthor=E.+J.+Brignoleauthor=M.+J.+Longauthor=J.+Chittuluruauthor=C.+L.+Drennanauthor=F.+J.+Asturiasauthor=J.+Stubbe&title=Clofarabine+targets+the+large+subunit+%28alpha%29+of+human+ribonucleotide+reductase+in+live+cells+by+assembly+into+persistent+hexamers&doi=10.1016%2Fj.chembiol.2012.05.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Clofarabine Targets the Large Subunit (α) of Human Ribonucleotide Reductase in Live Cells by Assembly into Persistent Hexamers</span></div><div class="casAuthors">Aye, Yimon; Brignole, Edward J.; Long, Marcus J. C.; Chittuluru, Johnathan; Drennan, Catherine L.; Asturias, Francisco J.; Stubbe, JoAnne</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">799-805</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Clofarabine (ClF) is a drug used in the treatment of leukemia.  One of its primary targets is human ribonucleotide reductase (hRNR), a dual-subunit, (α2)m(β2)n, regulatory enzyme indispensable in de novo dNTP synthesis.  We report that, in live mammalian cells, ClF targets hRNR by converting its α-subunit into kinetically stable hexamers.  We established mammalian expression platforms that enabled isolation of functional α and characterization of its altered oligomeric assocns. in response to ClF treatment.  Size exclusion chromatog. and electron microscopy documented persistence of in-cell-assembled-α6.  Our data validate hRNR as an important target of ClF, provide evidence that in vivo α's quaternary structure can be perturbed by a nonnatural ligand, and suggest small-mol.-promoted, persistent hexamerization as a strategy to modulate hRNR activity.  These studies lay foundations for documentation of RNR oligomeric state within a cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBYPDbbSuMoLVg90H21EOLACvtfcHk0ljzgCRNuyeOyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFahs7zF&md5=efa9aa560ce94086fea32214bb1f350a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.05.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.05.015%26sid%3Dliteratum%253Aachs%26aulast%3DAye%26aufirst%3DY.%26aulast%3DBrignole%26aufirst%3DE.%2BJ.%26aulast%3DLong%26aufirst%3DM.%2BJ.%26aulast%3DChittuluru%26aufirst%3DJ.%26aulast%3DDrennan%26aufirst%3DC.%2BL.%26aulast%3DAsturias%26aufirst%3DF.%2BJ.%26aulast%3DStubbe%26aufirst%3DJ.%26atitle%3DClofarabine%2520targets%2520the%2520large%2520subunit%2520%2528alpha%2529%2520of%2520human%2520ribonucleotide%2520reductase%2520in%2520live%2520cells%2520by%2520assembly%2520into%2520persistent%2520hexamers%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D799%26epage%3D805%26doi%3D10.1016%2Fj.chembiol.2012.05.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Kashlan, O. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooperman, B. S.</span><span> </span><span class="NLM_article-title">Comprehensive model for allosteric regulation of mammalian ribonucleotide reductase: refinements and consequences</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">1696</span><span class="NLM_x">–</span> <span class="NLM_lpage">1706</span><span class="refDoi"> DOI: 10.1021/bi020634d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi020634d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlsFaksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=1696-1706&author=O.+B.+Kashlanauthor=B.+S.+Cooperman&title=Comprehensive+model+for+allosteric+regulation+of+mammalian+ribonucleotide+reductase%3A+refinements+and+consequences&doi=10.1021%2Fbi020634d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive Model for Allosteric Regulation of Mammalian Ribonucleotide Reductase: Refinements and Consequences</span></div><div class="casAuthors">Kashlan, Ossama B.; Cooperman, Barry S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1696-1706</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Redn. of NDPs by murine ribonucleotide reductase (mRR) requires catalytic (mR1) and free radical-contg. (mR2) subunits and is regulated by nucleoside triphosphate allosteric effectors.  Here we present the results of several studies that refine the recently presented comprehensive model for the allosteric control of mRR enzymic activity [Kashlan, O. B., et al. (2002) Biochem. 41, 462-474], in which nucleotide binding to the specificity site (s-site) drives formation of an active R12R22 dimer, ATP or dATP binding to the adenine site (a-site) drives formation of a tetramer, mR14a, which isomerizes to an inactive form, mR14b, and ATP binding to the hexamerization site (h-site) drives formation of an active R16R26 hexamer.  Anal. of the a-site D57N variant of mR1, which differs from wild-type mR1 (wt.-mR1) in that its RR activity is activated by both ATP and dATP, demonstrates that dATP activation of the D57N variant RR arises from a blockage in the formation of mR14b from mR14a, and provides strong evidence that mR14a forms active complexes with mR22.  We further demonstrate that (a) differences in the effects of ATP vs. dATP binding to the a-site of wt.-mR1 provide specific mechanisms by which the dATP/ATP ratio in mammalian cells could modulate in vivo RR enzymic activity, (b) the comprehensive model is valid over a range of Mg2+ concns. that include in vivo concns., and (c) equil. consts. derived for the comprehensive model can be used to simulate the distribution of R1 among dimer, tetramer, and hexamer forms in vivo.  Such simulations indicate that mR16 predominates over mR12 in the cytoplasm of normal mammalian cells, where the great majority of RR activity is located, but that mR12 may be important for nuclear RR activity and for RR activity in cells in which the level of ATP is depleted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLAXHh__c4ErVg90H21EOLACvtfcHk0ljzgCRNuyeOyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlsFaksw%253D%253D&md5=97e20a2b3a066c99e735736c0b1c47e6</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fbi020634d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi020634d%26sid%3Dliteratum%253Aachs%26aulast%3DKashlan%26aufirst%3DO.%2BB.%26aulast%3DCooperman%26aufirst%3DB.%2BS.%26atitle%3DComprehensive%2520model%2520for%2520allosteric%2520regulation%2520of%2520mammalian%2520ribonucleotide%2520reductase%253A%2520refinements%2520and%2520consequences%26jtitle%3DBiochemistry%26date%3D2003%26volume%3D42%26spage%3D1696%26epage%3D1706%26doi%3D10.1021%2Fbi020634d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Arnaoutov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dasso, M.</span><span> </span><span class="NLM_article-title">IRBIT is a novel regulator of ribonucleotide reductase in higher eukaryotes</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">345</span><span class="NLM_x">, </span> <span class="NLM_fpage">1512</span><span class="NLM_x">–</span> <span class="NLM_lpage">1515</span><span class="refDoi"> DOI: 10.1126/science.1251550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1126%2Fscience.1251550" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=345&publication_year=2014&pages=1512-1515&author=A.+Arnaoutovauthor=M.+Dasso&title=IRBIT+is+a+novel+regulator+of+ribonucleotide+reductase+in+higher+eukaryotes&doi=10.1126%2Fscience.1251550"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1126%2Fscience.1251550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1251550%26sid%3Dliteratum%253Aachs%26aulast%3DArnaoutov%26aufirst%3DA.%26aulast%3DDasso%26aufirst%3DM.%26atitle%3DIRBIT%2520is%2520a%2520novel%2520regulator%2520of%2520ribonucleotide%2520reductase%2520in%2520higher%2520eukaryotes%26jtitle%3DScience%26date%3D2014%26volume%3D345%26spage%3D1512%26epage%3D1515%26doi%3D10.1126%2Fscience.1251550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Ahmad, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wan, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jha, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motea, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berdis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dealwis, C.</span><span> </span><span class="NLM_article-title">Evaluating the therapeutic potential of a non-natural nucleotide that inhibits human ribonucleotide reductase</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">2077</span><span class="NLM_x">–</span> <span class="NLM_lpage">2086</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-12-0199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1158%2F1535-7163.MCT-12-0199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=22933704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVyksr%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=2077-2086&author=M.+F.+Ahmadauthor=Q.+Wanauthor=S.+Jhaauthor=E.+Moteaauthor=A.+Berdisauthor=C.+Dealwis&title=Evaluating+the+therapeutic+potential+of+a+non-natural+nucleotide+that+inhibits+human+ribonucleotide+reductase&doi=10.1158%2F1535-7163.MCT-12-0199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluating the Therapeutic Potential of a Non-Natural Nucleotide That Inhibits Human Ribonucleotide Reductase</span></div><div class="casAuthors">Ahmad, Md. Faiz; Wan, Qun; Jha, Shalini; Motea, Edward; Berdis, Anthony; Dealwis, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2077-2086</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Human ribonucleotide reductase (hRR) is the key enzyme involved in de novo dNTP synthesis and thus represents an important therapeutic target against hyperproliferative diseases, most notably cancer.  The purpose of this study was to evaluate the ability of non-natural indolyl-2'-deoxynucleoside triphosphates to inhibit the activity of hRR.  The structural similarities of these analogs with dATP predicted that they would inhibit hRR activity by binding to its allosteric sites.  In silico anal. and in vitro characterization identified one particular analog designated as 5-nitro-indolyl-2'-deoxyribose triphosphate (5-NITP) that inhibits hRR. 5-NITP binding to hRR was detd. by isothermal titrn. calorimetry.  X-ray crystal structure of 5-NITP bound to RR1 was detd.  Cell-based studies showed the anti-cancer effects of the corresponding non-natural nucleoside against leukemia cells. 5-NITP binds to hRR with micromolar affinity.  Binding does not induce hexamerization of hRR1 like dATP, the native allosteric inhibitor of hRR that binds with high affinity to the A-site.  The X-ray crystal structure of Saccharomyces cerevisiae RR1-5-NITP (ScRR1-5-NITP) complex detd. to 2.3 Å resoln. shows that 5-NITP does not bind to the A-site but rather at the S-site.  Regardless, 5-nitro-indolyl-2'-deoxynucleoside (5-NIdR) produces cytostatic and cytotoxic effects against human leukemia cells by altering cell-cycle progression.  Our studies provide useful insights toward developing new inhibitors with improved potency and efficacy against hRR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq13zZRjCX4ALVg90H21EOLACvtfcHk0li-XrurlLjKKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVyksr%252FM&md5=79ad691a31ad0710598684e14bcece3a</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-12-0199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-12-0199%26sid%3Dliteratum%253Aachs%26aulast%3DAhmad%26aufirst%3DM.%2BF.%26aulast%3DWan%26aufirst%3DQ.%26aulast%3DJha%26aufirst%3DS.%26aulast%3DMotea%26aufirst%3DE.%26aulast%3DBerdis%26aufirst%3DA.%26aulast%3DDealwis%26aufirst%3DC.%26atitle%3DEvaluating%2520the%2520therapeutic%2520potential%2520of%2520a%2520non-natural%2520nucleotide%2520that%2520inhibits%2520human%2520ribonucleotide%2520reductase%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2012%26volume%3D11%26spage%3D2077%26epage%3D2086%26doi%3D10.1158%2F1535-7163.MCT-12-0199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Jaffe, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stith, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawrence, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrake, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunbrack, R. L.</span><span> </span><span class="NLM_article-title">A new model for allosteric regulation of phenylalanine hydroxylase: implications for disease and therapeutics</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">530</span><span class="NLM_x">, </span> <span class="NLM_fpage">73</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span><span class="refDoi"> DOI: 10.1016/j.abb.2012.12.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1016%2Fj.abb.2012.12.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=23296088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3sXis1Orsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=530&publication_year=2013&pages=73-82&author=E.+K.+Jaffeauthor=L.+Stithauthor=S.+H.+Lawrenceauthor=M.+Andrakeauthor=R.+L.+Dunbrack&title=A+new+model+for+allosteric+regulation+of+phenylalanine+hydroxylase%3A+implications+for+disease+and+therapeutics&doi=10.1016%2Fj.abb.2012.12.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A new model for allosteric regulation of phenylalanine hydroxylase: Implications for disease and therapeutics</span></div><div class="casAuthors">Jaffe, Eileen K.; Stith, Linda; Lawrence, Sarah H.; Andrake, Mark; Dunbrack, Roland L., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">530</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">73-82</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The structural basis for allosteric regulation of phenylalanine hydroxylase (PAH), whose dysfunction causes phenylketonuria (PKU), is poorly understood.  A new morpheein model for PAH allostery is proposed to consist of a dissociative equil. between two architecturally different tetramers whose interconversion requires a ∼90° rotation between the PAH catalytic and regulatory domains, the latter of which contains an ACT domain.  This unprecedented model is supported by in vitro data on purified full length rat and human PAH.  The conformational change is both predicted to and shown to render the tetramers chromatog. separable using ion exchange methods.  One novel aspect of the activated tetramer model is an allosteric phenylalanine binding site at the intersubunit interface of ACT domains.  Amino acid ligand-stabilized ACT domain dimerization follows the multimerization and ligand binding behavior of ACT domains present in other proteins in the PDB.  Spectroscopic, chromatog., and electrophoretic methods demonstrate a PAH equil. consisting of two architecturally distinct tetramers as well as dimers.  We postulate that PKU-assocd. mutations may shift the PAH quaternary structure equil. in favor of the low activity assemblies.  Pharmacol. chaperones that stabilize the ACT:ACT interface can potentially provide PKU patients with a novel small mol. therapeutic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjgPGUSaTRErVg90H21EOLACvtfcHk0li-XrurlLjKKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXis1Orsrk%253D&md5=15a7753b982effcbff2b40f23db80b39</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2012.12.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2012.12.017%26sid%3Dliteratum%253Aachs%26aulast%3DJaffe%26aufirst%3DE.%2BK.%26aulast%3DStith%26aufirst%3DL.%26aulast%3DLawrence%26aufirst%3DS.%2BH.%26aulast%3DAndrake%26aufirst%3DM.%26aulast%3DDunbrack%26aufirst%3DR.%2BL.%26atitle%3DA%2520new%2520model%2520for%2520allosteric%2520regulation%2520of%2520phenylalanine%2520hydroxylase%253A%2520implications%2520for%2520disease%2520and%2520therapeutics%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2013%26volume%3D530%26spage%3D73%26epage%3D82%26doi%3D10.1016%2Fj.abb.2012.12.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Jaffe, E. K.</span><span> </span><span class="NLM_article-title">Impact of quaternary structure dynamics on allosteric drug discovery</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">55</span><span class="NLM_x">–</span> <span class="NLM_lpage">63</span><span class="refDoi"> DOI: 10.2174/1568026611313010006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.2174%2F1568026611313010006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=23409765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtleqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=55-63&author=E.+K.+Jaffe&title=Impact+of+quaternary+structure+dynamics+on+allosteric+drug+discovery&doi=10.2174%2F1568026611313010006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of quaternary structure dynamics on allosteric drug discovery</span></div><div class="casAuthors">Jaffe, Eileen K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-63</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The morpheein model of allosteric regulation draws attention to proteins that can exist as an equil. of functionally distinct assemblies where: one subunit conformation assembles into one multimer; a different subunit conformation assembles into a different multimer; and the various multimers are in a dynamic equil. whose position can be modulated by ligands that bind to a multimer-specific ligand binding site.  The case study of porphobilinogen synthase (PBGS) illustrates how such an equil. holds lessons for disease mechanisms, drug discovery, understanding drug side effects, and identifying proteins wherein drug discovery efforts might focus on quaternary structure dynamics.  The morpheein model of allostery has been proposed as applicable for a wide assortment of disease-assocd. proteins.  Herein we discuss quaternary structure dynamics aspects to drug discovery for the disease-assocd. putative morpheeins phenylalanine hydroxylase, HIV integrase, pyruvate kinase, and tumor necrosis factor α.  Also highlighted is the quaternary structure equil. of transthyretin and successful drug discovery efforts focused on controlling its quaternary structure dynamics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_MI6syFNLs7Vg90H21EOLACvtfcHk0li-XrurlLjKKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtleqtbk%253D&md5=e53431517b1eca6ab29617d46e7b3cd1</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.2174%2F1568026611313010006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026611313010006%26sid%3Dliteratum%253Aachs%26aulast%3DJaffe%26aufirst%3DE.%2BK.%26atitle%3DImpact%2520of%2520quaternary%2520structure%2520dynamics%2520on%2520allosteric%2520drug%2520discovery%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2013%26volume%3D13%26spage%3D55%26epage%3D63%26doi%3D10.2174%2F1568026611313010006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Jaffe, E. K.</span><span> </span><span class="NLM_article-title">Morpheeins - A new pathway for allosteric drug discovery</span> <span class="citation_source-journal">Open Conf. Proc. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.2174/22102892010010100001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.2174%2F22102892010010100001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=21643557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlKqur7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=1-6&author=E.+K.+Jaffe&title=Morpheeins+-+A+new+pathway+for+allosteric+drug+discovery&doi=10.2174%2F22102892010010100001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Morpheeins - a new pathway for allosteric drug discovery</span></div><div class="casAuthors">Jaffe, Eileen K.</div><div class="citationInfo"><span class="NLM_cas:title">Open Conference Proceedings Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-6</span>CODEN:
                <span class="NLM_cas:coden">OCPJC2</span>;
        ISSN:<span class="NLM_cas:issn">2210-2892</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The morpheein model of allosteric regulation can be applied as a novel approach to the discovery of small mol. allosteric modulators of protein function.  Morpheeins are homo-oligomeric proteins where, under physiol. conditions, the oligomer can dissoc., the dissocd. units can change conformation, and the altered conformational state can reassoc. to a structurally and functionally distinct oligomer.  This phenomenon serves as a basis for allostery, as a basis for conformational diseases, as a basis for drug discovery, and may be applicable to personalized medicine such as in the prediction of drug side effects.  Each of these relationships has been established for the prototype morpheein, porphobilinogen synthase, where the conformational disease is a porphyria and the drug application is in antimicrobial discovery.  These data are presented along with a discussion of other drug targets for which the morpheein model of allostery may apply.  Such targets include HIV integrase, TNFα, β-tryptase, and p53.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHwzTd8syeArVg90H21EOLACvtfcHk0lgM8St-7J9O4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlKqur7F&md5=bc56e399c1257be36aea71127427f920</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.2174%2F22102892010010100001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F22102892010010100001%26sid%3Dliteratum%253Aachs%26aulast%3DJaffe%26aufirst%3DE.%2BK.%26atitle%3DMorpheeins%2520-%2520A%2520new%2520pathway%2520for%2520allosteric%2520drug%2520discovery%26jtitle%3DOpen%2520Conf.%2520Proc.%2520J.%26date%3D2010%26volume%3D1%26spage%3D1%26epage%3D6%26doi%3D10.2174%2F22102892010010100001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G. A.</span><span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2719</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span><span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lgM8St-7J9O4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Wan, K. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, V. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Entzeroth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. A.</span><span> </span><span class="NLM_article-title">Differential scanning fluorimetry as secondary screening platform for small molecule inhibitors of Bcl-XL</span> <span class="citation_source-journal">Cell Cycle</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">3943</span><span class="NLM_x">–</span> <span class="NLM_lpage">3952</span><span class="refDoi"> DOI: 10.4161/cc.8.23.10114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.4161%2Fcc.8.23.10114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=19901550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotlWrtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=3943-3952&author=K.+F.+Wanauthor=S.+Wangauthor=C.+J.+Brownauthor=V.+C.+Yuauthor=M.+Entzerothauthor=D.+P.+Laneauthor=M.+A.+Lee&title=Differential+scanning+fluorimetry+as+secondary+screening+platform+for+small+molecule+inhibitors+of+Bcl-XL&doi=10.4161%2Fcc.8.23.10114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Differential scanning fluorimetry as secondary screening platform for small molecule inhibitors of Bcl-XL</span></div><div class="casAuthors">Wan, Kah Fei; Wang, Sifang; Brown, Christopher J.; Yu, Victor C.; Entzeroth, Michael; Lane, David P.; Lee, May Ann</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3943-3952</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Since apoptosis is impaired in malignant cells overexpressing prosurvival Bcl-2 proteins, drugs mimicking their natural antagonists, BH3-only proteins, might overcome chemoresistance.  Small mol. inhibitors of Bcl-XL function have been discovered from diverse structure classes using rational drug design as well as high-throughput screening (HTS) approaches.  However, most of the BH3 mimetics that have been identified via screening based on fluorescence polarization displayed an affinity for their presumed protein targets that is far lower than that of BH3-only proteins.  Therefore, it is important to establish a simple and inexpensive secondary platform for hit validation which is pertinent to current efforts for developing compds. that mimic the action of BH3-only proteins as novel anticancer agents.  These considerations prompted the authors to explore the differential scanning fluorometry (DSF) method that is based on energetic coupling between ligand binding and protein unfolding.  The authors have systematically tested known Bcl-XL/Bcl-2 inhibitors using DSF and have revealed distinct subsets of inhibitors.  More importantly, the authors report that some of these inhibitors interacted selectively with glutathione S-transferase tagged Bcl-XL, whereas certain inhibitors exhibited marked selectivity towards native untagged Bcl-XL.  Therefore, the authors propose that the affinity tag may cause a significant conformational switch in the Bcl-XL, which results in the selectivity for certain subsets of small mol. inhibitors.  This finding also implies that the previous screens involving tagged proteins need to be carefully reexamd. while further investigations must ensure that the right conformation of protein is used in future screens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGHLlHxk1TSbVg90H21EOLACvtfcHk0lgM8St-7J9O4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotlWrtbo%253D&md5=3c7bb4fcfcdd53c393b7e3e9088fb847</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.4161%2Fcc.8.23.10114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.8.23.10114%26sid%3Dliteratum%253Aachs%26aulast%3DWan%26aufirst%3DK.%2BF.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DBrown%26aufirst%3DC.%2BJ.%26aulast%3DYu%26aufirst%3DV.%2BC.%26aulast%3DEntzeroth%26aufirst%3DM.%26aulast%3DLane%26aufirst%3DD.%2BP.%26aulast%3DLee%26aufirst%3DM.%2BA.%26atitle%3DDifferential%2520scanning%2520fluorimetry%2520as%2520secondary%2520screening%2520platform%2520for%2520small%2520molecule%2520inhibitors%2520of%2520Bcl-XL%26jtitle%3DCell%2520Cycle%26date%3D2009%26volume%3D8%26spage%3D3943%26epage%3D3952%26doi%3D10.4161%2Fcc.8.23.10114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Shrake, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rupley, J. A.</span><span> </span><span class="NLM_article-title">Environment and exposure to solvent of protein atoms. Lysozyme and insulin</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">351</span><span class="NLM_x">–</span> <span class="NLM_lpage">371</span><span class="refDoi"> DOI: 10.1016/0022-2836(73)90011-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1016%2F0022-2836%2873%2990011-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=4760134" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADyaE3sXlsVelt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1973&pages=351-371&author=A.+Shrakeauthor=J.+A.+Rupley&title=Environment+and+exposure+to+solvent+of+protein+atoms.+Lysozyme+and+insulin&doi=10.1016%2F0022-2836%2873%2990011-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Environment and exposure to solvent of protein atoms.  Lysozyme and insulin</span></div><div class="casAuthors">Shrake, A.; Rupley, J. A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">351-71</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">A computer program is described for calcg. the environment and the exposure to solvent of atoms of a protein.  The computation is based on the at. coordinates of the protein and on assumptions like those of B. Lee and F. M. Richards (1971).  Results for lysozyme and insulin are presented.  Changes in exposure to solvent and in the nature of contacts that develop through folding, assocn. reactions, and crystn. are described numerically.  The computations suggest several generalizations.  (a) Lattice contacts within the protein crystal are characterized by a significantly smaller involvement of nonpolar side chains and a proportionately greater involvement of ionizable side chains than is found for protein folding or for protein assocn. reactions important for biol. function.  (b) In helical regions the carbonyl O of the 1st residue in the helix has high probability of being shielded from solvent.  (c) Glycine is among the residues having exposure least affected by folding; this accords with the expectation that it lies at bends of the peptide chain on the surface of the mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoU7V-RfwTrV7Vg90H21EOLACvtfcHk0lgFidcrWgIzqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3sXlsVelt7Y%253D&md5=de79172cb044d2c7545702f6503a891a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2873%2990011-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252873%252990011-9%26sid%3Dliteratum%253Aachs%26aulast%3DShrake%26aufirst%3DA.%26aulast%3DRupley%26aufirst%3DJ.%2BA.%26atitle%3DEnvironment%2520and%2520exposure%2520to%2520solvent%2520of%2520protein%2520atoms.%2520Lysozyme%2520and%2520insulin%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1973%26volume%3D79%26spage%3D351%26epage%3D371%26doi%3D10.1016%2F0022-2836%2873%2990011-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Stearns, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Losee, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernstein, J.</span><span> </span><span class="NLM_article-title">Hydroxyurea. A new type of potential antitumor agent</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1963</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">201</span><span class="refDoi"> DOI: 10.1021/jm00338a026</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00338a026" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1963&pages=201&author=B.+Stearnsauthor=K.+A.+Loseeauthor=J.+Bernstein&title=Hydroxyurea.+A+new+type+of+potential+antitumor+agent&doi=10.1021%2Fjm00338a026"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm00338a026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00338a026%26sid%3Dliteratum%253Aachs%26aulast%3DStearns%26aufirst%3DB.%26aulast%3DLosee%26aufirst%3DK.%2BA.%26aulast%3DBernstein%26aufirst%3DJ.%26atitle%3DHydroxyurea.%2520A%2520new%2520type%2520of%2520potential%2520antitumor%2520agent%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1963%26volume%3D6%26spage%3D201%26doi%3D10.1021%2Fjm00338a026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Donehower, R. C.</span><span> </span><span class="NLM_article-title">An overview of the clinical experience with hydroxyurea</span> <span class="citation_source-journal">Semin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">11</span><span class="NLM_x">–</span> <span class="NLM_lpage">19</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1641651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A280%3ADyaK38zltFWitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1992&pages=11-19&author=R.+C.+Donehower&title=An+overview+of+the+clinical+experience+with+hydroxyurea"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">An overview of the clinical experience with hydroxyurea</span></div><div class="casAuthors">Donehower R C</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in oncology</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3 Suppl 9</span>),
    <span class="NLM_cas:pages">11-9</span>
        ISSN:<span class="NLM_cas:issn">0093-7754</span>.
    </div><div class="casAbstract">In the more than 30 years since hydroxyurea was first evaluated clinically, a number of diverse and unique applications have been identified for its use.  Some of these applications were apparent to investigators almost from the beginning, while others are still evolving 3 decades later as a result of ongoing research.  An overview of clinical experience with hydroxyurea provides an historical perspective on a period encompassing virtually the entire era in which anticancer chemotherapy has been a recognized discipline.  The critical role of the ribonucleotide reductase reaction in the regulation of DNA synthesis makes this enzyme a particularly important target site for antineoplastic therapy.  The sustained interest in hydroxyurea, however, has been the result of careful clinical observations as well as basic laboratory studies, which support current attempts to identify additional uses for the drug.  This overview describes the clinical development of hydroxyurea, discusses current indications, and briefly describes areas of ongoing research that may expand the clinical indications for hydroxyurea.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTa1cLEoNUZPuqrn-c4Of3KfW6udTcc2eaJT3tSP0V4gbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK38zltFWitA%253D%253D&md5=944c23293942a08a460e1e638a29008d</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDonehower%26aufirst%3DR.%2BC.%26atitle%3DAn%2520overview%2520of%2520the%2520clinical%2520experience%2520with%2520hydroxyurea%26jtitle%3DSemin.%2520Oncol.%26date%3D1992%26volume%3D19%26spage%3D11%26epage%3D19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Winter, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higueruelo, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sigurdardottir, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span> </span><span class="NLM_article-title">Biophysical and computational fragment-based approaches to targeting protein-protein interactions: applications in structure-guided drug discovery</span> <span class="citation_source-journal">Q. Rev. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">383</span><span class="NLM_x">–</span> <span class="NLM_lpage">426</span><span class="refDoi"> DOI: 10.1017/S0033583512000108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1017%2FS0033583512000108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=22971516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslygtL7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2012&pages=383-426&author=A.+Winterauthor=A.+P.+Higuerueloauthor=M.+Marshauthor=A.+Sigurdardottirauthor=W.+R.+Pittauthor=T.+L.+Blundell&title=Biophysical+and+computational+fragment-based+approaches+to+targeting+protein-protein+interactions%3A+applications+in+structure-guided+drug+discovery&doi=10.1017%2FS0033583512000108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Biophysical and computational fragment-based approaches to targeting protein-protein interactions: applications in structure-guided drug discovery</span></div><div class="casAuthors">Winter, Anja; Higueruelo, Alicia P.; Marsh, May; Sigurdardottir, Anna; Pitt, Will R.; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Quarterly Reviews of Biophysics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">383-426</span>CODEN:
                <span class="NLM_cas:coden">QURBAW</span>;
        ISSN:<span class="NLM_cas:issn">0033-5835</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  Drug discovery has classically targeted the active sites of enzymes or ligand-binding sites of receptors and ion channels.  In an attempt to improve selectivity of drug candidates, modulation of protein-protein interfaces (PPIs) of multiprotein complexes that mediate conformation or colocation of components of cell-regulatory pathways has become a focus of interest.  However, PPIs in multiprotein systems continue to pose significant challenges, as they are generally large, flat and poor in distinguishing features, making the design of small mol. antagonists a difficult task.  Nevertheless, encouragement has come from the recognition that a few amino acids - so-called hotspots - may contribute the majority of interaction-free energy.  The challenges posed by protein-protein interactions have led to a wellspring of creative approaches, including proteomimetics, stapled α-helical peptides and a plethora of antibody inspired mol. designs.  Here, we review a more generic approach: fragment-based drug discovery.  Fragments allow novel areas of chem. space to be explored more efficiently, but the initial hits have low affinity.  This means that they will not normally disrupt PPIs, unless they are tethered, an approach that has been pioneered by Wells and co-workers.  An alternative fragment-based approach is to stabilize the uncomplexed components of the multiprotein system in soln. and employ conventional fragment-based screening.  Here, we describe the current knowledge of the structures and properties of protein-protein interactions and the small mols. that can modulate them.  We then describe the use of sensitive biophys. methods - NMR, X-ray crystallog., surface plasmon resonance, differential scanning fluorimetry or isothermal calorimetry - to screen and validate fragment binding.  Fragment hits can subsequently be evolved into larger mols. with higher affinity and potency.  These may provide new leads for drug candidates that target protein-protein interactions and have therapeutic value.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoninFnW9vZ5LVg90H21EOLACvtfcHk0lgFidcrWgIzqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslygtL7K&md5=c3ad9e944414331e1d07509d37583b7a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1017%2FS0033583512000108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0033583512000108%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DA.%26aulast%3DHigueruelo%26aufirst%3DA.%2BP.%26aulast%3DMarsh%26aufirst%3DM.%26aulast%3DSigurdardottir%26aufirst%3DA.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DBiophysical%2520and%2520computational%2520fragment-based%2520approaches%2520to%2520targeting%2520protein-protein%2520interactions%253A%2520applications%2520in%2520structure-guided%2520drug%2520discovery%26jtitle%3DQ.%2520Rev.%2520Biophys.%26date%3D2012%26volume%3D45%26spage%3D383%26epage%3D426%26doi%3D10.1017%2FS0033583512000108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Wells, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McClendon, C. L.</span><span> </span><span class="NLM_article-title">Reaching for high-hanging fruit in drug discovery at protein-protein interfaces</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">450</span><span class="NLM_x">, </span> <span class="NLM_fpage">1001</span><span class="NLM_x">–</span> <span class="NLM_lpage">1009</span><span class="refDoi"> DOI: 10.1038/nature06526</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1038%2Fnature06526" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=18075579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2007&pages=1001-1009&author=J.+A.+Wellsauthor=C.+L.+McClendon&title=Reaching+for+high-hanging+fruit+in+drug+discovery+at+protein-protein+interfaces&doi=10.1038%2Fnature06526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Reaching for high-hanging fruit in drug discovery at protein-protein interfaces</span></div><div class="casAuthors">Wells, James A.; McClendon, Christopher L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">450</span>
        (<span class="NLM_cas:issue">7172</span>),
    <span class="NLM_cas:pages">1001-1009</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Targeting the interfaces between proteins has huge therapeutic potential, but discovering small-mol. drugs that disrupt protein-protein interactions is an enormous challenge.  Several recent success stories, however, indicate that protein-protein interfaces might be more tractable than has been thought.  These studies discovered small mols. that bind with drug-like potencies to 'hotspots' on the contact surfaces involved in protein-protein interactions.  Remarkably, these small mols. bind deeper within the contact surface of the target protein, and bind with much higher efficiencies, than do the contact atoms of the natural protein partner.  Some of these small mols. are now making their way through clin. trials, so this high-hanging fruit might not be far out of reach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-nGR3BqqkrVg90H21EOLACvtfcHk0lh262CUGefDyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F&md5=e7ed69fd8c362a71c4b99f029c16a6fc</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnature06526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06526%26sid%3Dliteratum%253Aachs%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DMcClendon%26aufirst%3DC.%2BL.%26atitle%3DReaching%2520for%2520high-hanging%2520fruit%2520in%2520drug%2520discovery%2520at%2520protein-protein%2520interfaces%26jtitle%3DNature%26date%3D2007%26volume%3D450%26spage%3D1001%26epage%3D1009%26doi%3D10.1038%2Fnature06526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span> </span><span class="NLM_article-title">The druggable genome</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">730</span><span class="refDoi"> DOI: 10.1038/nrd892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1038%2Fnrd892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=12209152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=727-730&author=A.+L.+Hopkinsauthor=C.+R.+Groom&title=The+druggable+genome&doi=10.1038%2Fnrd892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: The druggable genome</span></div><div class="casAuthors">Hopkins, Andrew L.; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">727-730</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  An assessment of the no. of mol. targets that represent an opportunity for therapeutic intervention is crucial to the development of post-genomic research strategies within the pharmaceutical industry.  Now that we know the size of the human genome, it is interesting to consider just how many mol. targets this opportunity represents.  We start from the position that we understand the properties that are required for a good drug, and therefore must be able to understand what makes a good drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Uvqi8O9wzrVg90H21EOLACvtfcHk0lh262CUGefDyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D&md5=d6dc7c67822661fe03e920695def1329</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnrd892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd892%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520druggable%2520genome%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D727%26epage%3D730%26doi%3D10.1038%2Fnrd892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Mills, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jasensky, R. D.</span><span> </span><span class="NLM_article-title">Antimicrobial effects of N alpha-palmitoyl-L-lysyl-L-lysine ethyl ester dihydrochloride and its use to extend the shelf life of creamed cottage cheese</span> <span class="citation_source-journal">J. Agric. Food Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1980</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">812</span><span class="NLM_x">–</span> <span class="NLM_lpage">817</span><span class="refDoi"> DOI: 10.1021/jf60230a035</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jf60230a035" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1980&pages=812-817&author=C.+J.+Millsauthor=T.+Richardsonauthor=R.+D.+Jasensky&title=Antimicrobial+effects+of+N+alpha-palmitoyl-L-lysyl-L-lysine+ethyl+ester+dihydrochloride+and+its+use+to+extend+the+shelf+life+of+creamed+cottage+cheese&doi=10.1021%2Fjf60230a035"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjf60230a035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjf60230a035%26sid%3Dliteratum%253Aachs%26aulast%3DMills%26aufirst%3DC.%2BJ.%26aulast%3DRichardson%26aufirst%3DT.%26aulast%3DJasensky%26aufirst%3DR.%2BD.%26atitle%3DAntimicrobial%2520effects%2520of%2520N%2520alpha-palmitoyl-L-lysyl-L-lysine%2520ethyl%2520ester%2520dihydrochloride%2520and%2520its%2520use%2520to%2520extend%2520the%2520shelf%2520life%2520of%2520creamed%2520cottage%2520cheese%26jtitle%3DJ.%2520Agric.%2520Food%2520Chem.%26date%3D1980%26volume%3D28%26spage%3D812%26epage%3D817%26doi%3D10.1021%2Fjf60230a035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Cegielska, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, A.</span><span> </span><span class="NLM_article-title">The sfiA11 mutation prevents filamentation in a response to cell wall damage only in a recA+ genetic background</span> <span class="citation_source-journal">Mol. Gen. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">201</span><span class="NLM_x">, </span> <span class="NLM_fpage">537</span><span class="NLM_x">–</span> <span class="NLM_lpage">542</span><span class="refDoi"> DOI: 10.1007/BF00331352</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1007%2FBF00331352" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=3911030" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADyaL28Xot1ygtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=201&publication_year=1985&pages=537-542&author=A.+Cegielskaauthor=A.+Taylor&title=The+sfiA11+mutation+prevents+filamentation+in+a+response+to+cell+wall+damage+only+in+a+recA%2B+genetic+background&doi=10.1007%2FBF00331352"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The sfiA11 mutation prevents filamentation in a response to cell wall damage only a recA+ genetic background</span></div><div class="casAuthors">Cegielska, Aleksandra; Taylor, Alina</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and General Genetics</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">201</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">537-42</span>CODEN:
                <span class="NLM_cas:coden">MGGEAE</span>;
        ISSN:<span class="NLM_cas:issn">0026-8925</span>.
    </div><div class="casAbstract">N-α-Palmitoyl-L-lysyl-L-lysine dihydrochloride Et ester (PLL)  [7325-88-4] at sublethal doses causes filamentous growth of Escherichia coli strains except sfiA mutants, which divide normally in its presence.  PLL does not elicit the SOS responses, as judged by λ prophage induction, RecA protein synthesis, or the induction of the sfiA operon in a sfiA::lacZ fusion strain.  Filamentation caused by PLL may be not an SOS function and might be the result of membrane damage by PLL, which is an amphipathic compd. and which, at higher doses, causes cell lysis.  Apparently, basal levels of the sfiA gene product are sufficient to inhibit cell division in the presence of PLL.  The phenotype of the sfiA mutation in the presence of PLL requires a recA+ genetic background and does not occur in E. coli mutants recA1 sfiA11, recA13 sfiA11, recA56 sfiA11 and recA441 sfiA11.  All of these strains, except rec441 sfiA11, regain the ability of sfiA11 mutants to divide in the presence of PLL after transformation with the RecA-overproducing plasmid pXO2.  Apparently, the RecA protein pos. affects sfiA11-mediated cell division in the presence of PLL.  The basal level of the RecA protein in the recA+ sfiA11 strain is sufficient for this process.  An increased level due to overprodn. from the multicopy plasmid pXO2 exerts the same effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokYSeaiXlZI7Vg90H21EOLACvtfcHk0lhNJ9sS-Yl_pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28Xot1ygtA%253D%253D&md5=e0d7b238ad35b7539c1c132e83444758</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2FBF00331352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00331352%26sid%3Dliteratum%253Aachs%26aulast%3DCegielska%26aufirst%3DA.%26aulast%3DTaylor%26aufirst%3DA.%26atitle%3DThe%2520sfiA11%2520mutation%2520prevents%2520filamentation%2520in%2520a%2520response%2520to%2520cell%2520wall%2520damage%2520only%2520in%2520a%2520recA%252B%2520genetic%2520background%26jtitle%3DMol.%2520Gen.%2520Genet.%26date%3D1985%26volume%3D201%26spage%3D537%26epage%3D542%26doi%3D10.1007%2FBF00331352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Blundell, T. L.</span><span> </span><span class="NLM_article-title">Structure-based drug design</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">384</span><span class="NLM_x">, </span> <span class="NLM_fpage">23</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=1996&pages=23-26&author=T.+L.+Blundell&title=Structure-based+drug+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DStructure-based%2520drug%2520design%26jtitle%3DNature%26date%3D1996%26volume%3D384%26spage%3D23%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Hofer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crona, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Logan, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sjoberg, B. M.</span><span> </span><span class="NLM_article-title">DNA building blocks: keeping control of manufacture</span> <span class="citation_source-journal">Crit. Rev. Biochem. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">50</span><span class="NLM_x">–</span> <span class="NLM_lpage">63</span><span class="refDoi"> DOI: 10.3109/10409238.2011.630372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.3109%2F10409238.2011.630372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=22050358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1CitLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2012&pages=50-63&author=A.+Hoferauthor=M.+Cronaauthor=D.+T.+Loganauthor=B.+M.+Sjoberg&title=DNA+building+blocks%3A+keeping+control+of+manufacture&doi=10.3109%2F10409238.2011.630372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">DNA building blocks: keeping control of manufacture</span></div><div class="casAuthors">Hofer, Anders; Crona, Mikael; Logan, Derek T.; Sjoeberg, Britt-Marie</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Biochemistry and Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">50-63</span>CODEN:
                <span class="NLM_cas:coden">CRBBEJ</span>;
        ISSN:<span class="NLM_cas:issn">1040-9238</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Ribonucleotide reductase (RNR) is the only source for de novo prodn. of the four deoxyribonucleoside triphosphate (dNTP) building blocks needed for DNA synthesis and repair.  It is crucial that these dNTP pools are carefully balanced, since mutation rates increase when dNTP levels are either unbalanced or elevated.  RNR is the major player in this homeostasis, and with its four different substrates, four different allosteric effectors and two different effector binding sites, it has one of the most sophisticated allosteric regulations known today.  In the past few years, the structures of RNRs from several bacteria, yeast and man have been detd. in the presence of allosteric effectors and substrates, revealing new information about the mechanisms behind the allosteric regulation.  A common theme for all studied RNRs is a flexible loop that mediates modulatory effects from the allosteric specificity site (s-site) to the catalytic site for discrimination between the four substrates.  Much less is known about the allosteric activity site (a-site), which functions as an on-off switch for the enzyme's overall activity by binding ATP (activator) or dATP (inhibitor).  The two nucleotides induce formation of different enzyme oligomers, and a recent structure of a dATP-inhibited α6β2 complex from yeast suggested how its subunits interacted non-productively.  Interestingly, the oligomers formed and the details of their allosteric regulation differ between eukaryotes and Escherichia coli.  Nevertheless, these differences serve a common purpose in an essential enzyme whose allosteric regulation might date back to the era when the mol. mechanisms behind the central dogma evolved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptG-F1ry53D7Vg90H21EOLACvtfcHk0lhNJ9sS-Yl_pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1CitLzF&md5=eef605f2814628042efc94982d098682</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.3109%2F10409238.2011.630372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F10409238.2011.630372%26sid%3Dliteratum%253Aachs%26aulast%3DHofer%26aufirst%3DA.%26aulast%3DCrona%26aufirst%3DM.%26aulast%3DLogan%26aufirst%3DD.%2BT.%26aulast%3DSjoberg%26aufirst%3DB.%2BM.%26atitle%3DDNA%2520building%2520blocks%253A%2520keeping%2520control%2520of%2520manufacture%26jtitle%3DCrit.%2520Rev.%2520Biochem.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D47%26spage%3D50%26epage%3D63%26doi%3D10.3109%2F10409238.2011.630372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Estebanez-Perpina, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnold, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodrigues, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mar, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bateman, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallai, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baxter, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guy, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fletterick, R. J.</span><span> </span><span class="NLM_article-title">A surface on the androgen receptor that allosterically regulates coactivator binding</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">16074</span><span class="NLM_x">–</span> <span class="NLM_lpage">16079</span><span class="refDoi"> DOI: 10.1073/pnas.0708036104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1073%2Fpnas.0708036104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=17911242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtF2hu7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=16074-16079&author=E.+Estebanez-Perpinaauthor=L.+A.+Arnoldauthor=P.+Nguyenauthor=E.+D.+Rodriguesauthor=E.+Marauthor=R.+Batemanauthor=P.+Pallaiauthor=K.+M.+Shokatauthor=J.+D.+Baxterauthor=R.+K.+Guyauthor=P.+Webbauthor=R.+J.+Fletterick&title=A+surface+on+the+androgen+receptor+that+allosterically+regulates+coactivator+binding&doi=10.1073%2Fpnas.0708036104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">A surface on the androgen receptor that allosterically regulates coactivator binding</span></div><div class="casAuthors">Estebanez-Perpina, Eva; Arnold, Alexander A.; Nguyen, Phuong; Rodrigues, Edson Delgado; Mar, Ellena; Bateman, Raynard; Pallai, Peter; Shokat, Kevan M.; Baxter, John D.; Guy, R. Kiplin; Webb, Paul; Fletterick, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">16074-16079</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Current approaches to inhibit nuclear receptor (NR) activity target the hormone binding pocket but face limitations.  The authors have proposed that inhibitors, which bind to nuclear receptor surfaces that mediate assembly of the receptor's binding partners, might overcome some of these limitations.  The androgen receptor (AR) plays a central role in prostate cancer, but conventional inhibitors lose effectiveness as cancer treatments because anti-androgen resistance usually develops.  The authors conducted functional and x-ray screens to identify compds. that bind the AR surface and block binding of coactivators for AR activation function 2 (AF-2).  Four compds. that block coactivator binding in soln. with IC50 ≈ 50 μM and inhibit AF-2 activity in cells were detected: three nonsteroidal antiinflammatory drugs and the thyroid hormone 3,3',5-triiodothyroacetic acid.  Although visualization of compds. at the AR surface reveals weak binding at AF-2, the most potent inhibitors bind preferentially to a previously unknown regulatory surface cleft termed binding function (BF)-3, which is a known target for mutations in prostate cancer and androgen insensitivity syndrome.  X-ray structural anal. reveals that 3,3',5-triiodothyroacetic acid binding to BF-3 remodels the adjacent interaction site AF-2 to weaken coactivator binding.  Mutation of residues that form BF-3 inhibits AR function and AR AF-2 activity.  The authors propose that BF-3 is a previously unrecognized allosteric regulatory site needed for AR activity in vivo and a possible pharmaceutical target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa7awt1cHxHrVg90H21EOLACvtfcHk0lhNJ9sS-Yl_pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtF2hu7bM&md5=23e84e3c667ef01f560246a25fc227e1</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0708036104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0708036104%26sid%3Dliteratum%253Aachs%26aulast%3DEstebanez-Perpina%26aufirst%3DE.%26aulast%3DArnold%26aufirst%3DL.%2BA.%26aulast%3DNguyen%26aufirst%3DP.%26aulast%3DRodrigues%26aufirst%3DE.%2BD.%26aulast%3DMar%26aufirst%3DE.%26aulast%3DBateman%26aufirst%3DR.%26aulast%3DPallai%26aufirst%3DP.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DBaxter%26aufirst%3DJ.%2BD.%26aulast%3DGuy%26aufirst%3DR.%2BK.%26aulast%3DWebb%26aufirst%3DP.%26aulast%3DFletterick%26aufirst%3DR.%2BJ.%26atitle%3DA%2520surface%2520on%2520the%2520androgen%2520receptor%2520that%2520allosterically%2520regulates%2520coactivator%2520binding%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D16074%26epage%3D16079%26doi%3D10.1073%2Fpnas.0708036104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Ciulli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span> </span><span class="NLM_article-title">Probing hot spots at protein-ligand binding sites: a fragment-based approach using biophysical methods</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">4992</span><span class="NLM_x">–</span> <span class="NLM_lpage">5000</span><span class="refDoi"> DOI: 10.1021/jm060490r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060490r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1SksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=4992-5000&author=A.+Ciulliauthor=G.+Williamsauthor=A.+G.+Smithauthor=T.+L.+Blundellauthor=C.+Abell&title=Probing+hot+spots+at+protein-ligand+binding+sites%3A+a+fragment-based+approach+using+biophysical+methods&doi=10.1021%2Fjm060490r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Probing Hot Spots at Protein-Ligand Binding Sites: A Fragment-Based Approach Using Biophysical Methods</span></div><div class="casAuthors">Ciulli, Alessio; Williams, Glyn; Smith, Alison G.; Blundell, Tom L.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4992-5000</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mapping interactions at protein-ligand binding sites is an important aspect of understanding many biol. reactions and a key part of drug design.  In this paper, we have used a fragment-based approach to probe "hot spots" at the cofactor-binding site of a model dehydrogenase, Escherichia coli ketopantoate reductase.  Our strategy involved the breaking down of NADPH (Kd = 300 nM) into smaller fragments and the biophys. characterization of their binding using WaterLOGSY NMR spectroscopy, isothermal titrn. calorimetry (ITC), and inhibition studies.  The weak binding affinities of fragments were measured by direct ITC titrns. under low c value conditions.  The 2'-phosphate and the reduced nicotinamide groups were found to contribute a large part of the binding energy.  A combination of ITC and site-directed mutagenesis enabled us to locate the fragments at sep. hot spots on opposite ends of the cofactor-binding site.  This study has identified structural determinants for cofactor recognition that represent a blueprint for future inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAWWHBHKxvxbVg90H21EOLACvtfcHk0ljvXhmXW8VKWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1SksL0%253D&md5=b6788ad16d6009d7cacd3e429b13523b</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm060490r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060490r%26sid%3Dliteratum%253Aachs%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DWilliams%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DA.%2BG.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DProbing%2520hot%2520spots%2520at%2520protein-ligand%2520binding%2520sites%253A%2520a%2520fragment-based%2520approach%2520using%2520biophysical%2520methods%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D4992%26epage%3D5000%26doi%3D10.1021%2Fjm060490r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Edink, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rucktooa, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Retra, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akdemir, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nahar, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuiderveld, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Elk, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janssen, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Nierop, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Muijlwijk-Koezen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smit, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sixma, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span> </span><span class="NLM_article-title">Fragment growing induces conformational changes in acetylcholine-binding protein: a structural and thermodynamic analysis</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">5363</span><span class="NLM_x">–</span> <span class="NLM_lpage">5371</span><span class="refDoi"> DOI: 10.1021/ja110571r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja110571r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFeisrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=5363-5371&author=E.+Edinkauthor=P.+Rucktooaauthor=K.+Retraauthor=A.+Akdemirauthor=T.+Naharauthor=O.+Zuiderveldauthor=R.+van+Elkauthor=E.+Janssenauthor=P.+van+Nieropauthor=J.+van+Muijlwijk-Koezenauthor=A.+B.+Smitauthor=T.+K.+Sixmaauthor=R.+Leursauthor=I.+J.+de+Esch&title=Fragment+growing+induces+conformational+changes+in+acetylcholine-binding+protein%3A+a+structural+and+thermodynamic+analysis&doi=10.1021%2Fja110571r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment Growing Induces Conformational Changes in Acetylcholine-Binding Protein: A Structural and Thermodynamic Analysis</span></div><div class="casAuthors">Edink, Ewald; Rucktooa, Prakash; Retra, Kim; Akdemir, Atilla; Nahar, Tariq; Zuiderveld, Obbe; van Elk, Rene; Janssen, Elwin; van Nierop, Pim; van Muijlwijk-Koezen, Jacqueline; Smit, August B.; Sixma, Titia K.; Leurs, Rob; de Esch, Iwan J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5363-5371</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Optimization of fragment hits toward high-affinity lead compds. is a crucial aspect of fragment-based drug discovery (FBDD).  In the current study, we have successfully optimized a fragment by growing into a ligand-inducible subpocket of the binding site of acetylcholine-binding protein (AChBP).  This protein is a sol. homolog of the ligand binding domain (LBD) of Cys-loop receptors.  The fragment optimization was monitored with X-ray structures of ligand complexes and systematic thermodn. analyses using surface plasmon resonance (SPR) biosensor anal. and isothermal titrn. calorimetry (ITC).  Using site-directed mutagenesis and AChBP from different species, we find that specific changes in thermodn. binding profiles, are indicative of interactions with the ligand-inducible subpocket of AChBP.  This study illustrates that thermodn. anal. provides valuable information on ligand binding modes and is complementary to affinity data when guiding rational structure- and fragment-based discovery approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7-gLHcdzLv7Vg90H21EOLACvtfcHk0ljvXhmXW8VKWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFeisrc%253D&md5=413798f2128e309545de77f8e995b565</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fja110571r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja110571r%26sid%3Dliteratum%253Aachs%26aulast%3DEdink%26aufirst%3DE.%26aulast%3DRucktooa%26aufirst%3DP.%26aulast%3DRetra%26aufirst%3DK.%26aulast%3DAkdemir%26aufirst%3DA.%26aulast%3DNahar%26aufirst%3DT.%26aulast%3DZuiderveld%26aufirst%3DO.%26aulast%3Dvan%2BElk%26aufirst%3DR.%26aulast%3DJanssen%26aufirst%3DE.%26aulast%3Dvan%2BNierop%26aufirst%3DP.%26aulast%3Dvan%2BMuijlwijk-Koezen%26aufirst%3DJ.%26aulast%3DSmit%26aufirst%3DA.%2BB.%26aulast%3DSixma%26aufirst%3DT.%2BK.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26atitle%3DFragment%2520growing%2520induces%2520conformational%2520changes%2520in%2520acetylcholine-binding%2520protein%253A%2520a%2520structural%2520and%2520thermodynamic%2520analysis%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D5363%26epage%3D5371%26doi%3D10.1021%2Fja110571r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1750</span><span class="NLM_x">–</span> <span class="NLM_lpage">1759</span><span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0ljvXhmXW8VKWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">–</span> <span class="NLM_lpage">1749</span><span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lhz_QyXqMT6Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanschagrin, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span> </span><span class="NLM_article-title">Extra precision glide: docking and scoring incorporating a model of hydrophobic enclosure for protein-ligand complexes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6177</span><span class="NLM_x">–</span> <span class="NLM_lpage">6196</span><span class="refDoi"> DOI: 10.1021/jm051256o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+precision+glide%3A+docking+and+scoring+incorporating+a+model+of+hydrophobic+enclosure+for+protein-ligand+complexes&doi=10.1021%2Fjm051256o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span></div><div class="casAuthors">Friesner, Richard A.; Murphy, Robert B.; Repasky, Matthew P.; Frye, Leah L.; Greenwood, Jeremy R.; Halgren, Thomas A.; Sanschagrin, Paul C.; Mainz, Daniel T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6177-6196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel scoring function to est. protein-ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol.  In addn. to unique water desolvation energy terms, protein-ligand structural motifs leading to enhanced binding affinity are included:(1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral-neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged-charged hydrogen bonds.  The XP scoring function and docking protocol have been developed to reproduce exptl. binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, resp.) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance.  Enrichment results demonstrate the importance of the novel XP mol. recognition and water scoring in sepg. active and inactive ligands and avoiding false positives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZiwN12vaY7Vg90H21EOLACvtfcHk0lhz_QyXqMT6Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D&md5=ea428c82ead0d8c27f8c1a7b694a1edf</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520precision%2520glide%253A%2520docking%2520and%2520scoring%2520incorporating%2520a%2520model%2520of%2520hydrophobic%2520enclosure%2520for%2520protein-ligand%2520complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196%26doi%3D10.1021%2Fjm051256o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Pace, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajdos, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fee, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimsley, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, T.</span><span> </span><span class="NLM_article-title">How to measure and predict the molar absorption coefficient of a protein</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">2411</span><span class="NLM_x">–</span> <span class="NLM_lpage">2423</span><span class="refDoi"> DOI: 10.1002/pro.5560041120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1002%2Fpro.5560041120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=8563639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADyaK2MXpsVGqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1995&pages=2411-2423&author=C.+N.+Paceauthor=F.+Vajdosauthor=L.+Feeauthor=G.+Grimsleyauthor=T.+Gray&title=How+to+measure+and+predict+the+molar+absorption+coefficient+of+a+protein&doi=10.1002%2Fpro.5560041120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">How to measure and predict the molar absorption coefficient of a protein</span></div><div class="casAuthors">Pace, C. Nick; Vajdos, Felix; Fee, Lanette; Grimsley, Gerald; Gray, Theronica</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2411-23</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">The molar absorption coeff., ε, of a protein is usually based on concns. measured by dry wt., nitrogen, or amino acid anal.  The studies reported here suggest that the Edelhoch method is the best method for measuring ε for a protein.  (This method is described by Gill and von Hippel [1989, Anal Biochem 182:319-326] and is based on data from Edelhoch [1967, Biochem. 6:1948-1954].).  The av. ε values for these chromophores in a sample of 18 well-characterized proteins have been estd., and the ε values in water, propanol, 6 M guanidine hydrochloride (GdnHCl), and 8 M urea have been measured.  For Trp, the av. ε values for the proteins are intermediate between those measured in 6 M GdnHCl and those measured in propanol.  Based on a sample of 116 measured ε values for 80 proteins, the ε at 280 nm of a folded protein in water, ε(280), can best be predicted with this equation L ε(280) (M-1 cm-1) = (#Trp)(5,500) + (#Tyr)(1,490) + (#cystine)(125).  These ε(280) values are quite reliable for proteins contg. Trp residues, and less reliable for proteins that do not.  However, the Edelhoch method is convenient and accurate, and the best approach is to measure rather than predict ε.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDyrAj6uk11bVg90H21EOLACvtfcHk0lhz_QyXqMT6Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXpsVGqt7k%253D&md5=033237a125c7486331bf5180047c3cd4</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2Fpro.5560041120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.5560041120%26sid%3Dliteratum%253Aachs%26aulast%3DPace%26aufirst%3DC.%2BN.%26aulast%3DVajdos%26aufirst%3DF.%26aulast%3DFee%26aufirst%3DL.%26aulast%3DGrimsley%26aufirst%3DG.%26aulast%3DGray%26aufirst%3DT.%26atitle%3DHow%2520to%2520measure%2520and%2520predict%2520the%2520molar%2520absorption%2520coefficient%2520of%2520a%2520protein%26jtitle%3DProtein%2520Sci.%26date%3D1995%26volume%3D4%26spage%3D2411%26epage%3D2423%26doi%3D10.1002%2Fpro.5560041120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Krippendorff, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lienau, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reichel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huisinga, W.</span><span> </span><span class="NLM_article-title">Optimizing classification of drug-drug interaction potential for CYP450 isoenzyme inhibition assays in early drug discovery</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">92</span><span class="NLM_x">–</span> <span class="NLM_lpage">99</span><span class="refDoi"> DOI: 10.1177/1087057106295897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1177%2F1087057106295897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=17130250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtFGgsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=92-99&author=B.+F.+Krippendorffauthor=P.+Lienauauthor=A.+Reichelauthor=W.+Huisinga&title=Optimizing+classification+of+drug-drug+interaction+potential+for+CYP450+isoenzyme+inhibition+assays+in+early+drug+discovery&doi=10.1177%2F1087057106295897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing classification of drug-drug interaction potential for CYP450 isoenzyme inhibition assays in early drug discovery</span></div><div class="casAuthors">Krippendorff, Ben-Fillippo; Lienau, Philip; Reichel, Andreas; Huisinga, Wilhelm</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">92-99</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">In drug discovery, the potential of cytochrome P 450 inhibition of new chem. entities is frequently quantified in terms of IC50 values.  In early drug discovery, a risk classification into low, medium, or high potential inhibitors is often sufficient for ranking and prioritizing of compds.  Although often 6 or more inhibitor concns. are used to det. the IC50 value, the question arises whether it is possible to predict the risk class based on fewer inhibitor concns. with comparable reliability.  In this article, the authors propose a new integrated 2-point method with inhibitor concns. chosen in accordance with the risk classification.  They analyze its predictive power and the feasibility of not only classifying the compds. into different risk classes but also ranking those compds. that have been binned into the middle risk class.  The proposed integrated 2-point method is thus highly suitable for automation.  Altogether, it maintains the quality of the prediction while considerably reducing time and cost.  The proposed method is applicable to other IC50 assays and risk classifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouHK1-JW_iC7Vg90H21EOLACvtfcHk0lgQIG1Rt0y1ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtFGgsL8%253D&md5=09ffac9868b98f80c5af46b4bdc5f349</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1177%2F1087057106295897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057106295897%26sid%3Dliteratum%253Aachs%26aulast%3DKrippendorff%26aufirst%3DB.%2BF.%26aulast%3DLienau%26aufirst%3DP.%26aulast%3DReichel%26aufirst%3DA.%26aulast%3DHuisinga%26aufirst%3DW.%26atitle%3DOptimizing%2520classification%2520of%2520drug-drug%2520interaction%2520potential%2520for%2520CYP450%2520isoenzyme%2520inhibition%2520assays%2520in%2520early%2520drug%2520discovery%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2007%26volume%3D12%26spage%3D92%26epage%3D99%26doi%3D10.1177%2F1087057106295897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 8 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Wei Hu, Xu-Sheng Huang, Ji-Feng Wu, Liang Yang, Yong-Tang Zheng, Yue-Mao Shen, Zhi-Yu Li, <span class="NLM_string-name hlFld-ContribAuthor">Xun Li</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Novel Topoisomerase II Inhibitors by Medicinal Chemistry Approaches. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (20)
                                     , 8947-8980. <a href="https://doi.org/10.1021/acs.jmedchem.7b01202" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01202</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01202%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BNovel%252BTopoisomerase%252BII%252BInhibitors%252Bby%252BMedicinal%252BChemistry%252BApproaches%26aulast%3DHu%26aufirst%3DWei%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D16082017%26date%3D20062018%26date%3D05062018%26volume%3D61%26issue%3D20%26spage%3D8947%26epage%3D8980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sarah E.  Huff</span>, <span class="hlFld-ContribAuthor ">Faiz Ahmad  Mohammed</span>, <span class="hlFld-ContribAuthor ">Mu  Yang</span>, <span class="hlFld-ContribAuthor ">Prashansa  Agrawal</span>, <span class="hlFld-ContribAuthor ">John  Pink</span>, <span class="hlFld-ContribAuthor ">Michael E.  Harris</span>, <span class="hlFld-ContribAuthor ">Chris G.  Dealwis</span>, and <span class="hlFld-ContribAuthor ">Rajesh  Viswanathan</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Guided Synthesis and Mechanistic Studies Reveal Sweetspots on Naphthyl Salicyl Hydrazone Scaffold as Non-Nucleosidic Competitive, Reversible Inhibitors of Human Ribonucleotide Reductase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (3)
                                     , 666-680. <a href="https://doi.org/10.1021/acs.jmedchem.7b00530" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00530</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00530%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Guided%252BSynthesis%252Band%252BMechanistic%252BStudies%252BReveal%252BSweetspots%252Bon%252BNaphthyl%252BSalicyl%252BHydrazone%252BScaffold%252Bas%252BNon-Nucleosidic%252BCompetitive%25252C%252BReversible%252BInhibitors%252Bof%252BHuman%252BRibonucleotide%252BReductase%26aulast%3DHuff%26aufirst%3DSarah%2BE.%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D05042017%26date%3D05012018%26date%3D08022018%26date%3D18122017%26volume%3D61%26issue%3D3%26spage%3D666%26epage%3D680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tessianna A.  Misko</span>, <span class="hlFld-ContribAuthor ">Yi-Ting  Liu</span>, <span class="hlFld-ContribAuthor ">Michael E.  Harris</span>, <span class="hlFld-ContribAuthor ">Nancy L.  Oleinick</span>, <span class="hlFld-ContribAuthor ">John  Pink</span>, <span class="hlFld-ContribAuthor ">Hsueh-Yun  Lee</span>, <span class="hlFld-ContribAuthor ">Chris G.  Dealwis</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-guided design of anti-cancer ribonucleotide reductase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2019,</strong> <em>34 </em>
                                    (1)
                                     , 438-450. <a href="https://doi.org/10.1080/14756366.2018.1545226" title="DOI URL">https://doi.org/10.1080/14756366.2018.1545226</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14756366.2018.1545226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14756366.2018.1545226%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3DStructure-guided%252Bdesign%252Bof%252Banti-cancer%252Bribonucleotide%252Breductase%252Binhibitors%26aulast%3DMisko%26aufirst%3DTessianna%2BA.%26date%3D2019%26date%3D2019%26volume%3D34%26issue%3D1%26spage%3D438%26epage%3D450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vijay  Kumar</span>, <span class="hlFld-ContribAuthor ">Anchal  Sharma</span>, <span class="hlFld-ContribAuthor ">Shivendra  Pratap</span>, <span class="hlFld-ContribAuthor ">Pravindra  Kumar</span>. </span><span class="cited-content_cbyCitation_article-title">Biochemical and biophysical characterization of 1,4-naphthoquinone as a dual inhibitor of two key enzymes of type II fatty acid biosynthesis from Moraxella catarrhalis. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - Proteins and Proteomics</span><span> <strong>2018,</strong> <em>1866 </em>
                                    (11)
                                     , 1131-1142. <a href="https://doi.org/10.1016/j.bbapap.2018.08.008" title="DOI URL">https://doi.org/10.1016/j.bbapap.2018.08.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbapap.2018.08.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbapap.2018.08.008%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520Proteins%2520and%2520Proteomics%26atitle%3DBiochemical%252Band%252Bbiophysical%252Bcharacterization%252Bof%252B1%25252C4-naphthoquinone%252Bas%252Ba%252Bdual%252Binhibitor%252Bof%252Btwo%252Bkey%252Benzymes%252Bof%252Btype%252BII%252Bfatty%252Bacid%252Bbiosynthesis%252Bfrom%252BMoraxella%252Bcatarrhalis%26aulast%3DKumar%26aufirst%3DVijay%26date%3D2018%26volume%3D1866%26issue%3D11%26spage%3D1131%26epage%3D1142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mahmoud Mohammad  Yaseen</span>, <span class="hlFld-ContribAuthor ">Nizar Mohammad  Abuharfeil</span>, <span class="hlFld-ContribAuthor ">Mohammad Ali  Alqudah</span>, <span class="hlFld-ContribAuthor ">Mohammad Mahmoud  Yaseen</span>. </span><span class="cited-content_cbyCitation_article-title">Mechanisms and Factors That Drive Extensive Human Immunodeficiency Virus Type-1 Hypervariability: An Overview. </span><span class="cited-content_cbyCitation_journal-name">Viral Immunology</span><span> <strong>2017,</strong> <em>30 </em>
                                    (10)
                                     , 708-726. <a href="https://doi.org/10.1089/vim.2017.0065" title="DOI URL">https://doi.org/10.1089/vim.2017.0065</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1089/vim.2017.0065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1089%2Fvim.2017.0065%26sid%3Dliteratum%253Aachs%26jtitle%3DViral%2520Immunology%26atitle%3DMechanisms%252Band%252BFactors%252BThat%252BDrive%252BExtensive%252BHuman%252BImmunodeficiency%252BVirus%252BType-1%252BHypervariability%25253A%252BAn%252BOverview%26aulast%3DYaseen%26aufirst%3DMahmoud%2BMohammad%26date%3D2017%26volume%3D30%26issue%3D10%26spage%3D708%26epage%3D726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Geraldine  Graser-Loescher</span>, <span class="hlFld-ContribAuthor ">Agnes  Schoenhuber</span>, <span class="hlFld-ContribAuthor ">Caroline  Ciglenec</span>, <span class="hlFld-ContribAuthor ">Sabine  Eberl</span>, <span class="hlFld-ContribAuthor ">Georg  Krupitza</span>, <span class="hlFld-ContribAuthor ">Robert M.  Mader</span>, <span class="hlFld-ContribAuthor ">Surender S.  Jadav</span>, <span class="hlFld-ContribAuthor ">Venkatesan  Jayaprakash</span>, <span class="hlFld-ContribAuthor ">Monika  Fritzer-Szekeres</span>, <span class="hlFld-ContribAuthor ">Thomas  Szekeres</span>, <span class="hlFld-ContribAuthor ">Philipp  Saiko</span>. </span><span class="cited-content_cbyCitation_article-title">Thiosemicarbazone derivatives, thiazolyl hydrazones, effectively inhibit leukemic tumor cell growth: Down-regulation of ribonucleotide reductase activity and synergism with arabinofuranosylcytosine. </span><span class="cited-content_cbyCitation_journal-name">Food and Chemical Toxicology</span><span> <strong>2017,</strong> <em>108 </em>, 53-62. <a href="https://doi.org/10.1016/j.fct.2017.07.029" title="DOI URL">https://doi.org/10.1016/j.fct.2017.07.029</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.fct.2017.07.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.fct.2017.07.029%26sid%3Dliteratum%253Aachs%26jtitle%3DFood%2520and%2520Chemical%2520Toxicology%26atitle%3DThiosemicarbazone%252Bderivatives%25252C%252Bthiazolyl%252Bhydrazones%25252C%252Beffectively%252Binhibit%252Bleukemic%252Btumor%252Bcell%252Bgrowth%25253A%252BDown-regulation%252Bof%252Bribonucleotide%252Breductase%252Bactivity%252Band%252Bsynergism%252Bwith%252Barabinofuranosylcytosine%26aulast%3DGraser-Loescher%26aufirst%3DGeraldine%26date%3D2017%26volume%3D108%26spage%3D53%26epage%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Md. Faiz  Ahmad</span>, <span class="hlFld-ContribAuthor ">Intekhab  Alam</span>, <span class="hlFld-ContribAuthor ">Sarah E.  Huff</span>, <span class="hlFld-ContribAuthor ">John  Pink</span>, <span class="hlFld-ContribAuthor ">Sheryl A.  Flanagan</span>, <span class="hlFld-ContribAuthor ">Donna  Shewach</span>, <span class="hlFld-ContribAuthor ">Tessianna A.  Misko</span>, <span class="hlFld-ContribAuthor ">Nancy L.  Oleinick</span>, <span class="hlFld-ContribAuthor ">William E.  Harte</span>, <span class="hlFld-ContribAuthor ">Rajesh  Viswanathan</span>, <span class="hlFld-ContribAuthor ">Michael E.  Harris</span>, <span class="hlFld-ContribAuthor ">Chris Godfrey  Dealwis</span>. </span><span class="cited-content_cbyCitation_article-title">Potent competitive inhibition of human ribonucleotide reductase by a nonnucleoside small molecule. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2017,</strong> <em>114 </em>
                                    (31)
                                     , 8241-8246. <a href="https://doi.org/10.1073/pnas.1620220114" title="DOI URL">https://doi.org/10.1073/pnas.1620220114</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.1620220114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.1620220114%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DPotent%252Bcompetitive%252Binhibition%252Bof%252Bhuman%252Bribonucleotide%252Breductase%252Bby%252Ba%252Bnonnucleoside%252Bsmall%252Bmolecule%26aulast%3DAhmad%26aufirst%3DMd.%2BFaiz%26date%3D2017%26date%3D2017%26volume%3D114%26issue%3D31%26spage%3D8241%26epage%3D8246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mikael  Crona</span>, <span class="hlFld-ContribAuthor ">Paula  Codó</span>, <span class="hlFld-ContribAuthor ">Venkateswara Rao  Jonna</span>, <span class="hlFld-ContribAuthor ">Anders  Hofer</span>, <span class="hlFld-ContribAuthor ">Aristi P.  Fernandes</span>, <span class="hlFld-ContribAuthor ">Fredrik  Tholander</span>. </span><span class="cited-content_cbyCitation_article-title">A ribonucleotide reductase inhibitor with deoxyribonucleoside-reversible cytotoxicity. </span><span class="cited-content_cbyCitation_journal-name">Molecular Oncology</span><span> <strong>2016,</strong> <em>10 </em>
                                    (9)
                                     , 1375-1386. <a href="https://doi.org/10.1016/j.molonc.2016.07.008" title="DOI URL">https://doi.org/10.1016/j.molonc.2016.07.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molonc.2016.07.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molonc.2016.07.008%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Oncology%26atitle%3DA%252Bribonucleotide%252Breductase%252Binhibitor%252Bwith%252Bdeoxyribonucleoside-reversible%252Bcytotoxicity%26aulast%3DCrona%26aufirst%3DMikael%26date%3D2016%26date%3D2016%26volume%3D10%26issue%3D9%26spage%3D1375%26epage%3D1386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/medium/jm-2015-00929x_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00929&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/medium/jm-2015-00929x_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. hRRM1 drug binding sites and fluorescence quenching identifies RR inhibitors. (A) Structure of the hRRM1 dimer with drug-target sites mapped. The M-site is the binding site for the new class of modulators that is the subject of this study. The A-site controls activity. The S-site controls specificity. The C-site is the catalytic site. The P-site binds the smaller R2 subunit derived peptide. (B and C) Tryptophan fluorescence quenching of hRRM1 in the presence of compounds <b>4</b> and <b>10</b>, respectively. (D and E) No tryptophan fluorescence quenching of hRRM1 by compounds <b>100207</b> and <b>184612</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00929&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/medium/jm-2015-00929x_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compound <b>4</b> interactions with hRRM1. (A) Model of the hRRM1 hexamer based on the <i>S. cerevisiae</i> RR1 hexamer structure. Ribbon diagram of the hRRM1 hexamer packing arrangement. hRRM1 monomers are green and magenta. All of the four helix ATP-binding cones are red. The 16 N-terminal residues at the hexamer interface are in cyan. Effectors (TTP) bound at the S-site are drawn in brick red spheres, and compound <b>4</b> at the hexamer interface is drawn in blue spheres. (B) The |<i>F</i><sub>0</sub>| – |<i>F</i><sub>c</sub>| electron density for the phthalimide compound (blue) in complex with hRRM1 orthorhombic crystals. Density contoured at 3σ defines the phthalimide binding to hRRM1. (C) 2|<i>F</i><sub>0</sub>| – |<i>F</i><sub>c</sub>| electron density (blue) of the phthalimide compound contoured at 1σ after refinement in the phthalimide-hRRM1 orthorhombic complex. (D) Illustration of the A-site and M-site binding by dATP and compound <b>4</b>, respectively, Compound <b>4</b> is shown in magenta, and dATP is in yellow. (E) Lig plot analysis of compound <b>4</b> interactions with hRRM1. The phthalimide compound is shown in purple, carbon atoms in phthalimide are shown in black, oxygen atoms are in red, and nitrogen atoms are in blue. An amino acid residue from hRRM1 interacting with the phthalimide is shown in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00929&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/medium/jm-2015-00929x_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Growth inhibition of MDA-MB-231 and HCT-116 cancer cells in a moderate throughput screen of candidate RR inhibitors. Cells were treated with 3 doses of candidate drugs (1 μM, 10 μM, and 50 μM) for 3 days in a standard growth inhibition assay in 96 well plates. Growth inhibition in duplicate wells of each drug/dose/cell line was assessed by measuring relative DNA content per well compared with that of untreated cells. Drug effect (1-relative growth) is plotted for the 10 μM and 1 μM dose groups (50 μM groups were not included for clarity).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00929&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/medium/jm-2015-00929x_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Growth inhibition of cancer cell lines by compound <b>1</b>. Cells were seeded into 96 well plates (2000 cells/well) and the following day incubated with the indicated concentrations of gemcitabine alone or in combination with compound <b>1</b> for 24 h. At this time, drugs were removed and replaced with control media in the gemcitabine alone groups, or compound <b>1</b> at the indicated concentrations in the compound <b>1</b> alone or combination groups for an additional 48 h. Relative growth was assessed by measuring DNA content in each well. Each drug concentration was assayed utilizing five replicates for each cell line. Results are representative of at least 2 experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00929&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/medium/jm-2015-00929x_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Effect of phthalimide binding on oligomerization in hRRM1 using gel filtration chromatography. (A) Chromatogram of hRRM1 with 1 mM concentration of compound <b>4</b> is shown in red, where the native hRRM1 in the absence of compound <b>4</b> is shown in black. (B) Standard curve for the determination of molecular masses (<i>Mr</i>) of RR. <i>K</i><sub>av</sub> = (<i>V</i><sub>e</sub> – <i>V</i><sub>0</sub>)/(<i>V</i><sub>t</sub> – <i>V</i><sub>0</sub>), where <i>V</i><sub>e</sub> = elution volume, <i>V</i><sub>0</sub> = void volume, and <i>V</i><sub>t</sub> = total volume. (C) hRRM1 hexamerization in the presence of dATP; at 50 μM dATP, the hexamers species are predominant with a small amount of dimers. The hexamer to dimer ratio based on integration of the peaks is approximately 3-fold. (D) The chromatogram of hRRM1 in the presence of 1 mM compound <b>4</b> and 50 μM dATP. The hexamer to dimer ratio based on integration of the peaks is approximately 7-fold.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-24/acs.jmedchem.5b00929/20151218/images/large/jm-2015-00929x_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b00929&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i28">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34368" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34368" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 57 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Jordheim, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seve, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tredan, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dumontet, C.</span><span> </span><span class="NLM_article-title">The ribonucleotide reductase large subunit (RRM1) as a predictive factor in patients with cancer</span> <span class="citation_source-journal">Lancet Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">702</span><span class="refDoi"> DOI: 10.1016/S1470-2045(10)70244-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1016%2FS1470-2045%2810%2970244-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=21163702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3MXot1ejur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=693-702&author=L.+P.+Jordheimauthor=P.+Seveauthor=O.+Tredanauthor=C.+Dumontet&title=The+ribonucleotide+reductase+large+subunit+%28RRM1%29+as+a+predictive+factor+in+patients+with+cancer&doi=10.1016%2FS1470-2045%2810%2970244-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The ribonucleotide reductase large subunit (RRM1) as a predictive factor in patients with cancer</span></div><div class="casAuthors">Jordheim, Lars Petter; Seve, Pascal; Tredan, Olivier; Dumontet, Charles</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">693-702</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Summary: The large subunit of human ribonucleotide reductase, RRM1, is involved in the regulation of cell proliferation, cell migration, tumor and metastasis development, and the synthesis of deoxyribonucleotides for DNA synthesis.  It is also a cellular target for the chemotherapeutic agent, gemcitabine.  RRM1 has been studied in a large no. of patients with different types of cancer, such as non-small-cell lung cancer, pancreatic cancer, breast cancer, and biliary tract cancer, to establish its prognostic or predictive value when patients were treated with gemcitabine, and mRNA expression and genetic variants as detd. by genotyping have in some cases been assocd. with clin. outcome of patients with cancer.  Here, we review preclin. and clin. studies of RRM1 assessment and discuss the further steps in the development of this clin. pertinent biomarker.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo06llwPkTEqbVg90H21EOLACvtfcHk0ljr7682QnSTeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXot1ejur0%253D&md5=e64d3933721a6bcf856ae763ff8ec47a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2810%2970244-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252810%252970244-8%26sid%3Dliteratum%253Aachs%26aulast%3DJordheim%26aufirst%3DL.%2BP.%26aulast%3DSeve%26aufirst%3DP.%26aulast%3DTredan%26aufirst%3DO.%26aulast%3DDumontet%26aufirst%3DC.%26atitle%3DThe%2520ribonucleotide%2520reductase%2520large%2520subunit%2520%2528RRM1%2529%2520as%2520a%2520predictive%2520factor%2520in%2520patients%2520with%2520cancer%26jtitle%3DLancet%2520Oncol.%26date%3D2011%26volume%3D12%26spage%3D693%26epage%3D702%26doi%3D10.1016%2FS1470-2045%2810%2970244-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Jordheim, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guittet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lepoivre, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galmarini, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dumontet, C.</span><span> </span><span class="NLM_article-title">Increased expression of the large subunit of ribonucleotide reductase is involved in resistance to gemcitabine in human mammary adenocarcinoma cells</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1268</span><span class="NLM_x">–</span> <span class="NLM_lpage">1276</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-05-0121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1158%2F1535-7163.MCT-05-0121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=16093443" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnt1Shtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=1268-1276&author=L.+P.+Jordheimauthor=O.+Guittetauthor=M.+Lepoivreauthor=C.+M.+Galmariniauthor=C.+Dumontet&title=Increased+expression+of+the+large+subunit+of+ribonucleotide+reductase+is+involved+in+resistance+to+gemcitabine+in+human+mammary+adenocarcinoma+cells&doi=10.1158%2F1535-7163.MCT-05-0121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Increased expression of the large subunit of ribonucleotide reductase is involved in resistance to gemcitabine in human mammary adenocarcinoma cells</span></div><div class="casAuthors">Jordheim, Lars Petter; Guittet, Olivier; Lepoivre, Michel; Galmarini, Carlos M.; Dumontet, Charles</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1268-1276</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Resistance to cytotoxic nucleoside analogs is a major problem in cancer treatment.  The cellular mechanisms involved in this phenomenon have been studied for several years, and some factors have been identified.  However, this resistance seems to be multifactorial and more studies are needed to gain better insight into this domain.  For this purpose, we developed a gemcitabine-resistant cell line (MCF7 1K) from the human mammary adenocarcinoma MCF7 strain by prolonged exposure to gemcitabine in vitro.  MCF7 1K cells are highly resistant to gemcitabine (533-fold) and cross-resistance is obsd. with araC (47-fold), triapine (14-fold), and hydroxyurea (6.7-fold).  Quant. real-time reverse transcription-PCR and Western blot anal. showed an increase in the gene and protein expression of the large subunit of ribonucleotide reductase, R1.  Ribonucleotide reductase activity was also significantly increased in the gemcitabine-resistant cells.  Study of genomic DNA showed 12-fold increase in R1 gene dosage in MCF7 1K cells.  In contrast, the gene and protein expression of the small subunit of ribonucleotide reductase, R2, were not modified in this cell line.  These results show that gemcitabine resistance can be assocd. with genetic modifications of target genes in malignant cells, and suggest that the large subunit of human ribonucleotide reductase is involved in the cellular response to gemcitabine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9Uk0sNRqiYrVg90H21EOLACvtfcHk0ljr7682QnSTeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnt1Shtbw%253D&md5=4b78ae54a2d3ba8c1195ad2f1c34edd9</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-05-0121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-05-0121%26sid%3Dliteratum%253Aachs%26aulast%3DJordheim%26aufirst%3DL.%2BP.%26aulast%3DGuittet%26aufirst%3DO.%26aulast%3DLepoivre%26aufirst%3DM.%26aulast%3DGalmarini%26aufirst%3DC.%2BM.%26aulast%3DDumontet%26aufirst%3DC.%26atitle%3DIncreased%2520expression%2520of%2520the%2520large%2520subunit%2520of%2520ribonucleotide%2520reductase%2520is%2520involved%2520in%2520resistance%2520to%2520gemcitabine%2520in%2520human%2520mammary%2520adenocarcinoma%2520cells%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2005%26volume%3D4%26spage%3D1268%26epage%3D1276%26doi%3D10.1158%2F1535-7163.MCT-05-0121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Shao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, Y.</span><span> </span><span class="NLM_article-title">Ribonucleotide reductase inhibitors and future drug design</span> <span class="citation_source-journal">Curr. Cancer Drug Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">409</span><span class="NLM_x">–</span> <span class="NLM_lpage">431</span><span class="refDoi"> DOI: 10.2174/156800906777723949</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.2174%2F156800906777723949" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=16918309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD28XnslCmsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2006&pages=409-431&author=J.+Shaoauthor=B.+Zhouauthor=B.+Chuauthor=Y.+Yen&title=Ribonucleotide+reductase+inhibitors+and+future+drug+design&doi=10.2174%2F156800906777723949"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Ribonucleotide reductase inhibitors and future drug design</span></div><div class="casAuthors">Shao, J.; Zhou, B.; Chu, Bernard; Yen, Y.</div><div class="citationInfo"><span class="NLM_cas:title">Current Cancer Drug Targets</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">409-431</span>CODEN:
                <span class="NLM_cas:coden">CCDTB9</span>;
        ISSN:<span class="NLM_cas:issn">1568-0096</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Ribonucleotide reductase (RR) is a multisubunit enzyme responsible for the redn. of ribonucleotides to their corresponding deoxyribonucleotides, which are building blocks for DNA replication and repair.  The key role of RR in DNA synthesis and cell growth control has made it an important target for anticancer therapy.  Increased RR activity has been assocd. with malignant transformation and tumor cell growth.  Efforts for new RR inhibitors have been made in basic and translational research.  In recent years, several RR inhibitors, including Triapine, Gemcitabine, and GTI-2040, have entered clin. trial or application.  Furthermore, the discovery of p53R2, a p53-inducible form of the small subunit of RR, raises the interest to develop subunit-specific RR inhibitors for cancer treatment.  This review compiles recent studies on (1) the structure, function, and regulation of two forms of RR; (2) the role in tumorigenesis of RR and the effect of RR inhibition in cancer treatment; (3) the classification, mechanisms of action, antitumor activity, and clin. trial and application of new RR inhibitors that have been used in clin. cancer chemotherapy or are being evaluated in clin. trials; (4) novel approaches for future RR inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaMYCdtQEZLVg90H21EOLACvtfcHk0li8uuR2KDRoww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnslCmsL8%253D&md5=cafea6b6ddf5aff0a4d75c09b120c2b9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2174%2F156800906777723949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156800906777723949%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DChu%26aufirst%3DB.%26aulast%3DYen%26aufirst%3DY.%26atitle%3DRibonucleotide%2520reductase%2520inhibitors%2520and%2520future%2520drug%2520design%26jtitle%3DCurr.%2520Cancer%2520Drug%2520Targets%26date%3D2006%26volume%3D6%26spage%3D409%26epage%3D431%26doi%3D10.2174%2F156800906777723949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Tang, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. C.</span><span> </span><span class="NLM_article-title">Novel therapeutic targets for pancreatic cancer</span> <span class="citation_source-journal">World J. Gastroenterol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">10825</span><span class="NLM_x">–</span> <span class="NLM_lpage">10844</span><span class="refDoi"> DOI: 10.3748/wjg.v20.i31.10825</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.3748%2Fwjg.v20.i31.10825" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=25152585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFajurbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=10825-10844&author=S.+C.+Tangauthor=Y.+C.+Chen&title=Novel+therapeutic+targets+for+pancreatic+cancer&doi=10.3748%2Fwjg.v20.i31.10825"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Novel therapeutic targets for pancreatic cancer</span></div><div class="casAuthors">Tang, Shing-Chun; Chen, Yang-Chao</div><div class="citationInfo"><span class="NLM_cas:title">World Journal of Gastroenterology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">10825-10844, 21 pp.</span>CODEN:
                <span class="NLM_cas:coden">WJGAF2</span>;
        ISSN:<span class="NLM_cas:issn">1007-9327</span>.
    
            (<span class="NLM_cas:orgname">Baishideng Publishing Group Inc.</span>)
        </div><div class="casAbstract">A review.  Pancreatic cancer has become the fourth leading cause of cancer death in the last two decades.  Only 3%-15% of patients diagnosed with pancreatic cancer had 5 yr survival rate.  Drug resistance, high metastasis, poor prognosis and tumor relapse contributed to the malignancies and difficulties in treating pancreatic cancer.  The current std. chemotherapy for pancreatic cancer is gemcitabine, however its efficacy is far from satisfactory, one of the reasons is due to the complex tumor microenvironment which decreases effective drug delivery to target cancer cell.  Studies of the mol. pathol. of pancreatic cancer have revealed that activation of KRAS, overexpression of cyclooxygenase-2, inactivation of p16INK4A and loss of p53 activities occurred in pancreatic cancer.  Co-administration of gemcitabine and targeting the mol. pathol. events happened in pancreatic cancer has brought an enhanced therapeutic effectiveness of gemcitabine.  Therefore, studies looking for novel targets in hindering pancreatic tumor growth are emerging rapidly.  In order to give a better understanding of the current findings and to seek the direction in future pancreatic cancer research; in this review we will focus on targets suppressing tumor metastasis and progression, KRAS activated downstream effectors, the relationship of Notch signaling and Nodal/Activin signaling with pancreatic cancer cells, the current findings of non-coding RNAs in inhibiting pancreatic cancer cell proliferation, brief discussion in transcription remodeling by epigenetic modifiers (e.g., HDAC, BMI1, EZH2) and the plausible therapeutic applications of cancer stem cell and hyaluronan in tumor environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4pSDAY_HLObVg90H21EOLACvtfcHk0li8uuR2KDRoww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFajurbK&md5=50dfb899e09c4f2dfd7708ddbe828972</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3748%2Fwjg.v20.i31.10825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3748%252Fwjg.v20.i31.10825%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DS.%2BC.%26aulast%3DChen%26aufirst%3DY.%2BC.%26atitle%3DNovel%2520therapeutic%2520targets%2520for%2520pancreatic%2520cancer%26jtitle%3DWorld%2520J.%2520Gastroenterol.%26date%3D2014%26volume%3D20%26spage%3D10825%26epage%3D10844%26doi%3D10.3748%2Fwjg.v20.i31.10825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Huang, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chubb, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plunkett, W.</span><span> </span><span class="NLM_article-title">Termination of DNA synthesis by 9-beta-D-arabinofuranosyl-2-fluoroadenine. A mechanism for cytotoxicity</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">265</span><span class="NLM_x">, </span> <span class="NLM_fpage">16617</span><span class="NLM_x">–</span> <span class="NLM_lpage">16625</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1697861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADyaK3cXlvVCgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=265&publication_year=1990&pages=16617-16625&author=P.+Huangauthor=S.+Chubbauthor=W.+Plunkett&title=Termination+of+DNA+synthesis+by+9-beta-D-arabinofuranosyl-2-fluoroadenine.+A+mechanism+for+cytotoxicity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Termination of DNA synthesis by 9-β-D-arabinofuranosyl-2-fluoroadenine.  A mechanism for cytotoxicity</span></div><div class="casAuthors">Huang, Peng; Chubb, Sherri; Plunkett, William</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">265</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">16617-25</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">The action of 9-β-D-arabinofuranosyl-2-fluoroadenine (F-ara-A) on DNA synthesis was evaluated in whole cells and in vitro.  F-ara-A was converted to its 5'-triphosphate (F-ara-ATP) in cells and then incorporated into DNA in a self-limiting manner.  More than 94% of the analog ws incorporated into DNA at the 3'-termini, indicating a chain termination action.  In vitro DNA primer extension expts. further revealed that F-ara-ATP competed with dATP for incorporation into the A site of the extending DNA strand.  The incorporation of F-ara-AMP into DNA resulted in termination of DNA strand elongation.  Human DNA polymerase α incorporated more F-ara-AMP into DNA than polymerase ε (proliferating cell nuclear antigen-independent DNA polymerase δ) and was more sensitive to inhibition by F-ara-ATP.  DNA polymerase ε was able to excise the incorporated F-ara-AMP from DNA in vitro.  The incorporation of F-ara-AMP into DNA was linearly correlated both with inhibition of DNA synthesis and with loss of clonogenicity; thus it may be the mechanism of its cytotoxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplGaiLMmReHLVg90H21EOLACvtfcHk0li8uuR2KDRoww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXlvVCgurw%253D&md5=1ce760d8ae93741a29a2a690527a9449</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DP.%26aulast%3DChubb%26aufirst%3DS.%26aulast%3DPlunkett%26aufirst%3DW.%26atitle%3DTermination%2520of%2520DNA%2520synthesis%2520by%25209-beta-D-arabinofuranosyl-2-fluoroadenine.%2520A%2520mechanism%2520for%2520cytotoxicity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1990%26volume%3D265%26spage%3D16617%26epage%3D16625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Avery, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rehg, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lumm, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harwood, F. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santana, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blakley, R. L.</span><span> </span><span class="NLM_article-title">Biochemical pharmacology of 2-chlorodeoxyadenosine in malignant human hematopoietic cell lines and therapeutic effects of 2-bromodeoxyadenosine in drug combinations in mice</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">4972</span><span class="NLM_x">–</span> <span class="NLM_lpage">4978</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1989&pages=4972-4978&author=T.+L.+Averyauthor=J.+E.+Rehgauthor=W.+C.+Lummauthor=F.+C.+Harwoodauthor=V.+M.+Santanaauthor=R.+L.+Blakley&title=Biochemical+pharmacology+of+2-chlorodeoxyadenosine+in+malignant+human+hematopoietic+cell+lines+and+therapeutic+effects+of+2-bromodeoxyadenosine+in+drug+combinations+in+mice"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAvery%26aufirst%3DT.%2BL.%26aulast%3DRehg%26aufirst%3DJ.%2BE.%26aulast%3DLumm%26aufirst%3DW.%2BC.%26aulast%3DHarwood%26aufirst%3DF.%2BC.%26aulast%3DSantana%26aufirst%3DV.%2BM.%26aulast%3DBlakley%26aufirst%3DR.%2BL.%26atitle%3DBiochemical%2520pharmacology%2520of%25202-chlorodeoxyadenosine%2520in%2520malignant%2520human%2520hematopoietic%2520cell%2520lines%2520and%2520therapeutic%2520effects%2520of%25202-bromodeoxyadenosine%2520in%2520drug%2520combinations%2520in%2520mice%26jtitle%3DCancer%2520Res.%26date%3D1989%26volume%3D49%26spage%3D4972%26epage%3D4978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Griffig, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koob, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blakley, R. L.</span><span> </span><span class="NLM_article-title">Mechanisms of inhibition of DNA synthesis by 2-chlorodeoxyadenosine in human lymphoblastic cells</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6923</span><span class="NLM_x">–</span> <span class="NLM_lpage">6928</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1989&pages=6923-6928&author=J.+Griffigauthor=R.+Koobauthor=R.+L.+Blakley&title=Mechanisms+of+inhibition+of+DNA+synthesis+by+2-chlorodeoxyadenosine+in+human+lymphoblastic+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGriffig%26aufirst%3DJ.%26aulast%3DKoob%26aufirst%3DR.%26aulast%3DBlakley%26aufirst%3DR.%2BL.%26atitle%3DMechanisms%2520of%2520inhibition%2520of%2520DNA%2520synthesis%2520by%25202-chlorodeoxyadenosine%2520in%2520human%2520lymphoblastic%2520cells%26jtitle%3DCancer%2520Res.%26date%3D1989%26volume%3D49%26spage%3D6923%26epage%3D6928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Parker, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaddix, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brockman, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shortnacy, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montgomery, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Secrist, J. A.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, L. L.,  Jr.</span><span> </span><span class="NLM_article-title">Effects of 2-chloro-9-(2-deoxy-2-fluoro-beta-D-arabinofuranosyl)adenine on K562 cellular metabolism and the inhibition of human ribonucleotide reductase and DNA polymerases by its 5′-triphosphate</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2386</span><span class="NLM_x">–</span> <span class="NLM_lpage">2394</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1991&pages=2386-2394&author=W.+B.+Parkerauthor=S.+C.+Shaddixauthor=C.+H.+Changauthor=E.+L.+Whiteauthor=L.+M.+Roseauthor=R.+W.+Brockmanauthor=A.+T.+Shortnacyauthor=J.+A.+Montgomeryauthor=J.+A.+Secristauthor=L.+L.+Bennett&title=Effects+of+2-chloro-9-%282-deoxy-2-fluoro-beta-D-arabinofuranosyl%29adenine+on+K562+cellular+metabolism+and+the+inhibition+of+human+ribonucleotide+reductase+and+DNA+polymerases+by+its+5%E2%80%B2-triphosphate"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DParker%26aufirst%3DW.%2BB.%26aulast%3DShaddix%26aufirst%3DS.%2BC.%26aulast%3DChang%26aufirst%3DC.%2BH.%26aulast%3DWhite%26aufirst%3DE.%2BL.%26aulast%3DRose%26aufirst%3DL.%2BM.%26aulast%3DBrockman%26aufirst%3DR.%2BW.%26aulast%3DShortnacy%26aufirst%3DA.%2BT.%26aulast%3DMontgomery%26aufirst%3DJ.%2BA.%26aulast%3DSecrist%26aufirst%3DJ.%2BA.%26aulast%3DBennett%26aufirst%3DL.%2BL.%26atitle%3DEffects%2520of%25202-chloro-9-%25282-deoxy-2-fluoro-beta-D-arabinofuranosyl%2529adenine%2520on%2520K562%2520cellular%2520metabolism%2520and%2520the%2520inhibition%2520of%2520human%2520ribonucleotide%2520reductase%2520and%2520DNA%2520polymerases%2520by%2520its%25205%25E2%2580%25B2-triphosphate%26jtitle%3DCancer%2520Res.%26date%3D1991%26volume%3D51%26spage%3D2386%26epage%3D2394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Heinemann, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Y. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chubb, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hertel, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grindey, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plunkett, W.</span><span> </span><span class="NLM_article-title">Inhibition of ribonucleotide reduction in CCRF-CEM cells by 2′,2′-difluorodeoxycytidine</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">567</span><span class="NLM_x">–</span> <span class="NLM_lpage">572</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1990&pages=567-572&author=V.+Heinemannauthor=Y.+Z.+Xuauthor=S.+Chubbauthor=A.+Senauthor=L.+W.+Hertelauthor=G.+B.+Grindeyauthor=W.+Plunkett&title=Inhibition+of+ribonucleotide+reduction+in+CCRF-CEM+cells+by+2%E2%80%B2%2C2%E2%80%B2-difluorodeoxycytidine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHeinemann%26aufirst%3DV.%26aulast%3DXu%26aufirst%3DY.%2BZ.%26aulast%3DChubb%26aufirst%3DS.%26aulast%3DSen%26aufirst%3DA.%26aulast%3DHertel%26aufirst%3DL.%2BW.%26aulast%3DGrindey%26aufirst%3DG.%2BB.%26aulast%3DPlunkett%26aufirst%3DW.%26atitle%3DInhibition%2520of%2520ribonucleotide%2520reduction%2520in%2520CCRF-CEM%2520cells%2520by%25202%25E2%2580%25B2%252C2%25E2%2580%25B2-difluorodeoxycytidine%26jtitle%3DMol.%2520Pharmacol.%26date%3D1990%26volume%3D38%26spage%3D567%26epage%3D572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Parker, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaddix, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shewach, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hertel, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Secrist, J. A.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montgomery, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, L. L.,  Jr.</span><span> </span><span class="NLM_article-title">Comparison of the mechanism of cytotoxicity of 2-chloro-9-(2-deoxy-2- fluoro-beta-D-arabinofuranosyl)adenine, 2-chloro-9-(2-deoxy-2-fluoro- beta-D-ribofuranosyl)adenine, and 2-chloro-9-(2-deoxy-2,2-difluoro- beta-D-ribofuranosyl)adenine in CEM cells</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">515</span><span class="NLM_x">–</span> <span class="NLM_lpage">520</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=1999&pages=515-520&author=W.+B.+Parkerauthor=S.+C.+Shaddixauthor=L.+M.+Roseauthor=D.+S.+Shewachauthor=L.+W.+Hertelauthor=J.+A.+Secristauthor=J.+A.+Montgomeryauthor=L.+L.+Bennett&title=Comparison+of+the+mechanism+of+cytotoxicity+of+2-chloro-9-%282-deoxy-2-+fluoro-beta-D-arabinofuranosyl%29adenine%2C+2-chloro-9-%282-deoxy-2-fluoro-+beta-D-ribofuranosyl%29adenine%2C+and+2-chloro-9-%282-deoxy-2%2C2-difluoro-+beta-D-ribofuranosyl%29adenine+in+CEM+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DParker%26aufirst%3DW.%2BB.%26aulast%3DShaddix%26aufirst%3DS.%2BC.%26aulast%3DRose%26aufirst%3DL.%2BM.%26aulast%3DShewach%26aufirst%3DD.%2BS.%26aulast%3DHertel%26aufirst%3DL.%2BW.%26aulast%3DSecrist%26aufirst%3DJ.%2BA.%26aulast%3DMontgomery%26aufirst%3DJ.%2BA.%26aulast%3DBennett%26aufirst%3DL.%2BL.%26atitle%3DComparison%2520of%2520the%2520mechanism%2520of%2520cytotoxicity%2520of%25202-chloro-9-%25282-deoxy-2-%2520fluoro-beta-D-arabinofuranosyl%2529adenine%252C%25202-chloro-9-%25282-deoxy-2-fluoro-%2520beta-D-ribofuranosyl%2529adenine%252C%2520and%25202-chloro-9-%25282-deoxy-2%252C2-difluoro-%2520beta-D-ribofuranosyl%2529adenine%2520in%2520CEM%2520cells%26jtitle%3DMol.%2520Pharmacol.%26date%3D1999%26volume%3D55%26spage%3D515%26epage%3D520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Cerqueira, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandes, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramos, M. J.</span><span> </span><span class="NLM_article-title">Ribonucleotide reductase: a critical enzyme for cancer chemotherapy and antiviral agents</span> <span class="citation_source-journal">Recent Pat. Anti-Cancer Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">11</span><span class="NLM_x">–</span> <span class="NLM_lpage">29</span><span class="refDoi"> DOI: 10.2174/157489207779561408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.2174%2F157489207779561408" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=11-29&author=N.+M.+Cerqueiraauthor=P.+A.+Fernandesauthor=M.+J.+Ramos&title=Ribonucleotide+reductase%3A+a+critical+enzyme+for+cancer+chemotherapy+and+antiviral+agents&doi=10.2174%2F157489207779561408"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.2174%2F157489207779561408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157489207779561408%26sid%3Dliteratum%253Aachs%26aulast%3DCerqueira%26aufirst%3DN.%2BM.%26aulast%3DFernandes%26aufirst%3DP.%2BA.%26aulast%3DRamos%26aufirst%3DM.%2BJ.%26atitle%3DRibonucleotide%2520reductase%253A%2520a%2520critical%2520enzyme%2520for%2520cancer%2520chemotherapy%2520and%2520antiviral%2520agents%26jtitle%3DRecent%2520Pat.%2520Anti-Cancer%2520Drug%2520Discovery%26date%3D2007%26volume%3D2%26spage%3D11%26epage%3D29%26doi%3D10.2174%2F157489207779561408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohman, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stubbe, J.</span><span> </span><span class="NLM_article-title">Mechanism of inactivation of human ribonucleotide reductase with p53R2 by gemcitabine 5′-diphosphate</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">11612</span><span class="NLM_x">–</span> <span class="NLM_lpage">11621</span><span class="refDoi"> DOI: 10.1021/bi901588z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi901588z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVens7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=11612-11621&author=J.+Wangauthor=G.+J.+Lohmanauthor=J.+Stubbe&title=Mechanism+of+inactivation+of+human+ribonucleotide+reductase+with+p53R2+by+gemcitabine+5%E2%80%B2-diphosphate&doi=10.1021%2Fbi901588z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of Inactivation of Human Ribonucleotide Reductase with p53R2 by Gemcitabine 5'-Diphosphate</span></div><div class="casAuthors">Wang, Jun; Lohman, Gregory J. S.; Stubbe, JoAnne</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">11612-11621</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ribonucleotide reductases (RNRs) catalyze the conversion of nucleoside 5'-diphosphates to the corresponding deoxynucleotides, supplying the dNTPs required for DNA replication and DNA repair.  Class I RNRs require two subunits, α and β, for activity.  Humans possess two β subunits: one involved in S phase DNA replication (β), and a second one involved in mitochondrial DNA replication (β' or p53R2) and potentially DNA repair.  Gemcitabine (F2C) is used clin. as an anticancer agent, and its phosphorylated metabolites target many enzymes involved in nucleotide metab., including RNR.  The present investigation with α (specific activity of 400 nmol min-1 mg-1) and β' (0.6 Y·/β'2 and a specific activity of 420 nmol min-1 mg-1) establishes that F2CDP is a substoichiometric inactivator of RNR.  Incubation of this α/β' with [1'-3H]-F2CDP or [5-3H]-F2CDP and reisolation of the protein by Sephadex G-50 chromatog. resulted in recovery 0.5 equiv of covalently bound sugar and 0.03 equiv of tightly assocd. cytosine to α2.  SDS-PAGE anal. (loaded without boiling) of the inactivated RNR showed that 60% of α migrates as a 90 kDa protein and 40% as a 120 kDa protein.  Incubation of [1'-3H]-F2CDP with active site mutants C444S/A, C218S/A, and E431Q/D-α and the C-terminal tail C787S/A and C790S/A mutants reveals that no sugar label is bound to the active site mutants of α and that in the case of C218S-α, α migrates as a 90 kDa protein.  Anal. of the inactivated wt-α/β' RNR by size exclusion chromatog. indicates a quaternary structure of α6β'6.  A mechanism of inactivation common with hα/β is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9SOObMT8YvLVg90H21EOLACvtfcHk0ljf2lXd-RXGAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVens7zJ&md5=8b1f3f09511b91e2bf328e4f0df18d1a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fbi901588z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi901588z%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLohman%26aufirst%3DG.%2BJ.%26aulast%3DStubbe%26aufirst%3DJ.%26atitle%3DMechanism%2520of%2520inactivation%2520of%2520human%2520ribonucleotide%2520reductase%2520with%2520p53R2%2520by%2520gemcitabine%25205%25E2%2580%25B2-diphosphate%26jtitle%3DBiochemistry%26date%3D2009%26volume%3D48%26spage%3D11612%26epage%3D11621%26doi%3D10.1021%2Fbi901588z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Huang, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chubb, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hertel, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grindey, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plunkett, W.</span><span> </span><span class="NLM_article-title">Action of 2′,2′-difluorodeoxycytidine on DNA synthesis</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">6110</span><span class="NLM_x">–</span> <span class="NLM_lpage">6117</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1718594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADyaK38XnsFOqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1991&pages=6110-6117&author=P.+Huangauthor=S.+Chubbauthor=L.+W.+Hertelauthor=G.+B.+Grindeyauthor=W.+Plunkett&title=Action+of+2%E2%80%B2%2C2%E2%80%B2-difluorodeoxycytidine+on+DNA+synthesis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Action of 2',2'-difluorodeoxycytidine on DNA synthesis</span></div><div class="casAuthors">Huang, Peng; Chubb, Sherri; Hertel, Larry W.; Grindey, Gerald B.; Plunkett, William</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6110-17</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    </div><div class="casAbstract">The action of the new deoxycytidine analog 2',2'-difluorodeoxycytidine (dFdC) on DNA synthesis was investigated in whole cells and in vitro assay systems with purified DNA polymerases.  DNA synthesis in human lymphoblastoid CEM cells was inhibited by dFdC in a concn.-dependent manner that could not be reversed by exogenous deoxynucleosides.  The analog was incorporated into cellular DNA; most of the incorporated dFdC 5'-monophosphate (dFdCMP) residues were in internucleotide linkage.  In vitro DNA primer extension assays demonstrated that dFdC 5'-triphosphate (dFdCTP) competed with deoxycytidine triphosphate for incorporation into the C sites of the growing DNA strand.  The ratios of the apparent Km values for the incorporation of dFdCTP and dCTP into a C site of M13mp19 DNA were 21.8 and 22.9 for DNA polymerases α and ε, resp.  The apparent Ki values of dFdCTP were 11.2 μM for DNA polymerase α and 14.4 μM for polymerase ε.  After dFdCMP incorporation, the primer was extended by one deoxynucleotide before a major pause in the polymn. process was obsd.  This was in contrast to the action of arabinosylcytosine 5'-triphosphate, which caused both DNA polymerases α and ε to pause at the site of incorporation.  The 3'→5' exonuclease activity of DNA polymerase ε was essentially unable to excise nucleotides from DNA contg. dFdCMP at either the 3'-end or at an internal position, whereas arabinosylcytosine monophosphate was removed from the 3'-terminus at 37% the rate for deoxynucleotides.  The cytotoxic activity of dFdC was strongly correlated with the amt. of dFdCMP incorporated into cellular DNA.  These results demonstrate qual. and quant. differences in the mol. actions of dFdC and arabinosylcytosine on DNA metab., but are consistent with an important role for such incorporation in the toxicity of dFdC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofN5itudznZLVg90H21EOLACvtfcHk0lgxQTuFUxk3fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XnsFOqsA%253D%253D&md5=2e7f4ee419ecbee58845b1a983750858</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DP.%26aulast%3DChubb%26aufirst%3DS.%26aulast%3DHertel%26aufirst%3DL.%2BW.%26aulast%3DGrindey%26aufirst%3DG.%2BB.%26aulast%3DPlunkett%26aufirst%3DW.%26atitle%3DAction%2520of%25202%25E2%2580%25B2%252C2%25E2%2580%25B2-difluorodeoxycytidine%2520on%2520DNA%2520synthesis%26jtitle%3DCancer%2520Res.%26date%3D1991%26volume%3D51%26spage%3D6110%26epage%3D6117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Heinemann, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulz, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Issels, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plunkett, W.</span><span> </span><span class="NLM_article-title">Gemcitabine: a modulator of intracellular nucleotide and deoxynucleotide metabolism</span> <span class="citation_source-journal">Semin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">11</span><span class="NLM_x">–</span> <span class="NLM_lpage">18</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=7481839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADyaK2MXpsFGntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1995&pages=11-18&author=V.+Heinemannauthor=L.+Schulzauthor=R.+D.+Isselsauthor=W.+Plunkett&title=Gemcitabine%3A+a+modulator+of+intracellular+nucleotide+and+deoxynucleotide+metabolism"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Gemcitabine: a modulator of intracellular nucleotide and deoxynucleotide metabolism</span></div><div class="casAuthors">Heinemann, Volker; Schulz, Lothar; Issels, Rolf D.; Plunkett, William</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Oncology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">4, Suppl. 11</span>),
    <span class="NLM_cas:pages">11-18</span>CODEN:
                <span class="NLM_cas:coden">SOLGAV</span>;
        ISSN:<span class="NLM_cas:issn">0093-7754</span>.
    
            (<span class="NLM_cas:orgname">Saunders</span>)
        </div><div class="casAbstract">Gemcitabine (2',2'-difluorodeoxycytidine, dFdC) is a deoxycytidine (dCyd) analog that extensively modulates intracellular CTP and dCTP metab.  In Chinese hamster ovary (CHO) cells, a 4-h exposure to gemcitabine (100 μM) reduced cellular CTP and dCTP concns. to 5.9% and 50%, resp., of control values.  Intracellular UTP concns. increased, indicating a metabolic block at CTP synthetase.  Pool sizes of ATP and GTP remained unaffected.  In contrast, a CHO mutant deficient in deoxycytidine kinase, and thus unable to accumulate dFdCTP, maintained its CTP pools under identical conditions, suggesting that the CTP pool depletion was dependent on gemcitabine phosphorylation.  Neither 100 μM arabinosylcytosine nor 5 mM hydroxyurea affected CTP levels, indicating that inhibition of DNA synthesis by analog incorporation or by depletion of deoxynucleotide triphosphate pools was not the cause of the CTP pool perturbation.  Metabolic studies demonstrated that incorporation of [3H]uridine into the UTP pool was not impaired by gemcitabine, whereas the specific activity of the CTP pools decreased as a function of increasing gemcitabine concn. and time of exposure.  Comparable results were obtained by using 3-deazauridine, a known inhibitor of CTP synthetase.  It is concluded that high cellular concns. of dFdCTP deplete cellular CTP concns. by inhibition of the dCTP pool and also may be a limiting factor for RNA synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8-eqnW776frVg90H21EOLACvtfcHk0lgxQTuFUxk3fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXpsFGntbc%253D&md5=39daaaf3c2730983939356963cce40ed</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHeinemann%26aufirst%3DV.%26aulast%3DSchulz%26aufirst%3DL.%26aulast%3DIssels%26aufirst%3DR.%2BD.%26aulast%3DPlunkett%26aufirst%3DW.%26atitle%3DGemcitabine%253A%2520a%2520modulator%2520of%2520intracellular%2520nucleotide%2520and%2520deoxynucleotide%2520metabolism%26jtitle%3DSemin.%2520Oncol.%26date%3D1995%26volume%3D22%26spage%3D11%26epage%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Gmeiner, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pon, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pourquier, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span> </span><span class="NLM_article-title">Structural basis for topoisomerase I inhibition by nucleoside analogs</span> <span class="citation_source-journal">Nucleosides, Nucleotides Nucleic Acids</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">653</span><span class="NLM_x">–</span> <span class="NLM_lpage">658</span><span class="refDoi"> DOI: 10.1081/NCN-120022604</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1081%2FNCN-120022604" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=14565246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntFChtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=653-658&author=W.+H.+Gmeinerauthor=S.+Yuauthor=R.+T.+Ponauthor=P.+Pourquierauthor=Y.+Pommier&title=Structural+basis+for+topoisomerase+I+inhibition+by+nucleoside+analogs&doi=10.1081%2FNCN-120022604"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Topoisomerase I Inhibition by Nucleoside Analogs</span></div><div class="casAuthors">Gmeiner, William H.; Yu, Shuyuan; Pon, Richard T.; Pourquier, Philippe; Pommier, Yves</div><div class="citationInfo"><span class="NLM_cas:title">Nucleosides, Nucleotides & Nucleic Acids</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5-8</span>),
    <span class="NLM_cas:pages">653-658</span>CODEN:
                <span class="NLM_cas:coden">NNNAFY</span>;
        ISSN:<span class="NLM_cas:issn">1525-7770</span>.
    
            (<span class="NLM_cas:orgname">Marcel Dekker, Inc.</span>)
        </div><div class="casAbstract">A review.  Nucleoside analogs such as 1-β-D-arabinofuranosyl cytidine (AraC) and 2', 2'-difluoro deoxycytidine (dFdC) are important components of the anticancer chemotherapeutic arsenal and are among the most effective anticancer drugs currently available.  Although both AraCTP and dFdCTP impede DNA replication through pausing of DNA polymerases, both nucleoside analogs are ultimately incorporated into replicated DNA and interfere in DNA-mediated processes.  Our labs. are investigating the structural basis for the poisoning of topoisomerase I (top1) due to antipyrimidine incorporation into duplex DNA.  We recently reported that both AraC and dFdC induce formation of top1 cleavage complexes, and poisoning of top1 contributes to the anticancer activities of both these drugs.  Recent NMR and thermodn. studies from our labs. provide insight into the mechanism by which AraC and dFdC poison top 1.  NMR studies from our labs. have revealed that the arabinosyl sugar of AraC adopted a C2'-endo conformation.  Although this is a B-type sugar pucker characteristic of duplex DNA, the conformation is rigid, and this lack of flexibility probably contributes to inhibition of the religation step of the top 1 reaction.  In contrast to AraC, NMR studies revealed dFdC adopted a C3' endo sugar pucker characteristic of RNA, rather than DNA duplexes.  DFdC substitution enhanced formation of top1 cleavage complexes, but did not inhibit religation.  The enhancement of top1 cleavage complexes most likely results from a combination of conformational and electrostatic effects.  The structural effects of dFdC and AraC are being further investigated in duplex DNA with well-defined top1 cleavage sites to analyze more specifically how these structural perturbations lead to enzyme poisoning.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqU2NZ1ttFrlrVg90H21EOLACvtfcHk0lgxQTuFUxk3fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntFChtrs%253D&md5=c950a0a08d9777b215b38cfc639bc4db</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1081%2FNCN-120022604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1081%252FNCN-120022604%26sid%3Dliteratum%253Aachs%26aulast%3DGmeiner%26aufirst%3DW.%2BH.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DPon%26aufirst%3DR.%2BT.%26aulast%3DPourquier%26aufirst%3DP.%26aulast%3DPommier%26aufirst%3DY.%26atitle%3DStructural%2520basis%2520for%2520topoisomerase%2520I%2520inhibition%2520by%2520nucleoside%2520analogs%26jtitle%3DNucleosides%252C%2520Nucleotides%2520Nucleic%2520Acids%26date%3D2003%26volume%3D22%26spage%3D653%26epage%3D658%26doi%3D10.1081%2FNCN-120022604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Zhou, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yip, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaur, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Synold, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horne, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, Y.</span><span> </span><span class="NLM_article-title">A small-molecule blocking ribonucleotide reductase holoenzyme formation inhibits cancer cell growth and overcomes drug resistance</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">6484</span><span class="NLM_x">–</span> <span class="NLM_lpage">6493</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-1094</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1158%2F0008-5472.CAN-13-1094" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=6484-6493&author=B.+Zhouauthor=L.+Suauthor=S.+Huauthor=W.+Huauthor=M.+L.+Yipauthor=J.+Wuauthor=S.+Gaurauthor=D.+L.+Smithauthor=Y.+C.+Yuanauthor=T.+W.+Synoldauthor=D.+Horneauthor=Y.+Yen&title=A+small-molecule+blocking+ribonucleotide+reductase+holoenzyme+formation+inhibits+cancer+cell+growth+and+overcomes+drug+resistance&doi=10.1158%2F0008-5472.CAN-13-1094"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-1094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-1094%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DSu%26aufirst%3DL.%26aulast%3DHu%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DW.%26aulast%3DYip%26aufirst%3DM.%2BL.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DGaur%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DD.%2BL.%26aulast%3DYuan%26aufirst%3DY.%2BC.%26aulast%3DSynold%26aufirst%3DT.%2BW.%26aulast%3DHorne%26aufirst%3DD.%26aulast%3DYen%26aufirst%3DY.%26atitle%3DA%2520small-molecule%2520blocking%2520ribonucleotide%2520reductase%2520holoenzyme%2520formation%2520inhibits%2520cancer%2520cell%2520growth%2520and%2520overcomes%2520drug%2520resistance%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D6484%26epage%3D6493%26doi%3D10.1158%2F0008-5472.CAN-13-1094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Brown, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canellakis, Z. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundin, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reichard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thelander, L.</span><span> </span><span class="NLM_article-title">Ribonucleoside diphosphate reductase. Purification of the two subunits, proteins B1 and B2</span> <span class="citation_source-journal">Eur. J. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1969</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">561</span><span class="NLM_x">–</span> <span class="NLM_lpage">573</span><span class="refDoi"> DOI: 10.1111/j.1432-1033.1969.tb00646.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1111%2Fj.1432-1033.1969.tb00646.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=4896737" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADyaF1MXks1Ols7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1969&pages=561-573&author=N.+C.+Brownauthor=Z.+N.+Canellakisauthor=B.+Lundinauthor=P.+Reichardauthor=L.+Thelander&title=Ribonucleoside+diphosphate+reductase.+Purification+of+the+two+subunits%2C+proteins+B1+and+B2&doi=10.1111%2Fj.1432-1033.1969.tb00646.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Ribonucleoside diphosphate reductase.  Purification of the two subunits, proteins B1 and B2</span></div><div class="casAuthors">Brown, Neal Curtis; Canellakis, Z. N.; Lundin, B.; Reichard, Peter; Thelander, Lars</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Biochemistry</span>
        (<span class="NLM_cas:date">1969</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">561-73</span>CODEN:
                <span class="NLM_cas:coden">EJBCAI</span>;
        ISSN:<span class="NLM_cas:issn">0014-2956</span>.
    </div><div class="casAbstract">Ribonucleoside diphosphate reductase from Escherichia coli B consists of 2 nonidentical subunits, called proteins B1 and B2.  After purification from exts. of a partially derepressed thymine- requiring mutant, both subunits were obtained in essentially pure form.  Criteria of purity included the demonstration of const. specific enzyme activity in peaks from final chromatographic or electrophoretic steps and the behavior of the proteins during ultracentrifugation and anal. gel electrophoresis.  Detns. of mol. wts. by sedimentation equil. centrifugation gave values between 160,000 and 200,000 for protein B1 and 78,000 for protein B2.  Evidence was obtained for some dissocn. of protein B1 during centrifugation.  Sep., each subunit was completely inactive; together, under the proper conditions, they catalyzed the redn. of the 5'-diphosphates of cytidine, uridine, adenosine, and guanosine to the corresponding deoxyribonucleotides.  With the exception of deoxyadenosine 5'-triphosphate, the regulatory effects of nucleoside triphosphates were identical to those observed earlier with less pure prepns. of enzyme.  The general inhibitory effect of deoxyadenosine 5'-triphosphate at concns. higher than 1μM was obtained as before.  In addn., lower concns. of this nucleotide had a novel stimulatory effect on the redn. of CDP.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5SRt2D08lprVg90H21EOLACvtfcHk0lgdoxFfFHTIBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF1MXks1Ols7s%253D&md5=e41cb10135d1abc3918c947683dd2e06</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1111%2Fj.1432-1033.1969.tb00646.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1432-1033.1969.tb00646.x%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DN.%2BC.%26aulast%3DCanellakis%26aufirst%3DZ.%2BN.%26aulast%3DLundin%26aufirst%3DB.%26aulast%3DReichard%26aufirst%3DP.%26aulast%3DThelander%26aufirst%3DL.%26atitle%3DRibonucleoside%2520diphosphate%2520reductase.%2520Purification%2520of%2520the%2520two%2520subunits%252C%2520proteins%2520B1%2520and%2520B2%26jtitle%3DEur.%2520J.%2520Biochem.%26date%3D1969%26volume%3D9%26spage%3D561%26epage%3D573%26doi%3D10.1111%2Fj.1432-1033.1969.tb00646.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Brown, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reichard, P.</span><span> </span><span class="NLM_article-title">Role of effector binding in allosteric control of ribonucleoside diphosphate reductase</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1969</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">39</span><span class="NLM_x">–</span> <span class="NLM_lpage">55</span><span class="refDoi"> DOI: 10.1016/0022-2836(69)90056-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1016%2F0022-2836%2869%2990056-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=1969&pages=39-55&author=N.+C.+Brownauthor=P.+Reichard&title=Role+of+effector+binding+in+allosteric+control+of+ribonucleoside+diphosphate+reductase&doi=10.1016%2F0022-2836%2869%2990056-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2869%2990056-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252869%252990056-4%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DN.%2BC.%26aulast%3DReichard%26aufirst%3DP.%26atitle%3DRole%2520of%2520effector%2520binding%2520in%2520allosteric%2520control%2520of%2520ribonucleoside%2520diphosphate%2520reductase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1969%26volume%3D46%26spage%3D39%26epage%3D55%26doi%3D10.1016%2F0022-2836%2869%2990056-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Eriksson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uhlin, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramaswamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ekberg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Regnstrom, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sjoberg, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eklund, H.</span><span> </span><span class="NLM_article-title">Binding of allosteric effectors to ribonucleotide reductase protein R1: reduction of active-site cysteines promotes substrate binding</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1077</span><span class="NLM_x">–</span> <span class="NLM_lpage">1092</span><span class="refDoi"> DOI: 10.1016/S0969-2126(97)00259-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1016%2FS0969-2126%2897%2900259-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=9309223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADyaK2sXmt1Wgsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1997&pages=1077-1092&author=M.+Erikssonauthor=U.+Uhlinauthor=S.+Ramaswamyauthor=M.+Ekbergauthor=K.+Regnstromauthor=B.+M.+Sjobergauthor=H.+Eklund&title=Binding+of+allosteric+effectors+to+ribonucleotide+reductase+protein+R1%3A+reduction+of+active-site+cysteines+promotes+substrate+binding&doi=10.1016%2FS0969-2126%2897%2900259-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Binding of allosteric effectors to ribonucleotide reductase protein R1: reduction of active-site cysteines promotes substrate binding</span></div><div class="casAuthors">Eriksson, Mathias; Uhlin, Ulla; Ramaswamy, S.; Ekberg, Monica; Regnstrom, Karin; Sjoberg, Britt-Marie; Eklund, Hans</div><div class="citationInfo"><span class="NLM_cas:title">Structure (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1077-1092</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Current Biology</span>)
        </div><div class="casAbstract">Ribonucleotide reductase (RNR) is an essential enzyme in DNA synthesis, catalyzing all de novo synthesis of deoxyribonucleotides.  The enzyme comprises two dimers, termed R1 and R2, and contains the redox active cysteine residues, Cys462 and Cys225.  The redn. of ribonucleotides to deoxyribonucleotides involves the transfer of free radicals.  The pathway for the radical has previously been suggested from crystallog. results, and is supported by site-directed mutagenesis studies.  Most RNRs are allosterically regulated through two different nucleotide-binding sites: one site controls general activity and the other controls substrate specificity.  Our aim has been to crystallog. demonstrate substrate binding and to locate the two effector-binding sites.  We report here the first crystal structure of RNR R1 in a reduced form.  The structure shows that upon redn. of the redox active cysteines, the sulfur atom of Cys462 becomes deeply buried.  The more accessible Cys225 moves to the former position of Cys462 making room for the substrate.  In addn., the structures of R1 in complexes with effector, effector analog and effector plus substrate provide information about these binding sites.  The substrate GDP binds in a cleft between two domains with its β-phosphate bound to the N termini of two helixes; the ribose forms hydrogen bonds to conserved residues.  Binding of dTTP at the allosteric substrate specificity site stabilizes three loops close to the dimer interface and the active site, whereas the general allosteric binding site is positioned far from the active site.  Binding of substrate at the active site of the enzyme is structurally regulated in two ways: binding of the correct substrate is regulated by the binding of allosteric effectors, and binding of the actual substrate occurs primarily when the active-site cysteines are reduced.  One of the loops stabilized upon binding of dTTP participates in the formation of the substrate-binding site through direct interaction with the nucleotide base.  The general allosteric effector site, located far from the active site, appears to regulate subunit interactions within the holoenzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZ9X-VPWPN7Vg90H21EOLACvtfcHk0lgdoxFfFHTIBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmt1Wgsrc%253D&md5=3b1cb456c53f6d9ccd4f303e0f2c330d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2897%2900259-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252897%252900259-1%26sid%3Dliteratum%253Aachs%26aulast%3DEriksson%26aufirst%3DM.%26aulast%3DUhlin%26aufirst%3DU.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DEkberg%26aufirst%3DM.%26aulast%3DRegnstrom%26aufirst%3DK.%26aulast%3DSjoberg%26aufirst%3DB.%2BM.%26aulast%3DEklund%26aufirst%3DH.%26atitle%3DBinding%2520of%2520allosteric%2520effectors%2520to%2520ribonucleotide%2520reductase%2520protein%2520R1%253A%2520reduction%2520of%2520active-site%2520cysteines%2520promotes%2520substrate%2520binding%26jtitle%3DStructure%26date%3D1997%26volume%3D5%26spage%3D1077%26epage%3D1092%26doi%3D10.1016%2FS0969-2126%2897%2900259-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Larsson, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eliasson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reichard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Logan, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordlund, P.</span><span> </span><span class="NLM_article-title">Structural mechanism of allosteric substrate specificity regulation in a ribonucleotide reductase</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1142</span><span class="NLM_x">–</span> <span class="NLM_lpage">1149</span><span class="refDoi"> DOI: 10.1038/nsmb838</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1038%2Fnsmb838" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=15475969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpt1ejsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=1142-1149&author=K.+M.+Larssonauthor=A.+Jordanauthor=R.+Eliassonauthor=P.+Reichardauthor=D.+T.+Loganauthor=P.+Nordlund&title=Structural+mechanism+of+allosteric+substrate+specificity+regulation+in+a+ribonucleotide+reductase&doi=10.1038%2Fnsmb838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structural mechanism of allosteric substrate specificity regulation in a ribonucleotide reductase</span></div><div class="casAuthors">Larsson, Karl-Magnus; Jordan, Albert; Eliasson, Rolf; Reichard, Peter; Logan, Derek T.; Nordlund, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1142-1149</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Ribonucleotide reductases (RNRs) catalyze the redn. of ribonucleotides into deoxyribonucleotides, which constitute the precursor pools used for DNA synthesis and repair.  Imbalances in these pools increase mutational rates and are detrimental to the cell.  Balanced precursor pools are maintained primarily through the regulation of the RNR substrate specificity.  Here, the mol. mechanism of the allosteric substrate specificity regulation is revealed through the structures of a dimeric coenzyme B12-dependent RNR from Thermotoga maritima, both in complexes with four effector-substrate nucleotide pairs and in three complexes with only effector.  The mechanism is based on the flexibility of loop 2, a key structural element, which forms a bridge between the specificity effector and substrate nucleotides.  Substrate specificity is achieved as different effectors and their cognate substrates stabilize specific discrete loop 2 conformations.  The mechanism of substrate specificity regulation is probably general for most class I and class II RNRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkLGAmhsPRF7Vg90H21EOLACvtfcHk0lgbhvYXh5nSCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpt1ejsL0%253D&md5=07cdbaa804737ec8af326306b1e3b2b8</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnsmb838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb838%26sid%3Dliteratum%253Aachs%26aulast%3DLarsson%26aufirst%3DK.%2BM.%26aulast%3DJordan%26aufirst%3DA.%26aulast%3DEliasson%26aufirst%3DR.%26aulast%3DReichard%26aufirst%3DP.%26aulast%3DLogan%26aufirst%3DD.%2BT.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DStructural%2520mechanism%2520of%2520allosteric%2520substrate%2520specificity%2520regulation%2520in%2520a%2520ribonucleotide%2520reductase%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D1142%26epage%3D1149%26doi%3D10.1038%2Fnsmb838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Xu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faber, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uchiki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairman, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Racca, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dealwis, C.</span><span> </span><span class="NLM_article-title">Structures of eukaryotic ribonucleotide reductase I provide insights into dNTP regulation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">4022</span><span class="NLM_x">–</span> <span class="NLM_lpage">4027</span><span class="refDoi"> DOI: 10.1073/pnas.0600443103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1073%2Fpnas.0600443103" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=4022-4027&author=H.+Xuauthor=C.+Faberauthor=T.+Uchikiauthor=J.+W.+Fairmanauthor=J.+Raccaauthor=C.+Dealwis&title=Structures+of+eukaryotic+ribonucleotide+reductase+I+provide+insights+into+dNTP+regulation&doi=10.1073%2Fpnas.0600443103"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0600443103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0600443103%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DH.%26aulast%3DFaber%26aufirst%3DC.%26aulast%3DUchiki%26aufirst%3DT.%26aulast%3DFairman%26aufirst%3DJ.%2BW.%26aulast%3DRacca%26aufirst%3DJ.%26aulast%3DDealwis%26aufirst%3DC.%26atitle%3DStructures%2520of%2520eukaryotic%2520ribonucleotide%2520reductase%2520I%2520provide%2520insights%2520into%2520dNTP%2520regulation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D4022%26epage%3D4027%26doi%3D10.1073%2Fpnas.0600443103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Kumar, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdulovic, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viberg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nilsson, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunkel, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chabes, A.</span><span> </span><span class="NLM_article-title">Mechanisms of mutagenesis in vivo due to imbalanced dNTP pools</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">1360</span><span class="NLM_x">–</span> <span class="NLM_lpage">1371</span><span class="refDoi"> DOI: 10.1093/nar/gkq829</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1093%2Fnar%2Fgkq829" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=20961955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3MXis1Chs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=1360-1371&author=D.+Kumarauthor=A.+L.+Abdulovicauthor=J.+Vibergauthor=A.+K.+Nilssonauthor=T.+A.+Kunkelauthor=A.+Chabes&title=Mechanisms+of+mutagenesis+in+vivo+due+to+imbalanced+dNTP+pools&doi=10.1093%2Fnar%2Fgkq829"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of mutagenesis in vivo due to imbalanced dNTP pools</span></div><div class="casAuthors">Kumar, Dinesh; Abdulovic, Amy L.; Viberg, Joergen; Nilsson, Anna Karin; Kunkel, Thomas A.; Chabes, Andrei</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1360-1371</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The mechanisms by which imbalanced dNTPs induce mutations have been well characterized within a test tube, but not in vivo.  We have examd. mechanisms by which dNTP imbalances induce genome instability in strains of Saccharomyces cerevisiae with different amino acid substitutions in Rnr1, the large subunit of ribonucleotide reductase.  These strains have different dNTP imbalances that correlate with elevated CAN1 mutation rates, with both substitution and insertion-deletion rates increasing by 10- to 300-fold.  The locations of the mutations in a strain with elevated dTTP and dCTP are completely different from those in a strain with elevated dATP and dGTP.  Thus, imbalanced dNTPs reduce genome stability in a manner that is highly dependent on the nature and degree of the imbalance.  Mutagenesis is enhanced despite the availability of proofreading and mismatch repair.  The mutations can be explained by imbalanced dNTP-induced increases in misinsertion, strand misalignment and mismatch extension at the expense of proofreading.  This implies that the relative dNTP concns. measured in exts. are truly available to a replication fork in vivo.  An interesting mutational strand bias is obsd. in one rnr1 strain, suggesting that the S-phase checkpoint selectively prevents replication errors during leading strand replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEZ-Es5BD9hLVg90H21EOLACvtfcHk0lgbhvYXh5nSCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXis1Chs74%253D&md5=1d61a978fd83aae91563ab8f3994afd2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq829&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq829%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DD.%26aulast%3DAbdulovic%26aufirst%3DA.%2BL.%26aulast%3DViberg%26aufirst%3DJ.%26aulast%3DNilsson%26aufirst%3DA.%2BK.%26aulast%3DKunkel%26aufirst%3DT.%2BA.%26aulast%3DChabes%26aufirst%3DA.%26atitle%3DMechanisms%2520of%2520mutagenesis%2520in%2520vivo%2520due%2520to%2520imbalanced%2520dNTP%2520pools%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3D1360%26epage%3D1371%26doi%3D10.1093%2Fnar%2Fgkq829" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Ahmad, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaushal, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wan, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wijerathna, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">An, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dealwis, C. G.</span><span> </span><span class="NLM_article-title">Role of arginine 293 and glutamine 288 in communication between catalytic and allosteric sites in yeast ribonucleotide reductase</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">419</span><span class="NLM_x">, </span> <span class="NLM_fpage">315</span><span class="NLM_x">–</span> <span class="NLM_lpage">329</span><span class="refDoi"> DOI: 10.1016/j.jmb.2012.03.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1016%2Fj.jmb.2012.03.014" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=419&publication_year=2012&pages=315-329&author=M.+F.+Ahmadauthor=P.+S.+Kaushalauthor=Q.+Wanauthor=S.+R.+Wijerathnaauthor=X.+Anauthor=M.+Huangauthor=C.+G.+Dealwis&title=Role+of+arginine+293+and+glutamine+288+in+communication+between+catalytic+and+allosteric+sites+in+yeast+ribonucleotide+reductase&doi=10.1016%2Fj.jmb.2012.03.014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2012.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2012.03.014%26sid%3Dliteratum%253Aachs%26aulast%3DAhmad%26aufirst%3DM.%2BF.%26aulast%3DKaushal%26aufirst%3DP.%2BS.%26aulast%3DWan%26aufirst%3DQ.%26aulast%3DWijerathna%26aufirst%3DS.%2BR.%26aulast%3DAn%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DDealwis%26aufirst%3DC.%2BG.%26atitle%3DRole%2520of%2520arginine%2520293%2520and%2520glutamine%2520288%2520in%2520communication%2520between%2520catalytic%2520and%2520allosteric%2520sites%2520in%2520yeast%2520ribonucleotide%2520reductase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D419%26spage%3D315%26epage%3D329%26doi%3D10.1016%2Fj.jmb.2012.03.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Uhlin, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eklund, H.</span><span> </span><span class="NLM_article-title">Structure of ribonucleotide reductase protein R1</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">370</span><span class="NLM_x">, </span> <span class="NLM_fpage">533</span><span class="NLM_x">–</span> <span class="NLM_lpage">539</span><span class="refDoi"> DOI: 10.1038/370533a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1038%2F370533a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=8052308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADyaK2cXmt1CqsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=370&publication_year=1994&pages=533-539&author=U.+Uhlinauthor=H.+Eklund&title=Structure+of+ribonucleotide+reductase+protein+R1&doi=10.1038%2F370533a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of ribonucleotide reductase protein R1</span></div><div class="casAuthors">Uhlin, Ulla; Eklund, Hans</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">370</span>
        (<span class="NLM_cas:issue">6490</span>),
    <span class="NLM_cas:pages">533-9</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The crystal structure ribonucleotide reductase (I) subunit R1 (in complex with subunit R2) at 2.5 Å is reported.  The 3-dimensional structure of the R2 subunit was previously reported and refined at 2.2 Å.  The R2 tyrosyl radical-based I reaction involves 5 cysteines.  Two redox-active R1 cysteines (Cys-225 and Cys-462) are located at adjacent antiparallel strands in a new type of 10-stranded α/β-barrel, and 2 others (Cys-754 and Cys-759) at the C-terminal end in a flexible arm.  The 5th cysteine (Cys-439), in a loop in the center of the barrel, is positioned to initiate the radical reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW-kijq3QSZbVg90H21EOLACvtfcHk0lgbhvYXh5nSCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmt1CqsLc%253D&md5=89f9cdf8318bf918d61ef63012f24073</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2F370533a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F370533a0%26sid%3Dliteratum%253Aachs%26aulast%3DUhlin%26aufirst%3DU.%26aulast%3DEklund%26aufirst%3DH.%26atitle%3DStructure%2520of%2520ribonucleotide%2520reductase%2520protein%2520R1%26jtitle%3DNature%26date%3D1994%26volume%3D370%26spage%3D533%26epage%3D539%26doi%3D10.1038%2F370533a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Kashlan, O. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lear, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooperman, B. S.</span><span> </span><span class="NLM_article-title">A comprehensive model for the allosteric regulation of mammalian ribonucleotide reductase. Functional consequences of ATP- and dATP-induced oligomerization of the large subunit</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">462</span><span class="NLM_x">–</span> <span class="NLM_lpage">474</span><span class="refDoi"> DOI: 10.1021/bi011653a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi011653a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD3MXovF2rtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=462-474&author=O.+B.+Kashlanauthor=C.+P.+Scottauthor=J.+D.+Learauthor=B.+S.+Cooperman&title=A+comprehensive+model+for+the+allosteric+regulation+of+mammalian+ribonucleotide+reductase.+Functional+consequences+of+ATP-+and+dATP-induced+oligomerization+of+the+large+subunit&doi=10.1021%2Fbi011653a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A Comprehensive Model for the Allosteric Regulation of Mammalian Ribonucleotide Reductase. Functional Consequences of ATP- and dATP-Induced Oligomerization of the Large Subunit</span></div><div class="casAuthors">Kashlan, Ossama B.; Scott, Charles P.; Lear, James D.; Cooperman, Barry S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-474</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Redn. of NDPs by murine ribonucleotide reductase (mRR) requires catalytic (mR1) and free radical-contg. (mR2) subunits and is regulated by nucleoside triphosphate allosteric effectors.  Here we present a new, comprehensive, and quant. model for allosteric control of mRR enzymic activity based on mol. mass, ligand binding, and enzyme activity studies.  In this model, nucleotide binding to the specificity site (s-site) drives formation of an active R12R22 dimer, ATP or dATP binding to the adenine-specific site (a-site) results in formation of an inactive tetramer, and ATP binding to the newly described hexamerization site (h-site) drives formation of active R16R26 hexamer.  In contrast, an earlier phenomenol. model [Thelander, L., and Reichard, P. (1979) Annu. Rev. Biochem. 67, 71-98] (the "RT" model) ignores aggregation state changes and mistakenly rationalizes ATP activation vs. dATP inhibition as reflecting different functional consequences of ATP vs. dATP binding to the a-site.  Our results suggest that the R16R26 heterohexamer is the major active form of the enzyme in mammalian cells, and that the ATP concn. is the primary modulator of enzyme activity, coupling the rate of DNA biosynthesis with the energetic state of the cell.  Using the crystal structure of the Escherichia coli R1 hexamer as a model for the mR1 hexamer, a scheme is presented that rationalizes the slow isomerization of the tetramer form and suggests an explanation for the low enzymic activity of tetramers complexed with R2.  The similar specific activities of R12R22 and R16R26 are inconsistent with a proposed model for R22 docking with R12 [Uhlin, U., and Eklund, H. (1994) Nature 370, 533-539], and an alternative is suggested.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT8Nd4PPq087Vg90H21EOLACvtfcHk0lhTRo9rslTrwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXovF2rtLg%253D&md5=f1cd48444fc897f38017b562d49fc1e5</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fbi011653a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi011653a%26sid%3Dliteratum%253Aachs%26aulast%3DKashlan%26aufirst%3DO.%2BB.%26aulast%3DScott%26aufirst%3DC.%2BP.%26aulast%3DLear%26aufirst%3DJ.%2BD.%26aulast%3DCooperman%26aufirst%3DB.%2BS.%26atitle%3DA%2520comprehensive%2520model%2520for%2520the%2520allosteric%2520regulation%2520of%2520mammalian%2520ribonucleotide%2520reductase.%2520Functional%2520consequences%2520of%2520ATP-%2520and%2520dATP-induced%2520oligomerization%2520of%2520the%2520large%2520subunit%26jtitle%3DBiochemistry%26date%3D2002%26volume%3D41%26spage%3D462%26epage%3D474%26doi%3D10.1021%2Fbi011653a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Cooperman, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kashlan, O. B.</span><span> </span><span class="NLM_article-title">A comprehensive model for the allosteric regulation of Class Ia ribonucleotide reductases</span> <span class="citation_source-journal">Adv. Enzyme Regul.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span><span class="refDoi"> DOI: 10.1016/S0065-2571(02)00035-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1016%2FS0065-2571%2802%2900035-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2003&pages=167-82&author=B.+S.+Coopermanauthor=O.+B.+Kashlan&title=A+comprehensive+model+for+the+allosteric+regulation+of+Class+Ia+ribonucleotide+reductases&doi=10.1016%2FS0065-2571%2802%2900035-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS0065-2571%2802%2900035-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0065-2571%252802%252900035-3%26sid%3Dliteratum%253Aachs%26aulast%3DCooperman%26aufirst%3DB.%2BS.%26aulast%3DKashlan%26aufirst%3DO.%2BB.%26atitle%3DA%2520comprehensive%2520model%2520for%2520the%2520allosteric%2520regulation%2520of%2520Class%2520Ia%2520ribonucleotide%2520reductases%26jtitle%3DAdv.%2520Enzyme%2520Regul.%26date%3D2003%26volume%3D43%26spage%3D167%26epage%3D82%26doi%3D10.1016%2FS0065-2571%2802%2900035-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Rofougaran, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vodnala, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hofer, A.</span><span> </span><span class="NLM_article-title">Enzymatically active mammalian ribonucleotide reductase exists primarily as an alpha6beta2 octamer</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">27705</span><span class="NLM_x">–</span> <span class="NLM_lpage">27711</span><span class="refDoi"> DOI: 10.1074/jbc.M605573200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1074%2Fjbc.M605573200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=27705-27711&author=R.+Rofougaranauthor=M.+Vodnalaauthor=A.+Hofer&title=Enzymatically+active+mammalian+ribonucleotide+reductase+exists+primarily+as+an+alpha6beta2+octamer&doi=10.1074%2Fjbc.M605573200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M605573200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M605573200%26sid%3Dliteratum%253Aachs%26aulast%3DRofougaran%26aufirst%3DR.%26aulast%3DVodnala%26aufirst%3DM.%26aulast%3DHofer%26aufirst%3DA.%26atitle%3DEnzymatically%2520active%2520mammalian%2520ribonucleotide%2520reductase%2520exists%2520primarily%2520as%2520an%2520alpha6beta2%2520octamer%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D27705%26epage%3D27711%26doi%3D10.1074%2Fjbc.M605573200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohman, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stubbe, J.</span><span> </span><span class="NLM_article-title">Enhanced subunit interactions with gemcitabine-5′-diphosphate inhibit ribonucleotide reductases</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">14324</span><span class="NLM_x">–</span> <span class="NLM_lpage">14329</span><span class="refDoi"> DOI: 10.1073/pnas.0706803104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1073%2Fpnas.0706803104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=17726094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVGhtbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=14324-14329&author=J.+Wangauthor=G.+J.+Lohmanauthor=J.+Stubbe&title=Enhanced+subunit+interactions+with+gemcitabine-5%E2%80%B2-diphosphate+inhibit+ribonucleotide+reductases&doi=10.1073%2Fpnas.0706803104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced subunit interactions with gemcitabine-5'-diphosphate inhibit ribonucleotide reductases</span></div><div class="casAuthors">Wang, Jun; Lohman, Gregory J. S.; Stubbe, JoAnne</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">14324-14329</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Ribonucleotide reductases (RNRs) catalyze the conversion of nucleotides to deoxynucleotides in all organisms.  The class I RNRs are composed of two subunits, α and β, with proposed quaternary structures of α2β2, α6β2, or α6β6, depending on the organism.  The t subunits bind the nucleoside diphosphate substrates and the dNTP/ATP allosteric effectors that govern specificity and turnover.  The β2 subunit houses the diferric Y. (1 radical per β2) cofactor that is required to initiate nucleotide redn. 2',2'-Difluoro-2'-deoxycytidine (F2C) is presently used clin. in a variety of cancer treatments and the 5'-diphosphorylated F2C (F2CDP) is a potent inhibitor of RNRs.  The studies with [1'-3H]-F2CDP and [5-3H]-F2CDP have established that F2CDP is a substoichiometric mechanism based inhibitor (0.5 equiv F2CDP/α) of both the Escherichia coli and the human RNRs in the presence of reductant.  Inactivation is caused by covalent labeling of RNR by the sugar of F2CDP (0.5 equiv/α) and is accompanied by release of 0.5 equiv cytosine/α.  Inactivation also results in loss of 40% of β2 activity.  Studies using size exclusion chromatog. reveal that in the E. coli RNR, an α2β2 tight complex is generated subsequent to enzyme inactivation by F2CDP, whereas in the human RNR, an α6β6 tight complex is generated.  Isolation of these complexes establishes that the weak interactions of the subunits in the absence of nucleotides are substantially increased in the presence of F2CDP and ATP.  This information and the proposed asymmetry between the interactions of αnβn provide an explanation for complete inactivation of RNR with substoichiometric amts. of F2CDP.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoABu-J8ZIg4bVg90H21EOLACvtfcHk0lhTRo9rslTrwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVGhtbbP&md5=28970ac58ef9ada8c69ff4f88bb7704c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0706803104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0706803104%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLohman%26aufirst%3DG.%2BJ.%26aulast%3DStubbe%26aufirst%3DJ.%26atitle%3DEnhanced%2520subunit%2520interactions%2520with%2520gemcitabine-5%25E2%2580%25B2-diphosphate%2520inhibit%2520ribonucleotide%2520reductases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D14324%26epage%3D14329%26doi%3D10.1073%2Fpnas.0706803104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Fairman, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wijerathna, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmad, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakano, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jha, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prendergast, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Welin, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flodin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordlund, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walz, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dealwis, C. G.</span><span> </span><span class="NLM_article-title">Structural basis for allosteric regulation of human ribonucleotide reductase by nucleotide-induced oligomerization</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">316</span><span class="NLM_x">–</span> <span class="NLM_lpage">322</span><span class="refDoi"> DOI: 10.1038/nsmb.2007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1038%2Fnsmb.2007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=21336276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1aisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=316-322&author=J.+W.+Fairmanauthor=S.+R.+Wijerathnaauthor=M.+F.+Ahmadauthor=H.+Xuauthor=R.+Nakanoauthor=S.+Jhaauthor=J.+Prendergastauthor=R.+M.+Welinauthor=S.+Flodinauthor=A.+Roosauthor=P.+Nordlundauthor=Z.+Liauthor=T.+Walzauthor=C.+G.+Dealwis&title=Structural+basis+for+allosteric+regulation+of+human+ribonucleotide+reductase+by+nucleotide-induced+oligomerization&doi=10.1038%2Fnsmb.2007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for allosteric regulation of human ribonucleotide reductase by nucleotide-induced oligomerization</span></div><div class="casAuthors">Fairman, James Wesley; Wijerathna, Sanath Ranjan; Ahmad, Md Faiz; Xu, Hai; Nakano, Ryo; Jha, Shalini; Prendergast, Jay; Welin, R. Martin; Flodin, Susanne; Roos, Annette; Nordlund, Par; Li, Zongli; Walz, Thomas; Dealwis, Chris Godfrey</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">316-322</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Ribonucleotide reductase (RR) is an αnβn (RR1-RR2) complex that maintains balanced dNTP pools by reducing NDPs to dNDPs.  RR1 is the catalytic subunit, and RR2 houses the free radical required for catalysis.  RR is allosterically regulated by its activator ATP and its inhibitor dATP, which regulate RR activity by inducing oligomerization of RR1.  Here, we report the first X-ray structures of human RR1 bound to TTP alone, dATP alone, TTP-GDP, TTP-ATP, and TTP-dATP.  These structures provide insights into regulation of RR by ATP or dATP.  At physiol. dATP concns., RR1 forms inactive hexamers.  We detd. the first X-ray structure of the RR1-dATP hexamer and used single-particle electron microscopy to visualize the α6-ββ'-dATP holocomplex.  Site-directed mutagenesis and functional assays confirm that hexamerization is a prerequisite for inhibition by dATP.  Our data indicate a mechanism for regulating RR activity by dATP-induced oligomerization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeexjNT7fmrLVg90H21EOLACvtfcHk0lh60eEqGzY7Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1aisbk%253D&md5=e7e97e869a51f1724f5ccc2d4eaea116</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2007%26sid%3Dliteratum%253Aachs%26aulast%3DFairman%26aufirst%3DJ.%2BW.%26aulast%3DWijerathna%26aufirst%3DS.%2BR.%26aulast%3DAhmad%26aufirst%3DM.%2BF.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DNakano%26aufirst%3DR.%26aulast%3DJha%26aufirst%3DS.%26aulast%3DPrendergast%26aufirst%3DJ.%26aulast%3DWelin%26aufirst%3DR.%2BM.%26aulast%3DFlodin%26aufirst%3DS.%26aulast%3DRoos%26aufirst%3DA.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DWalz%26aufirst%3DT.%26aulast%3DDealwis%26aufirst%3DC.%2BG.%26atitle%3DStructural%2520basis%2520for%2520allosteric%2520regulation%2520of%2520human%2520ribonucleotide%2520reductase%2520by%2520nucleotide-induced%2520oligomerization%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D316%26epage%3D322%26doi%3D10.1038%2Fnsmb.2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Aye, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stubbe, J.</span><span> </span><span class="NLM_article-title">Clofarabine 5′-di and -triphosphates inhibit human ribonucleotide reductase by altering the quaternary structure of its large subunit</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">9815</span><span class="NLM_x">–</span> <span class="NLM_lpage">9820</span><span class="refDoi"> DOI: 10.1073/pnas.1013274108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1073%2Fpnas.1013274108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=21628579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvVyjt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=9815-9820&author=Y.+Ayeauthor=J.+Stubbe&title=Clofarabine+5%E2%80%B2-di+and+-triphosphates+inhibit+human+ribonucleotide+reductase+by+altering+the+quaternary+structure+of+its+large+subunit&doi=10.1073%2Fpnas.1013274108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Clofarabine 5'-di and -triphosphates inhibit human ribonucleotide reductase by altering the quaternary structure of its large subunit</span></div><div class="casAuthors">Aye, Yimon; Stubbe, Joanne</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9815-9820, S9815/1-S9815/11</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Human ribonucleotide reductases (hRNRs) catalyze the conversion of nucleotides to deoxynucleotides and are composed of α and β-subunits that form active αnβm (n, m = 2 or 6) complexes.  α Binds NDP substrates (CDP, UDP, ADP, and GDP, C site) as well as ATP and dNTPs (dATP, dGTP, TTP) allosteric effectors that control enzyme activity (A site) and substrate specificity (S site).  Clofarabine (ClF), an adenosine analog, is used in the treatment of refractory leukemias.  Its mode of cytotoxicity is thought to be assocd. in part with the triphosphate functioning as an allosteric inhibitor of hRNR.  Studies on the mechanism of inhibition of hRNR by ClF di- and triphosphates (ClFDP and ClFTP) are presented.  ClFTP is a reversible inhibitor (Ki = 40 nM) that rapidly inactivates hRNR.  However, with time, 50% of the activity is recovered.  D57N-α, a mutant with an altered A site, prevents inhibition by ClFTP, suggesting its A site binding.  ClFDP is a slow-binding, reversible inhibitor (Ki = 17 nM; t1/2 = 23 min).  CDP protects α from its inhibition.  The altered off-rate of CIFDP from E·ClFDP* by ClFTP (A site) or dGTP (S site) and its inhibition of D57N-α together implicate its C site binding.  Size exclusion chromatog. of hRNR or α alone with ClFDP or ClFTP, ±ATP or dGTP, reveals in each case that α forms a kinetically stable hexameric state.  This is the first example of hexamerization of a induced by an NDP analog that reversibly binds at the active site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpI_2N_uNXV87Vg90H21EOLACvtfcHk0lh60eEqGzY7Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvVyjt7o%253D&md5=1843df4bccc24864da4eb14c91bc84ff</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1013274108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1013274108%26sid%3Dliteratum%253Aachs%26aulast%3DAye%26aufirst%3DY.%26aulast%3DStubbe%26aufirst%3DJ.%26atitle%3DClofarabine%25205%25E2%2580%25B2-di%2520and%2520-triphosphates%2520inhibit%2520human%2520ribonucleotide%2520reductase%2520by%2520altering%2520the%2520quaternary%2520structure%2520of%2520its%2520large%2520subunit%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D9815%26epage%3D9820%26doi%3D10.1073%2Fpnas.1013274108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Aye, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brignole, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chittuluru, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drennan, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asturias, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stubbe, J.</span><span> </span><span class="NLM_article-title">Clofarabine targets the large subunit (alpha) of human ribonucleotide reductase in live cells by assembly into persistent hexamers</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">799</span><span class="NLM_x">–</span> <span class="NLM_lpage">805</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2012.05.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1016%2Fj.chembiol.2012.05.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=22840768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFahs7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=799-805&author=Y.+Ayeauthor=E.+J.+Brignoleauthor=M.+J.+Longauthor=J.+Chittuluruauthor=C.+L.+Drennanauthor=F.+J.+Asturiasauthor=J.+Stubbe&title=Clofarabine+targets+the+large+subunit+%28alpha%29+of+human+ribonucleotide+reductase+in+live+cells+by+assembly+into+persistent+hexamers&doi=10.1016%2Fj.chembiol.2012.05.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Clofarabine Targets the Large Subunit (α) of Human Ribonucleotide Reductase in Live Cells by Assembly into Persistent Hexamers</span></div><div class="casAuthors">Aye, Yimon; Brignole, Edward J.; Long, Marcus J. C.; Chittuluru, Johnathan; Drennan, Catherine L.; Asturias, Francisco J.; Stubbe, JoAnne</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">799-805</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Clofarabine (ClF) is a drug used in the treatment of leukemia.  One of its primary targets is human ribonucleotide reductase (hRNR), a dual-subunit, (α2)m(β2)n, regulatory enzyme indispensable in de novo dNTP synthesis.  We report that, in live mammalian cells, ClF targets hRNR by converting its α-subunit into kinetically stable hexamers.  We established mammalian expression platforms that enabled isolation of functional α and characterization of its altered oligomeric assocns. in response to ClF treatment.  Size exclusion chromatog. and electron microscopy documented persistence of in-cell-assembled-α6.  Our data validate hRNR as an important target of ClF, provide evidence that in vivo α's quaternary structure can be perturbed by a nonnatural ligand, and suggest small-mol.-promoted, persistent hexamerization as a strategy to modulate hRNR activity.  These studies lay foundations for documentation of RNR oligomeric state within a cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBYPDbbSuMoLVg90H21EOLACvtfcHk0lh60eEqGzY7Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFahs7zF&md5=efa9aa560ce94086fea32214bb1f350a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.05.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.05.015%26sid%3Dliteratum%253Aachs%26aulast%3DAye%26aufirst%3DY.%26aulast%3DBrignole%26aufirst%3DE.%2BJ.%26aulast%3DLong%26aufirst%3DM.%2BJ.%26aulast%3DChittuluru%26aufirst%3DJ.%26aulast%3DDrennan%26aufirst%3DC.%2BL.%26aulast%3DAsturias%26aufirst%3DF.%2BJ.%26aulast%3DStubbe%26aufirst%3DJ.%26atitle%3DClofarabine%2520targets%2520the%2520large%2520subunit%2520%2528alpha%2529%2520of%2520human%2520ribonucleotide%2520reductase%2520in%2520live%2520cells%2520by%2520assembly%2520into%2520persistent%2520hexamers%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D799%26epage%3D805%26doi%3D10.1016%2Fj.chembiol.2012.05.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Kashlan, O. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooperman, B. S.</span><span> </span><span class="NLM_article-title">Comprehensive model for allosteric regulation of mammalian ribonucleotide reductase: refinements and consequences</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">1696</span><span class="NLM_x">–</span> <span class="NLM_lpage">1706</span><span class="refDoi"> DOI: 10.1021/bi020634d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi020634d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD3sXlsFaksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=1696-1706&author=O.+B.+Kashlanauthor=B.+S.+Cooperman&title=Comprehensive+model+for+allosteric+regulation+of+mammalian+ribonucleotide+reductase%3A+refinements+and+consequences&doi=10.1021%2Fbi020634d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive Model for Allosteric Regulation of Mammalian Ribonucleotide Reductase: Refinements and Consequences</span></div><div class="casAuthors">Kashlan, Ossama B.; Cooperman, Barry S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1696-1706</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Redn. of NDPs by murine ribonucleotide reductase (mRR) requires catalytic (mR1) and free radical-contg. (mR2) subunits and is regulated by nucleoside triphosphate allosteric effectors.  Here we present the results of several studies that refine the recently presented comprehensive model for the allosteric control of mRR enzymic activity [Kashlan, O. B., et al. (2002) Biochem. 41, 462-474], in which nucleotide binding to the specificity site (s-site) drives formation of an active R12R22 dimer, ATP or dATP binding to the adenine site (a-site) drives formation of a tetramer, mR14a, which isomerizes to an inactive form, mR14b, and ATP binding to the hexamerization site (h-site) drives formation of an active R16R26 hexamer.  Anal. of the a-site D57N variant of mR1, which differs from wild-type mR1 (wt.-mR1) in that its RR activity is activated by both ATP and dATP, demonstrates that dATP activation of the D57N variant RR arises from a blockage in the formation of mR14b from mR14a, and provides strong evidence that mR14a forms active complexes with mR22.  We further demonstrate that (a) differences in the effects of ATP vs. dATP binding to the a-site of wt.-mR1 provide specific mechanisms by which the dATP/ATP ratio in mammalian cells could modulate in vivo RR enzymic activity, (b) the comprehensive model is valid over a range of Mg2+ concns. that include in vivo concns., and (c) equil. consts. derived for the comprehensive model can be used to simulate the distribution of R1 among dimer, tetramer, and hexamer forms in vivo.  Such simulations indicate that mR16 predominates over mR12 in the cytoplasm of normal mammalian cells, where the great majority of RR activity is located, but that mR12 may be important for nuclear RR activity and for RR activity in cells in which the level of ATP is depleted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLAXHh__c4ErVg90H21EOLACvtfcHk0liFtiptSea5NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXlsFaksw%253D%253D&md5=97e20a2b3a066c99e735736c0b1c47e6</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fbi020634d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi020634d%26sid%3Dliteratum%253Aachs%26aulast%3DKashlan%26aufirst%3DO.%2BB.%26aulast%3DCooperman%26aufirst%3DB.%2BS.%26atitle%3DComprehensive%2520model%2520for%2520allosteric%2520regulation%2520of%2520mammalian%2520ribonucleotide%2520reductase%253A%2520refinements%2520and%2520consequences%26jtitle%3DBiochemistry%26date%3D2003%26volume%3D42%26spage%3D1696%26epage%3D1706%26doi%3D10.1021%2Fbi020634d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Arnaoutov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dasso, M.</span><span> </span><span class="NLM_article-title">IRBIT is a novel regulator of ribonucleotide reductase in higher eukaryotes</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">345</span><span class="NLM_x">, </span> <span class="NLM_fpage">1512</span><span class="NLM_x">–</span> <span class="NLM_lpage">1515</span><span class="refDoi"> DOI: 10.1126/science.1251550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1126%2Fscience.1251550" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=345&publication_year=2014&pages=1512-1515&author=A.+Arnaoutovauthor=M.+Dasso&title=IRBIT+is+a+novel+regulator+of+ribonucleotide+reductase+in+higher+eukaryotes&doi=10.1126%2Fscience.1251550"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1126%2Fscience.1251550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1251550%26sid%3Dliteratum%253Aachs%26aulast%3DArnaoutov%26aufirst%3DA.%26aulast%3DDasso%26aufirst%3DM.%26atitle%3DIRBIT%2520is%2520a%2520novel%2520regulator%2520of%2520ribonucleotide%2520reductase%2520in%2520higher%2520eukaryotes%26jtitle%3DScience%26date%3D2014%26volume%3D345%26spage%3D1512%26epage%3D1515%26doi%3D10.1126%2Fscience.1251550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Ahmad, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wan, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jha, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motea, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berdis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dealwis, C.</span><span> </span><span class="NLM_article-title">Evaluating the therapeutic potential of a non-natural nucleotide that inhibits human ribonucleotide reductase</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">2077</span><span class="NLM_x">–</span> <span class="NLM_lpage">2086</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-12-0199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1158%2F1535-7163.MCT-12-0199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=22933704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVyksr%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=2077-2086&author=M.+F.+Ahmadauthor=Q.+Wanauthor=S.+Jhaauthor=E.+Moteaauthor=A.+Berdisauthor=C.+Dealwis&title=Evaluating+the+therapeutic+potential+of+a+non-natural+nucleotide+that+inhibits+human+ribonucleotide+reductase&doi=10.1158%2F1535-7163.MCT-12-0199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluating the Therapeutic Potential of a Non-Natural Nucleotide That Inhibits Human Ribonucleotide Reductase</span></div><div class="casAuthors">Ahmad, Md. Faiz; Wan, Qun; Jha, Shalini; Motea, Edward; Berdis, Anthony; Dealwis, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2077-2086</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Human ribonucleotide reductase (hRR) is the key enzyme involved in de novo dNTP synthesis and thus represents an important therapeutic target against hyperproliferative diseases, most notably cancer.  The purpose of this study was to evaluate the ability of non-natural indolyl-2'-deoxynucleoside triphosphates to inhibit the activity of hRR.  The structural similarities of these analogs with dATP predicted that they would inhibit hRR activity by binding to its allosteric sites.  In silico anal. and in vitro characterization identified one particular analog designated as 5-nitro-indolyl-2'-deoxyribose triphosphate (5-NITP) that inhibits hRR. 5-NITP binding to hRR was detd. by isothermal titrn. calorimetry.  X-ray crystal structure of 5-NITP bound to RR1 was detd.  Cell-based studies showed the anti-cancer effects of the corresponding non-natural nucleoside against leukemia cells. 5-NITP binds to hRR with micromolar affinity.  Binding does not induce hexamerization of hRR1 like dATP, the native allosteric inhibitor of hRR that binds with high affinity to the A-site.  The X-ray crystal structure of Saccharomyces cerevisiae RR1-5-NITP (ScRR1-5-NITP) complex detd. to 2.3 Å resoln. shows that 5-NITP does not bind to the A-site but rather at the S-site.  Regardless, 5-nitro-indolyl-2'-deoxynucleoside (5-NIdR) produces cytostatic and cytotoxic effects against human leukemia cells by altering cell-cycle progression.  Our studies provide useful insights toward developing new inhibitors with improved potency and efficacy against hRR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq13zZRjCX4ALVg90H21EOLACvtfcHk0ljWcVU1xOOnCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVyksr%252FM&md5=79ad691a31ad0710598684e14bcece3a</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-12-0199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-12-0199%26sid%3Dliteratum%253Aachs%26aulast%3DAhmad%26aufirst%3DM.%2BF.%26aulast%3DWan%26aufirst%3DQ.%26aulast%3DJha%26aufirst%3DS.%26aulast%3DMotea%26aufirst%3DE.%26aulast%3DBerdis%26aufirst%3DA.%26aulast%3DDealwis%26aufirst%3DC.%26atitle%3DEvaluating%2520the%2520therapeutic%2520potential%2520of%2520a%2520non-natural%2520nucleotide%2520that%2520inhibits%2520human%2520ribonucleotide%2520reductase%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2012%26volume%3D11%26spage%3D2077%26epage%3D2086%26doi%3D10.1158%2F1535-7163.MCT-12-0199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Jaffe, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stith, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawrence, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrake, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunbrack, R. L.</span><span> </span><span class="NLM_article-title">A new model for allosteric regulation of phenylalanine hydroxylase: implications for disease and therapeutics</span> <span class="citation_source-journal">Arch. Biochem. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">530</span><span class="NLM_x">, </span> <span class="NLM_fpage">73</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span><span class="refDoi"> DOI: 10.1016/j.abb.2012.12.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1016%2Fj.abb.2012.12.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=23296088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3sXis1Orsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=530&publication_year=2013&pages=73-82&author=E.+K.+Jaffeauthor=L.+Stithauthor=S.+H.+Lawrenceauthor=M.+Andrakeauthor=R.+L.+Dunbrack&title=A+new+model+for+allosteric+regulation+of+phenylalanine+hydroxylase%3A+implications+for+disease+and+therapeutics&doi=10.1016%2Fj.abb.2012.12.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A new model for allosteric regulation of phenylalanine hydroxylase: Implications for disease and therapeutics</span></div><div class="casAuthors">Jaffe, Eileen K.; Stith, Linda; Lawrence, Sarah H.; Andrake, Mark; Dunbrack, Roland L., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">530</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">73-82</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The structural basis for allosteric regulation of phenylalanine hydroxylase (PAH), whose dysfunction causes phenylketonuria (PKU), is poorly understood.  A new morpheein model for PAH allostery is proposed to consist of a dissociative equil. between two architecturally different tetramers whose interconversion requires a ∼90° rotation between the PAH catalytic and regulatory domains, the latter of which contains an ACT domain.  This unprecedented model is supported by in vitro data on purified full length rat and human PAH.  The conformational change is both predicted to and shown to render the tetramers chromatog. separable using ion exchange methods.  One novel aspect of the activated tetramer model is an allosteric phenylalanine binding site at the intersubunit interface of ACT domains.  Amino acid ligand-stabilized ACT domain dimerization follows the multimerization and ligand binding behavior of ACT domains present in other proteins in the PDB.  Spectroscopic, chromatog., and electrophoretic methods demonstrate a PAH equil. consisting of two architecturally distinct tetramers as well as dimers.  We postulate that PKU-assocd. mutations may shift the PAH quaternary structure equil. in favor of the low activity assemblies.  Pharmacol. chaperones that stabilize the ACT:ACT interface can potentially provide PKU patients with a novel small mol. therapeutic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjgPGUSaTRErVg90H21EOLACvtfcHk0ljWcVU1xOOnCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXis1Orsrk%253D&md5=15a7753b982effcbff2b40f23db80b39</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2012.12.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2012.12.017%26sid%3Dliteratum%253Aachs%26aulast%3DJaffe%26aufirst%3DE.%2BK.%26aulast%3DStith%26aufirst%3DL.%26aulast%3DLawrence%26aufirst%3DS.%2BH.%26aulast%3DAndrake%26aufirst%3DM.%26aulast%3DDunbrack%26aufirst%3DR.%2BL.%26atitle%3DA%2520new%2520model%2520for%2520allosteric%2520regulation%2520of%2520phenylalanine%2520hydroxylase%253A%2520implications%2520for%2520disease%2520and%2520therapeutics%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2013%26volume%3D530%26spage%3D73%26epage%3D82%26doi%3D10.1016%2Fj.abb.2012.12.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Jaffe, E. K.</span><span> </span><span class="NLM_article-title">Impact of quaternary structure dynamics on allosteric drug discovery</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">55</span><span class="NLM_x">–</span> <span class="NLM_lpage">63</span><span class="refDoi"> DOI: 10.2174/1568026611313010006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.2174%2F1568026611313010006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=23409765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtleqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=55-63&author=E.+K.+Jaffe&title=Impact+of+quaternary+structure+dynamics+on+allosteric+drug+discovery&doi=10.2174%2F1568026611313010006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of quaternary structure dynamics on allosteric drug discovery</span></div><div class="casAuthors">Jaffe, Eileen K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-63</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The morpheein model of allosteric regulation draws attention to proteins that can exist as an equil. of functionally distinct assemblies where: one subunit conformation assembles into one multimer; a different subunit conformation assembles into a different multimer; and the various multimers are in a dynamic equil. whose position can be modulated by ligands that bind to a multimer-specific ligand binding site.  The case study of porphobilinogen synthase (PBGS) illustrates how such an equil. holds lessons for disease mechanisms, drug discovery, understanding drug side effects, and identifying proteins wherein drug discovery efforts might focus on quaternary structure dynamics.  The morpheein model of allostery has been proposed as applicable for a wide assortment of disease-assocd. proteins.  Herein we discuss quaternary structure dynamics aspects to drug discovery for the disease-assocd. putative morpheeins phenylalanine hydroxylase, HIV integrase, pyruvate kinase, and tumor necrosis factor α.  Also highlighted is the quaternary structure equil. of transthyretin and successful drug discovery efforts focused on controlling its quaternary structure dynamics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_MI6syFNLs7Vg90H21EOLACvtfcHk0ljWcVU1xOOnCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtleqtbk%253D&md5=e53431517b1eca6ab29617d46e7b3cd1</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.2174%2F1568026611313010006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026611313010006%26sid%3Dliteratum%253Aachs%26aulast%3DJaffe%26aufirst%3DE.%2BK.%26atitle%3DImpact%2520of%2520quaternary%2520structure%2520dynamics%2520on%2520allosteric%2520drug%2520discovery%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2013%26volume%3D13%26spage%3D55%26epage%3D63%26doi%3D10.2174%2F1568026611313010006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Jaffe, E. K.</span><span> </span><span class="NLM_article-title">Morpheeins - A new pathway for allosteric drug discovery</span> <span class="citation_source-journal">Open Conf. Proc. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.2174/22102892010010100001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.2174%2F22102892010010100001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=21643557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlKqur7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=1-6&author=E.+K.+Jaffe&title=Morpheeins+-+A+new+pathway+for+allosteric+drug+discovery&doi=10.2174%2F22102892010010100001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Morpheeins - a new pathway for allosteric drug discovery</span></div><div class="casAuthors">Jaffe, Eileen K.</div><div class="citationInfo"><span class="NLM_cas:title">Open Conference Proceedings Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-6</span>CODEN:
                <span class="NLM_cas:coden">OCPJC2</span>;
        ISSN:<span class="NLM_cas:issn">2210-2892</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The morpheein model of allosteric regulation can be applied as a novel approach to the discovery of small mol. allosteric modulators of protein function.  Morpheeins are homo-oligomeric proteins where, under physiol. conditions, the oligomer can dissoc., the dissocd. units can change conformation, and the altered conformational state can reassoc. to a structurally and functionally distinct oligomer.  This phenomenon serves as a basis for allostery, as a basis for conformational diseases, as a basis for drug discovery, and may be applicable to personalized medicine such as in the prediction of drug side effects.  Each of these relationships has been established for the prototype morpheein, porphobilinogen synthase, where the conformational disease is a porphyria and the drug application is in antimicrobial discovery.  These data are presented along with a discussion of other drug targets for which the morpheein model of allostery may apply.  Such targets include HIV integrase, TNFα, β-tryptase, and p53.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHwzTd8syeArVg90H21EOLACvtfcHk0ljWcVU1xOOnCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlKqur7F&md5=bc56e399c1257be36aea71127427f920</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.2174%2F22102892010010100001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F22102892010010100001%26sid%3Dliteratum%253Aachs%26aulast%3DJaffe%26aufirst%3DE.%2BK.%26atitle%3DMorpheeins%2520-%2520A%2520new%2520pathway%2520for%2520allosteric%2520drug%2520discovery%26jtitle%3DOpen%2520Conf.%2520Proc.%2520J.%26date%3D2010%26volume%3D1%26spage%3D1%26epage%3D6%26doi%3D10.2174%2F22102892010010100001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G. A.</span><span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2719</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span><span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0liQTAvEZn9wiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Wan, K. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, V. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Entzeroth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. A.</span><span> </span><span class="NLM_article-title">Differential scanning fluorimetry as secondary screening platform for small molecule inhibitors of Bcl-XL</span> <span class="citation_source-journal">Cell Cycle</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">3943</span><span class="NLM_x">–</span> <span class="NLM_lpage">3952</span><span class="refDoi"> DOI: 10.4161/cc.8.23.10114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.4161%2Fcc.8.23.10114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=19901550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotlWrtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=3943-3952&author=K.+F.+Wanauthor=S.+Wangauthor=C.+J.+Brownauthor=V.+C.+Yuauthor=M.+Entzerothauthor=D.+P.+Laneauthor=M.+A.+Lee&title=Differential+scanning+fluorimetry+as+secondary+screening+platform+for+small+molecule+inhibitors+of+Bcl-XL&doi=10.4161%2Fcc.8.23.10114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Differential scanning fluorimetry as secondary screening platform for small molecule inhibitors of Bcl-XL</span></div><div class="casAuthors">Wan, Kah Fei; Wang, Sifang; Brown, Christopher J.; Yu, Victor C.; Entzeroth, Michael; Lane, David P.; Lee, May Ann</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3943-3952</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Since apoptosis is impaired in malignant cells overexpressing prosurvival Bcl-2 proteins, drugs mimicking their natural antagonists, BH3-only proteins, might overcome chemoresistance.  Small mol. inhibitors of Bcl-XL function have been discovered from diverse structure classes using rational drug design as well as high-throughput screening (HTS) approaches.  However, most of the BH3 mimetics that have been identified via screening based on fluorescence polarization displayed an affinity for their presumed protein targets that is far lower than that of BH3-only proteins.  Therefore, it is important to establish a simple and inexpensive secondary platform for hit validation which is pertinent to current efforts for developing compds. that mimic the action of BH3-only proteins as novel anticancer agents.  These considerations prompted the authors to explore the differential scanning fluorometry (DSF) method that is based on energetic coupling between ligand binding and protein unfolding.  The authors have systematically tested known Bcl-XL/Bcl-2 inhibitors using DSF and have revealed distinct subsets of inhibitors.  More importantly, the authors report that some of these inhibitors interacted selectively with glutathione S-transferase tagged Bcl-XL, whereas certain inhibitors exhibited marked selectivity towards native untagged Bcl-XL.  Therefore, the authors propose that the affinity tag may cause a significant conformational switch in the Bcl-XL, which results in the selectivity for certain subsets of small mol. inhibitors.  This finding also implies that the previous screens involving tagged proteins need to be carefully reexamd. while further investigations must ensure that the right conformation of protein is used in future screens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGHLlHxk1TSbVg90H21EOLACvtfcHk0liQTAvEZn9wiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotlWrtbo%253D&md5=3c7bb4fcfcdd53c393b7e3e9088fb847</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.4161%2Fcc.8.23.10114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.8.23.10114%26sid%3Dliteratum%253Aachs%26aulast%3DWan%26aufirst%3DK.%2BF.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DBrown%26aufirst%3DC.%2BJ.%26aulast%3DYu%26aufirst%3DV.%2BC.%26aulast%3DEntzeroth%26aufirst%3DM.%26aulast%3DLane%26aufirst%3DD.%2BP.%26aulast%3DLee%26aufirst%3DM.%2BA.%26atitle%3DDifferential%2520scanning%2520fluorimetry%2520as%2520secondary%2520screening%2520platform%2520for%2520small%2520molecule%2520inhibitors%2520of%2520Bcl-XL%26jtitle%3DCell%2520Cycle%26date%3D2009%26volume%3D8%26spage%3D3943%26epage%3D3952%26doi%3D10.4161%2Fcc.8.23.10114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Shrake, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rupley, J. A.</span><span> </span><span class="NLM_article-title">Environment and exposure to solvent of protein atoms. Lysozyme and insulin</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">351</span><span class="NLM_x">–</span> <span class="NLM_lpage">371</span><span class="refDoi"> DOI: 10.1016/0022-2836(73)90011-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1016%2F0022-2836%2873%2990011-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=4760134" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADyaE3sXlsVelt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1973&pages=351-371&author=A.+Shrakeauthor=J.+A.+Rupley&title=Environment+and+exposure+to+solvent+of+protein+atoms.+Lysozyme+and+insulin&doi=10.1016%2F0022-2836%2873%2990011-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Environment and exposure to solvent of protein atoms.  Lysozyme and insulin</span></div><div class="casAuthors">Shrake, A.; Rupley, J. A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">351-71</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">A computer program is described for calcg. the environment and the exposure to solvent of atoms of a protein.  The computation is based on the at. coordinates of the protein and on assumptions like those of B. Lee and F. M. Richards (1971).  Results for lysozyme and insulin are presented.  Changes in exposure to solvent and in the nature of contacts that develop through folding, assocn. reactions, and crystn. are described numerically.  The computations suggest several generalizations.  (a) Lattice contacts within the protein crystal are characterized by a significantly smaller involvement of nonpolar side chains and a proportionately greater involvement of ionizable side chains than is found for protein folding or for protein assocn. reactions important for biol. function.  (b) In helical regions the carbonyl O of the 1st residue in the helix has high probability of being shielded from solvent.  (c) Glycine is among the residues having exposure least affected by folding; this accords with the expectation that it lies at bends of the peptide chain on the surface of the mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoU7V-RfwTrV7Vg90H21EOLACvtfcHk0liQTAvEZn9wiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3sXlsVelt7Y%253D&md5=de79172cb044d2c7545702f6503a891a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2873%2990011-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252873%252990011-9%26sid%3Dliteratum%253Aachs%26aulast%3DShrake%26aufirst%3DA.%26aulast%3DRupley%26aufirst%3DJ.%2BA.%26atitle%3DEnvironment%2520and%2520exposure%2520to%2520solvent%2520of%2520protein%2520atoms.%2520Lysozyme%2520and%2520insulin%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1973%26volume%3D79%26spage%3D351%26epage%3D371%26doi%3D10.1016%2F0022-2836%2873%2990011-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Stearns, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Losee, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernstein, J.</span><span> </span><span class="NLM_article-title">Hydroxyurea. A new type of potential antitumor agent</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1963</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">201</span><span class="refDoi"> DOI: 10.1021/jm00338a026</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00338a026" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1963&pages=201&author=B.+Stearnsauthor=K.+A.+Loseeauthor=J.+Bernstein&title=Hydroxyurea.+A+new+type+of+potential+antitumor+agent&doi=10.1021%2Fjm00338a026"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm00338a026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00338a026%26sid%3Dliteratum%253Aachs%26aulast%3DStearns%26aufirst%3DB.%26aulast%3DLosee%26aufirst%3DK.%2BA.%26aulast%3DBernstein%26aufirst%3DJ.%26atitle%3DHydroxyurea.%2520A%2520new%2520type%2520of%2520potential%2520antitumor%2520agent%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1963%26volume%3D6%26spage%3D201%26doi%3D10.1021%2Fjm00338a026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Donehower, R. C.</span><span> </span><span class="NLM_article-title">An overview of the clinical experience with hydroxyurea</span> <span class="citation_source-journal">Semin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">11</span><span class="NLM_x">–</span> <span class="NLM_lpage">19</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1641651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A280%3ADyaK38zltFWitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1992&pages=11-19&author=R.+C.+Donehower&title=An+overview+of+the+clinical+experience+with+hydroxyurea"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">An overview of the clinical experience with hydroxyurea</span></div><div class="casAuthors">Donehower R C</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in oncology</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3 Suppl 9</span>),
    <span class="NLM_cas:pages">11-9</span>
        ISSN:<span class="NLM_cas:issn">0093-7754</span>.
    </div><div class="casAbstract">In the more than 30 years since hydroxyurea was first evaluated clinically, a number of diverse and unique applications have been identified for its use.  Some of these applications were apparent to investigators almost from the beginning, while others are still evolving 3 decades later as a result of ongoing research.  An overview of clinical experience with hydroxyurea provides an historical perspective on a period encompassing virtually the entire era in which anticancer chemotherapy has been a recognized discipline.  The critical role of the ribonucleotide reductase reaction in the regulation of DNA synthesis makes this enzyme a particularly important target site for antineoplastic therapy.  The sustained interest in hydroxyurea, however, has been the result of careful clinical observations as well as basic laboratory studies, which support current attempts to identify additional uses for the drug.  This overview describes the clinical development of hydroxyurea, discusses current indications, and briefly describes areas of ongoing research that may expand the clinical indications for hydroxyurea.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTa1cLEoNUZPuqrn-c4Of3KfW6udTcc2eYg4c4p8-KRvLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK38zltFWitA%253D%253D&md5=944c23293942a08a460e1e638a29008d</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDonehower%26aufirst%3DR.%2BC.%26atitle%3DAn%2520overview%2520of%2520the%2520clinical%2520experience%2520with%2520hydroxyurea%26jtitle%3DSemin.%2520Oncol.%26date%3D1992%26volume%3D19%26spage%3D11%26epage%3D19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Winter, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higueruelo, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sigurdardottir, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitt, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span> </span><span class="NLM_article-title">Biophysical and computational fragment-based approaches to targeting protein-protein interactions: applications in structure-guided drug discovery</span> <span class="citation_source-journal">Q. Rev. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">383</span><span class="NLM_x">–</span> <span class="NLM_lpage">426</span><span class="refDoi"> DOI: 10.1017/S0033583512000108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1017%2FS0033583512000108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=22971516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslygtL7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2012&pages=383-426&author=A.+Winterauthor=A.+P.+Higuerueloauthor=M.+Marshauthor=A.+Sigurdardottirauthor=W.+R.+Pittauthor=T.+L.+Blundell&title=Biophysical+and+computational+fragment-based+approaches+to+targeting+protein-protein+interactions%3A+applications+in+structure-guided+drug+discovery&doi=10.1017%2FS0033583512000108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Biophysical and computational fragment-based approaches to targeting protein-protein interactions: applications in structure-guided drug discovery</span></div><div class="casAuthors">Winter, Anja; Higueruelo, Alicia P.; Marsh, May; Sigurdardottir, Anna; Pitt, Will R.; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Quarterly Reviews of Biophysics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">383-426</span>CODEN:
                <span class="NLM_cas:coden">QURBAW</span>;
        ISSN:<span class="NLM_cas:issn">0033-5835</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  Drug discovery has classically targeted the active sites of enzymes or ligand-binding sites of receptors and ion channels.  In an attempt to improve selectivity of drug candidates, modulation of protein-protein interfaces (PPIs) of multiprotein complexes that mediate conformation or colocation of components of cell-regulatory pathways has become a focus of interest.  However, PPIs in multiprotein systems continue to pose significant challenges, as they are generally large, flat and poor in distinguishing features, making the design of small mol. antagonists a difficult task.  Nevertheless, encouragement has come from the recognition that a few amino acids - so-called hotspots - may contribute the majority of interaction-free energy.  The challenges posed by protein-protein interactions have led to a wellspring of creative approaches, including proteomimetics, stapled α-helical peptides and a plethora of antibody inspired mol. designs.  Here, we review a more generic approach: fragment-based drug discovery.  Fragments allow novel areas of chem. space to be explored more efficiently, but the initial hits have low affinity.  This means that they will not normally disrupt PPIs, unless they are tethered, an approach that has been pioneered by Wells and co-workers.  An alternative fragment-based approach is to stabilize the uncomplexed components of the multiprotein system in soln. and employ conventional fragment-based screening.  Here, we describe the current knowledge of the structures and properties of protein-protein interactions and the small mols. that can modulate them.  We then describe the use of sensitive biophys. methods - NMR, X-ray crystallog., surface plasmon resonance, differential scanning fluorimetry or isothermal calorimetry - to screen and validate fragment binding.  Fragment hits can subsequently be evolved into larger mols. with higher affinity and potency.  These may provide new leads for drug candidates that target protein-protein interactions and have therapeutic value.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoninFnW9vZ5LVg90H21EOLACvtfcHk0lgEtjf-78agHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslygtL7K&md5=c3ad9e944414331e1d07509d37583b7a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1017%2FS0033583512000108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0033583512000108%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DA.%26aulast%3DHigueruelo%26aufirst%3DA.%2BP.%26aulast%3DMarsh%26aufirst%3DM.%26aulast%3DSigurdardottir%26aufirst%3DA.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DBiophysical%2520and%2520computational%2520fragment-based%2520approaches%2520to%2520targeting%2520protein-protein%2520interactions%253A%2520applications%2520in%2520structure-guided%2520drug%2520discovery%26jtitle%3DQ.%2520Rev.%2520Biophys.%26date%3D2012%26volume%3D45%26spage%3D383%26epage%3D426%26doi%3D10.1017%2FS0033583512000108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Wells, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McClendon, C. L.</span><span> </span><span class="NLM_article-title">Reaching for high-hanging fruit in drug discovery at protein-protein interfaces</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">450</span><span class="NLM_x">, </span> <span class="NLM_fpage">1001</span><span class="NLM_x">–</span> <span class="NLM_lpage">1009</span><span class="refDoi"> DOI: 10.1038/nature06526</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1038%2Fnature06526" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=18075579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2007&pages=1001-1009&author=J.+A.+Wellsauthor=C.+L.+McClendon&title=Reaching+for+high-hanging+fruit+in+drug+discovery+at+protein-protein+interfaces&doi=10.1038%2Fnature06526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Reaching for high-hanging fruit in drug discovery at protein-protein interfaces</span></div><div class="casAuthors">Wells, James A.; McClendon, Christopher L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">450</span>
        (<span class="NLM_cas:issue">7172</span>),
    <span class="NLM_cas:pages">1001-1009</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Targeting the interfaces between proteins has huge therapeutic potential, but discovering small-mol. drugs that disrupt protein-protein interactions is an enormous challenge.  Several recent success stories, however, indicate that protein-protein interfaces might be more tractable than has been thought.  These studies discovered small mols. that bind with drug-like potencies to 'hotspots' on the contact surfaces involved in protein-protein interactions.  Remarkably, these small mols. bind deeper within the contact surface of the target protein, and bind with much higher efficiencies, than do the contact atoms of the natural protein partner.  Some of these small mols. are now making their way through clin. trials, so this high-hanging fruit might not be far out of reach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-nGR3BqqkrVg90H21EOLACvtfcHk0lgEtjf-78agHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F&md5=e7ed69fd8c362a71c4b99f029c16a6fc</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnature06526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06526%26sid%3Dliteratum%253Aachs%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DMcClendon%26aufirst%3DC.%2BL.%26atitle%3DReaching%2520for%2520high-hanging%2520fruit%2520in%2520drug%2520discovery%2520at%2520protein-protein%2520interfaces%26jtitle%3DNature%26date%3D2007%26volume%3D450%26spage%3D1001%26epage%3D1009%26doi%3D10.1038%2Fnature06526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span> </span><span class="NLM_article-title">The druggable genome</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">730</span><span class="refDoi"> DOI: 10.1038/nrd892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1038%2Fnrd892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=12209152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=727-730&author=A.+L.+Hopkinsauthor=C.+R.+Groom&title=The+druggable+genome&doi=10.1038%2Fnrd892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: The druggable genome</span></div><div class="casAuthors">Hopkins, Andrew L.; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">727-730</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  An assessment of the no. of mol. targets that represent an opportunity for therapeutic intervention is crucial to the development of post-genomic research strategies within the pharmaceutical industry.  Now that we know the size of the human genome, it is interesting to consider just how many mol. targets this opportunity represents.  We start from the position that we understand the properties that are required for a good drug, and therefore must be able to understand what makes a good drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Uvqi8O9wzrVg90H21EOLACvtfcHk0ljnGaSu_4YbvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D&md5=d6dc7c67822661fe03e920695def1329</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnrd892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd892%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520druggable%2520genome%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D727%26epage%3D730%26doi%3D10.1038%2Fnrd892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Mills, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jasensky, R. D.</span><span> </span><span class="NLM_article-title">Antimicrobial effects of N alpha-palmitoyl-L-lysyl-L-lysine ethyl ester dihydrochloride and its use to extend the shelf life of creamed cottage cheese</span> <span class="citation_source-journal">J. Agric. Food Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1980</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">812</span><span class="NLM_x">–</span> <span class="NLM_lpage">817</span><span class="refDoi"> DOI: 10.1021/jf60230a035</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jf60230a035" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1980&pages=812-817&author=C.+J.+Millsauthor=T.+Richardsonauthor=R.+D.+Jasensky&title=Antimicrobial+effects+of+N+alpha-palmitoyl-L-lysyl-L-lysine+ethyl+ester+dihydrochloride+and+its+use+to+extend+the+shelf+life+of+creamed+cottage+cheese&doi=10.1021%2Fjf60230a035"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjf60230a035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjf60230a035%26sid%3Dliteratum%253Aachs%26aulast%3DMills%26aufirst%3DC.%2BJ.%26aulast%3DRichardson%26aufirst%3DT.%26aulast%3DJasensky%26aufirst%3DR.%2BD.%26atitle%3DAntimicrobial%2520effects%2520of%2520N%2520alpha-palmitoyl-L-lysyl-L-lysine%2520ethyl%2520ester%2520dihydrochloride%2520and%2520its%2520use%2520to%2520extend%2520the%2520shelf%2520life%2520of%2520creamed%2520cottage%2520cheese%26jtitle%3DJ.%2520Agric.%2520Food%2520Chem.%26date%3D1980%26volume%3D28%26spage%3D812%26epage%3D817%26doi%3D10.1021%2Fjf60230a035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Cegielska, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, A.</span><span> </span><span class="NLM_article-title">The sfiA11 mutation prevents filamentation in a response to cell wall damage only in a recA+ genetic background</span> <span class="citation_source-journal">Mol. Gen. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">201</span><span class="NLM_x">, </span> <span class="NLM_fpage">537</span><span class="NLM_x">–</span> <span class="NLM_lpage">542</span><span class="refDoi"> DOI: 10.1007/BF00331352</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1007%2FBF00331352" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=3911030" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADyaL28Xot1ygtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=201&publication_year=1985&pages=537-542&author=A.+Cegielskaauthor=A.+Taylor&title=The+sfiA11+mutation+prevents+filamentation+in+a+response+to+cell+wall+damage+only+in+a+recA%2B+genetic+background&doi=10.1007%2FBF00331352"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The sfiA11 mutation prevents filamentation in a response to cell wall damage only a recA+ genetic background</span></div><div class="casAuthors">Cegielska, Aleksandra; Taylor, Alina</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and General Genetics</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">201</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">537-42</span>CODEN:
                <span class="NLM_cas:coden">MGGEAE</span>;
        ISSN:<span class="NLM_cas:issn">0026-8925</span>.
    </div><div class="casAbstract">N-α-Palmitoyl-L-lysyl-L-lysine dihydrochloride Et ester (PLL)  [7325-88-4] at sublethal doses causes filamentous growth of Escherichia coli strains except sfiA mutants, which divide normally in its presence.  PLL does not elicit the SOS responses, as judged by λ prophage induction, RecA protein synthesis, or the induction of the sfiA operon in a sfiA::lacZ fusion strain.  Filamentation caused by PLL may be not an SOS function and might be the result of membrane damage by PLL, which is an amphipathic compd. and which, at higher doses, causes cell lysis.  Apparently, basal levels of the sfiA gene product are sufficient to inhibit cell division in the presence of PLL.  The phenotype of the sfiA mutation in the presence of PLL requires a recA+ genetic background and does not occur in E. coli mutants recA1 sfiA11, recA13 sfiA11, recA56 sfiA11 and recA441 sfiA11.  All of these strains, except rec441 sfiA11, regain the ability of sfiA11 mutants to divide in the presence of PLL after transformation with the RecA-overproducing plasmid pXO2.  Apparently, the RecA protein pos. affects sfiA11-mediated cell division in the presence of PLL.  The basal level of the RecA protein in the recA+ sfiA11 strain is sufficient for this process.  An increased level due to overprodn. from the multicopy plasmid pXO2 exerts the same effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokYSeaiXlZI7Vg90H21EOLACvtfcHk0ljnGaSu_4YbvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28Xot1ygtA%253D%253D&md5=e0d7b238ad35b7539c1c132e83444758</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2FBF00331352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00331352%26sid%3Dliteratum%253Aachs%26aulast%3DCegielska%26aufirst%3DA.%26aulast%3DTaylor%26aufirst%3DA.%26atitle%3DThe%2520sfiA11%2520mutation%2520prevents%2520filamentation%2520in%2520a%2520response%2520to%2520cell%2520wall%2520damage%2520only%2520in%2520a%2520recA%252B%2520genetic%2520background%26jtitle%3DMol.%2520Gen.%2520Genet.%26date%3D1985%26volume%3D201%26spage%3D537%26epage%3D542%26doi%3D10.1007%2FBF00331352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Blundell, T. L.</span><span> </span><span class="NLM_article-title">Structure-based drug design</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">384</span><span class="NLM_x">, </span> <span class="NLM_fpage">23</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=1996&pages=23-26&author=T.+L.+Blundell&title=Structure-based+drug+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DStructure-based%2520drug%2520design%26jtitle%3DNature%26date%3D1996%26volume%3D384%26spage%3D23%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Hofer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crona, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Logan, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sjoberg, B. M.</span><span> </span><span class="NLM_article-title">DNA building blocks: keeping control of manufacture</span> <span class="citation_source-journal">Crit. Rev. Biochem. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">50</span><span class="NLM_x">–</span> <span class="NLM_lpage">63</span><span class="refDoi"> DOI: 10.3109/10409238.2011.630372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.3109%2F10409238.2011.630372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=22050358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1CitLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2012&pages=50-63&author=A.+Hoferauthor=M.+Cronaauthor=D.+T.+Loganauthor=B.+M.+Sjoberg&title=DNA+building+blocks%3A+keeping+control+of+manufacture&doi=10.3109%2F10409238.2011.630372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">DNA building blocks: keeping control of manufacture</span></div><div class="casAuthors">Hofer, Anders; Crona, Mikael; Logan, Derek T.; Sjoeberg, Britt-Marie</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Biochemistry and Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">50-63</span>CODEN:
                <span class="NLM_cas:coden">CRBBEJ</span>;
        ISSN:<span class="NLM_cas:issn">1040-9238</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Ribonucleotide reductase (RNR) is the only source for de novo prodn. of the four deoxyribonucleoside triphosphate (dNTP) building blocks needed for DNA synthesis and repair.  It is crucial that these dNTP pools are carefully balanced, since mutation rates increase when dNTP levels are either unbalanced or elevated.  RNR is the major player in this homeostasis, and with its four different substrates, four different allosteric effectors and two different effector binding sites, it has one of the most sophisticated allosteric regulations known today.  In the past few years, the structures of RNRs from several bacteria, yeast and man have been detd. in the presence of allosteric effectors and substrates, revealing new information about the mechanisms behind the allosteric regulation.  A common theme for all studied RNRs is a flexible loop that mediates modulatory effects from the allosteric specificity site (s-site) to the catalytic site for discrimination between the four substrates.  Much less is known about the allosteric activity site (a-site), which functions as an on-off switch for the enzyme's overall activity by binding ATP (activator) or dATP (inhibitor).  The two nucleotides induce formation of different enzyme oligomers, and a recent structure of a dATP-inhibited α6β2 complex from yeast suggested how its subunits interacted non-productively.  Interestingly, the oligomers formed and the details of their allosteric regulation differ between eukaryotes and Escherichia coli.  Nevertheless, these differences serve a common purpose in an essential enzyme whose allosteric regulation might date back to the era when the mol. mechanisms behind the central dogma evolved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptG-F1ry53D7Vg90H21EOLACvtfcHk0ljnGaSu_4YbvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1CitLzF&md5=eef605f2814628042efc94982d098682</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.3109%2F10409238.2011.630372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F10409238.2011.630372%26sid%3Dliteratum%253Aachs%26aulast%3DHofer%26aufirst%3DA.%26aulast%3DCrona%26aufirst%3DM.%26aulast%3DLogan%26aufirst%3DD.%2BT.%26aulast%3DSjoberg%26aufirst%3DB.%2BM.%26atitle%3DDNA%2520building%2520blocks%253A%2520keeping%2520control%2520of%2520manufacture%26jtitle%3DCrit.%2520Rev.%2520Biochem.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D47%26spage%3D50%26epage%3D63%26doi%3D10.3109%2F10409238.2011.630372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Estebanez-Perpina, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnold, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodrigues, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mar, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bateman, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallai, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baxter, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guy, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fletterick, R. J.</span><span> </span><span class="NLM_article-title">A surface on the androgen receptor that allosterically regulates coactivator binding</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">16074</span><span class="NLM_x">–</span> <span class="NLM_lpage">16079</span><span class="refDoi"> DOI: 10.1073/pnas.0708036104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1073%2Fpnas.0708036104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=17911242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtF2hu7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=16074-16079&author=E.+Estebanez-Perpinaauthor=L.+A.+Arnoldauthor=P.+Nguyenauthor=E.+D.+Rodriguesauthor=E.+Marauthor=R.+Batemanauthor=P.+Pallaiauthor=K.+M.+Shokatauthor=J.+D.+Baxterauthor=R.+K.+Guyauthor=P.+Webbauthor=R.+J.+Fletterick&title=A+surface+on+the+androgen+receptor+that+allosterically+regulates+coactivator+binding&doi=10.1073%2Fpnas.0708036104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">A surface on the androgen receptor that allosterically regulates coactivator binding</span></div><div class="casAuthors">Estebanez-Perpina, Eva; Arnold, Alexander A.; Nguyen, Phuong; Rodrigues, Edson Delgado; Mar, Ellena; Bateman, Raynard; Pallai, Peter; Shokat, Kevan M.; Baxter, John D.; Guy, R. Kiplin; Webb, Paul; Fletterick, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">16074-16079</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Current approaches to inhibit nuclear receptor (NR) activity target the hormone binding pocket but face limitations.  The authors have proposed that inhibitors, which bind to nuclear receptor surfaces that mediate assembly of the receptor's binding partners, might overcome some of these limitations.  The androgen receptor (AR) plays a central role in prostate cancer, but conventional inhibitors lose effectiveness as cancer treatments because anti-androgen resistance usually develops.  The authors conducted functional and x-ray screens to identify compds. that bind the AR surface and block binding of coactivators for AR activation function 2 (AF-2).  Four compds. that block coactivator binding in soln. with IC50 ≈ 50 μM and inhibit AF-2 activity in cells were detected: three nonsteroidal antiinflammatory drugs and the thyroid hormone 3,3',5-triiodothyroacetic acid.  Although visualization of compds. at the AR surface reveals weak binding at AF-2, the most potent inhibitors bind preferentially to a previously unknown regulatory surface cleft termed binding function (BF)-3, which is a known target for mutations in prostate cancer and androgen insensitivity syndrome.  X-ray structural anal. reveals that 3,3',5-triiodothyroacetic acid binding to BF-3 remodels the adjacent interaction site AF-2 to weaken coactivator binding.  Mutation of residues that form BF-3 inhibits AR function and AR AF-2 activity.  The authors propose that BF-3 is a previously unrecognized allosteric regulatory site needed for AR activity in vivo and a possible pharmaceutical target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa7awt1cHxHrVg90H21EOLACvtfcHk0lgV89MSc6bnYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtF2hu7bM&md5=23e84e3c667ef01f560246a25fc227e1</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0708036104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0708036104%26sid%3Dliteratum%253Aachs%26aulast%3DEstebanez-Perpina%26aufirst%3DE.%26aulast%3DArnold%26aufirst%3DL.%2BA.%26aulast%3DNguyen%26aufirst%3DP.%26aulast%3DRodrigues%26aufirst%3DE.%2BD.%26aulast%3DMar%26aufirst%3DE.%26aulast%3DBateman%26aufirst%3DR.%26aulast%3DPallai%26aufirst%3DP.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DBaxter%26aufirst%3DJ.%2BD.%26aulast%3DGuy%26aufirst%3DR.%2BK.%26aulast%3DWebb%26aufirst%3DP.%26aulast%3DFletterick%26aufirst%3DR.%2BJ.%26atitle%3DA%2520surface%2520on%2520the%2520androgen%2520receptor%2520that%2520allosterically%2520regulates%2520coactivator%2520binding%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D16074%26epage%3D16079%26doi%3D10.1073%2Fpnas.0708036104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Ciulli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span> </span><span class="NLM_article-title">Probing hot spots at protein-ligand binding sites: a fragment-based approach using biophysical methods</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">4992</span><span class="NLM_x">–</span> <span class="NLM_lpage">5000</span><span class="refDoi"> DOI: 10.1021/jm060490r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060490r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1SksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=4992-5000&author=A.+Ciulliauthor=G.+Williamsauthor=A.+G.+Smithauthor=T.+L.+Blundellauthor=C.+Abell&title=Probing+hot+spots+at+protein-ligand+binding+sites%3A+a+fragment-based+approach+using+biophysical+methods&doi=10.1021%2Fjm060490r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Probing Hot Spots at Protein-Ligand Binding Sites: A Fragment-Based Approach Using Biophysical Methods</span></div><div class="casAuthors">Ciulli, Alessio; Williams, Glyn; Smith, Alison G.; Blundell, Tom L.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4992-5000</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mapping interactions at protein-ligand binding sites is an important aspect of understanding many biol. reactions and a key part of drug design.  In this paper, we have used a fragment-based approach to probe "hot spots" at the cofactor-binding site of a model dehydrogenase, Escherichia coli ketopantoate reductase.  Our strategy involved the breaking down of NADPH (Kd = 300 nM) into smaller fragments and the biophys. characterization of their binding using WaterLOGSY NMR spectroscopy, isothermal titrn. calorimetry (ITC), and inhibition studies.  The weak binding affinities of fragments were measured by direct ITC titrns. under low c value conditions.  The 2'-phosphate and the reduced nicotinamide groups were found to contribute a large part of the binding energy.  A combination of ITC and site-directed mutagenesis enabled us to locate the fragments at sep. hot spots on opposite ends of the cofactor-binding site.  This study has identified structural determinants for cofactor recognition that represent a blueprint for future inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAWWHBHKxvxbVg90H21EOLACvtfcHk0lgV89MSc6bnYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1SksL0%253D&md5=b6788ad16d6009d7cacd3e429b13523b</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm060490r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060490r%26sid%3Dliteratum%253Aachs%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DWilliams%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DA.%2BG.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DProbing%2520hot%2520spots%2520at%2520protein-ligand%2520binding%2520sites%253A%2520a%2520fragment-based%2520approach%2520using%2520biophysical%2520methods%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D4992%26epage%3D5000%26doi%3D10.1021%2Fjm060490r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Edink, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rucktooa, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Retra, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akdemir, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nahar, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuiderveld, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Elk, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janssen, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Nierop, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Muijlwijk-Koezen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smit, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sixma, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span> </span><span class="NLM_article-title">Fragment growing induces conformational changes in acetylcholine-binding protein: a structural and thermodynamic analysis</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">5363</span><span class="NLM_x">–</span> <span class="NLM_lpage">5371</span><span class="refDoi"> DOI: 10.1021/ja110571r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja110571r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFeisrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=5363-5371&author=E.+Edinkauthor=P.+Rucktooaauthor=K.+Retraauthor=A.+Akdemirauthor=T.+Naharauthor=O.+Zuiderveldauthor=R.+van+Elkauthor=E.+Janssenauthor=P.+van+Nieropauthor=J.+van+Muijlwijk-Koezenauthor=A.+B.+Smitauthor=T.+K.+Sixmaauthor=R.+Leursauthor=I.+J.+de+Esch&title=Fragment+growing+induces+conformational+changes+in+acetylcholine-binding+protein%3A+a+structural+and+thermodynamic+analysis&doi=10.1021%2Fja110571r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment Growing Induces Conformational Changes in Acetylcholine-Binding Protein: A Structural and Thermodynamic Analysis</span></div><div class="casAuthors">Edink, Ewald; Rucktooa, Prakash; Retra, Kim; Akdemir, Atilla; Nahar, Tariq; Zuiderveld, Obbe; van Elk, Rene; Janssen, Elwin; van Nierop, Pim; van Muijlwijk-Koezen, Jacqueline; Smit, August B.; Sixma, Titia K.; Leurs, Rob; de Esch, Iwan J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5363-5371</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Optimization of fragment hits toward high-affinity lead compds. is a crucial aspect of fragment-based drug discovery (FBDD).  In the current study, we have successfully optimized a fragment by growing into a ligand-inducible subpocket of the binding site of acetylcholine-binding protein (AChBP).  This protein is a sol. homolog of the ligand binding domain (LBD) of Cys-loop receptors.  The fragment optimization was monitored with X-ray structures of ligand complexes and systematic thermodn. analyses using surface plasmon resonance (SPR) biosensor anal. and isothermal titrn. calorimetry (ITC).  Using site-directed mutagenesis and AChBP from different species, we find that specific changes in thermodn. binding profiles, are indicative of interactions with the ligand-inducible subpocket of AChBP.  This study illustrates that thermodn. anal. provides valuable information on ligand binding modes and is complementary to affinity data when guiding rational structure- and fragment-based discovery approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7-gLHcdzLv7Vg90H21EOLACvtfcHk0lgV89MSc6bnYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFeisrc%253D&md5=413798f2128e309545de77f8e995b565</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fja110571r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja110571r%26sid%3Dliteratum%253Aachs%26aulast%3DEdink%26aufirst%3DE.%26aulast%3DRucktooa%26aufirst%3DP.%26aulast%3DRetra%26aufirst%3DK.%26aulast%3DAkdemir%26aufirst%3DA.%26aulast%3DNahar%26aufirst%3DT.%26aulast%3DZuiderveld%26aufirst%3DO.%26aulast%3Dvan%2BElk%26aufirst%3DR.%26aulast%3DJanssen%26aufirst%3DE.%26aulast%3Dvan%2BNierop%26aufirst%3DP.%26aulast%3Dvan%2BMuijlwijk-Koezen%26aufirst%3DJ.%26aulast%3DSmit%26aufirst%3DA.%2BB.%26aulast%3DSixma%26aufirst%3DT.%2BK.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26atitle%3DFragment%2520growing%2520induces%2520conformational%2520changes%2520in%2520acetylcholine-binding%2520protein%253A%2520a%2520structural%2520and%2520thermodynamic%2520analysis%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D5363%26epage%3D5371%26doi%3D10.1021%2Fja110571r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1750</span><span class="NLM_x">–</span> <span class="NLM_lpage">1759</span><span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0liHQnBYXpPykg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">–</span> <span class="NLM_lpage">1749</span><span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0liHQnBYXpPykg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanschagrin, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span> </span><span class="NLM_article-title">Extra precision glide: docking and scoring incorporating a model of hydrophobic enclosure for protein-ligand complexes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6177</span><span class="NLM_x">–</span> <span class="NLM_lpage">6196</span><span class="refDoi"> DOI: 10.1021/jm051256o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+precision+glide%3A+docking+and+scoring+incorporating+a+model+of+hydrophobic+enclosure+for+protein-ligand+complexes&doi=10.1021%2Fjm051256o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span></div><div class="casAuthors">Friesner, Richard A.; Murphy, Robert B.; Repasky, Matthew P.; Frye, Leah L.; Greenwood, Jeremy R.; Halgren, Thomas A.; Sanschagrin, Paul C.; Mainz, Daniel T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6177-6196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel scoring function to est. protein-ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol.  In addn. to unique water desolvation energy terms, protein-ligand structural motifs leading to enhanced binding affinity are included:(1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral-neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged-charged hydrogen bonds.  The XP scoring function and docking protocol have been developed to reproduce exptl. binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, resp.) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance.  Enrichment results demonstrate the importance of the novel XP mol. recognition and water scoring in sepg. active and inactive ligands and avoiding false positives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZiwN12vaY7Vg90H21EOLACvtfcHk0liHQnBYXpPykg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D&md5=ea428c82ead0d8c27f8c1a7b694a1edf</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520precision%2520glide%253A%2520docking%2520and%2520scoring%2520incorporating%2520a%2520model%2520of%2520hydrophobic%2520enclosure%2520for%2520protein-ligand%2520complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196%26doi%3D10.1021%2Fjm051256o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Pace, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajdos, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fee, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimsley, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, T.</span><span> </span><span class="NLM_article-title">How to measure and predict the molar absorption coefficient of a protein</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">2411</span><span class="NLM_x">–</span> <span class="NLM_lpage">2423</span><span class="refDoi"> DOI: 10.1002/pro.5560041120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1002%2Fpro.5560041120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=8563639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADyaK2MXpsVGqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1995&pages=2411-2423&author=C.+N.+Paceauthor=F.+Vajdosauthor=L.+Feeauthor=G.+Grimsleyauthor=T.+Gray&title=How+to+measure+and+predict+the+molar+absorption+coefficient+of+a+protein&doi=10.1002%2Fpro.5560041120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">How to measure and predict the molar absorption coefficient of a protein</span></div><div class="casAuthors">Pace, C. Nick; Vajdos, Felix; Fee, Lanette; Grimsley, Gerald; Gray, Theronica</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2411-23</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">The molar absorption coeff., ε, of a protein is usually based on concns. measured by dry wt., nitrogen, or amino acid anal.  The studies reported here suggest that the Edelhoch method is the best method for measuring ε for a protein.  (This method is described by Gill and von Hippel [1989, Anal Biochem 182:319-326] and is based on data from Edelhoch [1967, Biochem. 6:1948-1954].).  The av. ε values for these chromophores in a sample of 18 well-characterized proteins have been estd., and the ε values in water, propanol, 6 M guanidine hydrochloride (GdnHCl), and 8 M urea have been measured.  For Trp, the av. ε values for the proteins are intermediate between those measured in 6 M GdnHCl and those measured in propanol.  Based on a sample of 116 measured ε values for 80 proteins, the ε at 280 nm of a folded protein in water, ε(280), can best be predicted with this equation L ε(280) (M-1 cm-1) = (#Trp)(5,500) + (#Tyr)(1,490) + (#cystine)(125).  These ε(280) values are quite reliable for proteins contg. Trp residues, and less reliable for proteins that do not.  However, the Edelhoch method is convenient and accurate, and the best approach is to measure rather than predict ε.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDyrAj6uk11bVg90H21EOLACvtfcHk0lgv-ay4MSe2bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXpsVGqt7k%253D&md5=033237a125c7486331bf5180047c3cd4</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2Fpro.5560041120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.5560041120%26sid%3Dliteratum%253Aachs%26aulast%3DPace%26aufirst%3DC.%2BN.%26aulast%3DVajdos%26aufirst%3DF.%26aulast%3DFee%26aufirst%3DL.%26aulast%3DGrimsley%26aufirst%3DG.%26aulast%3DGray%26aufirst%3DT.%26atitle%3DHow%2520to%2520measure%2520and%2520predict%2520the%2520molar%2520absorption%2520coefficient%2520of%2520a%2520protein%26jtitle%3DProtein%2520Sci.%26date%3D1995%26volume%3D4%26spage%3D2411%26epage%3D2423%26doi%3D10.1002%2Fpro.5560041120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Krippendorff, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lienau, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reichel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huisinga, W.</span><span> </span><span class="NLM_article-title">Optimizing classification of drug-drug interaction potential for CYP450 isoenzyme inhibition assays in early drug discovery</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">92</span><span class="NLM_x">–</span> <span class="NLM_lpage">99</span><span class="refDoi"> DOI: 10.1177/1087057106295897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=10.1177%2F1087057106295897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=17130250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtFGgsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=92-99&author=B.+F.+Krippendorffauthor=P.+Lienauauthor=A.+Reichelauthor=W.+Huisinga&title=Optimizing+classification+of+drug-drug+interaction+potential+for+CYP450+isoenzyme+inhibition+assays+in+early+drug+discovery&doi=10.1177%2F1087057106295897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing classification of drug-drug interaction potential for CYP450 isoenzyme inhibition assays in early drug discovery</span></div><div class="casAuthors">Krippendorff, Ben-Fillippo; Lienau, Philip; Reichel, Andreas; Huisinga, Wilhelm</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">92-99</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">In drug discovery, the potential of cytochrome P 450 inhibition of new chem. entities is frequently quantified in terms of IC50 values.  In early drug discovery, a risk classification into low, medium, or high potential inhibitors is often sufficient for ranking and prioritizing of compds.  Although often 6 or more inhibitor concns. are used to det. the IC50 value, the question arises whether it is possible to predict the risk class based on fewer inhibitor concns. with comparable reliability.  In this article, the authors propose a new integrated 2-point method with inhibitor concns. chosen in accordance with the risk classification.  They analyze its predictive power and the feasibility of not only classifying the compds. into different risk classes but also ranking those compds. that have been binned into the middle risk class.  The proposed integrated 2-point method is thus highly suitable for automation.  Altogether, it maintains the quality of the prediction while considerably reducing time and cost.  The proposed method is applicable to other IC50 assays and risk classifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouHK1-JW_iC7Vg90H21EOLACvtfcHk0lgv-ay4MSe2bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtFGgsL8%253D&md5=09ffac9868b98f80c5af46b4bdc5f349</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1177%2F1087057106295897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057106295897%26sid%3Dliteratum%253Aachs%26aulast%3DKrippendorff%26aufirst%3DB.%2BF.%26aulast%3DLienau%26aufirst%3DP.%26aulast%3DReichel%26aufirst%3DA.%26aulast%3DHuisinga%26aufirst%3DW.%26atitle%3DOptimizing%2520classification%2520of%2520drug-drug%2520interaction%2520potential%2520for%2520CYP450%2520isoenzyme%2520inhibition%2520assays%2520in%2520early%2520drug%2520discovery%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2007%26volume%3D12%26spage%3D92%26epage%3D99%26doi%3D10.1177%2F1087057106295897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PAW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PAW','PDB','3PAW'); return false;">PDB: 3PAW</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i22"><a href="/doi/suppl/10.1021/acs.jmedchem.5b00929">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_76652"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b00929">10.1021/acs.jmedchem.5b00929</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Experimental methods for virtual screening, RR inhibition assays, inhibition mechanism, growth inhibition screening assays for determining cellular toxicity, crystallization of hRRM1, data collection, and structure determination; docking scores for Schrödinger; fluorescence quenching data; medicinal chemistry data; growth inhibition data for all compounds tested in MDA-MB-231 breast cancer and HCT-116 colon cancer cell lines; median effect doses (Dm) for compound <b>1</b>; interactions between compound <b>4</b> and hRRM1 (within 5 Å); enzyme kinetic mechanism of compound <b>4</b> inhibition of hRRM1; docking poses for the 10 compounds in <a class="ref internalNav" href="#tbl1">Table 1</a> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b00929/suppl_file/jm5b00929_si_001.pdf">jm5b00929_si_001.pdf (10.04 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">4X3V.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b00929&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b00929%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2015.58.issue-24%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b00929" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67997745eae3d1df","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
