
<html lang="en"     class="pb-page"  data-request-id="80135c47-afe0-461d-a596-0ad78c6b807f"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.1c00401;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2021.64.issue-12;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="An Exploration of Chemical Properties Required for Cooperative Stabilization of the 14-3-3 Interaction with NF-κB—Utilizing a Reversible Covalent Tethering Approach" /></meta><meta name="dc.Creator" content="Madita  Wolter" /></meta><meta name="dc.Creator" content="Dario  Valenti" /></meta><meta name="dc.Creator" content="Peter J.  Cossar" /></meta><meta name="dc.Creator" content="Stanimira  Hristeva" /></meta><meta name="dc.Creator" content="Laura M.  Levy" /></meta><meta name="dc.Creator" content="Thorsten  Genski" /></meta><meta name="dc.Creator" content="Torsten  Hoffmann" /></meta><meta name="dc.Creator" content="Luc  Brunsveld" /></meta><meta name="dc.Creator" content="Dimitrios  Tzalis" /></meta><meta name="dc.Creator" content="Christian  Ottmann" /></meta><meta name="dc.Description" content="Protein–protein modulation has emerged as a proven approach to drug discovery. While significant progress has been gained in developing protein–protein interaction (PPI) inhibitors, the orthogonal ..." /></meta><meta name="Description" content="Protein–protein modulation has emerged as a proven approach to drug discovery. While significant progress has been gained in developing protein–protein interaction (PPI) inhibitors, the orthogonal ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 2, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00401" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 The Authors. Published by American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00401" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00401" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00401" /></link>
        
    
    

<title>An Exploration of Chemical Properties Required for Cooperative Stabilization of the 14-3-3 Interaction with NF-κB—Utilizing a Reversible Covalent Tethering Approach | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00401" /></meta><meta property="og:title" content="An Exploration of Chemical Properties Required for Cooperative Stabilization of the 14-3-3 Interaction with NF-κB—Utilizing a Reversible Covalent Tethering Approach" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0014.jpeg" /></meta><meta property="og:description" content="Protein–protein modulation has emerged as a proven approach to drug discovery. While significant progress has been gained in developing protein–protein interaction (PPI) inhibitors, the orthogonal approach of PPI stabilization lacks established methodologies for drug design. Here, we report the systematic ″bottom-up″ development of a reversible covalent PPI stabilizer. An imine bond was employed to anchor the stabilizer at the interface of the 14-3-3/p65 complex, leading to a molecular glue that elicited an 81-fold increase in complex stabilization. Utilizing protein crystallography and biophysical assays, we deconvoluted how chemical properties of a stabilizer translate to structural changes in the ternary 14-3-3/p65/molecular glue complex. Furthermore, we explore how this leads to high cooperativity and increased stability of the complex." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00401"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00401">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00401&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00401&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00401&amp;href=/doi/10.1021/acs.jmedchem.1c00401" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 12</span><span class="cit-fg-pageRange">, 8423-8436</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/12" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00397" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00404" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">An Exploration of Chemical Properties Required for Cooperative Stabilization of the 14-3-3 Interaction with NF-κB—Utilizing a Reversible Covalent Tethering Approach</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Madita Wolter</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Madita Wolter</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Engineering, Laboratory of Chemical Biology and Institute for Complex Molecular Systems, Eindhoven University of Technology, P.O. Box 513, 5600 MB Eindhoven, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Madita++Wolter">Madita Wolter</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-1430-9589" title="Orcid link">https://orcid.org/0000-0003-1430-9589</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dario Valenti</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dario Valenti</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Engineering, Laboratory of Chemical Biology and Institute for Complex Molecular Systems, Eindhoven University of Technology, P.O. Box 513, 5600 MB Eindhoven, The Netherlands</div><div class="loa-info-affiliations-info">Medicinal Chemistry, Taros Chemicals GmbH & Co. KG, Emil-Figge-Straße 76a, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dario++Valenti">Dario Valenti</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter J. Cossar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter J. Cossar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Engineering, Laboratory of Chemical Biology and Institute for Complex Molecular Systems, Eindhoven University of Technology, P.O. Box 513, 5600 MB Eindhoven, The Netherlands</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+J.++Cossar">Peter J. Cossar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stanimira Hristeva</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stanimira Hristeva</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Taros Chemicals GmbH & Co. KG, Emil-Figge-Straße 76a, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stanimira++Hristeva">Stanimira Hristeva</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Laura M. Levy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laura M. Levy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Taros Chemicals GmbH & Co. KG, Emil-Figge-Straße 76a, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laura+M.++Levy">Laura M. Levy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thorsten Genski</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thorsten Genski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Taros Chemicals GmbH & Co. KG, Emil-Figge-Straße 76a, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thorsten++Genski">Thorsten Genski</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Torsten Hoffmann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Torsten Hoffmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Taros Chemicals GmbH & Co. KG, Emil-Figge-Straße 76a, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Torsten++Hoffmann">Torsten Hoffmann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Luc Brunsveld</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Luc Brunsveld</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Engineering, Laboratory of Chemical Biology and Institute for Complex Molecular Systems, Eindhoven University of Technology, P.O. Box 513, 5600 MB Eindhoven, The Netherlands</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#abc785c9d9dec5d8ddcec7cfebdfdece85c5c7"><span class="__cf_email__" data-cfemail="d6baf8b4a4a3b8a5a0b3bab296a2a3b3f8b8ba">[email protected]</span></a>. Phone: +31 40−247-2870.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Luc++Brunsveld">Luc Brunsveld</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5675-511X" title="Orcid link">https://orcid.org/0000-0001-5675-511X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dimitrios Tzalis</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dimitrios Tzalis</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Taros Chemicals GmbH & Co. KG, Emil-Figge-Straße 76a, 44227 Dortmund, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#adc9d9d7ccc1c4deedd9ccdfc2de83c9c8"><span class="__cf_email__" data-cfemail="7e1a0a041f12170d3e0a1f0c110d501a1b">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dimitrios++Tzalis">Dimitrios Tzalis</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Christian Ottmann</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christian Ottmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Engineering, Laboratory of Chemical Biology and Institute for Complex Molecular Systems, Eindhoven University of Technology, P.O. Box 513, 5600 MB Eindhoven, The Netherlands</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#e380cd8c97978e828d8da3979686cd8d8f"><span class="__cf_email__" data-cfemail="5734793823233a3639391723223279393b">[email protected]</span></a>. Phone: +31 40−247-2835.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christian++Ottmann">Christian Ottmann</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-7315-0315" title="Orcid link">https://orcid.org/0000-0001-7315-0315</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00401&amp;href=/doi/10.1021%2Facs.jmedchem.1c00401" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 12</span><span class="cit-pageRange">, 8423–8436</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 2, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>5 March 2021</li><li><span class="item_label"><b>Published</b> online</span>2 June 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 June 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00401" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00401</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 The Authors. Published by American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="https://acsopenscience.org/open-access/licensing-options/"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by-nc"><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_cc-1612971093887.svg" alt="CC: Creative Commons" /></img><span><strong>CC: </strong>Creative Commons</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_by-1612971093257.svg" alt="BY: Credit must be given to the creator" /></img><span><strong>BY: </strong>Credit must be given to the creator</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_nc-1612971093307.svg" alt="NC: Only noncommercial uses of the work are permitted" /></img><span><strong>NC: </strong>Only noncommercial uses of the work are permitted</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_nd-1612971093637.svg" alt="ND: No derivatives or adaptations of the work are permitted" /></img><span><strong>ND: </strong>No derivatives or adaptations of the work are permitted</span></a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00401"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1309</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00401" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;An Exploration of Chemical Properties Required for Cooperative Stabilization of the 14-3-3 Interaction with NF-κB—Utilizing a Reversible Covalent Tethering Approach&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Madita&quot;,&quot;last_name&quot;:&quot;Wolter&quot;},{&quot;first_name&quot;:&quot;Dario&quot;,&quot;last_name&quot;:&quot;Valenti&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;J. Cossar&quot;},{&quot;first_name&quot;:&quot;Stanimira&quot;,&quot;last_name&quot;:&quot;Hristeva&quot;},{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;M. Levy&quot;},{&quot;first_name&quot;:&quot;Thorsten&quot;,&quot;last_name&quot;:&quot;Genski&quot;},{&quot;first_name&quot;:&quot;Torsten&quot;,&quot;last_name&quot;:&quot;Hoffmann&quot;},{&quot;first_name&quot;:&quot;Luc&quot;,&quot;last_name&quot;:&quot;Brunsveld&quot;},{&quot;first_name&quot;:&quot;Dimitrios&quot;,&quot;last_name&quot;:&quot;Tzalis&quot;},{&quot;first_name&quot;:&quot;Christian&quot;,&quot;last_name&quot;:&quot;Ottmann&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;02&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;8423-8436&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00401&quot;},&quot;abstract&quot;:&quot;Protein–protein modulation has emerged as a proven approach to drug discovery. While significant progress has been gained in developing protein–protein interaction (PPI) inhibitors, the orthogonal approach of PPI stabilization lacks established methodologies for drug design. Here, we report the systematic ″bottom-up″ development of a reversible covalent PPI stabilizer. An imine bond was employed to anchor the stabilizer at the interface of the 14-3-3/p65 complex, leading to a molecular glue that elicited an 81-fold increase in complex stabilization. Utilizing protein crystallography and biophysical assays, we deconvoluted how chemical properties of a stabilizer translate to structural changes in the ternary 14-3-3/p65/molecular glue complex. Furthermore, we explore how this leads to high cooperativity and increased stability of the complex.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00401&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00401" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00401&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00401" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00401&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00401" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00401&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00401&amp;href=/doi/10.1021/acs.jmedchem.1c00401" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00401" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00401" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (11 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00401&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00401%26sid%3Dliteratum%253Aachs%26pmid%3D34076416%26genre%3Darticle%26aulast%3DWolter%26date%3D2021%26atitle%3DAn%2BExploration%2Bof%2BChemical%2BProperties%2BRequired%2Bfor%2BCooperative%2BStabilization%2Bof%2Bthe%2B14-3-3%2BInteraction%2Bwith%2BNF-%25CE%25BAB%25E2%2580%2594Utilizing%2Ba%2BReversible%2BCovalent%2BTethering%2BApproach%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D12%26spage%3D8423%26epage%3D8436%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=291120" title="Hydrophobicity">Hydrophobicity</a>,</li><li><a href="/action/doSearch?ConceptID=291192" title="Stabilization">Stabilization</a>,</li><li><a href="/action/doSearch?ConceptID=292219" title="Titration">Titration</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/12" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Protein–protein modulation has emerged as a proven approach to drug discovery. While significant progress has been gained in developing protein–protein interaction (PPI) inhibitors, the orthogonal approach of PPI stabilization lacks established methodologies for drug design. Here, we report the systematic ″bottom-up″ development of a reversible covalent PPI stabilizer. An imine bond was employed to anchor the stabilizer at the interface of the 14-3-3/p65 complex, leading to a molecular glue that elicited an 81-fold increase in complex stabilization. Utilizing protein crystallography and biophysical assays, we deconvoluted how chemical properties of a stabilizer translate to structural changes in the ternary 14-3-3/p65/molecular glue complex. Furthermore, we explore how this leads to high cooperativity and increased stability of the complex.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72876" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72876" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein–protein interaction (PPI) modulation has fundamentally changed drug discovery, expanding the druggable proteome.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1−4)</a> This shift in perspective has rapidly expanded the number of entry points for novel therapeutic intervention for many diseases.<a onclick="showRef(event, 'ref2 ref5'); return false;" href="javascript:void(0);" class="ref ref2 ref5">(2,5)</a> The development of PPI inhibitors has progressed steadily, leading to the emergence of clinically approved PPI inhibitors such as Venetoclax for chronic lymphocytic leukemia or small lymphocytic lymphoma.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> In contrast, PPI stabilization has emerged slowly, with the retrospective elucidation of the mode of action of clinically approved immunosuppressants, rapamycin,<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> and the cytostatic paclitaxel.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> Successes of immunomodulators, lenalidomide, and cooperative PROTACs like AT1 provide valuable ″proofs-of-concepts″ for modulating cell homeostasis using synthetically derived PPI stabilizers.<a onclick="showRef(event, 'ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12">(10−12)</a> However, a systematic understanding of how to design small molecules that stabilize PPIs is urgently needed. The disconnection between these orthogonal PPI modulation approaches is primarily driven by a poor understanding of drug design rules for PPI stabilizers due to the complicated kinetics of dynamically interacting components that form the complex,<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> shallow surface pockets, and limited or no structural information about the essential interactions that drive complex formation. A major obstacle for the development of PPI stabilizers is identifying chemical starting points, as the transient nature of complexes makes detecting ligand binding challenging. Further, affinity of a ligand to a complex is not necessarily correlated to the ligand’s stabilizing activity.</div><div class="NLM_p">There is a significant need to develop novel drug discovery methodologies and alternative types of chemical matter to identify hit compounds for the development of PPI stabilizers. Fragment-based drug discovery has emerged as a promising approach leading to several clinically approved drugs.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> Since the clinical approval of ibrutinib, the stigma relating to covalent drugs and fragments has dissipated,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> resulting in a rise in the application of covalent tethering, particularly for challenging drug targets.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> The emergence of fragment tethering approaches using covalent chemistry has simplified the identification of hit compounds as binding fragments are easily detectable, and instrumentation is not reliant on detecting transient fragments/complex interactions to identify hits.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> Covalent fragment screening uses a covalent bond that is formed between a natural or engineered amino acid inside the targeted binding pocket and a reactive handle installed on the fragments.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Recent fragment screening approaches utilizing disulfide tethering have proven successful in identifying hit fragments for PPI stabilization. Such site-directed fragment identification has recently been reported by us for stabilization of the protein 14-3-3 in complex with estrogen receptor α (ERα)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and estrogen related receptor gamma (ERRγ)<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> utilizing a disulfide tethering approach. We have also reported a new aldimine-based covalent tethering approach for the 14-3-3/p65 subunit of NF-κB PPI.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> This approach employs a dynamic imine-based tether to anchor the fragment at the interface of the composite pocket. Lysine tethering is a particularly attractive approach as the large percentage of the proteinogenic amino acids makes it amendable to a wide array of chemical probe development projects.<a onclick="showRef(event, 'ref26 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28 ref29">(26−29)</a></div><div class="NLM_p">Targeting the p65 subunit of NF-κB is of specific interest since NF-κB is a homo- and/or heterodimeric transcription factor involved in the regulation of immune responses, cell proliferation, and inflammation and therefore is connected to cancer and autoimmune diseases, among others.<a onclick="showRef(event, 'ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32">(30−32)</a> Attempts to directly inhibit the transcriptional activity of NF-κB have typically failed due to the inability to identify NF-κB-targeting matter.<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a> Interestingly, increased transcriptional activity of p65 has been correlated with downregulation of 14-3-3 in studies on ischemia–reperfusion and breast cancer.<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> Also, upregulation of 14-3-3 has been shown to favor cytosolic localization of p65,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> subsequently preventing transcriptional activity. Stabilization of the 14-3-3/p65 complex could therefore furnish a novel entry point for targeting NF-κB and enabling a controlled therapeutic intervention.</div><div class="NLM_p">Point mutational studies on p65 revealed three potential 14-3-3 binding sites surrounding the phosphorylation sites S45, S281, and S340.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Binding affinities and structural information for two of these sites, pS45 and pS281, were reported, showing the direct physical interaction between both proteins.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Benzaldehyde-based molecular fragments were shown to bind specifically to Lys122 of 14-3-3 via imine bond formation, thereby stabilizing the interaction with the p65 motif around phosphorylation site pS45 via hydrophobic contacts with p65.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div><div class="NLM_p last">Here, we extend upon our initial communication reporting on an imine-based site-directed fragment approach to develop a 14-3-3/p65 molecular glue.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Critical to the development of molecular glues is a robust understanding of molecular interactions and structural changes in the protein–protein–ligand complex that result in cooperative behavior. To understand the chemical properties that produce cooperative ligands, we employed a fragment extension design process using structural information gathered from X-ray crystallography soaking experiments and fluorescence anisotropy (FA) measurements. This enabled us to design initial fragments into molecular glues that showed stabilizing activity for the 14-3-3/p65 complex, culminating with the discovery of compound <b>24j</b> that elicits an 81-fold stabilizing effect on the 14-3-3/p65 complex.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03005" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03005" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We have previously shown that aldimine bond formation is highly selective for Lys122 of 14-3-3 that lies at the interface between 14-3-3 and p65 in the composite binding pocket (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). The enhanced selectivity for Lys122 is the result of a combination of the local hydrophobic character of the composite pocket, a lowered p<i>K</i><sub>a</sub> of the lysine side chain, and the templating effects of p65 binding.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Given the intrinsically disordered nature of large parts of the p65 subunit of NF-κB (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B), we utilized a 13-mer phosphopeptide representing (EGRSAG pSer45 IPGRRS) the recognition sequence of 14-3-3 to expedite chemical matter elucidation. Our initial investigation used X-ray crystal soaking experiments and a fragment library of commercially available aldehydes (34 fragments) to identify four key chemotypes that induced imine bond formation with Lys122 within the 14-3-3/p65 composite pocket (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C,D): methylsulfonyl (<b>1</b> and <b>2</b>), 1-nitro,3-hydroxybenzene (<b>3</b>), methyl acetamide (<b>4</b>), and five-membered <i>N</i>-heterocycles (<b>5</b>, <b>6</b>, and <b>7</b>). Biochemical assessment of fragments <b>1–7</b> used a fluorescence anisotropy (FA) assay. To assess the fragments’ capacity to induce a ternary complex, the fragments were titrated to a solution of 100 nM fluorescently labeled p65_45 peptide (FITC-βAla-EGRSAG pSer45 IPGRRS) and 50 μM of 14-3-3γ. The subsequent outputs are herein termed ″half maximal complex concentration (CC<sub>50</sub>)″. Results from the assay showed that these fragments did not increase complex formation. However, the lack of increase in complexation at relevant fragment concentrations was in accordance with crystal data, which indicated little to no direct contacts between the fragments and the p65 peptide. Fragments that are detectable in crystallography experiments but do not induce a detectable effect on complex formation are termed ″silent-binders″. We sought to develop a structure–activity relationship (SAR) based upon these four chemotypes with the specific goal of developing molecular glues that orthosterically engage with p65 and stabilize the 14-3-3/p65 complex.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Exploring the p65/14-3-3 interface with imine forming fragments. (A) Overview of the composite binding pocket of 14-3-3 and the p65_45 peptide (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QHL">6QHL</a>).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> (B) Crystal structure of the NF-κB subunit p65 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1IKN">1IKN</a>,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> IκBα and p50 hidden for clarity) with known phosphorylated serine residues Ser281 and Ser45 labeled. (C) Interface of the ternary complex of <b>1</b>/p65_45/14-3-3 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YOW">6YOW</a>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Indicated are three subpockets (the roof of 14-3-3, the FC-binding pocket, and the deep binding pocket), which can be exploited by fragment extension. Shown are 14-3-3 (white sticks, surface and cartoon), p65_45 peptide (red sticks), and fragment <b>1</b> (cyan sticks). (D) Fragment hits based on X-crystallography screening with benzaldehyde binding to Lys122 of 14-3-3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To facilitate rapid optimization of the initial hit compounds into molecular glues, six focused libraries were designed based on the four chemotypes from the initial screen. Key to the development of molecular glues is understanding which specific interactions (hotspots) between the fragments and the 14-3-3/p65 complex lead to cooperative binding. Within our lab, we have developed a highly robust crystallography screening system to rapidly access X-ray crystal structures of soaked fragments. Utilizing this system, we gain unparalleled structural information that guides hit optimization.</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4">2.1.  Focused Library Development around the 3-Hydroxy,4-nitrobenzaldehyde Scaffold</h3><div class="NLM_p">Initial focused library development concentrated on extension of <b>3</b> based on crystal data, which showed complete coverage of the fragment by the electron density map, indicating a high occupancy for <b>3</b> within the composite pocket. Analysis of the X-ray crystal structure of <b>3</b> indicated that a relatively large solvent exposed pocket was present in front of the fragment formed by p65 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). We sought to explore this chemical space via extension of <b>3</b> from the 3-hydroxy position. To expedite exploration, focused libraries 1 and 2 were obtained from a commercial supplier and screened using X-ray crystallography (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Nitrobenzaldehyde structural analogues. (A) Initial fragment hit <b>3</b> covalently bound to the 14-3-3/p65_45 complex. The fragment can be extended from its <i>meta-</i>position (black arrow shows proposed vector for extension) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YOY">6YOY</a>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> (B) Ternary structure of <b>9a</b> (salmon sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NMH">7NMH</a>), p65_45 (red sticks), and 14-3-3 (white cartoon/sticks). Water molecules: red spheres; hydrogen bonds: black dashes. (C) Ternary structure of <b>10a</b>/p65_45/14-3-3 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJF">7BJF</a>). Details as in (B). (D) Ternary structure of <b>13e</b> (left; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJL">7BJL</a>), <b>13f</b> (middle; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJW">7BJW</a>), or <b>13l</b> (right; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BKH">7BKH</a>) binding to the p65_45/14-3-3 complex. Compounds are shown as yellow sticks; the rest are as described before. The 2Fo-Fc electron density map is contoured at 1σ for all.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Focused Fragment Libraries 1 to 3</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Analysis of focused libraries 1 and 2 using X-ray crystal soaking experiments showed three sulfonate fragments (<b>9a–c</b>) and four ester fragments (<b>10a, d, j</b>, and <b>m)</b> bound to Lys122 in the crystal structures (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B,C; <a href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_001.pdf" class="ext-link">Figure S1A,B; Table S1</a>). The various substituents elicited poor to moderate coverage by the electron density map, except for the sulfonates <b>9a</b> and <b>9b</b>. However, neither <b>9a</b> nor <b>9b</b> were active in functional FA assays (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_001.pdf" class="ext-link">Figure S1A</a>). For these smaller substitutes (<b>9a</b> and <b>9b</b>), no favorable contacts with the peptide could be observed, whereas the larger substitutions resulted in a loss of electron density, indicative of a high conformational freedom of the ester side chain (<b>9c</b>, <b>10a</b>, and <b>10j</b>; <a href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_001.pdf" class="ext-link">Figure S1A,B</a>).</div><div class="NLM_p last">An additional collection of fragments (focused library 3) was synthesized based on tertiary amines at the <i>meta</i>-position. The replacement of the phenol oxygen of <b>3</b> with a ternary amine increased the number of vectors for fragment extension, reduced conformational freedom, and enabled the exploration of a different chemical space. A one-step nucleophilic aromatic substitution using the key benzaldehyde intermediate <b>11</b> and the corresponding amine (<b>12a–u</b>) was employed to afford 21 analogues (<b>13a–u</b>) with yields ranging from 20 to 68% (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Focused fragment library 3 was then soaked into p65_45/14-3-3 crystals and tested in the FA assay. Of this collection, four fragments bound in crystallography experiments, with <b>13e</b>, <b>f</b>, <b>l</b>, and <b>q</b> showing significant electron density coverage (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_001.pdf" class="ext-link">Figure S1C, Table S1</a>). Notably, all four fragments contained six-membered saturated <i>N</i>-heterocyclic rings with polar functional groups that formed either direct hydrogen bonds with the backbone of p65 or water-mediated hydrogen bonds. The binding poses of these compounds were well defined, whereby the saturated <i>N</i>-heterocyclic ring extends toward the C-terminus of the p65 peptide. These results suggest that these fragments are shielded by the amphiphilic amino acids Pro47, Gly48, Arg49, and Arg50 of p65, which facilitate covalent tethering. Biophysical assessment of <b>13e</b>, <b>f</b>, <b>l</b>, and <b>q</b> using FA compound titration assays showed that the fragments did not elicit significant ternary complex formation at biochemically relevant concentrations (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_001.pdf" class="ext-link">Figure S1C</a>). Surprisingly, <b>13f</b> and <b>13l</b> showed no increased complex formation, considering that both fragments engage in hydrogen bonding with p65. Given the lack of 14-3-3/p65 increased complex formation in FA compound titration assays at concentrations practical for hit optimization, we shifted focus to the five-membered <i>N</i>-heterocycles chemotype.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6">2.2.  Focused Library Development around the 4-(1<i>H</i>-Imidazol-1-yl)benzaldehyde Scaffold</h3><div class="NLM_p">Fragment <b>5</b> was selected for fragment library development as the <i>para</i>-substitution proved to be more solvent exposed compared with <b>7</b> and the 1,3-substituted imidazole provided the possibility for fragment extension from three vectors of the <i>N</i>-heterocycle (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). A focused library of 13 fragments (focused library 4) was synthesized using a nucleophilic aromatic substitution reaction with cesium fluoride, triethylamine, 4-fluoro-nicotinaldehyde (<b>14</b>), and an array of substituted imidazoles (<b>15a–e</b>) or benzimidazoles (<b>15f–k</b>) (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). The starting reactant 4-fluoro-nicotinaldehyde (<b>14</b>) was used to improve the solubility of the fragments and to provide a further point for a polar interaction compared with <b>5</b>–<b>7</b>. The resulting library was then subjected to X-ray crystal soaking experiments and FA assay. Notably, fragments <b>16b</b>, <b>c</b>, <b>f</b>, and <b>g–k</b> were tested as mixtures of regioisomers.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Imidazole-benzaldehydes are stabilizing the p65_45/14-3-3 complex. (A) The imidazole moiety of <b>5</b> (teal sticks) offers three vectors for fragment growth (arrows) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YP3">6YP3</a>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> (B) Fragment induced p65_45/14-3-3γ ternary complex formation, measured with compound titrations in FA (r/mAU). The compounds were titrated to 50 μM 14-3-3γ and 100 nM p65_45. (C) Ternary structure of <b>16d</b> (light blue sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NM9">7NM9</a>) binding to the p65_45/14-3-3σΔC complex. (D) Mixture of structural isomers of <b>16j</b> and <b>16k</b>. (E, F) Crystal structure of <b>16k</b> binding to the p65_45/14-3-3σΔC complex (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NR7">7NR7</a>). The electron density map indicates two binding poses (E). Based on the position of the methoxy substitution, the 2-chloro substitution points either toward the peptide (<b>16k-5</b>) or to the FC pocket (<b>16k-6</b>). Both binding poses engage in hydrophobic contacts with 14-3-3 and p65 (sphere representation, F). For all: 14-3-3: white cartoon, surface, sticks; p65_45: red sticks; 2Fo-Fc electron density map (contoured at 1σ): gray mesh.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Focused Fragment Libraries 4 and 5</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Analysis of the fluorescence anisotropy assay identified that fragments <b>16d</b>, <b>e</b>, <b>j</b>, and <b>k</b> induced an increase in anisotropy (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B); however, only <b>16d</b>, <b>j</b>, and <b>k</b> were detected in the electron density map of soaking experiments. Notably, all three fragments showed a mixture of conformational poses (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C–F, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_001.pdf" class="ext-link">Figure S2, Table S2</a>). Both structural isomers of <b>16j</b> (2-methyl-5-methoxy-benzimidazole) and <b>16k</b> (2-chloro-5-methoxy-benzimidazole) were observed to bind within the composite binding pocket as result of the soaking experiment using mixtures of the structural isomers (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D–F). For instance, the chlorine atom in <b>16k</b> points toward the p65 peptide, while the methoxy-substitution can be detected in the 5-position (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E). For the other binding pose, the chlorine atom in <b>16k</b> points to the FC pocket, whereas the methoxy-substitution located in the 6-position is positioned above Ile46 of p65. Both structural isomers of <b>16k</b> engage in hydrophobic contacts with the roof of 14-3-3 and Ill46 of p65_45 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). Despite their similar binding poses, <b>16k</b> (CC<sub>50</sub> = 260 μM) elicited a significantly higher increase in anisotropy compared to <b>16j</b>, which induced a negligible ternary complex formation in the FA assay (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). The replacement of a chloro-moiety for a methyl group in <b>16k</b> (<b>16j</b>) on the imidazole ring has a significant impact on complex formation, resulting in diminished anisotropy.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8">2.3.  Focused Library Development around the 4-Formylbenzamide Scaffold</h3><div class="NLM_p">We next turned our attention to fragment extension based upon fragment <b>4</b>. Analysis of the co-crystal structure showed that the <i>N</i>-acetyl group of <b>4</b> probes up and toward the p65 peptide. To expediate the rapid parallel synthesis of focused library 5, the <i>N</i>-(4-formylphenyl)acetamide (<b>4</b>) was inverted to a <i>N</i>-substituted 4-formyl-benzamide (<b>19a–t</b>), improving the nucleophilicity of the amine in the corresponding amide coupling reactions as well as providing greater chemical diversity of the fragment extension (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Further, a previously published cluster analysis of occurring torsion (τ) angles of benzamide report a τ ≈ 30 and 150° between the benzene ring and the acetamide head group (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A).<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> We hypothesized that extension of the fragment from this vector provides the opportunity to engage with the p65 peptide and enhance fragment binding (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). Focused library 5 was synthesized using standard amide coupling conditions, 1-formylbenzoic acid (<b>17</b>), and amines <b>18a–t</b>. In total, 13 fragments (<b>19a–t</b>) with amide substitution in the <i>para</i>-position of the aldehyde functionality were synthesized, soaked in p65_45/14-3-3σΔC crystals, and tested in FA compound titrations. All fragments from library 5 were shown to bind within the composite binding pocket using X-ray crystallography (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_001.pdf" class="ext-link">Figure S3, Table S2</a>). Subsequent biophysical analysis identified that all compounds were silent binders, not inducing detectable formation of a ternary complex at assay relevant concentrations. X-ray co-crystallization experiments and an assessment of the C–C–C–N torsion angles provided an explanation for the lack of activity for this series of fragments. The electron density map of the aldehydes showed highly resolved electron density for the benzaldehyde ring and the amide of the benzamide. The carbonyl of the benzamide engages in polar contacts with the water shell of 14-3-3, thereby stabilizing this orientation of the fragments (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). The binding poses of the different R-substitutions are poorly resolved by the electron density map, suggesting a high level of conformational freedom and a high-level entropy of the substitutes, unfavorable for binding. Assessment of this library of fragments showed the R-substitution pointing toward the solution above the p65 peptide or Asp215 of 14-3-3 with limited possibilities to engage in favorable contacts. For a few fragments, the R-substitutes engage in polar contacts with the 14-3-3 water shell, exemplified by <b>19e</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). However, the additional polar contacts do not translate to significant increases in ternary complex formation potentially due to entropic penalties or lack of contacts with the p65 peptide. Given the lack of cooperative ternary complex formation of this library at biochemically relevant concentration, we shifted focus to the 4-formyl benzenesulfonamide chemotype.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Acetamide-benzaldehydes in complex with 14-3-3/p65_45. (A) 3D (beige sticks) and 2D structure of 4-formyl-benzamide. The C–C–C–N torsion angles (orange lines) cluster at τ ≈ 0 and 180°.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> (B) Initial fragment hit <b>4</b> can be extended from its <i>para-</i>position (black arrows; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YOX">6YOX</a>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> (C) Ternary structure of <b>19e</b> (green sticks) in complex with p65_45 (red sticks) and 14-3-3σΔC (white cartoon and sticks) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BIY">7BIY</a>). Hydrogen bonds are indicated with green dashes; the 2Fo-Fc electron density map is shown as gray mesh, contoured at 1σ.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9">2.4.  Focused Library Development around 4-Formyl-benzenesulfonamides</h3><div class="NLM_p">Having investigated focused libraries 1–5, we shifted our attention to development of a focused library based on fragment <b>1</b>. Analysis of the crystal structure of fragment <b>1</b>, specifically the torsion angle of τ = 90 ± 30° between the benzene ring and the mesyl group, proved interesting (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A).<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Extension of the fragment from the methyl group provided a potential point of reaching over the p65 peptide, trapping its binding to 14-3-3 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). Alternatively, we postulated that fragment extension could also result in a change in the conformation of the fragment leading to the occupation of the FC pocket and increasing 14-3-3-based affinity. To expedite fragment development, the methyl group was replaced for <i>N-</i>substituted amines to facilitate rapid access to a library of 25 structural analogues of 4-formyl-benzenosulfonamides. This library of sulfonamides was synthesized by conversion of sodium 2-formylbenzene-1-sulfonate (<b>20</b>) to 4-formyl-benzenesulfonyl chloride (<b>21</b>) using thionyl chloride in DMF. Subsequent coupling of <b>21</b> with <i>N</i>-substituted amines (<b>22a–z</b>) afforded fragments <b>23a–z</b>.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Sulfonamide-benzaldehydes are active in FA compound titrations. (A) 3D and 2D chemical structure of 4-formyl-benzenesulfonamide with a C–C–S–N torsion angle τ as indicated.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> (B) The benzenesulfonamides are extended toward the FC pocket or toward the p65 peptide (red; 14-3-3: white; both with sticks and transparent surface) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YOW">6YOW</a>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> (C) FA compound titrations in the presence of 50 μM 14-3-3γ and 100 nM p65_45. The compounds are divided into weak (left), medium (middle), and high (right) activity. The legends are sorted by activity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In contrast to focused libraries 1–5, most of the tested 4-formyl-benzenesulfonamide fragments showed significant stabilization in FA assays, allowing a differentiated SAR analysis (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). The activity ranged from a weak affinity with a small, not quantifiable increase in anisotropy to a two-digit micromolar CC<sub>50</sub> potency in compound titrations. From the focused library 6, nine fragments elicited CC<sub>50</sub> values ranging from 57 to 430 μM. An additional assessment of stabilization using the FA protein titration assay showed for 19 fragments a significant shift of apparent <i>K</i><sub>D</sub> values in the presence of 1 mM of the fragments. Fragments <b>23e</b>, <b>f</b>, <b>k</b>, <b>t</b>, <b>w</b>, and <b>z</b> showed <i>K</i><sub>D</sub> values of <30 μM compared to the binary complex of full-length 14-3-3 and the p65_45 peptide that gave a <i>K</i><sub>D</sub> = 300 ± 100 μM. Analysis of this subset of fragments showed that these shifts in <i>K</i><sub>D</sub>’s (stabilization factor, SFs range from ∼12- to 65-fold). Fragment <b>23z</b>, possessing a <i>N</i>-substituted 1,2,3,4-tetrahydroquinoline, showed the greatest SF of ∼65-fold.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Exploration of 4-Formyl-benzenesulfonamides<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0011.gif" alt="" id="gr10" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0012.gif" alt="" id="gr11" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Fit did not converge.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">CC<sub>50</sub>: values of compound titrations with 50 μM 14-3-3γ. <i>K</i><sub>D,app</sub>: value of protein titrations in the presence of 1 mM of the fragment. SF: the fold-change of apparent <i>K</i><sub>D</sub> in comparison to a DMSO control. ND: not determined. All fragments bound to the p65_45/14-3-3σΔC complex. For additional data, see <a href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_001.pdf" class="ext-link">Figure S4 and Table S3</a>.</p></div></div><div></div></div><div class="NLM_p">Comparison of X-ray crystal soaks provided an explanation for the FA assay results. Structural data showed that 2-methyl pyrrolidine (<b>23e</b>) and piperidine (<b>23f</b>) were tolerated within the composite binding pocket. Ring expansion to the <i>N</i>-methyl diazepane (<b>23 g</b>) proved to be detrimental to stabilization (CC<sub>50</sub> = 640 μM). Notably, polar functionalities within six-membered <i>N</i>-heterocycles were not well tolerated; specifically, hydrogen bond accepting and donating groups (<b>23h</b>, <b>n–u</b>) elicited high CC<sub>50</sub> values and modest SFs (6–16-fold). This can be exemplified by fragment <b>23f</b> containing piperidine (app <i>K</i><sub>D</sub> = 15 μM, SF =22) compared with <i>N</i>-methylpiperazine <b>23h</b> (app <i>K</i><sub>D</sub> = 100 μM, SF = 3.3) and morpholine <b>23u</b> (app <i>K</i><sub>D</sub> = 52 μM, SF = 6.3) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B). Analysis of the X-ray structures showed that introduction of a polar functionality typically resulted in reorientation of the R-substitute of the fragment toward the FC-binding pocket. Introduction of a hydrophobic functionality such as <b>23k</b> (app <i>K</i><sub>D</sub> = 27 μM, SF = 12.2) and <b>23v</b> (app <i>K</i><sub>D</sub> = 430 μM, SF = 10.6)<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> recovered stabilizing activity, with the fragments re-establishing hydrophobic contacts with the p65 peptide. Notably, large amphiphilic modifications were also not well tolerated, such as <b>23m</b> (app <i>K</i><sub>D</sub> = 74 μM, SF = 4.5) and <b>23n</b> (app <i>K</i><sub>D</sub> = 94 μM, SF = 3.5) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), with both <b>23m</b> and <b>23n</b> occupying solvent exposed space within the composite pocket but not engaging in contacts with p65. Interestingly, fragment <b>23w</b> (app <i>K</i><sub>D</sub> = 26 μM, SF = 12.7) showed a significant drop in stabilization with respect to <b>23f</b>. Crystal soaking experiments showed that fragment <b>23w</b> engages in hydrophobic interactions with Ile46, Pro47, and Gly48 of p65 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D). However, unlike other fragments, <b>23w</b> induces a conformational change in the C-terminus of the peptide shifting toward the pS45. Analysis of soaking experiment with fragment <b>23y</b> showed that the tetrahydroisoquinoline ring engages in hydrophobic contacts with the p65 peptide; however, unlike <b>23e</b> and <b>23f</b>, the bicyclic rings repulse Pro47 of p65 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>E). This provides an explanation for the poor stabilizing activity of <b>23y</b>. In contrast to 2-tetrahydroquinoline (<b>23y)</b>, 1-tetrahydroquinoline (<b>23z</b>) was highly tolerated, eliciting CC<sub>50</sub> = 57 μM and app <i>K</i><sub>D</sub> = 5.1 μM; this translated to ∼65-fold stabilization. X-ray crystallography analysis of fragment <b>23z</b> provided valuable explanation for the high stabilization observed in FA assays. Specifically, the bicyclic substructure in <b>23z</b> engages the p65 peptide and is positioned upwards occupying the hydrophobic roof of 14-3-3σΔC shaped by residues Ile219, Leu218, and Leu222 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>F). Further, <b>23z</b> also engages in direct hydrophobic contacts with Ile46 and Pro47 of p65. Most notably, the 14-3-3/p65/<b>23z</b> ternary complex formation results in a reorientation of the p65 peptide leading to additional 14-3-3/p65 contacts. Fragment <b>23z</b> appears to function as a template for the additional binding of p65, increasing cooperative behavior of the ternary complex. Fragment <b>23z</b> facilitates additional 14-3-3/p65 contacts by enabling the C-terminus of p65 to wrap over <b>23z</b> and engage in electrostatic contacts with 14-3-3. Specifically, salt bridges are observed between Arg49/Glu14 and Arg50/Asp215 of p65_45/14-3-3, respectively. These structural features translate to high activity in FA assays, with a CC<sub>50</sub> of 57 μM and an SF of 65 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>G). The additional interactions between 14-3-3 and p65 induced by the binding of <b>23z</b> lead to increased cooperativity of the ternary complex. This improved cooperative behavior directly translates into improved stabilization of the 14-3-3/p65 complex.</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Sulfonamide-benzaldehydes inducing complex stabilization. (A) Overlay of <b>23h</b> (green sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NLE">7NLE</a>), <b>23k</b> (orange sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NK5">7NK5</a>), <b>23f</b> (cyan sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NJB">7NJB</a>), and <b>23u</b> (yellow sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJB">7BJB</a>) binding to p65_45 (red sticks) and 14-3-3 (white cartoon and sticks). Hydrogen bonds are shown as black dashes, and hydrophobic contacts are indicated with transparent spheres. A conserved water molecule (small sphere) is shown colored based on the related fragment color. (B) Fragments <b>23h</b>, <b>k</b>, <b>f</b>, and <b>u</b> show complex stabilization in FA compound titrations (left; [14-3-3γ]: 50 μM, [p65_45]: 100 nM) and protein titrations (right; [fragment]: 1 mM, [p65_45]: 100 nM). (C) Overlay of <b>23n</b> (brown sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NM3">7NM3</a>) and <b>24m</b> (light blue sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NM1">7NM1</a>) in complex with 14-3-3σΔC/p65_45. (D) Crystal structures of <b>23w</b> (blue sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NLA">7NLA</a>) binding to the composite binding pocket of p65 (red sticks) and 14-3-3 (white cartoon and sticks). The 2Fo-Fc electron density map (1σ) is displayed as gray mesh, and hydrogen bonds are shown as black dashes. (E) Overlay of <b>23y</b> (green sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NK3">7NK3</a>) and <b>23z</b> (yellow; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NJ9">7NJ9</a>) binding to the p65_45/14-3-3 complex. Upon binding of <b>23y</b>, Pro47 changes its conformation (red sticks, as reference: Pro47 of the <b>23z</b>/p65_45/14-3-3σΔC complex shown as yellow sticks). (F) Ternary structure of <b>23z</b> (yellow sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NJ9">7NJ9</a>) binding to the p65_45/14-3-3 complex. Details as in (D). (G) Ternary complex formation of <b>23x</b>, <b>y</b>, and <b>z</b> was measured with FA compound titrations (left; [14-3-3γ]: 50 μM, [p65_45]: 100 nM) and the stabilizing activity of <b>23w</b>, <b>y</b>, and <b>z</b> with protein titrations (right; [fragment]: 1 mM, [p65_45]: 100 nM).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11">2.5.  Optimization of Fragment <b>23z</b></h3><div class="NLM_p">Encouraged by the high stabilizing effect of covalent fragment <b>23z</b>, we looked to develop a library of 10 sulfonylamides based on tricyclic fragment <b>23z</b>. Due to the rearrangement of the p65 peptide in the presence of <b>23z</b> forming a narrow and enclosed binding pocket, this limited the sites for fragment modifications to the 5-, 6-, or 7-position of the tetrahydroquinoline ring (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A). As a result, we focused on small modifications to build additional interactions with the hydrophobic patch in the roof of 14-3-3 or extension of the fragments over Ile46 of p65. Additionally, we investigated the addition of heteroatoms in the 4-position of the tetrahydroquinoline ring to engage with the backbone carbonyl or nitrogen of Pro47 or Gly48 to establish additional polar contacts. A small library of 10 analogues was synthesized using our previously mentioned sulfonamide coupling (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Optimization of <b>23z</b>. (A) Ternary structure of <b>23z</b> (yellow spheres) in complex with 14-3-3σΔC (white surface) and p65_45 (red cartoon, transparent spheres) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NJ9">7NJ9</a>). Carbons of the bicyclic head group are numbered. (B) Ternary structure of <b>24b</b> (orange sticks) in complex with p65_45/14-3-3σΔC (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BIQ">7BIQ</a>). The distance between the 2-methyl and water (red spheres) is indicated with black dashes. Polar contacts are shown as black dashed lines. (C) Structure of <b>24e</b> (cyan sticks) binding to the p65_45/14-3-3σΔC complex (as in A; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BI3">7BI3</a>). Hydrophobic contacts are indicated with transparent spheres. (D) FA compound titrations with 50 μM 14-3-3γ and 100 nM p65_45. (E) FA protein titrations with 1 mM fragment and 100 nM p65_45. (F) Structure of <b>24j</b> binding to the p65_45/14-3-3σΔC complex (as in A; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BIW">7BIW</a>). A beneficial hydrogen bond is formed between the backbone carbonyl of p65s’ Pro47 and <b>24j</b> (black dash).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Exploration of structural analogs of <b>23z</b><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0013.gif" alt="" id="gr12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Tested as an enantiomeric mixture.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">CC<sub>50</sub>: values of compound titrations with 50 μM 14-3-3γ. <i>K</i><sub>D,app</sub>: values of protein titrations in the presence of 1 mM of the fragment. SF: the fold-change of apparent <i>K</i><sub>D</sub> in comparison to a DMSO control. ND: not determined. All fragments bound to the p65_45/14-3-3σΔC complex. For additional data, see <a href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_001.pdf" class="ext-link">Figure S5 and Table S4</a>.</p></div></div><div></div></div><div class="NLM_p">Structural analysis of this library showed that this series of analogues retained a similar cooperative behavior as <b>23z</b>, inducing additional PPI contacts. Biophysical analysis using an FA assay provided valuable insight into the SAR around fragment <b>23z</b>. Analysis of the CC<sub>50</sub> concentration indicated that <b>23z</b> showed the highest ternary complex formation, with all other analogues in this series eliciting a drop in complexation (CC<sub>50</sub> values ranging from 110 to 310 μM). However, significant insight into complex stabilization was gained from the analysis of app <i>K</i><sub>D</sub> values. Substitution of <b>23z</b> for an indoline (<b>24a</b>) or racemic 2-methyl-tetrahydroquinoline (<b>24b</b>) was tolerated; however, a 2-fold reduction was observed in stabilization, with app <i>K</i><sub>D</sub> values of 11 and 9.6 μM, respectively. Structural analysis of <b>24a</b> showed additional space around the five-membered heterocycle potentially providing additional sites for functionalization in the future (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_001.pdf" class="ext-link">Figure S5</a>). Fragment <b>24b</b> showed excellent electron density across the entire fragment within the X-ray crystal structure, enabling unambiguous assignment of stereochemistry. From the structure, it was identified that the R-enantiomer exclusively bound in the crystal, suggesting that the pure R-enantiomer may have significantly underestimated CC<sub>50</sub> and app <i>K</i><sub>D</sub> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B). This site may provide an excellent point for fragment extension toward the p65 peptide. Addition of a halogen or methoxy in the 6-position resulted in a reduction in stabilization, with 6-fluoro (<b>24c</b>), 6-chloro (<b>24d</b>), or 6-methoxy (<b>24e</b>) substituted tetrahydroquinoline fragments affording app <i>K</i><sub>D</sub> values of 6.2, 9.9, and 10 μM, respectively. Analysis of soaking experiments shows that modifications in this position resulted in unfavorable contacts with the roof of 14-3-3 (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C). Addition of an oxygen to the saturated six-membered ring (<b>24f</b>) proved detrimental to stabilization (app <i>K</i><sub>D</sub> = 15 μM). Further addition of halogens or methoxy groups in the 6 or 7 positions did not re-establish lost stabilization, such as 6-chloro (<b>24h</b>), 6-methoxy (<b>24i</b>), or 7-fluoro (<b>24g</b>). Substitution of the benzomorpholine (<b>24f</b>) for tetrahydroquinoxaline (<b>24j</b>) resulted in an improvement in stabilization with <b>24j</b> eliciting an app <i>K</i><sub>D</sub> of 4.3 μM, translating into the highest stabilizing effect with an 81-fold stabilization (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D,E). Structural analysis showed that the introduction of a nitrogen atom in the saturated ring installs an additional hydrogen bond with the backbone carbonyl of Pro47 of p65 (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>F).</div><div class="NLM_p last">To assess the selective ternary complex formation of <b>24j</b> with 14-3-3/p65, the fragment was further screened in a compound titration FA assay against three well-established 14-3-3 interactions (p53, ERα, and TAZ; <a href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_001.pdf" class="ext-link">Figure S6</a>). Gratifyingly, no ternary complex formation was detected for p53, ERα, or TAZ complexes, proving that <b>24j</b> is not a pan-stabilizer of 14-3-3 interactions and indicating an initial selectivity of <b>24j</b> toward the p65/14-3-3 interaction.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">3.  Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44072" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44072" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Summarizing, here we build upon our initial report of a novel covalent fragment screening approach using an aldehyde fragment library. These covalent fragments form a covalent tether with Lys122 within the 14-3-3/p65 composite binding pocket. Specifically, we describe the optimization of initial hit covalent fragments into a p65/14-3-3 molecular glue that elicited an 81-fold stabilization of the 14-3-3/p65 complex. Critical to this success was the use of X-ray crystallography and FA measurements to develop a robust understanding of the structural activity relationship. Direct hydrophobic engagement with p65 was the driving interaction that established complex stabilization. Hydrophobic contacts between <b>23z</b> or <b>24j</b> and p65 resulted in a conformational change in the 14-3-3/p65 interface with extended interactions, in turn increasing the cooperativity of the ternary complex. Fragment <b>23z</b> or <b>24j</b> might serve as a valuable starting point for optimization for the development of a chemical probe to study the effects of 14-3-3-mediated regulation of p65 in a cellular context. Particularly, the aldehyde functionality requires additional investigations to validate its activity and oxidative stability in a cellular context such as with the aldehyde-containing drug voxelotor.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Alternatively, the aldehyde could be replaced with a more benign chemical handle that is less susceptible to oxidation, or the application of a prodrug approach to protect the aldehyde may need to be considered when moving toward cellular studies.</div><div class="NLM_p last">Considering the lack of rational methodologies to develop PPI stabilizers, here we describe a systematic approach to developing initial hit fragments into molecular glues. Observations from this research indicate that the direct engagement of molecular glues with the partner peptide is significant for stabilization. Specifically, hydrophobic contacts are effective to enhance ternary complex formation and support a cooperative binding mode. Analysis of SAR results indicates that fragments that facilitate templating of the p65 peptide and promotion of additional contacts between 14-3-3 and p65 resulted in increased complex stabilization. This research has direct applications to further development of PPI stabilizers as well as the field of PROTACs. There is a growing body of evidence showing that cooperative PROTACs, which stabilize direct PPIs, lead to improved efficacy, reducing problems associated with the hook-effect and the necessity for highly potent molecules.<a onclick="showRef(event, 'ref42 ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref42 ref43 ref44 ref45">(42−45)</a> Additionally, research from the field of PROTACs and our findings appear to be in agreement that the molecular glue/cooperative PROTAC potency of a ligand for a complex is not the driving force for increased complex stabilization; rather, the cooperativity behavior of the ternary complex appears to be the driving force for PPI stabilization.<a onclick="showRef(event, 'ref11 ref42 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref11 ref42 ref45 ref46">(11,42,45,46)</a></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">4.  Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76755" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76755" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15">4.1.  Protein Expression and Purification</h3><div class="NLM_p last">The 14-3-3 proteins were expressed and purified using standard protocols. In short, pPROEX HTb vectors encoding the 14-3-3σΔC (truncated C-terminus ΔC17) and 14-3-3γ isoform were transformed into BL21(DE3) cells. Protein expression was initiated with 0.4 mM IPTG at a cell density OD<sub>600</sub> = 0.8–1. The expression took place overnight at 18 °C. The cells were harvested by centrifugation (10,000<i>g</i>, 15 min) and resuspended in a lysis buffer (50 mM Tris/HCl pH 8, 300 mM NaCl, 12.5 mM imidazole, and 2 mM β-mercaptoethanol). The cells were lysed with a homogenizer, and the lysate was cleared via centrifugation (40,000<i>g</i>, 30 min). Ni-NTA columns were used to isolate the protein, which was washed with 10 CV lysis buffer and eluted with 250 mM imidazole (50 mM Tris/HCl pH 8, 300 mM NaCl, 250 mM imidazole, and 2 mM β-mercaptoethanol). The full-length 14-3-3γ was dialyzed against 25 mM HEPES pH 7.5, 100 mM NaCl, 10 mM MgCl<sub>2</sub>, and 0.5 mM Tris(2-carboxyethyl)phosphine) and stored at −80 °C. For the 14-3-3σΔC, the His6-tag was removed by the TEV protease; the TEV was removed with Ni-NTA columns. The rest of the imidazole of the 14-3-3σΔC solution was removed by size exclusion chromatography (20 mM HEPES pH 7.5, 150 mM NaCl, and 2 mM β-mercaptoethanol) and stored at −80 °C.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16">4.2.  X-ray Crystallography</h3><div class="NLM_p">Binary crystals with p65_45 peptide (sequence: EGRSAG pS<sub>45</sub> IPGRRS, C-terminus: amidation; N-terminus: acetylation)<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> and 14-3-3σΔC were grown at a 14-3-3σΔC concentration of 12 mg/mL in a 1:2 ratio with the acetylated peptide in 20 mM HEPES pH 7.5, 2 mM MgCl<sub>2</sub>, and 2 mM β-mercaptoethanol. This complexation mixture was incubated overnight. In a hanging drop setup, the complexation mixture was mixed in 1:2 ratio with the precipitation buffer (95 mM HEPES pH 7.5, 27–28% PEG400, 190 mM CaCl<sub>2</sub>, and 5% glycerol). For data acquisition, crystals were directly flash-frozen in liquid nitrogen.</div><div class="NLM_p last">Fragment soaks were performed by adding compounds in DMSO stock solutions direct to fully grown crystals with a final compound concentration of 10 mM (≤1% DMSO). The soaks were incubated for 7 days prior to data acquisition. Diffraction data were measured either at the P11 beamline of PetraIII (DESY campus, Hamburg, Germany) or the i-03/i-24 beamline of the diamond light source (Oxford, UK) or in-house. The diffraction data were integrated with the xia2/DIALS pipeline<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> followed by molecular replacement with MolRep.<a onclick="showRef(event, 'ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref49 ref50">(49,50)</a> The binary p65_45/14-3-3σΔC structure was used as a search model (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QHL">6QHL</a>). Model refinement took place in iterative cycles with Coot,<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Refmac5,<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> and phenix.refine.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> 3D structures of ligands were prepared using the fragment SMILES and elbow of the phenix suite.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Figures were generated with PyMOL (V2.0.6, Schrodinger LLC).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17">4.3.  Fluorescence Anisotropy Assays</h3><div class="NLM_p">Complex stabilization was measured using a fluorescently labeled p65_45 peptide (FITC-βAla-EGRSAG pS<sub>45</sub> IPGRRS) at a concentration of 100 nM throughout all assays. During compound titrations, the 14-3-3γ concentration was constant at 50 μM and the compound was titrated in a 1:1 dilution series. In protein titrations, 14-3-3γ was titrated in a 1:1 dilution series in the presence of 1 mM compound. The plates (Corning 384 well plates, black, round bottom, low binding) were incubated for 3 h at RT prior to fluorescence anisotropy (FA) measurements with the Tecan Infinite 500 plate reader (FITC dye: excitation 485 nm and emission 535 nm). Dilution series were prepared in an FA buffer (10 mM HEPES pH 7.4, 150 mM NaCl, and 0.1% Tween20).</div><div class="NLM_p">The activity of the hit compound <b>24j</b> was measured for the TAZ-peptide<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> (FITC-βAla-RSH pS89 SPASLQ), p53-peptide<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> (TAMRA-Ahx-SRAHSSHLKSKKGQSTSRHKKLMFK pT387 EGPDSD-COOH), or ERα-peptide<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> (FITC-O1Pen-AEGFPA pT594 V-COOH) and 14-3-3γ. Therefore, the compound was titrated in a 1:1 dilution series to a constant concentration of 14-3-3γ and 10 nM peptide. The protein concentration was adjusted to one-third of the <i>K</i><sub>D</sub> of the binary protein/peptide complex (final 14-3-3γ concentration for the measurement with TAZ: 0.1 μM; with p53: 0.3 μM and with ERα: 0.1 μM). For a better comparison, the baseline of each measurement was subtracted (Δ<i>r</i>/mAU).</div><div class="NLM_p last">All measurements were performed as single measurements. Data analysis was performed with Origin 2019 (V9.6.0.172, OriginLab Corporation) with the inbuilt ″Hill1″ function for data fitting. For data sets without an upper plateau, the Hill coefficient <i>n</i> was set to <i>n</i> = 1.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18">4.4.  Chemistry: General Information</h3><div class="NLM_p">All commercial chemicals were used as received. Reagents were used without further purification unless otherwise noted. TLC analysis was performed on TLC aluminum sheets, silica gel layer, ALUGRAM SIL G UV254, 20 × 20 cm by MACHEREY-NAGEL. TLC plates were analyzed by UV fluorescence (254 nm). UHPLC–MS analysis was performed using the UPLC Agilent Technologies 1290 Infinity coupled with the Agilent Technologies 6120 Quadrupole LC/MS DAD detector. Column: ACQUITY UHPLC BEH C18 (1.7 μm) 2.1 mm × 50 mm. Temperature: 40 °C. Detection: DAD + MS/6120 Quadrupole. Injected volume: 1 μL. Flow: 1.2 mL/min. Solvent A: water + 0.1% formic acid. Solvent B: acetonitrile + 0.1% formic acid. Gradient: 0 min 2% B, 0.2 min 2% B, 2.0 min 98% B, 2.2 min 98% B, 2.21 min 2% B, and 2.5 min 2% B. Preparative HPLC was performed using the UPLC Agilent Technologies 1260 Infinity coupled with the Agilent Technologies 6120 Quadrupole LC/MS. Column: Waters XBridge Prep C18 5 μm OBD 19 × 150 mm. Detection: DAD + MS/6120 Quadrupole. Flow: 32 mL/min. Solvent A: water + 0.1% formic acid. Solvent B: acetonitrile + 0.1% formic acid. Gradient: 0 min 77% A/23% B, 1 min 77% A/23% B, 9 min 16% A/84% B, 9.01 min 2% A/98% B, and 11 min 2% A/98% B. The purity of the synthesized compounds is ≥95%. <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were recorded on a Bruker 300 MHz spectrometer at ambient temperature. The chemical shifts are listed in ppm on the δ = scale, and coupling constants were recorded in hertz (Hz). Chemical shifts are calibrated relative to the signals corresponding of the nondeuterated solvent (CHCl<sub>3</sub>: δ = 7.26 ppm for 1H and 77.16 ppm for 13C; DMSO: δ = 2.50 ppm for 1H and 39.52 ppm for 13C). Abbreviations are used in the description of NMR data as follows: chemical shift (δ = ppm), multiplicity (s = singlet, d = doublet, t = triplet, m = multiplet, bs = broad singlet, dd = doublet of doublets, td = triplet of doublets), and coupling constant (<i>J</i> = Hz).</div><div class="NLM_p last">For general procedures, chemical characterization, NMR spectra, and LC–MS traces, see the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00401" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55421" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55421" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00401?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00401</a>. <ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional results and X-ray crystallography tables, chemical experimental, and chemical spectral data (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_001.pdf">jm1c00401_si_001.pdf (3.98 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_002.csv">jm1c00401_si_002.csv (7.09 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors have been deposited in the Protein Data Bank under the following accession codes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NMH">7NMH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BI3">7BI3</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BIQ">7BIQ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BIW">7BIW</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BIY">7BIY</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJB">7BJB</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJF">7BJF</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJL">7BJL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJW">7BJW</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BKH">7BKH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NJ9">7NJ9</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NJB">7NJB</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NK3">7NK3</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NK5">7NK5</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NLA">7NLA</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NLE">7NLE</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NM1">7NM1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NM3">7NM3</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NM9">7NM9</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NR7">7NR7</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NV4">7NV4</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NVI">7NVI</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NWS">7NWS</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NXS">7NXS</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NXT">7NXT</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NXW">7NXW</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NXY">7NXY</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NY4">7NY4</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NYE">7NYE</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NYF">7NYF</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NYG">7NYG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NZ6">7NZ6</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NZG">7NZG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NZK">7NZK</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NZV">7NZV</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O34">7O34</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3A">7O3A</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3F">7O3F</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3P">7O3P</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3Q">7O3Q</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3R">7O3R</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3S">7O3S</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O57">7O57</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O59">7O59</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5A">7O5A</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5C">7O5C</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5D">7O5D</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5F">7O5F</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5G">7O5G</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5O">7O5O</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5P">7O5P</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5S">7O5S</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5U">7O5U</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5X">7O5X</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6F">7O6F</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6G">7O6G</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6I">7O6I</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6J">7O6J</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6K">7O6K</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6M">7O6M</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6O">7O6O</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.1c00401" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24608" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24608" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Luc Brunsveld</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Engineering, Laboratory of Chemical Biology and Institute
for Complex Molecular Systems, Eindhoven
University of Technology, P.O. Box 513, 5600 MB Eindhoven, The Netherlands</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5675-511X" title="Orcid link">https://orcid.org/0000-0001-5675-511X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#cfa3e1adbdbaa1bcb9aaa3ab8fbbbaaae1a1a3"><span class="__cf_email__" data-cfemail="e38fcd8191968d9095868f87a3979686cd8d8f">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dimitrios Tzalis</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Taros Chemicals GmbH & Co.
KG, Emil-Figge-Straße 76a, 44227 Dortmund, Germany</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#4420303e25282d37043025362b376a2021"><span class="__cf_email__" data-cfemail="a4c0d0dec5c8cdd7e4d0c5d6cbd78ac0c1">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christian Ottmann</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Engineering, Laboratory of Chemical Biology and Institute
for Complex Molecular Systems, Eindhoven
University of Technology, P.O. Box 513, 5600 MB Eindhoven, The Netherlands</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-7315-0315" title="Orcid link">https://orcid.org/0000-0001-7315-0315</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#50337e3f24243d313e3e102425357e3e3c"><span class="__cf_email__" data-cfemail="abc885c4dfdfc6cac5c5ebdfdece85c5c7">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Madita Wolter</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Engineering, Laboratory of Chemical Biology and Institute
for Complex Molecular Systems, Eindhoven
University of Technology, P.O. Box 513, 5600 MB Eindhoven, The Netherlands</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-1430-9589" title="Orcid link">https://orcid.org/0000-0003-1430-9589</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dario Valenti</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Engineering, Laboratory of Chemical Biology and Institute
for Complex Molecular Systems, Eindhoven
University of Technology, P.O. Box 513, 5600 MB Eindhoven, The Netherlands</span>; 
    <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Taros Chemicals GmbH & Co.
KG, Emil-Figge-Straße 76a, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter J. Cossar</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Engineering, Laboratory of Chemical Biology and Institute
for Complex Molecular Systems, Eindhoven
University of Technology, P.O. Box 513, 5600 MB Eindhoven, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stanimira Hristeva</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Taros Chemicals GmbH & Co.
KG, Emil-Figge-Straße 76a, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura M. Levy</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Taros Chemicals GmbH & Co.
KG, Emil-Figge-Straße 76a, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thorsten Genski</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Taros Chemicals GmbH & Co.
KG, Emil-Figge-Straße 76a, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Torsten Hoffmann</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Taros Chemicals GmbH & Co.
KG, Emil-Figge-Straße 76a, 44227 Dortmund, Germany</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.W., D.V., and P.J.C. contributed equally. The manuscript was written through contributions of all authors. All of the authors approved the final version of the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>The research was supported by funding from the European Union through the TASPPI project (H2020-MSCA-ITN- 2015, grant number 675179), through the Eurotech Postdoctoral Fellow program (Marie Skłodowska-Curie Co-funded, grant number 754462), and through The Netherlands Organization for Scientific Research (NWO) via VICI grant 016.150.366 and via Gravity Program 024.001.035.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">L.B. and C.O. are scientific co-founders of Ambagon Therapeutics.<br /></br>The authors declare the following competing financial interest(s): LB and CO are co-founders of Ambagon Therapeutics.<br /></br></div></li></ul></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">app. <i>K</i><sub>D</sub></td><td class="NLM_def"><p class="first last">apparent <i>K</i><sub>D</sub></p></td></tr><tr><td class="NLM_term">CC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-maximal complex concentration</p></td></tr><tr><td class="NLM_term">FA</td><td class="NLM_def"><p class="first last">fluorescence anisotropy</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33400" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33400" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 54 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Inhibitors of Protein-Protein Interactions: Progressing towards the Dream</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">301</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/nrd1343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnrd1343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=15060526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BD2cXis1GktrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=301-317&author=M.+M.+R.+Arkinauthor=J.+A.+Wells&title=Small-Molecule+Inhibitors+of+Protein-Protein+Interactions%3A+Progressing+towards+the+Dream&doi=10.1038%2Fnrd1343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule inhibitors of protein-protein interactions: progressing towards the dream</span></div><div class="casAuthors">Arkin, Michelle R.; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">301-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein-protein interactions have a key role in most biol. processes, and offer attractive opportunities for therapeutic intervention.  Developing small mols. that modulate protein-protein interactions is difficult, owing to issues such as the lack of well-defined binding pockets.  Nevertheless, there was important progress in this endeavor in recent years.  Here, the authors use illustrative examples to discuss general strategies for addressing the challenges inherent in the discovery and characterization of small-mol. inhibitors of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7MiFDdLJu27Vg90H21EOLACvtfcHk0lipz2GzYZjHFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXis1GktrY%253D&md5=3e102c6207be420b4232056b4acdea12</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd1343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1343%26sid%3Dliteratum%253Aachs%26aulast%3DArkin%26aufirst%3DM.%2BM.%2BR.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSmall-Molecule%2520Inhibitors%2520of%2520Protein-Protein%2520Interactions%253A%2520Progressing%2520towards%2520the%2520Dream%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2004%26volume%3D3%26spage%3D301%26epage%3D317%26doi%3D10.1038%2Fnrd1343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span> <span> </span><span class="NLM_article-title">Recent Advances in the Development of Protein–Protein Interactions Modulators: Mechanisms and Clinical Trials</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">213</span>, <span class="refDoi"> DOI: 10.1038/s41392-020-00315-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fs41392-020-00315-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=32968059" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A280%3ADC%252BB3s%252Fis1GisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2020&pages=213&author=H.+Luauthor=Q.+Zhouauthor=J.+Heauthor=Z.+Jiangauthor=C.+Pengauthor=R.+Tongauthor=J.+Shi&title=Recent+Advances+in+the+Development+of+Protein%E2%80%93Protein+Interactions+Modulators%3A+Mechanisms+and+Clinical+Trials&doi=10.1038%2Fs41392-020-00315-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in the development of protein-protein interactions modulators: mechanisms and clinical trials</span></div><div class="casAuthors">Lu Haiying; Tong Rongsheng; Shi Jianyou; Zhou Qiaodan; He Jun; Jiang Zhongliang; Peng Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Signal transduction and targeted therapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">213</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein-protein interactions (PPIs) have pivotal roles in life processes.  The studies showed that aberrant PPIs are associated with various diseases, including cancer, infectious diseases, and neurodegenerative diseases.  Therefore, targeting PPIs is a direction in treating diseases and an essential strategy for the development of new drugs.  In the past few decades, the modulation of PPIs has been recognized as one of the most challenging drug discovery tasks.  In recent years, some PPIs modulators have entered clinical studies, some of which been approved for marketing, indicating that the modulators targeting PPIs have broad prospects.  Here, we summarize the recent advances in PPIs modulators, including small molecules, peptides, and antibodies, hoping to provide some guidance to the design of novel drugs targeting PPIs in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTce0zvk4iKluvW160gIxP_fW6udTcc2eaPzgBx27u6fbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3s%252Fis1GisQ%253D%253D&md5=ec2910a147480ed3d7119f4bfd36523a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fs41392-020-00315-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-020-00315-3%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DZ.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DTong%26aufirst%3DR.%26aulast%3DShi%26aufirst%3DJ.%26atitle%3DRecent%2520Advances%2520in%2520the%2520Development%2520of%2520Protein%25E2%2580%2593Protein%2520Interactions%2520Modulators%253A%2520Mechanisms%2520and%2520Clinical%2520Trials%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2020%26volume%3D5%26spage%3D213%26doi%3D10.1038%2Fs41392-020-00315-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milroy, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennig, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Modulators of Protein-Protein Interactions</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">4695</span>– <span class="NLM_lpage">4748</span>, <span class="refDoi"> DOI: 10.1021/cr400698c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr400698c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmt1Sgsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2014&pages=4695-4748&author=L.+G.+Milroyauthor=T.+N.+Grossmannauthor=S.+Hennigauthor=L.+Brunsveldauthor=C.+Ottmann&title=Modulators+of+Protein-Protein+Interactions&doi=10.1021%2Fcr400698c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Modulators of Protein-Protein Interactions</span></div><div class="casAuthors">Milroy, Lech-Gustav; Grossmann, Tom N.; Hennig, Sven; Brunsveld, Luc; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4695-4748</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Since Hedin's characterization of trypsin and antitrypsin in 1906, arguably the first account of a regulatory protein-protein interaction (PPI), contemporary understanding of proteins and PPIs has been progressively transformed by landmark conceptual and technol. advances in mol. cell biol., biochem. and biophysics, not least, the sequencing of the human genome and the ensuing genomic technologies.  Today, proteins can be viewed as the mol. smart phones of the cell, genetically programmed to enact specific cellular functions in response to external stimuli.  Individually, proteins perform essential functions such as catalysis and the transport of mols. and ions.  However,their effectiveness in the crowded cellular environment is only short-range and insufficient to sustain life without the involvement of other biomols. such as other proteins or metabolites.  This review begins with a up-to-date account of the principle biochem. techniques used to identify PPI modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrty0-k5R_QrVg90H21EOLACvtfcHk0liPa-hGtVOQbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmt1Sgsrs%253D&md5=89eddaa7ebaa7b9e4dfe705e86b55fe7</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fcr400698c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr400698c%26sid%3Dliteratum%253Aachs%26aulast%3DMilroy%26aufirst%3DL.%2BG.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DHennig%26aufirst%3DS.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DModulators%2520of%2520Protein-Protein%2520Interactions%26jtitle%3DChem.%2520Rev.%26date%3D2014%26volume%3D114%26spage%3D4695%26epage%3D4748%26doi%3D10.1021%2Fcr400698c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stevers, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sijbesma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKintosh, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obsil, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landrieu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cau, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karawajczyk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eickhoff, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doveston, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Modulators of 14-3-3 Protein–Protein Interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_fpage">3755</span>– <span class="NLM_lpage">3778</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00574</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00574" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=3755-3778&author=L.+M.+Steversauthor=E.+Sijbesmaauthor=M.+Bottaauthor=C.+MacKintoshauthor=T.+Obsilauthor=I.+Landrieuauthor=Y.+Cauauthor=A.+J.+Wilsonauthor=A.+Karawajczykauthor=J.+Eickhoffauthor=J.+Davisauthor=M.+Hannauthor=G.+O%E2%80%99Mahonyauthor=R.+G.+Dovestonauthor=L.+Brunsveldauthor=C.+Ottmann&title=Modulators+of+14-3-3+Protein%E2%80%93Protein+Interactions&doi=10.1021%2Facs.jmedchem.7b00574"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00574%26sid%3Dliteratum%253Aachs%26aulast%3DStevers%26aufirst%3DL.%2BM.%26aulast%3DSijbesma%26aufirst%3DE.%26aulast%3DBotta%26aufirst%3DM.%26aulast%3DMacKintosh%26aufirst%3DC.%26aulast%3DObsil%26aufirst%3DT.%26aulast%3DLandrieu%26aufirst%3DI.%26aulast%3DCau%26aufirst%3DY.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26aulast%3DKarawajczyk%26aufirst%3DA.%26aulast%3DEickhoff%26aufirst%3DJ.%26aulast%3DDavis%26aufirst%3DJ.%26aulast%3DHann%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DG.%26aulast%3DDoveston%26aufirst%3DR.%2BG.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DModulators%2520of%252014-3-3%2520Protein%25E2%2580%2593Protein%2520Interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26spage%3D3755%26epage%3D3778%26doi%3D10.1021%2Facs.jmedchem.7b00574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skidmore, J.</span></span> <span> </span><span class="NLM_article-title">Small Molecules, Big Targets: Drug Discovery Faces the Protein–Protein Interaction Challenge</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">533</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.29</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnrd.2016.29" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=27050677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=533-550&author=D.+E.+Scottauthor=A.+R.+Baylyauthor=C.+Abellauthor=J.+Skidmore&title=Small+Molecules%2C+Big+Targets%3A+Drug+Discovery+Faces+the+Protein%E2%80%93Protein+Interaction+Challenge&doi=10.1038%2Fnrd.2016.29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules, big targets: drug discovery faces the protein-protein interaction challenge</span></div><div class="casAuthors">Scott, Duncan E.; Bayly, Andrew R.; Abell, Chris; Skidmore, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">533-550</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) are of pivotal importance in the regulation of biol. systems and are consequently implicated in the development of disease states.  Recent work has begun to show that, with the right tools, certain classes of PPI can yield to the efforts of medicinal chemists to develop inhibitors, and the first PPI inhibitors have reached clin. development.  In this Review, we describe the research leading to these breakthroughs and highlight the existence of groups of structurally related PPIs within the PPI target class.  For each of these groups, we use examples of successful discovery efforts to illustrate the research strategies that have proved most useful.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK-aGzHa9mP7Vg90H21EOLACvtfcHk0ljwbu4wr_BmNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D&md5=9f0953567ae9338b8b36aa78b8160655</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.29&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.29%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DBayly%26aufirst%3DA.%2BR.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DSkidmore%26aufirst%3DJ.%26atitle%3DSmall%2520Molecules%252C%2520Big%2520Targets%253A%2520Drug%2520Discovery%2520Faces%2520the%2520Protein%25E2%2580%2593Protein%2520Interaction%2520Challenge%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2016%26volume%3D15%26spage%3D533%26epage%3D550%26doi%3D10.1038%2Fnrd.2016.29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blenis, J.</span></span> <span> </span><span class="NLM_article-title">Rapamycin: One Drug</span>. <i>Many Effects. Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">373</span>– <span class="NLM_lpage">379</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2014.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1016%2Fj.cmet.2014.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=24508508" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=373-379&author=J.+Liauthor=S.+G.+Kimauthor=J.+Blenis&title=Rapamycin%3A+One+Drug&doi=10.1016%2Fj.cmet.2014.01.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2014.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2014.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%2BG.%26aulast%3DBlenis%26aufirst%3DJ.%26atitle%3DRapamycin%253A%2520One%2520Drug%26jtitle%3DMany%2520Effects.%2520Cell%2520Metab.%26date%3D2014%26volume%3D19%26spage%3D373%26epage%3D379%26doi%3D10.1016%2Fj.cmet.2014.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kunz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. N.</span></span> <span> </span><span class="NLM_article-title">Cyclosporin A, FK506 and Rapamycin: More than Just Immunosuppression</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">334</span>– <span class="NLM_lpage">338</span>, <span class="refDoi"> DOI: 10.1016/0968-0004(93)90069-Y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1016%2F0968-0004%2893%2990069-Y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=7694398" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADyaK3sXlvFGjsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1993&pages=334-338&author=J.+Kunzauthor=M.+N.+Hall&title=Cyclosporin+A%2C+FK506+and+Rapamycin%3A+More+than+Just+Immunosuppression&doi=10.1016%2F0968-0004%2893%2990069-Y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclosporin A, FK506 and rapamycin: more than just immunosuppression</span></div><div class="casAuthors">Kunz, Jeannette; Hall, Michael N.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">334-8</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    </div><div class="casAbstract">A review with 35 refs.  The mechanisms of action of the immunosuppressive drugs cyclosphorin A(CsA), FK506 and rapamycin are strikingly conserved from yeast to human T cells.  Recent results obtained with yeast corroborate calcineurin as the target of CsA-cyclophilin and FK506-FKBP complexes, and reveal a phosphatidylinositol 3-kinase homolog as the target of the rapamycin-FKBP complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro4ILcAHS6mrVg90H21EOLACvtfcHk0lj6mAk8fztOxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXlvFGjsrg%253D&md5=3a97087838b164d58bb654a3defada7f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2F0968-0004%2893%2990069-Y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0968-0004%252893%252990069-Y%26sid%3Dliteratum%253Aachs%26aulast%3DKunz%26aufirst%3DJ.%26aulast%3DHall%26aufirst%3DM.%2BN.%26atitle%3DCyclosporin%2520A%252C%2520FK506%2520and%2520Rapamycin%253A%2520More%2520than%2520Just%2520Immunosuppression%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1993%26volume%3D18%26spage%3D334%26epage%3D338%26doi%3D10.1016%2F0968-0004%2893%2990069-Y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wani, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coggon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McPhail, A. T.</span></span> <span> </span><span class="NLM_article-title">Plant Antitumor Agents. VI. Isolation and Structure of Taxol, a Novel Antileukemic and Antitumor Agent from Taxus Brevifolia</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1971</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">2325</span>– <span class="NLM_lpage">2327</span>, <span class="refDoi"> DOI: 10.1021/ja00738a045</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00738a045" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADyaE3MXktVKisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1971&pages=2325-2327&author=M.+C.+Waniauthor=H.+L.+Taylorauthor=M.+E.+Wallauthor=P.+Coggonauthor=A.+T.+McPhail&title=Plant+Antitumor+Agents.+VI.+Isolation+and+Structure+of+Taxol%2C+a+Novel+Antileukemic+and+Antitumor+Agent+from+Taxus+Brevifolia&doi=10.1021%2Fja00738a045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Plant antitumor agents.  VI.  Isolation and structure of taxol, a novel antileukemic and antitumor agent from Taxus brevifolia</span></div><div class="casAuthors">Wani, Mansukhlal C.; Taylor, Harold Lawrence; Wall, Monroe E.; Coggon, Philip; McPhail, Andrew T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1971</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2325-7</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Taxol (I), a complex ester with a taxane ring system exhibiting potent antileukemic and antitumor properties, was isolated from Taxus brevifolia (yew).  BzNHCH(Ph)CH(OH)CO2Me and 5β,16-epoxy-1β,-2α,4α,7β,10β,13α-hexahydroxytax-11-en-9-one 4α-acetate 2α-benzoate are obtained by methanolysis of I.  Based on chem. evidence, I is 5β,16-epoxy-1β,2α,4α,7β,10β,13α-hexahydroxytax-11-en-9-one 4α,10β-diacetate 2α-benzoate 13α-(3-benzamido-2-hydroxy-3-phenylpropionate).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpexINv4qMTjbVg90H21EOLACvtfcHk0lj6mAk8fztOxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3MXktVKisLc%253D&md5=50c59ef3bf2ae493408bed2a76ca0c2e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fja00738a045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00738a045%26sid%3Dliteratum%253Aachs%26aulast%3DWani%26aufirst%3DM.%2BC.%26aulast%3DTaylor%26aufirst%3DH.%2BL.%26aulast%3DWall%26aufirst%3DM.%2BE.%26aulast%3DCoggon%26aufirst%3DP.%26aulast%3DMcPhail%26aufirst%3DA.%2BT.%26atitle%3DPlant%2520Antitumor%2520Agents.%2520VI.%2520Isolation%2520and%2520Structure%2520of%2520Taxol%252C%2520a%2520Novel%2520Antileukemic%2520and%2520Antitumor%2520Agent%2520from%2520Taxus%2520Brevifolia%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1971%26volume%3D93%26spage%3D2325%26epage%3D2327%26doi%3D10.1021%2Fja00738a045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joerger, M.</span></span> <span> </span><span class="NLM_article-title">Metabolism of the Taxanes Including Nab-Paclitaxel</span>. <i>Expert Opin. Drug Metab. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">691</span>– <span class="NLM_lpage">702</span>, <span class="refDoi"> DOI: 10.1517/17425255.2015.983074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1517%2F17425255.2015.983074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=25394848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotVeks74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=691-702&author=M.+Joerger&title=Metabolism+of+the+Taxanes+Including+Nab-Paclitaxel&doi=10.1517%2F17425255.2015.983074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism of the taxanes including nab-paclitaxel</span></div><div class="casAuthors">Joerger, Markus</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Metabolism & Toxicology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">691-702</span>CODEN:
                <span class="NLM_cas:coden">EODMAP</span>;
        ISSN:<span class="NLM_cas:issn">1742-5255</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The classical taxanes (paclitaxel, docetaxel), the newer taxane cabazitaxel and the nanoparticle-bound nab-paclitaxel are among the most widely used anticancer drugs.  Despite years of research, the optimal dosing regimen (weekly vs 3-weekly) and optimal dose is still controversial, as is the value of pharmacol. personalization of taxane dosing.  Areas covered: This review provides an overview of the pharmacol. properties of the taxanes, including metab., pharmacokinetics-pharmacodynamics and aspects in the clin. use of taxanes.  The latter includes the ongoing debate on the most active and safe regimen (paclitaxel, docetaxel, nab-paclitaxel), the recommended initial dose (cabazitaxel) and pharmacol. dosing individualization.  Expert opinion: Taxanes share the characteristics of extensive hepatic metab. and biliary excretion, the need for dose adaptation in patients with liver dysfunction, and substantial pharmacokinetic variability even after taking into account known covariates.  Data from clin. studies suggest that optimal scheduling of the taxanes is dependent not only on the specific taxane compd., but also on the tumor type and line of treatment.  Finally, treating oncologists should be aware of the substantial risk for drug-drug interactions that is a direct consequence of the complex hepatic metab. of the taxanes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrftQ6JPyajY7Vg90H21EOLACvtfcHk0lj6mAk8fztOxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotVeks74%253D&md5=7ab3c51d343b63c6917708c729912835</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1517%2F17425255.2015.983074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425255.2015.983074%26sid%3Dliteratum%253Aachs%26aulast%3DJoerger%26aufirst%3DM.%26atitle%3DMetabolism%2520of%2520the%2520Taxanes%2520Including%2520Nab-Paclitaxel%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2015%26volume%3D11%26spage%3D691%26epage%3D702%26doi%3D10.1517%2F17425255.2015.983074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petzold, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomä, N. H.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of Lenalidomide-Induced CK1α Degradation by the CRL4CRBN Ubiquitin Ligase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>532</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">130</span>, <span class="refDoi"> DOI: 10.1038/nature16979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnature16979" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=26909574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtFOis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=532&publication_year=2016&pages=127-130&author=G.+Petzoldauthor=E.+S.+Fischerauthor=N.+H.+Thom%C3%A4&title=Structural+Basis+of+Lenalidomide-Induced+CK1%CE%B1+Degradation+by+the+CRL4CRBN+Ubiquitin+Ligase&doi=10.1038%2Fnature16979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of lenalidomide-induced CK1α degradation by the CRL4CRBN ubiquitin ligase</span></div><div class="casAuthors">Petzold, Georg; Fischer, Eric S.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">532</span>
        (<span class="NLM_cas:issue">7597</span>),
    <span class="NLM_cas:pages">127-130</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thalidomide and its derivs., lenalidomide and pomalidomide, are immune modulatory drugs (IMiDs) used in the treatment of hematol. malignancies.  IMiDs bind CRBN, the substrate receptor of the CUL4-RBX1-DDB1-CRBN (also known as CRL4CRBN) E3 ubiquitin ligase, and inhibit ubiquitination of endogenous CRL4CRBN substrates.  Unexpectedly, IMiDs also repurpose the ligase to target new proteins for degrdn.  Lenalidomide induces degrdn. of the lymphoid transcription factors Ikaros and Aiolos (also known as IKZF1 and IKZF3), and casein kinase 1α (CK1α), which contributes to its clin. efficacy in the treatment of multiple myeloma and 5q-deletion assocd. myelodysplastic syndrome (del(5q) MDS), resp.  How lenalidomide alters the specificity of the ligase to degrade these proteins remains elusive.  Here we present the 2.45 Å crystal structure of DDB1-CRBN bound to lenalidomide and CK1α.  CRBN and lenalidomide jointly provide the binding interface for a CK1α β-hairpin-loop located in the kinase N-lobe.  We show that CK1α binding to CRL4CRBN is strictly dependent on the presence of an IMiD.  Binding of IKZF1 to CRBN similarly requires the compd. and both, IKZF1 and CK1α, use a related binding mode.  Our study provides a mechanistic explanation for the selective efficacy of lenalidomide in del(5q) MDS therapy.  We anticipate that high-affinity protein-protein interactions induced by small mols. will provide opportunities for drug development, particularly for targeted protein degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs4FxWriI1ObVg90H21EOLACvtfcHk0lhOQVmeO4mktg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtFOis78%253D&md5=320fc6ff811da335fa8e02feab2d702d</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnature16979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16979%26sid%3Dliteratum%253Aachs%26aulast%3DPetzold%26aufirst%3DG.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DStructural%2520Basis%2520of%2520Lenalidomide-Induced%2520CK1%25CE%25B1%2520Degradation%2520by%2520the%2520CRL4CRBN%2520Ubiquitin%2520Ligase%26jtitle%3DNature%26date%3D2016%26volume%3D532%26spage%3D127%26epage%3D130%26doi%3D10.1038%2Fnature16979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of PROTAC Cooperative Recognition for Selective Protein Degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.+H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+Basis+of+PROTAC+Cooperative+Recognition+for+Selective+Protein+Degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0lhOQVmeO4mktg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520Basis%2520of%2520PROTAC%2520Cooperative%2520Recognition%2520for%2520Selective%2520Protein%2520Degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maniaci, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Bifunctional Chemical Probes Inducing Protein–Protein Interactions</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">156</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2019.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1016%2Fj.cbpa.2019.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=31419624" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFyltbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2019&pages=145-156&author=C.+Maniaciauthor=A.+Ciulli&title=Bifunctional+Chemical+Probes+Inducing+Protein%E2%80%93Protein+Interactions&doi=10.1016%2Fj.cbpa.2019.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Bifunctional chemical probes inducing protein-protein interactions</span></div><div class="casAuthors">Maniaci, Chiara; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">145-156</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Inducing biomol. interactions with synthetic mols. to impact biol. function is a concept of enormous appeal.  Recent years have seen a resurgence of interest in designing bispecific mols. that serve as bridging agents to bring proteins together.  Pioneering structural and biophys. investigation of ternary complexes formed by mono-functional and bifunctional ligands highlights that proximity-induced stabilization or de novo formation of protein-protein interactions is a common feature of their mol. recognition.  In this review, we illustrate these concepts and advances with representative case studies, and highlight progress over the past three years, with particular focus on recruitment to E3 ubiquitin ligases by 'mol. glues' and chimeric dimerizers (PROTACs) for targeted protein degrdn.  This approach promises to significantly expand the range of tractable targets for chem. biol. and therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobXJM1PThHPbVg90H21EOLACvtfcHk0lhitJmLWBbUPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFyltbjF&md5=4302bb010003177f4f5e5c3d4a401dc9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2019.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2019.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DManiaci%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DBifunctional%2520Chemical%2520Probes%2520Inducing%2520Protein%25E2%2580%2593Protein%2520Interactions%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D52%26spage%3D145%26epage%3D156%26doi%3D10.1016%2Fj.cbpa.2019.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Vink, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrei, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milroy, L.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span> <span> </span><span class="NLM_article-title">Cooperativity Basis for Small-Molecule Stabilization of Protein–Protein Interactions</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2869</span>– <span class="NLM_lpage">2874</span>, <span class="refDoi"> DOI: 10.1039/C8SC05242E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1039%2FC8SC05242E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=30996864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVWnu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=2869-2874&author=P.+J.+de%0AVinkauthor=S.+A.+Andreiauthor=Y.+Higuchiauthor=C.+Ottmannauthor=L.-G.+Milroyauthor=L.+Brunsveld&title=Cooperativity+Basis+for+Small-Molecule+Stabilization+of+Protein%E2%80%93Protein+Interactions&doi=10.1039%2FC8SC05242E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Cooperativity basis for small-molecule stabilization of protein-protein interactions</span></div><div class="casAuthors">de Vink, Pim J.; Andrei, Sebastian A.; Higuchi, Yusuke; Ottmann, Christian; Milroy, Lech-Gustav; Brunsveld, Luc</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2869-2874</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A cooperativity framework to describe and interpret small-mol. stabilization of protein-protein interactions (PPI) is presented.  The stabilization of PPIs is a versatile and emerging therapeutic strategy to target specific combinations of protein partners within the protein interactome.  Currently, the potency of PPI stabilizers is typically expressed by their apparent affinity or EC50.  Here, we propose that the effect of a PPI stabilizer be best described involving the cooperativity factor, α, between the stabilizer and binding partners in addn. to the intrinsic affinity, KDII, of the stabilizer for one of the apo-proteins.  By way of illustration, we combine fluorescence polarization measurements with thermodn. modeling to det. the α and KDII for the PPI stabilization of 14-3-3 and TASK3 by fusicoccin-A (FC-A) and validate our approach by studying other PPI-partners of 14-3-3 proteins.  Finally, we characterize a library of different stabilizer compds., and perform structure-activity relationship studies in which mol. changes could be attributed to either changes in cooperativity or intrinsic affinity.  Such insights should aid in the development of more effective protein-protein stabilizer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSuxb4CPB-lbVg90H21EOLACvtfcHk0lhitJmLWBbUPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVWnu78%253D&md5=5c66eab5f8accfcb7a24f280e349d82b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1039%2FC8SC05242E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8SC05242E%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BVink%26aufirst%3DP.%2BJ.%26aulast%3DAndrei%26aufirst%3DS.%2BA.%26aulast%3DHiguchi%26aufirst%3DY.%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DMilroy%26aufirst%3DL.-G.%26aulast%3DBrunsveld%26aufirst%3DL.%26atitle%3DCooperativity%2520Basis%2520for%2520Small-Molecule%2520Stabilization%2520of%2520Protein%25E2%2580%2593Protein%2520Interactions%26jtitle%3DChem.%2520Sci.%26date%3D2019%26volume%3D10%26spage%3D2869%26epage%3D2874%26doi%3D10.1039%2FC8SC05242E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, J. R.</span></span> <span> </span><span class="NLM_article-title">Cooperativity in Macromolecular Assembly</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">458</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1038/nchembio.102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnchembio.102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=18641626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslClsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=458-465&author=J.+R.+Williamson&title=Cooperativity+in+Macromolecular+Assembly&doi=10.1038%2Fnchembio.102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Cooperativity in macromolecular assembly</span></div><div class="casAuthors">Williamson, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">458-465</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The thermodn. principle of cooperativity is used to drive the formation of specific macromol. complexes during the assembly of a macromol. machine.  Understanding cooperativity provides insight into the mechanisms that govern assembly and disassembly of multicomponent complexes.  Our understanding of assembly mechanisms is lagging considerably behind our understanding of the structure and function of these complexes.  A significant challenge remains in tackling the thermodn. and kinetics of the intermol. interactions required for all cellular functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppUr8iKTgUG7Vg90H21EOLACvtfcHk0lhitJmLWBbUPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslClsLc%253D&md5=37ec3e5140b64abf0c005a4f8f79b5bd</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.102%26sid%3Dliteratum%253Aachs%26aulast%3DWilliamson%26aufirst%3DJ.%2BR.%26atitle%3DCooperativity%2520in%2520Macromolecular%2520Assembly%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D458%26epage%3D465%26doi%3D10.1038%2Fnchembio.102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Twenty Years on: The Impact of Fragments on Drug Discovery</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">619</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+Years+on%3A+The+Impact+of+Fragments+on+Drug+Discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0lhvPjrKmrPMHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520Years%2520on%253A%2520The%2520Impact%2520of%2520Fragments%2520on%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2016%26volume%3D15%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Box, G.</span></span> <span> </span><span class="NLM_article-title">Perspective Fragment-Based Drug Discovery : Advancing Fragments in the Absence of Crystal Structures</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.10.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1016%2Fj.chembiol.2018.10.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=30482678" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVWisrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=9-15&author=D.+A.+Erlansonauthor=B.+J.+Davisauthor=W.+Jahnkeauthor=G.+Box&title=Perspective+Fragment-Based+Drug+Discovery+%3A+Advancing+Fragments+in+the+Absence+of+Crystal+Structures&doi=10.1016%2Fj.chembiol.2018.10.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery: Advancing Fragments in the Absence of Crystal Structures</span></div><div class="casAuthors">Erlanson, Daniel A.; Davis, Ben J.; Jahnke, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-15</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Fragment-based drug discovery typically requires an interplay between screening methods, structural methods, and medicinal chem.  X-ray crystallog. is generally the method of choice to obtain three-dimensional structures of the bound ligand/protein complex, but this can sometimes be difficult, particularly for early, low-affinity fragment hits.  In this Perspective, we discuss strategies to advance and evolve fragments in the absence of crystal structures of protein-fragment complexes, although the structure of the unliganded protein may be available.  The strategies can involve other structural techniques, such as NMR spectroscopy, mol. modeling, or a variety of chem. approaches.  Often, these strategies are aimed at guiding evolution of initial fragment hits to a stage where crystal structures can be obtained for further structure-based optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmStmkIpx6bLVg90H21EOLACvtfcHk0lhvPjrKmrPMHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVWisrzI&md5=2f5fb9d063b7ccabb4f7eb611c00ce30</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.10.001%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DDavis%26aufirst%3DB.%2BJ.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DBox%26aufirst%3DG.%26atitle%3DPerspective%2520Fragment-Based%2520Drug%2520Discovery%2520%253A%2520Advancing%2520Fragments%2520in%2520the%2520Absence%2520of%2520Crystal%2520Structures%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D9%26epage%3D15%26doi%3D10.1016%2Fj.chembiol.2018.10.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalton, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span> <span> </span><span class="NLM_article-title">Covalent Small Molecules as Enabling Platforms for Drug Discovery</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">1100</span>, <span class="refDoi"> DOI: 10.1002/cbic.201900674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1002%2Fcbic.201900674" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=1-1100&author=S.+E.+Daltonauthor=S.+Campos&title=Covalent+Small+Molecules+as+Enabling+Platforms+for+Drug+Discovery&doi=10.1002%2Fcbic.201900674"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201900674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201900674%26sid%3Dliteratum%253Aachs%26aulast%3DDalton%26aufirst%3DS.%2BE.%26aulast%3DCampos%26aufirst%3DS.%26atitle%3DCovalent%2520Small%2520Molecules%2520as%2520Enabling%2520Platforms%2520for%2520Drug%2520Discovery%26jtitle%3DChemBioChem%26date%3D2019%26spage%3D1%26epage%3D1100%26doi%3D10.1002%2Fcbic.201900674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gabizon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shraga, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehrtz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livnah, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shorer, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurwicz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avram, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aharoni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albeck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shulman, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herishanu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">London, N.</span></span> <span> </span><span class="NLM_article-title">Efficient Targeted Degradation via Reversible and Irreversible Covalent PROTACs</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">11734</span>– <span class="NLM_lpage">11742</span>, <span class="refDoi"> DOI: 10.1021/jacs.9b13907</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.9b13907" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosF2gtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2020&pages=11734-11742&author=R.+Gabizonauthor=A.+Shragaauthor=P.+Gehrtzauthor=E.+Livnahauthor=Y.+Shorerauthor=N.+Gurwiczauthor=L.+Avramauthor=T.+Ungerauthor=H.+Aharoniauthor=S.+Albeckauthor=A.+Brandisauthor=Z.+Shulmanauthor=B.+Katzauthor=Y.+Herishanuauthor=N.+London&title=Efficient+Targeted+Degradation+via+Reversible+and+Irreversible+Covalent+PROTACs&doi=10.1021%2Fjacs.9b13907"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Targeted Degradation via Reversible and Irreversible Covalent PROTACs</span></div><div class="casAuthors">Gabizon, Ronen; Shraga, Amit; Gehrtz, Paul; Livnah, Ella; Shorer, Yamit; Gurwicz, Neta; Avram, Liat; Unger, Tamar; Aharoni, Hila; Albeck, Shira; Brandis, Alexander; Shulman, Ziv; Katz, Ben-Zion; Herishanu, Yair; London, Nir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">11734-11742</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis targeting chimeras (PROTACs) represent an exciting inhibitory modality with many advantages, including substoichiometric degrdn. of targets.  Their scope, though, is still limited to date by the requirement for a sufficiently potent target binder.  A soln. that proved useful in tackling challenging targets is the use of electrophiles to allow irreversible binding to the target.  However, such binding will negate the catalytic nature of PROTACs.  Reversible covalent PROTACs potentially offer the best of both worlds.  They possess the potency and selectivity assocd. with the formation of the covalent bond, while being able to dissoc. and regenerate once the protein target is degraded.  Using Bruton's tyrosine kinase (BTK) as a clin. relevant model system, we show efficient degrdn. by noncovalent, irreversible covalent, and reversible covalent PROTACs, with <10 nM DC50's and >85% degrdn.  Our data suggest that part of the degrdn. by our irreversible covalent PROTACs is driven by reversible binding prior to covalent bond formation, while the reversible covalent PROTACs drive degrdn. primarily by covalent engagement.  The PROTACs showed enhanced inhibition of B cell activation compared to ibrutinib and exhibit potent degrdn. of BTK in patient-derived primary chronic lymphocytic leukemia cells.  The most potent reversible covalent PROTAC, RC-3(I), exhibited enhanced selectivity toward BTK compared to noncovalent and irreversible covalent PROTACs.  These compds. may pave the way for the design of covalent PROTACs for a wide variety of challenging targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHhULFyC9cFbVg90H21EOLACvtfcHk0lgwNltnXMgW4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosF2gtr0%253D&md5=dda97d35aa486202fd34ee0c8bccdab1</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjacs.9b13907&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.9b13907%26sid%3Dliteratum%253Aachs%26aulast%3DGabizon%26aufirst%3DR.%26aulast%3DShraga%26aufirst%3DA.%26aulast%3DGehrtz%26aufirst%3DP.%26aulast%3DLivnah%26aufirst%3DE.%26aulast%3DShorer%26aufirst%3DY.%26aulast%3DGurwicz%26aufirst%3DN.%26aulast%3DAvram%26aufirst%3DL.%26aulast%3DUnger%26aufirst%3DT.%26aulast%3DAharoni%26aufirst%3DH.%26aulast%3DAlbeck%26aufirst%3DS.%26aulast%3DBrandis%26aufirst%3DA.%26aulast%3DShulman%26aufirst%3DZ.%26aulast%3DKatz%26aufirst%3DB.%26aulast%3DHerishanu%26aufirst%3DY.%26aulast%3DLondon%26aufirst%3DN.%26atitle%3DEfficient%2520Targeted%2520Degradation%2520via%2520Reversible%2520and%2520Irreversible%2520Covalent%2520PROTACs%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2020%26volume%3D142%26spage%3D11734%26epage%3D11742%26doi%3D10.1021%2Fjacs.9b13907" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Resnick, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douangamath, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krojer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sethi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geurink, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aimon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amitai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairhead, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabizon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Straub, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szommer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaidman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coker, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovaa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">London, N.</span></span> <span> </span><span class="NLM_article-title">Rapid Covalent-Probe Discovery by Electrophile-Fragment Screening</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_fpage">8951</span>– <span class="NLM_lpage">8968</span>, <span class="refDoi"> DOI: 10.1021/jacs.9b02822</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.9b02822" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovVKmsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=8951-8968&author=E.+Resnickauthor=A.+Bradleyauthor=J.+Ganauthor=A.+Douangamathauthor=T.+Krojerauthor=R.+Sethiauthor=P.+P.+Geurinkauthor=A.+Aimonauthor=G.+Amitaiauthor=D.+Belliniauthor=J.+Bennettauthor=M.+Fairheadauthor=O.+Fedorovauthor=R.+Gabizonauthor=J.+Ganauthor=J.+Guoauthor=A.+Plotnikovauthor=V.+M.+Straubauthor=T.+Szommerauthor=S.+Velupillaiauthor=D.+Zaidmanauthor=Y.+Zhangauthor=A.+R.+Cokerauthor=P.+E.+Brennanauthor=H.+Ovaaauthor=F.+Von+Delftauthor=N.+London&title=Rapid+Covalent-Probe+Discovery+by+Electrophile-Fragment+Screening&doi=10.1021%2Fjacs.9b02822"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Covalent-Probe Discovery by Electrophile-Fragment Screening</span></div><div class="casAuthors">Resnick, Efrat; Bradley, Anthony; Gan, Jinrui; Douangamath, Alice; Krojer, Tobias; Sethi, Ritika; Geurink, Paul P.; Aimon, Anthony; Amitai, Gabriel; Bellini, Dom; Bennett, James; Fairhead, Michael; Fedorov, Oleg; Gabizon, Ronen; Gan, Jin; Guo, Jingxu; Plotnikov, Alexander; Reznik, Nava; Ruda, Gian Filippo; Diaz-Saez, Laura; Straub, Verena M.; Szommer, Tamas; Velupillai, Srikannathasan; Zaidman, Daniel; Zhang, Yanling; Coker, Alun R.; Dowson, Christopher G.; Barr, Haim M.; Wang, Chu; Huber, Kilian V. M.; Brennan, Paul E.; Ovaa, Huib; von Delft, Frank; London, Nir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8951-8968</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Covalent probes can display unmatched potency, selectivity, and duration of action; however, their discovery is challenging.  In principle, fragments that can irreversibly bind their target can overcome the low affinity that limits reversible fragment screening, but such electrophilic fragments were considered nonselective and were rarely screened.  We hypothesized that mild electrophiles might overcome the selectivity challenge and constructed a library of 993 mildly electrophilic fragments.  We characterized this library by a new high-throughput thiol-reactivity assay and screened them against 10 cysteine-contg. proteins.  Highly reactive and promiscuous fragments were rare and could be easily eliminated.  In contrast, we found hits for most targets.  Combining our approach with high-throughput crystallog. allowed rapid progression to potent and selective probes for two enzymes, the deubiquitinase OTUB2 and the pyrophosphatase NUDT7.  No inhibitors were previously known for either.  This study highlights the potential of electrophile-fragment screening as a practical and efficient tool for covalent-ligand discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGuVq_Zb2i_7Vg90H21EOLACvtfcHk0lgwNltnXMgW4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovVKmsrg%253D&md5=18d7a00b9ce67f4f23fbb65f473cac78</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjacs.9b02822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.9b02822%26sid%3Dliteratum%253Aachs%26aulast%3DResnick%26aufirst%3DE.%26aulast%3DBradley%26aufirst%3DA.%26aulast%3DGan%26aufirst%3DJ.%26aulast%3DDouangamath%26aufirst%3DA.%26aulast%3DKrojer%26aufirst%3DT.%26aulast%3DSethi%26aufirst%3DR.%26aulast%3DGeurink%26aufirst%3DP.%2BP.%26aulast%3DAimon%26aufirst%3DA.%26aulast%3DAmitai%26aufirst%3DG.%26aulast%3DBellini%26aufirst%3DD.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DFairhead%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DGabizon%26aufirst%3DR.%26aulast%3DGan%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DPlotnikov%26aufirst%3DA.%26aulast%3DStraub%26aufirst%3DV.%2BM.%26aulast%3DSzommer%26aufirst%3DT.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DZaidman%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCoker%26aufirst%3DA.%2BR.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DOvaa%26aufirst%3DH.%26aulast%3DVon%2BDelft%26aufirst%3DF.%26aulast%3DLondon%26aufirst%3DN.%26atitle%3DRapid%2520Covalent-Probe%2520Discovery%2520by%2520Electrophile-Fragment%2520Screening%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26spage%3D8951%26epage%3D8968%26doi%3D10.1021%2Fjacs.9b02822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Resnick, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douangamath, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krojer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sethi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geurink, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aimon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amitai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairhead, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabizon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reznik, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Díaz-Sáez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Straub, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szommer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaidman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coker, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowson, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barr, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovaa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">London, N.</span></span> <span> </span><span class="NLM_article-title">Rapid Covalent-Probe Discovery by Electrophile-Fragment Screening</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">8951</span>– <span class="NLM_lpage">8968</span>, <span class="refDoi"> DOI: 10.1021/jacs.9b02822</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.9b02822" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovVKmsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=8951-8968&author=E.+Resnickauthor=A.+Bradleyauthor=J.+Ganauthor=A.+Douangamathauthor=T.+Krojerauthor=R.+Sethiauthor=P.+P.+Geurinkauthor=A.+Aimonauthor=G.+Amitaiauthor=D.+Belliniauthor=J.+Bennettauthor=M.+Fairheadauthor=O.+Fedorovauthor=R.+Gabizonauthor=J.+Ganauthor=J.+Guoauthor=A.+Plotnikovauthor=N.+Reznikauthor=G.+F.+Rudaauthor=L.+D%C3%ADaz-S%C3%A1ezauthor=V.+M.+Straubauthor=T.+Szommerauthor=S.+Velupillaiauthor=D.+Zaidmanauthor=Y.+Zhangauthor=A.+R.+Cokerauthor=C.+G.+Dowsonauthor=H.+M.+Barrauthor=C.+Wangauthor=K.+V.+M.+Huberauthor=P.+E.+Brennanauthor=H.+Ovaaauthor=F.+Von+Delftauthor=N.+London&title=Rapid+Covalent-Probe+Discovery+by+Electrophile-Fragment+Screening&doi=10.1021%2Fjacs.9b02822"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Covalent-Probe Discovery by Electrophile-Fragment Screening</span></div><div class="casAuthors">Resnick, Efrat; Bradley, Anthony; Gan, Jinrui; Douangamath, Alice; Krojer, Tobias; Sethi, Ritika; Geurink, Paul P.; Aimon, Anthony; Amitai, Gabriel; Bellini, Dom; Bennett, James; Fairhead, Michael; Fedorov, Oleg; Gabizon, Ronen; Gan, Jin; Guo, Jingxu; Plotnikov, Alexander; Reznik, Nava; Ruda, Gian Filippo; Diaz-Saez, Laura; Straub, Verena M.; Szommer, Tamas; Velupillai, Srikannathasan; Zaidman, Daniel; Zhang, Yanling; Coker, Alun R.; Dowson, Christopher G.; Barr, Haim M.; Wang, Chu; Huber, Kilian V. M.; Brennan, Paul E.; Ovaa, Huib; von Delft, Frank; London, Nir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8951-8968</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Covalent probes can display unmatched potency, selectivity, and duration of action; however, their discovery is challenging.  In principle, fragments that can irreversibly bind their target can overcome the low affinity that limits reversible fragment screening, but such electrophilic fragments were considered nonselective and were rarely screened.  We hypothesized that mild electrophiles might overcome the selectivity challenge and constructed a library of 993 mildly electrophilic fragments.  We characterized this library by a new high-throughput thiol-reactivity assay and screened them against 10 cysteine-contg. proteins.  Highly reactive and promiscuous fragments were rare and could be easily eliminated.  In contrast, we found hits for most targets.  Combining our approach with high-throughput crystallog. allowed rapid progression to potent and selective probes for two enzymes, the deubiquitinase OTUB2 and the pyrophosphatase NUDT7.  No inhibitors were previously known for either.  This study highlights the potential of electrophile-fragment screening as a practical and efficient tool for covalent-ligand discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGuVq_Zb2i_7Vg90H21EOLACvtfcHk0lihnT1s2Gn8SA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovVKmsrg%253D&md5=18d7a00b9ce67f4f23fbb65f473cac78</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjacs.9b02822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.9b02822%26sid%3Dliteratum%253Aachs%26aulast%3DResnick%26aufirst%3DE.%26aulast%3DBradley%26aufirst%3DA.%26aulast%3DGan%26aufirst%3DJ.%26aulast%3DDouangamath%26aufirst%3DA.%26aulast%3DKrojer%26aufirst%3DT.%26aulast%3DSethi%26aufirst%3DR.%26aulast%3DGeurink%26aufirst%3DP.%2BP.%26aulast%3DAimon%26aufirst%3DA.%26aulast%3DAmitai%26aufirst%3DG.%26aulast%3DBellini%26aufirst%3DD.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DFairhead%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DGabizon%26aufirst%3DR.%26aulast%3DGan%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DPlotnikov%26aufirst%3DA.%26aulast%3DReznik%26aufirst%3DN.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DD%25C3%25ADaz-S%25C3%25A1ez%26aufirst%3DL.%26aulast%3DStraub%26aufirst%3DV.%2BM.%26aulast%3DSzommer%26aufirst%3DT.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DZaidman%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCoker%26aufirst%3DA.%2BR.%26aulast%3DDowson%26aufirst%3DC.%2BG.%26aulast%3DBarr%26aufirst%3DH.%2BM.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DOvaa%26aufirst%3DH.%26aulast%3DVon%2BDelft%26aufirst%3DF.%26aulast%3DLondon%26aufirst%3DN.%26atitle%3DRapid%2520Covalent-Probe%2520Discovery%2520by%2520Electrophile-Fragment%2520Screening%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D8951%26epage%3D8968%26doi%3D10.1021%2Fjacs.9b02822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hallenbeck, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogden, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sijbesma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renslo, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span> <span> </span><span class="NLM_article-title">A Liquid Chromatography/Mass Spectrometry Method for Screening Disulfide Tethering Fragments</span>. <i>SLAS Discov.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">183</span>– <span class="NLM_lpage">192</span>, <span class="refDoi"> DOI: 10.1177/2472555217732072</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1177%2F2472555217732072" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=28945980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFCqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=183-192&author=K.+K.+Hallenbeckauthor=J.+L.+Daviesauthor=C.+Merronauthor=P.+Ogdenauthor=E.+Sijbesmaauthor=C.+Ottmannauthor=A.+R.+Rensloauthor=C.+Wilsonauthor=M.+R.+Arkin&title=A+Liquid+Chromatography%2FMass+Spectrometry+Method+for+Screening+Disulfide+Tethering+Fragments&doi=10.1177%2F2472555217732072"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A Liquid Chromatography/Mass Spectrometry Method for Screening Disulfide Tethering Fragments</span></div><div class="casAuthors">Hallenbeck, Kenneth K.; Davies, Julia L.; Merron, Connie; Ogden, Pierce; Sijbesma, Eline; Ottmann, Christian; Renslo, Adam R.; Wilson, Christopher; Arkin, Michelle R.</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">183-192</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">We report the refinement of a high-throughput, liq. chromatog./mass spectrometry (LC/MS)-based screening method for the identification of covalent small-mol. binders to proteins.  Using a custom library of 1600 disulfide-capped fragments targeting surface cysteine residues, we optimize sample prepn., chromatog., and ionization conditions to maximize the reliability and flexibility of the approach.  Data collection at a rate of 84 s per sample balances speed with reliability for sustained screening over multiple, diverse projects run over a 24-mo period.  The method is applicable to protein targets of various classes and a range of mol. masses.  Data are processed in a custom pipeline that calcs. a percent bound value for each compd. and identifies false positives by calcg. significance of detected masses (signal significance).  An example pipeline is available through Biovia's ScienceCloud Protocol Exchange.  Data collection and anal. methods for the screening of covalent adducts of intact proteins are now fast enough to screen the largest covalent compd. libraries in 1 to 2 days.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokAicgwlw2rrVg90H21EOLACvtfcHk0lj5DBC2pbclBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFCqt70%253D&md5=8be6b7afd4854cf0e4361d5bd6743663</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1177%2F2472555217732072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555217732072%26sid%3Dliteratum%253Aachs%26aulast%3DHallenbeck%26aufirst%3DK.%2BK.%26aulast%3DDavies%26aufirst%3DJ.%2BL.%26aulast%3DMerron%26aufirst%3DC.%26aulast%3DOgden%26aufirst%3DP.%26aulast%3DSijbesma%26aufirst%3DE.%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DRenslo%26aufirst%3DA.%2BR.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26atitle%3DA%2520Liquid%2520Chromatography%252FMass%2520Spectrometry%2520Method%2520for%2520Screening%2520Disulfide%2520Tethering%2520Fragments%26jtitle%3DSLAS%2520Discov.%26date%3D2018%26volume%3D23%26spage%3D183%26epage%3D192%26doi%3D10.1177%2F2472555217732072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">London, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eidam, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibold, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cimermančič, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnet, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span> <span> </span><span class="NLM_article-title">Covalent Docking of Large Libraries for the Discovery of Chemical Probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1066</span>– <span class="NLM_lpage">1072</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnchembio.1666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=25344815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVSqsbjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=1066-1072&author=N.+Londonauthor=R.+M.+Millerauthor=S.+Krishnanauthor=K.+Uchidaauthor=J.+J.+Irwinauthor=O.+Eidamauthor=L.+Giboldauthor=P.+Cimerman%C4%8Di%C4%8Dauthor=R.+Bonnetauthor=B.+K.+Shoichetauthor=J.+Taunton&title=Covalent+Docking+of+Large+Libraries+for+the+Discovery+of+Chemical+Probes&doi=10.1038%2Fnchembio.1666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent docking of large libraries for the discovery of chemical probes</span></div><div class="casAuthors">London, Nir; Miller, Rand M.; Krishnan, Shyam; Uchida, Kenji; Irwin, John J.; Eidam, Oliv; Gibold, Lucie; Cimermancic, Peter; Bonnet, Richard; Shoichet, Brian K.; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1066-1072</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes that form a covalent bond with a protein target often show enhanced selectivity, potency and utility for biol. studies.  Despite these advantages, protein-reactive compds. are usually avoided in high-throughput screening campaigns.  Here we describe a general method (DOCKovalent) for screening large virtual libraries of electrophilic small mols.  We apply this method prospectively to discover reversible covalent fragments that target distinct protein nucleophiles, including the catalytic serine of AmpC β-lactamase and noncatalytic cysteines in RSK2, MSK1 and JAK3 kinases.  We identify submicromolar to low-nanomolar hits with high ligand efficiency, cellular activity and selectivity, including what are to our knowledge the first reported reversible covalent inhibitors of JAK3.  Crystal structures of inhibitor complexes with AmpC and RSK2 confirm the docking predictions and guide further optimization.  As covalent virtual screening may have broad utility for the rapid discovery of chem. probes, we have made the method freely available through an automated web server (http://covalent.docking.org/).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpngQdIYqslNLVg90H21EOLACvtfcHk0lj5DBC2pbclBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVSqsbjE&md5=4e5fbeb8f67cb82d464d57069ba0c93a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1666%26sid%3Dliteratum%253Aachs%26aulast%3DLondon%26aufirst%3DN.%26aulast%3DMiller%26aufirst%3DR.%2BM.%26aulast%3DKrishnan%26aufirst%3DS.%26aulast%3DUchida%26aufirst%3DK.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DEidam%26aufirst%3DO.%26aulast%3DGibold%26aufirst%3DL.%26aulast%3DCimerman%25C4%258Di%25C4%258D%26aufirst%3DP.%26aulast%3DBonnet%26aufirst%3DR.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DCovalent%2520Docking%2520of%2520Large%2520Libraries%2520for%2520the%2520Discovery%2520of%2520Chemical%2520Probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D1066%26epage%3D1072%26doi%3D10.1038%2Fnchembio.1666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sijbesma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallenbeck, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leysen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Vink, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skóra, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Site-Directed Fragment-Based Screening for the Discovery of Protein-Protein Interaction Stabilizers</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">3524</span>– <span class="NLM_lpage">3531</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b11658</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b11658" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVWjtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=3524-3531&author=E.+Sijbesmaauthor=K.+K.+Hallenbeckauthor=S.+Leysenauthor=P.+J.+De+Vinkauthor=L.+Sk%C3%B3raauthor=W.+Jahnkeauthor=L.+Brunsveldauthor=M.+R.+Arkinauthor=C.+Ottmann&title=Site-Directed+Fragment-Based+Screening+for+the+Discovery+of+Protein-Protein+Interaction+Stabilizers&doi=10.1021%2Fjacs.8b11658"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Site-Directed Fragment-Based Screening for the Discovery of Protein-Protein Interaction Stabilizers</span></div><div class="casAuthors">Sijbesma, Eline; Hallenbeck, Kenneth K.; Leysen, Seppe; de Vink, Pim J.; Skora, Lukasz; Jahnke, Wolfgang; Brunsveld, Luc; Arkin, Michelle R.; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3524-3531</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modulation of protein-protein interactions (PPIs) by small mols. has emerged as a valuable approach in drug discovery.  Compared to direct inhibition, PPI stabilization is vastly underexplored but has strong advantages, including the ability to gain selectivity by targeting an interface formed only upon assocn. of proteins.  Here, we present the application of a site-directed screening technique based on disulfide trapping (tethering) to select for fragments that enhance the affinity between protein partners.  We target the phosphorylation-dependent interaction between the hub protein 14-3-3σ and a peptide derived from Estrogen Receptor α (ERα), an important breast cancer target that is neg. regulated by 14-3-3σ.  We identify orthosteric stabilizers that increase 14-3-3/ERα affinity up to 40-fold and propose the mechanism of stabilization based on X-ray crystal structures.  These fragments already display partial selectivity toward ERα-like motifs over other representative 14-3-3 clients.  This first of its kind study illustrates the potential of the tethering approach to overcome the hurdles in systematic PPI stabilizer discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1_kzNxRmRPrVg90H21EOLACvtfcHk0lj5DBC2pbclBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVWjtrY%253D&md5=5e68f397675437fffdca4cc47ed8e078</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b11658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b11658%26sid%3Dliteratum%253Aachs%26aulast%3DSijbesma%26aufirst%3DE.%26aulast%3DHallenbeck%26aufirst%3DK.%2BK.%26aulast%3DLeysen%26aufirst%3DS.%26aulast%3DDe%2BVink%26aufirst%3DP.%2BJ.%26aulast%3DSk%25C3%25B3ra%26aufirst%3DL.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DSite-Directed%2520Fragment-Based%2520Screening%2520for%2520the%2520Discovery%2520of%2520Protein-Protein%2520Interaction%2520Stabilizers%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D3524%26epage%3D3531%26doi%3D10.1021%2Fjacs.8b11658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sijbesma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somsen, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miley, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leijten-van
de Gevel, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Fluorescence Anisotropy-Based Tethering for Discovery of Protein–Protein Interaction Stabilizers</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_fpage">3143</span>– <span class="NLM_lpage">3148</span>, <span class="refDoi"> DOI: 10.1021/acschembio.0c00646</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.0c00646" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitlagtLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=3143-3148&author=E.+Sijbesmaauthor=B.+A.+Somsenauthor=G.+P.+Mileyauthor=I.+A.+Leijten-van%0Ade+Gevelauthor=L.+Brunsveldauthor=M.+R.+Arkinauthor=C.+Ottmann&title=Fluorescence+Anisotropy-Based+Tethering+for+Discovery+of+Protein%E2%80%93Protein+Interaction+Stabilizers&doi=10.1021%2Facschembio.0c00646"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence Anisotropy-Based Tethering for Discovery of Protein-Protein Interaction Stabilizers</span></div><div class="casAuthors">Sijbesma, Eline; Somsen, Bente A.; Miley, Galen P.; Leijten-van de Gevel, Iris A.; Brunsveld, Luc; Arkin, Michelle R.; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3143-3148</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-protein interaction (PPI) networks are fundamental for cellular processes.  Small-mol. PPI enhancers have been shown to be powerful tools to fundamentally study PPIs and as starting points for potential new therapeutics.  Yet, systematic approaches for their discovery are not widely available, and the design prerequisites of "mol. glues" are poorly understood.  Covalent fragment-based screening can identify chem. starting points for these enhancers at specific sites in PPI interfaces.  We recently reported a mass spectrometry-based disulfide-trapping (tethering) approach for a cysteine residue in the hub protein 14-3-3, an important regulator of phosphorylated client proteins.  Here, we invert the strategy and report the development of a functional read-out for systematic identification of PPI enhancers based on fluorescence anisotropy (FA-tethering) with the reactive handle now on a client-derived peptide.  Using the DNA-binding domain of the nuclear receptor Estrogen Related Receptor gamma (ERRγ), we target a native cysteine positioned at the 14-3-3 PPI interface and identify several fragments that form a disulfide bond to ERRγ and stabilize the complex up to 5-fold.  Crystallog. indicates that fragments bind in a pocket comprised of 14-3-3 and the ERRγ phosphopeptide.  FA-tethering presents a streamlined methodol. to discover mol. glues for protein complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoey2CDD3wVnLVg90H21EOLACvtfcHk0likb5vv399pNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitlagtLnM&md5=0c6220dfa4281bdb7f196402ef983a57</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facschembio.0c00646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.0c00646%26sid%3Dliteratum%253Aachs%26aulast%3DSijbesma%26aufirst%3DE.%26aulast%3DSomsen%26aufirst%3DB.%2BA.%26aulast%3DMiley%26aufirst%3DG.%2BP.%26aulast%3DLeijten-van%2Bde%2BGevel%26aufirst%3DI.%2BA.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DFluorescence%2520Anisotropy-Based%2520Tethering%2520for%2520Discovery%2520of%2520Protein%25E2%2580%2593Protein%2520Interaction%2520Stabilizers%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26spage%3D3143%26epage%3D3148%26doi%3D10.1021%2Facschembio.0c00646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valenti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cossar, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hristeva, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzalis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Stabilizers of Protein–Protein Interactions through Imine-Based Tethering</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">21520</span>– <span class="NLM_lpage">21524</span>, <span class="refDoi"> DOI: 10.1002/anie.202008585</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1002%2Fanie.202008585" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFWmtL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=21520-21524&author=M.+Wolterauthor=D.+Valentiauthor=P.+J.+Cossarauthor=L.+M.+Levyauthor=S.+Hristevaauthor=T.+Genskiauthor=T.+Hoffmannauthor=L.+Brunsveldauthor=D.+Tzalisauthor=C.+Ottmann&title=Fragment-Based+Stabilizers+of+Protein%E2%80%93Protein+Interactions+through+Imine-Based+Tethering&doi=10.1002%2Fanie.202008585"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Stabilizers of Protein-Protein Interactions through Imine-Based Tethering</span></div><div class="casAuthors">Wolter, Madita; Valenti, Dario; Cossar, Peter J.; Levy, Laura M.; Hristeva, Stanimira; Genski, Thorsten; Hoffmann, Torsten; Brunsveld, Luc; Tzalis, Dimitrios; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">21520-21524</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Small-mol. stabilization of protein-protein interactions (PPIs) is a promising concept in drug discovery, however the question how to identify or design chem. starting points in a "bottom-up" approach is largely unanswered.  We report a novel concept for identifying initial chem. matter for PPI stabilization based on imine-forming fragments.  The imine bond offers a covalent anchor for site-directed fragment targeting, whereas its transient nature enables efficient anal. of structure-activity relationships.  This bond enables fragment identification and optimization using protein crystallog.  We report novel fragments that bind specifically to a lysine at the PPI interface of the p65-subunit-derived peptide of NF-κB with the adapter protein 14-3-3.  Those fragments that subsequently establish contacts with the p65-derived peptide, rather than with 14-3-3, efficiently stabilize the 14-3-3/p65 complex and offer novel starting points for mol. glues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV4so8tUhVerVg90H21EOLACvtfcHk0likb5vv399pNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFWmtL7F&md5=8a1e3e6d8f989a704f3d5c8213bfeeb3</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fanie.202008585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.202008585%26sid%3Dliteratum%253Aachs%26aulast%3DWolter%26aufirst%3DM.%26aulast%3DValenti%26aufirst%3DD.%26aulast%3DCossar%26aufirst%3DP.%2BJ.%26aulast%3DLevy%26aufirst%3DL.%2BM.%26aulast%3DHristeva%26aufirst%3DS.%26aulast%3DGenski%26aufirst%3DT.%26aulast%3DHoffmann%26aufirst%3DT.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DTzalis%26aufirst%3DD.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DFragment-Based%2520Stabilizers%2520of%2520Protein%25E2%2580%2593Protein%2520Interactions%2520through%2520Imine-Based%2520Tethering%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2020%26volume%3D59%26spage%3D21520%26epage%3D21524%26doi%3D10.1002%2Fanie.202008585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span> <span> </span><span class="NLM_article-title">Lysine-Targeting Covalent Inhibitors</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">15200</span>– <span class="NLM_lpage">15209</span>, <span class="refDoi"> DOI: 10.1002/anie.201707630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1002%2Fanie.201707630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslejtr3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=15200-15209&author=J.+Pettingerauthor=K.+Jonesauthor=M.+D.+Cheeseman&title=Lysine-Targeting+Covalent+Inhibitors&doi=10.1002%2Fanie.201707630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-Targeting Covalent Inhibitors</span></div><div class="casAuthors">Pettinger, Jonathan; Jones, Keith; Cheeseman, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">15200-15209</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Targeted covalent inhibitors have gained widespread attention in drug discovery as a validated method to circumvent acquired resistance in oncol.  This strategy exploits small-mol./protein crystal structures to design tightly binding ligands with appropriately positioned electrophilic warheads.  While most focus has been on targeting binding-site cysteine residues, targeting nucleophilic lysine residues can also represent a viable approach to irreversible inhibition.  However, owing to the basicity of the ε-amino group in lysine, this strategy generates a no. of specific challenges.  Herein, the authors review the key principles for inhibitor design, give historical examples, and present recent developments that demonstrate the potential of lysine targeting for future drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUIzd2aDc06bVg90H21EOLACvtfcHk0liSa_VdRCzOZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslejtr3P&md5=4a6bb561e53c165fa3c7c85953bb4736</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fanie.201707630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201707630%26sid%3Dliteratum%253Aachs%26aulast%3DPettinger%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26atitle%3DLysine-Targeting%2520Covalent%2520Inhibitors%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D15200%26epage%3D15209%26doi%3D10.1002%2Fanie.201707630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span> <span> </span><span class="NLM_article-title">Kinetic Optimization of Lysine-Targeting Covalent Inhibitors of HSP72</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11383</span>– <span class="NLM_lpage">11398</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01709</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01709" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFKksLrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11383-11398&author=J.+Pettingerauthor=M.+Carterauthor=K.+Jonesauthor=M.+D.+Cheeseman&title=Kinetic+Optimization+of+Lysine-Targeting+Covalent+Inhibitors+of+HSP72&doi=10.1021%2Facs.jmedchem.9b01709"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Optimization of Lysine-Targeting Covalent Inhibitors of HSP72</span></div><div class="casAuthors">Pettinger, Jonathan; Carter, Michael; Jones, Keith; Cheeseman, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11383-11398</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The covalent inhibition mechanism of action, which overcomes competition with high-affinity, high-abundance substrates of challenging protein targets, can deliver effective chem. probes and drugs.  The success of this strategy has centered on exposed cysteine residues as nucleophiles but the low abundance of cysteine in the proteome has limited its application.  We have recently reported our discovery that lysine-56 in the difficult-to-drug target HSP72 could form a covalent bond with a small-mol. inhibitor.  We now disclose the optimization of these targeted covalent inhibitors using rational design.  Essential to our optimization was the development of a new covalent fluorescence polarization assay, which allows for the direct measurement of the key kinetic parameter in covalent inhibitor design, kinact/KI, extrapolation of the underlying parameters, kinact and Ki, and direct comparison to reversible analogs.  Using our approach, we demonstrate a >100-fold enhancement in covalent efficiency and key learnings in lysine-selective electrophile optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK27fPoDap37Vg90H21EOLACvtfcHk0liSa_VdRCzOZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFKksLrI&md5=6c02be025216bdf8d031dbebbe126456</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01709%26sid%3Dliteratum%253Aachs%26aulast%3DPettinger%26aufirst%3DJ.%26aulast%3DCarter%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26atitle%3DKinetic%2520Optimization%2520of%2520Lysine-Targeting%2520Covalent%2520Inhibitors%2520of%2520HSP72%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11383%26epage%3D11398%26doi%3D10.1021%2Facs.jmedchem.9b01709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Bihan, Y.-V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Montfort, R. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span> <span> </span><span class="NLM_article-title">An Irreversible Inhibitor of HSP72 That Unexpectedly Targets Lysine-56</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">3536</span>– <span class="NLM_lpage">3540</span>, <span class="refDoi"> DOI: 10.1002/anie.201611907</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1002%2Fanie.201611907" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVeru7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=3536-3540&author=J.+Pettingerauthor=Y.-V.+Le+Bihanauthor=M.+Widyaauthor=R.+L.+M.+van+Montfortauthor=K.+Jonesauthor=M.+D.+Cheeseman&title=An+Irreversible+Inhibitor+of+HSP72+That+Unexpectedly+Targets+Lysine-56&doi=10.1002%2Fanie.201611907"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">An Irreversible Inhibitor of HSP72 that Unexpectedly Targets Lysine-56</span></div><div class="casAuthors">Pettinger, Jonathan; Le Bihan, Yann-Vai; Widya, Marcella; van Montfort, Rob L. M.; Jones, Keith; Cheeseman, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3536-3540</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The stress-inducible mol. chaperone, HSP72, is an important therapeutic target in oncol., but inhibiting this protein with small mols. has proven particularly challenging.  Validating HSP72 inhibitors in cells is difficult owing to competition with the high affinity and abundance of its endogenous nucleotide substrates.  We hypothesized this could be overcome using a cysteine-targeted irreversible inhibitor.  Using rational design, we adapted a validated 8-N-benzyladenosine ligand for covalent bond formation and confirmed targeted irreversible inhibition.  However, no cysteine in the protein was modified; instead, we demonstrate that lysine-56 is the key nucleophilic residue.  Targeting this lysine could lead to a new design paradigm for HSP72 chem. probes and drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0a7TsyW4PurVg90H21EOLACvtfcHk0liSa_VdRCzOZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtVeru7o%253D&md5=8b2cbd17140e5f0c6430d0164e03280a</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611907&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611907%26sid%3Dliteratum%253Aachs%26aulast%3DPettinger%26aufirst%3DJ.%26aulast%3DLe%2BBihan%26aufirst%3DY.-V.%26aulast%3DWidya%26aufirst%3DM.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%2BM.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26atitle%3DAn%2520Irreversible%2520Inhibitor%2520of%2520HSP72%2520That%2520Unexpectedly%2520Targets%2520Lysine-56%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D3536%26epage%3D3540%26doi%3D10.1002%2Fanie.201611907" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalton, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittus, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Convery, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bush, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span> <span> </span><span class="NLM_article-title">Selectively Targeting the Kinome-Conserved Lysine of PI3Kδ as a General Approach to Covalent Kinase Inhibition</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">932</span>– <span class="NLM_lpage">939</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b08979</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b08979" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFKitrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=932-939&author=S.+E.+Daltonauthor=L.+Dittusauthor=D.+A.+Thomasauthor=M.+A.+Converyauthor=J.+Nunesauthor=J.+T.+Bushauthor=J.+P.+Evansauthor=T.+Wernerauthor=M.+Bantscheffauthor=J.+A.+Murphyauthor=S.+Campos&title=Selectively+Targeting+the+Kinome-Conserved+Lysine+of+PI3K%CE%B4+as+a+General+Approach+to+Covalent+Kinase+Inhibition&doi=10.1021%2Fjacs.7b08979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Selectively Targeting the Kinome-Conserved Lysine of PI3Kδ as a General Approach to Covalent Kinase Inhibition</span></div><div class="casAuthors">Dalton, Samuel E.; Dittus, Lars; Thomas, Daniel A.; Convery, Maire A.; Nunes, Joao; Bush, Jacob T.; Evans, John P.; Werner, Thilo; Bantscheff, Marcus; Murphy, John A.; Campos, Sebastien</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">932-939</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective covalent inhibition of kinases by targeting poorly conserved cysteines has proven highly fruitful to date in the development of chem. probes and approved drugs.  However, this approach is limited to ∼200 kinases possessing such a cysteine near the ATP-binding pocket.  Herein, we report a novel approach to achieve selective, irreversible kinase inhibition, by targeting the conserved catalytic lysine residue.  We have illustrated our approach by developing selective, covalent PI3Kδ inhibitors that exhibit nanomolar potency in cellular assays, and a duration of action >48 h in CD4+ T cells.  Despite conservation of the lysine residue throughout the kinome, the lead compd. shows high levels of selectivity over a selection of lipid and protein kinases in biochem. assays, as well as covalent binding to very few off-target proteins in live-cell proteomic studies.  We anticipate this approach could offer a general strategy, as an alternative to targeting non-conserved cysteines, for the development of selective covalent kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGvT7h2D9zf7Vg90H21EOLACvtfcHk0lign39zTIFo3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFKitrrK&md5=90bb399303a942fb87f1b9096eb8bdc6</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b08979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b08979%26sid%3Dliteratum%253Aachs%26aulast%3DDalton%26aufirst%3DS.%2BE.%26aulast%3DDittus%26aufirst%3DL.%26aulast%3DThomas%26aufirst%3DD.%2BA.%26aulast%3DConvery%26aufirst%3DM.%2BA.%26aulast%3DNunes%26aufirst%3DJ.%26aulast%3DBush%26aufirst%3DJ.%2BT.%26aulast%3DEvans%26aufirst%3DJ.%2BP.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DMurphy%26aufirst%3DJ.%2BA.%26aulast%3DCampos%26aufirst%3DS.%26atitle%3DSelectively%2520Targeting%2520the%2520Kinome-Conserved%2520Lysine%2520of%2520PI3K%25CE%25B4%2520as%2520a%2520General%2520Approach%2520to%2520Covalent%2520Kinase%2520Inhibition%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D932%26epage%3D939%26doi%3D10.1021%2Fjacs.7b08979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, I. M.</span></span> <span> </span><span class="NLM_article-title">NF-ΚB Regulation in the Immune System</span>. <i>Nat. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">725</span>– <span class="NLM_lpage">734</span>, <span class="refDoi"> DOI: 10.1038/nri910</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnri910" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=12360211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsVGqsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=725-734&author=Q.+Liauthor=I.+M.+Verma&title=NF-%CE%9AB+Regulation+in+the+Immune+System&doi=10.1038%2Fnri910"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">NF-κB regulation in the immune system</span></div><div class="casAuthors">Li, Qiutang; Verma, Inder M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">725-734</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The nuclear factor-κB (NF-κB)/REL family of transcription factors has a central role in coordinating the expression of a wide variety of genes that control immune responses.  There has been intense scientific activity in the NF-κB field owing to the involvement of these factors in the activation and regulation of key mols. that are assocd. with diseases ranging from inflammation to cancer.  Here, the authors focus on the current understanding of NF-κB regulation and its role in the immune system and inflammatory diseases.  The authors also discuss the role of NF-κB proteins as potential therapeutic targets in clin. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAP3VO2ff1orVg90H21EOLACvtfcHk0lign39zTIFo3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsVGqsbg%253D&md5=5dbb972341f6e063df6d46389217f88e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnri910&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri910%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DVerma%26aufirst%3DI.%2BM.%26atitle%3DNF-%25CE%259AB%2520Regulation%2520in%2520the%2520Immune%2520System%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2002%26volume%3D2%26spage%3D725%26epage%3D734%26doi%3D10.1038%2Fnri910" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakanishi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toi, M.</span></span> <span> </span><span class="NLM_article-title">Nuclear Factor-ΚB Inhibitors as Sensitizers to Anticancer Drugs</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">297</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.1038/nrc1588</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnrc1588" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=15803156" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivVars7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2005&pages=297-309&author=C.+Nakanishiauthor=M.+Toi&title=Nuclear+Factor-%CE%9AB+Inhibitors+as+Sensitizers+to+Anticancer+Drugs&doi=10.1038%2Fnrc1588"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Nuclear factor-κB inhibitors as sensitizers to anticancer drugs</span></div><div class="casAuthors">Nakanishi, Chikashi; Toi, Masakazu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">297-309</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The cytotoxicity of chemotherapeutic agents is attributed to apoptosis.  Acquired resistance to the effects of chemotherapy has emerged as a significant impediment to effective cancer therapy.  One feature that cytotoxic treatments of cancer have in common is their activation of the transcription factor nuclear factor-κB (NF-κB), which regulates cell survival.  NF-κB activation suppresses the apoptotic potential of chemotherapeutic agents and contributes to resistance.  What evidence is there that inhibitors of NF-κB might promote apoptosis in cancer cells and can NF-κB inhibitors be used to overcome resistance to chemotherapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZsXlm8lIyTbVg90H21EOLACvtfcHk0liNR1iVQ4S5ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivVars7Y%253D&md5=4449832c63d81112812502b97d2fdaea</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnrc1588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1588%26sid%3Dliteratum%253Aachs%26aulast%3DNakanishi%26aufirst%3DC.%26aulast%3DToi%26aufirst%3DM.%26atitle%3DNuclear%2520Factor-%25CE%259AB%2520Inhibitors%2520as%2520Sensitizers%2520to%2520Anticancer%2520Drugs%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2005%26volume%3D5%26spage%3D297%26epage%3D309%26doi%3D10.1038%2Fnrc1588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karin, M.</span></span> <span> </span><span class="NLM_article-title">NF-κB, Inflammation, Immunity and Cancer: Coming of Age</span>. <i>Nat. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">309</span>– <span class="NLM_lpage">324</span>, <span class="refDoi"> DOI: 10.1038/nri.2017.142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnri.2017.142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=29379212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yjtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=309-324&author=K.+Taniguchiauthor=M.+Karin&title=NF-%CE%BAB%2C+Inflammation%2C+Immunity+and+Cancer%3A+Coming+of+Age&doi=10.1038%2Fnri.2017.142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">NF-κB, inflammation, immunity and cancer: coming of age</span></div><div class="casAuthors">Taniguchi, Koji; Karin, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">309-324</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Fourteen years have passed since nuclear factor-κB (NF-κB) was first shown to serve as a mol. lynchpin that links persistent infections and chronic inflammation to increased cancer risk.  The young field of inflammation and cancer has now come of age, and inflammation has been recognized by the broad cancer research community as a hallmark and cause of cancer.  Here, we discuss how the initial discovery of a role for NF-κB in linking inflammation and cancer led to an improved understanding of tumor-elicited inflammation and its effects on anticancer immunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbWVUEqQxv9rVg90H21EOLACvtfcHk0liNR1iVQ4S5ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yjtrk%253D&md5=02d3c489732baab402ddf87d18416bd6</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnri.2017.142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri.2017.142%26sid%3Dliteratum%253Aachs%26aulast%3DTaniguchi%26aufirst%3DK.%26aulast%3DKarin%26aufirst%3DM.%26atitle%3DNF-%25CE%25BAB%252C%2520Inflammation%252C%2520Immunity%2520and%2520Cancer%253A%2520Coming%2520of%2520Age%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2018%26volume%3D18%26spage%3D309%26epage%3D324%26doi%3D10.1038%2Fnri.2017.142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arepalli, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, J.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span> <span> </span><span class="NLM_article-title">Novel NF-κB Inhibitors: A Patent Review (2011 – 2014)</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">334</span>, <span class="refDoi"> DOI: 10.1517/13543776.2014.998199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1517%2F13543776.2014.998199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=25553724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2MXit12ltLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=319-334&author=S.+K.+Arepalliauthor=M.+Choiauthor=J.-K.+Jungauthor=H.+Lee&title=Novel+NF-%CE%BAB+Inhibitors%3A+A+Patent+Review+%282011+%E2%80%93+2014%29&doi=10.1517%2F13543776.2014.998199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Novel NF-κB inhibitors: a patent review (2011 - 2014)</span></div><div class="casAuthors">Arepalli, Sateesh Kumar; Choi, Minho; Jung, Jae-Kyung; Lee, Heesoon</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">319-334</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Importance of the field: The NF-κB family of transcription factors has a vital role in coordinating the expression of a wide variety of genes that control immune responses.  The dysfunction of NF-κB has been assocd. with diseases ranging from inflammation to cancer.  Areas covered: In this review, authors provided information on the role of NF-κB in human health and on recently developed inhibitors from patents granted in 2011 - 2014.  The reader will gain an understanding of the mechanisms of NF-κB function as well as the structures and biol. activities of recently developed NF-κB inhibitors.  Expert opinion: Constitutive NF-κB activation is likely involved in the pathogenesis of several diseases.  Researchers need to consider the specific mechanism of each invented NF-κB inhibitor, as well as the properties of the cellular, in vitro and in vivo assays and targeted diseases.  For new drug development, they need various biol. tests to assess the potential side effects and optimization of the biol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4fFLw8dJ5jLVg90H21EOLACvtfcHk0lh1dmTH0uVLSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXit12ltLk%253D&md5=59349fb2f77470bbdc0045ccf64257ac</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1517%2F13543776.2014.998199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2014.998199%26sid%3Dliteratum%253Aachs%26aulast%3DArepalli%26aufirst%3DS.%2BK.%26aulast%3DChoi%26aufirst%3DM.%26aulast%3DJung%26aufirst%3DJ.-K.%26aulast%3DLee%26aufirst%3DH.%26atitle%3DNovel%2520NF-%25CE%25BAB%2520Inhibitors%253A%2520A%2520Patent%2520Review%2520%25282011%2520%25E2%2580%2593%25202014%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2015%26volume%3D25%26spage%3D319%26epage%3D334%26doi%3D10.1517%2F13543776.2014.998199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaltschmidt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greiner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadhim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaltschmidt, C.</span></span> <span> </span><span class="NLM_article-title">Subunit-Specific Role of NF-κB in Cancer</span>. <i>Biomedicines</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">44</span>, <span class="refDoi"> DOI: 10.3390/biomedicines6020044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.3390%2Fbiomedicines6020044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvFaju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=44&author=B.+Kaltschmidtauthor=J.+Greinerauthor=H.+Kadhimauthor=C.+Kaltschmidt&title=Subunit-Specific+Role+of+NF-%CE%BAB+in+Cancer&doi=10.3390%2Fbiomedicines6020044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Subunit-specific role of NF-κB in cancer</span></div><div class="casAuthors">Kaltschmidt, Barbara; Greiner, Johannes F. W.; Kadhim, Hussamadin M.; Kaltschmidt, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Biomedicines</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">44/1-44/12</span>CODEN:
                <span class="NLM_cas:coden">BIOMID</span>;
        ISSN:<span class="NLM_cas:issn">2227-9059</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  The transcription factor NF-κ B is a key player in inflammation, cancer development, and progression.  NF-κ B stimulates cell proliferation, prevents apoptosis, and could promote tumor angiogenesis as well as metastasis.  Extending the commonly accepted role of NF-κ B in cancer formation and progression, different NF-κ B subunits have been shown to be active and of particular importance in distinct types of cancer.  Here, we summarize overexpression data of the NF-κ B subunits RELA, RELB, and c-REL (referring to the v-REL, which is the oncogene of Reticuloendotheliosis virus strain T) as well as of their upstream kinase inhibitor, namely inhibitor of κ B kinases (IKK), in different human cancers, assessed by database mining.  These data argue against a universal mechanism of cancer-mediated activation of NF-κ B, and suggest a much more elaborated mode of NF-κ B regulation, indicating a tumor type-specific upregulation of the NF-κ B subunits.  We further discuss recent findings showing the diverse roles of NF-κ B signaling in cancer development and metastasis in a subunit-specific manner, emphasizing their specific transcriptional activity and the role of autoregulation.  While non-canonical NF-κ B RELB signaling is described to be mostly present in hematol. cancers, solid cancers reveal constitutive canonical NF-κ B RELA or c-REL activity.  Providing a linkage to cancer therapy, we discuss the recently described pivotal role of NF-κ B c-REL in regulating cancer-targeting immune responses.  In addn., current strategies and ongoing clin. trials are summarized, which utilize genome editing or drugs to inhibit the NF-κ FB subunits for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrjn-IQVNxD7Vg90H21EOLACvtfcHk0lh1dmTH0uVLSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvFaju7c%253D&md5=53c399ab25acd3881d6c274f80e27dda</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.3390%2Fbiomedicines6020044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiomedicines6020044%26sid%3Dliteratum%253Aachs%26aulast%3DKaltschmidt%26aufirst%3DB.%26aulast%3DGreiner%26aufirst%3DJ.%26aulast%3DKadhim%26aufirst%3DH.%26aulast%3DKaltschmidt%26aufirst%3DC.%26atitle%3DSubunit-Specific%2520Role%2520of%2520NF-%25CE%25BAB%2520in%2520Cancer%26jtitle%3DBiomedicines%26date%3D2018%26volume%3D6%26spage%3D44%26doi%3D10.3390%2Fbiomedicines6020044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, D. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, C.</span></span> <span> </span><span class="NLM_article-title">14-3-3 Isoforms Differentially Regulate NFκB Signaling in the Brain After Ischemia-Reperfusion</span>. <i>Neurochem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">2354</span>– <span class="NLM_lpage">2362</span>, <span class="refDoi"> DOI: 10.1007/s11064-017-2255-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1007%2Fs11064-017-2255-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=28424948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtlalsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2017&pages=2354-2362&author=X.+Zhouauthor=D.+X.+Huauthor=R.+Q.+Chenauthor=X.+Q.+Chenauthor=W.+Dongauthor=C.+Yi&title=14-3-3+Isoforms+Differentially+Regulate+NF%CE%BAB+Signaling+in+the+Brain+After+Ischemia-Reperfusion&doi=10.1007%2Fs11064-017-2255-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">14-3-3 Isoforms Differentially Regulate NFκB Signaling in the Brain After Ischemia-Reperfusion</span></div><div class="casAuthors">Zhou, Xin-yu; Hu, Dian Xing; Chen, Ruo Qiao; Chen, Xiao Qian; Dong, Wan-li; Yi, Cheng-la</div><div class="citationInfo"><span class="NLM_cas:title">Neurochemical Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2354-2362</span>CODEN:
                <span class="NLM_cas:coden">NEREDZ</span>;
        ISSN:<span class="NLM_cas:issn">0364-3190</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Mammalian 14-3-3 isoforms exist predominantly in the brain and are heavily involved in neurol. diseases.  However, the isoform-specific role of 14-3-3 proteins in the brain remains largely unclear.  Here, we investigated the role of 14-3-3 isoforms in rat brains after transient middle cerebral artery occlusion and reperfusion.  14-3-3β, η, γ And ζ but not ε or τ were selectively upregulated in cerebral cortical neurons after ischemia-reperfusion (I/R).  Selectively, 14-3-3β, γ and ζ were translocated from cytoplasm into the nuclei of neurons after I/R.  14-3-3 Bound to p65 and suppressed p65 expression in N2a cells.  In the brain, 14-3-3 could either colocalize with p65 in the nuclei of neurons or segregate from p65 expression in cortical neurons after I/R.  All evidence together suggests that 14-3-3 isoforms are differentially induced to enter into the nuclei of neurons after I/R, which might regulate NFκB signaling directly or indirectly.  Since 14-3-3 proteins are essential for cell survival and NFκB is a key transcriptional factor, our data suggest that the 14-3-3/p65 signaling pathway might be a potential therapeutic target for stroke.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSZBOBLj9SuLVg90H21EOLACvtfcHk0lh1dmTH0uVLSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtlalsLk%253D&md5=d6f800fa4dcef5b1a776832138605862</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1007%2Fs11064-017-2255-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11064-017-2255-3%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DD.%2BX.%26aulast%3DChen%26aufirst%3DR.%2BQ.%26aulast%3DChen%26aufirst%3DX.%2BQ.%26aulast%3DDong%26aufirst%3DW.%26aulast%3DYi%26aufirst%3DC.%26atitle%3D14-3-3%2520Isoforms%2520Differentially%2520Regulate%2520NF%25CE%25BAB%2520Signaling%2520in%2520the%2520Brain%2520After%2520Ischemia-Reperfusion%26jtitle%3DNeurochem.%2520Res.%26date%3D2017%26volume%3D42%26spage%3D2354%26epage%3D2362%26doi%3D10.1007%2Fs11064-017-2255-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Inglés-Esteve, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalmases, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Carbonell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jené-Sanz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Bigas, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iglesias, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Herguido, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rovira, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albanell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomis, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espinosa, L.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Specific NF-κB Activity Contributes to the Tumor Suppressor Function of 14-3-3σ in Breast Cancer</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e38347</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0038347</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1371%2Fjournal.pone.0038347" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=22675457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC38XosVenurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&author=J.+Ingl%C3%A9s-Esteveauthor=M.+Moralesauthor=A.+Dalmasesauthor=R.+Garcia-Carbonellauthor=A.+Jen%C3%A9-Sanzauthor=N.+L%C3%B3pez-Bigasauthor=M.+Iglesiasauthor=C.+Ruiz-Herguidoauthor=A.+Roviraauthor=F.+Rojoauthor=J.+Albanellauthor=R.+R.+Gomisauthor=A.+Bigasauthor=L.+Espinosa&title=Inhibition+of+Specific+NF-%CE%BAB+Activity+Contributes+to+the+Tumor+Suppressor+Function+of+14-3-3%CF%83+in+Breast+Cancer&doi=10.1371%2Fjournal.pone.0038347"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of specific NF-κB activity contributes to the tumor suppressor function of 14-3-3σ in breast cancer</span></div><div class="casAuthors">Ingles-Esteve, Julia; Morales, Monica; Dalmases, Alba; Garcia-Carbonell, Ricard; Jene-Sanz, Alba; Lopez-Bigas, Nuria; Iglesias, Mar; Ruiz-Herguido, Cristina; Rovira, Ana; Rojo, Federico; Albanell, Joan; Gomis, Roger R.; Bigas, Anna; Espinosa, Lluis</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e38347</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">14-3-3σ Is frequently lost in human breast cancers by genetic deletion or promoter methylation.  We have now investigated the involvement of 14-3-3σ in the termination of NF-κB signal in mammary cells and its putative role in cancer relapse and metastasis.  Our results show that 14-3-3σ regulates nuclear export of p65-NF-κB following chronic TNFα stimulation.  Restoration of 14-3-3σ in breast cancer cells reduces migration capacity and metastatic abilities in vivo.  By microarray anal., we have identified a genetic signature that responds to TNFα in a 14-3-3σ-dependent manner and significantly assocs. with different breast and other types of cancer.  By interrogating public databases, we have found that over-expression of this signature correlates with poor relapse-free survival in breast cancer patients.  Finally, screening of 96 human breast tumors showed that NF-κB activation strictly correlates with the absence of 14-3-3σ and it is significantly assocd. with worse prognosis in the multivariate anal.  Our findings identify a genetic signature that is important for breast cancer prognosis and for future personalized treatments based on NF-κB targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIDCBuaY4ofrVg90H21EOLACvtfcHk0lhJ4lP3teD_7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XosVenurw%253D&md5=ce325cadb08b4f675e78804dd458777d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0038347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0038347%26sid%3Dliteratum%253Aachs%26aulast%3DIngl%25C3%25A9s-Esteve%26aufirst%3DJ.%26aulast%3DMorales%26aufirst%3DM.%26aulast%3DDalmases%26aufirst%3DA.%26aulast%3DGarcia-Carbonell%26aufirst%3DR.%26aulast%3DJen%25C3%25A9-Sanz%26aufirst%3DA.%26aulast%3DL%25C3%25B3pez-Bigas%26aufirst%3DN.%26aulast%3DIglesias%26aufirst%3DM.%26aulast%3DRuiz-Herguido%26aufirst%3DC.%26aulast%3DRovira%26aufirst%3DA.%26aulast%3DRojo%26aufirst%3DF.%26aulast%3DAlbanell%26aufirst%3DJ.%26aulast%3DGomis%26aufirst%3DR.%2BR.%26aulast%3DBigas%26aufirst%3DA.%26aulast%3DEspinosa%26aufirst%3DL.%26atitle%3DInhibition%2520of%2520Specific%2520NF-%25CE%25BAB%2520Activity%2520Contributes%2520to%2520the%2520Tumor%2520Suppressor%2520Function%2520of%252014-3-3%25CF%2583%2520in%2520Breast%2520Cancer%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26doi%3D10.1371%2Fjournal.pone.0038347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aguilera, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Majada, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inglés-Esteve, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodilla, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espinosa, L.</span></span> <span> </span><span class="NLM_article-title">Efficient Nuclear Export of P65-IκBα Complexes Requires 14-3-3 Proteins</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">2472</span>– <span class="NLM_lpage">2472</span>, <span class="refDoi"> DOI: 10.1242/jcs.192641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1242%2Fjcs.192641" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=27307493" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslaqtbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2016&pages=2472-2472&author=C.+Aguileraauthor=V.+Fern%C3%A1ndez-Majadaauthor=J.+Ingl%C3%A9s-Esteveauthor=V.+Rodillaauthor=A.+Bigasauthor=L.+Espinosa&title=Efficient+Nuclear+Export+of+P65-I%CE%BAB%CE%B1+Complexes+Requires+14-3-3+Proteins&doi=10.1242%2Fjcs.192641"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient nuclear export of p65-IκBα complexes requires 14-3-3 proteins [Erratum to document cited in CA146:021884]</span></div><div class="casAuthors">Aguilera, Cristina; Fernandez-Majada, Vanessa; Ingles-Esteve, Julia; Rodilla, Veronica; Bigas, Anna; Espinosa, Lluis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2472</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">In the original publication, the GST-14-3-3 Coormassie stain shown in figure 2b is inadvertently duplicated from that shown in Figure 2a and the GST coomassie stains in figure 2c are also duplicated; the corrections are provided here.  In the original publication, the annotation of the reordering of the GST-BD42 and GST-BD42A samples is omitted; the correction is provided here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhcDkXJT1Pt7Vg90H21EOLACvtfcHk0lhJ4lP3teD_7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslaqtbbP&md5=667196cfeb49a0e74999ad2c27a24ac7</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1242%2Fjcs.192641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.192641%26sid%3Dliteratum%253Aachs%26aulast%3DAguilera%26aufirst%3DC.%26aulast%3DFern%25C3%25A1ndez-Majada%26aufirst%3DV.%26aulast%3DIngl%25C3%25A9s-Esteve%26aufirst%3DJ.%26aulast%3DRodilla%26aufirst%3DV.%26aulast%3DBigas%26aufirst%3DA.%26aulast%3DEspinosa%26aufirst%3DL.%26atitle%3DEfficient%2520Nuclear%2520Export%2520of%2520P65-I%25CE%25BAB%25CE%25B1%2520Complexes%2520Requires%252014-3-3%2520Proteins%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2016%26volume%3D129%26spage%3D2472%26epage%3D2472%26doi%3D10.1242%2Fjcs.192641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vink, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neves, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srdanović, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landrieu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Selectivity via Cooperativity: Preferential Stabilization of the P65/14-3-3 Interaction with Semisynthetic Natural Products</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">11772</span>– <span class="NLM_lpage">11783</span>, <span class="refDoi"> DOI: 10.1021/jacs.0c02151</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.0c02151" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFSltrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2020&pages=11772-11783&author=M.+Wolterauthor=P.+de+Vinkauthor=J.+F.+Nevesauthor=S.+Srdanovi%C4%87author=Y.+Higuchiauthor=N.+Katoauthor=A.+Wilsonauthor=I.+Landrieuauthor=L.+Brunsveldauthor=C.+Ottmann&title=Selectivity+via+Cooperativity%3A+Preferential+Stabilization+of+the+P65%2F14-3-3+Interaction+with+Semisynthetic+Natural+Products&doi=10.1021%2Fjacs.0c02151"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Selectivity via Cooperativity: Preferential Stabilization of the p65/14-3-3 Interaction with Semisynthetic Natural Products</span></div><div class="casAuthors">Wolter, Madita; de Vink, Pim; Neves, Joao Filipe; Srdanovic, Sonja; Higuchi, Yusuke; Kato, Nobuo; Wilson, Andrew; Landrieu, Isabelle; Brunsveld, Luc; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">11772-11783</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Natural compds. are an important class of potent drug mols. including some retrospectively found to act as stabilizers of protein-protein interactions (PPIs).  However, the design of synthetic PPI stabilizers remains an understudied approach.  To date, there are limited examples where cooperativity has been utilized to guide the optimization of a PPI stabilizer.  The 14-3-3 scaffold proteins provide an excellent platform to explore PPI stabilization because these proteins mediate several hundred PPIs, and a class of natural compds., the fusicoccanes, are known to stabilize a subset of 14-3-3 protein interactions. 14-3-3 has been reported to neg. regulate the p65 subunit of the NF-κB transcription factor, which qualifies this protein complex as a potential target for drug discovery to control cell proliferation.  Here, we report the high-resoln. crystal structures of two 14-3-3 binding motifs of p65 in complex with 14-3-3.  A semisynthetic natural product deriv., DP-005, binds to an interface pocket of the p65/14-3-3 complex and concomitantly stabilizes it.  Cooperativity analyses of this interaction, and other disease relevant 14-3-3-PPIs, demonstrated selectivity of DP-005 for the p65/14-3-3 complex.  The adaptation of a cooperative binding model provided a general approach to characterize stabilization and to assay for selectivity of PPI stabilizers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpd9M4XEOVeZLVg90H21EOLACvtfcHk0ljrd7N90jPtog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFSltrbJ&md5=ffbb418ca8ce137467486c725eca0eaa</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjacs.0c02151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.0c02151%26sid%3Dliteratum%253Aachs%26aulast%3DWolter%26aufirst%3DM.%26aulast%3Dde%2BVink%26aufirst%3DP.%26aulast%3DNeves%26aufirst%3DJ.%2BF.%26aulast%3DSrdanovi%25C4%2587%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DY.%26aulast%3DKato%26aufirst%3DN.%26aulast%3DWilson%26aufirst%3DA.%26aulast%3DLandrieu%26aufirst%3DI.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DSelectivity%2520via%2520Cooperativity%253A%2520Preferential%2520Stabilization%2520of%2520the%2520P65%252F14-3-3%2520Interaction%2520with%2520Semisynthetic%2520Natural%2520Products%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2020%26volume%3D142%26spage%3D11772%26epage%3D11783%26doi%3D10.1021%2Fjacs.0c02151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huxford, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, G.</span></span> <span> </span><span class="NLM_article-title">The Crystal Structure of the IκBα/NF-κB Complex Reveals Mechanisms of NF-κB Inactivation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">759</span>– <span class="NLM_lpage">770</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(00)81699-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1016%2FS0092-8674%2800%2981699-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=9865694" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADyaK1MXivVGr" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=759-770&author=T.+Huxfordauthor=D.-B.+Huangauthor=S.+Malekauthor=G.+Ghosh&title=The+Crystal+Structure+of+the+I%CE%BAB%CE%B1%2FNF-%CE%BAB+Complex+Reveals+Mechanisms+of+NF-%CE%BAB+Inactivation&doi=10.1016%2FS0092-8674%2800%2981699-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The crystal structure of the IκBα/NF-κB complex reveals mechanisms of NF-κB inactivation</span></div><div class="casAuthors">Huxford, Tom; Huang, De-Bin; Malek, Shiva; Ghosh, Gourisankar</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">759-770</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">IκBα regulates the transcription factor NF-κB through the formation of stable IκBα/NF-κB complexes.  Prior to induction, IκBα retains NF-κB in the cytoplasm until the NF-κB activation signal is received.  After activation, NF-κB is removed from gene promoters through assocn. with nuclear IκBα, restoring the preinduction state.  The 2.3 Å crystal structure of IκBα in complex with the NF-κB p50/p65 heterodimer reveals mechanisms of these inhibitory activities.  The presence of IκBα allows large en bloc movement of the NF-κB p65 subunit amino-terminal domain.  This conformational change induces allosteric inhibition of NF-κB DNA binding.  Amino acid residues immediately preceding the nuclear localization signals of both NF-κB p50 and p65 subunits are tethered to the IκBα amino-terminal ankyrin repeats, impeding NF-κB from nuclear import machinery recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkLfVoA9yRbrVg90H21EOLACvtfcHk0ljrd7N90jPtog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXivVGr&md5=d9f6b8e5bd56fa9f19299c5eb08c0834</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981699-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981699-2%26sid%3Dliteratum%253Aachs%26aulast%3DHuxford%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DD.-B.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DGhosh%26aufirst%3DG.%26atitle%3DThe%2520Crystal%2520Structure%2520of%2520the%2520I%25CE%25BAB%25CE%25B1%252FNF-%25CE%25BAB%2520Complex%2520Reveals%2520Mechanisms%2520of%2520NF-%25CE%25BAB%2520Inactivation%26jtitle%3DCell%26date%3D1998%26volume%3D95%26spage%3D759%26epage%3D770%26doi%3D10.1016%2FS0092-8674%2800%2981699-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brameld, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reuter, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Conformational Preferences Derived from Crystal Structure Data. A Medicinal Chemistry Focused Analysis</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1021/ci7002494</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci7002494" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BD1cXksV2isA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=1-24&author=K.+A.+Brameldauthor=B.+Kuhnauthor=D.+C.+Reuterauthor=M.+Stahl&title=Small+Molecule+Conformational+Preferences+Derived+from+Crystal+Structure+Data.+A+Medicinal+Chemistry+Focused+Analysis&doi=10.1021%2Fci7002494"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule conformational preferences derived from crystal structure data. a medicinal chemistry focused analysis</span></div><div class="casAuthors">Brameld, Ken A.; Kuhn, Bernd; Reuter, Deborah C.; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-24</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Based on torsion angle distributions of frequently occurring substructures, conformation preferences of druglike mols. are presented, accompanied by a review of the relevant literature.  First, the relevance of the Cambridge Structural Database (CSD) for drug design is demonstrated by comparing substructures present in compds. entering clin. trials with those found in the CSD and protein-bound ligands in the Protein Data Bank (PDB).  Next, we briefly highlight preferred conformations of elementary acyclic systems, followed by a discussion of sulfonamide conformations.  Due to their central role in medicinal chem., we discuss properties of aryl ring substituents in depth, including biaryl systems and systems of two aryl rings connected by two acyclic bonds.  For a subset of torsion motifs, we also compare torsion angle histograms derived from CSD structures with those derived from ligands in the PDB.  Furthermore, selected properties of some six- and seven-membered ring systems are discussed.  The article closes with a section on attractive sulfur-oxygen contacts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBrdvMrRvKirVg90H21EOLACvtfcHk0ljwuBSIFt8ZsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXksV2isA%253D%253D&md5=289969338dcb94a145ce736d19d05cf4</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fci7002494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci7002494%26sid%3Dliteratum%253Aachs%26aulast%3DBrameld%26aufirst%3DK.%2BA.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DReuter%26aufirst%3DD.%2BC.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DSmall%2520Molecule%2520Conformational%2520Preferences%2520Derived%2520from%2520Crystal%2520Structure%2520Data.%2520A%2520Medicinal%2520Chemistry%2520Focused%2520Analysis%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D1%26epage%3D24%26doi%3D10.1021%2Fci7002494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oksenberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva-Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchaleelaha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patskovska, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patskovsky, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almo, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinha, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalf, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Archer, D. R.</span></span> <span> </span><span class="NLM_article-title">GBT440 Increases Haemoglobin Oxygen Affinity, Reduces Sickling and Prolongs RBC Half-Life in a Murine Model of Sickle Cell Disease</span>. <i>Br. J. Haematol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>175</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">153</span>, <span class="refDoi"> DOI: 10.1111/bjh.14214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1111%2Fbjh.14214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=27378309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFCkurfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2016&pages=141-153&author=D.+Oksenbergauthor=K.+Dufuauthor=M.+P.+Patelauthor=C.+Chuangauthor=Z.+Liauthor=Q.+Xuauthor=A.+Silva-Garciaauthor=C.+Zhouauthor=A.+Hutchaleelahaauthor=L.+Patskovskaauthor=Y.+Patskovskyauthor=S.+C.+Almoauthor=U.+Sinhaauthor=B.+W.+Metcalfauthor=D.+R.+Archer&title=GBT440+Increases+Haemoglobin+Oxygen+Affinity%2C+Reduces+Sickling+and+Prolongs+RBC+Half-Life+in+a+Murine+Model+of+Sickle+Cell+Disease&doi=10.1111%2Fbjh.14214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">GBT440 increases haemoglobin oxygen affinity, reduces sickling and prolongs RBC half-life in a murine model of sickle cell disease</span></div><div class="casAuthors">Oksenberg, Donna; Dufu, Kobina; Patel, Mira P.; Chuang, Chihyuan; Li, Zhe; Xu, Qing; Silva-Garcia, Abel; Zhou, Chengjing; Hutchaleelaha, Athiwat; Patskovska, Larysa; Patskovsky, Yury; Almo, Steven C.; Sinha, Uma; Metcalf, Brian W.; Archer, David R.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Haematology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">141-153</span>CODEN:
                <span class="NLM_cas:coden">BJHEAL</span>;
        ISSN:<span class="NLM_cas:issn">0007-1048</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Summary : A major driver of the pathophysiol. of sickle cell disease (SCD) is polymn. of deoxygenated Hb S (HbS), which leads to sickling and destruction of red blood cells (RBCs) and end-organ damage.  Pharmacol. increasing the proportion of oxygenated HbS in RBCs may inhibit polymn., prevent sickling and provide long term disease modification.  We report that GBT440, a small mol. which binds to the N-terminal α chain of Hb, increases HbS affinity for oxygen, delays in vitro HbS polymn. and prevents sickling of RBCs.  Moreover, in a murine model of SCD, GBT440 extends the half-life of RBCs, reduces reticulocyte counts and prevents ex vivoRBC sickling.  Importantly, oral dosing of GBT440 in animals demonstrates suitability for once daily dosing in humans and a highly selective partitioning into RBCs, which is a key therapeutic safety attribute.  Thus, GBT440 has the potential for clin. use as a disease-modifying agent in sickle cell patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9twqbw0I4ELVg90H21EOLACvtfcHk0ljwuBSIFt8ZsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFCkurfP&md5=07276e0b19eac3cb2baef00c872c9197</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1111%2Fbjh.14214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbjh.14214%26sid%3Dliteratum%253Aachs%26aulast%3DOksenberg%26aufirst%3DD.%26aulast%3DDufu%26aufirst%3DK.%26aulast%3DPatel%26aufirst%3DM.%2BP.%26aulast%3DChuang%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DSilva-Garcia%26aufirst%3DA.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DHutchaleelaha%26aufirst%3DA.%26aulast%3DPatskovska%26aufirst%3DL.%26aulast%3DPatskovsky%26aufirst%3DY.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DSinha%26aufirst%3DU.%26aulast%3DMetcalf%26aufirst%3DB.%2BW.%26aulast%3DArcher%26aufirst%3DD.%2BR.%26atitle%3DGBT440%2520Increases%2520Haemoglobin%2520Oxygen%2520Affinity%252C%2520Reduces%2520Sickling%2520and%2520Prolongs%2520RBC%2520Half-Life%2520in%2520a%2520Murine%2520Model%2520of%2520Sickle%2520Cell%2520Disease%26jtitle%3DBr.%2520J.%2520Haematol.%26date%3D2016%26volume%3D175%26spage%3D141%26epage%3D153%26doi%3D10.1111%2Fbjh.14214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Differential PROTAC Substrate Specificity Dictated by Orientation of Recruited E3 Ligase</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1038/s41467-018-08027-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fs41467-018-08027-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=30602773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlSru7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1-13&author=B.+E.+Smithauthor=S.+L.+Wangauthor=S.+Jaime-Figueroaauthor=A.+Harbinauthor=J.+Wangauthor=B.+D.+Hammanauthor=C.+M.+Crews&title=Differential+PROTAC+Substrate+Specificity+Dictated+by+Orientation+of+Recruited+E3+Ligase&doi=10.1038%2Fs41467-018-08027-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Myeloid lineage enhancers drive oncogene synergy in CEBPA/CSF3R mutant acute myeloid leukemia</span></div><div class="casAuthors">Braun, Theodore P.; Okhovat, Mariam; Coblentz, Cody; Carratt, Sarah A.; Foley, Amy; Schonrock, Zachary; Smith, Brittany M.; Nevonen, Kimberly; Davis, Brett; Garcia, Brianna; LaTocha, Dorian; Weeder, Benjamin R.; Grzadkowski, Michal R.; Estabrook, Joey C.; Manning, Hannah G.; Watanabe-Smith, Kevin; Jeng, Sophia; Smith, Jenny L.; Leonti, Amanda R.; Ries, Rhonda E.; McWeeney, Shannon; Di Genua, Cristina; Drissen, Roy; Nerlov, Claus; Meshinchi, Soheil; Carbone, Lucia; Druker, Brian J.; Maxson, Julia E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-15</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Acute Myeloid Leukemia (AML) develops due to the acquisition of mutations from multiple functional classes.  Here, we demonstrate that activating mutations in the granulocyte colony stimulating factor receptor (CSF3R), cooperate with loss of function mutations in the transcription factor CEBPA to promote acute leukemia development.  The interaction between these distinct classes of mutations occurs at the level of myeloid lineage enhancers where mutant CEBPA prevents activation of a subset of differentiation assocd. enhancers.  To confirm this enhancer-dependent mechanism, we demonstrate that CEBPA mutations must occur as the initial event in AML initiation.  This improved mechanistic understanding will facilitate therapeutic development targeting the intersection of oncogene cooperativity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonOzBTD_nh57Vg90H21EOLACvtfcHk0lgXzH03gV_Vvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlSru7jE&md5=edb5560e8b3b0da2368fa6e527b0feb0</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-08027-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-08027-7%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DWang%26aufirst%3DS.%2BL.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DHarbin%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DDifferential%2520PROTAC%2520Substrate%2520Specificity%2520Dictated%2520by%2520Orientation%2520of%2520Recruited%2520E3%2520Ligase%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D1%26epage%3D13%26doi%3D10.1038%2Fs41467-018-08027-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zoppi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maniaci, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieshofer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riching, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spallarossa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Iterative Design and Optimization of Initially Inactive Proteolysis Targeting Chimeras (PROTACs) Identify VZ185 as a Potent, Fast, and Selective von Hippel-Lindau (VHL) Based Dual Degrader Probe of BRD9 and BRD7</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">699</span>– <span class="NLM_lpage">726</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01413</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01413" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVyhsLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=699-726&author=V.+Zoppiauthor=S.+J.+Hughesauthor=C.+Maniaciauthor=A.+Testaauthor=T.+Gmaschitzauthor=C.+Wieshoferauthor=M.+Koeglauthor=K.+M.+Richingauthor=D.+L.+Danielsauthor=A.+Spallarossaauthor=A.+Ciulli&title=Iterative+Design+and+Optimization+of+Initially+Inactive+Proteolysis+Targeting+Chimeras+%28PROTACs%29+Identify+VZ185+as+a+Potent%2C+Fast%2C+and+Selective+von+Hippel-Lindau+%28VHL%29+Based+Dual+Degrader+Probe+of+BRD9+and+BRD7&doi=10.1021%2Facs.jmedchem.8b01413"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Iterative Design and Optimization of Initially Inactive Proteolysis Targeting Chimeras (PROTACs) Identify VZ185 as a Potent, Fast, and Selective von Hippel-Lindau (VHL) Based Dual Degrader Probe of BRD9 and BRD7</span></div><div class="casAuthors">Zoppi, Vittoria; Hughes, Scott J.; Maniaci, Chiara; Testa, Andrea; Gmaschitz, Teresa; Wieshofer, Corinna; Koegl, Manfred; Riching, Kristin M.; Daniels, Danette L.; Spallarossa, Andrea; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">699-726</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Developing PROTACs to redirect the ubiquitination activity of E3 ligases and potently degrade a target protein within cells can be a lengthy and unpredictable process, and it remains unclear whether any combination of E3 and target might be productive for degrdn.  The authors describe a probe-quality degrader for a ligase-target pair deemed unsuitable: the von Hippel-Lindau (VHL) and BRD9, a bromodomain-contg. subunit of the SWI/SNF chromatin remodeling complex BAF.  VHL-based degraders could be optimized from suboptimal compds. in two rounds by systematically varying conjugation patterns and linkers and monitoring cellular degrdn. activities, kinetic profiles, and ubiquitination, as well as ternary complex formation thermodn.  The emerged structure-activity relations guided the discovery of VZ185, a potent, fast, and selective degrader of BRD9 and of its close homolog BRD7.  The authors' findings qualify a new chem. tool for BRD7/9 knockdown and provide a roadmap for PROTAC development against seemingly incompatible target-ligase combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiPnajotKvqLVg90H21EOLACvtfcHk0lgXzH03gV_Vvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVyhsLjN&md5=be7938c26807a2535b7f9b7de8ebfbb0</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01413%26sid%3Dliteratum%253Aachs%26aulast%3DZoppi%26aufirst%3DV.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DManiaci%26aufirst%3DC.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DWieshofer%26aufirst%3DC.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DRiching%26aufirst%3DK.%2BM.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DSpallarossa%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DIterative%2520Design%2520and%2520Optimization%2520of%2520Initially%2520Inactive%2520Proteolysis%2520Targeting%2520Chimeras%2520%2528PROTACs%2529%2520Identify%2520VZ185%2520as%2520a%2520Potent%252C%2520Fast%252C%2520and%2520Selective%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520Based%2520Dual%2520Degrader%2520Probe%2520of%2520BRD9%2520and%2520BRD7%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D699%26epage%3D726%26doi%3D10.1021%2Facs.jmedchem.8b01413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zorba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borzilleri, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farley, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiemer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Irrizary, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czabaniuk, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuff, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niosi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luthra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Kattan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span> <span> </span><span class="NLM_article-title">Delineating the Role of Cooperativity in the Design of Potent PROTACs for BTK</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">E7285</span>– <span class="NLM_lpage">E7292</span>, <span class="refDoi"> DOI: 10.1073/pnas.1803662115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1073%2Fpnas.1803662115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=30012605" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=E7285-E7292&author=A.+Zorbaauthor=C.+Nguyenauthor=Y.+Xuauthor=J.+Starrauthor=K.+Borzilleriauthor=J.+Smithauthor=H.+Zhuauthor=K.+A.+Farleyauthor=W.+D.+Dingauthor=J.+Schiemerauthor=X.+Fengauthor=J.+S.+Changauthor=D.+P.+Uccelloauthor=J.+A.+Youngauthor=C.+N.+Garcia-Irrizaryauthor=L.+Czabaniukauthor=B.+Schuffauthor=R.+Oliverauthor=J.+Montgomeryauthor=M.+M.+Haywardauthor=J.+Coeauthor=J.+Chenauthor=M.+Niosiauthor=S.+Luthraauthor=J.+C.+Shahauthor=A.+El-Kattanauthor=X.+Qiuauthor=G.+M.+Westauthor=M.+C.+Noeauthor=V.+Shanmugasundaramauthor=A.+M.+Gilbertauthor=M.+F.+Brownauthor=M.+F.+Calabrese&title=Delineating+the+Role+of+Cooperativity+in+the+Design+of+Potent+PROTACs+for+BTK&doi=10.1073%2Fpnas.1803662115"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1803662115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1803662115%26sid%3Dliteratum%253Aachs%26aulast%3DZorba%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DStarr%26aufirst%3DJ.%26aulast%3DBorzilleri%26aufirst%3DK.%26aulast%3DSmith%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DFarley%26aufirst%3DK.%2BA.%26aulast%3DDing%26aufirst%3DW.%2BD.%26aulast%3DSchiemer%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DChang%26aufirst%3DJ.%2BS.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%26aulast%3DGarcia-Irrizary%26aufirst%3DC.%2BN.%26aulast%3DCzabaniuk%26aufirst%3DL.%26aulast%3DSchuff%26aufirst%3DB.%26aulast%3DOliver%26aufirst%3DR.%26aulast%3DMontgomery%26aufirst%3DJ.%26aulast%3DHayward%26aufirst%3DM.%2BM.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DNiosi%26aufirst%3DM.%26aulast%3DLuthra%26aufirst%3DS.%26aulast%3DShah%26aufirst%3DJ.%2BC.%26aulast%3DEl-Kattan%26aufirst%3DA.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DWest%26aufirst%3DG.%2BM.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26atitle%3DDelineating%2520the%2520Role%2520of%2520Cooperativity%2520in%2520the%2520Design%2520of%2520Potent%2520PROTACs%2520for%2520BTK%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2018%26volume%3D115%26spage%3DE7285%26epage%3DE7292%26doi%3D10.1073%2Fpnas.1803662115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farnaby, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">SPR-Measured Dissociation Kinetics of PROTAC Ternary Complexes Influence Target Degradation Rate</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">368</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00092</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00092" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVCisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=361-368&author=M.+J.+Royauthor=S.+Winklerauthor=S.+J.+Hughesauthor=C.+Whitworthauthor=M.+Galantauthor=W.+Farnabyauthor=K.+Rumpelauthor=A.+Ciulli&title=SPR-Measured+Dissociation+Kinetics+of+PROTAC+Ternary+Complexes+Influence+Target+Degradation+Rate&doi=10.1021%2Facschembio.9b00092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">SPR-Measured Dissociation Kinetics of PROTAC Ternary Complexes Influence Target Degradation Rate</span></div><div class="casAuthors">Roy, Michael J.; Winkler, Sandra; Hughes, Scott J.; Whitworth, Claire; Galant, Michael; Farnaby, William; Rumpel, Klaus; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">361-368</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bifunctional degrader mols., known as proteolysis-targeting chimeras (PROTACs), function by recruiting a target to an E3 ligase, forming a target/PROTAC/ligase ternary complex.  Despite the importance of this key intermediate species, no detailed validation of a method to directly det. binding parameters for ternary complex kinetics has been reported, and it remains to be addressed whether tuning the kinetics of PROTAC ternary complexes may be an effective strategy to improve the efficiency of targeted protein degrdn.  Here, we develop an SPR-based assay to quantify the stability of PROTAC-induced ternary complexes by measuring for the first time the kinetics of their formation and dissocn. in vitro using purified proteins.  We benchmark our assay using four PROTACs that target the bromodomains (BDs) of bromodomain and extraterminal domain proteins Brd2, Brd3, and Brd4 to the von Hippel-Lindau E3 ligase (VHL).  We reveal marked differences in ternary complex off-rates for different PROTACs that exhibit either pos. or neg. cooperativity for ternary complex formation relative to binary binding.  The pos. cooperative degrader MZ1 forms comparatively stable and long-lived ternary complexes with either Brd4BD2 or Brd2BD2 and VHL.  Equivalent complexes with Brd3BD2 are destabilized due to a single amino acid difference (Glu/Gly swap) present in the bromodomain.  We observe that this difference in ternary complex dissociative half-life correlates to a greater initial rate of intracellular degrdn. of Brd2 and Brd4 relative to Brd3.  These findings establish a novel assay to measure the kinetics of PROTAC ternary complexes and elucidate the important kinetic parameters that drive effective target degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzFhvuKYztcbVg90H21EOLACvtfcHk0lh-zan1iZqEsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVCisrg%253D&md5=629d1acbd218b668a2696b727946962b</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00092%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DM.%2BJ.%26aulast%3DWinkler%26aufirst%3DS.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DGalant%26aufirst%3DM.%26aulast%3DFarnaby%26aufirst%3DW.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSPR-Measured%2520Dissociation%2520Kinetics%2520of%2520PROTAC%2520Ternary%2520Complexes%2520Influence%2520Target%2520Degradation%2520Rate%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D361%26epage%3D368%26doi%3D10.1021%2Facschembio.9b00092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87.e5</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87.e5&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+Design+from+Selective+Degradation+with+a+Promiscuous+Warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0li1l4e1gatgjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520Design%2520from%2520Selective%2520Degradation%2520with%2520a%2520Promiscuous%2520Warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87.e5%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vink, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neves, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srdanovic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landrieu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Selectivity via Cooperativity: Preferential Stabilization of the P65/14-3-3 Interaction with Semi-Synthetic Natural Products</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">11772</span>– <span class="NLM_lpage">11783</span>, <span class="refDoi"> DOI: 10.1021/jacs.0c02151</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.0c02151" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFSltrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2020&pages=11772-11783&author=M.+Wolterauthor=P.+de+Vinkauthor=J.+F.+Nevesauthor=S.+Srdanovicauthor=Y.+Higuchiauthor=N.+Katoauthor=A.+J.+Wilsonauthor=I.+Landrieuauthor=L.+Brunsveldauthor=C.+Ottmann&title=Selectivity+via+Cooperativity%3A+Preferential+Stabilization+of+the+P65%2F14-3-3+Interaction+with+Semi-Synthetic+Natural+Products&doi=10.1021%2Fjacs.0c02151"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Selectivity via Cooperativity: Preferential Stabilization of the p65/14-3-3 Interaction with Semisynthetic Natural Products</span></div><div class="casAuthors">Wolter, Madita; de Vink, Pim; Neves, Joao Filipe; Srdanovic, Sonja; Higuchi, Yusuke; Kato, Nobuo; Wilson, Andrew; Landrieu, Isabelle; Brunsveld, Luc; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">11772-11783</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Natural compds. are an important class of potent drug mols. including some retrospectively found to act as stabilizers of protein-protein interactions (PPIs).  However, the design of synthetic PPI stabilizers remains an understudied approach.  To date, there are limited examples where cooperativity has been utilized to guide the optimization of a PPI stabilizer.  The 14-3-3 scaffold proteins provide an excellent platform to explore PPI stabilization because these proteins mediate several hundred PPIs, and a class of natural compds., the fusicoccanes, are known to stabilize a subset of 14-3-3 protein interactions. 14-3-3 has been reported to neg. regulate the p65 subunit of the NF-κB transcription factor, which qualifies this protein complex as a potential target for drug discovery to control cell proliferation.  Here, we report the high-resoln. crystal structures of two 14-3-3 binding motifs of p65 in complex with 14-3-3.  A semisynthetic natural product deriv., DP-005, binds to an interface pocket of the p65/14-3-3 complex and concomitantly stabilizes it.  Cooperativity analyses of this interaction, and other disease relevant 14-3-3-PPIs, demonstrated selectivity of DP-005 for the p65/14-3-3 complex.  The adaptation of a cooperative binding model provided a general approach to characterize stabilization and to assay for selectivity of PPI stabilizers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpd9M4XEOVeZLVg90H21EOLACvtfcHk0li1l4e1gatgjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFSltrbJ&md5=ffbb418ca8ce137467486c725eca0eaa</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjacs.0c02151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.0c02151%26sid%3Dliteratum%253Aachs%26aulast%3DWolter%26aufirst%3DM.%26aulast%3Dde%2BVink%26aufirst%3DP.%26aulast%3DNeves%26aufirst%3DJ.%2BF.%26aulast%3DSrdanovic%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DY.%26aulast%3DKato%26aufirst%3DN.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26aulast%3DLandrieu%26aufirst%3DI.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DSelectivity%2520via%2520Cooperativity%253A%2520Preferential%2520Stabilization%2520of%2520the%2520P65%252F14-3-3%2520Interaction%2520with%2520Semi-Synthetic%2520Natural%2520Products%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2020%26volume%3D142%26spage%3D11772%26epage%3D11783%26doi%3D10.1021%2Fjacs.0c02151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clabbers, M. T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruene, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkhurst, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrahams, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterman, D. G.</span></span> <span> </span><span class="NLM_article-title">Electron Diffraction Data Processing with DIALS</span>. <i>Acta Crystallogr. Sect. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">506</span>– <span class="NLM_lpage">518</span>, <span class="refDoi"> DOI: 10.1107/S2059798318007726</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1107%2FS2059798318007726" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtV2ltbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=506-518&author=M.+T.+B.+Clabbersauthor=T.+Grueneauthor=J.+M.+Parkhurstauthor=J.+P.+Abrahamsauthor=D.+G.+Waterman&title=Electron+Diffraction+Data+Processing+with+DIALS&doi=10.1107%2FS2059798318007726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Electron diffraction data processing with DIALS</span></div><div class="casAuthors">Clabbers, Max T. B.; Gruene, Tim; Parkhurst, James M.; Abrahams, Jan Pieter; Waterman, David G.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">506-518</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Electron diffraction is a relatively novel alternative to X-ray crystallog. for the structure detn. of macromols. from three-dimensional nanometer-sized crystals.  Nevertheless, the DIALS software package can, with specific adaptations, successfully process continuous-rotation electron diffraction data.  Pathologies encountered specifically in electron diffraction make data integration more challenging.  Errors can arise from instrumentation, such as beam drift or distorted diffraction patterns from lens imperfections.  The diffraction geometry brings addnl. challenges such as strong correlation between lattice parameters and detector distance.  These issues are compounded if calibration is incomplete, leading to uncertainty in exptl. geometry, such as the effective detector distance and the rotation rate or direction.  Dynamic scattering, absorption, radiation damage and incomplete wedges of data are addnl. factors that complicate data processing.  Here, recent features of DIALS as adapted to electron diffraction processing are shown, including diagnostics for problematic diffraction geometry refinement, refinement of a smoothly varying beam model and corrections for distorted diffraction images.  These novel features, combined with the existing tools in DIALS, make data integration and refinement feasible for electron crystallog., even in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ2xvp5nytBbVg90H21EOLACvtfcHk0liec8riCJzBgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtV2ltbzM&md5=0ef3d54b8435ed67f31774547a35d263</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS2059798318007726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798318007726%26sid%3Dliteratum%253Aachs%26aulast%3DClabbers%26aufirst%3DM.%2BT.%2BB.%26aulast%3DGruene%26aufirst%3DT.%26aulast%3DParkhurst%26aufirst%3DJ.%2BM.%26aulast%3DAbrahams%26aufirst%3DJ.%2BP.%26aulast%3DWaterman%26aufirst%3DD.%2BG.%26atitle%3DElectron%2520Diffraction%2520Data%2520Processing%2520with%2520DIALS%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520Struct.%2520Biol.%26date%3D2018%26volume%3D74%26spage%3D506%26epage%3D518%26doi%3D10.1107%2FS2059798318007726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">Model Preparation in MOLREP and Examples of Model Improvement Using X-Ray Data</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1107/S0907444907049839</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1107%2FS0907444907049839" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=18094465" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVGjt7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2008&pages=33-39&author=A.+A.+Lebedevauthor=A.+A.+Vaginauthor=G.+N.+Murshudov&title=Model+Preparation+in+MOLREP+and+Examples+of+Model+Improvement+Using+X-Ray+Data&doi=10.1107%2FS0907444907049839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Model preparation in MOLREP and examples of model improvement using X-ray data</span></div><div class="casAuthors">Lebedev, Andrey A.; Vagin, Alexei A.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-39</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The success of mol. replacement is critically dependent on the quality of the search model.  Several model-prepn. procedures are integrated in the mol.-replacement program MOLREP.  These include model modification on the basis of amino-acid sequence alignment and model correction based on anal. of the solvent-accessibility of the atoms.  The packing function used in MOLREP for the translational search is explained in the context of model prepn.  In difficult cases, bioinformatics-based modifications are not sufficient for successful mol. replacement.  An approach implemented in MOLREP for solving cases with translational noncrystallog. symmetry is an example of model prepn. in which anal. of x-ray data plays an essential role.  In addn., two examples are presented in which the x-ray data were used to refine partial models for subsequent use in mol. replacement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeCvDgb4tgK7Vg90H21EOLACvtfcHk0liec8riCJzBgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVGjt7nI&md5=8ccba5e3d8014210bed44fe310c4c33e</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0907444907049839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444907049839%26sid%3Dliteratum%253Aachs%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DModel%2520Preparation%2520in%2520MOLREP%2520and%2520Examples%2520of%2520Model%2520Improvement%2520Using%2520X-Ray%2520Data%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D2008%26volume%3D64%26spage%3D33%26epage%3D39%26doi%3D10.1107%2FS0907444907049839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">Molecular Replacement with MOLREP</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1107/S0907444909042589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1107%2FS0907444909042589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=20057045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=22-25&author=A.+Vaginauthor=A.+Teplyakov&title=Molecular+Replacement+with+MOLREP&doi=10.1107%2FS0907444909042589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular replacement with MOLREP</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-25</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement that utilizes a no. of original approaches to rotational and translational search and data prepn.  Since the first publication describing the program, MOLREP has acquired a variety of features that include weighting of the X-ray data and search models, multi-copy search, fitting the model into electron d., structural superposition of two models and rigid-body refinement.  The program can run in a fully automatic mode using optimized parameters calcd. from the input data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzj5SzYvyu_7Vg90H21EOLACvtfcHk0liec8riCJzBgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D&md5=820d114719aca209994ffb0403e3b20d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042589%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMolecular%2520Replacement%2520with%2520MOLREP%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D22%26epage%3D25%26doi%3D10.1107%2FS0907444909042589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: Model-Building Tools for Molecular Graphics</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-Building+Tools+for+Molecular+Graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lijUQXz8VHQwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-Building%2520Tools%2520for%2520Molecular%2520Graphics%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span> <span> </span><span class="NLM_article-title">REFMAC5 for the Refinement of Macromolecular Crystal Structures</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/S0907444911001314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1107%2FS0907444911001314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=21460454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=355-367&author=G.+N.+Murshudovauthor=P.+Skub%C3%A1kauthor=A.+A.+Lebedevauthor=N.+S.+Pannuauthor=R.+A.+Steinerauthor=R.+A.+Nichollsauthor=M.+D.+Winnauthor=F.+Longauthor=A.+A.+Vagin&title=REFMAC5+for+the+Refinement+of+Macromolecular+Crystal+Structures&doi=10.1107%2FS0907444911001314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 for the refinement of macromolecular crystal structures</span></div><div class="casAuthors">Murshudov, Garib N.; Skubak, Pavol; Lebedev, Andrey A.; Pannu, Navraj S.; Steiner, Roberto A.; Nicholls, Robert A.; Winn, Martyn D.; Long, Fei; Vagin, Alexei A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">This paper describes various components of the macromol. crystallog. refinement program REFMAC5, which is distributed as part of the CCP4 suite.  REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS exptl. diffraction data.  To ensure chem. and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization.  Reliable models at resolns. at least as low as 4 Å can be achieved thanks to low-resoln. refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on at. displacement parameters (ADPs) based on the Kullback-Leibler divergence.  REFMAC5 addnl. offers TLS parameterization and, when high-resoln. data are available, fast refinement of anisotropic ADPs.  Refinement in the presence of twinning is performed in a fully automated fashion.  REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resoln. spectrum encountered in macromol. crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Jq-i2qIq5rVg90H21EOLACvtfcHk0lijUQXz8VHQwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D&md5=f8f3202d246908500057ad7c71015b7b</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444911001314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911001314%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DSkub%25C3%25A1k%26aufirst%3DP.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26atitle%3DREFMAC5%2520for%2520the%2520Refinement%2520of%2520Macromolecular%2520Crystal%2520Structures%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D355%26epage%3D367%26doi%3D10.1107%2FS0907444911001314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title"><i>PHENIX</i> : A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX+%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lijUQXz8VHQwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%2520%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doveston, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuusk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrei, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leysen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castaldi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendricks, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Stabilization of the P53 – 14-3-3 Protein-Protein Interaction</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>591</i></span>,  <span class="NLM_fpage">2449</span>– <span class="NLM_lpage">2457</span>, <span class="refDoi"> DOI: 10.1002/1873-3468.12723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1002%2F1873-3468.12723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=28640363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1yitLnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=591&publication_year=2017&pages=2449-2457&author=R.+G.+Dovestonauthor=A.+Kuuskauthor=S.+Andreiauthor=S.+Leysenauthor=Q.+Caoauthor=P.+Castaldiauthor=A.+Hendricksauthor=H.+Chenauthor=H.+Boydauthor=C.+Ottmann&title=Small-Molecule+Stabilization+of+the+P53+%E2%80%93+14-3-3+Protein-Protein+Interaction&doi=10.1002%2F1873-3468.12723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule stabilization of the p53-14-3-3 protein-protein interaction</span></div><div class="casAuthors">Doveston, Richard G.; Kuusk, Ave; Andrei, Sebastian A.; Leysen, Seppe; Cao, Qing; Castaldi, Maria P.; Hendricks, Adam; Brunsveld, Luc; Chen, Hongming; Boyd, Helen; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">591</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2449-2457</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The 14-3-3 proteins are pos. regulators of tumor suppressor protein p53, the mutation of which is implicated in many human cancers.  Current strategies for targeting of p53 involve restoration of wild-type function or inhibition of the interaction with MDM2, its key neg. regulator.  Despite the efficacy of these strategies, the alternate approach of stabilizing the interaction of p53 with pos. regulators and, thus, enhancing tumor suppressor activity, has not been explored.  Here, we report the 1st example of small-mol. (fusicoccin A) stabilization of the 14-3-3-p53 protein-protein interaction (PPI) and demonstrate the potential of this approach as a therapeutic modality.  We also obsd. a disconnect between biophys. and crystallog. data in the presence of a stabilizing mol., which is unusual in 14-3-3 PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpw1xMdr0EvsrVg90H21EOLACvtfcHk0ljywzqxOCQ71g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1yitLnI&md5=3048c3d9827ef0a5a50b502fc5cf6f5d</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2F1873-3468.12723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1873-3468.12723%26sid%3Dliteratum%253Aachs%26aulast%3DDoveston%26aufirst%3DR.%2BG.%26aulast%3DKuusk%26aufirst%3DA.%26aulast%3DAndrei%26aufirst%3DS.%26aulast%3DLeysen%26aufirst%3DS.%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DCastaldi%26aufirst%3DP.%26aulast%3DHendricks%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DBoyd%26aufirst%3DH.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DSmall-Molecule%2520Stabilization%2520of%2520the%2520P53%2520%25E2%2580%2593%252014-3-3%2520Protein-Protein%2520Interaction%26jtitle%3DFEBS%2520Lett.%26date%3D2017%26volume%3D591%26spage%3D2449%26epage%3D2457%26doi%3D10.1002%2F1873-3468.12723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Exploring the p65/14-3-3 interface with imine forming fragments. (A) Overview of the composite binding pocket of 14-3-3 and the p65_45 peptide (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QHL">6QHL</a>).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> (B) Crystal structure of the NF-κB subunit p65 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1IKN">1IKN</a>,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> IκBα and p50 hidden for clarity) with known phosphorylated serine residues Ser281 and Ser45 labeled. (C) Interface of the ternary complex of <b>1</b>/p65_45/14-3-3 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YOW">6YOW</a>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Indicated are three subpockets (the roof of 14-3-3, the FC-binding pocket, and the deep binding pocket), which can be exploited by fragment extension. Shown are 14-3-3 (white sticks, surface and cartoon), p65_45 peptide (red sticks), and fragment <b>1</b> (cyan sticks). (D) Fragment hits based on X-crystallography screening with benzaldehyde binding to Lys122 of 14-3-3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Nitrobenzaldehyde structural analogues. (A) Initial fragment hit <b>3</b> covalently bound to the 14-3-3/p65_45 complex. The fragment can be extended from its <i>meta-</i>position (black arrow shows proposed vector for extension) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YOY">6YOY</a>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> (B) Ternary structure of <b>9a</b> (salmon sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NMH">7NMH</a>), p65_45 (red sticks), and 14-3-3 (white cartoon/sticks). Water molecules: red spheres; hydrogen bonds: black dashes. (C) Ternary structure of <b>10a</b>/p65_45/14-3-3 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJF">7BJF</a>). Details as in (B). (D) Ternary structure of <b>13e</b> (left; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJL">7BJL</a>), <b>13f</b> (middle; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJW">7BJW</a>), or <b>13l</b> (right; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BKH">7BKH</a>) binding to the p65_45/14-3-3 complex. Compounds are shown as yellow sticks; the rest are as described before. The 2Fo-Fc electron density map is contoured at 1σ for all.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Focused Fragment Libraries 1 to 3</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Imidazole-benzaldehydes are stabilizing the p65_45/14-3-3 complex. (A) The imidazole moiety of <b>5</b> (teal sticks) offers three vectors for fragment growth (arrows) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YP3">6YP3</a>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> (B) Fragment induced p65_45/14-3-3γ ternary complex formation, measured with compound titrations in FA (r/mAU). The compounds were titrated to 50 μM 14-3-3γ and 100 nM p65_45. (C) Ternary structure of <b>16d</b> (light blue sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NM9">7NM9</a>) binding to the p65_45/14-3-3σΔC complex. (D) Mixture of structural isomers of <b>16j</b> and <b>16k</b>. (E, F) Crystal structure of <b>16k</b> binding to the p65_45/14-3-3σΔC complex (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NR7">7NR7</a>). The electron density map indicates two binding poses (E). Based on the position of the methoxy substitution, the 2-chloro substitution points either toward the peptide (<b>16k-5</b>) or to the FC pocket (<b>16k-6</b>). Both binding poses engage in hydrophobic contacts with 14-3-3 and p65 (sphere representation, F). For all: 14-3-3: white cartoon, surface, sticks; p65_45: red sticks; 2Fo-Fc electron density map (contoured at 1σ): gray mesh.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Focused Fragment Libraries 4 and 5</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Acetamide-benzaldehydes in complex with 14-3-3/p65_45. (A) 3D (beige sticks) and 2D structure of 4-formyl-benzamide. The C–C–C–N torsion angles (orange lines) cluster at τ ≈ 0 and 180°.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> (B) Initial fragment hit <b>4</b> can be extended from its <i>para-</i>position (black arrows; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YOX">6YOX</a>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> (C) Ternary structure of <b>19e</b> (green sticks) in complex with p65_45 (red sticks) and 14-3-3σΔC (white cartoon and sticks) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BIY">7BIY</a>). Hydrogen bonds are indicated with green dashes; the 2Fo-Fc electron density map is shown as gray mesh, contoured at 1σ.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Sulfonamide-benzaldehydes are active in FA compound titrations. (A) 3D and 2D chemical structure of 4-formyl-benzenesulfonamide with a C–C–S–N torsion angle τ as indicated.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> (B) The benzenesulfonamides are extended toward the FC pocket or toward the p65 peptide (red; 14-3-3: white; both with sticks and transparent surface) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YOW">6YOW</a>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> (C) FA compound titrations in the presence of 50 μM 14-3-3γ and 100 nM p65_45. The compounds are divided into weak (left), medium (middle), and high (right) activity. The legends are sorted by activity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Sulfonamide-benzaldehydes inducing complex stabilization. (A) Overlay of <b>23h</b> (green sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NLE">7NLE</a>), <b>23k</b> (orange sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NK5">7NK5</a>), <b>23f</b> (cyan sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NJB">7NJB</a>), and <b>23u</b> (yellow sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJB">7BJB</a>) binding to p65_45 (red sticks) and 14-3-3 (white cartoon and sticks). Hydrogen bonds are shown as black dashes, and hydrophobic contacts are indicated with transparent spheres. A conserved water molecule (small sphere) is shown colored based on the related fragment color. (B) Fragments <b>23h</b>, <b>k</b>, <b>f</b>, and <b>u</b> show complex stabilization in FA compound titrations (left; [14-3-3γ]: 50 μM, [p65_45]: 100 nM) and protein titrations (right; [fragment]: 1 mM, [p65_45]: 100 nM). (C) Overlay of <b>23n</b> (brown sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NM3">7NM3</a>) and <b>24m</b> (light blue sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NM1">7NM1</a>) in complex with 14-3-3σΔC/p65_45. (D) Crystal structures of <b>23w</b> (blue sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NLA">7NLA</a>) binding to the composite binding pocket of p65 (red sticks) and 14-3-3 (white cartoon and sticks). The 2Fo-Fc electron density map (1σ) is displayed as gray mesh, and hydrogen bonds are shown as black dashes. (E) Overlay of <b>23y</b> (green sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NK3">7NK3</a>) and <b>23z</b> (yellow; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NJ9">7NJ9</a>) binding to the p65_45/14-3-3 complex. Upon binding of <b>23y</b>, Pro47 changes its conformation (red sticks, as reference: Pro47 of the <b>23z</b>/p65_45/14-3-3σΔC complex shown as yellow sticks). (F) Ternary structure of <b>23z</b> (yellow sticks; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NJ9">7NJ9</a>) binding to the p65_45/14-3-3 complex. Details as in (D). (G) Ternary complex formation of <b>23x</b>, <b>y</b>, and <b>z</b> was measured with FA compound titrations (left; [14-3-3γ]: 50 μM, [p65_45]: 100 nM) and the stabilizing activity of <b>23w</b>, <b>y</b>, and <b>z</b> with protein titrations (right; [fragment]: 1 mM, [p65_45]: 100 nM).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/medium/jm1c00401_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Optimization of <b>23z</b>. (A) Ternary structure of <b>23z</b> (yellow spheres) in complex with 14-3-3σΔC (white surface) and p65_45 (red cartoon, transparent spheres) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NJ9">7NJ9</a>). Carbons of the bicyclic head group are numbered. (B) Ternary structure of <b>24b</b> (orange sticks) in complex with p65_45/14-3-3σΔC (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BIQ">7BIQ</a>). The distance between the 2-methyl and water (red spheres) is indicated with black dashes. Polar contacts are shown as black dashed lines. (C) Structure of <b>24e</b> (cyan sticks) binding to the p65_45/14-3-3σΔC complex (as in A; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BI3">7BI3</a>). Hydrophobic contacts are indicated with transparent spheres. (D) FA compound titrations with 50 μM 14-3-3γ and 100 nM p65_45. (E) FA protein titrations with 1 mM fragment and 100 nM p65_45. (F) Structure of <b>24j</b> binding to the p65_45/14-3-3σΔC complex (as in A; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BIW">7BIW</a>). A beneficial hydrogen bond is formed between the backbone carbonyl of p65s’ Pro47 and <b>24j</b> (black dash).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00401/20210616/images/large/jm1c00401_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00401&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i23">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88466" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88466" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 54 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Inhibitors of Protein-Protein Interactions: Progressing towards the Dream</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">301</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/nrd1343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnrd1343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=15060526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BD2cXis1GktrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=301-317&author=M.+M.+R.+Arkinauthor=J.+A.+Wells&title=Small-Molecule+Inhibitors+of+Protein-Protein+Interactions%3A+Progressing+towards+the+Dream&doi=10.1038%2Fnrd1343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule inhibitors of protein-protein interactions: progressing towards the dream</span></div><div class="casAuthors">Arkin, Michelle R.; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">301-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein-protein interactions have a key role in most biol. processes, and offer attractive opportunities for therapeutic intervention.  Developing small mols. that modulate protein-protein interactions is difficult, owing to issues such as the lack of well-defined binding pockets.  Nevertheless, there was important progress in this endeavor in recent years.  Here, the authors use illustrative examples to discuss general strategies for addressing the challenges inherent in the discovery and characterization of small-mol. inhibitors of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7MiFDdLJu27Vg90H21EOLACvtfcHk0ljWhq2w1bbggw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXis1GktrY%253D&md5=3e102c6207be420b4232056b4acdea12</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd1343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1343%26sid%3Dliteratum%253Aachs%26aulast%3DArkin%26aufirst%3DM.%2BM.%2BR.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSmall-Molecule%2520Inhibitors%2520of%2520Protein-Protein%2520Interactions%253A%2520Progressing%2520towards%2520the%2520Dream%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2004%26volume%3D3%26spage%3D301%26epage%3D317%26doi%3D10.1038%2Fnrd1343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span> <span> </span><span class="NLM_article-title">Recent Advances in the Development of Protein–Protein Interactions Modulators: Mechanisms and Clinical Trials</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">213</span>, <span class="refDoi"> DOI: 10.1038/s41392-020-00315-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fs41392-020-00315-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=32968059" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A280%3ADC%252BB3s%252Fis1GisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2020&pages=213&author=H.+Luauthor=Q.+Zhouauthor=J.+Heauthor=Z.+Jiangauthor=C.+Pengauthor=R.+Tongauthor=J.+Shi&title=Recent+Advances+in+the+Development+of+Protein%E2%80%93Protein+Interactions+Modulators%3A+Mechanisms+and+Clinical+Trials&doi=10.1038%2Fs41392-020-00315-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in the development of protein-protein interactions modulators: mechanisms and clinical trials</span></div><div class="casAuthors">Lu Haiying; Tong Rongsheng; Shi Jianyou; Zhou Qiaodan; He Jun; Jiang Zhongliang; Peng Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Signal transduction and targeted therapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">213</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein-protein interactions (PPIs) have pivotal roles in life processes.  The studies showed that aberrant PPIs are associated with various diseases, including cancer, infectious diseases, and neurodegenerative diseases.  Therefore, targeting PPIs is a direction in treating diseases and an essential strategy for the development of new drugs.  In the past few decades, the modulation of PPIs has been recognized as one of the most challenging drug discovery tasks.  In recent years, some PPIs modulators have entered clinical studies, some of which been approved for marketing, indicating that the modulators targeting PPIs have broad prospects.  Here, we summarize the recent advances in PPIs modulators, including small molecules, peptides, and antibodies, hoping to provide some guidance to the design of novel drugs targeting PPIs in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTce0zvk4iKluvW160gIxP_fW6udTcc2eZKumtVRaNQ7rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3s%252Fis1GisQ%253D%253D&md5=ec2910a147480ed3d7119f4bfd36523a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fs41392-020-00315-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-020-00315-3%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DZ.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DTong%26aufirst%3DR.%26aulast%3DShi%26aufirst%3DJ.%26atitle%3DRecent%2520Advances%2520in%2520the%2520Development%2520of%2520Protein%25E2%2580%2593Protein%2520Interactions%2520Modulators%253A%2520Mechanisms%2520and%2520Clinical%2520Trials%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2020%26volume%3D5%26spage%3D213%26doi%3D10.1038%2Fs41392-020-00315-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milroy, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennig, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Modulators of Protein-Protein Interactions</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">4695</span>– <span class="NLM_lpage">4748</span>, <span class="refDoi"> DOI: 10.1021/cr400698c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr400698c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmt1Sgsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2014&pages=4695-4748&author=L.+G.+Milroyauthor=T.+N.+Grossmannauthor=S.+Hennigauthor=L.+Brunsveldauthor=C.+Ottmann&title=Modulators+of+Protein-Protein+Interactions&doi=10.1021%2Fcr400698c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Modulators of Protein-Protein Interactions</span></div><div class="casAuthors">Milroy, Lech-Gustav; Grossmann, Tom N.; Hennig, Sven; Brunsveld, Luc; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4695-4748</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Since Hedin's characterization of trypsin and antitrypsin in 1906, arguably the first account of a regulatory protein-protein interaction (PPI), contemporary understanding of proteins and PPIs has been progressively transformed by landmark conceptual and technol. advances in mol. cell biol., biochem. and biophysics, not least, the sequencing of the human genome and the ensuing genomic technologies.  Today, proteins can be viewed as the mol. smart phones of the cell, genetically programmed to enact specific cellular functions in response to external stimuli.  Individually, proteins perform essential functions such as catalysis and the transport of mols. and ions.  However,their effectiveness in the crowded cellular environment is only short-range and insufficient to sustain life without the involvement of other biomols. such as other proteins or metabolites.  This review begins with a up-to-date account of the principle biochem. techniques used to identify PPI modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrty0-k5R_QrVg90H21EOLACvtfcHk0ljWhq2w1bbggw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmt1Sgsrs%253D&md5=89eddaa7ebaa7b9e4dfe705e86b55fe7</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fcr400698c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr400698c%26sid%3Dliteratum%253Aachs%26aulast%3DMilroy%26aufirst%3DL.%2BG.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DHennig%26aufirst%3DS.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DModulators%2520of%2520Protein-Protein%2520Interactions%26jtitle%3DChem.%2520Rev.%26date%3D2014%26volume%3D114%26spage%3D4695%26epage%3D4748%26doi%3D10.1021%2Fcr400698c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stevers, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sijbesma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKintosh, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obsil, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landrieu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cau, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karawajczyk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eickhoff, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doveston, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Modulators of 14-3-3 Protein–Protein Interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_fpage">3755</span>– <span class="NLM_lpage">3778</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00574</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00574" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=3755-3778&author=L.+M.+Steversauthor=E.+Sijbesmaauthor=M.+Bottaauthor=C.+MacKintoshauthor=T.+Obsilauthor=I.+Landrieuauthor=Y.+Cauauthor=A.+J.+Wilsonauthor=A.+Karawajczykauthor=J.+Eickhoffauthor=J.+Davisauthor=M.+Hannauthor=G.+O%E2%80%99Mahonyauthor=R.+G.+Dovestonauthor=L.+Brunsveldauthor=C.+Ottmann&title=Modulators+of+14-3-3+Protein%E2%80%93Protein+Interactions&doi=10.1021%2Facs.jmedchem.7b00574"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00574%26sid%3Dliteratum%253Aachs%26aulast%3DStevers%26aufirst%3DL.%2BM.%26aulast%3DSijbesma%26aufirst%3DE.%26aulast%3DBotta%26aufirst%3DM.%26aulast%3DMacKintosh%26aufirst%3DC.%26aulast%3DObsil%26aufirst%3DT.%26aulast%3DLandrieu%26aufirst%3DI.%26aulast%3DCau%26aufirst%3DY.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26aulast%3DKarawajczyk%26aufirst%3DA.%26aulast%3DEickhoff%26aufirst%3DJ.%26aulast%3DDavis%26aufirst%3DJ.%26aulast%3DHann%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DG.%26aulast%3DDoveston%26aufirst%3DR.%2BG.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DModulators%2520of%252014-3-3%2520Protein%25E2%2580%2593Protein%2520Interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26spage%3D3755%26epage%3D3778%26doi%3D10.1021%2Facs.jmedchem.7b00574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skidmore, J.</span></span> <span> </span><span class="NLM_article-title">Small Molecules, Big Targets: Drug Discovery Faces the Protein–Protein Interaction Challenge</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">533</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.29</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnrd.2016.29" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=27050677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=533-550&author=D.+E.+Scottauthor=A.+R.+Baylyauthor=C.+Abellauthor=J.+Skidmore&title=Small+Molecules%2C+Big+Targets%3A+Drug+Discovery+Faces+the+Protein%E2%80%93Protein+Interaction+Challenge&doi=10.1038%2Fnrd.2016.29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules, big targets: drug discovery faces the protein-protein interaction challenge</span></div><div class="casAuthors">Scott, Duncan E.; Bayly, Andrew R.; Abell, Chris; Skidmore, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">533-550</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) are of pivotal importance in the regulation of biol. systems and are consequently implicated in the development of disease states.  Recent work has begun to show that, with the right tools, certain classes of PPI can yield to the efforts of medicinal chemists to develop inhibitors, and the first PPI inhibitors have reached clin. development.  In this Review, we describe the research leading to these breakthroughs and highlight the existence of groups of structurally related PPIs within the PPI target class.  For each of these groups, we use examples of successful discovery efforts to illustrate the research strategies that have proved most useful.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK-aGzHa9mP7Vg90H21EOLACvtfcHk0li_EWjY2L3oYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D&md5=9f0953567ae9338b8b36aa78b8160655</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.29&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.29%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DBayly%26aufirst%3DA.%2BR.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DSkidmore%26aufirst%3DJ.%26atitle%3DSmall%2520Molecules%252C%2520Big%2520Targets%253A%2520Drug%2520Discovery%2520Faces%2520the%2520Protein%25E2%2580%2593Protein%2520Interaction%2520Challenge%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2016%26volume%3D15%26spage%3D533%26epage%3D550%26doi%3D10.1038%2Fnrd.2016.29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blenis, J.</span></span> <span> </span><span class="NLM_article-title">Rapamycin: One Drug</span>. <i>Many Effects. Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">373</span>– <span class="NLM_lpage">379</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2014.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1016%2Fj.cmet.2014.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=24508508" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=373-379&author=J.+Liauthor=S.+G.+Kimauthor=J.+Blenis&title=Rapamycin%3A+One+Drug&doi=10.1016%2Fj.cmet.2014.01.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2014.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2014.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%2BG.%26aulast%3DBlenis%26aufirst%3DJ.%26atitle%3DRapamycin%253A%2520One%2520Drug%26jtitle%3DMany%2520Effects.%2520Cell%2520Metab.%26date%3D2014%26volume%3D19%26spage%3D373%26epage%3D379%26doi%3D10.1016%2Fj.cmet.2014.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kunz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. N.</span></span> <span> </span><span class="NLM_article-title">Cyclosporin A, FK506 and Rapamycin: More than Just Immunosuppression</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">334</span>– <span class="NLM_lpage">338</span>, <span class="refDoi"> DOI: 10.1016/0968-0004(93)90069-Y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1016%2F0968-0004%2893%2990069-Y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=7694398" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADyaK3sXlvFGjsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1993&pages=334-338&author=J.+Kunzauthor=M.+N.+Hall&title=Cyclosporin+A%2C+FK506+and+Rapamycin%3A+More+than+Just+Immunosuppression&doi=10.1016%2F0968-0004%2893%2990069-Y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclosporin A, FK506 and rapamycin: more than just immunosuppression</span></div><div class="casAuthors">Kunz, Jeannette; Hall, Michael N.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">334-8</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    </div><div class="casAbstract">A review with 35 refs.  The mechanisms of action of the immunosuppressive drugs cyclosphorin A(CsA), FK506 and rapamycin are strikingly conserved from yeast to human T cells.  Recent results obtained with yeast corroborate calcineurin as the target of CsA-cyclophilin and FK506-FKBP complexes, and reveal a phosphatidylinositol 3-kinase homolog as the target of the rapamycin-FKBP complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro4ILcAHS6mrVg90H21EOLACvtfcHk0li_EWjY2L3oYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXlvFGjsrg%253D&md5=3a97087838b164d58bb654a3defada7f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2F0968-0004%2893%2990069-Y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0968-0004%252893%252990069-Y%26sid%3Dliteratum%253Aachs%26aulast%3DKunz%26aufirst%3DJ.%26aulast%3DHall%26aufirst%3DM.%2BN.%26atitle%3DCyclosporin%2520A%252C%2520FK506%2520and%2520Rapamycin%253A%2520More%2520than%2520Just%2520Immunosuppression%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1993%26volume%3D18%26spage%3D334%26epage%3D338%26doi%3D10.1016%2F0968-0004%2893%2990069-Y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wani, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coggon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McPhail, A. T.</span></span> <span> </span><span class="NLM_article-title">Plant Antitumor Agents. VI. Isolation and Structure of Taxol, a Novel Antileukemic and Antitumor Agent from Taxus Brevifolia</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1971</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">2325</span>– <span class="NLM_lpage">2327</span>, <span class="refDoi"> DOI: 10.1021/ja00738a045</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00738a045" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADyaE3MXktVKisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1971&pages=2325-2327&author=M.+C.+Waniauthor=H.+L.+Taylorauthor=M.+E.+Wallauthor=P.+Coggonauthor=A.+T.+McPhail&title=Plant+Antitumor+Agents.+VI.+Isolation+and+Structure+of+Taxol%2C+a+Novel+Antileukemic+and+Antitumor+Agent+from+Taxus+Brevifolia&doi=10.1021%2Fja00738a045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Plant antitumor agents.  VI.  Isolation and structure of taxol, a novel antileukemic and antitumor agent from Taxus brevifolia</span></div><div class="casAuthors">Wani, Mansukhlal C.; Taylor, Harold Lawrence; Wall, Monroe E.; Coggon, Philip; McPhail, Andrew T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1971</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2325-7</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Taxol (I), a complex ester with a taxane ring system exhibiting potent antileukemic and antitumor properties, was isolated from Taxus brevifolia (yew).  BzNHCH(Ph)CH(OH)CO2Me and 5β,16-epoxy-1β,-2α,4α,7β,10β,13α-hexahydroxytax-11-en-9-one 4α-acetate 2α-benzoate are obtained by methanolysis of I.  Based on chem. evidence, I is 5β,16-epoxy-1β,2α,4α,7β,10β,13α-hexahydroxytax-11-en-9-one 4α,10β-diacetate 2α-benzoate 13α-(3-benzamido-2-hydroxy-3-phenylpropionate).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpexINv4qMTjbVg90H21EOLACvtfcHk0lgnTrcE8HmWvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3MXktVKisLc%253D&md5=50c59ef3bf2ae493408bed2a76ca0c2e</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fja00738a045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00738a045%26sid%3Dliteratum%253Aachs%26aulast%3DWani%26aufirst%3DM.%2BC.%26aulast%3DTaylor%26aufirst%3DH.%2BL.%26aulast%3DWall%26aufirst%3DM.%2BE.%26aulast%3DCoggon%26aufirst%3DP.%26aulast%3DMcPhail%26aufirst%3DA.%2BT.%26atitle%3DPlant%2520Antitumor%2520Agents.%2520VI.%2520Isolation%2520and%2520Structure%2520of%2520Taxol%252C%2520a%2520Novel%2520Antileukemic%2520and%2520Antitumor%2520Agent%2520from%2520Taxus%2520Brevifolia%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1971%26volume%3D93%26spage%3D2325%26epage%3D2327%26doi%3D10.1021%2Fja00738a045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joerger, M.</span></span> <span> </span><span class="NLM_article-title">Metabolism of the Taxanes Including Nab-Paclitaxel</span>. <i>Expert Opin. Drug Metab. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">691</span>– <span class="NLM_lpage">702</span>, <span class="refDoi"> DOI: 10.1517/17425255.2015.983074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1517%2F17425255.2015.983074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=25394848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotVeks74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=691-702&author=M.+Joerger&title=Metabolism+of+the+Taxanes+Including+Nab-Paclitaxel&doi=10.1517%2F17425255.2015.983074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism of the taxanes including nab-paclitaxel</span></div><div class="casAuthors">Joerger, Markus</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Metabolism & Toxicology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">691-702</span>CODEN:
                <span class="NLM_cas:coden">EODMAP</span>;
        ISSN:<span class="NLM_cas:issn">1742-5255</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The classical taxanes (paclitaxel, docetaxel), the newer taxane cabazitaxel and the nanoparticle-bound nab-paclitaxel are among the most widely used anticancer drugs.  Despite years of research, the optimal dosing regimen (weekly vs 3-weekly) and optimal dose is still controversial, as is the value of pharmacol. personalization of taxane dosing.  Areas covered: This review provides an overview of the pharmacol. properties of the taxanes, including metab., pharmacokinetics-pharmacodynamics and aspects in the clin. use of taxanes.  The latter includes the ongoing debate on the most active and safe regimen (paclitaxel, docetaxel, nab-paclitaxel), the recommended initial dose (cabazitaxel) and pharmacol. dosing individualization.  Expert opinion: Taxanes share the characteristics of extensive hepatic metab. and biliary excretion, the need for dose adaptation in patients with liver dysfunction, and substantial pharmacokinetic variability even after taking into account known covariates.  Data from clin. studies suggest that optimal scheduling of the taxanes is dependent not only on the specific taxane compd., but also on the tumor type and line of treatment.  Finally, treating oncologists should be aware of the substantial risk for drug-drug interactions that is a direct consequence of the complex hepatic metab. of the taxanes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrftQ6JPyajY7Vg90H21EOLACvtfcHk0lgnTrcE8HmWvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotVeks74%253D&md5=7ab3c51d343b63c6917708c729912835</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1517%2F17425255.2015.983074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425255.2015.983074%26sid%3Dliteratum%253Aachs%26aulast%3DJoerger%26aufirst%3DM.%26atitle%3DMetabolism%2520of%2520the%2520Taxanes%2520Including%2520Nab-Paclitaxel%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2015%26volume%3D11%26spage%3D691%26epage%3D702%26doi%3D10.1517%2F17425255.2015.983074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petzold, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomä, N. H.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of Lenalidomide-Induced CK1α Degradation by the CRL4CRBN Ubiquitin Ligase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>532</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">130</span>, <span class="refDoi"> DOI: 10.1038/nature16979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnature16979" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=26909574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtFOis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=532&publication_year=2016&pages=127-130&author=G.+Petzoldauthor=E.+S.+Fischerauthor=N.+H.+Thom%C3%A4&title=Structural+Basis+of+Lenalidomide-Induced+CK1%CE%B1+Degradation+by+the+CRL4CRBN+Ubiquitin+Ligase&doi=10.1038%2Fnature16979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of lenalidomide-induced CK1α degradation by the CRL4CRBN ubiquitin ligase</span></div><div class="casAuthors">Petzold, Georg; Fischer, Eric S.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">532</span>
        (<span class="NLM_cas:issue">7597</span>),
    <span class="NLM_cas:pages">127-130</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thalidomide and its derivs., lenalidomide and pomalidomide, are immune modulatory drugs (IMiDs) used in the treatment of hematol. malignancies.  IMiDs bind CRBN, the substrate receptor of the CUL4-RBX1-DDB1-CRBN (also known as CRL4CRBN) E3 ubiquitin ligase, and inhibit ubiquitination of endogenous CRL4CRBN substrates.  Unexpectedly, IMiDs also repurpose the ligase to target new proteins for degrdn.  Lenalidomide induces degrdn. of the lymphoid transcription factors Ikaros and Aiolos (also known as IKZF1 and IKZF3), and casein kinase 1α (CK1α), which contributes to its clin. efficacy in the treatment of multiple myeloma and 5q-deletion assocd. myelodysplastic syndrome (del(5q) MDS), resp.  How lenalidomide alters the specificity of the ligase to degrade these proteins remains elusive.  Here we present the 2.45 Å crystal structure of DDB1-CRBN bound to lenalidomide and CK1α.  CRBN and lenalidomide jointly provide the binding interface for a CK1α β-hairpin-loop located in the kinase N-lobe.  We show that CK1α binding to CRL4CRBN is strictly dependent on the presence of an IMiD.  Binding of IKZF1 to CRBN similarly requires the compd. and both, IKZF1 and CK1α, use a related binding mode.  Our study provides a mechanistic explanation for the selective efficacy of lenalidomide in del(5q) MDS therapy.  We anticipate that high-affinity protein-protein interactions induced by small mols. will provide opportunities for drug development, particularly for targeted protein degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs4FxWriI1ObVg90H21EOLACvtfcHk0lgnTrcE8HmWvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtFOis78%253D&md5=320fc6ff811da335fa8e02feab2d702d</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnature16979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16979%26sid%3Dliteratum%253Aachs%26aulast%3DPetzold%26aufirst%3DG.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DStructural%2520Basis%2520of%2520Lenalidomide-Induced%2520CK1%25CE%25B1%2520Degradation%2520by%2520the%2520CRL4CRBN%2520Ubiquitin%2520Ligase%26jtitle%3DNature%26date%3D2016%26volume%3D532%26spage%3D127%26epage%3D130%26doi%3D10.1038%2Fnature16979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of PROTAC Cooperative Recognition for Selective Protein Degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.+H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+Basis+of+PROTAC+Cooperative+Recognition+for+Selective+Protein+Degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0lhvGcIdsBhA4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520Basis%2520of%2520PROTAC%2520Cooperative%2520Recognition%2520for%2520Selective%2520Protein%2520Degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maniaci, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Bifunctional Chemical Probes Inducing Protein–Protein Interactions</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">156</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2019.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1016%2Fj.cbpa.2019.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=31419624" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFyltbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2019&pages=145-156&author=C.+Maniaciauthor=A.+Ciulli&title=Bifunctional+Chemical+Probes+Inducing+Protein%E2%80%93Protein+Interactions&doi=10.1016%2Fj.cbpa.2019.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Bifunctional chemical probes inducing protein-protein interactions</span></div><div class="casAuthors">Maniaci, Chiara; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">145-156</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Inducing biomol. interactions with synthetic mols. to impact biol. function is a concept of enormous appeal.  Recent years have seen a resurgence of interest in designing bispecific mols. that serve as bridging agents to bring proteins together.  Pioneering structural and biophys. investigation of ternary complexes formed by mono-functional and bifunctional ligands highlights that proximity-induced stabilization or de novo formation of protein-protein interactions is a common feature of their mol. recognition.  In this review, we illustrate these concepts and advances with representative case studies, and highlight progress over the past three years, with particular focus on recruitment to E3 ubiquitin ligases by 'mol. glues' and chimeric dimerizers (PROTACs) for targeted protein degrdn.  This approach promises to significantly expand the range of tractable targets for chem. biol. and therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobXJM1PThHPbVg90H21EOLACvtfcHk0lhvGcIdsBhA4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFyltbjF&md5=4302bb010003177f4f5e5c3d4a401dc9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2019.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2019.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DManiaci%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DBifunctional%2520Chemical%2520Probes%2520Inducing%2520Protein%25E2%2580%2593Protein%2520Interactions%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D52%26spage%3D145%26epage%3D156%26doi%3D10.1016%2Fj.cbpa.2019.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Vink, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrei, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milroy, L.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span> <span> </span><span class="NLM_article-title">Cooperativity Basis for Small-Molecule Stabilization of Protein–Protein Interactions</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2869</span>– <span class="NLM_lpage">2874</span>, <span class="refDoi"> DOI: 10.1039/C8SC05242E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1039%2FC8SC05242E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=30996864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVWnu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=2869-2874&author=P.+J.+de%0AVinkauthor=S.+A.+Andreiauthor=Y.+Higuchiauthor=C.+Ottmannauthor=L.-G.+Milroyauthor=L.+Brunsveld&title=Cooperativity+Basis+for+Small-Molecule+Stabilization+of+Protein%E2%80%93Protein+Interactions&doi=10.1039%2FC8SC05242E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Cooperativity basis for small-molecule stabilization of protein-protein interactions</span></div><div class="casAuthors">de Vink, Pim J.; Andrei, Sebastian A.; Higuchi, Yusuke; Ottmann, Christian; Milroy, Lech-Gustav; Brunsveld, Luc</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2869-2874</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A cooperativity framework to describe and interpret small-mol. stabilization of protein-protein interactions (PPI) is presented.  The stabilization of PPIs is a versatile and emerging therapeutic strategy to target specific combinations of protein partners within the protein interactome.  Currently, the potency of PPI stabilizers is typically expressed by their apparent affinity or EC50.  Here, we propose that the effect of a PPI stabilizer be best described involving the cooperativity factor, α, between the stabilizer and binding partners in addn. to the intrinsic affinity, KDII, of the stabilizer for one of the apo-proteins.  By way of illustration, we combine fluorescence polarization measurements with thermodn. modeling to det. the α and KDII for the PPI stabilization of 14-3-3 and TASK3 by fusicoccin-A (FC-A) and validate our approach by studying other PPI-partners of 14-3-3 proteins.  Finally, we characterize a library of different stabilizer compds., and perform structure-activity relationship studies in which mol. changes could be attributed to either changes in cooperativity or intrinsic affinity.  Such insights should aid in the development of more effective protein-protein stabilizer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSuxb4CPB-lbVg90H21EOLACvtfcHk0liUSoO_KNDA2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVWnu78%253D&md5=5c66eab5f8accfcb7a24f280e349d82b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1039%2FC8SC05242E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8SC05242E%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BVink%26aufirst%3DP.%2BJ.%26aulast%3DAndrei%26aufirst%3DS.%2BA.%26aulast%3DHiguchi%26aufirst%3DY.%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DMilroy%26aufirst%3DL.-G.%26aulast%3DBrunsveld%26aufirst%3DL.%26atitle%3DCooperativity%2520Basis%2520for%2520Small-Molecule%2520Stabilization%2520of%2520Protein%25E2%2580%2593Protein%2520Interactions%26jtitle%3DChem.%2520Sci.%26date%3D2019%26volume%3D10%26spage%3D2869%26epage%3D2874%26doi%3D10.1039%2FC8SC05242E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, J. R.</span></span> <span> </span><span class="NLM_article-title">Cooperativity in Macromolecular Assembly</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">458</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1038/nchembio.102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnchembio.102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=18641626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslClsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=458-465&author=J.+R.+Williamson&title=Cooperativity+in+Macromolecular+Assembly&doi=10.1038%2Fnchembio.102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Cooperativity in macromolecular assembly</span></div><div class="casAuthors">Williamson, James R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">458-465</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The thermodn. principle of cooperativity is used to drive the formation of specific macromol. complexes during the assembly of a macromol. machine.  Understanding cooperativity provides insight into the mechanisms that govern assembly and disassembly of multicomponent complexes.  Our understanding of assembly mechanisms is lagging considerably behind our understanding of the structure and function of these complexes.  A significant challenge remains in tackling the thermodn. and kinetics of the intermol. interactions required for all cellular functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppUr8iKTgUG7Vg90H21EOLACvtfcHk0liUSoO_KNDA2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslClsLc%253D&md5=37ec3e5140b64abf0c005a4f8f79b5bd</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.102%26sid%3Dliteratum%253Aachs%26aulast%3DWilliamson%26aufirst%3DJ.%2BR.%26atitle%3DCooperativity%2520in%2520Macromolecular%2520Assembly%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D458%26epage%3D465%26doi%3D10.1038%2Fnchembio.102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Twenty Years on: The Impact of Fragments on Drug Discovery</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">619</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+Years+on%3A+The+Impact+of+Fragments+on+Drug+Discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0liUSoO_KNDA2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520Years%2520on%253A%2520The%2520Impact%2520of%2520Fragments%2520on%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2016%26volume%3D15%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Box, G.</span></span> <span> </span><span class="NLM_article-title">Perspective Fragment-Based Drug Discovery : Advancing Fragments in the Absence of Crystal Structures</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.10.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1016%2Fj.chembiol.2018.10.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=30482678" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVWisrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=9-15&author=D.+A.+Erlansonauthor=B.+J.+Davisauthor=W.+Jahnkeauthor=G.+Box&title=Perspective+Fragment-Based+Drug+Discovery+%3A+Advancing+Fragments+in+the+Absence+of+Crystal+Structures&doi=10.1016%2Fj.chembiol.2018.10.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery: Advancing Fragments in the Absence of Crystal Structures</span></div><div class="casAuthors">Erlanson, Daniel A.; Davis, Ben J.; Jahnke, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-15</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Fragment-based drug discovery typically requires an interplay between screening methods, structural methods, and medicinal chem.  X-ray crystallog. is generally the method of choice to obtain three-dimensional structures of the bound ligand/protein complex, but this can sometimes be difficult, particularly for early, low-affinity fragment hits.  In this Perspective, we discuss strategies to advance and evolve fragments in the absence of crystal structures of protein-fragment complexes, although the structure of the unliganded protein may be available.  The strategies can involve other structural techniques, such as NMR spectroscopy, mol. modeling, or a variety of chem. approaches.  Often, these strategies are aimed at guiding evolution of initial fragment hits to a stage where crystal structures can be obtained for further structure-based optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmStmkIpx6bLVg90H21EOLACvtfcHk0liUSoO_KNDA2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVWisrzI&md5=2f5fb9d063b7ccabb4f7eb611c00ce30</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.10.001%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DDavis%26aufirst%3DB.%2BJ.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DBox%26aufirst%3DG.%26atitle%3DPerspective%2520Fragment-Based%2520Drug%2520Discovery%2520%253A%2520Advancing%2520Fragments%2520in%2520the%2520Absence%2520of%2520Crystal%2520Structures%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D9%26epage%3D15%26doi%3D10.1016%2Fj.chembiol.2018.10.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalton, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span> <span> </span><span class="NLM_article-title">Covalent Small Molecules as Enabling Platforms for Drug Discovery</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">1100</span>, <span class="refDoi"> DOI: 10.1002/cbic.201900674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1002%2Fcbic.201900674" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=1-1100&author=S.+E.+Daltonauthor=S.+Campos&title=Covalent+Small+Molecules+as+Enabling+Platforms+for+Drug+Discovery&doi=10.1002%2Fcbic.201900674"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201900674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201900674%26sid%3Dliteratum%253Aachs%26aulast%3DDalton%26aufirst%3DS.%2BE.%26aulast%3DCampos%26aufirst%3DS.%26atitle%3DCovalent%2520Small%2520Molecules%2520as%2520Enabling%2520Platforms%2520for%2520Drug%2520Discovery%26jtitle%3DChemBioChem%26date%3D2019%26spage%3D1%26epage%3D1100%26doi%3D10.1002%2Fcbic.201900674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gabizon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shraga, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehrtz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livnah, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shorer, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gurwicz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avram, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aharoni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albeck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shulman, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herishanu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">London, N.</span></span> <span> </span><span class="NLM_article-title">Efficient Targeted Degradation via Reversible and Irreversible Covalent PROTACs</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">11734</span>– <span class="NLM_lpage">11742</span>, <span class="refDoi"> DOI: 10.1021/jacs.9b13907</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.9b13907" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosF2gtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2020&pages=11734-11742&author=R.+Gabizonauthor=A.+Shragaauthor=P.+Gehrtzauthor=E.+Livnahauthor=Y.+Shorerauthor=N.+Gurwiczauthor=L.+Avramauthor=T.+Ungerauthor=H.+Aharoniauthor=S.+Albeckauthor=A.+Brandisauthor=Z.+Shulmanauthor=B.+Katzauthor=Y.+Herishanuauthor=N.+London&title=Efficient+Targeted+Degradation+via+Reversible+and+Irreversible+Covalent+PROTACs&doi=10.1021%2Fjacs.9b13907"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Targeted Degradation via Reversible and Irreversible Covalent PROTACs</span></div><div class="casAuthors">Gabizon, Ronen; Shraga, Amit; Gehrtz, Paul; Livnah, Ella; Shorer, Yamit; Gurwicz, Neta; Avram, Liat; Unger, Tamar; Aharoni, Hila; Albeck, Shira; Brandis, Alexander; Shulman, Ziv; Katz, Ben-Zion; Herishanu, Yair; London, Nir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">11734-11742</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis targeting chimeras (PROTACs) represent an exciting inhibitory modality with many advantages, including substoichiometric degrdn. of targets.  Their scope, though, is still limited to date by the requirement for a sufficiently potent target binder.  A soln. that proved useful in tackling challenging targets is the use of electrophiles to allow irreversible binding to the target.  However, such binding will negate the catalytic nature of PROTACs.  Reversible covalent PROTACs potentially offer the best of both worlds.  They possess the potency and selectivity assocd. with the formation of the covalent bond, while being able to dissoc. and regenerate once the protein target is degraded.  Using Bruton's tyrosine kinase (BTK) as a clin. relevant model system, we show efficient degrdn. by noncovalent, irreversible covalent, and reversible covalent PROTACs, with <10 nM DC50's and >85% degrdn.  Our data suggest that part of the degrdn. by our irreversible covalent PROTACs is driven by reversible binding prior to covalent bond formation, while the reversible covalent PROTACs drive degrdn. primarily by covalent engagement.  The PROTACs showed enhanced inhibition of B cell activation compared to ibrutinib and exhibit potent degrdn. of BTK in patient-derived primary chronic lymphocytic leukemia cells.  The most potent reversible covalent PROTAC, RC-3(I), exhibited enhanced selectivity toward BTK compared to noncovalent and irreversible covalent PROTACs.  These compds. may pave the way for the design of covalent PROTACs for a wide variety of challenging targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHhULFyC9cFbVg90H21EOLACvtfcHk0lg-tIoZVue31g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosF2gtr0%253D&md5=dda97d35aa486202fd34ee0c8bccdab1</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjacs.9b13907&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.9b13907%26sid%3Dliteratum%253Aachs%26aulast%3DGabizon%26aufirst%3DR.%26aulast%3DShraga%26aufirst%3DA.%26aulast%3DGehrtz%26aufirst%3DP.%26aulast%3DLivnah%26aufirst%3DE.%26aulast%3DShorer%26aufirst%3DY.%26aulast%3DGurwicz%26aufirst%3DN.%26aulast%3DAvram%26aufirst%3DL.%26aulast%3DUnger%26aufirst%3DT.%26aulast%3DAharoni%26aufirst%3DH.%26aulast%3DAlbeck%26aufirst%3DS.%26aulast%3DBrandis%26aufirst%3DA.%26aulast%3DShulman%26aufirst%3DZ.%26aulast%3DKatz%26aufirst%3DB.%26aulast%3DHerishanu%26aufirst%3DY.%26aulast%3DLondon%26aufirst%3DN.%26atitle%3DEfficient%2520Targeted%2520Degradation%2520via%2520Reversible%2520and%2520Irreversible%2520Covalent%2520PROTACs%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2020%26volume%3D142%26spage%3D11734%26epage%3D11742%26doi%3D10.1021%2Fjacs.9b13907" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Resnick, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douangamath, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krojer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sethi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geurink, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aimon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amitai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairhead, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabizon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Straub, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szommer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaidman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coker, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovaa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">London, N.</span></span> <span> </span><span class="NLM_article-title">Rapid Covalent-Probe Discovery by Electrophile-Fragment Screening</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_fpage">8951</span>– <span class="NLM_lpage">8968</span>, <span class="refDoi"> DOI: 10.1021/jacs.9b02822</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.9b02822" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovVKmsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=8951-8968&author=E.+Resnickauthor=A.+Bradleyauthor=J.+Ganauthor=A.+Douangamathauthor=T.+Krojerauthor=R.+Sethiauthor=P.+P.+Geurinkauthor=A.+Aimonauthor=G.+Amitaiauthor=D.+Belliniauthor=J.+Bennettauthor=M.+Fairheadauthor=O.+Fedorovauthor=R.+Gabizonauthor=J.+Ganauthor=J.+Guoauthor=A.+Plotnikovauthor=V.+M.+Straubauthor=T.+Szommerauthor=S.+Velupillaiauthor=D.+Zaidmanauthor=Y.+Zhangauthor=A.+R.+Cokerauthor=P.+E.+Brennanauthor=H.+Ovaaauthor=F.+Von+Delftauthor=N.+London&title=Rapid+Covalent-Probe+Discovery+by+Electrophile-Fragment+Screening&doi=10.1021%2Fjacs.9b02822"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Covalent-Probe Discovery by Electrophile-Fragment Screening</span></div><div class="casAuthors">Resnick, Efrat; Bradley, Anthony; Gan, Jinrui; Douangamath, Alice; Krojer, Tobias; Sethi, Ritika; Geurink, Paul P.; Aimon, Anthony; Amitai, Gabriel; Bellini, Dom; Bennett, James; Fairhead, Michael; Fedorov, Oleg; Gabizon, Ronen; Gan, Jin; Guo, Jingxu; Plotnikov, Alexander; Reznik, Nava; Ruda, Gian Filippo; Diaz-Saez, Laura; Straub, Verena M.; Szommer, Tamas; Velupillai, Srikannathasan; Zaidman, Daniel; Zhang, Yanling; Coker, Alun R.; Dowson, Christopher G.; Barr, Haim M.; Wang, Chu; Huber, Kilian V. M.; Brennan, Paul E.; Ovaa, Huib; von Delft, Frank; London, Nir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8951-8968</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Covalent probes can display unmatched potency, selectivity, and duration of action; however, their discovery is challenging.  In principle, fragments that can irreversibly bind their target can overcome the low affinity that limits reversible fragment screening, but such electrophilic fragments were considered nonselective and were rarely screened.  We hypothesized that mild electrophiles might overcome the selectivity challenge and constructed a library of 993 mildly electrophilic fragments.  We characterized this library by a new high-throughput thiol-reactivity assay and screened them against 10 cysteine-contg. proteins.  Highly reactive and promiscuous fragments were rare and could be easily eliminated.  In contrast, we found hits for most targets.  Combining our approach with high-throughput crystallog. allowed rapid progression to potent and selective probes for two enzymes, the deubiquitinase OTUB2 and the pyrophosphatase NUDT7.  No inhibitors were previously known for either.  This study highlights the potential of electrophile-fragment screening as a practical and efficient tool for covalent-ligand discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGuVq_Zb2i_7Vg90H21EOLACvtfcHk0lg-tIoZVue31g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovVKmsrg%253D&md5=18d7a00b9ce67f4f23fbb65f473cac78</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjacs.9b02822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.9b02822%26sid%3Dliteratum%253Aachs%26aulast%3DResnick%26aufirst%3DE.%26aulast%3DBradley%26aufirst%3DA.%26aulast%3DGan%26aufirst%3DJ.%26aulast%3DDouangamath%26aufirst%3DA.%26aulast%3DKrojer%26aufirst%3DT.%26aulast%3DSethi%26aufirst%3DR.%26aulast%3DGeurink%26aufirst%3DP.%2BP.%26aulast%3DAimon%26aufirst%3DA.%26aulast%3DAmitai%26aufirst%3DG.%26aulast%3DBellini%26aufirst%3DD.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DFairhead%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DGabizon%26aufirst%3DR.%26aulast%3DGan%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DPlotnikov%26aufirst%3DA.%26aulast%3DStraub%26aufirst%3DV.%2BM.%26aulast%3DSzommer%26aufirst%3DT.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DZaidman%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCoker%26aufirst%3DA.%2BR.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DOvaa%26aufirst%3DH.%26aulast%3DVon%2BDelft%26aufirst%3DF.%26aulast%3DLondon%26aufirst%3DN.%26atitle%3DRapid%2520Covalent-Probe%2520Discovery%2520by%2520Electrophile-Fragment%2520Screening%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26spage%3D8951%26epage%3D8968%26doi%3D10.1021%2Fjacs.9b02822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Resnick, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douangamath, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krojer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sethi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geurink, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aimon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amitai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairhead, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabizon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reznik, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Díaz-Sáez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Straub, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szommer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaidman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coker, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowson, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barr, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovaa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">London, N.</span></span> <span> </span><span class="NLM_article-title">Rapid Covalent-Probe Discovery by Electrophile-Fragment Screening</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">8951</span>– <span class="NLM_lpage">8968</span>, <span class="refDoi"> DOI: 10.1021/jacs.9b02822</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.9b02822" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovVKmsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=8951-8968&author=E.+Resnickauthor=A.+Bradleyauthor=J.+Ganauthor=A.+Douangamathauthor=T.+Krojerauthor=R.+Sethiauthor=P.+P.+Geurinkauthor=A.+Aimonauthor=G.+Amitaiauthor=D.+Belliniauthor=J.+Bennettauthor=M.+Fairheadauthor=O.+Fedorovauthor=R.+Gabizonauthor=J.+Ganauthor=J.+Guoauthor=A.+Plotnikovauthor=N.+Reznikauthor=G.+F.+Rudaauthor=L.+D%C3%ADaz-S%C3%A1ezauthor=V.+M.+Straubauthor=T.+Szommerauthor=S.+Velupillaiauthor=D.+Zaidmanauthor=Y.+Zhangauthor=A.+R.+Cokerauthor=C.+G.+Dowsonauthor=H.+M.+Barrauthor=C.+Wangauthor=K.+V.+M.+Huberauthor=P.+E.+Brennanauthor=H.+Ovaaauthor=F.+Von+Delftauthor=N.+London&title=Rapid+Covalent-Probe+Discovery+by+Electrophile-Fragment+Screening&doi=10.1021%2Fjacs.9b02822"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Covalent-Probe Discovery by Electrophile-Fragment Screening</span></div><div class="casAuthors">Resnick, Efrat; Bradley, Anthony; Gan, Jinrui; Douangamath, Alice; Krojer, Tobias; Sethi, Ritika; Geurink, Paul P.; Aimon, Anthony; Amitai, Gabriel; Bellini, Dom; Bennett, James; Fairhead, Michael; Fedorov, Oleg; Gabizon, Ronen; Gan, Jin; Guo, Jingxu; Plotnikov, Alexander; Reznik, Nava; Ruda, Gian Filippo; Diaz-Saez, Laura; Straub, Verena M.; Szommer, Tamas; Velupillai, Srikannathasan; Zaidman, Daniel; Zhang, Yanling; Coker, Alun R.; Dowson, Christopher G.; Barr, Haim M.; Wang, Chu; Huber, Kilian V. M.; Brennan, Paul E.; Ovaa, Huib; von Delft, Frank; London, Nir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8951-8968</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Covalent probes can display unmatched potency, selectivity, and duration of action; however, their discovery is challenging.  In principle, fragments that can irreversibly bind their target can overcome the low affinity that limits reversible fragment screening, but such electrophilic fragments were considered nonselective and were rarely screened.  We hypothesized that mild electrophiles might overcome the selectivity challenge and constructed a library of 993 mildly electrophilic fragments.  We characterized this library by a new high-throughput thiol-reactivity assay and screened them against 10 cysteine-contg. proteins.  Highly reactive and promiscuous fragments were rare and could be easily eliminated.  In contrast, we found hits for most targets.  Combining our approach with high-throughput crystallog. allowed rapid progression to potent and selective probes for two enzymes, the deubiquitinase OTUB2 and the pyrophosphatase NUDT7.  No inhibitors were previously known for either.  This study highlights the potential of electrophile-fragment screening as a practical and efficient tool for covalent-ligand discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGuVq_Zb2i_7Vg90H21EOLACvtfcHk0lhp42JLEz2vOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovVKmsrg%253D&md5=18d7a00b9ce67f4f23fbb65f473cac78</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjacs.9b02822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.9b02822%26sid%3Dliteratum%253Aachs%26aulast%3DResnick%26aufirst%3DE.%26aulast%3DBradley%26aufirst%3DA.%26aulast%3DGan%26aufirst%3DJ.%26aulast%3DDouangamath%26aufirst%3DA.%26aulast%3DKrojer%26aufirst%3DT.%26aulast%3DSethi%26aufirst%3DR.%26aulast%3DGeurink%26aufirst%3DP.%2BP.%26aulast%3DAimon%26aufirst%3DA.%26aulast%3DAmitai%26aufirst%3DG.%26aulast%3DBellini%26aufirst%3DD.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DFairhead%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DGabizon%26aufirst%3DR.%26aulast%3DGan%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DPlotnikov%26aufirst%3DA.%26aulast%3DReznik%26aufirst%3DN.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DD%25C3%25ADaz-S%25C3%25A1ez%26aufirst%3DL.%26aulast%3DStraub%26aufirst%3DV.%2BM.%26aulast%3DSzommer%26aufirst%3DT.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DZaidman%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCoker%26aufirst%3DA.%2BR.%26aulast%3DDowson%26aufirst%3DC.%2BG.%26aulast%3DBarr%26aufirst%3DH.%2BM.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DOvaa%26aufirst%3DH.%26aulast%3DVon%2BDelft%26aufirst%3DF.%26aulast%3DLondon%26aufirst%3DN.%26atitle%3DRapid%2520Covalent-Probe%2520Discovery%2520by%2520Electrophile-Fragment%2520Screening%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D8951%26epage%3D8968%26doi%3D10.1021%2Fjacs.9b02822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hallenbeck, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogden, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sijbesma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renslo, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span> <span> </span><span class="NLM_article-title">A Liquid Chromatography/Mass Spectrometry Method for Screening Disulfide Tethering Fragments</span>. <i>SLAS Discov.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">183</span>– <span class="NLM_lpage">192</span>, <span class="refDoi"> DOI: 10.1177/2472555217732072</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1177%2F2472555217732072" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=28945980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFCqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=183-192&author=K.+K.+Hallenbeckauthor=J.+L.+Daviesauthor=C.+Merronauthor=P.+Ogdenauthor=E.+Sijbesmaauthor=C.+Ottmannauthor=A.+R.+Rensloauthor=C.+Wilsonauthor=M.+R.+Arkin&title=A+Liquid+Chromatography%2FMass+Spectrometry+Method+for+Screening+Disulfide+Tethering+Fragments&doi=10.1177%2F2472555217732072"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A Liquid Chromatography/Mass Spectrometry Method for Screening Disulfide Tethering Fragments</span></div><div class="casAuthors">Hallenbeck, Kenneth K.; Davies, Julia L.; Merron, Connie; Ogden, Pierce; Sijbesma, Eline; Ottmann, Christian; Renslo, Adam R.; Wilson, Christopher; Arkin, Michelle R.</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">183-192</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">We report the refinement of a high-throughput, liq. chromatog./mass spectrometry (LC/MS)-based screening method for the identification of covalent small-mol. binders to proteins.  Using a custom library of 1600 disulfide-capped fragments targeting surface cysteine residues, we optimize sample prepn., chromatog., and ionization conditions to maximize the reliability and flexibility of the approach.  Data collection at a rate of 84 s per sample balances speed with reliability for sustained screening over multiple, diverse projects run over a 24-mo period.  The method is applicable to protein targets of various classes and a range of mol. masses.  Data are processed in a custom pipeline that calcs. a percent bound value for each compd. and identifies false positives by calcg. significance of detected masses (signal significance).  An example pipeline is available through Biovia's ScienceCloud Protocol Exchange.  Data collection and anal. methods for the screening of covalent adducts of intact proteins are now fast enough to screen the largest covalent compd. libraries in 1 to 2 days.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokAicgwlw2rrVg90H21EOLACvtfcHk0ljkxPSMd-PokQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFCqt70%253D&md5=8be6b7afd4854cf0e4361d5bd6743663</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1177%2F2472555217732072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555217732072%26sid%3Dliteratum%253Aachs%26aulast%3DHallenbeck%26aufirst%3DK.%2BK.%26aulast%3DDavies%26aufirst%3DJ.%2BL.%26aulast%3DMerron%26aufirst%3DC.%26aulast%3DOgden%26aufirst%3DP.%26aulast%3DSijbesma%26aufirst%3DE.%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DRenslo%26aufirst%3DA.%2BR.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26atitle%3DA%2520Liquid%2520Chromatography%252FMass%2520Spectrometry%2520Method%2520for%2520Screening%2520Disulfide%2520Tethering%2520Fragments%26jtitle%3DSLAS%2520Discov.%26date%3D2018%26volume%3D23%26spage%3D183%26epage%3D192%26doi%3D10.1177%2F2472555217732072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">London, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eidam, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibold, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cimermančič, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnet, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span> <span> </span><span class="NLM_article-title">Covalent Docking of Large Libraries for the Discovery of Chemical Probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1066</span>– <span class="NLM_lpage">1072</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnchembio.1666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=25344815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVSqsbjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=1066-1072&author=N.+Londonauthor=R.+M.+Millerauthor=S.+Krishnanauthor=K.+Uchidaauthor=J.+J.+Irwinauthor=O.+Eidamauthor=L.+Giboldauthor=P.+Cimerman%C4%8Di%C4%8Dauthor=R.+Bonnetauthor=B.+K.+Shoichetauthor=J.+Taunton&title=Covalent+Docking+of+Large+Libraries+for+the+Discovery+of+Chemical+Probes&doi=10.1038%2Fnchembio.1666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent docking of large libraries for the discovery of chemical probes</span></div><div class="casAuthors">London, Nir; Miller, Rand M.; Krishnan, Shyam; Uchida, Kenji; Irwin, John J.; Eidam, Oliv; Gibold, Lucie; Cimermancic, Peter; Bonnet, Richard; Shoichet, Brian K.; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1066-1072</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes that form a covalent bond with a protein target often show enhanced selectivity, potency and utility for biol. studies.  Despite these advantages, protein-reactive compds. are usually avoided in high-throughput screening campaigns.  Here we describe a general method (DOCKovalent) for screening large virtual libraries of electrophilic small mols.  We apply this method prospectively to discover reversible covalent fragments that target distinct protein nucleophiles, including the catalytic serine of AmpC β-lactamase and noncatalytic cysteines in RSK2, MSK1 and JAK3 kinases.  We identify submicromolar to low-nanomolar hits with high ligand efficiency, cellular activity and selectivity, including what are to our knowledge the first reported reversible covalent inhibitors of JAK3.  Crystal structures of inhibitor complexes with AmpC and RSK2 confirm the docking predictions and guide further optimization.  As covalent virtual screening may have broad utility for the rapid discovery of chem. probes, we have made the method freely available through an automated web server (http://covalent.docking.org/).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpngQdIYqslNLVg90H21EOLACvtfcHk0ljkxPSMd-PokQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVSqsbjE&md5=4e5fbeb8f67cb82d464d57069ba0c93a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1666%26sid%3Dliteratum%253Aachs%26aulast%3DLondon%26aufirst%3DN.%26aulast%3DMiller%26aufirst%3DR.%2BM.%26aulast%3DKrishnan%26aufirst%3DS.%26aulast%3DUchida%26aufirst%3DK.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DEidam%26aufirst%3DO.%26aulast%3DGibold%26aufirst%3DL.%26aulast%3DCimerman%25C4%258Di%25C4%258D%26aufirst%3DP.%26aulast%3DBonnet%26aufirst%3DR.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DCovalent%2520Docking%2520of%2520Large%2520Libraries%2520for%2520the%2520Discovery%2520of%2520Chemical%2520Probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D1066%26epage%3D1072%26doi%3D10.1038%2Fnchembio.1666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sijbesma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallenbeck, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leysen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Vink, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skóra, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Site-Directed Fragment-Based Screening for the Discovery of Protein-Protein Interaction Stabilizers</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">3524</span>– <span class="NLM_lpage">3531</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b11658</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b11658" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVWjtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=3524-3531&author=E.+Sijbesmaauthor=K.+K.+Hallenbeckauthor=S.+Leysenauthor=P.+J.+De+Vinkauthor=L.+Sk%C3%B3raauthor=W.+Jahnkeauthor=L.+Brunsveldauthor=M.+R.+Arkinauthor=C.+Ottmann&title=Site-Directed+Fragment-Based+Screening+for+the+Discovery+of+Protein-Protein+Interaction+Stabilizers&doi=10.1021%2Fjacs.8b11658"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Site-Directed Fragment-Based Screening for the Discovery of Protein-Protein Interaction Stabilizers</span></div><div class="casAuthors">Sijbesma, Eline; Hallenbeck, Kenneth K.; Leysen, Seppe; de Vink, Pim J.; Skora, Lukasz; Jahnke, Wolfgang; Brunsveld, Luc; Arkin, Michelle R.; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3524-3531</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modulation of protein-protein interactions (PPIs) by small mols. has emerged as a valuable approach in drug discovery.  Compared to direct inhibition, PPI stabilization is vastly underexplored but has strong advantages, including the ability to gain selectivity by targeting an interface formed only upon assocn. of proteins.  Here, we present the application of a site-directed screening technique based on disulfide trapping (tethering) to select for fragments that enhance the affinity between protein partners.  We target the phosphorylation-dependent interaction between the hub protein 14-3-3σ and a peptide derived from Estrogen Receptor α (ERα), an important breast cancer target that is neg. regulated by 14-3-3σ.  We identify orthosteric stabilizers that increase 14-3-3/ERα affinity up to 40-fold and propose the mechanism of stabilization based on X-ray crystal structures.  These fragments already display partial selectivity toward ERα-like motifs over other representative 14-3-3 clients.  This first of its kind study illustrates the potential of the tethering approach to overcome the hurdles in systematic PPI stabilizer discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1_kzNxRmRPrVg90H21EOLACvtfcHk0lh8YuVKML39pA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVWjtrY%253D&md5=5e68f397675437fffdca4cc47ed8e078</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b11658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b11658%26sid%3Dliteratum%253Aachs%26aulast%3DSijbesma%26aufirst%3DE.%26aulast%3DHallenbeck%26aufirst%3DK.%2BK.%26aulast%3DLeysen%26aufirst%3DS.%26aulast%3DDe%2BVink%26aufirst%3DP.%2BJ.%26aulast%3DSk%25C3%25B3ra%26aufirst%3DL.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DSite-Directed%2520Fragment-Based%2520Screening%2520for%2520the%2520Discovery%2520of%2520Protein-Protein%2520Interaction%2520Stabilizers%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D3524%26epage%3D3531%26doi%3D10.1021%2Fjacs.8b11658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sijbesma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somsen, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miley, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leijten-van
de Gevel, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Fluorescence Anisotropy-Based Tethering for Discovery of Protein–Protein Interaction Stabilizers</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_fpage">3143</span>– <span class="NLM_lpage">3148</span>, <span class="refDoi"> DOI: 10.1021/acschembio.0c00646</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.0c00646" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitlagtLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=3143-3148&author=E.+Sijbesmaauthor=B.+A.+Somsenauthor=G.+P.+Mileyauthor=I.+A.+Leijten-van%0Ade+Gevelauthor=L.+Brunsveldauthor=M.+R.+Arkinauthor=C.+Ottmann&title=Fluorescence+Anisotropy-Based+Tethering+for+Discovery+of+Protein%E2%80%93Protein+Interaction+Stabilizers&doi=10.1021%2Facschembio.0c00646"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence Anisotropy-Based Tethering for Discovery of Protein-Protein Interaction Stabilizers</span></div><div class="casAuthors">Sijbesma, Eline; Somsen, Bente A.; Miley, Galen P.; Leijten-van de Gevel, Iris A.; Brunsveld, Luc; Arkin, Michelle R.; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3143-3148</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-protein interaction (PPI) networks are fundamental for cellular processes.  Small-mol. PPI enhancers have been shown to be powerful tools to fundamentally study PPIs and as starting points for potential new therapeutics.  Yet, systematic approaches for their discovery are not widely available, and the design prerequisites of "mol. glues" are poorly understood.  Covalent fragment-based screening can identify chem. starting points for these enhancers at specific sites in PPI interfaces.  We recently reported a mass spectrometry-based disulfide-trapping (tethering) approach for a cysteine residue in the hub protein 14-3-3, an important regulator of phosphorylated client proteins.  Here, we invert the strategy and report the development of a functional read-out for systematic identification of PPI enhancers based on fluorescence anisotropy (FA-tethering) with the reactive handle now on a client-derived peptide.  Using the DNA-binding domain of the nuclear receptor Estrogen Related Receptor gamma (ERRγ), we target a native cysteine positioned at the 14-3-3 PPI interface and identify several fragments that form a disulfide bond to ERRγ and stabilize the complex up to 5-fold.  Crystallog. indicates that fragments bind in a pocket comprised of 14-3-3 and the ERRγ phosphopeptide.  FA-tethering presents a streamlined methodol. to discover mol. glues for protein complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoey2CDD3wVnLVg90H21EOLACvtfcHk0ljNIfXUTyUpMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitlagtLnM&md5=0c6220dfa4281bdb7f196402ef983a57</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facschembio.0c00646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.0c00646%26sid%3Dliteratum%253Aachs%26aulast%3DSijbesma%26aufirst%3DE.%26aulast%3DSomsen%26aufirst%3DB.%2BA.%26aulast%3DMiley%26aufirst%3DG.%2BP.%26aulast%3DLeijten-van%2Bde%2BGevel%26aufirst%3DI.%2BA.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DFluorescence%2520Anisotropy-Based%2520Tethering%2520for%2520Discovery%2520of%2520Protein%25E2%2580%2593Protein%2520Interaction%2520Stabilizers%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26spage%3D3143%26epage%3D3148%26doi%3D10.1021%2Facschembio.0c00646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valenti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cossar, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hristeva, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzalis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Stabilizers of Protein–Protein Interactions through Imine-Based Tethering</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">21520</span>– <span class="NLM_lpage">21524</span>, <span class="refDoi"> DOI: 10.1002/anie.202008585</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1002%2Fanie.202008585" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFWmtL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=21520-21524&author=M.+Wolterauthor=D.+Valentiauthor=P.+J.+Cossarauthor=L.+M.+Levyauthor=S.+Hristevaauthor=T.+Genskiauthor=T.+Hoffmannauthor=L.+Brunsveldauthor=D.+Tzalisauthor=C.+Ottmann&title=Fragment-Based+Stabilizers+of+Protein%E2%80%93Protein+Interactions+through+Imine-Based+Tethering&doi=10.1002%2Fanie.202008585"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Stabilizers of Protein-Protein Interactions through Imine-Based Tethering</span></div><div class="casAuthors">Wolter, Madita; Valenti, Dario; Cossar, Peter J.; Levy, Laura M.; Hristeva, Stanimira; Genski, Thorsten; Hoffmann, Torsten; Brunsveld, Luc; Tzalis, Dimitrios; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">21520-21524</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Small-mol. stabilization of protein-protein interactions (PPIs) is a promising concept in drug discovery, however the question how to identify or design chem. starting points in a "bottom-up" approach is largely unanswered.  We report a novel concept for identifying initial chem. matter for PPI stabilization based on imine-forming fragments.  The imine bond offers a covalent anchor for site-directed fragment targeting, whereas its transient nature enables efficient anal. of structure-activity relationships.  This bond enables fragment identification and optimization using protein crystallog.  We report novel fragments that bind specifically to a lysine at the PPI interface of the p65-subunit-derived peptide of NF-κB with the adapter protein 14-3-3.  Those fragments that subsequently establish contacts with the p65-derived peptide, rather than with 14-3-3, efficiently stabilize the 14-3-3/p65 complex and offer novel starting points for mol. glues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV4so8tUhVerVg90H21EOLACvtfcHk0ljNIfXUTyUpMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFWmtL7F&md5=8a1e3e6d8f989a704f3d5c8213bfeeb3</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fanie.202008585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.202008585%26sid%3Dliteratum%253Aachs%26aulast%3DWolter%26aufirst%3DM.%26aulast%3DValenti%26aufirst%3DD.%26aulast%3DCossar%26aufirst%3DP.%2BJ.%26aulast%3DLevy%26aufirst%3DL.%2BM.%26aulast%3DHristeva%26aufirst%3DS.%26aulast%3DGenski%26aufirst%3DT.%26aulast%3DHoffmann%26aufirst%3DT.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DTzalis%26aufirst%3DD.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DFragment-Based%2520Stabilizers%2520of%2520Protein%25E2%2580%2593Protein%2520Interactions%2520through%2520Imine-Based%2520Tethering%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2020%26volume%3D59%26spage%3D21520%26epage%3D21524%26doi%3D10.1002%2Fanie.202008585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span> <span> </span><span class="NLM_article-title">Lysine-Targeting Covalent Inhibitors</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">15200</span>– <span class="NLM_lpage">15209</span>, <span class="refDoi"> DOI: 10.1002/anie.201707630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1002%2Fanie.201707630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslejtr3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=15200-15209&author=J.+Pettingerauthor=K.+Jonesauthor=M.+D.+Cheeseman&title=Lysine-Targeting+Covalent+Inhibitors&doi=10.1002%2Fanie.201707630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-Targeting Covalent Inhibitors</span></div><div class="casAuthors">Pettinger, Jonathan; Jones, Keith; Cheeseman, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">15200-15209</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Targeted covalent inhibitors have gained widespread attention in drug discovery as a validated method to circumvent acquired resistance in oncol.  This strategy exploits small-mol./protein crystal structures to design tightly binding ligands with appropriately positioned electrophilic warheads.  While most focus has been on targeting binding-site cysteine residues, targeting nucleophilic lysine residues can also represent a viable approach to irreversible inhibition.  However, owing to the basicity of the ε-amino group in lysine, this strategy generates a no. of specific challenges.  Herein, the authors review the key principles for inhibitor design, give historical examples, and present recent developments that demonstrate the potential of lysine targeting for future drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUIzd2aDc06bVg90H21EOLACvtfcHk0ljeHzVsBFNAEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslejtr3P&md5=4a6bb561e53c165fa3c7c85953bb4736</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fanie.201707630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201707630%26sid%3Dliteratum%253Aachs%26aulast%3DPettinger%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26atitle%3DLysine-Targeting%2520Covalent%2520Inhibitors%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D15200%26epage%3D15209%26doi%3D10.1002%2Fanie.201707630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span> <span> </span><span class="NLM_article-title">Kinetic Optimization of Lysine-Targeting Covalent Inhibitors of HSP72</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11383</span>– <span class="NLM_lpage">11398</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01709</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01709" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFKksLrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11383-11398&author=J.+Pettingerauthor=M.+Carterauthor=K.+Jonesauthor=M.+D.+Cheeseman&title=Kinetic+Optimization+of+Lysine-Targeting+Covalent+Inhibitors+of+HSP72&doi=10.1021%2Facs.jmedchem.9b01709"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Optimization of Lysine-Targeting Covalent Inhibitors of HSP72</span></div><div class="casAuthors">Pettinger, Jonathan; Carter, Michael; Jones, Keith; Cheeseman, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11383-11398</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The covalent inhibition mechanism of action, which overcomes competition with high-affinity, high-abundance substrates of challenging protein targets, can deliver effective chem. probes and drugs.  The success of this strategy has centered on exposed cysteine residues as nucleophiles but the low abundance of cysteine in the proteome has limited its application.  We have recently reported our discovery that lysine-56 in the difficult-to-drug target HSP72 could form a covalent bond with a small-mol. inhibitor.  We now disclose the optimization of these targeted covalent inhibitors using rational design.  Essential to our optimization was the development of a new covalent fluorescence polarization assay, which allows for the direct measurement of the key kinetic parameter in covalent inhibitor design, kinact/KI, extrapolation of the underlying parameters, kinact and Ki, and direct comparison to reversible analogs.  Using our approach, we demonstrate a >100-fold enhancement in covalent efficiency and key learnings in lysine-selective electrophile optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK27fPoDap37Vg90H21EOLACvtfcHk0ljeHzVsBFNAEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFKksLrI&md5=6c02be025216bdf8d031dbebbe126456</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01709%26sid%3Dliteratum%253Aachs%26aulast%3DPettinger%26aufirst%3DJ.%26aulast%3DCarter%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26atitle%3DKinetic%2520Optimization%2520of%2520Lysine-Targeting%2520Covalent%2520Inhibitors%2520of%2520HSP72%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11383%26epage%3D11398%26doi%3D10.1021%2Facs.jmedchem.9b01709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Bihan, Y.-V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Montfort, R. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span> <span> </span><span class="NLM_article-title">An Irreversible Inhibitor of HSP72 That Unexpectedly Targets Lysine-56</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">3536</span>– <span class="NLM_lpage">3540</span>, <span class="refDoi"> DOI: 10.1002/anie.201611907</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1002%2Fanie.201611907" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVeru7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=3536-3540&author=J.+Pettingerauthor=Y.-V.+Le+Bihanauthor=M.+Widyaauthor=R.+L.+M.+van+Montfortauthor=K.+Jonesauthor=M.+D.+Cheeseman&title=An+Irreversible+Inhibitor+of+HSP72+That+Unexpectedly+Targets+Lysine-56&doi=10.1002%2Fanie.201611907"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">An Irreversible Inhibitor of HSP72 that Unexpectedly Targets Lysine-56</span></div><div class="casAuthors">Pettinger, Jonathan; Le Bihan, Yann-Vai; Widya, Marcella; van Montfort, Rob L. M.; Jones, Keith; Cheeseman, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3536-3540</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The stress-inducible mol. chaperone, HSP72, is an important therapeutic target in oncol., but inhibiting this protein with small mols. has proven particularly challenging.  Validating HSP72 inhibitors in cells is difficult owing to competition with the high affinity and abundance of its endogenous nucleotide substrates.  We hypothesized this could be overcome using a cysteine-targeted irreversible inhibitor.  Using rational design, we adapted a validated 8-N-benzyladenosine ligand for covalent bond formation and confirmed targeted irreversible inhibition.  However, no cysteine in the protein was modified; instead, we demonstrate that lysine-56 is the key nucleophilic residue.  Targeting this lysine could lead to a new design paradigm for HSP72 chem. probes and drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0a7TsyW4PurVg90H21EOLACvtfcHk0lgVgneyy4gPzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtVeru7o%253D&md5=8b2cbd17140e5f0c6430d0164e03280a</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611907&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611907%26sid%3Dliteratum%253Aachs%26aulast%3DPettinger%26aufirst%3DJ.%26aulast%3DLe%2BBihan%26aufirst%3DY.-V.%26aulast%3DWidya%26aufirst%3DM.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%2BM.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26atitle%3DAn%2520Irreversible%2520Inhibitor%2520of%2520HSP72%2520That%2520Unexpectedly%2520Targets%2520Lysine-56%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D3536%26epage%3D3540%26doi%3D10.1002%2Fanie.201611907" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalton, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittus, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Convery, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bush, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span> <span> </span><span class="NLM_article-title">Selectively Targeting the Kinome-Conserved Lysine of PI3Kδ as a General Approach to Covalent Kinase Inhibition</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">932</span>– <span class="NLM_lpage">939</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b08979</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b08979" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFKitrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=932-939&author=S.+E.+Daltonauthor=L.+Dittusauthor=D.+A.+Thomasauthor=M.+A.+Converyauthor=J.+Nunesauthor=J.+T.+Bushauthor=J.+P.+Evansauthor=T.+Wernerauthor=M.+Bantscheffauthor=J.+A.+Murphyauthor=S.+Campos&title=Selectively+Targeting+the+Kinome-Conserved+Lysine+of+PI3K%CE%B4+as+a+General+Approach+to+Covalent+Kinase+Inhibition&doi=10.1021%2Fjacs.7b08979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Selectively Targeting the Kinome-Conserved Lysine of PI3Kδ as a General Approach to Covalent Kinase Inhibition</span></div><div class="casAuthors">Dalton, Samuel E.; Dittus, Lars; Thomas, Daniel A.; Convery, Maire A.; Nunes, Joao; Bush, Jacob T.; Evans, John P.; Werner, Thilo; Bantscheff, Marcus; Murphy, John A.; Campos, Sebastien</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">932-939</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective covalent inhibition of kinases by targeting poorly conserved cysteines has proven highly fruitful to date in the development of chem. probes and approved drugs.  However, this approach is limited to ∼200 kinases possessing such a cysteine near the ATP-binding pocket.  Herein, we report a novel approach to achieve selective, irreversible kinase inhibition, by targeting the conserved catalytic lysine residue.  We have illustrated our approach by developing selective, covalent PI3Kδ inhibitors that exhibit nanomolar potency in cellular assays, and a duration of action >48 h in CD4+ T cells.  Despite conservation of the lysine residue throughout the kinome, the lead compd. shows high levels of selectivity over a selection of lipid and protein kinases in biochem. assays, as well as covalent binding to very few off-target proteins in live-cell proteomic studies.  We anticipate this approach could offer a general strategy, as an alternative to targeting non-conserved cysteines, for the development of selective covalent kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGvT7h2D9zf7Vg90H21EOLACvtfcHk0lgVgneyy4gPzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFKitrrK&md5=90bb399303a942fb87f1b9096eb8bdc6</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b08979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b08979%26sid%3Dliteratum%253Aachs%26aulast%3DDalton%26aufirst%3DS.%2BE.%26aulast%3DDittus%26aufirst%3DL.%26aulast%3DThomas%26aufirst%3DD.%2BA.%26aulast%3DConvery%26aufirst%3DM.%2BA.%26aulast%3DNunes%26aufirst%3DJ.%26aulast%3DBush%26aufirst%3DJ.%2BT.%26aulast%3DEvans%26aufirst%3DJ.%2BP.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DMurphy%26aufirst%3DJ.%2BA.%26aulast%3DCampos%26aufirst%3DS.%26atitle%3DSelectively%2520Targeting%2520the%2520Kinome-Conserved%2520Lysine%2520of%2520PI3K%25CE%25B4%2520as%2520a%2520General%2520Approach%2520to%2520Covalent%2520Kinase%2520Inhibition%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D932%26epage%3D939%26doi%3D10.1021%2Fjacs.7b08979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, I. M.</span></span> <span> </span><span class="NLM_article-title">NF-ΚB Regulation in the Immune System</span>. <i>Nat. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">725</span>– <span class="NLM_lpage">734</span>, <span class="refDoi"> DOI: 10.1038/nri910</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnri910" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=12360211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsVGqsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=725-734&author=Q.+Liauthor=I.+M.+Verma&title=NF-%CE%9AB+Regulation+in+the+Immune+System&doi=10.1038%2Fnri910"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">NF-κB regulation in the immune system</span></div><div class="casAuthors">Li, Qiutang; Verma, Inder M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">725-734</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The nuclear factor-κB (NF-κB)/REL family of transcription factors has a central role in coordinating the expression of a wide variety of genes that control immune responses.  There has been intense scientific activity in the NF-κB field owing to the involvement of these factors in the activation and regulation of key mols. that are assocd. with diseases ranging from inflammation to cancer.  Here, the authors focus on the current understanding of NF-κB regulation and its role in the immune system and inflammatory diseases.  The authors also discuss the role of NF-κB proteins as potential therapeutic targets in clin. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAP3VO2ff1orVg90H21EOLACvtfcHk0lh5hwe3hDiCIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsVGqsbg%253D&md5=5dbb972341f6e063df6d46389217f88e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnri910&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri910%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DVerma%26aufirst%3DI.%2BM.%26atitle%3DNF-%25CE%259AB%2520Regulation%2520in%2520the%2520Immune%2520System%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2002%26volume%3D2%26spage%3D725%26epage%3D734%26doi%3D10.1038%2Fnri910" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakanishi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toi, M.</span></span> <span> </span><span class="NLM_article-title">Nuclear Factor-ΚB Inhibitors as Sensitizers to Anticancer Drugs</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">297</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.1038/nrc1588</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnrc1588" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=15803156" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivVars7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2005&pages=297-309&author=C.+Nakanishiauthor=M.+Toi&title=Nuclear+Factor-%CE%9AB+Inhibitors+as+Sensitizers+to+Anticancer+Drugs&doi=10.1038%2Fnrc1588"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Nuclear factor-κB inhibitors as sensitizers to anticancer drugs</span></div><div class="casAuthors">Nakanishi, Chikashi; Toi, Masakazu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">297-309</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The cytotoxicity of chemotherapeutic agents is attributed to apoptosis.  Acquired resistance to the effects of chemotherapy has emerged as a significant impediment to effective cancer therapy.  One feature that cytotoxic treatments of cancer have in common is their activation of the transcription factor nuclear factor-κB (NF-κB), which regulates cell survival.  NF-κB activation suppresses the apoptotic potential of chemotherapeutic agents and contributes to resistance.  What evidence is there that inhibitors of NF-κB might promote apoptosis in cancer cells and can NF-κB inhibitors be used to overcome resistance to chemotherapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZsXlm8lIyTbVg90H21EOLACvtfcHk0lh5hwe3hDiCIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivVars7Y%253D&md5=4449832c63d81112812502b97d2fdaea</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnrc1588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1588%26sid%3Dliteratum%253Aachs%26aulast%3DNakanishi%26aufirst%3DC.%26aulast%3DToi%26aufirst%3DM.%26atitle%3DNuclear%2520Factor-%25CE%259AB%2520Inhibitors%2520as%2520Sensitizers%2520to%2520Anticancer%2520Drugs%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2005%26volume%3D5%26spage%3D297%26epage%3D309%26doi%3D10.1038%2Fnrc1588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karin, M.</span></span> <span> </span><span class="NLM_article-title">NF-κB, Inflammation, Immunity and Cancer: Coming of Age</span>. <i>Nat. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">309</span>– <span class="NLM_lpage">324</span>, <span class="refDoi"> DOI: 10.1038/nri.2017.142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fnri.2017.142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=29379212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yjtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=309-324&author=K.+Taniguchiauthor=M.+Karin&title=NF-%CE%BAB%2C+Inflammation%2C+Immunity+and+Cancer%3A+Coming+of+Age&doi=10.1038%2Fnri.2017.142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">NF-κB, inflammation, immunity and cancer: coming of age</span></div><div class="casAuthors">Taniguchi, Koji; Karin, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">309-324</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Fourteen years have passed since nuclear factor-κB (NF-κB) was first shown to serve as a mol. lynchpin that links persistent infections and chronic inflammation to increased cancer risk.  The young field of inflammation and cancer has now come of age, and inflammation has been recognized by the broad cancer research community as a hallmark and cause of cancer.  Here, we discuss how the initial discovery of a role for NF-κB in linking inflammation and cancer led to an improved understanding of tumor-elicited inflammation and its effects on anticancer immunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbWVUEqQxv9rVg90H21EOLACvtfcHk0lh5hwe3hDiCIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yjtrk%253D&md5=02d3c489732baab402ddf87d18416bd6</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnri.2017.142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri.2017.142%26sid%3Dliteratum%253Aachs%26aulast%3DTaniguchi%26aufirst%3DK.%26aulast%3DKarin%26aufirst%3DM.%26atitle%3DNF-%25CE%25BAB%252C%2520Inflammation%252C%2520Immunity%2520and%2520Cancer%253A%2520Coming%2520of%2520Age%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2018%26volume%3D18%26spage%3D309%26epage%3D324%26doi%3D10.1038%2Fnri.2017.142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arepalli, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, J.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span> <span> </span><span class="NLM_article-title">Novel NF-κB Inhibitors: A Patent Review (2011 – 2014)</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">334</span>, <span class="refDoi"> DOI: 10.1517/13543776.2014.998199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1517%2F13543776.2014.998199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=25553724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2MXit12ltLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=319-334&author=S.+K.+Arepalliauthor=M.+Choiauthor=J.-K.+Jungauthor=H.+Lee&title=Novel+NF-%CE%BAB+Inhibitors%3A+A+Patent+Review+%282011+%E2%80%93+2014%29&doi=10.1517%2F13543776.2014.998199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Novel NF-κB inhibitors: a patent review (2011 - 2014)</span></div><div class="casAuthors">Arepalli, Sateesh Kumar; Choi, Minho; Jung, Jae-Kyung; Lee, Heesoon</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">319-334</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Importance of the field: The NF-κB family of transcription factors has a vital role in coordinating the expression of a wide variety of genes that control immune responses.  The dysfunction of NF-κB has been assocd. with diseases ranging from inflammation to cancer.  Areas covered: In this review, authors provided information on the role of NF-κB in human health and on recently developed inhibitors from patents granted in 2011 - 2014.  The reader will gain an understanding of the mechanisms of NF-κB function as well as the structures and biol. activities of recently developed NF-κB inhibitors.  Expert opinion: Constitutive NF-κB activation is likely involved in the pathogenesis of several diseases.  Researchers need to consider the specific mechanism of each invented NF-κB inhibitor, as well as the properties of the cellular, in vitro and in vivo assays and targeted diseases.  For new drug development, they need various biol. tests to assess the potential side effects and optimization of the biol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4fFLw8dJ5jLVg90H21EOLACvtfcHk0li29Jaam7ptBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXit12ltLk%253D&md5=59349fb2f77470bbdc0045ccf64257ac</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1517%2F13543776.2014.998199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2014.998199%26sid%3Dliteratum%253Aachs%26aulast%3DArepalli%26aufirst%3DS.%2BK.%26aulast%3DChoi%26aufirst%3DM.%26aulast%3DJung%26aufirst%3DJ.-K.%26aulast%3DLee%26aufirst%3DH.%26atitle%3DNovel%2520NF-%25CE%25BAB%2520Inhibitors%253A%2520A%2520Patent%2520Review%2520%25282011%2520%25E2%2580%2593%25202014%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2015%26volume%3D25%26spage%3D319%26epage%3D334%26doi%3D10.1517%2F13543776.2014.998199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaltschmidt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greiner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadhim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaltschmidt, C.</span></span> <span> </span><span class="NLM_article-title">Subunit-Specific Role of NF-κB in Cancer</span>. <i>Biomedicines</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">44</span>, <span class="refDoi"> DOI: 10.3390/biomedicines6020044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.3390%2Fbiomedicines6020044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvFaju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=44&author=B.+Kaltschmidtauthor=J.+Greinerauthor=H.+Kadhimauthor=C.+Kaltschmidt&title=Subunit-Specific+Role+of+NF-%CE%BAB+in+Cancer&doi=10.3390%2Fbiomedicines6020044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Subunit-specific role of NF-κB in cancer</span></div><div class="casAuthors">Kaltschmidt, Barbara; Greiner, Johannes F. W.; Kadhim, Hussamadin M.; Kaltschmidt, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Biomedicines</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">44/1-44/12</span>CODEN:
                <span class="NLM_cas:coden">BIOMID</span>;
        ISSN:<span class="NLM_cas:issn">2227-9059</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  The transcription factor NF-κ B is a key player in inflammation, cancer development, and progression.  NF-κ B stimulates cell proliferation, prevents apoptosis, and could promote tumor angiogenesis as well as metastasis.  Extending the commonly accepted role of NF-κ B in cancer formation and progression, different NF-κ B subunits have been shown to be active and of particular importance in distinct types of cancer.  Here, we summarize overexpression data of the NF-κ B subunits RELA, RELB, and c-REL (referring to the v-REL, which is the oncogene of Reticuloendotheliosis virus strain T) as well as of their upstream kinase inhibitor, namely inhibitor of κ B kinases (IKK), in different human cancers, assessed by database mining.  These data argue against a universal mechanism of cancer-mediated activation of NF-κ B, and suggest a much more elaborated mode of NF-κ B regulation, indicating a tumor type-specific upregulation of the NF-κ B subunits.  We further discuss recent findings showing the diverse roles of NF-κ B signaling in cancer development and metastasis in a subunit-specific manner, emphasizing their specific transcriptional activity and the role of autoregulation.  While non-canonical NF-κ B RELB signaling is described to be mostly present in hematol. cancers, solid cancers reveal constitutive canonical NF-κ B RELA or c-REL activity.  Providing a linkage to cancer therapy, we discuss the recently described pivotal role of NF-κ B c-REL in regulating cancer-targeting immune responses.  In addn., current strategies and ongoing clin. trials are summarized, which utilize genome editing or drugs to inhibit the NF-κ FB subunits for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrjn-IQVNxD7Vg90H21EOLACvtfcHk0li29Jaam7ptBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvFaju7c%253D&md5=53c399ab25acd3881d6c274f80e27dda</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.3390%2Fbiomedicines6020044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiomedicines6020044%26sid%3Dliteratum%253Aachs%26aulast%3DKaltschmidt%26aufirst%3DB.%26aulast%3DGreiner%26aufirst%3DJ.%26aulast%3DKadhim%26aufirst%3DH.%26aulast%3DKaltschmidt%26aufirst%3DC.%26atitle%3DSubunit-Specific%2520Role%2520of%2520NF-%25CE%25BAB%2520in%2520Cancer%26jtitle%3DBiomedicines%26date%3D2018%26volume%3D6%26spage%3D44%26doi%3D10.3390%2Fbiomedicines6020044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, D. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, C.</span></span> <span> </span><span class="NLM_article-title">14-3-3 Isoforms Differentially Regulate NFκB Signaling in the Brain After Ischemia-Reperfusion</span>. <i>Neurochem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">2354</span>– <span class="NLM_lpage">2362</span>, <span class="refDoi"> DOI: 10.1007/s11064-017-2255-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1007%2Fs11064-017-2255-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=28424948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtlalsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2017&pages=2354-2362&author=X.+Zhouauthor=D.+X.+Huauthor=R.+Q.+Chenauthor=X.+Q.+Chenauthor=W.+Dongauthor=C.+Yi&title=14-3-3+Isoforms+Differentially+Regulate+NF%CE%BAB+Signaling+in+the+Brain+After+Ischemia-Reperfusion&doi=10.1007%2Fs11064-017-2255-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">14-3-3 Isoforms Differentially Regulate NFκB Signaling in the Brain After Ischemia-Reperfusion</span></div><div class="casAuthors">Zhou, Xin-yu; Hu, Dian Xing; Chen, Ruo Qiao; Chen, Xiao Qian; Dong, Wan-li; Yi, Cheng-la</div><div class="citationInfo"><span class="NLM_cas:title">Neurochemical Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2354-2362</span>CODEN:
                <span class="NLM_cas:coden">NEREDZ</span>;
        ISSN:<span class="NLM_cas:issn">0364-3190</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Mammalian 14-3-3 isoforms exist predominantly in the brain and are heavily involved in neurol. diseases.  However, the isoform-specific role of 14-3-3 proteins in the brain remains largely unclear.  Here, we investigated the role of 14-3-3 isoforms in rat brains after transient middle cerebral artery occlusion and reperfusion.  14-3-3β, η, γ And ζ but not ε or τ were selectively upregulated in cerebral cortical neurons after ischemia-reperfusion (I/R).  Selectively, 14-3-3β, γ and ζ were translocated from cytoplasm into the nuclei of neurons after I/R.  14-3-3 Bound to p65 and suppressed p65 expression in N2a cells.  In the brain, 14-3-3 could either colocalize with p65 in the nuclei of neurons or segregate from p65 expression in cortical neurons after I/R.  All evidence together suggests that 14-3-3 isoforms are differentially induced to enter into the nuclei of neurons after I/R, which might regulate NFκB signaling directly or indirectly.  Since 14-3-3 proteins are essential for cell survival and NFκB is a key transcriptional factor, our data suggest that the 14-3-3/p65 signaling pathway might be a potential therapeutic target for stroke.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSZBOBLj9SuLVg90H21EOLACvtfcHk0lg0-r9W81pm6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtlalsLk%253D&md5=d6f800fa4dcef5b1a776832138605862</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1007%2Fs11064-017-2255-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11064-017-2255-3%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DD.%2BX.%26aulast%3DChen%26aufirst%3DR.%2BQ.%26aulast%3DChen%26aufirst%3DX.%2BQ.%26aulast%3DDong%26aufirst%3DW.%26aulast%3DYi%26aufirst%3DC.%26atitle%3D14-3-3%2520Isoforms%2520Differentially%2520Regulate%2520NF%25CE%25BAB%2520Signaling%2520in%2520the%2520Brain%2520After%2520Ischemia-Reperfusion%26jtitle%3DNeurochem.%2520Res.%26date%3D2017%26volume%3D42%26spage%3D2354%26epage%3D2362%26doi%3D10.1007%2Fs11064-017-2255-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Inglés-Esteve, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalmases, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Carbonell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jené-Sanz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Bigas, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iglesias, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Herguido, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rovira, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albanell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomis, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espinosa, L.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Specific NF-κB Activity Contributes to the Tumor Suppressor Function of 14-3-3σ in Breast Cancer</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e38347</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0038347</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1371%2Fjournal.pone.0038347" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=22675457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC38XosVenurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&author=J.+Ingl%C3%A9s-Esteveauthor=M.+Moralesauthor=A.+Dalmasesauthor=R.+Garcia-Carbonellauthor=A.+Jen%C3%A9-Sanzauthor=N.+L%C3%B3pez-Bigasauthor=M.+Iglesiasauthor=C.+Ruiz-Herguidoauthor=A.+Roviraauthor=F.+Rojoauthor=J.+Albanellauthor=R.+R.+Gomisauthor=A.+Bigasauthor=L.+Espinosa&title=Inhibition+of+Specific+NF-%CE%BAB+Activity+Contributes+to+the+Tumor+Suppressor+Function+of+14-3-3%CF%83+in+Breast+Cancer&doi=10.1371%2Fjournal.pone.0038347"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of specific NF-κB activity contributes to the tumor suppressor function of 14-3-3σ in breast cancer</span></div><div class="casAuthors">Ingles-Esteve, Julia; Morales, Monica; Dalmases, Alba; Garcia-Carbonell, Ricard; Jene-Sanz, Alba; Lopez-Bigas, Nuria; Iglesias, Mar; Ruiz-Herguido, Cristina; Rovira, Ana; Rojo, Federico; Albanell, Joan; Gomis, Roger R.; Bigas, Anna; Espinosa, Lluis</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e38347</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">14-3-3σ Is frequently lost in human breast cancers by genetic deletion or promoter methylation.  We have now investigated the involvement of 14-3-3σ in the termination of NF-κB signal in mammary cells and its putative role in cancer relapse and metastasis.  Our results show that 14-3-3σ regulates nuclear export of p65-NF-κB following chronic TNFα stimulation.  Restoration of 14-3-3σ in breast cancer cells reduces migration capacity and metastatic abilities in vivo.  By microarray anal., we have identified a genetic signature that responds to TNFα in a 14-3-3σ-dependent manner and significantly assocs. with different breast and other types of cancer.  By interrogating public databases, we have found that over-expression of this signature correlates with poor relapse-free survival in breast cancer patients.  Finally, screening of 96 human breast tumors showed that NF-κB activation strictly correlates with the absence of 14-3-3σ and it is significantly assocd. with worse prognosis in the multivariate anal.  Our findings identify a genetic signature that is important for breast cancer prognosis and for future personalized treatments based on NF-κB targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIDCBuaY4ofrVg90H21EOLACvtfcHk0lg0-r9W81pm6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XosVenurw%253D&md5=ce325cadb08b4f675e78804dd458777d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0038347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0038347%26sid%3Dliteratum%253Aachs%26aulast%3DIngl%25C3%25A9s-Esteve%26aufirst%3DJ.%26aulast%3DMorales%26aufirst%3DM.%26aulast%3DDalmases%26aufirst%3DA.%26aulast%3DGarcia-Carbonell%26aufirst%3DR.%26aulast%3DJen%25C3%25A9-Sanz%26aufirst%3DA.%26aulast%3DL%25C3%25B3pez-Bigas%26aufirst%3DN.%26aulast%3DIglesias%26aufirst%3DM.%26aulast%3DRuiz-Herguido%26aufirst%3DC.%26aulast%3DRovira%26aufirst%3DA.%26aulast%3DRojo%26aufirst%3DF.%26aulast%3DAlbanell%26aufirst%3DJ.%26aulast%3DGomis%26aufirst%3DR.%2BR.%26aulast%3DBigas%26aufirst%3DA.%26aulast%3DEspinosa%26aufirst%3DL.%26atitle%3DInhibition%2520of%2520Specific%2520NF-%25CE%25BAB%2520Activity%2520Contributes%2520to%2520the%2520Tumor%2520Suppressor%2520Function%2520of%252014-3-3%25CF%2583%2520in%2520Breast%2520Cancer%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26doi%3D10.1371%2Fjournal.pone.0038347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aguilera, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Majada, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inglés-Esteve, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodilla, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espinosa, L.</span></span> <span> </span><span class="NLM_article-title">Efficient Nuclear Export of P65-IκBα Complexes Requires 14-3-3 Proteins</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">2472</span>– <span class="NLM_lpage">2472</span>, <span class="refDoi"> DOI: 10.1242/jcs.192641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1242%2Fjcs.192641" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=27307493" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslaqtbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2016&pages=2472-2472&author=C.+Aguileraauthor=V.+Fern%C3%A1ndez-Majadaauthor=J.+Ingl%C3%A9s-Esteveauthor=V.+Rodillaauthor=A.+Bigasauthor=L.+Espinosa&title=Efficient+Nuclear+Export+of+P65-I%CE%BAB%CE%B1+Complexes+Requires+14-3-3+Proteins&doi=10.1242%2Fjcs.192641"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient nuclear export of p65-IκBα complexes requires 14-3-3 proteins [Erratum to document cited in CA146:021884]</span></div><div class="casAuthors">Aguilera, Cristina; Fernandez-Majada, Vanessa; Ingles-Esteve, Julia; Rodilla, Veronica; Bigas, Anna; Espinosa, Lluis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2472</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">In the original publication, the GST-14-3-3 Coormassie stain shown in figure 2b is inadvertently duplicated from that shown in Figure 2a and the GST coomassie stains in figure 2c are also duplicated; the corrections are provided here.  In the original publication, the annotation of the reordering of the GST-BD42 and GST-BD42A samples is omitted; the correction is provided here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhcDkXJT1Pt7Vg90H21EOLACvtfcHk0li535ZrFSy1OA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslaqtbbP&md5=667196cfeb49a0e74999ad2c27a24ac7</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1242%2Fjcs.192641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.192641%26sid%3Dliteratum%253Aachs%26aulast%3DAguilera%26aufirst%3DC.%26aulast%3DFern%25C3%25A1ndez-Majada%26aufirst%3DV.%26aulast%3DIngl%25C3%25A9s-Esteve%26aufirst%3DJ.%26aulast%3DRodilla%26aufirst%3DV.%26aulast%3DBigas%26aufirst%3DA.%26aulast%3DEspinosa%26aufirst%3DL.%26atitle%3DEfficient%2520Nuclear%2520Export%2520of%2520P65-I%25CE%25BAB%25CE%25B1%2520Complexes%2520Requires%252014-3-3%2520Proteins%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2016%26volume%3D129%26spage%3D2472%26epage%3D2472%26doi%3D10.1242%2Fjcs.192641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vink, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neves, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srdanović, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landrieu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Selectivity via Cooperativity: Preferential Stabilization of the P65/14-3-3 Interaction with Semisynthetic Natural Products</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">11772</span>– <span class="NLM_lpage">11783</span>, <span class="refDoi"> DOI: 10.1021/jacs.0c02151</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.0c02151" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFSltrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2020&pages=11772-11783&author=M.+Wolterauthor=P.+de+Vinkauthor=J.+F.+Nevesauthor=S.+Srdanovi%C4%87author=Y.+Higuchiauthor=N.+Katoauthor=A.+Wilsonauthor=I.+Landrieuauthor=L.+Brunsveldauthor=C.+Ottmann&title=Selectivity+via+Cooperativity%3A+Preferential+Stabilization+of+the+P65%2F14-3-3+Interaction+with+Semisynthetic+Natural+Products&doi=10.1021%2Fjacs.0c02151"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Selectivity via Cooperativity: Preferential Stabilization of the p65/14-3-3 Interaction with Semisynthetic Natural Products</span></div><div class="casAuthors">Wolter, Madita; de Vink, Pim; Neves, Joao Filipe; Srdanovic, Sonja; Higuchi, Yusuke; Kato, Nobuo; Wilson, Andrew; Landrieu, Isabelle; Brunsveld, Luc; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">11772-11783</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Natural compds. are an important class of potent drug mols. including some retrospectively found to act as stabilizers of protein-protein interactions (PPIs).  However, the design of synthetic PPI stabilizers remains an understudied approach.  To date, there are limited examples where cooperativity has been utilized to guide the optimization of a PPI stabilizer.  The 14-3-3 scaffold proteins provide an excellent platform to explore PPI stabilization because these proteins mediate several hundred PPIs, and a class of natural compds., the fusicoccanes, are known to stabilize a subset of 14-3-3 protein interactions. 14-3-3 has been reported to neg. regulate the p65 subunit of the NF-κB transcription factor, which qualifies this protein complex as a potential target for drug discovery to control cell proliferation.  Here, we report the high-resoln. crystal structures of two 14-3-3 binding motifs of p65 in complex with 14-3-3.  A semisynthetic natural product deriv., DP-005, binds to an interface pocket of the p65/14-3-3 complex and concomitantly stabilizes it.  Cooperativity analyses of this interaction, and other disease relevant 14-3-3-PPIs, demonstrated selectivity of DP-005 for the p65/14-3-3 complex.  The adaptation of a cooperative binding model provided a general approach to characterize stabilization and to assay for selectivity of PPI stabilizers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpd9M4XEOVeZLVg90H21EOLACvtfcHk0li535ZrFSy1OA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFSltrbJ&md5=ffbb418ca8ce137467486c725eca0eaa</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjacs.0c02151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.0c02151%26sid%3Dliteratum%253Aachs%26aulast%3DWolter%26aufirst%3DM.%26aulast%3Dde%2BVink%26aufirst%3DP.%26aulast%3DNeves%26aufirst%3DJ.%2BF.%26aulast%3DSrdanovi%25C4%2587%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DY.%26aulast%3DKato%26aufirst%3DN.%26aulast%3DWilson%26aufirst%3DA.%26aulast%3DLandrieu%26aufirst%3DI.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DSelectivity%2520via%2520Cooperativity%253A%2520Preferential%2520Stabilization%2520of%2520the%2520P65%252F14-3-3%2520Interaction%2520with%2520Semisynthetic%2520Natural%2520Products%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2020%26volume%3D142%26spage%3D11772%26epage%3D11783%26doi%3D10.1021%2Fjacs.0c02151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huxford, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, G.</span></span> <span> </span><span class="NLM_article-title">The Crystal Structure of the IκBα/NF-κB Complex Reveals Mechanisms of NF-κB Inactivation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">759</span>– <span class="NLM_lpage">770</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(00)81699-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1016%2FS0092-8674%2800%2981699-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=9865694" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADyaK1MXivVGr" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=759-770&author=T.+Huxfordauthor=D.-B.+Huangauthor=S.+Malekauthor=G.+Ghosh&title=The+Crystal+Structure+of+the+I%CE%BAB%CE%B1%2FNF-%CE%BAB+Complex+Reveals+Mechanisms+of+NF-%CE%BAB+Inactivation&doi=10.1016%2FS0092-8674%2800%2981699-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The crystal structure of the IκBα/NF-κB complex reveals mechanisms of NF-κB inactivation</span></div><div class="casAuthors">Huxford, Tom; Huang, De-Bin; Malek, Shiva; Ghosh, Gourisankar</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">759-770</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">IκBα regulates the transcription factor NF-κB through the formation of stable IκBα/NF-κB complexes.  Prior to induction, IκBα retains NF-κB in the cytoplasm until the NF-κB activation signal is received.  After activation, NF-κB is removed from gene promoters through assocn. with nuclear IκBα, restoring the preinduction state.  The 2.3 Å crystal structure of IκBα in complex with the NF-κB p50/p65 heterodimer reveals mechanisms of these inhibitory activities.  The presence of IκBα allows large en bloc movement of the NF-κB p65 subunit amino-terminal domain.  This conformational change induces allosteric inhibition of NF-κB DNA binding.  Amino acid residues immediately preceding the nuclear localization signals of both NF-κB p50 and p65 subunits are tethered to the IκBα amino-terminal ankyrin repeats, impeding NF-κB from nuclear import machinery recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkLfVoA9yRbrVg90H21EOLACvtfcHk0lhTHGvPoLLXzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXivVGr&md5=d9f6b8e5bd56fa9f19299c5eb08c0834</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981699-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981699-2%26sid%3Dliteratum%253Aachs%26aulast%3DHuxford%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DD.-B.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DGhosh%26aufirst%3DG.%26atitle%3DThe%2520Crystal%2520Structure%2520of%2520the%2520I%25CE%25BAB%25CE%25B1%252FNF-%25CE%25BAB%2520Complex%2520Reveals%2520Mechanisms%2520of%2520NF-%25CE%25BAB%2520Inactivation%26jtitle%3DCell%26date%3D1998%26volume%3D95%26spage%3D759%26epage%3D770%26doi%3D10.1016%2FS0092-8674%2800%2981699-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brameld, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reuter, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Conformational Preferences Derived from Crystal Structure Data. A Medicinal Chemistry Focused Analysis</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1021/ci7002494</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci7002494" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BD1cXksV2isA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=1-24&author=K.+A.+Brameldauthor=B.+Kuhnauthor=D.+C.+Reuterauthor=M.+Stahl&title=Small+Molecule+Conformational+Preferences+Derived+from+Crystal+Structure+Data.+A+Medicinal+Chemistry+Focused+Analysis&doi=10.1021%2Fci7002494"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule conformational preferences derived from crystal structure data. a medicinal chemistry focused analysis</span></div><div class="casAuthors">Brameld, Ken A.; Kuhn, Bernd; Reuter, Deborah C.; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-24</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Based on torsion angle distributions of frequently occurring substructures, conformation preferences of druglike mols. are presented, accompanied by a review of the relevant literature.  First, the relevance of the Cambridge Structural Database (CSD) for drug design is demonstrated by comparing substructures present in compds. entering clin. trials with those found in the CSD and protein-bound ligands in the Protein Data Bank (PDB).  Next, we briefly highlight preferred conformations of elementary acyclic systems, followed by a discussion of sulfonamide conformations.  Due to their central role in medicinal chem., we discuss properties of aryl ring substituents in depth, including biaryl systems and systems of two aryl rings connected by two acyclic bonds.  For a subset of torsion motifs, we also compare torsion angle histograms derived from CSD structures with those derived from ligands in the PDB.  Furthermore, selected properties of some six- and seven-membered ring systems are discussed.  The article closes with a section on attractive sulfur-oxygen contacts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBrdvMrRvKirVg90H21EOLACvtfcHk0lhTHGvPoLLXzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXksV2isA%253D%253D&md5=289969338dcb94a145ce736d19d05cf4</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fci7002494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci7002494%26sid%3Dliteratum%253Aachs%26aulast%3DBrameld%26aufirst%3DK.%2BA.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DReuter%26aufirst%3DD.%2BC.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DSmall%2520Molecule%2520Conformational%2520Preferences%2520Derived%2520from%2520Crystal%2520Structure%2520Data.%2520A%2520Medicinal%2520Chemistry%2520Focused%2520Analysis%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D1%26epage%3D24%26doi%3D10.1021%2Fci7002494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oksenberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva-Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchaleelaha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patskovska, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patskovsky, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almo, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinha, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalf, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Archer, D. R.</span></span> <span> </span><span class="NLM_article-title">GBT440 Increases Haemoglobin Oxygen Affinity, Reduces Sickling and Prolongs RBC Half-Life in a Murine Model of Sickle Cell Disease</span>. <i>Br. J. Haematol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>175</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">153</span>, <span class="refDoi"> DOI: 10.1111/bjh.14214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1111%2Fbjh.14214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=27378309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFCkurfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2016&pages=141-153&author=D.+Oksenbergauthor=K.+Dufuauthor=M.+P.+Patelauthor=C.+Chuangauthor=Z.+Liauthor=Q.+Xuauthor=A.+Silva-Garciaauthor=C.+Zhouauthor=A.+Hutchaleelahaauthor=L.+Patskovskaauthor=Y.+Patskovskyauthor=S.+C.+Almoauthor=U.+Sinhaauthor=B.+W.+Metcalfauthor=D.+R.+Archer&title=GBT440+Increases+Haemoglobin+Oxygen+Affinity%2C+Reduces+Sickling+and+Prolongs+RBC+Half-Life+in+a+Murine+Model+of+Sickle+Cell+Disease&doi=10.1111%2Fbjh.14214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">GBT440 increases haemoglobin oxygen affinity, reduces sickling and prolongs RBC half-life in a murine model of sickle cell disease</span></div><div class="casAuthors">Oksenberg, Donna; Dufu, Kobina; Patel, Mira P.; Chuang, Chihyuan; Li, Zhe; Xu, Qing; Silva-Garcia, Abel; Zhou, Chengjing; Hutchaleelaha, Athiwat; Patskovska, Larysa; Patskovsky, Yury; Almo, Steven C.; Sinha, Uma; Metcalf, Brian W.; Archer, David R.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Haematology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">141-153</span>CODEN:
                <span class="NLM_cas:coden">BJHEAL</span>;
        ISSN:<span class="NLM_cas:issn">0007-1048</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Summary : A major driver of the pathophysiol. of sickle cell disease (SCD) is polymn. of deoxygenated Hb S (HbS), which leads to sickling and destruction of red blood cells (RBCs) and end-organ damage.  Pharmacol. increasing the proportion of oxygenated HbS in RBCs may inhibit polymn., prevent sickling and provide long term disease modification.  We report that GBT440, a small mol. which binds to the N-terminal α chain of Hb, increases HbS affinity for oxygen, delays in vitro HbS polymn. and prevents sickling of RBCs.  Moreover, in a murine model of SCD, GBT440 extends the half-life of RBCs, reduces reticulocyte counts and prevents ex vivoRBC sickling.  Importantly, oral dosing of GBT440 in animals demonstrates suitability for once daily dosing in humans and a highly selective partitioning into RBCs, which is a key therapeutic safety attribute.  Thus, GBT440 has the potential for clin. use as a disease-modifying agent in sickle cell patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9twqbw0I4ELVg90H21EOLACvtfcHk0lgrwpshtCJdTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFCkurfP&md5=07276e0b19eac3cb2baef00c872c9197</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1111%2Fbjh.14214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbjh.14214%26sid%3Dliteratum%253Aachs%26aulast%3DOksenberg%26aufirst%3DD.%26aulast%3DDufu%26aufirst%3DK.%26aulast%3DPatel%26aufirst%3DM.%2BP.%26aulast%3DChuang%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DSilva-Garcia%26aufirst%3DA.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DHutchaleelaha%26aufirst%3DA.%26aulast%3DPatskovska%26aufirst%3DL.%26aulast%3DPatskovsky%26aufirst%3DY.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DSinha%26aufirst%3DU.%26aulast%3DMetcalf%26aufirst%3DB.%2BW.%26aulast%3DArcher%26aufirst%3DD.%2BR.%26atitle%3DGBT440%2520Increases%2520Haemoglobin%2520Oxygen%2520Affinity%252C%2520Reduces%2520Sickling%2520and%2520Prolongs%2520RBC%2520Half-Life%2520in%2520a%2520Murine%2520Model%2520of%2520Sickle%2520Cell%2520Disease%26jtitle%3DBr.%2520J.%2520Haematol.%26date%3D2016%26volume%3D175%26spage%3D141%26epage%3D153%26doi%3D10.1111%2Fbjh.14214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Differential PROTAC Substrate Specificity Dictated by Orientation of Recruited E3 Ligase</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1038/s41467-018-08027-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1038%2Fs41467-018-08027-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=30602773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlSru7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1-13&author=B.+E.+Smithauthor=S.+L.+Wangauthor=S.+Jaime-Figueroaauthor=A.+Harbinauthor=J.+Wangauthor=B.+D.+Hammanauthor=C.+M.+Crews&title=Differential+PROTAC+Substrate+Specificity+Dictated+by+Orientation+of+Recruited+E3+Ligase&doi=10.1038%2Fs41467-018-08027-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Myeloid lineage enhancers drive oncogene synergy in CEBPA/CSF3R mutant acute myeloid leukemia</span></div><div class="casAuthors">Braun, Theodore P.; Okhovat, Mariam; Coblentz, Cody; Carratt, Sarah A.; Foley, Amy; Schonrock, Zachary; Smith, Brittany M.; Nevonen, Kimberly; Davis, Brett; Garcia, Brianna; LaTocha, Dorian; Weeder, Benjamin R.; Grzadkowski, Michal R.; Estabrook, Joey C.; Manning, Hannah G.; Watanabe-Smith, Kevin; Jeng, Sophia; Smith, Jenny L.; Leonti, Amanda R.; Ries, Rhonda E.; McWeeney, Shannon; Di Genua, Cristina; Drissen, Roy; Nerlov, Claus; Meshinchi, Soheil; Carbone, Lucia; Druker, Brian J.; Maxson, Julia E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-15</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Acute Myeloid Leukemia (AML) develops due to the acquisition of mutations from multiple functional classes.  Here, we demonstrate that activating mutations in the granulocyte colony stimulating factor receptor (CSF3R), cooperate with loss of function mutations in the transcription factor CEBPA to promote acute leukemia development.  The interaction between these distinct classes of mutations occurs at the level of myeloid lineage enhancers where mutant CEBPA prevents activation of a subset of differentiation assocd. enhancers.  To confirm this enhancer-dependent mechanism, we demonstrate that CEBPA mutations must occur as the initial event in AML initiation.  This improved mechanistic understanding will facilitate therapeutic development targeting the intersection of oncogene cooperativity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonOzBTD_nh57Vg90H21EOLACvtfcHk0lgrwpshtCJdTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlSru7jE&md5=edb5560e8b3b0da2368fa6e527b0feb0</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-08027-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-08027-7%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DWang%26aufirst%3DS.%2BL.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DHarbin%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DDifferential%2520PROTAC%2520Substrate%2520Specificity%2520Dictated%2520by%2520Orientation%2520of%2520Recruited%2520E3%2520Ligase%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D1%26epage%3D13%26doi%3D10.1038%2Fs41467-018-08027-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zoppi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maniaci, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieshofer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riching, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spallarossa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Iterative Design and Optimization of Initially Inactive Proteolysis Targeting Chimeras (PROTACs) Identify VZ185 as a Potent, Fast, and Selective von Hippel-Lindau (VHL) Based Dual Degrader Probe of BRD9 and BRD7</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">699</span>– <span class="NLM_lpage">726</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01413</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01413" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVyhsLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=699-726&author=V.+Zoppiauthor=S.+J.+Hughesauthor=C.+Maniaciauthor=A.+Testaauthor=T.+Gmaschitzauthor=C.+Wieshoferauthor=M.+Koeglauthor=K.+M.+Richingauthor=D.+L.+Danielsauthor=A.+Spallarossaauthor=A.+Ciulli&title=Iterative+Design+and+Optimization+of+Initially+Inactive+Proteolysis+Targeting+Chimeras+%28PROTACs%29+Identify+VZ185+as+a+Potent%2C+Fast%2C+and+Selective+von+Hippel-Lindau+%28VHL%29+Based+Dual+Degrader+Probe+of+BRD9+and+BRD7&doi=10.1021%2Facs.jmedchem.8b01413"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Iterative Design and Optimization of Initially Inactive Proteolysis Targeting Chimeras (PROTACs) Identify VZ185 as a Potent, Fast, and Selective von Hippel-Lindau (VHL) Based Dual Degrader Probe of BRD9 and BRD7</span></div><div class="casAuthors">Zoppi, Vittoria; Hughes, Scott J.; Maniaci, Chiara; Testa, Andrea; Gmaschitz, Teresa; Wieshofer, Corinna; Koegl, Manfred; Riching, Kristin M.; Daniels, Danette L.; Spallarossa, Andrea; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">699-726</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Developing PROTACs to redirect the ubiquitination activity of E3 ligases and potently degrade a target protein within cells can be a lengthy and unpredictable process, and it remains unclear whether any combination of E3 and target might be productive for degrdn.  The authors describe a probe-quality degrader for a ligase-target pair deemed unsuitable: the von Hippel-Lindau (VHL) and BRD9, a bromodomain-contg. subunit of the SWI/SNF chromatin remodeling complex BAF.  VHL-based degraders could be optimized from suboptimal compds. in two rounds by systematically varying conjugation patterns and linkers and monitoring cellular degrdn. activities, kinetic profiles, and ubiquitination, as well as ternary complex formation thermodn.  The emerged structure-activity relations guided the discovery of VZ185, a potent, fast, and selective degrader of BRD9 and of its close homolog BRD7.  The authors' findings qualify a new chem. tool for BRD7/9 knockdown and provide a roadmap for PROTAC development against seemingly incompatible target-ligase combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiPnajotKvqLVg90H21EOLACvtfcHk0li6zXcQlGhSLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVyhsLjN&md5=be7938c26807a2535b7f9b7de8ebfbb0</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01413%26sid%3Dliteratum%253Aachs%26aulast%3DZoppi%26aufirst%3DV.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DManiaci%26aufirst%3DC.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DWieshofer%26aufirst%3DC.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DRiching%26aufirst%3DK.%2BM.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DSpallarossa%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DIterative%2520Design%2520and%2520Optimization%2520of%2520Initially%2520Inactive%2520Proteolysis%2520Targeting%2520Chimeras%2520%2528PROTACs%2529%2520Identify%2520VZ185%2520as%2520a%2520Potent%252C%2520Fast%252C%2520and%2520Selective%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520Based%2520Dual%2520Degrader%2520Probe%2520of%2520BRD9%2520and%2520BRD7%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D699%26epage%3D726%26doi%3D10.1021%2Facs.jmedchem.8b01413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zorba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borzilleri, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farley, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiemer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Irrizary, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czabaniuk, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuff, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niosi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luthra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Kattan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span> <span> </span><span class="NLM_article-title">Delineating the Role of Cooperativity in the Design of Potent PROTACs for BTK</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">E7285</span>– <span class="NLM_lpage">E7292</span>, <span class="refDoi"> DOI: 10.1073/pnas.1803662115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1073%2Fpnas.1803662115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=30012605" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=E7285-E7292&author=A.+Zorbaauthor=C.+Nguyenauthor=Y.+Xuauthor=J.+Starrauthor=K.+Borzilleriauthor=J.+Smithauthor=H.+Zhuauthor=K.+A.+Farleyauthor=W.+D.+Dingauthor=J.+Schiemerauthor=X.+Fengauthor=J.+S.+Changauthor=D.+P.+Uccelloauthor=J.+A.+Youngauthor=C.+N.+Garcia-Irrizaryauthor=L.+Czabaniukauthor=B.+Schuffauthor=R.+Oliverauthor=J.+Montgomeryauthor=M.+M.+Haywardauthor=J.+Coeauthor=J.+Chenauthor=M.+Niosiauthor=S.+Luthraauthor=J.+C.+Shahauthor=A.+El-Kattanauthor=X.+Qiuauthor=G.+M.+Westauthor=M.+C.+Noeauthor=V.+Shanmugasundaramauthor=A.+M.+Gilbertauthor=M.+F.+Brownauthor=M.+F.+Calabrese&title=Delineating+the+Role+of+Cooperativity+in+the+Design+of+Potent+PROTACs+for+BTK&doi=10.1073%2Fpnas.1803662115"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1803662115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1803662115%26sid%3Dliteratum%253Aachs%26aulast%3DZorba%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DStarr%26aufirst%3DJ.%26aulast%3DBorzilleri%26aufirst%3DK.%26aulast%3DSmith%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DFarley%26aufirst%3DK.%2BA.%26aulast%3DDing%26aufirst%3DW.%2BD.%26aulast%3DSchiemer%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DChang%26aufirst%3DJ.%2BS.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%26aulast%3DGarcia-Irrizary%26aufirst%3DC.%2BN.%26aulast%3DCzabaniuk%26aufirst%3DL.%26aulast%3DSchuff%26aufirst%3DB.%26aulast%3DOliver%26aufirst%3DR.%26aulast%3DMontgomery%26aufirst%3DJ.%26aulast%3DHayward%26aufirst%3DM.%2BM.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DNiosi%26aufirst%3DM.%26aulast%3DLuthra%26aufirst%3DS.%26aulast%3DShah%26aufirst%3DJ.%2BC.%26aulast%3DEl-Kattan%26aufirst%3DA.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DWest%26aufirst%3DG.%2BM.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26atitle%3DDelineating%2520the%2520Role%2520of%2520Cooperativity%2520in%2520the%2520Design%2520of%2520Potent%2520PROTACs%2520for%2520BTK%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2018%26volume%3D115%26spage%3DE7285%26epage%3DE7292%26doi%3D10.1073%2Fpnas.1803662115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farnaby, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">SPR-Measured Dissociation Kinetics of PROTAC Ternary Complexes Influence Target Degradation Rate</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">368</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00092</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00092" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVCisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=361-368&author=M.+J.+Royauthor=S.+Winklerauthor=S.+J.+Hughesauthor=C.+Whitworthauthor=M.+Galantauthor=W.+Farnabyauthor=K.+Rumpelauthor=A.+Ciulli&title=SPR-Measured+Dissociation+Kinetics+of+PROTAC+Ternary+Complexes+Influence+Target+Degradation+Rate&doi=10.1021%2Facschembio.9b00092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">SPR-Measured Dissociation Kinetics of PROTAC Ternary Complexes Influence Target Degradation Rate</span></div><div class="casAuthors">Roy, Michael J.; Winkler, Sandra; Hughes, Scott J.; Whitworth, Claire; Galant, Michael; Farnaby, William; Rumpel, Klaus; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">361-368</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bifunctional degrader mols., known as proteolysis-targeting chimeras (PROTACs), function by recruiting a target to an E3 ligase, forming a target/PROTAC/ligase ternary complex.  Despite the importance of this key intermediate species, no detailed validation of a method to directly det. binding parameters for ternary complex kinetics has been reported, and it remains to be addressed whether tuning the kinetics of PROTAC ternary complexes may be an effective strategy to improve the efficiency of targeted protein degrdn.  Here, we develop an SPR-based assay to quantify the stability of PROTAC-induced ternary complexes by measuring for the first time the kinetics of their formation and dissocn. in vitro using purified proteins.  We benchmark our assay using four PROTACs that target the bromodomains (BDs) of bromodomain and extraterminal domain proteins Brd2, Brd3, and Brd4 to the von Hippel-Lindau E3 ligase (VHL).  We reveal marked differences in ternary complex off-rates for different PROTACs that exhibit either pos. or neg. cooperativity for ternary complex formation relative to binary binding.  The pos. cooperative degrader MZ1 forms comparatively stable and long-lived ternary complexes with either Brd4BD2 or Brd2BD2 and VHL.  Equivalent complexes with Brd3BD2 are destabilized due to a single amino acid difference (Glu/Gly swap) present in the bromodomain.  We observe that this difference in ternary complex dissociative half-life correlates to a greater initial rate of intracellular degrdn. of Brd2 and Brd4 relative to Brd3.  These findings establish a novel assay to measure the kinetics of PROTAC ternary complexes and elucidate the important kinetic parameters that drive effective target degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzFhvuKYztcbVg90H21EOLACvtfcHk0li6zXcQlGhSLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVCisrg%253D&md5=629d1acbd218b668a2696b727946962b</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00092%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DM.%2BJ.%26aulast%3DWinkler%26aufirst%3DS.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DGalant%26aufirst%3DM.%26aulast%3DFarnaby%26aufirst%3DW.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSPR-Measured%2520Dissociation%2520Kinetics%2520of%2520PROTAC%2520Ternary%2520Complexes%2520Influence%2520Target%2520Degradation%2520Rate%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D361%26epage%3D368%26doi%3D10.1021%2Facschembio.9b00092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87.e5</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87.e5&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+Design+from+Selective+Degradation+with+a+Promiscuous+Warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0ljoB5ZeTq3d3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520Design%2520from%2520Selective%2520Degradation%2520with%2520a%2520Promiscuous%2520Warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87.e5%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vink, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neves, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srdanovic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landrieu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Selectivity via Cooperativity: Preferential Stabilization of the P65/14-3-3 Interaction with Semi-Synthetic Natural Products</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">11772</span>– <span class="NLM_lpage">11783</span>, <span class="refDoi"> DOI: 10.1021/jacs.0c02151</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.0c02151" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFSltrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2020&pages=11772-11783&author=M.+Wolterauthor=P.+de+Vinkauthor=J.+F.+Nevesauthor=S.+Srdanovicauthor=Y.+Higuchiauthor=N.+Katoauthor=A.+J.+Wilsonauthor=I.+Landrieuauthor=L.+Brunsveldauthor=C.+Ottmann&title=Selectivity+via+Cooperativity%3A+Preferential+Stabilization+of+the+P65%2F14-3-3+Interaction+with+Semi-Synthetic+Natural+Products&doi=10.1021%2Fjacs.0c02151"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Selectivity via Cooperativity: Preferential Stabilization of the p65/14-3-3 Interaction with Semisynthetic Natural Products</span></div><div class="casAuthors">Wolter, Madita; de Vink, Pim; Neves, Joao Filipe; Srdanovic, Sonja; Higuchi, Yusuke; Kato, Nobuo; Wilson, Andrew; Landrieu, Isabelle; Brunsveld, Luc; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">11772-11783</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Natural compds. are an important class of potent drug mols. including some retrospectively found to act as stabilizers of protein-protein interactions (PPIs).  However, the design of synthetic PPI stabilizers remains an understudied approach.  To date, there are limited examples where cooperativity has been utilized to guide the optimization of a PPI stabilizer.  The 14-3-3 scaffold proteins provide an excellent platform to explore PPI stabilization because these proteins mediate several hundred PPIs, and a class of natural compds., the fusicoccanes, are known to stabilize a subset of 14-3-3 protein interactions. 14-3-3 has been reported to neg. regulate the p65 subunit of the NF-κB transcription factor, which qualifies this protein complex as a potential target for drug discovery to control cell proliferation.  Here, we report the high-resoln. crystal structures of two 14-3-3 binding motifs of p65 in complex with 14-3-3.  A semisynthetic natural product deriv., DP-005, binds to an interface pocket of the p65/14-3-3 complex and concomitantly stabilizes it.  Cooperativity analyses of this interaction, and other disease relevant 14-3-3-PPIs, demonstrated selectivity of DP-005 for the p65/14-3-3 complex.  The adaptation of a cooperative binding model provided a general approach to characterize stabilization and to assay for selectivity of PPI stabilizers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpd9M4XEOVeZLVg90H21EOLACvtfcHk0ljoB5ZeTq3d3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFSltrbJ&md5=ffbb418ca8ce137467486c725eca0eaa</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjacs.0c02151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.0c02151%26sid%3Dliteratum%253Aachs%26aulast%3DWolter%26aufirst%3DM.%26aulast%3Dde%2BVink%26aufirst%3DP.%26aulast%3DNeves%26aufirst%3DJ.%2BF.%26aulast%3DSrdanovic%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DY.%26aulast%3DKato%26aufirst%3DN.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26aulast%3DLandrieu%26aufirst%3DI.%26aulast%3DBrunsveld%26aufirst%3DL.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DSelectivity%2520via%2520Cooperativity%253A%2520Preferential%2520Stabilization%2520of%2520the%2520P65%252F14-3-3%2520Interaction%2520with%2520Semi-Synthetic%2520Natural%2520Products%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2020%26volume%3D142%26spage%3D11772%26epage%3D11783%26doi%3D10.1021%2Fjacs.0c02151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clabbers, M. T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruene, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkhurst, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrahams, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waterman, D. G.</span></span> <span> </span><span class="NLM_article-title">Electron Diffraction Data Processing with DIALS</span>. <i>Acta Crystallogr. Sect. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">506</span>– <span class="NLM_lpage">518</span>, <span class="refDoi"> DOI: 10.1107/S2059798318007726</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1107%2FS2059798318007726" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtV2ltbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=506-518&author=M.+T.+B.+Clabbersauthor=T.+Grueneauthor=J.+M.+Parkhurstauthor=J.+P.+Abrahamsauthor=D.+G.+Waterman&title=Electron+Diffraction+Data+Processing+with+DIALS&doi=10.1107%2FS2059798318007726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Electron diffraction data processing with DIALS</span></div><div class="casAuthors">Clabbers, Max T. B.; Gruene, Tim; Parkhurst, James M.; Abrahams, Jan Pieter; Waterman, David G.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">506-518</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Electron diffraction is a relatively novel alternative to X-ray crystallog. for the structure detn. of macromols. from three-dimensional nanometer-sized crystals.  Nevertheless, the DIALS software package can, with specific adaptations, successfully process continuous-rotation electron diffraction data.  Pathologies encountered specifically in electron diffraction make data integration more challenging.  Errors can arise from instrumentation, such as beam drift or distorted diffraction patterns from lens imperfections.  The diffraction geometry brings addnl. challenges such as strong correlation between lattice parameters and detector distance.  These issues are compounded if calibration is incomplete, leading to uncertainty in exptl. geometry, such as the effective detector distance and the rotation rate or direction.  Dynamic scattering, absorption, radiation damage and incomplete wedges of data are addnl. factors that complicate data processing.  Here, recent features of DIALS as adapted to electron diffraction processing are shown, including diagnostics for problematic diffraction geometry refinement, refinement of a smoothly varying beam model and corrections for distorted diffraction images.  These novel features, combined with the existing tools in DIALS, make data integration and refinement feasible for electron crystallog., even in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ2xvp5nytBbVg90H21EOLACvtfcHk0ljoB5ZeTq3d3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtV2ltbzM&md5=0ef3d54b8435ed67f31774547a35d263</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS2059798318007726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798318007726%26sid%3Dliteratum%253Aachs%26aulast%3DClabbers%26aufirst%3DM.%2BT.%2BB.%26aulast%3DGruene%26aufirst%3DT.%26aulast%3DParkhurst%26aufirst%3DJ.%2BM.%26aulast%3DAbrahams%26aufirst%3DJ.%2BP.%26aulast%3DWaterman%26aufirst%3DD.%2BG.%26atitle%3DElectron%2520Diffraction%2520Data%2520Processing%2520with%2520DIALS%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520Struct.%2520Biol.%26date%3D2018%26volume%3D74%26spage%3D506%26epage%3D518%26doi%3D10.1107%2FS2059798318007726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">Model Preparation in MOLREP and Examples of Model Improvement Using X-Ray Data</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1107/S0907444907049839</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1107%2FS0907444907049839" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=18094465" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVGjt7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2008&pages=33-39&author=A.+A.+Lebedevauthor=A.+A.+Vaginauthor=G.+N.+Murshudov&title=Model+Preparation+in+MOLREP+and+Examples+of+Model+Improvement+Using+X-Ray+Data&doi=10.1107%2FS0907444907049839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Model preparation in MOLREP and examples of model improvement using X-ray data</span></div><div class="casAuthors">Lebedev, Andrey A.; Vagin, Alexei A.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-39</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The success of mol. replacement is critically dependent on the quality of the search model.  Several model-prepn. procedures are integrated in the mol.-replacement program MOLREP.  These include model modification on the basis of amino-acid sequence alignment and model correction based on anal. of the solvent-accessibility of the atoms.  The packing function used in MOLREP for the translational search is explained in the context of model prepn.  In difficult cases, bioinformatics-based modifications are not sufficient for successful mol. replacement.  An approach implemented in MOLREP for solving cases with translational noncrystallog. symmetry is an example of model prepn. in which anal. of x-ray data plays an essential role.  In addn., two examples are presented in which the x-ray data were used to refine partial models for subsequent use in mol. replacement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeCvDgb4tgK7Vg90H21EOLACvtfcHk0liRWe4sJ0fGfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVGjt7nI&md5=8ccba5e3d8014210bed44fe310c4c33e</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0907444907049839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444907049839%26sid%3Dliteratum%253Aachs%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DModel%2520Preparation%2520in%2520MOLREP%2520and%2520Examples%2520of%2520Model%2520Improvement%2520Using%2520X-Ray%2520Data%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D2008%26volume%3D64%26spage%3D33%26epage%3D39%26doi%3D10.1107%2FS0907444907049839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">Molecular Replacement with MOLREP</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1107/S0907444909042589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1107%2FS0907444909042589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=20057045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=22-25&author=A.+Vaginauthor=A.+Teplyakov&title=Molecular+Replacement+with+MOLREP&doi=10.1107%2FS0907444909042589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular replacement with MOLREP</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-25</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement that utilizes a no. of original approaches to rotational and translational search and data prepn.  Since the first publication describing the program, MOLREP has acquired a variety of features that include weighting of the X-ray data and search models, multi-copy search, fitting the model into electron d., structural superposition of two models and rigid-body refinement.  The program can run in a fully automatic mode using optimized parameters calcd. from the input data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzj5SzYvyu_7Vg90H21EOLACvtfcHk0liRWe4sJ0fGfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D&md5=820d114719aca209994ffb0403e3b20d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042589%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMolecular%2520Replacement%2520with%2520MOLREP%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D22%26epage%3D25%26doi%3D10.1107%2FS0907444909042589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: Model-Building Tools for Molecular Graphics</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-Building+Tools+for+Molecular+Graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0liRWe4sJ0fGfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-Building%2520Tools%2520for%2520Molecular%2520Graphics%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span> <span> </span><span class="NLM_article-title">REFMAC5 for the Refinement of Macromolecular Crystal Structures</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/S0907444911001314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1107%2FS0907444911001314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=21460454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=355-367&author=G.+N.+Murshudovauthor=P.+Skub%C3%A1kauthor=A.+A.+Lebedevauthor=N.+S.+Pannuauthor=R.+A.+Steinerauthor=R.+A.+Nichollsauthor=M.+D.+Winnauthor=F.+Longauthor=A.+A.+Vagin&title=REFMAC5+for+the+Refinement+of+Macromolecular+Crystal+Structures&doi=10.1107%2FS0907444911001314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 for the refinement of macromolecular crystal structures</span></div><div class="casAuthors">Murshudov, Garib N.; Skubak, Pavol; Lebedev, Andrey A.; Pannu, Navraj S.; Steiner, Roberto A.; Nicholls, Robert A.; Winn, Martyn D.; Long, Fei; Vagin, Alexei A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">This paper describes various components of the macromol. crystallog. refinement program REFMAC5, which is distributed as part of the CCP4 suite.  REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS exptl. diffraction data.  To ensure chem. and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization.  Reliable models at resolns. at least as low as 4 Å can be achieved thanks to low-resoln. refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on at. displacement parameters (ADPs) based on the Kullback-Leibler divergence.  REFMAC5 addnl. offers TLS parameterization and, when high-resoln. data are available, fast refinement of anisotropic ADPs.  Refinement in the presence of twinning is performed in a fully automated fashion.  REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resoln. spectrum encountered in macromol. crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Jq-i2qIq5rVg90H21EOLACvtfcHk0lj3zeASw3H5og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D&md5=f8f3202d246908500057ad7c71015b7b</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444911001314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911001314%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DSkub%25C3%25A1k%26aufirst%3DP.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26atitle%3DREFMAC5%2520for%2520the%2520Refinement%2520of%2520Macromolecular%2520Crystal%2520Structures%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D355%26epage%3D367%26doi%3D10.1107%2FS0907444911001314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title"><i>PHENIX</i> : A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX+%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lj3zeASw3H5og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%2520%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doveston, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuusk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrei, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leysen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castaldi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendricks, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Stabilization of the P53 – 14-3-3 Protein-Protein Interaction</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>591</i></span>,  <span class="NLM_fpage">2449</span>– <span class="NLM_lpage">2457</span>, <span class="refDoi"> DOI: 10.1002/1873-3468.12723</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=10.1002%2F1873-3468.12723" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=28640363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1yitLnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=591&publication_year=2017&pages=2449-2457&author=R.+G.+Dovestonauthor=A.+Kuuskauthor=S.+Andreiauthor=S.+Leysenauthor=Q.+Caoauthor=P.+Castaldiauthor=A.+Hendricksauthor=H.+Chenauthor=H.+Boydauthor=C.+Ottmann&title=Small-Molecule+Stabilization+of+the+P53+%E2%80%93+14-3-3+Protein-Protein+Interaction&doi=10.1002%2F1873-3468.12723"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule stabilization of the p53-14-3-3 protein-protein interaction</span></div><div class="casAuthors">Doveston, Richard G.; Kuusk, Ave; Andrei, Sebastian A.; Leysen, Seppe; Cao, Qing; Castaldi, Maria P.; Hendricks, Adam; Brunsveld, Luc; Chen, Hongming; Boyd, Helen; Ottmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">591</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2449-2457</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The 14-3-3 proteins are pos. regulators of tumor suppressor protein p53, the mutation of which is implicated in many human cancers.  Current strategies for targeting of p53 involve restoration of wild-type function or inhibition of the interaction with MDM2, its key neg. regulator.  Despite the efficacy of these strategies, the alternate approach of stabilizing the interaction of p53 with pos. regulators and, thus, enhancing tumor suppressor activity, has not been explored.  Here, we report the 1st example of small-mol. (fusicoccin A) stabilization of the 14-3-3-p53 protein-protein interaction (PPI) and demonstrate the potential of this approach as a therapeutic modality.  We also obsd. a disconnect between biophys. and crystallog. data in the presence of a stabilizing mol., which is unusual in 14-3-3 PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpw1xMdr0EvsrVg90H21EOLACvtfcHk0lj3zeASw3H5og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1yitLnI&md5=3048c3d9827ef0a5a50b502fc5cf6f5d</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2F1873-3468.12723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1873-3468.12723%26sid%3Dliteratum%253Aachs%26aulast%3DDoveston%26aufirst%3DR.%2BG.%26aulast%3DKuusk%26aufirst%3DA.%26aulast%3DAndrei%26aufirst%3DS.%26aulast%3DLeysen%26aufirst%3DS.%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DCastaldi%26aufirst%3DP.%26aulast%3DHendricks%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DBoyd%26aufirst%3DH.%26aulast%3DOttmann%26aufirst%3DC.%26atitle%3DSmall-Molecule%2520Stabilization%2520of%2520the%2520P53%2520%25E2%2580%2593%252014-3-3%2520Protein-Protein%2520Interaction%26jtitle%3DFEBS%2520Lett.%26date%3D2017%26volume%3D591%26spage%3D2449%26epage%3D2457%26doi%3D10.1002%2F1873-3468.12723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QHL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QHL','PDB','6QHL'); return false;">PDB: 6QHL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1IKN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1IKN','PDB','1IKN'); return false;">PDB: 1IKN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YOW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YOW','PDB','6YOW'); return false;">PDB: 6YOW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YOY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YOY','PDB','6YOY'); return false;">PDB: 6YOY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NMH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NMH','PDB','7NMH'); return false;">PDB: 7NMH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJF','PDB','7BJF'); return false;">PDB: 7BJF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJL','PDB','7BJL'); return false;">PDB: 7BJL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJW','PDB','7BJW'); return false;">PDB: 7BJW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BKH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BKH','PDB','7BKH'); return false;">PDB: 7BKH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YP3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YP3','PDB','6YP3'); return false;">PDB: 6YP3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NM9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NM9','PDB','7NM9'); return false;">PDB: 7NM9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NR7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NR7','PDB','7NR7'); return false;">PDB: 7NR7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YOX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YOX','PDB','6YOX'); return false;">PDB: 6YOX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BIY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BIY','PDB','7BIY'); return false;">PDB: 7BIY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NLE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NLE','PDB','7NLE'); return false;">PDB: 7NLE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NK5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NK5','PDB','7NK5'); return false;">PDB: 7NK5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NJB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NJB','PDB','7NJB'); return false;">PDB: 7NJB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJB','PDB','7BJB'); return false;">PDB: 7BJB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NM3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NM3','PDB','7NM3'); return false;">PDB: 7NM3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NM1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NM1','PDB','7NM1'); return false;">PDB: 7NM1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NLA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NLA','PDB','7NLA'); return false;">PDB: 7NLA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NK3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NK3','PDB','7NK3'); return false;">PDB: 7NK3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NJ9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NJ9','PDB','7NJ9'); return false;">PDB: 7NJ9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BIQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BIQ','PDB','7BIQ'); return false;">PDB: 7BIQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BI3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BI3','PDB','7BI3'); return false;">PDB: 7BI3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BIW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BIW','PDB','7BIW'); return false;">PDB: 7BIW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NV4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NV4','PDB','7NV4'); return false;">PDB: 7NV4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NVI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NVI','PDB','7NVI'); return false;">PDB: 7NVI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NWS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NWS','PDB','7NWS'); return false;">PDB: 7NWS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NXS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NXS','PDB','7NXS'); return false;">PDB: 7NXS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NXT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NXT','PDB','7NXT'); return false;">PDB: 7NXT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NXW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NXW','PDB','7NXW'); return false;">PDB: 7NXW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NXY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NXY','PDB','7NXY'); return false;">PDB: 7NXY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NY4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NY4','PDB','7NY4'); return false;">PDB: 7NY4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NYE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NYE','PDB','7NYE'); return false;">PDB: 7NYE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NYF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NYF','PDB','7NYF'); return false;">PDB: 7NYF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NYG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NYG','PDB','7NYG'); return false;">PDB: 7NYG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NZ6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NZ6','PDB','7NZ6'); return false;">PDB: 7NZ6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NZG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NZG','PDB','7NZG'); return false;">PDB: 7NZG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NZK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NZK','PDB','7NZK'); return false;">PDB: 7NZK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NZV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NZV','PDB','7NZV'); return false;">PDB: 7NZV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O34" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O34','PDB','7O34'); return false;">PDB: 7O34</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3A','PDB','7O3A'); return false;">PDB: 7O3A</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3F','PDB','7O3F'); return false;">PDB: 7O3F</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3P" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3P','PDB','7O3P'); return false;">PDB: 7O3P</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3Q" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3Q','PDB','7O3Q'); return false;">PDB: 7O3Q</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3R" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3R','PDB','7O3R'); return false;">PDB: 7O3R</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3S','PDB','7O3S'); return false;">PDB: 7O3S</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O57" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O57','PDB','7O57'); return false;">PDB: 7O57</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O59" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O59','PDB','7O59'); return false;">PDB: 7O59</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5A','PDB','7O5A'); return false;">PDB: 7O5A</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5C" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5C','PDB','7O5C'); return false;">PDB: 7O5C</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5D','PDB','7O5D'); return false;">PDB: 7O5D</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5F','PDB','7O5F'); return false;">PDB: 7O5F</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5G" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5G','PDB','7O5G'); return false;">PDB: 7O5G</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5O" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5O','PDB','7O5O'); return false;">PDB: 7O5O</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5P" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5P','PDB','7O5P'); return false;">PDB: 7O5P</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5S','PDB','7O5S'); return false;">PDB: 7O5S</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5U" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5U','PDB','7O5U'); return false;">PDB: 7O5U</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5X" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5X','PDB','7O5X'); return false;">PDB: 7O5X</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6F','PDB','7O6F'); return false;">PDB: 7O6F</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6G" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6G','PDB','7O6G'); return false;">PDB: 7O6G</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6I','PDB','7O6I'); return false;">PDB: 7O6I</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6J','PDB','7O6J'); return false;">PDB: 7O6J</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6K" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6K','PDB','7O6K'); return false;">PDB: 7O6K</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6M" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6M','PDB','7O6M'); return false;">PDB: 7O6M</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6O" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6O','PDB','7O6O'); return false;">PDB: 7O6O</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i19"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00401">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_80747"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00401?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00401</a>. <ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional results and X-ray crystallography tables, chemical experimental, and chemical spectral data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_001.pdf">jm1c00401_si_001.pdf (3.98 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00401/suppl_file/jm1c00401_si_002.csv">jm1c00401_si_002.csv (7.09 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors have been deposited in the Protein Data Bank under the following accession codes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NMH">7NMH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BI3">7BI3</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BIQ">7BIQ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BIW">7BIW</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BIY">7BIY</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJB">7BJB</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJF">7BJF</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJL">7BJL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BJW">7BJW</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BKH">7BKH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NJ9">7NJ9</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NJB">7NJB</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NK3">7NK3</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NK5">7NK5</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NLA">7NLA</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NLE">7NLE</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NM1">7NM1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NM3">7NM3</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NM9">7NM9</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NR7">7NR7</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NV4">7NV4</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NVI">7NVI</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NWS">7NWS</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NXS">7NXS</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NXT">7NXT</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NXW">7NXW</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NXY">7NXY</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NY4">7NY4</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NYE">7NYE</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NYF">7NYF</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NYG">7NYG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NZ6">7NZ6</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NZG">7NZG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NZK">7NZK</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7NZV">7NZV</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O34">7O34</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3A">7O3A</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3F">7O3F</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3P">7O3P</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3Q">7O3Q</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3R">7O3R</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O3S">7O3S</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O57">7O57</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O59">7O59</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5A">7O5A</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5C">7O5C</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5D">7O5D</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5F">7O5F</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5G">7O5G</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5O">7O5O</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5P">7O5P</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5S">7O5S</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5U">7O5U</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O5X">7O5X</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6F">7O6F</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6G">7O6G</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6I">7O6I</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6J">7O6J</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6K">7O6K</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6M">7O6M</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7O6O">7O6O</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00401&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00401%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-12%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00401" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                11MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679922a2caad3ddd","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
