
<html lang="en"     class="pb-page"  data-request-id="f5944c30-eae1-4cf0-8c81-b04d18558e84"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;issue:issue:10.1021/jmcmar.2017.60.issue-1;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.6b01055"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of a Chemical Probe Bisamide (CCT251236): An Orally Bioavailable Efficacious Pirin Ligand from a Heat Shock Transcription Factor 1 (HSF1) Phenotypic Screen" /></meta><meta name="dc.Creator" content="Matthew D.  Cheeseman" /></meta><meta name="dc.Creator" content="Nicola E. A.  Chessum" /></meta><meta name="dc.Creator" content="Carl S.  Rye" /></meta><meta name="dc.Creator" content="A. Elisa  Pasqua" /></meta><meta name="dc.Creator" content="Michael J.  Tucker" /></meta><meta name="dc.Creator" content="Birgit  Wilding" /></meta><meta name="dc.Creator" content="Lindsay E.  Evans" /></meta><meta name="dc.Creator" content="Susan  Lepri" /></meta><meta name="dc.Creator" content="Meirion  Richards" /></meta><meta name="dc.Creator" content="Swee Y.  Sharp" /></meta><meta name="dc.Creator" content="Salyha  Ali" /></meta><meta name="dc.Creator" content="Martin  Rowlands" /></meta><meta name="dc.Creator" content="Lisa  O’Fee" /></meta><meta name="dc.Creator" content="Asadh  Miah" /></meta><meta name="dc.Creator" content="Angela  Hayes" /></meta><meta name="dc.Creator" content="Alan T.  Henley" /></meta><meta name="dc.Creator" content="Marissa  Powers" /></meta><meta name="dc.Creator" content="Robert  te Poele" /></meta><meta name="dc.Creator" content="Emmanuel  De Billy" /></meta><meta name="dc.Creator" content="Loredana  Pellegrino" /></meta><meta name="dc.Creator" content="Florence  Raynaud" /></meta><meta name="dc.Creator" content="Rosemary  Burke" /></meta><meta name="dc.Creator" content="Rob L. M.  van Montfort" /></meta><meta name="dc.Creator" content="Suzanne A.  Eccles" /></meta><meta name="dc.Creator" content="Paul  Workman" /></meta><meta name="dc.Creator" content="Keith  Jones" /></meta><meta name="dc.Description" content="Phenotypic screens, which focus on measuring and quantifying discrete cellular changes rather than affinity for individual recombinant proteins, have recently attracted renewed interest as an effic..." /></meta><meta name="Description" content="Phenotypic screens, which focus on measuring and quantifying discrete cellular changes rather than affinity for individual recombinant proteins, have recently attracted renewed interest as an effic..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 22, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b01055" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01055" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b01055" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01055" /></link>
        
    
    

<title>Discovery of a Chemical Probe Bisamide (CCT251236): An Orally Bioavailable Efficacious Pirin Ligand from a Heat Shock Transcription Factor 1 (HSF1) Phenotypic Screen | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b01055" /></meta><meta property="og:title" content="Discovery of a Chemical Probe Bisamide (CCT251236): An Orally Bioavailable Efficacious Pirin Ligand from a Heat Shock Transcription Factor 1 (HSF1) Phenotypic Screen" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0014.jpeg" /></meta><meta property="og:description" content="Phenotypic screens, which focus on measuring and quantifying discrete cellular changes rather than affinity for individual recombinant proteins, have recently attracted renewed interest as an efficient strategy for drug discovery. In this article, we describe the discovery of a new chemical probe, bisamide (CCT251236), identified using an unbiased phenotypic screen to detect inhibitors of the HSF1 stress pathway. The chemical probe is orally bioavailable and displays efficacy in a human ovarian carcinoma xenograft model. By developing cell-based SAR and using chemical proteomics, we identified pirin as a high affinity molecular target, which was confirmed by SPR and crystallography." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b01055"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01055">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b01055&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b01055&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b01055&amp;href=/doi/10.1021/acs.jmedchem.6b01055" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 180-201</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b00954" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b00917" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of a Chemical Probe Bisamide (CCT251236): An Orally Bioavailable Efficacious Pirin Ligand from a Heat Shock Transcription Factor 1 (HSF1) Phenotypic Screen</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+D.++Cheeseman">Matthew D. Cheeseman</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Nicola+E.+A.++Chessum">Nicola E. A. Chessum</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Carl+S.++Rye">Carl S. Rye</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=A.+Elisa++Pasqua">A. Elisa Pasqua</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael+J.++Tucker">Michael J. Tucker</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Birgit++Wilding">Birgit Wilding</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-1896-3708" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lindsay+E.++Evans">Lindsay E. Evans</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Susan++Lepri">Susan Lepri</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Meirion++Richards">Meirion Richards</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Swee+Y.++Sharp">Swee Y. Sharp</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Salyha++Ali">Salyha Ali</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Rowlands">Martin Rowlands</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lisa++O%E2%80%99Fee">Lisa O’Fee</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Asadh++Miah">Asadh Miah</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Angela++Hayes">Angela Hayes</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alan+T.++Henley">Alan T. Henley</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Marissa++Powers">Marissa Powers</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Robert++te+Poele">Robert te Poele</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Emmanuel++De+Billy">Emmanuel De Billy</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Loredana++Pellegrino">Loredana Pellegrino</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Florence++Raynaud">Florence Raynaud</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rosemary++Burke">Rosemary Burke</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rob+L.+M.++van+Montfort">Rob L. M. van Montfort</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Suzanne+A.++Eccles">Suzanne A. Eccles</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paul++Workman">Paul Workman</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Keith++Jones">Keith Jones</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-9440-4094" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Cancer Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Division of Structural Biology at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div><div class="corresp-info"><strong>*</strong>For K.J.: phone, +44 (0) 20 8722 4334; E-mail, <a href="/cdn-cgi/l/email-protection#59123c302d31771336373c2a19303a2b77383a772c32"><span class="__cf_email__" data-cfemail="064d636f726e284c69686375466f657428676528736d">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>For P.W.: E-mail, <a href="/cdn-cgi/l/email-protection#edbd8c9881c3ba829f86808c83ad848e9fc38c8ec39886"><span class="__cf_email__" data-cfemail="f4a4958198daa39b869f99959ab49d9786da9597da819f">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01055&amp;href=/doi/10.1021%2Facs.jmedchem.6b01055" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 180–201</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 23, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>15 July 2016</li><li><span class="item_label"><b>Published</b> online</span>22 December 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 January 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b01055" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01055</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by text">with CC-BY<a href="/page/policy/authorchoice_ccby_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b01055"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4113</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">28</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b01055" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of a Chemical Probe Bisamide (CCT251236): An Orally Bioavailable Efficacious Pirin Ligand from a Heat Shock Transcription Factor 1 (HSF1) Phenotypic Screen&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;D. Cheeseman&quot;},{&quot;first_name&quot;:&quot;Nicola&quot;,&quot;last_name&quot;:&quot;E. A. Chessum&quot;},{&quot;first_name&quot;:&quot;Carl&quot;,&quot;last_name&quot;:&quot;S. Rye&quot;},{&quot;first_name&quot;:&quot;A.&quot;,&quot;last_name&quot;:&quot;Elisa Pasqua&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;J. Tucker&quot;},{&quot;first_name&quot;:&quot;Birgit&quot;,&quot;last_name&quot;:&quot;Wilding&quot;},{&quot;first_name&quot;:&quot;Lindsay&quot;,&quot;last_name&quot;:&quot;E. Evans&quot;},{&quot;first_name&quot;:&quot;Susan&quot;,&quot;last_name&quot;:&quot;Lepri&quot;},{&quot;first_name&quot;:&quot;Meirion&quot;,&quot;last_name&quot;:&quot;Richards&quot;},{&quot;first_name&quot;:&quot;Swee&quot;,&quot;last_name&quot;:&quot;Y. Sharp&quot;},{&quot;first_name&quot;:&quot;Salyha&quot;,&quot;last_name&quot;:&quot;Ali&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Rowlands&quot;},{&quot;first_name&quot;:&quot;Lisa&quot;,&quot;last_name&quot;:&quot;O’Fee&quot;},{&quot;first_name&quot;:&quot;Asadh&quot;,&quot;last_name&quot;:&quot;Miah&quot;},{&quot;first_name&quot;:&quot;Angela&quot;,&quot;last_name&quot;:&quot;Hayes&quot;},{&quot;first_name&quot;:&quot;Alan&quot;,&quot;last_name&quot;:&quot;T. Henley&quot;},{&quot;first_name&quot;:&quot;Marissa&quot;,&quot;last_name&quot;:&quot;Powers&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;te Poele&quot;},{&quot;first_name&quot;:&quot;Emmanuel&quot;,&quot;last_name&quot;:&quot;De Billy&quot;},{&quot;first_name&quot;:&quot;Loredana&quot;,&quot;last_name&quot;:&quot;Pellegrino&quot;},{&quot;first_name&quot;:&quot;Florence&quot;,&quot;last_name&quot;:&quot;Raynaud&quot;},{&quot;first_name&quot;:&quot;Rosemary&quot;,&quot;last_name&quot;:&quot;Burke&quot;},{&quot;first_name&quot;:&quot;Rob&quot;,&quot;last_name&quot;:&quot;L. M. van Montfort&quot;},{&quot;first_name&quot;:&quot;Suzanne&quot;,&quot;last_name&quot;:&quot;A. Eccles&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;Workman&quot;},{&quot;first_name&quot;:&quot;Keith&quot;,&quot;last_name&quot;:&quot;Jones&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;22&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;180-201&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b01055&quot;},&quot;abstract&quot;:&quot;Phenotypic screens, which focus on measuring and quantifying discrete cellular changes rather than affinity for individual recombinant proteins, have recently attracted renewed interest as an efficient strategy for drug discovery. In this article, we describe the discovery of a new chemical probe, bisamide (CCT251236), identified using an unbiased phenotypic screen to detect inhibitors of the HSF1 stress pathway. The chemical probe is orally bioavailable and displays efficacy in a human ovarian carcinoma xenograft model. By developing cell-based SAR and using chemical proteomics, we identified pirin as a high affinity molecular target, which was confirmed by SPR and crystallography.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01055&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01055" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01055&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01055" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01055&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01055" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01055&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01055&amp;href=/doi/10.1021/acs.jmedchem.6b01055" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b01055" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b01055" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b01055%26sid%3Dliteratum%253Aachs%26pmid%3D28004573%26genre%3Darticle%26aulast%3DCheeseman%26date%3D2017%26atitle%3DDiscovery%2Bof%2Ba%2BChemical%2BProbe%2BBisamide%2B%2528CCT251236%2529%253A%2BAn%2BOrally%2BBioavailable%2BEfficacious%2BPirin%2BLigand%2Bfrom%2Ba%2BHeat%2BShock%2BTranscription%2BFactor%2B1%2B%2528HSF1%2529%2BPhenotypic%2BScreen%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D1%26spage%3D180%26epage%3D201%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (5)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291465" title="Structure activity relationship">Structure activity relationship</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/1" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/jmcmar.2017.60.issue-1/20170112/jmcmar.2017.60.issue-1.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Phenotypic screens, which focus on measuring and quantifying discrete cellular changes rather than affinity for individual recombinant proteins, have recently attracted renewed interest as an efficient strategy for drug discovery. In this article, we describe the discovery of a new chemical probe, bisamide (CCT251236), identified using an unbiased phenotypic screen to detect inhibitors of the HSF1 stress pathway. The chemical probe is orally bioavailable and displays efficacy in a human ovarian carcinoma xenograft model. By developing cell-based SAR and using chemical proteomics, we identified pirin as a high affinity molecular target, which was confirmed by SPR and crystallography.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25664" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25664" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Despite the recent extraordinary progress seen in cancer therapy using molecularly targeted drugs, the disease commonly remains resistant to effective long-term treatment. Even when excellent responses to drugs are initially observed, resistance is almost inevitable and patients are then left with few treatment options<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> as the discovery of targeted therapies in oncology has focused on relatively few protein families.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> To break this cycle and expand the treatment options for cancer patients, new approaches are needed to discover novel druggable protein targets.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a></div><div class="NLM_p">In phenotypic screens, small molecules undergo high-throughput screening against intact cells, rather than recombinant proteins, and discrete phenotypic changes in the cell are measured and quantified.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Interest in phenotypic screens has increased significantly in recent years due to their potential to effectively discover new drugs.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Phenotypic screens have several advantages over screens using recombinant proteins. Hits from a phenotypic screen will, by definition, be cell permeable and have cellular activity, potentially reducing optimization cycles and timelines. Also, because the screening approach is unbiased, established knowledge of the biology of molecular targets is not required. Finally, polypharmacology is often observed with small molecules and structurally related protein families; this can be crucial for efficacy and is perfectly compatible with a phenotypic screening approach.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">In contrast, progressing hits from a phenotypic screen can generate a number of unique challenges. Discovering pharmacodynamic (PD) biomarkers in vivo for use in animals can be difficult when developing hits from phenotypic screens, as the pathways commonly need to be activated with an external stimulus.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Cell-based screens are typically more expensive and time-consuming and so may require a greater commitment prior to the screening campaign.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Furthermore, molecular target identification, deconvolution, and validation are crucial steps if new chemical probes<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and drugs are to be discovered. These are often a bottleneck in phenotypic screening.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Even with successful target deconvolution, the target discovered may not be of interest; for example, the target may already be drugged or be a known antitarget. Finally, polypharmacology may be a serious impediment to compound progression because the interaction with multiple structurally related protein targets may prove impossible to deconvolute.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> These challenges alter the balance between prioritizing druglike properties of compounds and an efficient target identification strategy.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><div class="NLM_p last">To execute a successful phenotypic screening campaign, it is critical to select an appropriate phenotype for small molecule intervention. HSF1 is a transcription factor and the master regulator of the ancient, canonical heat shock response.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> A large body of work has verified the importance of HSF1 to tumorigenesis and cancer progression.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> HSF1 has been proposed to be activated by various elements of the cancer state, potentially reprogramming the transcriptome in a way that is overlapping with, but distinct from, the heat shock response.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Also, a strong correlation has been reported between the expression of activated HSF1 in tumors and adverse clinical outcomes.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> This evidence indicates that the inhibition of HSF1-mediated transcription could be a viable strategy in cancer treatment.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Moreover, inhibiting the HSF1 stress pathway would represent an attempt at targeting non-oncogene addiction and proteotoxic stress, which has been proposed to be advantageous.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> However, HSF1 is a ligandless transcription factor and so is unlikely to be amenable to standard drug discovery strategies and direct inhibition with small molecules. Therefore, we proposed that an inhibitor of HSF1-mediated transcription, which antagonized the HSF1 pathway but without necessarily binding directly to HSF1, could be discovered and developed via a cell-based phenotypic screen.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86045" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86045" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> HSF1 Phenotypic Assay</h3><div class="NLM_p">To observe HSF1-mediated transcription in an in vitro setting, the HSF1 pathway is activated by a validated heat shock protein 90 (HSP90) inhibitor,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> or another form of external stress,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> which initiates the heat shock response. Commonly, the output of the heat-shock response is quantified by measuring the induction of heat shock 70 kDa protein 1 (HSP72) expression, the stress-inducible HSP70 isoform.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> HSF1 pathway inhibitors are then defined by their ability to block the induction of HSP72. Several HSF1-mediated HSP72 induction inhibitors have been discovered via this method with different proposed molecular mechanisms of action (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).<a onclick="showRef(event, 'ref21 ref28'); return false;" href="javascript:void(0);" class="ref ref21 ref28">(21, 28)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Inhibitors of HSF1-mediated HSP72 induction.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">With the aim of discovering novel hit-matter that inhibits HSF1-mediated transcription, we previously carried out a cell-based high-throughput phenotypic screen in U2OS human osteosarcoma cells of ∼200000 compounds, including ∼35000 compounds from a kinase-focused library.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The screen quantified the inhibition of HSP72 induction using the Arrayscan assay<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> following treatment with the HSP90 inhibitor tanespimycin (17-AAG).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> The Bisamide Series</h3><div class="NLM_p">Using this screen, we identified a potent hit from the kinase-focused deck, bisamide <b>1</b> (CCT245232) (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Following resynthesis, the screening hit was confirmed and displayed a pIC<sub>50</sub> = 8.55 ± 0.09 (IC<sub>50</sub> = 2.8 nM, <i>n</i> = 49)<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> in our HSP72 cell-based enzyme-linked immunosorbent assay (ELISA) in U2OS cells. The HSP72 cell-based ELISA assay is an alternative assay format to the Arrayscan assay for quantifying the induced expression of HSP72 and was used as our primary phenotypic pathway assay throughout the study.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The IC<sub>50</sub> was defined as the concentration that inhibited the signal to 50% of the 17-AAG (250 nM) induced HSP72 expression, relative to the control 17-AAG alone (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for details). The clear structural feature defining this chemotype was the <i>N</i>,<i>N</i>′-4-methyl-1,3-phenylenediamide core (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. HSF1 pathway inhibitor, bisamide <b>1</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We had previously demonstrated that pan cyclin-dependent kinase (CDK) inhibitors, and potent CDK9 inhibitors in particular that act as transcription antagonists,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> can inhibit the HSF1-mediated HSP72 induction phenotype; we initially suspected a similar mechanism for this chemotype.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> However, upon biochemical screening of bisamide <b>1</b> against CDK2 and CDK9, no inhibition was observed (<10% inhibition at 1 μM, see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). Therefore, we hypothesized that bisamide <b>1</b> was acting through a different mechanism of action and consequently <b>1</b> was submitted for further characterization.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Hit Characterization: Kinase Activity</h3><div class="NLM_p last">HSF1 is regulated by multiple post-translational phosphorylations,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> so we hypothesized that kinase inhibition, other than CDK2 and CDK9, was causing the observed HSF1-mediated HSP72 induction inhibition. Bisamide <b>1</b> was screened against a broad kinase set using the KINOMEscan biochemical screening platform, a binding assay which gave single-point percentage inhibition values at 1 μM (DiscoverX, <a href="http://www.discoverx.com" class="extLink">http://www.discoverx.com</a>).<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Analysis of this data set for 442 kinases revealed that bisamide <b>1</b> inhibited only nine kinases >50% and four kinases >90% (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for details). To validate the kinase hits, orthogonal functional assays were carried out on the four kinases that displayed >90% inhibition. Of these, KIT, PDGFRA, and PDGFRB failed to confirm in their respective functional assays, all returning pIC<sub>50</sub> < 5 (IC<sub>50</sub> > 10000 nM). Bisamide <b>1</b> did display modest activity against BRAF, returning a pIC<sub>50</sub> = 6.38 (IC<sub>50</sub> = 420 nM, <i>n</i> = 1).<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Although it seemed unlikely that the modest biochemical BRAF activity would translate into cellular activity, seven commercially available<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> potent BRAF inhibitors of differing chemotypes were tested in our cellular HSF1-mediated HSP72 induction inhibition assay for comparison and none displayed pIC<sub>50</sub> > 5 (IC<sub>50</sub> < 10000 nM).<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> This indicated that BRAF inhibition was not an important contributor to the observed HSF1 transcription inhibition phenotype (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>).</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Hit Characterization: Cellular</h3><div class="NLM_p last">To establish whether bisamide <b>1</b> displays cellular activity without activation of the heat shock response using the HSP90 inhibitor 17-AAG, the growth inhibitory effects were assayed in the U2OS cell line using the CellTiter Blue (CTB) assay (Promega). Following 4 days of treatment, bisamide <b>1</b> displayed a highly potent pGI<sub>50</sub> = 7.74 ± 0.08 (GI<sub>50</sub> = 18 nM, <i>n</i> = 13), when compared to vehicle. To assess the broader single agent cellular activity of <b>1</b>, the compound was assayed for growth inhibition against a large, genetically diverse panel of human cancer cell lines (Genomics of Drug Sensitivity in Cancer, <a href="http://www.cancerrxgene.org" class="extLink">www.cancerrxgene.org</a>).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Of the 635 cell lines assayed, 628 (99%) displayed a pGI<sub>50</sub> > 6 (GI<sub>50</sub> < 1000 nM), 455 (72%) a pGI<sub>50</sub> > 7 (GI<sub>50</sub> < 100 nM), and 18 (2.8%) a pGI<sub>50</sub> > 8 (GI<sub>50</sub> < 10 nM), with no clear selectivity against any specific tissue type (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for details). These results suggested that bisamide <b>1</b> had wide-ranging anticancer activity and that activation of the HSF1 pathway with an HSP90 inhibitor was not required for <b>1</b> to inhibit cancer cell growth.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Cell-Based SAR</h3><div class="NLM_p">Given the complexity of factors underlying cell-based structure-activity relationships (SAR), we focused on matched pair changes to establish which features were crucial for the cellular activity of the bisamide chemotype. There were two primary aims to our medicinal chemistry strategy. First, because our biochemical kinase screening had failed to reveal any potential kinase targets, a broader target identification strategy would need to be developed. Second, although bisamide <b>1</b> had excellent cellular potency, the compound had poor aqueous solubility (kinetic solubility <2 μM), which would need to be addressed to validate HSF1 pathway inhibition as an anticancer strategy in an in vivo human tumor xenograft model. Both goals required us to improve our understanding of the cellular SAR, particularly to identify a vector-to-solvent through which either a chemical probe linker or a solubilizing group could be attached, without disrupting the ability of the compound to bind to the primary pharmacological target or targets.</div><div class="NLM_p">To develop the cell-based SAR, we switched to the human ovarian carcinoma cell line SK-OV-3, which was relatively sensitive to growth inhibition and is commonly used in translational drug discovery both in vitro<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> and in vivo.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> When the activity of bisamide <b>1</b> in the SK-OV-3 cell line was assayed, it gave a pIC<sub>50</sub> = 7.17 ± 0.07 (IC<sub>50</sub> = 68 nM, <i>n</i> = 4) using the HSP72 cell-based ELISA assay induced with 250 nM 17-AAG. Bisamide <b>1</b> also inhibited the proliferation of SK-OV-3 cells in the CTB assay with a pGI<sub>50</sub> = 8.08 ± 0.12 (GI<sub>50</sub> = 8.4 nM, <i>n</i> = 12).</div><div class="NLM_p">We first focused the exploration on the benzodioxane motif (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). The left-hand ring system analogues (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entries 1–6) were synthesized via a three-step procedure from the commercially available 2-methyl-5-nitroaniline <b>2</b>. Following amide bond formation, through the reaction of 2-methyl-5-nitroaniline <b>2</b> with 2-methylquinoline-6-carboxylic acid <b>3</b> and subsequent reduction of the nitro-group using iron, the second amide bond was generated using the corresponding carboxylic acid in a HATU-mediated coupling to afford the various benzodioxane analogues in good to moderate yields.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0008.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of the Benzodioxane Bisamide Replacements<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) oxalyl chloride, DMF, DCM, then, pyridine, DCM; (ii) Fe, NH<sub>4</sub>Cl, EtOH/H<sub>2</sub>O; (iii) RPhCO<sub>2</sub>H, HATU, DIPEA, DMF.</p></p></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Bisamide Analogues</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0010.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">All results are quoted as the geometric mean ± SEM, pIC<sub>50</sub>/pGI<sub>50</sub>/p<i>K</i><sub>D</sub> = −log IC<sub>50</sub>/GI<sub>50</sub>/<i>K</i><sub>D</sub> (M). The number of repeats, <i>n</i>, are described in parentheses. ND = not determined.</p></div><div class="footnote" id="t1fn2"><sup>Table b</sup><p class="last">Cell-based ELISA assay for inhibition of HSP72 induction; SK-OV-3 cells were pretreated with compound at the relevant concentration for 1 h before the addition of 250 nM 17-AAG. HSP72 levels were then quantified after 18 h.</p></div><div class="footnote" id="t1fn3"><sup>Table c</sup><p class="last">Growth inhibition was measured after 96 h of treatment and compared to vehicle control.</p></div><div class="footnote" id="t1fn4"><sup>Table d</sup><p class="last">p<i>K</i><sub>D</sub> values were measured through analysis of the sensorgram at equilibrium where possible. Values were then fitted to a one-site specific binding model using Graphpad Prism Version 6.</p></div></div><div></div></div><div class="NLM_p">Changing the 5-substituted dioxane isomer <b>1</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 1) to the 6-substituted dioxane isomer <b>6</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 2) resulted in a complete loss of cellular activity (pIC<sub>50</sub> < 5), indicating that the isomer of the benzodioxane was crucial. The acyclic dimethoxy analogue <b>7</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 3) also displayed a complete loss of cellular activity. Next, we investigated the role of the two oxygen atoms of the dioxane ring. Removing the <i>para</i>-oxygen of the dioxane to give the <i>meta</i>-chroman <b>8</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 4) resulted in a modest 5-fold decrease in cellular HSF1-mediated HSP72 induction inhibition and a 7-fold decrease in growth inhibition compared to <b>1</b>; while the <i>para</i>-chroman isomer <b>9</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 5) displayed a further 6-fold decrease in HSF1-mediated HSP72 induction inhibition and an 8-fold decrease in growth inhibition compared to <b>8</b>, which represents a 37- and 51-fold decrease in cellular activity respectively compared to the original hit, bisamide <b>1</b>. Surprisingly, isochroman <b>10</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 6) displayed no measurable cellular activity. From these results, it was clear that the benzodioxane moiety was crucial for the cellular activity of the bisamide series. Because the SAR surrounding the benzodioxane was steep and complex, we decided that this region was unlikely to be solvent exposed and so would be incompatible with linker and solubilizing group attachment.</div><div class="NLM_p">Second, we investigated the role of the central ring and amide moieties in the cellular activity of the bisamide chemotype. We hypothesized that the amide groups could play an important role in hydrogen bonding with a potential target and in controlling the overall shape of the ligand, while substitution at the central ring could lead to the discovery of solvent-exposed vectors. The synthesis of these analogues (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entries 7–10) was similar to that described in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Substitution at both the 6-position (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 7) and the 4-position (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 8) of the central benzene ring resulted in a complete loss of cellular activity, suggesting that neither of these vectors represented a viable route to solvent. Owing to the potential for the toluene methyl group to affect the conformation of the right-hand amide, we decided to leave this group unchanged. <i>N</i>-Methyl substitution of the left-hand amide (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 9) and an attempted sulfonamide bioisosteric replacement (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 10) also resulted in a complete loss of cellular activity. Again, owing to the steep SAR in this region of the chemotype, it was clearly incompatible with linker and solubilizing group attachment.</div><div class="NLM_p last">Finally, we examined the role of the 2-methylquinoline ring in the cellular activity of the bisamide series. All analogues (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entries 11–13) were synthesized via a method similar to that described in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. First, we removed the 2-methyl group, which is both lipophilic and has the potential to be a weak hydrogen bond donor, to give quinoline <b>15</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 11). Removal of this group resulted in no significant change in the cellular activity when compared to <b>1</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 1). Next we moved the quinoline nitrogen from the 1- to the 3-position to give isoquinoline <b>16</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 12). Again, no significant change in cellular activity was observed when compared to bisamide <b>1</b>. Finally, we partially reduced the quinoline ring to give tetrahydroquinoline <b>17</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 13). By changing to tetrahydroquinoline <b>17</b>, the nitrogen atom now acts as a hydrogen bond donor rather than an acceptor. Despite this reversal in anticipated binding properties, only a 3.5-fold decrease in HSF1-mediated HSP72 induction inhibition and no significant change in antiproliferative activity was observed when compared to bisamide <b>1</b>. The broad SAR of the quinoline region of the molecule suggested that this moiety is solvent exposed and so could be exploited for solubilizing group and linker attachment.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Exploiting Vectors-to-Solvent: Solubilizing Group Optimization</h3><div class="NLM_p">Owing to the complexities of the cellular SAR, attaching a solubilizing group to the solvent-exposed region of the ligand was considered a more expeditious strategy to generate an in vivo chemical probe suitable for use in animal models rather than attempting to optimize the intrinsic solubility of the bisamide series without insight from structural information. To address this objective, we adopted an iterative strategy to exploit the rapid synthesis of analogues that would exhibit improved physicochemical properties and demonstrate the appropriate mouse pharmacokinetic (PK) parameters for in vivo study (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0009.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Solubilizing Group Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) oxalyl chloride, DMF, 1,4-benzodioxane-6-carboxylic acid, DCM, then, 4-methyl-3-nitroaniline, pyridine, DCM, RT, then Pd/C (10%), H<sub>2</sub> (1 atm), EtOH; (ii) NaH, RCH<sub>2</sub>OH, THF, 0 °C to reflux; (iii) <i>n</i>-BuLi, CO<sub>2</sub>(s), THF, −78 °C; (iv) HATU, DIPEA, DMF, RT.</p></p></figure><div class="NLM_p">The optimal route to the synthesis of these analogues would be to introduce the solubilizing group via nucleophilic aromatic substitution on the quinoline as the final step; however, this gave consistently low yields. Therefore, the solubilizing group was introduced at the start of the synthesis. Formation of the precursor carboxylic acid <b>19</b> was achieved via lithium halogen exchange of the corresponding aryl bromide <b>20</b>. The resulting bisamides were then assayed for cellular activity and in vitro PK parameters (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Optimization of the Bisamide Solubilizing Group</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0011.gif" alt="" id="fx2" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0012.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">pGI<sub>50</sub> = −log GI<sub>50</sub> (M); geometric mean ± SEM, <i>n</i> = number of biological repeats in parentheses. For the corresponding SK-OV-3 HSP72 cell-based ELISA data See <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</p></div><div class="footnote" id="t2fn2"><sup>Table b</sup><p class="last">Mouse liver microsome (MLM) assay was carried out at Cyprotex, arithmetic mean of <i>n</i> = 2. In vitro Cl<sub>int</sub> is calculated from the half-life using standard procedures and assumes the fraction unbound in the assay is 1.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></p></div><div class="footnote" id="t2fn3"><sup>Table c</sup><p class="last">Kinetic solubility (KS) measured via an in-house HPLC method from phosphate buffer at pH 7.4, all values quoted to 1 SF, the dynamic range of the assay is 1–100 μM, arithmetic mean of <i>n</i> = 2.</p></div><div class="footnote" id="t2fn4"><sup>Table d</sup><p class="last">Measured via an in-house HPLC method, arithmetic mean of <i>n</i> = 2, all values quoted to 2 SF.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></p></div><div class="footnote" id="t2fn5"><sup>Table e</sup><p class="last">MoKa version 2.5.2, all values quoted to 2 SF. See <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for details.</p></div></div><div></div></div><div class="NLM_p last">We first attached the oxygen-linked ether chain to give glycol <b>21</b> (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, entry 2). No significant decrease in either HSF1-mediated HSP72 induction inhibition (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>) or the antiproliferative activity was observed when compared to lead bisamide <b>1</b> (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, entry 1). This suggested first that the oxygen linker had no detrimental effect on activity and, second, that this region of the molecule was indeed solvent exposed. To carry out multiparameter optimization of the solubilizing group, we used four in vitro properties to assess each compound’s potential for in vivo mouse PK: microsomal stability, kinetic solubility at pH 7.4, lipophilicity as measured by Log<i>D</i><sub>7.4</sub>, and the predicted basicity of the solubilizing group. The glycol <b>21</b> displayed an increase in lipophilicity compared to lead bisamide <b>1</b>; this increase was reflected in the 2.4-fold decrease in microsomal stability with a modest increase in kinetic solubility. However, glycol <b>21</b> remained a low solubility compound, which is inconsistent with good oral bioavailability. From these data, we concluded that the compound would need to be charged to deliver the correct balance of properties. To introduce a charged moiety, we first focused on the dimethylamine group with a 3-carbon linker. Dimethylamine <b>22</b> (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, entry 3) displayed no significant change in cellular activity despite this analogue being predominately charged at physiological pH, according to the calculated p<i>K</i><sub>a</sub>. Consistent with its cationic character, dimethylamine <b>22</b> displayed a 1.4 log unit decrease in lipophilicity when compared to <b>21</b>, which was accompanied by an 18-fold improvement in kinetic solubility. Unfortunately, the decrease in lipophilicity was not reflected in an improvement in microsomal stability. We hypothesized that this was due to oxidation of the <i>N</i>-methyl groups, so we moved to cyclic amines to reduce the CYP450 mediated degradation. Morpholine <b>23</b> (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, entry 4) showed little improvement in microsomal stability or kinetic solubility when compared to <b>21</b>. Analysis of the Log<i>D</i><sub>7.4</sub> and calculated p<i>K</i><sub>a</sub> for morpholine <b>23</b> suggested that the solubilizing group was not sufficiently basic to balance the physicochemical properties of the compound. Moving to piperidine <b>24</b> (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, entry 5), we observed a better balance in physicochemical properties, with the reduction in lipophilicity reflected in improvements in both microsomal stability and kinetic solubility. However, piperidine <b>24</b> was predicted to be highly basic and we were concerned that this would have a detrimental effect on permeability and therefore on oral bioavailability. To reduce the basicity of the solubilizing group without increasing lipophilicity, we reduced the linker length to give <b>25</b> (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, entry 6), thereby exploiting the inductive effect of the oxygen. Despite the 0.9 log unit decrease in predicted basicity, the kinetic solubility and lipophilicity were unchanged. Unfortunately, <b>25</b> displayed a decrease in microsomal stability, so we reduced the ring size to give pyrrolidine <b>26</b> (CCT251236, <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, entry 7),<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> which displayed the desired balance of in vitro properties, while maintaining excellent cellular activity with a pIC<sub>50</sub> = 7.73 ± 0.07 (IC<sub>50</sub> = 19 nM, <i>n</i> = 15, see Supporting Information, <a class="ref internalNav" href="#notes-1" aria-label="Table S3">Table S3</a> for details) for inhibition of HSF1-mediated HSP72 induction. The free GI<sub>50</sub> in SK-OV-3 cells was then calculated from the free fraction in the cell assay, which gave a free GI<sub>50</sub> = 1.1 nM.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Western blotting confirmed that pyrrolidine <b>26</b> blocked the HSF1-mediated induction of both HSP72 and HSP27 as representative heat shock proteins, following treatment with the HSP90 inhibitor 17-AAG. Also, qPCR analysis demonstrated that <b>26</b> inhibited the induction of HSP72 at the mRNA level, clearly blocking the induction of <i>HSPA1A</i> gene expression with a pIC<sub>50</sub> = 7.40 (IC<sub>50</sub> = 40 nM, <i>n</i> = 1, see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). Taking these results together, pyrrolidine <b>26</b> was selected for in vivo study.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Mouse Pharmacokinetics (PK)</h3><div class="NLM_p">To assess the potential of bisamide <b>26</b> as an in vivo chemical tool to study HSF1-mediated transcriptional activity inhibition, the compound was dosed in BALB/c mice at 5 mg/kg as an oral solution and iv bolus. Blood concentrations were then measured over a 24 h period (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Mouse Blood PK Parameters for Bisamide <b>26</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="/" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">mouse</th><th class="colsep0 rowsep0" align="center" char="/">dose po/iv (mg/kg)</th><th class="colsep0 rowsep0" align="center">AUC<sub class="stack">0–24h</sub><sup class="stack">PO</sup> (h·nM)<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">blood Cl (mL/min/kg)<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><i>V</i><sub>ss</sub> (L/kg)<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">half-life (h)</th><th class="colsep0 rowsep0" align="center">%<i>F</i></th><th class="colsep0 rowsep0" align="center" char="."><i>f</i><sub>ub</sub><a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub class="stack">u,0–24h</sub><sup class="stack">PO</sup> (h·nM)<a class="ref internalNav" href="#t3fn6" aria-label="f">f</a></th><th class="colsep0 rowsep0" align="center">Cl<sub>u</sub> (mL/min/kg)<a class="ref internalNav" href="#t3fn7" aria-label="g">g</a></th><th class="colsep0 rowsep0" align="center"><i>V</i><sub>du</sub> (L/kg)<a class="ref internalNav" href="#t3fn8" aria-label="h">h</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BALB/c</td><td class="colsep0 rowsep0" align="char" char="/">5/5</td><td class="colsep0 rowsep0" align="center">5800 (8100–4200)</td><td class="colsep0 rowsep0" align="center">9.2 (12–7.3)</td><td class="colsep0 rowsep0" align="center">4.2 (6.3–2.8)</td><td class="colsep0 rowsep0" align="center">5.3</td><td class="colsep0 rowsep0" align="center">39</td><td class="colsep0 rowsep0" align="char" char=".">0.0083<a class="ref internalNav" href="#t3fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="char" char=".">48</td><td class="colsep0 rowsep0" align="center">1100</td><td class="colsep0 rowsep0" align="center">510</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">athymic</td><td class="colsep0 rowsep0" align="char" char="/">20/0</td><td class="colsep0 rowsep0" align="center">1900 (2800–1400)</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="char" char=".">0.015<a class="ref internalNav" href="#t3fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="center">NA</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">All values are quoted to 2 SF. The 90% confidence intervals (CI) are in parentheses. NA = not applicable.</p></div><div class="footnote" id="t3fn2"><sup>b</sup><p class="last">The geometric mean of <i>n</i> = 3 individual mice.</p></div><div class="footnote" id="t3fn3"><sup>c</sup><p class="last"><i>f</i><sub>ub</sub> = <i>f</i><sub>up</sub>/B:P.</p></div><div class="footnote" id="t3fn4"><sup>d</sup><p class="last"><i>f</i><sub>up</sub> = 0.010 (0.011–0.0097), B:<a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="51011160">[email protected]</a> μM = 1.2:1 (1.4–1.0).</p></div><div class="footnote" id="t3fn5"><sup>e</sup><p class="last"><i>f</i><sub>up</sub> = 0.025 (0.030–0.020), B:<a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="93c3d3a2">[email protected]</a> μM = 1.7:1 (1.7–1.7).</p></div><div class="footnote" id="t3fn6"><sup>f</sup><p class="last">AUC<sub>u</sub> = AUC·f<sub>ub</sub>.</p></div><div class="footnote" id="t3fn7"><sup>g</sup><p class="last">Cl<sub>u</sub> = Cl/<i>f</i><sub>ub</sub>.</p></div><div class="footnote" id="t3fn8"><sup>h</sup><p class="last"><i>V</i><sub>du</sub> = <i>V</i><sub>ss</sub>/f<sub>ub</sub>.</p></div></div></div><div class="NLM_p last">Analysis of the mouse PK data revealed that bisamide <b>26</b> possessed low total blood clearance (10% hepatic blood flow)<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and moderate oral bioavailability, with a half-life sufficient to allow once-daily dosing. In vitro assessment of the plasma protein binding and the blood to plasma ratio revealed that <b>26</b> was highly bound to plasma proteins (∼99%); therefore, the unbound clearance was high, with a low free exposure from the 5 mg/kg oral dose, equivalent to a free <i>C</i><sub>av</sub><sup>0–24h</sup> = 2.0 nM.<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41, 42)</a> The high unbound volume of distribution indicates that <b>26</b> readily binds to tissues, consistent with the basicity of the compound.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Bisamide <b>26</b> Displays Efficacy in a Human Ovarian Carcinoma Xenograft Model</h3><div class="NLM_p">Despite the high unbound clearance of bisamide <b>26</b> in mouse, the good oral bioavailability, half-life, and excellent in vitro cellular activity (free GI<sub>50</sub> = 1.1 nM) encouraged us to test the potential efficacy of <b>26</b> in an in vivo human tumor xenograft model. On the basis of the free <i>C</i><sub>av</sub><sup>0–24h</sup> = 2.0 nM observed in nontumor bearing immunocompetent BALB/c mice following the 5 mg/kg po qd dose, a 20 mg/kg po qd dose in immunodeprived athymic mice was selected to cover ∼10 times the in vitro free GI<sub>50</sub> in SK-OV-3 cells.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> To our surprise, following this 20 mg/kg po dose, the free exposure was actually a disappointing AUC<sub>u</sub><sup>0–24h</sup> = 29 h·nM, equivalent to a free <i>C</i><sub>av</sub><sup>0–24h</sup> = 1.2 nM (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>, see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for details). Despite the lower than expected free exposure of bisamide <b>26</b>, this dose still represented coverage of the in vitro free GI<sub>50</sub> in SK-OV-3 cells and was well tolerated in a mouse multidose tolerability study;<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> therefore, the 20 mg/kg po qd dose was selected for further investigation.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></div><div class="NLM_p">SK-OV-3 cells were injected subcutaneously into athymic mice for tumor formation. Once tumors were established, the mice were randomized into treatment and control groups and were dosed orally once-a-day with either vehicle or 20 mg/kg of bisamide <b>26</b>. Tumor volumes and mouse body weights were measured throughout and tumor weights were measured at the end of the study, while total tumor concentrations were measured 2 and 6 h post final dose (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a> and <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>).</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Efficacy of bisamide <b>26</b> against SK-OV-3 human ovarian carcinoma xenograft model. Blue, vehicle control, <i>n</i> = 8; red, <b>26</b>, 20 mg/kg po qd, <i>n</i> = 8 (vehicle = 10% DMSO, 90% of a 25% (2-hydroxypropyl)-β-cyclodextrin in 50 mM citrate buffer pH 5). Error bars: arithmetic mean ± SEM. Dosing breaks were carried out on days 5–12, 14, 16, 18, 20, 22, 24, 26, 29, 31.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The mice were dosed intermittently throughout the study with <b>26</b> to maintain their condition, as assessed through monitoring body weights (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). Clear therapeutic efficacy was observed with bisamide <b>26</b>, with a tumor growth inhibition (%TGI)<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> of 70% based on final tumor volumes. The study was terminated after 33 days, and comparison of the control and treated arms indicated a 64% reduction in mean tumor weights (<i>p</i> = 0.015)<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> with total tumor concentrations of <b>26</b> as high as 940 nM, consistent with the compound’s basicity and high volume of distribution (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>).</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Target Identification</h3><div class="NLM_p">Following the successful efficacy study, the potential for bisamide <b>26</b> as a chemical probe to study the effects of HSF1 transcription inhibition, both in vitro and in vivo, were clear. However, the variety of HSF1 transcription inhibitors in the literature<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> suggests there are multiple mechanisms through which this phenotype may be observed, each with differing potential for drug discovery.</div><div class="NLM_p">To decipher the molecular mechanism of the bisamide series, we needed to discover their protein targets. The potency of the bisamide <b>26</b> in cell-based assays suggests that only high affinity efficacy and epistatic targets would be of interest.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Our biochemical kinase screening had indicated that there were no high affinity kinase targets. We therefore expanded our biochemical screening to include other protein families. Bisamide <b>26</b> was submitted to the Cerep Diversity Screen (Cerep, <a href="http://www.cerep.fr" class="extLink">http://www.cerep.fr</a>) comprising 98 molecular targets, including receptors and enzymes, measured at 10 μM.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Analysis of these screening data revealed that bisamide <b>26</b> displays a good selectivity profile. Only six targets (the receptors: adenosine A2A and A3, histamine H2 and H3, muscarinic, and the enzyme acetylcholine esterase) displayed >80% inhibition, and these generally represented the highly promiscuous receptor protein targets (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for details). No hits displayed sufficient activity to clearly relate the molecular target to the efficacious free concentrations achieved in vitro or in vivo.</div><div class="NLM_p">Because no clear protein families had been revealed from our biochemical screening, a different approach was necessary. We decided to exploit a chemical proteomics strategy and using our knowledge of the cellular SAR, we designed a protein pull-down chemical probe to identify high affinity molecular targets from a human cancer cell lysate (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>).</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Tool compounds for target identification. All quoted cellular activities are in the SK-OV-3 cell line, the numbers of repeats are in parentheses.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To carry out the chemical proteomics pull-down strategy, the bisamide warhead would need to be attached via a linker to a solid-phase bead, without disrupting binding to the molecular target or targets. The solubilizing group vector was the obvious choice for the linker attachment, but to test whether cellular activity was maintained, amide <b>27</b> was designed as a cell-permeable mimic of the solid-phase probe. Pleasingly, amide <b>27</b> maintained excellent cellular potency (SK-OV-3 pGI<sub>50</sub> = 8.04 ± 0.06, GI<sub>50</sub> = 9.1 nM, <i>n</i> = 4), so amine <b>29</b> was used for attachment to the bead via amide bond formation. In addition to bisamide <b>26</b>, tetrahydroquinoline <b>17</b> was selected as a second positive control due to its distinct structural difference. These two compounds would be used to displace specific molecular targets from the solid-phase probe. Also, to help distinguish false positives from the pull-down experiment, a physicochemically matched negative control was designed. The dioxane isomer <b>6</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 2) had previously been shown to lose all cellular activity. To physicochemically match the dioxane isomer <b>6</b> to bisamide <b>26</b>, the same ethoxy pyrrolidine solubilizing group was attached to give isomer <b>28</b>, which, as expected, displayed no measurable cellular activity.</div><div class="NLM_p">With all four probe molecules in hand, the compounds were submitted to the stable isotope labeling by amino acids in cell culture (SILAC) mass-spectrometry based pull-down assay (Evotec, <a href="http://https:www.evotec.com" class="extLink">https:www.evotec.com</a>) to identify and quantify molecular targets of the bisamide series from the lysate of SK-OV-3 cells. Each protein captured by the bead, whether specific or nonspecific, was analyzed in this methodology by quantitative mass spectrometry. By using SILAC to quantify the relative amounts of protein captured from the lysate, this approach can, in principle, determine the apparent affinity of every stable protein in the lysate for the surface-bound bisamide probe, although particularly low abundance proteins may not be discovered via this method and proteins that are unstable to the lysis conditions would not be detected. Displacement of the captured proteins with the free active bisamides <b>26</b> and <b>17</b> then allowed for estimation of their apparent affinities for specific protein targets. Proteins that were apparently displaced from the bead-bound probe by the inactive isomer <b>28</b> were considered nonspecific or irrelevant for the cellular activity and discarded. The putative protein targets could then be ranked based on their affinity for further investigation (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>, see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for details).<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Molecular Targets from the Pull-down Assay Using the Bisamide Probes in SK-OV-3 Cell Lysate<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0013.gif" alt="" id="fx4" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="center" /></col><col align="char" char="." /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">protein</th><th class="colsep0 rowsep0" align="center" char=".">protein EC<sub>50</sub> (μM)<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char="."><b>26</b> AC<sub>50</sub> (μM)<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center"><b>26</b><i>K</i><sub>i</sub><sup>app</sup> (μM)<a class="ref internalNav" href="#t4fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center" char="."><b>17</b> AC<sub>50</sub> (μM)<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center"><b>17</b> K<sub>i</sub><sup>app</sup> (μM)<a class="ref internalNav" href="#t4fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center"><b>28</b> AC<sub>50</sub> (μM)<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GSK3β</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="center">NA</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="center">NA<a class="ref internalNav" href="#t4fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="center">NA<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Pirin</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="center">0.028</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="center">0.19</td><td class="colsep0 rowsep0" align="center">NA<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">(A) SK-OV-3 cell lysate protein binding curves for immobilized <b>29</b>. (B) Protein displacement curves for pyrrolidine <b>26</b>. Curves are an average of two different mixing conditions. EC<sub>50</sub> and AC<sub>50</sub> are determined from analysis of the curves without limits.</p></div><div class="footnote" id="t4fn2"><sup>b</sup><p class="last">EC<sub>50</sub> is the apparent immobilized bisamide probe concentration at which 50% of the available protein is bound. The probe bisamide is assumed to be >10-fold molar excess over proteins in the lysate.</p></div><div class="footnote" id="t4fn3"><sup>c</sup><p class="last">AC<sub>50</sub> is the apparent bisamide concentration at which 50% of the protein is displaced from the immobilized bisamide probe. At least 2-fold enrichment of bound proteins when compared to the matrix.</p></div><div class="footnote" id="t4fn4"><sup>d</sup><p class="last"><i>K</i><sub>i</sub><sup>app</sup> values are estimated using the Cheng–Prusoff equation, apparent [<b>29</b>-immobilized] = 67 μM,<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> assuming no mass transport limitation, interactions are purely competitive and all interactions are direct.</p></div><div class="footnote" id="t4fn5"><sup>e</sup><p class="last">Not applicable (NA). <i>K</i><sub>i</sub><sup>app</sup> cannot be calculated if the interaction with the probe is indirect.</p></div></div></div><div class="NLM_p last">Analysis of these data from the quantified chemical proteomics pull-down experiment revealed very few molecular targets of the bisamides, consistent with the selectivity observed with our biochemical screening. After excluding highly promiscuous proteins commonly observed in pull-down experiments,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> three putative targets were identified. PDE6D displayed only weak apparent affinity for the bead-bound probe and little selectivity between active (<b>26</b> and <b>17</b>) and inactive (<b>28</b>) probes and so was not considered further. GSK3β displayed apparent high affinity for both active analogues, with no apparent affinity for the inactive analogue. However, GSK3β had previously been assayed in our biochemical kinase screening and both binding and functional assays (see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>) had demonstrated that the bisamide series have no measurable affinity for this kinase; we therefore concluded that GSK3β was potentially an indirect target of the bisamide series. The final putative protein target was pirin. The binding curve for pirin to the bead-bound bisamide probe was quite shallow; however, the displacement curves for pirin, which were very similar to the displacement curves for GSK3β with both cell active probe ligands, gave robust data with an apparent <i>K</i><sub>i</sub> = 28 nM for <b>26</b> and no affinity was observed with the inactive control. Using this method on a SK-OV-3 cell lysate, pirin emerged as the only remaining putative target of the bisamide series.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></div></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Pirin Validation</h3><div class="NLM_p">Very little is known about pirin, and few papers have been published discussing its possible function. Pirin was first identified in 1997 by Wendler et al. from a yeast two-hybrid screen to discover interactors of the transcription factor NFI/CTF1. Pirin was described as an iron-binding, metal-dependent protein, predominately localized within the nucleus, highly conserved across species, and ubiquitously expressed in all tissue types.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> Following this initial discovery, Scheidereit et al. reported that human pirin interacts with the proto-oncoprotein, BCL3, linking pirin with the NFκB pathway.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> The role of pirin in the NFκB pathway was further demonstrated by Lui et al.; using SPR, they revealed the metal-dependent formation of a pirin/p65/DNA complex and hypothesized the role of human pirin to be a redox-sensing transcription factor regulator.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> The redox activity of pirin has previously been discussed, as the expression level of pirin was proposed to be under the control of the transcription factor NRF2 through changes in cellular oxidative stress.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Wang et al. previously reported that human pirin interacts with the tumor suppressor protein EAF2/U19 and that exogenous overexpression of pirin increased colony formation in LNCaP human prostate cancer cells.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> In the human melanoma cell line WM266.4, Alcalay et al. demonstrated that shRNA knockdown of pirin could induce a senescence phenotype and suppress colony formation.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Also in the WM266.4 cell-line, Osada et al. demonstrated that siRNA knockdown of pirin could suppress cell migration.<a onclick="showRef(event, 'ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref61 ref62">(61, 62)</a> These results are consistent with the observation by Simizu et al. that siRNA knockdown of pirin suppressed migration of human adenocarcinoma HeLa cells and that pirin was important for epithelial–mesenchymal transition (EMT).<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> However, little evidence has emerged of an antiproliferative phenotype from genetic inhibition of pirin expression and our in-house data using siRNA in SK-OV-3 cells was consistent with this finding.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> Nonetheless, because our original screening paradigm was designed to discover inhibitors of transcription, the proposed role of pirin as a redox-sensitive transcription factor regulator was certainly intriguing.</div><div class="NLM_p">To confirm pirin as a high affinity molecular target of the bisamide series, we needed to assess the affinity of the compounds and establish SAR in an orthogonal assay format. Pirin has no known catalytic function and no known endogenous ligand in mammalian cells, consequently we decided to focus on surface plasmon resonance (SPR) to measure the affinity of the bisamide series for pirin. Purified recombinant human pirin was attached to the SPR chip through a standard amide coupling. The affinity of the bisamide ligand was assessed by equilibrium analysis of the resulting sensorgram (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>).</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Representative SPR sensorgram and binding isotherm of bisamide <b>26</b> bound to recombinant pirin. The dotted-line represents the time-point the equilibrium response was measured. The binding isotherm was fitted to a one-site specific binding model using Graphpad Prism 6.07.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The known pirin ligand TPh A<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> was used as a positive control and to protect the binding site during amine coupling to the SPR chip but displayed only modest affinity in our hands (p<i>K</i><sub>D</sub> = 5.77 ± 0.04, <i>K</i><sub>D</sub> = 1700 nM, <i>n</i> = 4).<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> The SPR sensorgram showed that bisamide <b>26</b> is indeed a tight-binding ligand of pirin. Equilibrium analysis revealed that bisamide <b>26</b> had a p<i>K</i><sub>D</sub> = 7.36 ± 0.01 (<i>K</i><sub>D</sub> = 44 nM, <i>n</i> = 3). However, the ratio of theoretical <i>R</i><sub>max</sub> to measured <i>R</i><sub>max</sub> from the binding isotherm was 0.40, indicating apparent substoichiometric binding. This could be due to the coupling of pirin to the SPR chip impairing protein folding or because pirin is a metal-dependent protein, and the metal was leaching out into the running buffer; both effects could negatively impact the apparent binding affinity and stoichiometry.</div><div class="NLM_p">To establish the SAR surrounding pirin binding, several bisamide analogues were selected for further study. We first focused on the two other analogues used in target identification. The cell active analogue tetrahydroquinoline <b>17</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 13) was also found to display tight-binding affinity for pirin; in contrast, the negative control dioxane isomer <b>28</b> (pIC<sub>50</sub> < 5.00, IC<sub>50</sub> > 10000 nM, <i>n</i> = 2) displayed no measurable affinity. These data suggested that pirin SAR and the cellular SAR were linked. Increasing the steric bulk of the solubilizing group, as exemplified by piperidine <b>24</b> (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, entry 4), again returned a high affinity pirin ligand (p<i>K</i><sub>D</sub> = 7.52 ± 0.02, <i>K</i><sub>D</sub> = 30 nM, <i>n</i> = 3), consistent with the potent cellular activity. However, structural changes to the benzodioxane amide motif proved more complex. The original hit, bisamide <b>1</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 1), was a tight-binding pirin ligand; whereas the cell inactive analogues, methylated amide <b>13</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 9) and dimethoxy <b>7</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 3), displayed no affinity for pirin. However, the two chroman isomers <b>8</b> and <b>9</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, Entries 4 and 5) were tight-binding ligands for pirin but their target affinity did not reflect their decrease in cellular activity when compared to bisamide <b>1</b>. Finally, the isochroman analogue <b>10</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entry 6) was still a high affinity ligand for pirin, despite displaying no cellular activity (see Supporting Information, <a class="ref internalNav" href="#notes-1" aria-label="Table S11">Table S11</a> for details).</div><div class="NLM_p">Pirin is a member of the cupin super family of proteins, which are so-called because they all possess a conserved β-barrel.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> However, due to minimal sequence homology between members of the cupin family, crystallography is often needed to identify them. The cupins display a huge array of functionality, including oxidases and isomerases.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> To better understand pirin SAR, we determined the crystal structure of efficacious bisamide <b>26</b> bound to pirin (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>).</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0006.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. X-ray crystal structure of bisamide <b>26</b>. PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JCT">5JCT</a>, Pymol image of pirin (light-blue cartoon representation) in complex with bisamide <b>26</b> (cyan stick representation), 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map contoured at 1.0 σ (blue mesh) and Pymol image of pirin (magenta, blue stick representation) in complex with bisamide <b>26</b> (cyan stick representation), distances shown are in Å, 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> contoured at 1.0 σ (blue mesh). Crystals belonged to the space group <i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub> and diffracted to a resolution of 1.73 Å.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Pirin is a metal-binding bicupin characterized by the two β-barrels that are clearly visible in the crystal structure.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> Pirin binds a single metal-ion in one of the β-barrels, which is where a deep, small molecule binding pocket is located. The metal-ion is clearly visible in the electron density, although it is unclear which metal-ion is actually bound. The metal-ion binding site is formed of three histidine residues (His56, His58, and His101), a glutamic acid (Glu103) and two water molecules, showing the metal-ion possesses octahedral coordination. Bisamide <b>26</b> forms no direct interactions with the metal-ion, instead a water-mediated interaction is created by the two amide functional groups, which form a hydrogen bonding pincer around this metal-coordinated water molecule, revealing why the two amides are crucial for binding. The methyl-distal amide also acts as a hydrogen-bond donor with an aspartic acid (Asp43) at the base of the binding site, explaining why methylation of that group is not tolerated. The methylpyridine ring of the quinoline substituent is clearly solvent exposed and the solubilizing group cannot be resolved due to flexibility, consistent with the broad SAR at this position and the eventual success of the pull-down probe. The SAR around the benzodioxane ring is less clear. From the cellular SAR, the oxygen atoms of the dioxane are crucial for activity. The narrow binding site is consistent with the cellularly inactive dioxane isomer <b>28</b> and the cellularly inactive dimethoxy analogue <b>7</b> having low affinity for pirin (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> and Supporting Information, <a class="ref internalNav" href="#notes-1" aria-label="Table S11">Table S11</a>), as both these groups would cause a steric clash. However, there are no clear interactions between the protein and the dioxane ring oxygens, so it is unsurprising that changing the positions of the oxygen atoms has little effect on pirin affinity, such that chromans <b>8</b>, <b>9</b>, and isochroman <b>10</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, entries 4–6) still display high affinity for pirin, in contrast to their complex cellular SAR.</div><div class="NLM_p">The apparent disconnect between the pirin affinity SAR and the antiproliferative cellular SAR led us to hypothesize that either simply binding to pirin was not enough to recapitulate the cellular phenotype or that binding to a second molecular target was also crucial. For example, in the study by Lui et al.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> the formation of protein-protein interactions with pirin was under the allosteric control of the redox-state of the iron bound in the protein. Ligands bound in the metal-binding site are therefore likely to act in an allosteric manner to regulate the function of pirin. It is commonly known that allosteric modulators display complex SAR, where small structural changes can have little effect on protein affinity but can drastically alter the observed phenotype, switching ligands between positive, negative, and neutral allosteric modulation.<a onclick="showRef(event, 'ref69 ref70'); return false;" href="javascript:void(0);" class="ref ref69 ref70">(69, 70)</a> The alternative hypothesis consistent with the observed complex cellular SAR was that a second high affinity protein target was required to explain the antiproliferative activity of the bisamide <b>26</b> and was not detected in our target-ID campaign due to either low abundance or poor stability in the lysate. The dual targeting of proteins is often observed; for example, CDK4 and CDK6, where simultaneous inhibition of both kinases is necessary to observe an antiproliferative phenotype<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> and owing to the similarity of their respective binding sites, small-molecule inhibitors inevitably inhibit both proteins.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> RNAi knockdown of pirin by several groups has demonstrated very limited effects on cell proliferation, and as pirin has no known enzymatic function, discovering and validating a cellular biomarker to investigate its role in the antiproliferative phenotype and in vivo efficacy is an ongoing challenge.</div><div class="NLM_p">One phenotype that has previously been associated with a small-molecule binding to pirin is cancer cell migration.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Miyazaki et al. demonstrated that their pirin ligand, TPh A, inhibited the migration of the melanoma cell line, WM266.4.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> To investigate whether our tool compound, bisamide <b>26</b>, phenocopied this distinct pirin-ligand chemotype, we decided to investigate its antimigratory activity using a scratch-wound assay. The scratch-wound assay is commonly used to assess the effects of small-molecules on cell migration.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> Bisamide <b>26</b> demonstrated excellent antiproliferative activity against the WM266.4 cell line (pGI<sub>50</sub> = 7.92 ± 0.10, GI<sub>50</sub> = 12 nM, <i>n</i> = 11). To exclude the effect of inhibiting proliferation on migration in the scratch-wound assay, the cells were plated at high confluency and the timeframe of the assay was reduced to 30 h. The relative wound-density was then measured at various concentrations and time-points. The negative control isomer <b>28</b> displayed no measurable antimigratory activity in stark contrast to the potent pirin ligand bisamide <b>26</b>, which was able to strongly inhibit the migration of WM266.4 cells at 100 nM (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>).</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0007.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Antimigratory activity of chemical probe bisamide <b>26</b>. (A) Wound healing images of WM266.4 cells; after 30 h, the wound has almost completely healed in the control in contrast to 100 nM <b>26</b>. (B) Quantification of the relative wound density reveals the maximum antimigratory activity is achieved at 100 nM <b>26</b>. Each point represents the arithmetic mean ±SEM of the study carried out in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35547" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35547" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Bisamide <b>26</b> displays excellent and wide-ranging in vitro cellular potency, inhibiting both HSF1-mediated transcriptional activity (defined by the inhibition of the induced expression of the heat-shock proteins HSP72 and HSP27 and <i>HSPA1A</i> mRNA) and cancer cell proliferation. Also, <b>26</b> has good mouse PK and demonstrated its anticancer effects in vivo at low free exposures with clear tumor growth inhibition at tolerable doses. Kinase screening and broader proteome screening, in addition to the chemical proteomics using SILAC quantified pull-down with rigorous controls and follow-up testing, indicated that pirin was the sole specific protein target that could be identified to date. SPR analysis of the binding of the bisamide series to pirin confirms that these compounds are high affinity ligands, with most aspects of the pirin SAR being consistent with the cellular SAR. However, there was an apparent disconnect surrounding the dioxane group of the chemotype. This requires further investigation to determine whether pirin is crucial for the in vitro antiproliferative activity and in vivo efficacy via an allosteric modulation mechanism or whether binding to a second high affinity protein target is needed to fully account for the observed phenotype. Because the SILAC quantified pull-down assay can only identify protein targets that are stable to cell lysis, additional in-cell target-ID methods are currently under investigation to probe for a potential second target. Nevertheless, the tool compound, bisamide <b>26</b>, demonstrated the previously proposed anticancer phenotype of pirin ligands by inhibiting the migration of WM266.4 melanoma cells in vitro. The role of pirin in the bisamide phenotype and the cellular effects of modulating the HSF1 pathway with bisamide <b>26</b> are currently also under investigation and will be reported subsequently.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30836" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30836" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Using a high-throughput screen to identify inhibitors of the HSF1-mediated stress pathway, we have discovered an extremely potent inhibitor of human cancer cell proliferation in vitro from the bisamide chemotype. By exploring the SAR from the cellular assays, we designed a chemical probe, bisamide <b>26</b>, which is highly potent and displays an appropriate mouse pharmacokinetic profile to significantly inhibit growth in a human ovarian carcinoma xenograft model. The chemical probe <b>26</b> was also used to design a chemical proteomic pull-down experiment, which identified the putative transcription factor regulator, pirin, as a protein target. The high affinity of chemical probe <b>26</b> for pirin was confirmed by SPR. Comparison of the biophysical with the cellular data indicated that active molecules bind pirin but that the cellular SAR is more complex, although <b>26</b> did display a potent inhibitory effect on the migration of human melanoma cells, consistent with the putative pirin cancer phenotype. Despite this, we propose that bisamide <b>26</b>, in combination with the physicochemically matched negative control dioxane isomer <b>28</b>, are promising chemical probes to investigate the role of HSF1 pathway inhibition and pirin binding in vitro and in vivo.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10287" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10287" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Experimental Procedures (Chemistry)</h3><div class="NLM_p last">All final compounds were screened through our in-house computational PAINS filter and gave no structural alerts as potential assay interference compounds.<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> Unless otherwise stated, reactions were conducted in oven-dried glassware under an atmosphere of nitrogen or argon using anhydrous solvents. All commercially obtained reagents and solvents were used as received. Thin layer chromatography (TLC) was performed on precoated aluminum sheets of silica (60 F254 nm, Merck) and visualized using short-wave UV light. Flash column chromatography was carried out on Merck silica gel 60 (partial size 40–65 μm). Column chromatography was also performed on a Biotage SP1 purification system using Biotage Flash silica cartridges (SNAP KP-Sil). Ion exchange chromatography was performed using acidic Isolute Flash SCX-II columns. Semipreparative HPLC was performed on an Agilent 6120 system, flow 20 mL/min, eluents 0.1% acetic acid in water and 0.1% acetic acid in methanol, gradient of 10–100% organic phase. <sup>1</sup>H NMR spectra were recorded on Bruker AMX500 (500 MHz) spectrometers using an internal deuterium lock. Chemical shifts are quoted in parts per million (ppm) using the following internal references: CDCl<sub>3</sub> (δH 7.26), MeOD (δH 3.31), and DMSO-<i>d</i><sub>6</sub> (δH 2.50). Signal multiplicities are recorded as singlet (s), doublet (d), triplet (t), quartet (q), multiplet (m), doublet of doublets (dd), doublet of doublet of doublets (ddd), broad (br), or obscured (obs). Coupling constants, <i>J</i>, were measured to the nearest 0.1 Hz. <sup>13</sup>C NMR spectra were recorded on Bruker AMX500 spectrometers at 126 MHz using an internal deuterium lock. Chemical shifts are quoted to 0.01 ppm, unless greater accuracy was required, using the following internal references: CDCl<sub>3</sub> (δC 77.0), MeOD (δC 49.0), and DMSO-<i>d</i><sub>6</sub> (δC 39.5). High resolution mass spectra were recorded on an Agilent 1200 series HPLC and diode array detector coupled to a 6210 time-of-flight mass spectrometer with dual multimode APCI/ESI source or on a Waters Acquity UPLC and diode array detector coupled to a Waters G2 QToF mass spectrometer fitted with a multimode ESI/APCI source. Analytical separation was carried out according to the methods listed below. The mobile phase was a mixture of methanol (solvent A) and water (solvent B), both containing formic acid at 0.1%, UV detection was at 254 nm. Method I: Agilent 1200 series HPLC, Merck Purospher STAR (RP-18e, 30 mm × 4 mm) column using a flow rate of 1.5 mL/min in a 4 min gradient elution. Gradient elution was as follows: 10:90 (A/B) to 90:10 (A/B) over 2.5 min, 90:10 (A/B) for 1 min, and then reversion back to 10:90 (A/B) over 0.3 min, finally 10:90 (A/B) for 0.2 min. Method II: Agilent 1200 series HPLC, Merck Chromolith flash column (RP-18e, 25 mm × 2 mm) at 30 °C using a flow rate of 0.75 mL/min in a 4 min gradient elution. Gradient elution was as follows: 5:95 (A/B) to 100:0 (A/B) over 2.5 min, 100:0 (A/B) for 1 min, and then reversion back to 5:95 (A/B) over 0.1 min, finally 5:95 (A/B) for 0.4 min. Method III: Waters Acquity UPLC, Phenomenex Kinetex XB-C18 column (30 mm × 2.1 mm, 1.7 μ, 100 A) at 30 °C using flow rate of 0.3 mL/min in a 4 min gradient elution. Gradient elution was as follows: 10:90 (A/B) to 90:10 (A/B) over 3 min, 90:10 (A/B) for 0.5 min, and then reversion back to 10:90 (A/B) over 0.3 min, finally 10:90 (A/B) for 0.2 min; Method IV: Waters Acquity UPLC, Phenomenex Kinetex C18 column (30 mm × 2.1 mm, 2.6 μ, 100A), flow rate and gradient elution according to Method III. The following reference masses were used for HRMS analysis: Agilent 1200 series, caffeine [M + H]<sup>+</sup> 195.087652, hexakis(1<i>H</i>,1<i>H</i>,3<i>H</i>-tetrafluoropentoxy)phosphazene [M + H]<sup>+</sup> 922.009798, and hexakis(2,2-difluoroethoxy)phosphazene [M + H]<sup>+</sup> 622.02896 or reserpine [M + H]<sup>+</sup> 609.280657. Waters Acquity UPLC: leucine enkephalin fragment ion [M + H]<sup>+</sup> 397.1876. All compounds were >95% purity by LCMS analysis unless otherwise stated.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> General Synthetic Procedures</h3><div id="sec5_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Method A</h4><div class="NLM_p last">Oxalyl chloride (1.2–1.7 equiv) was added dropwise to a solution of the carboxylic acid (1.0–1.5 equiv) and DMF in anhydrous dichloromethane. The reaction mixture was stirred at room temperature for 3–4 h and then concentrated. The remaining residue was redissolved in dichloromethane and concentrated again. Then, the aniline (1.0 equiv) was added to the remaining residue and the combined solids were dissolved in anhydrous pyridine or a mixture of anhydrous pyridine and dichloromethane. The resulting reaction mixture was stirred at room temperature overnight.</div></div><div id="sec5_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Method B</h4><div class="NLM_p last">1-[Bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate (HATU, 1.2–1.3 equiv) was added to a solution of the carboxylic acid (1.0–1.2 equiv) and <i>N</i>,<i>N</i>-diisopropylethylamine (2.0–5.0 equiv) in anhydrous DMF. The reaction mixture was stirred for 10 min before the aniline (1.0 equiv) was added. The reaction mixture was stirred at room temperature overnight. The reaction mixture was then diluted with water. The resulting precipitate was isolated by filtration and washed with water.</div></div><div id="sec5_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Method C</h4><div class="NLM_p last">The (3-nitrophenyl)amide (1.0 equiv), iron powder (9–11 equiv), and ammonium chloride (9–11 equiv) in ethanol/water (4/1) were added to a round-bottom flask and heated to reflux overnight. Then the reaction mixture was filtered over a short pad of Celite or silica gel. The filtrate was reduced in vacuo until dryness. The remaining residue was resuspended in dichloromethane and washed with satd NaHCO<sub>3</sub> (aq), water, and brine. The organic layer was dried over sodium sulfate or magnesium sulfate and reduced in vacuo until dryness to afford the corresponding (3-aminophenyl)amide.</div></div><div id="sec5_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Method D</h4><div class="NLM_p last">Palladium (10% on activated carbon, 42–68 mg/1 mmol) was added to a suspension of the nitro compound in a 1:1 mixture of ethanol/ethyl acetate and stirred under H<sub>2</sub> (1 atm.) at 28 °C overnight. The reaction mixture was then filtered through Celite and the Celite pad washed with further ethyl acetate. The filtrate was concentrated in vacuo to afford the corresponding aniline.</div></div><div id="sec5_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Method E</h4><div class="NLM_p last">NaH (60% in mineral oil, 1.1–2.2 equiv) was added to a solution of the alcohol (1.2–2.8 equiv) in anhydrous THF at 0 °C. The reaction was allowed to stir at 0 °C for 10 min, then at room temperature for 30 min. 6-Bromo-2-chloroquinoline was added and the resulting suspension heated to reflux for 1–16 h. The reaction was allowed to cool to room temperature, then diluted with water/satd NaHCO<sub>3</sub> (aq and extracted with dichloromethane (3×)). The organic layers were combined, washed with water, dried over magnesium sulfate, and concentrated in vacuo to afford the crude product.</div></div><div id="sec5_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Method F</h4><div class="NLM_p last"><i>n</i>-BuLi (1.2–2.2 equiv, 2.5 M in hexanes, freshly titrated before use using standard procedures) was added dropwise to a solution of the aryl bromide in anhydrous THF at −78 °C under nitrogen. The reaction was stirred at −78 °C for 40 min before solid CO<sub>2</sub> was added. After stirring for 5 min, the reaction was allowed to warm to room temperature. The reaction was quenched with water and concentrated under reduced pressure to remove the THF. The aqueous layer was washed with ethyl acetate, acidified to pH 3 by the addition of 2 M HCl (aq), and concentrated in vacuo to afford the crude product.</div></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Preparation of Compounds in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a></h3><div id="sec5_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 2-Methyl-<i>N</i>-(2-methyl-5-nitrophenyl)quinoline-6-carboxamide <b>4</b></h4><div class="NLM_p last">2-Methylquinoline-6-carboxylic acid <b>3</b> (3.69 g, 19.72 mmol), oxalyl chloride (1.8 mL, 20.15 mmol), and DMF (310 μL, 4.00 mmol) in anhydrous dichloromethane (35 mL) were reacted according to method A. 2-Methyl-5-nitroaniline <b>2</b> (3.0 g, 19.72 mmol), and the acid chloride were then reacted in anhydrous pyridine (35 mL). The reaction mixture was reduced in vacuo until dryness. The remaining residue was triturated with diethyl ether. The crude product was purified by column chromatography (Biotage, gradient of 0–50% ethanol in dichloromethane) to afford the product as a yellow amorphous solid (5.57 g, 88%). IR (thin film): ν<sub>max</sub> 1661, 1527, 1342, 1260 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.36 (s, 1H), 8.63 (d, <i>J</i> = 1.8 Hz, 1H), 8.43 (d, <i>J</i> = 8.4 Hz, 1H), 8.40 (d, <i>J</i> = 2.4 Hz, 1H), 8.25 (dd, <i>J</i> = 8.7, 2.0 Hz, 1H), 8.07 (d, <i>J</i> = 2.9 Hz, 1H), 8.05 (d, <i>J</i> = 3.0 Hz, 1H), 7.60 (d, <i>J</i> = 8.5 Hz, 1H), 7.54 (d, <i>J</i> = 8.4 Hz, 1H), 2.71 (s, 3H), 2.44 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.54, 161.05, 146.20, 144.90, 143.38, 142.15, 137.76, 132.04, 130.08, 129.93, 129.31, 126.62, 126.09, 124.16, 121.00, 120.94, 23.98, 18.82. HRMS (ESI<sup>+</sup>): calcd for C<sub>18</sub>H<sub>15</sub>NaN<sub>3</sub>O<sub>3</sub> (M + Na)<sup>+</sup>, 344.1006; found, 344.0999.</div></div><div id="sec5_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <i>N</i>-(5-Amino-2-methylphenyl)-2-methylquinoline-6-carboxamide <b>5</b></h4><div class="NLM_p last">2-Methyl-<i>N</i>-(2-methyl-5-nitrophenyl)quinoline-6-carboxamide <b>4</b> (4.0 g, 12.45 mmol), iron powder (6.95 g, 124.0 mmol), and ammonium chloride (2.12 g, 124.0 mmol) in ethanol (50 mL) and water (12.5 mL) were reacted according to method C to afford the product as a light-yellow amorphous solid (1.95 g, 54%). IR (thin film): ν<sub>max</sub> 3340, 3252, 1645, 1582, 1487, 1278 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.86 (s, 1H), 8.61–8.51 (m, 1H), 8.38 (d, <i>J</i> = 8.4 Hz, 1H), 8.21 (dd, <i>J</i> = 8.8, 1.7 Hz, 1H), 8.00 (d, <i>J</i> = 8.8 Hz, 1H), 7.51 (d, <i>J</i> = 8.4 Hz, 1H), 6.91 (d, <i>J</i> = 8.1 Hz, 1H), 6.65 (d, <i>J</i> = 2.0 Hz, 1H), 6.42 (dd, <i>J</i> = 8.1, 2.2 Hz, 1H), 4.94 (s, 2H), 2.70 (s, 3H), 2.09 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.18, 161.09, 148.86, 147.31, 137.58, 137.01, 132.25, 130.87, 128.72, 128.49, 128.38, 125.84, 123.38, 120.72, 112.75, 112.61, 25.49, 17.50. HRMS (ESI<sup>+</sup>): calcd for C<sub>18</sub>H<sub>18</sub>N<sub>3</sub>O (M + H)<sup>+</sup>, 292.1444; found, 292.1446.</div></div><div id="sec5_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-methylphenyl)-2-methylquinoline-6-carboxamide <b>1</b></h4><div class="NLM_p last">2-Methyl-6-quinolinecarboxylic acid (600 mg, 3.21 mmol), HATU (1.46 g, 3.85 mmol), and <i>N</i>-(5-amino-2-methylphenyl)-2-methylquinoline-6-carboxamide <b>5</b> (910 mg, 3.21 mmol) were reacted in <i>N</i>,<i>N</i>-diisopropylethylamine (1.12 mL, 6.41 mmol) and anhydrous DMF (25 mL) according to method B to afford the product as an off-white amorphous solid (1.38 g, 95%). IR (solid): ν<sub>max</sub> 2922, 1645, 1582, 1496, 1284, 1258, 1063 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.14 (s, 1H), 10.08 (s, 1H), 8.61 (s, 1H), 8.41 (br d, <i>J</i> = 7.3 Hz, 1H), 8.24 (d, <i>J</i> = 8.7 Hz, 1H), 8.03 (d, <i>J</i> = 8.8 Hz, 1H), 7.87 (d, <i>J</i> = 2.1 Hz, 1H), 7.64–7.56 (m, 2H), 7.54 (d, <i>J</i> = 2.1 Hz, 1H), 7.51 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 7.24 (d, <i>J</i> = 8.5 Hz, 1H), 6.98 (d, <i>J</i> = 8.4 Hz, 1H), 4.35–4.26 (m, 4H), 2.71 (s, 3H), 2.24 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.44, 164.82, 161.21, 148.93, 146.80, 143.39, 137.77, 137.62, 136.74, 131.97, 130.59, 129.23, 128.81, 128.64, 128.38, 128.14, 125.86, 123.44, 121.66, 119.07, 118.63, 117.30, 117.12, 64.86, 64.48, 25.51, 17.94. HRMS (ESI<sup>+</sup>): calcd for C<sub>27</sub>H<sub>24</sub>N<sub>3</sub>O<sub>4</sub> (M + H)<sup>+</sup>, 454.1761; found, 454.1733.</div></div><div id="sec5_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-5-carboxamido)-2-methylphenyl)-2-methylquinoline-6-carboxamide <b>6</b></h4><div class="NLM_p last">2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-5-carboxylic acid (46 mg, 0.26 mmol), HATU (127 mg, 0.34 mmol), and <i>N</i>-(5-amino-2-methylphenyl)-2-methylquinoline-6-carboxamide <b>5</b> (75 mg, 0.26 mmol) in <i>N</i>,<i>N</i>-diisopropylethylamine (0.14 mL, 0.82 mmol) and anhydrous DMF (2.5 mL) were reacted according to method B to afford the product as a white amorphous solid (95 mg, 81%). IR (solid): ν<sub>max</sub> 3370, 3256, 1644, 1598, 1532, 1450 1091, 743 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.16 (s, 1H), 10.09 (s, 1H), 8.61 (s, 1H), 8.41 (d, <i>J</i> = 8.4 Hz, 1H), 8.27–8.21 (m, 2H), 8.03 (d, <i>J</i> = 8.7 Hz, 1H), 7.82 (br s, 1H), 7.58–7.52 (m, 2H), 7.25 (d, <i>J</i> = 8.3 Hz, 1H), 7.18–7.11 (m, 2H), 7.02–6.98 (m, 1H), 6.92 (t, <i>J</i> = 7.8 Hz, 1H), 4.40–4.27 (m, 4H), 2.71 (s, 3H), 2.24 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.43, 164.26, 161.23, 148.94, 144.11, 141.65, 137.62, 137.52, 136.89, 131.90, 130.74, 129.44, 128.81, 128.67, 128.38, 126.14, 125.85, 123.45, 121.73, 121.17, 119.51, 118.44, 118.00, 64.93, 64.21, 25.51, 17.97. HRMS (ESI<sup>+</sup>): calcd for C<sub>27</sub>H<sub>24</sub>N<sub>3</sub>O<sub>4</sub> (M + H)<sup>+</sup>, 454.1761; found, 454.1760.</div></div><div id="sec5_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <i>N</i>-(5-(3,4-Dimethoxybenzamido)-2-methylphenyl)-2-methylquinoline-6-carboxamide <b>7</b></h4><div class="NLM_p last">3,4-Dimethoxybenzoic acid (34 mg, 0.189 mmol), HATU (82 mg, 0.215 mmol), and <i>N</i>-(5-amino-2-methylphenyl)-2-methylquinoline-6-carboxamide <b>5</b> (50 mg, 0.172 mmol) in <i>N</i>,<i>N</i>-diisopropylethylamine (0.075 mL, 0.429 mmol) and anhydrous DMF (1.2 mL) were reacted according to method B. The crude product was purified by flash column chromatography (1–5% methanol in dichloromethane) to afford compound <b>7</b> as a white amorphous solid (52 mg, 67%). IR (thin film): ν<sub>max</sub> 1648, 1601, 1508, 1418, 1268, 1225 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.13 (s, 1H), 10.09 (s, 1H), 8.61 (d, <i>J</i> = 2.0 Hz, 1H), 8.41 (d, <i>J</i> = 8.4 Hz, 1H), 8.24 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 8.03 (d, <i>J</i> = 8.8 Hz, 1H), 7.85 (d, <i>J</i> = 2.2 Hz, 1H), 7.62 (td, <i>J</i> = 8.8, 2.2 Hz, 2H), 7.55 (d, <i>J</i> = 2.1 Hz, 1H), 7.53 (d, <i>J</i> = 8.4 Hz, 1H), 7.26 (d, <i>J</i> = 8.5 Hz, 1H), 7.08 (d, <i>J</i> = 8.5 Hz, 1H), 3.85 (s, 3H), 3.84 (s, 3H), 2.71 (s, 3H), 2.26 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.01, 164.74, 160.76, 151.58, 148.48, 148.26, 137.32, 137.16, 136.27, 131.51, 130.15, 128.75, 128.37, 128.16, 127.91, 126.92, 125.40, 122.98, 121.03, 118.76, 118.32, 110.98, 110.91, 55.67, 55.62, 25.04, 17.46. HRMS (ESI<sup>+</sup>): calcd for C<sub>27</sub>H<sub>26</sub>N<sub>3</sub>O (M + H)<sup>+</sup>, 456.19178; found, 456.19189.</div></div><div id="sec5_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> <i>N</i>-(5-(Chroman-7-carboxamido)-2-methylphenyl)-2-methylquinoline-6-carboxamide <b>8</b></h4><div class="NLM_p last">2-Methyl-6-quinolinecarboxylic acid (75 mg, 0.42 mmol), HATU (182 mg, 0.48 mmol), and <i>N</i>-(5-amino-2-methylphenyl)-2-methylquinoline-6-carboxamide <b>5</b> (111 mg, 0.38 mmol) in <i>N</i>,<i>N</i>-diisopropylethylamine (0.20 mL, 1.15 mmol) and anhydrous DMF (3.5 mL) were reacted according to method B. The crude product was purified by column chromatography (Biotage, gradient of 0–10% methanol in dichloromethane) to afford an off-white amorphous solid. This material was triturated with diethyl ether to afford the product as a white amorphous solid (51 mg, 30%). IR (thin film): ν<sub>max</sub> 2957, 1651, 1599, 1527, 1499, 1310, 1283 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO<i>-d</i><sub>6</sub>) δ 10.14 (s, 1H), 10.13 (s, 1H), 8.61 (d, <i>J</i> = 1.8 Hz, 1H), 8.41 (d, <i>J</i> = 8.4 Hz, 1H), 8.24 (dd, <i>J</i> = 8.8, 1.9 Hz, 1H), 8.03 (d, <i>J</i> = 8.8 Hz, 1H), 7.89 (d, <i>J</i> = 2.0 Hz, 1H), 7.60 (dd, <i>J</i> = 8.3, 2.1 Hz, 1H), 7.53 (d, <i>J</i> = 8.4 Hz, 1H), 7.43 (dd, <i>J</i> = 7.9, 1.7 Hz, 1H), 7.36 (d, <i>J</i> = 1.6 Hz, 1H), 7.25 (d, <i>J</i> = 8.4 Hz, 1H), 7.19 (d, <i>J</i> = 7.9 Hz, 1H), 4.23–4.14 (m, 2H), 2.80 (t, <i>J</i> = 6.4 Hz, 2H), 2.71 (s, 3H), 2.25 (s, 3H), 1.94 (p, <i>J</i> = 6.3 Hz, 2H). <sup>13</sup>C NMR (126 MHz, DMSO<i>-d</i><sub>6</sub>) δ 165.44, 165.31, 161.21, 154.79, 148.93, 137.70, 137.63, 136.73, 134.25, 131.96, 130.61, 130.29, 129.32, 128.81, 128.64, 128.38, 126.70, 125.86, 123.44, 119.61, 119.08, 118.64, 115.94, 66.57, 25.51, 24.73, 22.00, 17.95. HRMS (ESI<sup>+</sup>): calcd for C<sub>28</sub>H<sub>26</sub>N<sub>3</sub>O<sub>3</sub> (M + H)<sup>+</sup>, 452.1969; found, 452.1956.</div></div><div id="sec5_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> <i>N</i>-(5-(Chroman-6-carboxamido)-2-methylphenyl)-2-methylquinoline-6-carboxamide <b>9</b></h4><div class="NLM_p last">Chroman-6-carboxylic acid (40 mg, 0.227 mmol), HATU (98 mg, 0.257 mmol), and <i>N</i>-(5-amino-2-methylphenyl)-2-methylquinoline-6-carboxamide <b>5</b> (60 mg, 0.206 mmol) in <i>N</i>,<i>N</i>-diisopropylethylamine (0.088 mL, 0.505 mmol) and anhydrous DMF (1.5 mL) were reacted according to method B. The crude product was purified by flash column chromatography (1–3.5% methanol in dichloromethane) to afford the product as a white amorphous solid (62 mg, 62%). IR (thin film): ν<sub>max</sub> 1647, 1600, 1528, 1495, 1318, 1256 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.13 (s, 1H), 10.04 (s, 1H), 8.61 (d, <i>J</i> = 2.0 Hz, 1H), 8.41 (d, <i>J</i> = 8.4 Hz, 1H), 8.24 (dd, <i>J</i> = 8.7, 2.0 Hz, 1H), 8.03 (d, <i>J</i> = 8.7 Hz, 1H), 7.87 (d, <i>J</i> = 2.2 Hz, 1H), 7.75 (d, <i>J</i> = 2.3 Hz, 1H), 7.72 (dd, <i>J</i> = 8.5, 2.3 Hz, 1H), 7.59 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 7.53 (d, <i>J</i> = 8.4 Hz, 1H), 7.24 (d, <i>J</i> = 8.6 Hz, 1H), 6.84 (d, <i>J</i> = 8.5 Hz, 1H), 4.23–4.17 (m, 2H), 2.82 (t, <i>J</i> = 6.4 Hz, 2H), 2.71 (s, 3H), 2.24 (s, 3H), 2.00–1.91 (m, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.98, 164.86, 160.75, 157.36, 148.47, 137.46, 137.16, 136.25, 131.51, 130.12, 129.78, 128.63, 128.36, 128.17, 127.91, 127.00, 126.26, 125.40, 122.98, 122.03, 118.54, 118.11, 116.07, 66.36, 25.05, 24.26, 21.54, 17.47. HRMS (ESI<sup>+</sup>): calcd for C<sub>28</sub>H<sub>26</sub>N<sub>3</sub>O (M + H)<sup>+</sup>, 452.19687; found, 452.19758.</div></div><div id="sec5_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> <i>N</i>-(5-(Isochroman-7-carboxamido)-2-methylphenyl)-2-methylquinoline-6-carboxamide <b>10</b></h4><div class="NLM_p">7-Bromoisochroman (152 mg, 0.71 mmol), Herrmann’s palladacycle (33.4 mg, 0.04 mmol), and tri-<i>tert</i>-butylphosphonium tetrafluoroborate (41 mg, 0.14 mmol) were combined with 2-(trimethylsilyl)ethanol (7.0 mL) in a microwave vial. Molybdenum hexacarbonyl (377 mg, 1.427 mmol) was then added, followed by DBU (1.0 M in THF, 2.57 mL, 2.57 mmol), and the vial promptly sealed. The reaction mixture was heated to 130 °C for 1 h in a microwave. After this time, further 2-(trimethylsilyl)ethanol (3.0 mL) was added and the reaction heated to 130 °C for 1 h under microwave irradiation. The reaction mixture was concentrated under high vacuum. The resulting residue was purified by column chromatography (Biotage, gradient of 0–100% ethyl acetate in cyclohexane) to afford 2-(trimethylsilyl)ethyl isochroman-7-carboxylate compound as a yellow oil (108 mg, 54%). This oil was only 80% pure but was used directly in the next step.<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a></div><div class="NLM_p">To a stirring solution of 2-(trimethylsilyl)ethyl isochroman-7-carboxylate (105 mg, 0.302 mmol) in THF (3 mL) at room temperature under argon was dropwise added TBAF 1.0 M in THF (0.45 mL, 0.45 mmol). The reaction was allowed to stir at room temperature for 2 h. The reaction mixture was then diluted with water (25 mL) and the THF removed in vacuo. The aqueous layer was washed with dichloromethane (15 mL). TLC/LCMS showed that there was product in both the aqueous and organic layers. Therefore, the organic layer was extracted with a portion of 1 M NaOH (15 mL), followed by water (15 mL). The combined aqueous layers were acidified (to ∼ pH 2) using 1 M HCl (aq) and extracted with dichloromethane (3 × 15 mL). The combined organic layer was concentrated in vacuo to afford isochroman-7-carboxylic acid as a colorless amorphous solid. This material was used in the next step without further purification.</div><div class="NLM_p last">Isochroman-7-carboxylic acid (50 mg, 0.28 mmol), HATU (130 mg, 0.34 mmol), and <i>N</i>-(5-amino-2-methylphenyl)-2-methylquinoline-6-carboxamide <b>5</b> (87 mg, 0.30 mmol) in <i>N</i>,<i>N</i>-diisopropylethylamine (0.14 mL, 0.82 mmol) and anhydrous DMF (2.5 mL) were reacted according to method B. The crude product was purified by column chromatography (Biotage, gradient of 0–5% methanol in dichloromethane) to afford a pale-yellow amorphous solid (78 mg, 63%). IR (thin film): ν<sub>max</sub> 2965, 1650, 1600, 1528, 1499, 1285 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.19 (s, 1H), 10.14 (s, 1H), 8.61 (d, <i>J</i> = 2.0 Hz, 1H), 8.41 (d, <i>J</i> = 8.4 Hz, 1H), 8.24 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 8.03 (d, <i>J</i> = 8.8 Hz, 1H), 7.88 (d, <i>J</i> = 2.1 Hz, 1H), 7.78 (dd, <i>J</i> = 8.0, 1.8 Hz, 1H), 7.67–7.65 (m, 1H), 7.60 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 7.53 (d, <i>J</i> = 8.4 Hz, 1H), 7.29 (d, <i>J</i> = 8.0 Hz, 1H), 7.26 (d, <i>J</i> = 8.6 Hz, 1H), 4.76 (s, 2H), 3.91 (t, <i>J</i> = 5.7 Hz, 2H), 2.86 (t, <i>J</i> = 5.7 Hz, 2H), 2.71 (s, 3H), 2.25 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.58, 165.46, 161.22, 148.91, 137.70, 137.66, 137.49, 136.76, 135.49, 132.87, 131.96, 130.65, 129.34, 129.26, 128.80, 128.65, 128.40, 125.96, 125.86, 124.37, 123.46, 119.05, 118.63, 67.49, 64.82, 28.24, 25.50, 17.95. HRMS (ESI<sup>+</sup>): calcd for C<sub>28</sub>H<sub>26</sub>N<sub>3</sub>O<sub>3</sub> (M + H)<sup>+</sup>, 452.1969; found, 452.1964.</div></div><div id="sec5_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> <i>N</i>-(3-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2,6-dimethylphenyl)-2-methylquinoline-6-carboxamide <b>11</b></h4><div class="NLM_p">2,6-Dimethylaniline (4.07 mL, 32.7 mmol) and sulfuric acid (30 mL) were added to a round-bottom flask. The solution was cooled on an ice bath and nitric acid (2.6 mL, 43.1 mmol, 69%) was added dropwise to the stirred solution. The resulting reaction mixture was warmed to room temperature and stirred for further 30 min at room temperature. The reaction mixture was then poured onto cold water and neutralized by addition of solid sodium hydroxide. 2,6-Dimethyl-3-nitroaniline was isolated by filtration and washed with copious amounts of water. The crude product was purified by column chromatography (Biotage, gradient of 0–40% ethyl acetate in cyclohexane) to afford the product as a brown amorphous solid (1.82 g, 33%). IR (solid): ν<sub>max</sub> 3421, 3349, 2919, 1634, 1513, 1460, 1348 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 6.99 (d, <i>J</i> = 8.1 Hz, 1H), 6.93 (d, <i>J</i> = 8.1 Hz, 1H), 5.21 (s, 2H), 2.15 (s, 3H), 2.14 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 150.03, 146.44, 127.90, 126.09, 113.71, 111.03, 18.71, 13.41. HRMS (ESI<sup>+</sup>): calcd for C<sub>8</sub>H<sub>11</sub>N<sub>2</sub>O<sub>2</sub> (M + H)<sup>+</sup>, 167.0815; found, 167.0831.</div><div class="NLM_p">2-Methylquinoline-6-carboxylic acid <b>3</b> (1.27 g, 6.77 mmol), oxalyl chloride (750 μL, 8.40 mmol), and DMF (110 μL, 1.42 mmol) in anhydrous dichloromethane (15 mL) were reacted according to method A. The resulting acid chloride and 2,6-dimethyl-3-nitroaniline (1.5 g, 3.01 mmol) were then reacted in anhydrous pyridine (20 mL). The reaction mixture was concentrated in vacuo. The remaining residue was redissolved in dichloromethane and washed with satd NaHCO<sub>3</sub> (aq), water, and brine. The organic layer was dried over sodium sulfate and reduced in vacuo until dryness to afford <i>N</i>-(2,6-dimethyl-3-nitrophenyl)-2-methylquinoline-6-carboxamide as a dark-red to brown amorphous solid (1.69 g). The crude product was used in the next synthetic step without further purification.</div><div class="NLM_p"><i>N</i>-(2,6-Dimethyl-3-nitrophenyl)-2-methylquinoline-6-carboxamide (1.50 g, 4.47 mmol), iron powder (2.50 g, 44.7 mmol), and ammonium chloride (762 mg, 44.7 mmol) in ethanol (40 mL) and water (10 mL) were reacted according to method C. The crude product was purified by column chromatography (Biotage, gradient of 0–30% ethanol in dichloromethane) to afford the product as a brown amorphous solid (1.14 g, 84%). IR (solid): ν<sub>max</sub> 3232, 2916, 1649, 1621, 1487, 1281 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.84 (s, 1H), 8.60 (d, <i>J</i> = 1.9 Hz, 1H), 8.39 (d, <i>J</i> = 8.4 Hz, 1H), 8.25 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 8.01 (d, <i>J</i> = 8.7 Hz, 1H), 7.51 (d, <i>J</i> = 8.4 Hz, 1H), 6.82 (d, <i>J</i> = 8.1 Hz, 1H), 6.56 (d, <i>J</i> = 8.1 Hz, 1H), 4.72 (s, 2H), 2.70 (s, 3H), 2.07 (s, 3H), 1.95 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.08, 161.05, 148.86, 145.57, 137.57, 135.64, 132.08, 128.77, 128.40, 128.27, 127.51, 125.89, 123.38, 123.09, 120.01, 113.46, 25.49, 18.15, 12.84. HRMS (ESI<sup>+</sup>): calcd for C<sub>19</sub>H<sub>20</sub>N<sub>3</sub>O (M + H)<sup>+</sup>, 306.1606; found, 306.1597.</div><div class="NLM_p last">2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxylic acid (415 mg, 2.30 mmol), oxalyl chloride (350 μL, 3.92 mmol), and DMF (50 μL, 646 mmol) in anhydrous dichloromethane (10 mL) were reacted according to method A. <i>N</i>-(3-Amino-2,6-dimethylphenyl)-2-methylquinoline-6-carboxamide (703 mg, 2.30 mmol) and the acid chloride were then reacted in anhydrous pyridine. The reaction mixture was diluted with diethyl ether. The product precipitated from the solution, isolated by filtration, and washed with water and diethyl ether. The crude product was purified by column chromatography (Biotage, gradient of 0–10% ethanol in dichloromethane) to afford compound <b>11</b> as a beige amorphous solid (237.4 mg, 22%). IR (thin film): ν<sub>max</sub> 1649, 1584, 1487, 1290, 1067 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.08 (s, 1H), 9.80 (s, 1H), 8.63 (s, 1H), 8.41 (d, <i>J</i> = 8.4 Hz, 1H), 8.27 (d, <i>J</i> = 10.8 Hz, 1H), 8.04 (d, <i>J</i> = 8.8 Hz, 1H), 7.55 (d, <i>J</i> = 2.0 Hz, 1H), 7.54–7.53 (m, 1H), 7.53–7.50 (m, 1H), 7.18 (m, 2H), 6.98 (d, <i>J</i> = 8.4 Hz, 1H), 4.32–4.29 (m, 4H), 2.71 (s, 3H), 2.24 (s, 3H), 2.10 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.22, 164.92, 161.17, 148.93, 146.73, 143.41, 137.60, 136.13, 135.30, 133.96, 132.95, 131.78, 128.86, 128.55, 128.27, 127.84, 127.38, 126.24, 125.90, 123.44, 121.61, 117.31, 117.12, 64.85, 64.47, 25.51, 18.59, 13.94. HRMS (ESI<sup>+</sup>): calcd for C<sub>28</sub>H<sub>26</sub>N<sub>3</sub>O<sub>4</sub> (M + H)<sup>+</sup>, 468.1923; found, 428.1932.</div></div><div id="sec5_3_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2,4-dimethylphenyl)-2-methylquinoline-6-carboxamide <b>12</b></h4><div class="NLM_p">2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxylic acid (542 mg, 3.01 mmol), oxalyl chloride (325 μL, 4.71 mmol), and DMF (50 μL, 0.646 mmol) in anhydrous dichloromethane (10 mL) were reacted according to method A. 2,4-Dimethyl-5-nitroaniline (500 mg, 3.01 mmol) and the acid chloride were then reacted in anhydrous pyridine (15 mL). The reaction mixture was concentrated in vacuo. The remaining residue was triturated with diethyl ether and dried in vacuo to give <i>N</i>-(2,4-dimethyl-5-nitrophenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide as a brown amorphous solid (1.11 g), which was used in the next synthetic step without further purification.</div><div class="NLM_p"><i>N</i>-(2,4-Dimethyl-5-nitrophenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide (988 mg, 2.08 mmol), ammonium chloride (394 mg, 23.1 mmol), and iron powder (1.29 g, 23.1 mmol) in ethanol (10 mL) and water (2.5 mL) were reacted according to method C to afford <i>N</i>-(5-amino-2,4-dimethylphenyl)-2,3-dihydrobenz[<i>b</i>][1,4]dioxine-6-carboxamide as a brown amorphous solid (353 mg, 57% over two steps). IR (solid): ν<sub>max</sub> 3352, 3265, 2918, 1634, 1613, 1489, 1288 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.43 (s, 1H), 7.49 (d, <i>J</i> = 2.0 Hz, 1H), 7.47 (dd, <i>J</i> = 8.3, 2.1 Hz, 1H), 6.95 (d, <i>J</i> = 8.3 Hz, 1H), 6.77 (s, 1H), 6.58 (s, 1H), 4.65 (s, 2H), 4.40–4.10 (m, 4H), 2.02 (s, 3H), 2.00 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.53, 146.53, 144.85, 143.36, 134.81, 131.82, 128.22, 121.45, 121.11, 119.64, 117.23, 117.00, 113.13, 64.81, 64.46, 17.41, 17.29. HRMS (ESI<sup>+</sup>): calcd for C<sub>17</sub>H<sub>19</sub>N<sub>2</sub>O<sub>3</sub> (M + H)<sup>+</sup>, 299.1396; found, 299.1555.</div><div class="NLM_p last">2-Methylquinoline-6-carboxylic acid <b>3</b> (188 mg, 1.00 mmol), oxalyl chloride (80 μL, 0.896 mmol), and DMF (20 μL, 0.258 mmol) in anhydrous dichloromethane (5 mL) were reacted according to method A. The resulting acid chloride and <i>N</i>-(5-amino-2,4-dimethylphenyl)-2,3-dihydrobenz[<i>b</i>][1,4]dioxine-6-carboxamide (201 mg, 0.674 mmol) were then reacted in anhydrous pyridine (10 mL). The reaction mixture was then diluted with dichloromethane and washed with satd NaHCO<sub>3</sub> (aq), water, and brine. The organic layer was dried over sodium sulfate and reduced in vacuo until dryness. The crude product was purified by column chromatography (Biotage, gradient of 0–5% ethanol in dichloromethane). The product was further purified by semipreparative HPLC, according to the method specified in the general procedures, to afford a beige amorphous solid (30.4 mg, 9.7%). IR (solid): ν<sub>max</sub> 1664, 1620, 1531, 1497, 1291 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.09 (s, 1H), 9.71 (s, 1H), 8.59 (s, 1H), 8.40 (d, <i>J</i> = 8.4 Hz, 1H), 8.29–8.16 (m, 1H), 8.02 (d, <i>J</i> = 8.8 Hz, 1H), 7.64–7.43 (m, 3H), 7.35 (s, 1H), 7.17 (s, 1H), 6.98 (d, <i>J</i> = 8.3 Hz, 1H), 4.39–4.15 (m, 4H), 2.70 (s, 3H), 2.25 (s, 3H), 2.20 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.48, 164.83, 161.18, 148.91, 146.74, 143.41, 137.60, 134.65, 134.46, 132.25, 132.01, 131.85, 131.63, 128.77, 128.61, 128.40, 127.87, 125.84, 125.00, 123.43, 121.60, 117.31, 117.11, 64.84, 64.48, 25.50, 17.97, 17.89. HRMS (ESI<sup>+</sup>): calcd for C<sub>28</sub>H<sub>26</sub>N<sub>3</sub>O<sub>4</sub> (M + H)<sup>+</sup>, 468.1923; found, 468.1914.</div></div><div id="sec5_3_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 2-Methyl-<i>N</i>-(2-methyl-5-(<i>N</i>-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)phenyl)quinoline-6-carboxamide <b>13</b></h4><div class="NLM_p">4-Methyl-3-nitroaniline <b>2</b> (5.15 g, 33.8 mmol) was dissolved in DMF (20 mL), and iodomethane (1.8 mL, 28.9 mmol) was added. Then, satd NaHCO<sub>3</sub> (aq) (20 mL, 28.2 mmol) was added portionwise to the stirred reaction mixture. The resulting reaction mixture was stirred at room temperature overnight. The reaction mixture was partitioned between dichloromethane and water. The aqueous layer was extracted with dichloromethane. The combined organic layers were then washed with water and brine. The crude product was purified by column chromatography (Biotage, gradient of 0–20% ethyl acetate in cyclohexane) to afford <i>N</i>,4-dimethyl-3-nitroaniline as a red amorphous solid (1.55 g, 33% yield). IR (thin film): ν<sub>max</sub> 3401, 2926, 1690, 1526, 1493, 1321, 1286 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.18 (d, <i>J</i> = 2.6 Hz, 1H), 7.10 (d, <i>J</i> = 8.3 Hz, 1H), 6.75 (dd, <i>J</i> = 8.3, 2.6 Hz, 1H), 3.91 (s, 1H), 2.87 (s, 3H), 2.47 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 149.77, 148.06, 133.16, 121.34, 117.63, 106.97, 30.65, 19.53. HRMS (ESI<sup>+</sup>): calcd for C<sub>8</sub>H<sub>11</sub>N<sub>2</sub>O<sub>2</sub> (M + H)<sup>+</sup>, 167.0820; found, 167.0825.</div><div class="NLM_p">2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxylic acid (1.63 g, 9.03 mmol), oxalyl chloride (970 μL, 10.86 mmol), and DMF (140 μL, 1.81 mmol) in anhydrous dichloromethane (15 mL) were reacted according to method A. The resulting acid chloride and <i>N</i>-4-dimethyl-3-nitroaniline (1.5 g, 9.03 mmol) were then reacted in anhydrous dichloromethane (15 mL) and pyridine (3.0 mL, 34.0 mmol). The reaction mixture was diluted with dichloromethane and washed with satd NH<sub>4</sub>Cl (aq). The aqueous layer was extracted with dichloromethane. The combined organic layers were washed with satd NH<sub>4</sub>Cl (aq), water, and brine. The organic layer was dried over sodium sulfate and reduced in vacuo until dryness to give <i>N</i>-methyl-<i>N</i>-(4-methyl-3-nitrophenyl)-2,3-dihydrobenz[<i>b</i>][1,4]dioxine-6-carboxamide (2.51 g), which was used in the next synthetic step without further purification.</div><div class="NLM_p"><i>N</i>-Methyl-<i>N</i>-(4-methyl-3-nitrophenyl)-2,3-dihydrobenz[<i>b</i>][1,4]dioxine-6-carboxamide (2.50 g, 7.61 mmol), iron powder (4.56 g, 82.0 mmol), and ammonium chloride (1.36 g, 80 mmol) in ethanol (20 mL) and water (5 mL) were reacted according to method C. The crude product was resuspended in water. The product was isolated by filtration and washed with water and diethyl ether. The product was isolated as a beige amorphous solid (1.18 g, 52%). IR (solid): ν<sub>max</sub> 3447, 3354, 2934, 1600, 1579, 1509, 1426, 1282 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 6.85–6.82 (m, 1H), 6.80 (d, <i>J</i> = 7.8 Hz, 1H), 6.78–6.72 (m, 1H), 6.66 (d, <i>J</i> = 8.4 Hz, 1H), 6.40–6.32 (m, 1H), 6.23–6.13 (m, 1H), 4.92 (s, 2H), 4.18 (d, <i>J</i> = 8.0 Hz, 4H), 3.23 (s, 3H), 1.97 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.76, 147.70, 144.89, 144.04, 142.72, 130.83, 129.67, 122.40, 119.85, 118.09, 116.54, 114.85, 112.22, 64.58, 64.32, 38.98, 17.45. HRMS (ESI<sup>+</sup>): calcd for C<sub>17</sub>H<sub>19</sub>N<sub>2</sub>O<sub>3</sub> (M + H)<sup>+</sup>, 300.1474; found, 299.1401.</div><div class="NLM_p last">2-Methylquinoline-6-carboxylic acid <b>3</b> (345 mg, 1.84 mmol), oxalyl chloride (180 μL, 2.06 mmol), and DMF (300 μL, 3.87 mmol) in anhydrous dichloromethane (10 mL) were reacted according to method A. The resulting acid chloride and <i>N</i>-(3-amino-4-methylphenyl)-<i>N</i>-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide (500 mg, 1.68 mmol) were then reacted in anhydrous dichloromethane (13 mL) and pyridine (1.2 mL, 14.89 mmol). The reaction mixture was diluted with water. The product was isolated by filtration and washed with water and diethyl ether. The crude product was then purified with a cation exchange sorbent (Isolute SCX-II, washing first with methanol, and then with 10% 2 M ammonia in methanol). The product was further purified by semipreparative HPLC according to the method specified in the general procedures to afford a yellow amorphous solid (22.4 mg, 2.9%). IR (solid): ν<sub>max</sub> 2927, 1624, 1493, 1282 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.09 (s, 1H), 8.59 (d, <i>J</i> = 1.8 Hz, 1H), 8.39 (d, <i>J</i> = 8.4 Hz, 1H), 8.21 (dd, <i>J</i> = 8.8, 1.9 Hz, 1H), 8.02 (d, <i>J</i> = 8.8 Hz, 1H), 7.52 (d, <i>J</i> = 8.4 Hz, 1H), 7.36 (d, <i>J</i> = 2.2 Hz, 1H), 7.18 (d, <i>J</i> = 8.2 Hz, 1H), 6.90 (dd, <i>J</i> = 8.1, 2.2 Hz, 1H), 6.87 (d, <i>J</i> = 2.0 Hz, 1H), 6.76 (d, <i>J</i> = 2.0 Hz, 1H), 6.68 (d, <i>J</i> = 8.4 Hz, 1H), 4.27–4.14 (m, 4H), 3.17 (s, 3H), 2.70 (s, 3H), 2.23 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.06, 161.29, 148.95, 144.99, 143.28, 142.94, 137.62, 137.39, 131.93, 131.75, 131.25, 129.44, 128.80, 128.75, 128.39, 125.81, 125.05, 124.52, 123.47, 122.42, 118.16, 116.71, 64.60, 64.33, 49.06, 38.94, 25.51, 18.04. HRMS (ESI<sup>+</sup>): calcd for C<sub>28</sub>H<sub>26</sub>N<sub>3</sub>O<sub>4</sub> (M + H)<sup>+</sup>, 468.1923; found, 468.1919.</div></div><div id="sec5_3_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-sulfonamido)-2-methylphenyl)-2-methylquinoline-6-carboxamide <b>14</b></h4><div class="NLM_p">2-Methylquinoline-6-carboxylic acid (3.69 g, 19.72 mmol), oxalyl chloride (1.8 mL, 20.15 mmol), and DMF (310 μL, 4.00 mmol) were reacted in anhydrous dichloromethane (35 mL) according to method A. The resulting acid chloride was reacted with 2-methyl-5-nitroaniline (3.0 g, 19.72 mmol) in anhydrous pyridine (35 mL). The reaction mixture was reduced in vacuo until dryness. The remaining residue was triturated with diethyl ether and dried in vacuo. The crude product was purified by column chromatography (Biotage, gradient of 0–50% ethanol in dichloromethane) to afford the product as a yellow amorphous solid (5.57 g, 88%). IR (thin film): ν<sub>max</sub> 1661, 1527, 1342, 1260 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.36 (s, 1H), 8.63 (d, <i>J</i> = 1.8 Hz, 1H), 8.43 (d, <i>J</i> = 8.4 Hz, 1H), 8.40 (d, <i>J</i> = 2.4 Hz, 1H), 8.25 (dd, <i>J</i> = 8.7, 2.0 Hz, 1H), 8.07 (d, <i>J</i> = 2.9 Hz, 1H), 8.05 (d, <i>J</i> = 3.0 Hz, 1H), 7.60 (d, <i>J</i> = 8.5 Hz, 1H), 7.54 (d, <i>J</i> = 8.4 Hz, 1H), 2.71 (s, 3H), 2.44 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.54, 161.05, 146.20, 144.90, 143.38, 142.15, 137.76, 132.04, 130.08, 129.93, 129.31, 126.62, 126.09, 124.16, 121.00, 120.94, 23.98, 18.82. HRMS (ESI<sup>+</sup>): calcd for C<sub>18</sub>H<sub>15</sub>NaN<sub>3</sub>O<sub>3</sub> (M + Na)<sup>+</sup>, 344.1006; found, 344.0999.</div><div class="NLM_p">2-Methyl-<i>N</i>-(2-methyl-5-nitrophenyl)quinoline-6-carboxamide (4.0 g, 12.45 mmol), iron powder (6.95 g, 124.0 mmol), and ammonium chloride (2.12 g, 124.0 mmol) in ethanol (50 mL) and water (12.5 mL) were reacted according to method C to afford <i>N</i>-(5-amino-2-methylphenyl)-2-methylquinoline-6-carboxamide as a light-yellow amorphous solid (1.95 g, 48% over two steps). IR (thin film): ν<sub>max</sub> 3340, 3252, 1645, 1582, 1487, 1278 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.86 (s, 1H), 8.61–8.51 (m, 1H), 8.38 (d, <i>J</i> = 8.4 Hz, 1H), 8.21 (dd, <i>J</i> = 8.8, 1.7 Hz, 1H), 8.00 (d, <i>J</i> = 8.8 Hz, 1H), 7.51 (d, <i>J</i> = 8.4 Hz, 1H), 6.91 (d, <i>J</i> = 8.1 Hz, 1H), 6.65 (d, <i>J</i> = 2.0 Hz, 1H), 6.42 (dd, <i>J</i> = 8.1, 2.2 Hz, 1H), 4.94 (s, 2H), 2.70 (s, 3H), 2.09 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.18, 161.09, 148.86, 147.31, 137.58, 137.01, 132.25, 130.87, 128.72, 128.49, 128.38, 125.84, 123.38, 120.72, 112.75, 112.61, 25.49, 17.50. HRMS (ESI<sup>+</sup>): calcd for C<sub>18</sub>H<sub>18</sub>N<sub>3</sub>O (M + H)<sup>+</sup>, 292.1444; found, 292.1446.</div><div class="NLM_p last"><i>N</i>-(5-Amino-2-methylphenyl)-2-methylquinoline-6-carboxamide (500 mg, 1.60 mmol) and 2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-sulfonyl chloride (591 mg, 2.39 mmol) and anhydrous pyridine (7 mL) were added to a round-bottom flask. The resulting reaction mixture was stirred at room temperature under argon overnight. The reaction mixture was diluted with diethyl ether. The product was isolated by filtration and washed with water and diethyl ether. The crude product was purified by column chromatography (Biotage, gradient of 0–10% ethanol in dichloromethane) to afford the product as a beige amorphous solid (90.7 mg, 12%). IR (thin film): ν<sub>max</sub> 1651, 1529, 1494, 1328, 1285, 1254, 1153 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.14 (s, 1H), 10.06 (s, 1H), 8.57 (d, <i>J</i> = 1.9 Hz, 1H), 8.40 (d, <i>J</i> = 8.4 Hz, 1H), 8.20 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 8.01 (d, <i>J</i> = 8.8 Hz, 1H), 7.52 (d, <i>J</i> = 8.4 Hz, 1H), 7.30–7.23 (m, 2H), 7.22 (d, <i>J</i> = 2.2 Hz, 1H), 7.14 (d, <i>J</i> = 8.4 Hz, 1H), 6.99 (d, <i>J</i> = 8.4 Hz, 1H), 6.94 (dd, <i>J</i> = 8.2, 2.2 Hz, 1H), 4.53–3.98 (m, 4H), 2.70 (s, 3H), 2.16 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.30, 161.25, 147.61, 143.69, 137.73, 137.72, 137.30, 136.26, 132.43, 131.78, 131.30, 129.59, 128.75, 128.72, 128.41, 125.83, 123.47, 120.81, 118.35, 118.00, 117.84, 116.11, 64.80, 64.48, 25.46, 17.83. HRMS (ESI<sup>+</sup>): calcd for C<sub>26</sub>H<sub>24</sub>N<sub>3</sub>O<sub>5</sub>S (M + H)<sup>+</sup>, 490.1436; found, 490.1436.</div></div><div id="sec5_3_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-methylphenyl)quinoline-6-carboxamide <b>15</b></h4><div class="NLM_p last">6-Quinolinecarboxylic acid (0.064 g, 0.369 mmol), HATU (0.160 g, 0.422 mmol), and <i>N</i>-(3-amino-4-methylphenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide <b>18</b> (0.100 g, 0.352 mmol) in <i>N</i>,<i>N</i>-diisopropylethylamine (0.123 mL, 0.703 mmol) and anhydrous DMF (2.5 mL) were reacted according to method B to afford a white amorphous solid (141 mg, 91%). IR (thin film): ν<sub>max</sub> 1649, 1599, 1528, 1504, 1318, 1290, 1260, 1066 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.18 (s, 1H), 10.08 (s, 1H), 9.02 (dd, <i>J</i> = 4.2, 1.9 Hz, 1H), 8.67 (d, <i>J</i> = 1.9 Hz, 1H), 8.55 (dd, <i>J</i> = 8.5, 1.9 Hz, 1H), 8.29 (dd, <i>J</i> = 8.8, 2.2 Hz, 1H), 8.15 (d, <i>J</i> = 8.8 Hz, 1H), 7.88 (d, <i>J</i> = 2.1 Hz, 1H), 7.65 (dd, <i>J</i> = 9.6, 2.2 Hz, 1H), 7.59 (dd, <i>J</i> = 8.5, 2.2 Hz, 1H), 7.54 (d, <i>J</i> = 2.2 Hz, 1H), 7.51 (dd, <i>J</i> = 8.5, 2.2 Hz, 1H), 7.25 (d, <i>J</i> = 8.5 Hz, 1H), 6.99 (d, <i>J</i> = 8.5 Hz, 1H), 4.33–4.29 (m, 4H), 2.25 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.38, 164.83, 152.68, 149.25, 146.82, 143.40, 137.80, 137.67, 136.67, 132.83, 130.63, 129.61, 129.24, 128.95, 128.47, 128.14, 127.61, 122.73, 121.67, 119.06, 118.68, 117.31, 117.13, 64.86, 64.49, 17.95. HRMS (ESI<sup>+</sup>): calcd for C<sub>26</sub>H<sub>22</sub>N<sub>3</sub>O<sub>4</sub> (M + H)<sup>+</sup>, 440.1605; found, 440.1768.</div></div><div id="sec5_3_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-methylphenyl)isoquinoline-7-carboxamide <b>16</b></h4><div class="NLM_p last">Isoquinoline-7-carboxylic acid (50 mg, 0.290 mmol), HATU (130 mg, 0.343 mmol), and <i>N</i>-(3-amino-4-methylphenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide <b>18</b> (75 mg, 0.264 mmol) in <i>N</i>,<i>N</i>-diisopropylethylamine (0.101 mL, 0.580 mmol) and anhydrous DMF (1.8 mL) were reacted according to method B. The crude product was purified by trituration and brief sonication in ethyl acetate, then isolated by filtration and washed with ethyl acetate to afford the product as a white amorphous solid (52 mg, 45%). IR (thin film): ν<sub>max</sub> 1641, 1599, 1531, 1504, 1319, 1295, 1243, 1065 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO<i>-d</i><sub>6</sub>) δ 10.23 (s, 1H), 10.08 (s, 1H), 9.48 (s, 1H), 8.81 (s, 1H), 8.62 (d, <i>J</i> = 5.7 Hz, 1H), 8.30 (dd, <i>J</i> = 8.6, 1.7 Hz, 1H), 8.12 (d, <i>J</i> = 8.6 Hz, 1H), 7.94 (d, <i>J</i> = 5.7 Hz, 1H), 7.89 (d, <i>J</i> = 2.1 Hz, 1H), 7.59 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 7.54 (d, <i>J</i> = 2.1 Hz, 1H), 7.51 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 7.25 (d, <i>J</i> = 8.6 Hz, 1H), 6.98 (d, <i>J</i> = 8.4 Hz, 1H), 4.32–4.29 (m, 4H), 2.25 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO<i>-d</i><sub>6</sub>) δ 165.08, 164.82, 153.21, 146.81, 143.39, 143.20, 137.80, 137.37, 136.53, 134.19, 130.65, 130.19, 129.23, 128.72, 128.11, 127.81, 127.48, 121.65, 121.41, 119.02, 118.74, 117.30, 117.12, 64.85, 64.48, 17.93. HRMS (ESI<sup>+</sup>): calcd for C<sub>26</sub>H<sub>22</sub>N<sub>3</sub>O<sub>4</sub> (M + H)<sup>+</sup>, 440.1605; found, 440.1604.</div></div><div id="sec5_3_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-methylphenyl)-1,2,3,4-tetrahydroquinoline-6-carboxamide <b>17</b></h4><div class="NLM_p last">A suspension of <i>N</i>-(5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2 methylphenyl)quinoline-6-carboxamide <b>15</b> (0.097 g, 0.22 mmol) and Pd/C (10%, 0.025 g) in methanol (8 mL), ethyl acetate (6 mL), and 10 drops of glacial acetic acid was stirred under 1 atm hydrogen at 40 °C overnight, filtered through Celite with ethyl acetate, and concentrated. The residue was purified by flash column chromatography using a gradient of 10–22% ethyl acetate in dichloromethane to afford the title compound (72 mg, 74%) as a white amorphous solid. IR (solid): ν<sub>max</sub> 2924, 1634, 1604, 1579, 1504, 1291, 1248, 1065, 888 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.02 (s, 1H), 9.35 (s, 1H), 7.77 (d, <i>J</i> = 2.2 Hz, 1H), 7.57–7.52 (m, 4H), 7.50 (dd, <i>J</i> = 8.5, 2.2 Hz, 1H), 7.18 (d, <i>J</i> = 8.4 Hz, 1H), 6.98 (d, <i>J</i> = 8.4 Hz, 1H), 6.47 (d, <i>J</i> = 9.0 Hz, 1H), 6.35 (br s, 1H), 4.34–4.27 (m, 4H), 3.27–3.20 (m, 2H), 2.72 (t, <i>J</i> = 6.2 Hz, 2H), 2.17 (s, 3H), 1.85–1.76 (m, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.65, 164.74, 148.79, 146.76, 143.38, 137.56, 137.50, 130.31, 129.43, 129.00, 128.21, 127.29, 121.64, 120.44, 119.09, 119.06, 117.93, 117.28, 117.11, 112.41, 64.85, 64.48, 41.05, 27.27, 21.50, 17.95. HRMS (ESI<sup>+</sup>): calcd for C<sub>26</sub>H<sub>26</sub>N<sub>3</sub>O<sub>4</sub> (M + H)<sup>+</sup>, 444.1718; found, 444.1812.</div></div><div id="sec5_3_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> <i>N</i>-(3-Amino-4-methylphenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide <b>18</b></h4><div class="NLM_p">1,4-Benzodioxane-6-carboxylic acid (2.486 g, 13.80 mmol), oxalyl chloride (1.40 mL, 16.6 mmol), and DMF (0.027 mL, 0.34 mmol) in anhydrous dichloromethane (34 mL) were reacted according to method A. The resulting acid chloride was dissolved in anhydrous dichloromethane (12 mL) and added dropwise to a solution of 4-methyl-3-nitroaniline <b>2</b> (2.100 g, 13.80 mmol) and pyridine (2.23 mL, 27.6 mmol) in anhydrous dichloromethane (35 mL). The reaction mixture was stirred at room temperature for 2 h and then concentrated. The resulting amorphous solid was suspended in methanol, diluted with water, and then isolated by filtration and washed with water to afford <i>N</i>-(4-methyl-3-nitrophenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide (4.24 g, 98%) as a pale-tan colored amorphous solid, which was used in the next step without further purification.</div><div class="NLM_p last">Palladium (10% on activated carbon, 0.567 g) and <i>N</i>-(4-methyl-3-nitrophenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide (4.237 g, 13.48 mmol) in ethanol (90 mL) and ethyl acetate (90 mL) were reacted according to method D to afford the desired product (3.803 g, 99%) as a pale-yellow amorphous solid. IR (thin film): ν<sub>max</sub> 1641, 1611, 1582, 1289, 1064 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.70 (s, 1H), 7.49 (d, <i>J</i> = 2.2 Hz, 1H), 7.46 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 7.10 (d, <i>J</i> = 2.0 Hz, 1H), 6.95 (d, <i>J</i> = 8.4 Hz, 1H), 6.83 (d, <i>J</i> = 8.1 Hz, 1H), 6.79 (dd, <i>J</i> = 8.1, 2.0 Hz, 1H), 4.81 (s, 2H), 4.32–4.26 (m, 4H), 2.01 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.48, 146.89, 146.54, 143.30, 138.09, 130.04, 128.60, 121.54, 117.18, 117.05, 109.26, 106.88, 64.82, 64.47, 17.46. (1 carbon missing) HRMS (ESI<sup>+</sup>): calcd for C<sub>16</sub>H<sub>17</sub>N<sub>2</sub>O<sub>3</sub> (M + H)<sup>+</sup>, 285.1234; found, 285.1233.</div></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> Preparation of Compounds in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a></h3><div id="sec5_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-methylphenyl)-2-(2-methoxyethoxy)quinoline-6-carboxamide <b>21</b></h4><div class="NLM_p">NaH (60% in mineral oil, 0.198 g, 4.950 mmol) was added to 2-methoxyethanol (4.0 mL, 50.7 mmol) at 0 °C under inert atmosphere. The reaction mixture was stirred at 0 °C for 10 min then at room temperature for 20 min before 6-bromo-2-chloroquinoline (0.400 g, 1.649 mmol) was added. This suspension was then gradually heated to 70 °C, the amorphous solid dissolved as heated, and the resulting solution was allowed to stir overnight. The reaction mixture was cooled to room temperature and diluted with water and the resulting precipitate isolated by filtration. The precipitate was washed with water and dried to afford 6-bromo-2-(2-methoxyethoxy)quinoline as a dull-pink amorphous solid (0.497 g), which was used in the next step without further purification.</div><div class="NLM_p"><i>n</i>-BuLi (1.84 M in hexanes) (0.578 mL, 1.063 mmol) was added dropwise to a solution of 6-bromo-2-(2-methoxyethoxy)quinoline (0.250 g, 0.886 mmol) in anhydrous THF (3.0 mL) at −78 °C and the resulting brown-yellow mixture was stirred at −78 °C for 35 min. Solid CO<sub>2</sub> was added, and the resulting orange mixture was stirred at −78 °C for 5 min, then it was allowed to warm to room temperature and concentrated under reduced pressure. The residue was dissolved in brine (5.0 mL) and washed with dichloromethane (1 × 5.0 mL). The aqueous phase was acidified with 2M HCl (aq) to pH 3, and the resulting precipitate was isolated by filtration and washed with water to afford 2-(2-methoxyethoxy)quinoline-6-carboxylic acid as a pale pink amorphous solid (0.127 g), which was used in next step without further purification.</div><div class="NLM_p last">2-(2-Methoxyethoxy)quinoline-6-carboxylic acid (0.048 g, 0.193 mmol), HATU (0.084 g, 0.220 mmol), and <i>N</i>-(3-amino-4-methylphenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide <b>18</b> (0.050 g, 0.176 mmol) in <i>N</i>,<i>N</i>-diisopropylethylamine (0.068 mL, 0.387 mmol) and anhydrous DMF (1.25 mL) were reacted according to method B to afford the product <b>21</b> as a white amorphous solid (87 mg, 56% over 3 steps). IR (thin film): ν<sub>max</sub> 2925, 1642, 1602, 1582, 1526, 1502, 1345, 1286, 1261, 816, 784 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.07 (app. d, <i>J</i> = 3.0 Hz, 2H), 8.57 (d, <i>J</i> = 2.0 Hz, 1H), 8.39 (d, <i>J</i> = 8.8 Hz, 1H), 8.22 (dd, <i>J</i> = 8.7, 2.0 Hz, 1H), 7.89–7.84 (m, 2H), 7.58 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 7.54 (d, <i>J</i> = 2.1 Hz, 1H), 7.51 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 7.24 (d, <i>J</i> = 8.6 Hz, 1H), 7.14 (d, <i>J</i> = 8.8 Hz, 1H), 6.98 (d, <i>J</i> = 8.4 Hz, 1H), 4.61–4.55 (m, 2H), 4.32–4.29 (m, 4H), 3.77–3.72 (m, 2H), 3.33 (s, 3H), 2.24 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.39, 164.81, 163.13, 147.93, 146.80, 143.38, 140.80, 137.75, 136.78, 130.57, 129.22, 128.90, 128.58, 128.15, 127.21, 126.76, 124.52, 121.65, 119.07, 118.57, 117.29, 117.12, 114.39, 70.56, 65.40, 64.85, 64.48, 58.58, 17.94. HRMS (ESI<sup>+</sup>): calcd for C<sub>29</sub>H<sub>28</sub>N<sub>3</sub>O<sub>6</sub> (M + H)<sup>+</sup>, 514.1973; found, 514.1969.</div></div><div id="sec5_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-methylphenyl)-2-(3-(dimethylamino)propoxy)quinoline-6-carboxamide <b>22</b></h4><div class="NLM_p">NaH (60% in mineral oil, 0.109 g, 2.72 mmol), 4-(2-hydroxyethyl)morpholine (0.337 mL, 2.78 mmol), and 6-bromo-2-chloroquinoline (0.3 g, 1.237 mmol) in anhydrous THF (4 mL) were reacted according to method E. The reaction mixture was heated for 1 h. The crude product was purified by flash column chromatography (4–6% methanol in dichloromethane) to afford 3-((6-bromoquinolin-2-yl)oxy)-<i>N</i>,<i>N</i>-dimethylpropan-1-amine as a pale-salmon colored oil (0.346 g), which was used directly in the next step.</div><div class="NLM_p"><i>n</i>-BuLi (1.62 M in hexanes, 0.788 mL, 1.277 mmol) and 3-((6-bromoquinolin-2-yl)oxy)-<i>N</i>,<i>N</i>-dimethylpropan-1-amine (0.329 g, 1.064 mmol) in anhydrous THF (3.5 mL) were reacted according to method F to afford 2-(3-(dimethylamino)propoxy)quinoline-6-carboxylic acid as a white amorphous solid (0.170 g), which was used in the next step without further purification.</div><div class="NLM_p last">2-(3-(Dimethylamino)propoxy)quinoline-6-carboxylic acid (91 mg, 0.220 mmol), HATU (104 mg, 0.275 mmol), and <i>N</i>-(3-amino-4-methylphenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide <b>18</b> (50 mg, 0.176 mmol) in <i>N</i>,<i>N</i>-diisopropylethylamine (0.154 mL, 0.879 mmol) and anhydrous DMF (1.5 mL) were reacted according to method B. The crude product was purified by flash column chromatography (3.5–6.6% 2M NH<sub>3</sub>-methanol in dichloromethane) to afford the product <b>22</b> as a light-tan amorphous solid (0.078 g, 38% over 3 steps). IR (thin film): ν<sub>max</sub> 2945, 1647, 1603, 1583, 1528, 1502, 1286, 1262 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.07 (s, 2H), 8.56 (d, <i>J</i> = 2.0 Hz, 1H), 8.37 (d, <i>J</i> = 8.8 Hz, 1H), 8.22 (dd, <i>J</i> = 8.7, 2.0 Hz, 1H), 7.88–7.84 (m, 2H), 7.58 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 7.54 (d, <i>J</i> = 2.1 Hz, 1H), 7.51 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 7.24 (d, <i>J</i> = 8.5 Hz, 1H), 7.10 (d, <i>J</i> = 8.8 Hz, 1H), 6.98 (d, <i>J</i> = 8.4 Hz, 1H), 4.47 (t, <i>J</i> = 6.7 Hz, 2H), 4.34–4.27 (m, 4H), 2.39 (t, <i>J</i> = 7.1 Hz, 2H), 2.24 (s, 3H), 2.17 (s, 6H), 1.93 (p, <i>J</i> = 6.8 Hz, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.94, 164.35, 162.88, 147.63, 146.34, 142.93, 140.17, 137.29, 136.33, 130.10, 130.01, 128.76, 128.39, 128.12, 127.69, 126.73, 123.98, 121.19, 118.62, 118.11, 116.84, 116.66, 113.98, 64.38, 64.02, 55.76, 45.21, 26.55, 17.48. HRMS (ESI<sup>+</sup>): calcd for C<sub>31</sub>H<sub>33</sub>N<sub>4</sub>O<sub>5</sub> (M + H)<sup>+</sup>, 541.2446; found, 541.24423.</div></div><div id="sec5_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-methylphenyl)-2-(3-morpholinopropoxy)quinoline-6-carboxamide <b>23</b></h4><div class="NLM_p">4-(3-Hydroxypropyl)morpholine (0.171 mL, 1.24 mmol), NaH (60% mineral oil, 0.047 g, 1.19 mmol), and 6-bromo-2-chloroquinoline (0.250 g, 1.03 mmol) in anhydrous THF (3.50 mL) were reacted according to method E. The reaction mixture was heated for 16 h. Purification using flash column chromatography (1.5–3.5% methanol in dichloromethane) gave 4-(3-((6-bromoquinolin-2-yl)oxy)propyl)morpholine (221 mg) as a white amorphous solid, which was used directly in the next step.</div><div class="NLM_p"><i>n</i>-BuLi (1.56 M in hexanes, 0.460 mL, 0.717 mmol) and 4-(3-((6-bromoquinolin-2-yl)oxy)propyl)morpholine (0.210 g, 0.598 mmol) in anhydrous THF (2.00 mL) were reacted according to method F to afford 2-(3-morpholinopropoxy)quinoline-6-carboxylic acid (0.273 g) as a yellow amorphous solid, which was used in the next step without further purification.</div><div class="NLM_p last">2-(3-Morpholinopropoxy)quinoline-6-carboxylic acid hydrochloride (0.076 g, 0.188 mmol), HATU (0.089 g, 0.234 mmol), and <i>N</i>-(3-amino-4-methylphenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide <b>18</b> (0.040 g, 0.141 mmol) in <i>N</i>,<i>N</i>-diisopropylethylamine (0.139 mL, 0.797 mmol) and anhydrous DMF (1.35 mL) were reacted according to method B. The crude product was purified by flash column chromatography to afford compound <b>23</b> as a white amorphous solid (51.0 mg, 47% over 3 steps). IR (film): ν<sub>max</sub> 3189, 2934, 1636, 1504, 1293, 1066, 819 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.07 (s, 2H), 8.56 (d, <i>J</i> = 2.1 Hz, 1H), 8.37 (d, <i>J</i> = 8.9 Hz, 1H), 8.22 (dd, <i>J</i> = 8.7, 2.0 Hz, 1H), 7.88–7.83 (m, 2H), 7.58 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 7.54 (d, <i>J</i> = 2.2 Hz, 1H), 7.51 (dd, <i>J</i> = 8.5, 2.2 Hz, 1H), 7.24 (d, <i>J</i> = 8.3 Hz, 1H), 7.10 (d, <i>J</i> = 8.8 Hz, 1H), 6.98 (d, <i>J</i> = 8.4 Hz, 1H), 4.49 (t, <i>J</i> = 6.6 Hz, 2H), 4.34–4.27 (m, 4H), 3.59 (t, <i>J</i> = 4.7 Hz, 4H), 2.50–2.44 (m, 2H), 2.39 (br s, 4H), 2.24 (s, 3H), 1.97 (p, <i>J</i> = 6.7 Hz, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.95, 164.36, 162.88, 147.62, 146.35, 142.94, 140.21, 137.30, 136.34, 130.12, 130.03, 128.77, 128.42, 128.14, 127.70, 126.72, 124.00, 121.21, 118.63, 118.12, 116.85, 116.67, 114.00, 66.20, 64.40, 64.33, 64.03, 54.93, 53.34, 25.47, 17.49 (2 carbons missing). HRMS (ESI<sup>+</sup>): calcd for C<sub>33</sub>H<sub>35</sub>N<sub>4</sub>O<sub>6</sub> (M + H)<sup>+</sup>, 583.2551; found, 583.2586.</div></div><div id="sec5_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-methylphenyl)-2-(3-(piperidin-1-yl)propoxy)quinoline-6-carboxamide <b>24</b></h4><div class="NLM_p">1-Piperidinepropanol (0.235 mL, 1.55 mmol), NaH (60% in mineral oil, 0.059 g), and 6-bromo-2-chloroquinoline (0.300 g, 1.24 mmol) in anhydrous THF (4.00 mL) were reacted according to method E. The reaction was heated for 5 h. Purification using flash column chromatography (2.5–6% methanol in dichloromethane) isolated 6-bromo-2-(3-(piperidin-1-yl)propoxy)quinoline (329 mg) as a yellow oil, which was used directly in the next step.</div><div class="NLM_p"><i>n</i>-BuLi (1.56 M in hexanes, 0.732 mL, 1.14 mmol) and 6-bromo-2-(3-(piperidin-1-yl)propoxy)quinoline (0.319 g, 0.913 mmol) in anhydrous THF (3.10 mL) were reacted according to method F to afford the crude product, which was used in the next step without further purification.</div><div class="NLM_p last">2-(3-(Piperidin-1-yl)propoxy)quinoline-6-carboxylic acid hydrochloride (76 mg, 0.188 mmol), HATU (89 mg, 0.234 mmol), and <i>N</i>-(3-amino-4-methylphenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide <b>18</b> (40 mg, 0.141 mmol) in <i>N</i>,<i>N</i>-diisopropylethylamine (0.139 mL, 0.797 mmol) and anhydrous DMF (1.35 mL) were reacted according to method B. The crude product was purified by flash column chromatography (3.5–15% methanol in dichloromethane) to afford compound <b>24</b> as a white amorphous solid (53 mg, 49% over 3 steps). IR (film): ν<sub>max</sub> 3257, 2928, 1601, 1526, 1284, 1064, 818 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.07 (s, 2H), 8.57 (d, <i>J</i> = 2.1 Hz, 1H), 8.38 (d, <i>J</i> = 8.9 Hz, 1H), 8.23 (dd, <i>J</i> = 8.8, 2.1 Hz, 1H), 7.89–7.83 (m, 2H), 7.58 (dd, <i>J</i> = 8.3, 2.1 Hz, 1H), 7.54 (d, <i>J</i> = 2.1 Hz, 1H), 7.51 (dd, <i>J</i> = 8.5, 2.2 Hz, 1H), 7.24 (d, <i>J</i> = 8.4 Hz, 1H), 7.10 (d, <i>J</i> = 8.8 Hz, 1H), 6.98 (d, <i>J</i> = 8.4 Hz, 1H), 4.49 (t, <i>J</i> = 6.5 Hz, 2H), 4.34–4.26 (m, 4H), 2.70–2.30 (m, 6H), 2.24 (s, 3H), 2.02 (br s, 2H), 1.56 (br s, 4H), 1.42 (br s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.94, 164.37, 162.80, 147.59, 146.35, 142.94, 140.25, 137.31, 136.33, 130.12, 130.06, 128.78, 128.44, 128.15, 127.69, 126.72, 124.03, 121.20, 118.64, 118.14, 116.85, 116.67, 113.96, 64.40, 64.21, 64.03, 54.82, 53.67, 24.99, 23.34, 17.48, (3 carbons missing). HRMS (ESI<sup>+</sup>): calcd for C<sub>34</sub>H<sub>37</sub>N<sub>4</sub>O<sub>5</sub> (M + H)<sup>+</sup>, 581.2758; found, 581.2759.</div></div><div id="sec5_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-methylphenyl)-2-(2-(piperidin-1-yl)ethoxy)quinoline-6-carboxamide <b>25</b></h4><div class="NLM_p">4-(2-Hydroxyethyl)piperidine (0.197 mL, 1.49 mmol), NaH (60% in mineral oil, 0.057 g, 1.42 mmol), and 6-bromo-2-chloroquinoline (0.300 g, 1.24 mmol) in anhydrous THF (4 mL) were reacted according to method E. The reaction mixture was heated for 5 h. Purification by flash column chromatography (4–5% methanol in dichloromethane) isolated 334 mg of a pale-yellow oil, which was used directly in the next step.</div><div class="NLM_p"><i>n</i>-BuLi (2.28 M in hexanes, 0.437 mL, 0.996 mmol) and 6-bromo-2-(2-(piperidin-1-yl)ethoxy)quinoline (0.204 g, 0.664 mmol) in anhydrous THF (2.20 mL) were reacted according to method F (a precipitate formed after addition of <i>n</i>-BuLi therefore further THF (1.0 mL) was added to facilitate stirring) to afford the crude product, which was used in the next step without further purification.</div><div class="NLM_p last">2-(2-(Piperidin-1-yl)ethoxy)quinoline-6-carboxylic acid hydrochloride (85%, 93 mg, 0.234 mmol), HATU (111 mg, 0.293 mmol), and <i>N</i>-(3-amino-4-methylphenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide <b>18</b> (50 mg, 0.176 mmol) in <i>N</i>,<i>N</i>-diisopropylethylamine (0.174 mL, 0.997 mmol) and anhydrous DMF (1.5 mL) were reacted according to method B. The crude product was purified by flash column chromatography (3.5–10% methanol in dichloromethane) to afford compound <b>25</b> as a white amorphous solid (75 mg, 75%). IR (film): ν<sub>max</sub> 3230, 2932, 1636, 1504, 1291, 1040, 822 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.07 (s, 2H), 8.57 (d, <i>J</i> = 1.9 Hz, 1H), 8.38 (d, <i>J</i> = 8.9 Hz, 1H), 8.22 (dd, <i>J</i> = 8.7, 2.0 Hz, 1H), 7.88–7.84 (m, 2H), 7.58 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 7.54 (d, <i>J</i> = 2.1 Hz, 1H), 7.51 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 7.24 (d, <i>J</i> = 8.5 Hz, 1H), 7.11 (d, <i>J</i> = 8.8 Hz, 1H), 6.98 (d, <i>J</i> = 8.4 Hz, 1H), 4.60–4.54 (m, 2H), 4.34–4.27 (m, 4H), 2.75 (br s, 2H), 2.61–2.36 (br s, 4H), 2.24 (s, 3H), 1.51 (br s, 4H), 1.39 (br s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.39, 164.81, 163.15, 148.00, 146.80, 143.39, 140.71, 137.75, 136.79, 130.57, 130.52, 129.23, 128.88, 128.57, 128.15, 127.21, 124.48, 121.65, 119.08, 118.58, 117.30, 117.12, 114.47, 64.86, 64.48, 63.73, 57.50, 54.76, 25.95, 24.30, 17.94 (2 carbons missing). HRMS (ESI<sup>+</sup>): calcd for C<sub>33</sub>H<sub>35</sub>N<sub>4</sub>O<sub>5</sub> (M + H)<sup>+</sup>, 567.2602; found, 567.2598.</div></div><div id="sec5_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-methylphenyl)-2-(2-(pyrrolidin-1-yl)ethoxy)quinoline-6-carboxamide <b>26</b></h4><div class="NLM_p">NaH (60% in mineral oil, 57 mg, 1.42 mmol), 4-(2-hydroxyethyl)pyrrolidine (171 mg, 1.48 mmol), and bromo-2-chloroquinoline (300 mg, 1.24 mmol) were reacted according to method E. The reaction was heated for 4 h. Purification by flash column chromatography using a gradient of 2–5% methanol in dichloromethane afforded a pale-yellow oil (329 mg), which was used directly in the next step.</div><div class="NLM_p"><i>n</i>-BuLi (1.62 M in hexanes, 0.76 mL, 1.23 mmol) and 6-bromo-2-(2-(pyrrolidin-1-yl)ethoxy)quinoline (38 mg, 0.990 mmol) in anhydrous THF (3.3 mL) were reacted according to method F to afford the crude product, which was used in the next step without further purification.</div><div class="NLM_p last">2-(2-(Pyrrolidin-1-yl)ethoxy)quinoline-6-carboxylic acid hydrochloride (89 mg, 0.23 mmol), HATU (111 mg, 0.29 mmol), and <i>N</i>-(3-amino-4-methylphenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide (50 mg, 0.18 mmol) in <i>N</i>,<i>N</i>-diisopropylethylamine (0.17 mL, 1.0 mmol) and anhydrous DMF (1.5 mL) was reacted according to method B. The crude product was purified by flash column chromatography using a gradient of 5–12% methanol in dichloromethane to afford the product <b>26</b> as a white amorphous solid (66 mg, 70%). IR (solid): ν<sub>max</sub> 1637, 1601, 1581, 1289, 1065, 819 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.07 (s, 2H), 8.57 (d, <i>J</i> = 2.0 Hz, 1H), 8.38 (d, <i>J</i> = 8.8 Hz, 1H), 8.22 (dd, <i>J</i> = 8.7, 2.0 Hz, 1H), 7.88–7.84 (m, 2H), 7.58 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 7.54 (d, <i>J</i> = 2.1 Hz, 1H), 7.51 (dd, <i>J</i> = 8.5, 2.2 Hz, 1H), 7.24 (d, <i>J</i> = 8.5 Hz, 1H), 7.12 (d, <i>J</i> = 8.8 Hz, 1H), 6.98 (d, <i>J</i> = 8.4 Hz, 1H), 4.57 (t, <i>J</i> = 5.8 Hz, 2H), 4.34–4.27 (m, 4H), 2.89 (br s, 2H), 2.58 (br s, 4H), 2.23 (s, 3H), 1.70 (br s, 4H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.39, 164.81, 163.15, 148.00, 146.80, 143.39, 140.72, 137.76, 136.79, 130.57, 130.52, 129.23, 128.88, 128.58, 128.15, 127.21, 124.49, 121.65, 119.08, 118.58, 117.30, 117.12, 114.48, 65.03, 64.86, 64.48, 54.58, 54.42, 40.32, 40.15, 23.61, 17.95. HRMS (ESI<sup>+</sup>): calcd for C<sub>32</sub>H<sub>33</sub>N<sub>4</sub>O<sub>5</sub> (M + H)<sup>+</sup>, 553.2446; found, 553.2467.</div></div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> Preparation of Compounds in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a></h3><div id="sec5_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> <i>N</i>-(4-Methyl-3-(6-(2-propionamidoethoxy)-2-naphthamido)phenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide <b>27</b></h4><div class="NLM_p last">Propionyl chloride (2.5 μL, 0.029 mmol) was added to a solution of 2-(2-aminoethoxy)-<i>N</i>-(5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-methylphenyl)quinoline-6-carboxamide <b>29</b> (0.013 g, 0.026 mmol) in anhydrous dichloromethane (0.75 mL) and DMF (0.15 mL). The reaction mixture was stirred at room temperature for 2.5 h, concentrated to remove dichloromethane, and then diluted with water, extracted with dichloromethane (3×), dried over magnesium sulfate, filtered, and concentrated. Heptane was added and concentrated (2×) to remove residual DMF. The crude material was purified by flash column chromatography using a gradient of 2–3.5% methanol in dichloromethane to afford the title compound <b>27</b> (10 mg, 71%) as a white amorphous solid. IR (thin film): ν<sub>max</sub> 2934, 1645, 1529, 1502, 1286, 1262, 1065 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.08 (s, 1H), 10.07 (s, 1H), 8.57 (d, <i>J</i> = 2.0 Hz, 1H), 8.39 (d, <i>J</i> = 8.8 Hz, 1H), 8.23 (dd, <i>J</i> = 8.7, 2.1 Hz, 1H), 8.04 (br t, <i>J</i> = 5.5 Hz, 1H), 7.88–7.84 (m, 2H), 7.58 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 7.54 (d, <i>J</i> = 2.1 Hz, 1H), 7.51 (dd, <i>J</i> = 8.5, 2.2 Hz, 1H), 7.24 (d, <i>J</i> = 8.4 Hz, 1H), 7.11 (d, <i>J</i> = 8.8 Hz, 1H), 6.98 (d, <i>J</i> = 8.4 Hz, 1H), 4.47 (t, <i>J</i> = 5.7 Hz, 2H), 4.34–4.27 (m, 4H), 3.51 (q, <i>J</i> = 5.7 Hz, 2H), 2.24 (s, 3H), 2.10 (q, <i>J</i> = 7.6 Hz, 2H), 0.99 (t, <i>J</i> = 7.6 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.66, 165.39, 164.81, 163.13, 147.95, 146.80, 143.39, 140.76, 137.75, 136.79, 130.58, 129.23, 128.91, 128.59, 128.15, 127.21, 124.54, 121.66, 119.08, 118.59, 117.30, 117.12, 114.42, 65.03, 64.86, 64.49, 38.44, 28.86, 17.95, 10.36, (1 carbon missing). HRMS (ESI<sup>+</sup>): calcd for C<sub>31</sub>H<sub>31</sub>N<sub>4</sub>O<sub>6</sub> (M + H)<sup>+</sup>, 555.2238; found, 555.2228.</div></div><div id="sec5_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-5-carboxamido)-2-methylphenyl)-2-(2-(pyrrolidin-1-yl)ethoxy)quinoline-6-carboxamide <b>28</b></h4><div class="NLM_p">2,3-Dihydrobenzo[<i>b</i>][1,4]dioxine-5-carboxylic acid (980 mg, 5.44 mmol), oxalyl chloride (0.552 mL, 6.53 mmol), and DMF (0.011 mL, 0.14 mmol) in anhydrous dichloromethane (14 mL) were reacted according to method A. The resulting acid chloride was dissolved in anhydrous dichloromethane (9 mL) and added dropwise to a solution of 4-methyl-3-nitroaniline <b>2</b> (828 mg, 5.44 mmol) and pyridine (0.88 mL, 10.88 mmol) in anhydrous dichloromethane (15 mL). A precipitate formed during the addition. The reaction mixture was stirred at room temperature overnight and then concentrated. The resulting amorphous solid was suspended in methanol, diluted with water, and then isolated by filtration and washed with water to afford <i>N</i>-(4-methyl-3-nitrophenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-5-carboxamide as a pale-brown amorphous solid (1.57 g, 92%). <sup>1</sup>H NMR (500 MHz, DMSO<i>-d</i><sub>6</sub>) δ 10.45 (s, 1H), 8.53 (d, <i>J</i> = 2.2 Hz, 1H), 7.88 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 7.49 (d, <i>J</i> = 8.5 Hz, 1H), 7.16 (dd, <i>J</i> = 7.6, 1.6 Hz, 1H), 7.04 (dd, <i>J</i> = 8.1, 1.6 Hz, 1H), 6.97–6.92 (m, 1H), 4.39–4.36 (m, 2H), 4.34–4.30 (m, 2H), 2.50 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO<i>-d</i><sub>6</sub>) δ 164.91, 149.06, 144.14, 141.73, 138.33, 133.53, 127.95, 125.57, 124.69, 121.71, 121.25, 119.85, 115.23, 64.96, 64.23, 19.68. HRMS (ESI<sup>+</sup>): calcd for C<sub>16</sub>H<sub>15</sub>N<sub>2</sub>O<sub>5</sub> (M + H)<sup>+</sup>, 315.0975; found, 315.0976.</div><div class="NLM_p">Palladium (10% on activated carbon, 0.325 g) and <i>N</i>-(4-methyl-3-nitrophenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-5-carboxamide (1.50 g, 4.77 mmol) in ethanol (32 mL) and ethyl acetate (32 mL) were reacted according to method D to afford <i>N</i>-(3-amino-4-methylphenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-5-carboxamide (1.30 g, 96%) as a white amorphous solid. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.35 (s, 1H), 7.80 (dd, <i>J</i> = 7.8, 1.6 Hz, 1H), 7.34 (br s, 1H), 7.03 (dd, <i>J</i> = 8.0, 1.7 Hz, 1H), 7.01–6.92 (m, 2H), 6.76 (dd, <i>J</i> = 8.0, 2.0 Hz, 1H), 4.49–4.44 (m, 3H), 4.36–4.30 (m, 3H), 3.54 (br s, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 162.58, 145.09, 143.61, 141.73, 137.17, 130.51, 124.23, 122.59, 121.51, 121.00, 118.39, 110.47, 107.22, 65.13, 63.59, 16.91. HRMS (ESI<sup>+</sup>): calcd for C<sub>16</sub>H<sub>17</sub>N<sub>2</sub>O<sub>3</sub> (M + H)<sup>+</sup>, 286.1265; found, 286.1263: calcd for C<sub>16</sub>H<sub>17</sub>N<sub>2</sub>NaO<sub>3</sub> (M + Na)<sup>+</sup>, 307.1053; found, 307.1048.</div><div class="NLM_p last">2-(2-(Pyrrolidin-1-yl)ethoxy)quinoline-6-carboxylic acid hydrochloride (95 mg, 0.253 mmol), HATU, (104 mg, 0.274 mmol), and <i>N</i>-(3-amino-4-methylphenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-5-carboxamide (0.060 g, 0.211 mmol) in <i>N</i>,<i>N</i>-diisopropylethylamine (0.147 mL, 0.844 mmol) and anhydrous DMF (2 mL) were reacted according to method B. The crude product was purified by flash column chromatography using a gradient of 5–10% methanol in dichloromethane to afford the title compound <b>28</b> as a cream amorphous solid (61 mg, 52%). IR (thin film): ν<sub>max</sub> 2926, 1651, 1619, 1601, 1527, 1470, 1282 cm<sup>–1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO<i>-d</i><sub>6</sub>) δ 10.10 (s, 1H), 10.09 (s, 1H), 8.57 (d, <i>J</i> = 2.0 Hz, 1H), 8.38 (d, <i>J</i> = 8.8 Hz, 1H), 8.23 (dd, <i>J</i> = 8.7, 2.0 Hz, 1H), 7.86 (d, <i>J</i> = 8.7 Hz, 1H), 7.81 (d, <i>J</i> = 2.1 Hz, 1H), 7.53 (dd, <i>J</i> = 8.3, 2.1 Hz, 1H), 7.24 (d, <i>J</i> = 8.4 Hz, 1H), 7.15 (dd, <i>J</i> = 7.6, 1.6 Hz, 1H), 7.12 (d, <i>J</i> = 8.8 Hz, 1H), 7.01 (dd, <i>J</i> = 8.0, 1.7 Hz, 1H), 6.92 (t, <i>J</i> = 7.8 Hz, 1H), 4.57 (t, <i>J</i> = 5.8 Hz, 2H), 4.40–4.35 (m, 2H), 4.30 (dt, <i>J</i> = 3.8, 2.3 Hz, 2H), 2.91 (br s, 2H), 2.60 (br s, 4H), 2.23 (s, 3H), 1.71 (s, 4H). <sup>13</sup>C NMR (126 MHz, DMSO<i>-d</i><sub>6</sub>) δ 165.38, 164.26, 163.15, 148.01, 144.11, 141.65, 140.73, 137.51, 136.95, 130.72, 130.45, 129.44, 128.90, 128.62, 127.20, 126.15, 124.49, 121.72, 121.17, 119.50, 118.45, 117.95, 114.48, 64.93, 64.21, 54.55, 54.42, 23.60, 17.98 (2 carbons missing). HRMS (ESI<sup>+</sup>): calcd for C<sub>32</sub>H<sub>33</sub>N<sub>4</sub>O<sub>5</sub> (M + H)<sup>+</sup>, 553.2445; found, 553.2424.</div></div><div id="sec5_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 2-(2-Aminoethoxy)-<i>N</i>-(5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-methylphenyl)quinoline-6-carboxamide <b>29</b></h4><div class="NLM_p">NaH (60% in mineral oil, 0.112 g, 2.79 mmol), <i>tert</i>-butyl <i>N</i>-(2-hydroxyethyl)carbamate (0.43 mL, 2.79 mmol), and 6-bromo-2-chloroquinoline (0.564 g, 2.33 mmol) in anhydrous THF (13 mL) were reacted according to method E. The reaction was heated for 5 h. Purification by flash column chromatography using a gradient of 20–25% diethyl ether in petroleum ether afforded <i>tert</i>-butyl (2-((6-bromoquinolin-2-yl)oxy)ethyl)carbamate as a white amorphous solid (248 mg, 29%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.92 (d, <i>J</i> = 8.9 Hz, 1H), 7.88 (d, <i>J</i> = 1.6 Hz, 1H), 7.74–7.66 (m, 2H), 6.94 (d, <i>J</i> = 8.8 Hz, 1H), 5.14 (br s, 1H), 4.56 (t, <i>J</i> = 5.1 Hz, 2H), 3.61 (br s, 2H), 1.46 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 145.03, 137.93, 132.86, 129.48, 128.93, 117.42, 113.99, 65.52, 40.14, 28.40, (2 carbons missing). HRMS (ESI<sup>+</sup>): calcd for C<sub>16</sub>H<sub>20</sub><sup>79</sup>BrN<sub>2</sub>O<sub>3</sub> (M + H)<sup>+</sup>, 367.0652; found, 367.0649.</div><div class="NLM_p"><i>n</i>-BuLi (2.12 M in hexanes, 0.350 mL, 0.743 mmol) and <i>tert</i>-butyl (2-((6-bromoquinolin-2-yl)oxy)ethyl)carbamate (0.124 g, 0.338 mmol) in anhydrous THF (3.0 mL) were reacted according to method F. The reaction was quenched with water and the reaction mixture concentrated to remove THF. The resulting aqueous solution was washed with ethyl acetate (1×) (brine was added to help clear the emulsion formed) and then acidified with 2M HCl to pH 2–3. The aqueous layer was extracted with dichloromethane (3×) and the combined organic layer dried over magnesium sulfate to afford 2-(2-((<i>tert</i>-butoxycarbonyl)amino)ethoxy)quinoline-6-carboxylic acid as a white amorphous solid (52 mg, 46%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.06 (br s, 1H), 8.56 (d, <i>J</i> = 1.9 Hz, 1H), 8.41 (d, <i>J</i> = 8.9 Hz, 1H), 8.13 (dd, <i>J</i> = 8.7, 2.0 Hz, 1H), 7.80 (d, <i>J</i> = 8.8 Hz, 1H), 7.07 (d, <i>J</i> = 8.9 Hz, 1H), 7.04 (t, <i>J</i> = 5.7 Hz, 1H), 4.43 (t, <i>J</i> = 5.8 Hz, 2H), 3.37 (q, <i>J</i> = 5.6 Hz, 2H), 1.37 (s, 9H). HRMS (ESI<sup>+</sup>): calcd for C<sub>17</sub>H<sub>21</sub>N<sub>2</sub>O<sub>5</sub> (M + H)<sup>+</sup>, 333.1445; found, 333.1447.</div><div class="NLM_p">2-(2-((<i>tert</i>-Butoxycarbonyl)amino)ethoxy)quinoline-6-carboxylic acid (46 mg, 0.138 mmol), HATU (66 mg, 0.173 mmol), and <i>N</i>-(3-aminophenyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide (39 mg, 0.138 mmol) in <i>N</i>,<i>N</i>-diisopropylethylamine (0.073 mL, 0.415 mmol) and anhydrous DMF (2 mL) were reacted according to method B. The crude material was purified by flash column chromatography using a gradient of 1–3% methanol in dichloromethane to afford the product as an off-white amorphous solid (77 mg, 56%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.07 (s, 1H), 10.07 (s, 1H), 8.57 (d, <i>J</i> = 2.1 Hz, 1H), 8.39 (d, <i>J</i> = 8.8 Hz, 1H), 8.22 (dd, <i>J</i> = 8.8, 2.1 Hz, 1H), 7.86 (d, <i>J</i> = 8.8 Hz, 1H), 7.86 (d, <i>J</i> = 2.2 Hz, 1H), 7.58 (dd, <i>J</i> = 8.5, 2.2 Hz, 1H), 7.54 (d, <i>J</i> = 2.0 Hz, 1H), 7.51 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 7.24 (d, <i>J</i> = 8.5 Hz, 1H), 7.09 (d, <i>J</i> = 8.9 Hz, 1H), 7.06 (t, <i>J</i> = 5.6 Hz, 1H), 6.98 (d, <i>J</i> = 8.4 Hz, 1H), 4.45 (t, <i>J</i> = 5.7 Hz, 2H), 4.33–4.28 (m, 2H), 3.39 (ap q, <i>J</i> = 5.7 Hz, 2H), 2.24 (s, 3H), 1.38 (s, 9H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.39, 164.81, 163.12, 147.95, 146.80, 143.39, 140.71, 137.76, 136.79, 130.57, 129.23, 128.88, 128.58, 128.15, 127.21, 124.53, 121.66, 119.08, 118.58, 117.30, 117.12, 114.47, 78.21, 65.13, 64.86, 64.48, 39.72, 28.70, 17.95 (2 carbons missing). HRMS (ESI<sup>+</sup>): calcd for C<sub>33</sub>H<sub>35</sub>N<sub>4</sub>O<sub>7</sub> (M + H)<sup>+</sup>, 599.2500; found, 599.2495.</div><div class="NLM_p last">Trifluoroacetic acid (0.75 mL) was added to a solution of <i>tert</i>-butyl (2-((6-((5-(2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamido)-2-methylphenyl)carbamoyl)quinolin-2-yl)oxy)ethyl)carbamate (0.039 g, 0.065 mmol) in anhydrous dichloromethane (0.75 mL), and the reaction mixture was stirred at room temperature for 75 min, concentrated, dichloromethane added, and concentrated again, diluted with a small amount of methanol, and then a 1:1 mixture of satd NaHCO<sub>3</sub> (aq) was slowly added. The resulting precipitate was isolated by filtration and washed with water. The crude material was purified by flash column chromatography using 10% methanol in dichloromethane then a gradient of 5–9% 2M NH<sub>3</sub> methanol/DCM to afford the title compound (22 mg, 68%) as an off-white amorphous solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.08 (s, 2H), 8.57 (d, <i>J</i> = 2.0 Hz, 1H), 8.39 (d, <i>J</i> = 8.9 Hz, 1H), 8.22 (dd, <i>J</i> = 8.7, 2.1 Hz, 1H), 7.89–7.84 (m, 2H), 7.58 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 7.54 (d, <i>J</i> = 2.1 Hz, 1H), 7.51 (dd, <i>J</i> = 8.5, 2.2 Hz, 1H), 7.24 (d, <i>J</i> = 8.4 Hz, 1H), 7.12 (d, <i>J</i> = 8.7 Hz, 1H), 6.99 (d, <i>J</i> = 8.4 Hz, 1H), 4.42 (t, <i>J</i> = 5.8 Hz, 2H), 4.33–4.28 (m, 4H), 2.99 (t, <i>J</i> = 5.9 Hz, 2H), 2.25 (s, 3H), 2.06 (br s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.41, 164.81, 163.40, 148.07, 146.81, 143.39, 140.64, 137.76, 136.80, 130.58, 130.48, 129.23, 128.86, 128.58, 128.15, 127.20, 124.48, 121.66, 119.08, 118.58, 117.31, 117.12, 114.51, 68.85, 64.86, 64.49, 41.13, 17.96. HRMS (ESI<sup>+</sup>): calcd for C<sub>28</sub>H<sub>27</sub>N<sub>4</sub>O<sub>5</sub> (M + H)<sup>+</sup>, 499.1976; found, 499.1974.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i58"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01055">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14645" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14645" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b01055" class="ext-link">10.1021/acs.jmedchem.6b01055</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">NMR spectra of final compounds, CDK inhibition assay, Cancergene cell line profiling, DiscoveRx KinomeScan, kinase inhibition assays, BRAF inhibitors, solubilizing group optimization, mouse pharmacokinetics, in vivo mouse efficacy studies, Cerep Diversity Screen, SILAC target identification, surface plasmon resonance (SPR), pirin crystallography, full experimental procedures (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">SMILES molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">1%DMSO WM266.4 migration assay VC (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_003.mov" class="ext-link">MOV</a>)</p></li><li><p class="inline">Bisamide 26 WM266.4 migration assay 100 nM (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_004.mov" class="ext-link">MOV</a>)</p></li><li><p class="inline">Bisamide 28 WM266.4 migration assay 100 nM (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_005.mov" class="ext-link">MOV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_001.pdf">jm6b01055_si_001.pdf (12.6 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_002.csv">jm6b01055_si_002.csv (2.02 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_003.mov">jm6b01055_si_003.mov (553.04 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_004.mov">jm6b01055_si_004.mov (555.48 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_005.mov">jm6b01055_si_005.mov (565.79 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and structure factors for the crystal structures of pirin with compound <b>26</b> can be accessed using PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JCT">5JCT</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.6b01055" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32324" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32324" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul Workman</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1848796d74364f776a7375797658717b6a36797b366d73"><span class="__cf_email__" data-cfemail="baeadbcfd694edd5c8d1d7dbd4fad3d9c894dbd994cfd1">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Keith Jones</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9440-4094" title="Orcid link">http://orcid.org/0000-0002-9440-4094</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#8bc0eee2ffe3a5c1e4e5eef8cbe2e8f9a5eae8a5fee0"><span class="__cf_email__" data-cfemail="4308262a372b6d092c2d2630032a20316d22206d3628">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew D. Cheeseman</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicola E. A. Chessum</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carl S. Rye</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">A. Elisa Pasqua</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael J. Tucker</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Birgit Wilding</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1896-3708" title="Orcid link">http://orcid.org/0000-0002-1896-3708</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lindsay E. Evans</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Susan Lepri</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Meirion Richards</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Swee Y. Sharp</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Salyha Ali</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Division
of Structural Biology at The Institute of
Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Rowlands</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lisa O’Fee</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Asadh Miah</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Angela Hayes</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alan T. Henley</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marissa Powers</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert te Poele</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emmanuel De Billy</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Loredana Pellegrino</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Florence Raynaud</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rosemary Burke</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rob L. M. van Montfort</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Division
of Structural Biology at The Institute of
Cancer Research, London SW7 3RP, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Suzanne A. Eccles</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit at The Institute of Cancer Research, London SW7 3RP, United Kingdom</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The Institute of Cancer Research has a potential financial interest in inhibitors of the HSF1 pathway and operates a Rewards to Discoverers.<br /></br></div></li></ul></div><div class="ack" id="ACK-d121e5847-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i61">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82772" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82772" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by Cancer Research UK grant numbers C309/A8274 and C309/A11566. We acknowledge CRUK Centre funding and NHS funding to the NIHR Biomedical Research Centre at The Institute of Cancer Research, the Royal Marsden Hospital, and funding from Cancer Research Technology Pioneer Fund and Battle Against Cancer Investment Trust. Professor Paul Workman is a CRUK Life Fellow. We thank Dr Nora Cronin and the staff of the European Synchrotron Radiation facility for their support during data collection.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i62" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i62"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i63" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i63"> Abbreviations Used</h2><tr><td class="NLM_term">CTB</td><td class="NLM_def"><p class="first last">Cell Titer Blue</p></td></tr><tr><td class="NLM_term">ELISA</td><td class="NLM_def"><p class="first last">enzyme-linked immunosorbent assay</p></td></tr><tr><td class="NLM_term">HSF1</td><td class="NLM_def"><p class="first last">heat shock transcription factor 1</p></td></tr><tr><td class="NLM_term">HSP70</td><td class="NLM_def"><p class="first last">70 kDa heat shock proteins</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure-activity relationship</p></td></tr><tr><td class="NLM_term">HSP72</td><td class="NLM_def"><p class="first last">heat shock 70 kDa protein 1</p></td></tr><tr><td class="NLM_term">HSP90</td><td class="NLM_def"><p class="first last">heat shock protein 90</p></td></tr><tr><td class="NLM_term">17-AAG</td><td class="NLM_def"><p class="first last">tanespimycin (17-<i>N</i>-allylamino-17-demethoxygeldanamycin)</p></td></tr><tr><td class="NLM_term">BRAF</td><td class="NLM_def"><p class="first last">B-Raf proto-oncogene, serine/threonine kinase</p></td></tr><tr><td class="NLM_term">KIT</td><td class="NLM_def"><p class="first last">tyrosine protein kinase KIT</p></td></tr><tr><td class="NLM_term">PDGFRA</td><td class="NLM_def"><p class="first last">platelet-derived growth factor receptor alpha polypeptide</p></td></tr><tr><td class="NLM_term">PDGFRB</td><td class="NLM_def"><p class="first last">β-type platelet-derived growth factor receptor</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screen</p></td></tr><tr><td class="NLM_term">nM</td><td class="NLM_def"><p class="first last">nanomolar</p></td></tr><tr><td class="NLM_term">μM</td><td class="NLM_def"><p class="first last">micromolar</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-minimal (50%) inhibitory concentration</p></td></tr><tr><td class="NLM_term">GI<sub>50</sub></td><td class="NLM_def"><p class="first last">concentration for 50% of maximal inhibition of cell proliferation</p></td></tr><tr><td class="NLM_term">CDK2</td><td class="NLM_def"><p class="first last">cyclin-dependent kinase 2</p></td></tr><tr><td class="NLM_term">CDK9</td><td class="NLM_def"><p class="first last">cyclin-dependent kinase 9</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">standard error of the mean</p></td></tr><tr><td class="NLM_term"><i>K</i></td><td class="NLM_def"><p class="first last">dissociation constant</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate</p></td></tr><tr><td class="NLM_term">MLM</td><td class="NLM_def"><p class="first last">mouse liver microsomes</p></td></tr><tr><td class="NLM_term">KS</td><td class="NLM_def"><p class="first last">kinetic solubility</p></td></tr><tr><td class="NLM_term">HBF</td><td class="NLM_def"><p class="first last">hepatic blood flow</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G protein-coupled receptors</p></td></tr><tr><td class="NLM_term">NFκB</td><td class="NLM_def"><p class="first last">nuclear factor kappa-light-chain-enhancer of activated B cells</p></td></tr><tr><td class="NLM_term">EAF2/U19</td><td class="NLM_def"><p class="first last">ELL-associated factor 2</p></td></tr><tr><td class="NLM_term">NRF2</td><td class="NLM_def"><p class="first last">nuclear factor erythroid 2 related factor 2</p></td></tr><tr><td class="NLM_term">BCL3</td><td class="NLM_def"><p class="first last">B-cell lymphoma 3-encoded protein</p></td></tr><tr><td class="NLM_term">NFI/CTF1</td><td class="NLM_def"><p class="first last">nuclear factor I</p></td></tr><tr><td class="NLM_term">p65</td><td class="NLM_def"><p class="first last">nuclear factor NF-kappa-B p65 subunit</p></td></tr><tr><td class="NLM_term">EMT</td><td class="NLM_def"><p class="first last">epithelial–mesenchymal transition</p></td></tr><tr><td class="NLM_term">GSK3β</td><td class="NLM_def"><p class="first last">glycogen synthase kinase 3 beta</p></td></tr><tr><td class="NLM_term">PDE6D</td><td class="NLM_def"><p class="first last">retinal rod rhodopsin-sensitive cGMP 3′,5′-cyclic phosphodiesterase subunit delta</p></td></tr><tr><td class="NLM_term">SILAC</td><td class="NLM_def"><p class="first last">stable isotope labeling by amino acids in cell culture</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">AUC</td><td class="NLM_def"><p class="first last">area under the curve</p></td></tr><tr><td class="NLM_term">AUC<sub>u</sub></td><td class="NLM_def"><p class="first last">unbound area under the curve</p></td></tr><tr><td class="NLM_term"><i>C</i><sub>av</sub><sup>0–24 h</sup></td><td class="NLM_def"><p class="first last">average concentration over a 24 h period</p></td></tr><tr><td class="NLM_term">po</td><td class="NLM_def"><p class="first last">per os, oral dose</p></td></tr><tr><td class="NLM_term">qd</td><td class="NLM_def"><p class="first last">quaqe die, once a day dose</p></td></tr><tr><td class="NLM_term">Cl</td><td class="NLM_def"><p class="first last">clearance</p></td></tr><tr><td class="NLM_term">Cl<sub>u</sub></td><td class="NLM_def"><p class="first last">unbound clearance</p></td></tr><tr><td class="NLM_term"><i>V</i><sub>ss</sub></td><td class="NLM_def"><p class="first last">volume of distribution at steady-state</p></td></tr><tr><td class="NLM_term"><i>V</i><sub>du</sub></td><td class="NLM_def"><p class="first last">unbound volume of distribution</p></td></tr><tr><td class="NLM_term">%<i>F</i></td><td class="NLM_def"><p class="first last">percentage oral bioavailability</p></td></tr><tr><td class="NLM_term"><i>f</i><sub>up</sub></td><td class="NLM_def"><p class="first last">fraction unbound in plasma</p></td></tr><tr><td class="NLM_term"><i>f</i><sub>ub</sub></td><td class="NLM_def"><p class="first last">fraction unbound in blood</p></td></tr><tr><td class="NLM_term"><i>f</i><sub>ua</sub></td><td class="NLM_def"><p class="first last">fraction unbound in the assay</p></td></tr><tr><td class="NLM_term">B:P</td><td class="NLM_def"><p class="first last">blood to plasma ratio</p></td></tr><tr><td class="NLM_term">CYP450</td><td class="NLM_def"><p class="first last">cytochrome P450</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last">diisopropylethylamine</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">satd</td><td class="NLM_def"><p class="first last">saturated</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i64">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18674" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18674" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 75 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Holohan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Schaeybroeck, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Longley, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, P. G.</span><span> </span><span class="NLM_article-title">Cancer drug resistance: an evolving paradigm</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">714</span><span class="NLM_x">–</span> <span class="NLM_lpage">726</span><span class="refDoi"> DOI: 10.1038/nrc3599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnrc3599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=24060863" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVyqurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=714-726&author=C.+Holohanauthor=S.+Van+Schaeybroeckauthor=D.+B.+Longleyauthor=P.+G.+Johnston&title=Cancer+drug+resistance%3A+an+evolving+paradigm&doi=10.1038%2Fnrc3599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1aR"><div class="casContent"><span class="casTitleNuber">1a</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer drug resistance: an evolving paradigm</span></div><div class="casAuthors">Holohan, Caitriona; Van Schaeybroeck, Sandra; Longley, Daniel B.; Johnston, Patrick G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">714-726</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Resistance to chemotherapy and molecularly targeted therapies is a major problem facing current cancer research.  The mechanisms of resistance to 'classical' cytotoxic chemotherapeutics and to therapies that are designed to be selective for specific mol. targets share many features, such as alterations in the drug target, activation of prosurvival pathways and ineffective induction of cell death.  With the increasing arsenal of anticancer agents, improving preclin. models and the advent of powerful high-throughput screening techniques, there are now unprecedented opportunities to understand and overcome drug resistance through the clin. assessment of rational therapeutic drug combinations and the use of predictive biomarkers to enable patient stratification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrykBEUSgvv7Vg90H21EOLACvtfcHk0liOcV_pJg_QZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVyqurnE&md5=b497a823bbd968904ce1591b918d0679</span></div><a href="/servlet/linkout?suffix=cit1a&amp;dbid=16384&amp;doi=10.1038%2Fnrc3599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3599%26sid%3Dliteratum%253Aachs%26aulast%3DHolohan%26aufirst%3DC.%26aulast%3DVan%2BSchaeybroeck%26aufirst%3DS.%26aulast%3DLongley%26aufirst%3DD.%2BB.%26aulast%3DJohnston%26aufirst%3DP.%2BG.%26atitle%3DCancer%2520drug%2520resistance%253A%2520an%2520evolving%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2013%26volume%3D13%26spage%3D714%26epage%3D726%26doi%3D10.1038%2Fnrc3599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit1b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Zahreddine, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borden, K. L. B.</span><span> </span><span class="NLM_article-title">Mechanisms and insights into drug resistance in cancer</span> <span class="citation_source-journal">Front. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="refDoi"> DOI: 10.3389/fphar.2013.00028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.3389%2Ffphar.2013.00028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=23504227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A280%3ADC%252BC3svnt1ymtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=28&author=H.+Zahreddineauthor=K.+L.+B.+Borden&title=Mechanisms+and+insights+into+drug+resistance+in+cancer&doi=10.3389%2Ffphar.2013.00028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1bR"><div class="casContent"><span class="casTitleNuber">1b</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms and insights into drug resistance in cancer</span></div><div class="casAuthors">Zahreddine Hiba; Borden Katherine L B</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">28</span>
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    </div><div class="casAbstract">Cancer drug resistance continues to be a major impediment in medical oncology.  Clinically, resistance can arise prior to or as a result of cancer therapy.  In this review, we discuss different mechanisms adapted by cancerous cells to resist treatment, including alteration in drug transport and metabolism, mutation and amplification of drug targets, as well as genetic rewiring which can lead to impaired apoptosis.  Tumor heterogeneity may also contribute to resistance, where small subpopulations of cells may acquire or stochastically already possess some of the features enabling them to emerge under selective drug pressure.  Making the problem even more challenging, some of these resistance pathways lead to multidrug resistance, generating an even more difficult clinical problem to overcome.  We provide examples of these mechanisms and some insights into how understanding these processes can influence the next generation of cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQFZ1-BkHcDE36HPRus0rAafW6udTcc2eajtacj8xmCybntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3svnt1ymtw%253D%253D&md5=e3b49dd34e6cc024e098fd053447a10d</span></div><a href="/servlet/linkout?suffix=cit1b&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2013.00028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2013.00028%26sid%3Dliteratum%253Aachs%26aulast%3DZahreddine%26aufirst%3DH.%26aulast%3DBorden%26aufirst%3DK.%2BL.%2BB.%26atitle%3DMechanisms%2520and%2520insights%2520into%2520drug%2520resistance%2520in%2520cancer%26jtitle%3DFront.%2520Pharmacol.%26date%3D2013%26volume%3D4%26spage%3D28%26doi%3D10.3389%2Ffphar.2013.00028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit1c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Garraway, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janne, P. A.</span><span> </span><span class="NLM_article-title">Circumventing cancer drug resistance in the era of personalized medicine</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">214</span><span class="NLM_x">–</span> <span class="NLM_lpage">226</span><span class="refDoi"> DOI: 10.1158/2159-8290.CD-12-0012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1158%2F2159-8290.CD-12-0012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=214-226&author=L.+A.+Garrawayauthor=P.+A.+Janne&title=Circumventing+cancer+drug+resistance+in+the+era+of+personalized+medicine&doi=10.1158%2F2159-8290.CD-12-0012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1c&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-12-0012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-12-0012%26sid%3Dliteratum%253Aachs%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26atitle%3DCircumventing%2520cancer%2520drug%2520resistance%2520in%2520the%2520era%2520of%2520personalized%2520medicine%26jtitle%3DCancer%2520Discovery%26date%3D2012%26volume%3D2%26spage%3D214%26epage%3D226%26doi%3D10.1158%2F2159-8290.CD-12-0012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit1d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banerji, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span> </span><span class="NLM_article-title">Combinatorial drug therapy for cancer in the post-genomic era</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">679</span><span class="NLM_x">–</span> <span class="NLM_lpage">692</span><span class="refDoi"> DOI: 10.1038/nbt.2284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnbt.2284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=22781697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvFGmtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=679-692&author=B.+Al-Lazikaniauthor=U.+Banerjiauthor=P.+Workman&title=Combinatorial+drug+therapy+for+cancer+in+the+post-genomic+era&doi=10.1038%2Fnbt.2284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1dR"><div class="casContent"><span class="casTitleNuber">1d</span><div class="casTitle"><span class="NLM_cas:atitle">Combinatorial drug therapy for cancer in the post-genomic era</span></div><div class="casAuthors">Al-Lazikani, Bissan; Banerji, Udai; Workman, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">679-692</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Over the past decade, whole genome sequencing and other 'omics' technologies have defined pathogenic driver mutations to which tumor cells are addicted.  Such addictions, synthetic lethalities and other tumor vulnerabilities have yielded novel targets for a new generation of cancer drugs to treat discrete, genetically defined patient subgroups.  This personalized cancer medicine strategy could eventually replace the conventional one-size-fits-all cytotoxic chemotherapy approach.  However, the extraordinary intratumor genetic heterogeneity in cancers revealed by deep sequencing explains why de novo and acquired resistance arise with molecularly targeted drugs and cytotoxic chemotherapy, limiting their utility.  One soln. to the enduring challenge of polygenic cancer drug resistance is rational combinatorial targeted therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon5IOOPfsgR7Vg90H21EOLACvtfcHk0lgs8GpIsb4HmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvFGmtLg%253D&md5=a61b1b222a7146a92366f57cb0e708ca</span></div><a href="/servlet/linkout?suffix=cit1d&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2284%26sid%3Dliteratum%253Aachs%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DBanerji%26aufirst%3DU.%26aulast%3DWorkman%26aufirst%3DP.%26atitle%3DCombinatorial%2520drug%2520therapy%2520for%2520cancer%2520in%2520the%2520post-genomic%2520era%26jtitle%3DNat.%2520Biotechnol.%26date%3D2012%26volume%3D30%26spage%3D679%26epage%3D692%26doi%3D10.1038%2Fnbt.2284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Kinch, M. S.</span><span> </span><span class="NLM_article-title">An analysis of FDA-approved drugs for oncology</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1831</span><span class="NLM_x">–</span> <span class="NLM_lpage">1835</span><span class="refDoi"> DOI: 10.1016/j.drudis.2014.08.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.drudis.2014.08.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=25172803" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A280%3ADC%252BC2M%252FmsFGmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=1831-1835&author=M.+S.+Kinch&title=An+analysis+of+FDA-approved+drugs+for+oncology&doi=10.1016%2Fj.drudis.2014.08.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2aR"><div class="casContent"><span class="casTitleNuber">2a</span><div class="casTitle"><span class="NLM_cas:atitle">An analysis of FDA-approved drugs for oncology</span></div><div class="casAuthors">Kinch Michael S</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1831-5</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cancer remains the second leading cause of death globally.  The number of new medicines targeting cancer has grown impressively since the 1990s.  On average, ten new drugs are introduced each year.  Such growth has partly been achieved by emphasizing biologics and orphan indications, which account for one-quarter and one-half of new oncology drugs, respectively.  The biotechnology industry likewise has become the primary driver of cancer drug development in terms of patents, preclinical and clinical research, although pharmaceutical companies are granted more FDA approvals.  Many targeting strategies have been successful but recent trends suggest that kinase targets, although tractable, might be overemphasized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR3e6HLwRjzXH6oPH0mYWL1fW6udTcc2eajtacj8xmCybntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M%252FmsFGmsw%253D%253D&md5=360c08e49bf159819dc5758b352e5016</span></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.08.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.08.007%26sid%3Dliteratum%253Aachs%26aulast%3DKinch%26aufirst%3DM.%2BS.%26atitle%3DAn%2520analysis%2520of%2520FDA-approved%2520drugs%2520for%2520oncology%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D1831%26epage%3D1835%26doi%3D10.1016%2Fj.drudis.2014.08.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit2b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Patel, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halling-Brown, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tym, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span> </span><span class="NLM_article-title">Objective assessment of cancer genes for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">35</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1038/nrd3913</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnrd3913" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=35-50&author=M.+N.+Patelauthor=M.+D.+Halling-Brownauthor=J.+E.+Tymauthor=P.+Workmanauthor=B.+Al-Lazikani&title=Objective+assessment+of+cancer+genes+for+drug+discovery&doi=10.1038%2Fnrd3913"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=10.1038%2Fnrd3913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3913%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DM.%2BN.%26aulast%3DHalling-Brown%26aufirst%3DM.%2BD.%26aulast%3DTym%26aufirst%3DJ.%2BE.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26atitle%3DObjective%2520assessment%2520of%2520cancer%2520genes%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D35%26epage%3D50%26doi%3D10.1038%2Fnrd3913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Kotz, J.</span><span> </span><span class="NLM_article-title">Phenotypic screening, take two</span> <span class="citation_source-journal">SciBX</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="refDoi"> DOI: 10.1038/scibx.2012.380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fscibx.2012.380" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2012&pages=15&author=J.+Kotz&title=Phenotypic+screening%2C+take+two&doi=10.1038%2Fscibx.2012.380"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3a&amp;dbid=16384&amp;doi=10.1038%2Fscibx.2012.380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fscibx.2012.380%26sid%3Dliteratum%253Aachs%26aulast%3DKotz%26aufirst%3DJ.%26atitle%3DPhenotypic%2520screening%252C%2520take%2520two%26jtitle%3DSciBX%26date%3D2012%26volume%3D5%26spage%3D15%26doi%3D10.1038%2Fscibx.2012.380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit3b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Tang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duggan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warder, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLoughlin, S. M.</span><span> </span><span class="NLM_article-title">Target identification of compounds from a cell viability phenotypic screen using a bead/lysate-based affinity capture platform</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">201</span><span class="NLM_x">–</span> <span class="NLM_lpage">211</span><span class="refDoi"> DOI: 10.1177/1087057115622431</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1177%2F1087057115622431" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=26676096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVSrtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=201-211&author=H.+Tangauthor=S.+Dugganauthor=P.+L.+Richardsonauthor=V.+Marinauthor=S.+E.+Warderauthor=S.+M.+McLoughlin&title=Target+identification+of+compounds+from+a+cell+viability+phenotypic+screen+using+a+bead%2Flysate-based+affinity+capture+platform&doi=10.1177%2F1087057115622431"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3bR"><div class="casContent"><span class="casTitleNuber">3b</span><div class="casTitle"><span class="NLM_cas:atitle">Target identification of compounds from a cell viability phenotypic screen using a bead/lysate-based affinity capture platform</span></div><div class="casAuthors">Tang, Hua; Duggan, Shannon; Richardson, Paul L.; Marin, Violeta; Warder, Scott E.; McLoughlin, Shaun M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">201-211</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The pharmaceutical industry has been continually challenged by dwindling target diversity.  To obviate this trend, phenotypic screens have been adopted, complementing target-centric screening approaches.  Phenotypic screens identify drug leads using clin. relevant and translatable mechanisms, remaining agnostic to targets.  While target anonymity is advantageous early in the drug discovery process, it poses challenges to hit progression, including the development of backup series, retaining desired pharmacol. during optimization, discovery of markers, and understanding mechanism-driven toxicity.  Consequently, significant effort has been expended to elaborate the targets and mechanisms at work for promising screening hits.  Affinity capture is commonly leveraged, where the compds. are linked to beads and targets are abstracted from cell homogenates.  This technique has proven effective for identifying targets of kinase, PARP, and HDAC inhibitors, and examples of new targets have been reported.  Herein, a three-pronged approach to target deconvolution by affinity capture is described, including the implementation of a uniqueness index that helps discriminate between bona fide targets and background.  The effectiveness of this approach is demonstrated using characterized compds. that act on known and noncanonical target classes.  The platform is subsequently applied to phenotypic screening hits, identifying candidate targets.  The success rate of bead-based affinity capture is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodk9IeGmf4ErVg90H21EOLACvtfcHk0lgvEN0LSN2_XA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVSrtbk%253D&md5=5951e68edfdfeda7e82d428e06ca018c</span></div><a href="/servlet/linkout?suffix=cit3b&amp;dbid=16384&amp;doi=10.1177%2F1087057115622431&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057115622431%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DH.%26aulast%3DDuggan%26aufirst%3DS.%26aulast%3DRichardson%26aufirst%3DP.%2BL.%26aulast%3DMarin%26aufirst%3DV.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DMcLoughlin%26aufirst%3DS.%2BM.%26atitle%3DTarget%2520identification%2520of%2520compounds%2520from%2520a%2520cell%2520viability%2520phenotypic%2520screen%2520using%2520a%2520bead%252Flysate-based%2520affinity%2520capture%2520platform%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2016%26volume%3D21%26spage%3D201%26epage%3D211%26doi%3D10.1177%2F1087057115622431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Moffat, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailey, D.</span><span> </span><span class="NLM_article-title">Phenotypic screening in cancer drug discovery - past, present and future</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">588</span><span class="NLM_x">–</span> <span class="NLM_lpage">602</span><span class="refDoi"> DOI: 10.1038/nrd4366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnrd4366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=25033736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=588-602&author=J.+G.+Moffatauthor=J.+Rudolphauthor=D.+Bailey&title=Phenotypic+screening+in+cancer+drug+discovery+-+past%2C+present+and+future&doi=10.1038%2Fnrd4366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4aR"><div class="casContent"><span class="casTitleNuber">4a</span><div class="casTitle"><span class="NLM_cas:atitle">Phenotypic screening in cancer drug discovery - past, present and future</span></div><div class="casAuthors">Moffat, John G.; Rudolph, Joachim; Bailey, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">588-602</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There has been a resurgence of interest in the use of phenotypic screens in drug discovery as an alternative to target-focused approaches.  Given that oncol. is currently the most active therapeutic area, and also one in which target-focused approaches have been particularly prominent in the past two decades, we investigated the contribution of phenotypic assays to oncol. drug discovery by analyzing the origins of all new small-mol. cancer drugs approved by the US Food and Drug Administration (FDA) over the past 15 years and those currently in clin. development.  Although the majority of these drugs originated from target-based discovery, we identified a significant no. whose discovery depended on phenotypic screening approaches.  We postulate that the contribution of phenotypic screening to cancer drug discovery has been hampered by a reliance on 'classical' nonspecific drug effects such as cytotoxicity and mitotic arrest, exacerbated by a paucity of mechanistically defined cellular models for therapeutically translatable cancer phenotypes.  However, tech. and biol. advances that enable such mechanistically informed phenotypic models have the potential to empower phenotypic drug discovery in oncol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTWtoTIgWqvbVg90H21EOLACvtfcHk0lgvEN0LSN2_XA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7L&md5=0e8efed90e5bc4b47f959b83a6ab5b96</span></div><a href="/servlet/linkout?suffix=cit4a&amp;dbid=16384&amp;doi=10.1038%2Fnrd4366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4366%26sid%3Dliteratum%253Aachs%26aulast%3DMoffat%26aufirst%3DJ.%2BG.%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DBailey%26aufirst%3DD.%26atitle%3DPhenotypic%2520screening%2520in%2520cancer%2520drug%2520discovery%2520-%2520past%252C%2520present%2520and%2520future%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D588%26epage%3D602%26doi%3D10.1038%2Fnrd4366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit4b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Vincent, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loria, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pregel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanton, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitching, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nocka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doyonnas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steppan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schroeter, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peakman, M.</span><span> </span><span class="NLM_article-title">Developing predictive assays: the phenotypic screening “rule of 3”</span> <span class="citation_source-journal">Sci. Transl. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">293ps15</span><span class="refDoi"> DOI: 10.1126/scitranslmed.aab1201</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1126%2Fscitranslmed.aab1201" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=26109101" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=293ps15&author=F.+Vincentauthor=P.+Loriaauthor=M.+Pregelauthor=R.+Stantonauthor=L.+Kitchingauthor=K.+Nockaauthor=R.+Doyonnasauthor=C.+Steppanauthor=A.+Gilbertauthor=T.+Schroeterauthor=M.+Peakman&title=Developing+predictive+assays%3A+the+phenotypic+screening+%E2%80%9Crule+of+3%E2%80%9D&doi=10.1126%2Fscitranslmed.aab1201"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4b&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aab1201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aab1201%26sid%3Dliteratum%253Aachs%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DLoria%26aufirst%3DP.%26aulast%3DPregel%26aufirst%3DM.%26aulast%3DStanton%26aufirst%3DR.%26aulast%3DKitching%26aufirst%3DL.%26aulast%3DNocka%26aufirst%3DK.%26aulast%3DDoyonnas%26aufirst%3DR.%26aulast%3DSteppan%26aufirst%3DC.%26aulast%3DGilbert%26aufirst%3DA.%26aulast%3DSchroeter%26aufirst%3DT.%26aulast%3DPeakman%26aufirst%3DM.%26atitle%3DDeveloping%2520predictive%2520assays%253A%2520the%2520phenotypic%2520screening%2520%25E2%2580%259Crule%2520of%25203%25E2%2580%259D%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2015%26volume%3D7%26spage%3D293ps15%26doi%3D10.1126%2Fscitranslmed.aab1201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit4c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Swinney, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anthony, J.</span><span> </span><span class="NLM_article-title">How were new medicines discovered?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">507</span><span class="NLM_x">–</span> <span class="NLM_lpage">519</span><span class="refDoi"> DOI: 10.1038/nrd3480</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnrd3480" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=21701501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlCmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=507-519&author=D.+C.+Swinneyauthor=J.+Anthony&title=How+were+new+medicines+discovered%3F&doi=10.1038%2Fnrd3480"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4cR"><div class="casContent"><span class="casTitleNuber">4c</span><div class="casTitle"><span class="NLM_cas:atitle">How were new medicines discovered?</span></div><div class="casAuthors">Swinney, David C.; Anthony, Jason</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">507-519</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Preclin. strategies that are used to identify potential drug candidates include target-based screening, phenotypic screening, modification of natural substances and biol.-based approaches.  To investigate whether some strategies have been more successful than others in the discovery of new drugs, we analyzed the discovery strategies and the mol. mechanism of action (MMOA) for new mol. entities and new biologics that were approved by the US Food and Drug Administration between 1999 and 2008.  Out of the 259 agents that were approved, 75 were first-in-class drugs with new MMOAs, and out of these, 50 (67%) were small mols. and 25 (33%) were biologics.  The results also show that the contribution of phenotypic screening to the discovery of first-in-class small-mol. drugs exceeded that of target-based approaches - with 28 and 17 of these drugs coming from the two approaches, resp. - in an era in which the major focus was on target-based approaches.  We postulate that a target-centric approach for first-in-class drugs, without consideration of an optimal MMOA, may contribute to the current high attrition rates and low productivity in pharmaceutical research and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE8Ah1iNz76bVg90H21EOLACvtfcHk0ljC7tw4-GbQ_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlCmsbs%253D&md5=7503620f8fbadb0a829f6ea2029936a2</span></div><a href="/servlet/linkout?suffix=cit4c&amp;dbid=16384&amp;doi=10.1038%2Fnrd3480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3480%26sid%3Dliteratum%253Aachs%26aulast%3DSwinney%26aufirst%3DD.%2BC.%26aulast%3DAnthony%26aufirst%3DJ.%26atitle%3DHow%2520were%2520new%2520medicines%2520discovered%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D507%26epage%3D519%26doi%3D10.1038%2Fnrd3480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit4d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Cheeseman, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faisal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rayter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbeau, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalusa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westlake, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Montfort, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linardopoulos, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, K.</span><span> </span><span class="NLM_article-title">Targeting the PPM1D phenotype; 2,4-bisarylthiazoles cause highly selective apoptosis in PPM1D amplified cell-lines</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">3469</span><span class="NLM_x">–</span> <span class="NLM_lpage">3474</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2014.05.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.bmcl.2014.05.067" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=3469-3474&author=M.+D.+Cheesemanauthor=A.+Faisalauthor=S.+Rayterauthor=O.+Barbeauauthor=A.+Kalusaauthor=M.+Westlakeauthor=R.+Burkeauthor=M.+Swanauthor=R.+van+Montfortauthor=S.+Linardopoulosauthor=K.+Jones&title=Targeting+the+PPM1D+phenotype%3B+2%2C4-bisarylthiazoles+cause+highly+selective+apoptosis+in+PPM1D+amplified+cell-lines&doi=10.1016%2Fj.bmcl.2014.05.067"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4d&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.05.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.05.067%26sid%3Dliteratum%253Aachs%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26aulast%3DFaisal%26aufirst%3DA.%26aulast%3DRayter%26aufirst%3DS.%26aulast%3DBarbeau%26aufirst%3DO.%26aulast%3DKalusa%26aufirst%3DA.%26aulast%3DWestlake%26aufirst%3DM.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DSwan%26aufirst%3DM.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%26aulast%3DLinardopoulos%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DK.%26atitle%3DTargeting%2520the%2520PPM1D%2520phenotype%253B%25202%252C4-bisarylthiazoles%2520cause%2520highly%2520selective%2520apoptosis%2520in%2520PPM1D%2520amplified%2520cell-lines%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D3469%26epage%3D3474%26doi%3D10.1016%2Fj.bmcl.2014.05.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Zheng, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorne, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKew, J. C.</span><span> </span><span class="NLM_article-title">Phenotypic screens as a renewed approach for drug discovery</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1067</span><span class="NLM_x">–</span> <span class="NLM_lpage">1073</span><span class="refDoi"> DOI: 10.1016/j.drudis.2013.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.drudis.2013.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=23850704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1emt7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=1067-1073&author=W.+Zhengauthor=N.+Thorneauthor=J.+C.+McKew&title=Phenotypic+screens+as+a+renewed+approach+for+drug+discovery&doi=10.1016%2Fj.drudis.2013.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5aR"><div class="casContent"><span class="casTitleNuber">5a</span><div class="casTitle"><span class="NLM_cas:atitle">Phenotypic screens as a renewed approach for drug discovery</span></div><div class="casAuthors">Zheng, Wei; Thorne, Natasha; McKew, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">21-22</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The significant redn. in the no. of newly approved drugs in the past decade has been partially attributed to failures in discovery and validation of new targets.  Evaluation of recently approved new drugs has revealed that the no. of approved drugs discovered through phenotypic screens, an original drug screening paradigm, has exceeded those discovered through the mol. target-based approach.  Phenotypic screening is thus gaining new momentum in drug discovery with the hope that this approach may revitalize drug discovery and improve the success rate of drug approval through the discovery of viable lead compds. and identification of novel drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_cPfNTmnrfbVg90H21EOLACvtfcHk0ljC7tw4-GbQ_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1emt7vO&md5=9d92e1f61043ac2dd11bfe5f32944cff</span></div><a href="/servlet/linkout?suffix=cit5a&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DThorne%26aufirst%3DN.%26aulast%3DMcKew%26aufirst%3DJ.%2BC.%26atitle%3DPhenotypic%2520screens%2520as%2520a%2520renewed%2520approach%2520for%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2013%26volume%3D18%26spage%3D1067%26epage%3D1073%26doi%3D10.1016%2Fj.drudis.2013.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit5b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Lee, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uhlik, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moxham, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomandl, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sall, D. J.</span><span> </span><span class="NLM_article-title">Modern phenotypic drug discovery is a viable, neoclassic pharma strategy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">4527</span><span class="NLM_x">–</span> <span class="NLM_lpage">4538</span><span class="refDoi"> DOI: 10.1021/jm201649s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201649s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1OitLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=4527-4538&author=J.+A.+Leeauthor=M.+T.+Uhlikauthor=C.+M.+Moxhamauthor=D.+Tomandlauthor=D.+J.+Sall&title=Modern+phenotypic+drug+discovery+is+a+viable%2C+neoclassic+pharma+strategy&doi=10.1021%2Fjm201649s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5bR"><div class="casContent"><span class="casTitleNuber">5b</span><div class="casTitle"><span class="NLM_cas:atitle">Modern Phenotypic Drug Discovery Is a Viable, Neoclassic Pharma Strategy</span></div><div class="casAuthors">Lee, Jonathan A.; Uhlik, Mark T.; Moxham, Christopher M.; Tomandl, Dirk; Sall, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4527-4538</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The pharmaceutical industry (Pharma) is currently facing unprecedented challenges.  In addn. to strategic patent expirations, the rate of drug launches has been essentially const. for 60 years with overall productivity falling since the 1970s.  Similarly, the appearance of novel targets to FDA approved drugs, a measure of industry innovation, has not dramatically improved since the 1980s.  Although Pharma productivity is a multifaceted problem, detailed analyses of comprehensive, industry-wide data indicate that late stage clin. failures are a major contributor that has been attributed to poor target validation (TV) and the lack of predictive biomarkers that translate to the clinic.  In the spirit of "reinventing innovation" Pharma needs to introspectively identify areas for improvement.  This communication considers how choices in drug screening strategies may relate to target validation issues and influence the probability of identifying novel medicines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8CcODiYYNgLVg90H21EOLACvtfcHk0ljC7tw4-GbQ_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1OitLs%253D&md5=1937641b632e348d7566c016ca4cf51d</span></div><a href="/servlet/linkout?suffix=cit5b&amp;dbid=16384&amp;doi=10.1021%2Fjm201649s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201649s%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%2BA.%26aulast%3DUhlik%26aufirst%3DM.%2BT.%26aulast%3DMoxham%26aufirst%3DC.%2BM.%26aulast%3DTomandl%26aufirst%3DD.%26aulast%3DSall%26aufirst%3DD.%2BJ.%26atitle%3DModern%2520phenotypic%2520drug%2520discovery%2520is%2520a%2520viable%252C%2520neoclassic%2520pharma%2520strategy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D4527%26epage%3D4538%26doi%3D10.1021%2Fjm201649s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit5c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Eder, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sedrani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiesmann, C.</span><span> </span><span class="NLM_article-title">The discovery of first-in-class drugs: origins and evolution</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">577</span><span class="NLM_x">–</span> <span class="NLM_lpage">587</span><span class="refDoi"> DOI: 10.1038/nrd4336</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnrd4336" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=25033734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=577-587&author=J.+Ederauthor=R.+Sedraniauthor=C.+Wiesmann&title=The+discovery+of+first-in-class+drugs%3A+origins+and+evolution&doi=10.1038%2Fnrd4336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5cR"><div class="casContent"><span class="casTitleNuber">5c</span><div class="casTitle"><span class="NLM_cas:atitle">The discovery of first-in-class drugs: origins and evolution</span></div><div class="casAuthors">Eder, Jorg; Sedrani, Richard; Wiesmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">577-587</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Anal. of the origins of new drugs approved by the US Food and Drug Administration (FDA) from 1999 to 2008 suggested that phenotypic screening strategies had been more productive than target-based approaches in the discovery of first-in-class small-mol. drugs.  However, given the relatively recent introduction of target-based approaches in the context of the long time frames of drug development, their full impact might not yet have become apparent.  Here, we present an anal. of the origins of all 113 first-in-class drugs approved by the FDA from 1999 to 2013, which shows that the majority (78) were discovered through target-based approaches (45 small-mol. drugs and 33 biologics).  In addn., of 33 drugs identified in the absence of a target hypothesis, 25 were found through a chemocentric approach in which compds. with known pharmacol. served as the starting point, with only eight coming from what we define here as phenotypic screening: testing a large no. of compds. in a target-agnostic assay that monitors phenotypic changes.  We also discuss the implications for drug discovery strategies, including viewing phenotypic screening as a novel discipline rather than as a neoclassical approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqthEelveRWN7Vg90H21EOLACvtfcHk0ljq7CLYPVY9rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7I&md5=b2061d96a71893a329b5b654acec5420</span></div><a href="/servlet/linkout?suffix=cit5c&amp;dbid=16384&amp;doi=10.1038%2Fnrd4336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4336%26sid%3Dliteratum%253Aachs%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DSedrani%26aufirst%3DR.%26aulast%3DWiesmann%26aufirst%3DC.%26atitle%3DThe%2520discovery%2520of%2520first-in-class%2520drugs%253A%2520origins%2520and%2520evolution%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D577%26epage%3D587%26doi%3D10.1038%2Fnrd4336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Anighoro, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rastelli, G.</span><span> </span><span class="NLM_article-title">Polypharmacology: challenges and opportunities in drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">7874</span><span class="NLM_x">–</span> <span class="NLM_lpage">7887</span><span class="refDoi"> DOI: 10.1021/jm5006463</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5006463" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7874-7887&author=A.+Anighoroauthor=J.+Bajorathauthor=G.+Rastelli&title=Polypharmacology%3A+challenges+and+opportunities+in+drug+discovery&doi=10.1021%2Fjm5006463"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6aR"><div class="casContent"><span class="casTitleNuber">6a</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology: Challenges and Opportunities in Drug Discovery</span></div><div class="casAuthors">Anighoro, Andrew; Bajorath, Jurgen; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7874-7887</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  At present, the legendary magic bullet, i.e., a drug with high potency and selectivity toward a specific biol. target, shares the spotlight with an emerging and alternative polypharmacol. approach.  Polypharmacol. suggests that more effective drugs can be developed by specifically modulating multiple targets.  It is generally thought that complex diseases such as cancer and central nervous system diseases may require complex therapeutic approaches.  In this respect, a drug that "hits" multiple sensitive nodes belonging to a network of interacting targets offers the potential for higher efficacy and may limit drawbacks generally arising from the use of a single-target drug or a combination of multiple drugs.  In this review, we will compare advantages and disadvantages of multitarget vs. combination therapies, discuss potential drug promiscuity arising from off-target effects, comment on drug repurposing, and introduce approaches to the computational design of multitarget drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFOTyEu80AZrVg90H21EOLACvtfcHk0ljq7CLYPVY9rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I&md5=5ab36c961a84462d90bc25130d90a281</span></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1021%2Fjm5006463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5006463%26sid%3Dliteratum%253Aachs%26aulast%3DAnighoro%26aufirst%3DA.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DPolypharmacology%253A%2520challenges%2520and%2520opportunities%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D7874%26epage%3D7887%26doi%3D10.1021%2Fjm5006463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit6b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Peters, J.</span><span> </span><span class="NLM_article-title">Polypharmacology – foe or friend?</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">8955</span><span class="NLM_x">–</span> <span class="NLM_lpage">8971</span><span class="refDoi"> DOI: 10.1021/jm400856t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400856t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1amtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8955-8971&author=J.+Peters&title=Polypharmacology+%E2%80%93+foe+or+friend%3F&doi=10.1021%2Fjm400856t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6bR"><div class="casContent"><span class="casTitleNuber">6b</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology - Foe or Friend?</span></div><div class="casAuthors">Peters, Jens-Uwe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8955-8971</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Polypharmacol. describes the activity of compds. at multiple targets.  Current research focuses on two aspects of polypharmacol.: (1) unintended polypharmacol. can lead to adverse effects; (2) polypharmacol. across several disease-relevant targets can improve therapeutic efficacy, prevent drug resistance, or reduce therapeutic-target-related adverse effects.  This perspective reviews these interconnected aspects of polypharmacol.  The first part discusses the relevance of polypharmacol. for the safety of drugs, the mitigation of safety risks, and methods to identify polypharmacol. compds. early in the drug discovery process.  The second part discusses the advantages of polypharmacol. in the treatment of multigenic diseases and infections, and opportunities for drug discovery and drug repurposing.  This perspective aims to provide a balanced view on polypharmacol., which can compromise the safety of drugs, but can also confer superior efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5vnFGPfTnGLVg90H21EOLACvtfcHk0ljq7CLYPVY9rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1amtrnO&md5=f4aeb6efddd4bfdf4e94656303323cba</span></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1021%2Fjm400856t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400856t%26sid%3Dliteratum%253Aachs%26aulast%3DPeters%26aufirst%3DJ.%26atitle%3DPolypharmacology%2520%25E2%2580%2593%2520foe%2520or%2520friend%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8955%26epage%3D8971%26doi%3D10.1021%2Fjm400856t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit6c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Medina-Franco, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giulianotti, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Welmaker, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houghten, R. A.</span><span> </span><span class="NLM_article-title">Shifting from the single to the multitarget paradigm in drug discovery</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">495</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1016/j.drudis.2013.01.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.drudis.2013.01.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=23340113" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A280%3ADC%252BC3szhsV2rsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=495-501&author=J.+L.+Medina-Francoauthor=M.+A.+Giulianottiauthor=G.+S.+Welmakerauthor=R.+A.+Houghten&title=Shifting+from+the+single+to+the+multitarget+paradigm+in+drug+discovery&doi=10.1016%2Fj.drudis.2013.01.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6cR"><div class="casContent"><span class="casTitleNuber">6c</span><div class="casTitle"><span class="NLM_cas:atitle">Shifting from the single to the multitarget paradigm in drug discovery</span></div><div class="casAuthors">Medina-Franco Jose L; Giulianotti Marc A; Welmaker Gregory S; Houghten Richard A</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9-10</span>),
    <span class="NLM_cas:pages">495-501</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Increasing evidence that several drug compounds exert their effects through interactions with multiple targets is boosting the development of research fields that challenge the data reductionism approach.  In this article, we review and discuss the concepts of drug repurposing, polypharmacology, chemogenomics, phenotypic screening and high-throughput in vivo testing of mixture-based libraries in an integrated manner.  These research fields offer alternatives to the current paradigm of drug discovery, from a one target-one drug model to a multiple-target approach.  Furthermore, the goals of lead identification are being expanded accordingly to identify not only 'key' compounds that fit with a single-target 'lock', but also 'master key' compounds that favorably interact with multiple targets (i.e. operate a set of desired locks to gain access to the expected clinical effects).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTOFtVP-kWcL7qxQ2mb9Pu8fW6udTcc2eZqhzaBqHh0ebntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3szhsV2rsQ%253D%253D&md5=b4f8c7d1ebbbf32ed4c6c87500baf66f</span></div><a href="/servlet/linkout?suffix=cit6c&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.01.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.01.008%26sid%3Dliteratum%253Aachs%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26aulast%3DGiulianotti%26aufirst%3DM.%2BA.%26aulast%3DWelmaker%26aufirst%3DG.%2BS.%26aulast%3DHoughten%26aufirst%3DR.%2BA.%26atitle%3DShifting%2520from%2520the%2520single%2520to%2520the%2520multitarget%2520paradigm%2520in%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2013%26volume%3D18%26spage%3D495%26epage%3D501%26doi%3D10.1016%2Fj.drudis.2013.01.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit6d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Jalencas, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mestres, J.</span><span> </span><span class="NLM_article-title">On the origins of drug polypharmacology</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">80</span><span class="NLM_x">–</span> <span class="NLM_lpage">87</span><span class="refDoi"> DOI: 10.1039/C2MD20242E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1039%2FC2MD20242E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVynu7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=80-87&author=X.+Jalencasauthor=J.+Mestres&title=On+the+origins+of+drug+polypharmacology&doi=10.1039%2FC2MD20242E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6dR"><div class="casContent"><span class="casTitleNuber">6d</span><div class="casTitle"><span class="NLM_cas:atitle">On the origins of drug polypharmacology</span></div><div class="casAuthors">Jalencas, Xavier; Mestres, Jordi</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">80-87</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The ability of small mols. to interact with multiple proteins is commonly referred to as polypharmacol.  The now widely accepted polypharmacol. of drugs is of particular interest for human health as it has implications beyond therapeutic efficacy, from anticipating adverse drug reactions to identifying potential repurposing opportunities.  There have been a no. of studies relating the extent of drug polypharmacol. to the physicochem. properties and fragment compn. of the drug itself, but also to the protein family and distant binding site similarities of the drug's primary target.  Taken together, all these observations lead to speculate that the origins of drug polypharmacol. may lie at the heart of protein evolution and that polypharmacol. may just be a reminiscent signature of some of the mechanisms of adaptation that primitive biol. systems developed to increase the chances of survival in a highly variable early chem. environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB1qsCiSthHLVg90H21EOLACvtfcHk0linw-f8EpdRiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVynu7vI&md5=77fe84731536e5008cca3b0faf1e58dd</span></div><a href="/servlet/linkout?suffix=cit6d&amp;dbid=16384&amp;doi=10.1039%2FC2MD20242E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2MD20242E%26sid%3Dliteratum%253Aachs%26aulast%3DJalencas%26aufirst%3DX.%26aulast%3DMestres%26aufirst%3DJ.%26atitle%3DOn%2520the%2520origins%2520of%2520drug%2520polypharmacology%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D80%26epage%3D87%26doi%3D10.1039%2FC2MD20242E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Swinney, D. C.</span><span> </span><span class="NLM_article-title">The value of translational biomarkers to phenotypic assays</span> <span class="citation_source-journal">Front. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">171</span><span class="refDoi"> DOI: 10.3389/fphar.2014.00171</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.3389%2Ffphar.2014.00171" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=25076910" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A280%3ADC%252BC2cbns1aguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=171&author=D.+C.+Swinney&title=The+value+of+translational+biomarkers+to+phenotypic+assays&doi=10.3389%2Ffphar.2014.00171"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The value of translational biomarkers to phenotypic assays</span></div><div class="casAuthors">Swinney David C</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">171</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Phenotypic assays are tools essential for drug discovery.  Phenotypic assays have different types of endpoints depending on the goals; (1) empirical endpoints for basic research to understand the underlying biology that will lead to identification of translation biomarkers, (2) empirical endpoints to identify undesired effects related to toxicity of drug candidates, and (3) knowledge-based endpoints (biomarkers) for drug discovery which ideally are translational biomarkers that will be used to identify new drug candidates and their corresponding molecular mechanisms of action.  The value of phenotypic assays is increased through effective alignment of phenotypic assay endpoints with the objectives of the relevant stage in the drug discovery and development cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSWdoDxmkrm475XHngOXiV_fW6udTcc2eZqhzaBqHh0ebntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cbns1aguw%253D%253D&md5=5de6ca2ed245be9bfdb9f082c2d41adc</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2014.00171&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2014.00171%26sid%3Dliteratum%253Aachs%26aulast%3DSwinney%26aufirst%3DD.%2BC.%26atitle%3DThe%2520value%2520of%2520translational%2520biomarkers%2520to%2520phenotypic%2520assays%26jtitle%3DFront.%2520Pharmacol.%26date%3D2014%26volume%3D5%26spage%3D171%26doi%3D10.3389%2Ffphar.2014.00171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guan, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mais, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span> </span><span class="NLM_article-title">Quality control of cell-based high-throughput drug screening</span> <span class="citation_source-journal">Acta Pharm. Sin. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">429</span><span class="NLM_x">–</span> <span class="NLM_lpage">438</span><span class="refDoi"> DOI: 10.1016/j.apsb.2012.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.apsb.2012.03.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12nt7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=429-438&author=Z.+Zhangauthor=N.+Guanauthor=T.+Liauthor=D.+E.+Maisauthor=M.+Wang&title=Quality+control+of+cell-based+high-throughput+drug+screening&doi=10.1016%2Fj.apsb.2012.03.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Quality control of cell-based high-throughput drug screening</span></div><div class="casAuthors">Zhang, Zhiyun; Guan, Ni; Li, Ting; Mais, Dale E.; Wang, Mingwei</div><div class="citationInfo"><span class="NLM_cas:title">Acta Pharmaceutica Sinica B</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">429-438</span>CODEN:
                <span class="NLM_cas:coden">APSBCW</span>;
        ISSN:<span class="NLM_cas:issn">2211-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The pharmaceutical industry is presently suffering difficult times due to low productivity of new mol. entities.  As a major source of drug leads, high-throughput screening (HTS) has been often criticized for its 'dead end' lead compds.  However, the fruitful achievements resulting from HTS technol. indicate that it remains a feasible way for drug innovation.  Because of increasing considerations of earlier stage ADMET (absorption, distribution, metab., excretion and toxicity) in drug development, cell-based HTS is highly recommended in modern drug discovery for its ability to detect more biol. relevant characteristics of compds. in living systems.  This review provides a systematic and practical description of vital points for conducting high quality cell-based HTS, from assay development to optimization, compd. management, data analyses, hit validation as well as lead identification.  Potential problems and solns. are also covered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkiaPQ7EjY6rVg90H21EOLACvtfcHk0linw-f8EpdRiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12nt7fP&md5=b4b20c6edb7047d27bd0f7e68b92e992</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2012.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2012.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DGuan%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DMais%26aufirst%3DD.%2BE.%26aulast%3DWang%26aufirst%3DM.%26atitle%3DQuality%2520control%2520of%2520cell-based%2520high-throughput%2520drug%2520screening%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2012%26volume%3D2%26spage%3D429%26epage%3D438%26doi%3D10.1016%2Fj.apsb.2012.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blagg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buser-Doepner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederiksen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimshaw, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hepworth, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotz, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mader, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller-Fahrnow, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Hagan, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundström, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toledo-Sherman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walpole, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walters, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willson, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuercher, W. J.</span><span> </span><span class="NLM_article-title">The promise and peril of chemical probes</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">536</span><span class="NLM_x">–</span> <span class="NLM_lpage">541</span><span class="refDoi"> DOI: 10.1038/nchembio.1867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnchembio.1867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=26196764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=536-541&author=C.+H.+Arrowsmithauthor=J.+E.+Audiaauthor=C.+Austinauthor=J.+Baellauthor=J.+Bennettauthor=J.+Blaggauthor=C.+Bountraauthor=P.+E.+Brennanauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=C.+Buser-Doepnerauthor=R.+M.+Campbellauthor=A.+J.+Carterauthor=P.+Cohenauthor=R.+A.+Copelandauthor=B.+Cravattauthor=J.+L.+Dahlinauthor=D.+Dhanakauthor=A.+M.+Edwardsauthor=M.+Frederiksenauthor=S.+V.+Fryeauthor=N.+Grayauthor=C.+E.+Grimshawauthor=D.+Hepworthauthor=T.+Howeauthor=K.+V.+Huberauthor=J.+Jinauthor=S.+Knappauthor=J.+D.+Kotzauthor=R.+G.+Krugerauthor=D.+Loweauthor=M.+M.+Maderauthor=B.+Marsdenauthor=A.+Mueller-Fahrnowauthor=S.+M%C3%BCllerauthor=R.+C.+O%E2%80%99Haganauthor=J.+P.+Overingtonauthor=D.+R.+Owenauthor=S.+H.+Rosenbergauthor=B.+Rothauthor=R.+Rossauthor=M.+Schapiraauthor=S.+L.+Schreiberauthor=B.+Shoichetauthor=M.+Sundstr%C3%B6mauthor=G.+Superti-Furgaauthor=J.+Tauntonauthor=L.+Toledo-Shermanauthor=C.+Walpoleauthor=M.+A.+Waltersauthor=T.+M.+Willsonauthor=P.+Workmanauthor=R.+N.+Youngauthor=W.+J.+Zuercher&title=The+promise+and+peril+of+chemical+probes&doi=10.1038%2Fnchembio.1867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9aR"><div class="casContent"><span class="casTitleNuber">9a</span><div class="casTitle"><span class="NLM_cas:atitle">The promise and peril of chemical probes</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Audia, James E.; Austin, Christopher; Baell, Jonathan; Bennett, Jonathan; Blagg, Julian; Bountra, Chas; Brennan, Paul E.; Brown, Peter J.; Bunnage, Mark E.; Buser-Doepner, Carolyn; Campbell, Robert M.; Carter, Adrian J.; Cohen, Philip; Copeland, Robert A.; Cravatt, Ben; Dahlin, Jayme L.; Dhanak, Dashyant; Edwards, Aled M.; Frye, Stephen V.; Gray, Nathanael; Grimshaw, Charles E.; Hepworth, David; Howe, Trevor; Huber, Kilian V. M.; Jin, Jian; Knapp, Stefan; Kotz, Joanne D.; Kruger, Ryan G.; Lowe, Derek; Mader, Mary M.; Marsden, Brian; Mueller-Fahrnow, Anke; Muller, Susanne; O'Hagan, Ronan C.; Overington, John P.; Owen, Dafydd R.; Rosenberg, Saul H.; Roth, Brian; Ross, Ruth; Schapira, Matthieu; Schreiber, Stuart L.; Shoichet, Brian; Sundstrom, Michael; Superti-Furga, Giulio; Taunton, Jack; Toledo-Sherman, Leticia; Walpole, Chris; Walters, Michael A.; Willson, Timothy M.; Workman, Paul; Young, Robert N.; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are powerful reagents with increasing impacts on biomedical research.  However, probes of poor quality or that are used incorrectly generate misleading results.  To help address these shortcomings, we will create a community-driven wiki resource to improve quality and convey current best practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-li4dxq487rVg90H21EOLACvtfcHk0liyAfb8tbY9-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO&md5=d8c7a7af94921ab4cdcc871487cceadd</span></div><a href="/servlet/linkout?suffix=cit9a&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1867%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DK.%2BV.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Hagan%26aufirst%3DR.%2BC.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DRoth%26aufirst%3DB.%26aulast%3DRoss%26aufirst%3DR.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DSundstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DWalpole%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BN.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DThe%2520promise%2520and%2520peril%2520of%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnchembio.1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit9b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, I.</span><span> </span><span class="NLM_article-title">Probing the probes: fitness factors for small molecule tools</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">561</span><span class="NLM_x">–</span> <span class="NLM_lpage">577</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2010.05.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.chembiol.2010.05.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=20609406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVWqs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=561-577&author=P.+Workmanauthor=I.+Collins&title=Probing+the+probes%3A+fitness+factors+for+small+molecule+tools&doi=10.1016%2Fj.chembiol.2010.05.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9bR"><div class="casContent"><span class="casTitleNuber">9b</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the Probes: Fitness Factors For Small Molecule Tools</span></div><div class="casAuthors">Workman, Paul; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">561-577</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Chem. probes for interrogating biol. processes are of considerable current interest.  Cell permeable small mol. tools have a major role in facilitating the functional annotation of the human genome, understanding both physiol. and pathol. processes, and validating new mol. targets.  To be valuable, chem. tools must satisfy necessary criteria and recent publications have suggested objective guidelines for what makes a useful chem. probe.  Although recognizing that such guidelines may be valuable, we caution against overly restrictive rules that may stifle innovation in favor of a "fit-for-purpose" approach.  Reviewing the literature and providing examples from the cancer field, we recommend a series of "fitness factors" to be considered when assessing chem. probes.  We hope this will encourage innovative chem. biol. research while minimizing the generation of poor quality and misleading biol. data, thus increasing understanding of the particular biol. area, to the benefit of basic research and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQtty9oZD17rVg90H21EOLACvtfcHk0liyAfb8tbY9-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVWqs78%253D&md5=16be01da2339173aae8207030bf83c12</span></div><a href="/servlet/linkout?suffix=cit9b&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.05.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.05.013%26sid%3Dliteratum%253Aachs%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DProbing%2520the%2520probes%253A%2520fitness%2520factors%2520for%2520small%2520molecule%2520tools%26jtitle%3DChem.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D561%26epage%3D577%26doi%3D10.1016%2Fj.chembiol.2010.05.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit9c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piantnitski Chekler, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, L. H.</span><span> </span><span class="NLM_article-title">Target validation using chemical probes</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">195</span><span class="NLM_x">–</span> <span class="NLM_lpage">199</span><span class="refDoi"> DOI: 10.1038/nchembio.1197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnchembio.1197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=23508172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=195-199&author=M.+E.+Bunnageauthor=E.+L.+Piantnitski+Cheklerauthor=L.+H.+Jones&title=Target+validation+using+chemical+probes&doi=10.1038%2Fnchembio.1197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9cR"><div class="casContent"><span class="casTitleNuber">9c</span><div class="casTitle"><span class="NLM_cas:atitle">Target validation using chemical probes</span></div><div class="casAuthors">Bunnage, Mark E.; Chekler, Eugene L. Piatnitski; Jones, Lyn H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">195-199</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review on the fully profiled chem. probes essential to support the unbiased interpretation of biol. expts. necessary for rigorous preclin. target validation.  We believe that by developing a 'chem. probe tool kit', and a framework for its use, chem. biol. can have a more central role in identifying targets of potential relevance to disease, avoiding many of the biases that complicate target validation as practiced currently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLwRTT4pXGxbVg90H21EOLACvtfcHk0liSCEz5czjekQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D&md5=af0c362882191de96a8e8698ff2c9ce8</span></div><a href="/servlet/linkout?suffix=cit9c&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1197%26sid%3Dliteratum%253Aachs%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DPiantnitski%2BChekler%26aufirst%3DE.%2BL.%26aulast%3DJones%26aufirst%3DL.%2BH.%26atitle%3DTarget%2520validation%2520using%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D195%26epage%3D199%26doi%3D10.1038%2Fnchembio.1197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Ziegler, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pries, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hedberg, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span> </span><span class="NLM_article-title">Target identification for small bioactive molecules: finding the needle in the haystack</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2744</span><span class="NLM_x">–</span> <span class="NLM_lpage">2792</span><span class="refDoi"> DOI: 10.1002/anie.201208749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1002%2Fanie.201208749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=23418026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXislSksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=2744-2792&author=S.+Zieglerauthor=V.+Priesauthor=C.+Hedbergauthor=H.+Waldmann&title=Target+identification+for+small+bioactive+molecules%3A+finding+the+needle+in+the+haystack&doi=10.1002%2Fanie.201208749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10aR"><div class="casContent"><span class="casTitleNuber">10a</span><div class="casTitle"><span class="NLM_cas:atitle">Target Identification for Small Bioactive Molecules: Finding the Needle in the Haystack</span></div><div class="casAuthors">Ziegler, Slava; Pries, Verena; Hedberg, Christian; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2744-2792</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Identification and confirmation of bioactive small-mol. targets is a crucial, often decisive step both in academic and pharmaceutical research.  Through the development and availability of several new exptl. techniques, target identification is, in principle, feasible, and the no. of successful examples steadily grows.  However, a generic methodol. that can successfully be applied in the majority of the cases has not yet been established.  Herein we summarize current methods for target identification of small mols., primarily for a chem. audience but also the biol. community, for example, the chemist or biologist attempting to identify the target of a given bioactive compd.  We describe the most frequently employed exptl. approaches for target identification and provide several representative examples illustrating the state-of-the-art.  Among the techniques currently available, protein affinity isolation using suitable small-mol. probes (pulldown) and subsequent mass spectrometric anal. of the isolated proteins appears to be most powerful and most frequently applied.  To provide guidance for rapid entry into the field and based on our own experience we propose a typical workflow for target identification, which centers on the application of chem. proteomics as the key step to generate hypotheses for potential target proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYC9-7L3abMrVg90H21EOLACvtfcHk0liSCEz5czjekQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXislSksLk%253D&md5=3a124205e3a57a53f479b889073bda76</span></div><a href="/servlet/linkout?suffix=cit10a&amp;dbid=16384&amp;doi=10.1002%2Fanie.201208749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201208749%26sid%3Dliteratum%253Aachs%26aulast%3DZiegler%26aufirst%3DS.%26aulast%3DPries%26aufirst%3DV.%26aulast%3DHedberg%26aufirst%3DC.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DTarget%2520identification%2520for%2520small%2520bioactive%2520molecules%253A%2520finding%2520the%2520needle%2520in%2520the%2520haystack%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D2744%26epage%3D2792%26doi%3D10.1002%2Fanie.201208749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit10b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Rix, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span> </span><span class="NLM_article-title">Target profiling of small molecules by chemical proteomics</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">616</span><span class="NLM_x">–</span> <span class="NLM_lpage">624</span><span class="refDoi"> DOI: 10.1038/nchembio.216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnchembio.216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=19690537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVWhsb7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=616-624&author=U.+Rixauthor=G.+Superti-Furga&title=Target+profiling+of+small+molecules+by+chemical+proteomics&doi=10.1038%2Fnchembio.216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10bR"><div class="casContent"><span class="casTitleNuber">10b</span><div class="casTitle"><span class="NLM_cas:atitle">Target profiling of small molecules by chemical proteomics</span></div><div class="casAuthors">Rix, Uwe; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">616-624</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The medical and pharmaceutical communities are facing a dire need for new druggable targets, while, paradoxically, the targets of some drugs that are in clin. use or development remain elusive.  Many compds. have been more promiscuous than originally anticipated, which can potentially lead to side effects, but which may also open up addnl. medical uses.  As we move toward systems biol. and personalized medicine, comprehensively detg. small mol.-target interaction profiles and mapping these on signaling and metabolic pathways will become increasingly necessary.  Chem. proteomics is a powerful mass spectrometry-based affinity chromatog. approach for identifying proteome-wide small mol.-protein interactions.  Here we will provide a crit. overview of the basic concepts and recent advances in chem. proteomics and review recent applications, with a particular emphasis on kinase inhibitors and natural products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBJz9OMT5Rp7Vg90H21EOLACvtfcHk0liSCEz5czjekQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVWhsb7K&md5=b357080902ea5e6cd6ab51792f327d0b</span></div><a href="/servlet/linkout?suffix=cit10b&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.216%26sid%3Dliteratum%253Aachs%26aulast%3DRix%26aufirst%3DU.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DTarget%2520profiling%2520of%2520small%2520molecules%2520by%2520chemical%2520proteomics%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D616%26epage%3D624%26doi%3D10.1038%2Fnchembio.216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Schirle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span> </span><span class="NLM_article-title">Identifying compound efficacy targets in phenotypic drug discovery</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">82</span><span class="NLM_x">–</span> <span class="NLM_lpage">89</span><span class="refDoi"> DOI: 10.1016/j.drudis.2015.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.drudis.2015.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=26272035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCisrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=82-89&author=M.+Schirleauthor=J.+L.+Jenkins&title=Identifying+compound+efficacy+targets+in+phenotypic+drug+discovery&doi=10.1016%2Fj.drudis.2015.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying compound efficacy targets in phenotypic drug discovery</span></div><div class="casAuthors">Schirle, Markus; Jenkins, Jeremy L.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">82-89</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The identification of the efficacy target(s) for hits from phenotypic compd. screens remains a key step to progress compds. into drug development.  In addn. to efficacy targets, the characterization of epistatic proteins influencing compd. activity often facilitates the elucidation of the underlying mechanism of action; and, further, early detn. of off-targets that cause potentially unwanted secondary phenotypes helps in assessing potential liabilities.  This short review discusses the most important technologies currently available for characterizing the direct and indirect target space of bioactive compds. following phenotypic screening.  We present a comprehensive strategy employing complementary approaches to balance individual technol. strengths and weaknesses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNnxrH8t-we7Vg90H21EOLACvtfcHk0liSCEz5czjekQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCisrvF&md5=b373d239d72058a1b826ae1550bb4693</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26atitle%3DIdentifying%2520compound%2520efficacy%2520targets%2520in%2520phenotypic%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D82%26epage%3D89%26doi%3D10.1016%2Fj.drudis.2015.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Hann, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keseru, G. M.</span><span> </span><span class="NLM_article-title">Finding the sweet spot: the role of nature and nurture in medicinal chemistry</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">355</span><span class="NLM_x">–</span> <span class="NLM_lpage">365</span><span class="refDoi"> DOI: 10.1038/nrd3701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnrd3701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=22543468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt1GqsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=355-365&author=M.+M.+Hannauthor=G.+M.+Keseru&title=Finding+the+sweet+spot%3A+the+role+of+nature+and+nurture+in+medicinal+chemistry&doi=10.1038%2Fnrd3701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Finding the sweet spot: the role of nature and nurture in medicinal chemistry</span></div><div class="casAuthors">Hann, Michael M.; Keserue, Gyoergy M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">355-365</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Given its position at the heart of small-mol. drug discovery, medicinal chem. has an important role in tackling the well-known productivity challenges in pharmaceutical research and development.  In recent years, extensive analyses of successful and failed discovery compds. and drug candidates have improved our understanding of the role of physicochem. properties in drug attrition.  Based on the clarified challenges in finding the 'sweet spot' in medicinal chem. programs, we suggest that this goal can be achieved through a combination of first identifying chem. starting points with appropriate 'nature' and then rigorously 'nurturing' them during lead optimization.  Here, we discuss scientific, strategic, organizational and cultural considerations for medicinal chem. practices, with the aim of promoting more effective use of what is already known, as well as a wider appreciation of the risks of pursuing suboptimal compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrURciGjB2A3LVg90H21EOLACvtfcHk0ljQ5mTCpR3tbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt1GqsLY%253D&md5=90170e3f4e69e52bfa38667df380516a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrd3701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3701%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26atitle%3DFinding%2520the%2520sweet%2520spot%253A%2520the%2520role%2520of%2520nature%2520and%2520nurture%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D355%26epage%3D365%26doi%3D10.1038%2Fnrd3701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Vihervaara, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sistonen, L.</span><span> </span><span class="NLM_article-title">HSF1 at a glance</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">261</span><span class="NLM_x">–</span> <span class="NLM_lpage">266</span><span class="refDoi"> DOI: 10.1242/jcs.132605</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1242%2Fjcs.132605" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=24421309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivFSgtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2014&pages=261-266&author=A.+Vihervaaraauthor=L.+Sistonen&title=HSF1+at+a+glance&doi=10.1242%2Fjcs.132605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13aR"><div class="casContent"><span class="casTitleNuber">13a</span><div class="casTitle"><span class="NLM_cas:atitle">HSF1 at a glance</span></div><div class="casAuthors">Vihervaara, Anniina; Sistonen, Lea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">261-266</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">A review.  Heat-shock factor 1 (HSF1) is an evolutionarily highly conserved transcription factor that coordinates stress-induced transcription and directs versatile physiol. processes in eukaryotes.  The central position of HSF1 in cellular homeostasis has been well demonstrated, mainly through its strong effect in transactivating genes that encode heat-shock proteins (HSPs).  However, recent genome-wide studies have revealed that HSF1 is capable of reprogramming transcription more extensively than previously assumed; it is also involved in a multitude of processes in stressed and non-stressed cells.  Consequently, the importance of HSF1 in fundamental physiol. events, including metab., gametogenesis and aging, has become apparent and its significance in pathologies, such as cancer progression, is now evident.  Here, the authors highlight recent advances in the HSF1 field, discuss the organismal control over HSF1, and present the processes that are mediated by HSF1 in the context of cell type, cell-cycle phase, physiol. condition and received stimuli.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopQBde2HgDRbVg90H21EOLACvtfcHk0ljQ5mTCpR3tbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXivFSgtro%253D&md5=0e818f15568b3027c7bc4bc6dbec1d31</span></div><a href="/servlet/linkout?suffix=cit13a&amp;dbid=16384&amp;doi=10.1242%2Fjcs.132605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.132605%26sid%3Dliteratum%253Aachs%26aulast%3DVihervaara%26aufirst%3DA.%26aulast%3DSistonen%26aufirst%3DL.%26atitle%3DHSF1%2520at%2520a%2520glance%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2014%26volume%3D127%26spage%3D261%26epage%3D266%26doi%3D10.1242%2Fjcs.132605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit13b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Anckar, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sistonen, L.</span><span> </span><span class="NLM_article-title">Regulation of HSF1 function in the heat stress response: implications in aging and disease</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">1089</span><span class="NLM_x">–</span> <span class="NLM_lpage">1115</span><span class="refDoi"> DOI: 10.1146/annurev-biochem-060809-095203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1146%2Fannurev-biochem-060809-095203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=21417720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVCntrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2011&pages=1089-1115&author=J.+Anckarauthor=L.+Sistonen&title=Regulation+of+HSF1+function+in+the+heat+stress+response%3A+implications+in+aging+and+disease&doi=10.1146%2Fannurev-biochem-060809-095203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13bR"><div class="casContent"><span class="casTitleNuber">13b</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of HSF1 function in the heat stress response: implications in aging and disease</span></div><div class="casAuthors">Anckar, Julius; Sistonen, Lea</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1089-1115</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  To dampen proteotoxic stresses and maintain protein homeostasis, organisms possess a stress-responsive mol. machinery that detects and neutralizes protein damage.  A prominent feature of stressed cells is the increased synthesis of heat shock proteins (Hsps) that aid in the refolding of misfolded peptides and restrain protein aggregation.  Transcriptional activation of the heat shock response is orchestrated by heat shock factor 1 (HSF1), which rapidly translocates to hsp genes and induces their expression.  Although the role of HSF1 in protecting cells and organisms against severe stress insults is well established, many aspects of how HSF1 senses qual. and quant. different forms of stresses have remained poorly understood.  Moreover, recent discoveries that HSF1 controls life span have prompted new ways of thinking about an old transcription factor.  Here, we review the established role of HSF1 in counteracting cell stress and prospect the role of HSF1 as a regulator of disease states and aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraZhoheJ2_J7Vg90H21EOLACvtfcHk0ljQ5mTCpR3tbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVCntrc%253D&md5=c8d529a0d535211cfbe9281c80202c3a</span></div><a href="/servlet/linkout?suffix=cit13b&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-060809-095203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-060809-095203%26sid%3Dliteratum%253Aachs%26aulast%3DAnckar%26aufirst%3DJ.%26aulast%3DSistonen%26aufirst%3DL.%26atitle%3DRegulation%2520of%2520HSF1%2520function%2520in%2520the%2520heat%2520stress%2520response%253A%2520implications%2520in%2520aging%2520and%2520disease%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2011%26volume%3D80%26spage%3D1089%26epage%3D1115%26doi%3D10.1146%2Fannurev-biochem-060809-095203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit13c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Jiang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmitt, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span> </span><span class="NLM_article-title">Multifaceted roles of HSF1 in cancer</span> <span class="citation_source-journal">Tumor Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">4923</span><span class="NLM_x">–</span> <span class="NLM_lpage">4931</span><span class="refDoi"> DOI: 10.1007/s13277-015-3674-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1007%2Fs13277-015-3674-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=26108999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtV2lsbfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=4923-4931&author=S.+Jiangauthor=K.+Tuauthor=Q.+Fuauthor=D.+C.+Schmittauthor=L.+Zhouauthor=N.+Luauthor=Y.+Zhao&title=Multifaceted+roles+of+HSF1+in+cancer&doi=10.1007%2Fs13277-015-3674-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13cR"><div class="casContent"><span class="casTitleNuber">13c</span><div class="casTitle"><span class="NLM_cas:atitle">Multifaceted roles of HSF1 in cancer</span></div><div class="casAuthors">Jiang, Sufang; Tu, Kailing; Fu, Qiang; Schmitt, David C.; Zhou, Lan; Lu, Na; Zhao, Yuhua</div><div class="citationInfo"><span class="NLM_cas:title">Tumor Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4923-4931</span>CODEN:
                <span class="NLM_cas:coden">TUMBEA</span>;
        ISSN:<span class="NLM_cas:issn">1010-4283</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Heat shock transcription factor 1 (HSF1) is the master regulator of the heat shock response.  Accumulating evidence shows that HSF1 is overexpressed in a variety of human cancers, is assocd. with cancer aggressiveness, and could serve as an independent diagnostic or prognostic biomarker.  In this review, we will provide an overview of the multifaceted roles of HSF1 in cancer, with a special focus on the four underlying mol. mechanisms involved.  First, HSF1 regulates the expression of heat shock proteins (HSPs) including HSP90, HSP70, and HSP27.  Second, HSF1 regulates cellular metab., including glycolysis and lipid metab.  Third, HSF1 serves as a regulator of different signaling pathways, such as HuR-HIF-1, Slug, protein kinase C (PKC), nuclear factor-kappaB (NF-κB), PI3K-AKT-mTOR, and mitogen-activated protein kinase (MAPK) pathways.  Finally, HSF1 regulates microRNAs (miRNAs) and long non-coding RNAs (lncRNAs).  Overall, HSF1 plays many important roles in cancer via regulating cell proliferation, anti-apoptosis, epithelial-mesenchymal transition (EMT), migration, invasion, and metastasis and may be a potential therapeutic target for human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ3CTGJiNLPrVg90H21EOLACvtfcHk0ljQ5mTCpR3tbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtV2lsbfO&md5=05d25d9d33dfb74351bc073d678640ee</span></div><a href="/servlet/linkout?suffix=cit13c&amp;dbid=16384&amp;doi=10.1007%2Fs13277-015-3674-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-015-3674-x%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DTu%26aufirst%3DK.%26aulast%3DFu%26aufirst%3DQ.%26aulast%3DSchmitt%26aufirst%3DD.%2BC.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DN.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DMultifaceted%2520roles%2520of%2520HSF1%2520in%2520cancer%26jtitle%3DTumor%2520Biol.%26date%3D2015%26volume%3D36%26spage%3D4923%26epage%3D4931%26doi%3D10.1007%2Fs13277-015-3674-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Whitesell, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindquist, S.</span><span> </span><span class="NLM_article-title">Inhibiting the transcription factor HSF1 as an anticancer strategy</span> <span class="citation_source-journal">Expert Opin. Ther. Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">469</span><span class="NLM_x">–</span> <span class="NLM_lpage">478</span><span class="refDoi"> DOI: 10.1517/14728220902832697</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1517%2F14728220902832697" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=19335068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjvVWnsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2009&pages=469-478&author=L.+Whitesellauthor=S.+Lindquist&title=Inhibiting+the+transcription+factor+HSF1+as+an+anticancer+strategy&doi=10.1517%2F14728220902832697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14aR"><div class="casContent"><span class="casTitleNuber">14a</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting the transcription factor HSF1 as an anticancer strategy</span></div><div class="casAuthors">Whitesell, Luke; Lindquist, Susan</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">469-478</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  In mammals, the cytoprotective heat-shock response is regulated primarily by heat shock factor 1 (HSF1).  Unfortunately, the effects of HSF1 also support the ability of cancer cells to accommodate imbalances in signaling and alterations in DNA, protein and energy metab. assocd. with oncogenesis.  The malignant lifestyle confers dependence on this non-oncogene', suggesting a therapeutic role for HSF1 inhibitors.  We begin with an overview of how HSF1 affects cancer biol. and how its activity is regulated.  We then summarize progress in discovery and development of HSF1 inhibitors, their current limitations and potential as anticancer agents with a fundamentally different scope of action from other clin. validated modulators of protein homeostasis.  It is likely that within the next 5 years usable inhibitors of HSF1 will be identified and in early pre-clin. evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_J1_jCERfLbVg90H21EOLACvtfcHk0lgih6orgmG9xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjvVWnsr8%253D&md5=adfbdeed040bf19a03f7e4d6f69c0c6c</span></div><a href="/servlet/linkout?suffix=cit14a&amp;dbid=16384&amp;doi=10.1517%2F14728220902832697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728220902832697%26sid%3Dliteratum%253Aachs%26aulast%3DWhitesell%26aufirst%3DL.%26aulast%3DLindquist%26aufirst%3DS.%26atitle%3DInhibiting%2520the%2520transcription%2520factor%2520HSF1%2520as%2520an%2520anticancer%2520strategy%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2009%26volume%3D13%26spage%3D469%26epage%3D478%26doi%3D10.1517%2F14728220902832697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit14b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Moore, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dewal, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nekongo, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santiago, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levine, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoulders, M. D.</span><span> </span><span class="NLM_article-title">Transportable, chemical genetic methodology for the small molecule-mediated inhibition of heat shock factor 1</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">200</span><span class="NLM_x">–</span> <span class="NLM_lpage">210</span><span class="refDoi"> DOI: 10.1021/acschembio.5b00740</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00740" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslSrtLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=200-210&author=C.+L.+Mooreauthor=M.+B.+Dewalauthor=E.+E.+Nekongoauthor=S.+Santiagoauthor=N.+B.+Luauthor=S.+S.+Levineauthor=M.+D.+Shoulders&title=Transportable%2C+chemical+genetic+methodology+for+the+small+molecule-mediated+inhibition+of+heat+shock+factor+1&doi=10.1021%2Facschembio.5b00740"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14bR"><div class="casContent"><span class="casTitleNuber">14b</span><div class="casTitle"><span class="NLM_cas:atitle">Transportable, Chemical Genetic Methodology for the Small Molecule-Mediated Inhibition of Heat Shock Factor 1</span></div><div class="casAuthors">Moore, Christopher L.; Dewal, Mahender B.; Nekongo, Emmanuel E.; Santiago, Sebasthian; Lu, Nancy B.; Levine, Stuart S.; Shoulders, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">200-210</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteostasis in the cytosol is governed by the heat shock response.  The master regulator of the heat shock response, heat shock factor 1 (HSF1), and key chaperones whose levels are HSF1-regulated have emerged as high-profile targets for therapeutic applications ranging from protein misfolding-related disorders to cancer.  Nonetheless, a generally applicable methodol. to selectively and potently inhibit endogenous HSF1 in a small mol.-dependent manner in disease model systems remains elusive.  Also problematic, the administration of even highly selective chaperone inhibitors often has the side effect of activating HSF1 and thereby inducing a compensatory heat shock response.  Herein, we report a ligand-regulatable, dominant neg. version of HSF1 that addresses these issues.  Our approach, which required engineering a new dominant neg. HSF1 variant, permits dosable inhibition of endogenous HSF1 with a selective small mol. in cell-based model systems of interest.  The methodol. allows us to uncouple the pleiotropic effects of chaperone inhibitors and environmental toxins from the concomitantly induced compensatory heat shock response.  Integration of our method with techniques to activate HSF1 enables the creation of cell lines in which the cytosolic proteostasis network can be up- or down-regulated by orthogonal small mols.  Selective, small mol.-mediated inhibition of HSF1 has distinctive implications for the proteostasis of both chaperone-dependent globular proteins and aggregation-prone intrinsically disordered proteins.  Altogether, this work provides crit. methods for continued exploration of the biol. roles of HSF1 and the therapeutic potential of heat shock response modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKGNceoLmk-rVg90H21EOLACvtfcHk0lgih6orgmG9xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslSrtLfJ&md5=2d04135b9393ae16ee1bce79c6d890be</span></div><a href="/servlet/linkout?suffix=cit14b&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00740&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00740%26sid%3Dliteratum%253Aachs%26aulast%3DMoore%26aufirst%3DC.%2BL.%26aulast%3DDewal%26aufirst%3DM.%2BB.%26aulast%3DNekongo%26aufirst%3DE.%2BE.%26aulast%3DSantiago%26aufirst%3DS.%26aulast%3DLu%26aufirst%3DN.%2BB.%26aulast%3DLevine%26aufirst%3DS.%2BS.%26aulast%3DShoulders%26aufirst%3DM.%2BD.%26atitle%3DTransportable%252C%2520chemical%2520genetic%2520methodology%2520for%2520the%2520small%2520molecule-mediated%2520inhibition%2520of%2520heat%2520shock%2520factor%25201%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D200%26epage%3D210%26doi%3D10.1021%2Facschembio.5b00740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit14c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">de Billy, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Travers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span> </span><span class="NLM_article-title">Shock about heat shock in cancer</span> <span class="citation_source-journal">Oncotarget</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">741</span><span class="NLM_x">–</span> <span class="NLM_lpage">743</span><span class="refDoi"> DOI: 10.18632/oncotarget.646</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.18632%2Foncotarget.646" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=741-743&author=E.+de+Billyauthor=J.+Traversauthor=P.+Workman&title=Shock+about+heat+shock+in+cancer&doi=10.18632%2Foncotarget.646"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14c&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.646%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BBilly%26aufirst%3DE.%26aulast%3DTravers%26aufirst%3DJ.%26aulast%3DWorkman%26aufirst%3DP.%26atitle%3DShock%2520about%2520heat%2520shock%2520in%2520cancer%26jtitle%3DOncotarget%26date%3D2012%26volume%3D3%26spage%3D741%26epage%3D743%26doi%3D10.18632%2Foncotarget.646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Mendillo, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santagata, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koeva, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tamimi, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraenkel, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ince, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitesell, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindquist, S.</span><span> </span><span class="NLM_article-title">HSF1 drives a transcriptional program distinct from heat shock to support highly malignant human cancers</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">150</span><span class="NLM_x">, </span> <span class="NLM_fpage">549</span><span class="NLM_x">–</span> <span class="NLM_lpage">562</span><span class="refDoi"> DOI: 10.1016/j.cell.2012.06.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.cell.2012.06.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=22863008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFGrsLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2012&pages=549-562&author=M.+L.+Mendilloauthor=S.+Santagataauthor=M.+Koevaauthor=G.+W.+Bellauthor=R.+Huauthor=R.+M.+Tamimiauthor=E.+Fraenkelauthor=T.+A.+Inceauthor=L.+Whitesellauthor=S.+Lindquist&title=HSF1+drives+a+transcriptional+program+distinct+from+heat+shock+to+support+highly+malignant+human+cancers&doi=10.1016%2Fj.cell.2012.06.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">HSF1 drives a transcriptional program distinct from heat shock to support highly malignant human cancers</span></div><div class="casAuthors">Mendillo, Marc L.; Santagata, Sandro; Koeva, Martina; Bell, George W.; Hu, Rong; Tamimi, Rulla M.; Fraenkel, Ernest; Ince, Tan A.; Whitesell, Luke; Lindquist, Susan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">549-562</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Heat-Shock Factor 1 (HSF1), master regulator of the heat-shock response, facilitates malignant transformation, cancer cell survival, and proliferation in model systems.  The common assumption is that these effects are mediated through regulation of heat-shock protein (HSP) expression.  However, the transcriptional network that HSF1 coordinates directly in malignancy and its relationship to the heat-shock response have never been defined.  By comparing cells with high and low malignant potential alongside their nontransformed counterparts, we identify an HSF1-regulated transcriptional program specific to highly malignant cells and distinct from heat shock.  Cancer-specific genes in this program support oncogenic processes: cell-cycle regulation, signaling, metab., adhesion and translation.  HSP genes are integral to this program, however, many are uniquely regulated in malignancy.  This HSF1 cancer program is active in breast, colon and lung tumors isolated directly from human patients and is strongly assocd. with metastasis and death.  Thus, HSF1 rewires the transcriptome in tumorigenesis, with prognostic and therapeutic implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-7fuiXrhe0bVg90H21EOLACvtfcHk0lgih6orgmG9xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFGrsLzE&md5=56a6bb78f044d25ead6338d2d9f82693</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.06.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.06.031%26sid%3Dliteratum%253Aachs%26aulast%3DMendillo%26aufirst%3DM.%2BL.%26aulast%3DSantagata%26aufirst%3DS.%26aulast%3DKoeva%26aufirst%3DM.%26aulast%3DBell%26aufirst%3DG.%2BW.%26aulast%3DHu%26aufirst%3DR.%26aulast%3DTamimi%26aufirst%3DR.%2BM.%26aulast%3DFraenkel%26aufirst%3DE.%26aulast%3DInce%26aufirst%3DT.%2BA.%26aulast%3DWhitesell%26aufirst%3DL.%26aulast%3DLindquist%26aufirst%3DS.%26atitle%3DHSF1%2520drives%2520a%2520transcriptional%2520program%2520distinct%2520from%2520heat%2520shock%2520to%2520support%2520highly%2520malignant%2520human%2520cancers%26jtitle%3DCell%26date%3D2012%26volume%3D150%26spage%3D549%26epage%3D562%26doi%3D10.1016%2Fj.cell.2012.06.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Santagata, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, N. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendillo, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hankinson, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnitt, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitesell, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tamimi, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindquist, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ince, T. A.</span><span> </span><span class="NLM_article-title">High levels of nuclear heat-shock factor 1 (HSF1) are associated with poor prognosis in breast cancer</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">18378</span><span class="NLM_x">–</span> <span class="NLM_lpage">18383</span><span class="refDoi"> DOI: 10.1073/pnas.1115031108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1073%2Fpnas.1115031108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=22042860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsF2itrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=18378-18383&author=S.+Santagataauthor=R.+Huauthor=N.+U.+Linauthor=M.+L.+Mendilloauthor=L.+C.+Collinsauthor=S.+E.+Hankinsonauthor=S.+J.+Schnittauthor=L.+Whitesellauthor=R.+M.+Tamimiauthor=S.+Lindquistauthor=T.+A.+Ince&title=High+levels+of+nuclear+heat-shock+factor+1+%28HSF1%29+are+associated+with+poor+prognosis+in+breast+cancer&doi=10.1073%2Fpnas.1115031108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16aR"><div class="casContent"><span class="casTitleNuber">16a</span><div class="casTitle"><span class="NLM_cas:atitle">High levels of nuclear heat-shock factor 1 (HSF1) are associated with poor prognosis in breast cancer</span></div><div class="casAuthors">Santagata, Sandro; Hu, Rong; Lin, Nancy U.; Mendillo, Marc L.; Collins, Laura C.; Hankinson, Susan E.; Schnitt, Stuart J.; Whitesell, Luke; Tamimi, Rulla M.; Lindquist, Susan; Ince, Tan A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">18378-18383, S18378/1-S18378/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Heat-shock factor 1 (HSF1) is the master transcriptional regulator of the cellular response to heat and a wide variety of other stressors.  We previously reported that HSF1 promotes the survival and proliferation of malignant cells.  At this time, however, the clin. and prognostic significance of HSF1 in cancer is unknown.  To address this issue breast cancer samples from 1,841 participants in the Nurses' Health Study were scored for levels of nuclear HSF1.  Assocns. of HSF1 status with clin. parameters and survival outcomes were investigated by Kaplan-Meier anal. and Cox proportional hazard models.  The assocns. were further delineated by Kaplan-Meier anal. using publicly available mRNA expression data.  Our results show that nuclear HSF1 levels were elevated in ∼80% of in situ and invasive breast carcinomas.  In invasive carcinomas, HSF1 expression was assocd. with high histol. grade, larger tumor size, and nodal involvement at diagnosis (P < 0.0001). By-using multivariate anal. to account for the effects of covariates, high HSF1 levels were found to be independently assocd. with increased mortality (hazards ratio: 1.62; 95% confidence interval: 1.21-2.17; P < 0.0013). This assocn. was seen in the estrogen receptor (ER)-pos. population (hazards ratio: 2.10; 95% confidence interval: 1.45-3.03; P < 0.0001). In public expression profiling data, high HSF1 mRNA levels were also assocd. with an increase in ER-pos. breast cancer-specific mortality.  We conclude that increased HSF1 is assocd. with reduced breast cancer survival.  The findings indicate that HSF1 should be evaluated prospectively as an independent prognostic indicator in ER-pos. breast cancer.  HSF1 may ultimately be a useful therapeutic target in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq93NKPzO0hArVg90H21EOLACvtfcHk0lhkmTGyn_4Rcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsF2itrvI&md5=097a836a6ec39049b70b60e6d42b4718</span></div><a href="/servlet/linkout?suffix=cit16a&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1115031108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1115031108%26sid%3Dliteratum%253Aachs%26aulast%3DSantagata%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DN.%2BU.%26aulast%3DMendillo%26aufirst%3DM.%2BL.%26aulast%3DCollins%26aufirst%3DL.%2BC.%26aulast%3DHankinson%26aufirst%3DS.%2BE.%26aulast%3DSchnitt%26aufirst%3DS.%2BJ.%26aulast%3DWhitesell%26aufirst%3DL.%26aulast%3DTamimi%26aufirst%3DR.%2BM.%26aulast%3DLindquist%26aufirst%3DS.%26aulast%3DInce%26aufirst%3DT.%2BA.%26atitle%3DHigh%2520levels%2520of%2520nuclear%2520heat-shock%2520factor%25201%2520%2528HSF1%2529%2520are%2520associated%2520with%2520poor%2520prognosis%2520in%2520breast%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D18378%26epage%3D18383%26doi%3D10.1073%2Fpnas.1115031108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit16b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Fang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, L.</span><span> </span><span class="NLM_article-title">Heat shock factor 1 promotes invasion and metastasis of hepatocellular carcinoma in vitro and in vivo</span> <span class="citation_source-journal">Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">1782</span><span class="NLM_x">–</span> <span class="NLM_lpage">1794</span><span class="refDoi"> DOI: 10.1002/cncr.26482</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1002%2Fcncr.26482" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=22009757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38XktFWmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2012&pages=1782-1794&author=F.+Fangauthor=R.+Changauthor=L.+Yang&title=Heat+shock+factor+1+promotes+invasion+and+metastasis+of+hepatocellular+carcinoma+in+vitro+and+in+vivo&doi=10.1002%2Fcncr.26482"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16bR"><div class="casContent"><span class="casTitleNuber">16b</span><div class="casTitle"><span class="NLM_cas:atitle">Heat shock factor 1 promotes invasion and metastasis of hepatocellular carcinoma in vitro and in vivo</span></div><div class="casAuthors">Fang, Feng; Chang, Ruimin; Yang, Lianyue</div><div class="citationInfo"><span class="NLM_cas:title">Cancer (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1782-1794</span>CODEN:
                <span class="NLM_cas:coden">CANCAR</span>;
        ISSN:<span class="NLM_cas:issn">0008-543X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">BACKGROUND: Heat shock factor 1 (HSF1) is a powerful, multifaceted modifier of carcinogenesis.  However, the clin. significance and biol. function of HSF1 in hepatocellular carcinoma (HCC) remain unknown.  METHODS: Quant. reverse transcriptase-polymerase chain reaction anal., Western blot anal., and immunohistochem. staining were used to detect expression levels of HSF1, and its correlation with clinicopathol. parameters and the prognosis for patients with HCC were analyzed.  In addn., the biol. function and mol. mechanisms of HSF1 in HCC were investigated in vitro and in vivo.  RESULTS: HSF1 levels were elevated predominantly in HCC, esp. in venous emboli from HCC (P < .05), and high expression levels of HSF1 were correlated significantly with multiple nodules, venous invasion, absence of capsular formation, and high Edmondson-Steiner grade as well as poor overall survival and disease-free survival in patients with HCC (P < .05).  Multivariate Cox regression anal. revealed that high HSF1 expression was an independent prognostic factor for overall survival in patients with HCC (relative risk, 4.874; P < .001).  Finally, HSF1 was capable of promoting HCC cell migration and invasion in vitro and in vivo by facilitating the expression and phosphorylation of heat shock protein 27.  CONCLUSIONS: Collectively, the current findings suggested that HSF1 may serve as a novel prognostic marker and therapeutic target for HCC.  Cancer 2012;. © 2011 American Cancer Society.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8_TwAVVM7yLVg90H21EOLACvtfcHk0lhkmTGyn_4Rcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktFWmsbs%253D&md5=f95636fc9d7e691b3f7bbfe4cec30425</span></div><a href="/servlet/linkout?suffix=cit16b&amp;dbid=16384&amp;doi=10.1002%2Fcncr.26482&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcncr.26482%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DF.%26aulast%3DChang%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DL.%26atitle%3DHeat%2520shock%2520factor%25201%2520promotes%2520invasion%2520and%2520metastasis%2520of%2520hepatocellular%2520carcinoma%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DCancer%26date%3D2012%26volume%3D118%26spage%3D1782%26epage%3D1794%26doi%3D10.1002%2Fcncr.26482" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit16c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Scott, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nogueira, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heffernan, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Doorn, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhakal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanna, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaskelioff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, C.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeid, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feinberg, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanede Woude, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Granter, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bosenberg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DePinho, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rimm, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chin, L.</span><span> </span><span class="NLM_article-title">Pro-invasion metastasis drivers in early stage melanoma are oncogenes</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">92</span><span class="NLM_x">–</span> <span class="NLM_lpage">103</span><span class="refDoi"> DOI: 10.1016/j.ccr.2011.05.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.ccr.2011.05.025" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=92-103&author=K.+L.+Scottauthor=C.+Nogueiraauthor=T.+P.+Heffernanauthor=R.+van+Doornauthor=S.+Dhakalauthor=J.+A.+Hannaauthor=C.+Minauthor=M.+Jaskelioffauthor=Y.+Xiaoauthor=C.-J.+Wuauthor=L.+A.+Cameronauthor=S.+R.+Perryauthor=R.+Zeidauthor=T.+Feinbergauthor=M.+Kimauthor=G.+Vanede+Woudeauthor=S.+R.+Granterauthor=M.+Bosenbergauthor=G.+C.+Chuauthor=R.+A.+DePinhoauthor=D.+L.+Rimmauthor=L.+Chin&title=Pro-invasion+metastasis+drivers+in+early+stage+melanoma+are+oncogenes&doi=10.1016%2Fj.ccr.2011.05.025"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16c&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.05.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.05.025%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DK.%2BL.%26aulast%3DNogueira%26aufirst%3DC.%26aulast%3DHeffernan%26aufirst%3DT.%2BP.%26aulast%3Dvan%2BDoorn%26aufirst%3DR.%26aulast%3DDhakal%26aufirst%3DS.%26aulast%3DHanna%26aufirst%3DJ.%2BA.%26aulast%3DMin%26aufirst%3DC.%26aulast%3DJaskelioff%26aufirst%3DM.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DC.-J.%26aulast%3DCameron%26aufirst%3DL.%2BA.%26aulast%3DPerry%26aufirst%3DS.%2BR.%26aulast%3DZeid%26aufirst%3DR.%26aulast%3DFeinberg%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DM.%26aulast%3DVanede%2BWoude%26aufirst%3DG.%26aulast%3DGranter%26aufirst%3DS.%2BR.%26aulast%3DBosenberg%26aufirst%3DM.%26aulast%3DChu%26aufirst%3DG.%2BC.%26aulast%3DDePinho%26aufirst%3DR.%2BA.%26aulast%3DRimm%26aufirst%3DD.%2BL.%26aulast%3DChin%26aufirst%3DL.%26atitle%3DPro-invasion%2520metastasis%2520drivers%2520in%2520early%2520stage%2520melanoma%2520are%2520oncogenes%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D20%26spage%3D92%26epage%3D103%26doi%3D10.1016%2Fj.ccr.2011.05.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="note"><p class="first last">For a discussion of transcription inhibitors in oncology, see:</p></div><div class="NLM_citation" id="cit17a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Bensaude, O.</span><span> </span><span class="NLM_article-title">Inhibiting eukaryotic transcription, which compound to choose? how to evaluate its activity?</span> <span class="citation_source-journal">Transcr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">103</span><span class="NLM_x">–</span> <span class="NLM_lpage">108</span><span class="refDoi"> DOI: 10.4161/trns.2.3.16172</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.4161%2Ftrns.2.3.16172" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=21922053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A280%3ADC%252BC2sbovVCrtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=103-108&author=O.+Bensaude&title=Inhibiting+eukaryotic+transcription%2C+which+compound+to+choose%3F+how+to+evaluate+its+activity%3F&doi=10.4161%2Ftrns.2.3.16172"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17aR"><div class="casContent"><span class="casTitleNuber">17a</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting eukaryotic transcription: Which compound to choose? How to evaluate its activity?</span></div><div class="casAuthors">Bensaude Olivier</div><div class="citationInfo"><span class="NLM_cas:title">Transcription</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">103-108</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">This review first discusses ways in which we can evaluate transcription inhibition, describe changes in nuclear structure due to transcription inhibition, and report on genes that are paradoxically stimulated by transcription inhibition.  Next, it summarizes the characteristics and mechanisms of commonly used inhibitors: α-amanitin is highly selective for RNAP II and RNAP III but its action is slow, actinomycin D is fast but its selectivity is poor, CDK9 inhibitors such as DRB and flavopiridol are fast and reversible but many genes escape transcription inhibition.  New compounds, such as triptolide, are fast and selective and able to completely arrest transcription by triggering rapid degradation of RNAP II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTeQjDGoYQ9mtHxxy173TfNfW6udTcc2eZ5G4PsJkUcQLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sbovVCrtw%253D%253D&md5=9fb95455b03ee7a9365b4c83435fa6ea</span></div><a href="/servlet/linkout?suffix=cit17a&amp;dbid=16384&amp;doi=10.4161%2Ftrns.2.3.16172&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Ftrns.2.3.16172%26sid%3Dliteratum%253Aachs%26aulast%3DBensaude%26aufirst%3DO.%26atitle%3DInhibiting%2520eukaryotic%2520transcription%252C%2520which%2520compound%2520to%2520choose%253F%2520how%2520to%2520evaluate%2520its%2520activity%253F%26jtitle%3DTranscr.%26date%3D2011%26volume%3D2%26spage%3D103%26epage%3D108%26doi%3D10.4161%2Ftrns.2.3.16172" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit17b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Stellrecht, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. S.</span><span> </span><span class="NLM_article-title">Transcription inhibition as a therapeutic target for cancer</span> <span class="citation_source-journal">Cancers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">4170</span><span class="NLM_x">–</span> <span class="NLM_lpage">4190</span><span class="refDoi"> DOI: 10.3390/cancers3044170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.3390%2Fcancers3044170" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=4170-4190&author=C.+M.+Stellrechtauthor=L.+S.+Chen&title=Transcription+inhibition+as+a+therapeutic+target+for+cancer&doi=10.3390%2Fcancers3044170"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17b&amp;dbid=16384&amp;doi=10.3390%2Fcancers3044170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers3044170%26sid%3Dliteratum%253Aachs%26aulast%3DStellrecht%26aufirst%3DC.%2BM.%26aulast%3DChen%26aufirst%3DL.%2BS.%26atitle%3DTranscription%2520inhibition%2520as%2520a%2520therapeutic%2520target%2520for%2520cancer%26jtitle%3DCancers%26date%3D2011%26volume%3D3%26spage%3D4170%26epage%3D4190%26doi%3D10.3390%2Fcancers3044170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit17c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Kwiatkowski, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahl, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ficarro, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dastur, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amzallag, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramaswamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tesar, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hannett, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMillin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sim, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Look, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitsiades, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weng, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benes, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marto, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Targeting transcription regulation in cancer with a covalent CDK7 inhibitor</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">511</span><span class="NLM_x">, </span> <span class="NLM_fpage">616</span><span class="NLM_x">–</span> <span class="NLM_lpage">620</span><span class="refDoi"> DOI: 10.1038/nature13393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnature13393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=25043025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1ChurnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=511&publication_year=2014&pages=616-620&author=N.+Kwiatkowskiauthor=T.+Zhangauthor=P.+B.+Rahlauthor=B.+J.+Abrahamauthor=J.+Reddyauthor=S.+B.+Ficarroauthor=A.+Dasturauthor=A.+Amzallagauthor=S.+Ramaswamyauthor=B.+Tesarauthor=C.+E.+Jenkinsauthor=N.+M.+Hannettauthor=D.+McMillinauthor=T.+Sandaauthor=T.+Simauthor=N.+D.+Kimauthor=T.+Lookauthor=C.+S.+Mitsiadesauthor=A.+P.+Wengauthor=J.+R.+Brownauthor=C.+H.+Benesauthor=J.+A.+Martoauthor=R.+A.+Youngauthor=N.+S.+Gray&title=Targeting+transcription+regulation+in+cancer+with+a+covalent+CDK7+inhibitor&doi=10.1038%2Fnature13393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17cR"><div class="casContent"><span class="casTitleNuber">17c</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting transcription regulation in cancer with a covalent CDK7 inhibitor</span></div><div class="casAuthors">Kwiatkowski, Nicholas; Zhang, Tinghu; Rahl, Peter B.; Abraham, Brian J.; Reddy, Jessica; Ficarro, Scott B.; Dastur, Anahita; Amzallag, Arnaud; Ramaswamy, Sridhar; Tesar, Bethany; Jenkins, Catherine E.; Hannett, Nancy M.; McMillin, Douglas; Sanda, Takaomi; Sim, Taebo; Kim, Nam Doo; Look, Thomas; Mitsiades, Constantine S.; Weng, Andrew P.; Brown, Jennifer R.; Benes, Cyril H.; Marto, Jarrod A.; Young, Richard A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">511</span>
        (<span class="NLM_cas:issue">7511</span>),
    <span class="NLM_cas:pages">616-620</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tumor oncogenes include transcription factors that co-opt the general transcriptional machinery to sustain the oncogenic state, but direct pharmacol. inhibition of transcription factors has so far proven difficult.  However, the transcriptional machinery contains various enzymic cofactors that can be targeted for the development of new therapeutic candidates, including cyclin-dependent kinases (CDKs).  Here the authors present the discovery and characterization of a covalent CDK7 inhibitor, THZ1, which has the unprecedented ability to target a remote cysteine residue located outside of the canonical kinase domain, providing an unanticipated means of achieving selectivity for CDK7.  Cancer cell-line profiling indicates that a subset of cancer cell lines, including human T-cell acute lymphoblastic leukemia (T-ALL), have exceptional sensitivity to THZ1.  Genome-wide anal. in Jurkat T-ALL cells shows that THZ1 disproportionally affects transcription of RUNX1 and suggests that sensitivity to THZ1 may be due to vulnerability conferred by the RUNX1 super-enhancer and the key role of RUNX1 in the core transcriptional regulatory circuitry of these tumor cells.  Pharmacol. modulation of CDK7 kinase activity may thus provide an approach to identify and treat tumor types that are dependent on transcription for maintenance of the oncogenic state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVD-DeTAQCqrVg90H21EOLACvtfcHk0lgTDbKFjxpLhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1ChurnF&md5=895a28fb536663a31e7558fe79ce628b</span></div><a href="/servlet/linkout?suffix=cit17c&amp;dbid=16384&amp;doi=10.1038%2Fnature13393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13393%26sid%3Dliteratum%253Aachs%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DAbraham%26aufirst%3DB.%2BJ.%26aulast%3DReddy%26aufirst%3DJ.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DDastur%26aufirst%3DA.%26aulast%3DAmzallag%26aufirst%3DA.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DTesar%26aufirst%3DB.%26aulast%3DJenkins%26aufirst%3DC.%2BE.%26aulast%3DHannett%26aufirst%3DN.%2BM.%26aulast%3DMcMillin%26aufirst%3DD.%26aulast%3DSanda%26aufirst%3DT.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DN.%2BD.%26aulast%3DLook%26aufirst%3DT.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26aulast%3DWeng%26aufirst%3DA.%2BP.%26aulast%3DBrown%26aufirst%3DJ.%2BR.%26aulast%3DBenes%26aufirst%3DC.%2BH.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520transcription%2520regulation%2520in%2520cancer%2520with%2520a%2520covalent%2520CDK7%2520inhibitor%26jtitle%3DNature%26date%3D2014%26volume%3D511%26spage%3D616%26epage%3D620%26doi%3D10.1038%2Fnature13393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit17d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Nguyen, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, S.</span><span> </span><span class="NLM_article-title">Dinaciclib (SCH727965) inhibits the unfolded protein response through a CDK1- and 5-dependent mechanism</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">662</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-13-0714</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1158%2F1535-7163.MCT-13-0714" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=24362465" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjslGit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=662-674&author=T.+K.+Nguyenauthor=S.+Grant&title=Dinaciclib+%28SCH727965%29+inhibits+the+unfolded+protein+response+through+a+CDK1-+and+5-dependent+mechanism&doi=10.1158%2F1535-7163.MCT-13-0714"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17dR"><div class="casContent"><span class="casTitleNuber">17d</span><div class="casTitle"><span class="NLM_cas:atitle">Dinaciclib (SCH727965) Inhibits the Unfolded Protein Response through a CDK1- and 5-Dependent Mechanism</span></div><div class="casAuthors">Nguyen, Tri K.; Grant, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">662-674</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Evidence implicating dysregulation of the IRE1/XBP-1s arm of the unfolded protein response (UPR) in cancer pathogenesis (e.g., multiple myeloma) has prompted the development of IRE1 RNase inhibitors.  Here, effects of cyclin-dependent kinase (CDK) inhibitor SCH727965 (dinaciclib) on the IRE1 arm of the UPR were examd. in human leukemia and myeloma cells.  Exposure of cells to extremely low (e.g., nmol/L) concns. of SCH727965, a potent inhibitor of CDKs 1/2/5/9, diminished XBP-1s and Grp78 induction by the endoplasmic reticulum (ER) stress-inducers thapsigargin and tunicamycin, while sharply inducing cell death.  SCH727965, in contrast to IRE1 RNase inhibitors, inhibited the UPR in assocn. with attenuation of XBP-1s nuclear localization and accumulation rather than transcription, translation, or XBP-1 splicing.  Notably, in human leukemia cells, CDK1 and 5 short hairpin RNA (shRNA) knockdown diminished Grp78 and XBP-1s upregulation while increasing thapsigargin lethality, arguing for a functional role for CDK1/5 in activation of the cytoprotective IRE1/XBP-1s arm of the UPR.  In contrast, CDK9 or 2 inhibitors or shRNA knockdown failed to downregulate XBP-1s or Grp78.  Furthermore, IRE1, XBP-1, or Grp78 knockdown significantly increased thapsigargin lethality, as obsd. with CDK1/5 inhibition/knockdown.  Finally, SCH727965 diminished myeloma cell growth in vivo in assocn. with XBP-1s downregulation.  Together, these findings demonstrate that SCH727965 acts at extremely low concns. to attenuate XBP-1s nuclear accumulation and Grp78 upregulation in response to ER stress inducers.  They also highlight a link between specific components of the cell-cycle regulatory app. (e.g., CDK1/5) and the cytoprotective IRE1/XBP-1s/Grp78 arm of the UPR that may be exploited therapeutically in UPR-driven malignancies.  Mol Cancer Ther; 13(3); 662-74. ©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGV9Rbcg389bVg90H21EOLACvtfcHk0lgTDbKFjxpLhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjslGit74%253D&md5=1ead94a6d179e340c952dca738f6c62c</span></div><a href="/servlet/linkout?suffix=cit17d&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-13-0714&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-13-0714%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DT.%2BK.%26aulast%3DGrant%26aufirst%3DS.%26atitle%3DDinaciclib%2520%2528SCH727965%2529%2520inhibits%2520the%2520unfolded%2520protein%2520response%2520through%2520a%2520CDK1-%2520and%25205-dependent%2520mechanism%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2014%26volume%3D13%26spage%3D662%26epage%3D674%26doi%3D10.1158%2F1535-7163.MCT-13-0714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit17e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Lam, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbas, A. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradshaw, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walsby, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pepper, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Targeting RNA transcription and translation in ovarian cancer cells with pharmacological inhibitor CDKI-73</span> <span class="citation_source-journal">Oncotarget</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">7691</span><span class="NLM_x">–</span> <span class="NLM_lpage">7704</span><span class="refDoi"> DOI: 10.18632/oncotarget.2296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.18632%2Foncotarget.2296" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=7691-7704&author=F.+Lamauthor=A.+Y.+Abbasauthor=H.+Shaoauthor=T.+Teoauthor=J.+Adamsauthor=P.+Liauthor=T.+D.+Bradshawauthor=P.+M.+Fischerauthor=E.+Walsbyauthor=C.+Pepperauthor=Y.+Chenauthor=J.+Dingauthor=S.+Wang&title=Targeting+RNA+transcription+and+translation+in+ovarian+cancer+cells+with+pharmacological+inhibitor+CDKI-73&doi=10.18632%2Foncotarget.2296"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17e&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2296%26sid%3Dliteratum%253Aachs%26aulast%3DLam%26aufirst%3DF.%26aulast%3DAbbas%26aufirst%3DA.%2BY.%26aulast%3DShao%26aufirst%3DH.%26aulast%3DTeo%26aufirst%3DT.%26aulast%3DAdams%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DBradshaw%26aufirst%3DT.%2BD.%26aulast%3DFischer%26aufirst%3DP.%2BM.%26aulast%3DWalsby%26aufirst%3DE.%26aulast%3DPepper%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DTargeting%2520RNA%2520transcription%2520and%2520translation%2520in%2520ovarian%2520cancer%2520cells%2520with%2520pharmacological%2520inhibitor%2520CDKI-73%26jtitle%3DOncotarget%26date%3D2014%26volume%3D5%26spage%3D7691%26epage%3D7704%26doi%3D10.18632%2Foncotarget.2296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit17f"><span><span class="NLM_label">(f) </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffiths, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Midgley, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnett, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grabarek, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ingram, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kontopidis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McClue, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McInnes, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLachlan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meades, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mezna, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuart, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheleva, D. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glover, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blake, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, P. M.</span><span> </span><span class="NLM_article-title">Discovery and characterization of 2-anilino-4-(thiazol-5-yl)pyrimidine transcriptional CDK inhibitors as anticancer agents</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">1111</span><span class="NLM_x">–</span> <span class="NLM_lpage">1121</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2010.07.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.chembiol.2010.07.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17f&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=21035734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlKhtLfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=1111-1121&author=S.+Wangauthor=G.+Griffithsauthor=C.+A.+Midgleyauthor=A.+L.+Barnettauthor=M.+Cooperauthor=J.+Grabarekauthor=L.+Ingramauthor=W.+Jacksonauthor=G.+Kontopidisauthor=S.+J.+McClueauthor=C.+McInnesauthor=J.+McLachlanauthor=C.+Meadesauthor=M.+Meznaauthor=I.+Stuartauthor=M.+P.+Thomasauthor=D.+I.+Zhelevaauthor=D.+P.+Laneauthor=R.+C.+Jacksonauthor=D.+M.+Gloverauthor=D.+G.+Blakeauthor=P.+M.+Fischer&title=Discovery+and+characterization+of+2-anilino-4-%28thiazol-5-yl%29pyrimidine+transcriptional+CDK+inhibitors+as+anticancer+agents&doi=10.1016%2Fj.chembiol.2010.07.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17fR"><div class="casContent"><span class="casTitleNuber">17f</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Characterization of 2-Anilino-4- (Thiazol-5-yl)Pyrimidine Transcriptional CDK Inhibitors as Anticancer Agents</span></div><div class="casAuthors">Wang, Shu-Dong; Griffiths, Gary; Midgley, Carol A.; Barnett, Anna L.; Cooper, Michael; Grabarek, Joanna; Ingram, Laura; Jackson, Wayne; Kontopidis, George; McClue, Steven J.; McInnes, Campbell; McLachlan, Janice; Meades, Christopher; Mezna, Mokdad; Stuart, Iain; Thomas, Mark P.; Zheleva, Daniella I.; Lane, David P.; Jackson, Robert C.; Glover, David M.; Blake, David G.; Fischer, Peter M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1111-1121</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The main difficulty in the development of ATP antagonist kinase inhibitors is target specificity, since the ATP-binding motif is present in many proteins.  We introduce a strategy that has allowed us to identify compds. from a kinase inhibitor library that block the cyclin-dependent kinases responsible for regulating transcription, i.e., CDK7 and esp. CDK9.  The screening cascade employs cellular phenotypic assays based on mitotic index and nuclear p53 protein accumulation.  This permitted us to classify compds. into transcriptional, cell cycle, and mitotic inhibitor groups.  We describe the characterization of the transcriptional inhibitor class in terms of kinase inhibition profile, cellular mode of action, and selectivity for transformed cells.  A structural selectivity rationale was used to optimize potency and biopharmaceutical properties and led to the development of a transcriptional inhibitor, 3,4-dimethyl-5-[2-(4-piperazin-1-yl-phenylamino)-pyrimidin-4-yl]-3H-thiazol-2-one, with anticancer activity in animal models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryOG4LTevHibVg90H21EOLACvtfcHk0li6Tj0M0CZKTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlKhtLfL&md5=08aff2706f6eda4ba1038327267a361e</span></div><a href="/servlet/linkout?suffix=cit17f&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.07.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.07.016%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DGriffiths%26aufirst%3DG.%26aulast%3DMidgley%26aufirst%3DC.%2BA.%26aulast%3DBarnett%26aufirst%3DA.%2BL.%26aulast%3DCooper%26aufirst%3DM.%26aulast%3DGrabarek%26aufirst%3DJ.%26aulast%3DIngram%26aufirst%3DL.%26aulast%3DJackson%26aufirst%3DW.%26aulast%3DKontopidis%26aufirst%3DG.%26aulast%3DMcClue%26aufirst%3DS.%2BJ.%26aulast%3DMcInnes%26aufirst%3DC.%26aulast%3DMcLachlan%26aufirst%3DJ.%26aulast%3DMeades%26aufirst%3DC.%26aulast%3DMezna%26aufirst%3DM.%26aulast%3DStuart%26aufirst%3DI.%26aulast%3DThomas%26aufirst%3DM.%2BP.%26aulast%3DZheleva%26aufirst%3DD.%2BI.%26aulast%3DLane%26aufirst%3DD.%2BP.%26aulast%3DJackson%26aufirst%3DR.%2BC.%26aulast%3DGlover%26aufirst%3DD.%2BM.%26aulast%3DBlake%26aufirst%3DD.%2BG.%26aulast%3DFischer%26aufirst%3DP.%2BM.%26atitle%3DDiscovery%2520and%2520characterization%2520of%25202-anilino-4-%2528thiazol-5-yl%2529pyrimidine%2520transcriptional%2520CDK%2520inhibitors%2520as%2520anticancer%2520agents%26jtitle%3DChem.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D1111%26epage%3D1121%26doi%3D10.1016%2Fj.chembiol.2010.07.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit17g"><span><span class="NLM_label">(g) </span><span class="NLM_contrib-group">Villicana, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zurita, M.</span><span> </span><span class="NLM_article-title">The basal transcription machinery as a target for cancer therapy</span> <span class="citation_source-journal">Cancer Cell Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">18</span><span class="refDoi"> DOI: 10.1186/1475-2867-14-18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17g&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1186%2F1475-2867-14-18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17g&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=24576043" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17g&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjsVartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=18&author=C.+Villicanaauthor=G.+Cruzauthor=M.+Zurita&title=The+basal+transcription+machinery+as+a+target+for+cancer+therapy&doi=10.1186%2F1475-2867-14-18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17gR"><div class="casContent"><span class="casTitleNuber">17g</span><div class="casTitle"><span class="NLM_cas:atitle">The basal transcription machinery as a target for cancer therapy</span></div><div class="casAuthors">Villicana, Claudia; Cruz, Grisel; Zurita, Mario</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell International</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">18/1-18/15, 15 pp.</span>CODEN:
                <span class="NLM_cas:coden">CCIACC</span>;
        ISSN:<span class="NLM_cas:issn">1475-2867</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">General transcription is required for the growth and survival of all living cells.  However, tumor cells require extraordinary levels of transcription, including the transcription of rRNA genes by RNA polymerase I (RNPI) and mRNA by RNA polymerase II (RNPII).  In fact, cancer cells have mutations that directly enhance transcription and are frequently required for cancer transformation.  For example, the recent discovery that MYC enhances the transcription of the majority genes in the genome correlates with the fact that several transcription interfering drugs preferentially kill cancer cells.  In recent years, advances in the mechanistic studies of the basal transcription machinery and the discovery of drugs that interfere with multiple components of transcription are being used to combat cancer.  For example, drugs such as triptolide that targets the general transcription factors TFIIH and JQ1 to inhibit BRD4 are administered to target the high proliferative rate of cancer cells.  Given the importance of finding new strategies to preferentially sensitize tumor cells, this review primarily focuses on several transcription inhibitory drugs to demonstrate that the basal transcription machinery constitutes a potential target for the design of novel cancer drugs.  We highlight the drugs' mechanisms for interfering with tumor cell survival, their importance in cancer treatment and the challenges of clin. application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7aEZQbw_0xLVg90H21EOLACvtfcHk0li6Tj0M0CZKTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjsVartbo%253D&md5=806c757e799d3a318038191c9e1c3520</span></div><a href="/servlet/linkout?suffix=cit17g&amp;dbid=16384&amp;doi=10.1186%2F1475-2867-14-18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1475-2867-14-18%26sid%3Dliteratum%253Aachs%26aulast%3DVillicana%26aufirst%3DC.%26aulast%3DCruz%26aufirst%3DG.%26aulast%3DZurita%26aufirst%3DM.%26atitle%3DThe%2520basal%2520transcription%2520machinery%2520as%2520a%2520target%2520for%2520cancer%2520therapy%26jtitle%3DCancer%2520Cell%2520Int.%26date%3D2014%26volume%3D14%26spage%3D18%26doi%3D10.1186%2F1475-2867-14-18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Solimini, N. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elledge, S. J.</span><span> </span><span class="NLM_article-title">Non-oncogene addiction and the stress phenotype of cancer cells</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">986</span><span class="NLM_x">–</span> <span class="NLM_lpage">988</span><span class="refDoi"> DOI: 10.1016/j.cell.2007.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.cell.2007.09.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2007&pages=986-988&author=N.+L.+Soliminiauthor=J.+Luoauthor=S.+J.+Elledge&title=Non-oncogene+addiction+and+the+stress+phenotype+of+cancer+cells&doi=10.1016%2Fj.cell.2007.09.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DSolimini%26aufirst%3DN.%2BL.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DElledge%26aufirst%3DS.%2BJ.%26atitle%3DNon-oncogene%2520addiction%2520and%2520the%2520stress%2520phenotype%2520of%2520cancer%2520cells%26jtitle%3DCell%26date%3D2007%26volume%3D130%26spage%3D986%26epage%3D988%26doi%3D10.1016%2Fj.cell.2007.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Zou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guettouche, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voellmy, R.</span><span> </span><span class="NLM_article-title">Repression of heat shock transcription factor HSF1 activation by HSP90 (HSP90 complex) that forms a stress-sensitive complex with HSF1</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">471</span><span class="NLM_x">–</span> <span class="NLM_lpage">480</span><span class="refDoi"> DOI: 10.1016/S0092-8674(00)81588-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2FS0092-8674%2800%2981588-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=9727490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADyaK1cXlslOrsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1998&pages=471-480&author=J.+Zouauthor=Y.+Guoauthor=T.+Guettoucheauthor=D.+F.+Smithauthor=R.+Voellmy&title=Repression+of+heat+shock+transcription+factor+HSF1+activation+by+HSP90+%28HSP90+complex%29+that+forms+a+stress-sensitive+complex+with+HSF1&doi=10.1016%2FS0092-8674%2800%2981588-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19aR"><div class="casContent"><span class="casTitleNuber">19a</span><div class="casTitle"><span class="NLM_cas:atitle">Repression of heat shock transcription factor HSF1 activation by HSP90 (HSP90 complex) that forms a stress-sensitive complex with HSF1</span></div><div class="casAuthors">Zou, Jiangying; Guo, Yongle; Guettouche, Toumy; Smith, David F.; Voellmy, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">471-480</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Heat shock and other proteotoxic stresses cause accumulation of nonnative proteins that trigger activation of heat shock protein (Hsp) genes.  A chaperone/Hsp functioning as repressor of heat shock transcription factor (HSF) could make activation of hsp genes dependent on protein unfolding.  In a novel in vitro system, in which human HSF1 can be activated by nonnative protein, heat, and geldanamycin, addn. of Hsp90 inhibits activation.  Redn. of the level of Hsp90 but not of Hsp/c70, Hop, Hip, p23, CyP40, or Hsp40 dramatically activates HSF1.  In vivo, geldanamycin activates HSF1 under conditions in which it is an Hsp90-specific reagent.  Hsp90-contg. HSF1 complex is present in the unstressed cell and dissocs. during stress.  We conclude that Hsp90, by itself and/or assocd. with multichaperone complexes, is a major repressor of HSF1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW_ZsrrM8VFrVg90H21EOLACvtfcHk0liIAZgd4-328w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXlslOrsrY%253D&md5=26024c9d4e94c7f46986320fc667fd6f</span></div><a href="/servlet/linkout?suffix=cit19a&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981588-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981588-3%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DGuettouche%26aufirst%3DT.%26aulast%3DSmith%26aufirst%3DD.%2BF.%26aulast%3DVoellmy%26aufirst%3DR.%26atitle%3DRepression%2520of%2520heat%2520shock%2520transcription%2520factor%2520HSF1%2520activation%2520by%2520HSP90%2520%2528HSP90%2520complex%2529%2520that%2520forms%2520a%2520stress-sensitive%2520complex%2520with%2520HSF1%26jtitle%3DCell%26date%3D1998%26volume%3D94%26spage%3D471%26epage%3D480%26doi%3D10.1016%2FS0092-8674%2800%2981588-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit19b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Doubrovin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Che, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serganova, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moroz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solit, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ageyeva, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kochetkova, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pillarsetti, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finn, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blasberg, R. G.</span><span> </span><span class="NLM_article-title">Monitoring the induction of heat shock factor 1/heat shock protein 70 expression following 17-allylamino-17-demethoxygeldanamycin treatment by positron emission tomography and optical reporter gene imaging</span> <span class="citation_source-journal">Mol. Imaging</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">67</span><span class="NLM_x">–</span> <span class="NLM_lpage">76</span><span class="refDoi"> DOI: 10.2310/7290.2011.00028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.2310%2F7290.2011.00028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=22418029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVGrtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=67-76&author=M.+Doubrovinauthor=J.+T.+Cheauthor=I.+Serganovaauthor=E.+Morozauthor=D.+B.+Solitauthor=L.+Ageyevaauthor=T.+Kochetkovaauthor=N.+Pillarsettiauthor=R.+Finnauthor=N.+Rosenauthor=R.+G.+Blasberg&title=Monitoring+the+induction+of+heat+shock+factor+1%2Fheat+shock+protein+70+expression+following+17-allylamino-17-demethoxygeldanamycin+treatment+by+positron+emission+tomography+and+optical+reporter+gene+imaging&doi=10.2310%2F7290.2011.00028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19bR"><div class="casContent"><span class="casTitleNuber">19b</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring the induction of heat shock factor 1/heat shock protein 70 expression following 17-allylamino-demethoxygeldanamycin treatment by positron emission tomography and optical reporter gene imaging</span></div><div class="casAuthors">Doubrovin, Mikhail; Che, Jian T.; Serganova, Inna; Moroz, Ekaterina; Solit, David B.; Ageyeva, Lyudmila; Kochetkova, Tatiana; Pillarsetti, Nagavarakishore; Finn, Ronald; Rosen, Neal; Blasberg, Ronald G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Imaging</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-76</span>CODEN:
                <span class="NLM_cas:coden">MIOMBP</span>;
        ISSN:<span class="NLM_cas:issn">1535-3508</span>.
    
            (<span class="NLM_cas:orgname">Decker Publishing</span>)
        </div><div class="casAbstract">The cell response to proteotoxic cell stresses is mediated primarily through activation of heat shock factor 1 (HSF1).  This transcription factor plays a major role in the regulation of the heat shock proteins (HSPs), including HSP70.  We demonstrate that an [124I]iodide-pQHNIG70 positron emission tomog. (PET) reporter system that includes an inducible HSP70 promoter can be used to image and monitor the activation of the HSF1/HSP70 transcription factor in response to drug treatment (17-allylamino-demethoxygeldanamycin [17-AAG]).  We developed a dual imaging reporter (pQHNIG70) for noninvasive imaging of the heat shock response in cell culture and living animals previously and now study HSF1/HSP70 reporter activation in both cell culture and tumor-bearing animals following exposure to 17-AAG.  17-AAG (10-1,000 nM) induced reporter expression; a 23-fold increase was obsd. by 60 h.  Good correspondence between reporter expression and HSP70 protein levels were obsd.  MicroPET imaging based on [124I]iodide accumulation in pQHNIG70-transduced RG2 xenografts showed a significant 6.2-fold reporter response to 17-AAG, with a corresponding increase in tumor HSP70 and in tumor human sodium iodide symporter and green fluorescent protein reporter proteins.  The HSF1 reporter system can be used to screen anticancer drugs for induction of cytotoxic stress and HSF1 activation both in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0HTURK3faW7Vg90H21EOLACvtfcHk0liIAZgd4-328w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVGrtLk%253D&md5=3a63cac21e10c6aa9c43df81d0390af9</span></div><a href="/servlet/linkout?suffix=cit19b&amp;dbid=16384&amp;doi=10.2310%2F7290.2011.00028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2310%252F7290.2011.00028%26sid%3Dliteratum%253Aachs%26aulast%3DDoubrovin%26aufirst%3DM.%26aulast%3DChe%26aufirst%3DJ.%2BT.%26aulast%3DSerganova%26aufirst%3DI.%26aulast%3DMoroz%26aufirst%3DE.%26aulast%3DSolit%26aufirst%3DD.%2BB.%26aulast%3DAgeyeva%26aufirst%3DL.%26aulast%3DKochetkova%26aufirst%3DT.%26aulast%3DPillarsetti%26aufirst%3DN.%26aulast%3DFinn%26aufirst%3DR.%26aulast%3DRosen%26aufirst%3DN.%26aulast%3DBlasberg%26aufirst%3DR.%2BG.%26atitle%3DMonitoring%2520the%2520induction%2520of%2520heat%2520shock%2520factor%25201%252Fheat%2520shock%2520protein%252070%2520expression%2520following%252017-allylamino-17-demethoxygeldanamycin%2520treatment%2520by%2520positron%2520emission%2520tomography%2520and%2520optical%2520reporter%2520gene%2520imaging%26jtitle%3DMol.%2520Imaging%26date%3D2012%26volume%3D11%26spage%3D67%26epage%3D76%26doi%3D10.2310%2F7290.2011.00028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit19c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Au, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barber, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, B.</span><span> </span><span class="NLM_article-title">Identification of inhibitors of HSF1 functional activity by high-content target-based screening</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1165</span><span class="NLM_x">–</span> <span class="NLM_lpage">1175</span><span class="refDoi"> DOI: 10.1177/1087057109347472</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1177%2F1087057109347472" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=19820069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVSltrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=1165-1175&author=Q.+Auauthor=Y.+Zhangauthor=J.+R.+Barberauthor=S.+C.+Ngauthor=B.+Zhang&title=Identification+of+inhibitors+of+HSF1+functional+activity+by+high-content+target-based+screening&doi=10.1177%2F1087057109347472"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19cR"><div class="casContent"><span class="casTitleNuber">19c</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of inhibitors of HSF1 functional activity by high-content target-based screening</span></div><div class="casAuthors">Au, Qingyan; Zhang, Yingjia; Barber, Jack R.; Ng, Shi Chung; Zhang, Bin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1165-1175</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Cancer cells are known to experience a high level of stress and may require const. repair for survival and proliferation.  Recent studies showed that inhibition of heat shock factor 1 (HSF1), the key regulator for the stress-activated transcription of heat shock protein (HSP), can reduce the tumorigenic potential of cancer cells.  Such a "nononcogene addiction" phenomenon makes HSF1 an attractive cancer drug target.  Here, the authors report an image-based high-content screening (HCS) assay for HSF1 functional inhibitors.  A heat shock-based methodol. was used to stimulate the stress response followed by quant. measurement of HSF1/HSP70 granules for compd.-induced inhibitory effects.  The authors discovered a small mol. from a compd. library that inhibits HSF1 granule formation substantially in heat-shocked HeLa cells with IC50 at 80 nM.  Electrophoretic mobility shift of HSF1 by this compd. suggested significant inhibition of HSF1 phosphorylation, accompanied by reduced expression levels of HSP70 and HSP90 after heat induction.  Importantly, HeLa cells stably transfected with HSF1 shRNA were more resistant to the compd. treatment under lethal temp. than cells contg. HSF1, further validating an HSF1-dependent mechanism of action.  The HCS assay the authors developed was robust with a Z' factor of 0.65 in a 384-well plate format, providing a valuable method for identifying small-mol. functional inhibitors of HSF1 for potential cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf3LVGRu4f5LVg90H21EOLACvtfcHk0liIAZgd4-328w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVSltrc%253D&md5=ec73b3c9f122bfe61d12e3562d6f4724</span></div><a href="/servlet/linkout?suffix=cit19c&amp;dbid=16384&amp;doi=10.1177%2F1087057109347472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109347472%26sid%3Dliteratum%253Aachs%26aulast%3DAu%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DBarber%26aufirst%3DJ.%2BR.%26aulast%3DNg%26aufirst%3DS.%2BC.%26aulast%3DZhang%26aufirst%3DB.%26atitle%3DIdentification%2520of%2520inhibitors%2520of%2520HSF1%2520functional%2520activity%2520by%2520high-content%2520target-based%2520screening%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26spage%3D1165%26epage%3D1175%26doi%3D10.1177%2F1087057109347472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">West, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morano, K. A.</span><span> </span><span class="NLM_article-title">Small molecule activators of the heat shock response: chemical properties, molecular targets, and therapeutic promise</span> <span class="citation_source-journal">Chem. Res. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">2036</span><span class="NLM_x">–</span> <span class="NLM_lpage">2053</span><span class="refDoi"> DOI: 10.1021/tx300264x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx300264x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2012&pages=2036-2053&author=J.+D.+Westauthor=Y.+Wangauthor=K.+A.+Morano&title=Small+molecule+activators+of+the+heat+shock+response%3A+chemical+properties%2C+molecular+targets%2C+and+therapeutic+promise&doi=10.1021%2Ftx300264x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Ftx300264x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx300264x%26sid%3Dliteratum%253Aachs%26aulast%3DWest%26aufirst%3DJ.%2BD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMorano%26aufirst%3DK.%2BA.%26atitle%3DSmall%2520molecule%2520activators%2520of%2520the%2520heat%2520shock%2520response%253A%2520chemical%2520properties%252C%2520molecular%2520targets%252C%2520and%2520therapeutic%2520promise%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2012%26volume%3D25%26spage%3D2036%26epage%3D2053%26doi%3D10.1021%2Ftx300264x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Santagata, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendillo, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subramanian, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perley, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roche, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narayan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koeva, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golub, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porco, J. A.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitesell, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindquist, S.</span><span> </span><span class="NLM_article-title">Tight coordination of protein translation and HSF1 activation supports the anabolic malignant state</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">341</span><span class="NLM_x">, </span> <span class="NLM_fpage">1238303</span><span class="refDoi"> DOI: 10.1126/science.1238303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1126%2Fscience.1238303" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=23869022" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A280%3ADC%252BC3sfisF2htg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=1238303&author=S.+Santagataauthor=M.+L.+Mendilloauthor=Y.+Tangauthor=A.+Subramanianauthor=C.+C.+Perleyauthor=S.+P.+Rocheauthor=B.+Wongauthor=R.+Narayanauthor=H.+Kwonauthor=M.+Koevaauthor=A.+Amonauthor=T.+R.+Golubauthor=J.+A.+Porcoauthor=L.+Whitesellauthor=S.+Lindquist&title=Tight+coordination+of+protein+translation+and+HSF1+activation+supports+the+anabolic+malignant+state&doi=10.1126%2Fscience.1238303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21aR"><div class="casContent"><span class="casTitleNuber">21a</span><div class="casTitle"><span class="NLM_cas:atitle">Tight coordination of protein translation and HSF1 activation supports the anabolic malignant state</span></div><div class="casAuthors">Santagata Sandro; Mendillo Marc L; Tang Yun-chi; Subramanian Aravind; Perley Casey C; Roche Stephane P; Wong Bang; Narayan Rajiv; Kwon Hyoungtae; Koeva Martina; Amon Angelika; Golub Todd R; Porco John A Jr; Whitesell Luke; Lindquist Susan</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6143</span>),
    <span class="NLM_cas:pages">1238303</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The ribosome is centrally situated to sense metabolic states, but whether its activity, in turn, coherently rewires transcriptional responses is unknown.  Here, through integrated chemical-genetic analyses, we found that a dominant transcriptional effect of blocking protein translation in cancer cells was inactivation of heat shock factor 1 (HSF1), a multifaceted transcriptional regulator of the heat-shock response and many other cellular processes essential for anabolic metabolism, cellular proliferation, and tumorigenesis.  These analyses linked translational flux to the regulation of HSF1 transcriptional activity and to the modulation of energy metabolism.  Targeting this link with translation initiation inhibitors such as rocaglates deprived cancer cells of their energy and chaperone armamentarium and selectively impaired the proliferation of both malignant and premalignant cells with early-stage oncogenic lesions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQMPgMUk-vccmlSCSgBhwJFfW6udTcc2eZt3BglzjjPIbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sfisF2htg%253D%253D&md5=8bcd13a849bdc03a186b072d6adec86f</span></div><a href="/servlet/linkout?suffix=cit21a&amp;dbid=16384&amp;doi=10.1126%2Fscience.1238303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1238303%26sid%3Dliteratum%253Aachs%26aulast%3DSantagata%26aufirst%3DS.%26aulast%3DMendillo%26aufirst%3DM.%2BL.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DPerley%26aufirst%3DC.%2BC.%26aulast%3DRoche%26aufirst%3DS.%2BP.%26aulast%3DWong%26aufirst%3DB.%26aulast%3DNarayan%26aufirst%3DR.%26aulast%3DKwon%26aufirst%3DH.%26aulast%3DKoeva%26aufirst%3DM.%26aulast%3DAmon%26aufirst%3DA.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DWhitesell%26aufirst%3DL.%26aulast%3DLindquist%26aufirst%3DS.%26atitle%3DTight%2520coordination%2520of%2520protein%2520translation%2520and%2520HSF1%2520activation%2520supports%2520the%2520anabolic%2520malignant%2520state%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D1238303%26doi%3D10.1126%2Fscience.1238303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit21b"><span><span class="NLM_label">(b) </span><div class="note"><p class="first last">Triptolide:</p></div><span class="NLM_contrib-group">Westerheide, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawahara, T. L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orton, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morimoto, R. I.</span><span> </span><span class="NLM_article-title">Triptolide, an inhibitor of the human heat shock response that enhances stress-induced cell death</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">9616</span><span class="NLM_x">–</span> <span class="NLM_lpage">9622</span><span class="refDoi"> DOI: 10.1074/jbc.M512044200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1074%2Fjbc.M512044200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=16469748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BD28XjtVSqs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=9616-9622&author=S.+D.+Westerheideauthor=T.+L.+A.+Kawaharaauthor=K.+Ortonauthor=R.+I.+Morimoto&title=Triptolide%2C+an+inhibitor+of+the+human+heat+shock+response+that+enhances+stress-induced+cell+death&doi=10.1074%2Fjbc.M512044200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21bR"><div class="casContent"><span class="casTitleNuber">21b</span><div class="casTitle"><span class="NLM_cas:atitle">Triptolide, an Inhibitor of the Human Heat Shock Response That Enhances Stress-induced Cell Death</span></div><div class="casAuthors">Westerheide, Sandy D.; Kawahara, Tiara L. A.; Orton, Kai; Morimoto, Richard I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">9616-9622</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Mol. chaperones, inducible by heat shock and a variety of other stresses, have crit. roles in protein homeostasis, balancing cell stress with adaptation, survival, and cell death mechanisms.  In transformed cells and tumors, chaperones are frequently overexpressed, with constitutive activation of the heat shock transcription factor HSF1 implicated in tumor formation.  Here, we describe the activity of triptolide, a diterpene triepoxide from the plant Triptergium wilfordii, as an inhibitor of the human heat shock response.  Triptolide treatment of human tissue culture cells prevented the inducible expression of heat shock genes, shown by suppression of an HSP70 promoter-reporter construct and by suppression of endogenous HSP70 gene expression.  Upon examg. the steps in the HSF1 activation pathway, we found that triptolide abrogates the transactivation function of HSF1 without interfering in the early events of trimer formation, hyperphosphorylation, and DNA binding.  The ability of triptolide to inhibit the heat shock response renders these cells sensitive to stress-induced cell death, which may be of great relevance to cancer treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBmhgTU2H_UbVg90H21EOLACvtfcHk0ljwaIxnpn8bFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjtVSqs7g%253D&md5=e0108de875d6bb30843d4f3b7315c46f</span></div><a href="/servlet/linkout?suffix=cit21b&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M512044200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M512044200%26sid%3Dliteratum%253Aachs%26aulast%3DWesterheide%26aufirst%3DS.%2BD.%26aulast%3DKawahara%26aufirst%3DT.%2BL.%2BA.%26aulast%3DOrton%26aufirst%3DK.%26aulast%3DMorimoto%26aufirst%3DR.%2BI.%26atitle%3DTriptolide%252C%2520an%2520inhibitor%2520of%2520the%2520human%2520heat%2520shock%2520response%2520that%2520enhances%2520stress-induced%2520cell%2520death%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D9616%26epage%3D9622%26doi%3D10.1074%2Fjbc.M512044200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit21c"><span><span class="NLM_label">(c) </span><div class="note"><p class="first last">BEZ235 and SNS-032:</p></div><span class="NLM_contrib-group">Acquaviva, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sequeira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bates, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Proia, D. A.</span><span> </span><span class="NLM_article-title">mTOR inhibition potentiates HSP90 inhibitor activity via cessation of HSP synthesis</span> <span class="citation_source-journal">Mol. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">703</span><span class="NLM_x">–</span> <span class="NLM_lpage">713</span><span class="refDoi"> DOI: 10.1158/1541-7786.MCR-13-0605</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1158%2F1541-7786.MCR-13-0605" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=24554781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnslCrsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=703-713&author=J.+Acquavivaauthor=S.+Heauthor=J.+Sangauthor=D.+L.+Smithauthor=M.+Sequeiraauthor=C.+Zhangauthor=R.+C.+Batesauthor=D.+A.+Proia&title=mTOR+inhibition+potentiates+HSP90+inhibitor+activity+via+cessation+of+HSP+synthesis&doi=10.1158%2F1541-7786.MCR-13-0605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21cR"><div class="casContent"><span class="casTitleNuber">21c</span><div class="casTitle"><span class="NLM_cas:atitle">mTOR Inhibition Potentiates HSP90 Inhibitor Activity via Cessation of HSP Synthesis</span></div><div class="casAuthors">Acquaviva, Jaime; He, Suqin; Sang, Jim; Smith, Donald L.; Sequeira, Manuel; Zhang, Chaohua; Bates, Richard C.; Proia, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">703-713</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Because of their pleiotropic effects on crit. oncoproteins, inhibitors of HSP90 represent a promising new class of therapeutic agents for the treatment of human cancer.  However, pharmacol. inactivation of HSP90 subsequently triggers a heat shock response that may mitigate the full therapeutic benefit of these compds.  To overcome this limitation, a clin. feasible method was sought to block HSP synthesis induced by the potent HSP90 inhibitor ganetespib.  An immunoassay screen of 322 late-stage or clin. approved drugs was performed to uncover compds. that could block upregulation of the stress-inducible HSP70 that results as a consequence of HSP90 blockade.  Interestingly, inhibitors of the phosphoinositide 3-kinase (PI3K)/mTOR class counteracted ganetespib-induced HSP70 upregulation at both the gene and protein level by suppressing nuclear translocation of heat shock factor 1 (HSF1), the dominant transcription factor controlling cellular stress responses.  This effect was conserved across multiple tumor types and was found to be regulated, in part, by mTOR-dependent translational activity.  Pretreatment with cycloheximide, PI3K/mTOR inhibitor, or an inhibitor of eIF4E (a translation initiation factor and downstream effector of mTOR) all reduced ganetespib-mediated nuclear HSF1 accumulation, indicating that mTOR blockade confers a neg. regulatory effect on HSF1 activity.  Moreover, combined therapy regimens with mTOR or dual PI3K/mTOR inhibitors potentiated the antitumor efficacy of ganetespib in multiple in vivo models.  Implications: Collectively these data identify a novel strategy to optimize the therapeutic potential of HSP90 inhibitors.  Mol Cancer Res; 12(5); 703-13. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUGLLojl79cLVg90H21EOLACvtfcHk0ljwaIxnpn8bFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnslCrsr0%253D&md5=70701fbc238094847be0503aaf5224fd</span></div><a href="/servlet/linkout?suffix=cit21c&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-13-0605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-13-0605%26sid%3Dliteratum%253Aachs%26aulast%3DAcquaviva%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DSang%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DD.%2BL.%26aulast%3DSequeira%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DBates%26aufirst%3DR.%2BC.%26aulast%3DProia%26aufirst%3DD.%2BA.%26atitle%3DmTOR%2520inhibition%2520potentiates%2520HSP90%2520inhibitor%2520activity%2520via%2520cessation%2520of%2520HSP%2520synthesis%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2014%26volume%3D12%26spage%3D703%26epage%3D713%26doi%3D10.1158%2F1541-7786.MCR-13-0605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit21d"><span><span class="NLM_label">(d) </span><div class="note"><p class="first last">For Dinaciclib and SNS-032 see ref <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">28</a></p></div>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=For+Dinaciclib+and+SNS-032+see+ref+28."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21d&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="note"><p class="first last">The compound library was provided by AstraZeneca, see ref <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">28</a>.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Powers, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valenti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miranda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maloney, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eccles, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span> </span><span class="NLM_article-title">Mode of cell death induced by the HSP90 inhibitor 17-AAG (tanespimycin) is dependent on the expression of pro-apoptotic BAX</span> <span class="citation_source-journal">Oncotarget</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1963</span><span class="NLM_x">–</span> <span class="NLM_lpage">1965</span><span class="refDoi"> DOI: 10.18632/oncotarget.1419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.18632%2Foncotarget.1419" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1963-1965&author=M.+V.+Powersauthor=M.+Valentiauthor=S.+Mirandaauthor=A.+Maloneyauthor=S.+A.+Ecclesauthor=G.+Thomasauthor=P.+A.+Clarkeauthor=P.+Workman&title=Mode+of+cell+death+induced+by+the+HSP90+inhibitor+17-AAG+%28tanespimycin%29+is+dependent+on+the+expression+of+pro-apoptotic+BAX&doi=10.18632%2Foncotarget.1419"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.1419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.1419%26sid%3Dliteratum%253Aachs%26aulast%3DPowers%26aufirst%3DM.%2BV.%26aulast%3DValenti%26aufirst%3DM.%26aulast%3DMiranda%26aufirst%3DS.%26aulast%3DMaloney%26aufirst%3DA.%26aulast%3DEccles%26aufirst%3DS.%2BA.%26aulast%3DThomas%26aufirst%3DG.%26aulast%3DClarke%26aufirst%3DP.%2BA.%26aulast%3DWorkman%26aufirst%3DP.%26atitle%3DMode%2520of%2520cell%2520death%2520induced%2520by%2520the%2520HSP90%2520inhibitor%252017-AAG%2520%2528tanespimycin%2529%2520is%2520dependent%2520on%2520the%2520expression%2520of%2520pro-apoptotic%2520BAX%26jtitle%3DOncotarget%26date%3D2013%26volume%3D4%26spage%3D1963%26epage%3D1965%26doi%3D10.18632%2Foncotarget.1419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Jones, K.; Rye, C.; Chessum, N.; Cheeseman, M.; Pasqua, A. E.; Pike, K. G.; Faulder, P. F.</span><span> </span><span class="NLM_article-title">Fused 1,4-dihydrodioxin derivatives as inhibitors of heat shock transcription factor 1</span>. WO 2015/049535 A1, April 9,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=K.+Jones&author=C.+Rye&author=N.+Chessum&author=M.+Cheeseman&author=A.+E.+Pasqua&author=K.+G.+Pike&author=P.+F.+Faulder&title=Fused+1%2C4-dihydrodioxin+derivatives+as+inhibitors+of+heat+shock+transcription+factor+1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DK.%26atitle%3DFused%25201%252C4-dihydrodioxin%2520derivatives%2520as%2520inhibitors%2520of%2520heat%2520shock%2520transcription%2520factor%25201%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="note"><p class="first last">pIC<sub>50</sub> = −log<sub>10</sub> IC<sub>50</sub> (M). All results are quoted to three significant figures as geometric mean ± standard error of the mean (SEM). The geometric mean and the number of repeats are quoted in the parenthesis.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Hardcastle, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boxall, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomlin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aherne, W.</span><span> </span><span class="NLM_article-title">Solid-phase immunoassays in mechanism-based drug discovery: their application in the development of inhibitors of the molecular chaperone heat-shock protein 90</span> <span class="citation_source-journal">Assay Drug Dev. Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">273</span><span class="NLM_x">–</span> <span class="NLM_lpage">285</span><span class="refDoi"> DOI: 10.1089/adt.2005.3.273</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1089%2Fadt.2005.3.273" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2005&pages=273-285&author=A.+Hardcastleauthor=K.+Boxallauthor=J.+Richardsauthor=P.+Tomlinauthor=S.+Sharpauthor=P.+Clarkeauthor=P.+Workmanauthor=W.+Aherne&title=Solid-phase+immunoassays+in+mechanism-based+drug+discovery%3A+their+application+in+the+development+of+inhibitors+of+the+molecular+chaperone+heat-shock+protein+90&doi=10.1089%2Fadt.2005.3.273"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1089%2Fadt.2005.3.273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2005.3.273%26sid%3Dliteratum%253Aachs%26aulast%3DHardcastle%26aufirst%3DA.%26aulast%3DBoxall%26aufirst%3DK.%26aulast%3DRichards%26aufirst%3DJ.%26aulast%3DTomlin%26aufirst%3DP.%26aulast%3DSharp%26aufirst%3DS.%26aulast%3DClarke%26aufirst%3DP.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DAherne%26aufirst%3DW.%26atitle%3DSolid-phase%2520immunoassays%2520in%2520mechanism-based%2520drug%2520discovery%253A%2520their%2520application%2520in%2520the%2520development%2520of%2520inhibitors%2520of%2520the%2520molecular%2520chaperone%2520heat-shock%2520protein%252090%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2005%26volume%3D3%26spage%3D273%26epage%3D285%26doi%3D10.1089%2Fadt.2005.3.273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Laitem, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaborowska, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isa, N. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kufs, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dienstbier, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, S.</span><span> </span><span class="NLM_article-title">CDK9 inhibitors define elongation checkpoints at both ends of RNA polymerase II-transcribed genes</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">396</span><span class="NLM_x">–</span> <span class="NLM_lpage">405</span><span class="refDoi"> DOI: 10.1038/nsmb.3000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnsmb.3000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=25849141" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlKltrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=396-405&author=C.+Laitemauthor=J.+Zaborowskaauthor=N.+F.+Isaauthor=J.+Kufsauthor=M.+Dienstbierauthor=S.+Murphy&title=CDK9+inhibitors+define+elongation+checkpoints+at+both+ends+of+RNA+polymerase+II-transcribed+genes&doi=10.1038%2Fnsmb.3000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">CDK9 inhibitors define elongation checkpoints at both ends of RNA polymerase II-transcribed genes</span></div><div class="casAuthors">Laitem, Clelia; Zaborowska, Justyna; Isa, Nur F.; Kufs, Johann; Dienstbier, Martin; Murphy, Shona</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">396-403</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Transcription through early-elongation checkpoints requires phosphorylation of neg. transcription elongation factors (NTEFs) by the cyclin-dependent kinase (CDK) 9.  Using CDK9 inhibitors and global run-on sequencing (GRO-seq), we have mapped CDK9 inhibitor-sensitive checkpoints genome wide in human cells.  Our data indicate that early-elongation checkpoints are a general feature of RNA polymerase (pol) II-transcribed human genes and occur independently of polymerase stalling.  Pol II that has negotiated the early-elongation checkpoint can elongate in the presence of inhibitors but, remarkably, terminates transcription prematurely close to the terminal polyadenylation (poly(A)) site.  Our anal. has revealed an unexpected poly(A)-assocd. elongation checkpoint, which has major implications for the regulation of gene expression.  Interestingly, the pattern of modification of the C-terminal domain of pol II terminated at this new checkpoint largely mirrors the pattern normally found downstream of the poly(A) site, thus suggesting common mechanisms of termination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDoG6k1kV9nrVg90H21EOLACvtfcHk0lgfiuDAygC5RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlKltrk%253D&md5=f95ac3fd82fc8d3ce6914567114b0d1f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3000%26sid%3Dliteratum%253Aachs%26aulast%3DLaitem%26aufirst%3DC.%26aulast%3DZaborowska%26aufirst%3DJ.%26aulast%3DIsa%26aufirst%3DN.%2BF.%26aulast%3DKufs%26aufirst%3DJ.%26aulast%3DDienstbier%26aufirst%3DM.%26aulast%3DMurphy%26aufirst%3DS.%26atitle%3DCDK9%2520inhibitors%2520define%2520elongation%2520checkpoints%2520at%2520both%2520ends%2520of%2520RNA%2520polymerase%2520II-transcribed%2520genes%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D396%26epage%3D405%26doi%3D10.1038%2Fnsmb.3000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Rye, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessum, N. E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamont, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pike, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faulder, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demeritt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kemmitt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tucker, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zani, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheeseman, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isaac, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodwin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boros, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raynaud, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayes, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henley, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Billy, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">te Poele, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’ Fee, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foote, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, K.</span><span> </span><span class="NLM_article-title">Discovery of 4,6-disubstituted pyrimidines as potent inhibitors of the heat shock factor (HSF1) stress pathway and CDK9</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1580</span><span class="NLM_x">–</span> <span class="NLM_lpage">1586</span><span class="refDoi"> DOI: 10.1039/C6MD00159A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1039%2FC6MD00159A" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=1580-1586&author=C.+S.+Ryeauthor=N.+E.+A.+Chessumauthor=S.+Lamontauthor=K.+G.+Pikeauthor=P.+Faulderauthor=J.+Demerittauthor=P.+Kemmittauthor=J.+Tuckerauthor=L.+Zaniauthor=M.+D.+Cheesemanauthor=R.+Isaacauthor=L.+Goodwinauthor=J.+Borosauthor=F.+Raynaudauthor=A.+Hayesauthor=A.+Henleyauthor=E.+de+Billyauthor=C.+J.+Lynchauthor=S.+Y.+Sharpauthor=R.+te+Poeleauthor=L.+O%E2%80%99+Feeauthor=K.+M.+Footeauthor=S.+Greenauthor=P.+Workmanauthor=K.+Jones&title=Discovery+of+4%2C6-disubstituted+pyrimidines+as+potent+inhibitors+of+the+heat+shock+factor+%28HSF1%29+stress+pathway+and+CDK9&doi=10.1039%2FC6MD00159A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2FC6MD00159A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00159A%26sid%3Dliteratum%253Aachs%26aulast%3DRye%26aufirst%3DC.%2BS.%26aulast%3DChessum%26aufirst%3DN.%2BE.%2BA.%26aulast%3DLamont%26aufirst%3DS.%26aulast%3DPike%26aufirst%3DK.%2BG.%26aulast%3DFaulder%26aufirst%3DP.%26aulast%3DDemeritt%26aufirst%3DJ.%26aulast%3DKemmitt%26aufirst%3DP.%26aulast%3DTucker%26aufirst%3DJ.%26aulast%3DZani%26aufirst%3DL.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26aulast%3DIsaac%26aufirst%3DR.%26aulast%3DGoodwin%26aufirst%3DL.%26aulast%3DBoros%26aufirst%3DJ.%26aulast%3DRaynaud%26aufirst%3DF.%26aulast%3DHayes%26aufirst%3DA.%26aulast%3DHenley%26aufirst%3DA.%26aulast%3Dde%2BBilly%26aufirst%3DE.%26aulast%3DLynch%26aufirst%3DC.%2BJ.%26aulast%3DSharp%26aufirst%3DS.%2BY.%26aulast%3Dte%2BPoele%26aufirst%3DR.%26aulast%3DO%25E2%2580%2599%2BFee%26aufirst%3DL.%26aulast%3DFoote%26aufirst%3DK.%2BM.%26aulast%3DGreen%26aufirst%3DS.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DJones%26aufirst%3DK.%26atitle%3DDiscovery%2520of%25204%252C6-disubstituted%2520pyrimidines%2520as%2520potent%2520inhibitors%2520of%2520the%2520heat%2520shock%2520factor%2520%2528HSF1%2529%2520stress%2520pathway%2520and%2520CDK9%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D1580%26epage%3D1586%26doi%3D10.1039%2FC6MD00159A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Guettouche, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boellmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voellmy, R.</span><span> </span><span class="NLM_article-title">Analysis of phosphorylation of human heat shock factor 1 in cells experiencing a stress</span> <span class="citation_source-journal">BMC Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">4</span><span class="refDoi"> DOI: 10.1186/1471-2091-6-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1186%2F1471-2091-6-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=15760475" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A280%3ADC%252BD2M7nvVGjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=4&author=T.+Guettoucheauthor=F.+Boellmannauthor=W.+S.+Laneauthor=R.+Voellmy&title=Analysis+of+phosphorylation+of+human+heat+shock+factor+1+in+cells+experiencing+a+stress&doi=10.1186%2F1471-2091-6-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29aR"><div class="casContent"><span class="casTitleNuber">29a</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of phosphorylation of human heat shock factor 1 in cells experiencing a stress</span></div><div class="casAuthors">Guettouche Toumy; Boellmann Frank; Lane William S; Voellmy Richard</div><div class="citationInfo"><span class="NLM_cas:title">BMC biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">4</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Heat shock factor (HSF/HSF1) not only is the transcription factor primarily responsible for the transcriptional response of cells to physical and chemical stress but also coregulates other important signaling pathways.  The factor mediates the stress-induced expression of heat shock or stress proteins (HSPs).  HSF/HSF1 is inactive in unstressed cells and is activated during stress.  Activation is accompanied by hyperphosphorylation of the factor.  The regulatory importance of this phosphorylation has remained incompletely understood.  Several previous studies on human HSF1 were concerned with phosphorylation on Ser303, Ser307 and Ser363, which phosphorylation appears to be related to factor deactivation subsequent to stress, and one study reported stress-induced phosphorylation of Ser230 contributing to factor activation.  However, no previous study attempted to fully describe the phosphorylation status of an HSF/HSF1 in stressed cells and to systematically identify phosphoresidues involved in factor activation.  The present study reports such an analysis for human HSF1 in heat-stressed cells.  RESULTS:  An alanine scan of all Ser, Thr and Tyr residues of human HSF1 was carried out using a validated transactivation assay, and residues phosphorylated in HSF1 were identified by mass spectrometry and sequencing.  HSF1 activated by heat treatment was phosphorylated on Ser121, Ser230, Ser292, Ser303, Ser307, Ser314, Ser319, Ser326, Ser344, Ser363, Ser419, and Ser444.  Phosphorylation of Ser326 but none of the other Ser residues was found to contribute significantly to activation of the factor by heat stress.  Phosphorylation on Ser326 increased rapidly during heat stress as shown by experiments using a pSer326 phosphopeptide antibody.  Heat stress-induced DNA binding and nuclear translocation of a S326A substitution mutant was not impaired in HSF1-negative cells, but the mutant stimulated HSP70 expression several times less well than wild type factor.  CONCLUSION:  Twelve Ser residues but no Thr or Tyr residues were identified that were phosphorylated in heat-activated HSF1.  Mutagenesis experiments and functional studies suggested that phosphorylation of HSF1 residue Ser326 plays a critical role in the induction of the factor's transcriptional competence by heat stress.  PhosphoSer326 also contributes to activation of HSF1 by chemical stress.  To date, no functional role could be ascribed to any of the other newly identified phosphoSer residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQfx34-qVWLnWhHGZei4OW9fW6udTcc2eaeDa9OgrLcibntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M7nvVGjsA%253D%253D&md5=b4717d4af602d104e4af57020e402f74</span></div><a href="/servlet/linkout?suffix=cit29a&amp;dbid=16384&amp;doi=10.1186%2F1471-2091-6-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2091-6-4%26sid%3Dliteratum%253Aachs%26aulast%3DGuettouche%26aufirst%3DT.%26aulast%3DBoellmann%26aufirst%3DF.%26aulast%3DLane%26aufirst%3DW.%2BS.%26aulast%3DVoellmy%26aufirst%3DR.%26atitle%3DAnalysis%2520of%2520phosphorylation%2520of%2520human%2520heat%2520shock%2520factor%25201%2520in%2520cells%2520experiencing%2520a%2520stress%26jtitle%3DBMC%2520Biochem.%26date%3D2005%26volume%3D6%26spage%3D4%26doi%3D10.1186%2F1471-2091-6-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit29b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Budzynski, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puustinen, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joutsen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sistonen, L.</span><span> </span><span class="NLM_article-title">Uncoupling stress-inducible phosphorylation of heat shock factor 1 from its activation</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">2530</span><span class="NLM_x">–</span> <span class="NLM_lpage">2540</span><span class="refDoi"> DOI: 10.1128/MCB.00816-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1128%2FMCB.00816-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=25963659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFamurrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=2530-2540&author=M.+A.+Budzynskiauthor=M.+C.+Puustinenauthor=J.+Joutsenauthor=L.+Sistonen&title=Uncoupling+stress-inducible+phosphorylation+of+heat+shock+factor+1+from+its+activation&doi=10.1128%2FMCB.00816-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29bR"><div class="casContent"><span class="casTitleNuber">29b</span><div class="casTitle"><span class="NLM_cas:atitle">Uncoupling stress-inducible phosphorylation of heat shock factor 1 from its activation</span></div><div class="casAuthors">Budzynski, Marek A.; Puustinen, Mikael C.; Joutsen, Jenny; Sistonen, Lea</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2530-2540</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">1098-5549</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">In mammals the stress-inducible expression of genes encoding heat shock proteins is under the control of the heat shock transcription factor 1 (HSF1).  Activation of HSF1 is a multistep process, involving trimerization, acquisition of DNA-binding and transcriptional activities, which coincide with several posttranslational modifications.  Stress-inducible phosphorylation of HSF1, or hyperphosphorylation, which occurs mainly within the regulatory domain (RD), has been proposed as a requirement for HSF-driven transcription and is widely used for assessing HSF1 activation.  Nonetheless, the contribution of hyperphosphorylation to the activity of HSF1 remains unknown.  In this study, we generated a phosphorylation-deficient HSF1 mutant (HSF1Δ∼PRD), where the 15 known phosphorylation sites within the RD were disrupted.  Our results show that the phosphorylation status of the RD does not affect the subcellular localization and DNA-binding activity of HSF1.  Surprisingly, under stress conditions, HSF1Δ∼PRD is a potent transactivator of both endogenous targets and a reporter gene, and HSF1Δ∼PRD has a reduced activation threshold.  Our results provide the first direct evidence for uncoupling stress-inducible phosphorylation of HSF1 from its activation, and we propose that the phosphorylation signature alone is not an appropriate marker for HSF1 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGri3_8rGXXWrrVg90H21EOLACvtfcHk0lh0ZN1z3fzYmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFamurrP&md5=baff5411c24b2d2167af35caf17480c5</span></div><a href="/servlet/linkout?suffix=cit29b&amp;dbid=16384&amp;doi=10.1128%2FMCB.00816-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00816-14%26sid%3Dliteratum%253Aachs%26aulast%3DBudzynski%26aufirst%3DM.%2BA.%26aulast%3DPuustinen%26aufirst%3DM.%2BC.%26aulast%3DJoutsen%26aufirst%3DJ.%26aulast%3DSistonen%26aufirst%3DL.%26atitle%3DUncoupling%2520stress-inducible%2520phosphorylation%2520of%2520heat%2520shock%2520factor%25201%2520from%2520its%2520activation%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2015%26volume%3D35%26spage%3D2530%26epage%3D2540%26doi%3D10.1128%2FMCB.00816-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span>KINOMEscan—World’s Largest Kinase Assay Panel; <span class="NLM_publisher-name">DiscoverX Corporation</span>: <span class="NLM_publisher-loc">San Diego, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span><a href="https://www.discoverx.com/services/drug-discovery-development-services/kinase-profiling/kinomescan" class="extLink">https://www.discoverx.com/services/drug-discovery-development-services/kinase-profiling/kinomescan</a> (accessed August 31, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=KINOMEscan%E2%80%94World%E2%80%99s+Largest+Kinase+Assay+Panel%3B+DiscoverX+Corporation%3A+San+Diego%2C+CA%2C+2016%3B+https%3A%2F%2Fwww.discoverx.com%2Fservices%2Fdrug-discovery-development-services%2Fkinase-profiling%2Fkinomescan+%28accessed+August+31%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DKINOMEscan%25E2%2580%2594World%25E2%2580%2599s%2520Largest%2520Kinase%2520Assay%2520Panel%26pub%3DDiscoverX%2520Corporation%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span>SelectScreen Screening and Profiling Services; <span class="NLM_publisher-name">Thermo Fisher Scientific Inc.</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span><a href="http://www.thermofisher.com/uk/en/home/products-and-services/services/custom-services/screening-and-profiling-services/selectscreen-profiling-service.html?icid=fr-dd-1" class="extLink">http://www.thermofisher.com/uk/en/home/products-and-services/services/custom-services/screening-and-profiling-services/selectscreen-profiling-service.html?icid=fr-dd-1</a> (accessed August 31, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=SelectScreen+Screening+and+Profiling+Services%3B+Thermo+Fisher+Scientific+Inc.%2C+2016%3B+http%3A%2F%2Fwww.thermofisher.com%2Fuk%2Fen%2Fhome%2Fproducts-and-services%2Fservices%2Fcustom-services%2Fscreening-and-profiling-services%2Fselectscreen-profiling-service.html%3Ficid%3Dfr-dd-1+%28accessed+August+31%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DSelectScreen%2520Screening%2520and%2520Profiling%2520Services%26pub%3DThermo%2520Fisher%2520Scientific%2520Inc%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span>All BRAF inhibitors were purchased from Selleckchem and were used without further purification, <a href="http://www.selleckchem.com" class="extLink">http://www.selleckchem.com</a> (accessedAugust 31,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>. See <span class="NLM_xref">Supporting Information</span> for details.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=All+BRAF+inhibitors+were+purchased+from+Selleckchem+and+were+used+without+further+purification%2C+http%3A%2F%2Fwww.selleckchem.com+%28accessedAugust+31%2C+2016%29+.+See+Supporting+Information+for+details."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="note"><p class="first last">All BRAF inhibitors were screened in the human ovarian carcinoma cell-line SK-OV-3.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Garnett, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edelman, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heidorn, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenman, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dastur, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lau, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greninger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, I. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soares, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iorio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surdez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milano, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bignell, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tam, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevenson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barthorpe, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lutz, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogera, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawrence, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mclaren-Douglas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitropoulos, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mironenko, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jewitt, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boisvert, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hur, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baselga, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stamenkovic, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelman, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delattre, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saez-Rodriguez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Settleman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Futreal, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haber, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratton, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramaswamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mcdermott, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benes, C. H.</span><span> </span><span class="NLM_article-title">Systematic identification of genomic markers of drug sensitivity in cancer cells</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">483</span><span class="NLM_x">, </span> <span class="NLM_fpage">570</span><span class="NLM_x">–</span> <span class="NLM_lpage">575</span><span class="refDoi"> DOI: 10.1038/nature11005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnature11005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=22460902" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVektL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=483&publication_year=2012&pages=570-575&author=M.+J.+Garnettauthor=E.+J.+Edelmanauthor=S.+J.+Heidornauthor=C.+D.+Greenmanauthor=A.+Dasturauthor=K.+W.+Lauauthor=P.+Greningerauthor=I.+R.+Thompsonauthor=X.+Luoauthor=J.+Soaresauthor=Q.+Liuauthor=F.+Iorioauthor=D.+Surdezauthor=L.+Chenauthor=R.+J.+Milanoauthor=G.+R.+Bignellauthor=A.+T.+Tamauthor=H.+Daviesauthor=J.+A.+Stevensonauthor=S.+Barthorpeauthor=S.+R.+Lutzauthor=F.+Kogeraauthor=K.+Lawrenceauthor=A.+Mclaren-Douglasauthor=X.+Mitropoulosauthor=T.+Mironenkoauthor=H.+Thiauthor=L.+Richardsonauthor=W.+Zhouauthor=F.+Jewittauthor=T.+Zhangauthor=P.+O%E2%80%99Brienauthor=J.+L.+Boisvertauthor=S.+Priceauthor=W.+Hurauthor=W.+Yangauthor=X.+Dengauthor=A.+Butlerauthor=H.+G.+Choiauthor=J.+W.+Changauthor=J.+Baselgaauthor=I.+Stamenkovicauthor=J.+A.+Engelmanauthor=S.+V.+Sharmaauthor=O.+Delattreauthor=J.+Saez-Rodriguezauthor=N.+S.+Grayauthor=J.+Settlemanauthor=P.+A.+Futrealauthor=D.+A.+Haberauthor=M.+R.+Strattonauthor=S.+Ramaswamyauthor=U.+Mcdermottauthor=C.+H.+Benes&title=Systematic+identification+of+genomic+markers+of+drug+sensitivity+in+cancer+cells&doi=10.1038%2Fnature11005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic identification of genomic markers of drug sensitivity in cancer cells</span></div><div class="casAuthors">Garnett, Mathew J.; Edelman, Elena J.; Heidorn, Sonja J.; Greenman, Chris D.; Dastur, Anahita; Lau, King Wai; Greninger, Patricia; Thompson, I. Richard; Luo, Xi; Soares, Jorge; Liu, Qingsong; Iorio, Francesco; Surdez, Didier; Chen, Li; Milano, Randy J.; Bignell, Graham R.; Tam, Ah T.; Davies, Helen; Stevenson, Jesse A.; Barthorpe, Syd; Lutz, Stephen R.; Kogera, Fiona; Lawrence, Karl; McLaren-Douglas, Anne; Mitropoulos, Xeni; Mironenko, Tatiana; Thi, Helen; Richardson, Laura; Zhou, Wenjun; Jewitt, Frances; Zhang, Tinghu; O'Brien, Patrick; Boisvert, Jessica L.; Price, Stacey; Hur, Wooyoung; Yang, Wanjuan; Deng, Xianming; Butler, Adam; Choi, Hwan Geun; Chang, Jae Won; Baselga, Jose; Stamenkovic, Ivan; Engelman, Jeffrey A.; Sharma, Sreenath V.; Delattre, Olivier; Saez-Rodriguez, Julio; Gray, Nathanael S.; Settleman, Jeffrey; Futreal, P. Andrew; Haber, Daniel A.; Stratton, Michael R.; Ramaswamy, Sridhar; McDermott, Ultan; Benes, Cyril H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">483</span>
        (<span class="NLM_cas:issue">7391</span>),
    <span class="NLM_cas:pages">570-575</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Clin. responses to anticancer therapies are often restricted to a subset of patients.  In some cases, mutated cancer genes are potent biomarkers for responses to targeted agents.  Here, to uncover new biomarkers of sensitivity and resistance to cancer therapeutics, we screened a panel of several hundred cancer cell lines-which represent much of the tissue-type and genetic diversity of human cancers-with 130 drugs under clin. and preclin. investigation.  In aggregate, we found that mutated cancer genes were assocd. with cellular response to most currently available cancer drugs.  Classic oncogene addiction paradigms were modified by addnl. tissue-specific or expression biomarkers, and some frequently mutated genes were assocd. with sensitivity to a broad range of therapeutic agents.  Unexpected relationships were revealed, including the marked sensitivity of Ewing's sarcoma cells harbouring the EWS (also known as EWSR1)-FLI1 gene translocation to poly(ADP-ribose) polymerase (PARP) inhibitors.  By linking drug activity to the functional complexity of cancer genomes, systematic pharmacogenomic profiling in cancer cell lines provides a powerful biomarker discovery platform to guide rational cancer therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3a_Jr2Bhx9bVg90H21EOLACvtfcHk0lh0ZN1z3fzYmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVektL4%253D&md5=540fc631fa43b55d0338fff4e46782b5</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnature11005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11005%26sid%3Dliteratum%253Aachs%26aulast%3DGarnett%26aufirst%3DM.%2BJ.%26aulast%3DEdelman%26aufirst%3DE.%2BJ.%26aulast%3DHeidorn%26aufirst%3DS.%2BJ.%26aulast%3DGreenman%26aufirst%3DC.%2BD.%26aulast%3DDastur%26aufirst%3DA.%26aulast%3DLau%26aufirst%3DK.%2BW.%26aulast%3DGreninger%26aufirst%3DP.%26aulast%3DThompson%26aufirst%3DI.%2BR.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DSoares%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DIorio%26aufirst%3DF.%26aulast%3DSurdez%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DMilano%26aufirst%3DR.%2BJ.%26aulast%3DBignell%26aufirst%3DG.%2BR.%26aulast%3DTam%26aufirst%3DA.%2BT.%26aulast%3DDavies%26aufirst%3DH.%26aulast%3DStevenson%26aufirst%3DJ.%2BA.%26aulast%3DBarthorpe%26aufirst%3DS.%26aulast%3DLutz%26aufirst%3DS.%2BR.%26aulast%3DKogera%26aufirst%3DF.%26aulast%3DLawrence%26aufirst%3DK.%26aulast%3DMclaren-Douglas%26aufirst%3DA.%26aulast%3DMitropoulos%26aufirst%3DX.%26aulast%3DMironenko%26aufirst%3DT.%26aulast%3DThi%26aufirst%3DH.%26aulast%3DRichardson%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DJewitt%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DP.%26aulast%3DBoisvert%26aufirst%3DJ.%2BL.%26aulast%3DPrice%26aufirst%3DS.%26aulast%3DHur%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DButler%26aufirst%3DA.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DChang%26aufirst%3DJ.%2BW.%26aulast%3DBaselga%26aufirst%3DJ.%26aulast%3DStamenkovic%26aufirst%3DI.%26aulast%3DEngelman%26aufirst%3DJ.%2BA.%26aulast%3DSharma%26aufirst%3DS.%2BV.%26aulast%3DDelattre%26aufirst%3DO.%26aulast%3DSaez-Rodriguez%26aufirst%3DJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DFutreal%26aufirst%3DP.%2BA.%26aulast%3DHaber%26aufirst%3DD.%2BA.%26aulast%3DStratton%26aufirst%3DM.%2BR.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DMcdermott%26aufirst%3DU.%26aulast%3DBenes%26aufirst%3DC.%2BH.%26atitle%3DSystematic%2520identification%2520of%2520genomic%2520markers%2520of%2520drug%2520sensitivity%2520in%2520cancer%2520cells%26jtitle%3DNature%26date%3D2012%26volume%3D483%26spage%3D570%26epage%3D575%26doi%3D10.1038%2Fnature11005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35a"><span><div class="note"><p class="first last">ATCC:</p></div><a href="http://www.lgcstandards-atcc.org/products/all/HTB-77.aspx?geo_country=gb" class="extLink">http://www.lgcstandards-atcc.org/products/all/HTB-77.aspx?geo_country=gb</a> (accessed August 31, 2016). For examples of using the SK-OV-3 cell-line in drug discovery, see:</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=ATCC%3Ahttp%3A%2F%2Fwww.lgcstandards-atcc.org%2Fproducts%2Fall%2FHTB-77.aspx%3Fgeo_country%3Dgb+%28accessed+August+31%2C+2016%29.+For+examples+of+using+the+SK-OV-3+cell-line+in+drug+discovery%2C+see%3A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit35a1"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Huang, R. Y-J</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuay, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, T. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asad, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, A. H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, V. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thiery, J. P.</span><span> </span><span class="NLM_article-title">Functional relevance of a six mesenchymal gene signature in epithelial-mesenchymal transition (EMT) reversal by the triple angiokinase inhibitor, nintedanib (BIBF1120)</span> <span class="citation_source-journal">Oncotarget</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">22098</span><span class="NLM_x">–</span> <span class="NLM_lpage">22113</span><span class="refDoi"> DOI: 10.18632/oncotarget.4300</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35a1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.18632%2Foncotarget.4300" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=22098-22113&author=R.+Y-J+Huangauthor=K.+T.+Kuayauthor=T.+Z.+Tanauthor=M.+Asadauthor=H.+M.+Tangauthor=A.+H.+C.+Ngauthor=J.+Yeauthor=V.+Y.+Chungauthor=J.+P.+Thiery&title=Functional+relevance+of+a+six+mesenchymal+gene+signature+in+epithelial-mesenchymal+transition+%28EMT%29+reversal+by+the+triple+angiokinase+inhibitor%2C+nintedanib+%28BIBF1120%29&doi=10.18632%2Foncotarget.4300"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35a1&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.4300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.4300%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DR.%2BY-J%26aulast%3DKuay%26aufirst%3DK.%2BT.%26aulast%3DTan%26aufirst%3DT.%2BZ.%26aulast%3DAsad%26aufirst%3DM.%26aulast%3DTang%26aufirst%3DH.%2BM.%26aulast%3DNg%26aufirst%3DA.%2BH.%2BC.%26aulast%3DYe%26aufirst%3DJ.%26aulast%3DChung%26aufirst%3DV.%2BY.%26aulast%3DThiery%26aufirst%3DJ.%2BP.%26atitle%3DFunctional%2520relevance%2520of%2520a%2520six%2520mesenchymal%2520gene%2520signature%2520in%2520epithelial-mesenchymal%2520transition%2520%2528EMT%2529%2520reversal%2520by%2520the%2520triple%2520angiokinase%2520inhibitor%252C%2520nintedanib%2520%2528BIBF1120%2529%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D22098%26epage%3D22113%26doi%3D10.18632%2Foncotarget.4300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit35b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shan, B.</span><span> </span><span class="NLM_article-title">Reversing drug resistance of cisplatin by HSP90 inhibitors in human ovarian cancer cells</span> <span class="citation_source-journal">Int. J. Clin. Exp. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">6687</span><span class="NLM_x">–</span> <span class="NLM_lpage">6701</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=26221207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsFektbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2015&pages=6687-6701&author=Z.+Zhangauthor=Z.+Xieauthor=G.+Sunauthor=P.+Yangauthor=J.+Liauthor=H.+Yangauthor=S.+Xiaoauthor=Y.+Liuauthor=H.+Qiuauthor=L.+Qinauthor=C.+Zhangauthor=F.+Zhangauthor=B.+Shan&title=Reversing+drug+resistance+of+cisplatin+by+HSP90+inhibitors+in+human+ovarian+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35bR"><div class="casContent"><span class="casTitleNuber">35b</span><div class="casTitle"><span class="NLM_cas:atitle">Reversing drug resistance of cisplatin by hsp90 inhibitors in human ovarian cancer cells</span></div><div class="casAuthors">Zhang, Zhengmao; Xie, Zhen; Sun, Guangyu; Yang, Pingfang; Li, Jia; Yang, Hongfang; Xiao, Shuang; Liu, Yang; Qiu, Hongbing; Qin, Lijun; Zhang, Chao; Zhang, Fenghua; Shan, Baoen</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Clinical and Experimental Medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">6687-6701</span>CODEN:
                <span class="NLM_cas:coden">IJCEFA</span>;
        ISSN:<span class="NLM_cas:issn">1940-5901</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">Objective: To investigate the mechanisms for reversing drug resistance of cisplatin (DDP) by Hsp90 inhibitors (geldanamycin (GA), 17-AAG, 17-DMAG) in human ovarian cancer.  Methods: Cell proliferation rate in DDP resistant human ovarian cancer cell line SKOV3/DDP and its parent cell line SKOV3 after treatment with Hsp90 inhibitors and/or DDP were tested by MTT assay, and the reversing fold (RF) of DDP by Hsp90 inhibitors was calcd.  Cell cycle and cell apoptosis status after treatment were analyzed by flow cytometry.  The expression of multiple drug resistance related genes was analyzed by RT-PCR and Western-blot.  Results: All three tested Hsp90 inhibitors synergistically inhibited the cell proliferation of SKOV3 with DDP and enhanced the sensitivity of SKOV3/DDP cells to DDP.  The RF of DDP by Hsp90 inhibitors were all more than two fold.  GA caused cell cycle arrest in G2/M phasein SKOV3 cells. 17-AAG increased cell apoptosis but did not change cell cycle in SKOV3/DDP cells.  The mRNA and protein expression levels of various drug resistant related genes including LRP, GST-π, p53, bcl-2, survivin, ERCC1, XRCC1, BRCA1 and BRCA2 were more dramatically altered by Hsp90 inhibitors and DDP in combination compared to Hsp90 inhibitors or DDP treatment alone.  Conclusions: Exposure of SKOV3/DDP cells to Hsp90 inhibitors and DDP in combination results in synergistic cytotoxic and pro-apoptotic effects.  Hsp90 inhibitors reverse the drug resistance of SKOV3/DDP cells to DDP by modifying the expression of multiple drug resistance related genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMXw59WByT0LVg90H21EOLACvtfcHk0ljf8n-YhEu9Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsFektbs%253D&md5=e49594173796f33dc082132a29ebd90b</span></div><a href="/servlet/linkout?suffix=cit35b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DXie%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DXiao%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DQiu%26aufirst%3DH.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DShan%26aufirst%3DB.%26atitle%3DReversing%2520drug%2520resistance%2520of%2520cisplatin%2520by%2520HSP90%2520inhibitors%2520in%2520human%2520ovarian%2520cancer%2520cells%26jtitle%3DInt.%2520J.%2520Clin.%2520Exp.%2520Med.%26date%3D2015%26volume%3D8%26spage%3D6687%26epage%3D6701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit35c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Wu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, L.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, M.-J.</span><span> </span><span class="NLM_article-title">The inhibitory effect of doxycycline on cisplatin-sensitive and -resistant epithelial ovarian cancer</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e89841</span><span class="refDoi"> DOI: 10.1371/journal.pone.0089841</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1371%2Fjournal.pone.0089841" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e89841&author=W.+Wuauthor=L.-H.+Yuauthor=B.+Maauthor=M.-J.+Xu&title=The+inhibitory+effect+of+doxycycline+on+cisplatin-sensitive+and+-resistant+epithelial+ovarian+cancer&doi=10.1371%2Fjournal.pone.0089841"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35c&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0089841&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0089841%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DL.-H.%26aulast%3DMa%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DM.-J.%26atitle%3DThe%2520inhibitory%2520effect%2520of%2520doxycycline%2520on%2520cisplatin-sensitive%2520and%2520-resistant%2520epithelial%2520ovarian%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De89841%26doi%3D10.1371%2Fjournal.pone.0089841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit35d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Fekete, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santiskulvong, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorigo, O.</span><span> </span><span class="NLM_article-title">Effect of PI3K/AKT pathway inhibition-mediated G1 arrest on chemosensitization in ovarian cancer cells</span> <span class="citation_source-journal">Anticancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">445</span><span class="NLM_x">–</span> <span class="NLM_lpage">452</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=22287731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38XjsF2ltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=445-452&author=M.+Feketeauthor=C.+Santiskulvongauthor=C.+Engauthor=O.+Dorigo&title=Effect+of+PI3K%2FAKT+pathway+inhibition-mediated+G1+arrest+on+chemosensitization+in+ovarian+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35dR"><div class="casContent"><span class="casTitleNuber">35d</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of PI3K/Akt pathway inhibition-mediated G1 arrest on chemosensitization in ovarian cancer cells</span></div><div class="casAuthors">Fekete, Mirela; Santiskulvong, Chintda; Eng, Carol; Dorigo, Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Anticancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">445-452</span>CODEN:
                <span class="NLM_cas:coden">ANTRD4</span>;
        ISSN:<span class="NLM_cas:issn">0250-7005</span>.
    
            (<span class="NLM_cas:orgname">International Institute of Anticancer Research</span>)
        </div><div class="casAbstract">Background: Pharmacol. inhibition of the phosphoinositide 3-kinase (PI3K)/Akt pathway prevents G1 cell cycle progression into S, resulting in G1 accumulation.  The hypothesis that this arrest might neg. impact on chemotherapeutic agents primarily effective in S, G2 or M-phase was investigated.  Materials and Methods: Inhibition of PI3K/Akt pathway signaling via LY294002 and Akti-1/2 was demonstrated by immunoblotting.  Cell cycle progression was detd. by flow cytometric anal.  Cell proliferation was assayed using the XTT cell viability assay.  The Chou and Talalay median effect principal was used to evaluate drug interaction.  Results: In SKOV3 and IGROV1 human ovarian cancer cells, LY294002 and Akti-1/2 increased the percentage of cells in G1 and reversed the cell cycle effects of cisplatin, paclitaxel, gemcitabine and topotecan.  Pathway blockade synergistically enhanced the cytotoxicity of cisplatin and paclitaxel, but antagonized gemcitabine and topotecan effects.  Conclusion: Pharmacol. PI3K/Akt inhibition antagonizes the efficacy of chemotherapeutic agents primarily effective in the S or G2-phase of the cell cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwbwYDK0hUlLVg90H21EOLACvtfcHk0lgRV52S08jDHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjsF2ltLg%253D&md5=d3e97952622e3271ddf47f02ea052c14</span></div><a href="/servlet/linkout?suffix=cit35d&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFekete%26aufirst%3DM.%26aulast%3DSantiskulvong%26aufirst%3DC.%26aulast%3DEng%26aufirst%3DC.%26aulast%3DDorigo%26aufirst%3DO.%26atitle%3DEffect%2520of%2520PI3K%252FAKT%2520pathway%2520inhibition-mediated%2520G1%2520arrest%2520on%2520chemosensitization%2520in%2520ovarian%2520cancer%2520cells%26jtitle%3DAnticancer%2520Res.%26date%3D2012%26volume%3D32%26spage%3D445%26epage%3D452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit35e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Tan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanizaki, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Z. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aref, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ercan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liao, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelletti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hur, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sim, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaudet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbie, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeh, J. R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yun, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammerman, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohammadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Development of covalent inhibitors that can overcome resistance to first-generation FGFR kinase inhibitors</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">E4869</span><span class="NLM_x">–</span> <span class="NLM_lpage">E4877</span><span class="refDoi"> DOI: 10.1073/pnas.1403438111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1073%2Fpnas.1403438111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=25349422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWmsL%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=E4869-E4877&author=L.+Tanauthor=J.+Wangauthor=J.+Tanizakiauthor=Z.+F.+Huangauthor=A.+R.+Arefauthor=M.+Rusanauthor=S.+J.+Zhuauthor=Y.+Y.+Zhangauthor=D.+Ercanauthor=R.+G.+Liaoauthor=M.+Capellettiauthor=W.+J.+Zhouauthor=W.+Hurauthor=N.+Kimauthor=T.+Simauthor=S.+Gaudetauthor=D.+A.+Barbieauthor=J.+R.+J.+Yehauthor=C.+H.+Yunauthor=P.+S.+Hammermanauthor=M.+Mohammadiauthor=P.+A.+Janneauthor=N.+S.+Gray&title=Development+of+covalent+inhibitors+that+can+overcome+resistance+to+first-generation+FGFR+kinase+inhibitors&doi=10.1073%2Fpnas.1403438111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35eR"><div class="casContent"><span class="casTitleNuber">35e</span><div class="casTitle"><span class="NLM_cas:atitle">Development of covalent inhibitors that can overcome resistance to first-generation FGFR kinase inhibitors</span></div><div class="casAuthors">Tan, Li; Wang, Jun; Tanizaki, Junko; Huang, Zhifeng; Aref, Amir R.; Rusan, Maria; Zhu, Su-Jie; Zhang, Yiyun; Ercan, Dalia; Liao, Rachel G.; Capelletti, Marzia; Zhou, Wenjun; Hur, Wooyoung; Kim, NamDoo; Sim, Taebo; Gaudet, Suzanne; Barbie, David A.; Yeh, Jing-Ruey Joanna; Yun, Cai-Hong; Hammerman, Peter S.; Mohammadi, Moosa; Janne, Pasi A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">E4869-E4877</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The human FGF receptors (FGFRs) play crit. roles in various human cancers, and several FGFR inhibitors are currently under clin. investigation.  Resistance usually results from selection for mutant kinases that are impervious to the action of the drug or from up-regulation of compensatory signaling pathways.  Preclin. studies have demonstrated that resistance to FGFR inhibitors can be acquired through mutations in the FGFR gatekeeper residue, as clin. obsd. for FGFR4 in embryonal rhabdomyosarcoma and neuroendocrine breast carcinomas.  Here the authors report on the use of a structure-based drug design to develop two selective, next-generation covalent FGFR inhibitors, the FGFR irreversible inhibitors I and II.  To our knowledge, I and II are the first inhibitors that can potently inhibit the proliferation of cells dependent upon the gatekeeper mutants of FGFR1 or FGFR2, which confer resistance to first-generation clin. FGFR inhibitors such as NVP-BGJ398 and AZD4547.  Because of the conformational flexibility of the reactive acrylamide substituent, II has the unprecedented ability to inhibit both the EGF receptor (EGFR) and FGFR covalently by targeting two distinct cysteine residues.  The authors report the cocrystal structure of FGFR4 with I, which unexpectedly exhibits a "DFG-out" covalent binding mode.  The structural basis for dual FGFR and EGFR targeting by FIIN3 also is illustrated by crystal structures of II bound with FGFR4 V550L and EGFR L858R.  These results have important implications for the design of covalent FGFR inhibitors that can overcome clin. resistance and provide the first example, to the knowledge, of a kinase inhibitor that covalently targets cysteines located in different positions within the ATP-binding pocket.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmCbTDKPA32LVg90H21EOLACvtfcHk0lgRV52S08jDHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWmsL%252FP&md5=fd4a7eac1e0c17e7c9f2651dd2d76ac7</span></div><a href="/servlet/linkout?suffix=cit35e&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1403438111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1403438111%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DTanizaki%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DZ.%2BF.%26aulast%3DAref%26aufirst%3DA.%2BR.%26aulast%3DRusan%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DS.%2BJ.%26aulast%3DZhang%26aufirst%3DY.%2BY.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DLiao%26aufirst%3DR.%2BG.%26aulast%3DCapelletti%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DW.%2BJ.%26aulast%3DHur%26aufirst%3DW.%26aulast%3DKim%26aufirst%3DN.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DGaudet%26aufirst%3DS.%26aulast%3DBarbie%26aufirst%3DD.%2BA.%26aulast%3DYeh%26aufirst%3DJ.%2BR.%2BJ.%26aulast%3DYun%26aufirst%3DC.%2BH.%26aulast%3DHammerman%26aufirst%3DP.%2BS.%26aulast%3DMohammadi%26aufirst%3DM.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDevelopment%2520of%2520covalent%2520inhibitors%2520that%2520can%2520overcome%2520resistance%2520to%2520first-generation%2520FGFR%2520kinase%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3DE4869%26epage%3DE4877%26doi%3D10.1073%2Fpnas.1403438111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="note"><p class="first last">For examples of SK-OV-3 murine xenograft studies, see:</p></div><div class="NLM_citation" id="cit36a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Shaw, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senterman, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dawson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crane, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanderhyden, B. C.</span><span> </span><span class="NLM_article-title">Characterization of intraperitoneal, orthotopic, and metastatic xenograft models of human ovarian cancer</span> <span class="citation_source-journal">Mol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1032</span><span class="NLM_x">–</span> <span class="NLM_lpage">1042</span><span class="refDoi"> DOI: 10.1016/j.ymthe.2004.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.ymthe.2004.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=15564135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVSntrfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=1032-1042&author=T.+J.+Shawauthor=M.+K.+Sentermanauthor=K.+Dawsonauthor=C.+A.+Craneauthor=B.+C.+Vanderhyden&title=Characterization+of+intraperitoneal%2C+orthotopic%2C+and+metastatic+xenograft+models+of+human+ovarian+cancer&doi=10.1016%2Fj.ymthe.2004.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36aR"><div class="casContent"><span class="casTitleNuber">36a</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of intraperitoneal, orthotopic, and metastatic xenograft models of human ovarian cancer</span></div><div class="casAuthors">Shaw, Tanya J.; Senterman, Mary K.; Dawson, Kerri; Crane, Colleen A.; Vanderhyden, Barbara C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1032-1042</span>CODEN:
                <span class="NLM_cas:coden">MTOHCK</span>;
        ISSN:<span class="NLM_cas:issn">1525-0016</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Improvement of ovarian cancer patient outcome requires well-characterized animal models in which to evaluate novel therapeutics.  Xenograft models are frequently used, but with little discussion of disease histol.  The objectives of this study were to inject 11 ovarian cancer cell lines i.p., and a subset intrabursally (ib; orthotopic), into nude mice and to analyze the resulting pathologies.  Eight of 11 lines injected i.p. formed tumors within 3 mo at variable rates with the following histol. subtype distribution: one endometrioid, one serous, one clear cell, and five undifferentiated.  Only mice injected with A2780-cp cells presented with ovarian-specific metastases (11 of 88), and the survival time of these animals was significantly shorter, which may be attributed to the higher proliferation rate as detd. by Ki67 positivity.  Addnl. anal. of the influence of the ovarian microenvironment on cell characteristics was conducted with ib injection of two cell lines (OVCA 429 and ES-2).  The site of injection did not affect the tumor histol., the effect on proliferation was cell-type dependent, and the tumor take rate (cell survival) was neg. affected for OVCA 429 cells.  The animal models described herein represent histol. distinct models of both early and late stage ovarian cancer useful for evaluation of therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHXUVG2I7lorVg90H21EOLACvtfcHk0lg8JeNN40HvJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVSntrfM&md5=56735d8bc8f748afa5c2905ddb923a02</span></div><a href="/servlet/linkout?suffix=cit36a&amp;dbid=16384&amp;doi=10.1016%2Fj.ymthe.2004.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymthe.2004.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DShaw%26aufirst%3DT.%2BJ.%26aulast%3DSenterman%26aufirst%3DM.%2BK.%26aulast%3DDawson%26aufirst%3DK.%26aulast%3DCrane%26aufirst%3DC.%2BA.%26aulast%3DVanderhyden%26aufirst%3DB.%2BC.%26atitle%3DCharacterization%2520of%2520intraperitoneal%252C%2520orthotopic%252C%2520and%2520metastatic%2520xenograft%2520models%2520of%2520human%2520ovarian%2520cancer%26jtitle%3DMol.%2520Ther.%26date%3D2004%26volume%3D10%26spage%3D1032%26epage%3D1042%26doi%3D10.1016%2Fj.ymthe.2004.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit36b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Faratian, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zweemer, A. J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagumo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sims, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muir, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodds, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mullen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Um, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kay, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harrison, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langdon, S. P.</span><span> </span><span class="NLM_article-title">Trastuzumab and pertuzumab produce changes in morphology and estrogen receptor signaling in ovarian cancer xenografts revealing new treatment strategies</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">4451</span><span class="NLM_x">–</span> <span class="NLM_lpage">4461</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-10-2461</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1158%2F1078-0432.CCR-10-2461" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=21571868" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXos1SmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=4451-4461&author=D.+Faratianauthor=A.+J.+M.+Zweemerauthor=Y.+Nagumoauthor=A.+H.+Simsauthor=M.+Muirauthor=M.+Doddsauthor=P.+Mullenauthor=I.+Umauthor=C.+Kayauthor=M.+Hasmannauthor=D.+J.+Harrisonauthor=S.+P.+Langdon&title=Trastuzumab+and+pertuzumab+produce+changes+in+morphology+and+estrogen+receptor+signaling+in+ovarian+cancer+xenografts+revealing+new+treatment+strategies&doi=10.1158%2F1078-0432.CCR-10-2461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36bR"><div class="casContent"><span class="casTitleNuber">36b</span><div class="casTitle"><span class="NLM_cas:atitle">Trastuzumab and Pertuzumab Produce Changes in Morphology and Estrogen Receptor Signaling in Ovarian Cancer Xenografts Revealing New Treatment Strategies</span></div><div class="casAuthors">Faratian, Dana; Zweemer, Annelien J. M.; Nagumo, Yoko; Sims, Andrew H.; Muir, Morwenna; Dodds, Michael; Mullen, Peter; Um, Inhwa; Kay, Charlene; Hasmann, Max; Harrison, David J.; Langdon, Simon P.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4451-4461</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: The aim of this study was to investigate the antitumor effects of HER2-directed combination therapy in ovarian cancer xenograft models to evaluate their potential.  The combinations of trastuzumab and pertuzumab, and trastuzumab and aromatase inhibitor therapy were investigated.  Exptl. Design: The effects of trastuzumab, pertuzumab, and letrozole on growth response, apoptosis, morphol., and gene and protein expression were evaluated in the SKOV3 ovarian cancer cell line xenograft and a panel of five human ovarian xenografts derived directly from clin. specimens.  RESULTS: The combination of HER2-directed antibodies showed enhanced antitumor activity compared with single antibody therapy in the SKOV3 xenograft model.  Apoptosis, morphol., and estrogen-regulated gene expression were modulated by these antibodies in both spatial and temporal manners.  A panel of ovarian cancer xenografts showed differential growth responses to the combination of trastuzumab and pertuzumab.  High HER2 expression and increasing HER3 protein expression on treatment were assocd. with growth response.  In trastuzumab-treated SKOV3 tumors, there was a change in tumor morphol., with a redn. in frequency of estrogen receptor alpha (ERα)-neg. clear cell areas.  Trastuzumab, but not pertuzumab, increased expression of ERα in SKOV3 xenografts when analyzed by quant. immunofluorescence.  ERα and downstream signaling targets were modulated by trastuzumab alone and in combination.  Trastuzumab enhanced the responsiveness of SKOV3 xenografts to letrozole when given in combination.  CONCLUSIONS: These data suggest that trastuzumab in combination with pertuzumab could be an effective approach in high HER2-expressing ovarian cancers and could also enhance sensitivity to endocrine therapy in ERα-pos. ovarian cancer.  Clin Cancer Res; 17(13); 4451-61.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmH-TKtfP9sbVg90H21EOLACvtfcHk0lg8JeNN40HvJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXos1SmtLs%253D&md5=8a94e84f7fd91bddb63e17c9efee130b</span></div><a href="/servlet/linkout?suffix=cit36b&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-10-2461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-10-2461%26sid%3Dliteratum%253Aachs%26aulast%3DFaratian%26aufirst%3DD.%26aulast%3DZweemer%26aufirst%3DA.%2BJ.%2BM.%26aulast%3DNagumo%26aufirst%3DY.%26aulast%3DSims%26aufirst%3DA.%2BH.%26aulast%3DMuir%26aufirst%3DM.%26aulast%3DDodds%26aufirst%3DM.%26aulast%3DMullen%26aufirst%3DP.%26aulast%3DUm%26aufirst%3DI.%26aulast%3DKay%26aufirst%3DC.%26aulast%3DHasmann%26aufirst%3DM.%26aulast%3DHarrison%26aufirst%3DD.%2BJ.%26aulast%3DLangdon%26aufirst%3DS.%2BP.%26atitle%3DTrastuzumab%2520and%2520pertuzumab%2520produce%2520changes%2520in%2520morphology%2520and%2520estrogen%2520receptor%2520signaling%2520in%2520ovarian%2520cancer%2520xenografts%2520revealing%2520new%2520treatment%2520strategies%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2011%26volume%3D17%26spage%3D4451%26epage%3D4461%26doi%3D10.1158%2F1078-0432.CCR-10-2461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit36c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Zhang, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bing, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Z. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Q.</span><span> </span><span class="NLM_article-title">Augmented inhibition of angiogenesis by combination of HER2 antibody ChA21 and trastuzumab in human ovarian carcinoma xenograft</span> <span class="citation_source-journal">J. Ovarian Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">20</span><span class="refDoi"> DOI: 10.1186/1757-2215-3-20</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1186%2F1757-2215-3-20" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=20723224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A280%3ADC%252BC3cfitVSqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2010&pages=20&author=A.+L.+Zhangauthor=G.+D.+Shenauthor=T.+Zhaoauthor=G.+H.+Zhangauthor=J.+Liuauthor=L.+H.+Songauthor=W.+Weiauthor=L.+Bingauthor=Z.+S.+Wuauthor=Q.+Wu&title=Augmented+inhibition+of+angiogenesis+by+combination+of+HER2+antibody+ChA21+and+trastuzumab+in+human+ovarian+carcinoma+xenograft&doi=10.1186%2F1757-2215-3-20"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36cR"><div class="casContent"><span class="casTitleNuber">36c</span><div class="casTitle"><span class="NLM_cas:atitle">Augmented inhibition of angiogenesis by combination of HER2 antibody chA21 and trastuzumab in human ovarian carcinoma xenograft</span></div><div class="casAuthors">Zhang Anli; Shen Guodong; Zhao Ting; Zhang Guihong; Liu Jing; Song Lihua; Wei Wei; Bing Ling; Wu Zhengsheng; Wu Qiang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of ovarian research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">20</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  chA21 is a novel tumor-inhibitory antibody which recognized subdomain I of HER2 extracellular domain with an epitope distinct from other HER2 antibodies.  Previously, we demonstrated that chA21 inhibits human ovarian carcinoma cell line SKOV-3 growth in vitro and in vivo study.  In this study, we further investigated the anti-angiogenic efficacy combination of chA21 with trastuzumab in SKOV-3 xenograft model.  METHODS:  Nude mice were s.c. challenged with SKOV-3 cells and received treatment of chA21 alone, trastuzumab alone or both antibodies together twice a week for 21 days.  Tumor volume and microvessel density (MVD) were evaluated.  The effect of chA21 plus trastuzumab treament on vascular endothelial growth factor (VEGF) secretion, endothelial cells proliferation and migration, and the status of HER2 downstream pathway AKT/phosphorylated AKT (pAKT) were evaluated in vitro.  RESULTS:  In vivo study combination of chA21 with trastuzumab resulted in reduce tumor growth and angiogenesis than each monotherapy.  In vitro study, the combination of chA21 with trastuzumab inhibits VEGF secretion, endothelial cells proliferation and migration.  Furthermore, the combination treatment inhibits pAKT expression.  CONCLUSION:  Our findings suggested that the combination of chA21 with trastuzumab can cause augmented inhibition of angiogenesis in SKOV-3 xenograft model.  Inhibition of agniogenesis may through suppression of AKT pathway.  The therapeutic benefits of combination chA21 with trastuzumab warrant further study in an attempt to make the translation into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQyFo7VwIDPg3l8icjszYl0fW6udTcc2ebY1RepTId-y7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cfitVSqug%253D%253D&md5=be2d626dd3ce612db75ba8dc25597223</span></div><a href="/servlet/linkout?suffix=cit36c&amp;dbid=16384&amp;doi=10.1186%2F1757-2215-3-20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1757-2215-3-20%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DA.%2BL.%26aulast%3DShen%26aufirst%3DG.%2BD.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DG.%2BH.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DL.%2BH.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DBing%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DZ.%2BS.%26aulast%3DWu%26aufirst%3DQ.%26atitle%3DAugmented%2520inhibition%2520of%2520angiogenesis%2520by%2520combination%2520of%2520HER2%2520antibody%2520ChA21%2520and%2520trastuzumab%2520in%2520human%2520ovarian%2520carcinoma%2520xenograft%26jtitle%3DJ.%2520Ovarian%2520Res.%26date%3D2010%26volume%3D3%26spage%3D20%26doi%3D10.1186%2F1757-2215-3-20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit36d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Li, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ji, Z.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhuo, G.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, R.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.-R.</span><span> </span><span class="NLM_article-title">Effects of a selective cyclooxygenase-1 inhibitor in SKOV-3 ovarian carcinoma xenograft-bearing mice</span> <span class="citation_source-journal">Med. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">98</span><span class="NLM_x">–</span> <span class="NLM_lpage">104</span><span class="refDoi"> DOI: 10.1007/s12032-009-9179-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1007%2Fs12032-009-9179-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=19235530" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXis1CmtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2010&pages=98-104&author=W.+Liauthor=Z.-L.+Jiauthor=G.-C.+Zhuoauthor=R.-J.+Xuauthor=J.+Wangauthor=H.-R.+Jiang&title=Effects+of+a+selective+cyclooxygenase-1+inhibitor+in+SKOV-3+ovarian+carcinoma+xenograft-bearing+mice&doi=10.1007%2Fs12032-009-9179-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36dR"><div class="casContent"><span class="casTitleNuber">36d</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of a selective cyclooxygenase-1 inhibitor in SKOV-3 ovarian carcinoma xenograft-bearing mice</span></div><div class="casAuthors">Li, Wei; Ji, Zhong-lei; Zhuo, Guang-chao; Xu, Ru-jun; Wang, Jie; Jiang, Hong-ru</div><div class="citationInfo"><span class="NLM_cas:title">Medical Oncology (Totowa, NJ, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">98-104</span>CODEN:
                <span class="NLM_cas:coden">MONCEZ</span>;
        ISSN:<span class="NLM_cas:issn">1357-0560</span>.
    
            (<span class="NLM_cas:orgname">Humana Press Inc.</span>)
        </div><div class="casAbstract">To evaluate the effect of a cyclooxygenase-1 (COX-1) inhibitor, SC-560, on the growth inhibition of s.c. human ovarian SKOV-3 carcinoma and on angiogenesis.  Human ovarian SKOV-3 carcinoma cells xenograft-bearing mice were treated with SC-560, a COX-1-selective inhibitor, 6 mg/kg alone i.g. daily, and i.p. injections of cisplatin 3 mg/kg every other day for 21 days.  Prostaglandin E2 (PGE2) levels were detd. by ELISA.  Microvessel d. (MVD) of ovarian carcinoma was detd. with anti-CD34 as the label by immunohistochem.  In addn., the expression of COX-1 at protein levels in the control group was detected by immunohistochem.  SC-560 reduced the growth of tumors when SKOV-3 cells were xenografted in nude female mice.  The inhibitory rates in SC-560 group and cisplatin group were 47.1% and 51.7%, resp., which is significant statistically compared to that of control group (all, P < 0.05).  In treatment groups, SC-560 significantly reduced intratumor PGE2 levels (P < 0.01).  MVDs in SC-560 group were 35.73 ± 9.87, which are significant statistically compared to that of control group (74.33 ± 9.50) (P < 0.01).  COX-1, not COX-2, protein levels are elevated in tumor tissues.  These findings may implicate COX-1 as a suitable target for the treatment of ovarian cancer and that antiangiogenic therapy can be used to inhibit ovarian cancer growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk6AnKHPpTlbVg90H21EOLACvtfcHk0lihLMKOboDpow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXis1CmtLc%253D&md5=d6bbd61aad82cfd68cdc4b45a0c9a769</span></div><a href="/servlet/linkout?suffix=cit36d&amp;dbid=16384&amp;doi=10.1007%2Fs12032-009-9179-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12032-009-9179-y%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DW.%26aulast%3DJi%26aufirst%3DZ.-L.%26aulast%3DZhuo%26aufirst%3DG.-C.%26aulast%3DXu%26aufirst%3DR.-J.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DH.-R.%26atitle%3DEffects%2520of%2520a%2520selective%2520cyclooxygenase-1%2520inhibitor%2520in%2520SKOV-3%2520ovarian%2520carcinoma%2520xenograft-bearing%2520mice%26jtitle%3DMed.%2520Oncol.%26date%3D2010%26volume%3D27%26spage%3D98%26epage%3D104%26doi%3D10.1007%2Fs12032-009-9179-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit36e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Bauerschlag, D. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schem, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiwari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egberts, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weigel, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalthoff, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jonat, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maass, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meinhold-Heerlein, I.</span><span> </span><span class="NLM_article-title">Sunitinib (SU11248) inhibits growth of human ovarian cancer in xenografted mice</span> <span class="citation_source-journal">Anticancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">3355</span><span class="NLM_x">–</span> <span class="NLM_lpage">3360</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=20944108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlGksbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2010&pages=3355-3360&author=D.+O.+Bauerschlagauthor=C.+Schemauthor=S.+Tiwariauthor=J.+H.+Egbertsauthor=M.+T.+Weigelauthor=H.+Kalthoffauthor=W.+Jonatauthor=N.+Maassauthor=I.+Meinhold-Heerlein&title=Sunitinib+%28SU11248%29+inhibits+growth+of+human+ovarian+cancer+in+xenografted+mice"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36eR"><div class="casContent"><span class="casTitleNuber">36e</span><div class="casTitle"><span class="NLM_cas:atitle">Sunitinib (SU11248) inhibits growth of human ovarian cancer in xenografted mice</span></div><div class="casAuthors">Bauerschlag, Dirk O.; Schem, Christian; Tiwari, Sanjay; Egberts, Jan H.; Weigel, Marion T.; Kalthoff, Holger; Jonat, Walter; Maass, Nicolai; Meinhold-Heerlein, Ivo</div><div class="citationInfo"><span class="NLM_cas:title">Anticancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3355-3360</span>CODEN:
                <span class="NLM_cas:coden">ANTRD4</span>;
        ISSN:<span class="NLM_cas:issn">0250-7005</span>.
    
            (<span class="NLM_cas:orgname">International Institute of Anticancer Research</span>)
        </div><div class="casAbstract">Background: Treatment of ovarian cancer is still challenging esp. in recurrent platinum refractory cases.  Sunitinib is a multi tyrosine kinase inhibitor targeting receptors for vascular endothelial growth factor and platelet-derived growth factor which play a role in tumor angiogenesis.  It has been approved for the treatment of recurrent gastro intestinal stroma tumors and metastatic renal cancer.  Materials and Methods: In this study, sunitinib was tested for its effectiveness as a single agent in an ovarian cancer xenograft mouse model.  Skov3 cells stably expressing firefly luciferase were injected into SCID beige mice.  Mice received either 40 mg/kg bodyweight sunitinib or vehicle control.  Tumor growth was monitored longitudinally by luciferase signal.  Results: Sunitinib significantly reduced tumor growth (p = 0.0052) and peritoneal metastases, and was assocd. with a significantly reduced microvessel d. count (p<0.001).  Conclusion: These results suggest that clin. trials are warranted for the evaluation of sunitinib for treatment of patients with recurrent or advanced ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAP9MVZ5R-JLVg90H21EOLACvtfcHk0lihLMKOboDpow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlGksbvJ&md5=418110b00488e2629b8b30d25f36053f</span></div><a href="/servlet/linkout?suffix=cit36e&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBauerschlag%26aufirst%3DD.%2BO.%26aulast%3DSchem%26aufirst%3DC.%26aulast%3DTiwari%26aufirst%3DS.%26aulast%3DEgberts%26aufirst%3DJ.%2BH.%26aulast%3DWeigel%26aufirst%3DM.%2BT.%26aulast%3DKalthoff%26aufirst%3DH.%26aulast%3DJonat%26aufirst%3DW.%26aulast%3DMaass%26aufirst%3DN.%26aulast%3DMeinhold-Heerlein%26aufirst%3DI.%26atitle%3DSunitinib%2520%2528SU11248%2529%2520inhibits%2520growth%2520of%2520human%2520ovarian%2520cancer%2520in%2520xenografted%2520mice%26jtitle%3DAnticancer%2520Res.%26date%3D2010%26volume%3D30%26spage%3D3355%26epage%3D3360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Obach, R. S.</span><span> </span><span class="NLM_article-title">Prediction of human clearance of twenty-nine drugs from hepatic microsomal intrinsic clearance data: an examination of in vitro half-life approach and nonspecific binding to microsomes</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">1350</span><span class="NLM_x">–</span> <span class="NLM_lpage">1359</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10534321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=1350-1359&author=R.+S.+Obach&title=Prediction+of+human+clearance+of+twenty-nine+drugs+from+hepatic+microsomal+intrinsic+clearance+data%3A+an+examination+of+in+vitro+half-life+approach+and+nonspecific+binding+to+microsomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of human clearance of twenty-nine drugs from hepatic microsomal intrinsic clearance data: an examination of in vitro half-life approach and nonspecific binding to microsomes</span></div><div class="casAuthors">Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1350-1359</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Twenty-nine drugs of disparate structures and physicochem. properties were used in an examn. of the capability of human liver microsomal lability data ("in vitro T1/2" approach) to be useful in the prediction of human clearance.  Addnl., the potential importance of nonspecific binding to microsomes in the in vitro incubation milieu for the accurate prediction of human clearance was investigated.  The compds. examd. demonstrated a wide range of microsomal metabolic labilities with scaled intrinsic clearance values ranging from less than 0.5 mL/min/kg to 189 mL/min/kg.  Microsomal binding was detd. at microsomal protein concns. used in the lability incubations.  For the 29 compds. studied, unbound fractions in microsomes ranged from 0.11 to 1.0.  Generally, basic compds. demonstrated the greatest extent of binding and neutral and acidic compds. the least extent of binding.  In the projection of human clearance values, basic and neutral compds. were well predicted when all binding considerations (blood and microsome) were disregarded, however, including both binding considerations also yielded reasonable predictions.  Including only blood binding yielded very poor projections of human clearance for these two types of compds.  However, for acidic compds., disregarding all binding considerations yielded poor predictions of human clearance.  It was generally most difficult to accurately predict clearance for this class of compds.; however the accuracy was best when all binding considerations were included.  Overall, inclusion of both blood and microsome binding values gave the best agreement between in vivo clearance values and clearance values projected from in vitro intrinsic clearance data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWp2hUTOxyubVg90H21EOLACvtfcHk0lihLMKOboDpow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D&md5=7804cd4dfb131caac45aff9b7a1df901</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DPrediction%2520of%2520human%2520clearance%2520of%2520twenty-nine%2520drugs%2520from%2520hepatic%2520microsomal%2520intrinsic%2520clearance%2520data%253A%2520an%2520examination%2520of%2520in%2520vitro%2520half-life%2520approach%2520and%2520nonspecific%2520binding%2520to%2520microsomes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1999%26volume%3D27%26spage%3D1350%26epage%3D1359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><div class="note"><p class="first last">The in-house HPLC method to measure Log<i>D</i><sub>7.4</sub> is based on:</p></div><span class="NLM_contrib-group">Kerns, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petusky, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kleintop, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huryn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McConnell, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, G.</span><span> </span><span class="NLM_article-title">Pharmaceutical profiling method for lipophilicity and integrity using liquid chromatography-mass spectrometry</span> <span class="citation_source-journal">J. Chromatogr. B: Anal. Technol. Biomed. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">791</span><span class="NLM_x">, </span> <span class="NLM_fpage">381</span><span class="NLM_x">–</span> <span class="NLM_lpage">388</span><span class="refDoi"> DOI: 10.1016/S1570-0232(03)00250-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2FS1570-0232%2803%2900250-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=12798198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkt1Ors7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=791&publication_year=2003&pages=381-388&author=E.+H.+Kernsauthor=L.+Diauthor=S.+Petuskyauthor=T.+Kleintopauthor=D.+Hurynauthor=O.+McConnellauthor=G.+Carter&title=Pharmaceutical+profiling+method+for+lipophilicity+and+integrity+using+liquid+chromatography-mass+spectrometry&doi=10.1016%2FS1570-0232%2803%2900250-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmaceutical profiling method for lipophilicity and integrity using liquid chromatography-mass spectrometry</span></div><div class="casAuthors">Kerns, Edward H.; Di, Li; Petusky, Susan; Kleintop, Teresa; Huryn, Donna; McConnell, Oliver; Carter, Guy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chromatography B: Analytical Technologies in the Biomedical and Life Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">791</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">381-388</span>CODEN:
                <span class="NLM_cas:coden">JCBAAI</span>;
        ISSN:<span class="NLM_cas:issn">1570-0232</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A method for the simultaneous profiling of sample lipophilicity, integrity, and purity was described.  The method is rapid and is applicable to high throughput profiling of pharmaceutical properties in drug discovery.  A short Polaris C18 column is used with a rapid, wide-polarity mobile phase gradient, UV detection, and MS anal.  The lipophilicity of each component is estd. from a calibration curve using six drug or org. compds. and plotting their resp. measured retention time vs. Log D7.4 (literature).  The correlation of Log D7.4 (literature) to Log D7.4 (HPLC) for 60 structurally diverse drugs has a correlation coeff. r2 of 0.89.  The method is applicable to compds. with MW>200 and retention time>1.5 min for rapid, initial pharmaceutical profiling in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1NzfnRCy_z7Vg90H21EOLACvtfcHk0ljjfgUbIA7dlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkt1Ors7k%253D&md5=8b20a63df17d424f0e2406fee659e8ea</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2FS1570-0232%2803%2900250-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1570-0232%252803%252900250-2%26sid%3Dliteratum%253Aachs%26aulast%3DKerns%26aufirst%3DE.%2BH.%26aulast%3DDi%26aufirst%3DL.%26aulast%3DPetusky%26aufirst%3DS.%26aulast%3DKleintop%26aufirst%3DT.%26aulast%3DHuryn%26aufirst%3DD.%26aulast%3DMcConnell%26aufirst%3DO.%26aulast%3DCarter%26aufirst%3DG.%26atitle%3DPharmaceutical%2520profiling%2520method%2520for%2520lipophilicity%2520and%2520integrity%2520using%2520liquid%2520chromatography-mass%2520spectrometry%26jtitle%3DJ.%2520Chromatogr.%2520B%253A%2520Anal.%2520Technol.%2520Biomed.%2520Life%2520Sci.%26date%3D2003%26volume%3D791%26spage%3D381%26epage%3D388%26doi%3D10.1016%2FS1570-0232%2803%2900250-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="note"><p class="first last">The free-fraction in the cell assay was measured by dialysis using standard techniques, <i>f</i><sub>ua</sub> = 0.47 (90%CI = 0.54–0.42). Free GI<sub>50</sub> = GI<sub>50</sub>·<i>f</i><sub>ua</sub></p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><div class="note"><p class="first last">Hepatic blood flow (HBF) in mouse was assumed to be 90 mL/min/kg.</p></div><span class="NLM_contrib-group">Sanoh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horiguchi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugihara, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotake, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tayama, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohshita, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tateno, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horie, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitamura, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohta, S.</span><span> </span><span class="NLM_article-title">Prediction of in vivo hepatic clearance and half-life of drug candidates in human using chimeric mice with humanized liver</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">322</span><span class="NLM_x">–</span> <span class="NLM_lpage">328</span><span class="refDoi"> DOI: 10.1124/dmd.111.040923</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1124%2Fdmd.111.040923" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=322-328&author=S.+Sanohauthor=A.+Horiguchiauthor=K.+Sugiharaauthor=Y.+Kotakeauthor=Y.+Tayamaauthor=H.+Ohshitaauthor=C.+Tatenoauthor=T.+Horieauthor=S.+Kitamuraauthor=S.+Ohta&title=Prediction+of+in+vivo+hepatic+clearance+and+half-life+of+drug+candidates+in+human+using+chimeric+mice+with+humanized+liver&doi=10.1124%2Fdmd.111.040923"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1124%2Fdmd.111.040923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.111.040923%26sid%3Dliteratum%253Aachs%26aulast%3DSanoh%26aufirst%3DS.%26aulast%3DHoriguchi%26aufirst%3DA.%26aulast%3DSugihara%26aufirst%3DK.%26aulast%3DKotake%26aufirst%3DY.%26aulast%3DTayama%26aufirst%3DY.%26aulast%3DOhshita%26aufirst%3DH.%26aulast%3DTateno%26aufirst%3DC.%26aulast%3DHorie%26aufirst%3DT.%26aulast%3DKitamura%26aufirst%3DS.%26aulast%3DOhta%26aufirst%3DS.%26atitle%3DPrediction%2520of%2520in%2520vivo%2520hepatic%2520clearance%2520and%2520half-life%2520of%2520drug%2520candidates%2520in%2520human%2520using%2520chimeric%2520mice%2520with%2520humanized%2520liver%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2012%26volume%3D40%26spage%3D322%26epage%3D328%26doi%3D10.1124%2Fdmd.111.040923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Smith, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerns, E. H.</span><span> </span><span class="NLM_article-title">The effect of plasma protein binding on in vivo efficacy: misconceptions in drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">929</span><span class="NLM_x">–</span> <span class="NLM_lpage">939</span><span class="refDoi"> DOI: 10.1038/nrd3287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnrd3287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=21119731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsV2jurfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=929-939&author=D.+A.+Smithauthor=L.+Diauthor=E.+H.+Kerns&title=The+effect+of+plasma+protein+binding+on+in+vivo+efficacy%3A+misconceptions+in+drug+discovery&doi=10.1038%2Fnrd3287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The effect of plasma protein binding on in vivo efficacy: misconceptions in drug discovery</span></div><div class="casAuthors">Smith, Dennis A.; Di, Li; Kerns, Edward H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">929-939</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Data from in vitro plasma protein binding expts. that det. the fraction of protein-bound drug are frequently used in drug discovery to guide structure design and to prioritize compds. for in vivo studies.  However, we consider that these practices are usually misleading, because in vivo efficacy is detd. by the free (unbound) drug concn. surrounding the therapeutic target, not by the free drug fraction.  These practices yield no enhancement of the in vivo free drug concn.  So, decisions based on free drug fraction could result in the wrong compds. being advanced through drug discovery programs.  This Perspective provides guidance on the application of plasma protein binding information in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOa3ippodhGbVg90H21EOLACvtfcHk0ljjfgUbIA7dlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsV2jurfI&md5=a45d00f68ab44580fa38a6acef49493a</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnrd3287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3287%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DD.%2BA.%26aulast%3DDi%26aufirst%3DL.%26aulast%3DKerns%26aufirst%3DE.%2BH.%26atitle%3DThe%2520effect%2520of%2520plasma%2520protein%2520binding%2520on%2520in%2520vivo%2520efficacy%253A%2520misconceptions%2520in%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26spage%3D929%26epage%3D939%26doi%3D10.1038%2Fnrd3287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="note"><p class="first last">C<sub>av</sub><sup>0–24h</sup> = AUC<sup>0–24h</sup>/24.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Smith, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaumont, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maurer, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di, Li.</span><span> </span><span class="NLM_article-title">Volume of distribution in drug design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">5691</span><span class="NLM_x">–</span> <span class="NLM_lpage">5698</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00201</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00201" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltFSlsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5691-5698&author=D.+A.+Smithauthor=K.+Beaumontauthor=T.+S.+Maurerauthor=Li.+Di&title=Volume+of+distribution+in+drug+design&doi=10.1021%2Facs.jmedchem.5b00201"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Volume of Distribution in Drug Design</span></div><div class="casAuthors">Smith, Dennis A.; Beaumont, Kevin; Maurer, Tristan S.; Di, Li</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5691-5698</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Vol. of distribution is one of the most important pharmacokinetic properties of a drug candidate.  It is a major determinant of half-life and dosing frequency of a drug.  For a similar log P, a basic mol. will tend to exhibit higher vol. of distribution than a neutral mol.  Acids often exhibit low vols. of distribution.  Although a design strategy against vol. of distribution can be advantageous in achieving desirable dosing regimen, it must be well-directed in order to avoid detrimental effects to other important properties.  Strategies to increase vol. of distribution include adding lipophilicity and introducing basic functional groups in a way that does not increase metabolic clearance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP7QwgPahf4bVg90H21EOLACvtfcHk0ljjfgUbIA7dlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltFSlsb0%253D&md5=2c7289b3602a8edef2d9ef0e6a82e483</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00201%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DD.%2BA.%26aulast%3DBeaumont%26aufirst%3DK.%26aulast%3DMaurer%26aufirst%3DT.%2BS.%26aulast%3DDi%26aufirst%3DLi.%26atitle%3DVolume%2520of%2520distribution%2520in%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5691%26epage%3D5698%26doi%3D10.1021%2Facs.jmedchem.5b00201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="note"><p class="first last">Assumes linear scaling of free exposure (AUC<sub>u</sub>) between different doses.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="note"><p class="first last">Two mice were dosed at 30 mg/kg po qd for 4 days and were then monitored for the following 3 days. Both mice retained good condition over that period as assessed by %body weight. See <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for details.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Smith, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houghton, P.</span><span> </span><span class="NLM_article-title">A proposal regarding reporting of in vitro testing results</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2828</span><span class="NLM_x">–</span> <span class="NLM_lpage">2833</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-13-0043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1158%2F1078-0432.CCR-13-0043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=23580781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXos12jtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=2828-2833&author=M.+A.+Smithauthor=P.+Houghton&title=A+proposal+regarding+reporting+of+in+vitro+testing+results&doi=10.1158%2F1078-0432.CCR-13-0043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A Proposal Regarding Reporting of In Vitro Testing Results</span></div><div class="casAuthors">Smith, Malcolm A.; Houghton, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2828-2833</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  The high rate of neg. clin. trials and failed drug development programs calls into question the use of preclin. testing as currently practiced.  An important issue for the in vitro testing of agents that have advanced into the clinic is the use of clin. irrelevant concns. in reports making claims for anticancer activity, as illustrated by publications for sorafenib, vorinostat, and metformin.  For sorafenib, high protein binding leads to a dichotomy between concns. active in the 10% serum conditions commonly used for in vitro testing and concns. active in plasma.  Failure to recognize this distinction leads to inappropriate claims of activity for sorafenib based on the micromolar concns. commonly used for in vitro testing in low serum conditions.  For vorinostat and metformin, results using in vitro concns. higher than those achievable in patients are reported despite the availability of publications describing human pharmacokinetic data for each agent.  We encourage journal editors and reviewers to pay greater attention to clin. relevant concns. when considering reports that include in vitro testing of agents for which human pharmacokinetic data are available.  Steps taken to more carefully scrutinize activity claims based on in vitro results can help direct researchers away from clin. irrelevant lines of research and toward lines of research that are more likely to lead to pos. clin. trials and to improved treatments for patients with cancer.  Clin Cancer Res; 19(11); 2828-33. ©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOC9I1yI2eYrVg90H21EOLACvtfcHk0liTCQeCK3vyew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXos12jtLo%253D&md5=0a93995f520c722080947689eef19aee</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-13-0043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-13-0043%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DM.%2BA.%26aulast%3DHoughton%26aufirst%3DP.%26atitle%3DA%2520proposal%2520regarding%2520reporting%2520of%2520in%2520vitro%2520testing%2520results%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2013%26volume%3D19%26spage%3D2828%26epage%3D2833%26doi%3D10.1158%2F1078-0432.CCR-13-0043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="note"><p class="first last">The tumor growth inhibition is defined as %TGI = (((<i>C</i><sub>t</sub> – <i>C</i><sub>0</sub>) – (<i>T</i><sub>t</sub> – <i>T</i><sub>0</sub>))/(<i>C</i><sub>t</sub> – <i>C</i><sub>0</sub>)) × 100; <i>T</i><sub>t</sub> = arithmetic mean volume of treated tumors at time = t; <i>T</i><sub>0</sub> = arithmetic mean volume of treated tumors at time = 0; <i>C</i><sub>t</sub> = arithmetic mean volume of control tumors at time = t; <i>C</i><sub>0</sub> = arithmetic mean volume control tumors at time = 0.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="note"><p class="first last">The <i>p</i>-value refers to the comparison of untransformed arithmetic means of the treated and control arms using an unpaired Student’s <i>t</i> test with Welch’s correction. The value is reported to two significant figures.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">de Billy, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powers, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span> </span><span class="NLM_article-title">Drugging the heat shock factor 1 pathway: exploitation of the critical cancer cell dependence on the guardian of the proteome</span> <span class="citation_source-journal">Cell Cycle</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">3806</span><span class="NLM_x">–</span> <span class="NLM_lpage">3808</span><span class="refDoi"> DOI: 10.4161/cc.8.23.10423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.4161%2Fcc.8.23.10423" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=3806-3808&author=E.+de+Billyauthor=M.+V.+Powersauthor=J.+R.+Smithauthor=P.+Workman&title=Drugging+the+heat+shock+factor+1+pathway%3A+exploitation+of+the+critical+cancer+cell+dependence+on+the+guardian+of+the+proteome&doi=10.4161%2Fcc.8.23.10423"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.4161%2Fcc.8.23.10423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.8.23.10423%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BBilly%26aufirst%3DE.%26aulast%3DPowers%26aufirst%3DM.%2BV.%26aulast%3DSmith%26aufirst%3DJ.%2BR.%26aulast%3DWorkman%26aufirst%3DP.%26atitle%3DDrugging%2520the%2520heat%2520shock%2520factor%25201%2520pathway%253A%2520exploitation%2520of%2520the%2520critical%2520cancer%2520cell%2520dependence%2520on%2520the%2520guardian%2520of%2520the%2520proteome%26jtitle%3DCell%2520Cycle%26date%3D2009%26volume%3D8%26spage%3D3806%26epage%3D3808%26doi%3D10.4161%2Fcc.8.23.10423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span>Diversity Profile—P9; <span class="NLM_publisher-name">Eurofins</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span><a href="http://www.cerep.fr/cerep/users/pages/catalog/profiles/DetailProfile.asp?profile=2121" class="extLink">http://www.cerep.fr/cerep/users/pages/catalog/profiles/DetailProfile.asp?profile=2121</a> (accessed August 31,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Diversity+Profile%E2%80%94P9%3B+Eurofins%2C+2016%3B+http%3A%2F%2Fwww.cerep.fr%2Fcerep%2Fusers%2Fpages%2Fcatalog%2Fprofiles%2FDetailProfile.asp%3Fprofile%3D2121+%28accessed+August+31%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DDiversity%2520Profile%25E2%2580%2594P9%26pub%3DEurofins%26date%3D2016%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51a"><span><span class="NLM_label">(a) </span>Target Identification and Validation; <span class="NLM_publisher-name">Evotec</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span><a href="https://www.evotec.com/uploads/media_library/36/2015-09_DDUp-Services_Target__identification.pdf" class="extLink">https://www.evotec.com/uploads/media_library/36/2015-09_DDUp-Services_Target__identification.pdf</a> (accessed August 31, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Target+Identification+and+Validation%3B+Evotec%2C+2016%3B+https%3A%2F%2Fwww.evotec.com%2Fuploads%2Fmedia_library%2F36%2F2015-09_DDUp-Services_Target__identification.pdf+%28accessed+August+31%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DTarget%2520Identification%2520and%2520Validation%26pub%3DEvotec%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit51b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Sharma, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bairlein, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greff, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kéri, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cox, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daub, H.</span><span> </span><span class="NLM_article-title">Proteomics strategy for quantitative protein interaction profiling in cell extracts</span> <span class="citation_source-journal">Nat. Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">741</span><span class="NLM_x">–</span> <span class="NLM_lpage">744</span><span class="refDoi"> DOI: 10.1038/nmeth.1373</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnmeth.1373" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=19749761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFaju73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2009&pages=741-744&author=K.+Sharmaauthor=C.+Weberauthor=M.+Bairleinauthor=Z.+Greffauthor=G.+K%C3%A9riauthor=J.+Coxauthor=J.+V.+Olsenauthor=H.+Daub&title=Proteomics+strategy+for+quantitative+protein+interaction+profiling+in+cell+extracts&doi=10.1038%2Fnmeth.1373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51bR"><div class="casContent"><span class="casTitleNuber">51b</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomics strategy for quantitative protein interaction profiling in cell extracts</span></div><div class="casAuthors">Sharma, Kirti; Weber, Christoph; Bairlein, Michaela; Greff, Zoltan; Keri, Gyoergy; Cox, Juergen; Olsen, Jesper V.; Daub, Henrik</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">741-744</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The authors report a proteomics strategy to both identify and quantify cellular target protein interactions with externally introduced ligands.  The authors detd. dissocn. consts. for target proteins interacting with the ligand of interest by combining quant. mass spectrometry with a defined set of affinity purifn. expts.  The authors demonstrate the general utility of this methodol. in interaction studies involving small-mol. kinase inhibitors, a tyrosine-phosphorylated peptide and an antibody as affinity ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9I5plLFxQRbVg90H21EOLACvtfcHk0liTCQeCK3vyew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFaju73O&md5=78732562cac1cc2af0938f5536134098</span></div><a href="/servlet/linkout?suffix=cit51b&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.1373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.1373%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DK.%26aulast%3DWeber%26aufirst%3DC.%26aulast%3DBairlein%26aufirst%3DM.%26aulast%3DGreff%26aufirst%3DZ.%26aulast%3DK%25C3%25A9ri%26aufirst%3DG.%26aulast%3DCox%26aufirst%3DJ.%26aulast%3DOlsen%26aufirst%3DJ.%2BV.%26aulast%3DDaub%26aufirst%3DH.%26atitle%3DProteomics%2520strategy%2520for%2520quantitative%2520protein%2520interaction%2520profiling%2520in%2520cell%2520extracts%26jtitle%3DNat.%2520Methods%26date%3D2009%26volume%3D6%26spage%3D741%26epage%3D744%26doi%3D10.1038%2Fnmeth.1373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Cheng, Y.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prusoff, W. H.</span><span> </span><span class="NLM_article-title">Relationship between the inhibition constant (K<sub>i</sub>) and the concentration of inhibitor which causes 50% inhibition (I<sub>50</sub>) of an enzymatic reaction</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">3099</span><span class="NLM_x">–</span> <span class="NLM_lpage">3108</span><span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=4202581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=Y.-C.+Chengauthor=W.+H.+Prusoff&title=Relationship+between+the+inhibition+constant+%28Ki%29+and+the+concentration+of+inhibitor+which+causes+50%25+inhibition+%28I50%29+of+an+enzymatic+reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Relation between the inhibition constant K1) and the concentration of inhibitor which causes fifty per cent inhibition (I50) of an enzymic reaction</span></div><div class="casAuthors">Cheng, Yung-Chi; Prusoff, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3099-108</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">The Ki and I50 values are equal where the kinetics are non- or uncompetitive, but not where they are competitive.  The relation between the 2 values was analyzed for non- and uncompetitive bi- and monosubstrate reactions and for competitive bisubstrate reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqodFY_8msz57Vg90H21EOLACvtfcHk0liTCQeCK3vyew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D&md5=fe7175ca300d8f4085df87af37f8656c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.-C.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520inhibition%2520constant%2520%2528Ki%2529%2520and%2520the%2520concentration%2520of%2520inhibitor%2520which%2520causes%252050%2525%2520inhibition%2520%2528I50%2529%2520of%2520an%2520enzymatic%2520reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Trinkle-Mulcahy, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boulon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lam, Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urcia, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boisvert, F.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vandermoere, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrice, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swift, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rothbauer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leonhardt, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamond, A.</span><span> </span><span class="NLM_article-title">Identifying specific protein interaction partners using quantitative mass spectrometry and bead proteomes</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">183</span><span class="NLM_x">, </span> <span class="NLM_fpage">223</span><span class="NLM_x">–</span> <span class="NLM_lpage">239</span><span class="refDoi"> DOI: 10.1083/jcb.200805092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1083%2Fjcb.200805092" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=183&publication_year=2008&pages=223-239&author=L.+Trinkle-Mulcahyauthor=S.+Boulonauthor=Y.+W.+Lamauthor=R.+Urciaauthor=F.-M.+Boisvertauthor=F.+Vandermoereauthor=N.+A.+Morriceauthor=S.+Swiftauthor=U.+Rothbauerauthor=H.+Leonhardtauthor=A.+Lamond&title=Identifying+specific+protein+interaction+partners+using+quantitative+mass+spectrometry+and+bead+proteomes&doi=10.1083%2Fjcb.200805092"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1083%2Fjcb.200805092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.200805092%26sid%3Dliteratum%253Aachs%26aulast%3DTrinkle-Mulcahy%26aufirst%3DL.%26aulast%3DBoulon%26aufirst%3DS.%26aulast%3DLam%26aufirst%3DY.%2BW.%26aulast%3DUrcia%26aufirst%3DR.%26aulast%3DBoisvert%26aufirst%3DF.-M.%26aulast%3DVandermoere%26aufirst%3DF.%26aulast%3DMorrice%26aufirst%3DN.%2BA.%26aulast%3DSwift%26aufirst%3DS.%26aulast%3DRothbauer%26aufirst%3DU.%26aulast%3DLeonhardt%26aufirst%3DH.%26aulast%3DLamond%26aufirst%3DA.%26atitle%3DIdentifying%2520specific%2520protein%2520interaction%2520partners%2520using%2520quantitative%2520mass%2520spectrometry%2520and%2520bead%2520proteomes%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2008%26volume%3D183%26spage%3D223%26epage%3D239%26doi%3D10.1083%2Fjcb.200805092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="note"><p class="first last">In-house data using a biotinylated bisamide derivative in a protein pull-down experiment also identified Pirin as a molecular target of the cellularly active bisamide series (data not shown).</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Wendler, W. M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kremmer, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forster, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winnacker, E. L.</span><span> </span><span class="NLM_article-title">Identification of pirin, a novel highly conserved nuclear protein</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">272</span><span class="NLM_x">, </span> <span class="NLM_fpage">8482</span><span class="NLM_x">–</span> <span class="NLM_lpage">8489</span><span class="refDoi"> DOI: 10.1074/jbc.272.13.8482</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1074%2Fjbc.272.13.8482" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=9079676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADyaK2sXitF2ntbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=8482-8489&author=W.+M.+F.+Wendlerauthor=E.+Kremmerauthor=R.+Forsterauthor=E.+L.+Winnacker&title=Identification+of+pirin%2C+a+novel+highly+conserved+nuclear+protein&doi=10.1074%2Fjbc.272.13.8482"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Pirin, a novel highly conserved nuclear protein</span></div><div class="casAuthors">Wendler, Wolfgang M. F.; Kremmer, Elisabeth; Foerster, Reinhold; Winnacker, Ernst-Ludwig</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">272</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">8482-8489</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">In this article we describe the mol. cloning of Pirin, a novel highly conserved 32-kDa protein consisting of 290 amino acids.  Pirin was isolated by a yeast two-hybrid screen as an interactor of nuclear factor I/CCAAT box transcription factor (NFI/CTF1), which is known to stimulate adenovirus DNA replication and RNA polymerase II-driven transcription.  Pirin mRNA is expressed weakly in all human tissues tested.  About 15% of all Pirin cDNAs contain a short 34-base pair insertion in their 5'-untranslated regions, indicative of alternative splicing processes.  Multiple Pirin transcripts are expressed in skeletal muscle and heart.  Western blots and immunopptns. employing monoclonal anti-Pirin antibodies reveal that Pirin is a nuclear protein.  Moreover, confocal immunofluorescence expts. demonstrate a predominant localization of Pirin within dot-like subnuclear structures.  Homol. searches using the BLAST algorithm indicate the existence of Pirin homologues in mouse and rat.  The N-terminal half of Pirin is significantly conserved between mammals, plants, fungi, and even prokaryotic organisms.  Genomic Southern and Western blots demonstrate the presence of Pirin genes and their expression, resp., in all mammalian cell lines tested.  The expression pattern, the concd. localization is subnuclear structures, and its interaction with NFI/CTF1 in the two-hybrid system classify Pirin as a putative NFI/CTF1 cofactor, which might help to gain new insights in NFI/CTF1 functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogmDofGnosCbVg90H21EOLACvtfcHk0lhEcyhFTWfYHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXitF2ntbo%253D&md5=59a1e6e21a872b8e3e905fba15cf3e16</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1074%2Fjbc.272.13.8482&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.272.13.8482%26sid%3Dliteratum%253Aachs%26aulast%3DWendler%26aufirst%3DW.%2BM.%2BF.%26aulast%3DKremmer%26aufirst%3DE.%26aulast%3DForster%26aufirst%3DR.%26aulast%3DWinnacker%26aufirst%3DE.%2BL.%26atitle%3DIdentification%2520of%2520pirin%252C%2520a%2520novel%2520highly%2520conserved%2520nuclear%2520protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1997%26volume%3D272%26spage%3D8482%26epage%3D8489%26doi%3D10.1074%2Fjbc.272.13.8482" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Dechend, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirano, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lehmann, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heissmeyer, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ansieau, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wulczyn, F. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheidereit, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leutz, A.</span><span> </span><span class="NLM_article-title">The BCL-3 oncoprotein acts as a bridging factor between NF-κB/REL and nuclear co-regulators</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">3316</span><span class="NLM_x">–</span> <span class="NLM_lpage">3323</span><span class="refDoi"> DOI: 10.1038/sj.onc.1202717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fsj.onc.1202717" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10362352" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADyaK1MXjvFyrsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1999&pages=3316-3323&author=R.+Dechendauthor=F.+Hiranoauthor=K.+Lehmannauthor=V.+Heissmeyerauthor=S.+Ansieauauthor=F.+G.+Wulczynauthor=C.+Scheidereitauthor=A.+Leutz&title=The+BCL-3+oncoprotein+acts+as+a+bridging+factor+between+NF-%CE%BAB%2FREL+and+nuclear+co-regulators&doi=10.1038%2Fsj.onc.1202717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">The Bcl-3 oncoprotein acts as a bridging factor between NF-κB/Rel and nuclear co-regulators</span></div><div class="casAuthors">Dechend, Ralf; Hirano, Fuminori; Lehmann, Kerstin; Heissmeyer, Vigo; Ansieau, Stephane; Wulczyn, F. Gregory; Scheidereit, Claus; Leutz, Achim</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3316-3323</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Stockton Press</span>)
        </div><div class="casAbstract">The proto-oncoprotein Bcl-3 is a member of the IκB family and is present predominantly in the nucleus.  To gain insight into specific nuclear functions of Bcl-3 we have isolated proteins that interact with its ankyrin repeat domain.  Using the yeast two-hybrid-system we identified four novel binding partners of Bcl-3 in addn. to NF-κB p50 and p52, previously known to assoc. with Bcl-3.  The novel Bcl-3 interactors Jab1, Pirin, Tip60 and Bard1 are nuclear proteins which also bind to other transcription factors including c-Jun, nuclear factor I (NFI), HIV-1 Tat or the tumor suppressor and PolII holoenzyme component Brca1, resp.  Bcl-3, p50, and either Bard1, Tip60 or Pirin are sequestered into quaternary complexes on NF-κB DNA binding sites, whereas Jab1 enhances p50-Bcl-3-DNA complex formation.  Furthermore, the histone acetylase Tip60 enhances Bcl-3-p50 activated transcription through an NF-κB binding site, indicating that quaternary complexes contg. Bcl-3 interactors modulate NF-κB driven gene expression.  These data implicate Bcl-3 as an adaptor between NF-κB p50/p52 and other transcription regulators and suggest that its gene activation function may at least in part be due to recruitment of the Tip60 histone acetylase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFdiEgJ7YIorVg90H21EOLACvtfcHk0lhEcyhFTWfYHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvFyrsrw%253D&md5=8f7de746b9e70f40703d5581034bb9e8</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1202717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1202717%26sid%3Dliteratum%253Aachs%26aulast%3DDechend%26aufirst%3DR.%26aulast%3DHirano%26aufirst%3DF.%26aulast%3DLehmann%26aufirst%3DK.%26aulast%3DHeissmeyer%26aufirst%3DV.%26aulast%3DAnsieau%26aufirst%3DS.%26aulast%3DWulczyn%26aufirst%3DF.%2BG.%26aulast%3DScheidereit%26aufirst%3DC.%26aulast%3DLeutz%26aufirst%3DA.%26atitle%3DThe%2520BCL-3%2520oncoprotein%2520acts%2520as%2520a%2520bridging%2520factor%2520between%2520NF-%25CE%25BAB%252FREL%2520and%2520nuclear%2520co-regulators%26jtitle%3DOncogene%26date%3D1999%26volume%3D18%26spage%3D3316%26epage%3D3323%26doi%3D10.1038%2Fsj.onc.1202717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rehmani, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Esaki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Serrano, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, A.</span><span> </span><span class="NLM_article-title">Pirin is an iron-dependent redox regulator of NF-κB</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">9722</span><span class="NLM_x">–</span> <span class="NLM_lpage">9727</span><span class="refDoi"> DOI: 10.1073/pnas.1221743110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1073%2Fpnas.1221743110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=23716661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOrt7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=9722-9727&author=F.+Liuauthor=I.+Rehmaniauthor=S.+Esakiauthor=R.+Fuauthor=L.+Chenauthor=V.+de+Serranoauthor=A.+Liu&title=Pirin+is+an+iron-dependent+redox+regulator+of+NF-%CE%BAB&doi=10.1073%2Fpnas.1221743110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Pirin is an iron-dependent redox regulator of NF-κB</span></div><div class="casAuthors">Liu, Fange; Rehmani, Imran; Esaki, Shingo; Fu, Rong; Chen, Lirong; de Serrano, Vesna; Liu, Aimin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9722-9727, S9722/1-S9722/11</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Pirin is a nuclear nonheme Fe protein of unknown function present in all human tissues.  Here the authors describe that pirin may act as a redox sensor for the nuclear factor κB (NF-κB) transcription factor, a crit. mediator of intracellular signaling that has been linked to cellular responses to proinflammatory signals and controls the expression of a vast array of genes involved in immune and stress responses.  Pirin's regulatory effect was tested with several metals and at different oxidns. states, and the authors' spectroscopic results show that only the ferric form of pirin substantially facilitates binding of NF-κB proteins to target κB genes, a finding that suggests that pirin performs a redox-sensing role in NF-κB regulation.  The mol. mechanism of such a metal identity- and redox state-dependent regulation is revealed by the authors' structural studies of pirin.  The ferrous and ferric pirin proteins differ only by one electron, yet they have distinct conformations.  The Fe center is shown to play an allosteric role on an β-shaped surface area that has two distinct conformations based on the identity and the formal redox state of the metal.  The authors show that the R-shaped area composes the interface for pirin-NF-κB binding that is responsible for modulation of NF-κB's DNA-binding properties.  The nonheme Fe protein pirin is proposed to serve as a reversible functional switch that enables NF-κB to respond to changes in the redox levels of the cell nucleus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7Tm-I0Q4mD7Vg90H21EOLACvtfcHk0lhEcyhFTWfYHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOrt7bF&md5=d81627e3a8aa09ee6a46a8557719394d</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1221743110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1221743110%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRehmani%26aufirst%3DI.%26aulast%3DEsaki%26aufirst%3DS.%26aulast%3DFu%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DL.%26aulast%3Dde%2BSerrano%26aufirst%3DV.%26aulast%3DLiu%26aufirst%3DA.%26atitle%3DPirin%2520is%2520an%2520iron-dependent%2520redox%2520regulator%2520of%2520NF-%25CE%25BAB%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D9722%26epage%3D9727%26doi%3D10.1073%2Fpnas.1221743110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Brzoska, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stepkowski, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruszewski, M.</span><span> </span><span class="NLM_article-title">Basal PIR expression in HeLa Cells is driven by NRF2 via evolutionary conserved antioxidant response element</span> <span class="citation_source-journal">Mol. Cell. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">389</span><span class="NLM_x">, </span> <span class="NLM_fpage">99</span><span class="NLM_x">–</span> <span class="NLM_lpage">111</span><span class="refDoi"> DOI: 10.1007/s11010-013-1931-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1007%2Fs11010-013-1931-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=2014&pages=99-111&author=K.+Brzoskaauthor=T.+M.+Stepkowskiauthor=M.+Kruszewski&title=Basal+PIR+expression+in+HeLa+Cells+is+driven+by+NRF2+via+evolutionary+conserved+antioxidant+response+element&doi=10.1007%2Fs11010-013-1931-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58a&amp;dbid=16384&amp;doi=10.1007%2Fs11010-013-1931-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11010-013-1931-0%26sid%3Dliteratum%253Aachs%26aulast%3DBrzoska%26aufirst%3DK.%26aulast%3DStepkowski%26aufirst%3DT.%2BM.%26aulast%3DKruszewski%26aufirst%3DM.%26atitle%3DBasal%2520PIR%2520expression%2520in%2520HeLa%2520Cells%2520is%2520driven%2520by%2520NRF2%2520via%2520evolutionary%2520conserved%2520antioxidant%2520response%2520element%26jtitle%3DMol.%2520Cell.%2520Biochem.%26date%3D2014%26volume%3D389%26spage%3D99%26epage%3D111%26doi%3D10.1007%2Fs11010-013-1931-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit58b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Gelbman, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heguy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Connor, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zabner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crystal, R. G.</span><span> </span><span class="NLM_article-title">Upregulation of pirin expression by chronic cigarette smoking is associated with bronchial epithelial cell apoptosis</span> <span class="citation_source-journal">Respir. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">10</span><span class="refDoi"> DOI: 10.1186/1465-9921-8-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1186%2F1465-9921-8-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=17288615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A280%3ADC%252BD2s7is1yitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=10&author=B.+D.+Gelbmanauthor=A.+Heguyauthor=T.+P.+O%E2%80%99Connorauthor=J.+Zabnerauthor=R.+G.+Crystal&title=Upregulation+of+pirin+expression+by+chronic+cigarette+smoking+is+associated+with+bronchial+epithelial+cell+apoptosis&doi=10.1186%2F1465-9921-8-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58bR"><div class="casContent"><span class="casTitleNuber">58b</span><div class="casTitle"><span class="NLM_cas:atitle">Upregulation of pirin expression by chronic cigarette smoking is associated with bronchial epithelial cell apoptosis</span></div><div class="casAuthors">Gelbman Brian D; Heguy Adriana; O'Connor Timothy P; Zabner Joseph; Crystal Ronald G</div><div class="citationInfo"><span class="NLM_cas:title">Respiratory research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Cigarette smoke disrupts the protective barrier established by the airway epithelium through direct damage to the epithelial cells, leading to cell death.  Since the morphology of the airway epithelium of smokers does not typically demonstrate necrosis, the most likely mechanism for epithelial cell death in response to cigarette smoke is apoptosis.  We hypothesized that cigarette smoke directly up-regulates expression of apoptotic genes, which could play a role in airway epithelial apoptosis.  METHODS:  Microarray analysis of airway epithelium obtained by bronchoscopy on matched cohorts of 13 phenotypically normal smokers and 9 non-smokers was used to identify specific genes modulated by smoking that were associated with apoptosis.  Among the up-regulated apoptotic genes was pirin (3.1-fold, p < 0.002), an iron-binding nuclear protein and transcription cofactor.  In vitro studies using human bronchial cells exposed to cigarette smoke extract (CSE) and an adenovirus vector encoding the pirin cDNA (AdPirin) were performed to test the direct effect of cigarette smoke on pirin expression and the effect of pirin expression on apoptosis.  RESULTS:  Quantitative TaqMan RT-PCR confirmed a 2-fold increase in pirin expression in the airway epithelium of smokers compared to non-smokers (p < 0.02).  CSE applied to primary human bronchial epithelial cell cultures demonstrated that pirin mRNA levels increase in a time-and concentration-dependent manner (p < 0.03, all conditions compared to controls).  Overexpression of pirin, using the vector AdPirin, in human bronchial epithelial cells was associated with an increase in the number of apoptotic cells assessed by both TUNEL assay (5-fold, p < 0.01) and ELISA for cytoplasmic nucleosomes (19.3-fold, p < 0.01) compared to control adenovirus vector.  CONCLUSION:  These observations suggest that up-regulation of pirin may represent one mechanism by which cigarette smoke induces apoptosis in the airway epithelium, an observation that has implications for the pathogenesis of cigarette smoke-induced diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQNual6ZRHCSQ5nADWVYHg1fW6udTcc2eaav-aPnBIMsLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s7is1yitw%253D%253D&md5=fb51b1dd8e5675bb53f1f3bc23ea3ce4</span></div><a href="/servlet/linkout?suffix=cit58b&amp;dbid=16384&amp;doi=10.1186%2F1465-9921-8-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1465-9921-8-10%26sid%3Dliteratum%253Aachs%26aulast%3DGelbman%26aufirst%3DB.%2BD.%26aulast%3DHeguy%26aufirst%3DA.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DT.%2BP.%26aulast%3DZabner%26aufirst%3DJ.%26aulast%3DCrystal%26aufirst%3DR.%2BG.%26atitle%3DUpregulation%2520of%2520pirin%2520expression%2520by%2520chronic%2520cigarette%2520smoking%2520is%2520associated%2520with%2520bronchial%2520epithelial%2520cell%2520apoptosis%26jtitle%3DRespir.%2520Res.%26date%3D2007%26volume%3D8%26spage%3D10%26doi%3D10.1186%2F1465-9921-8-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Qiao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span> </span><span class="NLM_article-title">Pirin down-regulates the EAF2/U19 protein and alleviates its growth inhibition in prostate cancer cells</span> <span class="citation_source-journal">Prostate</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x">, </span> <span class="NLM_fpage">113</span><span class="NLM_x">–</span> <span class="NLM_lpage">120</span><span class="refDoi"> DOI: 10.1002/pros.22729</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1002%2Fpros.22729" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=113-120&author=Z.+Qiaoauthor=D.+Wangauthor=J.+Hahnauthor=J.+Aiauthor=Z.+Wang&title=Pirin+down-regulates+the+EAF2%2FU19+protein+and+alleviates+its+growth+inhibition+in+prostate+cancer+cells&doi=10.1002%2Fpros.22729"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2Fpros.22729&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpros.22729%26sid%3Dliteratum%253Aachs%26aulast%3DQiao%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DJ.%26aulast%3DAi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DPirin%2520down-regulates%2520the%2520EAF2%252FU19%2520protein%2520and%2520alleviates%2520its%2520growth%2520inhibition%2520in%2520prostate%2520cancer%2520cells%26jtitle%3DProstate%26date%3D2014%26volume%3D74%26spage%3D113%26epage%3D120%26doi%3D10.1002%2Fpros.22729" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Licciulli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luise, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scafetta, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capra, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giardina, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nuciforo, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bosari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viale, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazzarol, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonelli, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanfrancone, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alcalay, M.</span><span> </span><span class="NLM_article-title">Pirin inhibits cellular senescence in melanocytic cells</span> <span class="citation_source-journal">Am. J. Pathol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">178</span><span class="NLM_x">, </span> <span class="NLM_fpage">2397</span><span class="NLM_x">–</span> <span class="NLM_lpage">2406</span><span class="refDoi"> DOI: 10.1016/j.ajpath.2011.01.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.ajpath.2011.01.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=21514450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntVyqs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=178&publication_year=2011&pages=2397-2406&author=S.+Licciulliauthor=C.+Luiseauthor=G.+Scafettaauthor=M.+Capraauthor=G.+Giardinaauthor=P.+Nuciforoauthor=S.+Bosariauthor=G.+Vialeauthor=G.+Mazzarolauthor=C.+Tonelliauthor=L.+Lanfranconeauthor=M.+Alcalay&title=Pirin+inhibits+cellular+senescence+in+melanocytic+cells&doi=10.1016%2Fj.ajpath.2011.01.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Pirin inhibits cellular senescence in melanocytic cells</span></div><div class="casAuthors">Licciulli, Silvia; Luise, Chiara; Scafetta, Gaia; Capra, Maria; Giardina, Giuseppina; Nuciforo, Paolo; Bosari, Silvano; Viale, Giuseppe; Mazzarol, Giovanni; Tonelli, Chiara; Lanfrancone, Luisa; Alcalay, Myriam</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">178</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2397-2406</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Cellular senescence has been widely recognized as a tumor suppressing mechanism that acts as a barrier to cancer development after oncogenic stimuli.  A prominent in vivo model of the senescence barrier is represented by nevi, which are composed of melanocytes that, after an initial phase of proliferation induced by activated oncogenes (most commonly BRAF), are blocked in a state of cellular senescence.  Transformation to melanoma occurs when genes involved in controlling senescence are mutated or silenced and cells reacquire the capacity to proliferate.  Pirin (PIR) is a highly conserved nuclear protein that likely functions as a transcriptional regulator whose expression levels are altered in different types of tumors.  We analyzed the expression pattern of PER in adult human tissues and found that it is expressed in melanocytes and has a complex pattern of regulation in nevi and melanoma: it is rarely detected in mature nevi, but is expressed at high levels in a subset of melanomas.  Loss of function and overexpression expts. in normal and transformed melanocytic cells revealed that PIR is involved in the neg. control of cellular senescence and that its expression is necessary to overcome the senescence barrier.  Our results suggest that PIR may have a relevant role in melanoma progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoqcH_XkOw-bVg90H21EOLACvtfcHk0lhI5o8PO_Sl1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntVyqs7o%253D&md5=c40cc6c5235026b31db04ef84aadd38e</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.ajpath.2011.01.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajpath.2011.01.019%26sid%3Dliteratum%253Aachs%26aulast%3DLicciulli%26aufirst%3DS.%26aulast%3DLuise%26aufirst%3DC.%26aulast%3DScafetta%26aufirst%3DG.%26aulast%3DCapra%26aufirst%3DM.%26aulast%3DGiardina%26aufirst%3DG.%26aulast%3DNuciforo%26aufirst%3DP.%26aulast%3DBosari%26aufirst%3DS.%26aulast%3DViale%26aufirst%3DG.%26aulast%3DMazzarol%26aufirst%3DG.%26aulast%3DTonelli%26aufirst%3DC.%26aulast%3DLanfrancone%26aufirst%3DL.%26aulast%3DAlcalay%26aufirst%3DM.%26atitle%3DPirin%2520inhibits%2520cellular%2520senescence%2520in%2520melanocytic%2520cells%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2011%26volume%3D178%26spage%3D2397%26epage%3D2406%26doi%3D10.1016%2Fj.ajpath.2011.01.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Miyazaki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simizu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okumura, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takagi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osada, H.</span><span> </span><span class="NLM_article-title">A small-molecule inhibitor shows that pirin regulates migration of melanoma cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">667</span><span class="NLM_x">–</span> <span class="NLM_lpage">673</span><span class="refDoi"> DOI: 10.1038/nchembio.423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnchembio.423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=20711196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVais7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=667-673&author=I.+Miyazakiauthor=S.+Simizuauthor=H.+Okumuraauthor=S.+Takagiauthor=H.+Osada&title=A+small-molecule+inhibitor+shows+that+pirin+regulates+migration+of+melanoma+cells&doi=10.1038%2Fnchembio.423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">A small-molecule inhibitor shows that pirin regulates migration of melanoma cells</span></div><div class="casAuthors">Miyazaki, Isao; Simizu, Siro; Okumura, Hideo; Takagi, Satoshi; Osada, Hiroyuki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">667-673</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The discovery of small mols. that bind to a specific target and disrupt the function of proteins is an important step in chem. biol., esp. for poorly characterized proteins.  Human pirin is a nuclear protein of unknown function that is widely expressed in punctate subnuclear structures in human tissues.  Here, we report the discovery of a small mol. that binds to pirin.  We detd. how the small mol. bound to pirin by solving the cocrystal structure.  Either knockdown of pirin or treatment with the small mol. inhibited melanoma cell migration.  Thus, inhibition of pirin by the small mol. has led to a greater understanding of the function of pirin and represents a new method of studying pirin-mediated signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIZDQkD9RCtrVg90H21EOLACvtfcHk0lhiI8CUJS_35g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVais7bF&md5=21479021d6460fa0381eced68c6ab799</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.423%26sid%3Dliteratum%253Aachs%26aulast%3DMiyazaki%26aufirst%3DI.%26aulast%3DSimizu%26aufirst%3DS.%26aulast%3DOkumura%26aufirst%3DH.%26aulast%3DTakagi%26aufirst%3DS.%26aulast%3DOsada%26aufirst%3DH.%26atitle%3DA%2520small-molecule%2520inhibitor%2520shows%2520that%2520pirin%2520regulates%2520migration%2520of%2520melanoma%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D667%26epage%3D673%26doi%3D10.1038%2Fnchembio.423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Yamaoka, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sumiyoshi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minakawa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sumida, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saito, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikoma, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mabuchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozawa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inagaki, Y.</span><span> </span><span class="NLM_article-title">A novel small compound accelerates dermal wound healing by modifying infiltration, proliferation and migration of distinct cellular components in mice</span> <span class="citation_source-journal">J. Dermatol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x">, </span> <span class="NLM_fpage">204</span><span class="NLM_x">–</span> <span class="NLM_lpage">213</span><span class="refDoi"> DOI: 10.1016/j.jdermsci.2014.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.jdermsci.2014.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=24702853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1elsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=204-213&author=H.+Yamaokaauthor=H.+Sumiyoshiauthor=K.+Higashiauthor=S.+Nakaoauthor=K.+Minakawaauthor=K.+Sumidaauthor=K.+Saitoauthor=N.+Ikomaauthor=T.+Mabuchiauthor=A.+Ozawaauthor=Y.+Inagaki&title=A+novel+small+compound+accelerates+dermal+wound+healing+by+modifying+infiltration%2C+proliferation+and+migration+of+distinct+cellular+components+in+mice&doi=10.1016%2Fj.jdermsci.2014.03.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">A novel small compound accelerates dermal wound healing by modifying infiltration, proliferation and migration of distinct cellular components in mice</span></div><div class="casAuthors">Yamaoka, Hanako; Sumiyoshi, Hideaki; Higashi, Kiyoshi; Nakao, Sachie; Minakawa, Kaori; Sumida, Kayo; Saito, Koichi; Ikoma, Norihiro; Mabuchi, Tomotaka; Ozawa, Akira; Inagaki, Yutaka</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Dermatological Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">204-213</span>CODEN:
                <span class="NLM_cas:coden">JDSCEI</span>;
        ISSN:<span class="NLM_cas:issn">0923-1811</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Impaired wound healing in skin ulcer is one of the major medical issues in the aged society.  Wound healing is a complex process orchestrated by a no. of humoral factors and cellular components.  TGF-β is known to stimulate collagen prodn. in dermal fibroblasts while inhibiting proliferation of epidermal keratinocyte.  A screening of small compds. that suppress type I collagen prodn. in fibroblasts has identified HSc025 that antagonizes the TGF-β/Smad signal.We examd. the effects of HSc025 on dermal wound healing and elucidated the underlying mechanisms.Effects of HSc025 on the wound closure process were evaluated in a murine full-thickness excisional wound healing model.  Cell proliferation and migration were estd. using primary cultures of human keratinocytes and fibroblasts.  Comprehensive analyses of gene expression profiles were performed using untreated and HSc025-treated fibroblasts.Oral HSc025 administration suppressed macrophage infiltration and accelerated wound closure as early as at day 2 after the dermal excision.  Treatment of cultured keratinocytes with HSc025 counteracted the inhibitory effects of TGF-β on cell proliferation and migration.  On the other hand, HSc025 stimulated migration, but not proliferation, of dermal fibroblasts independently of TGF-β.  Expts. using an artificial dermis graft revealed that HSc025 stimulated migration of collagen-producing cells into the graft tissue.  A cDNA microarray anal. of untreated and HSc025-treated fibroblasts identified pirin as a crit. mediator accelerating fibroblast migration.HSc025 accelerates wound healing by modifying infiltration, proliferation and migration of distinct cellular components, which provides a novel insight into the therapy for intractable skin ulcer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0cYiJTQDJ3rVg90H21EOLACvtfcHk0lhiI8CUJS_35g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1elsbw%253D&md5=d9d77d13a9040b13afcb5c693e59f766</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.jdermsci.2014.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jdermsci.2014.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DYamaoka%26aufirst%3DH.%26aulast%3DSumiyoshi%26aufirst%3DH.%26aulast%3DHigashi%26aufirst%3DK.%26aulast%3DNakao%26aufirst%3DS.%26aulast%3DMinakawa%26aufirst%3DK.%26aulast%3DSumida%26aufirst%3DK.%26aulast%3DSaito%26aufirst%3DK.%26aulast%3DIkoma%26aufirst%3DN.%26aulast%3DMabuchi%26aufirst%3DT.%26aulast%3DOzawa%26aufirst%3DA.%26aulast%3DInagaki%26aufirst%3DY.%26atitle%3DA%2520novel%2520small%2520compound%2520accelerates%2520dermal%2520wound%2520healing%2520by%2520modifying%2520infiltration%252C%2520proliferation%2520and%2520migration%2520of%2520distinct%2520cellular%2520components%2520in%2520mice%26jtitle%3DJ.%2520Dermatol.%2520Sci.%26date%3D2014%26volume%3D74%26spage%3D204%26epage%3D213%26doi%3D10.1016%2Fj.jdermsci.2014.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Komai, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niwa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasazawa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simizu, S.</span><span> </span><span class="NLM_article-title">Pirin regulates epithelial to mesenchymal transition independently of BCL3-SLUG signaling</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">589</span><span class="NLM_x">, </span> <span class="NLM_fpage">738</span><span class="NLM_x">–</span> <span class="NLM_lpage">743</span><span class="refDoi"> DOI: 10.1016/j.febslet.2015.01.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.febslet.2015.01.040" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=589&publication_year=2015&pages=738-743&author=K.+Komaiauthor=Y.+Niwaauthor=Y.+Sasazawaauthor=S.+Simizu&title=Pirin+regulates+epithelial+to+mesenchymal+transition+independently+of+BCL3-SLUG+signaling&doi=10.1016%2Fj.febslet.2015.01.040"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63a&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2015.01.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2015.01.040%26sid%3Dliteratum%253Aachs%26aulast%3DKomai%26aufirst%3DK.%26aulast%3DNiwa%26aufirst%3DY.%26aulast%3DSasazawa%26aufirst%3DY.%26aulast%3DSimizu%26aufirst%3DS.%26atitle%3DPirin%2520regulates%2520epithelial%2520to%2520mesenchymal%2520transition%2520independently%2520of%2520BCL3-SLUG%2520signaling%26jtitle%3DFEBS%2520Lett.%26date%3D2015%26volume%3D589%26spage%3D738%26epage%3D743%26doi%3D10.1016%2Fj.febslet.2015.01.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit63b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Marcucci, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stassi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Maria, R.</span><span> </span><span class="NLM_article-title">Epithelial–mesenchymal transition: a new target in anticancer drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">311</span><span class="NLM_x">–</span> <span class="NLM_lpage">325</span><span class="refDoi"> DOI: 10.1038/nrd.2015.13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnrd.2015.13" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=311-325&author=F.+Marcucciauthor=G.+Stassiauthor=R.+De+Maria&title=Epithelial%E2%80%93mesenchymal+transition%3A+a+new+target+in+anticancer+drug+discovery&doi=10.1038%2Fnrd.2015.13"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63b&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2015.13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2015.13%26sid%3Dliteratum%253Aachs%26aulast%3DMarcucci%26aufirst%3DF.%26aulast%3DStassi%26aufirst%3DG.%26aulast%3DDe%2BMaria%26aufirst%3DR.%26atitle%3DEpithelial%25E2%2580%2593mesenchymal%2520transition%253A%2520a%2520new%2520target%2520in%2520anticancer%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D311%26epage%3D325%26doi%3D10.1038%2Fnrd.2015.13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="note"><p class="first last">In-house studies using siRNA to inhibit pirin synthesis in SK-OV-3 cells demonstrated only limited effects on proliferation. Further studies are ongoing.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span>TPh A was purchased from Sigma-Aldrich: <a href="http://www.sigmaaldrich.com/catalog/product/sigma/t6205?lang=en&amp;region=GB" class="extLink">http://www.sigmaaldrich.com/catalog/product/sigma/t6205?lang=en&region=GB</a> (August 31,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=TPh+A+was+purchased+from+Sigma-Aldrich%3A+http%3A%2F%2Fwww.sigmaaldrich.com%2Fcatalog%2Fproduct%2Fsigma%2Ft6205%3Flang%3Den%26region%3DGB+%28August+31%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Dunwell, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Culham, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sosa-Aguirre, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodenough, P. W.</span><span> </span><span class="NLM_article-title">Evolution of functional diversity in the cupin superfamily</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">740</span><span class="NLM_x">–</span> <span class="NLM_lpage">746</span><span class="refDoi"> DOI: 10.1016/S0968-0004(01)01981-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2FS0968-0004%2801%2901981-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=11738598" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BD3MXovVeru7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2001&pages=740-746&author=J.+M.+Dunwellauthor=A.+Culhamauthor=C.+E.+Carterauthor=C.+R.+Sosa-Aguirreauthor=P.+W.+Goodenough&title=Evolution+of+functional+diversity+in+the+cupin+superfamily&doi=10.1016%2FS0968-0004%2801%2901981-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66aR"><div class="casContent"><span class="casTitleNuber">66a</span><div class="casTitle"><span class="NLM_cas:atitle">Evolution of functional diversity in the cupin superfamily</span></div><div class="casAuthors">Dunwell, Jim M.; Culham, Alastair; Carter, Carol E.; Sosa-Aguirre, Carlos R.; Goodenough, Peter W.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">740-746</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The cupin superfamily of proteins is among the most functionally diverse of any described to date.  It was named on the basis of the conserved β-barrel fold ('cupa' is the Latin term for a small barrel), and comprises both enzymic and non-enzymic members, which have either one or two cupin domains.  Within the conserved tertiary structure, the variety of biochem. function is provided by minor variation of the residues in the active site and the identity of the bound metal ion.  This review discusses the advantages of this particular scaffold and provides an evolutionary anal. of 18 different subclasses within the cupin superfamily.  This mol. archaeol. review describes the discovery of the early ancestors of the plant proteins that now make up a major part of the human diet.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRAWCXekIxMLVg90H21EOLACvtfcHk0lioLbrF1yGiGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXovVeru7w%253D&md5=b7c00861747d648509a3cca37676f4ec</span></div><a href="/servlet/linkout?suffix=cit66a&amp;dbid=16384&amp;doi=10.1016%2FS0968-0004%2801%2901981-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0004%252801%252901981-8%26sid%3Dliteratum%253Aachs%26aulast%3DDunwell%26aufirst%3DJ.%2BM.%26aulast%3DCulham%26aufirst%3DA.%26aulast%3DCarter%26aufirst%3DC.%2BE.%26aulast%3DSosa-Aguirre%26aufirst%3DC.%2BR.%26aulast%3DGoodenough%26aufirst%3DP.%2BW.%26atitle%3DEvolution%2520of%2520functional%2520diversity%2520in%2520the%2520cupin%2520superfamily%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2001%26volume%3D26%26spage%3D740%26epage%3D746%26doi%3D10.1016%2FS0968-0004%2801%2901981-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit66b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Dunwell, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purvis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuri, S.</span><span> </span><span class="NLM_article-title">Cupins: the most functionally diverse protein superfamily?</span> <span class="citation_source-journal">Phytochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">7</span><span class="NLM_x">–</span> <span class="NLM_lpage">17</span><span class="refDoi"> DOI: 10.1016/j.phytochem.2003.08.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.phytochem.2003.08.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=14697267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvFynsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2004&pages=7-17&author=J.+M.+Dunwellauthor=A.+Purvisauthor=S.+Khuri&title=Cupins%3A+the+most+functionally+diverse+protein+superfamily%3F&doi=10.1016%2Fj.phytochem.2003.08.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66bR"><div class="casContent"><span class="casTitleNuber">66b</span><div class="casTitle"><span class="NLM_cas:atitle">Cupins: the most functionally diverse protein superfamily?</span></div><div class="casAuthors">Dunwell, Jim M.; Purvis, Alan; Khuri, Sawsan</div><div class="citationInfo"><span class="NLM_cas:title">Phytochemistry (Elsevier)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7-17</span>CODEN:
                <span class="NLM_cas:coden">PYTCAS</span>;
        ISSN:<span class="NLM_cas:issn">0031-9422</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review.  The cupin superfamily of proteins, named on the basis of a conserved β-barrel fold ('cupa' is the Latin term for a small barrel), was originally discovered using a conserved motif found within germin and germin-like proteins from higher plants.  Previous anal. of cupins had identified some 18 different functional classes that range from single-domain bacterial enzymes such as isomerases and epimerases involved in the modification of cell wall carbohydrates, through to two-domain bicupins such as the desiccation-tolerant seed storage globulins, and multidomain transcription factors including one linked to the nodulation response in legumes.  Recent advances in comparative genomics, and the resoln. of many more 3-D structures have now revealed that the largest subset of the cupin superfamily is the 2-oxyglutarate-Fe2+ dependent dioxygenases.  The substrates for this subclass of enzyme are many and varied and in total amt. to probably 50-100 different biochem. reactions, including several involved in plant growth and development.  Although the majority of enzymic cupins contain iron as an active site metal, other members contain either copper, zinc, cobalt, nickel or manganese ions as a cofactor, with each cofactor allowing a different type of chem. to occur within the conserved tertiary structure.  This review discusses the range of structures and functions found in this most diverse of superfamilies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGLBE2G_DAE7Vg90H21EOLACvtfcHk0lioLbrF1yGiGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvFynsro%253D&md5=8af457482acd815ffc0d8ff05bf247c7</span></div><a href="/servlet/linkout?suffix=cit66b&amp;dbid=16384&amp;doi=10.1016%2Fj.phytochem.2003.08.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phytochem.2003.08.016%26sid%3Dliteratum%253Aachs%26aulast%3DDunwell%26aufirst%3DJ.%2BM.%26aulast%3DPurvis%26aufirst%3DA.%26aulast%3DKhuri%26aufirst%3DS.%26atitle%3DCupins%253A%2520the%2520most%2520functionally%2520diverse%2520protein%2520superfamily%253F%26jtitle%3DPhytochemistry%26date%3D2004%26volume%3D65%26spage%3D7%26epage%3D17%26doi%3D10.1016%2Fj.phytochem.2003.08.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Khuri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bakker, F. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunwell, J. M.</span><span> </span><span class="NLM_article-title">Phylogeny, function, and evolution of the cupins, a structurally conserved, functionally diverse superfamily of proteins</span> <span class="citation_source-journal">Mol. Biol. Evol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">593</span><span class="NLM_x">–</span> <span class="NLM_lpage">605</span><span class="refDoi"> DOI: 10.1093/oxfordjournals.molbev.a003840</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1093%2Foxfordjournals.molbev.a003840" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2001&pages=593-605&author=S.+Khuriauthor=F.+T.+Bakkerauthor=J.+M.+Dunwell&title=Phylogeny%2C+function%2C+and+evolution+of+the+cupins%2C+a+structurally+conserved%2C+functionally+diverse+superfamily+of+proteins&doi=10.1093%2Foxfordjournals.molbev.a003840"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67a&amp;dbid=16384&amp;doi=10.1093%2Foxfordjournals.molbev.a003840&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Foxfordjournals.molbev.a003840%26sid%3Dliteratum%253Aachs%26aulast%3DKhuri%26aufirst%3DS.%26aulast%3DBakker%26aufirst%3DF.%2BT.%26aulast%3DDunwell%26aufirst%3DJ.%2BM.%26atitle%3DPhylogeny%252C%2520function%252C%2520and%2520evolution%2520of%2520the%2520cupins%252C%2520a%2520structurally%2520conserved%252C%2520functionally%2520diverse%2520superfamily%2520of%2520proteins%26jtitle%3DMol.%2520Biol.%2520Evol.%26date%3D2001%26volume%3D18%26spage%3D593%26epage%3D605%26doi%3D10.1093%2Foxfordjournals.molbev.a003840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit67b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Uberto, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moomaw, E. W.</span><span> </span><span class="NLM_article-title">Protein similarity networks reveal relationships among sequence, structure, and function within the cupin superfamily</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e74477</span><span class="refDoi"> DOI: 10.1371/journal.pone.0074477</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1371%2Fjournal.pone.0074477" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e74477&author=R.+Ubertoauthor=E.+W.+Moomaw&title=Protein+similarity+networks+reveal+relationships+among+sequence%2C+structure%2C+and+function+within+the+cupin+superfamily&doi=10.1371%2Fjournal.pone.0074477"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67b&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0074477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0074477%26sid%3Dliteratum%253Aachs%26aulast%3DUberto%26aufirst%3DR.%26aulast%3DMoomaw%26aufirst%3DE.%2BW.%26atitle%3DProtein%2520similarity%2520networks%2520reveal%2520relationships%2520among%2520sequence%252C%2520structure%252C%2520and%2520function%2520within%2520the%2520cupin%2520superfamily%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De74477%26doi%3D10.1371%2Fjournal.pone.0074477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><div class="note"><p class="first last">PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1J1L">1J1L</a>:</p></div><span class="NLM_contrib-group">Pang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartlam, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyatake, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hisano, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, Z.</span><span> </span><span class="NLM_article-title">Crystal structure of human pirin an iron-binding nuclear protein and transcription cofactor</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">1491</span><span class="NLM_x">–</span> <span class="NLM_lpage">1498</span><div class="note"><p class="first last">For other examples of Pirin crystal structures, see: PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ACL">3ACL</a>, ref <a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">61</a>; PDBs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ACL">3ACL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ERO">4ERO</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EWA">4EWA</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EWD">4EWD</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EWE">4EWE</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GUL">4GUL</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HLT">4HLT</a>, ref <a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">57</a></p></div><span class="refDoi"> DOI: 10.1074/jbc.M310022200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1074%2Fjbc.M310022200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=1491-1498&author=H.+Pangauthor=M.+Bartlamauthor=Q.+Zengauthor=H.+Miyatakeauthor=T.+Hisanoauthor=K.+Mikiauthor=L.+L.+Wongauthor=G.+F.+Gaoauthor=Z.+Rao&title=Crystal+structure+of+human+pirin+an+iron-binding+nuclear+protein+and+transcription+cofactor&doi=10.1074%2Fjbc.M310022200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M310022200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M310022200%26sid%3Dliteratum%253Aachs%26aulast%3DPang%26aufirst%3DH.%26aulast%3DBartlam%26aufirst%3DM.%26aulast%3DZeng%26aufirst%3DQ.%26aulast%3DMiyatake%26aufirst%3DH.%26aulast%3DHisano%26aufirst%3DT.%26aulast%3DMiki%26aufirst%3DK.%26aulast%3DWong%26aufirst%3DL.%2BL.%26aulast%3DGao%26aufirst%3DG.%2BF.%26aulast%3DRao%26aufirst%3DZ.%26atitle%3DCrystal%2520structure%2520of%2520human%2520pirin%2520an%2520iron-binding%2520nuclear%2520protein%2520and%2520transcription%2520cofactor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D1491%26epage%3D1498%26doi%3D10.1074%2Fjbc.M310022200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Lindsley, C. W.</span><span> </span><span class="NLM_article-title">2013 Philip S. Portoghese Medicinal Chemistry Lectureship: Drug Discovery Targeting Allosteric Sites</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">7485</span><span class="NLM_x">–</span> <span class="NLM_lpage">7498</span><span class="refDoi"> DOI: 10.1021/jm5011786</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011786" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtLvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7485-7498&author=C.+W.+Lindsley&title=2013+Philip+S.+Portoghese+Medicinal+Chemistry+Lectureship%3A+Drug+Discovery+Targeting+Allosteric+Sites&doi=10.1021%2Fjm5011786"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">2013 Philip S. Portoghese Medicinal Chemistry Lectureship: Drug Discovery Targeting Allosteric Sites</span></div><div class="casAuthors">Lindsley, Craig W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7485-7498</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The identification of sites on receptors topog. distinct from the orthosteric sites, so-called allosteric sites, has heralded novel approaches and modes of pharmacol. for target modulation.  Over the past 20 years, our understanding of allosteric modulation has grown significantly, and numerous advantages, as well as caveats (e.g., flat structure-activity relationships, species differences, "mol. switches"), have been identified.  For multiple receptors and proteins, numerous examples have been described where unprecedented levels of selectivity are achieved along with improved physiochem. properties.  While not a panacea, these novel approaches represent exciting opportunities for tool compd. development to probe the pharmacol. and therapeutic potential of discrete mol. targets, as well as new medicines.  In this Perspective, in commemoration of the 2013 Philip S.  Portoghese Medicinal Chem. Lectureship (Lindsley, C. W. Adventures in allosteric drug discovery. Presented at the 246th National Meeting of the American Chem. Society, Indianapolis, IN, Sept. 10, 2013; The 2013 Portoghese Lectureship), several vignettes of drug discovery campaigns targeting novel allosteric mechanisms will be recounted, along with lessons learned and guidelines that have emerged for successful lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowaY56ZaUKc7Vg90H21EOLACvtfcHk0lioLbrF1yGiGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtLvE&md5=8372986d1e484efdd3e14312903d1d48</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fjm5011786&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011786%26sid%3Dliteratum%253Aachs%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26atitle%3D2013%2520Philip%2520S.%2520Portoghese%2520Medicinal%2520Chemistry%2520Lectureship%253A%2520Drug%2520Discovery%2520Targeting%2520Allosteric%2520Sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D7485%26epage%3D7498%26doi%3D10.1021%2Fjm5011786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Dosa, P. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amin, E. A.</span><span> </span><span class="NLM_article-title">Tactical approaches to interconverting GPCR agonists and antagonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">810</span><span class="NLM_x">–</span> <span class="NLM_lpage">840</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00982</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00982" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFamsr7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=810-840&author=P.+I.+Dosaauthor=E.+A.+Amin&title=Tactical+approaches+to+interconverting+GPCR+agonists+and+antagonists&doi=10.1021%2Facs.jmedchem.5b00982"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Tactical Approaches to Interconverting GPCR Agonists and Antagonists</span></div><div class="casAuthors">Dosa, Peter I.; Amin, Elizabeth Ambrose</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">810-840</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There are many reported examples of small structural modifications to GPCR-targeted ligands leading to major changes in their functional activity, converting agonists into antagonists or vice versa.  These shifts in functional activity are often accompanied by negligible changes in binding affinity.  The current perspective focuses on outlining and analyzing various approaches that have been used to interconvert GPCR agonists, partial agonists, and antagonists in order to achieve the intended functional activity at a GPCR of therapeutic interest.  An improved understanding of specific structural modifications that are likely to alter the functional activity of a GPCR ligand may be of use to researchers designing GPCR-targeted drugs and/or probe compds., specifically in cases where a particular ligand exhibits good potency but not the preferred functional activity at the GPCR of choice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohWg2-DQHq3LVg90H21EOLACvtfcHk0ljWDs1pxO8jSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFamsr7P&md5=99ffc0b892eb9c10aaf5d52a7d5dd8bc</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00982&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00982%26sid%3Dliteratum%253Aachs%26aulast%3DDosa%26aufirst%3DP.%2BI.%26aulast%3DAmin%26aufirst%3DE.%2BA.%26atitle%3DTactical%2520approaches%2520to%2520interconverting%2520GPCR%2520agonists%2520and%2520antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D810%26epage%3D840%26doi%3D10.1021%2Facs.jmedchem.5b00982" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Blomen, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Májek, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jae, L. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bigenzahn, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nieuwenhuis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staring, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sacco, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Diemen, F. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olk, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stukalov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marceau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janssen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carette, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colinge, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brummelkamp, T. R.</span><span> </span><span class="NLM_article-title">Gene essentiality and synthetic lethality in haploid human cells</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">350</span><span class="NLM_x">, </span> <span class="NLM_fpage">1092</span><span class="NLM_x">–</span> <span class="NLM_lpage">1096</span><span class="refDoi"> DOI: 10.1126/science.aac7557</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1126%2Fscience.aac7557" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=26472760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFamtL3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=350&publication_year=2015&pages=1092-1096&author=V.+A.+Blomenauthor=P.+M%C3%A1jekauthor=L.+T.+Jaeauthor=J.+W.+Bigenzahnauthor=J.+Nieuwenhuisauthor=J.+Staringauthor=R.+Saccoauthor=F.+R.+van+Diemenauthor=N.+Olkauthor=A.+Stukalovauthor=C.+Marceauauthor=H.+Janssenauthor=J.+E.+Caretteauthor=K.+L.+Bennettauthor=J.+Colingeauthor=G.+Superti-Furgaauthor=T.+R.+Brummelkamp&title=Gene+essentiality+and+synthetic+lethality+in+haploid+human+cells&doi=10.1126%2Fscience.aac7557"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Gene essentiality and synthetic lethality in haploid human cells</span></div><div class="casAuthors">Blomen, Vincent A.; Majek, Peter; Jae, Lucas T.; Bigenzahn, Johannes W.; Nieuwenhuis, Joppe; Staring, Jacqueline; Sacco, Roberto; van Diemen, Ferdy R.; Olk, Nadine; Stukalov, Alexey; Marceau, Caleb; Janssen, Hans; Carette, Jan E.; Bennett, Keiryn L.; Colinge, Jacques; Superti-Furga, Giulio; Brummelkamp, Thijn R.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">350</span>
        (<span class="NLM_cas:issue">6264</span>),
    <span class="NLM_cas:pages">1092-1096</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Although the genes essential for life have been identified in less complex model organisms, their elucidation in human cells has been hindered by tech. barriers.  We used extensive mutagenesis in haploid human cells to identify approx. 2000 genes required for optimal fitness under culture conditions.  To study the principles of genetic interactions in human cells, we created a synthetic lethality network focused on the secretory pathway based exclusively on mutations.  This revealed a genetic cross-talk governing Golgi homeostasis, an addnl. subunit of the human oligosaccharyltransferase complex, and a phosphatidylinositol 4-kinase β adaptor hijacked by viruses.  The synthetic lethality map parallels observations made in yeast and projects a route forward to reveal genetic networks in diverse aspects of human cell biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfqw6qG8hJG7Vg90H21EOLACvtfcHk0ljWDs1pxO8jSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFamtL3E&md5=bd6af396c2c5a67356d0fe578f64f1df</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1126%2Fscience.aac7557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aac7557%26sid%3Dliteratum%253Aachs%26aulast%3DBlomen%26aufirst%3DV.%2BA.%26aulast%3DM%25C3%25A1jek%26aufirst%3DP.%26aulast%3DJae%26aufirst%3DL.%2BT.%26aulast%3DBigenzahn%26aufirst%3DJ.%2BW.%26aulast%3DNieuwenhuis%26aufirst%3DJ.%26aulast%3DStaring%26aufirst%3DJ.%26aulast%3DSacco%26aufirst%3DR.%26aulast%3Dvan%2BDiemen%26aufirst%3DF.%2BR.%26aulast%3DOlk%26aufirst%3DN.%26aulast%3DStukalov%26aufirst%3DA.%26aulast%3DMarceau%26aufirst%3DC.%26aulast%3DJanssen%26aufirst%3DH.%26aulast%3DCarette%26aufirst%3DJ.%2BE.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DColinge%26aufirst%3DJ.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DBrummelkamp%26aufirst%3DT.%2BR.%26atitle%3DGene%2520essentiality%2520and%2520synthetic%2520lethality%2520in%2520haploid%2520human%2520cells%26jtitle%3DScience%26date%3D2015%26volume%3D350%26spage%3D1092%26epage%3D1096%26doi%3D10.1126%2Fscience.aac7557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birsoy, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krupczak, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Post, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lander, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabatini, D. M.</span><span> </span><span class="NLM_article-title">Identification and characterization of essential genes in the human genome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">350</span><span class="NLM_x">, </span> <span class="NLM_fpage">1096</span><span class="NLM_x">–</span> <span class="NLM_lpage">1101</span><span class="refDoi"> DOI: 10.1126/science.aac7041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1126%2Fscience.aac7041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=26472758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFamtL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=350&publication_year=2015&pages=1096-1101&author=T.+Wangauthor=K.+Birsoyauthor=N.+W.+Hughesauthor=K.+M.+Krupczakauthor=Y.+Postauthor=J.+J.+Weiauthor=E.+S.+Landerauthor=D.+M.+Sabatini&title=Identification+and+characterization+of+essential+genes+in+the+human+genome&doi=10.1126%2Fscience.aac7041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of essential genes in the human genome</span></div><div class="casAuthors">Wang, Tim; Birsoy, Kivanc; Hughes, Nicholas W.; Krupczak, Kevin M.; Post, Yorick; Wei, Jenny J.; Lander, Eric S.; Sabatini, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">350</span>
        (<span class="NLM_cas:issue">6264</span>),
    <span class="NLM_cas:pages">1096-1101</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Large-scale genetic anal. of lethal phenotypes has elucidated the mol. underpinnings of many biol. processes.  Using the bacterial clustered regularly interspaced short palindromic repeats (CRISPR) system, we constructed a genome-wide single-guide RNA library to screen for genes required for proliferation and survival in a human cancer cell line.  Our screen revealed the set of cell-essential genes, which was validated with an orthogonal gene-trap-based screen and comparison with yeast gene knockouts.  This set is enriched for genes that encode components of fundamental pathways, are expressed at high levels, and contain few inactivating polymorphisms in the human population.  We also uncovered a large group of uncharacterized genes involved in RNA processing, a no. of whose products localize to the nucleolus.  Last, screens in addnl. cell lines showed a high degree of overlap in gene essentiality but also revealed differences specific to each cell line and cancer type that reflect the developmental origin, oncogenic drivers, paralogous gene expression pattern, and chromosomal structure of each line.  These results demonstrate the power of CRISPR-based screens and suggest a general strategy for identifying liabilities in cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonjG02K7R4iLVg90H21EOLACvtfcHk0ljWDs1pxO8jSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFamtL3F&md5=61273ba03938104748f0458f662a3c34</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1126%2Fscience.aac7041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aac7041%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DT.%26aulast%3DBirsoy%26aufirst%3DK.%26aulast%3DHughes%26aufirst%3DN.%2BW.%26aulast%3DKrupczak%26aufirst%3DK.%2BM.%26aulast%3DPost%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DJ.%2BJ.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520essential%2520genes%2520in%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2015%26volume%3D350%26spage%3D1096%26epage%3D1101%26doi%3D10.1126%2Fscience.aac7041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Staben, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyle, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boggs, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burch, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chua, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiPasquale, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedman, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heise, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koeppen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mintzer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rouge, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tam, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoeflich, K. P.</span><span> </span><span class="NLM_article-title">Back pocket flexibility provides group II p21-activated kinase (PAK) selectivity for type I 1/2 kinase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">1033</span><span class="NLM_x">–</span> <span class="NLM_lpage">1045</span><span class="refDoi"> DOI: 10.1021/jm401768t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401768t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotVGmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1033-1045&author=S.+T.+Stabenauthor=J.+A.+Fengauthor=K.+Lyleauthor=M.+Belvinauthor=J.+Boggsauthor=J.+D.+Burchauthor=C.+Chuaauthor=H.+Cuiauthor=A.+G.+DiPasqualeauthor=L.+S.+Friedmanauthor=C.+Heiseauthor=H.+Koeppenauthor=A.+Koteyauthor=R.+Mintzerauthor=A.+Ohauthor=D.+A.+Robertsauthor=L.+Rougeauthor=J.+Rudolphauthor=C.+Tamauthor=W.+Wangauthor=Y.+Xiaoauthor=A.+Youngauthor=Y.+Zhangauthor=K.+P.+Hoeflich&title=Back+pocket+flexibility+provides+group+II+p21-activated+kinase+%28PAK%29+selectivity+for+type+I+1%2F2+kinase+inhibitors&doi=10.1021%2Fjm401768t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73aR"><div class="casContent"><span class="casTitleNuber">73a</span><div class="casTitle"><span class="NLM_cas:atitle">Back Pocket Flexibility Provides Group II p21-Activated Kinase (PAK) Selectivity for Type I 1/2 Kinase Inhibitors</span></div><div class="casAuthors">Staben, Steven T.; Feng, Jianwen A.; Lyle, Karen; Belvin, Marcia; Boggs, Jason; Burch, Jason D.; Chua, Ching-ching; Cui, Haifeng; DiPasquale, Antonio G.; Friedman, Lori S.; Heise, Christopher; Koeppen, Hartmut; Kotey, Adrian; Mintzer, Robert; Oh, Angela; Roberts, David Allen; Rouge, Lionel; Rudolph, Joachim; Tam, Christine; Wang, Weiru; Xiao, Yisong; Young, Amy; Zhang, Yamin; Hoeflich, Klaus P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1033-1045</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-based methods were used to design a potent and highly selective group II p21-activated kinase (PAK) inhibitor with a novel binding mode, compd. I.  Hydrophobic interactions within a lipophilic pocket past the methionine gatekeeper of group II PAKs approached by these type I 1/2 binders were found to be important for improving potency.  A structure-based hypothesis and strategy for achieving selectivity over group I PAKs, and the broad kinome, based on unique flexibility of this lipophilic pocket, is presented.  A concn.-dependent decrease in tumor cell migration and invasion in two triple-neg. breast cancer cell lines was obsd. with compd. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrogBhcjJGWm7Vg90H21EOLACvtfcHk0lg-kY63v0yJHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotVGmsQ%253D%253D&md5=08b11088bc4d806cc61e16c932b68cf6</span></div><a href="/servlet/linkout?suffix=cit73a&amp;dbid=16384&amp;doi=10.1021%2Fjm401768t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401768t%26sid%3Dliteratum%253Aachs%26aulast%3DStaben%26aufirst%3DS.%2BT.%26aulast%3DFeng%26aufirst%3DJ.%2BA.%26aulast%3DLyle%26aufirst%3DK.%26aulast%3DBelvin%26aufirst%3DM.%26aulast%3DBoggs%26aufirst%3DJ.%26aulast%3DBurch%26aufirst%3DJ.%2BD.%26aulast%3DChua%26aufirst%3DC.%26aulast%3DCui%26aufirst%3DH.%26aulast%3DDiPasquale%26aufirst%3DA.%2BG.%26aulast%3DFriedman%26aufirst%3DL.%2BS.%26aulast%3DHeise%26aufirst%3DC.%26aulast%3DKoeppen%26aufirst%3DH.%26aulast%3DKotey%26aufirst%3DA.%26aulast%3DMintzer%26aufirst%3DR.%26aulast%3DOh%26aufirst%3DA.%26aulast%3DRoberts%26aufirst%3DD.%2BA.%26aulast%3DRouge%26aufirst%3DL.%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DTam%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DYoung%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26atitle%3DBack%2520pocket%2520flexibility%2520provides%2520group%2520II%2520p21-activated%2520kinase%2520%2528PAK%2529%2520selectivity%2520for%2520type%2520I%25201%252F2%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D1033%26epage%3D1045%26doi%3D10.1021%2Fjm401768t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit73b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Ingrassia, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lefranc, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dewelle, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pottier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieu, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spiegl-Kreinecker, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauvage, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El Yazidi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dehoux, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Quaquebeke, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiss, R.</span><span> </span><span class="NLM_article-title">Structure–activity relationship analysis of novel derivatives of narciclasine (an amaryllidaceae isocarbostyril derivative) as potential anticancer agents</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1100</span><span class="NLM_x">–</span> <span class="NLM_lpage">1114</span><span class="refDoi"> DOI: 10.1021/jm8013585</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8013585" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=1100-1114&author=L.+Ingrassiaauthor=F.+Lefrancauthor=J.+Dewelleauthor=L.+Pottierauthor=V.+Mathieuauthor=S.+Spiegl-Kreineckerauthor=S.+Sauvageauthor=M.+El+Yazidiauthor=M.+Dehouxauthor=W.+Bergerauthor=E.+Van+Quaquebekeauthor=R.+Kiss&title=Structure%E2%80%93activity+relationship+analysis+of+novel+derivatives+of+narciclasine+%28an+amaryllidaceae+isocarbostyril+derivative%29+as+potential+anticancer+agents&doi=10.1021%2Fjm8013585"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73b&amp;dbid=16384&amp;doi=10.1021%2Fjm8013585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8013585%26sid%3Dliteratum%253Aachs%26aulast%3DIngrassia%26aufirst%3DL.%26aulast%3DLefranc%26aufirst%3DF.%26aulast%3DDewelle%26aufirst%3DJ.%26aulast%3DPottier%26aufirst%3DL.%26aulast%3DMathieu%26aufirst%3DV.%26aulast%3DSpiegl-Kreinecker%26aufirst%3DS.%26aulast%3DSauvage%26aufirst%3DS.%26aulast%3DEl%2BYazidi%26aufirst%3DM.%26aulast%3DDehoux%26aufirst%3DM.%26aulast%3DBerger%26aufirst%3DW.%26aulast%3DVan%2BQuaquebeke%26aufirst%3DE.%26aulast%3DKiss%26aufirst%3DR.%26atitle%3DStructure%25E2%2580%2593activity%2520relationship%2520analysis%2520of%2520novel%2520derivatives%2520of%2520narciclasine%2520%2528an%2520amaryllidaceae%2520isocarbostyril%2520derivative%2529%2520as%2520potential%2520anticancer%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D1100%26epage%3D1114%26doi%3D10.1021%2Fjm8013585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit73c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Singh, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tamang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajamohanan, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jima, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakraborty, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kundu, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaikwad, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khan, M. I.</span><span> </span><span class="NLM_article-title">Isolation, structure, and functional elucidation of a modified pentapeptide, Cysteine protease inhibitor (CPI-2081) from streptomyces species 2081 that exhibit inhibitory effect on cancer cell migration</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5121</span><span class="NLM_x">–</span> <span class="NLM_lpage">5128</span><span class="refDoi"> DOI: 10.1021/jm9014179</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9014179" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5121-5128&author=J.+P.+Singhauthor=S.+Tamangauthor=P.+R.+Rajamohananauthor=N.+C.+Jimaauthor=G.+Chakrabortyauthor=G.+C.+Kunduauthor=S.+M.+Gaikwadauthor=M.+I.+Khan&title=Isolation%2C+structure%2C+and+functional+elucidation+of+a+modified+pentapeptide%2C+Cysteine+protease+inhibitor+%28CPI-2081%29+from+streptomyces+species+2081+that+exhibit+inhibitory+effect+on+cancer+cell+migration&doi=10.1021%2Fjm9014179"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73c&amp;dbid=16384&amp;doi=10.1021%2Fjm9014179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9014179%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DJ.%2BP.%26aulast%3DTamang%26aufirst%3DS.%26aulast%3DRajamohanan%26aufirst%3DP.%2BR.%26aulast%3DJima%26aufirst%3DN.%2BC.%26aulast%3DChakraborty%26aufirst%3DG.%26aulast%3DKundu%26aufirst%3DG.%2BC.%26aulast%3DGaikwad%26aufirst%3DS.%2BM.%26aulast%3DKhan%26aufirst%3DM.%2BI.%26atitle%3DIsolation%252C%2520structure%252C%2520and%2520functional%2520elucidation%2520of%2520a%2520modified%2520pentapeptide%252C%2520Cysteine%2520protease%2520inhibitor%2520%2528CPI-2081%2529%2520from%2520streptomyces%2520species%25202081%2520that%2520exhibit%2520inhibitory%2520effect%2520on%2520cancer%2520cell%2520migration%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5121%26epage%3D5128%26doi%3D10.1021%2Fjm9014179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit73d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Nagababu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barui, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thulasiram, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devi, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Satyanarayana, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patra, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sreedhar, B.</span><span> </span><span class="NLM_article-title">Antiangiogenic activity of mononuclear copper(II) polypyridyl complexes for the treatment of cancers</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">5226</span><span class="NLM_x">–</span> <span class="NLM_lpage">5241</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00651</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00651" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVans7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5226-5241&author=P.+Nagababuauthor=A.+K.+Baruiauthor=B.+Thulasiramauthor=C.+S.+Deviauthor=S.+Satyanarayanaauthor=C.+R.+Patraauthor=B.+Sreedhar&title=Antiangiogenic+activity+of+mononuclear+copper%28II%29+polypyridyl+complexes+for+the+treatment+of+cancers&doi=10.1021%2Facs.jmedchem.5b00651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73dR"><div class="casContent"><span class="casTitleNuber">73d</span><div class="casTitle"><span class="NLM_cas:atitle">Antiangiogenic Activity of Mononuclear Copper(II) Polypyridyl Complexes for the Treatment of Cancers</span></div><div class="casAuthors">Nagababu, Penumaka; Barui, Ayan Kumar; Thulasiram, Bathini; Devi, C. Shobha; Satyanarayana, S.; Patra, Chitta Ranjan; Sreedhar, Bojja</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5226-5241</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of four new mononuclear copper(II) polypyridyl complexes (1-4) have been designed, developed, and thoroughly characterized by several physicochem. techniques.  The CT-DNA binding properties of 1-4 have been investigated by absorption, emission spectroscopy, and viscosity measurements.  All the complexes esp. 1 and 4 exhibit cytotoxicity toward several cancer cell lines, suggesting their anticancer properties as obsd. by several in vitro assays.  Addnl., the complexes show inhibition of endothelial cell (HUVECs) proliferation, indicating their antiangiogenic nature.  In vivo chick embryo angiogenesis assay again confirms the antiangiogenic properties of 1 and 4.  The formation of excessive intracellular ROS (H2O2 and O2•-) and upregulation of BAX induced by copper(II) complexes may be the plausible mechanisms behind their anticancer activities.  The present study may offer a basis for the development of new transition metal complexes through suitable choice of ligands for cancer therapeutics by controlling tumor angiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpo-a7VLgXWKbVg90H21EOLACvtfcHk0lilmccXgTB47g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVans7jI&md5=042e41e7346f577fe274d0f16903ec33</span></div><a href="/servlet/linkout?suffix=cit73d&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00651%26sid%3Dliteratum%253Aachs%26aulast%3DNagababu%26aufirst%3DP.%26aulast%3DBarui%26aufirst%3DA.%2BK.%26aulast%3DThulasiram%26aufirst%3DB.%26aulast%3DDevi%26aufirst%3DC.%2BS.%26aulast%3DSatyanarayana%26aufirst%3DS.%26aulast%3DPatra%26aufirst%3DC.%2BR.%26aulast%3DSreedhar%26aufirst%3DB.%26atitle%3DAntiangiogenic%2520activity%2520of%2520mononuclear%2520copper%2528II%2529%2520polypyridyl%2520complexes%2520for%2520the%2520treatment%2520of%2520cancers%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5226%26epage%3D5241%26doi%3D10.1021%2Facs.jmedchem.5b00651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit73e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Amin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wani, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farooq, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nayak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakraborty, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shankar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">ur Rasool, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koul, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goswami, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rai, R.</span><span> </span><span class="NLM_article-title">Inhibition of invasion in pancreatic cancer cells by conjugate of EPA with β3,3-Pip-OH via PI3K/Akt/NF-kB pathway</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1071</span><span class="NLM_x">–</span> <span class="NLM_lpage">1074</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00257</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00257" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=1071-1074&author=H.+Aminauthor=N.+A.+Waniauthor=S.+Farooqauthor=D.+Nayakauthor=S.+Chakrabortyauthor=S.+Shankarauthor=R.+ur+Rasoolauthor=S.+Koulauthor=A.+Goswamiauthor=R.+Rai&title=Inhibition+of+invasion+in+pancreatic+cancer+cells+by+conjugate+of+EPA+with+%CE%B23%2C3-Pip-OH+via+PI3K%2FAkt%2FNF-kB+pathway&doi=10.1021%2Facsmedchemlett.5b00257"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73e&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00257%26sid%3Dliteratum%253Aachs%26aulast%3DAmin%26aufirst%3DH.%26aulast%3DWani%26aufirst%3DN.%2BA.%26aulast%3DFarooq%26aufirst%3DS.%26aulast%3DNayak%26aufirst%3DD.%26aulast%3DChakraborty%26aufirst%3DS.%26aulast%3DShankar%26aufirst%3DS.%26aulast%3Dur%2BRasool%26aufirst%3DR.%26aulast%3DKoul%26aufirst%3DS.%26aulast%3DGoswami%26aufirst%3DA.%26aulast%3DRai%26aufirst%3DR.%26atitle%3DInhibition%2520of%2520invasion%2520in%2520pancreatic%2520cancer%2520cells%2520by%2520conjugate%2520of%2520EPA%2520with%2520%25CE%25B23%252C3-Pip-OH%2520via%2520PI3K%252FAkt%252FNF-kB%2520pathway%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D1071%26epage%3D1074%26doi%3D10.1021%2Facsmedchemlett.5b00257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="note"><p class="first last">In-house PAINS filter protocol: Biovia Pipeline Pilot version 9.5.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Lagerlund, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larhed, M.</span><span> </span><span class="NLM_article-title">Microwave-promoted aminocarbonylations of aryl chlorides using Mo(CO)<sub>6</sub> as a solid carbon monoxide source</span> <span class="citation_source-journal">J. Comb. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">4</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1021/cc050102r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cc050102r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Gms7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2006&pages=4-6&author=O.+Lagerlundauthor=M.+Larhed&title=Microwave-promoted+aminocarbonylations+of+aryl+chlorides+using+Mo%28CO%296+as+a+solid+carbon+monoxide+source&doi=10.1021%2Fcc050102r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Microwave-Promoted Aminocarbonylations of Aryl Chlorides Using Mo(CO)6 as a Solid Carbon Monoxide Source</span></div><div class="casAuthors">Lagerlund, Olof; Larhed, Mats</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Combinatorial Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-6</span>CODEN:
                <span class="NLM_cas:coden">JCCHFF</span>;
        ISSN:<span class="NLM_cas:issn">1520-4766</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Both electron-rich and electron-poor aryl chlorides serve as useful substrates in microwave-heated in situ aminocarbonylation reactions under noninert conditions.  Both aliph. amines and anilines provide benzamide products in good yields after only 15-25 min of irradn.  The reaction was catalyzed by Herrmann's palladacycle and [(t-Bu)3PH]BF4 in presence of DBU and Mo(CO)6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgdoXvL_HaVLVg90H21EOLACvtfcHk0liwsMYyBFMPdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Gms7nP&md5=979264c72a974f6ca64be5b3d44a0c88</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fcc050102r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcc050102r%26sid%3Dliteratum%253Aachs%26aulast%3DLagerlund%26aufirst%3DO.%26aulast%3DLarhed%26aufirst%3DM.%26atitle%3DMicrowave-promoted%2520aminocarbonylations%2520of%2520aryl%2520chlorides%2520using%2520Mo%2528CO%25296%2520as%2520a%2520solid%2520carbon%2520monoxide%2520source%26jtitle%3DJ.%2520Comb.%2520Chem.%26date%3D2006%26volume%3D8%26spage%3D4%26epage%3D6%26doi%3D10.1021%2Fcc050102r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1a','cit1b','cit1c','cit1d'],'ref2':['cit2a','cit2b'],'ref3':['cit3a','cit3b'],'ref4':['cit4a','cit4b','cit4c','cit4d'],'ref5':['cit5a','cit5b','cit5c'],'ref6':['cit6a','cit6b','cit6c','cit6d'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9a','cit9b','cit9c'],'ref10':['cit10a','cit10b'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13a','cit13b','cit13c'],'ref14':['cit14a','cit14b','cit14c'],'ref15':['cit15'],'ref16':['cit16a','cit16b','cit16c'],'ref17':['cit17a','cit17b','cit17c','cit17d','cit17e','cit17f','cit17g'],'ref18':['cit18'],'ref19':['cit19a','cit19b','cit19c'],'ref20':['cit20'],'ref21':['cit21a','cit21b','cit21c','cit21d'],'ref22':[],'ref23':['cit23'],'ref24':['cit24'],'ref25':[],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29a','cit29b'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':[],'ref34':['cit34'],'ref35':['cit35a','cit35a1','cit35b','cit35c','cit35d','cit35e'],'ref36':['cit36a','cit36b','cit36c','cit36d','cit36e'],'ref37':['cit37'],'ref38':['cit38'],'ref39':[],'ref40':['cit40'],'ref41':['cit41'],'ref42':[],'ref43':['cit43'],'ref44':[],'ref45':[],'ref46':['cit46'],'ref47':[],'ref48':[],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51a','cit51b'],'ref52':['cit52'],'ref53':['cit53'],'ref54':[],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58a','cit58b'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63a','cit63b'],'ref64':[],'ref65':['cit65'],'ref66':['cit66a','cit66b'],'ref67':['cit67a','cit67b'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73a','cit73b','cit73c','cit73d','cit73e'],'ref74':[],'ref75':['cit75']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 28 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Erika M. Lisabeth, Dylan Kahl, Indiwari Gopallawa, Sarah E. Haynes, Sean A. Misek, Phillip L. Campbell, Thomas S. Dexheimer, Dinesh Khanna, David A. Fox, Xiangshu Jin, Brent R. Martin, Scott D. Larsen, <span class="NLM_string-name hlFld-ContribAuthor">Richard R. Neubig</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Pirin as a Molecular Target of the CCG-1423/CCG-203971 Series of Antifibrotic and Antimetastatic Compounds. </span><span class="cited-content_cbyCitation_journal-name">ACS Pharmacology & Translational Science</span><span> <strong>2019,</strong> <em>2 </em>
                                    (2)
                                     , 92-100. <a href="https://doi.org/10.1021/acsptsci.8b00048" title="DOI URL">https://doi.org/10.1021/acsptsci.8b00048</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsptsci.8b00048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsptsci.8b00048%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Pharmacology%2520%2526%2520Translational%2520Science%26atitle%3DIdentification%252Bof%252BPirin%252Bas%252Ba%252BMolecular%252BTarget%252Bof%252Bthe%252BCCG-1423%25252FCCG-203971%252BSeries%252Bof%252BAntifibrotic%252Band%252BAntimetastatic%252BCompounds%26aulast%3DLisabeth%26aufirst%3DErika%2BM.%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D13112018%26date%3D27032019%26date%3D18032019%26volume%3D2%26issue%3D2%26spage%3D92%26epage%3D100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sabine Depauw, Mélanie Lambert, Samy Jambon, Ananya Paul, Paul Peixoto, Raja Nhili, Laura Marongiu, Martin Figeac, Christelle Dassi, Charles Paul-Constant, Benjamin Billoré, Arvind Kumar, Abdelbasset A. Farahat, Mohamed A. Ismail, Ekaterina Mineva, Daniel P. Sweat, Chad E. Stephens, David W. Boykin, W. David Wilson, <span class="NLM_string-name hlFld-ContribAuthor">Marie-Hélène David-Cordonnier</span>. </span><span class="cited-content_cbyCitation_article-title">Heterocyclic Diamidine DNA Ligands as HOXA9 Transcription Factor Inhibitors: Design, Molecular Evaluation, and Cellular Consequences in a HOXA9-Dependant Leukemia Cell Model. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (3)
                                     , 1306-1329. <a href="https://doi.org/10.1021/acs.jmedchem.8b01448" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01448</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01448&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01448%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHeterocyclic%252BDiamidine%252BDNA%252BLigands%252Bas%252BHOXA9%252BTranscription%252BFactor%252BInhibitors%25253A%252BDesign%25252C%252BMolecular%252BEvaluation%25252C%252Band%252BCellular%252BConsequences%252Bin%252Ba%252BHOXA9-Dependant%252BLeukemia%252BCell%252BModel%26aulast%3DDepauw%26aufirst%3DSabine%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D18092018%26date%3D01022019%26date%3D15012019%26volume%3D62%26issue%3D3%26spage%3D1306%26epage%3D1329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Joshua Meyers, Nicola E. A. Chessum, Salyha Ali, N. Yi Mok, Birgit Wilding, A. Elisa Pasqua, Martin Rowlands, Michael J. Tucker, Lindsay E. Evans, Carl S. Rye, Lisa O’Fee, Yann-Vaï Le Bihan, Rosemary Burke, Michael Carter, Paul Workman, Julian Blagg, Nathan Brown, Rob L. M. van Montfort, Keith Jones, <span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Cheeseman</span>. </span><span class="cited-content_cbyCitation_article-title">Privileged Structures and Polypharmacology within and between Protein Families. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>9 </em>
                                    (12)
                                     , 1199-1204. <a href="https://doi.org/10.1021/acsmedchemlett.8b00364" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.8b00364</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.8b00364&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.8b00364%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DPrivileged%252BStructures%252Band%252BPolypharmacology%252Bwithin%252Band%252Bbetween%252BProtein%252BFamilies%26aulast%3DMeyers%26aufirst%3DJoshua%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D09082018%26date%3D16112018%26date%3D26112018%26date%3D16112018%26volume%3D9%26issue%3D12%26spage%3D1199%26epage%3D1204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nicola E. A.  Chessum</span>, <span class="hlFld-ContribAuthor ">Swee Y.  Sharp</span>, <span class="hlFld-ContribAuthor ">John J.  Caldwell</span>, <span class="hlFld-ContribAuthor ">A. Elisa  Pasqua</span>, <span class="hlFld-ContribAuthor ">Birgit  Wilding</span>, <span class="hlFld-ContribAuthor ">Giampiero  Colombano</span>, <span class="hlFld-ContribAuthor ">Ian  Collins</span>, <span class="hlFld-ContribAuthor ">Bugra  Ozer</span>, <span class="hlFld-ContribAuthor ">Meirion  Richards</span>, <span class="hlFld-ContribAuthor ">Martin  Rowlands</span>, <span class="hlFld-ContribAuthor ">Mark  Stubbs</span>, <span class="hlFld-ContribAuthor ">Rosemary  Burke</span>, <span class="hlFld-ContribAuthor ">P. Craig  McAndrew</span>, <span class="hlFld-ContribAuthor ">Paul A.  Clarke</span>, <span class="hlFld-ContribAuthor ">Paul  Workman</span>, <span class="hlFld-ContribAuthor ">Matthew D.  Cheeseman</span>, and <span class="hlFld-ContribAuthor ">Keith  Jones</span>  . </span><span class="cited-content_cbyCitation_article-title">Demonstrating In-Cell Target Engagement Using a Pirin Protein Degradation Probe (CCT367766). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (3)
                                     , 918-933. <a href="https://doi.org/10.1021/acs.jmedchem.7b01406" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01406</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01406%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDemonstrating%252BIn-Cell%252BTarget%252BEngagement%252BUsing%252Ba%252BPirin%252BProtein%252BDegradation%252BProbe%252B%252528CCT367766%252529%26aulast%3DChessum%26aufirst%3DNicola%2BE.%2BA.%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D22092017%26date%3D05012018%26date%3D08022018%26date%3D14122017%26volume%3D61%26issue%3D3%26spage%3D918%26epage%3D933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthias  Bach</span>, <span class="hlFld-ContribAuthor ">Anna  Lehmann</span>, <span class="hlFld-ContribAuthor ">Daniela  Brünnert</span>, <span class="hlFld-ContribAuthor ">Jens T.  Vanselow</span>, <span class="hlFld-ContribAuthor ">Andreas  Hartung</span>, <span class="hlFld-ContribAuthor ">Ralf C.  Bargou</span>, <span class="hlFld-ContribAuthor ">Ulrike  Holzgrabe</span>, <span class="hlFld-ContribAuthor ">Andreas  Schlosser</span>, and <span class="hlFld-ContribAuthor ">Manik  Chatterjee</span>  . </span><span class="cited-content_cbyCitation_article-title">Ugi Reaction-Derived α-Acyl Aminocarboxamides Bind to Phosphatidylinositol 3-Kinase-Related Kinases, Inhibit HSF1-Dependent Heat Shock Response, and Induce Apoptosis in Multiple Myeloma Cells. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (10)
                                     , 4147-4160. <a href="https://doi.org/10.1021/acs.jmedchem.6b01613" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01613</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01613%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DUgi%252BReaction-Derived%252B%2525CE%2525B1-Acyl%252BAminocarboxamides%252BBind%252Bto%252BPhosphatidylinositol%252B3-Kinase-Related%252BKinases%25252C%252BInhibit%252BHSF1-Dependent%252BHeat%252BShock%252BResponse%25252C%252Band%252BInduce%252BApoptosis%252Bin%252BMultiple%252BMyeloma%252BCells%26aulast%3DBach%26aufirst%3DMatthias%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D03112016%26date%3D08052017%26date%3D25052017%26date%3D28042017%26volume%3D60%26issue%3D10%26spage%3D4147%26epage%3D4160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kathryn M.  Appleton</span>, <span class="hlFld-ContribAuthor ">Charuta C.  Palsuledesai</span>, <span class="hlFld-ContribAuthor ">Sean A.  Misek</span>, <span class="hlFld-ContribAuthor ">Maja  Blake</span>, <span class="hlFld-ContribAuthor ">Joseph  Zagorski</span>, <span class="hlFld-ContribAuthor ">Kathleen A.  Gallo</span>, <span class="hlFld-ContribAuthor ">Thomas S.  Dexheimer</span>, <span class="hlFld-ContribAuthor ">Richard R.  Neubig</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of the Myocardin-Related Transcription Factor Pathway Increases Efficacy of Trametinib in NRAS-Mutant Melanoma Cell Lines. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (9)
                                     , 2012. <a href="https://doi.org/10.3390/cancers13092012" title="DOI URL">https://doi.org/10.3390/cancers13092012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13092012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13092012%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DInhibition%252Bof%252Bthe%252BMyocardin-Related%252BTranscription%252BFactor%252BPathway%252BIncreases%252BEfficacy%252Bof%252BTrametinib%252Bin%252BNRAS-Mutant%252BMelanoma%252BCell%252BLines%26aulast%3DAppleton%26aufirst%3DKathryn%2BM.%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D9%26spage%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lyn H.  Jones</span>. </span><span class="cited-content_cbyCitation_article-title">Medicinal Chemical Biology. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 1-37. <a href="https://doi.org/10.1002/0471266949.bmc256" title="DOI URL">https://doi.org/10.1002/0471266949.bmc256</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/0471266949.bmc256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F0471266949.bmc256%26sid%3Dliteratum%253Aachs%26atitle%3DMedicinal%252BChemical%252BBiology%26aulast%3DJones%26aufirst%3DLyn%2BH.%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D37%26pub%3DWiley%26atitle%3DBurger%252527s%252BMedicinal%252BChemistry%252Band%252BDrug%252BDiscovery%26aulast%3DAbraham%26aufirst%3DDonald%2BJ.%26date%3D2003%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Radosław P.  Nowak</span>, <span class="hlFld-ContribAuthor ">Lyn H.  Jones</span>. </span><span class="cited-content_cbyCitation_article-title">Target Validation Using PROTACs: Applying the Four Pillars Framework. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2021,</strong> <em>26 </em>
                                    (4)
                                     , 474-483. <a href="https://doi.org/10.1177/2472555220979584" title="DOI URL">https://doi.org/10.1177/2472555220979584</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/2472555220979584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F2472555220979584%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DTarget%252BValidation%252BUsing%252BPROTACs%25253A%252BApplying%252Bthe%252BFour%252BPillars%252BFramework%26aulast%3DNowak%26aufirst%3DRados%25C5%2582aw%2BP.%26date%3D2021%26date%3D2020%26volume%3D26%26issue%3D4%26spage%3D474%26epage%3D483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongzhao  Wang</span>, <span class="hlFld-ContribAuthor ">Zijian  Qin</span>, <span class="hlFld-ContribAuthor ">Aixia  Yan</span>. </span><span class="cited-content_cbyCitation_article-title">Classification models and SAR analysis on CysLT1 receptor antagonists using machine learning algorithms. </span><span class="cited-content_cbyCitation_journal-name">Molecular Diversity</span><span> <strong>2021,</strong> <em>69 </em><a href="https://doi.org/10.1007/s11030-020-10165-4" title="DOI URL">https://doi.org/10.1007/s11030-020-10165-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11030-020-10165-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11030-020-10165-4%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Diversity%26atitle%3DClassification%252Bmodels%252Band%252BSAR%252Banalysis%252Bon%252BCysLT1%252Breceptor%252Bantagonists%252Busing%252Bmachine%252Blearning%252Balgorithms%26aulast%3DWang%26aufirst%3DHongzhao%26date%3D2021%26date%3D2021%26volume%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Frederico Hillesheim  Horst</span>, <span class="hlFld-ContribAuthor ">Carime Vitória da Silva  Rodrigues</span>, <span class="hlFld-ContribAuthor ">Pedro Henrique Pimenta Rocha  Carvalho</span>, <span class="hlFld-ContribAuthor ">Amanda Monteiro  Leite</span>, <span class="hlFld-ContribAuthor ">Ricardo Bentes  Azevedo</span>, <span class="hlFld-ContribAuthor ">Brenno A. D.  Neto</span>, <span class="hlFld-ContribAuthor ">José Raimundo  Corrêa</span>, <span class="hlFld-ContribAuthor ">Mônica Pereira  Garcia</span>, <span class="hlFld-ContribAuthor ">Saud  Alotaibi</span>, <span class="hlFld-ContribAuthor ">Mohamed  Henini</span>, <span class="hlFld-ContribAuthor ">Sacha Braun  Chaves</span>, <span class="hlFld-ContribAuthor ">Marcelo Oliveira  Rodrigues</span>. </span><span class="cited-content_cbyCitation_article-title">From cow manure to bioactive carbon dots: a light-up probe for bioimaging investigations, glucose detection and potential immunotherapy agent for melanoma skin cancer. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2021,</strong> <em>11 </em>
                                    (11)
                                     , 6346-6352. <a href="https://doi.org/10.1039/D0RA10859F" title="DOI URL">https://doi.org/10.1039/D0RA10859F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0RA10859F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0RA10859F%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DFrom%252Bcow%252Bmanure%252Bto%252Bbioactive%252Bcarbon%252Bdots%25253A%252Ba%252Blight-up%252Bprobe%252Bfor%252Bbioimaging%252Binvestigations%25252C%252Bglucose%252Bdetection%252Band%252Bpotential%252Bimmunotherapy%252Bagent%252Bfor%252Bmelanoma%252Bskin%252Bcancer%26aulast%3DHorst%26aufirst%3DFrederico%2BHillesheim%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D11%26spage%3D6346%26epage%3D6352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francisco  Perez-Dominguez</span>, <span class="hlFld-ContribAuthor ">Diego  Carrillo-Beltrán</span>, <span class="hlFld-ContribAuthor ">Rancés  Blanco</span>, <span class="hlFld-ContribAuthor ">Juan P.  Muñoz</span>, <span class="hlFld-ContribAuthor ">Grettell  León-Cruz</span>, <span class="hlFld-ContribAuthor ">Alejandro H.  Corvalan</span>, <span class="hlFld-ContribAuthor ">Ulises  Urzúa</span>, <span class="hlFld-ContribAuthor ">Gloria M.  Calaf</span>, <span class="hlFld-ContribAuthor ">Francisco  Aguayo</span>. </span><span class="cited-content_cbyCitation_article-title">Role of Pirin, an Oxidative Stress Sensor Protein, in Epithelial Carcinogenesis. </span><span class="cited-content_cbyCitation_journal-name">Biology</span><span> <strong>2021,</strong> <em>10 </em>
                                    (2)
                                     , 116. <a href="https://doi.org/10.3390/biology10020116" title="DOI URL">https://doi.org/10.3390/biology10020116</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biology10020116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiology10020116%26sid%3Dliteratum%253Aachs%26jtitle%3DBiology%26atitle%3DRole%252Bof%252BPirin%25252C%252Ban%252BOxidative%252BStress%252BSensor%252BProtein%25252C%252Bin%252BEpithelial%252BCarcinogenesis%26aulast%3DPerez-Dominguez%26aufirst%3DFrancisco%26date%3D2021%26date%3D2021%26volume%3D10%26issue%3D2%26spage%3D116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lyn H.  Jones</span>. </span><span class="cited-content_cbyCitation_article-title">Target Validation—Prosecuting the Target. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1016/B978-0-12-820472-6.00014-1" title="DOI URL">https://doi.org/10.1016/B978-0-12-820472-6.00014-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-820472-6.00014-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-820472-6.00014-1%26sid%3Dliteratum%253Aachs%26atitle%3DTarget%252BValidation%2525E2%252580%252594Prosecuting%252Bthe%252BTarget%26aulast%3DJones%26aufirst%3DLyn%2BH.%26date%3D2021%26pub%3DElsevier%26atitle%3DReference%252BModule%252Bin%252BBiomedical%252BSciences%26date%3D2021%26volume%3D34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John F.  Darby</span>, <span class="hlFld-ContribAuthor ">Lewis R.  Vidler</span>, <span class="hlFld-ContribAuthor ">Peter J.  Simpson</span>, <span class="hlFld-ContribAuthor ">Bissan  Al-Lazikani</span>, <span class="hlFld-ContribAuthor ">Stephen J.  Matthews</span>, <span class="hlFld-ContribAuthor ">Swee Y.  Sharp</span>, <span class="hlFld-ContribAuthor ">Laurence H.  Pearl</span>, <span class="hlFld-ContribAuthor ">Swen  Hoelder</span>, <span class="hlFld-ContribAuthor ">Paul  Workman</span>. </span><span class="cited-content_cbyCitation_article-title">Solution structure of the Hop TPR2A domain and investigation of target druggability by NMR, biochemical and in silico approaches. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2020,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-020-71969-w" title="DOI URL">https://doi.org/10.1038/s41598-020-71969-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-020-71969-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-020-71969-w%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DSolution%252Bstructure%252Bof%252Bthe%252BHop%252BTPR2A%252Bdomain%252Band%252Binvestigation%252Bof%252Btarget%252Bdruggability%252Bby%252BNMR%25252C%252Bbiochemical%252Band%252Bin%252Bsilico%252Bapproaches%26aulast%3DDarby%26aufirst%3DJohn%2BF.%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anna E.  Masser</span>, <span class="hlFld-ContribAuthor ">Michela  Ciccarelli</span>, <span class="hlFld-ContribAuthor ">Claes  Andréasson</span>. </span><span class="cited-content_cbyCitation_article-title">Hsf1 on a leash – controlling the heat shock response by chaperone titration. </span><span class="cited-content_cbyCitation_journal-name">Experimental Cell Research</span><span> <strong>2020,</strong> <em>396 </em>
                                    (1)
                                     , 112246. <a href="https://doi.org/10.1016/j.yexcr.2020.112246" title="DOI URL">https://doi.org/10.1016/j.yexcr.2020.112246</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.yexcr.2020.112246&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.yexcr.2020.112246%26sid%3Dliteratum%253Aachs%26jtitle%3DExperimental%2520Cell%2520Research%26atitle%3DHsf1%252Bon%252Ba%252Bleash%252B%2525E2%252580%252593%252Bcontrolling%252Bthe%252Bheat%252Bshock%252Bresponse%252Bby%252Bchaperone%252Btitration%26aulast%3DMasser%26aufirst%3DAnna%2BE.%26date%3D2020%26volume%3D396%26issue%3D1%26spage%3D112246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rebecca M.  Sebastian</span>, <span class="hlFld-ContribAuthor ">Matthew D.  Shoulders</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical Biology Framework to Illuminate Proteostasis. </span><span class="cited-content_cbyCitation_journal-name">Annual Review of Biochemistry</span><span> <strong>2020,</strong> <em>89 </em>
                                    (1)
                                     , 529-555. <a href="https://doi.org/10.1146/annurev-biochem-013118-111552" title="DOI URL">https://doi.org/10.1146/annurev-biochem-013118-111552</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1146/annurev-biochem-013118-111552&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1146%2Fannurev-biochem-013118-111552%26sid%3Dliteratum%253Aachs%26jtitle%3DAnnual%2520Review%2520of%2520Biochemistry%26atitle%3DChemical%252BBiology%252BFramework%252Bto%252BIlluminate%252BProteostasis%26aulast%3DSebastian%26aufirst%3DRebecca%2BM.%26date%3D2020%26volume%3D89%26issue%3D1%26spage%3D529%26epage%3D555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yonghua  Liu</span>, <span class="hlFld-ContribAuthor ">Yanbin  Wu</span>, <span class="hlFld-ContribAuthor ">lianqi  Sun</span>, <span class="hlFld-ContribAuthor ">Yuxi  Gu</span>, <span class="hlFld-ContribAuthor ">Laixing  Hu</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and structure-activity relationship study of water-soluble carbazole sulfonamide derivatives as new anticancer agents. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>191 </em>, 112181. <a href="https://doi.org/10.1016/j.ejmech.2020.112181" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112181</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112181%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Band%252Bstructure-activity%252Brelationship%252Bstudy%252Bof%252Bwater-soluble%252Bcarbazole%252Bsulfonamide%252Bderivatives%252Bas%252Bnew%252Banticancer%252Bagents%26aulast%3DLiu%26aufirst%3DYonghua%26date%3D2020%26volume%3D191%26spage%3D112181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel  Zhang</span>, <span class="hlFld-ContribAuthor ">Dorothy  Wang</span>, <span class="hlFld-ContribAuthor ">Bin  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Small Molecule Inhibitors Targeting Heat Shock Response Pathways: Lessons from Clinical and Preclinical Studies in Cancer Therapeutics. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 79-99. <a href="https://doi.org/10.1007/7515_2020_2" title="DOI URL">https://doi.org/10.1007/7515_2020_2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7515_2020_2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7515_2020_2%26sid%3Dliteratum%253Aachs%26atitle%3DSmall%252BMolecule%252BInhibitors%252BTargeting%252BHeat%252BShock%252BResponse%252BPathways%25253A%252BLessons%252Bfrom%252BClinical%252Band%252BPreclinical%252BStudies%252Bin%252BCancer%252BTherapeutics%26aulast%3DZhang%26aufirst%3DDaniel%26date%3D2020%26date%3D2020%26spage%3D79%26epage%3D99%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DHeat%252BShock%252BProteins%252Bin%252BHuman%252BDiseases%26aulast%3DAsea%26aufirst%3DAlexzander%2BA.%2BA.%26date%3D2021%26volume%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rajesh  Chopra</span>, <span class="hlFld-ContribAuthor ">Florence I.  Raynaud</span>. </span><span class="cited-content_cbyCitation_article-title">Preclinical Studies to Enable First in Human Clinical Trials. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 45-69. <a href="https://doi.org/10.1007/978-3-030-47682-3_3" title="DOI URL">https://doi.org/10.1007/978-3-030-47682-3_3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-47682-3_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-47682-3_3%26sid%3Dliteratum%253Aachs%26atitle%3DPreclinical%252BStudies%252Bto%252BEnable%252BFirst%252Bin%252BHuman%252BClinical%252BTrials%26aulast%3DChopra%26aufirst%3DRajesh%26date%3D2020%26date%3D2020%26spage%3D45%26epage%3D69%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DPhase%252BI%252BOncology%252BDrug%252BDevelopment%26aulast%3DYap%26aufirst%3DTimothy%2BA.%26date%3D2020%26volume%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bushu  Dong</span>, <span class="hlFld-ContribAuthor ">Alex M.  Jaeger</span>, <span class="hlFld-ContribAuthor ">Dennis J.  Thiele</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibiting Heat Shock Factor 1 in Cancer: A Unique Therapeutic Opportunity. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2019,</strong> <em>40 </em>
                                    (12)
                                     , 986-1005. <a href="https://doi.org/10.1016/j.tips.2019.10.008" title="DOI URL">https://doi.org/10.1016/j.tips.2019.10.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2019.10.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2019.10.008%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DInhibiting%252BHeat%252BShock%252BFactor%252B1%252Bin%252BCancer%25253A%252BA%252BUnique%252BTherapeutic%252BOpportunity%26aulast%3DDong%26aufirst%3DBushu%26date%3D2019%26volume%3D40%26issue%3D12%26spage%3D986%26epage%3D1005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olof  Svensson</span>, <span class="hlFld-ContribAuthor ">Maciej  Gilski</span>, <span class="hlFld-ContribAuthor ">Didier  Nurizzo</span>, <span class="hlFld-ContribAuthor ">Matthew W.  Bowler</span>. </span><span class="cited-content_cbyCitation_article-title">A comparative anatomy of protein crystals: lessons from the automatic processing of 56 000 samples. </span><span class="cited-content_cbyCitation_journal-name">IUCrJ</span><span> <strong>2019,</strong> <em>6 </em>
                                    (5)
                                     , 822-831. <a href="https://doi.org/10.1107/S2052252519008017" title="DOI URL">https://doi.org/10.1107/S2052252519008017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1107/S2052252519008017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1107%2FS2052252519008017%26sid%3Dliteratum%253Aachs%26jtitle%3DIUCrJ%26atitle%3DA%252Bcomparative%252Banatomy%252Bof%252Bprotein%252Bcrystals%25253A%252Blessons%252Bfrom%252Bthe%252Bautomatic%252Bprocessing%252Bof%252B56%2525E2%252580%252585000%252Bsamples%26aulast%3DSvensson%26aufirst%3DOlof%26date%3D2019%26date%3D2019%26volume%3D6%26issue%3D5%26spage%3D822%26epage%3D831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Richard L.  Carpenter</span>, <span class="hlFld-ContribAuthor ">Yesim  Gökmen-Polar</span>. </span><span class="cited-content_cbyCitation_article-title">HSF1 as a Cancer Biomarker and Therapeutic Target. </span><span class="cited-content_cbyCitation_journal-name">Current Cancer Drug Targets</span><span> <strong>2019,</strong> <em>19 </em>
                                    (7)
                                     , 515-524. <a href="https://doi.org/10.2174/1568009618666181018162117" title="DOI URL">https://doi.org/10.2174/1568009618666181018162117</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1568009618666181018162117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1568009618666181018162117%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Cancer%2520Drug%2520Targets%26atitle%3DHSF1%252Bas%252Ba%252BCancer%252BBiomarker%252Band%252BTherapeutic%252BTarget%26aulast%3DCarpenter%26aufirst%3DRichard%2BL.%26date%3D2019%26volume%3D19%26issue%3D7%26spage%3D515%26epage%3D524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edgar  Boczek</span>, <span class="hlFld-ContribAuthor ">Giorgio  Gaglia</span>, <span class="hlFld-ContribAuthor ">Maya  Olshina</span>, <span class="hlFld-ContribAuthor ">Shireen  Sarraf</span>. </span><span class="cited-content_cbyCitation_article-title">The first Autumn School on Proteostasis: from molecular mechanisms to organismal consequences. </span><span class="cited-content_cbyCitation_journal-name">Cell Stress and Chaperones</span><span> <strong>2019,</strong> <em>24 </em>
                                    (3)
                                     , 481-492. <a href="https://doi.org/10.1007/s12192-019-00998-w" title="DOI URL">https://doi.org/10.1007/s12192-019-00998-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s12192-019-00998-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs12192-019-00998-w%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Stress%2520and%2520Chaperones%26atitle%3DThe%252Bfirst%252BAutumn%252BSchool%252Bon%252BProteostasis%25253A%252Bfrom%252Bmolecular%252Bmechanisms%252Bto%252Borganismal%252Bconsequences%26aulast%3DBoczek%26aufirst%3DEdgar%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D3%26spage%3D481%26epage%3D492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Takayuki  Arai</span>, <span class="hlFld-ContribAuthor ">Satoko  Kojima</span>, <span class="hlFld-ContribAuthor ">Yasutaka  Yamada</span>, <span class="hlFld-ContribAuthor ">Sho  Sugawara</span>, <span class="hlFld-ContribAuthor ">Mayuko  Kato</span>, <span class="hlFld-ContribAuthor ">Kazuto  Yamazaki</span>, <span class="hlFld-ContribAuthor ">Yukio  Naya</span>, <span class="hlFld-ContribAuthor ">Tomohiko  Ichikawa</span>, <span class="hlFld-ContribAuthor ">Naohiko  Seki</span>. </span><span class="cited-content_cbyCitation_article-title">Pirin: a potential novel therapeutic target for castration‐resistant prostate cancer regulated by
              miR‐455‐5p. </span><span class="cited-content_cbyCitation_journal-name">Molecular Oncology</span><span> <strong>2019,</strong> <em>13 </em>
                                    (2)
                                     , 322-337. <a href="https://doi.org/10.1002/1878-0261.12405" title="DOI URL">https://doi.org/10.1002/1878-0261.12405</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/1878-0261.12405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F1878-0261.12405%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Oncology%26atitle%3DPirin%25253A%252Ba%252Bpotential%252Bnovel%252Btherapeutic%252Btarget%252Bfor%252Bcastration%2525E2%252580%252590resistant%252Bprostate%252Bcancer%252Bregulated%252Bby%252BmiR%2525E2%252580%252590455%2525E2%252580%2525905p%26aulast%3DArai%26aufirst%3DTakayuki%26date%3D2019%26date%3D2018%26volume%3D13%26issue%3D2%26spage%3D322%26epage%3D337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Min  Yan</span>, <span class="hlFld-ContribAuthor ">Dongheng  Zhou</span>, <span class="hlFld-ContribAuthor ">Yejun  Gao</span>, <span class="hlFld-ContribAuthor ">Yongmin  Ma</span>. </span><span class="cited-content_cbyCitation_article-title">Metal‐Free Synthesis of Methylene‐Bridged Bisamide via Selectfluor‐Mediated Oxidative Methylenation. </span><span class="cited-content_cbyCitation_journal-name">ChemistrySelect</span><span> <strong>2018,</strong> <em>3 </em>
                                    (46)
                                     , 13006-13009. <a href="https://doi.org/10.1002/slct.201802450" title="DOI URL">https://doi.org/10.1002/slct.201802450</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/slct.201802450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fslct.201802450%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistrySelect%26atitle%3DMetal%2525E2%252580%252590Free%252BSynthesis%252Bof%252BMethylene%2525E2%252580%252590Bridged%252BBisamide%252Bvia%252BSelectfluor%2525E2%252580%252590Mediated%252BOxidative%252BMethylenation%26aulast%3DYan%26aufirst%3DMin%26date%3D2018%26date%3D2018%26volume%3D3%26issue%3D46%26spage%3D13006%26epage%3D13009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chiranjeev  Sharma</span>, <span class="hlFld-ContribAuthor ">Young  Seo</span>. </span><span class="cited-content_cbyCitation_article-title">Small Molecule Inhibitors of HSF1-Activated Pathways as Potential Next-Generation Anticancer Therapeutics. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (11)
                                     , 2757. <a href="https://doi.org/10.3390/molecules23112757" title="DOI URL">https://doi.org/10.3390/molecules23112757</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23112757&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23112757%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DSmall%252BMolecule%252BInhibitors%252Bof%252BHSF1-Activated%252BPathways%252Bas%252BPotential%252BNext-Generation%252BAnticancer%252BTherapeutics%26aulast%3DSharma%26aufirst%3DChiranjeev%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D11%26spage%3D2757" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jacqueline H.L.  Fok</span>, <span class="hlFld-ContribAuthor ">Somaieh  Hedayat</span>, <span class="hlFld-ContribAuthor ">Lei  Zhang</span>, <span class="hlFld-ContribAuthor ">Lauren I.  Aronson</span>, <span class="hlFld-ContribAuthor ">Fabio  Mirabella</span>, <span class="hlFld-ContribAuthor ">Charlotte  Pawlyn</span>, <span class="hlFld-ContribAuthor ">Michael D.  Bright</span>, <span class="hlFld-ContribAuthor ">Christopher P.  Wardell</span>, <span class="hlFld-ContribAuthor ">Jonathan J.  Keats</span>, <span class="hlFld-ContribAuthor ">Emmanuel  De Billy</span>, <span class="hlFld-ContribAuthor ">Carl S.  Rye</span>, <span class="hlFld-ContribAuthor ">Nicola E.A.  Chessum</span>, <span class="hlFld-ContribAuthor ">Keith  Jones</span>, <span class="hlFld-ContribAuthor ">Gareth J.  Morgan</span>, <span class="hlFld-ContribAuthor ">Suzanne A.  Eccles</span>, <span class="hlFld-ContribAuthor ">Paul  Workman</span>, <span class="hlFld-ContribAuthor ">Faith E.  Davies</span>. </span><span class="cited-content_cbyCitation_article-title">HSF1 Is Essential for Myeloma Cell Survival and A Promising Therapeutic Target. </span><span class="cited-content_cbyCitation_journal-name">Clinical Cancer Research</span><span> <strong>2018,</strong> <em>24 </em>
                                    (10)
                                     , 2395-2407. <a href="https://doi.org/10.1158/1078-0432.CCR-17-1594" title="DOI URL">https://doi.org/10.1158/1078-0432.CCR-17-1594</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/1078-0432.CCR-17-1594&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F1078-0432.CCR-17-1594%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Cancer%2520Research%26atitle%3DHSF1%252BIs%252BEssential%252Bfor%252BMyeloma%252BCell%252BSurvival%252Band%252BA%252BPromising%252BTherapeutic%252BTarget%26aulast%3DFok%26aufirst%3DJacqueline%2BH.L.%26date%3D2018%26date%3D2018%26volume%3D24%26issue%3D10%26spage%3D2395%26epage%3D2407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olof  Svensson</span>, <span class="hlFld-ContribAuthor ">Maciej  Gilski</span>, <span class="hlFld-ContribAuthor ">Didier  Nurizzo</span>, <span class="hlFld-ContribAuthor ">Matthew W.  Bowler</span>. </span><span class="cited-content_cbyCitation_article-title">Multi-position data collection and dynamic beam sizing: recent improvements to the automatic data-collection algorithms on MASSIF-1. </span><span class="cited-content_cbyCitation_journal-name">Acta Crystallographica Section D Structural Biology</span><span> <strong>2018,</strong> <em>74 </em>
                                    (5)
                                     , 433-440. <a href="https://doi.org/10.1107/S2059798318003728" title="DOI URL">https://doi.org/10.1107/S2059798318003728</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1107/S2059798318003728&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1107%2FS2059798318003728%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Crystallographica%2520Section%2520D%2520Structural%2520Biology%26atitle%3DMulti-position%252Bdata%252Bcollection%252Band%252Bdynamic%252Bbeam%252Bsizing%25253A%252Brecent%252Bimprovements%252Bto%252Bthe%252Bautomatic%252Bdata-collection%252Balgorithms%252Bon%252BMASSIF-1%26aulast%3DSvensson%26aufirst%3DOlof%26date%3D2018%26date%3D2018%26volume%3D74%26issue%3D5%26spage%3D433%26epage%3D440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel  Zhang</span>, <span class="hlFld-ContribAuthor ">Bin  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">High Content Screening of Small Molecule Modulators Targeting Heat Shock Response Pathway. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 141-165. <a href="https://doi.org/10.1007/978-3-319-90725-3_8" title="DOI URL">https://doi.org/10.1007/978-3-319-90725-3_8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-319-90725-3_8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-319-90725-3_8%26sid%3Dliteratum%253Aachs%26atitle%3DHigh%252BContent%252BScreening%252Bof%252BSmall%252BMolecule%252BModulators%252BTargeting%252BHeat%252BShock%252BResponse%252BPathway%26aulast%3DZhang%26aufirst%3DDaniel%26date%3D2018%26date%3D2018%26spage%3D141%26epage%3D165%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DHeat%252BShock%252BProteins%252Band%252BStress%26aulast%3DAsea%26aufirst%3DAlexzander%2BA.%2BA.%26date%3D2018%26volume%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Inhibitors of HSF1-mediated HSP72 induction.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. HSF1 pathway inhibitor, bisamide <b>1</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0008.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of the Benzodioxane Bisamide Replacements<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) oxalyl chloride, DMF, DCM, then, pyridine, DCM; (ii) Fe, NH<sub>4</sub>Cl, EtOH/H<sub>2</sub>O; (iii) RPhCO<sub>2</sub>H, HATU, DIPEA, DMF.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0009.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Solubilizing Group Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) oxalyl chloride, DMF, 1,4-benzodioxane-6-carboxylic acid, DCM, then, 4-methyl-3-nitroaniline, pyridine, DCM, RT, then Pd/C (10%), H<sub>2</sub> (1 atm), EtOH; (ii) NaH, RCH<sub>2</sub>OH, THF, 0 °C to reflux; (iii) <i>n</i>-BuLi, CO<sub>2</sub>(s), THF, −78 °C; (iv) HATU, DIPEA, DMF, RT.</p></p></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Efficacy of bisamide <b>26</b> against SK-OV-3 human ovarian carcinoma xenograft model. Blue, vehicle control, <i>n</i> = 8; red, <b>26</b>, 20 mg/kg po qd, <i>n</i> = 8 (vehicle = 10% DMSO, 90% of a 25% (2-hydroxypropyl)-β-cyclodextrin in 50 mM citrate buffer pH 5). Error bars: arithmetic mean ± SEM. Dosing breaks were carried out on days 5–12, 14, 16, 18, 20, 22, 24, 26, 29, 31.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Tool compounds for target identification. All quoted cellular activities are in the SK-OV-3 cell line, the numbers of repeats are in parentheses.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Representative SPR sensorgram and binding isotherm of bisamide <b>26</b> bound to recombinant pirin. The dotted-line represents the time-point the equilibrium response was measured. The binding isotherm was fitted to a one-site specific binding model using Graphpad Prism 6.07.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0006.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. X-ray crystal structure of bisamide <b>26</b>. PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JCT">5JCT</a>, Pymol image of pirin (light-blue cartoon representation) in complex with bisamide <b>26</b> (cyan stick representation), 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map contoured at 1.0 σ (blue mesh) and Pymol image of pirin (magenta, blue stick representation) in complex with bisamide <b>26</b> (cyan stick representation), distances shown are in Å, 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> contoured at 1.0 σ (blue mesh). Crystals belonged to the space group <i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub> and diffracted to a resolution of 1.73 Å.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/medium/jm-2016-01055c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0007.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Antimigratory activity of chemical probe bisamide <b>26</b>. (A) Wound healing images of WM266.4 cells; after 30 h, the wound has almost completely healed in the control in contrast to 100 nM <b>26</b>. (B) Quantification of the relative wound density reveals the maximum antimigratory activity is achieved at 100 nM <b>26</b>. Each point represents the arithmetic mean ±SEM of the study carried out in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01055/20170306/images/large/jm-2016-01055c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01055&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i64">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53344" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53344" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 75 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Holohan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Schaeybroeck, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Longley, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, P. G.</span><span> </span><span class="NLM_article-title">Cancer drug resistance: an evolving paradigm</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">714</span><span class="NLM_x">–</span> <span class="NLM_lpage">726</span><span class="refDoi"> DOI: 10.1038/nrc3599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnrc3599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=24060863" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVyqurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=714-726&author=C.+Holohanauthor=S.+Van+Schaeybroeckauthor=D.+B.+Longleyauthor=P.+G.+Johnston&title=Cancer+drug+resistance%3A+an+evolving+paradigm&doi=10.1038%2Fnrc3599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1aR"><div class="casContent"><span class="casTitleNuber">1a</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer drug resistance: an evolving paradigm</span></div><div class="casAuthors">Holohan, Caitriona; Van Schaeybroeck, Sandra; Longley, Daniel B.; Johnston, Patrick G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">714-726</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Resistance to chemotherapy and molecularly targeted therapies is a major problem facing current cancer research.  The mechanisms of resistance to 'classical' cytotoxic chemotherapeutics and to therapies that are designed to be selective for specific mol. targets share many features, such as alterations in the drug target, activation of prosurvival pathways and ineffective induction of cell death.  With the increasing arsenal of anticancer agents, improving preclin. models and the advent of powerful high-throughput screening techniques, there are now unprecedented opportunities to understand and overcome drug resistance through the clin. assessment of rational therapeutic drug combinations and the use of predictive biomarkers to enable patient stratification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrykBEUSgvv7Vg90H21EOLACvtfcHk0ljr47UhGojY9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVyqurnE&md5=b497a823bbd968904ce1591b918d0679</span></div><a href="/servlet/linkout?suffix=cit1a&amp;dbid=16384&amp;doi=10.1038%2Fnrc3599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3599%26sid%3Dliteratum%253Aachs%26aulast%3DHolohan%26aufirst%3DC.%26aulast%3DVan%2BSchaeybroeck%26aufirst%3DS.%26aulast%3DLongley%26aufirst%3DD.%2BB.%26aulast%3DJohnston%26aufirst%3DP.%2BG.%26atitle%3DCancer%2520drug%2520resistance%253A%2520an%2520evolving%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2013%26volume%3D13%26spage%3D714%26epage%3D726%26doi%3D10.1038%2Fnrc3599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit1b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Zahreddine, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borden, K. L. B.</span><span> </span><span class="NLM_article-title">Mechanisms and insights into drug resistance in cancer</span> <span class="citation_source-journal">Front. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="refDoi"> DOI: 10.3389/fphar.2013.00028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.3389%2Ffphar.2013.00028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=23504227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A280%3ADC%252BC3svnt1ymtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=28&author=H.+Zahreddineauthor=K.+L.+B.+Borden&title=Mechanisms+and+insights+into+drug+resistance+in+cancer&doi=10.3389%2Ffphar.2013.00028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1bR"><div class="casContent"><span class="casTitleNuber">1b</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms and insights into drug resistance in cancer</span></div><div class="casAuthors">Zahreddine Hiba; Borden Katherine L B</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">28</span>
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    </div><div class="casAbstract">Cancer drug resistance continues to be a major impediment in medical oncology.  Clinically, resistance can arise prior to or as a result of cancer therapy.  In this review, we discuss different mechanisms adapted by cancerous cells to resist treatment, including alteration in drug transport and metabolism, mutation and amplification of drug targets, as well as genetic rewiring which can lead to impaired apoptosis.  Tumor heterogeneity may also contribute to resistance, where small subpopulations of cells may acquire or stochastically already possess some of the features enabling them to emerge under selective drug pressure.  Making the problem even more challenging, some of these resistance pathways lead to multidrug resistance, generating an even more difficult clinical problem to overcome.  We provide examples of these mechanisms and some insights into how understanding these processes can influence the next generation of cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQFZ1-BkHcDE36HPRus0rAafW6udTcc2ead65ULIgvHw7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3svnt1ymtw%253D%253D&md5=e3b49dd34e6cc024e098fd053447a10d</span></div><a href="/servlet/linkout?suffix=cit1b&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2013.00028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2013.00028%26sid%3Dliteratum%253Aachs%26aulast%3DZahreddine%26aufirst%3DH.%26aulast%3DBorden%26aufirst%3DK.%2BL.%2BB.%26atitle%3DMechanisms%2520and%2520insights%2520into%2520drug%2520resistance%2520in%2520cancer%26jtitle%3DFront.%2520Pharmacol.%26date%3D2013%26volume%3D4%26spage%3D28%26doi%3D10.3389%2Ffphar.2013.00028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit1c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Garraway, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janne, P. A.</span><span> </span><span class="NLM_article-title">Circumventing cancer drug resistance in the era of personalized medicine</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">214</span><span class="NLM_x">–</span> <span class="NLM_lpage">226</span><span class="refDoi"> DOI: 10.1158/2159-8290.CD-12-0012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1158%2F2159-8290.CD-12-0012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=214-226&author=L.+A.+Garrawayauthor=P.+A.+Janne&title=Circumventing+cancer+drug+resistance+in+the+era+of+personalized+medicine&doi=10.1158%2F2159-8290.CD-12-0012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1c&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-12-0012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-12-0012%26sid%3Dliteratum%253Aachs%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26atitle%3DCircumventing%2520cancer%2520drug%2520resistance%2520in%2520the%2520era%2520of%2520personalized%2520medicine%26jtitle%3DCancer%2520Discovery%26date%3D2012%26volume%3D2%26spage%3D214%26epage%3D226%26doi%3D10.1158%2F2159-8290.CD-12-0012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit1d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banerji, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span> </span><span class="NLM_article-title">Combinatorial drug therapy for cancer in the post-genomic era</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">679</span><span class="NLM_x">–</span> <span class="NLM_lpage">692</span><span class="refDoi"> DOI: 10.1038/nbt.2284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnbt.2284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=22781697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvFGmtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=679-692&author=B.+Al-Lazikaniauthor=U.+Banerjiauthor=P.+Workman&title=Combinatorial+drug+therapy+for+cancer+in+the+post-genomic+era&doi=10.1038%2Fnbt.2284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1dR"><div class="casContent"><span class="casTitleNuber">1d</span><div class="casTitle"><span class="NLM_cas:atitle">Combinatorial drug therapy for cancer in the post-genomic era</span></div><div class="casAuthors">Al-Lazikani, Bissan; Banerji, Udai; Workman, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">679-692</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Over the past decade, whole genome sequencing and other 'omics' technologies have defined pathogenic driver mutations to which tumor cells are addicted.  Such addictions, synthetic lethalities and other tumor vulnerabilities have yielded novel targets for a new generation of cancer drugs to treat discrete, genetically defined patient subgroups.  This personalized cancer medicine strategy could eventually replace the conventional one-size-fits-all cytotoxic chemotherapy approach.  However, the extraordinary intratumor genetic heterogeneity in cancers revealed by deep sequencing explains why de novo and acquired resistance arise with molecularly targeted drugs and cytotoxic chemotherapy, limiting their utility.  One soln. to the enduring challenge of polygenic cancer drug resistance is rational combinatorial targeted therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon5IOOPfsgR7Vg90H21EOLACvtfcHk0lhHrw2Bucqm2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvFGmtLg%253D&md5=a61b1b222a7146a92366f57cb0e708ca</span></div><a href="/servlet/linkout?suffix=cit1d&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2284%26sid%3Dliteratum%253Aachs%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DBanerji%26aufirst%3DU.%26aulast%3DWorkman%26aufirst%3DP.%26atitle%3DCombinatorial%2520drug%2520therapy%2520for%2520cancer%2520in%2520the%2520post-genomic%2520era%26jtitle%3DNat.%2520Biotechnol.%26date%3D2012%26volume%3D30%26spage%3D679%26epage%3D692%26doi%3D10.1038%2Fnbt.2284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Kinch, M. S.</span><span> </span><span class="NLM_article-title">An analysis of FDA-approved drugs for oncology</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1831</span><span class="NLM_x">–</span> <span class="NLM_lpage">1835</span><span class="refDoi"> DOI: 10.1016/j.drudis.2014.08.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.drudis.2014.08.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=25172803" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A280%3ADC%252BC2M%252FmsFGmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=1831-1835&author=M.+S.+Kinch&title=An+analysis+of+FDA-approved+drugs+for+oncology&doi=10.1016%2Fj.drudis.2014.08.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2aR"><div class="casContent"><span class="casTitleNuber">2a</span><div class="casTitle"><span class="NLM_cas:atitle">An analysis of FDA-approved drugs for oncology</span></div><div class="casAuthors">Kinch Michael S</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1831-5</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cancer remains the second leading cause of death globally.  The number of new medicines targeting cancer has grown impressively since the 1990s.  On average, ten new drugs are introduced each year.  Such growth has partly been achieved by emphasizing biologics and orphan indications, which account for one-quarter and one-half of new oncology drugs, respectively.  The biotechnology industry likewise has become the primary driver of cancer drug development in terms of patents, preclinical and clinical research, although pharmaceutical companies are granted more FDA approvals.  Many targeting strategies have been successful but recent trends suggest that kinase targets, although tractable, might be overemphasized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR3e6HLwRjzXH6oPH0mYWL1fW6udTcc2eavgMHHjcOlx7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M%252FmsFGmsw%253D%253D&md5=360c08e49bf159819dc5758b352e5016</span></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.08.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.08.007%26sid%3Dliteratum%253Aachs%26aulast%3DKinch%26aufirst%3DM.%2BS.%26atitle%3DAn%2520analysis%2520of%2520FDA-approved%2520drugs%2520for%2520oncology%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D1831%26epage%3D1835%26doi%3D10.1016%2Fj.drudis.2014.08.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit2b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Patel, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halling-Brown, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tym, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span> </span><span class="NLM_article-title">Objective assessment of cancer genes for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">35</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1038/nrd3913</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnrd3913" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=35-50&author=M.+N.+Patelauthor=M.+D.+Halling-Brownauthor=J.+E.+Tymauthor=P.+Workmanauthor=B.+Al-Lazikani&title=Objective+assessment+of+cancer+genes+for+drug+discovery&doi=10.1038%2Fnrd3913"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=10.1038%2Fnrd3913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3913%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DM.%2BN.%26aulast%3DHalling-Brown%26aufirst%3DM.%2BD.%26aulast%3DTym%26aufirst%3DJ.%2BE.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26atitle%3DObjective%2520assessment%2520of%2520cancer%2520genes%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D35%26epage%3D50%26doi%3D10.1038%2Fnrd3913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Kotz, J.</span><span> </span><span class="NLM_article-title">Phenotypic screening, take two</span> <span class="citation_source-journal">SciBX</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="refDoi"> DOI: 10.1038/scibx.2012.380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fscibx.2012.380" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2012&pages=15&author=J.+Kotz&title=Phenotypic+screening%2C+take+two&doi=10.1038%2Fscibx.2012.380"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3a&amp;dbid=16384&amp;doi=10.1038%2Fscibx.2012.380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fscibx.2012.380%26sid%3Dliteratum%253Aachs%26aulast%3DKotz%26aufirst%3DJ.%26atitle%3DPhenotypic%2520screening%252C%2520take%2520two%26jtitle%3DSciBX%26date%3D2012%26volume%3D5%26spage%3D15%26doi%3D10.1038%2Fscibx.2012.380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit3b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Tang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duggan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warder, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLoughlin, S. M.</span><span> </span><span class="NLM_article-title">Target identification of compounds from a cell viability phenotypic screen using a bead/lysate-based affinity capture platform</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">201</span><span class="NLM_x">–</span> <span class="NLM_lpage">211</span><span class="refDoi"> DOI: 10.1177/1087057115622431</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1177%2F1087057115622431" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=26676096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVSrtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=201-211&author=H.+Tangauthor=S.+Dugganauthor=P.+L.+Richardsonauthor=V.+Marinauthor=S.+E.+Warderauthor=S.+M.+McLoughlin&title=Target+identification+of+compounds+from+a+cell+viability+phenotypic+screen+using+a+bead%2Flysate-based+affinity+capture+platform&doi=10.1177%2F1087057115622431"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3bR"><div class="casContent"><span class="casTitleNuber">3b</span><div class="casTitle"><span class="NLM_cas:atitle">Target identification of compounds from a cell viability phenotypic screen using a bead/lysate-based affinity capture platform</span></div><div class="casAuthors">Tang, Hua; Duggan, Shannon; Richardson, Paul L.; Marin, Violeta; Warder, Scott E.; McLoughlin, Shaun M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">201-211</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The pharmaceutical industry has been continually challenged by dwindling target diversity.  To obviate this trend, phenotypic screens have been adopted, complementing target-centric screening approaches.  Phenotypic screens identify drug leads using clin. relevant and translatable mechanisms, remaining agnostic to targets.  While target anonymity is advantageous early in the drug discovery process, it poses challenges to hit progression, including the development of backup series, retaining desired pharmacol. during optimization, discovery of markers, and understanding mechanism-driven toxicity.  Consequently, significant effort has been expended to elaborate the targets and mechanisms at work for promising screening hits.  Affinity capture is commonly leveraged, where the compds. are linked to beads and targets are abstracted from cell homogenates.  This technique has proven effective for identifying targets of kinase, PARP, and HDAC inhibitors, and examples of new targets have been reported.  Herein, a three-pronged approach to target deconvolution by affinity capture is described, including the implementation of a uniqueness index that helps discriminate between bona fide targets and background.  The effectiveness of this approach is demonstrated using characterized compds. that act on known and noncanonical target classes.  The platform is subsequently applied to phenotypic screening hits, identifying candidate targets.  The success rate of bead-based affinity capture is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodk9IeGmf4ErVg90H21EOLACvtfcHk0lgRH1zXuhy3fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVSrtbk%253D&md5=5951e68edfdfeda7e82d428e06ca018c</span></div><a href="/servlet/linkout?suffix=cit3b&amp;dbid=16384&amp;doi=10.1177%2F1087057115622431&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057115622431%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DH.%26aulast%3DDuggan%26aufirst%3DS.%26aulast%3DRichardson%26aufirst%3DP.%2BL.%26aulast%3DMarin%26aufirst%3DV.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DMcLoughlin%26aufirst%3DS.%2BM.%26atitle%3DTarget%2520identification%2520of%2520compounds%2520from%2520a%2520cell%2520viability%2520phenotypic%2520screen%2520using%2520a%2520bead%252Flysate-based%2520affinity%2520capture%2520platform%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2016%26volume%3D21%26spage%3D201%26epage%3D211%26doi%3D10.1177%2F1087057115622431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Moffat, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailey, D.</span><span> </span><span class="NLM_article-title">Phenotypic screening in cancer drug discovery - past, present and future</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">588</span><span class="NLM_x">–</span> <span class="NLM_lpage">602</span><span class="refDoi"> DOI: 10.1038/nrd4366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnrd4366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=25033736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=588-602&author=J.+G.+Moffatauthor=J.+Rudolphauthor=D.+Bailey&title=Phenotypic+screening+in+cancer+drug+discovery+-+past%2C+present+and+future&doi=10.1038%2Fnrd4366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4aR"><div class="casContent"><span class="casTitleNuber">4a</span><div class="casTitle"><span class="NLM_cas:atitle">Phenotypic screening in cancer drug discovery - past, present and future</span></div><div class="casAuthors">Moffat, John G.; Rudolph, Joachim; Bailey, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">588-602</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There has been a resurgence of interest in the use of phenotypic screens in drug discovery as an alternative to target-focused approaches.  Given that oncol. is currently the most active therapeutic area, and also one in which target-focused approaches have been particularly prominent in the past two decades, we investigated the contribution of phenotypic assays to oncol. drug discovery by analyzing the origins of all new small-mol. cancer drugs approved by the US Food and Drug Administration (FDA) over the past 15 years and those currently in clin. development.  Although the majority of these drugs originated from target-based discovery, we identified a significant no. whose discovery depended on phenotypic screening approaches.  We postulate that the contribution of phenotypic screening to cancer drug discovery has been hampered by a reliance on 'classical' nonspecific drug effects such as cytotoxicity and mitotic arrest, exacerbated by a paucity of mechanistically defined cellular models for therapeutically translatable cancer phenotypes.  However, tech. and biol. advances that enable such mechanistically informed phenotypic models have the potential to empower phenotypic drug discovery in oncol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTWtoTIgWqvbVg90H21EOLACvtfcHk0lgRH1zXuhy3fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7L&md5=0e8efed90e5bc4b47f959b83a6ab5b96</span></div><a href="/servlet/linkout?suffix=cit4a&amp;dbid=16384&amp;doi=10.1038%2Fnrd4366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4366%26sid%3Dliteratum%253Aachs%26aulast%3DMoffat%26aufirst%3DJ.%2BG.%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DBailey%26aufirst%3DD.%26atitle%3DPhenotypic%2520screening%2520in%2520cancer%2520drug%2520discovery%2520-%2520past%252C%2520present%2520and%2520future%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D588%26epage%3D602%26doi%3D10.1038%2Fnrd4366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit4b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Vincent, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loria, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pregel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanton, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitching, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nocka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doyonnas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steppan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schroeter, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peakman, M.</span><span> </span><span class="NLM_article-title">Developing predictive assays: the phenotypic screening “rule of 3”</span> <span class="citation_source-journal">Sci. Transl. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">293ps15</span><span class="refDoi"> DOI: 10.1126/scitranslmed.aab1201</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1126%2Fscitranslmed.aab1201" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=26109101" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=293ps15&author=F.+Vincentauthor=P.+Loriaauthor=M.+Pregelauthor=R.+Stantonauthor=L.+Kitchingauthor=K.+Nockaauthor=R.+Doyonnasauthor=C.+Steppanauthor=A.+Gilbertauthor=T.+Schroeterauthor=M.+Peakman&title=Developing+predictive+assays%3A+the+phenotypic+screening+%E2%80%9Crule+of+3%E2%80%9D&doi=10.1126%2Fscitranslmed.aab1201"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4b&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aab1201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aab1201%26sid%3Dliteratum%253Aachs%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DLoria%26aufirst%3DP.%26aulast%3DPregel%26aufirst%3DM.%26aulast%3DStanton%26aufirst%3DR.%26aulast%3DKitching%26aufirst%3DL.%26aulast%3DNocka%26aufirst%3DK.%26aulast%3DDoyonnas%26aufirst%3DR.%26aulast%3DSteppan%26aufirst%3DC.%26aulast%3DGilbert%26aufirst%3DA.%26aulast%3DSchroeter%26aufirst%3DT.%26aulast%3DPeakman%26aufirst%3DM.%26atitle%3DDeveloping%2520predictive%2520assays%253A%2520the%2520phenotypic%2520screening%2520%25E2%2580%259Crule%2520of%25203%25E2%2580%259D%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2015%26volume%3D7%26spage%3D293ps15%26doi%3D10.1126%2Fscitranslmed.aab1201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit4c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Swinney, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anthony, J.</span><span> </span><span class="NLM_article-title">How were new medicines discovered?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">507</span><span class="NLM_x">–</span> <span class="NLM_lpage">519</span><span class="refDoi"> DOI: 10.1038/nrd3480</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnrd3480" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=21701501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlCmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=507-519&author=D.+C.+Swinneyauthor=J.+Anthony&title=How+were+new+medicines+discovered%3F&doi=10.1038%2Fnrd3480"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4cR"><div class="casContent"><span class="casTitleNuber">4c</span><div class="casTitle"><span class="NLM_cas:atitle">How were new medicines discovered?</span></div><div class="casAuthors">Swinney, David C.; Anthony, Jason</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">507-519</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Preclin. strategies that are used to identify potential drug candidates include target-based screening, phenotypic screening, modification of natural substances and biol.-based approaches.  To investigate whether some strategies have been more successful than others in the discovery of new drugs, we analyzed the discovery strategies and the mol. mechanism of action (MMOA) for new mol. entities and new biologics that were approved by the US Food and Drug Administration between 1999 and 2008.  Out of the 259 agents that were approved, 75 were first-in-class drugs with new MMOAs, and out of these, 50 (67%) were small mols. and 25 (33%) were biologics.  The results also show that the contribution of phenotypic screening to the discovery of first-in-class small-mol. drugs exceeded that of target-based approaches - with 28 and 17 of these drugs coming from the two approaches, resp. - in an era in which the major focus was on target-based approaches.  We postulate that a target-centric approach for first-in-class drugs, without consideration of an optimal MMOA, may contribute to the current high attrition rates and low productivity in pharmaceutical research and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE8Ah1iNz76bVg90H21EOLACvtfcHk0lhTAzcHA9UkYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlCmsbs%253D&md5=7503620f8fbadb0a829f6ea2029936a2</span></div><a href="/servlet/linkout?suffix=cit4c&amp;dbid=16384&amp;doi=10.1038%2Fnrd3480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3480%26sid%3Dliteratum%253Aachs%26aulast%3DSwinney%26aufirst%3DD.%2BC.%26aulast%3DAnthony%26aufirst%3DJ.%26atitle%3DHow%2520were%2520new%2520medicines%2520discovered%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D507%26epage%3D519%26doi%3D10.1038%2Fnrd3480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit4d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Cheeseman, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faisal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rayter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbeau, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalusa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westlake, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Montfort, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linardopoulos, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, K.</span><span> </span><span class="NLM_article-title">Targeting the PPM1D phenotype; 2,4-bisarylthiazoles cause highly selective apoptosis in PPM1D amplified cell-lines</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">3469</span><span class="NLM_x">–</span> <span class="NLM_lpage">3474</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2014.05.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.bmcl.2014.05.067" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=3469-3474&author=M.+D.+Cheesemanauthor=A.+Faisalauthor=S.+Rayterauthor=O.+Barbeauauthor=A.+Kalusaauthor=M.+Westlakeauthor=R.+Burkeauthor=M.+Swanauthor=R.+van+Montfortauthor=S.+Linardopoulosauthor=K.+Jones&title=Targeting+the+PPM1D+phenotype%3B+2%2C4-bisarylthiazoles+cause+highly+selective+apoptosis+in+PPM1D+amplified+cell-lines&doi=10.1016%2Fj.bmcl.2014.05.067"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4d&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.05.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.05.067%26sid%3Dliteratum%253Aachs%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26aulast%3DFaisal%26aufirst%3DA.%26aulast%3DRayter%26aufirst%3DS.%26aulast%3DBarbeau%26aufirst%3DO.%26aulast%3DKalusa%26aufirst%3DA.%26aulast%3DWestlake%26aufirst%3DM.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DSwan%26aufirst%3DM.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%26aulast%3DLinardopoulos%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DK.%26atitle%3DTargeting%2520the%2520PPM1D%2520phenotype%253B%25202%252C4-bisarylthiazoles%2520cause%2520highly%2520selective%2520apoptosis%2520in%2520PPM1D%2520amplified%2520cell-lines%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D3469%26epage%3D3474%26doi%3D10.1016%2Fj.bmcl.2014.05.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Zheng, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorne, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKew, J. C.</span><span> </span><span class="NLM_article-title">Phenotypic screens as a renewed approach for drug discovery</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1067</span><span class="NLM_x">–</span> <span class="NLM_lpage">1073</span><span class="refDoi"> DOI: 10.1016/j.drudis.2013.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.drudis.2013.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=23850704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1emt7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=1067-1073&author=W.+Zhengauthor=N.+Thorneauthor=J.+C.+McKew&title=Phenotypic+screens+as+a+renewed+approach+for+drug+discovery&doi=10.1016%2Fj.drudis.2013.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5aR"><div class="casContent"><span class="casTitleNuber">5a</span><div class="casTitle"><span class="NLM_cas:atitle">Phenotypic screens as a renewed approach for drug discovery</span></div><div class="casAuthors">Zheng, Wei; Thorne, Natasha; McKew, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">21-22</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The significant redn. in the no. of newly approved drugs in the past decade has been partially attributed to failures in discovery and validation of new targets.  Evaluation of recently approved new drugs has revealed that the no. of approved drugs discovered through phenotypic screens, an original drug screening paradigm, has exceeded those discovered through the mol. target-based approach.  Phenotypic screening is thus gaining new momentum in drug discovery with the hope that this approach may revitalize drug discovery and improve the success rate of drug approval through the discovery of viable lead compds. and identification of novel drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_cPfNTmnrfbVg90H21EOLACvtfcHk0lhTAzcHA9UkYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1emt7vO&md5=9d92e1f61043ac2dd11bfe5f32944cff</span></div><a href="/servlet/linkout?suffix=cit5a&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DThorne%26aufirst%3DN.%26aulast%3DMcKew%26aufirst%3DJ.%2BC.%26atitle%3DPhenotypic%2520screens%2520as%2520a%2520renewed%2520approach%2520for%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2013%26volume%3D18%26spage%3D1067%26epage%3D1073%26doi%3D10.1016%2Fj.drudis.2013.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit5b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Lee, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uhlik, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moxham, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomandl, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sall, D. J.</span><span> </span><span class="NLM_article-title">Modern phenotypic drug discovery is a viable, neoclassic pharma strategy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">4527</span><span class="NLM_x">–</span> <span class="NLM_lpage">4538</span><span class="refDoi"> DOI: 10.1021/jm201649s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201649s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1OitLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=4527-4538&author=J.+A.+Leeauthor=M.+T.+Uhlikauthor=C.+M.+Moxhamauthor=D.+Tomandlauthor=D.+J.+Sall&title=Modern+phenotypic+drug+discovery+is+a+viable%2C+neoclassic+pharma+strategy&doi=10.1021%2Fjm201649s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5bR"><div class="casContent"><span class="casTitleNuber">5b</span><div class="casTitle"><span class="NLM_cas:atitle">Modern Phenotypic Drug Discovery Is a Viable, Neoclassic Pharma Strategy</span></div><div class="casAuthors">Lee, Jonathan A.; Uhlik, Mark T.; Moxham, Christopher M.; Tomandl, Dirk; Sall, Daniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4527-4538</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The pharmaceutical industry (Pharma) is currently facing unprecedented challenges.  In addn. to strategic patent expirations, the rate of drug launches has been essentially const. for 60 years with overall productivity falling since the 1970s.  Similarly, the appearance of novel targets to FDA approved drugs, a measure of industry innovation, has not dramatically improved since the 1980s.  Although Pharma productivity is a multifaceted problem, detailed analyses of comprehensive, industry-wide data indicate that late stage clin. failures are a major contributor that has been attributed to poor target validation (TV) and the lack of predictive biomarkers that translate to the clinic.  In the spirit of "reinventing innovation" Pharma needs to introspectively identify areas for improvement.  This communication considers how choices in drug screening strategies may relate to target validation issues and influence the probability of identifying novel medicines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8CcODiYYNgLVg90H21EOLACvtfcHk0lhTAzcHA9UkYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1OitLs%253D&md5=1937641b632e348d7566c016ca4cf51d</span></div><a href="/servlet/linkout?suffix=cit5b&amp;dbid=16384&amp;doi=10.1021%2Fjm201649s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201649s%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%2BA.%26aulast%3DUhlik%26aufirst%3DM.%2BT.%26aulast%3DMoxham%26aufirst%3DC.%2BM.%26aulast%3DTomandl%26aufirst%3DD.%26aulast%3DSall%26aufirst%3DD.%2BJ.%26atitle%3DModern%2520phenotypic%2520drug%2520discovery%2520is%2520a%2520viable%252C%2520neoclassic%2520pharma%2520strategy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D4527%26epage%3D4538%26doi%3D10.1021%2Fjm201649s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit5c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Eder, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sedrani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiesmann, C.</span><span> </span><span class="NLM_article-title">The discovery of first-in-class drugs: origins and evolution</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">577</span><span class="NLM_x">–</span> <span class="NLM_lpage">587</span><span class="refDoi"> DOI: 10.1038/nrd4336</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnrd4336" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=25033734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=577-587&author=J.+Ederauthor=R.+Sedraniauthor=C.+Wiesmann&title=The+discovery+of+first-in-class+drugs%3A+origins+and+evolution&doi=10.1038%2Fnrd4336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5cR"><div class="casContent"><span class="casTitleNuber">5c</span><div class="casTitle"><span class="NLM_cas:atitle">The discovery of first-in-class drugs: origins and evolution</span></div><div class="casAuthors">Eder, Jorg; Sedrani, Richard; Wiesmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">577-587</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Anal. of the origins of new drugs approved by the US Food and Drug Administration (FDA) from 1999 to 2008 suggested that phenotypic screening strategies had been more productive than target-based approaches in the discovery of first-in-class small-mol. drugs.  However, given the relatively recent introduction of target-based approaches in the context of the long time frames of drug development, their full impact might not yet have become apparent.  Here, we present an anal. of the origins of all 113 first-in-class drugs approved by the FDA from 1999 to 2013, which shows that the majority (78) were discovered through target-based approaches (45 small-mol. drugs and 33 biologics).  In addn., of 33 drugs identified in the absence of a target hypothesis, 25 were found through a chemocentric approach in which compds. with known pharmacol. served as the starting point, with only eight coming from what we define here as phenotypic screening: testing a large no. of compds. in a target-agnostic assay that monitors phenotypic changes.  We also discuss the implications for drug discovery strategies, including viewing phenotypic screening as a novel discipline rather than as a neoclassical approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqthEelveRWN7Vg90H21EOLACvtfcHk0lhM3g9pr4005Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygsr7I&md5=b2061d96a71893a329b5b654acec5420</span></div><a href="/servlet/linkout?suffix=cit5c&amp;dbid=16384&amp;doi=10.1038%2Fnrd4336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4336%26sid%3Dliteratum%253Aachs%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DSedrani%26aufirst%3DR.%26aulast%3DWiesmann%26aufirst%3DC.%26atitle%3DThe%2520discovery%2520of%2520first-in-class%2520drugs%253A%2520origins%2520and%2520evolution%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D577%26epage%3D587%26doi%3D10.1038%2Fnrd4336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Anighoro, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rastelli, G.</span><span> </span><span class="NLM_article-title">Polypharmacology: challenges and opportunities in drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">7874</span><span class="NLM_x">–</span> <span class="NLM_lpage">7887</span><span class="refDoi"> DOI: 10.1021/jm5006463</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5006463" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7874-7887&author=A.+Anighoroauthor=J.+Bajorathauthor=G.+Rastelli&title=Polypharmacology%3A+challenges+and+opportunities+in+drug+discovery&doi=10.1021%2Fjm5006463"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6aR"><div class="casContent"><span class="casTitleNuber">6a</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology: Challenges and Opportunities in Drug Discovery</span></div><div class="casAuthors">Anighoro, Andrew; Bajorath, Jurgen; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7874-7887</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  At present, the legendary magic bullet, i.e., a drug with high potency and selectivity toward a specific biol. target, shares the spotlight with an emerging and alternative polypharmacol. approach.  Polypharmacol. suggests that more effective drugs can be developed by specifically modulating multiple targets.  It is generally thought that complex diseases such as cancer and central nervous system diseases may require complex therapeutic approaches.  In this respect, a drug that "hits" multiple sensitive nodes belonging to a network of interacting targets offers the potential for higher efficacy and may limit drawbacks generally arising from the use of a single-target drug or a combination of multiple drugs.  In this review, we will compare advantages and disadvantages of multitarget vs. combination therapies, discuss potential drug promiscuity arising from off-target effects, comment on drug repurposing, and introduce approaches to the computational design of multitarget drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFOTyEu80AZrVg90H21EOLACvtfcHk0lhM3g9pr4005Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I&md5=5ab36c961a84462d90bc25130d90a281</span></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1021%2Fjm5006463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5006463%26sid%3Dliteratum%253Aachs%26aulast%3DAnighoro%26aufirst%3DA.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DPolypharmacology%253A%2520challenges%2520and%2520opportunities%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D7874%26epage%3D7887%26doi%3D10.1021%2Fjm5006463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit6b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Peters, J.</span><span> </span><span class="NLM_article-title">Polypharmacology – foe or friend?</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">8955</span><span class="NLM_x">–</span> <span class="NLM_lpage">8971</span><span class="refDoi"> DOI: 10.1021/jm400856t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400856t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1amtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8955-8971&author=J.+Peters&title=Polypharmacology+%E2%80%93+foe+or+friend%3F&doi=10.1021%2Fjm400856t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6bR"><div class="casContent"><span class="casTitleNuber">6b</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology - Foe or Friend?</span></div><div class="casAuthors">Peters, Jens-Uwe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8955-8971</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Polypharmacol. describes the activity of compds. at multiple targets.  Current research focuses on two aspects of polypharmacol.: (1) unintended polypharmacol. can lead to adverse effects; (2) polypharmacol. across several disease-relevant targets can improve therapeutic efficacy, prevent drug resistance, or reduce therapeutic-target-related adverse effects.  This perspective reviews these interconnected aspects of polypharmacol.  The first part discusses the relevance of polypharmacol. for the safety of drugs, the mitigation of safety risks, and methods to identify polypharmacol. compds. early in the drug discovery process.  The second part discusses the advantages of polypharmacol. in the treatment of multigenic diseases and infections, and opportunities for drug discovery and drug repurposing.  This perspective aims to provide a balanced view on polypharmacol., which can compromise the safety of drugs, but can also confer superior efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5vnFGPfTnGLVg90H21EOLACvtfcHk0lhM3g9pr4005Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1amtrnO&md5=f4aeb6efddd4bfdf4e94656303323cba</span></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1021%2Fjm400856t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400856t%26sid%3Dliteratum%253Aachs%26aulast%3DPeters%26aufirst%3DJ.%26atitle%3DPolypharmacology%2520%25E2%2580%2593%2520foe%2520or%2520friend%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8955%26epage%3D8971%26doi%3D10.1021%2Fjm400856t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit6c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Medina-Franco, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giulianotti, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Welmaker, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houghten, R. A.</span><span> </span><span class="NLM_article-title">Shifting from the single to the multitarget paradigm in drug discovery</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">495</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1016/j.drudis.2013.01.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.drudis.2013.01.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=23340113" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A280%3ADC%252BC3szhsV2rsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=495-501&author=J.+L.+Medina-Francoauthor=M.+A.+Giulianottiauthor=G.+S.+Welmakerauthor=R.+A.+Houghten&title=Shifting+from+the+single+to+the+multitarget+paradigm+in+drug+discovery&doi=10.1016%2Fj.drudis.2013.01.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6cR"><div class="casContent"><span class="casTitleNuber">6c</span><div class="casTitle"><span class="NLM_cas:atitle">Shifting from the single to the multitarget paradigm in drug discovery</span></div><div class="casAuthors">Medina-Franco Jose L; Giulianotti Marc A; Welmaker Gregory S; Houghten Richard A</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9-10</span>),
    <span class="NLM_cas:pages">495-501</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Increasing evidence that several drug compounds exert their effects through interactions with multiple targets is boosting the development of research fields that challenge the data reductionism approach.  In this article, we review and discuss the concepts of drug repurposing, polypharmacology, chemogenomics, phenotypic screening and high-throughput in vivo testing of mixture-based libraries in an integrated manner.  These research fields offer alternatives to the current paradigm of drug discovery, from a one target-one drug model to a multiple-target approach.  Furthermore, the goals of lead identification are being expanded accordingly to identify not only 'key' compounds that fit with a single-target 'lock', but also 'master key' compounds that favorably interact with multiple targets (i.e. operate a set of desired locks to gain access to the expected clinical effects).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTOFtVP-kWcL7qxQ2mb9Pu8fW6udTcc2eZeVMQPePsekbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3szhsV2rsQ%253D%253D&md5=b4f8c7d1ebbbf32ed4c6c87500baf66f</span></div><a href="/servlet/linkout?suffix=cit6c&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.01.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.01.008%26sid%3Dliteratum%253Aachs%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26aulast%3DGiulianotti%26aufirst%3DM.%2BA.%26aulast%3DWelmaker%26aufirst%3DG.%2BS.%26aulast%3DHoughten%26aufirst%3DR.%2BA.%26atitle%3DShifting%2520from%2520the%2520single%2520to%2520the%2520multitarget%2520paradigm%2520in%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2013%26volume%3D18%26spage%3D495%26epage%3D501%26doi%3D10.1016%2Fj.drudis.2013.01.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit6d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Jalencas, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mestres, J.</span><span> </span><span class="NLM_article-title">On the origins of drug polypharmacology</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">80</span><span class="NLM_x">–</span> <span class="NLM_lpage">87</span><span class="refDoi"> DOI: 10.1039/C2MD20242E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1039%2FC2MD20242E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVynu7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=80-87&author=X.+Jalencasauthor=J.+Mestres&title=On+the+origins+of+drug+polypharmacology&doi=10.1039%2FC2MD20242E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6dR"><div class="casContent"><span class="casTitleNuber">6d</span><div class="casTitle"><span class="NLM_cas:atitle">On the origins of drug polypharmacology</span></div><div class="casAuthors">Jalencas, Xavier; Mestres, Jordi</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">80-87</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The ability of small mols. to interact with multiple proteins is commonly referred to as polypharmacol.  The now widely accepted polypharmacol. of drugs is of particular interest for human health as it has implications beyond therapeutic efficacy, from anticipating adverse drug reactions to identifying potential repurposing opportunities.  There have been a no. of studies relating the extent of drug polypharmacol. to the physicochem. properties and fragment compn. of the drug itself, but also to the protein family and distant binding site similarities of the drug's primary target.  Taken together, all these observations lead to speculate that the origins of drug polypharmacol. may lie at the heart of protein evolution and that polypharmacol. may just be a reminiscent signature of some of the mechanisms of adaptation that primitive biol. systems developed to increase the chances of survival in a highly variable early chem. environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB1qsCiSthHLVg90H21EOLACvtfcHk0liVSUSaIQrMZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVynu7vI&md5=77fe84731536e5008cca3b0faf1e58dd</span></div><a href="/servlet/linkout?suffix=cit6d&amp;dbid=16384&amp;doi=10.1039%2FC2MD20242E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2MD20242E%26sid%3Dliteratum%253Aachs%26aulast%3DJalencas%26aufirst%3DX.%26aulast%3DMestres%26aufirst%3DJ.%26atitle%3DOn%2520the%2520origins%2520of%2520drug%2520polypharmacology%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D80%26epage%3D87%26doi%3D10.1039%2FC2MD20242E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Swinney, D. C.</span><span> </span><span class="NLM_article-title">The value of translational biomarkers to phenotypic assays</span> <span class="citation_source-journal">Front. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">171</span><span class="refDoi"> DOI: 10.3389/fphar.2014.00171</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.3389%2Ffphar.2014.00171" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=25076910" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A280%3ADC%252BC2cbns1aguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=171&author=D.+C.+Swinney&title=The+value+of+translational+biomarkers+to+phenotypic+assays&doi=10.3389%2Ffphar.2014.00171"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The value of translational biomarkers to phenotypic assays</span></div><div class="casAuthors">Swinney David C</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">171</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Phenotypic assays are tools essential for drug discovery.  Phenotypic assays have different types of endpoints depending on the goals; (1) empirical endpoints for basic research to understand the underlying biology that will lead to identification of translation biomarkers, (2) empirical endpoints to identify undesired effects related to toxicity of drug candidates, and (3) knowledge-based endpoints (biomarkers) for drug discovery which ideally are translational biomarkers that will be used to identify new drug candidates and their corresponding molecular mechanisms of action.  The value of phenotypic assays is increased through effective alignment of phenotypic assay endpoints with the objectives of the relevant stage in the drug discovery and development cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSWdoDxmkrm475XHngOXiV_fW6udTcc2eaZsLGKY13a0bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cbns1aguw%253D%253D&md5=5de6ca2ed245be9bfdb9f082c2d41adc</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2014.00171&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2014.00171%26sid%3Dliteratum%253Aachs%26aulast%3DSwinney%26aufirst%3DD.%2BC.%26atitle%3DThe%2520value%2520of%2520translational%2520biomarkers%2520to%2520phenotypic%2520assays%26jtitle%3DFront.%2520Pharmacol.%26date%3D2014%26volume%3D5%26spage%3D171%26doi%3D10.3389%2Ffphar.2014.00171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guan, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mais, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span> </span><span class="NLM_article-title">Quality control of cell-based high-throughput drug screening</span> <span class="citation_source-journal">Acta Pharm. Sin. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">429</span><span class="NLM_x">–</span> <span class="NLM_lpage">438</span><span class="refDoi"> DOI: 10.1016/j.apsb.2012.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.apsb.2012.03.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12nt7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=429-438&author=Z.+Zhangauthor=N.+Guanauthor=T.+Liauthor=D.+E.+Maisauthor=M.+Wang&title=Quality+control+of+cell-based+high-throughput+drug+screening&doi=10.1016%2Fj.apsb.2012.03.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Quality control of cell-based high-throughput drug screening</span></div><div class="casAuthors">Zhang, Zhiyun; Guan, Ni; Li, Ting; Mais, Dale E.; Wang, Mingwei</div><div class="citationInfo"><span class="NLM_cas:title">Acta Pharmaceutica Sinica B</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">429-438</span>CODEN:
                <span class="NLM_cas:coden">APSBCW</span>;
        ISSN:<span class="NLM_cas:issn">2211-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The pharmaceutical industry is presently suffering difficult times due to low productivity of new mol. entities.  As a major source of drug leads, high-throughput screening (HTS) has been often criticized for its 'dead end' lead compds.  However, the fruitful achievements resulting from HTS technol. indicate that it remains a feasible way for drug innovation.  Because of increasing considerations of earlier stage ADMET (absorption, distribution, metab., excretion and toxicity) in drug development, cell-based HTS is highly recommended in modern drug discovery for its ability to detect more biol. relevant characteristics of compds. in living systems.  This review provides a systematic and practical description of vital points for conducting high quality cell-based HTS, from assay development to optimization, compd. management, data analyses, hit validation as well as lead identification.  Potential problems and solns. are also covered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkiaPQ7EjY6rVg90H21EOLACvtfcHk0lgdvPicQpiibg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12nt7fP&md5=b4b20c6edb7047d27bd0f7e68b92e992</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2012.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2012.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DGuan%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DMais%26aufirst%3DD.%2BE.%26aulast%3DWang%26aufirst%3DM.%26atitle%3DQuality%2520control%2520of%2520cell-based%2520high-throughput%2520drug%2520screening%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2012%26volume%3D2%26spage%3D429%26epage%3D438%26doi%3D10.1016%2Fj.apsb.2012.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blagg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buser-Doepner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederiksen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimshaw, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hepworth, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howe, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotz, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mader, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller-Fahrnow, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Hagan, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenberg, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundström, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toledo-Sherman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walpole, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walters, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willson, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuercher, W. J.</span><span> </span><span class="NLM_article-title">The promise and peril of chemical probes</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">536</span><span class="NLM_x">–</span> <span class="NLM_lpage">541</span><span class="refDoi"> DOI: 10.1038/nchembio.1867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnchembio.1867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=26196764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=536-541&author=C.+H.+Arrowsmithauthor=J.+E.+Audiaauthor=C.+Austinauthor=J.+Baellauthor=J.+Bennettauthor=J.+Blaggauthor=C.+Bountraauthor=P.+E.+Brennanauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=C.+Buser-Doepnerauthor=R.+M.+Campbellauthor=A.+J.+Carterauthor=P.+Cohenauthor=R.+A.+Copelandauthor=B.+Cravattauthor=J.+L.+Dahlinauthor=D.+Dhanakauthor=A.+M.+Edwardsauthor=M.+Frederiksenauthor=S.+V.+Fryeauthor=N.+Grayauthor=C.+E.+Grimshawauthor=D.+Hepworthauthor=T.+Howeauthor=K.+V.+Huberauthor=J.+Jinauthor=S.+Knappauthor=J.+D.+Kotzauthor=R.+G.+Krugerauthor=D.+Loweauthor=M.+M.+Maderauthor=B.+Marsdenauthor=A.+Mueller-Fahrnowauthor=S.+M%C3%BCllerauthor=R.+C.+O%E2%80%99Haganauthor=J.+P.+Overingtonauthor=D.+R.+Owenauthor=S.+H.+Rosenbergauthor=B.+Rothauthor=R.+Rossauthor=M.+Schapiraauthor=S.+L.+Schreiberauthor=B.+Shoichetauthor=M.+Sundstr%C3%B6mauthor=G.+Superti-Furgaauthor=J.+Tauntonauthor=L.+Toledo-Shermanauthor=C.+Walpoleauthor=M.+A.+Waltersauthor=T.+M.+Willsonauthor=P.+Workmanauthor=R.+N.+Youngauthor=W.+J.+Zuercher&title=The+promise+and+peril+of+chemical+probes&doi=10.1038%2Fnchembio.1867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9aR"><div class="casContent"><span class="casTitleNuber">9a</span><div class="casTitle"><span class="NLM_cas:atitle">The promise and peril of chemical probes</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Audia, James E.; Austin, Christopher; Baell, Jonathan; Bennett, Jonathan; Blagg, Julian; Bountra, Chas; Brennan, Paul E.; Brown, Peter J.; Bunnage, Mark E.; Buser-Doepner, Carolyn; Campbell, Robert M.; Carter, Adrian J.; Cohen, Philip; Copeland, Robert A.; Cravatt, Ben; Dahlin, Jayme L.; Dhanak, Dashyant; Edwards, Aled M.; Frye, Stephen V.; Gray, Nathanael; Grimshaw, Charles E.; Hepworth, David; Howe, Trevor; Huber, Kilian V. M.; Jin, Jian; Knapp, Stefan; Kotz, Joanne D.; Kruger, Ryan G.; Lowe, Derek; Mader, Mary M.; Marsden, Brian; Mueller-Fahrnow, Anke; Muller, Susanne; O'Hagan, Ronan C.; Overington, John P.; Owen, Dafydd R.; Rosenberg, Saul H.; Roth, Brian; Ross, Ruth; Schapira, Matthieu; Schreiber, Stuart L.; Shoichet, Brian; Sundstrom, Michael; Superti-Furga, Giulio; Taunton, Jack; Toledo-Sherman, Leticia; Walpole, Chris; Walters, Michael A.; Willson, Timothy M.; Workman, Paul; Young, Robert N.; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are powerful reagents with increasing impacts on biomedical research.  However, probes of poor quality or that are used incorrectly generate misleading results.  To help address these shortcomings, we will create a community-driven wiki resource to improve quality and convey current best practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-li4dxq487rVg90H21EOLACvtfcHk0ljxmTsxykgahg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO&md5=d8c7a7af94921ab4cdcc871487cceadd</span></div><a href="/servlet/linkout?suffix=cit9a&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1867%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%26aulast%3DDahlin%26aufirst%3DJ.%2BL.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DK.%2BV.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKotz%26aufirst%3DJ.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%2BM.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Hagan%26aufirst%3DR.%2BC.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DRoth%26aufirst%3DB.%26aulast%3DRoss%26aufirst%3DR.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DSundstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DWalpole%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BN.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26atitle%3DThe%2520promise%2520and%2520peril%2520of%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnchembio.1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit9b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, I.</span><span> </span><span class="NLM_article-title">Probing the probes: fitness factors for small molecule tools</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">561</span><span class="NLM_x">–</span> <span class="NLM_lpage">577</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2010.05.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.chembiol.2010.05.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=20609406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVWqs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=561-577&author=P.+Workmanauthor=I.+Collins&title=Probing+the+probes%3A+fitness+factors+for+small+molecule+tools&doi=10.1016%2Fj.chembiol.2010.05.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9bR"><div class="casContent"><span class="casTitleNuber">9b</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the Probes: Fitness Factors For Small Molecule Tools</span></div><div class="casAuthors">Workman, Paul; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">561-577</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Chem. probes for interrogating biol. processes are of considerable current interest.  Cell permeable small mol. tools have a major role in facilitating the functional annotation of the human genome, understanding both physiol. and pathol. processes, and validating new mol. targets.  To be valuable, chem. tools must satisfy necessary criteria and recent publications have suggested objective guidelines for what makes a useful chem. probe.  Although recognizing that such guidelines may be valuable, we caution against overly restrictive rules that may stifle innovation in favor of a "fit-for-purpose" approach.  Reviewing the literature and providing examples from the cancer field, we recommend a series of "fitness factors" to be considered when assessing chem. probes.  We hope this will encourage innovative chem. biol. research while minimizing the generation of poor quality and misleading biol. data, thus increasing understanding of the particular biol. area, to the benefit of basic research and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQtty9oZD17rVg90H21EOLACvtfcHk0ljxmTsxykgahg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVWqs78%253D&md5=16be01da2339173aae8207030bf83c12</span></div><a href="/servlet/linkout?suffix=cit9b&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.05.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.05.013%26sid%3Dliteratum%253Aachs%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DProbing%2520the%2520probes%253A%2520fitness%2520factors%2520for%2520small%2520molecule%2520tools%26jtitle%3DChem.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D561%26epage%3D577%26doi%3D10.1016%2Fj.chembiol.2010.05.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit9c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piantnitski Chekler, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, L. H.</span><span> </span><span class="NLM_article-title">Target validation using chemical probes</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">195</span><span class="NLM_x">–</span> <span class="NLM_lpage">199</span><span class="refDoi"> DOI: 10.1038/nchembio.1197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnchembio.1197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=23508172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=195-199&author=M.+E.+Bunnageauthor=E.+L.+Piantnitski+Cheklerauthor=L.+H.+Jones&title=Target+validation+using+chemical+probes&doi=10.1038%2Fnchembio.1197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9cR"><div class="casContent"><span class="casTitleNuber">9c</span><div class="casTitle"><span class="NLM_cas:atitle">Target validation using chemical probes</span></div><div class="casAuthors">Bunnage, Mark E.; Chekler, Eugene L. Piatnitski; Jones, Lyn H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">195-199</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review on the fully profiled chem. probes essential to support the unbiased interpretation of biol. expts. necessary for rigorous preclin. target validation.  We believe that by developing a 'chem. probe tool kit', and a framework for its use, chem. biol. can have a more central role in identifying targets of potential relevance to disease, avoiding many of the biases that complicate target validation as practiced currently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLwRTT4pXGxbVg90H21EOLACvtfcHk0li3VfkhRQSAZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D&md5=af0c362882191de96a8e8698ff2c9ce8</span></div><a href="/servlet/linkout?suffix=cit9c&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1197%26sid%3Dliteratum%253Aachs%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DPiantnitski%2BChekler%26aufirst%3DE.%2BL.%26aulast%3DJones%26aufirst%3DL.%2BH.%26atitle%3DTarget%2520validation%2520using%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D195%26epage%3D199%26doi%3D10.1038%2Fnchembio.1197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Ziegler, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pries, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hedberg, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span> </span><span class="NLM_article-title">Target identification for small bioactive molecules: finding the needle in the haystack</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2744</span><span class="NLM_x">–</span> <span class="NLM_lpage">2792</span><span class="refDoi"> DOI: 10.1002/anie.201208749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1002%2Fanie.201208749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=23418026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXislSksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=2744-2792&author=S.+Zieglerauthor=V.+Priesauthor=C.+Hedbergauthor=H.+Waldmann&title=Target+identification+for+small+bioactive+molecules%3A+finding+the+needle+in+the+haystack&doi=10.1002%2Fanie.201208749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10aR"><div class="casContent"><span class="casTitleNuber">10a</span><div class="casTitle"><span class="NLM_cas:atitle">Target Identification for Small Bioactive Molecules: Finding the Needle in the Haystack</span></div><div class="casAuthors">Ziegler, Slava; Pries, Verena; Hedberg, Christian; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2744-2792</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Identification and confirmation of bioactive small-mol. targets is a crucial, often decisive step both in academic and pharmaceutical research.  Through the development and availability of several new exptl. techniques, target identification is, in principle, feasible, and the no. of successful examples steadily grows.  However, a generic methodol. that can successfully be applied in the majority of the cases has not yet been established.  Herein we summarize current methods for target identification of small mols., primarily for a chem. audience but also the biol. community, for example, the chemist or biologist attempting to identify the target of a given bioactive compd.  We describe the most frequently employed exptl. approaches for target identification and provide several representative examples illustrating the state-of-the-art.  Among the techniques currently available, protein affinity isolation using suitable small-mol. probes (pulldown) and subsequent mass spectrometric anal. of the isolated proteins appears to be most powerful and most frequently applied.  To provide guidance for rapid entry into the field and based on our own experience we propose a typical workflow for target identification, which centers on the application of chem. proteomics as the key step to generate hypotheses for potential target proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYC9-7L3abMrVg90H21EOLACvtfcHk0li3VfkhRQSAZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXislSksLk%253D&md5=3a124205e3a57a53f479b889073bda76</span></div><a href="/servlet/linkout?suffix=cit10a&amp;dbid=16384&amp;doi=10.1002%2Fanie.201208749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201208749%26sid%3Dliteratum%253Aachs%26aulast%3DZiegler%26aufirst%3DS.%26aulast%3DPries%26aufirst%3DV.%26aulast%3DHedberg%26aufirst%3DC.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DTarget%2520identification%2520for%2520small%2520bioactive%2520molecules%253A%2520finding%2520the%2520needle%2520in%2520the%2520haystack%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D2744%26epage%3D2792%26doi%3D10.1002%2Fanie.201208749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit10b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Rix, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span> </span><span class="NLM_article-title">Target profiling of small molecules by chemical proteomics</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">616</span><span class="NLM_x">–</span> <span class="NLM_lpage">624</span><span class="refDoi"> DOI: 10.1038/nchembio.216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnchembio.216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=19690537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVWhsb7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=616-624&author=U.+Rixauthor=G.+Superti-Furga&title=Target+profiling+of+small+molecules+by+chemical+proteomics&doi=10.1038%2Fnchembio.216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10bR"><div class="casContent"><span class="casTitleNuber">10b</span><div class="casTitle"><span class="NLM_cas:atitle">Target profiling of small molecules by chemical proteomics</span></div><div class="casAuthors">Rix, Uwe; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">616-624</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The medical and pharmaceutical communities are facing a dire need for new druggable targets, while, paradoxically, the targets of some drugs that are in clin. use or development remain elusive.  Many compds. have been more promiscuous than originally anticipated, which can potentially lead to side effects, but which may also open up addnl. medical uses.  As we move toward systems biol. and personalized medicine, comprehensively detg. small mol.-target interaction profiles and mapping these on signaling and metabolic pathways will become increasingly necessary.  Chem. proteomics is a powerful mass spectrometry-based affinity chromatog. approach for identifying proteome-wide small mol.-protein interactions.  Here we will provide a crit. overview of the basic concepts and recent advances in chem. proteomics and review recent applications, with a particular emphasis on kinase inhibitors and natural products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBJz9OMT5Rp7Vg90H21EOLACvtfcHk0li3VfkhRQSAZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVWhsb7K&md5=b357080902ea5e6cd6ab51792f327d0b</span></div><a href="/servlet/linkout?suffix=cit10b&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.216%26sid%3Dliteratum%253Aachs%26aulast%3DRix%26aufirst%3DU.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DTarget%2520profiling%2520of%2520small%2520molecules%2520by%2520chemical%2520proteomics%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D616%26epage%3D624%26doi%3D10.1038%2Fnchembio.216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Schirle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span> </span><span class="NLM_article-title">Identifying compound efficacy targets in phenotypic drug discovery</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">82</span><span class="NLM_x">–</span> <span class="NLM_lpage">89</span><span class="refDoi"> DOI: 10.1016/j.drudis.2015.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.drudis.2015.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=26272035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlCisrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=82-89&author=M.+Schirleauthor=J.+L.+Jenkins&title=Identifying+compound+efficacy+targets+in+phenotypic+drug+discovery&doi=10.1016%2Fj.drudis.2015.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying compound efficacy targets in phenotypic drug discovery</span></div><div class="casAuthors">Schirle, Markus; Jenkins, Jeremy L.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">82-89</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The identification of the efficacy target(s) for hits from phenotypic compd. screens remains a key step to progress compds. into drug development.  In addn. to efficacy targets, the characterization of epistatic proteins influencing compd. activity often facilitates the elucidation of the underlying mechanism of action; and, further, early detn. of off-targets that cause potentially unwanted secondary phenotypes helps in assessing potential liabilities.  This short review discusses the most important technologies currently available for characterizing the direct and indirect target space of bioactive compds. following phenotypic screening.  We present a comprehensive strategy employing complementary approaches to balance individual technol. strengths and weaknesses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNnxrH8t-we7Vg90H21EOLACvtfcHk0lh99QNOeZm37A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlCisrvF&md5=b373d239d72058a1b826ae1550bb4693</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26atitle%3DIdentifying%2520compound%2520efficacy%2520targets%2520in%2520phenotypic%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D82%26epage%3D89%26doi%3D10.1016%2Fj.drudis.2015.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Hann, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keseru, G. M.</span><span> </span><span class="NLM_article-title">Finding the sweet spot: the role of nature and nurture in medicinal chemistry</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">355</span><span class="NLM_x">–</span> <span class="NLM_lpage">365</span><span class="refDoi"> DOI: 10.1038/nrd3701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnrd3701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=22543468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt1GqsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=355-365&author=M.+M.+Hannauthor=G.+M.+Keseru&title=Finding+the+sweet+spot%3A+the+role+of+nature+and+nurture+in+medicinal+chemistry&doi=10.1038%2Fnrd3701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Finding the sweet spot: the role of nature and nurture in medicinal chemistry</span></div><div class="casAuthors">Hann, Michael M.; Keserue, Gyoergy M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">355-365</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Given its position at the heart of small-mol. drug discovery, medicinal chem. has an important role in tackling the well-known productivity challenges in pharmaceutical research and development.  In recent years, extensive analyses of successful and failed discovery compds. and drug candidates have improved our understanding of the role of physicochem. properties in drug attrition.  Based on the clarified challenges in finding the 'sweet spot' in medicinal chem. programs, we suggest that this goal can be achieved through a combination of first identifying chem. starting points with appropriate 'nature' and then rigorously 'nurturing' them during lead optimization.  Here, we discuss scientific, strategic, organizational and cultural considerations for medicinal chem. practices, with the aim of promoting more effective use of what is already known, as well as a wider appreciation of the risks of pursuing suboptimal compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrURciGjB2A3LVg90H21EOLACvtfcHk0lh99QNOeZm37A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt1GqsLY%253D&md5=90170e3f4e69e52bfa38667df380516a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrd3701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3701%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26atitle%3DFinding%2520the%2520sweet%2520spot%253A%2520the%2520role%2520of%2520nature%2520and%2520nurture%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D355%26epage%3D365%26doi%3D10.1038%2Fnrd3701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Vihervaara, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sistonen, L.</span><span> </span><span class="NLM_article-title">HSF1 at a glance</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">261</span><span class="NLM_x">–</span> <span class="NLM_lpage">266</span><span class="refDoi"> DOI: 10.1242/jcs.132605</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1242%2Fjcs.132605" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=24421309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivFSgtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2014&pages=261-266&author=A.+Vihervaaraauthor=L.+Sistonen&title=HSF1+at+a+glance&doi=10.1242%2Fjcs.132605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13aR"><div class="casContent"><span class="casTitleNuber">13a</span><div class="casTitle"><span class="NLM_cas:atitle">HSF1 at a glance</span></div><div class="casAuthors">Vihervaara, Anniina; Sistonen, Lea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">261-266</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">A review.  Heat-shock factor 1 (HSF1) is an evolutionarily highly conserved transcription factor that coordinates stress-induced transcription and directs versatile physiol. processes in eukaryotes.  The central position of HSF1 in cellular homeostasis has been well demonstrated, mainly through its strong effect in transactivating genes that encode heat-shock proteins (HSPs).  However, recent genome-wide studies have revealed that HSF1 is capable of reprogramming transcription more extensively than previously assumed; it is also involved in a multitude of processes in stressed and non-stressed cells.  Consequently, the importance of HSF1 in fundamental physiol. events, including metab., gametogenesis and aging, has become apparent and its significance in pathologies, such as cancer progression, is now evident.  Here, the authors highlight recent advances in the HSF1 field, discuss the organismal control over HSF1, and present the processes that are mediated by HSF1 in the context of cell type, cell-cycle phase, physiol. condition and received stimuli.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopQBde2HgDRbVg90H21EOLACvtfcHk0lgtjDZKM7nMdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXivFSgtro%253D&md5=0e818f15568b3027c7bc4bc6dbec1d31</span></div><a href="/servlet/linkout?suffix=cit13a&amp;dbid=16384&amp;doi=10.1242%2Fjcs.132605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.132605%26sid%3Dliteratum%253Aachs%26aulast%3DVihervaara%26aufirst%3DA.%26aulast%3DSistonen%26aufirst%3DL.%26atitle%3DHSF1%2520at%2520a%2520glance%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2014%26volume%3D127%26spage%3D261%26epage%3D266%26doi%3D10.1242%2Fjcs.132605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit13b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Anckar, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sistonen, L.</span><span> </span><span class="NLM_article-title">Regulation of HSF1 function in the heat stress response: implications in aging and disease</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">1089</span><span class="NLM_x">–</span> <span class="NLM_lpage">1115</span><span class="refDoi"> DOI: 10.1146/annurev-biochem-060809-095203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1146%2Fannurev-biochem-060809-095203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=21417720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVCntrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2011&pages=1089-1115&author=J.+Anckarauthor=L.+Sistonen&title=Regulation+of+HSF1+function+in+the+heat+stress+response%3A+implications+in+aging+and+disease&doi=10.1146%2Fannurev-biochem-060809-095203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13bR"><div class="casContent"><span class="casTitleNuber">13b</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of HSF1 function in the heat stress response: implications in aging and disease</span></div><div class="casAuthors">Anckar, Julius; Sistonen, Lea</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1089-1115</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  To dampen proteotoxic stresses and maintain protein homeostasis, organisms possess a stress-responsive mol. machinery that detects and neutralizes protein damage.  A prominent feature of stressed cells is the increased synthesis of heat shock proteins (Hsps) that aid in the refolding of misfolded peptides and restrain protein aggregation.  Transcriptional activation of the heat shock response is orchestrated by heat shock factor 1 (HSF1), which rapidly translocates to hsp genes and induces their expression.  Although the role of HSF1 in protecting cells and organisms against severe stress insults is well established, many aspects of how HSF1 senses qual. and quant. different forms of stresses have remained poorly understood.  Moreover, recent discoveries that HSF1 controls life span have prompted new ways of thinking about an old transcription factor.  Here, we review the established role of HSF1 in counteracting cell stress and prospect the role of HSF1 as a regulator of disease states and aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraZhoheJ2_J7Vg90H21EOLACvtfcHk0lgtjDZKM7nMdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVCntrc%253D&md5=c8d529a0d535211cfbe9281c80202c3a</span></div><a href="/servlet/linkout?suffix=cit13b&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-060809-095203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-060809-095203%26sid%3Dliteratum%253Aachs%26aulast%3DAnckar%26aufirst%3DJ.%26aulast%3DSistonen%26aufirst%3DL.%26atitle%3DRegulation%2520of%2520HSF1%2520function%2520in%2520the%2520heat%2520stress%2520response%253A%2520implications%2520in%2520aging%2520and%2520disease%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2011%26volume%3D80%26spage%3D1089%26epage%3D1115%26doi%3D10.1146%2Fannurev-biochem-060809-095203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit13c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Jiang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmitt, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span> </span><span class="NLM_article-title">Multifaceted roles of HSF1 in cancer</span> <span class="citation_source-journal">Tumor Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">4923</span><span class="NLM_x">–</span> <span class="NLM_lpage">4931</span><span class="refDoi"> DOI: 10.1007/s13277-015-3674-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1007%2Fs13277-015-3674-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=26108999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtV2lsbfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=4923-4931&author=S.+Jiangauthor=K.+Tuauthor=Q.+Fuauthor=D.+C.+Schmittauthor=L.+Zhouauthor=N.+Luauthor=Y.+Zhao&title=Multifaceted+roles+of+HSF1+in+cancer&doi=10.1007%2Fs13277-015-3674-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13cR"><div class="casContent"><span class="casTitleNuber">13c</span><div class="casTitle"><span class="NLM_cas:atitle">Multifaceted roles of HSF1 in cancer</span></div><div class="casAuthors">Jiang, Sufang; Tu, Kailing; Fu, Qiang; Schmitt, David C.; Zhou, Lan; Lu, Na; Zhao, Yuhua</div><div class="citationInfo"><span class="NLM_cas:title">Tumor Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4923-4931</span>CODEN:
                <span class="NLM_cas:coden">TUMBEA</span>;
        ISSN:<span class="NLM_cas:issn">1010-4283</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Heat shock transcription factor 1 (HSF1) is the master regulator of the heat shock response.  Accumulating evidence shows that HSF1 is overexpressed in a variety of human cancers, is assocd. with cancer aggressiveness, and could serve as an independent diagnostic or prognostic biomarker.  In this review, we will provide an overview of the multifaceted roles of HSF1 in cancer, with a special focus on the four underlying mol. mechanisms involved.  First, HSF1 regulates the expression of heat shock proteins (HSPs) including HSP90, HSP70, and HSP27.  Second, HSF1 regulates cellular metab., including glycolysis and lipid metab.  Third, HSF1 serves as a regulator of different signaling pathways, such as HuR-HIF-1, Slug, protein kinase C (PKC), nuclear factor-kappaB (NF-κB), PI3K-AKT-mTOR, and mitogen-activated protein kinase (MAPK) pathways.  Finally, HSF1 regulates microRNAs (miRNAs) and long non-coding RNAs (lncRNAs).  Overall, HSF1 plays many important roles in cancer via regulating cell proliferation, anti-apoptosis, epithelial-mesenchymal transition (EMT), migration, invasion, and metastasis and may be a potential therapeutic target for human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ3CTGJiNLPrVg90H21EOLACvtfcHk0lgtjDZKM7nMdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtV2lsbfO&md5=05d25d9d33dfb74351bc073d678640ee</span></div><a href="/servlet/linkout?suffix=cit13c&amp;dbid=16384&amp;doi=10.1007%2Fs13277-015-3674-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-015-3674-x%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DTu%26aufirst%3DK.%26aulast%3DFu%26aufirst%3DQ.%26aulast%3DSchmitt%26aufirst%3DD.%2BC.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DN.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DMultifaceted%2520roles%2520of%2520HSF1%2520in%2520cancer%26jtitle%3DTumor%2520Biol.%26date%3D2015%26volume%3D36%26spage%3D4923%26epage%3D4931%26doi%3D10.1007%2Fs13277-015-3674-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Whitesell, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindquist, S.</span><span> </span><span class="NLM_article-title">Inhibiting the transcription factor HSF1 as an anticancer strategy</span> <span class="citation_source-journal">Expert Opin. Ther. Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">469</span><span class="NLM_x">–</span> <span class="NLM_lpage">478</span><span class="refDoi"> DOI: 10.1517/14728220902832697</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1517%2F14728220902832697" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=19335068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjvVWnsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2009&pages=469-478&author=L.+Whitesellauthor=S.+Lindquist&title=Inhibiting+the+transcription+factor+HSF1+as+an+anticancer+strategy&doi=10.1517%2F14728220902832697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14aR"><div class="casContent"><span class="casTitleNuber">14a</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting the transcription factor HSF1 as an anticancer strategy</span></div><div class="casAuthors">Whitesell, Luke; Lindquist, Susan</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">469-478</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  In mammals, the cytoprotective heat-shock response is regulated primarily by heat shock factor 1 (HSF1).  Unfortunately, the effects of HSF1 also support the ability of cancer cells to accommodate imbalances in signaling and alterations in DNA, protein and energy metab. assocd. with oncogenesis.  The malignant lifestyle confers dependence on this non-oncogene', suggesting a therapeutic role for HSF1 inhibitors.  We begin with an overview of how HSF1 affects cancer biol. and how its activity is regulated.  We then summarize progress in discovery and development of HSF1 inhibitors, their current limitations and potential as anticancer agents with a fundamentally different scope of action from other clin. validated modulators of protein homeostasis.  It is likely that within the next 5 years usable inhibitors of HSF1 will be identified and in early pre-clin. evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_J1_jCERfLbVg90H21EOLACvtfcHk0lgtjDZKM7nMdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjvVWnsr8%253D&md5=adfbdeed040bf19a03f7e4d6f69c0c6c</span></div><a href="/servlet/linkout?suffix=cit14a&amp;dbid=16384&amp;doi=10.1517%2F14728220902832697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728220902832697%26sid%3Dliteratum%253Aachs%26aulast%3DWhitesell%26aufirst%3DL.%26aulast%3DLindquist%26aufirst%3DS.%26atitle%3DInhibiting%2520the%2520transcription%2520factor%2520HSF1%2520as%2520an%2520anticancer%2520strategy%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2009%26volume%3D13%26spage%3D469%26epage%3D478%26doi%3D10.1517%2F14728220902832697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit14b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Moore, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dewal, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nekongo, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santiago, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levine, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoulders, M. D.</span><span> </span><span class="NLM_article-title">Transportable, chemical genetic methodology for the small molecule-mediated inhibition of heat shock factor 1</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">200</span><span class="NLM_x">–</span> <span class="NLM_lpage">210</span><span class="refDoi"> DOI: 10.1021/acschembio.5b00740</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00740" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslSrtLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=200-210&author=C.+L.+Mooreauthor=M.+B.+Dewalauthor=E.+E.+Nekongoauthor=S.+Santiagoauthor=N.+B.+Luauthor=S.+S.+Levineauthor=M.+D.+Shoulders&title=Transportable%2C+chemical+genetic+methodology+for+the+small+molecule-mediated+inhibition+of+heat+shock+factor+1&doi=10.1021%2Facschembio.5b00740"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14bR"><div class="casContent"><span class="casTitleNuber">14b</span><div class="casTitle"><span class="NLM_cas:atitle">Transportable, Chemical Genetic Methodology for the Small Molecule-Mediated Inhibition of Heat Shock Factor 1</span></div><div class="casAuthors">Moore, Christopher L.; Dewal, Mahender B.; Nekongo, Emmanuel E.; Santiago, Sebasthian; Lu, Nancy B.; Levine, Stuart S.; Shoulders, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">200-210</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteostasis in the cytosol is governed by the heat shock response.  The master regulator of the heat shock response, heat shock factor 1 (HSF1), and key chaperones whose levels are HSF1-regulated have emerged as high-profile targets for therapeutic applications ranging from protein misfolding-related disorders to cancer.  Nonetheless, a generally applicable methodol. to selectively and potently inhibit endogenous HSF1 in a small mol.-dependent manner in disease model systems remains elusive.  Also problematic, the administration of even highly selective chaperone inhibitors often has the side effect of activating HSF1 and thereby inducing a compensatory heat shock response.  Herein, we report a ligand-regulatable, dominant neg. version of HSF1 that addresses these issues.  Our approach, which required engineering a new dominant neg. HSF1 variant, permits dosable inhibition of endogenous HSF1 with a selective small mol. in cell-based model systems of interest.  The methodol. allows us to uncouple the pleiotropic effects of chaperone inhibitors and environmental toxins from the concomitantly induced compensatory heat shock response.  Integration of our method with techniques to activate HSF1 enables the creation of cell lines in which the cytosolic proteostasis network can be up- or down-regulated by orthogonal small mols.  Selective, small mol.-mediated inhibition of HSF1 has distinctive implications for the proteostasis of both chaperone-dependent globular proteins and aggregation-prone intrinsically disordered proteins.  Altogether, this work provides crit. methods for continued exploration of the biol. roles of HSF1 and the therapeutic potential of heat shock response modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKGNceoLmk-rVg90H21EOLACvtfcHk0liq-lilH36npA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslSrtLfJ&md5=2d04135b9393ae16ee1bce79c6d890be</span></div><a href="/servlet/linkout?suffix=cit14b&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00740&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00740%26sid%3Dliteratum%253Aachs%26aulast%3DMoore%26aufirst%3DC.%2BL.%26aulast%3DDewal%26aufirst%3DM.%2BB.%26aulast%3DNekongo%26aufirst%3DE.%2BE.%26aulast%3DSantiago%26aufirst%3DS.%26aulast%3DLu%26aufirst%3DN.%2BB.%26aulast%3DLevine%26aufirst%3DS.%2BS.%26aulast%3DShoulders%26aufirst%3DM.%2BD.%26atitle%3DTransportable%252C%2520chemical%2520genetic%2520methodology%2520for%2520the%2520small%2520molecule-mediated%2520inhibition%2520of%2520heat%2520shock%2520factor%25201%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D200%26epage%3D210%26doi%3D10.1021%2Facschembio.5b00740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit14c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">de Billy, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Travers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span> </span><span class="NLM_article-title">Shock about heat shock in cancer</span> <span class="citation_source-journal">Oncotarget</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">741</span><span class="NLM_x">–</span> <span class="NLM_lpage">743</span><span class="refDoi"> DOI: 10.18632/oncotarget.646</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.18632%2Foncotarget.646" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=741-743&author=E.+de+Billyauthor=J.+Traversauthor=P.+Workman&title=Shock+about+heat+shock+in+cancer&doi=10.18632%2Foncotarget.646"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14c&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.646%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BBilly%26aufirst%3DE.%26aulast%3DTravers%26aufirst%3DJ.%26aulast%3DWorkman%26aufirst%3DP.%26atitle%3DShock%2520about%2520heat%2520shock%2520in%2520cancer%26jtitle%3DOncotarget%26date%3D2012%26volume%3D3%26spage%3D741%26epage%3D743%26doi%3D10.18632%2Foncotarget.646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Mendillo, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santagata, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koeva, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tamimi, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraenkel, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ince, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitesell, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindquist, S.</span><span> </span><span class="NLM_article-title">HSF1 drives a transcriptional program distinct from heat shock to support highly malignant human cancers</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">150</span><span class="NLM_x">, </span> <span class="NLM_fpage">549</span><span class="NLM_x">–</span> <span class="NLM_lpage">562</span><span class="refDoi"> DOI: 10.1016/j.cell.2012.06.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.cell.2012.06.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=22863008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFGrsLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2012&pages=549-562&author=M.+L.+Mendilloauthor=S.+Santagataauthor=M.+Koevaauthor=G.+W.+Bellauthor=R.+Huauthor=R.+M.+Tamimiauthor=E.+Fraenkelauthor=T.+A.+Inceauthor=L.+Whitesellauthor=S.+Lindquist&title=HSF1+drives+a+transcriptional+program+distinct+from+heat+shock+to+support+highly+malignant+human+cancers&doi=10.1016%2Fj.cell.2012.06.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">HSF1 drives a transcriptional program distinct from heat shock to support highly malignant human cancers</span></div><div class="casAuthors">Mendillo, Marc L.; Santagata, Sandro; Koeva, Martina; Bell, George W.; Hu, Rong; Tamimi, Rulla M.; Fraenkel, Ernest; Ince, Tan A.; Whitesell, Luke; Lindquist, Susan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">549-562</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Heat-Shock Factor 1 (HSF1), master regulator of the heat-shock response, facilitates malignant transformation, cancer cell survival, and proliferation in model systems.  The common assumption is that these effects are mediated through regulation of heat-shock protein (HSP) expression.  However, the transcriptional network that HSF1 coordinates directly in malignancy and its relationship to the heat-shock response have never been defined.  By comparing cells with high and low malignant potential alongside their nontransformed counterparts, we identify an HSF1-regulated transcriptional program specific to highly malignant cells and distinct from heat shock.  Cancer-specific genes in this program support oncogenic processes: cell-cycle regulation, signaling, metab., adhesion and translation.  HSP genes are integral to this program, however, many are uniquely regulated in malignancy.  This HSF1 cancer program is active in breast, colon and lung tumors isolated directly from human patients and is strongly assocd. with metastasis and death.  Thus, HSF1 rewires the transcriptome in tumorigenesis, with prognostic and therapeutic implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-7fuiXrhe0bVg90H21EOLACvtfcHk0liq-lilH36npA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFGrsLzE&md5=56a6bb78f044d25ead6338d2d9f82693</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.06.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.06.031%26sid%3Dliteratum%253Aachs%26aulast%3DMendillo%26aufirst%3DM.%2BL.%26aulast%3DSantagata%26aufirst%3DS.%26aulast%3DKoeva%26aufirst%3DM.%26aulast%3DBell%26aufirst%3DG.%2BW.%26aulast%3DHu%26aufirst%3DR.%26aulast%3DTamimi%26aufirst%3DR.%2BM.%26aulast%3DFraenkel%26aufirst%3DE.%26aulast%3DInce%26aufirst%3DT.%2BA.%26aulast%3DWhitesell%26aufirst%3DL.%26aulast%3DLindquist%26aufirst%3DS.%26atitle%3DHSF1%2520drives%2520a%2520transcriptional%2520program%2520distinct%2520from%2520heat%2520shock%2520to%2520support%2520highly%2520malignant%2520human%2520cancers%26jtitle%3DCell%26date%3D2012%26volume%3D150%26spage%3D549%26epage%3D562%26doi%3D10.1016%2Fj.cell.2012.06.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Santagata, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, N. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendillo, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hankinson, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnitt, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitesell, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tamimi, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindquist, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ince, T. A.</span><span> </span><span class="NLM_article-title">High levels of nuclear heat-shock factor 1 (HSF1) are associated with poor prognosis in breast cancer</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">18378</span><span class="NLM_x">–</span> <span class="NLM_lpage">18383</span><span class="refDoi"> DOI: 10.1073/pnas.1115031108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1073%2Fpnas.1115031108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=22042860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsF2itrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=18378-18383&author=S.+Santagataauthor=R.+Huauthor=N.+U.+Linauthor=M.+L.+Mendilloauthor=L.+C.+Collinsauthor=S.+E.+Hankinsonauthor=S.+J.+Schnittauthor=L.+Whitesellauthor=R.+M.+Tamimiauthor=S.+Lindquistauthor=T.+A.+Ince&title=High+levels+of+nuclear+heat-shock+factor+1+%28HSF1%29+are+associated+with+poor+prognosis+in+breast+cancer&doi=10.1073%2Fpnas.1115031108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16aR"><div class="casContent"><span class="casTitleNuber">16a</span><div class="casTitle"><span class="NLM_cas:atitle">High levels of nuclear heat-shock factor 1 (HSF1) are associated with poor prognosis in breast cancer</span></div><div class="casAuthors">Santagata, Sandro; Hu, Rong; Lin, Nancy U.; Mendillo, Marc L.; Collins, Laura C.; Hankinson, Susan E.; Schnitt, Stuart J.; Whitesell, Luke; Tamimi, Rulla M.; Lindquist, Susan; Ince, Tan A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">18378-18383, S18378/1-S18378/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Heat-shock factor 1 (HSF1) is the master transcriptional regulator of the cellular response to heat and a wide variety of other stressors.  We previously reported that HSF1 promotes the survival and proliferation of malignant cells.  At this time, however, the clin. and prognostic significance of HSF1 in cancer is unknown.  To address this issue breast cancer samples from 1,841 participants in the Nurses' Health Study were scored for levels of nuclear HSF1.  Assocns. of HSF1 status with clin. parameters and survival outcomes were investigated by Kaplan-Meier anal. and Cox proportional hazard models.  The assocns. were further delineated by Kaplan-Meier anal. using publicly available mRNA expression data.  Our results show that nuclear HSF1 levels were elevated in ∼80% of in situ and invasive breast carcinomas.  In invasive carcinomas, HSF1 expression was assocd. with high histol. grade, larger tumor size, and nodal involvement at diagnosis (P < 0.0001). By-using multivariate anal. to account for the effects of covariates, high HSF1 levels were found to be independently assocd. with increased mortality (hazards ratio: 1.62; 95% confidence interval: 1.21-2.17; P < 0.0013). This assocn. was seen in the estrogen receptor (ER)-pos. population (hazards ratio: 2.10; 95% confidence interval: 1.45-3.03; P < 0.0001). In public expression profiling data, high HSF1 mRNA levels were also assocd. with an increase in ER-pos. breast cancer-specific mortality.  We conclude that increased HSF1 is assocd. with reduced breast cancer survival.  The findings indicate that HSF1 should be evaluated prospectively as an independent prognostic indicator in ER-pos. breast cancer.  HSF1 may ultimately be a useful therapeutic target in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq93NKPzO0hArVg90H21EOLACvtfcHk0liq-lilH36npA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsF2itrvI&md5=097a836a6ec39049b70b60e6d42b4718</span></div><a href="/servlet/linkout?suffix=cit16a&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1115031108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1115031108%26sid%3Dliteratum%253Aachs%26aulast%3DSantagata%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DN.%2BU.%26aulast%3DMendillo%26aufirst%3DM.%2BL.%26aulast%3DCollins%26aufirst%3DL.%2BC.%26aulast%3DHankinson%26aufirst%3DS.%2BE.%26aulast%3DSchnitt%26aufirst%3DS.%2BJ.%26aulast%3DWhitesell%26aufirst%3DL.%26aulast%3DTamimi%26aufirst%3DR.%2BM.%26aulast%3DLindquist%26aufirst%3DS.%26aulast%3DInce%26aufirst%3DT.%2BA.%26atitle%3DHigh%2520levels%2520of%2520nuclear%2520heat-shock%2520factor%25201%2520%2528HSF1%2529%2520are%2520associated%2520with%2520poor%2520prognosis%2520in%2520breast%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D18378%26epage%3D18383%26doi%3D10.1073%2Fpnas.1115031108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit16b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Fang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, L.</span><span> </span><span class="NLM_article-title">Heat shock factor 1 promotes invasion and metastasis of hepatocellular carcinoma in vitro and in vivo</span> <span class="citation_source-journal">Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">1782</span><span class="NLM_x">–</span> <span class="NLM_lpage">1794</span><span class="refDoi"> DOI: 10.1002/cncr.26482</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1002%2Fcncr.26482" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=22009757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38XktFWmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2012&pages=1782-1794&author=F.+Fangauthor=R.+Changauthor=L.+Yang&title=Heat+shock+factor+1+promotes+invasion+and+metastasis+of+hepatocellular+carcinoma+in+vitro+and+in+vivo&doi=10.1002%2Fcncr.26482"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16bR"><div class="casContent"><span class="casTitleNuber">16b</span><div class="casTitle"><span class="NLM_cas:atitle">Heat shock factor 1 promotes invasion and metastasis of hepatocellular carcinoma in vitro and in vivo</span></div><div class="casAuthors">Fang, Feng; Chang, Ruimin; Yang, Lianyue</div><div class="citationInfo"><span class="NLM_cas:title">Cancer (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1782-1794</span>CODEN:
                <span class="NLM_cas:coden">CANCAR</span>;
        ISSN:<span class="NLM_cas:issn">0008-543X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">BACKGROUND: Heat shock factor 1 (HSF1) is a powerful, multifaceted modifier of carcinogenesis.  However, the clin. significance and biol. function of HSF1 in hepatocellular carcinoma (HCC) remain unknown.  METHODS: Quant. reverse transcriptase-polymerase chain reaction anal., Western blot anal., and immunohistochem. staining were used to detect expression levels of HSF1, and its correlation with clinicopathol. parameters and the prognosis for patients with HCC were analyzed.  In addn., the biol. function and mol. mechanisms of HSF1 in HCC were investigated in vitro and in vivo.  RESULTS: HSF1 levels were elevated predominantly in HCC, esp. in venous emboli from HCC (P < .05), and high expression levels of HSF1 were correlated significantly with multiple nodules, venous invasion, absence of capsular formation, and high Edmondson-Steiner grade as well as poor overall survival and disease-free survival in patients with HCC (P < .05).  Multivariate Cox regression anal. revealed that high HSF1 expression was an independent prognostic factor for overall survival in patients with HCC (relative risk, 4.874; P < .001).  Finally, HSF1 was capable of promoting HCC cell migration and invasion in vitro and in vivo by facilitating the expression and phosphorylation of heat shock protein 27.  CONCLUSIONS: Collectively, the current findings suggested that HSF1 may serve as a novel prognostic marker and therapeutic target for HCC.  Cancer 2012;. © 2011 American Cancer Society.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8_TwAVVM7yLVg90H21EOLACvtfcHk0liaj-XOtIgW-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktFWmsbs%253D&md5=f95636fc9d7e691b3f7bbfe4cec30425</span></div><a href="/servlet/linkout?suffix=cit16b&amp;dbid=16384&amp;doi=10.1002%2Fcncr.26482&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcncr.26482%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DF.%26aulast%3DChang%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DL.%26atitle%3DHeat%2520shock%2520factor%25201%2520promotes%2520invasion%2520and%2520metastasis%2520of%2520hepatocellular%2520carcinoma%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DCancer%26date%3D2012%26volume%3D118%26spage%3D1782%26epage%3D1794%26doi%3D10.1002%2Fcncr.26482" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit16c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Scott, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nogueira, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heffernan, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Doorn, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhakal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanna, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaskelioff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, C.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeid, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feinberg, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanede Woude, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Granter, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bosenberg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DePinho, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rimm, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chin, L.</span><span> </span><span class="NLM_article-title">Pro-invasion metastasis drivers in early stage melanoma are oncogenes</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">92</span><span class="NLM_x">–</span> <span class="NLM_lpage">103</span><span class="refDoi"> DOI: 10.1016/j.ccr.2011.05.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.ccr.2011.05.025" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=92-103&author=K.+L.+Scottauthor=C.+Nogueiraauthor=T.+P.+Heffernanauthor=R.+van+Doornauthor=S.+Dhakalauthor=J.+A.+Hannaauthor=C.+Minauthor=M.+Jaskelioffauthor=Y.+Xiaoauthor=C.-J.+Wuauthor=L.+A.+Cameronauthor=S.+R.+Perryauthor=R.+Zeidauthor=T.+Feinbergauthor=M.+Kimauthor=G.+Vanede+Woudeauthor=S.+R.+Granterauthor=M.+Bosenbergauthor=G.+C.+Chuauthor=R.+A.+DePinhoauthor=D.+L.+Rimmauthor=L.+Chin&title=Pro-invasion+metastasis+drivers+in+early+stage+melanoma+are+oncogenes&doi=10.1016%2Fj.ccr.2011.05.025"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16c&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.05.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.05.025%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DK.%2BL.%26aulast%3DNogueira%26aufirst%3DC.%26aulast%3DHeffernan%26aufirst%3DT.%2BP.%26aulast%3Dvan%2BDoorn%26aufirst%3DR.%26aulast%3DDhakal%26aufirst%3DS.%26aulast%3DHanna%26aufirst%3DJ.%2BA.%26aulast%3DMin%26aufirst%3DC.%26aulast%3DJaskelioff%26aufirst%3DM.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DC.-J.%26aulast%3DCameron%26aufirst%3DL.%2BA.%26aulast%3DPerry%26aufirst%3DS.%2BR.%26aulast%3DZeid%26aufirst%3DR.%26aulast%3DFeinberg%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DM.%26aulast%3DVanede%2BWoude%26aufirst%3DG.%26aulast%3DGranter%26aufirst%3DS.%2BR.%26aulast%3DBosenberg%26aufirst%3DM.%26aulast%3DChu%26aufirst%3DG.%2BC.%26aulast%3DDePinho%26aufirst%3DR.%2BA.%26aulast%3DRimm%26aufirst%3DD.%2BL.%26aulast%3DChin%26aufirst%3DL.%26atitle%3DPro-invasion%2520metastasis%2520drivers%2520in%2520early%2520stage%2520melanoma%2520are%2520oncogenes%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D20%26spage%3D92%26epage%3D103%26doi%3D10.1016%2Fj.ccr.2011.05.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="note"><p class="first last">For a discussion of transcription inhibitors in oncology, see:</p></div><div class="NLM_citation" id="rightTab-cit17a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Bensaude, O.</span><span> </span><span class="NLM_article-title">Inhibiting eukaryotic transcription, which compound to choose? how to evaluate its activity?</span> <span class="citation_source-journal">Transcr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">103</span><span class="NLM_x">–</span> <span class="NLM_lpage">108</span><span class="refDoi"> DOI: 10.4161/trns.2.3.16172</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.4161%2Ftrns.2.3.16172" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=21922053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A280%3ADC%252BC2sbovVCrtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=103-108&author=O.+Bensaude&title=Inhibiting+eukaryotic+transcription%2C+which+compound+to+choose%3F+how+to+evaluate+its+activity%3F&doi=10.4161%2Ftrns.2.3.16172"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17aR"><div class="casContent"><span class="casTitleNuber">17a</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting eukaryotic transcription: Which compound to choose? How to evaluate its activity?</span></div><div class="casAuthors">Bensaude Olivier</div><div class="citationInfo"><span class="NLM_cas:title">Transcription</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">103-108</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">This review first discusses ways in which we can evaluate transcription inhibition, describe changes in nuclear structure due to transcription inhibition, and report on genes that are paradoxically stimulated by transcription inhibition.  Next, it summarizes the characteristics and mechanisms of commonly used inhibitors: α-amanitin is highly selective for RNAP II and RNAP III but its action is slow, actinomycin D is fast but its selectivity is poor, CDK9 inhibitors such as DRB and flavopiridol are fast and reversible but many genes escape transcription inhibition.  New compounds, such as triptolide, are fast and selective and able to completely arrest transcription by triggering rapid degradation of RNAP II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTeQjDGoYQ9mtHxxy173TfNfW6udTcc2ebF8Pyg1famSbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sbovVCrtw%253D%253D&md5=9fb95455b03ee7a9365b4c83435fa6ea</span></div><a href="/servlet/linkout?suffix=cit17a&amp;dbid=16384&amp;doi=10.4161%2Ftrns.2.3.16172&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Ftrns.2.3.16172%26sid%3Dliteratum%253Aachs%26aulast%3DBensaude%26aufirst%3DO.%26atitle%3DInhibiting%2520eukaryotic%2520transcription%252C%2520which%2520compound%2520to%2520choose%253F%2520how%2520to%2520evaluate%2520its%2520activity%253F%26jtitle%3DTranscr.%26date%3D2011%26volume%3D2%26spage%3D103%26epage%3D108%26doi%3D10.4161%2Ftrns.2.3.16172" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit17b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Stellrecht, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. S.</span><span> </span><span class="NLM_article-title">Transcription inhibition as a therapeutic target for cancer</span> <span class="citation_source-journal">Cancers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">4170</span><span class="NLM_x">–</span> <span class="NLM_lpage">4190</span><span class="refDoi"> DOI: 10.3390/cancers3044170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.3390%2Fcancers3044170" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=4170-4190&author=C.+M.+Stellrechtauthor=L.+S.+Chen&title=Transcription+inhibition+as+a+therapeutic+target+for+cancer&doi=10.3390%2Fcancers3044170"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17b&amp;dbid=16384&amp;doi=10.3390%2Fcancers3044170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers3044170%26sid%3Dliteratum%253Aachs%26aulast%3DStellrecht%26aufirst%3DC.%2BM.%26aulast%3DChen%26aufirst%3DL.%2BS.%26atitle%3DTranscription%2520inhibition%2520as%2520a%2520therapeutic%2520target%2520for%2520cancer%26jtitle%3DCancers%26date%3D2011%26volume%3D3%26spage%3D4170%26epage%3D4190%26doi%3D10.3390%2Fcancers3044170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit17c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Kwiatkowski, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahl, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ficarro, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dastur, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amzallag, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramaswamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tesar, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hannett, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMillin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sim, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Look, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitsiades, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weng, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benes, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marto, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Targeting transcription regulation in cancer with a covalent CDK7 inhibitor</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">511</span><span class="NLM_x">, </span> <span class="NLM_fpage">616</span><span class="NLM_x">–</span> <span class="NLM_lpage">620</span><span class="refDoi"> DOI: 10.1038/nature13393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnature13393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=25043025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1ChurnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=511&publication_year=2014&pages=616-620&author=N.+Kwiatkowskiauthor=T.+Zhangauthor=P.+B.+Rahlauthor=B.+J.+Abrahamauthor=J.+Reddyauthor=S.+B.+Ficarroauthor=A.+Dasturauthor=A.+Amzallagauthor=S.+Ramaswamyauthor=B.+Tesarauthor=C.+E.+Jenkinsauthor=N.+M.+Hannettauthor=D.+McMillinauthor=T.+Sandaauthor=T.+Simauthor=N.+D.+Kimauthor=T.+Lookauthor=C.+S.+Mitsiadesauthor=A.+P.+Wengauthor=J.+R.+Brownauthor=C.+H.+Benesauthor=J.+A.+Martoauthor=R.+A.+Youngauthor=N.+S.+Gray&title=Targeting+transcription+regulation+in+cancer+with+a+covalent+CDK7+inhibitor&doi=10.1038%2Fnature13393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17cR"><div class="casContent"><span class="casTitleNuber">17c</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting transcription regulation in cancer with a covalent CDK7 inhibitor</span></div><div class="casAuthors">Kwiatkowski, Nicholas; Zhang, Tinghu; Rahl, Peter B.; Abraham, Brian J.; Reddy, Jessica; Ficarro, Scott B.; Dastur, Anahita; Amzallag, Arnaud; Ramaswamy, Sridhar; Tesar, Bethany; Jenkins, Catherine E.; Hannett, Nancy M.; McMillin, Douglas; Sanda, Takaomi; Sim, Taebo; Kim, Nam Doo; Look, Thomas; Mitsiades, Constantine S.; Weng, Andrew P.; Brown, Jennifer R.; Benes, Cyril H.; Marto, Jarrod A.; Young, Richard A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">511</span>
        (<span class="NLM_cas:issue">7511</span>),
    <span class="NLM_cas:pages">616-620</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tumor oncogenes include transcription factors that co-opt the general transcriptional machinery to sustain the oncogenic state, but direct pharmacol. inhibition of transcription factors has so far proven difficult.  However, the transcriptional machinery contains various enzymic cofactors that can be targeted for the development of new therapeutic candidates, including cyclin-dependent kinases (CDKs).  Here the authors present the discovery and characterization of a covalent CDK7 inhibitor, THZ1, which has the unprecedented ability to target a remote cysteine residue located outside of the canonical kinase domain, providing an unanticipated means of achieving selectivity for CDK7.  Cancer cell-line profiling indicates that a subset of cancer cell lines, including human T-cell acute lymphoblastic leukemia (T-ALL), have exceptional sensitivity to THZ1.  Genome-wide anal. in Jurkat T-ALL cells shows that THZ1 disproportionally affects transcription of RUNX1 and suggests that sensitivity to THZ1 may be due to vulnerability conferred by the RUNX1 super-enhancer and the key role of RUNX1 in the core transcriptional regulatory circuitry of these tumor cells.  Pharmacol. modulation of CDK7 kinase activity may thus provide an approach to identify and treat tumor types that are dependent on transcription for maintenance of the oncogenic state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVD-DeTAQCqrVg90H21EOLACvtfcHk0lgzLf1M4PQMLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1ChurnF&md5=895a28fb536663a31e7558fe79ce628b</span></div><a href="/servlet/linkout?suffix=cit17c&amp;dbid=16384&amp;doi=10.1038%2Fnature13393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13393%26sid%3Dliteratum%253Aachs%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DAbraham%26aufirst%3DB.%2BJ.%26aulast%3DReddy%26aufirst%3DJ.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DDastur%26aufirst%3DA.%26aulast%3DAmzallag%26aufirst%3DA.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DTesar%26aufirst%3DB.%26aulast%3DJenkins%26aufirst%3DC.%2BE.%26aulast%3DHannett%26aufirst%3DN.%2BM.%26aulast%3DMcMillin%26aufirst%3DD.%26aulast%3DSanda%26aufirst%3DT.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DN.%2BD.%26aulast%3DLook%26aufirst%3DT.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26aulast%3DWeng%26aufirst%3DA.%2BP.%26aulast%3DBrown%26aufirst%3DJ.%2BR.%26aulast%3DBenes%26aufirst%3DC.%2BH.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520transcription%2520regulation%2520in%2520cancer%2520with%2520a%2520covalent%2520CDK7%2520inhibitor%26jtitle%3DNature%26date%3D2014%26volume%3D511%26spage%3D616%26epage%3D620%26doi%3D10.1038%2Fnature13393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit17d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Nguyen, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, S.</span><span> </span><span class="NLM_article-title">Dinaciclib (SCH727965) inhibits the unfolded protein response through a CDK1- and 5-dependent mechanism</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">662</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-13-0714</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1158%2F1535-7163.MCT-13-0714" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=24362465" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjslGit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=662-674&author=T.+K.+Nguyenauthor=S.+Grant&title=Dinaciclib+%28SCH727965%29+inhibits+the+unfolded+protein+response+through+a+CDK1-+and+5-dependent+mechanism&doi=10.1158%2F1535-7163.MCT-13-0714"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17dR"><div class="casContent"><span class="casTitleNuber">17d</span><div class="casTitle"><span class="NLM_cas:atitle">Dinaciclib (SCH727965) Inhibits the Unfolded Protein Response through a CDK1- and 5-Dependent Mechanism</span></div><div class="casAuthors">Nguyen, Tri K.; Grant, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">662-674</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Evidence implicating dysregulation of the IRE1/XBP-1s arm of the unfolded protein response (UPR) in cancer pathogenesis (e.g., multiple myeloma) has prompted the development of IRE1 RNase inhibitors.  Here, effects of cyclin-dependent kinase (CDK) inhibitor SCH727965 (dinaciclib) on the IRE1 arm of the UPR were examd. in human leukemia and myeloma cells.  Exposure of cells to extremely low (e.g., nmol/L) concns. of SCH727965, a potent inhibitor of CDKs 1/2/5/9, diminished XBP-1s and Grp78 induction by the endoplasmic reticulum (ER) stress-inducers thapsigargin and tunicamycin, while sharply inducing cell death.  SCH727965, in contrast to IRE1 RNase inhibitors, inhibited the UPR in assocn. with attenuation of XBP-1s nuclear localization and accumulation rather than transcription, translation, or XBP-1 splicing.  Notably, in human leukemia cells, CDK1 and 5 short hairpin RNA (shRNA) knockdown diminished Grp78 and XBP-1s upregulation while increasing thapsigargin lethality, arguing for a functional role for CDK1/5 in activation of the cytoprotective IRE1/XBP-1s arm of the UPR.  In contrast, CDK9 or 2 inhibitors or shRNA knockdown failed to downregulate XBP-1s or Grp78.  Furthermore, IRE1, XBP-1, or Grp78 knockdown significantly increased thapsigargin lethality, as obsd. with CDK1/5 inhibition/knockdown.  Finally, SCH727965 diminished myeloma cell growth in vivo in assocn. with XBP-1s downregulation.  Together, these findings demonstrate that SCH727965 acts at extremely low concns. to attenuate XBP-1s nuclear accumulation and Grp78 upregulation in response to ER stress inducers.  They also highlight a link between specific components of the cell-cycle regulatory app. (e.g., CDK1/5) and the cytoprotective IRE1/XBP-1s/Grp78 arm of the UPR that may be exploited therapeutically in UPR-driven malignancies.  Mol Cancer Ther; 13(3); 662-74. ©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGV9Rbcg389bVg90H21EOLACvtfcHk0lgzLf1M4PQMLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjslGit74%253D&md5=1ead94a6d179e340c952dca738f6c62c</span></div><a href="/servlet/linkout?suffix=cit17d&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-13-0714&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-13-0714%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DT.%2BK.%26aulast%3DGrant%26aufirst%3DS.%26atitle%3DDinaciclib%2520%2528SCH727965%2529%2520inhibits%2520the%2520unfolded%2520protein%2520response%2520through%2520a%2520CDK1-%2520and%25205-dependent%2520mechanism%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2014%26volume%3D13%26spage%3D662%26epage%3D674%26doi%3D10.1158%2F1535-7163.MCT-13-0714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit17e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Lam, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbas, A. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradshaw, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walsby, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pepper, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Targeting RNA transcription and translation in ovarian cancer cells with pharmacological inhibitor CDKI-73</span> <span class="citation_source-journal">Oncotarget</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">7691</span><span class="NLM_x">–</span> <span class="NLM_lpage">7704</span><span class="refDoi"> DOI: 10.18632/oncotarget.2296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.18632%2Foncotarget.2296" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=7691-7704&author=F.+Lamauthor=A.+Y.+Abbasauthor=H.+Shaoauthor=T.+Teoauthor=J.+Adamsauthor=P.+Liauthor=T.+D.+Bradshawauthor=P.+M.+Fischerauthor=E.+Walsbyauthor=C.+Pepperauthor=Y.+Chenauthor=J.+Dingauthor=S.+Wang&title=Targeting+RNA+transcription+and+translation+in+ovarian+cancer+cells+with+pharmacological+inhibitor+CDKI-73&doi=10.18632%2Foncotarget.2296"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17e&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2296%26sid%3Dliteratum%253Aachs%26aulast%3DLam%26aufirst%3DF.%26aulast%3DAbbas%26aufirst%3DA.%2BY.%26aulast%3DShao%26aufirst%3DH.%26aulast%3DTeo%26aufirst%3DT.%26aulast%3DAdams%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DBradshaw%26aufirst%3DT.%2BD.%26aulast%3DFischer%26aufirst%3DP.%2BM.%26aulast%3DWalsby%26aufirst%3DE.%26aulast%3DPepper%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DTargeting%2520RNA%2520transcription%2520and%2520translation%2520in%2520ovarian%2520cancer%2520cells%2520with%2520pharmacological%2520inhibitor%2520CDKI-73%26jtitle%3DOncotarget%26date%3D2014%26volume%3D5%26spage%3D7691%26epage%3D7704%26doi%3D10.18632%2Foncotarget.2296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit17f"><span><span class="NLM_label">(f) </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffiths, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Midgley, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnett, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grabarek, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ingram, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kontopidis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McClue, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McInnes, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLachlan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meades, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mezna, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuart, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheleva, D. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glover, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blake, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, P. M.</span><span> </span><span class="NLM_article-title">Discovery and characterization of 2-anilino-4-(thiazol-5-yl)pyrimidine transcriptional CDK inhibitors as anticancer agents</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">1111</span><span class="NLM_x">–</span> <span class="NLM_lpage">1121</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2010.07.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.chembiol.2010.07.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17f&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=21035734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlKhtLfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=1111-1121&author=S.+Wangauthor=G.+Griffithsauthor=C.+A.+Midgleyauthor=A.+L.+Barnettauthor=M.+Cooperauthor=J.+Grabarekauthor=L.+Ingramauthor=W.+Jacksonauthor=G.+Kontopidisauthor=S.+J.+McClueauthor=C.+McInnesauthor=J.+McLachlanauthor=C.+Meadesauthor=M.+Meznaauthor=I.+Stuartauthor=M.+P.+Thomasauthor=D.+I.+Zhelevaauthor=D.+P.+Laneauthor=R.+C.+Jacksonauthor=D.+M.+Gloverauthor=D.+G.+Blakeauthor=P.+M.+Fischer&title=Discovery+and+characterization+of+2-anilino-4-%28thiazol-5-yl%29pyrimidine+transcriptional+CDK+inhibitors+as+anticancer+agents&doi=10.1016%2Fj.chembiol.2010.07.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17fR"><div class="casContent"><span class="casTitleNuber">17f</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Characterization of 2-Anilino-4- (Thiazol-5-yl)Pyrimidine Transcriptional CDK Inhibitors as Anticancer Agents</span></div><div class="casAuthors">Wang, Shu-Dong; Griffiths, Gary; Midgley, Carol A.; Barnett, Anna L.; Cooper, Michael; Grabarek, Joanna; Ingram, Laura; Jackson, Wayne; Kontopidis, George; McClue, Steven J.; McInnes, Campbell; McLachlan, Janice; Meades, Christopher; Mezna, Mokdad; Stuart, Iain; Thomas, Mark P.; Zheleva, Daniella I.; Lane, David P.; Jackson, Robert C.; Glover, David M.; Blake, David G.; Fischer, Peter M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1111-1121</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The main difficulty in the development of ATP antagonist kinase inhibitors is target specificity, since the ATP-binding motif is present in many proteins.  We introduce a strategy that has allowed us to identify compds. from a kinase inhibitor library that block the cyclin-dependent kinases responsible for regulating transcription, i.e., CDK7 and esp. CDK9.  The screening cascade employs cellular phenotypic assays based on mitotic index and nuclear p53 protein accumulation.  This permitted us to classify compds. into transcriptional, cell cycle, and mitotic inhibitor groups.  We describe the characterization of the transcriptional inhibitor class in terms of kinase inhibition profile, cellular mode of action, and selectivity for transformed cells.  A structural selectivity rationale was used to optimize potency and biopharmaceutical properties and led to the development of a transcriptional inhibitor, 3,4-dimethyl-5-[2-(4-piperazin-1-yl-phenylamino)-pyrimidin-4-yl]-3H-thiazol-2-one, with anticancer activity in animal models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryOG4LTevHibVg90H21EOLACvtfcHk0lj-qH-Whtsw_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlKhtLfL&md5=08aff2706f6eda4ba1038327267a361e</span></div><a href="/servlet/linkout?suffix=cit17f&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.07.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.07.016%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DGriffiths%26aufirst%3DG.%26aulast%3DMidgley%26aufirst%3DC.%2BA.%26aulast%3DBarnett%26aufirst%3DA.%2BL.%26aulast%3DCooper%26aufirst%3DM.%26aulast%3DGrabarek%26aufirst%3DJ.%26aulast%3DIngram%26aufirst%3DL.%26aulast%3DJackson%26aufirst%3DW.%26aulast%3DKontopidis%26aufirst%3DG.%26aulast%3DMcClue%26aufirst%3DS.%2BJ.%26aulast%3DMcInnes%26aufirst%3DC.%26aulast%3DMcLachlan%26aufirst%3DJ.%26aulast%3DMeades%26aufirst%3DC.%26aulast%3DMezna%26aufirst%3DM.%26aulast%3DStuart%26aufirst%3DI.%26aulast%3DThomas%26aufirst%3DM.%2BP.%26aulast%3DZheleva%26aufirst%3DD.%2BI.%26aulast%3DLane%26aufirst%3DD.%2BP.%26aulast%3DJackson%26aufirst%3DR.%2BC.%26aulast%3DGlover%26aufirst%3DD.%2BM.%26aulast%3DBlake%26aufirst%3DD.%2BG.%26aulast%3DFischer%26aufirst%3DP.%2BM.%26atitle%3DDiscovery%2520and%2520characterization%2520of%25202-anilino-4-%2528thiazol-5-yl%2529pyrimidine%2520transcriptional%2520CDK%2520inhibitors%2520as%2520anticancer%2520agents%26jtitle%3DChem.%2520Biol.%26date%3D2010%26volume%3D17%26spage%3D1111%26epage%3D1121%26doi%3D10.1016%2Fj.chembiol.2010.07.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit17g"><span><span class="NLM_label">(g) </span><span class="NLM_contrib-group">Villicana, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zurita, M.</span><span> </span><span class="NLM_article-title">The basal transcription machinery as a target for cancer therapy</span> <span class="citation_source-journal">Cancer Cell Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">18</span><span class="refDoi"> DOI: 10.1186/1475-2867-14-18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17g&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1186%2F1475-2867-14-18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17g&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=24576043" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17g&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjsVartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=18&author=C.+Villicanaauthor=G.+Cruzauthor=M.+Zurita&title=The+basal+transcription+machinery+as+a+target+for+cancer+therapy&doi=10.1186%2F1475-2867-14-18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17gR"><div class="casContent"><span class="casTitleNuber">17g</span><div class="casTitle"><span class="NLM_cas:atitle">The basal transcription machinery as a target for cancer therapy</span></div><div class="casAuthors">Villicana, Claudia; Cruz, Grisel; Zurita, Mario</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell International</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">18/1-18/15, 15 pp.</span>CODEN:
                <span class="NLM_cas:coden">CCIACC</span>;
        ISSN:<span class="NLM_cas:issn">1475-2867</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">General transcription is required for the growth and survival of all living cells.  However, tumor cells require extraordinary levels of transcription, including the transcription of rRNA genes by RNA polymerase I (RNPI) and mRNA by RNA polymerase II (RNPII).  In fact, cancer cells have mutations that directly enhance transcription and are frequently required for cancer transformation.  For example, the recent discovery that MYC enhances the transcription of the majority genes in the genome correlates with the fact that several transcription interfering drugs preferentially kill cancer cells.  In recent years, advances in the mechanistic studies of the basal transcription machinery and the discovery of drugs that interfere with multiple components of transcription are being used to combat cancer.  For example, drugs such as triptolide that targets the general transcription factors TFIIH and JQ1 to inhibit BRD4 are administered to target the high proliferative rate of cancer cells.  Given the importance of finding new strategies to preferentially sensitize tumor cells, this review primarily focuses on several transcription inhibitory drugs to demonstrate that the basal transcription machinery constitutes a potential target for the design of novel cancer drugs.  We highlight the drugs' mechanisms for interfering with tumor cell survival, their importance in cancer treatment and the challenges of clin. application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7aEZQbw_0xLVg90H21EOLACvtfcHk0lj-qH-Whtsw_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjsVartbo%253D&md5=806c757e799d3a318038191c9e1c3520</span></div><a href="/servlet/linkout?suffix=cit17g&amp;dbid=16384&amp;doi=10.1186%2F1475-2867-14-18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1475-2867-14-18%26sid%3Dliteratum%253Aachs%26aulast%3DVillicana%26aufirst%3DC.%26aulast%3DCruz%26aufirst%3DG.%26aulast%3DZurita%26aufirst%3DM.%26atitle%3DThe%2520basal%2520transcription%2520machinery%2520as%2520a%2520target%2520for%2520cancer%2520therapy%26jtitle%3DCancer%2520Cell%2520Int.%26date%3D2014%26volume%3D14%26spage%3D18%26doi%3D10.1186%2F1475-2867-14-18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Solimini, N. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elledge, S. J.</span><span> </span><span class="NLM_article-title">Non-oncogene addiction and the stress phenotype of cancer cells</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">986</span><span class="NLM_x">–</span> <span class="NLM_lpage">988</span><span class="refDoi"> DOI: 10.1016/j.cell.2007.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.cell.2007.09.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2007&pages=986-988&author=N.+L.+Soliminiauthor=J.+Luoauthor=S.+J.+Elledge&title=Non-oncogene+addiction+and+the+stress+phenotype+of+cancer+cells&doi=10.1016%2Fj.cell.2007.09.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DSolimini%26aufirst%3DN.%2BL.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DElledge%26aufirst%3DS.%2BJ.%26atitle%3DNon-oncogene%2520addiction%2520and%2520the%2520stress%2520phenotype%2520of%2520cancer%2520cells%26jtitle%3DCell%26date%3D2007%26volume%3D130%26spage%3D986%26epage%3D988%26doi%3D10.1016%2Fj.cell.2007.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Zou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guettouche, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voellmy, R.</span><span> </span><span class="NLM_article-title">Repression of heat shock transcription factor HSF1 activation by HSP90 (HSP90 complex) that forms a stress-sensitive complex with HSF1</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">471</span><span class="NLM_x">–</span> <span class="NLM_lpage">480</span><span class="refDoi"> DOI: 10.1016/S0092-8674(00)81588-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2FS0092-8674%2800%2981588-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=9727490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADyaK1cXlslOrsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1998&pages=471-480&author=J.+Zouauthor=Y.+Guoauthor=T.+Guettoucheauthor=D.+F.+Smithauthor=R.+Voellmy&title=Repression+of+heat+shock+transcription+factor+HSF1+activation+by+HSP90+%28HSP90+complex%29+that+forms+a+stress-sensitive+complex+with+HSF1&doi=10.1016%2FS0092-8674%2800%2981588-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19aR"><div class="casContent"><span class="casTitleNuber">19a</span><div class="casTitle"><span class="NLM_cas:atitle">Repression of heat shock transcription factor HSF1 activation by HSP90 (HSP90 complex) that forms a stress-sensitive complex with HSF1</span></div><div class="casAuthors">Zou, Jiangying; Guo, Yongle; Guettouche, Toumy; Smith, David F.; Voellmy, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">471-480</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Heat shock and other proteotoxic stresses cause accumulation of nonnative proteins that trigger activation of heat shock protein (Hsp) genes.  A chaperone/Hsp functioning as repressor of heat shock transcription factor (HSF) could make activation of hsp genes dependent on protein unfolding.  In a novel in vitro system, in which human HSF1 can be activated by nonnative protein, heat, and geldanamycin, addn. of Hsp90 inhibits activation.  Redn. of the level of Hsp90 but not of Hsp/c70, Hop, Hip, p23, CyP40, or Hsp40 dramatically activates HSF1.  In vivo, geldanamycin activates HSF1 under conditions in which it is an Hsp90-specific reagent.  Hsp90-contg. HSF1 complex is present in the unstressed cell and dissocs. during stress.  We conclude that Hsp90, by itself and/or assocd. with multichaperone complexes, is a major repressor of HSF1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW_ZsrrM8VFrVg90H21EOLACvtfcHk0lj-qH-Whtsw_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXlslOrsrY%253D&md5=26024c9d4e94c7f46986320fc667fd6f</span></div><a href="/servlet/linkout?suffix=cit19a&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981588-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981588-3%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DGuettouche%26aufirst%3DT.%26aulast%3DSmith%26aufirst%3DD.%2BF.%26aulast%3DVoellmy%26aufirst%3DR.%26atitle%3DRepression%2520of%2520heat%2520shock%2520transcription%2520factor%2520HSF1%2520activation%2520by%2520HSP90%2520%2528HSP90%2520complex%2529%2520that%2520forms%2520a%2520stress-sensitive%2520complex%2520with%2520HSF1%26jtitle%3DCell%26date%3D1998%26volume%3D94%26spage%3D471%26epage%3D480%26doi%3D10.1016%2FS0092-8674%2800%2981588-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit19b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Doubrovin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Che, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serganova, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moroz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solit, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ageyeva, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kochetkova, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pillarsetti, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finn, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blasberg, R. G.</span><span> </span><span class="NLM_article-title">Monitoring the induction of heat shock factor 1/heat shock protein 70 expression following 17-allylamino-17-demethoxygeldanamycin treatment by positron emission tomography and optical reporter gene imaging</span> <span class="citation_source-journal">Mol. Imaging</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">67</span><span class="NLM_x">–</span> <span class="NLM_lpage">76</span><span class="refDoi"> DOI: 10.2310/7290.2011.00028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.2310%2F7290.2011.00028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=22418029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVGrtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=67-76&author=M.+Doubrovinauthor=J.+T.+Cheauthor=I.+Serganovaauthor=E.+Morozauthor=D.+B.+Solitauthor=L.+Ageyevaauthor=T.+Kochetkovaauthor=N.+Pillarsettiauthor=R.+Finnauthor=N.+Rosenauthor=R.+G.+Blasberg&title=Monitoring+the+induction+of+heat+shock+factor+1%2Fheat+shock+protein+70+expression+following+17-allylamino-17-demethoxygeldanamycin+treatment+by+positron+emission+tomography+and+optical+reporter+gene+imaging&doi=10.2310%2F7290.2011.00028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19bR"><div class="casContent"><span class="casTitleNuber">19b</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring the induction of heat shock factor 1/heat shock protein 70 expression following 17-allylamino-demethoxygeldanamycin treatment by positron emission tomography and optical reporter gene imaging</span></div><div class="casAuthors">Doubrovin, Mikhail; Che, Jian T.; Serganova, Inna; Moroz, Ekaterina; Solit, David B.; Ageyeva, Lyudmila; Kochetkova, Tatiana; Pillarsetti, Nagavarakishore; Finn, Ronald; Rosen, Neal; Blasberg, Ronald G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Imaging</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-76</span>CODEN:
                <span class="NLM_cas:coden">MIOMBP</span>;
        ISSN:<span class="NLM_cas:issn">1535-3508</span>.
    
            (<span class="NLM_cas:orgname">Decker Publishing</span>)
        </div><div class="casAbstract">The cell response to proteotoxic cell stresses is mediated primarily through activation of heat shock factor 1 (HSF1).  This transcription factor plays a major role in the regulation of the heat shock proteins (HSPs), including HSP70.  We demonstrate that an [124I]iodide-pQHNIG70 positron emission tomog. (PET) reporter system that includes an inducible HSP70 promoter can be used to image and monitor the activation of the HSF1/HSP70 transcription factor in response to drug treatment (17-allylamino-demethoxygeldanamycin [17-AAG]).  We developed a dual imaging reporter (pQHNIG70) for noninvasive imaging of the heat shock response in cell culture and living animals previously and now study HSF1/HSP70 reporter activation in both cell culture and tumor-bearing animals following exposure to 17-AAG.  17-AAG (10-1,000 nM) induced reporter expression; a 23-fold increase was obsd. by 60 h.  Good correspondence between reporter expression and HSP70 protein levels were obsd.  MicroPET imaging based on [124I]iodide accumulation in pQHNIG70-transduced RG2 xenografts showed a significant 6.2-fold reporter response to 17-AAG, with a corresponding increase in tumor HSP70 and in tumor human sodium iodide symporter and green fluorescent protein reporter proteins.  The HSF1 reporter system can be used to screen anticancer drugs for induction of cytotoxic stress and HSF1 activation both in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0HTURK3faW7Vg90H21EOLACvtfcHk0liLwk7HuoLbgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVGrtLk%253D&md5=3a63cac21e10c6aa9c43df81d0390af9</span></div><a href="/servlet/linkout?suffix=cit19b&amp;dbid=16384&amp;doi=10.2310%2F7290.2011.00028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2310%252F7290.2011.00028%26sid%3Dliteratum%253Aachs%26aulast%3DDoubrovin%26aufirst%3DM.%26aulast%3DChe%26aufirst%3DJ.%2BT.%26aulast%3DSerganova%26aufirst%3DI.%26aulast%3DMoroz%26aufirst%3DE.%26aulast%3DSolit%26aufirst%3DD.%2BB.%26aulast%3DAgeyeva%26aufirst%3DL.%26aulast%3DKochetkova%26aufirst%3DT.%26aulast%3DPillarsetti%26aufirst%3DN.%26aulast%3DFinn%26aufirst%3DR.%26aulast%3DRosen%26aufirst%3DN.%26aulast%3DBlasberg%26aufirst%3DR.%2BG.%26atitle%3DMonitoring%2520the%2520induction%2520of%2520heat%2520shock%2520factor%25201%252Fheat%2520shock%2520protein%252070%2520expression%2520following%252017-allylamino-17-demethoxygeldanamycin%2520treatment%2520by%2520positron%2520emission%2520tomography%2520and%2520optical%2520reporter%2520gene%2520imaging%26jtitle%3DMol.%2520Imaging%26date%3D2012%26volume%3D11%26spage%3D67%26epage%3D76%26doi%3D10.2310%2F7290.2011.00028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit19c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Au, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barber, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, B.</span><span> </span><span class="NLM_article-title">Identification of inhibitors of HSF1 functional activity by high-content target-based screening</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1165</span><span class="NLM_x">–</span> <span class="NLM_lpage">1175</span><span class="refDoi"> DOI: 10.1177/1087057109347472</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1177%2F1087057109347472" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=19820069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVSltrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=1165-1175&author=Q.+Auauthor=Y.+Zhangauthor=J.+R.+Barberauthor=S.+C.+Ngauthor=B.+Zhang&title=Identification+of+inhibitors+of+HSF1+functional+activity+by+high-content+target-based+screening&doi=10.1177%2F1087057109347472"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19cR"><div class="casContent"><span class="casTitleNuber">19c</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of inhibitors of HSF1 functional activity by high-content target-based screening</span></div><div class="casAuthors">Au, Qingyan; Zhang, Yingjia; Barber, Jack R.; Ng, Shi Chung; Zhang, Bin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1165-1175</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Cancer cells are known to experience a high level of stress and may require const. repair for survival and proliferation.  Recent studies showed that inhibition of heat shock factor 1 (HSF1), the key regulator for the stress-activated transcription of heat shock protein (HSP), can reduce the tumorigenic potential of cancer cells.  Such a "nononcogene addiction" phenomenon makes HSF1 an attractive cancer drug target.  Here, the authors report an image-based high-content screening (HCS) assay for HSF1 functional inhibitors.  A heat shock-based methodol. was used to stimulate the stress response followed by quant. measurement of HSF1/HSP70 granules for compd.-induced inhibitory effects.  The authors discovered a small mol. from a compd. library that inhibits HSF1 granule formation substantially in heat-shocked HeLa cells with IC50 at 80 nM.  Electrophoretic mobility shift of HSF1 by this compd. suggested significant inhibition of HSF1 phosphorylation, accompanied by reduced expression levels of HSP70 and HSP90 after heat induction.  Importantly, HeLa cells stably transfected with HSF1 shRNA were more resistant to the compd. treatment under lethal temp. than cells contg. HSF1, further validating an HSF1-dependent mechanism of action.  The HCS assay the authors developed was robust with a Z' factor of 0.65 in a 384-well plate format, providing a valuable method for identifying small-mol. functional inhibitors of HSF1 for potential cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf3LVGRu4f5LVg90H21EOLACvtfcHk0liLwk7HuoLbgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVSltrc%253D&md5=ec73b3c9f122bfe61d12e3562d6f4724</span></div><a href="/servlet/linkout?suffix=cit19c&amp;dbid=16384&amp;doi=10.1177%2F1087057109347472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109347472%26sid%3Dliteratum%253Aachs%26aulast%3DAu%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DBarber%26aufirst%3DJ.%2BR.%26aulast%3DNg%26aufirst%3DS.%2BC.%26aulast%3DZhang%26aufirst%3DB.%26atitle%3DIdentification%2520of%2520inhibitors%2520of%2520HSF1%2520functional%2520activity%2520by%2520high-content%2520target-based%2520screening%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26spage%3D1165%26epage%3D1175%26doi%3D10.1177%2F1087057109347472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">West, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morano, K. A.</span><span> </span><span class="NLM_article-title">Small molecule activators of the heat shock response: chemical properties, molecular targets, and therapeutic promise</span> <span class="citation_source-journal">Chem. Res. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">2036</span><span class="NLM_x">–</span> <span class="NLM_lpage">2053</span><span class="refDoi"> DOI: 10.1021/tx300264x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx300264x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2012&pages=2036-2053&author=J.+D.+Westauthor=Y.+Wangauthor=K.+A.+Morano&title=Small+molecule+activators+of+the+heat+shock+response%3A+chemical+properties%2C+molecular+targets%2C+and+therapeutic+promise&doi=10.1021%2Ftx300264x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Ftx300264x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx300264x%26sid%3Dliteratum%253Aachs%26aulast%3DWest%26aufirst%3DJ.%2BD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMorano%26aufirst%3DK.%2BA.%26atitle%3DSmall%2520molecule%2520activators%2520of%2520the%2520heat%2520shock%2520response%253A%2520chemical%2520properties%252C%2520molecular%2520targets%252C%2520and%2520therapeutic%2520promise%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2012%26volume%3D25%26spage%3D2036%26epage%3D2053%26doi%3D10.1021%2Ftx300264x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Santagata, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendillo, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subramanian, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perley, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roche, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narayan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koeva, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golub, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porco, J. A.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitesell, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindquist, S.</span><span> </span><span class="NLM_article-title">Tight coordination of protein translation and HSF1 activation supports the anabolic malignant state</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">341</span><span class="NLM_x">, </span> <span class="NLM_fpage">1238303</span><span class="refDoi"> DOI: 10.1126/science.1238303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1126%2Fscience.1238303" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=23869022" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A280%3ADC%252BC3sfisF2htg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=1238303&author=S.+Santagataauthor=M.+L.+Mendilloauthor=Y.+Tangauthor=A.+Subramanianauthor=C.+C.+Perleyauthor=S.+P.+Rocheauthor=B.+Wongauthor=R.+Narayanauthor=H.+Kwonauthor=M.+Koevaauthor=A.+Amonauthor=T.+R.+Golubauthor=J.+A.+Porcoauthor=L.+Whitesellauthor=S.+Lindquist&title=Tight+coordination+of+protein+translation+and+HSF1+activation+supports+the+anabolic+malignant+state&doi=10.1126%2Fscience.1238303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21aR"><div class="casContent"><span class="casTitleNuber">21a</span><div class="casTitle"><span class="NLM_cas:atitle">Tight coordination of protein translation and HSF1 activation supports the anabolic malignant state</span></div><div class="casAuthors">Santagata Sandro; Mendillo Marc L; Tang Yun-chi; Subramanian Aravind; Perley Casey C; Roche Stephane P; Wong Bang; Narayan Rajiv; Kwon Hyoungtae; Koeva Martina; Amon Angelika; Golub Todd R; Porco John A Jr; Whitesell Luke; Lindquist Susan</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6143</span>),
    <span class="NLM_cas:pages">1238303</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The ribosome is centrally situated to sense metabolic states, but whether its activity, in turn, coherently rewires transcriptional responses is unknown.  Here, through integrated chemical-genetic analyses, we found that a dominant transcriptional effect of blocking protein translation in cancer cells was inactivation of heat shock factor 1 (HSF1), a multifaceted transcriptional regulator of the heat-shock response and many other cellular processes essential for anabolic metabolism, cellular proliferation, and tumorigenesis.  These analyses linked translational flux to the regulation of HSF1 transcriptional activity and to the modulation of energy metabolism.  Targeting this link with translation initiation inhibitors such as rocaglates deprived cancer cells of their energy and chaperone armamentarium and selectively impaired the proliferation of both malignant and premalignant cells with early-stage oncogenic lesions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQMPgMUk-vccmlSCSgBhwJFfW6udTcc2ebDRD1M2r6q7rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sfisF2htg%253D%253D&md5=8bcd13a849bdc03a186b072d6adec86f</span></div><a href="/servlet/linkout?suffix=cit21a&amp;dbid=16384&amp;doi=10.1126%2Fscience.1238303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1238303%26sid%3Dliteratum%253Aachs%26aulast%3DSantagata%26aufirst%3DS.%26aulast%3DMendillo%26aufirst%3DM.%2BL.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DPerley%26aufirst%3DC.%2BC.%26aulast%3DRoche%26aufirst%3DS.%2BP.%26aulast%3DWong%26aufirst%3DB.%26aulast%3DNarayan%26aufirst%3DR.%26aulast%3DKwon%26aufirst%3DH.%26aulast%3DKoeva%26aufirst%3DM.%26aulast%3DAmon%26aufirst%3DA.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DWhitesell%26aufirst%3DL.%26aulast%3DLindquist%26aufirst%3DS.%26atitle%3DTight%2520coordination%2520of%2520protein%2520translation%2520and%2520HSF1%2520activation%2520supports%2520the%2520anabolic%2520malignant%2520state%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D1238303%26doi%3D10.1126%2Fscience.1238303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit21b"><span><span class="NLM_label">(b) </span><div class="note"><p class="first last">Triptolide:</p></div><span class="NLM_contrib-group">Westerheide, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawahara, T. L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orton, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morimoto, R. I.</span><span> </span><span class="NLM_article-title">Triptolide, an inhibitor of the human heat shock response that enhances stress-induced cell death</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">9616</span><span class="NLM_x">–</span> <span class="NLM_lpage">9622</span><span class="refDoi"> DOI: 10.1074/jbc.M512044200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1074%2Fjbc.M512044200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=16469748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BD28XjtVSqs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=9616-9622&author=S.+D.+Westerheideauthor=T.+L.+A.+Kawaharaauthor=K.+Ortonauthor=R.+I.+Morimoto&title=Triptolide%2C+an+inhibitor+of+the+human+heat+shock+response+that+enhances+stress-induced+cell+death&doi=10.1074%2Fjbc.M512044200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21bR"><div class="casContent"><span class="casTitleNuber">21b</span><div class="casTitle"><span class="NLM_cas:atitle">Triptolide, an Inhibitor of the Human Heat Shock Response That Enhances Stress-induced Cell Death</span></div><div class="casAuthors">Westerheide, Sandy D.; Kawahara, Tiara L. A.; Orton, Kai; Morimoto, Richard I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">9616-9622</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Mol. chaperones, inducible by heat shock and a variety of other stresses, have crit. roles in protein homeostasis, balancing cell stress with adaptation, survival, and cell death mechanisms.  In transformed cells and tumors, chaperones are frequently overexpressed, with constitutive activation of the heat shock transcription factor HSF1 implicated in tumor formation.  Here, we describe the activity of triptolide, a diterpene triepoxide from the plant Triptergium wilfordii, as an inhibitor of the human heat shock response.  Triptolide treatment of human tissue culture cells prevented the inducible expression of heat shock genes, shown by suppression of an HSP70 promoter-reporter construct and by suppression of endogenous HSP70 gene expression.  Upon examg. the steps in the HSF1 activation pathway, we found that triptolide abrogates the transactivation function of HSF1 without interfering in the early events of trimer formation, hyperphosphorylation, and DNA binding.  The ability of triptolide to inhibit the heat shock response renders these cells sensitive to stress-induced cell death, which may be of great relevance to cancer treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBmhgTU2H_UbVg90H21EOLACvtfcHk0lhXYYhIDUHRIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjtVSqs7g%253D&md5=e0108de875d6bb30843d4f3b7315c46f</span></div><a href="/servlet/linkout?suffix=cit21b&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M512044200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M512044200%26sid%3Dliteratum%253Aachs%26aulast%3DWesterheide%26aufirst%3DS.%2BD.%26aulast%3DKawahara%26aufirst%3DT.%2BL.%2BA.%26aulast%3DOrton%26aufirst%3DK.%26aulast%3DMorimoto%26aufirst%3DR.%2BI.%26atitle%3DTriptolide%252C%2520an%2520inhibitor%2520of%2520the%2520human%2520heat%2520shock%2520response%2520that%2520enhances%2520stress-induced%2520cell%2520death%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D9616%26epage%3D9622%26doi%3D10.1074%2Fjbc.M512044200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit21c"><span><span class="NLM_label">(c) </span><div class="note"><p class="first last">BEZ235 and SNS-032:</p></div><span class="NLM_contrib-group">Acquaviva, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sequeira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bates, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Proia, D. A.</span><span> </span><span class="NLM_article-title">mTOR inhibition potentiates HSP90 inhibitor activity via cessation of HSP synthesis</span> <span class="citation_source-journal">Mol. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">703</span><span class="NLM_x">–</span> <span class="NLM_lpage">713</span><span class="refDoi"> DOI: 10.1158/1541-7786.MCR-13-0605</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1158%2F1541-7786.MCR-13-0605" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=24554781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnslCrsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=703-713&author=J.+Acquavivaauthor=S.+Heauthor=J.+Sangauthor=D.+L.+Smithauthor=M.+Sequeiraauthor=C.+Zhangauthor=R.+C.+Batesauthor=D.+A.+Proia&title=mTOR+inhibition+potentiates+HSP90+inhibitor+activity+via+cessation+of+HSP+synthesis&doi=10.1158%2F1541-7786.MCR-13-0605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21cR"><div class="casContent"><span class="casTitleNuber">21c</span><div class="casTitle"><span class="NLM_cas:atitle">mTOR Inhibition Potentiates HSP90 Inhibitor Activity via Cessation of HSP Synthesis</span></div><div class="casAuthors">Acquaviva, Jaime; He, Suqin; Sang, Jim; Smith, Donald L.; Sequeira, Manuel; Zhang, Chaohua; Bates, Richard C.; Proia, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">703-713</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Because of their pleiotropic effects on crit. oncoproteins, inhibitors of HSP90 represent a promising new class of therapeutic agents for the treatment of human cancer.  However, pharmacol. inactivation of HSP90 subsequently triggers a heat shock response that may mitigate the full therapeutic benefit of these compds.  To overcome this limitation, a clin. feasible method was sought to block HSP synthesis induced by the potent HSP90 inhibitor ganetespib.  An immunoassay screen of 322 late-stage or clin. approved drugs was performed to uncover compds. that could block upregulation of the stress-inducible HSP70 that results as a consequence of HSP90 blockade.  Interestingly, inhibitors of the phosphoinositide 3-kinase (PI3K)/mTOR class counteracted ganetespib-induced HSP70 upregulation at both the gene and protein level by suppressing nuclear translocation of heat shock factor 1 (HSF1), the dominant transcription factor controlling cellular stress responses.  This effect was conserved across multiple tumor types and was found to be regulated, in part, by mTOR-dependent translational activity.  Pretreatment with cycloheximide, PI3K/mTOR inhibitor, or an inhibitor of eIF4E (a translation initiation factor and downstream effector of mTOR) all reduced ganetespib-mediated nuclear HSF1 accumulation, indicating that mTOR blockade confers a neg. regulatory effect on HSF1 activity.  Moreover, combined therapy regimens with mTOR or dual PI3K/mTOR inhibitors potentiated the antitumor efficacy of ganetespib in multiple in vivo models.  Implications: Collectively these data identify a novel strategy to optimize the therapeutic potential of HSP90 inhibitors.  Mol Cancer Res; 12(5); 703-13. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUGLLojl79cLVg90H21EOLACvtfcHk0lhXYYhIDUHRIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnslCrsr0%253D&md5=70701fbc238094847be0503aaf5224fd</span></div><a href="/servlet/linkout?suffix=cit21c&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-13-0605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-13-0605%26sid%3Dliteratum%253Aachs%26aulast%3DAcquaviva%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DSang%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DD.%2BL.%26aulast%3DSequeira%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DBates%26aufirst%3DR.%2BC.%26aulast%3DProia%26aufirst%3DD.%2BA.%26atitle%3DmTOR%2520inhibition%2520potentiates%2520HSP90%2520inhibitor%2520activity%2520via%2520cessation%2520of%2520HSP%2520synthesis%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2014%26volume%3D12%26spage%3D703%26epage%3D713%26doi%3D10.1158%2F1541-7786.MCR-13-0605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit21d"><span><span class="NLM_label">(d) </span><div class="note"><p class="first last">For Dinaciclib and SNS-032 see ref <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">28</a></p></div>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=For+Dinaciclib+and+SNS-032+see+ref+28."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21d&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="note"><p class="first last">The compound library was provided by AstraZeneca, see ref <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">28</a>.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Powers, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valenti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miranda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maloney, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eccles, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span> </span><span class="NLM_article-title">Mode of cell death induced by the HSP90 inhibitor 17-AAG (tanespimycin) is dependent on the expression of pro-apoptotic BAX</span> <span class="citation_source-journal">Oncotarget</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1963</span><span class="NLM_x">–</span> <span class="NLM_lpage">1965</span><span class="refDoi"> DOI: 10.18632/oncotarget.1419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.18632%2Foncotarget.1419" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1963-1965&author=M.+V.+Powersauthor=M.+Valentiauthor=S.+Mirandaauthor=A.+Maloneyauthor=S.+A.+Ecclesauthor=G.+Thomasauthor=P.+A.+Clarkeauthor=P.+Workman&title=Mode+of+cell+death+induced+by+the+HSP90+inhibitor+17-AAG+%28tanespimycin%29+is+dependent+on+the+expression+of+pro-apoptotic+BAX&doi=10.18632%2Foncotarget.1419"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.1419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.1419%26sid%3Dliteratum%253Aachs%26aulast%3DPowers%26aufirst%3DM.%2BV.%26aulast%3DValenti%26aufirst%3DM.%26aulast%3DMiranda%26aufirst%3DS.%26aulast%3DMaloney%26aufirst%3DA.%26aulast%3DEccles%26aufirst%3DS.%2BA.%26aulast%3DThomas%26aufirst%3DG.%26aulast%3DClarke%26aufirst%3DP.%2BA.%26aulast%3DWorkman%26aufirst%3DP.%26atitle%3DMode%2520of%2520cell%2520death%2520induced%2520by%2520the%2520HSP90%2520inhibitor%252017-AAG%2520%2528tanespimycin%2529%2520is%2520dependent%2520on%2520the%2520expression%2520of%2520pro-apoptotic%2520BAX%26jtitle%3DOncotarget%26date%3D2013%26volume%3D4%26spage%3D1963%26epage%3D1965%26doi%3D10.18632%2Foncotarget.1419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Jones, K.; Rye, C.; Chessum, N.; Cheeseman, M.; Pasqua, A. E.; Pike, K. G.; Faulder, P. F.</span><span> </span><span class="NLM_article-title">Fused 1,4-dihydrodioxin derivatives as inhibitors of heat shock transcription factor 1</span>. WO 2015/049535 A1, April 9,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=K.+Jones&author=C.+Rye&author=N.+Chessum&author=M.+Cheeseman&author=A.+E.+Pasqua&author=K.+G.+Pike&author=P.+F.+Faulder&title=Fused+1%2C4-dihydrodioxin+derivatives+as+inhibitors+of+heat+shock+transcription+factor+1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DK.%26atitle%3DFused%25201%252C4-dihydrodioxin%2520derivatives%2520as%2520inhibitors%2520of%2520heat%2520shock%2520transcription%2520factor%25201%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="note"><p class="first last">pIC<sub>50</sub> = −log<sub>10</sub> IC<sub>50</sub> (M). All results are quoted to three significant figures as geometric mean ± standard error of the mean (SEM). The geometric mean and the number of repeats are quoted in the parenthesis.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Hardcastle, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boxall, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomlin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aherne, W.</span><span> </span><span class="NLM_article-title">Solid-phase immunoassays in mechanism-based drug discovery: their application in the development of inhibitors of the molecular chaperone heat-shock protein 90</span> <span class="citation_source-journal">Assay Drug Dev. Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">273</span><span class="NLM_x">–</span> <span class="NLM_lpage">285</span><span class="refDoi"> DOI: 10.1089/adt.2005.3.273</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1089%2Fadt.2005.3.273" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2005&pages=273-285&author=A.+Hardcastleauthor=K.+Boxallauthor=J.+Richardsauthor=P.+Tomlinauthor=S.+Sharpauthor=P.+Clarkeauthor=P.+Workmanauthor=W.+Aherne&title=Solid-phase+immunoassays+in+mechanism-based+drug+discovery%3A+their+application+in+the+development+of+inhibitors+of+the+molecular+chaperone+heat-shock+protein+90&doi=10.1089%2Fadt.2005.3.273"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1089%2Fadt.2005.3.273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2005.3.273%26sid%3Dliteratum%253Aachs%26aulast%3DHardcastle%26aufirst%3DA.%26aulast%3DBoxall%26aufirst%3DK.%26aulast%3DRichards%26aufirst%3DJ.%26aulast%3DTomlin%26aufirst%3DP.%26aulast%3DSharp%26aufirst%3DS.%26aulast%3DClarke%26aufirst%3DP.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DAherne%26aufirst%3DW.%26atitle%3DSolid-phase%2520immunoassays%2520in%2520mechanism-based%2520drug%2520discovery%253A%2520their%2520application%2520in%2520the%2520development%2520of%2520inhibitors%2520of%2520the%2520molecular%2520chaperone%2520heat-shock%2520protein%252090%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2005%26volume%3D3%26spage%3D273%26epage%3D285%26doi%3D10.1089%2Fadt.2005.3.273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Laitem, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaborowska, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isa, N. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kufs, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dienstbier, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, S.</span><span> </span><span class="NLM_article-title">CDK9 inhibitors define elongation checkpoints at both ends of RNA polymerase II-transcribed genes</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">396</span><span class="NLM_x">–</span> <span class="NLM_lpage">405</span><span class="refDoi"> DOI: 10.1038/nsmb.3000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnsmb.3000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=25849141" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtlKltrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=396-405&author=C.+Laitemauthor=J.+Zaborowskaauthor=N.+F.+Isaauthor=J.+Kufsauthor=M.+Dienstbierauthor=S.+Murphy&title=CDK9+inhibitors+define+elongation+checkpoints+at+both+ends+of+RNA+polymerase+II-transcribed+genes&doi=10.1038%2Fnsmb.3000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">CDK9 inhibitors define elongation checkpoints at both ends of RNA polymerase II-transcribed genes</span></div><div class="casAuthors">Laitem, Clelia; Zaborowska, Justyna; Isa, Nur F.; Kufs, Johann; Dienstbier, Martin; Murphy, Shona</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">396-403</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Transcription through early-elongation checkpoints requires phosphorylation of neg. transcription elongation factors (NTEFs) by the cyclin-dependent kinase (CDK) 9.  Using CDK9 inhibitors and global run-on sequencing (GRO-seq), we have mapped CDK9 inhibitor-sensitive checkpoints genome wide in human cells.  Our data indicate that early-elongation checkpoints are a general feature of RNA polymerase (pol) II-transcribed human genes and occur independently of polymerase stalling.  Pol II that has negotiated the early-elongation checkpoint can elongate in the presence of inhibitors but, remarkably, terminates transcription prematurely close to the terminal polyadenylation (poly(A)) site.  Our anal. has revealed an unexpected poly(A)-assocd. elongation checkpoint, which has major implications for the regulation of gene expression.  Interestingly, the pattern of modification of the C-terminal domain of pol II terminated at this new checkpoint largely mirrors the pattern normally found downstream of the poly(A) site, thus suggesting common mechanisms of termination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDoG6k1kV9nrVg90H21EOLACvtfcHk0lglEjpIqPzPNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtlKltrk%253D&md5=f95ac3fd82fc8d3ce6914567114b0d1f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3000%26sid%3Dliteratum%253Aachs%26aulast%3DLaitem%26aufirst%3DC.%26aulast%3DZaborowska%26aufirst%3DJ.%26aulast%3DIsa%26aufirst%3DN.%2BF.%26aulast%3DKufs%26aufirst%3DJ.%26aulast%3DDienstbier%26aufirst%3DM.%26aulast%3DMurphy%26aufirst%3DS.%26atitle%3DCDK9%2520inhibitors%2520define%2520elongation%2520checkpoints%2520at%2520both%2520ends%2520of%2520RNA%2520polymerase%2520II-transcribed%2520genes%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D396%26epage%3D405%26doi%3D10.1038%2Fnsmb.3000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Rye, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessum, N. E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamont, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pike, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faulder, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demeritt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kemmitt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tucker, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zani, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheeseman, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isaac, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodwin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boros, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raynaud, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayes, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henley, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Billy, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">te Poele, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’ Fee, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foote, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, K.</span><span> </span><span class="NLM_article-title">Discovery of 4,6-disubstituted pyrimidines as potent inhibitors of the heat shock factor (HSF1) stress pathway and CDK9</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1580</span><span class="NLM_x">–</span> <span class="NLM_lpage">1586</span><span class="refDoi"> DOI: 10.1039/C6MD00159A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1039%2FC6MD00159A" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=1580-1586&author=C.+S.+Ryeauthor=N.+E.+A.+Chessumauthor=S.+Lamontauthor=K.+G.+Pikeauthor=P.+Faulderauthor=J.+Demerittauthor=P.+Kemmittauthor=J.+Tuckerauthor=L.+Zaniauthor=M.+D.+Cheesemanauthor=R.+Isaacauthor=L.+Goodwinauthor=J.+Borosauthor=F.+Raynaudauthor=A.+Hayesauthor=A.+Henleyauthor=E.+de+Billyauthor=C.+J.+Lynchauthor=S.+Y.+Sharpauthor=R.+te+Poeleauthor=L.+O%E2%80%99+Feeauthor=K.+M.+Footeauthor=S.+Greenauthor=P.+Workmanauthor=K.+Jones&title=Discovery+of+4%2C6-disubstituted+pyrimidines+as+potent+inhibitors+of+the+heat+shock+factor+%28HSF1%29+stress+pathway+and+CDK9&doi=10.1039%2FC6MD00159A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2FC6MD00159A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00159A%26sid%3Dliteratum%253Aachs%26aulast%3DRye%26aufirst%3DC.%2BS.%26aulast%3DChessum%26aufirst%3DN.%2BE.%2BA.%26aulast%3DLamont%26aufirst%3DS.%26aulast%3DPike%26aufirst%3DK.%2BG.%26aulast%3DFaulder%26aufirst%3DP.%26aulast%3DDemeritt%26aufirst%3DJ.%26aulast%3DKemmitt%26aufirst%3DP.%26aulast%3DTucker%26aufirst%3DJ.%26aulast%3DZani%26aufirst%3DL.%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26aulast%3DIsaac%26aufirst%3DR.%26aulast%3DGoodwin%26aufirst%3DL.%26aulast%3DBoros%26aufirst%3DJ.%26aulast%3DRaynaud%26aufirst%3DF.%26aulast%3DHayes%26aufirst%3DA.%26aulast%3DHenley%26aufirst%3DA.%26aulast%3Dde%2BBilly%26aufirst%3DE.%26aulast%3DLynch%26aufirst%3DC.%2BJ.%26aulast%3DSharp%26aufirst%3DS.%2BY.%26aulast%3Dte%2BPoele%26aufirst%3DR.%26aulast%3DO%25E2%2580%2599%2BFee%26aufirst%3DL.%26aulast%3DFoote%26aufirst%3DK.%2BM.%26aulast%3DGreen%26aufirst%3DS.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DJones%26aufirst%3DK.%26atitle%3DDiscovery%2520of%25204%252C6-disubstituted%2520pyrimidines%2520as%2520potent%2520inhibitors%2520of%2520the%2520heat%2520shock%2520factor%2520%2528HSF1%2529%2520stress%2520pathway%2520and%2520CDK9%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D1580%26epage%3D1586%26doi%3D10.1039%2FC6MD00159A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Guettouche, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boellmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voellmy, R.</span><span> </span><span class="NLM_article-title">Analysis of phosphorylation of human heat shock factor 1 in cells experiencing a stress</span> <span class="citation_source-journal">BMC Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">4</span><span class="refDoi"> DOI: 10.1186/1471-2091-6-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1186%2F1471-2091-6-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=15760475" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A280%3ADC%252BD2M7nvVGjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=4&author=T.+Guettoucheauthor=F.+Boellmannauthor=W.+S.+Laneauthor=R.+Voellmy&title=Analysis+of+phosphorylation+of+human+heat+shock+factor+1+in+cells+experiencing+a+stress&doi=10.1186%2F1471-2091-6-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29aR"><div class="casContent"><span class="casTitleNuber">29a</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of phosphorylation of human heat shock factor 1 in cells experiencing a stress</span></div><div class="casAuthors">Guettouche Toumy; Boellmann Frank; Lane William S; Voellmy Richard</div><div class="citationInfo"><span class="NLM_cas:title">BMC biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">4</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Heat shock factor (HSF/HSF1) not only is the transcription factor primarily responsible for the transcriptional response of cells to physical and chemical stress but also coregulates other important signaling pathways.  The factor mediates the stress-induced expression of heat shock or stress proteins (HSPs).  HSF/HSF1 is inactive in unstressed cells and is activated during stress.  Activation is accompanied by hyperphosphorylation of the factor.  The regulatory importance of this phosphorylation has remained incompletely understood.  Several previous studies on human HSF1 were concerned with phosphorylation on Ser303, Ser307 and Ser363, which phosphorylation appears to be related to factor deactivation subsequent to stress, and one study reported stress-induced phosphorylation of Ser230 contributing to factor activation.  However, no previous study attempted to fully describe the phosphorylation status of an HSF/HSF1 in stressed cells and to systematically identify phosphoresidues involved in factor activation.  The present study reports such an analysis for human HSF1 in heat-stressed cells.  RESULTS:  An alanine scan of all Ser, Thr and Tyr residues of human HSF1 was carried out using a validated transactivation assay, and residues phosphorylated in HSF1 were identified by mass spectrometry and sequencing.  HSF1 activated by heat treatment was phosphorylated on Ser121, Ser230, Ser292, Ser303, Ser307, Ser314, Ser319, Ser326, Ser344, Ser363, Ser419, and Ser444.  Phosphorylation of Ser326 but none of the other Ser residues was found to contribute significantly to activation of the factor by heat stress.  Phosphorylation on Ser326 increased rapidly during heat stress as shown by experiments using a pSer326 phosphopeptide antibody.  Heat stress-induced DNA binding and nuclear translocation of a S326A substitution mutant was not impaired in HSF1-negative cells, but the mutant stimulated HSP70 expression several times less well than wild type factor.  CONCLUSION:  Twelve Ser residues but no Thr or Tyr residues were identified that were phosphorylated in heat-activated HSF1.  Mutagenesis experiments and functional studies suggested that phosphorylation of HSF1 residue Ser326 plays a critical role in the induction of the factor's transcriptional competence by heat stress.  PhosphoSer326 also contributes to activation of HSF1 by chemical stress.  To date, no functional role could be ascribed to any of the other newly identified phosphoSer residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQfx34-qVWLnWhHGZei4OW9fW6udTcc2eYH8t720XAH2rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M7nvVGjsA%253D%253D&md5=b4717d4af602d104e4af57020e402f74</span></div><a href="/servlet/linkout?suffix=cit29a&amp;dbid=16384&amp;doi=10.1186%2F1471-2091-6-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2091-6-4%26sid%3Dliteratum%253Aachs%26aulast%3DGuettouche%26aufirst%3DT.%26aulast%3DBoellmann%26aufirst%3DF.%26aulast%3DLane%26aufirst%3DW.%2BS.%26aulast%3DVoellmy%26aufirst%3DR.%26atitle%3DAnalysis%2520of%2520phosphorylation%2520of%2520human%2520heat%2520shock%2520factor%25201%2520in%2520cells%2520experiencing%2520a%2520stress%26jtitle%3DBMC%2520Biochem.%26date%3D2005%26volume%3D6%26spage%3D4%26doi%3D10.1186%2F1471-2091-6-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit29b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Budzynski, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puustinen, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joutsen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sistonen, L.</span><span> </span><span class="NLM_article-title">Uncoupling stress-inducible phosphorylation of heat shock factor 1 from its activation</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">2530</span><span class="NLM_x">–</span> <span class="NLM_lpage">2540</span><span class="refDoi"> DOI: 10.1128/MCB.00816-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1128%2FMCB.00816-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=25963659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFamurrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=2530-2540&author=M.+A.+Budzynskiauthor=M.+C.+Puustinenauthor=J.+Joutsenauthor=L.+Sistonen&title=Uncoupling+stress-inducible+phosphorylation+of+heat+shock+factor+1+from+its+activation&doi=10.1128%2FMCB.00816-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29bR"><div class="casContent"><span class="casTitleNuber">29b</span><div class="casTitle"><span class="NLM_cas:atitle">Uncoupling stress-inducible phosphorylation of heat shock factor 1 from its activation</span></div><div class="casAuthors">Budzynski, Marek A.; Puustinen, Mikael C.; Joutsen, Jenny; Sistonen, Lea</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2530-2540</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">1098-5549</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">In mammals the stress-inducible expression of genes encoding heat shock proteins is under the control of the heat shock transcription factor 1 (HSF1).  Activation of HSF1 is a multistep process, involving trimerization, acquisition of DNA-binding and transcriptional activities, which coincide with several posttranslational modifications.  Stress-inducible phosphorylation of HSF1, or hyperphosphorylation, which occurs mainly within the regulatory domain (RD), has been proposed as a requirement for HSF-driven transcription and is widely used for assessing HSF1 activation.  Nonetheless, the contribution of hyperphosphorylation to the activity of HSF1 remains unknown.  In this study, we generated a phosphorylation-deficient HSF1 mutant (HSF1Δ∼PRD), where the 15 known phosphorylation sites within the RD were disrupted.  Our results show that the phosphorylation status of the RD does not affect the subcellular localization and DNA-binding activity of HSF1.  Surprisingly, under stress conditions, HSF1Δ∼PRD is a potent transactivator of both endogenous targets and a reporter gene, and HSF1Δ∼PRD has a reduced activation threshold.  Our results provide the first direct evidence for uncoupling stress-inducible phosphorylation of HSF1 from its activation, and we propose that the phosphorylation signature alone is not an appropriate marker for HSF1 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGri3_8rGXXWrrVg90H21EOLACvtfcHk0licANkTC5C6iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFamurrP&md5=baff5411c24b2d2167af35caf17480c5</span></div><a href="/servlet/linkout?suffix=cit29b&amp;dbid=16384&amp;doi=10.1128%2FMCB.00816-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00816-14%26sid%3Dliteratum%253Aachs%26aulast%3DBudzynski%26aufirst%3DM.%2BA.%26aulast%3DPuustinen%26aufirst%3DM.%2BC.%26aulast%3DJoutsen%26aufirst%3DJ.%26aulast%3DSistonen%26aufirst%3DL.%26atitle%3DUncoupling%2520stress-inducible%2520phosphorylation%2520of%2520heat%2520shock%2520factor%25201%2520from%2520its%2520activation%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2015%26volume%3D35%26spage%3D2530%26epage%3D2540%26doi%3D10.1128%2FMCB.00816-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span>KINOMEscan—World’s Largest Kinase Assay Panel; <span class="NLM_publisher-name">DiscoverX Corporation</span>: <span class="NLM_publisher-loc">San Diego, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span><a href="https://www.discoverx.com/services/drug-discovery-development-services/kinase-profiling/kinomescan" class="extLink">https://www.discoverx.com/services/drug-discovery-development-services/kinase-profiling/kinomescan</a> (accessed August 31, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=KINOMEscan%E2%80%94World%E2%80%99s+Largest+Kinase+Assay+Panel%3B+DiscoverX+Corporation%3A+San+Diego%2C+CA%2C+2016%3B+https%3A%2F%2Fwww.discoverx.com%2Fservices%2Fdrug-discovery-development-services%2Fkinase-profiling%2Fkinomescan+%28accessed+August+31%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DKINOMEscan%25E2%2580%2594World%25E2%2580%2599s%2520Largest%2520Kinase%2520Assay%2520Panel%26pub%3DDiscoverX%2520Corporation%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span>SelectScreen Screening and Profiling Services; <span class="NLM_publisher-name">Thermo Fisher Scientific Inc.</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span><a href="http://www.thermofisher.com/uk/en/home/products-and-services/services/custom-services/screening-and-profiling-services/selectscreen-profiling-service.html?icid=fr-dd-1" class="extLink">http://www.thermofisher.com/uk/en/home/products-and-services/services/custom-services/screening-and-profiling-services/selectscreen-profiling-service.html?icid=fr-dd-1</a> (accessed August 31, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=SelectScreen+Screening+and+Profiling+Services%3B+Thermo+Fisher+Scientific+Inc.%2C+2016%3B+http%3A%2F%2Fwww.thermofisher.com%2Fuk%2Fen%2Fhome%2Fproducts-and-services%2Fservices%2Fcustom-services%2Fscreening-and-profiling-services%2Fselectscreen-profiling-service.html%3Ficid%3Dfr-dd-1+%28accessed+August+31%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DSelectScreen%2520Screening%2520and%2520Profiling%2520Services%26pub%3DThermo%2520Fisher%2520Scientific%2520Inc%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span>All BRAF inhibitors were purchased from Selleckchem and were used without further purification, <a href="http://www.selleckchem.com" class="extLink">http://www.selleckchem.com</a> (accessedAugust 31,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>. See <span class="NLM_xref">Supporting Information</span> for details.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=All+BRAF+inhibitors+were+purchased+from+Selleckchem+and+were+used+without+further+purification%2C+http%3A%2F%2Fwww.selleckchem.com+%28accessedAugust+31%2C+2016%29+.+See+Supporting+Information+for+details."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="note"><p class="first last">All BRAF inhibitors were screened in the human ovarian carcinoma cell-line SK-OV-3.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Garnett, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edelman, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heidorn, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenman, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dastur, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lau, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greninger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, I. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soares, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iorio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surdez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milano, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bignell, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tam, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevenson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barthorpe, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lutz, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogera, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawrence, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mclaren-Douglas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitropoulos, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mironenko, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jewitt, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boisvert, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hur, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baselga, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stamenkovic, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelman, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delattre, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saez-Rodriguez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Settleman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Futreal, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haber, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratton, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramaswamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mcdermott, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benes, C. H.</span><span> </span><span class="NLM_article-title">Systematic identification of genomic markers of drug sensitivity in cancer cells</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">483</span><span class="NLM_x">, </span> <span class="NLM_fpage">570</span><span class="NLM_x">–</span> <span class="NLM_lpage">575</span><span class="refDoi"> DOI: 10.1038/nature11005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnature11005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=22460902" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVektL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=483&publication_year=2012&pages=570-575&author=M.+J.+Garnettauthor=E.+J.+Edelmanauthor=S.+J.+Heidornauthor=C.+D.+Greenmanauthor=A.+Dasturauthor=K.+W.+Lauauthor=P.+Greningerauthor=I.+R.+Thompsonauthor=X.+Luoauthor=J.+Soaresauthor=Q.+Liuauthor=F.+Iorioauthor=D.+Surdezauthor=L.+Chenauthor=R.+J.+Milanoauthor=G.+R.+Bignellauthor=A.+T.+Tamauthor=H.+Daviesauthor=J.+A.+Stevensonauthor=S.+Barthorpeauthor=S.+R.+Lutzauthor=F.+Kogeraauthor=K.+Lawrenceauthor=A.+Mclaren-Douglasauthor=X.+Mitropoulosauthor=T.+Mironenkoauthor=H.+Thiauthor=L.+Richardsonauthor=W.+Zhouauthor=F.+Jewittauthor=T.+Zhangauthor=P.+O%E2%80%99Brienauthor=J.+L.+Boisvertauthor=S.+Priceauthor=W.+Hurauthor=W.+Yangauthor=X.+Dengauthor=A.+Butlerauthor=H.+G.+Choiauthor=J.+W.+Changauthor=J.+Baselgaauthor=I.+Stamenkovicauthor=J.+A.+Engelmanauthor=S.+V.+Sharmaauthor=O.+Delattreauthor=J.+Saez-Rodriguezauthor=N.+S.+Grayauthor=J.+Settlemanauthor=P.+A.+Futrealauthor=D.+A.+Haberauthor=M.+R.+Strattonauthor=S.+Ramaswamyauthor=U.+Mcdermottauthor=C.+H.+Benes&title=Systematic+identification+of+genomic+markers+of+drug+sensitivity+in+cancer+cells&doi=10.1038%2Fnature11005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic identification of genomic markers of drug sensitivity in cancer cells</span></div><div class="casAuthors">Garnett, Mathew J.; Edelman, Elena J.; Heidorn, Sonja J.; Greenman, Chris D.; Dastur, Anahita; Lau, King Wai; Greninger, Patricia; Thompson, I. Richard; Luo, Xi; Soares, Jorge; Liu, Qingsong; Iorio, Francesco; Surdez, Didier; Chen, Li; Milano, Randy J.; Bignell, Graham R.; Tam, Ah T.; Davies, Helen; Stevenson, Jesse A.; Barthorpe, Syd; Lutz, Stephen R.; Kogera, Fiona; Lawrence, Karl; McLaren-Douglas, Anne; Mitropoulos, Xeni; Mironenko, Tatiana; Thi, Helen; Richardson, Laura; Zhou, Wenjun; Jewitt, Frances; Zhang, Tinghu; O'Brien, Patrick; Boisvert, Jessica L.; Price, Stacey; Hur, Wooyoung; Yang, Wanjuan; Deng, Xianming; Butler, Adam; Choi, Hwan Geun; Chang, Jae Won; Baselga, Jose; Stamenkovic, Ivan; Engelman, Jeffrey A.; Sharma, Sreenath V.; Delattre, Olivier; Saez-Rodriguez, Julio; Gray, Nathanael S.; Settleman, Jeffrey; Futreal, P. Andrew; Haber, Daniel A.; Stratton, Michael R.; Ramaswamy, Sridhar; McDermott, Ultan; Benes, Cyril H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">483</span>
        (<span class="NLM_cas:issue">7391</span>),
    <span class="NLM_cas:pages">570-575</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Clin. responses to anticancer therapies are often restricted to a subset of patients.  In some cases, mutated cancer genes are potent biomarkers for responses to targeted agents.  Here, to uncover new biomarkers of sensitivity and resistance to cancer therapeutics, we screened a panel of several hundred cancer cell lines-which represent much of the tissue-type and genetic diversity of human cancers-with 130 drugs under clin. and preclin. investigation.  In aggregate, we found that mutated cancer genes were assocd. with cellular response to most currently available cancer drugs.  Classic oncogene addiction paradigms were modified by addnl. tissue-specific or expression biomarkers, and some frequently mutated genes were assocd. with sensitivity to a broad range of therapeutic agents.  Unexpected relationships were revealed, including the marked sensitivity of Ewing's sarcoma cells harbouring the EWS (also known as EWSR1)-FLI1 gene translocation to poly(ADP-ribose) polymerase (PARP) inhibitors.  By linking drug activity to the functional complexity of cancer genomes, systematic pharmacogenomic profiling in cancer cell lines provides a powerful biomarker discovery platform to guide rational cancer therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3a_Jr2Bhx9bVg90H21EOLACvtfcHk0licANkTC5C6iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVektL4%253D&md5=540fc631fa43b55d0338fff4e46782b5</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnature11005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11005%26sid%3Dliteratum%253Aachs%26aulast%3DGarnett%26aufirst%3DM.%2BJ.%26aulast%3DEdelman%26aufirst%3DE.%2BJ.%26aulast%3DHeidorn%26aufirst%3DS.%2BJ.%26aulast%3DGreenman%26aufirst%3DC.%2BD.%26aulast%3DDastur%26aufirst%3DA.%26aulast%3DLau%26aufirst%3DK.%2BW.%26aulast%3DGreninger%26aufirst%3DP.%26aulast%3DThompson%26aufirst%3DI.%2BR.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DSoares%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DIorio%26aufirst%3DF.%26aulast%3DSurdez%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DMilano%26aufirst%3DR.%2BJ.%26aulast%3DBignell%26aufirst%3DG.%2BR.%26aulast%3DTam%26aufirst%3DA.%2BT.%26aulast%3DDavies%26aufirst%3DH.%26aulast%3DStevenson%26aufirst%3DJ.%2BA.%26aulast%3DBarthorpe%26aufirst%3DS.%26aulast%3DLutz%26aufirst%3DS.%2BR.%26aulast%3DKogera%26aufirst%3DF.%26aulast%3DLawrence%26aufirst%3DK.%26aulast%3DMclaren-Douglas%26aufirst%3DA.%26aulast%3DMitropoulos%26aufirst%3DX.%26aulast%3DMironenko%26aufirst%3DT.%26aulast%3DThi%26aufirst%3DH.%26aulast%3DRichardson%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DJewitt%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DP.%26aulast%3DBoisvert%26aufirst%3DJ.%2BL.%26aulast%3DPrice%26aufirst%3DS.%26aulast%3DHur%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DButler%26aufirst%3DA.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DChang%26aufirst%3DJ.%2BW.%26aulast%3DBaselga%26aufirst%3DJ.%26aulast%3DStamenkovic%26aufirst%3DI.%26aulast%3DEngelman%26aufirst%3DJ.%2BA.%26aulast%3DSharma%26aufirst%3DS.%2BV.%26aulast%3DDelattre%26aufirst%3DO.%26aulast%3DSaez-Rodriguez%26aufirst%3DJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DFutreal%26aufirst%3DP.%2BA.%26aulast%3DHaber%26aufirst%3DD.%2BA.%26aulast%3DStratton%26aufirst%3DM.%2BR.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DMcdermott%26aufirst%3DU.%26aulast%3DBenes%26aufirst%3DC.%2BH.%26atitle%3DSystematic%2520identification%2520of%2520genomic%2520markers%2520of%2520drug%2520sensitivity%2520in%2520cancer%2520cells%26jtitle%3DNature%26date%3D2012%26volume%3D483%26spage%3D570%26epage%3D575%26doi%3D10.1038%2Fnature11005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35a"><span><div class="note"><p class="first last">ATCC:</p></div><a href="http://www.lgcstandards-atcc.org/products/all/HTB-77.aspx?geo_country=gb" class="extLink">http://www.lgcstandards-atcc.org/products/all/HTB-77.aspx?geo_country=gb</a> (accessed August 31, 2016). For examples of using the SK-OV-3 cell-line in drug discovery, see:</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=ATCC%3Ahttp%3A%2F%2Fwww.lgcstandards-atcc.org%2Fproducts%2Fall%2FHTB-77.aspx%3Fgeo_country%3Dgb+%28accessed+August+31%2C+2016%29.+For+examples+of+using+the+SK-OV-3+cell-line+in+drug+discovery%2C+see%3A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit35a1"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Huang, R. Y-J</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuay, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, T. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asad, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, A. H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, V. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thiery, J. P.</span><span> </span><span class="NLM_article-title">Functional relevance of a six mesenchymal gene signature in epithelial-mesenchymal transition (EMT) reversal by the triple angiokinase inhibitor, nintedanib (BIBF1120)</span> <span class="citation_source-journal">Oncotarget</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">22098</span><span class="NLM_x">–</span> <span class="NLM_lpage">22113</span><span class="refDoi"> DOI: 10.18632/oncotarget.4300</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35a1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.18632%2Foncotarget.4300" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=22098-22113&author=R.+Y-J+Huangauthor=K.+T.+Kuayauthor=T.+Z.+Tanauthor=M.+Asadauthor=H.+M.+Tangauthor=A.+H.+C.+Ngauthor=J.+Yeauthor=V.+Y.+Chungauthor=J.+P.+Thiery&title=Functional+relevance+of+a+six+mesenchymal+gene+signature+in+epithelial-mesenchymal+transition+%28EMT%29+reversal+by+the+triple+angiokinase+inhibitor%2C+nintedanib+%28BIBF1120%29&doi=10.18632%2Foncotarget.4300"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35a1&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.4300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.4300%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DR.%2BY-J%26aulast%3DKuay%26aufirst%3DK.%2BT.%26aulast%3DTan%26aufirst%3DT.%2BZ.%26aulast%3DAsad%26aufirst%3DM.%26aulast%3DTang%26aufirst%3DH.%2BM.%26aulast%3DNg%26aufirst%3DA.%2BH.%2BC.%26aulast%3DYe%26aufirst%3DJ.%26aulast%3DChung%26aufirst%3DV.%2BY.%26aulast%3DThiery%26aufirst%3DJ.%2BP.%26atitle%3DFunctional%2520relevance%2520of%2520a%2520six%2520mesenchymal%2520gene%2520signature%2520in%2520epithelial-mesenchymal%2520transition%2520%2528EMT%2529%2520reversal%2520by%2520the%2520triple%2520angiokinase%2520inhibitor%252C%2520nintedanib%2520%2528BIBF1120%2529%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D22098%26epage%3D22113%26doi%3D10.18632%2Foncotarget.4300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit35b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shan, B.</span><span> </span><span class="NLM_article-title">Reversing drug resistance of cisplatin by HSP90 inhibitors in human ovarian cancer cells</span> <span class="citation_source-journal">Int. J. Clin. Exp. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">6687</span><span class="NLM_x">–</span> <span class="NLM_lpage">6701</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=26221207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsFektbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2015&pages=6687-6701&author=Z.+Zhangauthor=Z.+Xieauthor=G.+Sunauthor=P.+Yangauthor=J.+Liauthor=H.+Yangauthor=S.+Xiaoauthor=Y.+Liuauthor=H.+Qiuauthor=L.+Qinauthor=C.+Zhangauthor=F.+Zhangauthor=B.+Shan&title=Reversing+drug+resistance+of+cisplatin+by+HSP90+inhibitors+in+human+ovarian+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35bR"><div class="casContent"><span class="casTitleNuber">35b</span><div class="casTitle"><span class="NLM_cas:atitle">Reversing drug resistance of cisplatin by hsp90 inhibitors in human ovarian cancer cells</span></div><div class="casAuthors">Zhang, Zhengmao; Xie, Zhen; Sun, Guangyu; Yang, Pingfang; Li, Jia; Yang, Hongfang; Xiao, Shuang; Liu, Yang; Qiu, Hongbing; Qin, Lijun; Zhang, Chao; Zhang, Fenghua; Shan, Baoen</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Clinical and Experimental Medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">6687-6701</span>CODEN:
                <span class="NLM_cas:coden">IJCEFA</span>;
        ISSN:<span class="NLM_cas:issn">1940-5901</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">Objective: To investigate the mechanisms for reversing drug resistance of cisplatin (DDP) by Hsp90 inhibitors (geldanamycin (GA), 17-AAG, 17-DMAG) in human ovarian cancer.  Methods: Cell proliferation rate in DDP resistant human ovarian cancer cell line SKOV3/DDP and its parent cell line SKOV3 after treatment with Hsp90 inhibitors and/or DDP were tested by MTT assay, and the reversing fold (RF) of DDP by Hsp90 inhibitors was calcd.  Cell cycle and cell apoptosis status after treatment were analyzed by flow cytometry.  The expression of multiple drug resistance related genes was analyzed by RT-PCR and Western-blot.  Results: All three tested Hsp90 inhibitors synergistically inhibited the cell proliferation of SKOV3 with DDP and enhanced the sensitivity of SKOV3/DDP cells to DDP.  The RF of DDP by Hsp90 inhibitors were all more than two fold.  GA caused cell cycle arrest in G2/M phasein SKOV3 cells. 17-AAG increased cell apoptosis but did not change cell cycle in SKOV3/DDP cells.  The mRNA and protein expression levels of various drug resistant related genes including LRP, GST-π, p53, bcl-2, survivin, ERCC1, XRCC1, BRCA1 and BRCA2 were more dramatically altered by Hsp90 inhibitors and DDP in combination compared to Hsp90 inhibitors or DDP treatment alone.  Conclusions: Exposure of SKOV3/DDP cells to Hsp90 inhibitors and DDP in combination results in synergistic cytotoxic and pro-apoptotic effects.  Hsp90 inhibitors reverse the drug resistance of SKOV3/DDP cells to DDP by modifying the expression of multiple drug resistance related genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMXw59WByT0LVg90H21EOLACvtfcHk0ljpDQWBSZLmcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsFektbs%253D&md5=e49594173796f33dc082132a29ebd90b</span></div><a href="/servlet/linkout?suffix=cit35b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DXie%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DXiao%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DQiu%26aufirst%3DH.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DShan%26aufirst%3DB.%26atitle%3DReversing%2520drug%2520resistance%2520of%2520cisplatin%2520by%2520HSP90%2520inhibitors%2520in%2520human%2520ovarian%2520cancer%2520cells%26jtitle%3DInt.%2520J.%2520Clin.%2520Exp.%2520Med.%26date%3D2015%26volume%3D8%26spage%3D6687%26epage%3D6701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit35c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Wu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, L.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, M.-J.</span><span> </span><span class="NLM_article-title">The inhibitory effect of doxycycline on cisplatin-sensitive and -resistant epithelial ovarian cancer</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e89841</span><span class="refDoi"> DOI: 10.1371/journal.pone.0089841</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1371%2Fjournal.pone.0089841" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e89841&author=W.+Wuauthor=L.-H.+Yuauthor=B.+Maauthor=M.-J.+Xu&title=The+inhibitory+effect+of+doxycycline+on+cisplatin-sensitive+and+-resistant+epithelial+ovarian+cancer&doi=10.1371%2Fjournal.pone.0089841"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35c&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0089841&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0089841%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DL.-H.%26aulast%3DMa%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DM.-J.%26atitle%3DThe%2520inhibitory%2520effect%2520of%2520doxycycline%2520on%2520cisplatin-sensitive%2520and%2520-resistant%2520epithelial%2520ovarian%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De89841%26doi%3D10.1371%2Fjournal.pone.0089841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit35d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Fekete, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santiskulvong, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorigo, O.</span><span> </span><span class="NLM_article-title">Effect of PI3K/AKT pathway inhibition-mediated G1 arrest on chemosensitization in ovarian cancer cells</span> <span class="citation_source-journal">Anticancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">445</span><span class="NLM_x">–</span> <span class="NLM_lpage">452</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=22287731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC38XjsF2ltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=445-452&author=M.+Feketeauthor=C.+Santiskulvongauthor=C.+Engauthor=O.+Dorigo&title=Effect+of+PI3K%2FAKT+pathway+inhibition-mediated+G1+arrest+on+chemosensitization+in+ovarian+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35dR"><div class="casContent"><span class="casTitleNuber">35d</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of PI3K/Akt pathway inhibition-mediated G1 arrest on chemosensitization in ovarian cancer cells</span></div><div class="casAuthors">Fekete, Mirela; Santiskulvong, Chintda; Eng, Carol; Dorigo, Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Anticancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">445-452</span>CODEN:
                <span class="NLM_cas:coden">ANTRD4</span>;
        ISSN:<span class="NLM_cas:issn">0250-7005</span>.
    
            (<span class="NLM_cas:orgname">International Institute of Anticancer Research</span>)
        </div><div class="casAbstract">Background: Pharmacol. inhibition of the phosphoinositide 3-kinase (PI3K)/Akt pathway prevents G1 cell cycle progression into S, resulting in G1 accumulation.  The hypothesis that this arrest might neg. impact on chemotherapeutic agents primarily effective in S, G2 or M-phase was investigated.  Materials and Methods: Inhibition of PI3K/Akt pathway signaling via LY294002 and Akti-1/2 was demonstrated by immunoblotting.  Cell cycle progression was detd. by flow cytometric anal.  Cell proliferation was assayed using the XTT cell viability assay.  The Chou and Talalay median effect principal was used to evaluate drug interaction.  Results: In SKOV3 and IGROV1 human ovarian cancer cells, LY294002 and Akti-1/2 increased the percentage of cells in G1 and reversed the cell cycle effects of cisplatin, paclitaxel, gemcitabine and topotecan.  Pathway blockade synergistically enhanced the cytotoxicity of cisplatin and paclitaxel, but antagonized gemcitabine and topotecan effects.  Conclusion: Pharmacol. PI3K/Akt inhibition antagonizes the efficacy of chemotherapeutic agents primarily effective in the S or G2-phase of the cell cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwbwYDK0hUlLVg90H21EOLACvtfcHk0lhJ5TWMWyMWYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjsF2ltLg%253D&md5=d3e97952622e3271ddf47f02ea052c14</span></div><a href="/servlet/linkout?suffix=cit35d&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFekete%26aufirst%3DM.%26aulast%3DSantiskulvong%26aufirst%3DC.%26aulast%3DEng%26aufirst%3DC.%26aulast%3DDorigo%26aufirst%3DO.%26atitle%3DEffect%2520of%2520PI3K%252FAKT%2520pathway%2520inhibition-mediated%2520G1%2520arrest%2520on%2520chemosensitization%2520in%2520ovarian%2520cancer%2520cells%26jtitle%3DAnticancer%2520Res.%26date%3D2012%26volume%3D32%26spage%3D445%26epage%3D452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit35e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Tan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanizaki, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Z. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aref, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ercan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liao, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelletti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hur, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sim, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaudet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbie, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeh, J. R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yun, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammerman, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohammadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Development of covalent inhibitors that can overcome resistance to first-generation FGFR kinase inhibitors</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">E4869</span><span class="NLM_x">–</span> <span class="NLM_lpage">E4877</span><span class="refDoi"> DOI: 10.1073/pnas.1403438111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1073%2Fpnas.1403438111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=25349422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWmsL%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=E4869-E4877&author=L.+Tanauthor=J.+Wangauthor=J.+Tanizakiauthor=Z.+F.+Huangauthor=A.+R.+Arefauthor=M.+Rusanauthor=S.+J.+Zhuauthor=Y.+Y.+Zhangauthor=D.+Ercanauthor=R.+G.+Liaoauthor=M.+Capellettiauthor=W.+J.+Zhouauthor=W.+Hurauthor=N.+Kimauthor=T.+Simauthor=S.+Gaudetauthor=D.+A.+Barbieauthor=J.+R.+J.+Yehauthor=C.+H.+Yunauthor=P.+S.+Hammermanauthor=M.+Mohammadiauthor=P.+A.+Janneauthor=N.+S.+Gray&title=Development+of+covalent+inhibitors+that+can+overcome+resistance+to+first-generation+FGFR+kinase+inhibitors&doi=10.1073%2Fpnas.1403438111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35eR"><div class="casContent"><span class="casTitleNuber">35e</span><div class="casTitle"><span class="NLM_cas:atitle">Development of covalent inhibitors that can overcome resistance to first-generation FGFR kinase inhibitors</span></div><div class="casAuthors">Tan, Li; Wang, Jun; Tanizaki, Junko; Huang, Zhifeng; Aref, Amir R.; Rusan, Maria; Zhu, Su-Jie; Zhang, Yiyun; Ercan, Dalia; Liao, Rachel G.; Capelletti, Marzia; Zhou, Wenjun; Hur, Wooyoung; Kim, NamDoo; Sim, Taebo; Gaudet, Suzanne; Barbie, David A.; Yeh, Jing-Ruey Joanna; Yun, Cai-Hong; Hammerman, Peter S.; Mohammadi, Moosa; Janne, Pasi A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">E4869-E4877</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The human FGF receptors (FGFRs) play crit. roles in various human cancers, and several FGFR inhibitors are currently under clin. investigation.  Resistance usually results from selection for mutant kinases that are impervious to the action of the drug or from up-regulation of compensatory signaling pathways.  Preclin. studies have demonstrated that resistance to FGFR inhibitors can be acquired through mutations in the FGFR gatekeeper residue, as clin. obsd. for FGFR4 in embryonal rhabdomyosarcoma and neuroendocrine breast carcinomas.  Here the authors report on the use of a structure-based drug design to develop two selective, next-generation covalent FGFR inhibitors, the FGFR irreversible inhibitors I and II.  To our knowledge, I and II are the first inhibitors that can potently inhibit the proliferation of cells dependent upon the gatekeeper mutants of FGFR1 or FGFR2, which confer resistance to first-generation clin. FGFR inhibitors such as NVP-BGJ398 and AZD4547.  Because of the conformational flexibility of the reactive acrylamide substituent, II has the unprecedented ability to inhibit both the EGF receptor (EGFR) and FGFR covalently by targeting two distinct cysteine residues.  The authors report the cocrystal structure of FGFR4 with I, which unexpectedly exhibits a "DFG-out" covalent binding mode.  The structural basis for dual FGFR and EGFR targeting by FIIN3 also is illustrated by crystal structures of II bound with FGFR4 V550L and EGFR L858R.  These results have important implications for the design of covalent FGFR inhibitors that can overcome clin. resistance and provide the first example, to the knowledge, of a kinase inhibitor that covalently targets cysteines located in different positions within the ATP-binding pocket.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmCbTDKPA32LVg90H21EOLACvtfcHk0lhJ5TWMWyMWYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWmsL%252FP&md5=fd4a7eac1e0c17e7c9f2651dd2d76ac7</span></div><a href="/servlet/linkout?suffix=cit35e&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1403438111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1403438111%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DTanizaki%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DZ.%2BF.%26aulast%3DAref%26aufirst%3DA.%2BR.%26aulast%3DRusan%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DS.%2BJ.%26aulast%3DZhang%26aufirst%3DY.%2BY.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DLiao%26aufirst%3DR.%2BG.%26aulast%3DCapelletti%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DW.%2BJ.%26aulast%3DHur%26aufirst%3DW.%26aulast%3DKim%26aufirst%3DN.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DGaudet%26aufirst%3DS.%26aulast%3DBarbie%26aufirst%3DD.%2BA.%26aulast%3DYeh%26aufirst%3DJ.%2BR.%2BJ.%26aulast%3DYun%26aufirst%3DC.%2BH.%26aulast%3DHammerman%26aufirst%3DP.%2BS.%26aulast%3DMohammadi%26aufirst%3DM.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDevelopment%2520of%2520covalent%2520inhibitors%2520that%2520can%2520overcome%2520resistance%2520to%2520first-generation%2520FGFR%2520kinase%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3DE4869%26epage%3DE4877%26doi%3D10.1073%2Fpnas.1403438111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="note"><p class="first last">For examples of SK-OV-3 murine xenograft studies, see:</p></div><div class="NLM_citation" id="rightTab-cit36a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Shaw, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senterman, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dawson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crane, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanderhyden, B. C.</span><span> </span><span class="NLM_article-title">Characterization of intraperitoneal, orthotopic, and metastatic xenograft models of human ovarian cancer</span> <span class="citation_source-journal">Mol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1032</span><span class="NLM_x">–</span> <span class="NLM_lpage">1042</span><span class="refDoi"> DOI: 10.1016/j.ymthe.2004.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.ymthe.2004.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=15564135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVSntrfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=1032-1042&author=T.+J.+Shawauthor=M.+K.+Sentermanauthor=K.+Dawsonauthor=C.+A.+Craneauthor=B.+C.+Vanderhyden&title=Characterization+of+intraperitoneal%2C+orthotopic%2C+and+metastatic+xenograft+models+of+human+ovarian+cancer&doi=10.1016%2Fj.ymthe.2004.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36aR"><div class="casContent"><span class="casTitleNuber">36a</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of intraperitoneal, orthotopic, and metastatic xenograft models of human ovarian cancer</span></div><div class="casAuthors">Shaw, Tanya J.; Senterman, Mary K.; Dawson, Kerri; Crane, Colleen A.; Vanderhyden, Barbara C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1032-1042</span>CODEN:
                <span class="NLM_cas:coden">MTOHCK</span>;
        ISSN:<span class="NLM_cas:issn">1525-0016</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Improvement of ovarian cancer patient outcome requires well-characterized animal models in which to evaluate novel therapeutics.  Xenograft models are frequently used, but with little discussion of disease histol.  The objectives of this study were to inject 11 ovarian cancer cell lines i.p., and a subset intrabursally (ib; orthotopic), into nude mice and to analyze the resulting pathologies.  Eight of 11 lines injected i.p. formed tumors within 3 mo at variable rates with the following histol. subtype distribution: one endometrioid, one serous, one clear cell, and five undifferentiated.  Only mice injected with A2780-cp cells presented with ovarian-specific metastases (11 of 88), and the survival time of these animals was significantly shorter, which may be attributed to the higher proliferation rate as detd. by Ki67 positivity.  Addnl. anal. of the influence of the ovarian microenvironment on cell characteristics was conducted with ib injection of two cell lines (OVCA 429 and ES-2).  The site of injection did not affect the tumor histol., the effect on proliferation was cell-type dependent, and the tumor take rate (cell survival) was neg. affected for OVCA 429 cells.  The animal models described herein represent histol. distinct models of both early and late stage ovarian cancer useful for evaluation of therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHXUVG2I7lorVg90H21EOLACvtfcHk0lhJ5TWMWyMWYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVSntrfM&md5=56735d8bc8f748afa5c2905ddb923a02</span></div><a href="/servlet/linkout?suffix=cit36a&amp;dbid=16384&amp;doi=10.1016%2Fj.ymthe.2004.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymthe.2004.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DShaw%26aufirst%3DT.%2BJ.%26aulast%3DSenterman%26aufirst%3DM.%2BK.%26aulast%3DDawson%26aufirst%3DK.%26aulast%3DCrane%26aufirst%3DC.%2BA.%26aulast%3DVanderhyden%26aufirst%3DB.%2BC.%26atitle%3DCharacterization%2520of%2520intraperitoneal%252C%2520orthotopic%252C%2520and%2520metastatic%2520xenograft%2520models%2520of%2520human%2520ovarian%2520cancer%26jtitle%3DMol.%2520Ther.%26date%3D2004%26volume%3D10%26spage%3D1032%26epage%3D1042%26doi%3D10.1016%2Fj.ymthe.2004.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit36b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Faratian, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zweemer, A. J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagumo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sims, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muir, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodds, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mullen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Um, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kay, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harrison, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langdon, S. P.</span><span> </span><span class="NLM_article-title">Trastuzumab and pertuzumab produce changes in morphology and estrogen receptor signaling in ovarian cancer xenografts revealing new treatment strategies</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">4451</span><span class="NLM_x">–</span> <span class="NLM_lpage">4461</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-10-2461</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1158%2F1078-0432.CCR-10-2461" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=21571868" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXos1SmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=4451-4461&author=D.+Faratianauthor=A.+J.+M.+Zweemerauthor=Y.+Nagumoauthor=A.+H.+Simsauthor=M.+Muirauthor=M.+Doddsauthor=P.+Mullenauthor=I.+Umauthor=C.+Kayauthor=M.+Hasmannauthor=D.+J.+Harrisonauthor=S.+P.+Langdon&title=Trastuzumab+and+pertuzumab+produce+changes+in+morphology+and+estrogen+receptor+signaling+in+ovarian+cancer+xenografts+revealing+new+treatment+strategies&doi=10.1158%2F1078-0432.CCR-10-2461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36bR"><div class="casContent"><span class="casTitleNuber">36b</span><div class="casTitle"><span class="NLM_cas:atitle">Trastuzumab and Pertuzumab Produce Changes in Morphology and Estrogen Receptor Signaling in Ovarian Cancer Xenografts Revealing New Treatment Strategies</span></div><div class="casAuthors">Faratian, Dana; Zweemer, Annelien J. M.; Nagumo, Yoko; Sims, Andrew H.; Muir, Morwenna; Dodds, Michael; Mullen, Peter; Um, Inhwa; Kay, Charlene; Hasmann, Max; Harrison, David J.; Langdon, Simon P.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4451-4461</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: The aim of this study was to investigate the antitumor effects of HER2-directed combination therapy in ovarian cancer xenograft models to evaluate their potential.  The combinations of trastuzumab and pertuzumab, and trastuzumab and aromatase inhibitor therapy were investigated.  Exptl. Design: The effects of trastuzumab, pertuzumab, and letrozole on growth response, apoptosis, morphol., and gene and protein expression were evaluated in the SKOV3 ovarian cancer cell line xenograft and a panel of five human ovarian xenografts derived directly from clin. specimens.  RESULTS: The combination of HER2-directed antibodies showed enhanced antitumor activity compared with single antibody therapy in the SKOV3 xenograft model.  Apoptosis, morphol., and estrogen-regulated gene expression were modulated by these antibodies in both spatial and temporal manners.  A panel of ovarian cancer xenografts showed differential growth responses to the combination of trastuzumab and pertuzumab.  High HER2 expression and increasing HER3 protein expression on treatment were assocd. with growth response.  In trastuzumab-treated SKOV3 tumors, there was a change in tumor morphol., with a redn. in frequency of estrogen receptor alpha (ERα)-neg. clear cell areas.  Trastuzumab, but not pertuzumab, increased expression of ERα in SKOV3 xenografts when analyzed by quant. immunofluorescence.  ERα and downstream signaling targets were modulated by trastuzumab alone and in combination.  Trastuzumab enhanced the responsiveness of SKOV3 xenografts to letrozole when given in combination.  CONCLUSIONS: These data suggest that trastuzumab in combination with pertuzumab could be an effective approach in high HER2-expressing ovarian cancers and could also enhance sensitivity to endocrine therapy in ERα-pos. ovarian cancer.  Clin Cancer Res; 17(13); 4451-61.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmH-TKtfP9sbVg90H21EOLACvtfcHk0lhAwQh38neTAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXos1SmtLs%253D&md5=8a94e84f7fd91bddb63e17c9efee130b</span></div><a href="/servlet/linkout?suffix=cit36b&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-10-2461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-10-2461%26sid%3Dliteratum%253Aachs%26aulast%3DFaratian%26aufirst%3DD.%26aulast%3DZweemer%26aufirst%3DA.%2BJ.%2BM.%26aulast%3DNagumo%26aufirst%3DY.%26aulast%3DSims%26aufirst%3DA.%2BH.%26aulast%3DMuir%26aufirst%3DM.%26aulast%3DDodds%26aufirst%3DM.%26aulast%3DMullen%26aufirst%3DP.%26aulast%3DUm%26aufirst%3DI.%26aulast%3DKay%26aufirst%3DC.%26aulast%3DHasmann%26aufirst%3DM.%26aulast%3DHarrison%26aufirst%3DD.%2BJ.%26aulast%3DLangdon%26aufirst%3DS.%2BP.%26atitle%3DTrastuzumab%2520and%2520pertuzumab%2520produce%2520changes%2520in%2520morphology%2520and%2520estrogen%2520receptor%2520signaling%2520in%2520ovarian%2520cancer%2520xenografts%2520revealing%2520new%2520treatment%2520strategies%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2011%26volume%3D17%26spage%3D4451%26epage%3D4461%26doi%3D10.1158%2F1078-0432.CCR-10-2461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit36c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Zhang, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bing, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Z. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Q.</span><span> </span><span class="NLM_article-title">Augmented inhibition of angiogenesis by combination of HER2 antibody ChA21 and trastuzumab in human ovarian carcinoma xenograft</span> <span class="citation_source-journal">J. Ovarian Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">20</span><span class="refDoi"> DOI: 10.1186/1757-2215-3-20</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1186%2F1757-2215-3-20" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=20723224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A280%3ADC%252BC3cfitVSqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2010&pages=20&author=A.+L.+Zhangauthor=G.+D.+Shenauthor=T.+Zhaoauthor=G.+H.+Zhangauthor=J.+Liuauthor=L.+H.+Songauthor=W.+Weiauthor=L.+Bingauthor=Z.+S.+Wuauthor=Q.+Wu&title=Augmented+inhibition+of+angiogenesis+by+combination+of+HER2+antibody+ChA21+and+trastuzumab+in+human+ovarian+carcinoma+xenograft&doi=10.1186%2F1757-2215-3-20"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36cR"><div class="casContent"><span class="casTitleNuber">36c</span><div class="casTitle"><span class="NLM_cas:atitle">Augmented inhibition of angiogenesis by combination of HER2 antibody chA21 and trastuzumab in human ovarian carcinoma xenograft</span></div><div class="casAuthors">Zhang Anli; Shen Guodong; Zhao Ting; Zhang Guihong; Liu Jing; Song Lihua; Wei Wei; Bing Ling; Wu Zhengsheng; Wu Qiang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of ovarian research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">20</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  chA21 is a novel tumor-inhibitory antibody which recognized subdomain I of HER2 extracellular domain with an epitope distinct from other HER2 antibodies.  Previously, we demonstrated that chA21 inhibits human ovarian carcinoma cell line SKOV-3 growth in vitro and in vivo study.  In this study, we further investigated the anti-angiogenic efficacy combination of chA21 with trastuzumab in SKOV-3 xenograft model.  METHODS:  Nude mice were s.c. challenged with SKOV-3 cells and received treatment of chA21 alone, trastuzumab alone or both antibodies together twice a week for 21 days.  Tumor volume and microvessel density (MVD) were evaluated.  The effect of chA21 plus trastuzumab treament on vascular endothelial growth factor (VEGF) secretion, endothelial cells proliferation and migration, and the status of HER2 downstream pathway AKT/phosphorylated AKT (pAKT) were evaluated in vitro.  RESULTS:  In vivo study combination of chA21 with trastuzumab resulted in reduce tumor growth and angiogenesis than each monotherapy.  In vitro study, the combination of chA21 with trastuzumab inhibits VEGF secretion, endothelial cells proliferation and migration.  Furthermore, the combination treatment inhibits pAKT expression.  CONCLUSION:  Our findings suggested that the combination of chA21 with trastuzumab can cause augmented inhibition of angiogenesis in SKOV-3 xenograft model.  Inhibition of agniogenesis may through suppression of AKT pathway.  The therapeutic benefits of combination chA21 with trastuzumab warrant further study in an attempt to make the translation into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQyFo7VwIDPg3l8icjszYl0fW6udTcc2eb322uGEoEqHbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cfitVSqug%253D%253D&md5=be2d626dd3ce612db75ba8dc25597223</span></div><a href="/servlet/linkout?suffix=cit36c&amp;dbid=16384&amp;doi=10.1186%2F1757-2215-3-20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1757-2215-3-20%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DA.%2BL.%26aulast%3DShen%26aufirst%3DG.%2BD.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DG.%2BH.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DL.%2BH.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DBing%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DZ.%2BS.%26aulast%3DWu%26aufirst%3DQ.%26atitle%3DAugmented%2520inhibition%2520of%2520angiogenesis%2520by%2520combination%2520of%2520HER2%2520antibody%2520ChA21%2520and%2520trastuzumab%2520in%2520human%2520ovarian%2520carcinoma%2520xenograft%26jtitle%3DJ.%2520Ovarian%2520Res.%26date%3D2010%26volume%3D3%26spage%3D20%26doi%3D10.1186%2F1757-2215-3-20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit36d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Li, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ji, Z.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhuo, G.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, R.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.-R.</span><span> </span><span class="NLM_article-title">Effects of a selective cyclooxygenase-1 inhibitor in SKOV-3 ovarian carcinoma xenograft-bearing mice</span> <span class="citation_source-journal">Med. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">98</span><span class="NLM_x">–</span> <span class="NLM_lpage">104</span><span class="refDoi"> DOI: 10.1007/s12032-009-9179-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1007%2Fs12032-009-9179-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=19235530" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXis1CmtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2010&pages=98-104&author=W.+Liauthor=Z.-L.+Jiauthor=G.-C.+Zhuoauthor=R.-J.+Xuauthor=J.+Wangauthor=H.-R.+Jiang&title=Effects+of+a+selective+cyclooxygenase-1+inhibitor+in+SKOV-3+ovarian+carcinoma+xenograft-bearing+mice&doi=10.1007%2Fs12032-009-9179-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36dR"><div class="casContent"><span class="casTitleNuber">36d</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of a selective cyclooxygenase-1 inhibitor in SKOV-3 ovarian carcinoma xenograft-bearing mice</span></div><div class="casAuthors">Li, Wei; Ji, Zhong-lei; Zhuo, Guang-chao; Xu, Ru-jun; Wang, Jie; Jiang, Hong-ru</div><div class="citationInfo"><span class="NLM_cas:title">Medical Oncology (Totowa, NJ, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">98-104</span>CODEN:
                <span class="NLM_cas:coden">MONCEZ</span>;
        ISSN:<span class="NLM_cas:issn">1357-0560</span>.
    
            (<span class="NLM_cas:orgname">Humana Press Inc.</span>)
        </div><div class="casAbstract">To evaluate the effect of a cyclooxygenase-1 (COX-1) inhibitor, SC-560, on the growth inhibition of s.c. human ovarian SKOV-3 carcinoma and on angiogenesis.  Human ovarian SKOV-3 carcinoma cells xenograft-bearing mice were treated with SC-560, a COX-1-selective inhibitor, 6 mg/kg alone i.g. daily, and i.p. injections of cisplatin 3 mg/kg every other day for 21 days.  Prostaglandin E2 (PGE2) levels were detd. by ELISA.  Microvessel d. (MVD) of ovarian carcinoma was detd. with anti-CD34 as the label by immunohistochem.  In addn., the expression of COX-1 at protein levels in the control group was detected by immunohistochem.  SC-560 reduced the growth of tumors when SKOV-3 cells were xenografted in nude female mice.  The inhibitory rates in SC-560 group and cisplatin group were 47.1% and 51.7%, resp., which is significant statistically compared to that of control group (all, P < 0.05).  In treatment groups, SC-560 significantly reduced intratumor PGE2 levels (P < 0.01).  MVDs in SC-560 group were 35.73 ± 9.87, which are significant statistically compared to that of control group (74.33 ± 9.50) (P < 0.01).  COX-1, not COX-2, protein levels are elevated in tumor tissues.  These findings may implicate COX-1 as a suitable target for the treatment of ovarian cancer and that antiangiogenic therapy can be used to inhibit ovarian cancer growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk6AnKHPpTlbVg90H21EOLACvtfcHk0lhAwQh38neTAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXis1CmtLc%253D&md5=d6bbd61aad82cfd68cdc4b45a0c9a769</span></div><a href="/servlet/linkout?suffix=cit36d&amp;dbid=16384&amp;doi=10.1007%2Fs12032-009-9179-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12032-009-9179-y%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DW.%26aulast%3DJi%26aufirst%3DZ.-L.%26aulast%3DZhuo%26aufirst%3DG.-C.%26aulast%3DXu%26aufirst%3DR.-J.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DH.-R.%26atitle%3DEffects%2520of%2520a%2520selective%2520cyclooxygenase-1%2520inhibitor%2520in%2520SKOV-3%2520ovarian%2520carcinoma%2520xenograft-bearing%2520mice%26jtitle%3DMed.%2520Oncol.%26date%3D2010%26volume%3D27%26spage%3D98%26epage%3D104%26doi%3D10.1007%2Fs12032-009-9179-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit36e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Bauerschlag, D. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schem, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiwari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egberts, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weigel, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalthoff, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jonat, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maass, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meinhold-Heerlein, I.</span><span> </span><span class="NLM_article-title">Sunitinib (SU11248) inhibits growth of human ovarian cancer in xenografted mice</span> <span class="citation_source-journal">Anticancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">3355</span><span class="NLM_x">–</span> <span class="NLM_lpage">3360</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=20944108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlGksbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2010&pages=3355-3360&author=D.+O.+Bauerschlagauthor=C.+Schemauthor=S.+Tiwariauthor=J.+H.+Egbertsauthor=M.+T.+Weigelauthor=H.+Kalthoffauthor=W.+Jonatauthor=N.+Maassauthor=I.+Meinhold-Heerlein&title=Sunitinib+%28SU11248%29+inhibits+growth+of+human+ovarian+cancer+in+xenografted+mice"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36eR"><div class="casContent"><span class="casTitleNuber">36e</span><div class="casTitle"><span class="NLM_cas:atitle">Sunitinib (SU11248) inhibits growth of human ovarian cancer in xenografted mice</span></div><div class="casAuthors">Bauerschlag, Dirk O.; Schem, Christian; Tiwari, Sanjay; Egberts, Jan H.; Weigel, Marion T.; Kalthoff, Holger; Jonat, Walter; Maass, Nicolai; Meinhold-Heerlein, Ivo</div><div class="citationInfo"><span class="NLM_cas:title">Anticancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3355-3360</span>CODEN:
                <span class="NLM_cas:coden">ANTRD4</span>;
        ISSN:<span class="NLM_cas:issn">0250-7005</span>.
    
            (<span class="NLM_cas:orgname">International Institute of Anticancer Research</span>)
        </div><div class="casAbstract">Background: Treatment of ovarian cancer is still challenging esp. in recurrent platinum refractory cases.  Sunitinib is a multi tyrosine kinase inhibitor targeting receptors for vascular endothelial growth factor and platelet-derived growth factor which play a role in tumor angiogenesis.  It has been approved for the treatment of recurrent gastro intestinal stroma tumors and metastatic renal cancer.  Materials and Methods: In this study, sunitinib was tested for its effectiveness as a single agent in an ovarian cancer xenograft mouse model.  Skov3 cells stably expressing firefly luciferase were injected into SCID beige mice.  Mice received either 40 mg/kg bodyweight sunitinib or vehicle control.  Tumor growth was monitored longitudinally by luciferase signal.  Results: Sunitinib significantly reduced tumor growth (p = 0.0052) and peritoneal metastases, and was assocd. with a significantly reduced microvessel d. count (p<0.001).  Conclusion: These results suggest that clin. trials are warranted for the evaluation of sunitinib for treatment of patients with recurrent or advanced ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAP9MVZ5R-JLVg90H21EOLACvtfcHk0lhAwQh38neTAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlGksbvJ&md5=418110b00488e2629b8b30d25f36053f</span></div><a href="/servlet/linkout?suffix=cit36e&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBauerschlag%26aufirst%3DD.%2BO.%26aulast%3DSchem%26aufirst%3DC.%26aulast%3DTiwari%26aufirst%3DS.%26aulast%3DEgberts%26aufirst%3DJ.%2BH.%26aulast%3DWeigel%26aufirst%3DM.%2BT.%26aulast%3DKalthoff%26aufirst%3DH.%26aulast%3DJonat%26aufirst%3DW.%26aulast%3DMaass%26aufirst%3DN.%26aulast%3DMeinhold-Heerlein%26aufirst%3DI.%26atitle%3DSunitinib%2520%2528SU11248%2529%2520inhibits%2520growth%2520of%2520human%2520ovarian%2520cancer%2520in%2520xenografted%2520mice%26jtitle%3DAnticancer%2520Res.%26date%3D2010%26volume%3D30%26spage%3D3355%26epage%3D3360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Obach, R. S.</span><span> </span><span class="NLM_article-title">Prediction of human clearance of twenty-nine drugs from hepatic microsomal intrinsic clearance data: an examination of in vitro half-life approach and nonspecific binding to microsomes</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">1350</span><span class="NLM_x">–</span> <span class="NLM_lpage">1359</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10534321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=1350-1359&author=R.+S.+Obach&title=Prediction+of+human+clearance+of+twenty-nine+drugs+from+hepatic+microsomal+intrinsic+clearance+data%3A+an+examination+of+in+vitro+half-life+approach+and+nonspecific+binding+to+microsomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of human clearance of twenty-nine drugs from hepatic microsomal intrinsic clearance data: an examination of in vitro half-life approach and nonspecific binding to microsomes</span></div><div class="casAuthors">Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1350-1359</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Twenty-nine drugs of disparate structures and physicochem. properties were used in an examn. of the capability of human liver microsomal lability data ("in vitro T1/2" approach) to be useful in the prediction of human clearance.  Addnl., the potential importance of nonspecific binding to microsomes in the in vitro incubation milieu for the accurate prediction of human clearance was investigated.  The compds. examd. demonstrated a wide range of microsomal metabolic labilities with scaled intrinsic clearance values ranging from less than 0.5 mL/min/kg to 189 mL/min/kg.  Microsomal binding was detd. at microsomal protein concns. used in the lability incubations.  For the 29 compds. studied, unbound fractions in microsomes ranged from 0.11 to 1.0.  Generally, basic compds. demonstrated the greatest extent of binding and neutral and acidic compds. the least extent of binding.  In the projection of human clearance values, basic and neutral compds. were well predicted when all binding considerations (blood and microsome) were disregarded, however, including both binding considerations also yielded reasonable predictions.  Including only blood binding yielded very poor projections of human clearance for these two types of compds.  However, for acidic compds., disregarding all binding considerations yielded poor predictions of human clearance.  It was generally most difficult to accurately predict clearance for this class of compds.; however the accuracy was best when all binding considerations were included.  Overall, inclusion of both blood and microsome binding values gave the best agreement between in vivo clearance values and clearance values projected from in vitro intrinsic clearance data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWp2hUTOxyubVg90H21EOLACvtfcHk0lihMlpjsc80-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D&md5=7804cd4dfb131caac45aff9b7a1df901</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DPrediction%2520of%2520human%2520clearance%2520of%2520twenty-nine%2520drugs%2520from%2520hepatic%2520microsomal%2520intrinsic%2520clearance%2520data%253A%2520an%2520examination%2520of%2520in%2520vitro%2520half-life%2520approach%2520and%2520nonspecific%2520binding%2520to%2520microsomes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1999%26volume%3D27%26spage%3D1350%26epage%3D1359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><div class="note"><p class="first last">The in-house HPLC method to measure Log<i>D</i><sub>7.4</sub> is based on:</p></div><span class="NLM_contrib-group">Kerns, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petusky, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kleintop, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huryn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McConnell, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, G.</span><span> </span><span class="NLM_article-title">Pharmaceutical profiling method for lipophilicity and integrity using liquid chromatography-mass spectrometry</span> <span class="citation_source-journal">J. Chromatogr. B: Anal. Technol. Biomed. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">791</span><span class="NLM_x">, </span> <span class="NLM_fpage">381</span><span class="NLM_x">–</span> <span class="NLM_lpage">388</span><span class="refDoi"> DOI: 10.1016/S1570-0232(03)00250-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2FS1570-0232%2803%2900250-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=12798198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkt1Ors7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=791&publication_year=2003&pages=381-388&author=E.+H.+Kernsauthor=L.+Diauthor=S.+Petuskyauthor=T.+Kleintopauthor=D.+Hurynauthor=O.+McConnellauthor=G.+Carter&title=Pharmaceutical+profiling+method+for+lipophilicity+and+integrity+using+liquid+chromatography-mass+spectrometry&doi=10.1016%2FS1570-0232%2803%2900250-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmaceutical profiling method for lipophilicity and integrity using liquid chromatography-mass spectrometry</span></div><div class="casAuthors">Kerns, Edward H.; Di, Li; Petusky, Susan; Kleintop, Teresa; Huryn, Donna; McConnell, Oliver; Carter, Guy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chromatography B: Analytical Technologies in the Biomedical and Life Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">791</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">381-388</span>CODEN:
                <span class="NLM_cas:coden">JCBAAI</span>;
        ISSN:<span class="NLM_cas:issn">1570-0232</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A method for the simultaneous profiling of sample lipophilicity, integrity, and purity was described.  The method is rapid and is applicable to high throughput profiling of pharmaceutical properties in drug discovery.  A short Polaris C18 column is used with a rapid, wide-polarity mobile phase gradient, UV detection, and MS anal.  The lipophilicity of each component is estd. from a calibration curve using six drug or org. compds. and plotting their resp. measured retention time vs. Log D7.4 (literature).  The correlation of Log D7.4 (literature) to Log D7.4 (HPLC) for 60 structurally diverse drugs has a correlation coeff. r2 of 0.89.  The method is applicable to compds. with MW>200 and retention time>1.5 min for rapid, initial pharmaceutical profiling in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1NzfnRCy_z7Vg90H21EOLACvtfcHk0lihMlpjsc80-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkt1Ors7k%253D&md5=8b20a63df17d424f0e2406fee659e8ea</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2FS1570-0232%2803%2900250-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1570-0232%252803%252900250-2%26sid%3Dliteratum%253Aachs%26aulast%3DKerns%26aufirst%3DE.%2BH.%26aulast%3DDi%26aufirst%3DL.%26aulast%3DPetusky%26aufirst%3DS.%26aulast%3DKleintop%26aufirst%3DT.%26aulast%3DHuryn%26aufirst%3DD.%26aulast%3DMcConnell%26aufirst%3DO.%26aulast%3DCarter%26aufirst%3DG.%26atitle%3DPharmaceutical%2520profiling%2520method%2520for%2520lipophilicity%2520and%2520integrity%2520using%2520liquid%2520chromatography-mass%2520spectrometry%26jtitle%3DJ.%2520Chromatogr.%2520B%253A%2520Anal.%2520Technol.%2520Biomed.%2520Life%2520Sci.%26date%3D2003%26volume%3D791%26spage%3D381%26epage%3D388%26doi%3D10.1016%2FS1570-0232%2803%2900250-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="note"><p class="first last">The free-fraction in the cell assay was measured by dialysis using standard techniques, <i>f</i><sub>ua</sub> = 0.47 (90%CI = 0.54–0.42). Free GI<sub>50</sub> = GI<sub>50</sub>·<i>f</i><sub>ua</sub></p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><div class="note"><p class="first last">Hepatic blood flow (HBF) in mouse was assumed to be 90 mL/min/kg.</p></div><span class="NLM_contrib-group">Sanoh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horiguchi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugihara, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotake, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tayama, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohshita, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tateno, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horie, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitamura, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohta, S.</span><span> </span><span class="NLM_article-title">Prediction of in vivo hepatic clearance and half-life of drug candidates in human using chimeric mice with humanized liver</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">322</span><span class="NLM_x">–</span> <span class="NLM_lpage">328</span><span class="refDoi"> DOI: 10.1124/dmd.111.040923</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1124%2Fdmd.111.040923" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=322-328&author=S.+Sanohauthor=A.+Horiguchiauthor=K.+Sugiharaauthor=Y.+Kotakeauthor=Y.+Tayamaauthor=H.+Ohshitaauthor=C.+Tatenoauthor=T.+Horieauthor=S.+Kitamuraauthor=S.+Ohta&title=Prediction+of+in+vivo+hepatic+clearance+and+half-life+of+drug+candidates+in+human+using+chimeric+mice+with+humanized+liver&doi=10.1124%2Fdmd.111.040923"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1124%2Fdmd.111.040923&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.111.040923%26sid%3Dliteratum%253Aachs%26aulast%3DSanoh%26aufirst%3DS.%26aulast%3DHoriguchi%26aufirst%3DA.%26aulast%3DSugihara%26aufirst%3DK.%26aulast%3DKotake%26aufirst%3DY.%26aulast%3DTayama%26aufirst%3DY.%26aulast%3DOhshita%26aufirst%3DH.%26aulast%3DTateno%26aufirst%3DC.%26aulast%3DHorie%26aufirst%3DT.%26aulast%3DKitamura%26aufirst%3DS.%26aulast%3DOhta%26aufirst%3DS.%26atitle%3DPrediction%2520of%2520in%2520vivo%2520hepatic%2520clearance%2520and%2520half-life%2520of%2520drug%2520candidates%2520in%2520human%2520using%2520chimeric%2520mice%2520with%2520humanized%2520liver%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2012%26volume%3D40%26spage%3D322%26epage%3D328%26doi%3D10.1124%2Fdmd.111.040923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Smith, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerns, E. H.</span><span> </span><span class="NLM_article-title">The effect of plasma protein binding on in vivo efficacy: misconceptions in drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">929</span><span class="NLM_x">–</span> <span class="NLM_lpage">939</span><span class="refDoi"> DOI: 10.1038/nrd3287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnrd3287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=21119731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsV2jurfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=929-939&author=D.+A.+Smithauthor=L.+Diauthor=E.+H.+Kerns&title=The+effect+of+plasma+protein+binding+on+in+vivo+efficacy%3A+misconceptions+in+drug+discovery&doi=10.1038%2Fnrd3287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The effect of plasma protein binding on in vivo efficacy: misconceptions in drug discovery</span></div><div class="casAuthors">Smith, Dennis A.; Di, Li; Kerns, Edward H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">929-939</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Data from in vitro plasma protein binding expts. that det. the fraction of protein-bound drug are frequently used in drug discovery to guide structure design and to prioritize compds. for in vivo studies.  However, we consider that these practices are usually misleading, because in vivo efficacy is detd. by the free (unbound) drug concn. surrounding the therapeutic target, not by the free drug fraction.  These practices yield no enhancement of the in vivo free drug concn.  So, decisions based on free drug fraction could result in the wrong compds. being advanced through drug discovery programs.  This Perspective provides guidance on the application of plasma protein binding information in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOa3ippodhGbVg90H21EOLACvtfcHk0lihMlpjsc80-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsV2jurfI&md5=a45d00f68ab44580fa38a6acef49493a</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnrd3287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3287%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DD.%2BA.%26aulast%3DDi%26aufirst%3DL.%26aulast%3DKerns%26aufirst%3DE.%2BH.%26atitle%3DThe%2520effect%2520of%2520plasma%2520protein%2520binding%2520on%2520in%2520vivo%2520efficacy%253A%2520misconceptions%2520in%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26spage%3D929%26epage%3D939%26doi%3D10.1038%2Fnrd3287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="note"><p class="first last">C<sub>av</sub><sup>0–24h</sup> = AUC<sup>0–24h</sup>/24.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Smith, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beaumont, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maurer, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di, Li.</span><span> </span><span class="NLM_article-title">Volume of distribution in drug design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">5691</span><span class="NLM_x">–</span> <span class="NLM_lpage">5698</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00201</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00201" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltFSlsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5691-5698&author=D.+A.+Smithauthor=K.+Beaumontauthor=T.+S.+Maurerauthor=Li.+Di&title=Volume+of+distribution+in+drug+design&doi=10.1021%2Facs.jmedchem.5b00201"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Volume of Distribution in Drug Design</span></div><div class="casAuthors">Smith, Dennis A.; Beaumont, Kevin; Maurer, Tristan S.; Di, Li</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5691-5698</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Vol. of distribution is one of the most important pharmacokinetic properties of a drug candidate.  It is a major determinant of half-life and dosing frequency of a drug.  For a similar log P, a basic mol. will tend to exhibit higher vol. of distribution than a neutral mol.  Acids often exhibit low vols. of distribution.  Although a design strategy against vol. of distribution can be advantageous in achieving desirable dosing regimen, it must be well-directed in order to avoid detrimental effects to other important properties.  Strategies to increase vol. of distribution include adding lipophilicity and introducing basic functional groups in a way that does not increase metabolic clearance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP7QwgPahf4bVg90H21EOLACvtfcHk0liagQZYzpEn7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltFSlsb0%253D&md5=2c7289b3602a8edef2d9ef0e6a82e483</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00201%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DD.%2BA.%26aulast%3DBeaumont%26aufirst%3DK.%26aulast%3DMaurer%26aufirst%3DT.%2BS.%26aulast%3DDi%26aufirst%3DLi.%26atitle%3DVolume%2520of%2520distribution%2520in%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5691%26epage%3D5698%26doi%3D10.1021%2Facs.jmedchem.5b00201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="note"><p class="first last">Assumes linear scaling of free exposure (AUC<sub>u</sub>) between different doses.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="note"><p class="first last">Two mice were dosed at 30 mg/kg po qd for 4 days and were then monitored for the following 3 days. Both mice retained good condition over that period as assessed by %body weight. See <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> for details.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Smith, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houghton, P.</span><span> </span><span class="NLM_article-title">A proposal regarding reporting of in vitro testing results</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2828</span><span class="NLM_x">–</span> <span class="NLM_lpage">2833</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-13-0043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1158%2F1078-0432.CCR-13-0043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=23580781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXos12jtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=2828-2833&author=M.+A.+Smithauthor=P.+Houghton&title=A+proposal+regarding+reporting+of+in+vitro+testing+results&doi=10.1158%2F1078-0432.CCR-13-0043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A Proposal Regarding Reporting of In Vitro Testing Results</span></div><div class="casAuthors">Smith, Malcolm A.; Houghton, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2828-2833</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  The high rate of neg. clin. trials and failed drug development programs calls into question the use of preclin. testing as currently practiced.  An important issue for the in vitro testing of agents that have advanced into the clinic is the use of clin. irrelevant concns. in reports making claims for anticancer activity, as illustrated by publications for sorafenib, vorinostat, and metformin.  For sorafenib, high protein binding leads to a dichotomy between concns. active in the 10% serum conditions commonly used for in vitro testing and concns. active in plasma.  Failure to recognize this distinction leads to inappropriate claims of activity for sorafenib based on the micromolar concns. commonly used for in vitro testing in low serum conditions.  For vorinostat and metformin, results using in vitro concns. higher than those achievable in patients are reported despite the availability of publications describing human pharmacokinetic data for each agent.  We encourage journal editors and reviewers to pay greater attention to clin. relevant concns. when considering reports that include in vitro testing of agents for which human pharmacokinetic data are available.  Steps taken to more carefully scrutinize activity claims based on in vitro results can help direct researchers away from clin. irrelevant lines of research and toward lines of research that are more likely to lead to pos. clin. trials and to improved treatments for patients with cancer.  Clin Cancer Res; 19(11); 2828-33. ©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOC9I1yI2eYrVg90H21EOLACvtfcHk0liagQZYzpEn7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXos12jtLo%253D&md5=0a93995f520c722080947689eef19aee</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-13-0043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-13-0043%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DM.%2BA.%26aulast%3DHoughton%26aufirst%3DP.%26atitle%3DA%2520proposal%2520regarding%2520reporting%2520of%2520in%2520vitro%2520testing%2520results%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2013%26volume%3D19%26spage%3D2828%26epage%3D2833%26doi%3D10.1158%2F1078-0432.CCR-13-0043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="note"><p class="first last">The tumor growth inhibition is defined as %TGI = (((<i>C</i><sub>t</sub> – <i>C</i><sub>0</sub>) – (<i>T</i><sub>t</sub> – <i>T</i><sub>0</sub>))/(<i>C</i><sub>t</sub> – <i>C</i><sub>0</sub>)) × 100; <i>T</i><sub>t</sub> = arithmetic mean volume of treated tumors at time = t; <i>T</i><sub>0</sub> = arithmetic mean volume of treated tumors at time = 0; <i>C</i><sub>t</sub> = arithmetic mean volume of control tumors at time = t; <i>C</i><sub>0</sub> = arithmetic mean volume control tumors at time = 0.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="note"><p class="first last">The <i>p</i>-value refers to the comparison of untransformed arithmetic means of the treated and control arms using an unpaired Student’s <i>t</i> test with Welch’s correction. The value is reported to two significant figures.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">de Billy, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powers, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Workman, P.</span><span> </span><span class="NLM_article-title">Drugging the heat shock factor 1 pathway: exploitation of the critical cancer cell dependence on the guardian of the proteome</span> <span class="citation_source-journal">Cell Cycle</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">3806</span><span class="NLM_x">–</span> <span class="NLM_lpage">3808</span><span class="refDoi"> DOI: 10.4161/cc.8.23.10423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.4161%2Fcc.8.23.10423" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=3806-3808&author=E.+de+Billyauthor=M.+V.+Powersauthor=J.+R.+Smithauthor=P.+Workman&title=Drugging+the+heat+shock+factor+1+pathway%3A+exploitation+of+the+critical+cancer+cell+dependence+on+the+guardian+of+the+proteome&doi=10.4161%2Fcc.8.23.10423"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.4161%2Fcc.8.23.10423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.8.23.10423%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BBilly%26aufirst%3DE.%26aulast%3DPowers%26aufirst%3DM.%2BV.%26aulast%3DSmith%26aufirst%3DJ.%2BR.%26aulast%3DWorkman%26aufirst%3DP.%26atitle%3DDrugging%2520the%2520heat%2520shock%2520factor%25201%2520pathway%253A%2520exploitation%2520of%2520the%2520critical%2520cancer%2520cell%2520dependence%2520on%2520the%2520guardian%2520of%2520the%2520proteome%26jtitle%3DCell%2520Cycle%26date%3D2009%26volume%3D8%26spage%3D3806%26epage%3D3808%26doi%3D10.4161%2Fcc.8.23.10423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span>Diversity Profile—P9; <span class="NLM_publisher-name">Eurofins</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span><a href="http://www.cerep.fr/cerep/users/pages/catalog/profiles/DetailProfile.asp?profile=2121" class="extLink">http://www.cerep.fr/cerep/users/pages/catalog/profiles/DetailProfile.asp?profile=2121</a> (accessed August 31,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Diversity+Profile%E2%80%94P9%3B+Eurofins%2C+2016%3B+http%3A%2F%2Fwww.cerep.fr%2Fcerep%2Fusers%2Fpages%2Fcatalog%2Fprofiles%2FDetailProfile.asp%3Fprofile%3D2121+%28accessed+August+31%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DDiversity%2520Profile%25E2%2580%2594P9%26pub%3DEurofins%26date%3D2016%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51a"><span><span class="NLM_label">(a) </span>Target Identification and Validation; <span class="NLM_publisher-name">Evotec</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span><a href="https://www.evotec.com/uploads/media_library/36/2015-09_DDUp-Services_Target__identification.pdf" class="extLink">https://www.evotec.com/uploads/media_library/36/2015-09_DDUp-Services_Target__identification.pdf</a> (accessed August 31, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Target+Identification+and+Validation%3B+Evotec%2C+2016%3B+https%3A%2F%2Fwww.evotec.com%2Fuploads%2Fmedia_library%2F36%2F2015-09_DDUp-Services_Target__identification.pdf+%28accessed+August+31%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DTarget%2520Identification%2520and%2520Validation%26pub%3DEvotec%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit51b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Sharma, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bairlein, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greff, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kéri, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cox, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daub, H.</span><span> </span><span class="NLM_article-title">Proteomics strategy for quantitative protein interaction profiling in cell extracts</span> <span class="citation_source-journal">Nat. Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">741</span><span class="NLM_x">–</span> <span class="NLM_lpage">744</span><span class="refDoi"> DOI: 10.1038/nmeth.1373</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnmeth.1373" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=19749761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFaju73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2009&pages=741-744&author=K.+Sharmaauthor=C.+Weberauthor=M.+Bairleinauthor=Z.+Greffauthor=G.+K%C3%A9riauthor=J.+Coxauthor=J.+V.+Olsenauthor=H.+Daub&title=Proteomics+strategy+for+quantitative+protein+interaction+profiling+in+cell+extracts&doi=10.1038%2Fnmeth.1373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51bR"><div class="casContent"><span class="casTitleNuber">51b</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomics strategy for quantitative protein interaction profiling in cell extracts</span></div><div class="casAuthors">Sharma, Kirti; Weber, Christoph; Bairlein, Michaela; Greff, Zoltan; Keri, Gyoergy; Cox, Juergen; Olsen, Jesper V.; Daub, Henrik</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">741-744</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The authors report a proteomics strategy to both identify and quantify cellular target protein interactions with externally introduced ligands.  The authors detd. dissocn. consts. for target proteins interacting with the ligand of interest by combining quant. mass spectrometry with a defined set of affinity purifn. expts.  The authors demonstrate the general utility of this methodol. in interaction studies involving small-mol. kinase inhibitors, a tyrosine-phosphorylated peptide and an antibody as affinity ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9I5plLFxQRbVg90H21EOLACvtfcHk0lhOfX-PtKbLOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFaju73O&md5=78732562cac1cc2af0938f5536134098</span></div><a href="/servlet/linkout?suffix=cit51b&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.1373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.1373%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DK.%26aulast%3DWeber%26aufirst%3DC.%26aulast%3DBairlein%26aufirst%3DM.%26aulast%3DGreff%26aufirst%3DZ.%26aulast%3DK%25C3%25A9ri%26aufirst%3DG.%26aulast%3DCox%26aufirst%3DJ.%26aulast%3DOlsen%26aufirst%3DJ.%2BV.%26aulast%3DDaub%26aufirst%3DH.%26atitle%3DProteomics%2520strategy%2520for%2520quantitative%2520protein%2520interaction%2520profiling%2520in%2520cell%2520extracts%26jtitle%3DNat.%2520Methods%26date%3D2009%26volume%3D6%26spage%3D741%26epage%3D744%26doi%3D10.1038%2Fnmeth.1373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Cheng, Y.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prusoff, W. H.</span><span> </span><span class="NLM_article-title">Relationship between the inhibition constant (K<sub>i</sub>) and the concentration of inhibitor which causes 50% inhibition (I<sub>50</sub>) of an enzymatic reaction</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">3099</span><span class="NLM_x">–</span> <span class="NLM_lpage">3108</span><span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=4202581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=Y.-C.+Chengauthor=W.+H.+Prusoff&title=Relationship+between+the+inhibition+constant+%28Ki%29+and+the+concentration+of+inhibitor+which+causes+50%25+inhibition+%28I50%29+of+an+enzymatic+reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Relation between the inhibition constant K1) and the concentration of inhibitor which causes fifty per cent inhibition (I50) of an enzymic reaction</span></div><div class="casAuthors">Cheng, Yung-Chi; Prusoff, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3099-108</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">The Ki and I50 values are equal where the kinetics are non- or uncompetitive, but not where they are competitive.  The relation between the 2 values was analyzed for non- and uncompetitive bi- and monosubstrate reactions and for competitive bisubstrate reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqodFY_8msz57Vg90H21EOLACvtfcHk0lhOfX-PtKbLOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D&md5=fe7175ca300d8f4085df87af37f8656c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.-C.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520inhibition%2520constant%2520%2528Ki%2529%2520and%2520the%2520concentration%2520of%2520inhibitor%2520which%2520causes%252050%2525%2520inhibition%2520%2528I50%2529%2520of%2520an%2520enzymatic%2520reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Trinkle-Mulcahy, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boulon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lam, Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urcia, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boisvert, F.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vandermoere, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrice, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swift, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rothbauer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leonhardt, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamond, A.</span><span> </span><span class="NLM_article-title">Identifying specific protein interaction partners using quantitative mass spectrometry and bead proteomes</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">183</span><span class="NLM_x">, </span> <span class="NLM_fpage">223</span><span class="NLM_x">–</span> <span class="NLM_lpage">239</span><span class="refDoi"> DOI: 10.1083/jcb.200805092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1083%2Fjcb.200805092" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=183&publication_year=2008&pages=223-239&author=L.+Trinkle-Mulcahyauthor=S.+Boulonauthor=Y.+W.+Lamauthor=R.+Urciaauthor=F.-M.+Boisvertauthor=F.+Vandermoereauthor=N.+A.+Morriceauthor=S.+Swiftauthor=U.+Rothbauerauthor=H.+Leonhardtauthor=A.+Lamond&title=Identifying+specific+protein+interaction+partners+using+quantitative+mass+spectrometry+and+bead+proteomes&doi=10.1083%2Fjcb.200805092"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1083%2Fjcb.200805092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.200805092%26sid%3Dliteratum%253Aachs%26aulast%3DTrinkle-Mulcahy%26aufirst%3DL.%26aulast%3DBoulon%26aufirst%3DS.%26aulast%3DLam%26aufirst%3DY.%2BW.%26aulast%3DUrcia%26aufirst%3DR.%26aulast%3DBoisvert%26aufirst%3DF.-M.%26aulast%3DVandermoere%26aufirst%3DF.%26aulast%3DMorrice%26aufirst%3DN.%2BA.%26aulast%3DSwift%26aufirst%3DS.%26aulast%3DRothbauer%26aufirst%3DU.%26aulast%3DLeonhardt%26aufirst%3DH.%26aulast%3DLamond%26aufirst%3DA.%26atitle%3DIdentifying%2520specific%2520protein%2520interaction%2520partners%2520using%2520quantitative%2520mass%2520spectrometry%2520and%2520bead%2520proteomes%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2008%26volume%3D183%26spage%3D223%26epage%3D239%26doi%3D10.1083%2Fjcb.200805092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="note"><p class="first last">In-house data using a biotinylated bisamide derivative in a protein pull-down experiment also identified Pirin as a molecular target of the cellularly active bisamide series (data not shown).</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Wendler, W. M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kremmer, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forster, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winnacker, E. L.</span><span> </span><span class="NLM_article-title">Identification of pirin, a novel highly conserved nuclear protein</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">272</span><span class="NLM_x">, </span> <span class="NLM_fpage">8482</span><span class="NLM_x">–</span> <span class="NLM_lpage">8489</span><span class="refDoi"> DOI: 10.1074/jbc.272.13.8482</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1074%2Fjbc.272.13.8482" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=9079676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADyaK2sXitF2ntbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=8482-8489&author=W.+M.+F.+Wendlerauthor=E.+Kremmerauthor=R.+Forsterauthor=E.+L.+Winnacker&title=Identification+of+pirin%2C+a+novel+highly+conserved+nuclear+protein&doi=10.1074%2Fjbc.272.13.8482"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Pirin, a novel highly conserved nuclear protein</span></div><div class="casAuthors">Wendler, Wolfgang M. F.; Kremmer, Elisabeth; Foerster, Reinhold; Winnacker, Ernst-Ludwig</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">272</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">8482-8489</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">In this article we describe the mol. cloning of Pirin, a novel highly conserved 32-kDa protein consisting of 290 amino acids.  Pirin was isolated by a yeast two-hybrid screen as an interactor of nuclear factor I/CCAAT box transcription factor (NFI/CTF1), which is known to stimulate adenovirus DNA replication and RNA polymerase II-driven transcription.  Pirin mRNA is expressed weakly in all human tissues tested.  About 15% of all Pirin cDNAs contain a short 34-base pair insertion in their 5'-untranslated regions, indicative of alternative splicing processes.  Multiple Pirin transcripts are expressed in skeletal muscle and heart.  Western blots and immunopptns. employing monoclonal anti-Pirin antibodies reveal that Pirin is a nuclear protein.  Moreover, confocal immunofluorescence expts. demonstrate a predominant localization of Pirin within dot-like subnuclear structures.  Homol. searches using the BLAST algorithm indicate the existence of Pirin homologues in mouse and rat.  The N-terminal half of Pirin is significantly conserved between mammals, plants, fungi, and even prokaryotic organisms.  Genomic Southern and Western blots demonstrate the presence of Pirin genes and their expression, resp., in all mammalian cell lines tested.  The expression pattern, the concd. localization is subnuclear structures, and its interaction with NFI/CTF1 in the two-hybrid system classify Pirin as a putative NFI/CTF1 cofactor, which might help to gain new insights in NFI/CTF1 functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogmDofGnosCbVg90H21EOLACvtfcHk0lhOfX-PtKbLOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXitF2ntbo%253D&md5=59a1e6e21a872b8e3e905fba15cf3e16</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1074%2Fjbc.272.13.8482&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.272.13.8482%26sid%3Dliteratum%253Aachs%26aulast%3DWendler%26aufirst%3DW.%2BM.%2BF.%26aulast%3DKremmer%26aufirst%3DE.%26aulast%3DForster%26aufirst%3DR.%26aulast%3DWinnacker%26aufirst%3DE.%2BL.%26atitle%3DIdentification%2520of%2520pirin%252C%2520a%2520novel%2520highly%2520conserved%2520nuclear%2520protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1997%26volume%3D272%26spage%3D8482%26epage%3D8489%26doi%3D10.1074%2Fjbc.272.13.8482" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Dechend, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirano, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lehmann, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heissmeyer, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ansieau, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wulczyn, F. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheidereit, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leutz, A.</span><span> </span><span class="NLM_article-title">The BCL-3 oncoprotein acts as a bridging factor between NF-κB/REL and nuclear co-regulators</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">3316</span><span class="NLM_x">–</span> <span class="NLM_lpage">3323</span><span class="refDoi"> DOI: 10.1038/sj.onc.1202717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fsj.onc.1202717" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10362352" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADyaK1MXjvFyrsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1999&pages=3316-3323&author=R.+Dechendauthor=F.+Hiranoauthor=K.+Lehmannauthor=V.+Heissmeyerauthor=S.+Ansieauauthor=F.+G.+Wulczynauthor=C.+Scheidereitauthor=A.+Leutz&title=The+BCL-3+oncoprotein+acts+as+a+bridging+factor+between+NF-%CE%BAB%2FREL+and+nuclear+co-regulators&doi=10.1038%2Fsj.onc.1202717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">The Bcl-3 oncoprotein acts as a bridging factor between NF-κB/Rel and nuclear co-regulators</span></div><div class="casAuthors">Dechend, Ralf; Hirano, Fuminori; Lehmann, Kerstin; Heissmeyer, Vigo; Ansieau, Stephane; Wulczyn, F. Gregory; Scheidereit, Claus; Leutz, Achim</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3316-3323</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Stockton Press</span>)
        </div><div class="casAbstract">The proto-oncoprotein Bcl-3 is a member of the IκB family and is present predominantly in the nucleus.  To gain insight into specific nuclear functions of Bcl-3 we have isolated proteins that interact with its ankyrin repeat domain.  Using the yeast two-hybrid-system we identified four novel binding partners of Bcl-3 in addn. to NF-κB p50 and p52, previously known to assoc. with Bcl-3.  The novel Bcl-3 interactors Jab1, Pirin, Tip60 and Bard1 are nuclear proteins which also bind to other transcription factors including c-Jun, nuclear factor I (NFI), HIV-1 Tat or the tumor suppressor and PolII holoenzyme component Brca1, resp.  Bcl-3, p50, and either Bard1, Tip60 or Pirin are sequestered into quaternary complexes on NF-κB DNA binding sites, whereas Jab1 enhances p50-Bcl-3-DNA complex formation.  Furthermore, the histone acetylase Tip60 enhances Bcl-3-p50 activated transcription through an NF-κB binding site, indicating that quaternary complexes contg. Bcl-3 interactors modulate NF-κB driven gene expression.  These data implicate Bcl-3 as an adaptor between NF-κB p50/p52 and other transcription regulators and suggest that its gene activation function may at least in part be due to recruitment of the Tip60 histone acetylase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFdiEgJ7YIorVg90H21EOLACvtfcHk0lhrEs5Ki55bcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvFyrsrw%253D&md5=8f7de746b9e70f40703d5581034bb9e8</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1202717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1202717%26sid%3Dliteratum%253Aachs%26aulast%3DDechend%26aufirst%3DR.%26aulast%3DHirano%26aufirst%3DF.%26aulast%3DLehmann%26aufirst%3DK.%26aulast%3DHeissmeyer%26aufirst%3DV.%26aulast%3DAnsieau%26aufirst%3DS.%26aulast%3DWulczyn%26aufirst%3DF.%2BG.%26aulast%3DScheidereit%26aufirst%3DC.%26aulast%3DLeutz%26aufirst%3DA.%26atitle%3DThe%2520BCL-3%2520oncoprotein%2520acts%2520as%2520a%2520bridging%2520factor%2520between%2520NF-%25CE%25BAB%252FREL%2520and%2520nuclear%2520co-regulators%26jtitle%3DOncogene%26date%3D1999%26volume%3D18%26spage%3D3316%26epage%3D3323%26doi%3D10.1038%2Fsj.onc.1202717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rehmani, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Esaki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Serrano, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, A.</span><span> </span><span class="NLM_article-title">Pirin is an iron-dependent redox regulator of NF-κB</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">9722</span><span class="NLM_x">–</span> <span class="NLM_lpage">9727</span><span class="refDoi"> DOI: 10.1073/pnas.1221743110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1073%2Fpnas.1221743110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=23716661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOrt7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=9722-9727&author=F.+Liuauthor=I.+Rehmaniauthor=S.+Esakiauthor=R.+Fuauthor=L.+Chenauthor=V.+de+Serranoauthor=A.+Liu&title=Pirin+is+an+iron-dependent+redox+regulator+of+NF-%CE%BAB&doi=10.1073%2Fpnas.1221743110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Pirin is an iron-dependent redox regulator of NF-κB</span></div><div class="casAuthors">Liu, Fange; Rehmani, Imran; Esaki, Shingo; Fu, Rong; Chen, Lirong; de Serrano, Vesna; Liu, Aimin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9722-9727, S9722/1-S9722/11</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Pirin is a nuclear nonheme Fe protein of unknown function present in all human tissues.  Here the authors describe that pirin may act as a redox sensor for the nuclear factor κB (NF-κB) transcription factor, a crit. mediator of intracellular signaling that has been linked to cellular responses to proinflammatory signals and controls the expression of a vast array of genes involved in immune and stress responses.  Pirin's regulatory effect was tested with several metals and at different oxidns. states, and the authors' spectroscopic results show that only the ferric form of pirin substantially facilitates binding of NF-κB proteins to target κB genes, a finding that suggests that pirin performs a redox-sensing role in NF-κB regulation.  The mol. mechanism of such a metal identity- and redox state-dependent regulation is revealed by the authors' structural studies of pirin.  The ferrous and ferric pirin proteins differ only by one electron, yet they have distinct conformations.  The Fe center is shown to play an allosteric role on an β-shaped surface area that has two distinct conformations based on the identity and the formal redox state of the metal.  The authors show that the R-shaped area composes the interface for pirin-NF-κB binding that is responsible for modulation of NF-κB's DNA-binding properties.  The nonheme Fe protein pirin is proposed to serve as a reversible functional switch that enables NF-κB to respond to changes in the redox levels of the cell nucleus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7Tm-I0Q4mD7Vg90H21EOLACvtfcHk0lhrEs5Ki55bcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOrt7bF&md5=d81627e3a8aa09ee6a46a8557719394d</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1221743110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1221743110%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRehmani%26aufirst%3DI.%26aulast%3DEsaki%26aufirst%3DS.%26aulast%3DFu%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DL.%26aulast%3Dde%2BSerrano%26aufirst%3DV.%26aulast%3DLiu%26aufirst%3DA.%26atitle%3DPirin%2520is%2520an%2520iron-dependent%2520redox%2520regulator%2520of%2520NF-%25CE%25BAB%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D9722%26epage%3D9727%26doi%3D10.1073%2Fpnas.1221743110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Brzoska, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stepkowski, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruszewski, M.</span><span> </span><span class="NLM_article-title">Basal PIR expression in HeLa Cells is driven by NRF2 via evolutionary conserved antioxidant response element</span> <span class="citation_source-journal">Mol. Cell. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">389</span><span class="NLM_x">, </span> <span class="NLM_fpage">99</span><span class="NLM_x">–</span> <span class="NLM_lpage">111</span><span class="refDoi"> DOI: 10.1007/s11010-013-1931-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1007%2Fs11010-013-1931-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=2014&pages=99-111&author=K.+Brzoskaauthor=T.+M.+Stepkowskiauthor=M.+Kruszewski&title=Basal+PIR+expression+in+HeLa+Cells+is+driven+by+NRF2+via+evolutionary+conserved+antioxidant+response+element&doi=10.1007%2Fs11010-013-1931-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58a&amp;dbid=16384&amp;doi=10.1007%2Fs11010-013-1931-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11010-013-1931-0%26sid%3Dliteratum%253Aachs%26aulast%3DBrzoska%26aufirst%3DK.%26aulast%3DStepkowski%26aufirst%3DT.%2BM.%26aulast%3DKruszewski%26aufirst%3DM.%26atitle%3DBasal%2520PIR%2520expression%2520in%2520HeLa%2520Cells%2520is%2520driven%2520by%2520NRF2%2520via%2520evolutionary%2520conserved%2520antioxidant%2520response%2520element%26jtitle%3DMol.%2520Cell.%2520Biochem.%26date%3D2014%26volume%3D389%26spage%3D99%26epage%3D111%26doi%3D10.1007%2Fs11010-013-1931-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit58b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Gelbman, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heguy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Connor, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zabner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crystal, R. G.</span><span> </span><span class="NLM_article-title">Upregulation of pirin expression by chronic cigarette smoking is associated with bronchial epithelial cell apoptosis</span> <span class="citation_source-journal">Respir. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">10</span><span class="refDoi"> DOI: 10.1186/1465-9921-8-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1186%2F1465-9921-8-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=17288615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A280%3ADC%252BD2s7is1yitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=10&author=B.+D.+Gelbmanauthor=A.+Heguyauthor=T.+P.+O%E2%80%99Connorauthor=J.+Zabnerauthor=R.+G.+Crystal&title=Upregulation+of+pirin+expression+by+chronic+cigarette+smoking+is+associated+with+bronchial+epithelial+cell+apoptosis&doi=10.1186%2F1465-9921-8-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58bR"><div class="casContent"><span class="casTitleNuber">58b</span><div class="casTitle"><span class="NLM_cas:atitle">Upregulation of pirin expression by chronic cigarette smoking is associated with bronchial epithelial cell apoptosis</span></div><div class="casAuthors">Gelbman Brian D; Heguy Adriana; O'Connor Timothy P; Zabner Joseph; Crystal Ronald G</div><div class="citationInfo"><span class="NLM_cas:title">Respiratory research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Cigarette smoke disrupts the protective barrier established by the airway epithelium through direct damage to the epithelial cells, leading to cell death.  Since the morphology of the airway epithelium of smokers does not typically demonstrate necrosis, the most likely mechanism for epithelial cell death in response to cigarette smoke is apoptosis.  We hypothesized that cigarette smoke directly up-regulates expression of apoptotic genes, which could play a role in airway epithelial apoptosis.  METHODS:  Microarray analysis of airway epithelium obtained by bronchoscopy on matched cohorts of 13 phenotypically normal smokers and 9 non-smokers was used to identify specific genes modulated by smoking that were associated with apoptosis.  Among the up-regulated apoptotic genes was pirin (3.1-fold, p < 0.002), an iron-binding nuclear protein and transcription cofactor.  In vitro studies using human bronchial cells exposed to cigarette smoke extract (CSE) and an adenovirus vector encoding the pirin cDNA (AdPirin) were performed to test the direct effect of cigarette smoke on pirin expression and the effect of pirin expression on apoptosis.  RESULTS:  Quantitative TaqMan RT-PCR confirmed a 2-fold increase in pirin expression in the airway epithelium of smokers compared to non-smokers (p < 0.02).  CSE applied to primary human bronchial epithelial cell cultures demonstrated that pirin mRNA levels increase in a time-and concentration-dependent manner (p < 0.03, all conditions compared to controls).  Overexpression of pirin, using the vector AdPirin, in human bronchial epithelial cells was associated with an increase in the number of apoptotic cells assessed by both TUNEL assay (5-fold, p < 0.01) and ELISA for cytoplasmic nucleosomes (19.3-fold, p < 0.01) compared to control adenovirus vector.  CONCLUSION:  These observations suggest that up-regulation of pirin may represent one mechanism by which cigarette smoke induces apoptosis in the airway epithelium, an observation that has implications for the pathogenesis of cigarette smoke-induced diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQNual6ZRHCSQ5nADWVYHg1fW6udTcc2eYMRhOEP1AvZLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s7is1yitw%253D%253D&md5=fb51b1dd8e5675bb53f1f3bc23ea3ce4</span></div><a href="/servlet/linkout?suffix=cit58b&amp;dbid=16384&amp;doi=10.1186%2F1465-9921-8-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1465-9921-8-10%26sid%3Dliteratum%253Aachs%26aulast%3DGelbman%26aufirst%3DB.%2BD.%26aulast%3DHeguy%26aufirst%3DA.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DT.%2BP.%26aulast%3DZabner%26aufirst%3DJ.%26aulast%3DCrystal%26aufirst%3DR.%2BG.%26atitle%3DUpregulation%2520of%2520pirin%2520expression%2520by%2520chronic%2520cigarette%2520smoking%2520is%2520associated%2520with%2520bronchial%2520epithelial%2520cell%2520apoptosis%26jtitle%3DRespir.%2520Res.%26date%3D2007%26volume%3D8%26spage%3D10%26doi%3D10.1186%2F1465-9921-8-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Qiao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span> </span><span class="NLM_article-title">Pirin down-regulates the EAF2/U19 protein and alleviates its growth inhibition in prostate cancer cells</span> <span class="citation_source-journal">Prostate</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x">, </span> <span class="NLM_fpage">113</span><span class="NLM_x">–</span> <span class="NLM_lpage">120</span><span class="refDoi"> DOI: 10.1002/pros.22729</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1002%2Fpros.22729" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=113-120&author=Z.+Qiaoauthor=D.+Wangauthor=J.+Hahnauthor=J.+Aiauthor=Z.+Wang&title=Pirin+down-regulates+the+EAF2%2FU19+protein+and+alleviates+its+growth+inhibition+in+prostate+cancer+cells&doi=10.1002%2Fpros.22729"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2Fpros.22729&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpros.22729%26sid%3Dliteratum%253Aachs%26aulast%3DQiao%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DJ.%26aulast%3DAi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26atitle%3DPirin%2520down-regulates%2520the%2520EAF2%252FU19%2520protein%2520and%2520alleviates%2520its%2520growth%2520inhibition%2520in%2520prostate%2520cancer%2520cells%26jtitle%3DProstate%26date%3D2014%26volume%3D74%26spage%3D113%26epage%3D120%26doi%3D10.1002%2Fpros.22729" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Licciulli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luise, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scafetta, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capra, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giardina, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nuciforo, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bosari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viale, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazzarol, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonelli, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanfrancone, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alcalay, M.</span><span> </span><span class="NLM_article-title">Pirin inhibits cellular senescence in melanocytic cells</span> <span class="citation_source-journal">Am. J. Pathol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">178</span><span class="NLM_x">, </span> <span class="NLM_fpage">2397</span><span class="NLM_x">–</span> <span class="NLM_lpage">2406</span><span class="refDoi"> DOI: 10.1016/j.ajpath.2011.01.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.ajpath.2011.01.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=21514450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntVyqs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=178&publication_year=2011&pages=2397-2406&author=S.+Licciulliauthor=C.+Luiseauthor=G.+Scafettaauthor=M.+Capraauthor=G.+Giardinaauthor=P.+Nuciforoauthor=S.+Bosariauthor=G.+Vialeauthor=G.+Mazzarolauthor=C.+Tonelliauthor=L.+Lanfranconeauthor=M.+Alcalay&title=Pirin+inhibits+cellular+senescence+in+melanocytic+cells&doi=10.1016%2Fj.ajpath.2011.01.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Pirin inhibits cellular senescence in melanocytic cells</span></div><div class="casAuthors">Licciulli, Silvia; Luise, Chiara; Scafetta, Gaia; Capra, Maria; Giardina, Giuseppina; Nuciforo, Paolo; Bosari, Silvano; Viale, Giuseppe; Mazzarol, Giovanni; Tonelli, Chiara; Lanfrancone, Luisa; Alcalay, Myriam</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">178</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2397-2406</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Cellular senescence has been widely recognized as a tumor suppressing mechanism that acts as a barrier to cancer development after oncogenic stimuli.  A prominent in vivo model of the senescence barrier is represented by nevi, which are composed of melanocytes that, after an initial phase of proliferation induced by activated oncogenes (most commonly BRAF), are blocked in a state of cellular senescence.  Transformation to melanoma occurs when genes involved in controlling senescence are mutated or silenced and cells reacquire the capacity to proliferate.  Pirin (PIR) is a highly conserved nuclear protein that likely functions as a transcriptional regulator whose expression levels are altered in different types of tumors.  We analyzed the expression pattern of PER in adult human tissues and found that it is expressed in melanocytes and has a complex pattern of regulation in nevi and melanoma: it is rarely detected in mature nevi, but is expressed at high levels in a subset of melanomas.  Loss of function and overexpression expts. in normal and transformed melanocytic cells revealed that PIR is involved in the neg. control of cellular senescence and that its expression is necessary to overcome the senescence barrier.  Our results suggest that PIR may have a relevant role in melanoma progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoqcH_XkOw-bVg90H21EOLACvtfcHk0liFDCgVkce-SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntVyqs7o%253D&md5=c40cc6c5235026b31db04ef84aadd38e</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.ajpath.2011.01.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajpath.2011.01.019%26sid%3Dliteratum%253Aachs%26aulast%3DLicciulli%26aufirst%3DS.%26aulast%3DLuise%26aufirst%3DC.%26aulast%3DScafetta%26aufirst%3DG.%26aulast%3DCapra%26aufirst%3DM.%26aulast%3DGiardina%26aufirst%3DG.%26aulast%3DNuciforo%26aufirst%3DP.%26aulast%3DBosari%26aufirst%3DS.%26aulast%3DViale%26aufirst%3DG.%26aulast%3DMazzarol%26aufirst%3DG.%26aulast%3DTonelli%26aufirst%3DC.%26aulast%3DLanfrancone%26aufirst%3DL.%26aulast%3DAlcalay%26aufirst%3DM.%26atitle%3DPirin%2520inhibits%2520cellular%2520senescence%2520in%2520melanocytic%2520cells%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2011%26volume%3D178%26spage%3D2397%26epage%3D2406%26doi%3D10.1016%2Fj.ajpath.2011.01.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Miyazaki, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simizu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okumura, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takagi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osada, H.</span><span> </span><span class="NLM_article-title">A small-molecule inhibitor shows that pirin regulates migration of melanoma cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">667</span><span class="NLM_x">–</span> <span class="NLM_lpage">673</span><span class="refDoi"> DOI: 10.1038/nchembio.423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnchembio.423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=20711196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVais7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=667-673&author=I.+Miyazakiauthor=S.+Simizuauthor=H.+Okumuraauthor=S.+Takagiauthor=H.+Osada&title=A+small-molecule+inhibitor+shows+that+pirin+regulates+migration+of+melanoma+cells&doi=10.1038%2Fnchembio.423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">A small-molecule inhibitor shows that pirin regulates migration of melanoma cells</span></div><div class="casAuthors">Miyazaki, Isao; Simizu, Siro; Okumura, Hideo; Takagi, Satoshi; Osada, Hiroyuki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">667-673</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The discovery of small mols. that bind to a specific target and disrupt the function of proteins is an important step in chem. biol., esp. for poorly characterized proteins.  Human pirin is a nuclear protein of unknown function that is widely expressed in punctate subnuclear structures in human tissues.  Here, we report the discovery of a small mol. that binds to pirin.  We detd. how the small mol. bound to pirin by solving the cocrystal structure.  Either knockdown of pirin or treatment with the small mol. inhibited melanoma cell migration.  Thus, inhibition of pirin by the small mol. has led to a greater understanding of the function of pirin and represents a new method of studying pirin-mediated signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIZDQkD9RCtrVg90H21EOLACvtfcHk0liFDCgVkce-SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVais7bF&md5=21479021d6460fa0381eced68c6ab799</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.423%26sid%3Dliteratum%253Aachs%26aulast%3DMiyazaki%26aufirst%3DI.%26aulast%3DSimizu%26aufirst%3DS.%26aulast%3DOkumura%26aufirst%3DH.%26aulast%3DTakagi%26aufirst%3DS.%26aulast%3DOsada%26aufirst%3DH.%26atitle%3DA%2520small-molecule%2520inhibitor%2520shows%2520that%2520pirin%2520regulates%2520migration%2520of%2520melanoma%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D667%26epage%3D673%26doi%3D10.1038%2Fnchembio.423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Yamaoka, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sumiyoshi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minakawa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sumida, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saito, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikoma, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mabuchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozawa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inagaki, Y.</span><span> </span><span class="NLM_article-title">A novel small compound accelerates dermal wound healing by modifying infiltration, proliferation and migration of distinct cellular components in mice</span> <span class="citation_source-journal">J. Dermatol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x">, </span> <span class="NLM_fpage">204</span><span class="NLM_x">–</span> <span class="NLM_lpage">213</span><span class="refDoi"> DOI: 10.1016/j.jdermsci.2014.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.jdermsci.2014.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=24702853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1elsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=204-213&author=H.+Yamaokaauthor=H.+Sumiyoshiauthor=K.+Higashiauthor=S.+Nakaoauthor=K.+Minakawaauthor=K.+Sumidaauthor=K.+Saitoauthor=N.+Ikomaauthor=T.+Mabuchiauthor=A.+Ozawaauthor=Y.+Inagaki&title=A+novel+small+compound+accelerates+dermal+wound+healing+by+modifying+infiltration%2C+proliferation+and+migration+of+distinct+cellular+components+in+mice&doi=10.1016%2Fj.jdermsci.2014.03.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">A novel small compound accelerates dermal wound healing by modifying infiltration, proliferation and migration of distinct cellular components in mice</span></div><div class="casAuthors">Yamaoka, Hanako; Sumiyoshi, Hideaki; Higashi, Kiyoshi; Nakao, Sachie; Minakawa, Kaori; Sumida, Kayo; Saito, Koichi; Ikoma, Norihiro; Mabuchi, Tomotaka; Ozawa, Akira; Inagaki, Yutaka</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Dermatological Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">204-213</span>CODEN:
                <span class="NLM_cas:coden">JDSCEI</span>;
        ISSN:<span class="NLM_cas:issn">0923-1811</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Impaired wound healing in skin ulcer is one of the major medical issues in the aged society.  Wound healing is a complex process orchestrated by a no. of humoral factors and cellular components.  TGF-β is known to stimulate collagen prodn. in dermal fibroblasts while inhibiting proliferation of epidermal keratinocyte.  A screening of small compds. that suppress type I collagen prodn. in fibroblasts has identified HSc025 that antagonizes the TGF-β/Smad signal.We examd. the effects of HSc025 on dermal wound healing and elucidated the underlying mechanisms.Effects of HSc025 on the wound closure process were evaluated in a murine full-thickness excisional wound healing model.  Cell proliferation and migration were estd. using primary cultures of human keratinocytes and fibroblasts.  Comprehensive analyses of gene expression profiles were performed using untreated and HSc025-treated fibroblasts.Oral HSc025 administration suppressed macrophage infiltration and accelerated wound closure as early as at day 2 after the dermal excision.  Treatment of cultured keratinocytes with HSc025 counteracted the inhibitory effects of TGF-β on cell proliferation and migration.  On the other hand, HSc025 stimulated migration, but not proliferation, of dermal fibroblasts independently of TGF-β.  Expts. using an artificial dermis graft revealed that HSc025 stimulated migration of collagen-producing cells into the graft tissue.  A cDNA microarray anal. of untreated and HSc025-treated fibroblasts identified pirin as a crit. mediator accelerating fibroblast migration.HSc025 accelerates wound healing by modifying infiltration, proliferation and migration of distinct cellular components, which provides a novel insight into the therapy for intractable skin ulcer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0cYiJTQDJ3rVg90H21EOLACvtfcHk0liFDCgVkce-SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1elsbw%253D&md5=d9d77d13a9040b13afcb5c693e59f766</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.jdermsci.2014.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jdermsci.2014.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DYamaoka%26aufirst%3DH.%26aulast%3DSumiyoshi%26aufirst%3DH.%26aulast%3DHigashi%26aufirst%3DK.%26aulast%3DNakao%26aufirst%3DS.%26aulast%3DMinakawa%26aufirst%3DK.%26aulast%3DSumida%26aufirst%3DK.%26aulast%3DSaito%26aufirst%3DK.%26aulast%3DIkoma%26aufirst%3DN.%26aulast%3DMabuchi%26aufirst%3DT.%26aulast%3DOzawa%26aufirst%3DA.%26aulast%3DInagaki%26aufirst%3DY.%26atitle%3DA%2520novel%2520small%2520compound%2520accelerates%2520dermal%2520wound%2520healing%2520by%2520modifying%2520infiltration%252C%2520proliferation%2520and%2520migration%2520of%2520distinct%2520cellular%2520components%2520in%2520mice%26jtitle%3DJ.%2520Dermatol.%2520Sci.%26date%3D2014%26volume%3D74%26spage%3D204%26epage%3D213%26doi%3D10.1016%2Fj.jdermsci.2014.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Komai, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niwa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasazawa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simizu, S.</span><span> </span><span class="NLM_article-title">Pirin regulates epithelial to mesenchymal transition independently of BCL3-SLUG signaling</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">589</span><span class="NLM_x">, </span> <span class="NLM_fpage">738</span><span class="NLM_x">–</span> <span class="NLM_lpage">743</span><span class="refDoi"> DOI: 10.1016/j.febslet.2015.01.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.febslet.2015.01.040" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=589&publication_year=2015&pages=738-743&author=K.+Komaiauthor=Y.+Niwaauthor=Y.+Sasazawaauthor=S.+Simizu&title=Pirin+regulates+epithelial+to+mesenchymal+transition+independently+of+BCL3-SLUG+signaling&doi=10.1016%2Fj.febslet.2015.01.040"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63a&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2015.01.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2015.01.040%26sid%3Dliteratum%253Aachs%26aulast%3DKomai%26aufirst%3DK.%26aulast%3DNiwa%26aufirst%3DY.%26aulast%3DSasazawa%26aufirst%3DY.%26aulast%3DSimizu%26aufirst%3DS.%26atitle%3DPirin%2520regulates%2520epithelial%2520to%2520mesenchymal%2520transition%2520independently%2520of%2520BCL3-SLUG%2520signaling%26jtitle%3DFEBS%2520Lett.%26date%3D2015%26volume%3D589%26spage%3D738%26epage%3D743%26doi%3D10.1016%2Fj.febslet.2015.01.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit63b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Marcucci, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stassi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Maria, R.</span><span> </span><span class="NLM_article-title">Epithelial–mesenchymal transition: a new target in anticancer drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">311</span><span class="NLM_x">–</span> <span class="NLM_lpage">325</span><span class="refDoi"> DOI: 10.1038/nrd.2015.13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1038%2Fnrd.2015.13" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=311-325&author=F.+Marcucciauthor=G.+Stassiauthor=R.+De+Maria&title=Epithelial%E2%80%93mesenchymal+transition%3A+a+new+target+in+anticancer+drug+discovery&doi=10.1038%2Fnrd.2015.13"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63b&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2015.13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2015.13%26sid%3Dliteratum%253Aachs%26aulast%3DMarcucci%26aufirst%3DF.%26aulast%3DStassi%26aufirst%3DG.%26aulast%3DDe%2BMaria%26aufirst%3DR.%26atitle%3DEpithelial%25E2%2580%2593mesenchymal%2520transition%253A%2520a%2520new%2520target%2520in%2520anticancer%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D311%26epage%3D325%26doi%3D10.1038%2Fnrd.2015.13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="note"><p class="first last">In-house studies using siRNA to inhibit pirin synthesis in SK-OV-3 cells demonstrated only limited effects on proliferation. Further studies are ongoing.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span>TPh A was purchased from Sigma-Aldrich: <a href="http://www.sigmaaldrich.com/catalog/product/sigma/t6205?lang=en&amp;region=GB" class="extLink">http://www.sigmaaldrich.com/catalog/product/sigma/t6205?lang=en&region=GB</a> (August 31,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=TPh+A+was+purchased+from+Sigma-Aldrich%3A+http%3A%2F%2Fwww.sigmaaldrich.com%2Fcatalog%2Fproduct%2Fsigma%2Ft6205%3Flang%3Den%26region%3DGB+%28August+31%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Dunwell, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Culham, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sosa-Aguirre, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodenough, P. W.</span><span> </span><span class="NLM_article-title">Evolution of functional diversity in the cupin superfamily</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">740</span><span class="NLM_x">–</span> <span class="NLM_lpage">746</span><span class="refDoi"> DOI: 10.1016/S0968-0004(01)01981-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2FS0968-0004%2801%2901981-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=11738598" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BD3MXovVeru7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2001&pages=740-746&author=J.+M.+Dunwellauthor=A.+Culhamauthor=C.+E.+Carterauthor=C.+R.+Sosa-Aguirreauthor=P.+W.+Goodenough&title=Evolution+of+functional+diversity+in+the+cupin+superfamily&doi=10.1016%2FS0968-0004%2801%2901981-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66aR"><div class="casContent"><span class="casTitleNuber">66a</span><div class="casTitle"><span class="NLM_cas:atitle">Evolution of functional diversity in the cupin superfamily</span></div><div class="casAuthors">Dunwell, Jim M.; Culham, Alastair; Carter, Carol E.; Sosa-Aguirre, Carlos R.; Goodenough, Peter W.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">740-746</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The cupin superfamily of proteins is among the most functionally diverse of any described to date.  It was named on the basis of the conserved β-barrel fold ('cupa' is the Latin term for a small barrel), and comprises both enzymic and non-enzymic members, which have either one or two cupin domains.  Within the conserved tertiary structure, the variety of biochem. function is provided by minor variation of the residues in the active site and the identity of the bound metal ion.  This review discusses the advantages of this particular scaffold and provides an evolutionary anal. of 18 different subclasses within the cupin superfamily.  This mol. archaeol. review describes the discovery of the early ancestors of the plant proteins that now make up a major part of the human diet.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRAWCXekIxMLVg90H21EOLACvtfcHk0li_4S0u1jScUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXovVeru7w%253D&md5=b7c00861747d648509a3cca37676f4ec</span></div><a href="/servlet/linkout?suffix=cit66a&amp;dbid=16384&amp;doi=10.1016%2FS0968-0004%2801%2901981-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0004%252801%252901981-8%26sid%3Dliteratum%253Aachs%26aulast%3DDunwell%26aufirst%3DJ.%2BM.%26aulast%3DCulham%26aufirst%3DA.%26aulast%3DCarter%26aufirst%3DC.%2BE.%26aulast%3DSosa-Aguirre%26aufirst%3DC.%2BR.%26aulast%3DGoodenough%26aufirst%3DP.%2BW.%26atitle%3DEvolution%2520of%2520functional%2520diversity%2520in%2520the%2520cupin%2520superfamily%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2001%26volume%3D26%26spage%3D740%26epage%3D746%26doi%3D10.1016%2FS0968-0004%2801%2901981-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit66b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Dunwell, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purvis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuri, S.</span><span> </span><span class="NLM_article-title">Cupins: the most functionally diverse protein superfamily?</span> <span class="citation_source-journal">Phytochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">7</span><span class="NLM_x">–</span> <span class="NLM_lpage">17</span><span class="refDoi"> DOI: 10.1016/j.phytochem.2003.08.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1016%2Fj.phytochem.2003.08.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=14697267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvFynsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2004&pages=7-17&author=J.+M.+Dunwellauthor=A.+Purvisauthor=S.+Khuri&title=Cupins%3A+the+most+functionally+diverse+protein+superfamily%3F&doi=10.1016%2Fj.phytochem.2003.08.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66bR"><div class="casContent"><span class="casTitleNuber">66b</span><div class="casTitle"><span class="NLM_cas:atitle">Cupins: the most functionally diverse protein superfamily?</span></div><div class="casAuthors">Dunwell, Jim M.; Purvis, Alan; Khuri, Sawsan</div><div class="citationInfo"><span class="NLM_cas:title">Phytochemistry (Elsevier)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7-17</span>CODEN:
                <span class="NLM_cas:coden">PYTCAS</span>;
        ISSN:<span class="NLM_cas:issn">0031-9422</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review.  The cupin superfamily of proteins, named on the basis of a conserved β-barrel fold ('cupa' is the Latin term for a small barrel), was originally discovered using a conserved motif found within germin and germin-like proteins from higher plants.  Previous anal. of cupins had identified some 18 different functional classes that range from single-domain bacterial enzymes such as isomerases and epimerases involved in the modification of cell wall carbohydrates, through to two-domain bicupins such as the desiccation-tolerant seed storage globulins, and multidomain transcription factors including one linked to the nodulation response in legumes.  Recent advances in comparative genomics, and the resoln. of many more 3-D structures have now revealed that the largest subset of the cupin superfamily is the 2-oxyglutarate-Fe2+ dependent dioxygenases.  The substrates for this subclass of enzyme are many and varied and in total amt. to probably 50-100 different biochem. reactions, including several involved in plant growth and development.  Although the majority of enzymic cupins contain iron as an active site metal, other members contain either copper, zinc, cobalt, nickel or manganese ions as a cofactor, with each cofactor allowing a different type of chem. to occur within the conserved tertiary structure.  This review discusses the range of structures and functions found in this most diverse of superfamilies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGLBE2G_DAE7Vg90H21EOLACvtfcHk0li_4S0u1jScUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvFynsro%253D&md5=8af457482acd815ffc0d8ff05bf247c7</span></div><a href="/servlet/linkout?suffix=cit66b&amp;dbid=16384&amp;doi=10.1016%2Fj.phytochem.2003.08.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phytochem.2003.08.016%26sid%3Dliteratum%253Aachs%26aulast%3DDunwell%26aufirst%3DJ.%2BM.%26aulast%3DPurvis%26aufirst%3DA.%26aulast%3DKhuri%26aufirst%3DS.%26atitle%3DCupins%253A%2520the%2520most%2520functionally%2520diverse%2520protein%2520superfamily%253F%26jtitle%3DPhytochemistry%26date%3D2004%26volume%3D65%26spage%3D7%26epage%3D17%26doi%3D10.1016%2Fj.phytochem.2003.08.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Khuri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bakker, F. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunwell, J. M.</span><span> </span><span class="NLM_article-title">Phylogeny, function, and evolution of the cupins, a structurally conserved, functionally diverse superfamily of proteins</span> <span class="citation_source-journal">Mol. Biol. Evol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">593</span><span class="NLM_x">–</span> <span class="NLM_lpage">605</span><span class="refDoi"> DOI: 10.1093/oxfordjournals.molbev.a003840</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1093%2Foxfordjournals.molbev.a003840" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2001&pages=593-605&author=S.+Khuriauthor=F.+T.+Bakkerauthor=J.+M.+Dunwell&title=Phylogeny%2C+function%2C+and+evolution+of+the+cupins%2C+a+structurally+conserved%2C+functionally+diverse+superfamily+of+proteins&doi=10.1093%2Foxfordjournals.molbev.a003840"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67a&amp;dbid=16384&amp;doi=10.1093%2Foxfordjournals.molbev.a003840&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Foxfordjournals.molbev.a003840%26sid%3Dliteratum%253Aachs%26aulast%3DKhuri%26aufirst%3DS.%26aulast%3DBakker%26aufirst%3DF.%2BT.%26aulast%3DDunwell%26aufirst%3DJ.%2BM.%26atitle%3DPhylogeny%252C%2520function%252C%2520and%2520evolution%2520of%2520the%2520cupins%252C%2520a%2520structurally%2520conserved%252C%2520functionally%2520diverse%2520superfamily%2520of%2520proteins%26jtitle%3DMol.%2520Biol.%2520Evol.%26date%3D2001%26volume%3D18%26spage%3D593%26epage%3D605%26doi%3D10.1093%2Foxfordjournals.molbev.a003840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit67b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Uberto, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moomaw, E. W.</span><span> </span><span class="NLM_article-title">Protein similarity networks reveal relationships among sequence, structure, and function within the cupin superfamily</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e74477</span><span class="refDoi"> DOI: 10.1371/journal.pone.0074477</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1371%2Fjournal.pone.0074477" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e74477&author=R.+Ubertoauthor=E.+W.+Moomaw&title=Protein+similarity+networks+reveal+relationships+among+sequence%2C+structure%2C+and+function+within+the+cupin+superfamily&doi=10.1371%2Fjournal.pone.0074477"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67b&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0074477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0074477%26sid%3Dliteratum%253Aachs%26aulast%3DUberto%26aufirst%3DR.%26aulast%3DMoomaw%26aufirst%3DE.%2BW.%26atitle%3DProtein%2520similarity%2520networks%2520reveal%2520relationships%2520among%2520sequence%252C%2520structure%252C%2520and%2520function%2520within%2520the%2520cupin%2520superfamily%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De74477%26doi%3D10.1371%2Fjournal.pone.0074477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><div class="note"><p class="first last">PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1J1L">1J1L</a>:</p></div><span class="NLM_contrib-group">Pang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartlam, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyatake, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hisano, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, Z.</span><span> </span><span class="NLM_article-title">Crystal structure of human pirin an iron-binding nuclear protein and transcription cofactor</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">1491</span><span class="NLM_x">–</span> <span class="NLM_lpage">1498</span><div class="note"><p class="first last">For other examples of Pirin crystal structures, see: PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ACL">3ACL</a>, ref <a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">61</a>; PDBs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ACL">3ACL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ERO">4ERO</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EWA">4EWA</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EWD">4EWD</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EWE">4EWE</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GUL">4GUL</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HLT">4HLT</a>, ref <a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">57</a></p></div><span class="refDoi"> DOI: 10.1074/jbc.M310022200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1074%2Fjbc.M310022200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=1491-1498&author=H.+Pangauthor=M.+Bartlamauthor=Q.+Zengauthor=H.+Miyatakeauthor=T.+Hisanoauthor=K.+Mikiauthor=L.+L.+Wongauthor=G.+F.+Gaoauthor=Z.+Rao&title=Crystal+structure+of+human+pirin+an+iron-binding+nuclear+protein+and+transcription+cofactor&doi=10.1074%2Fjbc.M310022200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M310022200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M310022200%26sid%3Dliteratum%253Aachs%26aulast%3DPang%26aufirst%3DH.%26aulast%3DBartlam%26aufirst%3DM.%26aulast%3DZeng%26aufirst%3DQ.%26aulast%3DMiyatake%26aufirst%3DH.%26aulast%3DHisano%26aufirst%3DT.%26aulast%3DMiki%26aufirst%3DK.%26aulast%3DWong%26aufirst%3DL.%2BL.%26aulast%3DGao%26aufirst%3DG.%2BF.%26aulast%3DRao%26aufirst%3DZ.%26atitle%3DCrystal%2520structure%2520of%2520human%2520pirin%2520an%2520iron-binding%2520nuclear%2520protein%2520and%2520transcription%2520cofactor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D1491%26epage%3D1498%26doi%3D10.1074%2Fjbc.M310022200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Lindsley, C. W.</span><span> </span><span class="NLM_article-title">2013 Philip S. Portoghese Medicinal Chemistry Lectureship: Drug Discovery Targeting Allosteric Sites</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">7485</span><span class="NLM_x">–</span> <span class="NLM_lpage">7498</span><span class="refDoi"> DOI: 10.1021/jm5011786</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011786" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtLvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7485-7498&author=C.+W.+Lindsley&title=2013+Philip+S.+Portoghese+Medicinal+Chemistry+Lectureship%3A+Drug+Discovery+Targeting+Allosteric+Sites&doi=10.1021%2Fjm5011786"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">2013 Philip S. Portoghese Medicinal Chemistry Lectureship: Drug Discovery Targeting Allosteric Sites</span></div><div class="casAuthors">Lindsley, Craig W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7485-7498</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The identification of sites on receptors topog. distinct from the orthosteric sites, so-called allosteric sites, has heralded novel approaches and modes of pharmacol. for target modulation.  Over the past 20 years, our understanding of allosteric modulation has grown significantly, and numerous advantages, as well as caveats (e.g., flat structure-activity relationships, species differences, "mol. switches"), have been identified.  For multiple receptors and proteins, numerous examples have been described where unprecedented levels of selectivity are achieved along with improved physiochem. properties.  While not a panacea, these novel approaches represent exciting opportunities for tool compd. development to probe the pharmacol. and therapeutic potential of discrete mol. targets, as well as new medicines.  In this Perspective, in commemoration of the 2013 Philip S.  Portoghese Medicinal Chem. Lectureship (Lindsley, C. W. Adventures in allosteric drug discovery. Presented at the 246th National Meeting of the American Chem. Society, Indianapolis, IN, Sept. 10, 2013; The 2013 Portoghese Lectureship), several vignettes of drug discovery campaigns targeting novel allosteric mechanisms will be recounted, along with lessons learned and guidelines that have emerged for successful lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowaY56ZaUKc7Vg90H21EOLACvtfcHk0lhyehJUeuedcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtLvE&md5=8372986d1e484efdd3e14312903d1d48</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fjm5011786&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011786%26sid%3Dliteratum%253Aachs%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26atitle%3D2013%2520Philip%2520S.%2520Portoghese%2520Medicinal%2520Chemistry%2520Lectureship%253A%2520Drug%2520Discovery%2520Targeting%2520Allosteric%2520Sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D7485%26epage%3D7498%26doi%3D10.1021%2Fjm5011786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Dosa, P. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amin, E. A.</span><span> </span><span class="NLM_article-title">Tactical approaches to interconverting GPCR agonists and antagonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">810</span><span class="NLM_x">–</span> <span class="NLM_lpage">840</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00982</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00982" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFamsr7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=810-840&author=P.+I.+Dosaauthor=E.+A.+Amin&title=Tactical+approaches+to+interconverting+GPCR+agonists+and+antagonists&doi=10.1021%2Facs.jmedchem.5b00982"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Tactical Approaches to Interconverting GPCR Agonists and Antagonists</span></div><div class="casAuthors">Dosa, Peter I.; Amin, Elizabeth Ambrose</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">810-840</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There are many reported examples of small structural modifications to GPCR-targeted ligands leading to major changes in their functional activity, converting agonists into antagonists or vice versa.  These shifts in functional activity are often accompanied by negligible changes in binding affinity.  The current perspective focuses on outlining and analyzing various approaches that have been used to interconvert GPCR agonists, partial agonists, and antagonists in order to achieve the intended functional activity at a GPCR of therapeutic interest.  An improved understanding of specific structural modifications that are likely to alter the functional activity of a GPCR ligand may be of use to researchers designing GPCR-targeted drugs and/or probe compds., specifically in cases where a particular ligand exhibits good potency but not the preferred functional activity at the GPCR of choice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohWg2-DQHq3LVg90H21EOLACvtfcHk0lhyehJUeuedcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFamsr7P&md5=99ffc0b892eb9c10aaf5d52a7d5dd8bc</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00982&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00982%26sid%3Dliteratum%253Aachs%26aulast%3DDosa%26aufirst%3DP.%2BI.%26aulast%3DAmin%26aufirst%3DE.%2BA.%26atitle%3DTactical%2520approaches%2520to%2520interconverting%2520GPCR%2520agonists%2520and%2520antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D810%26epage%3D840%26doi%3D10.1021%2Facs.jmedchem.5b00982" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Blomen, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Májek, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jae, L. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bigenzahn, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nieuwenhuis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staring, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sacco, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Diemen, F. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olk, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stukalov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marceau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janssen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carette, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colinge, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brummelkamp, T. R.</span><span> </span><span class="NLM_article-title">Gene essentiality and synthetic lethality in haploid human cells</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">350</span><span class="NLM_x">, </span> <span class="NLM_fpage">1092</span><span class="NLM_x">–</span> <span class="NLM_lpage">1096</span><span class="refDoi"> DOI: 10.1126/science.aac7557</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1126%2Fscience.aac7557" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=26472760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFamtL3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=350&publication_year=2015&pages=1092-1096&author=V.+A.+Blomenauthor=P.+M%C3%A1jekauthor=L.+T.+Jaeauthor=J.+W.+Bigenzahnauthor=J.+Nieuwenhuisauthor=J.+Staringauthor=R.+Saccoauthor=F.+R.+van+Diemenauthor=N.+Olkauthor=A.+Stukalovauthor=C.+Marceauauthor=H.+Janssenauthor=J.+E.+Caretteauthor=K.+L.+Bennettauthor=J.+Colingeauthor=G.+Superti-Furgaauthor=T.+R.+Brummelkamp&title=Gene+essentiality+and+synthetic+lethality+in+haploid+human+cells&doi=10.1126%2Fscience.aac7557"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Gene essentiality and synthetic lethality in haploid human cells</span></div><div class="casAuthors">Blomen, Vincent A.; Majek, Peter; Jae, Lucas T.; Bigenzahn, Johannes W.; Nieuwenhuis, Joppe; Staring, Jacqueline; Sacco, Roberto; van Diemen, Ferdy R.; Olk, Nadine; Stukalov, Alexey; Marceau, Caleb; Janssen, Hans; Carette, Jan E.; Bennett, Keiryn L.; Colinge, Jacques; Superti-Furga, Giulio; Brummelkamp, Thijn R.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">350</span>
        (<span class="NLM_cas:issue">6264</span>),
    <span class="NLM_cas:pages">1092-1096</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Although the genes essential for life have been identified in less complex model organisms, their elucidation in human cells has been hindered by tech. barriers.  We used extensive mutagenesis in haploid human cells to identify approx. 2000 genes required for optimal fitness under culture conditions.  To study the principles of genetic interactions in human cells, we created a synthetic lethality network focused on the secretory pathway based exclusively on mutations.  This revealed a genetic cross-talk governing Golgi homeostasis, an addnl. subunit of the human oligosaccharyltransferase complex, and a phosphatidylinositol 4-kinase β adaptor hijacked by viruses.  The synthetic lethality map parallels observations made in yeast and projects a route forward to reveal genetic networks in diverse aspects of human cell biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfqw6qG8hJG7Vg90H21EOLACvtfcHk0lhyehJUeuedcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFamtL3E&md5=bd6af396c2c5a67356d0fe578f64f1df</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1126%2Fscience.aac7557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aac7557%26sid%3Dliteratum%253Aachs%26aulast%3DBlomen%26aufirst%3DV.%2BA.%26aulast%3DM%25C3%25A1jek%26aufirst%3DP.%26aulast%3DJae%26aufirst%3DL.%2BT.%26aulast%3DBigenzahn%26aufirst%3DJ.%2BW.%26aulast%3DNieuwenhuis%26aufirst%3DJ.%26aulast%3DStaring%26aufirst%3DJ.%26aulast%3DSacco%26aufirst%3DR.%26aulast%3Dvan%2BDiemen%26aufirst%3DF.%2BR.%26aulast%3DOlk%26aufirst%3DN.%26aulast%3DStukalov%26aufirst%3DA.%26aulast%3DMarceau%26aufirst%3DC.%26aulast%3DJanssen%26aufirst%3DH.%26aulast%3DCarette%26aufirst%3DJ.%2BE.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DColinge%26aufirst%3DJ.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DBrummelkamp%26aufirst%3DT.%2BR.%26atitle%3DGene%2520essentiality%2520and%2520synthetic%2520lethality%2520in%2520haploid%2520human%2520cells%26jtitle%3DScience%26date%3D2015%26volume%3D350%26spage%3D1092%26epage%3D1096%26doi%3D10.1126%2Fscience.aac7557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birsoy, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krupczak, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Post, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lander, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabatini, D. M.</span><span> </span><span class="NLM_article-title">Identification and characterization of essential genes in the human genome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">350</span><span class="NLM_x">, </span> <span class="NLM_fpage">1096</span><span class="NLM_x">–</span> <span class="NLM_lpage">1101</span><span class="refDoi"> DOI: 10.1126/science.aac7041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=10.1126%2Fscience.aac7041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=26472758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFamtL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=350&publication_year=2015&pages=1096-1101&author=T.+Wangauthor=K.+Birsoyauthor=N.+W.+Hughesauthor=K.+M.+Krupczakauthor=Y.+Postauthor=J.+J.+Weiauthor=E.+S.+Landerauthor=D.+M.+Sabatini&title=Identification+and+characterization+of+essential+genes+in+the+human+genome&doi=10.1126%2Fscience.aac7041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of essential genes in the human genome</span></div><div class="casAuthors">Wang, Tim; Birsoy, Kivanc; Hughes, Nicholas W.; Krupczak, Kevin M.; Post, Yorick; Wei, Jenny J.; Lander, Eric S.; Sabatini, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">350</span>
        (<span class="NLM_cas:issue">6264</span>),
    <span class="NLM_cas:pages">1096-1101</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Large-scale genetic anal. of lethal phenotypes has elucidated the mol. underpinnings of many biol. processes.  Using the bacterial clustered regularly interspaced short palindromic repeats (CRISPR) system, we constructed a genome-wide single-guide RNA library to screen for genes required for proliferation and survival in a human cancer cell line.  Our screen revealed the set of cell-essential genes, which was validated with an orthogonal gene-trap-based screen and comparison with yeast gene knockouts.  This set is enriched for genes that encode components of fundamental pathways, are expressed at high levels, and contain few inactivating polymorphisms in the human population.  We also uncovered a large group of uncharacterized genes involved in RNA processing, a no. of whose products localize to the nucleolus.  Last, screens in addnl. cell lines showed a high degree of overlap in gene essentiality but also revealed differences specific to each cell line and cancer type that reflect the developmental origin, oncogenic drivers, paralogous gene expression pattern, and chromosomal structure of each line.  These results demonstrate the power of CRISPR-based screens and suggest a general strategy for identifying liabilities in cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonjG02K7R4iLVg90H21EOLACvtfcHk0ljV0ak1I2-IrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFamtL3F&md5=61273ba03938104748f0458f662a3c34</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1126%2Fscience.aac7041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aac7041%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DT.%26aulast%3DBirsoy%26aufirst%3DK.%26aulast%3DHughes%26aufirst%3DN.%2BW.%26aulast%3DKrupczak%26aufirst%3DK.%2BM.%26aulast%3DPost%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DJ.%2BJ.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520essential%2520genes%2520in%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2015%26volume%3D350%26spage%3D1096%26epage%3D1101%26doi%3D10.1126%2Fscience.aac7041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Staben, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyle, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belvin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boggs, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burch, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chua, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiPasquale, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedman, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heise, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koeppen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mintzer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rouge, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tam, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoeflich, K. P.</span><span> </span><span class="NLM_article-title">Back pocket flexibility provides group II p21-activated kinase (PAK) selectivity for type I 1/2 kinase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">1033</span><span class="NLM_x">–</span> <span class="NLM_lpage">1045</span><span class="refDoi"> DOI: 10.1021/jm401768t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401768t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotVGmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1033-1045&author=S.+T.+Stabenauthor=J.+A.+Fengauthor=K.+Lyleauthor=M.+Belvinauthor=J.+Boggsauthor=J.+D.+Burchauthor=C.+Chuaauthor=H.+Cuiauthor=A.+G.+DiPasqualeauthor=L.+S.+Friedmanauthor=C.+Heiseauthor=H.+Koeppenauthor=A.+Koteyauthor=R.+Mintzerauthor=A.+Ohauthor=D.+A.+Robertsauthor=L.+Rougeauthor=J.+Rudolphauthor=C.+Tamauthor=W.+Wangauthor=Y.+Xiaoauthor=A.+Youngauthor=Y.+Zhangauthor=K.+P.+Hoeflich&title=Back+pocket+flexibility+provides+group+II+p21-activated+kinase+%28PAK%29+selectivity+for+type+I+1%2F2+kinase+inhibitors&doi=10.1021%2Fjm401768t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73aR"><div class="casContent"><span class="casTitleNuber">73a</span><div class="casTitle"><span class="NLM_cas:atitle">Back Pocket Flexibility Provides Group II p21-Activated Kinase (PAK) Selectivity for Type I 1/2 Kinase Inhibitors</span></div><div class="casAuthors">Staben, Steven T.; Feng, Jianwen A.; Lyle, Karen; Belvin, Marcia; Boggs, Jason; Burch, Jason D.; Chua, Ching-ching; Cui, Haifeng; DiPasquale, Antonio G.; Friedman, Lori S.; Heise, Christopher; Koeppen, Hartmut; Kotey, Adrian; Mintzer, Robert; Oh, Angela; Roberts, David Allen; Rouge, Lionel; Rudolph, Joachim; Tam, Christine; Wang, Weiru; Xiao, Yisong; Young, Amy; Zhang, Yamin; Hoeflich, Klaus P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1033-1045</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-based methods were used to design a potent and highly selective group II p21-activated kinase (PAK) inhibitor with a novel binding mode, compd. I.  Hydrophobic interactions within a lipophilic pocket past the methionine gatekeeper of group II PAKs approached by these type I 1/2 binders were found to be important for improving potency.  A structure-based hypothesis and strategy for achieving selectivity over group I PAKs, and the broad kinome, based on unique flexibility of this lipophilic pocket, is presented.  A concn.-dependent decrease in tumor cell migration and invasion in two triple-neg. breast cancer cell lines was obsd. with compd. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrogBhcjJGWm7Vg90H21EOLACvtfcHk0ljV0ak1I2-IrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotVGmsQ%253D%253D&md5=08b11088bc4d806cc61e16c932b68cf6</span></div><a href="/servlet/linkout?suffix=cit73a&amp;dbid=16384&amp;doi=10.1021%2Fjm401768t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401768t%26sid%3Dliteratum%253Aachs%26aulast%3DStaben%26aufirst%3DS.%2BT.%26aulast%3DFeng%26aufirst%3DJ.%2BA.%26aulast%3DLyle%26aufirst%3DK.%26aulast%3DBelvin%26aufirst%3DM.%26aulast%3DBoggs%26aufirst%3DJ.%26aulast%3DBurch%26aufirst%3DJ.%2BD.%26aulast%3DChua%26aufirst%3DC.%26aulast%3DCui%26aufirst%3DH.%26aulast%3DDiPasquale%26aufirst%3DA.%2BG.%26aulast%3DFriedman%26aufirst%3DL.%2BS.%26aulast%3DHeise%26aufirst%3DC.%26aulast%3DKoeppen%26aufirst%3DH.%26aulast%3DKotey%26aufirst%3DA.%26aulast%3DMintzer%26aufirst%3DR.%26aulast%3DOh%26aufirst%3DA.%26aulast%3DRoberts%26aufirst%3DD.%2BA.%26aulast%3DRouge%26aufirst%3DL.%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DTam%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DYoung%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26atitle%3DBack%2520pocket%2520flexibility%2520provides%2520group%2520II%2520p21-activated%2520kinase%2520%2528PAK%2529%2520selectivity%2520for%2520type%2520I%25201%252F2%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D1033%26epage%3D1045%26doi%3D10.1021%2Fjm401768t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit73b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Ingrassia, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lefranc, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dewelle, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pottier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathieu, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spiegl-Kreinecker, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauvage, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El Yazidi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dehoux, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Quaquebeke, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiss, R.</span><span> </span><span class="NLM_article-title">Structure–activity relationship analysis of novel derivatives of narciclasine (an amaryllidaceae isocarbostyril derivative) as potential anticancer agents</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1100</span><span class="NLM_x">–</span> <span class="NLM_lpage">1114</span><span class="refDoi"> DOI: 10.1021/jm8013585</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8013585" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=1100-1114&author=L.+Ingrassiaauthor=F.+Lefrancauthor=J.+Dewelleauthor=L.+Pottierauthor=V.+Mathieuauthor=S.+Spiegl-Kreineckerauthor=S.+Sauvageauthor=M.+El+Yazidiauthor=M.+Dehouxauthor=W.+Bergerauthor=E.+Van+Quaquebekeauthor=R.+Kiss&title=Structure%E2%80%93activity+relationship+analysis+of+novel+derivatives+of+narciclasine+%28an+amaryllidaceae+isocarbostyril+derivative%29+as+potential+anticancer+agents&doi=10.1021%2Fjm8013585"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73b&amp;dbid=16384&amp;doi=10.1021%2Fjm8013585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8013585%26sid%3Dliteratum%253Aachs%26aulast%3DIngrassia%26aufirst%3DL.%26aulast%3DLefranc%26aufirst%3DF.%26aulast%3DDewelle%26aufirst%3DJ.%26aulast%3DPottier%26aufirst%3DL.%26aulast%3DMathieu%26aufirst%3DV.%26aulast%3DSpiegl-Kreinecker%26aufirst%3DS.%26aulast%3DSauvage%26aufirst%3DS.%26aulast%3DEl%2BYazidi%26aufirst%3DM.%26aulast%3DDehoux%26aufirst%3DM.%26aulast%3DBerger%26aufirst%3DW.%26aulast%3DVan%2BQuaquebeke%26aufirst%3DE.%26aulast%3DKiss%26aufirst%3DR.%26atitle%3DStructure%25E2%2580%2593activity%2520relationship%2520analysis%2520of%2520novel%2520derivatives%2520of%2520narciclasine%2520%2528an%2520amaryllidaceae%2520isocarbostyril%2520derivative%2529%2520as%2520potential%2520anticancer%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D1100%26epage%3D1114%26doi%3D10.1021%2Fjm8013585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit73c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Singh, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tamang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajamohanan, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jima, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakraborty, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kundu, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaikwad, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khan, M. I.</span><span> </span><span class="NLM_article-title">Isolation, structure, and functional elucidation of a modified pentapeptide, Cysteine protease inhibitor (CPI-2081) from streptomyces species 2081 that exhibit inhibitory effect on cancer cell migration</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5121</span><span class="NLM_x">–</span> <span class="NLM_lpage">5128</span><span class="refDoi"> DOI: 10.1021/jm9014179</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9014179" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5121-5128&author=J.+P.+Singhauthor=S.+Tamangauthor=P.+R.+Rajamohananauthor=N.+C.+Jimaauthor=G.+Chakrabortyauthor=G.+C.+Kunduauthor=S.+M.+Gaikwadauthor=M.+I.+Khan&title=Isolation%2C+structure%2C+and+functional+elucidation+of+a+modified+pentapeptide%2C+Cysteine+protease+inhibitor+%28CPI-2081%29+from+streptomyces+species+2081+that+exhibit+inhibitory+effect+on+cancer+cell+migration&doi=10.1021%2Fjm9014179"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73c&amp;dbid=16384&amp;doi=10.1021%2Fjm9014179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9014179%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DJ.%2BP.%26aulast%3DTamang%26aufirst%3DS.%26aulast%3DRajamohanan%26aufirst%3DP.%2BR.%26aulast%3DJima%26aufirst%3DN.%2BC.%26aulast%3DChakraborty%26aufirst%3DG.%26aulast%3DKundu%26aufirst%3DG.%2BC.%26aulast%3DGaikwad%26aufirst%3DS.%2BM.%26aulast%3DKhan%26aufirst%3DM.%2BI.%26atitle%3DIsolation%252C%2520structure%252C%2520and%2520functional%2520elucidation%2520of%2520a%2520modified%2520pentapeptide%252C%2520Cysteine%2520protease%2520inhibitor%2520%2528CPI-2081%2529%2520from%2520streptomyces%2520species%25202081%2520that%2520exhibit%2520inhibitory%2520effect%2520on%2520cancer%2520cell%2520migration%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5121%26epage%3D5128%26doi%3D10.1021%2Fjm9014179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit73d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Nagababu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barui, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thulasiram, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devi, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Satyanarayana, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patra, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sreedhar, B.</span><span> </span><span class="NLM_article-title">Antiangiogenic activity of mononuclear copper(II) polypyridyl complexes for the treatment of cancers</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">5226</span><span class="NLM_x">–</span> <span class="NLM_lpage">5241</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00651</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00651" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVans7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5226-5241&author=P.+Nagababuauthor=A.+K.+Baruiauthor=B.+Thulasiramauthor=C.+S.+Deviauthor=S.+Satyanarayanaauthor=C.+R.+Patraauthor=B.+Sreedhar&title=Antiangiogenic+activity+of+mononuclear+copper%28II%29+polypyridyl+complexes+for+the+treatment+of+cancers&doi=10.1021%2Facs.jmedchem.5b00651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73dR"><div class="casContent"><span class="casTitleNuber">73d</span><div class="casTitle"><span class="NLM_cas:atitle">Antiangiogenic Activity of Mononuclear Copper(II) Polypyridyl Complexes for the Treatment of Cancers</span></div><div class="casAuthors">Nagababu, Penumaka; Barui, Ayan Kumar; Thulasiram, Bathini; Devi, C. Shobha; Satyanarayana, S.; Patra, Chitta Ranjan; Sreedhar, Bojja</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5226-5241</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of four new mononuclear copper(II) polypyridyl complexes (1-4) have been designed, developed, and thoroughly characterized by several physicochem. techniques.  The CT-DNA binding properties of 1-4 have been investigated by absorption, emission spectroscopy, and viscosity measurements.  All the complexes esp. 1 and 4 exhibit cytotoxicity toward several cancer cell lines, suggesting their anticancer properties as obsd. by several in vitro assays.  Addnl., the complexes show inhibition of endothelial cell (HUVECs) proliferation, indicating their antiangiogenic nature.  In vivo chick embryo angiogenesis assay again confirms the antiangiogenic properties of 1 and 4.  The formation of excessive intracellular ROS (H2O2 and O2•-) and upregulation of BAX induced by copper(II) complexes may be the plausible mechanisms behind their anticancer activities.  The present study may offer a basis for the development of new transition metal complexes through suitable choice of ligands for cancer therapeutics by controlling tumor angiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpo-a7VLgXWKbVg90H21EOLACvtfcHk0lgM8Lv4s9SUMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVans7jI&md5=042e41e7346f577fe274d0f16903ec33</span></div><a href="/servlet/linkout?suffix=cit73d&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00651%26sid%3Dliteratum%253Aachs%26aulast%3DNagababu%26aufirst%3DP.%26aulast%3DBarui%26aufirst%3DA.%2BK.%26aulast%3DThulasiram%26aufirst%3DB.%26aulast%3DDevi%26aufirst%3DC.%2BS.%26aulast%3DSatyanarayana%26aufirst%3DS.%26aulast%3DPatra%26aufirst%3DC.%2BR.%26aulast%3DSreedhar%26aufirst%3DB.%26atitle%3DAntiangiogenic%2520activity%2520of%2520mononuclear%2520copper%2528II%2529%2520polypyridyl%2520complexes%2520for%2520the%2520treatment%2520of%2520cancers%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5226%26epage%3D5241%26doi%3D10.1021%2Facs.jmedchem.5b00651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit73e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Amin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wani, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farooq, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nayak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakraborty, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shankar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">ur Rasool, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koul, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goswami, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rai, R.</span><span> </span><span class="NLM_article-title">Inhibition of invasion in pancreatic cancer cells by conjugate of EPA with β3,3-Pip-OH via PI3K/Akt/NF-kB pathway</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1071</span><span class="NLM_x">–</span> <span class="NLM_lpage">1074</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00257</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00257" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=1071-1074&author=H.+Aminauthor=N.+A.+Waniauthor=S.+Farooqauthor=D.+Nayakauthor=S.+Chakrabortyauthor=S.+Shankarauthor=R.+ur+Rasoolauthor=S.+Koulauthor=A.+Goswamiauthor=R.+Rai&title=Inhibition+of+invasion+in+pancreatic+cancer+cells+by+conjugate+of+EPA+with+%CE%B23%2C3-Pip-OH+via+PI3K%2FAkt%2FNF-kB+pathway&doi=10.1021%2Facsmedchemlett.5b00257"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73e&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00257%26sid%3Dliteratum%253Aachs%26aulast%3DAmin%26aufirst%3DH.%26aulast%3DWani%26aufirst%3DN.%2BA.%26aulast%3DFarooq%26aufirst%3DS.%26aulast%3DNayak%26aufirst%3DD.%26aulast%3DChakraborty%26aufirst%3DS.%26aulast%3DShankar%26aufirst%3DS.%26aulast%3Dur%2BRasool%26aufirst%3DR.%26aulast%3DKoul%26aufirst%3DS.%26aulast%3DGoswami%26aufirst%3DA.%26aulast%3DRai%26aufirst%3DR.%26atitle%3DInhibition%2520of%2520invasion%2520in%2520pancreatic%2520cancer%2520cells%2520by%2520conjugate%2520of%2520EPA%2520with%2520%25CE%25B23%252C3-Pip-OH%2520via%2520PI3K%252FAkt%252FNF-kB%2520pathway%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D1071%26epage%3D1074%26doi%3D10.1021%2Facsmedchemlett.5b00257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="note"><p class="first last">In-house PAINS filter protocol: Biovia Pipeline Pilot version 9.5.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Lagerlund, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larhed, M.</span><span> </span><span class="NLM_article-title">Microwave-promoted aminocarbonylations of aryl chlorides using Mo(CO)<sub>6</sub> as a solid carbon monoxide source</span> <span class="citation_source-journal">J. Comb. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">4</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1021/cc050102r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cc050102r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Gms7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2006&pages=4-6&author=O.+Lagerlundauthor=M.+Larhed&title=Microwave-promoted+aminocarbonylations+of+aryl+chlorides+using+Mo%28CO%296+as+a+solid+carbon+monoxide+source&doi=10.1021%2Fcc050102r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Microwave-Promoted Aminocarbonylations of Aryl Chlorides Using Mo(CO)6 as a Solid Carbon Monoxide Source</span></div><div class="casAuthors">Lagerlund, Olof; Larhed, Mats</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Combinatorial Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-6</span>CODEN:
                <span class="NLM_cas:coden">JCCHFF</span>;
        ISSN:<span class="NLM_cas:issn">1520-4766</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Both electron-rich and electron-poor aryl chlorides serve as useful substrates in microwave-heated in situ aminocarbonylation reactions under noninert conditions.  Both aliph. amines and anilines provide benzamide products in good yields after only 15-25 min of irradn.  The reaction was catalyzed by Herrmann's palladacycle and [(t-Bu)3PH]BF4 in presence of DBU and Mo(CO)6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgdoXvL_HaVLVg90H21EOLACvtfcHk0lgM8Lv4s9SUMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Gms7nP&md5=979264c72a974f6ca64be5b3d44a0c88</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fcc050102r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcc050102r%26sid%3Dliteratum%253Aachs%26aulast%3DLagerlund%26aufirst%3DO.%26aulast%3DLarhed%26aufirst%3DM.%26atitle%3DMicrowave-promoted%2520aminocarbonylations%2520of%2520aryl%2520chlorides%2520using%2520Mo%2528CO%25296%2520as%2520a%2520solid%2520carbon%2520monoxide%2520source%26jtitle%3DJ.%2520Comb.%2520Chem.%26date%3D2006%26volume%3D8%26spage%3D4%26epage%3D6%26doi%3D10.1021%2Fcc050102r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JCT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JCT','PDB','5JCT'); return false;">PDB: 5JCT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1J1L" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1J1L','PDB','1J1L'); return false;">PDB: 1J1L</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ACL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ACL','PDB','3ACL'); return false;">PDB: 3ACL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ERO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ERO','PDB','4ERO'); return false;">PDB: 4ERO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EWA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EWA','PDB','4EWA'); return false;">PDB: 4EWA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EWD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EWD','PDB','4EWD'); return false;">PDB: 4EWD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EWE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4EWE','PDB','4EWE'); return false;">PDB: 4EWE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GUL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GUL','PDB','4GUL'); return false;">PDB: 4GUL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HLT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HLT','PDB','4HLT'); return false;">PDB: 4HLT</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i58"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01055">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_11656"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b01055">10.1021/acs.jmedchem.6b01055</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">NMR spectra of final compounds, CDK inhibition assay, Cancergene cell line profiling, DiscoveRx KinomeScan, kinase inhibition assays, BRAF inhibitors, solubilizing group optimization, mouse pharmacokinetics, in vivo mouse efficacy studies, Cerep Diversity Screen, SILAC target identification, surface plasmon resonance (SPR), pirin crystallography, full experimental procedures (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_001.pdf">PDF</a>)</p></li><li><p class="inline">SMILES molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_002.csv">CSV</a>)</p></li><li><p class="inline">1%DMSO WM266.4 migration assay VC (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_003.mov">MOV</a>)</p></li><li><p class="inline">Bisamide 26 WM266.4 migration assay 100 nM (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_004.mov">MOV</a>)</p></li><li><p class="inline">Bisamide 28 WM266.4 migration assay 100 nM (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_005.mov">MOV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_001.pdf">jm6b01055_si_001.pdf (12.6 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_002.csv">jm6b01055_si_002.csv (2.02 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_003.mov">jm6b01055_si_003.mov (553.04 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_004.mov">jm6b01055_si_004.mov (555.48 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01055/suppl_file/jm6b01055_si_005.mov">jm6b01055_si_005.mov (565.79 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and structure factors for the crystal structures of pirin with compound <b>26</b> can be accessed using PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JCT">5JCT</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-1%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b01055" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b01055" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799d9464bfa3d8e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
