
<html lang="en"     class="pb-page"  data-request-id="1c433bd0-9f7d-40ac-a0ed-f0b7a345cf01"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2021.64.issue-6;website:website:acspubs;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.0c02159;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Conformational Strain of Macrocyclic Peptides in Ligand–Receptor Complexes Based on Advanced Refinement of Bound-State Conformers" /></meta><meta name="dc.Creator" content="Alexander C.  Brueckner" /></meta><meta name="dc.Creator" content="Qiaolin  Deng" /></meta><meta name="dc.Creator" content="Ann E.  Cleves" /></meta><meta name="dc.Creator" content="Charles A.  Lesburg" /></meta><meta name="dc.Creator" content="Juan C.  Alvarez" /></meta><meta name="dc.Creator" content="Mikhail Y.  Reibarkh" /></meta><meta name="dc.Creator" content="Edward C.  Sherer" /></meta><meta name="dc.Creator" content="Ajay N.  Jain" /></meta><meta name="dc.Description" content="Macrocyclic peptides are an important modality in drug discovery, but molecular design is limited due to the complexity of their conformational landscape. To better understand conformational propen..." /></meta><meta name="Description" content="Macrocyclic peptides are an important modality in drug discovery, but molecular design is limited due to the complexity of their conformational landscape. To better understand conformational propen..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 16, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02159" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 The Authors. Published by American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02159" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02159" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02159" /></link>
        
    
    

<title>Conformational Strain of Macrocyclic Peptides in Ligand–Receptor Complexes Based on Advanced Refinement of Bound-State Conformers | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02159" /></meta><meta property="og:title" content="Conformational Strain of Macrocyclic Peptides in Ligand–Receptor Complexes Based on Advanced Refinement of Bound-State Conformers" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0014.jpeg" /></meta><meta property="og:description" content="Macrocyclic peptides are an important modality in drug discovery, but molecular design is limited due to the complexity of their conformational landscape. To better understand conformational propensities, global strain energies were estimated for 156 protein-macrocyclic peptide cocrystal structures. Unexpectedly large strain energies were observed when the bound-state conformations were modeled with positional restraints. Instead, low-energy conformer ensembles were generated using xGen that fit experimental X-ray electron density maps and gave reasonable strain energy estimates. The ensembles featured significant conformational adjustments while still fitting the electron density as well or better than the original coordinates. Strain estimates suggest the interaction energy in protein–ligand complexes can offset a greater amount of strain for macrocyclic peptides than for small molecules and non-peptidic macrocycles. Across all molecular classes, the approximate upper bound on global strain energies had the same relationship with molecular size, and bound-state ensembles from xGen yielded favorable binding energy estimates." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02159"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02159">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02159&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02159&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02159&amp;href=/doi/10.1021/acs.jmedchem.0c02159" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 3282-3298</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c02156" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c02191" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Conformational Strain of Macrocyclic Peptides in Ligand–Receptor Complexes Based on Advanced Refinement of Bound-State Conformers</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Alexander C. Brueckner</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alexander C. Brueckner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational & Structural Chemistry, Merck & Co Inc, 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#640508011c050a0001164a06161101070f0a011624090116070f4a070b09"><span class="__cf_email__" data-cfemail="8feee3eaf7eee1ebeafda1edfdfaeaece4e1eafdcfe2eafdece4a1ece0e2">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alexander+C.++Brueckner">Alexander C. Brueckner</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5866-0878" title="Orcid link">http://orcid.org/0000-0001-5866-0878</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qiaolin Deng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qiaolin Deng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational & Structural Chemistry, Merck & Co Inc, 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qiaolin++Deng">Qiaolin Deng</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6378-3872" title="Orcid link">http://orcid.org/0000-0001-6378-3872</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ann E. Cleves</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ann E. Cleves</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bioengineering and Therapeutic Sciences, University of California San Francisco, Box 0128, San Francisco, California 94158, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ann+E.++Cleves">Ann E. Cleves</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1622-2770" title="Orcid link">http://orcid.org/0000-0002-1622-2770</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Charles A. Lesburg</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Charles A. Lesburg</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational and Structural Chemistry, Merck and Co Inc, 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Charles+A.++Lesburg">Charles A. Lesburg</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7245-7331" title="Orcid link">http://orcid.org/0000-0001-7245-7331</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Juan C. Alvarez</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Juan C. Alvarez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational and Structural Chemistry, Merck and Co Inc, 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Juan+C.++Alvarez">Juan C. Alvarez</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mikhail Y. Reibarkh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mikhail Y. Reibarkh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Analytical Research and Development, Merck & Co Inc, 126 East Lincoln Avenue, Rahway, New Jersey 07065, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mikhail+Y.++Reibarkh">Mikhail Y. Reibarkh</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6589-707X" title="Orcid link">http://orcid.org/0000-0002-6589-707X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Edward C. Sherer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Edward C. Sherer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Analytical Research and Development, Merck & Co Inc, 126 East Lincoln Avenue, Rahway, New Jersey 07065, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Edward+C.++Sherer">Edward C. Sherer</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8178-9186" title="Orcid link">http://orcid.org/0000-0001-8178-9186</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Ajay N. Jain</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ajay N. Jain</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Bioengineering and Therapeutic Sciences, University of California San Francisco, Box 0128, San Francisco, California 94158, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#dabbb0bbb3b49ab0bbb3b4b6bbb8f4b5a8bd"><span class="__cf_email__" data-cfemail="6d0c070c04032d070c0403010c0f43021f0a">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ajay+N.++Jain">Ajay N. Jain</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4641-8501" title="Orcid link">http://orcid.org/0000-0003-4641-8501</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02159&amp;href=/doi/10.1021%2Facs.jmedchem.0c02159" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 3282–3298</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 16, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>14 December 2020</li><li><span class="item_label"><b>Published</b> online</span>16 March 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 March 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02159" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02159</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 The Authors. Published by American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="https://acsopenscience.org/open-access/licensing-options/"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by"><a href="https://creativecommons.org/licenses/by/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_cc-1612971093887.svg" alt="CC: Creative Commons" /></img><span><strong>CC: </strong>Creative Commons</span></a><a href="https://creativecommons.org/licenses/by/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_by-1612971093257.svg" alt="BY: Credit must be given to the creator" /></img><span><strong>BY: </strong>Credit must be given to the creator</span></a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02159"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1593</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02159" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Conformational Strain of Macrocyclic Peptides in Ligand–Receptor Complexes Based on Advanced Refinement of Bound-State Conformers&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Alexander&quot;,&quot;last_name&quot;:&quot;C. Brueckner&quot;},{&quot;first_name&quot;:&quot;Qiaolin&quot;,&quot;last_name&quot;:&quot;Deng&quot;},{&quot;first_name&quot;:&quot;Ann&quot;,&quot;last_name&quot;:&quot;E. Cleves&quot;},{&quot;first_name&quot;:&quot;Charles&quot;,&quot;last_name&quot;:&quot;A. Lesburg&quot;},{&quot;first_name&quot;:&quot;Juan&quot;,&quot;last_name&quot;:&quot;C. Alvarez&quot;},{&quot;first_name&quot;:&quot;Mikhail&quot;,&quot;last_name&quot;:&quot;Y. Reibarkh&quot;},{&quot;first_name&quot;:&quot;Edward&quot;,&quot;last_name&quot;:&quot;C. Sherer&quot;},{&quot;first_name&quot;:&quot;Ajay&quot;,&quot;last_name&quot;:&quot;N. Jain&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;16&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;3282-3298&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02159&quot;},&quot;abstract&quot;:&quot;Macrocyclic peptides are an important modality in drug discovery, but molecular design is limited due to the complexity of their conformational landscape. To better understand conformational propensities, global strain energies were estimated for 156 protein-macrocyclic peptide cocrystal structures. Unexpectedly large strain energies were observed when the bound-state conformations were modeled with positional restraints. Instead, low-energy conformer ensembles were generated using xGen that fit experimental X-ray electron density maps and gave reasonable strain energy estimates. The ensembles featured significant conformational adjustments while still fitting the electron density as well or better than the original coordinates. Strain estimates suggest the interaction energy in protein–ligand complexes can offset a greater amount of strain for macrocyclic peptides than for small molecules and non-peptidic macrocycles. Across all molecular classes, the approximate upper bound on global strain energies had the&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02159&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02159" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02159&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02159" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02159&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02159" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02159&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02159&amp;href=/doi/10.1021/acs.jmedchem.0c02159" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02159" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02159" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (10 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02159%26sid%3Dliteratum%253Aachs%26pmid%3D33724820%26genre%3Darticle%26aulast%3DBrueckner%26date%3D2021%26atitle%3DConformational%2BStrain%2Bof%2BMacrocyclic%2BPeptides%2Bin%2BLigand%25E2%2580%2593Receptor%2BComplexes%2BBased%2Bon%2BAdvanced%2BRefinement%2Bof%2BBound-State%2BConformers%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D6%26spage%3D3282%26epage%3D3298%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=290898" title="Energy">Energy</a>,</li><li><a href="/action/doSearch?ConceptID=292444" title="Molecular structure">Molecular structure</a>,</li><li><a href="/action/doSearch?ConceptID=291163" title="Macrocycles">Macrocycles</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/jmcmar.2021.64.issue-6/20210325/jmcmar.2021.64.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Macrocyclic peptides are an important modality in drug discovery, but molecular design is limited due to the complexity of their conformational landscape. To better understand conformational propensities, global strain energies were estimated for 156 protein-macrocyclic peptide cocrystal structures. Unexpectedly large strain energies were observed when the bound-state conformations were modeled with positional restraints. Instead, low-energy conformer ensembles were generated using xGen that fit experimental X-ray electron density maps and gave reasonable strain energy estimates. The ensembles featured significant conformational adjustments while still fitting the electron density as well or better than the original coordinates. Strain estimates suggest the interaction energy in protein–ligand complexes can offset a greater amount of strain for macrocyclic peptides than for small molecules and non-peptidic macrocycles. Across all molecular classes, the approximate upper bound on global strain energies had the same relationship with molecular size, and bound-state ensembles from xGen yielded favorable binding energy estimates.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53007" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53007" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein–protein associations are often mediated through large and relatively flat surfaces (1500–3000 Å<sup>2</sup>), to which drug-like small molecules (<500 Da) are often unable to bind tightly and selectively.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5 ref6">(1−6)</a> Peptides, however, can bind potently and selectively to these challenging drug targets due to their size and atomic composition. By mimicking endogenous bioactive molecules and protein secondary structure, a network of interactions between the peptide and the flat protein surface results in sufficient interaction energy to overcome major costs to binding.<a onclick="showRef(event, 'ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15">(7−15)</a></div><div class="NLM_p">Unfortunately, linear acyclic peptides suffer several disadvantages, such as weak binding affinity (large entropic penalty), low bioavailability, and poor <i>in vivo</i> stability<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> that have historically rendered them difficult to develop into marketed drugs. On the other hand, macrocyclic peptides represent a therapeutic modality that has the potential to combine the best properties of small molecule ligands (oral bioavailability and cell permeability) and antibodies (high specificity and the ability to target shallow or flat protein surfaces).<a onclick="showRef(event, 'ref4 ref9 ref16 ref17 ref18 ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref4 ref9 ref16 ref17 ref18 ref19 ref20 ref21 ref22">(4,9,16−22)</a> Macrocyclization of peptides constrains conformational space and imparts some structural preorganization, thereby decreasing the number of low-energy conformations.<a onclick="showRef(event, 'ref4 ref7 ref16 ref23 ref24 ref25 ref26 ref27 ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref4 ref7 ref16 ref23 ref24 ref25 ref26 ref27 ref28 ref29 ref30 ref31">(4,7,16,23−31)</a> This reduces the entropic penalty of binding and increases proteolytic stability compared to acyclic peptides.<a onclick="showRef(event, 'ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref32 ref33 ref34">(32−34)</a></div><div class="NLM_p">Although properties of macrocyclic peptides (so-called beyond Rule of 5; bRo5) lie outside the range of small molecule physical properties expected for “drug-likeness”,<a onclick="showRef(event, 'ref35 ref36 ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref37 ref38 ref39">(35−39)</a> they have gained substantial interest in pharmaceutical development.<a onclick="showRef(event, 'ref4 ref40 ref41 ref42 ref43 ref44 ref45 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref4 ref40 ref41 ref42 ref43 ref44 ref45 ref46 ref47">(4,40−47)</a> Ease of synthesis, large-scale library exploration, diversity,<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> and rapid screening capabilities (e.g., phage and mRNA display) have all been contributing factors to the increased pursuit of macrocyclic peptides as drug candidates. Increased adoption of peptides as a therapeutic modality in recent years is evidenced by steadily increasing numbers of publications,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> as well as several cyclic peptides entering clinical trials.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div><div class="NLM_p">Computational modeling has been widely adopted by pharmaceutical companies to aid drug design efforts.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> However, computational investigation of macrocyclic peptides is hampered by the challenges associated with exploring their conformational space. Efficient enumeration of conformers is difficult due to the constraints imposed by the closure of the macrocyclic ring systems, treatment of standard and nonstandard amino acids, incorporation of non-peptidic chemical content, and molecular size. Additionally, the vast number of possible conformers<a onclick="showRef(event, 'ref52 ref53 ref54 ref55 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref52 ref53 ref54 ref55 ref56 ref57">(52−57)</a> creates challenges in ensemble interpretation, identification of the relevant populations, and downstream calculations including prediction of drug-like attributes such as cell permeability, solubility, and other physicochemical properties.<a onclick="showRef(event, 'ref15 ref58'); return false;" href="javascript:void(0);" class="ref ref15 ref58">(15,58)</a></div><div class="NLM_p">Understanding the energetic strain window for peptidic macrocycles that can be compensated through interaction with protein targets would facilitate design efforts that employ conformationally-dependent modeling workflows. In the absence of experimental constraints on the bioactive conformer space, it remains a significant challenge to establish realistic limits on allowable conformational ensemble energy windows. A recent study reported by Zivanovic et al. quantified the global strain energy, defined as the difference between the global minimum solution-state and bound-state conformational energy, tolerated by drug-like small molecules using high-accuracy quantum mechanical calculations.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> They found that the global strain energy for a data set of 123 small molecules (200–700 Da) ranged from 0 to 7.1 kcal/mol, where 73% of the ligands fell below 1.8 kcal/mol. A similar understanding of global strain for macrocyclic peptides would allow conformational workflows to filter out high-energy conformers, which otherwise exceed the relevant threshold value. This significant reduction in conformational space would mitigate challenges in design and in computational complexity (both concerning time and hardware memory).</div><div class="NLM_p">Global strain energy values depend directly on the method used to estimate the energy of the bound-state and global minimum solution-state conformation. Here, we used an iterative implementation of ForceGen to locate the global minimum solution-state conformation as it is uniquely positioned to handle macrocyclic peptides by non-stochastically searching the conformational space with physical movements.<a onclick="showRef(event, 'ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref60 ref61">(60,61)</a> For bound-state ligand conformations, atomic coordinates from X-ray crystallographic models generally require some type of re-refinement to overcome small coordinate deviations that lead to erroneously high force field-based energy values.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> Historically, ligand heavy atoms have been restrained to their original positions with a square-welled quadratic positional penalty.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> However, Perola and Charifson’s influential study<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> facilitated the discussion around bound-state ligand modeling, including studies benchmarking modeling methodologies,<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> data set re-analyses,<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> and new techniques to better model the bioactive conformation including quantum mechanics/molecular mechanics (QM/MM)<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> and electron-density fitting.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a></div><div class="NLM_p last">Here, the newly introduced xGen<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> methodology is used to model the bound-state conformations and estimate the global strain energy for a data set of 156 macrocyclic peptide cocrystal structures. xGen uses a unique real-space refinement approach to identify low-energy conformer ensembles that fit the experimental electron density better than a single ligand conformer. The global strain energies and occupancy-weighted conformer ensembles resulting from the xGen procedure are compared to those of several other bound-state modeling methodologies and discussed in detail, including major implications for the design of macrocyclic peptidic drug candidates.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42564" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42564" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Macrocyclic Peptide Data Set Composition</h3><div class="NLM_p">A macrocyclic peptide data set containing 156 cocrystal structures with a total of 311 ligand-binding site pairs was curated (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Hydrolases make up nearly a quarter of the protein classifications represented, followed by those associated with the immune system (e.g., antigen-binding fragments—Fabs) and isomerases (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). The majority of the structures were deposited between 2010 and 2017, highlighting the increased interest in macrocyclic peptides as therapeutics in recent years (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B).<a onclick="showRef(event, 'ref4 ref40 ref41 ref42 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref4 ref40 ref41 ref42 ref43 ref44">(4,40−44)</a> Peptide sizes ranged from 4 to 24 residues with the bulk of the data set between 5 and 14 residues (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). This size range bridges the gap between small molecules (<500 Da) and therapeutic proteins (e.g., insulin; >3000 Da), agreeing with the bRo5 classification by Doak et al.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Crystal structure resolutions ranged from 1.0 to 3.0 Å with only a few structures outside this range (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D). Additional details can be found in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Composition of the macrocyclic peptide data set. See the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for a list of PDB IDs. (A) Classification of proteins. Only sectors >5% of the total population are labeled. Distribution of (B) deposited structures by year and (C) ligands by size. (D) Cumulative population of PDB structures within resolution cutoffs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Refining Modeled Bound-State Conformations Using Conformational Ensembles</h3><div class="NLM_p">When ligands bind to protein targets, some adopt conformations very similar to low-energy solution-state conformers (e.g., stapled peptides rigidified by design<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a>), while most require some degree of conformer reorganization, which can be characterized in terms of strain energy: the difference in energy between these two reference points. Calculated strain energy is dependent on the precise coordinates used to model the bound-state conformation. Owing to variations between force fields and techniques used to solve ligand-bound cocrystal structures, some form of atomic coordinate adjustment is required to accurately estimate strain energy. A widely used approach has been to restrain the ligand’s heavy atoms to their original positions with a square-welled quadratic positional penalty while performing a local minimization using a force field.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> For example, Perola and Charifson<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> used this method for a small molecule data set, using a 0.5 Å half-width with a 500 kcal/mol/Å<sup>2</sup> force constant. The nominal energy of the bound-state conformer is assigned based on the relaxed version, whose atomic deviations from the starting conformer are generally small.</div><div class="NLM_p">The implicit assumption underlying such an approach is that the bound-state ligand coordinates are inherently correct. A gross error in a modeled ligand conformation, for example, a 90° dihedral angle for an amide bond, will impart high strain energy, owing to positional restraint penalties. Peptidic macrocycles are among the most challenging ligands to fit into X-ray electron density, which raises concerns about strain estimates based on the nominal modeled ligand coordinates. The xGen real-space ligand refinement approach is integrated within the ForceGen conformational search procedures, which have been optimized for application to macrocyclic ligands.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> The method seeks to build conformer ensembles that are simultaneously low in energy and provide a better fit to the experimental electron density [real-space correlation coefficient (RSCC) and real-space residual (RSR)].</div><div class="NLM_p">In some cases, fitting the electron density perfectly with a single conformer results in large geometric distortions of the ligand, leading to artificially high-energy conformations. In contrast, xGen<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> first generates conformers that balance fit to the electron density and geometric distortions. The resulting conformers are re-minimized in two ways to form a “conformer trio”: one being the original conformer, one with a focus on fitting the electron density, and one with a focus on resolving geometric distortions. Conformer neighborhoods are then formed from a subset of the conformers in the resulting pool: those that fit the electron density up to a certain cutoff (90% of the maximum density fit), are within 3.0 kcal/mol of the minimum energy, and are geometrically similar to other neighborhood members (scaled root-mean-square deviation (rmsd) ≤ 0.65 Å). The final xGen conformer ensemble is optimized by selecting from the members of the conformer neighborhoods to fit real-space electron density. The ensemble is weighted by occupancy per conformer, without using B-factors to model atomic motion or uncertainty. The results of applying the xGen procedure on a small macrocyclic inhibitor of BACE1 are shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>. Note that overfitting can be a concern when modeling X-ray electron density with conformational ensembles, and this aspect was analyzed and discussed extensively in a method-focused study.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Conformational search and ensemble derivation. (A) All conformers resulting from a restrained search of the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DV1">3DV1</a> ligand, blending force field energetics with a quantitative reward for matching electron density. (B) Single high-quality conformer trio, representing both good fit to the density (orange) and low energy (yellow) along with a conformer with low rmsd to both (slate). (C) Occupancy-weighted conformer ensemble with the 1.0σ experimental density contour (gray mesh) and the corresponding calculated real-space density contour (cyan dots).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">xGen was applied using standard parameters on all 311 macrocyclic peptide binding sites (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for additional details). This process provided conformer ensembles with an average of 3.5 conformers per peptidic macrocycle. Just under one-fifth of the cases had a single conformer, the largest ensemble had 11 members, and around three-quarters fell into the 3–7 conformer range. The RSCC values improved marginally but consistently (average increase of 0.010 and <i>p</i> < 10<sup>–7</sup> by paired <i>t</i>-test), and the RSR values also improved consistently but very slightly (average decrease of 0.003 and <i>p</i> < 0.05 by paired <i>t</i>-test). The improvement in RSCC and RSR values indicates that nearly all the xGen conformer ensembles fit the experimental electron density as well, or better than, the deposited structures.</div><div class="NLM_p">The qualitative difference between the deposited ligand coordinates of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O4Y">5O4Y</a> (green) and the 5-member xGen conformer ensemble (orange) is illustrated in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>. The positions labeled 1–3 each show places where the deposited ligand contained a <i>cis</i>-amide, where the optimal xGen ensemble produced a lower energy <i>trans</i>-amide within the macrocyclic ring backbone. The xGen <i>trans</i>-amide 2 also introduced a conformational change in the neighboring thioether linkage. While one might observe experimental support for <i>cis</i>-amide configurations (e.g., assay data for N-methylated variants), no such data were provided in the original publication, and it seems likely that these high-energy amide geometries simply resulted from the difficulty in fitting large macrocyclic peptides into X-ray density using commonly available tools.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> The positions labeled 4–6 each show side chain rotameric variations. As will be discussed later, both the deposited single-conformer model and the alternative ensemble fit the electron density equally well, but the conformers comprising the latter model yield much lower strain estimates. Global strain estimation based on the original model of the bound-state ligand produces erroneously high values, even when making use of standard approaches for local relaxation.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Example of alternative fits to electron density for<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O4Y">5O4Y</a>. The atomic coordinates of the deposited ligand model are shown in green sticks, with a set of five conformers comprising an xGen ensemble shown in orange. The electron density contour from the 2|Fo| – |Fc| map is shown at 1.0σ. Red numbers 1–3 mark a position where a high-energy <i>cis</i>-amide in the deposited coordinates is flipped to a low-energy <i>trans</i>-amide in the xGen ensemble. Red numbers 4–6 show alternative side chain rotamers in the xGen ensemble compared to the deposited coordinates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In addition to the standard square-welled quadratic restraint and xGen methods, two additional methods for ligand refinement were studied for improving strain estimates. Each placed a square-welled quadratic restraint on heavy-atom positions to hold them near the original modeled atomic coordinates (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for additional details). Rather than applying the same half-widths to all heavy atoms, for one (termed “B-factor binning”) the half-widths of the flat wells were set based on three classes for atomic B-factors: low, medium, and high uncertainty. For the other (termed “coordinate uncertainty”) the half-widths were set based on explicitly calculated atomic-coordinate uncertainty. The diffraction precision index is a global measure of the precision of atomic positions from experimental crystallographic parameters, from which the coordinate error of each atom can be calculated using its B-factor.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Defining the Global Minimum Conformation</h3><div class="NLM_p">Exploring conformational space for peptide macrocycles is challenging and identifying the true global minimum energy value under any force field is computationally infeasible, as doing so requires exhaustive search at relatively fine granularity in dihedral angle space. Here, an estimate for the global minimum was determined using an iterative application of the fast and accurate ForceGen method,<a onclick="showRef(event, 'ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref60 ref61">(60,61)</a> beginning from two different input conformers for each ligand. The first conformer was defined by the deposited crystal structure ligand coordinates while the second was a randomized conformation (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for additional details).</div><div class="NLM_p last">The nominal global minima identified using this procedure is limited by the accuracy of the MMFF94sf force field and the thoroughness with which the conformational landscape is traversed. The lower bound on success in finding true global minima can be estimated based on the relationship between total molecular flexibility (the number of rotatable macrocyclic ring bonds plus the total number of rotatable exocyclic bonds) and success in identifying close matches to experimentally determined ligand conformations (either bound to proteins or within small-molecule crystal structures). Beginning from a randomized input conformer, the previously documented success rate for identifying a deposited ligand conformer (with 2.0 Å rmsd) with ForceGen was over 90% for cases with total flexibility up to 36,<a onclick="showRef(event, 'ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref60 ref61">(60,61)</a> which accounts for half of the current data set. For the remaining half, the success rate was slightly under 60%. Overall, given the iterative and more thorough procedure employed here than in the original benchmark studies, the lower bound for which estimates of global minima will be accurate is projected to be 75–85%. Note, however, the central practical goal of this study is to identify energy windows for conformational exploration, where the low-energy reference point is defined by the specific application and procedure of the conformational search to locate the global minima. In practical (non-iterative) applications of the ForceGen search procedure, the size of the energy windows will likely be larger than required as the extensive procedure employed here is expected to find the global minimum more often.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Strain Energy of Macrocyclic Peptides</h3><div class="NLM_p">The procedure described above for identifying a global minimum conformation produces a single final conformer and associated energy. A complication with such a procedure is that force field energy landscapes are notoriously frustrated surfaces, and it is possible to find singularities where small changes in coordinates lead to large changes in energy. To mitigate this issue, a Boltzmann-weighted ensemble energy value was calculated by a stochastic sampling of small coordinate perturbations followed by minimization (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details). This increased the estimate of the global minimum energy by an average of 0.9 kcal/mol across the 156 individual complexes. The largest such change was 4.5 kcal/mol.</div><div class="NLM_p">The procedure for identifying optimal bound-state conformer ensembles produces a small number of conformers, each of which is somewhat biased toward fitting electron density rather than minimizing internal energy. Recall, however, that these conformers are part of neighborhoods in which another neighborhood member was biased toward low energy (see <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, yellow conformer). We applied the same Boltzmann-weighted ensemble procedure to the collection of low-energy biased conformers with a square-welled quadratic restraint on heavy atoms to prevent excessive deviation from the original ensemble. In each case, the smallest rmsd of any conformer in the final Boltzmann-weighted ensemble to a conformer within the xGen ensemble was noted to quantify the distance between the locally optimal minimum and a conformer with direct experimental support from the X-ray data.</div><div class="NLM_p"><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> (blue lines) shows the cumulative histograms of rmsd from the crystallographic support (A) and strain energy (B) for the xGen ensembles. The deviations from the xGen ensembles by the Boltzmann-weighted minima were typically 0.25 Å (90th percentile rmsd = 0.45 Å). Median strain energy was 8.0 kcal/mol (90th percentile strain energy = 21.5 kcal/mol). A truncated data set where duplicate peptides were removed (e.g., cyclosporin) contained 116 cocrystal structures compared with 156 in the full data set. The median and the 90th percentile strain energy with the xGen method in the truncated data set shifted from 8.0 to 10.0 and 21.5 to 24.5 kcal/mol, respectively, indicating that duplicate peptides in the data set did not bias the results to a meaningful degree. The widely used square-welled restraint approach for ligand coordinate relaxation was also employed beginning from the originally modeled PDB ligand coordinates (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details), with results shown in red-dotted lines in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>. The quadratic restraint approach on deposited coordinates yielded slightly larger deviations than seen with the xGen method, but 90% of the deviations were 0.45 Å or less. However, strain energy values using the standard approach were roughly double the magnitude of those seen using the xGen ensembles (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). The fraction of cases with strain values of 20.0 kcal/mol or higher was 29%, compared with just 12% for the xGen ensembles. The changes in estimated strain magnitude were large enough to have clear practical implications, and the changes were also consistent (90th percentile global strain = 21.5 vs 32.5 kcal/mol, respectively; paired <i>t</i>-test, <i>p</i> < 10<sup>–12</sup>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Deviation from the crystallographic experimental support for the macrocyclic peptide data set. (B) Cumulative histogram of global strain energy. Red-dotted lines are for the square-welled quadratic positional restraint approach. Blue solid lines are for the xGen electron density fitting approach. Yellow dot-dashed lines are for the B-factor binning approach. Gray-dashed lines are for the coordinate uncertainty approach. The xGen electron density fitting retained high fidelity to the crystallographic data while producing the lowest strain estimates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The B-factor binning approach (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>; yellow dot-dashed lines) produced slightly lower strain estimates than the xGen ensemble approach; however, the magnitude of deviation from the modeled bound-state coordinates was more than twice as high. The coordinate uncertainty approach (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>; gray-dashed lines) produced larger structural deviations than either the xGen ensemble or standard positional restraint methods (though less than the B-factor binning approach), but it yielded strain estimates midway between those of the xGen and the standard positional restraint approaches. The xGen method re-fits the electron density with chemically sensible conformational ensembles, and it produced the lowest strain estimates of the approaches that retained high fidelity to the crystallographic data.</div><div class="NLM_p last">The case illustrated in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> is one of several cases where ligands with extremely large strain energies estimated from the standard positional restraint method (>100 kcal/mol) had significantly reduced strain energy estimates derived from the xGen methodology. The deposited coordinates for the macrocyclic peptide ligand in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O4Y">5O4Y</a> (15-residues; MW = 1869.2 Da) fit the electron density well (RSCC = 0.92). However, when the square-welled quadratic positional restraint approach was applied, the strain energy was 111.2 kcal/mol, both an outlier and a physically unrealistic value. The xGen ensemble fits the electron density as well as the deposited coordinates (RSCC = 0.92). However, the strain energy dropped nearly 10-fold to 11.6 kcal/mol. Additional qualitative analysis of the xGen conformer ensembles is described below, including a detailed discussion on the major conformational changes that led to the dramatic energy change in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O4Y">5O4Y</a>.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Effects of Force Field and Dielectric Constant</h3><div class="NLM_p">The ForceGen and xGen methods employ a variant of the MMFF94s force field, typically applied using a dielectric constant consistent with an aqueous solution (80.0).<a onclick="showRef(event, 'ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref60 ref61">(60,61)</a> Because the macrocyclic search procedure makes use of force field calculations as a tightly coupled aspect of the search process, it was not possible to test other force fields in a directly comparable manner. As an approximation, OPLS3e was used to locally optimize the xGen ensembles (with a positional restraint) and the global minimum conformer pools (without a positional restraint), with the difference between the minima from the two being an alternative estimate of the global strain energy. While the OPLS3e force field yielded slightly higher strain estimates, the magnitude and trends were similar.</div><div class="NLM_p last">Solvent-exposed ligands experience a range of dielectric constants,<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> from bulk solvent to the peptidic target background, which influences conformational energies. Within the macrocyclic peptide data set, there is a wide variation in the degree of binding-site contact area, which would suggest modeling the bound ligand states using a lower dielectric constant. On a randomly selected subset for 10% of cases, strain estimates were calculated with a series of different dielectric constants (ranging from 2.0 to 20.0) for the bound ligand states, with the energy for the global minima unchanged (reflecting the nominal solution conformational ensembles). While the results varied, generally, the bound conformation energy estimates decreased with decreasing dielectric values. However, given the large variation in protein–ligand contact areas, all-atom simulations in an explicit solvent would appear to be a better approach, and these are planned for future work.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Conformational Strain for Different Molecular Classes</h3><div class="NLM_p">The xGen electron density fitting procedure was applied to a non-peptidic macrocycle set and a small molecule set to contextualize macrocyclic peptide strain energy with other molecular classes (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details on the latter two data sets). Analogous plots to those shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> for the xGen results but with data from all three data sets included are in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>. The rmsd distribution was nearly identical for all data sets despite the diversity in size, topology, and chemical composition of the different molecular classes (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). However, strain energies were observed to have significantly larger values for some of the macrocyclic peptides compared to the non-peptidic macrocycles, or small molecules (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, vertical lines; 90th percentile = 21.5 vs 7.0 vs 5.0 kcal/mol, respectively). Note that this is also true when normalized by the heavy atom count (HAC), with the median strain per HAC being roughly 0.05–0.06 kcal/mol/atom for the small molecules and non-peptidic macrocycles but roughly double that for the peptide macrocycles. Importantly, not all peptide macrocycles have high strain. Rather, the allowable strain energy range is broader for this class of compounds, and, as will be discussed, can be balanced by favorable intermolecular interactions with the protein.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Deviations from the xGen ensembles by the Boltzmann-weighted minima were identical for all molecular classes. (B) Cumulative histogram of strain energy. Blue solid lines are for the macrocyclic peptide data set, green-dotted lines are for the non-peptidic macrocycle data set, and the purple-dashed lines are for the small molecule data set. All results are obtained from the electron density fitting approach (xGen). Vertical lines correspond to the 90th percentile for each data set, colored respectively. Strain energy estimates suggest the interaction energy in protein–ligand complexes can offset a greater amount of strain for macrocyclic peptides than for non-peptidic macrocycles or small molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">For all three molecular classes, a lower-right triangular distribution was observed when the global strain energies were plotted against HAC (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). More than 95% of the strain energies fit under a linear upper bound [<i>y</i> = 0.3 × (HAC-10)] and a lower bound of 0. Note that strain values calculated using the Boltzmann-weighted approach can be slightly negative (up to −0.6 kcal/mol) when the members of the conformational ensemble each have energy close to the global minimum. This simple relationship provides a means to quickly estimate the allowable energy window required for a thorough conformational search to identify conformers close to the bound-state ensemble. For example, if a ligand has 120 heavy atoms, the bound-state conformation is estimated to be within 33 kcal/mol of the global minimum (<i>y</i> = 0.3 × (120 – 10) = 33 kcal/mol).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Relationship between HAC and global strain energy featuring a lower-right triangular distribution. Blue squares are for the macrocyclic peptide data set, green circles are for the non-peptidic macrocycle data set, and purple triangles are for the small molecule data set. The black-dashed line is an approximate upper bound of the estimated strain energy.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The HAC range where molecules from all three data sets overlap (HAC of 65 or less) can be captured by an energy window of 16.5 kcal/mol, though very few examples exceeded the common conformational search energy window default value of 10.0 kcal/mol (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). Within this size range, there does not appear to be a distributional difference in strain energy as it relates to the molecular size, regardless of molecular class. Above 65 heavy atoms, only peptidic macrocycles remain, and the size range of 66–110 was well populated in the data set, exhibiting a similar lower-right triangular character to that with the smaller ligands. For the largest examples, with more than 110 heavy atoms, it is not clear whether the lack of a population with very low strain energy is due to the scarcity of ligands in this size range or some other effect.</div><div class="NLM_p last">The relationship between global strain and HAC was highly statistically significant as measured by Kendall‘s τ<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> (<i>p</i> ≪ 0.001) whether considering the entire set (341 cases, τ = 0.58), the small molecule set (38 cases, τ = 0.46), the non-peptide macrocycles (147 cases, τ = 0.37), or just the peptide macrocycles (156 cases, τ = 0.44). HAC is very strongly correlated with molecular weight, and it is also correlated with other characteristics such as the total atom count or the number of rotatable bonds. Particularly for the peptide macrocycle class, HAC is also correlated with the number of hydrogen-bond donors and acceptors. Due to its simplicity and the invariance of the relationship to global strain across molecular classes, HAC is preferable to other surrogates for estimating the energetic window size given a molecule.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Structural Characteristics of the Modeled Bound-State Conformational Ensembles</h3><div class="NLM_p">Several striking differences between the deposited coordinates and the occupancy-weighted conformer ensembles from xGen were observed. For 81% of the data set, the experimental electron density was better described by the inclusion of more than one conformer (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A). <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H0I">1H0I</a> contains a 5-residue macrocyclic peptide bound to chitinase B. The macrocycle in the deposited structure and final xGen ensemble overlay nearly perfectly—the only difference is an alternative side chain rotamer for a second conformer in the xGen ensemble (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B). Both the deposited structure and the xGen ensemble fit the electron density well, with the xGen ensemble fitting marginally better (RSCC = 0.89 and 0.92, respectively). The crystal structure for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MF8">1MF8</a> (an 11-residue macrocyclic peptide bound to cyclophilin) was solved at 3.10 Å resolution. The lower resolution is reflected in less detail in the electron density and leads to an ensemble of nine conformers better explaining the electron density than the single deposited conformer (RSCC = 0.89 and 0.80, respectively). For the 19% of xGen ensembles containing a single conformer, the distribution of crystal structure resolution and peptide size was nearly identical to the full data set; the single conformer ensembles were not clearly indicative of any particular type of example.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Number of conformers in the xGen ensembles. (A) Roughly 80% of the data set had more than one conformer in the final xGen ensemble. (B) Overlay of the deposited ligand conformer for<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H0I">1H0I</a> (green) and the two alternative conformers in the xGen ensemble (orange). (C) Overlay of the deposited ligand conformer for<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MF8">1MF8</a> (green) and the nine alternative conformers in the xGen ensemble (orange). Isosurface contour for ρ<sub>calc</sub> corresponding to 1.0σ for the xGen ensemble is shown as cyan dots. <i>E</i><sub>pos. res.</sub> and <i>E</i><sub>xGen</sub> are the estimated global strain energies for the positional restraint and density fitting methodologies, respectively. Energies are in kcal/mol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Not only do the xGen ensembles account for the electron density as well or better than the deposited conformers, but they also offer additional possibilities for ligand design. In 27% of the data set, backbone residues in the xGen ensemble were flipped relative to the deposited conformer coordinates (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A). Residue flipping produces alternative conformations that present different modification vectors and can, therefore, offer additional design opportunities. The consequence could be even more detrimental if a less relevant (or incorrect) conformer was suggested from standard ligand fitting methods and subsequent design ideas were misinformed.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Backbone residue flipping. (A) 27% of the data set featured a backbone residue flip in the final xGen ensemble relative to the deposited conformer. (B) Overlay of the deposited ligand conformer for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O4Y">5O4Y</a> (green) and the five alternative conformers in the xGen ensemble (orange). (C) Detailed view of a selection of backbone residues in the deposited conformer featuring <i>cis</i>-amide 1 from <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> (red text). (D) Detailed view of the same residues in the final xGen ensemble, now featuring a <i>trans</i>-amide (green text) and a new intramolecular hydrogen bond. Isosurface contour for ρ<sub>calc</sub> corresponding to 1.0σ for the xGen ensemble is shown as cyan dots. <i>E</i><sub>pos. res.</sub> and <i>E</i><sub>xGen</sub> are the estimated global strain energies for the positional restraint and density fitting methodologies, respectively. Distances are in Å and energies are in kcal/mol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">For example, the xGen ensemble for the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O4Y">5O4Y</a> ligand discussed earlier features five backbone residues in a different isomeric state relative to the deposited conformation (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B). In the deposited conformer, the backbone features a high energy <i>cis</i>-amide where the nitrogen points outside of the macrocyclic ring (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>C; amide 1 in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). In the xGen ensemble, the same backbone residue is flipped to a low energy <i>trans</i>-amide where the nitrogen now engages in a new trans-annular hydrogen bond (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>D). Although the deposited conformer and the xGen ensemble represent equivalent numerical fits to the electron density (RSCC = 0.92 for both), the xGen ensemble features conformers that are vastly lower in energy and differentially informs ligand design. Although <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O4Y">5O4Y</a> contains several <i>N</i>-methylated residues, the <i>cis</i>-amide nitrogen from the deposited ligand model is solvent-exposed and could be proposed as another site for N-methylation if improving permeability was of interest. However, the xGen ensemble shows the same residue participating in an intramolecular hydrogen bond where N-methylation at this position would disrupt the bound-state configuration and be detrimental to binding affinity. In contrast, targeting a cross-link to enforce this conformation may improve both potency and permeability.</div><div class="NLM_p">In total, 49% of the data set featured non-peptidic linkers to close the macrocyclic ring (i.e., disulfide bonds, alkyl chains, etc.). Of this subset, 44% of the xGen ensembles had alternative linker rotamers to the deposited conformers (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A). This was a surprisingly large portion of the ligands. However, the linker atoms were generally solvent-exposed in the bound state, and often had the largest B-factors, implying a large degree of structural disorder. xGen’s approach is to explicitly model the alternate low-energy conformational variants, leading to the large percentage of alternative linker rotamers observed.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Alternative linker rotamers. (A) 44% of the data set with non-peptidic linkers had alternative linker rotamers. (B) Overlay of deposited ligand conformer for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZQW">4ZQW</a> (green) and the five alternative conformers in the xGen ensemble (orange). (C) Overlay of the deposited ligand conformer for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWM">1VWM</a> (green) and the two alternative conformers in the xGen ensemble (orange). Major linker rotamers are noted in the red text. Isosurface contour for ρ<sub>calc</sub> corresponding to 1.0σ for the xGen ensemble is shown as cyan dots. <i>E</i><sub>pos. res.</sub> and <i>E</i><sub>xGen</sub> are the estimated global strain energies for the positional restraint and density fitting methodologies, respectively. Energies are in kcal/mol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">For example, the xGen final ensemble for the ligand in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZQW">4ZQW</a> (a 13-residue macrocyclic peptide bound to immunity protein CdiI) features five alternative conformers where much of the difference from the deposited coordinates is in the alkyl linker (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B). Alternative positions for the neighboring amide hydrogen bond donor and acceptor groups are a direct result of the linker rotamers, with major implications for protein–ligand or solvent–ligand interactions. An example where the xGen ensemble dramatically improves the fit to the electron density is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWM">1VWM</a>, a 6-residue macrocyclic peptide bound to streptavidin solved at a high resolution (1.60 Å). The ligand in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWM">1VWM</a> features a disulfide linker with two alternative rotamers in the xGen ensemble. The linker’s deposited coordinates place the acetyl group on the N-terminus outside the electron density envelope. The two alternative xGen conformers place the disulfide linker in different regions of the electron density, allowing the acetyl group to fit back within the electron density. This leads to a significantly better fit to the electron density overall for the xGen ensemble (RSCC = 0.63 and 0.77, respectively). It is important to note that, besides improving electron density fit, the calculated strain energy was also significantly reduced from 14.3 to 7.2 kcal/mol for the positional restraint method and xGen method, respectively. Disulfide linkers—a common motif in therapeutic cyclic peptides—were often found in a high-energy conformation in the deposited coordinates. The <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWM">1VWM</a> ligand example demonstrates that re-refining ligands with xGen yields much lower-energy disulfide linker conformers while improving the fit to electron density.</div><div class="NLM_p">Finally, in 90% of the data set, alternative macrocycle side chain rotamers were observed in the final xGen ensembles (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A). Side chains are often directly involved in interactions with the protein, so this frequency of alternative conformers was notable with major implications for drug design. Hydrophobic groups were among the most prevalent to have alternative rotamers in the data set. The xGen ensemble for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1C5F">1C5F</a> (an 11-residue macrocyclic peptide bound to cyclophilin) explains the electron density equally well as the deposited coordinates (RSCC = 0.94 and 0.93, respectively). However, the xGen ensemble with three alternative conformations shows rotamers for the Leu and Bmt. In terms of ligand design, hydrophobic group rotamers may be in solvent-exposed regions or may contact flexible regions of the protein target. Although less common, hydrophilic groups were also observed to have alternative rotamers. <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X6S">4X6S</a> is an 11-residue macrocyclic peptide bound to GRB7. The xGen six-conformer ensemble explained the electron density equally well to the deposited coordinates (RSCC = 0.88 for both). However, several hydrophilic side chains featured alternative rotamers in the xGen ensemble, including the C-terminal amide, the carboxyphenylalanine, and the Trp side chain (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). The Trp side chain is particularly interesting in that while all rotamers fit the electron density, there is a rotamer-dependent hydrogen bond donor, and the implications for protein–ligand or solvent–ligand interactions are significant. These additional rotamers may influence the design of viable variants and impart a better understanding of the thermodynamics of binding.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Alternative side chain rotamers. (A) 90% of the data set had alternative side chain rotamers. (B) Overlay of the deposited ligand conformer for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1C5F">1C5F</a> (green) and the three alternative conformers in the xGen ensemble (orange). (C) Overlay of the deposited ligand conformer for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X6S">4X6S</a> (green) and the six alternative conformers in the xGen ensemble (orange). Major side chain rotamers are noted in the red text. Isosurface contour for ρ<sub>calc</sub> corresponding to 1.0σ for the xGen ensemble is shown as cyan dots. <i>E</i><sub>pos. res.</sub> and <i>E</i><sub>xGen</sub> are the estimated global strain energies for the positional restraint and density fitting methodologies, respectively. Energies are in kcal/mol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Although the xGen methodology, in its current implementation, does not directly consider the protein, better intermolecular interactions were often observed when the aforementioned conformational adjustments were compared to the deposited ligand coordinates. For example, the deposited 16-residue macrocyclic peptide ligand in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5B4W">5B4W</a> fits the electron density well (RSCC = 0.91). The arginine side chain featured one strong and one weak intermolecular hydrogen bond with the Asp414 residue of the Plexin B1 protein target (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>B; yellow- and red-dotted lines, respectively). The xGen ensemble, generated in the absence of Plexin B1, fits the electron density equally well (RSCC = 0.91) but features an arginine rotamer that engages with the Asp414 residue with two strong hydrogen bonds (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>C; yellow-dotted lines). Looking at the deposited coordinates from a design perspective, modifications to enhance the weak hydrogen bond could be pursued. However, the xGen ensemble shows that such modifications may not be necessary, and efforts should be directed toward other regions of the peptide.</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Alternative intermolecular interactions. (A) Overlay of the deposited ligand conformer for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5B4W">5B4W</a> (green) and the three alternative conformers in the xGen ensemble (orange) in the crystal structure. Plexin B1 is shown in purple. (B) Detailed view of the deposited peptide arginine side chain and Plexin B1 Asp414 featuring one strong and one weak hydrogen bond (yellow- and red-dotted lines, respectively). (C) Detailed view of the same residues in the final xGen ensemble, now featuring an arginine side chain rotamer with two strong intermolecular hydrogen bonds (yellow-dotted lines). Isosurface contour for ρ<sub>calc</sub> corresponding to 1.0σ for the xGen ensemble is shown as cyan dots. <i>E</i><sub>pos. res.</sub> and <i>E</i><sub>xGen</sub> are the estimated global strain energies for the positional restraint and density fitting methodologies, respectively. Distances are in Å and energies are in kcal/mol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">More broadly with respect to the structural characteristics of macrocyclic peptides, the comprehensive review by Malde et al.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> points out that there are a substantial number of reported macrocyclic peptide structures for which ϕ/ψ angles deviate widely from well-established expectations. Indeed, the case of 2BCD described earlier actually contained a 90° Ω angle, which was remedied through the xGen refinement procedure. Given the difficulty of exploring different explanations to X-ray density fitting for large peptide macrocycles, making use of systematic search and fitting procedures such as those presented here are suggested for cases in which aberrant geometries are identified. A comprehensive analysis of ϕ/ψ angle distributions is beyond the scope of the present study, but such an analysis is enabled by the data archive associated with this paper (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c02159/suppl_file/jm0c02159_si_001.pdf" class="ext-link">Supporting Information</a>).</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Global Strain Energy Compared to Estimated Binding Enthalpy</h3><div class="NLM_p">The foregoing discussion of energetics has focused exclusively on internal ligand strain induced by the movement from a solvated environment to one bound to a protein. Strain energy estimates for all three molecular classes decreased using the xGen ensemble approach compared with deposited ligand model coordinates. Apart from the practical issue of estimating the size of energy windows for conformational search, there remains a basic thermodynamic question: are the estimated strain energy values small enough such that the energy of protein–ligand association can overcome the cost of pushing the ligands away from their energetically preferred conformations?</div><div class="NLM_p">To estimate the net binding enthalpy, the xGen ensembles were subjected to local optimization within their cognate binding sites using a scoring function for molecular docking (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details). <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a> shows the cumulative histograms of the difference between the optimized intermolecular interaction energy values (in kcal/mol) and the strain values, both derived from the final xGen conformer ensembles. For the non-macrocyclic small molecules and the non-peptide macrocycles, close to 90% of cases had net energy values of −10 kcal/mol or lower, with all cases but a handful of the non-peptide macrocycles producing negative values.</div><figure id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Cumulative histogram of enthalpy. Blue solid lines are for the macrocyclic peptide data set, green-dotted lines are for the non-peptidic macrocycle data set, and the purple-dashed lines are for the small molecule data set. All results are from docking the electron density fitting approach ensembles (xGen).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">However, 10% of the peptide macrocycle cases yielded positive values. Upon further inspection, most positive enthalpy cases arose from multiple neighboring proteins or biological assemblies in the larger unit cell binding with the ligand. The range of −10 to 0 kcal/mol showed a deviation between the enthalpy estimate for the macrocyclic peptides compared with the other two data sets. As with the positive enthalpy cases, some were affected by binding with multiple neighboring proteins or biological assemblies in the larger unit cell. Overall, there was a weak correlation between estimated enthalpy and strain energy (<i>R</i><sup>2</sup> = 0.33) across the combined data set, as would be expected given that both quantities were related to the ligand size. Perhaps most remarkably, nearly 70% of the peptide macrocycle cases followed essentially the same distribution of net estimated enthalpy as the other two molecular classes, with a value of −10 kcal/mol or better.</div><div class="NLM_p last">The xGen-based enthalpy estimates were consistently more favorable than those derived from the deposited crystallographic ligand coordinates: a net improvement of 2.2 kcal/mol from the small molecules (<i>p</i> < 10<sup>–4</sup> by paired <i>t</i>-test), 2.0 kcal/mol for non-peptide macrocycles (<i>p</i> < 10<sup>–8</sup>), and 6.6 kcal/mol for the peptide macrocycles (<i>p</i> < 10<sup>–11</sup>). Nearly 30% of the peptide macrocycle examples had net positive estimated enthalpy when beginning from the deposited ligand coordinates.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72387" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72387" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The xGen methodology was used to refine bound-state conformations of macrocyclic peptide cocrystal structures by building conformational ensembles that are low-energy and fit experimental electron density, ultimately providing more explicit possibilities for ligand design. Iterative conformational searches were used to identify the ForceGen global minimum conformations. The resulting estimates of global strain were substantially lower than those obtained using the originally deposited ligand coordinates, and they provide an upper-bound on ligand strain, applicable across molecular classes, that depends only on the molecular size as approximated by HAC. Applying xGen to deposited macrocyclic peptide coordinates resulted in conformational adjustments for the peptide backbones in 27%, the non-peptidic linkers in 44%, and the side chains in 90% of the data set. Estimates of binding energy revealed that the strain values were low enough in nearly all cases to yield net favorable values for the difference between intermolecular enthalpy and ligand strain.</div><div class="NLM_p last">Computational enumeration of conformational space is ubiquitous in computer-aided drug design, and it poses serious challenges for macrocyclic peptides with their vast conformational landscapes. This work offers a simple-to-calculate estimate for the maximal strain to be incurred for a macrocycle of a given size. Additionally, the broader workflow described here may help to address the enduring problem of understanding permeability and other properties of macrocyclic peptides, as extensive efforts have gone into studying the relationship between their permeability and conformational flexibility.<a onclick="showRef(event, 'ref1 ref7 ref14 ref15 ref24 ref48 ref49 ref74 ref75'); return false;" href="javascript:void(0);" class="ref ref1 ref7 ref14 ref15 ref24 ref48 ref49 ref74 ref75">(1,7,14,15,24,48,49,74,75)</a></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45038" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45038" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The data archive for this paper contains all molecular structural data and detailed computational procedures underlying the foregoing results (available to researchers at <a href="https://www.jainlab.org" class="extLink">https://www.jainlab.org</a> among the downloadable data sets as the “Peptide Strain Archive”). Here, descriptions of newly curated data and key procedures will be presented, with additional details present within the data archive. Crystal structures were obtained from the Protein Data Bank (PDB).<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> All compounds described in this study have been previously disclosed and have been characterized by X-ray crystallographic experiments, with details regarding purity available in the original publications or associated with the RCSB PDB deposition codes.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Molecular Data Sets</h3><div class="NLM_p">The review by Malde et al.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> in 2019 provides a good overview of available macrocyclic peptide cocrystal structures with protein targets. After extraction of all structures referenced in the review, 156 cocrystal structures of macrocyclic peptides complexed with protein targets with available density map coefficients were identified. Duplicate ligands across different PDB codes were retained in the data set given that different conditions were often used in the crystallization processes (different proteins, pH, etc.). Of the structures, 77 of the PDBs contained multiple copies of the ligand in the asymmetric unit. Each copy was extracted, providing 311 complexes in the data set. The strain energies reported represent the minimum strain calculated across the asymmetric unit copies. The list of macrocyclic peptide-containing PDB entries is:<ul class="NLM_list-list_type-label" id="list1"><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BM2">1BM2</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BZH">1BZH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1C5F">1C5F</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1E4W">1E4W</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H0G">1H0G</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H0I">1H0I</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1IKF">1IKF</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M63">1M63</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MF8">1MF8</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PCG">1PCG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QNG">1QNG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QNH">1QNH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QR3">1QR3</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1URC">1URC</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWB">1VWB</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWC">1VWC</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWD">1VWD</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWE">1VWE</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWF">1VWF</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWG">1VWG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWH">1VWH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWM">1VWM</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWN">1VWN</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWO">1VWO</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWP">1VWP</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1W9U">1W9U</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1W9V">1W9V</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1WAW">1WAW</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1WB0">1WB0</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1XQ7">1XQ7</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YIT">1YIT</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2BCD">2BCD</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2C9T">2C9T</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CV3">2CV3</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ESL">2ESL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2F58">2F58</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OJU">2OJU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2POY">2POY</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VDN">2VDN</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WFJ">2WFJ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X2C">2X2C</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X7K">2X7K</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z6W">2Z6W</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AV9">3AV9</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVA">3AVA</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVB">3AVB</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVC">3AVC</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVF">3AVF</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVG">3AVG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVH">3AVH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVI">3AVI</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVJ">3AVJ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVK">3AVK</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVL">3AVL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVM">3AVM</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVN">3AVN</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BO7">3BO7</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EOV">3EOV</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MGN">3MGN</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OE0">3OE0</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3P8F">3P8F</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PMP">3PMP</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP4">3PP4</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PQZ">3PQZ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QN7">3QN7</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VVR">3VVR</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VVS">3VVS</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WBN">3WBN</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WDC">3WDC</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WDD">3WDD</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WDE">3WDE</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WMG">3WMG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WNE">3WNE</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WNF">3WNF</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WNG">3WNG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WNH">3WNH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DGC">4DGC</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GLY">4GLY</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GVU">4GVU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HGC">4HGC</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HY7">4HY7</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IB5">4IB5</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IPZ">4IPZ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JJM">4JJM</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K8Y">4K8Y</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KEL">4KEL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTS">4KTS</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTU">4KTU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L3O">4L3O</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4M1D">4M1D</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MNX">4MNX</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MNY">4MNY</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MQ9">4MQ9</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OIN">4OIN</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OIP">4OIP</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OIQ">4OIQ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OIR">4OIR</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TOT">4TOT</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4U0G">4U0G</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4W4Z">4W4Z</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4W50">4W50</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X1N">4X1N</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X1Q">4X1Q</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X1R">4X1R</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X1S">4X1S</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X6S">4X6S</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YV9">4YV9</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Z0F">4Z0F</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZJX">4ZJX</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZQW">4ZQW</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AGU">5AGU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5B4W">5B4W</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CS2">5CS2</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EOC">5EOC</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ETU">5ETU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EUK">5EUK</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5F88">5F88</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FF6">5FF6</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GLH">5GLH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I2I">5I2I</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IOP">5IOP</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IR1">5IR1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ITF">5ITF</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IV2">5IV2</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IVZ">5IVZ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JR2">5JR2</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KEZ">5KEZ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KGN">5KGN</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LRG">5LRG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LRJ">5LRJ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LRK">5LRK</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LY1">5LY1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LY2">5LY2</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N99">5N99</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NES">5NES</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NEY">5NEY</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NF0">5NF0</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O45">5O45</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O4Y">5O4Y</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T1K">5T1K</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T1L">5T1L</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T1M">5T1M</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TH2">5TH2</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VBL">5VBL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VFC">5VFC</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VI6">5VI6</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W5S">5W5S</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W5U">5W5U</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W6I">5W6I</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W6R">5W6R</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W6T">5W6T</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XCO">5XCO</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XN3">5XN3</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B67">6B67</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C1Q">6C1Q</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C1R">6C1R</a>.</p></li></ul></div><div class="NLM_p">Perola and Charifson’s small molecule data set<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> was used to compare strain energy determined for canonical small molecules to strain energy estimated for macrocyclic peptides. Because the strain energy calculations reported here rely on electron density maps, it is noted that only 38 protein–ligand complexes from the original small-molecule data set had map coefficient files available. Each asymmetric unit ligand copy was extracted, amounting to 52 complexes in the data set. The list of small molecule-containing PDB entries is:<ul class="NLM_list-list_type-label" id="list2"><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=13GS">13GS</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1A4K">1A4K</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1A8T">1A8T</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1AFQ">1AFQ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ATL">1ATL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1AZM">1AZM</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BR6">1BR6</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1CET">1CET</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1DIB">1DIB</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1EVE">1EVE</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1EZQ">1EZQ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1F0R">1F0R</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1F0T">1F0T</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FCX">1FCX</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FCZ">1FCZ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FJS">1FJS</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FM6">1FM6</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FM9">1FM9</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1GWX">1GWX</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H1P">1H1P</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H1S">1H1S</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1I7Z">1I7Z</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1IY7">1IY7</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1K1J">1K1J</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1K7E">1K7E</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1K7F">1K7F</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LQD">1LQD</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M48">1M48</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MQ5">1MQ5</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MQ6">1MQ6</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NHU">1NHU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NHV">1NHV</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QHI">1QHI</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1THL">1THL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YDR">1YDR</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YDS">1YDS</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YDT">1YDT</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PCP">2PCP</a>.</p></li></ul></div><div class="NLM_p last">Jain et al.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> previously investigated the xGen methodology on 147 non-peptidic macrocycles, and the same data set was used here.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Ligand Preparation</h3><div class="NLM_p last">All peptide and small-molecule ligands were extracted from the PDB files using PyMOL version 2.3.1.<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> Ligands were protonated as expected at physiological pH using automated procedures within the Surflex Tools module, as previously described.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> Resulting structures were manually checked for correct bond orders, protonation states, and atom types with their original publications. Several ligands were missing atoms or side chain residues in the deposited ligand coordinates. To retain a direct comparison between experimental data and computational structures, the missing atoms were not added. Non-peptide macrocycles were prepared as previously described.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Real-Space Refinement of Ligands into Bound-State Conformational Ensembles</h3><div class="NLM_p">The xGen methodology and validation have been presented previously.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> Here, the standard “strict” procedure was employed to produce parsimonious conformational ensembles beginning from the deposited ligand and protein coordinates. The procedure first builds an idealized approximation to real-space X-ray electron density on a 0.25 Å grid surrounding the ligand (the <b>xgen_build</b> procedure). Next, beginning from the original crystallographic coordinates, the ligand is subjected to a conformational search using an objective function that combines force field energy with a measure of fit to electron density (the <b>xgen_refine</b> procedure). The resulting conformer pool is then expanded by taking each conformer and optimizing one copy toward better density congruence and another copy toward lower energy (the <b>xgen_expand</b> procedure). Finally, an occupancy-weighted conformer ensemble is generated (the <b>xgen_ensemble</b> procedure).</div><div class="NLM_p last">As described earlier, the conformers of the final ensemble are each part of a conformer neighborhood, all of whose members have small coordinate deviations from one another. Within each ensemble conformer’s neighborhood is at least one member that was optimized toward low energy. It is the set of lowest energy neighborhood members corresponding to each of the xGen ensemble members that form the input to the Boltzmann weighting procedure. The conformational ensemble that optimally fits the electron density is also provided to that procedure, and it is used to measure the deviation of the minima from the direct crystallographic experimental support.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Re-refining of Deposited Ligand Coordinates</h3><div class="NLM_p last">For the deposited ligand coordinates, the standard approach is to employ a flat-bottomed quadratic penalty during an energy minimization, preventing atoms from deviating much from their original positions. One common parameterization is to use a half-width of 0.5 Å a penalty of 500.0 kcal/mol/Å<sup>2</sup> (i.e., as used by Perola and Charifson). Here, we employed a half-width of 0.2 Å and a penalty of 1.0 kcal/mol/Å<sup>2</sup> to avoid excessively acute penalties on atoms that were questionably fit into electron density in their deposited coordinates. While there were differences between the two different parameterizations on a case by case basis, the overall results were statistically indistinguishable for the resulting estimates of both strain energy and deviation from the crystallographic support. Note that the single conformer emerging from the de-straining procedure was subjected to the same Boltzmann weighting procedure for final global strain estimation as were the xGen ensembles.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Search Procedures to Estimate Global Minima</h3><div class="NLM_p last">Each prepared ligand was subjected to an iterative conformational search beginning from its deposited coordinates and also beginning from a randomized conformer. Iterative searches were performed to ensure the global minimum was located, with each successive round being initiated from the prior round’s lowest energy conformer. Iterations terminated when either: (1) the same lowest energy structure was identified for three successive iterations, or (2) no new minimum was found for three consecutive rounds. Conformers identified during this search were combined into a single pool for the input to the Boltzmann weighting procedure.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Boltzmann Weighting Procedure</h3><div class="NLM_p last">To avoid energetic singularities and to make robust estimates of energy for conformers, we employed a procedure that repeatedly perturbs and minimizes input conformers. The Boltzmann weighting procedure first identifies the <i>N</i> lowest energy non-redundant conformers from an input pool (default <i>N</i> = 100 and default redundancy rmsd = 0.01 Å). For each such conformer, atomic coordinates are perturbed by a random value uniformly distributed at the interval [−0.01,+0.01], and minimization is performed, optionally with restraints to deter excessive deviation from the original coordinates. Five such perturbations and minimizations are performed. The energy for each conformer is computed as the Boltzmann average of the five local minima. The final reported energy value is computed as the Boltzmann average of each of these values for the N conformers. For the global minimum procedure, no positional restraints were used. For the bound-state energy estimates, this same procedure was applied, however, a flat-bottomed quadratic positional restraint was employed using a 0.1 Å half-width and a penalty of 1.0 kcal/mol/Å<sup>2</sup>.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Intermolecular Energy Estimation</h3><div class="NLM_p last">The Surflex-Dock scoring function has been extensively validated for both pose prediction and virtual screening.<a onclick="showRef(event, 'ref78 ref79 ref80 ref81 ref82'); return false;" href="javascript:void(0);" class="ref ref78 ref79 ref80 ref81 ref82">(78−82)</a> The function has been tuned to predict binding affinities, though the accuracy of such predictions is not thought to be sufficient to rank-order compounds for lead optimization. However, gross estimates of intermolecular binding free energy can be obtained by docking a ligand beginning with an agnostic initial state, or, when the bound ligand configuration has been determined, through local optimization. Here, given either xGen ensembles or deposited ligand coordinates, estimates were derived for the interaction energy between the protein and the ligand (using the Surflex Docking module’s <b>opt</b> procedure). The procedure provides both the total nominally predicted free energy of binding and the energy attributable to the hydrophobic and polar scoring function terms, without including terms for entropic fixation of rotatable bonds and rigid-body movement. The latter value was used as a rough guide to identify cases where estimates of global strain energy for a ligand were clearly too large to be supported by the interactions apparent in the bound complex.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Alternative Methods to Re-refine Bound-State Conformers</h3><div class="NLM_p">As mentioned earlier, in addition to the standard positional restraint approach with local minimization and the xGen ensemble fitting approach, two additional methods were tried. These were effectively a combination of the two other methods, where specific restraints were placed on each ligand atom based on its assigned B-factor from the deposited PDB structure. B-factors (also called the temperature factors, Debye–Waller factors, or atomic displacement parameters) derive their theoretical basis from the expected impact of the thermal motion of atoms on X-ray scattering.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> In practice, B-factors reflect broader aspects of the uncertainty in an atomic position, and they are assigned to minimize the deviation between calculated and experimental electron density. The xGen approach conducts a full conformational search subject to force field energy and a term that rewards fit to electron density. These two alternative methods conduct a full conformational search subject to the same force field and use a term that penalizes deviation from the original ligand coordinates.</div><div class="NLM_p last">The penalty is, again, a flat-bottomed quadratic potential (with a force constant of 1.0 kcal/mol/Å<sup>2</sup>), but the half-widths were assigned in a manner dependent on atomic B-factors. For the B-factor binning approach, the half-widths were set at 0.5 Å for B-factors ≤ 30, 1.0 Å B-factors > 30 and ≤60, and 5.0 Å for B-factors > 60. For the coordinate uncertainty approach, half-widths were set according to the calculated coordinated error, derived from the overall diffraction precision index of the structure,<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> combined with each atom’s B-factor using the standard formula.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22"><a href="/doi/suppl/10.1021/acs.jmedchem.0c02159" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58610" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58610" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c02159?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02159</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">High-level overview of available data archive (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02159/suppl_file/jm0c02159_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02159/suppl_file/jm0c02159_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02159/suppl_file/jm0c02159_si_001.pdf">jm0c02159_si_001.pdf (60.08 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02159/suppl_file/jm0c02159_si_002.csv">jm0c02159_si_002.csv (86.19 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c02159" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18328" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18328" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alexander C. Brueckner</span> - <span class="hlFld-Affiliation affiliation">Computational
& Structural Chemistry, Merck &
Co Inc, 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5866-0878" title="Orcid link">http://orcid.org/0000-0001-5866-0878</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f3929f968b929d979681dd9181869690989d9681b39e96819098dd909c9e"><span class="__cf_email__" data-cfemail="4e2f222b362f202a2b3c602c3c3b2b2d25202b3c0e232b3c2d25602d2123">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ajay N. Jain</span> - <span class="hlFld-Affiliation affiliation">Bioengineering
and Therapeutic Sciences, University of
California San Francisco, Box 0128, San Francisco, California 94158, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4641-8501" title="Orcid link">http://orcid.org/0000-0003-4641-8501</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#20414a41494e604a41494e4c41420e4f5247"><span class="__cf_email__" data-cfemail="9bfaf1faf2f5dbf1faf2f5f7faf9b5f4e9fc">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qiaolin Deng</span> - <span class="hlFld-Affiliation affiliation">Computational
& Structural Chemistry, Merck &
Co Inc, 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6378-3872" title="Orcid link">http://orcid.org/0000-0001-6378-3872</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ann E. Cleves</span> - <span class="hlFld-Affiliation affiliation">Bioengineering
and Therapeutic Sciences, University of
California San Francisco, Box 0128, San Francisco, California 94158, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1622-2770" title="Orcid link">http://orcid.org/0000-0002-1622-2770</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Charles A. Lesburg</span> - <span class="hlFld-Affiliation affiliation">Computational
and Structural Chemistry, Merck and Co Inc, 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7245-7331" title="Orcid link">http://orcid.org/0000-0001-7245-7331</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Juan C. Alvarez</span> - <span class="hlFld-Affiliation affiliation">Computational
and Structural Chemistry, Merck and Co Inc, 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mikhail Y. Reibarkh</span> - <span class="hlFld-Affiliation affiliation">Analytical
Research and Development, Merck & Co
Inc, 126 East Lincoln
Avenue, Rahway, New Jersey 07065, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6589-707X" title="Orcid link">http://orcid.org/0000-0002-6589-707X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Edward C. Sherer</span> - <span class="hlFld-Affiliation affiliation">Analytical
Research and Development, Merck & Co
Inc, 126 East Lincoln
Avenue, Rahway, New Jersey 07065, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8178-9186" title="Orcid link">http://orcid.org/0000-0001-8178-9186</a></span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): A.N.J. and A.E.C. have a financial interest in BioPharmics LLC, a biotechnology company whose main focus is in the development of methods for computational modeling in drug discovery.<br /></br></div></li></ul></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">bRo5</td><td class="NLM_def"><p class="first last">beyond Rule of 5</p></td></tr><tr><td class="NLM_term">Da</td><td class="NLM_def"><p class="first last">Dalton</p></td></tr><tr><td class="NLM_term">QM/MM</td><td class="NLM_def"><p class="first last">quantum mechanics/molecular mechanics</p></td></tr><tr><td class="NLM_term">Fab</td><td class="NLM_def"><p class="first last">antigen-binding fragments</p></td></tr><tr><td class="NLM_term">RSCC</td><td class="NLM_def"><p class="first last">real-space correlation coefficient</p></td></tr><tr><td class="NLM_term">RSR</td><td class="NLM_def"><p class="first last">real-space residual</p></td></tr><tr><td class="NLM_term">BACE1</td><td class="NLM_def"><p class="first last">beta-site APP cleaving enzyme 1</p></td></tr><tr><td class="NLM_term">HAC</td><td class="NLM_def"><p class="first last">heavy atom count</p></td></tr><tr><td class="NLM_term">Bmt</td><td class="NLM_def"><p class="first last">4-methyl-4-[(<i>E</i>)-2-butenyl]-4,<i>N</i>-methyl-threonine</p></td></tr><tr><td class="NLM_term">GRB7</td><td class="NLM_def"><p class="first last">growth factor receptor bound protein 7</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87275" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87275" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 82 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Over, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grönberg, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgendorf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muncipinto, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellisson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, M. W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvall, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Impact of Stereospecific Intramolecular Hydrogen Bonding on Cell Permeability and Physicochemical Properties</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2746</span>– <span class="NLM_lpage">2754</span>, <span class="refDoi"> DOI: 10.1021/jm500059t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500059t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFKiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=2746-2754&author=B.+Overauthor=P.+McCarrenauthor=P.+Arturssonauthor=M.+Foleyauthor=F.+Giordanettoauthor=G.+Gr%C3%B6nbergauthor=C.+Hilgendorfauthor=M.+D.+Leeauthor=P.+Matssonauthor=G.+Muncipintoauthor=M.+Pellissonauthor=M.+W.+D.+Perryauthor=R.+Svenssonauthor=J.+R.+Duvallauthor=J.+Kihlberg&title=Impact+of+Stereospecific+Intramolecular+Hydrogen+Bonding+on+Cell+Permeability+and+Physicochemical+Properties&doi=10.1021%2Fjm500059t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Stereospecific Intramolecular Hydrogen Bonding on Cell Permeability and Physicochemical Properties</span></div><div class="casAuthors">Over, Bjoern; McCarren, Patrick; Artursson, Per; Foley, Michael; Giordanetto, Fabrizio; Groenberg, Gunnar; Hilgendorf, Constanze; Lee, Maurice D.; Matsson, Paer; Muncipinto, Giovanni; Pellisson, Melanie; Perry, Matthew W. D.; Svensson, Richard; Duvall, Jeremy R.; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2746-2754</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Profiling of eight stereoisomeric T. cruzi growth inhibitors revealed vastly different in vitro properties such as soly., lipophilicity, pKa, and cell permeability for two sets of four stereoisomers.  Using computational chem. and NMR spectroscopy, we identified the formation of an intramol. NH→NR3 hydrogen bond in the set of stereoisomers displaying lower soly., higher lipophilicity, and higher cell permeability.  The intramol. hydrogen bond resulted in a significant pKa difference that accounts for the other structure-property relationships.  Application of this knowledge could be of particular value to maintain the delicate balance of size, soly., and lipophilicity required for cell penetration and oral administration for chem. probes or therapeutics with properties at, or beyond, Lipinski's rule of 5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR_VAbJqqxirVg90H21EOLACvtfcHk0ljiYSgwwoDoIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFKiurg%253D&md5=4227f825b82f5d17fab3f0de43806a5d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fjm500059t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500059t%26sid%3Dliteratum%253Aachs%26aulast%3DOver%26aufirst%3DB.%26aulast%3DMcCarren%26aufirst%3DP.%26aulast%3DArtursson%26aufirst%3DP.%26aulast%3DFoley%26aufirst%3DM.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DGr%25C3%25B6nberg%26aufirst%3DG.%26aulast%3DHilgendorf%26aufirst%3DC.%26aulast%3DLee%26aufirst%3DM.%2BD.%26aulast%3DMatsson%26aufirst%3DP.%26aulast%3DMuncipinto%26aufirst%3DG.%26aulast%3DPellisson%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DM.%2BW.%2BD.%26aulast%3DSvensson%26aufirst%3DR.%26aulast%3DDuvall%26aufirst%3DJ.%2BR.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DImpact%2520of%2520Stereospecific%2520Intramolecular%2520Hydrogen%2520Bonding%2520on%2520Cell%2520Permeability%2520and%2520Physicochemical%2520Properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D2746%26epage%3D2754%26doi%3D10.1021%2Fjm500059t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chirgadze, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanaraj, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyvonen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Innis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parisini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellegrini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayed, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sibanda, B. L.</span></span> <span> </span><span class="NLM_article-title">Protein-Protein Interactions in Receptor Activation and Intracellular Signalling</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>381</i></span>,  <span class="NLM_fpage">955</span>– <span class="NLM_lpage">959</span>, <span class="refDoi"> DOI: 10.1515/bc.2000.117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1515%2FBC.2000.117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=11076027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnvFSqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=381&publication_year=2000&pages=955-959&author=T.+L.+Blundellauthor=D.+F.+Burkeauthor=D.+Chirgadzeauthor=V.+Dhanarajauthor=M.+Hyvonenauthor=C.+A.+Innisauthor=E.+Parisiniauthor=L.+Pellegriniauthor=M.+Sayedauthor=B.+L.+Sibanda&title=Protein-Protein+Interactions+in+Receptor+Activation+and+Intracellular+Signalling&doi=10.1515%2Fbc.2000.117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-protein interactions in receptor activation and intracellular signalling</span></div><div class="casAuthors">Blundell, Tom L.; Burke, David F.; Chirgadze, Dimitri; Dhanaraj, Venugopal; Hyvonen, Marko; Innis, C. Axel; Parisini, Emilio; Pellegrini, Luca; Sayed, Muhammed; Sibanda, B. Lynn</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">381</span>
        (<span class="NLM_cas:issue">9/10</span>),
    <span class="NLM_cas:pages">955-959</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">A review, with 27 refs.  We review here signalling complexes that we have defined using X-ray anal. in our lab.  They include growth factors and their receptors: nerve growth factor (NGF) and its hetero-hexameric 7S NGF storage complex, hepatocyte growth factor/scatter factor (HGF/SF) NK1 dimers and fibroblast growth factor (FGF1) in complex with its receptor (FGFR2) ectodomain and heparin.  We also review our recent structural studies on intracellular signalling complexes, focusing on phosducin transducin Gβγ, CK2 protein kinase and its complexes, and the cyclin D-dependent kinase, Cdk6, bound to the cell cycle inhibitor p19INK4d.  Comparing the structures of these complexes with others we show that the surface area buried in signalling interactions does not always give a good indication of the strength of the interactions.  We show that conformational changes are often important in complexes with intermediate buried surface areas of 1500 to 2000 Å2, such as Cdk6 INK4 interactions.  Some interactions involve recognition of continuous epitopes, where there is no necessity for a tertiary structure and very often the binding conformation is induced during the process of interaction, for example phosducin binding to the βγ subunits (Gtβγ) of the heterotrimeric G protein transducin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOdDcfG3EA1rVg90H21EOLACvtfcHk0ljiYSgwwoDoIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnvFSqsbY%253D&md5=6d210079a72e28d158cd032b3f62fc0b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1515%2FBC.2000.117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252FBC.2000.117%26sid%3Dliteratum%253Aachs%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DBurke%26aufirst%3DD.%2BF.%26aulast%3DChirgadze%26aufirst%3DD.%26aulast%3DDhanaraj%26aufirst%3DV.%26aulast%3DHyvonen%26aufirst%3DM.%26aulast%3DInnis%26aufirst%3DC.%2BA.%26aulast%3DParisini%26aufirst%3DE.%26aulast%3DPellegrini%26aufirst%3DL.%26aulast%3DSayed%26aufirst%3DM.%26aulast%3DSibanda%26aufirst%3DB.%2BL.%26atitle%3DProtein-Protein%2520Interactions%2520in%2520Receptor%2520Activation%2520and%2520Intracellular%2520Signalling%26jtitle%3DBiol.%2520Chem.%26date%3D2000%26volume%3D381%26spage%3D955%26epage%3D959%26doi%3D10.1515%2Fbc.2000.117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, P.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Protein-Protein Interaction Inhibitors: Therapeutic Potential in Light of Molecular Size, Chemical Space, and Ligand Binding Efficiency Considerations</span>. <i>IUBMB Life</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">724</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1002/iub.383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fiub.383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=20979208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlGrsL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2010&pages=724-731&author=P.+Buchwald&title=Small-Molecule+Protein-Protein+Interaction+Inhibitors%3A+Therapeutic+Potential+in+Light+of+Molecular+Size%2C+Chemical+Space%2C+and+Ligand+Binding+Efficiency+Considerations&doi=10.1002%2Fiub.383"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule protein-protein interaction inhibitors: therapeutic potential in light of molecular size, chemical space, and ligand binding efficiency considerations</span></div><div class="casAuthors">Buchwald, Peter</div><div class="citationInfo"><span class="NLM_cas:title">IUBMB Life</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">724-731</span>CODEN:
                <span class="NLM_cas:coden">IULIF8</span>;
        ISSN:<span class="NLM_cas:issn">1521-6543</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Inc.</span>)
        </div><div class="casAbstract">A review.  As the ultimate function of proteins depends to a great extent on their binding partners, protein-protein interactions (PPIs) represent a treasure trove of possible new therapeutic targets.  Unfortunately, interfaces involved in PPIs are not well-suited for effective small mol. binding.  Nevertheless, successful examples of small-mol. PPI inhibitors (PPIIs) are beginning to accumulate, and the sheer no. of PPIs that form the human interactome implies that, despite the relative unsuitability of PPIs to serve as "druggable" targets, small-mol. PPIIs can still provide novel pharmacol. tools and new innovative drugs in at least some areas.  Here, after some illustrative examples, accumulating information on the binding efficiency, mol. size, and chem. space requirements will be briefly reviewed.  Therapeutic success can only be achieved if these considerations are incorporated into the search process and if careful medicinal chem. approaches are used to address the absorption, distribution, metab., and excretion requirements of larger mols. that are often needed for this target class due to the lower efficiency of binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPrt8JCmQ-mLVg90H21EOLACvtfcHk0liv-d2peKRRiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlGrsL%252FM&md5=ae27f95b3ef88b20ad2973927eaf4c91</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fiub.383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fiub.383%26sid%3Dliteratum%253Aachs%26aulast%3DBuchwald%26aufirst%3DP.%26atitle%3DSmall-Molecule%2520Protein-Protein%2520Interaction%2520Inhibitors%253A%2520Therapeutic%2520Potential%2520in%2520Light%2520of%2520Molecular%2520Size%252C%2520Chemical%2520Space%252C%2520and%2520Ligand%2520Binding%2520Efficiency%2520Considerations%26jtitle%3DIUBMB%2520Life%26date%3D2010%26volume%3D62%26spage%3D724%26epage%3D731%26doi%3D10.1002%2Fiub.383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D.</span></span> <span> </span><span class="NLM_article-title">The Future of Peptide-Based Drugs</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">136</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1111/cbdd.12055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1111%2Fcbdd.12055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=23253135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlt1yktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2013&pages=136-147&author=D.+J.+Craikauthor=D.+P.+Fairlieauthor=S.+Lirasauthor=D.+Price&title=The+Future+of+Peptide-Based+Drugs&doi=10.1111%2Fcbdd.12055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The future of peptide-based drugs</span></div><div class="casAuthors">Craik, David J.; Fairlie, David P.; Liras, Spiros; Price, David</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">136-147</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The suite of currently used drugs can be divided into two categories - traditional "small mol." drugs with typical mol. wts. of <500 Da but with oral bioavailability, and much larger "biologics" typically >5000 Da that are not orally bioavailable and need to be delivered via injection.  Due to their small size, conventional small mol. drugs may suffer from reduced target selectivity that often ultimately manifests in human side-effects, whereas protein therapeutics tend to be exquisitely specific for their targets due to many more interactions with them, but this comes at a cost of low bioavailability, poor membrane permeability, and metabolic instability.  The time has now come to reinvestigate new drug leads that fit between these two mol. wt. extremes, with the goal of combining advantages of small mols. (cost, conformational restriction, membrane permeability, metabolic stability, oral bioavailability) with those of proteins (natural components, target specificity, high potency).  This article uses selected examples of peptides to highlight the importance of peptide drugs, some potential new opportunities for their exploitation, and some difficult challenges ahead in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlaEZbrPXLPbVg90H21EOLACvtfcHk0liv-d2peKRRiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlt1yktA%253D%253D&md5=1e7e4b7e725541587fad855e6a28cadf</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12055%26sid%3Dliteratum%253Aachs%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DD.%26atitle%3DThe%2520Future%2520of%2520Peptide-Based%2520Drugs%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2013%26volume%3D81%26spage%3D136%26epage%3D147%26doi%3D10.1111%2Fcbdd.12055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span> <span> </span><span class="NLM_article-title">The Druggable Genome</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1038/nrd892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1038%2Fnrd892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=12209152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=727-730&author=A.+L.+Hopkinsauthor=C.+R.+Groom&title=The+Druggable+Genome&doi=10.1038%2Fnrd892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: The druggable genome</span></div><div class="casAuthors">Hopkins, Andrew L.; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">727-730</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  An assessment of the no. of mol. targets that represent an opportunity for therapeutic intervention is crucial to the development of post-genomic research strategies within the pharmaceutical industry.  Now that we know the size of the human genome, it is interesting to consider just how many mol. targets this opportunity represents.  We start from the position that we understand the properties that are required for a good drug, and therefore must be able to understand what makes a good drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Uvqi8O9wzrVg90H21EOLACvtfcHk0liv-d2peKRRiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D&md5=d6dc7c67822661fe03e920695def1329</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd892%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520Druggable%2520Genome%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D727%26epage%3D730%26doi%3D10.1038%2Fnrd892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raj, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">Plucking the High Hanging Fruit: A Systematic Approach for Targeting Protein-Protein Interactions</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4051</span>– <span class="NLM_lpage">4057</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2012.11.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1016%2Fj.bmc.2012.11.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=23267671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvV2rsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=4051-4057&author=M.+Rajauthor=B.+N.+Bullockauthor=P.+S.+Arora&title=Plucking+the+High+Hanging+Fruit%3A+A+Systematic+Approach+for+Targeting+Protein-Protein+Interactions&doi=10.1016%2Fj.bmc.2012.11.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Plucking the high hanging fruit: A systematic approach for targeting protein-protein interactions</span></div><div class="casAuthors">Raj, Monika; Bullock, Brooke N.; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4051-4057</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Development of specific ligands for protein targets that help decode the complexities of protein-protein interaction networks is a key goal for the field of chem. biol.  Despite the emergence of powerful in silico and exptl. high-throughput screening strategies, the discovery of synthetic ligands that selectively modulate protein-protein interactions remains a challenge for bioorg. and medicinal chemists.  This Perspective discusses emerging principles for the rational design of PPI inhibitors.  Fundamentally, the approach seeks to adapt nature's protein recognition principles for the design of suitable secondary structure mimetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorpTCgOTASB7Vg90H21EOLACvtfcHk0liv-d2peKRRiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvV2rsrbN&md5=970cdca83336f20387e060810f7f261c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2012.11.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2012.11.023%26sid%3Dliteratum%253Aachs%26aulast%3DRaj%26aufirst%3DM.%26aulast%3DBullock%26aufirst%3DB.%2BN.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DPlucking%2520the%2520High%2520Hanging%2520Fruit%253A%2520A%2520Systematic%2520Approach%2520for%2520Targeting%2520Protein-Protein%2520Interactions%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D4051%26epage%3D4057%26doi%3D10.1016%2Fj.bmc.2012.11.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Northfield, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colless, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaousis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamernig, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, R.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Rational Design and Synthesis of an Orally Bioavailable Peptide Guided by NMR Amide Temperature Coefficients</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">17504</span>– <span class="NLM_lpage">17509</span>, <span class="refDoi"> DOI: 10.1073/pnas.1417611111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1073%2Fpnas.1417611111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=25416591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKmu7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=17504-17509&author=C.+K.+Wangauthor=S.+E.+Northfieldauthor=B.+Collessauthor=S.+Chaousisauthor=I.+Hamernigauthor=R.-J.+Lohmanauthor=D.+S.+Nielsenauthor=C.+I.+Schroederauthor=S.+Lirasauthor=D.+A.+Priceauthor=D.+P.+Fairlieauthor=D.+J.+Craik&title=Rational+Design+and+Synthesis+of+an+Orally+Bioavailable+Peptide+Guided+by+NMR+Amide+Temperature+Coefficients&doi=10.1073%2Fpnas.1417611111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Rational design and synthesis of an orally bioavailable peptide guided by NMR amide temperature coefficients</span></div><div class="casAuthors">Wang, Conan K.; Northfield, Susan E.; Colless, Barbara; Chaousis, Stephanie; Hamernig, Ingrid; Lohman, Rink-Jan; Nielsen, Daniel S.; Schroeder, Christina I.; Liras, Spiros; Price, David A.; Fairlie, David P.; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">17504-17509</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Enhancing the oral bioavailability of peptide drug leads is a major challenge in drug design.  As such, methods to address this challenge are highly sought after by the pharmaceutical industry.  Here, we propose a strategy to identify appropriate amides for N-methylation using temp. coeffs. measured by NMR to identify exposed amides in cyclic peptides.  N-methylation effectively caps these amides, modifying the overall solvation properties of the peptides and making them more membrane permeable.  The approach for identifying sites for N-methylation is a rapid alternative to the elucidation of 3D structures of peptide drug leads, which has been a commonly used structure-guided approach in the past.  Five leucine-rich peptide scaffolds are reported with selectively designed N-methylated derivs.  In vitro membrane permeability was assessed by parallel artificial membrane permeability assay and Caco-2 assay.  The most promising N-methylated peptide was then tested in vivo.  Here we report a novel peptide (15), which displayed an oral bioavailability of 33% in a rat model, thus validating the design approach.  We show that this approach can also be used to explain the notable increase in oral bioavailability of a somatostatin analog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ1pwUl6qUw7Vg90H21EOLACvtfcHk0ljU1wL7LHRMxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKmu7vE&md5=02803e516bf514e3d941769b7bbf3607</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1417611111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1417611111%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%2BK.%26aulast%3DNorthfield%26aufirst%3DS.%2BE.%26aulast%3DColless%26aufirst%3DB.%26aulast%3DChaousis%26aufirst%3DS.%26aulast%3DHamernig%26aufirst%3DI.%26aulast%3DLohman%26aufirst%3DR.-J.%26aulast%3DNielsen%26aufirst%3DD.%2BS.%26aulast%3DSchroeder%26aufirst%3DC.%2BI.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DRational%2520Design%2520and%2520Synthesis%2520of%2520an%2520Orally%2520Bioavailable%2520Peptide%2520Guided%2520by%2520NMR%2520Amide%2520Temperature%2520Coefficients%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D17504%26epage%3D17509%26doi%3D10.1073%2Fpnas.1417611111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cardote, T. A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Cyclic and Macrocyclic Peptides as Chemical Tools to Recognise Protein Surfaces and Probe Protein-Protein Interactions</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">787</span>– <span class="NLM_lpage">794</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fcmdc.201500450" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=26563831" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVajsL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=787-794&author=T.+A.+F.+Cardoteauthor=A.+Ciulli&title=Cyclic+and+Macrocyclic+Peptides+as+Chemical+Tools+to+Recognise+Protein+Surfaces+and+Probe+Protein-Protein+Interactions&doi=10.1002%2Fcmdc.201500450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic and Macrocyclic Peptides as Chemical Tools To Recognise Protein Surfaces and Probe Protein-Protein Interactions</span></div><div class="casAuthors">Cardote, Teresa A. F.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">787-794</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Targeting protein surfaces and protein-protein interactions (PPIs) with small mols. is a frontier goal of chem. biol. and provides attractive therapeutic opportunities in drug discovery.  The mol. properties of protein surfaces, including their shallow features and lack of deep binding pockets, pose significant challenges, and as a result have proved difficult to target.  Peptides are ideal candidates for this mission due to their ability to closely mimic many structural features of protein interfaces.  However, their inherently low intracellular stability and permeability and high in vivo clearance have thus far limited their biol. applications.  One way to improve these properties is to constrain the secondary structure of linear peptides by cyclization.  Herein we review various classes of cyclic and macrocyclic peptides as chem. probes of protein surfaces and modulators of PPIs.  The growing interest in this area and recent advances provide evidence of the potential of developing peptide-like mols. that specifically target these interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRiqVM7-eZhbVg90H21EOLACvtfcHk0ljU1wL7LHRMxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVajsL%252FE&md5=a67ba0ee61764a882d5a37b474570343</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500450%26sid%3Dliteratum%253Aachs%26aulast%3DCardote%26aufirst%3DT.%2BA.%2BF.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DCyclic%2520and%2520Macrocyclic%2520Peptides%2520as%2520Chemical%2520Tools%2520to%2520Recognise%2520Protein%2520Surfaces%2520and%2520Probe%2520Protein-Protein%2520Interactions%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D787%26epage%3D794%26doi%3D10.1002%2Fcmdc.201500450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClendon, C. L.</span></span> <span> </span><span class="NLM_article-title">Reaching for High-Hanging Fruit in Drug Discovery at Protein-Protein Interfaces</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>450</i></span>,  <span class="NLM_fpage">1001</span>– <span class="NLM_lpage">1009</span>, <span class="refDoi"> DOI: 10.1038/nature06526</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1038%2Fnature06526" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=18075579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2007&pages=1001-1009&author=J.+A.+Wellsauthor=C.+L.+McClendon&title=Reaching+for+High-Hanging+Fruit+in+Drug+Discovery+at+Protein-Protein+Interfaces&doi=10.1038%2Fnature06526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Reaching for high-hanging fruit in drug discovery at protein-protein interfaces</span></div><div class="casAuthors">Wells, James A.; McClendon, Christopher L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">450</span>
        (<span class="NLM_cas:issue">7172</span>),
    <span class="NLM_cas:pages">1001-1009</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Targeting the interfaces between proteins has huge therapeutic potential, but discovering small-mol. drugs that disrupt protein-protein interactions is an enormous challenge.  Several recent success stories, however, indicate that protein-protein interfaces might be more tractable than has been thought.  These studies discovered small mols. that bind with drug-like potencies to 'hotspots' on the contact surfaces involved in protein-protein interactions.  Remarkably, these small mols. bind deeper within the contact surface of the target protein, and bind with much higher efficiencies, than do the contact atoms of the natural protein partner.  Some of these small mols. are now making their way through clin. trials, so this high-hanging fruit might not be far out of reach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-nGR3BqqkrVg90H21EOLACvtfcHk0ljU1wL7LHRMxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F&md5=e7ed69fd8c362a71c4b99f029c16a6fc</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnature06526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06526%26sid%3Dliteratum%253Aachs%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DMcClendon%26aufirst%3DC.%2BL.%26atitle%3DReaching%2520for%2520High-Hanging%2520Fruit%2520in%2520Drug%2520Discovery%2520at%2520Protein-Protein%2520Interfaces%26jtitle%3DNature%26date%3D2007%26volume%3D450%26spage%3D1001%26epage%3D1009%26doi%3D10.1038%2Fnature06526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.</span></span> <span> </span><span class="NLM_article-title">Selective, Cell-Permeable Nonphosphorylated Bicyclic Peptidyl Inhibitor Against Peptidyl-Prolyl Isomerase Pin1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6306</span>– <span class="NLM_lpage">6312</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00411</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00411" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2lt77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6306-6312&author=B.+Jiangauthor=D.+Pei&title=Selective%2C+Cell-Permeable+Nonphosphorylated+Bicyclic+Peptidyl+Inhibitor+Against+Peptidyl-Prolyl+Isomerase+Pin1&doi=10.1021%2Facs.jmedchem.5b00411"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A Selective, Cell-Permeable Nonphosphorylated Bicyclic Peptidyl Inhibitor against Peptidyl-Prolyl Isomerase Pin1</span></div><div class="casAuthors">Jiang, Bisheng; Pei, Dehua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6306-6312</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pin1 regulates the levels and functions of phosphoproteins by catalyzing phosphorylation-dependent cis/trans isomerization of peptidyl-prolyl bonds.  Previous Pin1 inhibitors contained phosphoamino acids, which are metabolically unstable and have poor membrane permeability.  In this work, the authors report a cell-permeable and metabolically stable nonphosphorylated bicyclic peptide as a potent and selective Pin1 inhibitor, which inhibited the intracellular Pin1 activity in cultured mammalian cells but had little effect on other isomerases such as Pin4, FKBP12, or cyclophilin A.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrK8No2ty2NM7Vg90H21EOLACvtfcHk0lj7zzdbXmVBFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2lt77P&md5=735a73f73d5aa05ab7f046e7e693b135</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00411%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DPei%26aufirst%3DD.%26atitle%3DSelective%252C%2520Cell-Permeable%2520Nonphosphorylated%2520Bicyclic%2520Peptidyl%2520Inhibitor%2520Against%2520Peptidyl-Prolyl%2520Isomerase%2520Pin1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6306%26epage%3D6312%26doi%3D10.1021%2Facs.jmedchem.5b00411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwyer, J. J.</span></span> <span> </span><span class="NLM_article-title">Lead Discovery and Optimization Strategies for Peptide Macrocycles</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">479</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2014.07.083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1016%2Fj.ejmech.2014.07.083" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=25109255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1GktrrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2015&pages=471-479&author=A.+Bhatauthor=L.+R.+Robertsauthor=J.+J.+Dwyer&title=Lead+Discovery+and+Optimization+Strategies+for+Peptide+Macrocycles&doi=10.1016%2Fj.ejmech.2014.07.083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Lead discovery and optimization strategies for peptide macrocycles</span></div><div class="casAuthors">Bhat, Abhijit; Roberts, Lee R.; Dwyer, John J.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">471-479</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Peptide macrocycles represent a chem. space where the best of biol. tools can synergize with the best of chem. approaches in the quest for leads against undruggable targets.  Peptide macrocycles offer some key advantages in both lead discovery and lead optimization phases of drug discovery when compared to natural product and synthetic macrocycles.  In addn., they are uniquely positioned to capitalize on the therapeutic potential of peptides because cyclization can help drive selectivity, potency and overcome the common limitations of metabolic instability of peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrYEBhURBm07Vg90H21EOLACvtfcHk0lj7zzdbXmVBFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1GktrrM&md5=2e12788610c87e9c774dcf3804855cf4</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.07.083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.07.083%26sid%3Dliteratum%253Aachs%26aulast%3DBhat%26aufirst%3DA.%26aulast%3DRoberts%26aufirst%3DL.%2BR.%26aulast%3DDwyer%26aufirst%3DJ.%2BJ.%26atitle%3DLead%2520Discovery%2520and%2520Optimization%2520Strategies%2520for%2520Peptide%2520Macrocycles%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D94%26spage%3D471%26epage%3D479%26doi%3D10.1016%2Fj.ejmech.2014.07.083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobritzsch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">How Beyond Rule of 5 Drugs and Clinical Candidates Bind to Their Targets</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2312</span>– <span class="NLM_lpage">2327</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01286</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01286" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2312-2327&author=B.+C.+Doakauthor=J.+Zhengauthor=D.+Dobritzschauthor=J.+Kihlberg&title=How+Beyond+Rule+of+5+Drugs+and+Clinical+Candidates+Bind+to+Their+Targets&doi=10.1021%2Facs.jmedchem.5b01286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">How Beyond Rule of 5 Drugs and Clinical Candidates Bind to Their Targets</span></div><div class="casAuthors">Doak, Bradley C.; Zheng, Jie; Dobritzsch, Doreen; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2312-2327</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To improve discovery of drugs for difficult targets, the opportunities of chem. space beyond the rule of 5 (bRo5) were examd. by retrospective anal. of a comprehensive set of structures for complexes between drugs and clin. candidates and their targets.  The anal. illustrates the potential of compds. far beyond rule of 5 space to modulate novel and difficult target classes that have large, flat, and groove-shaped binding sites.  However, ligand efficiencies are significantly reduced for flat- and groove-shape binding sites, suggesting that adjustments of how to use such metrics are required.  Ligands bRo5 appear to benefit from an appropriate balance between rigidity and flexibility to bind with sufficient affinity to their targets, with macrocycles and nonmacrocycles being found to have similar flexibility.  However, macrocycles were more disk- and spherelike, which may contribute to their superior binding to flat sites, while rigidification of nonmacrocycles lead to rodlike ligands that bind well to groove-shaped binding sites.  These insights should contribute to altering perceptions of what targets are considered "druggable" and provide support for drug design in beyond rule of 5 space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZD-h9VodiPLVg90H21EOLACvtfcHk0lj7zzdbXmVBFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F&md5=a417b605d1a3ffbc7ee84706401f01f4</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01286%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DDobritzsch%26aufirst%3DD.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DHow%2520Beyond%2520Rule%2520of%25205%2520Drugs%2520and%2520Clinical%2520Candidates%2520Bind%2520to%2520Their%2520Targets%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2312%26epage%3D2327%26doi%3D10.1021%2Facs.jmedchem.5b01286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, M. L.</span></span> <span> </span><span class="NLM_article-title">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1961</span>– <span class="NLM_lpage">2004</span>, <span class="refDoi"> DOI: 10.1021/jm1012374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1961-2004&author=E.+Marsaultauthor=M.+L.+Peterson&title=Macrocycles+Are+Great+Cycles%3A+Applications%2C+Opportunities%2C+and+Challenges+of+Synthetic+Macrocycles+in+Drug+Discovery&doi=10.1021%2Fjm1012374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span></div><div class="casAuthors">Marsault, Eric; Peterson, Mark L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1961-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles occupy a unique segment of chem. space.  In the past decade, their chem. diversity expanded significantly, supported by advances in bioinformatics and synthetic methodol.  As a consequence, this structural type has now been successfully tested on most biol. target classes.  The goal of this article is to put into perspective the current applications, opportunities, and challenges assocd. with synthetic macrocycles in drug discovery.  Accordingly, the first part of this article is dedicated to the drug discovery aspects of macrocycles and highlights salient features of their medicinal chem.  This section is organized by target class, a choice aimed at providing the reader an appreciation of the structural diversity generated for each class.  To give the reader an appreciation of the tools available to construct macrocyclic scaffolds, the site and method of the pivotal macrocyclization step are indicated in the figures.  Readers are referred to the source articles for further details.  In the second part, the technologies and synthetic approaches that already have demonstrated utility or possess a high potential for macrocycle-based drug discovery are discussed.  Finally, a perspective on the future of synthetic macrocycles in medicinal chem. is offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygQLE6JraHrVg90H21EOLACvtfcHk0lgFhor3nAW4Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D&md5=97c1120896e90a85790a9ac1274259f3</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm1012374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012374%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26atitle%3DMacrocycles%2520Are%2520Great%2520Cycles%253A%2520Applications%252C%2520Opportunities%252C%2520and%2520Challenges%2520of%2520Synthetic%2520Macrocycles%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1961%26epage%3D2004%26doi%3D10.1021%2Fjm1012374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Northfield, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swedberg, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colless, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaousis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Exploring Experimental and Computational Markers of Cyclic Peptides: Charting Islands of Permeability</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">213</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2015.04.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1016%2Fj.ejmech.2015.04.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=25974856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotlKhu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2015&pages=202-213&author=C.+K.+Wangauthor=S.+E.+Northfieldauthor=J.+E.+Swedbergauthor=B.+Collessauthor=S.+Chaousisauthor=D.+A.+Priceauthor=S.+Lirasauthor=D.+J.+Craik&title=Exploring+Experimental+and+Computational+Markers+of+Cyclic+Peptides%3A+Charting+Islands+of+Permeability&doi=10.1016%2Fj.ejmech.2015.04.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring experimental and computational markers of cyclic peptides: Charting islands of permeability</span></div><div class="casAuthors">Wang, Conan K.; Northfield, Susan E.; Swedberg, Joakim E.; Colless, Barbara; Chaousis, Stephanie; Price, David A.; Liras, Spiros; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">202-213</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">An increasing no. of macrocyclic peptides that cross biol. membranes are being reported, suggesting that it might be possible to develop peptides into orally bioavailable therapeutics; however, current understanding of what makes macrocyclic peptides cell permeable is still limited.  Here, we synthesized 62 cyclic hexapeptides and characterized their permeability using in vitro assays commonly used to predict in vivo absorption rates, i.e. the Caco-2 and PAMPA assays.  We correlated permeability with exptl. measured parameters of peptide conformation obtained using rapid methods based on chromatog. and NMR spectroscopy.  Based on these correlations, we propose a model describing the interplay between peptide permeability, lipophilicity and hydrogen bonding potential.  Specifically, peptides with very high permeability have high lipophilicity and few solvent hydrogen bond interactions, whereas peptides with very low permeability have low lipophilicity or many solvent interactions.  Our model is supported by mol. dynamics simulations of the cyclic peptides calcd. in explicit solvent, providing a structural basis for the obsd. correlations.  This prospective exploration into biomarkers of peptide permeability has the potential to unlock wider opportunities for development of peptides into drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYs3AwkeaC6rVg90H21EOLACvtfcHk0lgFhor3nAW4Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotlKhu70%253D&md5=300b043854615dc799a79646931cb87f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2015.04.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2015.04.049%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%2BK.%26aulast%3DNorthfield%26aufirst%3DS.%2BE.%26aulast%3DSwedberg%26aufirst%3DJ.%2BE.%26aulast%3DColless%26aufirst%3DB.%26aulast%3DChaousis%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DExploring%2520Experimental%2520and%2520Computational%2520Markers%2520of%2520Cyclic%2520Peptides%253A%2520Charting%2520Islands%2520of%2520Permeability%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D97%26spage%3D202%26epage%3D213%26doi%3D10.1016%2Fj.ejmech.2015.04.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thansandote, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dexter, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span> <span> </span><span class="NLM_article-title">Improving the Passive Permeability of Macrocyclic Peptides: Balancing Permeability with Other Physicochemical Properties</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">322</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2014.11.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1016%2Fj.bmc.2014.11.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=25533323" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVWktL3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=322-327&author=P.+Thansandoteauthor=R.+M.+Harrisauthor=H.+L.+Dexterauthor=G.+L.+Simpsonauthor=S.+Palauthor=R.+J.+Uptonauthor=K.+Valko&title=Improving+the+Passive+Permeability+of+Macrocyclic+Peptides%3A+Balancing+Permeability+with+Other+Physicochemical+Properties&doi=10.1016%2Fj.bmc.2014.11.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Improving the passive permeability of macrocyclic peptides: Balancing permeability with other physicochemical properties</span></div><div class="casAuthors">Thansandote, Praew; Harris, Robert M.; Dexter, Hannah L.; Simpson, Graham L.; Pal, Sandeep; Upton, Richard J.; Valko, Klara</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">322-327</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A no. of methods to improve the passive permeability of a set of cyclic peptides have been investigated using 6- and 7-mer macrocyclic templates.  In many cases the peptides were designed by mol. dynamics calcns. to evaluate the methods.  The aim of this study was not only to improve passive permeability, but also to balance permeability with other physicochem. properties with the goal of understanding and applying the knowledge to develop active cyclic peptides into drug candidates.  Evaluation of the methods herein suggest that increasing passive permeability often occurs at the expense of soly. and lipophilicity.  Computational methods can be useful when attempting to predict and design features to balance these properties, though limitations were obsd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqx--UWVWxM07Vg90H21EOLACvtfcHk0lgFhor3nAW4Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVWktL3O&md5=a9eb20c8e03b890c1d4616a8bdebf26c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2014.11.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2014.11.034%26sid%3Dliteratum%253Aachs%26aulast%3DThansandote%26aufirst%3DP.%26aulast%3DHarris%26aufirst%3DR.%2BM.%26aulast%3DDexter%26aufirst%3DH.%2BL.%26aulast%3DSimpson%26aufirst%3DG.%2BL.%26aulast%3DPal%26aufirst%3DS.%26aulast%3DUpton%26aufirst%3DR.%2BJ.%26aulast%3DValko%26aufirst%3DK.%26atitle%3DImproving%2520the%2520Passive%2520Permeability%2520of%2520Macrocyclic%2520Peptides%253A%2520Balancing%2520Permeability%2520with%2520Other%2520Physicochemical%2520Properties%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D322%26epage%3D327%26doi%3D10.1016%2Fj.bmc.2014.11.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradov, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Peptides as Drug Candidates: Recent Progress and Remaining Challenges</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">4167</span>– <span class="NLM_lpage">4181</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b13178</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b13178" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtFCqtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=4167-4181&author=A.+A.+Vinogradovauthor=Y.+Yinauthor=H.+Suga&title=Macrocyclic+Peptides+as+Drug+Candidates%3A+Recent+Progress+and+Remaining+Challenges&doi=10.1021%2Fjacs.8b13178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Peptides as Drug Candidates: Recent Progress and Remaining Challenges</span></div><div class="casAuthors">Vinogradov, Alexander A.; Yin, Yizhen; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4167-4181</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Peptides as a therapeutic modality attract much attention due to their synthetic accessibility, high degree of specific binding, and the ability to target protein surfaces traditionally considered "undruggable".  Unfortunately, at the same time, other pharmacol. properties of a generic peptide, such as metabolic stability and cell permeability, are quite poor, which limits the success of de novo discovered biol. active peptides as drug candidates.  Here, we review how macrocyclization as well as the incorporation of nonproteogenic amino acids and various conjugation strategies may be utilized to improve on these characteristics to create better drug candidates.  We analyze recent progress and remaining challenges in improving individual pharmacol. properties of bioactive peptides, and offer our opinion on interfacing these, often conflicting, considerations, to create balanced drug candidates as a potential way to make further progress in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXRPkCAGk8-rVg90H21EOLACvtfcHk0ljLAAE2Q9Dgog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtFCqtb4%253D&md5=59b6197874d4709a74089367a09119cc</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b13178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b13178%26sid%3Dliteratum%253Aachs%26aulast%3DVinogradov%26aufirst%3DA.%2BA.%26aulast%3DYin%26aufirst%3DY.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DMacrocyclic%2520Peptides%2520as%2520Drug%2520Candidates%253A%2520Recent%2520Progress%2520and%2520Remaining%2520Challenges%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D4167%26epage%3D4181%26doi%3D10.1021%2Fjacs.8b13178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Driggers, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrett, N. K.</span></span> <span> </span><span class="NLM_article-title">The Exploration of Macrocycles for Drug Discovery - An Underexploited Structural Class</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">608</span>– <span class="NLM_lpage">624</span>, <span class="refDoi"> DOI: 10.1038/nrd2590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1038%2Fnrd2590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=18591981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=608-624&author=E.+M.+Driggersauthor=S.+P.+Haleauthor=J.+Leeauthor=N.+K.+Terrett&title=The+Exploration+of+Macrocycles+for+Drug+Discovery+-+An+Underexploited+Structural+Class&doi=10.1038%2Fnrd2590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The exploration of macrocycles for drug discovery - an underexploited structural class</span></div><div class="casAuthors">Driggers, Edward M.; Hale, Stephen P.; Lee, Jinbo; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">608-624</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Natural products comprised of a macrocycle ring structure have proven their therapeutic applications as antibiotics, immunosuppressants as well as anticancer agents.  Despite this, macrocyclic compds. remain under-explored.  Terrett and colleagues review the properties and features of current macrocycle drugs, emphasizing the vast potential of synthetic macrocycles in drug discovery.  Macrocyclic natural products have evolved to fulfil numerous biochem. functions, and their profound pharmacol. properties have led to their development as drugs.  A macrocycle provides diverse functionality and stereochem. complexity in a conformationally pre-organized ring structure.  This can result in high affinity and selectivity for protein targets, while preserving sufficient bioavailability to reach intracellular locations.  Despite these valuable characteristics, and the proven success of more than 100 marketed macrocycle drugs derived from natural products, this structural class has been poorly explored within drug discovery.  This is in part due to concerns about synthetic intractability and non-drug-like properties.  This Review describes the growing body of data in favor of macrocyclic therapeutics, and demonstrates that this class of compds. can be both fully drug-like in its properties and readily prepd. owing to recent advances in synthetic medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDtW8LpE4ru7Vg90H21EOLACvtfcHk0ljLAAE2Q9Dgog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D&md5=16e091818053e5b163b81078a8d03094</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnrd2590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2590%26sid%3Dliteratum%253Aachs%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520Exploration%2520of%2520Macrocycles%2520for%2520Drug%2520Discovery%2520-%2520An%2520Underexploited%2520Structural%2520Class%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D608%26epage%3D624%26doi%3D10.1038%2Fnrd2590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chennamadhavuni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">How Proteins Bind Macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-731&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+Proteins+Bind+Macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0ljLAAE2Q9Dgog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520Proteins%2520Bind%2520Macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D731%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biron, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovadia, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langenegger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brueggen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoyer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jelinek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span> <span> </span><span class="NLM_article-title">Improving Oral Bioavailability of Peptides by Multiple N-Methylation: Somatostatin Analogues</span>. <i>Angew. Chem. Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">2595</span>– <span class="NLM_lpage">2599</span>, <span class="refDoi"> DOI: 10.1002/anie.200705797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fanie.200705797" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFartLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=2595-2599&author=E.+Bironauthor=J.+Chatterjeeauthor=O.+Ovadiaauthor=D.+Langeneggerauthor=J.+Brueggenauthor=D.+Hoyerauthor=H.+A.+Schmidauthor=R.+Jelinekauthor=C.+Gilonauthor=A.+Hoffmanauthor=H.+Kessler&title=Improving+Oral+Bioavailability+of+Peptides+by+Multiple+N-Methylation%3A+Somatostatin+Analogues&doi=10.1002%2Fanie.200705797"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Improving oral bioavailability of peptides by multiple N-methylation: somatostatin analogues</span></div><div class="casAuthors">Biron, Eric; Chatterjee, Jayanta; Ovadia, Oded; Langenegger, Daniel; Brueggen, Joseph; Hoyer, Daniel; Schmid, Herbert A.; Jelnick, Raz; Gilon, Chaim; Hoffman, Amnon; Kessler, Horst</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2595-2599</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A complete library of the N-methylated somatostatin cyclopeptidic analog Veber-Hirschmann peptide cyclo(-PFwKTF-) has been prepd. with the aim of improving its bioavailability.  Several analogs from the library were found to bind to the somatostatin receptor in the nanomolar range and one of them shows a significant oral bioavailability of 10%.  Conformational anal. shows that N-methylation is allowed at specific positions without affecting the bioactive conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo64OMbjkAtErVg90H21EOLACvtfcHk0ljLAAE2Q9Dgog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFartLw%253D&md5=42c312786da82941dde6671b1130b61b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fanie.200705797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200705797%26sid%3Dliteratum%253Aachs%26aulast%3DBiron%26aufirst%3DE.%26aulast%3DChatterjee%26aufirst%3DJ.%26aulast%3DOvadia%26aufirst%3DO.%26aulast%3DLangenegger%26aufirst%3DD.%26aulast%3DBrueggen%26aufirst%3DJ.%26aulast%3DHoyer%26aufirst%3DD.%26aulast%3DSchmid%26aufirst%3DH.%2BA.%26aulast%3DJelinek%26aufirst%3DR.%26aulast%3DGilon%26aufirst%3DC.%26aulast%3DHoffman%26aufirst%3DA.%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DImproving%2520Oral%2520Bioavailability%2520of%2520Peptides%2520by%2520Multiple%2520N-Methylation%253A%2520Somatostatin%2520Analogues%26jtitle%3DAngew.%2520Chem.%2520Int.%2520Ed.%26date%3D2008%26volume%3D47%26spage%3D2595%26epage%3D2599%26doi%3D10.1002%2Fanie.200705797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beck, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiran, M. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovadia, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span> <span> </span><span class="NLM_article-title">Intestinal Permeability of Cyclic Peptides: Common Key Backbone Motifs Identified</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">12125</span>– <span class="NLM_lpage">12133</span>, <span class="refDoi"> DOI: 10.1021/ja303200d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja303200d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpt1Cisbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=12125-12133&author=J.+G.+Beckauthor=J.+Chatterjeeauthor=B.+Lauferauthor=M.+U.+Kiranauthor=A.+O.+Frankauthor=S.+Neubauerauthor=O.+Ovadiaauthor=S.+Greenbergauthor=C.+Gilonauthor=A.+Hoffmanauthor=H.+Kessler&title=Intestinal+Permeability+of+Cyclic+Peptides%3A+Common+Key+Backbone+Motifs+Identified&doi=10.1021%2Fja303200d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Intestinal Permeability of Cyclic Peptides: Common Key Backbone Motifs Identified</span></div><div class="casAuthors">Beck, Johannes G.; Chatterjee, Jayanta; Laufer, Burkhardt; Kiran, Marelli Udaya; Frank, Andreas O.; Neubauer, Stefanie; Ovadia, Oded; Greenberg, Sarit; Gilon, Chaim; Hoffman, Amnon; Kessler, Horst</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">12125-12133</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Insufficient oral bioavailability is considered as a key limitation for the widespread development of peptides as therapeutics.  While the oral bioavailability of small org. compds. is often estd. from simple rules, similar rules do not apply to peptides, and even the high oral bioavailability that is described for a small no. of peptides is not well understood.  Here we present two highly Caco-2 permeable template structures based on a library of 54 cyclo(-d-Ala-Ala5-) peptides with different N-methylation patterns.  The first all-trans template structure possesses two β-turns of type II along Ala6-d-Ala1 and Ala3-Ala4 and is only found for one peptide with two N-Me groups at d-Ala1 and Ala6 [NMe(1,6)].  The second single-cis template possesses a characteristic cis peptide bond preceding Ala5, which results in type VI β-turn geometry along Ala4-Ala5.  Although the second template structure is found in seven peptides carrying N-Me groups on Ala5, high Caco-2 permeability is only found for a subgroup of two of them [NMe(1,5) and NMe(1,2,4,5)], suggesting that N-methylation of d-Ala1 is a prerequisite for high permeability of the second template structure.  The structural similarity of the second template structure with the orally bioavailable somatostatin analog cyclo(-Pro-Phe-NMe-d-Trp-NMe-Lys-Thr-NMe-Phe-), and the striking resemblance with both β-turns of the orally bioavailable peptide cyclosporine A, suggests that the introduction of bioactive sequences on the highly Caco-2 permeable templates may result in potent orally bioavailable drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4AC8OVfXJMbVg90H21EOLACvtfcHk0lgcpTHgT13OqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpt1Cisbw%253D&md5=ecf4dc5aa62a8ebb7c5cc99e8009f528</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fja303200d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja303200d%26sid%3Dliteratum%253Aachs%26aulast%3DBeck%26aufirst%3DJ.%2BG.%26aulast%3DChatterjee%26aufirst%3DJ.%26aulast%3DLaufer%26aufirst%3DB.%26aulast%3DKiran%26aufirst%3DM.%2BU.%26aulast%3DFrank%26aufirst%3DA.%2BO.%26aulast%3DNeubauer%26aufirst%3DS.%26aulast%3DOvadia%26aufirst%3DO.%26aulast%3DGreenberg%26aufirst%3DS.%26aulast%3DGilon%26aufirst%3DC.%26aulast%3DHoffman%26aufirst%3DA.%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DIntestinal%2520Permeability%2520of%2520Cyclic%2520Peptides%253A%2520Common%2520Key%2520Backbone%2520Motifs%2520Identified%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D12125%26epage%3D12133%26doi%3D10.1021%2Fja303200d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, R.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diness, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Total Synthesis, Structure, and Oral Absorption of a Thiazole Cyclic Peptide, Sanguinamide A</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">5720</span>– <span class="NLM_lpage">5723</span>, <span class="refDoi"> DOI: 10.1021/ol3027347</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol3027347" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1WqsrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=5720-5723&author=D.+S.+Nielsenauthor=H.+N.+Hoangauthor=R.-J.+Lohmanauthor=F.+Dinessauthor=D.+P.+Fairlie&title=Total+Synthesis%2C+Structure%2C+and+Oral+Absorption+of+a+Thiazole+Cyclic+Peptide%2C+Sanguinamide+A&doi=10.1021%2Fol3027347"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Total synthesis, structure, and oral absorption of a thiazole cyclic peptide, sanguinamide A</span></div><div class="casAuthors">Nielsen, Daniel S.; Hoang, Huy N.; Lohman, Rink-Jan; Diness, Frederik; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5720-5723</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The first total synthesis and three-dimensional soln. structure are reported for sanguinamide A, a thiazole-contg. cyclic peptide from the sea slug H. sanguineus.  Soln. phase fragment synthesis, solid phase fragment assembly, and soln. macrocyclization were combined to give (I) in 10% yield.  Spectral properties were identical for the natural product, requiring revision of its structure from cis- to trans- amide bond.  Intramol. transannular hydrogen bonds help to bury polar atoms, which enables oral absorption from the gut.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojYQbE5PbtKbVg90H21EOLACvtfcHk0lgcpTHgT13OqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1WqsrfP&md5=be27e8d9c27fbbe58bbc263be17008f7</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fol3027347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol3027347%26sid%3Dliteratum%253Aachs%26aulast%3DNielsen%26aufirst%3DD.%2BS.%26aulast%3DHoang%26aufirst%3DH.%2BN.%26aulast%3DLohman%26aufirst%3DR.-J.%26aulast%3DDiness%26aufirst%3DF.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DTotal%2520Synthesis%252C%2520Structure%252C%2520and%2520Oral%2520Absorption%2520of%2520a%2520Thiazole%2520Cyclic%2520Peptide%252C%2520Sanguinamide%2520A%26jtitle%3DOrg.%2520Lett.%26date%3D2012%26volume%3D14%26spage%3D5720%26epage%3D5723%26doi%3D10.1021%2Fol3027347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conibear, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaousis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johan Rosengren, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, C. I.</span></span> <span> </span><span class="NLM_article-title">Approaches to the Stabilization of Bioactive Epitopes by Grafting and Peptide Cyclization</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1002/bip.22767</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fbip.22767" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=26566734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVSgtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2016&pages=89-100&author=A.+C.+Conibearauthor=S.+Chaousisauthor=T.+Durekauthor=K.+Johan+Rosengrenauthor=D.+J.+Craikauthor=C.+I.+Schroeder&title=Approaches+to+the+Stabilization+of+Bioactive+Epitopes+by+Grafting+and+Peptide+Cyclization&doi=10.1002%2Fbip.22767"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Approaches to the stabilization of bioactive epitopes by grafting and peptide cyclization</span></div><div class="casAuthors">Conibear, Anne C.; Chaousis, Stephanie; Durek, Thomas; Johan Rosengren, K.; Craik, David J.; Schroeder, Christina I.</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">89-100</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Peptides are attracting increasing interest from the pharmaceutical industry because of their specificity and ability to address novel targets, including protein-protein interactions.  However, typically they require stabilization for therapeutic applications owing to their susceptibility to degrdn. by proteases.  Advances in the ability to chem. synthesize peptides and the development of new side-chain and backbone ligation strategies provide new tools to stabilize bioactive peptide epitopes.  Two such epitopes are LyP1, a nine residue peptide that localizes to tumor cells and has potential as an anticancer therapeutic, and RGDS, a tetrapeptide shown to bind to survivin and induce apoptosis.  Here we applied a variety of strategies for the stabilization of LyP1 and RGDS, including side-chain cyclization using "click" chem. and "grafting" the epitopes into two naturally occurring cyclic peptide scaffolds, i.e., θ-defensins and cyclotides.  NMR data showed that the three-disulfide θ-defensin and cyclotide scaffolds accommodated the LyP1 and RGDS epitopes but that scaffolds with fewer disulfide bonds were structurally compromised by inclusion of the LyP1 epitope.  LyP1, LyP1-, and RGDS-grafted peptides that were largely unstructured also had reduced resistance to degrdn. in human serum, showing that grafting into a stable cyclic scaffold is an effective strategy for increasing the stability of a bioactive peptide epitope.  Overall, the study demonstrates several methods for stabilizing peptide epitopes using side-chain or backbone cyclization and illustrates their potential in peptide drug design. © 2015 Wiley Periodicals, Inc.  Biopolymers (Pept Sci) 106: 89-100, 2016.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy6IO4ryYFMLVg90H21EOLACvtfcHk0lgcpTHgT13OqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVSgtb8%253D&md5=7a96a5f30b56a7f95a14fe28167c8e11</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fbip.22767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.22767%26sid%3Dliteratum%253Aachs%26aulast%3DConibear%26aufirst%3DA.%2BC.%26aulast%3DChaousis%26aufirst%3DS.%26aulast%3DDurek%26aufirst%3DT.%26aulast%3DJohan%2BRosengren%26aufirst%3DK.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DSchroeder%26aufirst%3DC.%2BI.%26atitle%3DApproaches%2520to%2520the%2520Stabilization%2520of%2520Bioactive%2520Epitopes%2520by%2520Grafting%2520and%2520Peptide%2520Cyclization%26jtitle%3DBiopolymers%26date%3D2016%26volume%3D106%26spage%3D89%26epage%3D100%26doi%3D10.1002%2Fbip.22767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okumu, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauletti, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander
Velde, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siahaan, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borchardt, R. T.</span></span> <span> </span><span class="NLM_article-title">Effect of Restricted Conformational Flexibility on the Permeation of Model Hexapeptides Across Caco-2 Cell Monolayers</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">175</span>, <span class="refDoi"> DOI: 10.1023/a:1012092409216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1023%2FA%3A1012092409216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=9090704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADyaK2sXitFyqt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1997&pages=169-175&author=F.+W.+Okumuauthor=G.+M.+Paulettiauthor=D.+G.+Vander%0AVeldeauthor=T.+J.+Siahaanauthor=R.+T.+Borchardt&title=Effect+of+Restricted+Conformational+Flexibility+on+the+Permeation+of+Model+Hexapeptides+Across+Caco-2+Cell+Monolayers&doi=10.1023%2Fa%3A1012092409216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of restricted conformational flexibility on the permeation of model hexapeptides across Caco-2 cell monolayers</span></div><div class="casAuthors">Okumu, Franklin W.; Pauletti, Giovanni M.; Vander Velde, David G.; Siahaan, Teruna J.; Borchardt, Ronald T.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">169-175</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Plenum</span>)
        </div><div class="casAbstract">The purpose of this study was to det. how restricted conformational flexibility of hexapeptides influences their cellular permeation characteristics.  Linear (Ac-Trp-Ala-Gly-Gly-X-Ala-NH2; X = Asp, Asn, Lys) and cyclic (cyclo[Trp-Ala-Gly-Gly-X-Ala]; X = Asp, Asn, Lys) hexapeptides were synthesized, and their transport characteristics were assessed using the Caco-2 cell culture model.  The lipophilicities of the hexapeptides were detd. using an immobilized artificial membrane.  Diffusion coeffs. used to calc. mol. radii were detd. by NMR.  Two-dimensional NMR spectroscopy, CD, and mol. dynamic simulations were used to elucidate the most favorable soln. structure of the cyclic Asp-contg. peptide.  The cyclic hexapeptides used in this study were 2-3 times more able to permeate (e.g., Papp = 9.3 ± 0.3 × 10-8 cm/s, X = Asp) the Caco-2 cell monolayer than were their linear analogs (e.g., Papp = 3.2 ± 0.3 × 10-8 cm/s, X = Asp).  In contrast to the linear hexapeptides, the flux of the cyclic hexapeptides was independent of charge.  The cyclic hexapeptides were shown to be more lipophilic than the linear hexapeptides as detd. by their retention times on an immobilized phospholipid column.  Detn. of mol. radii by two different techniques suggests little or no difference in size between the linear and cyclic hexapeptides.  Spectroscopic data indicate that the Asp-contg. linear hexapeptide exists in a dynamic equil. between random coil and β-turn structures while the cyclic Asp-contg. hexapeptide exists in a well-defined compact amphiphilic structure contg. two β-turns.  Cyclization of the linear hexapeptides increased their lipophilicities.  The increased permeation characteristics of the cyclic hexapeptides as compared to their linear analogs appears to be due to an increase in their flux via the transcellular route because of these increased lipophilicities.  Structural analyses of the cyclic Asp-contg. hexapeptide suggest that its well-defined soln. structure and, specifically the existence of two β-turns, explain its greater lipophilicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq53aKqMa4N-bVg90H21EOLACvtfcHk0ljDGpPShbmzvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXitFyqt7w%253D&md5=d5369a0565f8046f7e822f705ce0e859</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1023%2FA%3A1012092409216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1012092409216%26sid%3Dliteratum%253Aachs%26aulast%3DOkumu%26aufirst%3DF.%2BW.%26aulast%3DPauletti%26aufirst%3DG.%2BM.%26aulast%3DVander%2BVelde%26aufirst%3DD.%2BG.%26aulast%3DSiahaan%26aufirst%3DT.%2BJ.%26aulast%3DBorchardt%26aufirst%3DR.%2BT.%26atitle%3DEffect%2520of%2520Restricted%2520Conformational%2520Flexibility%2520on%2520the%2520Permeation%2520of%2520Model%2520Hexapeptides%2520Across%2520Caco-2%2520Cell%2520Monolayers%26jtitle%3DPharm.%2520Res.%26date%3D1997%26volume%3D14%26spage%3D169%26epage%3D175%26doi%3D10.1023%2Fa%3A1012092409216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rand, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, S. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotter, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalgutkar, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varma, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farley, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khunte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Limberakis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathiowetz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Optimizing PK Properties of Cyclic Peptides: The Effect of Side Chain Substitutions on Permeability and Clearance</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1282</span>– <span class="NLM_lpage">1289</span>, <span class="refDoi"> DOI: 10.1039/c2md20203d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1039%2FC2MD20203D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=23133740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVWns7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1282-1289&author=A.+C.+Randauthor=S.+S.+F.+Leungauthor=H.+Engauthor=C.+J.+Rotterauthor=R.+Sharmaauthor=A.+S.+Kalgutkarauthor=Y.+Zhangauthor=M.+V.+Varmaauthor=K.+A.+Farleyauthor=B.+Khunteauthor=C.+Limberakisauthor=D.+A.+Priceauthor=S.+Lirasauthor=A.+M.+Mathiowetzauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Optimizing+PK+Properties+of+Cyclic+Peptides%3A+The+Effect+of+Side+Chain+Substitutions+on+Permeability+and+Clearance&doi=10.1039%2Fc2md20203d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing PK properties of cyclic peptides: the effect of side chain substitutions on permeability and clearance</span></div><div class="casAuthors">Rand, Arthur C.; Leung, Siegfried S. F.; Eng, Heather; Rotter, Charles J.; Sharma, Raman; Kalgutkar, Amit S.; Zhang, Yizhong; Varma, Manthena V.; Farley, Kathleen A.; Khunte, Bhagyashree; Limberakis, Chris; Price, David A.; Liras, Spiros; Mathiowetz, Alan M.; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1282-1289</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A series of cyclic peptides were designed and prepd. to investigate the physicochem. properties that affect oral bioavailability of this chemotype in rats.  In particular, the ionization state of the peptide was examd. by the incorporation of naturally occurring amino acid residues that are charged in differing regions of the gut.  In addn., data was generated in a variety of in vitro assays and the usefulness of this data in predicting the subsequent oral bioavailability obsd. in the rat is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr-tYW5ziK7bVg90H21EOLACvtfcHk0ljDGpPShbmzvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVWns7%252FM&md5=fe1c9f3c7c3e6d48e4e11ee4b5818ef7</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FC2MD20203D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2MD20203D%26sid%3Dliteratum%253Aachs%26aulast%3DRand%26aufirst%3DA.%2BC.%26aulast%3DLeung%26aufirst%3DS.%2BS.%2BF.%26aulast%3DEng%26aufirst%3DH.%26aulast%3DRotter%26aufirst%3DC.%2BJ.%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DVarma%26aufirst%3DM.%2BV.%26aulast%3DFarley%26aufirst%3DK.%2BA.%26aulast%3DKhunte%26aufirst%3DB.%26aulast%3DLimberakis%26aufirst%3DC.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DMathiowetz%26aufirst%3DA.%2BM.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DOptimizing%2520PK%2520Properties%2520of%2520Cyclic%2520Peptides%253A%2520The%2520Effect%2520of%2520Side%2520Chain%2520Substitutions%2520on%2520Permeability%2520and%2520Clearance%26jtitle%3DMedChemComm%26date%3D2012%26volume%3D3%26spage%3D1282%26epage%3D1289%26doi%3D10.1039%2Fc2md20203d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">March, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbenante, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergman, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brinkworth, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickramasinghe, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Begun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Substrate-Based Cyclic Peptidomimetics of Phe-Ile-Val That Inhibit HIV-1 Protease Using a Novel Enzyme-Binding Mode</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">3375</span>– <span class="NLM_lpage">3379</span>, <span class="refDoi"> DOI: 10.1021/ja953790z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja953790z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADyaK28Xhslagsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=1996&pages=3375-3379&author=D.+R.+Marchauthor=G.+Abbenanteauthor=D.+A.+Bergmanauthor=R.+I.+Brinkworthauthor=W.+Wickramasingheauthor=J.+Begunauthor=J.+L.+Martinauthor=D.+P.+Fairlie&title=Substrate-Based+Cyclic+Peptidomimetics+of+Phe-Ile-Val+That+Inhibit+HIV-1+Protease+Using+a+Novel+Enzyme-Binding+Mode&doi=10.1021%2Fja953790z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate-based cyclic peptidomimetics of Phe-Ile-Val that inhibit HIV-1 protease using a novel enzyme-binding mode</span></div><div class="casAuthors">March, Darren R.; Abbenante, Giovanni; Bergman, Douglas A.; Brinkworth, Ross I.; Wickramasinghe, Wasantha; Begun, Jake; Martin, Jennifer L.; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3375-9</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Results are presented for inhibitors of HIV-1 protease that demonstrate a new strategy for developing peptidomimetics, involving the replacement of flexible segments of peptide substrates with conformationally constrained hydrolytically-stable macrocyclic structural mimics.  A 15-membered macrocycle that imitates the tripeptide Phe-Ile-Val was designed and incorporated into the C-terminus of Ac-Leu-Val-Phe-CHOHCH2-{Phe-Ile-Val}-NH2, an inhibitor of HIV-1 protease derived from a substrate sequence.  Advantages of the macrocycle over the acyclic peptide include constraining its components into their bioactive conformation and protecting the amide bonds from enzymic degrdn., the cycle being stable to acid, gastric proteases, and plasma.  Mol. modeling and X-ray structural studies reveal that the cyclic inhibitors have a unique enzyme-binding mode, the sterically unencumbered hydroxyethylamine isostere binds via both its hydroxyl and protonated nitrogen to the anionic Asp25 catalytic residues.  The novel macrocycle superimposes well on the linear peptidic inhibitor for which it was designed as a structural mimic.  Structural mimicry led to functional mimicry as shown by comparable inhibition of the protease by cyclic and acyclic mols.  Further modification of the acyclic N-terminus (Leu-Val-Phe) gave stable, water-sol., potent inhibitors of HIV-1 protease.  This approach may have general application to the development of mimetics of other bioactive peptides, including inhibitors of other enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpleApAoH1u-bVg90H21EOLACvtfcHk0liolPA7jafHcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xhslagsr8%253D&md5=716171632ee911b5e5be59a87a584a50</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fja953790z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja953790z%26sid%3Dliteratum%253Aachs%26aulast%3DMarch%26aufirst%3DD.%2BR.%26aulast%3DAbbenante%26aufirst%3DG.%26aulast%3DBergman%26aufirst%3DD.%2BA.%26aulast%3DBrinkworth%26aufirst%3DR.%2BI.%26aulast%3DWickramasinghe%26aufirst%3DW.%26aulast%3DBegun%26aufirst%3DJ.%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DSubstrate-Based%2520Cyclic%2520Peptidomimetics%2520of%2520Phe-Ile-Val%2520That%2520Inhibit%2520HIV-1%2520Protease%2520Using%2520a%2520Novel%2520Enzyme-Binding%2520Mode%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1996%26volume%3D118%26spage%3D3375%26epage%3D3379%26doi%3D10.1021%2Fja953790z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adessi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soto, C.</span></span> <span> </span><span class="NLM_article-title">Converting a Peptide into a Drug: Strategies to Improve Stability and Bioavailability</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">963</span>– <span class="NLM_lpage">978</span>, <span class="refDoi"> DOI: 10.2174/0929867024606731</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.2174%2F0929867024606731" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=11966456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD38XjtlKjtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=963-978&author=C.+Adessiauthor=C.+Soto&title=Converting+a+Peptide+into+a+Drug%3A+Strategies+to+Improve+Stability+and+Bioavailability&doi=10.2174%2F0929867024606731"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Converting a peptide into a drug: Strategies to improve stability and bioavailability</span></div><div class="casAuthors">Adessi, Celine; Soto, Claudio</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">963-978</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers</span>)
        </div><div class="casAbstract">A review.  The discovery of peptide hormones, growth factors and neuropeptides implicated in vital biol. functions of our organism has increased interest in therapeutic use of short peptides.  However, the development of peptides as clin. useful drugs is greatly limited by their poor metabolic stability and low bioavailability, which is due in part to their inability to readily cross membrane barriers such as the intestinal and blood-brain barriers.  The aim of peptide medicinal chem. is, therefore, to develop strategies to overcome these problems.  Recent progress in chem. synthesis and design have resulted in several strategies for producing modified peptides and mimetics with lower susceptibility to proteolysis and improved bioavailability, which has increased the probability of obtaining useful drugs structurally related to parent peptides.  This review describes different exptl. approaches to transforming a peptide into a potential drug and provides examples of the usefulness of these strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_7AaDVSzEfbVg90H21EOLACvtfcHk0liolPA7jafHcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjtlKjtbk%253D&md5=95cae8662dbf6eee4737276b40bf7209</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.2174%2F0929867024606731&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867024606731%26sid%3Dliteratum%253Aachs%26aulast%3DAdessi%26aufirst%3DC.%26aulast%3DSoto%26aufirst%3DC.%26atitle%3DConverting%2520a%2520Peptide%2520into%2520a%2520Drug%253A%2520Strategies%2520to%2520Improve%2520Stability%2520and%2520Bioavailability%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2002%26volume%3D9%26spage%3D963%26epage%3D978%26doi%3D10.2174%2F0929867024606731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halle, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selincer, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byk, G.</span></span> <span> </span><span class="NLM_article-title">Backbone Cyclization: A New Method for Conferring Conformational Constraint on Peptides</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">745</span>– <span class="NLM_lpage">750</span>, <span class="refDoi"> DOI: 10.1002/bip.360310619</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fbip.360310619" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1718473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADyaK3MXltFGjur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1991&pages=745-750&author=C.+Gilonauthor=D.+Halleauthor=M.+Chorevauthor=Z.+Selincerauthor=G.+Byk&title=Backbone+Cyclization%3A+A+New+Method+for+Conferring+Conformational+Constraint+on+Peptides&doi=10.1002%2Fbip.360310619"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Backbone cyclization:  a new method for conferring conformational constraint on peptides</span></div><div class="casAuthors">Gilon, Chaim; Halle, David; Chorev, Michael; Selinger, Zvi; Byk, Gerardo</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">745-50</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    </div><div class="casAbstract">This article describes a new concept of medium- and long-range cyclization of peptides through "backbone cyclization".  In this approach, conformational constraints are conferred on a peptide by linking ω-substituted alkylidene chains replacing Nα or Cα hydrogens in a peptide backbone.  Backbone cyclization, which is divided into N-backbone and C-backbone cyclizations, allow for new modes of cyclization in addn. to the classical ones that are limited to cyclization through the side chains and/or the amino or carboxyl terminal groups.  The article also describes the application of the N-backbone cyclization to the active region of substance P.  Conformational constraints of this peptide by the classical cyclization modes led to inactive analogs whereas N-backbone cyclization provided an active, selective, and metabolically stable analog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo10ie5nZFwl7Vg90H21EOLACvtfcHk0liolPA7jafHcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXltFGjur0%253D&md5=de539b2a1242d0ec87e3c50186bc8811</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fbip.360310619&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.360310619%26sid%3Dliteratum%253Aachs%26aulast%3DGilon%26aufirst%3DC.%26aulast%3DHalle%26aufirst%3DD.%26aulast%3DChorev%26aufirst%3DM.%26aulast%3DSelincer%26aufirst%3DZ.%26aulast%3DByk%26aufirst%3DG.%26atitle%3DBackbone%2520Cyclization%253A%2520A%2520New%2520Method%2520for%2520Conferring%2520Conformational%2520Constraint%2520on%2520Peptides%26jtitle%3DBiopolymers%26date%3D1991%26volume%3D31%26spage%3D745%26epage%3D750%26doi%3D10.1002%2Fbip.360310619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pauletti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangwar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siahaan, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aubé, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borchardt, R. T.</span></span> <span> </span><span class="NLM_article-title">Improvement of Oral Peptide Bioavailability: Peptidomimetics and Prodrug Strategies</span>. <i>Adv. Drug Deliv. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">256</span>, <span class="refDoi"> DOI: 10.1016/s0169-409x(97)00045-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1016%2FS0169-409X%2897%2900045-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10837560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjs12ktL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1997&pages=235-256&author=G.+Paulettiauthor=S.+Gangwarauthor=T.+J.+Siahaanauthor=J.+Aub%C3%A9author=R.+T.+Borchardt&title=Improvement+of+Oral+Peptide+Bioavailability%3A+Peptidomimetics+and+Prodrug+Strategies&doi=10.1016%2Fs0169-409x%2897%2900045-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Improvement of oral peptide bioavailability: Peptidomimetics and prodrug strategies</span></div><div class="casAuthors">Pauletti, Giovanni M.; Gangwar, Sanjeev; Siahaan, Teruna J.; Aube, Jeffrey; Borchardt, Ronald T.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2,3</span>),
    <span class="NLM_cas:pages">235-256</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review with 209 refs.  Clin. development of orally active peptide drugs has been restricted by their unfavorable physicochem. properties, which limit their intestinal mucosal permeation and their lack of stability against enzymic degrdn.  Successful oral delivery of peptides will depend, therefore, on strategies designed to alter the physicochem. characteristics of these potential drugs, without changing their biol. activity, in order to overcome the phys. and biochem. barrier properties of the intestinal cells.  This manuscript will focus on the physiol. limitations for oral peptide delivery and on various strategies using chem. modifications to improve oral bioavailability of peptide-based drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_apmBvn28dLVg90H21EOLACvtfcHk0ljHp3tyINPJ2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjs12ktL0%253D&md5=de147c42769ae91c7ad3d49321f3ddc3</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2897%2900045-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252897%252900045-8%26sid%3Dliteratum%253Aachs%26aulast%3DPauletti%26aufirst%3DG.%26aulast%3DGangwar%26aufirst%3DS.%26aulast%3DSiahaan%26aufirst%3DT.%2BJ.%26aulast%3DAub%25C3%25A9%26aufirst%3DJ.%26aulast%3DBorchardt%26aufirst%3DR.%2BT.%26atitle%3DImprovement%2520of%2520Oral%2520Peptide%2520Bioavailability%253A%2520Peptidomimetics%2520and%2520Prodrug%2520Strategies%26jtitle%3DAdv.%2520Drug%2520Deliv.%2520Rev.%26date%3D1997%26volume%3D27%26spage%3D235%26epage%3D256%26doi%3D10.1016%2Fs0169-409x%2897%2900045-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burton, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conradi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, N. F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgers, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borchardt, R. T.</span></span> <span> </span><span class="NLM_article-title">How Structural Features Influence the Biomembrane Permeability of Peptides</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">1336</span>– <span class="NLM_lpage">1340</span>, <span class="refDoi"> DOI: 10.1021/js960067d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1021%2Fjs960067d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=8961149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADyaK28Xms1Cmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=1996&pages=1336-1340&author=P.+S.+Burtonauthor=R.+A.+Conradiauthor=N.+F.+H.+Hoauthor=A.+R.+Hilgersauthor=R.+T.+Borchardt&title=How+Structural+Features+Influence+the+Biomembrane+Permeability+of+Peptides&doi=10.1021%2Fjs960067d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">How Structural Features Influence the Biomembrane Permeability of Peptides</span></div><div class="casAuthors">Burton, Philip S.; Conradi, Robert A.; Ho, Norman F. H.; Hilgers, Allen R.; Borchardt, Ronald T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1336-1340</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 55 refs.  Successful drug development requires not only optimization of specific and potent pharmacol. activity at the target site, but also efficient delivery to that site.  Many promising new peptides with novel therapeutic potential for the treatment of AIDS, cardiovascular diseases, and CNS disorders have been identified, yet their clin. utility has been limited by delivery problems.  Along with metab., a major factor contributing to the poor bioavailability of peptides is thought to be inefficient transport across cell membranes.  At the present time, the reasons for this poor transport are poorly understood.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAZOZxHGFusbVg90H21EOLACvtfcHk0ljHp3tyINPJ2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xms1Cmt7g%253D&md5=600e4f60ad068cf6f1f542fa1116c79d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjs960067d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjs960067d%26sid%3Dliteratum%253Aachs%26aulast%3DBurton%26aufirst%3DP.%2BS.%26aulast%3DConradi%26aufirst%3DR.%2BA.%26aulast%3DHo%26aufirst%3DN.%2BF.%2BH.%26aulast%3DHilgers%26aufirst%3DA.%2BR.%26aulast%3DBorchardt%26aufirst%3DR.%2BT.%26atitle%3DHow%2520Structural%2520Features%2520Influence%2520the%2520Biomembrane%2520Permeability%2520of%2520Peptides%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D1996%26volume%3D85%26spage%3D1336%26epage%3D1340%26doi%3D10.1021%2Fjs960067d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGeary, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Peptidomimetics: Potential for Drug Development</span>. <i>Curr. Opin. Drug Discov. Dev.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">208</span>– <span class="NLM_lpage">217</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=19649821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADyaK1MXntlyksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1998&pages=208-217&author=R.+P.+McGearyauthor=D.+P.+Fairlie&title=Macrocyclic+Peptidomimetics%3A+Potential+for+Drug+Development"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic peptidomimetics: potential for drug development</span></div><div class="casAuthors">McGeary, Ross P.; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">208-217</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Current Drugs Ltd.</span>)
        </div><div class="casAbstract">A review with 57 refs. focusing mainly on the structure-activity relationship for synthetic macrocyclic peptidomimetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyu9dgfXnpG7Vg90H21EOLACvtfcHk0ljHp3tyINPJ2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntlyksw%253D%253D&md5=2d443820e8b50d42df4d34f115dee467</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMcGeary%26aufirst%3DR.%2BP.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DMacrocyclic%2520Peptidomimetics%253A%2520Potential%2520for%2520Drug%2520Development%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discov.%2520Dev.%26date%3D1998%26volume%3D1%26spage%3D208%26epage%3D217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhardwaj, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahl, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmore, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheneval, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchko, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulavarti, S. V. S. R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eletsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnsen, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greisen, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linsky, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonneau, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coutsias, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correnti, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szyperski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span> <span> </span><span class="NLM_article-title">Accurate De Novo Design of Hyperstable Constrained Peptides</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>538</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">335</span>, <span class="refDoi"> DOI: 10.1038/nature19791</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1038%2Fnature19791" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=27626386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWiurnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=538&publication_year=2016&pages=329-335&author=G.+Bhardwajauthor=V.+K.+Mulliganauthor=C.+D.+Bahlauthor=J.+M.+Gilmoreauthor=P.+J.+Harveyauthor=O.+Chenevalauthor=G.+W.+Buchkoauthor=S.+V.+S.+R.+K.+Pulavartiauthor=A.+Eletskyauthor=P.-S.+Huangauthor=W.+A.+Johnsenauthor=P.+J.+Greisenauthor=G.+J.+Rocklinauthor=Y.+Songauthor=T.+W.+Linskyauthor=A.+Watkinsauthor=S.+A.+Rettieauthor=X.+Xuauthor=L.+P.+Carterauthor=R.+Bonneauauthor=J.+M.+Olsonauthor=E.+Coutsiasauthor=C.+E.+Correntiauthor=T.+Szyperskiauthor=D.+J.+Craikauthor=D.+Bakerauthor=D.+Baker&title=Accurate+De+Novo+Design+of+Hyperstable+Constrained+Peptides&doi=10.1038%2Fnature19791"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate de novo design of hyperstable constrained peptides</span></div><div class="casAuthors">Bhardwaj, Gaurav; Mulligan, Vikram Khipple; Bahl, Christopher D.; Gilmore, Jason M.; Harvey, Peta J.; Cheneval, Olivier; Buchko, Garry W.; Pulavarti, Surya V. S. R. K.; Kaas, Quentin; Eletsky, Alexander; Huang, Po-Ssu; Johnsen, William A.; Greisen, Per Jr; Rocklin, Gabriel J.; Song, Yifan; Linsky, Thomas W.; Watkins, Andrew; Rettie, Stephen A.; Xu, Xianzhong; Carter, Lauren P.; Bonneau, Richard; Olson, James M.; Coutsias, Evangelos; Correnti, Colin E.; Szyperski, Thomas; Craik, David J.; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">538</span>
        (<span class="NLM_cas:issue">7625</span>),
    <span class="NLM_cas:pages">329-335</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Naturally occurring, pharmacol. active peptides constrained with covalent crosslinks generally have shapes that have evolved to fit precisely into binding pockets on their targets.  Such peptides can have excellent pharmaceutical properties, combining the stability and tissue penetration of small-mol. drugs with the specificity of much larger protein therapeutics.  The ability to design constrained peptides with precisely specified tertiary structures would enable the design of shape-complementary inhibitors of arbitrary targets.  Here we describe the development of computational methods for accurate de novo design of conformationally restricted peptides, and the use of these methods to design 18-47 residue, disulfide-crosslinked peptides, a subset of which are heterochiral and/or N-C backbone-cyclized.  Both genetically encodable and non-canonical peptides are exceptionally stable to thermal and chem. denaturation, and 12 exptl. detd. X-ray and NMR structures are nearly identical to the computational design models.  The computational design methods and stable scaffolds presented here provide the basis for development of a new generation of peptide-based drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHeMGaFeL_kbVg90H21EOLACvtfcHk0lj-ZFhHTFU40A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWiurnL&md5=2d2b551ad78a2d76c99eeaba0c763c8e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnature19791&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature19791%26sid%3Dliteratum%253Aachs%26aulast%3DBhardwaj%26aufirst%3DG.%26aulast%3DMulligan%26aufirst%3DV.%2BK.%26aulast%3DBahl%26aufirst%3DC.%2BD.%26aulast%3DGilmore%26aufirst%3DJ.%2BM.%26aulast%3DHarvey%26aufirst%3DP.%2BJ.%26aulast%3DCheneval%26aufirst%3DO.%26aulast%3DBuchko%26aufirst%3DG.%2BW.%26aulast%3DPulavarti%26aufirst%3DS.%2BV.%2BS.%2BR.%2BK.%26aulast%3DEletsky%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DP.-S.%26aulast%3DJohnsen%26aufirst%3DW.%2BA.%26aulast%3DGreisen%26aufirst%3DP.%2BJ.%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLinsky%26aufirst%3DT.%2BW.%26aulast%3DWatkins%26aufirst%3DA.%26aulast%3DRettie%26aufirst%3DS.%2BA.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DCarter%26aufirst%3DL.%2BP.%26aulast%3DBonneau%26aufirst%3DR.%26aulast%3DOlson%26aufirst%3DJ.%2BM.%26aulast%3DCoutsias%26aufirst%3DE.%26aulast%3DCorrenti%26aufirst%3DC.%2BE.%26aulast%3DSzyperski%26aufirst%3DT.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DBaker%26aufirst%3DD.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DAccurate%2520De%2520Novo%2520Design%2520of%2520Hyperstable%2520Constrained%2520Peptides%26jtitle%3DNature%26date%3D2016%26volume%3D538%26spage%3D329%26epage%3D335%26doi%3D10.1038%2Fnature19791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soumana, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurt Yilmaz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prachanronarong, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aydin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Structural and Thermodynamic Effects of Macrocyclization in HCV NS3/4A Inhibitor MK-5172</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">900</span>– <span class="NLM_lpage">909</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00647</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00647" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVGitb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=900-909&author=D.+I.+Soumanaauthor=N.+Kurt+Yilmazauthor=K.+L.+Prachanronarongauthor=C.+Aydinauthor=A.+Aliauthor=C.+A.+Schiffer&title=Structural+and+Thermodynamic+Effects+of+Macrocyclization+in+HCV+NS3%2F4A+Inhibitor+MK-5172&doi=10.1021%2Facschembio.5b00647"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and Thermodynamic Effects of Macrocyclization in HCV NS3/4A Inhibitor MK-5172</span></div><div class="casAuthors">Soumana, Djade I.; Kurt Yilmaz, Nese; Prachanronarong, Kristina L.; Aydin, Cihan; Ali, Akbar; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">900-909</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recent advances in direct-acting antivirals against Hepatitis C Virus (HCV) have led to the development of potent inhibitors, including MK-5172, that target the viral NS3/4A protease with relatively low susceptibility to resistance.  MK-5172 has a P2-P4 macrocycle and a unique binding mode among current protease inhibitors where the P2 quinoxaline packs against the catalytic residues H57 and D81.  However, the effect of macrocyclization on this binding mode is not clear, as is the relation between macrocyclization, thermodn. stabilization, and susceptibility to the resistance mutation A156T.  We have detd. high-resoln. crystal structures of linear and P1-P3 macrocyclic analogs of MK-5172 bound to WT and A156T protease and compared these structures, their mol. dynamics, and exptl. binding thermodn. to the parent compd.  We find that the "unique" binding mode of MK-5172 is conserved even when the P2-P4 macrocycle is removed or replaced with a P1-P3 macrocycle.  While beneficial to decreasing the entropic penalty assocd. with binding, the constraint exerted by the P2-P4 macrocycle prevents efficient rearrangement to accommodate the A156T mutation, a deficit alleviated in the linear and P1-P3 analogs.  Design of macrocyclic inhibitors against NS3/4A needs to achieve the best balance between exerting optimal conformational constraint for enhancing potency, fitting within the substrate envelope and allowing adaptability to be robust against resistance mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVaszg0NfWS7Vg90H21EOLACvtfcHk0lj-ZFhHTFU40A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVGitb3F&md5=4f3cadfdedbb5af22d0ef55c088d3687</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00647%26sid%3Dliteratum%253Aachs%26aulast%3DSoumana%26aufirst%3DD.%2BI.%26aulast%3DKurt%2BYilmaz%26aufirst%3DN.%26aulast%3DPrachanronarong%26aufirst%3DK.%2BL.%26aulast%3DAydin%26aufirst%3DC.%26aulast%3DAli%26aufirst%3DA.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DStructural%2520and%2520Thermodynamic%2520Effects%2520of%2520Macrocyclization%2520in%2520HCV%2520NS3%252F4A%2520Inhibitor%2520MK-5172%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D900%26epage%3D909%26doi%3D10.1021%2Facschembio.5b00647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delorbe, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clements, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whiddon, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. F.</span></span> <span> </span><span class="NLM_article-title">Thermodynamic and Structural Effects of Macrocyclic Constraints in Protein-Ligand Interactions</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">448</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/ml100142y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100142y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A280%3ADC%252BC2srmt1OisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=448-452&author=J.+E.+Delorbeauthor=J.+H.+Clementsauthor=B.+B.+Whiddonauthor=S.+F.+Martin&title=Thermodynamic+and+Structural+Effects+of+Macrocyclic+Constraints+in+Protein-Ligand+Interactions&doi=10.1021%2Fml100142y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Thermodynamic and Structural Effects of Macrocyclization as a Constraining Method in Protein-Ligand Interactions</span></div><div class="casAuthors">Delorbe John E; Clements John H; Whiddon Benjamin B; Martin Stephen F</div><div class="citationInfo"><span class="NLM_cas:title">ACS medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">448-452</span>
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    </div><div class="casAbstract">The thermodynamic and structural effects of macrocyclization as a tactic for stabilizing the biologically-active conformation of Grb2 SH2 binding peptides were investigated using isothermal titration calorimetry and x-ray crystallography. 23-Membered macrocycles containing the sequence pYVN were slightly more potent than their linear controls; however, preorganization did not necessarily eventuate in a more favorable binding entropy.  Structures of complexes of macrocycle 7 and its acyclic control 8 are similar except for differences in relative orientations of corresponding atoms in the linking moieties of 7 and 8.  There are no differences in the number of direct or water-mediated protein-ligand contacts that might account for the less favorable binding enthalpy of 7; however, an intramolecular hydrogen bond between the pY and pY+3 residues in 8 that is absent in 7 may be a factor.  These studies highlight the difficulties associated with correlating energetics and structure in protein-ligand interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTdL3gvK7FnJAwxfk_AGB-EfW6udTcc2eaMPNenmONj7rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srmt1OisA%253D%253D&md5=bec2aeb7e9127f0f6310dbccf2663c76</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fml100142y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100142y%26sid%3Dliteratum%253Aachs%26aulast%3DDelorbe%26aufirst%3DJ.%2BE.%26aulast%3DClements%26aufirst%3DJ.%2BH.%26aulast%3DWhiddon%26aufirst%3DB.%2BB.%26aulast%3DMartin%26aufirst%3DS.%2BF.%26atitle%3DThermodynamic%2520and%2520Structural%2520Effects%2520of%2520Macrocyclic%2520Constraints%2520in%2520Protein-Ligand%2520Interactions%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D1%26spage%3D448%26epage%3D452%26doi%3D10.1021%2Fml100142y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hruby, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Obeidi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazmierski, W.</span></span> <span> </span><span class="NLM_article-title">Emerging Approaches in the Molecular Design of Receptor-Selective Peptide Ligands: Conformational, Topographical and Dynamic Considerations</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>268</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1042/bj2680249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1042%2Fbj2680249" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=2163604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADyaK3cXktFOqs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=268&publication_year=1990&pages=249-262&author=V.+J.+Hrubyauthor=F.+Al-Obeidiauthor=W.+Kazmierski&title=Emerging+Approaches+in+the+Molecular+Design+of+Receptor-Selective+Peptide+Ligands%3A+Conformational%2C+Topographical+and+Dynamic+Considerations&doi=10.1042%2Fbj2680249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging approaches in the molecular design of receptor-selective peptide ligands:  conformational, topographical and dynamic considerations</span></div><div class="casAuthors">Hruby, Victor J.; Al-Obeidi, Fahad; Kazmierski, Wieslaw</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">268</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-62</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    </div><div class="casAbstract">A review, with 107 refs., of the design of peptide and protein ligands (hormones) with specific phys., chem., and biol. properties.  Specific topics discussed were conformational constraint, structure-biol. activities relations, and rational drugs as well as topog. design possibilities and prospects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7mFpCF_I0xrVg90H21EOLACvtfcHk0lhWftynTjmzyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXktFOqs7Y%253D&md5=f0b6ef00b0d688997db977667caee697</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1042%2Fbj2680249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj2680249%26sid%3Dliteratum%253Aachs%26aulast%3DHruby%26aufirst%3DV.%2BJ.%26aulast%3DAl-Obeidi%26aufirst%3DF.%26aulast%3DKazmierski%26aufirst%3DW.%26atitle%3DEmerging%2520Approaches%2520in%2520the%2520Molecular%2520Design%2520of%2520Receptor-Selective%2520Peptide%2520Ligands%253A%2520Conformational%252C%2520Topographical%2520and%2520Dynamic%2520Considerations%26jtitle%3DBiochem.%2520J.%26date%3D1990%26volume%3D268%26spage%3D249%26epage%3D262%26doi%3D10.1042%2Fbj2680249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and Computational Approaches to Estimate Solubility and Permeability in Drug Discovery and Development Settings</span>. <i>Adv. Drug Deliv. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1016/s0169-409x(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+Computational+Approaches+to+Estimate+Solubility+and+Permeability+in+Drug+Discovery+and+Development+Settings&doi=10.1016%2Fs0169-409x%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0lhWftynTjmzyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520Computational%2520Approaches%2520to%2520Estimate%2520Solubility%2520and%2520Permeability%2520in%2520Drug%2520Discovery%2520and%2520Development%2520Settings%26jtitle%3DAdv.%2520Drug%2520Deliv.%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2Fs0169-409x%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veber, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopple, K. D.</span></span> <span> </span><span class="NLM_article-title">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">2615</span>– <span class="NLM_lpage">2623</span>, <span class="refDoi"> DOI: 10.1021/jm020017n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020017n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2615-2623&author=D.+F.+Veberauthor=S.+R.+Johnsonauthor=H.-Y.+Chengauthor=B.+R.+Smithauthor=K.+W.+Wardauthor=K.+D.+Kopple&title=Molecular+Properties+That+Influence+the+Oral+Bioavailability+of+Drug+Candidates&doi=10.1021%2Fjm020017n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span></div><div class="casAuthors">Veber, Daniel F.; Johnson, Stephen R.; Cheng, Hung-Yuan; Smith, Brian R.; Ward, Keith W.; Kopple, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2615-2623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oral bioavailability measurements in rats for over 1100 drug candidates studied at Smith-Kline Beecham Pharmaceuticals (now Glaxo Smith-Kline) have allowed us to analyze the relative importance of mol. properties considered to influence that drug property.  Reduced mol. flexibility, as measured by the no. of rotatable bonds, and low polar surface area or total hydrogen bond count (sum of donors and acceptors) are found to be important predictors of good oral bioavailability, independent of mol. wt.  That on av. both the no. of rotatable bonds and polar surface area or hydrogen bond count tend to increase with mol. wt. may in part explain the success of the mol. wt. parameter in predicting oral bioavailability.  The commonly applied mol. wt. cutoff at 500 does not itself significantly sep. compds. with poor oral bioavailability from those with acceptable values in this extensive data set.  Our observations suggest that compds. which meet only the 2 criteria of (1) 10 or fewer rotatable bonds and (2) polar surface area ≤140 Å2 (or 12 or fewer H-bond donors and acceptors) will have a high probability of good oral bioavailability in the rat.  Data sets for the artificial membrane permeation rate and for clearance in the rat were also examd.  Reduced polar surface area correlates better with increased permeation rate than does lipophilicity (C log P), and increased rotatable bond count has a neg. effect on the permeation rate.  A threshold permeation rate is a prerequisite of oral bioavailability.  The rotatable bond count does not correlate with the data examd. here for the in vivo clearance rate in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3JtJEkO7yVLVg90H21EOLACvtfcHk0lhWftynTjmzyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D&md5=eaad26ed6a259de82ad65a8834fc397d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm020017n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020017n%26sid%3Dliteratum%253Aachs%26aulast%3DVeber%26aufirst%3DD.%2BF.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DCheng%26aufirst%3DH.-Y.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DKopple%26aufirst%3DK.%2BD.%26atitle%3DMolecular%2520Properties%2520That%2520Influence%2520the%2520Oral%2520Bioavailability%2520of%2520Drug%2520Candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2615%26epage%3D2623%26doi%3D10.1021%2Fjm020017n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vieth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutherland, J. J.</span></span> <span> </span><span class="NLM_article-title">Dependence of Molecular Properties on Proteomic Family for Marketed Oral Drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">3451</span>– <span class="NLM_lpage">3453</span>, <span class="refDoi"> DOI: 10.1021/jm0603825</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0603825" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD28XksFKks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=3451-3453&author=M.+Viethauthor=J.+J.+Sutherland&title=Dependence+of+Molecular+Properties+on+Proteomic+Family+for+Marketed+Oral+Drugs&doi=10.1021%2Fjm0603825"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Dependence of Molecular Properties on Proteomic Family for Marketed Oral Drugs</span></div><div class="casAuthors">Vieth, Michal; Sutherland, Jeffrey J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3451-3453</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An assocn. of drugs with their proteomic family reveals that mol. properties of drugs targeting proteases, lipid and peptide G-protein-coupled receptors (GPCRs), and nuclear hormone receptors significantly exceed limits for some properties in the "rule of five", while drugs targeting cytochrome P450s, biogenic amine GPCRs, and transporters have significantly lower values for certain properties.  Also, the variation in drug targets appears to be a factor explaining increasing mol. wt. over time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOC7vpvPxrV7Vg90H21EOLACvtfcHk0lhWftynTjmzyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XksFKks70%253D&md5=9272583ace87f6080bd0e1b01ad7c657</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm0603825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0603825%26sid%3Dliteratum%253Aachs%26aulast%3DVieth%26aufirst%3DM.%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26atitle%3DDependence%2520of%2520Molecular%2520Properties%2520on%2520Proteomic%2520Family%2520for%2520Marketed%2520Oral%2520Drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D3451%26epage%3D3453%26doi%3D10.1021%2Fjm0603825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paolini, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapland, R. H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Hoorn, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Global Mapping of Pharmacological Space</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">805</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1038/nbt1228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1038%2Fnbt1228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=16841068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=805-815&author=G.+V.+Paoliniauthor=R.+H.+B.+Shaplandauthor=W.+P.+van+Hoornauthor=J.+S.+Masonauthor=A.+L.+Hopkins&title=Global+Mapping+of+Pharmacological+Space&doi=10.1038%2Fnbt1228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Global mapping of pharmacological space</span></div><div class="casAuthors">Paolini, Gaia V.; Shapland, Richard H. B.; van Hoorn, Willem P.; Mason, Jonathan S.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">805-815</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We present the global mapping of pharmacol. space by the integration of several vast sources of medicinal chem. structure-activity relationships (SAR) data.  Our comprehensive mapping of pharmacol. space enables us to identify confidently the human targets for which chem. tools and drugs have been discovered to date.  The integration of SAR data from diverse sources by unique canonical chem. structure, protein sequence and disease indication enables the construction of a ligand-target matrix to explore the global relationships between chem. structure and biol. targets.  Using the data matrix, we are able to catalog the links between proteins in chem. space as a polypharmacol. interaction network.  We demonstrate that probabilistic models can be used to predict pharmacol. from a large knowledge base.  The relationships between proteins, chem. structures and drug-like properties provide a framework for developing a probabilistic approach to drug discovery that can be exploited to increase research productivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV_CE47zZq8LVg90H21EOLACvtfcHk0ljkf2UlEzE79g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D&md5=f559b34692cc903a1b503deb07030c5d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnbt1228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1228%26sid%3Dliteratum%253Aachs%26aulast%3DPaolini%26aufirst%3DG.%2BV.%26aulast%3DShapland%26aufirst%3DR.%2BH.%2BB.%26aulast%3Dvan%2BHoorn%26aufirst%3DW.%2BP.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DGlobal%2520Mapping%2520of%2520Pharmacological%2520Space%26jtitle%3DNat.%2520Biotechnol.%26date%3D2006%26volume%3D24%26spage%3D805%26epage%3D815%26doi%3D10.1038%2Fnbt1228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gottschling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuster, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holzmann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span> <span> </span><span class="NLM_article-title">Combinatorial and Rational Strategies to Develop Nonpeptidic Α4β7-Integrin Antagonists from Cyclic Peptides</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">3007</span>– <span class="NLM_lpage">3011</span>, <span class="refDoi"> DOI: 10.1002/1521-3773(20020816)41:16<3007::aid-anie3007>3.0.co;2-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2F1521-3773%2820020816%2941%3A16%3C3007%3A%3AAID-ANIE3007%3E3.0.CO%3B2-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD38XmslSrt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=3007-3011&author=D.+Gottschlingauthor=J.+Boerauthor=A.+Schusterauthor=B.+Holzmannauthor=H.+Kessler&title=Combinatorial+and+Rational+Strategies+to+Develop+Nonpeptidic+%CE%914%CE%B27-Integrin+Antagonists+from+Cyclic+Peptides&doi=10.1002%2F1521-3773%2820020816%2941%3A16%3C3007%3A%3Aaid-anie3007%3E3.0.co%3B2-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Combinatorial and rational strategies to develop non-peptidic α4β7-integrin antagonists from cyclic peptides</span></div><div class="casAuthors">Gottschling, Dirk; Boer, Jurgen; Schuster, Anja; Holzmann, Bernhard; Kessler, Horst</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3007-3011</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">Non-peptidic compds. derived from cyclic peptides such as cyclo(F-L-D-F-D-p) were developed for inhibiting the α4β7-integrin/MAdCAM-1 interaction.  It possible to develop low-mol. wt. non-peptidic α4β7-integrin antagonists by using rational and combinatorial strategies.  A stepwise procedure, i.e., protein sequence → cyclic, constrained peptides → peptidomimetics → nonpeptide, can be a valuable method to develop new drugs.  Peptidomimetics based on isoquinoline-3-carbonyl-Leu-Asp-Thr-OH were prepd. and tested for their effect on α4β7- and α4β1-integrin-mediated cell adhesion to MAdCAM-1 and VCAM-1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1wX_AdejE77Vg90H21EOLACvtfcHk0ljkf2UlEzE79g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmslSrt7o%253D&md5=b9c124df0448bbfdb393502362f99101</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2F1521-3773%2820020816%2941%3A16%3C3007%3A%3AAID-ANIE3007%3E3.0.CO%3B2-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1521-3773%252820020816%252941%253A16%253C3007%253A%253AAID-ANIE3007%253E3.0.CO%253B2-3%26sid%3Dliteratum%253Aachs%26aulast%3DGottschling%26aufirst%3DD.%26aulast%3DBoer%26aufirst%3DJ.%26aulast%3DSchuster%26aufirst%3DA.%26aulast%3DHolzmann%26aufirst%3DB.%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DCombinatorial%2520and%2520Rational%2520Strategies%2520to%2520Develop%2520Nonpeptidic%2520%25CE%25914%25CE%25B27-Integrin%2520Antagonists%2520from%2520Cyclic%2520Peptides%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2002%26volume%3D41%26spage%3D3007%26epage%3D3011%26doi%3D10.1002%2F1521-3773%2820020816%2941%3A16%3C3007%3A%3Aaid-anie3007%3E3.0.co%3B2-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mallinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span> <span> </span><span class="NLM_article-title">Macrocycles in New Drug Discovery</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1409</span>– <span class="NLM_lpage">1438</span>, <span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-1438&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+New+Drug+Discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0ljkf2UlEzE79g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520New%2520Drug%2520Discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D1438%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yudin, A. K.</span></span> <span> </span><span class="NLM_article-title">Contemporary Strategies for Peptide Macrocyclization</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">509</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1038/nchem.1062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1038%2Fnchem.1062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=21697871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvFWjtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=509-524&author=C.+J.+Whiteauthor=A.+K.+Yudin&title=Contemporary+Strategies+for+Peptide+Macrocyclization&doi=10.1038%2Fnchem.1062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Contemporary strategies for peptide macrocyclization</span></div><div class="casAuthors">White, Christopher J.; Yudin, Andrei K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">509-524</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Peptide macrocycles have found applications that range from drug discovery to nanomaterials.  These ring-shaped mols. have shown remarkable capacity for functional fine-tuning.  Such capacity is enabled by the possibility of adjusting the peptide conformation using the techniques of chem. synthesis.  Cyclic peptides have been difficult, and often impossible, to prep. using traditional synthetic methods.  For macrocyclization to occur, the activated peptide must adopt an entropically disfavored pre-cyclization conformation before forming the desired product.  Here, recent solns. to some of the major challenges in this important area of contemporary synthesis were reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUTda6OkfQprVg90H21EOLACvtfcHk0lj0YZuATITpDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvFWjtrc%253D&md5=32dfd94692ca20901dc8828bcd27ae8c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1062%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DC.%2BJ.%26aulast%3DYudin%26aufirst%3DA.%2BK.%26atitle%3DContemporary%2520Strategies%2520for%2520Peptide%2520Macrocyclization%26jtitle%3DNat.%2520Chem.%26date%3D2011%26volume%3D3%26spage%3D509%26epage%3D524%26doi%3D10.1038%2Fnchem.1062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passioura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Technologies for the Synthesis of MRNA-Encoding Libraries and Discovery of Bioactive Natural Product-Inspired Non-Traditional Macrocyclic Peptides</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">3502</span>– <span class="NLM_lpage">3528</span>, <span class="refDoi"> DOI: 10.3390/molecules18033502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.3390%2Fmolecules18033502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=23507778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlt1GitLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=3502-3528&author=K.+Itoauthor=T.+Passiouraauthor=H.+Suga&title=Technologies+for+the+Synthesis+of+MRNA-Encoding+Libraries+and+Discovery+of+Bioactive+Natural+Product-Inspired+Non-Traditional+Macrocyclic+Peptides&doi=10.3390%2Fmolecules18033502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Technologies for the synthesis of mRNA-encoding libraries and discovery of bioactive natural product-inspired non-traditional macrocyclic peptides</span></div><div class="casAuthors">Ito, Kenichiro; Passioura, Toby; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3502-3528</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">In this review, we discuss emerging technologies for drug discovery, which yields novel mol. scaffolds based on natural product-inspired non-traditional peptides expressed using the translation machinery.  Unlike natural products, these technologies allow for constructing mRNA-encoding libraries of macrocyclic peptides contg. non-canonical sidechains and N-methyl-modified backbones.  The complexity of sequence space in such libraries reaches as high as a trillion (>1012), affording initial hits of high affinity ligands against protein targets.  Although this article comprehensively covers several related technologies, we discuss in greater detail the tech. development and advantages of the Random non-std. Peptide Integration Discovery (RaPID) system, including the recent identification of inhibitors against various therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3aPtwP6VYNrVg90H21EOLACvtfcHk0lj0YZuATITpDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlt1GitLw%253D&md5=71b8bd1771f6d01a7fce404ec6de8db5</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.3390%2Fmolecules18033502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules18033502%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DK.%26aulast%3DPassioura%26aufirst%3DT.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DTechnologies%2520for%2520the%2520Synthesis%2520of%2520MRNA-Encoding%2520Libraries%2520and%2520Discovery%2520of%2520Bioactive%2520Natural%2520Product-Inspired%2520Non-Traditional%2520Macrocyclic%2520Peptides%26jtitle%3DMolecules%26date%3D2013%26volume%3D18%26spage%3D3502%26epage%3D3528%26doi%3D10.3390%2Fmolecules18033502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schlippe, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Josephson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szostak, J. W.</span></span> <span> </span><span class="NLM_article-title">In Vitro Selection of Highly Modified Cyclic Peptides That Act as Tight Binding Inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">10469</span>– <span class="NLM_lpage">10477</span>, <span class="refDoi"> DOI: 10.1021/ja301017y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja301017y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A280%3ADC%252BC38vns1CqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=10469-10477&author=Y.+V.+Schlippeauthor=M.+C.+Hartmanauthor=K.+Josephsonauthor=J.+W.+Szostak&title=In+Vitro+Selection+of+Highly+Modified+Cyclic+Peptides+That+Act+as+Tight+Binding+Inhibitors&doi=10.1021%2Fja301017y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro selection of highly modified cyclic peptides that act as tight binding inhibitors</span></div><div class="casAuthors">Schlippe Yollete V Guillen; Hartman Matthew C T; Josephson Kristopher; Szostak Jack W</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">10469-77</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">There is a great demand for the discovery of new therapeutic molecules that combine the high specificity and affinity of biologic drugs with the bioavailability and lower cost of small molecules.  Small, natural-product-like peptides hold great promise in bridging this gap; however, access to libraries of these compounds has been a limitation.  Since ribosomal peptides may be subjected to in vitro selection techniques, the generation of extremely large libraries (>10(13)) of highly modified macrocyclic peptides may provide a powerful alternative for the generation and selection of new useful bioactive molecules.  Moreover, the incorporation of many non-proteinogenic amino acids into ribosomal peptides in conjunction with macrocyclization should enhance the drug-like features of these libraries.  Here we show that mRNA-display, a technique that allows the in vitro selection of peptides, can be applied to the evolution of macrocyclic peptides that contain a majority of unnatural amino acids.  We describe the isolation and characterization of two such unnatural cyclic peptides that bind the protease thrombin with low nanomolar affinity, and we show that the unnatural residues in these peptides are essential for the observed high-affinity binding.  We demonstrate that the selected peptides are tight-binding inhibitors of thrombin, with K(i)(app) values in the low nanomolar range.  The ability to evolve highly modified macrocyclic peptides in the laboratory is the first crucial step toward the facile generation of useful molecular reagents and therapeutic lead molecules that combine the advantageous features of biologics with those of small-molecule drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSohKaxZ0ovut1QT_1O5WBYfW6udTcc2eYDiOAVmDkgIbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38vns1CqsQ%253D%253D&md5=8b83efc2d96fcab98ec42039f63695fb</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fja301017y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja301017y%26sid%3Dliteratum%253Aachs%26aulast%3DSchlippe%26aufirst%3DY.%2BV.%26aulast%3DHartman%26aufirst%3DM.%2BC.%26aulast%3DJosephson%26aufirst%3DK.%26aulast%3DSzostak%26aufirst%3DJ.%2BW.%26atitle%3DIn%2520Vitro%2520Selection%2520of%2520Highly%2520Modified%2520Cyclic%2520Peptides%2520That%2520Act%2520as%2520Tight%2520Binding%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D10469%26epage%3D10477%26doi%3D10.1021%2Fja301017y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joo, S.-H.</span></span> <span> </span><span class="NLM_article-title">Cyclic Peptides as Therapeutic Agents and Biochemical Tools</span>. <i>Biomol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.4062/biomolther.2012.20.1.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.4062%2Fbiomolther.2012.20.1.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC38Xns12hsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=19-26&author=S.-H.+Joo&title=Cyclic+Peptides+as+Therapeutic+Agents+and+Biochemical+Tools&doi=10.4062%2Fbiomolther.2012.20.1.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic peptides as therapeutic agents and biochemical tools</span></div><div class="casAuthors">Joo, Sang Hoon</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecules & Therapeutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-26</span>CODEN:
                <span class="NLM_cas:coden">BTIHA3</span>;
        ISSN:<span class="NLM_cas:issn">1976-9148</span>.
    
            (<span class="NLM_cas:orgname">Korean Society of Applied Pharmacology</span>)
        </div><div class="casAbstract">A review.  There are many cyclic peptides with diverse biol. activities, such as antibacterial activity, immunosuppressive activity, and anti-tumor activity, and so on.  Encouraged by natural cyclic peptides with biol. activity, efforts have been made to develop cyclic peptides with both genetic and synthetic methods.  The genetic methods include phage display, intein-based cyclic peptides, and mRNA display.  The synthetic methods involve individual synthesis, parallel synthesis, as well as split-and-pool synthesis.  Recent development of cyclic peptide library based on split-and-pool synthesis allows on-bead screening, in-soln. screening, and microarray screening of cyclic peptides for biol. activity.  Cyclic peptides will be useful as receptor agonist/antagonist, RNA binding mol., enzyme inhibitor and so on, and more cyclic peptides will emerge as therapeutic agents and biochem. tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP1NpYU4J1frVg90H21EOLACvtfcHk0lj0YZuATITpDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xns12hsLw%253D&md5=02d12ac01606f28c812815cf9ed70f6f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.4062%2Fbiomolther.2012.20.1.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4062%252Fbiomolther.2012.20.1.019%26sid%3Dliteratum%253Aachs%26aulast%3DJoo%26aufirst%3DS.-H.%26atitle%3DCyclic%2520Peptides%2520as%2520Therapeutic%2520Agents%2520and%2520Biochemical%2520Tools%26jtitle%3DBiomol.%2520Ther.%26date%3D2012%26volume%3D20%26spage%3D19%26epage%3D26%26doi%3D10.4062%2Fbiomolther.2012.20.1.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oyelere, A. K.</span></span> <span> </span><span class="NLM_article-title">Macrocycles in Medicinal Chemistry and Drug Discovery</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1359</span>– <span class="NLM_lpage">1360</span>, <span class="refDoi"> DOI: 10.2174/156802610792232097</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.2174%2F156802610792232097" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=20919989" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFajurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=1359-1360&author=A.+K.+Oyelere&title=Macrocycles+in+Medicinal+Chemistry+and+Drug+Discovery&doi=10.2174%2F156802610792232097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in medicinal chemistry and drug discovery</span></div><div class="casAuthors">Oyelere, Adegboyega K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1359-1360</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnTtPNYBDVObVg90H21EOLACvtfcHk0lgvaJ4Q18c6mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFajurrJ&md5=77fe8dc96350b3cbfa5403cc45a9e8fa</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.2174%2F156802610792232097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802610792232097%26sid%3Dliteratum%253Aachs%26aulast%3DOyelere%26aufirst%3DA.%2BK.%26atitle%3DMacrocycles%2520in%2520Medicinal%2520Chemistry%2520and%2520Drug%2520Discovery%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2010%26volume%3D10%26spage%3D1359%26epage%3D1360%26doi%3D10.2174%2F156802610792232097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drahl, C.</span></span> <span> </span><span class="NLM_article-title">Big Hopes Ride on Big Rings</span>. <i>Chem. Eng. News</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">54</span>– <span class="NLM_lpage">57</span>, <span class="refDoi"> DOI: 10.1021/cen-v087n036.p054</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cen-v087n036.p054" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2009&pages=54-57&author=C.+Drahl&title=Big+Hopes+Ride+on+Big+Rings&doi=10.1021%2Fcen-v087n036.p054"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fcen-v087n036.p054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcen-v087n036.p054%26sid%3Dliteratum%253Aachs%26aulast%3DDrahl%26aufirst%3DC.%26atitle%3DBig%2520Hopes%2520Ride%2520on%2520Big%2520Rings%26jtitle%3DChem.%2520Eng.%2520News%26date%3D2009%26volume%3D87%26spage%3D54%26epage%3D57%26doi%3D10.1021%2Fcen-v087n036.p054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zorzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deyle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Cyclic Peptide Therapeutics: Past, Present and Future</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">29</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1016%2Fj.cbpa.2017.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=28249193" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsFGksLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=24-29&author=A.+Zorziauthor=K.+Deyleauthor=C.+Heinis&title=Cyclic+Peptide+Therapeutics%3A+Past%2C+Present+and+Future&doi=10.1016%2Fj.cbpa.2017.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic peptide therapeutics: past, present and future</span></div><div class="casAuthors">Zorzi, Alessandro; Deyle, Kaycie; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">24-29</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cyclic peptides combine several favorable properties such as good binding affinity, target selectivity and low toxicity that make them an attractive modality for the development of therapeutics.  Over 40 cyclic peptide drugs are currently in clin. use and around one new cyclic peptide drug enters the market every year on av.  The vast majority of clin. approved cyclic peptides are derived from natural products, such as antimicrobials or human peptide hormones.  New powerful techniques based on rational design and in vitro evolution have enabled the de novo development of cyclic peptide ligands to targets for which nature does not offer solns.  A look at the cyclic peptides currently under clin. evaluation shows that several have been developed using such techniques.  This new source for cyclic peptide ligands introduces a freshness to the field, and it is likely that de novo developed cyclic peptides will be in clin. use in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwchcMOj1bSbVg90H21EOLACvtfcHk0lgvaJ4Q18c6mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsFGksLg%253D&md5=4b9581365b92b649dad8c7b22c5bd9c3</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DZorzi%26aufirst%3DA.%26aulast%3DDeyle%26aufirst%3DK.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DCyclic%2520Peptide%2520Therapeutics%253A%2520Past%252C%2520Present%2520and%2520Future%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D24%26epage%3D29%26doi%3D10.1016%2Fj.cbpa.2017.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwochert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pye, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naylor, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez
Valcarcel, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawada, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanco, M.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Stereochemistry Balances Cell Permeability and Solubility in the Naturally Derived Phepropeptin Cyclic Peptides</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">757</span>– <span class="NLM_lpage">761</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00100</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00100" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC28XpsVCltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=757-761&author=J.+Schwochertauthor=Y.+Laoauthor=C.+R.+Pyeauthor=M.+R.+Naylorauthor=P.+V.+Desaiauthor=I.+C.+Gonzalez%0AValcarcelauthor=J.+A.+Barrettauthor=G.+Sawadaauthor=M.-J.+Blancoauthor=R.+S.+Lokey&title=Stereochemistry+Balances+Cell+Permeability+and+Solubility+in+the+Naturally+Derived+Phepropeptin+Cyclic+Peptides&doi=10.1021%2Facsmedchemlett.6b00100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Stereochemistry Balances Cell Permeability and Solubility in the Naturally Derived Phepropeptin Cyclic Peptides</span></div><div class="casAuthors">Schwochert, Joshua; Lao, Yongtong; Pye, Cameron R.; Naylor, Matthew R.; Desai, Prashant V.; Gonzalez-Valcarcel, Isabel C.; Barrett, Jaclyn A.; Sawada, Geri; Blanco, Maria-Jesus; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">757-761</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclic peptide (CP) natural products provide useful model systems for mapping "beyond-Rule-of-5" (bRo5) space.  We identified the phepropeptins as natural product CPs with potential cell permeability.  Synthesis of the phepropeptins and epimeric analogs revealed much more rapid cellular permeability for the natural stereochem. pattern.  Despite being more cell permeable, the natural compds. exhibited similar aq. soly. as the corresponding epimers, a phenomenon explained by solvent-dependent conformational flexibility among the natural compds.  When analyzing the polarity of the soln. structures we found that neither the no. of hydrogen bonds nor the total polar surface area accurately represents the solvation energies of the high and low dielec. conformations.  This work adds to a growing no. of natural CPs whose solvent-dependent conformational behavior allows for a balance between aq. soly. and cell permeability, highlighting structural flexibility as an important consideration in the design of mols. in bRo5 chem. space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwMDd9pbrhsbVg90H21EOLACvtfcHk0lgvaJ4Q18c6mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpsVCltLg%253D&md5=305053943e9f640caad43423e5b3b3ff</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00100%26sid%3Dliteratum%253Aachs%26aulast%3DSchwochert%26aufirst%3DJ.%26aulast%3DLao%26aufirst%3DY.%26aulast%3DPye%26aufirst%3DC.%2BR.%26aulast%3DNaylor%26aufirst%3DM.%2BR.%26aulast%3DDesai%26aufirst%3DP.%2BV.%26aulast%3DGonzalez%2BValcarcel%26aufirst%3DI.%2BC.%26aulast%3DBarrett%26aufirst%3DJ.%2BA.%26aulast%3DSawada%26aufirst%3DG.%26aulast%3DBlanco%26aufirst%3DM.-J.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DStereochemistry%2520Balances%2520Cell%2520Permeability%2520and%2520Solubility%2520in%2520the%2520Naturally%2520Derived%2520Phepropeptin%2520Cyclic%2520Peptides%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D757%26epage%3D761%26doi%3D10.1021%2Facsmedchemlett.6b00100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwochert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkeley, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponkey, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, S. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khunte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Limberakis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalgutkar, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathiowetz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Peptide to Peptoid Substitutions Increase Cell Permeability in Cyclic Hexapeptides</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2928</span>– <span class="NLM_lpage">2931</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.5b01162</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.5b01162" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2MXps1arsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=2928-2931&author=J.+Schwochertauthor=R.+Turnerauthor=M.+Thangauthor=R.+F.+Berkeleyauthor=A.+R.+Ponkeyauthor=K.+M.+Rodriguezauthor=S.+S.+F.+Leungauthor=B.+Khunteauthor=G.+Goetzauthor=C.+Limberakisauthor=A.+S.+Kalgutkarauthor=H.+Engauthor=M.+J.+Shapiroauthor=A.+M.+Mathiowetzauthor=D.+A.+Priceauthor=S.+Lirasauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Peptide+to+Peptoid+Substitutions+Increase+Cell+Permeability+in+Cyclic+Hexapeptides&doi=10.1021%2Facs.orglett.5b01162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide to Peptoid Substitutions Increase Cell Permeability in Cyclic Hexapeptides</span></div><div class="casAuthors">Schwochert, Joshua; Turner, Rushia; Thang, Melissa; Berkeley, Ray F.; Ponkey, Alexandra R.; Rodriguez, Kelsie M.; Leung, Siegfried S. F.; Khunte, Bhagyashree; Goetz, Gilles; Limberakis, Chris; Kalgutkar, Amit S.; Eng, Heather; Shapiro, Michael J.; Mathiowetz, Alan M.; Price, David A.; Liras, Spiros; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2928-2931</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The effect of peptide-to-peptoid substitutions on the passive membrane permeability of an N-methylated cyclic hexapeptide is examd.  In general, substitutions maintained permeability but increased conformational heterogeneity.  Diversification with nonproteinogenic side chains increased permeability up to 3-fold.  Addnl., the conformational impact of peptoid substitutions within a β-turn are explored.  Based on these results, the strategic incorporation of peptoid residues into cyclic peptides can maintain or improve cell permeability, while increasing access to diverse side-chain functionality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofUG1_hYdVn7Vg90H21EOLACvtfcHk0ljlNE4WCdJX1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXps1arsrs%253D&md5=156985175ddcafd34a67e81713934040</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.5b01162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.5b01162%26sid%3Dliteratum%253Aachs%26aulast%3DSchwochert%26aufirst%3DJ.%26aulast%3DTurner%26aufirst%3DR.%26aulast%3DThang%26aufirst%3DM.%26aulast%3DBerkeley%26aufirst%3DR.%2BF.%26aulast%3DPonkey%26aufirst%3DA.%2BR.%26aulast%3DRodriguez%26aufirst%3DK.%2BM.%26aulast%3DLeung%26aufirst%3DS.%2BS.%2BF.%26aulast%3DKhunte%26aufirst%3DB.%26aulast%3DGoetz%26aufirst%3DG.%26aulast%3DLimberakis%26aufirst%3DC.%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DEng%26aufirst%3DH.%26aulast%3DShapiro%26aufirst%3DM.%2BJ.%26aulast%3DMathiowetz%26aufirst%3DA.%2BM.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DPeptide%2520to%2520Peptoid%2520Substitutions%2520Increase%2520Cell%2520Permeability%2520in%2520Cyclic%2520Hexapeptides%26jtitle%3DOrg.%2520Lett.%26date%3D2015%26volume%3D17%26spage%3D2928%26epage%3D2931%26doi%3D10.1021%2Facs.orglett.5b01162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, C.</span></span> <span> </span><span class="NLM_article-title">Constrained Peptides’ Time to Shine?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">533</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1038%2Fnrd.2018.125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=30057410" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVSht7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=531-533&author=C.+Morrison&title=Constrained+Peptides%E2%80%99+Time+to+Shine%3F&doi=10.1038%2Fnrd.2018.125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Constrained peptides' time to shine?</span></div><div class="casAuthors">Morrison, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">531-533</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Constrained peptides have long tantalized drug developers with their potential ability to combine the best attributes of antibodies and small mols.  Finally, a handful of constrained peptides are in late-stage clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ9AS6knVUNrVg90H21EOLACvtfcHk0ljlNE4WCdJX1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVSht7fP&md5=35e88bd4a715f26738f600598bc75ab2</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.125%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DC.%26atitle%3DConstrained%2520Peptides%25E2%2580%2599%2520Time%2520to%2520Shine%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D531%26epage%3D533%26doi%3D10.1038%2Fnrd.2018.125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warr, W. A.</span></span> <span> </span><span class="NLM_article-title">A CADD-Alog of Strategies in Pharma</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">247</span>, <span class="refDoi"> DOI: 10.1007/s10822-017-0017-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1007%2Fs10822-017-0017-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=28315993" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2sXks12msbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=245-247&author=W.+A.+Warr&title=A+CADD-Alog+of+Strategies+in+Pharma&doi=10.1007%2Fs10822-017-0017-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">A CADD-alog of strategies in pharma</span></div><div class="casAuthors">Warr, Wendy A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">245-247</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A special issue on computer-aided drug design (CADD) strategies in pharma discusses how CADD groups in different environments work.  Perspectives were collected from authors in 11 organizations: four big pharmaceutical companies, one major biotechnol. company, one smaller biotech, one private pharmaceutical company, two contract research organizations (CROs), one university, and one that spans the breadth of big pharmaceutical companies and one smaller biotech.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm15mndFA_ybVg90H21EOLACvtfcHk0ljlNE4WCdJX1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXks12msbY%253D&md5=3e12bbefdad84473fc013467a32ab20a</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1007%2Fs10822-017-0017-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-017-0017-6%26sid%3Dliteratum%253Aachs%26aulast%3DWarr%26aufirst%3DW.%2BA.%26atitle%3DA%2520CADD-Alog%2520of%2520Strategies%2520in%2520Pharma%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2017%26volume%3D31%26spage%3D245%26epage%3D247%26doi%3D10.1007%2Fs10822-017-0017-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, K. L.</span></span> <span> </span><span class="NLM_article-title">Large Cyclodextrins</span>. <i>J. Inclusion Phenom. Macrocyclic Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1023/a:1020494503684</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1023%2FA%3A1020494503684" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsVKku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2002&pages=1-13&author=K.+L.+Larsen&title=Large+Cyclodextrins&doi=10.1023%2Fa%3A1020494503684"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Large Cyclodextrins</span></div><div class="casAuthors">Larsen, Kim Lambertsen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Inclusion Phenomena and Macrocyclic Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">JIPCF5</span>;
        ISSN:<span class="NLM_cas:issn">1388-3127</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A review with refs.  The existence of large cyclodextrins, cyclic α-D-(1→4) glucans with a degree of polymn. higher than eight, has been proven during the past decade.  A no. of 4-α-glucanotransferases have been shown to be able to produce large cyclodextrins consisting of up to several hundred glycosyl units, from both amylose and amylopectin.  Large cyclodextrins with degree of polymn. up to 31 have been isolated to purity by use of elaborate purifn. schemes, enabling studies of their structural and complex forming properties.  The solid state structures of the large cyclodextrins with a degree of polymn. 10, 14 and 26, resp., have revealed interesting new structural features of this family of mols.  This review summarizes the studies of the large cyclodextrins, a varied and highly interesting group of mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3FwWddKMD37Vg90H21EOLACvtfcHk0lgL_rQUcnUvoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsVKku7k%253D&md5=042d7663281078400947579627e0fc1c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1023%2FA%3A1020494503684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1020494503684%26sid%3Dliteratum%253Aachs%26aulast%3DLarsen%26aufirst%3DK.%2BL.%26atitle%3DLarge%2520Cyclodextrins%26jtitle%3DJ.%2520Inclusion%2520Phenom.%2520Macrocyclic%2520Chem.%26date%3D2002%26volume%3D43%26spage%3D1%26epage%3D13%26doi%3D10.1023%2Fa%3A1020494503684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raithby, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shields, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, F. H.</span></span> <span> </span><span class="NLM_article-title">Conformational Analysis of Macrocyclic Ether Ligands. II. 1,4,7,10,13-Pentaoxacyclopentadecane and 1,4,7,10,13-Pentathiacyclopentadecane</span>. <i>Acta Crystallogr., Sect. B: Struct. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">476</span>– <span class="NLM_lpage">489</span>, <span class="refDoi"> DOI: 10.1107/s0108768196015303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1107%2FS0108768196015303" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADyaK2sXktFWitbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=476-489&author=P.+R.+Raithbyauthor=G.+P.+Shieldsauthor=F.+H.+Allen&title=Conformational+Analysis+of+Macrocyclic+Ether+Ligands.+II.+1%2C4%2C7%2C10%2C13-Pentaoxacyclopentadecane+and+1%2C4%2C7%2C10%2C13-Pentathiacyclopentadecane&doi=10.1107%2Fs0108768196015303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational analysis of macrocyclic ether ligands. II. 1,4,7,10,13-Pentaoxacyclopentadecane and 1,4,7,10,13-pentathiacyclopentadecane</span></div><div class="casAuthors">Raithby, P. R.; Shields, G. P.; Allen, F. H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section B: Structural Science</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">B53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">476-489</span>CODEN:
                <span class="NLM_cas:coden">ASBSDK</span>;
        ISSN:<span class="NLM_cas:issn">0108-7681</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">Crystallog. results retrieved from the Cambridge Structural Database (CSD) were used to perform a systematic conformational classification of free and metal-coordinated unsatd. 15-membered oxa and thia macrocycles using symmetry-modified Jarvis-Patrick cluster anal.  Relative mol. mechanics energies of the obsd. conformations are compared with the cluster populations.  With oxa donors a uniangular and a [348] conformer predominate for larger metal ions; these lie above the donor atom plane with 1-6 addnl. ligands bound on the same side.  With smaller cations an anangular conformer is adopted, the O atoms describing the equatorial plane of a pentagonal bipyramid.  Other conformers occur as dictated by the coordination environment, particularly if not all donor atoms are metal-bound; in some cases the conformation is detd. by a H-bonded network.  In some thia examples the ligand binds to an axial/apical and four equatorial sites of the coordination polyhedron; in others contg. AuI or AgI the metal is linearly or tetrahedrally coordinated with addnl. M-S interactions.  With mixed donors, the hard/soft characteristics of the metal det. the coordination mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHw7WxdXKu4bVg90H21EOLACvtfcHk0lgL_rQUcnUvoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktFWitbg%253D&md5=dc8a6059021d4613c7f50c61fd5e53ef</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0108768196015303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0108768196015303%26sid%3Dliteratum%253Aachs%26aulast%3DRaithby%26aufirst%3DP.%2BR.%26aulast%3DShields%26aufirst%3DG.%2BP.%26aulast%3DAllen%26aufirst%3DF.%2BH.%26atitle%3DConformational%2520Analysis%2520of%2520Macrocyclic%2520Ether%2520Ligands.%2520II.%25201%252C4%252C7%252C10%252C13-Pentaoxacyclopentadecane%2520and%25201%252C4%252C7%252C10%252C13-Pentathiacyclopentadecane%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520B%253A%2520Struct.%2520Sci.%26date%3D1997%26volume%3D53%26spage%3D476%26epage%3D489%26doi%3D10.1107%2Fs0108768196015303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonnet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agrafiotis, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E.</span></span> <span> </span><span class="NLM_article-title">Conformational Analysis of Macrocycles: Finding What Common Search Methods Miss</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2242</span>– <span class="NLM_lpage">2259</span>, <span class="refDoi"> DOI: 10.1021/ci900238a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900238a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1ahtbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=2242-2259&author=P.+Bonnetauthor=D.+K.+Agrafiotisauthor=F.+Zhuauthor=E.+Martin&title=Conformational+Analysis+of+Macrocycles%3A+Finding+What+Common+Search+Methods+Miss&doi=10.1021%2Fci900238a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Analysis of Macrocycles: Finding What Common Search Methods Miss</span></div><div class="casAuthors">Bonnet, Pascal; Agrafiotis, Dimitris K.; Zhu, Fangqiang; Martin, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2242-2259</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As computational drug design becomes increasingly reliant on virtual screening and on high-throughput 3D modeling, the need for fast, robust, and reliable methods for sampling mol. conformations has become greater than ever.  Furthermore, chem. novelty is at a premium, forcing medicinal chemists to explore more complex structural motifs and unusual topologies.  This necessitates the use of conformational sampling techniques that work well in all cases.  Here, we compare the performance of several popular conformational search algorithms on three broad classes of macrocyclic mols.  These methods include Catalyst, CAESAR, MacroModel, MOE, Omega, Rubicon and two newer self-organizing algorithms known as stochastic proximity embedding (SPE) and self-organizing superimposition (SOS) that have been developed at Johnson & Johnson.  Our results show a compelling advantage for the three distance geometry methods (SOS, SPE, and Rubicon) followed to a lesser extent by MacroModel.  The remaining techniques, particularly those based on systematic search, often failed to identify any of the lowest energy conformations and are unsuitable for this class of structures.  Taken together with our previous study on drug-like mols., these results suggest that SPE and SOS are two of the most robust and universally applicable conformational search methods, with the latter being preferred because of its superior speed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhIMu1b7U477Vg90H21EOLACvtfcHk0lgL_rQUcnUvoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1ahtbnF&md5=9b4a5919415ef0660c4f9ff18d9b76f4</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fci900238a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900238a%26sid%3Dliteratum%253Aachs%26aulast%3DBonnet%26aufirst%3DP.%26aulast%3DAgrafiotis%26aufirst%3DD.%2BK.%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DMartin%26aufirst%3DE.%26atitle%3DConformational%2520Analysis%2520of%2520Macrocycles%253A%2520Finding%2520What%2520Common%2520Search%2520Methods%2520Miss%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D2242%26epage%3D2259%26doi%3D10.1021%2Fci900238a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reetz, M. T.</span></span> <span> </span><span class="NLM_article-title">Utility of B-Factors in Protein Science: Interpreting Rigidity, Flexibility, and Internal Motion and Engineering Thermostability</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">1626</span>– <span class="NLM_lpage">1665</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.8b00290</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00290" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVSit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2019&pages=1626-1665&author=Z.+Sunauthor=Q.+Liuauthor=G.+Quauthor=Y.+Fengauthor=M.+T.+Reetz&title=Utility+of+B-Factors+in+Protein+Science%3A+Interpreting+Rigidity%2C+Flexibility%2C+and+Internal+Motion+and+Engineering+Thermostability&doi=10.1021%2Facs.chemrev.8b00290"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Utility of B-Factors in Protein Science: Interpreting Rigidity, Flexibility, and Internal Motion and Engineering Thermostability</span></div><div class="casAuthors">Sun, Zhoutong; Liu, Qian; Qu, Ge; Feng, Yan; Reetz, Manfred T.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1626-1665</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The term B-factor, sometimes called the Debye-Waller factor, temp. factor, or at. displacement parameter, is used in protein crystallog. to describe the attenuation of X-ray or neutron scattering caused by thermal motion.  This review begins with analyses of early protein studies which suggested that B-factors, available from the Protein Data Bank, can be used to identify the flexibility of atoms, side chains, or even whole regions.  This requires a technique for obtaining normalized B-factors.  Since then the exploitation of B-factors has been extensively elaborated and applied in a variety of studies with quite different goals, all having in common the identification and interpretation of rigidity, flexibility, and/or internal motion which are crucial in enzymes and in proteins in general.  Importantly, this review includes a discussion of limitations and possible pitfalls when using B-factors.  A second research area, which likewise exploits B-factors, is also reviewed, namely, the development of the so-called B-FIT-directed evolution method for increasing the thermostability of enzymes as catalysts in org. chem. and biotechnol.  In both research areas, a max. of structural and mechanistic insights is gained when B-factor analyses are combined with other exptl. and computational techniques.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB4_fJMa4APbVg90H21EOLACvtfcHk0lgL_rQUcnUvoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVSit78%253D&md5=9bcf5fb15301278e8a13f250d38cad54</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00290%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DQu%26aufirst%3DG.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DReetz%26aufirst%3DM.%2BT.%26atitle%3DUtility%2520of%2520B-Factors%2520in%2520Protein%2520Science%253A%2520Interpreting%2520Rigidity%252C%2520Flexibility%252C%2520and%2520Internal%2520Motion%2520and%2520Engineering%2520Thermostability%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26spage%3D1626%26epage%3D1665%26doi%3D10.1021%2Facs.chemrev.8b00290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otero-Ramirez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passioura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Structural Features and Binding Modes of Thioether-Cyclized Peptide Ligands</span>. <i>Biomedicines</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">116</span>, <span class="refDoi"> DOI: 10.3390/biomedicines6040116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.3390%2Fbiomedicines6040116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1WjsbrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=116&author=M.+Otero-Ramirezauthor=T.+Passiouraauthor=H.+Suga&title=Structural+Features+and+Binding+Modes+of+Thioether-Cyclized+Peptide+Ligands&doi=10.3390%2Fbiomedicines6040116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Structural features and binding modes of thioether-cyclized peptide ligands</span></div><div class="casAuthors">Otero-Ramirez, Manuel E.; Passioura, Toby; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Biomedicines</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">116</span>CODEN:
                <span class="NLM_cas:coden">BIOMID</span>;
        ISSN:<span class="NLM_cas:issn">2227-9059</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Macrocyclic peptides are an emerging class of bioactive compds. for therapeutic use.  In part, this is because they are capable of high potency and excellent target affinity and selectivity.  Over the last decade, several biochem. techniques have been developed for the identification of bioactive macrocyclic peptides, allowing for the rapid isolation of high affinity ligands to a target of interest.  A common feature of these techniques is a general reliance on thioether formation to effect macrocyclization.  Increasingly, the compds. identified using these approaches have been subjected to x-ray crystallog. anal. bound to their resp. targets, providing detailed structural information about their conformation and mechanism of target binding.  The present review provides an overview of the target bound thioether-closed macrocyclic peptide structures that have been obtained to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-pfa5vlPS97Vg90H21EOLACvtfcHk0lhOfcXPsDQidA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1WjsbrF&md5=447cd5c4ca6d10ba1fa2cc423b2b2184</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.3390%2Fbiomedicines6040116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiomedicines6040116%26sid%3Dliteratum%253Aachs%26aulast%3DOtero-Ramirez%26aufirst%3DM.%26aulast%3DPassioura%26aufirst%3DT.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DStructural%2520Features%2520and%2520Binding%2520Modes%2520of%2520Thioether-Cyclized%2520Peptide%2520Ligands%26jtitle%3DBiomedicines%26date%3D2018%26volume%3D6%26spage%3D116%26doi%3D10.3390%2Fbiomedicines6040116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malde, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Crystal Structures of Protein-Bound Cyclic Peptides</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">9861</span>– <span class="NLM_lpage">9914</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.8b00807</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00807" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos1Wltrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2019&pages=9861-9914&author=A.+K.+Maldeauthor=T.+A.+Hillauthor=A.+Iyerauthor=D.+P.+Fairlie&title=Crystal+Structures+of+Protein-Bound+Cyclic+Peptides&doi=10.1021%2Facs.chemrev.8b00807"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of protein-bound cyclic peptides</span></div><div class="casAuthors">Malde, Alpeshkumar K.; Hill, Timothy A.; Iyer, Abishek; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9861-9914</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Cyclization is an important post-translational modification of peptides and proteins that confers key advantages such as protection from proteolytic degrdn., altered soly., membrane permeability, bioavailability, and esp. restricted conformational freedom in water that allows the peptide backbone to adopt the major secondary structure elements found in proteins.  Non-ribosomal synthesis in bacteria, fungi, and plants or synthetic chem. can introduce unnatural amino acids and non-peptidic constraints that modify peptide backbones and side chains to fine-tune cyclic peptide structure.  Structures can be potentially altered further upon binding to a protein in biol. environments.  Here we analyze three-dimensional crystal structures for 211 bioactive cyclic peptides bound to 65 different proteins.  The protein-bound cyclic peptides were examd. for similarities and differences in bonding modes, for main-chain and side-chain structure, and for the importance of polarity, hydrogen bonds, hydrophobic effects, and water mols. in interactions with proteins.  Many protein-bound cyclic peptides show backbone structures like those (strands, sheets, turns, helixes, loops, or distorted variations) found at protein-protein binding interfaces.  However, the notion of macrocycles simply as privileged scaffolds that primarily project side-chain substituents for complementary interactions with proteins is dispelled here.  Unlike small-mol. drugs, the cyclic peptides do not rely mainly upon hydrophobic and van der Waals interactions for protein binding; they also use their main chain and side chains to form polar contacts and hydrogen bonds with proteins.  Compared to small-mol. ligands, cyclic peptides can bind across larger, polar, and water-exposed protein surface areas, making many more contacts that can increase affinity, selectivity, biol. activity, and ligand-receptor residence time.  Cyclic peptides have a greater capacity than small-mol. drugs to modulate protein-protein interfaces that involve large, shallow, dynamic, polar, and water-exposed protein surfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnAtNoQP4lbbVg90H21EOLACvtfcHk0lhOfcXPsDQidA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos1Wltrg%253D&md5=38614ab1898f056757a3671554505a24</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00807&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00807%26sid%3Dliteratum%253Aachs%26aulast%3DMalde%26aufirst%3DA.%2BK.%26aulast%3DHill%26aufirst%3DT.%2BA.%26aulast%3DIyer%26aufirst%3DA.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DCrystal%2520Structures%2520of%2520Protein-Bound%2520Cyclic%2520Peptides%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26spage%3D9861%26epage%3D9914%26doi%3D10.1021%2Facs.chemrev.8b00807" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guéret, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adihou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalakrishnan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemurell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span></span> <span> </span><span class="NLM_article-title">New Modalities for Challenging Targets in Drug Discovery</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">10294</span>– <span class="NLM_lpage">10323</span>, <span class="refDoi"> DOI: 10.1002/anie.201611914</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fanie.201611914" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1WlsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=10294-10323&author=E.+Valeurauthor=S.+M.+Gu%C3%A9retauthor=H.+Adihouauthor=R.+Gopalakrishnanauthor=M.+Lemurellauthor=H.+Waldmannauthor=T.+N.+Grossmannauthor=A.+T.+Plowright&title=New+Modalities+for+Challenging+Targets+in+Drug+Discovery&doi=10.1002%2Fanie.201611914"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">New Modalities for Challenging Targets in Drug Discovery</span></div><div class="casAuthors">Valeur, Eric; Gueret, Stephanie M.; Adihou, Helene; Gopalakrishnan, Ranganath; Lemurell, Malin; Waldmann, Herbert; Grossmann, Tom N.; Plowright, Alleyn T.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">10294-10323</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The ever-increasing understanding of biol. systems is providing a range of exciting novel biol. targets, whose modulation may enable novel therapeutic options for many diseases.  These targets include protein-protein and protein-nucleic acid interactions, which are, however, often refractory to classical small-mol. approaches.  Other types of mols., or modalities, are therefore required to address these targets, which has led several academic research groups and pharmaceutical companies to increasingly use the concept of so-called "new modalities".  This review defines for the first time the scope of this term, which includes novel peptidic scaffolds, oligonucleotides, hybrids, mol. conjugates, as well as new uses of classical small mols.  The authors provide the most representative examples of these modalities to target large binding surface areas such as those found in protein-protein interactions and for biol. processes at the center of cell regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3BpAx5oKYa7Vg90H21EOLACvtfcHk0lhOfcXPsDQidA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1WlsLvP&md5=e09ef5bb8dbbf09c05b1a2340aa431ec</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611914&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611914%26sid%3Dliteratum%253Aachs%26aulast%3DValeur%26aufirst%3DE.%26aulast%3DGu%25C3%25A9ret%26aufirst%3DS.%2BM.%26aulast%3DAdihou%26aufirst%3DH.%26aulast%3DGopalakrishnan%26aufirst%3DR.%26aulast%3DLemurell%26aufirst%3DM.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26atitle%3DNew%2520Modalities%2520for%2520Challenging%2520Targets%2520in%2520Drug%2520Discovery%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D10294%26epage%3D10323%26doi%3D10.1002%2Fanie.201611914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zivanovic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colizzi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hospital, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soliva, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orozco, M.</span></span> <span> </span><span class="NLM_article-title">Exploring the Conformational Landscape of Bioactive Small Molecules</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">6575</span>– <span class="NLM_lpage">6585</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.0c00304</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.0c00304" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFOis7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=6575-6585&author=S.+Zivanovicauthor=F.+Colizziauthor=D.+Morenoauthor=A.+Hospitalauthor=R.+Solivaauthor=M.+Orozco&title=Exploring+the+Conformational+Landscape+of+Bioactive+Small+Molecules&doi=10.1021%2Facs.jctc.0c00304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring the Conformational Landscape of Bioactive Small Molecules</span></div><div class="casAuthors">Zivanovic, Sanja; Colizzi, Francesco; Moreno, David; Hospital, Adam; Soliva, Robert; Orozco, Modesto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">6575-6585</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">By using a combination of classical Hamiltonian replica exchange with high-level quantum mech. calcns. on more than one hundred drug-like mols., we explored here the energy cost assocd. with binding of drug-like mols. to target macromols.  We found that, in general, the drug-like mols. present bound to proteins in the Protein Data Bank (PDB) can access easily the bioactive conformation and in fact for 73% of the studied mols. the "bioactive" conformation is within 3kBT from the most-stable conformation in soln. as detd. by DFT/SCRF calcns.  Cases with large differences between the most-stable and the bioactive conformations appear in ligands recognized by ionic contacts, or very large structures establishing many favorable interactions with the protein.  There are also a few cases where we obsd. a non-negligible uncertainty related to the exptl. structure deposited in PDB.  Remarkably, the rough automatic force field used here provides reasonable ests. of the conformational ensemble of drugs in soln.  The outlined protocol can be used to better est. the cost of adopting the bioactive conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXCdIaj3lGALVg90H21EOLACvtfcHk0lgi3L-wJCyyZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFOis7jE&md5=73de708c7dce6920169e9f5c6547272d</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.0c00304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.0c00304%26sid%3Dliteratum%253Aachs%26aulast%3DZivanovic%26aufirst%3DS.%26aulast%3DColizzi%26aufirst%3DF.%26aulast%3DMoreno%26aufirst%3DD.%26aulast%3DHospital%26aufirst%3DA.%26aulast%3DSoliva%26aufirst%3DR.%26aulast%3DOrozco%26aufirst%3DM.%26atitle%3DExploring%2520the%2520Conformational%2520Landscape%2520of%2520Bioactive%2520Small%2520Molecules%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2020%26volume%3D16%26spage%3D6575%26epage%3D6585%26doi%3D10.1021%2Facs.jctc.0c00304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleves, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherer, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reibarkh, M. Y.</span></span> <span> </span><span class="NLM_article-title">Complex Macrocycle Exploration: Parallel, Heuristic, and Constraint-Based Conformer Generation Using ForceGen</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.1007/s10822-019-00203-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1007%2Fs10822-019-00203-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=31054028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovVOkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=531-558&author=A.+N.+Jainauthor=A.+E.+Clevesauthor=Q.+Gaoauthor=X.+Wangauthor=Y.+Liuauthor=E.+C.+Shererauthor=M.+Y.+Reibarkh&title=Complex+Macrocycle+Exploration%3A+Parallel%2C+Heuristic%2C+and+Constraint-Based+Conformer+Generation+Using+ForceGen&doi=10.1007%2Fs10822-019-00203-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Complex macrocycle exploration: parallel, heuristic, and constraint-based conformer generation using ForceGen</span></div><div class="casAuthors">Jain, Ajay N.; Cleves, Ann E.; Gao, Qi; Wang, Xiao; Liu, Yizhou; Sherer, Edward C.; Reibarkh, Mikhail Y.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">531-558</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">ForceGen is a template-free, non-stochastic approach for 2D to 3D structure generation and conformational elaboration for small mols., including both non-macrocycles and macrocycles.  For conformational search of non-macrocycles, ForceGen is both faster and more accurate than the best of all tested methods on a very large, independently curated benchmark of 2859 PDB ligands.  In this study, the primary results are on macrocycles, including results for 431 unique examples from four sep. benchmarks.  These include complex peptide and peptide-like cases that can form networks of internal hydrogen bonds.  By making use of new phys. movements ("flips" of near-linear sub-cycles and explicit formation of hydrogen bonds), ForceGen exhibited statistically significantly better performance for overall RMS deviation from exptl. coordinates than all other approaches.  The algorithmic approach offers natural parallelization across multiple computing-cores.  On a modest multi-core workstation, for all but the most complex macrocycles, median wall-clock times were generally under a minute in fast search mode and under 2 min using thorough search.  On the most complex cases (roughly cyclic decapeptides and larger) explicit exploration of likely hydrogen bonding networks yielded marked improvements, but with calcn. times increasing to several minutes and in some cases to roughly an hour for fast search.  In complex cases, utilization of NMR data to constrain conformational search produces accurate conformational ensembles representative of soln. state macrocycle behavior.  On macrocycles of typical complexity (up to 21 rotatable macrocyclic and exocyclic bonds), design-focused macrocycle optimization can be practically supported by computational chem. at interactive time-scales, with conformational ensemble accuracy equaling what is seen with non-macrocyclic ligands.  For more complex macrocycles, inclusion of sparse biophys. data is a helpful adjunct to computation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIA5XjyJWMwLVg90H21EOLACvtfcHk0lgi3L-wJCyyZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovVOkurg%253D&md5=8fb9b7aa12c3dbbca57ebc4265a4a204</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1007%2Fs10822-019-00203-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-019-00203-1%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26aulast%3DCleves%26aufirst%3DA.%2BE.%26aulast%3DGao%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DSherer%26aufirst%3DE.%2BC.%26aulast%3DReibarkh%26aufirst%3DM.%2BY.%26atitle%3DComplex%2520Macrocycle%2520Exploration%253A%2520Parallel%252C%2520Heuristic%252C%2520and%2520Constraint-Based%2520Conformer%2520Generation%2520Using%2520ForceGen%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2019%26volume%3D33%26spage%3D531%26epage%3D558%26doi%3D10.1007%2Fs10822-019-00203-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleves, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. N.</span></span> <span> </span><span class="NLM_article-title">ForceGen 3D Structure and Conformer Generation: From Small Lead-like Molecules to Macrocyclic Drugs</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">419</span>– <span class="NLM_lpage">439</span>, <span class="refDoi"> DOI: 10.1007/s10822-017-0015-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1007%2Fs10822-017-0015-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=28289981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFymtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=419-439&author=A.+E.+Clevesauthor=A.+N.+Jain&title=ForceGen+3D+Structure+and+Conformer+Generation%3A+From+Small+Lead-like+Molecules+to+Macrocyclic+Drugs&doi=10.1007%2Fs10822-017-0015-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">ForceGen 3D structure and conformer generation: from small lead-like molecules to macrocyclic drugs</span></div><div class="casAuthors">Cleves, Ann E.; Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">419-439</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">We introduce the ForceGen method for 3D structure generation and conformer elaboration of drug-like small mols.  ForceGen is novel, avoiding use of distance geometry, mol. templates, or simulation-oriented stochastic sampling.  The method is primarily driven by the mol. force field, implemented using an extension of MMFF94s and a partial charge estimator based on electronegativity-equalization.  The force field is coupled to algorithms for direct sampling of realistic phys. movements made by small mols.  Results are presented on a std. benchmark from the Cambridge Crystallog. Database of 480 drug-like small mols., including full structure generation from SMILES strings.  Reprodn. of protein-bound crystallog. ligand poses is demonstrated on four carefully curated data sets: the ConfGen Set (667 ligands), the PINC cross-docking benchmark (1062 ligands), a large set of macrocyclic ligands (182 total with typical ring sizes of 12-23 atoms), and a commonly used benchmark for evaluating macrocycle conformer generation (30 ligands total).  Results compare favorably to alternative methods, and performance on macrocyclic compds. approaches that obsd. on non-macrocycles while yielding a roughly 100-fold speed improvement over alternative MD-based methods with comparable performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFpG4EEfPBG7Vg90H21EOLACvtfcHk0lgi3L-wJCyyZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFymtLg%253D&md5=917729bd50c89ef5bfd73d347c8ebf53</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1007%2Fs10822-017-0015-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-017-0015-8%26sid%3Dliteratum%253Aachs%26aulast%3DCleves%26aufirst%3DA.%2BE.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DForceGen%25203D%2520Structure%2520and%2520Conformer%2520Generation%253A%2520From%2520Small%2520Lead-like%2520Molecules%2520to%2520Macrocyclic%2520Drugs%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2017%26volume%3D31%26spage%3D419%26epage%3D439%26doi%3D10.1007%2Fs10822-017-0015-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perola, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charifson, P. S.</span></span> <span> </span><span class="NLM_article-title">Conformational Analysis of Drug-Like Molecules Bound to Proteins: An Extensive Study of Ligand Reorganization upon Binding</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">2499</span>– <span class="NLM_lpage">2510</span>, <span class="refDoi"> DOI: 10.1021/jm030563w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030563w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD2cXivVans78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2499-2510&author=E.+Perolaauthor=P.+S.+Charifson&title=Conformational+Analysis+of+Drug-Like+Molecules+Bound+to+Proteins%3A+An+Extensive+Study+of+Ligand+Reorganization+upon+Binding&doi=10.1021%2Fjm030563w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Analysis of Drug-Like Molecules Bound to Proteins: An Extensive Study of Ligand Reorganization upon Binding</span></div><div class="casAuthors">Perola, Emanuele; Charifson, Paul S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2499-2510</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This paper describes a large-scale study on the nature and the energetics of the conformational changes drug-like mols. experience upon binding.  Ligand strain energies and conformational reorganization were analyzed with different computational methods on 150 crystal structures of pharmaceutically relevant protein-ligand complexes.  The common knowledge that ligands rarely bind in their lowest calcd. energy conformation was confirmed.  Addnl., the authors found that over 60% of the ligands do not bind in a local min. conformation.  While approx. 60% of the ligands were calcd. to bind with strain energies lower than 5 kcal/mol, strain energies over 9 kcal/mol were calcd. in at least 10% of the cases regardless of the method used.  A clear correlation was found between acceptable strain energy and ligand flexibility, while there was no correlation between strain energy and binding affinity, thus indicating that expensive conformational rearrangements can be tolerated in some cases without overly penalizing the tightness of binding.  On the basis of the trends obsd., thresholds for the acceptable strain energies of bioactive conformations were defined with consideration of the impact of ligand flexibility.  An anal. of the degree of folding of the bound ligands confirmed the general tendency of small mols. to bind in an extended conformation.  The results suggest that the unfolding of hydrophobic ligands during binding, which exposes hydrophobic surfaces to contact with protein residues, could be one of the factors accounting for high reorganization energies.  Finally, different methods for conformational anal. were evaluated, and guidelines were defined to maximize the prevalence of bioactive conformations in computationally generated ensembles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMrN0dFKglcbVg90H21EOLACvtfcHk0lgi3L-wJCyyZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXivVans78%253D&md5=8f54a8285ef1ffc5e1a19f748f4674bf</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fjm030563w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030563w%26sid%3Dliteratum%253Aachs%26aulast%3DPerola%26aufirst%3DE.%26aulast%3DCharifson%26aufirst%3DP.%2BS.%26atitle%3DConformational%2520Analysis%2520of%2520Drug-Like%2520Molecules%2520Bound%2520to%2520Proteins%253A%2520An%2520Extensive%2520Study%2520of%2520Ligand%2520Reorganization%2520upon%2520Binding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2499%26epage%3D2510%26doi%3D10.1021%2Fjm030563w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boström, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norrby, P.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liljefors, T.</span></span> <span> </span><span class="NLM_article-title">Conformational Energy Penalties of Protein-Bound Ligands</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">383</span>– <span class="NLM_lpage">396</span>, <span class="refDoi"> DOI: 10.1023/a:1008007507641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1023%2FA%3A1008007507641" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=9777496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADyaK1cXms1elsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1998&pages=383-396&author=J.+Bostr%C3%B6mauthor=P.-O.+Norrbyauthor=T.+Liljefors&title=Conformational+Energy+Penalties+of+Protein-Bound+Ligands&doi=10.1023%2Fa%3A1008007507641"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational energy penalties of protein-bound ligands</span></div><div class="casAuthors">Bostrom, Jonas; Norrby, Per-Ola; Liljefors, Tommy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">383-396</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">The conformational energies required for ligands to adopt their bioactive conformations were calcd. for 33 ligand-protein complexes including 28 different ligands.  In order to monitor the force field dependence of the results, two force fields, MM3* and AMBER*, were employed for the calcns.  Conformational analyses were performed in vacuo and in aq. soln. by using the generalized Born/solvent accessible surface (GB/SA) solvation model.  The protein-bound conformations were relaxed by using flat-bottomed Cartesian constraints.  For about 70% of the ligand-protein complexes studied, the conformational energies of the bioactive conformations were calcd. to be ≤3 kcal/mol.  It is demonstrated that the aq. conformational ensemble for the unbound ligand must be used as a ref. state in this type of calcns.  The calcns. for the ligand-protein complexes with conformational energy penalties of the ligand calcd. to be larger than 3 kcal/mol suffer from uncertainties in the interpretation of the exptl. data or limitations of the computational methods.  For example, in the case of long-chain flexible ligands (e.g. fatty acids), it is demonstrated that several conformations may be found which are very similar to the conformation detd. by x-ray crystallog. and which display significantly lower conformational energy penalties for binding than obtained by using the exptl. conformation.  For strongly polar mols., e.g. amino acids, the results indicate that further developments of the force fields and of the dielec. continuum solvation model are required for reliable calcns. on the conformational properties of this type of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOYXSRdHvDEbVg90H21EOLACvtfcHk0liIWYU7w9Q9JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXms1elsr4%253D&md5=d3108c97e8b080d5567b65653f9eff0d</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008007507641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008007507641%26sid%3Dliteratum%253Aachs%26aulast%3DBostr%25C3%25B6m%26aufirst%3DJ.%26aulast%3DNorrby%26aufirst%3DP.-O.%26aulast%3DLiljefors%26aufirst%3DT.%26atitle%3DConformational%2520Energy%2520Penalties%2520of%2520Protein-Bound%2520Ligands%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D1998%26volume%3D12%26spage%3D383%26epage%3D396%26doi%3D10.1023%2Fa%3A1008007507641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peach, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cachau, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicklaus, M. C.</span></span> <span> </span><span class="NLM_article-title">Conformational Energy Range of Ligands in Protein Crystal Structures: The Difficult Quest for Accurate Understanding</span>. <i>J. Mol. Recognit.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>30</i></span>, <span class="NLM_elocation-id">e2618</span> <span class="refDoi"> DOI: 10.1002/jmr.2618</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fjmr.2618" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2017&author=M.+L.+Peachauthor=R.+E.+Cachauauthor=M.+C.+Nicklaus&title=Conformational+Energy+Range+of+Ligands+in+Protein+Crystal+Structures%3A+The+Difficult+Quest+for+Accurate+Understanding&doi=10.1002%2Fjmr.2618"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1002%2Fjmr.2618&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmr.2618%26sid%3Dliteratum%253Aachs%26aulast%3DPeach%26aufirst%3DM.%2BL.%26aulast%3DCachau%26aufirst%3DR.%2BE.%26aulast%3DNicklaus%26aufirst%3DM.%2BC.%26atitle%3DConformational%2520Energy%2520Range%2520of%2520Ligands%2520in%2520Protein%2520Crystal%2520Structures%253A%2520The%2520Difficult%2520Quest%2520for%2520Accurate%2520Understanding%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D2017%26volume%3D30%26doi%3D10.1002%2Fjmr.2618" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Butler, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luque, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barril, X.</span></span> <span> </span><span class="NLM_article-title">Toward Accurate Relative Energy Predictions of the Bioactive Conformation of Drugs</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">601</span>– <span class="NLM_lpage">610</span>, <span class="refDoi"> DOI: 10.1002/jcc.21087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fjcc.21087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=18711721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD1MXit1Knsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=601-610&author=K.+T.+Butlerauthor=F.+J.+Luqueauthor=X.+Barril&title=Toward+Accurate+Relative+Energy+Predictions+of+the+Bioactive+Conformation+of+Drugs&doi=10.1002%2Fjcc.21087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Toward accurate relative energy predictions of the bioactive conformation of drugs</span></div><div class="casAuthors">Butler, Keith T.; Luque, F. Javier; Barril, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">601-610</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Quantifying the relative energy of a ligand in its target-bound state (i.e. the bioactive conformation) is essential to understand the process of mol. recognition, to optimize the potency of bioactive mols. and to increase the accuracy of structure-based drug design methods.  This is, nevertheless, seriously hampered by 2 inter-related issues, namely the difficulty in carrying out an exhaustive sampling of the conformational space and the shortcomings of the energy functions, usually based on parametric methods of limited accuracy.  Matters are further complicated by the exptl. uncertainty on the at. coordinates, which precludes a univocal definition of the bioactive conformation.  In this article the authors investigate the relative energy of bioactive conformations introducing 2 major improvements over previous studies: the use sophisticated QM-based methods to take into account both the internal energy of the ligand and the solvation effect, and the application of phys. meaningful constraints to refine the bioactive conformation.  On a set of 99 drug-like mols., the authors find that, contrary to previous observations, 2 thirds of bioactive conformations lie within 0.5 kcal mol-1 of a local min., with penalties above 2.0 kcal mol-1 being generally attributable to structural detn. inaccuracies.  The methodol. herein described opens the door to obtain quant. ests. of the energy of bioactive conformations and can be used both as an aid in refining crystallog. structures and as a tool in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoukwCddY9zl7Vg90H21EOLACvtfcHk0liIWYU7w9Q9JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXit1Knsrw%253D&md5=31ddffa90fb7f61e96b3b97ddeaac367</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21087%26sid%3Dliteratum%253Aachs%26aulast%3DButler%26aufirst%3DK.%2BT.%26aulast%3DLuque%26aufirst%3DF.%2BJ.%26aulast%3DBarril%26aufirst%3DX.%26atitle%3DToward%2520Accurate%2520Relative%2520Energy%2520Predictions%2520of%2520the%2520Bioactive%2520Conformation%2520of%2520Drugs%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D601%26epage%3D610%26doi%3D10.1002%2Fjcc.21087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borbulevych, O. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westerhoff, L. M.</span></span> <span> </span><span class="NLM_article-title">The Critical Role of QM/MM X-Ray Refinement and Accurate Tautomer/Protomer Determination in Structure-Based Drug Design</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">19</span>, <span class="refDoi"> DOI: 10.1007/s10822-020-00354-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1007%2Fs10822-020-00354-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=31792884" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=1-19&author=O.+Y.+Borbulevychauthor=R.+I.+Martinauthor=L.+M.+Westerhoff&title=The+Critical+Role+of+QM%2FMM+X-Ray+Refinement+and+Accurate+Tautomer%2FProtomer+Determination+in+Structure-Based+Drug+Design&doi=10.1007%2Fs10822-020-00354-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1007%2Fs10822-020-00354-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-020-00354-6%26sid%3Dliteratum%253Aachs%26aulast%3DBorbulevych%26aufirst%3DO.%2BY.%26aulast%3DMartin%26aufirst%3DR.%2BI.%26aulast%3DWesterhoff%26aufirst%3DL.%2BM.%26atitle%3DThe%2520Critical%2520Role%2520of%2520QM%252FMM%2520X-Ray%2520Refinement%2520and%2520Accurate%2520Tautomer%252FProtomer%2520Determination%2520in%2520Structure-Based%2520Drug%2520Design%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2020%26spage%3D1%26epage%3D19%26doi%3D10.1007%2Fs10822-020-00354-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janowski, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">York, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span> <span> </span><span class="NLM_article-title">Improved Ligand Geometries in Crystallographic Refinement Using AFITT in PHENIX</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">1062</span>– <span class="NLM_lpage">1072</span>, <span class="refDoi"> DOI: 10.1107/s2059798316012225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1107%2FS2059798316012225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVOhtr7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2016&pages=1062-1072&author=P.+A.+Janowskiauthor=N.+W.+Moriartyauthor=B.+P.+Kelleyauthor=D.+A.+Caseauthor=D.+M.+Yorkauthor=P.+D.+Adamsauthor=G.+L.+Warren&title=Improved+Ligand+Geometries+in+Crystallographic+Refinement+Using+AFITT+in+PHENIX&doi=10.1107%2Fs2059798316012225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Improved ligand geometries in crystallographic refinement using AFITT in PHENIX</span></div><div class="casAuthors">Janowski, Pawel A.; Moriarty, Nigel W.; Kelley, Brian P.; Case, David A.; York, Darrin M.; Adams, Paul D.; Warren, Gregory L.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1062-1072</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Modern crystal structure refinement programs rely on geometry restraints to overcome the challenge of a low data-to-parameter ratio.  While the classical Engh and Huber restraints work well for std. amino-acid residues, the chem. complexity of small-mol. ligands presents a particular challenge.  Most current approaches either limit ligand restraints to those that can be readily described in the Crystallog. Information File (CIF) format, thus sacrificing chem. flexibility and energetic accuracy, or they employ protocols that substantially lengthen the refinement time, potentially hindering rapid automated refinement workflows.  PHENIX-AFITT refinement uses a full mol.-mechanics force field for user-selected small-mol. ligands during refinement, eliminating the potentially difficult problem of finding or generating high-quality geometry restraints.  It is fully integrated with a std. refinement protocol and requires practically no addnl. steps from the user, making it ideal for high-throughput workflows.  PHENIX-AFITT refinements also handle multiple ligands in a single model, alternate conformations and covalently bound ligands.  Here, the results of combining AFITT and the PHENIX software suite on a data set of 189 protein-ligand PDB structures are presented.  Refinements using PHENIX-AFITT significantly reduce ligand conformational energy and lead to improved geometries without detriment to the fit to the exptl. data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAQdfkCh2PTbVg90H21EOLACvtfcHk0liLElVtMtGuYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVOhtr7J&md5=648534959c1ec463fc63856e56701988</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1107%2FS2059798316012225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798316012225%26sid%3Dliteratum%253Aachs%26aulast%3DJanowski%26aufirst%3DP.%2BA.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DKelley%26aufirst%3DB.%2BP.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DYork%26aufirst%3DD.%2BM.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26atitle%3DImproved%2520Ligand%2520Geometries%2520in%2520Crystallographic%2520Refinement%2520Using%2520AFITT%2520in%2520PHENIX%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2016%26volume%3D72%26spage%3D1062%26epage%3D1072%26doi%3D10.1107%2Fs2059798316012225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleves, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brueckner, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesburg, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherer, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reibarkh, M. Y.</span></span> <span> </span><span class="NLM_article-title">XGen: Real-Space Fitting of Complex Ligand Conformational Ensembles to X-Ray Electron Density Maps</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">10509</span>– <span class="NLM_lpage">10528</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01373</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01373" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslClsbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=10509-10528&author=A.+N.+Jainauthor=A.+E.+Clevesauthor=A.+C.+Bruecknerauthor=C.+A.+Lesburgauthor=Q.+Dengauthor=E.+C.+Shererauthor=M.+Y.+Reibarkh&title=XGen%3A+Real-Space+Fitting+of+Complex+Ligand+Conformational+Ensembles+to+X-Ray+Electron+Density+Maps&doi=10.1021%2Facs.jmedchem.0c01373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">XGen: Real-Space Fitting of Complex Ligand Conformational Ensembles to X-ray Electron Density Maps</span></div><div class="casAuthors">Jain, Ajay N.; Cleves, Ann E.; Brueckner, Alexander C.; Lesburg, Charles A.; Deng, Qiaolin; Sherer, Edward C.; Reibarkh, Mikhail Y.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10509-10528</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report a new method for X-ray d. ligand fitting and refinement that is suitable for a wide variety of small-mol. ligands, including macrocycles.  The approach (called "xGen") augments a force field energy calcn. with an electron d. fitting restraint that yields an energy reward during the restrained conformational search.  The resulting conformer pools balance goodness-of-fit with ligand strain.  Real-space refinement from pre-existing ligand coordinates of 150 macrocycles resulted in occupancy-weighted conformational ensembles that exhibited low strain energy.  The xGen ensembles improved upon electron d. fit compared with the PDB ref. coordinates without making use of atom-specific B-factors.  Similarly, on nonmacrocycles, de novo fitting produced occupancy-weighted ensembles of many conformers that were generally better-quality d. fits than the deposited primary/alternate conformational pairs.  The results suggest ubiquitous low-energy ligand conformational ensembles in X-ray diffraction data and provide an alternative to using B-factors as model parameters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraxQ4QG2Vcd7Vg90H21EOLACvtfcHk0liLElVtMtGuYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslClsbbF&md5=5ee4bb4268777ab8fbdcaf8adab2a586</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01373%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26aulast%3DCleves%26aufirst%3DA.%2BE.%26aulast%3DBrueckner%26aufirst%3DA.%2BC.%26aulast%3DLesburg%26aufirst%3DC.%2BA.%26aulast%3DDeng%26aufirst%3DQ.%26aulast%3DSherer%26aufirst%3DE.%2BC.%26aulast%3DReibarkh%26aufirst%3DM.%2BY.%26atitle%3DXGen%253A%2520Real-Space%2520Fitting%2520of%2520Complex%2520Ligand%2520Conformational%2520Ensembles%2520to%2520X-Ray%2520Electron%2520Density%2520Maps%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D10509%26epage%3D10528%26doi%3D10.1021%2Facs.jmedchem.0c01373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schade, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bazin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bent, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannifer, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickford, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prior, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irving, S. L.</span></span> <span> </span><span class="NLM_article-title">Design and Structure of Stapled Peptides Binding to Estrogen Receptors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">9696</span>– <span class="NLM_lpage">9699</span>, <span class="refDoi"> DOI: 10.1021/ja202946k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja202946k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntVems7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=9696-9699&author=C.+Phillipsauthor=L.+R.+Robertsauthor=M.+Schadeauthor=R.+Bazinauthor=A.+Bentauthor=N.+L.+Daviesauthor=R.+Mooreauthor=A.+D.+Panniferauthor=A.+R.+Pickfordauthor=S.+H.+Priorauthor=C.+M.+Readauthor=A.+Scottauthor=D.+G.+Brownauthor=B.+Xuauthor=S.+L.+Irving&title=Design+and+Structure+of+Stapled+Peptides+Binding+to+Estrogen+Receptors&doi=10.1021%2Fja202946k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Structure of Stapled Peptides Binding to Estrogen Receptors</span></div><div class="casAuthors">Phillips, Chris; Roberts, Lee R.; Schade, Markus; Bazin, Richard; Bent, Andrew; Davies, Nichola L.; Moore, Rob; Pannifer, Andrew D.; Pickford, Andrew R.; Prior, Stephen H.; Read, Christopher M.; Scott, Andrew; Brown, David G.; Xu, Bin; Irving, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">9696-9699</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Synthetic peptides that specifically bind nuclear hormone receptors offer an alternative approach to small mols. for the modulation of receptor signaling and subsequent gene expression.  Here the authors describe the design of a series of novel stapled peptides that bind the coactivator peptide site of estrogen receptors.  Using a no. of biophys. techniques, including crystal structure anal. of receptor-stapled peptide complexes, the authors describe in detail the mol. interactions and demonstrate that all-hydrocarbon staples modulate mol. recognition events.  The findings have implications for the design of stapled peptides in general.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-9uAz-tBuO7Vg90H21EOLACvtfcHk0liLElVtMtGuYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntVems7s%253D&md5=0c80b8dc5c2c36188f93659b5d1a2d00</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fja202946k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja202946k%26sid%3Dliteratum%253Aachs%26aulast%3DPhillips%26aufirst%3DC.%26aulast%3DRoberts%26aufirst%3DL.%2BR.%26aulast%3DSchade%26aufirst%3DM.%26aulast%3DBazin%26aufirst%3DR.%26aulast%3DBent%26aufirst%3DA.%26aulast%3DDavies%26aufirst%3DN.%2BL.%26aulast%3DMoore%26aufirst%3DR.%26aulast%3DPannifer%26aufirst%3DA.%2BD.%26aulast%3DPickford%26aufirst%3DA.%2BR.%26aulast%3DPrior%26aufirst%3DS.%2BH.%26aulast%3DRead%26aufirst%3DC.%2BM.%26aulast%3DScott%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DXu%26aufirst%3DB.%26aulast%3DIrving%26aufirst%3DS.%2BL.%26atitle%3DDesign%2520and%2520Structure%2520of%2520Stapled%2520Peptides%2520Binding%2520to%2520Estrogen%2520Receptors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D9696%26epage%3D9699%26doi%3D10.1021%2Fja202946k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Magiera-Mularz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skalniak, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zak, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musielak, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudzinska-Szostak, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berlicki, Ł.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kocik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grudnik, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sala, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarganes-Tzitzikas, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaabani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dömling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holak, T. A.</span></span> <span> </span><span class="NLM_article-title">Bioactive Macrocyclic Inhibitors of the PD-1/PD-L1 Immune Checkpoint</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">13732</span>– <span class="NLM_lpage">13735</span>, <span class="refDoi"> DOI: 10.1002/anie.201707707</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fanie.201707707" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOit7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=13732-13735&author=K.+Magiera-Mularzauthor=L.+Skalniakauthor=K.+M.+Zakauthor=B.+Musielakauthor=E.+Rudzinska-Szostakauthor=%C5%81.+Berlickiauthor=J.+Kocikauthor=P.+Grudnikauthor=D.+Salaauthor=T.+Zarganes-Tzitzikasauthor=S.+Shaabaniauthor=A.+D%C3%B6mlingauthor=G.+Dubinauthor=T.+A.+Holak&title=Bioactive+Macrocyclic+Inhibitors+of+the+PD-1%2FPD-L1+Immune+Checkpoint&doi=10.1002%2Fanie.201707707"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactive Macrocyclic Inhibitors of the PD-1/PD-L1 Immune Checkpoint</span></div><div class="casAuthors">Magiera-Mularz, Katarzyna; Skalniak, Lukasz; Zak, Krzysztof M.; Musielak, Bogdan; Rudzinska-Szostak, Ewa; Berlicki, Lukasz; Kocik, Justyna; Grudnik, Przemyslaw; Sala, Dominik; Zarganes-Tzitzikas, Tryfon; Shaabani, Shabnam; Doemling, Alexander; Dubin, Grzegorz; Holak, Tad A.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">13732-13735</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Blockade of the immunoinhibitory PD-1/PD-L1 pathway using monoclonal antibodies has shown impressive results with durable clin. antitumor responses.  Anti-PD-1 and anti-PD-L1 antibodies have now been approved for the treatment of a no. of tumor types, whereas the development of small mols. targeting immune checkpoints lags far behind.  We characterized two classes of macrocyclic-peptide inhibitors directed at the PD-1/PD-L1 pathway.  We show that these macrocyclic compds. act by directly binding to PD-L1 and that they are capable of antagonizing PD-L1 signaling and, similarly to antibodies, can restore the function of T-cells.  We also provide the crystal structures of two of these small-mol. inhibitors bound to PD-L1.  The structures provide a rationale for the checkpoint inhibition by these small mols., and a description of their small mol./PD-L1 interfaces provides a blueprint for the design of small-mol. inhibitors of the PD-1/PD-L1 pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxuarSIlQEobVg90H21EOLACvtfcHk0lg4AUH6xSztOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOit7%252FO&md5=b1634408007340ca0990d2607c96c1a9</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1002%2Fanie.201707707&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201707707%26sid%3Dliteratum%253Aachs%26aulast%3DMagiera-Mularz%26aufirst%3DK.%26aulast%3DSkalniak%26aufirst%3DL.%26aulast%3DZak%26aufirst%3DK.%2BM.%26aulast%3DMusielak%26aufirst%3DB.%26aulast%3DRudzinska-Szostak%26aufirst%3DE.%26aulast%3DBerlicki%26aufirst%3D%25C5%2581.%26aulast%3DKocik%26aufirst%3DJ.%26aulast%3DGrudnik%26aufirst%3DP.%26aulast%3DSala%26aufirst%3DD.%26aulast%3DZarganes-Tzitzikas%26aufirst%3DT.%26aulast%3DShaabani%26aufirst%3DS.%26aulast%3DD%25C3%25B6mling%26aufirst%3DA.%26aulast%3DDubin%26aufirst%3DG.%26aulast%3DHolak%26aufirst%3DT.%2BA.%26atitle%3DBioactive%2520Macrocyclic%2520Inhibitors%2520of%2520the%2520PD-1%252FPD-L1%2520Immune%2520Checkpoint%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D13732%26epage%3D13735%26doi%3D10.1002%2Fanie.201707707" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gurusaran, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shankar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagarajan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helliwell, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekar, K.</span></span> <span> </span><span class="NLM_article-title">Do We See What We Should See? Describing Non-Covalent Interactions in Protein Structures Including Precision</span>. <i>IUCrJ</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">74</span>– <span class="NLM_lpage">81</span>, <span class="refDoi"> DOI: 10.1107/s2052252513031485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1107%2FS2052252513031485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=25075321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotFWmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2014&pages=74-81&author=M.+Gurusaranauthor=M.+Shankarauthor=R.+Nagarajanauthor=J.+R.+Helliwellauthor=K.+Sekar&title=Do+We+See+What+We+Should+See%3F+Describing+Non-Covalent+Interactions+in+Protein+Structures+Including+Precision&doi=10.1107%2Fs2052252513031485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Do we see what we should see? Describing non-covalent interactions in protein structures including precision</span></div><div class="casAuthors">Gurusaran, Manickam; Shankar, Mani; Nagarajan, Raju; Helliwell, John R.; Sekar, Kanagaraj</div><div class="citationInfo"><span class="NLM_cas:title">IUCrJ</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">74-81</span>CODEN:
                <span class="NLM_cas:coden">IUCRAJ</span>;
        ISSN:<span class="NLM_cas:issn">2052-2525</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The power of X-ray crystal structure anal. as a technique is to 'see where the atoms are'.  The results are extensively used by a wide variety of research communities.  However, this 'seeing where the atoms are' can give a false sense of security unless the precision of the placement of the atoms has been taken into account.  Indeed, the presentation of bond distances and angles to a false precision (i.e. to too many decimal places) is commonplace.  This article has three themes.  Firstly, a basis for a proper representation of protein crystal structure results is detailed and demonstrated with respect to analyses of Protein Data Bank entries.  The basis for establishing the precision of placement of each atom in a protein crystal structure is non-trivial.  Secondly, a knowledge base harnessing such a descriptor of precision is presented.  It is applied here to the case of salt bridges, i.e. ion pairs, in protein structures; this is the most fundamental place to start with such structure-precision representations since salt bridges are one of the tenets of protein structure stability.  Ion pairs also play a central role in protein oligomerization, mol. recognition of ligands and substrates, allosteric regulation, domain motion and α-helix capping.  A new knowledge base, SBPS (Salt Bridges in Protein Structures), takes these structural precisions into account and is the first of its kind.  The third theme of the article is to indicate natural extensions of the need for such a description of precision, such as those involving metalloproteins and the detn. of the protonation states of ionizable amino acids.  Overall, it is also noted that this work and these examples are also relevant to protein three-dimensional structure mol. graphics software.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqEll24sceprVg90H21EOLACvtfcHk0lg4AUH6xSztOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotFWmsA%253D%253D&md5=8ae721a59f55521ba14113362bbd059f</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1107%2FS2052252513031485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2052252513031485%26sid%3Dliteratum%253Aachs%26aulast%3DGurusaran%26aufirst%3DM.%26aulast%3DShankar%26aufirst%3DM.%26aulast%3DNagarajan%26aufirst%3DR.%26aulast%3DHelliwell%26aufirst%3DJ.%2BR.%26aulast%3DSekar%26aufirst%3DK.%26atitle%3DDo%2520We%2520See%2520What%2520We%2520Should%2520See%253F%2520Describing%2520Non-Covalent%2520Interactions%2520in%2520Protein%2520Structures%2520Including%2520Precision%26jtitle%3DIUCrJ%26date%3D2014%26volume%3D1%26spage%3D74%26epage%3D81%26doi%3D10.1107%2Fs2052252513031485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexov, E.</span></span> <span> </span><span class="NLM_article-title">On the Dielectric “Constant” of Proteins: Smooth Dielectric Function for Macromolecular Modeling and Its Implementation in DelPhi</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2136</span>, <span class="refDoi"> DOI: 10.1021/ct400065j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400065j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjvVKrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=2126-2136&author=L.+Liauthor=C.+Liauthor=Z.+Zhangauthor=E.+Alexov&title=On+the+Dielectric+%E2%80%9CConstant%E2%80%9D+of+Proteins%3A+Smooth+Dielectric+Function+for+Macromolecular+Modeling+and+Its+Implementation+in+DelPhi&doi=10.1021%2Fct400065j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">On the Dielectric "Constant" of Proteins: Smooth Dielectric Function for Macromolecular Modeling and Its Implementation in DelPhi</span></div><div class="casAuthors">Li, Lin; Li, Chuan; Zhang, Zhe; Alexov, Emil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2126-2136</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Implicit methods for modeling protein electrostatics require dielec. properties of the system to be known, in particular, the value of the dielec. const. of the protein.  While numerous values of the internal protein dielec. const. were reported in the literature, still there is no consensus of what the optimal value is.  Perhaps this is due to the fact that the protein dielec. const. is not a "const." but is a complex function reflecting the properties of the protein's structure and sequence.  Here, we report an implementation of a Gaussian-based approach to deliver the dielec. const. distribution throughout the protein and surrounding water phase by utilizing the 3D structure of the corresponding macromol.  In contrast to previous reports, we construct a smooth dielec. function throughout the space of the system to be modeled rather than just constructing a "Gaussian surface" or smoothing mol.-water boundary.  Anal. on a large set of proteins shows that (a) the av. dielec. const. inside the protein is relatively low, about 6-7, and reaches a value of about 20-30 at the protein's surface, and (b) high av. local dielec. const. values are assocd. with charged residues while low dielec. const. values are automatically assigned to the regions occupied by hydrophobic residues.  In terms of energetics, a benchmarking test was carried out against the exptl. pKa's of 89 residues in staphylococcal nuclease (SNase) and showed that it results in a much better RMSD (= 1.77 pK) than the corresponding calcns. done with a homogeneous high dielec. const. with an optimal value of 10 (RMSD = 2.43 pK).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuokYHiKExsrVg90H21EOLACvtfcHk0lg4AUH6xSztOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjvVKrs7g%253D&md5=ce5218db0e9312a4e378239d340fe799</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fct400065j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400065j%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DAlexov%26aufirst%3DE.%26atitle%3DOn%2520the%2520Dielectric%2520%25E2%2580%259CConstant%25E2%2580%259D%2520of%2520Proteins%253A%2520Smooth%2520Dielectric%2520Function%2520for%2520Macromolecular%2520Modeling%2520and%2520Its%2520Implementation%2520in%2520DelPhi%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D2126%26epage%3D2136%26doi%3D10.1021%2Fct400065j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kendall, M. G.</span></span> <span> </span><span class="NLM_article-title">A New Measure of Rank Correlation</span>. <i>Biometrika</i> <span class="NLM_year" style="font-weight: bold;">1938</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">93</span>, <span class="refDoi"> DOI: 10.2307/2332226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1093%2Fbiomet%2F30.1-2.81" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1938&pages=81-93&author=M.+G.+Kendall&title=A+New+Measure+of+Rank+Correlation&doi=10.2307%2F2332226"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1093%2Fbiomet%2F30.1-2.81&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbiomet%252F30.1-2.81%26sid%3Dliteratum%253Aachs%26aulast%3DKendall%26aufirst%3DM.%2BG.%26atitle%3DA%2520New%2520Measure%2520of%2520Rank%2520Correlation%26jtitle%3DBiometrika%26date%3D1938%26volume%3D30%26spage%3D81%26epage%3D93%26doi%3D10.2307%2F2332226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, R.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucke, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Flexibility versus Rigidity for Orally Bioavailable Cyclic Hexapeptides</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">2289</span>– <span class="NLM_lpage">2293</span>, <span class="refDoi"> DOI: 10.1002/cbic.201500441</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fcbic.201500441" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=26336864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Wjt73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=2289-2293&author=D.+S.+Nielsenauthor=R.-J.+Lohmanauthor=H.+N.+Hoangauthor=T.+A.+Hillauthor=A.+Jonesauthor=A.+J.+Luckeauthor=D.+P.+Fairlie&title=Flexibility+versus+Rigidity+for+Orally+Bioavailable+Cyclic+Hexapeptides&doi=10.1002%2Fcbic.201500441"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Flexibility versus Rigidity for Orally Bioavailable Cyclic Hexapeptides</span></div><div class="casAuthors">Nielsen, Daniel S.; Lohman, Rink-Jan; Hoang, Huy N.; Hill, Timothy A.; Jones, Alun; Lucke, Andrew J.; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2289-2293</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cyclic peptides and macrocycles have the potential to be membrane permeable and orally bioavailable, despite often not complying with the "rule of five" used in medicinal chem. to guide the discovery of oral drugs.  Here we compare solvent-dependent three-dimensional structures of three cyclic hexapeptides contg. D-amino acids, prolines, and intramol. hydrogen bonds.  Conformational rigidity rather than flexibility resulted in higher membrane permeability, metabolic stability and oral bioavailability, consistent with less polar surface exposure to solvent and a reduced entropy penalty for transition between polar and nonpolar environments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpArFfvUbUBA7Vg90H21EOLACvtfcHk0lgF1TpdbWxcKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Wjt73I&md5=b0cf19a1ee875648d518dcc0befd9c28</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201500441&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201500441%26sid%3Dliteratum%253Aachs%26aulast%3DNielsen%26aufirst%3DD.%2BS.%26aulast%3DLohman%26aufirst%3DR.-J.%26aulast%3DHoang%26aufirst%3DH.%2BN.%26aulast%3DHill%26aufirst%3DT.%2BA.%26aulast%3DJones%26aufirst%3DA.%26aulast%3DLucke%26aufirst%3DA.%2BJ.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DFlexibility%2520versus%2520Rigidity%2520for%2520Orally%2520Bioavailable%2520Cyclic%2520Hexapeptides%26jtitle%3DChemBioChem%26date%3D2015%26volume%3D16%26spage%3D2289%26epage%3D2293%26doi%3D10.1002%2Fcbic.201500441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millhauser, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Testing the Conformational Hypothesis of Passive Membrane Permeability Using Synthetic Cyclic Peptide Diastereomers</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">2510</span>– <span class="NLM_lpage">2511</span>, <span class="refDoi"> DOI: 10.1021/ja0563455</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0563455" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVGqu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=2510-2511&author=T.+Rezaiauthor=B.+Yuauthor=G.+L.+Millhauserauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Testing+the+Conformational+Hypothesis+of+Passive+Membrane+Permeability+Using+Synthetic+Cyclic+Peptide+Diastereomers&doi=10.1021%2Fja0563455"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Testing the conformational hypothesis of passive membrane permeability using synthetic cyclic peptide diastereomers</span></div><div class="casAuthors">Rezai, Taha; Yu, Bin; Millhauser, Glenn L.; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2510-2511</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Little is known about the effect of conformation on passive membrane diffusion rates in small mols.  Evidence suggests that intramol. hydrogen bonding may play a role by reducing the energetic cost of desolvating hydrogen bond donors, esp. amide N-H groups.  We set out to test this hypothesis by investigating the passive membrane diffusion characteristics of a series of cyclic peptide diastereomers based on the sequence cyclo[Leu-Leu-Leu-Leu-Pro-Tyr].  We identified two cyclic hexapeptide diastereomers based on this sequence, whose membrane diffusion rates differed by nearly two log units.  Results of soln. NMR studies and hydrogen/deuterium (H/D) exchange expts. showed that membrane diffusion rates correlated with the degree of intramol. hydrogen bonding and H/D exchange rates.  The most permeable diastereomer, cyclo[D-Leu-D-Leu-Leu-D-Leu-Pro-Tyr], exhibited a passive membrane diffusion rate comparable to that of the orally available drug cyclosporine A.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqooDObTy1BXrVg90H21EOLACvtfcHk0lgF1TpdbWxcKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVGqu7k%253D&md5=52e5cac6648efb70e743f294e3f8c4ec</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fja0563455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0563455%26sid%3Dliteratum%253Aachs%26aulast%3DRezai%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DMillhauser%26aufirst%3DG.%2BL.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DTesting%2520the%2520Conformational%2520Hypothesis%2520of%2520Passive%2520Membrane%2520Permeability%2520Using%2520Synthetic%2520Cyclic%2520Peptide%2520Diastereomers%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D2510%26epage%3D2511%26doi%3D10.1021%2Fja0563455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilliland, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindyalov, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">The Protein Data Bank</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lgF1TpdbWxcKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span> <i>The PyMOL Molecular
Graphics System</i>, Version 2.3.1; <span class="NLM_publisher-name">Schrödinger,
LLC</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+The+PyMOL+Molecular%0AGraphics+System%2C+Version+2.3.1%3B+Schr%C3%B6dinger%2C%0ALLC."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520PyMOL%2520Molecular%250AGraphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. N.</span></span> <span> </span><span class="NLM_article-title">Surflex: Fully Automatic Flexible Molecular Docking Using a Molecular Similarity-Based Search Engine</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">499</span>– <span class="NLM_lpage">511</span>, <span class="refDoi"> DOI: 10.1021/jm020406h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020406h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkvFKjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=499-511&author=A.+N.+Jain&title=Surflex%3A+Fully+Automatic+Flexible+Molecular+Docking+Using+a+Molecular+Similarity-Based+Search+Engine&doi=10.1021%2Fjm020406h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Surflex: fully automatic flexible molecular docking using a molecular similarity-based search engine</span></div><div class="casAuthors">Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">499-511</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Surflex is a fully automatic flexible mol. docking algorithm that combines the scoring function from the Hammerhead docking system with a search engine that relies on a surface-based mol. similarity method as a means to rapidly generate suitable putative poses for mol. fragments.  Results are presented evaluating reliability and accuracy of dockings compared with crystallog. exptl. results on 81 protein/ligand pairs of substantial structural diversity.  In over 80% of the complexes, Surflex's highest scoring docked pose was within 2.5 Å root-mean-square deviation (rmsd), with over 90% of the complexes having one of the top ranked poses within 2.5 Å rmsd.  Results are also presented assessing Surflex's utility as a screening tool on two protein targets (thymidine kinase and estrogen receptor) using data sets on which competing methods were run.  Performance of Surflex was significantly better, with true pos. rates of greater than 80% at false pos. rates of less than 1%.  Docking time was roughly linear in no. of rotatable bonds, beginning with a few seconds for rigid mols. and adding approx. 10 s per rotatable bond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrawn5WHbx5p7Vg90H21EOLACvtfcHk0liY3ezMjnT4PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkvFKjsA%253D%253D&md5=67a08aafeadf69101d56b2f60a922359</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1021%2Fjm020406h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020406h%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DSurflex%253A%2520Fully%2520Automatic%2520Flexible%2520Molecular%2520Docking%2520Using%2520a%2520Molecular%2520Similarity-Based%2520Search%2520Engine%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D499%26epage%3D511%26doi%3D10.1021%2Fjm020406h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spitzer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. N.</span></span> <span> </span><span class="NLM_article-title">Surflex-Dock: Docking Benchmarks and Real-World Application</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">699</span>, <span class="refDoi"> DOI: 10.1007/s10822-011-9533-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1007%2Fs10822-011-9533-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=22569590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVelu7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=687-699&author=R.+Spitzerauthor=A.+N.+Jain&title=Surflex-Dock%3A+Docking+Benchmarks+and+Real-World+Application&doi=10.1007%2Fs10822-011-9533-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Surflex-Dock: Docking benchmarks and real-world application</span></div><div class="casAuthors">Spitzer, Russell; Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">687-699</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Benchmarks for mol. docking have historically focused on re-docking the cognate ligand of a well-detd. protein-ligand complex to measure geometric pose prediction accuracy, and measurement of virtual screening performance has been focused on increasingly large and diverse sets of target protein structures, cognate ligands, and various types of decoy sets.  Here, pose prediction is reported on the Astex Diverse set of 85 protein ligand complexes, and virtual screening performance is reported on the DUD set of 40 protein targets.  In both cases, prepd. structures of targets and ligands were provided by symposium organizers.  The re-prepd. data sets yielded results not significantly different than previous reports of Surflex-Dock on the two benchmarks.  Minor changes to protein coordinates resulting from complex pre-optimization had large effects on obsd. performance, highlighting the limitations of cognate ligand re-docking for pose prediction assessment.  Docking protocols developed for cross-docking, which address protein flexibility and produce discrete families of predicted poses, produced substantially better performance for pose prediction.  Performance on virtual screening performance was shown to benefit by employing and combining multiple screening methods: docking, 2D mol. similarity, and 3D mol. similarity.  In addn., use of multiple protein conformations significantly improved screening enrichment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPgES-yaoU4rVg90H21EOLACvtfcHk0liY3ezMjnT4PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVelu7nN&md5=78637e767d2ad764f4b89c24ef3c7a56</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9533-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9533-y%26sid%3Dliteratum%253Aachs%26aulast%3DSpitzer%26aufirst%3DR.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DSurflex-Dock%253A%2520Docking%2520Benchmarks%2520and%2520Real-World%2520Application%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D687%26epage%3D699%26doi%3D10.1007%2Fs10822-011-9533-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleves, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. N.</span></span> <span> </span><span class="NLM_article-title">Knowledge-Guided Docking: Accurate Prospective Prediction of Bound Configurations of Novel Ligands Using Surflex-Dock</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">485</span>– <span class="NLM_lpage">509</span>, <span class="refDoi"> DOI: 10.1007/s10822-015-9846-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1007%2Fs10822-015-9846-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=25940276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnvVShsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=485-509&author=A.+E.+Clevesauthor=A.+N.+Jain&title=Knowledge-Guided+Docking%3A+Accurate+Prospective+Prediction+of+Bound+Configurations+of+Novel+Ligands+Using+Surflex-Dock&doi=10.1007%2Fs10822-015-9846-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Knowledge-guided docking: accurate prospective prediction of bound configurations of novel ligands using Surflex-Dock</span></div><div class="casAuthors">Cleves, Ann E.; Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">485-509</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Prediction of the bound configuration of small-mol. ligands that differ substantially from the cognate ligand of a protein co-crystal structure is much more challenging than re-docking the cognate ligand.  Success rates for cross-docking in the range of 20-30 % are common.  We present an approach that uses structural information known prior to a particular cutoff-date to make predictions on ligands whose bounds structures were detd. later.  The knowledge-guided docking protocol was tested on a set of ten protein targets using a total of 949 ligands.  The benchmark data set, called PINC ("PINC Is Not Cognate"), is publicly available.  Protein pocket similarity was used to choose representative structures for ensemble-docking.  The docking protocol made use of known ligand poses prior to the cutoff-date, both to help guide the configurational search and to adjust the rank of predicted poses.  Overall, the top-scoring pose family was correct over 60 % of the time, with the top-two pose families approaching a 75 % success rate.  Correct poses among all those predicted were identified nearly 90 % of the time.  The largest improvements came from the use of mol. similarity to improve ligand pose rankings and the strategy for identifying representative protein structures.  With the exception of a single outlier target, the knowledge-guided docking protocol produced results matching the quality of cognate-ligand re-docking, but it did so on a very challenging temporally-segregated cross-docking benchmark.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnNcQG8X--3bVg90H21EOLACvtfcHk0lgJ_RXLNfYsgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnvVShsLo%253D&md5=b0deddab12f7289136703e2c4c75f266</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9846-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9846-3%26sid%3Dliteratum%253Aachs%26aulast%3DCleves%26aufirst%3DA.%2BE.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DKnowledge-Guided%2520Docking%253A%2520Accurate%2520Prospective%2520Prediction%2520of%2520Bound%2520Configurations%2520of%2520Novel%2520Ligands%2520Using%2520Surflex-Dock%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D485%26epage%3D509%26doi%3D10.1007%2Fs10822-015-9846-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pham, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. N.</span></span> <span> </span><span class="NLM_article-title">Customizing Scoring Functions for Docking</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">286</span>, <span class="refDoi"> DOI: 10.1007/s10822-008-9174-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1007%2Fs10822-008-9174-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=18273558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD1cXksVKks7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=269-286&author=T.+A.+Phamauthor=A.+N.+Jain&title=Customizing+Scoring+Functions+for+Docking&doi=10.1007%2Fs10822-008-9174-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Customizing scoring functions for docking</span></div><div class="casAuthors">Pham, Tuan A.; Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">269-286</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Empirical scoring functions used in protein-ligand docking calcns. are typically trained on a dataset of complexes with known affinities with the aim of generalizing across different docking applications.  The authors report a novel method of scoring-function optimization that supports the use of addnl. information to constrain scoring function parameters, which can be used to focus a scoring function's training towards a particular application, such as screening enrichment.  The approach combines multiple instance learning, pos. data in the form of ligands of protein binding sites of known and unknown affinity and binding geometry, and neg. (decoy) data of ligands thought not to bind particular protein binding sites or known not to bind in particular geometries.  Performance of the method for the Surflex-Dock scoring function is shown in cross-validation studies and in eight blind test cases.  Tuned functions optimized with a sufficient amt. of data exhibited either improved or undiminished screening performance relative to the original function across all eight complexes.  Anal. of the changes to the scoring function suggest that modifications can be learned that are related to protein-specific features such as active-site mobility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ2udGn_HJLrVg90H21EOLACvtfcHk0lgJ_RXLNfYsgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXksVKks7k%253D&md5=ec0b2d3159f8970abcd431b085df0a7d</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1007%2Fs10822-008-9174-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-008-9174-y%26sid%3Dliteratum%253Aachs%26aulast%3DPham%26aufirst%3DT.%2BA.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DCustomizing%2520Scoring%2520Functions%2520for%2520Docking%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2008%26volume%3D22%26spage%3D269%26epage%3D286%26doi%3D10.1007%2Fs10822-008-9174-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleves, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. N.</span></span> <span> </span><span class="NLM_article-title">Structure- and Ligand-Based Virtual Screening on DUD-E+: Performance Dependence on Approximations to the Binding Pocket</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4296</span>– <span class="NLM_lpage">4310</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00115</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00115" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BB3cXms1yjsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=4296-4310&author=A.+E.+Clevesauthor=A.+N.+Jain&title=Structure-+and+Ligand-Based+Virtual+Screening+on+DUD-E%2B%3A+Performance+Dependence+on+Approximations+to+the+Binding+Pocket&doi=10.1021%2Facs.jcim.0c00115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Structure- and Ligand-Based Virtual Screening on DUD-E+: Performance Dependence on Approximations to the Binding Pocket</span></div><div class="casAuthors">Cleves, Ann E.; Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4296-4310</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using the DUD-E+ benchmark, we explore the impact of using a single protein pocket or ligand for virtual screening compared with using ensembles of alternative pockets, ligands, and sets thereof.  For both structure-based and ligand-based approaches, the precise characterization of the binding site in question had a significant impact on screening performance.  Using the single original DUD-E protein, Surflex-Dock yielded mean ROC area of 0.81 ± 0.11.  Using the cognate ligand instead, with the eSim method for screening, yielded 0.77 ± 0.14.  Moving to ensembles of five protein pocket variants increased docking performance to 0.84 ± 0.09.  Results for the analogous ligand-based approach (using the five crystallog. aligned cognate ligands) was 0.83 ± 0.11.  Using the same ligands, but making use of an automatically generated mutual alignment, yielded mean AUC nearly as good as from single-structure docking: 0.80 ± 0.12.  Detailed results and statistical analyses show that structure- and ligand-based methods are complementary and can be fruitfully combined to enhance screening efficiency.  A hybrid approach combining ensemble docking with eSim-based screening produced the best and most consistent performance (mean ROC area of 0.89 ± 0.08 and 1% early enrichment of 46-fold).  Based on results from both the docking and ligand-similarity approaches, it is clearly unwise to make use of a single arbitrarily chosen protein structure for docking or single ligand query for similarity-based screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvrSodjvqflLVg90H21EOLACvtfcHk0liczBi37n-BDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXms1yjsLg%253D&md5=da167891883e8450776b3983053fb93a</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00115%26sid%3Dliteratum%253Aachs%26aulast%3DCleves%26aufirst%3DA.%2BE.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DStructure-%2520and%2520Ligand-Based%2520Virtual%2520Screening%2520on%2520DUD-E%252B%253A%2520Performance%2520Dependence%2520on%2520Approximations%2520to%2520the%2520Binding%2520Pocket%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D4296%26epage%3D4310%26doi%3D10.1021%2Facs.jcim.0c00115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Composition of the macrocyclic peptide data set. See the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for a list of PDB IDs. (A) Classification of proteins. Only sectors >5% of the total population are labeled. Distribution of (B) deposited structures by year and (C) ligands by size. (D) Cumulative population of PDB structures within resolution cutoffs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Conformational search and ensemble derivation. (A) All conformers resulting from a restrained search of the <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DV1">3DV1</a> ligand, blending force field energetics with a quantitative reward for matching electron density. (B) Single high-quality conformer trio, representing both good fit to the density (orange) and low energy (yellow) along with a conformer with low rmsd to both (slate). (C) Occupancy-weighted conformer ensemble with the 1.0σ experimental density contour (gray mesh) and the corresponding calculated real-space density contour (cyan dots).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Example of alternative fits to electron density for<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O4Y">5O4Y</a>. The atomic coordinates of the deposited ligand model are shown in green sticks, with a set of five conformers comprising an xGen ensemble shown in orange. The electron density contour from the 2|Fo| – |Fc| map is shown at 1.0σ. Red numbers 1–3 mark a position where a high-energy <i>cis</i>-amide in the deposited coordinates is flipped to a low-energy <i>trans</i>-amide in the xGen ensemble. Red numbers 4–6 show alternative side chain rotamers in the xGen ensemble compared to the deposited coordinates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Deviation from the crystallographic experimental support for the macrocyclic peptide data set. (B) Cumulative histogram of global strain energy. Red-dotted lines are for the square-welled quadratic positional restraint approach. Blue solid lines are for the xGen electron density fitting approach. Yellow dot-dashed lines are for the B-factor binning approach. Gray-dashed lines are for the coordinate uncertainty approach. The xGen electron density fitting retained high fidelity to the crystallographic data while producing the lowest strain estimates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Deviations from the xGen ensembles by the Boltzmann-weighted minima were identical for all molecular classes. (B) Cumulative histogram of strain energy. Blue solid lines are for the macrocyclic peptide data set, green-dotted lines are for the non-peptidic macrocycle data set, and the purple-dashed lines are for the small molecule data set. All results are obtained from the electron density fitting approach (xGen). Vertical lines correspond to the 90th percentile for each data set, colored respectively. Strain energy estimates suggest the interaction energy in protein–ligand complexes can offset a greater amount of strain for macrocyclic peptides than for non-peptidic macrocycles or small molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Relationship between HAC and global strain energy featuring a lower-right triangular distribution. Blue squares are for the macrocyclic peptide data set, green circles are for the non-peptidic macrocycle data set, and purple triangles are for the small molecule data set. The black-dashed line is an approximate upper bound of the estimated strain energy.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Number of conformers in the xGen ensembles. (A) Roughly 80% of the data set had more than one conformer in the final xGen ensemble. (B) Overlay of the deposited ligand conformer for<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H0I">1H0I</a> (green) and the two alternative conformers in the xGen ensemble (orange). (C) Overlay of the deposited ligand conformer for<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MF8">1MF8</a> (green) and the nine alternative conformers in the xGen ensemble (orange). Isosurface contour for ρ<sub>calc</sub> corresponding to 1.0σ for the xGen ensemble is shown as cyan dots. <i>E</i><sub>pos. res.</sub> and <i>E</i><sub>xGen</sub> are the estimated global strain energies for the positional restraint and density fitting methodologies, respectively. Energies are in kcal/mol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Backbone residue flipping. (A) 27% of the data set featured a backbone residue flip in the final xGen ensemble relative to the deposited conformer. (B) Overlay of the deposited ligand conformer for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O4Y">5O4Y</a> (green) and the five alternative conformers in the xGen ensemble (orange). (C) Detailed view of a selection of backbone residues in the deposited conformer featuring <i>cis</i>-amide 1 from <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> (red text). (D) Detailed view of the same residues in the final xGen ensemble, now featuring a <i>trans</i>-amide (green text) and a new intramolecular hydrogen bond. Isosurface contour for ρ<sub>calc</sub> corresponding to 1.0σ for the xGen ensemble is shown as cyan dots. <i>E</i><sub>pos. res.</sub> and <i>E</i><sub>xGen</sub> are the estimated global strain energies for the positional restraint and density fitting methodologies, respectively. Distances are in Å and energies are in kcal/mol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Alternative linker rotamers. (A) 44% of the data set with non-peptidic linkers had alternative linker rotamers. (B) Overlay of deposited ligand conformer for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZQW">4ZQW</a> (green) and the five alternative conformers in the xGen ensemble (orange). (C) Overlay of the deposited ligand conformer for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWM">1VWM</a> (green) and the two alternative conformers in the xGen ensemble (orange). Major linker rotamers are noted in the red text. Isosurface contour for ρ<sub>calc</sub> corresponding to 1.0σ for the xGen ensemble is shown as cyan dots. <i>E</i><sub>pos. res.</sub> and <i>E</i><sub>xGen</sub> are the estimated global strain energies for the positional restraint and density fitting methodologies, respectively. Energies are in kcal/mol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Alternative side chain rotamers. (A) 90% of the data set had alternative side chain rotamers. (B) Overlay of the deposited ligand conformer for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1C5F">1C5F</a> (green) and the three alternative conformers in the xGen ensemble (orange). (C) Overlay of the deposited ligand conformer for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X6S">4X6S</a> (green) and the six alternative conformers in the xGen ensemble (orange). Major side chain rotamers are noted in the red text. Isosurface contour for ρ<sub>calc</sub> corresponding to 1.0σ for the xGen ensemble is shown as cyan dots. <i>E</i><sub>pos. res.</sub> and <i>E</i><sub>xGen</sub> are the estimated global strain energies for the positional restraint and density fitting methodologies, respectively. Energies are in kcal/mol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Alternative intermolecular interactions. (A) Overlay of the deposited ligand conformer for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5B4W">5B4W</a> (green) and the three alternative conformers in the xGen ensemble (orange) in the crystal structure. Plexin B1 is shown in purple. (B) Detailed view of the deposited peptide arginine side chain and Plexin B1 Asp414 featuring one strong and one weak hydrogen bond (yellow- and red-dotted lines, respectively). (C) Detailed view of the same residues in the final xGen ensemble, now featuring an arginine side chain rotamer with two strong intermolecular hydrogen bonds (yellow-dotted lines). Isosurface contour for ρ<sub>calc</sub> corresponding to 1.0σ for the xGen ensemble is shown as cyan dots. <i>E</i><sub>pos. res.</sub> and <i>E</i><sub>xGen</sub> are the estimated global strain energies for the positional restraint and density fitting methodologies, respectively. Distances are in Å and energies are in kcal/mol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/medium/jm0c02159_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Cumulative histogram of enthalpy. Blue solid lines are for the macrocyclic peptide data set, green-dotted lines are for the non-peptidic macrocycle data set, and the purple-dashed lines are for the small molecule data set. All results are from docking the electron density fitting approach ensembles (xGen).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02159/20210409/images/large/jm0c02159_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02159&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i24">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21126" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21126" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 82 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Over, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grönberg, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgendorf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muncipinto, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellisson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, M. W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvall, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Impact of Stereospecific Intramolecular Hydrogen Bonding on Cell Permeability and Physicochemical Properties</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2746</span>– <span class="NLM_lpage">2754</span>, <span class="refDoi"> DOI: 10.1021/jm500059t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500059t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFKiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=2746-2754&author=B.+Overauthor=P.+McCarrenauthor=P.+Arturssonauthor=M.+Foleyauthor=F.+Giordanettoauthor=G.+Gr%C3%B6nbergauthor=C.+Hilgendorfauthor=M.+D.+Leeauthor=P.+Matssonauthor=G.+Muncipintoauthor=M.+Pellissonauthor=M.+W.+D.+Perryauthor=R.+Svenssonauthor=J.+R.+Duvallauthor=J.+Kihlberg&title=Impact+of+Stereospecific+Intramolecular+Hydrogen+Bonding+on+Cell+Permeability+and+Physicochemical+Properties&doi=10.1021%2Fjm500059t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Stereospecific Intramolecular Hydrogen Bonding on Cell Permeability and Physicochemical Properties</span></div><div class="casAuthors">Over, Bjoern; McCarren, Patrick; Artursson, Per; Foley, Michael; Giordanetto, Fabrizio; Groenberg, Gunnar; Hilgendorf, Constanze; Lee, Maurice D.; Matsson, Paer; Muncipinto, Giovanni; Pellisson, Melanie; Perry, Matthew W. D.; Svensson, Richard; Duvall, Jeremy R.; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2746-2754</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Profiling of eight stereoisomeric T. cruzi growth inhibitors revealed vastly different in vitro properties such as soly., lipophilicity, pKa, and cell permeability for two sets of four stereoisomers.  Using computational chem. and NMR spectroscopy, we identified the formation of an intramol. NH→NR3 hydrogen bond in the set of stereoisomers displaying lower soly., higher lipophilicity, and higher cell permeability.  The intramol. hydrogen bond resulted in a significant pKa difference that accounts for the other structure-property relationships.  Application of this knowledge could be of particular value to maintain the delicate balance of size, soly., and lipophilicity required for cell penetration and oral administration for chem. probes or therapeutics with properties at, or beyond, Lipinski's rule of 5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR_VAbJqqxirVg90H21EOLACvtfcHk0likbCp3HrerPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFKiurg%253D&md5=4227f825b82f5d17fab3f0de43806a5d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fjm500059t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500059t%26sid%3Dliteratum%253Aachs%26aulast%3DOver%26aufirst%3DB.%26aulast%3DMcCarren%26aufirst%3DP.%26aulast%3DArtursson%26aufirst%3DP.%26aulast%3DFoley%26aufirst%3DM.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DGr%25C3%25B6nberg%26aufirst%3DG.%26aulast%3DHilgendorf%26aufirst%3DC.%26aulast%3DLee%26aufirst%3DM.%2BD.%26aulast%3DMatsson%26aufirst%3DP.%26aulast%3DMuncipinto%26aufirst%3DG.%26aulast%3DPellisson%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DM.%2BW.%2BD.%26aulast%3DSvensson%26aufirst%3DR.%26aulast%3DDuvall%26aufirst%3DJ.%2BR.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DImpact%2520of%2520Stereospecific%2520Intramolecular%2520Hydrogen%2520Bonding%2520on%2520Cell%2520Permeability%2520and%2520Physicochemical%2520Properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D2746%26epage%3D2754%26doi%3D10.1021%2Fjm500059t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chirgadze, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanaraj, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyvonen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Innis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parisini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellegrini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayed, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sibanda, B. L.</span></span> <span> </span><span class="NLM_article-title">Protein-Protein Interactions in Receptor Activation and Intracellular Signalling</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>381</i></span>,  <span class="NLM_fpage">955</span>– <span class="NLM_lpage">959</span>, <span class="refDoi"> DOI: 10.1515/bc.2000.117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1515%2FBC.2000.117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=11076027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnvFSqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=381&publication_year=2000&pages=955-959&author=T.+L.+Blundellauthor=D.+F.+Burkeauthor=D.+Chirgadzeauthor=V.+Dhanarajauthor=M.+Hyvonenauthor=C.+A.+Innisauthor=E.+Parisiniauthor=L.+Pellegriniauthor=M.+Sayedauthor=B.+L.+Sibanda&title=Protein-Protein+Interactions+in+Receptor+Activation+and+Intracellular+Signalling&doi=10.1515%2Fbc.2000.117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-protein interactions in receptor activation and intracellular signalling</span></div><div class="casAuthors">Blundell, Tom L.; Burke, David F.; Chirgadze, Dimitri; Dhanaraj, Venugopal; Hyvonen, Marko; Innis, C. Axel; Parisini, Emilio; Pellegrini, Luca; Sayed, Muhammed; Sibanda, B. Lynn</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">381</span>
        (<span class="NLM_cas:issue">9/10</span>),
    <span class="NLM_cas:pages">955-959</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">A review, with 27 refs.  We review here signalling complexes that we have defined using X-ray anal. in our lab.  They include growth factors and their receptors: nerve growth factor (NGF) and its hetero-hexameric 7S NGF storage complex, hepatocyte growth factor/scatter factor (HGF/SF) NK1 dimers and fibroblast growth factor (FGF1) in complex with its receptor (FGFR2) ectodomain and heparin.  We also review our recent structural studies on intracellular signalling complexes, focusing on phosducin transducin Gβγ, CK2 protein kinase and its complexes, and the cyclin D-dependent kinase, Cdk6, bound to the cell cycle inhibitor p19INK4d.  Comparing the structures of these complexes with others we show that the surface area buried in signalling interactions does not always give a good indication of the strength of the interactions.  We show that conformational changes are often important in complexes with intermediate buried surface areas of 1500 to 2000 Å2, such as Cdk6 INK4 interactions.  Some interactions involve recognition of continuous epitopes, where there is no necessity for a tertiary structure and very often the binding conformation is induced during the process of interaction, for example phosducin binding to the βγ subunits (Gtβγ) of the heterotrimeric G protein transducin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOdDcfG3EA1rVg90H21EOLACvtfcHk0lhH9Y3nzhpK0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnvFSqsbY%253D&md5=6d210079a72e28d158cd032b3f62fc0b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1515%2FBC.2000.117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252FBC.2000.117%26sid%3Dliteratum%253Aachs%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DBurke%26aufirst%3DD.%2BF.%26aulast%3DChirgadze%26aufirst%3DD.%26aulast%3DDhanaraj%26aufirst%3DV.%26aulast%3DHyvonen%26aufirst%3DM.%26aulast%3DInnis%26aufirst%3DC.%2BA.%26aulast%3DParisini%26aufirst%3DE.%26aulast%3DPellegrini%26aufirst%3DL.%26aulast%3DSayed%26aufirst%3DM.%26aulast%3DSibanda%26aufirst%3DB.%2BL.%26atitle%3DProtein-Protein%2520Interactions%2520in%2520Receptor%2520Activation%2520and%2520Intracellular%2520Signalling%26jtitle%3DBiol.%2520Chem.%26date%3D2000%26volume%3D381%26spage%3D955%26epage%3D959%26doi%3D10.1515%2Fbc.2000.117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, P.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule Protein-Protein Interaction Inhibitors: Therapeutic Potential in Light of Molecular Size, Chemical Space, and Ligand Binding Efficiency Considerations</span>. <i>IUBMB Life</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">724</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1002/iub.383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fiub.383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=20979208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlGrsL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2010&pages=724-731&author=P.+Buchwald&title=Small-Molecule+Protein-Protein+Interaction+Inhibitors%3A+Therapeutic+Potential+in+Light+of+Molecular+Size%2C+Chemical+Space%2C+and+Ligand+Binding+Efficiency+Considerations&doi=10.1002%2Fiub.383"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule protein-protein interaction inhibitors: therapeutic potential in light of molecular size, chemical space, and ligand binding efficiency considerations</span></div><div class="casAuthors">Buchwald, Peter</div><div class="citationInfo"><span class="NLM_cas:title">IUBMB Life</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">724-731</span>CODEN:
                <span class="NLM_cas:coden">IULIF8</span>;
        ISSN:<span class="NLM_cas:issn">1521-6543</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Inc.</span>)
        </div><div class="casAbstract">A review.  As the ultimate function of proteins depends to a great extent on their binding partners, protein-protein interactions (PPIs) represent a treasure trove of possible new therapeutic targets.  Unfortunately, interfaces involved in PPIs are not well-suited for effective small mol. binding.  Nevertheless, successful examples of small-mol. PPI inhibitors (PPIIs) are beginning to accumulate, and the sheer no. of PPIs that form the human interactome implies that, despite the relative unsuitability of PPIs to serve as "druggable" targets, small-mol. PPIIs can still provide novel pharmacol. tools and new innovative drugs in at least some areas.  Here, after some illustrative examples, accumulating information on the binding efficiency, mol. size, and chem. space requirements will be briefly reviewed.  Therapeutic success can only be achieved if these considerations are incorporated into the search process and if careful medicinal chem. approaches are used to address the absorption, distribution, metab., and excretion requirements of larger mols. that are often needed for this target class due to the lower efficiency of binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPrt8JCmQ-mLVg90H21EOLACvtfcHk0lhH9Y3nzhpK0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlGrsL%252FM&md5=ae27f95b3ef88b20ad2973927eaf4c91</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fiub.383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fiub.383%26sid%3Dliteratum%253Aachs%26aulast%3DBuchwald%26aufirst%3DP.%26atitle%3DSmall-Molecule%2520Protein-Protein%2520Interaction%2520Inhibitors%253A%2520Therapeutic%2520Potential%2520in%2520Light%2520of%2520Molecular%2520Size%252C%2520Chemical%2520Space%252C%2520and%2520Ligand%2520Binding%2520Efficiency%2520Considerations%26jtitle%3DIUBMB%2520Life%26date%3D2010%26volume%3D62%26spage%3D724%26epage%3D731%26doi%3D10.1002%2Fiub.383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D.</span></span> <span> </span><span class="NLM_article-title">The Future of Peptide-Based Drugs</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">136</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1111/cbdd.12055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1111%2Fcbdd.12055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=23253135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlt1yktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2013&pages=136-147&author=D.+J.+Craikauthor=D.+P.+Fairlieauthor=S.+Lirasauthor=D.+Price&title=The+Future+of+Peptide-Based+Drugs&doi=10.1111%2Fcbdd.12055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The future of peptide-based drugs</span></div><div class="casAuthors">Craik, David J.; Fairlie, David P.; Liras, Spiros; Price, David</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">136-147</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The suite of currently used drugs can be divided into two categories - traditional "small mol." drugs with typical mol. wts. of <500 Da but with oral bioavailability, and much larger "biologics" typically >5000 Da that are not orally bioavailable and need to be delivered via injection.  Due to their small size, conventional small mol. drugs may suffer from reduced target selectivity that often ultimately manifests in human side-effects, whereas protein therapeutics tend to be exquisitely specific for their targets due to many more interactions with them, but this comes at a cost of low bioavailability, poor membrane permeability, and metabolic instability.  The time has now come to reinvestigate new drug leads that fit between these two mol. wt. extremes, with the goal of combining advantages of small mols. (cost, conformational restriction, membrane permeability, metabolic stability, oral bioavailability) with those of proteins (natural components, target specificity, high potency).  This article uses selected examples of peptides to highlight the importance of peptide drugs, some potential new opportunities for their exploitation, and some difficult challenges ahead in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlaEZbrPXLPbVg90H21EOLACvtfcHk0lhH9Y3nzhpK0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlt1yktA%253D%253D&md5=1e7e4b7e725541587fad855e6a28cadf</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12055%26sid%3Dliteratum%253Aachs%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DD.%26atitle%3DThe%2520Future%2520of%2520Peptide-Based%2520Drugs%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2013%26volume%3D81%26spage%3D136%26epage%3D147%26doi%3D10.1111%2Fcbdd.12055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span> <span> </span><span class="NLM_article-title">The Druggable Genome</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1038/nrd892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1038%2Fnrd892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=12209152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=727-730&author=A.+L.+Hopkinsauthor=C.+R.+Groom&title=The+Druggable+Genome&doi=10.1038%2Fnrd892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Opinion: The druggable genome</span></div><div class="casAuthors">Hopkins, Andrew L.; Groom, Colin R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">727-730</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  An assessment of the no. of mol. targets that represent an opportunity for therapeutic intervention is crucial to the development of post-genomic research strategies within the pharmaceutical industry.  Now that we know the size of the human genome, it is interesting to consider just how many mol. targets this opportunity represents.  We start from the position that we understand the properties that are required for a good drug, and therefore must be able to understand what makes a good drug target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Uvqi8O9wzrVg90H21EOLACvtfcHk0li8Y3kCsu8g-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmslamtrY%253D&md5=d6dc7c67822661fe03e920695def1329</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd892%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26atitle%3DThe%2520Druggable%2520Genome%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D727%26epage%3D730%26doi%3D10.1038%2Fnrd892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raj, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">Plucking the High Hanging Fruit: A Systematic Approach for Targeting Protein-Protein Interactions</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4051</span>– <span class="NLM_lpage">4057</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2012.11.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1016%2Fj.bmc.2012.11.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=23267671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvV2rsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=4051-4057&author=M.+Rajauthor=B.+N.+Bullockauthor=P.+S.+Arora&title=Plucking+the+High+Hanging+Fruit%3A+A+Systematic+Approach+for+Targeting+Protein-Protein+Interactions&doi=10.1016%2Fj.bmc.2012.11.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Plucking the high hanging fruit: A systematic approach for targeting protein-protein interactions</span></div><div class="casAuthors">Raj, Monika; Bullock, Brooke N.; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4051-4057</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Development of specific ligands for protein targets that help decode the complexities of protein-protein interaction networks is a key goal for the field of chem. biol.  Despite the emergence of powerful in silico and exptl. high-throughput screening strategies, the discovery of synthetic ligands that selectively modulate protein-protein interactions remains a challenge for bioorg. and medicinal chemists.  This Perspective discusses emerging principles for the rational design of PPI inhibitors.  Fundamentally, the approach seeks to adapt nature's protein recognition principles for the design of suitable secondary structure mimetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorpTCgOTASB7Vg90H21EOLACvtfcHk0li8Y3kCsu8g-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvV2rsrbN&md5=970cdca83336f20387e060810f7f261c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2012.11.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2012.11.023%26sid%3Dliteratum%253Aachs%26aulast%3DRaj%26aufirst%3DM.%26aulast%3DBullock%26aufirst%3DB.%2BN.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DPlucking%2520the%2520High%2520Hanging%2520Fruit%253A%2520A%2520Systematic%2520Approach%2520for%2520Targeting%2520Protein-Protein%2520Interactions%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D4051%26epage%3D4057%26doi%3D10.1016%2Fj.bmc.2012.11.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Northfield, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colless, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaousis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamernig, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, R.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Rational Design and Synthesis of an Orally Bioavailable Peptide Guided by NMR Amide Temperature Coefficients</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">17504</span>– <span class="NLM_lpage">17509</span>, <span class="refDoi"> DOI: 10.1073/pnas.1417611111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1073%2Fpnas.1417611111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=25416591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKmu7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=17504-17509&author=C.+K.+Wangauthor=S.+E.+Northfieldauthor=B.+Collessauthor=S.+Chaousisauthor=I.+Hamernigauthor=R.-J.+Lohmanauthor=D.+S.+Nielsenauthor=C.+I.+Schroederauthor=S.+Lirasauthor=D.+A.+Priceauthor=D.+P.+Fairlieauthor=D.+J.+Craik&title=Rational+Design+and+Synthesis+of+an+Orally+Bioavailable+Peptide+Guided+by+NMR+Amide+Temperature+Coefficients&doi=10.1073%2Fpnas.1417611111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Rational design and synthesis of an orally bioavailable peptide guided by NMR amide temperature coefficients</span></div><div class="casAuthors">Wang, Conan K.; Northfield, Susan E.; Colless, Barbara; Chaousis, Stephanie; Hamernig, Ingrid; Lohman, Rink-Jan; Nielsen, Daniel S.; Schroeder, Christina I.; Liras, Spiros; Price, David A.; Fairlie, David P.; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">17504-17509</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Enhancing the oral bioavailability of peptide drug leads is a major challenge in drug design.  As such, methods to address this challenge are highly sought after by the pharmaceutical industry.  Here, we propose a strategy to identify appropriate amides for N-methylation using temp. coeffs. measured by NMR to identify exposed amides in cyclic peptides.  N-methylation effectively caps these amides, modifying the overall solvation properties of the peptides and making them more membrane permeable.  The approach for identifying sites for N-methylation is a rapid alternative to the elucidation of 3D structures of peptide drug leads, which has been a commonly used structure-guided approach in the past.  Five leucine-rich peptide scaffolds are reported with selectively designed N-methylated derivs.  In vitro membrane permeability was assessed by parallel artificial membrane permeability assay and Caco-2 assay.  The most promising N-methylated peptide was then tested in vivo.  Here we report a novel peptide (15), which displayed an oral bioavailability of 33% in a rat model, thus validating the design approach.  We show that this approach can also be used to explain the notable increase in oral bioavailability of a somatostatin analog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ1pwUl6qUw7Vg90H21EOLACvtfcHk0li8Y3kCsu8g-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKmu7vE&md5=02803e516bf514e3d941769b7bbf3607</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1417611111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1417611111%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%2BK.%26aulast%3DNorthfield%26aufirst%3DS.%2BE.%26aulast%3DColless%26aufirst%3DB.%26aulast%3DChaousis%26aufirst%3DS.%26aulast%3DHamernig%26aufirst%3DI.%26aulast%3DLohman%26aufirst%3DR.-J.%26aulast%3DNielsen%26aufirst%3DD.%2BS.%26aulast%3DSchroeder%26aufirst%3DC.%2BI.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DRational%2520Design%2520and%2520Synthesis%2520of%2520an%2520Orally%2520Bioavailable%2520Peptide%2520Guided%2520by%2520NMR%2520Amide%2520Temperature%2520Coefficients%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D17504%26epage%3D17509%26doi%3D10.1073%2Fpnas.1417611111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cardote, T. A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Cyclic and Macrocyclic Peptides as Chemical Tools to Recognise Protein Surfaces and Probe Protein-Protein Interactions</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">787</span>– <span class="NLM_lpage">794</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fcmdc.201500450" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=26563831" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVajsL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=787-794&author=T.+A.+F.+Cardoteauthor=A.+Ciulli&title=Cyclic+and+Macrocyclic+Peptides+as+Chemical+Tools+to+Recognise+Protein+Surfaces+and+Probe+Protein-Protein+Interactions&doi=10.1002%2Fcmdc.201500450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic and Macrocyclic Peptides as Chemical Tools To Recognise Protein Surfaces and Probe Protein-Protein Interactions</span></div><div class="casAuthors">Cardote, Teresa A. F.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">787-794</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Targeting protein surfaces and protein-protein interactions (PPIs) with small mols. is a frontier goal of chem. biol. and provides attractive therapeutic opportunities in drug discovery.  The mol. properties of protein surfaces, including their shallow features and lack of deep binding pockets, pose significant challenges, and as a result have proved difficult to target.  Peptides are ideal candidates for this mission due to their ability to closely mimic many structural features of protein interfaces.  However, their inherently low intracellular stability and permeability and high in vivo clearance have thus far limited their biol. applications.  One way to improve these properties is to constrain the secondary structure of linear peptides by cyclization.  Herein we review various classes of cyclic and macrocyclic peptides as chem. probes of protein surfaces and modulators of PPIs.  The growing interest in this area and recent advances provide evidence of the potential of developing peptide-like mols. that specifically target these interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRiqVM7-eZhbVg90H21EOLACvtfcHk0lhITfmMqE4Prg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVajsL%252FE&md5=a67ba0ee61764a882d5a37b474570343</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500450%26sid%3Dliteratum%253Aachs%26aulast%3DCardote%26aufirst%3DT.%2BA.%2BF.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DCyclic%2520and%2520Macrocyclic%2520Peptides%2520as%2520Chemical%2520Tools%2520to%2520Recognise%2520Protein%2520Surfaces%2520and%2520Probe%2520Protein-Protein%2520Interactions%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D787%26epage%3D794%26doi%3D10.1002%2Fcmdc.201500450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClendon, C. L.</span></span> <span> </span><span class="NLM_article-title">Reaching for High-Hanging Fruit in Drug Discovery at Protein-Protein Interfaces</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>450</i></span>,  <span class="NLM_fpage">1001</span>– <span class="NLM_lpage">1009</span>, <span class="refDoi"> DOI: 10.1038/nature06526</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1038%2Fnature06526" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=18075579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2007&pages=1001-1009&author=J.+A.+Wellsauthor=C.+L.+McClendon&title=Reaching+for+High-Hanging+Fruit+in+Drug+Discovery+at+Protein-Protein+Interfaces&doi=10.1038%2Fnature06526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Reaching for high-hanging fruit in drug discovery at protein-protein interfaces</span></div><div class="casAuthors">Wells, James A.; McClendon, Christopher L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">450</span>
        (<span class="NLM_cas:issue">7172</span>),
    <span class="NLM_cas:pages">1001-1009</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Targeting the interfaces between proteins has huge therapeutic potential, but discovering small-mol. drugs that disrupt protein-protein interactions is an enormous challenge.  Several recent success stories, however, indicate that protein-protein interfaces might be more tractable than has been thought.  These studies discovered small mols. that bind with drug-like potencies to 'hotspots' on the contact surfaces involved in protein-protein interactions.  Remarkably, these small mols. bind deeper within the contact surface of the target protein, and bind with much higher efficiencies, than do the contact atoms of the natural protein partner.  Some of these small mols. are now making their way through clin. trials, so this high-hanging fruit might not be far out of reach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-nGR3BqqkrVg90H21EOLACvtfcHk0lhITfmMqE4Prg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVaqtr7F&md5=e7ed69fd8c362a71c4b99f029c16a6fc</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnature06526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06526%26sid%3Dliteratum%253Aachs%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DMcClendon%26aufirst%3DC.%2BL.%26atitle%3DReaching%2520for%2520High-Hanging%2520Fruit%2520in%2520Drug%2520Discovery%2520at%2520Protein-Protein%2520Interfaces%26jtitle%3DNature%26date%3D2007%26volume%3D450%26spage%3D1001%26epage%3D1009%26doi%3D10.1038%2Fnature06526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.</span></span> <span> </span><span class="NLM_article-title">Selective, Cell-Permeable Nonphosphorylated Bicyclic Peptidyl Inhibitor Against Peptidyl-Prolyl Isomerase Pin1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6306</span>– <span class="NLM_lpage">6312</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00411</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00411" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2lt77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6306-6312&author=B.+Jiangauthor=D.+Pei&title=Selective%2C+Cell-Permeable+Nonphosphorylated+Bicyclic+Peptidyl+Inhibitor+Against+Peptidyl-Prolyl+Isomerase+Pin1&doi=10.1021%2Facs.jmedchem.5b00411"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A Selective, Cell-Permeable Nonphosphorylated Bicyclic Peptidyl Inhibitor against Peptidyl-Prolyl Isomerase Pin1</span></div><div class="casAuthors">Jiang, Bisheng; Pei, Dehua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6306-6312</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pin1 regulates the levels and functions of phosphoproteins by catalyzing phosphorylation-dependent cis/trans isomerization of peptidyl-prolyl bonds.  Previous Pin1 inhibitors contained phosphoamino acids, which are metabolically unstable and have poor membrane permeability.  In this work, the authors report a cell-permeable and metabolically stable nonphosphorylated bicyclic peptide as a potent and selective Pin1 inhibitor, which inhibited the intracellular Pin1 activity in cultured mammalian cells but had little effect on other isomerases such as Pin4, FKBP12, or cyclophilin A.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrK8No2ty2NM7Vg90H21EOLACvtfcHk0lhITfmMqE4Prg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2lt77P&md5=735a73f73d5aa05ab7f046e7e693b135</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00411%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DPei%26aufirst%3DD.%26atitle%3DSelective%252C%2520Cell-Permeable%2520Nonphosphorylated%2520Bicyclic%2520Peptidyl%2520Inhibitor%2520Against%2520Peptidyl-Prolyl%2520Isomerase%2520Pin1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6306%26epage%3D6312%26doi%3D10.1021%2Facs.jmedchem.5b00411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwyer, J. J.</span></span> <span> </span><span class="NLM_article-title">Lead Discovery and Optimization Strategies for Peptide Macrocycles</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">479</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2014.07.083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1016%2Fj.ejmech.2014.07.083" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=25109255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1GktrrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2015&pages=471-479&author=A.+Bhatauthor=L.+R.+Robertsauthor=J.+J.+Dwyer&title=Lead+Discovery+and+Optimization+Strategies+for+Peptide+Macrocycles&doi=10.1016%2Fj.ejmech.2014.07.083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Lead discovery and optimization strategies for peptide macrocycles</span></div><div class="casAuthors">Bhat, Abhijit; Roberts, Lee R.; Dwyer, John J.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">471-479</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Peptide macrocycles represent a chem. space where the best of biol. tools can synergize with the best of chem. approaches in the quest for leads against undruggable targets.  Peptide macrocycles offer some key advantages in both lead discovery and lead optimization phases of drug discovery when compared to natural product and synthetic macrocycles.  In addn., they are uniquely positioned to capitalize on the therapeutic potential of peptides because cyclization can help drive selectivity, potency and overcome the common limitations of metabolic instability of peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrYEBhURBm07Vg90H21EOLACvtfcHk0lgzzgXjB2W0cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1GktrrM&md5=2e12788610c87e9c774dcf3804855cf4</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.07.083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.07.083%26sid%3Dliteratum%253Aachs%26aulast%3DBhat%26aufirst%3DA.%26aulast%3DRoberts%26aufirst%3DL.%2BR.%26aulast%3DDwyer%26aufirst%3DJ.%2BJ.%26atitle%3DLead%2520Discovery%2520and%2520Optimization%2520Strategies%2520for%2520Peptide%2520Macrocycles%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D94%26spage%3D471%26epage%3D479%26doi%3D10.1016%2Fj.ejmech.2014.07.083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobritzsch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">How Beyond Rule of 5 Drugs and Clinical Candidates Bind to Their Targets</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2312</span>– <span class="NLM_lpage">2327</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01286</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01286" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2312-2327&author=B.+C.+Doakauthor=J.+Zhengauthor=D.+Dobritzschauthor=J.+Kihlberg&title=How+Beyond+Rule+of+5+Drugs+and+Clinical+Candidates+Bind+to+Their+Targets&doi=10.1021%2Facs.jmedchem.5b01286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">How Beyond Rule of 5 Drugs and Clinical Candidates Bind to Their Targets</span></div><div class="casAuthors">Doak, Bradley C.; Zheng, Jie; Dobritzsch, Doreen; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2312-2327</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To improve discovery of drugs for difficult targets, the opportunities of chem. space beyond the rule of 5 (bRo5) were examd. by retrospective anal. of a comprehensive set of structures for complexes between drugs and clin. candidates and their targets.  The anal. illustrates the potential of compds. far beyond rule of 5 space to modulate novel and difficult target classes that have large, flat, and groove-shaped binding sites.  However, ligand efficiencies are significantly reduced for flat- and groove-shape binding sites, suggesting that adjustments of how to use such metrics are required.  Ligands bRo5 appear to benefit from an appropriate balance between rigidity and flexibility to bind with sufficient affinity to their targets, with macrocycles and nonmacrocycles being found to have similar flexibility.  However, macrocycles were more disk- and spherelike, which may contribute to their superior binding to flat sites, while rigidification of nonmacrocycles lead to rodlike ligands that bind well to groove-shaped binding sites.  These insights should contribute to altering perceptions of what targets are considered "druggable" and provide support for drug design in beyond rule of 5 space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZD-h9VodiPLVg90H21EOLACvtfcHk0lgzzgXjB2W0cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F&md5=a417b605d1a3ffbc7ee84706401f01f4</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01286%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DDobritzsch%26aufirst%3DD.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DHow%2520Beyond%2520Rule%2520of%25205%2520Drugs%2520and%2520Clinical%2520Candidates%2520Bind%2520to%2520Their%2520Targets%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2312%26epage%3D2327%26doi%3D10.1021%2Facs.jmedchem.5b01286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, M. L.</span></span> <span> </span><span class="NLM_article-title">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1961</span>– <span class="NLM_lpage">2004</span>, <span class="refDoi"> DOI: 10.1021/jm1012374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1961-2004&author=E.+Marsaultauthor=M.+L.+Peterson&title=Macrocycles+Are+Great+Cycles%3A+Applications%2C+Opportunities%2C+and+Challenges+of+Synthetic+Macrocycles+in+Drug+Discovery&doi=10.1021%2Fjm1012374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span></div><div class="casAuthors">Marsault, Eric; Peterson, Mark L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1961-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles occupy a unique segment of chem. space.  In the past decade, their chem. diversity expanded significantly, supported by advances in bioinformatics and synthetic methodol.  As a consequence, this structural type has now been successfully tested on most biol. target classes.  The goal of this article is to put into perspective the current applications, opportunities, and challenges assocd. with synthetic macrocycles in drug discovery.  Accordingly, the first part of this article is dedicated to the drug discovery aspects of macrocycles and highlights salient features of their medicinal chem.  This section is organized by target class, a choice aimed at providing the reader an appreciation of the structural diversity generated for each class.  To give the reader an appreciation of the tools available to construct macrocyclic scaffolds, the site and method of the pivotal macrocyclization step are indicated in the figures.  Readers are referred to the source articles for further details.  In the second part, the technologies and synthetic approaches that already have demonstrated utility or possess a high potential for macrocycle-based drug discovery are discussed.  Finally, a perspective on the future of synthetic macrocycles in medicinal chem. is offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygQLE6JraHrVg90H21EOLACvtfcHk0lgzzgXjB2W0cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D&md5=97c1120896e90a85790a9ac1274259f3</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm1012374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012374%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26atitle%3DMacrocycles%2520Are%2520Great%2520Cycles%253A%2520Applications%252C%2520Opportunities%252C%2520and%2520Challenges%2520of%2520Synthetic%2520Macrocycles%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1961%26epage%3D2004%26doi%3D10.1021%2Fjm1012374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Northfield, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swedberg, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colless, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaousis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Exploring Experimental and Computational Markers of Cyclic Peptides: Charting Islands of Permeability</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">213</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2015.04.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1016%2Fj.ejmech.2015.04.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=25974856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotlKhu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2015&pages=202-213&author=C.+K.+Wangauthor=S.+E.+Northfieldauthor=J.+E.+Swedbergauthor=B.+Collessauthor=S.+Chaousisauthor=D.+A.+Priceauthor=S.+Lirasauthor=D.+J.+Craik&title=Exploring+Experimental+and+Computational+Markers+of+Cyclic+Peptides%3A+Charting+Islands+of+Permeability&doi=10.1016%2Fj.ejmech.2015.04.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring experimental and computational markers of cyclic peptides: Charting islands of permeability</span></div><div class="casAuthors">Wang, Conan K.; Northfield, Susan E.; Swedberg, Joakim E.; Colless, Barbara; Chaousis, Stephanie; Price, David A.; Liras, Spiros; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">202-213</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">An increasing no. of macrocyclic peptides that cross biol. membranes are being reported, suggesting that it might be possible to develop peptides into orally bioavailable therapeutics; however, current understanding of what makes macrocyclic peptides cell permeable is still limited.  Here, we synthesized 62 cyclic hexapeptides and characterized their permeability using in vitro assays commonly used to predict in vivo absorption rates, i.e. the Caco-2 and PAMPA assays.  We correlated permeability with exptl. measured parameters of peptide conformation obtained using rapid methods based on chromatog. and NMR spectroscopy.  Based on these correlations, we propose a model describing the interplay between peptide permeability, lipophilicity and hydrogen bonding potential.  Specifically, peptides with very high permeability have high lipophilicity and few solvent hydrogen bond interactions, whereas peptides with very low permeability have low lipophilicity or many solvent interactions.  Our model is supported by mol. dynamics simulations of the cyclic peptides calcd. in explicit solvent, providing a structural basis for the obsd. correlations.  This prospective exploration into biomarkers of peptide permeability has the potential to unlock wider opportunities for development of peptides into drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYs3AwkeaC6rVg90H21EOLACvtfcHk0ljCidKWWpF8cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotlKhu70%253D&md5=300b043854615dc799a79646931cb87f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2015.04.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2015.04.049%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%2BK.%26aulast%3DNorthfield%26aufirst%3DS.%2BE.%26aulast%3DSwedberg%26aufirst%3DJ.%2BE.%26aulast%3DColless%26aufirst%3DB.%26aulast%3DChaousis%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DExploring%2520Experimental%2520and%2520Computational%2520Markers%2520of%2520Cyclic%2520Peptides%253A%2520Charting%2520Islands%2520of%2520Permeability%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D97%26spage%3D202%26epage%3D213%26doi%3D10.1016%2Fj.ejmech.2015.04.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thansandote, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dexter, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span> <span> </span><span class="NLM_article-title">Improving the Passive Permeability of Macrocyclic Peptides: Balancing Permeability with Other Physicochemical Properties</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">322</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2014.11.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1016%2Fj.bmc.2014.11.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=25533323" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVWktL3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=322-327&author=P.+Thansandoteauthor=R.+M.+Harrisauthor=H.+L.+Dexterauthor=G.+L.+Simpsonauthor=S.+Palauthor=R.+J.+Uptonauthor=K.+Valko&title=Improving+the+Passive+Permeability+of+Macrocyclic+Peptides%3A+Balancing+Permeability+with+Other+Physicochemical+Properties&doi=10.1016%2Fj.bmc.2014.11.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Improving the passive permeability of macrocyclic peptides: Balancing permeability with other physicochemical properties</span></div><div class="casAuthors">Thansandote, Praew; Harris, Robert M.; Dexter, Hannah L.; Simpson, Graham L.; Pal, Sandeep; Upton, Richard J.; Valko, Klara</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">322-327</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A no. of methods to improve the passive permeability of a set of cyclic peptides have been investigated using 6- and 7-mer macrocyclic templates.  In many cases the peptides were designed by mol. dynamics calcns. to evaluate the methods.  The aim of this study was not only to improve passive permeability, but also to balance permeability with other physicochem. properties with the goal of understanding and applying the knowledge to develop active cyclic peptides into drug candidates.  Evaluation of the methods herein suggest that increasing passive permeability often occurs at the expense of soly. and lipophilicity.  Computational methods can be useful when attempting to predict and design features to balance these properties, though limitations were obsd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqx--UWVWxM07Vg90H21EOLACvtfcHk0ljCidKWWpF8cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVWktL3O&md5=a9eb20c8e03b890c1d4616a8bdebf26c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2014.11.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2014.11.034%26sid%3Dliteratum%253Aachs%26aulast%3DThansandote%26aufirst%3DP.%26aulast%3DHarris%26aufirst%3DR.%2BM.%26aulast%3DDexter%26aufirst%3DH.%2BL.%26aulast%3DSimpson%26aufirst%3DG.%2BL.%26aulast%3DPal%26aufirst%3DS.%26aulast%3DUpton%26aufirst%3DR.%2BJ.%26aulast%3DValko%26aufirst%3DK.%26atitle%3DImproving%2520the%2520Passive%2520Permeability%2520of%2520Macrocyclic%2520Peptides%253A%2520Balancing%2520Permeability%2520with%2520Other%2520Physicochemical%2520Properties%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D322%26epage%3D327%26doi%3D10.1016%2Fj.bmc.2014.11.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradov, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Peptides as Drug Candidates: Recent Progress and Remaining Challenges</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">4167</span>– <span class="NLM_lpage">4181</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b13178</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b13178" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtFCqtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=4167-4181&author=A.+A.+Vinogradovauthor=Y.+Yinauthor=H.+Suga&title=Macrocyclic+Peptides+as+Drug+Candidates%3A+Recent+Progress+and+Remaining+Challenges&doi=10.1021%2Fjacs.8b13178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Peptides as Drug Candidates: Recent Progress and Remaining Challenges</span></div><div class="casAuthors">Vinogradov, Alexander A.; Yin, Yizhen; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4167-4181</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Peptides as a therapeutic modality attract much attention due to their synthetic accessibility, high degree of specific binding, and the ability to target protein surfaces traditionally considered "undruggable".  Unfortunately, at the same time, other pharmacol. properties of a generic peptide, such as metabolic stability and cell permeability, are quite poor, which limits the success of de novo discovered biol. active peptides as drug candidates.  Here, we review how macrocyclization as well as the incorporation of nonproteogenic amino acids and various conjugation strategies may be utilized to improve on these characteristics to create better drug candidates.  We analyze recent progress and remaining challenges in improving individual pharmacol. properties of bioactive peptides, and offer our opinion on interfacing these, often conflicting, considerations, to create balanced drug candidates as a potential way to make further progress in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXRPkCAGk8-rVg90H21EOLACvtfcHk0ljCidKWWpF8cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtFCqtb4%253D&md5=59b6197874d4709a74089367a09119cc</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b13178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b13178%26sid%3Dliteratum%253Aachs%26aulast%3DVinogradov%26aufirst%3DA.%2BA.%26aulast%3DYin%26aufirst%3DY.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DMacrocyclic%2520Peptides%2520as%2520Drug%2520Candidates%253A%2520Recent%2520Progress%2520and%2520Remaining%2520Challenges%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D4167%26epage%3D4181%26doi%3D10.1021%2Fjacs.8b13178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Driggers, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrett, N. K.</span></span> <span> </span><span class="NLM_article-title">The Exploration of Macrocycles for Drug Discovery - An Underexploited Structural Class</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">608</span>– <span class="NLM_lpage">624</span>, <span class="refDoi"> DOI: 10.1038/nrd2590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1038%2Fnrd2590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=18591981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=608-624&author=E.+M.+Driggersauthor=S.+P.+Haleauthor=J.+Leeauthor=N.+K.+Terrett&title=The+Exploration+of+Macrocycles+for+Drug+Discovery+-+An+Underexploited+Structural+Class&doi=10.1038%2Fnrd2590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The exploration of macrocycles for drug discovery - an underexploited structural class</span></div><div class="casAuthors">Driggers, Edward M.; Hale, Stephen P.; Lee, Jinbo; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">608-624</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Natural products comprised of a macrocycle ring structure have proven their therapeutic applications as antibiotics, immunosuppressants as well as anticancer agents.  Despite this, macrocyclic compds. remain under-explored.  Terrett and colleagues review the properties and features of current macrocycle drugs, emphasizing the vast potential of synthetic macrocycles in drug discovery.  Macrocyclic natural products have evolved to fulfil numerous biochem. functions, and their profound pharmacol. properties have led to their development as drugs.  A macrocycle provides diverse functionality and stereochem. complexity in a conformationally pre-organized ring structure.  This can result in high affinity and selectivity for protein targets, while preserving sufficient bioavailability to reach intracellular locations.  Despite these valuable characteristics, and the proven success of more than 100 marketed macrocycle drugs derived from natural products, this structural class has been poorly explored within drug discovery.  This is in part due to concerns about synthetic intractability and non-drug-like properties.  This Review describes the growing body of data in favor of macrocyclic therapeutics, and demonstrates that this class of compds. can be both fully drug-like in its properties and readily prepd. owing to recent advances in synthetic medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDtW8LpE4ru7Vg90H21EOLACvtfcHk0ljCidKWWpF8cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D&md5=16e091818053e5b163b81078a8d03094</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnrd2590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2590%26sid%3Dliteratum%253Aachs%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520Exploration%2520of%2520Macrocycles%2520for%2520Drug%2520Discovery%2520-%2520An%2520Underexploited%2520Structural%2520Class%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D608%26epage%3D624%26doi%3D10.1038%2Fnrd2590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chennamadhavuni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">How Proteins Bind Macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-731&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+Proteins+Bind+Macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0ljP0DMzJQX69Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520Proteins%2520Bind%2520Macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D731%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biron, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovadia, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langenegger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brueggen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoyer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jelinek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span> <span> </span><span class="NLM_article-title">Improving Oral Bioavailability of Peptides by Multiple N-Methylation: Somatostatin Analogues</span>. <i>Angew. Chem. Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">2595</span>– <span class="NLM_lpage">2599</span>, <span class="refDoi"> DOI: 10.1002/anie.200705797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fanie.200705797" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFartLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=2595-2599&author=E.+Bironauthor=J.+Chatterjeeauthor=O.+Ovadiaauthor=D.+Langeneggerauthor=J.+Brueggenauthor=D.+Hoyerauthor=H.+A.+Schmidauthor=R.+Jelinekauthor=C.+Gilonauthor=A.+Hoffmanauthor=H.+Kessler&title=Improving+Oral+Bioavailability+of+Peptides+by+Multiple+N-Methylation%3A+Somatostatin+Analogues&doi=10.1002%2Fanie.200705797"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Improving oral bioavailability of peptides by multiple N-methylation: somatostatin analogues</span></div><div class="casAuthors">Biron, Eric; Chatterjee, Jayanta; Ovadia, Oded; Langenegger, Daniel; Brueggen, Joseph; Hoyer, Daniel; Schmid, Herbert A.; Jelnick, Raz; Gilon, Chaim; Hoffman, Amnon; Kessler, Horst</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2595-2599</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A complete library of the N-methylated somatostatin cyclopeptidic analog Veber-Hirschmann peptide cyclo(-PFwKTF-) has been prepd. with the aim of improving its bioavailability.  Several analogs from the library were found to bind to the somatostatin receptor in the nanomolar range and one of them shows a significant oral bioavailability of 10%.  Conformational anal. shows that N-methylation is allowed at specific positions without affecting the bioactive conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo64OMbjkAtErVg90H21EOLACvtfcHk0ljP0DMzJQX69Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFartLw%253D&md5=42c312786da82941dde6671b1130b61b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fanie.200705797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200705797%26sid%3Dliteratum%253Aachs%26aulast%3DBiron%26aufirst%3DE.%26aulast%3DChatterjee%26aufirst%3DJ.%26aulast%3DOvadia%26aufirst%3DO.%26aulast%3DLangenegger%26aufirst%3DD.%26aulast%3DBrueggen%26aufirst%3DJ.%26aulast%3DHoyer%26aufirst%3DD.%26aulast%3DSchmid%26aufirst%3DH.%2BA.%26aulast%3DJelinek%26aufirst%3DR.%26aulast%3DGilon%26aufirst%3DC.%26aulast%3DHoffman%26aufirst%3DA.%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DImproving%2520Oral%2520Bioavailability%2520of%2520Peptides%2520by%2520Multiple%2520N-Methylation%253A%2520Somatostatin%2520Analogues%26jtitle%3DAngew.%2520Chem.%2520Int.%2520Ed.%26date%3D2008%26volume%3D47%26spage%3D2595%26epage%3D2599%26doi%3D10.1002%2Fanie.200705797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beck, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiran, M. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovadia, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span> <span> </span><span class="NLM_article-title">Intestinal Permeability of Cyclic Peptides: Common Key Backbone Motifs Identified</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">12125</span>– <span class="NLM_lpage">12133</span>, <span class="refDoi"> DOI: 10.1021/ja303200d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja303200d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpt1Cisbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=12125-12133&author=J.+G.+Beckauthor=J.+Chatterjeeauthor=B.+Lauferauthor=M.+U.+Kiranauthor=A.+O.+Frankauthor=S.+Neubauerauthor=O.+Ovadiaauthor=S.+Greenbergauthor=C.+Gilonauthor=A.+Hoffmanauthor=H.+Kessler&title=Intestinal+Permeability+of+Cyclic+Peptides%3A+Common+Key+Backbone+Motifs+Identified&doi=10.1021%2Fja303200d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Intestinal Permeability of Cyclic Peptides: Common Key Backbone Motifs Identified</span></div><div class="casAuthors">Beck, Johannes G.; Chatterjee, Jayanta; Laufer, Burkhardt; Kiran, Marelli Udaya; Frank, Andreas O.; Neubauer, Stefanie; Ovadia, Oded; Greenberg, Sarit; Gilon, Chaim; Hoffman, Amnon; Kessler, Horst</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">12125-12133</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Insufficient oral bioavailability is considered as a key limitation for the widespread development of peptides as therapeutics.  While the oral bioavailability of small org. compds. is often estd. from simple rules, similar rules do not apply to peptides, and even the high oral bioavailability that is described for a small no. of peptides is not well understood.  Here we present two highly Caco-2 permeable template structures based on a library of 54 cyclo(-d-Ala-Ala5-) peptides with different N-methylation patterns.  The first all-trans template structure possesses two β-turns of type II along Ala6-d-Ala1 and Ala3-Ala4 and is only found for one peptide with two N-Me groups at d-Ala1 and Ala6 [NMe(1,6)].  The second single-cis template possesses a characteristic cis peptide bond preceding Ala5, which results in type VI β-turn geometry along Ala4-Ala5.  Although the second template structure is found in seven peptides carrying N-Me groups on Ala5, high Caco-2 permeability is only found for a subgroup of two of them [NMe(1,5) and NMe(1,2,4,5)], suggesting that N-methylation of d-Ala1 is a prerequisite for high permeability of the second template structure.  The structural similarity of the second template structure with the orally bioavailable somatostatin analog cyclo(-Pro-Phe-NMe-d-Trp-NMe-Lys-Thr-NMe-Phe-), and the striking resemblance with both β-turns of the orally bioavailable peptide cyclosporine A, suggests that the introduction of bioactive sequences on the highly Caco-2 permeable templates may result in potent orally bioavailable drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4AC8OVfXJMbVg90H21EOLACvtfcHk0ljP0DMzJQX69Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpt1Cisbw%253D&md5=ecf4dc5aa62a8ebb7c5cc99e8009f528</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fja303200d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja303200d%26sid%3Dliteratum%253Aachs%26aulast%3DBeck%26aufirst%3DJ.%2BG.%26aulast%3DChatterjee%26aufirst%3DJ.%26aulast%3DLaufer%26aufirst%3DB.%26aulast%3DKiran%26aufirst%3DM.%2BU.%26aulast%3DFrank%26aufirst%3DA.%2BO.%26aulast%3DNeubauer%26aufirst%3DS.%26aulast%3DOvadia%26aufirst%3DO.%26aulast%3DGreenberg%26aufirst%3DS.%26aulast%3DGilon%26aufirst%3DC.%26aulast%3DHoffman%26aufirst%3DA.%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DIntestinal%2520Permeability%2520of%2520Cyclic%2520Peptides%253A%2520Common%2520Key%2520Backbone%2520Motifs%2520Identified%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D12125%26epage%3D12133%26doi%3D10.1021%2Fja303200d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, R.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diness, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Total Synthesis, Structure, and Oral Absorption of a Thiazole Cyclic Peptide, Sanguinamide A</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">5720</span>– <span class="NLM_lpage">5723</span>, <span class="refDoi"> DOI: 10.1021/ol3027347</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol3027347" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1WqsrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2012&pages=5720-5723&author=D.+S.+Nielsenauthor=H.+N.+Hoangauthor=R.-J.+Lohmanauthor=F.+Dinessauthor=D.+P.+Fairlie&title=Total+Synthesis%2C+Structure%2C+and+Oral+Absorption+of+a+Thiazole+Cyclic+Peptide%2C+Sanguinamide+A&doi=10.1021%2Fol3027347"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Total synthesis, structure, and oral absorption of a thiazole cyclic peptide, sanguinamide A</span></div><div class="casAuthors">Nielsen, Daniel S.; Hoang, Huy N.; Lohman, Rink-Jan; Diness, Frederik; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5720-5723</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The first total synthesis and three-dimensional soln. structure are reported for sanguinamide A, a thiazole-contg. cyclic peptide from the sea slug H. sanguineus.  Soln. phase fragment synthesis, solid phase fragment assembly, and soln. macrocyclization were combined to give (I) in 10% yield.  Spectral properties were identical for the natural product, requiring revision of its structure from cis- to trans- amide bond.  Intramol. transannular hydrogen bonds help to bury polar atoms, which enables oral absorption from the gut.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojYQbE5PbtKbVg90H21EOLACvtfcHk0liYpqWlHTC_oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1WqsrfP&md5=be27e8d9c27fbbe58bbc263be17008f7</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fol3027347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol3027347%26sid%3Dliteratum%253Aachs%26aulast%3DNielsen%26aufirst%3DD.%2BS.%26aulast%3DHoang%26aufirst%3DH.%2BN.%26aulast%3DLohman%26aufirst%3DR.-J.%26aulast%3DDiness%26aufirst%3DF.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DTotal%2520Synthesis%252C%2520Structure%252C%2520and%2520Oral%2520Absorption%2520of%2520a%2520Thiazole%2520Cyclic%2520Peptide%252C%2520Sanguinamide%2520A%26jtitle%3DOrg.%2520Lett.%26date%3D2012%26volume%3D14%26spage%3D5720%26epage%3D5723%26doi%3D10.1021%2Fol3027347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conibear, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaousis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johan Rosengren, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, C. I.</span></span> <span> </span><span class="NLM_article-title">Approaches to the Stabilization of Bioactive Epitopes by Grafting and Peptide Cyclization</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1002/bip.22767</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fbip.22767" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=26566734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVSgtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2016&pages=89-100&author=A.+C.+Conibearauthor=S.+Chaousisauthor=T.+Durekauthor=K.+Johan+Rosengrenauthor=D.+J.+Craikauthor=C.+I.+Schroeder&title=Approaches+to+the+Stabilization+of+Bioactive+Epitopes+by+Grafting+and+Peptide+Cyclization&doi=10.1002%2Fbip.22767"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Approaches to the stabilization of bioactive epitopes by grafting and peptide cyclization</span></div><div class="casAuthors">Conibear, Anne C.; Chaousis, Stephanie; Durek, Thomas; Johan Rosengren, K.; Craik, David J.; Schroeder, Christina I.</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">89-100</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Peptides are attracting increasing interest from the pharmaceutical industry because of their specificity and ability to address novel targets, including protein-protein interactions.  However, typically they require stabilization for therapeutic applications owing to their susceptibility to degrdn. by proteases.  Advances in the ability to chem. synthesize peptides and the development of new side-chain and backbone ligation strategies provide new tools to stabilize bioactive peptide epitopes.  Two such epitopes are LyP1, a nine residue peptide that localizes to tumor cells and has potential as an anticancer therapeutic, and RGDS, a tetrapeptide shown to bind to survivin and induce apoptosis.  Here we applied a variety of strategies for the stabilization of LyP1 and RGDS, including side-chain cyclization using "click" chem. and "grafting" the epitopes into two naturally occurring cyclic peptide scaffolds, i.e., θ-defensins and cyclotides.  NMR data showed that the three-disulfide θ-defensin and cyclotide scaffolds accommodated the LyP1 and RGDS epitopes but that scaffolds with fewer disulfide bonds were structurally compromised by inclusion of the LyP1 epitope.  LyP1, LyP1-, and RGDS-grafted peptides that were largely unstructured also had reduced resistance to degrdn. in human serum, showing that grafting into a stable cyclic scaffold is an effective strategy for increasing the stability of a bioactive peptide epitope.  Overall, the study demonstrates several methods for stabilizing peptide epitopes using side-chain or backbone cyclization and illustrates their potential in peptide drug design. © 2015 Wiley Periodicals, Inc.  Biopolymers (Pept Sci) 106: 89-100, 2016.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy6IO4ryYFMLVg90H21EOLACvtfcHk0liYpqWlHTC_oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVSgtb8%253D&md5=7a96a5f30b56a7f95a14fe28167c8e11</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fbip.22767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.22767%26sid%3Dliteratum%253Aachs%26aulast%3DConibear%26aufirst%3DA.%2BC.%26aulast%3DChaousis%26aufirst%3DS.%26aulast%3DDurek%26aufirst%3DT.%26aulast%3DJohan%2BRosengren%26aufirst%3DK.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DSchroeder%26aufirst%3DC.%2BI.%26atitle%3DApproaches%2520to%2520the%2520Stabilization%2520of%2520Bioactive%2520Epitopes%2520by%2520Grafting%2520and%2520Peptide%2520Cyclization%26jtitle%3DBiopolymers%26date%3D2016%26volume%3D106%26spage%3D89%26epage%3D100%26doi%3D10.1002%2Fbip.22767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okumu, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauletti, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander
Velde, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siahaan, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borchardt, R. T.</span></span> <span> </span><span class="NLM_article-title">Effect of Restricted Conformational Flexibility on the Permeation of Model Hexapeptides Across Caco-2 Cell Monolayers</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">175</span>, <span class="refDoi"> DOI: 10.1023/a:1012092409216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1023%2FA%3A1012092409216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=9090704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADyaK2sXitFyqt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1997&pages=169-175&author=F.+W.+Okumuauthor=G.+M.+Paulettiauthor=D.+G.+Vander%0AVeldeauthor=T.+J.+Siahaanauthor=R.+T.+Borchardt&title=Effect+of+Restricted+Conformational+Flexibility+on+the+Permeation+of+Model+Hexapeptides+Across+Caco-2+Cell+Monolayers&doi=10.1023%2Fa%3A1012092409216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of restricted conformational flexibility on the permeation of model hexapeptides across Caco-2 cell monolayers</span></div><div class="casAuthors">Okumu, Franklin W.; Pauletti, Giovanni M.; Vander Velde, David G.; Siahaan, Teruna J.; Borchardt, Ronald T.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">169-175</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Plenum</span>)
        </div><div class="casAbstract">The purpose of this study was to det. how restricted conformational flexibility of hexapeptides influences their cellular permeation characteristics.  Linear (Ac-Trp-Ala-Gly-Gly-X-Ala-NH2; X = Asp, Asn, Lys) and cyclic (cyclo[Trp-Ala-Gly-Gly-X-Ala]; X = Asp, Asn, Lys) hexapeptides were synthesized, and their transport characteristics were assessed using the Caco-2 cell culture model.  The lipophilicities of the hexapeptides were detd. using an immobilized artificial membrane.  Diffusion coeffs. used to calc. mol. radii were detd. by NMR.  Two-dimensional NMR spectroscopy, CD, and mol. dynamic simulations were used to elucidate the most favorable soln. structure of the cyclic Asp-contg. peptide.  The cyclic hexapeptides used in this study were 2-3 times more able to permeate (e.g., Papp = 9.3 ± 0.3 × 10-8 cm/s, X = Asp) the Caco-2 cell monolayer than were their linear analogs (e.g., Papp = 3.2 ± 0.3 × 10-8 cm/s, X = Asp).  In contrast to the linear hexapeptides, the flux of the cyclic hexapeptides was independent of charge.  The cyclic hexapeptides were shown to be more lipophilic than the linear hexapeptides as detd. by their retention times on an immobilized phospholipid column.  Detn. of mol. radii by two different techniques suggests little or no difference in size between the linear and cyclic hexapeptides.  Spectroscopic data indicate that the Asp-contg. linear hexapeptide exists in a dynamic equil. between random coil and β-turn structures while the cyclic Asp-contg. hexapeptide exists in a well-defined compact amphiphilic structure contg. two β-turns.  Cyclization of the linear hexapeptides increased their lipophilicities.  The increased permeation characteristics of the cyclic hexapeptides as compared to their linear analogs appears to be due to an increase in their flux via the transcellular route because of these increased lipophilicities.  Structural analyses of the cyclic Asp-contg. hexapeptide suggest that its well-defined soln. structure and, specifically the existence of two β-turns, explain its greater lipophilicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq53aKqMa4N-bVg90H21EOLACvtfcHk0liYpqWlHTC_oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXitFyqt7w%253D&md5=d5369a0565f8046f7e822f705ce0e859</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1023%2FA%3A1012092409216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1012092409216%26sid%3Dliteratum%253Aachs%26aulast%3DOkumu%26aufirst%3DF.%2BW.%26aulast%3DPauletti%26aufirst%3DG.%2BM.%26aulast%3DVander%2BVelde%26aufirst%3DD.%2BG.%26aulast%3DSiahaan%26aufirst%3DT.%2BJ.%26aulast%3DBorchardt%26aufirst%3DR.%2BT.%26atitle%3DEffect%2520of%2520Restricted%2520Conformational%2520Flexibility%2520on%2520the%2520Permeation%2520of%2520Model%2520Hexapeptides%2520Across%2520Caco-2%2520Cell%2520Monolayers%26jtitle%3DPharm.%2520Res.%26date%3D1997%26volume%3D14%26spage%3D169%26epage%3D175%26doi%3D10.1023%2Fa%3A1012092409216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rand, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, S. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotter, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalgutkar, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varma, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farley, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khunte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Limberakis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathiowetz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Optimizing PK Properties of Cyclic Peptides: The Effect of Side Chain Substitutions on Permeability and Clearance</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1282</span>– <span class="NLM_lpage">1289</span>, <span class="refDoi"> DOI: 10.1039/c2md20203d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1039%2FC2MD20203D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=23133740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVWns7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1282-1289&author=A.+C.+Randauthor=S.+S.+F.+Leungauthor=H.+Engauthor=C.+J.+Rotterauthor=R.+Sharmaauthor=A.+S.+Kalgutkarauthor=Y.+Zhangauthor=M.+V.+Varmaauthor=K.+A.+Farleyauthor=B.+Khunteauthor=C.+Limberakisauthor=D.+A.+Priceauthor=S.+Lirasauthor=A.+M.+Mathiowetzauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Optimizing+PK+Properties+of+Cyclic+Peptides%3A+The+Effect+of+Side+Chain+Substitutions+on+Permeability+and+Clearance&doi=10.1039%2Fc2md20203d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing PK properties of cyclic peptides: the effect of side chain substitutions on permeability and clearance</span></div><div class="casAuthors">Rand, Arthur C.; Leung, Siegfried S. F.; Eng, Heather; Rotter, Charles J.; Sharma, Raman; Kalgutkar, Amit S.; Zhang, Yizhong; Varma, Manthena V.; Farley, Kathleen A.; Khunte, Bhagyashree; Limberakis, Chris; Price, David A.; Liras, Spiros; Mathiowetz, Alan M.; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1282-1289</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A series of cyclic peptides were designed and prepd. to investigate the physicochem. properties that affect oral bioavailability of this chemotype in rats.  In particular, the ionization state of the peptide was examd. by the incorporation of naturally occurring amino acid residues that are charged in differing regions of the gut.  In addn., data was generated in a variety of in vitro assays and the usefulness of this data in predicting the subsequent oral bioavailability obsd. in the rat is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr-tYW5ziK7bVg90H21EOLACvtfcHk0lgSRXImkwv5Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVWns7%252FM&md5=fe1c9f3c7c3e6d48e4e11ee4b5818ef7</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FC2MD20203D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2MD20203D%26sid%3Dliteratum%253Aachs%26aulast%3DRand%26aufirst%3DA.%2BC.%26aulast%3DLeung%26aufirst%3DS.%2BS.%2BF.%26aulast%3DEng%26aufirst%3DH.%26aulast%3DRotter%26aufirst%3DC.%2BJ.%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DVarma%26aufirst%3DM.%2BV.%26aulast%3DFarley%26aufirst%3DK.%2BA.%26aulast%3DKhunte%26aufirst%3DB.%26aulast%3DLimberakis%26aufirst%3DC.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DMathiowetz%26aufirst%3DA.%2BM.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DOptimizing%2520PK%2520Properties%2520of%2520Cyclic%2520Peptides%253A%2520The%2520Effect%2520of%2520Side%2520Chain%2520Substitutions%2520on%2520Permeability%2520and%2520Clearance%26jtitle%3DMedChemComm%26date%3D2012%26volume%3D3%26spage%3D1282%26epage%3D1289%26doi%3D10.1039%2Fc2md20203d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">March, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbenante, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergman, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brinkworth, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickramasinghe, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Begun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Substrate-Based Cyclic Peptidomimetics of Phe-Ile-Val That Inhibit HIV-1 Protease Using a Novel Enzyme-Binding Mode</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">3375</span>– <span class="NLM_lpage">3379</span>, <span class="refDoi"> DOI: 10.1021/ja953790z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja953790z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADyaK28Xhslagsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=1996&pages=3375-3379&author=D.+R.+Marchauthor=G.+Abbenanteauthor=D.+A.+Bergmanauthor=R.+I.+Brinkworthauthor=W.+Wickramasingheauthor=J.+Begunauthor=J.+L.+Martinauthor=D.+P.+Fairlie&title=Substrate-Based+Cyclic+Peptidomimetics+of+Phe-Ile-Val+That+Inhibit+HIV-1+Protease+Using+a+Novel+Enzyme-Binding+Mode&doi=10.1021%2Fja953790z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate-based cyclic peptidomimetics of Phe-Ile-Val that inhibit HIV-1 protease using a novel enzyme-binding mode</span></div><div class="casAuthors">March, Darren R.; Abbenante, Giovanni; Bergman, Douglas A.; Brinkworth, Ross I.; Wickramasinghe, Wasantha; Begun, Jake; Martin, Jennifer L.; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3375-9</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Results are presented for inhibitors of HIV-1 protease that demonstrate a new strategy for developing peptidomimetics, involving the replacement of flexible segments of peptide substrates with conformationally constrained hydrolytically-stable macrocyclic structural mimics.  A 15-membered macrocycle that imitates the tripeptide Phe-Ile-Val was designed and incorporated into the C-terminus of Ac-Leu-Val-Phe-CHOHCH2-{Phe-Ile-Val}-NH2, an inhibitor of HIV-1 protease derived from a substrate sequence.  Advantages of the macrocycle over the acyclic peptide include constraining its components into their bioactive conformation and protecting the amide bonds from enzymic degrdn., the cycle being stable to acid, gastric proteases, and plasma.  Mol. modeling and X-ray structural studies reveal that the cyclic inhibitors have a unique enzyme-binding mode, the sterically unencumbered hydroxyethylamine isostere binds via both its hydroxyl and protonated nitrogen to the anionic Asp25 catalytic residues.  The novel macrocycle superimposes well on the linear peptidic inhibitor for which it was designed as a structural mimic.  Structural mimicry led to functional mimicry as shown by comparable inhibition of the protease by cyclic and acyclic mols.  Further modification of the acyclic N-terminus (Leu-Val-Phe) gave stable, water-sol., potent inhibitors of HIV-1 protease.  This approach may have general application to the development of mimetics of other bioactive peptides, including inhibitors of other enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpleApAoH1u-bVg90H21EOLACvtfcHk0lgSRXImkwv5Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xhslagsr8%253D&md5=716171632ee911b5e5be59a87a584a50</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fja953790z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja953790z%26sid%3Dliteratum%253Aachs%26aulast%3DMarch%26aufirst%3DD.%2BR.%26aulast%3DAbbenante%26aufirst%3DG.%26aulast%3DBergman%26aufirst%3DD.%2BA.%26aulast%3DBrinkworth%26aufirst%3DR.%2BI.%26aulast%3DWickramasinghe%26aufirst%3DW.%26aulast%3DBegun%26aufirst%3DJ.%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DSubstrate-Based%2520Cyclic%2520Peptidomimetics%2520of%2520Phe-Ile-Val%2520That%2520Inhibit%2520HIV-1%2520Protease%2520Using%2520a%2520Novel%2520Enzyme-Binding%2520Mode%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1996%26volume%3D118%26spage%3D3375%26epage%3D3379%26doi%3D10.1021%2Fja953790z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adessi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soto, C.</span></span> <span> </span><span class="NLM_article-title">Converting a Peptide into a Drug: Strategies to Improve Stability and Bioavailability</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">963</span>– <span class="NLM_lpage">978</span>, <span class="refDoi"> DOI: 10.2174/0929867024606731</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.2174%2F0929867024606731" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=11966456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD38XjtlKjtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=963-978&author=C.+Adessiauthor=C.+Soto&title=Converting+a+Peptide+into+a+Drug%3A+Strategies+to+Improve+Stability+and+Bioavailability&doi=10.2174%2F0929867024606731"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Converting a peptide into a drug: Strategies to improve stability and bioavailability</span></div><div class="casAuthors">Adessi, Celine; Soto, Claudio</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">963-978</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers</span>)
        </div><div class="casAbstract">A review.  The discovery of peptide hormones, growth factors and neuropeptides implicated in vital biol. functions of our organism has increased interest in therapeutic use of short peptides.  However, the development of peptides as clin. useful drugs is greatly limited by their poor metabolic stability and low bioavailability, which is due in part to their inability to readily cross membrane barriers such as the intestinal and blood-brain barriers.  The aim of peptide medicinal chem. is, therefore, to develop strategies to overcome these problems.  Recent progress in chem. synthesis and design have resulted in several strategies for producing modified peptides and mimetics with lower susceptibility to proteolysis and improved bioavailability, which has increased the probability of obtaining useful drugs structurally related to parent peptides.  This review describes different exptl. approaches to transforming a peptide into a potential drug and provides examples of the usefulness of these strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_7AaDVSzEfbVg90H21EOLACvtfcHk0lgSRXImkwv5Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjtlKjtbk%253D&md5=95cae8662dbf6eee4737276b40bf7209</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.2174%2F0929867024606731&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867024606731%26sid%3Dliteratum%253Aachs%26aulast%3DAdessi%26aufirst%3DC.%26aulast%3DSoto%26aufirst%3DC.%26atitle%3DConverting%2520a%2520Peptide%2520into%2520a%2520Drug%253A%2520Strategies%2520to%2520Improve%2520Stability%2520and%2520Bioavailability%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2002%26volume%3D9%26spage%3D963%26epage%3D978%26doi%3D10.2174%2F0929867024606731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halle, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selincer, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byk, G.</span></span> <span> </span><span class="NLM_article-title">Backbone Cyclization: A New Method for Conferring Conformational Constraint on Peptides</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">745</span>– <span class="NLM_lpage">750</span>, <span class="refDoi"> DOI: 10.1002/bip.360310619</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fbip.360310619" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1718473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADyaK3MXltFGjur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1991&pages=745-750&author=C.+Gilonauthor=D.+Halleauthor=M.+Chorevauthor=Z.+Selincerauthor=G.+Byk&title=Backbone+Cyclization%3A+A+New+Method+for+Conferring+Conformational+Constraint+on+Peptides&doi=10.1002%2Fbip.360310619"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Backbone cyclization:  a new method for conferring conformational constraint on peptides</span></div><div class="casAuthors">Gilon, Chaim; Halle, David; Chorev, Michael; Selinger, Zvi; Byk, Gerardo</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">745-50</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    </div><div class="casAbstract">This article describes a new concept of medium- and long-range cyclization of peptides through "backbone cyclization".  In this approach, conformational constraints are conferred on a peptide by linking ω-substituted alkylidene chains replacing Nα or Cα hydrogens in a peptide backbone.  Backbone cyclization, which is divided into N-backbone and C-backbone cyclizations, allow for new modes of cyclization in addn. to the classical ones that are limited to cyclization through the side chains and/or the amino or carboxyl terminal groups.  The article also describes the application of the N-backbone cyclization to the active region of substance P.  Conformational constraints of this peptide by the classical cyclization modes led to inactive analogs whereas N-backbone cyclization provided an active, selective, and metabolically stable analog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo10ie5nZFwl7Vg90H21EOLACvtfcHk0lhWvdKEjPbRvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXltFGjur0%253D&md5=de539b2a1242d0ec87e3c50186bc8811</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fbip.360310619&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.360310619%26sid%3Dliteratum%253Aachs%26aulast%3DGilon%26aufirst%3DC.%26aulast%3DHalle%26aufirst%3DD.%26aulast%3DChorev%26aufirst%3DM.%26aulast%3DSelincer%26aufirst%3DZ.%26aulast%3DByk%26aufirst%3DG.%26atitle%3DBackbone%2520Cyclization%253A%2520A%2520New%2520Method%2520for%2520Conferring%2520Conformational%2520Constraint%2520on%2520Peptides%26jtitle%3DBiopolymers%26date%3D1991%26volume%3D31%26spage%3D745%26epage%3D750%26doi%3D10.1002%2Fbip.360310619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pauletti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangwar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siahaan, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aubé, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borchardt, R. T.</span></span> <span> </span><span class="NLM_article-title">Improvement of Oral Peptide Bioavailability: Peptidomimetics and Prodrug Strategies</span>. <i>Adv. Drug Deliv. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">256</span>, <span class="refDoi"> DOI: 10.1016/s0169-409x(97)00045-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1016%2FS0169-409X%2897%2900045-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10837560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjs12ktL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1997&pages=235-256&author=G.+Paulettiauthor=S.+Gangwarauthor=T.+J.+Siahaanauthor=J.+Aub%C3%A9author=R.+T.+Borchardt&title=Improvement+of+Oral+Peptide+Bioavailability%3A+Peptidomimetics+and+Prodrug+Strategies&doi=10.1016%2Fs0169-409x%2897%2900045-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Improvement of oral peptide bioavailability: Peptidomimetics and prodrug strategies</span></div><div class="casAuthors">Pauletti, Giovanni M.; Gangwar, Sanjeev; Siahaan, Teruna J.; Aube, Jeffrey; Borchardt, Ronald T.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2,3</span>),
    <span class="NLM_cas:pages">235-256</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review with 209 refs.  Clin. development of orally active peptide drugs has been restricted by their unfavorable physicochem. properties, which limit their intestinal mucosal permeation and their lack of stability against enzymic degrdn.  Successful oral delivery of peptides will depend, therefore, on strategies designed to alter the physicochem. characteristics of these potential drugs, without changing their biol. activity, in order to overcome the phys. and biochem. barrier properties of the intestinal cells.  This manuscript will focus on the physiol. limitations for oral peptide delivery and on various strategies using chem. modifications to improve oral bioavailability of peptide-based drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_apmBvn28dLVg90H21EOLACvtfcHk0lhWvdKEjPbRvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjs12ktL0%253D&md5=de147c42769ae91c7ad3d49321f3ddc3</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2897%2900045-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252897%252900045-8%26sid%3Dliteratum%253Aachs%26aulast%3DPauletti%26aufirst%3DG.%26aulast%3DGangwar%26aufirst%3DS.%26aulast%3DSiahaan%26aufirst%3DT.%2BJ.%26aulast%3DAub%25C3%25A9%26aufirst%3DJ.%26aulast%3DBorchardt%26aufirst%3DR.%2BT.%26atitle%3DImprovement%2520of%2520Oral%2520Peptide%2520Bioavailability%253A%2520Peptidomimetics%2520and%2520Prodrug%2520Strategies%26jtitle%3DAdv.%2520Drug%2520Deliv.%2520Rev.%26date%3D1997%26volume%3D27%26spage%3D235%26epage%3D256%26doi%3D10.1016%2Fs0169-409x%2897%2900045-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burton, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conradi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, N. F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgers, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borchardt, R. T.</span></span> <span> </span><span class="NLM_article-title">How Structural Features Influence the Biomembrane Permeability of Peptides</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">1336</span>– <span class="NLM_lpage">1340</span>, <span class="refDoi"> DOI: 10.1021/js960067d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1021%2Fjs960067d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=8961149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADyaK28Xms1Cmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=1996&pages=1336-1340&author=P.+S.+Burtonauthor=R.+A.+Conradiauthor=N.+F.+H.+Hoauthor=A.+R.+Hilgersauthor=R.+T.+Borchardt&title=How+Structural+Features+Influence+the+Biomembrane+Permeability+of+Peptides&doi=10.1021%2Fjs960067d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">How Structural Features Influence the Biomembrane Permeability of Peptides</span></div><div class="casAuthors">Burton, Philip S.; Conradi, Robert A.; Ho, Norman F. H.; Hilgers, Allen R.; Borchardt, Ronald T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1336-1340</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 55 refs.  Successful drug development requires not only optimization of specific and potent pharmacol. activity at the target site, but also efficient delivery to that site.  Many promising new peptides with novel therapeutic potential for the treatment of AIDS, cardiovascular diseases, and CNS disorders have been identified, yet their clin. utility has been limited by delivery problems.  Along with metab., a major factor contributing to the poor bioavailability of peptides is thought to be inefficient transport across cell membranes.  At the present time, the reasons for this poor transport are poorly understood.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAZOZxHGFusbVg90H21EOLACvtfcHk0lhWvdKEjPbRvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xms1Cmt7g%253D&md5=600e4f60ad068cf6f1f542fa1116c79d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjs960067d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjs960067d%26sid%3Dliteratum%253Aachs%26aulast%3DBurton%26aufirst%3DP.%2BS.%26aulast%3DConradi%26aufirst%3DR.%2BA.%26aulast%3DHo%26aufirst%3DN.%2BF.%2BH.%26aulast%3DHilgers%26aufirst%3DA.%2BR.%26aulast%3DBorchardt%26aufirst%3DR.%2BT.%26atitle%3DHow%2520Structural%2520Features%2520Influence%2520the%2520Biomembrane%2520Permeability%2520of%2520Peptides%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D1996%26volume%3D85%26spage%3D1336%26epage%3D1340%26doi%3D10.1021%2Fjs960067d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGeary, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Peptidomimetics: Potential for Drug Development</span>. <i>Curr. Opin. Drug Discov. Dev.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">208</span>– <span class="NLM_lpage">217</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=19649821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADyaK1MXntlyksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1998&pages=208-217&author=R.+P.+McGearyauthor=D.+P.+Fairlie&title=Macrocyclic+Peptidomimetics%3A+Potential+for+Drug+Development"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic peptidomimetics: potential for drug development</span></div><div class="casAuthors">McGeary, Ross P.; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">208-217</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Current Drugs Ltd.</span>)
        </div><div class="casAbstract">A review with 57 refs. focusing mainly on the structure-activity relationship for synthetic macrocyclic peptidomimetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyu9dgfXnpG7Vg90H21EOLACvtfcHk0liW73GlRtzLYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntlyksw%253D%253D&md5=2d443820e8b50d42df4d34f115dee467</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMcGeary%26aufirst%3DR.%2BP.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DMacrocyclic%2520Peptidomimetics%253A%2520Potential%2520for%2520Drug%2520Development%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discov.%2520Dev.%26date%3D1998%26volume%3D1%26spage%3D208%26epage%3D217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhardwaj, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahl, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmore, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheneval, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchko, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulavarti, S. V. S. R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eletsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnsen, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greisen, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linsky, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonneau, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coutsias, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correnti, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szyperski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span> <span> </span><span class="NLM_article-title">Accurate De Novo Design of Hyperstable Constrained Peptides</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>538</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">335</span>, <span class="refDoi"> DOI: 10.1038/nature19791</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1038%2Fnature19791" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=27626386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWiurnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=538&publication_year=2016&pages=329-335&author=G.+Bhardwajauthor=V.+K.+Mulliganauthor=C.+D.+Bahlauthor=J.+M.+Gilmoreauthor=P.+J.+Harveyauthor=O.+Chenevalauthor=G.+W.+Buchkoauthor=S.+V.+S.+R.+K.+Pulavartiauthor=A.+Eletskyauthor=P.-S.+Huangauthor=W.+A.+Johnsenauthor=P.+J.+Greisenauthor=G.+J.+Rocklinauthor=Y.+Songauthor=T.+W.+Linskyauthor=A.+Watkinsauthor=S.+A.+Rettieauthor=X.+Xuauthor=L.+P.+Carterauthor=R.+Bonneauauthor=J.+M.+Olsonauthor=E.+Coutsiasauthor=C.+E.+Correntiauthor=T.+Szyperskiauthor=D.+J.+Craikauthor=D.+Bakerauthor=D.+Baker&title=Accurate+De+Novo+Design+of+Hyperstable+Constrained+Peptides&doi=10.1038%2Fnature19791"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate de novo design of hyperstable constrained peptides</span></div><div class="casAuthors">Bhardwaj, Gaurav; Mulligan, Vikram Khipple; Bahl, Christopher D.; Gilmore, Jason M.; Harvey, Peta J.; Cheneval, Olivier; Buchko, Garry W.; Pulavarti, Surya V. S. R. K.; Kaas, Quentin; Eletsky, Alexander; Huang, Po-Ssu; Johnsen, William A.; Greisen, Per Jr; Rocklin, Gabriel J.; Song, Yifan; Linsky, Thomas W.; Watkins, Andrew; Rettie, Stephen A.; Xu, Xianzhong; Carter, Lauren P.; Bonneau, Richard; Olson, James M.; Coutsias, Evangelos; Correnti, Colin E.; Szyperski, Thomas; Craik, David J.; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">538</span>
        (<span class="NLM_cas:issue">7625</span>),
    <span class="NLM_cas:pages">329-335</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Naturally occurring, pharmacol. active peptides constrained with covalent crosslinks generally have shapes that have evolved to fit precisely into binding pockets on their targets.  Such peptides can have excellent pharmaceutical properties, combining the stability and tissue penetration of small-mol. drugs with the specificity of much larger protein therapeutics.  The ability to design constrained peptides with precisely specified tertiary structures would enable the design of shape-complementary inhibitors of arbitrary targets.  Here we describe the development of computational methods for accurate de novo design of conformationally restricted peptides, and the use of these methods to design 18-47 residue, disulfide-crosslinked peptides, a subset of which are heterochiral and/or N-C backbone-cyclized.  Both genetically encodable and non-canonical peptides are exceptionally stable to thermal and chem. denaturation, and 12 exptl. detd. X-ray and NMR structures are nearly identical to the computational design models.  The computational design methods and stable scaffolds presented here provide the basis for development of a new generation of peptide-based drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHeMGaFeL_kbVg90H21EOLACvtfcHk0liW73GlRtzLYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWiurnL&md5=2d2b551ad78a2d76c99eeaba0c763c8e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnature19791&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature19791%26sid%3Dliteratum%253Aachs%26aulast%3DBhardwaj%26aufirst%3DG.%26aulast%3DMulligan%26aufirst%3DV.%2BK.%26aulast%3DBahl%26aufirst%3DC.%2BD.%26aulast%3DGilmore%26aufirst%3DJ.%2BM.%26aulast%3DHarvey%26aufirst%3DP.%2BJ.%26aulast%3DCheneval%26aufirst%3DO.%26aulast%3DBuchko%26aufirst%3DG.%2BW.%26aulast%3DPulavarti%26aufirst%3DS.%2BV.%2BS.%2BR.%2BK.%26aulast%3DEletsky%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DP.-S.%26aulast%3DJohnsen%26aufirst%3DW.%2BA.%26aulast%3DGreisen%26aufirst%3DP.%2BJ.%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLinsky%26aufirst%3DT.%2BW.%26aulast%3DWatkins%26aufirst%3DA.%26aulast%3DRettie%26aufirst%3DS.%2BA.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DCarter%26aufirst%3DL.%2BP.%26aulast%3DBonneau%26aufirst%3DR.%26aulast%3DOlson%26aufirst%3DJ.%2BM.%26aulast%3DCoutsias%26aufirst%3DE.%26aulast%3DCorrenti%26aufirst%3DC.%2BE.%26aulast%3DSzyperski%26aufirst%3DT.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DBaker%26aufirst%3DD.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DAccurate%2520De%2520Novo%2520Design%2520of%2520Hyperstable%2520Constrained%2520Peptides%26jtitle%3DNature%26date%3D2016%26volume%3D538%26spage%3D329%26epage%3D335%26doi%3D10.1038%2Fnature19791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soumana, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurt Yilmaz, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prachanronarong, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aydin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, C. A.</span></span> <span> </span><span class="NLM_article-title">Structural and Thermodynamic Effects of Macrocyclization in HCV NS3/4A Inhibitor MK-5172</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">900</span>– <span class="NLM_lpage">909</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00647</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00647" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVGitb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=900-909&author=D.+I.+Soumanaauthor=N.+Kurt+Yilmazauthor=K.+L.+Prachanronarongauthor=C.+Aydinauthor=A.+Aliauthor=C.+A.+Schiffer&title=Structural+and+Thermodynamic+Effects+of+Macrocyclization+in+HCV+NS3%2F4A+Inhibitor+MK-5172&doi=10.1021%2Facschembio.5b00647"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and Thermodynamic Effects of Macrocyclization in HCV NS3/4A Inhibitor MK-5172</span></div><div class="casAuthors">Soumana, Djade I.; Kurt Yilmaz, Nese; Prachanronarong, Kristina L.; Aydin, Cihan; Ali, Akbar; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">900-909</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recent advances in direct-acting antivirals against Hepatitis C Virus (HCV) have led to the development of potent inhibitors, including MK-5172, that target the viral NS3/4A protease with relatively low susceptibility to resistance.  MK-5172 has a P2-P4 macrocycle and a unique binding mode among current protease inhibitors where the P2 quinoxaline packs against the catalytic residues H57 and D81.  However, the effect of macrocyclization on this binding mode is not clear, as is the relation between macrocyclization, thermodn. stabilization, and susceptibility to the resistance mutation A156T.  We have detd. high-resoln. crystal structures of linear and P1-P3 macrocyclic analogs of MK-5172 bound to WT and A156T protease and compared these structures, their mol. dynamics, and exptl. binding thermodn. to the parent compd.  We find that the "unique" binding mode of MK-5172 is conserved even when the P2-P4 macrocycle is removed or replaced with a P1-P3 macrocycle.  While beneficial to decreasing the entropic penalty assocd. with binding, the constraint exerted by the P2-P4 macrocycle prevents efficient rearrangement to accommodate the A156T mutation, a deficit alleviated in the linear and P1-P3 analogs.  Design of macrocyclic inhibitors against NS3/4A needs to achieve the best balance between exerting optimal conformational constraint for enhancing potency, fitting within the substrate envelope and allowing adaptability to be robust against resistance mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVaszg0NfWS7Vg90H21EOLACvtfcHk0liW73GlRtzLYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVGitb3F&md5=4f3cadfdedbb5af22d0ef55c088d3687</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00647%26sid%3Dliteratum%253Aachs%26aulast%3DSoumana%26aufirst%3DD.%2BI.%26aulast%3DKurt%2BYilmaz%26aufirst%3DN.%26aulast%3DPrachanronarong%26aufirst%3DK.%2BL.%26aulast%3DAydin%26aufirst%3DC.%26aulast%3DAli%26aufirst%3DA.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DStructural%2520and%2520Thermodynamic%2520Effects%2520of%2520Macrocyclization%2520in%2520HCV%2520NS3%252F4A%2520Inhibitor%2520MK-5172%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D900%26epage%3D909%26doi%3D10.1021%2Facschembio.5b00647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delorbe, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clements, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whiddon, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. F.</span></span> <span> </span><span class="NLM_article-title">Thermodynamic and Structural Effects of Macrocyclic Constraints in Protein-Ligand Interactions</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">448</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/ml100142y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100142y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A280%3ADC%252BC2srmt1OisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=448-452&author=J.+E.+Delorbeauthor=J.+H.+Clementsauthor=B.+B.+Whiddonauthor=S.+F.+Martin&title=Thermodynamic+and+Structural+Effects+of+Macrocyclic+Constraints+in+Protein-Ligand+Interactions&doi=10.1021%2Fml100142y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Thermodynamic and Structural Effects of Macrocyclization as a Constraining Method in Protein-Ligand Interactions</span></div><div class="casAuthors">Delorbe John E; Clements John H; Whiddon Benjamin B; Martin Stephen F</div><div class="citationInfo"><span class="NLM_cas:title">ACS medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">448-452</span>
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    </div><div class="casAbstract">The thermodynamic and structural effects of macrocyclization as a tactic for stabilizing the biologically-active conformation of Grb2 SH2 binding peptides were investigated using isothermal titration calorimetry and x-ray crystallography. 23-Membered macrocycles containing the sequence pYVN were slightly more potent than their linear controls; however, preorganization did not necessarily eventuate in a more favorable binding entropy.  Structures of complexes of macrocycle 7 and its acyclic control 8 are similar except for differences in relative orientations of corresponding atoms in the linking moieties of 7 and 8.  There are no differences in the number of direct or water-mediated protein-ligand contacts that might account for the less favorable binding enthalpy of 7; however, an intramolecular hydrogen bond between the pY and pY+3 residues in 8 that is absent in 7 may be a factor.  These studies highlight the difficulties associated with correlating energetics and structure in protein-ligand interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTdL3gvK7FnJAwxfk_AGB-EfW6udTcc2eYJUfloAJ1ghLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srmt1OisA%253D%253D&md5=bec2aeb7e9127f0f6310dbccf2663c76</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fml100142y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100142y%26sid%3Dliteratum%253Aachs%26aulast%3DDelorbe%26aufirst%3DJ.%2BE.%26aulast%3DClements%26aufirst%3DJ.%2BH.%26aulast%3DWhiddon%26aufirst%3DB.%2BB.%26aulast%3DMartin%26aufirst%3DS.%2BF.%26atitle%3DThermodynamic%2520and%2520Structural%2520Effects%2520of%2520Macrocyclic%2520Constraints%2520in%2520Protein-Ligand%2520Interactions%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D1%26spage%3D448%26epage%3D452%26doi%3D10.1021%2Fml100142y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hruby, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Obeidi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazmierski, W.</span></span> <span> </span><span class="NLM_article-title">Emerging Approaches in the Molecular Design of Receptor-Selective Peptide Ligands: Conformational, Topographical and Dynamic Considerations</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>268</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1042/bj2680249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1042%2Fbj2680249" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=2163604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADyaK3cXktFOqs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=268&publication_year=1990&pages=249-262&author=V.+J.+Hrubyauthor=F.+Al-Obeidiauthor=W.+Kazmierski&title=Emerging+Approaches+in+the+Molecular+Design+of+Receptor-Selective+Peptide+Ligands%3A+Conformational%2C+Topographical+and+Dynamic+Considerations&doi=10.1042%2Fbj2680249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging approaches in the molecular design of receptor-selective peptide ligands:  conformational, topographical and dynamic considerations</span></div><div class="casAuthors">Hruby, Victor J.; Al-Obeidi, Fahad; Kazmierski, Wieslaw</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">268</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-62</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    </div><div class="casAbstract">A review, with 107 refs., of the design of peptide and protein ligands (hormones) with specific phys., chem., and biol. properties.  Specific topics discussed were conformational constraint, structure-biol. activities relations, and rational drugs as well as topog. design possibilities and prospects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7mFpCF_I0xrVg90H21EOLACvtfcHk0lgS7ImNWPczRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXktFOqs7Y%253D&md5=f0b6ef00b0d688997db977667caee697</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1042%2Fbj2680249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj2680249%26sid%3Dliteratum%253Aachs%26aulast%3DHruby%26aufirst%3DV.%2BJ.%26aulast%3DAl-Obeidi%26aufirst%3DF.%26aulast%3DKazmierski%26aufirst%3DW.%26atitle%3DEmerging%2520Approaches%2520in%2520the%2520Molecular%2520Design%2520of%2520Receptor-Selective%2520Peptide%2520Ligands%253A%2520Conformational%252C%2520Topographical%2520and%2520Dynamic%2520Considerations%26jtitle%3DBiochem.%2520J.%26date%3D1990%26volume%3D268%26spage%3D249%26epage%3D262%26doi%3D10.1042%2Fbj2680249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and Computational Approaches to Estimate Solubility and Permeability in Drug Discovery and Development Settings</span>. <i>Adv. Drug Deliv. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1016/s0169-409x(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+Computational+Approaches+to+Estimate+Solubility+and+Permeability+in+Drug+Discovery+and+Development+Settings&doi=10.1016%2Fs0169-409x%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0lgS7ImNWPczRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520Computational%2520Approaches%2520to%2520Estimate%2520Solubility%2520and%2520Permeability%2520in%2520Drug%2520Discovery%2520and%2520Development%2520Settings%26jtitle%3DAdv.%2520Drug%2520Deliv.%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2Fs0169-409x%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veber, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopple, K. D.</span></span> <span> </span><span class="NLM_article-title">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">2615</span>– <span class="NLM_lpage">2623</span>, <span class="refDoi"> DOI: 10.1021/jm020017n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020017n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2615-2623&author=D.+F.+Veberauthor=S.+R.+Johnsonauthor=H.-Y.+Chengauthor=B.+R.+Smithauthor=K.+W.+Wardauthor=K.+D.+Kopple&title=Molecular+Properties+That+Influence+the+Oral+Bioavailability+of+Drug+Candidates&doi=10.1021%2Fjm020017n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span></div><div class="casAuthors">Veber, Daniel F.; Johnson, Stephen R.; Cheng, Hung-Yuan; Smith, Brian R.; Ward, Keith W.; Kopple, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2615-2623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oral bioavailability measurements in rats for over 1100 drug candidates studied at Smith-Kline Beecham Pharmaceuticals (now Glaxo Smith-Kline) have allowed us to analyze the relative importance of mol. properties considered to influence that drug property.  Reduced mol. flexibility, as measured by the no. of rotatable bonds, and low polar surface area or total hydrogen bond count (sum of donors and acceptors) are found to be important predictors of good oral bioavailability, independent of mol. wt.  That on av. both the no. of rotatable bonds and polar surface area or hydrogen bond count tend to increase with mol. wt. may in part explain the success of the mol. wt. parameter in predicting oral bioavailability.  The commonly applied mol. wt. cutoff at 500 does not itself significantly sep. compds. with poor oral bioavailability from those with acceptable values in this extensive data set.  Our observations suggest that compds. which meet only the 2 criteria of (1) 10 or fewer rotatable bonds and (2) polar surface area ≤140 Å2 (or 12 or fewer H-bond donors and acceptors) will have a high probability of good oral bioavailability in the rat.  Data sets for the artificial membrane permeation rate and for clearance in the rat were also examd.  Reduced polar surface area correlates better with increased permeation rate than does lipophilicity (C log P), and increased rotatable bond count has a neg. effect on the permeation rate.  A threshold permeation rate is a prerequisite of oral bioavailability.  The rotatable bond count does not correlate with the data examd. here for the in vivo clearance rate in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3JtJEkO7yVLVg90H21EOLACvtfcHk0ljxuJM2FjezEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D&md5=eaad26ed6a259de82ad65a8834fc397d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm020017n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020017n%26sid%3Dliteratum%253Aachs%26aulast%3DVeber%26aufirst%3DD.%2BF.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DCheng%26aufirst%3DH.-Y.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DKopple%26aufirst%3DK.%2BD.%26atitle%3DMolecular%2520Properties%2520That%2520Influence%2520the%2520Oral%2520Bioavailability%2520of%2520Drug%2520Candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2615%26epage%3D2623%26doi%3D10.1021%2Fjm020017n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vieth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutherland, J. J.</span></span> <span> </span><span class="NLM_article-title">Dependence of Molecular Properties on Proteomic Family for Marketed Oral Drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">3451</span>– <span class="NLM_lpage">3453</span>, <span class="refDoi"> DOI: 10.1021/jm0603825</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0603825" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD28XksFKks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=3451-3453&author=M.+Viethauthor=J.+J.+Sutherland&title=Dependence+of+Molecular+Properties+on+Proteomic+Family+for+Marketed+Oral+Drugs&doi=10.1021%2Fjm0603825"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Dependence of Molecular Properties on Proteomic Family for Marketed Oral Drugs</span></div><div class="casAuthors">Vieth, Michal; Sutherland, Jeffrey J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3451-3453</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An assocn. of drugs with their proteomic family reveals that mol. properties of drugs targeting proteases, lipid and peptide G-protein-coupled receptors (GPCRs), and nuclear hormone receptors significantly exceed limits for some properties in the "rule of five", while drugs targeting cytochrome P450s, biogenic amine GPCRs, and transporters have significantly lower values for certain properties.  Also, the variation in drug targets appears to be a factor explaining increasing mol. wt. over time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOC7vpvPxrV7Vg90H21EOLACvtfcHk0ljxuJM2FjezEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XksFKks70%253D&md5=9272583ace87f6080bd0e1b01ad7c657</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm0603825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0603825%26sid%3Dliteratum%253Aachs%26aulast%3DVieth%26aufirst%3DM.%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26atitle%3DDependence%2520of%2520Molecular%2520Properties%2520on%2520Proteomic%2520Family%2520for%2520Marketed%2520Oral%2520Drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D3451%26epage%3D3453%26doi%3D10.1021%2Fjm0603825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paolini, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapland, R. H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Hoorn, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Global Mapping of Pharmacological Space</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">805</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1038/nbt1228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1038%2Fnbt1228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=16841068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=805-815&author=G.+V.+Paoliniauthor=R.+H.+B.+Shaplandauthor=W.+P.+van+Hoornauthor=J.+S.+Masonauthor=A.+L.+Hopkins&title=Global+Mapping+of+Pharmacological+Space&doi=10.1038%2Fnbt1228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Global mapping of pharmacological space</span></div><div class="casAuthors">Paolini, Gaia V.; Shapland, Richard H. B.; van Hoorn, Willem P.; Mason, Jonathan S.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">805-815</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We present the global mapping of pharmacol. space by the integration of several vast sources of medicinal chem. structure-activity relationships (SAR) data.  Our comprehensive mapping of pharmacol. space enables us to identify confidently the human targets for which chem. tools and drugs have been discovered to date.  The integration of SAR data from diverse sources by unique canonical chem. structure, protein sequence and disease indication enables the construction of a ligand-target matrix to explore the global relationships between chem. structure and biol. targets.  Using the data matrix, we are able to catalog the links between proteins in chem. space as a polypharmacol. interaction network.  We demonstrate that probabilistic models can be used to predict pharmacol. from a large knowledge base.  The relationships between proteins, chem. structures and drug-like properties provide a framework for developing a probabilistic approach to drug discovery that can be exploited to increase research productivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV_CE47zZq8LVg90H21EOLACvtfcHk0ljxuJM2FjezEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D&md5=f559b34692cc903a1b503deb07030c5d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnbt1228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1228%26sid%3Dliteratum%253Aachs%26aulast%3DPaolini%26aufirst%3DG.%2BV.%26aulast%3DShapland%26aufirst%3DR.%2BH.%2BB.%26aulast%3Dvan%2BHoorn%26aufirst%3DW.%2BP.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DGlobal%2520Mapping%2520of%2520Pharmacological%2520Space%26jtitle%3DNat.%2520Biotechnol.%26date%3D2006%26volume%3D24%26spage%3D805%26epage%3D815%26doi%3D10.1038%2Fnbt1228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gottschling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuster, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holzmann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span> <span> </span><span class="NLM_article-title">Combinatorial and Rational Strategies to Develop Nonpeptidic Α4β7-Integrin Antagonists from Cyclic Peptides</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">3007</span>– <span class="NLM_lpage">3011</span>, <span class="refDoi"> DOI: 10.1002/1521-3773(20020816)41:16<3007::aid-anie3007>3.0.co;2-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2F1521-3773%2820020816%2941%3A16%3C3007%3A%3AAID-ANIE3007%3E3.0.CO%3B2-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD38XmslSrt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=3007-3011&author=D.+Gottschlingauthor=J.+Boerauthor=A.+Schusterauthor=B.+Holzmannauthor=H.+Kessler&title=Combinatorial+and+Rational+Strategies+to+Develop+Nonpeptidic+%CE%914%CE%B27-Integrin+Antagonists+from+Cyclic+Peptides&doi=10.1002%2F1521-3773%2820020816%2941%3A16%3C3007%3A%3Aaid-anie3007%3E3.0.co%3B2-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Combinatorial and rational strategies to develop non-peptidic α4β7-integrin antagonists from cyclic peptides</span></div><div class="casAuthors">Gottschling, Dirk; Boer, Jurgen; Schuster, Anja; Holzmann, Bernhard; Kessler, Horst</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3007-3011</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">Non-peptidic compds. derived from cyclic peptides such as cyclo(F-L-D-F-D-p) were developed for inhibiting the α4β7-integrin/MAdCAM-1 interaction.  It possible to develop low-mol. wt. non-peptidic α4β7-integrin antagonists by using rational and combinatorial strategies.  A stepwise procedure, i.e., protein sequence → cyclic, constrained peptides → peptidomimetics → nonpeptide, can be a valuable method to develop new drugs.  Peptidomimetics based on isoquinoline-3-carbonyl-Leu-Asp-Thr-OH were prepd. and tested for their effect on α4β7- and α4β1-integrin-mediated cell adhesion to MAdCAM-1 and VCAM-1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1wX_AdejE77Vg90H21EOLACvtfcHk0lihEIu0F7tnMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmslSrt7o%253D&md5=b9c124df0448bbfdb393502362f99101</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2F1521-3773%2820020816%2941%3A16%3C3007%3A%3AAID-ANIE3007%3E3.0.CO%3B2-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1521-3773%252820020816%252941%253A16%253C3007%253A%253AAID-ANIE3007%253E3.0.CO%253B2-3%26sid%3Dliteratum%253Aachs%26aulast%3DGottschling%26aufirst%3DD.%26aulast%3DBoer%26aufirst%3DJ.%26aulast%3DSchuster%26aufirst%3DA.%26aulast%3DHolzmann%26aufirst%3DB.%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DCombinatorial%2520and%2520Rational%2520Strategies%2520to%2520Develop%2520Nonpeptidic%2520%25CE%25914%25CE%25B27-Integrin%2520Antagonists%2520from%2520Cyclic%2520Peptides%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2002%26volume%3D41%26spage%3D3007%26epage%3D3011%26doi%3D10.1002%2F1521-3773%2820020816%2941%3A16%3C3007%3A%3Aaid-anie3007%3E3.0.co%3B2-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mallinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span> <span> </span><span class="NLM_article-title">Macrocycles in New Drug Discovery</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1409</span>– <span class="NLM_lpage">1438</span>, <span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-1438&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+New+Drug+Discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0lihEIu0F7tnMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520New%2520Drug%2520Discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D1438%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yudin, A. K.</span></span> <span> </span><span class="NLM_article-title">Contemporary Strategies for Peptide Macrocyclization</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">509</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1038/nchem.1062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1038%2Fnchem.1062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=21697871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvFWjtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=509-524&author=C.+J.+Whiteauthor=A.+K.+Yudin&title=Contemporary+Strategies+for+Peptide+Macrocyclization&doi=10.1038%2Fnchem.1062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Contemporary strategies for peptide macrocyclization</span></div><div class="casAuthors">White, Christopher J.; Yudin, Andrei K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">509-524</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Peptide macrocycles have found applications that range from drug discovery to nanomaterials.  These ring-shaped mols. have shown remarkable capacity for functional fine-tuning.  Such capacity is enabled by the possibility of adjusting the peptide conformation using the techniques of chem. synthesis.  Cyclic peptides have been difficult, and often impossible, to prep. using traditional synthetic methods.  For macrocyclization to occur, the activated peptide must adopt an entropically disfavored pre-cyclization conformation before forming the desired product.  Here, recent solns. to some of the major challenges in this important area of contemporary synthesis were reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUTda6OkfQprVg90H21EOLACvtfcHk0lihEIu0F7tnMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvFWjtrc%253D&md5=32dfd94692ca20901dc8828bcd27ae8c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1062%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DC.%2BJ.%26aulast%3DYudin%26aufirst%3DA.%2BK.%26atitle%3DContemporary%2520Strategies%2520for%2520Peptide%2520Macrocyclization%26jtitle%3DNat.%2520Chem.%26date%3D2011%26volume%3D3%26spage%3D509%26epage%3D524%26doi%3D10.1038%2Fnchem.1062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passioura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Technologies for the Synthesis of MRNA-Encoding Libraries and Discovery of Bioactive Natural Product-Inspired Non-Traditional Macrocyclic Peptides</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">3502</span>– <span class="NLM_lpage">3528</span>, <span class="refDoi"> DOI: 10.3390/molecules18033502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.3390%2Fmolecules18033502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=23507778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlt1GitLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=3502-3528&author=K.+Itoauthor=T.+Passiouraauthor=H.+Suga&title=Technologies+for+the+Synthesis+of+MRNA-Encoding+Libraries+and+Discovery+of+Bioactive+Natural+Product-Inspired+Non-Traditional+Macrocyclic+Peptides&doi=10.3390%2Fmolecules18033502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Technologies for the synthesis of mRNA-encoding libraries and discovery of bioactive natural product-inspired non-traditional macrocyclic peptides</span></div><div class="casAuthors">Ito, Kenichiro; Passioura, Toby; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3502-3528</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">In this review, we discuss emerging technologies for drug discovery, which yields novel mol. scaffolds based on natural product-inspired non-traditional peptides expressed using the translation machinery.  Unlike natural products, these technologies allow for constructing mRNA-encoding libraries of macrocyclic peptides contg. non-canonical sidechains and N-methyl-modified backbones.  The complexity of sequence space in such libraries reaches as high as a trillion (>1012), affording initial hits of high affinity ligands against protein targets.  Although this article comprehensively covers several related technologies, we discuss in greater detail the tech. development and advantages of the Random non-std. Peptide Integration Discovery (RaPID) system, including the recent identification of inhibitors against various therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3aPtwP6VYNrVg90H21EOLACvtfcHk0lihEIu0F7tnMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlt1GitLw%253D&md5=71b8bd1771f6d01a7fce404ec6de8db5</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.3390%2Fmolecules18033502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules18033502%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DK.%26aulast%3DPassioura%26aufirst%3DT.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DTechnologies%2520for%2520the%2520Synthesis%2520of%2520MRNA-Encoding%2520Libraries%2520and%2520Discovery%2520of%2520Bioactive%2520Natural%2520Product-Inspired%2520Non-Traditional%2520Macrocyclic%2520Peptides%26jtitle%3DMolecules%26date%3D2013%26volume%3D18%26spage%3D3502%26epage%3D3528%26doi%3D10.3390%2Fmolecules18033502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schlippe, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Josephson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szostak, J. W.</span></span> <span> </span><span class="NLM_article-title">In Vitro Selection of Highly Modified Cyclic Peptides That Act as Tight Binding Inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">10469</span>– <span class="NLM_lpage">10477</span>, <span class="refDoi"> DOI: 10.1021/ja301017y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja301017y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A280%3ADC%252BC38vns1CqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=10469-10477&author=Y.+V.+Schlippeauthor=M.+C.+Hartmanauthor=K.+Josephsonauthor=J.+W.+Szostak&title=In+Vitro+Selection+of+Highly+Modified+Cyclic+Peptides+That+Act+as+Tight+Binding+Inhibitors&doi=10.1021%2Fja301017y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro selection of highly modified cyclic peptides that act as tight binding inhibitors</span></div><div class="casAuthors">Schlippe Yollete V Guillen; Hartman Matthew C T; Josephson Kristopher; Szostak Jack W</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">10469-77</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">There is a great demand for the discovery of new therapeutic molecules that combine the high specificity and affinity of biologic drugs with the bioavailability and lower cost of small molecules.  Small, natural-product-like peptides hold great promise in bridging this gap; however, access to libraries of these compounds has been a limitation.  Since ribosomal peptides may be subjected to in vitro selection techniques, the generation of extremely large libraries (>10(13)) of highly modified macrocyclic peptides may provide a powerful alternative for the generation and selection of new useful bioactive molecules.  Moreover, the incorporation of many non-proteinogenic amino acids into ribosomal peptides in conjunction with macrocyclization should enhance the drug-like features of these libraries.  Here we show that mRNA-display, a technique that allows the in vitro selection of peptides, can be applied to the evolution of macrocyclic peptides that contain a majority of unnatural amino acids.  We describe the isolation and characterization of two such unnatural cyclic peptides that bind the protease thrombin with low nanomolar affinity, and we show that the unnatural residues in these peptides are essential for the observed high-affinity binding.  We demonstrate that the selected peptides are tight-binding inhibitors of thrombin, with K(i)(app) values in the low nanomolar range.  The ability to evolve highly modified macrocyclic peptides in the laboratory is the first crucial step toward the facile generation of useful molecular reagents and therapeutic lead molecules that combine the advantageous features of biologics with those of small-molecule drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSohKaxZ0ovut1QT_1O5WBYfW6udTcc2eYHDAfxNxDekrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38vns1CqsQ%253D%253D&md5=8b83efc2d96fcab98ec42039f63695fb</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fja301017y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja301017y%26sid%3Dliteratum%253Aachs%26aulast%3DSchlippe%26aufirst%3DY.%2BV.%26aulast%3DHartman%26aufirst%3DM.%2BC.%26aulast%3DJosephson%26aufirst%3DK.%26aulast%3DSzostak%26aufirst%3DJ.%2BW.%26atitle%3DIn%2520Vitro%2520Selection%2520of%2520Highly%2520Modified%2520Cyclic%2520Peptides%2520That%2520Act%2520as%2520Tight%2520Binding%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D10469%26epage%3D10477%26doi%3D10.1021%2Fja301017y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joo, S.-H.</span></span> <span> </span><span class="NLM_article-title">Cyclic Peptides as Therapeutic Agents and Biochemical Tools</span>. <i>Biomol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.4062/biomolther.2012.20.1.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.4062%2Fbiomolther.2012.20.1.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC38Xns12hsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=19-26&author=S.-H.+Joo&title=Cyclic+Peptides+as+Therapeutic+Agents+and+Biochemical+Tools&doi=10.4062%2Fbiomolther.2012.20.1.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic peptides as therapeutic agents and biochemical tools</span></div><div class="casAuthors">Joo, Sang Hoon</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecules & Therapeutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-26</span>CODEN:
                <span class="NLM_cas:coden">BTIHA3</span>;
        ISSN:<span class="NLM_cas:issn">1976-9148</span>.
    
            (<span class="NLM_cas:orgname">Korean Society of Applied Pharmacology</span>)
        </div><div class="casAbstract">A review.  There are many cyclic peptides with diverse biol. activities, such as antibacterial activity, immunosuppressive activity, and anti-tumor activity, and so on.  Encouraged by natural cyclic peptides with biol. activity, efforts have been made to develop cyclic peptides with both genetic and synthetic methods.  The genetic methods include phage display, intein-based cyclic peptides, and mRNA display.  The synthetic methods involve individual synthesis, parallel synthesis, as well as split-and-pool synthesis.  Recent development of cyclic peptide library based on split-and-pool synthesis allows on-bead screening, in-soln. screening, and microarray screening of cyclic peptides for biol. activity.  Cyclic peptides will be useful as receptor agonist/antagonist, RNA binding mol., enzyme inhibitor and so on, and more cyclic peptides will emerge as therapeutic agents and biochem. tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP1NpYU4J1frVg90H21EOLACvtfcHk0lhHiQAcJmULYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xns12hsLw%253D&md5=02d12ac01606f28c812815cf9ed70f6f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.4062%2Fbiomolther.2012.20.1.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4062%252Fbiomolther.2012.20.1.019%26sid%3Dliteratum%253Aachs%26aulast%3DJoo%26aufirst%3DS.-H.%26atitle%3DCyclic%2520Peptides%2520as%2520Therapeutic%2520Agents%2520and%2520Biochemical%2520Tools%26jtitle%3DBiomol.%2520Ther.%26date%3D2012%26volume%3D20%26spage%3D19%26epage%3D26%26doi%3D10.4062%2Fbiomolther.2012.20.1.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oyelere, A. K.</span></span> <span> </span><span class="NLM_article-title">Macrocycles in Medicinal Chemistry and Drug Discovery</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1359</span>– <span class="NLM_lpage">1360</span>, <span class="refDoi"> DOI: 10.2174/156802610792232097</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.2174%2F156802610792232097" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=20919989" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFajurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=1359-1360&author=A.+K.+Oyelere&title=Macrocycles+in+Medicinal+Chemistry+and+Drug+Discovery&doi=10.2174%2F156802610792232097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in medicinal chemistry and drug discovery</span></div><div class="casAuthors">Oyelere, Adegboyega K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1359-1360</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnTtPNYBDVObVg90H21EOLACvtfcHk0lhHiQAcJmULYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFajurrJ&md5=77fe8dc96350b3cbfa5403cc45a9e8fa</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.2174%2F156802610792232097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802610792232097%26sid%3Dliteratum%253Aachs%26aulast%3DOyelere%26aufirst%3DA.%2BK.%26atitle%3DMacrocycles%2520in%2520Medicinal%2520Chemistry%2520and%2520Drug%2520Discovery%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2010%26volume%3D10%26spage%3D1359%26epage%3D1360%26doi%3D10.2174%2F156802610792232097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drahl, C.</span></span> <span> </span><span class="NLM_article-title">Big Hopes Ride on Big Rings</span>. <i>Chem. Eng. News</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">54</span>– <span class="NLM_lpage">57</span>, <span class="refDoi"> DOI: 10.1021/cen-v087n036.p054</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cen-v087n036.p054" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2009&pages=54-57&author=C.+Drahl&title=Big+Hopes+Ride+on+Big+Rings&doi=10.1021%2Fcen-v087n036.p054"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fcen-v087n036.p054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcen-v087n036.p054%26sid%3Dliteratum%253Aachs%26aulast%3DDrahl%26aufirst%3DC.%26atitle%3DBig%2520Hopes%2520Ride%2520on%2520Big%2520Rings%26jtitle%3DChem.%2520Eng.%2520News%26date%3D2009%26volume%3D87%26spage%3D54%26epage%3D57%26doi%3D10.1021%2Fcen-v087n036.p054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zorzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deyle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Cyclic Peptide Therapeutics: Past, Present and Future</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">29</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1016%2Fj.cbpa.2017.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=28249193" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsFGksLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=24-29&author=A.+Zorziauthor=K.+Deyleauthor=C.+Heinis&title=Cyclic+Peptide+Therapeutics%3A+Past%2C+Present+and+Future&doi=10.1016%2Fj.cbpa.2017.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic peptide therapeutics: past, present and future</span></div><div class="casAuthors">Zorzi, Alessandro; Deyle, Kaycie; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">24-29</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cyclic peptides combine several favorable properties such as good binding affinity, target selectivity and low toxicity that make them an attractive modality for the development of therapeutics.  Over 40 cyclic peptide drugs are currently in clin. use and around one new cyclic peptide drug enters the market every year on av.  The vast majority of clin. approved cyclic peptides are derived from natural products, such as antimicrobials or human peptide hormones.  New powerful techniques based on rational design and in vitro evolution have enabled the de novo development of cyclic peptide ligands to targets for which nature does not offer solns.  A look at the cyclic peptides currently under clin. evaluation shows that several have been developed using such techniques.  This new source for cyclic peptide ligands introduces a freshness to the field, and it is likely that de novo developed cyclic peptides will be in clin. use in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwchcMOj1bSbVg90H21EOLACvtfcHk0lhWbZcE6fL34Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsFGksLg%253D&md5=4b9581365b92b649dad8c7b22c5bd9c3</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DZorzi%26aufirst%3DA.%26aulast%3DDeyle%26aufirst%3DK.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DCyclic%2520Peptide%2520Therapeutics%253A%2520Past%252C%2520Present%2520and%2520Future%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D24%26epage%3D29%26doi%3D10.1016%2Fj.cbpa.2017.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwochert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pye, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naylor, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez
Valcarcel, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawada, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanco, M.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Stereochemistry Balances Cell Permeability and Solubility in the Naturally Derived Phepropeptin Cyclic Peptides</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">757</span>– <span class="NLM_lpage">761</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00100</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00100" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC28XpsVCltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=757-761&author=J.+Schwochertauthor=Y.+Laoauthor=C.+R.+Pyeauthor=M.+R.+Naylorauthor=P.+V.+Desaiauthor=I.+C.+Gonzalez%0AValcarcelauthor=J.+A.+Barrettauthor=G.+Sawadaauthor=M.-J.+Blancoauthor=R.+S.+Lokey&title=Stereochemistry+Balances+Cell+Permeability+and+Solubility+in+the+Naturally+Derived+Phepropeptin+Cyclic+Peptides&doi=10.1021%2Facsmedchemlett.6b00100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Stereochemistry Balances Cell Permeability and Solubility in the Naturally Derived Phepropeptin Cyclic Peptides</span></div><div class="casAuthors">Schwochert, Joshua; Lao, Yongtong; Pye, Cameron R.; Naylor, Matthew R.; Desai, Prashant V.; Gonzalez-Valcarcel, Isabel C.; Barrett, Jaclyn A.; Sawada, Geri; Blanco, Maria-Jesus; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">757-761</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclic peptide (CP) natural products provide useful model systems for mapping "beyond-Rule-of-5" (bRo5) space.  We identified the phepropeptins as natural product CPs with potential cell permeability.  Synthesis of the phepropeptins and epimeric analogs revealed much more rapid cellular permeability for the natural stereochem. pattern.  Despite being more cell permeable, the natural compds. exhibited similar aq. soly. as the corresponding epimers, a phenomenon explained by solvent-dependent conformational flexibility among the natural compds.  When analyzing the polarity of the soln. structures we found that neither the no. of hydrogen bonds nor the total polar surface area accurately represents the solvation energies of the high and low dielec. conformations.  This work adds to a growing no. of natural CPs whose solvent-dependent conformational behavior allows for a balance between aq. soly. and cell permeability, highlighting structural flexibility as an important consideration in the design of mols. in bRo5 chem. space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwMDd9pbrhsbVg90H21EOLACvtfcHk0lhWbZcE6fL34Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpsVCltLg%253D&md5=305053943e9f640caad43423e5b3b3ff</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00100%26sid%3Dliteratum%253Aachs%26aulast%3DSchwochert%26aufirst%3DJ.%26aulast%3DLao%26aufirst%3DY.%26aulast%3DPye%26aufirst%3DC.%2BR.%26aulast%3DNaylor%26aufirst%3DM.%2BR.%26aulast%3DDesai%26aufirst%3DP.%2BV.%26aulast%3DGonzalez%2BValcarcel%26aufirst%3DI.%2BC.%26aulast%3DBarrett%26aufirst%3DJ.%2BA.%26aulast%3DSawada%26aufirst%3DG.%26aulast%3DBlanco%26aufirst%3DM.-J.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DStereochemistry%2520Balances%2520Cell%2520Permeability%2520and%2520Solubility%2520in%2520the%2520Naturally%2520Derived%2520Phepropeptin%2520Cyclic%2520Peptides%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D757%26epage%3D761%26doi%3D10.1021%2Facsmedchemlett.6b00100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwochert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkeley, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponkey, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, S. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khunte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Limberakis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalgutkar, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathiowetz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Peptide to Peptoid Substitutions Increase Cell Permeability in Cyclic Hexapeptides</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2928</span>– <span class="NLM_lpage">2931</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.5b01162</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.5b01162" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2MXps1arsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=2928-2931&author=J.+Schwochertauthor=R.+Turnerauthor=M.+Thangauthor=R.+F.+Berkeleyauthor=A.+R.+Ponkeyauthor=K.+M.+Rodriguezauthor=S.+S.+F.+Leungauthor=B.+Khunteauthor=G.+Goetzauthor=C.+Limberakisauthor=A.+S.+Kalgutkarauthor=H.+Engauthor=M.+J.+Shapiroauthor=A.+M.+Mathiowetzauthor=D.+A.+Priceauthor=S.+Lirasauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Peptide+to+Peptoid+Substitutions+Increase+Cell+Permeability+in+Cyclic+Hexapeptides&doi=10.1021%2Facs.orglett.5b01162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide to Peptoid Substitutions Increase Cell Permeability in Cyclic Hexapeptides</span></div><div class="casAuthors">Schwochert, Joshua; Turner, Rushia; Thang, Melissa; Berkeley, Ray F.; Ponkey, Alexandra R.; Rodriguez, Kelsie M.; Leung, Siegfried S. F.; Khunte, Bhagyashree; Goetz, Gilles; Limberakis, Chris; Kalgutkar, Amit S.; Eng, Heather; Shapiro, Michael J.; Mathiowetz, Alan M.; Price, David A.; Liras, Spiros; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2928-2931</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The effect of peptide-to-peptoid substitutions on the passive membrane permeability of an N-methylated cyclic hexapeptide is examd.  In general, substitutions maintained permeability but increased conformational heterogeneity.  Diversification with nonproteinogenic side chains increased permeability up to 3-fold.  Addnl., the conformational impact of peptoid substitutions within a β-turn are explored.  Based on these results, the strategic incorporation of peptoid residues into cyclic peptides can maintain or improve cell permeability, while increasing access to diverse side-chain functionality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofUG1_hYdVn7Vg90H21EOLACvtfcHk0lhWbZcE6fL34Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXps1arsrs%253D&md5=156985175ddcafd34a67e81713934040</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.5b01162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.5b01162%26sid%3Dliteratum%253Aachs%26aulast%3DSchwochert%26aufirst%3DJ.%26aulast%3DTurner%26aufirst%3DR.%26aulast%3DThang%26aufirst%3DM.%26aulast%3DBerkeley%26aufirst%3DR.%2BF.%26aulast%3DPonkey%26aufirst%3DA.%2BR.%26aulast%3DRodriguez%26aufirst%3DK.%2BM.%26aulast%3DLeung%26aufirst%3DS.%2BS.%2BF.%26aulast%3DKhunte%26aufirst%3DB.%26aulast%3DGoetz%26aufirst%3DG.%26aulast%3DLimberakis%26aufirst%3DC.%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DEng%26aufirst%3DH.%26aulast%3DShapiro%26aufirst%3DM.%2BJ.%26aulast%3DMathiowetz%26aufirst%3DA.%2BM.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DPeptide%2520to%2520Peptoid%2520Substitutions%2520Increase%2520Cell%2520Permeability%2520in%2520Cyclic%2520Hexapeptides%26jtitle%3DOrg.%2520Lett.%26date%3D2015%26volume%3D17%26spage%3D2928%26epage%3D2931%26doi%3D10.1021%2Facs.orglett.5b01162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, C.</span></span> <span> </span><span class="NLM_article-title">Constrained Peptides’ Time to Shine?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">533</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1038%2Fnrd.2018.125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=30057410" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVSht7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=531-533&author=C.+Morrison&title=Constrained+Peptides%E2%80%99+Time+to+Shine%3F&doi=10.1038%2Fnrd.2018.125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Constrained peptides' time to shine?</span></div><div class="casAuthors">Morrison, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">531-533</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Constrained peptides have long tantalized drug developers with their potential ability to combine the best attributes of antibodies and small mols.  Finally, a handful of constrained peptides are in late-stage clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ9AS6knVUNrVg90H21EOLACvtfcHk0ljkYLKxrOedFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVSht7fP&md5=35e88bd4a715f26738f600598bc75ab2</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.125%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DC.%26atitle%3DConstrained%2520Peptides%25E2%2580%2599%2520Time%2520to%2520Shine%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D531%26epage%3D533%26doi%3D10.1038%2Fnrd.2018.125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warr, W. A.</span></span> <span> </span><span class="NLM_article-title">A CADD-Alog of Strategies in Pharma</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">247</span>, <span class="refDoi"> DOI: 10.1007/s10822-017-0017-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1007%2Fs10822-017-0017-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=28315993" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2sXks12msbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=245-247&author=W.+A.+Warr&title=A+CADD-Alog+of+Strategies+in+Pharma&doi=10.1007%2Fs10822-017-0017-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">A CADD-alog of strategies in pharma</span></div><div class="casAuthors">Warr, Wendy A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">245-247</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A special issue on computer-aided drug design (CADD) strategies in pharma discusses how CADD groups in different environments work.  Perspectives were collected from authors in 11 organizations: four big pharmaceutical companies, one major biotechnol. company, one smaller biotech, one private pharmaceutical company, two contract research organizations (CROs), one university, and one that spans the breadth of big pharmaceutical companies and one smaller biotech.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm15mndFA_ybVg90H21EOLACvtfcHk0ljkYLKxrOedFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXks12msbY%253D&md5=3e12bbefdad84473fc013467a32ab20a</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1007%2Fs10822-017-0017-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-017-0017-6%26sid%3Dliteratum%253Aachs%26aulast%3DWarr%26aufirst%3DW.%2BA.%26atitle%3DA%2520CADD-Alog%2520of%2520Strategies%2520in%2520Pharma%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2017%26volume%3D31%26spage%3D245%26epage%3D247%26doi%3D10.1007%2Fs10822-017-0017-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, K. L.</span></span> <span> </span><span class="NLM_article-title">Large Cyclodextrins</span>. <i>J. Inclusion Phenom. Macrocyclic Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1023/a:1020494503684</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1023%2FA%3A1020494503684" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsVKku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2002&pages=1-13&author=K.+L.+Larsen&title=Large+Cyclodextrins&doi=10.1023%2Fa%3A1020494503684"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Large Cyclodextrins</span></div><div class="casAuthors">Larsen, Kim Lambertsen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Inclusion Phenomena and Macrocyclic Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">JIPCF5</span>;
        ISSN:<span class="NLM_cas:issn">1388-3127</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A review with refs.  The existence of large cyclodextrins, cyclic α-D-(1→4) glucans with a degree of polymn. higher than eight, has been proven during the past decade.  A no. of 4-α-glucanotransferases have been shown to be able to produce large cyclodextrins consisting of up to several hundred glycosyl units, from both amylose and amylopectin.  Large cyclodextrins with degree of polymn. up to 31 have been isolated to purity by use of elaborate purifn. schemes, enabling studies of their structural and complex forming properties.  The solid state structures of the large cyclodextrins with a degree of polymn. 10, 14 and 26, resp., have revealed interesting new structural features of this family of mols.  This review summarizes the studies of the large cyclodextrins, a varied and highly interesting group of mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3FwWddKMD37Vg90H21EOLACvtfcHk0ljkYLKxrOedFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsVKku7k%253D&md5=042d7663281078400947579627e0fc1c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1023%2FA%3A1020494503684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1020494503684%26sid%3Dliteratum%253Aachs%26aulast%3DLarsen%26aufirst%3DK.%2BL.%26atitle%3DLarge%2520Cyclodextrins%26jtitle%3DJ.%2520Inclusion%2520Phenom.%2520Macrocyclic%2520Chem.%26date%3D2002%26volume%3D43%26spage%3D1%26epage%3D13%26doi%3D10.1023%2Fa%3A1020494503684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raithby, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shields, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, F. H.</span></span> <span> </span><span class="NLM_article-title">Conformational Analysis of Macrocyclic Ether Ligands. II. 1,4,7,10,13-Pentaoxacyclopentadecane and 1,4,7,10,13-Pentathiacyclopentadecane</span>. <i>Acta Crystallogr., Sect. B: Struct. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">476</span>– <span class="NLM_lpage">489</span>, <span class="refDoi"> DOI: 10.1107/s0108768196015303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1107%2FS0108768196015303" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADyaK2sXktFWitbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=476-489&author=P.+R.+Raithbyauthor=G.+P.+Shieldsauthor=F.+H.+Allen&title=Conformational+Analysis+of+Macrocyclic+Ether+Ligands.+II.+1%2C4%2C7%2C10%2C13-Pentaoxacyclopentadecane+and+1%2C4%2C7%2C10%2C13-Pentathiacyclopentadecane&doi=10.1107%2Fs0108768196015303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational analysis of macrocyclic ether ligands. II. 1,4,7,10,13-Pentaoxacyclopentadecane and 1,4,7,10,13-pentathiacyclopentadecane</span></div><div class="casAuthors">Raithby, P. R.; Shields, G. P.; Allen, F. H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section B: Structural Science</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">B53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">476-489</span>CODEN:
                <span class="NLM_cas:coden">ASBSDK</span>;
        ISSN:<span class="NLM_cas:issn">0108-7681</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">Crystallog. results retrieved from the Cambridge Structural Database (CSD) were used to perform a systematic conformational classification of free and metal-coordinated unsatd. 15-membered oxa and thia macrocycles using symmetry-modified Jarvis-Patrick cluster anal.  Relative mol. mechanics energies of the obsd. conformations are compared with the cluster populations.  With oxa donors a uniangular and a [348] conformer predominate for larger metal ions; these lie above the donor atom plane with 1-6 addnl. ligands bound on the same side.  With smaller cations an anangular conformer is adopted, the O atoms describing the equatorial plane of a pentagonal bipyramid.  Other conformers occur as dictated by the coordination environment, particularly if not all donor atoms are metal-bound; in some cases the conformation is detd. by a H-bonded network.  In some thia examples the ligand binds to an axial/apical and four equatorial sites of the coordination polyhedron; in others contg. AuI or AgI the metal is linearly or tetrahedrally coordinated with addnl. M-S interactions.  With mixed donors, the hard/soft characteristics of the metal det. the coordination mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHw7WxdXKu4bVg90H21EOLACvtfcHk0ling90vLSL-JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktFWitbg%253D&md5=dc8a6059021d4613c7f50c61fd5e53ef</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0108768196015303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0108768196015303%26sid%3Dliteratum%253Aachs%26aulast%3DRaithby%26aufirst%3DP.%2BR.%26aulast%3DShields%26aufirst%3DG.%2BP.%26aulast%3DAllen%26aufirst%3DF.%2BH.%26atitle%3DConformational%2520Analysis%2520of%2520Macrocyclic%2520Ether%2520Ligands.%2520II.%25201%252C4%252C7%252C10%252C13-Pentaoxacyclopentadecane%2520and%25201%252C4%252C7%252C10%252C13-Pentathiacyclopentadecane%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520B%253A%2520Struct.%2520Sci.%26date%3D1997%26volume%3D53%26spage%3D476%26epage%3D489%26doi%3D10.1107%2Fs0108768196015303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonnet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agrafiotis, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E.</span></span> <span> </span><span class="NLM_article-title">Conformational Analysis of Macrocycles: Finding What Common Search Methods Miss</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2242</span>– <span class="NLM_lpage">2259</span>, <span class="refDoi"> DOI: 10.1021/ci900238a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900238a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1ahtbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=2242-2259&author=P.+Bonnetauthor=D.+K.+Agrafiotisauthor=F.+Zhuauthor=E.+Martin&title=Conformational+Analysis+of+Macrocycles%3A+Finding+What+Common+Search+Methods+Miss&doi=10.1021%2Fci900238a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Analysis of Macrocycles: Finding What Common Search Methods Miss</span></div><div class="casAuthors">Bonnet, Pascal; Agrafiotis, Dimitris K.; Zhu, Fangqiang; Martin, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2242-2259</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As computational drug design becomes increasingly reliant on virtual screening and on high-throughput 3D modeling, the need for fast, robust, and reliable methods for sampling mol. conformations has become greater than ever.  Furthermore, chem. novelty is at a premium, forcing medicinal chemists to explore more complex structural motifs and unusual topologies.  This necessitates the use of conformational sampling techniques that work well in all cases.  Here, we compare the performance of several popular conformational search algorithms on three broad classes of macrocyclic mols.  These methods include Catalyst, CAESAR, MacroModel, MOE, Omega, Rubicon and two newer self-organizing algorithms known as stochastic proximity embedding (SPE) and self-organizing superimposition (SOS) that have been developed at Johnson & Johnson.  Our results show a compelling advantage for the three distance geometry methods (SOS, SPE, and Rubicon) followed to a lesser extent by MacroModel.  The remaining techniques, particularly those based on systematic search, often failed to identify any of the lowest energy conformations and are unsuitable for this class of structures.  Taken together with our previous study on drug-like mols., these results suggest that SPE and SOS are two of the most robust and universally applicable conformational search methods, with the latter being preferred because of its superior speed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhIMu1b7U477Vg90H21EOLACvtfcHk0ling90vLSL-JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1ahtbnF&md5=9b4a5919415ef0660c4f9ff18d9b76f4</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fci900238a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900238a%26sid%3Dliteratum%253Aachs%26aulast%3DBonnet%26aufirst%3DP.%26aulast%3DAgrafiotis%26aufirst%3DD.%2BK.%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DMartin%26aufirst%3DE.%26atitle%3DConformational%2520Analysis%2520of%2520Macrocycles%253A%2520Finding%2520What%2520Common%2520Search%2520Methods%2520Miss%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D2242%26epage%3D2259%26doi%3D10.1021%2Fci900238a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reetz, M. T.</span></span> <span> </span><span class="NLM_article-title">Utility of B-Factors in Protein Science: Interpreting Rigidity, Flexibility, and Internal Motion and Engineering Thermostability</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">1626</span>– <span class="NLM_lpage">1665</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.8b00290</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00290" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVSit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2019&pages=1626-1665&author=Z.+Sunauthor=Q.+Liuauthor=G.+Quauthor=Y.+Fengauthor=M.+T.+Reetz&title=Utility+of+B-Factors+in+Protein+Science%3A+Interpreting+Rigidity%2C+Flexibility%2C+and+Internal+Motion+and+Engineering+Thermostability&doi=10.1021%2Facs.chemrev.8b00290"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Utility of B-Factors in Protein Science: Interpreting Rigidity, Flexibility, and Internal Motion and Engineering Thermostability</span></div><div class="casAuthors">Sun, Zhoutong; Liu, Qian; Qu, Ge; Feng, Yan; Reetz, Manfred T.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1626-1665</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The term B-factor, sometimes called the Debye-Waller factor, temp. factor, or at. displacement parameter, is used in protein crystallog. to describe the attenuation of X-ray or neutron scattering caused by thermal motion.  This review begins with analyses of early protein studies which suggested that B-factors, available from the Protein Data Bank, can be used to identify the flexibility of atoms, side chains, or even whole regions.  This requires a technique for obtaining normalized B-factors.  Since then the exploitation of B-factors has been extensively elaborated and applied in a variety of studies with quite different goals, all having in common the identification and interpretation of rigidity, flexibility, and/or internal motion which are crucial in enzymes and in proteins in general.  Importantly, this review includes a discussion of limitations and possible pitfalls when using B-factors.  A second research area, which likewise exploits B-factors, is also reviewed, namely, the development of the so-called B-FIT-directed evolution method for increasing the thermostability of enzymes as catalysts in org. chem. and biotechnol.  In both research areas, a max. of structural and mechanistic insights is gained when B-factor analyses are combined with other exptl. and computational techniques.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB4_fJMa4APbVg90H21EOLACvtfcHk0ling90vLSL-JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVSit78%253D&md5=9bcf5fb15301278e8a13f250d38cad54</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00290%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DQu%26aufirst%3DG.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DReetz%26aufirst%3DM.%2BT.%26atitle%3DUtility%2520of%2520B-Factors%2520in%2520Protein%2520Science%253A%2520Interpreting%2520Rigidity%252C%2520Flexibility%252C%2520and%2520Internal%2520Motion%2520and%2520Engineering%2520Thermostability%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26spage%3D1626%26epage%3D1665%26doi%3D10.1021%2Facs.chemrev.8b00290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otero-Ramirez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passioura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Structural Features and Binding Modes of Thioether-Cyclized Peptide Ligands</span>. <i>Biomedicines</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">116</span>, <span class="refDoi"> DOI: 10.3390/biomedicines6040116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.3390%2Fbiomedicines6040116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1WjsbrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=116&author=M.+Otero-Ramirezauthor=T.+Passiouraauthor=H.+Suga&title=Structural+Features+and+Binding+Modes+of+Thioether-Cyclized+Peptide+Ligands&doi=10.3390%2Fbiomedicines6040116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Structural features and binding modes of thioether-cyclized peptide ligands</span></div><div class="casAuthors">Otero-Ramirez, Manuel E.; Passioura, Toby; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Biomedicines</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">116</span>CODEN:
                <span class="NLM_cas:coden">BIOMID</span>;
        ISSN:<span class="NLM_cas:issn">2227-9059</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Macrocyclic peptides are an emerging class of bioactive compds. for therapeutic use.  In part, this is because they are capable of high potency and excellent target affinity and selectivity.  Over the last decade, several biochem. techniques have been developed for the identification of bioactive macrocyclic peptides, allowing for the rapid isolation of high affinity ligands to a target of interest.  A common feature of these techniques is a general reliance on thioether formation to effect macrocyclization.  Increasingly, the compds. identified using these approaches have been subjected to x-ray crystallog. anal. bound to their resp. targets, providing detailed structural information about their conformation and mechanism of target binding.  The present review provides an overview of the target bound thioether-closed macrocyclic peptide structures that have been obtained to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-pfa5vlPS97Vg90H21EOLACvtfcHk0ljw6ugUmhIDpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1WjsbrF&md5=447cd5c4ca6d10ba1fa2cc423b2b2184</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.3390%2Fbiomedicines6040116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiomedicines6040116%26sid%3Dliteratum%253Aachs%26aulast%3DOtero-Ramirez%26aufirst%3DM.%26aulast%3DPassioura%26aufirst%3DT.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DStructural%2520Features%2520and%2520Binding%2520Modes%2520of%2520Thioether-Cyclized%2520Peptide%2520Ligands%26jtitle%3DBiomedicines%26date%3D2018%26volume%3D6%26spage%3D116%26doi%3D10.3390%2Fbiomedicines6040116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malde, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Crystal Structures of Protein-Bound Cyclic Peptides</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">9861</span>– <span class="NLM_lpage">9914</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.8b00807</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00807" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos1Wltrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2019&pages=9861-9914&author=A.+K.+Maldeauthor=T.+A.+Hillauthor=A.+Iyerauthor=D.+P.+Fairlie&title=Crystal+Structures+of+Protein-Bound+Cyclic+Peptides&doi=10.1021%2Facs.chemrev.8b00807"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of protein-bound cyclic peptides</span></div><div class="casAuthors">Malde, Alpeshkumar K.; Hill, Timothy A.; Iyer, Abishek; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9861-9914</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Cyclization is an important post-translational modification of peptides and proteins that confers key advantages such as protection from proteolytic degrdn., altered soly., membrane permeability, bioavailability, and esp. restricted conformational freedom in water that allows the peptide backbone to adopt the major secondary structure elements found in proteins.  Non-ribosomal synthesis in bacteria, fungi, and plants or synthetic chem. can introduce unnatural amino acids and non-peptidic constraints that modify peptide backbones and side chains to fine-tune cyclic peptide structure.  Structures can be potentially altered further upon binding to a protein in biol. environments.  Here we analyze three-dimensional crystal structures for 211 bioactive cyclic peptides bound to 65 different proteins.  The protein-bound cyclic peptides were examd. for similarities and differences in bonding modes, for main-chain and side-chain structure, and for the importance of polarity, hydrogen bonds, hydrophobic effects, and water mols. in interactions with proteins.  Many protein-bound cyclic peptides show backbone structures like those (strands, sheets, turns, helixes, loops, or distorted variations) found at protein-protein binding interfaces.  However, the notion of macrocycles simply as privileged scaffolds that primarily project side-chain substituents for complementary interactions with proteins is dispelled here.  Unlike small-mol. drugs, the cyclic peptides do not rely mainly upon hydrophobic and van der Waals interactions for protein binding; they also use their main chain and side chains to form polar contacts and hydrogen bonds with proteins.  Compared to small-mol. ligands, cyclic peptides can bind across larger, polar, and water-exposed protein surface areas, making many more contacts that can increase affinity, selectivity, biol. activity, and ligand-receptor residence time.  Cyclic peptides have a greater capacity than small-mol. drugs to modulate protein-protein interfaces that involve large, shallow, dynamic, polar, and water-exposed protein surfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnAtNoQP4lbbVg90H21EOLACvtfcHk0ljw6ugUmhIDpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos1Wltrg%253D&md5=38614ab1898f056757a3671554505a24</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00807&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00807%26sid%3Dliteratum%253Aachs%26aulast%3DMalde%26aufirst%3DA.%2BK.%26aulast%3DHill%26aufirst%3DT.%2BA.%26aulast%3DIyer%26aufirst%3DA.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DCrystal%2520Structures%2520of%2520Protein-Bound%2520Cyclic%2520Peptides%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26spage%3D9861%26epage%3D9914%26doi%3D10.1021%2Facs.chemrev.8b00807" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guéret, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adihou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalakrishnan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemurell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span></span> <span> </span><span class="NLM_article-title">New Modalities for Challenging Targets in Drug Discovery</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">10294</span>– <span class="NLM_lpage">10323</span>, <span class="refDoi"> DOI: 10.1002/anie.201611914</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fanie.201611914" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1WlsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=10294-10323&author=E.+Valeurauthor=S.+M.+Gu%C3%A9retauthor=H.+Adihouauthor=R.+Gopalakrishnanauthor=M.+Lemurellauthor=H.+Waldmannauthor=T.+N.+Grossmannauthor=A.+T.+Plowright&title=New+Modalities+for+Challenging+Targets+in+Drug+Discovery&doi=10.1002%2Fanie.201611914"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">New Modalities for Challenging Targets in Drug Discovery</span></div><div class="casAuthors">Valeur, Eric; Gueret, Stephanie M.; Adihou, Helene; Gopalakrishnan, Ranganath; Lemurell, Malin; Waldmann, Herbert; Grossmann, Tom N.; Plowright, Alleyn T.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">10294-10323</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The ever-increasing understanding of biol. systems is providing a range of exciting novel biol. targets, whose modulation may enable novel therapeutic options for many diseases.  These targets include protein-protein and protein-nucleic acid interactions, which are, however, often refractory to classical small-mol. approaches.  Other types of mols., or modalities, are therefore required to address these targets, which has led several academic research groups and pharmaceutical companies to increasingly use the concept of so-called "new modalities".  This review defines for the first time the scope of this term, which includes novel peptidic scaffolds, oligonucleotides, hybrids, mol. conjugates, as well as new uses of classical small mols.  The authors provide the most representative examples of these modalities to target large binding surface areas such as those found in protein-protein interactions and for biol. processes at the center of cell regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3BpAx5oKYa7Vg90H21EOLACvtfcHk0ljw6ugUmhIDpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1WlsLvP&md5=e09ef5bb8dbbf09c05b1a2340aa431ec</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611914&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611914%26sid%3Dliteratum%253Aachs%26aulast%3DValeur%26aufirst%3DE.%26aulast%3DGu%25C3%25A9ret%26aufirst%3DS.%2BM.%26aulast%3DAdihou%26aufirst%3DH.%26aulast%3DGopalakrishnan%26aufirst%3DR.%26aulast%3DLemurell%26aufirst%3DM.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26atitle%3DNew%2520Modalities%2520for%2520Challenging%2520Targets%2520in%2520Drug%2520Discovery%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D10294%26epage%3D10323%26doi%3D10.1002%2Fanie.201611914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zivanovic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colizzi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hospital, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soliva, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orozco, M.</span></span> <span> </span><span class="NLM_article-title">Exploring the Conformational Landscape of Bioactive Small Molecules</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">6575</span>– <span class="NLM_lpage">6585</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.0c00304</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.0c00304" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFOis7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=6575-6585&author=S.+Zivanovicauthor=F.+Colizziauthor=D.+Morenoauthor=A.+Hospitalauthor=R.+Solivaauthor=M.+Orozco&title=Exploring+the+Conformational+Landscape+of+Bioactive+Small+Molecules&doi=10.1021%2Facs.jctc.0c00304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring the Conformational Landscape of Bioactive Small Molecules</span></div><div class="casAuthors">Zivanovic, Sanja; Colizzi, Francesco; Moreno, David; Hospital, Adam; Soliva, Robert; Orozco, Modesto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">6575-6585</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">By using a combination of classical Hamiltonian replica exchange with high-level quantum mech. calcns. on more than one hundred drug-like mols., we explored here the energy cost assocd. with binding of drug-like mols. to target macromols.  We found that, in general, the drug-like mols. present bound to proteins in the Protein Data Bank (PDB) can access easily the bioactive conformation and in fact for 73% of the studied mols. the "bioactive" conformation is within 3kBT from the most-stable conformation in soln. as detd. by DFT/SCRF calcns.  Cases with large differences between the most-stable and the bioactive conformations appear in ligands recognized by ionic contacts, or very large structures establishing many favorable interactions with the protein.  There are also a few cases where we obsd. a non-negligible uncertainty related to the exptl. structure deposited in PDB.  Remarkably, the rough automatic force field used here provides reasonable ests. of the conformational ensemble of drugs in soln.  The outlined protocol can be used to better est. the cost of adopting the bioactive conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXCdIaj3lGALVg90H21EOLACvtfcHk0ljw6ugUmhIDpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFOis7jE&md5=73de708c7dce6920169e9f5c6547272d</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.0c00304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.0c00304%26sid%3Dliteratum%253Aachs%26aulast%3DZivanovic%26aufirst%3DS.%26aulast%3DColizzi%26aufirst%3DF.%26aulast%3DMoreno%26aufirst%3DD.%26aulast%3DHospital%26aufirst%3DA.%26aulast%3DSoliva%26aufirst%3DR.%26aulast%3DOrozco%26aufirst%3DM.%26atitle%3DExploring%2520the%2520Conformational%2520Landscape%2520of%2520Bioactive%2520Small%2520Molecules%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2020%26volume%3D16%26spage%3D6575%26epage%3D6585%26doi%3D10.1021%2Facs.jctc.0c00304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleves, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherer, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reibarkh, M. Y.</span></span> <span> </span><span class="NLM_article-title">Complex Macrocycle Exploration: Parallel, Heuristic, and Constraint-Based Conformer Generation Using ForceGen</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.1007/s10822-019-00203-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1007%2Fs10822-019-00203-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=31054028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovVOkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=531-558&author=A.+N.+Jainauthor=A.+E.+Clevesauthor=Q.+Gaoauthor=X.+Wangauthor=Y.+Liuauthor=E.+C.+Shererauthor=M.+Y.+Reibarkh&title=Complex+Macrocycle+Exploration%3A+Parallel%2C+Heuristic%2C+and+Constraint-Based+Conformer+Generation+Using+ForceGen&doi=10.1007%2Fs10822-019-00203-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Complex macrocycle exploration: parallel, heuristic, and constraint-based conformer generation using ForceGen</span></div><div class="casAuthors">Jain, Ajay N.; Cleves, Ann E.; Gao, Qi; Wang, Xiao; Liu, Yizhou; Sherer, Edward C.; Reibarkh, Mikhail Y.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">531-558</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">ForceGen is a template-free, non-stochastic approach for 2D to 3D structure generation and conformational elaboration for small mols., including both non-macrocycles and macrocycles.  For conformational search of non-macrocycles, ForceGen is both faster and more accurate than the best of all tested methods on a very large, independently curated benchmark of 2859 PDB ligands.  In this study, the primary results are on macrocycles, including results for 431 unique examples from four sep. benchmarks.  These include complex peptide and peptide-like cases that can form networks of internal hydrogen bonds.  By making use of new phys. movements ("flips" of near-linear sub-cycles and explicit formation of hydrogen bonds), ForceGen exhibited statistically significantly better performance for overall RMS deviation from exptl. coordinates than all other approaches.  The algorithmic approach offers natural parallelization across multiple computing-cores.  On a modest multi-core workstation, for all but the most complex macrocycles, median wall-clock times were generally under a minute in fast search mode and under 2 min using thorough search.  On the most complex cases (roughly cyclic decapeptides and larger) explicit exploration of likely hydrogen bonding networks yielded marked improvements, but with calcn. times increasing to several minutes and in some cases to roughly an hour for fast search.  In complex cases, utilization of NMR data to constrain conformational search produces accurate conformational ensembles representative of soln. state macrocycle behavior.  On macrocycles of typical complexity (up to 21 rotatable macrocyclic and exocyclic bonds), design-focused macrocycle optimization can be practically supported by computational chem. at interactive time-scales, with conformational ensemble accuracy equaling what is seen with non-macrocyclic ligands.  For more complex macrocycles, inclusion of sparse biophys. data is a helpful adjunct to computation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIA5XjyJWMwLVg90H21EOLACvtfcHk0lhARy7t5GfTPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovVOkurg%253D&md5=8fb9b7aa12c3dbbca57ebc4265a4a204</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1007%2Fs10822-019-00203-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-019-00203-1%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26aulast%3DCleves%26aufirst%3DA.%2BE.%26aulast%3DGao%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DSherer%26aufirst%3DE.%2BC.%26aulast%3DReibarkh%26aufirst%3DM.%2BY.%26atitle%3DComplex%2520Macrocycle%2520Exploration%253A%2520Parallel%252C%2520Heuristic%252C%2520and%2520Constraint-Based%2520Conformer%2520Generation%2520Using%2520ForceGen%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2019%26volume%3D33%26spage%3D531%26epage%3D558%26doi%3D10.1007%2Fs10822-019-00203-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleves, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. N.</span></span> <span> </span><span class="NLM_article-title">ForceGen 3D Structure and Conformer Generation: From Small Lead-like Molecules to Macrocyclic Drugs</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">419</span>– <span class="NLM_lpage">439</span>, <span class="refDoi"> DOI: 10.1007/s10822-017-0015-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1007%2Fs10822-017-0015-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=28289981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFymtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=419-439&author=A.+E.+Clevesauthor=A.+N.+Jain&title=ForceGen+3D+Structure+and+Conformer+Generation%3A+From+Small+Lead-like+Molecules+to+Macrocyclic+Drugs&doi=10.1007%2Fs10822-017-0015-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">ForceGen 3D structure and conformer generation: from small lead-like molecules to macrocyclic drugs</span></div><div class="casAuthors">Cleves, Ann E.; Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">419-439</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">We introduce the ForceGen method for 3D structure generation and conformer elaboration of drug-like small mols.  ForceGen is novel, avoiding use of distance geometry, mol. templates, or simulation-oriented stochastic sampling.  The method is primarily driven by the mol. force field, implemented using an extension of MMFF94s and a partial charge estimator based on electronegativity-equalization.  The force field is coupled to algorithms for direct sampling of realistic phys. movements made by small mols.  Results are presented on a std. benchmark from the Cambridge Crystallog. Database of 480 drug-like small mols., including full structure generation from SMILES strings.  Reprodn. of protein-bound crystallog. ligand poses is demonstrated on four carefully curated data sets: the ConfGen Set (667 ligands), the PINC cross-docking benchmark (1062 ligands), a large set of macrocyclic ligands (182 total with typical ring sizes of 12-23 atoms), and a commonly used benchmark for evaluating macrocycle conformer generation (30 ligands total).  Results compare favorably to alternative methods, and performance on macrocyclic compds. approaches that obsd. on non-macrocycles while yielding a roughly 100-fold speed improvement over alternative MD-based methods with comparable performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFpG4EEfPBG7Vg90H21EOLACvtfcHk0lhARy7t5GfTPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFymtLg%253D&md5=917729bd50c89ef5bfd73d347c8ebf53</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1007%2Fs10822-017-0015-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-017-0015-8%26sid%3Dliteratum%253Aachs%26aulast%3DCleves%26aufirst%3DA.%2BE.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DForceGen%25203D%2520Structure%2520and%2520Conformer%2520Generation%253A%2520From%2520Small%2520Lead-like%2520Molecules%2520to%2520Macrocyclic%2520Drugs%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2017%26volume%3D31%26spage%3D419%26epage%3D439%26doi%3D10.1007%2Fs10822-017-0015-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perola, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charifson, P. S.</span></span> <span> </span><span class="NLM_article-title">Conformational Analysis of Drug-Like Molecules Bound to Proteins: An Extensive Study of Ligand Reorganization upon Binding</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">2499</span>– <span class="NLM_lpage">2510</span>, <span class="refDoi"> DOI: 10.1021/jm030563w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030563w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD2cXivVans78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2499-2510&author=E.+Perolaauthor=P.+S.+Charifson&title=Conformational+Analysis+of+Drug-Like+Molecules+Bound+to+Proteins%3A+An+Extensive+Study+of+Ligand+Reorganization+upon+Binding&doi=10.1021%2Fjm030563w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Analysis of Drug-Like Molecules Bound to Proteins: An Extensive Study of Ligand Reorganization upon Binding</span></div><div class="casAuthors">Perola, Emanuele; Charifson, Paul S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2499-2510</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This paper describes a large-scale study on the nature and the energetics of the conformational changes drug-like mols. experience upon binding.  Ligand strain energies and conformational reorganization were analyzed with different computational methods on 150 crystal structures of pharmaceutically relevant protein-ligand complexes.  The common knowledge that ligands rarely bind in their lowest calcd. energy conformation was confirmed.  Addnl., the authors found that over 60% of the ligands do not bind in a local min. conformation.  While approx. 60% of the ligands were calcd. to bind with strain energies lower than 5 kcal/mol, strain energies over 9 kcal/mol were calcd. in at least 10% of the cases regardless of the method used.  A clear correlation was found between acceptable strain energy and ligand flexibility, while there was no correlation between strain energy and binding affinity, thus indicating that expensive conformational rearrangements can be tolerated in some cases without overly penalizing the tightness of binding.  On the basis of the trends obsd., thresholds for the acceptable strain energies of bioactive conformations were defined with consideration of the impact of ligand flexibility.  An anal. of the degree of folding of the bound ligands confirmed the general tendency of small mols. to bind in an extended conformation.  The results suggest that the unfolding of hydrophobic ligands during binding, which exposes hydrophobic surfaces to contact with protein residues, could be one of the factors accounting for high reorganization energies.  Finally, different methods for conformational anal. were evaluated, and guidelines were defined to maximize the prevalence of bioactive conformations in computationally generated ensembles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMrN0dFKglcbVg90H21EOLACvtfcHk0lhARy7t5GfTPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXivVans78%253D&md5=8f54a8285ef1ffc5e1a19f748f4674bf</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fjm030563w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030563w%26sid%3Dliteratum%253Aachs%26aulast%3DPerola%26aufirst%3DE.%26aulast%3DCharifson%26aufirst%3DP.%2BS.%26atitle%3DConformational%2520Analysis%2520of%2520Drug-Like%2520Molecules%2520Bound%2520to%2520Proteins%253A%2520An%2520Extensive%2520Study%2520of%2520Ligand%2520Reorganization%2520upon%2520Binding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2499%26epage%3D2510%26doi%3D10.1021%2Fjm030563w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boström, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norrby, P.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liljefors, T.</span></span> <span> </span><span class="NLM_article-title">Conformational Energy Penalties of Protein-Bound Ligands</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">383</span>– <span class="NLM_lpage">396</span>, <span class="refDoi"> DOI: 10.1023/a:1008007507641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1023%2FA%3A1008007507641" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=9777496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADyaK1cXms1elsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1998&pages=383-396&author=J.+Bostr%C3%B6mauthor=P.-O.+Norrbyauthor=T.+Liljefors&title=Conformational+Energy+Penalties+of+Protein-Bound+Ligands&doi=10.1023%2Fa%3A1008007507641"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational energy penalties of protein-bound ligands</span></div><div class="casAuthors">Bostrom, Jonas; Norrby, Per-Ola; Liljefors, Tommy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">383-396</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">The conformational energies required for ligands to adopt their bioactive conformations were calcd. for 33 ligand-protein complexes including 28 different ligands.  In order to monitor the force field dependence of the results, two force fields, MM3* and AMBER*, were employed for the calcns.  Conformational analyses were performed in vacuo and in aq. soln. by using the generalized Born/solvent accessible surface (GB/SA) solvation model.  The protein-bound conformations were relaxed by using flat-bottomed Cartesian constraints.  For about 70% of the ligand-protein complexes studied, the conformational energies of the bioactive conformations were calcd. to be ≤3 kcal/mol.  It is demonstrated that the aq. conformational ensemble for the unbound ligand must be used as a ref. state in this type of calcns.  The calcns. for the ligand-protein complexes with conformational energy penalties of the ligand calcd. to be larger than 3 kcal/mol suffer from uncertainties in the interpretation of the exptl. data or limitations of the computational methods.  For example, in the case of long-chain flexible ligands (e.g. fatty acids), it is demonstrated that several conformations may be found which are very similar to the conformation detd. by x-ray crystallog. and which display significantly lower conformational energy penalties for binding than obtained by using the exptl. conformation.  For strongly polar mols., e.g. amino acids, the results indicate that further developments of the force fields and of the dielec. continuum solvation model are required for reliable calcns. on the conformational properties of this type of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOYXSRdHvDEbVg90H21EOLACvtfcHk0lgpzEK1T94FBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXms1elsr4%253D&md5=d3108c97e8b080d5567b65653f9eff0d</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008007507641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008007507641%26sid%3Dliteratum%253Aachs%26aulast%3DBostr%25C3%25B6m%26aufirst%3DJ.%26aulast%3DNorrby%26aufirst%3DP.-O.%26aulast%3DLiljefors%26aufirst%3DT.%26atitle%3DConformational%2520Energy%2520Penalties%2520of%2520Protein-Bound%2520Ligands%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D1998%26volume%3D12%26spage%3D383%26epage%3D396%26doi%3D10.1023%2Fa%3A1008007507641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peach, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cachau, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicklaus, M. C.</span></span> <span> </span><span class="NLM_article-title">Conformational Energy Range of Ligands in Protein Crystal Structures: The Difficult Quest for Accurate Understanding</span>. <i>J. Mol. Recognit.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>30</i></span>, <span class="NLM_elocation-id">e2618</span> <span class="refDoi"> DOI: 10.1002/jmr.2618</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fjmr.2618" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2017&author=M.+L.+Peachauthor=R.+E.+Cachauauthor=M.+C.+Nicklaus&title=Conformational+Energy+Range+of+Ligands+in+Protein+Crystal+Structures%3A+The+Difficult+Quest+for+Accurate+Understanding&doi=10.1002%2Fjmr.2618"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1002%2Fjmr.2618&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmr.2618%26sid%3Dliteratum%253Aachs%26aulast%3DPeach%26aufirst%3DM.%2BL.%26aulast%3DCachau%26aufirst%3DR.%2BE.%26aulast%3DNicklaus%26aufirst%3DM.%2BC.%26atitle%3DConformational%2520Energy%2520Range%2520of%2520Ligands%2520in%2520Protein%2520Crystal%2520Structures%253A%2520The%2520Difficult%2520Quest%2520for%2520Accurate%2520Understanding%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D2017%26volume%3D30%26doi%3D10.1002%2Fjmr.2618" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Butler, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luque, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barril, X.</span></span> <span> </span><span class="NLM_article-title">Toward Accurate Relative Energy Predictions of the Bioactive Conformation of Drugs</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">601</span>– <span class="NLM_lpage">610</span>, <span class="refDoi"> DOI: 10.1002/jcc.21087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fjcc.21087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=18711721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD1MXit1Knsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=601-610&author=K.+T.+Butlerauthor=F.+J.+Luqueauthor=X.+Barril&title=Toward+Accurate+Relative+Energy+Predictions+of+the+Bioactive+Conformation+of+Drugs&doi=10.1002%2Fjcc.21087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Toward accurate relative energy predictions of the bioactive conformation of drugs</span></div><div class="casAuthors">Butler, Keith T.; Luque, F. Javier; Barril, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">601-610</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Quantifying the relative energy of a ligand in its target-bound state (i.e. the bioactive conformation) is essential to understand the process of mol. recognition, to optimize the potency of bioactive mols. and to increase the accuracy of structure-based drug design methods.  This is, nevertheless, seriously hampered by 2 inter-related issues, namely the difficulty in carrying out an exhaustive sampling of the conformational space and the shortcomings of the energy functions, usually based on parametric methods of limited accuracy.  Matters are further complicated by the exptl. uncertainty on the at. coordinates, which precludes a univocal definition of the bioactive conformation.  In this article the authors investigate the relative energy of bioactive conformations introducing 2 major improvements over previous studies: the use sophisticated QM-based methods to take into account both the internal energy of the ligand and the solvation effect, and the application of phys. meaningful constraints to refine the bioactive conformation.  On a set of 99 drug-like mols., the authors find that, contrary to previous observations, 2 thirds of bioactive conformations lie within 0.5 kcal mol-1 of a local min., with penalties above 2.0 kcal mol-1 being generally attributable to structural detn. inaccuracies.  The methodol. herein described opens the door to obtain quant. ests. of the energy of bioactive conformations and can be used both as an aid in refining crystallog. structures and as a tool in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoukwCddY9zl7Vg90H21EOLACvtfcHk0lgpzEK1T94FBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXit1Knsrw%253D&md5=31ddffa90fb7f61e96b3b97ddeaac367</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21087%26sid%3Dliteratum%253Aachs%26aulast%3DButler%26aufirst%3DK.%2BT.%26aulast%3DLuque%26aufirst%3DF.%2BJ.%26aulast%3DBarril%26aufirst%3DX.%26atitle%3DToward%2520Accurate%2520Relative%2520Energy%2520Predictions%2520of%2520the%2520Bioactive%2520Conformation%2520of%2520Drugs%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D601%26epage%3D610%26doi%3D10.1002%2Fjcc.21087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borbulevych, O. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westerhoff, L. M.</span></span> <span> </span><span class="NLM_article-title">The Critical Role of QM/MM X-Ray Refinement and Accurate Tautomer/Protomer Determination in Structure-Based Drug Design</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">19</span>, <span class="refDoi"> DOI: 10.1007/s10822-020-00354-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1007%2Fs10822-020-00354-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=31792884" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=1-19&author=O.+Y.+Borbulevychauthor=R.+I.+Martinauthor=L.+M.+Westerhoff&title=The+Critical+Role+of+QM%2FMM+X-Ray+Refinement+and+Accurate+Tautomer%2FProtomer+Determination+in+Structure-Based+Drug+Design&doi=10.1007%2Fs10822-020-00354-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1007%2Fs10822-020-00354-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-020-00354-6%26sid%3Dliteratum%253Aachs%26aulast%3DBorbulevych%26aufirst%3DO.%2BY.%26aulast%3DMartin%26aufirst%3DR.%2BI.%26aulast%3DWesterhoff%26aufirst%3DL.%2BM.%26atitle%3DThe%2520Critical%2520Role%2520of%2520QM%252FMM%2520X-Ray%2520Refinement%2520and%2520Accurate%2520Tautomer%252FProtomer%2520Determination%2520in%2520Structure-Based%2520Drug%2520Design%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2020%26spage%3D1%26epage%3D19%26doi%3D10.1007%2Fs10822-020-00354-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janowski, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">York, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span> <span> </span><span class="NLM_article-title">Improved Ligand Geometries in Crystallographic Refinement Using AFITT in PHENIX</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">1062</span>– <span class="NLM_lpage">1072</span>, <span class="refDoi"> DOI: 10.1107/s2059798316012225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1107%2FS2059798316012225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVOhtr7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2016&pages=1062-1072&author=P.+A.+Janowskiauthor=N.+W.+Moriartyauthor=B.+P.+Kelleyauthor=D.+A.+Caseauthor=D.+M.+Yorkauthor=P.+D.+Adamsauthor=G.+L.+Warren&title=Improved+Ligand+Geometries+in+Crystallographic+Refinement+Using+AFITT+in+PHENIX&doi=10.1107%2Fs2059798316012225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Improved ligand geometries in crystallographic refinement using AFITT in PHENIX</span></div><div class="casAuthors">Janowski, Pawel A.; Moriarty, Nigel W.; Kelley, Brian P.; Case, David A.; York, Darrin M.; Adams, Paul D.; Warren, Gregory L.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1062-1072</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Modern crystal structure refinement programs rely on geometry restraints to overcome the challenge of a low data-to-parameter ratio.  While the classical Engh and Huber restraints work well for std. amino-acid residues, the chem. complexity of small-mol. ligands presents a particular challenge.  Most current approaches either limit ligand restraints to those that can be readily described in the Crystallog. Information File (CIF) format, thus sacrificing chem. flexibility and energetic accuracy, or they employ protocols that substantially lengthen the refinement time, potentially hindering rapid automated refinement workflows.  PHENIX-AFITT refinement uses a full mol.-mechanics force field for user-selected small-mol. ligands during refinement, eliminating the potentially difficult problem of finding or generating high-quality geometry restraints.  It is fully integrated with a std. refinement protocol and requires practically no addnl. steps from the user, making it ideal for high-throughput workflows.  PHENIX-AFITT refinements also handle multiple ligands in a single model, alternate conformations and covalently bound ligands.  Here, the results of combining AFITT and the PHENIX software suite on a data set of 189 protein-ligand PDB structures are presented.  Refinements using PHENIX-AFITT significantly reduce ligand conformational energy and lead to improved geometries without detriment to the fit to the exptl. data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAQdfkCh2PTbVg90H21EOLACvtfcHk0lgs3Z6l5vuqiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVOhtr7J&md5=648534959c1ec463fc63856e56701988</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1107%2FS2059798316012225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798316012225%26sid%3Dliteratum%253Aachs%26aulast%3DJanowski%26aufirst%3DP.%2BA.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DKelley%26aufirst%3DB.%2BP.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DYork%26aufirst%3DD.%2BM.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26atitle%3DImproved%2520Ligand%2520Geometries%2520in%2520Crystallographic%2520Refinement%2520Using%2520AFITT%2520in%2520PHENIX%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2016%26volume%3D72%26spage%3D1062%26epage%3D1072%26doi%3D10.1107%2Fs2059798316012225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleves, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brueckner, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesburg, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherer, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reibarkh, M. Y.</span></span> <span> </span><span class="NLM_article-title">XGen: Real-Space Fitting of Complex Ligand Conformational Ensembles to X-Ray Electron Density Maps</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">10509</span>– <span class="NLM_lpage">10528</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01373</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01373" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslClsbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=10509-10528&author=A.+N.+Jainauthor=A.+E.+Clevesauthor=A.+C.+Bruecknerauthor=C.+A.+Lesburgauthor=Q.+Dengauthor=E.+C.+Shererauthor=M.+Y.+Reibarkh&title=XGen%3A+Real-Space+Fitting+of+Complex+Ligand+Conformational+Ensembles+to+X-Ray+Electron+Density+Maps&doi=10.1021%2Facs.jmedchem.0c01373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">XGen: Real-Space Fitting of Complex Ligand Conformational Ensembles to X-ray Electron Density Maps</span></div><div class="casAuthors">Jain, Ajay N.; Cleves, Ann E.; Brueckner, Alexander C.; Lesburg, Charles A.; Deng, Qiaolin; Sherer, Edward C.; Reibarkh, Mikhail Y.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10509-10528</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report a new method for X-ray d. ligand fitting and refinement that is suitable for a wide variety of small-mol. ligands, including macrocycles.  The approach (called "xGen") augments a force field energy calcn. with an electron d. fitting restraint that yields an energy reward during the restrained conformational search.  The resulting conformer pools balance goodness-of-fit with ligand strain.  Real-space refinement from pre-existing ligand coordinates of 150 macrocycles resulted in occupancy-weighted conformational ensembles that exhibited low strain energy.  The xGen ensembles improved upon electron d. fit compared with the PDB ref. coordinates without making use of atom-specific B-factors.  Similarly, on nonmacrocycles, de novo fitting produced occupancy-weighted ensembles of many conformers that were generally better-quality d. fits than the deposited primary/alternate conformational pairs.  The results suggest ubiquitous low-energy ligand conformational ensembles in X-ray diffraction data and provide an alternative to using B-factors as model parameters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraxQ4QG2Vcd7Vg90H21EOLACvtfcHk0lgs3Z6l5vuqiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslClsbbF&md5=5ee4bb4268777ab8fbdcaf8adab2a586</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01373%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26aulast%3DCleves%26aufirst%3DA.%2BE.%26aulast%3DBrueckner%26aufirst%3DA.%2BC.%26aulast%3DLesburg%26aufirst%3DC.%2BA.%26aulast%3DDeng%26aufirst%3DQ.%26aulast%3DSherer%26aufirst%3DE.%2BC.%26aulast%3DReibarkh%26aufirst%3DM.%2BY.%26atitle%3DXGen%253A%2520Real-Space%2520Fitting%2520of%2520Complex%2520Ligand%2520Conformational%2520Ensembles%2520to%2520X-Ray%2520Electron%2520Density%2520Maps%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D10509%26epage%3D10528%26doi%3D10.1021%2Facs.jmedchem.0c01373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schade, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bazin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bent, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannifer, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickford, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prior, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irving, S. L.</span></span> <span> </span><span class="NLM_article-title">Design and Structure of Stapled Peptides Binding to Estrogen Receptors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">9696</span>– <span class="NLM_lpage">9699</span>, <span class="refDoi"> DOI: 10.1021/ja202946k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja202946k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntVems7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=9696-9699&author=C.+Phillipsauthor=L.+R.+Robertsauthor=M.+Schadeauthor=R.+Bazinauthor=A.+Bentauthor=N.+L.+Daviesauthor=R.+Mooreauthor=A.+D.+Panniferauthor=A.+R.+Pickfordauthor=S.+H.+Priorauthor=C.+M.+Readauthor=A.+Scottauthor=D.+G.+Brownauthor=B.+Xuauthor=S.+L.+Irving&title=Design+and+Structure+of+Stapled+Peptides+Binding+to+Estrogen+Receptors&doi=10.1021%2Fja202946k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Structure of Stapled Peptides Binding to Estrogen Receptors</span></div><div class="casAuthors">Phillips, Chris; Roberts, Lee R.; Schade, Markus; Bazin, Richard; Bent, Andrew; Davies, Nichola L.; Moore, Rob; Pannifer, Andrew D.; Pickford, Andrew R.; Prior, Stephen H.; Read, Christopher M.; Scott, Andrew; Brown, David G.; Xu, Bin; Irving, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">9696-9699</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Synthetic peptides that specifically bind nuclear hormone receptors offer an alternative approach to small mols. for the modulation of receptor signaling and subsequent gene expression.  Here the authors describe the design of a series of novel stapled peptides that bind the coactivator peptide site of estrogen receptors.  Using a no. of biophys. techniques, including crystal structure anal. of receptor-stapled peptide complexes, the authors describe in detail the mol. interactions and demonstrate that all-hydrocarbon staples modulate mol. recognition events.  The findings have implications for the design of stapled peptides in general.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-9uAz-tBuO7Vg90H21EOLACvtfcHk0lgs3Z6l5vuqiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntVems7s%253D&md5=0c80b8dc5c2c36188f93659b5d1a2d00</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fja202946k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja202946k%26sid%3Dliteratum%253Aachs%26aulast%3DPhillips%26aufirst%3DC.%26aulast%3DRoberts%26aufirst%3DL.%2BR.%26aulast%3DSchade%26aufirst%3DM.%26aulast%3DBazin%26aufirst%3DR.%26aulast%3DBent%26aufirst%3DA.%26aulast%3DDavies%26aufirst%3DN.%2BL.%26aulast%3DMoore%26aufirst%3DR.%26aulast%3DPannifer%26aufirst%3DA.%2BD.%26aulast%3DPickford%26aufirst%3DA.%2BR.%26aulast%3DPrior%26aufirst%3DS.%2BH.%26aulast%3DRead%26aufirst%3DC.%2BM.%26aulast%3DScott%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DXu%26aufirst%3DB.%26aulast%3DIrving%26aufirst%3DS.%2BL.%26atitle%3DDesign%2520and%2520Structure%2520of%2520Stapled%2520Peptides%2520Binding%2520to%2520Estrogen%2520Receptors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D9696%26epage%3D9699%26doi%3D10.1021%2Fja202946k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Magiera-Mularz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skalniak, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zak, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musielak, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudzinska-Szostak, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berlicki, Ł.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kocik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grudnik, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sala, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarganes-Tzitzikas, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaabani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dömling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holak, T. A.</span></span> <span> </span><span class="NLM_article-title">Bioactive Macrocyclic Inhibitors of the PD-1/PD-L1 Immune Checkpoint</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">13732</span>– <span class="NLM_lpage">13735</span>, <span class="refDoi"> DOI: 10.1002/anie.201707707</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fanie.201707707" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOit7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=13732-13735&author=K.+Magiera-Mularzauthor=L.+Skalniakauthor=K.+M.+Zakauthor=B.+Musielakauthor=E.+Rudzinska-Szostakauthor=%C5%81.+Berlickiauthor=J.+Kocikauthor=P.+Grudnikauthor=D.+Salaauthor=T.+Zarganes-Tzitzikasauthor=S.+Shaabaniauthor=A.+D%C3%B6mlingauthor=G.+Dubinauthor=T.+A.+Holak&title=Bioactive+Macrocyclic+Inhibitors+of+the+PD-1%2FPD-L1+Immune+Checkpoint&doi=10.1002%2Fanie.201707707"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactive Macrocyclic Inhibitors of the PD-1/PD-L1 Immune Checkpoint</span></div><div class="casAuthors">Magiera-Mularz, Katarzyna; Skalniak, Lukasz; Zak, Krzysztof M.; Musielak, Bogdan; Rudzinska-Szostak, Ewa; Berlicki, Lukasz; Kocik, Justyna; Grudnik, Przemyslaw; Sala, Dominik; Zarganes-Tzitzikas, Tryfon; Shaabani, Shabnam; Doemling, Alexander; Dubin, Grzegorz; Holak, Tad A.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">13732-13735</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Blockade of the immunoinhibitory PD-1/PD-L1 pathway using monoclonal antibodies has shown impressive results with durable clin. antitumor responses.  Anti-PD-1 and anti-PD-L1 antibodies have now been approved for the treatment of a no. of tumor types, whereas the development of small mols. targeting immune checkpoints lags far behind.  We characterized two classes of macrocyclic-peptide inhibitors directed at the PD-1/PD-L1 pathway.  We show that these macrocyclic compds. act by directly binding to PD-L1 and that they are capable of antagonizing PD-L1 signaling and, similarly to antibodies, can restore the function of T-cells.  We also provide the crystal structures of two of these small-mol. inhibitors bound to PD-L1.  The structures provide a rationale for the checkpoint inhibition by these small mols., and a description of their small mol./PD-L1 interfaces provides a blueprint for the design of small-mol. inhibitors of the PD-1/PD-L1 pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxuarSIlQEobVg90H21EOLACvtfcHk0liKW8iohVcMPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOit7%252FO&md5=b1634408007340ca0990d2607c96c1a9</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1002%2Fanie.201707707&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201707707%26sid%3Dliteratum%253Aachs%26aulast%3DMagiera-Mularz%26aufirst%3DK.%26aulast%3DSkalniak%26aufirst%3DL.%26aulast%3DZak%26aufirst%3DK.%2BM.%26aulast%3DMusielak%26aufirst%3DB.%26aulast%3DRudzinska-Szostak%26aufirst%3DE.%26aulast%3DBerlicki%26aufirst%3D%25C5%2581.%26aulast%3DKocik%26aufirst%3DJ.%26aulast%3DGrudnik%26aufirst%3DP.%26aulast%3DSala%26aufirst%3DD.%26aulast%3DZarganes-Tzitzikas%26aufirst%3DT.%26aulast%3DShaabani%26aufirst%3DS.%26aulast%3DD%25C3%25B6mling%26aufirst%3DA.%26aulast%3DDubin%26aufirst%3DG.%26aulast%3DHolak%26aufirst%3DT.%2BA.%26atitle%3DBioactive%2520Macrocyclic%2520Inhibitors%2520of%2520the%2520PD-1%252FPD-L1%2520Immune%2520Checkpoint%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D13732%26epage%3D13735%26doi%3D10.1002%2Fanie.201707707" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gurusaran, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shankar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagarajan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helliwell, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekar, K.</span></span> <span> </span><span class="NLM_article-title">Do We See What We Should See? Describing Non-Covalent Interactions in Protein Structures Including Precision</span>. <i>IUCrJ</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">74</span>– <span class="NLM_lpage">81</span>, <span class="refDoi"> DOI: 10.1107/s2052252513031485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1107%2FS2052252513031485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=25075321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotFWmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2014&pages=74-81&author=M.+Gurusaranauthor=M.+Shankarauthor=R.+Nagarajanauthor=J.+R.+Helliwellauthor=K.+Sekar&title=Do+We+See+What+We+Should+See%3F+Describing+Non-Covalent+Interactions+in+Protein+Structures+Including+Precision&doi=10.1107%2Fs2052252513031485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Do we see what we should see? Describing non-covalent interactions in protein structures including precision</span></div><div class="casAuthors">Gurusaran, Manickam; Shankar, Mani; Nagarajan, Raju; Helliwell, John R.; Sekar, Kanagaraj</div><div class="citationInfo"><span class="NLM_cas:title">IUCrJ</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">74-81</span>CODEN:
                <span class="NLM_cas:coden">IUCRAJ</span>;
        ISSN:<span class="NLM_cas:issn">2052-2525</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The power of X-ray crystal structure anal. as a technique is to 'see where the atoms are'.  The results are extensively used by a wide variety of research communities.  However, this 'seeing where the atoms are' can give a false sense of security unless the precision of the placement of the atoms has been taken into account.  Indeed, the presentation of bond distances and angles to a false precision (i.e. to too many decimal places) is commonplace.  This article has three themes.  Firstly, a basis for a proper representation of protein crystal structure results is detailed and demonstrated with respect to analyses of Protein Data Bank entries.  The basis for establishing the precision of placement of each atom in a protein crystal structure is non-trivial.  Secondly, a knowledge base harnessing such a descriptor of precision is presented.  It is applied here to the case of salt bridges, i.e. ion pairs, in protein structures; this is the most fundamental place to start with such structure-precision representations since salt bridges are one of the tenets of protein structure stability.  Ion pairs also play a central role in protein oligomerization, mol. recognition of ligands and substrates, allosteric regulation, domain motion and α-helix capping.  A new knowledge base, SBPS (Salt Bridges in Protein Structures), takes these structural precisions into account and is the first of its kind.  The third theme of the article is to indicate natural extensions of the need for such a description of precision, such as those involving metalloproteins and the detn. of the protonation states of ionizable amino acids.  Overall, it is also noted that this work and these examples are also relevant to protein three-dimensional structure mol. graphics software.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqEll24sceprVg90H21EOLACvtfcHk0liKW8iohVcMPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotFWmsA%253D%253D&md5=8ae721a59f55521ba14113362bbd059f</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1107%2FS2052252513031485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2052252513031485%26sid%3Dliteratum%253Aachs%26aulast%3DGurusaran%26aufirst%3DM.%26aulast%3DShankar%26aufirst%3DM.%26aulast%3DNagarajan%26aufirst%3DR.%26aulast%3DHelliwell%26aufirst%3DJ.%2BR.%26aulast%3DSekar%26aufirst%3DK.%26atitle%3DDo%2520We%2520See%2520What%2520We%2520Should%2520See%253F%2520Describing%2520Non-Covalent%2520Interactions%2520in%2520Protein%2520Structures%2520Including%2520Precision%26jtitle%3DIUCrJ%26date%3D2014%26volume%3D1%26spage%3D74%26epage%3D81%26doi%3D10.1107%2Fs2052252513031485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexov, E.</span></span> <span> </span><span class="NLM_article-title">On the Dielectric “Constant” of Proteins: Smooth Dielectric Function for Macromolecular Modeling and Its Implementation in DelPhi</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2136</span>, <span class="refDoi"> DOI: 10.1021/ct400065j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400065j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjvVKrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=2126-2136&author=L.+Liauthor=C.+Liauthor=Z.+Zhangauthor=E.+Alexov&title=On+the+Dielectric+%E2%80%9CConstant%E2%80%9D+of+Proteins%3A+Smooth+Dielectric+Function+for+Macromolecular+Modeling+and+Its+Implementation+in+DelPhi&doi=10.1021%2Fct400065j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">On the Dielectric "Constant" of Proteins: Smooth Dielectric Function for Macromolecular Modeling and Its Implementation in DelPhi</span></div><div class="casAuthors">Li, Lin; Li, Chuan; Zhang, Zhe; Alexov, Emil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2126-2136</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Implicit methods for modeling protein electrostatics require dielec. properties of the system to be known, in particular, the value of the dielec. const. of the protein.  While numerous values of the internal protein dielec. const. were reported in the literature, still there is no consensus of what the optimal value is.  Perhaps this is due to the fact that the protein dielec. const. is not a "const." but is a complex function reflecting the properties of the protein's structure and sequence.  Here, we report an implementation of a Gaussian-based approach to deliver the dielec. const. distribution throughout the protein and surrounding water phase by utilizing the 3D structure of the corresponding macromol.  In contrast to previous reports, we construct a smooth dielec. function throughout the space of the system to be modeled rather than just constructing a "Gaussian surface" or smoothing mol.-water boundary.  Anal. on a large set of proteins shows that (a) the av. dielec. const. inside the protein is relatively low, about 6-7, and reaches a value of about 20-30 at the protein's surface, and (b) high av. local dielec. const. values are assocd. with charged residues while low dielec. const. values are automatically assigned to the regions occupied by hydrophobic residues.  In terms of energetics, a benchmarking test was carried out against the exptl. pKa's of 89 residues in staphylococcal nuclease (SNase) and showed that it results in a much better RMSD (= 1.77 pK) than the corresponding calcns. done with a homogeneous high dielec. const. with an optimal value of 10 (RMSD = 2.43 pK).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuokYHiKExsrVg90H21EOLACvtfcHk0liKW8iohVcMPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjvVKrs7g%253D&md5=ce5218db0e9312a4e378239d340fe799</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fct400065j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400065j%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DAlexov%26aufirst%3DE.%26atitle%3DOn%2520the%2520Dielectric%2520%25E2%2580%259CConstant%25E2%2580%259D%2520of%2520Proteins%253A%2520Smooth%2520Dielectric%2520Function%2520for%2520Macromolecular%2520Modeling%2520and%2520Its%2520Implementation%2520in%2520DelPhi%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D2126%26epage%3D2136%26doi%3D10.1021%2Fct400065j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kendall, M. G.</span></span> <span> </span><span class="NLM_article-title">A New Measure of Rank Correlation</span>. <i>Biometrika</i> <span class="NLM_year" style="font-weight: bold;">1938</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">93</span>, <span class="refDoi"> DOI: 10.2307/2332226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1093%2Fbiomet%2F30.1-2.81" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1938&pages=81-93&author=M.+G.+Kendall&title=A+New+Measure+of+Rank+Correlation&doi=10.2307%2F2332226"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1093%2Fbiomet%2F30.1-2.81&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbiomet%252F30.1-2.81%26sid%3Dliteratum%253Aachs%26aulast%3DKendall%26aufirst%3DM.%2BG.%26atitle%3DA%2520New%2520Measure%2520of%2520Rank%2520Correlation%26jtitle%3DBiometrika%26date%3D1938%26volume%3D30%26spage%3D81%26epage%3D93%26doi%3D10.2307%2F2332226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, R.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucke, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Flexibility versus Rigidity for Orally Bioavailable Cyclic Hexapeptides</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">2289</span>– <span class="NLM_lpage">2293</span>, <span class="refDoi"> DOI: 10.1002/cbic.201500441</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1002%2Fcbic.201500441" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=26336864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Wjt73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=2289-2293&author=D.+S.+Nielsenauthor=R.-J.+Lohmanauthor=H.+N.+Hoangauthor=T.+A.+Hillauthor=A.+Jonesauthor=A.+J.+Luckeauthor=D.+P.+Fairlie&title=Flexibility+versus+Rigidity+for+Orally+Bioavailable+Cyclic+Hexapeptides&doi=10.1002%2Fcbic.201500441"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Flexibility versus Rigidity for Orally Bioavailable Cyclic Hexapeptides</span></div><div class="casAuthors">Nielsen, Daniel S.; Lohman, Rink-Jan; Hoang, Huy N.; Hill, Timothy A.; Jones, Alun; Lucke, Andrew J.; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2289-2293</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cyclic peptides and macrocycles have the potential to be membrane permeable and orally bioavailable, despite often not complying with the "rule of five" used in medicinal chem. to guide the discovery of oral drugs.  Here we compare solvent-dependent three-dimensional structures of three cyclic hexapeptides contg. D-amino acids, prolines, and intramol. hydrogen bonds.  Conformational rigidity rather than flexibility resulted in higher membrane permeability, metabolic stability and oral bioavailability, consistent with less polar surface exposure to solvent and a reduced entropy penalty for transition between polar and nonpolar environments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpArFfvUbUBA7Vg90H21EOLACvtfcHk0ljPievfZ6yxow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Wjt73I&md5=b0cf19a1ee875648d518dcc0befd9c28</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201500441&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201500441%26sid%3Dliteratum%253Aachs%26aulast%3DNielsen%26aufirst%3DD.%2BS.%26aulast%3DLohman%26aufirst%3DR.-J.%26aulast%3DHoang%26aufirst%3DH.%2BN.%26aulast%3DHill%26aufirst%3DT.%2BA.%26aulast%3DJones%26aufirst%3DA.%26aulast%3DLucke%26aufirst%3DA.%2BJ.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DFlexibility%2520versus%2520Rigidity%2520for%2520Orally%2520Bioavailable%2520Cyclic%2520Hexapeptides%26jtitle%3DChemBioChem%26date%3D2015%26volume%3D16%26spage%3D2289%26epage%3D2293%26doi%3D10.1002%2Fcbic.201500441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millhauser, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Testing the Conformational Hypothesis of Passive Membrane Permeability Using Synthetic Cyclic Peptide Diastereomers</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">2510</span>– <span class="NLM_lpage">2511</span>, <span class="refDoi"> DOI: 10.1021/ja0563455</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0563455" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVGqu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=2510-2511&author=T.+Rezaiauthor=B.+Yuauthor=G.+L.+Millhauserauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Testing+the+Conformational+Hypothesis+of+Passive+Membrane+Permeability+Using+Synthetic+Cyclic+Peptide+Diastereomers&doi=10.1021%2Fja0563455"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Testing the conformational hypothesis of passive membrane permeability using synthetic cyclic peptide diastereomers</span></div><div class="casAuthors">Rezai, Taha; Yu, Bin; Millhauser, Glenn L.; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2510-2511</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Little is known about the effect of conformation on passive membrane diffusion rates in small mols.  Evidence suggests that intramol. hydrogen bonding may play a role by reducing the energetic cost of desolvating hydrogen bond donors, esp. amide N-H groups.  We set out to test this hypothesis by investigating the passive membrane diffusion characteristics of a series of cyclic peptide diastereomers based on the sequence cyclo[Leu-Leu-Leu-Leu-Pro-Tyr].  We identified two cyclic hexapeptide diastereomers based on this sequence, whose membrane diffusion rates differed by nearly two log units.  Results of soln. NMR studies and hydrogen/deuterium (H/D) exchange expts. showed that membrane diffusion rates correlated with the degree of intramol. hydrogen bonding and H/D exchange rates.  The most permeable diastereomer, cyclo[D-Leu-D-Leu-Leu-D-Leu-Pro-Tyr], exhibited a passive membrane diffusion rate comparable to that of the orally available drug cyclosporine A.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqooDObTy1BXrVg90H21EOLACvtfcHk0ljPievfZ6yxow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVGqu7k%253D&md5=52e5cac6648efb70e743f294e3f8c4ec</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fja0563455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0563455%26sid%3Dliteratum%253Aachs%26aulast%3DRezai%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DMillhauser%26aufirst%3DG.%2BL.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DTesting%2520the%2520Conformational%2520Hypothesis%2520of%2520Passive%2520Membrane%2520Permeability%2520Using%2520Synthetic%2520Cyclic%2520Peptide%2520Diastereomers%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D2510%26epage%3D2511%26doi%3D10.1021%2Fja0563455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilliland, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindyalov, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">The Protein Data Bank</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lhl7pBQrMHm8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span> <i>The PyMOL Molecular
Graphics System</i>, Version 2.3.1; <span class="NLM_publisher-name">Schrödinger,
LLC</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+The+PyMOL+Molecular%0AGraphics+System%2C+Version+2.3.1%3B+Schr%C3%B6dinger%2C%0ALLC."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520PyMOL%2520Molecular%250AGraphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. N.</span></span> <span> </span><span class="NLM_article-title">Surflex: Fully Automatic Flexible Molecular Docking Using a Molecular Similarity-Based Search Engine</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">499</span>– <span class="NLM_lpage">511</span>, <span class="refDoi"> DOI: 10.1021/jm020406h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020406h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkvFKjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=499-511&author=A.+N.+Jain&title=Surflex%3A+Fully+Automatic+Flexible+Molecular+Docking+Using+a+Molecular+Similarity-Based+Search+Engine&doi=10.1021%2Fjm020406h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Surflex: fully automatic flexible molecular docking using a molecular similarity-based search engine</span></div><div class="casAuthors">Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">499-511</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Surflex is a fully automatic flexible mol. docking algorithm that combines the scoring function from the Hammerhead docking system with a search engine that relies on a surface-based mol. similarity method as a means to rapidly generate suitable putative poses for mol. fragments.  Results are presented evaluating reliability and accuracy of dockings compared with crystallog. exptl. results on 81 protein/ligand pairs of substantial structural diversity.  In over 80% of the complexes, Surflex's highest scoring docked pose was within 2.5 Å root-mean-square deviation (rmsd), with over 90% of the complexes having one of the top ranked poses within 2.5 Å rmsd.  Results are also presented assessing Surflex's utility as a screening tool on two protein targets (thymidine kinase and estrogen receptor) using data sets on which competing methods were run.  Performance of Surflex was significantly better, with true pos. rates of greater than 80% at false pos. rates of less than 1%.  Docking time was roughly linear in no. of rotatable bonds, beginning with a few seconds for rigid mols. and adding approx. 10 s per rotatable bond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrawn5WHbx5p7Vg90H21EOLACvtfcHk0lhl7pBQrMHm8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkvFKjsA%253D%253D&md5=67a08aafeadf69101d56b2f60a922359</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1021%2Fjm020406h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020406h%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DSurflex%253A%2520Fully%2520Automatic%2520Flexible%2520Molecular%2520Docking%2520Using%2520a%2520Molecular%2520Similarity-Based%2520Search%2520Engine%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D499%26epage%3D511%26doi%3D10.1021%2Fjm020406h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spitzer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. N.</span></span> <span> </span><span class="NLM_article-title">Surflex-Dock: Docking Benchmarks and Real-World Application</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">699</span>, <span class="refDoi"> DOI: 10.1007/s10822-011-9533-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1007%2Fs10822-011-9533-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=22569590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVelu7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=687-699&author=R.+Spitzerauthor=A.+N.+Jain&title=Surflex-Dock%3A+Docking+Benchmarks+and+Real-World+Application&doi=10.1007%2Fs10822-011-9533-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Surflex-Dock: Docking benchmarks and real-world application</span></div><div class="casAuthors">Spitzer, Russell; Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">687-699</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Benchmarks for mol. docking have historically focused on re-docking the cognate ligand of a well-detd. protein-ligand complex to measure geometric pose prediction accuracy, and measurement of virtual screening performance has been focused on increasingly large and diverse sets of target protein structures, cognate ligands, and various types of decoy sets.  Here, pose prediction is reported on the Astex Diverse set of 85 protein ligand complexes, and virtual screening performance is reported on the DUD set of 40 protein targets.  In both cases, prepd. structures of targets and ligands were provided by symposium organizers.  The re-prepd. data sets yielded results not significantly different than previous reports of Surflex-Dock on the two benchmarks.  Minor changes to protein coordinates resulting from complex pre-optimization had large effects on obsd. performance, highlighting the limitations of cognate ligand re-docking for pose prediction assessment.  Docking protocols developed for cross-docking, which address protein flexibility and produce discrete families of predicted poses, produced substantially better performance for pose prediction.  Performance on virtual screening performance was shown to benefit by employing and combining multiple screening methods: docking, 2D mol. similarity, and 3D mol. similarity.  In addn., use of multiple protein conformations significantly improved screening enrichment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPgES-yaoU4rVg90H21EOLACvtfcHk0lhl7pBQrMHm8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVelu7nN&md5=78637e767d2ad764f4b89c24ef3c7a56</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9533-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9533-y%26sid%3Dliteratum%253Aachs%26aulast%3DSpitzer%26aufirst%3DR.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DSurflex-Dock%253A%2520Docking%2520Benchmarks%2520and%2520Real-World%2520Application%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D687%26epage%3D699%26doi%3D10.1007%2Fs10822-011-9533-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleves, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. N.</span></span> <span> </span><span class="NLM_article-title">Knowledge-Guided Docking: Accurate Prospective Prediction of Bound Configurations of Novel Ligands Using Surflex-Dock</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">485</span>– <span class="NLM_lpage">509</span>, <span class="refDoi"> DOI: 10.1007/s10822-015-9846-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1007%2Fs10822-015-9846-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=25940276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnvVShsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=485-509&author=A.+E.+Clevesauthor=A.+N.+Jain&title=Knowledge-Guided+Docking%3A+Accurate+Prospective+Prediction+of+Bound+Configurations+of+Novel+Ligands+Using+Surflex-Dock&doi=10.1007%2Fs10822-015-9846-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Knowledge-guided docking: accurate prospective prediction of bound configurations of novel ligands using Surflex-Dock</span></div><div class="casAuthors">Cleves, Ann E.; Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">485-509</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Prediction of the bound configuration of small-mol. ligands that differ substantially from the cognate ligand of a protein co-crystal structure is much more challenging than re-docking the cognate ligand.  Success rates for cross-docking in the range of 20-30 % are common.  We present an approach that uses structural information known prior to a particular cutoff-date to make predictions on ligands whose bounds structures were detd. later.  The knowledge-guided docking protocol was tested on a set of ten protein targets using a total of 949 ligands.  The benchmark data set, called PINC ("PINC Is Not Cognate"), is publicly available.  Protein pocket similarity was used to choose representative structures for ensemble-docking.  The docking protocol made use of known ligand poses prior to the cutoff-date, both to help guide the configurational search and to adjust the rank of predicted poses.  Overall, the top-scoring pose family was correct over 60 % of the time, with the top-two pose families approaching a 75 % success rate.  Correct poses among all those predicted were identified nearly 90 % of the time.  The largest improvements came from the use of mol. similarity to improve ligand pose rankings and the strategy for identifying representative protein structures.  With the exception of a single outlier target, the knowledge-guided docking protocol produced results matching the quality of cognate-ligand re-docking, but it did so on a very challenging temporally-segregated cross-docking benchmark.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnNcQG8X--3bVg90H21EOLACvtfcHk0lhl7pBQrMHm8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnvVShsLo%253D&md5=b0deddab12f7289136703e2c4c75f266</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9846-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9846-3%26sid%3Dliteratum%253Aachs%26aulast%3DCleves%26aufirst%3DA.%2BE.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DKnowledge-Guided%2520Docking%253A%2520Accurate%2520Prospective%2520Prediction%2520of%2520Bound%2520Configurations%2520of%2520Novel%2520Ligands%2520Using%2520Surflex-Dock%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D485%26epage%3D509%26doi%3D10.1007%2Fs10822-015-9846-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pham, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. N.</span></span> <span> </span><span class="NLM_article-title">Customizing Scoring Functions for Docking</span>. <i>J. Comput. Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">286</span>, <span class="refDoi"> DOI: 10.1007/s10822-008-9174-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=10.1007%2Fs10822-008-9174-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=18273558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BD1cXksVKks7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=269-286&author=T.+A.+Phamauthor=A.+N.+Jain&title=Customizing+Scoring+Functions+for+Docking&doi=10.1007%2Fs10822-008-9174-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Customizing scoring functions for docking</span></div><div class="casAuthors">Pham, Tuan A.; Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">269-286</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Empirical scoring functions used in protein-ligand docking calcns. are typically trained on a dataset of complexes with known affinities with the aim of generalizing across different docking applications.  The authors report a novel method of scoring-function optimization that supports the use of addnl. information to constrain scoring function parameters, which can be used to focus a scoring function's training towards a particular application, such as screening enrichment.  The approach combines multiple instance learning, pos. data in the form of ligands of protein binding sites of known and unknown affinity and binding geometry, and neg. (decoy) data of ligands thought not to bind particular protein binding sites or known not to bind in particular geometries.  Performance of the method for the Surflex-Dock scoring function is shown in cross-validation studies and in eight blind test cases.  Tuned functions optimized with a sufficient amt. of data exhibited either improved or undiminished screening performance relative to the original function across all eight complexes.  Anal. of the changes to the scoring function suggest that modifications can be learned that are related to protein-specific features such as active-site mobility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ2udGn_HJLrVg90H21EOLACvtfcHk0liIX9WnLM5kMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXksVKks7k%253D&md5=ec0b2d3159f8970abcd431b085df0a7d</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1007%2Fs10822-008-9174-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-008-9174-y%26sid%3Dliteratum%253Aachs%26aulast%3DPham%26aufirst%3DT.%2BA.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DCustomizing%2520Scoring%2520Functions%2520for%2520Docking%26jtitle%3DJ.%2520Comput.%2520Aided%2520Mol.%2520Des.%26date%3D2008%26volume%3D22%26spage%3D269%26epage%3D286%26doi%3D10.1007%2Fs10822-008-9174-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleves, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, A. N.</span></span> <span> </span><span class="NLM_article-title">Structure- and Ligand-Based Virtual Screening on DUD-E+: Performance Dependence on Approximations to the Binding Pocket</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4296</span>– <span class="NLM_lpage">4310</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00115</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00115" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;key=1%3ACAS%3A528%3ADC%252BB3cXms1yjsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=4296-4310&author=A.+E.+Clevesauthor=A.+N.+Jain&title=Structure-+and+Ligand-Based+Virtual+Screening+on+DUD-E%2B%3A+Performance+Dependence+on+Approximations+to+the+Binding+Pocket&doi=10.1021%2Facs.jcim.0c00115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Structure- and Ligand-Based Virtual Screening on DUD-E+: Performance Dependence on Approximations to the Binding Pocket</span></div><div class="casAuthors">Cleves, Ann E.; Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4296-4310</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using the DUD-E+ benchmark, we explore the impact of using a single protein pocket or ligand for virtual screening compared with using ensembles of alternative pockets, ligands, and sets thereof.  For both structure-based and ligand-based approaches, the precise characterization of the binding site in question had a significant impact on screening performance.  Using the single original DUD-E protein, Surflex-Dock yielded mean ROC area of 0.81 ± 0.11.  Using the cognate ligand instead, with the eSim method for screening, yielded 0.77 ± 0.14.  Moving to ensembles of five protein pocket variants increased docking performance to 0.84 ± 0.09.  Results for the analogous ligand-based approach (using the five crystallog. aligned cognate ligands) was 0.83 ± 0.11.  Using the same ligands, but making use of an automatically generated mutual alignment, yielded mean AUC nearly as good as from single-structure docking: 0.80 ± 0.12.  Detailed results and statistical analyses show that structure- and ligand-based methods are complementary and can be fruitfully combined to enhance screening efficiency.  A hybrid approach combining ensemble docking with eSim-based screening produced the best and most consistent performance (mean ROC area of 0.89 ± 0.08 and 1% early enrichment of 46-fold).  Based on results from both the docking and ligand-similarity approaches, it is clearly unwise to make use of a single arbitrarily chosen protein structure for docking or single ligand query for similarity-based screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvrSodjvqflLVg90H21EOLACvtfcHk0liIX9WnLM5kMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXms1yjsLg%253D&md5=da167891883e8450776b3983053fb93a</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00115%26sid%3Dliteratum%253Aachs%26aulast%3DCleves%26aufirst%3DA.%2BE.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DStructure-%2520and%2520Ligand-Based%2520Virtual%2520Screening%2520on%2520DUD-E%252B%253A%2520Performance%2520Dependence%2520on%2520Approximations%2520to%2520the%2520Binding%2520Pocket%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D4296%26epage%3D4310%26doi%3D10.1021%2Facs.jcim.0c00115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DV1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DV1','PDB','3DV1'); return false;">PDB: 3DV1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O4Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O4Y','PDB','5O4Y'); return false;">PDB: 5O4Y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H0I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H0I','PDB','1H0I'); return false;">PDB: 1H0I</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MF8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MF8','PDB','1MF8'); return false;">PDB: 1MF8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZQW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZQW','PDB','4ZQW'); return false;">PDB: 4ZQW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWM','PDB','1VWM'); return false;">PDB: 1VWM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1C5F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1C5F','PDB','1C5F'); return false;">PDB: 1C5F</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X6S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X6S','PDB','4X6S'); return false;">PDB: 4X6S</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5B4W" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5B4W','PDB','5B4W'); return false;">PDB: 5B4W</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BM2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BM2','PDB','1BM2'); return false;">PDB: 1BM2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BZH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BZH','PDB','1BZH'); return false;">PDB: 1BZH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1E4W" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1E4W','PDB','1E4W'); return false;">PDB: 1E4W</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H0G" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H0G','PDB','1H0G'); return false;">PDB: 1H0G</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1IKF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1IKF','PDB','1IKF'); return false;">PDB: 1IKF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M63" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M63','PDB','1M63'); return false;">PDB: 1M63</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PCG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PCG','PDB','1PCG'); return false;">PDB: 1PCG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QNG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QNG','PDB','1QNG'); return false;">PDB: 1QNG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QNH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QNH','PDB','1QNH'); return false;">PDB: 1QNH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QR3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QR3','PDB','1QR3'); return false;">PDB: 1QR3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1URC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1URC','PDB','1URC'); return false;">PDB: 1URC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWB','PDB','1VWB'); return false;">PDB: 1VWB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWC','PDB','1VWC'); return false;">PDB: 1VWC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWD','PDB','1VWD'); return false;">PDB: 1VWD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWE','PDB','1VWE'); return false;">PDB: 1VWE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWF','PDB','1VWF'); return false;">PDB: 1VWF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWG','PDB','1VWG'); return false;">PDB: 1VWG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWH','PDB','1VWH'); return false;">PDB: 1VWH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWN','PDB','1VWN'); return false;">PDB: 1VWN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWO','PDB','1VWO'); return false;">PDB: 1VWO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VWP','PDB','1VWP'); return false;">PDB: 1VWP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1W9U" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1W9U','PDB','1W9U'); return false;">PDB: 1W9U</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1W9V" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1W9V','PDB','1W9V'); return false;">PDB: 1W9V</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1WAW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1WAW','PDB','1WAW'); return false;">PDB: 1WAW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1WB0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1WB0','PDB','1WB0'); return false;">PDB: 1WB0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1XQ7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1XQ7','PDB','1XQ7'); return false;">PDB: 1XQ7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YIT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YIT','PDB','1YIT'); return false;">PDB: 1YIT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2BCD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2BCD','PDB','2BCD'); return false;">PDB: 2BCD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2C9T" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2C9T','PDB','2C9T'); return false;">PDB: 2C9T</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CV3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2CV3','PDB','2CV3'); return false;">PDB: 2CV3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ESL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ESL','PDB','2ESL'); return false;">PDB: 2ESL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2F58" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2F58','PDB','2F58'); return false;">PDB: 2F58</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OJU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OJU','PDB','2OJU'); return false;">PDB: 2OJU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2POY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2POY','PDB','2POY'); return false;">PDB: 2POY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VDN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VDN','PDB','2VDN'); return false;">PDB: 2VDN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WFJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WFJ','PDB','2WFJ'); return false;">PDB: 2WFJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X2C" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X2C','PDB','2X2C'); return false;">PDB: 2X2C</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X7K" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2X7K','PDB','2X7K'); return false;">PDB: 2X7K</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z6W" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z6W','PDB','2Z6W'); return false;">PDB: 2Z6W</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AV9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AV9','PDB','3AV9'); return false;">PDB: 3AV9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVA','PDB','3AVA'); return false;">PDB: 3AVA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVB','PDB','3AVB'); return false;">PDB: 3AVB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVC','PDB','3AVC'); return false;">PDB: 3AVC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVF','PDB','3AVF'); return false;">PDB: 3AVF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVG','PDB','3AVG'); return false;">PDB: 3AVG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVH','PDB','3AVH'); return false;">PDB: 3AVH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVI','PDB','3AVI'); return false;">PDB: 3AVI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVJ','PDB','3AVJ'); return false;">PDB: 3AVJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVK','PDB','3AVK'); return false;">PDB: 3AVK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVL','PDB','3AVL'); return false;">PDB: 3AVL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVM','PDB','3AVM'); return false;">PDB: 3AVM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVN','PDB','3AVN'); return false;">PDB: 3AVN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BO7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BO7','PDB','3BO7'); return false;">PDB: 3BO7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EOV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EOV','PDB','3EOV'); return false;">PDB: 3EOV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MGN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MGN','PDB','3MGN'); return false;">PDB: 3MGN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OE0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OE0','PDB','3OE0'); return false;">PDB: 3OE0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3P8F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3P8F','PDB','3P8F'); return false;">PDB: 3P8F</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PMP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PMP','PDB','3PMP'); return false;">PDB: 3PMP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP4','PDB','3PP4'); return false;">PDB: 3PP4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PQZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PQZ','PDB','3PQZ'); return false;">PDB: 3PQZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QN7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QN7','PDB','3QN7'); return false;">PDB: 3QN7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VVR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VVR','PDB','3VVR'); return false;">PDB: 3VVR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VVS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VVS','PDB','3VVS'); return false;">PDB: 3VVS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WBN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WBN','PDB','3WBN'); return false;">PDB: 3WBN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WDC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WDC','PDB','3WDC'); return false;">PDB: 3WDC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WDD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WDD','PDB','3WDD'); return false;">PDB: 3WDD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WDE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WDE','PDB','3WDE'); return false;">PDB: 3WDE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WMG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WMG','PDB','3WMG'); return false;">PDB: 3WMG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WNE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WNE','PDB','3WNE'); return false;">PDB: 3WNE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WNF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WNF','PDB','3WNF'); return false;">PDB: 3WNF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WNG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WNG','PDB','3WNG'); return false;">PDB: 3WNG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WNH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WNH','PDB','3WNH'); return false;">PDB: 3WNH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DGC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DGC','PDB','4DGC'); return false;">PDB: 4DGC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GLY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GLY','PDB','4GLY'); return false;">PDB: 4GLY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GVU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GVU','PDB','4GVU'); return false;">PDB: 4GVU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HGC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HGC','PDB','4HGC'); return false;">PDB: 4HGC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HY7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HY7','PDB','4HY7'); return false;">PDB: 4HY7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IB5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IB5','PDB','4IB5'); return false;">PDB: 4IB5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IPZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IPZ','PDB','4IPZ'); return false;">PDB: 4IPZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JJM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JJM','PDB','4JJM'); return false;">PDB: 4JJM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K8Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K8Y','PDB','4K8Y'); return false;">PDB: 4K8Y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KEL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KEL','PDB','4KEL'); return false;">PDB: 4KEL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTS','PDB','4KTS'); return false;">PDB: 4KTS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KTU','PDB','4KTU'); return false;">PDB: 4KTU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L3O" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L3O','PDB','4L3O'); return false;">PDB: 4L3O</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4M1D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4M1D','PDB','4M1D'); return false;">PDB: 4M1D</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MNX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MNX','PDB','4MNX'); return false;">PDB: 4MNX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MNY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MNY','PDB','4MNY'); return false;">PDB: 4MNY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MQ9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MQ9','PDB','4MQ9'); return false;">PDB: 4MQ9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OIN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OIN','PDB','4OIN'); return false;">PDB: 4OIN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OIP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OIP','PDB','4OIP'); return false;">PDB: 4OIP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OIQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OIQ','PDB','4OIQ'); return false;">PDB: 4OIQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OIR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4OIR','PDB','4OIR'); return false;">PDB: 4OIR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TOT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TOT','PDB','4TOT'); return false;">PDB: 4TOT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4U0G" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4U0G','PDB','4U0G'); return false;">PDB: 4U0G</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4W4Z" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4W4Z','PDB','4W4Z'); return false;">PDB: 4W4Z</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4W50" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4W50','PDB','4W50'); return false;">PDB: 4W50</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X1N" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X1N','PDB','4X1N'); return false;">PDB: 4X1N</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X1Q" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X1Q','PDB','4X1Q'); return false;">PDB: 4X1Q</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X1R" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X1R','PDB','4X1R'); return false;">PDB: 4X1R</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X1S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4X1S','PDB','4X1S'); return false;">PDB: 4X1S</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YV9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YV9','PDB','4YV9'); return false;">PDB: 4YV9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Z0F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Z0F','PDB','4Z0F'); return false;">PDB: 4Z0F</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZJX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZJX','PDB','4ZJX'); return false;">PDB: 4ZJX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AGU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AGU','PDB','5AGU'); return false;">PDB: 5AGU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CS2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CS2','PDB','5CS2'); return false;">PDB: 5CS2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EOC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EOC','PDB','5EOC'); return false;">PDB: 5EOC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ETU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ETU','PDB','5ETU'); return false;">PDB: 5ETU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EUK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EUK','PDB','5EUK'); return false;">PDB: 5EUK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5F88" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5F88','PDB','5F88'); return false;">PDB: 5F88</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FF6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FF6','PDB','5FF6'); return false;">PDB: 5FF6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GLH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GLH','PDB','5GLH'); return false;">PDB: 5GLH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I2I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I2I','PDB','5I2I'); return false;">PDB: 5I2I</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IOP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IOP','PDB','5IOP'); return false;">PDB: 5IOP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IR1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IR1','PDB','5IR1'); return false;">PDB: 5IR1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ITF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ITF','PDB','5ITF'); return false;">PDB: 5ITF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IV2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IV2','PDB','5IV2'); return false;">PDB: 5IV2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IVZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IVZ','PDB','5IVZ'); return false;">PDB: 5IVZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JR2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JR2','PDB','5JR2'); return false;">PDB: 5JR2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KEZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KEZ','PDB','5KEZ'); return false;">PDB: 5KEZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KGN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KGN','PDB','5KGN'); return false;">PDB: 5KGN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LRG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LRG','PDB','5LRG'); return false;">PDB: 5LRG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LRJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LRJ','PDB','5LRJ'); return false;">PDB: 5LRJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LRK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LRK','PDB','5LRK'); return false;">PDB: 5LRK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LY1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LY1','PDB','5LY1'); return false;">PDB: 5LY1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LY2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LY2','PDB','5LY2'); return false;">PDB: 5LY2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N99" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N99','PDB','5N99'); return false;">PDB: 5N99</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NES" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NES','PDB','5NES'); return false;">PDB: 5NES</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NEY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NEY','PDB','5NEY'); return false;">PDB: 5NEY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NF0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NF0','PDB','5NF0'); return false;">PDB: 5NF0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O45" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5O45','PDB','5O45'); return false;">PDB: 5O45</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T1K" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T1K','PDB','5T1K'); return false;">PDB: 5T1K</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T1L" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T1L','PDB','5T1L'); return false;">PDB: 5T1L</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T1M" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T1M','PDB','5T1M'); return false;">PDB: 5T1M</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TH2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TH2','PDB','5TH2'); return false;">PDB: 5TH2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VBL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VBL','PDB','5VBL'); return false;">PDB: 5VBL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VFC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VFC','PDB','5VFC'); return false;">PDB: 5VFC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VI6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VI6','PDB','5VI6'); return false;">PDB: 5VI6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W5S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W5S','PDB','5W5S'); return false;">PDB: 5W5S</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W5U" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W5U','PDB','5W5U'); return false;">PDB: 5W5U</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W6I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W6I','PDB','5W6I'); return false;">PDB: 5W6I</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W6R" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W6R','PDB','5W6R'); return false;">PDB: 5W6R</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W6T" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5W6T','PDB','5W6T'); return false;">PDB: 5W6T</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XCO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XCO','PDB','5XCO'); return false;">PDB: 5XCO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XN3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XN3','PDB','5XN3'); return false;">PDB: 5XN3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B67" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B67','PDB','6B67'); return false;">PDB: 6B67</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C1Q" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C1Q','PDB','6C1Q'); return false;">PDB: 6C1Q</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C1R" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C1R','PDB','6C1R'); return false;">PDB: 6C1R</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=13GS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=13GS','PDB','13GS'); return false;">PDB: 13GS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1A4K" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1A4K','PDB','1A4K'); return false;">PDB: 1A4K</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1A8T" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1A8T','PDB','1A8T'); return false;">PDB: 1A8T</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1AFQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1AFQ','PDB','1AFQ'); return false;">PDB: 1AFQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ATL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ATL','PDB','1ATL'); return false;">PDB: 1ATL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1AZM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1AZM','PDB','1AZM'); return false;">PDB: 1AZM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BR6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BR6','PDB','1BR6'); return false;">PDB: 1BR6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1CET" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1CET','PDB','1CET'); return false;">PDB: 1CET</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1DIB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1DIB','PDB','1DIB'); return false;">PDB: 1DIB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1EVE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1EVE','PDB','1EVE'); return false;">PDB: 1EVE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1EZQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1EZQ','PDB','1EZQ'); return false;">PDB: 1EZQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1F0R" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1F0R','PDB','1F0R'); return false;">PDB: 1F0R</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1F0T" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1F0T','PDB','1F0T'); return false;">PDB: 1F0T</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FCX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FCX','PDB','1FCX'); return false;">PDB: 1FCX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FCZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FCZ','PDB','1FCZ'); return false;">PDB: 1FCZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FJS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FJS','PDB','1FJS'); return false;">PDB: 1FJS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FM6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FM6','PDB','1FM6'); return false;">PDB: 1FM6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FM9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FM9','PDB','1FM9'); return false;">PDB: 1FM9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1GWX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1GWX','PDB','1GWX'); return false;">PDB: 1GWX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H1P" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H1P','PDB','1H1P'); return false;">PDB: 1H1P</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H1S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H1S','PDB','1H1S'); return false;">PDB: 1H1S</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1I7Z" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1I7Z','PDB','1I7Z'); return false;">PDB: 1I7Z</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1IY7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1IY7','PDB','1IY7'); return false;">PDB: 1IY7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1K1J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1K1J','PDB','1K1J'); return false;">PDB: 1K1J</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1K7E" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1K7E','PDB','1K7E'); return false;">PDB: 1K7E</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1K7F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1K7F','PDB','1K7F'); return false;">PDB: 1K7F</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LQD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1LQD','PDB','1LQD'); return false;">PDB: 1LQD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M48" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1M48','PDB','1M48'); return false;">PDB: 1M48</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MQ5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MQ5','PDB','1MQ5'); return false;">PDB: 1MQ5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MQ6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MQ6','PDB','1MQ6'); return false;">PDB: 1MQ6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NHU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NHU','PDB','1NHU'); return false;">PDB: 1NHU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NHV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NHV','PDB','1NHV'); return false;">PDB: 1NHV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QHI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1QHI','PDB','1QHI'); return false;">PDB: 1QHI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1THL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1THL','PDB','1THL'); return false;">PDB: 1THL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YDR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YDR','PDB','1YDR'); return false;">PDB: 1YDR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YDS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YDS','PDB','1YDS'); return false;">PDB: 1YDS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YDT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YDT','PDB','1YDT'); return false;">PDB: 1YDT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PCP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PCP','PDB','2PCP'); return false;">PDB: 2PCP</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i22"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02159">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_75834"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c02159?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02159</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">High-level overview of available data archive (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02159/suppl_file/jm0c02159_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02159/suppl_file/jm0c02159_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02159/suppl_file/jm0c02159_si_001.pdf">jm0c02159_si_001.pdf (60.08 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02159/suppl_file/jm0c02159_si_002.csv">jm0c02159_si_002.csv (86.19 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02159&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-6%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02159%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02159" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                10MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67991d8d094f24c3","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
